<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[2208.03392] Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions</title><meta property="og:description" content="With the advent of the  Internet of Things (IoT),  Artificial Intelligence (AI),  Machine Learning (ML), and  Deep Learning (DL) algorithms, the landscape of data-driven medical applications has emerged as a promising …">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/2208.03392">

<!--Generated on Wed Mar 13 18:05:04 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="keywords" lang="en" content="
Artificial Intelligence,  Communication,  Data Privacy,  Edge Computing,  Federated Learning,  Foundational Model,  Large Language Model,  Medical Applications,  Security.
">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<h1 class="ltx_title ltx_title_document">Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Ashish Rauniyar,  Desta Haileselassie Hagos,  Debesh Jha,  Jan Erik Håkegård,  Ulas Bagci,  Danda B. Rawat,  Vladimir Vlassov
</span><span class="ltx_author_notes">A. Rauniyar and J. E. Håkegård are with Sustainable Communication Technologies (SCT), SINTEF Digital, Trondheim, 7034, Norway (e-mail: ashish.rauniyar@sintef.no; jan.e.hakegard@sintef.no).D. H. Hagos and DB. Rawat are with the DoD Center of Excellence in Artificial Intelligence and Machine Learning (CoE-AIML), College of Engineering and Architecture (CEA), Department of Electrical Engineering and Computer Science, Howard University, Washington DC, USA (e-mail: desta.hagos@howard.edu; danda.rawat@howard.edu).D. Jha and U. Bagci are with Machine &amp; Hybrid Intelligence Lab, Department of Radiology, Northwestern University, USA (e-mail: debesh.jha@northwestern.edu; ulas.bagci@northwestern.edu).V. Vlassov is with the Department of Computer Science, School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology, Stockholm, Sweden (e-mail: vladv@kth.se).This research work was supported by the COPS (Comprehensive Privacy and Security for Resilient CPS/IoT) project funded by the Research Council of Norway under project number:300102.Accepted for publication in IEEE Internet of Things Journal.</span></span>
</div>

<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p id="id8.id1" class="ltx_p">With the advent of the  <span title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Internet of Things</span></span> (<abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr>),  <span title="Artificial Intelligence" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Artificial Intelligence</span></span> (<abbr title="Artificial Intelligence" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">AI</span></abbr>),  <span title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Machine Learning</span></span> (<abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr>), and  <span title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Deep Learning</span></span> (<abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr>) algorithms, the landscape of data-driven medical applications has emerged as a promising avenue for designing robust and scalable diagnostic and prognostic models from medical data. This has gained a lot of attention from both academia and industry, leading to significant improvements in healthcare quality. However, the adoption of AI-driven medical applications still faces tough challenges, including meeting security, privacy, and quality of service (QoS) standards. Recent developments in  <span title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Federated Learning</span></span> (<abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr>) have made it possible to train complex machine-learned models in a distributed manner and has become an active research domain, particularly processing the medical data at the edge of the network in a decentralized way to preserve privacy and address security concerns. To this end, in this paper, we explore the present and future of FL technology in medical applications where data sharing is a significant challenge. We delve into the current research trends and their outcomes, unravelling the complexities of designing reliable and scalable <abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr> models. Our paper outlines the fundamental statistical issues in FL, tackles device-related problems, addresses security challenges, and navigates the complexity of privacy concerns, all while highlighting its transformative potential in the medical field. Our study primarily focuses on medical applications of <abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr>, particularly in the context of global cancer diagnosis. We highlight the potential of FL to enable computer-aided diagnosis tools that address this challenge with greater effectiveness than traditional data-driven methods. Recent literature has shown that FL models are robust and generalize well to new data, which is essential for medical applications. We hope that this comprehensive review will serve as a checkpoint for the field, summarizing the current state-of-the-art and identifying open problems and future research directions.</p>
</div>
<div class="ltx_keywords">
<h6 class="ltx_title ltx_title_keywords">Index Terms: </h6>
Artificial Intelligence, Communication, Data Privacy, Edge Computing, Federated Learning, Foundational Model, Large Language Model, Medical Applications, Security.

</div>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">I </span><span id="S1.1.1" class="ltx_text ltx_font_smallcaps">Introduction</span>
</h2>

<div id="S1.p1" class="ltx_para">
<p id="S1.p1.1" class="ltx_p"><span id="S1.p1.1.1" class="ltx_text" style="color:#000000;">The rapid advancements in modern technology, particularly within the domains of the <abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr> and <abbr title="Artificial Intelligence" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">AI</span></abbr>, have catalyzed significant improvements in healthcare, enhancing both the quality and longevity of human life <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib1" title="" class="ltx_ref">1</a>, <a href="#bib.bib2" title="" class="ltx_ref">2</a>, <a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite>. An illustrative example of this transformative potential arises from the collaboration of researchers from Google Inc. and affiliated institutions, wherein they showcased the remarkable potential of AI <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib4" title="" class="ltx_ref">4</a>]</cite>. The study achieved diagnostic accuracy akin to physicians by training an AI system on an extensive dataset comprising tens of thousands of images. This proficiency extended to the precise identification of referable diabetic retinopathy while also revealing hitherto unrecognized correlations between distinct image patterns in fundus photographs and cardiovascular risk factors. Further innovation surfaces through ingeniously devised AI-powered <abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr> systems, detailed in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite>, which meticulously track bodily movements, temperature fluctuations, and acoustic signals. This pioneering approach holds immense promise, particularly in promptly identifying the onset of epileptic seizures. Notably, these technological breakthroughs have been pivotal in addressing the challenges posed by the COVID-19 pandemic, where AI and IoT-driven solutions have played a crucial role in medical treatment and healthcare support <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib6" title="" class="ltx_ref">6</a>]</cite>. Amidst this backdrop, medical data, including imaging and pathological data, are harnessed to train AI models capable of accurately identifying and predicting diseases, thereby augmenting the efficacy of healthcare diagnostics.</span></p>
</div>
<div id="S1.p2" class="ltx_para ltx_indent">
<p id="S1.p2.1" class="ltx_p"><span id="S1.p2.1.1" class="ltx_text" style="color:#000000;">The paradigm of data-driven medical applications has emerged as a promising avenue for creating robust and scalable predictive algorithms from medical data, garnering attention across academic and industrial circles  </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.p2.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib7" title="" class="ltx_ref">7</a><span id="S1.p2.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S1.p2.1.4" class="ltx_text" style="color:#000000;">. Although these applications have undeniably improved healthcare quality </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.p2.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib8" title="" class="ltx_ref">8</a><span id="S1.p2.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S1.p2.1.7" class="ltx_text" style="color:#000000;">, their adoption is hindered by challenges related to security, privacy, and  </span><span title="Quality of Service" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Quality of Service</span></span><span id="S1.p2.1.8" class="ltx_text" style="color:#000000;"> (</span><abbr title="Quality of Service" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">QoS</span></abbr><span id="S1.p2.1.9" class="ltx_text" style="color:#000000;">). In particular, concerns about data privacy loom large, with data owners, such as individuals, patients, and hospitals, apprehensive about sharing sensitive information with external entities. This apprehension has spurred the enactment of stringent legislation, exemplified by the United States Consumer Privacy Bill of Rights and the European Commission’s  </span><span title="General Data Protection Regulation" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">General Data Protection Regulation</span></span><span id="S1.p2.1.10" class="ltx_text" style="color:#000000;"> (</span><abbr title="General Data Protection Regulation" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">GDPR</span></abbr><span id="S1.p2.1.11" class="ltx_text" style="color:#000000;">), aimed at safeguarding user privacy </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.p2.1.12.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib9" title="" class="ltx_ref">9</a>, <a href="#bib.bib10" title="" class="ltx_ref">10</a>, <a href="#bib.bib11" title="" class="ltx_ref">11</a>, <a href="#bib.bib12" title="" class="ltx_ref">12</a><span id="S1.p2.1.13.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S1.p2.1.14" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<figure id="S1.F1" class="ltx_figure"><img src="/html/2208.03392/assets/x1.png" id="S1.F1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="499" height="282" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_figure"><span id="S1.F1.4.1.1" class="ltx_text" style="font-size:90%;">Figure 1</span>: </span><span id="S1.F1.5.2" class="ltx_text" style="font-size:90%;">Taxonomy of the topics covered in this work.</span></figcaption>
</figure>
<figure id="S1.T1" class="ltx_table">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_table"><span id="S1.T1.4.1.1" class="ltx_text" style="font-size:90%;">TABLE I</span>: </span><span id="S1.T1.5.2" class="ltx_text" style="font-size:90%;">Abbreviations used in our paper.</span></figcaption>
<table id="S1.T1.6" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S1.T1.6.1.1" class="ltx_tr">
<th id="S1.T1.6.1.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.1.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Acronym</span></th>
<th id="S1.T1.6.1.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.1.1.2.1" class="ltx_text ltx_font_bold" style="color:#000000;">Definition</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S1.T1.6.2.1" class="ltx_tr">
<td id="S1.T1.6.2.1.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.2.1.1.1" class="ltx_text" style="color:#000000;">AI</span></td>
<td id="S1.T1.6.2.1.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.2.1.2.1" class="ltx_text" style="color:#000000;">Artificial Intelligence</span></td>
</tr>
<tr id="S1.T1.6.3.2" class="ltx_tr">
<td id="S1.T1.6.3.2.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.3.2.1.1" class="ltx_text" style="color:#000000;">ASD</span></td>
<td id="S1.T1.6.3.2.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.3.2.2.1" class="ltx_text" style="color:#000000;">Autism Spectrum Disorder</span></td>
</tr>
<tr id="S1.T1.6.4.3" class="ltx_tr">
<td id="S1.T1.6.4.3.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.4.3.1.1" class="ltx_text" style="color:#000000;">CFL</span></td>
<td id="S1.T1.6.4.3.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.4.3.2.1" class="ltx_text" style="color:#000000;">Clustered Federated Learning</span></td>
</tr>
<tr id="S1.T1.6.5.4" class="ltx_tr">
<td id="S1.T1.6.5.4.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.5.4.1.1" class="ltx_text" style="color:#000000;">CILL</span></td>
<td id="S1.T1.6.5.4.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.5.4.2.1" class="ltx_text" style="color:#000000;">Cyclic Institutional Incremental Learning</span></td>
</tr>
<tr id="S1.T1.6.6.5" class="ltx_tr">
<td id="S1.T1.6.6.5.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.6.5.1.1" class="ltx_text" style="color:#000000;">CNN</span></td>
<td id="S1.T1.6.6.5.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.6.5.2.1" class="ltx_text" style="color:#000000;">Convolutional Neural Networks</span></td>
</tr>
<tr id="S1.T1.6.7.6" class="ltx_tr">
<td id="S1.T1.6.7.6.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.7.6.1.1" class="ltx_text" style="color:#000000;">CPU</span></td>
<td id="S1.T1.6.7.6.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.7.6.2.1" class="ltx_text" style="color:#000000;">Central Processing Unit</span></td>
</tr>
<tr id="S1.T1.6.8.7" class="ltx_tr">
<td id="S1.T1.6.8.7.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.8.7.1.1" class="ltx_text" style="color:#000000;">CT</span></td>
<td id="S1.T1.6.8.7.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.8.7.2.1" class="ltx_text" style="color:#000000;">Computed Tomography</span></td>
</tr>
<tr id="S1.T1.6.9.8" class="ltx_tr">
<td id="S1.T1.6.9.8.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.9.8.1.1" class="ltx_text" style="color:#000000;">DL</span></td>
<td id="S1.T1.6.9.8.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.9.8.2.1" class="ltx_text" style="color:#000000;">Deep Learning</span></td>
</tr>
<tr id="S1.T1.6.10.9" class="ltx_tr">
<td id="S1.T1.6.10.9.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.10.9.1.1" class="ltx_text" style="color:#000000;">DP</span></td>
<td id="S1.T1.6.10.9.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.10.9.2.1" class="ltx_text" style="color:#000000;">Differential Privacy</span></td>
</tr>
<tr id="S1.T1.6.11.10" class="ltx_tr">
<td id="S1.T1.6.11.10.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.11.10.1.1" class="ltx_text" style="color:#000000;">Edge-FL</span></td>
<td id="S1.T1.6.11.10.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.11.10.2.1" class="ltx_text" style="color:#000000;">Edge Computing-based Federated Learning</span></td>
</tr>
<tr id="S1.T1.6.12.11" class="ltx_tr">
<td id="S1.T1.6.12.11.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.12.11.1.1" class="ltx_text" style="color:#000000;">EHR</span></td>
<td id="S1.T1.6.12.11.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.12.11.2.1" class="ltx_text" style="color:#000000;">Electronic Health Record</span></td>
</tr>
<tr id="S1.T1.6.13.12" class="ltx_tr">
<td id="S1.T1.6.13.12.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.13.12.1.1" class="ltx_text" style="color:#000000;">EMRs</span></td>
<td id="S1.T1.6.13.12.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.13.12.2.1" class="ltx_text" style="color:#000000;">Electronic Medical Records</span></td>
</tr>
<tr id="S1.T1.6.14.13" class="ltx_tr">
<td id="S1.T1.6.14.13.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.14.13.1.1" class="ltx_text" style="color:#000000;">FADL</span></td>
<td id="S1.T1.6.14.13.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.14.13.2.1" class="ltx_text" style="color:#000000;">Federated Autonomous Deep Learning</span></td>
</tr>
<tr id="S1.T1.6.15.14" class="ltx_tr">
<td id="S1.T1.6.15.14.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.15.14.1.1" class="ltx_text" style="color:#000000;">FAIR</span></td>
<td id="S1.T1.6.15.14.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.15.14.2.1" class="ltx_text" style="color:#000000;">Findable, Accessible, Interoperable, Reusable</span></td>
</tr>
<tr id="S1.T1.6.16.15" class="ltx_tr">
<td id="S1.T1.6.16.15.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.16.15.1.1" class="ltx_text" style="color:#000000;">FATE</span></td>
<td id="S1.T1.6.16.15.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.16.15.2.1" class="ltx_text" style="color:#000000;">Federated AI Technology Enabler Framework</span></td>
</tr>
<tr id="S1.T1.6.17.16" class="ltx_tr">
<td id="S1.T1.6.17.16.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.17.16.1.1" class="ltx_text" style="color:#000000;">FATHOM</span></td>
<td id="S1.T1.6.17.16.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.17.16.2.1" class="ltx_text" style="color:#000000;">Federated Multi-task Hierarchical Attention Mode</span></td>
</tr>
<tr id="S1.T1.6.18.17" class="ltx_tr">
<td id="S1.T1.6.18.17.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.18.17.1.1" class="ltx_text" style="color:#000000;">FedAvg</span></td>
<td id="S1.T1.6.18.17.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.18.17.2.1" class="ltx_text" style="color:#000000;">Federated Averaging</span></td>
</tr>
<tr id="S1.T1.6.19.18" class="ltx_tr">
<td id="S1.T1.6.19.18.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.19.18.1.1" class="ltx_text" style="color:#000000;">FL</span></td>
<td id="S1.T1.6.19.18.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.19.18.2.1" class="ltx_text" style="color:#000000;">Federated Learning</span></td>
</tr>
<tr id="S1.T1.6.20.19" class="ltx_tr">
<td id="S1.T1.6.20.19.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.20.19.1.1" class="ltx_text" style="color:#000000;">FML</span></td>
<td id="S1.T1.6.20.19.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.20.19.2.1" class="ltx_text" style="color:#000000;">Federated Machine Learning</span></td>
</tr>
<tr id="S1.T1.6.21.20" class="ltx_tr">
<td id="S1.T1.6.21.20.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.21.20.1.1" class="ltx_text" style="color:#000000;">FDL</span></td>
<td id="S1.T1.6.21.20.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.21.20.2.1" class="ltx_text" style="color:#000000;">Federated Deep Learning</span></td>
</tr>
<tr id="S1.T1.6.22.21" class="ltx_tr">
<td id="S1.T1.6.22.21.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.22.21.1.1" class="ltx_text" style="color:#000000;">FMs</span></td>
<td id="S1.T1.6.22.21.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.22.21.2.1" class="ltx_text" style="color:#000000;">Foundational Models</span></td>
</tr>
<tr id="S1.T1.6.23.22" class="ltx_tr">
<td id="S1.T1.6.23.22.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.23.22.1.1" class="ltx_text" style="color:#000000;">GCAE</span></td>
<td id="S1.T1.6.23.22.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.23.22.2.1" class="ltx_text" style="color:#000000;">Generative Convolutional Autoencoder</span></td>
</tr>
<tr id="S1.T1.6.24.23" class="ltx_tr">
<td id="S1.T1.6.24.23.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.24.23.1.1" class="ltx_text" style="color:#000000;">GDPR</span></td>
<td id="S1.T1.6.24.23.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.24.23.2.1" class="ltx_text" style="color:#000000;">General Data Protection Regulation</span></td>
</tr>
<tr id="S1.T1.6.25.24" class="ltx_tr">
<td id="S1.T1.6.25.24.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.25.24.1.1" class="ltx_text" style="color:#000000;">GPU</span></td>
<td id="S1.T1.6.25.24.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.25.24.2.1" class="ltx_text" style="color:#000000;">Graphics Processing Unit</span></td>
</tr>
<tr id="S1.T1.6.26.25" class="ltx_tr">
<td id="S1.T1.6.26.25.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.26.25.1.1" class="ltx_text" style="color:#000000;">HIPAA</span></td>
<td id="S1.T1.6.26.25.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.26.25.2.1" class="ltx_text" style="color:#000000;">Health Insurance Portability and Accountability Act</span></td>
</tr>
<tr id="S1.T1.6.27.26" class="ltx_tr">
<td id="S1.T1.6.27.26.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.27.26.1.1" class="ltx_text" style="color:#000000;">I.I.D</span></td>
<td id="S1.T1.6.27.26.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.27.26.2.1" class="ltx_text" style="color:#000000;">Identical and Independently Distributed</span></td>
</tr>
<tr id="S1.T1.6.28.27" class="ltx_tr">
<td id="S1.T1.6.28.27.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.28.27.1.1" class="ltx_text" style="color:#000000;">IoMT</span></td>
<td id="S1.T1.6.28.27.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.28.27.2.1" class="ltx_text" style="color:#000000;">Internet of Medical Things</span></td>
</tr>
<tr id="S1.T1.6.29.28" class="ltx_tr">
<td id="S1.T1.6.29.28.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.29.28.1.1" class="ltx_text" style="color:#000000;">IoT</span></td>
<td id="S1.T1.6.29.28.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.29.28.2.1" class="ltx_text" style="color:#000000;">Internet of Things</span></td>
</tr>
<tr id="S1.T1.6.30.29" class="ltx_tr">
<td id="S1.T1.6.30.29.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.30.29.1.1" class="ltx_text" style="color:#000000;">IPFS</span></td>
<td id="S1.T1.6.30.29.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.30.29.2.1" class="ltx_text" style="color:#000000;">Interplanetary File System</span></td>
</tr>
<tr id="S1.T1.6.31.30" class="ltx_tr">
<td id="S1.T1.6.31.30.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.31.30.1.1" class="ltx_text" style="color:#000000;">MEC</span></td>
<td id="S1.T1.6.31.30.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.31.30.2.1" class="ltx_text" style="color:#000000;">Mobile Edge Computing</span></td>
</tr>
<tr id="S1.T1.6.32.31" class="ltx_tr">
<td id="S1.T1.6.32.31.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.32.31.1.1" class="ltx_text" style="color:#000000;">ML</span></td>
<td id="S1.T1.6.32.31.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.32.31.2.1" class="ltx_text" style="color:#000000;">Machine Learning</span></td>
</tr>
<tr id="S1.T1.6.33.32" class="ltx_tr">
<td id="S1.T1.6.33.32.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.33.32.1.1" class="ltx_text" style="color:#000000;">MRI</span></td>
<td id="S1.T1.6.33.32.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.33.32.2.1" class="ltx_text" style="color:#000000;">Magnetic Resonance Imaging</span></td>
</tr>
<tr id="S1.T1.6.34.33" class="ltx_tr">
<td id="S1.T1.6.34.33.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.34.33.1.1" class="ltx_text" style="color:#000000;">NER</span></td>
<td id="S1.T1.6.34.33.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.34.33.2.1" class="ltx_text" style="color:#000000;">Named Entity Recognition</span></td>
</tr>
<tr id="S1.T1.6.35.34" class="ltx_tr">
<td id="S1.T1.6.35.34.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.35.34.1.1" class="ltx_text" style="color:#000000;">NLP</span></td>
<td id="S1.T1.6.35.34.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.35.34.2.1" class="ltx_text" style="color:#000000;">Natural Language Processing</span></td>
</tr>
<tr id="S1.T1.6.36.35" class="ltx_tr">
<td id="S1.T1.6.36.35.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.36.35.1.1" class="ltx_text" style="color:#000000;">Open-FL</span></td>
<td id="S1.T1.6.36.35.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.36.35.2.1" class="ltx_text" style="color:#000000;">Open Federated Learning</span></td>
</tr>
<tr id="S1.T1.6.37.36" class="ltx_tr">
<td id="S1.T1.6.37.36.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.37.36.1.1" class="ltx_text" style="color:#000000;">QoS</span></td>
<td id="S1.T1.6.37.36.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.37.36.2.1" class="ltx_text" style="color:#000000;">Quality of Service</span></td>
</tr>
<tr id="S1.T1.6.38.37" class="ltx_tr">
<td id="S1.T1.6.38.37.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.38.37.1.1" class="ltx_text" style="color:#000000;">TFF</span></td>
<td id="S1.T1.6.38.37.2" class="ltx_td ltx_align_left ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.38.37.2.1" class="ltx_text" style="color:#000000;">Tensor Flow Federated</span></td>
</tr>
<tr id="S1.T1.6.39.38" class="ltx_tr">
<td id="S1.T1.6.39.38.1" class="ltx_td ltx_align_left ltx_border_b ltx_border_r ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.39.38.1.1" class="ltx_text" style="color:#000000;">UCADI</span></td>
<td id="S1.T1.6.39.38.2" class="ltx_td ltx_align_left ltx_border_b ltx_border_t" style="padding-top:0.65pt;padding-bottom:0.65pt;"><span id="S1.T1.6.39.38.2.1" class="ltx_text" style="color:#000000;">Unified Computed Tomography AI Diagnostic Initiative</span></td>
</tr>
</tbody>
</table>
</figure>
<div id="S1.p3" class="ltx_para ltx_indent">
<p id="S1.p3.1" class="ltx_p"><span id="S1.p3.1.1" class="ltx_text" style="color:#000000;">The conventional approach of storing and processing data on distant cloud servers presents challenges in the context of medical data. Centralized </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S1.p3.1.2" class="ltx_text" style="color:#000000;"> model training involves aggregating data onto a single machine or cluster, raising privacy concerns and practical limitations </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.p3.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib13" title="" class="ltx_ref">13</a>, <a href="#bib.bib14" title="" class="ltx_ref">14</a><span id="S1.p3.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S1.p3.1.5" class="ltx_text" style="color:#000000;">. Furthermore, the trend is shifting towards distributed data storage and analysis at the network edge, driven by real-time requirements, latency concerns, and privacy considerations </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.p3.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib15" title="" class="ltx_ref">15</a>, <a href="#bib.bib16" title="" class="ltx_ref">16</a>, <a href="#bib.bib17" title="" class="ltx_ref">17</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib19" title="" class="ltx_ref">19</a>, <a href="#bib.bib20" title="" class="ltx_ref">20</a><span id="S1.p3.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S1.p3.1.8" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S1.p4" class="ltx_para">
<p id="S1.p4.1" class="ltx_p"><span id="S1.p4.1.1" class="ltx_text" style="color:#000000;">In this landscape, the convergence of edge computing-based AI (a.k.a. EdgeAI) has emerged as a potent strategy <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib21" title="" class="ltx_ref">21</a>, <a href="#bib.bib22" title="" class="ltx_ref">22</a>, <a href="#bib.bib23" title="" class="ltx_ref">23</a>, <a href="#bib.bib24" title="" class="ltx_ref">24</a>]</cite>. EdgeAI leverages the processing capabilities of distributed devices, enabling localized <abbr title="Artificial Intelligence" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">AI</span></abbr> operations without violating data regulations. FL takes this a step further, allowing collaborative model training across multiple healthcare datasets without sharing sensitive patient data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib25" title="" class="ltx_ref">25</a>, <a href="#bib.bib26" title="" class="ltx_ref">26</a>]</cite>. This approach addresses privacy concerns and boosts training efficiency by tapping into distributed datasets and resources. FL’s potential is underscored by its applicability in healthcare, promising significant advancements in patient care and public health systems <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib27" title="" class="ltx_ref">27</a>, <a href="#bib.bib28" title="" class="ltx_ref">28</a>, <a href="#bib.bib29" title="" class="ltx_ref">29</a>, <a href="#bib.bib30" title="" class="ltx_ref">30</a>]</cite>.</span></p>
</div>
<figure id="S1.T2" class="ltx_table">
<figcaption class="ltx_caption" style="color:#000000;"><span class="ltx_tag ltx_tag_table"><span id="S1.T2.4.1.1" class="ltx_text" style="font-size:90%;">TABLE II</span>: </span><span id="S1.T2.5.2" class="ltx_text" style="font-size:90%;">Comparison and overview of recent surveys in FL.</span></figcaption>
<table id="S1.T2.6" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S1.T2.6.1.1" class="ltx_tr">
<th id="S1.T2.6.1.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_r ltx_border_t"><span id="S1.T2.6.1.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Publications</span></th>
<th id="S1.T2.6.1.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_t"><span id="S1.T2.6.1.1.2.1" class="ltx_text ltx_font_bold" style="color:#000000;">Main Research Focus and Scope</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S1.T2.6.2.1" class="ltx_tr">
<th id="S1.T2.6.2.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t"><span id="S1.T2.6.2.1.1.1" class="ltx_text" style="color:#000000;">This Survey</span></th>
<td id="S1.T2.6.2.1.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.2.1.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.2.1.2.2" class="ltx_text" style="color:#000000;">  Presents a comprehensive survey on the use of FL in the medical domain.</span>
</td>
</tr>
<tr id="S1.T2.6.3.2" class="ltx_tr">
<th id="S1.T2.6.3.2.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.3.2.2" class="ltx_td ltx_align_left">
<span id="S1.T2.6.3.2.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.3.2.2.2" class="ltx_text" style="color:#000000;">  Provides a holistic taxonomy on the use of several technologies in conjugation with FL, especially for medical applications.</span>
</td>
</tr>
<tr id="S1.T2.6.4.3" class="ltx_tr">
<th id="S1.T2.6.4.3.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.4.3.2" class="ltx_td ltx_align_left">
<span id="S1.T2.6.4.3.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.4.3.2.2" class="ltx_text" style="color:#000000;">  Covers the state-of-the-art works targeting the most commonly caused cancer both in terms of incidence and mortality and the</span>
</td>
</tr>
<tr id="S1.T2.6.5.4" class="ltx_tr">
<th id="S1.T2.6.5.4.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.5.4.2" class="ltx_td ltx_align_left"><span id="S1.T2.6.5.4.2.1" class="ltx_text" style="color:#000000;">       application of FL in developing computer-aided diagnosis tools.</span></td>
</tr>
<tr id="S1.T2.6.6.5" class="ltx_tr">
<th id="S1.T2.6.6.5.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.6.5.2" class="ltx_td ltx_align_left">
<span id="S1.T2.6.6.5.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.6.5.2.2" class="ltx_text" style="color:#000000;">  Discusses the main challenges, open research problems, and future research directions of FL in the context of the medical domain.</span>
</td>
</tr>
<tr id="S1.T2.6.7.6" class="ltx_tr">
<th id="S1.T2.6.7.6.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.7.6.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.7.6.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib31" title="" class="ltx_ref">31</a><span id="S1.T2.6.7.6.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.7.6.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.7.6.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.7.6.2.2" class="ltx_text" style="color:#000000;">  Focuses on comparing different ML-based deployment architectures for FL.</span>
</td>
</tr>
<tr id="S1.T2.6.8.7" class="ltx_tr">
<th id="S1.T2.6.8.7.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.8.7.2" class="ltx_td ltx_align_left">
<span id="S1.T2.6.8.7.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.8.7.2.2" class="ltx_text" style="color:#000000;">  Covers privacy and security, and resource management.</span>
</td>
</tr>
<tr id="S1.T2.6.9.8" class="ltx_tr">
<th id="S1.T2.6.9.8.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.9.8.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.9.8.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib32" title="" class="ltx_ref">32</a><span id="S1.T2.6.9.8.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.9.8.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.9.8.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.9.8.2.2" class="ltx_text" style="color:#000000;">  Focuses on FL enabling technologies, protocols, and applications.</span>
</td>
</tr>
<tr id="S1.T2.6.10.9" class="ltx_tr">
<th id="S1.T2.6.10.9.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.10.9.2" class="ltx_td ltx_align_left">
<span id="S1.T2.6.10.9.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.10.9.2.2" class="ltx_text" style="color:#000000;">  Provides a summary of the relevant FL protocols, platforms, and some real-life use-cases for FL.</span>
</td>
</tr>
<tr id="S1.T2.6.11.10" class="ltx_tr">
<th id="S1.T2.6.11.10.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.11.10.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.11.10.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib33" title="" class="ltx_ref">33</a><span id="S1.T2.6.11.10.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.11.10.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.11.10.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.11.10.2.2" class="ltx_text" style="color:#000000;">  Focuses on exploring learning algorithms to improve the federated averaging algorithm.</span>
</td>
</tr>
<tr id="S1.T2.6.12.11" class="ltx_tr">
<th id="S1.T2.6.12.11.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.12.11.2" class="ltx_td ltx_align_left">
<span id="S1.T2.6.12.11.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.12.11.2.2" class="ltx_text" style="color:#000000;">  Reviews model fusion methods for FL.</span>
</td>
</tr>
<tr id="S1.T2.6.13.12" class="ltx_tr">
<th id="S1.T2.6.13.12.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.13.12.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.13.12.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib34" title="" class="ltx_ref">34</a><span id="S1.T2.6.13.12.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.13.12.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.13.12.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.13.12.2.2" class="ltx_text" style="color:#000000;">  Summarizes the development prospects of FL in industrial field.</span>
</td>
</tr>
<tr id="S1.T2.6.14.13" class="ltx_tr">
<th id="S1.T2.6.14.13.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.14.13.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.14.13.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib35" title="" class="ltx_ref">35</a><span id="S1.T2.6.14.13.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.14.13.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.14.13.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.14.13.2.2" class="ltx_text" style="color:#000000;">  Explores the definition of FL systems and analyzed the FL system components.</span>
</td>
</tr>
<tr id="S1.T2.6.15.14" class="ltx_tr">
<th id="S1.T2.6.15.14.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.15.14.2" class="ltx_td ltx_align_left">
<span id="S1.T2.6.15.14.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.15.14.2.2" class="ltx_text" style="color:#000000;">  Categorizes FL systems according to data distribution, ML model, privacy mechanism, communication architecture,</span>
</td>
</tr>
<tr id="S1.T2.6.16.15" class="ltx_tr">
<th id="S1.T2.6.16.15.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.16.15.2" class="ltx_td ltx_align_left"><span id="S1.T2.6.16.15.2.1" class="ltx_text" style="color:#000000;">      scale of federation and motivation of federation.</span></td>
</tr>
<tr id="S1.T2.6.17.16" class="ltx_tr">
<th id="S1.T2.6.17.16.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.17.16.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.17.16.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib36" title="" class="ltx_ref">36</a><span id="S1.T2.6.17.16.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.17.16.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.17.16.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.17.16.2.2" class="ltx_text" style="color:#000000;">  Surveys the applications of FL for mobile edge computing network optimization.</span>
</td>
</tr>
<tr id="S1.T2.6.18.17" class="ltx_tr">
<th id="S1.T2.6.18.17.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.18.17.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.18.17.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib37" title="" class="ltx_ref">37</a><span id="S1.T2.6.18.17.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.18.17.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.18.17.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.18.17.2.2" class="ltx_text" style="color:#000000;">  Investigates industrial application trends of FL, essential factors affecting the quality of FL models, and compares FL and</span>
</td>
</tr>
<tr id="S1.T2.6.19.18" class="ltx_tr">
<th id="S1.T2.6.19.18.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.19.18.2" class="ltx_td ltx_align_left"><span id="S1.T2.6.19.18.2.1" class="ltx_text" style="color:#000000;">      non-FL algorithms in terms of learning quality.</span></td>
</tr>
<tr id="S1.T2.6.20.19" class="ltx_tr">
<th id="S1.T2.6.20.19.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.20.19.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.20.19.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib25" title="" class="ltx_ref">25</a><span id="S1.T2.6.20.19.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.20.19.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.20.19.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.20.19.2.2" class="ltx_text" style="color:#000000;">  Investigates FL as a potential solution for the future of digital health and highlights the challenges of FL on digital health.</span>
</td>
</tr>
<tr id="S1.T2.6.21.20" class="ltx_tr">
<th id="S1.T2.6.21.20.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.21.20.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.21.20.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib38" title="" class="ltx_ref">38</a><span id="S1.T2.6.21.20.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.21.20.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.21.20.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.21.20.2.2" class="ltx_text" style="color:#000000;">  Discusses the application of FL for various fields in smart cities, including communication security and privacy issues.</span>
</td>
</tr>
<tr id="S1.T2.6.22.21" class="ltx_tr">
<th id="S1.T2.6.22.21.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.22.21.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.22.21.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib39" title="" class="ltx_ref">39</a><span id="S1.T2.6.22.21.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.22.21.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.22.21.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.22.21.2.2" class="ltx_text" style="color:#000000;">  Provides a systematic literature review on the FL and its applicability for confidential healthcare datasets.</span>
</td>
</tr>
<tr id="S1.T2.6.23.22" class="ltx_tr">
<th id="S1.T2.6.23.22.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.23.22.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.23.22.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib26" title="" class="ltx_ref">26</a>, <a href="#bib.bib40" title="" class="ltx_ref">40</a><span id="S1.T2.6.23.22.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.23.22.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.23.22.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.23.22.2.2" class="ltx_text" style="color:#000000;">  Describes the role of FL in Internet of Medical Things (IoMT) networks for privacy preservation.</span>
</td>
</tr>
<tr id="S1.T2.6.24.23" class="ltx_tr">
<th id="S1.T2.6.24.23.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.24.23.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.24.23.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib41" title="" class="ltx_ref">41</a><span id="S1.T2.6.24.23.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.24.23.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.24.23.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.24.23.2.2" class="ltx_text" style="color:#000000;">  Presents a systematic literature review on current research about FL in the context of EHR data.</span>
</td>
</tr>
<tr id="S1.T2.6.25.24" class="ltx_tr">
<th id="S1.T2.6.25.24.1" class="ltx_td ltx_th ltx_th_row ltx_border_r"></th>
<td id="S1.T2.6.25.24.2" class="ltx_td ltx_align_left">
<span id="S1.T2.6.25.24.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.25.24.2.2" class="ltx_text" style="color:#000000;">  Discusses a general architecture for FL based on the primary findings from the literature analysis.</span>
</td>
</tr>
<tr id="S1.T2.6.26.25" class="ltx_tr">
<th id="S1.T2.6.26.25.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t">
<span id="S1.T2.6.26.25.1.1" class="ltx_text" style="color:#000000;">Ref </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.T2.6.26.25.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib42" title="" class="ltx_ref">42</a><span id="S1.T2.6.26.25.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite>
</th>
<td id="S1.T2.6.26.25.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S1.T2.6.26.25.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.26.25.2.2" class="ltx_text" style="color:#000000;">  Presents a survey on the use of FL in smart healthcare.</span>
</td>
</tr>
<tr id="S1.T2.6.27.26" class="ltx_tr">
<th id="S1.T2.6.27.26.1" class="ltx_td ltx_th ltx_th_row ltx_border_b ltx_border_r"></th>
<td id="S1.T2.6.27.26.2" class="ltx_td ltx_align_left ltx_border_b">
<span id="S1.T2.6.27.26.2.1" class="ltx_text ltx_inline-block" style="width:0.0pt;color:#000000;">•</span><span id="S1.T2.6.27.26.2.2" class="ltx_text" style="color:#000000;">  Describes FL designs and some emerging applications of FL in healthcare domains.</span>
</td>
</tr>
</tbody>
</table>
</figure>
<section id="S1.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S1.SS1.4.1.1" class="ltx_text">I-A</span> </span><span id="S1.SS1.5.2" class="ltx_text ltx_font_italic">Scope and Contributions</span>
</h3>

<div id="S1.SS1.p1" class="ltx_para">
<p id="S1.SS1.p1.1" class="ltx_p"><span id="S1.SS1.p1.1.1" class="ltx_text" style="color:#000000;">The overarching goal of this survey is to explore the integration of FL with emerging technologies tailored for medical applications. While FL’s versatility spans across multiple domains, its fusion with transformative innovations like IoT, blockchain, cloud and edge computing, and AI bears particularly impactful implications within the realm of healthcare. The safeguarding of health-related data poses distinct challenges owing to stringent regulations governing their acquisition and use. Even data anonymization falls short of guaranteeing privacy, as researchers in academia, industry, and health regulatory bodies concur that erasing patient metadata alone doesn’t suffice. In light of these unique hurdles in medical data governance and privacy, our survey aspires to answer the following research questions (RQs):</span></p>
<ul id="S1.I1" class="ltx_itemize">
<li id="S1.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I1.i1.p1" class="ltx_para">
<p id="S1.I1.i1.p1.1" class="ltx_p"><span id="S1.I1.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">RQ1</span><span id="S1.I1.i1.p1.1.2" class="ltx_text" style="color:#000000;">. What constitutes the fundamental concept of FL, and what advantages does it offer within the realm of medical applications?</span></p>
</div>
</li>
<li id="S1.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I1.i2.p1" class="ltx_para">
<p id="S1.I1.i2.p1.1" class="ltx_p"><span id="S1.I1.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">RQ2</span><span id="S1.I1.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Within the healthcare domain, how do various FL frameworks differ, and what distinctive features characterize each of these frameworks?</span></p>
</div>
</li>
<li id="S1.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I1.i3.p1" class="ltx_para">
<p id="S1.I1.i3.p1.1" class="ltx_p"><span id="S1.I1.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">RQ3</span><span id="S1.I1.i3.p1.1.2" class="ltx_text" style="color:#000000;">. What are the prevailing emerging technologies, and how is FL integrated with these emerging technologies to address healthcare challenges?</span></p>
</div>
</li>
<li id="S1.I1.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I1.i4.p1" class="ltx_para">
<p id="S1.I1.i4.p1.1" class="ltx_p"><span id="S1.I1.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">RQ4</span><span id="S1.I1.i4.p1.1.2" class="ltx_text" style="color:#000000;">. In practice, what are the tangible use cases of FL in the medical field?</span></p>
</div>
</li>
<li id="S1.I1.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I1.i5.p1" class="ltx_para">
<p id="S1.I1.i5.p1.1" class="ltx_p"><span id="S1.I1.i5.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">RQ5</span><span id="S1.I1.i5.p1.1.2" class="ltx_text" style="color:#000000;">. Specifically, in the context of combating global cancer burdens, how is FL utilized to develop medical diagnostic tools? Is there empirical evidence supporting the superior performance and generalizability of FL compared to other data-driven AI models in healthcare?</span></p>
</div>
</li>
<li id="S1.I1.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I1.i6.p1" class="ltx_para">
<p id="S1.I1.i6.p1.1" class="ltx_p"><span id="S1.I1.i6.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">RQ6</span><span id="S1.I1.i6.p1.1.2" class="ltx_text" style="color:#000000;">. Which open-source FL frameworks are currently available, and which among them is most suitable for deployment in medical settings?</span></p>
</div>
</li>
<li id="S1.I1.i7" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I1.i7.p1" class="ltx_para">
<p id="S1.I1.i7.p1.1" class="ltx_p"><span id="S1.I1.i7.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">RQ7</span><span id="S1.I1.i7.p1.1.2" class="ltx_text" style="color:#000000;">. What are the primary obstacles and significant challenges faced by FL when applied in healthcare contexts?</span></p>
</div>
</li>
<li id="S1.I1.i8" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I1.i8.p1" class="ltx_para">
<p id="S1.I1.i8.p1.1" class="ltx_p"><span id="S1.I1.i8.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">RQ8</span><span id="S1.I1.i8.p1.1.2" class="ltx_text" style="color:#000000;">. What open issues and future research directions emerge for FL within the medical domain?</span></p>
</div>
</li>
</ul>
<p id="S1.SS1.p1.2" class="ltx_p"><span id="S1.SS1.p1.2.1" class="ltx_text" style="color:#000000;">Given the unique challenges in preserving privacy in medical data and based on the above-formulated RQs, our paper aims to provide a comprehensive taxonomy that clarifies the interplay between FL and emerging technologies in the healthcare sector. Furthermore, it explores the practical, real-world applications of FL in healthcare, delving into the challenges faced within this context and identifying open problems. Additionally, the paper outlines potential future research directions for FL in the medical field.</span></p>
</div>
<div id="S1.SS1.p2" class="ltx_para">
<p id="S1.SS1.p2.1" class="ltx_p"><span id="S1.SS1.p2.1.1" class="ltx_text" style="color:#000000;">Existing survey studies, exemplified by </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.SS1.p2.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib31" title="" class="ltx_ref">31</a>, <a href="#bib.bib32" title="" class="ltx_ref">32</a>, <a href="#bib.bib33" title="" class="ltx_ref">33</a>, <a href="#bib.bib34" title="" class="ltx_ref">34</a>, <a href="#bib.bib35" title="" class="ltx_ref">35</a>, <a href="#bib.bib36" title="" class="ltx_ref">36</a>, <a href="#bib.bib25" title="" class="ltx_ref">25</a>, <a href="#bib.bib38" title="" class="ltx_ref">38</a><span id="S1.SS1.p2.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S1.SS1.p2.1.4" class="ltx_text" style="color:#000000;">, predominantly concentrate on facets such as general architecture, models, security, and privacy algorithms for FL that are unrelated to medical applications. Scarcely few, such as </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S1.SS1.p2.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib39" title="" class="ltx_ref">39</a>, <a href="#bib.bib26" title="" class="ltx_ref">26</a>, <a href="#bib.bib40" title="" class="ltx_ref">40</a>, <a href="#bib.bib41" title="" class="ltx_ref">41</a>, <a href="#bib.bib42" title="" class="ltx_ref">42</a><span id="S1.SS1.p2.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S1.SS1.p2.1.7" class="ltx_text" style="color:#000000;">, delve into FL within the medical domain, offering insights into works tailored for healthcare. However, the primary focus of our study rests on medical applications, where we underscore the weighty burden of
global cancer and illuminate the potency of FL in engendering
computer-aided diagnosis tools that address this challenge with
heightened efficacy. Thus, our survey’s exclusive purview lies in scrutinizing FL’s integration with emergent technologies dedicated to medical applications. Distinct from other FL-focused surveys, this article distinguishes itself by offering a comprehensive taxonomy encompassing the synergy of emerging technologies and FL, with a specific emphasis on medical applications. Fig. </span><a href="#S1.F1" title="Figure 1 ‣ I Introduction ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">1</span></a><span id="S1.SS1.p2.1.8" class="ltx_text" style="color:#000000;"> illustrates the taxonomy of the subjects elucidated within this survey, while Table </span><a href="#S1.T2" title="TABLE II ‣ I Introduction ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">II</span></a><span id="S1.SS1.p2.1.9" class="ltx_text" style="color:#000000;"> provides a meticulous juxtaposition of related surveys in the field. In Section III, we expound upon FL’s integration with emerging technologies within the healthcare context.</span></p>
</div>
<div id="S1.SS1.p3" class="ltx_para ltx_indent">
<p id="S1.SS1.p3.1" class="ltx_p"><span id="S1.SS1.p3.1.1" class="ltx_text" style="color:#000000;">The primary contributions of our work relative to the recent literature include:</span></p>
</div>
<div id="S1.SS1.p4" class="ltx_para">
<ul id="S1.I2" class="ltx_itemize">
<li id="S1.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i1.p1" class="ltx_para">
<p id="S1.I2.i1.p1.1" class="ltx_p"><span id="S1.I2.i1.p1.1.1" class="ltx_text" style="color:#000000;">A comprehensive survey on the integration of FL with emerging technologies for medical applications, bridging the gap between technical rigour and healthcare context.</span></p>
</div>
</li>
<li id="S1.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i2.p1" class="ltx_para">
<p id="S1.I2.i2.p1.1" class="ltx_p"><span id="S1.I2.i2.p1.1.1" class="ltx_text" style="color:#000000;">Exploration of emerging technologies that synergize with FL to address medical challenges.</span></p>
</div>
</li>
<li id="S1.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i3.p1" class="ltx_para">
<p id="S1.I2.i3.p1.1" class="ltx_p"><span id="S1.I2.i3.p1.1.1" class="ltx_text" style="color:#000000;">In-depth coverage of the recent works on the global cancer burden (targeting most commonly caused cancer in terms of incidence and mortality) and the application of </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S1.I2.i3.p1.1.2" class="ltx_text" style="color:#000000;"> in developing computer-aided diagnosis tools.</span></p>
</div>
</li>
<li id="S1.I2.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i4.p1" class="ltx_para">
<p id="S1.I2.i4.p1.1" class="ltx_p"><span id="S1.I2.i4.p1.1.1" class="ltx_text" style="color:#000000;">Overview and comparison of existing open-source FL software frameworks.</span></p>
</div>
</li>
<li id="S1.I2.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i5.p1" class="ltx_para">
<p id="S1.I2.i5.p1.1" class="ltx_p"><span id="S1.I2.i5.p1.1.1" class="ltx_text" style="color:#000000;">Identification of key challenges and open problems of FL within the medical domain.</span></p>
</div>
</li>
<li id="S1.I2.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i6.p1" class="ltx_para">
<p id="S1.I2.i6.p1.1" class="ltx_p"><span id="S1.I2.i6.p1.1.1" class="ltx_text" style="color:#000000;">Finally, this survey highlights the open problems of FL and offers future research directions to advance FL’s role in healthcare.</span></p>
</div>
</li>
</ul>
</div>
</section>
<section id="S1.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S1.SS2.4.1.1" class="ltx_text">I-B</span> </span><span id="S1.SS2.5.2" class="ltx_text ltx_font_italic">Paper Organizations</span>
</h3>

<div id="S1.SS2.p1" class="ltx_para">
<p id="S1.SS2.p1.1" class="ltx_p"><span id="S1.SS2.p1.1.1" class="ltx_text" style="color:#000000;">The rest of the paper is organized as follows. Section </span><a href="#S2" title="II FL Concepts and Frameworks in the Medical Domain ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">II</span></a><span id="S1.SS2.p1.1.2" class="ltx_text" style="color:#000000;"> briefly gives an overview of the FL concept and frameworks for FL in the medical domain. We explain the existing works in FL for medical applications in conjugation with other emerging technologies in Section </span><a href="#S3" title="III FL and Its Applications with Other Emerging Technologies in the Context of Healthcare ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">III</span></a><span id="S1.SS2.p1.1.3" class="ltx_text" style="color:#000000;">. In Section </span><a href="#S4" title="IV Practical Application of FL in Medical Domain ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">IV</span></a><span id="S1.SS2.p1.1.4" class="ltx_text" style="color:#000000;">, we explain the practical application of FL in medical domain and illustrate how </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S1.SS2.p1.1.5" class="ltx_text" style="color:#000000;"> is applied in developing medical diagnosis tools, particularly in addressing the global burden of cancer. Open-source FL software frameworks are presented in Section </span><a href="#S5" title="V Open-Source FL Software Frameworks ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">V</span></a><span id="S1.SS2.p1.1.6" class="ltx_text" style="color:#000000;">. We examine the challenges of FL from a medical perspective in Section </span><a href="#S6" title="VI Challenges of in Healthcare ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">VI</span></a><span id="S1.SS2.p1.1.7" class="ltx_text" style="color:#000000;">. Some open research problems and future research directions of FL in medical applications are presented in Section </span><a href="#S7" title="VII Open Problems and Future Research Directions of FL in Medical Context ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">VII</span></a><span id="S1.SS2.p1.1.8" class="ltx_text" style="color:#000000;">. Finally, in Section </span><a href="#S8" title="VIII Conclusions ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">VIII</span></a><span id="S1.SS2.p1.1.9" class="ltx_text" style="color:#000000;">, we provide concluding remarks.</span></p>
</div>
<figure id="S1.F2" class="ltx_figure"><img src="/html/2208.03392/assets/x2.png" id="S1.F2.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="346" height="195" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_figure"><span id="S1.F2.5.1.1" class="ltx_text" style="font-size:90%;">Figure 2</span>: </span><span id="S1.F2.6.2" class="ltx_text" style="font-size:90%;">Classical architecture of <abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr>. The clients send local model updates trained using the local dataset to the server for aggregation. The central server finally aggregates the local models transmitted by the participating clients and sends the current updated global model back to each participating client.</span></figcaption>
</figure>
<figure id="S1.F3" class="ltx_figure"><img src="/html/2208.03392/assets/images/Healthcare_FL_Framework.png" id="S1.F3.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="449" height="250" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_figure"><span id="S1.F3.4.1.1" class="ltx_text" style="font-size:90%;">Figure 3</span>: </span><span id="S1.F3.5.2" class="ltx_text" style="font-size:90%;">FL framework for the medical domain.</span></figcaption>
</figure>
</section>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section" style="color:#000000;">
<span class="ltx_tag ltx_tag_section">II </span><span id="S2.1.1" class="ltx_text ltx_font_smallcaps">FL Concepts and Frameworks in the Medical Domain</span>
</h2>

<div id="S2.p1" class="ltx_para">
<p id="S2.p1.1" class="ltx_p"><span id="S2.p1.1.1" class="ltx_text" style="color:#000000;">Training complex ML models in a distributed medical environment is founded on the principles of trust, collaboration, efficiency, and scalability. In this paradigm, instead of centralizing all medical data on a single server for training, data remains on individual healthcare organizations or local hospital servers. This approach presents several key advantages:</span></p>
<ul id="S2.I1" class="ltx_itemize">
<li id="S2.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I1.i1.p1" class="ltx_para">
<p id="S2.I1.i1.p1.1" class="ltx_p"><span id="S2.I1.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Privacy-Preserving</span><span id="S2.I1.i1.p1.1.2" class="ltx_text" style="color:#000000;">. Individual healthcare data remains decentralized, addressing concerns of data privacy and ownership. This is especially critical in medical applications where patient confidentiality is paramount.</span></p>
</div>
</li>
<li id="S2.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I1.i2.p1" class="ltx_para">
<p id="S2.I1.i2.p1.1" class="ltx_p"><span id="S2.I1.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Sovereignty</span><span id="S2.I1.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Data remains under the control of its owners, which is crucial in sensitive domains like healthcare. Hospitals, clinics, and patients maintain authority over their respective data sources.</span></p>
</div>
</li>
<li id="S2.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I1.i3.p1" class="ltx_para">
<p id="S2.I1.i3.p1.1" class="ltx_p"><span id="S2.I1.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Reduced Data Transfer</span><span id="S2.I1.i3.p1.1.2" class="ltx_text" style="color:#000000;">. Since data doesn’t need to be transmitted to a central server, bandwidth and latency issues are minimized, making it ideal for real-time applications such as immersive augmented reality and metaverse in healthcare.</span></p>
</div>
</li>
<li id="S2.I1.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I1.i4.p1" class="ltx_para">
<p id="S2.I1.i4.p1.1" class="ltx_p"><span id="S2.I1.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Scalability</span><span id="S2.I1.i4.p1.1.2" class="ltx_text" style="color:#000000;">. Distributed training allows incorporating learning from a multitude of sources, enabling the creation of more robust and accurate models as the healthcare dataset size expands.</span></p>
</div>
</li>
<li id="S2.I1.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I1.i5.p1" class="ltx_para">
<p id="S2.I1.i5.p1.1" class="ltx_p"><span id="S2.I1.i5.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Energy Efficiency</span><span id="S2.I1.i5.p1.1.2" class="ltx_text" style="color:#000000;">. Healthcare devices in the network can contribute to model training while leveraging local processing power, reducing the need for massive data transfers.</span></p>
</div>
</li>
</ul>
</div>
<div id="S2.p2" class="ltx_para">
<p id="S2.p2.1" class="ltx_p"><span id="S2.p2.1.1" class="ltx_text" style="color:#000000;">Apart from the aforementioned advantages, in the healthcare domain, FL uniquely addresses privacy and security concerns by design:</span></p>
<ul id="S2.I2" class="ltx_itemize">
<li id="S2.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I2.i1.p1" class="ltx_para">
<p id="S2.I2.i1.p1.1" class="ltx_p"><span id="S2.I2.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Localization</span><span id="S2.I2.i1.p1.1.2" class="ltx_text" style="color:#000000;">. FL operates on decentralized data sources, ensuring that sensitive medical information remains within the confines of its origin, reducing the risk of unauthorized access.</span></p>
</div>
</li>
<li id="S2.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I2.i2.p1" class="ltx_para">
<p id="S2.I2.i2.p1.1" class="ltx_p"><span id="S2.I2.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Aggregation</span><span id="S2.I2.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Instead of sharing raw healthcare data, FL aggregates model updates or gradients from multiple devices, preserving individual healthcare data while generating collective insights.</span></p>
</div>
</li>
<li id="S2.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I2.i3.p1" class="ltx_para">
<p id="S2.I2.i3.p1.1" class="ltx_p"><span id="S2.I2.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Differential Privacy</span><span id="S2.I2.i3.p1.1.2" class="ltx_text" style="color:#000000;">. Techniques such as differential privacy can be integrated into FL, adding noise to aggregated updates to prevent the reconstruction of individual healthcare data.</span></p>
</div>
</li>
<li id="S2.I2.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I2.i4.p1" class="ltx_para">
<p id="S2.I2.i4.p1.1" class="ltx_p"><span id="S2.I2.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Anonymization</span><span id="S2.I2.i4.p1.1.2" class="ltx_text" style="color:#000000;">. FL facilitates model training without requiring explicit data sharing, allowing health institutions to anonymize patient data and still contribute to model improvement.</span></p>
</div>
</li>
<li id="S2.I2.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I2.i5.p1" class="ltx_para">
<p id="S2.I2.i5.p1.1" class="ltx_p"><span id="S2.I2.i5.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Secure Communication</span><span id="S2.I2.i5.p1.1.2" class="ltx_text" style="color:#000000;">. Encrypted communication channels protect data during transmission between devices and the central server, mitigating interception risks.</span></p>
</div>
</li>
</ul>
<p id="S2.p2.2" class="ltx_p"><span id="S2.p2.2.1" class="ltx_text" style="color:#000000;">In the context of medical applications, FL’s role is pivotal. By enabling collaborative model training on decentralized data, FL optimally balances the need for improved healthcare insights with stringent privacy and security concerns. The approach ensures that sensitive patient information remains confidential while allowing the medical community to collectively advance diagnostic accuracy, treatment, and overall patient care.</span>
<br class="ltx_break"><span id="S2.p2.2.2" class="ltx_text" style="color:#000000;">
A classical </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S2.p2.2.3" class="ltx_text" style="color:#000000;"> architecture is shown in Fig. </span><a href="#S1.F2" title="Figure 2 ‣ I-B Paper Organizations ‣ I Introduction ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">2</span></a><span id="S2.p2.2.4" class="ltx_text" style="color:#000000;">. It consists of a centralized global server that broadcasts the AI/ML model parameters to the clients in the FL network. Clients are selected by the central server, either randomly or through a client selection algorithm </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.p2.2.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib43" title="" class="ltx_ref">43</a><span id="S2.p2.2.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.p2.2.7" class="ltx_text" style="color:#000000;">. Upon receiving global model parameters, chosen clients proceed to train the model using their localized data. These clients then communicate their local model parameters back to the server, which aggregates and synthesizes them into a global model.</span>
<br class="ltx_break"><span id="S2.p2.2.8" class="ltx_text" style="color:#000000;">A fundamental algorithm within the FL realm is </span><em id="S2.p2.2.9" class="ltx_emph ltx_font_italic" style="color:#000000;">FedAvg</em><span id="S2.p2.2.10" class="ltx_text" style="color:#000000;">, first introduced by Google </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.p2.2.11.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib44" title="" class="ltx_ref">44</a><span id="S2.p2.2.12.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.p2.2.13" class="ltx_text" style="color:#000000;">. The global server employs this algorithm for aggregating the local parameters of diverse clients in each iteration </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.p2.2.14.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib45" title="" class="ltx_ref">45</a>, <a href="#bib.bib46" title="" class="ltx_ref">46</a><span id="S2.p2.2.15.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.p2.2.16" class="ltx_text" style="color:#000000;">. This iterative process continues until the desired convergence or a specified number of iterations is achieved on the global centralized server. While FedAvg effectively operates in the presence of non-iid data, its performance in non-iid data scenarios lacks theoretical guarantees within convex optimization settings </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.p2.2.17.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib46" title="" class="ltx_ref">46</a><span id="S2.p2.2.18.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.p2.2.19" class="ltx_text" style="color:#000000;">.</span>
<br class="ltx_break"><span id="S2.p2.2.20" class="ltx_text" style="color:#000000;">An analogous framework for FL within the medical domain is showcased in Fig. </span><a href="#S1.F3" title="Figure 3 ‣ I-B Paper Organizations ‣ I Introduction ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">3</span></a><span id="S2.p2.2.21" class="ltx_text" style="color:#000000;">, wherein potential clients include mobile healthcare, hospital healthcare, and home healthcare. Notably, these domains may exhibit varying data distributions. The application of FL to these diverse domains hinges on their data characteristics to overcome security, privacy, healthcare system, and device challenges. For instance, mobile healthcare encompasses myriad smart healthcare devices, such as smartwatches, smartphones, and health monitoring gadgets, utilized by individuals for health tracking. Challenges encompass energy efficiency, communication, computation, and privacy considerations. Conversely, hospital healthcare involves larger institutions with substantial resources and computational capabilities, which mitigate concerns like client dropouts and straggler issues. Lastly, home healthcare involves medical services provided within residential settings, often comprising numerous smart healthcare devices. Remote patient monitoring falls under home healthcare, providing personalized care to patients in familiar surroundings.</span></p>
</div>
<div id="S2.p3" class="ltx_para">
<p id="S2.p3.1" class="ltx_p"><span id="S2.p3.1.1" class="ltx_text" style="color:#000000;">In Fig. </span><a href="#S1.F3" title="Figure 3 ‣ I-B Paper Organizations ‣ I Introduction ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">3</span></a><span id="S2.p3.1.2" class="ltx_text" style="color:#000000;">, we emphasize the pivotal role of FL aggregation in orchestrating collaborative convergence from disparate data sources while upholding the sanctity of sensitive medical information. The aggregation process melds locally refined model updates or gradients from diverse decentralized devices or data sources. This integral step is central to fashioning an enhanced global model that elevates accuracy while steadfastly guarding data privacy and security.</span></p>
</div>
<div id="S2.p4" class="ltx_para">
<p id="S2.p4.1" class="ltx_p"><span id="S2.p4.1.1" class="ltx_text" style="color:#000000;">Especially in the medical sector, where patient confidentiality is paramount, FL aggregation serves manifold purposes:</span></p>
<ul id="S2.I3" class="ltx_itemize">
<li id="S2.I3.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I3.i1.p1" class="ltx_para">
<p id="S2.I3.i1.p1.1" class="ltx_p"><span id="S2.I3.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Privacy Preservation</span><span id="S2.I3.i1.p1.1.2" class="ltx_text" style="color:#000000;">. FL aggregation guarantees the localization and security of individual patient data at its source, with the central server exclusively receiving aggregated model updates. This circumvents the need for raw data exchange, ensuring strict compliance with data protection regulations such as GDPR.</span></p>
</div>
</li>
<li id="S2.I3.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I3.i2.p1" class="ltx_para">
<p id="S2.I3.i2.p1.1" class="ltx_p"><span id="S2.I3.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Collaborative Insights</span><span id="S2.I3.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Aggregation brings together diverse datasets from various healthcare institutions or patient devices. This enables the generation of a more robust and accurate global model, which collectively learns from a broader spectrum of patient demographics and medical conditions.</span></p>
</div>
</li>
<li id="S2.I3.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I3.i3.p1" class="ltx_para">
<p id="S2.I3.i3.p1.1" class="ltx_p"><span id="S2.I3.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Enhanced Diagnostics</span><span id="S2.I3.i3.p1.1.2" class="ltx_text" style="color:#000000;">. In the medical realm, the FL-aggregated model can yield improved diagnostic accuracy and predictive capabilities. For instance, a model trained on data from different hospitals can offer more generalized insights that benefit the entire healthcare community.</span></p>
</div>
</li>
<li id="S2.I3.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I3.i4.p1" class="ltx_para">
<p id="S2.I3.i4.p1.1" class="ltx_p"><span id="S2.I3.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Customization</span><span id="S2.I3.i4.p1.1.2" class="ltx_text" style="color:#000000;">. FL aggregation allows for model customization while maintaining data security. Different institutions or regions can fine-tune the aggregated model based on local data nuances without directly sharing sensitive information.</span></p>
</div>
</li>
<li id="S2.I3.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S2.I3.i5.p1" class="ltx_para">
<p id="S2.I3.i5.p1.1" class="ltx_p"><span id="S2.I3.i5.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Decentralized Expertise</span><span id="S2.I3.i5.p1.1.2" class="ltx_text" style="color:#000000;">. Aggregation facilitates the amalgamation of domain-specific expertise from various medical practitioners. This collective knowledge can lead to more accurate models that cater to specific medical conditions.</span></p>
</div>
</li>
</ul>
</div>
<div id="S2.p5" class="ltx_para">
<p id="S2.p5.1" class="ltx_p"><span id="S2.p5.1.1" class="ltx_text" style="color:#000000;">In essence, FL aggregation harmonizes the collective intelligence of individual data sources while preserving privacy, ensuring security, and yielding a globally refined model. This unified model, generated collaboratively from diverse medical data points, holds significant potential to drive advancements in medical research, diagnostics, and treatment while upholding the ethical imperatives of data privacy and security.</span></p>
</div>
<div id="S2.p6" class="ltx_para">
<p id="S2.p6.1" class="ltx_p"><span id="S2.p6.1.1" class="ltx_text" style="color:#000000;">The categorization of FL frameworks within the medical domain is primarily determined by data distributions, as outlined below:</span></p>
</div>
<section id="S2.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S2.SS1.4.1.1" class="ltx_text">II-A</span> </span><span id="S2.SS1.5.2" class="ltx_text ltx_font_italic">Horizontal FL Framework</span>
</h3>

<div id="S2.SS1.p1" class="ltx_para">
<p id="S2.SS1.p1.1" class="ltx_p"><span id="S2.SS1.p1.1.1" class="ltx_text" style="color:#000000;">Horizontal FL operates on datasets that share identical feature spaces across all devices. This implies that Medical Client A and Medical Client B possess the same set of features. For example, consider two regional healthcare institutions catering to separate user groups in distinct regions, yet having overlapping health diagnoses, resulting in identical feature spaces (e.g., age, gender, cholesterol levels, blood pressure) </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.SS1.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib47" title="" class="ltx_ref">47</a><span id="S2.SS1.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.SS1.p1.1.4" class="ltx_text" style="color:#000000;">. In this case, each hospital’s data represents a horizontal partition of the overall dataset. The hospitals can collaborate using Horizontal FL to train a model that learns from the collective data without sharing individual patient records. The final model is a shared model that benefits from the diverse patient populations across different regions. Addressing limited sample sizes in data training, horizontal FL adopts a data-split approach. Li et al. exemplified this with an autism spectrum disorder (ASD) prediction scenario involving four medical institutions across diverse locations </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.SS1.p1.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib48" title="" class="ltx_ref">48</a><span id="S2.SS1.p1.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.SS1.p1.1.7" class="ltx_text" style="color:#000000;">. All these institutions share the same user features, and a global model is collaboratively trained within the FL framework using patient samples from all nodes.</span></p>
</div>
</section>
<section id="S2.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S2.SS2.4.1.1" class="ltx_text">II-B</span> </span><span id="S2.SS2.5.2" class="ltx_text ltx_font_italic">Vertical FL Framework</span>
</h3>

<div id="S2.SS2.p1" class="ltx_para">
<p id="S2.SS2.p1.1" class="ltx_p"><span id="S2.SS2.p1.1.1" class="ltx_text" style="color:#000000;">Vertical FL entails collaborative training of a global model from varied feature spaces using diverse datasets </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.SS2.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib49" title="" class="ltx_ref">49</a><span id="S2.SS2.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.SS2.p1.1.4" class="ltx_text" style="color:#000000;">. This approach increases the feature dimension during data training. Consider a large hospital and a medical insurance company coexisting in a location; they might share a common user database. However, their feature spaces could significantly differ, with the hospital focusing on patient treatment history and the insurance institution managing medical bills, receipts, and claims. Vertical FL capitalizes on these distinct datasets to offer location-specific health recommendations. Vertical FL allows these medical institutions to merge their data for a holistic analysis without compromising data privacy. They exchange only the relevant features (e.g., combining demographic data with genetic markers) needed for a particular research task, such as identifying genetic predispositions to certain medical conditions. This approach ensures that sensitive patient information remains confidential. An example is provided by Cha et al., where an autoencoder FL model transforms client user features into a latent dimension for vertically partitioned medical data </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.SS2.p1.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib50" title="" class="ltx_ref">50</a><span id="S2.SS2.p1.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.SS2.p1.1.7" class="ltx_text" style="color:#000000;">.</span></p>
</div>
</section>
<section id="S2.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S2.SS3.4.1.1" class="ltx_text">II-C</span> </span><span id="S2.SS3.5.2" class="ltx_text ltx_font_italic">Federated Transfer Learning Framework</span>
</h3>

<div id="S2.SS3.p1" class="ltx_para">
<p id="S2.SS3.p1.1" class="ltx_p"><span id="S2.SS3.p1.1.1" class="ltx_text" style="color:#000000;">Federated transfer learning emerges when vertical FL incorporates a pre-trained model, initially trained on a similar dataset to address a distinct problem. This strategy is especially useful when datasets differ not just in samples but also in feature spaces. It facilitates training a customized health model tailored to user-specific attributes. Specifically, in this framework, medical institutions collaborate by not only sharing model updates but also transferring knowledge gained from related tasks, enhancing the performance of their models. Consider a consortium of healthcare institutions aiming to build a personalized cancer prediction model. Each institution has access to data on different cancer types (e.g., breast cancer, lung cancer) and wants to leverage the insights gained from one type to improve predictions for another while maintaining data privacy. In Federated Transfer Learning, institutions initially train models independently for their specific cancer types. Afterwards, they share model updates and insights with the central model, allowing it to benefit from the collective knowledge. For instance, discoveries made during breast cancer prediction can be transferred to improve the lung cancer prediction model and vice versa. This approach optimizes model performance and generalization without compromising data confidentiality. An illustrative instance is </span><em id="S2.SS3.p1.1.2" class="ltx_emph ltx_font_italic" style="color:#000000;">FedHealth</em><span id="S2.SS3.p1.1.3" class="ltx_text" style="color:#000000;"> proposed by Chen et al., which aggregates data from diverse organizations </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.SS3.p1.1.4.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib51" title="" class="ltx_ref">51</a><span id="S2.SS3.p1.1.5.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S2.SS3.p1.1.6" class="ltx_text" style="color:#000000;">. </span><em id="S2.SS3.p1.1.7" class="ltx_emph ltx_font_italic" style="color:#000000;">FedHealth</em><span id="S2.SS3.p1.1.8" class="ltx_text" style="color:#000000;"> creates personalized models for each organization using federated transfer learning. The FL model initially learns human activity recognition tasks and then employs transfer learning to extend this classification to categorize Parkinson’s disease. This approach generates a global model for a specific disease prediction that can also be applied to other medical challenges.</span></p>
</div>
</section>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section" style="color:#000000;">
<span class="ltx_tag ltx_tag_section">III </span><span id="S3.1.1" class="ltx_text ltx_font_smallcaps">FL and Its Applications with Other Emerging Technologies in the Context of Healthcare</span>
</h2>

<div id="S3.p1" class="ltx_para">
<p id="S3.p1.1" class="ltx_p"><span id="S3.p1.1.1" class="ltx_text" style="color:#000000;">FL stands out with its unique ability to forge robust and dependable ML models, all without necessitating the sharing of raw data. Its potential applications span diverse domains such as healthcare </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib25" title="" class="ltx_ref">25</a>, <a href="#bib.bib52" title="" class="ltx_ref">52</a>, <a href="#bib.bib53" title="" class="ltx_ref">53</a><span id="S3.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.p1.1.4" class="ltx_text" style="color:#000000;">,  </span><span title="Natural Language Processing" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Natural Language Processing</span></span><span id="S3.p1.1.5" class="ltx_text" style="color:#000000;"> (</span><abbr title="Natural Language Processing" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">NLP</span></abbr><span id="S3.p1.1.6" class="ltx_text" style="color:#000000;">) </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.p1.1.7.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib54" title="" class="ltx_ref">54</a><span id="S3.p1.1.8.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.p1.1.9" class="ltx_text" style="color:#000000;">, transportation</span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.p1.1.10.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib55" title="" class="ltx_ref">55</a>, <a href="#bib.bib56" title="" class="ltx_ref">56</a><span id="S3.p1.1.11.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.p1.1.12" class="ltx_text" style="color:#000000;">, and finance </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.p1.1.13.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib57" title="" class="ltx_ref">57</a><span id="S3.p1.1.14.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.p1.1.15" class="ltx_text" style="color:#000000;">. In an era characterized by the proliferation of networked </span><abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr><span id="S3.p1.1.16" class="ltx_text" style="color:#000000;"> devices, amassing and securing the deluge of data they generate pose substantial challenges. The conundrum of preserving the confidentiality and privacy of medical records, in particular, has become a focal point for researchers spanning industry, academia, and medical research. Within this landscape, FL techniques have emerged as indispensable privacy preservation tools in both industrial and medical AI applications. Recent studies underscore the efficiency gains and relief they offer to global healthcare systems </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.p1.1.17.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib58" title="" class="ltx_ref">58</a><span id="S3.p1.1.18.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.p1.1.19" class="ltx_text" style="color:#000000;">. Furthermore, FL models exhibit heightened robustness and efficacy compared to traditional data-driven medical applications </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.p1.1.20.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib59" title="" class="ltx_ref">59</a>, <a href="#bib.bib60" title="" class="ltx_ref">60</a><span id="S3.p1.1.21.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.p1.1.22" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.p2" class="ltx_para">
<p id="S3.p2.1" class="ltx_p"><span id="S3.p2.1.1" class="ltx_text" style="color:#000000;">To broaden its horizons and expand into diverse application domains, FL seamlessly collaborates with other cutting-edge technologies. This synergy accelerates the collective training of ML and DL models without necessitating the centralization of data sharing. In the ensuing sections, we delve into how FL harmoniously interfaces with contemporary technologies, with a specific focus on its application within the medical domain.</span></p>
</div>
<section id="S3.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S3.SS1.4.1.1" class="ltx_text">III-A</span> </span><span id="S3.SS1.5.2" class="ltx_text ltx_font_italic">Blockchain-based FL in Healthcare</span>
</h3>

<div id="S3.SS1.p1" class="ltx_para">
<p id="S3.SS1.p1.1" class="ltx_p"><span id="S3.SS1.p1.1.1" class="ltx_text" style="color:#000000;">FL is heralded for safeguarding the privacy of raw data on individual clients while enabling collaborative model development, as depicted in Fig. </span><a href="#S1.F2" title="Figure 2 ‣ I-B Paper Organizations ‣ I Introduction ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">2</span></a><span id="S3.SS1.p1.1.2" class="ltx_text" style="color:#000000;">. However, inherent challenges persist (refer to Section </span><a href="#S6" title="VI Challenges of in Healthcare ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">VI</span></a><span id="S3.SS1.p1.1.3" class="ltx_text" style="color:#000000;">). To surmount these hurdles and augment FL’s security, scalability, and performance, a powerful ally emerges in the form of blockchain technology </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p1.1.4.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib61" title="" class="ltx_ref">61</a>, <a href="#bib.bib62" title="" class="ltx_ref">62</a>, <a href="#bib.bib63" title="" class="ltx_ref">63</a><span id="S3.SS1.p1.1.5.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p1.1.6" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS1.p2" class="ltx_para">
<p id="S3.SS1.p2.1" class="ltx_p"><span id="S3.SS1.p2.1.1" class="ltx_text" style="color:#000000;">At its core, blockchain is a decentralized, public ledger technology fostering collaborative learning across devices without reliance on a central aggregator </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p2.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib61" title="" class="ltx_ref">61</a>, <a href="#bib.bib62" title="" class="ltx_ref">62</a>, <a href="#bib.bib63" title="" class="ltx_ref">63</a><span id="S3.SS1.p2.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p2.1.4" class="ltx_text" style="color:#000000;">. Its versatility spans a gamut of data-centric domains, with healthcare being a prominent arena. While originally devised for managing Bitcoin transactions and cryptocurrencies </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p2.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib64" title="" class="ltx_ref">64</a><span id="S3.SS1.p2.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p2.1.7" class="ltx_text" style="color:#000000;">, blockchain’s potential stretches far beyond. It has been instrumental in reshaping various sectors, including business, transportation, logistics, and healthcare </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p2.1.8.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib65" title="" class="ltx_ref">65</a>, <a href="#bib.bib66" title="" class="ltx_ref">66</a><span id="S3.SS1.p2.1.9.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p2.1.10" class="ltx_text" style="color:#000000;">. In healthcare, blockchain’s capabilities offer secure management of electronic health records </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p2.1.11.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib67" title="" class="ltx_ref">67</a><span id="S3.SS1.p2.1.12.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p2.1.13" class="ltx_text" style="color:#000000;"> and have prompted extensive research exploration </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p2.1.14.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib68" title="" class="ltx_ref">68</a>, <a href="#bib.bib69" title="" class="ltx_ref">69</a>, <a href="#bib.bib70" title="" class="ltx_ref">70</a>, <a href="#bib.bib1" title="" class="ltx_ref">1</a>, <a href="#bib.bib71" title="" class="ltx_ref">71</a>, <a href="#bib.bib72" title="" class="ltx_ref">72</a>, <a href="#bib.bib71" title="" class="ltx_ref">71</a>, <a href="#bib.bib73" title="" class="ltx_ref">73</a><span id="S3.SS1.p2.1.15.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p2.1.16" class="ltx_text" style="color:#000000;">. The amalgamation of blockchain and FL provides an avenue for secure health data storage and efficient deployment of FL applications in the healthcare sector.</span></p>
</div>
<div id="S3.SS1.p3" class="ltx_para">
<p id="S3.SS1.p3.1" class="ltx_p"><span id="S3.SS1.p3.1.1" class="ltx_text" style="color:#000000;">In this symbiotic alliance, blockchain enhances FL’s robustness and privacy. Pioneering endeavors like the one outlined in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p3.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib74" title="" class="ltx_ref">74</a><span id="S3.SS1.p3.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p3.1.4" class="ltx_text" style="color:#000000;"> showcase a secure, decentralized architecture employing privacy-preserving encryption techniques for FL within healthcare, leveraging the Ethereum blockchain. This architecture logs network events while preserving patient identities through advanced encryption protocols. Such secured frameworks hold immense potential for privacy-conscious AI applications in healthcare </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p3.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib74" title="" class="ltx_ref">74</a><span id="S3.SS1.p3.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p3.1.7" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS1.p4" class="ltx_para">
<p id="S3.SS1.p4.1" class="ltx_p"><span id="S3.SS1.p4.1.1" class="ltx_text" style="color:#000000;">The amalgamation of decentralized data and collaborative FL approaches </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p4.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib75" title="" class="ltx_ref">75</a><span id="S3.SS1.p4.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p4.1.4" class="ltx_text" style="color:#000000;"> aligns seamlessly with the decentralized nature of blockchain and can expedite the development of AI applications in healthcare. For instance, the architecture detailed in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p4.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib76" title="" class="ltx_ref">76</a><span id="S3.SS1.p4.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p4.1.7" class="ltx_text" style="color:#000000;"> combines blockchain and FL to secure multi-agent systems, promoting cooperation among individual agent units for  </span><span title="Internet of Medical Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Internet of Medical Things</span></span><span id="S3.SS1.p4.1.8" class="ltx_text" style="color:#000000;"> (</span><abbr title="Internet of Medical Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoMT</span></abbr><span id="S3.SS1.p4.1.9" class="ltx_text" style="color:#000000;">). The results demonstrate significant promise, achieving an 80% accuracy in skin cancer classification </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p4.1.10.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib76" title="" class="ltx_ref">76</a><span id="S3.SS1.p4.1.11.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p4.1.12" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS1.p5" class="ltx_para">
<p id="S3.SS1.p5.1" class="ltx_p"><span id="S3.SS1.p5.1.1" class="ltx_text" style="color:#000000;">Blockchain’s potential extends to clinical trials and precision medicine </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p5.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib77" title="" class="ltx_ref">77</a><span id="S3.SS1.p5.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p5.1.4" class="ltx_text" style="color:#000000;">, offering a distributed parallel computing architecture for precision medicine through FL and transfer learning </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p5.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib77" title="" class="ltx_ref">77</a><span id="S3.SS1.p5.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p5.1.7" class="ltx_text" style="color:#000000;">. This innovation facilitates decentralized processing of extensive medical data. Disease diagnosis also benefits from blockchain; Health-Chain </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p5.1.8.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib78" title="" class="ltx_ref">78</a><span id="S3.SS1.p5.1.9.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p5.1.10" class="ltx_text" style="color:#000000;">, for instance, presents a blockchain-based decentralized privacy-preserving cross-institution disease classification framework. Employing differential privacy and pseudo-identity mechanisms, it addresses data privacy issues effectively. The authors’ experiments, focusing on breast cancer diagnosis and ECG arrhythmia classification, demonstrate the efficiency and effectiveness of Health-Chain </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS1.p5.1.11.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib78" title="" class="ltx_ref">78</a><span id="S3.SS1.p5.1.12.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS1.p5.1.13" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS1.p6" class="ltx_para">
<p id="S3.SS1.p6.1" class="ltx_p"><span id="S3.SS1.p6.1.1" class="ltx_text" style="color:#000000;">In essence, the symbiotic integration of blockchain-based FL in healthcare holds significant promise for the industry. Here are some key points highlighting its importance:</span></p>
<ul id="S3.I1" class="ltx_itemize">
<li id="S3.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I1.i1.p1" class="ltx_para">
<p id="S3.I1.i1.p1.1" class="ltx_p"><span id="S3.I1.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Enhanced Data Security</span><span id="S3.I1.i1.p1.1.2" class="ltx_text" style="color:#000000;">. Blockchain’s inherent security features, such as decentralized and immutable ledgers, bolster the privacy and integrity of healthcare data. When combined with FL, this integration ensures that patient information remains highly secure.</span></p>
</div>
</li>
<li id="S3.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I1.i2.p1" class="ltx_para">
<p id="S3.I1.i2.p1.1" class="ltx_p"><span id="S3.I1.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Transparency and Accountability</span><span id="S3.I1.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Blockchain’s transparency allows patients to have greater control and visibility over who accesses their health data and for what purposes. FL, in tandem, maintains data privacy while providing an auditable and transparent record of model updates and access requests, enhancing trust among stakeholders.</span></p>
</div>
</li>
<li id="S3.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I1.i3.p1" class="ltx_para">
<p id="S3.I1.i3.p1.1" class="ltx_p"><span id="S3.I1.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Interoperability</span><span id="S3.I1.i3.p1.1.2" class="ltx_text" style="color:#000000;">. Blockchain facilitates interoperability among disparate healthcare systems and institutions. FL’s federated approach extends this interoperability to machine learning models, enabling seamless collaboration and knowledge sharing across healthcare providers and researchers.</span></p>
</div>
</li>
<li id="S3.I1.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I1.i4.p1" class="ltx_para">
<p id="S3.I1.i4.p1.1" class="ltx_p"><span id="S3.I1.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Consent Management</span><span id="S3.I1.i4.p1.1.2" class="ltx_text" style="color:#000000;">. Blockchain-based smart contracts can manage patient consent for data sharing and model training. FL can then ensure that only authorized parties access and utilize patient data, aligning with evolving data protection regulations.</span></p>
</div>
</li>
<li id="S3.I1.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I1.i5.p1" class="ltx_para">
<p id="S3.I1.i5.p1.1" class="ltx_p"><span id="S3.I1.i5.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Monetization</span><span id="S3.I1.i5.p1.1.2" class="ltx_text" style="color:#000000;">. Patients can potentially benefit from sharing their health data through blockchain-based tokens or incentives. FL ensures data privacy while allowing patients to have control over how their data is used, possibly leading to new revenue-sharing models.</span></p>
</div>
</li>
<li id="S3.I1.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I1.i6.p1" class="ltx_para">
<p id="S3.I1.i6.p1.1" class="ltx_p"><span id="S3.I1.i6.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Research Advancements</span><span id="S3.I1.i6.p1.1.2" class="ltx_text" style="color:#000000;">. The combination of blockchain and FL accelerates medical research by facilitating secure and collaborative model training across institutions and geographies, leading to the development of more accurate diagnostic and treatment tools.</span></p>
</div>
</li>
<li id="S3.I1.i7" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I1.i7.p1" class="ltx_para">
<p id="S3.I1.i7.p1.1" class="ltx_p"><span id="S3.I1.i7.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Regulatory Compliance</span><span id="S3.I1.i7.p1.1.2" class="ltx_text" style="color:#000000;">. Healthcare is heavily regulated, and blockchain-based FL can assist in complying with data protection laws like HIPAA or GDPR by providing a robust framework for data management and privacy.</span></p>
</div>
</li>
</ul>
<p id="S3.SS1.p6.2" class="ltx_p"><span id="S3.SS1.p6.2.1" class="ltx_text" style="color:#000000;">In summary, the integration of blockchain and FL in healthcare not only addresses critical data privacy concerns but also enhances data sharing, research capabilities, and patient empowerment, ultimately driving innovation and improving patient care.</span></p>
</div>
</section>
<section id="S3.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S3.SS2.4.1.1" class="ltx_text">III-B</span> </span><span id="S3.SS2.5.2" class="ltx_text ltx_font_italic">FL Enabled IoT-based Healthcare Monitoring</span>
</h3>

<div id="S3.SS2.p1" class="ltx_para">
<p id="S3.SS2.p1.1" class="ltx_p"><span id="S3.SS2.p1.1.1" class="ltx_text" style="color:#000000;">The advent of </span><abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr><span id="S3.SS2.p1.1.2" class="ltx_text" style="color:#000000;"> technology has orchestrated a revolutionary landscape, intertwining seamless connectivity, smart devices, and enhanced productivity </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS2.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib79" title="" class="ltx_ref">79</a><span id="S3.SS2.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS2.p1.1.5" class="ltx_text" style="color:#000000;">. This evolution has been steered by the integration of a multitude of distributed smart devices and sensors, orchestrating the real-time generation of user data across various applications. Among the diverse domains that have been profoundly impacted by the </span><abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr><span id="S3.SS2.p1.1.6" class="ltx_text" style="color:#000000;"> revolution, healthcare stands out prominently </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS2.p1.1.7.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib80" title="" class="ltx_ref">80</a><span id="S3.SS2.p1.1.8.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS2.p1.1.9" class="ltx_text" style="color:#000000;">. Combining IoT with FL in healthcare merges real-time data from IoT devices with privacy-preserving FL model training. Notably, the work outlined in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS2.p1.1.10.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib80" title="" class="ltx_ref">80</a><span id="S3.SS2.p1.1.11.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS2.p1.1.12" class="ltx_text" style="color:#000000;"> delves into the intricacies of this fusion, elucidating the interplay between </span><abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr><span id="S3.SS2.p1.1.13" class="ltx_text" style="color:#000000;"> and healthcare applications. This exploration is complemented by an insightful discussion on the security challenges encountered within this convergence and potential strategies to address them.</span></p>
</div>
<div id="S3.SS2.p2" class="ltx_para">
<p id="S3.SS2.p2.1" class="ltx_p"><span id="S3.SS2.p2.1.1" class="ltx_text" style="color:#000000;">The </span><abbr title="Internet of Medical Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoMT</span></abbr><span id="S3.SS2.p2.1.2" class="ltx_text" style="color:#000000;"> paradigm represents yet another transformative facet within the realm of IoT and healthcare technology </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS2.p2.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib81" title="" class="ltx_ref">81</a><span id="S3.SS2.p2.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS2.p2.1.5" class="ltx_text" style="color:#000000;">. As these innovative technologies continue to proliferate, they give rise to a profusion of data streams from integrated devices. However, a distinctive feature emerges-thanks to </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S3.SS2.p2.1.6" class="ltx_text" style="color:#000000;"> techniques-allowing data evaluation to transpire locally at the edge devices. This decentralized approach, a cornerstone of </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S3.SS2.p2.1.7" class="ltx_text" style="color:#000000;">, ensures data privacy for </span><abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr><span id="S3.SS2.p2.1.8" class="ltx_text" style="color:#000000;"> devices, particularly critical within sensitive domains such as healthcare. It distinguishes </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S3.SS2.p2.1.9" class="ltx_text" style="color:#000000;"> from conventional centralized </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S3.SS2.p2.1.10" class="ltx_text" style="color:#000000;"> methods, promising a balance between data utility and confidentiality. In a parallel vein, Zhao et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS2.p2.1.11.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib82" title="" class="ltx_ref">82</a><span id="S3.SS2.p2.1.12.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS2.p2.1.13" class="ltx_text" style="color:#000000;"> presents a pioneering endeavor that harnesses </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S3.SS2.p2.1.14" class="ltx_text" style="color:#000000;">, differential privacy, and blockchain to empower manufacturers to assess </span><abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr><span id="S3.SS2.p2.1.15" class="ltx_text" style="color:#000000;">-derived data securely and effectively, aligning with the narrative of enhanced healthcare monitoring.</span></p>
</div>
<div id="S3.SS2.p3" class="ltx_para">
<p id="S3.SS2.p3.1" class="ltx_p"><span id="S3.SS2.p3.1.1" class="ltx_text" style="color:#000000;">The confluence of </span><abbr title="Internet of Things" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IoT</span></abbr><span id="S3.SS2.p3.1.2" class="ltx_text" style="color:#000000;"> and </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S3.SS2.p3.1.3" class="ltx_text" style="color:#000000;"> holds great potential, fundamentally transforming healthcare and beyond. Here are some key points highlighting its importance:</span></p>
</div>
<div id="S3.SS2.p4" class="ltx_para">
<ul id="S3.I2" class="ltx_itemize">
<li id="S3.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I2.i1.p1" class="ltx_para">
<p id="S3.I2.i1.p1.1" class="ltx_p"><span id="S3.I2.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Real-time Data Processing</span><span id="S3.I2.i1.p1.1.2" class="ltx_text" style="color:#000000;">. IoT devices continuously collect and transmit patient health data, enabling healthcare providers to access real-time information. FL can be used to collaboratively analyze this data while preserving patient privacy, leading to timely insights and interventions.</span></p>
</div>
</li>
<li id="S3.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I2.i2.p1" class="ltx_para">
<p id="S3.I2.i2.p1.1" class="ltx_p"><span id="S3.I2.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Remote Patient Monitoring</span><span id="S3.I2.i2.p1.1.2" class="ltx_text" style="color:#000000;">. IoT devices can remotely monitor patients’ vital signs and chronic conditions. FL allows this data to be aggregated and analyzed collectively, enabling healthcare professionals to make informed decisions about patient care without compromising individual privacy.</span></p>
</div>
</li>
<li id="S3.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I2.i3.p1" class="ltx_para">
<p id="S3.I2.i3.p1.1" class="ltx_p"><span id="S3.I2.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Early Disease Diagnosis and Prognosis</span><span id="S3.I2.i3.p1.1.2" class="ltx_text" style="color:#000000;">. By analyzing IoT-generated data with FL models, healthcare providers can predict and detect diseases at an early stage. This can lead to timely interventions and improved patient outcomes.</span></p>
</div>
</li>
<li id="S3.I2.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I2.i4.p1" class="ltx_para">
<p id="S3.I2.i4.p1.1" class="ltx_p"><span id="S3.I2.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Reduced Healthcare Costs</span><span id="S3.I2.i4.p1.1.2" class="ltx_text" style="color:#000000;">. The integration of IoT and FL can help reduce healthcare costs by preventing hospital readmissions through continuous monitoring, optimizing treatment plans, and reducing unnecessary medical interventions.</span></p>
</div>
</li>
<li id="S3.I2.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I2.i5.p1" class="ltx_para">
<p id="S3.I2.i5.p1.1" class="ltx_p"><span id="S3.I2.i5.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Scalability</span><span id="S3.I2.i5.p1.1.2" class="ltx_text" style="color:#000000;">. The IoT ecosystem is continually expanding, and FL can scale to accommodate a growing number of IoT devices and data sources, making it suitable for large-scale healthcare applications.</span></p>
</div>
</li>
<li id="S3.I2.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I2.i6.p1" class="ltx_para">
<p id="S3.I2.i6.p1.1" class="ltx_p"><span id="S3.I2.i6.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Improved Patient Engagement</span><span id="S3.I2.i6.p1.1.2" class="ltx_text" style="color:#000000;">. IoT devices can engage patients in their healthcare by providing them with valuable insights into their health status. FL can further enhance this engagement by offering personalized recommendations and feedback.</span></p>
</div>
</li>
</ul>
</div>
<div id="S3.SS2.p5" class="ltx_para">
<p id="S3.SS2.p5.1" class="ltx_p"><span id="S3.SS2.p5.1.1" class="ltx_text" style="color:#000000;">In summary, IoT devices collect patient data, while FL enables collaborative model building across decentralized sources without sharing raw data. This empowers personalized treatment recommendations, real-time monitoring, and alerts. Patient privacy is upheld as data remains local, meeting stringent regulatory standards. Moreover, diverse IoT data enriches models, adapting them to changing conditions and ensuring scalability. This convergence, as evidenced by Zhao et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS2.p5.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib82" title="" class="ltx_ref">82</a><span id="S3.SS2.p5.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS2.p5.1.4" class="ltx_text" style="color:#000000;">, holds the promise of revolutionizing data evaluation methodologies, ensuring privacy, security, and progress in the context of medical applications.</span></p>
</div>
</section>
<section id="S3.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S3.SS3.4.1.1" class="ltx_text">III-C</span> </span><span id="S3.SS3.5.2" class="ltx_text ltx_font_italic">Edge Computing Assisted FL for Healthcare</span>
</h3>

<figure id="S3.F4" class="ltx_figure"><img src="/html/2208.03392/assets/images/edge_computing.png" id="S3.F4.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="439" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_figure"><span id="S3.F4.4.1.1" class="ltx_text" style="font-size:90%;">Figure 4</span>: </span><span id="S3.F4.5.2" class="ltx_text" style="font-size:90%;">General architecture of edge computing in healthcare.</span></figcaption>
</figure>
<div id="S3.SS3.p1" class="ltx_para">
<p id="S3.SS3.p1.1" class="ltx_p"><span id="S3.SS3.p1.1.1" class="ltx_text" style="color:#000000;">In the ever-evolving landscape of healthcare, the fusion of edge computing and FL emerges as a transformative force. Edge computing extends the realms of conventional cloud computing by relocating computing resources and data storage closer to data sources, ushering in improvements in network resilience, availability, latency, and bandwidth </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib84" title="" class="ltx_ref">84</a>, <a href="#bib.bib85" title="" class="ltx_ref">85</a><span id="S3.SS3.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p1.1.4" class="ltx_text" style="color:#000000;">. Fig. </span><a href="#S3.F4" title="Figure 4 ‣ III-C Edge Computing Assisted FL for Healthcare ‣ III FL and Its Applications with Other Emerging Technologies in the Context of Healthcare ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">4</span></a><span id="S3.SS3.p1.1.5" class="ltx_text" style="color:#000000;"> elucidates the architecture of an edge computing platform, decentralizing processing power, intelligence, and communication capabilities to the myriad edge devices. This shift substantially reduces response times and curtails the deluge of data streaming onto the internet, a boon for time-sensitive applications.</span></p>
</div>
<div id="S3.SS3.p2" class="ltx_para">
<p id="S3.SS3.p2.1" class="ltx_p"><span id="S3.SS3.p2.1.1" class="ltx_text" style="color:#000000;">A pioneering FL framework, FedHome </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p2.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib86" title="" class="ltx_ref">86</a><span id="S3.SS3.p2.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p2.1.4" class="ltx_text" style="color:#000000;">, exemplifies the integration of cloud and edge resources to personalize in-home health monitoring. FedHome refines models by employing a class-balanced dataset generated from user data, effectively addressing statistical and communication challenges. It employs a lightweight model, the  </span><span title="Generative Convolutional Autoencoder" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Generative Convolutional Autoencoder</span></span><span id="S3.SS3.p2.1.5" class="ltx_text" style="color:#000000;"> (</span><abbr title="Generative Convolutional Autoencoder" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">GCAE</span></abbr><span id="S3.SS3.p2.1.6" class="ltx_text" style="color:#000000;">), for seamless data transfer between the cloud and the edges, significantly reducing communication costs. This remarkable integration yields an impressive accuracy of 95.41%, outperforming conventional neural network techniques by over 7.49%. Compared to other FL systems, FedHome achieves accuracy improvements exceeding 10%, positioning it as a potent tool for future in-home health monitoring </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p2.1.7.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib86" title="" class="ltx_ref">86</a><span id="S3.SS3.p2.1.8.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p2.1.9" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS3.p3" class="ltx_para">
<p id="S3.SS3.p3.1" class="ltx_p"><span id="S3.SS3.p3.1.1" class="ltx_text" style="color:#000000;">In parallel, innovative approaches continue to surface. A fog computing-based prototype </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p3.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib87" title="" class="ltx_ref">87</a><span id="S3.SS3.p3.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p3.1.4" class="ltx_text" style="color:#000000;"> demonstrates a fourfold acceleration in the computation process for mobile patients. Another trailblazing framework, BodyEdge</span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p3.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib88" title="" class="ltx_ref">88</a><span id="S3.SS3.p3.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p3.1.7" class="ltx_text" style="color:#000000;">, comprises a miniature mobile client module on mobile devices and an edge gateway, featuring a multi-radio interface for dependable connectivity and multi-technology communication. Designed to thrive in local environments, this healthcare framework significantly reduces data traffic to the internet, achieving response times commensurate with healthcare IoT applications</span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p3.1.8.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib88" title="" class="ltx_ref">88</a><span id="S3.SS3.p3.1.9.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p3.1.10" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS3.p4" class="ltx_para">
<p id="S3.SS3.p4.1" class="ltx_p"><span id="S3.SS3.p4.1.1" class="ltx_text" style="color:#000000;">FL, with its innate compatibility with edge computing, holds substantial promise. Unlike traditional centralized approaches, edge computing obviates the need to transport data from distributed edge devices to a central cloud platform. This synergy leverages the computational resources of edge servers and the data harnessed from edge devices, a match made in healthcare innovation heaven. Personalized edge-assisted FL </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p4.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib89" title="" class="ltx_ref">89</a><span id="S3.SS3.p4.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p4.1.4" class="ltx_text" style="color:#000000;"> offers a prime example, catering to healthcare analytics rooted in user-generated data. This framework employs pre-trained models to glean tailored insights, examining diseases by monitoring mobility levels and behaviors through wearable devices. The result is a privacy-respecting and cloud-resource-efficient solution</span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p4.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib89" title="" class="ltx_ref">89</a><span id="S3.SS3.p4.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p4.1.7" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS3.p5" class="ltx_para">
<p id="S3.SS3.p5.1" class="ltx_p"><span id="S3.SS3.p5.1.1" class="ltx_text" style="color:#000000;">To bolster integrity and security in the realm of distributed IoT devices, the fusion of FL and blockchain within edge computing architectures comes to the fore </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p5.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib90" title="" class="ltx_ref">90</a><span id="S3.SS3.p5.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p5.1.4" class="ltx_text" style="color:#000000;">. Recent advancements in blockchain and  </span><span title="Mobile Edge Computing" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Mobile Edge Computing</span></span><span id="S3.SS3.p5.1.5" class="ltx_text" style="color:#000000;"> (</span><abbr title="Mobile Edge Computing" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">MEC</span></abbr><span id="S3.SS3.p5.1.6" class="ltx_text" style="color:#000000;">) herald fresh opportunities for healthcare transformation </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p5.1.7.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib91" title="" class="ltx_ref">91</a>, <a href="#bib.bib92" title="" class="ltx_ref">92</a>, <a href="#bib.bib93" title="" class="ltx_ref">93</a>, <a href="#bib.bib94" title="" class="ltx_ref">94</a>, <a href="#bib.bib95" title="" class="ltx_ref">95</a><span id="S3.SS3.p5.1.8.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p5.1.9" class="ltx_text" style="color:#000000;">. Decentralized architectures, like the one proposed in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p5.1.10.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib96" title="" class="ltx_ref">96</a><span id="S3.SS3.p5.1.11.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p5.1.12" class="ltx_text" style="color:#000000;">, employ blockchain and MEC for secure  </span><span title="Electronic Medical Record" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Electronic Medical Record</span></span><span id="S3.SS3.p5.1.13" class="ltx_text" style="color:#000000;"> (</span><abbr title="Electronic Medical Record" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">EMR</span></abbr><span id="S3.SS3.p5.1.14" class="ltx_text" style="color:#000000;">) sharing among federated hospitals. Such systems deploy decentralized EMR storage on MEC servers, supporting secure, distributed data sharing through the  </span><span title="Interplanetary File System" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Interplanetary File System</span></span><span id="S3.SS3.p5.1.15" class="ltx_text" style="color:#000000;"> (</span><abbr title="Interplanetary File System" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">IPFS</span></abbr><span id="S3.SS3.p5.1.16" class="ltx_text" style="color:#000000;">). For energy-efficient computation offloading decisions within MEC systems, deep reinforcement learning techniques and FL frameworks with mobile edge systems step in, conserving IoT device energy while ensuring service quality </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p5.1.17.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib97" title="" class="ltx_ref">97</a><span id="S3.SS3.p5.1.18.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p5.1.19" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS3.p6" class="ltx_para">
<p id="S3.SS3.p6.1" class="ltx_p"><span id="S3.SS3.p6.1.1" class="ltx_text" style="color:#000000;">Ensuring the sanctity of personal data in smart healthcare systems is paramount. The lightweight privacy protection protocol presented in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p6.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib81" title="" class="ltx_ref">81</a><span id="S3.SS3.p6.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p6.1.4" class="ltx_text" style="color:#000000;"> operates within an edge computing environment, leveraging shared secrets and weight masks to safeguard user privacy. This protocol maintains gradient privacy, fortifying model accuracy while repelling equipment dropouts and collusion attacks. It employs algorithms rooted in digital signatures and hash functions, preserving message integrity and thwarting replay attacks. When compared to differential privacy, this approach boasts a 40% efficiency improvement while upholding the same safety and efficacy standards as FL </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.SS3.p6.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib81" title="" class="ltx_ref">81</a><span id="S3.SS3.p6.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S3.SS3.p6.1.7" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S3.SS3.p7" class="ltx_para">
<p id="S3.SS3.p7.1" class="ltx_p"><span id="S3.SS3.p7.1.1" class="ltx_text" style="color:#000000;">At this juncture, we want to highlight that the symbiotic integration of Edge Computing-based FL in healthcare is a transformative development with several key points of significance:</span></p>
<ul id="S3.I3" class="ltx_itemize">
<li id="S3.I3.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I3.i1.p1" class="ltx_para">
<p id="S3.I3.i1.p1.1" class="ltx_p"><span id="S3.I3.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Low Latency Data Processing</span><span id="S3.I3.i1.p1.1.2" class="ltx_text" style="color:#000000;">. Edge computing brings computation and data processing closer to the source of data, reducing latency. When combined with FL, this enables real-time analysis of healthcare data from IoT devices, wearables, and sensors, ensuring timely responses to critical medical conditions.</span></p>
</div>
</li>
<li id="S3.I3.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I3.i2.p1" class="ltx_para">
<p id="S3.I3.i2.p1.1" class="ltx_p"><span id="S3.I3.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Privacy-Preserving Data Processing</span><span id="S3.I3.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Edge devices can locally process and aggregate data, allowing FL to perform model training without the need to centralize sensitive patient information. This preserves patient privacy and complies with data protection regulations.</span></p>
</div>
</li>
<li id="S3.I3.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I3.i3.p1" class="ltx_para">
<p id="S3.I3.i3.p1.1" class="ltx_p"><span id="S3.I3.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Enhanced Data Security</span><span id="S3.I3.i3.p1.1.2" class="ltx_text" style="color:#000000;">. Edge devices can implement security measures locally, protecting data as it is collected and processed. FL extends this security to model training, ensuring that patient data remains secure throughout the entire process.</span></p>
</div>
</li>
<li id="S3.I3.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I3.i4.p1" class="ltx_para">
<p id="S3.I3.i4.p1.1" class="ltx_p"><span id="S3.I3.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Edge AI for Diagnosis</span><span id="S3.I3.i4.p1.1.2" class="ltx_text" style="color:#000000;">. Combining Edge Computing with FL enables AI-powered diagnosis and decision support at the edge. This can lead to faster and more accurate diagnosis in scenarios like medical imaging analysis.</span></p>
</div>
</li>
<li id="S3.I3.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I3.i5.p1" class="ltx_para">
<p id="S3.I3.i5.p1.1" class="ltx_p"><span id="S3.I3.i5.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Edge Devices Diversity</span><span id="S3.I3.i5.p1.1.2" class="ltx_text" style="color:#000000;">. Healthcare relies on a diverse range of devices, from wearable fitness trackers to sophisticated medical equipment. Edge-based FL can adapt to this diversity, providing a flexible platform for healthcare applications.</span></p>
</div>
</li>
<li id="S3.I3.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S3.I3.i6.p1" class="ltx_para">
<p id="S3.I3.i6.p1.1" class="ltx_p"><span id="S3.I3.i6.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Cost-Efficiency</span><span id="S3.I3.i6.p1.1.2" class="ltx_text" style="color:#000000;">. Edge computing can reduce the costs associated with data transfer and centralized cloud infrastructure. FL’s decentralized training further optimizes resource usage.</span></p>
</div>
</li>
</ul>
<p id="S3.SS3.p7.2" class="ltx_p"><span id="S3.SS3.p7.2.1" class="ltx_text" style="color:#000000;">In summation, the integration of edge computing and FL unlocks new horizons in healthcare, driving innovation by reducing latency, conserving bandwidth, and safeguarding data privacy. This powerful synergy has the potential to revolutionize healthcare delivery, especially in remote and resource-constrained settings.</span></p>
</div>
</section>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section" style="color:#000000;">
<span class="ltx_tag ltx_tag_section">IV </span><span id="S4.1.1" class="ltx_text ltx_font_smallcaps">Practical Application of FL in Medical Domain</span>
</h2>

<div id="S4.p1" class="ltx_para">
<p id="S4.p1.1" class="ltx_p"><span id="S4.p1.1.1" class="ltx_text" style="color:#000000;">In this era of data-driven decision-making, the convergence of AI and healthcare holds tremendous promise. Among the various AI methodologies, FL emerges as a trailblazing approach that marries the potential of machine learning with the sensitivity of medical data. For example, medical institutions across the globe collaborate seamlessly, sharing insights and expertise, without compromising the privacy and security of patient records. FL makes this vision a reality by revolutionizing how healthcare organizations leverage data. It ushers in an era where medical breakthroughs are a collective endeavor and where the protection of individual privacy is of utmost importance. In this section, we embark through the practical applications of FL in the medical domain, exploring how this groundbreaking technology is reshaping patient care, research, and healthcare systems worldwide.</span></p>
</div>
<section id="S4.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S4.SS1.4.1.1" class="ltx_text">IV-A</span> </span><span id="S4.SS1.5.2" class="ltx_text ltx_font_italic">FL-enabled COVID-19 Detection</span>
</h3>

<div id="S4.SS1.p1" class="ltx_para">
<p id="S4.SS1.p1.1" class="ltx_p"><span id="S4.SS1.p1.1.1" class="ltx_text" style="color:#000000;">The realm of healthcare witnessed a transformative shift with the advent of smart healthcare and IoMT technologies. Amid global health crises such as the COVID-19 pandemic, the need for rapid, collaborative responses became paramount. FL, known for its privacy-preserving capabilities, emerged as a powerful tool in the fight against the pandemic. Researchers have harnessed FL’s potential to contribute to the analysis of the COVID-19 outbreak, leading to a wealth of innovative applications </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib98" title="" class="ltx_ref">98</a>, <a href="#bib.bib99" title="" class="ltx_ref">99</a>, <a href="#bib.bib100" title="" class="ltx_ref">100</a>, <a href="#bib.bib101" title="" class="ltx_ref">101</a>, <a href="#bib.bib102" title="" class="ltx_ref">102</a>, <a href="#bib.bib103" title="" class="ltx_ref">103</a><span id="S4.SS1.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p1.1.4" class="ltx_text" style="color:#000000;">. A comprehensive review of FL in COVID-19 detection can be found in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p1.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib104" title="" class="ltx_ref">104</a><span id="S4.SS1.p1.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p1.1.7" class="ltx_text" style="color:#000000;">, while </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p1.1.8.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib105" title="" class="ltx_ref">105</a><span id="S4.SS1.p1.1.9.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p1.1.10" class="ltx_text" style="color:#000000;"> offers insights into recent FL applications, limitations, and challenges in both COVID-19 and non-COVID-19 scenarios. Specifically, how the FL could provide valuable care during and post-COVID-19 was a question of interest.</span></p>
</div>
<div id="S4.SS1.p2" class="ltx_para ltx_indent">
<p id="S4.SS1.p2.1" class="ltx_p"><span id="S4.SS1.p2.1.1" class="ltx_text" style="color:#000000;">Realizing the lack of generalization in AI models, which prevents them from being used in clinical settings, one of the pioneering applications of FL in healthcare is the Unified  </span><span title="Computed Tomography" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Computed Tomography</span></span><span id="S4.SS1.p2.1.2" class="ltx_text" style="color:#000000;"> (</span><abbr title="Computed Tomography" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">CT</span></abbr><span id="S4.SS1.p2.1.3" class="ltx_text" style="color:#000000;">)-COVID AI Diagnostic Initiative (UCADI) </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p2.1.4.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib99" title="" class="ltx_ref">99</a>, <a href="#bib.bib106" title="" class="ltx_ref">106</a><span id="S4.SS1.p2.1.5.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p2.1.6" class="ltx_text" style="color:#000000;">. UCADI facilitates global collaboration in building a clinic CT-COVID AI application. By uniting efforts worldwide, it leverages diverse data sources for enhanced diagnostic accuracy while preserving privacy. FL has also shown promise in predicting mortality among hospitalized COVID-19 patients </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p2.1.7.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib101" title="" class="ltx_ref">101</a>, <a href="#bib.bib107" title="" class="ltx_ref">107</a><span id="S4.SS1.p2.1.8.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p2.1.9" class="ltx_text" style="color:#000000;">. By aggregating patient data from multiple hospitals within a health system, FL models outperformed locally trained models in predicting outcomes, demonstrating the potential to develop effective predictive tools without compromising patient privacy. Another groundbreaking initiative, the </span><abbr title="Electronic Medical Record" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">EMR</span></abbr><span id="S4.SS1.p2.1.10" class="ltx_text" style="color:#000000;"> Chest X-ray AI model (EXAM) </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p2.1.11.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib98" title="" class="ltx_ref">98</a><span id="S4.SS1.p2.1.12.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p2.1.13" class="ltx_text" style="color:#000000;">, utilizes FL to estimate future oxygen requirements for symptomatic COVID-19 patients. Drawing data from 20 global institutes, EXAM leverages vital signs, laboratory data, and chest X-rays to improve patient care. Collaborative FL frameworks enable medical institutions to screen COVID-19 from chest X-ray images without sharing patient data </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p2.1.14.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib108" title="" class="ltx_ref">108</a><span id="S4.SS1.p2.1.15.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p2.1.16" class="ltx_text" style="color:#000000;">. Numerous studies have proposed and verified FL’s effectiveness in COVID-19 X-ray data training and deployment experiments </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p2.1.17.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib109" title="" class="ltx_ref">109</a>, <a href="#bib.bib110" title="" class="ltx_ref">110</a>, <a href="#bib.bib111" title="" class="ltx_ref">111</a>, <a href="#bib.bib112" title="" class="ltx_ref">112</a>, <a href="#bib.bib113" title="" class="ltx_ref">113</a>, <a href="#bib.bib114" title="" class="ltx_ref">114</a>, <a href="#bib.bib115" title="" class="ltx_ref">115</a>, <a href="#bib.bib116" title="" class="ltx_ref">116</a>, <a href="#bib.bib117" title="" class="ltx_ref">117</a>, <a href="#bib.bib118" title="" class="ltx_ref">118</a>, <a href="#bib.bib119" title="" class="ltx_ref">119</a>, <a href="#bib.bib120" title="" class="ltx_ref">120</a>, <a href="#bib.bib121" title="" class="ltx_ref">121</a>, <a href="#bib.bib122" title="" class="ltx_ref">122</a><span id="S4.SS1.p2.1.18.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p2.1.19" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S4.SS1.p3" class="ltx_para ltx_indent">
<p id="S4.SS1.p3.1" class="ltx_p"><span id="S4.SS1.p3.1.1" class="ltx_text" style="color:#000000;">Hybrid framework for COVID-19 prediction via federated </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S4.SS1.p3.1.2" class="ltx_text" style="color:#000000;"> models is investigated in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p3.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib123" title="" class="ltx_ref">123</a><span id="S4.SS1.p3.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p3.1.5" class="ltx_text" style="color:#000000;">. An asynchronously updating FL model for mobile and deployable resource nodes to create local AI models for COVID-19 detection without explicit radiograph data exchange with the cloud is proposed in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p3.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib124" title="" class="ltx_ref">124</a><span id="S4.SS1.p3.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p3.1.8" class="ltx_text" style="color:#000000;">. In </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p3.1.9.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib125" title="" class="ltx_ref">125</a><span id="S4.SS1.p3.1.10.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p3.1.11" class="ltx_text" style="color:#000000;">, Qayyum et al. utilized the emerging concept of clustered FL (CFL) for an automatic diagnosis of COVID-19 at the edge by training a multi-modal ML model capable of detecting COVID-19 in both X-ray and Ultrasound imagery. The authors also envisaged that CFL is found to cope better with the divergence in data distribution from different sources than the conventional FL technique. Similarly, in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p3.1.12.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib126" title="" class="ltx_ref">126</a><span id="S4.SS1.p3.1.13.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p3.1.14" class="ltx_text" style="color:#000000;">, Chen et al. discussed the multi-modal COVID-19 discovery based on collaborative FL. Moreover, in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p3.1.15.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib127" title="" class="ltx_ref">127</a><span id="S4.SS1.p3.1.16.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p3.1.17" class="ltx_text" style="color:#000000;">, FL was used to predict acute kidney injury within three and seven days of admission in 4029 adults hospitalized with COVID-19 at five socio-demographically diverse New York City hospitals using demographics, comorbidities, vital signs, and laboratory values. The prediction performance of FL models was often found to be higher than that of single-hospital models and comparable to that of pooled-data models.</span></p>
</div>
<div id="S4.SS1.p4" class="ltx_para ltx_indent">
<p id="S4.SS1.p4.1" class="ltx_p"><span id="S4.SS1.p4.1.1" class="ltx_text" style="color:#000000;">A 5G-enabled architecture of auxiliary COVID-19 diagnosis based on FL for various institutions and central cloud cooperation was proposed by Wang et al. in order to realize the sharing of diagnosis models with high generalization performance </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p4.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib128" title="" class="ltx_ref">128</a><span id="S4.SS1.p4.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p4.1.4" class="ltx_text" style="color:#000000;">. A COVID-19 diagnosis model cognition framework was built on sharing and updating the model adaptively between the central and distributed nodes to interchange models and parameters. Federated </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S4.SS1.p4.1.5" class="ltx_text" style="color:#000000;"> for detecting COVID-19 lung abnormalities in CT through a multinational validation study was conducted in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p4.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib103" title="" class="ltx_ref">103</a><span id="S4.SS1.p4.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p4.1.8" class="ltx_text" style="color:#000000;">. The authors investigated FL strategies for developing an AI model for COVID-19 medical image diagnostics with good generalization capabilities on unseen multinational datasets. A secure international informatics FL infrastructure to investigate COVID-19 is initiated in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p4.1.9.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib100" title="" class="ltx_ref">100</a><span id="S4.SS1.p4.1.10.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p4.1.11" class="ltx_text" style="color:#000000;">. To address the privacy dilemma for COVID-19 data, the SCOR consortium has been formed that has created a ready-to-deploy secure FL infrastructure based on privacy and security technology. For COVID-19 impacted region segmentation in 3D chest CT, the authors in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS1.p4.1.12.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib102" title="" class="ltx_ref">102</a><span id="S4.SS1.p4.1.13.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS1.p4.1.14" class="ltx_text" style="color:#000000;"> presented a federated semi-supervised learning architecture using multinational COVID-19 data from China, Italy, &amp; Japan. The proposed approach can extract useful information from clients who only have unlabeled data.</span></p>
</div>
<div id="S4.SS1.p5" class="ltx_para ltx_indent">
<p id="S4.SS1.p5.1" class="ltx_p"><span id="S4.SS1.p5.1.1" class="ltx_text" style="color:#000000;">These applications exemplify how FL has emerged as a critical tool in the medical domain, offering global collaboration, predictive power, and privacy preservation in the fight against COVID-19 and beyond. During the pandemic, through FL, medical institutions worldwide could share insights and collectively train predictive models for virus spread, vaccine development, and treatment strategies while safeguarding patient information. FL’s decentralized nature allowed for real-time data analysis and rapid response. Healthcare professionals and researchers could continuously update and refine models as new data became available, leading to more accurate predictions and recommendations. Additionally, FL facilitated global collaboration, enabling experts from various regions to pool their data and expertise to combat the virus effectively. This collaborative approach was instrumental in accelerating vaccine research and development, as researchers worldwide could collectively analyze clinical trial data without compromising data privacy. Overall, FL emerged as a crucial tool in the fight against COVID-19 by enabling privacy-preserving data collaboration, real-time analysis, and global cooperation among healthcare professionals and researchers.</span></p>
</div>
</section>
<section id="S4.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S4.SS2.4.1.1" class="ltx_text">IV-B</span> </span><span id="S4.SS2.5.2" class="ltx_text ltx_font_italic">FL-enabled Activity Recognition for Elderly Care</span>
</h3>

<div id="S4.SS2.p1" class="ltx_para">
<p id="S4.SS2.p1.1" class="ltx_p"><span id="S4.SS2.p1.1.1" class="ltx_text" style="color:#000000;">The realm of human activities encompasses a diverse spectrum, ranging from sedentary behaviors such as sitting and standing to dynamic actions like walking and running. The domain of activity recognition, driven by computer vision, holds immense potential to revolutionize elderly care. It not only facilitates preventive and predictive interventions but also paves the way for personalized care through intelligent fall detection solutions, as demonstrated in the work by Brenvcivc et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS2.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib129" title="" class="ltx_ref">129</a><span id="S4.SS2.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS2.p1.1.4" class="ltx_text" style="color:#000000;">. Within this context, </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS2.p1.1.5" class="ltx_text" style="color:#000000;"> emerges as a compelling arena for advancing activity recognition’s role in elderly care, ushering in solutions tailored for aging populations while meticulously safeguarding their privacy.</span></p>
</div>
<div id="S4.SS2.p2" class="ltx_para">
<p id="S4.SS2.p2.1" class="ltx_p"><span id="S4.SS2.p2.1.1" class="ltx_text" style="color:#000000;">A pivotal study conducted by Sozinov et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS2.p2.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib130" title="" class="ltx_ref">130</a><span id="S4.SS2.p2.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS2.p2.1.4" class="ltx_text" style="color:#000000;"> illuminates the landscape of FL’s contribution to activity recognition. Their work compared the performance of FL against centralized learning methodologies, employing deep neural network architectures and softmax regression models. The investigation spanned both synthetic and real-world datasets, delving into crucial aspects such as transmission costs and the influence of erroneous clients with compromised data. The findings unveiled a nuanced scenario—while FL models exhibited slightly diminished accuracy compared to their centralized counterparts (achieving up to 89% accuracy as opposed to 93% in centralized models), the outcomes remained notably acceptable. This study underscores the potential of FL to deliver proficient models for human activity recognition tasks. Stepping further, the pursuit of efficient feature representation finds its expression in the innovative Federated Multi-Task Hierarchical Attention Model (FATHOM) proposed by Chen et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS2.p2.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib131" title="" class="ltx_ref">131</a><span id="S4.SS2.p2.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS2.p2.1.7" class="ltx_text" style="color:#000000;">. This novel framework intertwines classification and regression models from diverse sensors through a federated approach. By addressing the intricacies of multi-sensor data, FATHOM aspires to enhance the accuracy of activity recognition while embracing the distributed nature of FL. The architecture introduced by Bram et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS2.p2.1.8.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib132" title="" class="ltx_ref">132</a><span id="S4.SS2.p2.1.9.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS2.p2.1.10" class="ltx_text" style="color:#000000;"> introduces a powerful  </span><span title="Federated Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Federated Machine Learning</span></span><span id="S4.SS2.p2.1.11" class="ltx_text" style="color:#000000;"> (</span><abbr title="Federated Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FML</span></abbr><span id="S4.SS2.p2.1.12" class="ltx_text" style="color:#000000;">) software framework designed to harness the potential of deep neural networks. This architecture gracefully accommodates vast quantities of unlabeled data often dispersed across distributed clients, leveraging FL’s decentralized training paradigm to yield various neural network architectures. Intriguingly, FL’s applicability extends to the realm of elderly populations grappling with Alzheimer’s and dementia, as explored in Hesseberg’s work </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS2.p2.1.13.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib133" title="" class="ltx_ref">133</a><span id="S4.SS2.p2.1.14.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS2.p2.1.15" class="ltx_text" style="color:#000000;">. This study delves into FL’s potential to cater to the unique challenges faced by this demographic, opening avenues for enhancing care and understanding within the context of cognitive impairments.</span></p>
</div>
<div id="S4.SS2.p3" class="ltx_para">
<p id="S4.SS2.p3.1" class="ltx_p"><span id="S4.SS2.p3.1.1" class="ltx_text" style="color:#000000;">In the realm of activity recognition for enhanced elderly care, FL emerges as a potent tool, ushering in solutions that resonate with privacy-centric, personalized care paradigms. The studies elucidated here underscore FL’s capacity to augment accuracy, feature representation, and interdisciplinary collaborations, rendering it a pivotal catalyst in redefining elderly care in the era of advanced technology.</span></p>
</div>
</section>
<section id="S4.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S4.SS3.4.1.1" class="ltx_text">IV-C</span> </span><span id="S4.SS3.5.2" class="ltx_text ltx_font_italic">Practical Use Cases of FL for Medical Diagnostic Tools</span>
</h3>

<div id="S4.SS3.p1" class="ltx_para">
<p id="S4.SS3.p1.1" class="ltx_p"><span id="S4.SS3.p1.1.1" class="ltx_text" style="color:#000000;">Here, we will illustrate how </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p1.1.2" class="ltx_text" style="color:#000000;"> is applied in developing medical diagnosis tools, particularly in addressing the global burden of cancer and other non-communicable diseases.</span></p>
</div>
<div id="S4.SS3.p2" class="ltx_para">
<p id="S4.SS3.p2.1" class="ltx_p"><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p2.1.1" class="ltx_text" style="color:#000000;"> is being increasingly explored in medical image analysis to train the </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S4.SS3.p2.1.2" class="ltx_text" style="color:#000000;"> models on the large datasets distributed across multi-center and cross-border </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p2.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib134" title="" class="ltx_ref">134</a>, <a href="#bib.bib135" title="" class="ltx_ref">135</a>, <a href="#bib.bib136" title="" class="ltx_ref">136</a><span id="S4.SS3.p2.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p2.1.5" class="ltx_text" style="color:#000000;">. It allows each medical center to train a global model collaboratively in FL settings. Cross-domain collaborative and decentralized learning has several advantages. In the field of medical imaging, data annotation is one of the crucial and labour-intensive tasks. Using </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p2.1.6" class="ltx_text" style="color:#000000;">, different institutions can benefit from each other’s annotations without even sharing them. Training </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S4.SS3.p2.1.7" class="ltx_text" style="color:#000000;"> algorithms require high computational power and memory space. Using </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p2.1.8" class="ltx_text" style="color:#000000;"> can help in efficient training and memory consumption for AI-assisted medical image analysis algorithms. In this Section, we will briefly outline the existing works on different types of medical use cases and their application using </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p2.1.9" class="ltx_text" style="color:#000000;">. In this regard, we will cover examples of the most commonly occurring cancer in terms of incidence and the most deadly cancer in terms of mortality. We also discuss the application of </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p2.1.10" class="ltx_text" style="color:#000000;"> in developing the computer-aided diagnosis tool for those medical cases. The most frequently occurring cancer in terms of incidence are breast, lung, colorectum, prostate, stomach, liver, etc. (please refer to Fig. </span><a href="#S4.F5" title="Figure 5 ‣ IV-C Practical Use Cases of FL for Medical Diagnostic Tools ‣ IV Practical Application of FL in Medical Domain ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">5</span></a><span id="S4.SS3.p2.1.11" class="ltx_text" style="color:#000000;">). The most lethal cancers are lung cancer, colorectum cancer, liver cancer, stomach cancer, breast cancer, esophagus cancer, pancreas, etc. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p2.1.12.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib137" title="" class="ltx_ref">137</a><span id="S4.SS3.p2.1.13.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p2.1.14" class="ltx_text" style="color:#000000;"> (see Fig. </span><a href="#S4.F6" title="Figure 6 ‣ IV-C Practical Use Cases of FL for Medical Diagnostic Tools ‣ IV Practical Application of FL in Medical Domain ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">6</span></a><span id="S4.SS3.p2.1.15" class="ltx_text" style="color:#000000;">). Below, we briefly describe the most commonly caused cancer both in terms of incidence and mortality.</span></p>
</div>
<div id="S4.SS3.p3" class="ltx_para ltx_noindent">
<p id="S4.SS3.p3.1" class="ltx_p"><span id="S4.SS3.p3.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Glioblastoma tumor detection</span><span id="S4.SS3.p3.1.2" class="ltx_text" style="color:#000000;">. Researchers from a medical center at the University of Pennsylvania, Penn Medicine, and Intel Labs have used </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p3.1.3" class="ltx_text" style="color:#000000;"> to develop a model that can detect glioblastoma tumors in  </span><span title="Magnetic Resonance Imaging" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Magnetic Resonance Imaging</span></span><span id="S4.SS3.p3.1.4" class="ltx_text" style="color:#000000;"> (</span><abbr title="Magnetic Resonance Imaging" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">MRI</span></abbr><span id="S4.SS3.p3.1.5" class="ltx_text" style="color:#000000;">) images with 33% higher accuracy than traditional methods </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p3.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib138" title="" class="ltx_ref">138</a><span id="S4.SS3.p3.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p3.1.8" class="ltx_text" style="color:#000000;">. The model was trained on data from 71 sites across 6 continents, without any of the data ever leaving the individual institutions. This allowed the researchers to build a more robust and generalizable </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p3.1.9" class="ltx_text" style="color:#000000;"> model that can be used to diagnose glioblastoma in patients around the world.</span></p>
</div>
<figure id="S4.F5" class="ltx_figure"><img src="/html/2208.03392/assets/x3.png" id="S4.F5.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="597" height="1004" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_figure"><span id="S4.F5.7.1.1" class="ltx_text" style="font-size:90%;">Figure 5</span>: </span><span id="S4.F5.8.2" class="ltx_text" style="font-size:90%;">Estimated number of cancer incidences worldwide in 2020 (including both sexes and all ages). Breast cancer remains the leading cause of cancer, followed by lung, colorectum, prostate, stomach, liver, and esophagus cancer (GLOBOCAN 2020 statistics) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib137" title="" class="ltx_ref">137</a>]</cite>.</span></figcaption>
</figure>
<figure id="S4.F6" class="ltx_figure"><img src="/html/2208.03392/assets/x4.png" id="S4.F6.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="597" height="1004" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_figure"><span id="S4.F6.7.1.1" class="ltx_text" style="font-size:90%;">Figure 6</span>: </span><span id="S4.F6.8.2" class="ltx_text" style="font-size:90%;">Estimated number of cancer mortality in 2020 worldwide, including both sexes and all ages. Lung cancer remained the leading cause of cancer-related mortality, followed by colorectum, liver, stomach, breast, oesophagus, pancreas, and other cancer (GLOBOCAN 2020 statistics) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib137" title="" class="ltx_ref">137</a>]</cite>.</span></figcaption>
</figure>
<div id="S4.SS3.p4" class="ltx_para ltx_noindent">
<p id="S4.SS3.p4.1" class="ltx_p"><span id="S4.SS3.p4.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Lung cancer detection</span><span id="S4.SS3.p4.1.2" class="ltx_text" style="color:#000000;">. Lung cancer is the second leading cause of cancer in terms of incidence and the leading cause of cancer-related death worldwide. It accounts for 11.4% of all cancer-related incidence and accounts for 18% of global cancer-related death </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p4.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib137" title="" class="ltx_ref">137</a><span id="S4.SS3.p4.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p4.1.5" class="ltx_text" style="color:#000000;">. Adnan et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p4.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib139" title="" class="ltx_ref">139</a><span id="S4.SS3.p4.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p4.1.8" class="ltx_text" style="color:#000000;"> studied the effects of IID and non-IID distributions along with the healthcare providers by conducting a case study. They proposed a deferentially private FL framework as the potential solution to analyze histopathology images. For the experimentation, they used the publicly available Cancer Genome Atlas (TCGA) dataset for simulating a distributed environment and empirically compared the performance of private, distributed training to conventional training. Their work revealed differential private learning as a potential framework for the collaborative and decentralized development of </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S4.SS3.p4.1.9" class="ltx_text" style="color:#000000;"> models in medical image analysis tasks. In </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p4.1.10.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib140" title="" class="ltx_ref">140</a><span id="S4.SS3.p4.1.11.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p4.1.12" class="ltx_text" style="color:#000000;">, a team of researchers from the University of Pennsylvania have used </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p4.1.13" class="ltx_text" style="color:#000000;"> to develop a model that can detect lung cancer in CT scans. The model was trained on data from 15 different hospitals, without any of the data ever leaving the individual hospitals. This allowed the researchers to build a model that is robust to the different imaging protocols used by different hospitals and researchers.</span></p>
</div>
<div id="S4.SS3.p5" class="ltx_para ltx_noindent">
<p id="S4.SS3.p5.1" class="ltx_p"><span id="S4.SS3.p5.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Breast cancer detection</span><span id="S4.SS3.p5.1.2" class="ltx_text" style="color:#000000;">. Breast cancer is the first leading cause of cancer and the third leading cause of mortality. The Global Cancer Observatory (GLOBOCAN) accounts for 11.7% of the incidence and 6.9% of the mortality </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p5.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib137" title="" class="ltx_ref">137</a><span id="S4.SS3.p5.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p5.1.5" class="ltx_text" style="color:#000000;">. Roth et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p5.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib134" title="" class="ltx_ref">134</a><span id="S4.SS3.p5.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p5.1.8" class="ltx_text" style="color:#000000;"> used FL to train a </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S4.SS3.p5.1.9" class="ltx_text" style="color:#000000;">-based model for breast density classification using a collaborative approach with seven clinical institutions worldwide. Their experimental results showed that the model trained under an FL setting outperformed the individually trained model on each institute’s local data by 6.3% on average. Additionally, a relative improvement of 45.8% was observed in the model’s generalizability when the FL-based model was evaluated on the external testing datasets from other participating sites. These improvements were explained by the larger training datasets obtained from seven different institutions under FL settings without requiring to share any data and respecting privacy. Similarly, Sanchez et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p5.1.10.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib141" title="" class="ltx_ref">141</a><span id="S4.SS3.p5.1.11.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p5.1.12" class="ltx_text" style="color:#000000;"> defined a memory-aware curriculum learning method
for the </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p5.1.13" class="ltx_text" style="color:#000000;"> settings. Their curriculum controlled the order of the training samples and paid special attention to forgotten samples after the deployment of the global model. Their approach combined the unsupervised domain adaptation to deal with the domain shift problem while preserving data privacy. They evaluated their method on three clinical datasets from different centers. Their results showed that federated adversarial learning was advantageous for multi-site breast cancer classification. In </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p5.1.14.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib142" title="" class="ltx_ref">142</a><span id="S4.SS3.p5.1.15.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p5.1.16" class="ltx_text" style="color:#000000;">, the authors propose a novel method for </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p5.1.17" class="ltx_text" style="color:#000000;"> that uses curriculum learning to improve the classification performance of breast cancer models. Their method, called memory-aware curriculum </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p5.1.18" class="ltx_text" style="color:#000000;">, prioritizes the training samples that are forgotten after the deployment of the global model. This helps to ensure that the local models are better able to generalize to new data. Other researchers from the University of Pennsylvania have implemented a </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p5.1.19" class="ltx_text" style="color:#000000;">-based to calculate breast percent density </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p5.1.20.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib143" title="" class="ltx_ref">143</a><span id="S4.SS3.p5.1.21.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p5.1.22" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<div id="S4.SS3.p6" class="ltx_para ltx_noindent">
<p id="S4.SS3.p6.1" class="ltx_p"><span id="S4.SS3.p6.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Brain cancer detection</span><span id="S4.SS3.p6.1.2" class="ltx_text" style="color:#000000;">. </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p6.1.3" class="ltx_text" style="color:#000000;"> can be used to develop a model that performs brain cancer segmentation in medical imaging data, such as </span><abbr title="Magnetic Resonance Imaging" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">MRI</span></abbr><span id="S4.SS3.p6.1.4" class="ltx_text" style="color:#000000;"> scans. The model can learn from diverse datasets from different hospitals, enhancing its accuracy and generalization across various brain cancer types </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p6.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib135" title="" class="ltx_ref">135</a><span id="S4.SS3.p6.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p6.1.7" class="ltx_text" style="color:#000000;">. Brain tumour segmentation is an important process for separating tumours from normal tissue. In the clinical setting, it can provide key details for each part of the tumour that is useful for diagnosis and treatment planning. </span><abbr title="Magnetic Resonance Imaging" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">MRI</span></abbr><span id="S4.SS3.p6.1.8" class="ltx_text" style="color:#000000;"> has been extensively used by medical professionals for diagnosing brain tissue </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p6.1.9.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib144" title="" class="ltx_ref">144</a><span id="S4.SS3.p6.1.10.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p6.1.11" class="ltx_text" style="color:#000000;">. However, a manual process is time-consuming and can only be accomplished by professional neuroradiologists </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p6.1.12.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib145" title="" class="ltx_ref">145</a><span id="S4.SS3.p6.1.13.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p6.1.14" class="ltx_text" style="color:#000000;">. An automatic brain tumour segmentation can significantly impact brain tumour diagnosis and treatment. Li et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p6.1.15.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib146" title="" class="ltx_ref">146</a><span id="S4.SS3.p6.1.16.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p6.1.17" class="ltx_text" style="color:#000000;"> implemented the feasibility of using the differential-privacy technique to protect patient data in the </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p6.1.18" class="ltx_text" style="color:#000000;"> setup. As a use case, they considered the practical learning systems for brain tumour segmentation on the BraTS 2018 dataset </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p6.1.19.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib144" title="" class="ltx_ref">144</a>, <a href="#bib.bib147" title="" class="ltx_ref">147</a><span id="S4.SS3.p6.1.20.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p6.1.21" class="ltx_text" style="color:#000000;">. For the evaluation, they compared federated vs centralized data training. Their decentralized model converged at 300 training epochs, whereas the </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p6.1.22" class="ltx_text" style="color:#000000;"> converged at about 600 epochs. The data-centralized training took about 205.70s per epoch, whereas 65.45s plus small overheads.</span></p>
</div>
<div id="S4.SS3.p7" class="ltx_para">
<p id="S4.SS3.p7.1" class="ltx_p"><span id="S4.SS3.p7.1.1" class="ltx_text" style="color:#000000;">Similarly, Sheller et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p7.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib148" title="" class="ltx_ref">148</a><span id="S4.SS3.p7.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p7.1.4" class="ltx_text" style="color:#000000;"> performed a multi-institutional study where they studied the feasibility of brain tumour segmentation. Their federated semantic segmentation model achieved a dice coefficient of 0.8520 on multimodel brain scans and achieved a dice coefficient of 0.8620 with the model trained by sharing the dataset. Due to the lack of a centralized public dataset, they used 2018 BraTs data for training their </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p7.1.5" class="ltx_text" style="color:#000000;"> based effective segmentation model by iteratively aggregating the locally trained UNet segmentation models at a centralized server. They compared the performance of </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p7.1.6" class="ltx_text" style="color:#000000;"> models with traditional institutional incremental learning (IIL) and cyclic institutional incremental learning (CILL) and showed improvement over both techniques.</span></p>
</div>
<div id="S4.SS3.p8" class="ltx_para ltx_noindent">
<p id="S4.SS3.p8.1" class="ltx_p"><span id="S4.SS3.p8.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Colorectal polyp segmentation</span><span id="S4.SS3.p8.1.2" class="ltx_text" style="color:#000000;">.
Colorectal cancer is the third leading cause of cancer-related incidence and the second leading cause of death </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p8.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib137" title="" class="ltx_ref">137</a><span id="S4.SS3.p8.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p8.1.5" class="ltx_text" style="color:#000000;">. Colonoscopy is the gold standard for the early detection of cancer. However, there is a high adenoma miss-rate of 20-24% </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p8.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib149" title="" class="ltx_ref">149</a><span id="S4.SS3.p8.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p8.1.8" class="ltx_text" style="color:#000000;">. Moreover, the examination procedure is expensive and resource-demanding. A computer-aided diagnosis system can help detect neoplastic polyps during screening and aid in the early detection of cancer. FL can be one of the potential solutions to aid the development of computer-aided diagnosis systems for colonoscopy screening. To the best of our knowledge, there is no collaboration on developing decentralized AI in colorectal polyp segmentation. The reason for this is that the availability of a diverse and multi-center dataset remains one of the major challenges. To address the challenge of the lack of a multi-center dataset in the colonoscopy domain, Ali et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p8.1.9.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib150" title="" class="ltx_ref">150</a><span id="S4.SS3.p8.1.10.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p8.1.11" class="ltx_text" style="color:#000000;"> have curated and publicly released the PolypGen dataset, a six-center dataset that is suitable for exploring </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p8.1.12" class="ltx_text" style="color:#000000;">. PolypGen consists of 3762 polyps frames from more than 300 patients’ datasets. It consists of datasets from six unique clinical institutions (Norway, France, Italy, Egypt, and the UK). Each center dataset is stored in a separate folder. Thus, this dataset can be advantageous for training multiple </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S4.SS3.p8.1.13" class="ltx_text" style="color:#000000;">-based models independently on the colonoscopy datasets on separate computers and sharing the learned model weight. Although there has been work done on PolypGen dataset </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p8.1.14.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib151" title="" class="ltx_ref">151</a><span id="S4.SS3.p8.1.15.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p8.1.16" class="ltx_text" style="color:#000000;">, training the model on the FL-based paradigm still remains an open research area.</span></p>
</div>
<div id="S4.SS3.p9" class="ltx_para ltx_noindent">
<p id="S4.SS3.p9.1" class="ltx_p"><span id="S4.SS3.p9.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Prostate cancer segmentation</span><span id="S4.SS3.p9.1.2" class="ltx_text" style="color:#000000;">. Prostate cancer is the second leading cause of cancer-related death (after lung cancer) in the American male population </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p9.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib152" title="" class="ltx_ref">152</a><span id="S4.SS3.p9.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p9.1.5" class="ltx_text" style="color:#000000;">. An accurate and automatic prostate segmentation can play an important role in facilitating diagnostic and therapeutic applications such as early detection of cancer, patient management, treatment planning, and surgical planning </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p9.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib153" title="" class="ltx_ref">153</a><span id="S4.SS3.p9.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p9.1.8" class="ltx_text" style="color:#000000;">. Manual segmentation is the commonly used method for the segmentation of prostate and prostate regions despite being a time-consuming and subjective process. In this regard, Sarma et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p9.1.9.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib154" title="" class="ltx_ref">154</a><span id="S4.SS3.p9.1.10.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p9.1.11" class="ltx_text" style="color:#000000;"> performed a multicenter study on the whole prostate segmentation on the axial T2- weighted </span><abbr title="Magnetic Resonance Imaging" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">MRI</span></abbr><span id="S4.SS3.p9.1.12" class="ltx_text" style="color:#000000;"> scans. They used the 3D Hybrid Anisotropic Hybrid Network </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p9.1.13.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib155" title="" class="ltx_ref">155</a><span id="S4.SS3.p9.1.14.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p9.1.15" class="ltx_text" style="color:#000000;"> model to train on each institution’s data. Additionally, a single model was trained using </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p9.1.16" class="ltx_text" style="color:#000000;"> across all three institutions. </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p9.1.17" class="ltx_text" style="color:#000000;"> based model had better performance and generalizability than the model trained on single institutions.</span></p>
</div>
<div id="S4.SS3.p10" class="ltx_para">
<p id="S4.SS3.p10.1" class="ltx_p"><span id="S4.SS3.p10.1.1" class="ltx_text" style="color:#000000;">Xia et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p10.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib45" title="" class="ltx_ref">45</a><span id="S4.SS3.p10.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p10.1.4" class="ltx_text" style="color:#000000;"> proposed addressing the federated domain generalization problem. They proposed Federated Domain Generalization (FedG), a generalizable model that aims to learn a federated model from multiple distributed source generalizations that suffered from performance drops on unseen data outside of the federation. They trained FedG on a six prostrate dataset. The experimental results showed superior efficacy of FedG over the state-of-the-art methods. Similarly, Liu et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p10.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib156" title="" class="ltx_ref">156</a><span id="S4.SS3.p10.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p10.1.7" class="ltx_text" style="color:#000000;"> highlighted a critical challenge in FL, where models trained within the federation may experience a significant performance drop when tested on entirely new hospitals located outside of the federation. In response to this challenge, they introduced a pioneering solution known as “Episodic Learning in Continuous Frequency Space (EL-CFS)”. This approach empowers each client to leverage multi-source data distributions, all while navigating the complex landscape of decentralized data. The effectiveness of EL-CFS becomes evident through its application to two distinct medical image segmentation datasets. The first dataset involves prostate MRI segmentation, encompassing data from six different medical centers. The second dataset comprises retinal fundus images, specifically focusing on the optic disc and optic cup, collected from four diverse medical centers. Comparative analyses, both qualitatively and quantitatively, consistently demonstrate the superior performance and efficiency of the EL-CFS algorithm.</span></p>
</div>
<div id="S4.SS3.p11" class="ltx_para ltx_noindent">
<p id="S4.SS3.p11.1" class="ltx_p"><span id="S4.SS3.p11.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Small bowel disease detection</span><span id="S4.SS3.p11.1.2" class="ltx_text" style="color:#000000;">.
Capsule endoscopy is the procedure to examine small intestine disease. The physicians recommend a capsule endoscopy procedure for the identification of gastrointestinal bleeding, diagnosing inflammatory disease (Crohn’s disease), cancer, small intestine polyp, etc. The development of a </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S4.SS3.p11.1.3" class="ltx_text" style="color:#000000;">-based computer-aided diagnosis system can improve the anomaly detection rate in the small intestine and reduce manual labour. However, the capsule endoscopy dataset is scarce, and no multi-center dataset is publicly available for experimentation. One of the large-scale datasets in this field is KvasirCapsule dataset </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p11.1.4.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib157" title="" class="ltx_ref">157</a><span id="S4.SS3.p11.1.5.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p11.1.6" class="ltx_text" style="color:#000000;"> that contains 4,741,504 image frames. This dataset was collected at Baerum hospital at Vestre Viken (VV) Hospital Trust in Norway. VV consists of four hospitals; therefore, this dataset can be treated as a multi-center dataset. However, the dataset from each center is not provided in a separate folder and no information is provided about the labelled and unlabelled videos at which hospital they were collected. To the best of our knowledge, there is no work on video capsule endoscopy for small bowel disease detection or gastrointestinal tract bleeding detection using </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p11.1.7" class="ltx_text" style="color:#000000;">. Both exploring the dataset in the </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p11.1.8" class="ltx_text" style="color:#000000;"> setting and the use of </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p11.1.9" class="ltx_text" style="color:#000000;"> for small bowel disease detection remain open issues.</span></p>
</div>
<div id="S4.SS3.p12" class="ltx_para ltx_noindent">
<p id="S4.SS3.p12.1" class="ltx_p"><span id="S4.SS3.p12.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Liver segmentation</span><span id="S4.SS3.p12.1.2" class="ltx_text" style="color:#000000;">.
Liver disease is of high importance because of its high incidence. It is the third leading cause of mortality in both men and women </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p12.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib137" title="" class="ltx_ref">137</a><span id="S4.SS3.p12.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p12.1.5" class="ltx_text" style="color:#000000;"> (see Fig. </span><a href="#S4.F6" title="Figure 6 ‣ IV-C Practical Use Cases of FL for Medical Diagnostic Tools ‣ IV Practical Application of FL in Medical Domain ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">6</span></a><span id="S4.SS3.p12.1.6" class="ltx_text" style="color:#000000;">). An early diagnosis of liver disease can play an important role in treating and preventing damage. Liver segmentation is the most commonly adopted technique for diagnosis and follow-up treatment. </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p12.1.7" class="ltx_text" style="color:#000000;">-based solutions recently gained popularity by training the shared global models by distributed clients with heterogeneous image datasets. Bernecker et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p12.1.8.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib158" title="" class="ltx_ref">158</a><span id="S4.SS3.p12.1.9.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p12.1.10" class="ltx_text" style="color:#000000;"> proposed two </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p12.1.11" class="ltx_text" style="color:#000000;"> algorithms, namely FedNorm and FedNorm+, that were based on modality-based normalization techniques. They validated their method on the multi-modal and multi-institutional datasets (6 centers, 428 patients). They obtained the high performance of dice coefficients up to 0.9610 and consistently outperformed locally trained models that were trained in each center. In some cases, the federated models also even surpassed centralized models. Similarly, recently Xu et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p12.1.12.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib159" title="" class="ltx_ref">159</a><span id="S4.SS3.p12.1.13.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p12.1.14" class="ltx_text" style="color:#000000;"> presented a new federated multi-organ segmentation U-Net for the segmentation of multiple organs. Here, the multi-encoding U-Net (MENU-Net) extracts organ-specific features via encoding sub-networks. Out of the different sub-networks, each of the sub-network is considered an expert of a specific organ and is trained for that client. They have regularized the training of the MENU-Net by designing an auxiliary generic decoder. This was performed to encourage organ-specific feature enhancement (for example, becoming distinctive and informative). They have trained their method on four abdominal CT image datasets (liver, kidney, pancreas, and BTCV). The extensive experimental results on four different datasets showed the potential of the proposed method in solving the partial level problem in </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p12.1.15" class="ltx_text" style="color:#000000;"> context.</span></p>
</div>
<div id="S4.SS3.p13" class="ltx_para ltx_noindent">
<p id="S4.SS3.p13.1" class="ltx_p"><span id="S4.SS3.p13.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Pancreas segmentation</span><span id="S4.SS3.p13.1.2" class="ltx_text" style="color:#000000;">.
Pancreas cancer is among the most lethal malignant tumours characterized by high mortality </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p13.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib160" title="" class="ltx_ref">160</a><span id="S4.SS3.p13.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p13.1.5" class="ltx_text" style="color:#000000;">. The GLOBOCAN 2020 statistic shows that it is the seventh leading lethal cancer (see Fig. </span><a href="#S4.F6" title="Figure 6 ‣ IV-C Practical Use Cases of FL for Medical Diagnostic Tools ‣ IV Practical Application of FL in Medical Domain ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">6</span></a><span id="S4.SS3.p13.1.6" class="ltx_text" style="color:#000000;">). The five-year survival rate of the patient with the disease is less than nine years </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p13.1.7.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib161" title="" class="ltx_ref">161</a><span id="S4.SS3.p13.1.8.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p13.1.9" class="ltx_text" style="color:#000000;">. Therefore, accurate segmentation of pancreatic cancer can play an important role in clinical diagnosis and treatment. In this respect, Shen et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p13.1.10.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib162" title="" class="ltx_ref">162</a><span id="S4.SS3.p13.1.11.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p13.1.12" class="ltx_text" style="color:#000000;"> investigated heterogeneous optimization methods with abdominal CT images from the pancreas and pancreatic tumours in </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p13.1.13" class="ltx_text" style="color:#000000;"> based settings. They utilized three publicly available annotated pancreas segmentation datasets (one consisting of pancreas and tumour, whereas the other consists of healthy pancreas cases) to model three different heterogeneous clients during </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p13.1.14" class="ltx_text" style="color:#000000;"> settings. They evaluated each method’s performance in the </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p13.1.15" class="ltx_text" style="color:#000000;"> settings and compared their proposed global model’s performance with </span><em id="S4.SS3.p13.1.16" class="ltx_emph ltx_font_italic" style="color:#000000;">FedAvg</em><span id="S4.SS3.p13.1.17" class="ltx_text" style="color:#000000;"> and </span><em id="S4.SS3.p13.1.18" class="ltx_emph ltx_font_italic" style="color:#000000;">FedProx</em><span id="S4.SS3.p13.1.19" class="ltx_text" style="color:#000000;">. Their experimental results revealed that global models were better than local models for all datasets. Similarly, Wang et al. </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p13.1.20.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib163" title="" class="ltx_ref">163</a><span id="S4.SS3.p13.1.21.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p13.1.22" class="ltx_text" style="color:#000000;"> trained the </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p13.1.23" class="ltx_text" style="color:#000000;"> models for the segmentation of the pancreas without data sharing. Their dataset (abdominal CT images) consisted of healthy and unhealthy pancreas collected from two different institutions in Taiwan and Japan. The model utilized for segmentation was coarse-to-fine network architecture search </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p13.1.24.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib164" title="" class="ltx_ref">164</a><span id="S4.SS3.p13.1.25.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p13.1.26" class="ltx_text" style="color:#000000;"> along with variational auto-encoder </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p13.1.27.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib165" title="" class="ltx_ref">165</a><span id="S4.SS3.p13.1.28.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p13.1.29" class="ltx_text" style="color:#000000;"> at the encoder endpoint. The quantitative results comparison between </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p13.1.30" class="ltx_text" style="color:#000000;"> settings and locally trained algorithms showed that </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p13.1.31" class="ltx_text" style="color:#000000;"> framework was better in dealing with unbalanced data distributions and successfully delivered better generalizable models compared to that of standalone training.</span></p>
</div>
<div id="S4.SS3.p14" class="ltx_para ltx_noindent">
<p id="S4.SS3.p14.1" class="ltx_p"><span id="S4.SS3.p14.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Treatment planning</span><span id="S4.SS3.p14.1.2" class="ltx_text" style="color:#000000;">.


Beyond its diagnostic applications, FL finds valuable utility in enhancing treatment responses. A prominent example of its application in the medical field lies within radiation therapy. FL-based models provide a collaborative solution that empowers radiation oncologists to devise the most optimal treatment plans tailored to individual patients, a process known as personalized treatment planning </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p14.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib166" title="" class="ltx_ref">166</a><span id="S4.SS3.p14.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p14.1.5" class="ltx_text" style="color:#000000;">. This approach proves particularly advantageous in conditions like lung cancer, where radiation therapy is utilized. The scope of FL’s contributions extends beyond personalized treatment. It encompasses various facets of radiotherapy, including contouring, optimal dose prediction, tumor motion forecasting, image guidance, and survival prognosis. These advancements pave the way for precision and effectiveness in patient treatment, ultimately benefiting those undergoing therapy. Moreover, the incorporation of FL not only optimizes therapeutic outcomes but also ensures the security and confidentiality of patient data in alignment with regulatory standards. This dual focus on treatment efficacy and data privacy safeguards sensitive medical information. As distributed technology continues to evolve, we can anticipate further innovative applications of FL in the ongoing battle against cancer and other chronic diseases.</span></p>
</div>
<figure id="S4.T3" class="ltx_table">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_table"><span id="S4.T3.11.1.1" class="ltx_text" style="font-size:90%;">TABLE III</span>: </span><span id="S4.T3.12.2" class="ltx_text" style="font-size:90%;">Enhanced performance and generalizability of FL in clinical use <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib154" title="" class="ltx_ref">154</a>]</cite>.</span></figcaption>
<table id="S4.T3.4" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S4.T3.4.5.1" class="ltx_tr">
<th id="S4.T3.4.5.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_t" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.4.5.1.1.1" class="ltx_text" style="color:#000000;">Architecture</span></th>
<th id="S4.T3.4.5.1.2" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.4.5.1.2.1" class="ltx_text" style="color:#000000;">Institution</span></th>
<th id="S4.T3.4.5.1.3" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.4.5.1.3.1" class="ltx_text" style="color:#000000;">Dice coefficient</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S4.T3.1.1" class="ltx_tr">
<th id="S4.T3.1.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_t" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.1.1.2.1" class="ltx_text" style="color:#000000;">Private model</span></th>
<td id="S4.T3.1.1.3" class="ltx_td ltx_align_center ltx_border_t" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.1.1.3.1" class="ltx_text" style="color:#000000;">NCI</span></td>
<td id="S4.T3.1.1.1" class="ltx_td ltx_align_center ltx_border_t" style="padding-top:2.5pt;padding-bottom:2.5pt;">
<span id="S4.T3.1.1.1.1" class="ltx_text" style="color:#000000;">0.872 </span><math id="S4.T3.1.1.1.m1.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="S4.T3.1.1.1.m1.1a"><mo mathcolor="#000000" id="S4.T3.1.1.1.m1.1.1" xref="S4.T3.1.1.1.m1.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="S4.T3.1.1.1.m1.1b"><csymbol cd="latexml" id="S4.T3.1.1.1.m1.1.1.cmml" xref="S4.T3.1.1.1.m1.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="S4.T3.1.1.1.m1.1c">\pm</annotation></semantics></math><span id="S4.T3.1.1.1.2" class="ltx_text" style="color:#000000;"> 0.062</span>
</td>
</tr>
<tr id="S4.T3.2.2" class="ltx_tr">
<th id="S4.T3.2.2.2" class="ltx_td ltx_align_left ltx_th ltx_th_row" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.2.2.2.1" class="ltx_text" style="color:#000000;">Private model</span></th>
<td id="S4.T3.2.2.3" class="ltx_td ltx_align_center" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.2.2.3.1" class="ltx_text" style="color:#000000;">SUNY</span></td>
<td id="S4.T3.2.2.1" class="ltx_td ltx_align_center" style="padding-top:2.5pt;padding-bottom:2.5pt;">
<span id="S4.T3.2.2.1.1" class="ltx_text" style="color:#000000;">0.838 </span><math id="S4.T3.2.2.1.m1.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="S4.T3.2.2.1.m1.1a"><mo mathcolor="#000000" id="S4.T3.2.2.1.m1.1.1" xref="S4.T3.2.2.1.m1.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="S4.T3.2.2.1.m1.1b"><csymbol cd="latexml" id="S4.T3.2.2.1.m1.1.1.cmml" xref="S4.T3.2.2.1.m1.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="S4.T3.2.2.1.m1.1c">\pm</annotation></semantics></math><span id="S4.T3.2.2.1.2" class="ltx_text" style="color:#000000;"> 0.043</span>
</td>
</tr>
<tr id="S4.T3.3.3" class="ltx_tr">
<th id="S4.T3.3.3.2" class="ltx_td ltx_align_left ltx_th ltx_th_row" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.3.3.2.1" class="ltx_text" style="color:#000000;">Private model</span></th>
<td id="S4.T3.3.3.3" class="ltx_td ltx_align_center" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.3.3.3.1" class="ltx_text" style="color:#000000;">UCLA</span></td>
<td id="S4.T3.3.3.1" class="ltx_td ltx_align_center" style="padding-top:2.5pt;padding-bottom:2.5pt;">
<span id="S4.T3.3.3.1.1" class="ltx_text" style="color:#000000;">0.812 </span><math id="S4.T3.3.3.1.m1.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="S4.T3.3.3.1.m1.1a"><mo mathcolor="#000000" id="S4.T3.3.3.1.m1.1.1" xref="S4.T3.3.3.1.m1.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="S4.T3.3.3.1.m1.1b"><csymbol cd="latexml" id="S4.T3.3.3.1.m1.1.1.cmml" xref="S4.T3.3.3.1.m1.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="S4.T3.3.3.1.m1.1c">\pm</annotation></semantics></math><span id="S4.T3.3.3.1.2" class="ltx_text" style="color:#000000;"> 0.136</span>
</td>
</tr>
<tr id="S4.T3.4.4" class="ltx_tr">
<th id="S4.T3.4.4.2" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_b" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.4.4.2.1" class="ltx_text" style="color:#000000;">FL Model</span></th>
<td id="S4.T3.4.4.3" class="ltx_td ltx_align_center ltx_border_b" style="padding-top:2.5pt;padding-bottom:2.5pt;"><span id="S4.T3.4.4.3.1" class="ltx_text" style="color:#000000;">Collaborative</span></td>
<td id="S4.T3.4.4.1" class="ltx_td ltx_align_center ltx_border_b" style="padding-top:2.5pt;padding-bottom:2.5pt;">
<span id="S4.T3.4.4.1.1" class="ltx_text" style="color:#000000;">0.889 </span><math id="S4.T3.4.4.1.m1.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="S4.T3.4.4.1.m1.1a"><mo mathcolor="#000000" id="S4.T3.4.4.1.m1.1.1" xref="S4.T3.4.4.1.m1.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="S4.T3.4.4.1.m1.1b"><csymbol cd="latexml" id="S4.T3.4.4.1.m1.1.1.cmml" xref="S4.T3.4.4.1.m1.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="S4.T3.4.4.1.m1.1c">\pm</annotation></semantics></math><span id="S4.T3.4.4.1.2" class="ltx_text" style="color:#000000;"> 0.036</span>
</td>
</tr>
</tbody>
</table>
</figure>
<div id="S4.SS3.p15" class="ltx_para">
<p id="S4.SS3.p15.1" class="ltx_p"><span id="S4.SS3.p15.1.1" class="ltx_text" style="color:#000000;">In </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p15.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib154" title="" class="ltx_ref">154</a><span id="S4.SS3.p15.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p15.1.4" class="ltx_text" style="color:#000000;">, the efficacy of FL has been exemplified through a multi-institutional collaboration involving three esteemed academic establishments: the University of California, Los Angeles (UCLA), the State University of New York (SUNY) Upstate Medical University, and the National Cancer Institute (NCI) in the United States. This endeavour harnessed authentic clinical prostate imaging data to underscore the substantial capabilities of the FL framework, as medical imaging data for each of these intuitions was vital, and data privacy was the stringent requirement. Applied to the domain of medical image analysis, the study focused on the intricate task of whole prostate segmentation, which constitutes an initial stride in the accurate diagnosis of cancer through </span><abbr title="Magnetic Resonance Imaging" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">MRI</span></abbr><span id="S4.SS3.p15.1.5" class="ltx_text" style="color:#000000;"> scans and in guiding fusion-based interventions. This research manifests the broader implications of FL’s proficiency, with specific emphasis on its generalization capacity. The approach employed entailed the training and aggregation of models in a federated manner, thereby engendering a composite model equipped with predictive weights of a generalized nature, adaptable to the idiosyncrasies of each institutional dataset. Remarkably, this composite model exhibited good generalizability, as corroborated by its superior performance on both held-out test sets from the individual institutions and on external validation dataset.</span></p>
</div>
<div id="S4.SS3.p16" class="ltx_para">
<p id="S4.SS3.p16.1" class="ltx_p"><span id="S4.SS3.p16.1.1" class="ltx_text" style="color:#000000;">An insightful evaluation of the findings, as elucidated in Table </span><a href="#S4.T3" title="TABLE III ‣ IV-C Practical Use Cases of FL for Medical Diagnostic Tools ‣ IV Practical Application of FL in Medical Domain ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">III</span></a><span id="S4.SS3.p16.1.2" class="ltx_text" style="color:#000000;">, underscores the intricacies of performance. The private models, while showcasing a diverse spectrum of competencies on the ProstateX dataset </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S4.SS3.p16.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib167" title="" class="ltx_ref">167</a><span id="S4.SS3.p16.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S4.SS3.p16.1.5" class="ltx_text" style="color:#000000;"> (as evidenced by Dice coefficient values ranging from 0.812 to 0.872), were collectively overshadowed by the </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S4.SS3.p16.1.6" class="ltx_text" style="color:#000000;"> model. This latter model, exemplifying an aggregate mean Dice coefficient of 0.889, emerged as the definitive winner in performance.</span></p>
</div>
<div id="S4.SS3.p17" class="ltx_para">
<p id="S4.SS3.p17.1" class="ltx_p"><span id="S4.SS3.p17.1.1" class="ltx_text" style="color:#000000;">Employing rigorous statistical analyses, the study reaffirmed the empirical superiority of the FL paradigm over the individual private model. This substantiates the effectiveness and practical viability of FL’s capacity to transcend the limitations of isolated data silos and underscores its pivotal role in bolstering generalizability and predictive efficacy across diverse healthcare datasets.</span></p>
</div>
</section>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section" style="color:#000000;">
<span class="ltx_tag ltx_tag_section">V </span><span id="S5.1.1" class="ltx_text ltx_font_smallcaps">Open-Source FL Software Frameworks</span>
</h2>

<div id="S5.p1" class="ltx_para">
<p id="S5.p1.1" class="ltx_p"><span id="S5.p1.1.1" class="ltx_text" style="color:#000000;">In this section, we delve into the landscape of open-source </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S5.p1.1.2" class="ltx_text" style="color:#000000;"> software frameworks, pivotal tools for rapid prototyping, experimentation, and validation of novel FL algorithms. Open-source FL software in healthcare ensures transparency and trust. Healthcare practitioners and institutions can inspect the code, verify security measures, and understand the algorithms being used, which is crucial for maintaining data privacy and complying with healthcare regulations. Open source also promotes innovation. Developers from diverse backgrounds can freely build upon existing FL solutions, tailoring them to address unique challenges in healthcare, such as disease diagnosis, drug discovery, or treatment personalization. This accelerates the development of FL applications in the medical field. While some proprietary FL frameworks exist, such as IBM FL </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib168" title="" class="ltx_ref">168</a><span id="S5.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.p1.1.5" class="ltx_text" style="color:#000000;">, NVIDIA CLARA </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.p1.1.6.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib169" title="" class="ltx_ref">169</a><span id="S5.p1.1.7.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.p1.1.8" class="ltx_text" style="color:#000000;">, Substra </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.p1.1.9.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib170" title="" class="ltx_ref">170</a><span id="S5.p1.1.10.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.p1.1.11" class="ltx_text" style="color:#000000;"> and Sherpa.AI FL framework </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.p1.1.12.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib171" title="" class="ltx_ref">171</a><span id="S5.p1.1.13.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.p1.1.14" class="ltx_text" style="color:#000000;"> their lack of comprehensive documentation and intricate details can hinder the seamless exploration of new FL methodologies. Consequently, we focus here on elucidating the available open-source FL software frameworks. The ensuing sub-sections unravel the nuances of each framework, and a comprehensive comparison is distilled in Table</span><a href="#S5.T4" title="TABLE IV ‣ V Open-Source FL Software Frameworks ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">IV</span></a><span id="S5.p1.1.15" class="ltx_text" style="color:#000000;">.</span></p>
</div>
<figure id="S5.T4" class="ltx_table">
<figcaption class="ltx_caption ltx_centering" style="color:#000000;"><span class="ltx_tag ltx_tag_table"><span id="S5.T4.4.1.1" class="ltx_text" style="font-size:90%;">TABLE IV</span>: </span><span id="S5.T4.5.2" class="ltx_text" style="font-size:90%;">Comparison of open-source FL software frameworks.</span></figcaption>
<table id="S5.T4.6" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S5.T4.6.1.1" class="ltx_tr">
<th id="S5.T4.6.1.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Features</span></th>
<td id="S5.T4.6.1.1.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.2.1" class="ltx_text ltx_font_bold" style="color:#000000;">FATE</span></td>
<td id="S5.T4.6.1.1.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.3.1" class="ltx_text ltx_font_bold" style="color:#000000;">TFF</span></td>
<td id="S5.T4.6.1.1.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.4.1" class="ltx_text ltx_font_bold" style="color:#000000;">OpenFL</span></td>
<td id="S5.T4.6.1.1.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.5.1" class="ltx_text ltx_font_bold" style="color:#000000;">Fed-BioMed</span></td>
<td id="S5.T4.6.1.1.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.6.1" class="ltx_text ltx_font_bold" style="color:#000000;">PySyft</span></td>
<td id="S5.T4.6.1.1.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.7.1" class="ltx_text ltx_font_bold" style="color:#000000;">FedML</span></td>
<td id="S5.T4.6.1.1.8" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.8.1" class="ltx_text ltx_font_bold" style="color:#000000;">LEAF</span></td>
<td id="S5.T4.6.1.1.9" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.1.1.9.1" class="ltx_text ltx_font_bold" style="color:#000000;">PaddleFL</span></td>
<td id="S5.T4.6.1.1.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S5.T4.6.1.1.10.1" class="ltx_text ltx_font_bold" style="color:#000000;">PrivacyFL</span></td>
</tr>
<tr id="S5.T4.6.2.2" class="ltx_tr">
<th id="S5.T4.6.2.2.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.1.1" class="ltx_text" style="color:#000000;">FL model with different datasets</span></th>
<td id="S5.T4.6.2.2.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.2.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.2.2.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.3.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.2.2.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.4.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.2.2.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.5.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.2.2.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.6.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.2.2.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.7.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.2.2.8" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.8.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.2.2.9" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.2.2.9.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.2.2.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S5.T4.6.2.2.10.1" class="ltx_text" style="color:#000000;">Yes</span></td>
</tr>
<tr id="S5.T4.6.3.3" class="ltx_tr">
<th id="S5.T4.6.3.3.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.1.1" class="ltx_text" style="color:#000000;">FL attack simulator</span></th>
<td id="S5.T4.6.3.3.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.2.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.3.3.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.3.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.3.3.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.4.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.3.3.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.5.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.3.3.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.6.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.3.3.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.7.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.3.3.8" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.8.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.3.3.9" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.3.3.9.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.3.3.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S5.T4.6.3.3.10.1" class="ltx_text" style="color:#000000;">Yes</span></td>
</tr>
<tr id="S5.T4.6.4.4" class="ltx_tr">
<th id="S5.T4.6.4.4.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.1.1" class="ltx_text" style="color:#000000;">Documentation and tutorials details</span></th>
<td id="S5.T4.6.4.4.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.2.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.4.4.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.3.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.4.4.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.4.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.4.4.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.5.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.4.4.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.6.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.4.4.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.7.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.4.4.8" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.8.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.4.4.9" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.4.4.9.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.4.4.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S5.T4.6.4.4.10.1" class="ltx_text" style="color:#000000;">No</span></td>
</tr>
<tr id="S5.T4.6.5.5" class="ltx_tr">
<th id="S5.T4.6.5.5.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.1.1" class="ltx_text" style="color:#000000;">Other library support</span></th>
<td id="S5.T4.6.5.5.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.2.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.5.5.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.3.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.5.5.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.4.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.5.5.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.5.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.5.5.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.6.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.5.5.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.7.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.5.5.8" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.8.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.5.5.9" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.5.5.9.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.5.5.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S5.T4.6.5.5.10.1" class="ltx_text" style="color:#000000;">No</span></td>
</tr>
<tr id="S5.T4.6.6.6" class="ltx_tr">
<th id="S5.T4.6.6.6.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.1.1" class="ltx_text" style="color:#000000;">Data Partitioning</span></th>
<td id="S5.T4.6.6.6.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.2.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.6.6.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.3.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.6.6.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.4.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.6.6.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.5.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.6.6.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.6.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.6.6.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.7.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.6.6.8" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.8.1" class="ltx_text" style="color:#000000;">Partial</span></td>
<td id="S5.T4.6.6.6.9" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.6.6.9.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.6.6.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S5.T4.6.6.6.10.1" class="ltx_text" style="color:#000000;">Partial</span></td>
</tr>
<tr id="S5.T4.6.7.7" class="ltx_tr">
<th id="S5.T4.6.7.7.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t"><span id="S5.T4.6.7.7.1.1" class="ltx_text" style="color:#000000;">Hardware support</span></th>
<td id="S5.T4.6.7.7.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.7.7.2.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
<td id="S5.T4.6.7.7.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S5.T4.6.7.7.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S5.T4.6.7.7.3.1.1" class="ltx_tr">
<td id="S5.T4.6.7.7.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S5.T4.6.7.7.3.1.1.1.1" class="ltx_text" style="color:#000000;">GPUs</span></td>
</tr>
<tr id="S5.T4.6.7.7.3.1.2" class="ltx_tr">
<td id="S5.T4.6.7.7.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S5.T4.6.7.7.3.1.2.1.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
</tr>
</table>
</td>
<td id="S5.T4.6.7.7.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.7.7.4.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
<td id="S5.T4.6.7.7.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S5.T4.6.7.7.5.1" class="ltx_tabular ltx_align_middle">
<tr id="S5.T4.6.7.7.5.1.1" class="ltx_tr">
<td id="S5.T4.6.7.7.5.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S5.T4.6.7.7.5.1.1.1.1" class="ltx_text" style="color:#000000;">GPUs</span></td>
</tr>
<tr id="S5.T4.6.7.7.5.1.2" class="ltx_tr">
<td id="S5.T4.6.7.7.5.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S5.T4.6.7.7.5.1.2.1.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
</tr>
</table>
</td>
<td id="S5.T4.6.7.7.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S5.T4.6.7.7.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S5.T4.6.7.7.6.1.1" class="ltx_tr">
<td id="S5.T4.6.7.7.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S5.T4.6.7.7.6.1.1.1.1" class="ltx_text" style="color:#000000;">GPUs</span></td>
</tr>
<tr id="S5.T4.6.7.7.6.1.2" class="ltx_tr">
<td id="S5.T4.6.7.7.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S5.T4.6.7.7.6.1.2.1.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
</tr>
</table>
</td>
<td id="S5.T4.6.7.7.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S5.T4.6.7.7.7.1" class="ltx_tabular ltx_align_middle">
<tr id="S5.T4.6.7.7.7.1.1" class="ltx_tr">
<td id="S5.T4.6.7.7.7.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S5.T4.6.7.7.7.1.1.1.1" class="ltx_text" style="color:#000000;">GPUs</span></td>
</tr>
<tr id="S5.T4.6.7.7.7.1.2" class="ltx_tr">
<td id="S5.T4.6.7.7.7.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S5.T4.6.7.7.7.1.2.1.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
</tr>
</table>
</td>
<td id="S5.T4.6.7.7.8" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.7.7.8.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
<td id="S5.T4.6.7.7.9" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S5.T4.6.7.7.9.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
<td id="S5.T4.6.7.7.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S5.T4.6.7.7.10.1" class="ltx_text" style="color:#000000;">CPUs</span></td>
</tr>
<tr id="S5.T4.6.8.8" class="ltx_tr">
<th id="S5.T4.6.8.8.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.1.1" class="ltx_text" style="color:#000000;">FL specific for medical applications</span></th>
<td id="S5.T4.6.8.8.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.2.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.8.8.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.3.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.8.8.4" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.4.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.8.8.5" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.5.1" class="ltx_text" style="color:#000000;">Yes</span></td>
<td id="S5.T4.6.8.8.6" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.6.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.8.8.7" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.7.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.8.8.8" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.8.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.8.8.9" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S5.T4.6.8.8.9.1" class="ltx_text" style="color:#000000;">No</span></td>
<td id="S5.T4.6.8.8.10" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S5.T4.6.8.8.10.1" class="ltx_text" style="color:#000000;">No</span></td>
</tr>
</tbody>
</table>
</figure>
<section id="S5.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS1.4.1.1" class="ltx_text">V-A</span> </span><span id="S5.SS1.5.2" class="ltx_text ltx_font_italic">Federated AI Technology Enabler Framework (FATE)</span>
</h3>

<div id="S5.SS1.p1" class="ltx_para">
<p id="S5.SS1.p1.1" class="ltx_p"><span id="S5.SS1.p1.1.1" class="ltx_text" style="color:#000000;">FATE, is a compelling open-source endeavor spearheaded by Webank, engineered to establish a secure computing platform for the federated AI ecosystem </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS1.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib172" title="" class="ltx_ref">172</a><span id="S5.SS1.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS1.p1.1.4" class="ltx_text" style="color:#000000;">. With FATE, diverse secure computation protocols seamlessly interlace, enabling collaborative data analysis while adhering to stringent data protection regulations. Notably, FATE’s script-based interface streamlines integration, albeit relying on command-line options and a specialized yet underspecified domain-specific language.</span></p>
</div>
</section>
<section id="S5.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS2.4.1.1" class="ltx_text">V-B</span> </span><span id="S5.SS2.5.2" class="ltx_text ltx_font_italic"> <span title="Tensor Flow Federated" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Tensor Flow Federated</span></span> (<abbr title="Tensor Flow Federated" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">TFF</span></abbr>)</span>
</h3>

<div id="S5.SS2.p1" class="ltx_para">
<p id="S5.SS2.p1.1" class="ltx_p"><abbr title="Tensor Flow Federated" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">TFF</span></abbr><span id="S5.SS2.p1.1.1" class="ltx_text" style="color:#000000;">, a prominent open-source venture, serves as a foundational framework for decentralized </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S5.SS2.p1.1.2" class="ltx_text" style="color:#000000;"> on distributed data </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS2.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib173" title="" class="ltx_ref">173</a><span id="S5.SS2.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS2.p1.1.5" class="ltx_text" style="color:#000000;">. TFF boasts dual layers—FL API and Federated Core API—catering to both ML developers and systems researchers. This multi-layered approach enables innovative algorithmic experiments, simulation of FL protocols, and model compatibility with TensorFlow models. However, it’s noteworthy that TFF’s arsenal lacks dedicated mechanisms for differential privacy, posing a minor constraint for privacy-preserving innovations.</span></p>
</div>
</section>
<section id="S5.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS3.4.1.1" class="ltx_text">V-C</span> </span><span id="S5.SS3.5.2" class="ltx_text ltx_font_italic">Open-<abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr> (OpenFL)</span>
</h3>

<div id="S5.SS3.p1" class="ltx_para">
<p id="S5.SS3.p1.1" class="ltx_p"><span id="S5.SS3.p1.1.1" class="ltx_text" style="color:#000000;">OpenFL emerges as a versatile open-source framework specifically tailored for training </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S5.SS3.p1.1.2" class="ltx_text" style="color:#000000;"> models under the paradigm of FL, particularly in the medical landscape </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS3.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib174" title="" class="ltx_ref">174</a><span id="S5.SS3.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS3.p1.1.5" class="ltx_text" style="color:#000000;">. This dynamic framework harmoniously coexists with TensorFlow and PyTorch training pipelines, and its adaptability extends to a plethora of ML and </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S5.SS3.p1.1.6" class="ltx_text" style="color:#000000;"> frameworks. With OpenFL, practitioners can power the Federated Tumor Segmentation (FeTS) initiative (Federated Tumor Segmentation)</span><span id="footnote1" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_tag ltx_tag_note">1</span>https://www.med.upenn.edu/cbica/fets/</span></span></span><span id="S5.SS3.p1.1.7" class="ltx_text" style="color:#000000;">, a real-world medical FL platform for tumour segmentation. This endorsement underscores OpenFL’s suitability for medical applications, where data privacy and collaboration are paramount. The framework’s architecture, encompassing both collaborator and aggregator components, orchestrates the training process on local datasets and the subsequent aggregation of model updates, facilitating collaborative learning. However, OpenFL currently lacks dedicated differential privacy mechanisms.</span></p>
</div>
</section>
<section id="S5.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS4.4.1.1" class="ltx_text">V-D</span> </span><span id="S5.SS4.5.2" class="ltx_text ltx_font_italic">Fed-BioMed</span>
</h3>

<div id="S5.SS4.p1" class="ltx_para">
<p id="S5.SS4.p1.1" class="ltx_p"><span id="S5.SS4.p1.1.1" class="ltx_text" style="color:#000000;">Fed-BioMed</span><span id="footnote2" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">2</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">2</sup><span class="ltx_tag ltx_tag_note">2</span>https://fedbiomed.gitlabpages.inria.fr/</span></span></span><span id="S5.SS4.p1.1.2" class="ltx_text" style="color:#000000;"> is an open-source initiative aimed at enabling biomedical research using non-centralized statistical analysis and </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S5.SS4.p1.1.3" class="ltx_text" style="color:#000000;"> methodologies </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS4.p1.1.4.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib175" title="" class="ltx_ref">175</a><span id="S5.SS4.p1.1.5.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS4.p1.1.6" class="ltx_text" style="color:#000000;">. The project, which is currently based on Python, PyTorch, and Scikit-learn, allows for the development and deployment of FL analysis in real-world </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S5.SS4.p1.1.7" class="ltx_text" style="color:#000000;"> systems. Fed-BioMed is an ongoing initiative, and it still needs to be developed to be scalable and fully deployed in a practical healthcare setting.</span></p>
</div>
</section>
<section id="S5.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS5.4.1.1" class="ltx_text">V-E</span> </span><span id="S5.SS5.5.2" class="ltx_text ltx_font_italic">PySyft</span>
</h3>

<div id="S5.SS5.p1" class="ltx_para">
<p id="S5.SS5.p1.1" class="ltx_p"><span id="S5.SS5.p1.1.1" class="ltx_text" style="color:#000000;">PySyft is an open-source Python3 library that utilizes FL, differential privacy, and encrypted computations to enable FL for research purposes </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS5.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib176" title="" class="ltx_ref">176</a><span id="S5.SS5.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS5.p1.1.4" class="ltx_text" style="color:#000000;">. It was created by the OpenMined</span><span id="footnote3" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">3</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">3</sup><span class="ltx_tag ltx_tag_note">3</span>https://www.openmined.org/</span></span></span><span id="S5.SS5.p1.1.5" class="ltx_text" style="color:#000000;"> community and mostly works with </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S5.SS5.p1.1.6" class="ltx_text" style="color:#000000;"> frameworks like PyTorch and TensorFlow. PySyft defines objects, </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S5.SS5.p1.1.7" class="ltx_text" style="color:#000000;"> methods, and abstractions. With PySyft, it is not possible as of now to work on ML/AI projects requiring network communication. This would necessitate the use of another package known as PyGrid</span><span id="footnote4" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">4</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">4</sup><span class="ltx_tag ltx_tag_note">4</span>https://blog.openmined.org/what-is-pygrid-demo/</span></span></span><span id="S5.SS5.p1.1.8" class="ltx_text" style="color:#000000;">. Moreover, it does not provide any DP mechanism nor any DP algorithm.</span></p>
</div>
</section>
<section id="S5.SS6" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS6.4.1.1" class="ltx_text">V-F</span> </span><span id="S5.SS6.5.2" class="ltx_text ltx_font_italic">FedML</span>
</h3>

<div id="S5.SS6.p1" class="ltx_para">
<p id="S5.SS6.p1.1" class="ltx_p"><span id="S5.SS6.p1.1.1" class="ltx_text" style="color:#000000;">FedML positions itself as a research-oriented open-source benchmark and framework, expertly engineered to streamline FL algorithm development and performance evaluations </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS6.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib177" title="" class="ltx_ref">177</a><span id="S5.SS6.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS6.p1.1.4" class="ltx_text" style="color:#000000;">. This versatile framework accommodates diverse computing paradigms, ranging from on-device training for edge devices to distributed computing and single-machine simulations. Its lightweight Edge AI SDK finds application in a range of hardware setups, including edge GPUs, smartphones, and IoT devices. It’s worth noting that while FedML’s potential is significant, its current documentation requires enhancement, and differential privacy components are not yet integrated.</span></p>
</div>
</section>
<section id="S5.SS7" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS7.4.1.1" class="ltx_text">V-G</span> </span><span id="S5.SS7.5.2" class="ltx_text ltx_font_italic">LEAF</span>
</h3>

<div id="S5.SS7.p1" class="ltx_para">
<p id="S5.SS7.p1.1" class="ltx_p"><span id="S5.SS7.p1.1.1" class="ltx_text" style="color:#000000;">LEAF, a benchmarking framework dedicated to FL, serves as an invaluable resource for researchers probing diverse domains like multi-task learning, meta-learning, and on-device learning </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS7.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib178" title="" class="ltx_ref">178</a><span id="S5.SS7.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS7.p1.1.4" class="ltx_text" style="color:#000000;">. Crafted through a mosaic of open-source datasets, statistics, and reference implementations, LEAF empowers researchers to explore novel methodologies with a level of pragmatism unprecedented in prior benchmarks. It’s pertinent to observe that while LEAF offers a substantial suite of resources, official documentation and dedicated benchmarks for differential privacy are areas that warrant further attention.</span></p>
</div>
</section>
<section id="S5.SS8" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS8.4.1.1" class="ltx_text">V-H</span> </span><span id="S5.SS8.5.2" class="ltx_text ltx_font_italic">PaddleFL</span>
</h3>

<div id="S5.SS8.p1" class="ltx_para">
<p id="S5.SS8.p1.1" class="ltx_p"><span id="S5.SS8.p1.1.1" class="ltx_text" style="color:#000000;">PaddleFL, fortified by PaddlePaddle, emerges as a robust open-source FL framework designed to cater to industrial-scale parallel distributed </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S5.SS8.p1.1.2" class="ltx_text" style="color:#000000;"> </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS8.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib179" title="" class="ltx_ref">179</a><span id="S5.SS8.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS8.p1.1.5" class="ltx_text" style="color:#000000;">. Its architecture equips researchers to swiftly reproduce and evaluate diverse FL techniques and algorithms, ensuring compatibility with an array of parallel distributed clusters. Most notably, PaddleFL leverages differential private stochastic gradient descent, offering enhanced privacy-preserving capabilities. It’s essential to acknowledge that PaddleFL, despite its strengths, currently grapples with inadequately detailed documentation and a need for streamlined extensibility.</span></p>
</div>
</section>
<section id="S5.SS9" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS9.4.1.1" class="ltx_text">V-I</span> </span><span id="S5.SS9.5.2" class="ltx_text ltx_font_italic">PrivacyFL</span>
</h3>

<div id="S5.SS9.p1" class="ltx_para">
<p id="S5.SS9.p1.1" class="ltx_p"><span id="S5.SS9.p1.1.1" class="ltx_text" style="color:#000000;">PrivacyFL, a scalable and configurable open-source FL framework, positions itself as a versatile solution that couples advanced analytics with security measures </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S5.SS9.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib180" title="" class="ltx_ref">180</a><span id="S5.SS9.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S5.SS9.p1.1.4" class="ltx_text" style="color:#000000;">. Armed with features like latency simulation, resilience against client departure or failure, and differential privacy mechanisms, PrivacyFL caters to diverse requirements of FL within an array of scenarios. It’s important to note that while PrivacyFL excels in configurability, it remains an emerging framework and could benefit from comprehensive documentation to ease the implementation of new FL algorithms.</span>
<br class="ltx_break"></p>
</div>
<div id="S5.SS9.p2" class="ltx_para">
<p id="S5.SS9.p2.1" class="ltx_p"><span id="S5.SS9.p2.1.1" class="ltx_text" style="color:#000000;">Overall, these open-source software development for FL Learning in healthcare not only democratizes access to advanced machine learning tools but also encourages collaboration, innovation, transparency, and the growth of a vibrant community dedicated to improving patient care and healthcare outcomes. Based on our findings, we have identified OpenFL and Fed-BioMed as particularly suitable FL software frameworks for medical applications. OpenFL, with its emphasis on collaborative learning and compatibility with medical data paradigms, positions itself as a promising tool for enhancing medical research and diagnostics. On the other hand, Fed-BioMed’s focus on enabling biomedical research through decentralized statistical analysis and </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S5.SS9.p2.1.2" class="ltx_text" style="color:#000000;"> methodologies aligns well with the intricacies of medical data and privacy considerations.</span></p>
</div>
</section>
</section>
<section id="S6" class="ltx_section">
<h2 class="ltx_title ltx_title_section" style="color:#000000;">
<span class="ltx_tag ltx_tag_section">VI </span><span id="S6.1.1" class="ltx_text ltx_font_smallcaps">Challenges of <abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr> in Healthcare</span>
</h2>

<div id="S6.p1" class="ltx_para">
<p id="S6.p1.1" class="ltx_p"><span id="S6.p1.1.1" class="ltx_text" style="color:#000000;">While FL offers significant promise in the context of the healthcare setting, it is essential to acknowledge and address its associated challenges and limitations. Some of the main challenges of FL in real-world medical environments include communication, statistical, computational, and security and privacy challenges </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib181" title="" class="ltx_ref">181</a><span id="S6.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.p1.1.4" class="ltx_text" style="color:#000000;">. In the following sub-sections, we explain these challenges in detail, emphasizing their relevance within the medical domain.</span></p>
</div>
<section id="S6.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS1.4.1.1" class="ltx_text">VI-A</span> </span><span id="S6.SS1.5.2" class="ltx_text ltx_font_italic">Communication Challenges</span>
</h3>

<div id="S6.SS1.p1" class="ltx_para">
<p id="S6.SS1.p1.1" class="ltx_p"><span id="S6.SS1.p1.1.1" class="ltx_text" style="color:#000000;">Effective communication is important in </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S6.SS1.p1.1.2" class="ltx_text" style="color:#000000;">, where nodes or clients exchange information in a centralized FL system or within a distributed setup. FL requires multiple communication rounds among these entities to train a global model, which can strain communication bandwidth resources </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS1.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib182" title="" class="ltx_ref">182</a><span id="S6.SS1.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS1.p1.1.5" class="ltx_text" style="color:#000000;">. In healthcare, where facilities may be geographically dispersed, unreliable or unstable network connections can hinder the FL process. This network reliability disparity across healthcare providers poses a significant hurdle, potentially impeding collaborative training of robust machine learning models. Here are some key aspects to consider:</span></p>
</div>
<div id="S6.SS1.p2" class="ltx_para">
<ul id="S6.I1" class="ltx_itemize">
<li id="S6.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I1.i1.p1" class="ltx_para">
<p id="S6.I1.i1.p1.1" class="ltx_p"><span id="S6.I1.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Bandwidth Bottlenecks</span><span id="S6.I1.i1.p1.1.2" class="ltx_text" style="color:#000000;">. </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S6.I1.i1.p1.1.3" class="ltx_text" style="color:#000000;"> necessitates continuous communication between clients and a central server or between clients themselves in distributed FL. Each communication round involves the exchange of model updates or gradients, which can consume substantial communication bandwidth. In healthcare environments with a high number of participating nodes, this can strain available resources, leading to bandwidth bottlenecks that slow down the FL process.</span></p>
</div>
</li>
<li id="S6.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I1.i2.p1" class="ltx_para">
<p id="S6.I1.i2.p1.1" class="ltx_p"><span id="S6.I1.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Geographical Disparities</span><span id="S6.I1.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Healthcare providers are often geographically dispersed. This geographical diversity can result in unreliable or unstable network connections, which can hinder the smooth flow of data and model updates. Rural healthcare facilities, in particular, may face connectivity challenges, making it difficult for them to actively participate in FL processes.</span></p>
</div>
</li>
<li id="S6.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I1.i3.p1" class="ltx_para">
<p id="S6.I1.i3.p1.1" class="ltx_p"><span id="S6.I1.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Latency Issues</span><span id="S6.I1.i3.p1.1.2" class="ltx_text" style="color:#000000;">. Real-time decision-making is critical in healthcare, and FL’s reliance on communication rounds introduces latency. For time-sensitive applications, such as telemedicine or remote patient monitoring, delays in communication can have adverse consequences. Reducing latency while ensuring data security remains a complex challenge in implementing FL in healthcare.</span></p>
</div>
</li>
<li id="S6.I1.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I1.i4.p1" class="ltx_para">
<p id="S6.I1.i4.p1.1" class="ltx_p"><span id="S6.I1.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Privacy Concerns</span><span id="S6.I1.i4.p1.1.2" class="ltx_text" style="color:#000000;">. Communication during FL must be secure to protect sensitive healthcare data. Ensuring end-to-end encryption and data integrity during data transmission is crucial to mitigate privacy risks. Healthcare regulations, such as  </span><span title="Health Insurance Portability and Accountability Act" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Health Insurance Portability and Accountability Act</span></span><span id="S6.I1.i4.p1.1.3" class="ltx_text" style="color:#000000;"> (</span><abbr title="Health Insurance Portability and Accountability Act" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">HIPAA</span></abbr><span id="S6.I1.i4.p1.1.4" class="ltx_text" style="color:#000000;">) in the United States and </span><abbr title="General Data Protection Regulation" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">GDPR</span></abbr><span id="S6.I1.i4.p1.1.5" class="ltx_text" style="color:#000000;"> in Europe, impose stringent data security and privacy requirements, making robust communication protocols essential.</span></p>
</div>
</li>
</ul>
</div>
</section>
<section id="S6.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS2.4.1.1" class="ltx_text">VI-B</span> </span><span id="S6.SS2.5.2" class="ltx_text ltx_font_italic">Statistical Challenges</span>
</h3>

<div id="S6.SS2.p1" class="ltx_para">
<p id="S6.SS2.p1.1" class="ltx_p"><span id="S6.SS2.p1.1.1" class="ltx_text" style="color:#000000;">The statistical challenge, i.e., data distribution among the available clients in the network participating in the FL process, is one of the main challenges of FL, particularly in healthcare applications. EHR is considered one of the prominent sources of healthcare data for FL applications. A detailed survey of existing works on EHR data for FL applications is outlined in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS2.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib183" title="" class="ltx_ref">183</a><span id="S6.SS2.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS2.p1.1.4" class="ltx_text" style="color:#000000;">. It is to be noted that healthcare data is inherently diverse and subject to variations across different facilities and regions. Here’s a deeper exploration of these challenges:</span></p>
</div>
<div id="S6.SS2.p2" class="ltx_para">
<ul id="S6.I2" class="ltx_itemize">
<li id="S6.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I2.i1.p1" class="ltx_para">
<p id="S6.I2.i1.p1.1" class="ltx_p"><span id="S6.I2.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Heterogeneity</span><span id="S6.I2.i1.p1.1.2" class="ltx_text" style="color:#000000;">. Healthcare data comes from various sources including EHRs, medical imaging, wearable devices, and genomics. Hence, handling and federating this heterogeneous data is understandably a significant challenge. EHRs are a primary data source for healthcare FL, but they can be heterogeneous due to variations in data formats, coding systems, and data collection practices. Integrating and harmonizing such diverse data sources for effective FL is challenging.</span></p>
</div>
</li>
<li id="S6.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I2.i2.p1" class="ltx_para">
<p id="S6.I2.i2.p1.1" class="ltx_p"><span id="S6.I2.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Bias and Generalizability</span><span id="S6.I2.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Training machine learning models on data from a single hospital or clinic can introduce bias and limit the model’s generalizability. Patient populations, disease prevalence, and treatment protocols may vary across healthcare facilities, leading to model biases that may not translate well to other settings. Achieving model fairness and robustness against bias is a complex statistical problem.</span></p>
</div>
</li>
<li id="S6.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I2.i3.p1" class="ltx_para">
<p id="S6.I2.i3.p1.1" class="ltx_p"><span id="S6.I2.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Sparsity</span><span id="S6.I2.i3.p1.1.2" class="ltx_text" style="color:#000000;">. Smaller healthcare clinics, especially in remote or underserved areas, may have limited EHR data. This data sparsity can hinder their participation in FL processes. Moreover, these clinics may lack the resources to label medical data comprehensively, leading to unlabelled data, which is challenging to leverage effectively in machine learning.</span></p>
</div>
</li>
<li id="S6.I2.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I2.i4.p1" class="ltx_para">
<p id="S6.I2.i4.p1.1" class="ltx_p"><span id="S6.I2.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Non-I.I.D. Data Distribution</span><span id="S6.I2.i4.p1.1.2" class="ltx_text" style="color:#000000;">. The common assumption in federated optimization is Identically and Independently Distributed (I.I.D.) data across clients. However, in healthcare FL, this assumption often doesn’t hold true due to the inherent variations in healthcare data. Non-I.I.D. data distribution complicates the optimization process, increases the risk of straggler clients, and may add modelling, analysis, and evaluation complexity for FL algorithms </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.I2.i4.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib184" title="" class="ltx_ref">184</a><span id="S6.I2.i4.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.I2.i4.p1.1.5" class="ltx_text" style="color:#000000;">.</span></p>
</div>
</li>
</ul>
</div>
</section>
<section id="S6.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS3.4.1.1" class="ltx_text">VI-C</span> </span><span id="S6.SS3.5.2" class="ltx_text ltx_font_italic">Computational Challenges</span>
</h3>

<div id="S6.SS3.p1" class="ltx_para">
<p id="S6.SS3.p1.1" class="ltx_p"><span id="S6.SS3.p1.1.1" class="ltx_text" style="color:#000000;">It has been predicted that more than 80% generated data by the edge devices will be processed at the edge of the network itself by 2030 </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS3.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib185" title="" class="ltx_ref">185</a><span id="S6.SS3.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS3.p1.1.4" class="ltx_text" style="color:#000000;">. Thus, there is an increased interest in using AI either on edge devices or at the edge of the network. The computation challenges in healthcare-based FL are intertwined with the increasing adoption of Edge-AI and the diverse computational capabilities of healthcare devices. These challenges are pivotal for efficient model training and deployment:</span></p>
</div>
<div id="S6.SS3.p2" class="ltx_para">
<ul id="S6.I3" class="ltx_itemize">
<li id="S6.I3.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I3.i1.p1" class="ltx_para">
<p id="S6.I3.i1.p1.1" class="ltx_p"><span id="S6.I3.i1.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Edge Device Proliferation</span><span id="S6.I3.i1.p1.1.2" class="ltx_text" style="color:#000000;">. The healthcare landscape is witnessing a proliferation of edge devices, including wearable health gadgets, smartwatches, and IoT sensors. These devices generate a substantial volume of healthcare data and present opportunities for on-device AI. However, their computational capacities, storage capabilities, and energy profiles vary significantly.</span></p>
</div>
</li>
<li id="S6.I3.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I3.i2.p1" class="ltx_para">
<p id="S6.I3.i2.p1.1" class="ltx_p"><span id="S6.I3.i2.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Device Reliability</span><span id="S6.I3.i2.p1.1.2" class="ltx_text" style="color:#000000;">. Many edge devices are energy-constrained and may stop working during FL iterations. This can affect data quality and introduce unpredictability into the FL process. Ensuring that these devices reliably participate in FL while managing their limited resources remains a computational challenge </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.I3.i2.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib186" title="" class="ltx_ref">186</a><span id="S6.I3.i2.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.I3.i2.p1.1.5" class="ltx_text" style="color:#000000;">.</span></p>
</div>
</li>
<li id="S6.I3.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I3.i3.p1" class="ltx_para">
<p id="S6.I3.i3.p1.1" class="ltx_p"><span id="S6.I3.i3.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Lightweight Models</span><span id="S6.I3.i3.p1.1.2" class="ltx_text" style="color:#000000;">. In cross-device FL settings, where on-device AI is prevalent, machine learning models must be lightweight and energy-efficient to run effectively on resource-constrained devices. Balancing model performance with resource constraints is an ongoing challenge in healthcare FL.</span></p>
</div>
</li>
<li id="S6.I3.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I3.i4.p1" class="ltx_para">
<p id="S6.I3.i4.p1.1" class="ltx_p"><span id="S6.I3.i4.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Data Disparity in Cross-Silo FL</span><span id="S6.I3.i4.p1.1.2" class="ltx_text" style="color:#000000;">. Cross-silo FL, involving multiple healthcare institutions, is a common architecture in healthcare. However, the data disparity among these institutions may require a higher number of FL iterations to achieve the desired model accuracy on a global server </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.I3.i4.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib187" title="" class="ltx_ref">187</a>, <a href="#bib.bib188" title="" class="ltx_ref">188</a><span id="S6.I3.i4.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.I3.i4.p1.1.5" class="ltx_text" style="color:#000000;">. This increased computation time poses challenges in maintaining efficiency and responsiveness in healthcare applications.</span></p>
</div>
</li>
<li id="S6.I3.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S6.I3.i5.p1" class="ltx_para">
<p id="S6.I3.i5.p1.1" class="ltx_p"><span id="S6.I3.i5.p1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Scalability</span><span id="S6.I3.i5.p1.1.2" class="ltx_text" style="color:#000000;">. </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S6.I3.i5.p1.1.3" class="ltx_text" style="color:#000000;"> models can be computationally expensive to train and deploy. Therefore, scalability can be a critical challenge for modern healthcare organizations that have limited resources </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.I3.i5.p1.1.4.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib181" title="" class="ltx_ref">181</a>, <a href="#bib.bib189" title="" class="ltx_ref">189</a><span id="S6.I3.i5.p1.1.5.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.I3.i5.p1.1.6" class="ltx_text" style="color:#000000;">.</span></p>
</div>
</li>
</ul>
</div>
</section>
<section id="S6.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS4.4.1.1" class="ltx_text">VI-D</span> </span><span id="S6.SS4.5.2" class="ltx_text ltx_font_italic">Security and Privacy Challenges</span>
</h3>

<div id="S6.SS4.p1" class="ltx_para">
<p id="S6.SS4.p1.1" class="ltx_p"><span id="S6.SS4.p1.1.1" class="ltx_text" style="color:#000000;">Security and privacy are paramount in FL-based healthcare due to the sensitive nature of medical data. Ensuring data protection while fostering collaborative model training is a complex endeavor. Moreover, the use of </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S6.SS4.p1.1.2" class="ltx_text" style="color:#000000;"> in the healthcare setting is subject to a variety of regulations which can vary from country to country </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS4.p1.1.3.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib190" title="" class="ltx_ref">190</a><span id="S6.SS4.p1.1.4.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS4.p1.1.5" class="ltx_text" style="color:#000000;">. Hence, compliance with healthcare regulations like </span><abbr title="Health Insurance Portability and Accountability Act" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">HIPAA</span></abbr><span id="S6.SS4.p1.1.6" class="ltx_text" style="color:#000000;"> </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS4.p1.1.7.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib191" title="" class="ltx_ref">191</a>, <a href="#bib.bib192" title="" class="ltx_ref">192</a>, <a href="#bib.bib193" title="" class="ltx_ref">193</a><span id="S6.SS4.p1.1.8.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS4.p1.1.9" class="ltx_text" style="color:#000000;"> and </span><abbr title="General Data Protection Regulation" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">GDPR</span></abbr><span id="S6.SS4.p1.1.10" class="ltx_text" style="color:#000000;"> </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS4.p1.1.11.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib11" title="" class="ltx_ref">11</a><span id="S6.SS4.p1.1.12.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS4.p1.1.13" class="ltx_text" style="color:#000000;"> adds complexity to the security and privacy implementations in FL settings. As shown in Fig. </span><a href="#S1.F2" title="Figure 2 ‣ I-B Paper Organizations ‣ I Introduction ‣ Federated Learning for Medical Applications: A Taxonomy, Current Trends, Challenges, and Future Research Directions" class="ltx_ref" style="color:#000000;"><span class="ltx_text ltx_ref_tag">2</span></a><span id="S6.SS4.p1.1.14" class="ltx_text" style="color:#000000;">, without having direct access to the client’s local data, FL systems allow distributed clients to collaborate and train </span><abbr title="Machine Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">ML</span></abbr><span id="S6.SS4.p1.1.15" class="ltx_text" style="color:#000000;"> and </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S6.SS4.p1.1.16" class="ltx_text" style="color:#000000;"> models by sharing training parameters. However, instead of centralizing raw data, sharing gradient updates to a central server trained on the local clients could lead to reverse engineering attacks by passively intercepting the gradients exchanged during the training process </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS4.p1.1.17.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib194" title="" class="ltx_ref">194</a>, <a href="#bib.bib195" title="" class="ltx_ref">195</a><span id="S6.SS4.p1.1.18.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS4.p1.1.19" class="ltx_text" style="color:#000000;">. This means malicious clients can introduce a backdoor functionality that compromises the underlying FL system during the training process of the global federated model </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS4.p1.1.20.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib196" title="" class="ltx_ref">196</a><span id="S6.SS4.p1.1.21.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS4.p1.1.22" class="ltx_text" style="color:#000000;">, which can be against the </span><abbr title="General Data Protection Regulation" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">GDPR</span></abbr><span id="S6.SS4.p1.1.23" class="ltx_text" style="color:#000000;"> since medical data are highly sensitive and private data. An attacker can use a backdoor functionality to mislabel specific jobs without affecting the overall accuracy of the global model. One possible option to address such a critical issue is to encrypt the global gradients shared in the distributed network. Differential privacy at the client level can help protect against backdoor attacks, but it comes at the cost of significantly reducing the performance of the global model </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS4.p1.1.24.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib197" title="" class="ltx_ref">197</a><span id="S6.SS4.p1.1.25.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS4.p1.1.26" class="ltx_text" style="color:#000000;">. Other existing technical approaches also prevent a passive attacker from violating the privacy of data and information leakage by exploiting the global model outputs using malicious model updates. For example, the work in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS4.p1.1.27.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib198" title="" class="ltx_ref">198</a><span id="S6.SS4.p1.1.28.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS4.p1.1.29" class="ltx_text" style="color:#000000;"> presents a novel cryptographic key generation and sharing approach that leverages additive homomorphic encryption to maximize the confidentiality of federated gradient descent in the training of deep neural networks without any loss of accuracy. The work in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S6.SS4.p1.1.30.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib199" title="" class="ltx_ref">199</a><span id="S6.SS4.p1.1.31.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S6.SS4.p1.1.32" class="ltx_text" style="color:#000000;"> identifies and examines some of the security vulnerabilities and threats in FL systems and provides insights into the existing defence techniques and future approaches for improving the security and privacy of FL implementations.</span></p>
</div>
<div id="S6.SS4.p2" class="ltx_para">
<p id="S6.SS4.p2.1" class="ltx_p"><span id="S6.SS4.p2.1.1" class="ltx_text" style="color:#000000;">In summary, deploying FL in healthcare presents multifaceted challenges that demand innovative solutions addressing communication, statistical, computation, security, and privacy concerns. Addressing these challenges is essential to harness the full potential of FL while safeguarding the integrity and privacy of healthcare data.</span></p>
</div>
</section>
</section>
<section id="S7" class="ltx_section">
<h2 class="ltx_title ltx_title_section" style="color:#000000;">
<span class="ltx_tag ltx_tag_section">VII </span><span id="S7.1.1" class="ltx_text ltx_font_smallcaps">Open Problems and Future Research Directions of FL in Medical Context</span>
</h2>

<div id="S7.p1" class="ltx_para">
<p id="S7.p1.1" class="ltx_p"><span id="S7.p1.1.1" class="ltx_text" style="color:#000000;">In the preceding sections, we discussed the evolving landscape of </span><abbr title="Federated Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FL</span></abbr><span id="S7.p1.1.2" class="ltx_text" style="color:#000000;"> within the healthcare domain and its profound potential. However, FL in healthcare is still in its nascent stage, and numerous research endeavors are required to establish a practical framework for its application in medical settings. This section highlights the research gap and outlines promising avenues for future exploration in line with the earlier challenges.</span></p>
</div>
<section id="S7.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS1.4.1.1" class="ltx_text">VII-A</span> </span><span id="S7.SS1.5.2" class="ltx_text ltx_font_italic">Efficient Hyperparameter Optimization</span>
</h3>

<div id="S7.SS1.p1" class="ltx_para">
<p id="S7.SS1.p1.1" class="ltx_p"><span id="S7.SS1.p1.1.1" class="ltx_text" style="color:#000000;">One of the open problems in the FL research is hyperparameter optimization. Even before starting an FL process, an AI model with properly optimized hyperparameters need to be designed that can be communicated from the FL server to multiple clients to train on their private medical data. This hyperparameter optimization is difficult to achieve on a federated medical dataset since the clients do not want to participate and contribute before benefiting from the FL algorithm through their involvement in the system. One of the possible ways to achieve this is to design an appropriate auction mechanism with incentives that would motivate the clients to contribute toward hyperparameter optimization </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS1.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib200" title="" class="ltx_ref">200</a><span id="S7.SS1.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS1.p1.1.4" class="ltx_text" style="color:#000000;">.</span></p>
</div>
</section>
<section id="S7.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS2.4.1.1" class="ltx_text">VII-B</span> </span><span id="S7.SS2.5.2" class="ltx_text ltx_font_italic">Security and Privacy</span>
</h3>

<div id="S7.SS2.p1" class="ltx_para">
<p id="S7.SS2.p1.1" class="ltx_p"><span id="S7.SS2.p1.1.1" class="ltx_text" style="color:#000000;">Security remains a paramount concern in FL systems, especially in medical contexts where privacy is a stringiest requirement. FL’s collaborative nature, with participation from multiple clients, elevates susceptibility to security attacks, including model poisoning </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS2.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib201" title="" class="ltx_ref">201</a><span id="S7.SS2.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS2.p1.1.4" class="ltx_text" style="color:#000000;">. A malicious client can misclassify their input and influence the global model on the server. This situation is highly undesirable in a medical environment. Although some of the work in </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS2.p1.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib202" title="" class="ltx_ref">202</a><span id="S7.SS2.p1.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS2.p1.1.7" class="ltx_text" style="color:#000000;"> have introduced methods to distinguish between benign and malicious models. They have their own limitations, as such methods can only be applied in controlled settings under specific environments. Therefore, this is an open and challenging research issue that needs to be solved before deploying FL in a practical medical environment.</span></p>
</div>
</section>
<section id="S7.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS3.4.1.1" class="ltx_text">VII-C</span> </span><span id="S7.SS3.5.2" class="ltx_text ltx_font_italic">Efficient Communication Paradigm</span>
</h3>

<div id="S7.SS3.p1" class="ltx_para">
<p id="S7.SS3.p1.1" class="ltx_p"><span id="S7.SS3.p1.1.1" class="ltx_text" style="color:#000000;">Communication efficiency is pivotal in FL systems, especially when numerous clients, such as smartwatches and health gadgets, are involved. To address the communication bottleneck, research must focus on designing lightweight AI models that maintain accuracy while minimizing the data transferred during uplink and downlink communications. Strategies such as sparsification, subsampling, and quantization offer avenues for reducing message sizes and optimizing bandwidth utilization </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS3.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib203" title="" class="ltx_ref">203</a><span id="S7.SS3.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS3.p1.1.4" class="ltx_text" style="color:#000000;">. Additionally, exploring methods to minimize uplink communication rounds and enhance convergence speed while conserving client energy represents a promising research direction.</span></p>
</div>
</section>
<section id="S7.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS4.4.1.1" class="ltx_text">VII-D</span> </span><span id="S7.SS4.5.2" class="ltx_text ltx_font_italic">Solving Medical Data Heterogeneity and Statistical Issues</span>
</h3>

<div id="S7.SS4.p1" class="ltx_para">
<p id="S7.SS4.p1.1" class="ltx_p"><span id="S7.SS4.p1.1.1" class="ltx_text" style="color:#000000;">Because of FL’s distributed nature, data distributions across medical institutions are frequently heterogeneous. Several research studies have pointed out that the FL performance degrades with increasing degrees of data heterogeneity. Thus, another open problem to solve in FL is its medical data heterogeneity. FL can combine medical data, but combining horizontal medical data from medical institutions across regions and longitudinal medical data from the same patient across hospitals remains one of the most difficult challenges in FL. Although a few works such as </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS4.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib204" title="" class="ltx_ref">204</a>, <a href="#bib.bib135" title="" class="ltx_ref">135</a><span id="S7.SS4.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS4.p1.1.4" class="ltx_text" style="color:#000000;"> have investigated the medical data heterogeneity problem across multiple institutions, further research is essential to generalize results and address the nuances of medical data heterogeneity comprehensively.</span></p>
</div>
</section>
<section id="S7.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS5.4.1.1" class="ltx_text">VII-E</span> </span><span id="S7.SS5.5.2" class="ltx_text ltx_font_italic">Designing Intelligent Incentive Algorithms</span>
</h3>

<div id="S7.SS5.p1" class="ltx_para">
<p id="S7.SS5.p1.1" class="ltx_p"><span id="S7.SS5.p1.1.1" class="ltx_text" style="color:#000000;">Motivating federated medical clients to actively participate in FL processes is vital for fostering the development of robust models. Crafting incentive mechanisms encouraging honest participation and rewarding high-quality data contributions is a pressing research area </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS5.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib205" title="" class="ltx_ref">205</a><span id="S7.SS5.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS5.p1.1.4" class="ltx_text" style="color:#000000;">. Simultaneously, developing intelligent FL algorithms that can discern and exclude clients with poor-quality data can enhance model generalizability </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS5.p1.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib206" title="" class="ltx_ref">206</a><span id="S7.SS5.p1.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS5.p1.1.7" class="ltx_text" style="color:#000000;">.</span></p>
</div>
</section>
<section id="S7.SS6" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS6.4.1.1" class="ltx_text">VII-F</span> </span><span id="S7.SS6.5.2" class="ltx_text ltx_font_italic">Integrating FL into the Foundational Models</span>
</h3>

<div id="S7.SS6.p1" class="ltx_para">
<p id="S7.SS6.p1.1" class="ltx_p"><span title="Foundational Models" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_long">Foundational Models</span></span><span id="S7.SS6.p1.1.1" class="ltx_text" style="color:#000000;"> (</span><abbr title="Foundational Models" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FMs</span></abbr><span id="S7.SS6.p1.1.2" class="ltx_text" style="color:#000000;">) are a large deep-learning network that is pre-trained on a massive amount of data and designed to be adopted for a wide variety of downstream tasks. FL can potentially empower the </span><abbr title="Foundational Models" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">FMs</span></abbr><span id="S7.SS6.p1.1.3" class="ltx_text" style="color:#000000;">, encompassing </span><span id="S7.SS6.p1.1.4" class="ltx_text ltx_font_italic" style="color:#000000;">pre-training, fine-tuning and downstream applications</span><span id="S7.SS6.p1.1.5" class="ltx_text" style="color:#000000;">. Through FL, it is possible to collect a large dataset from different medical centers across the border during the </span><span id="S7.SS6.p1.1.6" class="ltx_text ltx_font_italic" style="color:#000000;">pre-training</span><span id="S7.SS6.p1.1.7" class="ltx_text" style="color:#000000;">. Collaborative learning can help develop robust and generalized FM models for various applications and domains. Additionally, FL can play an important role in </span><span id="S7.SS6.p1.1.8" class="ltx_text ltx_font_italic" style="color:#000000;">fine-tuning</span><span id="S7.SS6.p1.1.9" class="ltx_text" style="color:#000000;"> of FMs by incorporating incremental data, which helps to remain up-to-date and adaptable. One of the examples of such models is GPT-JT (with six billion parameters), which has outperformed many FMs with significantly larger parameter counts</span><span id="footnote5" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">5</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">5</sup><span class="ltx_tag ltx_tag_note">5</span><span class="ltx_ref ltx_nolink ltx_url ltx_font_typewriter ltx_ref_self">https://together.ai/blog/releasing-v1-of-gpt-jt-powered-by-open-source-ai</span></span></span></span><span id="S7.SS6.p1.1.10" class="ltx_text" style="color:#000000;">. This shows that further </span><span id="S7.SS6.p1.1.11" class="ltx_text ltx_font_italic" style="color:#000000;">fine-tuning</span><span id="S7.SS6.p1.1.12" class="ltx_text" style="color:#000000;"> can improve the performance of FMs. Moreover, FL enables the distributed utilization of FMs from </span><span id="S7.SS6.p1.1.13" class="ltx_text ltx_font_italic" style="color:#000000;">downstream tasks</span><span id="S7.SS6.p1.1.14" class="ltx_text" style="color:#000000;">. The distributed approach empowers diverse participants to collaborate, share insights, and drive continuous improvement in FM applications. Some examples of </span><span id="S7.SS6.p1.1.15" class="ltx_text ltx_font_italic" style="color:#000000;">downstream tasks</span><span id="S7.SS6.p1.1.16" class="ltx_text" style="color:#000000;"> include sentiment analysis, question answering, translation, image classification, object detection, speech recognition, etc. The downstream tasks benefit in terms of privacy preservation, better generalization, adaptability, and resource utilization.</span></p>
</div>
</section>
<section id="S7.SS7" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS7.4.1.1" class="ltx_text">VII-G</span> </span><span id="S7.SS7.5.2" class="ltx_text ltx_font_italic">Generative Pre-trained Large Language Model (FL-GPT)</span>
</h3>

<div id="S7.SS7.p1" class="ltx_para">
<p id="S7.SS7.p1.1" class="ltx_p"><span id="S7.SS7.p1.1.1" class="ltx_text" style="color:#000000;">The pre-trained large language models (LLM) mostly built upon the transformer architecture (e.g., BERT, GPT, Amazon Titan, AI21 Jurassic, Cohere), have garnered huge interest in healthcare applications, especially in administrative tasks (</span><span id="S7.SS7.p1.1.2" class="ltx_text ltx_font_italic" style="color:#000000;">such as generating letters, discharge summaries in clinics and aiding in disease diagnosis. The other application includes helping medical professionals to use LLM’s model (for example, ChatGPT) for training, education and clinical research</span><span id="S7.SS7.p1.1.3" class="ltx_text" style="color:#000000;">. However, the need for substantial healthcare data to effectively train these models raises privacy and security concerns. Addressing this, FL can be used as a crucial approach, offering privacy-preserving model training across decentralized healthcare institutions. FL not only safeguards sensitive patient information but also enables the development of specialized, context-aware models to suit the diverse healthcare landscape. The research direction calls for optimizing FL methodologies to ensure efficient, secure, and collaborative training of a specialized FL-enabled generative pre-trained large language model (FL-GPT) for healthcare, addressing communication efficiency, model convergence, and privacy preservation while enhancing collaboration and data integration among healthcare organizations. This trajectory holds great promise to redefine healthcare, ensuring patient data privacy while enhancing care and diagnostics.</span></p>
</div>
</section>
<section id="S7.SS8" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS8.4.1.1" class="ltx_text">VII-H</span> </span><span id="S7.SS8.5.2" class="ltx_text ltx_font_italic">Image Generative AI models</span>
</h3>

<div id="S7.SS8.p1" class="ltx_para">
<p id="S7.SS8.p1.1" class="ltx_p"><span id="S7.SS8.p1.1.1" class="ltx_text" style="color:#000000;">Image generative AI models (for example, Stable Diffusion, DALL-E2, CLIP, and Imagen) help to generate realistic medical images for radiological applications. One example is that by using a stable diffusion model, it is possible to generate liver tumours with different sizes, shapes, orientations, textures, intensities, and locations. Synthetically generating hard-to-find tumours, such as small liver tumours with less than 10 mm in size or rare tumours, helps to include such datasets in the training process, which can further boost the network’s performance, leading to better robustness and generalizability. Further optimization under the FL can improve the performance of different radiological imaging applications, such as diagnostic and prognosis assistance, helping in early disease diagnosis and treatment decisions.</span></p>
</div>
</section>
<section id="S7.SS9" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS9.4.1.1" class="ltx_text">VII-I</span> </span><span id="S7.SS9.5.2" class="ltx_text ltx_font_italic">Contrastive Learning for Unlabelled Medical Data </span>
</h3>

<div id="S7.SS9.p1" class="ltx_para">
<p id="S7.SS9.p1.1" class="ltx_p"><span id="S7.SS9.p1.1.1" class="ltx_text" style="color:#000000;">Most of the research works in FL are concentrated on the supervised learning models where the datasets are labeled </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS9.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib33" title="" class="ltx_ref">33</a>, <a href="#bib.bib207" title="" class="ltx_ref">207</a><span id="S7.SS9.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS9.p1.1.4" class="ltx_text" style="color:#000000;">. However, unlabelled datasets could grow in numbers when small health organizations and big hospitals participate in the FL for training a global model. To overcome this unlabelled datasets problem, semi-supervised and unsupervised learning have recently been used in FL by some researchers </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS9.p1.1.5.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib208" title="" class="ltx_ref">208</a>, <a href="#bib.bib209" title="" class="ltx_ref">209</a>, <a href="#bib.bib210" title="" class="ltx_ref">210</a>, <a href="#bib.bib211" title="" class="ltx_ref">211</a><span id="S7.SS9.p1.1.6.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS9.p1.1.7" class="ltx_text" style="color:#000000;">. Although semi-supervised and unsupervised learning can solve the problem related to the unlabelled datasets, they often fail to achieve high performance in medical image datasets with </span><abbr title="Deep Learning" class="ltx_glossaryref"><span class="ltx_text ltx_glossary_short">DL</span></abbr><span id="S7.SS9.p1.1.8" class="ltx_text" style="color:#000000;"> models. In this regard, contrastive learning has recently become a popular research field for learning unlabeled data representations by training an FL model on unlabeled data </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS9.p1.1.9.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib212" title="" class="ltx_ref">212</a><span id="S7.SS9.p1.1.10.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS9.p1.1.11" class="ltx_text" style="color:#000000;">. Contrastive learning, also known as self-learning, is a pre-training procedure in which the model attempts to learn similar and distinct data samples from an unlabeled data distribution </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS9.p1.1.12.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib213" title="" class="ltx_ref">213</a>, <a href="#bib.bib214" title="" class="ltx_ref">214</a><span id="S7.SS9.p1.1.13.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS9.p1.1.14" class="ltx_text" style="color:#000000;">. Thus, contrastive learning seems very relevant for applying FL in the medical domain as it can be used on unlabelled and non-IID medical data. This is an interesting research direction, and more research is needed to be explored in FL settings.</span></p>
</div>
</section>
<section id="S7.SS10" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS10.4.1.1" class="ltx_text">VII-J</span> </span><span id="S7.SS10.5.2" class="ltx_text ltx_font_italic">Benchmarking FL</span>
</h3>

<div id="S7.SS10.p1" class="ltx_para">
<p id="S7.SS10.p1.1" class="ltx_p"><span id="S7.SS10.p1.1.1" class="ltx_text" style="color:#000000;">As FL gains prominence in the medical domain, it becomes imperative to establish robust benchmarking tools and frameworks for empirical evaluation and comparison. Expanding existing implementations and fostering the availability of open FL-based medical datasets for research communities are essential steps to promote reproducibility, generalizability, and innovation in FL solutions. Also, open FL-based medical datasets should be made available for the research community for benchmarking FL algorithms suitable for medical applications.</span></p>
</div>
</section>
<section id="S7.SS11" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection" style="color:#000000;">
<span class="ltx_tag ltx_tag_subsection"><span id="S7.SS11.4.1.1" class="ltx_text">VII-K</span> </span><span id="S7.SS11.5.2" class="ltx_text ltx_font_italic">Regulatory Frameworks for Medical Applications</span>
</h3>

<div id="S7.SS11.p1" class="ltx_para">
<p id="S7.SS11.p1.1" class="ltx_p"><span id="S7.SS11.p1.1.1" class="ltx_text" style="color:#000000;">Considering the strict regulations such as GDPR on the security and privacy of medical data, a guideline needs to be developed that allows researchers to evaluate actual risks and concerns related to the medical data when using it in the FL framework. So far, we do not have enough knowledge and the possibility to check if privacy regulations of medical data are being violated when using it for FL applications. This calls for joint cooperation between computer science and law community researchers to address the security and privacy regulations of medical data in the FL framework. Moreover, enhancing the explainability of FL models in medical contexts is an open challenge that warrants exploration </span><cite class="ltx_cite ltx_citemacro_cite"><span id="S7.SS11.p1.1.2.1" class="ltx_text" style="color:#000000;">[</span><a href="#bib.bib215" title="" class="ltx_ref">215</a>, <a href="#bib.bib216" title="" class="ltx_ref">216</a>, <a href="#bib.bib217" title="" class="ltx_ref">217</a><span id="S7.SS11.p1.1.3.2" class="ltx_text" style="color:#000000;">]</span></cite><span id="S7.SS11.p1.1.4" class="ltx_text" style="color:#000000;">.</span></p>
</div>
</section>
</section>
<section id="S8" class="ltx_section">
<h2 class="ltx_title ltx_title_section" style="color:#000000;">
<span class="ltx_tag ltx_tag_section">VIII </span><span id="S8.1.1" class="ltx_text ltx_font_smallcaps">Conclusions</span>
</h2>

<div id="S8.p1" class="ltx_para">
<p id="S8.p1.1" class="ltx_p"><span id="S8.p1.1.1" class="ltx_text" style="color:#000000;">In this paper, we explored the pivotal role of FL in privacy-preserving medical applications where direct access to sensitive medical data is limited. FL, being a decentralized and robust framework, is well-suited for healthcare. Our discussion has highlighted how FL and emerging technologies address the unique challenges in the medical field, especially in the context of addressing global cancer burdens. Our research demonstrates how FL can enable the creation of computer-aided diagnosis tools that are more effective than traditional data-driven medical applications. However, several challenges remain, such as system and statistical differences, communication limitations, and security and privacy concerns. We have given a comprehensive overview of FL and its applications, using cutting-edge technologies to overcome the intricate challenges in the medical domain. Throughout this article, we have presented the latest research findings showcasing the evolving use of FL in medical contexts. These findings emphasize the dynamic nature of FL research in healthcare. To inspire future research, we have explored specific challenges in FL in medical scenarios, including differences between healthcare institutions, communication issues in FL networks, and the need for strong security and privacy measures. These challenges define the current state of FL in healthcare and guide future investigations. Addressing these challenges and advancing FL in medical applications requires collaboration across diverse interdisciplinary research communities, including health, computer science, and law. Combining expertise from these fields is crucial for designing reliable and scalable FL models that can drive healthcare innovations based on data.</span>
<br class="ltx_break"><span id="S8.p1.1.2" class="ltx_text" style="color:#000000;">In future, we anticipate the practical deployment of FL for medical applications, leading to the development of more secured protocols. Additionally, we anticipate the emergence of energy-efficient communication paradigms tailored to the unique requirements of FL networks. The translation of FL technology into real-world healthcare solutions is poised to revolutionize the healthcare landscape by addressing complex challenges.</span>
<br class="ltx_break"><span id="S8.p1.1.3" class="ltx_text" style="color:#000000;">In conclusion, this survey paper serves as a crucial reference, summarizing the current state of FL in medical applications. It also acts as a guide for researchers, pointing out open problems and future research directions in this field. We strongly believe that FL has the potential to greatly benefit healthcare by improving the efficiency of healthcare systems worldwide.</span></p>
</div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography" style="color:#000000;">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock"><span id="bib.bib1.1.1" class="ltx_text" style="color:#000000;">
Y. A. Qadri, A. Nauman, Y. B. Zikria, A. V. Vasilakos, and S. W. Kim, “The future of healthcare internet of things: a survey of emerging technologies,” </span><em id="bib.bib1.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Communications Surveys &amp; Tutorials</em><span id="bib.bib1.3.3" class="ltx_text" style="color:#000000;">, vol. 22, no. 2, pp. 1121–1167, 2020.
</span>
</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock"><span id="bib.bib2.1.1" class="ltx_text" style="color:#000000;">
D. V. Dimitrov, “Medical internet of things and big data in healthcare,” </span><em id="bib.bib2.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Healthcare informatics research</em><span id="bib.bib2.3.3" class="ltx_text" style="color:#000000;">, vol. 22, no. 3, pp. 156–163, 2016.
</span>
</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock"><span id="bib.bib3.1.1" class="ltx_text" style="color:#000000;">
S. U. Amin and M. S. Hossain, “Edge intelligence and internet of things in healthcare: A survey,” </span><em id="bib.bib3.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Access</em><span id="bib.bib3.3.3" class="ltx_text" style="color:#000000;">, vol. 9, pp. 45–59, 2021.
</span>
</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock"><span id="bib.bib4.1.1" class="ltx_text" style="color:#000000;">
V. Gulshan, L. Peng, M. Coram, M. C. Stumpe, D. Wu, A. Narayanaswamy, S. Venugopalan, K. Widner, T. Madams, J. Cuadros, </span><em id="bib.bib4.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib4.3.3" class="ltx_text" style="color:#000000;">, “Development and validation of a deep learning algorithm for detection of diabetic retinopathy in retinal fundus photographs,” </span><em id="bib.bib4.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">jama</em><span id="bib.bib4.5.5" class="ltx_text" style="color:#000000;">, vol. 316, no. 22, pp. 2402–2410, 2016.
</span>
</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock"><span id="bib.bib5.1.1" class="ltx_text" style="color:#000000;">
M. A. Sayeed, S. P. Mohanty, E. Kougianos, and H. P. Zaveri, “Neuro-detect: a machine learning-based fast and accurate seizure detection system in the iomt,” </span><em id="bib.bib5.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Consumer Electronics</em><span id="bib.bib5.3.3" class="ltx_text" style="color:#000000;">, vol. 65, no. 3, pp. 359–368, 2019.
</span>
</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock"><span id="bib.bib6.1.1" class="ltx_text" style="color:#000000;">
Z. Allam and D. S. Jones, “On the coronavirus (COVID-19) outbreak and the smart city network: universal data sharing standards coupled with artificial intelligence (AI) to benefit urban health monitoring and management,” in </span><em id="bib.bib6.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proc. of the Healthcare</em><span id="bib.bib6.3.3" class="ltx_text" style="color:#000000;">, vol. 8, no. 1, 2020, p. 46.
</span>
</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock"><span id="bib.bib7.1.1" class="ltx_text" style="color:#000000;">
Q. Cai, H. Wang, Z. Li, and X. Liu, “A survey on multimodal data-driven smart healthcare systems: approaches and applications,” </span><em id="bib.bib7.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Access</em><span id="bib.bib7.3.3" class="ltx_text" style="color:#000000;">, vol. 7, pp. 133 583–133 599, 2019.
</span>
</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock"><span id="bib.bib8.1.1" class="ltx_text" style="color:#000000;">
F. Cascini, F. Santaroni, R. Lanzetti, G. Failla, A. Gentili, and W. Ricciardi, “Developing a data-driven approach in order to improve the safety and quality of patient care,” </span><em id="bib.bib8.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Frontiers in public health</em><span id="bib.bib8.3.3" class="ltx_text" style="color:#000000;">, p. 592, 2021.
</span>
</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock"><span id="bib.bib9.1.1" class="ltx_text" style="color:#000000;">
J. Andrew and M. Baker, “The general data protection regulation in the age of surveillance capitalism,” </span><em id="bib.bib9.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of Business Ethics</em><span id="bib.bib9.3.3" class="ltx_text" style="color:#000000;">, vol. 168, no. 3, pp. 565–578, 2021.
</span>
</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock"><span id="bib.bib10.1.1" class="ltx_text" style="color:#000000;">
W. Hartzog and N. Richards, “Privacy’s constitutional moment and the limits of data protection,” </span><em id="bib.bib10.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">BCL Rev.</em><span id="bib.bib10.3.3" class="ltx_text" style="color:#000000;">, vol. 61, p. 1687, 2020.
</span>
</span>
</li>
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock"><span id="bib.bib11.1.1" class="ltx_text" style="color:#000000;">
B. Yuan and J. Li, “The policy effect of the general data protection regulation (GDPR) on the digital public health sector in the european union: an empirical investigation,” </span><em id="bib.bib11.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">International journal of environmental research and public health</em><span id="bib.bib11.3.3" class="ltx_text" style="color:#000000;">, vol. 16, no. 6, p. 1070, 2019.
</span>
</span>
</li>
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock"><span id="bib.bib12.1.1" class="ltx_text" style="color:#000000;">
C. J. Hoofnagle, B. van der Sloot, and F. Z. Borgesius, “The european union general data protection regulation: what it is and what it means,” </span><em id="bib.bib12.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Information &amp; Communications Technology Law</em><span id="bib.bib12.3.3" class="ltx_text" style="color:#000000;">, vol. 28, no. 1, pp. 65–98, 2019.
</span>
</span>
</li>
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock"><span id="bib.bib13.1.1" class="ltx_text" style="color:#000000;">
M. M. John, H. H. Olsson, and J. Bosch, “Developing ML/DL models: a design framework,” in </span><em id="bib.bib13.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Software and System Processes</em><span id="bib.bib13.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 1–10.
</span>
</span>
</li>
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock"><span id="bib.bib14.1.1" class="ltx_text" style="color:#000000;">
D. Xu, T. Li, Y. Li, X. Su, S. Tarkoma, T. Jiang, J. Crowcroft, and P. Hui, “Edge intelligence: Architectures, challenges, and applications,” </span><em id="bib.bib14.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2003.12172</em><span id="bib.bib14.3.3" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock"><span id="bib.bib15.1.1" class="ltx_text" style="color:#000000;">
W. Yu, F. Liang, X. He, W. G. Hatcher, C. Lu, J. Lin, and X. Yang, “A survey on the edge computing for the Internet of Things,” </span><em id="bib.bib15.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Access</em><span id="bib.bib15.3.3" class="ltx_text" style="color:#000000;">, vol. 6, pp. 6900–6919, 2017.
</span>
</span>
</li>
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock"><span id="bib.bib16.1.1" class="ltx_text" style="color:#000000;">
X. Chen, Q. Shi, L. Yang, and J. Xu, “ThriftyEdge: Resource-efficient edge computing for intelligent IoT applications,” </span><em id="bib.bib16.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE network</em><span id="bib.bib16.3.3" class="ltx_text" style="color:#000000;">, vol. 32, no. 1, pp. 61–65, 2018.
</span>
</span>
</li>
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock"><span id="bib.bib17.1.1" class="ltx_text" style="color:#000000;">
W. Rafique, L. Qi, I. Yaqoob, M. Imran, R. U. Rasool, and W. Dou, “Complementing IoT services through software defined networking and edge computing: A comprehensive survey,” </span><em id="bib.bib17.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Communications Surveys &amp; Tutorials</em><span id="bib.bib17.3.3" class="ltx_text" style="color:#000000;">, vol. 22, no. 3, pp. 1761–1804, 2020.
</span>
</span>
</li>
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock"><span id="bib.bib18.1.1" class="ltx_text" style="color:#000000;">
K. Sha, T. A. Yang, W. Wei, and S. Davari, “A survey of edge computing-based designs for IoT security,” </span><em id="bib.bib18.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Digital Communications and Networks</em><span id="bib.bib18.3.3" class="ltx_text" style="color:#000000;">, vol. 6, no. 2, pp. 195–202, 2020.
</span>
</span>
</li>
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock"><span id="bib.bib19.1.1" class="ltx_text" style="color:#000000;">
P. Zhang, C. Jiang, X. Pang, and Y. Qian, “Stec-iot: A security tactic by virtualizing edge computing on iot,” </span><em id="bib.bib19.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Internet of Things Journal</em><span id="bib.bib19.3.3" class="ltx_text" style="color:#000000;">, vol. 8, no. 4, pp. 2459–2467, 2020.
</span>
</span>
</li>
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock"><span id="bib.bib20.1.1" class="ltx_text" style="color:#000000;">
W. Shi, J. Cao, Q. Zhang, Y. Li, and L. Xu, “Edge computing: Vision and challenges,” </span><em id="bib.bib20.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Internet of Things journal</em><span id="bib.bib20.3.3" class="ltx_text" style="color:#000000;">, vol. 3, no. 5, pp. 637–646, 2016.
</span>
</span>
</li>
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock"><span id="bib.bib21.1.1" class="ltx_text" style="color:#000000;">
E. Li, L. Zeng, Z. Zhou, and X. Chen, “Edge ai: On-demand accelerating deep neural network inference via edge computing,” </span><em id="bib.bib21.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Wireless Communications</em><span id="bib.bib21.3.3" class="ltx_text" style="color:#000000;">, pp. 447–457, 2019.
</span>
</span>
</li>
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[22]</span>
<span class="ltx_bibblock"><span id="bib.bib22.1.1" class="ltx_text" style="color:#000000;">
T. Rausch, W. Hummer, V. Muthusamy, A. Rashed, and S. Dustdar, “Towards a serverless platform for edge AI,” in </span><em id="bib.bib22.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the USENIX Workshop on Hot Topics in Edge Computing</em><span id="bib.bib22.3.3" class="ltx_text" style="color:#000000;">, 2019.
</span>
</span>
</li>
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[23]</span>
<span class="ltx_bibblock"><span id="bib.bib23.1.1" class="ltx_text" style="color:#000000;">
Y. Shi, K. Yang, T. Jiang, J. Zhang, and K. B. Letaief, “Communication-efficient edge ai: Algorithms and systems,” </span><em id="bib.bib23.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Communications Surveys &amp; Tutorials</em><span id="bib.bib23.3.3" class="ltx_text" style="color:#000000;">, pp. 2167–2191, 2020.
</span>
</span>
</li>
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[24]</span>
<span class="ltx_bibblock"><span id="bib.bib24.1.1" class="ltx_text" style="color:#000000;">
P. P. Ray, D. Dash, and D. De, “Edge computing for internet of things: A survey, e-healthcare case study and future direction,” </span><em id="bib.bib24.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of Network and Computer Applications</em><span id="bib.bib24.3.3" class="ltx_text" style="color:#000000;">, vol. 140, pp. 1–22, 2019.
</span>
</span>
</li>
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[25]</span>
<span class="ltx_bibblock"><span id="bib.bib25.1.1" class="ltx_text" style="color:#000000;">
N. Rieke, J. Hancox, W. Li, F. Milletari, H. R. Roth, S. Albarqouni, S. Bakas, M. N. Galtier, B. A. Landman, K. Maier-Hein, </span><em id="bib.bib25.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib25.3.3" class="ltx_text" style="color:#000000;">, “The future of digital health with federated learning,” </span><em id="bib.bib25.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">NPJ digital medicine</em><span id="bib.bib25.5.5" class="ltx_text" style="color:#000000;">, vol. 3, no. 1, pp. 1–7, 2020.
</span>
</span>
</li>
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[26]</span>
<span class="ltx_bibblock"><span id="bib.bib26.1.1" class="ltx_text" style="color:#000000;">
J. Xu, B. S. Glicksberg, C. Su, P. Walker, J. Bian, and F. Wang, “Federated learning for healthcare informatics,” </span><em id="bib.bib26.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of Healthcare Informatics Research</em><span id="bib.bib26.3.3" class="ltx_text" style="color:#000000;">, vol. 5, no. 1, pp. 1–19, 2021.
</span>
</span>
</li>
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[27]</span>
<span class="ltx_bibblock"><span id="bib.bib27.1.1" class="ltx_text" style="color:#000000;">
J. Luo, M. Wu, D. Gopukumar, and Y. Zhao, “Big data application in biomedical research and health care: a literature review,” </span><em id="bib.bib27.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Biomedical informatics insights</em><span id="bib.bib27.3.3" class="ltx_text" style="color:#000000;">, vol. 8, pp. BII–S31 559, 2016.
</span>
</span>
</li>
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[28]</span>
<span class="ltx_bibblock"><span id="bib.bib28.1.1" class="ltx_text" style="color:#000000;">
Z. Lv, H. Song, P. Basanta-Val, A. Steed, and M. Jo, “Next-generation big data analytics: State of the art, challenges, and future research topics,” </span><em id="bib.bib28.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Industrial Informatics</em><span id="bib.bib28.3.3" class="ltx_text" style="color:#000000;">, vol. 13, no. 4, pp. 1891–1899, 2017.
</span>
</span>
</li>
<li id="bib.bib29" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[29]</span>
<span class="ltx_bibblock"><span id="bib.bib29.1.1" class="ltx_text" style="color:#000000;">
L. Maier-Hein, S. S. Vedula, S. Speidel, N. Navab, R. Kikinis, A. Park, M. Eisenmann, H. Feussner, G. Forestier, S. Giannarou, </span><em id="bib.bib29.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib29.3.3" class="ltx_text" style="color:#000000;">, “Surgical data science for next-generation interventions,” </span><em id="bib.bib29.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Nature Biomedical Engineering</em><span id="bib.bib29.5.5" class="ltx_text" style="color:#000000;">, vol. 1, no. 9, pp. 691–696, 2017.
</span>
</span>
</li>
<li id="bib.bib30" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[30]</span>
<span class="ltx_bibblock"><span id="bib.bib30.1.1" class="ltx_text" style="color:#000000;">
J. J. Rasouli, J. Shao, S. Neifert, W. N. Gibbs, G. Habboub, M. P. Steinmetz, E. Benzel, and T. E. Mroz, “Artificial intelligence and robotics in spine surgery,” </span><em id="bib.bib30.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Global Spine Journal</em><span id="bib.bib30.3.3" class="ltx_text" style="color:#000000;">, vol. 11, no. 4, pp. 556–564, 2021.
</span>
</span>
</li>
<li id="bib.bib31" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[31]</span>
<span class="ltx_bibblock"><span id="bib.bib31.1.1" class="ltx_text" style="color:#000000;">
S. AbdulRahman, H. Tout, H. Ould-Slimane, A. Mourad, C. Talhi, and M. Guizani, “A survey on federated learning: The journey from centralized to distributed on-site learning and beyond,” </span><em id="bib.bib31.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Internet of Things Journal</em><span id="bib.bib31.3.3" class="ltx_text" style="color:#000000;">, vol. 8, no. 7, pp. 5476–5497, 2020.
</span>
</span>
</li>
<li id="bib.bib32" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[32]</span>
<span class="ltx_bibblock"><span id="bib.bib32.1.1" class="ltx_text" style="color:#000000;">
M. Aledhari, R. Razzak, R. M. Parizi, and F. Saeed, “Federated learning: A survey on enabling technologies, protocols, and applications,” </span><em id="bib.bib32.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Access</em><span id="bib.bib32.3.3" class="ltx_text" style="color:#000000;">, vol. 8, pp. 140 699–140 725, 2020.
</span>
</span>
</li>
<li id="bib.bib33" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[33]</span>
<span class="ltx_bibblock"><span id="bib.bib33.1.1" class="ltx_text" style="color:#000000;">
S. Ji, T. Saravirta, S. Pan, G. Long, and A. Walid, “Emerging trends in federated learning: From model fusion to federated x learning,” </span><em id="bib.bib33.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2102.12920</em><span id="bib.bib33.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib34" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[34]</span>
<span class="ltx_bibblock"><span id="bib.bib34.1.1" class="ltx_text" style="color:#000000;">
L. Li, Y. Fan, M. Tse, and K.-Y. Lin, “A review of applications in federated learning,” </span><em id="bib.bib34.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Computers &amp; Industrial Engineering</em><span id="bib.bib34.3.3" class="ltx_text" style="color:#000000;">, vol. 149, p. 106854, 2020.
</span>
</span>
</li>
<li id="bib.bib35" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[35]</span>
<span class="ltx_bibblock"><span id="bib.bib35.1.1" class="ltx_text" style="color:#000000;">
Q. Li, Z. Wen, Z. Wu, S. Hu, N. Wang, Y. Li, X. Liu, and B. He, “A survey on federated learning systems: vision, hype and reality for data privacy and protection,” </span><em id="bib.bib35.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Knowledge and Data Engineering</em><span id="bib.bib35.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib36" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[36]</span>
<span class="ltx_bibblock"><span id="bib.bib36.1.1" class="ltx_text" style="color:#000000;">
W. Y. B. Lim, N. C. Luong, D. T. Hoang, Y. Jiao, Y.-C. Liang, Q. Yang, D. Niyato, and C. Miao, “Federated learning in mobile edge networks: A comprehensive survey,” </span><em id="bib.bib36.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Communications Surveys &amp; Tutorials</em><span id="bib.bib36.3.3" class="ltx_text" style="color:#000000;">, vol. 22, no. 3, pp. 2031–2063, 2020.
</span>
</span>
</li>
<li id="bib.bib37" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[37]</span>
<span class="ltx_bibblock"><span id="bib.bib37.1.1" class="ltx_text" style="color:#000000;">
Y. Liu, L. Zhang, N. Ge, and G. Li, “A systematic literature review on federated learning: From a model quality perspective,” 2020.
</span>
</span>
</li>
<li id="bib.bib38" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[38]</span>
<span class="ltx_bibblock"><span id="bib.bib38.1.1" class="ltx_text" style="color:#000000;">
Z. Zheng, Y. Zhou, Y. Sun, Z. Wang, B. Liu, and K. Li, “Applications of federated learning in smart cities: recent advances, taxonomy, and open challenges,” </span><em id="bib.bib38.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Connection Science</em><span id="bib.bib38.3.3" class="ltx_text" style="color:#000000;">, vol. 34, no. 1, pp. 1–28, 2022.
</span>
</span>
</li>
<li id="bib.bib39" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[39]</span>
<span class="ltx_bibblock"><span id="bib.bib39.1.1" class="ltx_text" style="color:#000000;">
B. Pfitzner, N. Steckhan, and B. Arnrich, “Federated learning in a medical context: A systematic literature review,” </span><em id="bib.bib39.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">ACM Transactions on Internet Technology (TOIT)</em><span id="bib.bib39.3.3" class="ltx_text" style="color:#000000;">, vol. 21, no. 2, pp. 1–31, 2021.
</span>
</span>
</li>
<li id="bib.bib40" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[40]</span>
<span class="ltx_bibblock"><span id="bib.bib40.1.1" class="ltx_text" style="color:#000000;">
M. Ali, F. Naeem, M. Tariq, and G. Kaddoum, “Federated learning for privacy preservation in smart healthcare systems: A comprehensive survey,” </span><em id="bib.bib40.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2203.09702</em><span id="bib.bib40.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib41" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[41]</span>
<span class="ltx_bibblock"><span id="bib.bib41.1.1" class="ltx_text" style="color:#000000;">
R. S. Antunes, C. A. da Costa, A. Küderle, I. A. Yari, and B. Eskofier, “Federated learning for healthcare: Systematic review and architecture proposal,” </span><em id="bib.bib41.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">ACM Transactions on Intelligent Systems and Technology (TIST)</em><span id="bib.bib41.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib42" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[42]</span>
<span class="ltx_bibblock"><span id="bib.bib42.1.1" class="ltx_text" style="color:#000000;">
D. C. Nguyen, Q.-V. Pham, P. N. Pathirana, M. Ding, A. Seneviratne, Z. Lin, O. Dobre, and W.-J. Hwang, “Federated learning for smart healthcare: A survey,” </span><em id="bib.bib42.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">ACM Computing Surveys (CSUR)</em><span id="bib.bib42.3.3" class="ltx_text" style="color:#000000;">, vol. 55, no. 3, pp. 1–37, 2022.
</span>
</span>
</li>
<li id="bib.bib43" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[43]</span>
<span class="ltx_bibblock"><span id="bib.bib43.1.1" class="ltx_text" style="color:#000000;">
H. Yu, Z. Liu, Y. Liu, T. Chen, M. Cong, X. Weng, D. Niyato, and Q. Yang, “A sustainable incentive scheme for federated learning,” </span><em id="bib.bib43.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Intelligent Systems</em><span id="bib.bib43.3.3" class="ltx_text" style="color:#000000;">, vol. 35, no. 4, pp. 58–69, 2020.
</span>
</span>
</li>
<li id="bib.bib44" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[44]</span>
<span class="ltx_bibblock"><span id="bib.bib44.1.1" class="ltx_text" style="color:#000000;">
B. McMahan, E. Moore, D. Ramage, S. Hampson, and B. A. y Arcas, “Communication-efficient learning of deep networks from decentralized data,” in </span><em id="bib.bib44.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the Artificial intelligence and statistics</em><span id="bib.bib44.3.3" class="ltx_text" style="color:#000000;">, 2017, pp. 1273–1282.
</span>
</span>
</li>
<li id="bib.bib45" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[45]</span>
<span class="ltx_bibblock"><span id="bib.bib45.1.1" class="ltx_text" style="color:#000000;">
Y. Xia, D. Yang, W. Li, A. Myronenko, D. Xu, H. Obinata, H. Mori, P. An, S. Harmon, E. Turkbey, </span><em id="bib.bib45.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib45.3.3" class="ltx_text" style="color:#000000;">, “Auto-fedavg: Learnable federated averaging for multi-institutional medical image segmentation,” </span><em id="bib.bib45.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2104.10195</em><span id="bib.bib45.5.5" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib46" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[46]</span>
<span class="ltx_bibblock"><span id="bib.bib46.1.1" class="ltx_text" style="color:#000000;">
X. Li, K. Huang, W. Yang, S. Wang, and Z. Zhang, “On the convergence of fedavg on non-iid data,” </span><em id="bib.bib46.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:1907.02189</em><span id="bib.bib46.3.3" class="ltx_text" style="color:#000000;">, 2019.
</span>
</span>
</li>
<li id="bib.bib47" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[47]</span>
<span class="ltx_bibblock"><span id="bib.bib47.1.1" class="ltx_text" style="color:#000000;">
W. Lu, J. Wang, Y. Chen, X. Qin, R. Xu, D. Dimitriadis, and T. Qin, “Personalized federated learning with adaptive batchnorm for healthcare,” </span><em id="bib.bib47.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Big Data</em><span id="bib.bib47.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib48" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[48]</span>
<span class="ltx_bibblock"><span id="bib.bib48.1.1" class="ltx_text" style="color:#000000;">
X. Li, Y. Gu, N. Dvornek, L. H. Staib, P. Ventola, and J. S. Duncan, “Multi-site fmri analysis using privacy-preserving federated learning and domain adaptation: Abide results,” </span><em id="bib.bib48.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Medical Image Analysis</em><span id="bib.bib48.3.3" class="ltx_text" style="color:#000000;">, vol. 65, p. 101765, 2020.
</span>
</span>
</li>
<li id="bib.bib49" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[49]</span>
<span class="ltx_bibblock"><span id="bib.bib49.1.1" class="ltx_text" style="color:#000000;">
D. Romanini, A. J. Hall, P. Papadopoulos, T. Titcombe, A. Ismail, T. Cebere, R. Sandmann, R. Roehm, and M. A. Hoeh, “Pyvertical: A vertical federated learning framework for multi-headed splitnn,” </span><em id="bib.bib49.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2104.00489</em><span id="bib.bib49.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib50" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[50]</span>
<span class="ltx_bibblock"><span id="bib.bib50.1.1" class="ltx_text" style="color:#000000;">
D. Cha, M. Sung, Y.-R. Park, </span><em id="bib.bib50.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib50.3.3" class="ltx_text" style="color:#000000;">, “Implementing vertical federated learning using autoencoders: Practical application, generalizability, and utility study,” </span><em id="bib.bib50.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">JMIR medical informatics</em><span id="bib.bib50.5.5" class="ltx_text" style="color:#000000;">, vol. 9, no. 6, p. e26598, 2021.
</span>
</span>
</li>
<li id="bib.bib51" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[51]</span>
<span class="ltx_bibblock"><span id="bib.bib51.1.1" class="ltx_text" style="color:#000000;">
Y. Chen, X. Qin, J. Wang, C. Yu, and W. Gao, “Fedhealth: A federated transfer learning framework for wearable healthcare,” </span><em id="bib.bib51.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Intelligent Systems</em><span id="bib.bib51.3.3" class="ltx_text" style="color:#000000;">, vol. 35, no. 4, pp. 83–93, 2020.
</span>
</span>
</li>
<li id="bib.bib52" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[52]</span>
<span class="ltx_bibblock"><span id="bib.bib52.1.1" class="ltx_text" style="color:#000000;">
T. S. Brisimi, R. Chen, T. Mela, A. Olshevsky, I. C. Paschalidis, and W. Shi, “Federated learning of predictive models from federated electronic health records,” </span><em id="bib.bib52.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">International journal of medical informatics</em><span id="bib.bib52.3.3" class="ltx_text" style="color:#000000;">, vol. 112, pp. 59–67, 2018.
</span>
</span>
</li>
<li id="bib.bib53" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[53]</span>
<span class="ltx_bibblock"><span id="bib.bib53.1.1" class="ltx_text" style="color:#000000;">
L. Huang, A. L. Shea, H. Qian, A. Masurkar, H. Deng, and D. Liu, “Patient clustering improves efficiency of federated machine learning to predict mortality and hospital stay time using distributed electronic medical records,” </span><em id="bib.bib53.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of biomedical informatics</em><span id="bib.bib53.3.3" class="ltx_text" style="color:#000000;">, vol. 99, p. 103291, 2019.
</span>
</span>
</li>
<li id="bib.bib54" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[54]</span>
<span class="ltx_bibblock"><span id="bib.bib54.1.1" class="ltx_text" style="color:#000000;">
B. Y. Lin, C. He, Z. Zeng, H. Wang, Y. Huang, M. Soltanolkotabi, X. Ren, and S. Avestimehr, “FedNLP: Benchmarking Federated Learning Methods for Natural Language Processing Tasks,” </span><em id="bib.bib54.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2104.08815</em><span id="bib.bib54.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib55" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[55]</span>
<span class="ltx_bibblock"><span id="bib.bib55.1.1" class="ltx_text" style="color:#000000;">
Y. Liu, J. James, J. Kang, D. Niyato, and S. Zhang, “Privacy-preserving traffic flow prediction: A federated learning approach,” </span><em id="bib.bib55.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Internet of Things Journal</em><span id="bib.bib55.3.3" class="ltx_text" style="color:#000000;">, vol. 7, no. 8, pp. 7751–7763, 2020.
</span>
</span>
</li>
<li id="bib.bib56" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[56]</span>
<span class="ltx_bibblock"><span id="bib.bib56.1.1" class="ltx_text" style="color:#000000;">
K. Tan, D. Bremner, J. Le Kernec, and M. Imran, “Federated machine learning in vehicular networks: A summary of recent applications,” in </span><em id="bib.bib56.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the 2020 International Conference on UK-China Emerging Technologies (UCET)</em><span id="bib.bib56.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 1–4.
</span>
</span>
</li>
<li id="bib.bib57" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[57]</span>
<span class="ltx_bibblock"><span id="bib.bib57.1.1" class="ltx_text" style="color:#000000;">
Y. Cheng, Y. Liu, T. Chen, and Q. Yang, “Federated learning for privacy-preserving AI,” </span><em id="bib.bib57.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Communications of the ACM</em><span id="bib.bib57.3.3" class="ltx_text" style="color:#000000;">, vol. 63, no. 12, pp. 33–36, 2020.
</span>
</span>
</li>
<li id="bib.bib58" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[58]</span>
<span class="ltx_bibblock"><span id="bib.bib58.1.1" class="ltx_text" style="color:#000000;">
B. Shen, J. Guo, and Y. Yang, “MedChain: Efficient healthcare data sharing via blockchain,” </span><em id="bib.bib58.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Applied sciences</em><span id="bib.bib58.3.3" class="ltx_text" style="color:#000000;">, vol. 9, no. 6, p. 1207, 2019.
</span>
</span>
</li>
<li id="bib.bib59" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[59]</span>
<span class="ltx_bibblock"><span id="bib.bib59.1.1" class="ltx_text" style="color:#000000;">
M. Grama, M. Musat, L. Muñoz-González, J. Passerat-Palmbach, D. Rueckert, and A. Alansary, “Robust aggregation for adaptive privacy preserving federated learning in healthcare,” </span><em id="bib.bib59.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2009.08294</em><span id="bib.bib59.3.3" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib60" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[60]</span>
<span class="ltx_bibblock"><span id="bib.bib60.1.1" class="ltx_text" style="color:#000000;">
M. Fang, X. Cao, J. Jia, and N. Gong, “Local Model Poisoning Attacks to Byzantine-Robust Federated Learning,” in </span><em id="bib.bib60.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the 29th USENIX Security Symposium (USENIX Security 20)</em><span id="bib.bib60.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 1605–1622.
</span>
</span>
</li>
<li id="bib.bib61" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[61]</span>
<span class="ltx_bibblock"><span id="bib.bib61.1.1" class="ltx_text" style="color:#000000;">
X. Bao, C. Su, Y. Xiong, W. Huang, and Y. Hu, “Flchain: A blockchain for auditable federated learning with trust and incentive,” in </span><em id="bib.bib61.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Big Data Computing and Communications (BIGCOM)</em><span id="bib.bib61.3.3" class="ltx_text" style="color:#000000;">, 2019, pp. 151–159.
</span>
</span>
</li>
<li id="bib.bib62" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[62]</span>
<span class="ltx_bibblock"><span id="bib.bib62.1.1" class="ltx_text" style="color:#000000;">
H. B. Desai, M. S. Ozdayi, and M. Kantarcioglu, “Blockfla: Accountable federated learning via hybrid blockchain architecture,” in </span><em id="bib.bib62.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the eleventh ACM conference on data and application security and privacy</em><span id="bib.bib62.3.3" class="ltx_text" style="color:#000000;">, 2021, pp. 101–112.
</span>
</span>
</li>
<li id="bib.bib63" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[63]</span>
<span class="ltx_bibblock"><span id="bib.bib63.1.1" class="ltx_text" style="color:#000000;">
P. Ramanan and K. Nakayama, “Baffle: Blockchain based aggregator free federated learning,” in </span><em id="bib.bib63.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Blockchain (Blockchain)</em><span id="bib.bib63.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 72–81.
</span>
</span>
</li>
<li id="bib.bib64" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[64]</span>
<span class="ltx_bibblock"><span id="bib.bib64.1.1" class="ltx_text" style="color:#000000;">
S. Nakamoto, “Bitcoin: A peer-to-peer electronic cash system,” </span><em id="bib.bib64.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Decentralized Business Review</em><span id="bib.bib64.3.3" class="ltx_text" style="color:#000000;">, p. 21260, 2008.
</span>
</span>
</li>
<li id="bib.bib65" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[65]</span>
<span class="ltx_bibblock"><span id="bib.bib65.1.1" class="ltx_text" style="color:#000000;">
J. Frizzo-Barker, P. A. Chow-White, P. R. Adams, J. Mentanko, D. Ha, and S. Green, “Blockchain as a disruptive technology for business: A systematic review,” </span><em id="bib.bib65.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">International Journal of Information Management</em><span id="bib.bib65.3.3" class="ltx_text" style="color:#000000;">, vol. 51, p. 102029, 2020.
</span>
</span>
</li>
<li id="bib.bib66" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[66]</span>
<span class="ltx_bibblock"><span id="bib.bib66.1.1" class="ltx_text" style="color:#000000;">
M. Pournader, Y. Shi, S. Seuring, and S. L. Koh, “Blockchain applications in supply chains, transport and logistics: a systematic review of the literature,” </span><em id="bib.bib66.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">International Journal of Production Research</em><span id="bib.bib66.3.3" class="ltx_text" style="color:#000000;">, vol. 58, no. 7, pp. 2063–2081, 2020.
</span>
</span>
</li>
<li id="bib.bib67" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[67]</span>
<span class="ltx_bibblock"><span id="bib.bib67.1.1" class="ltx_text" style="color:#000000;">
M. A. Engelhardt, “Hitching healthcare to the chain: An introduction to blockchain technology in the healthcare sector,” </span><em id="bib.bib67.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Technology Innovation Management Review</em><span id="bib.bib67.3.3" class="ltx_text" style="color:#000000;">, vol. 7, no. 10, 2017.
</span>
</span>
</li>
<li id="bib.bib68" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[68]</span>
<span class="ltx_bibblock"><span id="bib.bib68.1.1" class="ltx_text" style="color:#000000;">
C. C. Agbo, Q. H. Mahmoud, and J. M. Eklund, “Blockchain technology in healthcare: a systematic review,” in </span><em id="bib.bib68.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the Healthcare</em><span id="bib.bib68.3.3" class="ltx_text" style="color:#000000;">, vol. 7, no. 2, 2019, p. 56.
</span>
</span>
</li>
<li id="bib.bib69" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[69]</span>
<span class="ltx_bibblock"><span id="bib.bib69.1.1" class="ltx_text" style="color:#000000;">
K. N. Griggs, O. Ossipova, C. P. Kohlios, A. N. Baccarini, E. A. Howson, and T. Hayajneh, “Healthcare blockchain system using smart contracts for secure automated remote patient monitoring,” </span><em id="bib.bib69.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of medical systems</em><span id="bib.bib69.3.3" class="ltx_text" style="color:#000000;">, vol. 42, no. 7, pp. 1–7, 2018.
</span>
</span>
</li>
<li id="bib.bib70" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[70]</span>
<span class="ltx_bibblock"><span id="bib.bib70.1.1" class="ltx_text" style="color:#000000;">
M. Hölbl, M. Kompara, A. Kamišalić, and L. Nemec Zlatolas, “A systematic review of the use of blockchain in healthcare,” </span><em id="bib.bib70.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Symmetry</em><span id="bib.bib70.3.3" class="ltx_text" style="color:#000000;">, vol. 10, no. 10, p. 470, 2018.
</span>
</span>
</li>
<li id="bib.bib71" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[71]</span>
<span class="ltx_bibblock"><span id="bib.bib71.1.1" class="ltx_text" style="color:#000000;">
H. D. Zubaydi, Y.-W. Chong, K. Ko, S. M. Hanshi, and S. Karuppayah, “A review on the role of blockchain technology in the healthcare domain,” </span><em id="bib.bib71.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Electronics</em><span id="bib.bib71.3.3" class="ltx_text" style="color:#000000;">, vol. 8, no. 6, p. 679, 2019.
</span>
</span>
</li>
<li id="bib.bib72" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[72]</span>
<span class="ltx_bibblock"><span id="bib.bib72.1.1" class="ltx_text" style="color:#000000;">
A. Hasselgren, K. Kralevska, D. Gligoroski, S. A. Pedersen, and A. Faxvaag, “Blockchain in healthcare and health sciences—A scoping review,” </span><em id="bib.bib72.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">International Journal of Medical Informatics</em><span id="bib.bib72.3.3" class="ltx_text" style="color:#000000;">, vol. 134, p. 104040, 2020.
</span>
</span>
</li>
<li id="bib.bib73" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[73]</span>
<span class="ltx_bibblock"><span id="bib.bib73.1.1" class="ltx_text" style="color:#000000;">
A. A. Vazirani, O. O’Donoghue, D. Brindley, and E. Meinert, “Blockchain vehicles for efficient medical record management,” </span><em id="bib.bib73.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">NPJ digital medicine</em><span id="bib.bib73.3.3" class="ltx_text" style="color:#000000;">, vol. 3, no. 1, pp. 1–5, 2020.
</span>
</span>
</li>
<li id="bib.bib74" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[74]</span>
<span class="ltx_bibblock"><span id="bib.bib74.1.1" class="ltx_text" style="color:#000000;">
J. Passerat-Palmbach, T. Farnan, R. Miller, M. S. Gross, H. L. Flannery, and B. Gleim, “A blockchain-orchestrated federated learning architecture for healthcare consortia,” </span><em id="bib.bib74.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:1910.12603</em><span id="bib.bib74.3.3" class="ltx_text" style="color:#000000;">, 2019.
</span>
</span>
</li>
<li id="bib.bib75" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[75]</span>
<span class="ltx_bibblock"><span id="bib.bib75.1.1" class="ltx_text" style="color:#000000;">
K. Bonawitz, V. Ivanov, B. Kreuter, A. Marcedone, H. B. McMahan, S. Patel, D. Ramage, A. Segal, and K. Seth, “Practical secure aggregation for privacy-preserving machine learning,” in </span><em id="bib.bib75.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">proceedings of the ACM SIGSAC Conference on Computer and Communications Security</em><span id="bib.bib75.3.3" class="ltx_text" style="color:#000000;">, 2017, pp. 1175–1191.
</span>
</span>
</li>
<li id="bib.bib76" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[76]</span>
<span class="ltx_bibblock"><span id="bib.bib76.1.1" class="ltx_text" style="color:#000000;">
D. Połap, G. Srivastava, and K. Yu, “Agent architecture of an intelligent medical system based on federated learning and blockchain technology,” </span><em id="bib.bib76.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of Information Security and Applications</em><span id="bib.bib76.3.3" class="ltx_text" style="color:#000000;">, vol. 58, p. 102748, 2021.
</span>
</span>
</li>
<li id="bib.bib77" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[77]</span>
<span class="ltx_bibblock"><span id="bib.bib77.1.1" class="ltx_text" style="color:#000000;">
Z. Shae and J. Tsai, “Transform blockchain into distributed parallel computing architecture for precision medicine,” in </span><em id="bib.bib77.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Distributed Computing Systems (ICDCS)</em><span id="bib.bib77.3.3" class="ltx_text" style="color:#000000;">, 2018, pp. 1290–1299.
</span>
</span>
</li>
<li id="bib.bib78" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[78]</span>
<span class="ltx_bibblock"><span id="bib.bib78.1.1" class="ltx_text" style="color:#000000;">
X. Chen, X. Wang, and K. Yang, “Asynchronous Blockchain-based privacy-preserving training framework for disease diagnosis,” in </span><em id="bib.bib78.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Big Data (Big Data)</em><span id="bib.bib78.3.3" class="ltx_text" style="color:#000000;">, 2019, pp. 5469–5473.
</span>
</span>
</li>
<li id="bib.bib79" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[79]</span>
<span class="ltx_bibblock"><span id="bib.bib79.1.1" class="ltx_text" style="color:#000000;">
W. S. W. Care, “Internet of Things,” </span><em id="bib.bib79.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Smart Healthcare Service, Wellness IT</em><span id="bib.bib79.3.3" class="ltx_text" style="color:#000000;">, 2016.
</span>
</span>
</li>
<li id="bib.bib80" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[80]</span>
<span class="ltx_bibblock"><span id="bib.bib80.1.1" class="ltx_text" style="color:#000000;">
L. M. Dang, M. Piran, D. Han, K. Min, H. Moon, </span><em id="bib.bib80.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib80.3.3" class="ltx_text" style="color:#000000;">, “A survey on internet of things and cloud computing for healthcare,” </span><em id="bib.bib80.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Electronics</em><span id="bib.bib80.5.5" class="ltx_text" style="color:#000000;">, vol. 8, no. 7, p. 768, 2019.
</span>
</span>
</li>
<li id="bib.bib81" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[81]</span>
<span class="ltx_bibblock"><span id="bib.bib81.1.1" class="ltx_text" style="color:#000000;">
R. Wang, J. Lai, Z. Zhang, X. Li, P. Vijayakumar, and M. Karuppiah, “Privacy-Preserving Federated Learning for Internet of Medical Things under Edge Computing,” </span><em id="bib.bib81.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Journal of Biomedical and Health Informatics</em><span id="bib.bib81.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib82" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[82]</span>
<span class="ltx_bibblock"><span id="bib.bib82.1.1" class="ltx_text" style="color:#000000;">
Y. Zhao, J. Zhao, L. Jiang, R. Tan, D. Niyato, Z. Li, L. Lyu, and Y. Liu, “Privacy-preserving blockchain-based federated learning for IoT devices,” </span><em id="bib.bib82.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE IoT Journal</em><span id="bib.bib82.3.3" class="ltx_text" style="color:#000000;">, vol. 8, no. 3, pp. 1817–1829, 2020.
</span>
</span>
</li>
<li id="bib.bib83" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[83]</span>
<span class="ltx_bibblock"><span id="bib.bib83.1.1" class="ltx_text" style="color:#000000;">
H. G. Abreha, C. J. Bernardos, A. D. L. Oliva, L. Cominardi, and A. Azcorra, “Monitoring in fog computing: state-of-the-art and research challenges,” </span><em id="bib.bib83.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">International Journal of Ad Hoc and Ubiquitous Computing</em><span id="bib.bib83.3.3" class="ltx_text" style="color:#000000;">, vol. 36, no. 2, pp. 114–130, 2021.
</span>
</span>
</li>
<li id="bib.bib84" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[84]</span>
<span class="ltx_bibblock"><span id="bib.bib84.1.1" class="ltx_text" style="color:#000000;">
Y. Mao, C. You, J. Zhang, K. Huang, and K. B. Letaief, “A survey on mobile edge computing: The communication perspective,” </span><em id="bib.bib84.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE communications surveys &amp; tutorials</em><span id="bib.bib84.3.3" class="ltx_text" style="color:#000000;">, vol. 19, no. 4, pp. 2322–2358, 2017.
</span>
</span>
</li>
<li id="bib.bib85" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[85]</span>
<span class="ltx_bibblock"><span id="bib.bib85.1.1" class="ltx_text" style="color:#000000;">
P. Mach and Z. Becvar, “Mobile edge computing: A survey on architecture and computation offloading,” </span><em id="bib.bib85.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Communications Surveys &amp; Tutorials</em><span id="bib.bib85.3.3" class="ltx_text" style="color:#000000;">, vol. 19, no. 3, pp. 1628–1656, 2017.
</span>
</span>
</li>
<li id="bib.bib86" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[86]</span>
<span class="ltx_bibblock"><span id="bib.bib86.1.1" class="ltx_text" style="color:#000000;">
Q. Wu, X. Chen, Z. Zhou, and J. Zhang, “FedHome: Cloud-edge based personalized federated learning for in-home health monitoring,” </span><em id="bib.bib86.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Mobile Computing</em><span id="bib.bib86.3.3" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib87" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[87]</span>
<span class="ltx_bibblock"><span id="bib.bib87.1.1" class="ltx_text" style="color:#000000;">
M. García-Valls, C. Calva-Urrego, and A. García-Fornes, “Accelerating smart eHealth services execution at the fog computing infrastructure,” </span><em id="bib.bib87.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Future generation computer systems</em><span id="bib.bib87.3.3" class="ltx_text" style="color:#000000;">, vol. 108, pp. 882–893, 2020.
</span>
</span>
</li>
<li id="bib.bib88" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[88]</span>
<span class="ltx_bibblock"><span id="bib.bib88.1.1" class="ltx_text" style="color:#000000;">
P. Pace, G. Aloi, R. Gravina, G. Caliciuri, G. Fortino, and A. Liotta, “An edge-based architecture to support efficient applications for healthcare industry 4.0,” </span><em id="bib.bib88.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Industrial Informatics</em><span id="bib.bib88.3.3" class="ltx_text" style="color:#000000;">, vol. 15, no. 1, pp. 481–489, 2018.
</span>
</span>
</li>
<li id="bib.bib89" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[89]</span>
<span class="ltx_bibblock"><span id="bib.bib89.1.1" class="ltx_text" style="color:#000000;">
S. Hakak, S. Ray, W. Z. Khan, and E. Scheme, “A framework for edge-assisted healthcare data analytics using federated learning,” in </span><em id="bib.bib89.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Big Data (Big Data)</em><span id="bib.bib89.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 3423–3427.
</span>
</span>
</li>
<li id="bib.bib90" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[90]</span>
<span class="ltx_bibblock"><span id="bib.bib90.1.1" class="ltx_text" style="color:#000000;">
D. C. Nguyen, M. Ding, Q.-V. Pham, P. N. Pathirana, L. B. Le, A. Seneviratne, J. Li, D. Niyato, and H. V. Poor, “Federated learning meets blockchain in edge computing: Opportunities and challenges,” </span><em id="bib.bib90.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Internet of Things Journal</em><span id="bib.bib90.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib91" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[91]</span>
<span class="ltx_bibblock"><span id="bib.bib91.1.1" class="ltx_text" style="color:#000000;">
M. A. Rahman, M. S. Hossain, G. Loukas, E. Hassanain, S. S. Rahman, M. F. Alhamid, and M. Guizani, “Blockchain-based mobile edge computing framework for secure therapy applications,” </span><em id="bib.bib91.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Access</em><span id="bib.bib91.3.3" class="ltx_text" style="color:#000000;">, vol. 6, pp. 72 469–72 478, 2018.
</span>
</span>
</li>
<li id="bib.bib92" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[92]</span>
<span class="ltx_bibblock"><span id="bib.bib92.1.1" class="ltx_text" style="color:#000000;">
M. Asif-Ur-Rahman, F. Afsana, M. Mahmud, M. S. Kaiser, M. R. Ahmed, O. Kaiwartya, and A. James-Taylor, “Toward a heterogeneous mist, fog, and cloud-based framework for the internet of healthcare things,” </span><em id="bib.bib92.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE IoT Journal</em><span id="bib.bib92.3.3" class="ltx_text" style="color:#000000;">, vol. 6, no. 3, pp. 4049–4062, 2018.
</span>
</span>
</li>
<li id="bib.bib93" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[93]</span>
<span class="ltx_bibblock"><span id="bib.bib93.1.1" class="ltx_text" style="color:#000000;">
A. A. Abdellatif, A. Z. Al-Marridi, A. Mohamed, A. Erbad, C. F. Chiasserini, and A. Refaey, “sshealth: toward secure, blockchain-enabled healthcare systems,” </span><em id="bib.bib93.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Network</em><span id="bib.bib93.3.3" class="ltx_text" style="color:#000000;">, vol. 34, no. 4, pp. 312–319, 2020.
</span>
</span>
</li>
<li id="bib.bib94" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[94]</span>
<span class="ltx_bibblock"><span id="bib.bib94.1.1" class="ltx_text" style="color:#000000;">
M. A. Rahman, E. Hassanain, M. M. Rashid, S. J. Barnes, and M. S. Hossain, “Spatial blockchain-based secure mass screening framework for children with dyslexia,” </span><em id="bib.bib94.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Access</em><span id="bib.bib94.3.3" class="ltx_text" style="color:#000000;">, vol. 6, 2018.
</span>
</span>
</li>
<li id="bib.bib95" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[95]</span>
<span class="ltx_bibblock"><span id="bib.bib95.1.1" class="ltx_text" style="color:#000000;">
T.-T. Kuo, H.-E. Kim, and L. Ohno-Machado, “Blockchain distributed ledger technologies for biomedical and health care applications,” </span><em id="bib.bib95.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of the American Medical Informatics Association</em><span id="bib.bib95.3.3" class="ltx_text" style="color:#000000;">, vol. 24, no. 6, pp. 1211–1220, 2017.
</span>
</span>
</li>
<li id="bib.bib96" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[96]</span>
<span class="ltx_bibblock"><span id="bib.bib96.1.1" class="ltx_text" style="color:#000000;">
D. C. Nguyen, P. N. Pathirana, M. Ding, and A. Seneviratne, “Blockchain and edge computing for decentralized EMRs sharing in federated healthcare,” in </span><em id="bib.bib96.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the GLOBECOM 2020-2020 IEEE Global Communications Conference</em><span id="bib.bib96.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 1–6.
</span>
</span>
</li>
<li id="bib.bib97" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[97]</span>
<span class="ltx_bibblock"><span id="bib.bib97.1.1" class="ltx_text" style="color:#000000;">
X. Wang, Y. Han, C. Wang, Q. Zhao, X. Chen, and M. Chen, “In-edge ai: Intelligentizing mobile edge computing, caching and communication by federated learning,” </span><em id="bib.bib97.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Network</em><span id="bib.bib97.3.3" class="ltx_text" style="color:#000000;">, vol. 33, no. 5, 2019.
</span>
</span>
</li>
<li id="bib.bib98" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[98]</span>
<span class="ltx_bibblock"><span id="bib.bib98.1.1" class="ltx_text" style="color:#000000;">
I. Dayan, H. R. Roth, A. Zhong, A. Harouni, A. Gentili, A. Z. Abidin, A. Liu, A. B. Costa, B. J. Wood, C.-S. Tsai, </span><em id="bib.bib98.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib98.3.3" class="ltx_text" style="color:#000000;">, “Federated learning for predicting clinical outcomes in patients with COVID-19,” </span><em id="bib.bib98.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Nature medicine</em><span id="bib.bib98.5.5" class="ltx_text" style="color:#000000;">, vol. 27, no. 10, pp. 1735–1743, 2021.
</span>
</span>
</li>
<li id="bib.bib99" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[99]</span>
<span class="ltx_bibblock"><span id="bib.bib99.1.1" class="ltx_text" style="color:#000000;">
Y. Xu, L. Ma, F. Yang, Y. Chen, K. Ma, J. Yang, X. Yang, Y. Chen, C. Shu, Z. Fan, </span><em id="bib.bib99.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib99.3.3" class="ltx_text" style="color:#000000;">, “A collaborative online AI engine for CT-based COVID-19 diagnosis,” </span><em id="bib.bib99.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">medRxiv</em><span id="bib.bib99.5.5" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib100" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[100]</span>
<span class="ltx_bibblock"><span id="bib.bib100.1.1" class="ltx_text" style="color:#000000;">
J. L. Raisaro, F. Marino, J. Troncoso-Pastoriza, R. Beau-Lejdstrom, R. Bellazzi, R. Murphy, E. V. Bernstam, H. Wang, M. Bucalo, Y. Chen, </span><em id="bib.bib100.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib100.3.3" class="ltx_text" style="color:#000000;">, “SCOR: A secure international informatics infrastructure to investigate COVID-19,” </span><em id="bib.bib100.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of the American Medical Informatics Association</em><span id="bib.bib100.5.5" class="ltx_text" style="color:#000000;">, vol. 27, no. 11, pp. 1721–1726, 2020.
</span>
</span>
</li>
<li id="bib.bib101" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[101]</span>
<span class="ltx_bibblock"><span id="bib.bib101.1.1" class="ltx_text" style="color:#000000;">
A. Vaid, S. K. Jaladanki, J. Xu, S. Teng, A. Kumar, S. Lee, S. Somani, I. Paranjpe, J. K. De Freitas, T. Wanyan, </span><em id="bib.bib101.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib101.3.3" class="ltx_text" style="color:#000000;">, “Federated learning of electronic health records to improve mortality prediction in hospitalized patients with COVID-19: Machine learning approach,” </span><em id="bib.bib101.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">JMIR medical informatics</em><span id="bib.bib101.5.5" class="ltx_text" style="color:#000000;">, vol. 9, no. 1, p. e24207, 2021.
</span>
</span>
</li>
<li id="bib.bib102" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[102]</span>
<span class="ltx_bibblock"><span id="bib.bib102.1.1" class="ltx_text" style="color:#000000;">
D. Yang, Z. Xu, W. Li, A. Myronenko, H. R. Roth, S. Harmon, S. Xu, B. Turkbey, E. Turkbey, X. Wang, </span><em id="bib.bib102.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib102.3.3" class="ltx_text" style="color:#000000;">, “Federated semi-supervised learning for COVID region segmentation in chest CT using multi-national data from China, Italy, Japan,” </span><em id="bib.bib102.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Medical image analysis</em><span id="bib.bib102.5.5" class="ltx_text" style="color:#000000;">, vol. 70, p. 101992, 2021.
</span>
</span>
</li>
<li id="bib.bib103" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[103]</span>
<span class="ltx_bibblock"><span id="bib.bib103.1.1" class="ltx_text" style="color:#000000;">
Q. Dou, T. Y. So, M. Jiang, Q. Liu, V. Vardhanabhuti, G. Kaissis, Z. Li, W. Si, H. H. Lee, K. Yu, </span><em id="bib.bib103.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib103.3.3" class="ltx_text" style="color:#000000;">, “Federated deep learning for detecting COVID-19 lung abnormalities in CT: a privacy-preserving multinational validation study,” </span><em id="bib.bib103.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">NPJ digital medicine</em><span id="bib.bib103.5.5" class="ltx_text" style="color:#000000;">, vol. 4, no. 1, pp. 1–11, 2021.
</span>
</span>
</li>
<li id="bib.bib104" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[104]</span>
<span class="ltx_bibblock"><span id="bib.bib104.1.1" class="ltx_text" style="color:#000000;">
S. Naz, K. T. Phan, and Y.-P. P. Chen, “A comprehensive review of federated learning for covid-19 detection,” </span><em id="bib.bib104.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">International Journal of Intelligent Systems</em><span id="bib.bib104.3.3" class="ltx_text" style="color:#000000;">, vol. 37, no. 3, pp. 2371–2392, 2022.
</span>
</span>
</li>
<li id="bib.bib105" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[105]</span>
<span class="ltx_bibblock"><span id="bib.bib105.1.1" class="ltx_text" style="color:#000000;">
F. Qian and A. Zhang, “The value of federated learning during and post-covid-19,” </span><em id="bib.bib105.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">International Journal for Quality in Health Care</em><span id="bib.bib105.3.3" class="ltx_text" style="color:#000000;">, vol. 33, no. 1, p. mzab010, 2021.
</span>
</span>
</li>
<li id="bib.bib106" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[106]</span>
<span class="ltx_bibblock"><span id="bib.bib106.1.1" class="ltx_text" style="color:#000000;">
X. Bai, H. Wang, L. Ma, Y. Xu, J. Gan, Z. Fan, F. Yang, K. Ma, J. Yang, S. Bai, </span><em id="bib.bib106.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib106.3.3" class="ltx_text" style="color:#000000;">, “Advancing covid-19 diagnosis with privacy-preserving collaboration in artificial intelligence,” </span><em id="bib.bib106.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Nature Machine Intelligence</em><span id="bib.bib106.5.5" class="ltx_text" style="color:#000000;">, vol. 3, no. 12, pp. 1081–1089, 2021.
</span>
</span>
</li>
<li id="bib.bib107" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[107]</span>
<span class="ltx_bibblock"><span id="bib.bib107.1.1" class="ltx_text" style="color:#000000;">
A. Vaid, S. K. Jaladanki, J. Xu, S. Teng, A. Kumar, S. Lee, S. Somani, I. Paranjpe, J. K. De Freitas, T. Wanyan, </span><em id="bib.bib107.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib107.3.3" class="ltx_text" style="color:#000000;">, “Federated learning of electronic health records improves mortality prediction in patients hospitalized with covid-19,” </span><em id="bib.bib107.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">medRxiv</em><span id="bib.bib107.5.5" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib108" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[108]</span>
<span class="ltx_bibblock"><span id="bib.bib108.1.1" class="ltx_text" style="color:#000000;">
I. Feki, S. Ammar, Y. Kessentini, and K. Muhammad, “Federated learning for covid-19 screening from chest x-ray images,” </span><em id="bib.bib108.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Applied Soft Computing</em><span id="bib.bib108.3.3" class="ltx_text" style="color:#000000;">, vol. 106, p. 107330, 2021.
</span>
</span>
</li>
<li id="bib.bib109" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[109]</span>
<span class="ltx_bibblock"><span id="bib.bib109.1.1" class="ltx_text" style="color:#000000;">
B. Yan, J. Wang, J. Cheng, Y. Zhou, Y. Zhang, Y. Yang, L. Liu, H. Zhao, C. Wang, and B. Liu, “Experiments of federated learning for covid-19 chest x-ray images,” in </span><em id="bib.bib109.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Artificial Intelligence and Security</em><span id="bib.bib109.3.3" class="ltx_text" style="color:#000000;">, 2021, pp. 41–53.
</span>
</span>
</li>
<li id="bib.bib110" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[110]</span>
<span class="ltx_bibblock"><span id="bib.bib110.1.1" class="ltx_text" style="color:#000000;">
B. Liu, B. Yan, Y. Zhou, Y. Yang, and Y. Zhang, “Experiments of federated learning for covid-19 chest x-ray images,” </span><em id="bib.bib110.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2007.05592</em><span id="bib.bib110.3.3" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib111" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[111]</span>
<span class="ltx_bibblock"><span id="bib.bib111.1.1" class="ltx_text" style="color:#000000;">
W. Zhang, T. Zhou, Q. Lu, X. Wang, C. Zhu, H. Sun, Z. Wang, S. K. Lo, and F.-Y. Wang, “Dynamic-fusion-based federated learning for covid-19 detection,” </span><em id="bib.bib111.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Internet of Things Journal</em><span id="bib.bib111.3.3" class="ltx_text" style="color:#000000;">, vol. 8, no. 21, pp. 15 884–15 891, 2021.
</span>
</span>
</li>
<li id="bib.bib112" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[112]</span>
<span class="ltx_bibblock"><span id="bib.bib112.1.1" class="ltx_text" style="color:#000000;">
M. Abdul Salam, S. Taha, and M. Ramadan, “COVID-19 detection using federated machine learning,” </span><em id="bib.bib112.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">PLoS One</em><span id="bib.bib112.3.3" class="ltx_text" style="color:#000000;">, vol. 16, no. 6, 2021.
</span>
</span>
</li>
<li id="bib.bib113" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[113]</span>
<span class="ltx_bibblock"><span id="bib.bib113.1.1" class="ltx_text" style="color:#000000;">
A. Bhattacharya, M. Gawali, J. Seth, and V. Kulkarni, “Application of federated learning in building a robust covid-19 chest x-ray classification model,” </span><em id="bib.bib113.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2204.10505</em><span id="bib.bib113.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib114" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[114]</span>
<span class="ltx_bibblock"><span id="bib.bib114.1.1" class="ltx_text" style="color:#000000;">
T.-T. Ho </span><em id="bib.bib114.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib114.3.3" class="ltx_text" style="color:#000000;">, “Dpcovid: Privacy-preserving federated covid-19 detection,” </span><em id="bib.bib114.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2110.13760</em><span id="bib.bib114.5.5" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib115" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[115]</span>
<span class="ltx_bibblock"><span id="bib.bib115.1.1" class="ltx_text" style="color:#000000;">
J. de Moura, J. Novo, and M. Ortega, “Fully automatic deep convolutional approaches for the analysis of covid-19 using chest x-ray images,” </span><em id="bib.bib115.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Applied Soft Computing</em><span id="bib.bib115.3.3" class="ltx_text" style="color:#000000;">, vol. 115, p. 108190, 2022.
</span>
</span>
</li>
<li id="bib.bib116" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[116]</span>
<span class="ltx_bibblock"><span id="bib.bib116.1.1" class="ltx_text" style="color:#000000;">
A. E. Cetinkaya, M. Akin, and S. Sagiroglu, “A communication efficient federated learning approach to multi chest diseases classification,” in </span><em id="bib.bib116.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Computer Science and Engineering (UBMK)</em><span id="bib.bib116.3.3" class="ltx_text" style="color:#000000;">, 2021, pp. 429–434.
</span>
</span>
</li>
<li id="bib.bib117" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[117]</span>
<span class="ltx_bibblock"><span id="bib.bib117.1.1" class="ltx_text" style="color:#000000;">
A. Ulhaq and O. Burmeister, “Covid-19 imaging data privacy by federated learning design: A theoretical framework,” </span><em id="bib.bib117.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2010.06177</em><span id="bib.bib117.3.3" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib118" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[118]</span>
<span class="ltx_bibblock"><span id="bib.bib118.1.1" class="ltx_text" style="color:#000000;">
S. Banerjee, R. Misra, M. Prasad, E. Elmroth, and M. H. Bhuyan, “Multi-diseases classification from chest-x-ray: A federated deep learning approach,” in </span><em id="bib.bib118.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the Australasian Joint Conference on Artificial Intelligence</em><span id="bib.bib118.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 3–15.
</span>
</span>
</li>
<li id="bib.bib119" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[119]</span>
<span class="ltx_bibblock"><span id="bib.bib119.1.1" class="ltx_text" style="color:#000000;">
M. U. Alam and R. Rahmani, “Federated semi-supervised multi-task learning to detect covid-19 and lungs segmentation marking using chest radiography images and raspberry pi devices: An internet of medical things application,” </span><em id="bib.bib119.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Sensors</em><span id="bib.bib119.3.3" class="ltx_text" style="color:#000000;">, vol. 21, no. 15, p. 5025, 2021.
</span>
</span>
</li>
<li id="bib.bib120" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[120]</span>
<span class="ltx_bibblock"><span id="bib.bib120.1.1" class="ltx_text" style="color:#000000;">
F. Ślazyk, P. Jabłecki, A. Lisowska, M. Malawski, and S. Płotka, “Cxr-fl: Deep learning-based chest x-ray image analysis using federated learning,” </span><em id="bib.bib120.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2204.05203</em><span id="bib.bib120.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib121" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[121]</span>
<span class="ltx_bibblock"><span id="bib.bib121.1.1" class="ltx_text" style="color:#000000;">
Y. Cao, “Near real-time federated machine learning approach over chest computed tomography for covid-19 diagnosis,” in </span><em id="bib.bib121.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Applications and Techniques in Information Security</em><span id="bib.bib121.3.3" class="ltx_text" style="color:#000000;">, 2021, pp. 21–36.
</span>
</span>
</li>
<li id="bib.bib122" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[122]</span>
<span class="ltx_bibblock"><span id="bib.bib122.1.1" class="ltx_text" style="color:#000000;">
L. Boyi, Y. Bingjie, Z. Yize, Y. Yifan, and Z. Yixian, “Experiments of federated learning for covid-19 chest x-ray images,” </span><em id="bib.bib122.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2007.05592</em><span id="bib.bib122.3.3" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib123" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[123]</span>
<span class="ltx_bibblock"><span id="bib.bib123.1.1" class="ltx_text" style="color:#000000;">
A. Kallel, M. Rekik, and M. Khemakhem, “Hybrid-based framework for covid-19 prediction via federated machine learning models,” </span><em id="bib.bib123.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">The Journal of Supercomputing</em><span id="bib.bib123.3.3" class="ltx_text" style="color:#000000;">, vol. 78, no. 5, pp. 7078–7105, 2022.
</span>
</span>
</li>
<li id="bib.bib124" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[124]</span>
<span class="ltx_bibblock"><span id="bib.bib124.1.1" class="ltx_text" style="color:#000000;">
S. Sakib, M. M. Fouda, Z. M. Fadlullah, and N. Nasser, “On covid-19 prediction using asynchronous federated learning-based agile radiograph screening booths,” in </span><em id="bib.bib124.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the ICC 2021-IEEE International Conference on Communications</em><span id="bib.bib124.3.3" class="ltx_text" style="color:#000000;">, 2021, pp. 1–6.
</span>
</span>
</li>
<li id="bib.bib125" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[125]</span>
<span class="ltx_bibblock"><span id="bib.bib125.1.1" class="ltx_text" style="color:#000000;">
A. Qayyum, K. Ahmad, M. A. Ahsan, A. Al-Fuqaha, and J. Qadir, “Collaborative federated learning for healthcare: Multi-modal covid-19 diagnosis at the edge,” </span><em id="bib.bib125.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2101.07511</em><span id="bib.bib125.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib126" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[126]</span>
<span class="ltx_bibblock"><span id="bib.bib126.1.1" class="ltx_text" style="color:#000000;">
X. Chen, Y. Shao, Z. Xue, and Z. Yu, “Multi-Modal COVID-19 Discovery With Collaborative Federated Learning,” in </span><em id="bib.bib126.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Cloud Computing and Intelligent Systems (CCIS)</em><span id="bib.bib126.3.3" class="ltx_text" style="color:#000000;">, 2021, pp. 52–56.
</span>
</span>
</li>
<li id="bib.bib127" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[127]</span>
<span class="ltx_bibblock"><span id="bib.bib127.1.1" class="ltx_text" style="color:#000000;">
S. K. Jaladanki, A. Vaid, A. S. Sawant, J. Xu, K. Shah, S. Dellepiane, I. Paranjpe, L. Chan, P. Kovatch, A. W. Charney, </span><em id="bib.bib127.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib127.3.3" class="ltx_text" style="color:#000000;">, “Development of a federated learning approach to predict acute kidney injury in adult hospitalized patients with covid-19 in new york city,” </span><em id="bib.bib127.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">medRxiv</em><span id="bib.bib127.5.5" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib128" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[128]</span>
<span class="ltx_bibblock"><span id="bib.bib128.1.1" class="ltx_text" style="color:#000000;">
R. Wang, J. Xu, Y. Ma, M. Talha, M. S. Al-Rakhami, and A. Ghoneim, “Auxiliary diagnosis of covid-19 based on 5g-enabled federated learning,” </span><em id="bib.bib128.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Ieee Network</em><span id="bib.bib128.3.3" class="ltx_text" style="color:#000000;">, vol. 35, no. 3, pp. 14–20, 2021.
</span>
</span>
</li>
<li id="bib.bib129" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[129]</span>
<span class="ltx_bibblock"><span id="bib.bib129.1.1" class="ltx_text" style="color:#000000;">
N. S. Brenčič, M. Dragoi, I. Mocanu, and T. Winiarski, “Intuitive and intelligent solutions for elderly care,” in </span><em id="bib.bib129.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Digital Health in Focus of Predictive, Preventive and Personalised Medicine</em><span id="bib.bib129.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 101–108.
</span>
</span>
</li>
<li id="bib.bib130" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[130]</span>
<span class="ltx_bibblock"><span id="bib.bib130.1.1" class="ltx_text" style="color:#000000;">
K. Sozinov, V. Vlassov, and S. Girdzijauskas, “Human activity recognition using federated learning,” in </span><em id="bib.bib130.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the Intl Conf on Parallel &amp; Distributed Processing with Applications, Ubiquitous Computing &amp; Communications, Big Data &amp; Cloud Computing, Social Computing &amp; Networking, Sustainable Computing &amp; Communications (ISPA/IUCC/BDCloud/SocialCom/SustainCom)</em><span id="bib.bib130.3.3" class="ltx_text" style="color:#000000;">, 2018, pp. 1103–1111.
</span>
</span>
</li>
<li id="bib.bib131" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[131]</span>
<span class="ltx_bibblock"><span id="bib.bib131.1.1" class="ltx_text" style="color:#000000;">
Y. Chen, Y. Ning, Z. Chai, and H. Rangwala, “Federated multi-task hierarchical attention model for sensor analytics,” </span><em id="bib.bib131.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:1905.05142</em><span id="bib.bib131.3.3" class="ltx_text" style="color:#000000;">, 2019.
</span>
</span>
</li>
<li id="bib.bib132" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[132]</span>
<span class="ltx_bibblock"><span id="bib.bib132.1.1" class="ltx_text" style="color:#000000;">
B. van Berlo, “Unsupervised feature learning in a federated setting for human activity detection,” Master’s thesis, Eindhoven University of Technology, Netherlands, 2019.
</span>
</span>
</li>
<li id="bib.bib133" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[133]</span>
<span class="ltx_bibblock"><span id="bib.bib133.1.1" class="ltx_text" style="color:#000000;">
R. Hesseberg and P. Minne, “Federated learning for dementia classification in a european multicentre dementia study,” Master’s thesis, University of Stavanger, Norway, 2020.
</span>
</span>
</li>
<li id="bib.bib134" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[134]</span>
<span class="ltx_bibblock"><span id="bib.bib134.1.1" class="ltx_text" style="color:#000000;">
H. R. Roth, K. Chang, P. Singh, N. Neumark, W. Li, V. Gupta, S. Gupta, L. Qu, A. Ihsani, B. C. Bizzo, </span><em id="bib.bib134.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib134.3.3" class="ltx_text" style="color:#000000;">, “Federated learning for breast density classification: A real-world implementation,” in </span><em id="bib.bib134.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Domain Adaptation and Representation Transfer, and Distributed and Collaborative Learning</em><span id="bib.bib134.5.5" class="ltx_text" style="color:#000000;">, 2020, pp. 181–191.
</span>
</span>
</li>
<li id="bib.bib135" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[135]</span>
<span class="ltx_bibblock"><span id="bib.bib135.1.1" class="ltx_text" style="color:#000000;">
M. J. Sheller, B. Edwards, G. A. Reina, J. Martin, S. Pati, A. Kotrotsou, M. Milchenko, W. Xu, D. Marcus, R. R. Colen, </span><em id="bib.bib135.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib135.3.3" class="ltx_text" style="color:#000000;">, “Federated learning in medicine: facilitating multi-institutional collaborations without sharing patient data,” </span><em id="bib.bib135.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Scientific reports</em><span id="bib.bib135.5.5" class="ltx_text" style="color:#000000;">, vol. 10, no. 1, p. 12598, 2020.
</span>
</span>
</li>
<li id="bib.bib136" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[136]</span>
<span class="ltx_bibblock"><span id="bib.bib136.1.1" class="ltx_text" style="color:#000000;">
S. W. Remedios, J. A. Butman, B. A. Landman, and D. L. Pham, “Federated gradient averaging for multi-site training with momentum-based optimizers,” in </span><em id="bib.bib136.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Domain Adaptation and Representation Transfer, and Distributed and Collaborative Learning</em><span id="bib.bib136.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 170–180.
</span>
</span>
</li>
<li id="bib.bib137" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[137]</span>
<span class="ltx_bibblock"><span id="bib.bib137.1.1" class="ltx_text" style="color:#000000;">
H. Sung, J. Ferlay, R. L. Siegel, M. Laversanne, I. Soerjomataram, A. Jemal, and F. Bray, “Global cancer statistics 2020: Globocan estimates of incidence and mortality worldwide for 36 cancers in 185 countries,” </span><em id="bib.bib137.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">CA: a cancer journal for clinicians</em><span id="bib.bib137.3.3" class="ltx_text" style="color:#000000;">, vol. 71, no. 3, pp. 209–249, 2021.
</span>
</span>
</li>
<li id="bib.bib138" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[138]</span>
<span class="ltx_bibblock"><span id="bib.bib138.1.1" class="ltx_text" style="color:#000000;">
S. Pati, U. Baid, B. Edwards, M. Sheller, S.-H. Wang, G. A. Reina, P. Foley, A. Gruzdev, D. Karkada, C. Davatzikos, </span><em id="bib.bib138.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib138.3.3" class="ltx_text" style="color:#000000;">, “Federated learning enables big data for rare cancer boundary detection,” </span><em id="bib.bib138.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Nature communications</em><span id="bib.bib138.5.5" class="ltx_text" style="color:#000000;">, vol. 13, no. 1, p. 7346, 2022.
</span>
</span>
</li>
<li id="bib.bib139" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[139]</span>
<span class="ltx_bibblock"><span id="bib.bib139.1.1" class="ltx_text" style="color:#000000;">
M. Adnan, S. Kalra, J. C. Cresswell, G. W. Taylor, and H. R. Tizhoosh, “Federated learning and differential privacy for medical image analysis,” </span><em id="bib.bib139.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Scientific reports</em><span id="bib.bib139.3.3" class="ltx_text" style="color:#000000;">, vol. 12, no. 1, pp. 1–10, 2022.
</span>
</span>
</li>
<li id="bib.bib140" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[140]</span>
<span class="ltx_bibblock"><span id="bib.bib140.1.1" class="ltx_text" style="color:#000000;">
G. T. Kennedy, F. S. Azari, E. Bernstein, B. Nadeem, A. Chang, A. Segil, S. Carlin, N. T. Sullivan, E. Encarnado, C. Desphande, </span><em id="bib.bib140.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib140.3.3" class="ltx_text" style="color:#000000;">, “Targeted detection of cancer at the cellular level during biopsy by near-infrared confocal laser endomicroscopy,” </span><em id="bib.bib140.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Nature Communications</em><span id="bib.bib140.5.5" class="ltx_text" style="color:#000000;">, vol. 13, no. 1, p. 2711, 2022.
</span>
</span>
</li>
<li id="bib.bib141" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[141]</span>
<span class="ltx_bibblock"><span id="bib.bib141.1.1" class="ltx_text" style="color:#000000;">
A. Jiménez-Sánchez, M. Tardy, M. A. G. Ballester, D. Mateus, and G. Piella, “Memory-aware curriculum federated learning for breast cancer classification,” </span><em id="bib.bib141.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2107.02504</em><span id="bib.bib141.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib142" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[142]</span>
<span class="ltx_bibblock"><span id="bib.bib142.1.1" class="ltx_text" style="color:#000000;">
A. Jiménez-Sánchez, M. Tardy, M. A. González Ballester, D. Mateus, and G. Piella, “Memory-aware curriculum federated learning for breast cancer classification,” </span><em id="bib.bib142.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Computer Methods and Programs in Biomedicine</em><span id="bib.bib142.3.3" class="ltx_text" style="color:#000000;">, vol. 229, p. 107318, 2023. [Online]. Available: </span><span class="ltx_ref ltx_nolink ltx_url ltx_ref_self" style="color:#000000;">https://www.sciencedirect.com/science/article/pii/S016926072200699X</span><span id="bib.bib142.4.4" class="ltx_text" style="color:#000000;">
</span>
</span>
</li>
<li id="bib.bib143" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[143]</span>
<span class="ltx_bibblock"><span id="bib.bib143.1.1" class="ltx_text" style="color:#000000;">
R. Muthukrishnan, A. Heyler, K. Katti, S. Pati, W. Mankowski, A. Alahari, M. Sanborn, E. F. Conant, C. Scott, S. Winham, </span><em id="bib.bib143.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib143.3.3" class="ltx_text" style="color:#000000;">, “MammoDL: mammographic breast density estimation using federated learning,” </span><em id="bib.bib143.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2206.05575</em><span id="bib.bib143.5.5" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib144" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[144]</span>
<span class="ltx_bibblock"><span id="bib.bib144.1.1" class="ltx_text" style="color:#000000;">
S. Bakas, H. Akbari, A. Sotiras, M. Bilello, M. Rozycki, J. S. Kirby, J. B. Freymann, K. Farahani, and C. Davatzikos, “Advancing the cancer genome atlas glioma mri collections with expert segmentation labels and radiomic features,” </span><em id="bib.bib144.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Scientific data</em><span id="bib.bib144.3.3" class="ltx_text" style="color:#000000;">, vol. 4, no. 1, pp. 1–13, 2017.
</span>
</span>
</li>
<li id="bib.bib145" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[145]</span>
<span class="ltx_bibblock"><span id="bib.bib145.1.1" class="ltx_text" style="color:#000000;">
Z. Tang, S. Ahmad, P.-T. Yap, and D. Shen, “Multi-atlas segmentation of mr tumor brain images using low-rank based image recovery,” </span><em id="bib.bib145.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE transactions on medical imaging</em><span id="bib.bib145.3.3" class="ltx_text" style="color:#000000;">, vol. 37, no. 10, pp. 2224–2235, 2018.
</span>
</span>
</li>
<li id="bib.bib146" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[146]</span>
<span class="ltx_bibblock"><span id="bib.bib146.1.1" class="ltx_text" style="color:#000000;">
W. Li, F. Milletarì, D. Xu, N. Rieke, J. Hancox, W. Zhu, M. Baust, Y. Cheng, S. Ourselin, M. J. Cardoso, </span><em id="bib.bib146.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib146.3.3" class="ltx_text" style="color:#000000;">, “Privacy-preserving federated brain tumour segmentation,” in </span><em id="bib.bib146.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Proc. of the Intl. workshop on machine learning in medical imaging</em><span id="bib.bib146.5.5" class="ltx_text" style="color:#000000;">, 2019, pp. 133–141.
</span>
</span>
</li>
<li id="bib.bib147" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[147]</span>
<span class="ltx_bibblock"><span id="bib.bib147.1.1" class="ltx_text" style="color:#000000;">
B. H. Menze, A. Jakab, S. Bauer, J. Kalpathy-Cramer, K. Farahani, J. Kirby, Y. Burren, N. Porz, J. Slotboom, R. Wiest, </span><em id="bib.bib147.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib147.3.3" class="ltx_text" style="color:#000000;">, “The multimodal brain tumor image segmentation benchmark (brats),” </span><em id="bib.bib147.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE transactions on medical imaging</em><span id="bib.bib147.5.5" class="ltx_text" style="color:#000000;">, vol. 34, no. 10, pp. 1993–2024, 2014.
</span>
</span>
</li>
<li id="bib.bib148" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[148]</span>
<span class="ltx_bibblock"><span id="bib.bib148.1.1" class="ltx_text" style="color:#000000;">
M. J. Sheller, G. A. Reina, B. Edwards, J. Martin, and S. Bakas, “Multi-institutional deep learning modeling without sharing patient data: A feasibility study on brain tumor segmentation,” in </span><em id="bib.bib148.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International MICCAI Brainlesion Workshop</em><span id="bib.bib148.3.3" class="ltx_text" style="color:#000000;">, 2018, pp. 92–104.
</span>
</span>
</li>
<li id="bib.bib149" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[149]</span>
<span class="ltx_bibblock"><span id="bib.bib149.1.1" class="ltx_text" style="color:#000000;">
A. Leufkens, M. Van Oijen, F. Vleggaar, and P. Siersema, “Factors influencing the miss rate of polyps in a back-to-back colonoscopy study,” </span><em id="bib.bib149.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Endoscopy</em><span id="bib.bib149.3.3" class="ltx_text" style="color:#000000;">, vol. 44, no. 05, pp. 470–475, 2012.
</span>
</span>
</li>
<li id="bib.bib150" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[150]</span>
<span class="ltx_bibblock"><span id="bib.bib150.1.1" class="ltx_text" style="color:#000000;">
S. Ali, D. Jha, N. Ghatwary, S. Realdon, R. Cannizzaro, O. E. Salem, D. Lamarque, C. Daul, K. V. Anonsen, M. A. Riegler, </span><em id="bib.bib150.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib150.3.3" class="ltx_text" style="color:#000000;">, “Polypgen: A multi-center polyp detection and segmentation dataset for generalisability assessment,” </span><em id="bib.bib150.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2106.04463</em><span id="bib.bib150.5.5" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib151" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[151]</span>
<span class="ltx_bibblock"><span id="bib.bib151.1.1" class="ltx_text" style="color:#000000;">
S. Ali, N. Ghatwary, D. Jha, E. Isik-Polat, G. Polat, C. Yang, W. Li, A. Galdran, M.-Á. G. Ballester, V. Thambawita, </span><em id="bib.bib151.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib151.3.3" class="ltx_text" style="color:#000000;">, “Assessing generalisability of deep learning-based polyp detection and segmentation methods through a computer vision challenge,” </span><em id="bib.bib151.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2202.12031</em><span id="bib.bib151.5.5" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib152" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[152]</span>
<span class="ltx_bibblock"><span id="bib.bib152.1.1" class="ltx_text" style="color:#000000;">
N. Aldoj, S. Lukas, M. Dewey, and T. Penzkofer, “Semi-automatic classification of prostate cancer on multi-parametric mr imaging using a multi-channel 3d convolutional neural network,” </span><em id="bib.bib152.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">European radiology</em><span id="bib.bib152.3.3" class="ltx_text" style="color:#000000;">, vol. 30, no. 2, pp. 1243–1253, 2020.
</span>
</span>
</li>
<li id="bib.bib153" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[153]</span>
<span class="ltx_bibblock"><span id="bib.bib153.1.1" class="ltx_text" style="color:#000000;">
Y. Wang, J.-Z. Cheng, D. Ni, M. Lin, J. Qin, X. Luo, M. Xu, X. Xie, and P. A. Heng, “Towards personalized statistical deformable model and hybrid point matching for robust mr-trus registration,” </span><em id="bib.bib153.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE transactions on medical imaging</em><span id="bib.bib153.3.3" class="ltx_text" style="color:#000000;">, vol. 35, no. 2, pp. 589–604, 2015.
</span>
</span>
</li>
<li id="bib.bib154" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[154]</span>
<span class="ltx_bibblock"><span id="bib.bib154.1.1" class="ltx_text" style="color:#000000;">
K. V. Sarma, S. Harmon, T. Sanford, H. R. Roth, Z. Xu, J. Tetreault, D. Xu, M. G. Flores, A. G. Raman, R. Kulkarni, </span><em id="bib.bib154.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib154.3.3" class="ltx_text" style="color:#000000;">, “Federated learning improves site performance in multicenter deep learning without data sharing,” </span><em id="bib.bib154.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of the American Medical Informatics Association</em><span id="bib.bib154.5.5" class="ltx_text" style="color:#000000;">, vol. 28, no. 6, pp. 1259–1264, 2021.
</span>
</span>
</li>
<li id="bib.bib155" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[155]</span>
<span class="ltx_bibblock"><span id="bib.bib155.1.1" class="ltx_text" style="color:#000000;">
S. Liu, D. Xu, S. K. Zhou, O. Pauly, S. Grbic, T. Mertelmeier, J. Wicklein, A. Jerebko, W. Cai, and D. Comaniciu, “3d anisotropic hybrid network: Transferring convolutional features from 2d images to 3d anisotropic volumes,” in </span><em id="bib.bib155.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International conference on medical image computing and computer-assisted intervention</em><span id="bib.bib155.3.3" class="ltx_text" style="color:#000000;">, 2018, pp. 851–858.
</span>
</span>
</li>
<li id="bib.bib156" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[156]</span>
<span class="ltx_bibblock"><span id="bib.bib156.1.1" class="ltx_text" style="color:#000000;">
Q. Liu, C. Chen, J. Qin, Q. Dou, and P.-A. Heng, “Feddg: Federated domain generalization on medical image segmentation via episodic learning in continuous frequency space,” in </span><em id="bib.bib156.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition</em><span id="bib.bib156.3.3" class="ltx_text" style="color:#000000;">, 2021, pp. 1013–1023.
</span>
</span>
</li>
<li id="bib.bib157" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[157]</span>
<span class="ltx_bibblock"><span id="bib.bib157.1.1" class="ltx_text" style="color:#000000;">
P. H. Smedsrud, V. Thambawita, S. A. Hicks, H. Gjestang, O. O. Nedrejord, E. Næss, H. Borgli, D. Jha, T. J. D. Berstad, S. L. Eskeland, </span><em id="bib.bib157.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib157.3.3" class="ltx_text" style="color:#000000;">, “Kvasir-capsule, a video capsule endoscopy dataset,” </span><em id="bib.bib157.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Scientific Data</em><span id="bib.bib157.5.5" class="ltx_text" style="color:#000000;">, vol. 8, no. 1, pp. 1–10, 2021.
</span>
</span>
</li>
<li id="bib.bib158" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[158]</span>
<span class="ltx_bibblock"><span id="bib.bib158.1.1" class="ltx_text" style="color:#000000;">
T. Bernecker, A. Peters, C. L. Schlett, F. Bamberg, F. Theis, D. Rueckert, J. Weiß, and S. Albarqouni, “Fednorm: Modality-based normalization in federated learning for multi-modal liver segmentation,” </span><em id="bib.bib158.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2205.11096</em><span id="bib.bib158.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib159" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[159]</span>
<span class="ltx_bibblock"><span id="bib.bib159.1.1" class="ltx_text" style="color:#000000;">
X. Xu and P. Yan, “Federated multi-organ segmentation with partially labeled data,” </span><em id="bib.bib159.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2206.07156</em><span id="bib.bib159.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib160" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[160]</span>
<span class="ltx_bibblock"><span id="bib.bib160.1.1" class="ltx_text" style="color:#000000;">
M. Hidalgo, “Pancreatic cancer,” </span><em id="bib.bib160.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">New England Journal of Medicine</em><span id="bib.bib160.3.3" class="ltx_text" style="color:#000000;">, vol. 362, no. 17, pp. 1605–1617, 2010.
</span>
</span>
</li>
<li id="bib.bib161" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[161]</span>
<span class="ltx_bibblock"><span id="bib.bib161.1.1" class="ltx_text" style="color:#000000;">
P. Rawla, T. Sunkara, and V. Gaduputi, “Epidemiology of pancreatic cancer: global trends, etiology and risk factors,” </span><em id="bib.bib161.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">World journal of oncology</em><span id="bib.bib161.3.3" class="ltx_text" style="color:#000000;">, vol. 10, no. 1, p. 10, 2019.
</span>
</span>
</li>
<li id="bib.bib162" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[162]</span>
<span class="ltx_bibblock"><span id="bib.bib162.1.1" class="ltx_text" style="color:#000000;">
C. Shen, P. Wang, H. R. Roth, D. Yang, D. Xu, M. Oda, W. Wang, C.-S. Fuh, P.-T. Chen, K.-L. Liu, </span><em id="bib.bib162.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib162.3.3" class="ltx_text" style="color:#000000;">, “Multi-task federated learning for heterogeneous pancreas segmentation,” in </span><em id="bib.bib162.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Clinical Image-Based Procedures, Distributed and Collaborative Learning, Artificial Intelligence for Combating COVID-19 and Secure and Privacy-Preserving Machine Learning</em><span id="bib.bib162.5.5" class="ltx_text" style="color:#000000;">, 2021, pp. 101–110.
</span>
</span>
</li>
<li id="bib.bib163" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[163]</span>
<span class="ltx_bibblock"><span id="bib.bib163.1.1" class="ltx_text" style="color:#000000;">
P. Wang, C. Shen, H. R. Roth, D. Yang, D. Xu, M. Oda, K. Misawa, P.-T. Chen, K.-L. Liu, W.-C. Liao, </span><em id="bib.bib163.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib163.3.3" class="ltx_text" style="color:#000000;">, “Automated pancreas segmentation using multi-institutional collaborative deep learning,” in </span><em id="bib.bib163.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Domain Adaptation and Representation Transfer, and Distributed and Collaborative Learning</em><span id="bib.bib163.5.5" class="ltx_text" style="color:#000000;">, 2020, pp. 192–200.
</span>
</span>
</li>
<li id="bib.bib164" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[164]</span>
<span class="ltx_bibblock"><span id="bib.bib164.1.1" class="ltx_text" style="color:#000000;">
Q. Yu, D. Yang, H. Roth, Y. Bai, Y. Zhang, A. L. Yuille, and D. Xu, “C2fnas: Coarse-to-fine neural architecture search for 3d medical image segmentation,” in </span><em id="bib.bib164.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition</em><span id="bib.bib164.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 4126–4135.
</span>
</span>
</li>
<li id="bib.bib165" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[165]</span>
<span class="ltx_bibblock"><span id="bib.bib165.1.1" class="ltx_text" style="color:#000000;">
A. Myronenko, “3d mri brain tumor segmentation using autoencoder regularization,” in </span><em id="bib.bib165.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International MICCAI Brainlesion Workshop</em><span id="bib.bib165.3.3" class="ltx_text" style="color:#000000;">, 2018, pp. 311–320.
</span>
</span>
</li>
<li id="bib.bib166" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[166]</span>
<span class="ltx_bibblock"><span id="bib.bib166.1.1" class="ltx_text" style="color:#000000;">
W. P. Smith, M. Kim, C. Holdsworth, J. Liao, and M. H. Phillips, “Personalized treatment planning with a model of radiation therapy outcomes for use in multiobjective optimization of imrt plans for prostate cancer,” </span><em id="bib.bib166.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Radiation Oncology</em><span id="bib.bib166.3.3" class="ltx_text" style="color:#000000;">, vol. 11, no. 1, pp. 1–14, 2016.
</span>
</span>
</li>
<li id="bib.bib167" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[167]</span>
<span class="ltx_bibblock"><span id="bib.bib167.1.1" class="ltx_text" style="color:#000000;">
S. G. Armato III, H. Huisman, K. Drukker, L. Hadjiiski, J. S. Kirby, N. Petrick, G. Redmond, M. L. Giger, K. Cha, A. Mamonov, </span><em id="bib.bib167.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib167.3.3" class="ltx_text" style="color:#000000;">, “Prostatex challenges for computerized classification of prostate lesions from multiparametric magnetic resonance images,” </span><em id="bib.bib167.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of Medical Imaging</em><span id="bib.bib167.5.5" class="ltx_text" style="color:#000000;">, vol. 5, no. 4, pp. 044 501–044 501, 2018.
</span>
</span>
</li>
<li id="bib.bib168" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[168]</span>
<span class="ltx_bibblock"><span id="bib.bib168.1.1" class="ltx_text" style="color:#000000;">
H. Ludwig, N. Baracaldo, G. Thomas, Y. Zhou, A. Anwar, S. Rajamoni, Y. Ong, J. Radhakrishnan, A. Verma, M. Sinn, </span><em id="bib.bib168.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib168.3.3" class="ltx_text" style="color:#000000;">, “Ibm federated learning: an enterprise framework white paper v0. 1,” </span><em id="bib.bib168.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2007.10987</em><span id="bib.bib168.5.5" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib169" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[169]</span>
<span class="ltx_bibblock"><span id="bib.bib169.1.1" class="ltx_text" style="color:#000000;">
“An application framework optimized for healthcare and life sciences developers,” Dec 2019. [Online]. Available: </span><span class="ltx_ref ltx_nolink ltx_url ltx_ref_self" style="color:#000000;">https://developer.nvidia.com/clara</span><span id="bib.bib169.2.2" class="ltx_text" style="color:#000000;">
</span>
</span>
</li>
<li id="bib.bib170" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[170]</span>
<span class="ltx_bibblock"><span id="bib.bib170.1.1" class="ltx_text" style="color:#000000;">
M. N. Galtier and C. Marini, “Substra: a framework for privacy-preserving, traceable and collaborative machine learning,” </span><em id="bib.bib170.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:1910.11567</em><span id="bib.bib170.3.3" class="ltx_text" style="color:#000000;">, 2019.
</span>
</span>
</li>
<li id="bib.bib171" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[171]</span>
<span class="ltx_bibblock"><span id="bib.bib171.1.1" class="ltx_text" style="color:#000000;">
N. Rodríguez-Barroso, G. Stipcich, D. Jiménez-López, J. A. Ruiz-Millán, E. Martínez-Cámara, G. González-Seco, M. V. Luzón, M. A. Veganzones, and F. Herrera, “Federated learning and differential privacy: Software tools analysis, the sherpa. ai fl framework and methodological guidelines for preserving data privacy,” </span><em id="bib.bib171.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Information Fusion</em><span id="bib.bib171.3.3" class="ltx_text" style="color:#000000;">, vol. 64, pp. 270–292, 2020.
</span>
</span>
</li>
<li id="bib.bib172" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[172]</span>
<span class="ltx_bibblock"><span id="bib.bib172.1.1" class="ltx_text" style="color:#000000;">
“An industrial grade federated learning framework,” Oct 2019. [Online]. Available: </span><span class="ltx_ref ltx_nolink ltx_url ltx_ref_self" style="color:#000000;">https://fate.fedai.org/</span><span id="bib.bib172.2.2" class="ltx_text" style="color:#000000;">
</span>
</span>
</li>
<li id="bib.bib173" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[173]</span>
<span class="ltx_bibblock"><span id="bib.bib173.1.1" class="ltx_text" style="color:#000000;">
“Tensorflow federated: Machine learning on decentralized data,” Feb 2019. [Online]. Available: </span><span class="ltx_ref ltx_nolink ltx_url ltx_ref_self" style="color:#000000;">https://www.tensorflow.org/federated</span><span id="bib.bib173.2.2" class="ltx_text" style="color:#000000;">
</span>
</span>
</li>
<li id="bib.bib174" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[174]</span>
<span class="ltx_bibblock"><span id="bib.bib174.1.1" class="ltx_text" style="color:#000000;">
G. A. Reina, A. Gruzdev, P. Foley, O. Perepelkina, M. Sharma, I. Davidyuk, I. Trushkin, M. Radionov, A. Mokrov, D. Agapov, </span><em id="bib.bib174.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib174.3.3" class="ltx_text" style="color:#000000;">, “Openfl: An open-source framework for federated learning,” </span><em id="bib.bib174.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2105.06413</em><span id="bib.bib174.5.5" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib175" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[175]</span>
<span class="ltx_bibblock"><span id="bib.bib175.1.1" class="ltx_text" style="color:#000000;">
S. Silva, A. Altmann, B. Gutman, and M. Lorenzi, “Fed-biomed: A general open-source frontend framework for federated learning in healthcare,” in </span><em id="bib.bib175.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Domain Adaptation and Representation Transfer, and Distributed and Collaborative Learning</em><span id="bib.bib175.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 201–210.
</span>
</span>
</li>
<li id="bib.bib176" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[176]</span>
<span class="ltx_bibblock"><span id="bib.bib176.1.1" class="ltx_text" style="color:#000000;">
“Pysyft,” Jan 2020. [Online]. Available: </span><span class="ltx_ref ltx_nolink ltx_url ltx_ref_self" style="color:#000000;">https://blog.openmined.org/tag/pysyft/</span><span id="bib.bib176.2.2" class="ltx_text" style="color:#000000;">
</span>
</span>
</li>
<li id="bib.bib177" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[177]</span>
<span class="ltx_bibblock"><span id="bib.bib177.1.1" class="ltx_text" style="color:#000000;">
C. He, S. Li, J. So, X. Zeng, M. Zhang, H. Wang, X. Wang, P. Vepakomma, A. Singh, H. Qiu, </span><em id="bib.bib177.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib177.3.3" class="ltx_text" style="color:#000000;">, “Fedml: A research library and benchmark for federated machine learning,” </span><em id="bib.bib177.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2007.13518</em><span id="bib.bib177.5.5" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib178" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[178]</span>
<span class="ltx_bibblock"><span id="bib.bib178.1.1" class="ltx_text" style="color:#000000;">
S. Caldas, S. M. K. Duddu, P. Wu, T. Li, J. Konečnỳ, H. B. McMahan, V. Smith, and A. Talwalkar, “Leaf: A benchmark for federated settings,” </span><em id="bib.bib178.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:1812.01097</em><span id="bib.bib178.3.3" class="ltx_text" style="color:#000000;">, 2018.
</span>
</span>
</li>
<li id="bib.bib179" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[179]</span>
<span class="ltx_bibblock"><span id="bib.bib179.1.1" class="ltx_text" style="color:#000000;">
“Paddlefl,” Aug 2016. [Online]. Available: </span><span class="ltx_ref ltx_nolink ltx_url ltx_ref_self" style="color:#000000;">https://github.com/paddlepaddle/paddle</span><span id="bib.bib179.2.2" class="ltx_text" style="color:#000000;">
</span>
</span>
</li>
<li id="bib.bib180" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[180]</span>
<span class="ltx_bibblock"><span id="bib.bib180.1.1" class="ltx_text" style="color:#000000;">
V. Mugunthan, A. Peraire-Bueno, and L. Kagal, “Privacyfl: A simulator for privacy-preserving and secure federated learning,” in </span><em id="bib.bib180.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the 29th ACM International Conference on Information &amp; Knowledge Management</em><span id="bib.bib180.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 3085–3092.
</span>
</span>
</li>
<li id="bib.bib181" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[181]</span>
<span class="ltx_bibblock"><span id="bib.bib181.1.1" class="ltx_text" style="color:#000000;">
T. Li, A. K. Sahu, A. Talwalkar, and V. Smith, “Federated learning: Challenges, methods, and future directions,” </span><em id="bib.bib181.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE signal processing magazine</em><span id="bib.bib181.3.3" class="ltx_text" style="color:#000000;">, vol. 37, no. 3, pp. 50–60, 2020.
</span>
</span>
</li>
<li id="bib.bib182" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[182]</span>
<span class="ltx_bibblock"><span id="bib.bib182.1.1" class="ltx_text" style="color:#000000;">
S. Zheng, C. Shen, and X. Chen, “Design and analysis of uplink and downlink communications for federated learning,” </span><em id="bib.bib182.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Journal on Selected Areas in Communications</em><span id="bib.bib182.3.3" class="ltx_text" style="color:#000000;">, pp. 2150–2167, 2020.
</span>
</span>
</li>
<li id="bib.bib183" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[183]</span>
<span class="ltx_bibblock"><span id="bib.bib183.1.1" class="ltx_text" style="color:#000000;">
T. K. Dang, X. Lan, J. Weng, and M. Feng, “Federated learning for electronic health records,” </span><em id="bib.bib183.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">ACM Transactions on Intelligent Systems and Technology (TIST)</em><span id="bib.bib183.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib184" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[184]</span>
<span class="ltx_bibblock"><span id="bib.bib184.1.1" class="ltx_text" style="color:#000000;">
J. Ding and B. Ding, “Interval privacy: A framework for privacy-preserving data collection,” </span><em id="bib.bib184.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Signal Processing</em><span id="bib.bib184.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib185" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[185]</span>
<span class="ltx_bibblock"><span id="bib.bib185.1.1" class="ltx_text" style="color:#000000;">
E. Peltonen, M. Bennis, M. Capobianco, M. Debbah, A. Ding, F. Gil-Castiñeira, M. Jurmu, T. Karvonen, M. Kelanti, A. Kliks, </span><em id="bib.bib185.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">et al.</em><span id="bib.bib185.3.3" class="ltx_text" style="color:#000000;">, “6g white paper on edge intelligence,” </span><em id="bib.bib185.4.4" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2004.14850</em><span id="bib.bib185.5.5" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib186" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[186]</span>
<span class="ltx_bibblock"><span id="bib.bib186.1.1" class="ltx_text" style="color:#000000;">
S. K. Lo, Q. Lu, L. Zhu, H.-y. Paik, X. Xu, and C. Wang, “Architectural patterns for the design of federated learning systems,” </span><em id="bib.bib186.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of Systems and Software</em><span id="bib.bib186.3.3" class="ltx_text" style="color:#000000;">, p. 111357, 2022.
</span>
</span>
</li>
<li id="bib.bib187" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[187]</span>
<span class="ltx_bibblock"><span id="bib.bib187.1.1" class="ltx_text" style="color:#000000;">
H. Diddee and B. Kansra, “Crosspriv: User privacy preservation model for cross-silo federated software,” in </span><em id="bib.bib187.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the IEEE/ACM International Conference on Automated Software Engineering (ASE)</em><span id="bib.bib187.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 1370–1372.
</span>
</span>
</li>
<li id="bib.bib188" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[188]</span>
<span class="ltx_bibblock"><span id="bib.bib188.1.1" class="ltx_text" style="color:#000000;">
T. Nguyen, M. Dakka, S. Diakiw, M. VerMilyea, M. Perugini, J. Hall, and D. Perugini, “A novel decentralized federated learning approach to train on globally distributed, poor quality, and protected private medical data,” </span><em id="bib.bib188.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Scientific Reports</em><span id="bib.bib188.3.3" class="ltx_text" style="color:#000000;">, vol. 12, no. 1, pp. 1–12, 2022.
</span>
</span>
</li>
<li id="bib.bib189" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[189]</span>
<span class="ltx_bibblock"><span id="bib.bib189.1.1" class="ltx_text" style="color:#000000;">
J. S.-P. Díaz and Á. L. García, “Study of the performance and scalability of federated learning for medical imaging with intermittent clients,” </span><em id="bib.bib189.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Neurocomputing</em><span id="bib.bib189.3.3" class="ltx_text" style="color:#000000;">, vol. 518, pp. 142–154, 2023.
</span>
</span>
</li>
<li id="bib.bib190" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[190]</span>
<span class="ltx_bibblock"><span id="bib.bib190.1.1" class="ltx_text" style="color:#000000;">
J. L. Fernández-Alemán, I. C. Señor, P. Á. O. Lozoya, and A. Toval, “Security and privacy in electronic health records: A systematic literature review,” </span><em id="bib.bib190.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of biomedical informatics</em><span id="bib.bib190.3.3" class="ltx_text" style="color:#000000;">, vol. 46, no. 3, pp. 541–562, 2013.
</span>
</span>
</li>
<li id="bib.bib191" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[191]</span>
<span class="ltx_bibblock"><span id="bib.bib191.1.1" class="ltx_text" style="color:#000000;">
R. Wu, G.-J. Ahn, and H. Hu, “Towards HIPAA-compliant healthcare systems,” in </span><em id="bib.bib191.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the 2nd ACM SIGHIT International Health Informatics Symposium</em><span id="bib.bib191.3.3" class="ltx_text" style="color:#000000;">, 2012, pp. 593–602.
</span>
</span>
</li>
<li id="bib.bib192" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[192]</span>
<span class="ltx_bibblock"><span id="bib.bib192.1.1" class="ltx_text" style="color:#000000;">
G. J. Annas, “Hipaa regulations: a new era of medical-record privacy?” </span><em id="bib.bib192.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">New England Journal of Medicine</em><span id="bib.bib192.3.3" class="ltx_text" style="color:#000000;">, vol. 348, p. 1486, 2003.
</span>
</span>
</li>
<li id="bib.bib193" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[193]</span>
<span class="ltx_bibblock"><span id="bib.bib193.1.1" class="ltx_text" style="color:#000000;">
Y. B. Choi, K. E. Capitan, J. S. Krause, and M. M. Streeper, “Challenges associated with privacy in health care industry: implementation of HIPAA and the security rules,” </span><em id="bib.bib193.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of medical systems</em><span id="bib.bib193.3.3" class="ltx_text" style="color:#000000;">, vol. 30, pp. 57–64, 2006.
</span>
</span>
</li>
<li id="bib.bib194" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[194]</span>
<span class="ltx_bibblock"><span id="bib.bib194.1.1" class="ltx_text" style="color:#000000;">
M. Fredrikson, S. Jha, and T. Ristenpart, “Model inversion attacks that exploit confidence information and basic countermeasures,” in </span><em id="bib.bib194.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the ACM SIGSAC conference on computer and communications security</em><span id="bib.bib194.3.3" class="ltx_text" style="color:#000000;">, 2015, pp. 1322–1333.
</span>
</span>
</li>
<li id="bib.bib195" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[195]</span>
<span class="ltx_bibblock"><span id="bib.bib195.1.1" class="ltx_text" style="color:#000000;">
M. Fredrikson, E. Lantz, S. Jha, S. Lin, D. Page, and T. Ristenpart, “Privacy in Pharmacogenetics: An End-to-End Case Study of Personalized Warfarin Dosing,” in </span><em id="bib.bib195.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the USENIX Security Symposium (USENIX Security 14)</em><span id="bib.bib195.3.3" class="ltx_text" style="color:#000000;">, 2014, pp. 17–32.
</span>
</span>
</li>
<li id="bib.bib196" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[196]</span>
<span class="ltx_bibblock"><span id="bib.bib196.1.1" class="ltx_text" style="color:#000000;">
E. Bagdasaryan, A. Veit, Y. Hua, D. Estrin, and V. Shmatikov, “How to backdoor federated learning,” in </span><em id="bib.bib196.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Artificial Intelligence and Statistics</em><span id="bib.bib196.3.3" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib197" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[197]</span>
<span class="ltx_bibblock"><span id="bib.bib197.1.1" class="ltx_text" style="color:#000000;">
R. C. Geyer, T. Klein, and M. Nabi, “Differentially private federated learning: A client level perspective,” </span><em id="bib.bib197.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:1712.07557</em><span id="bib.bib197.3.3" class="ltx_text" style="color:#000000;">, 2017.
</span>
</span>
</li>
<li id="bib.bib198" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[198]</span>
<span class="ltx_bibblock"><span id="bib.bib198.1.1" class="ltx_text" style="color:#000000;">
K. Jayaram, A. Verma, A. Verma, G. Thomas, and C. Sutcher-Shepard, “Mystiko: Cloud-mediated, private, federated gradient descent,” in </span><em id="bib.bib198.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Cloud Computing (CLOUD)</em><span id="bib.bib198.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 201–210.
</span>
</span>
</li>
<li id="bib.bib199" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[199]</span>
<span class="ltx_bibblock"><span id="bib.bib199.1.1" class="ltx_text" style="color:#000000;">
V. Mothukuri, R. M. Parizi, S. Pouriyeh, Y. Huang, A. Dehghantanha, and G. Srivastava, “A survey on security and privacy of federated learning,” </span><em id="bib.bib199.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Future Generation Computer Systems</em><span id="bib.bib199.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib200" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[200]</span>
<span class="ltx_bibblock"><span id="bib.bib200.1.1" class="ltx_text" style="color:#000000;">
Y. Deng, F. Lyu, J. Ren, H. Wu, Y. Zhou, Y. Zhang, and X. Shen, “Auction: Automated and quality-aware client selection framework for efficient federated learning,” </span><em id="bib.bib200.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Transactions on Parallel and Distributed Systems</em><span id="bib.bib200.3.3" class="ltx_text" style="color:#000000;">, vol. 33, no. 8, pp. 1996–2009, 2021.
</span>
</span>
</li>
<li id="bib.bib201" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[201]</span>
<span class="ltx_bibblock"><span id="bib.bib201.1.1" class="ltx_text" style="color:#000000;">
A. N. Bhagoji, S. Chakraborty, P. Mittal, and S. Calo, “Analyzing federated learning through an adversarial lens,” in </span><em id="bib.bib201.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the International Conference on Machine Learning</em><span id="bib.bib201.3.3" class="ltx_text" style="color:#000000;">, 2019, pp. 634–643.
</span>
</span>
</li>
<li id="bib.bib202" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[202]</span>
<span class="ltx_bibblock"><span id="bib.bib202.1.1" class="ltx_text" style="color:#000000;">
M. Malekzadeh, B. Hasircioglu, N. Mital, K. Katarya, M. Ozfatura, and D. Gündüz, “Dopamine: Differentially private secure federated learning on medical data,” in </span><em id="bib.bib202.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the Second AAAI Workshop on Privacy-Preserving Artificial Intelligence, Virtual Worskhop</em><span id="bib.bib202.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib203" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[203]</span>
<span class="ltx_bibblock"><span id="bib.bib203.1.1" class="ltx_text" style="color:#000000;">
G. Paragliola and A. Coronato, “Definition of a novel federated learning approach to reduce communication costs,” </span><em id="bib.bib203.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Expert Systems with Applications</em><span id="bib.bib203.3.3" class="ltx_text" style="color:#000000;">, vol. 189, p. 116109, 2022.
</span>
</span>
</li>
<li id="bib.bib204" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[204]</span>
<span class="ltx_bibblock"><span id="bib.bib204.1.1" class="ltx_text" style="color:#000000;">
L. Qu, N. Balachandar, and D. L. Rubin, “An experimental study of data heterogeneity in federated learning methods for medical imaging,” </span><em id="bib.bib204.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2107.08371</em><span id="bib.bib204.3.3" class="ltx_text" style="color:#000000;">, 2021.
</span>
</span>
</li>
<li id="bib.bib205" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[205]</span>
<span class="ltx_bibblock"><span id="bib.bib205.1.1" class="ltx_text" style="color:#000000;">
H. Yu, Z. Liu, Y. Liu, T. Chen, M. Cong, X. Weng, D. Niyato, and Q. Yang, “A fairness-aware incentive scheme for federated learning,” in </span><em id="bib.bib205.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the AAAI/ACM Conference on AI, Ethics, and Society</em><span id="bib.bib205.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 393–399.
</span>
</span>
</li>
<li id="bib.bib206" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[206]</span>
<span class="ltx_bibblock"><span id="bib.bib206.1.1" class="ltx_text" style="color:#000000;">
J. Kang, Z. Xiong, D. Niyato, S. Xie, and J. Zhang, “Incentive mechanism for reliable federated learning: A joint optimization approach to combining reputation and contract theory,” </span><em id="bib.bib206.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">IEEE Internet of Things Journal</em><span id="bib.bib206.3.3" class="ltx_text" style="color:#000000;">, vol. 6, no. 6, pp. 10 700–10 714, 2019.
</span>
</span>
</li>
<li id="bib.bib207" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[207]</span>
<span class="ltx_bibblock"><span id="bib.bib207.1.1" class="ltx_text" style="color:#000000;">
A. Fallah, A. Mokhtari, and A. Ozdaglar, “Personalized federated learning with theoretical guarantees: A model-agnostic meta-learning approach,” </span><em id="bib.bib207.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Advances in Neural Information Processing Systems</em><span id="bib.bib207.3.3" class="ltx_text" style="color:#000000;">, vol. 33, pp. 3557–3568, 2020.
</span>
</span>
</li>
<li id="bib.bib208" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[208]</span>
<span class="ltx_bibblock"><span id="bib.bib208.1.1" class="ltx_text" style="color:#000000;">
H. Kassem, D. Alapatt, P. Mascagni, A. Consortium, A. Karargyris, and N. Padoy, “Federated cycling (fedcy): Semi-supervised federated learning of surgical phases,” </span><em id="bib.bib208.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2203.07345</em><span id="bib.bib208.3.3" class="ltx_text" style="color:#000000;">, 2022.
</span>
</span>
</li>
<li id="bib.bib209" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[209]</span>
<span class="ltx_bibblock"><span id="bib.bib209.1.1" class="ltx_text" style="color:#000000;">
S. Itahara, T. Nishio, Y. Koda, M. Morikura, and K. Yamamoto, “Distillation-based semi-supervised federated learning for communication-efficient collaborative training with non-iid private data,” </span><em id="bib.bib209.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:2008.06180</em><span id="bib.bib209.3.3" class="ltx_text" style="color:#000000;">, 2020.
</span>
</span>
</li>
<li id="bib.bib210" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[210]</span>
<span class="ltx_bibblock"><span id="bib.bib210.1.1" class="ltx_text" style="color:#000000;">
N. Görnitz, A. Porbadnigk, A. Binder, C. Sannelli, M. Braun, K.-R. Müller, and M. Kloft, “Learning and evaluation in presence of non-iid label noise,” in </span><em id="bib.bib210.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Artificial Intelligence and Statistics</em><span id="bib.bib210.3.3" class="ltx_text" style="color:#000000;">, 2014, pp. 293–302.
</span>
</span>
</li>
<li id="bib.bib211" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[211]</span>
<span class="ltx_bibblock"><span id="bib.bib211.1.1" class="ltx_text" style="color:#000000;">
H. Zhu, J. Xu, S. Liu, and Y. Jin, “Federated learning on non-iid data: A survey,” </span><em id="bib.bib211.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Neurocomputing</em><span id="bib.bib211.3.3" class="ltx_text" style="color:#000000;">, vol. 465, pp. 371–390, 2021.
</span>
</span>
</li>
<li id="bib.bib212" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[212]</span>
<span class="ltx_bibblock"><span id="bib.bib212.1.1" class="ltx_text" style="color:#000000;">
Q. Li, B. He, and D. Song, “Model-contrastive federated learning,” in </span><em id="bib.bib212.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition</em><span id="bib.bib212.3.3" class="ltx_text" style="color:#000000;">, 2021, pp. 10 713–10 722.
</span>
</span>
</li>
<li id="bib.bib213" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[213]</span>
<span class="ltx_bibblock"><span id="bib.bib213.1.1" class="ltx_text" style="color:#000000;">
K. He, H. Fan, Y. Wu, S. Xie, and R. Girshick, “Momentum contrast for unsupervised visual representation learning,” in </span><em id="bib.bib213.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proceedings of the IEEE/CVF conference on computer vision and pattern recognition</em><span id="bib.bib213.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 9729–9738.
</span>
</span>
</li>
<li id="bib.bib214" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[214]</span>
<span class="ltx_bibblock"><span id="bib.bib214.1.1" class="ltx_text" style="color:#000000;">
T. Chen, S. Kornblith, M. Norouzi, and G. Hinton, “A simple framework for contrastive learning of visual representations,” in </span><em id="bib.bib214.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Proc. of the Intl. conference on machine learning</em><span id="bib.bib214.3.3" class="ltx_text" style="color:#000000;">, 2020, pp. 1597–1607.
</span>
</span>
</li>
<li id="bib.bib215" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[215]</span>
<span class="ltx_bibblock"><span id="bib.bib215.1.1" class="ltx_text" style="color:#000000;">
W. Samek, T. Wiegand, and K.-R. Müller, “Explainable artificial intelligence: Understanding, visualizing and interpreting deep learning models,” </span><em id="bib.bib215.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:1708.08296</em><span id="bib.bib215.3.3" class="ltx_text" style="color:#000000;">, 2017.
</span>
</span>
</li>
<li id="bib.bib216" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[216]</span>
<span class="ltx_bibblock"><span id="bib.bib216.1.1" class="ltx_text" style="color:#000000;">
P. Chen, X. Du, Z. Lu, J. Wu, and P. C. Hung, “Evfl: An explainable vertical federated learning for data-oriented artificial intelligence systems,” </span><em id="bib.bib216.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">Journal of Systems Architecture</em><span id="bib.bib216.3.3" class="ltx_text" style="color:#000000;">, vol. 126, p. 102474, 2022.
</span>
</span>
</li>
<li id="bib.bib217" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[217]</span>
<span class="ltx_bibblock"><span id="bib.bib217.1.1" class="ltx_text" style="color:#000000;">
G. Wang, “Interpret federated learning with shapley values,” </span><em id="bib.bib217.2.2" class="ltx_emph ltx_font_italic" style="color:#000000;">arXiv preprint arXiv:1905.04519</em><span id="bib.bib217.3.3" class="ltx_text" style="color:#000000;">, 2019.
</span>
</span>
</li>
</ul>
</section>
<figure id="id1" class="ltx_float biography">
<table id="id1.1" class="ltx_tabular">
<tr id="id1.1.1" class="ltx_tr">
<td id="id1.1.1.1" class="ltx_td"><img src="/html/2208.03392/assets/images/Ashish_R.jpeg" id="id1.1.1.1.g1" class="ltx_graphics ltx_img_portrait" width="97" height="125" alt="[Uncaptioned image]"></td>
<td id="id1.1.1.2" class="ltx_td">
<span id="id1.1.1.2.1" class="ltx_inline-block">
<span id="id1.1.1.2.1.1" class="ltx_p"><span id="id1.1.1.2.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">ASHISH RAUNIYAR</span><span id="id1.1.1.2.1.1.2" class="ltx_text" style="color:#000000;"> 
is currently working as a Research Scientist at SINTEF Digital, Norway.
He received a Ph.D. Degree in Computer Science from the University of Oslo, Norway in 2021. He was a graduate research assistant at Wireless Emerging Networking System (WENS) Lab, where he completed his Master’s degree in IT Convergence Engineering at Kumoh National Institute of Technology, South Korea. He is a recipient of Best Paper Awards at the 2020 IEEE 43rd International Conference on Telecommunications and Signal Processing (TSP), Milan, Italy, 28th IEEE International Telecommunication Networks and Applications Conference (ITNAC), 2018, Sydney, Australia, and AI-DLDA 2018 International Summer School on Artificial Intelligence, Udine-Italy, 2018. He was also selected as “Top 200 Young Researchers in Computer Science &amp; Mathematics” and invited to attend Heidelberg Laureate Forum, Heidelberg, Germany in 2017, and Global Young Scientist Summit, Singapore in 2020. He also won the European Satellite Navigation Competition (ESNC) in 2017. His main research interest includes 5G/6G Signal Processing, Autonomous Systems and Networks, Internet of Things, Machine Learning, Wireless Communications, and Computer Networking.</span></span>
</span>
</td>
</tr>
</table>
</figure>
<figure id="id2" class="ltx_float biography">
<table id="id2.1" class="ltx_tabular">
<tr id="id2.1.1" class="ltx_tr">
<td id="id2.1.1.1" class="ltx_td"><img src="/html/2208.03392/assets/images/desta_photo.jpeg" id="id2.1.1.1.g1" class="ltx_graphics ltx_img_portrait" width="92" height="125" alt="[Uncaptioned image]"></td>
<td id="id2.1.1.2" class="ltx_td">
<span id="id2.1.1.2.1" class="ltx_inline-block">
<span id="id2.1.1.2.1.1" class="ltx_p"><span id="id2.1.1.2.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">DESTA HAILESELASSIE HAGOS</span><span id="id2.1.1.2.1.1.2" class="ltx_text" style="color:#000000;"> 
received a Ph.D. degree in Computer Science from the University of Oslo, Faculty of Mathematics and Natural Sciences, Norway, in April 2020. Currently, he is a Postdoctoral Research Fellow at the DoD Center of Excellence in Artificial Intelligence and Machine Learning (CoE-AIML), College of Engineering and Architecture (CEA), Department of Electrical Engineering and Computer Science at Howard University, Washington DC, USA. Previously, he was a Postdoctoral Research Fellow at the Division of Software and Computer Systems (SCS), Department of Computer Science, School of Electrical Engineering and Computer Science (EECS), KTH Royal Institute of Technology, Stockholm, Sweden, working on the H2020-EU project, ExtremeEarth: From Copernicus Big Data to Extreme Earth Analytics. He received his B.Sc. degree in Computer Science from Mekelle University, Department of Computer Science, Mekelle, Tigray, in 2008. He obtained his M.Sc. degree in Computer Science and Engineering specializing in Mobile Systems from Luleå University of Technology, Department of Computer Science Electrical and Space Engineering, Sweden, in June 2012. His current research interests are in the areas of Machine Learning, Deep Learning, and Artificial Intelligence.</span></span>
</span>
</td>
</tr>
</table>
</figure>
<figure id="id3" class="ltx_float biography">
<table id="id3.1" class="ltx_tabular">
<tr id="id3.1.1" class="ltx_tr">
<td id="id3.1.1.1" class="ltx_td"><img src="/html/2208.03392/assets/images/Debesh.jpg" id="id3.1.1.1.g1" class="ltx_graphics ltx_img_square" width="100" height="100" alt="[Uncaptioned image]"></td>
<td id="id3.1.1.2" class="ltx_td">
<span id="id3.1.1.2.1" class="ltx_inline-block">
<span id="id3.1.1.2.1.1" class="ltx_p"><span id="id3.1.1.2.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">DEBESH JHA</span><span id="id3.1.1.2.1.1.2" class="ltx_text" style="color:#000000;">  is currently working as a Research Associate at the Machine &amp; Hybrid Intelligence Lab, Department of Radiology, Northwestern University. He received a Ph.D. in Computer Science from the UiT The Arctic University of Norway in 2022. Previously, he worked as a researcher at UiT The Arctic University of Norway, Tromsø, Norway. During his Ph.D., he was also affiliated with Simula Research Laboratory, Oslo, Norway, and Simula Metropolitan Center for Digital Engineering, Oslo, Norway. He was a graduate research assistant at the Digital Media Computing (DMC) Lab, where he completed his Master’s Degree in Information and Communication Engineering from the Chosun University, Republic of Korea. He is a recipient of the First-ever Paper with Code Contributor Award in 2022. He is also the recipient of the Best Paper Award at the International Conference on Electronics, Information, and Communication (ICEIC 2018), Hawaii, USA, and the Best Student Paper Award finalist at the IEEE 33rd International Symposium on Computer Based Medical Systems (CBMS 2020). His research interest includes Medical Image Analysis, Computer-aided Diagnosis and Detection, Machine Learning, Deep Learning, Computer Vision, and their application in Biomedical and Clinical Imaging.</span></span>
</span>
</td>
</tr>
</table>
</figure>
<figure id="id4" class="ltx_float biography">
<table id="id4.1" class="ltx_tabular">
<tr id="id4.1.1" class="ltx_tr">
<td id="id4.1.1.1" class="ltx_td"><img src="/html/2208.03392/assets/images/Jan.jpeg" id="id4.1.1.1.g1" class="ltx_graphics ltx_img_square" width="110" height="107" alt="[Uncaptioned image]"></td>
<td id="id4.1.1.2" class="ltx_td">
<span id="id4.1.1.2.1" class="ltx_inline-block">
<span id="id4.1.1.2.1.1" class="ltx_p"><span id="id4.1.1.2.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">JAN ERIK HÅKEGÅRD</span><span id="id4.1.1.2.1.1.2" class="ltx_text" style="color:#000000;"> 
received the degree Sivilingeniør (M.Sc.) in 1990 from the Department of Electronical Engineering and Informatics, The Norwegian Institute of Technology (NTH), Trondheim, Norway. In 1997, he received a Docteur (Ph.D.) degree in Electronics and Communications at ENST, site de Toulouse, France. Since 1997, he has been with SINTEF Digital, working on research and development projects related to various types of Wireless Communication Systems. He is currently leading SINTEF activities within the Satellite and Terrestrial Communication Systems.</span></span>
</span>
</td>
</tr>
</table>
</figure>
<figure id="id5" class="ltx_float biography">
<table id="id5.1" class="ltx_tabular">
<tr id="id5.1.1" class="ltx_tr">
<td id="id5.1.1.1" class="ltx_td"><img src="/html/2208.03392/assets/images/ulas.png" id="id5.1.1.1.g1" class="ltx_graphics ltx_img_square" width="110" height="113" alt="[Uncaptioned image]"></td>
<td id="id5.1.1.2" class="ltx_td">
<span id="id5.1.1.2.1" class="ltx_inline-block">
<span id="id5.1.1.2.1.1" class="ltx_p"><span id="id5.1.1.2.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">ULAS BAGCI, Ph.D.,</span><span id="id5.1.1.2.1.1.2" class="ltx_text" style="color:#000000;">  is an Associate Professor at the Northwestern University’s Radiology, BME, and ECE departments, and He holds a courtesy professorship at the Center for Research in Computer Vision (CRCV), University of Central Florida (UCF). His research interests are Artificial Intelligence, Machine Learning, and their applications in Biomedical and Clinical Imaging. Dr. Bagci has more than 250 peer-reviewed articles on these topics. Previously, he was a staff scientist and lab co-manager at the National Institutes of Health’s radiology and imaging sciences department, center for infectious disease imaging. Dr. Bagci holds NIH grants and serves as a steering committee member of AIR (artificial intelligence resource) at the NIH. Dr. Bagci also serves as an area chair for MICCAI for several years and he is an associate editor of top-tier journals in his fields such as IEEE Transactions on Medical Imaging, Medical Physics, and Medical Image Analysis. He has several international and national recognitions including Best Paper and Reviewer Awards.</span></span>
</span>
</td>
</tr>
</table>
</figure>
<figure id="id6" class="ltx_float biography">
<table id="id6.1" class="ltx_tabular">
<tr id="id6.1.1" class="ltx_tr">
<td id="id6.1.1.1" class="ltx_td"><img src="/html/2208.03392/assets/images/Danda_Rawat.png" id="id6.1.1.1.g1" class="ltx_graphics ltx_img_square" width="100" height="106" alt="[Uncaptioned image]"></td>
<td id="id6.1.1.2" class="ltx_td">
<span id="id6.1.1.2.1" class="ltx_inline-block">
<span id="id6.1.1.2.1.1" class="ltx_p"><span id="id6.1.1.2.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">Dr. DANDA B. RAWAT</span><span id="id6.1.1.2.1.1.2" class="ltx_text" style="color:#000000;"> 
is the Associate Dean for Research &amp; Graduate Studies, a Full Professor in the Department of Electrical Engineering &amp; Computer Science (EECS), Founding Director of the Howard University Data Science &amp; Cybersecurity Center, Founding Director of the DoD Center of Excellence in Artificial Intelligence &amp; Machine Learning (CoE-AIML), Director of Cyber-security and Wireless Networking Innovations (CWiNs) Research Lab, Graduate Program Director of Howard CS Graduate Programs and Director of Graduate Cybersecurity Certificate Program at Howard University, Washington, DC, USA. Dr. Rawat is engaged in research and teaching in the areas of cybersecurity, machine learning, big data analytics, and wireless networking for emerging networked systems including cyber-physical systems (eHealth, energy, transportation), Internet-of-Things, multi-domain operations, smart cities, software-defined systems, and vehicular networks.</span></span>
</span>
</td>
</tr>
</table>
</figure>
<figure id="id7" class="ltx_float biography">
<table id="id7.1" class="ltx_tabular">
<tr id="id7.1.1" class="ltx_tr">
<td id="id7.1.1.1" class="ltx_td"><img src="/html/2208.03392/assets/images/vladimir_vlassov.jpeg" id="id7.1.1.1.g1" class="ltx_graphics ltx_img_square" width="100" height="100" alt="[Uncaptioned image]"></td>
<td id="id7.1.1.2" class="ltx_td">
<span id="id7.1.1.2.1" class="ltx_inline-block">
<span id="id7.1.1.2.1.1" class="ltx_p"><span id="id7.1.1.2.1.1.1" class="ltx_text ltx_font_bold" style="color:#000000;">VLADIMIR VLASSOV</span><span id="id7.1.1.2.1.1.2" class="ltx_text" style="color:#000000;"> 
is a professor in Computer Systems at the Department of Computer Science, School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology in Stockholm, Sweden. He was a visiting scientist with the Massachusetts Institute of Technology (1998) and at the University of Massachusetts Amherst (2004), USA. He has participated in a number of research projects funded by the European Commission, Swedish funding agencies, and the National Science Foundation (NSF) USA. He was one of the coordinators of the EMJD-DC Erasmus Mundus Joint Doctorate in Distributed Computing. Currently, he is a principal investigator from KTH in the H2020-EU project “ExtremeEarth: From Copernicus Big Data to Extreme Earth Analytics” (2018-2020). His research covers several areas in computer science, including Data-intensive Computing, Stream Processing, Scalable Distributed Deep Learning, Distributed Systems, Autonomic Computing, Cloud, and Edge Computing.</span></span>
</span>
</td>
</tr>
</table>
</figure>
<div class="ltx_pagination ltx_role_newpage"></div>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/2208.03391" class="ar5iv-nav-button ar5iv-nav-button-prev">◄</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/2208.03392" class="ar5iv-text-button ar5iv-severity-warning">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+2208.03392">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/2208.03392" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/2208.03393" class="ar5iv-nav-button ar5iv-nav-button-next">►</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Wed Mar 13 18:05:04 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "×";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
