<!DOCTYPE html>
<html lang="en">
<head>
<meta content="text/html; charset=utf-8" http-equiv="content-type"/>
<title>A review of Feature Selection on Knowledge Graphs</title>
<!--Generated on Fri Jun 21 04:49:06 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport"/>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv.0.7.9.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv-fonts.0.7.9.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/latexml_styles.css" rel="stylesheet" type="text/css"/>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/html2canvas/1.3.3/html2canvas.min.js"></script>
<script src="/static/browse/0.3.4/js/addons_new.js"></script>
<script src="/static/browse/0.3.4/js/feedbackOverlay.js"></script>
<base href="/html/2406.14864v1/"/></head>
<body>
<nav class="ltx_page_navbar">
<nav class="ltx_TOC">
<ol class="ltx_toclist">
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S1" title="In A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1 </span>Introduction</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S1.SS1" title="In 1 Introduction ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1.1 </span>Brief Introduction to Knowledge Graphs</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S1.SS2" title="In 1 Introduction ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1.2 </span>Importance of Feature Selection</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S1.SS3" title="In 1 Introduction ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1.3 </span>Overview of the Relationship between Knowledge Graphs and Feature Selection</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2" title="In A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2 </span>Background and Key Concepts</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection">
<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.SS1" title="In 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.1 </span>Definition and Structure of Knowledge Graphs</span></a>
<ol class="ltx_toclist ltx_toclist_subsection">
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.SS1.SSS1" title="In 2.1 Definition and Structure of Knowledge Graphs ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.1.1 </span>Ontologies</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.SS1.SSS2" title="In 2.1 Definition and Structure of Knowledge Graphs ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.1.2 </span>Example: Bio2RDF</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_subsection">
<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.SS2" title="In 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2 </span>Structuring Domain Knowledge with RDF</span></a>
<ol class="ltx_toclist ltx_toclist_subsection">
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.SS2.SSS1" title="In 2.2 Structuring Domain Knowledge with RDF ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2.1 </span>RDF</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.SS2.SSS2" title="In 2.2 Structuring Domain Knowledge with RDF ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2.2 </span>Ontologies</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.SS3" title="In 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.3 </span>Leveraging Graph Databases</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.SS4" title="In 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.4 </span>Visual Demonstration of ADKGs of Varying Sizes</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3" title="In A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3 </span>Feature Selection on Knowledge Graphs</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.SS1" title="In 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1 </span>Causal Discovery-Search Algorithm</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.SS2" title="In 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.2 </span>Feature Selection-Dimensionality Reduction</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.SS3" title="In 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3 </span>Data Linking and Data Integration-Similarity Based Methods</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.SS4" title="In 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.4 </span>Knowledge Graph Embeddings-Vector Embeddings</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.SS5" title="In 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.5 </span>Deep Learning-Advanced Network representation Learning</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.SS6" title="In 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.6 </span>Comparative Analysis of Different Approaches</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S4" title="In A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4 </span>Challenges and Opportunities in Knowledge Graph Feature Selection</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S4.SS1" title="In 4 Challenges and Opportunities in Knowledge Graph Feature Selection ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.1 </span>Challenges</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S4.SS2" title="In 4 Challenges and Opportunities in Knowledge Graph Feature Selection ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.2 </span>Future Directions</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S5" title="In A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5 </span>Conclusion</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S6" title="In A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">6 </span>Funding</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.SS1" title="In Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">6.1 </span>Appendix A. Table of Acronyms</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.SS2" title="In Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">6.2 </span>Appendix B. A more detailed description of KGs of sizes tiny, small, and medium</span></a></li>
</ol></nav>
</nav>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<div class="ltx_para" id="p1">
<span class="ltx_ERROR undefined" id="p1.1">\journaltitle</span>
<p class="ltx_p" id="p1.2">Briefings in Bioinformatics
<span class="ltx_ERROR undefined" id="p1.2.1">\DOI</span>DOI HERE


<span class="ltx_ERROR undefined" id="p1.2.2">\access</span>Advance Access Publication Date: Day Month Year
<span class="ltx_ERROR undefined" id="p1.2.3">\appnotes</span>Paper</p>
</div>
<div class="ltx_para" id="p2">
<span class="ltx_ERROR undefined" id="p2.1">\authormark</span>
<p class="ltx_p" id="p2.2">Author Name et al.</p>
</div>
<div class="ltx_para" id="p3">
<span class="ltx_ERROR undefined" id="p3.2">\corresp</span>
<p class="ltx_p" id="p3.1">[<math alttext="\ast" class="ltx_Math" display="inline" id="p3.1.m1.1"><semantics id="p3.1.m1.1a"><mo id="p3.1.m1.1.1" xref="p3.1.m1.1.1.cmml">∗</mo><annotation-xml encoding="MathML-Content" id="p3.1.m1.1b"><ci id="p3.1.m1.1.1.cmml" xref="p3.1.m1.1.1">∗</ci></annotation-xml><annotation encoding="application/x-tex" id="p3.1.m1.1c">\ast</annotation><annotation encoding="application/x-llamapun" id="p3.1.m1.1d">∗</annotation></semantics></math>]Corresponding author. <a class="ltx_ref ltx_href" href="email:Jason.Moore@csmc.edu" title="">Jason.Moore@csmc.edu</a></p>
</div>
<div class="ltx_para" id="p4">
<p class="ltx_p" id="p4.1">0Year
0Year
0Year</p>
</div>
<h1 class="ltx_title ltx_title_document">A review of Feature Selection on Knowledge Graphs</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Sisi Shao<span class="ltx_ERROR undefined" id="id1.1.id1">\ORCID</span>0009-0000-9783-9205
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Pedro Henrique Ribeiro
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Christina Ramirez<span class="ltx_ERROR undefined" id="id2.1.id1">\ORCID</span>0000-0002-8435-0416
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Jason H. Moore<span class="ltx_ERROR undefined" id="id3.1.id1">\ORCID</span>0000-0002-5015-1099
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_address"><span class="ltx_ERROR undefined" id="id4.2.id1">\orgdiv</span>Department of Biostatistics, <span class="ltx_ERROR undefined" id="id5.3.id2">\orgname</span>Fielding School of Public Health at University of California, Los Angeles, <span class="ltx_ERROR undefined" id="id6.4.id3">\orgaddress</span><span class="ltx_ERROR undefined" id="id7.5.id4">\street</span>650 Charles E Young Dr S, <span class="ltx_ERROR undefined" id="id8.6.id5">\postcode</span>90095-1772, <span class="ltx_ERROR undefined" id="id9.7.id6">\state</span>California, <span class="ltx_ERROR undefined" id="id10.8.id7">\country</span>Country
</span>
<span class="ltx_contact ltx_role_address"><span class="ltx_ERROR undefined" id="id11.9.id1">\orgdiv</span>Department of Computational Biomedicine, <span class="ltx_ERROR undefined" id="id12.10.id2">\orgname</span>Cedars-Sinai Medical Center, <span class="ltx_ERROR undefined" id="id13.11.id3">\orgaddress</span><span class="ltx_ERROR undefined" id="id14.12.id4">\street</span>8700 Beverly Blvd, <span class="ltx_ERROR undefined" id="id15.13.id5">\postcode</span>90048, <span class="ltx_ERROR undefined" id="id16.14.id6">\state</span>California, <span class="ltx_ERROR undefined" id="id17.15.id7">\country</span>United States
</span></span></span>
</div>
<div class="ltx_dates">(2024; 2024; Date; Date; Date)</div>
<h1 class="ltx_title ltx_title_document">A review of feature selection strategies utilizing graph data structures and knowledge graphs</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Sisi Shao<span class="ltx_ERROR undefined" id="id1.1.id1">\ORCID</span>0009-0000-9783-9205
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Pedro Henrique Ribeiro
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Christina Ramirez<span class="ltx_ERROR undefined" id="id2.1.id1">\ORCID</span>0000-0002-8435-0416
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Jason H. Moore<span class="ltx_ERROR undefined" id="id3.1.id1">\ORCID</span>0000-0002-5015-1099
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_address"><span class="ltx_ERROR undefined" id="id4.2.id1">\orgdiv</span>Department of Biostatistics, <span class="ltx_ERROR undefined" id="id5.3.id2">\orgname</span>Fielding School of Public Health at University of California, Los Angeles, <span class="ltx_ERROR undefined" id="id6.4.id3">\orgaddress</span><span class="ltx_ERROR undefined" id="id7.5.id4">\street</span>650 Charles E Young Dr S, <span class="ltx_ERROR undefined" id="id8.6.id5">\postcode</span>90095-1772, <span class="ltx_ERROR undefined" id="id9.7.id6">\state</span>California, <span class="ltx_ERROR undefined" id="id10.8.id7">\country</span>Country
</span>
<span class="ltx_contact ltx_role_address"><span class="ltx_ERROR undefined" id="id11.9.id1">\orgdiv</span>Department of Computational Biomedicine, <span class="ltx_ERROR undefined" id="id12.10.id2">\orgname</span>Cedars-Sinai Medical Center, <span class="ltx_ERROR undefined" id="id13.11.id3">\orgaddress</span><span class="ltx_ERROR undefined" id="id14.12.id4">\street</span>8700 Beverly Blvd, <span class="ltx_ERROR undefined" id="id15.13.id5">\postcode</span>90048, <span class="ltx_ERROR undefined" id="id16.14.id6">\state</span>California, <span class="ltx_ERROR undefined" id="id17.15.id7">\country</span>United States
</span></span></span>
</div>
<div class="ltx_dates">(2024; 2024; Date; Date; Date)</div>
<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p class="ltx_p" id="id18.id1">Feature selection in Knowledge Graphs (KGs) are increasingly utilized in diverse domains, including biomedical research, Natural Language Processing (NLP), and personalized recommendation systems. This paper delves into the methodologies for feature selection within KGs, emphasizing their roles in enhancing machine learning (ML) model efficacy, hypothesis generation, and interpretability. Through this comprehensive review, we aim to catalyze further innovation in feature selection for KGs, paving the way for more insightful, efficient, and interpretable analytical models across various domains. Our exploration reveals the critical importance of scalability, accuracy, and interpretability in feature selection techniques, advocating for the integration of domain knowledge to refine the selection process. We highlight the burgeoning potential of multi-objective optimization and interdisciplinary collaboration in advancing KG feature selection, underscoring the transformative impact of such methodologies on precision medicine, among other fields. The paper concludes by charting future directions, including the development of scalable, dynamic feature selection algorithms and the integration of explainable AI principles to foster transparency and trust in KG-driven models.</p>
</div>
<div class="ltx_classification">
<h6 class="ltx_title ltx_title_classification">keywords: </h6>Feature Selection, Knowledge Graphs, Deep Learning, Precision Medicine, Explainable AI.
</div>
<section class="ltx_section" id="S1">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>
<section class="ltx_subsection" id="S1.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.1 </span>Brief Introduction to Knowledge Graphs</h3>
<div class="ltx_para" id="S1.SS1.p1">
<p class="ltx_p" id="S1.SS1.p1.1">In the era of large-scale digital information, Knowledge Graphs (KGs) are an increasingly popular tool for organizing data and information<cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib1" title="">chicaiza2021comprehensive </a></cite>. At their core, KGs are characterized by representing entities and their relationships through triplets (subject-predicate-object), allowing for in-depth data analysis and the development of personalized care strategies. For instance, a triplet like “Cyclophosphamide - treats - Cancer” demonstrates KGs’ potential in drug discovery and repurposing. Platforms like Bio2RDF have been instrumental in exploring the complex relationships between genetics, diseases, and environmental factors. KGs thereby facilitate a comprehensive approach to healthcare; this approach supports a wide range of applications, from advanced decision-support systems to personalized medicine and innovative drug discovery methods <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib2" title="">belleau2008bio2rdf, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib3" title="">hasan2020knowledge, </a>)</cite>.</p>
</div>
<div class="ltx_para" id="S1.SS1.p2">
<p class="ltx_p" id="S1.SS1.p2.1">One of the most well-known uses for KGs is in the development of web-based technologies, including search engines and the Semantic Web (an extension of the World Wide Web that enables data to be shared and reused across applications). Google KG, DBpedia, and Yet Another Great Ontology (YAGO) utilize the principles of the Semantic Web and Linked Open Data (LOD–a method of publishing structured data so that it can be interlinked and become more useful) to create extensive networks of nodes and edges, representing the intricate relationships within vast datasets and enabling enhanced query processing and analytics capabilities. The contributions of scholars such as Fensel et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib4" title="">fensel2020introduction </a></cite>, Bonner et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib5" title="">bonner2022understanding </a></cite>, and Yang et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib6" title="">yang2023comprehensive </a></cite> have been crucial in shedding light on the foundational aspects and ongoing evolution of these systems.</p>
</div>
<div class="ltx_para" id="S1.SS1.p3">
<p class="ltx_p" id="S1.SS1.p3.1">As biology continues to advance, we’re accumulating a vast amount of knowledge about genes, proteins, chemicals, cells, diseases, and other biological entities along with their complex interactions which are intricate and multifaceted<cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib7" title="">levine2019biological </a></cite>. To make sense of this complexity, KGs have emerged as a powerful tool for organizing and connecting this information. In the realm of precision medicine, KGs have been used to consolidate disparate biomedical data, and been applied to improving the effectiveness of personalized patient care by systematically utilizing genetic, environmental, and lifestyle information. This application is exemplified by PrimeKG, which significantly contributes to creating a comprehensive medical knowledge base by integrating a wide ontology with data from various sources, including genomic databases, thereby supporting detailed medical research and personalized care planning <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib8" title="">chandak2023building, </a>)</cite>.</p>
</div>
<div class="ltx_para" id="S1.SS1.p4">
<p class="ltx_p" id="S1.SS1.p4.1">At their core, KGs are characterized by representing entities and their relationships through triplets (subject-predicate-object), allowing for in-depth data analysis and the development of personalized care strategies. For instance, a triplet like “Cyclophosphamide - treats - Cancer” demonstrates KGs’ potential in drug discovery and repurposing. Platforms like Bio2RDF have been instrumental in exploring the complex relationships between genetics, diseases, and environmental factors. KGs thereby facilitate a comprehensive approach to healthcare; this approach supports a wide range of applications, from advanced decision-support systems to personalized medicine and innovative drug discovery methods <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib2" title="">belleau2008bio2rdf, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib3" title="">hasan2020knowledge, </a>)</cite>.</p>
</div>
<div class="ltx_para" id="S1.SS1.p5">
<p class="ltx_p" id="S1.SS1.p5.1">The integration and analysis of data from biomedical research and clinical practice through KGs provide a dynamic platform for advancements in understanding and treating diseases. The academic discourse on feature selection methods applied to KGs, as highlighted by the studies referenced, underscores their transformative potential in various domains, particularly in advancing personalized medicine and healthcare outcomes.</p>
</div>
</section>
<section class="ltx_subsection" id="S1.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.2 </span>Importance of Feature Selection</h3>
<div class="ltx_para" id="S1.SS2.p1">
<p class="ltx_p" id="S1.SS2.p1.1">Feature selection involves choosing a subset of input variables most relevant for analysis, crucial in modern ML research due to the vast amounts of data ranging from petabytes to exabytes. As datasets grow in size and complexity, identifying important attributes is essential to address the ”curse of dimensionality” <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib9" title="">bellman1966dynamic, </a>)</cite>, which can degrade model performance. Reducing the feature set helps mitigate overfitting and improves computational efficiency <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib10" title="">ferreira2012efficient, </a>)</cite>. This reduction aids model interpretability in critical domains like healthcare and finance <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib11" title="">lahmiri2016features, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib12" title="">huda2016hybrid, </a>)</cite> and enhances the model’s generalizability to new data, a cornerstone for practical applications <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib13" title="">forster2000key, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib14" title="">saari2010generalizability, </a>)</cite>. Streamlined models, requiring fewer computational resources, are beneficial in resource-constrained scenarios like edge computing <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib15" title="">bikku2016hadoop, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib16" title="">mohammed2020edge, </a>)</cite>. With big data’s growing influence, especially in healthcare projected to reach $79.23 billion by 2028, feature selection is increasingly vital to ensure robust and applicable models.</p>
</div>
<div class="ltx_para" id="S1.SS2.p2">
<p class="ltx_p" id="S1.SS2.p2.1">Often in ML, feature selection refers to selecting columns of a tabular dataset. In this paper, we take a broader view, including selecting nodes or entities for hypothesis generation and further investigation. For example, a knowledge graph (KG) with genes and diseases can hypothesize new subsets of genes related to a specific disease.</p>
</div>
<div class="ltx_para" id="S1.SS2.p3">
<p class="ltx_p" id="S1.SS2.p3.1">Recognizing various feature selection methods, such as algorithmic techniques, statistical analyses <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib17" title="">jovic2015review, </a>)</cite>, and expert insights, we now explore the relationship between KGs and feature selection, highlighting how these frameworks can enhance the feature selection process.</p>
</div>
<figure class="ltx_figure" id="S1.F1"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_square" height="514" id="S1.F1.g1" src="x1.png" width="627"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S1.F1.2.1.1" style="font-size:90%;">Figure 1</span>: </span><span class="ltx_text" id="S1.F1.3.2" style="font-size:90%;">An integrated overview of KGs encompassing RDF structuring, Ontological frameworks, and GDB management, illustrating the flow from data sources to semantic querying and storage. Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S1.F1" title="Figure 1 ‣ 1.2 Importance of Feature Selection ‣ 1 Introduction ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">1</span></a> delineates the contribution of varied scholarly and scientific data sources—such as Google Scholar, PubMed, arXiv, and DrugBank—in providing raw data inputs. These inputs are then semantically encoded via the RDF, using triples that consist of subjects, predicates, and objects, alongside URIs that ensure the unique identification and integration of data entities across the KG. At the heart of the semantic structure are ontologies, exemplified here by the Unified Medical Language System (UMLS), which define the schema for the KG by outlining the essential relationships and attributes of the domain-specific entities. This ontology-based schema informs the organization and representation of knowledge within GDBs, such as Neo4j, which are specialized for storing and operationalizing the complex relational data of KGs. The central round-edged box showcases the role of query languages, with Cypher portrayed as a model for extracting information from GDBs through its intuitive syntax and pattern matching capabilities. The graphic elucidation of the query output illustrates a network of nodes and edges, representing the intricate interrelations and potential analytical insights derived from KGs. Each cluster within the network, designated as A, B, and C, symbolizes distinct subsets or aspects of the graph database that have been queried. </span></figcaption>
</figure>
</section>
<section class="ltx_subsection" id="S1.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.3 </span>Overview of the Relationship between Knowledge Graphs and Feature Selection</h3>
<div class="ltx_para" id="S1.SS3.p1">
<p class="ltx_p" id="S1.SS3.p1.1">The integration of KGs with feature selection processes marks a pivotal advancement in the realm of ML, particularly enhancing the capabilities of predictive models. Notably, many AI/ML systems remain largely unaware of domain-specific knowledge, such as biomedical information, which humans routinely leverage to solve complex problems. This oversight highlights the potential of KGs, with their rich web of entities, attributes, and interconnections, to bridge this gap. KGs play a crucial role across diverse domains such as the Semantic Web, NLP, and comprehensive data integration efforts, providing a structured representation that significantly aids in the precision of feature selection. This critical phase in ML aims at pinpointing the most relevant data attributes to optimize model performance, reduce over-fitting, and enhance interpretability.</p>
</div>
<div class="ltx_para" id="S1.SS3.p2">
<p class="ltx_p" id="S1.SS3.p2.1">However, integrating KGs into feature selection is challenging, encompassing issues of scalability, KG integrity, and adaptation to diverse domains. These challenges call for a concerted research effort aimed at developing scalable algorithms that efficiently navigate expansive KGs, enhance KG completeness, and foster the integration of varied data sources. This multidisciplinary arena benefits immensely from the combined expertise in knowledge representation, ML, and domain-specific areas, underscoring the critical need for a harmonious blend of structured knowledge with empirical, data-driven approaches.</p>
</div>
<div class="ltx_para" id="S1.SS3.p3">
<p class="ltx_p" id="S1.SS3.p3.1">To this end, exploring innovative methodologies becomes paramount. Approaches such as embedding-based feature selection and the application of graph neural networks (GNNs) demonstrate the potential of leveraging KGs’ unique characteristics for feature selection. These methodologies offer scalable and effective solutions for managing the high-dimensional spaces inherent to KGs, thus facilitating a more nuanced and comprehensive analysis of data.</p>
</div>
<div class="ltx_para" id="S1.SS3.p4">
<p class="ltx_p" id="S1.SS3.p4.1">Moreover, the dynamic nature of KGs, with their constantly evolving entities and relationships, necessitates feature selection methods that are not only adaptive but also capable of real-time updates. This adaptability ensures the relevance and efficacy of selected features in the face of new information, thereby maintaining the integrity and applicability of ML models in rapidly changing scenarios.</p>
</div>
</section>
</section>
<section class="ltx_section" id="S2">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Background and Key Concepts</h2>
<section class="ltx_subsection" id="S2.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.1 </span>Definition and Structure of Knowledge Graphs</h3>
<div class="ltx_para" id="S2.SS1.p1">
<p class="ltx_p" id="S2.SS1.p1.1">KGs categorize and link data for domain-specific knowledge discovery.</p>
</div>
<section class="ltx_subsubsection" id="S2.SS1.SSS1">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.1.1 </span>Ontologies</h4>
<div class="ltx_para" id="S2.SS1.SSS1.p1">
<p class="ltx_p" id="S2.SS1.SSS1.p1.1">KGs use ontologies to define relationships and model semantics <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib18" title="">staab2010handbook, </a>)</cite>. Ontologies categorize concepts, allowing flexible queries. Bio2RDF, for example, defines classes like ”proteins” and ”chemical entities,” and their relationships using RDF triples.</p>
</div>
</section>
<section class="ltx_subsubsection" id="S2.SS1.SSS2">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.1.2 </span>Example: Bio2RDF</h4>
<div class="ltx_para" id="S2.SS1.SSS2.p1">
<p class="ltx_p" id="S2.SS1.SSS2.p1.1">Bio2RDF integrates datasets like DrugBank <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib19" title="">wishart2018drugbank, </a>)</cite>, SIDER <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib20" title="">kuhn2016sider, </a>)</cite>, and KEGG <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib21" title="">kanehisa2000kegg, </a>)</cite> into a unified RDF structure, enhancing data interoperability and supporting complex queries.</p>
</div>
<div class="ltx_para" id="S2.SS1.SSS2.p2">
<ul class="ltx_itemize" id="S2.I1">
<li class="ltx_item" id="S2.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S2.I1.i1.p1">
<p class="ltx_p" id="S2.I1.i1.p1.1"><span class="ltx_text ltx_font_bold" id="S2.I1.i1.p1.1.1">Nodes:</span> Tagged with URIs, representing biomedical entities like genes and drugs.</p>
</div>
</li>
<li class="ltx_item" id="S2.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S2.I1.i2.p1">
<p class="ltx_p" id="S2.I1.i2.p1.1"><span class="ltx_text ltx_font_bold" id="S2.I1.i2.p1.1.1">Relationships:</span> Include ”targets” and ”is affected by,” illustrating drug-protein interactions and genetic influences.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section class="ltx_subsection" id="S2.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.2 </span>Structuring Domain Knowledge with RDF</h3>
<section class="ltx_subsubsection" id="S2.SS2.SSS1">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.2.1 </span>RDF</h4>
<div class="ltx_para" id="S2.SS2.SSS1.p1">
<p class="ltx_p" id="S2.SS2.SSS1.p1.1">RDF provides a structure for semantic representation in KGs <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib22" title="">bodenreider2004unified, </a>)</cite>. It formalizes relationships as triplets (subject-predicate-object) forming a graph <math alttext="G=\{(s,p,o)\}" class="ltx_Math" display="inline" id="S2.SS2.SSS1.p1.1.m1.4"><semantics id="S2.SS2.SSS1.p1.1.m1.4a"><mrow id="S2.SS2.SSS1.p1.1.m1.4.4" xref="S2.SS2.SSS1.p1.1.m1.4.4.cmml"><mi id="S2.SS2.SSS1.p1.1.m1.4.4.3" xref="S2.SS2.SSS1.p1.1.m1.4.4.3.cmml">G</mi><mo id="S2.SS2.SSS1.p1.1.m1.4.4.2" xref="S2.SS2.SSS1.p1.1.m1.4.4.2.cmml">=</mo><mrow id="S2.SS2.SSS1.p1.1.m1.4.4.1.1" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.2.cmml"><mo id="S2.SS2.SSS1.p1.1.m1.4.4.1.1.2" stretchy="false" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.2.cmml">{</mo><mrow id="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.2" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.1.cmml"><mo id="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.2.1" stretchy="false" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.1.cmml">(</mo><mi id="S2.SS2.SSS1.p1.1.m1.1.1" xref="S2.SS2.SSS1.p1.1.m1.1.1.cmml">s</mi><mo id="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.2.2" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.1.cmml">,</mo><mi id="S2.SS2.SSS1.p1.1.m1.2.2" xref="S2.SS2.SSS1.p1.1.m1.2.2.cmml">p</mi><mo id="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.2.3" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.1.cmml">,</mo><mi id="S2.SS2.SSS1.p1.1.m1.3.3" xref="S2.SS2.SSS1.p1.1.m1.3.3.cmml">o</mi><mo id="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.2.4" stretchy="false" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.1.cmml">)</mo></mrow><mo id="S2.SS2.SSS1.p1.1.m1.4.4.1.1.3" stretchy="false" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.2.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS1.p1.1.m1.4b"><apply id="S2.SS2.SSS1.p1.1.m1.4.4.cmml" xref="S2.SS2.SSS1.p1.1.m1.4.4"><eq id="S2.SS2.SSS1.p1.1.m1.4.4.2.cmml" xref="S2.SS2.SSS1.p1.1.m1.4.4.2"></eq><ci id="S2.SS2.SSS1.p1.1.m1.4.4.3.cmml" xref="S2.SS2.SSS1.p1.1.m1.4.4.3">𝐺</ci><set id="S2.SS2.SSS1.p1.1.m1.4.4.1.2.cmml" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.1"><vector id="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.1.cmml" xref="S2.SS2.SSS1.p1.1.m1.4.4.1.1.1.2"><ci id="S2.SS2.SSS1.p1.1.m1.1.1.cmml" xref="S2.SS2.SSS1.p1.1.m1.1.1">𝑠</ci><ci id="S2.SS2.SSS1.p1.1.m1.2.2.cmml" xref="S2.SS2.SSS1.p1.1.m1.2.2">𝑝</ci><ci id="S2.SS2.SSS1.p1.1.m1.3.3.cmml" xref="S2.SS2.SSS1.p1.1.m1.3.3">𝑜</ci></vector></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS1.p1.1.m1.4c">G=\{(s,p,o)\}</annotation><annotation encoding="application/x-llamapun" id="S2.SS2.SSS1.p1.1.m1.4d">italic_G = { ( italic_s , italic_p , italic_o ) }</annotation></semantics></math>. RDF enhances data interlinking and queryability <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib23" title="">donnelly2006snomed, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib24" title="">nelson2011normalized, </a>)</cite>.</p>
</div>
</section>
<section class="ltx_subsubsection" id="S2.SS2.SSS2">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.2.2 </span>Ontologies</h4>
<div class="ltx_para" id="S2.SS2.SSS2.p1">
<p class="ltx_p" id="S2.SS2.SSS2.p1.1">Ontologies in KGs categorize and describe concepts with flexible relationships. They enhance querying capabilities by defining both specific and abstract relationships, as seen in Bio2RDF and AlzKB.</p>
</div>
</section>
</section>
<section class="ltx_subsection" id="S2.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.3 </span>Leveraging Graph Databases</h3>
<div class="ltx_para" id="S2.SS3.p1">
<p class="ltx_p" id="S2.SS3.p1.1">Graph Databases (GDBs) like Neo4j manage complex data relationships within KGs, enabling efficient semantic analysis <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib25" title="">miller2013graph, </a>)</cite>. Freebase and query languages like Cypher and SPARQL extend GDB functionality for intuitive querying <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib26" title="">bollacker2008freebase, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib27" title="">francis2018cypher, </a>)</cite>.</p>
</div>
</section>
<section class="ltx_subsection" id="S2.SS4">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.4 </span>Visual Demonstration of ADKGs of Varying Sizes</h3>
<div class="ltx_para" id="S2.SS4.p1">
<p class="ltx_p" id="S2.SS4.p1.1">We use AlzKb, an Alzheimer’s Disease KG, to demonstrate varying KG sizes. Figures represent tiny (Cypher query limit 8), small (Cypher query limit 15), and medium (CYpher query limit 200) KGs. A tiny KG example is shown in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.F2" title="Figure 2 ‣ 2.4 Visual Demonstration of ADKGs of Varying Sizes ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">2</span></a>.</p>
</div>
<figure class="ltx_figure" id="S2.F2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="901" id="S2.F2.g1" src="extracted/5682567/figs/tinyKGexample3.png" width="1970"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S2.F2.2.1.1" style="font-size:90%;">Figure 2</span>: </span><span class="ltx_text" id="S2.F2.3.2" style="font-size:90%;">A Tiny-sized ADKG (Yellow Node: AD; Purple Nodes: Genes; Green Nodes: Drugs) <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib28" title="">alzheimersknowledgebase, </a>)</cite>. There are five instances of the “Chemical binds gene” relationship (light purple arrows), where a chemical is shown to interact directly with a gene; six instances of the “Gene associates with disease” relationship (yellow arrows), representing genes that have an association with AD; one instance of the “Chemical decreases expression”
relationship (dark green arrow), indicating a chemical that downregulates or decreases the expression of a gene; one instance of “Gene regulates gene” (purple arrow), suggesting a regulatory interaction between two genes, PPARG and TPI1. More detailed information on genes and drugs is given in the Appendix B. </span></figcaption>
</figure>
</section>
</section>
<section class="ltx_section" id="S3">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Feature Selection on Knowledge Graphs</h2>
<div class="ltx_para" id="S3.p1">
<p class="ltx_p" id="S3.p1.1">We categorize and evaluate the methodological frameworks delineated within the referenced manuscripts in this section. Below we elaborate on four distinct KG feature selection methods, including search algorithms, similarity-based methods, vector embeddings, and advanced network representation learning, available in the most current literature to the best of our present knowledge.</p>
</div>
<section class="ltx_subsection" id="S3.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.1 </span>Causal Discovery-Search Algorithm</h3>
<div class="ltx_para" id="S3.SS1.p1">
<p class="ltx_p" id="S3.SS1.p1.1">The goal of causal discovery is to move beyond merely describing correlated events to identifying the direction of influence between observed phenomena. The challenge in causality analysis lies in capturing the complex interactions between variables. Typically, these relationships are formalized using causal graphs, where nodes represent variables and directed edges denote causal effects.</p>
</div>
<div class="ltx_para" id="S3.SS1.p2">
<p class="ltx_p" id="S3.SS1.p2.1">In medicine, the gold standard for establishing causal relationships, including confounding, collider, mediation, moderation, reverse causality, effect modification, causal chain, and causal graph, is through randomized controlled trials (RCTs). However, various analytical methods can infer causal relationships from observational data. Researchers must consider other measured or unmeasured variables that may act as confounders, mediators, or colliders. For a comprehensive review of causal discovery, we recommend this survey paper by Zanga et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib29" title="">zanga2022survey </a></cite>.</p>
</div>
<div class="ltx_para" id="S3.SS1.p3">
<p class="ltx_p" id="S3.SS1.p3.1">There has been a lot of work recently on building automated methods, generally utilizing natural language processing (NLP) techniques, to extract causal relations from the scientific literature. These KGs can then be used to consolidate knowledge and form inferences and hypotheses about how different variables interact. Causal analysis can then be used to identify features that have causal effects on downstream variables.</p>
</div>
<div class="ltx_para" id="S3.SS1.p4">
<p class="ltx_p" id="S3.SS1.p4.1">The study by Malec et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib30" title="">malec2023causal </a></cite> introduced a novel causal feature selection framework using the ”ADKG” knowledge graph. This ADKG was constructed from post-2010 PubMed biomedical literature and an ontology-grounded KG via the PheKnowLator workflow <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib31" title="">callahan2019pheknowlator, </a>)</cite>. The authors used PubMed identifiers and machine reading systems like EIDOS, REACH, and SemRep within the INDRA ecosystem <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib32" title="">gyori2017word, </a>)</cite> to extract data. INDRA assembles knowledge into a model of causal molecular interactions <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib33" title="">perez2009semantics, </a>)</cite>, resulting in an OWL ontology <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib34" title="">horrocks2005owl, </a>)</cite>.</p>
</div>
<div class="ltx_para" id="S3.SS1.p5">
<p class="ltx_p" id="S3.SS1.p5.1">The study aimed to enhance causal feature selection with the ADKG. Hygiene steps were performed, and logical entailments were initially omitted. Predicates were mapped to the Relation Ontology (RO) to provide logical definitions and infer additional knowledge. Forward-chaining inference using CLIPS generated new triples based on RO properties, with belief scores assigned. Integration with PheKnowLator facilitated path search algorithms, reweighting edges with hierarchical relationships for optimized path searches. Competency questions, such as causal relationships between depression and AD, were addressed using SPARQL queries and Dijkstra’s shortest path algorithm <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib33" title="">perez2009semantics, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib35" title="">ducharme2013learning, </a>)</cite>.</p>
</div>
<div class="ltx_para" id="S3.SS1.p6">
<p class="ltx_p" id="S3.SS1.p6.1">Dijkstra’s algorithm applied to the ADKG identified shortest paths connecting genes and diseases, highlighting direct relationships <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib30" title="">malec2023causal, </a>)</cite>. These paths were analyzed to identify potential confounders, colliders, and mediators. Confounders influence both exposure and outcome, colliders are influenced by both, and mediators act as intermediaries. Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F3" title="Figure 3 ‣ 3.1 Causal Discovery-Search Algorithm ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">3</span></a> illustrates identifying a potential confounder between AD and depression using Dijkstra’s algorithm. The study identified 126 unique potential confounders, 29 colliders, and 18 potential mediators, showcasing the ADKG’s ability to uncover intricate relationships that traditional searches might miss.</p>
</div>
<figure class="ltx_figure" id="S3.F3"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="190" id="S3.F3.g1" src="extracted/5682567/figs/causalfeature.png" width="923"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F3.2.1.1" style="font-size:90%;">Figure 3</span>: </span><span class="ltx_text" id="S3.F3.3.2" style="font-size:90%;">Illustration of Inflammatory Response (pink node) as a Potential Confounder in the Association Between AD (left yellow node) and Depression (right yellow node). The diagram represents the shortest paths (through orange nodes) identified by Dijkstra’s algorithm. The two green paths also connect inflammatory response with AD and Depression but both of them are one unit longer than the orange ones. Consequently, Dijkstra’s algorithm picks the shortest path.</span></figcaption>
</figure>
</section>
<section class="ltx_subsection" id="S3.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.2 </span>Feature Selection-Dimensionality Reduction</h3>
<div class="ltx_para" id="S3.SS2.p1">
<p class="ltx_p" id="S3.SS2.p1.1">KGs can be utilized to perform feature selection for high-dimensional tabular datasets. In this scenario, nodes in the graph may relate to the columns, or features, of the tabular dataset. These subsets of features then be analyzed with methods such as machine learning. Below, we outline a few examples of graph-based methods for selecting subsets of features.</p>
<ul class="ltx_itemize" id="S3.I1">
<li class="ltx_item" id="S3.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I1.i1.p1">
<p class="ltx_p" id="S3.I1.i1.p1.1">Fang et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib36" title="">fang2019diagnosis </a></cite> developed an information theory approach informed by a KG to select features for training machine learning models. The goal of their study was to develop a predictive model of Chronic Obstructive Pulmonary Disease (COPD) from a tabular dataset including twenty-eight features representing medical tests and patient symptoms. First, a KG was constructed by integrating electronic medical records (EMRs) and domain-specific biomedical knowledge to identify and represent relationships among diseases, symptoms, causes, risk factors, drugs, side effects, and more. The features of the tabular dataset corresponded to nodes in the KG. Their algorithm, CMFS-<math alttext="\eta" class="ltx_Math" display="inline" id="S3.I1.i1.p1.1.m1.1"><semantics id="S3.I1.i1.p1.1.m1.1a"><mi id="S3.I1.i1.p1.1.m1.1.1" xref="S3.I1.i1.p1.1.m1.1.1.cmml">η</mi><annotation-xml encoding="MathML-Content" id="S3.I1.i1.p1.1.m1.1b"><ci id="S3.I1.i1.p1.1.m1.1.1.cmml" xref="S3.I1.i1.p1.1.m1.1.1">𝜂</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.I1.i1.p1.1.m1.1c">\eta</annotation><annotation encoding="application/x-llamapun" id="S3.I1.i1.p1.1.m1.1d">italic_η</annotation></semantics></math>, uses the weights between features in the KG to iteratively add or remove features from the set according to an information-theory-based heuristic. The study used this approach to select subsets of the corresponding features of the tabular dataset to train an SVM model.</p>
</div>
</li>
<li class="ltx_item" id="S3.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I1.i2.p1">
<p class="ltx_p" id="S3.I1.i2.p1.1">Ma et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib37" title="">ma2020knowledge </a></cite> sought to develop a model to predict whether a given Android app contained malware based on the Android API calls contained in the source code. First, they used the official documentation to construct a KG containing all API entities, such as classes and methods, as well as relationships between entities, such as return types and inheritance. Next, they identified a set of permissions considered to be highly sensitive that was required for each API entity. The study created a binary feature vector for each application based on whether or not a given entity was present in the code. To reduce the size of the binary feature vector, the authors selected only entities that were between one to four hops from a node requiring sensitive permission. ¡As not all entities contained explicit links in the documentation, an LSTM model was used to identify an additional subset of entities that shared similar descriptions with entities that require sensitive permissions.¿ This feature vector could then be used to train a classification model. A detailed description of how sensitive APIs, nodes in the KGs, are selected is shown in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F4" title="Figure 4 ‣ 2nd item ‣ 3.2 Feature Selection-Dimensionality Reduction ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">4</span></a>.</p>
</div>
<figure class="ltx_figure" id="S3.F4"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="117" id="S3.F4.g1" src="extracted/5682567/figs/android4.png" width="195"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F4.2.1.1" style="font-size:90%;">Figure 4</span>: </span><span class="ltx_text" id="S3.F4.3.2" style="font-size:90%;">Example of Direct and Indirect Dangerous APIs Selection Enabled by the Android API KG. The golden-orange rounded rectangle in the figure signifies a dangerous API called “getCall-CapablePhoneAccounts,” which facilitates the retrieval of Phone Account Handles for making and receiving calls. The light-yellow rounded rectangles are APIs directly connected to the Dangerous API, up to four degrees of separation through hyperlinks, with the understanding that links beyond this do not markedly enhance classification accuracy. The Siamese-BiLSTM network comes into play by identifying indirectly connected, potentially dangerous APIs—represented by the red rounded rectangle, such as “READ SMS,” which allows reading SMS messages but lacks a direct hyperlink or descriptive connection to other APIs. By embedding API descriptions into a vector space using Word2Vec and processing them through a Bidirectional LSTM, the network encodes the APIs’ textual data from both directions for a full context capture. These encoded vectors are then condensed through a dense layer into a final representation. Comparing these representations enables the network to detect hidden APIs that, while not directly linked, share sensitive characteristics with the known dangerous API, thereby revealing hidden dangers through textual similarity rather than explicit interlinking.</span></figcaption>
</figure>
</li>
<li class="ltx_item" id="S3.I1.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I1.i3.p1">
<p class="ltx_p" id="S3.I1.i3.p1.1">Jaworsky et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib38" title="">jaworsky2023interrelated </a></cite>, developed an unsupervised approach for selecting significantly interrelated features and eliminating redundant features from a KG, which they applied to a health survey dataset published by the Behavioral Risk Factor Surveillance System (BRFSS) <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib39" title="">centers2014behavioral, </a>)</cite>. The algorithm works by iteratively ranking scores based on their connections. This feature selection approach is divided into four main steps outlined in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F5" title="Figure 5 ‣ 3rd item ‣ 3.2 Feature Selection-Dimensionality Reduction ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">5</span></a>.</p>
</div>
<figure class="ltx_figure" id="S3.F5"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="127" id="S3.F5.g1" src="extracted/5682567/figs/health_survey2.png" width="223"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F5.2.1.1" style="font-size:90%;">Figure 5</span>: </span><span class="ltx_text" id="S3.F5.3.2" style="font-size:90%;">Illustration of interrelated feature selection procedure. 1) In the data filtering step as shown in part (a), states lacking lung cancer cases are excluded after referencing previous surveys spanning several years. 2) Features with over 50% missing values are eliminated. Then a KG is constructed from the remaining features. 3) A KG driven algorithm is used to transform the health survey question list to a data set with significantly interrelated features. 4) Finally, a binary relevance classifier (a special case of multi-label classifier) is proposed to predict the likelihood of multiple diseases by identifying 1-to-many cancer relationship. In part (b), the KG driven algorithm starts with the initial threshold 100%. Then it loops through existing features and compute weights for each (features with more edges will get more weights). By sorting the weights, the features with highest weights are kept and the threshold is subtracted by 1%. The algorithm is iterated until the stopping criterion is met.</span></figcaption>
</figure>
</li>
<li class="ltx_item" id="S3.I1.i4" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I1.i4.p1">
<p class="ltx_p" id="S3.I1.i4.p1.1">In the Hadith Corpus KG created by Mohammed et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib40" title="">atef2022feature </a></cite>, nodes represent distinct features and semantic categories derived from Hadith texts. Features include specific Islamic terms like “prayer” or “fasting,” while categories encompass broader thematic areas like rituals, ethics, jurisprudence, and other domains of Islamic scholarship. Edges in this KG quantify associations between features and categories based on co-occurrence frequency.</p>
</div>
<figure class="ltx_figure" id="S3.F6"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="455" id="S3.F6.g1" src="extracted/5682567/figs/ACO2.png" width="1201"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F6.12.6.1" style="font-size:90%;">Figure 6</span>: </span><span class="ltx_text" id="S3.F6.10.5" style="font-size:90%;">A demonstration of simplified ACO (Ant Colony Optimization) feature selection on Hadith Corpus KG. Here, two ants named Ben and Joe traverse the KG, with Ben starting at the “Zakat” node and moving to “Fasting”
across iterations, and Joe beginning his journey at a randomly selected node “Sawm”. The pheromone and heuristic values, represented by the green and red numbers above and below the edges, are aggregated outcomes of the explorations conducted by all ants in the system. Parameters <math alttext="\alpha" class="ltx_Math" display="inline" id="S3.F6.6.1.m1.1"><semantics id="S3.F6.6.1.m1.1b"><mi id="S3.F6.6.1.m1.1.1" xref="S3.F6.6.1.m1.1.1.cmml">α</mi><annotation-xml encoding="MathML-Content" id="S3.F6.6.1.m1.1c"><ci id="S3.F6.6.1.m1.1.1.cmml" xref="S3.F6.6.1.m1.1.1">𝛼</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F6.6.1.m1.1d">\alpha</annotation><annotation encoding="application/x-llamapun" id="S3.F6.6.1.m1.1e">italic_α</annotation></semantics></math> and <math alttext="\beta" class="ltx_Math" display="inline" id="S3.F6.7.2.m2.1"><semantics id="S3.F6.7.2.m2.1b"><mi id="S3.F6.7.2.m2.1.1" xref="S3.F6.7.2.m2.1.1.cmml">β</mi><annotation-xml encoding="MathML-Content" id="S3.F6.7.2.m2.1c"><ci id="S3.F6.7.2.m2.1.1.cmml" xref="S3.F6.7.2.m2.1.1">𝛽</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F6.7.2.m2.1d">\beta</annotation><annotation encoding="application/x-llamapun" id="S3.F6.7.2.m2.1e">italic_β</annotation></semantics></math> determine the relative influence of pheromone trails and heuristic information, respectively, while the evaporation rate <math alttext="\rho" class="ltx_Math" display="inline" id="S3.F6.8.3.m3.1"><semantics id="S3.F6.8.3.m3.1b"><mi id="S3.F6.8.3.m3.1.1" xref="S3.F6.8.3.m3.1.1.cmml">ρ</mi><annotation-xml encoding="MathML-Content" id="S3.F6.8.3.m3.1c"><ci id="S3.F6.8.3.m3.1.1.cmml" xref="S3.F6.8.3.m3.1.1">𝜌</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F6.8.3.m3.1d">\rho</annotation><annotation encoding="application/x-llamapun" id="S3.F6.8.3.m3.1e">italic_ρ</annotation></semantics></math> ensures flexibility in pathfinding, preventing premature convergence on suboptimal routes. The collective pheromone deposit <math alttext="\Delta" class="ltx_Math" display="inline" id="S3.F6.9.4.m4.1"><semantics id="S3.F6.9.4.m4.1b"><mi id="S3.F6.9.4.m4.1.1" mathvariant="normal" xref="S3.F6.9.4.m4.1.1.cmml">Δ</mi><annotation-xml encoding="MathML-Content" id="S3.F6.9.4.m4.1c"><ci id="S3.F6.9.4.m4.1.1.cmml" xref="S3.F6.9.4.m4.1.1">Δ</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F6.9.4.m4.1d">\Delta</annotation><annotation encoding="application/x-llamapun" id="S3.F6.9.4.m4.1e">roman_Δ</annotation></semantics></math> between “Fasting” and “Sawm” by Ben and Joe is a cumulative measure reflecting the alignment of the Hadith content with specific categories, denoted by the pink nodes. The probability that Ben chooses “Sawm” as the next feature is computed as a normalized version of <math alttext="\text{Pheromone}^{\alpha}\times\text{Heuristics}^{\beta}" class="ltx_Math" display="inline" id="S3.F6.10.5.m5.1"><semantics id="S3.F6.10.5.m5.1b"><mrow id="S3.F6.10.5.m5.1.1" xref="S3.F6.10.5.m5.1.1.cmml"><msup id="S3.F6.10.5.m5.1.1.2" xref="S3.F6.10.5.m5.1.1.2.cmml"><mtext id="S3.F6.10.5.m5.1.1.2.2" xref="S3.F6.10.5.m5.1.1.2.2a.cmml">Pheromone</mtext><mi id="S3.F6.10.5.m5.1.1.2.3" xref="S3.F6.10.5.m5.1.1.2.3.cmml">α</mi></msup><mo id="S3.F6.10.5.m5.1.1.1" lspace="0.222em" rspace="0.222em" xref="S3.F6.10.5.m5.1.1.1.cmml">×</mo><msup id="S3.F6.10.5.m5.1.1.3" xref="S3.F6.10.5.m5.1.1.3.cmml"><mtext id="S3.F6.10.5.m5.1.1.3.2" xref="S3.F6.10.5.m5.1.1.3.2a.cmml">Heuristics</mtext><mi id="S3.F6.10.5.m5.1.1.3.3" xref="S3.F6.10.5.m5.1.1.3.3.cmml">β</mi></msup></mrow><annotation-xml encoding="MathML-Content" id="S3.F6.10.5.m5.1c"><apply id="S3.F6.10.5.m5.1.1.cmml" xref="S3.F6.10.5.m5.1.1"><times id="S3.F6.10.5.m5.1.1.1.cmml" xref="S3.F6.10.5.m5.1.1.1"></times><apply id="S3.F6.10.5.m5.1.1.2.cmml" xref="S3.F6.10.5.m5.1.1.2"><csymbol cd="ambiguous" id="S3.F6.10.5.m5.1.1.2.1.cmml" xref="S3.F6.10.5.m5.1.1.2">superscript</csymbol><ci id="S3.F6.10.5.m5.1.1.2.2a.cmml" xref="S3.F6.10.5.m5.1.1.2.2"><mtext id="S3.F6.10.5.m5.1.1.2.2.cmml" xref="S3.F6.10.5.m5.1.1.2.2">Pheromone</mtext></ci><ci id="S3.F6.10.5.m5.1.1.2.3.cmml" xref="S3.F6.10.5.m5.1.1.2.3">𝛼</ci></apply><apply id="S3.F6.10.5.m5.1.1.3.cmml" xref="S3.F6.10.5.m5.1.1.3"><csymbol cd="ambiguous" id="S3.F6.10.5.m5.1.1.3.1.cmml" xref="S3.F6.10.5.m5.1.1.3">superscript</csymbol><ci id="S3.F6.10.5.m5.1.1.3.2a.cmml" xref="S3.F6.10.5.m5.1.1.3.2"><mtext id="S3.F6.10.5.m5.1.1.3.2.cmml" xref="S3.F6.10.5.m5.1.1.3.2">Heuristics</mtext></ci><ci id="S3.F6.10.5.m5.1.1.3.3.cmml" xref="S3.F6.10.5.m5.1.1.3.3">𝛽</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F6.10.5.m5.1d">\text{Pheromone}^{\alpha}\times\text{Heuristics}^{\beta}</annotation><annotation encoding="application/x-llamapun" id="S3.F6.10.5.m5.1e">Pheromone start_POSTSUPERSCRIPT italic_α end_POSTSUPERSCRIPT × Heuristics start_POSTSUPERSCRIPT italic_β end_POSTSUPERSCRIPT</annotation></semantics></math> (see the middle right of the figure). In this instance, the focus is to reinforce the linkage between fasting-related Hadiths and the “Physical Acts” category, differentiating it from the “Spiritual Practice” category and the “Forms of Worship” category, which are more aligned with spiritual benefits and devotional acts. </span></figcaption>
</figure>
<div class="ltx_para" id="S3.I1.i4.p2">
<p class="ltx_p" id="S3.I1.i4.p2.1">Feature selection for text classification is guided by Ant Colony Optimization (ACO) <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib41" title="">dorigo2005ant, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib42" title="">dorigo2006ant, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib43" title="">dorigo2019ant, </a>)</cite>. ACO is a probabilistic technique for solving computational problems which can be reduced to finding good paths through graphs. Inspired by the behavior of ants finding the shortest path from their colony to food sources, ACO is a part of swarm intelligence methods and a subset of evolutionary algorithms. Initially, several paths are randomly constructed, and after traversing a path, an ant deposits Pheromones along it (typically inversely proportional to path length), so shorter paths receive more pheromones. Over time, the pheromones evaporate, reducing their attractive strength to prevent premature convergence. When choosing their paths, ants probabilistically prefer paths with stronger pheromone concentrations while also exploring new paths to avoid local optima. The process is repeated until convergence. In this way, ACO balances between exploring new feature paths (exploration) and intensifying the search around promising features found in previous iterations (exploitation), adapting dynamically to find optimal feature sets for text classification <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib44" title="">parpinelli2002data, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib45" title="">martens2007classification, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib46" title="">aghdam2009text, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib47" title="">onan2023srl, </a>)</cite>. The pheromone trail and PageRank-like heuristic measure guide this optimization. We provide a graphical illustration of the ACO feature selection process in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F6" title="Figure 6 ‣ 4th item ‣ 3.2 Feature Selection-Dimensionality Reduction ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">6</span></a>.</p>
</div>
<div class="ltx_para" id="S3.I1.i4.p3">
<p class="ltx_p" id="S3.I1.i4.p3.1">This study demonstrates that integrating ACO into Arabic text classification yields a notable 3% average increase in accuracy, F1 score, recall, and precision compared to conventional methods like Naive Bayes, Random Forest, Decision Trees, and XGBoost, contributing significantly to the field of Arabic text classification.</p>
</div>
</li>
</ul>
</div>
</section>
<section class="ltx_subsection" id="S3.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.3 </span>Data Linking and Data Integration-Similarity Based Methods</h3>
<div class="ltx_para" id="S3.SS3.p1">
<ul class="ltx_itemize" id="S3.I2">
<li class="ltx_item" id="S3.I2.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I2.i1.p1">
<p class="ltx_p" id="S3.I2.i1.p1.1">Data linkage and data integration refer to the process of combining different sources of data<cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib48" title="">chang2018making </a></cite>. As KGs are developed to summarize large data, they can be great, easy-to-use tools for adding additional data and context to make ML workflows. For example, features of a given dataset can be expanded to include additional information per sample based what we know about a given feature. In Li et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib49" title="">li2020feature </a></cite>, the study collected data on self-reported student anxiety levels as well as basic information such as age, gender, grade, and home address. They then used the “Own-Think KG” (see Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F7" title="Figure 7 ‣ 3.4 Knowledge Graph Embeddings-Vector Embeddings ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">7</span></a>) as well as “DBpedia,” both known for their credibility and encyclopedic nature, to identify other features for their analysis based on the home address, including weather, population size, and GDP at both the district and regional area levels. These KGs follow a clear and explainable three-tuple storage structure, consisting of entities, attributes, and values, making them suitable for non-numerical feature generation. Importantly, they offer online querying capabilities, eliminating the need to download extensive datasets <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib50" title="">auer2007dbpedia, </a>)</cite>.</p>
</div>
</li>
</ul>
</div>
</section>
<section class="ltx_subsection" id="S3.SS4">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.4 </span>Knowledge Graph Embeddings-Vector Embeddings</h3>
<div class="ltx_para" id="S3.SS4.p1">
<p class="ltx_p" id="S3.SS4.p1.1">The embedding-focused approach in feature selection, exemplified by methods like the DistMult <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib51" title="">yang2014embedding </a></cite>, ComplEX <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib52" title="">trouillon2016complex, </a>)</cite>, TransE <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib53" title="">bordes2013translating, </a>)</cite>, and RESCAL <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib54" title="">nickel2011three, </a>)</cite>, and FeaBI <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib55" title="">ismaeil2023feabi </a></cite>, RippleNet <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib56" title="">wang2018ripplenet, </a>)</cite> seeks to represent nodes in a continuous vector space that capture deep semantic relationships and properties. This is a similar concept to word embeddings. Whereas in word embeddings, similar vectors capture similar semantic meaning, with similar words having similar representation, graph node embeddings capture relationship similarity within the graph network. The approach is popular for various applications, including link prediction <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib57" title="">kumar2020link, </a>)</cite> and entity classification <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib58" title="">al2020named, </a>)</cite>. Link prediction serves several purposes, from selecting movies a user would be interested in, to predicting drug-target interactions. Several methods have been developed to leverage embeddings for recommendation algorithms.</p>
</div>
<figure class="ltx_figure" id="S3.F7"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="160" id="S3.F7.g1" src="extracted/5682567/figs/Own-Think2.png" width="209"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F7.2.1.1" style="font-size:90%;">Figure 7</span>: </span><span class="ltx_text" id="S3.F7.3.2" style="font-size:90%;">Own-Think KG Advantage over Tradition One-hot Encoding. Consider a dataset that includes information about various cities, Beijing, Shanghai, and Hong Kong, where each city is represented by non-numerical discrete features such as its name. In a traditional dataset, this name might be converted into a numerical form using techniques like one-hot encoding. However, this process strips the city’s name of any contextual information about the city itself. Using a KG like the Own-Think KG, we can query additional information about each city to enrich the features, such as geographical, economic, demographic, cultural features, and so on. </span></figcaption>
</figure>
<figure class="ltx_figure" id="S3.F8">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_3">
<div class="ltx_block ltx_figure_panel ltx_minipage ltx_align_center ltx_align_middle" id="S3.F8.1" style="width:138.8pt;">
<img alt="Refer to caption" class="ltx_graphics ltx_img_square" height="579" id="S3.F8.1.g1" src="extracted/5682567/figs/beijing3.png" width="598"/>
</div>
</div>
<div class="ltx_flex_cell ltx_flex_size_3">
<div class="ltx_block ltx_figure_panel ltx_minipage ltx_align_center ltx_align_middle" id="S3.F8.2" style="width:143.1pt;">
<img alt="Refer to caption" class="ltx_graphics ltx_img_square" height="615" id="S3.F8.2.g1" src="extracted/5682567/figs/hongkong3.png" width="598"/>
</div>
</div>
<div class="ltx_flex_cell ltx_flex_size_3">
<div class="ltx_block ltx_figure_panel ltx_minipage ltx_align_center ltx_align_middle" id="S3.F8.3" style="width:138.8pt;">
<img alt="Refer to caption" class="ltx_graphics ltx_img_square" height="577" id="S3.F8.3.g1" src="extracted/5682567/figs/shanghai3.png" width="598"/>
</div>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F8.5.1.1" style="font-size:90%;">Figure 8</span>: </span><span class="ltx_text" id="S3.F8.6.2" style="font-size:90%;">Demonstration of Non-numeric Discrete Features Enrichment and Selection by Own-think KG. The figure includes enriched information for Beijing, Hong Kong, and Shanghai. For example, the additional features for Shanghai provided by the Own-Think KG (see Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F7" title="Figure 7 ‣ 3.4 Knowledge Graph Embeddings-Vector Embeddings ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">7</span></a>) detail Shanghai’s population size, average temperature, latitude, longitude, and GDP. contribute to a richer, more nuanced profile of Shanghai, compared to a one-hot encoding representation of each city, and offer additional insight as to how each aspect of a city may relate to the analysis at hand.</span></figcaption>
</figure>
<div class="ltx_para" id="S3.SS4.p2">
<p class="ltx_p" id="S3.SS4.p2.1"><span class="ltx_text ltx_font_italic" id="S3.SS4.p2.1.1">Embedding via DistMult:</span></p>
<ol class="ltx_enumerate" id="S3.I3">
<li class="ltx_item" id="S3.I3.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span>
<div class="ltx_para" id="S3.I3.i1.p1">
<p class="ltx_p" id="S3.I3.i1.p1.1">The DistMult method, designed to predict missing relationships or facts within a KG <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib59" title="">chen2020knowledge, </a>)</cite>, embeds entities and their interactions as vectors, inherently performing feature selection by:</p>
<ul class="ltx_itemize" id="S3.I3.i1.I1">
<li class="ltx_item" id="S3.I3.i1.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I3.i1.I1.i1.p1">
<p class="ltx_p" id="S3.I3.i1.I1.i1.p1.1"><span class="ltx_text ltx_font_italic" id="S3.I3.i1.I1.i1.p1.1.1">Capturing Semantic Similarities:</span> Entities with closer interactional kinship within the KG are embedded proximately, emphasizing features underlying these semantic similarities.</p>
</div>
</li>
<li class="ltx_item" id="S3.I3.i1.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I3.i1.I1.i2.p1">
<p class="ltx_p" id="S3.I3.i1.I1.i2.p1.1"><span class="ltx_text ltx_font_italic" id="S3.I3.i1.I1.i2.p1.1.1">Highlighting Relevant Interactions:</span> DistMult accentuates features defining the interactions, such as biological pathways or chemical properties relevant to the interaction.</p>
</div>
</li>
</ul>
</div>
</li>
<li class="ltx_item" id="S3.I3.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span>
<div class="ltx_para" id="S3.I3.i2.p1">
<p class="ltx_p" id="S3.I3.i2.p1.1"><span class="ltx_text ltx_font_italic" id="S3.I3.i2.p1.1.1">Optimization of Feature Representation:</span> The DistMult training process fine-tunes the entity and relation representations in the vector space, adjusting the significance of various attributes to enhance model accuracy.</p>
</div>
</li>
</ol>
</div>
<div class="ltx_para" id="S3.SS4.p3">
<ul class="ltx_itemize" id="S3.I4">
<li class="ltx_item" id="S3.I4.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I4.i1.p1">
<p class="ltx_p" id="S3.I4.i1.p1.1">One relatively simple strategy for edge prediction is to first create embeddings for each node and then train a classification algorithm to predict whether or not a connection exists between two nodes given their embeddings. For example, Wang et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib60" title="">wang2022kg </a></cite> utilized this strategy to predict drug-target interactions. In this study, the authors created node embeddings from a KG that contained known drug-target interactions. Next, they trained a deep learning model that took in a pair of embeddings (one drug and one target) to predict whether or not this pair was an existing edge in the graph. The authors showed that the model was able to identify some known interactions that were removed from the training set.</p>
</div>
</li>
<li class="ltx_item" id="S3.I4.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I4.i2.p1">
<p class="ltx_p" id="S3.I4.i2.p1.1">A unique example comes from Wang et al., who proposed a hybrid KG embedding and path-based method in a recommendation algorithm they named RippleNet <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib56" title="">wang2018ripplenet, </a>)</cite>. In this context, the KG contains nodes representing items that can be recommended, for example, movies, along with other nodes representing other features associated with each item, such as actors, genres, or release date. Edges associations between items and features, for example, a movie and its actors. In addition, there is a separate matrix that contains the interactions between each user and item. The goal is to predict the likelihood of a user selecting an item given the KG and the user’s prior interactions. The algorithm begins by initializing the representation of each item based on the user’s click history. Next, the algorithm iterates over items that are increasing hops from items the user had already interacted with. The end result is an embedding of the relevance of each item that is combined with the initial vector representation with a model for the final prediction of the likelihood of selecting that item. This was later extended by Wang et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib61" title="">wang2021multitask </a></cite> by having a combined deep framework that is simultaneously trained on a KG embedding task in addition to learning the recommendation task. The model architecture features shared latent features between the two tasks, with the idea being that the inclusion of the embedding task will enhance the latent representations. We give an illustration of Ripp-MKR in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F9" title="Figure 9 ‣ 2nd item ‣ 3.4 Knowledge Graph Embeddings-Vector Embeddings ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">9</span></a>.</p>
</div>
<figure class="ltx_figure" id="S3.F9"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="692" id="S3.F9.g1" src="extracted/5682567/figs/multitask.png" width="1018"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F9.42.18.1" style="font-size:90%;">Figure 9</span>: </span><span class="ltx_text" id="S3.F9.34.17" style="font-size:90%;">Illustration of Ripp-MKR Feature Learning Mechanisms. The Ripp-MKR model involves a recommendation system KG with nodes representing users, movies, genres, and actors. In this KG, relationships such as “Alice watched Barbie of Swan Lake,” “Barbie of Swan Lake is starred by Barbie,” and “Barbie of Swan Lake has genre Animation” are examples of how the system is structured (see <span class="ltx_text ltx_font_bold" id="S3.F9.34.17.1">KG Construction</span>). Taking Alice as the initial point, we construct a historical set, <math alttext="V_{\text{Alice}}" class="ltx_Math" display="inline" id="S3.F9.18.1.m1.1"><semantics id="S3.F9.18.1.m1.1b"><msub id="S3.F9.18.1.m1.1.1" xref="S3.F9.18.1.m1.1.1.cmml"><mi id="S3.F9.18.1.m1.1.1.2" xref="S3.F9.18.1.m1.1.1.2.cmml">V</mi><mtext id="S3.F9.18.1.m1.1.1.3" xref="S3.F9.18.1.m1.1.1.3a.cmml">Alice</mtext></msub><annotation-xml encoding="MathML-Content" id="S3.F9.18.1.m1.1c"><apply id="S3.F9.18.1.m1.1.1.cmml" xref="S3.F9.18.1.m1.1.1"><csymbol cd="ambiguous" id="S3.F9.18.1.m1.1.1.1.cmml" xref="S3.F9.18.1.m1.1.1">subscript</csymbol><ci id="S3.F9.18.1.m1.1.1.2.cmml" xref="S3.F9.18.1.m1.1.1.2">𝑉</ci><ci id="S3.F9.18.1.m1.1.1.3a.cmml" xref="S3.F9.18.1.m1.1.1.3"><mtext id="S3.F9.18.1.m1.1.1.3.cmml" mathsize="70%" xref="S3.F9.18.1.m1.1.1.3">Alice</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.18.1.m1.1d">V_{\text{Alice}}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.18.1.m1.1e">italic_V start_POSTSUBSCRIPT Alice end_POSTSUBSCRIPT</annotation></semantics></math>, comprising Alice’s movie-watching history, which includes three movies (see <span class="ltx_text ltx_font_bold" id="S3.F9.34.17.2">RippleNet</span>). RippleNet then extends Alice’s preference for the Barbie series to other movies with similar genres and actors, like “Barbie.” The <span class="ltx_text ltx_font_bold" id="S3.F9.34.17.3">KG Embedding Module</span> (KGE) refines Alice’s embedding, <math alttext="u_{\text{Alice}}" class="ltx_Math" display="inline" id="S3.F9.19.2.m2.1"><semantics id="S3.F9.19.2.m2.1b"><msub id="S3.F9.19.2.m2.1.1" xref="S3.F9.19.2.m2.1.1.cmml"><mi id="S3.F9.19.2.m2.1.1.2" xref="S3.F9.19.2.m2.1.1.2.cmml">u</mi><mtext id="S3.F9.19.2.m2.1.1.3" xref="S3.F9.19.2.m2.1.1.3a.cmml">Alice</mtext></msub><annotation-xml encoding="MathML-Content" id="S3.F9.19.2.m2.1c"><apply id="S3.F9.19.2.m2.1.1.cmml" xref="S3.F9.19.2.m2.1.1"><csymbol cd="ambiguous" id="S3.F9.19.2.m2.1.1.1.cmml" xref="S3.F9.19.2.m2.1.1">subscript</csymbol><ci id="S3.F9.19.2.m2.1.1.2.cmml" xref="S3.F9.19.2.m2.1.1.2">𝑢</ci><ci id="S3.F9.19.2.m2.1.1.3a.cmml" xref="S3.F9.19.2.m2.1.1.3"><mtext id="S3.F9.19.2.m2.1.1.3.cmml" mathsize="70%" xref="S3.F9.19.2.m2.1.1.3">Alice</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.19.2.m2.1d">u_{\text{Alice}}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.19.2.m2.1e">italic_u start_POSTSUBSCRIPT Alice end_POSTSUBSCRIPT</annotation></semantics></math>, by aggregating all the <math alttext="k" class="ltx_Math" display="inline" id="S3.F9.20.3.m3.1"><semantics id="S3.F9.20.3.m3.1b"><mi id="S3.F9.20.3.m3.1.1" xref="S3.F9.20.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S3.F9.20.3.m3.1c"><ci id="S3.F9.20.3.m3.1.1.cmml" xref="S3.F9.20.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.20.3.m3.1d">k</annotation><annotation encoding="application/x-llamapun" id="S3.F9.20.3.m3.1e">italic_k</annotation></semantics></math>-hop softmax-weighted tail embeddings <math alttext="t_{i}" class="ltx_Math" display="inline" id="S3.F9.21.4.m4.1"><semantics id="S3.F9.21.4.m4.1b"><msub id="S3.F9.21.4.m4.1.1" xref="S3.F9.21.4.m4.1.1.cmml"><mi id="S3.F9.21.4.m4.1.1.2" xref="S3.F9.21.4.m4.1.1.2.cmml">t</mi><mi id="S3.F9.21.4.m4.1.1.3" xref="S3.F9.21.4.m4.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="S3.F9.21.4.m4.1c"><apply id="S3.F9.21.4.m4.1.1.cmml" xref="S3.F9.21.4.m4.1.1"><csymbol cd="ambiguous" id="S3.F9.21.4.m4.1.1.1.cmml" xref="S3.F9.21.4.m4.1.1">subscript</csymbol><ci id="S3.F9.21.4.m4.1.1.2.cmml" xref="S3.F9.21.4.m4.1.1.2">𝑡</ci><ci id="S3.F9.21.4.m4.1.1.3.cmml" xref="S3.F9.21.4.m4.1.1.3">𝑖</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.21.4.m4.1d">t_{i}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.21.4.m4.1e">italic_t start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT</annotation></semantics></math>, for instance, “Animation” and “Barbie” (see <span class="ltx_text ltx_font_bold" id="S3.F9.34.17.4">KG Construction</span>). This refined embedding, <math alttext="u_{\text{Alice}}" class="ltx_Math" display="inline" id="S3.F9.22.5.m5.1"><semantics id="S3.F9.22.5.m5.1b"><msub id="S3.F9.22.5.m5.1.1" xref="S3.F9.22.5.m5.1.1.cmml"><mi id="S3.F9.22.5.m5.1.1.2" xref="S3.F9.22.5.m5.1.1.2.cmml">u</mi><mtext id="S3.F9.22.5.m5.1.1.3" xref="S3.F9.22.5.m5.1.1.3a.cmml">Alice</mtext></msub><annotation-xml encoding="MathML-Content" id="S3.F9.22.5.m5.1c"><apply id="S3.F9.22.5.m5.1.1.cmml" xref="S3.F9.22.5.m5.1.1"><csymbol cd="ambiguous" id="S3.F9.22.5.m5.1.1.1.cmml" xref="S3.F9.22.5.m5.1.1">subscript</csymbol><ci id="S3.F9.22.5.m5.1.1.2.cmml" xref="S3.F9.22.5.m5.1.1.2">𝑢</ci><ci id="S3.F9.22.5.m5.1.1.3a.cmml" xref="S3.F9.22.5.m5.1.1.3"><mtext id="S3.F9.22.5.m5.1.1.3.cmml" mathsize="70%" xref="S3.F9.22.5.m5.1.1.3">Alice</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.22.5.m5.1d">u_{\text{Alice}}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.22.5.m5.1e">italic_u start_POSTSUBSCRIPT Alice end_POSTSUBSCRIPT</annotation></semantics></math>, is processed through an <math alttext="L" class="ltx_Math" display="inline" id="S3.F9.23.6.m6.1"><semantics id="S3.F9.23.6.m6.1b"><mi id="S3.F9.23.6.m6.1.1" xref="S3.F9.23.6.m6.1.1.cmml">L</mi><annotation-xml encoding="MathML-Content" id="S3.F9.23.6.m6.1c"><ci id="S3.F9.23.6.m6.1.1.cmml" xref="S3.F9.23.6.m6.1.1">𝐿</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.23.6.m6.1d">L</annotation><annotation encoding="application/x-llamapun" id="S3.F9.23.6.m6.1e">italic_L</annotation></semantics></math>-layer MLP to derive a nuanced user vector, <math alttext="U_{\text{Alice}}^{L}" class="ltx_Math" display="inline" id="S3.F9.24.7.m7.1"><semantics id="S3.F9.24.7.m7.1b"><msubsup id="S3.F9.24.7.m7.1.1" xref="S3.F9.24.7.m7.1.1.cmml"><mi id="S3.F9.24.7.m7.1.1.2.2" xref="S3.F9.24.7.m7.1.1.2.2.cmml">U</mi><mtext id="S3.F9.24.7.m7.1.1.2.3" xref="S3.F9.24.7.m7.1.1.2.3a.cmml">Alice</mtext><mi id="S3.F9.24.7.m7.1.1.3" xref="S3.F9.24.7.m7.1.1.3.cmml">L</mi></msubsup><annotation-xml encoding="MathML-Content" id="S3.F9.24.7.m7.1c"><apply id="S3.F9.24.7.m7.1.1.cmml" xref="S3.F9.24.7.m7.1.1"><csymbol cd="ambiguous" id="S3.F9.24.7.m7.1.1.1.cmml" xref="S3.F9.24.7.m7.1.1">superscript</csymbol><apply id="S3.F9.24.7.m7.1.1.2.cmml" xref="S3.F9.24.7.m7.1.1"><csymbol cd="ambiguous" id="S3.F9.24.7.m7.1.1.2.1.cmml" xref="S3.F9.24.7.m7.1.1">subscript</csymbol><ci id="S3.F9.24.7.m7.1.1.2.2.cmml" xref="S3.F9.24.7.m7.1.1.2.2">𝑈</ci><ci id="S3.F9.24.7.m7.1.1.2.3a.cmml" xref="S3.F9.24.7.m7.1.1.2.3"><mtext id="S3.F9.24.7.m7.1.1.2.3.cmml" mathsize="70%" xref="S3.F9.24.7.m7.1.1.2.3">Alice</mtext></ci></apply><ci id="S3.F9.24.7.m7.1.1.3.cmml" xref="S3.F9.24.7.m7.1.1.3">𝐿</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.24.7.m7.1d">U_{\text{Alice}}^{L}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.24.7.m7.1e">italic_U start_POSTSUBSCRIPT Alice end_POSTSUBSCRIPT start_POSTSUPERSCRIPT italic_L end_POSTSUPERSCRIPT</annotation></semantics></math>. The KGE is informed by the interactions among movies, genres, and actors. The <span class="ltx_text ltx_font_bold" id="S3.F9.34.17.5">Cross and Compress Unit</span> (CCU) examines the interactions between different genres by calculating the outer product of the movie vector <math alttext="v" class="ltx_Math" display="inline" id="S3.F9.25.8.m8.1"><semantics id="S3.F9.25.8.m8.1b"><mi id="S3.F9.25.8.m8.1.1" xref="S3.F9.25.8.m8.1.1.cmml">v</mi><annotation-xml encoding="MathML-Content" id="S3.F9.25.8.m8.1c"><ci id="S3.F9.25.8.m8.1.1.cmml" xref="S3.F9.25.8.m8.1.1">𝑣</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.25.8.m8.1d">v</annotation><annotation encoding="application/x-llamapun" id="S3.F9.25.8.m8.1e">italic_v</annotation></semantics></math> (e.g., “Skyfall”) and an entity vector <math alttext="e" class="ltx_Math" display="inline" id="S3.F9.26.9.m9.1"><semantics id="S3.F9.26.9.m9.1b"><mi id="S3.F9.26.9.m9.1.1" xref="S3.F9.26.9.m9.1.1.cmml">e</mi><annotation-xml encoding="MathML-Content" id="S3.F9.26.9.m9.1c"><ci id="S3.F9.26.9.m9.1.1.cmml" xref="S3.F9.26.9.m9.1.1">𝑒</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.26.9.m9.1d">e</annotation><annotation encoding="application/x-llamapun" id="S3.F9.26.9.m9.1e">italic_e</annotation></semantics></math> from the set <math alttext="S_{\text{Skyfall}}" class="ltx_Math" display="inline" id="S3.F9.27.10.m10.1"><semantics id="S3.F9.27.10.m10.1b"><msub id="S3.F9.27.10.m10.1.1" xref="S3.F9.27.10.m10.1.1.cmml"><mi id="S3.F9.27.10.m10.1.1.2" xref="S3.F9.27.10.m10.1.1.2.cmml">S</mi><mtext id="S3.F9.27.10.m10.1.1.3" xref="S3.F9.27.10.m10.1.1.3a.cmml">Skyfall</mtext></msub><annotation-xml encoding="MathML-Content" id="S3.F9.27.10.m10.1c"><apply id="S3.F9.27.10.m10.1.1.cmml" xref="S3.F9.27.10.m10.1.1"><csymbol cd="ambiguous" id="S3.F9.27.10.m10.1.1.1.cmml" xref="S3.F9.27.10.m10.1.1">subscript</csymbol><ci id="S3.F9.27.10.m10.1.1.2.cmml" xref="S3.F9.27.10.m10.1.1.2">𝑆</ci><ci id="S3.F9.27.10.m10.1.1.3a.cmml" xref="S3.F9.27.10.m10.1.1.3"><mtext id="S3.F9.27.10.m10.1.1.3.cmml" mathsize="70%" xref="S3.F9.27.10.m10.1.1.3">Skyfall</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.27.10.m10.1d">S_{\text{Skyfall}}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.27.10.m10.1e">italic_S start_POSTSUBSCRIPT Skyfall end_POSTSUBSCRIPT</annotation></semantics></math>, which includes entities related to “Skyfall” in the KG. After performing the outer product between <math alttext="v" class="ltx_Math" display="inline" id="S3.F9.28.11.m11.1"><semantics id="S3.F9.28.11.m11.1b"><mi id="S3.F9.28.11.m11.1.1" xref="S3.F9.28.11.m11.1.1.cmml">v</mi><annotation-xml encoding="MathML-Content" id="S3.F9.28.11.m11.1c"><ci id="S3.F9.28.11.m11.1.1.cmml" xref="S3.F9.28.11.m11.1.1">𝑣</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.28.11.m11.1d">v</annotation><annotation encoding="application/x-llamapun" id="S3.F9.28.11.m11.1e">italic_v</annotation></semantics></math> and each <math alttext="e\in S_{\text{Skyfall}}" class="ltx_Math" display="inline" id="S3.F9.29.12.m12.1"><semantics id="S3.F9.29.12.m12.1b"><mrow id="S3.F9.29.12.m12.1.1" xref="S3.F9.29.12.m12.1.1.cmml"><mi id="S3.F9.29.12.m12.1.1.2" xref="S3.F9.29.12.m12.1.1.2.cmml">e</mi><mo id="S3.F9.29.12.m12.1.1.1" xref="S3.F9.29.12.m12.1.1.1.cmml">∈</mo><msub id="S3.F9.29.12.m12.1.1.3" xref="S3.F9.29.12.m12.1.1.3.cmml"><mi id="S3.F9.29.12.m12.1.1.3.2" xref="S3.F9.29.12.m12.1.1.3.2.cmml">S</mi><mtext id="S3.F9.29.12.m12.1.1.3.3" xref="S3.F9.29.12.m12.1.1.3.3a.cmml">Skyfall</mtext></msub></mrow><annotation-xml encoding="MathML-Content" id="S3.F9.29.12.m12.1c"><apply id="S3.F9.29.12.m12.1.1.cmml" xref="S3.F9.29.12.m12.1.1"><in id="S3.F9.29.12.m12.1.1.1.cmml" xref="S3.F9.29.12.m12.1.1.1"></in><ci id="S3.F9.29.12.m12.1.1.2.cmml" xref="S3.F9.29.12.m12.1.1.2">𝑒</ci><apply id="S3.F9.29.12.m12.1.1.3.cmml" xref="S3.F9.29.12.m12.1.1.3"><csymbol cd="ambiguous" id="S3.F9.29.12.m12.1.1.3.1.cmml" xref="S3.F9.29.12.m12.1.1.3">subscript</csymbol><ci id="S3.F9.29.12.m12.1.1.3.2.cmml" xref="S3.F9.29.12.m12.1.1.3.2">𝑆</ci><ci id="S3.F9.29.12.m12.1.1.3.3a.cmml" xref="S3.F9.29.12.m12.1.1.3.3"><mtext id="S3.F9.29.12.m12.1.1.3.3.cmml" mathsize="70%" xref="S3.F9.29.12.m12.1.1.3.3">Skyfall</mtext></ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.29.12.m12.1d">e\in S_{\text{Skyfall}}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.29.12.m12.1e">italic_e ∈ italic_S start_POSTSUBSCRIPT Skyfall end_POSTSUBSCRIPT</annotation></semantics></math> <math alttext="L" class="ltx_Math" display="inline" id="S3.F9.30.13.m13.1"><semantics id="S3.F9.30.13.m13.1b"><mi id="S3.F9.30.13.m13.1.1" xref="S3.F9.30.13.m13.1.1.cmml">L</mi><annotation-xml encoding="MathML-Content" id="S3.F9.30.13.m13.1c"><ci id="S3.F9.30.13.m13.1.1.cmml" xref="S3.F9.30.13.m13.1.1">𝐿</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.30.13.m13.1d">L</annotation><annotation encoding="application/x-llamapun" id="S3.F9.30.13.m13.1e">italic_L</annotation></semantics></math> times, the final latent feature vector, <math alttext="V_{\text{Skyfall}}^{L}" class="ltx_Math" display="inline" id="S3.F9.31.14.m14.1"><semantics id="S3.F9.31.14.m14.1b"><msubsup id="S3.F9.31.14.m14.1.1" xref="S3.F9.31.14.m14.1.1.cmml"><mi id="S3.F9.31.14.m14.1.1.2.2" xref="S3.F9.31.14.m14.1.1.2.2.cmml">V</mi><mtext id="S3.F9.31.14.m14.1.1.2.3" xref="S3.F9.31.14.m14.1.1.2.3a.cmml">Skyfall</mtext><mi id="S3.F9.31.14.m14.1.1.3" xref="S3.F9.31.14.m14.1.1.3.cmml">L</mi></msubsup><annotation-xml encoding="MathML-Content" id="S3.F9.31.14.m14.1c"><apply id="S3.F9.31.14.m14.1.1.cmml" xref="S3.F9.31.14.m14.1.1"><csymbol cd="ambiguous" id="S3.F9.31.14.m14.1.1.1.cmml" xref="S3.F9.31.14.m14.1.1">superscript</csymbol><apply id="S3.F9.31.14.m14.1.1.2.cmml" xref="S3.F9.31.14.m14.1.1"><csymbol cd="ambiguous" id="S3.F9.31.14.m14.1.1.2.1.cmml" xref="S3.F9.31.14.m14.1.1">subscript</csymbol><ci id="S3.F9.31.14.m14.1.1.2.2.cmml" xref="S3.F9.31.14.m14.1.1.2.2">𝑉</ci><ci id="S3.F9.31.14.m14.1.1.2.3a.cmml" xref="S3.F9.31.14.m14.1.1.2.3"><mtext id="S3.F9.31.14.m14.1.1.2.3.cmml" mathsize="70%" xref="S3.F9.31.14.m14.1.1.2.3">Skyfall</mtext></ci></apply><ci id="S3.F9.31.14.m14.1.1.3.cmml" xref="S3.F9.31.14.m14.1.1.3">𝐿</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.31.14.m14.1d">V_{\text{Skyfall}}^{L}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.31.14.m14.1e">italic_V start_POSTSUBSCRIPT Skyfall end_POSTSUBSCRIPT start_POSTSUPERSCRIPT italic_L end_POSTSUPERSCRIPT</annotation></semantics></math>, for “Skyfall” is obtained by taking the expectation over the L outer products. The <span class="ltx_text ltx_font_bold" id="S3.F9.34.17.6">Recommendation Module</span> then selects the movie with the highest sigmoid probability from the inner product of <math alttext="U_{\text{Alice}}^{L}" class="ltx_Math" display="inline" id="S3.F9.32.15.m15.1"><semantics id="S3.F9.32.15.m15.1b"><msubsup id="S3.F9.32.15.m15.1.1" xref="S3.F9.32.15.m15.1.1.cmml"><mi id="S3.F9.32.15.m15.1.1.2.2" xref="S3.F9.32.15.m15.1.1.2.2.cmml">U</mi><mtext id="S3.F9.32.15.m15.1.1.2.3" xref="S3.F9.32.15.m15.1.1.2.3a.cmml">Alice</mtext><mi id="S3.F9.32.15.m15.1.1.3" xref="S3.F9.32.15.m15.1.1.3.cmml">L</mi></msubsup><annotation-xml encoding="MathML-Content" id="S3.F9.32.15.m15.1c"><apply id="S3.F9.32.15.m15.1.1.cmml" xref="S3.F9.32.15.m15.1.1"><csymbol cd="ambiguous" id="S3.F9.32.15.m15.1.1.1.cmml" xref="S3.F9.32.15.m15.1.1">superscript</csymbol><apply id="S3.F9.32.15.m15.1.1.2.cmml" xref="S3.F9.32.15.m15.1.1"><csymbol cd="ambiguous" id="S3.F9.32.15.m15.1.1.2.1.cmml" xref="S3.F9.32.15.m15.1.1">subscript</csymbol><ci id="S3.F9.32.15.m15.1.1.2.2.cmml" xref="S3.F9.32.15.m15.1.1.2.2">𝑈</ci><ci id="S3.F9.32.15.m15.1.1.2.3a.cmml" xref="S3.F9.32.15.m15.1.1.2.3"><mtext id="S3.F9.32.15.m15.1.1.2.3.cmml" mathsize="70%" xref="S3.F9.32.15.m15.1.1.2.3">Alice</mtext></ci></apply><ci id="S3.F9.32.15.m15.1.1.3.cmml" xref="S3.F9.32.15.m15.1.1.3">𝐿</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.32.15.m15.1d">U_{\text{Alice}}^{L}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.32.15.m15.1e">italic_U start_POSTSUBSCRIPT Alice end_POSTSUBSCRIPT start_POSTSUPERSCRIPT italic_L end_POSTSUPERSCRIPT</annotation></semantics></math> and <math alttext="V_{\text{Skyfall}}^{L}" class="ltx_Math" display="inline" id="S3.F9.33.16.m16.1"><semantics id="S3.F9.33.16.m16.1b"><msubsup id="S3.F9.33.16.m16.1.1" xref="S3.F9.33.16.m16.1.1.cmml"><mi id="S3.F9.33.16.m16.1.1.2.2" xref="S3.F9.33.16.m16.1.1.2.2.cmml">V</mi><mtext id="S3.F9.33.16.m16.1.1.2.3" xref="S3.F9.33.16.m16.1.1.2.3a.cmml">Skyfall</mtext><mi id="S3.F9.33.16.m16.1.1.3" xref="S3.F9.33.16.m16.1.1.3.cmml">L</mi></msubsup><annotation-xml encoding="MathML-Content" id="S3.F9.33.16.m16.1c"><apply id="S3.F9.33.16.m16.1.1.cmml" xref="S3.F9.33.16.m16.1.1"><csymbol cd="ambiguous" id="S3.F9.33.16.m16.1.1.1.cmml" xref="S3.F9.33.16.m16.1.1">superscript</csymbol><apply id="S3.F9.33.16.m16.1.1.2.cmml" xref="S3.F9.33.16.m16.1.1"><csymbol cd="ambiguous" id="S3.F9.33.16.m16.1.1.2.1.cmml" xref="S3.F9.33.16.m16.1.1">subscript</csymbol><ci id="S3.F9.33.16.m16.1.1.2.2.cmml" xref="S3.F9.33.16.m16.1.1.2.2">𝑉</ci><ci id="S3.F9.33.16.m16.1.1.2.3a.cmml" xref="S3.F9.33.16.m16.1.1.2.3"><mtext id="S3.F9.33.16.m16.1.1.2.3.cmml" mathsize="70%" xref="S3.F9.33.16.m16.1.1.2.3">Skyfall</mtext></ci></apply><ci id="S3.F9.33.16.m16.1.1.3.cmml" xref="S3.F9.33.16.m16.1.1.3">𝐿</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.33.16.m16.1d">V_{\text{Skyfall}}^{L}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.33.16.m16.1e">italic_V start_POSTSUBSCRIPT Skyfall end_POSTSUBSCRIPT start_POSTSUPERSCRIPT italic_L end_POSTSUPERSCRIPT</annotation></semantics></math>, denoted by <math alttext="\widehat{y}_{\text{Alice,Skyfall}}" class="ltx_Math" display="inline" id="S3.F9.34.17.m17.1"><semantics id="S3.F9.34.17.m17.1b"><msub id="S3.F9.34.17.m17.1.1" xref="S3.F9.34.17.m17.1.1.cmml"><mover accent="true" id="S3.F9.34.17.m17.1.1.2" xref="S3.F9.34.17.m17.1.1.2.cmml"><mi id="S3.F9.34.17.m17.1.1.2.2" xref="S3.F9.34.17.m17.1.1.2.2.cmml">y</mi><mo id="S3.F9.34.17.m17.1.1.2.1" xref="S3.F9.34.17.m17.1.1.2.1.cmml">^</mo></mover><mtext id="S3.F9.34.17.m17.1.1.3" xref="S3.F9.34.17.m17.1.1.3a.cmml">Alice,Skyfall</mtext></msub><annotation-xml encoding="MathML-Content" id="S3.F9.34.17.m17.1c"><apply id="S3.F9.34.17.m17.1.1.cmml" xref="S3.F9.34.17.m17.1.1"><csymbol cd="ambiguous" id="S3.F9.34.17.m17.1.1.1.cmml" xref="S3.F9.34.17.m17.1.1">subscript</csymbol><apply id="S3.F9.34.17.m17.1.1.2.cmml" xref="S3.F9.34.17.m17.1.1.2"><ci id="S3.F9.34.17.m17.1.1.2.1.cmml" xref="S3.F9.34.17.m17.1.1.2.1">^</ci><ci id="S3.F9.34.17.m17.1.1.2.2.cmml" xref="S3.F9.34.17.m17.1.1.2.2">𝑦</ci></apply><ci id="S3.F9.34.17.m17.1.1.3a.cmml" xref="S3.F9.34.17.m17.1.1.3"><mtext id="S3.F9.34.17.m17.1.1.3.cmml" mathsize="70%" xref="S3.F9.34.17.m17.1.1.3">Alice,Skyfall</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F9.34.17.m17.1d">\widehat{y}_{\text{Alice,Skyfall}}</annotation><annotation encoding="application/x-llamapun" id="S3.F9.34.17.m17.1e">over^ start_ARG italic_y end_ARG start_POSTSUBSCRIPT Alice,Skyfall end_POSTSUBSCRIPT</annotation></semantics></math>. From potential next movies like “Skyfall,” “Inception,” and “Barbie: Fairytopia,” Ripp-MKR recommends “Barbie: Fairytopia” to Alice as it has the highest probability value, indicating it as the most suitable next watch.</span></figcaption>
</figure>
</li>
<li class="ltx_item" id="S3.I4.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I4.i3.p1">
<p class="ltx_p" id="S3.I4.i3.p1.1">Ismaeil et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib55" title="">ismaeil2023feabi </a></cite> introduced a method, FeaBI, to generate interpretable KG entity embeddings. First, a standard KG embedding is calculated. Additionally, a few categories of features for each node are extracted to form a vector, including the types of edges or relations it has, the types of nodes it is connected to, sequences of edge types of a certain length, and graph structural statistics. Next, regression random forest models are trained to predict each of the original embedding dimensions from its extracted feature vector. The random forest model ranks features based on their importance for the reconstruction task. These rankings can be used to better understand the information captured by embeddings. Additionally, a smaller subset of the feature vector can be selected for the most important features and used in place of the original embedding for more interpretable analysis.</p>
</div>
</li>
</ul>
</div>
</section>
<section class="ltx_subsection" id="S3.SS5">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.5 </span>Deep Learning-Advanced Network representation Learning</h3>
<div class="ltx_para" id="S3.SS5.p1">
<p class="ltx_p" id="S3.SS5.p1.1">Deep Learning models are designed to capture high-level, abstract representations of data. This ability allows them to capture meaningful insights from KGs, thereby enhancing applications in various domains, including personalized recommendations and predictive healthcare analytics.</p>
</div>
<div class="ltx_para" id="S3.SS5.p2">
<ul class="ltx_itemize" id="S3.I5">
<li class="ltx_item" id="S3.I5.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I5.i1.p1">
<p class="ltx_p" id="S3.I5.i1.p1.1">Anelli et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib62" title="">anelli2021sparse </a></cite> proposes KGFlex, a recommendation system <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib63" title="">shani2011evaluating, </a>)</cite> that integrates KG-based feature selection to improve the personalization and accuracy of recommendations. They use the notion of multi-hop predicates <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib64" title="">zhang2021cone, </a>)</cite> (i.e., considering chains of predicates that connect two entities at a high depth) to construct the semantic features on a KG. For instance, <math alttext="\text{A}\rightarrow\text{B}\rightarrow\text{C}" class="ltx_Math" display="inline" id="S3.I5.i1.p1.1.m1.1"><semantics id="S3.I5.i1.p1.1.m1.1a"><mrow id="S3.I5.i1.p1.1.m1.1.1" xref="S3.I5.i1.p1.1.m1.1.1.cmml"><mtext id="S3.I5.i1.p1.1.m1.1.1.2" xref="S3.I5.i1.p1.1.m1.1.1.2a.cmml">A</mtext><mo id="S3.I5.i1.p1.1.m1.1.1.3" stretchy="false" xref="S3.I5.i1.p1.1.m1.1.1.3.cmml">→</mo><mtext id="S3.I5.i1.p1.1.m1.1.1.4" xref="S3.I5.i1.p1.1.m1.1.1.4a.cmml">B</mtext><mo id="S3.I5.i1.p1.1.m1.1.1.5" stretchy="false" xref="S3.I5.i1.p1.1.m1.1.1.5.cmml">→</mo><mtext id="S3.I5.i1.p1.1.m1.1.1.6" xref="S3.I5.i1.p1.1.m1.1.1.6a.cmml">C</mtext></mrow><annotation-xml encoding="MathML-Content" id="S3.I5.i1.p1.1.m1.1b"><apply id="S3.I5.i1.p1.1.m1.1.1.cmml" xref="S3.I5.i1.p1.1.m1.1.1"><and id="S3.I5.i1.p1.1.m1.1.1a.cmml" xref="S3.I5.i1.p1.1.m1.1.1"></and><apply id="S3.I5.i1.p1.1.m1.1.1b.cmml" xref="S3.I5.i1.p1.1.m1.1.1"><ci id="S3.I5.i1.p1.1.m1.1.1.3.cmml" xref="S3.I5.i1.p1.1.m1.1.1.3">→</ci><ci id="S3.I5.i1.p1.1.m1.1.1.2a.cmml" xref="S3.I5.i1.p1.1.m1.1.1.2"><mtext id="S3.I5.i1.p1.1.m1.1.1.2.cmml" xref="S3.I5.i1.p1.1.m1.1.1.2">A</mtext></ci><ci id="S3.I5.i1.p1.1.m1.1.1.4a.cmml" xref="S3.I5.i1.p1.1.m1.1.1.4"><mtext id="S3.I5.i1.p1.1.m1.1.1.4.cmml" xref="S3.I5.i1.p1.1.m1.1.1.4">B</mtext></ci></apply><apply id="S3.I5.i1.p1.1.m1.1.1c.cmml" xref="S3.I5.i1.p1.1.m1.1.1"><ci id="S3.I5.i1.p1.1.m1.1.1.5.cmml" xref="S3.I5.i1.p1.1.m1.1.1.5">→</ci><share href="https://arxiv.org/html/2406.14864v1#S3.I5.i1.p1.1.m1.1.1.4.cmml" id="S3.I5.i1.p1.1.m1.1.1d.cmml" xref="S3.I5.i1.p1.1.m1.1.1"></share><ci id="S3.I5.i1.p1.1.m1.1.1.6a.cmml" xref="S3.I5.i1.p1.1.m1.1.1.6"><mtext id="S3.I5.i1.p1.1.m1.1.1.6.cmml" xref="S3.I5.i1.p1.1.m1.1.1.6">C</mtext></ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.I5.i1.p1.1.m1.1c">\text{A}\rightarrow\text{B}\rightarrow\text{C}</annotation><annotation encoding="application/x-llamapun" id="S3.I5.i1.p1.1.m1.1d">A → B → C</annotation></semantics></math> is a 2-hop predicate.</p>
</div>
<div class="ltx_para" id="S3.I5.i1.p2">
<p class="ltx_p" id="S3.I5.i1.p2.3">In the feature selection step, KGFlex utilizes the concepts of entropy and information gain <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib65" title="">shannon1948mathematical, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib66" title="">rokach2005top, </a>)</cite> to assess how significant and relevant a feature is to a user when determining whether to engage with an item or not, i.e., to watch a movie or not. The features, represented as <math alttext="\langle" class="ltx_Math" display="inline" id="S3.I5.i1.p2.1.m1.1"><semantics id="S3.I5.i1.p2.1.m1.1a"><mo id="S3.I5.i1.p2.1.m1.1.1" stretchy="false" xref="S3.I5.i1.p2.1.m1.1.1.cmml">⟨</mo><annotation-xml encoding="MathML-Content" id="S3.I5.i1.p2.1.m1.1b"><ci id="S3.I5.i1.p2.1.m1.1.1.cmml" xref="S3.I5.i1.p2.1.m1.1.1">⟨</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.I5.i1.p2.1.m1.1c">\langle</annotation><annotation encoding="application/x-llamapun" id="S3.I5.i1.p2.1.m1.1d">⟨</annotation></semantics></math>predicate<span class="ltx_text ltx_markedasmath" id="S3.I5.i1.p2.3.1">,</span>entity<math alttext="\rangle" class="ltx_Math" display="inline" id="S3.I5.i1.p2.3.m3.1"><semantics id="S3.I5.i1.p2.3.m3.1a"><mo id="S3.I5.i1.p2.3.m3.1.1" stretchy="false" xref="S3.I5.i1.p2.3.m3.1.1.cmml">⟩</mo><annotation-xml encoding="MathML-Content" id="S3.I5.i1.p2.3.m3.1b"><ci id="S3.I5.i1.p2.3.m3.1.1.cmml" xref="S3.I5.i1.p2.3.m3.1.1">⟩</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.I5.i1.p2.3.m3.1c">\rangle</annotation><annotation encoding="application/x-llamapun" id="S3.I5.i1.p2.3.m3.1d">⟩</annotation></semantics></math> pairs, are then embedded in a latent space to construct the user-item interaction along with user embeddings via DL methods. For a particular user, the items with higher user-item interactions are recommended. All the embeddings and model parameters in KGFlex are learned from the Bayesian Personalized Ranking (BPR) optimization criterion <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib67" title="">rendle2012bpr, </a>)</cite>. The whole procedure is visualized in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F10" title="Figure 10 ‣ 1st item ‣ 3.5 Deep Learning-Advanced Network representation Learning ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">10</span></a>.</p>
</div>
<figure class="ltx_figure" id="S3.F10"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="600" id="S3.F10.g1" src="extracted/5682567/figs/sparse_new.png" width="904"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F10.44.22.1" style="font-size:90%;">Figure 10</span>: </span><span class="ltx_text" id="S3.F10.42.21" style="font-size:90%;">Illustration of KGFlex feature selection and recommendation procedure. We start with a KG with 6 nodes and 6 predicates (edges/relations). A feature set <math alttext="\mathcal{F}" class="ltx_Math" display="inline" id="S3.F10.22.1.m1.1"><semantics id="S3.F10.22.1.m1.1b"><mi class="ltx_font_mathcaligraphic" id="S3.F10.22.1.m1.1.1" xref="S3.F10.22.1.m1.1.1.cmml">ℱ</mi><annotation-xml encoding="MathML-Content" id="S3.F10.22.1.m1.1c"><ci id="S3.F10.22.1.m1.1.1.cmml" xref="S3.F10.22.1.m1.1.1">ℱ</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.22.1.m1.1d">\mathcal{F}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.22.1.m1.1e">caligraphic_F</annotation></semantics></math> is constructed where each element is of form <math alttext="\langle\text{predicate, node}\rangle" class="ltx_Math" display="inline" id="S3.F10.23.2.m2.1"><semantics id="S3.F10.23.2.m2.1b"><mrow id="S3.F10.23.2.m2.1.2.2" xref="S3.F10.23.2.m2.1.2.1.cmml"><mo id="S3.F10.23.2.m2.1.2.2.1" stretchy="false" xref="S3.F10.23.2.m2.1.2.1.1.cmml">⟨</mo><mtext id="S3.F10.23.2.m2.1.1" xref="S3.F10.23.2.m2.1.1a.cmml">predicate, node</mtext><mo id="S3.F10.23.2.m2.1.2.2.2" stretchy="false" xref="S3.F10.23.2.m2.1.2.1.1.cmml">⟩</mo></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.23.2.m2.1c"><apply id="S3.F10.23.2.m2.1.2.1.cmml" xref="S3.F10.23.2.m2.1.2.2"><csymbol cd="latexml" id="S3.F10.23.2.m2.1.2.1.1.cmml" xref="S3.F10.23.2.m2.1.2.2.1">delimited-⟨⟩</csymbol><ci id="S3.F10.23.2.m2.1.1a.cmml" xref="S3.F10.23.2.m2.1.1"><mtext id="S3.F10.23.2.m2.1.1.cmml" xref="S3.F10.23.2.m2.1.1">predicate, node</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.23.2.m2.1d">\langle\text{predicate, node}\rangle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.23.2.m2.1e">⟨ predicate, node ⟩</annotation></semantics></math>; for instance, from node A we can get to node B via a black predicate, then a feature is constructed as <math alttext="\langle" class="ltx_Math" display="inline" id="S3.F10.24.3.m3.1"><semantics id="S3.F10.24.3.m3.1b"><mo id="S3.F10.24.3.m3.1.1" stretchy="false" xref="S3.F10.24.3.m3.1.1.cmml">⟨</mo><annotation-xml encoding="MathML-Content" id="S3.F10.24.3.m3.1c"><ci id="S3.F10.24.3.m3.1.1.cmml" xref="S3.F10.24.3.m3.1.1">⟨</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.24.3.m3.1d">\langle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.24.3.m3.1e">⟨</annotation></semantics></math>Black, B<math alttext="\rangle" class="ltx_Math" display="inline" id="S3.F10.25.4.m4.1"><semantics id="S3.F10.25.4.m4.1b"><mo id="S3.F10.25.4.m4.1.1" stretchy="false" xref="S3.F10.25.4.m4.1.1.cmml">⟩</mo><annotation-xml encoding="MathML-Content" id="S3.F10.25.4.m4.1c"><ci id="S3.F10.25.4.m4.1.1.cmml" xref="S3.F10.25.4.m4.1.1">⟩</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.25.4.m4.1d">\rangle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.25.4.m4.1e">⟩</annotation></semantics></math>. We construct a global embedding set <math alttext="\mathcal{G}" class="ltx_Math" display="inline" id="S3.F10.26.5.m5.1"><semantics id="S3.F10.26.5.m5.1b"><mi class="ltx_font_mathcaligraphic" id="S3.F10.26.5.m5.1.1" xref="S3.F10.26.5.m5.1.1.cmml">𝒢</mi><annotation-xml encoding="MathML-Content" id="S3.F10.26.5.m5.1c"><ci id="S3.F10.26.5.m5.1.1.cmml" xref="S3.F10.26.5.m5.1.1">𝒢</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.26.5.m5.1d">\mathcal{G}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.26.5.m5.1e">caligraphic_G</annotation></semantics></math> representing each feature in <math alttext="\mathcal{F}" class="ltx_Math" display="inline" id="S3.F10.27.6.m6.1"><semantics id="S3.F10.27.6.m6.1b"><mi class="ltx_font_mathcaligraphic" id="S3.F10.27.6.m6.1.1" xref="S3.F10.27.6.m6.1.1.cmml">ℱ</mi><annotation-xml encoding="MathML-Content" id="S3.F10.27.6.m6.1c"><ci id="S3.F10.27.6.m6.1.1.cmml" xref="S3.F10.27.6.m6.1.1">ℱ</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.27.6.m6.1d">\mathcal{F}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.27.6.m6.1e">caligraphic_F</annotation></semantics></math>, and a user-feature embedding set <math alttext="\mathcal{P}" class="ltx_Math" display="inline" id="S3.F10.28.7.m7.1"><semantics id="S3.F10.28.7.m7.1b"><mi class="ltx_font_mathcaligraphic" id="S3.F10.28.7.m7.1.1" xref="S3.F10.28.7.m7.1.1.cmml">𝒫</mi><annotation-xml encoding="MathML-Content" id="S3.F10.28.7.m7.1c"><ci id="S3.F10.28.7.m7.1.1.cmml" xref="S3.F10.28.7.m7.1.1">𝒫</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.28.7.m7.1d">\mathcal{P}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.28.7.m7.1e">caligraphic_P</annotation></semantics></math> for each pair of user and feature. All embeddings and parameters in KGFlex are learned via DL methods with the BPR optimization criterion. We then associate each user-feature pair with an information gain <math alttext="\mathbb{IG}" class="ltx_Math" display="inline" id="S3.F10.29.8.m8.1"><semantics id="S3.F10.29.8.m8.1b"><mrow id="S3.F10.29.8.m8.1.1" xref="S3.F10.29.8.m8.1.1.cmml"><mi id="S3.F10.29.8.m8.1.1.2" xref="S3.F10.29.8.m8.1.1.2.cmml">𝕀</mi><mo id="S3.F10.29.8.m8.1.1.1" xref="S3.F10.29.8.m8.1.1.1.cmml">⁢</mo><mi id="S3.F10.29.8.m8.1.1.3" xref="S3.F10.29.8.m8.1.1.3.cmml">𝔾</mi></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.29.8.m8.1c"><apply id="S3.F10.29.8.m8.1.1.cmml" xref="S3.F10.29.8.m8.1.1"><times id="S3.F10.29.8.m8.1.1.1.cmml" xref="S3.F10.29.8.m8.1.1.1"></times><ci id="S3.F10.29.8.m8.1.1.2.cmml" xref="S3.F10.29.8.m8.1.1.2">𝕀</ci><ci id="S3.F10.29.8.m8.1.1.3.cmml" xref="S3.F10.29.8.m8.1.1.3">𝔾</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.29.8.m8.1d">\mathbb{IG}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.29.8.m8.1e">blackboard_I blackboard_G</annotation></semantics></math>, which measures the expected reduction in information entropy from a prior node to a new node that acquires some information. For instance, suppose a user is currently at node <math alttext="A" class="ltx_Math" display="inline" id="S3.F10.30.9.m9.1"><semantics id="S3.F10.30.9.m9.1b"><mi id="S3.F10.30.9.m9.1.1" xref="S3.F10.30.9.m9.1.1.cmml">A</mi><annotation-xml encoding="MathML-Content" id="S3.F10.30.9.m9.1c"><ci id="S3.F10.30.9.m9.1.1.cmml" xref="S3.F10.30.9.m9.1.1">𝐴</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.30.9.m9.1d">A</annotation><annotation encoding="application/x-llamapun" id="S3.F10.30.9.m9.1e">italic_A</annotation></semantics></math>. The computed information gain <math alttext="\mathbb{IG}" class="ltx_Math" display="inline" id="S3.F10.31.10.m10.1"><semantics id="S3.F10.31.10.m10.1b"><mrow id="S3.F10.31.10.m10.1.1" xref="S3.F10.31.10.m10.1.1.cmml"><mi id="S3.F10.31.10.m10.1.1.2" xref="S3.F10.31.10.m10.1.1.2.cmml">𝕀</mi><mo id="S3.F10.31.10.m10.1.1.1" xref="S3.F10.31.10.m10.1.1.1.cmml">⁢</mo><mi id="S3.F10.31.10.m10.1.1.3" xref="S3.F10.31.10.m10.1.1.3.cmml">𝔾</mi></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.31.10.m10.1c"><apply id="S3.F10.31.10.m10.1.1.cmml" xref="S3.F10.31.10.m10.1.1"><times id="S3.F10.31.10.m10.1.1.1.cmml" xref="S3.F10.31.10.m10.1.1.1"></times><ci id="S3.F10.31.10.m10.1.1.2.cmml" xref="S3.F10.31.10.m10.1.1.2">𝕀</ci><ci id="S3.F10.31.10.m10.1.1.3.cmml" xref="S3.F10.31.10.m10.1.1.3">𝔾</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.31.10.m10.1d">\mathbb{IG}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.31.10.m10.1e">blackboard_I blackboard_G</annotation></semantics></math>(<math alttext="\langle" class="ltx_Math" display="inline" id="S3.F10.32.11.m11.1"><semantics id="S3.F10.32.11.m11.1b"><mo id="S3.F10.32.11.m11.1.1" stretchy="false" xref="S3.F10.32.11.m11.1.1.cmml">⟨</mo><annotation-xml encoding="MathML-Content" id="S3.F10.32.11.m11.1c"><ci id="S3.F10.32.11.m11.1.1.cmml" xref="S3.F10.32.11.m11.1.1">⟨</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.32.11.m11.1d">\langle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.32.11.m11.1e">⟨</annotation></semantics></math>Black, B<math alttext="\rangle" class="ltx_Math" display="inline" id="S3.F10.33.12.m12.1"><semantics id="S3.F10.33.12.m12.1b"><mo id="S3.F10.33.12.m12.1.1" stretchy="false" xref="S3.F10.33.12.m12.1.1.cmml">⟩</mo><annotation-xml encoding="MathML-Content" id="S3.F10.33.12.m12.1c"><ci id="S3.F10.33.12.m12.1.1.cmml" xref="S3.F10.33.12.m12.1.1">⟩</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.33.12.m12.1d">\rangle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.33.12.m12.1e">⟩</annotation></semantics></math>)=1, <math alttext="\mathbb{IG}" class="ltx_Math" display="inline" id="S3.F10.34.13.m13.1"><semantics id="S3.F10.34.13.m13.1b"><mrow id="S3.F10.34.13.m13.1.1" xref="S3.F10.34.13.m13.1.1.cmml"><mi id="S3.F10.34.13.m13.1.1.2" xref="S3.F10.34.13.m13.1.1.2.cmml">𝕀</mi><mo id="S3.F10.34.13.m13.1.1.1" xref="S3.F10.34.13.m13.1.1.1.cmml">⁢</mo><mi id="S3.F10.34.13.m13.1.1.3" xref="S3.F10.34.13.m13.1.1.3.cmml">𝔾</mi></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.34.13.m13.1c"><apply id="S3.F10.34.13.m13.1.1.cmml" xref="S3.F10.34.13.m13.1.1"><times id="S3.F10.34.13.m13.1.1.1.cmml" xref="S3.F10.34.13.m13.1.1.1"></times><ci id="S3.F10.34.13.m13.1.1.2.cmml" xref="S3.F10.34.13.m13.1.1.2">𝕀</ci><ci id="S3.F10.34.13.m13.1.1.3.cmml" xref="S3.F10.34.13.m13.1.1.3">𝔾</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.34.13.m13.1d">\mathbb{IG}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.34.13.m13.1e">blackboard_I blackboard_G</annotation></semantics></math>(<math alttext="\langle" class="ltx_Math" display="inline" id="S3.F10.35.14.m14.1"><semantics id="S3.F10.35.14.m14.1b"><mo id="S3.F10.35.14.m14.1.1" stretchy="false" xref="S3.F10.35.14.m14.1.1.cmml">⟨</mo><annotation-xml encoding="MathML-Content" id="S3.F10.35.14.m14.1c"><ci id="S3.F10.35.14.m14.1.1.cmml" xref="S3.F10.35.14.m14.1.1">⟨</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.35.14.m14.1d">\langle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.35.14.m14.1e">⟨</annotation></semantics></math>Orange, E<math alttext="\rangle" class="ltx_Math" display="inline" id="S3.F10.36.15.m15.1"><semantics id="S3.F10.36.15.m15.1b"><mo id="S3.F10.36.15.m15.1.1" stretchy="false" xref="S3.F10.36.15.m15.1.1.cmml">⟩</mo><annotation-xml encoding="MathML-Content" id="S3.F10.36.15.m15.1c"><ci id="S3.F10.36.15.m15.1.1.cmml" xref="S3.F10.36.15.m15.1.1">⟩</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.36.15.m15.1d">\rangle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.36.15.m15.1e">⟩</annotation></semantics></math>)=0 and <math alttext="\mathbb{IG}" class="ltx_Math" display="inline" id="S3.F10.37.16.m16.1"><semantics id="S3.F10.37.16.m16.1b"><mrow id="S3.F10.37.16.m16.1.1" xref="S3.F10.37.16.m16.1.1.cmml"><mi id="S3.F10.37.16.m16.1.1.2" xref="S3.F10.37.16.m16.1.1.2.cmml">𝕀</mi><mo id="S3.F10.37.16.m16.1.1.1" xref="S3.F10.37.16.m16.1.1.1.cmml">⁢</mo><mi id="S3.F10.37.16.m16.1.1.3" xref="S3.F10.37.16.m16.1.1.3.cmml">𝔾</mi></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.37.16.m16.1c"><apply id="S3.F10.37.16.m16.1.1.cmml" xref="S3.F10.37.16.m16.1.1"><times id="S3.F10.37.16.m16.1.1.1.cmml" xref="S3.F10.37.16.m16.1.1.1"></times><ci id="S3.F10.37.16.m16.1.1.2.cmml" xref="S3.F10.37.16.m16.1.1.2">𝕀</ci><ci id="S3.F10.37.16.m16.1.1.3.cmml" xref="S3.F10.37.16.m16.1.1.3">𝔾</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.37.16.m16.1d">\mathbb{IG}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.37.16.m16.1e">blackboard_I blackboard_G</annotation></semantics></math>(<math alttext="\langle" class="ltx_Math" display="inline" id="S3.F10.38.17.m17.1"><semantics id="S3.F10.38.17.m17.1b"><mo id="S3.F10.38.17.m17.1.1" stretchy="false" xref="S3.F10.38.17.m17.1.1.cmml">⟨</mo><annotation-xml encoding="MathML-Content" id="S3.F10.38.17.m17.1c"><ci id="S3.F10.38.17.m17.1.1.cmml" xref="S3.F10.38.17.m17.1.1">⟨</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.38.17.m17.1d">\langle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.38.17.m17.1e">⟨</annotation></semantics></math>Black, D<math alttext="\rangle" class="ltx_Math" display="inline" id="S3.F10.39.18.m18.1"><semantics id="S3.F10.39.18.m18.1b"><mo id="S3.F10.39.18.m18.1.1" stretchy="false" xref="S3.F10.39.18.m18.1.1.cmml">⟩</mo><annotation-xml encoding="MathML-Content" id="S3.F10.39.18.m18.1c"><ci id="S3.F10.39.18.m18.1.1.cmml" xref="S3.F10.39.18.m18.1.1">⟩</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.39.18.m18.1d">\rangle</annotation><annotation encoding="application/x-llamapun" id="S3.F10.39.18.m18.1e">⟩</annotation></semantics></math>)=1 means the nodes <math alttext="B,D" class="ltx_Math" display="inline" id="S3.F10.40.19.m19.2"><semantics id="S3.F10.40.19.m19.2b"><mrow id="S3.F10.40.19.m19.2.3.2" xref="S3.F10.40.19.m19.2.3.1.cmml"><mi id="S3.F10.40.19.m19.1.1" xref="S3.F10.40.19.m19.1.1.cmml">B</mi><mo id="S3.F10.40.19.m19.2.3.2.1" xref="S3.F10.40.19.m19.2.3.1.cmml">,</mo><mi id="S3.F10.40.19.m19.2.2" xref="S3.F10.40.19.m19.2.2.cmml">D</mi></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.40.19.m19.2c"><list id="S3.F10.40.19.m19.2.3.1.cmml" xref="S3.F10.40.19.m19.2.3.2"><ci id="S3.F10.40.19.m19.1.1.cmml" xref="S3.F10.40.19.m19.1.1">𝐵</ci><ci id="S3.F10.40.19.m19.2.2.cmml" xref="S3.F10.40.19.m19.2.2">𝐷</ci></list></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.40.19.m19.2d">B,D</annotation><annotation encoding="application/x-llamapun" id="S3.F10.40.19.m19.2e">italic_B , italic_D</annotation></semantics></math> and the predicate “Black” have influential impacts on the user’s next move. Finally, for each user, we compute the user-item interaction <math alttext="\mathbb{X}" class="ltx_Math" display="inline" id="S3.F10.41.20.m20.1"><semantics id="S3.F10.41.20.m20.1b"><mi id="S3.F10.41.20.m20.1.1" xref="S3.F10.41.20.m20.1.1.cmml">𝕏</mi><annotation-xml encoding="MathML-Content" id="S3.F10.41.20.m20.1c"><ci id="S3.F10.41.20.m20.1.1.cmml" xref="S3.F10.41.20.m20.1.1">𝕏</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.41.20.m20.1d">\mathbb{X}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.41.20.m20.1e">blackboard_X</annotation></semantics></math> and recommend items to him with the highest <math alttext="\mathbb{X}" class="ltx_Math" display="inline" id="S3.F10.42.21.m21.1"><semantics id="S3.F10.42.21.m21.1b"><mi id="S3.F10.42.21.m21.1.1" xref="S3.F10.42.21.m21.1.1.cmml">𝕏</mi><annotation-xml encoding="MathML-Content" id="S3.F10.42.21.m21.1c"><ci id="S3.F10.42.21.m21.1.1.cmml" xref="S3.F10.42.21.m21.1.1">𝕏</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.42.21.m21.1d">\mathbb{X}</annotation><annotation encoding="application/x-llamapun" id="S3.F10.42.21.m21.1e">blackboard_X</annotation></semantics></math> values. </span></figcaption>
</figure>
<div class="ltx_para" id="S3.I5.i1.p3">
<p class="ltx_p" id="S3.I5.i1.p3.1">The performance of KGFlex is evaluated on three datasets from various domains, <em class="ltx_emph ltx_font_italic" id="S3.I5.i1.p3.1.1">Yahoo! Movies</em>, <em class="ltx_emph ltx_font_italic" id="S3.I5.i1.p3.1.2">MovieLens</em>, and <em class="ltx_emph ltx_font_italic" id="S3.I5.i1.p3.1.3">Facebook Books</em>. The experiments are designed to test the performance of KGFlex in terms of the Gini Index <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib68" title="">gastwirth1972estimation, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib69" title="">castells2021novelty, </a>)</cite>). KGFlex outperforms certain latent factor models such as kaHFM <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib70" title="">anelli2019make, </a>)</cite>, Item-kNN <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib71" title="">koren2010factor, </a>)</cite>, NeuMF <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib72" title="">he2017neural, </a>)</cite> and BPR-MF <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib67" title="">rendle2012bpr, </a>)</cite> by an average of 18%. It also surpasses other key metrics, such as Item Coverage <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib73" title="">adomavicius2011improving, </a>)</cite>, in the recommendations it generates. Additionally, it excels in metrics like ACLT <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib74" title="">abdollahpouri2019managing, </a>)</cite>, PopREO, and PopRSP <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib75" title="">zhu2020measuring, </a>)</cite>, which measure recommendation performance concerning the underrepresentation of rare items. It is occasionally outperformed only by kaHFM in top-10 recommendations.</p>
</div>
</li>
<li class="ltx_item" id="S3.I5.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I5.i2.p1">
<p class="ltx_p" id="S3.I5.i2.p1.1">Su et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib76" title="">su2022attention </a></cite> presents an attention-based KG representation learning framework, named DDKG, aimed at feature representation and selection to improve drug-drug interaction (DDI) prediction. This approach allows for end-to-end prediction of DDIs. We summarize the DDKG into the below four main parts:</p>
<ul class="ltx_itemize" id="S3.I5.i2.I1">
<li class="ltx_item" id="S3.I5.i2.I1.ix1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">a.</span>
<div class="ltx_para" id="S3.I5.i2.I1.ix1.p1">
<p class="ltx_p" id="S3.I5.i2.I1.ix1.p1.1"><em class="ltx_emph ltx_font_italic" id="S3.I5.i2.I1.ix1.p1.1.1">KG Construction:</em> The KG construction amalgamates the Simplified Molecular Input Line Entry System (SMILES), SMILES-associated triple facts, and entities such as proteins and diseases. For example, we have two drugs, A and B, and we integrate their SMILES sequences alongside their relationships (e.g., “targets”) with diseases into the KG.</p>
</div>
</li>
<li class="ltx_item" id="S3.I5.i2.I1.ix2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">b.</span>
<div class="ltx_para" id="S3.I5.i2.I1.ix2.p1">
<p class="ltx_p" id="S3.I5.i2.I1.ix2.p1.1"><em class="ltx_emph ltx_font_italic" id="S3.I5.i2.I1.ix2.p1.1.1">Drug Embedding Initialization:</em> DDKG uses an encoder-decoder layer to learn the initial embeddings of drug nodes, mainly from the SMILES sequences in the KG. This step transforms the SMILES sequences of drugs A and B into vector representations that capture their chemical structure and properties.</p>
</div>
</li>
<li class="ltx_item" id="S3.I5.i2.I1.ix3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">c.</span>
<div class="ltx_para" id="S3.I5.i2.I1.ix3.p1">
<p class="ltx_p" id="S3.I5.i2.I1.ix3.p1.1"><em class="ltx_emph ltx_font_italic" id="S3.I5.i2.I1.ix3.p1.1.1">Drug Representation Learning:</em> This part, consisting of three elements, serves as the key feature selection step in DDKG.</p>
<ul class="ltx_itemize" id="S3.I5.i2.I1.ix3.I1">
<li class="ltx_item" id="S3.I5.i2.I1.ix3.I1.ix1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">-</span>
<div class="ltx_para" id="S3.I5.i2.I1.ix3.I1.ix1.p1">
<p class="ltx_p" id="S3.I5.i2.I1.ix3.I1.ix1.p1.1"><span class="ltx_text ltx_font_italic" id="S3.I5.i2.I1.ix3.I1.ix1.p1.1.1">Neighborhood Sampling:</span> For each drug node, a fixed-size set of neighboring nodes is selected. The significance of each neighbor is determined by <em class="ltx_emph ltx_font_italic" id="S3.I5.i2.I1.ix3.I1.ix1.p1.1.2">attention weights</em>, which are calculated based on the embeddings of the nodes and the types of relationships among them. This step ensures only the most relevant neighbors (in terms of both graph structure and drug relationships) are considered for further computation.</p>
</div>
</li>
<li class="ltx_item" id="S3.I5.i2.I1.ix3.I1.ix2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">-</span>
<div class="ltx_para" id="S3.I5.i2.I1.ix3.I1.ix2.p1">
<p class="ltx_p" id="S3.I5.i2.I1.ix3.I1.ix2.p1.1"><span class="ltx_text ltx_font_italic" id="S3.I5.i2.I1.ix3.I1.ix2.p1.1.1">Information Propagation: </span> It involves calculating a weighted sum of the neighbor embeddings. The attention weights (calculated in the previous step) are used to determine how much each neighbor’s information should contribute to the drug node’s new representation. This ensures that more relevant neighbors have a bigger impact on the final representation.</p>
</div>
</li>
<li class="ltx_item" id="S3.I5.i2.I1.ix3.I1.ix3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">-</span>
<div class="ltx_para" id="S3.I5.i2.I1.ix3.I1.ix3.p1">
<p class="ltx_p" id="S3.I5.i2.I1.ix3.I1.ix3.p1.1"><span class="ltx_text ltx_font_italic" id="S3.I5.i2.I1.ix3.I1.ix3.p1.1.1">Information Aggregation:</span> The weighted sum of the neighbor embeddings is combined with the drug node’s initial embedding. A final global representation of a drug node is obtained.</p>
</div>
</li>
</ul>
</div>
</li>
<li class="ltx_item" id="S3.I5.i2.I1.ix4" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">d.</span>
<div class="ltx_para" id="S3.I5.i2.I1.ix4.p1">
<p class="ltx_p" id="S3.I5.i2.I1.ix4.p1.1"><em class="ltx_emph ltx_font_italic" id="S3.I5.i2.I1.ix4.p1.1.1">DDI Prediction:</em> For a queried pair of drugs, DDKG estimates their interaction probability by simply multiplying their final respective representations derived in <em class="ltx_emph ltx_font_italic" id="S3.I5.i2.I1.ix4.p1.1.2">c</em>.</p>
</div>
</li>
</ul>
</div>
</li>
<li class="ltx_item" id="S3.I5.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S3.I5.i3.p1">
<p class="ltx_p" id="S3.I5.i3.p1.1">In the work by Hsieh et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib77" title="">hsieh2021drug </a></cite>, a Graph Neural Networks (GNN) <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib78" title="">zhou2020graph, </a>)</cite> is employed to advance the feature selection (drug selection) process for COVID-19 treatment from a drug-target interaction network (see Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F11" title="Figure 11 ‣ 3rd item ‣ 3.5 Deep Learning-Advanced Network representation Learning ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">11</span></a>). The authors first constructed a COVID-19 KG (see the top-left region in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S3.F11" title="Figure 11 ‣ 3rd item ‣ 3.5 Deep Learning-Advanced Network representation Learning ‣ 3 Feature Selection on Knowledge Graphs ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">11</span></a>) and generated embeddings using a GNN. The method involves transferring knowledge from another drug repurposing KG (see top-right region) and learning high-dimensional embeddings for drugs that encapsulate the complex pharmacological characteristics of drugs (see middle region). By utilizing a ranking model informed by Bayesian pairwise ranking loss, this approach prioritizes potential drug candidates for downstream tasks such as gene set enrichment analysis (see middle-left region), Retrospective in vitro drug screening (see middle-right region), etc. Top 22 promising drugs including Aspirin, Acetaminophen and Teicoplanin are highlighted in the paper, demonstrating the rapid identification of candidate drugs for COVID‐19 treatment.</p>
</div>
<figure class="ltx_figure" id="S3.F11"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="569" id="S3.F11.g1" src="extracted/5682567/figs/GNN.png" width="1030"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F11.8.1.1" style="font-size:90%;">Figure 11</span>: </span><span class="ltx_text" id="S3.F11.9.2" style="font-size:90%;">Feature selection (drug selection) via GNN embedding and drug ranking. The authors first construct a <span class="ltx_text ltx_font_bold" id="S3.F11.9.2.1">COVID-19 KG</span> containing different types of nodes (including 3635 drugs) and interactions. The variational graph autoencoder with GraphSAGE messages passing <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib79" title="">kipf2016variational, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib80" title="">hamilton2017inductive, </a>)</cite>, a specific type of GNN, was used to derive the drug embedding (the grey squares in <span class="ltx_text ltx_font_bold" id="S3.F11.9.2.2">Feature Selection</span>) by transferring a drug repurposing KG (DRKG) <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib81" title="">zeng2020repurpose, </a>)</cite> to boost the representativeness. Initial drug ranking using Bayesian pairwise ranking loss is applied to rank and select possibly potent drugs out of all candidates, hence serving as a feature selection step. The model efficacy was demonstrated using different validations. For instance, the authors perform <span class="ltx_text ltx_font_bold" id="S3.F11.9.2.3">Genetic Validation</span> by identifying significant associations between SARS-CoV-2 and selected drugs. <span class="ltx_text ltx_font_bold" id="S3.F11.9.2.4">Drug Screening Validation</span> is also performed by retrospectively comparing selected drugs with effective drugs in various in vitro drug screening experiments. In the <span class="ltx_text ltx_font_bold" id="S3.F11.9.2.5">Population-based Validation</span>, the proposed method identified six drugs administered to the COVID-19 patients out of ten positive drugs that were effective in the electronic health records. In addition, <span class="ltx_text ltx_font_bold" id="S3.F11.9.2.6">Drug Combination Search</span> for improving the COVID-19 treatment efficacy is conducted on the selected drugs. All validation results testify the capability of the proposed method speeding up the discovery of candidate drugs for treating COVID-19.</span></figcaption>
</figure>
</li>
</ul>
</div>
</section>
<section class="ltx_subsection" id="S3.SS6">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.6 </span>Comparative Analysis of Different Approaches</h3>
<div class="ltx_para" id="S3.SS6.p1">
<p class="ltx_p" id="S3.SS6.p1.1">We evaluate the methodologies from referenced manuscripts, focusing on their advantages and disadvantages.</p>
</div>
<div class="ltx_para" id="S3.SS6.p2">
<p class="ltx_p" id="S3.SS6.p2.1"><span class="ltx_text ltx_font_bold" id="S3.SS6.p2.1.1">1. Search Algorithms</span>
Used in the Hadith Corpus KG <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib16" title="">mohammed2020edge </a></cite> with the ACO algorithm and in COPD diagnosis <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib36" title="">fang2019diagnosis </a></cite> with the CMFS-<math alttext="\eta" class="ltx_Math" display="inline" id="S3.SS6.p2.1.m1.1"><semantics id="S3.SS6.p2.1.m1.1a"><mi id="S3.SS6.p2.1.m1.1.1" xref="S3.SS6.p2.1.m1.1.1.cmml">η</mi><annotation-xml encoding="MathML-Content" id="S3.SS6.p2.1.m1.1b"><ci id="S3.SS6.p2.1.m1.1.1.cmml" xref="S3.SS6.p2.1.m1.1.1">𝜂</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.SS6.p2.1.m1.1c">\eta</annotation><annotation encoding="application/x-llamapun" id="S3.SS6.p2.1.m1.1d">italic_η</annotation></semantics></math> algorithm. These methods highlight the importance of selecting appropriate strategies based on specific dataset requirements.</p>
</div>
<div class="ltx_para" id="S3.SS6.p3">
<p class="ltx_p" id="S3.SS6.p3.1"><span class="ltx_text ltx_font_bold" id="S3.SS6.p3.1.1">2. Vector Embeddings</span>
This approach, exemplified by the DistMult Algorithm and FeaBI, moves away from explicit path searches to embedding entities in a continuous vector space. It captures deep semantic relationships, facilitating the identification of intricate patterns relevant to complex domains like drug discovery <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib43" title="">dorigo2019ant </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib40" title="">atef2022feature </a></cite>.</p>
</div>
<div class="ltx_para" id="S3.SS6.p4">
<p class="ltx_p" id="S3.SS6.p4.1"><span class="ltx_text ltx_font_bold" id="S3.SS6.p4.1.1">3. Similarity-based Methods</span>
These methods compare entities within a graph to identify similarities using metrics like cosine similarity or Jaccard index. They are beneficial for clustering or recommendation systems, as demonstrated by Ma et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib37" title="">ma2020knowledge </a></cite> in Android malware classification and Jaworsky et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib38" title="">jaworsky2023interrelated </a></cite> in health survey datasets.</p>
</div>
<div class="ltx_para" id="S3.SS6.p5">
<p class="ltx_p" id="S3.SS6.p5.1"><span class="ltx_text ltx_font_bold" id="S3.SS6.p5.1.1">4. Advanced Network Representation Learning</span>
Utilizes deep learning models to interpret and analyze KGs, capturing high-level data representations. Examples include KGFLEX for optimizing recommendation systems and DDKG for drug-drug interaction predictions, showcasing the power of GNN frameworks in feature selection <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib36" title="">fang2019diagnosis </a></cite>.</p>
</div>
<div class="ltx_para" id="S3.SS6.p6">
<p class="ltx_p" id="S3.SS6.p6.1"><span class="ltx_text ltx_font_bold" id="S3.SS6.p6.1.1">Comparison and Contrast</span>
Search algorithms and similarity-based methods provide direct, interpretable insights into KG structures, making them suitable for applications requiring clarity and precision. In contrast, vector embeddings and advanced network representation learning offer a nuanced understanding of data, identifying complex patterns and relationships. These latter methods are valuable for scenarios where data relationships are not straightforward, enabling flexible and powerful KG modeling for predictive analytics. The drug ranking technique by Hsieh et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib77" title="">hsieh2021drug </a></cite> demonstrates the intersection of vector embeddings and advanced network learning, highlighting their transformative potential in feature selection.</p>
</div>
<figure class="ltx_table" id="S3.T1">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table"><span class="ltx_text" id="S3.T1.2.1.1" style="font-size:90%;">Table 1</span>: </span><span class="ltx_text" id="S3.T1.3.2" style="font-size:90%;">Comparison of Feature Selection Methods for KGs</span></figcaption>
<table class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle" id="S3.T1.4">
<thead class="ltx_thead">
<tr class="ltx_tr" id="S3.T1.4.1.1">
<th class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_tt" id="S3.T1.4.1.1.1">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.1.1.1.1">
<span class="ltx_p" id="S3.T1.4.1.1.1.1.1"><span class="ltx_text ltx_font_bold" id="S3.T1.4.1.1.1.1.1.1">Method</span></span>
</span>
</th>
<th class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_tt" id="S3.T1.4.1.1.2">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.1.1.2.1">
<span class="ltx_p" id="S3.T1.4.1.1.2.1.1"><span class="ltx_text ltx_font_bold" id="S3.T1.4.1.1.2.1.1.1">Pros</span></span>
</span>
</th>
<th class="ltx_td ltx_nopad_r ltx_align_justify ltx_th ltx_th_column ltx_border_tt" id="S3.T1.4.1.1.3">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.1.1.3.1">
<span class="ltx_p" id="S3.T1.4.1.1.3.1.1"><span class="ltx_text ltx_font_bold" id="S3.T1.4.1.1.3.1.1.1">Cons</span></span>
</span>
</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="S3.T1.4.2.1">
<td class="ltx_td ltx_align_justify ltx_border_t" id="S3.T1.4.2.1.1">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.2.1.1.1">
<span class="ltx_p" id="S3.T1.4.2.1.1.1.1"><span class="ltx_text ltx_font_bold" id="S3.T1.4.2.1.1.1.1.1">Search Algorithms</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_border_t" id="S3.T1.4.2.1.2">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.2.1.2.1">
<span class="ltx_p" id="S3.T1.4.2.1.2.1.1">Efficient and precise in known domains. Straightforward implementation.</span>
</span>
</td>
<td class="ltx_td ltx_nopad_r ltx_align_justify ltx_border_t" id="S3.T1.4.2.1.3">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.2.1.3.1">
<span class="ltx_p" id="S3.T1.4.2.1.3.1.1">May miss novel connections. Less adaptive to new patterns.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S3.T1.4.3.2">
<td class="ltx_td ltx_align_justify" id="S3.T1.4.3.2.1">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.3.2.1.1">
<span class="ltx_p" id="S3.T1.4.3.2.1.1.1"><span class="ltx_text ltx_font_bold" id="S3.T1.4.3.2.1.1.1.1">Vector Embeddings</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify" id="S3.T1.4.3.2.2">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.3.2.2.1">
<span class="ltx_p" id="S3.T1.4.3.2.2.1.1">Captures deep semantic relationships. Scalable to large KGs. Enhances predictive power.</span>
</span>
</td>
<td class="ltx_td ltx_nopad_r ltx_align_justify" id="S3.T1.4.3.2.3">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.3.2.3.1">
<span class="ltx_p" id="S3.T1.4.3.2.3.1.1">Challenges in interpretability. High initial training cost.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S3.T1.4.4.3">
<td class="ltx_td ltx_align_justify" id="S3.T1.4.4.3.1">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.4.3.1.1">
<span class="ltx_p" id="S3.T1.4.4.3.1.1.1"><span class="ltx_text ltx_font_bold" id="S3.T1.4.4.3.1.1.1.1">Similarity-based Methods</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify" id="S3.T1.4.4.3.2">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.4.3.2.1">
<span class="ltx_p" id="S3.T1.4.4.3.2.1.1">Easy to understand. Efficient for clustering/recommendations.</span>
</span>
</td>
<td class="ltx_td ltx_nopad_r ltx_align_justify" id="S3.T1.4.4.3.3">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.4.3.3.1">
<span class="ltx_p" id="S3.T1.4.4.3.3.1.1">Reliant on similarity metric quality. Computational challenges with large KGs.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S3.T1.4.5.4">
<td class="ltx_td ltx_align_justify ltx_border_bb" id="S3.T1.4.5.4.1">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.5.4.1.1">
<span class="ltx_p" id="S3.T1.4.5.4.1.1.1"><span class="ltx_text ltx_font_bold" id="S3.T1.4.5.4.1.1.1.1">Advanced Network Representation Learning</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_border_bb" id="S3.T1.4.5.4.2">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.5.4.2.1">
<span class="ltx_p" id="S3.T1.4.5.4.2.1.1">Learns complex representations. Integrates heterogeneous data. Versatile in application.</span>
</span>
</td>
<td class="ltx_td ltx_nopad_r ltx_align_justify ltx_border_bb" id="S3.T1.4.5.4.3">
<span class="ltx_inline-block ltx_align_top" id="S3.T1.4.5.4.3.1">
<span class="ltx_p" id="S3.T1.4.5.4.3.1.1">Computationally intensive. Complex model structure.</span>
</span>
</td>
</tr>
</tbody>
</table>
</figure>
</section>
</section>
<section class="ltx_section" id="S4">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Challenges and Opportunities in Knowledge Graph Feature Selection</h2>
<div class="ltx_para" id="S4.p1">
<p class="ltx_p" id="S4.p1.1">Knowledge Graphs (KGs) are transforming data-driven fields like biomedical research, bioinformatics, and recommendation systems. They offer significant analytical capabilities but also present challenges and opportunities, especially in feature selection for machine learning models.</p>
</div>
<section class="ltx_subsection" id="S4.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Challenges</h3>
<div class="ltx_para" id="S4.SS1.p1">
<p class="ltx_p" id="S4.SS1.p1.1">Feature selection in KGs faces several hurdles:</p>
</div>
<div class="ltx_para" id="S4.SS1.p2">
<ol class="ltx_enumerate" id="S4.I1">
<li class="ltx_item" id="S4.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span>
<div class="ltx_para" id="S4.I1.i1.p1">
<p class="ltx_p" id="S4.I1.i1.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I1.i1.p1.1.1">High Dimensionality and Complexity:</span> KGs encompass numerous entities and relationships, creating high-dimensional spaces that challenge traditional feature selection methods.</p>
</div>
</li>
<li class="ltx_item" id="S4.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span>
<div class="ltx_para" id="S4.I1.i2.p1">
<p class="ltx_p" id="S4.I1.i2.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I1.i2.p1.1.1">Data Heterogeneity:</span> KGs integrate diverse data types (numerical, categorical, textual) from various sources, necessitating robust feature selection techniques.</p>
</div>
</li>
<li class="ltx_item" id="S4.I1.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span>
<div class="ltx_para" id="S4.I1.i3.p1">
<p class="ltx_p" id="S4.I1.i3.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I1.i3.p1.1.1">Interpretability:</span> Enhancing interpretability is crucial, especially in fields like healthcare, where understanding why features are selected is essential.</p>
</div>
</li>
</ol>
</div>
</section>
<section class="ltx_subsection" id="S4.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>Future Directions</h3>
<div class="ltx_para" id="S4.SS2.p1">
<p class="ltx_p" id="S4.SS2.p1.1">Several promising research avenues could redefine KG feature selection:</p>
</div>
<div class="ltx_para" id="S4.SS2.p2">
<ul class="ltx_itemize" id="S4.I2">
<li class="ltx_item" id="S4.I2.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i1.p1">
<p class="ltx_p" id="S4.I2.i1.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i1.p1.1.1">Causal Inference Techniques:</span> Applying causal inference techniques to KGs can refine feature selection strategies <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib30" title="">malec2023causal, </a>)</cite>.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i2.p1">
<p class="ltx_p" id="S4.I2.i2.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i2.p1.1.1">Embedding KGs into Feature Matrices:</span> Creating feature matrices from KGs facilitates downstream tasks and enhances model performance <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib82" title="">strande2017evaluating, </a>)</cite>.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i3.p1">
<p class="ltx_p" id="S4.I2.i3.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i3.p1.1.1">Novel Algorithms:</span> Exploring algorithms like Ant Colony Optimization (ACO) introduces new approaches to feature selection within KGs <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib43" title="">dorigo2019ant, </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib40" title="">atef2022feature, </a>)</cite>.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i4" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i4.p1">
<p class="ltx_p" id="S4.I2.i4.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i4.p1.1.1">Multi-objective Optimization:</span> Using multi-objective optimization techniques offers a refined methodology for feature selection, balancing criteria like redundancy and relevance <cite class="ltx_cite ltx_citemacro_citep">(<a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib83" title="">mouret2015illuminating, </a>)</cite>.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i5" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i5.p1">
<p class="ltx_p" id="S4.I2.i5.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i5.p1.1.1">Interdisciplinary Integration:</span> Combining KGs with quantum computing, reinforcement learning (RL), and federated learning (FL) can enhance feature selection. Quantum-enhanced selection addresses scalability, RL refines the process based on feedback, and FL enables decentralized selection, preserving privacy <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib84" title="">ma2021quantum </a>; <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib85" title="">huang2022fedcke </a></cite>.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i6" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i6.p1">
<p class="ltx_p" id="S4.I2.i6.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i6.p1.1.1">Semantic Enrichment and XAI:</span> Leveraging the semantic information in KGs and applying Explainable AI principles can improve feature selection and model interpretability.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i7" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i7.p1">
<p class="ltx_p" id="S4.I2.i7.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i7.p1.1.1">Domain Knowledge Integration:</span> Integrating domain-specific knowledge into the feature selection process results in more effective selections, particularly in specialized fields like genomics and pharmacology.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i8" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i8.p1">
<p class="ltx_p" id="S4.I2.i8.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i8.p1.1.1">Multi-modal Data Fusion:</span> Combining various data sources into KGs offers a holistic view and unlocks new insights and applications.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i9" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i9.p1">
<p class="ltx_p" id="S4.I2.i9.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i9.p1.1.1">Dynamic KGs and Real-time Feature Selection:</span> Developing methods for real-time feature selection as KGs evolve can lead to more agile models, critical in rapidly changing domains like social media analysis.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i10" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i10.p1">
<p class="ltx_p" id="S4.I2.i10.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i10.p1.1.1">Collaborative KG Frameworks:</span> Creating frameworks for sharing and integrating KGs can enhance feature diversity and quality, fostering standardized protocols and benchmarks.</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i11" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="S4.I2.i11.p1">
<p class="ltx_p" id="S4.I2.i11.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I2.i11.p1.1.1">Ethical Considerations:</span> Prioritizing ethical considerations and bias mitigation in KG feature selection ensures fairness and equity in applications.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section class="ltx_section" id="S5">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Conclusion</h2>
<div class="ltx_para" id="S5.p1">
<p class="ltx_p" id="S5.p1.1">Examining these methodologies underscores the importance of scalability, accuracy, and interpretability in feature selection processes. As KGs grow, developing scalable algorithms that efficiently process large-scale KGs without losing information granularity is paramount. This requires a balanced approach that leverages KGs’ rich semantic relationships while addressing computational challenges.</p>
</div>
</section>
<section class="ltx_section" id="Sx1">
<h2 class="ltx_title ltx_title_section">Key Points of the Paper</h2>
<div class="ltx_para" id="Sx1.p1">
<ul class="ltx_itemize" id="Sx1.I1">
<li class="ltx_item" id="Sx1.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Sx1.I1.i1.p1">
<p class="ltx_p" id="Sx1.I1.i1.p1.1">Emphasizes combining feature selection techniques with KGs to enhance predictive modeling in biomedical research.</p>
</div>
</li>
<li class="ltx_item" id="Sx1.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Sx1.I1.i2.p1">
<p class="ltx_p" id="Sx1.I1.i2.p1.1">Shows significant applications in bioinformatics, improving disease prediction and drug discovery processes.</p>
</div>
</li>
<li class="ltx_item" id="Sx1.I1.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Sx1.I1.i3.p1">
<p class="ltx_p" id="Sx1.I1.i3.p1.1">Discusses challenges like computational complexity and the need for comprehensive KGs, proposing future research to develop efficient algorithms and integrate additional data sources.</p>
</div>
</li>
</ul>
</div>
</section>
<section class="ltx_section" id="S6">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Funding</h2>
<div class="ltx_para" id="S6.p1">
<p class="ltx_p" id="S6.p1.1">This work was funded by the National Institutes of Health (NIH) [U01 AG066833].</p>
</div>
</section>
<section class="ltx_section" id="Sx2">
<h2 class="ltx_title ltx_title_section">Appendix</h2>
<section class="ltx_subsection" id="Sx2.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.1 </span>Appendix A. Table of Acronyms</h3>
<div class="ltx_para" id="Sx2.SS1.p1">
<p class="ltx_p" id="Sx2.SS1.p1.1">Table 2 lists the Table of Acronyms for this paper.</p>
</div>
<figure class="ltx_table" id="Sx2.T2">
<table class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle" id="Sx2.T2.2">
<thead class="ltx_thead">
<tr class="ltx_tr" id="Sx2.T2.2.1.1">
<th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_r ltx_border_tt" id="Sx2.T2.2.1.1.1">Abbreviation</th>
<th class="ltx_td ltx_align_justify ltx_align_top ltx_th ltx_th_column ltx_border_tt" id="Sx2.T2.2.1.1.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.1.1.2.1">
<span class="ltx_p" id="Sx2.T2.2.1.1.2.1.1" style="width:156.5pt;">Definition</span>
</span>
</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="Sx2.T2.2.2.1">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r ltx_border_t" id="Sx2.T2.2.2.1.1">ACLT</th>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Sx2.T2.2.2.1.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.2.1.2.1">
<span class="ltx_p" id="Sx2.T2.2.2.1.2.1.1" style="width:156.5pt;">Average Coverage of Long Tail items</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.3.2">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.3.2.1">ACO</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.3.2.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.3.2.2.1">
<span class="ltx_p" id="Sx2.T2.2.3.2.2.1.1" style="width:156.5pt;">Ant Colony Optimization</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.4.3">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.4.3.1">AD</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.4.3.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.4.3.2.1">
<span class="ltx_p" id="Sx2.T2.2.4.3.2.1.1" style="width:156.5pt;">Alzheimer’s Disease</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.5.4">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.5.4.1">ADKG</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.5.4.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.5.4.2.1">
<span class="ltx_p" id="Sx2.T2.2.5.4.2.1.1" style="width:156.5pt;">Alzheimer’s Disease Knowledge Graph</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.6.5">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.6.5.1">AI</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.6.5.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.6.5.2.1">
<span class="ltx_p" id="Sx2.T2.2.6.5.2.1.1" style="width:156.5pt;">Artificial Intelligence</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.7.6">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.7.6.1">AlzKb</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.7.6.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.7.6.2.1">
<span class="ltx_p" id="Sx2.T2.2.7.6.2.1.1" style="width:156.5pt;">Alzheimer’s Disease Knowledge Base</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.8.7">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.8.7.1">APOE</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.8.7.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.8.7.2.1">
<span class="ltx_p" id="Sx2.T2.2.8.7.2.1.1" style="width:156.5pt;">Apolipoprotein E</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.9.8">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.9.8.1">AUC</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.9.8.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.9.8.2.1">
<span class="ltx_p" id="Sx2.T2.2.9.8.2.1.1" style="width:156.5pt;">Area Under the Curve</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.10.9">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.10.9.1">Bi-LSTM</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.10.9.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.10.9.2.1">
<span class="ltx_p" id="Sx2.T2.2.10.9.2.1.1" style="width:156.5pt;">Bidirectional Long Short-Term Memory</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.11.10">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.11.10.1">BPR</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.11.10.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.11.10.2.1">
<span class="ltx_p" id="Sx2.T2.2.11.10.2.1.1" style="width:156.5pt;">Bayesian Personalized Ranking</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.12.11">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.12.11.1">COPD</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.12.11.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.12.11.2.1">
<span class="ltx_p" id="Sx2.T2.2.12.11.2.1.1" style="width:156.5pt;">Chronic Obstructive Pulmonary Disease</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.13.12">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.13.12.1">CYP2D6</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.13.12.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.13.12.2.1">
<span class="ltx_p" id="Sx2.T2.2.13.12.2.1.1" style="width:156.5pt;">Cytochrome P450 2D6</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.14.13">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.14.13.1">DDI</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.14.13.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.14.13.2.1">
<span class="ltx_p" id="Sx2.T2.2.14.13.2.1.1" style="width:156.5pt;">drug-drug interaction</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.15.14">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.15.14.1">DistMult</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.15.14.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.15.14.2.1">
<span class="ltx_p" id="Sx2.T2.2.15.14.2.1.1" style="width:156.5pt;">The Distributed Multinomial Method</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.16.15">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.16.15.1">DL</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.16.15.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.16.15.2.1">
<span class="ltx_p" id="Sx2.T2.2.16.15.2.1.1" style="width:156.5pt;">Deep Learning</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.17.16">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.17.16.1">DR</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.17.16.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.17.16.2.1">
<span class="ltx_p" id="Sx2.T2.2.17.16.2.1.1" style="width:156.5pt;">Dimensionality/Dimension Reduction</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.18.17">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.18.17.1">DSA-SVM</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.18.17.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.18.17.2.1">
<span class="ltx_p" id="Sx2.T2.2.18.17.2.1.1" style="width:156.5pt;">Direct Search Simulated Annealing with Support Vector Machine</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.19.18">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.19.18.1">DTP</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.19.18.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.19.18.2.1">
<span class="ltx_p" id="Sx2.T2.2.19.18.2.1.1" style="width:156.5pt;">Drug-target Pairs</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.20.19">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.20.19.1">GDB</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.20.19.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.20.19.2.1">
<span class="ltx_p" id="Sx2.T2.2.20.19.2.1.1" style="width:156.5pt;">Graph Database</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.21.20">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.21.20.1">GNN</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.21.20.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.21.20.2.1">
<span class="ltx_p" id="Sx2.T2.2.21.20.2.1.1" style="width:156.5pt;">Graph Neural Network</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.22.21">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.22.21.1">HMOX1</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.22.21.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.22.21.2.1">
<span class="ltx_p" id="Sx2.T2.2.22.21.2.1.1" style="width:156.5pt;">Heme Oxygenase 1</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.23.22">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.23.22.1">KEGG</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.23.22.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.23.22.2.1">
<span class="ltx_p" id="Sx2.T2.2.23.22.2.1.1" style="width:156.5pt;">Kyoto Encyclopedia of Genes and Genomes</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.24.23">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.24.23.1">KG</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.24.23.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.24.23.2.1">
<span class="ltx_p" id="Sx2.T2.2.24.23.2.1.1" style="width:156.5pt;">Knowledge Graph</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.25.24">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.25.24.1">LDA</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.25.24.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.25.24.2.1">
<span class="ltx_p" id="Sx2.T2.2.25.24.2.1.1" style="width:156.5pt;">Linear Discriminant Analysis</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.26.25">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.26.25.1">LLE</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.26.25.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.26.25.2.1">
<span class="ltx_p" id="Sx2.T2.2.26.25.2.1.1" style="width:156.5pt;">Local Linear Embedding</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.27.26">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.27.26.1">ML</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.27.26.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.27.26.2.1">
<span class="ltx_p" id="Sx2.T2.2.27.26.2.1.1" style="width:156.5pt;">Machine Learning</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.28.27">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.28.27.1">MLP</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.28.27.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.28.27.2.1">
<span class="ltx_p" id="Sx2.T2.2.28.27.2.1.1" style="width:156.5pt;">Multiple Layer Perceptron</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.29.28">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.29.28.1">MQL</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.29.28.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.29.28.2.1">
<span class="ltx_p" id="Sx2.T2.2.29.28.2.1.1" style="width:156.5pt;">Metaweb Query Language</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.30.29">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.30.29.1">MTHFR</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.30.29.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.30.29.2.1">
<span class="ltx_p" id="Sx2.T2.2.30.29.2.1.1" style="width:156.5pt;">Methylenetetrahydrofolate Reductase</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.31.30">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.31.30.1">RDF</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.31.30.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.31.30.2.1">
<span class="ltx_p" id="Sx2.T2.2.31.30.2.1.1" style="width:156.5pt;">Resource Description Framework</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.32.31">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.32.31.1">RFE</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.32.31.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.32.31.2.1">
<span class="ltx_p" id="Sx2.T2.2.32.31.2.1.1" style="width:156.5pt;">Recursive Feature Elimination</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.33.32">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.33.32.1">nDCG</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.33.32.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.33.32.2.1">
<span class="ltx_p" id="Sx2.T2.2.33.32.2.1.1" style="width:156.5pt;">Normalized Discount Cumulative Gain</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.34.33">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.34.33.1">NLP</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.34.33.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.34.33.2.1">
<span class="ltx_p" id="Sx2.T2.2.34.33.2.1.1" style="width:156.5pt;">Natural Language Processing</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.35.34">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.35.34.1">NOS3</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.35.34.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.35.34.2.1">
<span class="ltx_p" id="Sx2.T2.2.35.34.2.1.1" style="width:156.5pt;">Nitric Oxide Synthase 3</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.36.35">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.36.35.1">OWL</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.36.35.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.36.35.2.1">
<span class="ltx_p" id="Sx2.T2.2.36.35.2.1.1" style="width:156.5pt;">The Web Ontology Language</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.37.36">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.37.36.1">PCA</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.37.36.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.37.36.2.1">
<span class="ltx_p" id="Sx2.T2.2.37.36.2.1.1" style="width:156.5pt;">Principal Component Analysis</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.38.37">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.38.37.1">PPARG</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.38.37.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.38.37.2.1">
<span class="ltx_p" id="Sx2.T2.2.38.37.2.1.1" style="width:156.5pt;">Peroxisome Proliferator-Activated Receptor Gamma</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.39.38">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.39.38.1">RDF</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.39.38.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.39.38.2.1">
<span class="ltx_p" id="Sx2.T2.2.39.38.2.1.1" style="width:156.5pt;">Resource Description Framework</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.40.39">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.40.39.1">RFE</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.40.39.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.40.39.2.1">
<span class="ltx_p" id="Sx2.T2.2.40.39.2.1.1" style="width:156.5pt;">Recursive Feature Elimination</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.41.40">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.41.40.1">RO</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.41.40.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.41.40.2.1">
<span class="ltx_p" id="Sx2.T2.2.41.40.2.1.1" style="width:156.5pt;">Relation Ontology</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.42.41">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.42.41.1">TPI1</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.42.41.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.42.41.2.1">
<span class="ltx_p" id="Sx2.T2.2.42.41.2.1.1" style="width:156.5pt;">Triosephosphate Isomerase 1</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.43.42">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.43.42.1">URIs</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.43.42.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.43.42.2.1">
<span class="ltx_p" id="Sx2.T2.2.43.42.2.1.1" style="width:156.5pt;">Uniform Resource Identifiers</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.44.43">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.44.43.1">UMLS</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.44.43.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.44.43.2.1">
<span class="ltx_p" id="Sx2.T2.2.44.43.2.1.1" style="width:156.5pt;">Unified Medical Language System</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.45.44">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_r" id="Sx2.T2.2.45.44.1">W3C</th>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Sx2.T2.2.45.44.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.45.44.2.1">
<span class="ltx_p" id="Sx2.T2.2.45.44.2.1.1" style="width:156.5pt;">World Wide Web Consortium</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Sx2.T2.2.46.45">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_bb ltx_border_r" id="Sx2.T2.2.46.45.1">YAGO</th>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Sx2.T2.2.46.45.2">
<span class="ltx_inline-block ltx_align_top" id="Sx2.T2.2.46.45.2.1">
<span class="ltx_p" id="Sx2.T2.2.46.45.2.1.1" style="width:156.5pt;">Yet Another Great Ontology</span>
</span>
</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table"><span class="ltx_text" id="Sx2.T2.3.1.1" style="font-size:90%;">Table 2</span>: </span><span class="ltx_text" id="Sx2.T2.4.2" style="font-size:90%;">Table of Acronyms</span></figcaption>
</figure>
</section>
<section class="ltx_subsection" id="Sx2.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.2 </span>Appendix B. A more detailed description of KGs of sizes tiny, small, and medium</h3>
<div class="ltx_para" id="Sx2.SS2.p1">
<p class="ltx_p" id="Sx2.SS2.p1.1">Within each of the three graphs(see Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.F2" title="Figure 2 ‣ 2.4 Visual Demonstration of ADKGs of Varying Sizes ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">2</span></a>, Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F12" title="Figure 12 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">12</span></a>, and Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F13" title="Figure 13 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">13</span></a>), the nodes and their connections are represented by distinct colors and arrow types to convey different biological relationships:</p>
</div>
<div class="ltx_para" id="Sx2.SS2.p2">
<p class="ltx_p" id="Sx2.SS2.p2.1">Orange (see Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F12" title="Figure 12 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">12</span></a> and Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F13" title="Figure 13 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">13</span></a>) and Yellow (see Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.F2" title="Figure 2 ‣ 2.4 Visual Demonstration of ADKGs of Varying Sizes ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">2</span></a>) nodes represent the disease entity, with AD positioned as the central node, highlighting it as the primary focus of this network.</p>
</div>
<div class="ltx_para" id="Sx2.SS2.p3">
<p class="ltx_p" id="Sx2.SS2.p3.1">Purple nodes signify genes, which are implicated in AD through various associations such as genetic risk factors, differential gene expression, or other genetic interactions.</p>
</div>
<div class="ltx_para" id="Sx2.SS2.p4">
<p class="ltx_p" id="Sx2.SS2.p4.1">Green nodes denote chemicals, encompassing drugs, vitamins, or other bioactive molecules. These external agents are potential modulators of gene function or disease pathology.</p>
</div>
<div class="ltx_para" id="Sx2.SS2.p5">
<ul class="ltx_itemize" id="Sx2.I1">
<li class="ltx_item" id="Sx2.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Sx2.I1.i1.p1">
<p class="ltx_p" id="Sx2.I1.i1.p1.1">There are five instances of the “Chemical binds gene” relationship (light purple arrows in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.F2" title="Figure 2 ‣ 2.4 Visual Demonstration of ADKGs of Varying Sizes ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">2</span></a> and coffee arrows in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F12" title="Figure 12 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">12</span></a> and Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F13" title="Figure 13 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">13</span></a>), where a chemical is shown to interact directly with a gene. This does not necessarily indicate an increase or decrease in gene expression, but rather a physical or functional interaction. For example, one of the edges indicates folic acid, a form of vitamin B that is vital for making DNA and other genetic material, binds the MTHFR gene. MTHFR plays a crucial role in processing amino acids, the building blocks of proteins. Variants of this gene can affect homocysteine levels in the blood. Deficiencies in folic acid are linked to elevated homocysteine levels, which may increase AD risk.</p>
</div>
</li>
<li class="ltx_item" id="Sx2.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Sx2.I1.i2.p1">
<p class="ltx_p" id="Sx2.I1.i2.p1.1">There are six instances of the “Gene associates with disease” relationship (yellow arrows in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#S2.F2" title="Figure 2 ‣ 2.4 Visual Demonstration of ADKGs of Varying Sizes ‣ 2 Background and Key Concepts ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">2</span></a> and red arrows in Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F12" title="Figure 12 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">12</span></a> and Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F13" title="Figure 13 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">13</span></a>), representing genes that have an association with AD. These relationships might represent genetic risk factors, genes involved in the pathology of the disease, or genes that could be potential targets for therapeutic intervention. For instance, the NOS3 gene is associated with AD. It is involved in the generation of nitric oxide, a molecule that aids in blood vessel dilation. Impairment in NOS3 function can affect blood flow in the brain, potentially impacting Alzheimer’s disease pathology.</p>
</div>
</li>
<li class="ltx_item" id="Sx2.I1.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Sx2.I1.i3.p1">
<p class="ltx_p" id="Sx2.I1.i3.p1.1">There are three instances of the
“Chemical increases expression”
relationship (pink arrows), denoting chemicals that are known to upregulate or increase the expression of certain genes. For instance, Vitamin A increases the expression of HMOX1, a gene-encoding enzyme in response to oxidative stress, which is a contributing factor in neuronal damage observed in AD.</p>
</div>
</li>
<li class="ltx_item" id="Sx2.I1.i4" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Sx2.I1.i4.p1">
<p class="ltx_p" id="Sx2.I1.i4.p1.1">There is one instance of the
“Chemical decreases expression”
relationship (green arrow), indicating a chemical that downregulates or decreases the expression of a gene. Namely, Cyclosporine, an immunosuppressant that may inhibit the formation of amyloid plaques, a hallmark of AD, decreases the expression of TPI1, an enzyme that plays a crucial role in glycolysis, a metabolic pathway that occurs in the cytoplasm of cells.</p>
</div>
</li>
<li class="ltx_item" id="Sx2.I1.i5" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Sx2.I1.i5.p1">
<p class="ltx_p" id="Sx2.I1.i5.p1.1">There is one instance of “Gene regulates gene” (purple arrow), suggesting a regulatory interaction between two genes, PPARG and TPI1. For context, PPARG is a gene that codes for a protein that regulates fatty acid storage and glucose metabolism. It is a target for some drugs that might influence Alzheimer’s disease progression.</p>
</div>
</li>
</ul>
</div>
<div class="ltx_para" id="Sx2.SS2.p6">
<p class="ltx_p" id="Sx2.SS2.p6.1">Figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F12" title="Figure 12 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">12</span></a> provides an example of small-sized ADKG with 23 nodes and 32 edges (setting the Cypher limit clause to 15) and figure <a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#Sx2.F13" title="Figure 13 ‣ 6.2 Appendix B. A more detailed description of KGs of sizes tiny, small, and medium ‣ Appendix ‣ A review of Feature Selection on Knowledge Graphs"><span class="ltx_text ltx_ref_tag">13</span></a> provides an example of medium-sized ADKG with 156 nodes and 288 edges (setting the Cypher limit clause to 200). In addition to the relationship types described above, the medium-sized ADKG also demonstrates the “DRUGTREATDISEASE” (gold arrows) and “GENEINTERACTSWITHGENE” (brown arrows) relationships. As the size of KGs continues to expand, the challenge of comprehending the intricate web of entities and relationships within them becomes daunting for human observers. Consequently, there arises an urgent need for the development of sophisticated computational tools capable of effectively managing these vast KGs.</p>
</div>
<figure class="ltx_figure" id="Sx2.F12">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" height="109" id="Sx2.F12.g1" src="extracted/5682567/smallKGexample.png" width="223"/></div>
<div class="ltx_flex_cell ltx_flex_size_2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" height="100" id="Sx2.F12.g2" src="extracted/5682567/SmallDesc.jpg" width="363"/></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="Sx2.F12.2.1.1" style="font-size:90%;">Figure 12</span>: </span><span class="ltx_text" id="Sx2.F12.3.2" style="font-size:90%;">A Small-sized ADKG (Orange Node: AD; Purple Nodes: Genes; Green Nodes: Drugs) <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib28" title="">alzheimersknowledgebase </a></cite></span></figcaption>
</figure>
<figure class="ltx_figure" id="Sx2.F13">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" height="110" id="Sx2.F13.g1" src="extracted/5682567/mediumKGexample.png" width="223"/></div>
<div class="ltx_flex_cell ltx_flex_size_2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" height="116" id="Sx2.F13.g2" src="extracted/5682567/MediumDesc.jpg" width="354"/></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="Sx2.F13.2.1.1" style="font-size:90%;">Figure 13</span>: </span><span class="ltx_text" id="Sx2.F13.3.2" style="font-size:90%;">A Medium-sized ADKG (Orange Node: AD; Purple Nodes: Genes; Green Nodes: Drugs) <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2406.14864v1#bib.bib28" title="">alzheimersknowledgebase </a></cite></span></figcaption>
</figure>
</section>
</section>
<section class="ltx_bibliography" id="bib">
<h2 class="ltx_title ltx_title_bibliography">References</h2>
<ul class="ltx_biblist">
<li class="ltx_bibitem" id="bib.bib1">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">
Janneth Chicaiza and Priscila Valdiviezo-Diaz.

</span>
<span class="ltx_bibblock">A comprehensive survey of knowledge graph-based recommender systems: Technologies, development, and contributions.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib1.1.1">Information</span>, 12(6):232, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib2">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">
François Belleau, Marc-Alexandre Nolin, Nicole Tourigny, Philippe Rigault, and Jean Morissette.

</span>
<span class="ltx_bibblock">Bio2rdf: towards a mashup to build bioinformatics knowledge systems.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib2.1.1">Journal of biomedical informatics</span>, 41(5):706–716, 2008.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib3">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">
SM Shamimul Hasan, Donna Rivera, Xiao-Cheng Wu, Eric B Durbin, J Blair Christian, and Georgia Tourassi.

</span>
<span class="ltx_bibblock">Knowledge graph-enabled cancer data analytics.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib3.1.1">IEEE journal of biomedical and health informatics</span>, 24(7):1952–1967, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib4">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">
Dieter Fensel, Umutcan Şimşek, Kevin Angele, Elwin Huaman, Elias Kärle, Oleksandra Panasiuk, Ioan Toma, Jürgen Umbrich, Alexander Wahler, Dieter Fensel, et al.

</span>
<span class="ltx_bibblock">Introduction: what is a knowledge graph?

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib4.1.1">Knowledge graphs: Methodology, tools and selected use cases</span>, pages 1–10, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib5">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">
Stephen Bonner, Ian P Barrett, Cheng Ye, Rowan Swiers, Ola Engkvist, Charles Tapley Hoyt, and William L Hamilton.

</span>
<span class="ltx_bibblock">Understanding the performance of knowledge graph embeddings in drug discovery.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib5.1.1">Artificial Intelligence in the Life Sciences</span>, 2:100036, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib6">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">
Yang Yang, Yuwei Lu, and Wenying Yan.

</span>
<span class="ltx_bibblock">A comprehensive review on knowledge graphs for complex diseases.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib6.1.1">Briefings in Bioinformatics</span>, 24(1):bbac543, 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib7">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">
Beth Levine and Guido Kroemer.

</span>
<span class="ltx_bibblock">Biological functions of autophagy genes: a disease perspective.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib7.1.1">Cell</span>, 176(1):11–42, 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib8">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">
Payal Chandak, Kexin Huang, and Marinka Zitnik.

</span>
<span class="ltx_bibblock">Building a knowledge graph to enable precision medicine.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib8.1.1">Scientific Data</span>, 10(1):67, 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib9">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">
Richard Bellman.

</span>
<span class="ltx_bibblock">Dynamic programming.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib9.1.1">Science</span>, 153(3731):34–37, 1966.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib10">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">
Artur J Ferreira and Mário AT Figueiredo.

</span>
<span class="ltx_bibblock">Efficient feature selection filters for high-dimensional data.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib10.1.1">Pattern recognition letters</span>, 33(13):1794–1804, 2012.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib11">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock">
Salim Lahmiri.

</span>
<span class="ltx_bibblock">Features selection, data mining and finacial risk classification: a comparative study.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib11.1.1">Intelligent Systems in Accounting, Finance and Management</span>, 23(4):265–275, 2016.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib12">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock">
Shamsul Huda, John Yearwood, Herbert F Jelinek, Mohammad Mehedi Hassan, Giancarlo Fortino, and Michael Buckland.

</span>
<span class="ltx_bibblock">A hybrid feature selection with ensemble classification for imbalanced healthcare data: A case study for brain tumor diagnosis.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib12.1.1">IEEE access</span>, 4:9145–9154, 2016.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib13">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock">
Malcolm R Forster.

</span>
<span class="ltx_bibblock">Key concepts in model selection: Performance and generalizability.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib13.1.1">Journal of mathematical psychology</span>, 44(1):205–231, 2000.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib14">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock">
Pasi Saari, Tuomas Eerola, and Olivier Lartillot.

</span>
<span class="ltx_bibblock">Generalizability and simplicity as criteria in feature selection: Application to mood classification in music.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib14.1.1">IEEE Transactions on audio, speech, and language processing</span>, 19(6):1802–1812, 2010.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib15">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock">
Thulasi Bikku, N Sambasiva Rao, and Ananda Rao Akepogu.

</span>
<span class="ltx_bibblock">Hadoop based feature selection and decision making models on big data.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib15.1.1">Indian Journal of Science and Technology</span>, 9(10):1–6, 2016.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib16">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock">
Bushra Mohammed, Mosab Hamdan, Joseph Stephen Bassi, Haitham A Jamil, Suleman Khan, Abdallah Elhigazi, Danda B Rawat, Ismahani Binti Ismail, and Muhammad Nadzir Marsono.

</span>
<span class="ltx_bibblock">Edge computing intelligence using robust feature selection for network traffic classification in internet-of-things.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib16.1.1">IEEE Access</span>, 8:224059–224070, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib17">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock">
Alan Jović, Karla Brkić, and Nikola Bogunović.

</span>
<span class="ltx_bibblock">A review of feature selection methods with applications.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib17.1.1">2015 38th international convention on information and communication technology, electronics and microelectronics (MIPRO)</span>, pages 1200–1205. Ieee, 2015.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib18">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock">
Steffen Staab and Rudi Studer.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib18.1.1">Handbook on ontologies</span>.

</span>
<span class="ltx_bibblock">Springer Science &amp; Business Media, 2010.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib19">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock">
David S Wishart, Yannick D Feunang, An C Guo, Elvis J Lo, Ana Marcu, Jason R Grant, Tanvir Sajed, Daniel Johnson, Carin Li, Zinat Sayeeda, et al.

</span>
<span class="ltx_bibblock">Drugbank 5.0: a major update to the drugbank database for 2018.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib19.1.1">Nucleic acids research</span>, 46(D1):D1074–D1082, 2018.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib20">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock">
Michael Kuhn, Ivica Letunic, Lars Juhl Jensen, and Peer Bork.

</span>
<span class="ltx_bibblock">The sider database of drugs and side effects.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib20.1.1">Nucleic acids research</span>, 44(D1):D1075–D1079, 2016.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib21">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock">
Minoru Kanehisa and Susumu Goto.

</span>
<span class="ltx_bibblock">Kegg: kyoto encyclopedia of genes and genomes.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib21.1.1">Nucleic acids research</span>, 28(1):27–30, 2000.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib22">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[22]</span>
<span class="ltx_bibblock">
Olivier Bodenreider.

</span>
<span class="ltx_bibblock">The unified medical language system (umls): integrating biomedical terminology.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib22.1.1">Nucleic acids research</span>, 32(suppl_1):D267–D270, 2004.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib23">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[23]</span>
<span class="ltx_bibblock">
Kevin Donnelly et al.

</span>
<span class="ltx_bibblock">Snomed-ct: The advanced terminology and coding system for ehealth.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib23.1.1">Studies in health technology and informatics</span>, 121:279, 2006.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib24">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[24]</span>
<span class="ltx_bibblock">
Stuart J Nelson, Kelly Zeng, John Kilbourne, Tammy Powell, and Robin Moore.

</span>
<span class="ltx_bibblock">Normalized names for clinical drugs: Rxnorm at 6 years.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib24.1.1">Journal of the American Medical Informatics Association</span>, 18(4):441–448, 2011.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib25">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[25]</span>
<span class="ltx_bibblock">
Justin J Miller.

</span>
<span class="ltx_bibblock">Graph database applications and concepts with neo4j.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib25.1.1">Proceedings of the southern association for information systems conference, Atlanta, GA, USA</span>, volume 2324, pages 141–147, 2013.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib26">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[26]</span>
<span class="ltx_bibblock">
Kurt Bollacker, Colin Evans, Praveen Paritosh, Tim Sturge, and Jamie Taylor.

</span>
<span class="ltx_bibblock">Freebase: a collaboratively created graph database for structuring human knowledge.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib26.1.1">Proceedings of the 2008 ACM SIGMOD international conference on Management of data</span>, pages 1247–1250, 2008.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib27">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[27]</span>
<span class="ltx_bibblock">
Nadime Francis, Alastair Green, Paolo Guagliardo, Leonid Libkin, Tobias Lindaaker, Victor Marsault, Stefan Plantikow, Mats Rydberg, Petra Selmer, and Andrés Taylor.

</span>
<span class="ltx_bibblock">Cypher: An evolving query language for property graphs.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib27.1.1">Proceedings of the 2018 international conference on management of data</span>, pages 1433–1445, 2018.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib28">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[28]</span>
<span class="ltx_bibblock">
Joseph D. Romano, Van Truong, Rachit Kumar, Mythreye Venkatesan, Britney E. Graham, Yun Hao, Nick Matsumoto, Xi Li, Zhiping Wang, Marylyn Ritchie, Li Shen, and Jason H. Moore.

</span>
<span class="ltx_bibblock">The alzheimer’s knowledge base - a knowledge graph for therapeutic discovery in alzheimer’s disease research.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib28.1.1">Preprint</span>, February 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib29">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[29]</span>
<span class="ltx_bibblock">
Alessio Zanga, Elif Ozkirimli, and Fabio Stella.

</span>
<span class="ltx_bibblock">A survey on causal discovery: Theory and practice.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib29.1.1">International Journal of Approximate Reasoning</span>, 151:101–129, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib30">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[30]</span>
<span class="ltx_bibblock">
Scott A Malec, Sanya B Taneja, Steven M Albert, C Elizabeth Shaaban, Helmet T Karim, Arthur S Levine, Paul Munro, Tiffany J Callahan, and Richard D Boyce.

</span>
<span class="ltx_bibblock">Causal feature selection using a knowledge graph combining structured knowledge from the biomedical literature and ontologies: a use case studying depression as a risk factor for alzheimer’s disease.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib30.1.1">Journal of Biomedical Informatics</span>, 142:104368, 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib31">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[31]</span>
<span class="ltx_bibblock">
Terry Callahan.

</span>
<span class="ltx_bibblock">Pheknowlator.

</span>
<span class="ltx_bibblock"><a class="ltx_ref ltx_url ltx_font_typewriter" href="https://doi.org/10.5281/zenodo.3401437" title="">https://doi.org/10.5281/zenodo.3401437</a>, 2019.

</span>
<span class="ltx_bibblock">Available online at Zenodo.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib32">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[32]</span>
<span class="ltx_bibblock">
Benjamin M Gyori, John A Bachman, Kartik Subramanian, Jeremy L Muhlich, Lucian Galescu, and Peter K Sorger.

</span>
<span class="ltx_bibblock">From word models to executable models of signaling networks using automated assembly.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib32.1.1">Molecular systems biology</span>, 13(11):954, 2017.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib33">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[33]</span>
<span class="ltx_bibblock">
Jorge Pérez, Marcelo Arenas, and Claudio Gutierrez.

</span>
<span class="ltx_bibblock">Semantics and complexity of sparql.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib33.1.1">ACM Transactions on Database Systems (TODS)</span>, 34(3):1–45, 2009.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib34">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[34]</span>
<span class="ltx_bibblock">
Ian Horrocks, Peter F Patel-Schneider, Sean Bechhofer, and Dmitry Tsarkov.

</span>
<span class="ltx_bibblock">Owl rules: A proposal and prototype implementation.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib34.1.1">Journal of web semantics</span>, 3(1):23–40, 2005.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib35">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[35]</span>
<span class="ltx_bibblock">
Bob DuCharme.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib35.1.1">Learning SPARQL: querying and updating with SPARQL 1.1</span>.

</span>
<span class="ltx_bibblock">” O’Reilly Media, Inc.”, 2013.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib36">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[36]</span>
<span class="ltx_bibblock">
Youli Fang, Hong Wang, Lutong Wang, Ruitong Di, and Yongqiang Song.

</span>
<span class="ltx_bibblock">Diagnosis of copd based on a knowledge graph and integrated model.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib36.1.1">IEEE Access</span>, 7:46004–46013, 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib37">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[37]</span>
<span class="ltx_bibblock">
Duoyuan Ma, Yude Bai, Zhenchang Xing, Lintan Sun, and Xiaohong Li.

</span>
<span class="ltx_bibblock">A knowledge graph-based sensitive feature selection for android malware classification.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib37.1.1">2020 27th Asia-Pacific Software Engineering Conference (APSEC)</span>, pages 188–197, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib38">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[38]</span>
<span class="ltx_bibblock">
Markian Jaworsky, Xiaohui Tao, Lei Pan, Shiva Raj Pokhrel, Jianming Yong, and Ji Zhang.

</span>
<span class="ltx_bibblock">Interrelated feature selection from health surveys using domain knowledge graph.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib38.1.1">Health Information Science and Systems</span>, 11(1):54, 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib39">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[39]</span>
<span class="ltx_bibblock">
Carol Pierannunzi, Shaohua Sean Hu, and Lina Balluz.

</span>
<span class="ltx_bibblock">A systematic review of publications assessing reliability and validity of the behavioral risk factor surveillance system (brfss), 2004–2011.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib39.1.1">BMC medical research methodology</span>, 13(1):1–14, 2013.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib40">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[40]</span>
<span class="ltx_bibblock">
Mohamed Atef Mosa.

</span>
<span class="ltx_bibblock">Feature selection based on aco and knowledge graph for arabic text classification.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib40.1.1">Journal of Experimental &amp; Theoretical Artificial Intelligence</span>, pages 1–18, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib41">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[41]</span>
<span class="ltx_bibblock">
Marco Dorigo and Christian Blum.

</span>
<span class="ltx_bibblock">Ant colony optimization theory: A survey.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib41.1.1">Theoretical computer science</span>, 344(2-3):243–278, 2005.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib42">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[42]</span>
<span class="ltx_bibblock">
Marco Dorigo, Mauro Birattari, and Thomas Stutzle.

</span>
<span class="ltx_bibblock">Ant colony optimization.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib42.1.1">IEEE computational intelligence magazine</span>, 1(4):28–39, 2006.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib43">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[43]</span>
<span class="ltx_bibblock">
Marco Dorigo and Thomas Stützle.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib43.1.1">Ant colony optimization: overview and recent advances</span>.

</span>
<span class="ltx_bibblock">Springer, 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib44">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[44]</span>
<span class="ltx_bibblock">
Rafael S Parpinelli, Heitor S Lopes, and Alex Alves Freitas.

</span>
<span class="ltx_bibblock">Data mining with an ant colony optimization algorithm.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib44.1.1">IEEE transactions on evolutionary computation</span>, 6(4):321–332, 2002.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib45">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[45]</span>
<span class="ltx_bibblock">
David Martens, Manu De Backer, Raf Haesen, Jan Vanthienen, Monique Snoeck, and Bart Baesens.

</span>
<span class="ltx_bibblock">Classification with ant colony optimization.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib45.1.1">IEEE transactions on evolutionary computation</span>, 11(5):651–665, 2007.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib46">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[46]</span>
<span class="ltx_bibblock">
Mehdi Hosseinzadeh Aghdam, Nasser Ghasem-Aghaee, and Mohammad Ehsan Basiri.

</span>
<span class="ltx_bibblock">Text feature selection using ant colony optimization.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib46.1.1">Expert systems with applications</span>, 36(3):6843–6853, 2009.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib47">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[47]</span>
<span class="ltx_bibblock">
Aytuğ Onan.

</span>
<span class="ltx_bibblock">Srl-aco: A text augmentation framework based on semantic role labeling and ant colony optimization.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib47.1.1">Journal of King Saud University-Computer and Information Sciences</span>, page 101611, 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib48">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[48]</span>
<span class="ltx_bibblock">
Hyejung Chang.

</span>
<span class="ltx_bibblock">Making sense of the big picture: Data linkage and integration in the era of big data.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib48.1.1">Healthcare Informatics Research</span>, 24(4):251, 2018.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib49">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[49]</span>
<span class="ltx_bibblock">
Li Li, Haolin Yang, Yueming Jiao, and Kuo-Yi Lin.

</span>
<span class="ltx_bibblock">Feature generation based on knowledge graph.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib49.1.1">IFAC-PapersOnLine</span>, 53(5):774–779, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib50">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[50]</span>
<span class="ltx_bibblock">
Sören Auer, Christian Bizer, Georgi Kobilarov, Jens Lehmann, Richard Cyganiak, and Zachary Ives.

</span>
<span class="ltx_bibblock">Dbpedia: A nucleus for a web of open data.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib50.1.1">international semantic web conference</span>, pages 722–735. Springer, 2007.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib51">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[51]</span>
<span class="ltx_bibblock">
Bishan Yang, Wen-tau Yih, Xiaodong He, Jianfeng Gao, and Li Deng.

</span>
<span class="ltx_bibblock">Embedding entities and relations for learning and inference in knowledge bases.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib51.1.1">arXiv preprint arXiv:1412.6575</span>, 2014.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib52">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[52]</span>
<span class="ltx_bibblock">
Théo Trouillon, Johannes Welbl, Sebastian Riedel, Éric Gaussier, and Guillaume Bouchard.

</span>
<span class="ltx_bibblock">Complex embeddings for simple link prediction.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib52.1.1">International conference on machine learning</span>, pages 2071–2080. PMLR, 2016.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib53">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[53]</span>
<span class="ltx_bibblock">
Antoine Bordes, Nicolas Usunier, Alberto Garcia-Duran, Jason Weston, and Oksana Yakhnenko.

</span>
<span class="ltx_bibblock">Translating embeddings for modeling multi-relational data.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib53.1.1">Advances in neural information processing systems</span>, 26, 2013.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib54">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[54]</span>
<span class="ltx_bibblock">
Maximilian Nickel, Volker Tresp, Hans-Peter Kriegel, et al.

</span>
<span class="ltx_bibblock">A three-way model for collective learning on multi-relational data.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib54.1.1">Icml</span>, volume 11, pages 3104482–3104584, 2011.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib55">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[55]</span>
<span class="ltx_bibblock">
Youmna Ismaeil, Daria Stepanova, Trung-Kien Tran, and Hendrik Blockeel.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib55.1.1">FeaBI: A Feature Selection-Based Framework for Interpreting KG Embeddings</span>, pages 599–617.

</span>
<span class="ltx_bibblock">2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib56">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[56]</span>
<span class="ltx_bibblock">
Hongwei Wang, Fuzheng Zhang, Jialin Wang, Miao Zhao, Wenjie Li, Xing Xie, and Minyi Guo.

</span>
<span class="ltx_bibblock">Ripplenet: Propagating user preferences on the knowledge graph for recommender systems.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib56.1.1">Proceedings of the 27th ACM international conference on information and knowledge management</span>, pages 417–426, 2018.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib57">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[57]</span>
<span class="ltx_bibblock">
Ajay Kumar, Shashank Sheshar Singh, Kuldeep Singh, and Bhaskar Biswas.

</span>
<span class="ltx_bibblock">Link prediction techniques, applications, and performance: A survey.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib57.1.1">Physica A: Statistical Mechanics and its Applications</span>, 553:124289, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib58">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[58]</span>
<span class="ltx_bibblock">
Tareq Al-Moslmi, Marc Gallofré Ocaña, Andreas L Opdahl, and Csaba Veres.

</span>
<span class="ltx_bibblock">Named entity extraction for knowledge graphs: A literature overview.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib58.1.1">IEEE Access</span>, 8:32862–32881, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib59">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[59]</span>
<span class="ltx_bibblock">
Zhe Chen, Yuehan Wang, Bin Zhao, Jing Cheng, Xin Zhao, and Zongtao Duan.

</span>
<span class="ltx_bibblock">Knowledge graph completion: A review.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib59.1.1">Ieee Access</span>, 8:192435–192456, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib60">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[60]</span>
<span class="ltx_bibblock">
Shudong Wang, Zhenzhen Du, Mao Ding, Alfonso Rodriguez-Paton, and Tao Song.

</span>
<span class="ltx_bibblock">Kg-dti: a knowledge graph based deep learning method for drug-target interaction predictions and alzheimer’s disease drug repositions.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib60.1.1">Applied Intelligence</span>, 52(1):846–857, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib61">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[61]</span>
<span class="ltx_bibblock">
YueQun Wang, LiYan Dong, YongLi Li, and Hao Zhang.

</span>
<span class="ltx_bibblock">Multitask feature learning approach for knowledge graph enhanced recommendations with ripplenet.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib61.1.1">Plos one</span>, 16(5):e0251162, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib62">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[62]</span>
<span class="ltx_bibblock">
Vito Walter Anelli, Tommaso Di Noia, Eugenio Di Sciascio, Antonio Ferrara, and Alberto Carlo Maria Mancino.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib62.1.1">Sparse feature factorization for recommender systems with knowledge graphs</span>, pages 154–165.

</span>
<span class="ltx_bibblock">2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib63">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[63]</span>
<span class="ltx_bibblock">
Guy Shani and Asela Gunawardana.

</span>
<span class="ltx_bibblock">Evaluating recommendation systems.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib63.1.1">Recommender systems handbook</span>, pages 257–297, 2011.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib64">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[64]</span>
<span class="ltx_bibblock">
Zhanqiu Zhang, Jie Wang, Jiajun Chen, Shuiwang Ji, and Feng Wu.

</span>
<span class="ltx_bibblock">Cone: Cone embeddings for multi-hop reasoning over knowledge graphs.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib64.1.1">Advances in Neural Information Processing Systems</span>, 34:19172–19183, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib65">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[65]</span>
<span class="ltx_bibblock">
Claude Elwood Shannon.

</span>
<span class="ltx_bibblock">A mathematical theory of communication.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib65.1.1">The Bell system technical journal</span>, 27(3):379–423, 1948.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib66">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[66]</span>
<span class="ltx_bibblock">
Lior Rokach and Oded Maimon.

</span>
<span class="ltx_bibblock">Top-down induction of decision trees classifiers-a survey.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib66.1.1">IEEE Transactions on Systems, Man, and Cybernetics, Part C (Applications and Reviews)</span>, 35(4):476–487, 2005.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib67">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[67]</span>
<span class="ltx_bibblock">
Steffen Rendle, Christoph Freudenthaler, Zeno Gantner, and Lars Schmidt-Thieme.

</span>
<span class="ltx_bibblock">Bpr: Bayesian personalized ranking from implicit feedback.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib67.1.1">arXiv preprint arXiv:1205.2618</span>, 2012.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib68">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[68]</span>
<span class="ltx_bibblock">
Joseph L Gastwirth.

</span>
<span class="ltx_bibblock">The estimation of the lorenz curve and gini index.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib68.1.1">The review of economics and statistics</span>, pages 306–316, 1972.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib69">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[69]</span>
<span class="ltx_bibblock">
Pablo Castells, Neil Hurley, and Saul Vargas.

</span>
<span class="ltx_bibblock">Novelty and diversity in recommender systems.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib69.1.1">Recommender systems handbook</span>, pages 603–646. Springer, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib70">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[70]</span>
<span class="ltx_bibblock">
Vito Walter Anelli, Tommaso Di Noia, Eugenio Di Sciascio, Azzurra Ragone, and Joseph Trotta.

</span>
<span class="ltx_bibblock">How to make latent factors interpretable by feeding factorization machines with knowledge graphs.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib70.1.1">The Semantic Web–ISWC 2019: 18th International Semantic Web Conference, Auckland, New Zealand, October 26–30, 2019, Proceedings, Part I 18</span>, pages 38–56. Springer, 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib71">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[71]</span>
<span class="ltx_bibblock">
Yehuda Koren.

</span>
<span class="ltx_bibblock">Factor in the neighbors: Scalable and accurate collaborative filtering.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib71.1.1">ACM Transactions on Knowledge Discovery from Data (TKDD)</span>, 4(1):1–24, 2010.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib72">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[72]</span>
<span class="ltx_bibblock">
Xiangnan He and Tat-Seng Chua.

</span>
<span class="ltx_bibblock">Neural factorization machines for sparse predictive analytics.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib72.1.1">Proceedings of the 40th International ACM SIGIR conference on Research and Development in Information Retrieval</span>, pages 355–364, 2017.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib73">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[73]</span>
<span class="ltx_bibblock">
Gediminas Adomavicius and YoungOk Kwon.

</span>
<span class="ltx_bibblock">Improving aggregate recommendation diversity using ranking-based techniques.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib73.1.1">IEEE Transactions on Knowledge and Data Engineering</span>, 24(5):896–911, 2011.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib74">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[74]</span>
<span class="ltx_bibblock">
Himan Abdollahpouri, Robin Burke, and Bamshad Mobasher.

</span>
<span class="ltx_bibblock">Managing popularity bias in recommender systems with personalized re-ranking.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib74.1.1">arXiv preprint arXiv:1901.07555</span>, 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib75">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[75]</span>
<span class="ltx_bibblock">
Ziwei Zhu, Jianling Wang, and James Caverlee.

</span>
<span class="ltx_bibblock">Measuring and mitigating item under-recommendation bias in personalized ranking systems.

</span>
<span class="ltx_bibblock">In <span class="ltx_text ltx_font_italic" id="bib.bib75.1.1">Proceedings of the 43rd international ACM SIGIR conference on research and development in information retrieval</span>, pages 449–458, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib76">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[76]</span>
<span class="ltx_bibblock">
Xiaorui Su, Lun Hu, Zhuhong You, Pengwei Hu, and Bowei Zhao.

</span>
<span class="ltx_bibblock">Attention-based knowledge graph representation learning for predicting drug-drug interactions.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib76.1.1">Briefings in bioinformatics</span>, 23(3):bbac140, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib77">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[77]</span>
<span class="ltx_bibblock">
Kanglin Hsieh, Yinyin Wang, Luyao Chen, Zhongming Zhao, Sean Savitz, Xiaoqian Jiang, Jing Tang, and Yejin Kim.

</span>
<span class="ltx_bibblock">Drug repurposing for covid-19 using graph neural network and harmonizing multiple evidence.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib77.1.1">Scientific reports</span>, 11(1):23179, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib78">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[78]</span>
<span class="ltx_bibblock">
Jie Zhou, Ganqu Cui, Shengding Hu, Zhengyan Zhang, Cheng Yang, Zhiyuan Liu, Lifeng Wang, Changcheng Li, and Maosong Sun.

</span>
<span class="ltx_bibblock">Graph neural networks: A review of methods and applications.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib78.1.1">AI open</span>, 1:57–81, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib79">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[79]</span>
<span class="ltx_bibblock">
Thomas N Kipf and Max Welling.

</span>
<span class="ltx_bibblock">Variational graph auto-encoders.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib79.1.1">arXiv preprint arXiv:1611.07308</span>, 2016.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib80">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[80]</span>
<span class="ltx_bibblock">
Will Hamilton, Zhitao Ying, and Jure Leskovec.

</span>
<span class="ltx_bibblock">Inductive representation learning on large graphs.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib80.1.1">Advances in neural information processing systems</span>, 30, 2017.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib81">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[81]</span>
<span class="ltx_bibblock">
Xiangxiang Zeng, Xiang Song, Tengfei Ma, Xiaoqin Pan, Yadi Zhou, Yuan Hou, Zheng Zhang, Kenli Li, George Karypis, and Feixiong Cheng.

</span>
<span class="ltx_bibblock">Repurpose open data to discover therapeutics for covid-19 using deep learning.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib81.1.1">Journal of proteome research</span>, 19(11):4624–4636, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib82">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[82]</span>
<span class="ltx_bibblock">
Natasha T Strande, Erin Rooney Riggs, Adam H Buchanan, Ozge Ceyhan-Birsoy, Marina DiStefano, Selina S Dwight, Jenny Goldstein, Rajarshi Ghosh, Bryce A Seifert, Tam P Sneddon, et al.

</span>
<span class="ltx_bibblock">Evaluating the clinical validity of gene-disease associations: an evidence-based framework developed by the clinical genome resource.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib82.1.1">The American Journal of Human Genetics</span>, 100(6):895–906, 2017.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib83">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[83]</span>
<span class="ltx_bibblock">
Jean-Baptiste Mouret and Jeff Clune.

</span>
<span class="ltx_bibblock">Illuminating search spaces by mapping elites.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib83.1.1">arXiv preprint arXiv:1504.04909</span>, 2015.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib84">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[84]</span>
<span class="ltx_bibblock">
Yunpu Ma and Volker Tresp.

</span>
<span class="ltx_bibblock">Quantum machine learning algorithm for knowledge graphs.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib84.1.1">ACM Transactions on Quantum Computing</span>, 2(3):1–28, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib85">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[85]</span>
<span class="ltx_bibblock">
Wei Huang, Jia Liu, Tianrui Li, Shenggong Ji, Dexian Wang, and Tianqiang Huang.

</span>
<span class="ltx_bibblock">Fedcke: Cross-domain knowledge graph embedding in federated learning.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib85.1.1">IEEE Transactions on Big Data</span>, 2022.

</span>
</li>
</ul>
</section>
<div class="ltx_para" id="p5">
<span class="ltx_ERROR undefined" id="p5.1">{biography}</span>
<p class="ltx_p" id="p5.2">Sisi Shao is currently a PhD student at the Department of Biostatistics, UCLA Fielding School of Public Health. She received a M.Sc. degree in Financial Engineering from UCLA Anderson School of Management. Her research interest includes artificial intelligence, automated machine learning methods, and statistical methods for large-scale multivariate time series and longitudinal data.</p>
</div>
<div class="ltx_para" id="p6">
<span class="ltx_ERROR undefined" id="p6.1">{biography}</span>
<p class="ltx_p" id="p6.2">Pedro Henrique Ribeiro received a M.S.E. in Bioengineering from the University of Pennsylvania and a B.A. in Computer Science from Oberlin College. He is currently a research data scientist at the Cedars-Sinai Department of Computational Biomedicine. His main research interests are in machine learning and evolutionary algorithms.</p>
</div>
<div class="ltx_para" id="p7">
<span class="ltx_ERROR undefined" id="p7.1">{biography}</span>
<p class="ltx_p" id="p7.2">Christina Ramirez  received her Ph.D. degree in Statistics/Social Science from the California Institute of Technology, Pasadena, CA, USA, in 1999. She is currently a Professor of Biostatistics at the UCLA Fielding School of Public Health. Her main research interests include HIV pathogenesis, HIV drug resistance mutation/recombination, viral fitness, coreceptor utilization, and high-dimensional data analysis.</p>
</div>
<div class="ltx_para" id="p8">
<span class="ltx_ERROR undefined" id="p8.1">{biography}</span>
<p class="ltx_p" id="p8.2">Jason H. Moore is Chair of the Department of Computational Biomedicine at Cedars-Sinai Medical Center where he also serves as Director of the Center for Artificial Intelligence Research and Education. His research focuses on the development and application of artificial intelligence and machine learning methods for the analysis of biomedical and clinical data with the goal of improving health.</p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
</article>
</div>
<footer class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Fri Jun 21 04:49:06 2024 by <a class="ltx_LaTeXML_logo" href="http://dlmf.nist.gov/LaTeXML/"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img alt="Mascot Sammy" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg=="/></a>
</div></footer>
</div>
</body>
</html>
