<!DOCTYPE html>
<html lang="en">
<head>
<meta content="text/html; charset=utf-8" http-equiv="content-type"/>
<title>Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging</title>
<!--Generated on Tue Oct  1 22:12:04 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport"/>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv.0.7.9.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv-fonts.0.7.9.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/latexml_styles.css" rel="stylesheet" type="text/css"/>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/html2canvas/1.3.3/html2canvas.min.js"></script>
<script src="/static/browse/0.3.4/js/addons_new.js"></script>
<script src="/static/browse/0.3.4/js/feedbackOverlay.js"></script>
<meta content="Metastatic Breast Cancer IF Imaging CTCs Detection" lang="en" name="keywords"/>
<base href="/html/2410.02988v1/"/></head>
<body>
<nav class="ltx_page_navbar">
<nav class="ltx_TOC">
<ol class="ltx_toclist">
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S1" title="In Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1 </span>Introduction</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S2" title="In Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2 </span>Background</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3" title="In Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3 </span>Methods</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.SS1" title="In 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1 </span>Cell Detection</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.SS2" title="In 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.2 </span>Nuclear Segmentation</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.SS3" title="In 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3 </span>Cell Segmentation</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.SS4" title="In 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.4 </span>Feature Extraction</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S4" title="In Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4 </span>Experiments</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S4.SS1" title="In 4 Experiments ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.1 </span>Ground Truth Data</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S4.SS2" title="In 4 Experiments ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.2 </span>CTC Classifier Training</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S4.SS3" title="In 4 Experiments ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.3 </span>Results</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S5" title="In Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5 </span>Conclusion</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S5.SS0.SSS1" title="In 5 Conclusion ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5.0.1 </span>Acknowledgements</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S5.SS0.SSS2" title="In 5 Conclusion ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5.0.2 </span><span class="ltx_ERROR undefined">\discintname</span></span></a></li>
</ol>
</li>
</ol></nav>
</nav>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line"><span class="ltx_note ltx_role_institutetext" id="id1"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_note_type">institutetext: </span>Epic Sciences, San Diego, CA, USA</span></span></span>
<h1 class="ltx_title ltx_title_document">Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Evan Schwab
</span></span>
<span class="ltx_author_before">‚ÄÉ‚ÄÉ</span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Bharat Annaldas
</span></span>
<span class="ltx_author_before">‚ÄÉ‚ÄÉ</span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Nisha Ramesh
</span></span>
<span class="ltx_author_before">‚ÄÉ‚ÄÉ</span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Anna Lundberg
</span></span>
<span class="ltx_author_before">‚ÄÉ‚ÄÉ</span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Vishal Shelke
</span></span>
<span class="ltx_author_before">‚ÄÉ‚ÄÉ</span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Xinran Xu
</span></span>
<span class="ltx_author_before">‚ÄÉ‚ÄÉ</span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Cole Gilbertson
</span></span>
<span class="ltx_author_before">‚ÄÉ‚ÄÉ</span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Jiyun Byun
</span></span>
<span class="ltx_author_before">‚ÄÉ‚ÄÉ</span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Ernest T. Lam
</span></span>
</div>
<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p class="ltx_p" id="id2.2">Liquid biopsies (eg., blood draws) offer a less invasive and non-localized alternative to tissue biopsies for monitoring the progression of metastatic breast cancer (mBCa). Immunofluoresence (IF) microscopy is a tool to image and analyze millions of blood cells in a patient sample. By detecting and genetically sequencing circulating tumor cells (CTCs) in the blood, personalized treatment plans are achievable for various cancer subtypes. However, CTCs are rare (about 1 in 2M), making manual CTC detection very difficult. In addition, clinicians rely on quantitative cellular biomarkers to manually classify CTCs. This requires prior tasks of cell detection, segmentation and feature extraction. To assist clinicians, we have developed a fully automated machine learning-based production-level pipeline to efficiently detect, segment and classify CTCs in multi-channel IF images. We achieve over <math alttext="99\%" class="ltx_Math" display="inline" id="id1.1.m1.1"><semantics id="id1.1.m1.1a"><mrow id="id1.1.m1.1.1" xref="id1.1.m1.1.1.cmml"><mn id="id1.1.m1.1.1.2" xref="id1.1.m1.1.1.2.cmml">99</mn><mo id="id1.1.m1.1.1.1" xref="id1.1.m1.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="id1.1.m1.1b"><apply id="id1.1.m1.1.1.cmml" xref="id1.1.m1.1.1"><csymbol cd="latexml" id="id1.1.m1.1.1.1.cmml" xref="id1.1.m1.1.1.1">percent</csymbol><cn id="id1.1.m1.1.1.2.cmml" type="integer" xref="id1.1.m1.1.1.2">99</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="id1.1.m1.1c">99\%</annotation><annotation encoding="application/x-llamapun" id="id1.1.m1.1d">99 %</annotation></semantics></math> sensitivity and <math alttext="97\%" class="ltx_Math" display="inline" id="id2.2.m2.1"><semantics id="id2.2.m2.1a"><mrow id="id2.2.m2.1.1" xref="id2.2.m2.1.1.cmml"><mn id="id2.2.m2.1.1.2" xref="id2.2.m2.1.1.2.cmml">97</mn><mo id="id2.2.m2.1.1.1" xref="id2.2.m2.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="id2.2.m2.1b"><apply id="id2.2.m2.1.1.cmml" xref="id2.2.m2.1.1"><csymbol cd="latexml" id="id2.2.m2.1.1.1.cmml" xref="id2.2.m2.1.1.1">percent</csymbol><cn id="id2.2.m2.1.1.2.cmml" type="integer" xref="id2.2.m2.1.1.2">97</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="id2.2.m2.1c">97\%</annotation><annotation encoding="application/x-llamapun" id="id2.2.m2.1d">97 %</annotation></semantics></math> specificity on 9,533 cells from 15 mBCa patients. Our pipeline has been successfully deployed on real mBCa patients, reducing a patient average of 14M detected cells to only 335 CTC candidates for manual review.</p>
</div>
<div class="ltx_keywords">
<h6 class="ltx_title ltx_title_keywords">Keywords: </h6>Metastatic Breast Cancer IF Imaging CTCs Detection
</div>
<section class="ltx_section" id="S1">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>
<div class="ltx_para" id="S1.p1">
<p class="ltx_p" id="S1.p1.1">Liquid biopsies (eg., blood draws) offer a less invasive and non-localized alternative to tissue biopsies for a more continuous monitoring of metastatic breast cancer (mBCa). Immunofluoresence (IF) microscopy is a tool to image and analyze millions of cells in a sample of blood. Circulating tumor cells (CTCs) in the blood are indicative of metastasis and the combination of protein biomarker and single-cell genetic analyses of CTCs provides clinicians with a comprehensive cancer profile for mBCa patients from a single blood draw <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib16" title="">16</a>, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib18" title="">18</a>]</cite>.</p>
</div>
<div class="ltx_para" id="S1.p2">
<p class="ltx_p" id="S1.p2.1">In mBCa patients, CTCs average 1 in every 2M cells <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib18" title="">18</a>]</cite>, motivating the need for machine learning (ML) to reduce the burden of manual classification by automatically detecting and classifying CTCs. Deep learning has been developed for classifying CTCs in multiple IF modalities <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib1" title="">1</a>, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib2" title="">2</a>, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib8" title="">8</a>, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib12" title="">12</a>, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib15" title="">15</a>, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib20" title="">20</a>, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib21" title="">21</a>]</cite>. However, these methods fall short of providing a fully automated pipeline to detect and classify CTCs to present for manual review. Furthermore, deep learning loses the utility and interpretability of biomarker features, which clinicians already rely on for manual classification.</p>
</div>
<figure class="ltx_figure" id="S1.F1"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="482" id="S1.F1.g1" src="x1.png" width="1196"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S1.F1.2.1.1" style="font-size:90%;">Figure 1</span>: </span><span class="ltx_text" id="S1.F1.3.2" style="font-size:90%;">Example user interface of ML classified CTC candidates output by our pipeline and presented to clinicians (bottom row thumbnails). One thumbnail is selected for review and cell and nuclear MFI values reported per channel. The yellow nuclear mask is overlayed on each channel and the color composite. </span></figcaption>
</figure>
<div class="ltx_para" id="S1.p3">
<p class="ltx_p" id="S1.p3.1">In this work, we have developed a fully automated ML-based production-level pipeline to efficiently detect, segment, and classify mBCa CTCs in multi-channel IF images to present to clinicians for final confirmation and genetic sequencing. Our automated pipeline, which we call <span class="ltx_text ltx_font_bold" id="S1.p3.1.1">BR</span>east cancer <span class="ltx_text ltx_font_bold" id="S1.p3.1.2">I</span>maging <span class="ltx_text ltx_font_bold" id="S1.p3.1.3">A</span>lgorithm (BRIA), combines image processing, deep learning and interpretable feature-based ML. BRIA has been fully deployed at Epic Sciences within a comprehensive clinical workflow that integrates patient sample collection, slide preparation, fluorescence scanning, cloud-based analysis using the proposed pipeline, database management, quality control, and reporting through a proprietary clinical viewer that interacts with data sources through an API.</p>
</div>
<div class="ltx_para" id="S1.p4">
<p class="ltx_p" id="S1.p4.1">In Sec.¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S2" title="2 Background ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">2</span></a> we provide a background on IF imaging and the clinical workflow. Then, in Sec.¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3" title="3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">3</span></a> we present each component of the BRIA pipeline and share our ground truth labeling and experimental results in Sec.¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S4" title="4 Experiments ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">4</span></a>.</p>
</div>
</section>
<section class="ltx_section" id="S2">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Background</h2>
<div class="ltx_para" id="S2.p1">
<p class="ltx_p" id="S2.p1.1">Widefield fluorescence microscopy is an IF imaging technique used to capture 2D images of blood samples collected on whole slides <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib10" title="">10</a>, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib19" title="">19</a>]</cite>.
Fluorescent dyes are applied in an assay system to capture specific biomarkers of interest and are digitized in multiple imaging channels. At Epic Sciences, we have developed a clinical diagnostic test for mBCa, called DefineMBC, which uses IF imaging as one of the core components for protein biomarker expression analysis. The DefineMBC assay includes DAPI, CK, and CD45/31 channels which are designed to visualize cell components and provide inclusionary and exclusionary biomarkers for CTCs.<span class="ltx_note ltx_role_footnote" id="footnote1"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_tag ltx_tag_note">1</span>BRIA is agnostic to additional BCa biomarker channels like HER2 or ER.</span></span></span> The DAPI stain (4‚Äô,6-diamidino-2-phenylindole) is used to highlight the nucleus of the cell. The CK stain (Cytokeratin) is an indicator of CTCs and CD45/31<span class="ltx_note ltx_role_footnote" id="footnote2"><sup class="ltx_note_mark">2</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">2</sup><span class="ltx_tag ltx_tag_note">2</span>Cluster of differentiation (CD) 31 is a protein associated with white blood cells and endothelial cells. CD 45 is a protein marker of leukocyte lineage.</span></span></span> is an indicator of non-CTCs like white blood cells. Therefore, high CK and low CD45/31 values are indicative of CTCs (see Fig.¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S2.F2" title="Figure 2 ‚Ä£ 2 Background ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">2</span></a> for examples).</p>
</div>
<div class="ltx_para" id="S2.p2">
<p class="ltx_p" id="S2.p2.2">These channel values are summarized using statistical features such as the mean florescence intensity (MFI), ie. the average pixel intensity within an object like the cell or nucleus, for each channel (e.g., Nuclear CK MFI). To compute these features, each individual cell on a slide of roughly 3M cells must be detected and cropped as individual thumbnail images. Then the cell and nucleus need to be segmented to compute MFIs for each channel. In the absence of ML, the standard practice to classify CTCs is to first apply biomarker thresholds (eg. Nuclear CK MFI <math alttext="&gt;\!269" class="ltx_Math" display="inline" id="S2.p2.1.m1.1"><semantics id="S2.p2.1.m1.1a"><mrow id="S2.p2.1.m1.1.1" xref="S2.p2.1.m1.1.1.cmml"><mi id="S2.p2.1.m1.1.1.2" xref="S2.p2.1.m1.1.1.2.cmml"></mi><mo id="S2.p2.1.m1.1.1.1" rspace="0.108em" xref="S2.p2.1.m1.1.1.1.cmml">&gt;</mo><mn id="S2.p2.1.m1.1.1.3" xref="S2.p2.1.m1.1.1.3.cmml">269</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.p2.1.m1.1b"><apply id="S2.p2.1.m1.1.1.cmml" xref="S2.p2.1.m1.1.1"><gt id="S2.p2.1.m1.1.1.1.cmml" xref="S2.p2.1.m1.1.1.1"></gt><csymbol cd="latexml" id="S2.p2.1.m1.1.1.2.cmml" xref="S2.p2.1.m1.1.1.2">absent</csymbol><cn id="S2.p2.1.m1.1.1.3.cmml" type="integer" xref="S2.p2.1.m1.1.1.3">269</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.p2.1.m1.1c">&gt;\!269</annotation><annotation encoding="application/x-llamapun" id="S2.p2.1.m1.1d">&gt; 269</annotation></semantics></math> and Nuclear CD45/31 MFI <math alttext="\leq\!3000" class="ltx_Math" display="inline" id="S2.p2.2.m2.1"><semantics id="S2.p2.2.m2.1a"><mrow id="S2.p2.2.m2.1.1" xref="S2.p2.2.m2.1.1.cmml"><mi id="S2.p2.2.m2.1.1.2" xref="S2.p2.2.m2.1.1.2.cmml"></mi><mo id="S2.p2.2.m2.1.1.1" rspace="0.108em" xref="S2.p2.2.m2.1.1.1.cmml">‚â§</mo><mn id="S2.p2.2.m2.1.1.3" xref="S2.p2.2.m2.1.1.3.cmml">3000</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.p2.2.m2.1b"><apply id="S2.p2.2.m2.1.1.cmml" xref="S2.p2.2.m2.1.1"><leq id="S2.p2.2.m2.1.1.1.cmml" xref="S2.p2.2.m2.1.1.1"></leq><csymbol cd="latexml" id="S2.p2.2.m2.1.1.2.cmml" xref="S2.p2.2.m2.1.1.2">absent</csymbol><cn id="S2.p2.2.m2.1.1.3.cmml" type="integer" xref="S2.p2.2.m2.1.1.3">3000</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.p2.2.m2.1c">\leq\!3000</annotation><annotation encoding="application/x-llamapun" id="S2.p2.2.m2.1d">‚â§ 3000</annotation></semantics></math>, etc.) to filter 99% of non-CTCs and artefacts in our data. However, validating biomarker thresholds is a challenging task and clinicians are still left with 10‚Äôs of thousands of CTC candidates per patient to sort through.</p>
</div>
<figure class="ltx_figure" id="S2.F2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="715" id="S2.F2.g1" src="x2.png" width="1196"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S2.F2.2.1.1" style="font-size:90%;">Figure 2</span>: </span><span class="ltx_text" id="S2.F2.3.2" style="font-size:90%;">Example CTCs, non-CTCs, and artefacts in 3-channel IF images with a color composite. CTCs have prominent CK signal and low CD45/31. Some Non-CTCs are visually similar which makes manual classification difficult.</span></figcaption>
</figure>
<div class="ltx_para" id="S2.p3">
<p class="ltx_p" id="S2.p3.1">Within this workflow, clinicians manually classify CTCs based on MFI values and the visual inspection of features like cell size, shape, texture and other biomarkers. Because clinicians already rely heuristically on a list of qualitative cellular features, we are motivated to quantify these biomarkers and utilize interpretable feature-based ML instead of deep learning to classify CTCs.</p>
</div>
<div class="ltx_para" id="S2.p4">
<p class="ltx_p" id="S2.p4.1">The BRIA pipeline assists clinicians by automating these steps of cell detection, nuclear and cell segmentation, feature extraction and CTC classification to present CTC candidates to clinicians for final review. Figure¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S1.F1" title="Figure 1 ‚Ä£ 1 Introduction ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">1</span></a> shows a user interface of CTC candidates presented to clinicians for manual review. For this slide example, the pipeline reduced 3M total cells to 48 CTC candidates instead of what would be 10s of thousands of candidates to sort through.</p>
</div>
</section>
<section class="ltx_section" id="S3">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Methods</h2>
<div class="ltx_para" id="S3.p1">
<p class="ltx_p" id="S3.p1.1">We outline the main steps of the BRIA pipeline leading up to CTC classification in Sec¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.SS1" title="3.1 Cell Detection ‚Ä£ 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">3.1</span></a> cell detection, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.SS2" title="3.2 Nuclear Segmentation ‚Ä£ 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">3.2</span></a> nucleus segmentation, <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.SS3" title="3.3 Cell Segmentation ‚Ä£ 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">3.3</span></a> cell segmentation, and <a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.SS4" title="3.4 Feature Extraction ‚Ä£ 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">3.4</span></a> feature extraction. These steps are also required for manual classification in the absence of an ML CTC classifier. See Figure¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S3.F3" title="Figure 3 ‚Ä£ 3.2 Nuclear Segmentation ‚Ä£ 3 Methods ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">3</span></a> for a visual overview.</p>
</div>
<section class="ltx_subsection" id="S3.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.1 </span>Cell Detection</h3>
<div class="ltx_para" id="S3.SS1.p1">
<p class="ltx_p" id="S3.SS1.p1.2">The first step of BRIA is to efficiently detect each individual cell center and crop thumbnail images for each cell. Whole slides are too large to image fully and so a grid of 588 (<math alttext="14\!\times\!42" class="ltx_Math" display="inline" id="S3.SS1.p1.1.m1.1"><semantics id="S3.SS1.p1.1.m1.1a"><mrow id="S3.SS1.p1.1.m1.1.1" xref="S3.SS1.p1.1.m1.1.1.cmml"><mn id="S3.SS1.p1.1.m1.1.1.2" xref="S3.SS1.p1.1.m1.1.1.2.cmml">14</mn><mo id="S3.SS1.p1.1.m1.1.1.1" lspace="0.052em" rspace="0.052em" xref="S3.SS1.p1.1.m1.1.1.1.cmml">√ó</mo><mn id="S3.SS1.p1.1.m1.1.1.3" xref="S3.SS1.p1.1.m1.1.1.3.cmml">42</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS1.p1.1.m1.1b"><apply id="S3.SS1.p1.1.m1.1.1.cmml" xref="S3.SS1.p1.1.m1.1.1"><times id="S3.SS1.p1.1.m1.1.1.1.cmml" xref="S3.SS1.p1.1.m1.1.1.1"></times><cn id="S3.SS1.p1.1.m1.1.1.2.cmml" type="integer" xref="S3.SS1.p1.1.m1.1.1.2">14</cn><cn id="S3.SS1.p1.1.m1.1.1.3.cmml" type="integer" xref="S3.SS1.p1.1.m1.1.1.3">42</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p1.1.m1.1c">14\!\times\!42</annotation><annotation encoding="application/x-llamapun" id="S3.SS1.p1.1.m1.1d">14 √ó 42</annotation></semantics></math>) fields of view (FOVs) of size <math alttext="2040\!\times\!2040" class="ltx_Math" display="inline" id="S3.SS1.p1.2.m2.1"><semantics id="S3.SS1.p1.2.m2.1a"><mrow id="S3.SS1.p1.2.m2.1.1" xref="S3.SS1.p1.2.m2.1.1.cmml"><mn id="S3.SS1.p1.2.m2.1.1.2" xref="S3.SS1.p1.2.m2.1.1.2.cmml">2040</mn><mo id="S3.SS1.p1.2.m2.1.1.1" lspace="0.052em" rspace="0.052em" xref="S3.SS1.p1.2.m2.1.1.1.cmml">√ó</mo><mn id="S3.SS1.p1.2.m2.1.1.3" xref="S3.SS1.p1.2.m2.1.1.3.cmml">2040</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS1.p1.2.m2.1b"><apply id="S3.SS1.p1.2.m2.1.1.cmml" xref="S3.SS1.p1.2.m2.1.1"><times id="S3.SS1.p1.2.m2.1.1.1.cmml" xref="S3.SS1.p1.2.m2.1.1.1"></times><cn id="S3.SS1.p1.2.m2.1.1.2.cmml" type="integer" xref="S3.SS1.p1.2.m2.1.1.2">2040</cn><cn id="S3.SS1.p1.2.m2.1.1.3.cmml" type="integer" xref="S3.SS1.p1.2.m2.1.1.3">2040</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p1.2.m2.1c">2040\!\times\!2040</annotation><annotation encoding="application/x-llamapun" id="S3.SS1.p1.2.m2.1d">2040 √ó 2040</annotation></semantics></math> pixels are imaged and processed in parallel and stitched together.
We compared three classical cell detection methods including watershed, radial symmetry <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib11" title="">11</a>]</cite> and the Laplacian of Gaussian (LoG) <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib14" title="">14</a>]</cite>. These are applied to the DAPI channel which highlights the nucleus of each cell (CTCs and non-CTCs alike).</p>
</div>
<div class="ltx_para" id="S3.SS1.p2">
<p class="ltx_p" id="S3.SS1.p2.5">For validation, the centroid coordinates of 24,110 cells from 5 FOVs were manually identified. We measured performance based on the cell count output by the detection algorithm as well as the average distance between the ground truth and estimated centroids. With an exhaustive parameter search, the best performing algorithm was the LoG with cell count F1 score of 0.997 and a 1.12 <math alttext="\mu" class="ltx_Math" display="inline" id="S3.SS1.p2.1.m1.1"><semantics id="S3.SS1.p2.1.m1.1a"><mi id="S3.SS1.p2.1.m1.1.1" xref="S3.SS1.p2.1.m1.1.1.cmml">Œº</mi><annotation-xml encoding="MathML-Content" id="S3.SS1.p2.1.m1.1b"><ci id="S3.SS1.p2.1.m1.1.1.cmml" xref="S3.SS1.p2.1.m1.1.1">ùúá</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p2.1.m1.1c">\mu</annotation><annotation encoding="application/x-llamapun" id="S3.SS1.p2.1.m1.1d">italic_Œº</annotation></semantics></math>m average distance between estimated and ground truth centroids. This is an acceptable error compared to the average radius of a cell nucleus of about 5 <math alttext="\mu" class="ltx_Math" display="inline" id="S3.SS1.p2.2.m2.1"><semantics id="S3.SS1.p2.2.m2.1a"><mi id="S3.SS1.p2.2.m2.1.1" xref="S3.SS1.p2.2.m2.1.1.cmml">Œº</mi><annotation-xml encoding="MathML-Content" id="S3.SS1.p2.2.m2.1b"><ci id="S3.SS1.p2.2.m2.1.1.cmml" xref="S3.SS1.p2.2.m2.1.1">ùúá</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p2.2.m2.1c">\mu</annotation><annotation encoding="application/x-llamapun" id="S3.SS1.p2.2.m2.1d">italic_Œº</annotation></semantics></math>m and whole cell size of up to 15 <math alttext="\mu" class="ltx_Math" display="inline" id="S3.SS1.p2.3.m3.1"><semantics id="S3.SS1.p2.3.m3.1a"><mi id="S3.SS1.p2.3.m3.1.1" xref="S3.SS1.p2.3.m3.1.1.cmml">Œº</mi><annotation-xml encoding="MathML-Content" id="S3.SS1.p2.3.m3.1b"><ci id="S3.SS1.p2.3.m3.1.1.cmml" xref="S3.SS1.p2.3.m3.1.1">ùúá</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p2.3.m3.1c">\mu</annotation><annotation encoding="application/x-llamapun" id="S3.SS1.p2.3.m3.1d">italic_Œº</annotation></semantics></math>m. Small <math alttext="24\!\times\!24" class="ltx_Math" display="inline" id="S3.SS1.p2.4.m4.1"><semantics id="S3.SS1.p2.4.m4.1a"><mrow id="S3.SS1.p2.4.m4.1.1" xref="S3.SS1.p2.4.m4.1.1.cmml"><mn id="S3.SS1.p2.4.m4.1.1.2" xref="S3.SS1.p2.4.m4.1.1.2.cmml">24</mn><mo id="S3.SS1.p2.4.m4.1.1.1" lspace="0.052em" rspace="0.052em" xref="S3.SS1.p2.4.m4.1.1.1.cmml">√ó</mo><mn id="S3.SS1.p2.4.m4.1.1.3" xref="S3.SS1.p2.4.m4.1.1.3.cmml">24</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS1.p2.4.m4.1b"><apply id="S3.SS1.p2.4.m4.1.1.cmml" xref="S3.SS1.p2.4.m4.1.1"><times id="S3.SS1.p2.4.m4.1.1.1.cmml" xref="S3.SS1.p2.4.m4.1.1.1"></times><cn id="S3.SS1.p2.4.m4.1.1.2.cmml" type="integer" xref="S3.SS1.p2.4.m4.1.1.2">24</cn><cn id="S3.SS1.p2.4.m4.1.1.3.cmml" type="integer" xref="S3.SS1.p2.4.m4.1.1.3">24</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p2.4.m4.1c">24\!\times\!24</annotation><annotation encoding="application/x-llamapun" id="S3.SS1.p2.4.m4.1d">24 √ó 24</annotation></semantics></math> pixel thumbnail images are cropped around the centroids based on the average size of nuclei. In total, our cell detection algorithm takes 10 min to detect <math alttext="\sim 3" class="ltx_Math" display="inline" id="S3.SS1.p2.5.m5.1"><semantics id="S3.SS1.p2.5.m5.1a"><mrow id="S3.SS1.p2.5.m5.1.1" xref="S3.SS1.p2.5.m5.1.1.cmml"><mi id="S3.SS1.p2.5.m5.1.1.2" xref="S3.SS1.p2.5.m5.1.1.2.cmml"></mi><mo id="S3.SS1.p2.5.m5.1.1.1" xref="S3.SS1.p2.5.m5.1.1.1.cmml">‚àº</mo><mn id="S3.SS1.p2.5.m5.1.1.3" xref="S3.SS1.p2.5.m5.1.1.3.cmml">3</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS1.p2.5.m5.1b"><apply id="S3.SS1.p2.5.m5.1.1.cmml" xref="S3.SS1.p2.5.m5.1.1"><csymbol cd="latexml" id="S3.SS1.p2.5.m5.1.1.1.cmml" xref="S3.SS1.p2.5.m5.1.1.1">similar-to</csymbol><csymbol cd="latexml" id="S3.SS1.p2.5.m5.1.1.2.cmml" xref="S3.SS1.p2.5.m5.1.1.2">absent</csymbol><cn id="S3.SS1.p2.5.m5.1.1.3.cmml" type="integer" xref="S3.SS1.p2.5.m5.1.1.3">3</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p2.5.m5.1c">\sim 3</annotation><annotation encoding="application/x-llamapun" id="S3.SS1.p2.5.m5.1d">‚àº 3</annotation></semantics></math>M cells from a single slide.</p>
</div>
</section>
<section class="ltx_subsection" id="S3.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.2 </span>Nuclear Segmentation</h3>
<div class="ltx_para" id="S3.SS2.p1">
<p class="ltx_p" id="S3.SS2.p1.1">Once a cell is detected, segmentation of the nucleus is important for extracting features like nuclear MFI for each channel.
Since the DAPI channel is already designed to highlight nuclei, nuclear segmentation can be accomplished with classical image processing.
Given each thumbnail image, we compute the normal vector field of each pixel to the center and the gradient vector field. These two fields are multiplied using a Gaussian weighted dot product with a radius parameter output by the detection algorithm. This results in a transformed image that isolates the cell of interest. We compared the watershed method and Otsu‚Äôs method <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib7" title="">7</a>]</cite> applied the transformed image to segment the nucleus.</p>
</div>
<div class="ltx_para" id="S3.SS2.p2">
<p class="ltx_p" id="S3.SS2.p2.1">To evaluate performance, the nuclei of the same set of 24,110 cells that were used for the cell detection ground truth were also manually segmented by 40 expert annotators. On a sample of 25 nuclei, we used the Simultaneous Truth and Performance Level Estimation (STAPLE) <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib17" title="">17</a>]</cite> to measure the concordance between annotators and found 93% sensitivity for segmenting the nucleus correctly. Watershed achieved F1=0.83 while Otsu‚Äôs method achieved F1=0.934 which is a good performance in comparison to human annotators. With parallelization the nuclear segmentation step takes an average of 10 min per slide.</p>
</div>
<figure class="ltx_figure" id="S3.F3"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="497" id="S3.F3.g1" src="x3.png" width="1196"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S3.F3.2.1.1" style="font-size:90%;">Figure 3</span>: </span><span class="ltx_text" id="S3.F3.3.2" style="font-size:90%;">BRIA Pipeline Overview. After IF image acquisition and slide processing, the main algorithmic steps include Cell Detection, Nuclear and Cell Segmentation, Feature Extraction, and CTC Classification.</span></figcaption>
</figure>
</section>
<section class="ltx_subsection" id="S3.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.3 </span>Cell Segmentation</h3>
<div class="ltx_para" id="S3.SS3.p1">
<p class="ltx_p" id="S3.SS3.p1.1">For whole cell segmentation we utilize all three IF channels to account for signal variation across each channel for both CTCs and non-CTCs alike. To address the scarcity of CTCs in our data, simulated montages of CTCs are created to increase their occurrence by placing manually segmented CTCs at randomly generated coordinates on a simulated background image based on real IF images.</p>
</div>
<div class="ltx_para" id="S3.SS3.p2">
<p class="ltx_p" id="S3.SS3.p2.1">We utilize a 3-channel U-Net <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib9" title="">9</a>]</cite> to segment cells within overlapping <math alttext="512\!\times\!512" class="ltx_Math" display="inline" id="S3.SS3.p2.1.m1.1"><semantics id="S3.SS3.p2.1.m1.1a"><mrow id="S3.SS3.p2.1.m1.1.1" xref="S3.SS3.p2.1.m1.1.1.cmml"><mn id="S3.SS3.p2.1.m1.1.1.2" xref="S3.SS3.p2.1.m1.1.1.2.cmml">512</mn><mo id="S3.SS3.p2.1.m1.1.1.1" lspace="0.052em" rspace="0.052em" xref="S3.SS3.p2.1.m1.1.1.1.cmml">√ó</mo><mn id="S3.SS3.p2.1.m1.1.1.3" xref="S3.SS3.p2.1.m1.1.1.3.cmml">512</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS3.p2.1.m1.1b"><apply id="S3.SS3.p2.1.m1.1.1.cmml" xref="S3.SS3.p2.1.m1.1.1"><times id="S3.SS3.p2.1.m1.1.1.1.cmml" xref="S3.SS3.p2.1.m1.1.1.1"></times><cn id="S3.SS3.p2.1.m1.1.1.2.cmml" type="integer" xref="S3.SS3.p2.1.m1.1.1.2">512</cn><cn id="S3.SS3.p2.1.m1.1.1.3.cmml" type="integer" xref="S3.SS3.p2.1.m1.1.1.3">512</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS3.p2.1.m1.1c">512\!\times\!512</annotation><annotation encoding="application/x-llamapun" id="S3.SS3.p2.1.m1.1d">512 √ó 512</annotation></semantics></math> image patches within each FOV. Overlap was enforced to avoid cutting cells at the edge of a patch. To accelerate this process we use our cell detection output to discard patches that do not contain any cells. The outputs of the U-Net are pixel-level probability maps for three classes: cell, boundary, and background. Including the boundary as a third class proved useful in separating cell clusters. The probability maps for each overlapping patch are merged into a single FOV by taking the maximum among the cell, boundary, and inverted background probability maps. Then instance segmentation is performed using watershed to identify the masks of each individual cell. These masks are mapped back to each thumbnail using the cell centroids.</p>
</div>
<div class="ltx_para" id="S3.SS3.p3">
<p class="ltx_p" id="S3.SS3.p3.1">Training was performed on 22 patient slides and 9 CTC montages with a total of <math alttext="10,443" class="ltx_Math" display="inline" id="S3.SS3.p3.1.m1.2"><semantics id="S3.SS3.p3.1.m1.2a"><mrow id="S3.SS3.p3.1.m1.2.3.2" xref="S3.SS3.p3.1.m1.2.3.1.cmml"><mn id="S3.SS3.p3.1.m1.1.1" xref="S3.SS3.p3.1.m1.1.1.cmml">10</mn><mo id="S3.SS3.p3.1.m1.2.3.2.1" xref="S3.SS3.p3.1.m1.2.3.1.cmml">,</mo><mn id="S3.SS3.p3.1.m1.2.2" xref="S3.SS3.p3.1.m1.2.2.cmml">443</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS3.p3.1.m1.2b"><list id="S3.SS3.p3.1.m1.2.3.1.cmml" xref="S3.SS3.p3.1.m1.2.3.2"><cn id="S3.SS3.p3.1.m1.1.1.cmml" type="integer" xref="S3.SS3.p3.1.m1.1.1">10</cn><cn id="S3.SS3.p3.1.m1.2.2.cmml" type="integer" xref="S3.SS3.p3.1.m1.2.2">443</cn></list></annotation-xml><annotation encoding="application/x-tex" id="S3.SS3.p3.1.m1.2c">10,443</annotation><annotation encoding="application/x-llamapun" id="S3.SS3.p3.1.m1.2d">10 , 443</annotation></semantics></math> manually segmented cells. The STAPLE algorithm was again used to evaluate human level segmentation performance. We gave 10 annotators 1,206 cells (including 165 CTCs) to manually segment and their concordance level was an F1=0.936. In comparison our U-Net achieved a comparable F1=0.933 averaged over 5-fold cross validation of our training set. The best performing parameters were learning rate=0.0001, dropout=0.3, weighted classes, network depth=4, epochs=300, and Adam optimizer. Our algorithm took an average of 126s per FOV and with parallelization, segmentation of an entire slide (588 FOVs, 3M cells) was accomplished in 39 min.</p>
</div>
</section>
<section class="ltx_subsection" id="S3.SS4">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.4 </span>Feature Extraction</h3>
<div class="ltx_para" id="S3.SS4.p1">
<p class="ltx_p" id="S3.SS4.p1.1">Feature-based ML is important for clinicians to better interpret classification results. Since clinicians already rely on quantitative values like nuclear MFI and visually inspect cell morphology and texture, we are motivated to extract additional morphology, intensity, and texture features from the nucleus, whole cell, and entire thumbnail image. In total we extract 122 features which are summarized with equations in Supp. Table 1.</p>
</div>
<div class="ltx_para" id="S3.SS4.p2">
<p class="ltx_p" id="S3.SS4.p2.1">We extract 8 morphology features, 4 each from the nuclear and cell masks: size, roundness, elongation and the first Hu moment <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib3" title="">3</a>]</cite>, which captures more subtle shape variability. (BCa CTCs are often larger than non-CTCs.)
We next compute a total of 44 intensity features from nuclear and cell masks across the three channels: MFI, lower, median, and upper quartiles, interquartile range, as well as Pearson‚Äôs correlations and Ranked-Weighted Co-Localizations <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib13" title="">13</a>]</cite> between channels. We also extract CK specific features like CK+ ratio, defined as the number of CK pixels that are greater than a cutoff value, divided by the area of the mask. CTCs will exhibit a higher CK+ ratio. We also compute the mean and standard deviation of pixels in the entire CK channel thumbnail which may help eliminate CK+ artefacts like flares.</p>
</div>
<div class="ltx_para" id="S3.SS4.p3">
<p class="ltx_p" id="S3.SS4.p3.2">Finally, we extract 70 texture features. First, a 32D Gabor feature vector is constructed using the mean and standard deviation of filtered images for 16 parameter combinations: <math alttext="\theta\!=\!0^{\circ},45^{\circ},90^{\circ},135^{\circ},\lambda\!=\!0.1,0.4" class="ltx_Math" display="inline" id="S3.SS4.p3.1.m1.4"><semantics id="S3.SS4.p3.1.m1.4a"><mrow id="S3.SS4.p3.1.m1.4.4.2" xref="S3.SS4.p3.1.m1.4.4.3.cmml"><mrow id="S3.SS4.p3.1.m1.3.3.1.1" xref="S3.SS4.p3.1.m1.3.3.1.1.cmml"><mi id="S3.SS4.p3.1.m1.3.3.1.1.6" xref="S3.SS4.p3.1.m1.3.3.1.1.6.cmml">Œ∏</mi><mo id="S3.SS4.p3.1.m1.3.3.1.1.5" lspace="0.108em" rspace="0.108em" xref="S3.SS4.p3.1.m1.3.3.1.1.5.cmml">=</mo><mrow id="S3.SS4.p3.1.m1.3.3.1.1.4.4" xref="S3.SS4.p3.1.m1.3.3.1.1.4.5.cmml"><msup id="S3.SS4.p3.1.m1.3.3.1.1.1.1.1" xref="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.cmml"><mn id="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.2" xref="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.2.cmml">0</mn><mo id="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.3" xref="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.3.cmml">‚àò</mo></msup><mo id="S3.SS4.p3.1.m1.3.3.1.1.4.4.5" xref="S3.SS4.p3.1.m1.3.3.1.1.4.5.cmml">,</mo><msup id="S3.SS4.p3.1.m1.3.3.1.1.2.2.2" xref="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.cmml"><mn id="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.2" xref="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.2.cmml">45</mn><mo id="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.3" xref="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.3.cmml">‚àò</mo></msup><mo id="S3.SS4.p3.1.m1.3.3.1.1.4.4.6" xref="S3.SS4.p3.1.m1.3.3.1.1.4.5.cmml">,</mo><msup id="S3.SS4.p3.1.m1.3.3.1.1.3.3.3" xref="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.cmml"><mn id="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.2" xref="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.2.cmml">90</mn><mo id="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.3" xref="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.3.cmml">‚àò</mo></msup><mo id="S3.SS4.p3.1.m1.3.3.1.1.4.4.7" xref="S3.SS4.p3.1.m1.3.3.1.1.4.5.cmml">,</mo><msup id="S3.SS4.p3.1.m1.3.3.1.1.4.4.4" xref="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.cmml"><mn id="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.2" xref="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.2.cmml">135</mn><mo id="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.3" xref="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.3.cmml">‚àò</mo></msup></mrow></mrow><mo id="S3.SS4.p3.1.m1.4.4.2.3" xref="S3.SS4.p3.1.m1.4.4.3a.cmml">,</mo><mrow id="S3.SS4.p3.1.m1.4.4.2.2" xref="S3.SS4.p3.1.m1.4.4.2.2.cmml"><mi id="S3.SS4.p3.1.m1.4.4.2.2.2" xref="S3.SS4.p3.1.m1.4.4.2.2.2.cmml">Œª</mi><mo id="S3.SS4.p3.1.m1.4.4.2.2.1" lspace="0.108em" rspace="0.108em" xref="S3.SS4.p3.1.m1.4.4.2.2.1.cmml">=</mo><mrow id="S3.SS4.p3.1.m1.4.4.2.2.3.2" xref="S3.SS4.p3.1.m1.4.4.2.2.3.1.cmml"><mn id="S3.SS4.p3.1.m1.1.1" xref="S3.SS4.p3.1.m1.1.1.cmml">0.1</mn><mo id="S3.SS4.p3.1.m1.4.4.2.2.3.2.1" xref="S3.SS4.p3.1.m1.4.4.2.2.3.1.cmml">,</mo><mn id="S3.SS4.p3.1.m1.2.2" xref="S3.SS4.p3.1.m1.2.2.cmml">0.4</mn></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.SS4.p3.1.m1.4b"><apply id="S3.SS4.p3.1.m1.4.4.3.cmml" xref="S3.SS4.p3.1.m1.4.4.2"><csymbol cd="ambiguous" id="S3.SS4.p3.1.m1.4.4.3a.cmml" xref="S3.SS4.p3.1.m1.4.4.2.3">formulae-sequence</csymbol><apply id="S3.SS4.p3.1.m1.3.3.1.1.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1"><eq id="S3.SS4.p3.1.m1.3.3.1.1.5.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.5"></eq><ci id="S3.SS4.p3.1.m1.3.3.1.1.6.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.6">ùúÉ</ci><list id="S3.SS4.p3.1.m1.3.3.1.1.4.5.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.4.4"><apply id="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.1.1.1"><csymbol cd="ambiguous" id="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.1.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.1.1.1">superscript</csymbol><cn id="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.2.cmml" type="integer" xref="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.2">0</cn><compose id="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.3.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.1.1.1.3"></compose></apply><apply id="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.2.2.2"><csymbol cd="ambiguous" id="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.1.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.2.2.2">superscript</csymbol><cn id="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.2.cmml" type="integer" xref="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.2">45</cn><compose id="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.3.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.2.2.2.3"></compose></apply><apply id="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.3.3.3"><csymbol cd="ambiguous" id="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.1.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.3.3.3">superscript</csymbol><cn id="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.2.cmml" type="integer" xref="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.2">90</cn><compose id="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.3.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.3.3.3.3"></compose></apply><apply id="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.4.4.4"><csymbol cd="ambiguous" id="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.1.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.4.4.4">superscript</csymbol><cn id="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.2.cmml" type="integer" xref="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.2">135</cn><compose id="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.3.cmml" xref="S3.SS4.p3.1.m1.3.3.1.1.4.4.4.3"></compose></apply></list></apply><apply id="S3.SS4.p3.1.m1.4.4.2.2.cmml" xref="S3.SS4.p3.1.m1.4.4.2.2"><eq id="S3.SS4.p3.1.m1.4.4.2.2.1.cmml" xref="S3.SS4.p3.1.m1.4.4.2.2.1"></eq><ci id="S3.SS4.p3.1.m1.4.4.2.2.2.cmml" xref="S3.SS4.p3.1.m1.4.4.2.2.2">ùúÜ</ci><list id="S3.SS4.p3.1.m1.4.4.2.2.3.1.cmml" xref="S3.SS4.p3.1.m1.4.4.2.2.3.2"><cn id="S3.SS4.p3.1.m1.1.1.cmml" type="float" xref="S3.SS4.p3.1.m1.1.1">0.1</cn><cn id="S3.SS4.p3.1.m1.2.2.cmml" type="float" xref="S3.SS4.p3.1.m1.2.2">0.4</cn></list></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS4.p3.1.m1.4c">\theta\!=\!0^{\circ},45^{\circ},90^{\circ},135^{\circ},\lambda\!=\!0.1,0.4</annotation><annotation encoding="application/x-llamapun" id="S3.SS4.p3.1.m1.4d">italic_Œ∏ = 0 start_POSTSUPERSCRIPT ‚àò end_POSTSUPERSCRIPT , 45 start_POSTSUPERSCRIPT ‚àò end_POSTSUPERSCRIPT , 90 start_POSTSUPERSCRIPT ‚àò end_POSTSUPERSCRIPT , 135 start_POSTSUPERSCRIPT ‚àò end_POSTSUPERSCRIPT , italic_Œª = 0.1 , 0.4</annotation></semantics></math>, and <math alttext="\sigma\!=\!1,3" class="ltx_Math" display="inline" id="S3.SS4.p3.2.m2.2"><semantics id="S3.SS4.p3.2.m2.2a"><mrow id="S3.SS4.p3.2.m2.2.3" xref="S3.SS4.p3.2.m2.2.3.cmml"><mi id="S3.SS4.p3.2.m2.2.3.2" xref="S3.SS4.p3.2.m2.2.3.2.cmml">œÉ</mi><mo id="S3.SS4.p3.2.m2.2.3.1" lspace="0.108em" rspace="0.108em" xref="S3.SS4.p3.2.m2.2.3.1.cmml">=</mo><mrow id="S3.SS4.p3.2.m2.2.3.3.2" xref="S3.SS4.p3.2.m2.2.3.3.1.cmml"><mn id="S3.SS4.p3.2.m2.1.1" xref="S3.SS4.p3.2.m2.1.1.cmml">1</mn><mo id="S3.SS4.p3.2.m2.2.3.3.2.1" xref="S3.SS4.p3.2.m2.2.3.3.1.cmml">,</mo><mn id="S3.SS4.p3.2.m2.2.2" xref="S3.SS4.p3.2.m2.2.2.cmml">3</mn></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.SS4.p3.2.m2.2b"><apply id="S3.SS4.p3.2.m2.2.3.cmml" xref="S3.SS4.p3.2.m2.2.3"><eq id="S3.SS4.p3.2.m2.2.3.1.cmml" xref="S3.SS4.p3.2.m2.2.3.1"></eq><ci id="S3.SS4.p3.2.m2.2.3.2.cmml" xref="S3.SS4.p3.2.m2.2.3.2">ùúé</ci><list id="S3.SS4.p3.2.m2.2.3.3.1.cmml" xref="S3.SS4.p3.2.m2.2.3.3.2"><cn id="S3.SS4.p3.2.m2.1.1.cmml" type="integer" xref="S3.SS4.p3.2.m2.1.1">1</cn><cn id="S3.SS4.p3.2.m2.2.2.cmml" type="integer" xref="S3.SS4.p3.2.m2.2.2">3</cn></list></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS4.p3.2.m2.2c">\sigma\!=\!1,3</annotation><annotation encoding="application/x-llamapun" id="S3.SS4.p3.2.m2.2d">italic_œÉ = 1 , 3</annotation></semantics></math>, selected based on cell size. Gabor features identify frequency changes in an image at various orientations and sizes. This helps to identify small dye-aggregates, flares as well as CTCs in an image. Next, a 32D Laws <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib4" title="">4</a>]</cite> feature vector is constructed by ordered multiplications of 1D filters L5 (Level), E5 (Edge), S5 (Spot), and R5 (Ripple) to detect spatial patterns.
Finally, Local Binary Pattern (LBP) <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib6" title="">6</a>]</cite> encodes edges, corners, raised areas, and lines.
LBP results in a transformed image per channel and we calculate the correlation and normalized mutual information between each channel pair, resulting in 6 final texture features.
</p>
</div>
</section>
</section>
<section class="ltx_section" id="S4">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Experiments</h2>
<div class="ltx_para" id="S4.p1">
<p class="ltx_p" id="S4.p1.1">Once cells are detected, segmented and features are extracted using the steps of the pipeline described above, we are ready to collect and label ground truth data to train and evaluate a CTC classifier. The CTC classifier is used to replace the standard rule-based biomarker thresholding and substantially reduce the number of CTC candidates presented to clinicians for final review.</p>
</div>
<section class="ltx_subsection" id="S4.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Ground Truth Data</h3>
<div class="ltx_para" id="S4.SS1.p1">
<p class="ltx_p" id="S4.SS1.p1.1">Ground truth data is collected from 15 mBCa patients using the same cell detection and segmentation steps of the pipeline. From 15 patients, a total of 241,644,731 cells were identified by the cell detection algorithm. Then 99.8% of non-CTCs and artefacts were filtered using the rule-based thresholds described in Section¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S2" title="2 Background ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">2</span></a> leaving 500,255 CTC candidates presented for manual classification. (Without a CTC classifier, this is the number of candidates clinicians would have to manually review in their workflow.) Ground truth CTCs were labeled by four annotators and subjected to a round of adjudication for consensus. They each labeled a comparable number of non-CTCs and non-cellular artefacts.</p>
</div>
<figure class="ltx_table" id="S4.T1">
<table class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle" id="S4.T1.2">
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="S4.T1.2.1.1" style="background-color:#B5DEE8;">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_rr ltx_border_t" id="S4.T1.2.1.1.1"><span class="ltx_text ltx_font_bold" id="S4.T1.2.1.1.1.1" style="background-color:#B5DEE8;">Dataset</span></th>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.1.1.2"><span class="ltx_text ltx_font_bold" id="S4.T1.2.1.1.2.1" style="background-color:#B5DEE8;">Patients</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.1.1.3"><span class="ltx_text ltx_font_bold" id="S4.T1.2.1.1.3.1" style="background-color:#B5DEE8;">Labeled Samples</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.1.1.4"><span class="ltx_text ltx_font_bold" id="S4.T1.2.1.1.4.1" style="background-color:#B5DEE8;">CTCs</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.1.1.5"><span class="ltx_text ltx_font_bold" id="S4.T1.2.1.1.5.1" style="background-color:#B5DEE8;">Non-CTCs</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.1.1.6"><span class="ltx_text ltx_font_bold" id="S4.T1.2.1.1.6.1" style="background-color:#B5DEE8;">Artefacts</span></td>
</tr>
<tr class="ltx_tr" id="S4.T1.2.2.2">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_rr ltx_border_t" id="S4.T1.2.2.2.1"><span class="ltx_text ltx_font_bold" id="S4.T1.2.2.2.1.1">Training</span></th>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.2.2.2">7</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.2.2.3">4,680</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.2.2.4">1,667</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.2.2.5">1,632</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.2.2.6">1,381</td>
</tr>
<tr class="ltx_tr" id="S4.T1.2.3.3">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_rr ltx_border_t" id="S4.T1.2.3.3.1"><span class="ltx_text ltx_font_bold" id="S4.T1.2.3.3.1.1">Verification</span></th>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.3.3.2">5</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.3.3.3">1,931</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.3.3.4">340</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.3.3.5">882</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.3.3.6">709</td>
</tr>
<tr class="ltx_tr" id="S4.T1.2.4.4">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_rr ltx_border_t" id="S4.T1.2.4.4.1"><span class="ltx_text ltx_font_bold" id="S4.T1.2.4.4.1.1">Validation</span></th>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.4.4.2">3</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.4.4.3">2,922</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.4.4.4">1,221</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.4.4.5">969</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T1.2.4.4.6">732</td>
</tr>
<tr class="ltx_tr" id="S4.T1.2.5.5">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_b ltx_border_l ltx_border_rr ltx_border_t" id="S4.T1.2.5.5.1"><span class="ltx_text ltx_font_bold" id="S4.T1.2.5.5.1.1">Total</span></th>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T1.2.5.5.2">15</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T1.2.5.5.3">9,533</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T1.2.5.5.4">3,228</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T1.2.5.5.5">3,483</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T1.2.5.5.6">2,822</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table"><span class="ltx_text" id="S4.T1.3.1.1" style="font-size:90%;">Table 1</span>: </span><span class="ltx_text" id="S4.T1.4.2" style="font-size:90%;">Summary of class sample counts for each dataset split.</span></figcaption>
</figure>
<div class="ltx_para" id="S4.SS1.p2">
<p class="ltx_p" id="S4.SS1.p2.1">The result was 9,533 labeled samples with 3,228 CTCs, 3,483 non-CTCs, and 2,822 artefacts. The ground truth data was split at the patient level into training, verification, and validation sets, stratified by number of CTCs (See Table¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S4.T1" title="Table 1 ‚Ä£ 4.1 Ground Truth Data ‚Ä£ 4 Experiments ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">1</span></a>).</p>
</div>
</section>
<section class="ltx_subsection" id="S4.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>CTC Classifier Training</h3>
<div class="ltx_para" id="S4.SS2.p1">
<p class="ltx_p" id="S4.SS2.p1.2">For CTC classification, we combined the non-CTC and artefacts into a single negative class for binary classification. We evaluated four SVM models, including a linear SVM and three non-linear kernel SVMs: radial basis function (RBF), sigmoid, and polynomial. To optimize the classifier performance, the hyper-parameter tuning of <math alttext="C" class="ltx_Math" display="inline" id="S4.SS2.p1.1.m1.1"><semantics id="S4.SS2.p1.1.m1.1a"><mi id="S4.SS2.p1.1.m1.1.1" xref="S4.SS2.p1.1.m1.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="S4.SS2.p1.1.m1.1b"><ci id="S4.SS2.p1.1.m1.1.1.cmml" xref="S4.SS2.p1.1.m1.1.1">ùê∂</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p1.1.m1.1c">C</annotation><annotation encoding="application/x-llamapun" id="S4.SS2.p1.1.m1.1d">italic_C</annotation></semantics></math>, <math alttext="\gamma" class="ltx_Math" display="inline" id="S4.SS2.p1.2.m2.1"><semantics id="S4.SS2.p1.2.m2.1a"><mi id="S4.SS2.p1.2.m2.1.1" xref="S4.SS2.p1.2.m2.1.1.cmml">Œ≥</mi><annotation-xml encoding="MathML-Content" id="S4.SS2.p1.2.m2.1b"><ci id="S4.SS2.p1.2.m2.1.1.cmml" xref="S4.SS2.p1.2.m2.1.1">ùõæ</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p1.2.m2.1c">\gamma</annotation><annotation encoding="application/x-llamapun" id="S4.SS2.p1.2.m2.1d">italic_Œ≥</annotation></semantics></math>, and polynomial degree were performed using grid search over 5-fold cross validation. All image features were normalized using min-max normalization fit on the training dataset and applied to the verification and validation sets. For our application, it is imperative for patient outcomes to miss as few CTCs as possible (false negatives). Alternatively, allowing false positive CTC candidates only increases the burden of the manual reviewers. Therefore, optimal performance hinges on maximizing sensitivity while controlling for specificity.</p>
</div>
<figure class="ltx_figure" id="S4.F4"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="483" id="S4.F4.g1" src="x4.png" width="1196"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" id="S4.F4.2.1.1" style="font-size:90%;">Figure 4</span>: </span><span class="ltx_text" id="S4.F4.3.2" style="font-size:90%;">Top ten features weights for True Positive (CTC) and True Negative (non-CTC/artefact) classes using SHapley Additive exPlanations (SHAP) <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#bib.bib5" title="">5</a>]</cite> averaged over 100 random samples per class in the verification set. (Note: Nuc_IQR_ck is the inter-quartile range (IQR) of pixels in the nucleus (nuc) in CK; STD_ck is the standard deviation of CK; Cell_Coloc_ck_dapi is the co-localization of pixels between CK and DAPI within the cell. See Supp. Table 1 for feature definitions.)
</span></figcaption>
</figure>
<figure class="ltx_table" id="S4.T2">
<table class="ltx_tabular ltx_centering ltx_align_middle" id="S4.T2.2">
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="S4.T2.2.1.1" style="background-color:#B5DEE8;">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_rr ltx_border_t" id="S4.T2.2.1.1.1"><span class="ltx_text ltx_font_bold" id="S4.T2.2.1.1.1.1" style="background-color:#B5DEE8;">Dataset</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.1.1.2"><span class="ltx_text ltx_font_bold" id="S4.T2.2.1.1.2.1" style="background-color:#B5DEE8;">True Pos.</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.1.1.3"><span class="ltx_text ltx_font_bold" id="S4.T2.2.1.1.3.1" style="background-color:#B5DEE8;">True Neg.</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.1.1.4"><span class="ltx_text ltx_font_bold" id="S4.T2.2.1.1.4.1" style="background-color:#B5DEE8;">False Pos.</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.1.1.5"><span class="ltx_text ltx_font_bold" id="S4.T2.2.1.1.5.1" style="background-color:#B5DEE8;">False Neg.</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.1.1.6"><span class="ltx_text ltx_font_bold" id="S4.T2.2.1.1.6.1" style="background-color:#B5DEE8;">Sens.</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.1.1.7"><span class="ltx_text ltx_font_bold" id="S4.T2.2.1.1.7.1" style="background-color:#B5DEE8;">Spec.</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.1.1.8"><span class="ltx_text ltx_font_bold" id="S4.T2.2.1.1.8.1" style="background-color:#B5DEE8;">Acc.</span></td>
</tr>
<tr class="ltx_tr" id="S4.T2.2.2.2">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_rr ltx_border_t" id="S4.T2.2.2.2.1"><span class="ltx_text ltx_font_bold" id="S4.T2.2.2.2.1.1">Training</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.2.2.2">1,667</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.2.2.3">3,007</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.2.2.4">6</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.2.2.5">0</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.2.2.6">100%</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.2.2.7">99.8%</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.2.2.8">99.9%</td>
</tr>
<tr class="ltx_tr" id="S4.T2.2.3.3">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_rr ltx_border_t" id="S4.T2.2.3.3.1"><span class="ltx_text ltx_font_bold" id="S4.T2.2.3.3.1.1">Verification</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.3.3.2">340</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.3.3.3">1,571</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.3.3.4">20</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.3.3.5">0</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.3.3.6">100%</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.3.3.7">98.7%</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="S4.T2.2.3.3.8">99.0%</td>
</tr>
<tr class="ltx_tr" id="S4.T2.2.4.4">
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_l ltx_border_rr ltx_border_t" id="S4.T2.2.4.4.1"><span class="ltx_text ltx_font_bold" id="S4.T2.2.4.4.1.1">Validation</span></td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T2.2.4.4.2">1,210</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T2.2.4.4.3">1,648</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T2.2.4.4.4">53</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T2.2.4.4.5">11</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T2.2.4.4.6">99.1%</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T2.2.4.4.7">96.9%</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="S4.T2.2.4.4.8">97.8%</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table"><span class="ltx_text" id="S4.T2.3.1.1" style="font-size:90%;">Table 2</span>: </span><span class="ltx_text" id="S4.T2.4.2" style="font-size:90%;">Summary of True Positive, True Negative, False Positive, False Negative counts and Sensitivity, Specificity, and Accuracy for each dataset split.</span></figcaption>
</figure>
</section>
<section class="ltx_subsection" id="S4.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.3 </span>Results</h3>
<div class="ltx_para" id="S4.SS3.p1">
<p class="ltx_p" id="S4.SS3.p1.2">The SVM with RBF Kernel and hyper-parameters of <math alttext="C\!=\!10" class="ltx_Math" display="inline" id="S4.SS3.p1.1.m1.1"><semantics id="S4.SS3.p1.1.m1.1a"><mrow id="S4.SS3.p1.1.m1.1.1" xref="S4.SS3.p1.1.m1.1.1.cmml"><mi id="S4.SS3.p1.1.m1.1.1.2" xref="S4.SS3.p1.1.m1.1.1.2.cmml">C</mi><mo id="S4.SS3.p1.1.m1.1.1.1" lspace="0.108em" rspace="0.108em" xref="S4.SS3.p1.1.m1.1.1.1.cmml">=</mo><mn id="S4.SS3.p1.1.m1.1.1.3" xref="S4.SS3.p1.1.m1.1.1.3.cmml">10</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.SS3.p1.1.m1.1b"><apply id="S4.SS3.p1.1.m1.1.1.cmml" xref="S4.SS3.p1.1.m1.1.1"><eq id="S4.SS3.p1.1.m1.1.1.1.cmml" xref="S4.SS3.p1.1.m1.1.1.1"></eq><ci id="S4.SS3.p1.1.m1.1.1.2.cmml" xref="S4.SS3.p1.1.m1.1.1.2">ùê∂</ci><cn id="S4.SS3.p1.1.m1.1.1.3.cmml" type="integer" xref="S4.SS3.p1.1.m1.1.1.3">10</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS3.p1.1.m1.1c">C\!=\!10</annotation><annotation encoding="application/x-llamapun" id="S4.SS3.p1.1.m1.1d">italic_C = 10</annotation></semantics></math> and <math alttext="\gamma\!=\!1" class="ltx_Math" display="inline" id="S4.SS3.p1.2.m2.1"><semantics id="S4.SS3.p1.2.m2.1a"><mrow id="S4.SS3.p1.2.m2.1.1" xref="S4.SS3.p1.2.m2.1.1.cmml"><mi id="S4.SS3.p1.2.m2.1.1.2" xref="S4.SS3.p1.2.m2.1.1.2.cmml">Œ≥</mi><mo id="S4.SS3.p1.2.m2.1.1.1" lspace="0.108em" rspace="0.108em" xref="S4.SS3.p1.2.m2.1.1.1.cmml">=</mo><mn id="S4.SS3.p1.2.m2.1.1.3" xref="S4.SS3.p1.2.m2.1.1.3.cmml">1</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.SS3.p1.2.m2.1b"><apply id="S4.SS3.p1.2.m2.1.1.cmml" xref="S4.SS3.p1.2.m2.1.1"><eq id="S4.SS3.p1.2.m2.1.1.1.cmml" xref="S4.SS3.p1.2.m2.1.1.1"></eq><ci id="S4.SS3.p1.2.m2.1.1.2.cmml" xref="S4.SS3.p1.2.m2.1.1.2">ùõæ</ci><cn id="S4.SS3.p1.2.m2.1.1.3.cmml" type="integer" xref="S4.SS3.p1.2.m2.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS3.p1.2.m2.1c">\gamma\!=\!1</annotation><annotation encoding="application/x-llamapun" id="S4.SS3.p1.2.m2.1d">italic_Œ≥ = 1</annotation></semantics></math> scored the highest average accuracy across the 5 folds. The chosen model was retrained on the complete training set and a 0.3 probability threshold was selected to maintain 100% sensitivity on the training and verification sets with greater than 98% specificity. Of the 26 total false positives in the training and verification sets, 8 were actually Non-CTCs and 18 were determined to be artefacts. Since artefacts are typically easier to inspect by humans this majority of false positives further alleviates the manual review of CTC candidates. We then applied our model to the hold-out validation set and achieved over 99% sensitivity and 97% specificity. See Table¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S4.T2" title="Table 2 ‚Ä£ 4.2 CTC Classifier Training ‚Ä£ 4 Experiments ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">2</span></a> for these results.</p>
</div>
<div class="ltx_para" id="S4.SS3.p2">
<p class="ltx_p" id="S4.SS3.p2.1">The ranked feature weights in Fig.¬†<a class="ltx_ref" href="https://arxiv.org/html/2410.02988v1#S4.F4" title="Figure 4 ‚Ä£ 4.2 CTC Classifier Training ‚Ä£ 4 Experiments ‚Ä£ Fully Automated CTC Detection, Segmentation and Classification for Multi-Channel IF Imaging"><span class="ltx_text ltx_ref_tag">4</span></a> indicate that CK intensity features are most important for the CTC class while DAPI and CD45/31 features are more useful for classifying non-CTCs/artefacts. This result coincides with clinical importance of CK in classifying CTCs and can be used for additional interpretation by clinicians.</p>
</div>
<div class="ltx_para" id="S4.SS3.p3">
<p class="ltx_p" id="S4.SS3.p3.1">Finally, we applied BRIA on the full patient data of the combined training and verification sets. With a total of 12 patients, 171M cells were detected (avg. 14M per patient). By biomarker thresholding, 401,608 CTC candidates would have been presented for manual review. In contrast, our ML CTC classifier achieved a <math alttext="100\times" class="ltx_math_unparsed" display="inline" id="S4.SS3.p3.1.m1.1"><semantics id="S4.SS3.p3.1.m1.1a"><mrow id="S4.SS3.p3.1.m1.1b"><mn id="S4.SS3.p3.1.m1.1.1">100</mn><mo id="S4.SS3.p3.1.m1.1.2" lspace="0.222em">√ó</mo></mrow><annotation encoding="application/x-tex" id="S4.SS3.p3.1.m1.1c">100\times</annotation><annotation encoding="application/x-llamapun" id="S4.SS3.p3.1.m1.1d">100 √ó</annotation></semantics></math> reduction for a total of 4,019 CTC candidates (avg. 335 per patient) of which 2,007 are known to be true positive CTCs. Because we achieved 100% sensitivity in the combined training and verification sets, we know that no CTCs were missed on the full patient data. This showcases the true clinical value of our ML-based pipeline to substantially reduce manual workloads.</p>
</div>
<div class="ltx_para" id="S4.SS3.p4">
<p class="ltx_p" id="S4.SS3.p4.1">The BRIA pipeline is configured to leverage AWS Batch jobs that run in parallel, enabling the simultaneous analysis of multiple slides. Each slide takes an average of 90 minutes to complete the analysis. The pipeline requires 32 GB of memory and up to 12 GB of temporary storage for decompressed TIFF image files, while the analysis output files, including slide QC, identified CTC MFI values in JSON format, and cell thumbnails as PNG images, require less than 50 MB of S3 storage.</p>
</div>
</section>
</section>
<section class="ltx_section" id="S5">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Conclusion</h2>
<div class="ltx_para" id="S5.p1">
<p class="ltx_p" id="S5.p1.1">In conclusion, we have demonstrated the clinical utility of our fully automated BRIA pipeline for detection, segmentation, feature extraction, and classification of mBCa CTCs in multi-channel IF imaging. This work is important to deliver patient-specific profiles for mBCa by assisting clinicians in detecting the rare occurrences of CTCs in liquid biopsies. Validated on a hold-out set with over 99% sensitivity our CTC classifier detects nearly all CTCs while reducing substantial burden by presenting 100x fewer candidates for manual confirmation. With BRIA in production, future efforts will focus on continuous learning to monitor and maintain performance of CTC classification over time and expansion to additional cancer types for additional generalization and validation.</p>
</div>
<div class="ltx_para" id="S5.p2">
<span class="ltx_ERROR undefined" id="S5.p2.1">{credits}</span>
</div>
<section class="ltx_subsubsection" id="S5.SS0.SSS1">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.0.1 </span>Acknowledgements</h4>
<div class="ltx_para" id="S5.SS0.SSS1.p1">
<p class="ltx_p" id="S5.SS0.SSS1.p1.1">We thank our ground truth labeling teams, our medical director and the Assay Development, MTT and Clinical groups at Epic Sciences for their support in this work.</p>
</div>
</section>
<section class="ltx_subsubsection" id="S5.SS0.SSS2">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.0.2 </span><span class="ltx_ERROR undefined" id="S5.SS0.SSS2.1.1">\discintname</span>
</h4>
<div class="ltx_para" id="S5.SS0.SSS2.p1">
<p class="ltx_p" id="S5.SS0.SSS2.p1.1">The authors are current or past employees of Epic Sciences and may own company stock options.</p>
</div>
</section>
</section>
<section class="ltx_bibliography" id="bib">
<h2 class="ltx_title ltx_title_bibliography">References</h2>
<ul class="ltx_biblist">
<li class="ltx_bibitem" id="bib.bib1">
<span class="ltx_tag ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">
Guo, Z., Lin, X., Hui, Y., Wang, J., Zhang, Q., Kong, F.: Circulating tumor cell identification based on deep learning. Frontiers in Oncology <span class="ltx_text ltx_font_bold" id="bib.bib1.1.1">12</span>, 843879 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib2">
<span class="ltx_tag ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">
Hashimoto, K., Kamiya, T., Li, G., Yoneda, K., Tanaka, F.: Automatic identification of tumor cells for circulating tumor cells by convolutional neural networks. International Journal of Innovative Computing, Information and Control <span class="ltx_text ltx_font_bold" id="bib.bib2.1.1">19</span>(1) (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib3">
<span class="ltx_tag ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">
Hu, M.K.: Visual Pattern Recognition by Moment Invariants. In: IRE Trans. Info. Theory. vol.¬†IT-8, pp. 179‚Äì187 (January 1962)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib4">
<span class="ltx_tag ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">
Laws, K.: Texture Image Segmentation (January 1980)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib5">
<span class="ltx_tag ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">
Lundberg, S.M., Lee, S.I.: A unified approach to interpreting model predictions. In: Guyon, I., Luxburg, U.V., Bengio, S., Wallach, H., Fergus, R., Vishwanathan, S., Garnett, R. (eds.) Advances in Neural Information Processing Systems 30, pp. 4765‚Äì4774. Curran Associates, Inc. (2017)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib6">
<span class="ltx_tag ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">
Ojala, T., Pietik√§inen, M., Harwood, D.: A comparative study of texture measures with classification based on featured distributions. Pattern recognition <span class="ltx_text ltx_font_bold" id="bib.bib6.1.1">29</span>(1), 51‚Äì59 (1996)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib7">
<span class="ltx_tag ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">
Otsu, N.: A threshold selection method from gray level histograms. IEEE Trans. Systems, Man and Cybernetics <span class="ltx_text ltx_font_bold" id="bib.bib7.1.1">9</span>, 62‚Äì66 (Mar 1979)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib8">
<span class="ltx_tag ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">
Park, J., Ha, S., Kim, J., Song, J.W., Hyun, K.A., Kamiya, T., Jung, H.I.: Classification of circulating tumor cell clusters by morphological characteristics using convolutional neural network-support vector machine. Sensors and Actuators B: Chemical <span class="ltx_text ltx_font_bold" id="bib.bib8.1.1">401</span>, 134896 (2024)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib9">
<span class="ltx_tag ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">
Ronneberger, O., P.Fischer, Brox, T.: U-net: Convolutional networks for biomedical image segmentation. In: Medical Image Computing and Computer-Assisted Intervention (MICCAI). LNCS, vol.¬†9351, pp. 234‚Äì241. Springer (2015)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib10">
<span class="ltx_tag ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">
Scher, H., Armstrong, A., Schonhoft, J., Gill, A., Zhao, J., Barnett, E., Carbone, E., Lu, J., Antonarakis, E., Luo, J., et¬†al.: Development and validation of circulating tumour cell enumeration (Epic Sciences) as a prognostic biomarker in men with metastatic castration-resistant prostate cancer. European Journal of Cancer <span class="ltx_text ltx_font_bold" id="bib.bib10.1.1">150</span>, 83‚Äì94 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib11">
<span class="ltx_tag ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock">
Schmitt, O., Hasse, M.: Radial symmetries based decomposition of cell clusters in binary and gray level images. Pattern Recognition <span class="ltx_text ltx_font_bold" id="bib.bib11.1.1">41</span>(6), 1905‚Äì1923 (Jun 2008)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib12">
<span class="ltx_tag ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock">
Shen, C., Rawal, S., Brown, R., Zhou, H., Agarwal, A., Watson, M.A., Cote, R.J., Yang, C.: Automatic detection of circulating tumor cells and cancer associated fibroblasts using deep learning. Scientific reports <span class="ltx_text ltx_font_bold" id="bib.bib12.1.1">13</span>(1), ¬†5708 (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib13">
<span class="ltx_tag ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock">
Singan, V.R.: Dual channel rank-based intensity weighting for quantitative co-localization of microscopy images. BMC Bioinformatics <span class="ltx_text ltx_font_bold" id="bib.bib13.1.1">12</span>, 1471‚Äì2105 (Oct 2011)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib14">
<span class="ltx_tag ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock">
Stegmaier, J., Otte, J.C., Kobitski, A., Bartschat, A., Garcia, A., Nienhaus, G.U., Str√§hle, U., Mikut, R.: Fast segmentation of stained nuclei in terabyte-scale, time resolved 3d microscopy image stacks. PLoS ONE <span class="ltx_text ltx_font_bold" id="bib.bib14.1.1">9</span>(2), e90036 (02 2014)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib15">
<span class="ltx_tag ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock">
Tsuji, K., Lu, H., Tan, J.K., Kim, H., Yoneda, K., Tanaka, F.: Detection of circulating tumor cells in fluorescence microscopy images based on ann classifier. Mobile Networks and Applications <span class="ltx_text ltx_font_bold" id="bib.bib15.1.1">25</span>, 1042‚Äì1051 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib16">
<span class="ltx_tag ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock">
Vidlarova, M., Rehulkova, A., Stejskal, P., Prokopova, A., Slavik, H., Hajduch, M., Srovnal, J.: Recent advances in methods for circulating tumor cell detection. International Journal of Molecular Sciences <span class="ltx_text ltx_font_bold" id="bib.bib16.1.1">24</span>(4), ¬†3902 (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib17">
<span class="ltx_tag ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock">
Warfield, S.K., Zou, K.H., Wells, W.M.: Simultaneous truth and performance level estimation (staple): an algorithm for the validation of image segmentation. IEEE transactions on medical imaging <span class="ltx_text ltx_font_bold" id="bib.bib17.1.1">23</span>(7), 903‚Äì921 (2004)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib18">
<span class="ltx_tag ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock">
Werner, S.L., Graf, R.P., Landers, M., Valenta, D.T., Schroeder, M., Greene, S.B., Bales, N., Dittamore, R., Marrinucci, D.: Analytical validation and capabilities of the Epic CTC platform: enrichment-free circulating tumour cell detection and characterization. Journal of Circulating Biomarkers <span class="ltx_text ltx_font_bold" id="bib.bib18.1.1">4</span>, ¬†3 (2015)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib19">
<span class="ltx_tag ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock">
Wilson, M.: Introduction to widefield microscopy (Jun 2017), <a class="ltx_ref" href="https://www.leica-microsystems.com/science-lab/introduction-to-widefield-microscopy/" title="">https://www.leica-microsystems.com/science-lab/introduction-to-widefield-microscopy/</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib20">
<span class="ltx_tag ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock">
Zeune, L.L., Boink, Y.E., van Dalum, G., Nanou, A., de¬†Wit, S., Andree, K.C., Swennenhuis, J.F., van Gils, S.A., Terstappen, L.W., Brune, C.: Deep learning of circulating tumour cells. Nature Machine Intelligence <span class="ltx_text ltx_font_bold" id="bib.bib20.1.1">2</span>(2), 124‚Äì133 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib21">
<span class="ltx_tag ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock">
Zhang, A., Zou, Z., Liu, Y., Chen, Y., Yang, Y., Chen, Y., Law, B.N.F.: Automated detection of circulating tumor cells using faster region convolution neural network. Journal of Medical Imaging and Health Informatics <span class="ltx_text ltx_font_bold" id="bib.bib21.1.1">9</span>(1), 167‚Äì174 (2019)

</span>
</li>
</ul>
</section>
<section class="ltx_section" id="Sx1">
<h2 class="ltx_title ltx_title_section">Supplementary</h2>
<figure class="ltx_table" id="Sx1.T1">
<table class="ltx_tabular ltx_centering ltx_align_middle" id="Sx1.T1.23">
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="Sx1.T1.23.24.1" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.23.24.1.1"><span class="ltx_text ltx_font_bold" id="Sx1.T1.23.24.1.1.1" style="background-color:#F2F2F2;">Object</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.24.1.2"><span class="ltx_text ltx_font_bold" id="Sx1.T1.23.24.1.2.1" style="background-color:#F2F2F2;">Feature</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.24.1.3"><span class="ltx_text ltx_font_bold" id="Sx1.T1.23.24.1.3.1" style="background-color:#F2F2F2;">Dim</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.24.1.4"><span class="ltx_text ltx_font_bold" id="Sx1.T1.23.24.1.4.1" style="background-color:#F2F2F2;">Equation</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.24.1.5"><span class="ltx_text ltx_font_bold" id="Sx1.T1.23.24.1.5.1" style="background-color:#F2F2F2;">Description</span></td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.25.2" style="background-color:#B5DEE8;">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_tt" colspan="5" id="Sx1.T1.23.25.2.1"><span class="ltx_text ltx_font_bold" id="Sx1.T1.23.25.2.1.1" style="background-color:#B5DEE8;">Morphology Features (8)</span></td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.2.2">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.2.2.3"><span class="ltx_text" id="Sx1.T1.2.2.3.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.2.2.4"><span class="ltx_text" id="Sx1.T1.2.2.4.1">Size</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.2.2.5"><span class="ltx_text" id="Sx1.T1.2.2.5.1">2</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.1.1.1"><span class="ltx_text" id="Sx1.T1.1.1.1.1"><math alttext="A=|O|" class="ltx_Math" display="inline" id="Sx1.T1.1.1.1.1.m1.1"><semantics id="Sx1.T1.1.1.1.1.m1.1a"><mrow id="Sx1.T1.1.1.1.1.m1.1.2" xref="Sx1.T1.1.1.1.1.m1.1.2.cmml"><mi id="Sx1.T1.1.1.1.1.m1.1.2.2" xref="Sx1.T1.1.1.1.1.m1.1.2.2.cmml">A</mi><mo id="Sx1.T1.1.1.1.1.m1.1.2.1" xref="Sx1.T1.1.1.1.1.m1.1.2.1.cmml">=</mo><mrow id="Sx1.T1.1.1.1.1.m1.1.2.3.2" xref="Sx1.T1.1.1.1.1.m1.1.2.3.1.cmml"><mo id="Sx1.T1.1.1.1.1.m1.1.2.3.2.1" stretchy="false" xref="Sx1.T1.1.1.1.1.m1.1.2.3.1.1.cmml">|</mo><mi id="Sx1.T1.1.1.1.1.m1.1.1" xref="Sx1.T1.1.1.1.1.m1.1.1.cmml">O</mi><mo id="Sx1.T1.1.1.1.1.m1.1.2.3.2.2" stretchy="false" xref="Sx1.T1.1.1.1.1.m1.1.2.3.1.1.cmml">|</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.1.1.1.1.m1.1b"><apply id="Sx1.T1.1.1.1.1.m1.1.2.cmml" xref="Sx1.T1.1.1.1.1.m1.1.2"><eq id="Sx1.T1.1.1.1.1.m1.1.2.1.cmml" xref="Sx1.T1.1.1.1.1.m1.1.2.1"></eq><ci id="Sx1.T1.1.1.1.1.m1.1.2.2.cmml" xref="Sx1.T1.1.1.1.1.m1.1.2.2">ùê¥</ci><apply id="Sx1.T1.1.1.1.1.m1.1.2.3.1.cmml" xref="Sx1.T1.1.1.1.1.m1.1.2.3.2"><abs id="Sx1.T1.1.1.1.1.m1.1.2.3.1.1.cmml" xref="Sx1.T1.1.1.1.1.m1.1.2.3.2.1"></abs><ci id="Sx1.T1.1.1.1.1.m1.1.1.cmml" xref="Sx1.T1.1.1.1.1.m1.1.1">ùëÇ</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.1.1.1.1.m1.1c">A=|O|</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.1.1.1.1.m1.1d">italic_A = | italic_O |</annotation></semantics></math></span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.2.2.2">Area of object <math alttext="O" class="ltx_Math" display="inline" id="Sx1.T1.2.2.2.m1.1"><semantics id="Sx1.T1.2.2.2.m1.1a"><mi id="Sx1.T1.2.2.2.m1.1.1" xref="Sx1.T1.2.2.2.m1.1.1.cmml">O</mi><annotation-xml encoding="MathML-Content" id="Sx1.T1.2.2.2.m1.1b"><ci id="Sx1.T1.2.2.2.m1.1.1.cmml" xref="Sx1.T1.2.2.2.m1.1.1">ùëÇ</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.2.2.2.m1.1c">O</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.2.2.2.m1.1d">italic_O</annotation></semantics></math>.</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.3.3">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.3.3.2" rowspan="2"><span class="ltx_text" id="Sx1.T1.3.3.2.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.3.3.3" rowspan="2"><span class="ltx_text" id="Sx1.T1.3.3.3.1">Roundness</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.3.3.4" rowspan="2"><span class="ltx_text" id="Sx1.T1.3.3.4.1">2</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.3.3.1" rowspan="2"><span class="ltx_text" id="Sx1.T1.3.3.1.1"><math alttext="R=4\pi A/C^{2}_{p}" class="ltx_Math" display="inline" id="Sx1.T1.3.3.1.1.m1.1"><semantics id="Sx1.T1.3.3.1.1.m1.1a"><mrow id="Sx1.T1.3.3.1.1.m1.1.1" xref="Sx1.T1.3.3.1.1.m1.1.1.cmml"><mi id="Sx1.T1.3.3.1.1.m1.1.1.2" xref="Sx1.T1.3.3.1.1.m1.1.1.2.cmml">R</mi><mo id="Sx1.T1.3.3.1.1.m1.1.1.1" xref="Sx1.T1.3.3.1.1.m1.1.1.1.cmml">=</mo><mrow id="Sx1.T1.3.3.1.1.m1.1.1.3" xref="Sx1.T1.3.3.1.1.m1.1.1.3.cmml"><mrow id="Sx1.T1.3.3.1.1.m1.1.1.3.2" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.cmml"><mn id="Sx1.T1.3.3.1.1.m1.1.1.3.2.2" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.2.cmml">4</mn><mo id="Sx1.T1.3.3.1.1.m1.1.1.3.2.1" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.1.cmml">‚Å¢</mo><mi id="Sx1.T1.3.3.1.1.m1.1.1.3.2.3" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.3.cmml">œÄ</mi><mo id="Sx1.T1.3.3.1.1.m1.1.1.3.2.1a" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.1.cmml">‚Å¢</mo><mi id="Sx1.T1.3.3.1.1.m1.1.1.3.2.4" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.4.cmml">A</mi></mrow><mo id="Sx1.T1.3.3.1.1.m1.1.1.3.1" xref="Sx1.T1.3.3.1.1.m1.1.1.3.1.cmml">/</mo><msubsup id="Sx1.T1.3.3.1.1.m1.1.1.3.3" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3.cmml"><mi id="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.2" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.2.cmml">C</mi><mi id="Sx1.T1.3.3.1.1.m1.1.1.3.3.3" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3.3.cmml">p</mi><mn id="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.3" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.3.cmml">2</mn></msubsup></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.3.3.1.1.m1.1b"><apply id="Sx1.T1.3.3.1.1.m1.1.1.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1"><eq id="Sx1.T1.3.3.1.1.m1.1.1.1.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.1"></eq><ci id="Sx1.T1.3.3.1.1.m1.1.1.2.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.2">ùëÖ</ci><apply id="Sx1.T1.3.3.1.1.m1.1.1.3.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3"><divide id="Sx1.T1.3.3.1.1.m1.1.1.3.1.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.1"></divide><apply id="Sx1.T1.3.3.1.1.m1.1.1.3.2.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2"><times id="Sx1.T1.3.3.1.1.m1.1.1.3.2.1.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.1"></times><cn id="Sx1.T1.3.3.1.1.m1.1.1.3.2.2.cmml" type="integer" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.2">4</cn><ci id="Sx1.T1.3.3.1.1.m1.1.1.3.2.3.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.3">ùúã</ci><ci id="Sx1.T1.3.3.1.1.m1.1.1.3.2.4.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.2.4">ùê¥</ci></apply><apply id="Sx1.T1.3.3.1.1.m1.1.1.3.3.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3"><csymbol cd="ambiguous" id="Sx1.T1.3.3.1.1.m1.1.1.3.3.1.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3">subscript</csymbol><apply id="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3"><csymbol cd="ambiguous" id="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.1.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3">superscript</csymbol><ci id="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.2.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.2">ùê∂</ci><cn id="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.3.cmml" type="integer" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3.2.3">2</cn></apply><ci id="Sx1.T1.3.3.1.1.m1.1.1.3.3.3.cmml" xref="Sx1.T1.3.3.1.1.m1.1.1.3.3.3">ùëù</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.3.3.1.1.m1.1c">R=4\pi A/C^{2}_{p}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.3.3.1.1.m1.1d">italic_R = 4 italic_œÄ italic_A / italic_C start_POSTSUPERSCRIPT 2 end_POSTSUPERSCRIPT start_POSTSUBSCRIPT italic_p end_POSTSUBSCRIPT</annotation></semantics></math></span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.3.3.5">Closeness to a circle.</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.4.4">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.4.4.1">
<math alttext="C_{p}" class="ltx_Math" display="inline" id="Sx1.T1.4.4.1.m1.1"><semantics id="Sx1.T1.4.4.1.m1.1a"><msub id="Sx1.T1.4.4.1.m1.1.1" xref="Sx1.T1.4.4.1.m1.1.1.cmml"><mi id="Sx1.T1.4.4.1.m1.1.1.2" xref="Sx1.T1.4.4.1.m1.1.1.2.cmml">C</mi><mi id="Sx1.T1.4.4.1.m1.1.1.3" xref="Sx1.T1.4.4.1.m1.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx1.T1.4.4.1.m1.1b"><apply id="Sx1.T1.4.4.1.m1.1.1.cmml" xref="Sx1.T1.4.4.1.m1.1.1"><csymbol cd="ambiguous" id="Sx1.T1.4.4.1.m1.1.1.1.cmml" xref="Sx1.T1.4.4.1.m1.1.1">subscript</csymbol><ci id="Sx1.T1.4.4.1.m1.1.1.2.cmml" xref="Sx1.T1.4.4.1.m1.1.1.2">ùê∂</ci><ci id="Sx1.T1.4.4.1.m1.1.1.3.cmml" xref="Sx1.T1.4.4.1.m1.1.1.3">ùëù</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.4.4.1.m1.1c">C_{p}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.4.4.1.m1.1d">italic_C start_POSTSUBSCRIPT italic_p end_POSTSUBSCRIPT</annotation></semantics></math>, convex perimeter.</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.6.6">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.6.6.3"><span class="ltx_text" id="Sx1.T1.6.6.3.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.6.6.4"><span class="ltx_text" id="Sx1.T1.6.6.4.1">Elongation</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.6.6.5"><span class="ltx_text" id="Sx1.T1.6.6.5.1">2</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.5.5.1"><span class="ltx_text" id="Sx1.T1.5.5.1.1"><math alttext="E=4A/\pi l_{m}^{2}" class="ltx_Math" display="inline" id="Sx1.T1.5.5.1.1.m1.1"><semantics id="Sx1.T1.5.5.1.1.m1.1a"><mrow id="Sx1.T1.5.5.1.1.m1.1.1" xref="Sx1.T1.5.5.1.1.m1.1.1.cmml"><mi id="Sx1.T1.5.5.1.1.m1.1.1.2" xref="Sx1.T1.5.5.1.1.m1.1.1.2.cmml">E</mi><mo id="Sx1.T1.5.5.1.1.m1.1.1.1" xref="Sx1.T1.5.5.1.1.m1.1.1.1.cmml">=</mo><mrow id="Sx1.T1.5.5.1.1.m1.1.1.3" xref="Sx1.T1.5.5.1.1.m1.1.1.3.cmml"><mrow id="Sx1.T1.5.5.1.1.m1.1.1.3.2" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.cmml"><mrow id="Sx1.T1.5.5.1.1.m1.1.1.3.2.2" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.cmml"><mn id="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.2" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.2.cmml">4</mn><mo id="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.1" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.1.cmml">‚Å¢</mo><mi id="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.3" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.3.cmml">A</mi></mrow><mo id="Sx1.T1.5.5.1.1.m1.1.1.3.2.1" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.1.cmml">/</mo><mi id="Sx1.T1.5.5.1.1.m1.1.1.3.2.3" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.3.cmml">œÄ</mi></mrow><mo id="Sx1.T1.5.5.1.1.m1.1.1.3.1" xref="Sx1.T1.5.5.1.1.m1.1.1.3.1.cmml">‚Å¢</mo><msubsup id="Sx1.T1.5.5.1.1.m1.1.1.3.3" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3.cmml"><mi id="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.2" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.2.cmml">l</mi><mi id="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.3" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.3.cmml">m</mi><mn id="Sx1.T1.5.5.1.1.m1.1.1.3.3.3" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3.3.cmml">2</mn></msubsup></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.5.5.1.1.m1.1b"><apply id="Sx1.T1.5.5.1.1.m1.1.1.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1"><eq id="Sx1.T1.5.5.1.1.m1.1.1.1.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.1"></eq><ci id="Sx1.T1.5.5.1.1.m1.1.1.2.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.2">ùê∏</ci><apply id="Sx1.T1.5.5.1.1.m1.1.1.3.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3"><times id="Sx1.T1.5.5.1.1.m1.1.1.3.1.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.1"></times><apply id="Sx1.T1.5.5.1.1.m1.1.1.3.2.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2"><divide id="Sx1.T1.5.5.1.1.m1.1.1.3.2.1.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.1"></divide><apply id="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.2"><times id="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.1.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.1"></times><cn id="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.2.cmml" type="integer" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.2">4</cn><ci id="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.3.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.2.3">ùê¥</ci></apply><ci id="Sx1.T1.5.5.1.1.m1.1.1.3.2.3.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.2.3">ùúã</ci></apply><apply id="Sx1.T1.5.5.1.1.m1.1.1.3.3.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3"><csymbol cd="ambiguous" id="Sx1.T1.5.5.1.1.m1.1.1.3.3.1.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3">superscript</csymbol><apply id="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3"><csymbol cd="ambiguous" id="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.1.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3">subscript</csymbol><ci id="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.2.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.2">ùëô</ci><ci id="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.3.cmml" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3.2.3">ùëö</ci></apply><cn id="Sx1.T1.5.5.1.1.m1.1.1.3.3.3.cmml" type="integer" xref="Sx1.T1.5.5.1.1.m1.1.1.3.3.3">2</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.5.5.1.1.m1.1c">E=4A/\pi l_{m}^{2}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.5.5.1.1.m1.1d">italic_E = 4 italic_A / italic_œÄ italic_l start_POSTSUBSCRIPT italic_m end_POSTSUBSCRIPT start_POSTSUPERSCRIPT 2 end_POSTSUPERSCRIPT</annotation></semantics></math></span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.6.6.2">Area to major axis <math alttext="l_{m}" class="ltx_Math" display="inline" id="Sx1.T1.6.6.2.m1.1"><semantics id="Sx1.T1.6.6.2.m1.1a"><msub id="Sx1.T1.6.6.2.m1.1.1" xref="Sx1.T1.6.6.2.m1.1.1.cmml"><mi id="Sx1.T1.6.6.2.m1.1.1.2" xref="Sx1.T1.6.6.2.m1.1.1.2.cmml">l</mi><mi id="Sx1.T1.6.6.2.m1.1.1.3" xref="Sx1.T1.6.6.2.m1.1.1.3.cmml">m</mi></msub><annotation-xml encoding="MathML-Content" id="Sx1.T1.6.6.2.m1.1b"><apply id="Sx1.T1.6.6.2.m1.1.1.cmml" xref="Sx1.T1.6.6.2.m1.1.1"><csymbol cd="ambiguous" id="Sx1.T1.6.6.2.m1.1.1.1.cmml" xref="Sx1.T1.6.6.2.m1.1.1">subscript</csymbol><ci id="Sx1.T1.6.6.2.m1.1.1.2.cmml" xref="Sx1.T1.6.6.2.m1.1.1.2">ùëô</ci><ci id="Sx1.T1.6.6.2.m1.1.1.3.cmml" xref="Sx1.T1.6.6.2.m1.1.1.3">ùëö</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.6.6.2.m1.1c">l_{m}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.6.6.2.m1.1d">italic_l start_POSTSUBSCRIPT italic_m end_POSTSUBSCRIPT</annotation></semantics></math>
</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.8.8">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.8.8.3" rowspan="2"><span class="ltx_text" id="Sx1.T1.8.8.3.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.7.7.1">
<math alttext="1^{\text{st}}" class="ltx_Math" display="inline" id="Sx1.T1.7.7.1.m1.1"><semantics id="Sx1.T1.7.7.1.m1.1a"><msup id="Sx1.T1.7.7.1.m1.1.1" xref="Sx1.T1.7.7.1.m1.1.1.cmml"><mn id="Sx1.T1.7.7.1.m1.1.1.2" xref="Sx1.T1.7.7.1.m1.1.1.2.cmml">1</mn><mtext id="Sx1.T1.7.7.1.m1.1.1.3" xref="Sx1.T1.7.7.1.m1.1.1.3a.cmml">st</mtext></msup><annotation-xml encoding="MathML-Content" id="Sx1.T1.7.7.1.m1.1b"><apply id="Sx1.T1.7.7.1.m1.1.1.cmml" xref="Sx1.T1.7.7.1.m1.1.1"><csymbol cd="ambiguous" id="Sx1.T1.7.7.1.m1.1.1.1.cmml" xref="Sx1.T1.7.7.1.m1.1.1">superscript</csymbol><cn id="Sx1.T1.7.7.1.m1.1.1.2.cmml" type="integer" xref="Sx1.T1.7.7.1.m1.1.1.2">1</cn><ci id="Sx1.T1.7.7.1.m1.1.1.3a.cmml" xref="Sx1.T1.7.7.1.m1.1.1.3"><mtext id="Sx1.T1.7.7.1.m1.1.1.3.cmml" mathsize="70%" xref="Sx1.T1.7.7.1.m1.1.1.3">st</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.7.7.1.m1.1c">1^{\text{st}}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.7.7.1.m1.1d">1 start_POSTSUPERSCRIPT st end_POSTSUPERSCRIPT</annotation></semantics></math> Hu</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.8.8.4" rowspan="2"><span class="ltx_text" id="Sx1.T1.8.8.4.1">2</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.8.8.2" rowspan="2"><span class="ltx_text" id="Sx1.T1.8.8.2.1"><math alttext="M_{1}=\mu_{2,0}+\mu_{0,2}" class="ltx_Math" display="inline" id="Sx1.T1.8.8.2.1.m1.4"><semantics id="Sx1.T1.8.8.2.1.m1.4a"><mrow id="Sx1.T1.8.8.2.1.m1.4.5" xref="Sx1.T1.8.8.2.1.m1.4.5.cmml"><msub id="Sx1.T1.8.8.2.1.m1.4.5.2" xref="Sx1.T1.8.8.2.1.m1.4.5.2.cmml"><mi id="Sx1.T1.8.8.2.1.m1.4.5.2.2" xref="Sx1.T1.8.8.2.1.m1.4.5.2.2.cmml">M</mi><mn id="Sx1.T1.8.8.2.1.m1.4.5.2.3" xref="Sx1.T1.8.8.2.1.m1.4.5.2.3.cmml">1</mn></msub><mo id="Sx1.T1.8.8.2.1.m1.4.5.1" xref="Sx1.T1.8.8.2.1.m1.4.5.1.cmml">=</mo><mrow id="Sx1.T1.8.8.2.1.m1.4.5.3" xref="Sx1.T1.8.8.2.1.m1.4.5.3.cmml"><msub id="Sx1.T1.8.8.2.1.m1.4.5.3.2" xref="Sx1.T1.8.8.2.1.m1.4.5.3.2.cmml"><mi id="Sx1.T1.8.8.2.1.m1.4.5.3.2.2" xref="Sx1.T1.8.8.2.1.m1.4.5.3.2.2.cmml">Œº</mi><mrow id="Sx1.T1.8.8.2.1.m1.2.2.2.4" xref="Sx1.T1.8.8.2.1.m1.2.2.2.3.cmml"><mn id="Sx1.T1.8.8.2.1.m1.1.1.1.1" xref="Sx1.T1.8.8.2.1.m1.1.1.1.1.cmml">2</mn><mo id="Sx1.T1.8.8.2.1.m1.2.2.2.4.1" xref="Sx1.T1.8.8.2.1.m1.2.2.2.3.cmml">,</mo><mn id="Sx1.T1.8.8.2.1.m1.2.2.2.2" xref="Sx1.T1.8.8.2.1.m1.2.2.2.2.cmml">0</mn></mrow></msub><mo id="Sx1.T1.8.8.2.1.m1.4.5.3.1" xref="Sx1.T1.8.8.2.1.m1.4.5.3.1.cmml">+</mo><msub id="Sx1.T1.8.8.2.1.m1.4.5.3.3" xref="Sx1.T1.8.8.2.1.m1.4.5.3.3.cmml"><mi id="Sx1.T1.8.8.2.1.m1.4.5.3.3.2" xref="Sx1.T1.8.8.2.1.m1.4.5.3.3.2.cmml">Œº</mi><mrow id="Sx1.T1.8.8.2.1.m1.4.4.2.4" xref="Sx1.T1.8.8.2.1.m1.4.4.2.3.cmml"><mn id="Sx1.T1.8.8.2.1.m1.3.3.1.1" xref="Sx1.T1.8.8.2.1.m1.3.3.1.1.cmml">0</mn><mo id="Sx1.T1.8.8.2.1.m1.4.4.2.4.1" xref="Sx1.T1.8.8.2.1.m1.4.4.2.3.cmml">,</mo><mn id="Sx1.T1.8.8.2.1.m1.4.4.2.2" xref="Sx1.T1.8.8.2.1.m1.4.4.2.2.cmml">2</mn></mrow></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.8.8.2.1.m1.4b"><apply id="Sx1.T1.8.8.2.1.m1.4.5.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5"><eq id="Sx1.T1.8.8.2.1.m1.4.5.1.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.1"></eq><apply id="Sx1.T1.8.8.2.1.m1.4.5.2.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.2"><csymbol cd="ambiguous" id="Sx1.T1.8.8.2.1.m1.4.5.2.1.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.2">subscript</csymbol><ci id="Sx1.T1.8.8.2.1.m1.4.5.2.2.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.2.2">ùëÄ</ci><cn id="Sx1.T1.8.8.2.1.m1.4.5.2.3.cmml" type="integer" xref="Sx1.T1.8.8.2.1.m1.4.5.2.3">1</cn></apply><apply id="Sx1.T1.8.8.2.1.m1.4.5.3.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.3"><plus id="Sx1.T1.8.8.2.1.m1.4.5.3.1.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.3.1"></plus><apply id="Sx1.T1.8.8.2.1.m1.4.5.3.2.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.3.2"><csymbol cd="ambiguous" id="Sx1.T1.8.8.2.1.m1.4.5.3.2.1.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.3.2">subscript</csymbol><ci id="Sx1.T1.8.8.2.1.m1.4.5.3.2.2.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.3.2.2">ùúá</ci><list id="Sx1.T1.8.8.2.1.m1.2.2.2.3.cmml" xref="Sx1.T1.8.8.2.1.m1.2.2.2.4"><cn id="Sx1.T1.8.8.2.1.m1.1.1.1.1.cmml" type="integer" xref="Sx1.T1.8.8.2.1.m1.1.1.1.1">2</cn><cn id="Sx1.T1.8.8.2.1.m1.2.2.2.2.cmml" type="integer" xref="Sx1.T1.8.8.2.1.m1.2.2.2.2">0</cn></list></apply><apply id="Sx1.T1.8.8.2.1.m1.4.5.3.3.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.3.3"><csymbol cd="ambiguous" id="Sx1.T1.8.8.2.1.m1.4.5.3.3.1.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.3.3">subscript</csymbol><ci id="Sx1.T1.8.8.2.1.m1.4.5.3.3.2.cmml" xref="Sx1.T1.8.8.2.1.m1.4.5.3.3.2">ùúá</ci><list id="Sx1.T1.8.8.2.1.m1.4.4.2.3.cmml" xref="Sx1.T1.8.8.2.1.m1.4.4.2.4"><cn id="Sx1.T1.8.8.2.1.m1.3.3.1.1.cmml" type="integer" xref="Sx1.T1.8.8.2.1.m1.3.3.1.1">0</cn><cn id="Sx1.T1.8.8.2.1.m1.4.4.2.2.cmml" type="integer" xref="Sx1.T1.8.8.2.1.m1.4.4.2.2">2</cn></list></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.8.8.2.1.m1.4c">M_{1}=\mu_{2,0}+\mu_{0,2}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.8.8.2.1.m1.4d">italic_M start_POSTSUBSCRIPT 1 end_POSTSUBSCRIPT = italic_Œº start_POSTSUBSCRIPT 2 , 0 end_POSTSUBSCRIPT + italic_Œº start_POSTSUBSCRIPT 0 , 2 end_POSTSUBSCRIPT</annotation></semantics></math></span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.8.8.5">Shape descriptor using</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.9.9">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.9.9.2">Moment</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.9.9.1">moments, <math alttext="\mu_{0,2},\mu_{2,0}" class="ltx_Math" display="inline" id="Sx1.T1.9.9.1.m1.6"><semantics id="Sx1.T1.9.9.1.m1.6a"><mrow id="Sx1.T1.9.9.1.m1.6.6.2" xref="Sx1.T1.9.9.1.m1.6.6.3.cmml"><msub id="Sx1.T1.9.9.1.m1.5.5.1.1" xref="Sx1.T1.9.9.1.m1.5.5.1.1.cmml"><mi id="Sx1.T1.9.9.1.m1.5.5.1.1.2" xref="Sx1.T1.9.9.1.m1.5.5.1.1.2.cmml">Œº</mi><mrow id="Sx1.T1.9.9.1.m1.2.2.2.4" xref="Sx1.T1.9.9.1.m1.2.2.2.3.cmml"><mn id="Sx1.T1.9.9.1.m1.1.1.1.1" xref="Sx1.T1.9.9.1.m1.1.1.1.1.cmml">0</mn><mo id="Sx1.T1.9.9.1.m1.2.2.2.4.1" xref="Sx1.T1.9.9.1.m1.2.2.2.3.cmml">,</mo><mn id="Sx1.T1.9.9.1.m1.2.2.2.2" xref="Sx1.T1.9.9.1.m1.2.2.2.2.cmml">2</mn></mrow></msub><mo id="Sx1.T1.9.9.1.m1.6.6.2.3" xref="Sx1.T1.9.9.1.m1.6.6.3.cmml">,</mo><msub id="Sx1.T1.9.9.1.m1.6.6.2.2" xref="Sx1.T1.9.9.1.m1.6.6.2.2.cmml"><mi id="Sx1.T1.9.9.1.m1.6.6.2.2.2" xref="Sx1.T1.9.9.1.m1.6.6.2.2.2.cmml">Œº</mi><mrow id="Sx1.T1.9.9.1.m1.4.4.2.4" xref="Sx1.T1.9.9.1.m1.4.4.2.3.cmml"><mn id="Sx1.T1.9.9.1.m1.3.3.1.1" xref="Sx1.T1.9.9.1.m1.3.3.1.1.cmml">2</mn><mo id="Sx1.T1.9.9.1.m1.4.4.2.4.1" xref="Sx1.T1.9.9.1.m1.4.4.2.3.cmml">,</mo><mn id="Sx1.T1.9.9.1.m1.4.4.2.2" xref="Sx1.T1.9.9.1.m1.4.4.2.2.cmml">0</mn></mrow></msub></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.9.9.1.m1.6b"><list id="Sx1.T1.9.9.1.m1.6.6.3.cmml" xref="Sx1.T1.9.9.1.m1.6.6.2"><apply id="Sx1.T1.9.9.1.m1.5.5.1.1.cmml" xref="Sx1.T1.9.9.1.m1.5.5.1.1"><csymbol cd="ambiguous" id="Sx1.T1.9.9.1.m1.5.5.1.1.1.cmml" xref="Sx1.T1.9.9.1.m1.5.5.1.1">subscript</csymbol><ci id="Sx1.T1.9.9.1.m1.5.5.1.1.2.cmml" xref="Sx1.T1.9.9.1.m1.5.5.1.1.2">ùúá</ci><list id="Sx1.T1.9.9.1.m1.2.2.2.3.cmml" xref="Sx1.T1.9.9.1.m1.2.2.2.4"><cn id="Sx1.T1.9.9.1.m1.1.1.1.1.cmml" type="integer" xref="Sx1.T1.9.9.1.m1.1.1.1.1">0</cn><cn id="Sx1.T1.9.9.1.m1.2.2.2.2.cmml" type="integer" xref="Sx1.T1.9.9.1.m1.2.2.2.2">2</cn></list></apply><apply id="Sx1.T1.9.9.1.m1.6.6.2.2.cmml" xref="Sx1.T1.9.9.1.m1.6.6.2.2"><csymbol cd="ambiguous" id="Sx1.T1.9.9.1.m1.6.6.2.2.1.cmml" xref="Sx1.T1.9.9.1.m1.6.6.2.2">subscript</csymbol><ci id="Sx1.T1.9.9.1.m1.6.6.2.2.2.cmml" xref="Sx1.T1.9.9.1.m1.6.6.2.2.2">ùúá</ci><list id="Sx1.T1.9.9.1.m1.4.4.2.3.cmml" xref="Sx1.T1.9.9.1.m1.4.4.2.4"><cn id="Sx1.T1.9.9.1.m1.3.3.1.1.cmml" type="integer" xref="Sx1.T1.9.9.1.m1.3.3.1.1">2</cn><cn id="Sx1.T1.9.9.1.m1.4.4.2.2.cmml" type="integer" xref="Sx1.T1.9.9.1.m1.4.4.2.2">0</cn></list></apply></list></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.9.9.1.m1.6c">\mu_{0,2},\mu_{2,0}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.9.9.1.m1.6d">italic_Œº start_POSTSUBSCRIPT 0 , 2 end_POSTSUBSCRIPT , italic_Œº start_POSTSUBSCRIPT 2 , 0 end_POSTSUBSCRIPT</annotation></semantics></math>
</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.26.3" style="background-color:#B5DEE8;">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_tt" colspan="5" id="Sx1.T1.23.26.3.1"><span class="ltx_text ltx_font_bold" id="Sx1.T1.23.26.3.1.1" style="background-color:#B5DEE8;">Intensity Features (44)</span></td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.10.10">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.10.10.2" rowspan="2"><span class="ltx_text" id="Sx1.T1.10.10.2.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.10.10.3" rowspan="2"><span class="ltx_text" id="Sx1.T1.10.10.3.1">MFI</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.10.10.4" rowspan="2"><span class="ltx_text" id="Sx1.T1.10.10.4.1">6</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.10.10.1" rowspan="2"><span class="ltx_text" id="Sx1.T1.10.10.1.1"><math alttext="\text{MFI}_{ch}=\frac{1}{A}\sum_{p\in O}I_{ch}(p)" class="ltx_Math" display="inline" id="Sx1.T1.10.10.1.1.m1.1"><semantics id="Sx1.T1.10.10.1.1.m1.1a"><mrow id="Sx1.T1.10.10.1.1.m1.1.2" xref="Sx1.T1.10.10.1.1.m1.1.2.cmml"><msub id="Sx1.T1.10.10.1.1.m1.1.2.2" xref="Sx1.T1.10.10.1.1.m1.1.2.2.cmml"><mtext id="Sx1.T1.10.10.1.1.m1.1.2.2.2" xref="Sx1.T1.10.10.1.1.m1.1.2.2.2a.cmml">MFI</mtext><mrow id="Sx1.T1.10.10.1.1.m1.1.2.2.3" xref="Sx1.T1.10.10.1.1.m1.1.2.2.3.cmml"><mi id="Sx1.T1.10.10.1.1.m1.1.2.2.3.2" xref="Sx1.T1.10.10.1.1.m1.1.2.2.3.2.cmml">c</mi><mo id="Sx1.T1.10.10.1.1.m1.1.2.2.3.1" xref="Sx1.T1.10.10.1.1.m1.1.2.2.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.10.10.1.1.m1.1.2.2.3.3" xref="Sx1.T1.10.10.1.1.m1.1.2.2.3.3.cmml">h</mi></mrow></msub><mo id="Sx1.T1.10.10.1.1.m1.1.2.1" xref="Sx1.T1.10.10.1.1.m1.1.2.1.cmml">=</mo><mrow id="Sx1.T1.10.10.1.1.m1.1.2.3" xref="Sx1.T1.10.10.1.1.m1.1.2.3.cmml"><mfrac id="Sx1.T1.10.10.1.1.m1.1.2.3.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.2.cmml"><mn id="Sx1.T1.10.10.1.1.m1.1.2.3.2.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.2.2.cmml">1</mn><mi id="Sx1.T1.10.10.1.1.m1.1.2.3.2.3" xref="Sx1.T1.10.10.1.1.m1.1.2.3.2.3.cmml">A</mi></mfrac><mo id="Sx1.T1.10.10.1.1.m1.1.2.3.1" xref="Sx1.T1.10.10.1.1.m1.1.2.3.1.cmml">‚Å¢</mo><mrow id="Sx1.T1.10.10.1.1.m1.1.2.3.3" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.cmml"><msub id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.cmml"><mo id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.2.cmml">‚àë</mo><mrow id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.cmml"><mi id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.2.cmml">p</mi><mo id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.1" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.1.cmml">‚àà</mo><mi id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.3" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.3.cmml">O</mi></mrow></msub><mrow id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.cmml"><msub id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.cmml"><mi id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.2.cmml">I</mi><mrow id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.cmml"><mi id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.2.cmml">c</mi><mo id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.1" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.3" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.3.cmml">h</mi></mrow></msub><mo id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.1" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.1.cmml">‚Å¢</mo><mrow id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.3.2" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.cmml"><mo id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.3.2.1" stretchy="false" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.cmml">(</mo><mi id="Sx1.T1.10.10.1.1.m1.1.1" xref="Sx1.T1.10.10.1.1.m1.1.1.cmml">p</mi><mo id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.3.2.2" stretchy="false" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.cmml">)</mo></mrow></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.10.10.1.1.m1.1b"><apply id="Sx1.T1.10.10.1.1.m1.1.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2"><eq id="Sx1.T1.10.10.1.1.m1.1.2.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.1"></eq><apply id="Sx1.T1.10.10.1.1.m1.1.2.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.2"><csymbol cd="ambiguous" id="Sx1.T1.10.10.1.1.m1.1.2.2.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.2">subscript</csymbol><ci id="Sx1.T1.10.10.1.1.m1.1.2.2.2a.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.2.2"><mtext id="Sx1.T1.10.10.1.1.m1.1.2.2.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.2.2">MFI</mtext></ci><apply id="Sx1.T1.10.10.1.1.m1.1.2.2.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.2.3"><times id="Sx1.T1.10.10.1.1.m1.1.2.2.3.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.2.3.1"></times><ci id="Sx1.T1.10.10.1.1.m1.1.2.2.3.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.2.3.2">ùëê</ci><ci id="Sx1.T1.10.10.1.1.m1.1.2.2.3.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.2.3.3">‚Ñé</ci></apply></apply><apply id="Sx1.T1.10.10.1.1.m1.1.2.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3"><times id="Sx1.T1.10.10.1.1.m1.1.2.3.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.1"></times><apply id="Sx1.T1.10.10.1.1.m1.1.2.3.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.2"><divide id="Sx1.T1.10.10.1.1.m1.1.2.3.2.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.2"></divide><cn id="Sx1.T1.10.10.1.1.m1.1.2.3.2.2.cmml" type="integer" xref="Sx1.T1.10.10.1.1.m1.1.2.3.2.2">1</cn><ci id="Sx1.T1.10.10.1.1.m1.1.2.3.2.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.2.3">ùê¥</ci></apply><apply id="Sx1.T1.10.10.1.1.m1.1.2.3.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3"><apply id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1"><csymbol cd="ambiguous" id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1">subscript</csymbol><sum id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.2"></sum><apply id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3"><in id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.1"></in><ci id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.2">ùëù</ci><ci id="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.1.3.3">ùëÇ</ci></apply></apply><apply id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2"><times id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.1"></times><apply id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2"><csymbol cd="ambiguous" id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2">subscript</csymbol><ci id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.2">ùêº</ci><apply id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3"><times id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.1"></times><ci id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.2.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.2">ùëê</ci><ci id="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.3.cmml" xref="Sx1.T1.10.10.1.1.m1.1.2.3.3.2.2.3.3">‚Ñé</ci></apply></apply><ci id="Sx1.T1.10.10.1.1.m1.1.1.cmml" xref="Sx1.T1.10.10.1.1.m1.1.1">ùëù</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.10.10.1.1.m1.1c">\text{MFI}_{ch}=\frac{1}{A}\sum_{p\in O}I_{ch}(p)</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.10.10.1.1.m1.1d">MFI start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT = divide start_ARG 1 end_ARG start_ARG italic_A end_ARG ‚àë start_POSTSUBSCRIPT italic_p ‚àà italic_O end_POSTSUBSCRIPT italic_I start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT ( italic_p )</annotation></semantics></math></span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.10.10.5">Mean Fluorescence Intensity</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.12.12">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.12.12.2">of object <math alttext="O" class="ltx_Math" display="inline" id="Sx1.T1.11.11.1.m1.1"><semantics id="Sx1.T1.11.11.1.m1.1a"><mi id="Sx1.T1.11.11.1.m1.1.1" xref="Sx1.T1.11.11.1.m1.1.1.cmml">O</mi><annotation-xml encoding="MathML-Content" id="Sx1.T1.11.11.1.m1.1b"><ci id="Sx1.T1.11.11.1.m1.1.1.cmml" xref="Sx1.T1.11.11.1.m1.1.1">ùëÇ</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.11.11.1.m1.1c">O</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.11.11.1.m1.1d">italic_O</annotation></semantics></math> for channel <math alttext="I_{ch}" class="ltx_Math" display="inline" id="Sx1.T1.12.12.2.m2.1"><semantics id="Sx1.T1.12.12.2.m2.1a"><msub id="Sx1.T1.12.12.2.m2.1.1" xref="Sx1.T1.12.12.2.m2.1.1.cmml"><mi id="Sx1.T1.12.12.2.m2.1.1.2" xref="Sx1.T1.12.12.2.m2.1.1.2.cmml">I</mi><mrow id="Sx1.T1.12.12.2.m2.1.1.3" xref="Sx1.T1.12.12.2.m2.1.1.3.cmml"><mi id="Sx1.T1.12.12.2.m2.1.1.3.2" xref="Sx1.T1.12.12.2.m2.1.1.3.2.cmml">c</mi><mo id="Sx1.T1.12.12.2.m2.1.1.3.1" xref="Sx1.T1.12.12.2.m2.1.1.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.12.12.2.m2.1.1.3.3" xref="Sx1.T1.12.12.2.m2.1.1.3.3.cmml">h</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx1.T1.12.12.2.m2.1b"><apply id="Sx1.T1.12.12.2.m2.1.1.cmml" xref="Sx1.T1.12.12.2.m2.1.1"><csymbol cd="ambiguous" id="Sx1.T1.12.12.2.m2.1.1.1.cmml" xref="Sx1.T1.12.12.2.m2.1.1">subscript</csymbol><ci id="Sx1.T1.12.12.2.m2.1.1.2.cmml" xref="Sx1.T1.12.12.2.m2.1.1.2">ùêº</ci><apply id="Sx1.T1.12.12.2.m2.1.1.3.cmml" xref="Sx1.T1.12.12.2.m2.1.1.3"><times id="Sx1.T1.12.12.2.m2.1.1.3.1.cmml" xref="Sx1.T1.12.12.2.m2.1.1.3.1"></times><ci id="Sx1.T1.12.12.2.m2.1.1.3.2.cmml" xref="Sx1.T1.12.12.2.m2.1.1.3.2">ùëê</ci><ci id="Sx1.T1.12.12.2.m2.1.1.3.3.cmml" xref="Sx1.T1.12.12.2.m2.1.1.3.3">‚Ñé</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.12.12.2.m2.1c">I_{ch}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.12.12.2.m2.1d">italic_I start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT</annotation></semantics></math>
</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.15.15">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.15.15.4" rowspan="2"><span class="ltx_text" id="Sx1.T1.15.15.4.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.15.15.5">Quartile</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.15.15.6" rowspan="2"><span class="ltx_text" id="Sx1.T1.15.15.6.1">24</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.15.15.3">
<math alttext="\text{LQI}_{ch}" class="ltx_Math" display="inline" id="Sx1.T1.13.13.1.m1.1"><semantics id="Sx1.T1.13.13.1.m1.1a"><msub id="Sx1.T1.13.13.1.m1.1.1" xref="Sx1.T1.13.13.1.m1.1.1.cmml"><mtext id="Sx1.T1.13.13.1.m1.1.1.2" xref="Sx1.T1.13.13.1.m1.1.1.2a.cmml">LQI</mtext><mrow id="Sx1.T1.13.13.1.m1.1.1.3" xref="Sx1.T1.13.13.1.m1.1.1.3.cmml"><mi id="Sx1.T1.13.13.1.m1.1.1.3.2" xref="Sx1.T1.13.13.1.m1.1.1.3.2.cmml">c</mi><mo id="Sx1.T1.13.13.1.m1.1.1.3.1" xref="Sx1.T1.13.13.1.m1.1.1.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.13.13.1.m1.1.1.3.3" xref="Sx1.T1.13.13.1.m1.1.1.3.3.cmml">h</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx1.T1.13.13.1.m1.1b"><apply id="Sx1.T1.13.13.1.m1.1.1.cmml" xref="Sx1.T1.13.13.1.m1.1.1"><csymbol cd="ambiguous" id="Sx1.T1.13.13.1.m1.1.1.1.cmml" xref="Sx1.T1.13.13.1.m1.1.1">subscript</csymbol><ci id="Sx1.T1.13.13.1.m1.1.1.2a.cmml" xref="Sx1.T1.13.13.1.m1.1.1.2"><mtext id="Sx1.T1.13.13.1.m1.1.1.2.cmml" xref="Sx1.T1.13.13.1.m1.1.1.2">LQI</mtext></ci><apply id="Sx1.T1.13.13.1.m1.1.1.3.cmml" xref="Sx1.T1.13.13.1.m1.1.1.3"><times id="Sx1.T1.13.13.1.m1.1.1.3.1.cmml" xref="Sx1.T1.13.13.1.m1.1.1.3.1"></times><ci id="Sx1.T1.13.13.1.m1.1.1.3.2.cmml" xref="Sx1.T1.13.13.1.m1.1.1.3.2">ùëê</ci><ci id="Sx1.T1.13.13.1.m1.1.1.3.3.cmml" xref="Sx1.T1.13.13.1.m1.1.1.3.3">‚Ñé</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.13.13.1.m1.1c">\text{LQI}_{ch}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.13.13.1.m1.1d">LQI start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT</annotation></semantics></math>, <math alttext="\text{MQI}_{ch}" class="ltx_Math" display="inline" id="Sx1.T1.14.14.2.m2.1"><semantics id="Sx1.T1.14.14.2.m2.1a"><msub id="Sx1.T1.14.14.2.m2.1.1" xref="Sx1.T1.14.14.2.m2.1.1.cmml"><mtext id="Sx1.T1.14.14.2.m2.1.1.2" xref="Sx1.T1.14.14.2.m2.1.1.2a.cmml">MQI</mtext><mrow id="Sx1.T1.14.14.2.m2.1.1.3" xref="Sx1.T1.14.14.2.m2.1.1.3.cmml"><mi id="Sx1.T1.14.14.2.m2.1.1.3.2" xref="Sx1.T1.14.14.2.m2.1.1.3.2.cmml">c</mi><mo id="Sx1.T1.14.14.2.m2.1.1.3.1" xref="Sx1.T1.14.14.2.m2.1.1.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.14.14.2.m2.1.1.3.3" xref="Sx1.T1.14.14.2.m2.1.1.3.3.cmml">h</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx1.T1.14.14.2.m2.1b"><apply id="Sx1.T1.14.14.2.m2.1.1.cmml" xref="Sx1.T1.14.14.2.m2.1.1"><csymbol cd="ambiguous" id="Sx1.T1.14.14.2.m2.1.1.1.cmml" xref="Sx1.T1.14.14.2.m2.1.1">subscript</csymbol><ci id="Sx1.T1.14.14.2.m2.1.1.2a.cmml" xref="Sx1.T1.14.14.2.m2.1.1.2"><mtext id="Sx1.T1.14.14.2.m2.1.1.2.cmml" xref="Sx1.T1.14.14.2.m2.1.1.2">MQI</mtext></ci><apply id="Sx1.T1.14.14.2.m2.1.1.3.cmml" xref="Sx1.T1.14.14.2.m2.1.1.3"><times id="Sx1.T1.14.14.2.m2.1.1.3.1.cmml" xref="Sx1.T1.14.14.2.m2.1.1.3.1"></times><ci id="Sx1.T1.14.14.2.m2.1.1.3.2.cmml" xref="Sx1.T1.14.14.2.m2.1.1.3.2">ùëê</ci><ci id="Sx1.T1.14.14.2.m2.1.1.3.3.cmml" xref="Sx1.T1.14.14.2.m2.1.1.3.3">‚Ñé</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.14.14.2.m2.1c">\text{MQI}_{ch}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.14.14.2.m2.1d">MQI start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT</annotation></semantics></math>, <math alttext="\text{UQI}_{ch}" class="ltx_Math" display="inline" id="Sx1.T1.15.15.3.m3.1"><semantics id="Sx1.T1.15.15.3.m3.1a"><msub id="Sx1.T1.15.15.3.m3.1.1" xref="Sx1.T1.15.15.3.m3.1.1.cmml"><mtext id="Sx1.T1.15.15.3.m3.1.1.2" xref="Sx1.T1.15.15.3.m3.1.1.2a.cmml">UQI</mtext><mrow id="Sx1.T1.15.15.3.m3.1.1.3" xref="Sx1.T1.15.15.3.m3.1.1.3.cmml"><mi id="Sx1.T1.15.15.3.m3.1.1.3.2" xref="Sx1.T1.15.15.3.m3.1.1.3.2.cmml">c</mi><mo id="Sx1.T1.15.15.3.m3.1.1.3.1" xref="Sx1.T1.15.15.3.m3.1.1.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.15.15.3.m3.1.1.3.3" xref="Sx1.T1.15.15.3.m3.1.1.3.3.cmml">h</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx1.T1.15.15.3.m3.1b"><apply id="Sx1.T1.15.15.3.m3.1.1.cmml" xref="Sx1.T1.15.15.3.m3.1.1"><csymbol cd="ambiguous" id="Sx1.T1.15.15.3.m3.1.1.1.cmml" xref="Sx1.T1.15.15.3.m3.1.1">subscript</csymbol><ci id="Sx1.T1.15.15.3.m3.1.1.2a.cmml" xref="Sx1.T1.15.15.3.m3.1.1.2"><mtext id="Sx1.T1.15.15.3.m3.1.1.2.cmml" xref="Sx1.T1.15.15.3.m3.1.1.2">UQI</mtext></ci><apply id="Sx1.T1.15.15.3.m3.1.1.3.cmml" xref="Sx1.T1.15.15.3.m3.1.1.3"><times id="Sx1.T1.15.15.3.m3.1.1.3.1.cmml" xref="Sx1.T1.15.15.3.m3.1.1.3.1"></times><ci id="Sx1.T1.15.15.3.m3.1.1.3.2.cmml" xref="Sx1.T1.15.15.3.m3.1.1.3.2">ùëê</ci><ci id="Sx1.T1.15.15.3.m3.1.1.3.3.cmml" xref="Sx1.T1.15.15.3.m3.1.1.3.3">‚Ñé</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.15.15.3.m3.1c">\text{UQI}_{ch}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.15.15.3.m3.1d">UQI start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT</annotation></semantics></math>,</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.15.15.7">Lower, median, upper, inter-</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.16.16">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.16.16.2">Intensities</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.16.16.1"><math alttext="\text{IQR}_{ch}\!=\!\text{UQI}_{ch}\!-\!\text{LQI}_{ch}" class="ltx_Math" display="inline" id="Sx1.T1.16.16.1.m1.1"><semantics id="Sx1.T1.16.16.1.m1.1a"><mrow id="Sx1.T1.16.16.1.m1.1.1" xref="Sx1.T1.16.16.1.m1.1.1.cmml"><msub id="Sx1.T1.16.16.1.m1.1.1.2" xref="Sx1.T1.16.16.1.m1.1.1.2.cmml"><mtext id="Sx1.T1.16.16.1.m1.1.1.2.2" xref="Sx1.T1.16.16.1.m1.1.1.2.2a.cmml">IQR</mtext><mrow id="Sx1.T1.16.16.1.m1.1.1.2.3" xref="Sx1.T1.16.16.1.m1.1.1.2.3.cmml"><mi id="Sx1.T1.16.16.1.m1.1.1.2.3.2" xref="Sx1.T1.16.16.1.m1.1.1.2.3.2.cmml">c</mi><mo id="Sx1.T1.16.16.1.m1.1.1.2.3.1" xref="Sx1.T1.16.16.1.m1.1.1.2.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.16.16.1.m1.1.1.2.3.3" xref="Sx1.T1.16.16.1.m1.1.1.2.3.3.cmml">h</mi></mrow></msub><mo id="Sx1.T1.16.16.1.m1.1.1.1" lspace="0.108em" rspace="0.108em" xref="Sx1.T1.16.16.1.m1.1.1.1.cmml">=</mo><mrow id="Sx1.T1.16.16.1.m1.1.1.3" xref="Sx1.T1.16.16.1.m1.1.1.3.cmml"><msub id="Sx1.T1.16.16.1.m1.1.1.3.2" xref="Sx1.T1.16.16.1.m1.1.1.3.2.cmml"><mtext id="Sx1.T1.16.16.1.m1.1.1.3.2.2" xref="Sx1.T1.16.16.1.m1.1.1.3.2.2a.cmml">UQI</mtext><mrow id="Sx1.T1.16.16.1.m1.1.1.3.2.3" xref="Sx1.T1.16.16.1.m1.1.1.3.2.3.cmml"><mi id="Sx1.T1.16.16.1.m1.1.1.3.2.3.2" xref="Sx1.T1.16.16.1.m1.1.1.3.2.3.2.cmml">c</mi><mo id="Sx1.T1.16.16.1.m1.1.1.3.2.3.1" xref="Sx1.T1.16.16.1.m1.1.1.3.2.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.16.16.1.m1.1.1.3.2.3.3" xref="Sx1.T1.16.16.1.m1.1.1.3.2.3.3.cmml">h</mi></mrow></msub><mo id="Sx1.T1.16.16.1.m1.1.1.3.1" rspace="0.052em" xref="Sx1.T1.16.16.1.m1.1.1.3.1.cmml">‚àí</mo><msub id="Sx1.T1.16.16.1.m1.1.1.3.3" xref="Sx1.T1.16.16.1.m1.1.1.3.3.cmml"><mtext id="Sx1.T1.16.16.1.m1.1.1.3.3.2" xref="Sx1.T1.16.16.1.m1.1.1.3.3.2a.cmml">LQI</mtext><mrow id="Sx1.T1.16.16.1.m1.1.1.3.3.3" xref="Sx1.T1.16.16.1.m1.1.1.3.3.3.cmml"><mi id="Sx1.T1.16.16.1.m1.1.1.3.3.3.2" xref="Sx1.T1.16.16.1.m1.1.1.3.3.3.2.cmml">c</mi><mo id="Sx1.T1.16.16.1.m1.1.1.3.3.3.1" xref="Sx1.T1.16.16.1.m1.1.1.3.3.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.16.16.1.m1.1.1.3.3.3.3" xref="Sx1.T1.16.16.1.m1.1.1.3.3.3.3.cmml">h</mi></mrow></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.16.16.1.m1.1b"><apply id="Sx1.T1.16.16.1.m1.1.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1"><eq id="Sx1.T1.16.16.1.m1.1.1.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1.1"></eq><apply id="Sx1.T1.16.16.1.m1.1.1.2.cmml" xref="Sx1.T1.16.16.1.m1.1.1.2"><csymbol cd="ambiguous" id="Sx1.T1.16.16.1.m1.1.1.2.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1.2">subscript</csymbol><ci id="Sx1.T1.16.16.1.m1.1.1.2.2a.cmml" xref="Sx1.T1.16.16.1.m1.1.1.2.2"><mtext id="Sx1.T1.16.16.1.m1.1.1.2.2.cmml" xref="Sx1.T1.16.16.1.m1.1.1.2.2">IQR</mtext></ci><apply id="Sx1.T1.16.16.1.m1.1.1.2.3.cmml" xref="Sx1.T1.16.16.1.m1.1.1.2.3"><times id="Sx1.T1.16.16.1.m1.1.1.2.3.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1.2.3.1"></times><ci id="Sx1.T1.16.16.1.m1.1.1.2.3.2.cmml" xref="Sx1.T1.16.16.1.m1.1.1.2.3.2">ùëê</ci><ci id="Sx1.T1.16.16.1.m1.1.1.2.3.3.cmml" xref="Sx1.T1.16.16.1.m1.1.1.2.3.3">‚Ñé</ci></apply></apply><apply id="Sx1.T1.16.16.1.m1.1.1.3.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3"><minus id="Sx1.T1.16.16.1.m1.1.1.3.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.1"></minus><apply id="Sx1.T1.16.16.1.m1.1.1.3.2.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.2"><csymbol cd="ambiguous" id="Sx1.T1.16.16.1.m1.1.1.3.2.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.2">subscript</csymbol><ci id="Sx1.T1.16.16.1.m1.1.1.3.2.2a.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.2.2"><mtext id="Sx1.T1.16.16.1.m1.1.1.3.2.2.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.2.2">UQI</mtext></ci><apply id="Sx1.T1.16.16.1.m1.1.1.3.2.3.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.2.3"><times id="Sx1.T1.16.16.1.m1.1.1.3.2.3.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.2.3.1"></times><ci id="Sx1.T1.16.16.1.m1.1.1.3.2.3.2.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.2.3.2">ùëê</ci><ci id="Sx1.T1.16.16.1.m1.1.1.3.2.3.3.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.2.3.3">‚Ñé</ci></apply></apply><apply id="Sx1.T1.16.16.1.m1.1.1.3.3.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.3"><csymbol cd="ambiguous" id="Sx1.T1.16.16.1.m1.1.1.3.3.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.3">subscript</csymbol><ci id="Sx1.T1.16.16.1.m1.1.1.3.3.2a.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.3.2"><mtext id="Sx1.T1.16.16.1.m1.1.1.3.3.2.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.3.2">LQI</mtext></ci><apply id="Sx1.T1.16.16.1.m1.1.1.3.3.3.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.3.3"><times id="Sx1.T1.16.16.1.m1.1.1.3.3.3.1.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.3.3.1"></times><ci id="Sx1.T1.16.16.1.m1.1.1.3.3.3.2.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.3.3.2">ùëê</ci><ci id="Sx1.T1.16.16.1.m1.1.1.3.3.3.3.cmml" xref="Sx1.T1.16.16.1.m1.1.1.3.3.3.3">‚Ñé</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.16.16.1.m1.1c">\text{IQR}_{ch}\!=\!\text{UQI}_{ch}\!-\!\text{LQI}_{ch}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.16.16.1.m1.1d">IQR start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT = UQI start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT - LQI start_POSTSUBSCRIPT italic_c italic_h end_POSTSUBSCRIPT</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.16.16.3">quartile range per channel</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.17.17">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.17.17.2" rowspan="2"><span class="ltx_text" id="Sx1.T1.17.17.2.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.17.17.3" rowspan="2"><span class="ltx_text" id="Sx1.T1.17.17.3.1">CK+ ratio</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.17.17.4" rowspan="2"><span class="ltx_text" id="Sx1.T1.17.17.4.1">2</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.17.17.1" rowspan="2"><span class="ltx_text" id="Sx1.T1.17.17.1.1"><math alttext="CK_{+}\!=\!|I_{ck+}(p)_{p\in O}|/A" class="ltx_Math" display="inline" id="Sx1.T1.17.17.1.1.m1.2"><semantics id="Sx1.T1.17.17.1.1.m1.2a"><mrow id="Sx1.T1.17.17.1.1.m1.2.2" xref="Sx1.T1.17.17.1.1.m1.2.2.cmml"><mrow id="Sx1.T1.17.17.1.1.m1.2.2.3" xref="Sx1.T1.17.17.1.1.m1.2.2.3.cmml"><mi id="Sx1.T1.17.17.1.1.m1.2.2.3.2" xref="Sx1.T1.17.17.1.1.m1.2.2.3.2.cmml">C</mi><mo id="Sx1.T1.17.17.1.1.m1.2.2.3.1" xref="Sx1.T1.17.17.1.1.m1.2.2.3.1.cmml">‚Å¢</mo><msub id="Sx1.T1.17.17.1.1.m1.2.2.3.3" xref="Sx1.T1.17.17.1.1.m1.2.2.3.3.cmml"><mi id="Sx1.T1.17.17.1.1.m1.2.2.3.3.2" xref="Sx1.T1.17.17.1.1.m1.2.2.3.3.2.cmml">K</mi><mo id="Sx1.T1.17.17.1.1.m1.2.2.3.3.3" xref="Sx1.T1.17.17.1.1.m1.2.2.3.3.3.cmml">+</mo></msub></mrow><mo id="Sx1.T1.17.17.1.1.m1.2.2.2" rspace="0.108em" xref="Sx1.T1.17.17.1.1.m1.2.2.2.cmml">=</mo><mrow id="Sx1.T1.17.17.1.1.m1.2.2.1" xref="Sx1.T1.17.17.1.1.m1.2.2.1.cmml"><mrow id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.2.cmml"><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.2" stretchy="false" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.2.1.cmml">|</mo><mrow id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.cmml"><msub id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.cmml"><mi id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.2" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.2.cmml">I</mi><mrow id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.cmml"><mrow id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.cmml"><mi id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.2" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.2.cmml">c</mi><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.1" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.1.cmml">‚Å¢</mo><mi id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.3" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.3.cmml">k</mi></mrow><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.3" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.3.cmml">+</mo></mrow></msub><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.1" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.1.cmml">‚Å¢</mo><msub id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.cmml"><mrow id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.2.2" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.cmml"><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.2.2.1" stretchy="false" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.cmml">(</mo><mi id="Sx1.T1.17.17.1.1.m1.1.1" xref="Sx1.T1.17.17.1.1.m1.1.1.cmml">p</mi><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.2.2.2" stretchy="false" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.cmml">)</mo></mrow><mrow id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.cmml"><mi id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.2" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.2.cmml">p</mi><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.1" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.1.cmml">‚àà</mo><mi id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.3" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.3.cmml">O</mi></mrow></msub></mrow><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.3" stretchy="false" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.2.1.cmml">|</mo></mrow><mo id="Sx1.T1.17.17.1.1.m1.2.2.1.2" xref="Sx1.T1.17.17.1.1.m1.2.2.1.2.cmml">/</mo><mi id="Sx1.T1.17.17.1.1.m1.2.2.1.3" xref="Sx1.T1.17.17.1.1.m1.2.2.1.3.cmml">A</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.17.17.1.1.m1.2b"><apply id="Sx1.T1.17.17.1.1.m1.2.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2"><eq id="Sx1.T1.17.17.1.1.m1.2.2.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.2"></eq><apply id="Sx1.T1.17.17.1.1.m1.2.2.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.3"><times id="Sx1.T1.17.17.1.1.m1.2.2.3.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.3.1"></times><ci id="Sx1.T1.17.17.1.1.m1.2.2.3.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.3.2">ùê∂</ci><apply id="Sx1.T1.17.17.1.1.m1.2.2.3.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.3.3"><csymbol cd="ambiguous" id="Sx1.T1.17.17.1.1.m1.2.2.3.3.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.3.3">subscript</csymbol><ci id="Sx1.T1.17.17.1.1.m1.2.2.3.3.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.3.3.2">ùêæ</ci><plus id="Sx1.T1.17.17.1.1.m1.2.2.3.3.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.3.3.3"></plus></apply></apply><apply id="Sx1.T1.17.17.1.1.m1.2.2.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1"><divide id="Sx1.T1.17.17.1.1.m1.2.2.1.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.2"></divide><apply id="Sx1.T1.17.17.1.1.m1.2.2.1.1.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1"><abs id="Sx1.T1.17.17.1.1.m1.2.2.1.1.2.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.2"></abs><apply id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1"><times id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.1"></times><apply id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2"><csymbol cd="ambiguous" id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2">subscript</csymbol><ci id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.2">ùêº</ci><apply id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3"><csymbol cd="latexml" id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3">limit-from</csymbol><apply id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2"><times id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.1"></times><ci id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.2">ùëê</ci><ci id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.2.3">ùëò</ci></apply><plus id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.2.3.3"></plus></apply></apply><apply id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3"><csymbol cd="ambiguous" id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3">subscript</csymbol><ci id="Sx1.T1.17.17.1.1.m1.1.1.cmml" xref="Sx1.T1.17.17.1.1.m1.1.1">ùëù</ci><apply id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3"><in id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.1.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.1"></in><ci id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.2.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.2">ùëù</ci><ci id="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.1.1.1.3.3.3">ùëÇ</ci></apply></apply></apply></apply><ci id="Sx1.T1.17.17.1.1.m1.2.2.1.3.cmml" xref="Sx1.T1.17.17.1.1.m1.2.2.1.3">ùê¥</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.17.17.1.1.m1.2c">CK_{+}\!=\!|I_{ck+}(p)_{p\in O}|/A</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.17.17.1.1.m1.2d">italic_C italic_K start_POSTSUBSCRIPT + end_POSTSUBSCRIPT = | italic_I start_POSTSUBSCRIPT italic_c italic_k + end_POSTSUBSCRIPT ( italic_p ) start_POSTSUBSCRIPT italic_p ‚àà italic_O end_POSTSUBSCRIPT | / italic_A</annotation></semantics></math></span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.17.17.5">Fraction of pixels greater</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.27.4">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.27.4.1">than CK cutoff in object</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.28.5">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.23.28.5.1" rowspan="2"><span class="ltx_text" id="Sx1.T1.23.28.5.1.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.28.5.2">Channel</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.28.5.3" rowspan="2"><span class="ltx_text" id="Sx1.T1.23.28.5.3.1">6</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.28.5.4" rowspan="2"><span class="ltx_text" id="Sx1.T1.23.28.5.4.1">Pearson‚Äôs Correlation</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.28.5.5">Similarity of pixels between</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.29.6">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.29.6.1">Correlation</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.29.6.2">two channels within object</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.30.7">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.23.30.7.1" rowspan="2"><span class="ltx_text" id="Sx1.T1.23.30.7.1.1">Nuc/Cell</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.30.7.2">Channel</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.30.7.3" rowspan="2"><span class="ltx_text" id="Sx1.T1.23.30.7.3.1">4</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.30.7.4">Ranked-Weighted</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.30.7.5">Co-occurrence and corr.</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.31.8">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.31.8.1">Co-localization</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.31.8.2">Co-localization</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.31.8.3">of two channels in object</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.18.18">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.18.18.2">CK</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.18.18.3">Statistics</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.18.18.4">2</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.18.18.1"><math alttext="\mu_{ck},\sigma_{ck}" class="ltx_Math" display="inline" id="Sx1.T1.18.18.1.m1.2"><semantics id="Sx1.T1.18.18.1.m1.2a"><mrow id="Sx1.T1.18.18.1.m1.2.2.2" xref="Sx1.T1.18.18.1.m1.2.2.3.cmml"><msub id="Sx1.T1.18.18.1.m1.1.1.1.1" xref="Sx1.T1.18.18.1.m1.1.1.1.1.cmml"><mi id="Sx1.T1.18.18.1.m1.1.1.1.1.2" xref="Sx1.T1.18.18.1.m1.1.1.1.1.2.cmml">Œº</mi><mrow id="Sx1.T1.18.18.1.m1.1.1.1.1.3" xref="Sx1.T1.18.18.1.m1.1.1.1.1.3.cmml"><mi id="Sx1.T1.18.18.1.m1.1.1.1.1.3.2" xref="Sx1.T1.18.18.1.m1.1.1.1.1.3.2.cmml">c</mi><mo id="Sx1.T1.18.18.1.m1.1.1.1.1.3.1" xref="Sx1.T1.18.18.1.m1.1.1.1.1.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.18.18.1.m1.1.1.1.1.3.3" xref="Sx1.T1.18.18.1.m1.1.1.1.1.3.3.cmml">k</mi></mrow></msub><mo id="Sx1.T1.18.18.1.m1.2.2.2.3" xref="Sx1.T1.18.18.1.m1.2.2.3.cmml">,</mo><msub id="Sx1.T1.18.18.1.m1.2.2.2.2" xref="Sx1.T1.18.18.1.m1.2.2.2.2.cmml"><mi id="Sx1.T1.18.18.1.m1.2.2.2.2.2" xref="Sx1.T1.18.18.1.m1.2.2.2.2.2.cmml">œÉ</mi><mrow id="Sx1.T1.18.18.1.m1.2.2.2.2.3" xref="Sx1.T1.18.18.1.m1.2.2.2.2.3.cmml"><mi id="Sx1.T1.18.18.1.m1.2.2.2.2.3.2" xref="Sx1.T1.18.18.1.m1.2.2.2.2.3.2.cmml">c</mi><mo id="Sx1.T1.18.18.1.m1.2.2.2.2.3.1" xref="Sx1.T1.18.18.1.m1.2.2.2.2.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.18.18.1.m1.2.2.2.2.3.3" xref="Sx1.T1.18.18.1.m1.2.2.2.2.3.3.cmml">k</mi></mrow></msub></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.18.18.1.m1.2b"><list id="Sx1.T1.18.18.1.m1.2.2.3.cmml" xref="Sx1.T1.18.18.1.m1.2.2.2"><apply id="Sx1.T1.18.18.1.m1.1.1.1.1.cmml" xref="Sx1.T1.18.18.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="Sx1.T1.18.18.1.m1.1.1.1.1.1.cmml" xref="Sx1.T1.18.18.1.m1.1.1.1.1">subscript</csymbol><ci id="Sx1.T1.18.18.1.m1.1.1.1.1.2.cmml" xref="Sx1.T1.18.18.1.m1.1.1.1.1.2">ùúá</ci><apply id="Sx1.T1.18.18.1.m1.1.1.1.1.3.cmml" xref="Sx1.T1.18.18.1.m1.1.1.1.1.3"><times id="Sx1.T1.18.18.1.m1.1.1.1.1.3.1.cmml" xref="Sx1.T1.18.18.1.m1.1.1.1.1.3.1"></times><ci id="Sx1.T1.18.18.1.m1.1.1.1.1.3.2.cmml" xref="Sx1.T1.18.18.1.m1.1.1.1.1.3.2">ùëê</ci><ci id="Sx1.T1.18.18.1.m1.1.1.1.1.3.3.cmml" xref="Sx1.T1.18.18.1.m1.1.1.1.1.3.3">ùëò</ci></apply></apply><apply id="Sx1.T1.18.18.1.m1.2.2.2.2.cmml" xref="Sx1.T1.18.18.1.m1.2.2.2.2"><csymbol cd="ambiguous" id="Sx1.T1.18.18.1.m1.2.2.2.2.1.cmml" xref="Sx1.T1.18.18.1.m1.2.2.2.2">subscript</csymbol><ci id="Sx1.T1.18.18.1.m1.2.2.2.2.2.cmml" xref="Sx1.T1.18.18.1.m1.2.2.2.2.2">ùúé</ci><apply id="Sx1.T1.18.18.1.m1.2.2.2.2.3.cmml" xref="Sx1.T1.18.18.1.m1.2.2.2.2.3"><times id="Sx1.T1.18.18.1.m1.2.2.2.2.3.1.cmml" xref="Sx1.T1.18.18.1.m1.2.2.2.2.3.1"></times><ci id="Sx1.T1.18.18.1.m1.2.2.2.2.3.2.cmml" xref="Sx1.T1.18.18.1.m1.2.2.2.2.3.2">ùëê</ci><ci id="Sx1.T1.18.18.1.m1.2.2.2.2.3.3.cmml" xref="Sx1.T1.18.18.1.m1.2.2.2.2.3.3">ùëò</ci></apply></apply></list></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.18.18.1.m1.2c">\mu_{ck},\sigma_{ck}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.18.18.1.m1.2d">italic_Œº start_POSTSUBSCRIPT italic_c italic_k end_POSTSUBSCRIPT , italic_œÉ start_POSTSUBSCRIPT italic_c italic_k end_POSTSUBSCRIPT</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.18.18.5">Mean &amp; Std. Dev. of CK</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.32.9" style="background-color:#B5DEE8;">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_tt" colspan="5" id="Sx1.T1.23.32.9.1"><span class="ltx_text ltx_font_bold" id="Sx1.T1.23.32.9.1.1" style="background-color:#B5DEE8;">Texture Features (70)</span></td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.19.19">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.19.19.2" rowspan="2"><span class="ltx_text" id="Sx1.T1.19.19.2.1">CK</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.19.19.3">2D Gabor</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.19.19.4" rowspan="2"><span class="ltx_text" id="Sx1.T1.19.19.4.1">32</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.19.19.1" rowspan="2"><span class="ltx_text" id="Sx1.T1.19.19.1.1"><math alttext="g(\lambda,\theta,\sigma,\psi,\gamma)" class="ltx_Math" display="inline" id="Sx1.T1.19.19.1.1.m1.5"><semantics id="Sx1.T1.19.19.1.1.m1.5a"><mrow id="Sx1.T1.19.19.1.1.m1.5.6" xref="Sx1.T1.19.19.1.1.m1.5.6.cmml"><mi id="Sx1.T1.19.19.1.1.m1.5.6.2" xref="Sx1.T1.19.19.1.1.m1.5.6.2.cmml">g</mi><mo id="Sx1.T1.19.19.1.1.m1.5.6.1" xref="Sx1.T1.19.19.1.1.m1.5.6.1.cmml">‚Å¢</mo><mrow id="Sx1.T1.19.19.1.1.m1.5.6.3.2" xref="Sx1.T1.19.19.1.1.m1.5.6.3.1.cmml"><mo id="Sx1.T1.19.19.1.1.m1.5.6.3.2.1" stretchy="false" xref="Sx1.T1.19.19.1.1.m1.5.6.3.1.cmml">(</mo><mi id="Sx1.T1.19.19.1.1.m1.1.1" xref="Sx1.T1.19.19.1.1.m1.1.1.cmml">Œª</mi><mo id="Sx1.T1.19.19.1.1.m1.5.6.3.2.2" xref="Sx1.T1.19.19.1.1.m1.5.6.3.1.cmml">,</mo><mi id="Sx1.T1.19.19.1.1.m1.2.2" xref="Sx1.T1.19.19.1.1.m1.2.2.cmml">Œ∏</mi><mo id="Sx1.T1.19.19.1.1.m1.5.6.3.2.3" xref="Sx1.T1.19.19.1.1.m1.5.6.3.1.cmml">,</mo><mi id="Sx1.T1.19.19.1.1.m1.3.3" xref="Sx1.T1.19.19.1.1.m1.3.3.cmml">œÉ</mi><mo id="Sx1.T1.19.19.1.1.m1.5.6.3.2.4" xref="Sx1.T1.19.19.1.1.m1.5.6.3.1.cmml">,</mo><mi id="Sx1.T1.19.19.1.1.m1.4.4" xref="Sx1.T1.19.19.1.1.m1.4.4.cmml">œà</mi><mo id="Sx1.T1.19.19.1.1.m1.5.6.3.2.5" xref="Sx1.T1.19.19.1.1.m1.5.6.3.1.cmml">,</mo><mi id="Sx1.T1.19.19.1.1.m1.5.5" xref="Sx1.T1.19.19.1.1.m1.5.5.cmml">Œ≥</mi><mo id="Sx1.T1.19.19.1.1.m1.5.6.3.2.6" stretchy="false" xref="Sx1.T1.19.19.1.1.m1.5.6.3.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.19.19.1.1.m1.5b"><apply id="Sx1.T1.19.19.1.1.m1.5.6.cmml" xref="Sx1.T1.19.19.1.1.m1.5.6"><times id="Sx1.T1.19.19.1.1.m1.5.6.1.cmml" xref="Sx1.T1.19.19.1.1.m1.5.6.1"></times><ci id="Sx1.T1.19.19.1.1.m1.5.6.2.cmml" xref="Sx1.T1.19.19.1.1.m1.5.6.2">ùëî</ci><vector id="Sx1.T1.19.19.1.1.m1.5.6.3.1.cmml" xref="Sx1.T1.19.19.1.1.m1.5.6.3.2"><ci id="Sx1.T1.19.19.1.1.m1.1.1.cmml" xref="Sx1.T1.19.19.1.1.m1.1.1">ùúÜ</ci><ci id="Sx1.T1.19.19.1.1.m1.2.2.cmml" xref="Sx1.T1.19.19.1.1.m1.2.2">ùúÉ</ci><ci id="Sx1.T1.19.19.1.1.m1.3.3.cmml" xref="Sx1.T1.19.19.1.1.m1.3.3">ùúé</ci><ci id="Sx1.T1.19.19.1.1.m1.4.4.cmml" xref="Sx1.T1.19.19.1.1.m1.4.4">ùúì</ci><ci id="Sx1.T1.19.19.1.1.m1.5.5.cmml" xref="Sx1.T1.19.19.1.1.m1.5.5">ùõæ</ci></vector></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.19.19.1.1.m1.5c">g(\lambda,\theta,\sigma,\psi,\gamma)</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.19.19.1.1.m1.5d">italic_g ( italic_Œª , italic_Œ∏ , italic_œÉ , italic_œà , italic_Œ≥ )</annotation></semantics></math></span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.19.19.5">Frequency patterns in</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.33.10">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.33.10.1">Filters</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.33.10.2">various orientations</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.34.11">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.23.34.11.1" rowspan="2"><span class="ltx_text" id="Sx1.T1.23.34.11.1.1">CK</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.34.11.2">2D Laws</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.34.11.3" rowspan="2"><span class="ltx_text" id="Sx1.T1.23.34.11.3.1">32</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.34.11.4">Pairs of Level, Edge,</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.23.34.11.5">Spatial patterns in</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.35.12">
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.35.12.1">Filters</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.35.12.2">Spot, and Ripple filters</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.23.35.12.3">various orientations</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.20.20">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" id="Sx1.T1.20.20.2">DAPI,</td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.20.20.3">LBP</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t" id="Sx1.T1.20.20.4" rowspan="3"><span class="ltx_text" id="Sx1.T1.20.20.4.1">6</span></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.20.20.1"><math alttext="LBP(p)\!=\!\sum_{i}s(n\textsubscript{i}-p)2^{i}" class="ltx_Math" display="inline" id="Sx1.T1.20.20.1.m1.2"><semantics id="Sx1.T1.20.20.1.m1.2a"><mrow id="Sx1.T1.20.20.1.m1.2.2" xref="Sx1.T1.20.20.1.m1.2.2.cmml"><mrow id="Sx1.T1.20.20.1.m1.2.2.3" xref="Sx1.T1.20.20.1.m1.2.2.3.cmml"><mi id="Sx1.T1.20.20.1.m1.2.2.3.2" xref="Sx1.T1.20.20.1.m1.2.2.3.2.cmml">L</mi><mo id="Sx1.T1.20.20.1.m1.2.2.3.1" xref="Sx1.T1.20.20.1.m1.2.2.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.20.20.1.m1.2.2.3.3" xref="Sx1.T1.20.20.1.m1.2.2.3.3.cmml">B</mi><mo id="Sx1.T1.20.20.1.m1.2.2.3.1a" xref="Sx1.T1.20.20.1.m1.2.2.3.1.cmml">‚Å¢</mo><mi id="Sx1.T1.20.20.1.m1.2.2.3.4" xref="Sx1.T1.20.20.1.m1.2.2.3.4.cmml">P</mi><mo id="Sx1.T1.20.20.1.m1.2.2.3.1b" xref="Sx1.T1.20.20.1.m1.2.2.3.1.cmml">‚Å¢</mo><mrow id="Sx1.T1.20.20.1.m1.2.2.3.5.2" xref="Sx1.T1.20.20.1.m1.2.2.3.cmml"><mo id="Sx1.T1.20.20.1.m1.2.2.3.5.2.1" stretchy="false" xref="Sx1.T1.20.20.1.m1.2.2.3.cmml">(</mo><mi id="Sx1.T1.20.20.1.m1.1.1" xref="Sx1.T1.20.20.1.m1.1.1.cmml">p</mi><mo id="Sx1.T1.20.20.1.m1.2.2.3.5.2.2" stretchy="false" xref="Sx1.T1.20.20.1.m1.2.2.3.cmml">)</mo></mrow></mrow><mo id="Sx1.T1.20.20.1.m1.2.2.2" lspace="0.108em" rspace="0.0539em" xref="Sx1.T1.20.20.1.m1.2.2.2.cmml">=</mo><mrow id="Sx1.T1.20.20.1.m1.2.2.1" xref="Sx1.T1.20.20.1.m1.2.2.1.cmml"><msub id="Sx1.T1.20.20.1.m1.2.2.1.2" xref="Sx1.T1.20.20.1.m1.2.2.1.2.cmml"><mo id="Sx1.T1.20.20.1.m1.2.2.1.2.2" lspace="0.0539em" xref="Sx1.T1.20.20.1.m1.2.2.1.2.2.cmml">‚àë</mo><mi id="Sx1.T1.20.20.1.m1.2.2.1.2.3" xref="Sx1.T1.20.20.1.m1.2.2.1.2.3.cmml">i</mi></msub><mrow id="Sx1.T1.20.20.1.m1.2.2.1.1" xref="Sx1.T1.20.20.1.m1.2.2.1.1.cmml"><mi id="Sx1.T1.20.20.1.m1.2.2.1.1.3" xref="Sx1.T1.20.20.1.m1.2.2.1.1.3.cmml">s</mi><mo id="Sx1.T1.20.20.1.m1.2.2.1.1.2" xref="Sx1.T1.20.20.1.m1.2.2.1.1.2.cmml">‚Å¢</mo><mrow id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.cmml"><mo id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.2" stretchy="false" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.cmml">(</mo><mrow id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.cmml"><mrow id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.cmml"><mi id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.2" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.2.cmml">n</mi><mo id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.1" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.1.cmml">‚Å¢</mo><mtext id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.3" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.3b.cmml"><sub class="ltx_sub" id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.3.1nest">i</sub></mtext></mrow><mo id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.1" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.1.cmml">‚àí</mo><mi id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.3" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.3.cmml">p</mi></mrow><mo id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.3" stretchy="false" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.cmml">)</mo></mrow><mo id="Sx1.T1.20.20.1.m1.2.2.1.1.2a" xref="Sx1.T1.20.20.1.m1.2.2.1.1.2.cmml">‚Å¢</mo><msup id="Sx1.T1.20.20.1.m1.2.2.1.1.4" xref="Sx1.T1.20.20.1.m1.2.2.1.1.4.cmml"><mn id="Sx1.T1.20.20.1.m1.2.2.1.1.4.2" xref="Sx1.T1.20.20.1.m1.2.2.1.1.4.2.cmml">2</mn><mi id="Sx1.T1.20.20.1.m1.2.2.1.1.4.3" xref="Sx1.T1.20.20.1.m1.2.2.1.1.4.3.cmml">i</mi></msup></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.20.20.1.m1.2b"><apply id="Sx1.T1.20.20.1.m1.2.2.cmml" xref="Sx1.T1.20.20.1.m1.2.2"><eq id="Sx1.T1.20.20.1.m1.2.2.2.cmml" xref="Sx1.T1.20.20.1.m1.2.2.2"></eq><apply id="Sx1.T1.20.20.1.m1.2.2.3.cmml" xref="Sx1.T1.20.20.1.m1.2.2.3"><times id="Sx1.T1.20.20.1.m1.2.2.3.1.cmml" xref="Sx1.T1.20.20.1.m1.2.2.3.1"></times><ci id="Sx1.T1.20.20.1.m1.2.2.3.2.cmml" xref="Sx1.T1.20.20.1.m1.2.2.3.2">ùêø</ci><ci id="Sx1.T1.20.20.1.m1.2.2.3.3.cmml" xref="Sx1.T1.20.20.1.m1.2.2.3.3">ùêµ</ci><ci id="Sx1.T1.20.20.1.m1.2.2.3.4.cmml" xref="Sx1.T1.20.20.1.m1.2.2.3.4">ùëÉ</ci><ci id="Sx1.T1.20.20.1.m1.1.1.cmml" xref="Sx1.T1.20.20.1.m1.1.1">ùëù</ci></apply><apply id="Sx1.T1.20.20.1.m1.2.2.1.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1"><apply id="Sx1.T1.20.20.1.m1.2.2.1.2.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.2"><csymbol cd="ambiguous" id="Sx1.T1.20.20.1.m1.2.2.1.2.1.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.2">subscript</csymbol><sum id="Sx1.T1.20.20.1.m1.2.2.1.2.2.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.2.2"></sum><ci id="Sx1.T1.20.20.1.m1.2.2.1.2.3.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.2.3">ùëñ</ci></apply><apply id="Sx1.T1.20.20.1.m1.2.2.1.1.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1"><times id="Sx1.T1.20.20.1.m1.2.2.1.1.2.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.2"></times><ci id="Sx1.T1.20.20.1.m1.2.2.1.1.3.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.3">ùë†</ci><apply id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1"><minus id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.1.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.1"></minus><apply id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2"><times id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.1.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.1"></times><ci id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.2.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.2">ùëõ</ci><ci id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.3b.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.3"><mtext id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.3.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.3"><sub class="ltx_sub" id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.2.3.1anest">i</sub></mtext></ci></apply><ci id="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.3.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.1.1.1.3">ùëù</ci></apply><apply id="Sx1.T1.20.20.1.m1.2.2.1.1.4.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.4"><csymbol cd="ambiguous" id="Sx1.T1.20.20.1.m1.2.2.1.1.4.1.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.4">superscript</csymbol><cn id="Sx1.T1.20.20.1.m1.2.2.1.1.4.2.cmml" type="integer" xref="Sx1.T1.20.20.1.m1.2.2.1.1.4.2">2</cn><ci id="Sx1.T1.20.20.1.m1.2.2.1.1.4.3.cmml" xref="Sx1.T1.20.20.1.m1.2.2.1.1.4.3">ùëñ</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.20.20.1.m1.2c">LBP(p)\!=\!\sum_{i}s(n\textsubscript{i}-p)2^{i}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.20.20.1.m1.2d">italic_L italic_B italic_P ( italic_p ) = ‚àë start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT italic_s ( italic_n - italic_p ) 2 start_POSTSUPERSCRIPT italic_i end_POSTSUPERSCRIPT</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_r ltx_border_t" id="Sx1.T1.20.20.5">Local geometric pattern</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.21.21">
<td class="ltx_td ltx_align_center ltx_border_l ltx_border_r" id="Sx1.T1.21.21.2">CK,</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.21.21.3">Correlation</td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.21.21.1"><math alttext="s(x)\!=\!1\text{ if }x\!&gt;\!1,\text{ else }0" class="ltx_Math" display="inline" id="Sx1.T1.21.21.1.m1.3"><semantics id="Sx1.T1.21.21.1.m1.3a"><mrow id="Sx1.T1.21.21.1.m1.3.3.2" xref="Sx1.T1.21.21.1.m1.3.3.3.cmml"><mrow id="Sx1.T1.21.21.1.m1.2.2.1.1" xref="Sx1.T1.21.21.1.m1.2.2.1.1.cmml"><mrow id="Sx1.T1.21.21.1.m1.2.2.1.1.2" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2.cmml"><mi id="Sx1.T1.21.21.1.m1.2.2.1.1.2.2" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2.2.cmml">s</mi><mo id="Sx1.T1.21.21.1.m1.2.2.1.1.2.1" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2.1.cmml">‚Å¢</mo><mrow id="Sx1.T1.21.21.1.m1.2.2.1.1.2.3.2" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2.cmml"><mo id="Sx1.T1.21.21.1.m1.2.2.1.1.2.3.2.1" stretchy="false" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2.cmml">(</mo><mi id="Sx1.T1.21.21.1.m1.1.1" xref="Sx1.T1.21.21.1.m1.1.1.cmml">x</mi><mo id="Sx1.T1.21.21.1.m1.2.2.1.1.2.3.2.2" stretchy="false" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2.cmml">)</mo></mrow></mrow><mo id="Sx1.T1.21.21.1.m1.2.2.1.1.3" lspace="0.108em" rspace="0.108em" xref="Sx1.T1.21.21.1.m1.2.2.1.1.3.cmml">=</mo><mrow id="Sx1.T1.21.21.1.m1.2.2.1.1.4" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.cmml"><mn id="Sx1.T1.21.21.1.m1.2.2.1.1.4.2" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.2.cmml">1</mn><mo id="Sx1.T1.21.21.1.m1.2.2.1.1.4.1" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.1.cmml">‚Å¢</mo><mtext id="Sx1.T1.21.21.1.m1.2.2.1.1.4.3" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.3a.cmml">¬†if¬†</mtext><mo id="Sx1.T1.21.21.1.m1.2.2.1.1.4.1a" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.1.cmml">‚Å¢</mo><mi id="Sx1.T1.21.21.1.m1.2.2.1.1.4.4" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.4.cmml">x</mi></mrow><mo id="Sx1.T1.21.21.1.m1.2.2.1.1.5" lspace="0.108em" rspace="0.108em" xref="Sx1.T1.21.21.1.m1.2.2.1.1.5.cmml">&gt;</mo><mn id="Sx1.T1.21.21.1.m1.2.2.1.1.6" xref="Sx1.T1.21.21.1.m1.2.2.1.1.6.cmml">1</mn></mrow><mo id="Sx1.T1.21.21.1.m1.3.3.2.3" xref="Sx1.T1.21.21.1.m1.3.3.3a.cmml">,</mo><mrow id="Sx1.T1.21.21.1.m1.3.3.2.2" xref="Sx1.T1.21.21.1.m1.3.3.2.2.cmml"><mtext id="Sx1.T1.21.21.1.m1.3.3.2.2.2" xref="Sx1.T1.21.21.1.m1.3.3.2.2.2a.cmml">¬†else¬†</mtext><mo id="Sx1.T1.21.21.1.m1.3.3.2.2.1" xref="Sx1.T1.21.21.1.m1.3.3.2.2.1.cmml">‚Å¢</mo><mn id="Sx1.T1.21.21.1.m1.3.3.2.2.3" xref="Sx1.T1.21.21.1.m1.3.3.2.2.3.cmml">0</mn></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx1.T1.21.21.1.m1.3b"><apply id="Sx1.T1.21.21.1.m1.3.3.3.cmml" xref="Sx1.T1.21.21.1.m1.3.3.2"><csymbol cd="ambiguous" id="Sx1.T1.21.21.1.m1.3.3.3a.cmml" xref="Sx1.T1.21.21.1.m1.3.3.2.3">formulae-sequence</csymbol><apply id="Sx1.T1.21.21.1.m1.2.2.1.1.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1"><and id="Sx1.T1.21.21.1.m1.2.2.1.1a.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1"></and><apply id="Sx1.T1.21.21.1.m1.2.2.1.1b.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1"><eq id="Sx1.T1.21.21.1.m1.2.2.1.1.3.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.3"></eq><apply id="Sx1.T1.21.21.1.m1.2.2.1.1.2.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2"><times id="Sx1.T1.21.21.1.m1.2.2.1.1.2.1.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2.1"></times><ci id="Sx1.T1.21.21.1.m1.2.2.1.1.2.2.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.2.2">ùë†</ci><ci id="Sx1.T1.21.21.1.m1.1.1.cmml" xref="Sx1.T1.21.21.1.m1.1.1">ùë•</ci></apply><apply id="Sx1.T1.21.21.1.m1.2.2.1.1.4.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4"><times id="Sx1.T1.21.21.1.m1.2.2.1.1.4.1.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.1"></times><cn id="Sx1.T1.21.21.1.m1.2.2.1.1.4.2.cmml" type="integer" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.2">1</cn><ci id="Sx1.T1.21.21.1.m1.2.2.1.1.4.3a.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.3"><mtext id="Sx1.T1.21.21.1.m1.2.2.1.1.4.3.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.3">¬†if¬†</mtext></ci><ci id="Sx1.T1.21.21.1.m1.2.2.1.1.4.4.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.4.4">ùë•</ci></apply></apply><apply id="Sx1.T1.21.21.1.m1.2.2.1.1c.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1"><gt id="Sx1.T1.21.21.1.m1.2.2.1.1.5.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1.5"></gt><share href="https://arxiv.org/html/2410.02988v1#Sx1.T1.21.21.1.m1.2.2.1.1.4.cmml" id="Sx1.T1.21.21.1.m1.2.2.1.1d.cmml" xref="Sx1.T1.21.21.1.m1.2.2.1.1"></share><cn id="Sx1.T1.21.21.1.m1.2.2.1.1.6.cmml" type="integer" xref="Sx1.T1.21.21.1.m1.2.2.1.1.6">1</cn></apply></apply><apply id="Sx1.T1.21.21.1.m1.3.3.2.2.cmml" xref="Sx1.T1.21.21.1.m1.3.3.2.2"><times id="Sx1.T1.21.21.1.m1.3.3.2.2.1.cmml" xref="Sx1.T1.21.21.1.m1.3.3.2.2.1"></times><ci id="Sx1.T1.21.21.1.m1.3.3.2.2.2a.cmml" xref="Sx1.T1.21.21.1.m1.3.3.2.2.2"><mtext id="Sx1.T1.21.21.1.m1.3.3.2.2.2.cmml" xref="Sx1.T1.21.21.1.m1.3.3.2.2.2">¬†else¬†</mtext></ci><cn id="Sx1.T1.21.21.1.m1.3.3.2.2.3.cmml" type="integer" xref="Sx1.T1.21.21.1.m1.3.3.2.2.3">0</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.21.21.1.m1.3c">s(x)\!=\!1\text{ if }x\!&gt;\!1,\text{ else }0</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.21.21.1.m1.3d">italic_s ( italic_x ) = 1 if italic_x &gt; 1 , else 0</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_r" id="Sx1.T1.21.21.4">encodings and similarity</td>
</tr>
<tr class="ltx_tr" id="Sx1.T1.23.23">
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_l ltx_border_r" id="Sx1.T1.23.23.3">CD45/31</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r" id="Sx1.T1.23.23.4">&amp; Mutal Info</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r" id="Sx1.T1.23.23.2">pixel <math alttext="n_{i}" class="ltx_Math" display="inline" id="Sx1.T1.22.22.1.m1.1"><semantics id="Sx1.T1.22.22.1.m1.1a"><msub id="Sx1.T1.22.22.1.m1.1.1" xref="Sx1.T1.22.22.1.m1.1.1.cmml"><mi id="Sx1.T1.22.22.1.m1.1.1.2" xref="Sx1.T1.22.22.1.m1.1.1.2.cmml">n</mi><mi id="Sx1.T1.22.22.1.m1.1.1.3" xref="Sx1.T1.22.22.1.m1.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="Sx1.T1.22.22.1.m1.1b"><apply id="Sx1.T1.22.22.1.m1.1.1.cmml" xref="Sx1.T1.22.22.1.m1.1.1"><csymbol cd="ambiguous" id="Sx1.T1.22.22.1.m1.1.1.1.cmml" xref="Sx1.T1.22.22.1.m1.1.1">subscript</csymbol><ci id="Sx1.T1.22.22.1.m1.1.1.2.cmml" xref="Sx1.T1.22.22.1.m1.1.1.2">ùëõ</ci><ci id="Sx1.T1.22.22.1.m1.1.1.3.cmml" xref="Sx1.T1.22.22.1.m1.1.1.3">ùëñ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.22.22.1.m1.1c">n_{i}</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.22.22.1.m1.1d">italic_n start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT</annotation></semantics></math> around <math alttext="p" class="ltx_Math" display="inline" id="Sx1.T1.23.23.2.m2.1"><semantics id="Sx1.T1.23.23.2.m2.1a"><mi id="Sx1.T1.23.23.2.m2.1.1" xref="Sx1.T1.23.23.2.m2.1.1.cmml">p</mi><annotation-xml encoding="MathML-Content" id="Sx1.T1.23.23.2.m2.1b"><ci id="Sx1.T1.23.23.2.m2.1.1.cmml" xref="Sx1.T1.23.23.2.m2.1.1">ùëù</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx1.T1.23.23.2.m2.1c">p</annotation><annotation encoding="application/x-llamapun" id="Sx1.T1.23.23.2.m2.1d">italic_p</annotation></semantics></math>
</td>
<td class="ltx_td ltx_align_center ltx_border_b ltx_border_r" id="Sx1.T1.23.23.5">between channel LBPs</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table"><span class="ltx_text" id="Sx1.T1.25.1.1" style="font-size:90%;">Table 1</span>: </span><span class="ltx_text" id="Sx1.T1.26.2" style="font-size:90%;">Summary of 122 Morphology, Intensity and Texture features extracted from 3-channel IF images (DAPI, CK, CD45/31) within objects of nucleus (Nuc.), cell or full image thumbnail.</span></figcaption>
</figure>
</section>
</article>
</div>
<footer class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Tue Oct  1 22:12:04 2024 by <a class="ltx_LaTeXML_logo" href="http://dlmf.nist.gov/LaTeXML/"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img alt="Mascot Sammy" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg=="/></a>
</div></footer>
</div>
</body>
</html>
