<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[1309.7402] Untitled Document</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Untitled Document">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="Untitled Document">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/1309.7402">

<!--Generated on Fri Mar 15 12:53:41 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document">
<section id="Sx1" class="ltx_section">
<h2 class="ltx_title ltx_title_section"></h2>

</section>
<section id="Sx2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Joint assembly and genetic mapping of the Atlantic horseshoe crab genome reveals ancient whole genome duplication.</h2>

<div id="Sx2.p1" class="ltx_para">
<p id="Sx2.p1.1" class="ltx_p">Carlos Nossa<sup id="Sx2.p1.1.1" class="ltx_sup">1</sup>,
Paul Havlak<sup id="Sx2.p1.1.2" class="ltx_sup">1</sup>,
Jia-Xing Yue<sup id="Sx2.p1.1.3" class="ltx_sup">1</sup>,
Jie Lv<sup id="Sx2.p1.1.4" class="ltx_sup">1</sup>,
Kim Vincent<sup id="Sx2.p1.1.5" class="ltx_sup">1</sup>,
H Jane Brockmann<sup id="Sx2.p1.1.6" class="ltx_sup">3</sup>,
Nicholas H Putnam<sup id="Sx2.p1.1.7" class="ltx_sup">1,2</sup></p>
</div>
<div id="Sx2.p2" class="ltx_para">
<p id="Sx2.p2.1" class="ltx_p"><span id="Sx2.p2.1.1" class="ltx_text" style="font-size:90%;">(1) Department of Ecology and Evolutionary Biology, and (2) Department of
Biochemistry and Cell Biology, Rice University, P.O. Box 1892, Houston
TX 77251-1892; (3)
Department of Biology, P.O.B. 11-8525, University of Florida,
Gainesville, FL 32611-8525.</span></p>
</div>
<div id="Sx2.p3" class="ltx_para">
<p id="Sx2.p3.1" class="ltx_p"><span id="Sx2.p3.1.1" class="ltx_text" style="font-size:90%;">Address for correspondence:</span></p>
</div>
<div id="Sx2.p4" class="ltx_para">
<p id="Sx2.p4.1" class="ltx_p"><span id="Sx2.p4.1.1" class="ltx_text" style="font-size:90%;">nputnam@rice.edu</span></p>
</div>
<div id="Sx2.p5" class="ltx_para">
<p id="Sx2.p5.1" class="ltx_p"><span id="Sx2.p5.1.1" class="ltx_text" style="font-size:90%;">Nicholas Putnam</span></p>
</div>
<div id="Sx2.p6" class="ltx_para">
<p id="Sx2.p6.1" class="ltx_p"><span id="Sx2.p6.1.1" class="ltx_text" style="font-size:90%;">Ecology and Evolutionary Biology – MS 170</span></p>
</div>
<div id="Sx2.p7" class="ltx_para">
<p id="Sx2.p7.1" class="ltx_p"><span id="Sx2.p7.1.1" class="ltx_text" style="font-size:90%;">Rice University</span></p>
</div>
<div id="Sx2.p8" class="ltx_para">
<p id="Sx2.p8.1" class="ltx_p"><span id="Sx2.p8.1.1" class="ltx_text" style="font-size:90%;">P.O. Box 1892</span></p>
</div>
<div id="Sx2.p9" class="ltx_para">
<p id="Sx2.p9.1" class="ltx_p"><span id="Sx2.p9.1.1" class="ltx_text" style="font-size:90%;">Houston, TX 77251-1892</span></p>
</div>
<div id="Sx2.p10" class="ltx_para">
<p id="Sx2.p10.1" class="ltx_p">Keyword: Genotyping-by-Sequencing (GBS)</p>
</div>
<div id="Sx2.p11" class="ltx_para">
<p id="Sx2.p11.1" class="ltx_p">Keyword: Genetic linkage mapping</p>
</div>
<div id="Sx2.p12" class="ltx_para">
<p id="Sx2.p12.1" class="ltx_p">Keyword: Genome evolution</p>
</div>
<div id="Sx2.p13" class="ltx_para">
<p id="Sx2.p13.1" class="ltx_p">Keywork: Limulus polyphemus</p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
</section>
<section id="Sx3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Abstract</h2>

<div id="Sx3.p1" class="ltx_para">
<p id="Sx3.p1.1" class="ltx_p">Horseshoe crabs are marine arthropods with a fossil record extending
back approximately 450 million years.
They exhibit remarkable morphological stability over their long
evolutionary history, retaining a number of ancestral arthropod
traits, and are often cited as
examples of “living fossils.”
As arthropods, they belong to
the <em id="Sx3.p1.1.1" class="ltx_emph ltx_font_italic">Ecdysozoa</em>, an ancient super-phylum whose sequenced genomes
(including insects and nematodes) have thus far shown more divergence
from the ancestral pattern of eumetazoan genome organization than
cnidarians, deuterostomes, and lophotrochozoans. However,
much of ecdysozoan diversity remains unrepresented in comparative
genomic analyses.
Here we use a new strategy of combined <em id="Sx3.p1.1.2" class="ltx_emph ltx_font_italic">de novo</em> assembly and
genetic mapping to examine the chromosome-scale genome organization
of the Atlantic
horseshoe crab <em id="Sx3.p1.1.3" class="ltx_emph ltx_font_italic">Limulus polyphemus</em>. We constructed a genetic
linkage map of this 2.7 Gbp genome by sequencing the nuclear DNA of 34
wild-collected, full-sibling embryos and their parents at a mean
redundancy of 1.1x per sample. The map includes 84,307 sequence
markers and 5,775 candidate conserved protein coding
genes. Comparison to other metazoan genomes shows that
the <em id="Sx3.p1.1.4" class="ltx_emph ltx_font_italic">L. polyphemus</em> genome preserves ancestral bilaterian linkage
groups, and that
a common ancestor of modern horseshoe crabs underwent one or more ancient
whole genome duplications (WGDs) ~300 MYA, followed by extensive
chromosome fusion.</p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
</section>
<section id="Sx4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Introduction</h2>

<div id="Sx4.p1" class="ltx_para">
<p id="Sx4.p1.1" class="ltx_p">Comparative analysis of genome sequences from diverse metazoans has
revealed much about their evolution over hundreds of millions of
years. The discovery of extensive gene homology across large
evolutionary distances has allowed researchers to track chromsome
rearrangements and whole genome duplications. The resulting value
of whole chromosome sequences presents a challenge for existing
whole genome shotgun (WGS) assembly strategies.</p>
</div>
<div id="Sx4.p2" class="ltx_para">
<p id="Sx4.p2.1" class="ltx_p">Whole genome duplication events were long
suspected<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib1" title="" class="ltx_ref">1</a>]</cite>, but only the availability of
genome sequences has allowed confirmation of them in fungal,
vertebrate, plant, and ciliate
lineages<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib2" title="" class="ltx_ref">2</a>, <a href="#bib.bib3" title="" class="ltx_ref">3</a>, <a href="#bib.bib4" title="" class="ltx_ref">4</a>, <a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite>.
In contrast, when only a few chordate, insect, and nematode genomes were
available, conservation of gene linkage (i.e. synteny) and gene order
were observed only between closely-related species, and consequently
were not expected to be conserved between phyla. As more metazoan
genomes have been sequenced, it has become clear that long-range
linkage has been conserved over long time scales in many lineages.</p>
</div>
<div id="Sx4.p3" class="ltx_para">
<p id="Sx4.p3.1" class="ltx_p">Sequencing the genomes of representatives of chordate, mollusk, annelid,
cnidarian, placozoan and sponge clades has identified 17 or 18
ancestral linkage groups
(ALGs)<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib6" title="" class="ltx_ref">6</a>, <a href="#bib.bib7" title="" class="ltx_ref">7</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib9" title="" class="ltx_ref">9</a>, <a href="#bib.bib10" title="" class="ltx_ref">10</a>]</cite>.
Each of these ALGs consists of a set of ancestral genes whose
descendants share conserved synteny in
multiple sequenced genomes. These ALGs have been interpreted to
correspond to ancestral metazoan chromosomes, and correlations
between inferred rates of gene movement between ALGs across the
metazoan tree suggest that these ancestral linkage relationships are
conserved through the action of selective constraints on a subset of
genes<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib11" title="" class="ltx_ref">11</a>]</cite>.</p>
</div>
<div id="Sx4.p4" class="ltx_para">
<p id="Sx4.p4.1" class="ltx_p">The relatively small number of genomes from anciently
distinct metazoan lineages and the fragmented nature of draft
genome assemblies still limit both the search for ancient whole genome
duplications and the power of the data to constrain
models of chromosome-scale genome structure evolution.
While WGS sequencing technology and assembly
methods are active areas of research and technological development,
and have improved at a dramatic pace in recent years, high quality <em id="Sx4.p4.1.1" class="ltx_emph ltx_font_italic">de novo</em> assembly of large, complex metazoan genomes remains a difficult
and resource-intensive problem. Without genetic or physical maps, or
reliance on a high-quality reference genome of a closely-related
species, WGS sequencing projects still typically produce assemblies
containing thousands of scaffolds, hundreds of scaffolds incorrectly
joining sequence from different chromosomes, or
both<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib12" title="" class="ltx_ref">12</a>]</cite>.</p>
</div>
<div id="Sx4.p5" class="ltx_para">
<p id="Sx4.p5.1" class="ltx_p">Next-generation sequencing has greatly reduced the cost of
constructing high density genetic maps by eliminating the need to
develop and genotype polymorphic markers
individually<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib13" title="" class="ltx_ref">13</a>]</cite>. This has been achieved
either by focusing sequence coverage within or adjacent to genomic
regions of distinct biochemical character, such as restriction sites with
RAD-seq and related methods<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib14" title="" class="ltx_ref">14</a>, <a href="#bib.bib15" title="" class="ltx_ref">15</a>]</cite>, or by combining
information across regions using a reference genome
sequence<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>, <a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite>.
While RAD-seq is applicable to organisms lacking a reference genome
assembly, it is not directly applicable to comparisons of genome
organization across long evolutionary time spans because such
comparisons rely on the identification of homologous sequence markers
(typically protein-coding genes), which typically have only a small overlap with
the restriction-associated markers.</p>
</div>
<div id="Sx4.p6" class="ltx_para">
<p id="Sx4.p6.1" class="ltx_p">Here we present a genotype-by-sequencing method for constructing a
high-density genetic map using low-coverage, low-cost, whole genome
sequencing data from the offspring of a wild cross. In this joint
assembly and mapping (JAM) approach the traditionally independent and
sequential steps of genome assembly, polymorphic marker
identification, and genetic map construction are combined. Existing
assemblers expect lower densities of sequence polymorphism, deeper
coverage, greater computer memory, or more aggressive quality trimming
that decrease sequence coverage<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib19" title="" class="ltx_ref">19</a>, <a href="#bib.bib20" title="" class="ltx_ref">20</a>]</cite>.
Our current implementation focuses on conservative assembly of short
scaffolds sufficient for map construction, but our results suggest
that further integration of genetic mapping information within whole
genome shotgun assembly methods can be a cost effective
way to produce assemblies of large, complex genomes with
chromosome-scale contiguity.</p>
</div>
<div id="Sx4.p7" class="ltx_para">
<p id="Sx4.p7.1" class="ltx_p">We have applied this approach to produce a genetic
map of the genome of the Atlantic horseshoe crab, <em id="Sx4.p7.1.1" class="ltx_emph ltx_font_italic">Limulus polyphemus</em>.
Horseshoe crabs are marine arthropods with a fossil record extending
back ~450 million years<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib21" title="" class="ltx_ref">21</a>]</cite>.
They exhibit remarkable morphological stability over their long
evolutionary history, retaining a number of ancestral arthropod
traits<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib22" title="" class="ltx_ref">22</a>]</cite>, and are often cited as
examples of “living fossils.”
<em id="Sx4.p7.1.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> has
a large genome about 90% the size of the human genome. It is an
important species from ecological, commercial, and conservation
perspectives<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib23" title="" class="ltx_ref">23</a>]</cite>, that has been used as a
model system for research in behavioral ecology, physiology and
development<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib24" title="" class="ltx_ref">24</a>]</cite>. The map and SNP
markers described here will be a resource for the <em id="Sx4.p7.1.3" class="ltx_emph ltx_font_italic">L. polyphemus</em>
genome project, research in horseshoe crab population biology, and
comparisons of metazoan genome organization. By anchoring protein
coding genes to this map, we are able to extend anaylsis of
ancestral linkage groups and whole genome duplications to the
chelicerate lineage.</p>
</div>
</section>
<section id="Sx5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Results</h2>

<section id="Sx5.SSx1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Assembly and Mapping</h3>

<div id="Sx5.SSx1.p1" class="ltx_para">
<p id="Sx5.SSx1.p1.1" class="ltx_p">The JAM method is designed to produce a combined assembly of
polymorphic sequences, tagged by genomic regions with at most one SNP
per <em id="Sx5.SSx1.p1.1.1" class="ltx_emph ltx_font_italic">k</em>-mer window (Methods). Starting with genomic reads from a
mating pair of adult <em id="Sx5.SSx1.p1.1.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> and 34 offspring (~100 bp reads
on ~300 bp inserts), we tabulated high-quality 23-mers occurring in
two sequencing runs (and thus also in at least two individuals).</p>
</div>
<div id="Sx5.SSx1.p2" class="ltx_para">
<p id="Sx5.SSx1.p2.2" class="ltx_p">Fitting Poisson models for unduplicated sequence to the frequencies of these
filtered 23-mers suggests that 1.1 billion genomic loci are unique at this resolution.
Of these <em id="Sx5.SSx1.p2.2.1" class="ltx_emph ltx_font_italic">k</em>-loci, 63 percent are modeled as homozygous,
27 percent as paired major-minor alleles,
and the remaining 10 percent as tied allele pairs (Figure <a href="#Sx5.F1" title="Figure 1 ‣ Assembly and Mapping ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>).
The corresponding SNPs, if always at least 23 bases apart,
would be 1.6 percent of bases in the <em id="Sx5.SSx1.p2.2.2" class="ltx_emph ltx_font_italic">k</em>-loci.
Dividing the total number of filtered 23-mers by the modeled
homozygous depth of coverage <math id="Sx5.SSx1.p2.1.m1.1" class="ltx_Math" alttext="d=38.9" display="inline"><semantics id="Sx5.SSx1.p2.1.m1.1a"><mrow id="Sx5.SSx1.p2.1.m1.1.1" xref="Sx5.SSx1.p2.1.m1.1.1.cmml"><mi id="Sx5.SSx1.p2.1.m1.1.1.2" xref="Sx5.SSx1.p2.1.m1.1.1.2.cmml">d</mi><mo id="Sx5.SSx1.p2.1.m1.1.1.1" xref="Sx5.SSx1.p2.1.m1.1.1.1.cmml">=</mo><mn id="Sx5.SSx1.p2.1.m1.1.1.3" xref="Sx5.SSx1.p2.1.m1.1.1.3.cmml">38.9</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx5.SSx1.p2.1.m1.1b"><apply id="Sx5.SSx1.p2.1.m1.1.1.cmml" xref="Sx5.SSx1.p2.1.m1.1.1"><eq id="Sx5.SSx1.p2.1.m1.1.1.1.cmml" xref="Sx5.SSx1.p2.1.m1.1.1.1"></eq><ci id="Sx5.SSx1.p2.1.m1.1.1.2.cmml" xref="Sx5.SSx1.p2.1.m1.1.1.2">𝑑</ci><cn type="float" id="Sx5.SSx1.p2.1.m1.1.1.3.cmml" xref="Sx5.SSx1.p2.1.m1.1.1.3">38.9</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx1.p2.1.m1.1c">d=38.9</annotation></semantics></math> yields an estimated genome
size of 2.74 billion bases, consistent with the measured DNA content of 2.8
pg (978 Mb <math id="Sx5.SSx1.p2.2.m2.1" class="ltx_Math" alttext="\simeq" display="inline"><semantics id="Sx5.SSx1.p2.2.m2.1a"><mo id="Sx5.SSx1.p2.2.m2.1.1" xref="Sx5.SSx1.p2.2.m2.1.1.cmml">≃</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx1.p2.2.m2.1b"><csymbol cd="latexml" id="Sx5.SSx1.p2.2.m2.1.1.cmml" xref="Sx5.SSx1.p2.2.m2.1.1">similar-to-or-equals</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx1.p2.2.m2.1c">\simeq</annotation></semantics></math> 1 pg DNA)<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib25" title="" class="ltx_ref">25</a>]</cite>.</p>
</div>
<figure id="Sx5.F1" class="ltx_figure"><img src="/html/1309.7402/assets/x1.png" id="Sx5.F1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="415" height="321" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>Fitting Poisson distributions to Limulus 23mer frequencies (filtered as described below)</figcaption>
</figure>
<div id="Sx5.SSx1.p3" class="ltx_para">
<p id="Sx5.SSx1.p3.1" class="ltx_p">We categorize specific 23-mers by their edit distances to others,
having no neighbors within a single base substitution
(unique tags) or with a single mutually unique neighbor (“SNPmer pair” tags).
A subset of these, including SNPmer pairs for approximately 7.9 million SNPs,
constitute the tags used for contigging and scaffolding.
The SNP-mer pairs account for about 45 percent
of the modeled fraction of alleles,
the others missed from similarity to other sequences
(e.g. due to repeats) or distance from each other
(because of indels or multiple SNPs per 23-mer).</p>
</div>
<div id="Sx5.SSx1.p4" class="ltx_para">
<p id="Sx5.SSx1.p4.1" class="ltx_p">Chaining these 23-mers together (see Methods) produces an initial 6.6
million contigs, 3.9 million of which are linkable by paired reads for
scaffolding. Applying Bambus<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib26" title="" class="ltx_ref">26</a>]</cite> produces 944 thousand scaffolds spanning
1.3 million bases (Table <a href="#Sx5.T1" title="Table 1 ‣ Assembly and Mapping ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>). These scaffolds serve as
markers incorporating multiple <em id="Sx5.SSx1.p4.1.1" class="ltx_emph ltx_font_italic">k</em>-loci, including SNPmer pairs
used to identify haplotypes.</p>
</div>
<figure id="Sx5.T1" class="ltx_table">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 1: </span><em id="Sx5.T1.2.1" class="ltx_emph ltx_font_italic">K</em>-mer contig and scaffold statistics</figcaption>
<table id="Sx5.T1.3" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="Sx5.T1.3.1.1" class="ltx_tr">
<th id="Sx5.T1.3.1.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_column">Assembled</th>
<th id="Sx5.T1.3.1.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_column">count</th>
<th id="Sx5.T1.3.1.1.3" class="ltx_td ltx_align_left ltx_th ltx_th_column">total (bp)</th>
<th id="Sx5.T1.3.1.1.4" class="ltx_td ltx_align_right ltx_th ltx_th_column">avg. span (bp)</th>
<th id="Sx5.T1.3.1.1.5" class="ltx_td ltx_align_right ltx_th ltx_th_column">n50 span (bp)</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="Sx5.T1.3.2.1" class="ltx_tr">
<td id="Sx5.T1.3.2.1.1" class="ltx_td ltx_align_left ltx_border_t">
<em id="Sx5.T1.3.2.1.1.1" class="ltx_emph ltx_font_italic">k</em>-mer contigs</td>
<td id="Sx5.T1.3.2.1.2" class="ltx_td ltx_align_left ltx_border_t">6,614,434</td>
<td id="Sx5.T1.3.2.1.3" class="ltx_td ltx_align_left ltx_border_t">1,240,275,515</td>
<td id="Sx5.T1.3.2.1.4" class="ltx_td ltx_align_right ltx_border_t">188</td>
<td id="Sx5.T1.3.2.1.5" class="ltx_td ltx_align_right ltx_border_t">418</td>
</tr>
<tr id="Sx5.T1.3.3.2" class="ltx_tr">
<td id="Sx5.T1.3.3.2.1" class="ltx_td ltx_align_left">linkable contigs</td>
<td id="Sx5.T1.3.3.2.2" class="ltx_td ltx_align_left">3,925,844</td>
<td id="Sx5.T1.3.3.2.3" class="ltx_td ltx_align_left">1,137,576,911</td>
<td id="Sx5.T1.3.3.2.4" class="ltx_td ltx_align_right">290</td>
<td id="Sx5.T1.3.3.2.5" class="ltx_td ltx_align_right">460</td>
</tr>
<tr id="Sx5.T1.3.4.3" class="ltx_tr">
<td id="Sx5.T1.3.4.3.1" class="ltx_td ltx_align_left">initial scaffolds</td>
<td id="Sx5.T1.3.4.3.2" class="ltx_td ltx_align_left">944,246</td>
<td id="Sx5.T1.3.4.3.3" class="ltx_td ltx_align_left">1,261,263,172</td>
<td id="Sx5.T1.3.4.3.4" class="ltx_td ltx_align_right">1336</td>
<td id="Sx5.T1.3.4.3.5" class="ltx_td ltx_align_right">3047</td>
</tr>
<tr id="Sx5.T1.3.5.4" class="ltx_tr">
<td id="Sx5.T1.3.5.4.1" class="ltx_td ltx_align_left">reference scaffolds</td>
<td id="Sx5.T1.3.5.4.2" class="ltx_td ltx_align_left">944,246</td>
<td id="Sx5.T1.3.5.4.3" class="ltx_td ltx_align_left">1,295,334,515</td>
<td id="Sx5.T1.3.5.4.4" class="ltx_td ltx_align_right">1372</td>
<td id="Sx5.T1.3.5.4.5" class="ltx_td ltx_align_right">2930</td>
</tr>
<tr id="Sx5.T1.3.6.5" class="ltx_tr">
<td id="Sx5.T1.3.6.5.1" class="ltx_td ltx_align_left">reference bases</td>
<td id="Sx5.T1.3.6.5.2" class="ltx_td"></td>
<td id="Sx5.T1.3.6.5.3" class="ltx_td ltx_align_left">1,131,458,744</td>
<td id="Sx5.T1.3.6.5.4" class="ltx_td ltx_align_right">1198</td>
<td id="Sx5.T1.3.6.5.5" class="ltx_td ltx_align_right">2553</td>
</tr>
</tbody>
</table>
</figure>
<div id="Sx5.SSx1.p5" class="ltx_para">
<p id="Sx5.SSx1.p5.1" class="ltx_p">After assembly, the mean density of single nucleotide polymorphisms (SNPs) across the
four haplotypes in assembled regions was estimated based on read
re-alignments to be 7.6 per
thousand bases. We jointly inferred the phases of these SNPs and
segregation pattern (offspring genotypes) in the mapping cross for
each marker in a maximum likelihood framework (Methods). We focused on the
91,320 markers with at least 18 inferred bi-allelic SNPs for
constructing the linkage map. These markers grouped into 1,908
high-confidence map bins (<em id="Sx5.SSx1.p5.1.1" class="ltx_emph ltx_font_italic">i.e.</em> unique segregation patterns, assumed
to correspond to loci in the genome uninterrupted by meiotic
recombination in the cross<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib27" title="" class="ltx_ref">27</a>]</cite>). Map
bins fell into 32 linkage groups, close to the 26
pairs (2N=52) previously found in a cytogenetic analysis of two
chromosome spreads<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib28" title="" class="ltx_ref">28</a>]</cite>. 20 map bins were
removed for having inconsistent positions in the maternal and paternal
maps, and 12 were singletons.</p>
</div>
<div id="Sx5.SSx1.p6" class="ltx_para">
<p id="Sx5.SSx1.p6.1" class="ltx_p">To estimate the frequency of incorrect genotype calls as a function of
the log likelihood difference between the called and alternative
genotype (genotype confidence score), including contributions from
uncertainty in SNP-mer identification, assembly, and sampling noise,
we carried out a simulation of the library pooling and sequencing, <math id="Sx5.SSx1.p6.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx5.SSx1.p6.1.m1.1a"><mi id="Sx5.SSx1.p6.1.m1.1.1" xref="Sx5.SSx1.p6.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx5.SSx1.p6.1.m1.1b"><ci id="Sx5.SSx1.p6.1.m1.1.1.cmml" xref="Sx5.SSx1.p6.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx1.p6.1.m1.1c">k</annotation></semantics></math>-mer assembly and genotype inference protocols, using the sequenced <em id="Sx5.SSx1.p6.1.1" class="ltx_emph ltx_font_italic">Ciona intestinalis</em> genome as a starting point.</p>
</div>
<div id="Sx5.SSx1.p7" class="ltx_para">
<p id="Sx5.SSx1.p7.3" class="ltx_p">In the simulated <em id="Sx5.SSx1.p7.3.1" class="ltx_emph ltx_font_italic">C. intestinalis</em> data set (Methods), a single stretched
exponential distribution provided a good fit to the frequency of genotype calling
errors as a function of the call confidence score for scores up to
6, or down to error frequency of about 1%. The observed error
frequency declined more slowly for higher confidence scores. The
minimum <math id="Sx5.SSx1.p7.1.m1.1" class="ltx_Math" alttext="\chi^{2}" display="inline"><semantics id="Sx5.SSx1.p7.1.m1.1a"><msup id="Sx5.SSx1.p7.1.m1.1.1" xref="Sx5.SSx1.p7.1.m1.1.1.cmml"><mi id="Sx5.SSx1.p7.1.m1.1.1.2" xref="Sx5.SSx1.p7.1.m1.1.1.2.cmml">χ</mi><mn id="Sx5.SSx1.p7.1.m1.1.1.3" xref="Sx5.SSx1.p7.1.m1.1.1.3.cmml">2</mn></msup><annotation-xml encoding="MathML-Content" id="Sx5.SSx1.p7.1.m1.1b"><apply id="Sx5.SSx1.p7.1.m1.1.1.cmml" xref="Sx5.SSx1.p7.1.m1.1.1"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.1.m1.1.1.1.cmml" xref="Sx5.SSx1.p7.1.m1.1.1">superscript</csymbol><ci id="Sx5.SSx1.p7.1.m1.1.1.2.cmml" xref="Sx5.SSx1.p7.1.m1.1.1.2">𝜒</ci><cn type="integer" id="Sx5.SSx1.p7.1.m1.1.1.3.cmml" xref="Sx5.SSx1.p7.1.m1.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx1.p7.1.m1.1c">\chi^{2}</annotation></semantics></math> fit used for estimating the genotyping error rate in the
<em id="Sx5.SSx1.p7.3.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> map bins
was <math id="Sx5.SSx1.p7.2.m2.1" class="ltx_Math" alttext="p_{e}(s)=a_{1}e^{-s^{c_{1}}/b_{1}}+a_{2}e^{-s^{c_{2}}/b_{2}}" display="inline"><semantics id="Sx5.SSx1.p7.2.m2.1a"><mrow id="Sx5.SSx1.p7.2.m2.1.2" xref="Sx5.SSx1.p7.2.m2.1.2.cmml"><mrow id="Sx5.SSx1.p7.2.m2.1.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.2.cmml"><msub id="Sx5.SSx1.p7.2.m2.1.2.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.2.2.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.2.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.2.2.2.cmml">p</mi><mi id="Sx5.SSx1.p7.2.m2.1.2.2.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.2.2.3.cmml">e</mi></msub><mo lspace="0em" rspace="0em" id="Sx5.SSx1.p7.2.m2.1.2.2.1" xref="Sx5.SSx1.p7.2.m2.1.2.2.1.cmml">​</mo><mrow id="Sx5.SSx1.p7.2.m2.1.2.2.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.2.cmml"><mo stretchy="false" id="Sx5.SSx1.p7.2.m2.1.2.2.3.2.1" xref="Sx5.SSx1.p7.2.m2.1.2.2.cmml">(</mo><mi id="Sx5.SSx1.p7.2.m2.1.1" xref="Sx5.SSx1.p7.2.m2.1.1.cmml">s</mi><mo stretchy="false" id="Sx5.SSx1.p7.2.m2.1.2.2.3.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.2.cmml">)</mo></mrow></mrow><mo id="Sx5.SSx1.p7.2.m2.1.2.1" xref="Sx5.SSx1.p7.2.m2.1.2.1.cmml">=</mo><mrow id="Sx5.SSx1.p7.2.m2.1.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.cmml"><mrow id="Sx5.SSx1.p7.2.m2.1.2.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.cmml"><msub id="Sx5.SSx1.p7.2.m2.1.2.3.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.2.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.2.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.2.2.cmml">a</mi><mn id="Sx5.SSx1.p7.2.m2.1.2.3.2.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.2.3.cmml">1</mn></msub><mo lspace="0em" rspace="0em" id="Sx5.SSx1.p7.2.m2.1.2.3.2.1" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.1.cmml">​</mo><msup id="Sx5.SSx1.p7.2.m2.1.2.3.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.2.cmml">e</mi><mrow id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.cmml"><mo id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3a" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.cmml">−</mo><mrow id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.cmml"><msup id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.2.cmml">s</mi><msub id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.2.cmml">c</mi><mn id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.3.cmml">1</mn></msub></msup><mo id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.1" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.1.cmml">/</mo><msub id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.2.cmml">b</mi><mn id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.3.cmml">1</mn></msub></mrow></mrow></msup></mrow><mo id="Sx5.SSx1.p7.2.m2.1.2.3.1" xref="Sx5.SSx1.p7.2.m2.1.2.3.1.cmml">+</mo><mrow id="Sx5.SSx1.p7.2.m2.1.2.3.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.cmml"><msub id="Sx5.SSx1.p7.2.m2.1.2.3.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.2.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.3.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.2.2.cmml">a</mi><mn id="Sx5.SSx1.p7.2.m2.1.2.3.3.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.2.3.cmml">2</mn></msub><mo lspace="0em" rspace="0em" id="Sx5.SSx1.p7.2.m2.1.2.3.3.1" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.1.cmml">​</mo><msup id="Sx5.SSx1.p7.2.m2.1.2.3.3.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.2.cmml">e</mi><mrow id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.cmml"><mo id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3a" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.cmml">−</mo><mrow id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.cmml"><msup id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.2.cmml">s</mi><msub id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.2.cmml">c</mi><mn id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.3.cmml">2</mn></msub></msup><mo id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.1" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.1.cmml">/</mo><msub id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.cmml"><mi id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.2" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.2.cmml">b</mi><mn id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.3" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.3.cmml">2</mn></msub></mrow></mrow></msup></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx5.SSx1.p7.2.m2.1b"><apply id="Sx5.SSx1.p7.2.m2.1.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2"><eq id="Sx5.SSx1.p7.2.m2.1.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.1"></eq><apply id="Sx5.SSx1.p7.2.m2.1.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.2"><times id="Sx5.SSx1.p7.2.m2.1.2.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.2.1"></times><apply id="Sx5.SSx1.p7.2.m2.1.2.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.2.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.2.2">subscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.2.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.2.2.2">𝑝</ci><ci id="Sx5.SSx1.p7.2.m2.1.2.2.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.2.2.3">𝑒</ci></apply><ci id="Sx5.SSx1.p7.2.m2.1.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.1">𝑠</ci></apply><apply id="Sx5.SSx1.p7.2.m2.1.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3"><plus id="Sx5.SSx1.p7.2.m2.1.2.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.1"></plus><apply id="Sx5.SSx1.p7.2.m2.1.2.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2"><times id="Sx5.SSx1.p7.2.m2.1.2.3.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.1"></times><apply id="Sx5.SSx1.p7.2.m2.1.2.3.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.2.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.2">subscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.2.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.2.2">𝑎</ci><cn type="integer" id="Sx5.SSx1.p7.2.m2.1.2.3.2.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.2.3">1</cn></apply><apply id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3">superscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.2">𝑒</ci><apply id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3"><minus id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3"></minus><apply id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2"><divide id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.1"></divide><apply id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2">superscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.2">𝑠</ci><apply id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3">subscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.2">𝑐</ci><cn type="integer" id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.2.3.3">1</cn></apply></apply><apply id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3">subscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.2">𝑏</ci><cn type="integer" id="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.2.3.3.2.3.3">1</cn></apply></apply></apply></apply></apply><apply id="Sx5.SSx1.p7.2.m2.1.2.3.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3"><times id="Sx5.SSx1.p7.2.m2.1.2.3.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.1"></times><apply id="Sx5.SSx1.p7.2.m2.1.2.3.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.3.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.2">subscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.3.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.2.2">𝑎</ci><cn type="integer" id="Sx5.SSx1.p7.2.m2.1.2.3.3.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.2.3">2</cn></apply><apply id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3">superscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.2">𝑒</ci><apply id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3"><minus id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3"></minus><apply id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2"><divide id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.1"></divide><apply id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2">superscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.2">𝑠</ci><apply id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3">subscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.2">𝑐</ci><cn type="integer" id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.2.3.3">2</cn></apply></apply><apply id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.1.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3">subscript</csymbol><ci id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.2.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.2">𝑏</ci><cn type="integer" id="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.3.cmml" xref="Sx5.SSx1.p7.2.m2.1.2.3.3.3.3.2.3.3">2</cn></apply></apply></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx1.p7.2.m2.1c">p_{e}(s)=a_{1}e^{-s^{c_{1}}/b_{1}}+a_{2}e^{-s^{c_{2}}/b_{2}}</annotation></semantics></math>, with
parameter values <math id="Sx5.SSx1.p7.3.m3.2" class="ltx_Math" alttext="a_{1}=0.49,b_{1}=2.08,c_{1}=1.26,a_{2}=5.47,b_{2}=0.17,c_{2}=0.16" display="inline"><semantics id="Sx5.SSx1.p7.3.m3.2a"><mrow id="Sx5.SSx1.p7.3.m3.2.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.3.cmml"><mrow id="Sx5.SSx1.p7.3.m3.1.1.1.1" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.cmml"><msub id="Sx5.SSx1.p7.3.m3.1.1.1.1.2" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.2.cmml"><mi id="Sx5.SSx1.p7.3.m3.1.1.1.1.2.2" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.2.2.cmml">a</mi><mn id="Sx5.SSx1.p7.3.m3.1.1.1.1.2.3" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.2.3.cmml">1</mn></msub><mo id="Sx5.SSx1.p7.3.m3.1.1.1.1.1" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.1.cmml">=</mo><mn id="Sx5.SSx1.p7.3.m3.1.1.1.1.3" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.3.cmml">0.49</mn></mrow><mo id="Sx5.SSx1.p7.3.m3.2.2.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.3a.cmml">,</mo><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.3.cmml"><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.cmml"><msub id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.cmml"><mi id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.2.cmml">b</mi><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.3.cmml">1</mn></msub><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.1.cmml">=</mo><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.3.cmml">2.08</mn></mrow><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.3a.cmml">,</mo><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.3.cmml"><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.cmml"><msub id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.cmml"><mi id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.2.cmml">c</mi><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.3.cmml">1</mn></msub><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.1.cmml">=</mo><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.3.cmml">1.26</mn></mrow><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.3a.cmml">,</mo><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.3.cmml"><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.cmml"><msub id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.cmml"><mi id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.2.cmml">a</mi><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.3.cmml">2</mn></msub><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.1.cmml">=</mo><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.3.cmml">5.47</mn></mrow><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.3a.cmml">,</mo><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.3.cmml"><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.cmml"><msub id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.cmml"><mi id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.2.cmml">b</mi><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.3.cmml">2</mn></msub><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.1.cmml">=</mo><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.3.cmml">0.17</mn></mrow><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.3a.cmml">,</mo><mrow id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.cmml"><msub id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.cmml"><mi id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.2" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.2.cmml">c</mi><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.3.cmml">2</mn></msub><mo id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.1" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.1.cmml">=</mo><mn id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.3" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.3.cmml">0.16</mn></mrow></mrow></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx5.SSx1.p7.3.m3.2b"><apply id="Sx5.SSx1.p7.3.m3.2.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.3a.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.3">formulae-sequence</csymbol><apply id="Sx5.SSx1.p7.3.m3.1.1.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.1.1.1.1"><eq id="Sx5.SSx1.p7.3.m3.1.1.1.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.1"></eq><apply id="Sx5.SSx1.p7.3.m3.1.1.1.1.2.cmml" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.1.1.1.1.2.1.cmml" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.2">subscript</csymbol><ci id="Sx5.SSx1.p7.3.m3.1.1.1.1.2.2.cmml" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.2.2">𝑎</ci><cn type="integer" id="Sx5.SSx1.p7.3.m3.1.1.1.1.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.2.3">1</cn></apply><cn type="float" id="Sx5.SSx1.p7.3.m3.1.1.1.1.3.cmml" xref="Sx5.SSx1.p7.3.m3.1.1.1.1.3">0.49</cn></apply><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.3a.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.3">formulae-sequence</csymbol><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1"><eq id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.1"></eq><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2">subscript</csymbol><ci id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.2">𝑏</ci><cn type="integer" id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.2.3">1</cn></apply><cn type="float" id="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.1.1.3">2.08</cn></apply><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.3a.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.3">formulae-sequence</csymbol><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1"><eq id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.1"></eq><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2">subscript</csymbol><ci id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.2">𝑐</ci><cn type="integer" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.2.3">1</cn></apply><cn type="float" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.1.1.3">1.26</cn></apply><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.3a.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.3">formulae-sequence</csymbol><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1"><eq id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.1"></eq><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2">subscript</csymbol><ci id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.2">𝑎</ci><cn type="integer" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.2.3">2</cn></apply><cn type="float" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.1.1.3">5.47</cn></apply><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.3a.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.3">formulae-sequence</csymbol><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1"><eq id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.1"></eq><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2">subscript</csymbol><ci id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.2">𝑏</ci><cn type="integer" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.2.3">2</cn></apply><cn type="float" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.1.1.3">0.17</cn></apply><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2"><eq id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.1"></eq><apply id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2"><csymbol cd="ambiguous" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.1.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2">subscript</csymbol><ci id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.2.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.2">𝑐</ci><cn type="integer" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.2.3">2</cn></apply><cn type="float" id="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.3.cmml" xref="Sx5.SSx1.p7.3.m3.2.2.2.2.2.2.2.2.2.2.2.2.3">0.16</cn></apply></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx1.p7.3.m3.2c">a_{1}=0.49,b_{1}=2.08,c_{1}=1.26,a_{2}=5.47,b_{2}=0.17,c_{2}=0.16</annotation></semantics></math>
(Figure <a href="#Sx5.F2" title="Figure 2 ‣ Assembly and Mapping ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>).</p>
</div>
<figure id="Sx5.F2" class="ltx_figure"><img src="/html/1309.7402/assets/error-fig.png" id="Sx5.F2.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="538" height="404" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Genotype call error rate as a function of call confidence score for bins of 10,000 calls in simulated <em id="Sx5.F2.8.1" class="ltx_emph ltx_font_italic">Ciona intestinalis</em> genome data. The stippled blue region shows 95% confidence intervals of the Bayesian posterior probability distribution of the underlying error rate computed from the Beta distribution Beta<math id="Sx5.F2.4.m1.2" class="ltx_Math" alttext="(n_{e}+1,n_{c}-n_{e}+1)" display="inline"><semantics id="Sx5.F2.4.m1.2b"><mrow id="Sx5.F2.4.m1.2.2.2" xref="Sx5.F2.4.m1.2.2.3.cmml"><mo stretchy="false" id="Sx5.F2.4.m1.2.2.2.3" xref="Sx5.F2.4.m1.2.2.3.cmml">(</mo><mrow id="Sx5.F2.4.m1.1.1.1.1" xref="Sx5.F2.4.m1.1.1.1.1.cmml"><msub id="Sx5.F2.4.m1.1.1.1.1.2" xref="Sx5.F2.4.m1.1.1.1.1.2.cmml"><mi id="Sx5.F2.4.m1.1.1.1.1.2.2" xref="Sx5.F2.4.m1.1.1.1.1.2.2.cmml">n</mi><mi id="Sx5.F2.4.m1.1.1.1.1.2.3" xref="Sx5.F2.4.m1.1.1.1.1.2.3.cmml">e</mi></msub><mo id="Sx5.F2.4.m1.1.1.1.1.1" xref="Sx5.F2.4.m1.1.1.1.1.1.cmml">+</mo><mn id="Sx5.F2.4.m1.1.1.1.1.3" xref="Sx5.F2.4.m1.1.1.1.1.3.cmml">1</mn></mrow><mo id="Sx5.F2.4.m1.2.2.2.4" xref="Sx5.F2.4.m1.2.2.3.cmml">,</mo><mrow id="Sx5.F2.4.m1.2.2.2.2" xref="Sx5.F2.4.m1.2.2.2.2.cmml"><mrow id="Sx5.F2.4.m1.2.2.2.2.2" xref="Sx5.F2.4.m1.2.2.2.2.2.cmml"><msub id="Sx5.F2.4.m1.2.2.2.2.2.2" xref="Sx5.F2.4.m1.2.2.2.2.2.2.cmml"><mi id="Sx5.F2.4.m1.2.2.2.2.2.2.2" xref="Sx5.F2.4.m1.2.2.2.2.2.2.2.cmml">n</mi><mi id="Sx5.F2.4.m1.2.2.2.2.2.2.3" xref="Sx5.F2.4.m1.2.2.2.2.2.2.3.cmml">c</mi></msub><mo id="Sx5.F2.4.m1.2.2.2.2.2.1" xref="Sx5.F2.4.m1.2.2.2.2.2.1.cmml">−</mo><msub id="Sx5.F2.4.m1.2.2.2.2.2.3" xref="Sx5.F2.4.m1.2.2.2.2.2.3.cmml"><mi id="Sx5.F2.4.m1.2.2.2.2.2.3.2" xref="Sx5.F2.4.m1.2.2.2.2.2.3.2.cmml">n</mi><mi id="Sx5.F2.4.m1.2.2.2.2.2.3.3" xref="Sx5.F2.4.m1.2.2.2.2.2.3.3.cmml">e</mi></msub></mrow><mo id="Sx5.F2.4.m1.2.2.2.2.1" xref="Sx5.F2.4.m1.2.2.2.2.1.cmml">+</mo><mn id="Sx5.F2.4.m1.2.2.2.2.3" xref="Sx5.F2.4.m1.2.2.2.2.3.cmml">1</mn></mrow><mo stretchy="false" id="Sx5.F2.4.m1.2.2.2.5" xref="Sx5.F2.4.m1.2.2.3.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="Sx5.F2.4.m1.2c"><interval closure="open" id="Sx5.F2.4.m1.2.2.3.cmml" xref="Sx5.F2.4.m1.2.2.2"><apply id="Sx5.F2.4.m1.1.1.1.1.cmml" xref="Sx5.F2.4.m1.1.1.1.1"><plus id="Sx5.F2.4.m1.1.1.1.1.1.cmml" xref="Sx5.F2.4.m1.1.1.1.1.1"></plus><apply id="Sx5.F2.4.m1.1.1.1.1.2.cmml" xref="Sx5.F2.4.m1.1.1.1.1.2"><csymbol cd="ambiguous" id="Sx5.F2.4.m1.1.1.1.1.2.1.cmml" xref="Sx5.F2.4.m1.1.1.1.1.2">subscript</csymbol><ci id="Sx5.F2.4.m1.1.1.1.1.2.2.cmml" xref="Sx5.F2.4.m1.1.1.1.1.2.2">𝑛</ci><ci id="Sx5.F2.4.m1.1.1.1.1.2.3.cmml" xref="Sx5.F2.4.m1.1.1.1.1.2.3">𝑒</ci></apply><cn type="integer" id="Sx5.F2.4.m1.1.1.1.1.3.cmml" xref="Sx5.F2.4.m1.1.1.1.1.3">1</cn></apply><apply id="Sx5.F2.4.m1.2.2.2.2.cmml" xref="Sx5.F2.4.m1.2.2.2.2"><plus id="Sx5.F2.4.m1.2.2.2.2.1.cmml" xref="Sx5.F2.4.m1.2.2.2.2.1"></plus><apply id="Sx5.F2.4.m1.2.2.2.2.2.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2"><minus id="Sx5.F2.4.m1.2.2.2.2.2.1.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.1"></minus><apply id="Sx5.F2.4.m1.2.2.2.2.2.2.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.2"><csymbol cd="ambiguous" id="Sx5.F2.4.m1.2.2.2.2.2.2.1.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.2">subscript</csymbol><ci id="Sx5.F2.4.m1.2.2.2.2.2.2.2.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.2.2">𝑛</ci><ci id="Sx5.F2.4.m1.2.2.2.2.2.2.3.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.2.3">𝑐</ci></apply><apply id="Sx5.F2.4.m1.2.2.2.2.2.3.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.3"><csymbol cd="ambiguous" id="Sx5.F2.4.m1.2.2.2.2.2.3.1.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.3">subscript</csymbol><ci id="Sx5.F2.4.m1.2.2.2.2.2.3.2.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.3.2">𝑛</ci><ci id="Sx5.F2.4.m1.2.2.2.2.2.3.3.cmml" xref="Sx5.F2.4.m1.2.2.2.2.2.3.3">𝑒</ci></apply></apply><cn type="integer" id="Sx5.F2.4.m1.2.2.2.2.3.cmml" xref="Sx5.F2.4.m1.2.2.2.2.3">1</cn></apply></interval></annotation-xml><annotation encoding="application/x-tex" id="Sx5.F2.4.m1.2d">(n_{e}+1,n_{c}-n_{e}+1)</annotation></semantics></math> conjugate the assumed binomial distribution of observed errors, where <math id="Sx5.F2.5.m2.1" class="ltx_Math" alttext="n_{e}" display="inline"><semantics id="Sx5.F2.5.m2.1b"><msub id="Sx5.F2.5.m2.1.1" xref="Sx5.F2.5.m2.1.1.cmml"><mi id="Sx5.F2.5.m2.1.1.2" xref="Sx5.F2.5.m2.1.1.2.cmml">n</mi><mi id="Sx5.F2.5.m2.1.1.3" xref="Sx5.F2.5.m2.1.1.3.cmml">e</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.F2.5.m2.1c"><apply id="Sx5.F2.5.m2.1.1.cmml" xref="Sx5.F2.5.m2.1.1"><csymbol cd="ambiguous" id="Sx5.F2.5.m2.1.1.1.cmml" xref="Sx5.F2.5.m2.1.1">subscript</csymbol><ci id="Sx5.F2.5.m2.1.1.2.cmml" xref="Sx5.F2.5.m2.1.1.2">𝑛</ci><ci id="Sx5.F2.5.m2.1.1.3.cmml" xref="Sx5.F2.5.m2.1.1.3">𝑒</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.F2.5.m2.1d">n_{e}</annotation></semantics></math> and <math id="Sx5.F2.6.m3.1" class="ltx_Math" alttext="n_{c}" display="inline"><semantics id="Sx5.F2.6.m3.1b"><msub id="Sx5.F2.6.m3.1.1" xref="Sx5.F2.6.m3.1.1.cmml"><mi id="Sx5.F2.6.m3.1.1.2" xref="Sx5.F2.6.m3.1.1.2.cmml">n</mi><mi id="Sx5.F2.6.m3.1.1.3" xref="Sx5.F2.6.m3.1.1.3.cmml">c</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.F2.6.m3.1c"><apply id="Sx5.F2.6.m3.1.1.cmml" xref="Sx5.F2.6.m3.1.1"><csymbol cd="ambiguous" id="Sx5.F2.6.m3.1.1.1.cmml" xref="Sx5.F2.6.m3.1.1">subscript</csymbol><ci id="Sx5.F2.6.m3.1.1.2.cmml" xref="Sx5.F2.6.m3.1.1.2">𝑛</ci><ci id="Sx5.F2.6.m3.1.1.3.cmml" xref="Sx5.F2.6.m3.1.1.3">𝑐</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.F2.6.m3.1d">n_{c}</annotation></semantics></math> are the number of errors and number of calls in each bin respectively.</figcaption>
</figure>
<div id="Sx5.SSx1.p8" class="ltx_para">
<p id="Sx5.SSx1.p8.1" class="ltx_p">Applying this model to the <em id="Sx5.SSx1.p8.1.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> marker genome calls, we
estimated that the genotype calling error rate in the map bin
representative markers was 0.0099. 51% of adjacent map bin pairs are
separated by a single inferred recombination event in the cross, and 94% are
separated by three or fewer recombinants in each parent.</p>
</div>
<div id="Sx5.SSx1.p9" class="ltx_para">
<p id="Sx5.SSx1.p9.1" class="ltx_p">Of the 91,320 markers with at least 18 putative SNPs, 84,307 (92%)
were assigned to their closest map bins with a threshold of <math id="Sx5.SSx1.p9.1.m1.1" class="ltx_Math" alttext="p&lt;10^{-6}" display="inline"><semantics id="Sx5.SSx1.p9.1.m1.1a"><mrow id="Sx5.SSx1.p9.1.m1.1.1" xref="Sx5.SSx1.p9.1.m1.1.1.cmml"><mi id="Sx5.SSx1.p9.1.m1.1.1.2" xref="Sx5.SSx1.p9.1.m1.1.1.2.cmml">p</mi><mo id="Sx5.SSx1.p9.1.m1.1.1.1" xref="Sx5.SSx1.p9.1.m1.1.1.1.cmml">&lt;</mo><msup id="Sx5.SSx1.p9.1.m1.1.1.3" xref="Sx5.SSx1.p9.1.m1.1.1.3.cmml"><mn id="Sx5.SSx1.p9.1.m1.1.1.3.2" xref="Sx5.SSx1.p9.1.m1.1.1.3.2.cmml">10</mn><mrow id="Sx5.SSx1.p9.1.m1.1.1.3.3" xref="Sx5.SSx1.p9.1.m1.1.1.3.3.cmml"><mo id="Sx5.SSx1.p9.1.m1.1.1.3.3a" xref="Sx5.SSx1.p9.1.m1.1.1.3.3.cmml">−</mo><mn id="Sx5.SSx1.p9.1.m1.1.1.3.3.2" xref="Sx5.SSx1.p9.1.m1.1.1.3.3.2.cmml">6</mn></mrow></msup></mrow><annotation-xml encoding="MathML-Content" id="Sx5.SSx1.p9.1.m1.1b"><apply id="Sx5.SSx1.p9.1.m1.1.1.cmml" xref="Sx5.SSx1.p9.1.m1.1.1"><lt id="Sx5.SSx1.p9.1.m1.1.1.1.cmml" xref="Sx5.SSx1.p9.1.m1.1.1.1"></lt><ci id="Sx5.SSx1.p9.1.m1.1.1.2.cmml" xref="Sx5.SSx1.p9.1.m1.1.1.2">𝑝</ci><apply id="Sx5.SSx1.p9.1.m1.1.1.3.cmml" xref="Sx5.SSx1.p9.1.m1.1.1.3"><csymbol cd="ambiguous" id="Sx5.SSx1.p9.1.m1.1.1.3.1.cmml" xref="Sx5.SSx1.p9.1.m1.1.1.3">superscript</csymbol><cn type="integer" id="Sx5.SSx1.p9.1.m1.1.1.3.2.cmml" xref="Sx5.SSx1.p9.1.m1.1.1.3.2">10</cn><apply id="Sx5.SSx1.p9.1.m1.1.1.3.3.cmml" xref="Sx5.SSx1.p9.1.m1.1.1.3.3"><minus id="Sx5.SSx1.p9.1.m1.1.1.3.3.1.cmml" xref="Sx5.SSx1.p9.1.m1.1.1.3.3"></minus><cn type="integer" id="Sx5.SSx1.p9.1.m1.1.1.3.3.2.cmml" xref="Sx5.SSx1.p9.1.m1.1.1.3.3.2">6</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx1.p9.1.m1.1c">p&lt;10^{-6}</annotation></semantics></math> (Methods), for an estimated genome-wide average density of one mapped
sequence marker every 32 kb. A mean of 45 markers were mapped to each map
bin, and the number of markers mapped was used to estimate the
relative physical size of map bins. 46% of the scaffolds with 12-17 SNPs
could be placed with the same threshold, for an additional 32,688 markers, or
one marker every 23 kb.</p>
</div>
<div id="Sx5.SSx1.p10" class="ltx_para">
<p id="Sx5.SSx1.p10.1" class="ltx_p">The total length of the scaffolds assigned to map bins was 411 Mb, and
they contained 2.67 million bi-allelic SNPs assigned a phase with a
posterior probability of at least 0.99. Of these, 72% were inferred
to be unique to one of the four parental chromosomes. This is close
to the 74% predicted under the finite sites neutral coalescent
model given the observed SNP density<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib29" title="" class="ltx_ref">29</a>]</cite>.</p>
</div>
<figure id="Sx5.F3" class="ltx_figure"><img src="/html/1309.7402/assets/x2.png" id="Sx5.F3.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="415" height="581" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>Each of the numbered blocks represents one of the 32 linkage groups of the <em id="Sx5.F3.2.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> genetic map, and is composed of four columns: Two bands of the triangular matrices in which the color scale indicates the fraction of samples showing recombination between pairs of markers; maternal recombination frequency is shown on the left, paternal on the right. A column labeled “ALG” indicates segments of significant (p&lt;0.05 in Fisher’s Exact Test, after Bonferroni correction for multiple tests) conservation of gene content with ancestral bilaterian linage groups. The column labeled HOX shows the map positions and types of predicted homeobox transcription factor genes. The two color scales are for: recombination frequency between pairs of markers and log p-value for enrichment in gene content with ancestral linkage groups.</figcaption>
</figure>
</section>
<section id="Sx5.SSx2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Sequence composition and recombination rate</h3>

<div id="Sx5.SSx2.p1" class="ltx_para">
<p id="Sx5.SSx2.p1.2" class="ltx_p">In the scaffolds longer than 1kb (mean length 2.9 kb), the <math id="Sx5.SSx2.p1.1.m1.1" class="ltx_Math" alttext="G/C" display="inline"><semantics id="Sx5.SSx2.p1.1.m1.1a"><mrow id="Sx5.SSx2.p1.1.m1.1.1" xref="Sx5.SSx2.p1.1.m1.1.1.cmml"><mi id="Sx5.SSx2.p1.1.m1.1.1.2" xref="Sx5.SSx2.p1.1.m1.1.1.2.cmml">G</mi><mo id="Sx5.SSx2.p1.1.m1.1.1.1" xref="Sx5.SSx2.p1.1.m1.1.1.1.cmml">/</mo><mi id="Sx5.SSx2.p1.1.m1.1.1.3" xref="Sx5.SSx2.p1.1.m1.1.1.3.cmml">C</mi></mrow><annotation-xml encoding="MathML-Content" id="Sx5.SSx2.p1.1.m1.1b"><apply id="Sx5.SSx2.p1.1.m1.1.1.cmml" xref="Sx5.SSx2.p1.1.m1.1.1"><divide id="Sx5.SSx2.p1.1.m1.1.1.1.cmml" xref="Sx5.SSx2.p1.1.m1.1.1.1"></divide><ci id="Sx5.SSx2.p1.1.m1.1.1.2.cmml" xref="Sx5.SSx2.p1.1.m1.1.1.2">𝐺</ci><ci id="Sx5.SSx2.p1.1.m1.1.1.3.cmml" xref="Sx5.SSx2.p1.1.m1.1.1.3">𝐶</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx2.p1.1.m1.1c">G/C</annotation></semantics></math> base
content was 33.3 <math id="Sx5.SSx2.p1.2.m2.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx2.p1.2.m2.1a"><mo id="Sx5.SSx2.p1.2.m2.1.1" xref="Sx5.SSx2.p1.2.m2.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx2.p1.2.m2.1b"><csymbol cd="latexml" id="Sx5.SSx2.p1.2.m2.1.1.cmml" xref="Sx5.SSx2.p1.2.m2.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx2.p1.2.m2.1c">\pm</annotation></semantics></math> 2.8 percent, and the local relative frequency
of CpG dinucleotides was bimodally distributed, with about 30% of sequences exhibiting depletion of
CpG. TpG and CpA dinucleotides were over-represented on average
and their local densities
negatively correlated with CpG density, suggesting ongoing germ-line
CpG methylation for a fraction of the genome<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib30" title="" class="ltx_ref">30</a>]</cite>.</p>
</div>
<div id="Sx5.SSx2.p2" class="ltx_para">
<p id="Sx5.SSx2.p2.2" class="ltx_p">The mean maternal and paternal recombination rates were estimated to
be 1.28 and 0.76 centimorgans per megabase respectively, consistent
with expectation based on genome size<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib31" title="" class="ltx_ref">31</a>]</cite>.
We did not observe evidence of segregation distortion
for any map bins.
Estimated
local recombination rates in the two parents are correlated across the
genome with and <math id="Sx5.SSx2.p2.1.m1.1" class="ltx_Math" alttext="r^{2}" display="inline"><semantics id="Sx5.SSx2.p2.1.m1.1a"><msup id="Sx5.SSx2.p2.1.m1.1.1" xref="Sx5.SSx2.p2.1.m1.1.1.cmml"><mi id="Sx5.SSx2.p2.1.m1.1.1.2" xref="Sx5.SSx2.p2.1.m1.1.1.2.cmml">r</mi><mn id="Sx5.SSx2.p2.1.m1.1.1.3" xref="Sx5.SSx2.p2.1.m1.1.1.3.cmml">2</mn></msup><annotation-xml encoding="MathML-Content" id="Sx5.SSx2.p2.1.m1.1b"><apply id="Sx5.SSx2.p2.1.m1.1.1.cmml" xref="Sx5.SSx2.p2.1.m1.1.1"><csymbol cd="ambiguous" id="Sx5.SSx2.p2.1.m1.1.1.1.cmml" xref="Sx5.SSx2.p2.1.m1.1.1">superscript</csymbol><ci id="Sx5.SSx2.p2.1.m1.1.1.2.cmml" xref="Sx5.SSx2.p2.1.m1.1.1.2">𝑟</ci><cn type="integer" id="Sx5.SSx2.p2.1.m1.1.1.3.cmml" xref="Sx5.SSx2.p2.1.m1.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx2.p2.1.m1.1c">r^{2}</annotation></semantics></math> = 7.1% ;
p&lt;1e-29, and the mean local recombination rate is correlated with
local SNP density, <math id="Sx5.SSx2.p2.2.m2.1" class="ltx_Math" alttext="r^{2}" display="inline"><semantics id="Sx5.SSx2.p2.2.m2.1a"><msup id="Sx5.SSx2.p2.2.m2.1.1" xref="Sx5.SSx2.p2.2.m2.1.1.cmml"><mi id="Sx5.SSx2.p2.2.m2.1.1.2" xref="Sx5.SSx2.p2.2.m2.1.1.2.cmml">r</mi><mn id="Sx5.SSx2.p2.2.m2.1.1.3" xref="Sx5.SSx2.p2.2.m2.1.1.3.cmml">2</mn></msup><annotation-xml encoding="MathML-Content" id="Sx5.SSx2.p2.2.m2.1b"><apply id="Sx5.SSx2.p2.2.m2.1.1.cmml" xref="Sx5.SSx2.p2.2.m2.1.1"><csymbol cd="ambiguous" id="Sx5.SSx2.p2.2.m2.1.1.1.cmml" xref="Sx5.SSx2.p2.2.m2.1.1">superscript</csymbol><ci id="Sx5.SSx2.p2.2.m2.1.1.2.cmml" xref="Sx5.SSx2.p2.2.m2.1.1.2">𝑟</ci><cn type="integer" id="Sx5.SSx2.p2.2.m2.1.1.3.cmml" xref="Sx5.SSx2.p2.2.m2.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx2.p2.2.m2.1c">r^{2}</annotation></semantics></math> = 9.7% ; p&lt;1e-40.</p>
</div>
</section>
<section id="Sx5.SSx3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Ancestral linkage group conservation</h3>

<div id="Sx5.SSx3.p1" class="ltx_para">
<p id="Sx5.SSx3.p1.1" class="ltx_p">34,942 scaffolds have significant sequence conservation with 10,399
predicted proteins of the tick <em id="Sx5.SSx3.p1.1.1" class="ltx_emph ltx_font_italic">Ixodes scapularis</em><cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib32" title="" class="ltx_ref">32</a>]</cite>. 6,246 of these hits formed
reciprocal best pairs, of which 5,775 (92%) could be placed on the
linkage map at a threshold of p&lt;<math id="Sx5.SSx3.p1.1.m1.1" class="ltx_Math" alttext="10^{-6}" display="inline"><semantics id="Sx5.SSx3.p1.1.m1.1a"><msup id="Sx5.SSx3.p1.1.m1.1.1" xref="Sx5.SSx3.p1.1.m1.1.1.cmml"><mn id="Sx5.SSx3.p1.1.m1.1.1.2" xref="Sx5.SSx3.p1.1.m1.1.1.2.cmml">10</mn><mrow id="Sx5.SSx3.p1.1.m1.1.1.3" xref="Sx5.SSx3.p1.1.m1.1.1.3.cmml"><mo id="Sx5.SSx3.p1.1.m1.1.1.3a" xref="Sx5.SSx3.p1.1.m1.1.1.3.cmml">−</mo><mn id="Sx5.SSx3.p1.1.m1.1.1.3.2" xref="Sx5.SSx3.p1.1.m1.1.1.3.2.cmml">6</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="Sx5.SSx3.p1.1.m1.1b"><apply id="Sx5.SSx3.p1.1.m1.1.1.cmml" xref="Sx5.SSx3.p1.1.m1.1.1"><csymbol cd="ambiguous" id="Sx5.SSx3.p1.1.m1.1.1.1.cmml" xref="Sx5.SSx3.p1.1.m1.1.1">superscript</csymbol><cn type="integer" id="Sx5.SSx3.p1.1.m1.1.1.2.cmml" xref="Sx5.SSx3.p1.1.m1.1.1.2">10</cn><apply id="Sx5.SSx3.p1.1.m1.1.1.3.cmml" xref="Sx5.SSx3.p1.1.m1.1.1.3"><minus id="Sx5.SSx3.p1.1.m1.1.1.3.1.cmml" xref="Sx5.SSx3.p1.1.m1.1.1.3"></minus><cn type="integer" id="Sx5.SSx3.p1.1.m1.1.1.3.2.cmml" xref="Sx5.SSx3.p1.1.m1.1.1.3.2">6</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx3.p1.1.m1.1c">10^{-6}</annotation></semantics></math>. These were used as
conserved markers for comparisons of genome organization. When
linkage groups were divided into 108 non-overlapping bins of 1,000
markers, 52 had significant (p&lt;0.05, after Bonferroni correction for
1,944 pairwise tests) enrichment in shared orthologs (or “hit”) with at
least one of eighteen ancestral chordate linkage
groups<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib7" title="" class="ltx_ref">7</a>]</cite>. A hidden Markov model
segmentation algorithm<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib6" title="" class="ltx_ref">6</a>]</cite> identified 40 breakpoints
in ALG composition in the linkage groups. 72% of the genome is
spanned by 53 intervening segments that hit one or (for eight of them)
two ALGs (Figures <a href="#Sx5.F4" title="Figure 4 ‣ Ancestral linkage group conservation ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a> and <a href="#Sx5.F5" title="Figure 5 ‣ Ancestral linkage group conservation ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>). Each of
the eighteen ancestral ALGs has at least one hit among the 45
segments with a unique hit to the ALGs.</p>
</div>
<figure id="Sx5.F4" class="ltx_figure"><img src="/html/1309.7402/assets/x3.png" id="Sx5.F4.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="277" height="466" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 4: </span>Limulus-Human macro-synteny dot plot. Blue points indicate the position of human genes in reconstructed ancestral chordate ALGs (vertical displacement) and their candidate orthologs in the 30 <em id="Sx5.F4.2.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> linkage groups (horizontal displacement).</figcaption>
</figure>
<figure id="Sx5.F5" class="ltx_figure"><img src="/html/1309.7402/assets/x4.png" id="Sx5.F5.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="277" height="466" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 5: </span>Limulus-Human macro-synteny dot plot as in Figure <a href="#Sx5.F4" title="Figure 4 ‣ Ancestral linkage group conservation ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>, showing breaks introduced by hidden Markov model segmentation of the linkage groups as vertical lines.</figcaption>
</figure>
</section>
<section id="Sx5.SSx4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Whole genome duplications</h3>

<div id="Sx5.SSx4.p1" class="ltx_para">
<p id="Sx5.SSx4.p1.1" class="ltx_p">Whole genome duplication (WGD), or polyploidization is a rare but
dramatic genetic mutation event which doubles the size of a genome and
creates a redundant pair of copies from every gene. Because it
creates redundant copies of genes for entire biochemical pathways and
genetic networks, it has been proposed that it creates unique raw
material for the evolution of novel biological functions and increased
complexity.</p>
</div>
</section>
<section id="Sx5.SSx5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Homeobox gene clusters</h3>

<div id="Sx5.SSx5.p1" class="ltx_para">
<p id="Sx5.SSx5.p1.1" class="ltx_p">Homeobox genes encode a large family of transcription factors involved
in diverse embryonic patterning and structure formation processes of
eukaryotes. As a particular subfamily of homeobox genes, the Hox
cluster is known to control metazoan body patterning along the
anterior-posterior axis. We identified 155 scaffolds with significant
homology to predicted chelicerate homeobox gene sequences in public
databases. We classified these sequences into homeobox subfamilies (Methods)
and placed them on the map by best hit. Two large clusters of Hox
genes are found on linkage group (LG) 15 and LG 21, each containing multiple members
of the anterior, central and posterior classes. There are also two
parahox cluster homologs, each with three homeobox genes: <em id="Sx5.SSx5.p1.1.1" class="ltx_emph ltx_font_italic">gsx</em> and
<em id="Sx5.SSx5.p1.1.2" class="ltx_emph ltx_font_italic">cdx</em> orthologs and a third homeobox gene not confidently assigned
to a subfamily in our analysis (LG 5 and LG 19). There are two smaller
clusters containing multiple hox genes (LG 18 and LG 20), and clusters
of other homeobox genes, including members of the <em id="Sx5.SSx5.p1.1.3" class="ltx_emph ltx_font_italic">msx</em>, <em id="Sx5.SSx5.p1.1.4" class="ltx_emph ltx_font_italic">lbx</em>, <em id="Sx5.SSx5.p1.1.5" class="ltx_emph ltx_font_italic">nk</em>,
<em id="Sx5.SSx5.p1.1.6" class="ltx_emph ltx_font_italic">evx</em>, and <em id="Sx5.SSx5.p1.1.7" class="ltx_emph ltx_font_italic">gbx</em> families (Figure 1).</p>
</div>
</section>
<section id="Sx5.SSx6" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Genomic distribution of paralogous genes</h3>

<div id="Sx5.SSx6.p1" class="ltx_para">
<p id="Sx5.SSx6.p1.2" class="ltx_p">WGD creates many pairs of duplicate genes or “paralogs”. The
distinctive features of these genes have been used to infer WGD events
in fungal, vertebrate and plant genomes<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib2" title="" class="ltx_ref">2</a>, <a href="#bib.bib3" title="" class="ltx_ref">3</a>, <a href="#bib.bib4" title="" class="ltx_ref">4</a>]</cite>.
We examined the genomic distribution of 2,716 pairs of candidate
paralogous gene markers in <em id="Sx5.SSx6.p1.2.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> for signatures of WGD. In
45% of these pairs both markers mapped to the same chromosome,
compared to 5.3 <math id="Sx5.SSx6.p1.1.m1.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p1.1.m1.1a"><mo id="Sx5.SSx6.p1.1.m1.1.1" xref="Sx5.SSx6.p1.1.m1.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p1.1.m1.1b"><csymbol cd="latexml" id="Sx5.SSx6.p1.1.m1.1.1.cmml" xref="Sx5.SSx6.p1.1.m1.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p1.1.m1.1c">\pm</annotation></semantics></math> 0.5 % in 1000 datasets with randomly-permuted
paralogous gene identities. The mapped positions of pairs within the
chromosomes were highly correlated (average <math id="Sx5.SSx6.p1.2.m2.1" class="ltx_Math" alttext="r^{2}" display="inline"><semantics id="Sx5.SSx6.p1.2.m2.1a"><msup id="Sx5.SSx6.p1.2.m2.1.1" xref="Sx5.SSx6.p1.2.m2.1.1.cmml"><mi id="Sx5.SSx6.p1.2.m2.1.1.2" xref="Sx5.SSx6.p1.2.m2.1.1.2.cmml">r</mi><mn id="Sx5.SSx6.p1.2.m2.1.1.3" xref="Sx5.SSx6.p1.2.m2.1.1.3.cmml">2</mn></msup><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p1.2.m2.1b"><apply id="Sx5.SSx6.p1.2.m2.1.1.cmml" xref="Sx5.SSx6.p1.2.m2.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p1.2.m2.1.1.1.cmml" xref="Sx5.SSx6.p1.2.m2.1.1">superscript</csymbol><ci id="Sx5.SSx6.p1.2.m2.1.1.2.cmml" xref="Sx5.SSx6.p1.2.m2.1.1.2">𝑟</ci><cn type="integer" id="Sx5.SSx6.p1.2.m2.1.1.3.cmml" xref="Sx5.SSx6.p1.2.m2.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p1.2.m2.1c">r^{2}</annotation></semantics></math> = 0.81, and exceeding
0.95 for 8 of the large chromosomes; Figure <a href="#Sx7.F8" title="Figure 8 ‣ Ka and Ks estimation for paralogs and T. tridentatus orthologs ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>),
suggesting that many of the pairs represent recent tandem gene
duplicates or single genes fragmented across multiple markers. In the following, these
same-chromosome paralogs are referred to as “tandem” duplicates.</p>
</div>
<div id="Sx5.SSx6.p2" class="ltx_para">
<p id="Sx5.SSx6.p2.11" class="ltx_p">Inter-chromosomal duplicates are clustered into conserved paralogous
micro-synteny blocks (or “paralogons”<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite>): there are 25 pairs of
loci, each with at least six (<math id="Sx5.SSx6.p2.1.m1.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p2.1.m1.1a"><msub id="Sx5.SSx6.p2.1.m1.1.1" xref="Sx5.SSx6.p2.1.m1.1.1.cmml"><mi id="Sx5.SSx6.p2.1.m1.1.1.2" xref="Sx5.SSx6.p2.1.m1.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p2.1.m1.1.1.3" xref="Sx5.SSx6.p2.1.m1.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.1.m1.1b"><apply id="Sx5.SSx6.p2.1.m1.1.1.cmml" xref="Sx5.SSx6.p2.1.m1.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p2.1.m1.1.1.1.cmml" xref="Sx5.SSx6.p2.1.m1.1.1">subscript</csymbol><ci id="Sx5.SSx6.p2.1.m1.1.1.2.cmml" xref="Sx5.SSx6.p2.1.m1.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p2.1.m1.1.1.3.cmml" xref="Sx5.SSx6.p2.1.m1.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.1.m1.1c">m_{p}</annotation></semantics></math> =6) paralog pairs clustered with
a maximum gap (max-gap) of 300 markers between adjacent paralogs in
each cluster. These clusters span 25,044 markers, or 30% of the map, after removing
redundancy from paralogons with overlapping footprints. In 1000
datasets with randomly-permuted paralogous gene identities, the
maximum number of such clusters observed was 11; the mean and standard
deviation were 3.9 <math id="Sx5.SSx6.p2.2.m2.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p2.2.m2.1a"><mo id="Sx5.SSx6.p2.2.m2.1.1" xref="Sx5.SSx6.p2.2.m2.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.2.m2.1b"><csymbol cd="latexml" id="Sx5.SSx6.p2.2.m2.1.1.cmml" xref="Sx5.SSx6.p2.2.m2.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.2.m2.1c">\pm</annotation></semantics></math> 1.0. The observed clustering into paralogons
was greater than that in the randomized datasets over a broad range of
choices of max-gap and <math id="Sx5.SSx6.p2.3.m3.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p2.3.m3.1a"><msub id="Sx5.SSx6.p2.3.m3.1.1" xref="Sx5.SSx6.p2.3.m3.1.1.cmml"><mi id="Sx5.SSx6.p2.3.m3.1.1.2" xref="Sx5.SSx6.p2.3.m3.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p2.3.m3.1.1.3" xref="Sx5.SSx6.p2.3.m3.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.3.m3.1b"><apply id="Sx5.SSx6.p2.3.m3.1.1.cmml" xref="Sx5.SSx6.p2.3.m3.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p2.3.m3.1.1.1.cmml" xref="Sx5.SSx6.p2.3.m3.1.1">subscript</csymbol><ci id="Sx5.SSx6.p2.3.m3.1.1.2.cmml" xref="Sx5.SSx6.p2.3.m3.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p2.3.m3.1.1.3.cmml" xref="Sx5.SSx6.p2.3.m3.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.3.m3.1c">m_{p}</annotation></semantics></math>. For example, for max-gap=100, <math id="Sx5.SSx6.p2.4.m4.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p2.4.m4.1a"><msub id="Sx5.SSx6.p2.4.m4.1.1" xref="Sx5.SSx6.p2.4.m4.1.1.cmml"><mi id="Sx5.SSx6.p2.4.m4.1.1.2" xref="Sx5.SSx6.p2.4.m4.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p2.4.m4.1.1.3" xref="Sx5.SSx6.p2.4.m4.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.4.m4.1b"><apply id="Sx5.SSx6.p2.4.m4.1.1.cmml" xref="Sx5.SSx6.p2.4.m4.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p2.4.m4.1.1.1.cmml" xref="Sx5.SSx6.p2.4.m4.1.1">subscript</csymbol><ci id="Sx5.SSx6.p2.4.m4.1.1.2.cmml" xref="Sx5.SSx6.p2.4.m4.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p2.4.m4.1.1.3.cmml" xref="Sx5.SSx6.p2.4.m4.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.4.m4.1c">m_{p}</annotation></semantics></math> =3
there are 52 clusters vs. 3.5 <math id="Sx5.SSx6.p2.5.m5.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p2.5.m5.1a"><mo id="Sx5.SSx6.p2.5.m5.1.1" xref="Sx5.SSx6.p2.5.m5.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.5.m5.1b"><csymbol cd="latexml" id="Sx5.SSx6.p2.5.m5.1.1.cmml" xref="Sx5.SSx6.p2.5.m5.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.5.m5.1c">\pm</annotation></semantics></math> 1.9, range 0-10; for max-gap=500,
<math id="Sx5.SSx6.p2.6.m6.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p2.6.m6.1a"><msub id="Sx5.SSx6.p2.6.m6.1.1" xref="Sx5.SSx6.p2.6.m6.1.1.cmml"><mi id="Sx5.SSx6.p2.6.m6.1.1.2" xref="Sx5.SSx6.p2.6.m6.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p2.6.m6.1.1.3" xref="Sx5.SSx6.p2.6.m6.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.6.m6.1b"><apply id="Sx5.SSx6.p2.6.m6.1.1.cmml" xref="Sx5.SSx6.p2.6.m6.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p2.6.m6.1.1.1.cmml" xref="Sx5.SSx6.p2.6.m6.1.1">subscript</csymbol><ci id="Sx5.SSx6.p2.6.m6.1.1.2.cmml" xref="Sx5.SSx6.p2.6.m6.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p2.6.m6.1.1.3.cmml" xref="Sx5.SSx6.p2.6.m6.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.6.m6.1c">m_{p}</annotation></semantics></math> = 9 there are 12 vs. 2.9 <math id="Sx5.SSx6.p2.7.m7.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p2.7.m7.1a"><mo id="Sx5.SSx6.p2.7.m7.1.1" xref="Sx5.SSx6.p2.7.m7.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.7.m7.1b"><csymbol cd="latexml" id="Sx5.SSx6.p2.7.m7.1.1.cmml" xref="Sx5.SSx6.p2.7.m7.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.7.m7.1c">\pm</annotation></semantics></math> 1.7, range 0-9. Because of the
large proportion of apparent tandem gene duplicates (45%), this
randomization scheme increases the number of inter-chromosomal paralog
pairs relative to the data, making it a conservative significance test
for inter-chromosomal paralog clustering. When genes with tandem
duplicates are excluded from the randomizations, the observed number
of clusters is greater than the maximum observed in 1000
randomizations for all the combinations of max-gap in the set
(100, 200, 300, 400, 500, 600) and <math id="Sx5.SSx6.p2.8.m8.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p2.8.m8.1a"><msub id="Sx5.SSx6.p2.8.m8.1.1" xref="Sx5.SSx6.p2.8.m8.1.1.cmml"><mi id="Sx5.SSx6.p2.8.m8.1.1.2" xref="Sx5.SSx6.p2.8.m8.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p2.8.m8.1.1.3" xref="Sx5.SSx6.p2.8.m8.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.8.m8.1b"><apply id="Sx5.SSx6.p2.8.m8.1.1.cmml" xref="Sx5.SSx6.p2.8.m8.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p2.8.m8.1.1.1.cmml" xref="Sx5.SSx6.p2.8.m8.1.1">subscript</csymbol><ci id="Sx5.SSx6.p2.8.m8.1.1.2.cmml" xref="Sx5.SSx6.p2.8.m8.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p2.8.m8.1.1.3.cmml" xref="Sx5.SSx6.p2.8.m8.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.8.m8.1c">m_{p}</annotation></semantics></math> in the set (3, 4, 5, 6, 7, 8,
9). 23
max-gap=600, <math id="Sx5.SSx6.p2.9.m9.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p2.9.m9.1a"><msub id="Sx5.SSx6.p2.9.m9.1.1" xref="Sx5.SSx6.p2.9.m9.1.1.cmml"><mi id="Sx5.SSx6.p2.9.m9.1.1.2" xref="Sx5.SSx6.p2.9.m9.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p2.9.m9.1.1.3" xref="Sx5.SSx6.p2.9.m9.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.9.m9.1b"><apply id="Sx5.SSx6.p2.9.m9.1.1.cmml" xref="Sx5.SSx6.p2.9.m9.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p2.9.m9.1.1.1.cmml" xref="Sx5.SSx6.p2.9.m9.1.1">subscript</csymbol><ci id="Sx5.SSx6.p2.9.m9.1.1.2.cmml" xref="Sx5.SSx6.p2.9.m9.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p2.9.m9.1.1.3.cmml" xref="Sx5.SSx6.p2.9.m9.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.9.m9.1c">m_{p}</annotation></semantics></math> = 7 clusters span 59% of the map, compared to respective mean
number and map coverage of 3.3 <math id="Sx5.SSx6.p2.10.m10.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p2.10.m10.1a"><mo id="Sx5.SSx6.p2.10.m10.1.1" xref="Sx5.SSx6.p2.10.m10.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.10.m10.1b"><csymbol cd="latexml" id="Sx5.SSx6.p2.10.m10.1.1.cmml" xref="Sx5.SSx6.p2.10.m10.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.10.m10.1c">\pm</annotation></semantics></math> 1.8, and 13 <math id="Sx5.SSx6.p2.11.m11.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p2.11.m11.1a"><mo id="Sx5.SSx6.p2.11.m11.1.1" xref="Sx5.SSx6.p2.11.m11.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p2.11.m11.1b"><csymbol cd="latexml" id="Sx5.SSx6.p2.11.m11.1.1.cmml" xref="Sx5.SSx6.p2.11.m11.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p2.11.m11.1c">\pm</annotation></semantics></math> 6 %
in these randomizations.</p>
</div>
<div id="Sx5.SSx6.p3" class="ltx_para">
<p id="Sx5.SSx6.p3.2" class="ltx_p">Among the marker pairs mapping to different chromosomes, we find a
significant excess of pairs relating segments derived from the same
ancestral linkage group (ALG) relative to randomization controls (247
pairs vs 102 <math id="Sx5.SSx6.p3.1.m1.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p3.1.m1.1a"><mo id="Sx5.SSx6.p3.1.m1.1.1" xref="Sx5.SSx6.p3.1.m1.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p3.1.m1.1b"><csymbol cd="latexml" id="Sx5.SSx6.p3.1.m1.1.1.cmml" xref="Sx5.SSx6.p3.1.m1.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p3.1.m1.1c">\pm</annotation></semantics></math> 11, p&lt;0.001 in randomizations of all genes; 202 vs
46 <math id="Sx5.SSx6.p3.2.m2.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p3.2.m2.1a"><mo id="Sx5.SSx6.p3.2.m2.1.1" xref="Sx5.SSx6.p3.2.m2.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p3.2.m2.1b"><csymbol cd="latexml" id="Sx5.SSx6.p3.2.m2.1.1.cmml" xref="Sx5.SSx6.p3.2.m2.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p3.2.m2.1c">\pm</annotation></semantics></math> 7 when genes in tandem duplicates are excluded). This pattern is consistent with the
creation of these segments by duplication (rather than fission).</p>
</div>
<div id="Sx5.SSx6.p4" class="ltx_para">
<p id="Sx5.SSx6.p4.8" class="ltx_p">The max-gap clusters have a significant amount of overlap among their
footprints. For example, the footprints of the max-gap=600, <math id="Sx5.SSx6.p4.1.m1.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p4.1.m1.1a"><msub id="Sx5.SSx6.p4.1.m1.1.1" xref="Sx5.SSx6.p4.1.m1.1.1.cmml"><mi id="Sx5.SSx6.p4.1.m1.1.1.2" xref="Sx5.SSx6.p4.1.m1.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p4.1.m1.1.1.3" xref="Sx5.SSx6.p4.1.m1.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p4.1.m1.1b"><apply id="Sx5.SSx6.p4.1.m1.1.1.cmml" xref="Sx5.SSx6.p4.1.m1.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p4.1.m1.1.1.1.cmml" xref="Sx5.SSx6.p4.1.m1.1.1">subscript</csymbol><ci id="Sx5.SSx6.p4.1.m1.1.1.2.cmml" xref="Sx5.SSx6.p4.1.m1.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p4.1.m1.1.1.3.cmml" xref="Sx5.SSx6.p4.1.m1.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p4.1.m1.1c">m_{p}</annotation></semantics></math> = 7 clusters had a
total length of 72,072 markers, but a net footprint after redundnacy
removal of 49,545 markers. We examined the relationships among the
paralogons for evidence of successive rounds of duplication. We
considered a graph in which nodes correspond to merged, non-redundant
paralogon footprint regions. Nodes are connected with edges if a
max-gap cluster connects the two nodes. The average clustering
coefficient of this graph is equal to the probability that footprints
<math id="Sx5.SSx6.p4.2.m2.1" class="ltx_Math" alttext="a" display="inline"><semantics id="Sx5.SSx6.p4.2.m2.1a"><mi id="Sx5.SSx6.p4.2.m2.1.1" xref="Sx5.SSx6.p4.2.m2.1.1.cmml">a</mi><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p4.2.m2.1b"><ci id="Sx5.SSx6.p4.2.m2.1.1.cmml" xref="Sx5.SSx6.p4.2.m2.1.1">𝑎</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p4.2.m2.1c">a</annotation></semantics></math> and <math id="Sx5.SSx6.p4.3.m3.1" class="ltx_Math" alttext="c" display="inline"><semantics id="Sx5.SSx6.p4.3.m3.1a"><mi id="Sx5.SSx6.p4.3.m3.1.1" xref="Sx5.SSx6.p4.3.m3.1.1.cmml">c</mi><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p4.3.m3.1b"><ci id="Sx5.SSx6.p4.3.m3.1.1.cmml" xref="Sx5.SSx6.p4.3.m3.1.1">𝑐</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p4.3.m3.1c">c</annotation></semantics></math> share a max-gap cluster, given that there are edges
<math id="Sx5.SSx6.p4.4.m4.2" class="ltx_Math" alttext="(a,b)" display="inline"><semantics id="Sx5.SSx6.p4.4.m4.2a"><mrow id="Sx5.SSx6.p4.4.m4.2.3.2" xref="Sx5.SSx6.p4.4.m4.2.3.1.cmml"><mo stretchy="false" id="Sx5.SSx6.p4.4.m4.2.3.2.1" xref="Sx5.SSx6.p4.4.m4.2.3.1.cmml">(</mo><mi id="Sx5.SSx6.p4.4.m4.1.1" xref="Sx5.SSx6.p4.4.m4.1.1.cmml">a</mi><mo id="Sx5.SSx6.p4.4.m4.2.3.2.2" xref="Sx5.SSx6.p4.4.m4.2.3.1.cmml">,</mo><mi id="Sx5.SSx6.p4.4.m4.2.2" xref="Sx5.SSx6.p4.4.m4.2.2.cmml">b</mi><mo stretchy="false" id="Sx5.SSx6.p4.4.m4.2.3.2.3" xref="Sx5.SSx6.p4.4.m4.2.3.1.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p4.4.m4.2b"><interval closure="open" id="Sx5.SSx6.p4.4.m4.2.3.1.cmml" xref="Sx5.SSx6.p4.4.m4.2.3.2"><ci id="Sx5.SSx6.p4.4.m4.1.1.cmml" xref="Sx5.SSx6.p4.4.m4.1.1">𝑎</ci><ci id="Sx5.SSx6.p4.4.m4.2.2.cmml" xref="Sx5.SSx6.p4.4.m4.2.2">𝑏</ci></interval></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p4.4.m4.2c">(a,b)</annotation></semantics></math> and <math id="Sx5.SSx6.p4.5.m5.2" class="ltx_Math" alttext="(b,c)" display="inline"><semantics id="Sx5.SSx6.p4.5.m5.2a"><mrow id="Sx5.SSx6.p4.5.m5.2.3.2" xref="Sx5.SSx6.p4.5.m5.2.3.1.cmml"><mo stretchy="false" id="Sx5.SSx6.p4.5.m5.2.3.2.1" xref="Sx5.SSx6.p4.5.m5.2.3.1.cmml">(</mo><mi id="Sx5.SSx6.p4.5.m5.1.1" xref="Sx5.SSx6.p4.5.m5.1.1.cmml">b</mi><mo id="Sx5.SSx6.p4.5.m5.2.3.2.2" xref="Sx5.SSx6.p4.5.m5.2.3.1.cmml">,</mo><mi id="Sx5.SSx6.p4.5.m5.2.2" xref="Sx5.SSx6.p4.5.m5.2.2.cmml">c</mi><mo stretchy="false" id="Sx5.SSx6.p4.5.m5.2.3.2.3" xref="Sx5.SSx6.p4.5.m5.2.3.1.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p4.5.m5.2b"><interval closure="open" id="Sx5.SSx6.p4.5.m5.2.3.1.cmml" xref="Sx5.SSx6.p4.5.m5.2.3.2"><ci id="Sx5.SSx6.p4.5.m5.1.1.cmml" xref="Sx5.SSx6.p4.5.m5.1.1">𝑏</ci><ci id="Sx5.SSx6.p4.5.m5.2.2.cmml" xref="Sx5.SSx6.p4.5.m5.2.2">𝑐</ci></interval></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p4.5.m5.2c">(b,c)</annotation></semantics></math> in the graph. We compared the clustering
coefficients to those found in random Erdős-Rényi graphs with the same
number of nodes and edge probability as the observed graph. We found
that the observed data shows significantly more clustering than these
random graphs for a wide range of choices of max-gap and <math id="Sx5.SSx6.p4.6.m6.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p4.6.m6.1a"><msub id="Sx5.SSx6.p4.6.m6.1.1" xref="Sx5.SSx6.p4.6.m6.1.1.cmml"><mi id="Sx5.SSx6.p4.6.m6.1.1.2" xref="Sx5.SSx6.p4.6.m6.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p4.6.m6.1.1.3" xref="Sx5.SSx6.p4.6.m6.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p4.6.m6.1b"><apply id="Sx5.SSx6.p4.6.m6.1.1.cmml" xref="Sx5.SSx6.p4.6.m6.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p4.6.m6.1.1.1.cmml" xref="Sx5.SSx6.p4.6.m6.1.1">subscript</csymbol><ci id="Sx5.SSx6.p4.6.m6.1.1.2.cmml" xref="Sx5.SSx6.p4.6.m6.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p4.6.m6.1.1.3.cmml" xref="Sx5.SSx6.p4.6.m6.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p4.6.m6.1c">m_{p}</annotation></semantics></math>. For
example, for max-gap 600, <math id="Sx5.SSx6.p4.7.m7.1" class="ltx_Math" alttext="m_{p}" display="inline"><semantics id="Sx5.SSx6.p4.7.m7.1a"><msub id="Sx5.SSx6.p4.7.m7.1.1" xref="Sx5.SSx6.p4.7.m7.1.1.cmml"><mi id="Sx5.SSx6.p4.7.m7.1.1.2" xref="Sx5.SSx6.p4.7.m7.1.1.2.cmml">m</mi><mi id="Sx5.SSx6.p4.7.m7.1.1.3" xref="Sx5.SSx6.p4.7.m7.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p4.7.m7.1b"><apply id="Sx5.SSx6.p4.7.m7.1.1.cmml" xref="Sx5.SSx6.p4.7.m7.1.1"><csymbol cd="ambiguous" id="Sx5.SSx6.p4.7.m7.1.1.1.cmml" xref="Sx5.SSx6.p4.7.m7.1.1">subscript</csymbol><ci id="Sx5.SSx6.p4.7.m7.1.1.2.cmml" xref="Sx5.SSx6.p4.7.m7.1.1.2">𝑚</ci><ci id="Sx5.SSx6.p4.7.m7.1.1.3.cmml" xref="Sx5.SSx6.p4.7.m7.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p4.7.m7.1c">m_{p}</annotation></semantics></math> =7, the average clustering coefficient is
0.19, while 10,000 random graphs had coefficients of 0.034 <math id="Sx5.SSx6.p4.8.m8.1" class="ltx_Math" alttext="\pm" display="inline"><semantics id="Sx5.SSx6.p4.8.m8.1a"><mo id="Sx5.SSx6.p4.8.m8.1.1" xref="Sx5.SSx6.p4.8.m8.1.1.cmml">±</mo><annotation-xml encoding="MathML-Content" id="Sx5.SSx6.p4.8.m8.1b"><csymbol cd="latexml" id="Sx5.SSx6.p4.8.m8.1.1.cmml" xref="Sx5.SSx6.p4.8.m8.1.1">plus-or-minus</csymbol></annotation-xml><annotation encoding="application/x-tex" id="Sx5.SSx6.p4.8.m8.1c">\pm</annotation></semantics></math> 0.042,
p=0.0039.</p>
</div>
</section>
<section id="Sx5.SSx7" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Age distribution of paralogous genes</h3>

<div id="Sx5.SSx7.p1" class="ltx_para">
<p id="Sx5.SSx7.p1.2" class="ltx_p">Because WGD events create many paralogs at the same time, they leave
characteristic peaks in the age distribution of paralogous genes. In
<em id="Sx5.SSx7.p1.2.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> the distribution shows peaks centered at 0.71 and 1.34
substitutions per synonymous site (K<sub id="Sx5.SSx7.p1.2.2" class="ltx_sub"><span id="Sx5.SSx7.p1.2.2.1" class="ltx_text ltx_font_italic">s</span></sub>), values within the
approximately linear response range of K<sub id="Sx5.SSx7.p1.2.3" class="ltx_sub"><span id="Sx5.SSx7.p1.2.3.1" class="ltx_text ltx_font_italic">s</span></sub> estimates to WGD age<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib33" title="" class="ltx_ref">33</a>]</cite> (Figure <a href="#Sx7.F9" title="Figure 9 ‣ Ka and Ks estimation for paralogs and T. tridentatus orthologs ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">9</span></a>).
For comparison, the synonymous site divergence between an Asian
horseshoe crab species <em id="Sx5.SSx7.p1.2.4" class="ltx_emph ltx_font_italic">Tachypleus tridentatus</em> and <em id="Sx5.SSx7.p1.2.5" class="ltx_emph ltx_font_italic">L. polyphemus</em>
has a mode of 0.35. The common ancestor of these species has been
estimated to have lived 114-154 MYA, coincident with the opening of
the Atlantic ocean<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib34" title="" class="ltx_ref">34</a>]</cite>, suggesting a WGD event
230-310 MYA, and possibly an older one 450-600 MYA.</p>
</div>
</section>
</section>
<section id="Sx6" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Discussion</h2>

<div id="Sx6.p1" class="ltx_para">
<p id="Sx6.p1.1" class="ltx_p">Our results demonstrate that a low cost, combined approach to whole
genome sequencing and genetic mapping can be used to efficiently
create a very high density genetic recombination map for a non-model
organism with a large genome. Because the approach uses genome-wide
sequencing, a large number of sequence markers can be anchored to the
map, allowing comparisons of genome organization at the chromosome
scale over very large evolutionary divergences. The identification of
chromosomal segments with significant gene composition homology to
each of the chordate ALGs shows that the predominance of fusion and
mixing of ancestral linkage groups previously observed in analyzed
ecdysozoan genomes<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib10" title="" class="ltx_ref">10</a>]</cite> is not ancestral to or
universal in the clade.</p>
</div>
<div id="Sx6.p2" class="ltx_para">
<p id="Sx6.p2.1" class="ltx_p">The map allows quantiative characterization of other features of
chromosome-scale organization, such as the correlation between local
recombination rate and polymorphism levels. Similar positive
correlations between local recombination rate and polymorphism level
have been observed in other metazoans including
humans<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib35" title="" class="ltx_ref">35</a>, <a href="#bib.bib36" title="" class="ltx_ref">36</a>, <a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>
and plants<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>, <a href="#bib.bib39" title="" class="ltx_ref">39</a>]</cite>. Future
comparisons to more closely related chelicerates will allow tests to
distinguish whether these rates are positively correlated with
inter-specific divergence, consistent with a neutral process of
correlated mutation and recombination
rates<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib40" title="" class="ltx_ref">40</a>]</cite>. Alternatively, the association
could be explained by hitchhiking and background selection<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib41" title="" class="ltx_ref">41</a>]</cite>.</p>
</div>
<div id="Sx6.p3" class="ltx_para">
<p id="Sx6.p3.1" class="ltx_p">The existence of duplicated hox and parahox clusters on four
different chromosomes is highly suggestive of multiple whole genome
duplication. Hox clusters have not been found in duplicate copies
except in vertebrates where they have been created by whole genome
duplication, and have only rarely been subsequently lost.</p>
</div>
<div id="Sx6.p4" class="ltx_para">
<p id="Sx6.p4.1" class="ltx_p">The enrichment of inter-chromosomal paralog pairs in segments of the
same ALG origin is consistent with their creation by duplication
(rather than fission), although because small-scale duplication is
biased toward local (tandem) duplication, fission of segments could
also leave behind an enrichment of paralogs. Such a mechanism,
however, would not create the observed organization of
paralogs, <em id="Sx6.p4.1.1" class="ltx_emph ltx_font_italic">i.e.</em> their clustering into “paralogons”. The fact
that these paralogons span a large portion of the map (59%) suggests
that it was a whole genome duplication, rather than segmental
duplications that have rise to the pattern.</p>
</div>
<div id="Sx6.p5" class="ltx_para">
<p id="Sx6.p5.1" class="ltx_p">The double-peaked shape of the distribution of synonymous site
divergence between pairs of paralogs, combined with the existence of
two small clusters of HOX genes in addition to the two complete HOX
clusters suggests that there may have been two rounds of whole genome
duplication in the horseshoe crab lineage.</p>
</div>
<div id="Sx6.p6" class="ltx_para">
<p id="Sx6.p6.1" class="ltx_p">WGDs preceded major species radiations in vertebrates, angiosperms and
teleost fish and the importance of their role in evolution is the
subject of long-running debate<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib2" title="" class="ltx_ref">2</a>, <a href="#bib.bib3" title="" class="ltx_ref">3</a>, <a href="#bib.bib4" title="" class="ltx_ref">4</a>, <a href="#bib.bib1" title="" class="ltx_ref">1</a>]</cite>.
The discovery of whole genome duplication in an invertebrate, and
during horseshoe crabs’ long and famously conservative evolutionary
history suggests that such events may have been more common than
previously assumed in metazoan evolution, and that while they may have
provided raw material for adaptive evolution in some cases, they are
not evolutionary drivers.</p>
</div>
</section>
<section id="Sx7" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Methods</h2>

<section id="Sx7.SSx1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Joint assembly and mapping (JAM) overview</h3>

<div id="Sx7.SSx1.p1" class="ltx_para">
<p id="Sx7.SSx1.p1.1" class="ltx_p">Barcoded genomic DNA libraries were created, pooled, and sequenced in
four lanes on the Illumina HiSeq2000 platform for a mating pair of <em id="Sx7.SSx1.p1.1.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> and 34 offspring (Methods).</p>
</div>
<div id="Sx7.SSx1.p2" class="ltx_para">
<p id="Sx7.SSx1.p2.6" class="ltx_p">The JAM method proceeds through three major phases: 1. The frequencies
of DNA sub-sequences of fixed length <math id="Sx7.SSx1.p2.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx1.p2.1.m1.1a"><mi id="Sx7.SSx1.p2.1.m1.1.1" xref="Sx7.SSx1.p2.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx1.p2.1.m1.1b"><ci id="Sx7.SSx1.p2.1.m1.1.1.cmml" xref="Sx7.SSx1.p2.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx1.p2.1.m1.1c">k</annotation></semantics></math> (<math id="Sx7.SSx1.p2.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx1.p2.2.m2.1a"><mi id="Sx7.SSx1.p2.2.m2.1.1" xref="Sx7.SSx1.p2.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx1.p2.2.m2.1b"><ci id="Sx7.SSx1.p2.2.m2.1.1.cmml" xref="Sx7.SSx1.p2.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx1.p2.2.m2.1c">k</annotation></semantics></math>-mers) are profiled to
characterize the quality, uniqueness, polymorphisms and repetition in
genomic reads, using software we developed building on work from the
Atlas assembler<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib42" title="" class="ltx_ref">42</a>]</cite>. Allelic pairs of <math id="Sx7.SSx1.p2.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx1.p2.3.m3.1a"><mi id="Sx7.SSx1.p2.3.m3.1.1" xref="Sx7.SSx1.p2.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx1.p2.3.m3.1b"><ci id="Sx7.SSx1.p2.3.m3.1.1.cmml" xref="Sx7.SSx1.p2.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx1.p2.3.m3.1c">k</annotation></semantics></math>-mers representing alternate forms of single nucleotide polymorphisms (SNPs)
are identified and tracked through the subsequent steps. 2.
Contigs are assembled on a graph of unique <math id="Sx7.SSx1.p2.4.m4.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx1.p2.4.m4.1a"><mi id="Sx7.SSx1.p2.4.m4.1.1" xref="Sx7.SSx1.p2.4.m4.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx1.p2.4.m4.1b"><ci id="Sx7.SSx1.p2.4.m4.1.1.cmml" xref="Sx7.SSx1.p2.4.m4.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx1.p2.4.m4.1c">k</annotation></semantics></math>-mers  and paired SNP <math id="Sx7.SSx1.p2.5.m5.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx1.p2.5.m5.1a"><mi id="Sx7.SSx1.p2.5.m5.1.1" xref="Sx7.SSx1.p2.5.m5.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx1.p2.5.m5.1b"><ci id="Sx7.SSx1.p2.5.m5.1.1.cmml" xref="Sx7.SSx1.p2.5.m5.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx1.p2.5.m5.1c">k</annotation></semantics></math>-mers  sampled to reduce memory usage, then ordered and oriented
using the Bambus
scaffolder<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib26" title="" class="ltx_ref">26</a>, <a href="#bib.bib43" title="" class="ltx_ref">43</a>]</cite>. Each
multi-SNP scaffold is treated as a single marker for the linkage
mapping steps. 3. The paired SNP <math id="Sx7.SSx1.p2.6.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx1.p2.6.m6.1a"><mi id="Sx7.SSx1.p2.6.m6.1.1" xref="Sx7.SSx1.p2.6.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx1.p2.6.m6.1b"><ci id="Sx7.SSx1.p2.6.m6.1.1.cmml" xref="Sx7.SSx1.p2.6.m6.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx1.p2.6.m6.1c">k</annotation></semantics></math>-mers  (SNPmers) in each scaffold are combined with the read,
mate-pair, and parent- or offspring-library associations of their
alleles for haplotype phasing and construction of a high density
genetic linkage map.</p>
</div>
</section>
<section id="Sx7.SSx2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Sampling and sequencing</h3>

<div id="Sx7.SSx2.p1" class="ltx_para">
<p id="Sx7.SSx2.p1.1" class="ltx_p">Tissue samples were collected from the third walking leg of a
monandrous pair of horseshoe crabs nesting at high tide on the beach
at Seahorse Key, an island along the west coast of north Florida, on
27 March 2010. The eggs laid by this pair were collected 6 hr later
when the tide had receded and reared in plastic dishes as previously
described<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite>. Trilobite larvae hatched from
the eggs 4 weeks later. Tissue samples and larvae were preserved in
RNALater. Genomic DNA purification and library construction were carried out
using Qiagen DNAEasy, Illumina TruSeq and Nextera kits, following
manufacturers’ protocols. Barcoded samples were pooled and sequenced
on on the Illumina HiSeq2000 platform.</p>
</div>
<div id="Sx7.SSx2.p2" class="ltx_para">
<p id="Sx7.SSx2.p2.1" class="ltx_p">Limulus larvae were processed as follows; each larva, suspended in 100 <math id="Sx7.SSx2.p2.1.m1.1" class="ltx_Math" alttext="\mu\text{L}" display="inline"><semantics id="Sx7.SSx2.p2.1.m1.1a"><mrow id="Sx7.SSx2.p2.1.m1.1.1" xref="Sx7.SSx2.p2.1.m1.1.1.cmml"><mi id="Sx7.SSx2.p2.1.m1.1.1.2" xref="Sx7.SSx2.p2.1.m1.1.1.2.cmml">μ</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx2.p2.1.m1.1.1.1" xref="Sx7.SSx2.p2.1.m1.1.1.1.cmml">​</mo><mtext id="Sx7.SSx2.p2.1.m1.1.1.3" xref="Sx7.SSx2.p2.1.m1.1.1.3a.cmml">L</mtext></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx2.p2.1.m1.1b"><apply id="Sx7.SSx2.p2.1.m1.1.1.cmml" xref="Sx7.SSx2.p2.1.m1.1.1"><times id="Sx7.SSx2.p2.1.m1.1.1.1.cmml" xref="Sx7.SSx2.p2.1.m1.1.1.1"></times><ci id="Sx7.SSx2.p2.1.m1.1.1.2.cmml" xref="Sx7.SSx2.p2.1.m1.1.1.2">𝜇</ci><ci id="Sx7.SSx2.p2.1.m1.1.1.3a.cmml" xref="Sx7.SSx2.p2.1.m1.1.1.3"><mtext id="Sx7.SSx2.p2.1.m1.1.1.3.cmml" xref="Sx7.SSx2.p2.1.m1.1.1.3">L</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx2.p2.1.m1.1c">\mu\text{L}</annotation></semantics></math> of RNAlater and
stored at -80 °C in a 1.5 mL Eppendorf tube, was thawed on ice, after which RNAlater was
removed. DNA was extracted using the Qiagen DNAEasy kit per manufacturer’s protocols.
DNA was quantified using picogreen DNA quantitation kit. To prepare TruSeq libraries,
DNA was first purified another time using zymo genomic DNA clean columns per manufacturer’s protocols.
Adult <em id="Sx7.SSx2.p2.1.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> DNA was prepared as above, but using claw tissue rather than whole larvae.
All DNA extracts were tested by gel electrophoresis to ensure DNA was not degraded.
TruSeq libraries were prepared at University of Georgia’s Georgia Genomics Facility.
1-5 micrograms of sample DNA was subjected to fragmentation using Covaris sonicator.
Fragmented DNA was then used for library construction using Illumina TruSeq library prep kits.
Libraries were pooled together in equimolar amounts (for 10 larvae) and used for sequencing.
For samples 11-34, Library prep was switched from TruSeq to Nextera kits.
Nextera library preparation was performed according to manufacturer’s protocol.
The Nextera library product was quantified by picogreen, and fragment size distribution was checked by using Lonza flash gel,
to ensure that fragment size distribution was between 300-1000 bp.
Sample libraries were pooled in equimolar concentrations and sent for sequencing.
Library pools were sequenced on the Illumina HiSeq2000 platform at Medical College of Wisconsin Sequencing Service Core Facility.</p>
</div>
</section>
<section id="Sx7.SSx3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<math id="Sx7.SSx3.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.1.m1.1b"><mi id="Sx7.SSx3.1.m1.1.1" xref="Sx7.SSx3.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.1.m1.1c"><ci id="Sx7.SSx3.1.m1.1.1.cmml" xref="Sx7.SSx3.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.1.m1.1d">k</annotation></semantics></math>-mer  decomposition</h3>

<div id="Sx7.SSx3.p1" class="ltx_para">
<p id="Sx7.SSx3.p1.8" class="ltx_p">We determine a lower bound on the <math id="Sx7.SSx3.p1.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p1.1.m1.1a"><mi id="Sx7.SSx3.p1.1.m1.1.1" xref="Sx7.SSx3.p1.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.1.m1.1b"><ci id="Sx7.SSx3.p1.1.m1.1.1.cmml" xref="Sx7.SSx3.p1.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.1.m1.1c">k</annotation></semantics></math>-mer  size long enough for a given
expectation of uniquenes in a random genome. While increasing <em id="Sx7.SSx3.p1.8.1" class="ltx_emph ltx_font_italic">k</em>
reduces the rate of coincidentally repeated <math id="Sx7.SSx3.p1.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p1.2.m2.1a"><mi id="Sx7.SSx3.p1.2.m2.1.1" xref="Sx7.SSx3.p1.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.2.m2.1b"><ci id="Sx7.SSx3.p1.2.m2.1.1.cmml" xref="Sx7.SSx3.p1.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.2.m2.1c">k</annotation></semantics></math>-mers, it also reduces
the effective depth of coverage due to untrimmed errors and edge
effects at read ends — and increases the cases of multiple SNPs per
<math id="Sx7.SSx3.p1.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p1.3.m3.1a"><mi id="Sx7.SSx3.p1.3.m3.1.1" xref="Sx7.SSx3.p1.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.3.m3.1b"><ci id="Sx7.SSx3.p1.3.m3.1.1.cmml" xref="Sx7.SSx3.p1.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.3.m3.1c">k</annotation></semantics></math>-mer  locus, which are not tracked in our current software implementation. We
can approximately model a genome-scale string <em id="Sx7.SSx3.p1.8.2" class="ltx_emph ltx_font_italic">G</em> of random nucleotides as
<math id="Sx7.SSx3.p1.4.m4.1" class="ltx_Math" alttext="G" display="inline"><semantics id="Sx7.SSx3.p1.4.m4.1a"><mi id="Sx7.SSx3.p1.4.m4.1.1" xref="Sx7.SSx3.p1.4.m4.1.1.cmml">G</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.4.m4.1b"><ci id="Sx7.SSx3.p1.4.m4.1.1.cmml" xref="Sx7.SSx3.p1.4.m4.1.1">𝐺</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.4.m4.1c">G</annotation></semantics></math> samples taken with uniform probability from the space of all <math id="Sx7.SSx3.p1.5.m5.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p1.5.m5.1a"><mi id="Sx7.SSx3.p1.5.m5.1.1" xref="Sx7.SSx3.p1.5.m5.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.5.m5.1b"><ci id="Sx7.SSx3.p1.5.m5.1.1.cmml" xref="Sx7.SSx3.p1.5.m5.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.5.m5.1c">k</annotation></semantics></math>-mers (of size <math id="Sx7.SSx3.p1.6.m6.1" class="ltx_Math" alttext="4^{k}/2" display="inline"><semantics id="Sx7.SSx3.p1.6.m6.1a"><mrow id="Sx7.SSx3.p1.6.m6.1.1" xref="Sx7.SSx3.p1.6.m6.1.1.cmml"><msup id="Sx7.SSx3.p1.6.m6.1.1.2" xref="Sx7.SSx3.p1.6.m6.1.1.2.cmml"><mn id="Sx7.SSx3.p1.6.m6.1.1.2.2" xref="Sx7.SSx3.p1.6.m6.1.1.2.2.cmml">4</mn><mi id="Sx7.SSx3.p1.6.m6.1.1.2.3" xref="Sx7.SSx3.p1.6.m6.1.1.2.3.cmml">k</mi></msup><mo id="Sx7.SSx3.p1.6.m6.1.1.1" xref="Sx7.SSx3.p1.6.m6.1.1.1.cmml">/</mo><mn id="Sx7.SSx3.p1.6.m6.1.1.3" xref="Sx7.SSx3.p1.6.m6.1.1.3.cmml">2</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.6.m6.1b"><apply id="Sx7.SSx3.p1.6.m6.1.1.cmml" xref="Sx7.SSx3.p1.6.m6.1.1"><divide id="Sx7.SSx3.p1.6.m6.1.1.1.cmml" xref="Sx7.SSx3.p1.6.m6.1.1.1"></divide><apply id="Sx7.SSx3.p1.6.m6.1.1.2.cmml" xref="Sx7.SSx3.p1.6.m6.1.1.2"><csymbol cd="ambiguous" id="Sx7.SSx3.p1.6.m6.1.1.2.1.cmml" xref="Sx7.SSx3.p1.6.m6.1.1.2">superscript</csymbol><cn type="integer" id="Sx7.SSx3.p1.6.m6.1.1.2.2.cmml" xref="Sx7.SSx3.p1.6.m6.1.1.2.2">4</cn><ci id="Sx7.SSx3.p1.6.m6.1.1.2.3.cmml" xref="Sx7.SSx3.p1.6.m6.1.1.2.3">𝑘</ci></apply><cn type="integer" id="Sx7.SSx3.p1.6.m6.1.1.3.cmml" xref="Sx7.SSx3.p1.6.m6.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.6.m6.1c">4^{k}/2</annotation></semantics></math> for odd <math id="Sx7.SSx3.p1.7.m7.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p1.7.m7.1a"><mi id="Sx7.SSx3.p1.7.m7.1.1" xref="Sx7.SSx3.p1.7.m7.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.7.m7.1b"><ci id="Sx7.SSx3.p1.7.m7.1.1.cmml" xref="Sx7.SSx3.p1.7.m7.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.7.m7.1c">k</annotation></semantics></math>-mers  treating reverse-complements as same; slightly
more for even <em id="Sx7.SSx3.p1.8.3" class="ltx_emph ltx_font_italic">k</em>). The Poisson distribution then gives the probability that a
location in <em id="Sx7.SSx3.p1.8.4" class="ltx_emph ltx_font_italic">G</em> has its own, unique <math id="Sx7.SSx3.p1.8.m8.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p1.8.m8.1a"><mi id="Sx7.SSx3.p1.8.m8.1.1" xref="Sx7.SSx3.p1.8.m8.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.8.m8.1b"><ci id="Sx7.SSx3.p1.8.m8.1.1.cmml" xref="Sx7.SSx3.p1.8.m8.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.8.m8.1c">k</annotation></semantics></math>-mer  (shared with no other location) as</p>
<blockquote id="Sx7.SSx3.p1.10" class="ltx_quote">
<p id="Sx7.SSx3.p1.10.2" class="ltx_p"><math id="Sx7.SSx3.p1.9.1.m1.1" class="ltx_Math" alttext="e^{-\lambda}" display="inline"><semantics id="Sx7.SSx3.p1.9.1.m1.1a"><msup id="Sx7.SSx3.p1.9.1.m1.1.1" xref="Sx7.SSx3.p1.9.1.m1.1.1.cmml"><mi id="Sx7.SSx3.p1.9.1.m1.1.1.2" xref="Sx7.SSx3.p1.9.1.m1.1.1.2.cmml">e</mi><mrow id="Sx7.SSx3.p1.9.1.m1.1.1.3" xref="Sx7.SSx3.p1.9.1.m1.1.1.3.cmml"><mo id="Sx7.SSx3.p1.9.1.m1.1.1.3a" xref="Sx7.SSx3.p1.9.1.m1.1.1.3.cmml">−</mo><mi id="Sx7.SSx3.p1.9.1.m1.1.1.3.2" xref="Sx7.SSx3.p1.9.1.m1.1.1.3.2.cmml">λ</mi></mrow></msup><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.9.1.m1.1b"><apply id="Sx7.SSx3.p1.9.1.m1.1.1.cmml" xref="Sx7.SSx3.p1.9.1.m1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx3.p1.9.1.m1.1.1.1.cmml" xref="Sx7.SSx3.p1.9.1.m1.1.1">superscript</csymbol><ci id="Sx7.SSx3.p1.9.1.m1.1.1.2.cmml" xref="Sx7.SSx3.p1.9.1.m1.1.1.2">𝑒</ci><apply id="Sx7.SSx3.p1.9.1.m1.1.1.3.cmml" xref="Sx7.SSx3.p1.9.1.m1.1.1.3"><minus id="Sx7.SSx3.p1.9.1.m1.1.1.3.1.cmml" xref="Sx7.SSx3.p1.9.1.m1.1.1.3"></minus><ci id="Sx7.SSx3.p1.9.1.m1.1.1.3.2.cmml" xref="Sx7.SSx3.p1.9.1.m1.1.1.3.2">𝜆</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.9.1.m1.1c">e^{-\lambda}</annotation></semantics></math>, where <math id="Sx7.SSx3.p1.10.2.m2.1" class="ltx_Math" alttext="\lambda=G/(4k/2)=2G/4k" display="inline"><semantics id="Sx7.SSx3.p1.10.2.m2.1a"><mrow id="Sx7.SSx3.p1.10.2.m2.1.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.cmml"><mi id="Sx7.SSx3.p1.10.2.m2.1.1.3" xref="Sx7.SSx3.p1.10.2.m2.1.1.3.cmml">λ</mi><mo id="Sx7.SSx3.p1.10.2.m2.1.1.4" xref="Sx7.SSx3.p1.10.2.m2.1.1.4.cmml">=</mo><mrow id="Sx7.SSx3.p1.10.2.m2.1.1.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.cmml"><mi id="Sx7.SSx3.p1.10.2.m2.1.1.1.3" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.3.cmml">G</mi><mo id="Sx7.SSx3.p1.10.2.m2.1.1.1.2" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.2.cmml">/</mo><mrow id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.cmml"><mo stretchy="false" id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.2" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.cmml">(</mo><mrow id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.cmml"><mrow id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.cmml"><mn id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.2" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.2.cmml">4</mn><mo lspace="0em" rspace="0em" id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.1.cmml">​</mo><mi id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.3" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.3.cmml">k</mi></mrow><mo id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.1.cmml">/</mo><mn id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.3" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.3.cmml">2</mn></mrow><mo stretchy="false" id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.3" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="Sx7.SSx3.p1.10.2.m2.1.1.5" xref="Sx7.SSx3.p1.10.2.m2.1.1.5.cmml">=</mo><mrow id="Sx7.SSx3.p1.10.2.m2.1.1.6" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.cmml"><mrow id="Sx7.SSx3.p1.10.2.m2.1.1.6.2" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.cmml"><mrow id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.cmml"><mn id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.2" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.2.cmml">2</mn><mo lspace="0em" rspace="0em" id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.1.cmml">​</mo><mi id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.3" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.3.cmml">G</mi></mrow><mo id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.1.cmml">/</mo><mn id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.3" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.3.cmml">4</mn></mrow><mo lspace="0em" rspace="0em" id="Sx7.SSx3.p1.10.2.m2.1.1.6.1" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.1.cmml">​</mo><mi id="Sx7.SSx3.p1.10.2.m2.1.1.6.3" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.3.cmml">k</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.10.2.m2.1b"><apply id="Sx7.SSx3.p1.10.2.m2.1.1.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1"><and id="Sx7.SSx3.p1.10.2.m2.1.1a.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1"></and><apply id="Sx7.SSx3.p1.10.2.m2.1.1b.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1"><eq id="Sx7.SSx3.p1.10.2.m2.1.1.4.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.4"></eq><ci id="Sx7.SSx3.p1.10.2.m2.1.1.3.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.3">𝜆</ci><apply id="Sx7.SSx3.p1.10.2.m2.1.1.1.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1"><divide id="Sx7.SSx3.p1.10.2.m2.1.1.1.2.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.2"></divide><ci id="Sx7.SSx3.p1.10.2.m2.1.1.1.3.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.3">𝐺</ci><apply id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1"><divide id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.1.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.1"></divide><apply id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2"><times id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.1.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.1"></times><cn type="integer" id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.2.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.2">4</cn><ci id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.3.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.2.3">𝑘</ci></apply><cn type="integer" id="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.3.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.1.1.1.1.3">2</cn></apply></apply></apply><apply id="Sx7.SSx3.p1.10.2.m2.1.1c.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1"><eq id="Sx7.SSx3.p1.10.2.m2.1.1.5.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.5"></eq><share href="#Sx7.SSx3.p1.10.2.m2.1.1.1.cmml" id="Sx7.SSx3.p1.10.2.m2.1.1d.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1"></share><apply id="Sx7.SSx3.p1.10.2.m2.1.1.6.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6"><times id="Sx7.SSx3.p1.10.2.m2.1.1.6.1.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.1"></times><apply id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2"><divide id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.1.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.1"></divide><apply id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2"><times id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.1.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.1"></times><cn type="integer" id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.2.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.2">2</cn><ci id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.3.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.2.3">𝐺</ci></apply><cn type="integer" id="Sx7.SSx3.p1.10.2.m2.1.1.6.2.3.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.2.3">4</cn></apply><ci id="Sx7.SSx3.p1.10.2.m2.1.1.6.3.cmml" xref="Sx7.SSx3.p1.10.2.m2.1.1.6.3">𝑘</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.10.2.m2.1c">\lambda=G/(4k/2)=2G/4k</annotation></semantics></math>.</p>
</blockquote>
<p id="Sx7.SSx3.p1.14" class="ltx_p">The probability of a location sharing its <em id="Sx7.SSx3.p1.14.1" class="ltx_emph ltx_font_italic">kmer</em> is then <math id="Sx7.SSx3.p1.11.m1.1" class="ltx_Math" alttext="1-e^{-\lambda}" display="inline"><semantics id="Sx7.SSx3.p1.11.m1.1a"><mrow id="Sx7.SSx3.p1.11.m1.1.1" xref="Sx7.SSx3.p1.11.m1.1.1.cmml"><mn id="Sx7.SSx3.p1.11.m1.1.1.2" xref="Sx7.SSx3.p1.11.m1.1.1.2.cmml">1</mn><mo id="Sx7.SSx3.p1.11.m1.1.1.1" xref="Sx7.SSx3.p1.11.m1.1.1.1.cmml">−</mo><msup id="Sx7.SSx3.p1.11.m1.1.1.3" xref="Sx7.SSx3.p1.11.m1.1.1.3.cmml"><mi id="Sx7.SSx3.p1.11.m1.1.1.3.2" xref="Sx7.SSx3.p1.11.m1.1.1.3.2.cmml">e</mi><mrow id="Sx7.SSx3.p1.11.m1.1.1.3.3" xref="Sx7.SSx3.p1.11.m1.1.1.3.3.cmml"><mo id="Sx7.SSx3.p1.11.m1.1.1.3.3a" xref="Sx7.SSx3.p1.11.m1.1.1.3.3.cmml">−</mo><mi id="Sx7.SSx3.p1.11.m1.1.1.3.3.2" xref="Sx7.SSx3.p1.11.m1.1.1.3.3.2.cmml">λ</mi></mrow></msup></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.11.m1.1b"><apply id="Sx7.SSx3.p1.11.m1.1.1.cmml" xref="Sx7.SSx3.p1.11.m1.1.1"><minus id="Sx7.SSx3.p1.11.m1.1.1.1.cmml" xref="Sx7.SSx3.p1.11.m1.1.1.1"></minus><cn type="integer" id="Sx7.SSx3.p1.11.m1.1.1.2.cmml" xref="Sx7.SSx3.p1.11.m1.1.1.2">1</cn><apply id="Sx7.SSx3.p1.11.m1.1.1.3.cmml" xref="Sx7.SSx3.p1.11.m1.1.1.3"><csymbol cd="ambiguous" id="Sx7.SSx3.p1.11.m1.1.1.3.1.cmml" xref="Sx7.SSx3.p1.11.m1.1.1.3">superscript</csymbol><ci id="Sx7.SSx3.p1.11.m1.1.1.3.2.cmml" xref="Sx7.SSx3.p1.11.m1.1.1.3.2">𝑒</ci><apply id="Sx7.SSx3.p1.11.m1.1.1.3.3.cmml" xref="Sx7.SSx3.p1.11.m1.1.1.3.3"><minus id="Sx7.SSx3.p1.11.m1.1.1.3.3.1.cmml" xref="Sx7.SSx3.p1.11.m1.1.1.3.3"></minus><ci id="Sx7.SSx3.p1.11.m1.1.1.3.3.2.cmml" xref="Sx7.SSx3.p1.11.m1.1.1.3.3.2">𝜆</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.11.m1.1c">1-e^{-\lambda}</annotation></semantics></math>; thus, to
limit the maximum rate <em id="Sx7.SSx3.p1.14.2" class="ltx_emph ltx_font_italic">R</em> of <em id="Sx7.SSx3.p1.14.3" class="ltx_emph ltx_font_italic">G</em>-locations sharing <math id="Sx7.SSx3.p1.12.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p1.12.m2.1a"><mi id="Sx7.SSx3.p1.12.m2.1.1" xref="Sx7.SSx3.p1.12.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.12.m2.1b"><ci id="Sx7.SSx3.p1.12.m2.1.1.cmml" xref="Sx7.SSx3.p1.12.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.12.m2.1c">k</annotation></semantics></math>-mers, we
require <math id="Sx7.SSx3.p1.13.m3.2" class="ltx_Math" alttext="k\geq\lceil\log_{4}(-2G/\ln(1-R))\rceil" display="inline"><semantics id="Sx7.SSx3.p1.13.m3.2a"><mrow id="Sx7.SSx3.p1.13.m3.2.2" xref="Sx7.SSx3.p1.13.m3.2.2.cmml"><mi id="Sx7.SSx3.p1.13.m3.2.2.3" xref="Sx7.SSx3.p1.13.m3.2.2.3.cmml">k</mi><mo id="Sx7.SSx3.p1.13.m3.2.2.2" xref="Sx7.SSx3.p1.13.m3.2.2.2.cmml">≥</mo><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.2.cmml"><mo stretchy="false" id="Sx7.SSx3.p1.13.m3.2.2.1.1.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.2.1.cmml">⌈</mo><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.3.cmml"><msub id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.cmml"><mi id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.2.cmml">log</mi><mn id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.3" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.3.cmml">4</mn></msub><mo id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2a" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.3.cmml">⁡</mo><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.3.cmml"><mo stretchy="false" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.3.cmml">(</mo><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.cmml"><mo id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1a" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.cmml">−</mo><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.cmml"><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.cmml"><mn id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.2.cmml">2</mn><mo lspace="0em" rspace="0em" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.3" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.3.cmml">G</mi></mrow><mo id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.2.cmml">/</mo><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.2.cmml"><mi id="Sx7.SSx3.p1.13.m3.1.1" xref="Sx7.SSx3.p1.13.m3.1.1.cmml">ln</mi><mo id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1a" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.2.cmml">⁡</mo><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.2.cmml"><mo stretchy="false" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.2.cmml">(</mo><mrow id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.cmml"><mn id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.2" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.2.cmml">1</mn><mo id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.1" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.1.cmml">−</mo><mi id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.3" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.3.cmml">R</mi></mrow><mo stretchy="false" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.3" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.2.cmml">)</mo></mrow></mrow></mrow></mrow><mo stretchy="false" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.3" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.3.cmml">)</mo></mrow></mrow><mo stretchy="false" id="Sx7.SSx3.p1.13.m3.2.2.1.1.3" xref="Sx7.SSx3.p1.13.m3.2.2.1.2.1.cmml">⌉</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.13.m3.2b"><apply id="Sx7.SSx3.p1.13.m3.2.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2"><geq id="Sx7.SSx3.p1.13.m3.2.2.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.2"></geq><ci id="Sx7.SSx3.p1.13.m3.2.2.3.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.3">𝑘</ci><apply id="Sx7.SSx3.p1.13.m3.2.2.1.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1"><ceiling id="Sx7.SSx3.p1.13.m3.2.2.1.2.1.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.2"></ceiling><apply id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.3.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2"><apply id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.1.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1">subscript</csymbol><log id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.2"></log><cn type="integer" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.3.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.1.1.3">4</cn></apply><apply id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1"><minus id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1"></minus><apply id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1"><divide id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.2"></divide><apply id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3"><times id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.1.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.1"></times><cn type="integer" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.2">2</cn><ci id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.3.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.3.3">𝐺</ci></apply><apply id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1"><ln id="Sx7.SSx3.p1.13.m3.1.1.cmml" xref="Sx7.SSx3.p1.13.m3.1.1"></ln><apply id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1"><minus id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.1.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.1"></minus><cn type="integer" id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.2.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.2">1</cn><ci id="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.3.cmml" xref="Sx7.SSx3.p1.13.m3.2.2.1.1.1.2.2.1.1.1.1.1.1.3">𝑅</ci></apply></apply></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.13.m3.2c">k\geq\lceil\log_{4}(-2G/\ln(1-R))\rceil</annotation></semantics></math>. For
example, for a mammalian-scale genome of approximately 3 billion bases, and <em id="Sx7.SSx3.p1.14.4" class="ltx_emph ltx_font_italic">R</em> = 0.1%, we choose <math id="Sx7.SSx3.p1.14.m4.1" class="ltx_Math" alttext="k\geq 22" display="inline"><semantics id="Sx7.SSx3.p1.14.m4.1a"><mrow id="Sx7.SSx3.p1.14.m4.1.1" xref="Sx7.SSx3.p1.14.m4.1.1.cmml"><mi id="Sx7.SSx3.p1.14.m4.1.1.2" xref="Sx7.SSx3.p1.14.m4.1.1.2.cmml">k</mi><mo id="Sx7.SSx3.p1.14.m4.1.1.1" xref="Sx7.SSx3.p1.14.m4.1.1.1.cmml">≥</mo><mn id="Sx7.SSx3.p1.14.m4.1.1.3" xref="Sx7.SSx3.p1.14.m4.1.1.3.cmml">22</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p1.14.m4.1b"><apply id="Sx7.SSx3.p1.14.m4.1.1.cmml" xref="Sx7.SSx3.p1.14.m4.1.1"><geq id="Sx7.SSx3.p1.14.m4.1.1.1.cmml" xref="Sx7.SSx3.p1.14.m4.1.1.1"></geq><ci id="Sx7.SSx3.p1.14.m4.1.1.2.cmml" xref="Sx7.SSx3.p1.14.m4.1.1.2">𝑘</ci><cn type="integer" id="Sx7.SSx3.p1.14.m4.1.1.3.cmml" xref="Sx7.SSx3.p1.14.m4.1.1.3">22</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p1.14.m4.1c">k\geq 22</annotation></semantics></math>. For <em id="Sx7.SSx3.p1.14.5" class="ltx_emph ltx_font_italic">Limulus polyphemus</em>, the Animal Genome Size Database<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib25" title="" class="ltx_ref">25</a>]</cite> reports an estimated haploid genome size of 2.80 pg and, as each picogram represents almost a billion nucleotide base-pairs of DNA, the mammalian-scale choice of <em id="Sx7.SSx3.p1.14.6" class="ltx_emph ltx_font_italic">k</em> applies.<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib45" title="" class="ltx_ref">45</a>, <a href="#bib.bib19" title="" class="ltx_ref">19</a>]</cite></p>
</div>
<div id="Sx7.SSx3.p2" class="ltx_para">
<p id="Sx7.SSx3.p2.13" class="ltx_p">This lower bound ignores chemical and biological sequence biases, so
selecting <math id="Sx7.SSx3.p2.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p2.1.m1.1a"><mi id="Sx7.SSx3.p2.1.m1.1.1" xref="Sx7.SSx3.p2.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.1.m1.1b"><ci id="Sx7.SSx3.p2.1.m1.1.1.cmml" xref="Sx7.SSx3.p2.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.1.m1.1c">k</annotation></semantics></math> for a real genome project requires attention to error
rates, repeats tandem and interspersed, and genome size, all known
vaguely, if at all, before sequencing. Studying the <math id="Sx7.SSx3.p2.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p2.2.m2.1a"><mi id="Sx7.SSx3.p2.2.m2.1.1" xref="Sx7.SSx3.p2.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.2.m2.1b"><ci id="Sx7.SSx3.p2.2.m2.1.1.cmml" xref="Sx7.SSx3.p2.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.2.m2.1c">k</annotation></semantics></math>-mer distributions after sequencing can clarify these genomic properties as
we select <em id="Sx7.SSx3.p2.13.1" class="ltx_emph ltx_font_italic">k</em> to maximize the net yield of candidate <math id="Sx7.SSx3.p2.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p2.3.m3.1a"><mi id="Sx7.SSx3.p2.3.m3.1.1" xref="Sx7.SSx3.p2.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.3.m3.1b"><ci id="Sx7.SSx3.p2.3.m3.1.1.cmml" xref="Sx7.SSx3.p2.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.3.m3.1c">k</annotation></semantics></math>-mer  tags,
between errors and with at most one SNP location, in sequencing reads.
We convert Illumina/Solexa FASTQ format (paying attention to the
different quality encodings of the software versions) into FASTA
format<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib46" title="" class="ltx_ref">46</a>]</cite>, masking (replacing with ‘N’) any base
with Phred-scale<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib47" title="" class="ltx_ref">47</a>]</cite> quality below 20, and
soft-masking (representing in lower case) other bases with quality
below 30. For initial trimming experiments, we vary these quality
thresholds as indicated below. We store <math id="Sx7.SSx3.p2.4.m4.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p2.4.m4.1a"><mi id="Sx7.SSx3.p2.4.m4.1.1" xref="Sx7.SSx3.p2.4.m4.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.4.m4.1b"><ci id="Sx7.SSx3.p2.4.m4.1.1.cmml" xref="Sx7.SSx3.p2.4.m4.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.4.m4.1c">k</annotation></semantics></math>-mers  in hash tables with
open addressing<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib48" title="" class="ltx_ref">48</a>]</cite>, supporting odd <math id="Sx7.SSx3.p2.5.m5.1" class="ltx_Math" alttext="k&lt;=31" display="inline"><semantics id="Sx7.SSx3.p2.5.m5.1a"><mrow id="Sx7.SSx3.p2.5.m5.1.1" xref="Sx7.SSx3.p2.5.m5.1.1.cmml"><mi id="Sx7.SSx3.p2.5.m5.1.1.2" xref="Sx7.SSx3.p2.5.m5.1.1.2.cmml">k</mi><mo id="Sx7.SSx3.p2.5.m5.1.1.1" xref="Sx7.SSx3.p2.5.m5.1.1.1.cmml">&lt;=</mo><mn id="Sx7.SSx3.p2.5.m5.1.1.3" xref="Sx7.SSx3.p2.5.m5.1.1.3.cmml">31</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.5.m5.1b"><apply id="Sx7.SSx3.p2.5.m5.1.1.cmml" xref="Sx7.SSx3.p2.5.m5.1.1"><leq id="Sx7.SSx3.p2.5.m5.1.1.1.cmml" xref="Sx7.SSx3.p2.5.m5.1.1.1"></leq><ci id="Sx7.SSx3.p2.5.m5.1.1.2.cmml" xref="Sx7.SSx3.p2.5.m5.1.1.2">𝑘</ci><cn type="integer" id="Sx7.SSx3.p2.5.m5.1.1.3.cmml" xref="Sx7.SSx3.p2.5.m5.1.1.3">31</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.5.m5.1c">k&lt;=31</annotation></semantics></math>. We tally for each kmer a bit vector for presence or absence in up
to 64 libraries, and an overall count of occurrences in all libraries
(count limited to <math id="Sx7.SSx3.p2.6.m6.1" class="ltx_Math" alttext="64-2k" display="inline"><semantics id="Sx7.SSx3.p2.6.m6.1a"><mrow id="Sx7.SSx3.p2.6.m6.1.1" xref="Sx7.SSx3.p2.6.m6.1.1.cmml"><mn id="Sx7.SSx3.p2.6.m6.1.1.2" xref="Sx7.SSx3.p2.6.m6.1.1.2.cmml">64</mn><mo id="Sx7.SSx3.p2.6.m6.1.1.1" xref="Sx7.SSx3.p2.6.m6.1.1.1.cmml">−</mo><mrow id="Sx7.SSx3.p2.6.m6.1.1.3" xref="Sx7.SSx3.p2.6.m6.1.1.3.cmml"><mn id="Sx7.SSx3.p2.6.m6.1.1.3.2" xref="Sx7.SSx3.p2.6.m6.1.1.3.2.cmml">2</mn><mo lspace="0em" rspace="0em" id="Sx7.SSx3.p2.6.m6.1.1.3.1" xref="Sx7.SSx3.p2.6.m6.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx3.p2.6.m6.1.1.3.3" xref="Sx7.SSx3.p2.6.m6.1.1.3.3.cmml">k</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.6.m6.1b"><apply id="Sx7.SSx3.p2.6.m6.1.1.cmml" xref="Sx7.SSx3.p2.6.m6.1.1"><minus id="Sx7.SSx3.p2.6.m6.1.1.1.cmml" xref="Sx7.SSx3.p2.6.m6.1.1.1"></minus><cn type="integer" id="Sx7.SSx3.p2.6.m6.1.1.2.cmml" xref="Sx7.SSx3.p2.6.m6.1.1.2">64</cn><apply id="Sx7.SSx3.p2.6.m6.1.1.3.cmml" xref="Sx7.SSx3.p2.6.m6.1.1.3"><times id="Sx7.SSx3.p2.6.m6.1.1.3.1.cmml" xref="Sx7.SSx3.p2.6.m6.1.1.3.1"></times><cn type="integer" id="Sx7.SSx3.p2.6.m6.1.1.3.2.cmml" xref="Sx7.SSx3.p2.6.m6.1.1.3.2">2</cn><ci id="Sx7.SSx3.p2.6.m6.1.1.3.3.cmml" xref="Sx7.SSx3.p2.6.m6.1.1.3.3">𝑘</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.6.m6.1c">64-2k</annotation></semantics></math> bits) Where the kmer hash would be too large
for available memories, we sample the kmers using a hash-slicing
factor <math id="Sx7.SSx3.p2.7.m7.1" class="ltx_Math" alttext="S" display="inline"><semantics id="Sx7.SSx3.p2.7.m7.1a"><mi id="Sx7.SSx3.p2.7.m7.1.1" xref="Sx7.SSx3.p2.7.m7.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.7.m7.1b"><ci id="Sx7.SSx3.p2.7.m7.1.1.cmml" xref="Sx7.SSx3.p2.7.m7.1.1">𝑆</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.7.m7.1c">S</annotation></semantics></math> (must be prime). Representing each kmer as an integer in
<math id="Sx7.SSx3.p2.8.m8.1" class="ltx_Math" alttext="[0..{4^{k}}-1]" display="inline"><semantics id="Sx7.SSx3.p2.8.m8.1a"><mrow id="Sx7.SSx3.p2.8.m8.1.1.1" xref="Sx7.SSx3.p2.8.m8.1.1.2.cmml"><mo stretchy="false" id="Sx7.SSx3.p2.8.m8.1.1.1.2" xref="Sx7.SSx3.p2.8.m8.1.1.2.1.cmml">[</mo><mrow id="Sx7.SSx3.p2.8.m8.1.1.1.1" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.cmml"><msup id="Sx7.SSx3.p2.8.m8.1.1.1.1.2" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.2.cmml"><mn id="Sx7.SSx3.p2.8.m8.1.1.1.1.2.2" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.2.2.cmml">0..4</mn><mi id="Sx7.SSx3.p2.8.m8.1.1.1.1.2.3" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.2.3.cmml">k</mi></msup><mo id="Sx7.SSx3.p2.8.m8.1.1.1.1.1" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.1.cmml">−</mo><mn id="Sx7.SSx3.p2.8.m8.1.1.1.1.3" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.3.cmml">1</mn></mrow><mo stretchy="false" id="Sx7.SSx3.p2.8.m8.1.1.1.3" xref="Sx7.SSx3.p2.8.m8.1.1.2.1.cmml">]</mo></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.8.m8.1b"><apply id="Sx7.SSx3.p2.8.m8.1.1.2.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1"><csymbol cd="latexml" id="Sx7.SSx3.p2.8.m8.1.1.2.1.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1.2">delimited-[]</csymbol><apply id="Sx7.SSx3.p2.8.m8.1.1.1.1.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1.1"><minus id="Sx7.SSx3.p2.8.m8.1.1.1.1.1.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.1"></minus><apply id="Sx7.SSx3.p2.8.m8.1.1.1.1.2.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.2"><csymbol cd="ambiguous" id="Sx7.SSx3.p2.8.m8.1.1.1.1.2.1.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.2">superscript</csymbol><cn type="float" id="Sx7.SSx3.p2.8.m8.1.1.1.1.2.2.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.2.2">0..4</cn><ci id="Sx7.SSx3.p2.8.m8.1.1.1.1.2.3.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.2.3">𝑘</ci></apply><cn type="integer" id="Sx7.SSx3.p2.8.m8.1.1.1.1.3.cmml" xref="Sx7.SSx3.p2.8.m8.1.1.1.1.3">1</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.8.m8.1c">[0..{4^{k}}-1]</annotation></semantics></math>, slice <math id="Sx7.SSx3.p2.9.m9.1" class="ltx_Math" alttext="s" display="inline"><semantics id="Sx7.SSx3.p2.9.m9.1a"><mi id="Sx7.SSx3.p2.9.m9.1.1" xref="Sx7.SSx3.p2.9.m9.1.1.cmml">s</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.9.m9.1b"><ci id="Sx7.SSx3.p2.9.m9.1.1.cmml" xref="Sx7.SSx3.p2.9.m9.1.1">𝑠</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.9.m9.1c">s</annotation></semantics></math> consists of those kmers whose remainder on
division by <math id="Sx7.SSx3.p2.10.m10.1" class="ltx_Math" alttext="S" display="inline"><semantics id="Sx7.SSx3.p2.10.m10.1a"><mi id="Sx7.SSx3.p2.10.m10.1.1" xref="Sx7.SSx3.p2.10.m10.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.10.m10.1b"><ci id="Sx7.SSx3.p2.10.m10.1.1.cmml" xref="Sx7.SSx3.p2.10.m10.1.1">𝑆</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.10.m10.1c">S</annotation></semantics></math> is <math id="Sx7.SSx3.p2.11.m11.1" class="ltx_Math" alttext="s" display="inline"><semantics id="Sx7.SSx3.p2.11.m11.1a"><mi id="Sx7.SSx3.p2.11.m11.1.1" xref="Sx7.SSx3.p2.11.m11.1.1.cmml">s</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.11.m11.1b"><ci id="Sx7.SSx3.p2.11.m11.1.1.cmml" xref="Sx7.SSx3.p2.11.m11.1.1">𝑠</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.11.m11.1c">s</annotation></semantics></math>. We can tabulate one slice for a representative
sample of <math id="Sx7.SSx3.p2.12.m12.1" class="ltx_Math" alttext="1/S" display="inline"><semantics id="Sx7.SSx3.p2.12.m12.1a"><mrow id="Sx7.SSx3.p2.12.m12.1.1" xref="Sx7.SSx3.p2.12.m12.1.1.cmml"><mn id="Sx7.SSx3.p2.12.m12.1.1.2" xref="Sx7.SSx3.p2.12.m12.1.1.2.cmml">1</mn><mo id="Sx7.SSx3.p2.12.m12.1.1.1" xref="Sx7.SSx3.p2.12.m12.1.1.1.cmml">/</mo><mi id="Sx7.SSx3.p2.12.m12.1.1.3" xref="Sx7.SSx3.p2.12.m12.1.1.3.cmml">S</mi></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.12.m12.1b"><apply id="Sx7.SSx3.p2.12.m12.1.1.cmml" xref="Sx7.SSx3.p2.12.m12.1.1"><divide id="Sx7.SSx3.p2.12.m12.1.1.1.cmml" xref="Sx7.SSx3.p2.12.m12.1.1.1"></divide><cn type="integer" id="Sx7.SSx3.p2.12.m12.1.1.2.cmml" xref="Sx7.SSx3.p2.12.m12.1.1.2">1</cn><ci id="Sx7.SSx3.p2.12.m12.1.1.3.cmml" xref="Sx7.SSx3.p2.12.m12.1.1.3">𝑆</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.12.m12.1c">1/S</annotation></semantics></math> kmers (for initial estimation of depth of coverage and
genome size) or, using <math id="Sx7.SSx3.p2.13.m13.1" class="ltx_Math" alttext="S" display="inline"><semantics id="Sx7.SSx3.p2.13.m13.1a"><mi id="Sx7.SSx3.p2.13.m13.1.1" xref="Sx7.SSx3.p2.13.m13.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p2.13.m13.1b"><ci id="Sx7.SSx3.p2.13.m13.1.1.cmml" xref="Sx7.SSx3.p2.13.m13.1.1">𝑆</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p2.13.m13.1c">S</annotation></semantics></math> independent jobs, collect information for
kmers in all slices. Our hash tables store odd-length kmers so that
reverse-complementary sequences can be combined without the ambiguous
orientation of palindromic sequences (e.g., <span id="Sx7.SSx3.p2.13.2" class="ltx_text ltx_font_typewriter">ATCGAT</span>).</p>
</div>
<div id="Sx7.SSx3.p3" class="ltx_para">
<p id="Sx7.SSx3.p3.1" class="ltx_p">After selecting <em id="Sx7.SSx3.p3.1.1" class="ltx_emph ltx_font_italic">k</em> as described above and making a full tabulation of
kmer counts and bit vectors, we filter out kmers not expected to
represent genomic sequence. <math id="Sx7.SSx3.p3.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx3.p3.1.m1.1a"><mi id="Sx7.SSx3.p3.1.m1.1.1" xref="Sx7.SSx3.p3.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p3.1.m1.1b"><ci id="Sx7.SSx3.p3.1.m1.1.1.cmml" xref="Sx7.SSx3.p3.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p3.1.m1.1c">k</annotation></semantics></math>-mers  were required to have three copies
in the total sequence set, with at least one copy in the initial run
and one in the second run. This was partly to filter out incomplete
adapter sequences, which can be difficult to trim but which were
different in the two runs.</p>
</div>
<div id="Sx7.SSx3.p4" class="ltx_para">
<p id="Sx7.SSx3.p4.5" class="ltx_p">Extending methods developed for the Atlas assembler<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib42" title="" class="ltx_ref">42</a>]</cite> to heterozygous sequences,
Figure <a href="#Sx5.F1" title="Figure 1 ‣ Assembly and Mapping ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> gives a rough decomposition of the <em id="Sx7.SSx3.p4.5.1" class="ltx_emph ltx_font_italic">k</em>-mer
frequency distribution for 23-mers with quality <math id="Sx7.SSx3.p4.1.m1.1" class="ltx_Math" alttext="\geq 20" display="inline"><semantics id="Sx7.SSx3.p4.1.m1.1a"><mrow id="Sx7.SSx3.p4.1.m1.1.1" xref="Sx7.SSx3.p4.1.m1.1.1.cmml"><mi id="Sx7.SSx3.p4.1.m1.1.1.2" xref="Sx7.SSx3.p4.1.m1.1.1.2.cmml"></mi><mo id="Sx7.SSx3.p4.1.m1.1.1.1" xref="Sx7.SSx3.p4.1.m1.1.1.1.cmml">≥</mo><mn id="Sx7.SSx3.p4.1.m1.1.1.3" xref="Sx7.SSx3.p4.1.m1.1.1.3.cmml">20</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p4.1.m1.1b"><apply id="Sx7.SSx3.p4.1.m1.1.1.cmml" xref="Sx7.SSx3.p4.1.m1.1.1"><geq id="Sx7.SSx3.p4.1.m1.1.1.1.cmml" xref="Sx7.SSx3.p4.1.m1.1.1.1"></geq><csymbol cd="latexml" id="Sx7.SSx3.p4.1.m1.1.1.2.cmml" xref="Sx7.SSx3.p4.1.m1.1.1.2">absent</csymbol><cn type="integer" id="Sx7.SSx3.p4.1.m1.1.1.3.cmml" xref="Sx7.SSx3.p4.1.m1.1.1.3">20</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p4.1.m1.1c">\geq 20</annotation></semantics></math>,
minimizing the square of the residuals of <em id="Sx7.SSx3.p4.5.2" class="ltx_emph ltx_font_italic">k</em>-mer counts on frequencies 3
through 70 while not exceeding the observed counts.
Four linked distributions model fractions of the genome as
monoallelic or biallelic: homozygous regions with <math id="Sx7.SSx3.p4.2.m2.1" class="ltx_Math" alttext="d=38.9" display="inline"><semantics id="Sx7.SSx3.p4.2.m2.1a"><mrow id="Sx7.SSx3.p4.2.m2.1.1" xref="Sx7.SSx3.p4.2.m2.1.1.cmml"><mi id="Sx7.SSx3.p4.2.m2.1.1.2" xref="Sx7.SSx3.p4.2.m2.1.1.2.cmml">d</mi><mo id="Sx7.SSx3.p4.2.m2.1.1.1" xref="Sx7.SSx3.p4.2.m2.1.1.1.cmml">=</mo><mn id="Sx7.SSx3.p4.2.m2.1.1.3" xref="Sx7.SSx3.p4.2.m2.1.1.3.cmml">38.9</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p4.2.m2.1b"><apply id="Sx7.SSx3.p4.2.m2.1.1.cmml" xref="Sx7.SSx3.p4.2.m2.1.1"><eq id="Sx7.SSx3.p4.2.m2.1.1.1.cmml" xref="Sx7.SSx3.p4.2.m2.1.1.1"></eq><ci id="Sx7.SSx3.p4.2.m2.1.1.2.cmml" xref="Sx7.SSx3.p4.2.m2.1.1.2">𝑑</ci><cn type="float" id="Sx7.SSx3.p4.2.m2.1.1.3.cmml" xref="Sx7.SSx3.p4.2.m2.1.1.3">38.9</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p4.2.m2.1c">d=38.9</annotation></semantics></math>-fold coverage
(dark blue), minor alleles covered at <math id="Sx7.SSx3.p4.3.m3.1" class="ltx_Math" alttext="d/4" display="inline"><semantics id="Sx7.SSx3.p4.3.m3.1a"><mrow id="Sx7.SSx3.p4.3.m3.1.1" xref="Sx7.SSx3.p4.3.m3.1.1.cmml"><mi id="Sx7.SSx3.p4.3.m3.1.1.2" xref="Sx7.SSx3.p4.3.m3.1.1.2.cmml">d</mi><mo id="Sx7.SSx3.p4.3.m3.1.1.1" xref="Sx7.SSx3.p4.3.m3.1.1.1.cmml">/</mo><mn id="Sx7.SSx3.p4.3.m3.1.1.3" xref="Sx7.SSx3.p4.3.m3.1.1.3.cmml">4</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p4.3.m3.1b"><apply id="Sx7.SSx3.p4.3.m3.1.1.cmml" xref="Sx7.SSx3.p4.3.m3.1.1"><divide id="Sx7.SSx3.p4.3.m3.1.1.1.cmml" xref="Sx7.SSx3.p4.3.m3.1.1.1"></divide><ci id="Sx7.SSx3.p4.3.m3.1.1.2.cmml" xref="Sx7.SSx3.p4.3.m3.1.1.2">𝑑</ci><cn type="integer" id="Sx7.SSx3.p4.3.m3.1.1.3.cmml" xref="Sx7.SSx3.p4.3.m3.1.1.3">4</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p4.3.m3.1c">d/4</annotation></semantics></math> (green), tied alleles at
<math id="Sx7.SSx3.p4.4.m4.1" class="ltx_Math" alttext="d/2" display="inline"><semantics id="Sx7.SSx3.p4.4.m4.1a"><mrow id="Sx7.SSx3.p4.4.m4.1.1" xref="Sx7.SSx3.p4.4.m4.1.1.cmml"><mi id="Sx7.SSx3.p4.4.m4.1.1.2" xref="Sx7.SSx3.p4.4.m4.1.1.2.cmml">d</mi><mo id="Sx7.SSx3.p4.4.m4.1.1.1" xref="Sx7.SSx3.p4.4.m4.1.1.1.cmml">/</mo><mn id="Sx7.SSx3.p4.4.m4.1.1.3" xref="Sx7.SSx3.p4.4.m4.1.1.3.cmml">2</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p4.4.m4.1b"><apply id="Sx7.SSx3.p4.4.m4.1.1.cmml" xref="Sx7.SSx3.p4.4.m4.1.1"><divide id="Sx7.SSx3.p4.4.m4.1.1.1.cmml" xref="Sx7.SSx3.p4.4.m4.1.1.1"></divide><ci id="Sx7.SSx3.p4.4.m4.1.1.2.cmml" xref="Sx7.SSx3.p4.4.m4.1.1.2">𝑑</ci><cn type="integer" id="Sx7.SSx3.p4.4.m4.1.1.3.cmml" xref="Sx7.SSx3.p4.4.m4.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p4.4.m4.1c">d/2</annotation></semantics></math> (red) and major alleles at <math id="Sx7.SSx3.p4.5.m5.1" class="ltx_Math" alttext="3d/4" display="inline"><semantics id="Sx7.SSx3.p4.5.m5.1a"><mrow id="Sx7.SSx3.p4.5.m5.1.1" xref="Sx7.SSx3.p4.5.m5.1.1.cmml"><mrow id="Sx7.SSx3.p4.5.m5.1.1.2" xref="Sx7.SSx3.p4.5.m5.1.1.2.cmml"><mn id="Sx7.SSx3.p4.5.m5.1.1.2.2" xref="Sx7.SSx3.p4.5.m5.1.1.2.2.cmml">3</mn><mo lspace="0em" rspace="0em" id="Sx7.SSx3.p4.5.m5.1.1.2.1" xref="Sx7.SSx3.p4.5.m5.1.1.2.1.cmml">​</mo><mi id="Sx7.SSx3.p4.5.m5.1.1.2.3" xref="Sx7.SSx3.p4.5.m5.1.1.2.3.cmml">d</mi></mrow><mo id="Sx7.SSx3.p4.5.m5.1.1.1" xref="Sx7.SSx3.p4.5.m5.1.1.1.cmml">/</mo><mn id="Sx7.SSx3.p4.5.m5.1.1.3" xref="Sx7.SSx3.p4.5.m5.1.1.3.cmml">4</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx3.p4.5.m5.1b"><apply id="Sx7.SSx3.p4.5.m5.1.1.cmml" xref="Sx7.SSx3.p4.5.m5.1.1"><divide id="Sx7.SSx3.p4.5.m5.1.1.1.cmml" xref="Sx7.SSx3.p4.5.m5.1.1.1"></divide><apply id="Sx7.SSx3.p4.5.m5.1.1.2.cmml" xref="Sx7.SSx3.p4.5.m5.1.1.2"><times id="Sx7.SSx3.p4.5.m5.1.1.2.1.cmml" xref="Sx7.SSx3.p4.5.m5.1.1.2.1"></times><cn type="integer" id="Sx7.SSx3.p4.5.m5.1.1.2.2.cmml" xref="Sx7.SSx3.p4.5.m5.1.1.2.2">3</cn><ci id="Sx7.SSx3.p4.5.m5.1.1.2.3.cmml" xref="Sx7.SSx3.p4.5.m5.1.1.2.3">𝑑</ci></apply><cn type="integer" id="Sx7.SSx3.p4.5.m5.1.1.3.cmml" xref="Sx7.SSx3.p4.5.m5.1.1.3">4</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx3.p4.5.m5.1c">3d/4</annotation></semantics></math> (purple). This fit
is robust enough to confirm the abundance of major-minor allele pairs
(27 percent of <em id="Sx7.SSx3.p4.5.3" class="ltx_emph ltx_font_italic">k</em>-loci, vs. 10 percent for tied alleles),
with the broader peaks in the data than in the fitted curves
consistent with less uniform sampling (for example, varying coverage
of parents and offspring).
The Poisson decomposition suggests a density of polymorphisms of 1.2%
in major-minor allele pairs, based on dividing the modeled number of
such sequenced pairs by <em id="Sx7.SSx3.p4.5.4" class="ltx_emph ltx_font_italic">k</em> (assuming most polymorphisms are SNPs
spaced at least <em id="Sx7.SSx3.p4.5.5" class="ltx_emph ltx_font_italic">k</em> bases apart), by <em id="Sx7.SSx3.p4.5.6" class="ltx_emph ltx_font_italic">d</em> (the estimated depth of
sequencing) and by the estimated genome size of 2.74 billion bases.</p>
</div>
</section>
<section id="Sx7.SSx4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">SNPmer identification </h3>

<div id="Sx7.SSx4.p1" class="ltx_para">
<p id="Sx7.SSx4.p1.1" class="ltx_p">The filtered kmer counts, computed in parallel, are loaded into a hash
table with additional fields to track kmers that are uniquely within
one mismatch of each other. Because this step analyzes all (non-error)
<math id="Sx7.SSx4.p1.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p1.1.m1.1a"><mi id="Sx7.SSx4.p1.1.m1.1.1" xref="Sx7.SSx4.p1.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p1.1.m1.1b"><ci id="Sx7.SSx4.p1.1.m1.1.1.cmml" xref="Sx7.SSx4.p1.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p1.1.m1.1c">k</annotation></semantics></math>-mers  in one table, this requires a single large-memory processor (on the
order of 32 GiB).</p>
</div>
<div id="Sx7.SSx4.p2" class="ltx_para">
<p id="Sx7.SSx4.p2.5" class="ltx_p">For each <math id="Sx7.SSx4.p2.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p2.1.m1.1a"><mi id="Sx7.SSx4.p2.1.m1.1.1" xref="Sx7.SSx4.p2.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p2.1.m1.1b"><ci id="Sx7.SSx4.p2.1.m1.1.1.cmml" xref="Sx7.SSx4.p2.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p2.1.m1.1c">k</annotation></semantics></math>-mer, we check all its <math id="Sx7.SSx4.p2.2.m2.1" class="ltx_Math" alttext="3k" display="inline"><semantics id="Sx7.SSx4.p2.2.m2.1a"><mrow id="Sx7.SSx4.p2.2.m2.1.1" xref="Sx7.SSx4.p2.2.m2.1.1.cmml"><mn id="Sx7.SSx4.p2.2.m2.1.1.2" xref="Sx7.SSx4.p2.2.m2.1.1.2.cmml">3</mn><mo lspace="0em" rspace="0em" id="Sx7.SSx4.p2.2.m2.1.1.1" xref="Sx7.SSx4.p2.2.m2.1.1.1.cmml">​</mo><mi id="Sx7.SSx4.p2.2.m2.1.1.3" xref="Sx7.SSx4.p2.2.m2.1.1.3.cmml">k</mi></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p2.2.m2.1b"><apply id="Sx7.SSx4.p2.2.m2.1.1.cmml" xref="Sx7.SSx4.p2.2.m2.1.1"><times id="Sx7.SSx4.p2.2.m2.1.1.1.cmml" xref="Sx7.SSx4.p2.2.m2.1.1.1"></times><cn type="integer" id="Sx7.SSx4.p2.2.m2.1.1.2.cmml" xref="Sx7.SSx4.p2.2.m2.1.1.2">3</cn><ci id="Sx7.SSx4.p2.2.m2.1.1.3.cmml" xref="Sx7.SSx4.p2.2.m2.1.1.3">𝑘</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p2.2.m2.1c">3k</annotation></semantics></math> one-substitution neighbors,
The <math id="Sx7.SSx4.p2.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p2.3.m3.1a"><mi id="Sx7.SSx4.p2.3.m3.1.1" xref="Sx7.SSx4.p2.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p2.3.m3.1b"><ci id="Sx7.SSx4.p2.3.m3.1.1.cmml" xref="Sx7.SSx4.p2.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p2.3.m3.1c">k</annotation></semantics></math>-mers  are partitioned each into one of three categories:
<em id="Sx7.SSx4.p2.5.1" class="ltx_emph ltx_font_italic">unique</em>: having no edit-neighbors within one
substitution; <em id="Sx7.SSx4.p2.5.2" class="ltx_emph ltx_font_italic">ambiguous</em>: having either multiple one-substitution
neighbors, or one neighbor that has multiple neighbors; or
<em id="Sx7.SSx4.p2.5.3" class="ltx_emph ltx_font_italic">partnered</em>: uniquely pairable with exactly one other
<math id="Sx7.SSx4.p2.4.m4.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p2.4.m4.1a"><mi id="Sx7.SSx4.p2.4.m4.1.1" xref="Sx7.SSx4.p2.4.m4.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p2.4.m4.1b"><ci id="Sx7.SSx4.p2.4.m4.1.1.cmml" xref="Sx7.SSx4.p2.4.m4.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p2.4.m4.1c">k</annotation></semantics></math>-mer  differing by one substitution, such <em id="Sx7.SSx4.p2.5.4" class="ltx_emph ltx_font_italic">kmers</em> also known from now on as SNPmers
or SNPmer pairs. For each SNPmer, we save the position of the
substitution, a bitmask for the change (transition, complement, or
non-complement transversion),
and whether the canonical form of the partner in the
table has the same sense or is reverse complemented with respect to
this <math id="Sx7.SSx4.p2.5.m5.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p2.5.m5.1a"><mi id="Sx7.SSx4.p2.5.m5.1.1" xref="Sx7.SSx4.p2.5.m5.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p2.5.m5.1b"><ci id="Sx7.SSx4.p2.5.m5.1.1.cmml" xref="Sx7.SSx4.p2.5.m5.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p2.5.m5.1c">k</annotation></semantics></math>-mer.</p>
</div>
<div id="Sx7.SSx4.p3" class="ltx_para">
<p id="Sx7.SSx4.p3.10" class="ltx_p">Only partnered and unique <math id="Sx7.SSx4.p3.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.1.m1.1a"><mi id="Sx7.SSx4.p3.1.m1.1.1" xref="Sx7.SSx4.p3.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.1.m1.1b"><ci id="Sx7.SSx4.p3.1.m1.1.1.cmml" xref="Sx7.SSx4.p3.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.1.m1.1c">k</annotation></semantics></math>-mers  will be further tracked. While this limited
method cannot identify <math id="Sx7.SSx4.p3.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.2.m2.1a"><mi id="Sx7.SSx4.p3.2.m2.1.1" xref="Sx7.SSx4.p3.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.2.m2.1b"><ci id="Sx7.SSx4.p3.2.m2.1.1.cmml" xref="Sx7.SSx4.p3.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.2.m2.1c">k</annotation></semantics></math>-mers  for genomic SNP and non-SNP locations with complete
confidence, false pairing or missed pairing should have limited effects on
subsequent analysis.
False pairing, due to coincidental similarities or repeats, will
combine nodes of the <math id="Sx7.SSx4.p3.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.3.m3.1a"><mi id="Sx7.SSx4.p3.3.m3.1.1" xref="Sx7.SSx4.p3.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.3.m3.1b"><ci id="Sx7.SSx4.p3.3.m3.1.1.cmml" xref="Sx7.SSx4.p3.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.3.m3.1c">k</annotation></semantics></math>-mer  graph (see below) and cause possible misassembly: for
our purposes, noise in the scaffolding, haplotype phasing, and linkage
analysis. Missed pairing can happend from indel polymorphisms, SNPs separated
by fewer than <math id="Sx7.SSx4.p3.4.m4.1" class="ltx_Math" alttext="k-1" display="inline"><semantics id="Sx7.SSx4.p3.4.m4.1a"><mrow id="Sx7.SSx4.p3.4.m4.1.1" xref="Sx7.SSx4.p3.4.m4.1.1.cmml"><mi id="Sx7.SSx4.p3.4.m4.1.1.2" xref="Sx7.SSx4.p3.4.m4.1.1.2.cmml">k</mi><mo id="Sx7.SSx4.p3.4.m4.1.1.1" xref="Sx7.SSx4.p3.4.m4.1.1.1.cmml">−</mo><mn id="Sx7.SSx4.p3.4.m4.1.1.3" xref="Sx7.SSx4.p3.4.m4.1.1.3.cmml">1</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.4.m4.1b"><apply id="Sx7.SSx4.p3.4.m4.1.1.cmml" xref="Sx7.SSx4.p3.4.m4.1.1"><minus id="Sx7.SSx4.p3.4.m4.1.1.1.cmml" xref="Sx7.SSx4.p3.4.m4.1.1.1"></minus><ci id="Sx7.SSx4.p3.4.m4.1.1.2.cmml" xref="Sx7.SSx4.p3.4.m4.1.1.2">𝑘</ci><cn type="integer" id="Sx7.SSx4.p3.4.m4.1.1.3.cmml" xref="Sx7.SSx4.p3.4.m4.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.4.m4.1c">k-1</annotation></semantics></math> positions, failure to sequence minor alleles, or
ambiguity due to too many similar <math id="Sx7.SSx4.p3.5.m5.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.5.m5.1a"><mi id="Sx7.SSx4.p3.5.m5.1.1" xref="Sx7.SSx4.p3.5.m5.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.5.m5.1b"><ci id="Sx7.SSx4.p3.5.m5.1.1.cmml" xref="Sx7.SSx4.p3.5.m5.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.5.m5.1c">k</annotation></semantics></math>-mers. Ambiguously non-unique <math id="Sx7.SSx4.p3.6.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.6.m6.1a"><mi id="Sx7.SSx4.p3.6.m6.1.1" xref="Sx7.SSx4.p3.6.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.6.m6.1b"><ci id="Sx7.SSx4.p3.6.m6.1.1.cmml" xref="Sx7.SSx4.p3.6.m6.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.6.m6.1c">k</annotation></semantics></math>-mers  will
be skipped over (reducing connectivity of the <math id="Sx7.SSx4.p3.7.m7.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.7.m7.1a"><mi id="Sx7.SSx4.p3.7.m7.1.1" xref="Sx7.SSx4.p3.7.m7.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.7.m7.1b"><ci id="Sx7.SSx4.p3.7.m7.1.1.cmml" xref="Sx7.SSx4.p3.7.m7.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.7.m7.1c">k</annotation></semantics></math>-mer  graph if there are too
many in a row). While allelic <math id="Sx7.SSx4.p3.8.m8.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.8.m8.1a"><mi id="Sx7.SSx4.p3.8.m8.1.1" xref="Sx7.SSx4.p3.8.m8.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.8.m8.1b"><ci id="Sx7.SSx4.p3.8.m8.1.1.cmml" xref="Sx7.SSx4.p3.8.m8.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.8.m8.1c">k</annotation></semantics></math>-mers  misidentified as unique will cause
more forks in the <math id="Sx7.SSx4.p3.9.m9.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.9.m9.1a"><mi id="Sx7.SSx4.p3.9.m9.1.1" xref="Sx7.SSx4.p3.9.m9.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.9.m9.1b"><ci id="Sx7.SSx4.p3.9.m9.1.1.cmml" xref="Sx7.SSx4.p3.9.m9.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.9.m9.1c">k</annotation></semantics></math>-mer  graph, those from major-alleles
will still be chained together with flanking unique sequence, provided
that major allele <math id="Sx7.SSx4.p3.10.m10.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx4.p3.10.m10.1a"><mi id="Sx7.SSx4.p3.10.m10.1.1" xref="Sx7.SSx4.p3.10.m10.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx4.p3.10.m10.1b"><ci id="Sx7.SSx4.p3.10.m10.1.1.cmml" xref="Sx7.SSx4.p3.10.m10.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx4.p3.10.m10.1c">k</annotation></semantics></math>-mers  have at least twice the frequency as those for the
minor allele.</p>
</div>
<div id="Sx7.SSx4.p4" class="ltx_para">
<p id="Sx7.SSx4.p4.1" class="ltx_p">Table <a href="#Sx7.T2" title="Table 2 ‣ SNPmer identification ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> shows the totals and percentages of the
different kmer categories, counting each SNPmer pair as one
kmer. SNPmer pairs account for 16.3 percent of the putative
genomically unique 23mer markers; dividing by 23 gives us the fraction
of bases in those markers that are putative SNPs: 0.71 percent.</p>
</div>
<figure id="Sx7.T2" class="ltx_table">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 2: </span>K-mer categories, counting a SNPmer pair as one kmer</figcaption>
<table id="Sx7.T2.1.1" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="Sx7.T2.1.1.1" class="ltx_tr">
<th id="Sx7.T2.1.1.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_column">
<math id="Sx7.T2.1.1.1.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.T2.1.1.1.1.m1.1a"><mi id="Sx7.T2.1.1.1.1.m1.1.1" xref="Sx7.T2.1.1.1.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.T2.1.1.1.1.m1.1b"><ci id="Sx7.T2.1.1.1.1.m1.1.1.cmml" xref="Sx7.T2.1.1.1.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.T2.1.1.1.1.m1.1c">k</annotation></semantics></math>-mer  type</th>
<th id="Sx7.T2.1.1.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_column"># Distinct</th>
<th id="Sx7.T2.1.1.1.3" class="ltx_td ltx_align_left ltx_th ltx_th_column">Percentage</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="Sx7.T2.1.1.2.1" class="ltx_tr">
<td id="Sx7.T2.1.1.2.1.1" class="ltx_td ltx_align_left ltx_border_t">no partner/unique</td>
<td id="Sx7.T2.1.1.2.1.2" class="ltx_td ltx_align_left ltx_border_t">946,431,901</td>
<td id="Sx7.T2.1.1.2.1.3" class="ltx_td ltx_align_left ltx_border_t">55.48%</td>
</tr>
<tr id="Sx7.T2.1.1.3.2" class="ltx_tr">
<td id="Sx7.T2.1.1.3.2.1" class="ltx_td ltx_align_left">partnered/SNPmer</td>
<td id="Sx7.T2.1.1.3.2.2" class="ltx_td ltx_align_left">184,756,149</td>
<td id="Sx7.T2.1.1.3.2.3" class="ltx_td ltx_align_left">10.83%</td>
</tr>
<tr id="Sx7.T2.1.1.4.3" class="ltx_tr">
<td id="Sx7.T2.1.1.4.3.1" class="ltx_td ltx_align_left">ambiguous</td>
<td id="Sx7.T2.1.1.4.3.2" class="ltx_td ltx_align_left">574,557,296</td>
<td id="Sx7.T2.1.1.4.3.3" class="ltx_td ltx_align_left">33.68%</td>
</tr>
<tr id="Sx7.T2.1.1.5.4" class="ltx_tr">
<td id="Sx7.T2.1.1.5.4.1" class="ltx_td ltx_align_left">TOTAL</td>
<td id="Sx7.T2.1.1.5.4.2" class="ltx_td ltx_align_left">1,705,745,346</td>
<td id="Sx7.T2.1.1.5.4.3" class="ltx_td"></td>
</tr>
</tbody>
</table>
</figure>
</section>
<section id="Sx7.SSx5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Subset <math id="Sx7.SSx5.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.1.m1.1b"><mi id="Sx7.SSx5.1.m1.1.1" xref="Sx7.SSx5.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.1.m1.1c"><ci id="Sx7.SSx5.1.m1.1.1.cmml" xref="Sx7.SSx5.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.1.m1.1d">k</annotation></semantics></math>-mer  selection </h3>

<div id="Sx7.SSx5.p1" class="ltx_para">
<p id="Sx7.SSx5.p1.2" class="ltx_p">To reduce the memory requirements of our <math id="Sx7.SSx5.p1.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p1.1.m1.1a"><mi id="Sx7.SSx5.p1.1.m1.1.1" xref="Sx7.SSx5.p1.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p1.1.m1.1b"><ci id="Sx7.SSx5.p1.1.m1.1.1.cmml" xref="Sx7.SSx5.p1.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p1.1.m1.1c">k</annotation></semantics></math>-mer  assembly graph, we
select a roughly one-tenth subset of the <math id="Sx7.SSx5.p1.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p1.2.m2.1a"><mi id="Sx7.SSx5.p1.2.m2.1.1" xref="Sx7.SSx5.p1.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p1.2.m2.1b"><ci id="Sx7.SSx5.p1.2.m2.1.1.cmml" xref="Sx7.SSx5.p1.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p1.2.m2.1c">k</annotation></semantics></math>-mers .</p>
</div>
<div id="Sx7.SSx5.p2" class="ltx_para">
<p id="Sx7.SSx5.p2.2" class="ltx_p">In the case of a true SNP at least <em id="Sx7.SSx5.p2.2.1" class="ltx_emph ltx_font_italic">k-1</em> bases from other SNPs and gaps in error-free coverage of either allele, there will be <em id="Sx7.SSx5.p2.2.2" class="ltx_emph ltx_font_italic">k</em> covering SNPmer pairs (provided that covering <math id="Sx7.SSx5.p2.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p2.1.m1.1a"><mi id="Sx7.SSx5.p2.1.m1.1.1" xref="Sx7.SSx5.p2.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p2.1.m1.1b"><ci id="Sx7.SSx5.p2.1.m1.1.1.cmml" xref="Sx7.SSx5.p2.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p2.1.m1.1c">k</annotation></semantics></math>-mers  are also uniquely pairable). By taking only SNPmer pairs with the substitution in particular positions, we can reduce the size of the graph and its redundancy. Analyzing the distribution of edit position for all the SNPmer pairs, we observe an enrichment for edits near the ends due to sequencing errors. By selecting positions 3, 12 and 21 of 23-base SNPmers, we avoid the most problematic positions and reduce this portion of <math id="Sx7.SSx5.p2.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p2.2.m2.1a"><mi id="Sx7.SSx5.p2.2.m2.1.1" xref="Sx7.SSx5.p2.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p2.2.m2.1b"><ci id="Sx7.SSx5.p2.2.m2.1.1.cmml" xref="Sx7.SSx5.p2.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p2.2.m2.1c">k</annotation></semantics></math>-mer  nodes by a factor of 7.67.</p>
</div>
<div id="Sx7.SSx5.p3" class="ltx_para">
<p id="Sx7.SSx5.p3.6" class="ltx_p">Unlike for SNPmers, there are no canonical positions that identify the
unique, unpaired <math id="Sx7.SSx5.p3.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p3.1.m1.1a"><mi id="Sx7.SSx5.p3.1.m1.1.1" xref="Sx7.SSx5.p3.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p3.1.m1.1b"><ci id="Sx7.SSx5.p3.1.m1.1.1.cmml" xref="Sx7.SSx5.p3.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p3.1.m1.1c">k</annotation></semantics></math>-mers  . Several mechanisms have been proposed for sampling <math id="Sx7.SSx5.p3.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p3.2.m2.1a"><mi id="Sx7.SSx5.p3.2.m2.1.1" xref="Sx7.SSx5.p3.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p3.2.m2.1b"><ci id="Sx7.SSx5.p3.2.m2.1.1.cmml" xref="Sx7.SSx5.p3.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p3.2.m2.1c">k</annotation></semantics></math>-mers  in a representative way<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib49" title="" class="ltx_ref">49</a>, <a href="#bib.bib50" title="" class="ltx_ref">50</a>]</cite>. We use the more pseudo-random hash-slicing rule, already discussed above, to sample a single slice of <math id="Sx7.SSx5.p3.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p3.3.m3.1a"><mi id="Sx7.SSx5.p3.3.m3.1.1" xref="Sx7.SSx5.p3.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p3.3.m3.1b"><ci id="Sx7.SSx5.p3.3.m3.1.1.cmml" xref="Sx7.SSx5.p3.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p3.3.m3.1c">k</annotation></semantics></math>-mers  those whose integer encodings are congruent to a particular slice number <math id="Sx7.SSx5.p3.4.m4.1" class="ltx_Math" alttext="s" display="inline"><semantics id="Sx7.SSx5.p3.4.m4.1a"><mi id="Sx7.SSx5.p3.4.m4.1.1" xref="Sx7.SSx5.p3.4.m4.1.1.cmml">s</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p3.4.m4.1b"><ci id="Sx7.SSx5.p3.4.m4.1.1.cmml" xref="Sx7.SSx5.p3.4.m4.1.1">𝑠</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p3.4.m4.1c">s</annotation></semantics></math>, modulo <math id="Sx7.SSx5.p3.5.m5.1" class="ltx_Math" alttext="S" display="inline"><semantics id="Sx7.SSx5.p3.5.m5.1a"><mi id="Sx7.SSx5.p3.5.m5.1.1" xref="Sx7.SSx5.p3.5.m5.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p3.5.m5.1b"><ci id="Sx7.SSx5.p3.5.m5.1.1.cmml" xref="Sx7.SSx5.p3.5.m5.1.1">𝑆</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p3.5.m5.1c">S</annotation></semantics></math> (the hash slicing factor). We have found that on the finished human genome (results not shown), hash slicing is effectively a Poisson sampling, with sampled <math id="Sx7.SSx5.p3.6.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p3.6.m6.1a"><mi id="Sx7.SSx5.p3.6.m6.1.1" xref="Sx7.SSx5.p3.6.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p3.6.m6.1b"><ci id="Sx7.SSx5.p3.6.m6.1.1.cmml" xref="Sx7.SSx5.p3.6.m6.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p3.6.m6.1c">k</annotation></semantics></math>-mers  spaced according to an exponential distribution.</p>
</div>
<div id="Sx7.SSx5.p4" class="ltx_para">
<p id="Sx7.SSx5.p4.2" class="ltx_p">A caveat in applying hash slicing is that taking the remainder modulo a prime is not very pseudo-random for Mersenne primes (equal to <math id="Sx7.SSx5.p4.1.m1.1" class="ltx_Math" alttext="2^{p-1}" display="inline"><semantics id="Sx7.SSx5.p4.1.m1.1a"><msup id="Sx7.SSx5.p4.1.m1.1.1" xref="Sx7.SSx5.p4.1.m1.1.1.cmml"><mn id="Sx7.SSx5.p4.1.m1.1.1.2" xref="Sx7.SSx5.p4.1.m1.1.1.2.cmml">2</mn><mrow id="Sx7.SSx5.p4.1.m1.1.1.3" xref="Sx7.SSx5.p4.1.m1.1.1.3.cmml"><mi id="Sx7.SSx5.p4.1.m1.1.1.3.2" xref="Sx7.SSx5.p4.1.m1.1.1.3.2.cmml">p</mi><mo id="Sx7.SSx5.p4.1.m1.1.1.3.1" xref="Sx7.SSx5.p4.1.m1.1.1.3.1.cmml">−</mo><mn id="Sx7.SSx5.p4.1.m1.1.1.3.3" xref="Sx7.SSx5.p4.1.m1.1.1.3.3.cmml">1</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p4.1.m1.1b"><apply id="Sx7.SSx5.p4.1.m1.1.1.cmml" xref="Sx7.SSx5.p4.1.m1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx5.p4.1.m1.1.1.1.cmml" xref="Sx7.SSx5.p4.1.m1.1.1">superscript</csymbol><cn type="integer" id="Sx7.SSx5.p4.1.m1.1.1.2.cmml" xref="Sx7.SSx5.p4.1.m1.1.1.2">2</cn><apply id="Sx7.SSx5.p4.1.m1.1.1.3.cmml" xref="Sx7.SSx5.p4.1.m1.1.1.3"><minus id="Sx7.SSx5.p4.1.m1.1.1.3.1.cmml" xref="Sx7.SSx5.p4.1.m1.1.1.3.1"></minus><ci id="Sx7.SSx5.p4.1.m1.1.1.3.2.cmml" xref="Sx7.SSx5.p4.1.m1.1.1.3.2">𝑝</ci><cn type="integer" id="Sx7.SSx5.p4.1.m1.1.1.3.3.cmml" xref="Sx7.SSx5.p4.1.m1.1.1.3.3">1</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p4.1.m1.1c">2^{p-1}</annotation></semantics></math> for some <em id="Sx7.SSx5.p4.2.1" class="ltx_emph ltx_font_italic">p</em>), when <math id="Sx7.SSx5.p4.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p4.2.m2.1a"><mi id="Sx7.SSx5.p4.2.m2.1.1" xref="Sx7.SSx5.p4.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p4.2.m2.1b"><ci id="Sx7.SSx5.p4.2.m2.1.1.cmml" xref="Sx7.SSx5.p4.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p4.2.m2.1c">k</annotation></semantics></math>-mers  are represented in base-4 encoding<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib48" title="" class="ltx_ref">48</a>]</cite>. We therefore pick a slicing factor of 11, the smallest non-Mersenne prime greater than our SNPmer sampling factor.</p>
</div>
<div id="Sx7.SSx5.p5" class="ltx_para">
<p id="Sx7.SSx5.p5.3" class="ltx_p">The resulting <math id="Sx7.SSx5.p5.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p5.1.m1.1a"><mi id="Sx7.SSx5.p5.1.m1.1.1" xref="Sx7.SSx5.p5.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p5.1.m1.1b"><ci id="Sx7.SSx5.p5.1.m1.1.1.cmml" xref="Sx7.SSx5.p5.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p5.1.m1.1c">k</annotation></semantics></math>-mer  subset has 86.0 million unpaired <math id="Sx7.SSx5.p5.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p5.2.m2.1a"><mi id="Sx7.SSx5.p5.2.m2.1.1" xref="Sx7.SSx5.p5.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p5.2.m2.1b"><ci id="Sx7.SSx5.p5.2.m2.1.1.cmml" xref="Sx7.SSx5.p5.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p5.2.m2.1c">k</annotation></semantics></math>-mers  and 24.0 million SNPmer pairs, each reduced as predicted, for a total factor of 9.8 reduction in <math id="Sx7.SSx5.p5.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx5.p5.3.m3.1a"><mi id="Sx7.SSx5.p5.3.m3.1.1" xref="Sx7.SSx5.p5.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx5.p5.3.m3.1b"><ci id="Sx7.SSx5.p5.3.m3.1.1.cmml" xref="Sx7.SSx5.p5.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx5.p5.3.m3.1c">k</annotation></semantics></math>-mer  nodes for the next step.</p>
</div>
</section>
<section id="Sx7.SSx6" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Contigging and scaffolding</h3>

<div id="Sx7.SSx6.p1" class="ltx_para">
<p id="Sx7.SSx6.p1.6" class="ltx_p">Each of the sampled unique <math id="Sx7.SSx6.p1.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p1.1.m1.1a"><mi id="Sx7.SSx6.p1.1.m1.1.1" xref="Sx7.SSx6.p1.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p1.1.m1.1b"><ci id="Sx7.SSx6.p1.1.m1.1.1.cmml" xref="Sx7.SSx6.p1.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p1.1.m1.1c">k</annotation></semantics></math>-mers  and SNPmer pairs is a node in the
<math id="Sx7.SSx6.p1.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p1.2.m2.1a"><mi id="Sx7.SSx6.p1.2.m2.1.1" xref="Sx7.SSx6.p1.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p1.2.m2.1b"><ci id="Sx7.SSx6.p1.2.m2.1.1.cmml" xref="Sx7.SSx6.p1.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p1.2.m2.1c">k</annotation></semantics></math>-mer  graph. Nodes are connected when the corresponding <math id="Sx7.SSx6.p1.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p1.3.m3.1a"><mi id="Sx7.SSx6.p1.3.m3.1.1" xref="Sx7.SSx6.p1.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p1.3.m3.1b"><ci id="Sx7.SSx6.p1.3.m3.1.1.cmml" xref="Sx7.SSx6.p1.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p1.3.m3.1c">k</annotation></semantics></math>-mers appear consecutively in at least one read of the input (any
intervening <math id="Sx7.SSx6.p1.4.m4.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p1.4.m4.1a"><mi id="Sx7.SSx6.p1.4.m4.1.1" xref="Sx7.SSx6.p1.4.m4.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p1.4.m4.1b"><ci id="Sx7.SSx6.p1.4.m4.1.1.cmml" xref="Sx7.SSx6.p1.4.m4.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p1.4.m4.1c">k</annotation></semantics></math>-mers  having been skipped due to sampling or
ambiguity). The relative orientation, distance and number of
supporting reads of the <math id="Sx7.SSx6.p1.5.m5.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p1.5.m5.1a"><mi id="Sx7.SSx6.p1.5.m5.1.1" xref="Sx7.SSx6.p1.5.m5.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p1.5.m5.1b"><ci id="Sx7.SSx6.p1.5.m5.1.1.cmml" xref="Sx7.SSx6.p1.5.m5.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p1.5.m5.1c">k</annotation></semantics></math>-mers  is stored in the edge. When
conflicting distance or relative orientation is observed among
different reads for the same pair of <math id="Sx7.SSx6.p1.6.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p1.6.m6.1a"><mi id="Sx7.SSx6.p1.6.m6.1.1" xref="Sx7.SSx6.p1.6.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p1.6.m6.1b"><ci id="Sx7.SSx6.p1.6.m6.1.1.cmml" xref="Sx7.SSx6.p1.6.m6.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p1.6.m6.1c">k</annotation></semantics></math>-mer  nodes, all edges from both
nodes in the corresponding direction are ignored in contigging.</p>
</div>
<div id="Sx7.SSx6.p2" class="ltx_para">
<p id="Sx7.SSx6.p2.1" class="ltx_p">Robust edges for one direction from a node are defined as those
supported by a supermajority of the reads containing the <math id="Sx7.SSx6.p2.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p2.1.m1.1a"><mi id="Sx7.SSx6.p2.1.m1.1.1" xref="Sx7.SSx6.p2.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p2.1.m1.1b"><ci id="Sx7.SSx6.p2.1.m1.1.1.cmml" xref="Sx7.SSx6.p2.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p2.1.m1.1c">k</annotation></semantics></math>-mer: the
number of supporting reads is greater than or equal to both (1) two
plus the sum of the read counts for all other edges in that direction
and (2) twice the read count of the next-most supported edge in the
same direction. By this construction, a node has at most one robust
edge in each direction.</p>
</div>
<div id="Sx7.SSx6.p3" class="ltx_para">
<p id="Sx7.SSx6.p3.1" class="ltx_p">A mutually robust edge is defined as one that is robust going in both
directions between the two nodes it connects.</p>
</div>
<div id="Sx7.SSx6.p4" class="ltx_para">
<p id="Sx7.SSx6.p4.2" class="ltx_p">Contigs are the connected components of the subgraph consisting of
mutually robust edges. Singleton and circular contigs are reported for
diagnostic purposes, but ignored in subsequent analysis. Each
retained “<math id="Sx7.SSx6.p4.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p4.1.m1.1a"><mi id="Sx7.SSx6.p4.1.m1.1.1" xref="Sx7.SSx6.p4.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p4.1.m1.1b"><ci id="Sx7.SSx6.p4.1.m1.1.1.cmml" xref="Sx7.SSx6.p4.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p4.1.m1.1c">k</annotation></semantics></math>-mer  contig” of Table <a href="#Sx5.T1" title="Table 1 ‣ Assembly and Mapping ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> therefore represents
a chain of nodes for SNPmers and unique <math id="Sx7.SSx6.p4.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p4.2.m2.1a"><mi id="Sx7.SSx6.p4.2.m2.1.1" xref="Sx7.SSx6.p4.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p4.2.m2.1b"><ci id="Sx7.SSx6.p4.2.m2.1.1.cmml" xref="Sx7.SSx6.p4.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p4.2.m2.1c">k</annotation></semantics></math>-mers  not shared with other
contigs.</p>
</div>
<div id="Sx7.SSx6.p5" class="ltx_para">
<p id="Sx7.SSx6.p5.2" class="ltx_p">After assembly of <math id="Sx7.SSx6.p5.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p5.1.m1.1a"><mi id="Sx7.SSx6.p5.1.m1.1.1" xref="Sx7.SSx6.p5.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p5.1.m1.1b"><ci id="Sx7.SSx6.p5.1.m1.1.1.cmml" xref="Sx7.SSx6.p5.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p5.1.m1.1c">k</annotation></semantics></math>-mer  contigs, we connect them in longer structures
using the Bambus scaffolder<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib26" title="" class="ltx_ref">26</a>]</cite>. This
requires mapping templates (read pairs) to contigs, based on
shared <math id="Sx7.SSx6.p5.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p5.2.m2.1a"><mi id="Sx7.SSx6.p5.2.m2.1.1" xref="Sx7.SSx6.p5.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p5.2.m2.1b"><ci id="Sx7.SSx6.p5.2.m2.1.1.cmml" xref="Sx7.SSx6.p5.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p5.2.m2.1c">k</annotation></semantics></math>-mer  content, and dividing the resulting graph of contigs
linked by templates into batches small enough for Bambus to
process.</p>
</div>
<div id="Sx7.SSx6.p6" class="ltx_para">
<p id="Sx7.SSx6.p6.3" class="ltx_p">Contigs are chains of <math id="Sx7.SSx6.p6.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p6.1.m1.1a"><mi id="Sx7.SSx6.p6.1.m1.1.1" xref="Sx7.SSx6.p6.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p6.1.m1.1b"><ci id="Sx7.SSx6.p6.1.m1.1.1.cmml" xref="Sx7.SSx6.p6.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p6.1.m1.1c">k</annotation></semantics></math>-mer  nodes (unique or SNPmers). Since each
node can appear only once in the contigs, contigs can be represented
with a variant of the <math id="Sx7.SSx6.p6.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p6.2.m2.1a"><mi id="Sx7.SSx6.p6.2.m2.1.1" xref="Sx7.SSx6.p6.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p6.2.m2.1b"><ci id="Sx7.SSx6.p6.2.m2.1.1.cmml" xref="Sx7.SSx6.p6.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p6.2.m2.1c">k</annotation></semantics></math>-mer  hash. This allows efficient tabulation of
template-to-contig overlaps in a single pass through the read
ckdata. Each overlap is detected by the number of shared <math id="Sx7.SSx6.p6.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p6.3.m3.1a"><mi id="Sx7.SSx6.p6.3.m3.1.1" xref="Sx7.SSx6.p6.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p6.3.m3.1b"><ci id="Sx7.SSx6.p6.3.m3.1.1.cmml" xref="Sx7.SSx6.p6.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p6.3.m3.1c">k</annotation></semantics></math>-mers  on a
consistent diagonal (<em id="Sx7.SSx6.p6.3.1" class="ltx_emph ltx_font_italic">i.e.</em> relative sequence offset) and their span
on read and contig. Templates that overlap multiple contigs can be
used to order and orient those contigs.</p>
</div>
<div id="Sx7.SSx6.p7" class="ltx_para">
<p id="Sx7.SSx6.p7.3" class="ltx_p">We present templates linking contigs to Bambus using AMOS format<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib43" title="" class="ltx_ref">43</a>]</cite> for the reads (template ends) mapped to
each contig. Reads are included only for the contig with which it
shares the most <math id="Sx7.SSx6.p7.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p7.1.m1.1a"><mi id="Sx7.SSx6.p7.1.m1.1.1" xref="Sx7.SSx6.p7.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p7.1.m1.1b"><ci id="Sx7.SSx6.p7.1.m1.1.1.cmml" xref="Sx7.SSx6.p7.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p7.1.m1.1c">k</annotation></semantics></math>-mers, if the span of those {<math id="Sx7.SSx6.p7.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p7.2.m2.1a"><mi id="Sx7.SSx6.p7.2.m2.1.1" xref="Sx7.SSx6.p7.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p7.2.m2.1b"><ci id="Sx7.SSx6.p7.2.m2.1.1.cmml" xref="Sx7.SSx6.p7.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p7.2.m2.1c">k</annotation></semantics></math>-mers} is <math id="Sx7.SSx6.p7.3.m3.1" class="ltx_Math" alttext="\geq k" display="inline"><semantics id="Sx7.SSx6.p7.3.m3.1a"><mrow id="Sx7.SSx6.p7.3.m3.1.1" xref="Sx7.SSx6.p7.3.m3.1.1.cmml"><mi id="Sx7.SSx6.p7.3.m3.1.1.2" xref="Sx7.SSx6.p7.3.m3.1.1.2.cmml"></mi><mo id="Sx7.SSx6.p7.3.m3.1.1.1" xref="Sx7.SSx6.p7.3.m3.1.1.1.cmml">≥</mo><mi id="Sx7.SSx6.p7.3.m3.1.1.3" xref="Sx7.SSx6.p7.3.m3.1.1.3.cmml">k</mi></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p7.3.m3.1b"><apply id="Sx7.SSx6.p7.3.m3.1.1.cmml" xref="Sx7.SSx6.p7.3.m3.1.1"><geq id="Sx7.SSx6.p7.3.m3.1.1.1.cmml" xref="Sx7.SSx6.p7.3.m3.1.1.1"></geq><csymbol cd="latexml" id="Sx7.SSx6.p7.3.m3.1.1.2.cmml" xref="Sx7.SSx6.p7.3.m3.1.1.2">absent</csymbol><ci id="Sx7.SSx6.p7.3.m3.1.1.3.cmml" xref="Sx7.SSx6.p7.3.m3.1.1.3">𝑘</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p7.3.m3.1c">\geq k</annotation></semantics></math> and
the other read-end of the template similarly qualifies in a different
contig. Bambus then infers links between contigs by matching template
identifiers shared by reads in different “linkable contigs”.</p>
</div>
<div id="Sx7.SSx6.p8" class="ltx_para">
<p id="Sx7.SSx6.p8.1" class="ltx_p">To generate independent AMOS files for each Bambus run, the large
graph of template links is partitioned into connected components.
These components are grouped into batches for a reasonably
load-balanced parallel computation with no template links between
batches. The results are summarized as “initial scaffolds” in Table
<a href="#Sx5.T1" title="Table 1 ‣ Assembly and Mapping ‣ Results" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.</p>
</div>
<div id="Sx7.SSx6.p9" class="ltx_para">
<p id="Sx7.SSx6.p9.3" class="ltx_p">Consensus sequence representations for {<math id="Sx7.SSx6.p9.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p9.1.m1.1a"><mi id="Sx7.SSx6.p9.1.m1.1.1" xref="Sx7.SSx6.p9.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p9.1.m1.1b"><ci id="Sx7.SSx6.p9.1.m1.1.1.cmml" xref="Sx7.SSx6.p9.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p9.1.m1.1c">k</annotation></semantics></math>-mer} contigs and scaffolds
were generated in two phases. In the first phase, sequence spanned by
selected SNPmers and subset {<math id="Sx7.SSx6.p9.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p9.2.m2.1a"><mi id="Sx7.SSx6.p9.2.m2.1.1" xref="Sx7.SSx6.p9.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p9.2.m2.1b"><ci id="Sx7.SSx6.p9.2.m2.1.1.cmml" xref="Sx7.SSx6.p9.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p9.2.m2.1c">k</annotation></semantics></math>-mers} (see sections <a href="#Sx7.SSx4" title="SNPmer identification ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_title">SNPmer identification</span></a> and
<a href="#Sx7.SSx5" title="Subset 𝑘-mer selection ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_title">Subset <math class="ltx_Math" alttext="k" display="inline"><semantics><mi>k</mi><annotation-xml encoding="MathML-Content"><ci>𝑘</ci></annotation-xml><annotation encoding="application/x-tex">k</annotation></semantics></math>-mer  selection</span></a> above) are joined together, separated by a number of Ns
corresponding to the number of bases not spanned by {<math id="Sx7.SSx6.p9.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx6.p9.3.m3.1a"><mi id="Sx7.SSx6.p9.3.m3.1.1" xref="Sx7.SSx6.p9.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx6.p9.3.m3.1b"><ci id="Sx7.SSx6.p9.3.m3.1.1.cmml" xref="Sx7.SSx6.p9.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx6.p9.3.m3.1c">k</annotation></semantics></math>-mers} in the
subset. In the second phase, a single pass is made through the read
data set, and stretches of Ns that are spanned by single reads are
replaced by the sequence of the read.</p>
</div>
</section>
<section id="Sx7.SSx7" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">SNP phase and genotype inference </h3>

<div id="Sx7.SSx7.p1" class="ltx_para">
<p id="Sx7.SSx7.p1.1" class="ltx_p">Each scaffold of the <math id="Sx7.SSx7.p1.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx7.p1.1.m1.1a"><mi id="Sx7.SSx7.p1.1.m1.1.1" xref="Sx7.SSx7.p1.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p1.1.m1.1b"><ci id="Sx7.SSx7.p1.1.m1.1.1.cmml" xref="Sx7.SSx7.p1.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p1.1.m1.1c">k</annotation></semantics></math>-mer  assembly constitutes a
candidate marker for mapping. While the depth of sequence coverage on
each member of the mapping panel is too low ( about 1X ) to directly
infer the genotype of individual members of the mapping pannel at
individual SNPs, the tight linkage between SNPs within markers means
that learning a sample’s genotype at any one reveals it at the
others, effectively amplifying the sequence coverage by a factor
proportional to
the number of SNPs within the marker. This is the same principle
exploited in genotype by sequencing (GBS) approaches to genetic
mapping in the presence of reference genomes, for example in
recombinant inbred lines of reference rice strains<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite>, and in crosses between <em id="Sx7.SSx7.p1.1.1" class="ltx_emph ltx_font_italic">Drosophila</em>
species with sequenced genomes<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite>. We use the same principle to genotype offspring in the context of a
cross between two outbred individuals, simultaneously inferring the phases of
the SNPs (<em id="Sx7.SSx7.p1.1.2" class="ltx_emph ltx_font_italic">i.e.</em> which bases appears on each of the four parental
chromosomes in the cross).</p>
</div>
<div id="Sx7.SSx7.p2" class="ltx_para">
<p id="Sx7.SSx7.p2.1" class="ltx_p">While the data will be insufficient to infer genotypes at many
markers, all those where confident inferences can be made can be used
to build the linkage map.</p>
</div>
<div id="Sx7.SSx7.p3" class="ltx_para">
<p id="Sx7.SSx7.p3.8" class="ltx_p">For the purposes of genotype inference, a marker is treated as a
collection of <math id="Sx7.SSx7.p3.1.m1.1" class="ltx_Math" alttext="m" display="inline"><semantics id="Sx7.SSx7.p3.1.m1.1a"><mi id="Sx7.SSx7.p3.1.m1.1.1" xref="Sx7.SSx7.p3.1.m1.1.1.cmml">m</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p3.1.m1.1b"><ci id="Sx7.SSx7.p3.1.m1.1.1.cmml" xref="Sx7.SSx7.p3.1.m1.1.1">𝑚</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p3.1.m1.1c">m</annotation></semantics></math> SNPs (indexed in the following by <math id="Sx7.SSx7.p3.2.m2.4" class="ltx_Math" alttext="i\in\{1,2,...,m\}" display="inline"><semantics id="Sx7.SSx7.p3.2.m2.4a"><mrow id="Sx7.SSx7.p3.2.m2.4.5" xref="Sx7.SSx7.p3.2.m2.4.5.cmml"><mi id="Sx7.SSx7.p3.2.m2.4.5.2" xref="Sx7.SSx7.p3.2.m2.4.5.2.cmml">i</mi><mo id="Sx7.SSx7.p3.2.m2.4.5.1" xref="Sx7.SSx7.p3.2.m2.4.5.1.cmml">∈</mo><mrow id="Sx7.SSx7.p3.2.m2.4.5.3.2" xref="Sx7.SSx7.p3.2.m2.4.5.3.1.cmml"><mo stretchy="false" id="Sx7.SSx7.p3.2.m2.4.5.3.2.1" xref="Sx7.SSx7.p3.2.m2.4.5.3.1.cmml">{</mo><mn id="Sx7.SSx7.p3.2.m2.1.1" xref="Sx7.SSx7.p3.2.m2.1.1.cmml">1</mn><mo id="Sx7.SSx7.p3.2.m2.4.5.3.2.2" xref="Sx7.SSx7.p3.2.m2.4.5.3.1.cmml">,</mo><mn id="Sx7.SSx7.p3.2.m2.2.2" xref="Sx7.SSx7.p3.2.m2.2.2.cmml">2</mn><mo id="Sx7.SSx7.p3.2.m2.4.5.3.2.3" xref="Sx7.SSx7.p3.2.m2.4.5.3.1.cmml">,</mo><mi mathvariant="normal" id="Sx7.SSx7.p3.2.m2.3.3" xref="Sx7.SSx7.p3.2.m2.3.3.cmml">…</mi><mo id="Sx7.SSx7.p3.2.m2.4.5.3.2.4" xref="Sx7.SSx7.p3.2.m2.4.5.3.1.cmml">,</mo><mi id="Sx7.SSx7.p3.2.m2.4.4" xref="Sx7.SSx7.p3.2.m2.4.4.cmml">m</mi><mo stretchy="false" id="Sx7.SSx7.p3.2.m2.4.5.3.2.5" xref="Sx7.SSx7.p3.2.m2.4.5.3.1.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p3.2.m2.4b"><apply id="Sx7.SSx7.p3.2.m2.4.5.cmml" xref="Sx7.SSx7.p3.2.m2.4.5"><in id="Sx7.SSx7.p3.2.m2.4.5.1.cmml" xref="Sx7.SSx7.p3.2.m2.4.5.1"></in><ci id="Sx7.SSx7.p3.2.m2.4.5.2.cmml" xref="Sx7.SSx7.p3.2.m2.4.5.2">𝑖</ci><set id="Sx7.SSx7.p3.2.m2.4.5.3.1.cmml" xref="Sx7.SSx7.p3.2.m2.4.5.3.2"><cn type="integer" id="Sx7.SSx7.p3.2.m2.1.1.cmml" xref="Sx7.SSx7.p3.2.m2.1.1">1</cn><cn type="integer" id="Sx7.SSx7.p3.2.m2.2.2.cmml" xref="Sx7.SSx7.p3.2.m2.2.2">2</cn><ci id="Sx7.SSx7.p3.2.m2.3.3.cmml" xref="Sx7.SSx7.p3.2.m2.3.3">…</ci><ci id="Sx7.SSx7.p3.2.m2.4.4.cmml" xref="Sx7.SSx7.p3.2.m2.4.4">𝑚</ci></set></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p3.2.m2.4c">i\in\{1,2,...,m\}</annotation></semantics></math>), that have been inferred to be closely linked on the
genome via the <math id="Sx7.SSx7.p3.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx7.p3.3.m3.1a"><mi id="Sx7.SSx7.p3.3.m3.1.1" xref="Sx7.SSx7.p3.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p3.3.m3.1b"><ci id="Sx7.SSx7.p3.3.m3.1.1.cmml" xref="Sx7.SSx7.p3.3.m3.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p3.3.m3.1c">k</annotation></semantics></math>-mer  assembly step.
If the four parental chromosomes are labeled <math id="Sx7.SSx7.p3.4.m4.2" class="ltx_Math" alttext="a,b" display="inline"><semantics id="Sx7.SSx7.p3.4.m4.2a"><mrow id="Sx7.SSx7.p3.4.m4.2.3.2" xref="Sx7.SSx7.p3.4.m4.2.3.1.cmml"><mi id="Sx7.SSx7.p3.4.m4.1.1" xref="Sx7.SSx7.p3.4.m4.1.1.cmml">a</mi><mo id="Sx7.SSx7.p3.4.m4.2.3.2.1" xref="Sx7.SSx7.p3.4.m4.2.3.1.cmml">,</mo><mi id="Sx7.SSx7.p3.4.m4.2.2" xref="Sx7.SSx7.p3.4.m4.2.2.cmml">b</mi></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p3.4.m4.2b"><list id="Sx7.SSx7.p3.4.m4.2.3.1.cmml" xref="Sx7.SSx7.p3.4.m4.2.3.2"><ci id="Sx7.SSx7.p3.4.m4.1.1.cmml" xref="Sx7.SSx7.p3.4.m4.1.1">𝑎</ci><ci id="Sx7.SSx7.p3.4.m4.2.2.cmml" xref="Sx7.SSx7.p3.4.m4.2.2">𝑏</ci></list></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p3.4.m4.2c">a,b</annotation></semantics></math> in one parent and
<math id="Sx7.SSx7.p3.5.m5.2" class="ltx_Math" alttext="c,d" display="inline"><semantics id="Sx7.SSx7.p3.5.m5.2a"><mrow id="Sx7.SSx7.p3.5.m5.2.3.2" xref="Sx7.SSx7.p3.5.m5.2.3.1.cmml"><mi id="Sx7.SSx7.p3.5.m5.1.1" xref="Sx7.SSx7.p3.5.m5.1.1.cmml">c</mi><mo id="Sx7.SSx7.p3.5.m5.2.3.2.1" xref="Sx7.SSx7.p3.5.m5.2.3.1.cmml">,</mo><mi id="Sx7.SSx7.p3.5.m5.2.2" xref="Sx7.SSx7.p3.5.m5.2.2.cmml">d</mi></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p3.5.m5.2b"><list id="Sx7.SSx7.p3.5.m5.2.3.1.cmml" xref="Sx7.SSx7.p3.5.m5.2.3.2"><ci id="Sx7.SSx7.p3.5.m5.1.1.cmml" xref="Sx7.SSx7.p3.5.m5.1.1">𝑐</ci><ci id="Sx7.SSx7.p3.5.m5.2.2.cmml" xref="Sx7.SSx7.p3.5.m5.2.2">𝑑</ci></list></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p3.5.m5.2c">c,d</annotation></semantics></math> in the other, then the genotyping problem is to infer which of
the four possible segregation states or genotypes <math id="Sx7.SSx7.p3.6.m6.4" class="ltx_Math" alttext="ac,ad,bc,bd" display="inline"><semantics id="Sx7.SSx7.p3.6.m6.4a"><mrow id="Sx7.SSx7.p3.6.m6.4.4.4" xref="Sx7.SSx7.p3.6.m6.4.4.5.cmml"><mrow id="Sx7.SSx7.p3.6.m6.1.1.1.1" xref="Sx7.SSx7.p3.6.m6.1.1.1.1.cmml"><mi id="Sx7.SSx7.p3.6.m6.1.1.1.1.2" xref="Sx7.SSx7.p3.6.m6.1.1.1.1.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p3.6.m6.1.1.1.1.1" xref="Sx7.SSx7.p3.6.m6.1.1.1.1.1.cmml">​</mo><mi id="Sx7.SSx7.p3.6.m6.1.1.1.1.3" xref="Sx7.SSx7.p3.6.m6.1.1.1.1.3.cmml">c</mi></mrow><mo id="Sx7.SSx7.p3.6.m6.4.4.4.5" xref="Sx7.SSx7.p3.6.m6.4.4.5.cmml">,</mo><mrow id="Sx7.SSx7.p3.6.m6.2.2.2.2" xref="Sx7.SSx7.p3.6.m6.2.2.2.2.cmml"><mi id="Sx7.SSx7.p3.6.m6.2.2.2.2.2" xref="Sx7.SSx7.p3.6.m6.2.2.2.2.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p3.6.m6.2.2.2.2.1" xref="Sx7.SSx7.p3.6.m6.2.2.2.2.1.cmml">​</mo><mi id="Sx7.SSx7.p3.6.m6.2.2.2.2.3" xref="Sx7.SSx7.p3.6.m6.2.2.2.2.3.cmml">d</mi></mrow><mo id="Sx7.SSx7.p3.6.m6.4.4.4.6" xref="Sx7.SSx7.p3.6.m6.4.4.5.cmml">,</mo><mrow id="Sx7.SSx7.p3.6.m6.3.3.3.3" xref="Sx7.SSx7.p3.6.m6.3.3.3.3.cmml"><mi id="Sx7.SSx7.p3.6.m6.3.3.3.3.2" xref="Sx7.SSx7.p3.6.m6.3.3.3.3.2.cmml">b</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p3.6.m6.3.3.3.3.1" xref="Sx7.SSx7.p3.6.m6.3.3.3.3.1.cmml">​</mo><mi id="Sx7.SSx7.p3.6.m6.3.3.3.3.3" xref="Sx7.SSx7.p3.6.m6.3.3.3.3.3.cmml">c</mi></mrow><mo id="Sx7.SSx7.p3.6.m6.4.4.4.7" xref="Sx7.SSx7.p3.6.m6.4.4.5.cmml">,</mo><mrow id="Sx7.SSx7.p3.6.m6.4.4.4.4" xref="Sx7.SSx7.p3.6.m6.4.4.4.4.cmml"><mi id="Sx7.SSx7.p3.6.m6.4.4.4.4.2" xref="Sx7.SSx7.p3.6.m6.4.4.4.4.2.cmml">b</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p3.6.m6.4.4.4.4.1" xref="Sx7.SSx7.p3.6.m6.4.4.4.4.1.cmml">​</mo><mi id="Sx7.SSx7.p3.6.m6.4.4.4.4.3" xref="Sx7.SSx7.p3.6.m6.4.4.4.4.3.cmml">d</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p3.6.m6.4b"><list id="Sx7.SSx7.p3.6.m6.4.4.5.cmml" xref="Sx7.SSx7.p3.6.m6.4.4.4"><apply id="Sx7.SSx7.p3.6.m6.1.1.1.1.cmml" xref="Sx7.SSx7.p3.6.m6.1.1.1.1"><times id="Sx7.SSx7.p3.6.m6.1.1.1.1.1.cmml" xref="Sx7.SSx7.p3.6.m6.1.1.1.1.1"></times><ci id="Sx7.SSx7.p3.6.m6.1.1.1.1.2.cmml" xref="Sx7.SSx7.p3.6.m6.1.1.1.1.2">𝑎</ci><ci id="Sx7.SSx7.p3.6.m6.1.1.1.1.3.cmml" xref="Sx7.SSx7.p3.6.m6.1.1.1.1.3">𝑐</ci></apply><apply id="Sx7.SSx7.p3.6.m6.2.2.2.2.cmml" xref="Sx7.SSx7.p3.6.m6.2.2.2.2"><times id="Sx7.SSx7.p3.6.m6.2.2.2.2.1.cmml" xref="Sx7.SSx7.p3.6.m6.2.2.2.2.1"></times><ci id="Sx7.SSx7.p3.6.m6.2.2.2.2.2.cmml" xref="Sx7.SSx7.p3.6.m6.2.2.2.2.2">𝑎</ci><ci id="Sx7.SSx7.p3.6.m6.2.2.2.2.3.cmml" xref="Sx7.SSx7.p3.6.m6.2.2.2.2.3">𝑑</ci></apply><apply id="Sx7.SSx7.p3.6.m6.3.3.3.3.cmml" xref="Sx7.SSx7.p3.6.m6.3.3.3.3"><times id="Sx7.SSx7.p3.6.m6.3.3.3.3.1.cmml" xref="Sx7.SSx7.p3.6.m6.3.3.3.3.1"></times><ci id="Sx7.SSx7.p3.6.m6.3.3.3.3.2.cmml" xref="Sx7.SSx7.p3.6.m6.3.3.3.3.2">𝑏</ci><ci id="Sx7.SSx7.p3.6.m6.3.3.3.3.3.cmml" xref="Sx7.SSx7.p3.6.m6.3.3.3.3.3">𝑐</ci></apply><apply id="Sx7.SSx7.p3.6.m6.4.4.4.4.cmml" xref="Sx7.SSx7.p3.6.m6.4.4.4.4"><times id="Sx7.SSx7.p3.6.m6.4.4.4.4.1.cmml" xref="Sx7.SSx7.p3.6.m6.4.4.4.4.1"></times><ci id="Sx7.SSx7.p3.6.m6.4.4.4.4.2.cmml" xref="Sx7.SSx7.p3.6.m6.4.4.4.4.2">𝑏</ci><ci id="Sx7.SSx7.p3.6.m6.4.4.4.4.3.cmml" xref="Sx7.SSx7.p3.6.m6.4.4.4.4.3">𝑑</ci></apply></list></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p3.6.m6.4c">ac,ad,bc,bd</annotation></semantics></math>
describes each sample at each marker locus. We index samples with
<math id="Sx7.SSx7.p3.7.m7.1" class="ltx_Math" alttext="j" display="inline"><semantics id="Sx7.SSx7.p3.7.m7.1a"><mi id="Sx7.SSx7.p3.7.m7.1.1" xref="Sx7.SSx7.p3.7.m7.1.1.cmml">j</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p3.7.m7.1b"><ci id="Sx7.SSx7.p3.7.m7.1.1.cmml" xref="Sx7.SSx7.p3.7.m7.1.1">𝑗</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p3.7.m7.1c">j</annotation></semantics></math>, and denote a sample genotype by <math id="Sx7.SSx7.p3.8.m8.1" class="ltx_Math" alttext="g_{j}" display="inline"><semantics id="Sx7.SSx7.p3.8.m8.1a"><msub id="Sx7.SSx7.p3.8.m8.1.1" xref="Sx7.SSx7.p3.8.m8.1.1.cmml"><mi id="Sx7.SSx7.p3.8.m8.1.1.2" xref="Sx7.SSx7.p3.8.m8.1.1.2.cmml">g</mi><mi id="Sx7.SSx7.p3.8.m8.1.1.3" xref="Sx7.SSx7.p3.8.m8.1.1.3.cmml">j</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p3.8.m8.1b"><apply id="Sx7.SSx7.p3.8.m8.1.1.cmml" xref="Sx7.SSx7.p3.8.m8.1.1"><csymbol cd="ambiguous" id="Sx7.SSx7.p3.8.m8.1.1.1.cmml" xref="Sx7.SSx7.p3.8.m8.1.1">subscript</csymbol><ci id="Sx7.SSx7.p3.8.m8.1.1.2.cmml" xref="Sx7.SSx7.p3.8.m8.1.1.2">𝑔</ci><ci id="Sx7.SSx7.p3.8.m8.1.1.3.cmml" xref="Sx7.SSx7.p3.8.m8.1.1.3">𝑗</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p3.8.m8.1c">g_{j}</annotation></semantics></math>.</p>
</div>
<div id="Sx7.SSx7.p4" class="ltx_para">
<p id="Sx7.SSx7.p4.4" class="ltx_p">We assume that markers are very small compared to a
chromosome, and ignore the possibility of a recombination event within
individual markers.
The data used for inference of the offspring genotypes consist of the
number of reads from each barcoded sample <math id="Sx7.SSx7.p4.1.m1.1" class="ltx_Math" alttext="j" display="inline"><semantics id="Sx7.SSx7.p4.1.m1.1a"><mi id="Sx7.SSx7.p4.1.m1.1.1" xref="Sx7.SSx7.p4.1.m1.1.1.cmml">j</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p4.1.m1.1b"><ci id="Sx7.SSx7.p4.1.m1.1.1.cmml" xref="Sx7.SSx7.p4.1.m1.1.1">𝑗</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p4.1.m1.1c">j</annotation></semantics></math>
showing each of the four
possible DNA bases <math id="Sx7.SSx7.p4.2.m2.1" class="ltx_Math" alttext="b" display="inline"><semantics id="Sx7.SSx7.p4.2.m2.1a"><mi id="Sx7.SSx7.p4.2.m2.1.1" xref="Sx7.SSx7.p4.2.m2.1.1.cmml">b</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p4.2.m2.1b"><ci id="Sx7.SSx7.p4.2.m2.1.1.cmml" xref="Sx7.SSx7.p4.2.m2.1.1">𝑏</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p4.2.m2.1c">b</annotation></semantics></math> at each variable SNP position <math id="Sx7.SSx7.p4.3.m3.1" class="ltx_Math" alttext="i" display="inline"><semantics id="Sx7.SSx7.p4.3.m3.1a"><mi id="Sx7.SSx7.p4.3.m3.1.1" xref="Sx7.SSx7.p4.3.m3.1.1.cmml">i</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p4.3.m3.1b"><ci id="Sx7.SSx7.p4.3.m3.1.1.cmml" xref="Sx7.SSx7.p4.3.m3.1.1">𝑖</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p4.3.m3.1c">i</annotation></semantics></math>, which we denote <math id="Sx7.SSx7.p4.4.m4.1" class="ltx_Math" alttext="{n_{ij}^{b}}" display="inline"><semantics id="Sx7.SSx7.p4.4.m4.1a"><msubsup id="Sx7.SSx7.p4.4.m4.1.1" xref="Sx7.SSx7.p4.4.m4.1.1.cmml"><mi id="Sx7.SSx7.p4.4.m4.1.1.2.2" xref="Sx7.SSx7.p4.4.m4.1.1.2.2.cmml">n</mi><mrow id="Sx7.SSx7.p4.4.m4.1.1.2.3" xref="Sx7.SSx7.p4.4.m4.1.1.2.3.cmml"><mi id="Sx7.SSx7.p4.4.m4.1.1.2.3.2" xref="Sx7.SSx7.p4.4.m4.1.1.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p4.4.m4.1.1.2.3.1" xref="Sx7.SSx7.p4.4.m4.1.1.2.3.1.cmml">​</mo><mi id="Sx7.SSx7.p4.4.m4.1.1.2.3.3" xref="Sx7.SSx7.p4.4.m4.1.1.2.3.3.cmml">j</mi></mrow><mi id="Sx7.SSx7.p4.4.m4.1.1.3" xref="Sx7.SSx7.p4.4.m4.1.1.3.cmml">b</mi></msubsup><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p4.4.m4.1b"><apply id="Sx7.SSx7.p4.4.m4.1.1.cmml" xref="Sx7.SSx7.p4.4.m4.1.1"><csymbol cd="ambiguous" id="Sx7.SSx7.p4.4.m4.1.1.1.cmml" xref="Sx7.SSx7.p4.4.m4.1.1">superscript</csymbol><apply id="Sx7.SSx7.p4.4.m4.1.1.2.cmml" xref="Sx7.SSx7.p4.4.m4.1.1"><csymbol cd="ambiguous" id="Sx7.SSx7.p4.4.m4.1.1.2.1.cmml" xref="Sx7.SSx7.p4.4.m4.1.1">subscript</csymbol><ci id="Sx7.SSx7.p4.4.m4.1.1.2.2.cmml" xref="Sx7.SSx7.p4.4.m4.1.1.2.2">𝑛</ci><apply id="Sx7.SSx7.p4.4.m4.1.1.2.3.cmml" xref="Sx7.SSx7.p4.4.m4.1.1.2.3"><times id="Sx7.SSx7.p4.4.m4.1.1.2.3.1.cmml" xref="Sx7.SSx7.p4.4.m4.1.1.2.3.1"></times><ci id="Sx7.SSx7.p4.4.m4.1.1.2.3.2.cmml" xref="Sx7.SSx7.p4.4.m4.1.1.2.3.2">𝑖</ci><ci id="Sx7.SSx7.p4.4.m4.1.1.2.3.3.cmml" xref="Sx7.SSx7.p4.4.m4.1.1.2.3.3">𝑗</ci></apply></apply><ci id="Sx7.SSx7.p4.4.m4.1.1.3.cmml" xref="Sx7.SSx7.p4.4.m4.1.1.3">𝑏</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p4.4.m4.1c">{n_{ij}^{b}}</annotation></semantics></math>.</p>
</div>
<div id="Sx7.SSx7.p5" class="ltx_para">
<p id="Sx7.SSx7.p5.8" class="ltx_p">If the phase <math id="Sx7.SSx7.p5.1.m1.1" class="ltx_Math" alttext="\phi_{i}" display="inline"><semantics id="Sx7.SSx7.p5.1.m1.1a"><msub id="Sx7.SSx7.p5.1.m1.1.1" xref="Sx7.SSx7.p5.1.m1.1.1.cmml"><mi id="Sx7.SSx7.p5.1.m1.1.1.2" xref="Sx7.SSx7.p5.1.m1.1.1.2.cmml">ϕ</mi><mi id="Sx7.SSx7.p5.1.m1.1.1.3" xref="Sx7.SSx7.p5.1.m1.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p5.1.m1.1b"><apply id="Sx7.SSx7.p5.1.m1.1.1.cmml" xref="Sx7.SSx7.p5.1.m1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx7.p5.1.m1.1.1.1.cmml" xref="Sx7.SSx7.p5.1.m1.1.1">subscript</csymbol><ci id="Sx7.SSx7.p5.1.m1.1.1.2.cmml" xref="Sx7.SSx7.p5.1.m1.1.1.2">italic-ϕ</ci><ci id="Sx7.SSx7.p5.1.m1.1.1.3.cmml" xref="Sx7.SSx7.p5.1.m1.1.1.3">𝑖</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p5.1.m1.1c">\phi_{i}</annotation></semantics></math> of SNP <math id="Sx7.SSx7.p5.2.m2.1" class="ltx_Math" alttext="i" display="inline"><semantics id="Sx7.SSx7.p5.2.m2.1a"><mi id="Sx7.SSx7.p5.2.m2.1.1" xref="Sx7.SSx7.p5.2.m2.1.1.cmml">i</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p5.2.m2.1b"><ci id="Sx7.SSx7.p5.2.m2.1.1.cmml" xref="Sx7.SSx7.p5.2.m2.1.1">𝑖</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p5.2.m2.1c">i</annotation></semantics></math> were known, <em id="Sx7.SSx7.p5.8.1" class="ltx_emph ltx_font_italic">i.e.</em> which base is present in each
of the four parental chromosomes, then a choice of genotype <math id="Sx7.SSx7.p5.3.m3.1" class="ltx_Math" alttext="g_{j}" display="inline"><semantics id="Sx7.SSx7.p5.3.m3.1a"><msub id="Sx7.SSx7.p5.3.m3.1.1" xref="Sx7.SSx7.p5.3.m3.1.1.cmml"><mi id="Sx7.SSx7.p5.3.m3.1.1.2" xref="Sx7.SSx7.p5.3.m3.1.1.2.cmml">g</mi><mi id="Sx7.SSx7.p5.3.m3.1.1.3" xref="Sx7.SSx7.p5.3.m3.1.1.3.cmml">j</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p5.3.m3.1b"><apply id="Sx7.SSx7.p5.3.m3.1.1.cmml" xref="Sx7.SSx7.p5.3.m3.1.1"><csymbol cd="ambiguous" id="Sx7.SSx7.p5.3.m3.1.1.1.cmml" xref="Sx7.SSx7.p5.3.m3.1.1">subscript</csymbol><ci id="Sx7.SSx7.p5.3.m3.1.1.2.cmml" xref="Sx7.SSx7.p5.3.m3.1.1.2">𝑔</ci><ci id="Sx7.SSx7.p5.3.m3.1.1.3.cmml" xref="Sx7.SSx7.p5.3.m3.1.1.3">𝑗</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p5.3.m3.1c">g_{j}</annotation></semantics></math>
implies a specific homozygous or heterozygous state <math id="Sx7.SSx7.p5.4.m4.10" class="ltx_Math" alttext="s_{ij}\in S=\{AA,CC,TT,GG,AC,AT,AG,CT,CG,TG\}" display="inline"><semantics id="Sx7.SSx7.p5.4.m4.10a"><mrow id="Sx7.SSx7.p5.4.m4.10.10" xref="Sx7.SSx7.p5.4.m4.10.10.cmml"><msub id="Sx7.SSx7.p5.4.m4.10.10.12" xref="Sx7.SSx7.p5.4.m4.10.10.12.cmml"><mi id="Sx7.SSx7.p5.4.m4.10.10.12.2" xref="Sx7.SSx7.p5.4.m4.10.10.12.2.cmml">s</mi><mrow id="Sx7.SSx7.p5.4.m4.10.10.12.3" xref="Sx7.SSx7.p5.4.m4.10.10.12.3.cmml"><mi id="Sx7.SSx7.p5.4.m4.10.10.12.3.2" xref="Sx7.SSx7.p5.4.m4.10.10.12.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.10.10.12.3.1" xref="Sx7.SSx7.p5.4.m4.10.10.12.3.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.10.10.12.3.3" xref="Sx7.SSx7.p5.4.m4.10.10.12.3.3.cmml">j</mi></mrow></msub><mo id="Sx7.SSx7.p5.4.m4.10.10.13" xref="Sx7.SSx7.p5.4.m4.10.10.13.cmml">∈</mo><mi id="Sx7.SSx7.p5.4.m4.10.10.14" xref="Sx7.SSx7.p5.4.m4.10.10.14.cmml">S</mi><mo id="Sx7.SSx7.p5.4.m4.10.10.15" xref="Sx7.SSx7.p5.4.m4.10.10.15.cmml">=</mo><mrow id="Sx7.SSx7.p5.4.m4.10.10.10.10" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml"><mo stretchy="false" id="Sx7.SSx7.p5.4.m4.10.10.10.10.11" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">{</mo><mrow id="Sx7.SSx7.p5.4.m4.1.1.1.1.1" xref="Sx7.SSx7.p5.4.m4.1.1.1.1.1.cmml"><mi id="Sx7.SSx7.p5.4.m4.1.1.1.1.1.2" xref="Sx7.SSx7.p5.4.m4.1.1.1.1.1.2.cmml">A</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.1.1.1.1.1.1" xref="Sx7.SSx7.p5.4.m4.1.1.1.1.1.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.1.1.1.1.1.3" xref="Sx7.SSx7.p5.4.m4.1.1.1.1.1.3.cmml">A</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.12" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.2.2.2.2.2" xref="Sx7.SSx7.p5.4.m4.2.2.2.2.2.cmml"><mi id="Sx7.SSx7.p5.4.m4.2.2.2.2.2.2" xref="Sx7.SSx7.p5.4.m4.2.2.2.2.2.2.cmml">C</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.2.2.2.2.2.1" xref="Sx7.SSx7.p5.4.m4.2.2.2.2.2.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.2.2.2.2.2.3" xref="Sx7.SSx7.p5.4.m4.2.2.2.2.2.3.cmml">C</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.13" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.3.3.3.3.3" xref="Sx7.SSx7.p5.4.m4.3.3.3.3.3.cmml"><mi id="Sx7.SSx7.p5.4.m4.3.3.3.3.3.2" xref="Sx7.SSx7.p5.4.m4.3.3.3.3.3.2.cmml">T</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.3.3.3.3.3.1" xref="Sx7.SSx7.p5.4.m4.3.3.3.3.3.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.3.3.3.3.3.3" xref="Sx7.SSx7.p5.4.m4.3.3.3.3.3.3.cmml">T</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.14" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.4.4.4.4.4" xref="Sx7.SSx7.p5.4.m4.4.4.4.4.4.cmml"><mi id="Sx7.SSx7.p5.4.m4.4.4.4.4.4.2" xref="Sx7.SSx7.p5.4.m4.4.4.4.4.4.2.cmml">G</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.4.4.4.4.4.1" xref="Sx7.SSx7.p5.4.m4.4.4.4.4.4.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.4.4.4.4.4.3" xref="Sx7.SSx7.p5.4.m4.4.4.4.4.4.3.cmml">G</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.15" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.5.5.5.5.5" xref="Sx7.SSx7.p5.4.m4.5.5.5.5.5.cmml"><mi id="Sx7.SSx7.p5.4.m4.5.5.5.5.5.2" xref="Sx7.SSx7.p5.4.m4.5.5.5.5.5.2.cmml">A</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.5.5.5.5.5.1" xref="Sx7.SSx7.p5.4.m4.5.5.5.5.5.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.5.5.5.5.5.3" xref="Sx7.SSx7.p5.4.m4.5.5.5.5.5.3.cmml">C</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.16" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.6.6.6.6.6" xref="Sx7.SSx7.p5.4.m4.6.6.6.6.6.cmml"><mi id="Sx7.SSx7.p5.4.m4.6.6.6.6.6.2" xref="Sx7.SSx7.p5.4.m4.6.6.6.6.6.2.cmml">A</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.6.6.6.6.6.1" xref="Sx7.SSx7.p5.4.m4.6.6.6.6.6.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.6.6.6.6.6.3" xref="Sx7.SSx7.p5.4.m4.6.6.6.6.6.3.cmml">T</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.17" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.7.7.7.7.7" xref="Sx7.SSx7.p5.4.m4.7.7.7.7.7.cmml"><mi id="Sx7.SSx7.p5.4.m4.7.7.7.7.7.2" xref="Sx7.SSx7.p5.4.m4.7.7.7.7.7.2.cmml">A</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.7.7.7.7.7.1" xref="Sx7.SSx7.p5.4.m4.7.7.7.7.7.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.7.7.7.7.7.3" xref="Sx7.SSx7.p5.4.m4.7.7.7.7.7.3.cmml">G</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.18" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.8.8.8.8.8" xref="Sx7.SSx7.p5.4.m4.8.8.8.8.8.cmml"><mi id="Sx7.SSx7.p5.4.m4.8.8.8.8.8.2" xref="Sx7.SSx7.p5.4.m4.8.8.8.8.8.2.cmml">C</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.8.8.8.8.8.1" xref="Sx7.SSx7.p5.4.m4.8.8.8.8.8.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.8.8.8.8.8.3" xref="Sx7.SSx7.p5.4.m4.8.8.8.8.8.3.cmml">T</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.19" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.9.9.9.9.9" xref="Sx7.SSx7.p5.4.m4.9.9.9.9.9.cmml"><mi id="Sx7.SSx7.p5.4.m4.9.9.9.9.9.2" xref="Sx7.SSx7.p5.4.m4.9.9.9.9.9.2.cmml">C</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.9.9.9.9.9.1" xref="Sx7.SSx7.p5.4.m4.9.9.9.9.9.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.9.9.9.9.9.3" xref="Sx7.SSx7.p5.4.m4.9.9.9.9.9.3.cmml">G</mi></mrow><mo id="Sx7.SSx7.p5.4.m4.10.10.10.10.20" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">,</mo><mrow id="Sx7.SSx7.p5.4.m4.10.10.10.10.10" xref="Sx7.SSx7.p5.4.m4.10.10.10.10.10.cmml"><mi id="Sx7.SSx7.p5.4.m4.10.10.10.10.10.2" xref="Sx7.SSx7.p5.4.m4.10.10.10.10.10.2.cmml">T</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.4.m4.10.10.10.10.10.1" xref="Sx7.SSx7.p5.4.m4.10.10.10.10.10.1.cmml">​</mo><mi id="Sx7.SSx7.p5.4.m4.10.10.10.10.10.3" xref="Sx7.SSx7.p5.4.m4.10.10.10.10.10.3.cmml">G</mi></mrow><mo stretchy="false" id="Sx7.SSx7.p5.4.m4.10.10.10.10.21" xref="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p5.4.m4.10b"><apply id="Sx7.SSx7.p5.4.m4.10.10.cmml" xref="Sx7.SSx7.p5.4.m4.10.10"><and id="Sx7.SSx7.p5.4.m4.10.10a.cmml" xref="Sx7.SSx7.p5.4.m4.10.10"></and><apply id="Sx7.SSx7.p5.4.m4.10.10b.cmml" xref="Sx7.SSx7.p5.4.m4.10.10"><in id="Sx7.SSx7.p5.4.m4.10.10.13.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.13"></in><apply id="Sx7.SSx7.p5.4.m4.10.10.12.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.12"><csymbol cd="ambiguous" id="Sx7.SSx7.p5.4.m4.10.10.12.1.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.12">subscript</csymbol><ci id="Sx7.SSx7.p5.4.m4.10.10.12.2.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.12.2">𝑠</ci><apply id="Sx7.SSx7.p5.4.m4.10.10.12.3.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.12.3"><times id="Sx7.SSx7.p5.4.m4.10.10.12.3.1.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.12.3.1"></times><ci id="Sx7.SSx7.p5.4.m4.10.10.12.3.2.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.12.3.2">𝑖</ci><ci id="Sx7.SSx7.p5.4.m4.10.10.12.3.3.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.12.3.3">𝑗</ci></apply></apply><ci id="Sx7.SSx7.p5.4.m4.10.10.14.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.14">𝑆</ci></apply><apply id="Sx7.SSx7.p5.4.m4.10.10c.cmml" xref="Sx7.SSx7.p5.4.m4.10.10"><eq id="Sx7.SSx7.p5.4.m4.10.10.15.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.15"></eq><share href="#Sx7.SSx7.p5.4.m4.10.10.14.cmml" id="Sx7.SSx7.p5.4.m4.10.10d.cmml" xref="Sx7.SSx7.p5.4.m4.10.10"></share><set id="Sx7.SSx7.p5.4.m4.10.10.10.11.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.10.10"><apply id="Sx7.SSx7.p5.4.m4.1.1.1.1.1.cmml" xref="Sx7.SSx7.p5.4.m4.1.1.1.1.1"><times id="Sx7.SSx7.p5.4.m4.1.1.1.1.1.1.cmml" xref="Sx7.SSx7.p5.4.m4.1.1.1.1.1.1"></times><ci id="Sx7.SSx7.p5.4.m4.1.1.1.1.1.2.cmml" xref="Sx7.SSx7.p5.4.m4.1.1.1.1.1.2">𝐴</ci><ci id="Sx7.SSx7.p5.4.m4.1.1.1.1.1.3.cmml" xref="Sx7.SSx7.p5.4.m4.1.1.1.1.1.3">𝐴</ci></apply><apply id="Sx7.SSx7.p5.4.m4.2.2.2.2.2.cmml" xref="Sx7.SSx7.p5.4.m4.2.2.2.2.2"><times id="Sx7.SSx7.p5.4.m4.2.2.2.2.2.1.cmml" xref="Sx7.SSx7.p5.4.m4.2.2.2.2.2.1"></times><ci id="Sx7.SSx7.p5.4.m4.2.2.2.2.2.2.cmml" xref="Sx7.SSx7.p5.4.m4.2.2.2.2.2.2">𝐶</ci><ci id="Sx7.SSx7.p5.4.m4.2.2.2.2.2.3.cmml" xref="Sx7.SSx7.p5.4.m4.2.2.2.2.2.3">𝐶</ci></apply><apply id="Sx7.SSx7.p5.4.m4.3.3.3.3.3.cmml" xref="Sx7.SSx7.p5.4.m4.3.3.3.3.3"><times id="Sx7.SSx7.p5.4.m4.3.3.3.3.3.1.cmml" xref="Sx7.SSx7.p5.4.m4.3.3.3.3.3.1"></times><ci id="Sx7.SSx7.p5.4.m4.3.3.3.3.3.2.cmml" xref="Sx7.SSx7.p5.4.m4.3.3.3.3.3.2">𝑇</ci><ci id="Sx7.SSx7.p5.4.m4.3.3.3.3.3.3.cmml" xref="Sx7.SSx7.p5.4.m4.3.3.3.3.3.3">𝑇</ci></apply><apply id="Sx7.SSx7.p5.4.m4.4.4.4.4.4.cmml" xref="Sx7.SSx7.p5.4.m4.4.4.4.4.4"><times id="Sx7.SSx7.p5.4.m4.4.4.4.4.4.1.cmml" xref="Sx7.SSx7.p5.4.m4.4.4.4.4.4.1"></times><ci id="Sx7.SSx7.p5.4.m4.4.4.4.4.4.2.cmml" xref="Sx7.SSx7.p5.4.m4.4.4.4.4.4.2">𝐺</ci><ci id="Sx7.SSx7.p5.4.m4.4.4.4.4.4.3.cmml" xref="Sx7.SSx7.p5.4.m4.4.4.4.4.4.3">𝐺</ci></apply><apply id="Sx7.SSx7.p5.4.m4.5.5.5.5.5.cmml" xref="Sx7.SSx7.p5.4.m4.5.5.5.5.5"><times id="Sx7.SSx7.p5.4.m4.5.5.5.5.5.1.cmml" xref="Sx7.SSx7.p5.4.m4.5.5.5.5.5.1"></times><ci id="Sx7.SSx7.p5.4.m4.5.5.5.5.5.2.cmml" xref="Sx7.SSx7.p5.4.m4.5.5.5.5.5.2">𝐴</ci><ci id="Sx7.SSx7.p5.4.m4.5.5.5.5.5.3.cmml" xref="Sx7.SSx7.p5.4.m4.5.5.5.5.5.3">𝐶</ci></apply><apply id="Sx7.SSx7.p5.4.m4.6.6.6.6.6.cmml" xref="Sx7.SSx7.p5.4.m4.6.6.6.6.6"><times id="Sx7.SSx7.p5.4.m4.6.6.6.6.6.1.cmml" xref="Sx7.SSx7.p5.4.m4.6.6.6.6.6.1"></times><ci id="Sx7.SSx7.p5.4.m4.6.6.6.6.6.2.cmml" xref="Sx7.SSx7.p5.4.m4.6.6.6.6.6.2">𝐴</ci><ci id="Sx7.SSx7.p5.4.m4.6.6.6.6.6.3.cmml" xref="Sx7.SSx7.p5.4.m4.6.6.6.6.6.3">𝑇</ci></apply><apply id="Sx7.SSx7.p5.4.m4.7.7.7.7.7.cmml" xref="Sx7.SSx7.p5.4.m4.7.7.7.7.7"><times id="Sx7.SSx7.p5.4.m4.7.7.7.7.7.1.cmml" xref="Sx7.SSx7.p5.4.m4.7.7.7.7.7.1"></times><ci id="Sx7.SSx7.p5.4.m4.7.7.7.7.7.2.cmml" xref="Sx7.SSx7.p5.4.m4.7.7.7.7.7.2">𝐴</ci><ci id="Sx7.SSx7.p5.4.m4.7.7.7.7.7.3.cmml" xref="Sx7.SSx7.p5.4.m4.7.7.7.7.7.3">𝐺</ci></apply><apply id="Sx7.SSx7.p5.4.m4.8.8.8.8.8.cmml" xref="Sx7.SSx7.p5.4.m4.8.8.8.8.8"><times id="Sx7.SSx7.p5.4.m4.8.8.8.8.8.1.cmml" xref="Sx7.SSx7.p5.4.m4.8.8.8.8.8.1"></times><ci id="Sx7.SSx7.p5.4.m4.8.8.8.8.8.2.cmml" xref="Sx7.SSx7.p5.4.m4.8.8.8.8.8.2">𝐶</ci><ci id="Sx7.SSx7.p5.4.m4.8.8.8.8.8.3.cmml" xref="Sx7.SSx7.p5.4.m4.8.8.8.8.8.3">𝑇</ci></apply><apply id="Sx7.SSx7.p5.4.m4.9.9.9.9.9.cmml" xref="Sx7.SSx7.p5.4.m4.9.9.9.9.9"><times id="Sx7.SSx7.p5.4.m4.9.9.9.9.9.1.cmml" xref="Sx7.SSx7.p5.4.m4.9.9.9.9.9.1"></times><ci id="Sx7.SSx7.p5.4.m4.9.9.9.9.9.2.cmml" xref="Sx7.SSx7.p5.4.m4.9.9.9.9.9.2">𝐶</ci><ci id="Sx7.SSx7.p5.4.m4.9.9.9.9.9.3.cmml" xref="Sx7.SSx7.p5.4.m4.9.9.9.9.9.3">𝐺</ci></apply><apply id="Sx7.SSx7.p5.4.m4.10.10.10.10.10.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.10.10.10"><times id="Sx7.SSx7.p5.4.m4.10.10.10.10.10.1.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.10.10.10.1"></times><ci id="Sx7.SSx7.p5.4.m4.10.10.10.10.10.2.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.10.10.10.2">𝑇</ci><ci id="Sx7.SSx7.p5.4.m4.10.10.10.10.10.3.cmml" xref="Sx7.SSx7.p5.4.m4.10.10.10.10.10.3">𝐺</ci></apply></set></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p5.4.m4.10c">s_{ij}\in S=\{AA,CC,TT,GG,AC,AT,AG,CT,CG,TG\}</annotation></semantics></math> for SNP <math id="Sx7.SSx7.p5.5.m5.1" class="ltx_Math" alttext="i" display="inline"><semantics id="Sx7.SSx7.p5.5.m5.1a"><mi id="Sx7.SSx7.p5.5.m5.1.1" xref="Sx7.SSx7.p5.5.m5.1.1.cmml">i</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p5.5.m5.1b"><ci id="Sx7.SSx7.p5.5.m5.1.1.cmml" xref="Sx7.SSx7.p5.5.m5.1.1">𝑖</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p5.5.m5.1c">i</annotation></semantics></math> in
sample <math id="Sx7.SSx7.p5.6.m6.1" class="ltx_Math" alttext="j" display="inline"><semantics id="Sx7.SSx7.p5.6.m6.1a"><mi id="Sx7.SSx7.p5.6.m6.1.1" xref="Sx7.SSx7.p5.6.m6.1.1.cmml">j</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p5.6.m6.1b"><ci id="Sx7.SSx7.p5.6.m6.1.1.cmml" xref="Sx7.SSx7.p5.6.m6.1.1">𝑗</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p5.6.m6.1c">j</annotation></semantics></math>.
For a given phase and genotype, the likelihood function
for a given SNP position in a given
sample is given by either a binomial (for homozygous states) or trinomial
probability mass function
of the read counts, base-calling error rate <math id="Sx7.SSx7.p5.7.m7.1" class="ltx_Math" alttext="\epsilon" display="inline"><semantics id="Sx7.SSx7.p5.7.m7.1a"><mi id="Sx7.SSx7.p5.7.m7.1.1" xref="Sx7.SSx7.p5.7.m7.1.1.cmml">ϵ</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p5.7.m7.1b"><ci id="Sx7.SSx7.p5.7.m7.1.1.cmml" xref="Sx7.SSx7.p5.7.m7.1.1">italic-ϵ</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p5.7.m7.1c">\epsilon</annotation></semantics></math>, and the site
genotype <math id="Sx7.SSx7.p5.8.m8.1" class="ltx_Math" alttext="s_{ij}" display="inline"><semantics id="Sx7.SSx7.p5.8.m8.1a"><msub id="Sx7.SSx7.p5.8.m8.1.1" xref="Sx7.SSx7.p5.8.m8.1.1.cmml"><mi id="Sx7.SSx7.p5.8.m8.1.1.2" xref="Sx7.SSx7.p5.8.m8.1.1.2.cmml">s</mi><mrow id="Sx7.SSx7.p5.8.m8.1.1.3" xref="Sx7.SSx7.p5.8.m8.1.1.3.cmml"><mi id="Sx7.SSx7.p5.8.m8.1.1.3.2" xref="Sx7.SSx7.p5.8.m8.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p5.8.m8.1.1.3.1" xref="Sx7.SSx7.p5.8.m8.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx7.p5.8.m8.1.1.3.3" xref="Sx7.SSx7.p5.8.m8.1.1.3.3.cmml">j</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p5.8.m8.1b"><apply id="Sx7.SSx7.p5.8.m8.1.1.cmml" xref="Sx7.SSx7.p5.8.m8.1.1"><csymbol cd="ambiguous" id="Sx7.SSx7.p5.8.m8.1.1.1.cmml" xref="Sx7.SSx7.p5.8.m8.1.1">subscript</csymbol><ci id="Sx7.SSx7.p5.8.m8.1.1.2.cmml" xref="Sx7.SSx7.p5.8.m8.1.1.2">𝑠</ci><apply id="Sx7.SSx7.p5.8.m8.1.1.3.cmml" xref="Sx7.SSx7.p5.8.m8.1.1.3"><times id="Sx7.SSx7.p5.8.m8.1.1.3.1.cmml" xref="Sx7.SSx7.p5.8.m8.1.1.3.1"></times><ci id="Sx7.SSx7.p5.8.m8.1.1.3.2.cmml" xref="Sx7.SSx7.p5.8.m8.1.1.3.2">𝑖</ci><ci id="Sx7.SSx7.p5.8.m8.1.1.3.3.cmml" xref="Sx7.SSx7.p5.8.m8.1.1.3.3">𝑗</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p5.8.m8.1c">s_{ij}</annotation></semantics></math>:</p>
</div>
<div id="Sx7.SSx7.p6" class="ltx_para">
<table id="Sx7.E1" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="Sx7.E1.m1.12" class="ltx_math_unparsed" alttext="\mathbb{L}(\phi_{i},g_{j})=p(n_{ij}^{b}|\phi_{i},g_{j})=P_{m}(n_{ij}^{b},s_{ij},\epsilon)=\begin{cases}{n\choose m}\epsilon^{m}(1-\epsilon)^{n-m},&amp;\text{if }s_{ij}\text{ homozygous}\\
\frac{n!}{k!l!m!}\epsilon\prime^{m}(\frac{1-\epsilon\prime}{2})^{k+l},&amp;\text{if }s_{ij}\text{ heterozygous}\\
\end{cases}" display="block"><semantics id="Sx7.E1.m1.12a"><mrow id="Sx7.E1.m1.12.12"><mrow id="Sx7.E1.m1.9.9.2"><mi id="Sx7.E1.m1.9.9.2.4">𝕃</mi><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.9.9.2.3">​</mo><mrow id="Sx7.E1.m1.9.9.2.2.2"><mo stretchy="false" id="Sx7.E1.m1.9.9.2.2.2.3">(</mo><msub id="Sx7.E1.m1.8.8.1.1.1.1"><mi id="Sx7.E1.m1.8.8.1.1.1.1.2">ϕ</mi><mi id="Sx7.E1.m1.8.8.1.1.1.1.3">i</mi></msub><mo id="Sx7.E1.m1.9.9.2.2.2.4">,</mo><msub id="Sx7.E1.m1.9.9.2.2.2.2"><mi id="Sx7.E1.m1.9.9.2.2.2.2.2">g</mi><mi id="Sx7.E1.m1.9.9.2.2.2.2.3">j</mi></msub><mo stretchy="false" id="Sx7.E1.m1.9.9.2.2.2.5">)</mo></mrow></mrow><mo id="Sx7.E1.m1.12.12.7">=</mo><mrow id="Sx7.E1.m1.10.10.3"><mi id="Sx7.E1.m1.10.10.3.3">p</mi><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.10.10.3.2">​</mo><mrow id="Sx7.E1.m1.10.10.3.1.1"><mo stretchy="false" id="Sx7.E1.m1.10.10.3.1.1.2">(</mo><mrow id="Sx7.E1.m1.10.10.3.1.1.1"><msubsup id="Sx7.E1.m1.10.10.3.1.1.1.4"><mi id="Sx7.E1.m1.10.10.3.1.1.1.4.2.2">n</mi><mrow id="Sx7.E1.m1.10.10.3.1.1.1.4.2.3"><mi id="Sx7.E1.m1.10.10.3.1.1.1.4.2.3.2">i</mi><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.10.10.3.1.1.1.4.2.3.1">​</mo><mi id="Sx7.E1.m1.10.10.3.1.1.1.4.2.3.3">j</mi></mrow><mi id="Sx7.E1.m1.10.10.3.1.1.1.4.3">b</mi></msubsup><mo fence="false" id="Sx7.E1.m1.10.10.3.1.1.1.3">|</mo><mrow id="Sx7.E1.m1.10.10.3.1.1.1.2.2"><msub id="Sx7.E1.m1.10.10.3.1.1.1.1.1.1"><mi id="Sx7.E1.m1.10.10.3.1.1.1.1.1.1.2">ϕ</mi><mi id="Sx7.E1.m1.10.10.3.1.1.1.1.1.1.3">i</mi></msub><mo id="Sx7.E1.m1.10.10.3.1.1.1.2.2.3">,</mo><msub id="Sx7.E1.m1.10.10.3.1.1.1.2.2.2"><mi id="Sx7.E1.m1.10.10.3.1.1.1.2.2.2.2">g</mi><mi id="Sx7.E1.m1.10.10.3.1.1.1.2.2.2.3">j</mi></msub></mrow></mrow><mo stretchy="false" id="Sx7.E1.m1.10.10.3.1.1.3">)</mo></mrow></mrow><mo id="Sx7.E1.m1.12.12.8">=</mo><mrow id="Sx7.E1.m1.12.12.5"><msub id="Sx7.E1.m1.12.12.5.4"><mi id="Sx7.E1.m1.12.12.5.4.2">P</mi><mi id="Sx7.E1.m1.12.12.5.4.3">m</mi></msub><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.12.12.5.3">​</mo><mrow id="Sx7.E1.m1.12.12.5.2.2"><mo stretchy="false" id="Sx7.E1.m1.12.12.5.2.2.3">(</mo><msubsup id="Sx7.E1.m1.11.11.4.1.1.1"><mi id="Sx7.E1.m1.11.11.4.1.1.1.2.2">n</mi><mrow id="Sx7.E1.m1.11.11.4.1.1.1.2.3"><mi id="Sx7.E1.m1.11.11.4.1.1.1.2.3.2">i</mi><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.11.11.4.1.1.1.2.3.1">​</mo><mi id="Sx7.E1.m1.11.11.4.1.1.1.2.3.3">j</mi></mrow><mi id="Sx7.E1.m1.11.11.4.1.1.1.3">b</mi></msubsup><mo id="Sx7.E1.m1.12.12.5.2.2.4">,</mo><msub id="Sx7.E1.m1.12.12.5.2.2.2"><mi id="Sx7.E1.m1.12.12.5.2.2.2.2">s</mi><mrow id="Sx7.E1.m1.12.12.5.2.2.2.3"><mi id="Sx7.E1.m1.12.12.5.2.2.2.3.2">i</mi><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.12.12.5.2.2.2.3.1">​</mo><mi id="Sx7.E1.m1.12.12.5.2.2.2.3.3">j</mi></mrow></msub><mo id="Sx7.E1.m1.12.12.5.2.2.5">,</mo><mi id="Sx7.E1.m1.7.7">ϵ</mi><mo stretchy="false" id="Sx7.E1.m1.12.12.5.2.2.6">)</mo></mrow></mrow><mo id="Sx7.E1.m1.12.12.9">=</mo><mrow id="Sx7.E1.m1.6.6"><mo id="Sx7.E1.m1.6.6.7">{</mo><mtable columnspacing="5pt" displaystyle="true" rowspacing="0pt" id="Sx7.E1.m1.6.6.6"><mtr id="Sx7.E1.m1.6.6.6a"><mtd class="ltx_align_left" columnalign="left" id="Sx7.E1.m1.6.6.6b"><mrow id="Sx7.E1.m1.3.3.3.3.3.3.4"><mrow id="Sx7.E1.m1.3.3.3.3.3.3.4.1"><mrow id="Sx7.E1.m1.3.3.3.3.3.3.3.5"><mo id="Sx7.E1.m1.3.3.3.3.3.3.3.5.1">(</mo><mstyle displaystyle="false" id="Sx7.E1.m1.3.3.3.3.3.3.3.3.3"><mfrac linethickness="0pt" id="Sx7.E1.m1.3.3.3.3.3.3.3.3.3a"><mi id="Sx7.E1.m1.2.2.2.2.2.2.2.2.2.2">n</mi><mi id="Sx7.E1.m1.3.3.3.3.3.3.3.3.3.3">m</mi></mfrac></mstyle><mo id="Sx7.E1.m1.3.3.3.3.3.3.3.5.2">)</mo></mrow><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.3.3.3.3.3.3.4.1.2">​</mo><msup id="Sx7.E1.m1.3.3.3.3.3.3.4.1.3"><mi id="Sx7.E1.m1.3.3.3.3.3.3.4.1.3.2">ϵ</mi><mi id="Sx7.E1.m1.3.3.3.3.3.3.4.1.3.3">m</mi></msup><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.3.3.3.3.3.3.4.1.2a">​</mo><msup id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1"><mrow id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.1.1"><mo stretchy="false" id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.1.1.2">(</mo><mrow id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.1.1.1"><mn id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.1.1.1.2">1</mn><mo id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.1.1.1.1">−</mo><mi id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.1.1.1.3">ϵ</mi></mrow><mo stretchy="false" id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.1.1.3">)</mo></mrow><mrow id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.3"><mi id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.3.2">n</mi><mo id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.3.1">−</mo><mi id="Sx7.E1.m1.3.3.3.3.3.3.4.1.1.3.3">m</mi></mrow></msup></mrow><mo id="Sx7.E1.m1.3.3.3.3.3.3.4.2">,</mo></mrow></mtd><mtd class="ltx_align_left" columnalign="left" id="Sx7.E1.m1.6.6.6c"><mrow id="Sx7.E1.m1.4.4.4.4.4.1"><mtext id="Sx7.E1.m1.4.4.4.4.4.1.2">if </mtext><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.4.4.4.4.4.1.1">​</mo><msub id="Sx7.E1.m1.4.4.4.4.4.1.3"><mi id="Sx7.E1.m1.4.4.4.4.4.1.3.2">s</mi><mrow id="Sx7.E1.m1.4.4.4.4.4.1.3.3"><mi id="Sx7.E1.m1.4.4.4.4.4.1.3.3.2">i</mi><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.4.4.4.4.4.1.3.3.1">​</mo><mi id="Sx7.E1.m1.4.4.4.4.4.1.3.3.3">j</mi></mrow></msub><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.4.4.4.4.4.1.1a">​</mo><mtext id="Sx7.E1.m1.4.4.4.4.4.1.4"> homozygous</mtext></mrow></mtd></mtr><mtr id="Sx7.E1.m1.6.6.6d"><mtd class="ltx_align_left" columnalign="left" id="Sx7.E1.m1.6.6.6e"><mrow id="Sx7.E1.m1.5.5.5.5.1.1"><mstyle displaystyle="false" id="Sx7.E1.m1.5.5.5.5.1.1.1"><mfrac id="Sx7.E1.m1.5.5.5.5.1.1.1a"><mrow id="Sx7.E1.m1.5.5.5.5.1.1.1.2"><mi id="Sx7.E1.m1.5.5.5.5.1.1.1.2.2">n</mi><mo id="Sx7.E1.m1.5.5.5.5.1.1.1.2.1">!</mo></mrow><mrow id="Sx7.E1.m1.5.5.5.5.1.1.1.3"><mrow id="Sx7.E1.m1.5.5.5.5.1.1.1.3.2"><mi id="Sx7.E1.m1.5.5.5.5.1.1.1.3.2.2">k</mi><mo id="Sx7.E1.m1.5.5.5.5.1.1.1.3.2.1">!</mo></mrow><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.5.5.5.5.1.1.1.3.1">​</mo><mrow id="Sx7.E1.m1.5.5.5.5.1.1.1.3.3"><mi id="Sx7.E1.m1.5.5.5.5.1.1.1.3.3.2">l</mi><mo id="Sx7.E1.m1.5.5.5.5.1.1.1.3.3.1">!</mo></mrow><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.5.5.5.5.1.1.1.3.1a">​</mo><mrow id="Sx7.E1.m1.5.5.5.5.1.1.1.3.4"><mi id="Sx7.E1.m1.5.5.5.5.1.1.1.3.4.2">m</mi><mo id="Sx7.E1.m1.5.5.5.5.1.1.1.3.4.1">!</mo></mrow></mrow></mfrac></mstyle><mi id="Sx7.E1.m1.5.5.5.5.1.1.2">ϵ</mi><msup id="Sx7.E1.m1.5.5.5.5.1.1.3"><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.5.5.5.5.1.1.3.2">′</mo><mi id="Sx7.E1.m1.5.5.5.5.1.1.3.3">m</mi></msup><msup id="Sx7.E1.m1.5.5.5.5.1.1.4"><mrow id="Sx7.E1.m1.5.5.5.5.1.1.4.2"><mo stretchy="false" id="Sx7.E1.m1.5.5.5.5.1.1.4.2.1">(</mo><mstyle displaystyle="false" id="Sx7.E1.m1.5.5.5.5.1.1.4.2.2"><mfrac id="Sx7.E1.m1.5.5.5.5.1.1.4.2.2a"><mrow id="Sx7.E1.m1.5.5.5.5.1.1.4.2.2.2"><mn id="Sx7.E1.m1.5.5.5.5.1.1.4.2.2.2.1">1</mn><mo id="Sx7.E1.m1.5.5.5.5.1.1.4.2.2.2.2">−</mo><mi id="Sx7.E1.m1.5.5.5.5.1.1.4.2.2.2.3">ϵ</mi><mo lspace="0em" id="Sx7.E1.m1.5.5.5.5.1.1.4.2.2.2.4">′</mo></mrow><mn id="Sx7.E1.m1.5.5.5.5.1.1.4.2.2.3">2</mn></mfrac></mstyle><mo stretchy="false" id="Sx7.E1.m1.5.5.5.5.1.1.4.2.3">)</mo></mrow><mrow id="Sx7.E1.m1.5.5.5.5.1.1.4.3"><mi id="Sx7.E1.m1.5.5.5.5.1.1.4.3.2">k</mi><mo id="Sx7.E1.m1.5.5.5.5.1.1.4.3.1">+</mo><mi id="Sx7.E1.m1.5.5.5.5.1.1.4.3.3">l</mi></mrow></msup><mo id="Sx7.E1.m1.5.5.5.5.1.1.5">,</mo></mrow></mtd><mtd class="ltx_align_left" columnalign="left" id="Sx7.E1.m1.6.6.6f"><mrow id="Sx7.E1.m1.6.6.6.6.2.1"><mtext id="Sx7.E1.m1.6.6.6.6.2.1.2">if </mtext><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.6.6.6.6.2.1.1">​</mo><msub id="Sx7.E1.m1.6.6.6.6.2.1.3"><mi id="Sx7.E1.m1.6.6.6.6.2.1.3.2">s</mi><mrow id="Sx7.E1.m1.6.6.6.6.2.1.3.3"><mi id="Sx7.E1.m1.6.6.6.6.2.1.3.3.2">i</mi><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.6.6.6.6.2.1.3.3.1">​</mo><mi id="Sx7.E1.m1.6.6.6.6.2.1.3.3.3">j</mi></mrow></msub><mo lspace="0em" rspace="0em" id="Sx7.E1.m1.6.6.6.6.2.1.1a">​</mo><mtext id="Sx7.E1.m1.6.6.6.6.2.1.4"> heterozygous</mtext></mrow></mtd></mtr></mtable></mrow></mrow><annotation encoding="application/x-tex" id="Sx7.E1.m1.12b">\mathbb{L}(\phi_{i},g_{j})=p(n_{ij}^{b}|\phi_{i},g_{j})=P_{m}(n_{ij}^{b},s_{ij},\epsilon)=\begin{cases}{n\choose m}\epsilon^{m}(1-\epsilon)^{n-m},&amp;\text{if }s_{ij}\text{ homozygous}\\
\frac{n!}{k!l!m!}\epsilon\prime^{m}(\frac{1-\epsilon\prime}{2})^{k+l},&amp;\text{if }s_{ij}\text{ heterozygous}\\
\end{cases}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(1)</span></td>
</tr></tbody>
</table>
</div>
<div id="Sx7.SSx7.p7" class="ltx_para">
<p id="Sx7.SSx7.p7.8" class="ltx_p">where <math id="Sx7.SSx7.p7.1.m1.1" class="ltx_Math" alttext="n" display="inline"><semantics id="Sx7.SSx7.p7.1.m1.1a"><mi id="Sx7.SSx7.p7.1.m1.1.1" xref="Sx7.SSx7.p7.1.m1.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p7.1.m1.1b"><ci id="Sx7.SSx7.p7.1.m1.1.1.cmml" xref="Sx7.SSx7.p7.1.m1.1.1">𝑛</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p7.1.m1.1c">n</annotation></semantics></math> is the total number of reads at SNP <math id="Sx7.SSx7.p7.2.m2.1" class="ltx_Math" alttext="i" display="inline"><semantics id="Sx7.SSx7.p7.2.m2.1a"><mi id="Sx7.SSx7.p7.2.m2.1.1" xref="Sx7.SSx7.p7.2.m2.1.1.cmml">i</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p7.2.m2.1b"><ci id="Sx7.SSx7.p7.2.m2.1.1.cmml" xref="Sx7.SSx7.p7.2.m2.1.1">𝑖</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p7.2.m2.1c">i</annotation></semantics></math>; <math id="Sx7.SSx7.p7.3.m3.1" class="ltx_Math" alttext="m" display="inline"><semantics id="Sx7.SSx7.p7.3.m3.1a"><mi id="Sx7.SSx7.p7.3.m3.1.1" xref="Sx7.SSx7.p7.3.m3.1.1.cmml">m</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p7.3.m3.1b"><ci id="Sx7.SSx7.p7.3.m3.1.1.cmml" xref="Sx7.SSx7.p7.3.m3.1.1">𝑚</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p7.3.m3.1c">m</annotation></semantics></math> is the number of observations of bases not in <math id="Sx7.SSx7.p7.4.m4.1" class="ltx_Math" alttext="\sigma_{ij}" display="inline"><semantics id="Sx7.SSx7.p7.4.m4.1a"><msub id="Sx7.SSx7.p7.4.m4.1.1" xref="Sx7.SSx7.p7.4.m4.1.1.cmml"><mi id="Sx7.SSx7.p7.4.m4.1.1.2" xref="Sx7.SSx7.p7.4.m4.1.1.2.cmml">σ</mi><mrow id="Sx7.SSx7.p7.4.m4.1.1.3" xref="Sx7.SSx7.p7.4.m4.1.1.3.cmml"><mi id="Sx7.SSx7.p7.4.m4.1.1.3.2" xref="Sx7.SSx7.p7.4.m4.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p7.4.m4.1.1.3.1" xref="Sx7.SSx7.p7.4.m4.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx7.p7.4.m4.1.1.3.3" xref="Sx7.SSx7.p7.4.m4.1.1.3.3.cmml">j</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p7.4.m4.1b"><apply id="Sx7.SSx7.p7.4.m4.1.1.cmml" xref="Sx7.SSx7.p7.4.m4.1.1"><csymbol cd="ambiguous" id="Sx7.SSx7.p7.4.m4.1.1.1.cmml" xref="Sx7.SSx7.p7.4.m4.1.1">subscript</csymbol><ci id="Sx7.SSx7.p7.4.m4.1.1.2.cmml" xref="Sx7.SSx7.p7.4.m4.1.1.2">𝜎</ci><apply id="Sx7.SSx7.p7.4.m4.1.1.3.cmml" xref="Sx7.SSx7.p7.4.m4.1.1.3"><times id="Sx7.SSx7.p7.4.m4.1.1.3.1.cmml" xref="Sx7.SSx7.p7.4.m4.1.1.3.1"></times><ci id="Sx7.SSx7.p7.4.m4.1.1.3.2.cmml" xref="Sx7.SSx7.p7.4.m4.1.1.3.2">𝑖</ci><ci id="Sx7.SSx7.p7.4.m4.1.1.3.3.cmml" xref="Sx7.SSx7.p7.4.m4.1.1.3.3">𝑗</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p7.4.m4.1c">\sigma_{ij}</annotation></semantics></math> (<em id="Sx7.SSx7.p7.8.1" class="ltx_emph ltx_font_italic">i.e.</em> mismatches); <math id="Sx7.SSx7.p7.5.m5.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx7.p7.5.m5.1a"><mi id="Sx7.SSx7.p7.5.m5.1.1" xref="Sx7.SSx7.p7.5.m5.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p7.5.m5.1b"><ci id="Sx7.SSx7.p7.5.m5.1.1.cmml" xref="Sx7.SSx7.p7.5.m5.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p7.5.m5.1c">k</annotation></semantics></math> and <math id="Sx7.SSx7.p7.6.m6.1" class="ltx_Math" alttext="l" display="inline"><semantics id="Sx7.SSx7.p7.6.m6.1a"><mi id="Sx7.SSx7.p7.6.m6.1.1" xref="Sx7.SSx7.p7.6.m6.1.1.cmml">l</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p7.6.m6.1b"><ci id="Sx7.SSx7.p7.6.m6.1.1.cmml" xref="Sx7.SSx7.p7.6.m6.1.1">𝑙</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p7.6.m6.1c">l</annotation></semantics></math> are the counts for each of the two bases of <math id="Sx7.SSx7.p7.7.m7.1" class="ltx_Math" alttext="\sigma_{ij}" display="inline"><semantics id="Sx7.SSx7.p7.7.m7.1a"><msub id="Sx7.SSx7.p7.7.m7.1.1" xref="Sx7.SSx7.p7.7.m7.1.1.cmml"><mi id="Sx7.SSx7.p7.7.m7.1.1.2" xref="Sx7.SSx7.p7.7.m7.1.1.2.cmml">σ</mi><mrow id="Sx7.SSx7.p7.7.m7.1.1.3" xref="Sx7.SSx7.p7.7.m7.1.1.3.cmml"><mi id="Sx7.SSx7.p7.7.m7.1.1.3.2" xref="Sx7.SSx7.p7.7.m7.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx7.p7.7.m7.1.1.3.1" xref="Sx7.SSx7.p7.7.m7.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx7.p7.7.m7.1.1.3.3" xref="Sx7.SSx7.p7.7.m7.1.1.3.3.cmml">j</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx7.p7.7.m7.1b"><apply id="Sx7.SSx7.p7.7.m7.1.1.cmml" xref="Sx7.SSx7.p7.7.m7.1.1"><csymbol cd="ambiguous" id="Sx7.SSx7.p7.7.m7.1.1.1.cmml" xref="Sx7.SSx7.p7.7.m7.1.1">subscript</csymbol><ci id="Sx7.SSx7.p7.7.m7.1.1.2.cmml" xref="Sx7.SSx7.p7.7.m7.1.1.2">𝜎</ci><apply id="Sx7.SSx7.p7.7.m7.1.1.3.cmml" xref="Sx7.SSx7.p7.7.m7.1.1.3"><times id="Sx7.SSx7.p7.7.m7.1.1.3.1.cmml" xref="Sx7.SSx7.p7.7.m7.1.1.3.1"></times><ci id="Sx7.SSx7.p7.7.m7.1.1.3.2.cmml" xref="Sx7.SSx7.p7.7.m7.1.1.3.2">𝑖</ci><ci id="Sx7.SSx7.p7.7.m7.1.1.3.3.cmml" xref="Sx7.SSx7.p7.7.m7.1.1.3.3">𝑗</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx7.p7.7.m7.1c">\sigma_{ij}</annotation></semantics></math> for heterozygous sites, and <math id="Sx7.SSx7.p7.8.m8.1" class="ltx_math_unparsed" alttext="\epsilon\prime=2\epsilon/3" display="inline"><semantics id="Sx7.SSx7.p7.8.m8.1a"><mrow id="Sx7.SSx7.p7.8.m8.1b"><mi id="Sx7.SSx7.p7.8.m8.1.1">ϵ</mi><mo lspace="0em" rspace="0.1389em" id="Sx7.SSx7.p7.8.m8.1.2">′</mo><mo lspace="0.1389em" id="Sx7.SSx7.p7.8.m8.1.3">=</mo><mn id="Sx7.SSx7.p7.8.m8.1.4">2</mn><mi id="Sx7.SSx7.p7.8.m8.1.5">ϵ</mi><mo id="Sx7.SSx7.p7.8.m8.1.6">/</mo><mn id="Sx7.SSx7.p7.8.m8.1.7">3</mn></mrow><annotation encoding="application/x-tex" id="Sx7.SSx7.p7.8.m8.1c">\epsilon\prime=2\epsilon/3</annotation></semantics></math>.</p>
</div>
</section>
<section id="Sx7.SSx8" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Likelihood maximization</h3>

<div id="Sx7.SSx8.p1" class="ltx_para">
<p id="Sx7.SSx8.p1.3" class="ltx_p">Searching for an optimal choice of SNP phases <math id="Sx7.SSx8.p1.1.m1.1" class="ltx_Math" alttext="\phi_{i}" display="inline"><semantics id="Sx7.SSx8.p1.1.m1.1a"><msub id="Sx7.SSx8.p1.1.m1.1.1" xref="Sx7.SSx8.p1.1.m1.1.1.cmml"><mi id="Sx7.SSx8.p1.1.m1.1.1.2" xref="Sx7.SSx8.p1.1.m1.1.1.2.cmml">ϕ</mi><mi id="Sx7.SSx8.p1.1.m1.1.1.3" xref="Sx7.SSx8.p1.1.m1.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p1.1.m1.1b"><apply id="Sx7.SSx8.p1.1.m1.1.1.cmml" xref="Sx7.SSx8.p1.1.m1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx8.p1.1.m1.1.1.1.cmml" xref="Sx7.SSx8.p1.1.m1.1.1">subscript</csymbol><ci id="Sx7.SSx8.p1.1.m1.1.1.2.cmml" xref="Sx7.SSx8.p1.1.m1.1.1.2">italic-ϕ</ci><ci id="Sx7.SSx8.p1.1.m1.1.1.3.cmml" xref="Sx7.SSx8.p1.1.m1.1.1.3">𝑖</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p1.1.m1.1c">\phi_{i}</annotation></semantics></math> and sample genotypes <math id="Sx7.SSx8.p1.2.m2.1" class="ltx_Math" alttext="g_{j}" display="inline"><semantics id="Sx7.SSx8.p1.2.m2.1a"><msub id="Sx7.SSx8.p1.2.m2.1.1" xref="Sx7.SSx8.p1.2.m2.1.1.cmml"><mi id="Sx7.SSx8.p1.2.m2.1.1.2" xref="Sx7.SSx8.p1.2.m2.1.1.2.cmml">g</mi><mi id="Sx7.SSx8.p1.2.m2.1.1.3" xref="Sx7.SSx8.p1.2.m2.1.1.3.cmml">j</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p1.2.m2.1b"><apply id="Sx7.SSx8.p1.2.m2.1.1.cmml" xref="Sx7.SSx8.p1.2.m2.1.1"><csymbol cd="ambiguous" id="Sx7.SSx8.p1.2.m2.1.1.1.cmml" xref="Sx7.SSx8.p1.2.m2.1.1">subscript</csymbol><ci id="Sx7.SSx8.p1.2.m2.1.1.2.cmml" xref="Sx7.SSx8.p1.2.m2.1.1.2">𝑔</ci><ci id="Sx7.SSx8.p1.2.m2.1.1.3.cmml" xref="Sx7.SSx8.p1.2.m2.1.1.3">𝑗</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p1.2.m2.1c">g_{j}</annotation></semantics></math> is made difficult by the exponential size of the search space: for segregating bi-allelic SNP sites there are 14 possible phases to consider at each SNP site, so for a mapping panel of only 20 siblings and a marker containing only 10 SNPs, there are <math id="Sx7.SSx8.p1.3.m3.1" class="ltx_Math" alttext="4^{20}\times 14^{10}&gt;3\times 10^{23}" display="inline"><semantics id="Sx7.SSx8.p1.3.m3.1a"><mrow id="Sx7.SSx8.p1.3.m3.1.1" xref="Sx7.SSx8.p1.3.m3.1.1.cmml"><mrow id="Sx7.SSx8.p1.3.m3.1.1.2" xref="Sx7.SSx8.p1.3.m3.1.1.2.cmml"><msup id="Sx7.SSx8.p1.3.m3.1.1.2.2" xref="Sx7.SSx8.p1.3.m3.1.1.2.2.cmml"><mn id="Sx7.SSx8.p1.3.m3.1.1.2.2.2" xref="Sx7.SSx8.p1.3.m3.1.1.2.2.2.cmml">4</mn><mn id="Sx7.SSx8.p1.3.m3.1.1.2.2.3" xref="Sx7.SSx8.p1.3.m3.1.1.2.2.3.cmml">20</mn></msup><mo lspace="0.222em" rspace="0.222em" id="Sx7.SSx8.p1.3.m3.1.1.2.1" xref="Sx7.SSx8.p1.3.m3.1.1.2.1.cmml">×</mo><msup id="Sx7.SSx8.p1.3.m3.1.1.2.3" xref="Sx7.SSx8.p1.3.m3.1.1.2.3.cmml"><mn id="Sx7.SSx8.p1.3.m3.1.1.2.3.2" xref="Sx7.SSx8.p1.3.m3.1.1.2.3.2.cmml">14</mn><mn id="Sx7.SSx8.p1.3.m3.1.1.2.3.3" xref="Sx7.SSx8.p1.3.m3.1.1.2.3.3.cmml">10</mn></msup></mrow><mo id="Sx7.SSx8.p1.3.m3.1.1.1" xref="Sx7.SSx8.p1.3.m3.1.1.1.cmml">&gt;</mo><mrow id="Sx7.SSx8.p1.3.m3.1.1.3" xref="Sx7.SSx8.p1.3.m3.1.1.3.cmml"><mn id="Sx7.SSx8.p1.3.m3.1.1.3.2" xref="Sx7.SSx8.p1.3.m3.1.1.3.2.cmml">3</mn><mo lspace="0.222em" rspace="0.222em" id="Sx7.SSx8.p1.3.m3.1.1.3.1" xref="Sx7.SSx8.p1.3.m3.1.1.3.1.cmml">×</mo><msup id="Sx7.SSx8.p1.3.m3.1.1.3.3" xref="Sx7.SSx8.p1.3.m3.1.1.3.3.cmml"><mn id="Sx7.SSx8.p1.3.m3.1.1.3.3.2" xref="Sx7.SSx8.p1.3.m3.1.1.3.3.2.cmml">10</mn><mn id="Sx7.SSx8.p1.3.m3.1.1.3.3.3" xref="Sx7.SSx8.p1.3.m3.1.1.3.3.3.cmml">23</mn></msup></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p1.3.m3.1b"><apply id="Sx7.SSx8.p1.3.m3.1.1.cmml" xref="Sx7.SSx8.p1.3.m3.1.1"><gt id="Sx7.SSx8.p1.3.m3.1.1.1.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.1"></gt><apply id="Sx7.SSx8.p1.3.m3.1.1.2.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2"><times id="Sx7.SSx8.p1.3.m3.1.1.2.1.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.1"></times><apply id="Sx7.SSx8.p1.3.m3.1.1.2.2.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.2"><csymbol cd="ambiguous" id="Sx7.SSx8.p1.3.m3.1.1.2.2.1.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.2">superscript</csymbol><cn type="integer" id="Sx7.SSx8.p1.3.m3.1.1.2.2.2.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.2.2">4</cn><cn type="integer" id="Sx7.SSx8.p1.3.m3.1.1.2.2.3.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.2.3">20</cn></apply><apply id="Sx7.SSx8.p1.3.m3.1.1.2.3.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.3"><csymbol cd="ambiguous" id="Sx7.SSx8.p1.3.m3.1.1.2.3.1.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.3">superscript</csymbol><cn type="integer" id="Sx7.SSx8.p1.3.m3.1.1.2.3.2.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.3.2">14</cn><cn type="integer" id="Sx7.SSx8.p1.3.m3.1.1.2.3.3.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.2.3.3">10</cn></apply></apply><apply id="Sx7.SSx8.p1.3.m3.1.1.3.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.3"><times id="Sx7.SSx8.p1.3.m3.1.1.3.1.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.3.1"></times><cn type="integer" id="Sx7.SSx8.p1.3.m3.1.1.3.2.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.3.2">3</cn><apply id="Sx7.SSx8.p1.3.m3.1.1.3.3.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.3.3"><csymbol cd="ambiguous" id="Sx7.SSx8.p1.3.m3.1.1.3.3.1.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.3.3">superscript</csymbol><cn type="integer" id="Sx7.SSx8.p1.3.m3.1.1.3.3.2.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.3.3.2">10</cn><cn type="integer" id="Sx7.SSx8.p1.3.m3.1.1.3.3.3.cmml" xref="Sx7.SSx8.p1.3.m3.1.1.3.3.3">23</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p1.3.m3.1c">4^{20}\times 14^{10}&gt;3\times 10^{23}</annotation></semantics></math> combinations to consider. In simulation tests, we found that a variant of expectation maximization (EM), an iterative likelihood maximization method can accurately infer a large proportion of marker genotypes.</p>
</div>
<div id="Sx7.SSx8.p2" class="ltx_para">
<p id="Sx7.SSx8.p2.4" class="ltx_p">To initialize the iteration, the parental samples and a randomly selected offspring are, without loss of generality, assigned genotypes <math id="Sx7.SSx8.p2.1.m1.2" class="ltx_Math" alttext="(a,b)" display="inline"><semantics id="Sx7.SSx8.p2.1.m1.2a"><mrow id="Sx7.SSx8.p2.1.m1.2.3.2" xref="Sx7.SSx8.p2.1.m1.2.3.1.cmml"><mo stretchy="false" id="Sx7.SSx8.p2.1.m1.2.3.2.1" xref="Sx7.SSx8.p2.1.m1.2.3.1.cmml">(</mo><mi id="Sx7.SSx8.p2.1.m1.1.1" xref="Sx7.SSx8.p2.1.m1.1.1.cmml">a</mi><mo id="Sx7.SSx8.p2.1.m1.2.3.2.2" xref="Sx7.SSx8.p2.1.m1.2.3.1.cmml">,</mo><mi id="Sx7.SSx8.p2.1.m1.2.2" xref="Sx7.SSx8.p2.1.m1.2.2.cmml">b</mi><mo stretchy="false" id="Sx7.SSx8.p2.1.m1.2.3.2.3" xref="Sx7.SSx8.p2.1.m1.2.3.1.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p2.1.m1.2b"><interval closure="open" id="Sx7.SSx8.p2.1.m1.2.3.1.cmml" xref="Sx7.SSx8.p2.1.m1.2.3.2"><ci id="Sx7.SSx8.p2.1.m1.1.1.cmml" xref="Sx7.SSx8.p2.1.m1.1.1">𝑎</ci><ci id="Sx7.SSx8.p2.1.m1.2.2.cmml" xref="Sx7.SSx8.p2.1.m1.2.2">𝑏</ci></interval></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p2.1.m1.2c">(a,b)</annotation></semantics></math>, <math id="Sx7.SSx8.p2.2.m2.2" class="ltx_Math" alttext="(c,d)" display="inline"><semantics id="Sx7.SSx8.p2.2.m2.2a"><mrow id="Sx7.SSx8.p2.2.m2.2.3.2" xref="Sx7.SSx8.p2.2.m2.2.3.1.cmml"><mo stretchy="false" id="Sx7.SSx8.p2.2.m2.2.3.2.1" xref="Sx7.SSx8.p2.2.m2.2.3.1.cmml">(</mo><mi id="Sx7.SSx8.p2.2.m2.1.1" xref="Sx7.SSx8.p2.2.m2.1.1.cmml">c</mi><mo id="Sx7.SSx8.p2.2.m2.2.3.2.2" xref="Sx7.SSx8.p2.2.m2.2.3.1.cmml">,</mo><mi id="Sx7.SSx8.p2.2.m2.2.2" xref="Sx7.SSx8.p2.2.m2.2.2.cmml">d</mi><mo stretchy="false" id="Sx7.SSx8.p2.2.m2.2.3.2.3" xref="Sx7.SSx8.p2.2.m2.2.3.1.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p2.2.m2.2b"><interval closure="open" id="Sx7.SSx8.p2.2.m2.2.3.1.cmml" xref="Sx7.SSx8.p2.2.m2.2.3.2"><ci id="Sx7.SSx8.p2.2.m2.1.1.cmml" xref="Sx7.SSx8.p2.2.m2.1.1">𝑐</ci><ci id="Sx7.SSx8.p2.2.m2.2.2.cmml" xref="Sx7.SSx8.p2.2.m2.2.2">𝑑</ci></interval></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p2.2.m2.2c">(c,d)</annotation></semantics></math> and <math id="Sx7.SSx8.p2.3.m3.2" class="ltx_Math" alttext="(a,c)" display="inline"><semantics id="Sx7.SSx8.p2.3.m3.2a"><mrow id="Sx7.SSx8.p2.3.m3.2.3.2" xref="Sx7.SSx8.p2.3.m3.2.3.1.cmml"><mo stretchy="false" id="Sx7.SSx8.p2.3.m3.2.3.2.1" xref="Sx7.SSx8.p2.3.m3.2.3.1.cmml">(</mo><mi id="Sx7.SSx8.p2.3.m3.1.1" xref="Sx7.SSx8.p2.3.m3.1.1.cmml">a</mi><mo id="Sx7.SSx8.p2.3.m3.2.3.2.2" xref="Sx7.SSx8.p2.3.m3.2.3.1.cmml">,</mo><mi id="Sx7.SSx8.p2.3.m3.2.2" xref="Sx7.SSx8.p2.3.m3.2.2.cmml">c</mi><mo stretchy="false" id="Sx7.SSx8.p2.3.m3.2.3.2.3" xref="Sx7.SSx8.p2.3.m3.2.3.1.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p2.3.m3.2b"><interval closure="open" id="Sx7.SSx8.p2.3.m3.2.3.1.cmml" xref="Sx7.SSx8.p2.3.m3.2.3.2"><ci id="Sx7.SSx8.p2.3.m3.1.1.cmml" xref="Sx7.SSx8.p2.3.m3.1.1">𝑎</ci><ci id="Sx7.SSx8.p2.3.m3.2.2.cmml" xref="Sx7.SSx8.p2.3.m3.2.2">𝑐</ci></interval></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p2.3.m3.2c">(a,c)</annotation></semantics></math>. At each step, we calculate the conditional probability distributions over the possible SNP phases <math id="Sx7.SSx8.p2.4.m4.1" class="ltx_Math" alttext="p(\phi_{i})" display="inline"><semantics id="Sx7.SSx8.p2.4.m4.1a"><mrow id="Sx7.SSx8.p2.4.m4.1.1" xref="Sx7.SSx8.p2.4.m4.1.1.cmml"><mi id="Sx7.SSx8.p2.4.m4.1.1.3" xref="Sx7.SSx8.p2.4.m4.1.1.3.cmml">p</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx8.p2.4.m4.1.1.2" xref="Sx7.SSx8.p2.4.m4.1.1.2.cmml">​</mo><mrow id="Sx7.SSx8.p2.4.m4.1.1.1.1" xref="Sx7.SSx8.p2.4.m4.1.1.1.1.1.cmml"><mo stretchy="false" id="Sx7.SSx8.p2.4.m4.1.1.1.1.2" xref="Sx7.SSx8.p2.4.m4.1.1.1.1.1.cmml">(</mo><msub id="Sx7.SSx8.p2.4.m4.1.1.1.1.1" xref="Sx7.SSx8.p2.4.m4.1.1.1.1.1.cmml"><mi id="Sx7.SSx8.p2.4.m4.1.1.1.1.1.2" xref="Sx7.SSx8.p2.4.m4.1.1.1.1.1.2.cmml">ϕ</mi><mi id="Sx7.SSx8.p2.4.m4.1.1.1.1.1.3" xref="Sx7.SSx8.p2.4.m4.1.1.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="Sx7.SSx8.p2.4.m4.1.1.1.1.3" xref="Sx7.SSx8.p2.4.m4.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p2.4.m4.1b"><apply id="Sx7.SSx8.p2.4.m4.1.1.cmml" xref="Sx7.SSx8.p2.4.m4.1.1"><times id="Sx7.SSx8.p2.4.m4.1.1.2.cmml" xref="Sx7.SSx8.p2.4.m4.1.1.2"></times><ci id="Sx7.SSx8.p2.4.m4.1.1.3.cmml" xref="Sx7.SSx8.p2.4.m4.1.1.3">𝑝</ci><apply id="Sx7.SSx8.p2.4.m4.1.1.1.1.1.cmml" xref="Sx7.SSx8.p2.4.m4.1.1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx8.p2.4.m4.1.1.1.1.1.1.cmml" xref="Sx7.SSx8.p2.4.m4.1.1.1.1">subscript</csymbol><ci id="Sx7.SSx8.p2.4.m4.1.1.1.1.1.2.cmml" xref="Sx7.SSx8.p2.4.m4.1.1.1.1.1.2">italic-ϕ</ci><ci id="Sx7.SSx8.p2.4.m4.1.1.1.1.1.3.cmml" xref="Sx7.SSx8.p2.4.m4.1.1.1.1.1.3">𝑖</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p2.4.m4.1c">p(\phi_{i})</annotation></semantics></math> given the genotype assignments according to:</p>
</div>
<div id="Sx7.SSx8.p3" class="ltx_para">
<table id="Sx7.E2" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="Sx7.E2.m1.8" class="ltx_Math" alttext="p^{(t)}(\phi_{i})=p(\phi_{i}|\mathbf{g}^{(t)})=\prod_{\sigma\in S}p(n_{i\sigma}^{b}|\phi_{i},\mathbf{g}^{(t)})/(\sum_{k}\prod_{\tau\in S}p(n_{k\tau}^{b}|\phi_{k},\mathbf{g}^{(t)}))" display="block"><semantics id="Sx7.E2.m1.8a"><mrow id="Sx7.E2.m1.8.8" xref="Sx7.E2.m1.8.8.cmml"><mrow id="Sx7.E2.m1.5.5.1" xref="Sx7.E2.m1.5.5.1.cmml"><msup id="Sx7.E2.m1.5.5.1.3" xref="Sx7.E2.m1.5.5.1.3.cmml"><mi id="Sx7.E2.m1.5.5.1.3.2" xref="Sx7.E2.m1.5.5.1.3.2.cmml">p</mi><mrow id="Sx7.E2.m1.1.1.1.3" xref="Sx7.E2.m1.5.5.1.3.cmml"><mo stretchy="false" id="Sx7.E2.m1.1.1.1.3.1" xref="Sx7.E2.m1.5.5.1.3.cmml">(</mo><mi id="Sx7.E2.m1.1.1.1.1" xref="Sx7.E2.m1.1.1.1.1.cmml">t</mi><mo stretchy="false" id="Sx7.E2.m1.1.1.1.3.2" xref="Sx7.E2.m1.5.5.1.3.cmml">)</mo></mrow></msup><mo lspace="0em" rspace="0em" id="Sx7.E2.m1.5.5.1.2" xref="Sx7.E2.m1.5.5.1.2.cmml">​</mo><mrow id="Sx7.E2.m1.5.5.1.1.1" xref="Sx7.E2.m1.5.5.1.1.1.1.cmml"><mo stretchy="false" id="Sx7.E2.m1.5.5.1.1.1.2" xref="Sx7.E2.m1.5.5.1.1.1.1.cmml">(</mo><msub id="Sx7.E2.m1.5.5.1.1.1.1" xref="Sx7.E2.m1.5.5.1.1.1.1.cmml"><mi id="Sx7.E2.m1.5.5.1.1.1.1.2" xref="Sx7.E2.m1.5.5.1.1.1.1.2.cmml">ϕ</mi><mi id="Sx7.E2.m1.5.5.1.1.1.1.3" xref="Sx7.E2.m1.5.5.1.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="Sx7.E2.m1.5.5.1.1.1.3" xref="Sx7.E2.m1.5.5.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="Sx7.E2.m1.8.8.6" xref="Sx7.E2.m1.8.8.6.cmml">=</mo><mrow id="Sx7.E2.m1.6.6.2" xref="Sx7.E2.m1.6.6.2.cmml"><mi id="Sx7.E2.m1.6.6.2.3" xref="Sx7.E2.m1.6.6.2.3.cmml">p</mi><mo lspace="0em" rspace="0em" id="Sx7.E2.m1.6.6.2.2" xref="Sx7.E2.m1.6.6.2.2.cmml">​</mo><mrow id="Sx7.E2.m1.6.6.2.1.1" xref="Sx7.E2.m1.6.6.2.1.1.1.cmml"><mo stretchy="false" id="Sx7.E2.m1.6.6.2.1.1.2" xref="Sx7.E2.m1.6.6.2.1.1.1.cmml">(</mo><mrow id="Sx7.E2.m1.6.6.2.1.1.1" xref="Sx7.E2.m1.6.6.2.1.1.1.cmml"><msub id="Sx7.E2.m1.6.6.2.1.1.1.2" xref="Sx7.E2.m1.6.6.2.1.1.1.2.cmml"><mi id="Sx7.E2.m1.6.6.2.1.1.1.2.2" xref="Sx7.E2.m1.6.6.2.1.1.1.2.2.cmml">ϕ</mi><mi id="Sx7.E2.m1.6.6.2.1.1.1.2.3" xref="Sx7.E2.m1.6.6.2.1.1.1.2.3.cmml">i</mi></msub><mo fence="false" id="Sx7.E2.m1.6.6.2.1.1.1.1" xref="Sx7.E2.m1.6.6.2.1.1.1.1.cmml">|</mo><msup id="Sx7.E2.m1.6.6.2.1.1.1.3" xref="Sx7.E2.m1.6.6.2.1.1.1.3.cmml"><mi id="Sx7.E2.m1.6.6.2.1.1.1.3.2" xref="Sx7.E2.m1.6.6.2.1.1.1.3.2.cmml">𝐠</mi><mrow id="Sx7.E2.m1.2.2.1.3" xref="Sx7.E2.m1.6.6.2.1.1.1.3.cmml"><mo stretchy="false" id="Sx7.E2.m1.2.2.1.3.1" xref="Sx7.E2.m1.6.6.2.1.1.1.3.cmml">(</mo><mi id="Sx7.E2.m1.2.2.1.1" xref="Sx7.E2.m1.2.2.1.1.cmml">t</mi><mo stretchy="false" id="Sx7.E2.m1.2.2.1.3.2" xref="Sx7.E2.m1.6.6.2.1.1.1.3.cmml">)</mo></mrow></msup></mrow><mo stretchy="false" id="Sx7.E2.m1.6.6.2.1.1.3" xref="Sx7.E2.m1.6.6.2.1.1.1.cmml">)</mo></mrow></mrow><mo rspace="0.111em" id="Sx7.E2.m1.8.8.7" xref="Sx7.E2.m1.8.8.7.cmml">=</mo><mrow id="Sx7.E2.m1.8.8.4" xref="Sx7.E2.m1.8.8.4.cmml"><munder id="Sx7.E2.m1.8.8.4.3" xref="Sx7.E2.m1.8.8.4.3.cmml"><mo movablelimits="false" id="Sx7.E2.m1.8.8.4.3.2" xref="Sx7.E2.m1.8.8.4.3.2.cmml">∏</mo><mrow id="Sx7.E2.m1.8.8.4.3.3" xref="Sx7.E2.m1.8.8.4.3.3.cmml"><mi id="Sx7.E2.m1.8.8.4.3.3.2" xref="Sx7.E2.m1.8.8.4.3.3.2.cmml">σ</mi><mo id="Sx7.E2.m1.8.8.4.3.3.1" xref="Sx7.E2.m1.8.8.4.3.3.1.cmml">∈</mo><mi id="Sx7.E2.m1.8.8.4.3.3.3" xref="Sx7.E2.m1.8.8.4.3.3.3.cmml">S</mi></mrow></munder><mrow id="Sx7.E2.m1.8.8.4.2" xref="Sx7.E2.m1.8.8.4.2.cmml"><mrow id="Sx7.E2.m1.7.7.3.1.1" xref="Sx7.E2.m1.7.7.3.1.1.cmml"><mi id="Sx7.E2.m1.7.7.3.1.1.3" xref="Sx7.E2.m1.7.7.3.1.1.3.cmml">p</mi><mo lspace="0em" rspace="0em" id="Sx7.E2.m1.7.7.3.1.1.2" xref="Sx7.E2.m1.7.7.3.1.1.2.cmml">​</mo><mrow id="Sx7.E2.m1.7.7.3.1.1.1.1" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.cmml"><mo stretchy="false" id="Sx7.E2.m1.7.7.3.1.1.1.1.2" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.cmml">(</mo><mrow id="Sx7.E2.m1.7.7.3.1.1.1.1.1" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.cmml"><msubsup id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.cmml"><mi id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.2" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.2.cmml">n</mi><mrow id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.cmml"><mi id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.2" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.1" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.1.cmml">​</mo><mi id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.3" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.3.cmml">σ</mi></mrow><mi id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.3" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.3.cmml">b</mi></msubsup><mo fence="false" id="Sx7.E2.m1.7.7.3.1.1.1.1.1.3" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.3.cmml">|</mo><mrow id="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.3.cmml"><msub id="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.cmml"><mi id="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.2" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.2.cmml">ϕ</mi><mi id="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.3" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.3.cmml">i</mi></msub><mo id="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.3" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.3.cmml">,</mo><msup id="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.cmml"><mi id="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.2" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.2.cmml">𝐠</mi><mrow id="Sx7.E2.m1.3.3.1.3" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.cmml"><mo stretchy="false" id="Sx7.E2.m1.3.3.1.3.1" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.cmml">(</mo><mi id="Sx7.E2.m1.3.3.1.1" xref="Sx7.E2.m1.3.3.1.1.cmml">t</mi><mo stretchy="false" id="Sx7.E2.m1.3.3.1.3.2" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.cmml">)</mo></mrow></msup></mrow></mrow><mo stretchy="false" id="Sx7.E2.m1.7.7.3.1.1.1.1.3" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="Sx7.E2.m1.8.8.4.2.3" xref="Sx7.E2.m1.8.8.4.2.3.cmml">/</mo><mrow id="Sx7.E2.m1.8.8.4.2.2.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.cmml"><mo stretchy="false" id="Sx7.E2.m1.8.8.4.2.2.1.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.cmml">(</mo><mrow id="Sx7.E2.m1.8.8.4.2.2.1.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.cmml"><munder id="Sx7.E2.m1.8.8.4.2.2.1.1.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.2.cmml"><mo lspace="0em" movablelimits="false" rspace="0em" id="Sx7.E2.m1.8.8.4.2.2.1.1.2.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.2.2.cmml">∑</mo><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.2.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.2.3.cmml">k</mi></munder><mrow id="Sx7.E2.m1.8.8.4.2.2.1.1.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.cmml"><munder id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.cmml"><mo movablelimits="false" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.2.cmml">∏</mo><mrow id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.cmml"><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.2.cmml">τ</mi><mo id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.1.cmml">∈</mo><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.3.cmml">S</mi></mrow></munder><mrow id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.cmml"><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.3.cmml">p</mi><mo lspace="0em" rspace="0em" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.2.cmml">​</mo><mrow id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.cmml"><mo stretchy="false" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.cmml">(</mo><mrow id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.cmml"><msubsup id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.cmml"><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.2.cmml">n</mi><mrow id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.cmml"><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.1.cmml">​</mo><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.3.cmml">τ</mi></mrow><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.3.cmml">b</mi></msubsup><mo fence="false" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.3.cmml">|</mo><mrow id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.3.cmml"><msub id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.cmml"><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.2.cmml">ϕ</mi><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.3.cmml">k</mi></msub><mo id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.3.cmml">,</mo><msup id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.cmml"><mi id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.2.cmml">𝐠</mi><mrow id="Sx7.E2.m1.4.4.1.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.cmml"><mo stretchy="false" id="Sx7.E2.m1.4.4.1.3.1" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.cmml">(</mo><mi id="Sx7.E2.m1.4.4.1.1" xref="Sx7.E2.m1.4.4.1.1.cmml">t</mi><mo stretchy="false" id="Sx7.E2.m1.4.4.1.3.2" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.cmml">)</mo></mrow></msup></mrow></mrow><mo stretchy="false" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow></mrow></mrow><mo stretchy="false" id="Sx7.E2.m1.8.8.4.2.2.1.3" xref="Sx7.E2.m1.8.8.4.2.2.1.1.cmml">)</mo></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.E2.m1.8b"><apply id="Sx7.E2.m1.8.8.cmml" xref="Sx7.E2.m1.8.8"><and id="Sx7.E2.m1.8.8a.cmml" xref="Sx7.E2.m1.8.8"></and><apply id="Sx7.E2.m1.8.8b.cmml" xref="Sx7.E2.m1.8.8"><eq id="Sx7.E2.m1.8.8.6.cmml" xref="Sx7.E2.m1.8.8.6"></eq><apply id="Sx7.E2.m1.5.5.1.cmml" xref="Sx7.E2.m1.5.5.1"><times id="Sx7.E2.m1.5.5.1.2.cmml" xref="Sx7.E2.m1.5.5.1.2"></times><apply id="Sx7.E2.m1.5.5.1.3.cmml" xref="Sx7.E2.m1.5.5.1.3"><csymbol cd="ambiguous" id="Sx7.E2.m1.5.5.1.3.1.cmml" xref="Sx7.E2.m1.5.5.1.3">superscript</csymbol><ci id="Sx7.E2.m1.5.5.1.3.2.cmml" xref="Sx7.E2.m1.5.5.1.3.2">𝑝</ci><ci id="Sx7.E2.m1.1.1.1.1.cmml" xref="Sx7.E2.m1.1.1.1.1">𝑡</ci></apply><apply id="Sx7.E2.m1.5.5.1.1.1.1.cmml" xref="Sx7.E2.m1.5.5.1.1.1"><csymbol cd="ambiguous" id="Sx7.E2.m1.5.5.1.1.1.1.1.cmml" xref="Sx7.E2.m1.5.5.1.1.1">subscript</csymbol><ci id="Sx7.E2.m1.5.5.1.1.1.1.2.cmml" xref="Sx7.E2.m1.5.5.1.1.1.1.2">italic-ϕ</ci><ci id="Sx7.E2.m1.5.5.1.1.1.1.3.cmml" xref="Sx7.E2.m1.5.5.1.1.1.1.3">𝑖</ci></apply></apply><apply id="Sx7.E2.m1.6.6.2.cmml" xref="Sx7.E2.m1.6.6.2"><times id="Sx7.E2.m1.6.6.2.2.cmml" xref="Sx7.E2.m1.6.6.2.2"></times><ci id="Sx7.E2.m1.6.6.2.3.cmml" xref="Sx7.E2.m1.6.6.2.3">𝑝</ci><apply id="Sx7.E2.m1.6.6.2.1.1.1.cmml" xref="Sx7.E2.m1.6.6.2.1.1"><csymbol cd="latexml" id="Sx7.E2.m1.6.6.2.1.1.1.1.cmml" xref="Sx7.E2.m1.6.6.2.1.1.1.1">conditional</csymbol><apply id="Sx7.E2.m1.6.6.2.1.1.1.2.cmml" xref="Sx7.E2.m1.6.6.2.1.1.1.2"><csymbol cd="ambiguous" id="Sx7.E2.m1.6.6.2.1.1.1.2.1.cmml" xref="Sx7.E2.m1.6.6.2.1.1.1.2">subscript</csymbol><ci id="Sx7.E2.m1.6.6.2.1.1.1.2.2.cmml" xref="Sx7.E2.m1.6.6.2.1.1.1.2.2">italic-ϕ</ci><ci id="Sx7.E2.m1.6.6.2.1.1.1.2.3.cmml" xref="Sx7.E2.m1.6.6.2.1.1.1.2.3">𝑖</ci></apply><apply id="Sx7.E2.m1.6.6.2.1.1.1.3.cmml" xref="Sx7.E2.m1.6.6.2.1.1.1.3"><csymbol cd="ambiguous" id="Sx7.E2.m1.6.6.2.1.1.1.3.1.cmml" xref="Sx7.E2.m1.6.6.2.1.1.1.3">superscript</csymbol><ci id="Sx7.E2.m1.6.6.2.1.1.1.3.2.cmml" xref="Sx7.E2.m1.6.6.2.1.1.1.3.2">𝐠</ci><ci id="Sx7.E2.m1.2.2.1.1.cmml" xref="Sx7.E2.m1.2.2.1.1">𝑡</ci></apply></apply></apply></apply><apply id="Sx7.E2.m1.8.8c.cmml" xref="Sx7.E2.m1.8.8"><eq id="Sx7.E2.m1.8.8.7.cmml" xref="Sx7.E2.m1.8.8.7"></eq><share href="#Sx7.E2.m1.6.6.2.cmml" id="Sx7.E2.m1.8.8d.cmml" xref="Sx7.E2.m1.8.8"></share><apply id="Sx7.E2.m1.8.8.4.cmml" xref="Sx7.E2.m1.8.8.4"><apply id="Sx7.E2.m1.8.8.4.3.cmml" xref="Sx7.E2.m1.8.8.4.3"><csymbol cd="ambiguous" id="Sx7.E2.m1.8.8.4.3.1.cmml" xref="Sx7.E2.m1.8.8.4.3">subscript</csymbol><csymbol cd="latexml" id="Sx7.E2.m1.8.8.4.3.2.cmml" xref="Sx7.E2.m1.8.8.4.3.2">product</csymbol><apply id="Sx7.E2.m1.8.8.4.3.3.cmml" xref="Sx7.E2.m1.8.8.4.3.3"><in id="Sx7.E2.m1.8.8.4.3.3.1.cmml" xref="Sx7.E2.m1.8.8.4.3.3.1"></in><ci id="Sx7.E2.m1.8.8.4.3.3.2.cmml" xref="Sx7.E2.m1.8.8.4.3.3.2">𝜎</ci><ci id="Sx7.E2.m1.8.8.4.3.3.3.cmml" xref="Sx7.E2.m1.8.8.4.3.3.3">𝑆</ci></apply></apply><apply id="Sx7.E2.m1.8.8.4.2.cmml" xref="Sx7.E2.m1.8.8.4.2"><divide id="Sx7.E2.m1.8.8.4.2.3.cmml" xref="Sx7.E2.m1.8.8.4.2.3"></divide><apply id="Sx7.E2.m1.7.7.3.1.1.cmml" xref="Sx7.E2.m1.7.7.3.1.1"><times id="Sx7.E2.m1.7.7.3.1.1.2.cmml" xref="Sx7.E2.m1.7.7.3.1.1.2"></times><ci id="Sx7.E2.m1.7.7.3.1.1.3.cmml" xref="Sx7.E2.m1.7.7.3.1.1.3">𝑝</ci><apply id="Sx7.E2.m1.7.7.3.1.1.1.1.1.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1"><csymbol cd="latexml" id="Sx7.E2.m1.7.7.3.1.1.1.1.1.3.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.3">conditional</csymbol><apply id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4"><csymbol cd="ambiguous" id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.1.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4">superscript</csymbol><apply id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4"><csymbol cd="ambiguous" id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.1.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4">subscript</csymbol><ci id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.2.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.2">𝑛</ci><apply id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3"><times id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.1.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.1"></times><ci id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.2.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.2">𝑖</ci><ci id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.3.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.2.3.3">𝜎</ci></apply></apply><ci id="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.3.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.4.3">𝑏</ci></apply><list id="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.3.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2"><apply id="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.1.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.2.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.2">italic-ϕ</ci><ci id="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.3.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.1.1.1.3">𝑖</ci></apply><apply id="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2"><csymbol cd="ambiguous" id="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.1.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2">superscript</csymbol><ci id="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.2.cmml" xref="Sx7.E2.m1.7.7.3.1.1.1.1.1.2.2.2.2">𝐠</ci><ci id="Sx7.E2.m1.3.3.1.1.cmml" xref="Sx7.E2.m1.3.3.1.1">𝑡</ci></apply></list></apply></apply><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1"><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.2"><csymbol cd="ambiguous" id="Sx7.E2.m1.8.8.4.2.2.1.1.2.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.2">subscript</csymbol><sum id="Sx7.E2.m1.8.8.4.2.2.1.1.2.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.2.2"></sum><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.2.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.2.3">𝑘</ci></apply><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1"><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2"><csymbol cd="ambiguous" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2">subscript</csymbol><csymbol cd="latexml" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.2">product</csymbol><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3"><in id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.1"></in><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.2">𝜏</ci><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.2.3.3">𝑆</ci></apply></apply><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1"><times id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.2"></times><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.3">𝑝</ci><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1"><csymbol cd="latexml" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.3">conditional</csymbol><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4"><csymbol cd="ambiguous" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4">superscript</csymbol><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4"><csymbol cd="ambiguous" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4">subscript</csymbol><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.2">𝑛</ci><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3"><times id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.1"></times><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.2">𝑘</ci><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.2.3.3">𝜏</ci></apply></apply><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.4.3">𝑏</ci></apply><list id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2"><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.2">italic-ϕ</ci><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.3.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.1.1.1.3">𝑘</ci></apply><apply id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2"><csymbol cd="ambiguous" id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.1.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2">superscript</csymbol><ci id="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.2.cmml" xref="Sx7.E2.m1.8.8.4.2.2.1.1.1.1.1.1.1.2.2.2.2">𝐠</ci><ci id="Sx7.E2.m1.4.4.1.1.cmml" xref="Sx7.E2.m1.4.4.1.1">𝑡</ci></apply></list></apply></apply></apply></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.E2.m1.8c">p^{(t)}(\phi_{i})=p(\phi_{i}|\mathbf{g}^{(t)})=\prod_{\sigma\in S}p(n_{i\sigma}^{b}|\phi_{i},\mathbf{g}^{(t)})/(\sum_{k}\prod_{\tau\in S}p(n_{k\tau}^{b}|\phi_{k},\mathbf{g}^{(t)}))</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(2)</span></td>
</tr></tbody>
</table>
</div>
<div id="Sx7.SSx8.p4" class="ltx_para">
<p id="Sx7.SSx8.p4.8" class="ltx_p">where we have labeled the chosen values for the genotypes <math id="Sx7.SSx8.p4.1.m1.1" class="ltx_Math" alttext="g_{j}" display="inline"><semantics id="Sx7.SSx8.p4.1.m1.1a"><msub id="Sx7.SSx8.p4.1.m1.1.1" xref="Sx7.SSx8.p4.1.m1.1.1.cmml"><mi id="Sx7.SSx8.p4.1.m1.1.1.2" xref="Sx7.SSx8.p4.1.m1.1.1.2.cmml">g</mi><mi id="Sx7.SSx8.p4.1.m1.1.1.3" xref="Sx7.SSx8.p4.1.m1.1.1.3.cmml">j</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p4.1.m1.1b"><apply id="Sx7.SSx8.p4.1.m1.1.1.cmml" xref="Sx7.SSx8.p4.1.m1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx8.p4.1.m1.1.1.1.cmml" xref="Sx7.SSx8.p4.1.m1.1.1">subscript</csymbol><ci id="Sx7.SSx8.p4.1.m1.1.1.2.cmml" xref="Sx7.SSx8.p4.1.m1.1.1.2">𝑔</ci><ci id="Sx7.SSx8.p4.1.m1.1.1.3.cmml" xref="Sx7.SSx8.p4.1.m1.1.1.3">𝑗</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p4.1.m1.1c">g_{j}</annotation></semantics></math> at iteration <math id="Sx7.SSx8.p4.2.m2.1" class="ltx_Math" alttext="t" display="inline"><semantics id="Sx7.SSx8.p4.2.m2.1a"><mi id="Sx7.SSx8.p4.2.m2.1.1" xref="Sx7.SSx8.p4.2.m2.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p4.2.m2.1b"><ci id="Sx7.SSx8.p4.2.m2.1.1.cmml" xref="Sx7.SSx8.p4.2.m2.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p4.2.m2.1c">t</annotation></semantics></math> collectively by <math id="Sx7.SSx8.p4.3.m3.1" class="ltx_Math" alttext="\mathbf{g}^{(t)}" display="inline"><semantics id="Sx7.SSx8.p4.3.m3.1a"><msup id="Sx7.SSx8.p4.3.m3.1.2" xref="Sx7.SSx8.p4.3.m3.1.2.cmml"><mi id="Sx7.SSx8.p4.3.m3.1.2.2" xref="Sx7.SSx8.p4.3.m3.1.2.2.cmml">𝐠</mi><mrow id="Sx7.SSx8.p4.3.m3.1.1.1.3" xref="Sx7.SSx8.p4.3.m3.1.2.cmml"><mo stretchy="false" id="Sx7.SSx8.p4.3.m3.1.1.1.3.1" xref="Sx7.SSx8.p4.3.m3.1.2.cmml">(</mo><mi id="Sx7.SSx8.p4.3.m3.1.1.1.1" xref="Sx7.SSx8.p4.3.m3.1.1.1.1.cmml">t</mi><mo stretchy="false" id="Sx7.SSx8.p4.3.m3.1.1.1.3.2" xref="Sx7.SSx8.p4.3.m3.1.2.cmml">)</mo></mrow></msup><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p4.3.m3.1b"><apply id="Sx7.SSx8.p4.3.m3.1.2.cmml" xref="Sx7.SSx8.p4.3.m3.1.2"><csymbol cd="ambiguous" id="Sx7.SSx8.p4.3.m3.1.2.1.cmml" xref="Sx7.SSx8.p4.3.m3.1.2">superscript</csymbol><ci id="Sx7.SSx8.p4.3.m3.1.2.2.cmml" xref="Sx7.SSx8.p4.3.m3.1.2.2">𝐠</ci><ci id="Sx7.SSx8.p4.3.m3.1.1.1.1.cmml" xref="Sx7.SSx8.p4.3.m3.1.1.1.1">𝑡</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p4.3.m3.1c">\mathbf{g}^{(t)}</annotation></semantics></math>. <math id="Sx7.SSx8.p4.4.m4.1" class="ltx_Math" alttext="n_{i\sigma}^{b}" display="inline"><semantics id="Sx7.SSx8.p4.4.m4.1a"><msubsup id="Sx7.SSx8.p4.4.m4.1.1" xref="Sx7.SSx8.p4.4.m4.1.1.cmml"><mi id="Sx7.SSx8.p4.4.m4.1.1.2.2" xref="Sx7.SSx8.p4.4.m4.1.1.2.2.cmml">n</mi><mrow id="Sx7.SSx8.p4.4.m4.1.1.2.3" xref="Sx7.SSx8.p4.4.m4.1.1.2.3.cmml"><mi id="Sx7.SSx8.p4.4.m4.1.1.2.3.2" xref="Sx7.SSx8.p4.4.m4.1.1.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx8.p4.4.m4.1.1.2.3.1" xref="Sx7.SSx8.p4.4.m4.1.1.2.3.1.cmml">​</mo><mi id="Sx7.SSx8.p4.4.m4.1.1.2.3.3" xref="Sx7.SSx8.p4.4.m4.1.1.2.3.3.cmml">σ</mi></mrow><mi id="Sx7.SSx8.p4.4.m4.1.1.3" xref="Sx7.SSx8.p4.4.m4.1.1.3.cmml">b</mi></msubsup><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p4.4.m4.1b"><apply id="Sx7.SSx8.p4.4.m4.1.1.cmml" xref="Sx7.SSx8.p4.4.m4.1.1"><csymbol cd="ambiguous" id="Sx7.SSx8.p4.4.m4.1.1.1.cmml" xref="Sx7.SSx8.p4.4.m4.1.1">superscript</csymbol><apply id="Sx7.SSx8.p4.4.m4.1.1.2.cmml" xref="Sx7.SSx8.p4.4.m4.1.1"><csymbol cd="ambiguous" id="Sx7.SSx8.p4.4.m4.1.1.2.1.cmml" xref="Sx7.SSx8.p4.4.m4.1.1">subscript</csymbol><ci id="Sx7.SSx8.p4.4.m4.1.1.2.2.cmml" xref="Sx7.SSx8.p4.4.m4.1.1.2.2">𝑛</ci><apply id="Sx7.SSx8.p4.4.m4.1.1.2.3.cmml" xref="Sx7.SSx8.p4.4.m4.1.1.2.3"><times id="Sx7.SSx8.p4.4.m4.1.1.2.3.1.cmml" xref="Sx7.SSx8.p4.4.m4.1.1.2.3.1"></times><ci id="Sx7.SSx8.p4.4.m4.1.1.2.3.2.cmml" xref="Sx7.SSx8.p4.4.m4.1.1.2.3.2">𝑖</ci><ci id="Sx7.SSx8.p4.4.m4.1.1.2.3.3.cmml" xref="Sx7.SSx8.p4.4.m4.1.1.2.3.3">𝜎</ci></apply></apply><ci id="Sx7.SSx8.p4.4.m4.1.1.3.cmml" xref="Sx7.SSx8.p4.4.m4.1.1.3">𝑏</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p4.4.m4.1c">n_{i\sigma}^{b}</annotation></semantics></math> is the combined total number of observations of base <math id="Sx7.SSx8.p4.5.m5.1" class="ltx_Math" alttext="b" display="inline"><semantics id="Sx7.SSx8.p4.5.m5.1a"><mi id="Sx7.SSx8.p4.5.m5.1.1" xref="Sx7.SSx8.p4.5.m5.1.1.cmml">b</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p4.5.m5.1b"><ci id="Sx7.SSx8.p4.5.m5.1.1.cmml" xref="Sx7.SSx8.p4.5.m5.1.1">𝑏</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p4.5.m5.1c">b</annotation></semantics></math> at polymorphic SNP <math id="Sx7.SSx8.p4.6.m6.1" class="ltx_Math" alttext="i" display="inline"><semantics id="Sx7.SSx8.p4.6.m6.1a"><mi id="Sx7.SSx8.p4.6.m6.1.1" xref="Sx7.SSx8.p4.6.m6.1.1.cmml">i</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p4.6.m6.1b"><ci id="Sx7.SSx8.p4.6.m6.1.1.cmml" xref="Sx7.SSx8.p4.6.m6.1.1">𝑖</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p4.6.m6.1c">i</annotation></semantics></math> for all samples included at iteration step <math id="Sx7.SSx8.p4.7.m7.1" class="ltx_Math" alttext="t" display="inline"><semantics id="Sx7.SSx8.p4.7.m7.1a"><mi id="Sx7.SSx8.p4.7.m7.1.1" xref="Sx7.SSx8.p4.7.m7.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p4.7.m7.1b"><ci id="Sx7.SSx8.p4.7.m7.1.1.cmml" xref="Sx7.SSx8.p4.7.m7.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p4.7.m7.1c">t</annotation></semantics></math> which have genotype <math id="Sx7.SSx8.p4.8.m8.2" class="ltx_Math" alttext="s(\phi_{i},g_{j})=\sigma" display="inline"><semantics id="Sx7.SSx8.p4.8.m8.2a"><mrow id="Sx7.SSx8.p4.8.m8.2.2" xref="Sx7.SSx8.p4.8.m8.2.2.cmml"><mrow id="Sx7.SSx8.p4.8.m8.2.2.2" xref="Sx7.SSx8.p4.8.m8.2.2.2.cmml"><mi id="Sx7.SSx8.p4.8.m8.2.2.2.4" xref="Sx7.SSx8.p4.8.m8.2.2.2.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx8.p4.8.m8.2.2.2.3" xref="Sx7.SSx8.p4.8.m8.2.2.2.3.cmml">​</mo><mrow id="Sx7.SSx8.p4.8.m8.2.2.2.2.2" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.3.cmml"><mo stretchy="false" id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.3" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.3.cmml">(</mo><msub id="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1" xref="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.cmml"><mi id="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.2" xref="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.2.cmml">ϕ</mi><mi id="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.3" xref="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.3.cmml">i</mi></msub><mo id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.4" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.3.cmml">,</mo><msub id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.cmml"><mi id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.2" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.2.cmml">g</mi><mi id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.3" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.3.cmml">j</mi></msub><mo stretchy="false" id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.5" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.3.cmml">)</mo></mrow></mrow><mo id="Sx7.SSx8.p4.8.m8.2.2.3" xref="Sx7.SSx8.p4.8.m8.2.2.3.cmml">=</mo><mi id="Sx7.SSx8.p4.8.m8.2.2.4" xref="Sx7.SSx8.p4.8.m8.2.2.4.cmml">σ</mi></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p4.8.m8.2b"><apply id="Sx7.SSx8.p4.8.m8.2.2.cmml" xref="Sx7.SSx8.p4.8.m8.2.2"><eq id="Sx7.SSx8.p4.8.m8.2.2.3.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.3"></eq><apply id="Sx7.SSx8.p4.8.m8.2.2.2.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.2"><times id="Sx7.SSx8.p4.8.m8.2.2.2.3.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.2.3"></times><ci id="Sx7.SSx8.p4.8.m8.2.2.2.4.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.2.4">𝑠</ci><interval closure="open" id="Sx7.SSx8.p4.8.m8.2.2.2.2.3.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.2"><apply id="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.cmml" xref="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.1.cmml" xref="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1">subscript</csymbol><ci id="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.2.cmml" xref="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.2">italic-ϕ</ci><ci id="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.3.cmml" xref="Sx7.SSx8.p4.8.m8.1.1.1.1.1.1.3">𝑖</ci></apply><apply id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2"><csymbol cd="ambiguous" id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.1.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2">subscript</csymbol><ci id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.2.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.2">𝑔</ci><ci id="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.3.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.2.2.2.2.3">𝑗</ci></apply></interval></apply><ci id="Sx7.SSx8.p4.8.m8.2.2.4.cmml" xref="Sx7.SSx8.p4.8.m8.2.2.4">𝜎</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p4.8.m8.2c">s(\phi_{i},g_{j})=\sigma</annotation></semantics></math>.</p>
</div>
<div id="Sx7.SSx8.p5" class="ltx_para">
<p id="Sx7.SSx8.p5.2" class="ltx_p">On each iteration until all samples have been included, a randomly selected sample is added to the set after calculating
<math id="Sx7.SSx8.p5.1.m1.2" class="ltx_Math" alttext="p^{(t)}(\phi_{i})" display="inline"><semantics id="Sx7.SSx8.p5.1.m1.2a"><mrow id="Sx7.SSx8.p5.1.m1.2.2" xref="Sx7.SSx8.p5.1.m1.2.2.cmml"><msup id="Sx7.SSx8.p5.1.m1.2.2.3" xref="Sx7.SSx8.p5.1.m1.2.2.3.cmml"><mi id="Sx7.SSx8.p5.1.m1.2.2.3.2" xref="Sx7.SSx8.p5.1.m1.2.2.3.2.cmml">p</mi><mrow id="Sx7.SSx8.p5.1.m1.1.1.1.3" xref="Sx7.SSx8.p5.1.m1.2.2.3.cmml"><mo stretchy="false" id="Sx7.SSx8.p5.1.m1.1.1.1.3.1" xref="Sx7.SSx8.p5.1.m1.2.2.3.cmml">(</mo><mi id="Sx7.SSx8.p5.1.m1.1.1.1.1" xref="Sx7.SSx8.p5.1.m1.1.1.1.1.cmml">t</mi><mo stretchy="false" id="Sx7.SSx8.p5.1.m1.1.1.1.3.2" xref="Sx7.SSx8.p5.1.m1.2.2.3.cmml">)</mo></mrow></msup><mo lspace="0em" rspace="0em" id="Sx7.SSx8.p5.1.m1.2.2.2" xref="Sx7.SSx8.p5.1.m1.2.2.2.cmml">​</mo><mrow id="Sx7.SSx8.p5.1.m1.2.2.1.1" xref="Sx7.SSx8.p5.1.m1.2.2.1.1.1.cmml"><mo stretchy="false" id="Sx7.SSx8.p5.1.m1.2.2.1.1.2" xref="Sx7.SSx8.p5.1.m1.2.2.1.1.1.cmml">(</mo><msub id="Sx7.SSx8.p5.1.m1.2.2.1.1.1" xref="Sx7.SSx8.p5.1.m1.2.2.1.1.1.cmml"><mi id="Sx7.SSx8.p5.1.m1.2.2.1.1.1.2" xref="Sx7.SSx8.p5.1.m1.2.2.1.1.1.2.cmml">ϕ</mi><mi id="Sx7.SSx8.p5.1.m1.2.2.1.1.1.3" xref="Sx7.SSx8.p5.1.m1.2.2.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="Sx7.SSx8.p5.1.m1.2.2.1.1.3" xref="Sx7.SSx8.p5.1.m1.2.2.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p5.1.m1.2b"><apply id="Sx7.SSx8.p5.1.m1.2.2.cmml" xref="Sx7.SSx8.p5.1.m1.2.2"><times id="Sx7.SSx8.p5.1.m1.2.2.2.cmml" xref="Sx7.SSx8.p5.1.m1.2.2.2"></times><apply id="Sx7.SSx8.p5.1.m1.2.2.3.cmml" xref="Sx7.SSx8.p5.1.m1.2.2.3"><csymbol cd="ambiguous" id="Sx7.SSx8.p5.1.m1.2.2.3.1.cmml" xref="Sx7.SSx8.p5.1.m1.2.2.3">superscript</csymbol><ci id="Sx7.SSx8.p5.1.m1.2.2.3.2.cmml" xref="Sx7.SSx8.p5.1.m1.2.2.3.2">𝑝</ci><ci id="Sx7.SSx8.p5.1.m1.1.1.1.1.cmml" xref="Sx7.SSx8.p5.1.m1.1.1.1.1">𝑡</ci></apply><apply id="Sx7.SSx8.p5.1.m1.2.2.1.1.1.cmml" xref="Sx7.SSx8.p5.1.m1.2.2.1.1"><csymbol cd="ambiguous" id="Sx7.SSx8.p5.1.m1.2.2.1.1.1.1.cmml" xref="Sx7.SSx8.p5.1.m1.2.2.1.1">subscript</csymbol><ci id="Sx7.SSx8.p5.1.m1.2.2.1.1.1.2.cmml" xref="Sx7.SSx8.p5.1.m1.2.2.1.1.1.2">italic-ϕ</ci><ci id="Sx7.SSx8.p5.1.m1.2.2.1.1.1.3.cmml" xref="Sx7.SSx8.p5.1.m1.2.2.1.1.1.3">𝑖</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p5.1.m1.2c">p^{(t)}(\phi_{i})</annotation></semantics></math>. Then the next set of genotype assignments <math id="Sx7.SSx8.p5.2.m2.1" class="ltx_Math" alttext="\mathbf{g}^{(t+1)}" display="inline"><semantics id="Sx7.SSx8.p5.2.m2.1a"><msup id="Sx7.SSx8.p5.2.m2.1.2" xref="Sx7.SSx8.p5.2.m2.1.2.cmml"><mi id="Sx7.SSx8.p5.2.m2.1.2.2" xref="Sx7.SSx8.p5.2.m2.1.2.2.cmml">𝐠</mi><mrow id="Sx7.SSx8.p5.2.m2.1.1.1.1" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.cmml"><mo stretchy="false" id="Sx7.SSx8.p5.2.m2.1.1.1.1.2" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.cmml">(</mo><mrow id="Sx7.SSx8.p5.2.m2.1.1.1.1.1" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.cmml"><mi id="Sx7.SSx8.p5.2.m2.1.1.1.1.1.2" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.2.cmml">t</mi><mo id="Sx7.SSx8.p5.2.m2.1.1.1.1.1.1" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.1.cmml">+</mo><mn id="Sx7.SSx8.p5.2.m2.1.1.1.1.1.3" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.3.cmml">1</mn></mrow><mo stretchy="false" id="Sx7.SSx8.p5.2.m2.1.1.1.1.3" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.cmml">)</mo></mrow></msup><annotation-xml encoding="MathML-Content" id="Sx7.SSx8.p5.2.m2.1b"><apply id="Sx7.SSx8.p5.2.m2.1.2.cmml" xref="Sx7.SSx8.p5.2.m2.1.2"><csymbol cd="ambiguous" id="Sx7.SSx8.p5.2.m2.1.2.1.cmml" xref="Sx7.SSx8.p5.2.m2.1.2">superscript</csymbol><ci id="Sx7.SSx8.p5.2.m2.1.2.2.cmml" xref="Sx7.SSx8.p5.2.m2.1.2.2">𝐠</ci><apply id="Sx7.SSx8.p5.2.m2.1.1.1.1.1.cmml" xref="Sx7.SSx8.p5.2.m2.1.1.1.1"><plus id="Sx7.SSx8.p5.2.m2.1.1.1.1.1.1.cmml" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.1"></plus><ci id="Sx7.SSx8.p5.2.m2.1.1.1.1.1.2.cmml" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.2">𝑡</ci><cn type="integer" id="Sx7.SSx8.p5.2.m2.1.1.1.1.1.3.cmml" xref="Sx7.SSx8.p5.2.m2.1.1.1.1.1.3">1</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx8.p5.2.m2.1c">\mathbf{g}^{(t+1)}</annotation></semantics></math> are determined by
choosing those that maximize the expected value of the log likelihood:</p>
</div>
<div id="Sx7.SSx8.p6" class="ltx_para">
<table id="Sx7.E3" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="Sx7.E3.m1.7" class="ltx_Math" alttext="\mathbb{E}_{\phi|n,g_{j}}[\log L(g_{j};n,\phi)]=\sum_{i}p(\phi_{i})\log p(n_{ij}^{b}|g_{j},\phi_{i})" display="block"><semantics id="Sx7.E3.m1.7a"><mrow id="Sx7.E3.m1.7.7" xref="Sx7.E3.m1.7.7.cmml"><mrow id="Sx7.E3.m1.5.5.1" xref="Sx7.E3.m1.5.5.1.cmml"><msub id="Sx7.E3.m1.5.5.1.3" xref="Sx7.E3.m1.5.5.1.3.cmml"><mi id="Sx7.E3.m1.5.5.1.3.2" xref="Sx7.E3.m1.5.5.1.3.2.cmml">𝔼</mi><mrow id="Sx7.E3.m1.2.2.2" xref="Sx7.E3.m1.2.2.2.cmml"><mi id="Sx7.E3.m1.2.2.2.4" xref="Sx7.E3.m1.2.2.2.4.cmml">ϕ</mi><mo fence="false" id="Sx7.E3.m1.2.2.2.3" xref="Sx7.E3.m1.2.2.2.3.cmml">|</mo><mrow id="Sx7.E3.m1.2.2.2.2.1" xref="Sx7.E3.m1.2.2.2.2.2.cmml"><mi id="Sx7.E3.m1.1.1.1.1" xref="Sx7.E3.m1.1.1.1.1.cmml">n</mi><mo id="Sx7.E3.m1.2.2.2.2.1.2" xref="Sx7.E3.m1.2.2.2.2.2.cmml">,</mo><msub id="Sx7.E3.m1.2.2.2.2.1.1" xref="Sx7.E3.m1.2.2.2.2.1.1.cmml"><mi id="Sx7.E3.m1.2.2.2.2.1.1.2" xref="Sx7.E3.m1.2.2.2.2.1.1.2.cmml">g</mi><mi id="Sx7.E3.m1.2.2.2.2.1.1.3" xref="Sx7.E3.m1.2.2.2.2.1.1.3.cmml">j</mi></msub></mrow></mrow></msub><mo lspace="0em" rspace="0em" id="Sx7.E3.m1.5.5.1.2" xref="Sx7.E3.m1.5.5.1.2.cmml">​</mo><mrow id="Sx7.E3.m1.5.5.1.1.1" xref="Sx7.E3.m1.5.5.1.1.2.cmml"><mo stretchy="false" id="Sx7.E3.m1.5.5.1.1.1.2" xref="Sx7.E3.m1.5.5.1.1.2.1.cmml">[</mo><mrow id="Sx7.E3.m1.5.5.1.1.1.1" xref="Sx7.E3.m1.5.5.1.1.1.1.cmml"><mrow id="Sx7.E3.m1.5.5.1.1.1.1.3" xref="Sx7.E3.m1.5.5.1.1.1.1.3.cmml"><mi id="Sx7.E3.m1.5.5.1.1.1.1.3.1" xref="Sx7.E3.m1.5.5.1.1.1.1.3.1.cmml">log</mi><mo lspace="0.167em" id="Sx7.E3.m1.5.5.1.1.1.1.3a" xref="Sx7.E3.m1.5.5.1.1.1.1.3.cmml">⁡</mo><mi id="Sx7.E3.m1.5.5.1.1.1.1.3.2" xref="Sx7.E3.m1.5.5.1.1.1.1.3.2.cmml">L</mi></mrow><mo lspace="0em" rspace="0em" id="Sx7.E3.m1.5.5.1.1.1.1.2" xref="Sx7.E3.m1.5.5.1.1.1.1.2.cmml">​</mo><mrow id="Sx7.E3.m1.5.5.1.1.1.1.1.1" xref="Sx7.E3.m1.5.5.1.1.1.1.1.2.cmml"><mo stretchy="false" id="Sx7.E3.m1.5.5.1.1.1.1.1.1.2" xref="Sx7.E3.m1.5.5.1.1.1.1.1.2.cmml">(</mo><msub id="Sx7.E3.m1.5.5.1.1.1.1.1.1.1" xref="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.cmml"><mi id="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.2" xref="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.2.cmml">g</mi><mi id="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.3" xref="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.3.cmml">j</mi></msub><mo id="Sx7.E3.m1.5.5.1.1.1.1.1.1.3" xref="Sx7.E3.m1.5.5.1.1.1.1.1.2.cmml">;</mo><mi id="Sx7.E3.m1.3.3" xref="Sx7.E3.m1.3.3.cmml">n</mi><mo id="Sx7.E3.m1.5.5.1.1.1.1.1.1.4" xref="Sx7.E3.m1.5.5.1.1.1.1.1.2.cmml">,</mo><mi id="Sx7.E3.m1.4.4" xref="Sx7.E3.m1.4.4.cmml">ϕ</mi><mo stretchy="false" id="Sx7.E3.m1.5.5.1.1.1.1.1.1.5" xref="Sx7.E3.m1.5.5.1.1.1.1.1.2.cmml">)</mo></mrow></mrow><mo stretchy="false" id="Sx7.E3.m1.5.5.1.1.1.3" xref="Sx7.E3.m1.5.5.1.1.2.1.cmml">]</mo></mrow></mrow><mo rspace="0.111em" id="Sx7.E3.m1.7.7.4" xref="Sx7.E3.m1.7.7.4.cmml">=</mo><mrow id="Sx7.E3.m1.7.7.3" xref="Sx7.E3.m1.7.7.3.cmml"><munder id="Sx7.E3.m1.7.7.3.3" xref="Sx7.E3.m1.7.7.3.3.cmml"><mo movablelimits="false" id="Sx7.E3.m1.7.7.3.3.2" xref="Sx7.E3.m1.7.7.3.3.2.cmml">∑</mo><mi id="Sx7.E3.m1.7.7.3.3.3" xref="Sx7.E3.m1.7.7.3.3.3.cmml">i</mi></munder><mrow id="Sx7.E3.m1.7.7.3.2" xref="Sx7.E3.m1.7.7.3.2.cmml"><mi id="Sx7.E3.m1.7.7.3.2.4" xref="Sx7.E3.m1.7.7.3.2.4.cmml">p</mi><mo lspace="0em" rspace="0em" id="Sx7.E3.m1.7.7.3.2.3" xref="Sx7.E3.m1.7.7.3.2.3.cmml">​</mo><mrow id="Sx7.E3.m1.6.6.2.1.1.1" xref="Sx7.E3.m1.6.6.2.1.1.1.1.cmml"><mo stretchy="false" id="Sx7.E3.m1.6.6.2.1.1.1.2" xref="Sx7.E3.m1.6.6.2.1.1.1.1.cmml">(</mo><msub id="Sx7.E3.m1.6.6.2.1.1.1.1" xref="Sx7.E3.m1.6.6.2.1.1.1.1.cmml"><mi id="Sx7.E3.m1.6.6.2.1.1.1.1.2" xref="Sx7.E3.m1.6.6.2.1.1.1.1.2.cmml">ϕ</mi><mi id="Sx7.E3.m1.6.6.2.1.1.1.1.3" xref="Sx7.E3.m1.6.6.2.1.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="Sx7.E3.m1.6.6.2.1.1.1.3" xref="Sx7.E3.m1.6.6.2.1.1.1.1.cmml">)</mo></mrow><mo lspace="0.167em" rspace="0em" id="Sx7.E3.m1.7.7.3.2.3a" xref="Sx7.E3.m1.7.7.3.2.3.cmml">​</mo><mrow id="Sx7.E3.m1.7.7.3.2.5" xref="Sx7.E3.m1.7.7.3.2.5.cmml"><mi id="Sx7.E3.m1.7.7.3.2.5.1" xref="Sx7.E3.m1.7.7.3.2.5.1.cmml">log</mi><mo lspace="0.167em" id="Sx7.E3.m1.7.7.3.2.5a" xref="Sx7.E3.m1.7.7.3.2.5.cmml">⁡</mo><mi id="Sx7.E3.m1.7.7.3.2.5.2" xref="Sx7.E3.m1.7.7.3.2.5.2.cmml">p</mi></mrow><mo lspace="0em" rspace="0em" id="Sx7.E3.m1.7.7.3.2.3b" xref="Sx7.E3.m1.7.7.3.2.3.cmml">​</mo><mrow id="Sx7.E3.m1.7.7.3.2.2.1" xref="Sx7.E3.m1.7.7.3.2.2.1.1.cmml"><mo stretchy="false" id="Sx7.E3.m1.7.7.3.2.2.1.2" xref="Sx7.E3.m1.7.7.3.2.2.1.1.cmml">(</mo><mrow id="Sx7.E3.m1.7.7.3.2.2.1.1" xref="Sx7.E3.m1.7.7.3.2.2.1.1.cmml"><msubsup id="Sx7.E3.m1.7.7.3.2.2.1.1.4" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.cmml"><mi id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.2" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.2.cmml">n</mi><mrow id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.cmml"><mi id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.2" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.1" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.1.cmml">​</mo><mi id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.3" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.3.cmml">j</mi></mrow><mi id="Sx7.E3.m1.7.7.3.2.2.1.1.4.3" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.3.cmml">b</mi></msubsup><mo fence="false" id="Sx7.E3.m1.7.7.3.2.2.1.1.3" xref="Sx7.E3.m1.7.7.3.2.2.1.1.3.cmml">|</mo><mrow id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.3.cmml"><msub id="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1" xref="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.cmml"><mi id="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.2" xref="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.2.cmml">g</mi><mi id="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.3" xref="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.3.cmml">j</mi></msub><mo id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.3" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.3.cmml">,</mo><msub id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.cmml"><mi id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.2" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.2.cmml">ϕ</mi><mi id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.3" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.3.cmml">i</mi></msub></mrow></mrow><mo stretchy="false" id="Sx7.E3.m1.7.7.3.2.2.1.3" xref="Sx7.E3.m1.7.7.3.2.2.1.1.cmml">)</mo></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.E3.m1.7b"><apply id="Sx7.E3.m1.7.7.cmml" xref="Sx7.E3.m1.7.7"><eq id="Sx7.E3.m1.7.7.4.cmml" xref="Sx7.E3.m1.7.7.4"></eq><apply id="Sx7.E3.m1.5.5.1.cmml" xref="Sx7.E3.m1.5.5.1"><times id="Sx7.E3.m1.5.5.1.2.cmml" xref="Sx7.E3.m1.5.5.1.2"></times><apply id="Sx7.E3.m1.5.5.1.3.cmml" xref="Sx7.E3.m1.5.5.1.3"><csymbol cd="ambiguous" id="Sx7.E3.m1.5.5.1.3.1.cmml" xref="Sx7.E3.m1.5.5.1.3">subscript</csymbol><ci id="Sx7.E3.m1.5.5.1.3.2.cmml" xref="Sx7.E3.m1.5.5.1.3.2">𝔼</ci><apply id="Sx7.E3.m1.2.2.2.cmml" xref="Sx7.E3.m1.2.2.2"><csymbol cd="latexml" id="Sx7.E3.m1.2.2.2.3.cmml" xref="Sx7.E3.m1.2.2.2.3">conditional</csymbol><ci id="Sx7.E3.m1.2.2.2.4.cmml" xref="Sx7.E3.m1.2.2.2.4">italic-ϕ</ci><list id="Sx7.E3.m1.2.2.2.2.2.cmml" xref="Sx7.E3.m1.2.2.2.2.1"><ci id="Sx7.E3.m1.1.1.1.1.cmml" xref="Sx7.E3.m1.1.1.1.1">𝑛</ci><apply id="Sx7.E3.m1.2.2.2.2.1.1.cmml" xref="Sx7.E3.m1.2.2.2.2.1.1"><csymbol cd="ambiguous" id="Sx7.E3.m1.2.2.2.2.1.1.1.cmml" xref="Sx7.E3.m1.2.2.2.2.1.1">subscript</csymbol><ci id="Sx7.E3.m1.2.2.2.2.1.1.2.cmml" xref="Sx7.E3.m1.2.2.2.2.1.1.2">𝑔</ci><ci id="Sx7.E3.m1.2.2.2.2.1.1.3.cmml" xref="Sx7.E3.m1.2.2.2.2.1.1.3">𝑗</ci></apply></list></apply></apply><apply id="Sx7.E3.m1.5.5.1.1.2.cmml" xref="Sx7.E3.m1.5.5.1.1.1"><csymbol cd="latexml" id="Sx7.E3.m1.5.5.1.1.2.1.cmml" xref="Sx7.E3.m1.5.5.1.1.1.2">delimited-[]</csymbol><apply id="Sx7.E3.m1.5.5.1.1.1.1.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1"><times id="Sx7.E3.m1.5.5.1.1.1.1.2.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.2"></times><apply id="Sx7.E3.m1.5.5.1.1.1.1.3.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.3"><log id="Sx7.E3.m1.5.5.1.1.1.1.3.1.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.3.1"></log><ci id="Sx7.E3.m1.5.5.1.1.1.1.3.2.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.3.2">𝐿</ci></apply><list id="Sx7.E3.m1.5.5.1.1.1.1.1.2.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.1.1"><apply id="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.1.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.1.1.1">subscript</csymbol><ci id="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.2.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.2">𝑔</ci><ci id="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.3.cmml" xref="Sx7.E3.m1.5.5.1.1.1.1.1.1.1.3">𝑗</ci></apply><ci id="Sx7.E3.m1.3.3.cmml" xref="Sx7.E3.m1.3.3">𝑛</ci><ci id="Sx7.E3.m1.4.4.cmml" xref="Sx7.E3.m1.4.4">italic-ϕ</ci></list></apply></apply></apply><apply id="Sx7.E3.m1.7.7.3.cmml" xref="Sx7.E3.m1.7.7.3"><apply id="Sx7.E3.m1.7.7.3.3.cmml" xref="Sx7.E3.m1.7.7.3.3"><csymbol cd="ambiguous" id="Sx7.E3.m1.7.7.3.3.1.cmml" xref="Sx7.E3.m1.7.7.3.3">subscript</csymbol><sum id="Sx7.E3.m1.7.7.3.3.2.cmml" xref="Sx7.E3.m1.7.7.3.3.2"></sum><ci id="Sx7.E3.m1.7.7.3.3.3.cmml" xref="Sx7.E3.m1.7.7.3.3.3">𝑖</ci></apply><apply id="Sx7.E3.m1.7.7.3.2.cmml" xref="Sx7.E3.m1.7.7.3.2"><times id="Sx7.E3.m1.7.7.3.2.3.cmml" xref="Sx7.E3.m1.7.7.3.2.3"></times><ci id="Sx7.E3.m1.7.7.3.2.4.cmml" xref="Sx7.E3.m1.7.7.3.2.4">𝑝</ci><apply id="Sx7.E3.m1.6.6.2.1.1.1.1.cmml" xref="Sx7.E3.m1.6.6.2.1.1.1"><csymbol cd="ambiguous" id="Sx7.E3.m1.6.6.2.1.1.1.1.1.cmml" xref="Sx7.E3.m1.6.6.2.1.1.1">subscript</csymbol><ci id="Sx7.E3.m1.6.6.2.1.1.1.1.2.cmml" xref="Sx7.E3.m1.6.6.2.1.1.1.1.2">italic-ϕ</ci><ci id="Sx7.E3.m1.6.6.2.1.1.1.1.3.cmml" xref="Sx7.E3.m1.6.6.2.1.1.1.1.3">𝑖</ci></apply><apply id="Sx7.E3.m1.7.7.3.2.5.cmml" xref="Sx7.E3.m1.7.7.3.2.5"><log id="Sx7.E3.m1.7.7.3.2.5.1.cmml" xref="Sx7.E3.m1.7.7.3.2.5.1"></log><ci id="Sx7.E3.m1.7.7.3.2.5.2.cmml" xref="Sx7.E3.m1.7.7.3.2.5.2">𝑝</ci></apply><apply id="Sx7.E3.m1.7.7.3.2.2.1.1.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1"><csymbol cd="latexml" id="Sx7.E3.m1.7.7.3.2.2.1.1.3.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.3">conditional</csymbol><apply id="Sx7.E3.m1.7.7.3.2.2.1.1.4.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4"><csymbol cd="ambiguous" id="Sx7.E3.m1.7.7.3.2.2.1.1.4.1.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4">superscript</csymbol><apply id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4"><csymbol cd="ambiguous" id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.1.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4">subscript</csymbol><ci id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.2.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.2">𝑛</ci><apply id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3"><times id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.1.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.1"></times><ci id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.2.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.2">𝑖</ci><ci id="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.3.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.2.3.3">𝑗</ci></apply></apply><ci id="Sx7.E3.m1.7.7.3.2.2.1.1.4.3.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.4.3">𝑏</ci></apply><list id="Sx7.E3.m1.7.7.3.2.2.1.1.2.3.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.2"><apply id="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1"><csymbol cd="ambiguous" id="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.1.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1">subscript</csymbol><ci id="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.2.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.2">𝑔</ci><ci id="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.3.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.1.1.1.3">𝑗</ci></apply><apply id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2"><csymbol cd="ambiguous" id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.1.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2">subscript</csymbol><ci id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.2.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.2">italic-ϕ</ci><ci id="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.3.cmml" xref="Sx7.E3.m1.7.7.3.2.2.1.1.2.2.2.3">𝑖</ci></apply></list></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.E3.m1.7c">\mathbb{E}_{\phi|n,g_{j}}[\log L(g_{j};n,\phi)]=\sum_{i}p(\phi_{i})\log p(n_{ij}^{b}|g_{j},\phi_{i})</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(3)</span></td>
</tr></tbody>
</table>
</div>
<div id="Sx7.SSx8.p7" class="ltx_para">
<p id="Sx7.SSx8.p7.1" class="ltx_p">These steps are repeated until genotypes are being selected for all
samples, and the expected log likelihood stops increasing. At the end
of the iteration, the likelihood-maximizing genotypes are reported,
along with the log likelihood difference between the best and second
best choice of genotype for each sample, which provides an indicator
of the confidence in genotype call.
To gauge convergence, this procedure is repeated 5 times for each marker, with different random choices of initial conditions. Markers which do not identify the same ML genotype multiple times in independent runs are not included among the high confidence genotype calls.</p>
</div>
</section>
<section id="Sx7.SSx9" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Error model calibration </h3>

<div id="Sx7.SSx9.p1" class="ltx_para">
<p id="Sx7.SSx9.p1.1" class="ltx_p">The sequence of 14 <em id="Sx7.SSx9.p1.1.1" class="ltx_emph ltx_font_italic">Ciona intestinalis</em> autosomes were downloaded from Ensembl (www.ensembl.org).
These 14 chromosomes were used as the template in our genome
simulation. Based on their sequence length, We used a markovian
coalescent simulator <span id="Sx7.SSx9.p1.1.2" class="ltx_text ltx_font_typewriter">macs</span> <cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib51" title="" class="ltx_ref">51</a>]</cite> to generate four
haploid samples drawn from a population under neutral Wright-Fisher
model with population mutation rate of 0.012 and population
recombination rate of 0.0085. Using the <em id="Sx7.SSx9.p1.1.3" class="ltx_emph ltx_font_italic">C. intestinalis</em> genome as
the reference sequence, two diploid parental genomes were constructed
based on the <span id="Sx7.SSx9.p1.1.4" class="ltx_text ltx_font_typewriter">macs</span> output with realistic SNP and Indel models
inferred by several previous studies on the Ciona
genome<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib52" title="" class="ltx_ref">52</a>, <a href="#bib.bib53" title="" class="ltx_ref">53</a>, <a href="#bib.bib54" title="" class="ltx_ref">54</a>]</cite>. We
wrote a <span id="Sx7.SSx9.p1.1.5" class="ltx_text ltx_font_typewriter">perl</span> script to simulate the genomes of offspring generated
by the cross of the two simulated parents. The software
package <span id="Sx7.SSx9.p1.1.6" class="ltx_text ltx_font_typewriter">dwgsim<cite class="ltx_cite ltx_citemacro_citep"><span id="Sx7.SSx9.p1.1.6.1.1" class="ltx_text ltx_font_serif">[</span><a href="#bib.bib55" title="" class="ltx_ref">55</a><span id="Sx7.SSx9.p1.1.6.2.2" class="ltx_text ltx_font_serif">]</span></cite></span> was used to
generate Illumina paired-end reads based on our simulated genomes of
both parents and offsprings, with the coverage of 20X and 5X
respectively.</p>
</div>
<div id="Sx7.SSx9.p2" class="ltx_para">
<p id="Sx7.SSx9.p2.1" class="ltx_p">To estimate the frequency of incorrect genotype calls as a function of
the log likelihood difference between the called and alternative
genotype, including contributions from uncertainty in SNP-mer
identification, assembly, and sampling noise, we carried out a
simulation of the <math id="Sx7.SSx9.p2.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="Sx7.SSx9.p2.1.m1.1a"><mi id="Sx7.SSx9.p2.1.m1.1.1" xref="Sx7.SSx9.p2.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx9.p2.1.m1.1b"><ci id="Sx7.SSx9.p2.1.m1.1.1.cmml" xref="Sx7.SSx9.p2.1.m1.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx9.p2.1.m1.1c">k</annotation></semantics></math>-mer  assembly and genotype inference protocols
Among high-confidence genotype calls, the observed error frequency was
a function of call confidence score was well-fit by a sum of two
stretched exponential functions, allowing assignment of
error probabilities to individual genotype calls.</p>
</div>
</section>
<section id="Sx7.SSx10" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Linkage group construction</h3>

<div id="Sx7.SSx10.p1" class="ltx_para">
<p id="Sx7.SSx10.p1.4" class="ltx_p">We use the linkage p-value <math id="Sx7.SSx10.p1.1.m1.1" class="ltx_Math" alttext="p_{ab}" display="inline"><semantics id="Sx7.SSx10.p1.1.m1.1a"><msub id="Sx7.SSx10.p1.1.m1.1.1" xref="Sx7.SSx10.p1.1.m1.1.1.cmml"><mi id="Sx7.SSx10.p1.1.m1.1.1.2" xref="Sx7.SSx10.p1.1.m1.1.1.2.cmml">p</mi><mrow id="Sx7.SSx10.p1.1.m1.1.1.3" xref="Sx7.SSx10.p1.1.m1.1.1.3.cmml"><mi id="Sx7.SSx10.p1.1.m1.1.1.3.2" xref="Sx7.SSx10.p1.1.m1.1.1.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx10.p1.1.m1.1.1.3.1" xref="Sx7.SSx10.p1.1.m1.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx10.p1.1.m1.1.1.3.3" xref="Sx7.SSx10.p1.1.m1.1.1.3.3.cmml">b</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p1.1.m1.1b"><apply id="Sx7.SSx10.p1.1.m1.1.1.cmml" xref="Sx7.SSx10.p1.1.m1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx10.p1.1.m1.1.1.1.cmml" xref="Sx7.SSx10.p1.1.m1.1.1">subscript</csymbol><ci id="Sx7.SSx10.p1.1.m1.1.1.2.cmml" xref="Sx7.SSx10.p1.1.m1.1.1.2">𝑝</ci><apply id="Sx7.SSx10.p1.1.m1.1.1.3.cmml" xref="Sx7.SSx10.p1.1.m1.1.1.3"><times id="Sx7.SSx10.p1.1.m1.1.1.3.1.cmml" xref="Sx7.SSx10.p1.1.m1.1.1.3.1"></times><ci id="Sx7.SSx10.p1.1.m1.1.1.3.2.cmml" xref="Sx7.SSx10.p1.1.m1.1.1.3.2">𝑎</ci><ci id="Sx7.SSx10.p1.1.m1.1.1.3.3.cmml" xref="Sx7.SSx10.p1.1.m1.1.1.3.3">𝑏</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p1.1.m1.1c">p_{ab}</annotation></semantics></math> between pairs of map bins <math id="Sx7.SSx10.p1.2.m2.1" class="ltx_Math" alttext="a" display="inline"><semantics id="Sx7.SSx10.p1.2.m2.1a"><mi id="Sx7.SSx10.p1.2.m2.1.1" xref="Sx7.SSx10.p1.2.m2.1.1.cmml">a</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p1.2.m2.1b"><ci id="Sx7.SSx10.p1.2.m2.1.1.cmml" xref="Sx7.SSx10.p1.2.m2.1.1">𝑎</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p1.2.m2.1c">a</annotation></semantics></math> and <math id="Sx7.SSx10.p1.3.m3.1" class="ltx_Math" alttext="b" display="inline"><semantics id="Sx7.SSx10.p1.3.m3.1a"><mi id="Sx7.SSx10.p1.3.m3.1.1" xref="Sx7.SSx10.p1.3.m3.1.1.cmml">b</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p1.3.m3.1b"><ci id="Sx7.SSx10.p1.3.m3.1.1.cmml" xref="Sx7.SSx10.p1.3.m3.1.1">𝑏</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p1.3.m3.1c">b</annotation></semantics></math> defined as the minimum over the four
possible relabelings <math id="Sx7.SSx10.p1.4.m4.1" class="ltx_Math" alttext="r" display="inline"><semantics id="Sx7.SSx10.p1.4.m4.1a"><mi id="Sx7.SSx10.p1.4.m4.1.1" xref="Sx7.SSx10.p1.4.m4.1.1.cmml">r</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p1.4.m4.1b"><ci id="Sx7.SSx10.p1.4.m4.1.1.cmml" xref="Sx7.SSx10.p1.4.m4.1.1">𝑟</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p1.4.m4.1c">r</annotation></semantics></math> of the maternal and paternal chromosomes of the Binomial p-value for the number of matching genotypes:</p>
</div>
<div id="Sx7.SSx10.p2" class="ltx_para">
<table id="Sx7.E4" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="Sx7.E4.m1.5" class="ltx_Math" alttext="p_{ab}=\min_{r}\left[1-\sum_{i}^{m_{r}-1}{n\choose i}\frac{1}{2^{n}}\right]" display="block"><semantics id="Sx7.E4.m1.5a"><mrow id="Sx7.E4.m1.5.5" xref="Sx7.E4.m1.5.5.cmml"><msub id="Sx7.E4.m1.5.5.4" xref="Sx7.E4.m1.5.5.4.cmml"><mi id="Sx7.E4.m1.5.5.4.2" xref="Sx7.E4.m1.5.5.4.2.cmml">p</mi><mrow id="Sx7.E4.m1.5.5.4.3" xref="Sx7.E4.m1.5.5.4.3.cmml"><mi id="Sx7.E4.m1.5.5.4.3.2" xref="Sx7.E4.m1.5.5.4.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.E4.m1.5.5.4.3.1" xref="Sx7.E4.m1.5.5.4.3.1.cmml">​</mo><mi id="Sx7.E4.m1.5.5.4.3.3" xref="Sx7.E4.m1.5.5.4.3.3.cmml">b</mi></mrow></msub><mo id="Sx7.E4.m1.5.5.3" xref="Sx7.E4.m1.5.5.3.cmml">=</mo><mrow id="Sx7.E4.m1.5.5.2.2" xref="Sx7.E4.m1.5.5.2.3.cmml"><munder id="Sx7.E4.m1.4.4.1.1.1" xref="Sx7.E4.m1.4.4.1.1.1.cmml"><mi id="Sx7.E4.m1.4.4.1.1.1.2" xref="Sx7.E4.m1.4.4.1.1.1.2.cmml">min</mi><mi id="Sx7.E4.m1.4.4.1.1.1.3" xref="Sx7.E4.m1.4.4.1.1.1.3.cmml">r</mi></munder><mo id="Sx7.E4.m1.5.5.2.2a" xref="Sx7.E4.m1.5.5.2.3.cmml">⁡</mo><mrow id="Sx7.E4.m1.5.5.2.2.2" xref="Sx7.E4.m1.5.5.2.3.cmml"><mo id="Sx7.E4.m1.5.5.2.2.2.2" xref="Sx7.E4.m1.5.5.2.3.cmml">[</mo><mrow id="Sx7.E4.m1.5.5.2.2.2.1" xref="Sx7.E4.m1.5.5.2.2.2.1.cmml"><mn id="Sx7.E4.m1.5.5.2.2.2.1.2" xref="Sx7.E4.m1.5.5.2.2.2.1.2.cmml">1</mn><mo rspace="0.055em" id="Sx7.E4.m1.5.5.2.2.2.1.1" xref="Sx7.E4.m1.5.5.2.2.2.1.1.cmml">−</mo><mrow id="Sx7.E4.m1.5.5.2.2.2.1.3" xref="Sx7.E4.m1.5.5.2.2.2.1.3.cmml"><munderover id="Sx7.E4.m1.5.5.2.2.2.1.3.1" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.cmml"><mo movablelimits="false" rspace="0em" id="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.2" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.2.cmml">∑</mo><mi id="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.3" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.3.cmml">i</mi><mrow id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.cmml"><msub id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.cmml"><mi id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.2" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.2.cmml">m</mi><mi id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.3" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.3.cmml">r</mi></msub><mo id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.1" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.1.cmml">−</mo><mn id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.3" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.3.cmml">1</mn></mrow></munderover><mrow id="Sx7.E4.m1.5.5.2.2.2.1.3.2" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.cmml"><mrow id="Sx7.E4.m1.3.3.5" xref="Sx7.E4.m1.3.3.4.cmml"><mo id="Sx7.E4.m1.3.3.5.1" xref="Sx7.E4.m1.1.1.1.1.1.cmml">(</mo><mfrac linethickness="0pt" id="Sx7.E4.m1.3.3.3.3" xref="Sx7.E4.m1.3.3.4.cmml"><mi id="Sx7.E4.m1.2.2.2.2.2" xref="Sx7.E4.m1.2.2.2.2.2.cmml">n</mi><mi id="Sx7.E4.m1.3.3.3.3.3" xref="Sx7.E4.m1.3.3.3.3.3.cmml">i</mi></mfrac><mo id="Sx7.E4.m1.3.3.5.2" xref="Sx7.E4.m1.1.1.1.1.1.cmml">)</mo></mrow><mo lspace="0em" rspace="0em" id="Sx7.E4.m1.5.5.2.2.2.1.3.2.1" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.1.cmml">​</mo><mfrac id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.cmml"><mn id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.2" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.2.cmml">1</mn><msup id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.cmml"><mn id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.2" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.2.cmml">2</mn><mi id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.3" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.3.cmml">n</mi></msup></mfrac></mrow></mrow></mrow><mo id="Sx7.E4.m1.5.5.2.2.2.3" xref="Sx7.E4.m1.5.5.2.3.cmml">]</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.E4.m1.5b"><apply id="Sx7.E4.m1.5.5.cmml" xref="Sx7.E4.m1.5.5"><eq id="Sx7.E4.m1.5.5.3.cmml" xref="Sx7.E4.m1.5.5.3"></eq><apply id="Sx7.E4.m1.5.5.4.cmml" xref="Sx7.E4.m1.5.5.4"><csymbol cd="ambiguous" id="Sx7.E4.m1.5.5.4.1.cmml" xref="Sx7.E4.m1.5.5.4">subscript</csymbol><ci id="Sx7.E4.m1.5.5.4.2.cmml" xref="Sx7.E4.m1.5.5.4.2">𝑝</ci><apply id="Sx7.E4.m1.5.5.4.3.cmml" xref="Sx7.E4.m1.5.5.4.3"><times id="Sx7.E4.m1.5.5.4.3.1.cmml" xref="Sx7.E4.m1.5.5.4.3.1"></times><ci id="Sx7.E4.m1.5.5.4.3.2.cmml" xref="Sx7.E4.m1.5.5.4.3.2">𝑎</ci><ci id="Sx7.E4.m1.5.5.4.3.3.cmml" xref="Sx7.E4.m1.5.5.4.3.3">𝑏</ci></apply></apply><apply id="Sx7.E4.m1.5.5.2.3.cmml" xref="Sx7.E4.m1.5.5.2.2"><apply id="Sx7.E4.m1.4.4.1.1.1.cmml" xref="Sx7.E4.m1.4.4.1.1.1"><csymbol cd="ambiguous" id="Sx7.E4.m1.4.4.1.1.1.1.cmml" xref="Sx7.E4.m1.4.4.1.1.1">subscript</csymbol><min id="Sx7.E4.m1.4.4.1.1.1.2.cmml" xref="Sx7.E4.m1.4.4.1.1.1.2"></min><ci id="Sx7.E4.m1.4.4.1.1.1.3.cmml" xref="Sx7.E4.m1.4.4.1.1.1.3">𝑟</ci></apply><apply id="Sx7.E4.m1.5.5.2.2.2.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1"><minus id="Sx7.E4.m1.5.5.2.2.2.1.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.1"></minus><cn type="integer" id="Sx7.E4.m1.5.5.2.2.2.1.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.2">1</cn><apply id="Sx7.E4.m1.5.5.2.2.2.1.3.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3"><apply id="Sx7.E4.m1.5.5.2.2.2.1.3.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1"><csymbol cd="ambiguous" id="Sx7.E4.m1.5.5.2.2.2.1.3.1.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1">superscript</csymbol><apply id="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1"><csymbol cd="ambiguous" id="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1">subscript</csymbol><sum id="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.2"></sum><ci id="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.3.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.2.3">𝑖</ci></apply><apply id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3"><minus id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.1"></minus><apply id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2"><csymbol cd="ambiguous" id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2">subscript</csymbol><ci id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.2">𝑚</ci><ci id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.3.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.2.3">𝑟</ci></apply><cn type="integer" id="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.3.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.1.3.3">1</cn></apply></apply><apply id="Sx7.E4.m1.5.5.2.2.2.1.3.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2"><times id="Sx7.E4.m1.5.5.2.2.2.1.3.2.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.1"></times><apply id="Sx7.E4.m1.3.3.4.cmml" xref="Sx7.E4.m1.3.3.5"><csymbol cd="latexml" id="Sx7.E4.m1.1.1.1.1.1.cmml" xref="Sx7.E4.m1.3.3.5.1">binomial</csymbol><ci id="Sx7.E4.m1.2.2.2.2.2.cmml" xref="Sx7.E4.m1.2.2.2.2.2">𝑛</ci><ci id="Sx7.E4.m1.3.3.3.3.3.cmml" xref="Sx7.E4.m1.3.3.3.3.3">𝑖</ci></apply><apply id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2"><divide id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2"></divide><cn type="integer" id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.2">1</cn><apply id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3"><csymbol cd="ambiguous" id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.1.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3">superscript</csymbol><cn type="integer" id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.2.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.2">2</cn><ci id="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.3.cmml" xref="Sx7.E4.m1.5.5.2.2.2.1.3.2.2.3.3">𝑛</ci></apply></apply></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.E4.m1.5c">p_{ab}=\min_{r}\left[1-\sum_{i}^{m_{r}-1}{n\choose i}\frac{1}{2^{n}}\right]</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(4)</span></td>
</tr></tbody>
</table>
</div>
<div id="Sx7.SSx10.p3" class="ltx_para">
<p id="Sx7.SSx10.p3.3" class="ltx_p">where <math id="Sx7.SSx10.p3.1.m1.1" class="ltx_Math" alttext="n" display="inline"><semantics id="Sx7.SSx10.p3.1.m1.1a"><mi id="Sx7.SSx10.p3.1.m1.1.1" xref="Sx7.SSx10.p3.1.m1.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p3.1.m1.1b"><ci id="Sx7.SSx10.p3.1.m1.1.1.cmml" xref="Sx7.SSx10.p3.1.m1.1.1">𝑛</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p3.1.m1.1c">n</annotation></semantics></math> is the total number of sample genotype calls (68 in the present case, or 34 in each parent) and <math id="Sx7.SSx10.p3.2.m2.1" class="ltx_Math" alttext="m_{r}" display="inline"><semantics id="Sx7.SSx10.p3.2.m2.1a"><msub id="Sx7.SSx10.p3.2.m2.1.1" xref="Sx7.SSx10.p3.2.m2.1.1.cmml"><mi id="Sx7.SSx10.p3.2.m2.1.1.2" xref="Sx7.SSx10.p3.2.m2.1.1.2.cmml">m</mi><mi id="Sx7.SSx10.p3.2.m2.1.1.3" xref="Sx7.SSx10.p3.2.m2.1.1.3.cmml">r</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p3.2.m2.1b"><apply id="Sx7.SSx10.p3.2.m2.1.1.cmml" xref="Sx7.SSx10.p3.2.m2.1.1"><csymbol cd="ambiguous" id="Sx7.SSx10.p3.2.m2.1.1.1.cmml" xref="Sx7.SSx10.p3.2.m2.1.1">subscript</csymbol><ci id="Sx7.SSx10.p3.2.m2.1.1.2.cmml" xref="Sx7.SSx10.p3.2.m2.1.1.2">𝑚</ci><ci id="Sx7.SSx10.p3.2.m2.1.1.3.cmml" xref="Sx7.SSx10.p3.2.m2.1.1.3">𝑟</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p3.2.m2.1c">m_{r}</annotation></semantics></math> is the number of matching genotypes under relabeling <math id="Sx7.SSx10.p3.3.m3.1" class="ltx_Math" alttext="r" display="inline"><semantics id="Sx7.SSx10.p3.3.m3.1a"><mi id="Sx7.SSx10.p3.3.m3.1.1" xref="Sx7.SSx10.p3.3.m3.1.1.cmml">r</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p3.3.m3.1b"><ci id="Sx7.SSx10.p3.3.m3.1.1.cmml" xref="Sx7.SSx10.p3.3.m3.1.1">𝑟</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p3.3.m3.1c">r</annotation></semantics></math>.</p>
</div>
<div id="Sx7.SSx10.p4" class="ltx_para">
<p id="Sx7.SSx10.p4.1" class="ltx_p">We identified map bins with segregation patterns indicating either inconsistent placement in the maternal and paternal maps or genotyping error with a double threshold procedure as follows:</p>
</div>
<div id="Sx7.SSx10.p5" class="ltx_para">
<ol id="Sx7.I1" class="ltx_enumerate">
<li id="Sx7.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="Sx7.I1.i1.p1" class="ltx_para">
<p id="Sx7.I1.i1.p1.1" class="ltx_p">Map bins were partitioned into linkage groups by single linkage clustering at a threshold of <math id="Sx7.I1.i1.p1.1.m1.1" class="ltx_Math" alttext="p_{ab}&lt;p_{1}" display="inline"><semantics id="Sx7.I1.i1.p1.1.m1.1a"><mrow id="Sx7.I1.i1.p1.1.m1.1.1" xref="Sx7.I1.i1.p1.1.m1.1.1.cmml"><msub id="Sx7.I1.i1.p1.1.m1.1.1.2" xref="Sx7.I1.i1.p1.1.m1.1.1.2.cmml"><mi id="Sx7.I1.i1.p1.1.m1.1.1.2.2" xref="Sx7.I1.i1.p1.1.m1.1.1.2.2.cmml">p</mi><mrow id="Sx7.I1.i1.p1.1.m1.1.1.2.3" xref="Sx7.I1.i1.p1.1.m1.1.1.2.3.cmml"><mi id="Sx7.I1.i1.p1.1.m1.1.1.2.3.2" xref="Sx7.I1.i1.p1.1.m1.1.1.2.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.I1.i1.p1.1.m1.1.1.2.3.1" xref="Sx7.I1.i1.p1.1.m1.1.1.2.3.1.cmml">​</mo><mi id="Sx7.I1.i1.p1.1.m1.1.1.2.3.3" xref="Sx7.I1.i1.p1.1.m1.1.1.2.3.3.cmml">b</mi></mrow></msub><mo id="Sx7.I1.i1.p1.1.m1.1.1.1" xref="Sx7.I1.i1.p1.1.m1.1.1.1.cmml">&lt;</mo><msub id="Sx7.I1.i1.p1.1.m1.1.1.3" xref="Sx7.I1.i1.p1.1.m1.1.1.3.cmml"><mi id="Sx7.I1.i1.p1.1.m1.1.1.3.2" xref="Sx7.I1.i1.p1.1.m1.1.1.3.2.cmml">p</mi><mn id="Sx7.I1.i1.p1.1.m1.1.1.3.3" xref="Sx7.I1.i1.p1.1.m1.1.1.3.3.cmml">1</mn></msub></mrow><annotation-xml encoding="MathML-Content" id="Sx7.I1.i1.p1.1.m1.1b"><apply id="Sx7.I1.i1.p1.1.m1.1.1.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1"><lt id="Sx7.I1.i1.p1.1.m1.1.1.1.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.1"></lt><apply id="Sx7.I1.i1.p1.1.m1.1.1.2.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.2"><csymbol cd="ambiguous" id="Sx7.I1.i1.p1.1.m1.1.1.2.1.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.2">subscript</csymbol><ci id="Sx7.I1.i1.p1.1.m1.1.1.2.2.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.2.2">𝑝</ci><apply id="Sx7.I1.i1.p1.1.m1.1.1.2.3.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.2.3"><times id="Sx7.I1.i1.p1.1.m1.1.1.2.3.1.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.2.3.1"></times><ci id="Sx7.I1.i1.p1.1.m1.1.1.2.3.2.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.2.3.2">𝑎</ci><ci id="Sx7.I1.i1.p1.1.m1.1.1.2.3.3.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.2.3.3">𝑏</ci></apply></apply><apply id="Sx7.I1.i1.p1.1.m1.1.1.3.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.3"><csymbol cd="ambiguous" id="Sx7.I1.i1.p1.1.m1.1.1.3.1.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.3">subscript</csymbol><ci id="Sx7.I1.i1.p1.1.m1.1.1.3.2.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.3.2">𝑝</ci><cn type="integer" id="Sx7.I1.i1.p1.1.m1.1.1.3.3.cmml" xref="Sx7.I1.i1.p1.1.m1.1.1.3.3">1</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.I1.i1.p1.1.m1.1c">p_{ab}&lt;p_{1}</annotation></semantics></math>.</p>
</div>
</li>
<li id="Sx7.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="Sx7.I1.i2.p1" class="ltx_para">
<p id="Sx7.I1.i2.p1.2" class="ltx_p">Within each partition, map bins which formed articulation points<cite class="ltx_cite ltx_citemacro_citep">[<span id="Sx7.I1.i2.p1.2.1.1" class="ltx_text ltx_font_italic">i.e.</span> nodes which, if removed, would cause the linkage group to fall apart into two disconnected subgraphs; <a href="#bib.bib56" title="" class="ltx_ref">56</a>]</cite> in the graph of <math id="Sx7.I1.i2.p1.1.m1.1" class="ltx_Math" alttext="p_{ab}&lt;p_{2}" display="inline"><semantics id="Sx7.I1.i2.p1.1.m1.1a"><mrow id="Sx7.I1.i2.p1.1.m1.1.1" xref="Sx7.I1.i2.p1.1.m1.1.1.cmml"><msub id="Sx7.I1.i2.p1.1.m1.1.1.2" xref="Sx7.I1.i2.p1.1.m1.1.1.2.cmml"><mi id="Sx7.I1.i2.p1.1.m1.1.1.2.2" xref="Sx7.I1.i2.p1.1.m1.1.1.2.2.cmml">p</mi><mrow id="Sx7.I1.i2.p1.1.m1.1.1.2.3" xref="Sx7.I1.i2.p1.1.m1.1.1.2.3.cmml"><mi id="Sx7.I1.i2.p1.1.m1.1.1.2.3.2" xref="Sx7.I1.i2.p1.1.m1.1.1.2.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.I1.i2.p1.1.m1.1.1.2.3.1" xref="Sx7.I1.i2.p1.1.m1.1.1.2.3.1.cmml">​</mo><mi id="Sx7.I1.i2.p1.1.m1.1.1.2.3.3" xref="Sx7.I1.i2.p1.1.m1.1.1.2.3.3.cmml">b</mi></mrow></msub><mo id="Sx7.I1.i2.p1.1.m1.1.1.1" xref="Sx7.I1.i2.p1.1.m1.1.1.1.cmml">&lt;</mo><msub id="Sx7.I1.i2.p1.1.m1.1.1.3" xref="Sx7.I1.i2.p1.1.m1.1.1.3.cmml"><mi id="Sx7.I1.i2.p1.1.m1.1.1.3.2" xref="Sx7.I1.i2.p1.1.m1.1.1.3.2.cmml">p</mi><mn id="Sx7.I1.i2.p1.1.m1.1.1.3.3" xref="Sx7.I1.i2.p1.1.m1.1.1.3.3.cmml">2</mn></msub></mrow><annotation-xml encoding="MathML-Content" id="Sx7.I1.i2.p1.1.m1.1b"><apply id="Sx7.I1.i2.p1.1.m1.1.1.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1"><lt id="Sx7.I1.i2.p1.1.m1.1.1.1.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.1"></lt><apply id="Sx7.I1.i2.p1.1.m1.1.1.2.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.2"><csymbol cd="ambiguous" id="Sx7.I1.i2.p1.1.m1.1.1.2.1.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.2">subscript</csymbol><ci id="Sx7.I1.i2.p1.1.m1.1.1.2.2.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.2.2">𝑝</ci><apply id="Sx7.I1.i2.p1.1.m1.1.1.2.3.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.2.3"><times id="Sx7.I1.i2.p1.1.m1.1.1.2.3.1.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.2.3.1"></times><ci id="Sx7.I1.i2.p1.1.m1.1.1.2.3.2.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.2.3.2">𝑎</ci><ci id="Sx7.I1.i2.p1.1.m1.1.1.2.3.3.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.2.3.3">𝑏</ci></apply></apply><apply id="Sx7.I1.i2.p1.1.m1.1.1.3.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.3"><csymbol cd="ambiguous" id="Sx7.I1.i2.p1.1.m1.1.1.3.1.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.3">subscript</csymbol><ci id="Sx7.I1.i2.p1.1.m1.1.1.3.2.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.3.2">𝑝</ci><cn type="integer" id="Sx7.I1.i2.p1.1.m1.1.1.3.3.cmml" xref="Sx7.I1.i2.p1.1.m1.1.1.3.3">2</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.I1.i2.p1.1.m1.1c">p_{ab}&lt;p_{2}</annotation></semantics></math>, where <math id="Sx7.I1.i2.p1.2.m2.1" class="ltx_Math" alttext="p_{2}&gt;p_{1}" display="inline"><semantics id="Sx7.I1.i2.p1.2.m2.1a"><mrow id="Sx7.I1.i2.p1.2.m2.1.1" xref="Sx7.I1.i2.p1.2.m2.1.1.cmml"><msub id="Sx7.I1.i2.p1.2.m2.1.1.2" xref="Sx7.I1.i2.p1.2.m2.1.1.2.cmml"><mi id="Sx7.I1.i2.p1.2.m2.1.1.2.2" xref="Sx7.I1.i2.p1.2.m2.1.1.2.2.cmml">p</mi><mn id="Sx7.I1.i2.p1.2.m2.1.1.2.3" xref="Sx7.I1.i2.p1.2.m2.1.1.2.3.cmml">2</mn></msub><mo id="Sx7.I1.i2.p1.2.m2.1.1.1" xref="Sx7.I1.i2.p1.2.m2.1.1.1.cmml">&gt;</mo><msub id="Sx7.I1.i2.p1.2.m2.1.1.3" xref="Sx7.I1.i2.p1.2.m2.1.1.3.cmml"><mi id="Sx7.I1.i2.p1.2.m2.1.1.3.2" xref="Sx7.I1.i2.p1.2.m2.1.1.3.2.cmml">p</mi><mn id="Sx7.I1.i2.p1.2.m2.1.1.3.3" xref="Sx7.I1.i2.p1.2.m2.1.1.3.3.cmml">1</mn></msub></mrow><annotation-xml encoding="MathML-Content" id="Sx7.I1.i2.p1.2.m2.1b"><apply id="Sx7.I1.i2.p1.2.m2.1.1.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1"><gt id="Sx7.I1.i2.p1.2.m2.1.1.1.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.1"></gt><apply id="Sx7.I1.i2.p1.2.m2.1.1.2.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.2"><csymbol cd="ambiguous" id="Sx7.I1.i2.p1.2.m2.1.1.2.1.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.2">subscript</csymbol><ci id="Sx7.I1.i2.p1.2.m2.1.1.2.2.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.2.2">𝑝</ci><cn type="integer" id="Sx7.I1.i2.p1.2.m2.1.1.2.3.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.2.3">2</cn></apply><apply id="Sx7.I1.i2.p1.2.m2.1.1.3.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.3"><csymbol cd="ambiguous" id="Sx7.I1.i2.p1.2.m2.1.1.3.1.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.3">subscript</csymbol><ci id="Sx7.I1.i2.p1.2.m2.1.1.3.2.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.3.2">𝑝</ci><cn type="integer" id="Sx7.I1.i2.p1.2.m2.1.1.3.3.cmml" xref="Sx7.I1.i2.p1.2.m2.1.1.3.3">1</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.I1.i2.p1.2.m2.1c">p_{2}&gt;p_{1}</annotation></semantics></math>.</p>
</div>
</li>
</ol>
</div>
<div id="Sx7.SSx10.p6" class="ltx_para">
<p id="Sx7.SSx10.p6.11" class="ltx_p">This procedure identifies map bins which alone account for the merging of what would otherwise be two distinct partitions. We used the following pairs of thresholds <math id="Sx7.SSx10.p6.1.m1.2" class="ltx_Math" alttext="p_{1},p_{2}" display="inline"><semantics id="Sx7.SSx10.p6.1.m1.2a"><mrow id="Sx7.SSx10.p6.1.m1.2.2.2" xref="Sx7.SSx10.p6.1.m1.2.2.3.cmml"><msub id="Sx7.SSx10.p6.1.m1.1.1.1.1" xref="Sx7.SSx10.p6.1.m1.1.1.1.1.cmml"><mi id="Sx7.SSx10.p6.1.m1.1.1.1.1.2" xref="Sx7.SSx10.p6.1.m1.1.1.1.1.2.cmml">p</mi><mn id="Sx7.SSx10.p6.1.m1.1.1.1.1.3" xref="Sx7.SSx10.p6.1.m1.1.1.1.1.3.cmml">1</mn></msub><mo id="Sx7.SSx10.p6.1.m1.2.2.2.3" xref="Sx7.SSx10.p6.1.m1.2.2.3.cmml">,</mo><msub id="Sx7.SSx10.p6.1.m1.2.2.2.2" xref="Sx7.SSx10.p6.1.m1.2.2.2.2.cmml"><mi id="Sx7.SSx10.p6.1.m1.2.2.2.2.2" xref="Sx7.SSx10.p6.1.m1.2.2.2.2.2.cmml">p</mi><mn id="Sx7.SSx10.p6.1.m1.2.2.2.2.3" xref="Sx7.SSx10.p6.1.m1.2.2.2.2.3.cmml">2</mn></msub></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p6.1.m1.2b"><list id="Sx7.SSx10.p6.1.m1.2.2.3.cmml" xref="Sx7.SSx10.p6.1.m1.2.2.2"><apply id="Sx7.SSx10.p6.1.m1.1.1.1.1.cmml" xref="Sx7.SSx10.p6.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx10.p6.1.m1.1.1.1.1.1.cmml" xref="Sx7.SSx10.p6.1.m1.1.1.1.1">subscript</csymbol><ci id="Sx7.SSx10.p6.1.m1.1.1.1.1.2.cmml" xref="Sx7.SSx10.p6.1.m1.1.1.1.1.2">𝑝</ci><cn type="integer" id="Sx7.SSx10.p6.1.m1.1.1.1.1.3.cmml" xref="Sx7.SSx10.p6.1.m1.1.1.1.1.3">1</cn></apply><apply id="Sx7.SSx10.p6.1.m1.2.2.2.2.cmml" xref="Sx7.SSx10.p6.1.m1.2.2.2.2"><csymbol cd="ambiguous" id="Sx7.SSx10.p6.1.m1.2.2.2.2.1.cmml" xref="Sx7.SSx10.p6.1.m1.2.2.2.2">subscript</csymbol><ci id="Sx7.SSx10.p6.1.m1.2.2.2.2.2.cmml" xref="Sx7.SSx10.p6.1.m1.2.2.2.2.2">𝑝</ci><cn type="integer" id="Sx7.SSx10.p6.1.m1.2.2.2.2.3.cmml" xref="Sx7.SSx10.p6.1.m1.2.2.2.2.3">2</cn></apply></list></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p6.1.m1.2c">p_{1},p_{2}</annotation></semantics></math> to identify a total of 20 map bins for exclusion from the map:
10<sup id="Sx7.SSx10.p6.11.1" class="ltx_sup"><span id="Sx7.SSx10.p6.11.1.1" class="ltx_text ltx_font_italic">-7</span></sup>,10<sup id="Sx7.SSx10.p6.11.2" class="ltx_sup"><span id="Sx7.SSx10.p6.11.2.1" class="ltx_text ltx_font_italic">-6</span></sup>;
10<sup id="Sx7.SSx10.p6.11.3" class="ltx_sup"><span id="Sx7.SSx10.p6.11.3.1" class="ltx_text ltx_font_italic">-8</span></sup>,10<sup id="Sx7.SSx10.p6.11.4" class="ltx_sup"><span id="Sx7.SSx10.p6.11.4.1" class="ltx_text ltx_font_italic">-7</span></sup>;
10<sup id="Sx7.SSx10.p6.11.5" class="ltx_sup"><span id="Sx7.SSx10.p6.11.5.1" class="ltx_text ltx_font_italic">-9</span></sup>,10<sup id="Sx7.SSx10.p6.11.6" class="ltx_sup"><span id="Sx7.SSx10.p6.11.6.1" class="ltx_text ltx_font_italic">-8</span></sup>.
The remaining markers form locally consistent linkage groups in which all linkages defined at threshold <math id="Sx7.SSx10.p6.8.m8.1" class="ltx_Math" alttext="p_{1}" display="inline"><semantics id="Sx7.SSx10.p6.8.m8.1a"><msub id="Sx7.SSx10.p6.8.m8.1.1" xref="Sx7.SSx10.p6.8.m8.1.1.cmml"><mi id="Sx7.SSx10.p6.8.m8.1.1.2" xref="Sx7.SSx10.p6.8.m8.1.1.2.cmml">p</mi><mn id="Sx7.SSx10.p6.8.m8.1.1.3" xref="Sx7.SSx10.p6.8.m8.1.1.3.cmml">1</mn></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p6.8.m8.1b"><apply id="Sx7.SSx10.p6.8.m8.1.1.cmml" xref="Sx7.SSx10.p6.8.m8.1.1"><csymbol cd="ambiguous" id="Sx7.SSx10.p6.8.m8.1.1.1.cmml" xref="Sx7.SSx10.p6.8.m8.1.1">subscript</csymbol><ci id="Sx7.SSx10.p6.8.m8.1.1.2.cmml" xref="Sx7.SSx10.p6.8.m8.1.1.2">𝑝</ci><cn type="integer" id="Sx7.SSx10.p6.8.m8.1.1.3.cmml" xref="Sx7.SSx10.p6.8.m8.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p6.8.m8.1c">p_{1}</annotation></semantics></math> are corroborated by multiple linkages at <math id="Sx7.SSx10.p6.9.m9.1" class="ltx_Math" alttext="p_{2}" display="inline"><semantics id="Sx7.SSx10.p6.9.m9.1a"><msub id="Sx7.SSx10.p6.9.m9.1.1" xref="Sx7.SSx10.p6.9.m9.1.1.cmml"><mi id="Sx7.SSx10.p6.9.m9.1.1.2" xref="Sx7.SSx10.p6.9.m9.1.1.2.cmml">p</mi><mn id="Sx7.SSx10.p6.9.m9.1.1.3" xref="Sx7.SSx10.p6.9.m9.1.1.3.cmml">2</mn></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p6.9.m9.1b"><apply id="Sx7.SSx10.p6.9.m9.1.1.cmml" xref="Sx7.SSx10.p6.9.m9.1.1"><csymbol cd="ambiguous" id="Sx7.SSx10.p6.9.m9.1.1.1.cmml" xref="Sx7.SSx10.p6.9.m9.1.1">subscript</csymbol><ci id="Sx7.SSx10.p6.9.m9.1.1.2.cmml" xref="Sx7.SSx10.p6.9.m9.1.1.2">𝑝</ci><cn type="integer" id="Sx7.SSx10.p6.9.m9.1.1.3.cmml" xref="Sx7.SSx10.p6.9.m9.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p6.9.m9.1c">p_{2}</annotation></semantics></math>, for the above values of <math id="Sx7.SSx10.p6.10.m10.1" class="ltx_Math" alttext="p_{1}" display="inline"><semantics id="Sx7.SSx10.p6.10.m10.1a"><msub id="Sx7.SSx10.p6.10.m10.1.1" xref="Sx7.SSx10.p6.10.m10.1.1.cmml"><mi id="Sx7.SSx10.p6.10.m10.1.1.2" xref="Sx7.SSx10.p6.10.m10.1.1.2.cmml">p</mi><mn id="Sx7.SSx10.p6.10.m10.1.1.3" xref="Sx7.SSx10.p6.10.m10.1.1.3.cmml">1</mn></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p6.10.m10.1b"><apply id="Sx7.SSx10.p6.10.m10.1.1.cmml" xref="Sx7.SSx10.p6.10.m10.1.1"><csymbol cd="ambiguous" id="Sx7.SSx10.p6.10.m10.1.1.1.cmml" xref="Sx7.SSx10.p6.10.m10.1.1">subscript</csymbol><ci id="Sx7.SSx10.p6.10.m10.1.1.2.cmml" xref="Sx7.SSx10.p6.10.m10.1.1.2">𝑝</ci><cn type="integer" id="Sx7.SSx10.p6.10.m10.1.1.3.cmml" xref="Sx7.SSx10.p6.10.m10.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p6.10.m10.1c">p_{1}</annotation></semantics></math> and <math id="Sx7.SSx10.p6.11.m11.1" class="ltx_Math" alttext="p_{2}" display="inline"><semantics id="Sx7.SSx10.p6.11.m11.1a"><msub id="Sx7.SSx10.p6.11.m11.1.1" xref="Sx7.SSx10.p6.11.m11.1.1.cmml"><mi id="Sx7.SSx10.p6.11.m11.1.1.2" xref="Sx7.SSx10.p6.11.m11.1.1.2.cmml">p</mi><mn id="Sx7.SSx10.p6.11.m11.1.1.3" xref="Sx7.SSx10.p6.11.m11.1.1.3.cmml">2</mn></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx10.p6.11.m11.1b"><apply id="Sx7.SSx10.p6.11.m11.1.1.cmml" xref="Sx7.SSx10.p6.11.m11.1.1"><csymbol cd="ambiguous" id="Sx7.SSx10.p6.11.m11.1.1.1.cmml" xref="Sx7.SSx10.p6.11.m11.1.1">subscript</csymbol><ci id="Sx7.SSx10.p6.11.m11.1.1.2.cmml" xref="Sx7.SSx10.p6.11.m11.1.1.2">𝑝</ci><cn type="integer" id="Sx7.SSx10.p6.11.m11.1.1.3.cmml" xref="Sx7.SSx10.p6.11.m11.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx10.p6.11.m11.1c">p_{2}</annotation></semantics></math>.</p>
</div>
</section>
<section id="Sx7.SSx11" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Marker ordering</h3>

<div id="Sx7.SSx11.p1" class="ltx_para">
<p id="Sx7.SSx11.p1.6" class="ltx_p">Markers were ordered within each linkage group using the following
protocol. Within each linkage group a consistent labeling of the four parental chromosomes was achieved by constructing a graph <math id="Sx7.SSx11.p1.1.m1.1" class="ltx_Math" alttext="G" display="inline"><semantics id="Sx7.SSx11.p1.1.m1.1a"><mi id="Sx7.SSx11.p1.1.m1.1.1" xref="Sx7.SSx11.p1.1.m1.1.1.cmml">G</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx11.p1.1.m1.1b"><ci id="Sx7.SSx11.p1.1.m1.1.1.cmml" xref="Sx7.SSx11.p1.1.m1.1.1">𝐺</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx11.p1.1.m1.1c">G</annotation></semantics></math> in which nodes correspond to map bins and edges are weighted by linkage p-value <math id="Sx7.SSx11.p1.2.m2.1" class="ltx_Math" alttext="p_{ab}" display="inline"><semantics id="Sx7.SSx11.p1.2.m2.1a"><msub id="Sx7.SSx11.p1.2.m2.1.1" xref="Sx7.SSx11.p1.2.m2.1.1.cmml"><mi id="Sx7.SSx11.p1.2.m2.1.1.2" xref="Sx7.SSx11.p1.2.m2.1.1.2.cmml">p</mi><mrow id="Sx7.SSx11.p1.2.m2.1.1.3" xref="Sx7.SSx11.p1.2.m2.1.1.3.cmml"><mi id="Sx7.SSx11.p1.2.m2.1.1.3.2" xref="Sx7.SSx11.p1.2.m2.1.1.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx11.p1.2.m2.1.1.3.1" xref="Sx7.SSx11.p1.2.m2.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx11.p1.2.m2.1.1.3.3" xref="Sx7.SSx11.p1.2.m2.1.1.3.3.cmml">b</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx11.p1.2.m2.1b"><apply id="Sx7.SSx11.p1.2.m2.1.1.cmml" xref="Sx7.SSx11.p1.2.m2.1.1"><csymbol cd="ambiguous" id="Sx7.SSx11.p1.2.m2.1.1.1.cmml" xref="Sx7.SSx11.p1.2.m2.1.1">subscript</csymbol><ci id="Sx7.SSx11.p1.2.m2.1.1.2.cmml" xref="Sx7.SSx11.p1.2.m2.1.1.2">𝑝</ci><apply id="Sx7.SSx11.p1.2.m2.1.1.3.cmml" xref="Sx7.SSx11.p1.2.m2.1.1.3"><times id="Sx7.SSx11.p1.2.m2.1.1.3.1.cmml" xref="Sx7.SSx11.p1.2.m2.1.1.3.1"></times><ci id="Sx7.SSx11.p1.2.m2.1.1.3.2.cmml" xref="Sx7.SSx11.p1.2.m2.1.1.3.2">𝑎</ci><ci id="Sx7.SSx11.p1.2.m2.1.1.3.3.cmml" xref="Sx7.SSx11.p1.2.m2.1.1.3.3">𝑏</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx11.p1.2.m2.1c">p_{ab}</annotation></semantics></math> (as defined above). The local chromosome labels are updated at each map bin as it is reached in a traversal of the minimum spanning tree of <math id="Sx7.SSx11.p1.3.m3.1" class="ltx_Math" alttext="G" display="inline"><semantics id="Sx7.SSx11.p1.3.m3.1a"><mi id="Sx7.SSx11.p1.3.m3.1.1" xref="Sx7.SSx11.p1.3.m3.1.1.cmml">G</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx11.p1.3.m3.1b"><ci id="Sx7.SSx11.p1.3.m3.1.1.cmml" xref="Sx7.SSx11.p1.3.m3.1.1">𝐺</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx11.p1.3.m3.1c">G</annotation></semantics></math> to the labeling <math id="Sx7.SSx11.p1.4.m4.1" class="ltx_Math" alttext="r" display="inline"><semantics id="Sx7.SSx11.p1.4.m4.1a"><mi id="Sx7.SSx11.p1.4.m4.1.1" xref="Sx7.SSx11.p1.4.m4.1.1.cmml">r</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx11.p1.4.m4.1b"><ci id="Sx7.SSx11.p1.4.m4.1.1.cmml" xref="Sx7.SSx11.p1.4.m4.1.1">𝑟</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx11.p1.4.m4.1c">r</annotation></semantics></math> that maximizes <math id="Sx7.SSx11.p1.5.m5.1" class="ltx_Math" alttext="p_{ab}" display="inline"><semantics id="Sx7.SSx11.p1.5.m5.1a"><msub id="Sx7.SSx11.p1.5.m5.1.1" xref="Sx7.SSx11.p1.5.m5.1.1.cmml"><mi id="Sx7.SSx11.p1.5.m5.1.1.2" xref="Sx7.SSx11.p1.5.m5.1.1.2.cmml">p</mi><mrow id="Sx7.SSx11.p1.5.m5.1.1.3" xref="Sx7.SSx11.p1.5.m5.1.1.3.cmml"><mi id="Sx7.SSx11.p1.5.m5.1.1.3.2" xref="Sx7.SSx11.p1.5.m5.1.1.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx11.p1.5.m5.1.1.3.1" xref="Sx7.SSx11.p1.5.m5.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx11.p1.5.m5.1.1.3.3" xref="Sx7.SSx11.p1.5.m5.1.1.3.3.cmml">b</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx11.p1.5.m5.1b"><apply id="Sx7.SSx11.p1.5.m5.1.1.cmml" xref="Sx7.SSx11.p1.5.m5.1.1"><csymbol cd="ambiguous" id="Sx7.SSx11.p1.5.m5.1.1.1.cmml" xref="Sx7.SSx11.p1.5.m5.1.1">subscript</csymbol><ci id="Sx7.SSx11.p1.5.m5.1.1.2.cmml" xref="Sx7.SSx11.p1.5.m5.1.1.2">𝑝</ci><apply id="Sx7.SSx11.p1.5.m5.1.1.3.cmml" xref="Sx7.SSx11.p1.5.m5.1.1.3"><times id="Sx7.SSx11.p1.5.m5.1.1.3.1.cmml" xref="Sx7.SSx11.p1.5.m5.1.1.3.1"></times><ci id="Sx7.SSx11.p1.5.m5.1.1.3.2.cmml" xref="Sx7.SSx11.p1.5.m5.1.1.3.2">𝑎</ci><ci id="Sx7.SSx11.p1.5.m5.1.1.3.3.cmml" xref="Sx7.SSx11.p1.5.m5.1.1.3.3">𝑏</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx11.p1.5.m5.1c">p_{ab}</annotation></semantics></math> along the incident of <math id="Sx7.SSx11.p1.6.m6.1" class="ltx_Math" alttext="G" display="inline"><semantics id="Sx7.SSx11.p1.6.m6.1a"><mi id="Sx7.SSx11.p1.6.m6.1.1" xref="Sx7.SSx11.p1.6.m6.1.1.cmml">G</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx11.p1.6.m6.1b"><ci id="Sx7.SSx11.p1.6.m6.1.1.cmml" xref="Sx7.SSx11.p1.6.m6.1.1">𝐺</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx11.p1.6.m6.1c">G</annotation></semantics></math> used in the traversal.
Markers within each linkage group were clustered by hierarchical clustering (marker-marker distance metric: cosine of the angle between the vectors of recombination distances to the other map bins; distance updating method: average linkage) into a binary tree data structure with leaves representing map bins. A node in the right subtree of the root node was rotated, interchanging its left and right subtrees if its left subtree was not already closer (in average recombination distance) to the markers of the left subtree of the global root; and similarly for nodes in the left subtree of the root. An in-order traversal of the tree generates an ordering of the markers. Finally three reversals of the order of markers in segments of the map were added based on visual inspection of the recombination distance matrix.
In the final marker ordering, 51% of adjacent map bin pairs are separated by a single recombination event in the cross, and 94% are separated by three or fewer recombinants in each parent.</p>
</div>
</section>
<section id="Sx7.SSx12" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Placement of markers on the map </h3>

<div id="Sx7.SSx12.p1" class="ltx_para">
<p id="Sx7.SSx12.p1.7" class="ltx_p">To anchor additional markers to the map, we computed the <math id="Sx7.SSx12.p1.1.m1.1" class="ltx_Math" alttext="p_{ab}" display="inline"><semantics id="Sx7.SSx12.p1.1.m1.1a"><msub id="Sx7.SSx12.p1.1.m1.1.1" xref="Sx7.SSx12.p1.1.m1.1.1.cmml"><mi id="Sx7.SSx12.p1.1.m1.1.1.2" xref="Sx7.SSx12.p1.1.m1.1.1.2.cmml">p</mi><mrow id="Sx7.SSx12.p1.1.m1.1.1.3" xref="Sx7.SSx12.p1.1.m1.1.1.3.cmml"><mi id="Sx7.SSx12.p1.1.m1.1.1.3.2" xref="Sx7.SSx12.p1.1.m1.1.1.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx12.p1.1.m1.1.1.3.1" xref="Sx7.SSx12.p1.1.m1.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx12.p1.1.m1.1.1.3.3" xref="Sx7.SSx12.p1.1.m1.1.1.3.3.cmml">b</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx12.p1.1.m1.1b"><apply id="Sx7.SSx12.p1.1.m1.1.1.cmml" xref="Sx7.SSx12.p1.1.m1.1.1"><csymbol cd="ambiguous" id="Sx7.SSx12.p1.1.m1.1.1.1.cmml" xref="Sx7.SSx12.p1.1.m1.1.1">subscript</csymbol><ci id="Sx7.SSx12.p1.1.m1.1.1.2.cmml" xref="Sx7.SSx12.p1.1.m1.1.1.2">𝑝</ci><apply id="Sx7.SSx12.p1.1.m1.1.1.3.cmml" xref="Sx7.SSx12.p1.1.m1.1.1.3"><times id="Sx7.SSx12.p1.1.m1.1.1.3.1.cmml" xref="Sx7.SSx12.p1.1.m1.1.1.3.1"></times><ci id="Sx7.SSx12.p1.1.m1.1.1.3.2.cmml" xref="Sx7.SSx12.p1.1.m1.1.1.3.2">𝑎</ci><ci id="Sx7.SSx12.p1.1.m1.1.1.3.3.cmml" xref="Sx7.SSx12.p1.1.m1.1.1.3.3">𝑏</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx12.p1.1.m1.1c">p_{ab}</annotation></semantics></math> (see
above) between marker <math id="Sx7.SSx12.p1.2.m2.1" class="ltx_Math" alttext="a" display="inline"><semantics id="Sx7.SSx12.p1.2.m2.1a"><mi id="Sx7.SSx12.p1.2.m2.1.1" xref="Sx7.SSx12.p1.2.m2.1.1.cmml">a</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx12.p1.2.m2.1b"><ci id="Sx7.SSx12.p1.2.m2.1.1.cmml" xref="Sx7.SSx12.p1.2.m2.1.1">𝑎</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx12.p1.2.m2.1c">a</annotation></semantics></math> to be placed on the map and each map bin
<math id="Sx7.SSx12.p1.3.m3.1" class="ltx_Math" alttext="b" display="inline"><semantics id="Sx7.SSx12.p1.3.m3.1a"><mi id="Sx7.SSx12.p1.3.m3.1.1" xref="Sx7.SSx12.p1.3.m3.1.1.cmml">b</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx12.p1.3.m3.1b"><ci id="Sx7.SSx12.p1.3.m3.1.1.cmml" xref="Sx7.SSx12.p1.3.m3.1.1">𝑏</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx12.p1.3.m3.1c">b</annotation></semantics></math>. Marker <math id="Sx7.SSx12.p1.4.m4.1" class="ltx_Math" alttext="a" display="inline"><semantics id="Sx7.SSx12.p1.4.m4.1a"><mi id="Sx7.SSx12.p1.4.m4.1.1" xref="Sx7.SSx12.p1.4.m4.1.1.cmml">a</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx12.p1.4.m4.1b"><ci id="Sx7.SSx12.p1.4.m4.1.1.cmml" xref="Sx7.SSx12.p1.4.m4.1.1">𝑎</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx12.p1.4.m4.1c">a</annotation></semantics></math> is anchored to the map at the position of the bin <math id="Sx7.SSx12.p1.5.m5.1" class="ltx_Math" alttext="b" display="inline"><semantics id="Sx7.SSx12.p1.5.m5.1a"><mi id="Sx7.SSx12.p1.5.m5.1.1" xref="Sx7.SSx12.p1.5.m5.1.1.cmml">b</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx12.p1.5.m5.1b"><ci id="Sx7.SSx12.p1.5.m5.1.1.cmml" xref="Sx7.SSx12.p1.5.m5.1.1">𝑏</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx12.p1.5.m5.1c">b</annotation></semantics></math>
which minimizes <math id="Sx7.SSx12.p1.6.m6.1" class="ltx_Math" alttext="p_{ab}" display="inline"><semantics id="Sx7.SSx12.p1.6.m6.1a"><msub id="Sx7.SSx12.p1.6.m6.1.1" xref="Sx7.SSx12.p1.6.m6.1.1.cmml"><mi id="Sx7.SSx12.p1.6.m6.1.1.2" xref="Sx7.SSx12.p1.6.m6.1.1.2.cmml">p</mi><mrow id="Sx7.SSx12.p1.6.m6.1.1.3" xref="Sx7.SSx12.p1.6.m6.1.1.3.cmml"><mi id="Sx7.SSx12.p1.6.m6.1.1.3.2" xref="Sx7.SSx12.p1.6.m6.1.1.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx12.p1.6.m6.1.1.3.1" xref="Sx7.SSx12.p1.6.m6.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx12.p1.6.m6.1.1.3.3" xref="Sx7.SSx12.p1.6.m6.1.1.3.3.cmml">b</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx12.p1.6.m6.1b"><apply id="Sx7.SSx12.p1.6.m6.1.1.cmml" xref="Sx7.SSx12.p1.6.m6.1.1"><csymbol cd="ambiguous" id="Sx7.SSx12.p1.6.m6.1.1.1.cmml" xref="Sx7.SSx12.p1.6.m6.1.1">subscript</csymbol><ci id="Sx7.SSx12.p1.6.m6.1.1.2.cmml" xref="Sx7.SSx12.p1.6.m6.1.1.2">𝑝</ci><apply id="Sx7.SSx12.p1.6.m6.1.1.3.cmml" xref="Sx7.SSx12.p1.6.m6.1.1.3"><times id="Sx7.SSx12.p1.6.m6.1.1.3.1.cmml" xref="Sx7.SSx12.p1.6.m6.1.1.3.1"></times><ci id="Sx7.SSx12.p1.6.m6.1.1.3.2.cmml" xref="Sx7.SSx12.p1.6.m6.1.1.3.2">𝑎</ci><ci id="Sx7.SSx12.p1.6.m6.1.1.3.3.cmml" xref="Sx7.SSx12.p1.6.m6.1.1.3.3">𝑏</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx12.p1.6.m6.1c">p_{ab}</annotation></semantics></math> if <math id="Sx7.SSx12.p1.7.m7.1" class="ltx_Math" alttext="p_{ab}&lt;10^{-6}" display="inline"><semantics id="Sx7.SSx12.p1.7.m7.1a"><mrow id="Sx7.SSx12.p1.7.m7.1.1" xref="Sx7.SSx12.p1.7.m7.1.1.cmml"><msub id="Sx7.SSx12.p1.7.m7.1.1.2" xref="Sx7.SSx12.p1.7.m7.1.1.2.cmml"><mi id="Sx7.SSx12.p1.7.m7.1.1.2.2" xref="Sx7.SSx12.p1.7.m7.1.1.2.2.cmml">p</mi><mrow id="Sx7.SSx12.p1.7.m7.1.1.2.3" xref="Sx7.SSx12.p1.7.m7.1.1.2.3.cmml"><mi id="Sx7.SSx12.p1.7.m7.1.1.2.3.2" xref="Sx7.SSx12.p1.7.m7.1.1.2.3.2.cmml">a</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx12.p1.7.m7.1.1.2.3.1" xref="Sx7.SSx12.p1.7.m7.1.1.2.3.1.cmml">​</mo><mi id="Sx7.SSx12.p1.7.m7.1.1.2.3.3" xref="Sx7.SSx12.p1.7.m7.1.1.2.3.3.cmml">b</mi></mrow></msub><mo id="Sx7.SSx12.p1.7.m7.1.1.1" xref="Sx7.SSx12.p1.7.m7.1.1.1.cmml">&lt;</mo><msup id="Sx7.SSx12.p1.7.m7.1.1.3" xref="Sx7.SSx12.p1.7.m7.1.1.3.cmml"><mn id="Sx7.SSx12.p1.7.m7.1.1.3.2" xref="Sx7.SSx12.p1.7.m7.1.1.3.2.cmml">10</mn><mrow id="Sx7.SSx12.p1.7.m7.1.1.3.3" xref="Sx7.SSx12.p1.7.m7.1.1.3.3.cmml"><mo id="Sx7.SSx12.p1.7.m7.1.1.3.3a" xref="Sx7.SSx12.p1.7.m7.1.1.3.3.cmml">−</mo><mn id="Sx7.SSx12.p1.7.m7.1.1.3.3.2" xref="Sx7.SSx12.p1.7.m7.1.1.3.3.2.cmml">6</mn></mrow></msup></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx12.p1.7.m7.1b"><apply id="Sx7.SSx12.p1.7.m7.1.1.cmml" xref="Sx7.SSx12.p1.7.m7.1.1"><lt id="Sx7.SSx12.p1.7.m7.1.1.1.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.1"></lt><apply id="Sx7.SSx12.p1.7.m7.1.1.2.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.2"><csymbol cd="ambiguous" id="Sx7.SSx12.p1.7.m7.1.1.2.1.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.2">subscript</csymbol><ci id="Sx7.SSx12.p1.7.m7.1.1.2.2.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.2.2">𝑝</ci><apply id="Sx7.SSx12.p1.7.m7.1.1.2.3.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.2.3"><times id="Sx7.SSx12.p1.7.m7.1.1.2.3.1.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.2.3.1"></times><ci id="Sx7.SSx12.p1.7.m7.1.1.2.3.2.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.2.3.2">𝑎</ci><ci id="Sx7.SSx12.p1.7.m7.1.1.2.3.3.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.2.3.3">𝑏</ci></apply></apply><apply id="Sx7.SSx12.p1.7.m7.1.1.3.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.3"><csymbol cd="ambiguous" id="Sx7.SSx12.p1.7.m7.1.1.3.1.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.3">superscript</csymbol><cn type="integer" id="Sx7.SSx12.p1.7.m7.1.1.3.2.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.3.2">10</cn><apply id="Sx7.SSx12.p1.7.m7.1.1.3.3.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.3.3"><minus id="Sx7.SSx12.p1.7.m7.1.1.3.3.1.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.3.3"></minus><cn type="integer" id="Sx7.SSx12.p1.7.m7.1.1.3.3.2.cmml" xref="Sx7.SSx12.p1.7.m7.1.1.3.3.2">6</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx12.p1.7.m7.1c">p_{ab}&lt;10^{-6}</annotation></semantics></math>.</p>
</div>
</section>
<section id="Sx7.SSx13" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">SNP density estimation </h3>

<div id="Sx7.SSx13.p1" class="ltx_para">
<p id="Sx7.SSx13.p1.1" class="ltx_p">Illumina reads were mapped to the assembled scaffold sequences with <span id="Sx7.SSx13.p1.1.1" class="ltx_text ltx_font_typewriter">stampy<cite class="ltx_cite ltx_citemacro_citep"><span id="Sx7.SSx13.p1.1.1.1.1" class="ltx_text ltx_font_serif">[</span><a href="#bib.bib57" title="" class="ltx_ref">57</a><span id="Sx7.SSx13.p1.1.1.2.2" class="ltx_text ltx_font_serif">]</span></cite></span> using default settings. For a
sample of 9,228 scaffolds with lengths ranging from 5.0-5.5 kb,
sequence variants were called with <span id="Sx7.SSx13.p1.1.2" class="ltx_text ltx_font_typewriter">SAMtools<cite class="ltx_cite ltx_citemacro_citep"><span id="Sx7.SSx13.p1.1.2.1.1" class="ltx_text ltx_font_serif">[</span><a href="#bib.bib58" title="" class="ltx_ref">58</a><span id="Sx7.SSx13.p1.1.2.2.2" class="ltx_text ltx_font_serif">]</span></cite></span>
using a variant quality score threshold of 50, and ignoring indel
positions.</p>
</div>
<div id="Sx7.SSx13.p2" class="ltx_para">
<p id="Sx7.SSx13.p2.1" class="ltx_p">A SNP density of 0.76 % in four haplotypes which
corresponds to a predicted rate of pairwise sequence differences per
site of <math id="Sx7.SSx13.p2.1.m1.1" class="ltx_Math" alttext="\theta" display="inline"><semantics id="Sx7.SSx13.p2.1.m1.1a"><mi id="Sx7.SSx13.p2.1.m1.1.1" xref="Sx7.SSx13.p2.1.m1.1.1.cmml">θ</mi><annotation-xml encoding="MathML-Content" id="Sx7.SSx13.p2.1.m1.1b"><ci id="Sx7.SSx13.p2.1.m1.1.1.cmml" xref="Sx7.SSx13.p2.1.m1.1.1">𝜃</ci></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx13.p2.1.m1.1c">\theta</annotation></semantics></math> = 0.0042 under the finite sites model of mutation and
the neutral coalescent model of the relationships among sampled
alleles<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib59" title="" class="ltx_ref">59</a>]</cite>.</p>
</div>
</section>
<section id="Sx7.SSx14" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Estimation of local recombination rate</h3>

<div id="Sx7.SSx14.p1" class="ltx_para">
<p id="Sx7.SSx14.p1.1" class="ltx_p">To estimate the local recombination rate for each map bin, we computed
the linear regression of map distance in number of markers on physical
distance using up to 10 neighboring map bins in each direction along
the map (or fewer for bins within 10 map bins of the end of the
linkage group). Map distance was calculated from recombination
fraction using Haldane’s map distance <math id="Sx7.SSx14.p1.1.m1.1" class="ltx_Math" alttext="-\frac{1}{2}log(1-2r)" display="inline"><semantics id="Sx7.SSx14.p1.1.m1.1a"><mrow id="Sx7.SSx14.p1.1.m1.1.1" xref="Sx7.SSx14.p1.1.m1.1.1.cmml"><mo id="Sx7.SSx14.p1.1.m1.1.1a" xref="Sx7.SSx14.p1.1.m1.1.1.cmml">−</mo><mrow id="Sx7.SSx14.p1.1.m1.1.1.1" xref="Sx7.SSx14.p1.1.m1.1.1.1.cmml"><mfrac id="Sx7.SSx14.p1.1.m1.1.1.1.3" xref="Sx7.SSx14.p1.1.m1.1.1.1.3.cmml"><mn id="Sx7.SSx14.p1.1.m1.1.1.1.3.2" xref="Sx7.SSx14.p1.1.m1.1.1.1.3.2.cmml">1</mn><mn id="Sx7.SSx14.p1.1.m1.1.1.1.3.3" xref="Sx7.SSx14.p1.1.m1.1.1.1.3.3.cmml">2</mn></mfrac><mo lspace="0em" rspace="0em" id="Sx7.SSx14.p1.1.m1.1.1.1.2" xref="Sx7.SSx14.p1.1.m1.1.1.1.2.cmml">​</mo><mi id="Sx7.SSx14.p1.1.m1.1.1.1.4" xref="Sx7.SSx14.p1.1.m1.1.1.1.4.cmml">l</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx14.p1.1.m1.1.1.1.2a" xref="Sx7.SSx14.p1.1.m1.1.1.1.2.cmml">​</mo><mi id="Sx7.SSx14.p1.1.m1.1.1.1.5" xref="Sx7.SSx14.p1.1.m1.1.1.1.5.cmml">o</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx14.p1.1.m1.1.1.1.2b" xref="Sx7.SSx14.p1.1.m1.1.1.1.2.cmml">​</mo><mi id="Sx7.SSx14.p1.1.m1.1.1.1.6" xref="Sx7.SSx14.p1.1.m1.1.1.1.6.cmml">g</mi><mo lspace="0em" rspace="0em" id="Sx7.SSx14.p1.1.m1.1.1.1.2c" xref="Sx7.SSx14.p1.1.m1.1.1.1.2.cmml">​</mo><mrow id="Sx7.SSx14.p1.1.m1.1.1.1.1.1" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.cmml"><mo stretchy="false" id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.2" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.cmml">(</mo><mrow id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.cmml"><mn id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.2" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.2.cmml">1</mn><mo id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.1" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.1.cmml">−</mo><mrow id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.cmml"><mn id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.2" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.2.cmml">2</mn><mo lspace="0em" rspace="0em" id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.1" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.3" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.3.cmml">r</mi></mrow></mrow><mo stretchy="false" id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.3" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx14.p1.1.m1.1b"><apply id="Sx7.SSx14.p1.1.m1.1.1.cmml" xref="Sx7.SSx14.p1.1.m1.1.1"><minus id="Sx7.SSx14.p1.1.m1.1.1.2.cmml" xref="Sx7.SSx14.p1.1.m1.1.1"></minus><apply id="Sx7.SSx14.p1.1.m1.1.1.1.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1"><times id="Sx7.SSx14.p1.1.m1.1.1.1.2.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.2"></times><apply id="Sx7.SSx14.p1.1.m1.1.1.1.3.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.3"><divide id="Sx7.SSx14.p1.1.m1.1.1.1.3.1.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.3"></divide><cn type="integer" id="Sx7.SSx14.p1.1.m1.1.1.1.3.2.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.3.2">1</cn><cn type="integer" id="Sx7.SSx14.p1.1.m1.1.1.1.3.3.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.3.3">2</cn></apply><ci id="Sx7.SSx14.p1.1.m1.1.1.1.4.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.4">𝑙</ci><ci id="Sx7.SSx14.p1.1.m1.1.1.1.5.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.5">𝑜</ci><ci id="Sx7.SSx14.p1.1.m1.1.1.1.6.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.6">𝑔</ci><apply id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1"><minus id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.1.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.1"></minus><cn type="integer" id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.2.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.2">1</cn><apply id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3"><times id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.1.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.1"></times><cn type="integer" id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.2.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.2">2</cn><ci id="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.3.cmml" xref="Sx7.SSx14.p1.1.m1.1.1.1.1.1.1.3.3">𝑟</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx14.p1.1.m1.1c">-\frac{1}{2}log(1-2r)</annotation></semantics></math> <cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib60" title="" class="ltx_ref">60</a>]</cite>.</p>
</div>
</section>
<section id="Sx7.SSx15" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Ancestral linkage group conservation</h3>

<div id="Sx7.SSx15.p1" class="ltx_para">
<p id="Sx7.SSx15.p1.1" class="ltx_p">To compare the genome organization in <em id="Sx7.SSx15.p1.1.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> to the ancestral
metazoan ALGs, we used the reciprocal best blast hit (RBH) orthology
criterion in an alignment of the <em id="Sx7.SSx15.p1.1.2" class="ltx_emph ltx_font_italic">Ixodes scapularis</em> predicted
proteins<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib32" title="" class="ltx_ref">32</a>]</cite> to the consensus sequences for
the marker scaffolds. <em id="Sx7.SSx15.p1.1.3" class="ltx_emph ltx_font_italic">L. polyphemus</em>
scaffolds with RBH of e-value <math id="Sx7.SSx15.p1.1.m1.1" class="ltx_Math" alttext="&lt;=10^{\mathrm{-6}}" display="inline"><semantics id="Sx7.SSx15.p1.1.m1.1a"><mrow id="Sx7.SSx15.p1.1.m1.1.1" xref="Sx7.SSx15.p1.1.m1.1.1.cmml"><mi id="Sx7.SSx15.p1.1.m1.1.1.2" xref="Sx7.SSx15.p1.1.m1.1.1.2.cmml"></mi><mo id="Sx7.SSx15.p1.1.m1.1.1.1" xref="Sx7.SSx15.p1.1.m1.1.1.1.cmml">&lt;=</mo><msup id="Sx7.SSx15.p1.1.m1.1.1.3" xref="Sx7.SSx15.p1.1.m1.1.1.3.cmml"><mn id="Sx7.SSx15.p1.1.m1.1.1.3.2" xref="Sx7.SSx15.p1.1.m1.1.1.3.2.cmml">10</mn><mrow id="Sx7.SSx15.p1.1.m1.1.1.3.3" xref="Sx7.SSx15.p1.1.m1.1.1.3.3.cmml"><mo id="Sx7.SSx15.p1.1.m1.1.1.3.3a" xref="Sx7.SSx15.p1.1.m1.1.1.3.3.cmml">−</mo><mn id="Sx7.SSx15.p1.1.m1.1.1.3.3.2" xref="Sx7.SSx15.p1.1.m1.1.1.3.3.2.cmml">6</mn></mrow></msup></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx15.p1.1.m1.1b"><apply id="Sx7.SSx15.p1.1.m1.1.1.cmml" xref="Sx7.SSx15.p1.1.m1.1.1"><leq id="Sx7.SSx15.p1.1.m1.1.1.1.cmml" xref="Sx7.SSx15.p1.1.m1.1.1.1"></leq><csymbol cd="latexml" id="Sx7.SSx15.p1.1.m1.1.1.2.cmml" xref="Sx7.SSx15.p1.1.m1.1.1.2">absent</csymbol><apply id="Sx7.SSx15.p1.1.m1.1.1.3.cmml" xref="Sx7.SSx15.p1.1.m1.1.1.3"><csymbol cd="ambiguous" id="Sx7.SSx15.p1.1.m1.1.1.3.1.cmml" xref="Sx7.SSx15.p1.1.m1.1.1.3">superscript</csymbol><cn type="integer" id="Sx7.SSx15.p1.1.m1.1.1.3.2.cmml" xref="Sx7.SSx15.p1.1.m1.1.1.3.2">10</cn><apply id="Sx7.SSx15.p1.1.m1.1.1.3.3.cmml" xref="Sx7.SSx15.p1.1.m1.1.1.3.3"><minus id="Sx7.SSx15.p1.1.m1.1.1.3.3.1.cmml" xref="Sx7.SSx15.p1.1.m1.1.1.3.3"></minus><cn type="integer" id="Sx7.SSx15.p1.1.m1.1.1.3.3.2.cmml" xref="Sx7.SSx15.p1.1.m1.1.1.3.3.2">6</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx15.p1.1.m1.1c">&lt;=10^{\mathrm{-6}}</annotation></semantics></math> were assigned to the same
ancestral bilaterian gene orthology group as their <em id="Sx7.SSx15.p1.1.4" class="ltx_emph ltx_font_italic">I. scapularis</em> ortholog,
and thereby with human genes.
Regions of the map were tested for enrichment in genes from particular
ancestral linkage groups with Fisher’s Exact Test, and breakpoints in
ancestral linkage group composition were identified using a hidden
Markov model, as previously
described<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib6" title="" class="ltx_ref">6</a>, <a href="#bib.bib7" title="" class="ltx_ref">7</a>]</cite>.</p>
</div>
</section>
<section id="Sx7.SSx16" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Homeobox gene modeling </h3>

<div id="Sx7.SSx16.p1" class="ltx_para">
<p id="Sx7.SSx16.p1.1" class="ltx_p">We identified 155 marker scaffolds with a tblastx alignment of
e-value <math id="Sx7.SSx16.p1.1.m1.1" class="ltx_Math" alttext="&lt;10^{\mathrm{-6}}" display="inline"><semantics id="Sx7.SSx16.p1.1.m1.1a"><mrow id="Sx7.SSx16.p1.1.m1.1.1" xref="Sx7.SSx16.p1.1.m1.1.1.cmml"><mi id="Sx7.SSx16.p1.1.m1.1.1.2" xref="Sx7.SSx16.p1.1.m1.1.1.2.cmml"></mi><mo id="Sx7.SSx16.p1.1.m1.1.1.1" xref="Sx7.SSx16.p1.1.m1.1.1.1.cmml">&lt;</mo><msup id="Sx7.SSx16.p1.1.m1.1.1.3" xref="Sx7.SSx16.p1.1.m1.1.1.3.cmml"><mn id="Sx7.SSx16.p1.1.m1.1.1.3.2" xref="Sx7.SSx16.p1.1.m1.1.1.3.2.cmml">10</mn><mrow id="Sx7.SSx16.p1.1.m1.1.1.3.3" xref="Sx7.SSx16.p1.1.m1.1.1.3.3.cmml"><mo id="Sx7.SSx16.p1.1.m1.1.1.3.3a" xref="Sx7.SSx16.p1.1.m1.1.1.3.3.cmml">−</mo><mn id="Sx7.SSx16.p1.1.m1.1.1.3.3.2" xref="Sx7.SSx16.p1.1.m1.1.1.3.3.2.cmml">6</mn></mrow></msup></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx16.p1.1.m1.1b"><apply id="Sx7.SSx16.p1.1.m1.1.1.cmml" xref="Sx7.SSx16.p1.1.m1.1.1"><lt id="Sx7.SSx16.p1.1.m1.1.1.1.cmml" xref="Sx7.SSx16.p1.1.m1.1.1.1"></lt><csymbol cd="latexml" id="Sx7.SSx16.p1.1.m1.1.1.2.cmml" xref="Sx7.SSx16.p1.1.m1.1.1.2">absent</csymbol><apply id="Sx7.SSx16.p1.1.m1.1.1.3.cmml" xref="Sx7.SSx16.p1.1.m1.1.1.3"><csymbol cd="ambiguous" id="Sx7.SSx16.p1.1.m1.1.1.3.1.cmml" xref="Sx7.SSx16.p1.1.m1.1.1.3">superscript</csymbol><cn type="integer" id="Sx7.SSx16.p1.1.m1.1.1.3.2.cmml" xref="Sx7.SSx16.p1.1.m1.1.1.3.2">10</cn><apply id="Sx7.SSx16.p1.1.m1.1.1.3.3.cmml" xref="Sx7.SSx16.p1.1.m1.1.1.3.3"><minus id="Sx7.SSx16.p1.1.m1.1.1.3.3.1.cmml" xref="Sx7.SSx16.p1.1.m1.1.1.3.3"></minus><cn type="integer" id="Sx7.SSx16.p1.1.m1.1.1.3.3.2.cmml" xref="Sx7.SSx16.p1.1.m1.1.1.3.3.2">6</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx16.p1.1.m1.1c">&lt;10^{\mathrm{-6}}</annotation></semantics></math> to a set of chelicerate homeobox
gene sequences downloaded from Genbank using the
NCBI online query interface (genbank accessions
AF071402.1, AF071403.1, AF071405.1, AF071406.1, AF071407.1, AF085352.1, AF151986.1, AF151987.1, AF151988.1, AF151989.1, AF151990.1, AF151991.1, AF151992.1, AF151993.1, AF151994.1, AF151995.1, AF151996.1, AF151997.1, AF151998.1, AF151999.1, AF152000.1, AF237818.1, AJ005643.1, AJ007431.1, AJ007432.1, AJ007433.1, AJ007434.1, AJ007435.1, AJ007436.1, AJ007437.1, AM419029.1, AM419030.1, AM419031.1, AM419032.1, DQ315728.1, DQ315729.1, DQ315730.1, DQ315731.1, DQ315732.1, DQ315733.1, DQ315734.1, DQ315735.1, DQ315736.1, DQ315737.1, DQ315738.1, DQ315739.1, DQ315740.1, DQ315741.1, DQ315742.1, DQ315743.1, DQ315744.1, EU870887.1, EU870888.1, EU870889.1, HE608680.1, HE608681.1, HE608682.1, HE805493.1, HE805494.1, HE805495.1, HE805496.1, HE805497.1, HE805498.1, HE805499.1, HE805500.1, HE805501.1, HE805502.1, S70005.1, S70006.1, S70008.1, and S70010.1). The reads of each marker (those with best <span id="Sx7.SSx16.p1.1.1" class="ltx_text ltx_font_typewriter">stampy<cite class="ltx_cite ltx_citemacro_citep"><span id="Sx7.SSx16.p1.1.1.1.1" class="ltx_text ltx_font_serif">[</span><a href="#bib.bib57" title="" class="ltx_ref">57</a><span id="Sx7.SSx16.p1.1.1.2.2" class="ltx_text ltx_font_serif">]</span></cite></span> alignment to the scaffold) were reassembled with <span id="Sx7.SSx16.p1.1.2" class="ltx_text ltx_font_typewriter">PHRAP<cite class="ltx_cite ltx_citemacro_citep"><span id="Sx7.SSx16.p1.1.2.1.1" class="ltx_text ltx_font_serif">[</span><a href="#bib.bib61" title="" class="ltx_ref">61</a><span id="Sx7.SSx16.p1.1.2.2.2" class="ltx_text ltx_font_serif">]</span></cite></span>, with default parameters. The resulting contigs were aligned to a collection of homeobox-containing protein sequences (genbank accessions NP 001034497.1, NP 001034510.1, AAL71874.1, NP 001034505.1, NP 001036813.1, CAA66399.1, NP 001107762.1, NP 001107807.1, EEZ99256.1, NP 001034519.1, NP 476954.1, NP 032840.1, NP 031699.2, AAI37770.1, EEN68949.1, NP 523700.2, NP 001034511.2, AAK16421.1, and AAK16422.1) with exonerate<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib62" title="" class="ltx_ref">62</a>]</cite> in protein-to-genome mode. For each contig, the amino acid sequence predicted by the highest-scoring exonerate alignment was used in subsequent phylogenetic analysis, resulting in 104 putative homeobox-containing markers ranging in length from 18 to 147 amino acids.</p>
</div>
</section>
<section id="Sx7.SSx17" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Phylogenetic analysis of homeobox genes</h3>

<div id="Sx7.SSx17.p1" class="ltx_para">
<p id="Sx7.SSx17.p1.1" class="ltx_p">A multiple sequence alignment of the predicted homeobox sequences
combined with a collection of representative sequences from various
classes of homeobox genes was constructed with muscle3.8.31<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib63" title="" class="ltx_ref">63</a>]</cite> using default settings. The resulting
alignment was trimmed to a 63 amino acid segment spanning the
conserved homeodomain, and sequences with more than 50% gaps were
removed, leaving 93 predicted <em id="Sx7.SSx17.p1.1.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> homeobox genes in the
analysis. Bayesian phylogenetic analysis was carried out on the
resulting 178 taxon, 63 amino acid character matrix (Supplementary
File HombeoboxTable.html) using MrBayes
v3.2.1<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib64" title="" class="ltx_ref">64</a>]</cite> using a mixed model of amino acid
substituions, gamma-distributed rate variation among sites with fixed
shape parameter <math id="Sx7.SSx17.p1.1.m1.1" class="ltx_Math" alttext="\alpha=1.0" display="inline"><semantics id="Sx7.SSx17.p1.1.m1.1a"><mrow id="Sx7.SSx17.p1.1.m1.1.1" xref="Sx7.SSx17.p1.1.m1.1.1.cmml"><mi id="Sx7.SSx17.p1.1.m1.1.1.2" xref="Sx7.SSx17.p1.1.m1.1.1.2.cmml">α</mi><mo id="Sx7.SSx17.p1.1.m1.1.1.1" xref="Sx7.SSx17.p1.1.m1.1.1.1.cmml">=</mo><mn id="Sx7.SSx17.p1.1.m1.1.1.3" xref="Sx7.SSx17.p1.1.m1.1.1.3.cmml">1.0</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx17.p1.1.m1.1b"><apply id="Sx7.SSx17.p1.1.m1.1.1.cmml" xref="Sx7.SSx17.p1.1.m1.1.1"><eq id="Sx7.SSx17.p1.1.m1.1.1.1.cmml" xref="Sx7.SSx17.p1.1.m1.1.1.1"></eq><ci id="Sx7.SSx17.p1.1.m1.1.1.2.cmml" xref="Sx7.SSx17.p1.1.m1.1.1.2">𝛼</ci><cn type="float" id="Sx7.SSx17.p1.1.m1.1.1.3.cmml" xref="Sx7.SSx17.p1.1.m1.1.1.3">1.0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx17.p1.1.m1.1c">\alpha=1.0</annotation></semantics></math>, alignment gaps treated as missing data,
2,000,000 Monte Carlo steps, two independent runs with four Monte
Carlo chains, and the initial 25% of sampled trees were discarded as
“burn-in”. Monte Carlo appeared to reach convergence, with an average
standard deviation of the split frequencies of 0.022.
The majority-rule consensus of the sampled trees is shown in Figures
<a href="#Sx7.F6" title="Figure 6 ‣ Phylogenetic analysis of homeobox genes ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a> and <a href="#Sx7.F7" title="Figure 7 ‣ Phylogenetic analysis of homeobox genes ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>, and well-supported gene
clades (posterior probability greater than 0.95) were used to group
the predicted <em id="Sx7.SSx17.p1.1.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> genes into classes. The table in
Supplementary File HombeoboxTable.html lists the reassembled marker
contigs, their inferred hox gene class, and maximum likelihood map
positions. Predicted genes were anchored to the map as
described above.</p>
</div>
<figure id="Sx7.F6" class="ltx_figure"><img src="/html/1309.7402/assets/x5.png" id="Sx7.F6.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="475" height="615" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 6: </span>Unrooted phylogenetic tree of homeobox sequences (part 1). Nodes are labeled with Bayesian posterior probabilities. Highly supported partitions used to classify <em id="Sx7.F6.3.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> sequences are drawn in read, with the abbreviation for the class shown in large letters. <em id="Sx7.F6.4.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> homeobox sequences not grouped into one of these highly supported partitions are assigned to class “?”. For ease of display, a large subtree consisting of HOX and parahox genes has been pruned at the position labeled “HOX”, and is shown in Figure <a href="#Sx7.F7" title="Figure 7 ‣ Phylogenetic analysis of homeobox genes ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>.</figcaption>
</figure>
<figure id="Sx7.F7" class="ltx_figure"><img src="/html/1309.7402/assets/x6.png" id="Sx7.F7.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="475" height="615" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 7: </span>Phylogenetic tree of homeobox sequences, part 2. The rooted subtree pruned from the tree in Figure <a href="#Sx7.F6" title="Figure 6 ‣ Phylogenetic analysis of homeobox genes ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a>. Nodes are labeled with Bayesian posterior probabilities. Highly supported partitions used to classify <em id="Sx7.F7.3.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> sequences are drawn in read, with the abbreviation for the class shown in large letters. <em id="Sx7.F7.4.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> homeobox sequences not grouped into one of these highly supported partitions are assigned to class “hox?”.</figcaption>
</figure>
</section>
<section id="Sx7.SSx18" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">Genomic distribution of paralogs</h3>

<div id="Sx7.SSx18.p1" class="ltx_para">
<p id="Sx7.SSx18.p1.1" class="ltx_p">We identified 2716 pairs of <em id="Sx7.SSx18.p1.1.1" class="ltx_emph ltx_font_italic">Limulus</em> markers that can both be placed
on the map and have their best translated alignment to the same
<em id="Sx7.SSx18.p1.1.2" class="ltx_emph ltx_font_italic">I. scapularis</em> gene. (<em id="Sx7.SSx18.p1.1.3" class="ltx_emph ltx_font_italic">I. scapularis</em> genes with more than five
best-hit markers were excluded from seeding such pairs.) To estimate
the synonymous sequence divergence between pairs of candidate
<em id="Sx7.SSx18.p1.1.4" class="ltx_emph ltx_font_italic">L. polyphemus</em> paralogous gene pairs and <em id="Sx7.SSx18.p1.1.5" class="ltx_emph ltx_font_italic">L. polyphemus</em> genes and
their <em id="Sx7.SSx18.p1.1.6" class="ltx_emph ltx_font_italic">T. tridentatus</em> orthologs, we constructed codon alignments of
predicted coding sequence for estimation of synonymous sequence
divergence. Conserved clusters of paralogs were identified using a
variant of the “max-gap” criterion<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite> in
which two genes are placed in the same cluster if they and their
paralogs lie within threshold distance. In the analysis presented
here, the distance threshold used was 500 markers.</p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
</section>
<section id="Sx7.SSx19" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">K<sub id="Sx7.SSx19.3.1" class="ltx_sub"><span id="Sx7.SSx19.3.1.1" class="ltx_text ltx_font_italic">a</span></sub> and K<sub id="Sx7.SSx19.4.2" class="ltx_sub"><span id="Sx7.SSx19.4.2.1" class="ltx_text ltx_font_italic">s</span></sub> estimation for paralogs and <em id="Sx7.SSx19.5.3" class="ltx_emph ltx_font_italic">T. tridentatus</em> orthologs </h3>

<div id="Sx7.SSx19.p1" class="ltx_para">
<p id="Sx7.SSx19.p1.1" class="ltx_p">Figure <a href="#Sx7.F8" title="Figure 8 ‣ Ka and Ks estimation for paralogs and T. tridentatus orthologs ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a> shows the distribution across the map of pairs of candidate paralogs.</p>
</div>
<figure id="Sx7.F8" class="ltx_figure"><img src="/html/1309.7402/assets/x7.png" id="Sx7.F8.g1" class="ltx_graphics ltx_centering ltx_img_square" width="392" height="384" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 8: </span>Genomic distribution of candidate paralogs</figcaption>
</figure>
<div id="Sx7.SSx19.p2" class="ltx_para">
<p id="Sx7.SSx19.p2.1" class="ltx_p">To estimate the synonymous sequence divergence between pairs of candidate <em id="Sx7.SSx19.p2.1.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> paralogous gene pairs, and between <em id="Sx7.SSx19.p2.1.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> genes and their <em id="Sx7.SSx19.p2.1.3" class="ltx_emph ltx_font_italic">T. tridentatus</em> orthologs, we followed the following protocol.</p>
</div>
<div id="Sx7.SSx19.p3" class="ltx_para">
<ol id="Sx7.I2" class="ltx_enumerate">
<li id="Sx7.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="Sx7.I2.i1.p1" class="ltx_para">
<p id="Sx7.I2.i1.p1.1" class="ltx_p">Reassemble reads from each marker with <span id="Sx7.I2.i1.p1.1.1" class="ltx_text ltx_font_typewriter">PHRAP<cite class="ltx_cite ltx_citemacro_citep"><span id="Sx7.I2.i1.p1.1.1.1.1" class="ltx_text ltx_font_serif">[</span><a href="#bib.bib61" title="" class="ltx_ref">61</a><span id="Sx7.I2.i1.p1.1.1.2.2" class="ltx_text ltx_font_serif">]</span></cite></span>, and create a
predicted coding sequence using exonerate, as described for the
annotation of homeobox gene models (see <a href="#Sx7.SSx16" title="Homeobox gene modeling ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_title">Homeobox gene modeling</span></a> above).</p>
</div>
</li>
<li id="Sx7.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="Sx7.I2.i2.p1" class="ltx_para">
<p id="Sx7.I2.i2.p1.1" class="ltx_p">Combine the exonerate alignments of codons to amino acids to
create an alignment of codons for either a pair of
<em id="Sx7.I2.i2.p1.1.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> sequences, or for a <em id="Sx7.I2.i2.p1.1.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> - <em id="Sx7.I2.i2.p1.1.3" class="ltx_emph ltx_font_italic">T. tridentatus</em> sequence pair.</p>
</div>
</li>
<li id="Sx7.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span> 
<div id="Sx7.I2.i3.p1" class="ltx_para">
<p id="Sx7.I2.i3.p1.2" class="ltx_p">Use the method of Yang and Nielsen<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib65" title="" class="ltx_ref">65</a>]</cite> to estimate the synonymous and
non-synonymous substitution rates K<sub id="Sx7.I2.i3.p1.2.1" class="ltx_sub"><span id="Sx7.I2.i3.p1.2.1.1" class="ltx_text ltx_font_italic">a</span></sub> and K<sub id="Sx7.I2.i3.p1.2.2" class="ltx_sub"><span id="Sx7.I2.i3.p1.2.2.1" class="ltx_text ltx_font_italic">s</span></sub>, as implemented in
the KaKsCalculator package<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib66" title="" class="ltx_ref">66</a>]</cite>.</p>
</div>
</li>
<li id="Sx7.I2.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">4.</span> 
<div id="Sx7.I2.i4.p1" class="ltx_para">
<p id="Sx7.I2.i4.p1.2" class="ltx_p">Discard estimates based on fewer than 30 sites (30 synonymous sites for estimates of <math id="Sx7.I2.i4.p1.1.m1.1" class="ltx_Math" alttext="K_{s}" display="inline"><semantics id="Sx7.I2.i4.p1.1.m1.1a"><msub id="Sx7.I2.i4.p1.1.m1.1.1" xref="Sx7.I2.i4.p1.1.m1.1.1.cmml"><mi id="Sx7.I2.i4.p1.1.m1.1.1.2" xref="Sx7.I2.i4.p1.1.m1.1.1.2.cmml">K</mi><mi id="Sx7.I2.i4.p1.1.m1.1.1.3" xref="Sx7.I2.i4.p1.1.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.I2.i4.p1.1.m1.1b"><apply id="Sx7.I2.i4.p1.1.m1.1.1.cmml" xref="Sx7.I2.i4.p1.1.m1.1.1"><csymbol cd="ambiguous" id="Sx7.I2.i4.p1.1.m1.1.1.1.cmml" xref="Sx7.I2.i4.p1.1.m1.1.1">subscript</csymbol><ci id="Sx7.I2.i4.p1.1.m1.1.1.2.cmml" xref="Sx7.I2.i4.p1.1.m1.1.1.2">𝐾</ci><ci id="Sx7.I2.i4.p1.1.m1.1.1.3.cmml" xref="Sx7.I2.i4.p1.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.I2.i4.p1.1.m1.1c">K_{s}</annotation></semantics></math>, non-synonymous sites for <math id="Sx7.I2.i4.p1.2.m2.1" class="ltx_Math" alttext="K_{a}" display="inline"><semantics id="Sx7.I2.i4.p1.2.m2.1a"><msub id="Sx7.I2.i4.p1.2.m2.1.1" xref="Sx7.I2.i4.p1.2.m2.1.1.cmml"><mi id="Sx7.I2.i4.p1.2.m2.1.1.2" xref="Sx7.I2.i4.p1.2.m2.1.1.2.cmml">K</mi><mi id="Sx7.I2.i4.p1.2.m2.1.1.3" xref="Sx7.I2.i4.p1.2.m2.1.1.3.cmml">a</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.I2.i4.p1.2.m2.1b"><apply id="Sx7.I2.i4.p1.2.m2.1.1.cmml" xref="Sx7.I2.i4.p1.2.m2.1.1"><csymbol cd="ambiguous" id="Sx7.I2.i4.p1.2.m2.1.1.1.cmml" xref="Sx7.I2.i4.p1.2.m2.1.1">subscript</csymbol><ci id="Sx7.I2.i4.p1.2.m2.1.1.2.cmml" xref="Sx7.I2.i4.p1.2.m2.1.1.2">𝐾</ci><ci id="Sx7.I2.i4.p1.2.m2.1.1.3.cmml" xref="Sx7.I2.i4.p1.2.m2.1.1.3">𝑎</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.I2.i4.p1.2.m2.1c">K_{a}</annotation></semantics></math>).</p>
</div>
</li>
</ol>
</div>
<div id="Sx7.SSx19.p4" class="ltx_para">
<p id="Sx7.SSx19.p4.1" class="ltx_p">GenBank accessions for <em id="Sx7.SSx19.p4.1.1" class="ltx_emph ltx_font_italic">Tachypleus tridentatus</em> mRNA clones: JQ966943, AB353281, AB353280, HM156111, HQ221882, HQ221883, HQ221881, HQ386702, HM852953, TATTPP, TATPROCLOT, FN582225, FN582226, AF467804, AF227150, GQ260127, AF264067, AF264068, AB353279, AB005542, TATLICI, TATTGL, TATCFGB, TATLFC1, TATLFC2, AB201713, TATCFGA, TATLICI2, CS423581, CS423579, AB028144, AB201778, AB201776, AB201774, AB201772, AB201770, AB201768, AB201766, AB201779, AB201777, AB201775, AB201773, AB201771, AB201769, AB201767, AB201765, AB105059, AB002814, AX763473, TATCFBP, AB076186, AB076185, X04192, TATHCLL, AB037394, AB019116, AB019114, AB019112, AB019110, AB019108, AB019106, AB019104, AB019102, AB019100, AB019098, AB019096, AB019117, AB019115, AB019113, AB019111, AB019109, AB019107, AB019105, AB019103, AB019101, AB019099, AB019097, AB023783, AB024738, AB024739, AB024737, AB017484, D87214, D85756, D85341.</p>
</div>
<div id="Sx7.SSx19.p5" class="ltx_para">
<p id="Sx7.SSx19.p5.6" class="ltx_p">Figure <a href="#Sx7.F9" title="Figure 9 ‣ Ka and Ks estimation for paralogs and T. tridentatus orthologs ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">9</span></a> shows the distributions of K<sub id="Sx7.SSx19.p5.6.1" class="ltx_sub"><span id="Sx7.SSx19.p5.6.1.1" class="ltx_text ltx_font_italic">a</span></sub> and K<sub id="Sx7.SSx19.p5.6.2" class="ltx_sub"><span id="Sx7.SSx19.p5.6.2.1" class="ltx_text ltx_font_italic">s</span></sub> for paralogs and <em id="Sx7.SSx19.p5.6.3" class="ltx_emph ltx_font_italic">T. tridentatus</em> orthologs. To estimate the number and age of peaks in the un-saturated range<cite class="ltx_cite ltx_citemacro_citep">[<a href="#bib.bib33" title="" class="ltx_ref">33</a>]</cite> of the K<sub id="Sx7.SSx19.p5.6.4" class="ltx_sub"><span id="Sx7.SSx19.p5.6.4.1" class="ltx_text ltx_font_italic">s</span></sub> distribution (and of putative WGD events),
we fit a series of univariate normal mixture models, with 1, 2, 3, and
4 components to the paralog <math id="Sx7.SSx19.p5.4.m4.1" class="ltx_Math" alttext="K_{s}" display="inline"><semantics id="Sx7.SSx19.p5.4.m4.1a"><msub id="Sx7.SSx19.p5.4.m4.1.1" xref="Sx7.SSx19.p5.4.m4.1.1.cmml"><mi id="Sx7.SSx19.p5.4.m4.1.1.2" xref="Sx7.SSx19.p5.4.m4.1.1.2.cmml">K</mi><mi id="Sx7.SSx19.p5.4.m4.1.1.3" xref="Sx7.SSx19.p5.4.m4.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.SSx19.p5.4.m4.1b"><apply id="Sx7.SSx19.p5.4.m4.1.1.cmml" xref="Sx7.SSx19.p5.4.m4.1.1"><csymbol cd="ambiguous" id="Sx7.SSx19.p5.4.m4.1.1.1.cmml" xref="Sx7.SSx19.p5.4.m4.1.1">subscript</csymbol><ci id="Sx7.SSx19.p5.4.m4.1.1.2.cmml" xref="Sx7.SSx19.p5.4.m4.1.1.2">𝐾</ci><ci id="Sx7.SSx19.p5.4.m4.1.1.3.cmml" xref="Sx7.SSx19.p5.4.m4.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx19.p5.4.m4.1c">K_{s}</annotation></semantics></math> distribution in the
range <math id="Sx7.SSx19.p5.5.m5.1" class="ltx_Math" alttext="0&lt;K_{s}&lt;2.5" display="inline"><semantics id="Sx7.SSx19.p5.5.m5.1a"><mrow id="Sx7.SSx19.p5.5.m5.1.1" xref="Sx7.SSx19.p5.5.m5.1.1.cmml"><mn id="Sx7.SSx19.p5.5.m5.1.1.2" xref="Sx7.SSx19.p5.5.m5.1.1.2.cmml">0</mn><mo id="Sx7.SSx19.p5.5.m5.1.1.3" xref="Sx7.SSx19.p5.5.m5.1.1.3.cmml">&lt;</mo><msub id="Sx7.SSx19.p5.5.m5.1.1.4" xref="Sx7.SSx19.p5.5.m5.1.1.4.cmml"><mi id="Sx7.SSx19.p5.5.m5.1.1.4.2" xref="Sx7.SSx19.p5.5.m5.1.1.4.2.cmml">K</mi><mi id="Sx7.SSx19.p5.5.m5.1.1.4.3" xref="Sx7.SSx19.p5.5.m5.1.1.4.3.cmml">s</mi></msub><mo id="Sx7.SSx19.p5.5.m5.1.1.5" xref="Sx7.SSx19.p5.5.m5.1.1.5.cmml">&lt;</mo><mn id="Sx7.SSx19.p5.5.m5.1.1.6" xref="Sx7.SSx19.p5.5.m5.1.1.6.cmml">2.5</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.SSx19.p5.5.m5.1b"><apply id="Sx7.SSx19.p5.5.m5.1.1.cmml" xref="Sx7.SSx19.p5.5.m5.1.1"><and id="Sx7.SSx19.p5.5.m5.1.1a.cmml" xref="Sx7.SSx19.p5.5.m5.1.1"></and><apply id="Sx7.SSx19.p5.5.m5.1.1b.cmml" xref="Sx7.SSx19.p5.5.m5.1.1"><lt id="Sx7.SSx19.p5.5.m5.1.1.3.cmml" xref="Sx7.SSx19.p5.5.m5.1.1.3"></lt><cn type="integer" id="Sx7.SSx19.p5.5.m5.1.1.2.cmml" xref="Sx7.SSx19.p5.5.m5.1.1.2">0</cn><apply id="Sx7.SSx19.p5.5.m5.1.1.4.cmml" xref="Sx7.SSx19.p5.5.m5.1.1.4"><csymbol cd="ambiguous" id="Sx7.SSx19.p5.5.m5.1.1.4.1.cmml" xref="Sx7.SSx19.p5.5.m5.1.1.4">subscript</csymbol><ci id="Sx7.SSx19.p5.5.m5.1.1.4.2.cmml" xref="Sx7.SSx19.p5.5.m5.1.1.4.2">𝐾</ci><ci id="Sx7.SSx19.p5.5.m5.1.1.4.3.cmml" xref="Sx7.SSx19.p5.5.m5.1.1.4.3">𝑠</ci></apply></apply><apply id="Sx7.SSx19.p5.5.m5.1.1c.cmml" xref="Sx7.SSx19.p5.5.m5.1.1"><lt id="Sx7.SSx19.p5.5.m5.1.1.5.cmml" xref="Sx7.SSx19.p5.5.m5.1.1.5"></lt><share href="#Sx7.SSx19.p5.5.m5.1.1.4.cmml" id="Sx7.SSx19.p5.5.m5.1.1d.cmml" xref="Sx7.SSx19.p5.5.m5.1.1"></share><cn type="float" id="Sx7.SSx19.p5.5.m5.1.1.6.cmml" xref="Sx7.SSx19.p5.5.m5.1.1.6">2.5</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.SSx19.p5.5.m5.1c">0&lt;K_{s}&lt;2.5</annotation></semantics></math> and selected
the best model on the basis of Bayesian Information Criterion (Table <a href="#Sx7.T3" title="Table 3 ‣ Ka and Ks estimation for paralogs and T. tridentatus orthologs ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>).
The best model had two components, with means at 0.7 and 1.45 substitutions per site. The position of the peak at lowest K<sub id="Sx7.SSx19.p5.6.5" class="ltx_sub"><span id="Sx7.SSx19.p5.6.5.1" class="ltx_text ltx_font_italic">s</span></sub> was not sensitive to the addition of more mixture components.
Figure <a href="#Sx7.F10" title="Figure 10 ‣ Ka and Ks estimation for paralogs and T. tridentatus orthologs ‣ Methods" class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a> shows a comparison of the distribution and the components of the best-fitting model.
Gaussian mixture models were estimated in <span id="Sx7.SSx19.p5.6.6" class="ltx_text ltx_font_typewriter">R</span> with <span id="Sx7.SSx19.p5.6.7" class="ltx_text ltx_font_typewriter">mixtools<cite class="ltx_cite ltx_citemacro_citep"><span id="Sx7.SSx19.p5.6.7.1.1" class="ltx_text ltx_font_serif">[</span><a href="#bib.bib67" title="" class="ltx_ref">67</a><span id="Sx7.SSx19.p5.6.7.2.2" class="ltx_text ltx_font_serif">]</span></cite></span>.</p>
</div>
<figure id="Sx7.T3" class="ltx_table">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 3: </span>Mixture model fits to K<sub id="Sx7.T3.4.1" class="ltx_sub"><span id="Sx7.T3.4.1.1" class="ltx_text ltx_font_italic">s</span></sub> distribution. N is the number of mixture components, k the number model parameters, ln(L) the log likelihood of the data under the best fit model, BIC the Bayesian information criterion, AIC the Akaike information criterion.</figcaption>
<table id="Sx7.T3.5" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="Sx7.T3.5.1.1" class="ltx_tr">
<th id="Sx7.T3.5.1.1.1" class="ltx_td ltx_align_right ltx_th ltx_th_column">N</th>
<th id="Sx7.T3.5.1.1.2" class="ltx_td ltx_align_right ltx_th ltx_th_column">k</th>
<th id="Sx7.T3.5.1.1.3" class="ltx_td ltx_align_right ltx_th ltx_th_column">ln(L)</th>
<th id="Sx7.T3.5.1.1.4" class="ltx_td ltx_align_right ltx_th ltx_th_column">BIC</th>
<th id="Sx7.T3.5.1.1.5" class="ltx_td ltx_align_right ltx_th ltx_th_column">AIC</th>
<th id="Sx7.T3.5.1.1.6" class="ltx_td ltx_align_left ltx_th ltx_th_column">mixture components</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="Sx7.T3.5.2.1" class="ltx_tr">
<td id="Sx7.T3.5.2.1.1" class="ltx_td ltx_align_right ltx_border_t">1</td>
<td id="Sx7.T3.5.2.1.2" class="ltx_td ltx_align_right ltx_border_t">2</td>
<td id="Sx7.T3.5.2.1.3" class="ltx_td ltx_align_right ltx_border_t">-273.93</td>
<td id="Sx7.T3.5.2.1.4" class="ltx_td ltx_align_right ltx_border_t">559.74</td>
<td id="Sx7.T3.5.2.1.5" class="ltx_td ltx_align_right ltx_border_t">551.87</td>
<td id="Sx7.T3.5.2.1.6" class="ltx_td ltx_align_left ltx_border_t">1.32 ± 0.50</td>
</tr>
<tr id="Sx7.T3.5.3.2" class="ltx_tr">
<td id="Sx7.T3.5.3.2.1" class="ltx_td ltx_align_right">2</td>
<td id="Sx7.T3.5.3.2.2" class="ltx_td ltx_align_right">5</td>
<td id="Sx7.T3.5.3.2.3" class="ltx_td ltx_align_right">-259.32</td>
<td id="Sx7.T3.5.3.2.4" class="ltx_td ltx_align_right"><span id="Sx7.T3.5.3.2.4.1" class="ltx_text ltx_font_bold">548.31</span></td>
<td id="Sx7.T3.5.3.2.5" class="ltx_td ltx_align_right">528.64</td>
<td id="Sx7.T3.5.3.2.6" class="ltx_td ltx_align_left">0.70 ± 0.14 ; 1.45 ± 0.45</td>
</tr>
<tr id="Sx7.T3.5.4.3" class="ltx_tr">
<td id="Sx7.T3.5.4.3.1" class="ltx_td ltx_align_right">3</td>
<td id="Sx7.T3.5.4.3.2" class="ltx_td ltx_align_right">8</td>
<td id="Sx7.T3.5.4.3.3" class="ltx_td ltx_align_right">-253.36</td>
<td id="Sx7.T3.5.4.3.4" class="ltx_td ltx_align_right">554.19</td>
<td id="Sx7.T3.5.4.3.5" class="ltx_td ltx_align_right">522.71</td>
<td id="Sx7.T3.5.4.3.6" class="ltx_td ltx_align_left">0.71 ± 0.17 ; 1.34 ± 0.29 ; 2.09 ± 0.19</td>
</tr>
<tr id="Sx7.T3.5.5.4" class="ltx_tr">
<td id="Sx7.T3.5.5.4.1" class="ltx_td ltx_align_right">4</td>
<td id="Sx7.T3.5.5.4.2" class="ltx_td ltx_align_right">11</td>
<td id="Sx7.T3.5.5.4.3" class="ltx_td ltx_align_right">-251.41</td>
<td id="Sx7.T3.5.5.4.4" class="ltx_td ltx_align_right">568.11</td>
<td id="Sx7.T3.5.5.4.5" class="ltx_td ltx_align_right">524.82</td>
<td id="Sx7.T3.5.5.4.6" class="ltx_td ltx_align_left">0.74 ± 0.18 ; 1.34 ± 0.20 ; 1.70 ± 0.04 ; 2.02 ± 0.22</td>
</tr>
</tbody>
</table>
</figure>
<figure id="Sx7.F9" class="ltx_figure"><img src="/html/1309.7402/assets/x8.png" id="Sx7.F9.g1" class="ltx_graphics ltx_centering ltx_img_square" width="384" height="384" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 9: </span>Distribution of synonymous and non-synonymous sequence divergence rates for pairs <em id="Sx7.F9.4.1" class="ltx_emph ltx_font_italic">L. polyphemus</em> paralogs and <em id="Sx7.F9.5.2" class="ltx_emph ltx_font_italic">L. polyphemus</em> - <em id="Sx7.F9.6.3" class="ltx_emph ltx_font_italic">T. tridentatus</em> orthologs.</figcaption>
</figure>
<figure id="Sx7.F10" class="ltx_figure"><img src="/html/1309.7402/assets/x9.png" id="Sx7.F10.g1" class="ltx_graphics ltx_centering ltx_img_square" width="230" height="230" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 10: </span>Two component mixture model fit to the <math id="Sx7.F10.3.m1.1" class="ltx_Math" alttext="K_{s}" display="inline"><semantics id="Sx7.F10.3.m1.1b"><msub id="Sx7.F10.3.m1.1.1" xref="Sx7.F10.3.m1.1.1.cmml"><mi id="Sx7.F10.3.m1.1.1.2" xref="Sx7.F10.3.m1.1.1.2.cmml">K</mi><mi id="Sx7.F10.3.m1.1.1.3" xref="Sx7.F10.3.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="Sx7.F10.3.m1.1c"><apply id="Sx7.F10.3.m1.1.1.cmml" xref="Sx7.F10.3.m1.1.1"><csymbol cd="ambiguous" id="Sx7.F10.3.m1.1.1.1.cmml" xref="Sx7.F10.3.m1.1.1">subscript</csymbol><ci id="Sx7.F10.3.m1.1.1.2.cmml" xref="Sx7.F10.3.m1.1.1.2">𝐾</ci><ci id="Sx7.F10.3.m1.1.1.3.cmml" xref="Sx7.F10.3.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.F10.3.m1.1d">K_{s}</annotation></semantics></math> peak on the range <math id="Sx7.F10.4.m2.1" class="ltx_Math" alttext="0&lt;=K_{s}&lt;=2.5" display="inline"><semantics id="Sx7.F10.4.m2.1b"><mrow id="Sx7.F10.4.m2.1.1" xref="Sx7.F10.4.m2.1.1.cmml"><mn id="Sx7.F10.4.m2.1.1.2" xref="Sx7.F10.4.m2.1.1.2.cmml">0</mn><mo id="Sx7.F10.4.m2.1.1.3" xref="Sx7.F10.4.m2.1.1.3.cmml">&lt;=</mo><msub id="Sx7.F10.4.m2.1.1.4" xref="Sx7.F10.4.m2.1.1.4.cmml"><mi id="Sx7.F10.4.m2.1.1.4.2" xref="Sx7.F10.4.m2.1.1.4.2.cmml">K</mi><mi id="Sx7.F10.4.m2.1.1.4.3" xref="Sx7.F10.4.m2.1.1.4.3.cmml">s</mi></msub><mo id="Sx7.F10.4.m2.1.1.5" xref="Sx7.F10.4.m2.1.1.5.cmml">&lt;=</mo><mn id="Sx7.F10.4.m2.1.1.6" xref="Sx7.F10.4.m2.1.1.6.cmml">2.5</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx7.F10.4.m2.1c"><apply id="Sx7.F10.4.m2.1.1.cmml" xref="Sx7.F10.4.m2.1.1"><and id="Sx7.F10.4.m2.1.1a.cmml" xref="Sx7.F10.4.m2.1.1"></and><apply id="Sx7.F10.4.m2.1.1b.cmml" xref="Sx7.F10.4.m2.1.1"><leq id="Sx7.F10.4.m2.1.1.3.cmml" xref="Sx7.F10.4.m2.1.1.3"></leq><cn type="integer" id="Sx7.F10.4.m2.1.1.2.cmml" xref="Sx7.F10.4.m2.1.1.2">0</cn><apply id="Sx7.F10.4.m2.1.1.4.cmml" xref="Sx7.F10.4.m2.1.1.4"><csymbol cd="ambiguous" id="Sx7.F10.4.m2.1.1.4.1.cmml" xref="Sx7.F10.4.m2.1.1.4">subscript</csymbol><ci id="Sx7.F10.4.m2.1.1.4.2.cmml" xref="Sx7.F10.4.m2.1.1.4.2">𝐾</ci><ci id="Sx7.F10.4.m2.1.1.4.3.cmml" xref="Sx7.F10.4.m2.1.1.4.3">𝑠</ci></apply></apply><apply id="Sx7.F10.4.m2.1.1c.cmml" xref="Sx7.F10.4.m2.1.1"><leq id="Sx7.F10.4.m2.1.1.5.cmml" xref="Sx7.F10.4.m2.1.1.5"></leq><share href="#Sx7.F10.4.m2.1.1.4.cmml" id="Sx7.F10.4.m2.1.1d.cmml" xref="Sx7.F10.4.m2.1.1"></share><cn type="float" id="Sx7.F10.4.m2.1.1.6.cmml" xref="Sx7.F10.4.m2.1.1.6">2.5</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx7.F10.4.m2.1d">0&lt;=K_{s}&lt;=2.5</annotation></semantics></math>.</figcaption>
</figure>
</section>
</section>
<section id="Sx8" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Data Access</h2>

<div id="Sx8.p1" class="ltx_para">
<p id="Sx8.p1.1" class="ltx_p">The raw sequencing reads are currently being submitted through the NCBI SRA and are
accessible via NCBI BioProject accession PRJNA187356.</p>
</div>
</section>
<section id="Sx9" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Acknowledgements</h2>

<div id="Sx9.p1" class="ltx_para">
<p id="Sx9.p1.1" class="ltx_p">This research was supported by the National Science Foundation
(EF-0850294 and IOB 06-41750), the Beckman Young Investigator Program,
the University of Florida Division of Sponsored Research, the
Department of Biology, and the UF Marine Laboratory at Seahorse Key.</p>
</div>
</section>
<section id="Sx10" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Author contributions</h2>

<div id="Sx10.p1" class="ltx_para">
<p id="Sx10.p1.1" class="ltx_p">NHP conceived and led the project. All authors wrote the paper.
PH, NHP and J-XY wrote software. NHP, PH, J-XY and CWN
carried out sequence analysis. JB collected and raised samples. CWN
extracted genomic DNA and created the libraries for sequencing.</p>
</div>
</section>
<section id="Sx11" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Discosure Declaration</h2>

<div id="Sx11.p1" class="ltx_para">
<p id="Sx11.p1.1" class="ltx_p">The authors declare that they have no conflicts of interest.</p>
</div>
</section>
<section id="Sx12" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Figures</h2>

</section>
<section id="Sx13" class="ltx_section">
<h2 class="ltx_title ltx_title_section"></h2>

</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">
Ohno S (1970) Evolution by gene duplication.

</span>
<span class="ltx_bibblock">Springer-Verlag.


</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">
Wolfe KH, Shields DC (1997) Molecular evidence for an ancient duplication of
the entire yeast genome.

</span>
<span class="ltx_bibblock">Nature 387: 708–713.


</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">
McLysaght A, Hokamp K, Wolfe KH (2002) Extensive genomic duplication during
early chordate evolution.

</span>
<span class="ltx_bibblock">Nature Genetics 31: 200–204.


</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">
Simillion C, Vandepoele K, Montagu MCEV, Zabeau M, Peer YVd (2002) The hidden
duplication past of arabidopsis thaliana.

</span>
<span class="ltx_bibblock">Proceedings of the National Academy of Sciences 99: 13627–13632.


</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">
Aury JM, Jaillon O, Duret L, Noel B, Jubin C, et al. (2006) Global trends of
whole-genome duplications revealed by the ciliate paramecium tetraurelia.

</span>
<span class="ltx_bibblock">Nature 444: 171–178.


</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">
Putnam NH, Srivastava M, Hellsten U, Dirks B, Chapman J, et al. (2007) Sea
anemone genome reveals ancestral eumetazoan gene repertoire and genomic
organization.

</span>
<span class="ltx_bibblock">Science 317: 86 –94.


</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">
Putnam NH, Butts T, Ferrier DEK, Furlong RF, Hellsten U, et al. (2008) The
amphioxus genome and the evolution of the chordate karyotype.

</span>
<span class="ltx_bibblock">Nature 453: 1064–1071.


</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">
Srivastava M, Begovic E, Chapman J, Putnam NH, Hellsten U, et al. (2008) The
trichoplax genome and the nature of placozoans.

</span>
<span class="ltx_bibblock">Nature 454: 955–960.


</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">
Srivastava M, Simakov O, Chapman J, Fahey B, Gauthier MEA, et al. (2010) The
amphimedon queenslandica genome and the evolution of animal complexity.

</span>
<span class="ltx_bibblock">Nature 466: 720–726.


</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">
Simakov O, Marletaz F, Cho SJ, Edsinger-Gonzales E, Havlak P, et al. (2012)
Insights into bilaterian evolution from three spiralian genomes.

</span>
<span class="ltx_bibblock">Nature .


</span>
</li>
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock">
Lv J, Havlak P, Putnam N (2011) Constraints on genes shape long-term
conservation of macro-synteny in metazoan genomes.

</span>
<span class="ltx_bibblock">BMC Bioinformatics 12: S11.


</span>
</li>
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock">
Earl D, Bradnam K, John JS, Darling A, Lin D, et al. (2011) Assemblathon 1: A
competitive assessment of de novo short read assembly methods.

</span>
<span class="ltx_bibblock">Genome Research 21: 2224–2241.


</span>
</li>
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock">
Davey JW, Hohenlohe PA, Etter PD, Boone JQ, Catchen JM, et al. (2011)
Genome-wide genetic marker discovery and genotyping using next-generation
sequencing.

</span>
<span class="ltx_bibblock">Nature Reviews Genetics 12: 499–510.


</span>
</li>
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock">
Altshuler D, Pollara VJ, Cowles CR, Van Etten WJ, Baldwin J, et al. (2000) An
SNP map of the human genome generated by reduced representation shotgun
sequencing.

</span>
<span class="ltx_bibblock">Nature 407: 513–516.


</span>
</li>
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock">
Baird NA, Etter PD, Atwood TS, Currey MC, Shiver AL, et al. (2008) Rapid SNP
discovery and genetic mapping using sequenced RAD markers.

</span>
<span class="ltx_bibblock">PLoS ONE 3: e3376.


</span>
</li>
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock">
Huang X, Feng Q, Qian Q, Zhao Q, Wang L, et al. (2009) High-throughput
genotyping by whole-genome resequencing.

</span>
<span class="ltx_bibblock">Genome Research 19: 1068–1076.


</span>
</li>
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock">
Andolfatto P, Davison D, Erezyilmaz D, Hu TT, Mast J, et al. (2011) Multiplexed
shotgun genotyping for rapid and efficient genetic mapping.

</span>
<span class="ltx_bibblock">Genome Research 21: 610–617.


</span>
</li>
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock">
Chapman JA, Ho I, Sunkara S, Luo S, Schroth GP, et al. (2011) Meraculous: De
novo genome assembly with short paired-end reads.

</span>
<span class="ltx_bibblock">PLoS ONE 6: e23501.


</span>
</li>
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock">
Butler J, MacCallum I, Kleber M, Shlyakhter IA, Belmonte MK, et al. (2008)
ALLPATHS: de novo assembly of whole-genome shotgun microreads.

</span>
<span class="ltx_bibblock">Genome Research 18: 810–820.


</span>
</li>
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock">
Gnerre S, MacCallum I, Przybylski D, Ribeiro FJ, Burton JN, et al. (2011)
High-quality draft assemblies of mammalian genomes from massively parallel
sequence data.

</span>
<span class="ltx_bibblock">Proceedings of the National Academy of Sciences 108: 1513–1518.


</span>
</li>
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock">
Rudkin DM, Young GA (2009) Horseshoe crabs – an ancient ancestry revealed.

</span>
<span class="ltx_bibblock">In: Tanacredi JT, Botton ML, Smith D, editors, Biology and
Conservation of Horseshoe Crabs, Springer US. pp. 25–44.

</span>
<span class="ltx_bibblock">URL <span id="bib.bib21.1.1" class="ltx_text ltx_font_typewriter">http://www.springerlink.com/content/t231746300u44q55/abstract/</span>.


</span>
</li>
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[22]</span>
<span class="ltx_bibblock">
Fisher DC (1984) The xiphosurida: archetypes of bradytely?

</span>
<span class="ltx_bibblock">In: Eldrege N, Stanley SM, editors, Living Fossils, Springer Verlag.
pp. 196–213.


</span>
</li>
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[23]</span>
<span class="ltx_bibblock">
Berkson J, Shuster CN Jr (1999) The horseshoe crab: The battle for a true
multiple-use resource.

</span>
<span class="ltx_bibblock">Fisheries 24: 6–10.


</span>
</li>
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[24]</span>
<span class="ltx_bibblock">
Shuster CN Jr, Barlow RB, Brockmann HJ, editors (2003) The American Horseshoe
Crab.

</span>
<span class="ltx_bibblock">Cambridge, MA: Harvard University Press.


</span>
</li>
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[25]</span>
<span class="ltx_bibblock">
Gregory TR (2012).

</span>
<span class="ltx_bibblock">Animal genome size database.

</span>
<span class="ltx_bibblock">URL <span id="bib.bib25.1.1" class="ltx_text ltx_font_typewriter">Animal Genome Size Database</span>.


</span>
</li>
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[26]</span>
<span class="ltx_bibblock">
Pop M, Kosack DS, Salzberg SL (2004) Hierarchical scaffolding with bambus.

</span>
<span class="ltx_bibblock">Genome Research 14: 149–159.


</span>
</li>
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[27]</span>
<span class="ltx_bibblock">
van Os H, Andrzejewski S, Bakker E, Barrena I, Bryan GJ, et al. (2006)
Construction of a 10,000-marker ultradense genetic recombination map of
potato: Providing a framework for accelerated gene isolation and a genomewide
physical map.

</span>
<span class="ltx_bibblock">Genetics 173: 1075 –1087.


</span>
</li>
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[28]</span>
<span class="ltx_bibblock">
Iwasaki Y, Iwami T, Sekiguchi K (1988) Karyology.

</span>
<span class="ltx_bibblock">In: Sekiguchi K, editor, Biology of Horseshoe Crabs, Tokyo: Science
House Co. pp. 309–314.


</span>
</li>
<li id="bib.bib29" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[29]</span>
<span class="ltx_bibblock">
Cartwright RA, Hussin J, Keebler JEM, Stone EA, Awadalla P (2012) A
family-based probabilistic method for capturing de novo mutations from
high-throughput short-read sequencing data.

</span>
<span class="ltx_bibblock">Statistical applications in genetics and molecular biology 11.


</span>
</li>
<li id="bib.bib30" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[30]</span>
<span class="ltx_bibblock">
Bird AP (1980) DNA methylation and the frequency of CpG in animal DNA.

</span>
<span class="ltx_bibblock">Nucleic Acids Research 8: 1499–1504.


</span>
</li>
<li id="bib.bib31" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[31]</span>
<span class="ltx_bibblock">
Lynch M (2006) The origins of eukaryotic gene structure.

</span>
<span class="ltx_bibblock">Molecular Biology and Evolution 23: 450–468.


</span>
</li>
<li id="bib.bib32" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[32]</span>
<span class="ltx_bibblock">
VectorBase (2008).

</span>
<span class="ltx_bibblock">Ixodes scapularis annotation, IscaW1.

</span>
<span class="ltx_bibblock">URL <span id="bib.bib32.1.1" class="ltx_text ltx_font_typewriter">http://www.vectorbase.org</span>.


</span>
</li>
<li id="bib.bib33" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[33]</span>
<span class="ltx_bibblock">
Vanneste K, Van de Peer Y, Maere S (2013) Inference of genome duplications from
age distributions revisited.

</span>
<span class="ltx_bibblock">Molecular biology and evolution 30: 177–190.


</span>
</li>
<li id="bib.bib34" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[34]</span>
<span class="ltx_bibblock">
Obst M, Faurby S, Bussarawit S, Funch P (2012) Molecular phylogeny of extant
horseshoe crabs (xiphosura, limulidae) indicates paleogene diversification of
asian species.

</span>
<span class="ltx_bibblock">Molecular Phylogenetics and Evolution 62: 21–26.


</span>
</li>
<li id="bib.bib35" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[35]</span>
<span class="ltx_bibblock">
Begun DJ, Aquadro CF (1992) Levels of naturally occurring DNA polymorphism
correlate with recombination rates in d. melanogaster.

</span>
<span class="ltx_bibblock">, Published online: 09 April 1992; | doi:101038/356519a0
356: 519–520.


</span>
</li>
<li id="bib.bib36" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[36]</span>
<span class="ltx_bibblock">
Cutter AD, Payseur BA (2003) Selection at linked sites in the partial selfer
caenorhabditis elegans.

</span>
<span class="ltx_bibblock">Molecular Biology and Evolution 20: 665–673.


</span>
</li>
<li id="bib.bib37" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[37]</span>
<span class="ltx_bibblock">
Nachman MW (2001) Single nucleotide polymorphisms and recombination rate in
humans.

</span>
<span class="ltx_bibblock">Trends in Genetics 17: 481–485.


</span>
</li>
<li id="bib.bib38" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[38]</span>
<span class="ltx_bibblock">
Stephan W, Langley CH (1998) DNA polymorphism in lycopersicon and
crossing-over per physical length.

</span>
<span class="ltx_bibblock">Genetics 150: 1585–1593.


</span>
</li>
<li id="bib.bib39" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[39]</span>
<span class="ltx_bibblock">
Roselius K, Stephan W, Städler T (2005) The relationship of nucleotide
polymorphism, recombination rate and selection in wild tomato species.

</span>
<span class="ltx_bibblock">Genetics 171: 753–763.


</span>
</li>
<li id="bib.bib40" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[40]</span>
<span class="ltx_bibblock">
Hellmann I, Ebersberger I, Ptak SE, Pääbo S, Przeworski M (2003) A neutral
explanation for the correlation of diversity with recombination rates in
humans.

</span>
<span class="ltx_bibblock">The American Journal of Human Genetics 72: 1527–1535.


</span>
</li>
<li id="bib.bib41" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[41]</span>
<span class="ltx_bibblock">
Andolfatto P, Przeworski M (2001) Regions of lower crossing over harbor more
rare variants in african populations of drosophila melanogaster.

</span>
<span class="ltx_bibblock">Genetics 158: 657–665.


</span>
</li>
<li id="bib.bib42" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[42]</span>
<span class="ltx_bibblock">
Havlak P, Chen R, Durbin KJ, Egan A, Ren Y, et al. (2004) The atlas genome
assembly system.

</span>
<span class="ltx_bibblock">Genome Research 14: 721–732.


</span>
</li>
<li id="bib.bib43" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[43]</span>
<span class="ltx_bibblock">
Treangen TJ, Sommer DD, Angly FE, Koren S, Pop M (2011) Next generation
sequence assembly with AMOS.

</span>
<span class="ltx_bibblock">Current protocols in bioinformatics / editoral board, Andreas D
Baxevanis [et al] CHAPTER: Unit11.8.


</span>
</li>
<li id="bib.bib44" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[44]</span>
<span class="ltx_bibblock">
Johnson SL, Brockmann HJ (2010) Costs of multiple mates: an experimental study
in horseshoe crabs.

</span>
<span class="ltx_bibblock">Animal Behaviour 80: 773–782.


</span>
</li>
<li id="bib.bib45" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[45]</span>
<span class="ltx_bibblock">
Mullikin JC, Ning Z (2003) The phusion assembler.

</span>
<span class="ltx_bibblock">Genome Research 13: 81–90.


</span>
</li>
<li id="bib.bib46" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[46]</span>
<span class="ltx_bibblock">
Cock PJA, Fields CJ, Goto N, Heuer ML, Rice PM (2010) The sanger FASTQ file
format for sequences with quality scores, and the Solexa/Illumina FASTQ
variants.

</span>
<span class="ltx_bibblock">Nucleic Acids Research 38: 1767–1771.


</span>
</li>
<li id="bib.bib47" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[47]</span>
<span class="ltx_bibblock">
Ewing B, Green P (1998) Base-calling of automated sequencer traces
UsingPhred. II. error probabilities.

</span>
<span class="ltx_bibblock">Genome Research 8: 186–194.


</span>
</li>
<li id="bib.bib48" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[48]</span>
<span class="ltx_bibblock">
Knuth DE (1973) Searching and Sorting, volume 3 of <em id="bib.bib48.1.1" class="ltx_emph ltx_font_italic">The Art of Computer
Programming</em>.

</span>
<span class="ltx_bibblock">Reading, MA: Addison-Wesley.


</span>
</li>
<li id="bib.bib49" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[49]</span>
<span class="ltx_bibblock">
Roberts M, Hayes W, Hunt BR, Mount SM, Yorke JA (2004) Reducing storage
requirements for biological sequence comparison.

</span>
<span class="ltx_bibblock">Bioinformatics 20: 3363–3369.


</span>
</li>
<li id="bib.bib50" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[50]</span>
<span class="ltx_bibblock">
Ye C, Ma Z, Cannon C, Pop M, Yu D (2012) Exploiting sparseness in de novo
genome assembly.

</span>
<span class="ltx_bibblock">BMC Bioinformatics 13: S1.


</span>
</li>
<li id="bib.bib51" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[51]</span>
<span class="ltx_bibblock">
Chen GK, Marjoram P, Wall JD (2009) Fast and flexible simulation of DNA
sequence data.

</span>
<span class="ltx_bibblock">Genome Research 19: 136–142.


</span>
</li>
<li id="bib.bib52" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[52]</span>
<span class="ltx_bibblock">
Dehal P, Satou Y, Campbell RK, Chapman J, Degnan B, et al. (2002) The draft
genome of ciona intestinalis: Insights into chordate and vertebrate origins.

</span>
<span class="ltx_bibblock">Science 298: 2157–2167.


</span>
</li>
<li id="bib.bib53" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[53]</span>
<span class="ltx_bibblock">
Haubold B, Pfaffelhuber P, Lynch M (2010) mlRho – a program for estimating
the population mutation and recombination rates from shotgun-sequenced
diploid genomes.

</span>
<span class="ltx_bibblock">Molecular Ecology 19: 277–284.


</span>
</li>
<li id="bib.bib54" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[54]</span>
<span class="ltx_bibblock">
Small KS, Brudno M, Hill MM, Sidow A (2007) Extreme genomic variation in a
natural population.

</span>
<span class="ltx_bibblock">Proceedings of the National Academy of Sciences 104: 5698–5703.


</span>
</li>
<li id="bib.bib55" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[55]</span>
<span class="ltx_bibblock">
Homer N (2012).

</span>
<span class="ltx_bibblock">dwgsim.

</span>
<span class="ltx_bibblock">URL <span id="bib.bib55.1.1" class="ltx_text ltx_font_typewriter">http://sourceforge.net/apps/mediawiki/dnaa</span>.


</span>
</li>
<li id="bib.bib56" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[56]</span>
<span class="ltx_bibblock">
Hopcroft J, Tarjan R (1973) Algorithm 447: efficient algorithms for graph
manipulation.

</span>
<span class="ltx_bibblock">Commun ACM 16: 372–378.


</span>
</li>
<li id="bib.bib57" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[57]</span>
<span class="ltx_bibblock">
Lunter G, Goodson M (2011) Stampy: a statistical algorithm for sensitive and
fast mapping of illumina sequence reads.

</span>
<span class="ltx_bibblock">Genome research 21: 936–939.


</span>
</li>
<li id="bib.bib58" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[58]</span>
<span class="ltx_bibblock">
Li H (2011) A statistical framework for SNP calling, mutation discovery,
association mapping and population genetical parameter estimation from
sequencing data.

</span>
<span class="ltx_bibblock">Bioinformatics 27: 2987 –2993.


</span>
</li>
<li id="bib.bib59" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[59]</span>
<span class="ltx_bibblock">
Yang Z (1996) Statistical properties of a DNA sample under the finite-sites
model.

</span>
<span class="ltx_bibblock">Genetics 144: 1941–1950.


</span>
</li>
<li id="bib.bib60" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[60]</span>
<span class="ltx_bibblock">
Haldane JBS (1919) The combination of linkage values and the calculation of
distances between the loci of linked factors.

</span>
<span class="ltx_bibblock">Journal of Genetics 8: 299–309.


</span>
</li>
<li id="bib.bib61" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[61]</span>
<span class="ltx_bibblock">
Gordon D, Abajian C, Green P (1998) Consed: a graphical tool for sequence
finishing.

</span>
<span class="ltx_bibblock">Genome research 8: 195–202.


</span>
</li>
<li id="bib.bib62" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[62]</span>
<span class="ltx_bibblock">
Slater GS, Birney E (2005) Automated generation of heuristics for biological
sequence comparison.

</span>
<span class="ltx_bibblock">BMC Bioinformatics 6: 31.


</span>
</li>
<li id="bib.bib63" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[63]</span>
<span class="ltx_bibblock">
Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and
high throughput.

</span>
<span class="ltx_bibblock">Nucleic Acids Research 32: 1792–1797.


</span>
</li>
<li id="bib.bib64" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[64]</span>
<span class="ltx_bibblock">
Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference
under mixed models.

</span>
<span class="ltx_bibblock">Bioinformatics 19: 1572–1574.


</span>
</li>
<li id="bib.bib65" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[65]</span>
<span class="ltx_bibblock">
Yang Z, Nielsen R (2000) Estimating synonymous and nonsynonymous substitution
rates under realistic evolutionary models.

</span>
<span class="ltx_bibblock">Molecular Biology and Evolution 17: 32–43.


</span>
</li>
<li id="bib.bib66" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[66]</span>
<span class="ltx_bibblock">
Zhang Z, Li J, Zhao XQ, Wang J, Wong GKS, et al. (2006) KaKs calculator:
Calculating ka and ks through model selection and model averaging.

</span>
<span class="ltx_bibblock">Genomics, Proteomics &amp; Bioinformatics 4: 259–263.


</span>
</li>
<li id="bib.bib67" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[67]</span>
<span class="ltx_bibblock">
Benaglia T, Chauveau D, Hunter R David, Young S Derek (2009) mixtools: An r
package for analyzing finite mixture models.

</span>
<span class="ltx_bibblock">Journal of Statistical Software 32: 1–29.


</span>
</li>
</ul>
</section>
<section id="Sx13.SSx1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection"></h3>

</section>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/1309.7401" class="ar5iv-nav-button ar5iv-nav-button-prev">◄</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/1309.7402" class="ar5iv-text-button ar5iv-severity-warning">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+1309.7402">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/1309.7402" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/1309.7403" class="ar5iv-nav-button ar5iv-nav-button-next">►</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Fri Mar 15 12:53:41 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "×";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
