<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[2102.03314] On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022).</title><meta property="og:description" content="The availability of genomic data is essential to progress in biomedical research, personalized medicine, etc. However, its extreme sensitivity makes it problematic, if not outright impossible, to publish or share it.
A…">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022).">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022).">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/2102.03314">

<!--Generated on Thu Mar  7 02:27:27 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<h1 class="ltx_title ltx_font_bold ltx_title_document">On Utility and Privacy in Synthetic Genomic Data<span id="id1.id1" class="ltx_note ltx_role_thanks"><sup class="ltx_note_mark">†</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">†</sup><span class="ltx_note_type">thanks: </span>Published in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022).</span></span></span>
</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Bristena Oprisanu
<br class="ltx_break">UCL
<br class="ltx_break">bristena.oprisanu.10@ucl.ac.uk
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Georgi Ganev
<br class="ltx_break">UCL and Hazy
<br class="ltx_break">georgi.ganev.16@ucl.ac.uk
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Emiliano De Cristofaro
<br class="ltx_break">UCL and Alan Turing Institute
<br class="ltx_break">e.decristofaro@ucl.ac.uk
</span></span>
</div>

<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p id="id2.id1" class="ltx_p">The availability of genomic data is essential to progress in biomedical research, personalized medicine, etc. However, its extreme sensitivity makes it problematic, if not outright impossible, to publish or share it.
As a result, several initiatives have been launched to experiment with <span id="id2.id1.1" class="ltx_text ltx_font_italic">synthetic</span> genomic data, e.g., using generative models to learn the underlying distribution of the real data and generate artificial datasets that preserve its salient characteristics without exposing it.</p>
<p id="id3.id2" class="ltx_p">This paper provides the first evaluation of both utility and privacy protection of six state-of-the-art models for generating synthetic genomic data.
We assess the performance of the synthetic data on several common tasks, such as allele population statistics and linkage disequilibrium.
We then measure privacy through the lens of <span id="id3.id2.1" class="ltx_text ltx_font_italic">membership inference attacks</span>, i.e., inferring whether a record was part of the training data.
Our experiments show that no single approach to generate synthetic genomic data yields both high utility and strong privacy across the board.
Also, the size and nature of the training dataset matter.
Moreover, while some combinations of datasets and models produce synthetic data with distributions close to the real data, there often are target data points that are vulnerable to membership inference.
Looking forward, our techniques can be used by practitioners to assess the risks of deploying synthetic genomic data in the wild and serve as a benchmark for future work.</p>
</div>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>

<div id="S1.p1" class="ltx_para">
<p id="S1.p1.1" class="ltx_p">Progress in genome sequencing has paved the way towards preventing, diagnosing, and treating several diseases and conditions.
Much of this progress is dependent on the availability of genomic data.
Consequently, numerous initiatives have been established to support and encourage genomic data sharing.
For example, the International HapMap Project <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib42" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">42</span></a>]</cite> helped identify common genetic variations and study their involvement in human health and disease, while the 1000 Genomes Project <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib1" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">1</span></a>]</cite> created a catalog of human variation and genotype data.
Funding agencies, e.g., the National Institutes of Health (NIH), increasingly often make data sharing a requirement to fund grant applications <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib46" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">46</span></a>]</cite>.</p>
</div>
<div id="S1.p2" class="ltx_para">
<p id="S1.p2.1" class="ltx_p">Data sharing in genomics is crucial to enable progress in Precision Medicine <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib21" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">21</span></a>]</cite>.
However, this inherently conflicts with the need to protect individuals’ privacy.
Genomic data contains sensitive information related to heritage, predisposition to diseases, phenotype traits, etc., making it hard to anonymize <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib23" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">23</span></a>]</cite>.
Hiding “sensitive” portions of the genome is not effective, as they can still be inferred via high-order correlation models <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">54</span></a>]</cite>.<span id="footnote1" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_tag ltx_tag_note">1</span>For a thorough review of privacy threats in genomics, please see <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib4" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">4</span></a>, <a href="#bib.bib67" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">67</span></a>, <a href="#bib.bib44" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">44</span></a>]</cite>.</span></span></span></p>
</div>
<div id="S1.p3" class="ltx_para">
<p id="S1.p3.1" class="ltx_p">As a result, genomics researchers have begun to investigate the possibility of releasing <span id="S1.p3.1.1" class="ltx_text ltx_font_italic">synthetic</span> datasets rather than real/anonymized data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib52" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">52</span></a>]</cite>.
This follows a general trend in healthcare; for instance, the National Health Service (NHS) in England has recently concluded a project focused on releasing synthetic Emergency Room (“A&amp;E”) records <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib45" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">45</span></a>]</cite>.
Basically, the idea is to use generative models to learn to generate samples with the same characteristics—more precisely, with very close distributions—as the real data.
In other words, rather than releasing data of actual individuals, entities share artificially generated data so that the statistical properties of the original data are preserved, but minimizing the risk of malicious inference of sensitive information <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib15" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">15</span></a>]</cite>.</p>
</div>
<div id="S1.p4" class="ltx_para ltx_noindent">
<p id="S1.p4.1" class="ltx_p"><span id="S1.p4.1.1" class="ltx_text ltx_font_bold">Generative Models and Genomics.</span> In particular, previous work has experimented with both statistical and generative models.
Samani et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">54</span></a>]</cite> propose an inference model based on the recombination rate, which can also be used to generate new synthetic genomic samples.
Yelmen et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">71</span></a>]</cite> use Generative Adversarial Models (GANs) and Restricted Boltzmann Machines (RMBs) to mimic the distribution of real genomes and capture population structures. Finally, Killoran et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib36" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">36</span></a>]</cite> use ad-hoc training techniques for GANs and architectures for computer vision tasks.</p>
</div>
<div id="S1.p5" class="ltx_para ltx_noindent">
<p id="S1.p5.1" class="ltx_p"><span id="S1.p5.1.1" class="ltx_text ltx_font_bold">Technical Roadmap.</span> Prior work on synthetic genomic data (Section <a href="#S3" title="3 Synthetic Data Approaches in Genomics ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>) has not evaluated the privacy guarantees they provide and only evaluated utility from the shallow point of view of the statistical fidelity provided by the generative model.
To address this gap, we introduce a novel evaluation methodology and perform a series of experiments geared to quantify both the utility and the privacy of six state-of-the-art models used to generate human genomic synthetic data.</p>
</div>
<div id="S1.p6" class="ltx_para">
<p id="S1.p6.1" class="ltx_p">Our analysis unfolds along two main axes:</p>
<ol id="S1.I1" class="ltx_enumerate">
<li id="S1.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S1.I1.i1.p1" class="ltx_para">
<p id="S1.I1.i1.p1.1" class="ltx_p"><span id="S1.I1.i1.p1.1.1" class="ltx_text ltx_font_italic">Utility.</span> We focus on several, very common computational tasks on genomic data, measuring how well generative models preserve summary statistics (e.g., allele frequencies, population statistics) or linkage disequilibrium (see Section <a href="#S4" title="4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>).
We also compute the Kologorov-Smirnov(KS) test on the main metrics discussed as a non-parametric test.</p>
</div>
</li>
<li id="S1.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S1.I1.i2.p1" class="ltx_para">
<p id="S1.I1.i2.p1.1" class="ltx_p"><span id="S1.I1.i2.p1.1.1" class="ltx_text ltx_font_italic">Privacy.</span> We mount <span id="S1.I1.i2.p1.1.2" class="ltx_text ltx_font_italic">membership inference attacks</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib29" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">29</span></a>]</cite>, having an attacker infer whether a target record was part of the real data used to train the model producing the synthetic dataset, as opposed to releasing the real dataset (see Section <a href="#S5" title="5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>).
In the process, we also introduce a novel attack where the adversary only has <span id="S1.I1.i2.p1.1.3" class="ltx_text ltx_font_italic">partial</span> information for a target individual (Section <a href="#S5.SS2" title="5.2 Privacy Gain under Membership Inference Attack with Partial Information ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5.2</span></a>). We choose to study MIA as it is one of the most studied attacks mounted for both genomics data and machine learning, however, it has never been studied in the context of synthetic genomic data.</p>
</div>
</li>
</ol>
</div>
<div id="S1.p7" class="ltx_para ltx_noindent">
<p id="S1.p7.1" class="ltx_p"><span id="S1.p7.1.1" class="ltx_text ltx_font_bold">Main Findings.</span> Overall, our evaluation shows that there is no single approach for generating genomic synthetic data that performs well across the board, both in terms of utility and privacy.
Among other things, we find that:</p>
<ul id="S1.I2" class="ltx_itemize">
<li id="S1.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i1.p1" class="ltx_para">
<p id="S1.I2.i1.p1.1" class="ltx_p">A high-order correlation model specifically build for genomic data (Recomb) has the best utility metrics for small datasets but does so at the cost of privacy, even against weaker adversaries who only have partial information available.</p>
</div>
</li>
<li id="S1.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i2.p1" class="ltx_para">
<p id="S1.I2.i2.p1.1" class="ltx_p">The RBM model has a better performance with increasing dataset sizes, both in terms of utility and privacy, as targets get stronger privacy protection when synthetic data is generated using a larger training set.</p>
</div>
</li>
<li id="S1.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S1.I2.i3.p1" class="ltx_para">
<p id="S1.I2.i3.p1.1" class="ltx_p">Releasing synthetic datasets does not provide robust protection against membership inference attacks. We find cases where releasing the synthetic dataset sometimes offers better protection against membership inference attacks.
However, because of the randomness introduced by the generative models, one cannot meaningfully predict a target’s susceptibility to privacy attacks without fixing the training set and quantifying the respective privacy loss/gain for all targets in the set.</p>
</div>
</li>
</ul>
</div>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Preliminaries</h2>

<div id="S2.p1" class="ltx_para">
<p id="S2.p1.1" class="ltx_p">In this section, we introduce background information about genomics, and tools used to generate synthetic data, then, we present the privacy metrics and datasets used in our evaluation.</p>
</div>
<section id="S2.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.1 </span>Genomic Primer</h3>

<div id="S2.SS1.p1" class="ltx_para ltx_noindent">
<p id="S2.SS1.p1.1" class="ltx_p"><span id="S2.SS1.p1.1.1" class="ltx_text ltx_font_bold">Genomes and Genes.</span> The genome represents the entirety of an organism’s hereditary information.
It is encoded in DNA: each DNA strand comprises four chemical units, called nucleotides, represented by the letters A, C, G, and T.
The human genome consists of approximately 3 billion nucleotides packaged into thread-like structures called chromosomes.
The genome includes both the genes and the non-coding sequences of the DNA.
The former determine specific traits, characteristics, or control activity within an organism.
We refer to the group of genes inherited from a single parent as a <span id="S2.SS1.p1.1.2" class="ltx_text ltx_font_bold ltx_font_italic">haplotype</span>.
An <span id="S2.SS1.p1.1.3" class="ltx_text ltx_font_bold ltx_font_italic">allele</span> is a different variation of a gene; any individual inherits two alleles for each gene, one from each of their parents.
The genotype consists of the alleles that an organism has for a particular characteristic.</p>
</div>
<div id="S2.SS1.p2" class="ltx_para ltx_noindent">
<p id="S2.SS1.p2.1" class="ltx_p"><span id="S2.SS1.p2.1.1" class="ltx_text ltx_font_bold">SNPs and SNVs.</span> About 99.5% of the genome is shared among all humans; the rest differs due to genetic variations.
Single nucleotide polymorphisms (SNPs) are the most common type of genetic variation.
They occur at a single position in the genome and at least 1% of the population.
SNPs are usually biallelic and can be encoded by {0, 1, 2}, with 0 denoting a combination of two major (i.e., common) alleles, 2 a combination of two minor alleles, and 1 a combination of a major and a minor allele (which is also referred to as a heterozygous SNP).
Single nucleotide variants (SNVs) are single nucleotide positions in the genomic DNA at which different sequence alternative exists <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib17" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">17</span></a>]</cite>.</p>
</div>
<div id="S2.SS1.p3" class="ltx_para ltx_noindent">
<p id="S2.SS1.p3.1" class="ltx_p"><span id="S2.SS1.p3.1.1" class="ltx_text ltx_font_bold">Recombination Rate (RR).</span> Recombination is the process of determining the frequency with which characteristics are inherited together.
The RR is the probability that a transmitted haplotype constitutes a new combination of alleles different from that of either parental haplotype <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib14" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">14</span></a>]</cite>.</p>
</div>
<div id="S2.SS1.p4" class="ltx_para ltx_noindent">
<p id="S2.SS1.p4.1" class="ltx_p"><span id="S2.SS1.p4.1.1" class="ltx_text ltx_font_bold">Genome-Wide Association Studies (GWAS).</span> GWAS are hypothesis-free methods for identifying associations between genetic regions and traits.
A typical GWAS looks for common variants in several individuals, both with and without a trait, using genome-wide SNP arrays <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib43" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">43</span></a>, <a href="#bib.bib16" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">16</span></a>]</cite>.</p>
</div>
</section>
<section id="S2.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.2 </span>Membership Inference Attacks</h3>

<div id="S2.SS2.p1" class="ltx_para ltx_noindent">
<p id="S2.SS2.p1.1" class="ltx_p"><span id="S2.SS2.p1.1.1" class="ltx_text ltx_font_bold">Genomics.</span> A well-understood privacy threat in genomics is determining whether the data of a target individual is part of an aggregate genomic dataset or mixture.
This is known as a <span id="S2.SS2.p1.1.2" class="ltx_text ltx_font_italic">membership inference attack</span> (MIA) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib29" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">29</span></a>, <a href="#bib.bib66" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">66</span></a>, <a href="#bib.bib74" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">74</span></a>]</cite>. The ability to infer the presence of an individual’s data in a dataset constitutes an inherent privacy leak whenever the dataset has some sensitive attributes.
For instance, if a mixture includes DNA from patients with a specific disease, learning that a person is part of that mixture exposes their health status.</p>
</div>
<div id="S2.SS2.p2" class="ltx_para">
<p id="S2.SS2.p2.1" class="ltx_p">In general, genomic data contains extremely sensitive information about individuals; hence, MIAs against genomic data prompt severe privacy threats, including denial of life/health insurance, revealing predisposition to diseases, ancestry, etc.</p>
</div>
<div id="S2.SS2.p3" class="ltx_para ltx_noindent">
<p id="S2.SS2.p3.1" class="ltx_p"><span id="S2.SS2.p3.1.1" class="ltx_text ltx_font_bold">Machine Learning.</span> MIAs have also been studied in the context of machine learning to infer whether or not a target data point was used to train a target model.
This has been done both for discriminative <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib55" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">55</span></a>, <a href="#bib.bib53" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">53</span></a>, <a href="#bib.bib51" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">51</span></a>, <a href="#bib.bib11" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">11</span></a>, <a href="#bib.bib30" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">30</span></a>, <a href="#bib.bib38" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">38</span></a>, <a href="#bib.bib41" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">41</span></a>]</cite>
and generative models <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib24" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">24</span></a>, <a href="#bib.bib26" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">26</span></a>, <a href="#bib.bib11" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">11</span></a>]</cite>.
Inferring training set membership might yield serious privacy violations.
For instance, if a model for drug dose prediction is trained using data from patients with a certain disease, or synthetic health images are produced by a generative model trained on patients’ images, learning that data of a particular individual was part of the training set leaks information about that person’s health.
Overall, MIAs are also used as signals that access to a target model is “leaky” and can be a gateway to additional attacks <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib24" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">24</span></a>]</cite>.
This is the main reason we choose MIA as our leading privacy metric while deferring the quantification of other attacks (e.g., attribute inference or reconstruction attacks) to future work.</p>
</div>
</section>
<section id="S2.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.3 </span>Privacy Gain (PG) from Synthetic Data Release</h3>

<div id="S2.SS3.p1" class="ltx_para">
<p id="S2.SS3.p1.1" class="ltx_p">Our main privacy evaluation metric is denoted as <span id="S2.SS3.p1.1.1" class="ltx_text ltx_font_bold ltx_font_italic">Privacy Gain (PG)</span>, first proposed in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">59</span></a>]</cite>.
The PG quantifies the privacy advantage obtained by a target <math id="S2.SS3.p1.1.m1.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S2.SS3.p1.1.m1.1a"><mi id="S2.SS3.p1.1.m1.1.1" xref="S2.SS3.p1.1.m1.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.1.m1.1b"><ci id="S2.SS3.p1.1.m1.1.1.cmml" xref="S2.SS3.p1.1.m1.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.1.m1.1c">t</annotation></semantics></math>, vis-à-vis an MIA adversary, when a synthetic dataset is published instead of the real data.</p>
</div>
<div id="S2.SS3.p2" class="ltx_para ltx_noindent">
<p id="S2.SS3.p2.1" class="ltx_p"><span id="S2.SS3.p2.1.1" class="ltx_text ltx_font_bold">MIA Training (Algorithm <a href="#alg1" title="Algorithm 1 ‣ 2.3 Privacy Gain (PG) from Synthetic Data Release ‣ 2 Preliminaries ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>).</span> The adversary is trained as follows.
First, we take a reference dataset (which may or may not overlap with the real data used to generate the synthetic dataset) and use it to generate a synthetic dataset (lines 1–6).
This dataset is labeled as 0, i.e., it does not include the target record (line 7).
The target record is added to the dataset (line 8), and synthetic data is generated from the new dataset, which includes the target record; this dataset is labeled as 1 (lines 10-13).
These models are referred to as <span id="S2.SS3.p2.1.2" class="ltx_text ltx_font_italic">generative shadow</span> models.
Finally, the adversary uses the synthetic datasets to train a classifier (line 14), which distinguishes whether or not the target was used to train a generative model.</p>
</div>
<div id="S2.SS3.p3" class="ltx_para ltx_noindent">
<p id="S2.SS3.p3.6" class="ltx_p"><span id="S2.SS3.p3.6.1" class="ltx_text ltx_font_bold">PG Estimation (Algorithm <a href="#alg2" title="Algorithm 2 ‣ 2.3 Privacy Gain (PG) from Synthetic Data Release ‣ 2 Preliminaries ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>).</span> We estimate the PG for a fixed target record and input dataset as follows. First, the algorithm takes the target training set and generates <math id="S2.SS3.p3.1.m1.1" class="ltx_Math" alttext="n_{s}" display="inline"><semantics id="S2.SS3.p3.1.m1.1a"><msub id="S2.SS3.p3.1.m1.1.1" xref="S2.SS3.p3.1.m1.1.1.cmml"><mi id="S2.SS3.p3.1.m1.1.1.2" xref="S2.SS3.p3.1.m1.1.1.2.cmml">n</mi><mi id="S2.SS3.p3.1.m1.1.1.3" xref="S2.SS3.p3.1.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S2.SS3.p3.1.m1.1b"><apply id="S2.SS3.p3.1.m1.1.1.cmml" xref="S2.SS3.p3.1.m1.1.1"><csymbol cd="ambiguous" id="S2.SS3.p3.1.m1.1.1.1.cmml" xref="S2.SS3.p3.1.m1.1.1">subscript</csymbol><ci id="S2.SS3.p3.1.m1.1.1.2.cmml" xref="S2.SS3.p3.1.m1.1.1.2">𝑛</ci><ci id="S2.SS3.p3.1.m1.1.1.3.cmml" xref="S2.SS3.p3.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p3.1.m1.1c">n_{s}</annotation></semantics></math> synthetic datasets without the target record (lines 1–4).
Then, the target record is added to the training set, and <math id="S2.SS3.p3.2.m2.1" class="ltx_Math" alttext="n_{s}" display="inline"><semantics id="S2.SS3.p3.2.m2.1a"><msub id="S2.SS3.p3.2.m2.1.1" xref="S2.SS3.p3.2.m2.1.1.cmml"><mi id="S2.SS3.p3.2.m2.1.1.2" xref="S2.SS3.p3.2.m2.1.1.2.cmml">n</mi><mi id="S2.SS3.p3.2.m2.1.1.3" xref="S2.SS3.p3.2.m2.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S2.SS3.p3.2.m2.1b"><apply id="S2.SS3.p3.2.m2.1.1.cmml" xref="S2.SS3.p3.2.m2.1.1"><csymbol cd="ambiguous" id="S2.SS3.p3.2.m2.1.1.1.cmml" xref="S2.SS3.p3.2.m2.1.1">subscript</csymbol><ci id="S2.SS3.p3.2.m2.1.1.2.cmml" xref="S2.SS3.p3.2.m2.1.1.2">𝑛</ci><ci id="S2.SS3.p3.2.m2.1.1.3.cmml" xref="S2.SS3.p3.2.m2.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p3.2.m2.1c">n_{s}</annotation></semantics></math> of synthetic datasets are generated from this dataset (lines 5–9).
Then, the adversary trains their attack model <math id="S2.SS3.p3.3.m3.1" class="ltx_Math" alttext="\mathtt{MIATrain}" display="inline"><semantics id="S2.SS3.p3.3.m3.1a"><mi id="S2.SS3.p3.3.m3.1.1" xref="S2.SS3.p3.3.m3.1.1.cmml">𝙼𝙸𝙰𝚃𝚛𝚊𝚒𝚗</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p3.3.m3.1b"><ci id="S2.SS3.p3.3.m3.1.1.cmml" xref="S2.SS3.p3.3.m3.1.1">𝙼𝙸𝙰𝚃𝚛𝚊𝚒𝚗</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p3.3.m3.1c">\mathtt{MIATrain}</annotation></semantics></math> (line 10).
Finally, the PG for a target <math id="S2.SS3.p3.4.m4.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S2.SS3.p3.4.m4.1a"><mi id="S2.SS3.p3.4.m4.1.1" xref="S2.SS3.p3.4.m4.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p3.4.m4.1b"><ci id="S2.SS3.p3.4.m4.1.1.cmml" xref="S2.SS3.p3.4.m4.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p3.4.m4.1c">t</annotation></semantics></math> is computed as <math id="S2.SS3.p3.5.m5.1" class="ltx_Math" alttext="PG_{t}=\frac{1-\overline{MIA_{t}}(S_{test})}{2}" display="inline"><semantics id="S2.SS3.p3.5.m5.1a"><mrow id="S2.SS3.p3.5.m5.1.2" xref="S2.SS3.p3.5.m5.1.2.cmml"><mrow id="S2.SS3.p3.5.m5.1.2.2" xref="S2.SS3.p3.5.m5.1.2.2.cmml"><mi id="S2.SS3.p3.5.m5.1.2.2.2" xref="S2.SS3.p3.5.m5.1.2.2.2.cmml">P</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.5.m5.1.2.2.1" xref="S2.SS3.p3.5.m5.1.2.2.1.cmml">​</mo><msub id="S2.SS3.p3.5.m5.1.2.2.3" xref="S2.SS3.p3.5.m5.1.2.2.3.cmml"><mi id="S2.SS3.p3.5.m5.1.2.2.3.2" xref="S2.SS3.p3.5.m5.1.2.2.3.2.cmml">G</mi><mi id="S2.SS3.p3.5.m5.1.2.2.3.3" xref="S2.SS3.p3.5.m5.1.2.2.3.3.cmml">t</mi></msub></mrow><mo id="S2.SS3.p3.5.m5.1.2.1" xref="S2.SS3.p3.5.m5.1.2.1.cmml">=</mo><mfrac id="S2.SS3.p3.5.m5.1.1" xref="S2.SS3.p3.5.m5.1.1.cmml"><mrow id="S2.SS3.p3.5.m5.1.1.1" xref="S2.SS3.p3.5.m5.1.1.1.cmml"><mn id="S2.SS3.p3.5.m5.1.1.1.3" xref="S2.SS3.p3.5.m5.1.1.1.3.cmml">1</mn><mo id="S2.SS3.p3.5.m5.1.1.1.2" xref="S2.SS3.p3.5.m5.1.1.1.2.cmml">−</mo><mrow id="S2.SS3.p3.5.m5.1.1.1.1" xref="S2.SS3.p3.5.m5.1.1.1.1.cmml"><mover accent="true" id="S2.SS3.p3.5.m5.1.1.1.1.3" xref="S2.SS3.p3.5.m5.1.1.1.1.3.cmml"><mrow id="S2.SS3.p3.5.m5.1.1.1.1.3.2" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.cmml"><mi id="S2.SS3.p3.5.m5.1.1.1.1.3.2.2" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.5.m5.1.1.1.1.3.2.1" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.1.cmml">​</mo><mi id="S2.SS3.p3.5.m5.1.1.1.1.3.2.3" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.5.m5.1.1.1.1.3.2.1a" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.1.cmml">​</mo><msub id="S2.SS3.p3.5.m5.1.1.1.1.3.2.4" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.cmml"><mi id="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.2" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.2.cmml">A</mi><mi id="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.3" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.3.cmml">t</mi></msub></mrow><mo id="S2.SS3.p3.5.m5.1.1.1.1.3.1" xref="S2.SS3.p3.5.m5.1.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S2.SS3.p3.5.m5.1.1.1.1.2" xref="S2.SS3.p3.5.m5.1.1.1.1.2.cmml">​</mo><mrow id="S2.SS3.p3.5.m5.1.1.1.1.1.1" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S2.SS3.p3.5.m5.1.1.1.1.1.1.2" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.cmml">(</mo><msub id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.cmml"><mi id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.2" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.2.cmml">S</mi><mrow id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.cmml"><mi id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.2" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.1" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.3" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.1a" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.4" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.1b" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.5" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S2.SS3.p3.5.m5.1.1.1.1.1.1.3" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow></mrow><mn id="S2.SS3.p3.5.m5.1.1.3" xref="S2.SS3.p3.5.m5.1.1.3.cmml">2</mn></mfrac></mrow><annotation-xml encoding="MathML-Content" id="S2.SS3.p3.5.m5.1b"><apply id="S2.SS3.p3.5.m5.1.2.cmml" xref="S2.SS3.p3.5.m5.1.2"><eq id="S2.SS3.p3.5.m5.1.2.1.cmml" xref="S2.SS3.p3.5.m5.1.2.1"></eq><apply id="S2.SS3.p3.5.m5.1.2.2.cmml" xref="S2.SS3.p3.5.m5.1.2.2"><times id="S2.SS3.p3.5.m5.1.2.2.1.cmml" xref="S2.SS3.p3.5.m5.1.2.2.1"></times><ci id="S2.SS3.p3.5.m5.1.2.2.2.cmml" xref="S2.SS3.p3.5.m5.1.2.2.2">𝑃</ci><apply id="S2.SS3.p3.5.m5.1.2.2.3.cmml" xref="S2.SS3.p3.5.m5.1.2.2.3"><csymbol cd="ambiguous" id="S2.SS3.p3.5.m5.1.2.2.3.1.cmml" xref="S2.SS3.p3.5.m5.1.2.2.3">subscript</csymbol><ci id="S2.SS3.p3.5.m5.1.2.2.3.2.cmml" xref="S2.SS3.p3.5.m5.1.2.2.3.2">𝐺</ci><ci id="S2.SS3.p3.5.m5.1.2.2.3.3.cmml" xref="S2.SS3.p3.5.m5.1.2.2.3.3">𝑡</ci></apply></apply><apply id="S2.SS3.p3.5.m5.1.1.cmml" xref="S2.SS3.p3.5.m5.1.1"><divide id="S2.SS3.p3.5.m5.1.1.2.cmml" xref="S2.SS3.p3.5.m5.1.1"></divide><apply id="S2.SS3.p3.5.m5.1.1.1.cmml" xref="S2.SS3.p3.5.m5.1.1.1"><minus id="S2.SS3.p3.5.m5.1.1.1.2.cmml" xref="S2.SS3.p3.5.m5.1.1.1.2"></minus><cn type="integer" id="S2.SS3.p3.5.m5.1.1.1.3.cmml" xref="S2.SS3.p3.5.m5.1.1.1.3">1</cn><apply id="S2.SS3.p3.5.m5.1.1.1.1.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1"><times id="S2.SS3.p3.5.m5.1.1.1.1.2.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.2"></times><apply id="S2.SS3.p3.5.m5.1.1.1.1.3.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3"><ci id="S2.SS3.p3.5.m5.1.1.1.1.3.1.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.1">¯</ci><apply id="S2.SS3.p3.5.m5.1.1.1.1.3.2.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2"><times id="S2.SS3.p3.5.m5.1.1.1.1.3.2.1.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.1"></times><ci id="S2.SS3.p3.5.m5.1.1.1.1.3.2.2.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.2">𝑀</ci><ci id="S2.SS3.p3.5.m5.1.1.1.1.3.2.3.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.3">𝐼</ci><apply id="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.4"><csymbol cd="ambiguous" id="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.1.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.4">subscript</csymbol><ci id="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.2.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.2">𝐴</ci><ci id="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.3.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.3.2.4.3">𝑡</ci></apply></apply></apply><apply id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.1.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1">subscript</csymbol><ci id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.2.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.2">𝑆</ci><apply id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3"><times id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.1.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.1"></times><ci id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.2.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.2">𝑡</ci><ci id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.3.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.3">𝑒</ci><ci id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.4.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.4">𝑠</ci><ci id="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.5.cmml" xref="S2.SS3.p3.5.m5.1.1.1.1.1.1.1.3.5">𝑡</ci></apply></apply></apply></apply><cn type="integer" id="S2.SS3.p3.5.m5.1.1.3.cmml" xref="S2.SS3.p3.5.m5.1.1.3">2</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p3.5.m5.1c">PG_{t}=\frac{1-\overline{MIA_{t}}(S_{test})}{2}</annotation></semantics></math>, where <math id="S2.SS3.p3.6.m6.3" class="ltx_Math" alttext="\overline{MIA_{t}}(S_{test})=\sum_{S_{i}\in S_{test}}\frac{\Pr[MIA_{t}(S_{i})=1]}{2*n_{s}}" display="inline"><semantics id="S2.SS3.p3.6.m6.3a"><mrow id="S2.SS3.p3.6.m6.3.3" xref="S2.SS3.p3.6.m6.3.3.cmml"><mrow id="S2.SS3.p3.6.m6.3.3.1" xref="S2.SS3.p3.6.m6.3.3.1.cmml"><mover accent="true" id="S2.SS3.p3.6.m6.3.3.1.3" xref="S2.SS3.p3.6.m6.3.3.1.3.cmml"><mrow id="S2.SS3.p3.6.m6.3.3.1.3.2" xref="S2.SS3.p3.6.m6.3.3.1.3.2.cmml"><mi id="S2.SS3.p3.6.m6.3.3.1.3.2.2" xref="S2.SS3.p3.6.m6.3.3.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.1.3.2.1" xref="S2.SS3.p3.6.m6.3.3.1.3.2.1.cmml">​</mo><mi id="S2.SS3.p3.6.m6.3.3.1.3.2.3" xref="S2.SS3.p3.6.m6.3.3.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.1.3.2.1a" xref="S2.SS3.p3.6.m6.3.3.1.3.2.1.cmml">​</mo><msub id="S2.SS3.p3.6.m6.3.3.1.3.2.4" xref="S2.SS3.p3.6.m6.3.3.1.3.2.4.cmml"><mi id="S2.SS3.p3.6.m6.3.3.1.3.2.4.2" xref="S2.SS3.p3.6.m6.3.3.1.3.2.4.2.cmml">A</mi><mi id="S2.SS3.p3.6.m6.3.3.1.3.2.4.3" xref="S2.SS3.p3.6.m6.3.3.1.3.2.4.3.cmml">t</mi></msub></mrow><mo id="S2.SS3.p3.6.m6.3.3.1.3.1" xref="S2.SS3.p3.6.m6.3.3.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.1.2" xref="S2.SS3.p3.6.m6.3.3.1.2.cmml">​</mo><mrow id="S2.SS3.p3.6.m6.3.3.1.1.1" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.cmml"><mo stretchy="false" id="S2.SS3.p3.6.m6.3.3.1.1.1.2" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.cmml">(</mo><msub id="S2.SS3.p3.6.m6.3.3.1.1.1.1" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.cmml"><mi id="S2.SS3.p3.6.m6.3.3.1.1.1.1.2" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.2.cmml">S</mi><mrow id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.cmml"><mi id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.2" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.1" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.3" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.1a" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.4" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.1b" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.5" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S2.SS3.p3.6.m6.3.3.1.1.1.3" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.cmml">)</mo></mrow></mrow><mo rspace="0.111em" id="S2.SS3.p3.6.m6.3.3.2" xref="S2.SS3.p3.6.m6.3.3.2.cmml">=</mo><mrow id="S2.SS3.p3.6.m6.3.3.3" xref="S2.SS3.p3.6.m6.3.3.3.cmml"><msub id="S2.SS3.p3.6.m6.3.3.3.1" xref="S2.SS3.p3.6.m6.3.3.3.1.cmml"><mo id="S2.SS3.p3.6.m6.3.3.3.1.2" xref="S2.SS3.p3.6.m6.3.3.3.1.2.cmml">∑</mo><mrow id="S2.SS3.p3.6.m6.3.3.3.1.3" xref="S2.SS3.p3.6.m6.3.3.3.1.3.cmml"><msub id="S2.SS3.p3.6.m6.3.3.3.1.3.2" xref="S2.SS3.p3.6.m6.3.3.3.1.3.2.cmml"><mi id="S2.SS3.p3.6.m6.3.3.3.1.3.2.2" xref="S2.SS3.p3.6.m6.3.3.3.1.3.2.2.cmml">S</mi><mi id="S2.SS3.p3.6.m6.3.3.3.1.3.2.3" xref="S2.SS3.p3.6.m6.3.3.3.1.3.2.3.cmml">i</mi></msub><mo id="S2.SS3.p3.6.m6.3.3.3.1.3.1" xref="S2.SS3.p3.6.m6.3.3.3.1.3.1.cmml">∈</mo><msub id="S2.SS3.p3.6.m6.3.3.3.1.3.3" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.cmml"><mi id="S2.SS3.p3.6.m6.3.3.3.1.3.3.2" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.2.cmml">S</mi><mrow id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.cmml"><mi id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.2" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.1" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.1.cmml">​</mo><mi id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.3" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.1a" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.1.cmml">​</mo><mi id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.4" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.1b" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.1.cmml">​</mo><mi id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.5" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.5.cmml">t</mi></mrow></msub></mrow></msub><mfrac id="S2.SS3.p3.6.m6.2.2" xref="S2.SS3.p3.6.m6.2.2.cmml"><mrow id="S2.SS3.p3.6.m6.2.2.2.2" xref="S2.SS3.p3.6.m6.2.2.2.3.cmml"><mi id="S2.SS3.p3.6.m6.1.1.1.1" xref="S2.SS3.p3.6.m6.1.1.1.1.cmml">Pr</mi><mo id="S2.SS3.p3.6.m6.2.2.2.2a" xref="S2.SS3.p3.6.m6.2.2.2.3.cmml">⁡</mo><mrow id="S2.SS3.p3.6.m6.2.2.2.2.1" xref="S2.SS3.p3.6.m6.2.2.2.3.cmml"><mo stretchy="false" id="S2.SS3.p3.6.m6.2.2.2.2.1.2" xref="S2.SS3.p3.6.m6.2.2.2.3.cmml">[</mo><mrow id="S2.SS3.p3.6.m6.2.2.2.2.1.1" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.cmml"><mrow id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.cmml"><mi id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.3" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.3.cmml">M</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.2" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.2.cmml">​</mo><mi id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.4" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.4.cmml">I</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.2a" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.2.cmml">​</mo><msub id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.cmml"><mi id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.2" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.2.cmml">A</mi><mi id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.3" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.3.cmml">t</mi></msub><mo lspace="0em" rspace="0em" id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.2b" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.2.cmml">​</mo><mrow id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.2" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.cmml">(</mo><msub id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.cmml"><mi id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.2" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.2.cmml">S</mi><mi id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.3" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.3" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S2.SS3.p3.6.m6.2.2.2.2.1.1.2" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.2.cmml">=</mo><mn id="S2.SS3.p3.6.m6.2.2.2.2.1.1.3" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.3.cmml">1</mn></mrow><mo stretchy="false" id="S2.SS3.p3.6.m6.2.2.2.2.1.3" xref="S2.SS3.p3.6.m6.2.2.2.3.cmml">]</mo></mrow></mrow><mrow id="S2.SS3.p3.6.m6.2.2.4" xref="S2.SS3.p3.6.m6.2.2.4.cmml"><mn id="S2.SS3.p3.6.m6.2.2.4.2" xref="S2.SS3.p3.6.m6.2.2.4.2.cmml">2</mn><mo lspace="0.222em" rspace="0.222em" id="S2.SS3.p3.6.m6.2.2.4.1" xref="S2.SS3.p3.6.m6.2.2.4.1.cmml">∗</mo><msub id="S2.SS3.p3.6.m6.2.2.4.3" xref="S2.SS3.p3.6.m6.2.2.4.3.cmml"><mi id="S2.SS3.p3.6.m6.2.2.4.3.2" xref="S2.SS3.p3.6.m6.2.2.4.3.2.cmml">n</mi><mi id="S2.SS3.p3.6.m6.2.2.4.3.3" xref="S2.SS3.p3.6.m6.2.2.4.3.3.cmml">s</mi></msub></mrow></mfrac></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS3.p3.6.m6.3b"><apply id="S2.SS3.p3.6.m6.3.3.cmml" xref="S2.SS3.p3.6.m6.3.3"><eq id="S2.SS3.p3.6.m6.3.3.2.cmml" xref="S2.SS3.p3.6.m6.3.3.2"></eq><apply id="S2.SS3.p3.6.m6.3.3.1.cmml" xref="S2.SS3.p3.6.m6.3.3.1"><times id="S2.SS3.p3.6.m6.3.3.1.2.cmml" xref="S2.SS3.p3.6.m6.3.3.1.2"></times><apply id="S2.SS3.p3.6.m6.3.3.1.3.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3"><ci id="S2.SS3.p3.6.m6.3.3.1.3.1.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.1">¯</ci><apply id="S2.SS3.p3.6.m6.3.3.1.3.2.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.2"><times id="S2.SS3.p3.6.m6.3.3.1.3.2.1.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.2.1"></times><ci id="S2.SS3.p3.6.m6.3.3.1.3.2.2.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.2.2">𝑀</ci><ci id="S2.SS3.p3.6.m6.3.3.1.3.2.3.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.2.3">𝐼</ci><apply id="S2.SS3.p3.6.m6.3.3.1.3.2.4.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.2.4"><csymbol cd="ambiguous" id="S2.SS3.p3.6.m6.3.3.1.3.2.4.1.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.2.4">subscript</csymbol><ci id="S2.SS3.p3.6.m6.3.3.1.3.2.4.2.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.2.4.2">𝐴</ci><ci id="S2.SS3.p3.6.m6.3.3.1.3.2.4.3.cmml" xref="S2.SS3.p3.6.m6.3.3.1.3.2.4.3">𝑡</ci></apply></apply></apply><apply id="S2.SS3.p3.6.m6.3.3.1.1.1.1.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1"><csymbol cd="ambiguous" id="S2.SS3.p3.6.m6.3.3.1.1.1.1.1.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1">subscript</csymbol><ci id="S2.SS3.p3.6.m6.3.3.1.1.1.1.2.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.2">𝑆</ci><apply id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3"><times id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.1.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.1"></times><ci id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.2.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.2">𝑡</ci><ci id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.3.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.3">𝑒</ci><ci id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.4.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.4">𝑠</ci><ci id="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.5.cmml" xref="S2.SS3.p3.6.m6.3.3.1.1.1.1.3.5">𝑡</ci></apply></apply></apply><apply id="S2.SS3.p3.6.m6.3.3.3.cmml" xref="S2.SS3.p3.6.m6.3.3.3"><apply id="S2.SS3.p3.6.m6.3.3.3.1.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1"><csymbol cd="ambiguous" id="S2.SS3.p3.6.m6.3.3.3.1.1.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1">subscript</csymbol><sum id="S2.SS3.p3.6.m6.3.3.3.1.2.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.2"></sum><apply id="S2.SS3.p3.6.m6.3.3.3.1.3.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3"><in id="S2.SS3.p3.6.m6.3.3.3.1.3.1.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.1"></in><apply id="S2.SS3.p3.6.m6.3.3.3.1.3.2.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.2"><csymbol cd="ambiguous" id="S2.SS3.p3.6.m6.3.3.3.1.3.2.1.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.2">subscript</csymbol><ci id="S2.SS3.p3.6.m6.3.3.3.1.3.2.2.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.2.2">𝑆</ci><ci id="S2.SS3.p3.6.m6.3.3.3.1.3.2.3.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.2.3">𝑖</ci></apply><apply id="S2.SS3.p3.6.m6.3.3.3.1.3.3.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3"><csymbol cd="ambiguous" id="S2.SS3.p3.6.m6.3.3.3.1.3.3.1.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3">subscript</csymbol><ci id="S2.SS3.p3.6.m6.3.3.3.1.3.3.2.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.2">𝑆</ci><apply id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3"><times id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.1.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.1"></times><ci id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.2.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.2">𝑡</ci><ci id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.3.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.3">𝑒</ci><ci id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.4.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.4">𝑠</ci><ci id="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.5.cmml" xref="S2.SS3.p3.6.m6.3.3.3.1.3.3.3.5">𝑡</ci></apply></apply></apply></apply><apply id="S2.SS3.p3.6.m6.2.2.cmml" xref="S2.SS3.p3.6.m6.2.2"><divide id="S2.SS3.p3.6.m6.2.2.3.cmml" xref="S2.SS3.p3.6.m6.2.2"></divide><apply id="S2.SS3.p3.6.m6.2.2.2.3.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2"><ci id="S2.SS3.p3.6.m6.1.1.1.1.cmml" xref="S2.SS3.p3.6.m6.1.1.1.1">Pr</ci><apply id="S2.SS3.p3.6.m6.2.2.2.2.1.1.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1"><eq id="S2.SS3.p3.6.m6.2.2.2.2.1.1.2.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.2"></eq><apply id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1"><times id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.2.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.2"></times><ci id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.3.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.3">𝑀</ci><ci id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.4.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.4">𝐼</ci><apply id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5"><csymbol cd="ambiguous" id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.1.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5">subscript</csymbol><ci id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.2.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.2">𝐴</ci><ci id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.3.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.5.3">𝑡</ci></apply><apply id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.1.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1">subscript</csymbol><ci id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.2.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.2">𝑆</ci><ci id="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.3.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.1.1.1.1.3">𝑖</ci></apply></apply><cn type="integer" id="S2.SS3.p3.6.m6.2.2.2.2.1.1.3.cmml" xref="S2.SS3.p3.6.m6.2.2.2.2.1.1.3">1</cn></apply></apply><apply id="S2.SS3.p3.6.m6.2.2.4.cmml" xref="S2.SS3.p3.6.m6.2.2.4"><times id="S2.SS3.p3.6.m6.2.2.4.1.cmml" xref="S2.SS3.p3.6.m6.2.2.4.1"></times><cn type="integer" id="S2.SS3.p3.6.m6.2.2.4.2.cmml" xref="S2.SS3.p3.6.m6.2.2.4.2">2</cn><apply id="S2.SS3.p3.6.m6.2.2.4.3.cmml" xref="S2.SS3.p3.6.m6.2.2.4.3"><csymbol cd="ambiguous" id="S2.SS3.p3.6.m6.2.2.4.3.1.cmml" xref="S2.SS3.p3.6.m6.2.2.4.3">subscript</csymbol><ci id="S2.SS3.p3.6.m6.2.2.4.3.2.cmml" xref="S2.SS3.p3.6.m6.2.2.4.3.2">𝑛</ci><ci id="S2.SS3.p3.6.m6.2.2.4.3.3.cmml" xref="S2.SS3.p3.6.m6.2.2.4.3.3">𝑠</ci></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p3.6.m6.3c">\overline{MIA_{t}}(S_{test})=\sum_{S_{i}\in S_{test}}\frac{\Pr[MIA_{t}(S_{i})=1]}{2*n_{s}}</annotation></semantics></math> (lines 11–12).</p>
</div>
<div id="S2.SS3.p4" class="ltx_para">
<p id="S2.SS3.p4.1" class="ltx_p">Put simply, the PG quantifies the difference between the probability that an attacker correctly identifies that the target record belongs to the real dataset and that that the attacker correctly identifies that the target record was used in training a generative model that outputs a synthetic dataset.</p>
</div>
<figure id="S2.SS3.fig1" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="alg1" class="ltx_float ltx_figure_panel ltx_minipage ltx_align_top ltx_framed ltx_framed_top" style="width:216.8pt;">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_float"><span id="alg1.4.1.1" class="ltx_text ltx_font_bold">Algorithm 1</span> </span> <math id="alg1.2.m1.1" class="ltx_Math" alttext="\mathtt{MIATrain}" display="inline"><semantics id="alg1.2.m1.1b"><mi id="alg1.2.m1.1.1" xref="alg1.2.m1.1.1.cmml">𝙼𝙸𝙰𝚃𝚛𝚊𝚒𝚗</mi><annotation-xml encoding="MathML-Content" id="alg1.2.m1.1c"><ci id="alg1.2.m1.1.1.cmml" xref="alg1.2.m1.1.1">𝙼𝙸𝙰𝚃𝚛𝚊𝚒𝚗</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.2.m1.1d">\mathtt{MIATrain}</annotation></semantics></math> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">59</span></a>]</cite></figcaption>
<div id="alg1.5" class="ltx_listing ltx_listing">
<div id="alg1.l1" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">1:</span><span id="alg1.l1.1" class="ltx_text" style="font-size:90%;">A generative model GM(), the target record </span><math id="alg1.l1.m1.1" class="ltx_Math" alttext="t" display="inline"><semantics id="alg1.l1.m1.1a"><mi mathsize="90%" id="alg1.l1.m1.1.1" xref="alg1.l1.m1.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="alg1.l1.m1.1b"><ci id="alg1.l1.m1.1.1.cmml" xref="alg1.l1.m1.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1.m1.1c">t</annotation></semantics></math><span id="alg1.l1.2" class="ltx_text" style="font-size:90%;">, a reference dataset </span><math id="alg1.l1.m2.1" class="ltx_Math" alttext="R" display="inline"><semantics id="alg1.l1.m2.1a"><mi mathsize="90%" id="alg1.l1.m2.1.1" xref="alg1.l1.m2.1.1.cmml">R</mi><annotation-xml encoding="MathML-Content" id="alg1.l1.m2.1b"><ci id="alg1.l1.m2.1.1.cmml" xref="alg1.l1.m2.1.1">𝑅</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1.m2.1c">R</annotation></semantics></math><span id="alg1.l1.3" class="ltx_text" style="font-size:90%;"> of size </span><math id="alg1.l1.m3.1" class="ltx_Math" alttext="n" display="inline"><semantics id="alg1.l1.m3.1a"><mi mathsize="90%" id="alg1.l1.m3.1.1" xref="alg1.l1.m3.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="alg1.l1.m3.1b"><ci id="alg1.l1.m3.1.1.cmml" xref="alg1.l1.m3.1.1">𝑛</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1.m3.1c">n</annotation></semantics></math><span id="alg1.l1.4" class="ltx_text" style="font-size:90%;">, the number of synthetic test sets </span><math id="alg1.l1.m4.1" class="ltx_Math" alttext="n_{s}" display="inline"><semantics id="alg1.l1.m4.1a"><msub id="alg1.l1.m4.1.1" xref="alg1.l1.m4.1.1.cmml"><mi mathsize="90%" id="alg1.l1.m4.1.1.2" xref="alg1.l1.m4.1.1.2.cmml">n</mi><mi mathsize="90%" id="alg1.l1.m4.1.1.3" xref="alg1.l1.m4.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="alg1.l1.m4.1b"><apply id="alg1.l1.m4.1.1.cmml" xref="alg1.l1.m4.1.1"><csymbol cd="ambiguous" id="alg1.l1.m4.1.1.1.cmml" xref="alg1.l1.m4.1.1">subscript</csymbol><ci id="alg1.l1.m4.1.1.2.cmml" xref="alg1.l1.m4.1.1.2">𝑛</ci><ci id="alg1.l1.m4.1.1.3.cmml" xref="alg1.l1.m4.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1.m4.1c">n_{s}</annotation></semantics></math><span id="alg1.l1.5" class="ltx_text" style="font-size:90%;"> of size </span><math id="alg1.l1.m5.1" class="ltx_Math" alttext="m" display="inline"><semantics id="alg1.l1.m5.1a"><mi mathsize="90%" id="alg1.l1.m5.1.1" xref="alg1.l1.m5.1.1.cmml">m</mi><annotation-xml encoding="MathML-Content" id="alg1.l1.m5.1b"><ci id="alg1.l1.m5.1.1.cmml" xref="alg1.l1.m5.1.1">𝑚</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1.m5.1c">m</annotation></semantics></math><span id="alg1.l1.6" class="ltx_text" style="font-size:90%;">, and the number </span><math id="alg1.l1.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="alg1.l1.m6.1a"><mi mathsize="90%" id="alg1.l1.m6.1.1" xref="alg1.l1.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="alg1.l1.m6.1b"><ci id="alg1.l1.m6.1.1.cmml" xref="alg1.l1.m6.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1.m6.1c">k</annotation></semantics></math><span id="alg1.l1.7" class="ltx_text" style="font-size:90%;"> of shadow models. </span>
</div>
<div id="alg1.l2" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">2:</span><math id="alg1.l2.m1.1" class="ltx_Math" alttext="MIA_{t}()" display="inline"><semantics id="alg1.l2.m1.1a"><mrow id="alg1.l2.m1.1.1" xref="alg1.l2.m1.1.1.cmml"><mi mathsize="90%" id="alg1.l2.m1.1.1.2" xref="alg1.l2.m1.1.1.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="alg1.l2.m1.1.1.1" xref="alg1.l2.m1.1.1.1.cmml">​</mo><mi mathsize="90%" id="alg1.l2.m1.1.1.3" xref="alg1.l2.m1.1.1.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="alg1.l2.m1.1.1.1a" xref="alg1.l2.m1.1.1.1.cmml">​</mo><msub id="alg1.l2.m1.1.1.4" xref="alg1.l2.m1.1.1.4.cmml"><mi mathsize="90%" id="alg1.l2.m1.1.1.4.2" xref="alg1.l2.m1.1.1.4.2.cmml">A</mi><mi mathsize="90%" id="alg1.l2.m1.1.1.4.3" xref="alg1.l2.m1.1.1.4.3.cmml">t</mi></msub><mo lspace="0em" rspace="0em" id="alg1.l2.m1.1.1.1b" xref="alg1.l2.m1.1.1.1.cmml">​</mo><mrow id="alg1.l2.m1.1.1.5.2" xref="alg1.l2.m1.1.1.cmml"><mo maxsize="90%" minsize="90%" id="alg1.l2.m1.1.1.5.2.1" xref="alg1.l2.m1.1.1.5.1.cmml">(</mo><mo maxsize="90%" minsize="90%" id="alg1.l2.m1.1.1.5.2.2" xref="alg1.l2.m1.1.1.5.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l2.m1.1b"><apply id="alg1.l2.m1.1.1.cmml" xref="alg1.l2.m1.1.1"><times id="alg1.l2.m1.1.1.1.cmml" xref="alg1.l2.m1.1.1.1"></times><ci id="alg1.l2.m1.1.1.2.cmml" xref="alg1.l2.m1.1.1.2">𝑀</ci><ci id="alg1.l2.m1.1.1.3.cmml" xref="alg1.l2.m1.1.1.3">𝐼</ci><apply id="alg1.l2.m1.1.1.4.cmml" xref="alg1.l2.m1.1.1.4"><csymbol cd="ambiguous" id="alg1.l2.m1.1.1.4.1.cmml" xref="alg1.l2.m1.1.1.4">subscript</csymbol><ci id="alg1.l2.m1.1.1.4.2.cmml" xref="alg1.l2.m1.1.1.4.2">𝐴</ci><ci id="alg1.l2.m1.1.1.4.3.cmml" xref="alg1.l2.m1.1.1.4.3">𝑡</ci></apply><list id="alg1.l2.m1.1.1.5.1.cmml" xref="alg1.l2.m1.1.1.5.2.1"></list></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l2.m1.1c">MIA_{t}()</annotation></semantics></math><span id="alg1.l2.1" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg1.l3" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">3:</span><span id="alg1.l3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">for</span><span id="alg1.l3.2" class="ltx_text" style="font-size:90%;"> </span><math id="alg1.l3.m1.3" class="ltx_Math" alttext="i=1,\cdots,k" display="inline"><semantics id="alg1.l3.m1.3a"><mrow id="alg1.l3.m1.3.4" xref="alg1.l3.m1.3.4.cmml"><mi mathsize="90%" id="alg1.l3.m1.3.4.2" xref="alg1.l3.m1.3.4.2.cmml">i</mi><mo mathsize="90%" id="alg1.l3.m1.3.4.1" xref="alg1.l3.m1.3.4.1.cmml">=</mo><mrow id="alg1.l3.m1.3.4.3.2" xref="alg1.l3.m1.3.4.3.1.cmml"><mn mathsize="90%" id="alg1.l3.m1.1.1" xref="alg1.l3.m1.1.1.cmml">1</mn><mo mathsize="90%" id="alg1.l3.m1.3.4.3.2.1" xref="alg1.l3.m1.3.4.3.1.cmml">,</mo><mi mathsize="90%" mathvariant="normal" id="alg1.l3.m1.2.2" xref="alg1.l3.m1.2.2.cmml">⋯</mi><mo mathsize="90%" id="alg1.l3.m1.3.4.3.2.2" xref="alg1.l3.m1.3.4.3.1.cmml">,</mo><mi mathsize="90%" id="alg1.l3.m1.3.3" xref="alg1.l3.m1.3.3.cmml">k</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l3.m1.3b"><apply id="alg1.l3.m1.3.4.cmml" xref="alg1.l3.m1.3.4"><eq id="alg1.l3.m1.3.4.1.cmml" xref="alg1.l3.m1.3.4.1"></eq><ci id="alg1.l3.m1.3.4.2.cmml" xref="alg1.l3.m1.3.4.2">𝑖</ci><list id="alg1.l3.m1.3.4.3.1.cmml" xref="alg1.l3.m1.3.4.3.2"><cn type="integer" id="alg1.l3.m1.1.1.cmml" xref="alg1.l3.m1.1.1">1</cn><ci id="alg1.l3.m1.2.2.cmml" xref="alg1.l3.m1.2.2">⋯</ci><ci id="alg1.l3.m1.3.3.cmml" xref="alg1.l3.m1.3.3">𝑘</ci></list></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l3.m1.3c">i=1,\cdots,k</annotation></semantics></math><span id="alg1.l3.3" class="ltx_text" style="font-size:90%;"> </span><span id="alg1.l3.4" class="ltx_text ltx_font_bold" style="font-size:90%;">do</span><span id="alg1.l3.5" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg1.l4" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">4:</span><span id="alg1.l4.1" class="ltx_text" style="font-size:90%;">    </span><math id="alg1.l4.m1.1" class="ltx_Math" alttext="R_{i}\sim R^{n}" display="inline"><semantics id="alg1.l4.m1.1a"><mrow id="alg1.l4.m1.1.1" xref="alg1.l4.m1.1.1.cmml"><msub id="alg1.l4.m1.1.1.2" xref="alg1.l4.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l4.m1.1.1.2.2" xref="alg1.l4.m1.1.1.2.2.cmml">R</mi><mi mathsize="90%" id="alg1.l4.m1.1.1.2.3" xref="alg1.l4.m1.1.1.2.3.cmml">i</mi></msub><mo mathsize="90%" id="alg1.l4.m1.1.1.1" xref="alg1.l4.m1.1.1.1.cmml">∼</mo><msup id="alg1.l4.m1.1.1.3" xref="alg1.l4.m1.1.1.3.cmml"><mi mathsize="90%" id="alg1.l4.m1.1.1.3.2" xref="alg1.l4.m1.1.1.3.2.cmml">R</mi><mi mathsize="90%" id="alg1.l4.m1.1.1.3.3" xref="alg1.l4.m1.1.1.3.3.cmml">n</mi></msup></mrow><annotation-xml encoding="MathML-Content" id="alg1.l4.m1.1b"><apply id="alg1.l4.m1.1.1.cmml" xref="alg1.l4.m1.1.1"><csymbol cd="latexml" id="alg1.l4.m1.1.1.1.cmml" xref="alg1.l4.m1.1.1.1">similar-to</csymbol><apply id="alg1.l4.m1.1.1.2.cmml" xref="alg1.l4.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l4.m1.1.1.2.1.cmml" xref="alg1.l4.m1.1.1.2">subscript</csymbol><ci id="alg1.l4.m1.1.1.2.2.cmml" xref="alg1.l4.m1.1.1.2.2">𝑅</ci><ci id="alg1.l4.m1.1.1.2.3.cmml" xref="alg1.l4.m1.1.1.2.3">𝑖</ci></apply><apply id="alg1.l4.m1.1.1.3.cmml" xref="alg1.l4.m1.1.1.3"><csymbol cd="ambiguous" id="alg1.l4.m1.1.1.3.1.cmml" xref="alg1.l4.m1.1.1.3">superscript</csymbol><ci id="alg1.l4.m1.1.1.3.2.cmml" xref="alg1.l4.m1.1.1.3.2">𝑅</ci><ci id="alg1.l4.m1.1.1.3.3.cmml" xref="alg1.l4.m1.1.1.3.3">𝑛</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l4.m1.1c">R_{i}\sim R^{n}</annotation></semantics></math><span id="alg1.l4.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg1.l5" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">5:</span><span id="alg1.l5.1" class="ltx_text" style="font-size:90%;">    </span><math id="alg1.l5.m1.1" class="ltx_Math" alttext="f_{i}\sim" display="inline"><semantics id="alg1.l5.m1.1a"><mrow id="alg1.l5.m1.1.1" xref="alg1.l5.m1.1.1.cmml"><msub id="alg1.l5.m1.1.1.2" xref="alg1.l5.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l5.m1.1.1.2.2" xref="alg1.l5.m1.1.1.2.2.cmml">f</mi><mi mathsize="90%" id="alg1.l5.m1.1.1.2.3" xref="alg1.l5.m1.1.1.2.3.cmml">i</mi></msub><mo mathsize="90%" id="alg1.l5.m1.1.1.1" xref="alg1.l5.m1.1.1.1.cmml">∼</mo><mi id="alg1.l5.m1.1.1.3" xref="alg1.l5.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg1.l5.m1.1b"><apply id="alg1.l5.m1.1.1.cmml" xref="alg1.l5.m1.1.1"><csymbol cd="latexml" id="alg1.l5.m1.1.1.1.cmml" xref="alg1.l5.m1.1.1.1">similar-to</csymbol><apply id="alg1.l5.m1.1.1.2.cmml" xref="alg1.l5.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l5.m1.1.1.2.1.cmml" xref="alg1.l5.m1.1.1.2">subscript</csymbol><ci id="alg1.l5.m1.1.1.2.2.cmml" xref="alg1.l5.m1.1.1.2.2">𝑓</ci><ci id="alg1.l5.m1.1.1.2.3.cmml" xref="alg1.l5.m1.1.1.2.3">𝑖</ci></apply><csymbol cd="latexml" id="alg1.l5.m1.1.1.3.cmml" xref="alg1.l5.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l5.m1.1c">f_{i}\sim</annotation></semantics></math><span id="alg1.l5.2" class="ltx_text" style="font-size:90%;"> GM(</span><math id="alg1.l5.m2.1" class="ltx_Math" alttext="{R_{i}}" display="inline"><semantics id="alg1.l5.m2.1a"><msub id="alg1.l5.m2.1.1" xref="alg1.l5.m2.1.1.cmml"><mi mathsize="90%" id="alg1.l5.m2.1.1.2" xref="alg1.l5.m2.1.1.2.cmml">R</mi><mi mathsize="90%" id="alg1.l5.m2.1.1.3" xref="alg1.l5.m2.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="alg1.l5.m2.1b"><apply id="alg1.l5.m2.1.1.cmml" xref="alg1.l5.m2.1.1"><csymbol cd="ambiguous" id="alg1.l5.m2.1.1.1.cmml" xref="alg1.l5.m2.1.1">subscript</csymbol><ci id="alg1.l5.m2.1.1.2.cmml" xref="alg1.l5.m2.1.1.2">𝑅</ci><ci id="alg1.l5.m2.1.1.3.cmml" xref="alg1.l5.m2.1.1.3">𝑖</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l5.m2.1c">{R_{i}}</annotation></semantics></math><span id="alg1.l5.3" class="ltx_text" style="font-size:90%;">) 
</span>
</div>
<div id="alg1.l6" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">6:</span><span id="alg1.l6.1" class="ltx_text" style="font-size:90%;">    </span><span id="alg1.l6.2" class="ltx_text ltx_font_bold" style="font-size:90%;">for</span><span id="alg1.l6.3" class="ltx_text" style="font-size:90%;"> </span><math id="alg1.l6.m1.3" class="ltx_Math" alttext="j=1,\cdots,n_{s}" display="inline"><semantics id="alg1.l6.m1.3a"><mrow id="alg1.l6.m1.3.3" xref="alg1.l6.m1.3.3.cmml"><mi mathsize="90%" id="alg1.l6.m1.3.3.3" xref="alg1.l6.m1.3.3.3.cmml">j</mi><mo mathsize="90%" id="alg1.l6.m1.3.3.2" xref="alg1.l6.m1.3.3.2.cmml">=</mo><mrow id="alg1.l6.m1.3.3.1.1" xref="alg1.l6.m1.3.3.1.2.cmml"><mn mathsize="90%" id="alg1.l6.m1.1.1" xref="alg1.l6.m1.1.1.cmml">1</mn><mo mathsize="90%" id="alg1.l6.m1.3.3.1.1.2" xref="alg1.l6.m1.3.3.1.2.cmml">,</mo><mi mathsize="90%" mathvariant="normal" id="alg1.l6.m1.2.2" xref="alg1.l6.m1.2.2.cmml">⋯</mi><mo mathsize="90%" id="alg1.l6.m1.3.3.1.1.3" xref="alg1.l6.m1.3.3.1.2.cmml">,</mo><msub id="alg1.l6.m1.3.3.1.1.1" xref="alg1.l6.m1.3.3.1.1.1.cmml"><mi mathsize="90%" id="alg1.l6.m1.3.3.1.1.1.2" xref="alg1.l6.m1.3.3.1.1.1.2.cmml">n</mi><mi mathsize="90%" id="alg1.l6.m1.3.3.1.1.1.3" xref="alg1.l6.m1.3.3.1.1.1.3.cmml">s</mi></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l6.m1.3b"><apply id="alg1.l6.m1.3.3.cmml" xref="alg1.l6.m1.3.3"><eq id="alg1.l6.m1.3.3.2.cmml" xref="alg1.l6.m1.3.3.2"></eq><ci id="alg1.l6.m1.3.3.3.cmml" xref="alg1.l6.m1.3.3.3">𝑗</ci><list id="alg1.l6.m1.3.3.1.2.cmml" xref="alg1.l6.m1.3.3.1.1"><cn type="integer" id="alg1.l6.m1.1.1.cmml" xref="alg1.l6.m1.1.1">1</cn><ci id="alg1.l6.m1.2.2.cmml" xref="alg1.l6.m1.2.2">⋯</ci><apply id="alg1.l6.m1.3.3.1.1.1.cmml" xref="alg1.l6.m1.3.3.1.1.1"><csymbol cd="ambiguous" id="alg1.l6.m1.3.3.1.1.1.1.cmml" xref="alg1.l6.m1.3.3.1.1.1">subscript</csymbol><ci id="alg1.l6.m1.3.3.1.1.1.2.cmml" xref="alg1.l6.m1.3.3.1.1.1.2">𝑛</ci><ci id="alg1.l6.m1.3.3.1.1.1.3.cmml" xref="alg1.l6.m1.3.3.1.1.1.3">𝑠</ci></apply></list></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l6.m1.3c">j=1,\cdots,n_{s}</annotation></semantics></math><span id="alg1.l6.4" class="ltx_text" style="font-size:90%;"> </span><span id="alg1.l6.5" class="ltx_text ltx_font_bold" style="font-size:90%;">do</span><span id="alg1.l6.6" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg1.l7" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">7:</span><span id="alg1.l7.1" class="ltx_text" style="font-size:90%;">         </span><math id="alg1.l7.m1.1" class="ltx_Math" alttext="S_{j}^{m}\sim f_{i}" display="inline"><semantics id="alg1.l7.m1.1a"><mrow id="alg1.l7.m1.1.1" xref="alg1.l7.m1.1.1.cmml"><msubsup id="alg1.l7.m1.1.1.2" xref="alg1.l7.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l7.m1.1.1.2.2.2" xref="alg1.l7.m1.1.1.2.2.2.cmml">S</mi><mi mathsize="90%" id="alg1.l7.m1.1.1.2.2.3" xref="alg1.l7.m1.1.1.2.2.3.cmml">j</mi><mi mathsize="90%" id="alg1.l7.m1.1.1.2.3" xref="alg1.l7.m1.1.1.2.3.cmml">m</mi></msubsup><mo mathsize="90%" id="alg1.l7.m1.1.1.1" xref="alg1.l7.m1.1.1.1.cmml">∼</mo><msub id="alg1.l7.m1.1.1.3" xref="alg1.l7.m1.1.1.3.cmml"><mi mathsize="90%" id="alg1.l7.m1.1.1.3.2" xref="alg1.l7.m1.1.1.3.2.cmml">f</mi><mi mathsize="90%" id="alg1.l7.m1.1.1.3.3" xref="alg1.l7.m1.1.1.3.3.cmml">i</mi></msub></mrow><annotation-xml encoding="MathML-Content" id="alg1.l7.m1.1b"><apply id="alg1.l7.m1.1.1.cmml" xref="alg1.l7.m1.1.1"><csymbol cd="latexml" id="alg1.l7.m1.1.1.1.cmml" xref="alg1.l7.m1.1.1.1">similar-to</csymbol><apply id="alg1.l7.m1.1.1.2.cmml" xref="alg1.l7.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l7.m1.1.1.2.1.cmml" xref="alg1.l7.m1.1.1.2">superscript</csymbol><apply id="alg1.l7.m1.1.1.2.2.cmml" xref="alg1.l7.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l7.m1.1.1.2.2.1.cmml" xref="alg1.l7.m1.1.1.2">subscript</csymbol><ci id="alg1.l7.m1.1.1.2.2.2.cmml" xref="alg1.l7.m1.1.1.2.2.2">𝑆</ci><ci id="alg1.l7.m1.1.1.2.2.3.cmml" xref="alg1.l7.m1.1.1.2.2.3">𝑗</ci></apply><ci id="alg1.l7.m1.1.1.2.3.cmml" xref="alg1.l7.m1.1.1.2.3">𝑚</ci></apply><apply id="alg1.l7.m1.1.1.3.cmml" xref="alg1.l7.m1.1.1.3"><csymbol cd="ambiguous" id="alg1.l7.m1.1.1.3.1.cmml" xref="alg1.l7.m1.1.1.3">subscript</csymbol><ci id="alg1.l7.m1.1.1.3.2.cmml" xref="alg1.l7.m1.1.1.3.2">𝑓</ci><ci id="alg1.l7.m1.1.1.3.3.cmml" xref="alg1.l7.m1.1.1.3.3">𝑖</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l7.m1.1c">S_{j}^{m}\sim f_{i}</annotation></semantics></math><span id="alg1.l7.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg1.l8" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">8:</span><span id="alg1.l8.1" class="ltx_text" style="font-size:90%;">         </span><math id="alg1.l8.m1.1" class="ltx_Math" alttext="S_{\mathtt{train}}\leftarrow S_{\mathtt{train}}\cup S_{j}^{m}" display="inline"><semantics id="alg1.l8.m1.1a"><mrow id="alg1.l8.m1.1.1" xref="alg1.l8.m1.1.1.cmml"><msub id="alg1.l8.m1.1.1.2" xref="alg1.l8.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l8.m1.1.1.2.2" xref="alg1.l8.m1.1.1.2.2.cmml">S</mi><mi mathsize="90%" id="alg1.l8.m1.1.1.2.3" xref="alg1.l8.m1.1.1.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" stretchy="false" id="alg1.l8.m1.1.1.1" xref="alg1.l8.m1.1.1.1.cmml">←</mo><mrow id="alg1.l8.m1.1.1.3" xref="alg1.l8.m1.1.1.3.cmml"><msub id="alg1.l8.m1.1.1.3.2" xref="alg1.l8.m1.1.1.3.2.cmml"><mi mathsize="90%" id="alg1.l8.m1.1.1.3.2.2" xref="alg1.l8.m1.1.1.3.2.2.cmml">S</mi><mi mathsize="90%" id="alg1.l8.m1.1.1.3.2.3" xref="alg1.l8.m1.1.1.3.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" id="alg1.l8.m1.1.1.3.1" xref="alg1.l8.m1.1.1.3.1.cmml">∪</mo><msubsup id="alg1.l8.m1.1.1.3.3" xref="alg1.l8.m1.1.1.3.3.cmml"><mi mathsize="90%" id="alg1.l8.m1.1.1.3.3.2.2" xref="alg1.l8.m1.1.1.3.3.2.2.cmml">S</mi><mi mathsize="90%" id="alg1.l8.m1.1.1.3.3.2.3" xref="alg1.l8.m1.1.1.3.3.2.3.cmml">j</mi><mi mathsize="90%" id="alg1.l8.m1.1.1.3.3.3" xref="alg1.l8.m1.1.1.3.3.3.cmml">m</mi></msubsup></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l8.m1.1b"><apply id="alg1.l8.m1.1.1.cmml" xref="alg1.l8.m1.1.1"><ci id="alg1.l8.m1.1.1.1.cmml" xref="alg1.l8.m1.1.1.1">←</ci><apply id="alg1.l8.m1.1.1.2.cmml" xref="alg1.l8.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.2.1.cmml" xref="alg1.l8.m1.1.1.2">subscript</csymbol><ci id="alg1.l8.m1.1.1.2.2.cmml" xref="alg1.l8.m1.1.1.2.2">𝑆</ci><ci id="alg1.l8.m1.1.1.2.3.cmml" xref="alg1.l8.m1.1.1.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><apply id="alg1.l8.m1.1.1.3.cmml" xref="alg1.l8.m1.1.1.3"><union id="alg1.l8.m1.1.1.3.1.cmml" xref="alg1.l8.m1.1.1.3.1"></union><apply id="alg1.l8.m1.1.1.3.2.cmml" xref="alg1.l8.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.3.2.1.cmml" xref="alg1.l8.m1.1.1.3.2">subscript</csymbol><ci id="alg1.l8.m1.1.1.3.2.2.cmml" xref="alg1.l8.m1.1.1.3.2.2">𝑆</ci><ci id="alg1.l8.m1.1.1.3.2.3.cmml" xref="alg1.l8.m1.1.1.3.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><apply id="alg1.l8.m1.1.1.3.3.cmml" xref="alg1.l8.m1.1.1.3.3"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.3.3.1.cmml" xref="alg1.l8.m1.1.1.3.3">superscript</csymbol><apply id="alg1.l8.m1.1.1.3.3.2.cmml" xref="alg1.l8.m1.1.1.3.3"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.3.3.2.1.cmml" xref="alg1.l8.m1.1.1.3.3">subscript</csymbol><ci id="alg1.l8.m1.1.1.3.3.2.2.cmml" xref="alg1.l8.m1.1.1.3.3.2.2">𝑆</ci><ci id="alg1.l8.m1.1.1.3.3.2.3.cmml" xref="alg1.l8.m1.1.1.3.3.2.3">𝑗</ci></apply><ci id="alg1.l8.m1.1.1.3.3.3.cmml" xref="alg1.l8.m1.1.1.3.3.3">𝑚</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l8.m1.1c">S_{\mathtt{train}}\leftarrow S_{\mathtt{train}}\cup S_{j}^{m}</annotation></semantics></math><span id="alg1.l8.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg1.l9" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">9:</span><span id="alg1.l9.1" class="ltx_text" style="font-size:90%;">         </span><math id="alg1.l9.m1.1" class="ltx_Math" alttext="l_{\mathtt{train}}\leftarrow l_{\mathtt{train}}\cup 0" display="inline"><semantics id="alg1.l9.m1.1a"><mrow id="alg1.l9.m1.1.1" xref="alg1.l9.m1.1.1.cmml"><msub id="alg1.l9.m1.1.1.2" xref="alg1.l9.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l9.m1.1.1.2.2" xref="alg1.l9.m1.1.1.2.2.cmml">l</mi><mi mathsize="90%" id="alg1.l9.m1.1.1.2.3" xref="alg1.l9.m1.1.1.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" stretchy="false" id="alg1.l9.m1.1.1.1" xref="alg1.l9.m1.1.1.1.cmml">←</mo><mrow id="alg1.l9.m1.1.1.3" xref="alg1.l9.m1.1.1.3.cmml"><msub id="alg1.l9.m1.1.1.3.2" xref="alg1.l9.m1.1.1.3.2.cmml"><mi mathsize="90%" id="alg1.l9.m1.1.1.3.2.2" xref="alg1.l9.m1.1.1.3.2.2.cmml">l</mi><mi mathsize="90%" id="alg1.l9.m1.1.1.3.2.3" xref="alg1.l9.m1.1.1.3.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" id="alg1.l9.m1.1.1.3.1" xref="alg1.l9.m1.1.1.3.1.cmml">∪</mo><mn mathsize="90%" id="alg1.l9.m1.1.1.3.3" xref="alg1.l9.m1.1.1.3.3.cmml">0</mn></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l9.m1.1b"><apply id="alg1.l9.m1.1.1.cmml" xref="alg1.l9.m1.1.1"><ci id="alg1.l9.m1.1.1.1.cmml" xref="alg1.l9.m1.1.1.1">←</ci><apply id="alg1.l9.m1.1.1.2.cmml" xref="alg1.l9.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l9.m1.1.1.2.1.cmml" xref="alg1.l9.m1.1.1.2">subscript</csymbol><ci id="alg1.l9.m1.1.1.2.2.cmml" xref="alg1.l9.m1.1.1.2.2">𝑙</ci><ci id="alg1.l9.m1.1.1.2.3.cmml" xref="alg1.l9.m1.1.1.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><apply id="alg1.l9.m1.1.1.3.cmml" xref="alg1.l9.m1.1.1.3"><union id="alg1.l9.m1.1.1.3.1.cmml" xref="alg1.l9.m1.1.1.3.1"></union><apply id="alg1.l9.m1.1.1.3.2.cmml" xref="alg1.l9.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg1.l9.m1.1.1.3.2.1.cmml" xref="alg1.l9.m1.1.1.3.2">subscript</csymbol><ci id="alg1.l9.m1.1.1.3.2.2.cmml" xref="alg1.l9.m1.1.1.3.2.2">𝑙</ci><ci id="alg1.l9.m1.1.1.3.2.3.cmml" xref="alg1.l9.m1.1.1.3.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><cn type="integer" id="alg1.l9.m1.1.1.3.3.cmml" xref="alg1.l9.m1.1.1.3.3">0</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l9.m1.1c">l_{\mathtt{train}}\leftarrow l_{\mathtt{train}}\cup 0</annotation></semantics></math><span id="alg1.l9.2" class="ltx_text" style="font-size:90%;"> 
    </span>
</div>
<div id="alg1.l10" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">10:</span><span id="alg1.l10.1" class="ltx_text" style="font-size:90%;">    </span><math id="alg1.l10.m1.1" class="ltx_Math" alttext="R^{\prime}_{i}\leftarrow R_{i}\cup t" display="inline"><semantics id="alg1.l10.m1.1a"><mrow id="alg1.l10.m1.1.1" xref="alg1.l10.m1.1.1.cmml"><msubsup id="alg1.l10.m1.1.1.2" xref="alg1.l10.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l10.m1.1.1.2.2.2" xref="alg1.l10.m1.1.1.2.2.2.cmml">R</mi><mi mathsize="90%" id="alg1.l10.m1.1.1.2.3" xref="alg1.l10.m1.1.1.2.3.cmml">i</mi><mo mathsize="90%" id="alg1.l10.m1.1.1.2.2.3" xref="alg1.l10.m1.1.1.2.2.3.cmml">′</mo></msubsup><mo mathsize="90%" stretchy="false" id="alg1.l10.m1.1.1.1" xref="alg1.l10.m1.1.1.1.cmml">←</mo><mrow id="alg1.l10.m1.1.1.3" xref="alg1.l10.m1.1.1.3.cmml"><msub id="alg1.l10.m1.1.1.3.2" xref="alg1.l10.m1.1.1.3.2.cmml"><mi mathsize="90%" id="alg1.l10.m1.1.1.3.2.2" xref="alg1.l10.m1.1.1.3.2.2.cmml">R</mi><mi mathsize="90%" id="alg1.l10.m1.1.1.3.2.3" xref="alg1.l10.m1.1.1.3.2.3.cmml">i</mi></msub><mo mathsize="90%" id="alg1.l10.m1.1.1.3.1" xref="alg1.l10.m1.1.1.3.1.cmml">∪</mo><mi mathsize="90%" id="alg1.l10.m1.1.1.3.3" xref="alg1.l10.m1.1.1.3.3.cmml">t</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l10.m1.1b"><apply id="alg1.l10.m1.1.1.cmml" xref="alg1.l10.m1.1.1"><ci id="alg1.l10.m1.1.1.1.cmml" xref="alg1.l10.m1.1.1.1">←</ci><apply id="alg1.l10.m1.1.1.2.cmml" xref="alg1.l10.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l10.m1.1.1.2.1.cmml" xref="alg1.l10.m1.1.1.2">subscript</csymbol><apply id="alg1.l10.m1.1.1.2.2.cmml" xref="alg1.l10.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l10.m1.1.1.2.2.1.cmml" xref="alg1.l10.m1.1.1.2">superscript</csymbol><ci id="alg1.l10.m1.1.1.2.2.2.cmml" xref="alg1.l10.m1.1.1.2.2.2">𝑅</ci><ci id="alg1.l10.m1.1.1.2.2.3.cmml" xref="alg1.l10.m1.1.1.2.2.3">′</ci></apply><ci id="alg1.l10.m1.1.1.2.3.cmml" xref="alg1.l10.m1.1.1.2.3">𝑖</ci></apply><apply id="alg1.l10.m1.1.1.3.cmml" xref="alg1.l10.m1.1.1.3"><union id="alg1.l10.m1.1.1.3.1.cmml" xref="alg1.l10.m1.1.1.3.1"></union><apply id="alg1.l10.m1.1.1.3.2.cmml" xref="alg1.l10.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg1.l10.m1.1.1.3.2.1.cmml" xref="alg1.l10.m1.1.1.3.2">subscript</csymbol><ci id="alg1.l10.m1.1.1.3.2.2.cmml" xref="alg1.l10.m1.1.1.3.2.2">𝑅</ci><ci id="alg1.l10.m1.1.1.3.2.3.cmml" xref="alg1.l10.m1.1.1.3.2.3">𝑖</ci></apply><ci id="alg1.l10.m1.1.1.3.3.cmml" xref="alg1.l10.m1.1.1.3.3">𝑡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l10.m1.1c">R^{\prime}_{i}\leftarrow R_{i}\cup t</annotation></semantics></math><span id="alg1.l10.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg1.l11" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">11:</span><span id="alg1.l11.1" class="ltx_text" style="font-size:90%;">    </span><math id="alg1.l11.m1.1" class="ltx_Math" alttext="f^{\prime}_{i}\sim" display="inline"><semantics id="alg1.l11.m1.1a"><mrow id="alg1.l11.m1.1.1" xref="alg1.l11.m1.1.1.cmml"><msubsup id="alg1.l11.m1.1.1.2" xref="alg1.l11.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l11.m1.1.1.2.2.2" xref="alg1.l11.m1.1.1.2.2.2.cmml">f</mi><mi mathsize="90%" id="alg1.l11.m1.1.1.2.3" xref="alg1.l11.m1.1.1.2.3.cmml">i</mi><mo mathsize="90%" id="alg1.l11.m1.1.1.2.2.3" xref="alg1.l11.m1.1.1.2.2.3.cmml">′</mo></msubsup><mo mathsize="90%" id="alg1.l11.m1.1.1.1" xref="alg1.l11.m1.1.1.1.cmml">∼</mo><mi id="alg1.l11.m1.1.1.3" xref="alg1.l11.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg1.l11.m1.1b"><apply id="alg1.l11.m1.1.1.cmml" xref="alg1.l11.m1.1.1"><csymbol cd="latexml" id="alg1.l11.m1.1.1.1.cmml" xref="alg1.l11.m1.1.1.1">similar-to</csymbol><apply id="alg1.l11.m1.1.1.2.cmml" xref="alg1.l11.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l11.m1.1.1.2.1.cmml" xref="alg1.l11.m1.1.1.2">subscript</csymbol><apply id="alg1.l11.m1.1.1.2.2.cmml" xref="alg1.l11.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l11.m1.1.1.2.2.1.cmml" xref="alg1.l11.m1.1.1.2">superscript</csymbol><ci id="alg1.l11.m1.1.1.2.2.2.cmml" xref="alg1.l11.m1.1.1.2.2.2">𝑓</ci><ci id="alg1.l11.m1.1.1.2.2.3.cmml" xref="alg1.l11.m1.1.1.2.2.3">′</ci></apply><ci id="alg1.l11.m1.1.1.2.3.cmml" xref="alg1.l11.m1.1.1.2.3">𝑖</ci></apply><csymbol cd="latexml" id="alg1.l11.m1.1.1.3.cmml" xref="alg1.l11.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l11.m1.1c">f^{\prime}_{i}\sim</annotation></semantics></math><span id="alg1.l11.2" class="ltx_text" style="font-size:90%;"> GM(</span><math id="alg1.l11.m2.1" class="ltx_Math" alttext="{R^{\prime}_{i}}" display="inline"><semantics id="alg1.l11.m2.1a"><msubsup id="alg1.l11.m2.1.1" xref="alg1.l11.m2.1.1.cmml"><mi mathsize="90%" id="alg1.l11.m2.1.1.2.2" xref="alg1.l11.m2.1.1.2.2.cmml">R</mi><mi mathsize="90%" id="alg1.l11.m2.1.1.3" xref="alg1.l11.m2.1.1.3.cmml">i</mi><mo mathsize="90%" id="alg1.l11.m2.1.1.2.3" xref="alg1.l11.m2.1.1.2.3.cmml">′</mo></msubsup><annotation-xml encoding="MathML-Content" id="alg1.l11.m2.1b"><apply id="alg1.l11.m2.1.1.cmml" xref="alg1.l11.m2.1.1"><csymbol cd="ambiguous" id="alg1.l11.m2.1.1.1.cmml" xref="alg1.l11.m2.1.1">subscript</csymbol><apply id="alg1.l11.m2.1.1.2.cmml" xref="alg1.l11.m2.1.1"><csymbol cd="ambiguous" id="alg1.l11.m2.1.1.2.1.cmml" xref="alg1.l11.m2.1.1">superscript</csymbol><ci id="alg1.l11.m2.1.1.2.2.cmml" xref="alg1.l11.m2.1.1.2.2">𝑅</ci><ci id="alg1.l11.m2.1.1.2.3.cmml" xref="alg1.l11.m2.1.1.2.3">′</ci></apply><ci id="alg1.l11.m2.1.1.3.cmml" xref="alg1.l11.m2.1.1.3">𝑖</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l11.m2.1c">{R^{\prime}_{i}}</annotation></semantics></math><span id="alg1.l11.3" class="ltx_text" style="font-size:90%;">) 
</span>
</div>
<div id="alg1.l12" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">12:</span><span id="alg1.l12.1" class="ltx_text" style="font-size:90%;">    </span><span id="alg1.l12.2" class="ltx_text ltx_font_bold" style="font-size:90%;">for</span><span id="alg1.l12.3" class="ltx_text" style="font-size:90%;"> </span><math id="alg1.l12.m1.3" class="ltx_Math" alttext="j=1,\cdots,n_{s}" display="inline"><semantics id="alg1.l12.m1.3a"><mrow id="alg1.l12.m1.3.3" xref="alg1.l12.m1.3.3.cmml"><mi mathsize="90%" id="alg1.l12.m1.3.3.3" xref="alg1.l12.m1.3.3.3.cmml">j</mi><mo mathsize="90%" id="alg1.l12.m1.3.3.2" xref="alg1.l12.m1.3.3.2.cmml">=</mo><mrow id="alg1.l12.m1.3.3.1.1" xref="alg1.l12.m1.3.3.1.2.cmml"><mn mathsize="90%" id="alg1.l12.m1.1.1" xref="alg1.l12.m1.1.1.cmml">1</mn><mo mathsize="90%" id="alg1.l12.m1.3.3.1.1.2" xref="alg1.l12.m1.3.3.1.2.cmml">,</mo><mi mathsize="90%" mathvariant="normal" id="alg1.l12.m1.2.2" xref="alg1.l12.m1.2.2.cmml">⋯</mi><mo mathsize="90%" id="alg1.l12.m1.3.3.1.1.3" xref="alg1.l12.m1.3.3.1.2.cmml">,</mo><msub id="alg1.l12.m1.3.3.1.1.1" xref="alg1.l12.m1.3.3.1.1.1.cmml"><mi mathsize="90%" id="alg1.l12.m1.3.3.1.1.1.2" xref="alg1.l12.m1.3.3.1.1.1.2.cmml">n</mi><mi mathsize="90%" id="alg1.l12.m1.3.3.1.1.1.3" xref="alg1.l12.m1.3.3.1.1.1.3.cmml">s</mi></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l12.m1.3b"><apply id="alg1.l12.m1.3.3.cmml" xref="alg1.l12.m1.3.3"><eq id="alg1.l12.m1.3.3.2.cmml" xref="alg1.l12.m1.3.3.2"></eq><ci id="alg1.l12.m1.3.3.3.cmml" xref="alg1.l12.m1.3.3.3">𝑗</ci><list id="alg1.l12.m1.3.3.1.2.cmml" xref="alg1.l12.m1.3.3.1.1"><cn type="integer" id="alg1.l12.m1.1.1.cmml" xref="alg1.l12.m1.1.1">1</cn><ci id="alg1.l12.m1.2.2.cmml" xref="alg1.l12.m1.2.2">⋯</ci><apply id="alg1.l12.m1.3.3.1.1.1.cmml" xref="alg1.l12.m1.3.3.1.1.1"><csymbol cd="ambiguous" id="alg1.l12.m1.3.3.1.1.1.1.cmml" xref="alg1.l12.m1.3.3.1.1.1">subscript</csymbol><ci id="alg1.l12.m1.3.3.1.1.1.2.cmml" xref="alg1.l12.m1.3.3.1.1.1.2">𝑛</ci><ci id="alg1.l12.m1.3.3.1.1.1.3.cmml" xref="alg1.l12.m1.3.3.1.1.1.3">𝑠</ci></apply></list></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l12.m1.3c">j=1,\cdots,n_{s}</annotation></semantics></math><span id="alg1.l12.4" class="ltx_text" style="font-size:90%;"> </span><span id="alg1.l12.5" class="ltx_text ltx_font_bold" style="font-size:90%;">do</span><span id="alg1.l12.6" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg1.l13" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">13:</span><span id="alg1.l13.1" class="ltx_text" style="font-size:90%;">         </span><math id="alg1.l13.m1.1" class="ltx_Math" alttext="S_{j}^{m}\sim f^{\prime}_{i}" display="inline"><semantics id="alg1.l13.m1.1a"><mrow id="alg1.l13.m1.1.1" xref="alg1.l13.m1.1.1.cmml"><msubsup id="alg1.l13.m1.1.1.2" xref="alg1.l13.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l13.m1.1.1.2.2.2" xref="alg1.l13.m1.1.1.2.2.2.cmml">S</mi><mi mathsize="90%" id="alg1.l13.m1.1.1.2.2.3" xref="alg1.l13.m1.1.1.2.2.3.cmml">j</mi><mi mathsize="90%" id="alg1.l13.m1.1.1.2.3" xref="alg1.l13.m1.1.1.2.3.cmml">m</mi></msubsup><mo mathsize="90%" id="alg1.l13.m1.1.1.1" xref="alg1.l13.m1.1.1.1.cmml">∼</mo><msubsup id="alg1.l13.m1.1.1.3" xref="alg1.l13.m1.1.1.3.cmml"><mi mathsize="90%" id="alg1.l13.m1.1.1.3.2.2" xref="alg1.l13.m1.1.1.3.2.2.cmml">f</mi><mi mathsize="90%" id="alg1.l13.m1.1.1.3.3" xref="alg1.l13.m1.1.1.3.3.cmml">i</mi><mo mathsize="90%" id="alg1.l13.m1.1.1.3.2.3" xref="alg1.l13.m1.1.1.3.2.3.cmml">′</mo></msubsup></mrow><annotation-xml encoding="MathML-Content" id="alg1.l13.m1.1b"><apply id="alg1.l13.m1.1.1.cmml" xref="alg1.l13.m1.1.1"><csymbol cd="latexml" id="alg1.l13.m1.1.1.1.cmml" xref="alg1.l13.m1.1.1.1">similar-to</csymbol><apply id="alg1.l13.m1.1.1.2.cmml" xref="alg1.l13.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l13.m1.1.1.2.1.cmml" xref="alg1.l13.m1.1.1.2">superscript</csymbol><apply id="alg1.l13.m1.1.1.2.2.cmml" xref="alg1.l13.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l13.m1.1.1.2.2.1.cmml" xref="alg1.l13.m1.1.1.2">subscript</csymbol><ci id="alg1.l13.m1.1.1.2.2.2.cmml" xref="alg1.l13.m1.1.1.2.2.2">𝑆</ci><ci id="alg1.l13.m1.1.1.2.2.3.cmml" xref="alg1.l13.m1.1.1.2.2.3">𝑗</ci></apply><ci id="alg1.l13.m1.1.1.2.3.cmml" xref="alg1.l13.m1.1.1.2.3">𝑚</ci></apply><apply id="alg1.l13.m1.1.1.3.cmml" xref="alg1.l13.m1.1.1.3"><csymbol cd="ambiguous" id="alg1.l13.m1.1.1.3.1.cmml" xref="alg1.l13.m1.1.1.3">subscript</csymbol><apply id="alg1.l13.m1.1.1.3.2.cmml" xref="alg1.l13.m1.1.1.3"><csymbol cd="ambiguous" id="alg1.l13.m1.1.1.3.2.1.cmml" xref="alg1.l13.m1.1.1.3">superscript</csymbol><ci id="alg1.l13.m1.1.1.3.2.2.cmml" xref="alg1.l13.m1.1.1.3.2.2">𝑓</ci><ci id="alg1.l13.m1.1.1.3.2.3.cmml" xref="alg1.l13.m1.1.1.3.2.3">′</ci></apply><ci id="alg1.l13.m1.1.1.3.3.cmml" xref="alg1.l13.m1.1.1.3.3">𝑖</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l13.m1.1c">S_{j}^{m}\sim f^{\prime}_{i}</annotation></semantics></math><span id="alg1.l13.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg1.l14" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">14:</span><span id="alg1.l14.1" class="ltx_text" style="font-size:90%;">         </span><math id="alg1.l14.m1.1" class="ltx_Math" alttext="S_{\mathtt{train}}\leftarrow S_{\mathtt{train}}\cup S_{j}^{m}" display="inline"><semantics id="alg1.l14.m1.1a"><mrow id="alg1.l14.m1.1.1" xref="alg1.l14.m1.1.1.cmml"><msub id="alg1.l14.m1.1.1.2" xref="alg1.l14.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l14.m1.1.1.2.2" xref="alg1.l14.m1.1.1.2.2.cmml">S</mi><mi mathsize="90%" id="alg1.l14.m1.1.1.2.3" xref="alg1.l14.m1.1.1.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" stretchy="false" id="alg1.l14.m1.1.1.1" xref="alg1.l14.m1.1.1.1.cmml">←</mo><mrow id="alg1.l14.m1.1.1.3" xref="alg1.l14.m1.1.1.3.cmml"><msub id="alg1.l14.m1.1.1.3.2" xref="alg1.l14.m1.1.1.3.2.cmml"><mi mathsize="90%" id="alg1.l14.m1.1.1.3.2.2" xref="alg1.l14.m1.1.1.3.2.2.cmml">S</mi><mi mathsize="90%" id="alg1.l14.m1.1.1.3.2.3" xref="alg1.l14.m1.1.1.3.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" id="alg1.l14.m1.1.1.3.1" xref="alg1.l14.m1.1.1.3.1.cmml">∪</mo><msubsup id="alg1.l14.m1.1.1.3.3" xref="alg1.l14.m1.1.1.3.3.cmml"><mi mathsize="90%" id="alg1.l14.m1.1.1.3.3.2.2" xref="alg1.l14.m1.1.1.3.3.2.2.cmml">S</mi><mi mathsize="90%" id="alg1.l14.m1.1.1.3.3.2.3" xref="alg1.l14.m1.1.1.3.3.2.3.cmml">j</mi><mi mathsize="90%" id="alg1.l14.m1.1.1.3.3.3" xref="alg1.l14.m1.1.1.3.3.3.cmml">m</mi></msubsup></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l14.m1.1b"><apply id="alg1.l14.m1.1.1.cmml" xref="alg1.l14.m1.1.1"><ci id="alg1.l14.m1.1.1.1.cmml" xref="alg1.l14.m1.1.1.1">←</ci><apply id="alg1.l14.m1.1.1.2.cmml" xref="alg1.l14.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l14.m1.1.1.2.1.cmml" xref="alg1.l14.m1.1.1.2">subscript</csymbol><ci id="alg1.l14.m1.1.1.2.2.cmml" xref="alg1.l14.m1.1.1.2.2">𝑆</ci><ci id="alg1.l14.m1.1.1.2.3.cmml" xref="alg1.l14.m1.1.1.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><apply id="alg1.l14.m1.1.1.3.cmml" xref="alg1.l14.m1.1.1.3"><union id="alg1.l14.m1.1.1.3.1.cmml" xref="alg1.l14.m1.1.1.3.1"></union><apply id="alg1.l14.m1.1.1.3.2.cmml" xref="alg1.l14.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg1.l14.m1.1.1.3.2.1.cmml" xref="alg1.l14.m1.1.1.3.2">subscript</csymbol><ci id="alg1.l14.m1.1.1.3.2.2.cmml" xref="alg1.l14.m1.1.1.3.2.2">𝑆</ci><ci id="alg1.l14.m1.1.1.3.2.3.cmml" xref="alg1.l14.m1.1.1.3.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><apply id="alg1.l14.m1.1.1.3.3.cmml" xref="alg1.l14.m1.1.1.3.3"><csymbol cd="ambiguous" id="alg1.l14.m1.1.1.3.3.1.cmml" xref="alg1.l14.m1.1.1.3.3">superscript</csymbol><apply id="alg1.l14.m1.1.1.3.3.2.cmml" xref="alg1.l14.m1.1.1.3.3"><csymbol cd="ambiguous" id="alg1.l14.m1.1.1.3.3.2.1.cmml" xref="alg1.l14.m1.1.1.3.3">subscript</csymbol><ci id="alg1.l14.m1.1.1.3.3.2.2.cmml" xref="alg1.l14.m1.1.1.3.3.2.2">𝑆</ci><ci id="alg1.l14.m1.1.1.3.3.2.3.cmml" xref="alg1.l14.m1.1.1.3.3.2.3">𝑗</ci></apply><ci id="alg1.l14.m1.1.1.3.3.3.cmml" xref="alg1.l14.m1.1.1.3.3.3">𝑚</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l14.m1.1c">S_{\mathtt{train}}\leftarrow S_{\mathtt{train}}\cup S_{j}^{m}</annotation></semantics></math><span id="alg1.l14.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg1.l15" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">15:</span><span id="alg1.l15.1" class="ltx_text" style="font-size:90%;">         </span><math id="alg1.l15.m1.1" class="ltx_Math" alttext="l_{\mathtt{train}}\leftarrow l_{\mathtt{train}}\cup 1" display="inline"><semantics id="alg1.l15.m1.1a"><mrow id="alg1.l15.m1.1.1" xref="alg1.l15.m1.1.1.cmml"><msub id="alg1.l15.m1.1.1.2" xref="alg1.l15.m1.1.1.2.cmml"><mi mathsize="90%" id="alg1.l15.m1.1.1.2.2" xref="alg1.l15.m1.1.1.2.2.cmml">l</mi><mi mathsize="90%" id="alg1.l15.m1.1.1.2.3" xref="alg1.l15.m1.1.1.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" stretchy="false" id="alg1.l15.m1.1.1.1" xref="alg1.l15.m1.1.1.1.cmml">←</mo><mrow id="alg1.l15.m1.1.1.3" xref="alg1.l15.m1.1.1.3.cmml"><msub id="alg1.l15.m1.1.1.3.2" xref="alg1.l15.m1.1.1.3.2.cmml"><mi mathsize="90%" id="alg1.l15.m1.1.1.3.2.2" xref="alg1.l15.m1.1.1.3.2.2.cmml">l</mi><mi mathsize="90%" id="alg1.l15.m1.1.1.3.2.3" xref="alg1.l15.m1.1.1.3.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" id="alg1.l15.m1.1.1.3.1" xref="alg1.l15.m1.1.1.3.1.cmml">∪</mo><mn mathsize="90%" id="alg1.l15.m1.1.1.3.3" xref="alg1.l15.m1.1.1.3.3.cmml">1</mn></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l15.m1.1b"><apply id="alg1.l15.m1.1.1.cmml" xref="alg1.l15.m1.1.1"><ci id="alg1.l15.m1.1.1.1.cmml" xref="alg1.l15.m1.1.1.1">←</ci><apply id="alg1.l15.m1.1.1.2.cmml" xref="alg1.l15.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l15.m1.1.1.2.1.cmml" xref="alg1.l15.m1.1.1.2">subscript</csymbol><ci id="alg1.l15.m1.1.1.2.2.cmml" xref="alg1.l15.m1.1.1.2.2">𝑙</ci><ci id="alg1.l15.m1.1.1.2.3.cmml" xref="alg1.l15.m1.1.1.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><apply id="alg1.l15.m1.1.1.3.cmml" xref="alg1.l15.m1.1.1.3"><union id="alg1.l15.m1.1.1.3.1.cmml" xref="alg1.l15.m1.1.1.3.1"></union><apply id="alg1.l15.m1.1.1.3.2.cmml" xref="alg1.l15.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg1.l15.m1.1.1.3.2.1.cmml" xref="alg1.l15.m1.1.1.3.2">subscript</csymbol><ci id="alg1.l15.m1.1.1.3.2.2.cmml" xref="alg1.l15.m1.1.1.3.2.2">𝑙</ci><ci id="alg1.l15.m1.1.1.3.2.3.cmml" xref="alg1.l15.m1.1.1.3.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><cn type="integer" id="alg1.l15.m1.1.1.3.3.cmml" xref="alg1.l15.m1.1.1.3.3">1</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l15.m1.1c">l_{\mathtt{train}}\leftarrow l_{\mathtt{train}}\cup 1</annotation></semantics></math><span id="alg1.l15.2" class="ltx_text" style="font-size:90%;"> 
    </span>
</div>
<div id="alg1.l16" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">16:</span><math id="alg1.l16.m1.2" class="ltx_Math" alttext="MIA_{t}(){\leftarrow}\mathtt{Classifier}(S_{\mathtt{train}},l_{\mathtt{train}})" display="inline"><semantics id="alg1.l16.m1.2a"><mrow id="alg1.l16.m1.2.2" xref="alg1.l16.m1.2.2.cmml"><mrow id="alg1.l16.m1.2.2.4" xref="alg1.l16.m1.2.2.4.cmml"><mi mathsize="90%" id="alg1.l16.m1.2.2.4.2" xref="alg1.l16.m1.2.2.4.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="alg1.l16.m1.2.2.4.1" xref="alg1.l16.m1.2.2.4.1.cmml">​</mo><mi mathsize="90%" id="alg1.l16.m1.2.2.4.3" xref="alg1.l16.m1.2.2.4.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="alg1.l16.m1.2.2.4.1a" xref="alg1.l16.m1.2.2.4.1.cmml">​</mo><msub id="alg1.l16.m1.2.2.4.4" xref="alg1.l16.m1.2.2.4.4.cmml"><mi mathsize="90%" id="alg1.l16.m1.2.2.4.4.2" xref="alg1.l16.m1.2.2.4.4.2.cmml">A</mi><mi mathsize="90%" id="alg1.l16.m1.2.2.4.4.3" xref="alg1.l16.m1.2.2.4.4.3.cmml">t</mi></msub><mo lspace="0em" rspace="0em" id="alg1.l16.m1.2.2.4.1b" xref="alg1.l16.m1.2.2.4.1.cmml">​</mo><mrow id="alg1.l16.m1.2.2.4.5.2" xref="alg1.l16.m1.2.2.4.cmml"><mo maxsize="90%" minsize="90%" id="alg1.l16.m1.2.2.4.5.2.1" xref="alg1.l16.m1.2.2.4.5.1.cmml">(</mo><mo maxsize="90%" minsize="90%" id="alg1.l16.m1.2.2.4.5.2.2" xref="alg1.l16.m1.2.2.4.5.1.cmml">)</mo></mrow></mrow><mo mathsize="90%" stretchy="false" id="alg1.l16.m1.2.2.3" xref="alg1.l16.m1.2.2.3.cmml">←</mo><mrow id="alg1.l16.m1.2.2.2" xref="alg1.l16.m1.2.2.2.cmml"><mi mathsize="90%" id="alg1.l16.m1.2.2.2.4" xref="alg1.l16.m1.2.2.2.4.cmml">𝙲𝚕𝚊𝚜𝚜𝚒𝚏𝚒𝚎𝚛</mi><mo lspace="0em" rspace="0em" id="alg1.l16.m1.2.2.2.3" xref="alg1.l16.m1.2.2.2.3.cmml">​</mo><mrow id="alg1.l16.m1.2.2.2.2.2" xref="alg1.l16.m1.2.2.2.2.3.cmml"><mo maxsize="90%" minsize="90%" id="alg1.l16.m1.2.2.2.2.2.3" xref="alg1.l16.m1.2.2.2.2.3.cmml">(</mo><msub id="alg1.l16.m1.1.1.1.1.1.1" xref="alg1.l16.m1.1.1.1.1.1.1.cmml"><mi mathsize="90%" id="alg1.l16.m1.1.1.1.1.1.1.2" xref="alg1.l16.m1.1.1.1.1.1.1.2.cmml">S</mi><mi mathsize="90%" id="alg1.l16.m1.1.1.1.1.1.1.3" xref="alg1.l16.m1.1.1.1.1.1.1.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo mathsize="90%" id="alg1.l16.m1.2.2.2.2.2.4" xref="alg1.l16.m1.2.2.2.2.3.cmml">,</mo><msub id="alg1.l16.m1.2.2.2.2.2.2" xref="alg1.l16.m1.2.2.2.2.2.2.cmml"><mi mathsize="90%" id="alg1.l16.m1.2.2.2.2.2.2.2" xref="alg1.l16.m1.2.2.2.2.2.2.2.cmml">l</mi><mi mathsize="90%" id="alg1.l16.m1.2.2.2.2.2.2.3" xref="alg1.l16.m1.2.2.2.2.2.2.3.cmml">𝚝𝚛𝚊𝚒𝚗</mi></msub><mo maxsize="90%" minsize="90%" id="alg1.l16.m1.2.2.2.2.2.5" xref="alg1.l16.m1.2.2.2.2.3.cmml">)</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l16.m1.2b"><apply id="alg1.l16.m1.2.2.cmml" xref="alg1.l16.m1.2.2"><ci id="alg1.l16.m1.2.2.3.cmml" xref="alg1.l16.m1.2.2.3">←</ci><apply id="alg1.l16.m1.2.2.4.cmml" xref="alg1.l16.m1.2.2.4"><times id="alg1.l16.m1.2.2.4.1.cmml" xref="alg1.l16.m1.2.2.4.1"></times><ci id="alg1.l16.m1.2.2.4.2.cmml" xref="alg1.l16.m1.2.2.4.2">𝑀</ci><ci id="alg1.l16.m1.2.2.4.3.cmml" xref="alg1.l16.m1.2.2.4.3">𝐼</ci><apply id="alg1.l16.m1.2.2.4.4.cmml" xref="alg1.l16.m1.2.2.4.4"><csymbol cd="ambiguous" id="alg1.l16.m1.2.2.4.4.1.cmml" xref="alg1.l16.m1.2.2.4.4">subscript</csymbol><ci id="alg1.l16.m1.2.2.4.4.2.cmml" xref="alg1.l16.m1.2.2.4.4.2">𝐴</ci><ci id="alg1.l16.m1.2.2.4.4.3.cmml" xref="alg1.l16.m1.2.2.4.4.3">𝑡</ci></apply><list id="alg1.l16.m1.2.2.4.5.1.cmml" xref="alg1.l16.m1.2.2.4.5.2.1"></list></apply><apply id="alg1.l16.m1.2.2.2.cmml" xref="alg1.l16.m1.2.2.2"><times id="alg1.l16.m1.2.2.2.3.cmml" xref="alg1.l16.m1.2.2.2.3"></times><ci id="alg1.l16.m1.2.2.2.4.cmml" xref="alg1.l16.m1.2.2.2.4">𝙲𝚕𝚊𝚜𝚜𝚒𝚏𝚒𝚎𝚛</ci><interval closure="open" id="alg1.l16.m1.2.2.2.2.3.cmml" xref="alg1.l16.m1.2.2.2.2.2"><apply id="alg1.l16.m1.1.1.1.1.1.1.cmml" xref="alg1.l16.m1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="alg1.l16.m1.1.1.1.1.1.1.1.cmml" xref="alg1.l16.m1.1.1.1.1.1.1">subscript</csymbol><ci id="alg1.l16.m1.1.1.1.1.1.1.2.cmml" xref="alg1.l16.m1.1.1.1.1.1.1.2">𝑆</ci><ci id="alg1.l16.m1.1.1.1.1.1.1.3.cmml" xref="alg1.l16.m1.1.1.1.1.1.1.3">𝚝𝚛𝚊𝚒𝚗</ci></apply><apply id="alg1.l16.m1.2.2.2.2.2.2.cmml" xref="alg1.l16.m1.2.2.2.2.2.2"><csymbol cd="ambiguous" id="alg1.l16.m1.2.2.2.2.2.2.1.cmml" xref="alg1.l16.m1.2.2.2.2.2.2">subscript</csymbol><ci id="alg1.l16.m1.2.2.2.2.2.2.2.cmml" xref="alg1.l16.m1.2.2.2.2.2.2.2">𝑙</ci><ci id="alg1.l16.m1.2.2.2.2.2.2.3.cmml" xref="alg1.l16.m1.2.2.2.2.2.2.3">𝚝𝚛𝚊𝚒𝚗</ci></apply></interval></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l16.m1.2c">MIA_{t}(){\leftarrow}\mathtt{Classifier}(S_{\mathtt{train}},l_{\mathtt{train}})</annotation></semantics></math><span id="alg1.l16.1" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
</div>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="alg2" class="ltx_float ltx_figure_panel ltx_minipage ltx_align_center ltx_align_top ltx_framed ltx_framed_top" style="width:216.8pt;">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_float"><span id="alg2.4.1.1" class="ltx_text ltx_font_bold">Algorithm 2</span> </span> <math id="alg2.2.m1.1" class="ltx_Math" alttext="\mathtt{MIAGain}" display="inline"><semantics id="alg2.2.m1.1b"><mi id="alg2.2.m1.1.1" xref="alg2.2.m1.1.1.cmml">𝙼𝙸𝙰𝙶𝚊𝚒𝚗</mi><annotation-xml encoding="MathML-Content" id="alg2.2.m1.1c"><ci id="alg2.2.m1.1.1.cmml" xref="alg2.2.m1.1.1">𝙼𝙸𝙰𝙶𝚊𝚒𝚗</ci></annotation-xml><annotation encoding="application/x-tex" id="alg2.2.m1.1d">\mathtt{MIAGain}</annotation></semantics></math> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">59</span></a>]</cite></figcaption>
<div id="alg2.5" class="ltx_listing ltx_listing">
<div id="alg2.l1" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">1:</span><span id="alg2.l1.1" class="ltx_text" style="font-size:90%;">A generative model GM(), the target record </span><math id="alg2.l1.m1.1" class="ltx_Math" alttext="t" display="inline"><semantics id="alg2.l1.m1.1a"><mi mathsize="90%" id="alg2.l1.m1.1.1" xref="alg2.l1.m1.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="alg2.l1.m1.1b"><ci id="alg2.l1.m1.1.1.cmml" xref="alg2.l1.m1.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="alg2.l1.m1.1c">t</annotation></semantics></math><span id="alg2.l1.2" class="ltx_text" style="font-size:90%;">, the target training set </span><math id="alg2.l1.m2.1" class="ltx_Math" alttext="R^{t}_{out}" display="inline"><semantics id="alg2.l1.m2.1a"><msubsup id="alg2.l1.m2.1.1" xref="alg2.l1.m2.1.1.cmml"><mi mathsize="90%" id="alg2.l1.m2.1.1.2.2" xref="alg2.l1.m2.1.1.2.2.cmml">R</mi><mrow id="alg2.l1.m2.1.1.3" xref="alg2.l1.m2.1.1.3.cmml"><mi mathsize="90%" id="alg2.l1.m2.1.1.3.2" xref="alg2.l1.m2.1.1.3.2.cmml">o</mi><mo lspace="0em" rspace="0em" id="alg2.l1.m2.1.1.3.1" xref="alg2.l1.m2.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l1.m2.1.1.3.3" xref="alg2.l1.m2.1.1.3.3.cmml">u</mi><mo lspace="0em" rspace="0em" id="alg2.l1.m2.1.1.3.1a" xref="alg2.l1.m2.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l1.m2.1.1.3.4" xref="alg2.l1.m2.1.1.3.4.cmml">t</mi></mrow><mi mathsize="90%" id="alg2.l1.m2.1.1.2.3" xref="alg2.l1.m2.1.1.2.3.cmml">t</mi></msubsup><annotation-xml encoding="MathML-Content" id="alg2.l1.m2.1b"><apply id="alg2.l1.m2.1.1.cmml" xref="alg2.l1.m2.1.1"><csymbol cd="ambiguous" id="alg2.l1.m2.1.1.1.cmml" xref="alg2.l1.m2.1.1">subscript</csymbol><apply id="alg2.l1.m2.1.1.2.cmml" xref="alg2.l1.m2.1.1"><csymbol cd="ambiguous" id="alg2.l1.m2.1.1.2.1.cmml" xref="alg2.l1.m2.1.1">superscript</csymbol><ci id="alg2.l1.m2.1.1.2.2.cmml" xref="alg2.l1.m2.1.1.2.2">𝑅</ci><ci id="alg2.l1.m2.1.1.2.3.cmml" xref="alg2.l1.m2.1.1.2.3">𝑡</ci></apply><apply id="alg2.l1.m2.1.1.3.cmml" xref="alg2.l1.m2.1.1.3"><times id="alg2.l1.m2.1.1.3.1.cmml" xref="alg2.l1.m2.1.1.3.1"></times><ci id="alg2.l1.m2.1.1.3.2.cmml" xref="alg2.l1.m2.1.1.3.2">𝑜</ci><ci id="alg2.l1.m2.1.1.3.3.cmml" xref="alg2.l1.m2.1.1.3.3">𝑢</ci><ci id="alg2.l1.m2.1.1.3.4.cmml" xref="alg2.l1.m2.1.1.3.4">𝑡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l1.m2.1c">R^{t}_{out}</annotation></semantics></math><span id="alg2.l1.3" class="ltx_text" style="font-size:90%;"> of size </span><math id="alg2.l1.m3.1" class="ltx_Math" alttext="n" display="inline"><semantics id="alg2.l1.m3.1a"><mi mathsize="90%" id="alg2.l1.m3.1.1" xref="alg2.l1.m3.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="alg2.l1.m3.1b"><ci id="alg2.l1.m3.1.1.cmml" xref="alg2.l1.m3.1.1">𝑛</ci></annotation-xml><annotation encoding="application/x-tex" id="alg2.l1.m3.1c">n</annotation></semantics></math><span id="alg2.l1.4" class="ltx_text" style="font-size:90%;">, the size </span><math id="alg2.l1.m4.1" class="ltx_Math" alttext="m" display="inline"><semantics id="alg2.l1.m4.1a"><mi mathsize="90%" id="alg2.l1.m4.1.1" xref="alg2.l1.m4.1.1.cmml">m</mi><annotation-xml encoding="MathML-Content" id="alg2.l1.m4.1b"><ci id="alg2.l1.m4.1.1.cmml" xref="alg2.l1.m4.1.1">𝑚</ci></annotation-xml><annotation encoding="application/x-tex" id="alg2.l1.m4.1c">m</annotation></semantics></math><span id="alg2.l1.5" class="ltx_text" style="font-size:90%;"> of the synthetic dataset, the number </span><math id="alg2.l1.m5.1" class="ltx_Math" alttext="n_{s}" display="inline"><semantics id="alg2.l1.m5.1a"><msub id="alg2.l1.m5.1.1" xref="alg2.l1.m5.1.1.cmml"><mi mathsize="90%" id="alg2.l1.m5.1.1.2" xref="alg2.l1.m5.1.1.2.cmml">n</mi><mi mathsize="90%" id="alg2.l1.m5.1.1.3" xref="alg2.l1.m5.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="alg2.l1.m5.1b"><apply id="alg2.l1.m5.1.1.cmml" xref="alg2.l1.m5.1.1"><csymbol cd="ambiguous" id="alg2.l1.m5.1.1.1.cmml" xref="alg2.l1.m5.1.1">subscript</csymbol><ci id="alg2.l1.m5.1.1.2.cmml" xref="alg2.l1.m5.1.1.2">𝑛</ci><ci id="alg2.l1.m5.1.1.3.cmml" xref="alg2.l1.m5.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l1.m5.1c">n_{s}</annotation></semantics></math><span id="alg2.l1.6" class="ltx_text" style="font-size:90%;"> of synthetic test sets, a reference dataset </span><math id="alg2.l1.m6.1" class="ltx_Math" alttext="R^{a}" display="inline"><semantics id="alg2.l1.m6.1a"><msup id="alg2.l1.m6.1.1" xref="alg2.l1.m6.1.1.cmml"><mi mathsize="90%" id="alg2.l1.m6.1.1.2" xref="alg2.l1.m6.1.1.2.cmml">R</mi><mi mathsize="90%" id="alg2.l1.m6.1.1.3" xref="alg2.l1.m6.1.1.3.cmml">a</mi></msup><annotation-xml encoding="MathML-Content" id="alg2.l1.m6.1b"><apply id="alg2.l1.m6.1.1.cmml" xref="alg2.l1.m6.1.1"><csymbol cd="ambiguous" id="alg2.l1.m6.1.1.1.cmml" xref="alg2.l1.m6.1.1">superscript</csymbol><ci id="alg2.l1.m6.1.1.2.cmml" xref="alg2.l1.m6.1.1.2">𝑅</ci><ci id="alg2.l1.m6.1.1.3.cmml" xref="alg2.l1.m6.1.1.3">𝑎</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l1.m6.1c">R^{a}</annotation></semantics></math><span id="alg2.l1.7" class="ltx_text" style="font-size:90%;">, the number </span><math id="alg2.l1.m7.1" class="ltx_Math" alttext="k" display="inline"><semantics id="alg2.l1.m7.1a"><mi mathsize="90%" id="alg2.l1.m7.1.1" xref="alg2.l1.m7.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="alg2.l1.m7.1b"><ci id="alg2.l1.m7.1.1.cmml" xref="alg2.l1.m7.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="alg2.l1.m7.1c">k</annotation></semantics></math><span id="alg2.l1.8" class="ltx_text" style="font-size:90%;"> of shadow models.
</span>
</div>
<div id="alg2.l2" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">2:</span><math id="alg2.l2.m1.1" class="ltx_Math" alttext="PG_{t}" display="inline"><semantics id="alg2.l2.m1.1a"><mrow id="alg2.l2.m1.1.1" xref="alg2.l2.m1.1.1.cmml"><mi mathsize="90%" id="alg2.l2.m1.1.1.2" xref="alg2.l2.m1.1.1.2.cmml">P</mi><mo lspace="0em" rspace="0em" id="alg2.l2.m1.1.1.1" xref="alg2.l2.m1.1.1.1.cmml">​</mo><msub id="alg2.l2.m1.1.1.3" xref="alg2.l2.m1.1.1.3.cmml"><mi mathsize="90%" id="alg2.l2.m1.1.1.3.2" xref="alg2.l2.m1.1.1.3.2.cmml">G</mi><mi mathsize="90%" id="alg2.l2.m1.1.1.3.3" xref="alg2.l2.m1.1.1.3.3.cmml">t</mi></msub></mrow><annotation-xml encoding="MathML-Content" id="alg2.l2.m1.1b"><apply id="alg2.l2.m1.1.1.cmml" xref="alg2.l2.m1.1.1"><times id="alg2.l2.m1.1.1.1.cmml" xref="alg2.l2.m1.1.1.1"></times><ci id="alg2.l2.m1.1.1.2.cmml" xref="alg2.l2.m1.1.1.2">𝑃</ci><apply id="alg2.l2.m1.1.1.3.cmml" xref="alg2.l2.m1.1.1.3"><csymbol cd="ambiguous" id="alg2.l2.m1.1.1.3.1.cmml" xref="alg2.l2.m1.1.1.3">subscript</csymbol><ci id="alg2.l2.m1.1.1.3.2.cmml" xref="alg2.l2.m1.1.1.3.2">𝐺</ci><ci id="alg2.l2.m1.1.1.3.3.cmml" xref="alg2.l2.m1.1.1.3.3">𝑡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l2.m1.1c">PG_{t}</annotation></semantics></math><span id="alg2.l2.1" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg2.l3" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">3:</span><math id="alg2.l3.m1.1" class="ltx_Math" alttext="f_{out}\sim" display="inline"><semantics id="alg2.l3.m1.1a"><mrow id="alg2.l3.m1.1.1" xref="alg2.l3.m1.1.1.cmml"><msub id="alg2.l3.m1.1.1.2" xref="alg2.l3.m1.1.1.2.cmml"><mi mathsize="90%" id="alg2.l3.m1.1.1.2.2" xref="alg2.l3.m1.1.1.2.2.cmml">f</mi><mrow id="alg2.l3.m1.1.1.2.3" xref="alg2.l3.m1.1.1.2.3.cmml"><mi mathsize="90%" id="alg2.l3.m1.1.1.2.3.2" xref="alg2.l3.m1.1.1.2.3.2.cmml">o</mi><mo lspace="0em" rspace="0em" id="alg2.l3.m1.1.1.2.3.1" xref="alg2.l3.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l3.m1.1.1.2.3.3" xref="alg2.l3.m1.1.1.2.3.3.cmml">u</mi><mo lspace="0em" rspace="0em" id="alg2.l3.m1.1.1.2.3.1a" xref="alg2.l3.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l3.m1.1.1.2.3.4" xref="alg2.l3.m1.1.1.2.3.4.cmml">t</mi></mrow></msub><mo mathsize="90%" id="alg2.l3.m1.1.1.1" xref="alg2.l3.m1.1.1.1.cmml">∼</mo><mi id="alg2.l3.m1.1.1.3" xref="alg2.l3.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg2.l3.m1.1b"><apply id="alg2.l3.m1.1.1.cmml" xref="alg2.l3.m1.1.1"><csymbol cd="latexml" id="alg2.l3.m1.1.1.1.cmml" xref="alg2.l3.m1.1.1.1">similar-to</csymbol><apply id="alg2.l3.m1.1.1.2.cmml" xref="alg2.l3.m1.1.1.2"><csymbol cd="ambiguous" id="alg2.l3.m1.1.1.2.1.cmml" xref="alg2.l3.m1.1.1.2">subscript</csymbol><ci id="alg2.l3.m1.1.1.2.2.cmml" xref="alg2.l3.m1.1.1.2.2">𝑓</ci><apply id="alg2.l3.m1.1.1.2.3.cmml" xref="alg2.l3.m1.1.1.2.3"><times id="alg2.l3.m1.1.1.2.3.1.cmml" xref="alg2.l3.m1.1.1.2.3.1"></times><ci id="alg2.l3.m1.1.1.2.3.2.cmml" xref="alg2.l3.m1.1.1.2.3.2">𝑜</ci><ci id="alg2.l3.m1.1.1.2.3.3.cmml" xref="alg2.l3.m1.1.1.2.3.3">𝑢</ci><ci id="alg2.l3.m1.1.1.2.3.4.cmml" xref="alg2.l3.m1.1.1.2.3.4">𝑡</ci></apply></apply><csymbol cd="latexml" id="alg2.l3.m1.1.1.3.cmml" xref="alg2.l3.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l3.m1.1c">f_{out}\sim</annotation></semantics></math><span id="alg2.l3.1" class="ltx_text" style="font-size:90%;"> GM(</span><math id="alg2.l3.m2.1" class="ltx_Math" alttext="{R^{t}_{out}}" display="inline"><semantics id="alg2.l3.m2.1a"><msubsup id="alg2.l3.m2.1.1" xref="alg2.l3.m2.1.1.cmml"><mi mathsize="90%" id="alg2.l3.m2.1.1.2.2" xref="alg2.l3.m2.1.1.2.2.cmml">R</mi><mrow id="alg2.l3.m2.1.1.3" xref="alg2.l3.m2.1.1.3.cmml"><mi mathsize="90%" id="alg2.l3.m2.1.1.3.2" xref="alg2.l3.m2.1.1.3.2.cmml">o</mi><mo lspace="0em" rspace="0em" id="alg2.l3.m2.1.1.3.1" xref="alg2.l3.m2.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l3.m2.1.1.3.3" xref="alg2.l3.m2.1.1.3.3.cmml">u</mi><mo lspace="0em" rspace="0em" id="alg2.l3.m2.1.1.3.1a" xref="alg2.l3.m2.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l3.m2.1.1.3.4" xref="alg2.l3.m2.1.1.3.4.cmml">t</mi></mrow><mi mathsize="90%" id="alg2.l3.m2.1.1.2.3" xref="alg2.l3.m2.1.1.2.3.cmml">t</mi></msubsup><annotation-xml encoding="MathML-Content" id="alg2.l3.m2.1b"><apply id="alg2.l3.m2.1.1.cmml" xref="alg2.l3.m2.1.1"><csymbol cd="ambiguous" id="alg2.l3.m2.1.1.1.cmml" xref="alg2.l3.m2.1.1">subscript</csymbol><apply id="alg2.l3.m2.1.1.2.cmml" xref="alg2.l3.m2.1.1"><csymbol cd="ambiguous" id="alg2.l3.m2.1.1.2.1.cmml" xref="alg2.l3.m2.1.1">superscript</csymbol><ci id="alg2.l3.m2.1.1.2.2.cmml" xref="alg2.l3.m2.1.1.2.2">𝑅</ci><ci id="alg2.l3.m2.1.1.2.3.cmml" xref="alg2.l3.m2.1.1.2.3">𝑡</ci></apply><apply id="alg2.l3.m2.1.1.3.cmml" xref="alg2.l3.m2.1.1.3"><times id="alg2.l3.m2.1.1.3.1.cmml" xref="alg2.l3.m2.1.1.3.1"></times><ci id="alg2.l3.m2.1.1.3.2.cmml" xref="alg2.l3.m2.1.1.3.2">𝑜</ci><ci id="alg2.l3.m2.1.1.3.3.cmml" xref="alg2.l3.m2.1.1.3.3">𝑢</ci><ci id="alg2.l3.m2.1.1.3.4.cmml" xref="alg2.l3.m2.1.1.3.4">𝑡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l3.m2.1c">{R^{t}_{out}}</annotation></semantics></math><span id="alg2.l3.2" class="ltx_text" style="font-size:90%;">) 
</span>
</div>
<div id="alg2.l4" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">4:</span><span id="alg2.l4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">for</span><span id="alg2.l4.2" class="ltx_text" style="font-size:90%;"> </span><math id="alg2.l4.m1.3" class="ltx_Math" alttext="i=1,\cdots,n_{s}" display="inline"><semantics id="alg2.l4.m1.3a"><mrow id="alg2.l4.m1.3.3" xref="alg2.l4.m1.3.3.cmml"><mi mathsize="90%" id="alg2.l4.m1.3.3.3" xref="alg2.l4.m1.3.3.3.cmml">i</mi><mo mathsize="90%" id="alg2.l4.m1.3.3.2" xref="alg2.l4.m1.3.3.2.cmml">=</mo><mrow id="alg2.l4.m1.3.3.1.1" xref="alg2.l4.m1.3.3.1.2.cmml"><mn mathsize="90%" id="alg2.l4.m1.1.1" xref="alg2.l4.m1.1.1.cmml">1</mn><mo mathsize="90%" id="alg2.l4.m1.3.3.1.1.2" xref="alg2.l4.m1.3.3.1.2.cmml">,</mo><mi mathsize="90%" mathvariant="normal" id="alg2.l4.m1.2.2" xref="alg2.l4.m1.2.2.cmml">⋯</mi><mo mathsize="90%" id="alg2.l4.m1.3.3.1.1.3" xref="alg2.l4.m1.3.3.1.2.cmml">,</mo><msub id="alg2.l4.m1.3.3.1.1.1" xref="alg2.l4.m1.3.3.1.1.1.cmml"><mi mathsize="90%" id="alg2.l4.m1.3.3.1.1.1.2" xref="alg2.l4.m1.3.3.1.1.1.2.cmml">n</mi><mi mathsize="90%" id="alg2.l4.m1.3.3.1.1.1.3" xref="alg2.l4.m1.3.3.1.1.1.3.cmml">s</mi></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg2.l4.m1.3b"><apply id="alg2.l4.m1.3.3.cmml" xref="alg2.l4.m1.3.3"><eq id="alg2.l4.m1.3.3.2.cmml" xref="alg2.l4.m1.3.3.2"></eq><ci id="alg2.l4.m1.3.3.3.cmml" xref="alg2.l4.m1.3.3.3">𝑖</ci><list id="alg2.l4.m1.3.3.1.2.cmml" xref="alg2.l4.m1.3.3.1.1"><cn type="integer" id="alg2.l4.m1.1.1.cmml" xref="alg2.l4.m1.1.1">1</cn><ci id="alg2.l4.m1.2.2.cmml" xref="alg2.l4.m1.2.2">⋯</ci><apply id="alg2.l4.m1.3.3.1.1.1.cmml" xref="alg2.l4.m1.3.3.1.1.1"><csymbol cd="ambiguous" id="alg2.l4.m1.3.3.1.1.1.1.cmml" xref="alg2.l4.m1.3.3.1.1.1">subscript</csymbol><ci id="alg2.l4.m1.3.3.1.1.1.2.cmml" xref="alg2.l4.m1.3.3.1.1.1.2">𝑛</ci><ci id="alg2.l4.m1.3.3.1.1.1.3.cmml" xref="alg2.l4.m1.3.3.1.1.1.3">𝑠</ci></apply></list></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l4.m1.3c">i=1,\cdots,n_{s}</annotation></semantics></math><span id="alg2.l4.3" class="ltx_text" style="font-size:90%;"> </span><span id="alg2.l4.4" class="ltx_text ltx_font_bold" style="font-size:90%;">do</span><span id="alg2.l4.5" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg2.l5" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">5:</span><span id="alg2.l5.1" class="ltx_text" style="font-size:90%;">    </span><math id="alg2.l5.m1.1" class="ltx_Math" alttext="S_{i}\sim f_{out}^{m}" display="inline"><semantics id="alg2.l5.m1.1a"><mrow id="alg2.l5.m1.1.1" xref="alg2.l5.m1.1.1.cmml"><msub id="alg2.l5.m1.1.1.2" xref="alg2.l5.m1.1.1.2.cmml"><mi mathsize="90%" id="alg2.l5.m1.1.1.2.2" xref="alg2.l5.m1.1.1.2.2.cmml">S</mi><mi mathsize="90%" id="alg2.l5.m1.1.1.2.3" xref="alg2.l5.m1.1.1.2.3.cmml">i</mi></msub><mo mathsize="90%" id="alg2.l5.m1.1.1.1" xref="alg2.l5.m1.1.1.1.cmml">∼</mo><msubsup id="alg2.l5.m1.1.1.3" xref="alg2.l5.m1.1.1.3.cmml"><mi mathsize="90%" id="alg2.l5.m1.1.1.3.2.2" xref="alg2.l5.m1.1.1.3.2.2.cmml">f</mi><mrow id="alg2.l5.m1.1.1.3.2.3" xref="alg2.l5.m1.1.1.3.2.3.cmml"><mi mathsize="90%" id="alg2.l5.m1.1.1.3.2.3.2" xref="alg2.l5.m1.1.1.3.2.3.2.cmml">o</mi><mo lspace="0em" rspace="0em" id="alg2.l5.m1.1.1.3.2.3.1" xref="alg2.l5.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l5.m1.1.1.3.2.3.3" xref="alg2.l5.m1.1.1.3.2.3.3.cmml">u</mi><mo lspace="0em" rspace="0em" id="alg2.l5.m1.1.1.3.2.3.1a" xref="alg2.l5.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l5.m1.1.1.3.2.3.4" xref="alg2.l5.m1.1.1.3.2.3.4.cmml">t</mi></mrow><mi mathsize="90%" id="alg2.l5.m1.1.1.3.3" xref="alg2.l5.m1.1.1.3.3.cmml">m</mi></msubsup></mrow><annotation-xml encoding="MathML-Content" id="alg2.l5.m1.1b"><apply id="alg2.l5.m1.1.1.cmml" xref="alg2.l5.m1.1.1"><csymbol cd="latexml" id="alg2.l5.m1.1.1.1.cmml" xref="alg2.l5.m1.1.1.1">similar-to</csymbol><apply id="alg2.l5.m1.1.1.2.cmml" xref="alg2.l5.m1.1.1.2"><csymbol cd="ambiguous" id="alg2.l5.m1.1.1.2.1.cmml" xref="alg2.l5.m1.1.1.2">subscript</csymbol><ci id="alg2.l5.m1.1.1.2.2.cmml" xref="alg2.l5.m1.1.1.2.2">𝑆</ci><ci id="alg2.l5.m1.1.1.2.3.cmml" xref="alg2.l5.m1.1.1.2.3">𝑖</ci></apply><apply id="alg2.l5.m1.1.1.3.cmml" xref="alg2.l5.m1.1.1.3"><csymbol cd="ambiguous" id="alg2.l5.m1.1.1.3.1.cmml" xref="alg2.l5.m1.1.1.3">superscript</csymbol><apply id="alg2.l5.m1.1.1.3.2.cmml" xref="alg2.l5.m1.1.1.3"><csymbol cd="ambiguous" id="alg2.l5.m1.1.1.3.2.1.cmml" xref="alg2.l5.m1.1.1.3">subscript</csymbol><ci id="alg2.l5.m1.1.1.3.2.2.cmml" xref="alg2.l5.m1.1.1.3.2.2">𝑓</ci><apply id="alg2.l5.m1.1.1.3.2.3.cmml" xref="alg2.l5.m1.1.1.3.2.3"><times id="alg2.l5.m1.1.1.3.2.3.1.cmml" xref="alg2.l5.m1.1.1.3.2.3.1"></times><ci id="alg2.l5.m1.1.1.3.2.3.2.cmml" xref="alg2.l5.m1.1.1.3.2.3.2">𝑜</ci><ci id="alg2.l5.m1.1.1.3.2.3.3.cmml" xref="alg2.l5.m1.1.1.3.2.3.3">𝑢</ci><ci id="alg2.l5.m1.1.1.3.2.3.4.cmml" xref="alg2.l5.m1.1.1.3.2.3.4">𝑡</ci></apply></apply><ci id="alg2.l5.m1.1.1.3.3.cmml" xref="alg2.l5.m1.1.1.3.3">𝑚</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l5.m1.1c">S_{i}\sim f_{out}^{m}</annotation></semantics></math><span id="alg2.l5.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg2.l6" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">6:</span><span id="alg2.l6.1" class="ltx_text" style="font-size:90%;">    </span><math id="alg2.l6.m1.1" class="ltx_Math" alttext="S_{test}\leftarrow S_{test}\cup S_{i}" display="inline"><semantics id="alg2.l6.m1.1a"><mrow id="alg2.l6.m1.1.1" xref="alg2.l6.m1.1.1.cmml"><msub id="alg2.l6.m1.1.1.2" xref="alg2.l6.m1.1.1.2.cmml"><mi mathsize="90%" id="alg2.l6.m1.1.1.2.2" xref="alg2.l6.m1.1.1.2.2.cmml">S</mi><mrow id="alg2.l6.m1.1.1.2.3" xref="alg2.l6.m1.1.1.2.3.cmml"><mi mathsize="90%" id="alg2.l6.m1.1.1.2.3.2" xref="alg2.l6.m1.1.1.2.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="alg2.l6.m1.1.1.2.3.1" xref="alg2.l6.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l6.m1.1.1.2.3.3" xref="alg2.l6.m1.1.1.2.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="alg2.l6.m1.1.1.2.3.1a" xref="alg2.l6.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l6.m1.1.1.2.3.4" xref="alg2.l6.m1.1.1.2.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="alg2.l6.m1.1.1.2.3.1b" xref="alg2.l6.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l6.m1.1.1.2.3.5" xref="alg2.l6.m1.1.1.2.3.5.cmml">t</mi></mrow></msub><mo mathsize="90%" stretchy="false" id="alg2.l6.m1.1.1.1" xref="alg2.l6.m1.1.1.1.cmml">←</mo><mrow id="alg2.l6.m1.1.1.3" xref="alg2.l6.m1.1.1.3.cmml"><msub id="alg2.l6.m1.1.1.3.2" xref="alg2.l6.m1.1.1.3.2.cmml"><mi mathsize="90%" id="alg2.l6.m1.1.1.3.2.2" xref="alg2.l6.m1.1.1.3.2.2.cmml">S</mi><mrow id="alg2.l6.m1.1.1.3.2.3" xref="alg2.l6.m1.1.1.3.2.3.cmml"><mi mathsize="90%" id="alg2.l6.m1.1.1.3.2.3.2" xref="alg2.l6.m1.1.1.3.2.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="alg2.l6.m1.1.1.3.2.3.1" xref="alg2.l6.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l6.m1.1.1.3.2.3.3" xref="alg2.l6.m1.1.1.3.2.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="alg2.l6.m1.1.1.3.2.3.1a" xref="alg2.l6.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l6.m1.1.1.3.2.3.4" xref="alg2.l6.m1.1.1.3.2.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="alg2.l6.m1.1.1.3.2.3.1b" xref="alg2.l6.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l6.m1.1.1.3.2.3.5" xref="alg2.l6.m1.1.1.3.2.3.5.cmml">t</mi></mrow></msub><mo mathsize="90%" id="alg2.l6.m1.1.1.3.1" xref="alg2.l6.m1.1.1.3.1.cmml">∪</mo><msub id="alg2.l6.m1.1.1.3.3" xref="alg2.l6.m1.1.1.3.3.cmml"><mi mathsize="90%" id="alg2.l6.m1.1.1.3.3.2" xref="alg2.l6.m1.1.1.3.3.2.cmml">S</mi><mi mathsize="90%" id="alg2.l6.m1.1.1.3.3.3" xref="alg2.l6.m1.1.1.3.3.3.cmml">i</mi></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg2.l6.m1.1b"><apply id="alg2.l6.m1.1.1.cmml" xref="alg2.l6.m1.1.1"><ci id="alg2.l6.m1.1.1.1.cmml" xref="alg2.l6.m1.1.1.1">←</ci><apply id="alg2.l6.m1.1.1.2.cmml" xref="alg2.l6.m1.1.1.2"><csymbol cd="ambiguous" id="alg2.l6.m1.1.1.2.1.cmml" xref="alg2.l6.m1.1.1.2">subscript</csymbol><ci id="alg2.l6.m1.1.1.2.2.cmml" xref="alg2.l6.m1.1.1.2.2">𝑆</ci><apply id="alg2.l6.m1.1.1.2.3.cmml" xref="alg2.l6.m1.1.1.2.3"><times id="alg2.l6.m1.1.1.2.3.1.cmml" xref="alg2.l6.m1.1.1.2.3.1"></times><ci id="alg2.l6.m1.1.1.2.3.2.cmml" xref="alg2.l6.m1.1.1.2.3.2">𝑡</ci><ci id="alg2.l6.m1.1.1.2.3.3.cmml" xref="alg2.l6.m1.1.1.2.3.3">𝑒</ci><ci id="alg2.l6.m1.1.1.2.3.4.cmml" xref="alg2.l6.m1.1.1.2.3.4">𝑠</ci><ci id="alg2.l6.m1.1.1.2.3.5.cmml" xref="alg2.l6.m1.1.1.2.3.5">𝑡</ci></apply></apply><apply id="alg2.l6.m1.1.1.3.cmml" xref="alg2.l6.m1.1.1.3"><union id="alg2.l6.m1.1.1.3.1.cmml" xref="alg2.l6.m1.1.1.3.1"></union><apply id="alg2.l6.m1.1.1.3.2.cmml" xref="alg2.l6.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg2.l6.m1.1.1.3.2.1.cmml" xref="alg2.l6.m1.1.1.3.2">subscript</csymbol><ci id="alg2.l6.m1.1.1.3.2.2.cmml" xref="alg2.l6.m1.1.1.3.2.2">𝑆</ci><apply id="alg2.l6.m1.1.1.3.2.3.cmml" xref="alg2.l6.m1.1.1.3.2.3"><times id="alg2.l6.m1.1.1.3.2.3.1.cmml" xref="alg2.l6.m1.1.1.3.2.3.1"></times><ci id="alg2.l6.m1.1.1.3.2.3.2.cmml" xref="alg2.l6.m1.1.1.3.2.3.2">𝑡</ci><ci id="alg2.l6.m1.1.1.3.2.3.3.cmml" xref="alg2.l6.m1.1.1.3.2.3.3">𝑒</ci><ci id="alg2.l6.m1.1.1.3.2.3.4.cmml" xref="alg2.l6.m1.1.1.3.2.3.4">𝑠</ci><ci id="alg2.l6.m1.1.1.3.2.3.5.cmml" xref="alg2.l6.m1.1.1.3.2.3.5">𝑡</ci></apply></apply><apply id="alg2.l6.m1.1.1.3.3.cmml" xref="alg2.l6.m1.1.1.3.3"><csymbol cd="ambiguous" id="alg2.l6.m1.1.1.3.3.1.cmml" xref="alg2.l6.m1.1.1.3.3">subscript</csymbol><ci id="alg2.l6.m1.1.1.3.3.2.cmml" xref="alg2.l6.m1.1.1.3.3.2">𝑆</ci><ci id="alg2.l6.m1.1.1.3.3.3.cmml" xref="alg2.l6.m1.1.1.3.3.3">𝑖</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l6.m1.1c">S_{test}\leftarrow S_{test}\cup S_{i}</annotation></semantics></math><span id="alg2.l6.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg2.l7" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">7:</span><math id="alg2.l7.m1.1" class="ltx_Math" alttext="R^{t}_{in}\leftarrow R^{t}_{out}\cup t" display="inline"><semantics id="alg2.l7.m1.1a"><mrow id="alg2.l7.m1.1.1" xref="alg2.l7.m1.1.1.cmml"><msubsup id="alg2.l7.m1.1.1.2" xref="alg2.l7.m1.1.1.2.cmml"><mi mathsize="90%" id="alg2.l7.m1.1.1.2.2.2" xref="alg2.l7.m1.1.1.2.2.2.cmml">R</mi><mrow id="alg2.l7.m1.1.1.2.3" xref="alg2.l7.m1.1.1.2.3.cmml"><mi mathsize="90%" id="alg2.l7.m1.1.1.2.3.2" xref="alg2.l7.m1.1.1.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="alg2.l7.m1.1.1.2.3.1" xref="alg2.l7.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l7.m1.1.1.2.3.3" xref="alg2.l7.m1.1.1.2.3.3.cmml">n</mi></mrow><mi mathsize="90%" id="alg2.l7.m1.1.1.2.2.3" xref="alg2.l7.m1.1.1.2.2.3.cmml">t</mi></msubsup><mo mathsize="90%" stretchy="false" id="alg2.l7.m1.1.1.1" xref="alg2.l7.m1.1.1.1.cmml">←</mo><mrow id="alg2.l7.m1.1.1.3" xref="alg2.l7.m1.1.1.3.cmml"><msubsup id="alg2.l7.m1.1.1.3.2" xref="alg2.l7.m1.1.1.3.2.cmml"><mi mathsize="90%" id="alg2.l7.m1.1.1.3.2.2.2" xref="alg2.l7.m1.1.1.3.2.2.2.cmml">R</mi><mrow id="alg2.l7.m1.1.1.3.2.3" xref="alg2.l7.m1.1.1.3.2.3.cmml"><mi mathsize="90%" id="alg2.l7.m1.1.1.3.2.3.2" xref="alg2.l7.m1.1.1.3.2.3.2.cmml">o</mi><mo lspace="0em" rspace="0em" id="alg2.l7.m1.1.1.3.2.3.1" xref="alg2.l7.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l7.m1.1.1.3.2.3.3" xref="alg2.l7.m1.1.1.3.2.3.3.cmml">u</mi><mo lspace="0em" rspace="0em" id="alg2.l7.m1.1.1.3.2.3.1a" xref="alg2.l7.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l7.m1.1.1.3.2.3.4" xref="alg2.l7.m1.1.1.3.2.3.4.cmml">t</mi></mrow><mi mathsize="90%" id="alg2.l7.m1.1.1.3.2.2.3" xref="alg2.l7.m1.1.1.3.2.2.3.cmml">t</mi></msubsup><mo mathsize="90%" id="alg2.l7.m1.1.1.3.1" xref="alg2.l7.m1.1.1.3.1.cmml">∪</mo><mi mathsize="90%" id="alg2.l7.m1.1.1.3.3" xref="alg2.l7.m1.1.1.3.3.cmml">t</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg2.l7.m1.1b"><apply id="alg2.l7.m1.1.1.cmml" xref="alg2.l7.m1.1.1"><ci id="alg2.l7.m1.1.1.1.cmml" xref="alg2.l7.m1.1.1.1">←</ci><apply id="alg2.l7.m1.1.1.2.cmml" xref="alg2.l7.m1.1.1.2"><csymbol cd="ambiguous" id="alg2.l7.m1.1.1.2.1.cmml" xref="alg2.l7.m1.1.1.2">subscript</csymbol><apply id="alg2.l7.m1.1.1.2.2.cmml" xref="alg2.l7.m1.1.1.2"><csymbol cd="ambiguous" id="alg2.l7.m1.1.1.2.2.1.cmml" xref="alg2.l7.m1.1.1.2">superscript</csymbol><ci id="alg2.l7.m1.1.1.2.2.2.cmml" xref="alg2.l7.m1.1.1.2.2.2">𝑅</ci><ci id="alg2.l7.m1.1.1.2.2.3.cmml" xref="alg2.l7.m1.1.1.2.2.3">𝑡</ci></apply><apply id="alg2.l7.m1.1.1.2.3.cmml" xref="alg2.l7.m1.1.1.2.3"><times id="alg2.l7.m1.1.1.2.3.1.cmml" xref="alg2.l7.m1.1.1.2.3.1"></times><ci id="alg2.l7.m1.1.1.2.3.2.cmml" xref="alg2.l7.m1.1.1.2.3.2">𝑖</ci><ci id="alg2.l7.m1.1.1.2.3.3.cmml" xref="alg2.l7.m1.1.1.2.3.3">𝑛</ci></apply></apply><apply id="alg2.l7.m1.1.1.3.cmml" xref="alg2.l7.m1.1.1.3"><union id="alg2.l7.m1.1.1.3.1.cmml" xref="alg2.l7.m1.1.1.3.1"></union><apply id="alg2.l7.m1.1.1.3.2.cmml" xref="alg2.l7.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg2.l7.m1.1.1.3.2.1.cmml" xref="alg2.l7.m1.1.1.3.2">subscript</csymbol><apply id="alg2.l7.m1.1.1.3.2.2.cmml" xref="alg2.l7.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg2.l7.m1.1.1.3.2.2.1.cmml" xref="alg2.l7.m1.1.1.3.2">superscript</csymbol><ci id="alg2.l7.m1.1.1.3.2.2.2.cmml" xref="alg2.l7.m1.1.1.3.2.2.2">𝑅</ci><ci id="alg2.l7.m1.1.1.3.2.2.3.cmml" xref="alg2.l7.m1.1.1.3.2.2.3">𝑡</ci></apply><apply id="alg2.l7.m1.1.1.3.2.3.cmml" xref="alg2.l7.m1.1.1.3.2.3"><times id="alg2.l7.m1.1.1.3.2.3.1.cmml" xref="alg2.l7.m1.1.1.3.2.3.1"></times><ci id="alg2.l7.m1.1.1.3.2.3.2.cmml" xref="alg2.l7.m1.1.1.3.2.3.2">𝑜</ci><ci id="alg2.l7.m1.1.1.3.2.3.3.cmml" xref="alg2.l7.m1.1.1.3.2.3.3">𝑢</ci><ci id="alg2.l7.m1.1.1.3.2.3.4.cmml" xref="alg2.l7.m1.1.1.3.2.3.4">𝑡</ci></apply></apply><ci id="alg2.l7.m1.1.1.3.3.cmml" xref="alg2.l7.m1.1.1.3.3">𝑡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l7.m1.1c">R^{t}_{in}\leftarrow R^{t}_{out}\cup t</annotation></semantics></math><span id="alg2.l7.1" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg2.l8" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">8:</span><math id="alg2.l8.m1.1" class="ltx_Math" alttext="f_{in}\sim" display="inline"><semantics id="alg2.l8.m1.1a"><mrow id="alg2.l8.m1.1.1" xref="alg2.l8.m1.1.1.cmml"><msub id="alg2.l8.m1.1.1.2" xref="alg2.l8.m1.1.1.2.cmml"><mi mathsize="90%" id="alg2.l8.m1.1.1.2.2" xref="alg2.l8.m1.1.1.2.2.cmml">f</mi><mrow id="alg2.l8.m1.1.1.2.3" xref="alg2.l8.m1.1.1.2.3.cmml"><mi mathsize="90%" id="alg2.l8.m1.1.1.2.3.2" xref="alg2.l8.m1.1.1.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="alg2.l8.m1.1.1.2.3.1" xref="alg2.l8.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l8.m1.1.1.2.3.3" xref="alg2.l8.m1.1.1.2.3.3.cmml">n</mi></mrow></msub><mo mathsize="90%" id="alg2.l8.m1.1.1.1" xref="alg2.l8.m1.1.1.1.cmml">∼</mo><mi id="alg2.l8.m1.1.1.3" xref="alg2.l8.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg2.l8.m1.1b"><apply id="alg2.l8.m1.1.1.cmml" xref="alg2.l8.m1.1.1"><csymbol cd="latexml" id="alg2.l8.m1.1.1.1.cmml" xref="alg2.l8.m1.1.1.1">similar-to</csymbol><apply id="alg2.l8.m1.1.1.2.cmml" xref="alg2.l8.m1.1.1.2"><csymbol cd="ambiguous" id="alg2.l8.m1.1.1.2.1.cmml" xref="alg2.l8.m1.1.1.2">subscript</csymbol><ci id="alg2.l8.m1.1.1.2.2.cmml" xref="alg2.l8.m1.1.1.2.2">𝑓</ci><apply id="alg2.l8.m1.1.1.2.3.cmml" xref="alg2.l8.m1.1.1.2.3"><times id="alg2.l8.m1.1.1.2.3.1.cmml" xref="alg2.l8.m1.1.1.2.3.1"></times><ci id="alg2.l8.m1.1.1.2.3.2.cmml" xref="alg2.l8.m1.1.1.2.3.2">𝑖</ci><ci id="alg2.l8.m1.1.1.2.3.3.cmml" xref="alg2.l8.m1.1.1.2.3.3">𝑛</ci></apply></apply><csymbol cd="latexml" id="alg2.l8.m1.1.1.3.cmml" xref="alg2.l8.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l8.m1.1c">f_{in}\sim</annotation></semantics></math><span id="alg2.l8.1" class="ltx_text" style="font-size:90%;"> GM(</span><math id="alg2.l8.m2.1" class="ltx_Math" alttext="{R^{t}_{in}}" display="inline"><semantics id="alg2.l8.m2.1a"><msubsup id="alg2.l8.m2.1.1" xref="alg2.l8.m2.1.1.cmml"><mi mathsize="90%" id="alg2.l8.m2.1.1.2.2" xref="alg2.l8.m2.1.1.2.2.cmml">R</mi><mrow id="alg2.l8.m2.1.1.3" xref="alg2.l8.m2.1.1.3.cmml"><mi mathsize="90%" id="alg2.l8.m2.1.1.3.2" xref="alg2.l8.m2.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="alg2.l8.m2.1.1.3.1" xref="alg2.l8.m2.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l8.m2.1.1.3.3" xref="alg2.l8.m2.1.1.3.3.cmml">n</mi></mrow><mi mathsize="90%" id="alg2.l8.m2.1.1.2.3" xref="alg2.l8.m2.1.1.2.3.cmml">t</mi></msubsup><annotation-xml encoding="MathML-Content" id="alg2.l8.m2.1b"><apply id="alg2.l8.m2.1.1.cmml" xref="alg2.l8.m2.1.1"><csymbol cd="ambiguous" id="alg2.l8.m2.1.1.1.cmml" xref="alg2.l8.m2.1.1">subscript</csymbol><apply id="alg2.l8.m2.1.1.2.cmml" xref="alg2.l8.m2.1.1"><csymbol cd="ambiguous" id="alg2.l8.m2.1.1.2.1.cmml" xref="alg2.l8.m2.1.1">superscript</csymbol><ci id="alg2.l8.m2.1.1.2.2.cmml" xref="alg2.l8.m2.1.1.2.2">𝑅</ci><ci id="alg2.l8.m2.1.1.2.3.cmml" xref="alg2.l8.m2.1.1.2.3">𝑡</ci></apply><apply id="alg2.l8.m2.1.1.3.cmml" xref="alg2.l8.m2.1.1.3"><times id="alg2.l8.m2.1.1.3.1.cmml" xref="alg2.l8.m2.1.1.3.1"></times><ci id="alg2.l8.m2.1.1.3.2.cmml" xref="alg2.l8.m2.1.1.3.2">𝑖</ci><ci id="alg2.l8.m2.1.1.3.3.cmml" xref="alg2.l8.m2.1.1.3.3">𝑛</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l8.m2.1c">{R^{t}_{in}}</annotation></semantics></math><span id="alg2.l8.2" class="ltx_text" style="font-size:90%;">) 
</span>
</div>
<div id="alg2.l9" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">9:</span><span id="alg2.l9.1" class="ltx_text ltx_font_bold" style="font-size:90%;">for</span><span id="alg2.l9.2" class="ltx_text" style="font-size:90%;"> </span><math id="alg2.l9.m1.3" class="ltx_Math" alttext="i=1,\cdots,n_{s}" display="inline"><semantics id="alg2.l9.m1.3a"><mrow id="alg2.l9.m1.3.3" xref="alg2.l9.m1.3.3.cmml"><mi mathsize="90%" id="alg2.l9.m1.3.3.3" xref="alg2.l9.m1.3.3.3.cmml">i</mi><mo mathsize="90%" id="alg2.l9.m1.3.3.2" xref="alg2.l9.m1.3.3.2.cmml">=</mo><mrow id="alg2.l9.m1.3.3.1.1" xref="alg2.l9.m1.3.3.1.2.cmml"><mn mathsize="90%" id="alg2.l9.m1.1.1" xref="alg2.l9.m1.1.1.cmml">1</mn><mo mathsize="90%" id="alg2.l9.m1.3.3.1.1.2" xref="alg2.l9.m1.3.3.1.2.cmml">,</mo><mi mathsize="90%" mathvariant="normal" id="alg2.l9.m1.2.2" xref="alg2.l9.m1.2.2.cmml">⋯</mi><mo mathsize="90%" id="alg2.l9.m1.3.3.1.1.3" xref="alg2.l9.m1.3.3.1.2.cmml">,</mo><msub id="alg2.l9.m1.3.3.1.1.1" xref="alg2.l9.m1.3.3.1.1.1.cmml"><mi mathsize="90%" id="alg2.l9.m1.3.3.1.1.1.2" xref="alg2.l9.m1.3.3.1.1.1.2.cmml">n</mi><mi mathsize="90%" id="alg2.l9.m1.3.3.1.1.1.3" xref="alg2.l9.m1.3.3.1.1.1.3.cmml">s</mi></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg2.l9.m1.3b"><apply id="alg2.l9.m1.3.3.cmml" xref="alg2.l9.m1.3.3"><eq id="alg2.l9.m1.3.3.2.cmml" xref="alg2.l9.m1.3.3.2"></eq><ci id="alg2.l9.m1.3.3.3.cmml" xref="alg2.l9.m1.3.3.3">𝑖</ci><list id="alg2.l9.m1.3.3.1.2.cmml" xref="alg2.l9.m1.3.3.1.1"><cn type="integer" id="alg2.l9.m1.1.1.cmml" xref="alg2.l9.m1.1.1">1</cn><ci id="alg2.l9.m1.2.2.cmml" xref="alg2.l9.m1.2.2">⋯</ci><apply id="alg2.l9.m1.3.3.1.1.1.cmml" xref="alg2.l9.m1.3.3.1.1.1"><csymbol cd="ambiguous" id="alg2.l9.m1.3.3.1.1.1.1.cmml" xref="alg2.l9.m1.3.3.1.1.1">subscript</csymbol><ci id="alg2.l9.m1.3.3.1.1.1.2.cmml" xref="alg2.l9.m1.3.3.1.1.1.2">𝑛</ci><ci id="alg2.l9.m1.3.3.1.1.1.3.cmml" xref="alg2.l9.m1.3.3.1.1.1.3">𝑠</ci></apply></list></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l9.m1.3c">i=1,\cdots,n_{s}</annotation></semantics></math><span id="alg2.l9.3" class="ltx_text" style="font-size:90%;"> </span><span id="alg2.l9.4" class="ltx_text ltx_font_bold" style="font-size:90%;">do</span><span id="alg2.l9.5" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg2.l10" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">10:</span><span id="alg2.l10.1" class="ltx_text" style="font-size:90%;">    </span><math id="alg2.l10.m1.1" class="ltx_Math" alttext="S_{i}\sim f_{in}^{m}" display="inline"><semantics id="alg2.l10.m1.1a"><mrow id="alg2.l10.m1.1.1" xref="alg2.l10.m1.1.1.cmml"><msub id="alg2.l10.m1.1.1.2" xref="alg2.l10.m1.1.1.2.cmml"><mi mathsize="90%" id="alg2.l10.m1.1.1.2.2" xref="alg2.l10.m1.1.1.2.2.cmml">S</mi><mi mathsize="90%" id="alg2.l10.m1.1.1.2.3" xref="alg2.l10.m1.1.1.2.3.cmml">i</mi></msub><mo mathsize="90%" id="alg2.l10.m1.1.1.1" xref="alg2.l10.m1.1.1.1.cmml">∼</mo><msubsup id="alg2.l10.m1.1.1.3" xref="alg2.l10.m1.1.1.3.cmml"><mi mathsize="90%" id="alg2.l10.m1.1.1.3.2.2" xref="alg2.l10.m1.1.1.3.2.2.cmml">f</mi><mrow id="alg2.l10.m1.1.1.3.2.3" xref="alg2.l10.m1.1.1.3.2.3.cmml"><mi mathsize="90%" id="alg2.l10.m1.1.1.3.2.3.2" xref="alg2.l10.m1.1.1.3.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="alg2.l10.m1.1.1.3.2.3.1" xref="alg2.l10.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l10.m1.1.1.3.2.3.3" xref="alg2.l10.m1.1.1.3.2.3.3.cmml">n</mi></mrow><mi mathsize="90%" id="alg2.l10.m1.1.1.3.3" xref="alg2.l10.m1.1.1.3.3.cmml">m</mi></msubsup></mrow><annotation-xml encoding="MathML-Content" id="alg2.l10.m1.1b"><apply id="alg2.l10.m1.1.1.cmml" xref="alg2.l10.m1.1.1"><csymbol cd="latexml" id="alg2.l10.m1.1.1.1.cmml" xref="alg2.l10.m1.1.1.1">similar-to</csymbol><apply id="alg2.l10.m1.1.1.2.cmml" xref="alg2.l10.m1.1.1.2"><csymbol cd="ambiguous" id="alg2.l10.m1.1.1.2.1.cmml" xref="alg2.l10.m1.1.1.2">subscript</csymbol><ci id="alg2.l10.m1.1.1.2.2.cmml" xref="alg2.l10.m1.1.1.2.2">𝑆</ci><ci id="alg2.l10.m1.1.1.2.3.cmml" xref="alg2.l10.m1.1.1.2.3">𝑖</ci></apply><apply id="alg2.l10.m1.1.1.3.cmml" xref="alg2.l10.m1.1.1.3"><csymbol cd="ambiguous" id="alg2.l10.m1.1.1.3.1.cmml" xref="alg2.l10.m1.1.1.3">superscript</csymbol><apply id="alg2.l10.m1.1.1.3.2.cmml" xref="alg2.l10.m1.1.1.3"><csymbol cd="ambiguous" id="alg2.l10.m1.1.1.3.2.1.cmml" xref="alg2.l10.m1.1.1.3">subscript</csymbol><ci id="alg2.l10.m1.1.1.3.2.2.cmml" xref="alg2.l10.m1.1.1.3.2.2">𝑓</ci><apply id="alg2.l10.m1.1.1.3.2.3.cmml" xref="alg2.l10.m1.1.1.3.2.3"><times id="alg2.l10.m1.1.1.3.2.3.1.cmml" xref="alg2.l10.m1.1.1.3.2.3.1"></times><ci id="alg2.l10.m1.1.1.3.2.3.2.cmml" xref="alg2.l10.m1.1.1.3.2.3.2">𝑖</ci><ci id="alg2.l10.m1.1.1.3.2.3.3.cmml" xref="alg2.l10.m1.1.1.3.2.3.3">𝑛</ci></apply></apply><ci id="alg2.l10.m1.1.1.3.3.cmml" xref="alg2.l10.m1.1.1.3.3">𝑚</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l10.m1.1c">S_{i}\sim f_{in}^{m}</annotation></semantics></math><span id="alg2.l10.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg2.l11" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">11:</span><span id="alg2.l11.1" class="ltx_text" style="font-size:90%;">    </span><math id="alg2.l11.m1.1" class="ltx_Math" alttext="S_{test}\leftarrow S_{test}\cup S_{i}" display="inline"><semantics id="alg2.l11.m1.1a"><mrow id="alg2.l11.m1.1.1" xref="alg2.l11.m1.1.1.cmml"><msub id="alg2.l11.m1.1.1.2" xref="alg2.l11.m1.1.1.2.cmml"><mi mathsize="90%" id="alg2.l11.m1.1.1.2.2" xref="alg2.l11.m1.1.1.2.2.cmml">S</mi><mrow id="alg2.l11.m1.1.1.2.3" xref="alg2.l11.m1.1.1.2.3.cmml"><mi mathsize="90%" id="alg2.l11.m1.1.1.2.3.2" xref="alg2.l11.m1.1.1.2.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="alg2.l11.m1.1.1.2.3.1" xref="alg2.l11.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l11.m1.1.1.2.3.3" xref="alg2.l11.m1.1.1.2.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="alg2.l11.m1.1.1.2.3.1a" xref="alg2.l11.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l11.m1.1.1.2.3.4" xref="alg2.l11.m1.1.1.2.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="alg2.l11.m1.1.1.2.3.1b" xref="alg2.l11.m1.1.1.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l11.m1.1.1.2.3.5" xref="alg2.l11.m1.1.1.2.3.5.cmml">t</mi></mrow></msub><mo mathsize="90%" stretchy="false" id="alg2.l11.m1.1.1.1" xref="alg2.l11.m1.1.1.1.cmml">←</mo><mrow id="alg2.l11.m1.1.1.3" xref="alg2.l11.m1.1.1.3.cmml"><msub id="alg2.l11.m1.1.1.3.2" xref="alg2.l11.m1.1.1.3.2.cmml"><mi mathsize="90%" id="alg2.l11.m1.1.1.3.2.2" xref="alg2.l11.m1.1.1.3.2.2.cmml">S</mi><mrow id="alg2.l11.m1.1.1.3.2.3" xref="alg2.l11.m1.1.1.3.2.3.cmml"><mi mathsize="90%" id="alg2.l11.m1.1.1.3.2.3.2" xref="alg2.l11.m1.1.1.3.2.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="alg2.l11.m1.1.1.3.2.3.1" xref="alg2.l11.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l11.m1.1.1.3.2.3.3" xref="alg2.l11.m1.1.1.3.2.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="alg2.l11.m1.1.1.3.2.3.1a" xref="alg2.l11.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l11.m1.1.1.3.2.3.4" xref="alg2.l11.m1.1.1.3.2.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="alg2.l11.m1.1.1.3.2.3.1b" xref="alg2.l11.m1.1.1.3.2.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l11.m1.1.1.3.2.3.5" xref="alg2.l11.m1.1.1.3.2.3.5.cmml">t</mi></mrow></msub><mo mathsize="90%" id="alg2.l11.m1.1.1.3.1" xref="alg2.l11.m1.1.1.3.1.cmml">∪</mo><msub id="alg2.l11.m1.1.1.3.3" xref="alg2.l11.m1.1.1.3.3.cmml"><mi mathsize="90%" id="alg2.l11.m1.1.1.3.3.2" xref="alg2.l11.m1.1.1.3.3.2.cmml">S</mi><mi mathsize="90%" id="alg2.l11.m1.1.1.3.3.3" xref="alg2.l11.m1.1.1.3.3.3.cmml">i</mi></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg2.l11.m1.1b"><apply id="alg2.l11.m1.1.1.cmml" xref="alg2.l11.m1.1.1"><ci id="alg2.l11.m1.1.1.1.cmml" xref="alg2.l11.m1.1.1.1">←</ci><apply id="alg2.l11.m1.1.1.2.cmml" xref="alg2.l11.m1.1.1.2"><csymbol cd="ambiguous" id="alg2.l11.m1.1.1.2.1.cmml" xref="alg2.l11.m1.1.1.2">subscript</csymbol><ci id="alg2.l11.m1.1.1.2.2.cmml" xref="alg2.l11.m1.1.1.2.2">𝑆</ci><apply id="alg2.l11.m1.1.1.2.3.cmml" xref="alg2.l11.m1.1.1.2.3"><times id="alg2.l11.m1.1.1.2.3.1.cmml" xref="alg2.l11.m1.1.1.2.3.1"></times><ci id="alg2.l11.m1.1.1.2.3.2.cmml" xref="alg2.l11.m1.1.1.2.3.2">𝑡</ci><ci id="alg2.l11.m1.1.1.2.3.3.cmml" xref="alg2.l11.m1.1.1.2.3.3">𝑒</ci><ci id="alg2.l11.m1.1.1.2.3.4.cmml" xref="alg2.l11.m1.1.1.2.3.4">𝑠</ci><ci id="alg2.l11.m1.1.1.2.3.5.cmml" xref="alg2.l11.m1.1.1.2.3.5">𝑡</ci></apply></apply><apply id="alg2.l11.m1.1.1.3.cmml" xref="alg2.l11.m1.1.1.3"><union id="alg2.l11.m1.1.1.3.1.cmml" xref="alg2.l11.m1.1.1.3.1"></union><apply id="alg2.l11.m1.1.1.3.2.cmml" xref="alg2.l11.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg2.l11.m1.1.1.3.2.1.cmml" xref="alg2.l11.m1.1.1.3.2">subscript</csymbol><ci id="alg2.l11.m1.1.1.3.2.2.cmml" xref="alg2.l11.m1.1.1.3.2.2">𝑆</ci><apply id="alg2.l11.m1.1.1.3.2.3.cmml" xref="alg2.l11.m1.1.1.3.2.3"><times id="alg2.l11.m1.1.1.3.2.3.1.cmml" xref="alg2.l11.m1.1.1.3.2.3.1"></times><ci id="alg2.l11.m1.1.1.3.2.3.2.cmml" xref="alg2.l11.m1.1.1.3.2.3.2">𝑡</ci><ci id="alg2.l11.m1.1.1.3.2.3.3.cmml" xref="alg2.l11.m1.1.1.3.2.3.3">𝑒</ci><ci id="alg2.l11.m1.1.1.3.2.3.4.cmml" xref="alg2.l11.m1.1.1.3.2.3.4">𝑠</ci><ci id="alg2.l11.m1.1.1.3.2.3.5.cmml" xref="alg2.l11.m1.1.1.3.2.3.5">𝑡</ci></apply></apply><apply id="alg2.l11.m1.1.1.3.3.cmml" xref="alg2.l11.m1.1.1.3.3"><csymbol cd="ambiguous" id="alg2.l11.m1.1.1.3.3.1.cmml" xref="alg2.l11.m1.1.1.3.3">subscript</csymbol><ci id="alg2.l11.m1.1.1.3.3.2.cmml" xref="alg2.l11.m1.1.1.3.3.2">𝑆</ci><ci id="alg2.l11.m1.1.1.3.3.3.cmml" xref="alg2.l11.m1.1.1.3.3.3">𝑖</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l11.m1.1c">S_{test}\leftarrow S_{test}\cup S_{i}</annotation></semantics></math><span id="alg2.l11.2" class="ltx_text" style="font-size:90%;"> 
</span>
</div>
<div id="alg2.l12" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">12:</span><math id="alg2.l12.m1.1" class="ltx_math_unparsed" alttext="MIA_{t}()\leftarrow\mathtt{MIATrain}(" display="inline"><semantics id="alg2.l12.m1.1a"><mrow id="alg2.l12.m1.1b"><mi mathsize="90%" id="alg2.l12.m1.1.1">M</mi><mi mathsize="90%" id="alg2.l12.m1.1.2">I</mi><msub id="alg2.l12.m1.1.3"><mi mathsize="90%" id="alg2.l12.m1.1.3.2">A</mi><mi mathsize="90%" id="alg2.l12.m1.1.3.3">t</mi></msub><mrow id="alg2.l12.m1.1.4"><mo maxsize="90%" minsize="90%" id="alg2.l12.m1.1.4.1">(</mo><mo maxsize="90%" minsize="90%" id="alg2.l12.m1.1.4.2">)</mo></mrow><mo mathsize="90%" stretchy="false" id="alg2.l12.m1.1.5">←</mo><mi mathsize="90%" id="alg2.l12.m1.1.6">𝙼𝙸𝙰𝚃𝚛𝚊𝚒𝚗</mi><mo maxsize="90%" minsize="90%" id="alg2.l12.m1.1.7">(</mo></mrow><annotation encoding="application/x-tex" id="alg2.l12.m1.1c">MIA_{t}()\leftarrow\mathtt{MIATrain}(</annotation></semantics></math><span id="alg2.l12.1" class="ltx_text" style="font-size:90%;">GM()</span><math id="alg2.l12.m2.1" class="ltx_math_unparsed" alttext=",t,R^{a},n,m,n_{s},k)" display="inline"><semantics id="alg2.l12.m2.1a"><mrow id="alg2.l12.m2.1b"><mo mathsize="90%" id="alg2.l12.m2.1.1">,</mo><mi mathsize="90%" id="alg2.l12.m2.1.2">t</mi><mo mathsize="90%" id="alg2.l12.m2.1.3">,</mo><msup id="alg2.l12.m2.1.4"><mi mathsize="90%" id="alg2.l12.m2.1.4.2">R</mi><mi mathsize="90%" id="alg2.l12.m2.1.4.3">a</mi></msup><mo mathsize="90%" id="alg2.l12.m2.1.5">,</mo><mi mathsize="90%" id="alg2.l12.m2.1.6">n</mi><mo mathsize="90%" id="alg2.l12.m2.1.7">,</mo><mi mathsize="90%" id="alg2.l12.m2.1.8">m</mi><mo mathsize="90%" id="alg2.l12.m2.1.9">,</mo><msub id="alg2.l12.m2.1.10"><mi mathsize="90%" id="alg2.l12.m2.1.10.2">n</mi><mi mathsize="90%" id="alg2.l12.m2.1.10.3">s</mi></msub><mo mathsize="90%" id="alg2.l12.m2.1.11">,</mo><mi mathsize="90%" id="alg2.l12.m2.1.12">k</mi><mo maxsize="90%" minsize="90%" id="alg2.l12.m2.1.13">)</mo></mrow><annotation encoding="application/x-tex" id="alg2.l12.m2.1c">,t,R^{a},n,m,n_{s},k)</annotation></semantics></math><span id="alg2.l12.2" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg2.l13" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">13:</span><math id="alg2.l13.m1.3" class="ltx_Math" alttext="\overline{MIA_{t}}(S_{test})=\sum_{S_{i}\in S_{test}}\Pr[MIA_{t}(S_{i})=1]/2n_{s}" display="inline"><semantics id="alg2.l13.m1.3a"><mrow id="alg2.l13.m1.3.3" xref="alg2.l13.m1.3.3.cmml"><mrow id="alg2.l13.m1.2.2.1" xref="alg2.l13.m1.2.2.1.cmml"><mover accent="true" id="alg2.l13.m1.2.2.1.3" xref="alg2.l13.m1.2.2.1.3.cmml"><mrow id="alg2.l13.m1.2.2.1.3.2" xref="alg2.l13.m1.2.2.1.3.2.cmml"><mi mathsize="90%" id="alg2.l13.m1.2.2.1.3.2.2" xref="alg2.l13.m1.2.2.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.2.2.1.3.2.1" xref="alg2.l13.m1.2.2.1.3.2.1.cmml">​</mo><mi mathsize="90%" id="alg2.l13.m1.2.2.1.3.2.3" xref="alg2.l13.m1.2.2.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.2.2.1.3.2.1a" xref="alg2.l13.m1.2.2.1.3.2.1.cmml">​</mo><msub id="alg2.l13.m1.2.2.1.3.2.4" xref="alg2.l13.m1.2.2.1.3.2.4.cmml"><mi mathsize="90%" id="alg2.l13.m1.2.2.1.3.2.4.2" xref="alg2.l13.m1.2.2.1.3.2.4.2.cmml">A</mi><mi mathsize="90%" id="alg2.l13.m1.2.2.1.3.2.4.3" xref="alg2.l13.m1.2.2.1.3.2.4.3.cmml">t</mi></msub></mrow><mo mathsize="90%" id="alg2.l13.m1.2.2.1.3.1" xref="alg2.l13.m1.2.2.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="alg2.l13.m1.2.2.1.2" xref="alg2.l13.m1.2.2.1.2.cmml">​</mo><mrow id="alg2.l13.m1.2.2.1.1.1" xref="alg2.l13.m1.2.2.1.1.1.1.cmml"><mo maxsize="90%" minsize="90%" id="alg2.l13.m1.2.2.1.1.1.2" xref="alg2.l13.m1.2.2.1.1.1.1.cmml">(</mo><msub id="alg2.l13.m1.2.2.1.1.1.1" xref="alg2.l13.m1.2.2.1.1.1.1.cmml"><mi mathsize="90%" id="alg2.l13.m1.2.2.1.1.1.1.2" xref="alg2.l13.m1.2.2.1.1.1.1.2.cmml">S</mi><mrow id="alg2.l13.m1.2.2.1.1.1.1.3" xref="alg2.l13.m1.2.2.1.1.1.1.3.cmml"><mi mathsize="90%" id="alg2.l13.m1.2.2.1.1.1.1.3.2" xref="alg2.l13.m1.2.2.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.2.2.1.1.1.1.3.1" xref="alg2.l13.m1.2.2.1.1.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l13.m1.2.2.1.1.1.1.3.3" xref="alg2.l13.m1.2.2.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.2.2.1.1.1.1.3.1a" xref="alg2.l13.m1.2.2.1.1.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l13.m1.2.2.1.1.1.1.3.4" xref="alg2.l13.m1.2.2.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.2.2.1.1.1.1.3.1b" xref="alg2.l13.m1.2.2.1.1.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l13.m1.2.2.1.1.1.1.3.5" xref="alg2.l13.m1.2.2.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo maxsize="90%" minsize="90%" id="alg2.l13.m1.2.2.1.1.1.3" xref="alg2.l13.m1.2.2.1.1.1.1.cmml">)</mo></mrow></mrow><mo mathsize="90%" rspace="0.111em" id="alg2.l13.m1.3.3.3" xref="alg2.l13.m1.3.3.3.cmml">=</mo><mrow id="alg2.l13.m1.3.3.2" xref="alg2.l13.m1.3.3.2.cmml"><msub id="alg2.l13.m1.3.3.2.2" xref="alg2.l13.m1.3.3.2.2.cmml"><mo maxsize="90%" minsize="90%" stretchy="true" id="alg2.l13.m1.3.3.2.2.2" xref="alg2.l13.m1.3.3.2.2.2.cmml">∑</mo><mrow id="alg2.l13.m1.3.3.2.2.3" xref="alg2.l13.m1.3.3.2.2.3.cmml"><msub id="alg2.l13.m1.3.3.2.2.3.2" xref="alg2.l13.m1.3.3.2.2.3.2.cmml"><mi mathsize="90%" id="alg2.l13.m1.3.3.2.2.3.2.2" xref="alg2.l13.m1.3.3.2.2.3.2.2.cmml">S</mi><mi mathsize="90%" id="alg2.l13.m1.3.3.2.2.3.2.3" xref="alg2.l13.m1.3.3.2.2.3.2.3.cmml">i</mi></msub><mo mathsize="90%" id="alg2.l13.m1.3.3.2.2.3.1" xref="alg2.l13.m1.3.3.2.2.3.1.cmml">∈</mo><msub id="alg2.l13.m1.3.3.2.2.3.3" xref="alg2.l13.m1.3.3.2.2.3.3.cmml"><mi mathsize="90%" id="alg2.l13.m1.3.3.2.2.3.3.2" xref="alg2.l13.m1.3.3.2.2.3.3.2.cmml">S</mi><mrow id="alg2.l13.m1.3.3.2.2.3.3.3" xref="alg2.l13.m1.3.3.2.2.3.3.3.cmml"><mi mathsize="90%" id="alg2.l13.m1.3.3.2.2.3.3.3.2" xref="alg2.l13.m1.3.3.2.2.3.3.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.3.3.2.2.3.3.3.1" xref="alg2.l13.m1.3.3.2.2.3.3.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l13.m1.3.3.2.2.3.3.3.3" xref="alg2.l13.m1.3.3.2.2.3.3.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.3.3.2.2.3.3.3.1a" xref="alg2.l13.m1.3.3.2.2.3.3.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l13.m1.3.3.2.2.3.3.3.4" xref="alg2.l13.m1.3.3.2.2.3.3.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.3.3.2.2.3.3.3.1b" xref="alg2.l13.m1.3.3.2.2.3.3.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l13.m1.3.3.2.2.3.3.3.5" xref="alg2.l13.m1.3.3.2.2.3.3.3.5.cmml">t</mi></mrow></msub></mrow></msub><mrow id="alg2.l13.m1.3.3.2.1" xref="alg2.l13.m1.3.3.2.1.cmml"><mrow id="alg2.l13.m1.3.3.2.1.1" xref="alg2.l13.m1.3.3.2.1.1.cmml"><mrow id="alg2.l13.m1.3.3.2.1.1.1.1" xref="alg2.l13.m1.3.3.2.1.1.1.2.cmml"><mi mathsize="90%" id="alg2.l13.m1.1.1" xref="alg2.l13.m1.1.1.cmml">Pr</mi><mo id="alg2.l13.m1.3.3.2.1.1.1.1a" xref="alg2.l13.m1.3.3.2.1.1.1.2.cmml">⁡</mo><mrow id="alg2.l13.m1.3.3.2.1.1.1.1.1" xref="alg2.l13.m1.3.3.2.1.1.1.2.cmml"><mo maxsize="90%" minsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.2" xref="alg2.l13.m1.3.3.2.1.1.1.2.cmml">[</mo><mrow id="alg2.l13.m1.3.3.2.1.1.1.1.1.1" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.cmml"><mrow id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.cmml"><mi mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.3" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.3.cmml">M</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.2" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.2.cmml">​</mo><mi mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.4" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.4.cmml">I</mi><mo lspace="0em" rspace="0em" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.2a" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.2.cmml">​</mo><msub id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.cmml"><mi mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.2" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.2.cmml">A</mi><mi mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.3" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.3.cmml">t</mi></msub><mo lspace="0em" rspace="0em" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.2b" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.2.cmml">​</mo><mrow id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.cmml"><mo maxsize="90%" minsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.2" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.cmml">(</mo><msub id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.cmml"><mi mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.2" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.2.cmml">S</mi><mi mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.3" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.3.cmml">i</mi></msub><mo maxsize="90%" minsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.3" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.2" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.2.cmml">=</mo><mn mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.3" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.3.cmml">1</mn></mrow><mo maxsize="90%" minsize="90%" id="alg2.l13.m1.3.3.2.1.1.1.1.1.3" xref="alg2.l13.m1.3.3.2.1.1.1.2.cmml">]</mo></mrow></mrow><mo maxsize="90%" minsize="90%" stretchy="true" symmetric="true" id="alg2.l13.m1.3.3.2.1.1.2" xref="alg2.l13.m1.3.3.2.1.1.2.cmml">/</mo><mn mathsize="90%" id="alg2.l13.m1.3.3.2.1.1.3" xref="alg2.l13.m1.3.3.2.1.1.3.cmml">2</mn></mrow><mo lspace="0em" rspace="0em" id="alg2.l13.m1.3.3.2.1.2" xref="alg2.l13.m1.3.3.2.1.2.cmml">​</mo><msub id="alg2.l13.m1.3.3.2.1.3" xref="alg2.l13.m1.3.3.2.1.3.cmml"><mi mathsize="90%" id="alg2.l13.m1.3.3.2.1.3.2" xref="alg2.l13.m1.3.3.2.1.3.2.cmml">n</mi><mi mathsize="90%" id="alg2.l13.m1.3.3.2.1.3.3" xref="alg2.l13.m1.3.3.2.1.3.3.cmml">s</mi></msub></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg2.l13.m1.3b"><apply id="alg2.l13.m1.3.3.cmml" xref="alg2.l13.m1.3.3"><eq id="alg2.l13.m1.3.3.3.cmml" xref="alg2.l13.m1.3.3.3"></eq><apply id="alg2.l13.m1.2.2.1.cmml" xref="alg2.l13.m1.2.2.1"><times id="alg2.l13.m1.2.2.1.2.cmml" xref="alg2.l13.m1.2.2.1.2"></times><apply id="alg2.l13.m1.2.2.1.3.cmml" xref="alg2.l13.m1.2.2.1.3"><ci id="alg2.l13.m1.2.2.1.3.1.cmml" xref="alg2.l13.m1.2.2.1.3.1">¯</ci><apply id="alg2.l13.m1.2.2.1.3.2.cmml" xref="alg2.l13.m1.2.2.1.3.2"><times id="alg2.l13.m1.2.2.1.3.2.1.cmml" xref="alg2.l13.m1.2.2.1.3.2.1"></times><ci id="alg2.l13.m1.2.2.1.3.2.2.cmml" xref="alg2.l13.m1.2.2.1.3.2.2">𝑀</ci><ci id="alg2.l13.m1.2.2.1.3.2.3.cmml" xref="alg2.l13.m1.2.2.1.3.2.3">𝐼</ci><apply id="alg2.l13.m1.2.2.1.3.2.4.cmml" xref="alg2.l13.m1.2.2.1.3.2.4"><csymbol cd="ambiguous" id="alg2.l13.m1.2.2.1.3.2.4.1.cmml" xref="alg2.l13.m1.2.2.1.3.2.4">subscript</csymbol><ci id="alg2.l13.m1.2.2.1.3.2.4.2.cmml" xref="alg2.l13.m1.2.2.1.3.2.4.2">𝐴</ci><ci id="alg2.l13.m1.2.2.1.3.2.4.3.cmml" xref="alg2.l13.m1.2.2.1.3.2.4.3">𝑡</ci></apply></apply></apply><apply id="alg2.l13.m1.2.2.1.1.1.1.cmml" xref="alg2.l13.m1.2.2.1.1.1"><csymbol cd="ambiguous" id="alg2.l13.m1.2.2.1.1.1.1.1.cmml" xref="alg2.l13.m1.2.2.1.1.1">subscript</csymbol><ci id="alg2.l13.m1.2.2.1.1.1.1.2.cmml" xref="alg2.l13.m1.2.2.1.1.1.1.2">𝑆</ci><apply id="alg2.l13.m1.2.2.1.1.1.1.3.cmml" xref="alg2.l13.m1.2.2.1.1.1.1.3"><times id="alg2.l13.m1.2.2.1.1.1.1.3.1.cmml" xref="alg2.l13.m1.2.2.1.1.1.1.3.1"></times><ci id="alg2.l13.m1.2.2.1.1.1.1.3.2.cmml" xref="alg2.l13.m1.2.2.1.1.1.1.3.2">𝑡</ci><ci id="alg2.l13.m1.2.2.1.1.1.1.3.3.cmml" xref="alg2.l13.m1.2.2.1.1.1.1.3.3">𝑒</ci><ci id="alg2.l13.m1.2.2.1.1.1.1.3.4.cmml" xref="alg2.l13.m1.2.2.1.1.1.1.3.4">𝑠</ci><ci id="alg2.l13.m1.2.2.1.1.1.1.3.5.cmml" xref="alg2.l13.m1.2.2.1.1.1.1.3.5">𝑡</ci></apply></apply></apply><apply id="alg2.l13.m1.3.3.2.cmml" xref="alg2.l13.m1.3.3.2"><apply id="alg2.l13.m1.3.3.2.2.cmml" xref="alg2.l13.m1.3.3.2.2"><csymbol cd="ambiguous" id="alg2.l13.m1.3.3.2.2.1.cmml" xref="alg2.l13.m1.3.3.2.2">subscript</csymbol><sum id="alg2.l13.m1.3.3.2.2.2.cmml" xref="alg2.l13.m1.3.3.2.2.2"></sum><apply id="alg2.l13.m1.3.3.2.2.3.cmml" xref="alg2.l13.m1.3.3.2.2.3"><in id="alg2.l13.m1.3.3.2.2.3.1.cmml" xref="alg2.l13.m1.3.3.2.2.3.1"></in><apply id="alg2.l13.m1.3.3.2.2.3.2.cmml" xref="alg2.l13.m1.3.3.2.2.3.2"><csymbol cd="ambiguous" id="alg2.l13.m1.3.3.2.2.3.2.1.cmml" xref="alg2.l13.m1.3.3.2.2.3.2">subscript</csymbol><ci id="alg2.l13.m1.3.3.2.2.3.2.2.cmml" xref="alg2.l13.m1.3.3.2.2.3.2.2">𝑆</ci><ci id="alg2.l13.m1.3.3.2.2.3.2.3.cmml" xref="alg2.l13.m1.3.3.2.2.3.2.3">𝑖</ci></apply><apply id="alg2.l13.m1.3.3.2.2.3.3.cmml" xref="alg2.l13.m1.3.3.2.2.3.3"><csymbol cd="ambiguous" id="alg2.l13.m1.3.3.2.2.3.3.1.cmml" xref="alg2.l13.m1.3.3.2.2.3.3">subscript</csymbol><ci id="alg2.l13.m1.3.3.2.2.3.3.2.cmml" xref="alg2.l13.m1.3.3.2.2.3.3.2">𝑆</ci><apply id="alg2.l13.m1.3.3.2.2.3.3.3.cmml" xref="alg2.l13.m1.3.3.2.2.3.3.3"><times id="alg2.l13.m1.3.3.2.2.3.3.3.1.cmml" xref="alg2.l13.m1.3.3.2.2.3.3.3.1"></times><ci id="alg2.l13.m1.3.3.2.2.3.3.3.2.cmml" xref="alg2.l13.m1.3.3.2.2.3.3.3.2">𝑡</ci><ci id="alg2.l13.m1.3.3.2.2.3.3.3.3.cmml" xref="alg2.l13.m1.3.3.2.2.3.3.3.3">𝑒</ci><ci id="alg2.l13.m1.3.3.2.2.3.3.3.4.cmml" xref="alg2.l13.m1.3.3.2.2.3.3.3.4">𝑠</ci><ci id="alg2.l13.m1.3.3.2.2.3.3.3.5.cmml" xref="alg2.l13.m1.3.3.2.2.3.3.3.5">𝑡</ci></apply></apply></apply></apply><apply id="alg2.l13.m1.3.3.2.1.cmml" xref="alg2.l13.m1.3.3.2.1"><times id="alg2.l13.m1.3.3.2.1.2.cmml" xref="alg2.l13.m1.3.3.2.1.2"></times><apply id="alg2.l13.m1.3.3.2.1.1.cmml" xref="alg2.l13.m1.3.3.2.1.1"><divide id="alg2.l13.m1.3.3.2.1.1.2.cmml" xref="alg2.l13.m1.3.3.2.1.1.2"></divide><apply id="alg2.l13.m1.3.3.2.1.1.1.2.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1"><ci id="alg2.l13.m1.1.1.cmml" xref="alg2.l13.m1.1.1">Pr</ci><apply id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1"><eq id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.2.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.2"></eq><apply id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1"><times id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.2.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.2"></times><ci id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.3.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.3">𝑀</ci><ci id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.4.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.4">𝐼</ci><apply id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5"><csymbol cd="ambiguous" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.1.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5">subscript</csymbol><ci id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.2.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.2">𝐴</ci><ci id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.3.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.5.3">𝑡</ci></apply><apply id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.2.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.2">𝑆</ci><ci id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.3.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.1.1.1.1.3">𝑖</ci></apply></apply><cn type="integer" id="alg2.l13.m1.3.3.2.1.1.1.1.1.1.3.cmml" xref="alg2.l13.m1.3.3.2.1.1.1.1.1.1.3">1</cn></apply></apply><cn type="integer" id="alg2.l13.m1.3.3.2.1.1.3.cmml" xref="alg2.l13.m1.3.3.2.1.1.3">2</cn></apply><apply id="alg2.l13.m1.3.3.2.1.3.cmml" xref="alg2.l13.m1.3.3.2.1.3"><csymbol cd="ambiguous" id="alg2.l13.m1.3.3.2.1.3.1.cmml" xref="alg2.l13.m1.3.3.2.1.3">subscript</csymbol><ci id="alg2.l13.m1.3.3.2.1.3.2.cmml" xref="alg2.l13.m1.3.3.2.1.3.2">𝑛</ci><ci id="alg2.l13.m1.3.3.2.1.3.3.cmml" xref="alg2.l13.m1.3.3.2.1.3.3">𝑠</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l13.m1.3c">\overline{MIA_{t}}(S_{test})=\sum_{S_{i}\in S_{test}}\Pr[MIA_{t}(S_{i})=1]/2n_{s}</annotation></semantics></math><span id="alg2.l13.1" class="ltx_text" style="font-size:90%;">
</span>
</div>
<div id="alg2.l14" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline">14:</span><math id="alg2.l14.m1.1" class="ltx_Math" alttext="PG_{t}\leftarrow(1-\overline{MIA_{t}}(S_{test}))/2" display="inline"><semantics id="alg2.l14.m1.1a"><mrow id="alg2.l14.m1.1.1" xref="alg2.l14.m1.1.1.cmml"><mrow id="alg2.l14.m1.1.1.3" xref="alg2.l14.m1.1.1.3.cmml"><mi mathsize="90%" id="alg2.l14.m1.1.1.3.2" xref="alg2.l14.m1.1.1.3.2.cmml">P</mi><mo lspace="0em" rspace="0em" id="alg2.l14.m1.1.1.3.1" xref="alg2.l14.m1.1.1.3.1.cmml">​</mo><msub id="alg2.l14.m1.1.1.3.3" xref="alg2.l14.m1.1.1.3.3.cmml"><mi mathsize="90%" id="alg2.l14.m1.1.1.3.3.2" xref="alg2.l14.m1.1.1.3.3.2.cmml">G</mi><mi mathsize="90%" id="alg2.l14.m1.1.1.3.3.3" xref="alg2.l14.m1.1.1.3.3.3.cmml">t</mi></msub></mrow><mo mathsize="90%" stretchy="false" id="alg2.l14.m1.1.1.2" xref="alg2.l14.m1.1.1.2.cmml">←</mo><mrow id="alg2.l14.m1.1.1.1" xref="alg2.l14.m1.1.1.1.cmml"><mrow id="alg2.l14.m1.1.1.1.1.1" xref="alg2.l14.m1.1.1.1.1.1.1.cmml"><mo maxsize="90%" minsize="90%" id="alg2.l14.m1.1.1.1.1.1.2" xref="alg2.l14.m1.1.1.1.1.1.1.cmml">(</mo><mrow id="alg2.l14.m1.1.1.1.1.1.1" xref="alg2.l14.m1.1.1.1.1.1.1.cmml"><mn mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.3" xref="alg2.l14.m1.1.1.1.1.1.1.3.cmml">1</mn><mo mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.2" xref="alg2.l14.m1.1.1.1.1.1.1.2.cmml">−</mo><mrow id="alg2.l14.m1.1.1.1.1.1.1.1" xref="alg2.l14.m1.1.1.1.1.1.1.1.cmml"><mover accent="true" id="alg2.l14.m1.1.1.1.1.1.1.1.3" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.cmml"><mrow id="alg2.l14.m1.1.1.1.1.1.1.1.3.2" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.cmml"><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.2" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.1" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.1.cmml">​</mo><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.3" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.1a" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.1.cmml">​</mo><msub id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.cmml"><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.2" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.2.cmml">A</mi><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.3" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.3.cmml">t</mi></msub></mrow><mo mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.3.1" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="alg2.l14.m1.1.1.1.1.1.1.1.2" xref="alg2.l14.m1.1.1.1.1.1.1.1.2.cmml">​</mo><mrow id="alg2.l14.m1.1.1.1.1.1.1.1.1.1" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.cmml"><mo maxsize="90%" minsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.2" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.cmml">(</mo><msub id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.cmml"><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.2" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.2.cmml">S</mi><mrow id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.cmml"><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.2" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.1" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.3" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.1a" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.4" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.1b" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.1.cmml">​</mo><mi mathsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.5" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo maxsize="90%" minsize="90%" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.3" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow></mrow><mo maxsize="90%" minsize="90%" id="alg2.l14.m1.1.1.1.1.1.3" xref="alg2.l14.m1.1.1.1.1.1.1.cmml">)</mo></mrow><mo maxsize="90%" minsize="90%" stretchy="true" symmetric="true" id="alg2.l14.m1.1.1.1.2" xref="alg2.l14.m1.1.1.1.2.cmml">/</mo><mn mathsize="90%" id="alg2.l14.m1.1.1.1.3" xref="alg2.l14.m1.1.1.1.3.cmml">2</mn></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg2.l14.m1.1b"><apply id="alg2.l14.m1.1.1.cmml" xref="alg2.l14.m1.1.1"><ci id="alg2.l14.m1.1.1.2.cmml" xref="alg2.l14.m1.1.1.2">←</ci><apply id="alg2.l14.m1.1.1.3.cmml" xref="alg2.l14.m1.1.1.3"><times id="alg2.l14.m1.1.1.3.1.cmml" xref="alg2.l14.m1.1.1.3.1"></times><ci id="alg2.l14.m1.1.1.3.2.cmml" xref="alg2.l14.m1.1.1.3.2">𝑃</ci><apply id="alg2.l14.m1.1.1.3.3.cmml" xref="alg2.l14.m1.1.1.3.3"><csymbol cd="ambiguous" id="alg2.l14.m1.1.1.3.3.1.cmml" xref="alg2.l14.m1.1.1.3.3">subscript</csymbol><ci id="alg2.l14.m1.1.1.3.3.2.cmml" xref="alg2.l14.m1.1.1.3.3.2">𝐺</ci><ci id="alg2.l14.m1.1.1.3.3.3.cmml" xref="alg2.l14.m1.1.1.3.3.3">𝑡</ci></apply></apply><apply id="alg2.l14.m1.1.1.1.cmml" xref="alg2.l14.m1.1.1.1"><divide id="alg2.l14.m1.1.1.1.2.cmml" xref="alg2.l14.m1.1.1.1.2"></divide><apply id="alg2.l14.m1.1.1.1.1.1.1.cmml" xref="alg2.l14.m1.1.1.1.1.1"><minus id="alg2.l14.m1.1.1.1.1.1.1.2.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.2"></minus><cn type="integer" id="alg2.l14.m1.1.1.1.1.1.1.3.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.3">1</cn><apply id="alg2.l14.m1.1.1.1.1.1.1.1.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1"><times id="alg2.l14.m1.1.1.1.1.1.1.1.2.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.2"></times><apply id="alg2.l14.m1.1.1.1.1.1.1.1.3.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3"><ci id="alg2.l14.m1.1.1.1.1.1.1.1.3.1.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.1">¯</ci><apply id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2"><times id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.1.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.1"></times><ci id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.2.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.2">𝑀</ci><ci id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.3.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.3">𝐼</ci><apply id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4"><csymbol cd="ambiguous" id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.1.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4">subscript</csymbol><ci id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.2.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.2">𝐴</ci><ci id="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.3.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.3.2.4.3">𝑡</ci></apply></apply></apply><apply id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.2.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.2">𝑆</ci><apply id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3"><times id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.1.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.1"></times><ci id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.2.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.2">𝑡</ci><ci id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.3.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.3">𝑒</ci><ci id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.4.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.4">𝑠</ci><ci id="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.5.cmml" xref="alg2.l14.m1.1.1.1.1.1.1.1.1.1.1.3.5">𝑡</ci></apply></apply></apply></apply><cn type="integer" id="alg2.l14.m1.1.1.1.3.cmml" xref="alg2.l14.m1.1.1.1.3">2</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg2.l14.m1.1c">PG_{t}\leftarrow(1-\overline{MIA_{t}}(S_{test}))/2</annotation></semantics></math><span id="alg2.l14.1" class="ltx_text" style="font-size:90%;">
</span>
</div>
</div>
</figure>
</div>
</div>
</figure>
<div id="S2.SS3.p5" class="ltx_para ltx_noindent">
<p id="S2.SS3.p5.4" class="ltx_p"><span id="S2.SS3.p5.4.1" class="ltx_text ltx_font_bold">PG Values.</span> The PG ranges between 0, when publishing the synthetic dataset leads to the same privacy loss as publishing the real dataset (i.e., <math id="S2.SS3.p5.1.m1.1" class="ltx_Math" alttext="\overline{MIA_{t}}(S_{test})" display="inline"><semantics id="S2.SS3.p5.1.m1.1a"><mrow id="S2.SS3.p5.1.m1.1.1" xref="S2.SS3.p5.1.m1.1.1.cmml"><mover accent="true" id="S2.SS3.p5.1.m1.1.1.3" xref="S2.SS3.p5.1.m1.1.1.3.cmml"><mrow id="S2.SS3.p5.1.m1.1.1.3.2" xref="S2.SS3.p5.1.m1.1.1.3.2.cmml"><mi id="S2.SS3.p5.1.m1.1.1.3.2.2" xref="S2.SS3.p5.1.m1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.1.m1.1.1.3.2.1" xref="S2.SS3.p5.1.m1.1.1.3.2.1.cmml">​</mo><mi id="S2.SS3.p5.1.m1.1.1.3.2.3" xref="S2.SS3.p5.1.m1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.1.m1.1.1.3.2.1a" xref="S2.SS3.p5.1.m1.1.1.3.2.1.cmml">​</mo><msub id="S2.SS3.p5.1.m1.1.1.3.2.4" xref="S2.SS3.p5.1.m1.1.1.3.2.4.cmml"><mi id="S2.SS3.p5.1.m1.1.1.3.2.4.2" xref="S2.SS3.p5.1.m1.1.1.3.2.4.2.cmml">A</mi><mi id="S2.SS3.p5.1.m1.1.1.3.2.4.3" xref="S2.SS3.p5.1.m1.1.1.3.2.4.3.cmml">t</mi></msub></mrow><mo id="S2.SS3.p5.1.m1.1.1.3.1" xref="S2.SS3.p5.1.m1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S2.SS3.p5.1.m1.1.1.2" xref="S2.SS3.p5.1.m1.1.1.2.cmml">​</mo><mrow id="S2.SS3.p5.1.m1.1.1.1.1" xref="S2.SS3.p5.1.m1.1.1.1.1.1.cmml"><mo stretchy="false" id="S2.SS3.p5.1.m1.1.1.1.1.2" xref="S2.SS3.p5.1.m1.1.1.1.1.1.cmml">(</mo><msub id="S2.SS3.p5.1.m1.1.1.1.1.1" xref="S2.SS3.p5.1.m1.1.1.1.1.1.cmml"><mi id="S2.SS3.p5.1.m1.1.1.1.1.1.2" xref="S2.SS3.p5.1.m1.1.1.1.1.1.2.cmml">S</mi><mrow id="S2.SS3.p5.1.m1.1.1.1.1.1.3" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.cmml"><mi id="S2.SS3.p5.1.m1.1.1.1.1.1.3.2" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.1.m1.1.1.1.1.1.3.1" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.1.m1.1.1.1.1.1.3.3" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.1.m1.1.1.1.1.1.3.1a" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.1.m1.1.1.1.1.1.3.4" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.1.m1.1.1.1.1.1.3.1b" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.1.m1.1.1.1.1.1.3.5" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S2.SS3.p5.1.m1.1.1.1.1.3" xref="S2.SS3.p5.1.m1.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS3.p5.1.m1.1b"><apply id="S2.SS3.p5.1.m1.1.1.cmml" xref="S2.SS3.p5.1.m1.1.1"><times id="S2.SS3.p5.1.m1.1.1.2.cmml" xref="S2.SS3.p5.1.m1.1.1.2"></times><apply id="S2.SS3.p5.1.m1.1.1.3.cmml" xref="S2.SS3.p5.1.m1.1.1.3"><ci id="S2.SS3.p5.1.m1.1.1.3.1.cmml" xref="S2.SS3.p5.1.m1.1.1.3.1">¯</ci><apply id="S2.SS3.p5.1.m1.1.1.3.2.cmml" xref="S2.SS3.p5.1.m1.1.1.3.2"><times id="S2.SS3.p5.1.m1.1.1.3.2.1.cmml" xref="S2.SS3.p5.1.m1.1.1.3.2.1"></times><ci id="S2.SS3.p5.1.m1.1.1.3.2.2.cmml" xref="S2.SS3.p5.1.m1.1.1.3.2.2">𝑀</ci><ci id="S2.SS3.p5.1.m1.1.1.3.2.3.cmml" xref="S2.SS3.p5.1.m1.1.1.3.2.3">𝐼</ci><apply id="S2.SS3.p5.1.m1.1.1.3.2.4.cmml" xref="S2.SS3.p5.1.m1.1.1.3.2.4"><csymbol cd="ambiguous" id="S2.SS3.p5.1.m1.1.1.3.2.4.1.cmml" xref="S2.SS3.p5.1.m1.1.1.3.2.4">subscript</csymbol><ci id="S2.SS3.p5.1.m1.1.1.3.2.4.2.cmml" xref="S2.SS3.p5.1.m1.1.1.3.2.4.2">𝐴</ci><ci id="S2.SS3.p5.1.m1.1.1.3.2.4.3.cmml" xref="S2.SS3.p5.1.m1.1.1.3.2.4.3">𝑡</ci></apply></apply></apply><apply id="S2.SS3.p5.1.m1.1.1.1.1.1.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="S2.SS3.p5.1.m1.1.1.1.1.1.1.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1">subscript</csymbol><ci id="S2.SS3.p5.1.m1.1.1.1.1.1.2.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1.1.2">𝑆</ci><apply id="S2.SS3.p5.1.m1.1.1.1.1.1.3.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3"><times id="S2.SS3.p5.1.m1.1.1.1.1.1.3.1.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.1"></times><ci id="S2.SS3.p5.1.m1.1.1.1.1.1.3.2.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.2">𝑡</ci><ci id="S2.SS3.p5.1.m1.1.1.1.1.1.3.3.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.3">𝑒</ci><ci id="S2.SS3.p5.1.m1.1.1.1.1.1.3.4.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.4">𝑠</ci><ci id="S2.SS3.p5.1.m1.1.1.1.1.1.3.5.cmml" xref="S2.SS3.p5.1.m1.1.1.1.1.1.3.5">𝑡</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p5.1.m1.1c">\overline{MIA_{t}}(S_{test})</annotation></semantics></math> = 1) and 0.5, when publishing the synthetic dataset perfectly protects the target from MIA (i.e., <math id="S2.SS3.p5.2.m2.1" class="ltx_Math" alttext="\overline{MIA_{t}}(S_{test})" display="inline"><semantics id="S2.SS3.p5.2.m2.1a"><mrow id="S2.SS3.p5.2.m2.1.1" xref="S2.SS3.p5.2.m2.1.1.cmml"><mover accent="true" id="S2.SS3.p5.2.m2.1.1.3" xref="S2.SS3.p5.2.m2.1.1.3.cmml"><mrow id="S2.SS3.p5.2.m2.1.1.3.2" xref="S2.SS3.p5.2.m2.1.1.3.2.cmml"><mi id="S2.SS3.p5.2.m2.1.1.3.2.2" xref="S2.SS3.p5.2.m2.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.2.m2.1.1.3.2.1" xref="S2.SS3.p5.2.m2.1.1.3.2.1.cmml">​</mo><mi id="S2.SS3.p5.2.m2.1.1.3.2.3" xref="S2.SS3.p5.2.m2.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.2.m2.1.1.3.2.1a" xref="S2.SS3.p5.2.m2.1.1.3.2.1.cmml">​</mo><msub id="S2.SS3.p5.2.m2.1.1.3.2.4" xref="S2.SS3.p5.2.m2.1.1.3.2.4.cmml"><mi id="S2.SS3.p5.2.m2.1.1.3.2.4.2" xref="S2.SS3.p5.2.m2.1.1.3.2.4.2.cmml">A</mi><mi id="S2.SS3.p5.2.m2.1.1.3.2.4.3" xref="S2.SS3.p5.2.m2.1.1.3.2.4.3.cmml">t</mi></msub></mrow><mo id="S2.SS3.p5.2.m2.1.1.3.1" xref="S2.SS3.p5.2.m2.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S2.SS3.p5.2.m2.1.1.2" xref="S2.SS3.p5.2.m2.1.1.2.cmml">​</mo><mrow id="S2.SS3.p5.2.m2.1.1.1.1" xref="S2.SS3.p5.2.m2.1.1.1.1.1.cmml"><mo stretchy="false" id="S2.SS3.p5.2.m2.1.1.1.1.2" xref="S2.SS3.p5.2.m2.1.1.1.1.1.cmml">(</mo><msub id="S2.SS3.p5.2.m2.1.1.1.1.1" xref="S2.SS3.p5.2.m2.1.1.1.1.1.cmml"><mi id="S2.SS3.p5.2.m2.1.1.1.1.1.2" xref="S2.SS3.p5.2.m2.1.1.1.1.1.2.cmml">S</mi><mrow id="S2.SS3.p5.2.m2.1.1.1.1.1.3" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.cmml"><mi id="S2.SS3.p5.2.m2.1.1.1.1.1.3.2" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.2.m2.1.1.1.1.1.3.1" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.2.m2.1.1.1.1.1.3.3" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.2.m2.1.1.1.1.1.3.1a" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.2.m2.1.1.1.1.1.3.4" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.2.m2.1.1.1.1.1.3.1b" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.2.m2.1.1.1.1.1.3.5" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S2.SS3.p5.2.m2.1.1.1.1.3" xref="S2.SS3.p5.2.m2.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS3.p5.2.m2.1b"><apply id="S2.SS3.p5.2.m2.1.1.cmml" xref="S2.SS3.p5.2.m2.1.1"><times id="S2.SS3.p5.2.m2.1.1.2.cmml" xref="S2.SS3.p5.2.m2.1.1.2"></times><apply id="S2.SS3.p5.2.m2.1.1.3.cmml" xref="S2.SS3.p5.2.m2.1.1.3"><ci id="S2.SS3.p5.2.m2.1.1.3.1.cmml" xref="S2.SS3.p5.2.m2.1.1.3.1">¯</ci><apply id="S2.SS3.p5.2.m2.1.1.3.2.cmml" xref="S2.SS3.p5.2.m2.1.1.3.2"><times id="S2.SS3.p5.2.m2.1.1.3.2.1.cmml" xref="S2.SS3.p5.2.m2.1.1.3.2.1"></times><ci id="S2.SS3.p5.2.m2.1.1.3.2.2.cmml" xref="S2.SS3.p5.2.m2.1.1.3.2.2">𝑀</ci><ci id="S2.SS3.p5.2.m2.1.1.3.2.3.cmml" xref="S2.SS3.p5.2.m2.1.1.3.2.3">𝐼</ci><apply id="S2.SS3.p5.2.m2.1.1.3.2.4.cmml" xref="S2.SS3.p5.2.m2.1.1.3.2.4"><csymbol cd="ambiguous" id="S2.SS3.p5.2.m2.1.1.3.2.4.1.cmml" xref="S2.SS3.p5.2.m2.1.1.3.2.4">subscript</csymbol><ci id="S2.SS3.p5.2.m2.1.1.3.2.4.2.cmml" xref="S2.SS3.p5.2.m2.1.1.3.2.4.2">𝐴</ci><ci id="S2.SS3.p5.2.m2.1.1.3.2.4.3.cmml" xref="S2.SS3.p5.2.m2.1.1.3.2.4.3">𝑡</ci></apply></apply></apply><apply id="S2.SS3.p5.2.m2.1.1.1.1.1.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1"><csymbol cd="ambiguous" id="S2.SS3.p5.2.m2.1.1.1.1.1.1.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1">subscript</csymbol><ci id="S2.SS3.p5.2.m2.1.1.1.1.1.2.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1.1.2">𝑆</ci><apply id="S2.SS3.p5.2.m2.1.1.1.1.1.3.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3"><times id="S2.SS3.p5.2.m2.1.1.1.1.1.3.1.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.1"></times><ci id="S2.SS3.p5.2.m2.1.1.1.1.1.3.2.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.2">𝑡</ci><ci id="S2.SS3.p5.2.m2.1.1.1.1.1.3.3.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.3">𝑒</ci><ci id="S2.SS3.p5.2.m2.1.1.1.1.1.3.4.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.4">𝑠</ci><ci id="S2.SS3.p5.2.m2.1.1.1.1.1.3.5.cmml" xref="S2.SS3.p5.2.m2.1.1.1.1.1.3.5">𝑡</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p5.2.m2.1c">\overline{MIA_{t}}(S_{test})</annotation></semantics></math>=0).
This means that <math id="S2.SS3.p5.3.m3.1" class="ltx_Math" alttext="PG=0.25" display="inline"><semantics id="S2.SS3.p5.3.m3.1a"><mrow id="S2.SS3.p5.3.m3.1.1" xref="S2.SS3.p5.3.m3.1.1.cmml"><mrow id="S2.SS3.p5.3.m3.1.1.2" xref="S2.SS3.p5.3.m3.1.1.2.cmml"><mi id="S2.SS3.p5.3.m3.1.1.2.2" xref="S2.SS3.p5.3.m3.1.1.2.2.cmml">P</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.3.m3.1.1.2.1" xref="S2.SS3.p5.3.m3.1.1.2.1.cmml">​</mo><mi id="S2.SS3.p5.3.m3.1.1.2.3" xref="S2.SS3.p5.3.m3.1.1.2.3.cmml">G</mi></mrow><mo id="S2.SS3.p5.3.m3.1.1.1" xref="S2.SS3.p5.3.m3.1.1.1.cmml">=</mo><mn id="S2.SS3.p5.3.m3.1.1.3" xref="S2.SS3.p5.3.m3.1.1.3.cmml">0.25</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.SS3.p5.3.m3.1b"><apply id="S2.SS3.p5.3.m3.1.1.cmml" xref="S2.SS3.p5.3.m3.1.1"><eq id="S2.SS3.p5.3.m3.1.1.1.cmml" xref="S2.SS3.p5.3.m3.1.1.1"></eq><apply id="S2.SS3.p5.3.m3.1.1.2.cmml" xref="S2.SS3.p5.3.m3.1.1.2"><times id="S2.SS3.p5.3.m3.1.1.2.1.cmml" xref="S2.SS3.p5.3.m3.1.1.2.1"></times><ci id="S2.SS3.p5.3.m3.1.1.2.2.cmml" xref="S2.SS3.p5.3.m3.1.1.2.2">𝑃</ci><ci id="S2.SS3.p5.3.m3.1.1.2.3.cmml" xref="S2.SS3.p5.3.m3.1.1.2.3">𝐺</ci></apply><cn type="float" id="S2.SS3.p5.3.m3.1.1.3.cmml" xref="S2.SS3.p5.3.m3.1.1.3">0.25</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p5.3.m3.1c">PG=0.25</annotation></semantics></math> when the probability of the adversary inferring whether or not a target is part of the training set used to generate the synthetic dataset is the same as random guessing (i.e., <math id="S2.SS3.p5.4.m4.1" class="ltx_Math" alttext="\overline{MIA_{t}}(S_{test})=0.5" display="inline"><semantics id="S2.SS3.p5.4.m4.1a"><mrow id="S2.SS3.p5.4.m4.1.1" xref="S2.SS3.p5.4.m4.1.1.cmml"><mrow id="S2.SS3.p5.4.m4.1.1.1" xref="S2.SS3.p5.4.m4.1.1.1.cmml"><mover accent="true" id="S2.SS3.p5.4.m4.1.1.1.3" xref="S2.SS3.p5.4.m4.1.1.1.3.cmml"><mrow id="S2.SS3.p5.4.m4.1.1.1.3.2" xref="S2.SS3.p5.4.m4.1.1.1.3.2.cmml"><mi id="S2.SS3.p5.4.m4.1.1.1.3.2.2" xref="S2.SS3.p5.4.m4.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.4.m4.1.1.1.3.2.1" xref="S2.SS3.p5.4.m4.1.1.1.3.2.1.cmml">​</mo><mi id="S2.SS3.p5.4.m4.1.1.1.3.2.3" xref="S2.SS3.p5.4.m4.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.4.m4.1.1.1.3.2.1a" xref="S2.SS3.p5.4.m4.1.1.1.3.2.1.cmml">​</mo><msub id="S2.SS3.p5.4.m4.1.1.1.3.2.4" xref="S2.SS3.p5.4.m4.1.1.1.3.2.4.cmml"><mi id="S2.SS3.p5.4.m4.1.1.1.3.2.4.2" xref="S2.SS3.p5.4.m4.1.1.1.3.2.4.2.cmml">A</mi><mi id="S2.SS3.p5.4.m4.1.1.1.3.2.4.3" xref="S2.SS3.p5.4.m4.1.1.1.3.2.4.3.cmml">t</mi></msub></mrow><mo id="S2.SS3.p5.4.m4.1.1.1.3.1" xref="S2.SS3.p5.4.m4.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S2.SS3.p5.4.m4.1.1.1.2" xref="S2.SS3.p5.4.m4.1.1.1.2.cmml">​</mo><mrow id="S2.SS3.p5.4.m4.1.1.1.1.1" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S2.SS3.p5.4.m4.1.1.1.1.1.2" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.cmml">(</mo><msub id="S2.SS3.p5.4.m4.1.1.1.1.1.1" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.cmml"><mi id="S2.SS3.p5.4.m4.1.1.1.1.1.1.2" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.2.cmml">S</mi><mrow id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.cmml"><mi id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.2" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.1" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.3" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.1a" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.4" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.1b" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.5" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S2.SS3.p5.4.m4.1.1.1.1.1.3" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S2.SS3.p5.4.m4.1.1.2" xref="S2.SS3.p5.4.m4.1.1.2.cmml">=</mo><mn id="S2.SS3.p5.4.m4.1.1.3" xref="S2.SS3.p5.4.m4.1.1.3.cmml">0.5</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.SS3.p5.4.m4.1b"><apply id="S2.SS3.p5.4.m4.1.1.cmml" xref="S2.SS3.p5.4.m4.1.1"><eq id="S2.SS3.p5.4.m4.1.1.2.cmml" xref="S2.SS3.p5.4.m4.1.1.2"></eq><apply id="S2.SS3.p5.4.m4.1.1.1.cmml" xref="S2.SS3.p5.4.m4.1.1.1"><times id="S2.SS3.p5.4.m4.1.1.1.2.cmml" xref="S2.SS3.p5.4.m4.1.1.1.2"></times><apply id="S2.SS3.p5.4.m4.1.1.1.3.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3"><ci id="S2.SS3.p5.4.m4.1.1.1.3.1.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.1">¯</ci><apply id="S2.SS3.p5.4.m4.1.1.1.3.2.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.2"><times id="S2.SS3.p5.4.m4.1.1.1.3.2.1.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.2.1"></times><ci id="S2.SS3.p5.4.m4.1.1.1.3.2.2.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.2.2">𝑀</ci><ci id="S2.SS3.p5.4.m4.1.1.1.3.2.3.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.2.3">𝐼</ci><apply id="S2.SS3.p5.4.m4.1.1.1.3.2.4.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.2.4"><csymbol cd="ambiguous" id="S2.SS3.p5.4.m4.1.1.1.3.2.4.1.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.2.4">subscript</csymbol><ci id="S2.SS3.p5.4.m4.1.1.1.3.2.4.2.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.2.4.2">𝐴</ci><ci id="S2.SS3.p5.4.m4.1.1.1.3.2.4.3.cmml" xref="S2.SS3.p5.4.m4.1.1.1.3.2.4.3">𝑡</ci></apply></apply></apply><apply id="S2.SS3.p5.4.m4.1.1.1.1.1.1.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.SS3.p5.4.m4.1.1.1.1.1.1.1.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1">subscript</csymbol><ci id="S2.SS3.p5.4.m4.1.1.1.1.1.1.2.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.2">𝑆</ci><apply id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3"><times id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.1.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.1"></times><ci id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.2.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.2">𝑡</ci><ci id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.3.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.3">𝑒</ci><ci id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.4.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.4">𝑠</ci><ci id="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.5.cmml" xref="S2.SS3.p5.4.m4.1.1.1.1.1.1.3.5">𝑡</ci></apply></apply></apply><cn type="float" id="S2.SS3.p5.4.m4.1.1.3.cmml" xref="S2.SS3.p5.4.m4.1.1.3">0.5</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p5.4.m4.1c">\overline{MIA_{t}}(S_{test})=0.5</annotation></semantics></math>).</p>
</div>
<div id="S2.SS3.p6" class="ltx_para ltx_noindent">
<p id="S2.SS3.p6.1" class="ltx_p"><span id="S2.SS3.p6.1.1" class="ltx_text ltx_font_bold">Dimensionality Reduction.</span> To reduce the effects of high dimensionality, the attacker first maps the synthetic data to a lower feature space.
This helps detect the influence of the target record on the training dataset.
We experiment with four different feature sets, as done in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">59</span></a>]</cite>: namely, a <span id="S2.SS3.p6.1.2" class="ltx_text ltx_font_italic">naive</span> feature set, which encodes the number of distinct categories plus the most and least frequent category for each attribute, a <span id="S2.SS3.p6.1.3" class="ltx_text ltx_font_italic">histogram</span>, which computes the frequency counts for each attribute, a <span id="S2.SS3.p6.1.4" class="ltx_text ltx_font_italic">correlation</span>, which encodes pairwise correlations between attributes, and an <span id="S2.SS3.p6.1.5" class="ltx_text ltx_font_italic">ensemble</span> feature set, which combines all the previously mentioned feature sets.</p>
</div>
</section>
<section id="S2.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.4 </span>Datasets</h3>

<div id="S2.SS4.p1" class="ltx_para">
<p id="S2.SS4.p1.1" class="ltx_p">Our evaluation uses data from two projects: HapMap <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib42" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">42</span></a>]</cite> and the 1000 Genome Project <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib1" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">1</span></a>]</cite>.
More specifically, we use 1,000 SNPs from chromosome 13 from the following datasets:</p>
</div>
<div id="S2.SS4.p2" class="ltx_para">
<ol id="S2.I1" class="ltx_enumerate">
<li id="S2.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S2.I1.i1.p1" class="ltx_para">
<p id="S2.I1.i1.p1.1" class="ltx_p"><span id="S2.I1.i1.p1.1.1" class="ltx_text ltx_font_italic">CEU Population (HapMap).</span> Samples from 117 Utah residents with Northern and Western European ancestry, released in phase 2 of the HapMap project.</p>
</div>
</li>
<li id="S2.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S2.I1.i2.p1" class="ltx_para">
<p id="S2.I1.i2.p1.1" class="ltx_p"><span id="S2.I1.i2.p1.1.1" class="ltx_text ltx_font_italic">CHB Population (HapMap).</span> Samples from 120 Han Chinese individuals from Beijing, China.</p>
</div>
</li>
<li id="S2.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span> 
<div id="S2.I1.i3.p1" class="ltx_para">
<p id="S2.I1.i3.p1.1" class="ltx_p"><span id="S2.I1.i3.p1.1.1" class="ltx_text ltx_font_italic">1,000 Genomes.</span> Samples from 2,504 individuals from 26 different populations released from phase 3 of the 1000 Genomes project.</p>
</div>
</li>
</ol>
</div>
</section>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Synthetic Data Approaches in Genomics</h2>

<div id="S3.p1" class="ltx_para">
<p id="S3.p1.1" class="ltx_p">In this section, we provide an overview of the state-of-the-art models for generating synthetic genomic data.
In particular, we discuss the Recombination model presented by Samani et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">54</span></a>]</cite>, the RBM and GAN models proposed by Yelmen et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">71</span></a>]</cite>, and the WGAN model from Killoran et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib36" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">36</span></a>]</cite>.
We also introduce and consider two other “hybrid” models.</p>
</div>
<div id="S3.p2" class="ltx_para ltx_noindent">
<p id="S3.p2.1" class="ltx_p"><span id="S3.p2.1.1" class="ltx_text ltx_font_bold">Recombination Model (Recomb).</span>
Samani et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">54</span></a>]</cite> propose the use of a <span id="S3.p2.1.2" class="ltx_text ltx_font_italic">recombination model</span> as an inference method for quantifying individuals’ genomic privacy.
This statistical model is based on a high-order SNV correlation that relates linkage disequilibrium patterns to the underlying recombination rate.
<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">54</span></a>]</cite> shows how to use this method to generate synthetic samples and perform Principal Component Analysis (PCA).
The recombination model yields a distribution closer to the real data than models using only linkage disequilibrium and allele frequencies.
The model uses a “genetic map,” which includes the recombination rate.
This is provided with the dataset for the HapMap datasets, but not for the 1000 genomes data.
For the latter, we use the scripts from <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib47" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">47</span></a>]</cite> to generate the genomic map.</p>
</div>
<div id="S3.p3" class="ltx_para ltx_noindent">
<p id="S3.p3.1" class="ltx_p"><span id="S3.p3.1.1" class="ltx_text ltx_font_bold">Restricted Boltzmann Machines (RBMs).</span>
RBMs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib58" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">58</span></a>]</cite> are generative models geared to learn a probability distribution over a set of inputs.
RBMs are shallow, two-layer neural nets: the first is known as the “visible” (on input) layer and the second as the hidden layer.
The two layers are connected via a bipartite graph, i.e., every node in the visible layer is connected to every node in the hidden one, but no two nodes in the same group are connected, allowing for more efficient training algorithms.
The learning procedure consists of maximizing the likelihood function over the visible variables of the model.
The RBM models re-create data in an unsupervised manner through many forward and backward passes between the layers, corresponding to sampling from the learned distribution.
The output of the hidden layer passes through an activation function, which becomes the input for the former.</p>
</div>
<div id="S3.p4" class="ltx_para">
<p id="S3.p4.1" class="ltx_p">As mentioned, Yelmen et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">71</span></a>]</cite> use RBMs to generate synthetic genomic data.
In our evaluation, we follow the same RBM settings as  <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">71</span></a>]</cite>.
More specifically, we use a ReLu activation function, with the visible layer having the same size as the input we considered (1,000 features) and with the number of hidden nodes set to 100.
The learning rate is set to 0.01, the batch size to 32, and we iterate over 2,000 epochs.</p>
</div>
<div id="S3.p5" class="ltx_para ltx_noindent">
<p id="S3.p5.1" class="ltx_p"><span id="S3.p5.1.1" class="ltx_text ltx_font_bold">Generative Adversarial Networks (GANs).</span>
A GAN is an unsupervised deep learning model consisting of two neural networks, a generator and a discriminator, which “compete” against each other. During training, the generator’s goal is to produce synthetic data, and the discriminator evaluates them against real data samples to distinguish the synthetic from the real samples.
The training objective is to learn the data distribution so that the data samples produced by the generator cannot be distinguished from real data by the discriminator.</p>
</div>
<div id="S3.p6" class="ltx_para">
<p id="S3.p6.1" class="ltx_p">We use the GAN approach also proposed by Yelmen et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">71</span></a>]</cite>, mirroring their experimental settings.
That is, the generator model consists of an input layer with latent dimension set to 600 and two hidden layers, of sizes 512 and 1,024, respectively.
The discriminator consists of an input layer with a size equal to the number of SNPs evaluated (1,000) and two hidden layers of sizes 512 and 256, respectively, and an output layer of size 1.
The output layer for the generator uses tanh as an activation function, and the output layer for the discriminator uses the sigmoid activation function.
We compile both the generator and discriminator using the Adam optimization and binary cross-entropy as the loss function.</p>
</div>
<div id="S3.p7" class="ltx_para ltx_noindent">
<p id="S3.p7.1" class="ltx_p"><span id="S3.p7.1.1" class="ltx_text ltx_font_bold">Recombination RBM (Rec-RBM).</span>
To overcome issues caused by low numbers of training samples, we propose a hybrid approach between the Recomb and the RBM models.
We use the former to generate extra samples, which we then use, together with the real data samples, to train the RBM model with the same parameters as before.
We do so to explore whether having more data points to train the model improves the utility of the synthetic data.</p>
</div>
<div id="S3.p8" class="ltx_para ltx_noindent">
<p id="S3.p8.1" class="ltx_p"><span id="S3.p8.1.1" class="ltx_text ltx_font_bold">Recombination GAN (Rec-GAN).</span>
Like Rec-RBM, we use the Recomb model to generate extra training samples for the GAN model, using the same parameters as before.
Again, we want to study whether having a larger dataset available for training improves the synthetic data output’s overall utility.</p>
</div>
<div id="S3.p9" class="ltx_para ltx_noindent">
<p id="S3.p9.1" class="ltx_p"><span id="S3.p9.1.1" class="ltx_text ltx_font_bold">Wasserstein GAN (WGAN).</span>
Killoran et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib36" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">36</span></a>]</cite> propose an alternative GAN model by treating DNA sequences as a hybrid between natural language and computer vision data.
The sequences are one-hot encoded, the GAN is based on a WGAN architecture trained with a gradient penalty <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib22" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">22</span></a>]</cite>, and both the generator and discriminator use convolutional neural networks <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">37</span></a>]</cite> and a residual architecture <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib25" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">25</span></a>]</cite>, which includes skip connections that jump over some layers.
The authors also propose a joint method combining the GAN model with an activation maximization design <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">57</span></a>, <a href="#bib.bib73" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">73</span></a>, <a href="#bib.bib40" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">40</span></a>]</cite> to tune the sequences to have desired properties.
However, we do not include the joint model in our evaluation, as we focus on a range of statistics instead of a single desired property.</p>
</div>
<div id="S3.p10" class="ltx_para">
<p id="S3.p10.1" class="ltx_p">In our evaluation, we use the WGAN model with the default parameters from the implementation in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib35" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">35</span></a>]</cite>.
The generator consists of an input layer with a dimension of the latent space set to 100, followed by a hidden layer with a size 100 times the length of the sequence (1,000), which is then reshaped to (length of the sequence, 100), followed by 5 resblocks.
Finally, there is a 1-D convolutional layer followed by the output layer, which uses softmax.
The discriminator has a very similar architecture but in a different order – i.e., it starts with the input layer to which the one-hot sequences are fed, that is followed by the 1-D convolutional layer, then the 5 resblocks, followed by the reshape layer and the output layer of size 1.
We perform 5 discriminator updates for every generator update.
Both the generator and discriminator use Adam optimization, and their learning rates are set to 0.0001, while a gradient penalty adjusts the loss as mentioned.
We use a batch size of 64.
For more details please refer to Appendix <a href="#A1" title="Appendix A WGAN settings ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">A</span></a>.</p>
</div>
<figure id="S3.F1" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S3.F1.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x1.png" id="S3.F1.sf1.g1" class="ltx_graphics ltx_img_landscape" width="438" height="121" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S3.F1.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x2.png" id="S3.F1.sf2.g1" class="ltx_graphics ltx_img_landscape" width="438" height="121" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S3.F1.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x3.png" id="S3.F1.sf3.g1" class="ltx_graphics ltx_img_landscape" width="438" height="121" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(c) </span>1000 Genomes</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>Major allele frequencies for synthetic data generated by the models, plotted against the real data.</figcaption>
</figure>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Utility Evaluation</h2>

<div id="S4.p1" class="ltx_para">
<p id="S4.p1.1" class="ltx_p">This section presents a comprehensive utility evaluation of the synthetic data generated by the models introduced in Section <a href="#S3" title="3 Synthetic Data Approaches in Genomics ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>.
We look at common summary statistics used in genome-wide association studies, aiming to assess the accuracy loss due to the use of synthetic datasets. We choose summary statistics as our main utility evaluation metrics since prior work shows that they provide a good starting point for assessing the utility of synthetics genomic data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib7" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">7</span></a>, <a href="#bib.bib48" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">48</span></a>]</cite>.
While high(er) granularity may be required in some biomedical applications, if most generative models do not accurately replicate the high-level statistics of the real data, then it is extremely unlikely they would provide reasonable accuracy on single sequences.
More specifically, we analyze how well data generated by the generative models preserve allele frequencies, population statistics, and linkage disequilibrium. We also compute the KS two-sample test <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib27" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">27</span></a>]</cite> for the goodness of fit on the most relevant metrics in each subsection for each real dataset vs. the synthetic data.</p>
</div>
<section id="S4.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Allele Statistics</h3>

<figure id="S4.T1" class="ltx_table">
<table id="S4.T1.51" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S4.T1.51.52.1" class="ltx_tr">
<td id="S4.T1.51.52.1.1" class="ltx_td ltx_align_left ltx_border_tt"><span id="S4.T1.51.52.1.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Models</span></td>
<td id="S4.T1.51.52.1.2" class="ltx_td ltx_align_center ltx_border_tt" colspan="2"><span id="S4.T1.51.52.1.2.1" class="ltx_text ltx_font_bold" style="font-size:90%;">CEU</span></td>
<td id="S4.T1.51.52.1.3" class="ltx_td ltx_align_center ltx_border_tt" colspan="2"><span id="S4.T1.51.52.1.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">CHB</span></td>
<td id="S4.T1.51.52.1.4" class="ltx_td ltx_align_center ltx_border_tt" colspan="2"><span id="S4.T1.51.52.1.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">1000 Geno</span></td>
</tr>
<tr id="S4.T1.3.3" class="ltx_tr">
<td id="S4.T1.3.3.4" class="ltx_td"></td>
<td id="S4.T1.1.1.1" class="ltx_td ltx_align_left"><math id="S4.T1.1.1.1.m1.1" class="ltx_Math" alttext="D" display="inline"><semantics id="S4.T1.1.1.1.m1.1a"><mi mathsize="90%" id="S4.T1.1.1.1.m1.1.1" xref="S4.T1.1.1.1.m1.1.1.cmml">D</mi><annotation-xml encoding="MathML-Content" id="S4.T1.1.1.1.m1.1b"><ci id="S4.T1.1.1.1.m1.1.1.cmml" xref="S4.T1.1.1.1.m1.1.1">𝐷</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.1.1.1.m1.1c">D</annotation></semantics></math></td>
<td id="S4.T1.3.3.5" class="ltx_td ltx_align_right"><span id="S4.T1.3.3.5.1" class="ltx_text" style="font-size:90%;">p-value</span></td>
<td id="S4.T1.2.2.2" class="ltx_td ltx_align_right"><math id="S4.T1.2.2.2.m1.1" class="ltx_Math" alttext="D" display="inline"><semantics id="S4.T1.2.2.2.m1.1a"><mi mathsize="90%" id="S4.T1.2.2.2.m1.1.1" xref="S4.T1.2.2.2.m1.1.1.cmml">D</mi><annotation-xml encoding="MathML-Content" id="S4.T1.2.2.2.m1.1b"><ci id="S4.T1.2.2.2.m1.1.1.cmml" xref="S4.T1.2.2.2.m1.1.1">𝐷</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.2.2.2.m1.1c">D</annotation></semantics></math></td>
<td id="S4.T1.3.3.6" class="ltx_td ltx_align_right"><span id="S4.T1.3.3.6.1" class="ltx_text" style="font-size:90%;">p-value</span></td>
<td id="S4.T1.3.3.3" class="ltx_td ltx_align_right"><math id="S4.T1.3.3.3.m1.1" class="ltx_Math" alttext="D" display="inline"><semantics id="S4.T1.3.3.3.m1.1a"><mi mathsize="90%" id="S4.T1.3.3.3.m1.1.1" xref="S4.T1.3.3.3.m1.1.1.cmml">D</mi><annotation-xml encoding="MathML-Content" id="S4.T1.3.3.3.m1.1b"><ci id="S4.T1.3.3.3.m1.1.1.cmml" xref="S4.T1.3.3.3.m1.1.1">𝐷</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.3.3.3.m1.1c">D</annotation></semantics></math></td>
<td id="S4.T1.3.3.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.3.3.7.1" class="ltx_text" style="font-size:90%;">p-value</span></td>
</tr>
<tr id="S4.T1.51.53.2" class="ltx_tr">
<td id="S4.T1.51.53.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S4.T1.51.53.2.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">MAF</span></td>
</tr>
<tr id="S4.T1.51.54.3" class="ltx_tr">
<td id="S4.T1.51.54.3.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S4.T1.51.54.3.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Recomb</span></td>
<td id="S4.T1.51.54.3.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S4.T1.51.54.3.2.1" class="ltx_text" style="font-size:90%;">0.04</span></td>
<td id="S4.T1.51.54.3.3" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.51.54.3.3.1" class="ltx_text" style="font-size:90%;">0.36</span></td>
<td id="S4.T1.51.54.3.4" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.51.54.3.4.1" class="ltx_text" style="font-size:90%;">0.03</span></td>
<td id="S4.T1.51.54.3.5" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.51.54.3.5.1" class="ltx_text" style="font-size:90%;">0.53</span></td>
<td id="S4.T1.51.54.3.6" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.51.54.3.6.1" class="ltx_text" style="font-size:90%;">0.008</span></td>
<td id="S4.T1.51.54.3.7" class="ltx_td ltx_nopad_r ltx_align_right ltx_border_t"><span id="S4.T1.51.54.3.7.1" class="ltx_text" style="font-size:90%;">0.99</span></td>
</tr>
<tr id="S4.T1.5.5" class="ltx_tr">
<td id="S4.T1.5.5.3" class="ltx_td ltx_align_left"><span id="S4.T1.5.5.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">RBM</span></td>
<td id="S4.T1.5.5.4" class="ltx_td ltx_align_left"><span id="S4.T1.5.5.4.1" class="ltx_text" style="font-size:90%;">0.47</span></td>
<td id="S4.T1.4.4.1" class="ltx_td ltx_align_right"><math id="S4.T1.4.4.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.4.4.1.m1.1a"><mrow id="S4.T1.4.4.1.m1.1.1" xref="S4.T1.4.4.1.m1.1.1.cmml"><mi id="S4.T1.4.4.1.m1.1.1.2" xref="S4.T1.4.4.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.4.4.1.m1.1.1.1" xref="S4.T1.4.4.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.4.4.1.m1.1.1.3" xref="S4.T1.4.4.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.4.4.1.m1.1b"><apply id="S4.T1.4.4.1.m1.1.1.cmml" xref="S4.T1.4.4.1.m1.1.1"><lt id="S4.T1.4.4.1.m1.1.1.1.cmml" xref="S4.T1.4.4.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.4.4.1.m1.1.1.2.cmml" xref="S4.T1.4.4.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.4.4.1.m1.1.1.3.cmml" xref="S4.T1.4.4.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.4.4.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.5.5.5" class="ltx_td ltx_align_right"><span id="S4.T1.5.5.5.1" class="ltx_text" style="font-size:90%;">0.33</span></td>
<td id="S4.T1.5.5.2" class="ltx_td ltx_align_right"><math id="S4.T1.5.5.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.5.5.2.m1.1a"><mrow id="S4.T1.5.5.2.m1.1.1" xref="S4.T1.5.5.2.m1.1.1.cmml"><mi id="S4.T1.5.5.2.m1.1.1.2" xref="S4.T1.5.5.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.5.5.2.m1.1.1.1" xref="S4.T1.5.5.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.5.5.2.m1.1.1.3" xref="S4.T1.5.5.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.5.5.2.m1.1b"><apply id="S4.T1.5.5.2.m1.1.1.cmml" xref="S4.T1.5.5.2.m1.1.1"><lt id="S4.T1.5.5.2.m1.1.1.1.cmml" xref="S4.T1.5.5.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.5.5.2.m1.1.1.2.cmml" xref="S4.T1.5.5.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.5.5.2.m1.1.1.3.cmml" xref="S4.T1.5.5.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.5.5.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.5.5.6" class="ltx_td ltx_align_right"><span id="S4.T1.5.5.6.1" class="ltx_text" style="font-size:90%;">0.06</span></td>
<td id="S4.T1.5.5.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.5.5.7.1" class="ltx_text" style="font-size:90%;">0.03</span></td>
</tr>
<tr id="S4.T1.8.8" class="ltx_tr">
<td id="S4.T1.8.8.4" class="ltx_td ltx_align_left"><span id="S4.T1.8.8.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">GAN</span></td>
<td id="S4.T1.8.8.5" class="ltx_td ltx_align_left"><span id="S4.T1.8.8.5.1" class="ltx_text" style="font-size:90%;">0.37</span></td>
<td id="S4.T1.6.6.1" class="ltx_td ltx_align_right"><math id="S4.T1.6.6.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.6.6.1.m1.1a"><mrow id="S4.T1.6.6.1.m1.1.1" xref="S4.T1.6.6.1.m1.1.1.cmml"><mi id="S4.T1.6.6.1.m1.1.1.2" xref="S4.T1.6.6.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.6.6.1.m1.1.1.1" xref="S4.T1.6.6.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.6.6.1.m1.1.1.3" xref="S4.T1.6.6.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.6.6.1.m1.1b"><apply id="S4.T1.6.6.1.m1.1.1.cmml" xref="S4.T1.6.6.1.m1.1.1"><lt id="S4.T1.6.6.1.m1.1.1.1.cmml" xref="S4.T1.6.6.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.6.6.1.m1.1.1.2.cmml" xref="S4.T1.6.6.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.6.6.1.m1.1.1.3.cmml" xref="S4.T1.6.6.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.6.6.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.8.8.6" class="ltx_td ltx_align_right"><span id="S4.T1.8.8.6.1" class="ltx_text" style="font-size:90%;">0.52</span></td>
<td id="S4.T1.7.7.2" class="ltx_td ltx_align_right"><math id="S4.T1.7.7.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.7.7.2.m1.1a"><mrow id="S4.T1.7.7.2.m1.1.1" xref="S4.T1.7.7.2.m1.1.1.cmml"><mi id="S4.T1.7.7.2.m1.1.1.2" xref="S4.T1.7.7.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.7.7.2.m1.1.1.1" xref="S4.T1.7.7.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.7.7.2.m1.1.1.3" xref="S4.T1.7.7.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.7.7.2.m1.1b"><apply id="S4.T1.7.7.2.m1.1.1.cmml" xref="S4.T1.7.7.2.m1.1.1"><lt id="S4.T1.7.7.2.m1.1.1.1.cmml" xref="S4.T1.7.7.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.7.7.2.m1.1.1.2.cmml" xref="S4.T1.7.7.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.7.7.2.m1.1.1.3.cmml" xref="S4.T1.7.7.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.7.7.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.8.8.7" class="ltx_td ltx_align_right"><span id="S4.T1.8.8.7.1" class="ltx_text" style="font-size:90%;">0.58</span></td>
<td id="S4.T1.8.8.3" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.8.8.3.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.8.8.3.m1.1a"><mrow id="S4.T1.8.8.3.m1.1.1" xref="S4.T1.8.8.3.m1.1.1.cmml"><mi id="S4.T1.8.8.3.m1.1.1.2" xref="S4.T1.8.8.3.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.8.8.3.m1.1.1.1" xref="S4.T1.8.8.3.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.8.8.3.m1.1.1.3" xref="S4.T1.8.8.3.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.8.8.3.m1.1b"><apply id="S4.T1.8.8.3.m1.1.1.cmml" xref="S4.T1.8.8.3.m1.1.1"><lt id="S4.T1.8.8.3.m1.1.1.1.cmml" xref="S4.T1.8.8.3.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.8.8.3.m1.1.1.2.cmml" xref="S4.T1.8.8.3.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.8.8.3.m1.1.1.3.cmml" xref="S4.T1.8.8.3.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.8.8.3.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.11.11" class="ltx_tr">
<td id="S4.T1.11.11.4" class="ltx_td ltx_align_left"><span id="S4.T1.11.11.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Rec-RBM</span></td>
<td id="S4.T1.11.11.5" class="ltx_td ltx_align_left"><span id="S4.T1.11.11.5.1" class="ltx_text" style="font-size:90%;">0.31</span></td>
<td id="S4.T1.9.9.1" class="ltx_td ltx_align_right"><math id="S4.T1.9.9.1.m1.1" class="ltx_Math" alttext="{&lt;}0.01" display="inline"><semantics id="S4.T1.9.9.1.m1.1a"><mrow id="S4.T1.9.9.1.m1.1.1" xref="S4.T1.9.9.1.m1.1.1.cmml"><mi id="S4.T1.9.9.1.m1.1.1.2" xref="S4.T1.9.9.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.9.9.1.m1.1.1.1" xref="S4.T1.9.9.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.9.9.1.m1.1.1.3" xref="S4.T1.9.9.1.m1.1.1.3.cmml">0.01</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.9.9.1.m1.1b"><apply id="S4.T1.9.9.1.m1.1.1.cmml" xref="S4.T1.9.9.1.m1.1.1"><lt id="S4.T1.9.9.1.m1.1.1.1.cmml" xref="S4.T1.9.9.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.9.9.1.m1.1.1.2.cmml" xref="S4.T1.9.9.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.9.9.1.m1.1.1.3.cmml" xref="S4.T1.9.9.1.m1.1.1.3">0.01</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.9.9.1.m1.1c">{&lt;}0.01</annotation></semantics></math></td>
<td id="S4.T1.11.11.6" class="ltx_td ltx_align_right"><span id="S4.T1.11.11.6.1" class="ltx_text" style="font-size:90%;">0.23</span></td>
<td id="S4.T1.10.10.2" class="ltx_td ltx_align_right"><math id="S4.T1.10.10.2.m1.1" class="ltx_Math" alttext="{&lt;}0.01" display="inline"><semantics id="S4.T1.10.10.2.m1.1a"><mrow id="S4.T1.10.10.2.m1.1.1" xref="S4.T1.10.10.2.m1.1.1.cmml"><mi id="S4.T1.10.10.2.m1.1.1.2" xref="S4.T1.10.10.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.10.10.2.m1.1.1.1" xref="S4.T1.10.10.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.10.10.2.m1.1.1.3" xref="S4.T1.10.10.2.m1.1.1.3.cmml">0.01</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.10.10.2.m1.1b"><apply id="S4.T1.10.10.2.m1.1.1.cmml" xref="S4.T1.10.10.2.m1.1.1"><lt id="S4.T1.10.10.2.m1.1.1.1.cmml" xref="S4.T1.10.10.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.10.10.2.m1.1.1.2.cmml" xref="S4.T1.10.10.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.10.10.2.m1.1.1.3.cmml" xref="S4.T1.10.10.2.m1.1.1.3">0.01</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.10.10.2.m1.1c">{&lt;}0.01</annotation></semantics></math></td>
<td id="S4.T1.11.11.7" class="ltx_td ltx_align_right"><span id="S4.T1.11.11.7.1" class="ltx_text" style="font-size:90%;">0.08</span></td>
<td id="S4.T1.11.11.3" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.11.11.3.m1.1" class="ltx_Math" alttext="{&lt;}0.1" display="inline"><semantics id="S4.T1.11.11.3.m1.1a"><mrow id="S4.T1.11.11.3.m1.1.1" xref="S4.T1.11.11.3.m1.1.1.cmml"><mi id="S4.T1.11.11.3.m1.1.1.2" xref="S4.T1.11.11.3.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.11.11.3.m1.1.1.1" xref="S4.T1.11.11.3.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.11.11.3.m1.1.1.3" xref="S4.T1.11.11.3.m1.1.1.3.cmml">0.1</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.11.11.3.m1.1b"><apply id="S4.T1.11.11.3.m1.1.1.cmml" xref="S4.T1.11.11.3.m1.1.1"><lt id="S4.T1.11.11.3.m1.1.1.1.cmml" xref="S4.T1.11.11.3.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.11.11.3.m1.1.1.2.cmml" xref="S4.T1.11.11.3.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.11.11.3.m1.1.1.3.cmml" xref="S4.T1.11.11.3.m1.1.1.3">0.1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.11.11.3.m1.1c">{&lt;}0.1</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.13.13" class="ltx_tr">
<td id="S4.T1.13.13.3" class="ltx_td ltx_align_left"><span id="S4.T1.13.13.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Rec-GAN</span></td>
<td id="S4.T1.13.13.4" class="ltx_td ltx_align_left"><span id="S4.T1.13.13.4.1" class="ltx_text" style="font-size:90%;">0.54</span></td>
<td id="S4.T1.12.12.1" class="ltx_td ltx_align_right"><math id="S4.T1.12.12.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.12.12.1.m1.1a"><mrow id="S4.T1.12.12.1.m1.1.1" xref="S4.T1.12.12.1.m1.1.1.cmml"><mi id="S4.T1.12.12.1.m1.1.1.2" xref="S4.T1.12.12.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.12.12.1.m1.1.1.1" xref="S4.T1.12.12.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.12.12.1.m1.1.1.3" xref="S4.T1.12.12.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.12.12.1.m1.1b"><apply id="S4.T1.12.12.1.m1.1.1.cmml" xref="S4.T1.12.12.1.m1.1.1"><lt id="S4.T1.12.12.1.m1.1.1.1.cmml" xref="S4.T1.12.12.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.12.12.1.m1.1.1.2.cmml" xref="S4.T1.12.12.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.12.12.1.m1.1.1.3.cmml" xref="S4.T1.12.12.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.12.12.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.13.13.5" class="ltx_td ltx_align_right"><span id="S4.T1.13.13.5.1" class="ltx_text" style="font-size:90%;">0.61</span></td>
<td id="S4.T1.13.13.2" class="ltx_td ltx_align_right"><math id="S4.T1.13.13.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.13.13.2.m1.1a"><mrow id="S4.T1.13.13.2.m1.1.1" xref="S4.T1.13.13.2.m1.1.1.cmml"><mi id="S4.T1.13.13.2.m1.1.1.2" xref="S4.T1.13.13.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.13.13.2.m1.1.1.1" xref="S4.T1.13.13.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.13.13.2.m1.1.1.3" xref="S4.T1.13.13.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.13.13.2.m1.1b"><apply id="S4.T1.13.13.2.m1.1.1.cmml" xref="S4.T1.13.13.2.m1.1.1"><lt id="S4.T1.13.13.2.m1.1.1.1.cmml" xref="S4.T1.13.13.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.13.13.2.m1.1.1.2.cmml" xref="S4.T1.13.13.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.13.13.2.m1.1.1.3.cmml" xref="S4.T1.13.13.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.13.13.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.13.13.6" class="ltx_td ltx_align_right"><span id="S4.T1.13.13.6.1" class="ltx_text" style="font-size:90%;">0.863</span></td>
<td id="S4.T1.13.13.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.13.13.7.1" class="ltx_text" style="font-size:90%;">0</span></td>
</tr>
<tr id="S4.T1.15.15" class="ltx_tr">
<td id="S4.T1.15.15.3" class="ltx_td ltx_align_left"><span id="S4.T1.15.15.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">WGAN</span></td>
<td id="S4.T1.15.15.4" class="ltx_td ltx_align_left"><span id="S4.T1.15.15.4.1" class="ltx_text" style="font-size:90%;">0.10</span></td>
<td id="S4.T1.14.14.1" class="ltx_td ltx_align_right"><math id="S4.T1.14.14.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.14.14.1.m1.1a"><mrow id="S4.T1.14.14.1.m1.1.1" xref="S4.T1.14.14.1.m1.1.1.cmml"><mi id="S4.T1.14.14.1.m1.1.1.2" xref="S4.T1.14.14.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.14.14.1.m1.1.1.1" xref="S4.T1.14.14.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.14.14.1.m1.1.1.3" xref="S4.T1.14.14.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.14.14.1.m1.1b"><apply id="S4.T1.14.14.1.m1.1.1.cmml" xref="S4.T1.14.14.1.m1.1.1"><lt id="S4.T1.14.14.1.m1.1.1.1.cmml" xref="S4.T1.14.14.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.14.14.1.m1.1.1.2.cmml" xref="S4.T1.14.14.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.14.14.1.m1.1.1.3.cmml" xref="S4.T1.14.14.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.14.14.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.15.15.5" class="ltx_td ltx_align_right"><span id="S4.T1.15.15.5.1" class="ltx_text" style="font-size:90%;">0.14</span></td>
<td id="S4.T1.15.15.2" class="ltx_td ltx_align_right"><math id="S4.T1.15.15.2.m1.1" class="ltx_Math" alttext="{&lt;}0.1" display="inline"><semantics id="S4.T1.15.15.2.m1.1a"><mrow id="S4.T1.15.15.2.m1.1.1" xref="S4.T1.15.15.2.m1.1.1.cmml"><mi id="S4.T1.15.15.2.m1.1.1.2" xref="S4.T1.15.15.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.15.15.2.m1.1.1.1" xref="S4.T1.15.15.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.15.15.2.m1.1.1.3" xref="S4.T1.15.15.2.m1.1.1.3.cmml">0.1</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.15.15.2.m1.1b"><apply id="S4.T1.15.15.2.m1.1.1.cmml" xref="S4.T1.15.15.2.m1.1.1"><lt id="S4.T1.15.15.2.m1.1.1.1.cmml" xref="S4.T1.15.15.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.15.15.2.m1.1.1.2.cmml" xref="S4.T1.15.15.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.15.15.2.m1.1.1.3.cmml" xref="S4.T1.15.15.2.m1.1.1.3">0.1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.15.15.2.m1.1c">{&lt;}0.1</annotation></semantics></math></td>
<td id="S4.T1.15.15.6" class="ltx_td ltx_align_right"><span id="S4.T1.15.15.6.1" class="ltx_text" style="font-size:90%;">0.28</span></td>
<td id="S4.T1.15.15.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.15.15.7.1" class="ltx_text" style="font-size:90%;">0.82</span></td>
</tr>
<tr id="S4.T1.51.55.4" class="ltx_tr">
<td id="S4.T1.51.55.4.1" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S4.T1.51.55.4.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">SFS</span></td>
</tr>
<tr id="S4.T1.16.16" class="ltx_tr">
<td id="S4.T1.16.16.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S4.T1.16.16.2.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Recomb</span></td>
<td id="S4.T1.16.16.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S4.T1.16.16.3.1" class="ltx_text" style="font-size:90%;">0.07</span></td>
<td id="S4.T1.16.16.4" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.16.16.4.1" class="ltx_text" style="font-size:90%;">0.89</span></td>
<td id="S4.T1.16.16.5" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.16.16.5.1" class="ltx_text" style="font-size:90%;">0.07</span></td>
<td id="S4.T1.16.16.6" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.16.16.6.1" class="ltx_text" style="font-size:90%;">0.89</span></td>
<td id="S4.T1.16.16.7" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.16.16.7.1" class="ltx_text" style="font-size:90%;">0.18</span></td>
<td id="S4.T1.16.16.1" class="ltx_td ltx_nopad_r ltx_align_right ltx_border_t"><math id="S4.T1.16.16.1.m1.1" class="ltx_Math" alttext="{&lt;}0.1" display="inline"><semantics id="S4.T1.16.16.1.m1.1a"><mrow id="S4.T1.16.16.1.m1.1.1" xref="S4.T1.16.16.1.m1.1.1.cmml"><mi id="S4.T1.16.16.1.m1.1.1.2" xref="S4.T1.16.16.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.16.16.1.m1.1.1.1" xref="S4.T1.16.16.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.16.16.1.m1.1.1.3" xref="S4.T1.16.16.1.m1.1.1.3.cmml">0.1</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.16.16.1.m1.1b"><apply id="S4.T1.16.16.1.m1.1.1.cmml" xref="S4.T1.16.16.1.m1.1.1"><lt id="S4.T1.16.16.1.m1.1.1.1.cmml" xref="S4.T1.16.16.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.16.16.1.m1.1.1.2.cmml" xref="S4.T1.16.16.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.16.16.1.m1.1.1.3.cmml" xref="S4.T1.16.16.1.m1.1.1.3">0.1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.16.16.1.m1.1c">{&lt;}0.1</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.18.18" class="ltx_tr">
<td id="S4.T1.18.18.3" class="ltx_td ltx_align_left"><span id="S4.T1.18.18.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">RBM</span></td>
<td id="S4.T1.18.18.4" class="ltx_td ltx_align_left"><span id="S4.T1.18.18.4.1" class="ltx_text" style="font-size:90%;">0.34</span></td>
<td id="S4.T1.17.17.1" class="ltx_td ltx_align_right"><math id="S4.T1.17.17.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.17.17.1.m1.1a"><mrow id="S4.T1.17.17.1.m1.1.1" xref="S4.T1.17.17.1.m1.1.1.cmml"><mi id="S4.T1.17.17.1.m1.1.1.2" xref="S4.T1.17.17.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.17.17.1.m1.1.1.1" xref="S4.T1.17.17.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.17.17.1.m1.1.1.3" xref="S4.T1.17.17.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.17.17.1.m1.1b"><apply id="S4.T1.17.17.1.m1.1.1.cmml" xref="S4.T1.17.17.1.m1.1.1"><lt id="S4.T1.17.17.1.m1.1.1.1.cmml" xref="S4.T1.17.17.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.17.17.1.m1.1.1.2.cmml" xref="S4.T1.17.17.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.17.17.1.m1.1.1.3.cmml" xref="S4.T1.17.17.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.17.17.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.18.18.5" class="ltx_td ltx_align_right"><span id="S4.T1.18.18.5.1" class="ltx_text" style="font-size:90%;">0.28</span></td>
<td id="S4.T1.18.18.2" class="ltx_td ltx_align_right"><math id="S4.T1.18.18.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.18.18.2.m1.1a"><mrow id="S4.T1.18.18.2.m1.1.1" xref="S4.T1.18.18.2.m1.1.1.cmml"><mi id="S4.T1.18.18.2.m1.1.1.2" xref="S4.T1.18.18.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.18.18.2.m1.1.1.1" xref="S4.T1.18.18.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.18.18.2.m1.1.1.3" xref="S4.T1.18.18.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.18.18.2.m1.1b"><apply id="S4.T1.18.18.2.m1.1.1.cmml" xref="S4.T1.18.18.2.m1.1.1"><lt id="S4.T1.18.18.2.m1.1.1.1.cmml" xref="S4.T1.18.18.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.18.18.2.m1.1.1.2.cmml" xref="S4.T1.18.18.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.18.18.2.m1.1.1.3.cmml" xref="S4.T1.18.18.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.18.18.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.18.18.6" class="ltx_td ltx_align_right"><span id="S4.T1.18.18.6.1" class="ltx_text" style="font-size:90%;">0.12</span></td>
<td id="S4.T1.18.18.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.18.18.7.1" class="ltx_text" style="font-size:90%;">0.44</span></td>
</tr>
<tr id="S4.T1.21.21" class="ltx_tr">
<td id="S4.T1.21.21.4" class="ltx_td ltx_align_left"><span id="S4.T1.21.21.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">GAN</span></td>
<td id="S4.T1.21.21.5" class="ltx_td ltx_align_left"><span id="S4.T1.21.21.5.1" class="ltx_text" style="font-size:90%;">0.40</span></td>
<td id="S4.T1.19.19.1" class="ltx_td ltx_align_right"><math id="S4.T1.19.19.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.19.19.1.m1.1a"><mrow id="S4.T1.19.19.1.m1.1.1" xref="S4.T1.19.19.1.m1.1.1.cmml"><mi id="S4.T1.19.19.1.m1.1.1.2" xref="S4.T1.19.19.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.19.19.1.m1.1.1.1" xref="S4.T1.19.19.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.19.19.1.m1.1.1.3" xref="S4.T1.19.19.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.19.19.1.m1.1b"><apply id="S4.T1.19.19.1.m1.1.1.cmml" xref="S4.T1.19.19.1.m1.1.1"><lt id="S4.T1.19.19.1.m1.1.1.1.cmml" xref="S4.T1.19.19.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.19.19.1.m1.1.1.2.cmml" xref="S4.T1.19.19.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.19.19.1.m1.1.1.3.cmml" xref="S4.T1.19.19.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.19.19.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.21.21.6" class="ltx_td ltx_align_right"><span id="S4.T1.21.21.6.1" class="ltx_text" style="font-size:90%;">0.41</span></td>
<td id="S4.T1.20.20.2" class="ltx_td ltx_align_right"><math id="S4.T1.20.20.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.20.20.2.m1.1a"><mrow id="S4.T1.20.20.2.m1.1.1" xref="S4.T1.20.20.2.m1.1.1.cmml"><mi id="S4.T1.20.20.2.m1.1.1.2" xref="S4.T1.20.20.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.20.20.2.m1.1.1.1" xref="S4.T1.20.20.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.20.20.2.m1.1.1.3" xref="S4.T1.20.20.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.20.20.2.m1.1b"><apply id="S4.T1.20.20.2.m1.1.1.cmml" xref="S4.T1.20.20.2.m1.1.1"><lt id="S4.T1.20.20.2.m1.1.1.1.cmml" xref="S4.T1.20.20.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.20.20.2.m1.1.1.2.cmml" xref="S4.T1.20.20.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.20.20.2.m1.1.1.3.cmml" xref="S4.T1.20.20.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.20.20.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.21.21.7" class="ltx_td ltx_align_right"><span id="S4.T1.21.21.7.1" class="ltx_text" style="font-size:90%;">0.23</span></td>
<td id="S4.T1.21.21.3" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.21.21.3.m1.1" class="ltx_Math" alttext="{&lt;}0.01" display="inline"><semantics id="S4.T1.21.21.3.m1.1a"><mrow id="S4.T1.21.21.3.m1.1.1" xref="S4.T1.21.21.3.m1.1.1.cmml"><mi id="S4.T1.21.21.3.m1.1.1.2" xref="S4.T1.21.21.3.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.21.21.3.m1.1.1.1" xref="S4.T1.21.21.3.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.21.21.3.m1.1.1.3" xref="S4.T1.21.21.3.m1.1.1.3.cmml">0.01</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.21.21.3.m1.1b"><apply id="S4.T1.21.21.3.m1.1.1.cmml" xref="S4.T1.21.21.3.m1.1.1"><lt id="S4.T1.21.21.3.m1.1.1.1.cmml" xref="S4.T1.21.21.3.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.21.21.3.m1.1.1.2.cmml" xref="S4.T1.21.21.3.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.21.21.3.m1.1.1.3.cmml" xref="S4.T1.21.21.3.m1.1.1.3">0.01</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.21.21.3.m1.1c">{&lt;}0.01</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.23.23" class="ltx_tr">
<td id="S4.T1.23.23.3" class="ltx_td ltx_align_left"><span id="S4.T1.23.23.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Rec-RBM</span></td>
<td id="S4.T1.23.23.4" class="ltx_td ltx_align_left"><span id="S4.T1.23.23.4.1" class="ltx_text" style="font-size:90%;">0.26</span></td>
<td id="S4.T1.22.22.1" class="ltx_td ltx_align_right"><math id="S4.T1.22.22.1.m1.1" class="ltx_Math" alttext="{&lt;}0.01" display="inline"><semantics id="S4.T1.22.22.1.m1.1a"><mrow id="S4.T1.22.22.1.m1.1.1" xref="S4.T1.22.22.1.m1.1.1.cmml"><mi id="S4.T1.22.22.1.m1.1.1.2" xref="S4.T1.22.22.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.22.22.1.m1.1.1.1" xref="S4.T1.22.22.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.22.22.1.m1.1.1.3" xref="S4.T1.22.22.1.m1.1.1.3.cmml">0.01</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.22.22.1.m1.1b"><apply id="S4.T1.22.22.1.m1.1.1.cmml" xref="S4.T1.22.22.1.m1.1.1"><lt id="S4.T1.22.22.1.m1.1.1.1.cmml" xref="S4.T1.22.22.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.22.22.1.m1.1.1.2.cmml" xref="S4.T1.22.22.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.22.22.1.m1.1.1.3.cmml" xref="S4.T1.22.22.1.m1.1.1.3">0.01</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.22.22.1.m1.1c">{&lt;}0.01</annotation></semantics></math></td>
<td id="S4.T1.23.23.5" class="ltx_td ltx_align_right"><span id="S4.T1.23.23.5.1" class="ltx_text" style="font-size:90%;">0.23</span></td>
<td id="S4.T1.23.23.2" class="ltx_td ltx_align_right"><math id="S4.T1.23.23.2.m1.1" class="ltx_Math" alttext="{&lt;}0.01" display="inline"><semantics id="S4.T1.23.23.2.m1.1a"><mrow id="S4.T1.23.23.2.m1.1.1" xref="S4.T1.23.23.2.m1.1.1.cmml"><mi id="S4.T1.23.23.2.m1.1.1.2" xref="S4.T1.23.23.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.23.23.2.m1.1.1.1" xref="S4.T1.23.23.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.23.23.2.m1.1.1.3" xref="S4.T1.23.23.2.m1.1.1.3.cmml">0.01</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.23.23.2.m1.1b"><apply id="S4.T1.23.23.2.m1.1.1.cmml" xref="S4.T1.23.23.2.m1.1.1"><lt id="S4.T1.23.23.2.m1.1.1.1.cmml" xref="S4.T1.23.23.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.23.23.2.m1.1.1.2.cmml" xref="S4.T1.23.23.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.23.23.2.m1.1.1.3.cmml" xref="S4.T1.23.23.2.m1.1.1.3">0.01</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.23.23.2.m1.1c">{&lt;}0.01</annotation></semantics></math></td>
<td id="S4.T1.23.23.6" class="ltx_td ltx_align_right"><span id="S4.T1.23.23.6.1" class="ltx_text" style="font-size:90%;">0.06</span></td>
<td id="S4.T1.23.23.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.23.23.7.1" class="ltx_text" style="font-size:90%;">0.99</span></td>
</tr>
<tr id="S4.T1.26.26" class="ltx_tr">
<td id="S4.T1.26.26.4" class="ltx_td ltx_align_left"><span id="S4.T1.26.26.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Rec-GAN</span></td>
<td id="S4.T1.26.26.5" class="ltx_td ltx_align_left"><span id="S4.T1.26.26.5.1" class="ltx_text" style="font-size:90%;">0.64</span></td>
<td id="S4.T1.24.24.1" class="ltx_td ltx_align_right"><math id="S4.T1.24.24.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.24.24.1.m1.1a"><mrow id="S4.T1.24.24.1.m1.1.1" xref="S4.T1.24.24.1.m1.1.1.cmml"><mi id="S4.T1.24.24.1.m1.1.1.2" xref="S4.T1.24.24.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.24.24.1.m1.1.1.1" xref="S4.T1.24.24.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.24.24.1.m1.1.1.3" xref="S4.T1.24.24.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.24.24.1.m1.1b"><apply id="S4.T1.24.24.1.m1.1.1.cmml" xref="S4.T1.24.24.1.m1.1.1"><lt id="S4.T1.24.24.1.m1.1.1.1.cmml" xref="S4.T1.24.24.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.24.24.1.m1.1.1.2.cmml" xref="S4.T1.24.24.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.24.24.1.m1.1.1.3.cmml" xref="S4.T1.24.24.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.24.24.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.26.26.6" class="ltx_td ltx_align_right"><span id="S4.T1.26.26.6.1" class="ltx_text" style="font-size:90%;">0.58</span></td>
<td id="S4.T1.25.25.2" class="ltx_td ltx_align_right"><math id="S4.T1.25.25.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.25.25.2.m1.1a"><mrow id="S4.T1.25.25.2.m1.1.1" xref="S4.T1.25.25.2.m1.1.1.cmml"><mi id="S4.T1.25.25.2.m1.1.1.2" xref="S4.T1.25.25.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.25.25.2.m1.1.1.1" xref="S4.T1.25.25.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.25.25.2.m1.1.1.3" xref="S4.T1.25.25.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.25.25.2.m1.1b"><apply id="S4.T1.25.25.2.m1.1.1.cmml" xref="S4.T1.25.25.2.m1.1.1"><lt id="S4.T1.25.25.2.m1.1.1.1.cmml" xref="S4.T1.25.25.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.25.25.2.m1.1.1.2.cmml" xref="S4.T1.25.25.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.25.25.2.m1.1.1.3.cmml" xref="S4.T1.25.25.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.25.25.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.26.26.7" class="ltx_td ltx_align_right"><span id="S4.T1.26.26.7.1" class="ltx_text" style="font-size:90%;">0.25</span></td>
<td id="S4.T1.26.26.3" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.26.26.3.m1.1" class="ltx_Math" alttext="{&lt;}0.01" display="inline"><semantics id="S4.T1.26.26.3.m1.1a"><mrow id="S4.T1.26.26.3.m1.1.1" xref="S4.T1.26.26.3.m1.1.1.cmml"><mi id="S4.T1.26.26.3.m1.1.1.2" xref="S4.T1.26.26.3.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.26.26.3.m1.1.1.1" xref="S4.T1.26.26.3.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.26.26.3.m1.1.1.3" xref="S4.T1.26.26.3.m1.1.1.3.cmml">0.01</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.26.26.3.m1.1b"><apply id="S4.T1.26.26.3.m1.1.1.cmml" xref="S4.T1.26.26.3.m1.1.1"><lt id="S4.T1.26.26.3.m1.1.1.1.cmml" xref="S4.T1.26.26.3.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.26.26.3.m1.1.1.2.cmml" xref="S4.T1.26.26.3.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.26.26.3.m1.1.1.3.cmml" xref="S4.T1.26.26.3.m1.1.1.3">0.01</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.26.26.3.m1.1c">{&lt;}0.01</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.28.28" class="ltx_tr">
<td id="S4.T1.28.28.3" class="ltx_td ltx_align_left"><span id="S4.T1.28.28.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">WGAN</span></td>
<td id="S4.T1.28.28.4" class="ltx_td ltx_align_left"><span id="S4.T1.28.28.4.1" class="ltx_text" style="font-size:90%;">0.09</span></td>
<td id="S4.T1.28.28.5" class="ltx_td ltx_align_right"><span id="S4.T1.28.28.5.1" class="ltx_text" style="font-size:90%;">0.79</span></td>
<td id="S4.T1.28.28.6" class="ltx_td ltx_align_right"><span id="S4.T1.28.28.6.1" class="ltx_text" style="font-size:90%;">0.18</span></td>
<td id="S4.T1.27.27.1" class="ltx_td ltx_align_right"><math id="S4.T1.27.27.1.m1.1" class="ltx_Math" alttext="{&lt;}0.1" display="inline"><semantics id="S4.T1.27.27.1.m1.1a"><mrow id="S4.T1.27.27.1.m1.1.1" xref="S4.T1.27.27.1.m1.1.1.cmml"><mi id="S4.T1.27.27.1.m1.1.1.2" xref="S4.T1.27.27.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.27.27.1.m1.1.1.1" xref="S4.T1.27.27.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.27.27.1.m1.1.1.3" xref="S4.T1.27.27.1.m1.1.1.3.cmml">0.1</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.27.27.1.m1.1b"><apply id="S4.T1.27.27.1.m1.1.1.cmml" xref="S4.T1.27.27.1.m1.1.1"><lt id="S4.T1.27.27.1.m1.1.1.1.cmml" xref="S4.T1.27.27.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.27.27.1.m1.1.1.2.cmml" xref="S4.T1.27.27.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.27.27.1.m1.1.1.3.cmml" xref="S4.T1.27.27.1.m1.1.1.3">0.1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.27.27.1.m1.1c">{&lt;}0.1</annotation></semantics></math></td>
<td id="S4.T1.28.28.7" class="ltx_td ltx_align_right"><span id="S4.T1.28.28.7.1" class="ltx_text" style="font-size:90%;">0.19</span></td>
<td id="S4.T1.28.28.2" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.28.28.2.m1.1" class="ltx_Math" alttext="{&lt;}0.1" display="inline"><semantics id="S4.T1.28.28.2.m1.1a"><mrow id="S4.T1.28.28.2.m1.1.1" xref="S4.T1.28.28.2.m1.1.1.cmml"><mi id="S4.T1.28.28.2.m1.1.1.2" xref="S4.T1.28.28.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.28.28.2.m1.1.1.1" xref="S4.T1.28.28.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.28.28.2.m1.1.1.3" xref="S4.T1.28.28.2.m1.1.1.3.cmml">0.1</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.28.28.2.m1.1b"><apply id="S4.T1.28.28.2.m1.1.1.cmml" xref="S4.T1.28.28.2.m1.1.1"><lt id="S4.T1.28.28.2.m1.1.1.1.cmml" xref="S4.T1.28.28.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.28.28.2.m1.1.1.2.cmml" xref="S4.T1.28.28.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.28.28.2.m1.1.1.3.cmml" xref="S4.T1.28.28.2.m1.1.1.3">0.1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.28.28.2.m1.1c">{&lt;}0.1</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.51.56.5" class="ltx_tr">
<td id="S4.T1.51.56.5.1" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S4.T1.51.56.5.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">% Heterozygous Samples</span></td>
</tr>
<tr id="S4.T1.30.30" class="ltx_tr">
<td id="S4.T1.30.30.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S4.T1.30.30.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Recomb</span></td>
<td id="S4.T1.30.30.4" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.30.30.4.1" class="ltx_text" style="font-size:90%;">0.19</span></td>
<td id="S4.T1.30.30.5" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.30.30.5.1" class="ltx_text" style="font-size:90%;">0.47</span></td>
<td id="S4.T1.30.30.6" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.30.30.6.1" class="ltx_text" style="font-size:90%;">0.29</span></td>
<td id="S4.T1.29.29.1" class="ltx_td ltx_align_right ltx_border_t"><math id="S4.T1.29.29.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.29.29.1.m1.1a"><mrow id="S4.T1.29.29.1.m1.1.1" xref="S4.T1.29.29.1.m1.1.1.cmml"><mi id="S4.T1.29.29.1.m1.1.1.2" xref="S4.T1.29.29.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.29.29.1.m1.1.1.1" xref="S4.T1.29.29.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.29.29.1.m1.1.1.3" xref="S4.T1.29.29.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.29.29.1.m1.1b"><apply id="S4.T1.29.29.1.m1.1.1.cmml" xref="S4.T1.29.29.1.m1.1.1"><lt id="S4.T1.29.29.1.m1.1.1.1.cmml" xref="S4.T1.29.29.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.29.29.1.m1.1.1.2.cmml" xref="S4.T1.29.29.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.29.29.1.m1.1.1.3.cmml" xref="S4.T1.29.29.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.29.29.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.30.30.7" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.30.30.7.1" class="ltx_text" style="font-size:90%;">0.32</span></td>
<td id="S4.T1.30.30.2" class="ltx_td ltx_nopad_r ltx_align_right ltx_border_t"><math id="S4.T1.30.30.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.30.30.2.m1.1a"><mrow id="S4.T1.30.30.2.m1.1.1" xref="S4.T1.30.30.2.m1.1.1.cmml"><mi id="S4.T1.30.30.2.m1.1.1.2" xref="S4.T1.30.30.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.30.30.2.m1.1.1.1" xref="S4.T1.30.30.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.30.30.2.m1.1.1.3" xref="S4.T1.30.30.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.30.30.2.m1.1b"><apply id="S4.T1.30.30.2.m1.1.1.cmml" xref="S4.T1.30.30.2.m1.1.1"><lt id="S4.T1.30.30.2.m1.1.1.1.cmml" xref="S4.T1.30.30.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.30.30.2.m1.1.1.2.cmml" xref="S4.T1.30.30.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.30.30.2.m1.1.1.3.cmml" xref="S4.T1.30.30.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.30.30.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.32.32" class="ltx_tr">
<td id="S4.T1.32.32.3" class="ltx_td ltx_align_left"><span id="S4.T1.32.32.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">RBM</span></td>
<td id="S4.T1.32.32.4" class="ltx_td ltx_align_right"><span id="S4.T1.32.32.4.1" class="ltx_text" style="font-size:90%;">0.64</span></td>
<td id="S4.T1.31.31.1" class="ltx_td ltx_align_right"><math id="S4.T1.31.31.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.31.31.1.m1.1a"><mrow id="S4.T1.31.31.1.m1.1.1" xref="S4.T1.31.31.1.m1.1.1.cmml"><mi id="S4.T1.31.31.1.m1.1.1.2" xref="S4.T1.31.31.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.31.31.1.m1.1.1.1" xref="S4.T1.31.31.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.31.31.1.m1.1.1.3" xref="S4.T1.31.31.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.31.31.1.m1.1b"><apply id="S4.T1.31.31.1.m1.1.1.cmml" xref="S4.T1.31.31.1.m1.1.1"><lt id="S4.T1.31.31.1.m1.1.1.1.cmml" xref="S4.T1.31.31.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.31.31.1.m1.1.1.2.cmml" xref="S4.T1.31.31.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.31.31.1.m1.1.1.3.cmml" xref="S4.T1.31.31.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.31.31.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.32.32.5" class="ltx_td ltx_align_right"><span id="S4.T1.32.32.5.1" class="ltx_text" style="font-size:90%;">0.70</span></td>
<td id="S4.T1.32.32.2" class="ltx_td ltx_align_right"><math id="S4.T1.32.32.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.32.32.2.m1.1a"><mrow id="S4.T1.32.32.2.m1.1.1" xref="S4.T1.32.32.2.m1.1.1.cmml"><mi id="S4.T1.32.32.2.m1.1.1.2" xref="S4.T1.32.32.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.32.32.2.m1.1.1.1" xref="S4.T1.32.32.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.32.32.2.m1.1.1.3" xref="S4.T1.32.32.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.32.32.2.m1.1b"><apply id="S4.T1.32.32.2.m1.1.1.cmml" xref="S4.T1.32.32.2.m1.1.1"><lt id="S4.T1.32.32.2.m1.1.1.1.cmml" xref="S4.T1.32.32.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.32.32.2.m1.1.1.2.cmml" xref="S4.T1.32.32.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.32.32.2.m1.1.1.3.cmml" xref="S4.T1.32.32.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.32.32.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.32.32.6" class="ltx_td ltx_align_right"><span id="S4.T1.32.32.6.1" class="ltx_text" style="font-size:90%;">0.13</span></td>
<td id="S4.T1.32.32.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.32.32.7.1" class="ltx_text" style="font-size:90%;">0.34</span></td>
</tr>
<tr id="S4.T1.35.35" class="ltx_tr">
<td id="S4.T1.35.35.4" class="ltx_td ltx_align_left"><span id="S4.T1.35.35.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">GAN</span></td>
<td id="S4.T1.35.35.5" class="ltx_td ltx_align_right"><span id="S4.T1.35.35.5.1" class="ltx_text" style="font-size:90%;">0.90</span></td>
<td id="S4.T1.33.33.1" class="ltx_td ltx_align_right"><math id="S4.T1.33.33.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.33.33.1.m1.1a"><mrow id="S4.T1.33.33.1.m1.1.1" xref="S4.T1.33.33.1.m1.1.1.cmml"><mi id="S4.T1.33.33.1.m1.1.1.2" xref="S4.T1.33.33.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.33.33.1.m1.1.1.1" xref="S4.T1.33.33.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.33.33.1.m1.1.1.3" xref="S4.T1.33.33.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.33.33.1.m1.1b"><apply id="S4.T1.33.33.1.m1.1.1.cmml" xref="S4.T1.33.33.1.m1.1.1"><lt id="S4.T1.33.33.1.m1.1.1.1.cmml" xref="S4.T1.33.33.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.33.33.1.m1.1.1.2.cmml" xref="S4.T1.33.33.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.33.33.1.m1.1.1.3.cmml" xref="S4.T1.33.33.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.33.33.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.35.35.6" class="ltx_td ltx_align_right"><span id="S4.T1.35.35.6.1" class="ltx_text" style="font-size:90%;">1.00</span></td>
<td id="S4.T1.34.34.2" class="ltx_td ltx_align_right"><math id="S4.T1.34.34.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.34.34.2.m1.1a"><mrow id="S4.T1.34.34.2.m1.1.1" xref="S4.T1.34.34.2.m1.1.1.cmml"><mi id="S4.T1.34.34.2.m1.1.1.2" xref="S4.T1.34.34.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.34.34.2.m1.1.1.1" xref="S4.T1.34.34.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.34.34.2.m1.1.1.3" xref="S4.T1.34.34.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.34.34.2.m1.1b"><apply id="S4.T1.34.34.2.m1.1.1.cmml" xref="S4.T1.34.34.2.m1.1.1"><lt id="S4.T1.34.34.2.m1.1.1.1.cmml" xref="S4.T1.34.34.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.34.34.2.m1.1.1.2.cmml" xref="S4.T1.34.34.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.34.34.2.m1.1.1.3.cmml" xref="S4.T1.34.34.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.34.34.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.35.35.7" class="ltx_td ltx_align_right"><span id="S4.T1.35.35.7.1" class="ltx_text" style="font-size:90%;">0.57</span></td>
<td id="S4.T1.35.35.3" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.35.35.3.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.35.35.3.m1.1a"><mrow id="S4.T1.35.35.3.m1.1.1" xref="S4.T1.35.35.3.m1.1.1.cmml"><mi id="S4.T1.35.35.3.m1.1.1.2" xref="S4.T1.35.35.3.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.35.35.3.m1.1.1.1" xref="S4.T1.35.35.3.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.35.35.3.m1.1.1.3" xref="S4.T1.35.35.3.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.35.35.3.m1.1b"><apply id="S4.T1.35.35.3.m1.1.1.cmml" xref="S4.T1.35.35.3.m1.1.1"><lt id="S4.T1.35.35.3.m1.1.1.1.cmml" xref="S4.T1.35.35.3.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.35.35.3.m1.1.1.2.cmml" xref="S4.T1.35.35.3.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.35.35.3.m1.1.1.3.cmml" xref="S4.T1.35.35.3.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.35.35.3.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.38.38" class="ltx_tr">
<td id="S4.T1.38.38.4" class="ltx_td ltx_align_left"><span id="S4.T1.38.38.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Rec-RBM</span></td>
<td id="S4.T1.38.38.5" class="ltx_td ltx_align_right"><span id="S4.T1.38.38.5.1" class="ltx_text" style="font-size:90%;">0.99</span></td>
<td id="S4.T1.36.36.1" class="ltx_td ltx_align_right"><math id="S4.T1.36.36.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.36.36.1.m1.1a"><mrow id="S4.T1.36.36.1.m1.1.1" xref="S4.T1.36.36.1.m1.1.1.cmml"><mi id="S4.T1.36.36.1.m1.1.1.2" xref="S4.T1.36.36.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.36.36.1.m1.1.1.1" xref="S4.T1.36.36.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.36.36.1.m1.1.1.3" xref="S4.T1.36.36.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.36.36.1.m1.1b"><apply id="S4.T1.36.36.1.m1.1.1.cmml" xref="S4.T1.36.36.1.m1.1.1"><lt id="S4.T1.36.36.1.m1.1.1.1.cmml" xref="S4.T1.36.36.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.36.36.1.m1.1.1.2.cmml" xref="S4.T1.36.36.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.36.36.1.m1.1.1.3.cmml" xref="S4.T1.36.36.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.36.36.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.38.38.6" class="ltx_td ltx_align_right"><span id="S4.T1.38.38.6.1" class="ltx_text" style="font-size:90%;">1.00</span></td>
<td id="S4.T1.37.37.2" class="ltx_td ltx_align_right"><math id="S4.T1.37.37.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.37.37.2.m1.1a"><mrow id="S4.T1.37.37.2.m1.1.1" xref="S4.T1.37.37.2.m1.1.1.cmml"><mi id="S4.T1.37.37.2.m1.1.1.2" xref="S4.T1.37.37.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.37.37.2.m1.1.1.1" xref="S4.T1.37.37.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.37.37.2.m1.1.1.3" xref="S4.T1.37.37.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.37.37.2.m1.1b"><apply id="S4.T1.37.37.2.m1.1.1.cmml" xref="S4.T1.37.37.2.m1.1.1"><lt id="S4.T1.37.37.2.m1.1.1.1.cmml" xref="S4.T1.37.37.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.37.37.2.m1.1.1.2.cmml" xref="S4.T1.37.37.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.37.37.2.m1.1.1.3.cmml" xref="S4.T1.37.37.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.37.37.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.38.38.7" class="ltx_td ltx_align_right"><span id="S4.T1.38.38.7.1" class="ltx_text" style="font-size:90%;">0.40</span></td>
<td id="S4.T1.38.38.3" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.38.38.3.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.38.38.3.m1.1a"><mrow id="S4.T1.38.38.3.m1.1.1" xref="S4.T1.38.38.3.m1.1.1.cmml"><mi id="S4.T1.38.38.3.m1.1.1.2" xref="S4.T1.38.38.3.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.38.38.3.m1.1.1.1" xref="S4.T1.38.38.3.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.38.38.3.m1.1.1.3" xref="S4.T1.38.38.3.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.38.38.3.m1.1b"><apply id="S4.T1.38.38.3.m1.1.1.cmml" xref="S4.T1.38.38.3.m1.1.1"><lt id="S4.T1.38.38.3.m1.1.1.1.cmml" xref="S4.T1.38.38.3.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.38.38.3.m1.1.1.2.cmml" xref="S4.T1.38.38.3.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.38.38.3.m1.1.1.3.cmml" xref="S4.T1.38.38.3.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.38.38.3.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.41.41" class="ltx_tr">
<td id="S4.T1.41.41.4" class="ltx_td ltx_align_left"><span id="S4.T1.41.41.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Rec-GAN</span></td>
<td id="S4.T1.41.41.5" class="ltx_td ltx_align_right"><span id="S4.T1.41.41.5.1" class="ltx_text" style="font-size:90%;">0.55</span></td>
<td id="S4.T1.39.39.1" class="ltx_td ltx_align_right"><math id="S4.T1.39.39.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.39.39.1.m1.1a"><mrow id="S4.T1.39.39.1.m1.1.1" xref="S4.T1.39.39.1.m1.1.1.cmml"><mi id="S4.T1.39.39.1.m1.1.1.2" xref="S4.T1.39.39.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.39.39.1.m1.1.1.1" xref="S4.T1.39.39.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.39.39.1.m1.1.1.3" xref="S4.T1.39.39.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.39.39.1.m1.1b"><apply id="S4.T1.39.39.1.m1.1.1.cmml" xref="S4.T1.39.39.1.m1.1.1"><lt id="S4.T1.39.39.1.m1.1.1.1.cmml" xref="S4.T1.39.39.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.39.39.1.m1.1.1.2.cmml" xref="S4.T1.39.39.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.39.39.1.m1.1.1.3.cmml" xref="S4.T1.39.39.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.39.39.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.41.41.6" class="ltx_td ltx_align_right"><span id="S4.T1.41.41.6.1" class="ltx_text" style="font-size:90%;">0.68</span></td>
<td id="S4.T1.40.40.2" class="ltx_td ltx_align_right"><math id="S4.T1.40.40.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.40.40.2.m1.1a"><mrow id="S4.T1.40.40.2.m1.1.1" xref="S4.T1.40.40.2.m1.1.1.cmml"><mi id="S4.T1.40.40.2.m1.1.1.2" xref="S4.T1.40.40.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.40.40.2.m1.1.1.1" xref="S4.T1.40.40.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.40.40.2.m1.1.1.3" xref="S4.T1.40.40.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.40.40.2.m1.1b"><apply id="S4.T1.40.40.2.m1.1.1.cmml" xref="S4.T1.40.40.2.m1.1.1"><lt id="S4.T1.40.40.2.m1.1.1.1.cmml" xref="S4.T1.40.40.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.40.40.2.m1.1.1.2.cmml" xref="S4.T1.40.40.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.40.40.2.m1.1.1.3.cmml" xref="S4.T1.40.40.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.40.40.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.41.41.7" class="ltx_td ltx_align_right"><span id="S4.T1.41.41.7.1" class="ltx_text" style="font-size:90%;">0.52</span></td>
<td id="S4.T1.41.41.3" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.41.41.3.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.41.41.3.m1.1a"><mrow id="S4.T1.41.41.3.m1.1.1" xref="S4.T1.41.41.3.m1.1.1.cmml"><mi id="S4.T1.41.41.3.m1.1.1.2" xref="S4.T1.41.41.3.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.41.41.3.m1.1.1.1" xref="S4.T1.41.41.3.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.41.41.3.m1.1.1.3" xref="S4.T1.41.41.3.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.41.41.3.m1.1b"><apply id="S4.T1.41.41.3.m1.1.1.cmml" xref="S4.T1.41.41.3.m1.1.1"><lt id="S4.T1.41.41.3.m1.1.1.1.cmml" xref="S4.T1.41.41.3.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.41.41.3.m1.1.1.2.cmml" xref="S4.T1.41.41.3.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.41.41.3.m1.1.1.3.cmml" xref="S4.T1.41.41.3.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.41.41.3.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.44.44" class="ltx_tr">
<td id="S4.T1.44.44.4" class="ltx_td ltx_align_left"><span id="S4.T1.44.44.4.1" class="ltx_text ltx_font_bold" style="font-size:90%;">WGAN</span></td>
<td id="S4.T1.44.44.5" class="ltx_td ltx_align_right"><span id="S4.T1.44.44.5.1" class="ltx_text" style="font-size:90%;">0.17</span></td>
<td id="S4.T1.42.42.1" class="ltx_td ltx_align_right"><math id="S4.T1.42.42.1.m1.1" class="ltx_Math" alttext="{&lt;}0.1" display="inline"><semantics id="S4.T1.42.42.1.m1.1a"><mrow id="S4.T1.42.42.1.m1.1.1" xref="S4.T1.42.42.1.m1.1.1.cmml"><mi id="S4.T1.42.42.1.m1.1.1.2" xref="S4.T1.42.42.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.42.42.1.m1.1.1.1" xref="S4.T1.42.42.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.42.42.1.m1.1.1.3" xref="S4.T1.42.42.1.m1.1.1.3.cmml">0.1</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.42.42.1.m1.1b"><apply id="S4.T1.42.42.1.m1.1.1.cmml" xref="S4.T1.42.42.1.m1.1.1"><lt id="S4.T1.42.42.1.m1.1.1.1.cmml" xref="S4.T1.42.42.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.42.42.1.m1.1.1.2.cmml" xref="S4.T1.42.42.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.42.42.1.m1.1.1.3.cmml" xref="S4.T1.42.42.1.m1.1.1.3">0.1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.42.42.1.m1.1c">{&lt;}0.1</annotation></semantics></math></td>
<td id="S4.T1.44.44.6" class="ltx_td ltx_align_right"><span id="S4.T1.44.44.6.1" class="ltx_text" style="font-size:90%;">0.39</span></td>
<td id="S4.T1.43.43.2" class="ltx_td ltx_align_right"><math id="S4.T1.43.43.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.43.43.2.m1.1a"><mrow id="S4.T1.43.43.2.m1.1.1" xref="S4.T1.43.43.2.m1.1.1.cmml"><mi id="S4.T1.43.43.2.m1.1.1.2" xref="S4.T1.43.43.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.43.43.2.m1.1.1.1" xref="S4.T1.43.43.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.43.43.2.m1.1.1.3" xref="S4.T1.43.43.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.43.43.2.m1.1b"><apply id="S4.T1.43.43.2.m1.1.1.cmml" xref="S4.T1.43.43.2.m1.1.1"><lt id="S4.T1.43.43.2.m1.1.1.1.cmml" xref="S4.T1.43.43.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.43.43.2.m1.1.1.2.cmml" xref="S4.T1.43.43.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.43.43.2.m1.1.1.3.cmml" xref="S4.T1.43.43.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.43.43.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.44.44.7" class="ltx_td ltx_align_right"><span id="S4.T1.44.44.7.1" class="ltx_text" style="font-size:90%;">0.45</span></td>
<td id="S4.T1.44.44.3" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.44.44.3.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.44.44.3.m1.1a"><mrow id="S4.T1.44.44.3.m1.1.1" xref="S4.T1.44.44.3.m1.1.1.cmml"><mi id="S4.T1.44.44.3.m1.1.1.2" xref="S4.T1.44.44.3.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.44.44.3.m1.1.1.1" xref="S4.T1.44.44.3.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.44.44.3.m1.1.1.3" xref="S4.T1.44.44.3.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.44.44.3.m1.1b"><apply id="S4.T1.44.44.3.m1.1.1.cmml" xref="S4.T1.44.44.3.m1.1.1"><lt id="S4.T1.44.44.3.m1.1.1.1.cmml" xref="S4.T1.44.44.3.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.44.44.3.m1.1.1.2.cmml" xref="S4.T1.44.44.3.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.44.44.3.m1.1.1.3.cmml" xref="S4.T1.44.44.3.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.44.44.3.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.51.57.6" class="ltx_tr">
<td id="S4.T1.51.57.6.1" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S4.T1.51.57.6.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Linkage Disequilibrium</span></td>
</tr>
<tr id="S4.T1.46.46" class="ltx_tr">
<td id="S4.T1.46.46.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S4.T1.46.46.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Recomb</span></td>
<td id="S4.T1.46.46.4" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.46.46.4.1" class="ltx_text" style="font-size:90%;">0.11</span></td>
<td id="S4.T1.45.45.1" class="ltx_td ltx_align_right ltx_border_t"><math id="S4.T1.45.45.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.45.45.1.m1.1a"><mrow id="S4.T1.45.45.1.m1.1.1" xref="S4.T1.45.45.1.m1.1.1.cmml"><mi id="S4.T1.45.45.1.m1.1.1.2" xref="S4.T1.45.45.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.45.45.1.m1.1.1.1" xref="S4.T1.45.45.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.45.45.1.m1.1.1.3" xref="S4.T1.45.45.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.45.45.1.m1.1b"><apply id="S4.T1.45.45.1.m1.1.1.cmml" xref="S4.T1.45.45.1.m1.1.1"><lt id="S4.T1.45.45.1.m1.1.1.1.cmml" xref="S4.T1.45.45.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.45.45.1.m1.1.1.2.cmml" xref="S4.T1.45.45.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.45.45.1.m1.1.1.3.cmml" xref="S4.T1.45.45.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.45.45.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.46.46.5" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.46.46.5.1" class="ltx_text" style="font-size:90%;">0.12</span></td>
<td id="S4.T1.46.46.2" class="ltx_td ltx_align_right ltx_border_t"><math id="S4.T1.46.46.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.46.46.2.m1.1a"><mrow id="S4.T1.46.46.2.m1.1.1" xref="S4.T1.46.46.2.m1.1.1.cmml"><mi id="S4.T1.46.46.2.m1.1.1.2" xref="S4.T1.46.46.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.46.46.2.m1.1.1.1" xref="S4.T1.46.46.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.46.46.2.m1.1.1.3" xref="S4.T1.46.46.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.46.46.2.m1.1b"><apply id="S4.T1.46.46.2.m1.1.1.cmml" xref="S4.T1.46.46.2.m1.1.1"><lt id="S4.T1.46.46.2.m1.1.1.1.cmml" xref="S4.T1.46.46.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.46.46.2.m1.1.1.2.cmml" xref="S4.T1.46.46.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.46.46.2.m1.1.1.3.cmml" xref="S4.T1.46.46.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.46.46.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.46.46.6" class="ltx_td ltx_align_right ltx_border_t"><span id="S4.T1.46.46.6.1" class="ltx_text" style="font-size:90%;">0.31</span></td>
<td id="S4.T1.46.46.7" class="ltx_td ltx_nopad_r ltx_align_right ltx_border_t"><span id="S4.T1.46.46.7.1" class="ltx_text" style="font-size:90%;">0</span></td>
</tr>
<tr id="S4.T1.47.47" class="ltx_tr">
<td id="S4.T1.47.47.2" class="ltx_td ltx_align_left"><span id="S4.T1.47.47.2.1" class="ltx_text ltx_font_bold" style="font-size:90%;">RBM</span></td>
<td id="S4.T1.47.47.3" class="ltx_td ltx_align_right"><span id="S4.T1.47.47.3.1" class="ltx_text" style="font-size:90%;">0.61</span></td>
<td id="S4.T1.47.47.4" class="ltx_td ltx_align_right"><span id="S4.T1.47.47.4.1" class="ltx_text" style="font-size:90%;">0</span></td>
<td id="S4.T1.47.47.5" class="ltx_td ltx_align_right"><span id="S4.T1.47.47.5.1" class="ltx_text" style="font-size:90%;">0.69</span></td>
<td id="S4.T1.47.47.6" class="ltx_td ltx_align_right"><span id="S4.T1.47.47.6.1" class="ltx_text" style="font-size:90%;">0</span></td>
<td id="S4.T1.47.47.7" class="ltx_td ltx_align_right"><span id="S4.T1.47.47.7.1" class="ltx_text" style="font-size:90%;">0.08</span></td>
<td id="S4.T1.47.47.1" class="ltx_td ltx_nopad_r ltx_align_right"><math id="S4.T1.47.47.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.47.47.1.m1.1a"><mrow id="S4.T1.47.47.1.m1.1.1" xref="S4.T1.47.47.1.m1.1.1.cmml"><mi id="S4.T1.47.47.1.m1.1.1.2" xref="S4.T1.47.47.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.47.47.1.m1.1.1.1" xref="S4.T1.47.47.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.47.47.1.m1.1.1.3" xref="S4.T1.47.47.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.47.47.1.m1.1b"><apply id="S4.T1.47.47.1.m1.1.1.cmml" xref="S4.T1.47.47.1.m1.1.1"><lt id="S4.T1.47.47.1.m1.1.1.1.cmml" xref="S4.T1.47.47.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.47.47.1.m1.1.1.2.cmml" xref="S4.T1.47.47.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.47.47.1.m1.1.1.3.cmml" xref="S4.T1.47.47.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.47.47.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
</tr>
<tr id="S4.T1.49.49" class="ltx_tr">
<td id="S4.T1.49.49.3" class="ltx_td ltx_align_left"><span id="S4.T1.49.49.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">GAN</span></td>
<td id="S4.T1.49.49.4" class="ltx_td ltx_align_right"><span id="S4.T1.49.49.4.1" class="ltx_text" style="font-size:90%;">0.07</span></td>
<td id="S4.T1.48.48.1" class="ltx_td ltx_align_right"><math id="S4.T1.48.48.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.48.48.1.m1.1a"><mrow id="S4.T1.48.48.1.m1.1.1" xref="S4.T1.48.48.1.m1.1.1.cmml"><mi id="S4.T1.48.48.1.m1.1.1.2" xref="S4.T1.48.48.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.48.48.1.m1.1.1.1" xref="S4.T1.48.48.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.48.48.1.m1.1.1.3" xref="S4.T1.48.48.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.48.48.1.m1.1b"><apply id="S4.T1.48.48.1.m1.1.1.cmml" xref="S4.T1.48.48.1.m1.1.1"><lt id="S4.T1.48.48.1.m1.1.1.1.cmml" xref="S4.T1.48.48.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.48.48.1.m1.1.1.2.cmml" xref="S4.T1.48.48.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.48.48.1.m1.1.1.3.cmml" xref="S4.T1.48.48.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.48.48.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.49.49.5" class="ltx_td ltx_align_right"><span id="S4.T1.49.49.5.1" class="ltx_text" style="font-size:90%;">0.09</span></td>
<td id="S4.T1.49.49.2" class="ltx_td ltx_align_right"><math id="S4.T1.49.49.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.49.49.2.m1.1a"><mrow id="S4.T1.49.49.2.m1.1.1" xref="S4.T1.49.49.2.m1.1.1.cmml"><mi id="S4.T1.49.49.2.m1.1.1.2" xref="S4.T1.49.49.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.49.49.2.m1.1.1.1" xref="S4.T1.49.49.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.49.49.2.m1.1.1.3" xref="S4.T1.49.49.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.49.49.2.m1.1b"><apply id="S4.T1.49.49.2.m1.1.1.cmml" xref="S4.T1.49.49.2.m1.1.1"><lt id="S4.T1.49.49.2.m1.1.1.1.cmml" xref="S4.T1.49.49.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.49.49.2.m1.1.1.2.cmml" xref="S4.T1.49.49.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.49.49.2.m1.1.1.3.cmml" xref="S4.T1.49.49.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.49.49.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.49.49.6" class="ltx_td ltx_align_right"><span id="S4.T1.49.49.6.1" class="ltx_text" style="font-size:90%;">0.86</span></td>
<td id="S4.T1.49.49.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.49.49.7.1" class="ltx_text" style="font-size:90%;">0</span></td>
</tr>
<tr id="S4.T1.51.58.7" class="ltx_tr">
<td id="S4.T1.51.58.7.1" class="ltx_td ltx_align_left"><span id="S4.T1.51.58.7.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Rec-RBM</span></td>
<td id="S4.T1.51.58.7.2" class="ltx_td ltx_align_right"><span id="S4.T1.51.58.7.2.1" class="ltx_text" style="font-size:90%;">0.40</span></td>
<td id="S4.T1.51.58.7.3" class="ltx_td ltx_align_right"><span id="S4.T1.51.58.7.3.1" class="ltx_text" style="font-size:90%;">0</span></td>
<td id="S4.T1.51.58.7.4" class="ltx_td ltx_align_right"><span id="S4.T1.51.58.7.4.1" class="ltx_text" style="font-size:90%;">0.41</span></td>
<td id="S4.T1.51.58.7.5" class="ltx_td ltx_align_right"><span id="S4.T1.51.58.7.5.1" class="ltx_text" style="font-size:90%;">0</span></td>
<td id="S4.T1.51.58.7.6" class="ltx_td ltx_align_right"><span id="S4.T1.51.58.7.6.1" class="ltx_text" style="font-size:90%;">0.38</span></td>
<td id="S4.T1.51.58.7.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.51.58.7.7.1" class="ltx_text" style="font-size:90%;">0</span></td>
</tr>
<tr id="S4.T1.51.59.8" class="ltx_tr">
<td id="S4.T1.51.59.8.1" class="ltx_td ltx_align_left"><span id="S4.T1.51.59.8.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Rec-GAN</span></td>
<td id="S4.T1.51.59.8.2" class="ltx_td ltx_align_right"><span id="S4.T1.51.59.8.2.1" class="ltx_text" style="font-size:90%;">0.23</span></td>
<td id="S4.T1.51.59.8.3" class="ltx_td ltx_align_right"><span id="S4.T1.51.59.8.3.1" class="ltx_text" style="font-size:90%;">0</span></td>
<td id="S4.T1.51.59.8.4" class="ltx_td ltx_align_right"><span id="S4.T1.51.59.8.4.1" class="ltx_text" style="font-size:90%;">0.21</span></td>
<td id="S4.T1.51.59.8.5" class="ltx_td ltx_align_right"><span id="S4.T1.51.59.8.5.1" class="ltx_text" style="font-size:90%;">0</span></td>
<td id="S4.T1.51.59.8.6" class="ltx_td ltx_align_right"><span id="S4.T1.51.59.8.6.1" class="ltx_text" style="font-size:90%;">0.91</span></td>
<td id="S4.T1.51.59.8.7" class="ltx_td ltx_nopad_r ltx_align_right"><span id="S4.T1.51.59.8.7.1" class="ltx_text" style="font-size:90%;">0</span></td>
</tr>
<tr id="S4.T1.51.51" class="ltx_tr">
<td id="S4.T1.51.51.3" class="ltx_td ltx_align_left ltx_border_bb"><span id="S4.T1.51.51.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;">WGAN</span></td>
<td id="S4.T1.51.51.4" class="ltx_td ltx_align_right ltx_border_bb"><span id="S4.T1.51.51.4.1" class="ltx_text" style="font-size:90%;">0.10</span></td>
<td id="S4.T1.50.50.1" class="ltx_td ltx_align_right ltx_border_bb"><math id="S4.T1.50.50.1.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.50.50.1.m1.1a"><mrow id="S4.T1.50.50.1.m1.1.1" xref="S4.T1.50.50.1.m1.1.1.cmml"><mi id="S4.T1.50.50.1.m1.1.1.2" xref="S4.T1.50.50.1.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.50.50.1.m1.1.1.1" xref="S4.T1.50.50.1.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.50.50.1.m1.1.1.3" xref="S4.T1.50.50.1.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.50.50.1.m1.1b"><apply id="S4.T1.50.50.1.m1.1.1.cmml" xref="S4.T1.50.50.1.m1.1.1"><lt id="S4.T1.50.50.1.m1.1.1.1.cmml" xref="S4.T1.50.50.1.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.50.50.1.m1.1.1.2.cmml" xref="S4.T1.50.50.1.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.50.50.1.m1.1.1.3.cmml" xref="S4.T1.50.50.1.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.50.50.1.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.51.51.5" class="ltx_td ltx_align_right ltx_border_bb"><span id="S4.T1.51.51.5.1" class="ltx_text" style="font-size:90%;">0.09</span></td>
<td id="S4.T1.51.51.2" class="ltx_td ltx_align_right ltx_border_bb"><math id="S4.T1.51.51.2.m1.1" class="ltx_Math" alttext="{&lt;}0.001" display="inline"><semantics id="S4.T1.51.51.2.m1.1a"><mrow id="S4.T1.51.51.2.m1.1.1" xref="S4.T1.51.51.2.m1.1.1.cmml"><mi id="S4.T1.51.51.2.m1.1.1.2" xref="S4.T1.51.51.2.m1.1.1.2.cmml"></mi><mo mathsize="90%" id="S4.T1.51.51.2.m1.1.1.1" xref="S4.T1.51.51.2.m1.1.1.1.cmml">&lt;</mo><mn mathsize="90%" id="S4.T1.51.51.2.m1.1.1.3" xref="S4.T1.51.51.2.m1.1.1.3.cmml">0.001</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.51.51.2.m1.1b"><apply id="S4.T1.51.51.2.m1.1.1.cmml" xref="S4.T1.51.51.2.m1.1.1"><lt id="S4.T1.51.51.2.m1.1.1.1.cmml" xref="S4.T1.51.51.2.m1.1.1.1"></lt><csymbol cd="latexml" id="S4.T1.51.51.2.m1.1.1.2.cmml" xref="S4.T1.51.51.2.m1.1.1.2">absent</csymbol><cn type="float" id="S4.T1.51.51.2.m1.1.1.3.cmml" xref="S4.T1.51.51.2.m1.1.1.3">0.001</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.51.51.2.m1.1c">{&lt;}0.001</annotation></semantics></math></td>
<td id="S4.T1.51.51.6" class="ltx_td ltx_align_right ltx_border_bb"><span id="S4.T1.51.51.6.1" class="ltx_text" style="font-size:90%;">0.53</span></td>
<td id="S4.T1.51.51.7" class="ltx_td ltx_nopad_r ltx_align_right ltx_border_bb"><span id="S4.T1.51.51.7.1" class="ltx_text" style="font-size:90%;">0</span></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:90%;"><span class="ltx_tag ltx_tag_table">Table 1: </span>Two-sample (real vs. synthetic data) Kolmogorov-Smirnov test performed on the the main utility summary statistics presented in Section <a href="#S4" title="4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.</figcaption>
</figure>
<div id="S4.SS1.p1" class="ltx_para ltx_noindent">
<p id="S4.SS1.p1.1" class="ltx_p"><span id="S4.SS1.p1.1.1" class="ltx_text ltx_font_bold">Major Allele Frequency (MAF).</span> In population genetics, the <em id="S4.SS1.p1.1.2" class="ltx_emph ltx_font_italic">major allele frequency</em> (MAF) is routinely used to provide helpful information to differentiate between common and rare variants in the population, as it quantifies the frequency at which the most common allele occurs in a given population.
We start our utility analysis by comparing MAFs in the synthetic data vs. the real data.</p>
</div>
<div id="S4.SS1.p2" class="ltx_para">
<p id="S4.SS1.p2.1" class="ltx_p">In Fig. <a href="#S3.F1" title="Figure 1 ‣ 3 Synthetic Data Approaches in Genomics ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>, we plot the MAF at each position for the real datasets and the synthetic samples, over the CEU and CHB populations, and the 1000 Genomes dataset.
For CEU/CHB (Fig. <a href="#S3.F1.sf1" title="In Figure 1 ‣ 3 Synthetic Data Approaches in Genomics ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">1a</span></a>–<a href="#S3.F1.sf2" title="In Figure 1 ‣ 3 Synthetic Data Approaches in Genomics ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">1b</span></a>),the Recomb and WGAN replicate best the allele frequencies in the real data.
On the other hand, GAN and Rec-GAN fail to do so, and in fact, the generated samples seem random.
Even though not as close to the real frequencies as Recomb, the RBM model performs better than the GAN and Rec-GAN models.
In fact, RBM further improves when combined with Recomb (see Rec-RBM).</p>
</div>
<div id="S4.SS1.p3" class="ltx_para">
<p id="S4.SS1.p3.1" class="ltx_p">For 1000 Genomes (Fig. <a href="#S3.F1.sf3" title="In Figure 1 ‣ 3 Synthetic Data Approaches in Genomics ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">1c</span></a>),
Recomb’s MAF distribution is also similar to the real data’s.
However, RBM and Rec-RBM both display MAFs close to the real data, whereas, even with more training samples available, the GAN and Rec-GAN models still seem to produce random results.
Moreover, WGAN does not match the MAF distribution for this population as closely.
Overall, the difference in the MAF distributions across datasets is likely due to fewer samples available for the HapMap populations than the 1000 Genomes.
To further support our findings, we also compute the KS two-sample test <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib27" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">27</span></a>]</cite> on the statistics presented in this section (see Table <a href="#S4.T1" title="Table 1 ‣ 4.1 Allele Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>).
The test compares the agreement between the cumulative distributions of two independent samples.
If the resulting <math id="S4.SS1.p3.1.m1.1" class="ltx_Math" alttext="D" display="inline"><semantics id="S4.SS1.p3.1.m1.1a"><mi id="S4.SS1.p3.1.m1.1.1" xref="S4.SS1.p3.1.m1.1.1.cmml">D</mi><annotation-xml encoding="MathML-Content" id="S4.SS1.p3.1.m1.1b"><ci id="S4.SS1.p3.1.m1.1.1.cmml" xref="S4.SS1.p3.1.m1.1.1">𝐷</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.SS1.p3.1.m1.1c">D</annotation></semantics></math> value is low, and the p-value is high, we cannot reject the null hypothesis (i.e., there is no difference between the distributions).
For every two-samples test, the 95% critical value is approximately 0.195 (as we have 100 samples in each dataset), so we can reject the null hypothesis (that there is no difference between the distributions) for all synthetic data above this value.
For the CEU and CHB populations, we cannot reject the null hypothesis for samples generated by the Recomb and WGAN., whereas for the 1000 Genomes dataset we cannot reject the null hypothesis for Recomb, RBM and Rec-RBM.</p>
</div>
<div id="S4.SS1.p4" class="ltx_para ltx_noindent">
<p id="S4.SS1.p4.1" class="ltx_p"><span id="S4.SS1.p4.1.1" class="ltx_text ltx_font_bold">Alternate Allele Correlation (AAC).</span> To evaluate whether the real and synthetic data are <span id="S4.SS1.p4.1.2" class="ltx_text ltx_font_italic">genetically</span> different, in Appendix Fig. <a href="#S7.F12" title="Figure 12 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">12</span></a>, we plot the <em id="S4.SS1.p4.1.3" class="ltx_emph ltx_font_italic">alternate allele correlation</em> (AAC).
The more similar the two populations are, the closer the SNPs should be to the diagonal, as in the leftmost plots, where we have the real data against itself.
The strongest AAC is with the synthetic data generated by Recomb.
On the opposite side of the spectrum, the synthetic data generated by GAN and Rec-GAN have weak correlations.
For the CEU and CHB populations, Rec-RBM yields stronger AACs than simple RBM and the WGAN.
For the 1000 genomes dataset (Fig. <a href="#S7.F12.sf3" title="In Figure 12 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">12c</span></a>), there is a strong correlation between the alternate alleles for the real data and Recomb, RBM, Rec-RBM, and WGAN.</p>
</div>
<div id="S4.SS1.p5" class="ltx_para ltx_noindent">
<p id="S4.SS1.p5.1" class="ltx_p"><span id="S4.SS1.p5.1.1" class="ltx_text ltx_font_bold">Site Frequency Spectrum (SFS).</span> Another summary statistic that captures essential information about the underlying distribution of the allele frequencies of a given set of SNPs in a population or sample is the SFS <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib19" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">19</span></a>, <a href="#bib.bib18" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">18</span></a>]</cite>.
Basically, it provides a histogram whose size depends on the number of sequenced individuals.
In Fig. <a href="#S7.F13" title="Figure 13 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">13</span></a>, we plot the scaled folded SFS, which is the distribution of counts of minor alleles in a sample calculated over all segregating sites.
We scale this value so that a constant value is expected across the spectrum for neutral variation and constant population size, which yields the best visual comparisons.
If the distribution of allele frequencies for the synthetic samples matches that of the real data, we would expect to see the two spectra aligned.</p>
</div>
<div id="S4.SS1.p6" class="ltx_para">
<p id="S4.SS1.p6.1" class="ltx_p">With the HapMap populations (Fig. <a href="#S7.F13.sf1" title="In Figure 13 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">13a</span></a>– <a href="#S7.F13.sf2" title="In Figure 13 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">13b</span></a>), Rec-GAN suggests an excess of rare variants for a minor allele frequency around 0.1.
Whereas GAN seems to generate data closer to a neutral expectation, i.e., the synthetic dataset describes a more stable population.
Similarly, for the 1000 Genomes (Fig. <a href="#S7.F13.sf3" title="In Figure 13 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">13c</span></a>), Rec-GAN has an excess of rare variants for a minor allele frequency less than 0.1, and this is also displayed, at a lower scale, by the GAN-generated data.
After computing the KS-two sample test, we cannot reject the null hypothesis for the samples generated by the Recomb and the WGAN models, for both CEU and CHB populations.
We reject the null hypothesis for the 1000 Genomes dataset for synthetic data generated by the GAN and Rec-GAN.</p>
</div>
<figure id="S4.F2" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S4.F2.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x4.png" id="S4.F2.sf1.g1" class="ltx_graphics ltx_img_landscape" width="392" height="87" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU Population</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S4.F2.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x5.png" id="S4.F2.sf2.g1" class="ltx_graphics ltx_img_landscape" width="392" height="87" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB Population</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S4.F2.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x6.png" id="S4.F2.sf3.g1" class="ltx_graphics ltx_img_landscape" width="392" height="87" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(c) </span>1000 Genomes Population</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S4.F2.sf4" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x7.png" id="S4.F2.sf4.g1" class="ltx_graphics ltx_img_landscape" width="392" height="87" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(d) </span>1000 Genomes Population</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Percentage of heterozygous variants in each sample in the dataset for CEU, CHB populations, and the 1000 Genomes dataset.</figcaption>
</figure>
</section>
<section id="S4.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>Population Statistics</h3>

<div id="S4.SS2.p1" class="ltx_para">
<p id="S4.SS2.p1.1" class="ltx_p">Next, we study population statistics to determine how close to the real dataset is the synthetic data.
In particular, we look at the percentage of heterozygous variants, the fixation index, and the Euclidean Genetic Distance.</p>
</div>
<div id="S4.SS2.p2" class="ltx_para ltx_noindent">
<p id="S4.SS2.p2.1" class="ltx_p"><span id="S4.SS2.p2.1.1" class="ltx_text ltx_font_bold">Heterozygosity.</span> The condition of having two different alleles at a locus is denoted as heterozygosity.
The percentage of heterozygous variants is commonly used in population studies, as a low percentage of heterozygous variants implies less diversity in the population.
In Fig. <a href="#S4.F2.sf1" title="In Figure 2 ‣ 4.1 Allele Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">2a</span></a>–<a href="#S4.F2.sf2" title="In Figure 2 ‣ 4.1 Allele Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">2b</span></a>, we plot the percentage of heterozygous variants in each sample for the CEU/CHB populations, comparing the real statistics (blue/leftmost bars) vs. those computed on the synthetic data.
From a visual inspection, both CEU and CHB datasets, the Recomb and WGAN samples yield a similar distribution to the real data. However, when computing the KS statistic, we find that we cannot reject the null hypothesis only for those two models for the CEU datasets, and we reject the null hypothesis for all synthetic datasets generated from the CHB dataset.</p>
</div>
<div id="S4.SS2.p3" class="ltx_para">
<p id="S4.SS2.p3.1" class="ltx_p">For the GAN and RBM, the percentage of heterozygous samples decreases, suggesting that both models produce more homozygous variants.
Moreover, even though for the major allele frequencies, Rec-RBM produces variants with statistics closer to the real data, the percentage of heterozygous variants turns out to be the lowest for both populations.
By contrast, Rec-GAN produces a higher percentage of heterozygous variants than GAN, even though the major allele frequencies are not aligned with the original samples.
With the 1000 Genomes (Fig. <a href="#S4.F2.sf3" title="In Figure 2 ‣ 4.1 Allele Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">2c</span></a>), the % of heterozygous samples in the real data is lower across all samples.
Once again, and in line with previous results, GAN and the Rec-GAN significantly deviate from the %s of heterozygous samples found in the real data.
In fact, in Fig. <a href="#S4.F2.sf4" title="In Figure 2 ‣ 4.1 Allele Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">2d</span></a>, we remove the GAN and the Rec-GAN models in order to take a closer look at the other models; with more data samples, the samples generated yield percentages of heterozygous samples relatively similar to the real data. For this dataset, the only model for which we cannot reject the null hypothesis is the RBM.</p>
</div>
<div id="S4.SS2.p4" class="ltx_para ltx_noindent">
<p id="S4.SS2.p4.4" class="ltx_p"><span id="S4.SS2.p4.1.1" class="ltx_text ltx_font_bold">Fixation Index (<math id="S4.SS2.p4.1.1.m1.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.SS2.p4.1.1.m1.1a"><msub id="S4.SS2.p4.1.1.m1.1.1" xref="S4.SS2.p4.1.1.m1.1.1.cmml"><mi id="S4.SS2.p4.1.1.m1.1.1.2" xref="S4.SS2.p4.1.1.m1.1.1.2.cmml">F</mi><mrow id="S4.SS2.p4.1.1.m1.1.1.3" xref="S4.SS2.p4.1.1.m1.1.1.3.cmml"><mi id="S4.SS2.p4.1.1.m1.1.1.3.2" xref="S4.SS2.p4.1.1.m1.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p4.1.1.m1.1.1.3.1" xref="S4.SS2.p4.1.1.m1.1.1.3.1.cmml">​</mo><mi id="S4.SS2.p4.1.1.m1.1.1.3.3" xref="S4.SS2.p4.1.1.m1.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p4.1.1.m1.1b"><apply id="S4.SS2.p4.1.1.m1.1.1.cmml" xref="S4.SS2.p4.1.1.m1.1.1"><csymbol cd="ambiguous" id="S4.SS2.p4.1.1.m1.1.1.1.cmml" xref="S4.SS2.p4.1.1.m1.1.1">subscript</csymbol><ci id="S4.SS2.p4.1.1.m1.1.1.2.cmml" xref="S4.SS2.p4.1.1.m1.1.1.2">𝐹</ci><apply id="S4.SS2.p4.1.1.m1.1.1.3.cmml" xref="S4.SS2.p4.1.1.m1.1.1.3"><times id="S4.SS2.p4.1.1.m1.1.1.3.1.cmml" xref="S4.SS2.p4.1.1.m1.1.1.3.1"></times><ci id="S4.SS2.p4.1.1.m1.1.1.3.2.cmml" xref="S4.SS2.p4.1.1.m1.1.1.3.2">𝑆</ci><ci id="S4.SS2.p4.1.1.m1.1.1.3.3.cmml" xref="S4.SS2.p4.1.1.m1.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p4.1.1.m1.1c">F_{ST}</annotation></semantics></math>).</span> Another way to assess how <span id="S4.SS2.p4.4.2" class="ltx_text ltx_font_italic">different</span> are groups of populations from each other is to use the fixation index <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib28" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">28</span></a>]</cite>.
It provides a comparison of differences in allele frequency, with values ranging from 0 (not different) to 1 (completely different/no alleles in common).
In Fig. <a href="#S4.F3" title="Figure 3 ‣ 4.2 Population Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, we compare the <math id="S4.SS2.p4.2.m1.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.SS2.p4.2.m1.1a"><msub id="S4.SS2.p4.2.m1.1.1" xref="S4.SS2.p4.2.m1.1.1.cmml"><mi id="S4.SS2.p4.2.m1.1.1.2" xref="S4.SS2.p4.2.m1.1.1.2.cmml">F</mi><mrow id="S4.SS2.p4.2.m1.1.1.3" xref="S4.SS2.p4.2.m1.1.1.3.cmml"><mi id="S4.SS2.p4.2.m1.1.1.3.2" xref="S4.SS2.p4.2.m1.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p4.2.m1.1.1.3.1" xref="S4.SS2.p4.2.m1.1.1.3.1.cmml">​</mo><mi id="S4.SS2.p4.2.m1.1.1.3.3" xref="S4.SS2.p4.2.m1.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p4.2.m1.1b"><apply id="S4.SS2.p4.2.m1.1.1.cmml" xref="S4.SS2.p4.2.m1.1.1"><csymbol cd="ambiguous" id="S4.SS2.p4.2.m1.1.1.1.cmml" xref="S4.SS2.p4.2.m1.1.1">subscript</csymbol><ci id="S4.SS2.p4.2.m1.1.1.2.cmml" xref="S4.SS2.p4.2.m1.1.1.2">𝐹</ci><apply id="S4.SS2.p4.2.m1.1.1.3.cmml" xref="S4.SS2.p4.2.m1.1.1.3"><times id="S4.SS2.p4.2.m1.1.1.3.1.cmml" xref="S4.SS2.p4.2.m1.1.1.3.1"></times><ci id="S4.SS2.p4.2.m1.1.1.3.2.cmml" xref="S4.SS2.p4.2.m1.1.1.3.2">𝑆</ci><ci id="S4.SS2.p4.2.m1.1.1.3.3.cmml" xref="S4.SS2.p4.2.m1.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p4.2.m1.1c">F_{ST}</annotation></semantics></math> values for the real data against the synthetic samples.
For illustration purposes, we also include <math id="S4.SS2.p4.3.m2.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.SS2.p4.3.m2.1a"><msub id="S4.SS2.p4.3.m2.1.1" xref="S4.SS2.p4.3.m2.1.1.cmml"><mi id="S4.SS2.p4.3.m2.1.1.2" xref="S4.SS2.p4.3.m2.1.1.2.cmml">F</mi><mrow id="S4.SS2.p4.3.m2.1.1.3" xref="S4.SS2.p4.3.m2.1.1.3.cmml"><mi id="S4.SS2.p4.3.m2.1.1.3.2" xref="S4.SS2.p4.3.m2.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p4.3.m2.1.1.3.1" xref="S4.SS2.p4.3.m2.1.1.3.1.cmml">​</mo><mi id="S4.SS2.p4.3.m2.1.1.3.3" xref="S4.SS2.p4.3.m2.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p4.3.m2.1b"><apply id="S4.SS2.p4.3.m2.1.1.cmml" xref="S4.SS2.p4.3.m2.1.1"><csymbol cd="ambiguous" id="S4.SS2.p4.3.m2.1.1.1.cmml" xref="S4.SS2.p4.3.m2.1.1">subscript</csymbol><ci id="S4.SS2.p4.3.m2.1.1.2.cmml" xref="S4.SS2.p4.3.m2.1.1.2">𝐹</ci><apply id="S4.SS2.p4.3.m2.1.1.3.cmml" xref="S4.SS2.p4.3.m2.1.1.3"><times id="S4.SS2.p4.3.m2.1.1.3.1.cmml" xref="S4.SS2.p4.3.m2.1.1.3.1"></times><ci id="S4.SS2.p4.3.m2.1.1.3.2.cmml" xref="S4.SS2.p4.3.m2.1.1.3.2">𝑆</ci><ci id="S4.SS2.p4.3.m2.1.1.3.3.cmml" xref="S4.SS2.p4.3.m2.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p4.3.m2.1c">F_{ST}</annotation></semantics></math> of the real data against itself, which obviously yields <math id="S4.SS2.p4.4.m3.1" class="ltx_Math" alttext="F_{ST}=0" display="inline"><semantics id="S4.SS2.p4.4.m3.1a"><mrow id="S4.SS2.p4.4.m3.1.1" xref="S4.SS2.p4.4.m3.1.1.cmml"><msub id="S4.SS2.p4.4.m3.1.1.2" xref="S4.SS2.p4.4.m3.1.1.2.cmml"><mi id="S4.SS2.p4.4.m3.1.1.2.2" xref="S4.SS2.p4.4.m3.1.1.2.2.cmml">F</mi><mrow id="S4.SS2.p4.4.m3.1.1.2.3" xref="S4.SS2.p4.4.m3.1.1.2.3.cmml"><mi id="S4.SS2.p4.4.m3.1.1.2.3.2" xref="S4.SS2.p4.4.m3.1.1.2.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p4.4.m3.1.1.2.3.1" xref="S4.SS2.p4.4.m3.1.1.2.3.1.cmml">​</mo><mi id="S4.SS2.p4.4.m3.1.1.2.3.3" xref="S4.SS2.p4.4.m3.1.1.2.3.3.cmml">T</mi></mrow></msub><mo id="S4.SS2.p4.4.m3.1.1.1" xref="S4.SS2.p4.4.m3.1.1.1.cmml">=</mo><mn id="S4.SS2.p4.4.m3.1.1.3" xref="S4.SS2.p4.4.m3.1.1.3.cmml">0</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.SS2.p4.4.m3.1b"><apply id="S4.SS2.p4.4.m3.1.1.cmml" xref="S4.SS2.p4.4.m3.1.1"><eq id="S4.SS2.p4.4.m3.1.1.1.cmml" xref="S4.SS2.p4.4.m3.1.1.1"></eq><apply id="S4.SS2.p4.4.m3.1.1.2.cmml" xref="S4.SS2.p4.4.m3.1.1.2"><csymbol cd="ambiguous" id="S4.SS2.p4.4.m3.1.1.2.1.cmml" xref="S4.SS2.p4.4.m3.1.1.2">subscript</csymbol><ci id="S4.SS2.p4.4.m3.1.1.2.2.cmml" xref="S4.SS2.p4.4.m3.1.1.2.2">𝐹</ci><apply id="S4.SS2.p4.4.m3.1.1.2.3.cmml" xref="S4.SS2.p4.4.m3.1.1.2.3"><times id="S4.SS2.p4.4.m3.1.1.2.3.1.cmml" xref="S4.SS2.p4.4.m3.1.1.2.3.1"></times><ci id="S4.SS2.p4.4.m3.1.1.2.3.2.cmml" xref="S4.SS2.p4.4.m3.1.1.2.3.2">𝑆</ci><ci id="S4.SS2.p4.4.m3.1.1.2.3.3.cmml" xref="S4.SS2.p4.4.m3.1.1.2.3.3">𝑇</ci></apply></apply><cn type="integer" id="S4.SS2.p4.4.m3.1.1.3.cmml" xref="S4.SS2.p4.4.m3.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p4.4.m3.1c">F_{ST}=0</annotation></semantics></math>.</p>
</div>
<div id="S4.SS2.p5" class="ltx_para">
<p id="S4.SS2.p5.5" class="ltx_p">Recomb is once again the closest to the real data, which confirms the alignment from Fig. <a href="#S3.F1" title="Figure 1 ‣ 3 Synthetic Data Approaches in Genomics ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> of the allele frequencies of the synthetic recombination data with the real data.
The <math id="S4.SS2.p5.1.m1.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.SS2.p5.1.m1.1a"><msub id="S4.SS2.p5.1.m1.1.1" xref="S4.SS2.p5.1.m1.1.1.cmml"><mi id="S4.SS2.p5.1.m1.1.1.2" xref="S4.SS2.p5.1.m1.1.1.2.cmml">F</mi><mrow id="S4.SS2.p5.1.m1.1.1.3" xref="S4.SS2.p5.1.m1.1.1.3.cmml"><mi id="S4.SS2.p5.1.m1.1.1.3.2" xref="S4.SS2.p5.1.m1.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p5.1.m1.1.1.3.1" xref="S4.SS2.p5.1.m1.1.1.3.1.cmml">​</mo><mi id="S4.SS2.p5.1.m1.1.1.3.3" xref="S4.SS2.p5.1.m1.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p5.1.m1.1b"><apply id="S4.SS2.p5.1.m1.1.1.cmml" xref="S4.SS2.p5.1.m1.1.1"><csymbol cd="ambiguous" id="S4.SS2.p5.1.m1.1.1.1.cmml" xref="S4.SS2.p5.1.m1.1.1">subscript</csymbol><ci id="S4.SS2.p5.1.m1.1.1.2.cmml" xref="S4.SS2.p5.1.m1.1.1.2">𝐹</ci><apply id="S4.SS2.p5.1.m1.1.1.3.cmml" xref="S4.SS2.p5.1.m1.1.1.3"><times id="S4.SS2.p5.1.m1.1.1.3.1.cmml" xref="S4.SS2.p5.1.m1.1.1.3.1"></times><ci id="S4.SS2.p5.1.m1.1.1.3.2.cmml" xref="S4.SS2.p5.1.m1.1.1.3.2">𝑆</ci><ci id="S4.SS2.p5.1.m1.1.1.3.3.cmml" xref="S4.SS2.p5.1.m1.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p5.1.m1.1c">F_{ST}</annotation></semantics></math> value for the synthetic data produced by RBM is, for both CEU and CHB populations, less than 0.10; however, the hybrid Rec-RBM model further reduces this value to less than 0.04, and so does WGAN.
For both populations, data generated by GAN and Rec-GAN has the highest <math id="S4.SS2.p5.2.m2.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.SS2.p5.2.m2.1a"><msub id="S4.SS2.p5.2.m2.1.1" xref="S4.SS2.p5.2.m2.1.1.cmml"><mi id="S4.SS2.p5.2.m2.1.1.2" xref="S4.SS2.p5.2.m2.1.1.2.cmml">F</mi><mrow id="S4.SS2.p5.2.m2.1.1.3" xref="S4.SS2.p5.2.m2.1.1.3.cmml"><mi id="S4.SS2.p5.2.m2.1.1.3.2" xref="S4.SS2.p5.2.m2.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p5.2.m2.1.1.3.1" xref="S4.SS2.p5.2.m2.1.1.3.1.cmml">​</mo><mi id="S4.SS2.p5.2.m2.1.1.3.3" xref="S4.SS2.p5.2.m2.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p5.2.m2.1b"><apply id="S4.SS2.p5.2.m2.1.1.cmml" xref="S4.SS2.p5.2.m2.1.1"><csymbol cd="ambiguous" id="S4.SS2.p5.2.m2.1.1.1.cmml" xref="S4.SS2.p5.2.m2.1.1">subscript</csymbol><ci id="S4.SS2.p5.2.m2.1.1.2.cmml" xref="S4.SS2.p5.2.m2.1.1.2">𝐹</ci><apply id="S4.SS2.p5.2.m2.1.1.3.cmml" xref="S4.SS2.p5.2.m2.1.1.3"><times id="S4.SS2.p5.2.m2.1.1.3.1.cmml" xref="S4.SS2.p5.2.m2.1.1.3.1"></times><ci id="S4.SS2.p5.2.m2.1.1.3.2.cmml" xref="S4.SS2.p5.2.m2.1.1.3.2">𝑆</ci><ci id="S4.SS2.p5.2.m2.1.1.3.3.cmml" xref="S4.SS2.p5.2.m2.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p5.2.m2.1c">F_{ST}</annotation></semantics></math>, although, for the CHB population, the latter increases it, and for the CEU population reduces it.
Finally, for the 1000 genomes, Recomb, RBM, and Rec-RBM all have <math id="S4.SS2.p5.3.m3.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.SS2.p5.3.m3.1a"><msub id="S4.SS2.p5.3.m3.1.1" xref="S4.SS2.p5.3.m3.1.1.cmml"><mi id="S4.SS2.p5.3.m3.1.1.2" xref="S4.SS2.p5.3.m3.1.1.2.cmml">F</mi><mrow id="S4.SS2.p5.3.m3.1.1.3" xref="S4.SS2.p5.3.m3.1.1.3.cmml"><mi id="S4.SS2.p5.3.m3.1.1.3.2" xref="S4.SS2.p5.3.m3.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p5.3.m3.1.1.3.1" xref="S4.SS2.p5.3.m3.1.1.3.1.cmml">​</mo><mi id="S4.SS2.p5.3.m3.1.1.3.3" xref="S4.SS2.p5.3.m3.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p5.3.m3.1b"><apply id="S4.SS2.p5.3.m3.1.1.cmml" xref="S4.SS2.p5.3.m3.1.1"><csymbol cd="ambiguous" id="S4.SS2.p5.3.m3.1.1.1.cmml" xref="S4.SS2.p5.3.m3.1.1">subscript</csymbol><ci id="S4.SS2.p5.3.m3.1.1.2.cmml" xref="S4.SS2.p5.3.m3.1.1.2">𝐹</ci><apply id="S4.SS2.p5.3.m3.1.1.3.cmml" xref="S4.SS2.p5.3.m3.1.1.3"><times id="S4.SS2.p5.3.m3.1.1.3.1.cmml" xref="S4.SS2.p5.3.m3.1.1.3.1"></times><ci id="S4.SS2.p5.3.m3.1.1.3.2.cmml" xref="S4.SS2.p5.3.m3.1.1.3.2">𝑆</ci><ci id="S4.SS2.p5.3.m3.1.1.3.3.cmml" xref="S4.SS2.p5.3.m3.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p5.3.m3.1c">F_{ST}</annotation></semantics></math> close to the real data.
While still having a low <math id="S4.SS2.p5.4.m4.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.SS2.p5.4.m4.1a"><msub id="S4.SS2.p5.4.m4.1.1" xref="S4.SS2.p5.4.m4.1.1.cmml"><mi id="S4.SS2.p5.4.m4.1.1.2" xref="S4.SS2.p5.4.m4.1.1.2.cmml">F</mi><mrow id="S4.SS2.p5.4.m4.1.1.3" xref="S4.SS2.p5.4.m4.1.1.3.cmml"><mi id="S4.SS2.p5.4.m4.1.1.3.2" xref="S4.SS2.p5.4.m4.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p5.4.m4.1.1.3.1" xref="S4.SS2.p5.4.m4.1.1.3.1.cmml">​</mo><mi id="S4.SS2.p5.4.m4.1.1.3.3" xref="S4.SS2.p5.4.m4.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p5.4.m4.1b"><apply id="S4.SS2.p5.4.m4.1.1.cmml" xref="S4.SS2.p5.4.m4.1.1"><csymbol cd="ambiguous" id="S4.SS2.p5.4.m4.1.1.1.cmml" xref="S4.SS2.p5.4.m4.1.1">subscript</csymbol><ci id="S4.SS2.p5.4.m4.1.1.2.cmml" xref="S4.SS2.p5.4.m4.1.1.2">𝐹</ci><apply id="S4.SS2.p5.4.m4.1.1.3.cmml" xref="S4.SS2.p5.4.m4.1.1.3"><times id="S4.SS2.p5.4.m4.1.1.3.1.cmml" xref="S4.SS2.p5.4.m4.1.1.3.1"></times><ci id="S4.SS2.p5.4.m4.1.1.3.2.cmml" xref="S4.SS2.p5.4.m4.1.1.3.2">𝑆</ci><ci id="S4.SS2.p5.4.m4.1.1.3.3.cmml" xref="S4.SS2.p5.4.m4.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p5.4.m4.1c">F_{ST}</annotation></semantics></math>, WGAN has a slightly higher value.
Whereas, with GAN and Rec-GAN, <math id="S4.SS2.p5.5.m5.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.SS2.p5.5.m5.1a"><msub id="S4.SS2.p5.5.m5.1.1" xref="S4.SS2.p5.5.m5.1.1.cmml"><mi id="S4.SS2.p5.5.m5.1.1.2" xref="S4.SS2.p5.5.m5.1.1.2.cmml">F</mi><mrow id="S4.SS2.p5.5.m5.1.1.3" xref="S4.SS2.p5.5.m5.1.1.3.cmml"><mi id="S4.SS2.p5.5.m5.1.1.3.2" xref="S4.SS2.p5.5.m5.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.SS2.p5.5.m5.1.1.3.1" xref="S4.SS2.p5.5.m5.1.1.3.1.cmml">​</mo><mi id="S4.SS2.p5.5.m5.1.1.3.3" xref="S4.SS2.p5.5.m5.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p5.5.m5.1b"><apply id="S4.SS2.p5.5.m5.1.1.cmml" xref="S4.SS2.p5.5.m5.1.1"><csymbol cd="ambiguous" id="S4.SS2.p5.5.m5.1.1.1.cmml" xref="S4.SS2.p5.5.m5.1.1">subscript</csymbol><ci id="S4.SS2.p5.5.m5.1.1.2.cmml" xref="S4.SS2.p5.5.m5.1.1.2">𝐹</ci><apply id="S4.SS2.p5.5.m5.1.1.3.cmml" xref="S4.SS2.p5.5.m5.1.1.3"><times id="S4.SS2.p5.5.m5.1.1.3.1.cmml" xref="S4.SS2.p5.5.m5.1.1.3.1"></times><ci id="S4.SS2.p5.5.m5.1.1.3.2.cmml" xref="S4.SS2.p5.5.m5.1.1.3.2">𝑆</ci><ci id="S4.SS2.p5.5.m5.1.1.3.3.cmml" xref="S4.SS2.p5.5.m5.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p5.5.m5.1c">F_{ST}</annotation></semantics></math> significantly deviates from the real data, even with the increased number of samples of this dataset.</p>
</div>
<figure id="S4.F3" class="ltx_figure"><img src="/html/2102.03314/assets/x8.png" id="S4.F3.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="196" height="133" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>Fixation index values (<math id="S4.F3.2.m1.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S4.F3.2.m1.1b"><msub id="S4.F3.2.m1.1.1" xref="S4.F3.2.m1.1.1.cmml"><mi id="S4.F3.2.m1.1.1.2" xref="S4.F3.2.m1.1.1.2.cmml">F</mi><mrow id="S4.F3.2.m1.1.1.3" xref="S4.F3.2.m1.1.1.3.cmml"><mi id="S4.F3.2.m1.1.1.3.2" xref="S4.F3.2.m1.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S4.F3.2.m1.1.1.3.1" xref="S4.F3.2.m1.1.1.3.1.cmml">​</mo><mi id="S4.F3.2.m1.1.1.3.3" xref="S4.F3.2.m1.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S4.F3.2.m1.1c"><apply id="S4.F3.2.m1.1.1.cmml" xref="S4.F3.2.m1.1.1"><csymbol cd="ambiguous" id="S4.F3.2.m1.1.1.1.cmml" xref="S4.F3.2.m1.1.1">subscript</csymbol><ci id="S4.F3.2.m1.1.1.2.cmml" xref="S4.F3.2.m1.1.1.2">𝐹</ci><apply id="S4.F3.2.m1.1.1.3.cmml" xref="S4.F3.2.m1.1.1.3"><times id="S4.F3.2.m1.1.1.3.1.cmml" xref="S4.F3.2.m1.1.1.3.1"></times><ci id="S4.F3.2.m1.1.1.3.2.cmml" xref="S4.F3.2.m1.1.1.3.2">𝑆</ci><ci id="S4.F3.2.m1.1.1.3.3.cmml" xref="S4.F3.2.m1.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.F3.2.m1.1d">F_{ST}</annotation></semantics></math>) for the CEU and CHB populations, and the 1000 Genomes dataset.</figcaption>
</figure>
<div id="S4.SS2.p6" class="ltx_para ltx_noindent">
<p id="S4.SS2.p6.2" class="ltx_p"><span id="S4.SS2.p6.2.1" class="ltx_text ltx_font_bold">Euclidean Genetic Distance (EGD).</span> Since the fixation index does not easily allow for pairwise comparisons among populations, in Appendix Fig. <a href="#S7.F14" title="Figure 14 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">14</span></a>, we plot the Euclidean Genetic Distance (EGD) between the samples in each dataset.
EGD is routinely used as a measure of divergence between populations and shows the number of differences, or mutations, between two populations; a distance of <math id="S4.SS2.p6.1.m1.1" class="ltx_Math" alttext="0" display="inline"><semantics id="S4.SS2.p6.1.m1.1a"><mn id="S4.SS2.p6.1.m1.1.1" xref="S4.SS2.p6.1.m1.1.1.cmml">0</mn><annotation-xml encoding="MathML-Content" id="S4.SS2.p6.1.m1.1b"><cn type="integer" id="S4.SS2.p6.1.m1.1.1.cmml" xref="S4.SS2.p6.1.m1.1.1">0</cn></annotation-xml></semantics></math> means there is no difference in the results, i.e., there is an exact match.
From Fig. <a href="#S7.F14.sf1" title="In Figure 14 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">14a</span></a>–<a href="#S7.F14.sf2" title="In Figure 14 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">14b</span></a>, where the EGD on the diagonal is <math id="S4.SS2.p6.2.m2.1" class="ltx_Math" alttext="0" display="inline"><semantics id="S4.SS2.p6.2.m2.1a"><mn id="S4.SS2.p6.2.m2.1.1" xref="S4.SS2.p6.2.m2.1.1.cmml">0</mn><annotation-xml encoding="MathML-Content" id="S4.SS2.p6.2.m2.1b"><cn type="integer" id="S4.SS2.p6.2.m2.1.1.cmml" xref="S4.SS2.p6.2.m2.1.1">0</cn></annotation-xml></semantics></math>, we observe that, for both CEU and CHB populations, the synthetic samples generated by GAN are closer to each other than by the other models.
Rec-GAN generates samples with EGD close to 0, suggesting that there are very few differences between them and samples with a distance of around 30.
As for the other population statistics, Recomb generates samples that match the differences observed in the real data the closest for both populations.
For RBM, the samples generated have fewer differences than the real data.
Perhaps more interestingly, Rec-RBM yields samples with a higher divergence than the real data; this can be a consequence of the low percentage of heterozygous samples found in the synthetic samples generated by this model (recall Fig. <a href="#S4.F2" title="Figure 2 ‣ 4.1 Allele Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>).
The samples from WGAN match some of the differences observed in the real data, but the model also yields a few samples with a higher divergence.</p>
</div>
<figure id="S4.F4" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S4.F4.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/graphs/plots/ld_CEU.png" id="S4.F4.sf1.g1" class="ltx_graphics ltx_img_landscape" width="592" height="97" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S4.F4.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/graphs/plots/ld_CHB.png" id="S4.F4.sf2.g1" class="ltx_graphics ltx_img_landscape" width="592" height="97" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S4.F4.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/graphs/plots/ld_1000g.png" id="S4.F4.sf3.g1" class="ltx_graphics ltx_img_landscape" width="592" height="97" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(c) </span>1000 Genomes</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 4: </span>Pairwise Linkage Disequilibrium for Real vs. Synthetic Samples.</figcaption>
</figure>
<div id="S4.SS2.p7" class="ltx_para">
<p id="S4.SS2.p7.1" class="ltx_p">Finally, for the 1000 Genomes (Fig. <a href="#S7.F14.sf3" title="In Figure 14 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">14c</span></a>), all samples in the real data have closer EGDs between each other.
In fact, the samples generated by RBM yield a similar pattern in the EGD distances.
Although Recomb, Rec-RBM, and WGAN do too, they exhibit a lower distance, on average, between samples.
As for CEU/CHB populations, GAN and Rec-GAN models fail to capture the differences between samples.</p>
</div>
</section>
<section id="S4.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.3 </span>Linkage Disequilibrium (LD) Analysis</h3>

<div id="S4.SS3.p1" class="ltx_para">
<p id="S4.SS3.p1.1" class="ltx_p">Linkage disequilibrium (LD) captures the non-random association of alleles at two or more positions in a general population – i.e., those alleles do not occur randomly with respect to each other.
In Genome-Wide Association Studies, LD allows researchers to optimize genetic studies, e.g., by preventing genotyping SNPs that provide redundant information <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib9" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">9</span></a>]</cite>.
In Fig. <a href="#S4.F4" title="Figure 4 ‣ 4.2 Population Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>,
we plot the <math id="S4.SS3.p1.1.m1.1" class="ltx_Math" alttext="r^{2}" display="inline"><semantics id="S4.SS3.p1.1.m1.1a"><msup id="S4.SS3.p1.1.m1.1.1" xref="S4.SS3.p1.1.m1.1.1.cmml"><mi id="S4.SS3.p1.1.m1.1.1.2" xref="S4.SS3.p1.1.m1.1.1.2.cmml">r</mi><mn id="S4.SS3.p1.1.m1.1.1.3" xref="S4.SS3.p1.1.m1.1.1.3.cmml">2</mn></msup><annotation-xml encoding="MathML-Content" id="S4.SS3.p1.1.m1.1b"><apply id="S4.SS3.p1.1.m1.1.1.cmml" xref="S4.SS3.p1.1.m1.1.1"><csymbol cd="ambiguous" id="S4.SS3.p1.1.m1.1.1.1.cmml" xref="S4.SS3.p1.1.m1.1.1">superscript</csymbol><ci id="S4.SS3.p1.1.m1.1.1.2.cmml" xref="S4.SS3.p1.1.m1.1.1.2">𝑟</ci><cn type="integer" id="S4.SS3.p1.1.m1.1.1.3.cmml" xref="S4.SS3.p1.1.m1.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS3.p1.1.m1.1c">r^{2}</annotation></semantics></math> value for LD based on the Rogers-Huff method <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib50" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">50</span></a>]</cite>.
This ranges from 0 (there is no LD between the 2 SNPs) to 1 (the SNPs are in complete LD, i.e., the two SNPs have not been separated by recombination and have the same allele frequencies).</p>
</div>
<div id="S4.SS3.p2" class="ltx_para">
<p id="S4.SS3.p2.1" class="ltx_p">For CEU and CHB populations, RBM generates samples that display a stronger LD than the real data.
With more training samples, Rec-RBM yields a weaker LD but still stronger than the real data for both models.
On the other side of the spectrum, for Rec-GAN, the LD for the synthetic data is the weakest. For the 1000 Genomes, we find a stronger LD between the real samples than with the other two datasets.
RBM generates samples that are almost indistinguishable from the real data in terms of LD.
The LD in the synthetic datasets generated by Recomb, Rec-RBM, and WGAN have lower correlations than RBM, with GAN and Rec-GAN both failing to preserve the LD. In this case, the non-parametric KS test yields more surprising results for both the CEU and CHB populations. We find that we cannot reject the null hypothesis for samples generated by Recomb, GAN and WGAN. This implies that, even though the GAN model does not preserve well the allele frequencies or the population statistics, it manages to preserve the correlations between alleles.</p>
</div>
<div id="S4.SS3.p3" class="ltx_para">
<p id="S4.SS3.p3.1" class="ltx_p">For the 1000 Genomes data, the only generative models that yields samples which cannot be rejected under the null hypothesis is the RBM.</p>
</div>
</section>
<section id="S4.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.4 </span>Takeaways</h3>

<div id="S4.SS4.p1" class="ltx_para">
<p id="S4.SS4.p1.1" class="ltx_p">Our utility evaluation shows that there are only a handful of cases where generative models produce synthetic genomic data with high utility on popular tasks.</p>
</div>
<div id="S4.SS4.p2" class="ltx_para">
<p id="S4.SS4.p2.1" class="ltx_p">The Recomb model, which is based on high-order SNV correlations, generates synthetic data preserving most statistical properties displayed by the real data, even when few samples are available.
We get better utility when the genetic map is included with the data rather than generated from the existing data.
This conclusion is also supported by the fact that we cannot reject the null hypothesis for this model for most statistics analyzed (with the exception of the percentage of heterozygous samples for the CHB).</p>
</div>
<div id="S4.SS4.p3" class="ltx_para">
<p id="S4.SS4.p3.1" class="ltx_p">With RBM, more training samples improve the quality of the synthetic data, as evidenced by the difference between the HapMap populations and the 1000 Genomes dataset.</p>
</div>
<div id="S4.SS4.p4" class="ltx_para">
<p id="S4.SS4.p4.1" class="ltx_p">We also find that when few samples are available for training, the hybrid Rec-RBM model approach helps improve the quality of samples compared to just RBM, but the samples are not as close to the real data as for Recomb.
This is clear from the utility of the synthetic data on the two smaller HapMap datasets.
For the 1000 genomes, it is not surprising that Rec-RBM’s performance is worse than RBM since Recomb does not generate as “useful” samples as for the other two datasets.
Finally, the GAN and the Rec-GAN models generate samples with the lowest utility, regardless of the number of samples available for training. However, even though the allele and population statistics are now well preserved, our non-parametric test showcases that the correlations between alleles are preserved for the LD.
In contrast to the other GAN models studied, the data generated by WGAN preserves most statistical properties of the real data.</p>
</div>
<div id="S4.SS4.p5" class="ltx_para">
<p id="S4.SS4.p5.1" class="ltx_p">Overall, our analysis exposes the limitations, in terms of statistical utility, of using generative machine learning models to produce synthetic genomic data.</p>
</div>
</section>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Privacy Evaluation</h2>

<div id="S5.p1" class="ltx_para">
<p id="S5.p1.2" class="ltx_p">Next, we quantify the privacy provided by synthetic data by evaluating its vulnerability to Membership Inference Attacks (MIAs).
More precisely, we compute the Privacy Gain, PG, obtained by releasing a synthetic dataset instead of the real data (see Section <a href="#S2.SS3" title="2.3 Privacy Gain (PG) from Synthetic Data Release ‣ 2 Preliminaries ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.3</span></a>). Recall that if the synthetic data does not hide nor give any additional information to an MIA attacker, <math id="S5.p1.1.m1.1" class="ltx_Math" alttext="PG_{t}" display="inline"><semantics id="S5.p1.1.m1.1a"><mrow id="S5.p1.1.m1.1.1" xref="S5.p1.1.m1.1.1.cmml"><mi id="S5.p1.1.m1.1.1.2" xref="S5.p1.1.m1.1.1.2.cmml">P</mi><mo lspace="0em" rspace="0em" id="S5.p1.1.m1.1.1.1" xref="S5.p1.1.m1.1.1.1.cmml">​</mo><msub id="S5.p1.1.m1.1.1.3" xref="S5.p1.1.m1.1.1.3.cmml"><mi id="S5.p1.1.m1.1.1.3.2" xref="S5.p1.1.m1.1.1.3.2.cmml">G</mi><mi id="S5.p1.1.m1.1.1.3.3" xref="S5.p1.1.m1.1.1.3.3.cmml">t</mi></msub></mrow><annotation-xml encoding="MathML-Content" id="S5.p1.1.m1.1b"><apply id="S5.p1.1.m1.1.1.cmml" xref="S5.p1.1.m1.1.1"><times id="S5.p1.1.m1.1.1.1.cmml" xref="S5.p1.1.m1.1.1.1"></times><ci id="S5.p1.1.m1.1.1.2.cmml" xref="S5.p1.1.m1.1.1.2">𝑃</ci><apply id="S5.p1.1.m1.1.1.3.cmml" xref="S5.p1.1.m1.1.1.3"><csymbol cd="ambiguous" id="S5.p1.1.m1.1.1.3.1.cmml" xref="S5.p1.1.m1.1.1.3">subscript</csymbol><ci id="S5.p1.1.m1.1.1.3.2.cmml" xref="S5.p1.1.m1.1.1.3.2">𝐺</ci><ci id="S5.p1.1.m1.1.1.3.3.cmml" xref="S5.p1.1.m1.1.1.3.3">𝑡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.p1.1.m1.1c">PG_{t}</annotation></semantics></math>, for a target record <math id="S5.p1.2.m2.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S5.p1.2.m2.1a"><mi id="S5.p1.2.m2.1.1" xref="S5.p1.2.m2.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S5.p1.2.m2.1b"><ci id="S5.p1.2.m2.1.1.cmml" xref="S5.p1.2.m2.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.p1.2.m2.1c">t</annotation></semantics></math>, should have a value of around 0.25.</p>
</div>
<div id="S5.p2" class="ltx_para">
<p id="S5.p2.1" class="ltx_p">We present experiments for both a “standard” MIA and a novel attack, which we denote as <span id="S5.p2.1.1" class="ltx_text ltx_font_italic">MIA with partial information.</span>
The latter essentially assumes that the adversary only has access to partial data from the target sequence.
We exclude GAN and Rec-GAN from the evaluation since they yield poor utility performance, so there is not really any point in evaluating their privacy.</p>
</div>
<div id="S5.p3" class="ltx_para">
<p id="S5.p3.1" class="ltx_p">Throughout our evaluation, we randomly choose 10 targets from each dataset across 10 test runs.
In each run, we fix the target and sample a new training cohort.
We train the attack classifier using 5 shadow models, using 100 synthetic training sets for each of them.
We then compute the PG on 100 synthetic datasets, with a split of 50 sets generated from a training set including the target and 50 sets generated without.
Finally, we report the PG for each test and each target as the average PG across all synthetic datasets tested.</p>
</div>
<section id="S5.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.1 </span>Privacy Gain Under Membership Inference Attack</h3>

<div id="S5.SS1.p1" class="ltx_para ltx_noindent">
<p id="S5.SS1.p1.1" class="ltx_p"><span id="S5.SS1.p1.1.1" class="ltx_text ltx_font_bold">Threat Model.</span> We assume that the adversary has access to the target, to the synthetic datasets, as well as public datasets with similar distribution to the original data (which may or may not include the target) for training his shadow models. The goal of the adversary is to identify whether the target sequence was used for training the model that generated the synthetic dataset. This could lead to the leakage of additional information about the target, as for example if a synthetic dataset of cancer patients is released and the adversary can identify that the target is part of the training set, it would lead to exposing the cancer status of the target.</p>
</div>
<div id="S5.SS1.p2" class="ltx_para">
<p id="S5.SS1.p2.8" class="ltx_p">We use three adversarial classifiers: K-Nearest Neighbor (KNN), Logistic Regression (LogReg), and Random Forest (RandForest).
We use four feature sets, as described in Section <a href="#S2.SS3" title="2.3 Privacy Gain (PG) from Synthetic Data Release ‣ 2 Preliminaries ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.3</span></a>: Naive ( <svg id="S5.SS1.p2.1.pic1" class="ltx_picture" height="6.09" overflow="visible" version="1.1" width="6.09"><g transform="translate(0,6.09) matrix(1 0 0 -1 0 0) translate(3.04,0) translate(0,3.04)" fill="#1F77B4" stroke="#1F77B4" stroke-width="0.4pt" color="#1F77B4"><path d="M 0 0 M 2.77 0 C 2.77 1.53 1.53 2.77 0 2.77 C -1.53 2.77 -2.77 1.53 -2.77 0 C -2.77 -1.53 -1.53 -2.77 0 -2.77 C 1.53 -2.77 2.77 -1.53 2.77 0 Z M 0 0"></path></g></svg><math id="S5.SS1.p2.2.m1.1" class="ltx_Math" alttext="\mathtt{F_{Naive}}" display="inline"><semantics id="S5.SS1.p2.2.m1.1a"><msub id="S5.SS1.p2.2.m1.1.1" xref="S5.SS1.p2.2.m1.1.1.cmml"><mi id="S5.SS1.p2.2.m1.1.1.2" xref="S5.SS1.p2.2.m1.1.1.2.cmml">𝙵</mi><mi id="S5.SS1.p2.2.m1.1.1.3" xref="S5.SS1.p2.2.m1.1.1.3.cmml">𝙽𝚊𝚒𝚟𝚎</mi></msub><annotation-xml encoding="MathML-Content" id="S5.SS1.p2.2.m1.1b"><apply id="S5.SS1.p2.2.m1.1.1.cmml" xref="S5.SS1.p2.2.m1.1.1"><csymbol cd="ambiguous" id="S5.SS1.p2.2.m1.1.1.1.cmml" xref="S5.SS1.p2.2.m1.1.1">subscript</csymbol><ci id="S5.SS1.p2.2.m1.1.1.2.cmml" xref="S5.SS1.p2.2.m1.1.1.2">𝙵</ci><ci id="S5.SS1.p2.2.m1.1.1.3.cmml" xref="S5.SS1.p2.2.m1.1.1.3">𝙽𝚊𝚒𝚟𝚎</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p2.2.m1.1c">\mathtt{F_{Naive}}</annotation></semantics></math>), Histogram ( <svg id="S5.SS1.p2.3.pic2" class="ltx_picture" height="6.09" overflow="visible" version="1.1" width="6.09"><g transform="translate(0,6.09) matrix(1 0 0 -1 0 0) translate(3.04,0) translate(0,3.04)" fill="#FF7F0E" stroke="#FF7F0E" stroke-width="0.4pt" color="#FF7F0E"><path d="M 0 0 M 2.77 0 C 2.77 1.53 1.53 2.77 0 2.77 C -1.53 2.77 -2.77 1.53 -2.77 0 C -2.77 -1.53 -1.53 -2.77 0 -2.77 C 1.53 -2.77 2.77 -1.53 2.77 0 Z M 0 0"></path></g></svg><math id="S5.SS1.p2.4.m2.1" class="ltx_Math" alttext="\mathtt{F_{Hist}}" display="inline"><semantics id="S5.SS1.p2.4.m2.1a"><msub id="S5.SS1.p2.4.m2.1.1" xref="S5.SS1.p2.4.m2.1.1.cmml"><mi id="S5.SS1.p2.4.m2.1.1.2" xref="S5.SS1.p2.4.m2.1.1.2.cmml">𝙵</mi><mi id="S5.SS1.p2.4.m2.1.1.3" xref="S5.SS1.p2.4.m2.1.1.3.cmml">𝙷𝚒𝚜𝚝</mi></msub><annotation-xml encoding="MathML-Content" id="S5.SS1.p2.4.m2.1b"><apply id="S5.SS1.p2.4.m2.1.1.cmml" xref="S5.SS1.p2.4.m2.1.1"><csymbol cd="ambiguous" id="S5.SS1.p2.4.m2.1.1.1.cmml" xref="S5.SS1.p2.4.m2.1.1">subscript</csymbol><ci id="S5.SS1.p2.4.m2.1.1.2.cmml" xref="S5.SS1.p2.4.m2.1.1.2">𝙵</ci><ci id="S5.SS1.p2.4.m2.1.1.3.cmml" xref="S5.SS1.p2.4.m2.1.1.3">𝙷𝚒𝚜𝚝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p2.4.m2.1c">\mathtt{F_{Hist}}</annotation></semantics></math> ), Correlations ( <svg id="S5.SS1.p2.5.pic3" class="ltx_picture" height="6.09" overflow="visible" version="1.1" width="6.09"><g transform="translate(0,6.09) matrix(1 0 0 -1 0 0) translate(3.04,0) translate(0,3.04)" fill="#2CA02C" stroke="#2CA02C" stroke-width="0.4pt" color="#2CA02C"><path d="M 0 0 M 2.77 0 C 2.77 1.53 1.53 2.77 0 2.77 C -1.53 2.77 -2.77 1.53 -2.77 0 C -2.77 -1.53 -1.53 -2.77 0 -2.77 C 1.53 -2.77 2.77 -1.53 2.77 0 Z M 0 0"></path></g></svg><math id="S5.SS1.p2.6.m3.1" class="ltx_Math" alttext="\mathtt{F_{Corr}}" display="inline"><semantics id="S5.SS1.p2.6.m3.1a"><msub id="S5.SS1.p2.6.m3.1.1" xref="S5.SS1.p2.6.m3.1.1.cmml"><mi id="S5.SS1.p2.6.m3.1.1.2" xref="S5.SS1.p2.6.m3.1.1.2.cmml">𝙵</mi><mi id="S5.SS1.p2.6.m3.1.1.3" xref="S5.SS1.p2.6.m3.1.1.3.cmml">𝙲𝚘𝚛𝚛</mi></msub><annotation-xml encoding="MathML-Content" id="S5.SS1.p2.6.m3.1b"><apply id="S5.SS1.p2.6.m3.1.1.cmml" xref="S5.SS1.p2.6.m3.1.1"><csymbol cd="ambiguous" id="S5.SS1.p2.6.m3.1.1.1.cmml" xref="S5.SS1.p2.6.m3.1.1">subscript</csymbol><ci id="S5.SS1.p2.6.m3.1.1.2.cmml" xref="S5.SS1.p2.6.m3.1.1.2">𝙵</ci><ci id="S5.SS1.p2.6.m3.1.1.3.cmml" xref="S5.SS1.p2.6.m3.1.1.3">𝙲𝚘𝚛𝚛</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p2.6.m3.1c">\mathtt{F_{Corr}}</annotation></semantics></math>), and an Ensemble feature set ( <svg id="S5.SS1.p2.7.pic4" class="ltx_picture" height="6.09" overflow="visible" version="1.1" width="6.09"><g transform="translate(0,6.09) matrix(1 0 0 -1 0 0) translate(3.04,0) translate(0,3.04)" fill="#D62728" stroke="#D62728" stroke-width="0.4pt" color="#D62728"><path d="M 0 0 M 2.77 0 C 2.77 1.53 1.53 2.77 0 2.77 C -1.53 2.77 -2.77 1.53 -2.77 0 C -2.77 -1.53 -1.53 -2.77 0 -2.77 C 1.53 -2.77 2.77 -1.53 2.77 0 Z M 0 0"></path></g></svg><math id="S5.SS1.p2.8.m4.1" class="ltx_Math" alttext="\mathtt{F_{Ens}}" display="inline"><semantics id="S5.SS1.p2.8.m4.1a"><msub id="S5.SS1.p2.8.m4.1.1" xref="S5.SS1.p2.8.m4.1.1.cmml"><mi id="S5.SS1.p2.8.m4.1.1.2" xref="S5.SS1.p2.8.m4.1.1.2.cmml">𝙵</mi><mi id="S5.SS1.p2.8.m4.1.1.3" xref="S5.SS1.p2.8.m4.1.1.3.cmml">𝙴𝚗𝚜</mi></msub><annotation-xml encoding="MathML-Content" id="S5.SS1.p2.8.m4.1b"><apply id="S5.SS1.p2.8.m4.1.1.cmml" xref="S5.SS1.p2.8.m4.1.1"><csymbol cd="ambiguous" id="S5.SS1.p2.8.m4.1.1.1.cmml" xref="S5.SS1.p2.8.m4.1.1">subscript</csymbol><ci id="S5.SS1.p2.8.m4.1.1.2.cmml" xref="S5.SS1.p2.8.m4.1.1.2">𝙵</ci><ci id="S5.SS1.p2.8.m4.1.1.3.cmml" xref="S5.SS1.p2.8.m4.1.1.3">𝙴𝚗𝚜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p2.8.m4.1c">\mathtt{F_{Ens}}</annotation></semantics></math>).</p>
</div>
<figure id="S5.F5" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/graphs/plots/legend.png" id="S5.F5.g1" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="150" height="19" alt="Refer to caption"></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S5.F5.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x9.png" id="S5.F5.sf1.g1" class="ltx_graphics ltx_img_landscape" width="415" height="89" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU Population</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S5.F5.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x10.png" id="S5.F5.sf2.g1" class="ltx_graphics ltx_img_landscape" width="415" height="91" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB Population</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 5: </span>Privacy Gain (PG) of different models over the two HapMap populations.</figcaption>
</figure>
<section id="S5.SS1.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.1.1 </span>HapMap Populations</h4>

<div id="S5.SS1.SSS1.p1" class="ltx_para">
<p id="S5.SS1.SSS1.p1.1" class="ltx_p">In Fig. <a href="#S5.F5" title="Figure 5 ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>, we report the PG value for targets randomly chosen from the two HapMap populations.</p>
</div>
<div id="S5.SS1.SSS1.p2" class="ltx_para ltx_noindent">
<p id="S5.SS1.SSS1.p2.1" class="ltx_p"><span id="S5.SS1.SSS1.p2.1.1" class="ltx_text ltx_font_bold">KNN.</span> For CEU, using KNN (Fig. <a href="#S5.F5.sf1" title="In Figure 5 ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5a</span></a>, left), we find that over 74% of the targets in the synthetic dataset generated by Recomb have a PG lower than the random baseline (0.25) for the Ensemble, Correlations, and Histogram feature sets.
With RBM, there are between 84% and 88% of the targets, depending on the feature set, that have a PG of 0.25; in other words, for these targets, the adversary’s probability of inferring their presence in the training set is the same as random guessing.
However, between 10% and 15% of the targets have no PG at all, whereas there are between 1% and 4% of the targets, depending on the feature set, for which the synthetic data perfectly protects the target from MIA (PG=0.5).
With Rec-RBM, at least 59% of the targets have a PG of 0.25 under the four feature sets.
With WGAN, at least 48% of the targets have a PG of 0.25, depending on the feature set.</p>
</div>
<div id="S5.SS1.SSS1.p3" class="ltx_para">
<p id="S5.SS1.SSS1.p3.1" class="ltx_p">For the CHB population (Fig. <a href="#S5.F5.sf2" title="In Figure 5 ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5b</span></a>, left), we find that over 60% of the targets generated by the Recomb, with all features, have PG below the random guess baseline.
With RBM, between 89% and 97% of targets have PG of exactly 0.25 across all feature sets, i.e., the synthetic dataset generated by the RBM for these targets does not hide nor give new information to the attacker about their membership to the synthetic dataset.
Interestingly, for the Correlations feature set, no target has PG lower than 0.25.
As for the CEU population, about 50% of the targets across all feature sets have a PG of 0.25 for data from Rec-RBM and at least 47% of targets from WGAN.</p>
</div>
<figure id="S5.F6" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_2"><img src="/html/2102.03314/assets/graphs/plots/legend.png" id="S5.F6.1.g1" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="150" height="19" alt="Refer to caption"></div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S5.F6.sf1" class="ltx_figure ltx_figure_panel"><img src="/html/2102.03314/assets/x11.png" id="S5.F6.sf1.g1" class="ltx_graphics ltx_img_landscape" width="184" height="105" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>Random Target</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S5.F6.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x12.png" id="S5.F6.sf2.g1" class="ltx_graphics ltx_img_landscape" width="184" height="105" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>Outlier target</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 6: </span>Privacy Gain (PG) of RBM over the 1000 Genomes dataset. </figcaption>
</figure>
<div id="S5.SS1.SSS1.p4" class="ltx_para ltx_noindent">
<p id="S5.SS1.SSS1.p4.1" class="ltx_p"><span id="S5.SS1.SSS1.p4.1.1" class="ltx_text ltx_font_bold">LogReg.</span> Using LogReg, both Recomb and RBM have the lowest PG among all attack classifiers for both HapMap populations.
For CEU (Fig. <a href="#S5.F5.sf1" title="In Figure 5 ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5a</span></a>, middle), using the Histogram feature set, 94% (resp., 96%) of the targets from Recomb (resp., RBM) have PG below 0.25, which is the random guess baseline.
Under Correlations, 99% (resp., 97%) of the targets in Recomb (resp., RBM) have PG below 0.25,
while, for the Ensemble feature set, 96% (resp., 98%) of the targets from Recomb (resp., RBM) have PG below 0.25. With Rec-RBM, we find that between 52% and 56% of the targets across all feature sets have PG above 0.25, and with WGAN, between 50% and 57% of the targets across all feature sets have PG above 0.25.
Moreover, for the Rec-RBM and WGAN-generated data, no target consistently has a lower PG than the random guess baseline across all test runs.</p>
</div>
<div id="S5.SS1.SSS1.p5" class="ltx_para">
<p id="S5.SS1.SSS1.p5.1" class="ltx_p">For CHB (Fig. <a href="#S5.F5.sf2" title="In Figure 5 ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5b</span></a>, middle), with synthetic data generated by Recomb, the average PG is below the random baseline (0.25) for 99% of the targets in the Histogram feature set, 97% for Correlations, and 96% for Ensemble. For RBM, 79% of the targets in the Histogram feature set have a PG below 0.25. Under the Ensemble feature set, 84% of the targets have PG below 0.25. For synthetic data generated by Rec-RBM, we find that 54% of the targets from the Histogram and Ensemble feature sets have PG lower than 0.25, and 46% have PG over 0.25.
For the Naive and Correlations feature sets, respectively, 45% and 47% of the targets have PG lower than the random guess baseline.
For WGAN-generated data, the Correlations feature set yields most targets (55%) with PG<math id="S5.SS1.SSS1.p5.1.m1.1" class="ltx_Math" alttext="&lt;" display="inline"><semantics id="S5.SS1.SSS1.p5.1.m1.1a"><mo id="S5.SS1.SSS1.p5.1.m1.1.1" xref="S5.SS1.SSS1.p5.1.m1.1.1.cmml">&lt;</mo><annotation-xml encoding="MathML-Content" id="S5.SS1.SSS1.p5.1.m1.1b"><lt id="S5.SS1.SSS1.p5.1.m1.1.1.cmml" xref="S5.SS1.SSS1.p5.1.m1.1.1"></lt></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.SSS1.p5.1.m1.1c">&lt;</annotation></semantics></math>0.25.</p>
</div>
<div id="S5.SS1.SSS1.p6" class="ltx_para ltx_noindent">
<p id="S5.SS1.SSS1.p6.1" class="ltx_p"><span id="S5.SS1.SSS1.p6.1.1" class="ltx_text ltx_font_bold">RandForest.</span> When using RandForest as the attack classifier on data from the CEU population (Fig. <a href="#S5.F5.sf1" title="In Figure 5 ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5a</span></a>, right), with RBM-generated data, 73% of the targets from both Correlation and Histogram feature sets have lower PG than the random baseline.
This is for 69% and 62% of the targets with, respectively, Ensemble and Naive feature sets.
For the synthetic data generated by Rec-RBM, about 51% of targets have a PG of over 0.25, and 49% of the targets have PG of less than 0.25 across all feature sets.
For WGAN, between 46% and 59% of the targets from all feature sets have a PG less than the random guess baseline (0.25), with the Correlations feature set having the least percentage of vulnerable targets (59%).</p>
</div>
<div id="S5.SS1.SSS1.p7" class="ltx_para">
<p id="S5.SS1.SSS1.p7.2" class="ltx_p">For CHB (Fig. <a href="#S5.F5.sf2" title="In Figure 5 ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5b</span></a>, right), the lowest privacy gain for the samples generated by Recomb: over 79% of all targets for each of the four feature sets have PG lower than the random baseline.
For the synthetic samples from RBM, 71%, 61%, and 53% of the targets under the Naive, Histogram, respectively, Ensemble feature sets have PG lower than 0.25.
However, for the Correlations feature sets, we find that 55% of the targets have a PG of 0.25, meaning that those targets are protected from MIA.
For the synthetic samples generated by Rec-RBM, 54% of the targets from the Histogram and Ensemble feature sets and 47 and 45% of the targets from the Correlations and Naive, respectively, have PG lower than 0.25.
Finally, for the WGAN, we find that between 44% and 53% of the targets have PG<math id="S5.SS1.SSS1.p7.1.m1.1" class="ltx_Math" alttext="&gt;" display="inline"><semantics id="S5.SS1.SSS1.p7.1.m1.1a"><mo id="S5.SS1.SSS1.p7.1.m1.1.1" xref="S5.SS1.SSS1.p7.1.m1.1.1.cmml">&gt;</mo><annotation-xml encoding="MathML-Content" id="S5.SS1.SSS1.p7.1.m1.1b"><gt id="S5.SS1.SSS1.p7.1.m1.1.1.cmml" xref="S5.SS1.SSS1.p7.1.m1.1.1"></gt></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.SSS1.p7.1.m1.1c">&gt;</annotation></semantics></math>0.25 across all four feature sets.
To give a better idea of different trade-offs between utility and privacy, we also plot the PG vs. the <math id="S5.SS1.SSS1.p7.2.m2.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="S5.SS1.SSS1.p7.2.m2.1a"><msub id="S5.SS1.SSS1.p7.2.m2.1.1" xref="S5.SS1.SSS1.p7.2.m2.1.1.cmml"><mi id="S5.SS1.SSS1.p7.2.m2.1.1.2" xref="S5.SS1.SSS1.p7.2.m2.1.1.2.cmml">F</mi><mrow id="S5.SS1.SSS1.p7.2.m2.1.1.3" xref="S5.SS1.SSS1.p7.2.m2.1.1.3.cmml"><mi id="S5.SS1.SSS1.p7.2.m2.1.1.3.2" xref="S5.SS1.SSS1.p7.2.m2.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="S5.SS1.SSS1.p7.2.m2.1.1.3.1" xref="S5.SS1.SSS1.p7.2.m2.1.1.3.1.cmml">​</mo><mi id="S5.SS1.SSS1.p7.2.m2.1.1.3.3" xref="S5.SS1.SSS1.p7.2.m2.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S5.SS1.SSS1.p7.2.m2.1b"><apply id="S5.SS1.SSS1.p7.2.m2.1.1.cmml" xref="S5.SS1.SSS1.p7.2.m2.1.1"><csymbol cd="ambiguous" id="S5.SS1.SSS1.p7.2.m2.1.1.1.cmml" xref="S5.SS1.SSS1.p7.2.m2.1.1">subscript</csymbol><ci id="S5.SS1.SSS1.p7.2.m2.1.1.2.cmml" xref="S5.SS1.SSS1.p7.2.m2.1.1.2">𝐹</ci><apply id="S5.SS1.SSS1.p7.2.m2.1.1.3.cmml" xref="S5.SS1.SSS1.p7.2.m2.1.1.3"><times id="S5.SS1.SSS1.p7.2.m2.1.1.3.1.cmml" xref="S5.SS1.SSS1.p7.2.m2.1.1.3.1"></times><ci id="S5.SS1.SSS1.p7.2.m2.1.1.3.2.cmml" xref="S5.SS1.SSS1.p7.2.m2.1.1.3.2">𝑆</ci><ci id="S5.SS1.SSS1.p7.2.m2.1.1.3.3.cmml" xref="S5.SS1.SSS1.p7.2.m2.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.SSS1.p7.2.m2.1c">F_{ST}</annotation></semantics></math> (see Section <a href="#S4.SS2" title="4.2 Population Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">4.2</span></a>) for all the models presented in Appendix <a href="#A4" title="Appendix D Privacy Gain vs. 𝐹_{𝑆⁢𝑇} ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">D</span></a>.</p>
</div>
</section>
<section id="S5.SS1.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.1.2 </span>1000 Genomes Population</h4>

<div id="S5.SS1.SSS2.p1" class="ltx_para">
<p id="S5.SS1.SSS2.p1.1" class="ltx_p">For this dataset, we focus our analysis on the RBM model, as it generated synthetic data closest to the real data across all utility metrics evaluated.</p>
</div>
<div id="S5.SS1.SSS2.p2" class="ltx_para ltx_noindent">
<p id="S5.SS1.SSS2.p2.2" class="ltx_p"><span id="S5.SS1.SSS2.p2.2.1" class="ltx_text ltx_font_bold">Random Target.</span> In Figure <a href="#S5.F6.sf1" title="In Figure 6 ‣ 5.1.1 HapMap Populations ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">6a</span></a>, we plot the PG for the synthetic data generated by RBM for randomly chosen targets.
With the RandForest MIA classifier, we observe that 56%, 75%, 92%, and 50% of the targets have PG higher than the random guess baseline (PG<math id="S5.SS1.SSS2.p2.1.m1.1" class="ltx_Math" alttext="\geq" display="inline"><semantics id="S5.SS1.SSS2.p2.1.m1.1a"><mo id="S5.SS1.SSS2.p2.1.m1.1.1" xref="S5.SS1.SSS2.p2.1.m1.1.1.cmml">≥</mo><annotation-xml encoding="MathML-Content" id="S5.SS1.SSS2.p2.1.m1.1b"><geq id="S5.SS1.SSS2.p2.1.m1.1.1.cmml" xref="S5.SS1.SSS2.p2.1.m1.1.1"></geq></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.SSS2.p2.1.m1.1c">\geq</annotation></semantics></math>0.25) for, respectively, Naive, Histogram, Correlations, and Ensemble feature sets.
The high percentage of targets that have a PG<math id="S5.SS1.SSS2.p2.2.m2.1" class="ltx_Math" alttext="\geq" display="inline"><semantics id="S5.SS1.SSS2.p2.2.m2.1a"><mo id="S5.SS1.SSS2.p2.2.m2.1.1" xref="S5.SS1.SSS2.p2.2.m2.1.1.cmml">≥</mo><annotation-xml encoding="MathML-Content" id="S5.SS1.SSS2.p2.2.m2.1b"><geq id="S5.SS1.SSS2.p2.2.m2.1.1.cmml" xref="S5.SS1.SSS2.p2.2.m2.1.1"></geq></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.SSS2.p2.2.m2.1c">\geq</annotation></semantics></math>0.25 for the Correlations suggests that the impact of a single target in the training dataset of the RBM on the correlations of the synthetic dataset is minimal.</p>
</div>
<div id="S5.SS1.SSS2.p3" class="ltx_para">
<p id="S5.SS1.SSS2.p3.1" class="ltx_p">Then, with the LogReg classifier, we find a high variation in PG, similar to the HapMap populations in the case of Rec-RBM.
Under the Naive feature set, half of the targets have PG gain below the random guess baseline.
Similarly, 43%, 45%, and 43% of targets have a PG lower than 0.25 for, respectively, the Correlations, Histogram, and Ensemble feature sets.
Finally, for the KNN classifier, over 94% of the targets have a PG of 0.25 across all feature sets.</p>
</div>
<div id="S5.SS1.SSS2.p4" class="ltx_para ltx_noindent">
<p id="S5.SS1.SSS2.p4.1" class="ltx_p"><span id="S5.SS1.SSS2.p4.1.1" class="ltx_text ltx_font_bold">Outlier Target.</span> To better understand whether or not, with more training data, a target’s signal in the synthetic dataset is diluted, we also test an “extreme” outlier case.
That is, we craft an outlier target that has only minor alleles at all positions.
While we are aware that this case would be sporadic in a real-world scenario, our goal is to observe whether and how much this impacts PG.
To this end, in Figure <a href="#S5.F6.sf2" title="In Figure 6 ‣ 5.1.1 HapMap Populations ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">6b</span></a>, we plot the PG of this outlier case across 10 test runs.</p>
</div>
<div id="S5.SS1.SSS2.p5" class="ltx_para">
<p id="S5.SS1.SSS2.p5.1" class="ltx_p">With RandForest, we find that PG is below 0.25 for 8 of the 10 test runs under the Ensemble feature set.
In fact, this is the only combination between attack classifier and feature set for which a greater percentage of the targets have a lower privacy gain than in the random target case.
For the Naive feature set, PG is below the random baseline in only 3 of the test runs. For the Correlations and Histogram feature sets, all test runs yield PG of 0.25 or above.
With LogReg, 4 out of 10 of the test runs for the Naive, Histogram, and Correlations feature sets yield PG below 0.25. For Ensemble, this happens for 6 test runs. Finally, with KNN, across all feature sets, PG for all test runs is 0.25, i.e., the synthetic data does not disclose any membership information regarding the outlier.</p>
</div>
<div id="S5.SS1.SSS2.p6" class="ltx_para">
<p id="S5.SS1.SSS2.p6.1" class="ltx_p">While there are differences across classifiers/feature sets, the PG is centered around 0.25 for all test runs in the outlier target setting.
This evident from Figure <a href="#S5.F6.sf2" title="In Figure 6 ‣ 5.1.1 HapMap Populations ‣ 5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">6b</span></a>, which implies that, across all test runs, the accuracy of the MIA is not much better than random guessing.</p>
</div>
</section>
<section id="S5.SS1.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.1.3 </span>Takeaways</h4>

<div id="S5.SS1.SSS3.p1" class="ltx_para">
<p id="S5.SS1.SSS3.p1.1" class="ltx_p">The different combinations of datasets, attack classifiers, and feature sets yield varied results with respect to privacy.
This is due to two main reasons: first, not all classifiers have the same accuracy on tasks for the same dataset, as shown in previous work <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">3</span></a>]</cite>.
Second, the features that the generative model preserves after training will “reflect” in the synthetic data; thus, this will impact the PG based on the feature extraction method.
On the HapMap populations, while the utility experiments show that Recomb-generated synthetic data is “closest” to the real data, it does so with a significant privacy loss. The RBM-generated synthetic data is the most vulnerable under the LogReg classifier, with at least 70% of the targets across both populations and all feature sets having PG below the random guess baseline.
This suggests that, with few data samples available for training, the RBM model is likely to overfit and is thus susceptible to MIAs.</p>
</div>
<div id="S5.SS1.SSS3.p2" class="ltx_para">
<p id="S5.SS1.SSS3.p2.1" class="ltx_p">For Rec-RBM and WGAN, the attacker cannot reliably predict membership, i.e., extra samples from the Recomb model in the training of the Rec-RBM dilute the target’s signal in the training data.
However, for both models, we still find combinations of targets and training sets for the attack classifier for which PG is significantly lower than the random guess baseline; i.e., the synthetic data will still expose membership information about the respective targets.
On the 1000 Genomes, PG values have a higher variation overall when the target is chosen randomly from the dataset than for the two smaller HapMap datasets.
The results for RBM confirm our hypothesis that the target’s influence is diluted within larger datasets. Once again, this does not mean that MIAs are not possible for Rec-RBM and WGAN; it depends on the combination of the target, training set, attack classifier, and feature set.</p>
</div>
<div id="S5.SS1.SSS3.p3" class="ltx_para">
<p id="S5.SS1.SSS3.p3.1" class="ltx_p">However, with an “extreme” outlier (i.e., a target with minor alleles at all positions), the synthetic data generated by RBM does not have a big impact on PG.
In this case, the PG is actually close to the random guess baseline across all test runs.</p>
</div>
<div id="S5.SS1.SSS3.p4" class="ltx_para">
<p id="S5.SS1.SSS3.p4.1" class="ltx_p">We find that targets get very different levels of privacy protection, based on the combination of target and training set used to generate the synthetic data.
For a privacy mechanism to provide good privacy protection, it needs to be predictable, i.e., all targets should have a privacy gain above the random guess baseline, which our evaluation shows it is not the case for the data generated by the models we have evaluated.</p>
</div>
</section>
</section>
<section id="S5.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.2 </span>Privacy Gain under Membership Inference Attack with Partial Information</h3>

<div id="S5.SS2.p1" class="ltx_para">
<p id="S5.SS2.p1.1" class="ltx_p">Next, we introduce a novel attack, which we denote as MIA with Partial Information (MIA-PI).
Basically, we only give the attacker access to a fraction of SNVs from the target sequence, chosen at random.
The attacker then uses the Recombination model from <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">54</span></a>]</cite> as an inference method to predict the rest of the sequence.
Compared to the previous attack, the adversary trains their (attack) classifier using the sequence <span id="S5.SS2.p1.1.1" class="ltx_text ltx_font_italic">inferred</span> from the partial data.
Thus, the PG formula also needs to be adjusted to account for how likely an adversary is to identify a target from partial information.</p>
</div>
<div id="S5.SS2.p2" class="ltx_para ltx_noindent">
<p id="S5.SS2.p2.1" class="ltx_p"><span id="S5.SS2.p2.1.1" class="ltx_text ltx_font_bold">Threat Model.</span> We assume that, similar to the MIA attack, the adversary has access to the synthetic dataset, as well as public datasets with similar distribution to the original data (which may or may not include the target) for training his shadow models. In contrast with the previous MIA, the adversary does not obtain access to the full sequence of the target, but only to a percentage of the SNPs from the target sequence (as for example, the victim undertakes some specific genomic tests and the adversary learns that information about the victim from the outcome of those tests), as well as public knowledge about genomic data. The goal of the adversary is to identify if the partial target sequence was part of the training set which generated the synthetic data.</p>
</div>
<div id="S5.SS2.p3" class="ltx_para ltx_noindent">
<p id="S5.SS2.p3.3" class="ltx_p"><span id="S5.SS2.p3.3.1" class="ltx_text ltx_font_bold">PG for MIA-PI.</span> Assuming the attacker has partial information <math id="S5.SS2.p3.1.m1.1" class="ltx_Math" alttext="t^{\prime}" display="inline"><semantics id="S5.SS2.p3.1.m1.1a"><msup id="S5.SS2.p3.1.m1.1.1" xref="S5.SS2.p3.1.m1.1.1.cmml"><mi id="S5.SS2.p3.1.m1.1.1.2" xref="S5.SS2.p3.1.m1.1.1.2.cmml">t</mi><mo id="S5.SS2.p3.1.m1.1.1.3" xref="S5.SS2.p3.1.m1.1.1.3.cmml">′</mo></msup><annotation-xml encoding="MathML-Content" id="S5.SS2.p3.1.m1.1b"><apply id="S5.SS2.p3.1.m1.1.1.cmml" xref="S5.SS2.p3.1.m1.1.1"><csymbol cd="ambiguous" id="S5.SS2.p3.1.m1.1.1.1.cmml" xref="S5.SS2.p3.1.m1.1.1">superscript</csymbol><ci id="S5.SS2.p3.1.m1.1.1.2.cmml" xref="S5.SS2.p3.1.m1.1.1.2">𝑡</ci><ci id="S5.SS2.p3.1.m1.1.1.3.cmml" xref="S5.SS2.p3.1.m1.1.1.3">′</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p3.1.m1.1c">t^{\prime}</annotation></semantics></math> as a fraction of the SNVs from <math id="S5.SS2.p3.2.m2.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S5.SS2.p3.2.m2.1a"><mi id="S5.SS2.p3.2.m2.1.1" xref="S5.SS2.p3.2.m2.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S5.SS2.p3.2.m2.1b"><ci id="S5.SS2.p3.2.m2.1.1.cmml" xref="S5.SS2.p3.2.m2.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p3.2.m2.1c">t</annotation></semantics></math>, they first use the Recomb model, as an inference algorithm, to predict the rest of the SNVs from the target sequence, which we denote by <math id="S5.SS2.p3.3.m3.1" class="ltx_Math" alttext="t_{p}" display="inline"><semantics id="S5.SS2.p3.3.m3.1a"><msub id="S5.SS2.p3.3.m3.1.1" xref="S5.SS2.p3.3.m3.1.1.cmml"><mi id="S5.SS2.p3.3.m3.1.1.2" xref="S5.SS2.p3.3.m3.1.1.2.cmml">t</mi><mi id="S5.SS2.p3.3.m3.1.1.3" xref="S5.SS2.p3.3.m3.1.1.3.cmml">p</mi></msub><annotation-xml encoding="MathML-Content" id="S5.SS2.p3.3.m3.1b"><apply id="S5.SS2.p3.3.m3.1.1.cmml" xref="S5.SS2.p3.3.m3.1.1"><csymbol cd="ambiguous" id="S5.SS2.p3.3.m3.1.1.1.cmml" xref="S5.SS2.p3.3.m3.1.1">subscript</csymbol><ci id="S5.SS2.p3.3.m3.1.1.2.cmml" xref="S5.SS2.p3.3.m3.1.1.2">𝑡</ci><ci id="S5.SS2.p3.3.m3.1.1.3.cmml" xref="S5.SS2.p3.3.m3.1.1.3">𝑝</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p3.3.m3.1c">t_{p}</annotation></semantics></math>.
The privacy gain is computed as:</p>
</div>
<div id="S5.SS2.p4" class="ltx_para">
<table id="S5.Ex1" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S5.Ex1.m1.2" class="ltx_Math" alttext="PG_{t}=\frac{\overline{MIA_{t_{p}}}(R_{t})-\overline{MIA_{t_{p}}}(S_{test})}{2}\mbox{, where}\vspace*{-0.1cm}" display="block"><semantics id="S5.Ex1.m1.2a"><mrow id="S5.Ex1.m1.2.3" xref="S5.Ex1.m1.2.3.cmml"><mrow id="S5.Ex1.m1.2.3.2" xref="S5.Ex1.m1.2.3.2.cmml"><mi id="S5.Ex1.m1.2.3.2.2" xref="S5.Ex1.m1.2.3.2.2.cmml">P</mi><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.2.3.2.1" xref="S5.Ex1.m1.2.3.2.1.cmml">​</mo><msub id="S5.Ex1.m1.2.3.2.3" xref="S5.Ex1.m1.2.3.2.3.cmml"><mi id="S5.Ex1.m1.2.3.2.3.2" xref="S5.Ex1.m1.2.3.2.3.2.cmml">G</mi><mi id="S5.Ex1.m1.2.3.2.3.3" xref="S5.Ex1.m1.2.3.2.3.3.cmml">t</mi></msub></mrow><mo id="S5.Ex1.m1.2.3.1" xref="S5.Ex1.m1.2.3.1.cmml">=</mo><mrow id="S5.Ex1.m1.2.3.3" xref="S5.Ex1.m1.2.3.3.cmml"><mfrac id="S5.Ex1.m1.2.2" xref="S5.Ex1.m1.2.2.cmml"><mrow id="S5.Ex1.m1.2.2.2" xref="S5.Ex1.m1.2.2.2.cmml"><mrow id="S5.Ex1.m1.1.1.1.1" xref="S5.Ex1.m1.1.1.1.1.cmml"><mover accent="true" id="S5.Ex1.m1.1.1.1.1.3" xref="S5.Ex1.m1.1.1.1.1.3.cmml"><mrow id="S5.Ex1.m1.1.1.1.1.3.2" xref="S5.Ex1.m1.1.1.1.1.3.2.cmml"><mi id="S5.Ex1.m1.1.1.1.1.3.2.2" xref="S5.Ex1.m1.1.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.1.1.1.1.3.2.1" xref="S5.Ex1.m1.1.1.1.1.3.2.1.cmml">​</mo><mi id="S5.Ex1.m1.1.1.1.1.3.2.3" xref="S5.Ex1.m1.1.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.1.1.1.1.3.2.1a" xref="S5.Ex1.m1.1.1.1.1.3.2.1.cmml">​</mo><msub id="S5.Ex1.m1.1.1.1.1.3.2.4" xref="S5.Ex1.m1.1.1.1.1.3.2.4.cmml"><mi id="S5.Ex1.m1.1.1.1.1.3.2.4.2" xref="S5.Ex1.m1.1.1.1.1.3.2.4.2.cmml">A</mi><msub id="S5.Ex1.m1.1.1.1.1.3.2.4.3" xref="S5.Ex1.m1.1.1.1.1.3.2.4.3.cmml"><mi id="S5.Ex1.m1.1.1.1.1.3.2.4.3.2" xref="S5.Ex1.m1.1.1.1.1.3.2.4.3.2.cmml">t</mi><mi id="S5.Ex1.m1.1.1.1.1.3.2.4.3.3" xref="S5.Ex1.m1.1.1.1.1.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.Ex1.m1.1.1.1.1.3.1" xref="S5.Ex1.m1.1.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.1.1.1.1.2" xref="S5.Ex1.m1.1.1.1.1.2.cmml">​</mo><mrow id="S5.Ex1.m1.1.1.1.1.1.1" xref="S5.Ex1.m1.1.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.Ex1.m1.1.1.1.1.1.1.2" xref="S5.Ex1.m1.1.1.1.1.1.1.1.cmml">(</mo><msub id="S5.Ex1.m1.1.1.1.1.1.1.1" xref="S5.Ex1.m1.1.1.1.1.1.1.1.cmml"><mi id="S5.Ex1.m1.1.1.1.1.1.1.1.2" xref="S5.Ex1.m1.1.1.1.1.1.1.1.2.cmml">R</mi><mi id="S5.Ex1.m1.1.1.1.1.1.1.1.3" xref="S5.Ex1.m1.1.1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.Ex1.m1.1.1.1.1.1.1.3" xref="S5.Ex1.m1.1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S5.Ex1.m1.2.2.2.3" xref="S5.Ex1.m1.2.2.2.3.cmml">−</mo><mrow id="S5.Ex1.m1.2.2.2.2" xref="S5.Ex1.m1.2.2.2.2.cmml"><mover accent="true" id="S5.Ex1.m1.2.2.2.2.3" xref="S5.Ex1.m1.2.2.2.2.3.cmml"><mrow id="S5.Ex1.m1.2.2.2.2.3.2" xref="S5.Ex1.m1.2.2.2.2.3.2.cmml"><mi id="S5.Ex1.m1.2.2.2.2.3.2.2" xref="S5.Ex1.m1.2.2.2.2.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.2.2.2.2.3.2.1" xref="S5.Ex1.m1.2.2.2.2.3.2.1.cmml">​</mo><mi id="S5.Ex1.m1.2.2.2.2.3.2.3" xref="S5.Ex1.m1.2.2.2.2.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.2.2.2.2.3.2.1a" xref="S5.Ex1.m1.2.2.2.2.3.2.1.cmml">​</mo><msub id="S5.Ex1.m1.2.2.2.2.3.2.4" xref="S5.Ex1.m1.2.2.2.2.3.2.4.cmml"><mi id="S5.Ex1.m1.2.2.2.2.3.2.4.2" xref="S5.Ex1.m1.2.2.2.2.3.2.4.2.cmml">A</mi><msub id="S5.Ex1.m1.2.2.2.2.3.2.4.3" xref="S5.Ex1.m1.2.2.2.2.3.2.4.3.cmml"><mi id="S5.Ex1.m1.2.2.2.2.3.2.4.3.2" xref="S5.Ex1.m1.2.2.2.2.3.2.4.3.2.cmml">t</mi><mi id="S5.Ex1.m1.2.2.2.2.3.2.4.3.3" xref="S5.Ex1.m1.2.2.2.2.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.Ex1.m1.2.2.2.2.3.1" xref="S5.Ex1.m1.2.2.2.2.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.2.2.2.2.2" xref="S5.Ex1.m1.2.2.2.2.2.cmml">​</mo><mrow id="S5.Ex1.m1.2.2.2.2.1.1" xref="S5.Ex1.m1.2.2.2.2.1.1.1.cmml"><mo stretchy="false" id="S5.Ex1.m1.2.2.2.2.1.1.2" xref="S5.Ex1.m1.2.2.2.2.1.1.1.cmml">(</mo><msub id="S5.Ex1.m1.2.2.2.2.1.1.1" xref="S5.Ex1.m1.2.2.2.2.1.1.1.cmml"><mi id="S5.Ex1.m1.2.2.2.2.1.1.1.2" xref="S5.Ex1.m1.2.2.2.2.1.1.1.2.cmml">S</mi><mrow id="S5.Ex1.m1.2.2.2.2.1.1.1.3" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.cmml"><mi id="S5.Ex1.m1.2.2.2.2.1.1.1.3.2" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.2.2.2.2.1.1.1.3.1" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.Ex1.m1.2.2.2.2.1.1.1.3.3" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.2.2.2.2.1.1.1.3.1a" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.Ex1.m1.2.2.2.2.1.1.1.3.4" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.2.2.2.2.1.1.1.3.1b" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.Ex1.m1.2.2.2.2.1.1.1.3.5" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S5.Ex1.m1.2.2.2.2.1.1.3" xref="S5.Ex1.m1.2.2.2.2.1.1.1.cmml">)</mo></mrow></mrow></mrow><mn id="S5.Ex1.m1.2.2.4" xref="S5.Ex1.m1.2.2.4.cmml">2</mn></mfrac><mo lspace="0em" rspace="0em" id="S5.Ex1.m1.2.3.3.1" xref="S5.Ex1.m1.2.3.3.1.cmml">​</mo><mtext id="S5.Ex1.m1.2.3.3.2" xref="S5.Ex1.m1.2.3.3.2a.cmml">, where</mtext></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.Ex1.m1.2b"><apply id="S5.Ex1.m1.2.3.cmml" xref="S5.Ex1.m1.2.3"><eq id="S5.Ex1.m1.2.3.1.cmml" xref="S5.Ex1.m1.2.3.1"></eq><apply id="S5.Ex1.m1.2.3.2.cmml" xref="S5.Ex1.m1.2.3.2"><times id="S5.Ex1.m1.2.3.2.1.cmml" xref="S5.Ex1.m1.2.3.2.1"></times><ci id="S5.Ex1.m1.2.3.2.2.cmml" xref="S5.Ex1.m1.2.3.2.2">𝑃</ci><apply id="S5.Ex1.m1.2.3.2.3.cmml" xref="S5.Ex1.m1.2.3.2.3"><csymbol cd="ambiguous" id="S5.Ex1.m1.2.3.2.3.1.cmml" xref="S5.Ex1.m1.2.3.2.3">subscript</csymbol><ci id="S5.Ex1.m1.2.3.2.3.2.cmml" xref="S5.Ex1.m1.2.3.2.3.2">𝐺</ci><ci id="S5.Ex1.m1.2.3.2.3.3.cmml" xref="S5.Ex1.m1.2.3.2.3.3">𝑡</ci></apply></apply><apply id="S5.Ex1.m1.2.3.3.cmml" xref="S5.Ex1.m1.2.3.3"><times id="S5.Ex1.m1.2.3.3.1.cmml" xref="S5.Ex1.m1.2.3.3.1"></times><apply id="S5.Ex1.m1.2.2.cmml" xref="S5.Ex1.m1.2.2"><divide id="S5.Ex1.m1.2.2.3.cmml" xref="S5.Ex1.m1.2.2"></divide><apply id="S5.Ex1.m1.2.2.2.cmml" xref="S5.Ex1.m1.2.2.2"><minus id="S5.Ex1.m1.2.2.2.3.cmml" xref="S5.Ex1.m1.2.2.2.3"></minus><apply id="S5.Ex1.m1.1.1.1.1.cmml" xref="S5.Ex1.m1.1.1.1.1"><times id="S5.Ex1.m1.1.1.1.1.2.cmml" xref="S5.Ex1.m1.1.1.1.1.2"></times><apply id="S5.Ex1.m1.1.1.1.1.3.cmml" xref="S5.Ex1.m1.1.1.1.1.3"><ci id="S5.Ex1.m1.1.1.1.1.3.1.cmml" xref="S5.Ex1.m1.1.1.1.1.3.1">¯</ci><apply id="S5.Ex1.m1.1.1.1.1.3.2.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2"><times id="S5.Ex1.m1.1.1.1.1.3.2.1.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.1"></times><ci id="S5.Ex1.m1.1.1.1.1.3.2.2.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.2">𝑀</ci><ci id="S5.Ex1.m1.1.1.1.1.3.2.3.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.3">𝐼</ci><apply id="S5.Ex1.m1.1.1.1.1.3.2.4.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.4"><csymbol cd="ambiguous" id="S5.Ex1.m1.1.1.1.1.3.2.4.1.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.4">subscript</csymbol><ci id="S5.Ex1.m1.1.1.1.1.3.2.4.2.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.4.2">𝐴</ci><apply id="S5.Ex1.m1.1.1.1.1.3.2.4.3.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.4.3"><csymbol cd="ambiguous" id="S5.Ex1.m1.1.1.1.1.3.2.4.3.1.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.4.3">subscript</csymbol><ci id="S5.Ex1.m1.1.1.1.1.3.2.4.3.2.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.4.3.2">𝑡</ci><ci id="S5.Ex1.m1.1.1.1.1.3.2.4.3.3.cmml" xref="S5.Ex1.m1.1.1.1.1.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.Ex1.m1.1.1.1.1.1.1.1.cmml" xref="S5.Ex1.m1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.Ex1.m1.1.1.1.1.1.1.1.1.cmml" xref="S5.Ex1.m1.1.1.1.1.1.1">subscript</csymbol><ci id="S5.Ex1.m1.1.1.1.1.1.1.1.2.cmml" xref="S5.Ex1.m1.1.1.1.1.1.1.1.2">𝑅</ci><ci id="S5.Ex1.m1.1.1.1.1.1.1.1.3.cmml" xref="S5.Ex1.m1.1.1.1.1.1.1.1.3">𝑡</ci></apply></apply><apply id="S5.Ex1.m1.2.2.2.2.cmml" xref="S5.Ex1.m1.2.2.2.2"><times id="S5.Ex1.m1.2.2.2.2.2.cmml" xref="S5.Ex1.m1.2.2.2.2.2"></times><apply id="S5.Ex1.m1.2.2.2.2.3.cmml" xref="S5.Ex1.m1.2.2.2.2.3"><ci id="S5.Ex1.m1.2.2.2.2.3.1.cmml" xref="S5.Ex1.m1.2.2.2.2.3.1">¯</ci><apply id="S5.Ex1.m1.2.2.2.2.3.2.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2"><times id="S5.Ex1.m1.2.2.2.2.3.2.1.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.1"></times><ci id="S5.Ex1.m1.2.2.2.2.3.2.2.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.2">𝑀</ci><ci id="S5.Ex1.m1.2.2.2.2.3.2.3.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.3">𝐼</ci><apply id="S5.Ex1.m1.2.2.2.2.3.2.4.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.4"><csymbol cd="ambiguous" id="S5.Ex1.m1.2.2.2.2.3.2.4.1.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.4">subscript</csymbol><ci id="S5.Ex1.m1.2.2.2.2.3.2.4.2.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.4.2">𝐴</ci><apply id="S5.Ex1.m1.2.2.2.2.3.2.4.3.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.4.3"><csymbol cd="ambiguous" id="S5.Ex1.m1.2.2.2.2.3.2.4.3.1.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.4.3">subscript</csymbol><ci id="S5.Ex1.m1.2.2.2.2.3.2.4.3.2.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.4.3.2">𝑡</ci><ci id="S5.Ex1.m1.2.2.2.2.3.2.4.3.3.cmml" xref="S5.Ex1.m1.2.2.2.2.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.Ex1.m1.2.2.2.2.1.1.1.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1"><csymbol cd="ambiguous" id="S5.Ex1.m1.2.2.2.2.1.1.1.1.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1">subscript</csymbol><ci id="S5.Ex1.m1.2.2.2.2.1.1.1.2.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1.1.2">𝑆</ci><apply id="S5.Ex1.m1.2.2.2.2.1.1.1.3.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3"><times id="S5.Ex1.m1.2.2.2.2.1.1.1.3.1.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.1"></times><ci id="S5.Ex1.m1.2.2.2.2.1.1.1.3.2.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.2">𝑡</ci><ci id="S5.Ex1.m1.2.2.2.2.1.1.1.3.3.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.3">𝑒</ci><ci id="S5.Ex1.m1.2.2.2.2.1.1.1.3.4.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.4">𝑠</ci><ci id="S5.Ex1.m1.2.2.2.2.1.1.1.3.5.cmml" xref="S5.Ex1.m1.2.2.2.2.1.1.1.3.5">𝑡</ci></apply></apply></apply></apply><cn type="integer" id="S5.Ex1.m1.2.2.4.cmml" xref="S5.Ex1.m1.2.2.4">2</cn></apply><ci id="S5.Ex1.m1.2.3.3.2a.cmml" xref="S5.Ex1.m1.2.3.3.2"><mtext id="S5.Ex1.m1.2.3.3.2.cmml" xref="S5.Ex1.m1.2.3.3.2">, where</mtext></ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.Ex1.m1.2c">PG_{t}=\frac{\overline{MIA_{t_{p}}}(R_{t})-\overline{MIA_{t_{p}}}(S_{test})}{2}\mbox{, where}\vspace*{-0.1cm}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr></tbody>
</table>
<table id="S5.Ex2" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S5.Ex2.m1.3" class="ltx_Math" alttext="\overline{MIA_{t_{p}}}(S_{test})=\sum_{S_{i}\in S_{test}}\frac{\Pr[MIA_{t_{p}}(S_{i})=1]}{2*n_{s}}\mbox{, and}\vspace*{-0.1cm}" display="block"><semantics id="S5.Ex2.m1.3a"><mrow id="S5.Ex2.m1.3.3" xref="S5.Ex2.m1.3.3.cmml"><mrow id="S5.Ex2.m1.3.3.1" xref="S5.Ex2.m1.3.3.1.cmml"><mover accent="true" id="S5.Ex2.m1.3.3.1.3" xref="S5.Ex2.m1.3.3.1.3.cmml"><mrow id="S5.Ex2.m1.3.3.1.3.2" xref="S5.Ex2.m1.3.3.1.3.2.cmml"><mi id="S5.Ex2.m1.3.3.1.3.2.2" xref="S5.Ex2.m1.3.3.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.1.3.2.1" xref="S5.Ex2.m1.3.3.1.3.2.1.cmml">​</mo><mi id="S5.Ex2.m1.3.3.1.3.2.3" xref="S5.Ex2.m1.3.3.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.1.3.2.1a" xref="S5.Ex2.m1.3.3.1.3.2.1.cmml">​</mo><msub id="S5.Ex2.m1.3.3.1.3.2.4" xref="S5.Ex2.m1.3.3.1.3.2.4.cmml"><mi id="S5.Ex2.m1.3.3.1.3.2.4.2" xref="S5.Ex2.m1.3.3.1.3.2.4.2.cmml">A</mi><msub id="S5.Ex2.m1.3.3.1.3.2.4.3" xref="S5.Ex2.m1.3.3.1.3.2.4.3.cmml"><mi id="S5.Ex2.m1.3.3.1.3.2.4.3.2" xref="S5.Ex2.m1.3.3.1.3.2.4.3.2.cmml">t</mi><mi id="S5.Ex2.m1.3.3.1.3.2.4.3.3" xref="S5.Ex2.m1.3.3.1.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.Ex2.m1.3.3.1.3.1" xref="S5.Ex2.m1.3.3.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.1.2" xref="S5.Ex2.m1.3.3.1.2.cmml">​</mo><mrow id="S5.Ex2.m1.3.3.1.1.1" xref="S5.Ex2.m1.3.3.1.1.1.1.cmml"><mo stretchy="false" id="S5.Ex2.m1.3.3.1.1.1.2" xref="S5.Ex2.m1.3.3.1.1.1.1.cmml">(</mo><msub id="S5.Ex2.m1.3.3.1.1.1.1" xref="S5.Ex2.m1.3.3.1.1.1.1.cmml"><mi id="S5.Ex2.m1.3.3.1.1.1.1.2" xref="S5.Ex2.m1.3.3.1.1.1.1.2.cmml">S</mi><mrow id="S5.Ex2.m1.3.3.1.1.1.1.3" xref="S5.Ex2.m1.3.3.1.1.1.1.3.cmml"><mi id="S5.Ex2.m1.3.3.1.1.1.1.3.2" xref="S5.Ex2.m1.3.3.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.1.1.1.1.3.1" xref="S5.Ex2.m1.3.3.1.1.1.1.3.1.cmml">​</mo><mi id="S5.Ex2.m1.3.3.1.1.1.1.3.3" xref="S5.Ex2.m1.3.3.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.1.1.1.1.3.1a" xref="S5.Ex2.m1.3.3.1.1.1.1.3.1.cmml">​</mo><mi id="S5.Ex2.m1.3.3.1.1.1.1.3.4" xref="S5.Ex2.m1.3.3.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.1.1.1.1.3.1b" xref="S5.Ex2.m1.3.3.1.1.1.1.3.1.cmml">​</mo><mi id="S5.Ex2.m1.3.3.1.1.1.1.3.5" xref="S5.Ex2.m1.3.3.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S5.Ex2.m1.3.3.1.1.1.3" xref="S5.Ex2.m1.3.3.1.1.1.1.cmml">)</mo></mrow></mrow><mo rspace="0.111em" id="S5.Ex2.m1.3.3.2" xref="S5.Ex2.m1.3.3.2.cmml">=</mo><mrow id="S5.Ex2.m1.3.3.3" xref="S5.Ex2.m1.3.3.3.cmml"><munder id="S5.Ex2.m1.3.3.3.1" xref="S5.Ex2.m1.3.3.3.1.cmml"><mo movablelimits="false" id="S5.Ex2.m1.3.3.3.1.2" xref="S5.Ex2.m1.3.3.3.1.2.cmml">∑</mo><mrow id="S5.Ex2.m1.3.3.3.1.3" xref="S5.Ex2.m1.3.3.3.1.3.cmml"><msub id="S5.Ex2.m1.3.3.3.1.3.2" xref="S5.Ex2.m1.3.3.3.1.3.2.cmml"><mi id="S5.Ex2.m1.3.3.3.1.3.2.2" xref="S5.Ex2.m1.3.3.3.1.3.2.2.cmml">S</mi><mi id="S5.Ex2.m1.3.3.3.1.3.2.3" xref="S5.Ex2.m1.3.3.3.1.3.2.3.cmml">i</mi></msub><mo id="S5.Ex2.m1.3.3.3.1.3.1" xref="S5.Ex2.m1.3.3.3.1.3.1.cmml">∈</mo><msub id="S5.Ex2.m1.3.3.3.1.3.3" xref="S5.Ex2.m1.3.3.3.1.3.3.cmml"><mi id="S5.Ex2.m1.3.3.3.1.3.3.2" xref="S5.Ex2.m1.3.3.3.1.3.3.2.cmml">S</mi><mrow id="S5.Ex2.m1.3.3.3.1.3.3.3" xref="S5.Ex2.m1.3.3.3.1.3.3.3.cmml"><mi id="S5.Ex2.m1.3.3.3.1.3.3.3.2" xref="S5.Ex2.m1.3.3.3.1.3.3.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.3.1.3.3.3.1" xref="S5.Ex2.m1.3.3.3.1.3.3.3.1.cmml">​</mo><mi id="S5.Ex2.m1.3.3.3.1.3.3.3.3" xref="S5.Ex2.m1.3.3.3.1.3.3.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.3.1.3.3.3.1a" xref="S5.Ex2.m1.3.3.3.1.3.3.3.1.cmml">​</mo><mi id="S5.Ex2.m1.3.3.3.1.3.3.3.4" xref="S5.Ex2.m1.3.3.3.1.3.3.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.3.1.3.3.3.1b" xref="S5.Ex2.m1.3.3.3.1.3.3.3.1.cmml">​</mo><mi id="S5.Ex2.m1.3.3.3.1.3.3.3.5" xref="S5.Ex2.m1.3.3.3.1.3.3.3.5.cmml">t</mi></mrow></msub></mrow></munder><mrow id="S5.Ex2.m1.3.3.3.2" xref="S5.Ex2.m1.3.3.3.2.cmml"><mfrac id="S5.Ex2.m1.2.2" xref="S5.Ex2.m1.2.2.cmml"><mrow id="S5.Ex2.m1.2.2.2.2" xref="S5.Ex2.m1.2.2.2.3.cmml"><mi id="S5.Ex2.m1.1.1.1.1" xref="S5.Ex2.m1.1.1.1.1.cmml">Pr</mi><mo id="S5.Ex2.m1.2.2.2.2a" xref="S5.Ex2.m1.2.2.2.3.cmml">⁡</mo><mrow id="S5.Ex2.m1.2.2.2.2.1" xref="S5.Ex2.m1.2.2.2.3.cmml"><mo stretchy="false" id="S5.Ex2.m1.2.2.2.2.1.2" xref="S5.Ex2.m1.2.2.2.3.cmml">[</mo><mrow id="S5.Ex2.m1.2.2.2.2.1.1" xref="S5.Ex2.m1.2.2.2.2.1.1.cmml"><mrow id="S5.Ex2.m1.2.2.2.2.1.1.1" xref="S5.Ex2.m1.2.2.2.2.1.1.1.cmml"><mi id="S5.Ex2.m1.2.2.2.2.1.1.1.3" xref="S5.Ex2.m1.2.2.2.2.1.1.1.3.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.2.2.2.2.1.1.1.2" xref="S5.Ex2.m1.2.2.2.2.1.1.1.2.cmml">​</mo><mi id="S5.Ex2.m1.2.2.2.2.1.1.1.4" xref="S5.Ex2.m1.2.2.2.2.1.1.1.4.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.2.2.2.2.1.1.1.2a" xref="S5.Ex2.m1.2.2.2.2.1.1.1.2.cmml">​</mo><msub id="S5.Ex2.m1.2.2.2.2.1.1.1.5" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.cmml"><mi id="S5.Ex2.m1.2.2.2.2.1.1.1.5.2" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.2.cmml">A</mi><msub id="S5.Ex2.m1.2.2.2.2.1.1.1.5.3" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.cmml"><mi id="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.2" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.2.cmml">t</mi><mi id="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.3" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.3.cmml">p</mi></msub></msub><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.2.2.2.2.1.1.1.2b" xref="S5.Ex2.m1.2.2.2.2.1.1.1.2.cmml">​</mo><mrow id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.2" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.cmml">(</mo><msub id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.cmml"><mi id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.2" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.2.cmml">S</mi><mi id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.3" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.3" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S5.Ex2.m1.2.2.2.2.1.1.2" xref="S5.Ex2.m1.2.2.2.2.1.1.2.cmml">=</mo><mn id="S5.Ex2.m1.2.2.2.2.1.1.3" xref="S5.Ex2.m1.2.2.2.2.1.1.3.cmml">1</mn></mrow><mo stretchy="false" id="S5.Ex2.m1.2.2.2.2.1.3" xref="S5.Ex2.m1.2.2.2.3.cmml">]</mo></mrow></mrow><mrow id="S5.Ex2.m1.2.2.4" xref="S5.Ex2.m1.2.2.4.cmml"><mn id="S5.Ex2.m1.2.2.4.2" xref="S5.Ex2.m1.2.2.4.2.cmml">2</mn><mo lspace="0.222em" rspace="0.222em" id="S5.Ex2.m1.2.2.4.1" xref="S5.Ex2.m1.2.2.4.1.cmml">∗</mo><msub id="S5.Ex2.m1.2.2.4.3" xref="S5.Ex2.m1.2.2.4.3.cmml"><mi id="S5.Ex2.m1.2.2.4.3.2" xref="S5.Ex2.m1.2.2.4.3.2.cmml">n</mi><mi id="S5.Ex2.m1.2.2.4.3.3" xref="S5.Ex2.m1.2.2.4.3.3.cmml">s</mi></msub></mrow></mfrac><mo lspace="0em" rspace="0em" id="S5.Ex2.m1.3.3.3.2.1" xref="S5.Ex2.m1.3.3.3.2.1.cmml">​</mo><mtext id="S5.Ex2.m1.3.3.3.2.2" xref="S5.Ex2.m1.3.3.3.2.2a.cmml">, and</mtext></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.Ex2.m1.3b"><apply id="S5.Ex2.m1.3.3.cmml" xref="S5.Ex2.m1.3.3"><eq id="S5.Ex2.m1.3.3.2.cmml" xref="S5.Ex2.m1.3.3.2"></eq><apply id="S5.Ex2.m1.3.3.1.cmml" xref="S5.Ex2.m1.3.3.1"><times id="S5.Ex2.m1.3.3.1.2.cmml" xref="S5.Ex2.m1.3.3.1.2"></times><apply id="S5.Ex2.m1.3.3.1.3.cmml" xref="S5.Ex2.m1.3.3.1.3"><ci id="S5.Ex2.m1.3.3.1.3.1.cmml" xref="S5.Ex2.m1.3.3.1.3.1">¯</ci><apply id="S5.Ex2.m1.3.3.1.3.2.cmml" xref="S5.Ex2.m1.3.3.1.3.2"><times id="S5.Ex2.m1.3.3.1.3.2.1.cmml" xref="S5.Ex2.m1.3.3.1.3.2.1"></times><ci id="S5.Ex2.m1.3.3.1.3.2.2.cmml" xref="S5.Ex2.m1.3.3.1.3.2.2">𝑀</ci><ci id="S5.Ex2.m1.3.3.1.3.2.3.cmml" xref="S5.Ex2.m1.3.3.1.3.2.3">𝐼</ci><apply id="S5.Ex2.m1.3.3.1.3.2.4.cmml" xref="S5.Ex2.m1.3.3.1.3.2.4"><csymbol cd="ambiguous" id="S5.Ex2.m1.3.3.1.3.2.4.1.cmml" xref="S5.Ex2.m1.3.3.1.3.2.4">subscript</csymbol><ci id="S5.Ex2.m1.3.3.1.3.2.4.2.cmml" xref="S5.Ex2.m1.3.3.1.3.2.4.2">𝐴</ci><apply id="S5.Ex2.m1.3.3.1.3.2.4.3.cmml" xref="S5.Ex2.m1.3.3.1.3.2.4.3"><csymbol cd="ambiguous" id="S5.Ex2.m1.3.3.1.3.2.4.3.1.cmml" xref="S5.Ex2.m1.3.3.1.3.2.4.3">subscript</csymbol><ci id="S5.Ex2.m1.3.3.1.3.2.4.3.2.cmml" xref="S5.Ex2.m1.3.3.1.3.2.4.3.2">𝑡</ci><ci id="S5.Ex2.m1.3.3.1.3.2.4.3.3.cmml" xref="S5.Ex2.m1.3.3.1.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.Ex2.m1.3.3.1.1.1.1.cmml" xref="S5.Ex2.m1.3.3.1.1.1"><csymbol cd="ambiguous" id="S5.Ex2.m1.3.3.1.1.1.1.1.cmml" xref="S5.Ex2.m1.3.3.1.1.1">subscript</csymbol><ci id="S5.Ex2.m1.3.3.1.1.1.1.2.cmml" xref="S5.Ex2.m1.3.3.1.1.1.1.2">𝑆</ci><apply id="S5.Ex2.m1.3.3.1.1.1.1.3.cmml" xref="S5.Ex2.m1.3.3.1.1.1.1.3"><times id="S5.Ex2.m1.3.3.1.1.1.1.3.1.cmml" xref="S5.Ex2.m1.3.3.1.1.1.1.3.1"></times><ci id="S5.Ex2.m1.3.3.1.1.1.1.3.2.cmml" xref="S5.Ex2.m1.3.3.1.1.1.1.3.2">𝑡</ci><ci id="S5.Ex2.m1.3.3.1.1.1.1.3.3.cmml" xref="S5.Ex2.m1.3.3.1.1.1.1.3.3">𝑒</ci><ci id="S5.Ex2.m1.3.3.1.1.1.1.3.4.cmml" xref="S5.Ex2.m1.3.3.1.1.1.1.3.4">𝑠</ci><ci id="S5.Ex2.m1.3.3.1.1.1.1.3.5.cmml" xref="S5.Ex2.m1.3.3.1.1.1.1.3.5">𝑡</ci></apply></apply></apply><apply id="S5.Ex2.m1.3.3.3.cmml" xref="S5.Ex2.m1.3.3.3"><apply id="S5.Ex2.m1.3.3.3.1.cmml" xref="S5.Ex2.m1.3.3.3.1"><csymbol cd="ambiguous" id="S5.Ex2.m1.3.3.3.1.1.cmml" xref="S5.Ex2.m1.3.3.3.1">subscript</csymbol><sum id="S5.Ex2.m1.3.3.3.1.2.cmml" xref="S5.Ex2.m1.3.3.3.1.2"></sum><apply id="S5.Ex2.m1.3.3.3.1.3.cmml" xref="S5.Ex2.m1.3.3.3.1.3"><in id="S5.Ex2.m1.3.3.3.1.3.1.cmml" xref="S5.Ex2.m1.3.3.3.1.3.1"></in><apply id="S5.Ex2.m1.3.3.3.1.3.2.cmml" xref="S5.Ex2.m1.3.3.3.1.3.2"><csymbol cd="ambiguous" id="S5.Ex2.m1.3.3.3.1.3.2.1.cmml" xref="S5.Ex2.m1.3.3.3.1.3.2">subscript</csymbol><ci id="S5.Ex2.m1.3.3.3.1.3.2.2.cmml" xref="S5.Ex2.m1.3.3.3.1.3.2.2">𝑆</ci><ci id="S5.Ex2.m1.3.3.3.1.3.2.3.cmml" xref="S5.Ex2.m1.3.3.3.1.3.2.3">𝑖</ci></apply><apply id="S5.Ex2.m1.3.3.3.1.3.3.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3"><csymbol cd="ambiguous" id="S5.Ex2.m1.3.3.3.1.3.3.1.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3">subscript</csymbol><ci id="S5.Ex2.m1.3.3.3.1.3.3.2.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3.2">𝑆</ci><apply id="S5.Ex2.m1.3.3.3.1.3.3.3.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3.3"><times id="S5.Ex2.m1.3.3.3.1.3.3.3.1.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3.3.1"></times><ci id="S5.Ex2.m1.3.3.3.1.3.3.3.2.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3.3.2">𝑡</ci><ci id="S5.Ex2.m1.3.3.3.1.3.3.3.3.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3.3.3">𝑒</ci><ci id="S5.Ex2.m1.3.3.3.1.3.3.3.4.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3.3.4">𝑠</ci><ci id="S5.Ex2.m1.3.3.3.1.3.3.3.5.cmml" xref="S5.Ex2.m1.3.3.3.1.3.3.3.5">𝑡</ci></apply></apply></apply></apply><apply id="S5.Ex2.m1.3.3.3.2.cmml" xref="S5.Ex2.m1.3.3.3.2"><times id="S5.Ex2.m1.3.3.3.2.1.cmml" xref="S5.Ex2.m1.3.3.3.2.1"></times><apply id="S5.Ex2.m1.2.2.cmml" xref="S5.Ex2.m1.2.2"><divide id="S5.Ex2.m1.2.2.3.cmml" xref="S5.Ex2.m1.2.2"></divide><apply id="S5.Ex2.m1.2.2.2.3.cmml" xref="S5.Ex2.m1.2.2.2.2"><ci id="S5.Ex2.m1.1.1.1.1.cmml" xref="S5.Ex2.m1.1.1.1.1">Pr</ci><apply id="S5.Ex2.m1.2.2.2.2.1.1.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1"><eq id="S5.Ex2.m1.2.2.2.2.1.1.2.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.2"></eq><apply id="S5.Ex2.m1.2.2.2.2.1.1.1.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1"><times id="S5.Ex2.m1.2.2.2.2.1.1.1.2.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.2"></times><ci id="S5.Ex2.m1.2.2.2.2.1.1.1.3.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.3">𝑀</ci><ci id="S5.Ex2.m1.2.2.2.2.1.1.1.4.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.4">𝐼</ci><apply id="S5.Ex2.m1.2.2.2.2.1.1.1.5.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5"><csymbol cd="ambiguous" id="S5.Ex2.m1.2.2.2.2.1.1.1.5.1.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5">subscript</csymbol><ci id="S5.Ex2.m1.2.2.2.2.1.1.1.5.2.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.2">𝐴</ci><apply id="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.3"><csymbol cd="ambiguous" id="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.1.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.3">subscript</csymbol><ci id="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.2.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.2">𝑡</ci><ci id="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.3.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.5.3.3">𝑝</ci></apply></apply><apply id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.1.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1">subscript</csymbol><ci id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.2.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.2">𝑆</ci><ci id="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.3.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.1.1.1.1.3">𝑖</ci></apply></apply><cn type="integer" id="S5.Ex2.m1.2.2.2.2.1.1.3.cmml" xref="S5.Ex2.m1.2.2.2.2.1.1.3">1</cn></apply></apply><apply id="S5.Ex2.m1.2.2.4.cmml" xref="S5.Ex2.m1.2.2.4"><times id="S5.Ex2.m1.2.2.4.1.cmml" xref="S5.Ex2.m1.2.2.4.1"></times><cn type="integer" id="S5.Ex2.m1.2.2.4.2.cmml" xref="S5.Ex2.m1.2.2.4.2">2</cn><apply id="S5.Ex2.m1.2.2.4.3.cmml" xref="S5.Ex2.m1.2.2.4.3"><csymbol cd="ambiguous" id="S5.Ex2.m1.2.2.4.3.1.cmml" xref="S5.Ex2.m1.2.2.4.3">subscript</csymbol><ci id="S5.Ex2.m1.2.2.4.3.2.cmml" xref="S5.Ex2.m1.2.2.4.3.2">𝑛</ci><ci id="S5.Ex2.m1.2.2.4.3.3.cmml" xref="S5.Ex2.m1.2.2.4.3.3">𝑠</ci></apply></apply></apply><ci id="S5.Ex2.m1.3.3.3.2.2a.cmml" xref="S5.Ex2.m1.3.3.3.2.2"><mtext id="S5.Ex2.m1.3.3.3.2.2.cmml" xref="S5.Ex2.m1.3.3.3.2.2">, and</mtext></ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.Ex2.m1.3c">\overline{MIA_{t_{p}}}(S_{test})=\sum_{S_{i}\in S_{test}}\frac{\Pr[MIA_{t_{p}}(S_{i})=1]}{2*n_{s}}\mbox{, and}\vspace*{-0.1cm}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr></tbody>
</table>
<table id="S5.Ex3" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S5.Ex3.m1.3" class="ltx_Math" alttext="\overline{MIA_{t_{p}}}(R_{t})=\sum_{R_{i}\in R_{t}}\frac{\Pr[MIA_{t_{p}}(R_{i})=1]}{2*n_{s}}." display="block"><semantics id="S5.Ex3.m1.3a"><mrow id="S5.Ex3.m1.3.3.1" xref="S5.Ex3.m1.3.3.1.1.cmml"><mrow id="S5.Ex3.m1.3.3.1.1" xref="S5.Ex3.m1.3.3.1.1.cmml"><mrow id="S5.Ex3.m1.3.3.1.1.1" xref="S5.Ex3.m1.3.3.1.1.1.cmml"><mover accent="true" id="S5.Ex3.m1.3.3.1.1.1.3" xref="S5.Ex3.m1.3.3.1.1.1.3.cmml"><mrow id="S5.Ex3.m1.3.3.1.1.1.3.2" xref="S5.Ex3.m1.3.3.1.1.1.3.2.cmml"><mi id="S5.Ex3.m1.3.3.1.1.1.3.2.2" xref="S5.Ex3.m1.3.3.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.Ex3.m1.3.3.1.1.1.3.2.1" xref="S5.Ex3.m1.3.3.1.1.1.3.2.1.cmml">​</mo><mi id="S5.Ex3.m1.3.3.1.1.1.3.2.3" xref="S5.Ex3.m1.3.3.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.Ex3.m1.3.3.1.1.1.3.2.1a" xref="S5.Ex3.m1.3.3.1.1.1.3.2.1.cmml">​</mo><msub id="S5.Ex3.m1.3.3.1.1.1.3.2.4" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.cmml"><mi id="S5.Ex3.m1.3.3.1.1.1.3.2.4.2" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.2.cmml">A</mi><msub id="S5.Ex3.m1.3.3.1.1.1.3.2.4.3" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.cmml"><mi id="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.2" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.2.cmml">t</mi><mi id="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.3" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.Ex3.m1.3.3.1.1.1.3.1" xref="S5.Ex3.m1.3.3.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.Ex3.m1.3.3.1.1.1.2" xref="S5.Ex3.m1.3.3.1.1.1.2.cmml">​</mo><mrow id="S5.Ex3.m1.3.3.1.1.1.1.1" xref="S5.Ex3.m1.3.3.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.Ex3.m1.3.3.1.1.1.1.1.2" xref="S5.Ex3.m1.3.3.1.1.1.1.1.1.cmml">(</mo><msub id="S5.Ex3.m1.3.3.1.1.1.1.1.1" xref="S5.Ex3.m1.3.3.1.1.1.1.1.1.cmml"><mi id="S5.Ex3.m1.3.3.1.1.1.1.1.1.2" xref="S5.Ex3.m1.3.3.1.1.1.1.1.1.2.cmml">R</mi><mi id="S5.Ex3.m1.3.3.1.1.1.1.1.1.3" xref="S5.Ex3.m1.3.3.1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.Ex3.m1.3.3.1.1.1.1.1.3" xref="S5.Ex3.m1.3.3.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo rspace="0.111em" id="S5.Ex3.m1.3.3.1.1.2" xref="S5.Ex3.m1.3.3.1.1.2.cmml">=</mo><mrow id="S5.Ex3.m1.3.3.1.1.3" xref="S5.Ex3.m1.3.3.1.1.3.cmml"><munder id="S5.Ex3.m1.3.3.1.1.3.1" xref="S5.Ex3.m1.3.3.1.1.3.1.cmml"><mo movablelimits="false" id="S5.Ex3.m1.3.3.1.1.3.1.2" xref="S5.Ex3.m1.3.3.1.1.3.1.2.cmml">∑</mo><mrow id="S5.Ex3.m1.3.3.1.1.3.1.3" xref="S5.Ex3.m1.3.3.1.1.3.1.3.cmml"><msub id="S5.Ex3.m1.3.3.1.1.3.1.3.2" xref="S5.Ex3.m1.3.3.1.1.3.1.3.2.cmml"><mi id="S5.Ex3.m1.3.3.1.1.3.1.3.2.2" xref="S5.Ex3.m1.3.3.1.1.3.1.3.2.2.cmml">R</mi><mi id="S5.Ex3.m1.3.3.1.1.3.1.3.2.3" xref="S5.Ex3.m1.3.3.1.1.3.1.3.2.3.cmml">i</mi></msub><mo id="S5.Ex3.m1.3.3.1.1.3.1.3.1" xref="S5.Ex3.m1.3.3.1.1.3.1.3.1.cmml">∈</mo><msub id="S5.Ex3.m1.3.3.1.1.3.1.3.3" xref="S5.Ex3.m1.3.3.1.1.3.1.3.3.cmml"><mi id="S5.Ex3.m1.3.3.1.1.3.1.3.3.2" xref="S5.Ex3.m1.3.3.1.1.3.1.3.3.2.cmml">R</mi><mi id="S5.Ex3.m1.3.3.1.1.3.1.3.3.3" xref="S5.Ex3.m1.3.3.1.1.3.1.3.3.3.cmml">t</mi></msub></mrow></munder><mfrac id="S5.Ex3.m1.2.2" xref="S5.Ex3.m1.2.2.cmml"><mrow id="S5.Ex3.m1.2.2.2.2" xref="S5.Ex3.m1.2.2.2.3.cmml"><mi id="S5.Ex3.m1.1.1.1.1" xref="S5.Ex3.m1.1.1.1.1.cmml">Pr</mi><mo id="S5.Ex3.m1.2.2.2.2a" xref="S5.Ex3.m1.2.2.2.3.cmml">⁡</mo><mrow id="S5.Ex3.m1.2.2.2.2.1" xref="S5.Ex3.m1.2.2.2.3.cmml"><mo stretchy="false" id="S5.Ex3.m1.2.2.2.2.1.2" xref="S5.Ex3.m1.2.2.2.3.cmml">[</mo><mrow id="S5.Ex3.m1.2.2.2.2.1.1" xref="S5.Ex3.m1.2.2.2.2.1.1.cmml"><mrow id="S5.Ex3.m1.2.2.2.2.1.1.1" xref="S5.Ex3.m1.2.2.2.2.1.1.1.cmml"><mi id="S5.Ex3.m1.2.2.2.2.1.1.1.3" xref="S5.Ex3.m1.2.2.2.2.1.1.1.3.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.Ex3.m1.2.2.2.2.1.1.1.2" xref="S5.Ex3.m1.2.2.2.2.1.1.1.2.cmml">​</mo><mi id="S5.Ex3.m1.2.2.2.2.1.1.1.4" xref="S5.Ex3.m1.2.2.2.2.1.1.1.4.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.Ex3.m1.2.2.2.2.1.1.1.2a" xref="S5.Ex3.m1.2.2.2.2.1.1.1.2.cmml">​</mo><msub id="S5.Ex3.m1.2.2.2.2.1.1.1.5" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.cmml"><mi id="S5.Ex3.m1.2.2.2.2.1.1.1.5.2" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.2.cmml">A</mi><msub id="S5.Ex3.m1.2.2.2.2.1.1.1.5.3" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.cmml"><mi id="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.2" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.2.cmml">t</mi><mi id="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.3" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.3.cmml">p</mi></msub></msub><mo lspace="0em" rspace="0em" id="S5.Ex3.m1.2.2.2.2.1.1.1.2b" xref="S5.Ex3.m1.2.2.2.2.1.1.1.2.cmml">​</mo><mrow id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.2" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.cmml">(</mo><msub id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.cmml"><mi id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.2" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.2.cmml">R</mi><mi id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.3" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.3" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S5.Ex3.m1.2.2.2.2.1.1.2" xref="S5.Ex3.m1.2.2.2.2.1.1.2.cmml">=</mo><mn id="S5.Ex3.m1.2.2.2.2.1.1.3" xref="S5.Ex3.m1.2.2.2.2.1.1.3.cmml">1</mn></mrow><mo stretchy="false" id="S5.Ex3.m1.2.2.2.2.1.3" xref="S5.Ex3.m1.2.2.2.3.cmml">]</mo></mrow></mrow><mrow id="S5.Ex3.m1.2.2.4" xref="S5.Ex3.m1.2.2.4.cmml"><mn id="S5.Ex3.m1.2.2.4.2" xref="S5.Ex3.m1.2.2.4.2.cmml">2</mn><mo lspace="0.222em" rspace="0.222em" id="S5.Ex3.m1.2.2.4.1" xref="S5.Ex3.m1.2.2.4.1.cmml">∗</mo><msub id="S5.Ex3.m1.2.2.4.3" xref="S5.Ex3.m1.2.2.4.3.cmml"><mi id="S5.Ex3.m1.2.2.4.3.2" xref="S5.Ex3.m1.2.2.4.3.2.cmml">n</mi><mi id="S5.Ex3.m1.2.2.4.3.3" xref="S5.Ex3.m1.2.2.4.3.3.cmml">s</mi></msub></mrow></mfrac></mrow></mrow><mo lspace="0em" id="S5.Ex3.m1.3.3.1.2" xref="S5.Ex3.m1.3.3.1.1.cmml">.</mo></mrow><annotation-xml encoding="MathML-Content" id="S5.Ex3.m1.3b"><apply id="S5.Ex3.m1.3.3.1.1.cmml" xref="S5.Ex3.m1.3.3.1"><eq id="S5.Ex3.m1.3.3.1.1.2.cmml" xref="S5.Ex3.m1.3.3.1.1.2"></eq><apply id="S5.Ex3.m1.3.3.1.1.1.cmml" xref="S5.Ex3.m1.3.3.1.1.1"><times id="S5.Ex3.m1.3.3.1.1.1.2.cmml" xref="S5.Ex3.m1.3.3.1.1.1.2"></times><apply id="S5.Ex3.m1.3.3.1.1.1.3.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3"><ci id="S5.Ex3.m1.3.3.1.1.1.3.1.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.1">¯</ci><apply id="S5.Ex3.m1.3.3.1.1.1.3.2.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2"><times id="S5.Ex3.m1.3.3.1.1.1.3.2.1.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.1"></times><ci id="S5.Ex3.m1.3.3.1.1.1.3.2.2.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.2">𝑀</ci><ci id="S5.Ex3.m1.3.3.1.1.1.3.2.3.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.3">𝐼</ci><apply id="S5.Ex3.m1.3.3.1.1.1.3.2.4.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4"><csymbol cd="ambiguous" id="S5.Ex3.m1.3.3.1.1.1.3.2.4.1.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4">subscript</csymbol><ci id="S5.Ex3.m1.3.3.1.1.1.3.2.4.2.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.2">𝐴</ci><apply id="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.3"><csymbol cd="ambiguous" id="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.1.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.3">subscript</csymbol><ci id="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.2.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.2">𝑡</ci><ci id="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.3.cmml" xref="S5.Ex3.m1.3.3.1.1.1.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.Ex3.m1.3.3.1.1.1.1.1.1.cmml" xref="S5.Ex3.m1.3.3.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.Ex3.m1.3.3.1.1.1.1.1.1.1.cmml" xref="S5.Ex3.m1.3.3.1.1.1.1.1">subscript</csymbol><ci id="S5.Ex3.m1.3.3.1.1.1.1.1.1.2.cmml" xref="S5.Ex3.m1.3.3.1.1.1.1.1.1.2">𝑅</ci><ci id="S5.Ex3.m1.3.3.1.1.1.1.1.1.3.cmml" xref="S5.Ex3.m1.3.3.1.1.1.1.1.1.3">𝑡</ci></apply></apply><apply id="S5.Ex3.m1.3.3.1.1.3.cmml" xref="S5.Ex3.m1.3.3.1.1.3"><apply id="S5.Ex3.m1.3.3.1.1.3.1.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1"><csymbol cd="ambiguous" id="S5.Ex3.m1.3.3.1.1.3.1.1.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1">subscript</csymbol><sum id="S5.Ex3.m1.3.3.1.1.3.1.2.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.2"></sum><apply id="S5.Ex3.m1.3.3.1.1.3.1.3.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3"><in id="S5.Ex3.m1.3.3.1.1.3.1.3.1.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.1"></in><apply id="S5.Ex3.m1.3.3.1.1.3.1.3.2.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.2"><csymbol cd="ambiguous" id="S5.Ex3.m1.3.3.1.1.3.1.3.2.1.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.2">subscript</csymbol><ci id="S5.Ex3.m1.3.3.1.1.3.1.3.2.2.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.2.2">𝑅</ci><ci id="S5.Ex3.m1.3.3.1.1.3.1.3.2.3.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.2.3">𝑖</ci></apply><apply id="S5.Ex3.m1.3.3.1.1.3.1.3.3.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.3"><csymbol cd="ambiguous" id="S5.Ex3.m1.3.3.1.1.3.1.3.3.1.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.3">subscript</csymbol><ci id="S5.Ex3.m1.3.3.1.1.3.1.3.3.2.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.3.2">𝑅</ci><ci id="S5.Ex3.m1.3.3.1.1.3.1.3.3.3.cmml" xref="S5.Ex3.m1.3.3.1.1.3.1.3.3.3">𝑡</ci></apply></apply></apply><apply id="S5.Ex3.m1.2.2.cmml" xref="S5.Ex3.m1.2.2"><divide id="S5.Ex3.m1.2.2.3.cmml" xref="S5.Ex3.m1.2.2"></divide><apply id="S5.Ex3.m1.2.2.2.3.cmml" xref="S5.Ex3.m1.2.2.2.2"><ci id="S5.Ex3.m1.1.1.1.1.cmml" xref="S5.Ex3.m1.1.1.1.1">Pr</ci><apply id="S5.Ex3.m1.2.2.2.2.1.1.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1"><eq id="S5.Ex3.m1.2.2.2.2.1.1.2.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.2"></eq><apply id="S5.Ex3.m1.2.2.2.2.1.1.1.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1"><times id="S5.Ex3.m1.2.2.2.2.1.1.1.2.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.2"></times><ci id="S5.Ex3.m1.2.2.2.2.1.1.1.3.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.3">𝑀</ci><ci id="S5.Ex3.m1.2.2.2.2.1.1.1.4.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.4">𝐼</ci><apply id="S5.Ex3.m1.2.2.2.2.1.1.1.5.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5"><csymbol cd="ambiguous" id="S5.Ex3.m1.2.2.2.2.1.1.1.5.1.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5">subscript</csymbol><ci id="S5.Ex3.m1.2.2.2.2.1.1.1.5.2.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.2">𝐴</ci><apply id="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.3"><csymbol cd="ambiguous" id="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.1.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.3">subscript</csymbol><ci id="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.2.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.2">𝑡</ci><ci id="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.3.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.5.3.3">𝑝</ci></apply></apply><apply id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.1.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1">subscript</csymbol><ci id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.2.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.2">𝑅</ci><ci id="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.3.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.1.1.1.1.3">𝑖</ci></apply></apply><cn type="integer" id="S5.Ex3.m1.2.2.2.2.1.1.3.cmml" xref="S5.Ex3.m1.2.2.2.2.1.1.3">1</cn></apply></apply><apply id="S5.Ex3.m1.2.2.4.cmml" xref="S5.Ex3.m1.2.2.4"><times id="S5.Ex3.m1.2.2.4.1.cmml" xref="S5.Ex3.m1.2.2.4.1"></times><cn type="integer" id="S5.Ex3.m1.2.2.4.2.cmml" xref="S5.Ex3.m1.2.2.4.2">2</cn><apply id="S5.Ex3.m1.2.2.4.3.cmml" xref="S5.Ex3.m1.2.2.4.3"><csymbol cd="ambiguous" id="S5.Ex3.m1.2.2.4.3.1.cmml" xref="S5.Ex3.m1.2.2.4.3">subscript</csymbol><ci id="S5.Ex3.m1.2.2.4.3.2.cmml" xref="S5.Ex3.m1.2.2.4.3.2">𝑛</ci><ci id="S5.Ex3.m1.2.2.4.3.3.cmml" xref="S5.Ex3.m1.2.2.4.3.3">𝑠</ci></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.Ex3.m1.3c">\overline{MIA_{t_{p}}}(R_{t})=\sum_{R_{i}\in R_{t}}\frac{\Pr[MIA_{t_{p}}(R_{i})=1]}{2*n_{s}}.</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr></tbody>
</table>
</div>
<div id="S5.SS2.p5" class="ltx_para ltx_noindent">
<p id="S5.SS2.p5.1" class="ltx_p">That is, the privacy gain, in this case, is computed as the difference between the probability that the attacker, who has partial information about the target record, correctly identifies that the target is part of the real dataset versus the target being part of the training set used to generate the synthetic dataset.</p>
</div>
<div id="S5.SS2.p6" class="ltx_para">
<p id="S5.SS2.p6.9" class="ltx_p">As a result, PG now ranges between -0.5 and 0.5, where 0.5 means that having the real dataset <math id="S5.SS2.p6.1.m1.1" class="ltx_Math" alttext="R" display="inline"><semantics id="S5.SS2.p6.1.m1.1a"><mi id="S5.SS2.p6.1.m1.1.1" xref="S5.SS2.p6.1.m1.1.1.cmml">R</mi><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.1.m1.1b"><ci id="S5.SS2.p6.1.m1.1.1.cmml" xref="S5.SS2.p6.1.m1.1.1">𝑅</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.1.m1.1c">R</annotation></semantics></math> and the partial information <math id="S5.SS2.p6.2.m2.1" class="ltx_Math" alttext="t^{\prime}" display="inline"><semantics id="S5.SS2.p6.2.m2.1a"><msup id="S5.SS2.p6.2.m2.1.1" xref="S5.SS2.p6.2.m2.1.1.cmml"><mi id="S5.SS2.p6.2.m2.1.1.2" xref="S5.SS2.p6.2.m2.1.1.2.cmml">t</mi><mo id="S5.SS2.p6.2.m2.1.1.3" xref="S5.SS2.p6.2.m2.1.1.3.cmml">′</mo></msup><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.2.m2.1b"><apply id="S5.SS2.p6.2.m2.1.1.cmml" xref="S5.SS2.p6.2.m2.1.1"><csymbol cd="ambiguous" id="S5.SS2.p6.2.m2.1.1.1.cmml" xref="S5.SS2.p6.2.m2.1.1">superscript</csymbol><ci id="S5.SS2.p6.2.m2.1.1.2.cmml" xref="S5.SS2.p6.2.m2.1.1.2">𝑡</ci><ci id="S5.SS2.p6.2.m2.1.1.3.cmml" xref="S5.SS2.p6.2.m2.1.1.3">′</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.2.m2.1c">t^{\prime}</annotation></semantics></math> about the target allows the adversary to infer the membership of <math id="S5.SS2.p6.3.m3.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S5.SS2.p6.3.m3.1a"><mi id="S5.SS2.p6.3.m3.1.1" xref="S5.SS2.p6.3.m3.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.3.m3.1b"><ci id="S5.SS2.p6.3.m3.1.1.cmml" xref="S5.SS2.p6.3.m3.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.3.m3.1c">t</annotation></semantics></math> in <math id="S5.SS2.p6.4.m4.1" class="ltx_Math" alttext="R" display="inline"><semantics id="S5.SS2.p6.4.m4.1a"><mi id="S5.SS2.p6.4.m4.1.1" xref="S5.SS2.p6.4.m4.1.1.cmml">R</mi><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.4.m4.1b"><ci id="S5.SS2.p6.4.m4.1.1.cmml" xref="S5.SS2.p6.4.m4.1.1">𝑅</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.4.m4.1c">R</annotation></semantics></math>, while the synthetic dataset reduces the adversary’s chance of success (i.e. <math id="S5.SS2.p6.5.m5.1" class="ltx_Math" alttext="\overline{MIA_{t_{p}}}(R_{t})=1" display="inline"><semantics id="S5.SS2.p6.5.m5.1a"><mrow id="S5.SS2.p6.5.m5.1.1" xref="S5.SS2.p6.5.m5.1.1.cmml"><mrow id="S5.SS2.p6.5.m5.1.1.1" xref="S5.SS2.p6.5.m5.1.1.1.cmml"><mover accent="true" id="S5.SS2.p6.5.m5.1.1.1.3" xref="S5.SS2.p6.5.m5.1.1.1.3.cmml"><mrow id="S5.SS2.p6.5.m5.1.1.1.3.2" xref="S5.SS2.p6.5.m5.1.1.1.3.2.cmml"><mi id="S5.SS2.p6.5.m5.1.1.1.3.2.2" xref="S5.SS2.p6.5.m5.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.5.m5.1.1.1.3.2.1" xref="S5.SS2.p6.5.m5.1.1.1.3.2.1.cmml">​</mo><mi id="S5.SS2.p6.5.m5.1.1.1.3.2.3" xref="S5.SS2.p6.5.m5.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.5.m5.1.1.1.3.2.1a" xref="S5.SS2.p6.5.m5.1.1.1.3.2.1.cmml">​</mo><msub id="S5.SS2.p6.5.m5.1.1.1.3.2.4" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.cmml"><mi id="S5.SS2.p6.5.m5.1.1.1.3.2.4.2" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.2.cmml">A</mi><msub id="S5.SS2.p6.5.m5.1.1.1.3.2.4.3" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.cmml"><mi id="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.2" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.2.cmml">t</mi><mi id="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.3" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.SS2.p6.5.m5.1.1.1.3.1" xref="S5.SS2.p6.5.m5.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.SS2.p6.5.m5.1.1.1.2" xref="S5.SS2.p6.5.m5.1.1.1.2.cmml">​</mo><mrow id="S5.SS2.p6.5.m5.1.1.1.1.1" xref="S5.SS2.p6.5.m5.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS2.p6.5.m5.1.1.1.1.1.2" xref="S5.SS2.p6.5.m5.1.1.1.1.1.1.cmml">(</mo><msub id="S5.SS2.p6.5.m5.1.1.1.1.1.1" xref="S5.SS2.p6.5.m5.1.1.1.1.1.1.cmml"><mi id="S5.SS2.p6.5.m5.1.1.1.1.1.1.2" xref="S5.SS2.p6.5.m5.1.1.1.1.1.1.2.cmml">R</mi><mi id="S5.SS2.p6.5.m5.1.1.1.1.1.1.3" xref="S5.SS2.p6.5.m5.1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.SS2.p6.5.m5.1.1.1.1.1.3" xref="S5.SS2.p6.5.m5.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S5.SS2.p6.5.m5.1.1.2" xref="S5.SS2.p6.5.m5.1.1.2.cmml">=</mo><mn id="S5.SS2.p6.5.m5.1.1.3" xref="S5.SS2.p6.5.m5.1.1.3.cmml">1</mn></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.5.m5.1b"><apply id="S5.SS2.p6.5.m5.1.1.cmml" xref="S5.SS2.p6.5.m5.1.1"><eq id="S5.SS2.p6.5.m5.1.1.2.cmml" xref="S5.SS2.p6.5.m5.1.1.2"></eq><apply id="S5.SS2.p6.5.m5.1.1.1.cmml" xref="S5.SS2.p6.5.m5.1.1.1"><times id="S5.SS2.p6.5.m5.1.1.1.2.cmml" xref="S5.SS2.p6.5.m5.1.1.1.2"></times><apply id="S5.SS2.p6.5.m5.1.1.1.3.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3"><ci id="S5.SS2.p6.5.m5.1.1.1.3.1.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.1">¯</ci><apply id="S5.SS2.p6.5.m5.1.1.1.3.2.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2"><times id="S5.SS2.p6.5.m5.1.1.1.3.2.1.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.1"></times><ci id="S5.SS2.p6.5.m5.1.1.1.3.2.2.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.2">𝑀</ci><ci id="S5.SS2.p6.5.m5.1.1.1.3.2.3.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.3">𝐼</ci><apply id="S5.SS2.p6.5.m5.1.1.1.3.2.4.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4"><csymbol cd="ambiguous" id="S5.SS2.p6.5.m5.1.1.1.3.2.4.1.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4">subscript</csymbol><ci id="S5.SS2.p6.5.m5.1.1.1.3.2.4.2.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.2">𝐴</ci><apply id="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.3"><csymbol cd="ambiguous" id="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.1.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.3">subscript</csymbol><ci id="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.2.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.2">𝑡</ci><ci id="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.3.cmml" xref="S5.SS2.p6.5.m5.1.1.1.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.SS2.p6.5.m5.1.1.1.1.1.1.cmml" xref="S5.SS2.p6.5.m5.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS2.p6.5.m5.1.1.1.1.1.1.1.cmml" xref="S5.SS2.p6.5.m5.1.1.1.1.1">subscript</csymbol><ci id="S5.SS2.p6.5.m5.1.1.1.1.1.1.2.cmml" xref="S5.SS2.p6.5.m5.1.1.1.1.1.1.2">𝑅</ci><ci id="S5.SS2.p6.5.m5.1.1.1.1.1.1.3.cmml" xref="S5.SS2.p6.5.m5.1.1.1.1.1.1.3">𝑡</ci></apply></apply><cn type="integer" id="S5.SS2.p6.5.m5.1.1.3.cmml" xref="S5.SS2.p6.5.m5.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.5.m5.1c">\overline{MIA_{t_{p}}}(R_{t})=1</annotation></semantics></math> and <math id="S5.SS2.p6.6.m6.1" class="ltx_Math" alttext="\overline{MIA_{t_{p}}}(S_{test})=0" display="inline"><semantics id="S5.SS2.p6.6.m6.1a"><mrow id="S5.SS2.p6.6.m6.1.1" xref="S5.SS2.p6.6.m6.1.1.cmml"><mrow id="S5.SS2.p6.6.m6.1.1.1" xref="S5.SS2.p6.6.m6.1.1.1.cmml"><mover accent="true" id="S5.SS2.p6.6.m6.1.1.1.3" xref="S5.SS2.p6.6.m6.1.1.1.3.cmml"><mrow id="S5.SS2.p6.6.m6.1.1.1.3.2" xref="S5.SS2.p6.6.m6.1.1.1.3.2.cmml"><mi id="S5.SS2.p6.6.m6.1.1.1.3.2.2" xref="S5.SS2.p6.6.m6.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.6.m6.1.1.1.3.2.1" xref="S5.SS2.p6.6.m6.1.1.1.3.2.1.cmml">​</mo><mi id="S5.SS2.p6.6.m6.1.1.1.3.2.3" xref="S5.SS2.p6.6.m6.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.6.m6.1.1.1.3.2.1a" xref="S5.SS2.p6.6.m6.1.1.1.3.2.1.cmml">​</mo><msub id="S5.SS2.p6.6.m6.1.1.1.3.2.4" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.cmml"><mi id="S5.SS2.p6.6.m6.1.1.1.3.2.4.2" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.2.cmml">A</mi><msub id="S5.SS2.p6.6.m6.1.1.1.3.2.4.3" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.cmml"><mi id="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.2" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.2.cmml">t</mi><mi id="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.3" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.SS2.p6.6.m6.1.1.1.3.1" xref="S5.SS2.p6.6.m6.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.SS2.p6.6.m6.1.1.1.2" xref="S5.SS2.p6.6.m6.1.1.1.2.cmml">​</mo><mrow id="S5.SS2.p6.6.m6.1.1.1.1.1" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS2.p6.6.m6.1.1.1.1.1.2" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.cmml">(</mo><msub id="S5.SS2.p6.6.m6.1.1.1.1.1.1" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.cmml"><mi id="S5.SS2.p6.6.m6.1.1.1.1.1.1.2" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.2.cmml">S</mi><mrow id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.cmml"><mi id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.2" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.1" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.3" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.1a" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.4" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.1b" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.5" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S5.SS2.p6.6.m6.1.1.1.1.1.3" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S5.SS2.p6.6.m6.1.1.2" xref="S5.SS2.p6.6.m6.1.1.2.cmml">=</mo><mn id="S5.SS2.p6.6.m6.1.1.3" xref="S5.SS2.p6.6.m6.1.1.3.cmml">0</mn></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.6.m6.1b"><apply id="S5.SS2.p6.6.m6.1.1.cmml" xref="S5.SS2.p6.6.m6.1.1"><eq id="S5.SS2.p6.6.m6.1.1.2.cmml" xref="S5.SS2.p6.6.m6.1.1.2"></eq><apply id="S5.SS2.p6.6.m6.1.1.1.cmml" xref="S5.SS2.p6.6.m6.1.1.1"><times id="S5.SS2.p6.6.m6.1.1.1.2.cmml" xref="S5.SS2.p6.6.m6.1.1.1.2"></times><apply id="S5.SS2.p6.6.m6.1.1.1.3.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3"><ci id="S5.SS2.p6.6.m6.1.1.1.3.1.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.1">¯</ci><apply id="S5.SS2.p6.6.m6.1.1.1.3.2.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2"><times id="S5.SS2.p6.6.m6.1.1.1.3.2.1.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.1"></times><ci id="S5.SS2.p6.6.m6.1.1.1.3.2.2.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.2">𝑀</ci><ci id="S5.SS2.p6.6.m6.1.1.1.3.2.3.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.3">𝐼</ci><apply id="S5.SS2.p6.6.m6.1.1.1.3.2.4.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4"><csymbol cd="ambiguous" id="S5.SS2.p6.6.m6.1.1.1.3.2.4.1.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4">subscript</csymbol><ci id="S5.SS2.p6.6.m6.1.1.1.3.2.4.2.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.2">𝐴</ci><apply id="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.3"><csymbol cd="ambiguous" id="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.1.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.3">subscript</csymbol><ci id="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.2.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.2">𝑡</ci><ci id="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.3.cmml" xref="S5.SS2.p6.6.m6.1.1.1.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.SS2.p6.6.m6.1.1.1.1.1.1.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS2.p6.6.m6.1.1.1.1.1.1.1.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1">subscript</csymbol><ci id="S5.SS2.p6.6.m6.1.1.1.1.1.1.2.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.2">𝑆</ci><apply id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3"><times id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.1.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.1"></times><ci id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.2.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.2">𝑡</ci><ci id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.3.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.3">𝑒</ci><ci id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.4.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.4">𝑠</ci><ci id="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.5.cmml" xref="S5.SS2.p6.6.m6.1.1.1.1.1.1.3.5">𝑡</ci></apply></apply></apply><cn type="integer" id="S5.SS2.p6.6.m6.1.1.3.cmml" xref="S5.SS2.p6.6.m6.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.6.m6.1c">\overline{MIA_{t_{p}}}(S_{test})=0</annotation></semantics></math>).
A negative PG value means that publishing the synthetic data, instead of the real data, improves the adversary’s chance to correctly infer membership of the target <math id="S5.SS2.p6.7.m7.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S5.SS2.p6.7.m7.1a"><mi id="S5.SS2.p6.7.m7.1.1" xref="S5.SS2.p6.7.m7.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.7.m7.1b"><ci id="S5.SS2.p6.7.m7.1.1.cmml" xref="S5.SS2.p6.7.m7.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.7.m7.1c">t</annotation></semantics></math> (i.e. <math id="S5.SS2.p6.8.m8.2" class="ltx_Math" alttext="\overline{MIA_{t_{p}}}(R_{t})&lt;\overline{MIA_{t_{p}}}(S_{test})" display="inline"><semantics id="S5.SS2.p6.8.m8.2a"><mrow id="S5.SS2.p6.8.m8.2.2" xref="S5.SS2.p6.8.m8.2.2.cmml"><mrow id="S5.SS2.p6.8.m8.1.1.1" xref="S5.SS2.p6.8.m8.1.1.1.cmml"><mover accent="true" id="S5.SS2.p6.8.m8.1.1.1.3" xref="S5.SS2.p6.8.m8.1.1.1.3.cmml"><mrow id="S5.SS2.p6.8.m8.1.1.1.3.2" xref="S5.SS2.p6.8.m8.1.1.1.3.2.cmml"><mi id="S5.SS2.p6.8.m8.1.1.1.3.2.2" xref="S5.SS2.p6.8.m8.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.1.1.1.3.2.1" xref="S5.SS2.p6.8.m8.1.1.1.3.2.1.cmml">​</mo><mi id="S5.SS2.p6.8.m8.1.1.1.3.2.3" xref="S5.SS2.p6.8.m8.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.1.1.1.3.2.1a" xref="S5.SS2.p6.8.m8.1.1.1.3.2.1.cmml">​</mo><msub id="S5.SS2.p6.8.m8.1.1.1.3.2.4" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.cmml"><mi id="S5.SS2.p6.8.m8.1.1.1.3.2.4.2" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.2.cmml">A</mi><msub id="S5.SS2.p6.8.m8.1.1.1.3.2.4.3" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.cmml"><mi id="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.2" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.2.cmml">t</mi><mi id="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.3" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.SS2.p6.8.m8.1.1.1.3.1" xref="S5.SS2.p6.8.m8.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.1.1.1.2" xref="S5.SS2.p6.8.m8.1.1.1.2.cmml">​</mo><mrow id="S5.SS2.p6.8.m8.1.1.1.1.1" xref="S5.SS2.p6.8.m8.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS2.p6.8.m8.1.1.1.1.1.2" xref="S5.SS2.p6.8.m8.1.1.1.1.1.1.cmml">(</mo><msub id="S5.SS2.p6.8.m8.1.1.1.1.1.1" xref="S5.SS2.p6.8.m8.1.1.1.1.1.1.cmml"><mi id="S5.SS2.p6.8.m8.1.1.1.1.1.1.2" xref="S5.SS2.p6.8.m8.1.1.1.1.1.1.2.cmml">R</mi><mi id="S5.SS2.p6.8.m8.1.1.1.1.1.1.3" xref="S5.SS2.p6.8.m8.1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.SS2.p6.8.m8.1.1.1.1.1.3" xref="S5.SS2.p6.8.m8.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S5.SS2.p6.8.m8.2.2.3" xref="S5.SS2.p6.8.m8.2.2.3.cmml">&lt;</mo><mrow id="S5.SS2.p6.8.m8.2.2.2" xref="S5.SS2.p6.8.m8.2.2.2.cmml"><mover accent="true" id="S5.SS2.p6.8.m8.2.2.2.3" xref="S5.SS2.p6.8.m8.2.2.2.3.cmml"><mrow id="S5.SS2.p6.8.m8.2.2.2.3.2" xref="S5.SS2.p6.8.m8.2.2.2.3.2.cmml"><mi id="S5.SS2.p6.8.m8.2.2.2.3.2.2" xref="S5.SS2.p6.8.m8.2.2.2.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.2.2.2.3.2.1" xref="S5.SS2.p6.8.m8.2.2.2.3.2.1.cmml">​</mo><mi id="S5.SS2.p6.8.m8.2.2.2.3.2.3" xref="S5.SS2.p6.8.m8.2.2.2.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.2.2.2.3.2.1a" xref="S5.SS2.p6.8.m8.2.2.2.3.2.1.cmml">​</mo><msub id="S5.SS2.p6.8.m8.2.2.2.3.2.4" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.cmml"><mi id="S5.SS2.p6.8.m8.2.2.2.3.2.4.2" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.2.cmml">A</mi><msub id="S5.SS2.p6.8.m8.2.2.2.3.2.4.3" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.cmml"><mi id="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.2" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.2.cmml">t</mi><mi id="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.3" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.SS2.p6.8.m8.2.2.2.3.1" xref="S5.SS2.p6.8.m8.2.2.2.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.2.2.2.2" xref="S5.SS2.p6.8.m8.2.2.2.2.cmml">​</mo><mrow id="S5.SS2.p6.8.m8.2.2.2.1.1" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.cmml"><mo stretchy="false" id="S5.SS2.p6.8.m8.2.2.2.1.1.2" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.cmml">(</mo><msub id="S5.SS2.p6.8.m8.2.2.2.1.1.1" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.cmml"><mi id="S5.SS2.p6.8.m8.2.2.2.1.1.1.2" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.2.cmml">S</mi><mrow id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.cmml"><mi id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.2" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.1" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.3" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.1a" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.4" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.1b" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.5" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S5.SS2.p6.8.m8.2.2.2.1.1.3" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.cmml">)</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.8.m8.2b"><apply id="S5.SS2.p6.8.m8.2.2.cmml" xref="S5.SS2.p6.8.m8.2.2"><lt id="S5.SS2.p6.8.m8.2.2.3.cmml" xref="S5.SS2.p6.8.m8.2.2.3"></lt><apply id="S5.SS2.p6.8.m8.1.1.1.cmml" xref="S5.SS2.p6.8.m8.1.1.1"><times id="S5.SS2.p6.8.m8.1.1.1.2.cmml" xref="S5.SS2.p6.8.m8.1.1.1.2"></times><apply id="S5.SS2.p6.8.m8.1.1.1.3.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3"><ci id="S5.SS2.p6.8.m8.1.1.1.3.1.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.1">¯</ci><apply id="S5.SS2.p6.8.m8.1.1.1.3.2.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2"><times id="S5.SS2.p6.8.m8.1.1.1.3.2.1.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.1"></times><ci id="S5.SS2.p6.8.m8.1.1.1.3.2.2.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.2">𝑀</ci><ci id="S5.SS2.p6.8.m8.1.1.1.3.2.3.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.3">𝐼</ci><apply id="S5.SS2.p6.8.m8.1.1.1.3.2.4.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4"><csymbol cd="ambiguous" id="S5.SS2.p6.8.m8.1.1.1.3.2.4.1.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4">subscript</csymbol><ci id="S5.SS2.p6.8.m8.1.1.1.3.2.4.2.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.2">𝐴</ci><apply id="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.3"><csymbol cd="ambiguous" id="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.1.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.3">subscript</csymbol><ci id="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.2.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.2">𝑡</ci><ci id="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.3.cmml" xref="S5.SS2.p6.8.m8.1.1.1.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.SS2.p6.8.m8.1.1.1.1.1.1.cmml" xref="S5.SS2.p6.8.m8.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS2.p6.8.m8.1.1.1.1.1.1.1.cmml" xref="S5.SS2.p6.8.m8.1.1.1.1.1">subscript</csymbol><ci id="S5.SS2.p6.8.m8.1.1.1.1.1.1.2.cmml" xref="S5.SS2.p6.8.m8.1.1.1.1.1.1.2">𝑅</ci><ci id="S5.SS2.p6.8.m8.1.1.1.1.1.1.3.cmml" xref="S5.SS2.p6.8.m8.1.1.1.1.1.1.3">𝑡</ci></apply></apply><apply id="S5.SS2.p6.8.m8.2.2.2.cmml" xref="S5.SS2.p6.8.m8.2.2.2"><times id="S5.SS2.p6.8.m8.2.2.2.2.cmml" xref="S5.SS2.p6.8.m8.2.2.2.2"></times><apply id="S5.SS2.p6.8.m8.2.2.2.3.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3"><ci id="S5.SS2.p6.8.m8.2.2.2.3.1.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.1">¯</ci><apply id="S5.SS2.p6.8.m8.2.2.2.3.2.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2"><times id="S5.SS2.p6.8.m8.2.2.2.3.2.1.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.1"></times><ci id="S5.SS2.p6.8.m8.2.2.2.3.2.2.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.2">𝑀</ci><ci id="S5.SS2.p6.8.m8.2.2.2.3.2.3.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.3">𝐼</ci><apply id="S5.SS2.p6.8.m8.2.2.2.3.2.4.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4"><csymbol cd="ambiguous" id="S5.SS2.p6.8.m8.2.2.2.3.2.4.1.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4">subscript</csymbol><ci id="S5.SS2.p6.8.m8.2.2.2.3.2.4.2.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.2">𝐴</ci><apply id="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.3"><csymbol cd="ambiguous" id="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.1.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.3">subscript</csymbol><ci id="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.2.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.2">𝑡</ci><ci id="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.3.cmml" xref="S5.SS2.p6.8.m8.2.2.2.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.SS2.p6.8.m8.2.2.2.1.1.1.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1"><csymbol cd="ambiguous" id="S5.SS2.p6.8.m8.2.2.2.1.1.1.1.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1">subscript</csymbol><ci id="S5.SS2.p6.8.m8.2.2.2.1.1.1.2.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.2">𝑆</ci><apply id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3"><times id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.1.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.1"></times><ci id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.2.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.2">𝑡</ci><ci id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.3.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.3">𝑒</ci><ci id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.4.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.4">𝑠</ci><ci id="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.5.cmml" xref="S5.SS2.p6.8.m8.2.2.2.1.1.1.3.5">𝑡</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.8.m8.2c">\overline{MIA_{t_{p}}}(R_{t})&lt;\overline{MIA_{t_{p}}}(S_{test})</annotation></semantics></math>).
If publishing the synthetic data does not increase nor decrease the adversary’s inference powers, we should have PG=0 (i.e. <math id="S5.SS2.p6.9.m9.2" class="ltx_Math" alttext="\overline{MIA_{t_{p}}}(R_{t})=\overline{MIA_{t_{p}}}(S_{test})" display="inline"><semantics id="S5.SS2.p6.9.m9.2a"><mrow id="S5.SS2.p6.9.m9.2.2" xref="S5.SS2.p6.9.m9.2.2.cmml"><mrow id="S5.SS2.p6.9.m9.1.1.1" xref="S5.SS2.p6.9.m9.1.1.1.cmml"><mover accent="true" id="S5.SS2.p6.9.m9.1.1.1.3" xref="S5.SS2.p6.9.m9.1.1.1.3.cmml"><mrow id="S5.SS2.p6.9.m9.1.1.1.3.2" xref="S5.SS2.p6.9.m9.1.1.1.3.2.cmml"><mi id="S5.SS2.p6.9.m9.1.1.1.3.2.2" xref="S5.SS2.p6.9.m9.1.1.1.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.1.1.1.3.2.1" xref="S5.SS2.p6.9.m9.1.1.1.3.2.1.cmml">​</mo><mi id="S5.SS2.p6.9.m9.1.1.1.3.2.3" xref="S5.SS2.p6.9.m9.1.1.1.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.1.1.1.3.2.1a" xref="S5.SS2.p6.9.m9.1.1.1.3.2.1.cmml">​</mo><msub id="S5.SS2.p6.9.m9.1.1.1.3.2.4" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.cmml"><mi id="S5.SS2.p6.9.m9.1.1.1.3.2.4.2" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.2.cmml">A</mi><msub id="S5.SS2.p6.9.m9.1.1.1.3.2.4.3" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.cmml"><mi id="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.2" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.2.cmml">t</mi><mi id="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.3" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.SS2.p6.9.m9.1.1.1.3.1" xref="S5.SS2.p6.9.m9.1.1.1.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.1.1.1.2" xref="S5.SS2.p6.9.m9.1.1.1.2.cmml">​</mo><mrow id="S5.SS2.p6.9.m9.1.1.1.1.1" xref="S5.SS2.p6.9.m9.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS2.p6.9.m9.1.1.1.1.1.2" xref="S5.SS2.p6.9.m9.1.1.1.1.1.1.cmml">(</mo><msub id="S5.SS2.p6.9.m9.1.1.1.1.1.1" xref="S5.SS2.p6.9.m9.1.1.1.1.1.1.cmml"><mi id="S5.SS2.p6.9.m9.1.1.1.1.1.1.2" xref="S5.SS2.p6.9.m9.1.1.1.1.1.1.2.cmml">R</mi><mi id="S5.SS2.p6.9.m9.1.1.1.1.1.1.3" xref="S5.SS2.p6.9.m9.1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.SS2.p6.9.m9.1.1.1.1.1.3" xref="S5.SS2.p6.9.m9.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S5.SS2.p6.9.m9.2.2.3" xref="S5.SS2.p6.9.m9.2.2.3.cmml">=</mo><mrow id="S5.SS2.p6.9.m9.2.2.2" xref="S5.SS2.p6.9.m9.2.2.2.cmml"><mover accent="true" id="S5.SS2.p6.9.m9.2.2.2.3" xref="S5.SS2.p6.9.m9.2.2.2.3.cmml"><mrow id="S5.SS2.p6.9.m9.2.2.2.3.2" xref="S5.SS2.p6.9.m9.2.2.2.3.2.cmml"><mi id="S5.SS2.p6.9.m9.2.2.2.3.2.2" xref="S5.SS2.p6.9.m9.2.2.2.3.2.2.cmml">M</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.2.2.2.3.2.1" xref="S5.SS2.p6.9.m9.2.2.2.3.2.1.cmml">​</mo><mi id="S5.SS2.p6.9.m9.2.2.2.3.2.3" xref="S5.SS2.p6.9.m9.2.2.2.3.2.3.cmml">I</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.2.2.2.3.2.1a" xref="S5.SS2.p6.9.m9.2.2.2.3.2.1.cmml">​</mo><msub id="S5.SS2.p6.9.m9.2.2.2.3.2.4" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.cmml"><mi id="S5.SS2.p6.9.m9.2.2.2.3.2.4.2" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.2.cmml">A</mi><msub id="S5.SS2.p6.9.m9.2.2.2.3.2.4.3" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.cmml"><mi id="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.2" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.2.cmml">t</mi><mi id="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.3" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.3.cmml">p</mi></msub></msub></mrow><mo id="S5.SS2.p6.9.m9.2.2.2.3.1" xref="S5.SS2.p6.9.m9.2.2.2.3.1.cmml">¯</mo></mover><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.2.2.2.2" xref="S5.SS2.p6.9.m9.2.2.2.2.cmml">​</mo><mrow id="S5.SS2.p6.9.m9.2.2.2.1.1" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.cmml"><mo stretchy="false" id="S5.SS2.p6.9.m9.2.2.2.1.1.2" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.cmml">(</mo><msub id="S5.SS2.p6.9.m9.2.2.2.1.1.1" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.cmml"><mi id="S5.SS2.p6.9.m9.2.2.2.1.1.1.2" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.2.cmml">S</mi><mrow id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.cmml"><mi id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.2" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.1" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.3" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.1a" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.4" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.4.cmml">s</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.1b" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.1.cmml">​</mo><mi id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.5" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.5.cmml">t</mi></mrow></msub><mo stretchy="false" id="S5.SS2.p6.9.m9.2.2.2.1.1.3" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.cmml">)</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.p6.9.m9.2b"><apply id="S5.SS2.p6.9.m9.2.2.cmml" xref="S5.SS2.p6.9.m9.2.2"><eq id="S5.SS2.p6.9.m9.2.2.3.cmml" xref="S5.SS2.p6.9.m9.2.2.3"></eq><apply id="S5.SS2.p6.9.m9.1.1.1.cmml" xref="S5.SS2.p6.9.m9.1.1.1"><times id="S5.SS2.p6.9.m9.1.1.1.2.cmml" xref="S5.SS2.p6.9.m9.1.1.1.2"></times><apply id="S5.SS2.p6.9.m9.1.1.1.3.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3"><ci id="S5.SS2.p6.9.m9.1.1.1.3.1.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.1">¯</ci><apply id="S5.SS2.p6.9.m9.1.1.1.3.2.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2"><times id="S5.SS2.p6.9.m9.1.1.1.3.2.1.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.1"></times><ci id="S5.SS2.p6.9.m9.1.1.1.3.2.2.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.2">𝑀</ci><ci id="S5.SS2.p6.9.m9.1.1.1.3.2.3.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.3">𝐼</ci><apply id="S5.SS2.p6.9.m9.1.1.1.3.2.4.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4"><csymbol cd="ambiguous" id="S5.SS2.p6.9.m9.1.1.1.3.2.4.1.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4">subscript</csymbol><ci id="S5.SS2.p6.9.m9.1.1.1.3.2.4.2.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.2">𝐴</ci><apply id="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.3"><csymbol cd="ambiguous" id="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.1.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.3">subscript</csymbol><ci id="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.2.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.2">𝑡</ci><ci id="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.3.cmml" xref="S5.SS2.p6.9.m9.1.1.1.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.SS2.p6.9.m9.1.1.1.1.1.1.cmml" xref="S5.SS2.p6.9.m9.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS2.p6.9.m9.1.1.1.1.1.1.1.cmml" xref="S5.SS2.p6.9.m9.1.1.1.1.1">subscript</csymbol><ci id="S5.SS2.p6.9.m9.1.1.1.1.1.1.2.cmml" xref="S5.SS2.p6.9.m9.1.1.1.1.1.1.2">𝑅</ci><ci id="S5.SS2.p6.9.m9.1.1.1.1.1.1.3.cmml" xref="S5.SS2.p6.9.m9.1.1.1.1.1.1.3">𝑡</ci></apply></apply><apply id="S5.SS2.p6.9.m9.2.2.2.cmml" xref="S5.SS2.p6.9.m9.2.2.2"><times id="S5.SS2.p6.9.m9.2.2.2.2.cmml" xref="S5.SS2.p6.9.m9.2.2.2.2"></times><apply id="S5.SS2.p6.9.m9.2.2.2.3.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3"><ci id="S5.SS2.p6.9.m9.2.2.2.3.1.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.1">¯</ci><apply id="S5.SS2.p6.9.m9.2.2.2.3.2.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2"><times id="S5.SS2.p6.9.m9.2.2.2.3.2.1.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.1"></times><ci id="S5.SS2.p6.9.m9.2.2.2.3.2.2.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.2">𝑀</ci><ci id="S5.SS2.p6.9.m9.2.2.2.3.2.3.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.3">𝐼</ci><apply id="S5.SS2.p6.9.m9.2.2.2.3.2.4.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4"><csymbol cd="ambiguous" id="S5.SS2.p6.9.m9.2.2.2.3.2.4.1.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4">subscript</csymbol><ci id="S5.SS2.p6.9.m9.2.2.2.3.2.4.2.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.2">𝐴</ci><apply id="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.3"><csymbol cd="ambiguous" id="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.1.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.3">subscript</csymbol><ci id="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.2.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.2">𝑡</ci><ci id="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.3.cmml" xref="S5.SS2.p6.9.m9.2.2.2.3.2.4.3.3">𝑝</ci></apply></apply></apply></apply><apply id="S5.SS2.p6.9.m9.2.2.2.1.1.1.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1"><csymbol cd="ambiguous" id="S5.SS2.p6.9.m9.2.2.2.1.1.1.1.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1">subscript</csymbol><ci id="S5.SS2.p6.9.m9.2.2.2.1.1.1.2.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.2">𝑆</ci><apply id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3"><times id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.1.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.1"></times><ci id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.2.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.2">𝑡</ci><ci id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.3.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.3">𝑒</ci><ci id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.4.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.4">𝑠</ci><ci id="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.5.cmml" xref="S5.SS2.p6.9.m9.2.2.2.1.1.1.3.5">𝑡</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p6.9.m9.2c">\overline{MIA_{t_{p}}}(R_{t})=\overline{MIA_{t_{p}}}(S_{test})</annotation></semantics></math>).</p>
</div>
<div id="S5.SS2.p7" class="ltx_para ltx_noindent">
<p id="S5.SS2.p7.1" class="ltx_p"><span id="S5.SS2.p7.1.1" class="ltx_text ltx_font_bold">Experiments.</span> From the experiments presented in Section <a href="#S5.SS1" title="5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5.1</span></a>, we find that the attack classifier that yields the lowest PG is Logistic Regression; thus, we only experiment with that one to ease presentation.
In the following, we present the results of the MIA-PI experiments for the <span id="S5.SS2.p7.1.2" class="ltx_text ltx_font_bold ltx_font_italic">CEU population</span>, focusing on the <span id="S5.SS2.p7.1.3" class="ltx_text ltx_font_bold ltx_font_italic">Recomb and RBM models</span> (as mentioned, with a LogReg attack classifier).
We do so as these two models yield the lowest PG in Section <a href="#S5.SS1" title="5.1 Privacy Gain Under Membership Inference Attack ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">5.1</span></a>.</p>
</div>
<div id="S5.SS2.p8" class="ltx_para ltx_noindent">
<p id="S5.SS2.p8.1" class="ltx_p"><span id="S5.SS2.p8.1.1" class="ltx_text ltx_font_bold">Recomb.</span> In Figure <a href="#S5.F7" title="Figure 7 ‣ 5.2 Privacy Gain under Membership Inference Attack with Partial Information ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>, we plot the Cumulative Distribution Function (CDF) of the accuracy of the attack for Recomb when the adversary has access to the full sequence vs. partial information, specifically, a ratio of 0.05, 0.1, and 0.2 of the total SNVs from the target sequence.
Interestingly, even when only 0.05 of the target SNVs are available to the attacker, for 90% and 91% of the targets from the Histogram and respectively Ensemble feature sets, the attacker’s accuracy is still above the random guess baseline (50% accuracy).
Our intuition is that many targets are vulnerable to the attack, even with little partial information, since we use the Recomb model not only for the attack but also as an inference method to predict the rest of the sequence.</p>
</div>
<div id="S5.SS2.p9" class="ltx_para">
<p id="S5.SS2.p9.1" class="ltx_p">To explore how much of the MIA-PI vulnerability is due to the release of synthetic datasets, and not only by how much information the attacker has available, in Figure <a href="#S5.F8" title="Figure 8 ‣ 5.2 Privacy Gain under Membership Inference Attack with Partial Information ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>, we plot the CDF of the PG with MIA-PI.
In line with the accuracy results, we find that the PG is greater than 0 for at least 88% of the targets for all ratios of partial information tested in the case of the Correlations feature set.
However, for the other three feature sets, releasing the synthetic dataset instead of the real data decreases the privacy gain (i.e., PG&lt;0) for the majority of targets.
When the adversary has access to just 5% of the SNVs from the target, there is a negative PG for 61% of the targets under the Histogram and 62% of the targets under the Ensemble feature sets.
With 10% of the target sequence available, 54% and 60% of the targets under, respectively, the Histogram and the Ensemble feature sets have negative PG, and with 20%, these numbers go up to 64% and 67%.
There are more targets with negative PG with increasing partial information available to the attacker for the Naive feature set, i.e., 59%, 70%, and 84% with, respectively, 5%, 10%, and 20% of the target sequence available.
Overall, this shows that releasing the synthetic dataset instead of the synthetic data does not mitigate privacy, even when the attacker does not have access to the full sequence.</p>
</div>
<figure id="S5.F7" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/graphs/plots/legend.png" id="S5.F7.g1" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="105" height="13" alt="Refer to caption"></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/graphs/plots/mia_ceu_recomb_partial.png" id="S5.F7.g2" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="299" height="70" alt="Refer to caption"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 7: </span>Accuracy of the Membership Inference Attack with access to full and partial information (0.05, 0.1, and 0.2 ratio) for <span id="S5.F7.2.1" class="ltx_text ltx_font_bold ltx_font_italic">Recomb</span>.</figcaption>
</figure>
<figure id="S5.F8" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/graphs/plots/legend.png" id="S5.F8.g1" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="105" height="13" alt="Refer to caption"></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/x13.png" id="S5.F8.g2" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="230" height="51" alt="Refer to caption"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 8: </span>Privacy Gain (PG) for synthetic samples from <span id="S5.F8.2.1" class="ltx_text ltx_font_bold ltx_font_italic">Recomb</span>.</figcaption>
</figure>
<div id="S5.SS2.p10" class="ltx_para ltx_noindent">
<p id="S5.SS2.p10.1" class="ltx_p"><span id="S5.SS2.p10.1.1" class="ltx_text ltx_font_bold">RBM.</span> In Figure <a href="#S5.F9" title="Figure 9 ‣ 5.2 Privacy Gain under Membership Inference Attack with Partial Information ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">9</span></a>, we plot the CDF for the accuracy of the attack for the RBM model for both full and partial information about the target record available to the attacker.
Across all feature sets, there is an increase in the accuracy of the attack with more information available to the attacker, as is expected.
We also look at CDF for the PG in the case of partial information available to the attacker in Figure <a href="#S5.F10" title="Figure 10 ‣ 5.2 Privacy Gain under Membership Inference Attack with Partial Information ‣ 5 Privacy Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a>.
Once again, under the Naive feature set, increasing the partial information available to the attacker negatively correlates with the percentage of targets with a negative PG.
Under all other feature sets, for most targets, releasing the synthetic dataset instead of the real data yields a positive PG, meaning that releasing the synthetic dataset instead of the real dataset improves the PG.</p>
</div>
<div id="S5.SS2.p11" class="ltx_para ltx_noindent">
<p id="S5.SS2.p11.1" class="ltx_p"><span id="S5.SS2.p11.1.1" class="ltx_text ltx_font_bold">Takeaways.</span> We find that not even decreasing the attacker’s power by only giving him partial information from the target sequence mitigates privacy for the Recomb-generated synthetic data.
This is likely because, using the Recomb model as both generative and inference model, the adversary’s power is increased since the feature set extracted from the synthetic data will be closer to the feature set for the predicted target.</p>
</div>
<div id="S5.SS2.p12" class="ltx_para">
<p id="S5.SS2.p12.1" class="ltx_p">However, in this case, for RBM, we see an increase in the privacy gained by releasing synthetic data instead of real data.
This implies that, even if the RBM is likely to overfit when few samples are available for training, it does so on the predicted sequence of the target rather than on the full sequence and thus decreases the accuracy of the MIA.</p>
</div>
<div id="S5.SS2.p13" class="ltx_para">
<p id="S5.SS2.p13.1" class="ltx_p">Overall, not even with partial data from the target sequence, we obtain privacy gain values constantly better than random guessing, which, as mentioned before, indicates that synthetic data is not really a reliable privacy defense.</p>
</div>
<figure id="S5.F9" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/graphs/plots/legend.png" id="S5.F9.g1" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="105" height="13" alt="Refer to caption"></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/graphs/plots/mia_ceu_rbm_partial.png" id="S5.F9.g2" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="299" height="71" alt="Refer to caption"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 9: </span>Accuracy of the Membership Inference Attack with access to full and partial information (0.05, 0.1, and 0.2 ratio) for <span id="S5.F9.2.1" class="ltx_text ltx_font_bold ltx_font_italic">RBM</span>.</figcaption>
</figure>
<figure id="S5.F10" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/graphs/plots/legend.png" id="S5.F10.g1" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="105" height="13" alt="Refer to caption"></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/x14.png" id="S5.F10.g2" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="230" height="51" alt="Refer to caption"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 10: </span>Privacy Gain (PG) for synthetic samples generated by <span id="S5.F10.2.1" class="ltx_text ltx_font_bold ltx_font_italic">RBM</span>.</figcaption>
</figure>
</section>
</section>
<section id="S6" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Related Work</h2>

<div id="S6.p1" class="ltx_para">
<p id="S6.p1.1" class="ltx_p">In this section, we review related work on synthetic data, genomic privacy, and MIAs against machine learning models.</p>
</div>
<div id="S6.p2" class="ltx_para ltx_noindent">
<p id="S6.p2.1" class="ltx_p"><span id="S6.p2.1.1" class="ltx_text ltx_font_bold">Synthetic Data Initiatives.</span>
In recent years, researchers have focused on generating synthetic electronic health records (EHR), aiming to facilitate research in and adoption of machine learning in medicine.
Choi et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib13" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">13</span></a>]</cite> use a combination of
an autoencoder with a GAN model, called medGAN, to generate high-dimensional multi-label discrete data.
ADS-GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib72" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">72</span></a>]</cite> uses a quantifiable definition for “identifiability” combined with the discriminator’s loss to minimize the probability of patient’s re-identification, while CorGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib61" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">61</span></a>]</cite> combines convolutional GANs and convolutional autoencoders to capture the correlations between adjacent medical features.
Biswal et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">8</span></a>]</cite> use variational autoencoder to synthesize sequences of discrete EHR encounters and encounter features.</p>
</div>
<div id="S6.p3" class="ltx_para">
<p id="S6.p3.1" class="ltx_p">Other initiatives focus on synthetic data modeled on primary care data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib68" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">68</span></a>, <a href="#bib.bib63" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">63</span></a>, <a href="#bib.bib45" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">45</span></a>, <a href="#bib.bib64" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">64</span></a>]</cite>.
Researchers have also explored generating synthetic health patient data to detect cancer and other diseases, e.g., RDP-CGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib62" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">62</span></a>]</cite> combines convolutional GANs and convolutional autoencoders, both trained with Rényi differential privacy <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib39" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">39</span></a>]</cite>.</p>
</div>
<div id="S6.p4" class="ltx_para">
<p id="S6.p4.1" class="ltx_p">Specific to genomes is the work presented in Section <a href="#S3" title="3 Synthetic Data Approaches in Genomics ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">54</span></a>, <a href="#bib.bib71" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">71</span></a>, <a href="#bib.bib36" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">36</span></a>]</cite>, which we have evaluated in terms of their utility–privacy tradeoffs.</p>
</div>
<div id="S6.p5" class="ltx_para ltx_noindent">
<p id="S6.p5.1" class="ltx_p"><span id="S6.p5.1.1" class="ltx_text ltx_font_bold">Privacy in Genomics.</span> Researchers have focused on studying and mitigating privacy risks in genomics.
One of the first attacks on genomic data is the Membership Inference Attack proposed by Homer et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib29" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">29</span></a>]</cite>, showing that an adversary can infer the presence of an individual’s genotype within a complex DNA mixture.
This attack has been improved by Wang et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib66" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">66</span></a>]</cite> using correlation statistics of a few hundred SNPs.
Then, Im et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib32" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">32</span></a>]</cite> show that the summary information from genome-wide association studies, such as regression coefficients, can also reveal an individual’s participation within the respective study.
Membership inference has also been shown possible in the context of the Beacon network <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib56" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">56</span></a>, <a href="#bib.bib49" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">49</span></a>, <a href="#bib.bib65" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">65</span></a>]</cite>, a federated service that answers queries of the form “does your data have a specific nucleotide at a specific genomic coordinate?”.</p>
</div>
<div id="S6.p6" class="ltx_para">
<p id="S6.p6.1" class="ltx_p">Chen et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib12" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">12</span></a>]</cite> study the effects of differential privacy protection against membership inference attack on machine learning for genomic data. However, their study is focused on privacy leakage via providing access to trained classification models, whereas we study the privacy leakage from sharing synthetic datasets.</p>
</div>
<div id="S6.p7" class="ltx_para ltx_noindent">
<p id="S6.p7.1" class="ltx_p"><span id="S6.p7.1.1" class="ltx_text ltx_font_bold">MIAs against Machine Learning Models.</span>
MIAs have long been studied in the context of machine learning.
Shokri et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib55" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">55</span></a>]</cite> present the first attack against discriminative models, aiming to identify whether a data record was used in training, using an approach based on shadow models.
Hui et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib30" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">30</span></a>]</cite> also propose an attack against discriminative models, which probes the target model and infers membership directly from the probes instead of shadow models.
Hayes et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib24" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">24</span></a>]</cite> present the first MIA against generative models like GANs; they use a discriminator to output the data with the highest confidence values as the original training data.
Hilprecht et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib26" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">26</span></a>]</cite> study MIAs against both GANs and Variational AutoEncoders (VAEs),
while Chen et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib11" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">11</span></a>]</cite> propose a generic MIA model against GANs.</p>
</div>
<div id="S6.p8" class="ltx_para">
<p id="S6.p8.1" class="ltx_p">Stadler et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">59</span></a>]</cite> recently evaluate MIAs in the context of synthetic data and show that even access to a single synthetic dataset output by the target model can lead to privacy leakage.
We re-use their framework for quantifying the privacy gain when a synthetic dataset is released instead of the real dataset.
However, not only do we do so for a specific context (namely, genomics), but we also measure utility (while they only study privacy).
In fact, there are several distinguishing characteristics between <span id="S6.p8.1.1" class="ltx_text ltx_font_italic">“generic”</span> synthetic data and <span id="S6.p8.1.2" class="ltx_text ltx_font_italic">genomic</span> data, which makes the evaluation significantly different, if not harder; paramount among these is the fact that all features found in genomic sequences are correlated with each other, unlike with generic datasets.
Finally, we introduce and measure a novel attack whereby the attacker only has access to partial genomic information about the target.</p>
</div>
</section>
<section id="S7" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">7 </span>Conclusion</h2>

<div id="S7.p1" class="ltx_para">
<p id="S7.p1.1" class="ltx_p">This paper presented an in-depth measurement of state-of-the-art methods to generate synthetic genomic data.
We did so vis-à-vis their utility, with respect to several common analytical tasks performed by researchers and the privacy protection they provide compared to releasing real data.</p>
</div>
<div id="S7.p2" class="ltx_para">
<p id="S7.p2.1" class="ltx_p">High-quality synthetic data must accurately capture the relations between data points; however, this can enable attackers to infer sensitive information about the training data used to generate the synthetic data.
This was illustrated by the performance of the Recomb model on the HapMap datasets: while it achieves the best utility, it does so at the cost of significantly reducing privacy.</p>
</div>
<div id="S7.p3" class="ltx_para ltx_noindent">
<p id="S7.p3.1" class="ltx_p"><span id="S7.p3.1.1" class="ltx_text ltx_font_bold">Discussion.</span> Overall, we found that there is no single method that outperforms the others for all metrics and all datasets.
While this is perhaps not surprising, we are the first to present a systematic, re-usable methodology to analyze all kinds of methods to generate synthetic genomic data.
For instance, this allows us to be the first to show that, unlike what previously suggested in previous work, models based on a simple GAN architecture (i.e., GAN and Rec-GAN) are not a good fit for genomic data. In fact, these provide low (the lowest) utility across the board.
Our methodology also allowed us to shed light on the influence of the size of the training dataset, especially in the case of generative models.
For example, utility improves wit more samples in the hybrid Rec-RBM for the smaller HapMap datasets, and the RBM for the 1000 Genomes dataset; also, we measured a decrease in the number of targets exposed to membership inference.</p>
</div>
<div id="S7.p4" class="ltx_para">
<p id="S7.p4.1" class="ltx_p">We also introduced a new “MIA with partial information,” which shed light on the fact that not even decreasing the adversary’s power by limiting their knowledge of the target to partial information fully mitigates the privacy loss.
Finally, our modular evaluation framework paves the way for practitioners, scientists, and researchers to easily build on our work and assess the risks of deploying synthetic genomic data in the wild, for a wide range of applications, and serve as a benchmark for techniques proposed in the future.</p>
</div>
<div id="S7.p5" class="ltx_para ltx_noindent">
<p id="S7.p5.1" class="ltx_p"><span id="S7.p5.1.1" class="ltx_text ltx_font_bold">Limitations &amp; Future Work.</span> Our evaluation focuses on existing generative methods for synthetic genomic data; thus, we have not engaged in fine-tuning the (hyper-)parameters of the models evaluated.
Moreover, one might argue that the techniques we evaluate were not designed with privacy in mind, unlike previous work on differentially private generative models for images or clinical data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib2" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">2</span></a>, <a href="#bib.bib6" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">6</span></a>, <a href="#bib.bib10" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">10</span></a>]</cite>.
That is, it is not entirely surprising that they yield small privacy gains.
However, to the best of our knowledge, <span id="S7.p5.1.2" class="ltx_text ltx_font_italic">no differentially private generative model has been proposed for genomic data</span>, which is the focus of our study.</p>
</div>
<div id="S7.p6" class="ltx_para">
<p id="S7.p6.1" class="ltx_p">In fact, prior work has shown that, for precision medicine applications, the high dimensionality of the data tends to be a major limitation, resulting in poor utility for differentially private mechanisms <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib20" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">20</span></a>, <a href="#bib.bib5" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">5</span></a>, <a href="#bib.bib34" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">34</span></a>, <a href="#bib.bib70" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">70</span></a>, <a href="#bib.bib69" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">69</span></a>]</cite>.
Differentially private techniques for GWAS are also known to yield poor accuracy as the number of features is large, relative to the number of patients in a study <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib33" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">33</span></a>]</cite>.
Nonetheless, we plan to experiment with the possible adaptation of differentially private models to genomic data and evaluate them in future work.</p>
</div>
<div id="S7.p7" class="ltx_para">
<p id="S7.p7.1" class="ltx_p">Finally, future work should search for, and experiment with, genomics use cases that have more data points, possibly relying on further collaborations with biomedical researchers.
We also intend to extend our privacy evaluation to understand how much the privacy loss stemming from releasing (synthetic) datasets affects the relatives of those included in the training set of the corresponding generative models <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib31" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">31</span></a>, <a href="#bib.bib60" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">60</span></a>]</cite>.</p>
</div>
<div id="S7.p8" class="ltx_para ltx_noindent">
<p id="S7.p8.1" class="ltx_p"><span id="S7.p8.1.1" class="ltx_text ltx_font_bold">Acknowledgments.</span> This work was supported by a Google Faculty Award on “Enabling Progress in Genomic Research via Privacy Preserving Data Sharing.”</p>
</div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography" style="font-size:90%;">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib1.2.2.1" class="ltx_text" style="font-size:90%;">[1]</span></span>
<span class="ltx_bibblock"><span id="bib.bib1.4.1" class="ltx_text" style="font-size:90%;">
1000 Genomes Project Consortium.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib1.5.1" class="ltx_text" style="font-size:90%;">A global reference for human genetic variation.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib1.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Nature</span><span id="bib.bib1.7.2" class="ltx_text" style="font-size:90%;">, 526(7571), 2015.
</span>
</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib2.2.2.1" class="ltx_text" style="font-size:90%;">[2]</span></span>
<span class="ltx_bibblock"><span id="bib.bib2.4.1" class="ltx_text" style="font-size:90%;">
G. Acs, L. Melis, C. Castelluccia, and E. De Cristofaro.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib2.5.1" class="ltx_text" style="font-size:90%;">Differentially private mixture of generative neural networks.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib2.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">IEEE Transactions on Knowledge and Data Engineering</span><span id="bib.bib2.7.2" class="ltx_text" style="font-size:90%;">,
31(6):1109–1121, 2018.
</span>
</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib3.2.2.1" class="ltx_text" style="font-size:90%;">[3]</span></span>
<span class="ltx_bibblock"><span id="bib.bib3.4.1" class="ltx_text" style="font-size:90%;">
D. R. Amancio, C. H. Comin, D. Casanova, G. Travieso, O. M. Bruno, F. A.
Rodrigues, and L. da Fontoura Costa.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib3.5.1" class="ltx_text" style="font-size:90%;">A systematic comparison of supervised classifiers.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib3.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">PloS one</span><span id="bib.bib3.7.2" class="ltx_text" style="font-size:90%;">, 9(4), 2014.
</span>
</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib4.2.2.1" class="ltx_text" style="font-size:90%;">[4]</span></span>
<span class="ltx_bibblock"><span id="bib.bib4.4.1" class="ltx_text" style="font-size:90%;">
E. Ayday, E. De Cristofaro, J.-P. Hubaux, and G. Tsudik.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib4.5.1" class="ltx_text" style="font-size:90%;">Whole genome sequencing: Revolutionary medicine or privacy
nightmare?
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib4.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Computer</span><span id="bib.bib4.7.2" class="ltx_text" style="font-size:90%;">, 48(2), 2015.
</span>
</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib5.2.2.1" class="ltx_text" style="font-size:90%;">[5]</span></span>
<span class="ltx_bibblock"><span id="bib.bib5.4.1" class="ltx_text" style="font-size:90%;">
C.-A. Azencott.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib5.5.1" class="ltx_text" style="font-size:90%;">Machine learning and genomics: precision medicine versus patient
privacy.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib5.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Philosophical Transactions of the Royal Society A: Mathematical,
Physical and Engineering Sciences</span><span id="bib.bib5.7.2" class="ltx_text" style="font-size:90%;">, 376(2128), 2018.
</span>
</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib6.2.2.1" class="ltx_text" style="font-size:90%;">[6]</span></span>
<span class="ltx_bibblock"><span id="bib.bib6.4.1" class="ltx_text" style="font-size:90%;">
B. K. Beaulieu-Jones, Z. S. Wu, C. Williams, R. Lee, S. P. Bhavnani, J. B.
Byrd, and C. S. Greene.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib6.5.1" class="ltx_text" style="font-size:90%;">Privacy-preserving generative deep neural networks support clinical
data sharing.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib6.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Circulation: Cardiovascular Quality and Outcomes</span><span id="bib.bib6.7.2" class="ltx_text" style="font-size:90%;">,
12(7):e005122, 2019.
</span>
</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib7.2.2.1" class="ltx_text" style="font-size:90%;">[7]</span></span>
<span class="ltx_bibblock"><span id="bib.bib7.4.1" class="ltx_text" style="font-size:90%;">
C. Benner, A. S. Havulinna, M.-R. Järvelin, V. Salomaa, S. Ripatti, and
M. Pirinen.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib7.5.1" class="ltx_text" style="font-size:90%;">Prospects of fine-mapping trait-associated genomic regions by using
summary statistics from genome-wide association studies.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib7.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">The American Journal of Human Genetics</span><span id="bib.bib7.7.2" class="ltx_text" style="font-size:90%;">, 101(4):539–551, 2017.
</span>
</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib8.2.2.1" class="ltx_text" style="font-size:90%;">[8]</span></span>
<span class="ltx_bibblock"><span id="bib.bib8.4.1" class="ltx_text" style="font-size:90%;">
S. Biswal, S. Ghosh, J. Duke, B. Malin, W. Stewart, and J. Sun.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib8.5.1" class="ltx_text" style="font-size:90%;">EVA: Generating Longitudinal Electronic Health Records Using
Conditional Variational Autoencoders.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib8.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">arXiv:2012.10020</span><span id="bib.bib8.7.2" class="ltx_text" style="font-size:90%;">, 2020.
</span>
</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib9.2.2.1" class="ltx_text" style="font-size:90%;">[9]</span></span>
<span class="ltx_bibblock"><span id="bib.bib9.4.1" class="ltx_text" style="font-size:90%;">
W. S. Bush and J. H. Moore.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib9.5.1" class="ltx_text" style="font-size:90%;">Genome-wide association studies.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib9.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">PLoS Computational Biology</span><span id="bib.bib9.7.2" class="ltx_text" style="font-size:90%;">, 8(12), 2012.
</span>
</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib10.2.2.1" class="ltx_text" style="font-size:90%;">[10]</span></span>
<span class="ltx_bibblock"><span id="bib.bib10.4.1" class="ltx_text" style="font-size:90%;">
D. Chen, T. Orekondy, and M. Fritz.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib10.5.1" class="ltx_text" style="font-size:90%;">Gs-wgan: A gradient-sanitized approach for learning differentially
private generators.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib10.6.1" class="ltx_text" style="font-size:90%;">arXiv preprint arXiv:2006.08265, 2020.
</span>
</span>
</li>
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib11.2.2.1" class="ltx_text" style="font-size:90%;">[11]</span></span>
<span class="ltx_bibblock"><span id="bib.bib11.4.1" class="ltx_text" style="font-size:90%;">
D. Chen, N. Yu, Y. Zhang, and M. Fritz.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib11.5.1" class="ltx_text" style="font-size:90%;">GAN-Leaks: A taxonomy of membership inference attacks against GANs.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib11.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib11.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">ACM CCS</span><span id="bib.bib11.8.3" class="ltx_text" style="font-size:90%;">, 2020.
</span>
</span>
</li>
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib12.2.2.1" class="ltx_text" style="font-size:90%;">[12]</span></span>
<span class="ltx_bibblock"><span id="bib.bib12.4.1" class="ltx_text" style="font-size:90%;">
J. Chen, W. H. Wang, and X. Shi.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib12.5.1" class="ltx_text" style="font-size:90%;">Differential Privacy Protection Against Membership Inference Attack
on Machine Learning for Genomic Data.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://www.biorxiv.org/content/early/2020/08/04/2020.08.03.235416.full.pdf" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://www.biorxiv.org/content/early/2020/08/04/2020.08.03.235416.full.pdf</a><span id="bib.bib12.6.1" class="ltx_text" style="font-size:90%;">,
2020.
</span>
</span>
</li>
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib13.2.2.1" class="ltx_text" style="font-size:90%;">[13]</span></span>
<span class="ltx_bibblock"><span id="bib.bib13.4.1" class="ltx_text" style="font-size:90%;">
E. Choi, S. Biswal, B. Malin, J. Duke, W. F. Stewart, and J. Sun.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib13.5.1" class="ltx_text" style="font-size:90%;">Generating multi-label discrete patient records using generative
adversarial networks.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib13.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib13.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">Machine learning for Healthcare Conference</span><span id="bib.bib13.8.3" class="ltx_text" style="font-size:90%;">, 2017.
</span>
</span>
</li>
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib14.2.2.1" class="ltx_text" style="font-size:90%;">[14]</span></span>
<span class="ltx_bibblock"><span id="bib.bib14.4.1" class="ltx_text" style="font-size:90%;">
G. M. Clarke and L. R. Cardon.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib14.5.1" class="ltx_text" style="font-size:90%;">Disentangling linkage disequilibrium and linkage from dense
single-nucleotide polymorphism trio data.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib14.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Genetics</span><span id="bib.bib14.7.2" class="ltx_text" style="font-size:90%;">, 171(4), 2005.
</span>
</span>
</li>
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib15.2.2.1" class="ltx_text" style="font-size:90%;">[15]</span></span>
<span class="ltx_bibblock"><span id="bib.bib15.4.1" class="ltx_text" style="font-size:90%;">
H. Elias.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib15.5.1" class="ltx_text" style="font-size:90%;">Synthetic data – All the perks without the risk?
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://techhq.com/2020/10/synthetic-data-all-the-perks-without-the-risk/" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://techhq.com/2020/10/synthetic-data-all-the-perks-without-the-risk/</a><span id="bib.bib15.6.1" class="ltx_text" style="font-size:90%;">,
2020.
</span>
</span>
</li>
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib16.2.2.1" class="ltx_text" style="font-size:90%;">[16]</span></span>
<span class="ltx_bibblock"><span id="bib.bib16.4.1" class="ltx_text" style="font-size:90%;">
EMBL-EBI Training.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib16.5.1" class="ltx_text" style="font-size:90%;">What are genome wide association studies (GWAS)?
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://bit.ly/3wP0zA8" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://bit.ly/3wP0zA8</a><span id="bib.bib16.6.1" class="ltx_text" style="font-size:90%;">, 2021.
</span>
</span>
</li>
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib17.2.2.1" class="ltx_text" style="font-size:90%;">[17]</span></span>
<span class="ltx_bibblock"><span id="bib.bib17.4.1" class="ltx_text" style="font-size:90%;">
Ensembl Variation.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib17.5.1" class="ltx_text" style="font-size:90%;">Variant classification.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://m.ensembl.org/info/genome/variation/prediction/classification.html" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://m.ensembl.org/info/genome/variation/prediction/classification.html</a><span id="bib.bib17.6.1" class="ltx_text" style="font-size:90%;">,
2020.
</span>
</span>
</li>
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib18.2.2.1" class="ltx_text" style="font-size:90%;">[18]</span></span>
<span class="ltx_bibblock"><span id="bib.bib18.4.1" class="ltx_text" style="font-size:90%;">
S. N. Evans, Y. Shvets, and M. Slatkin.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib18.5.1" class="ltx_text" style="font-size:90%;">Non-equilibrium theory of the allele frequency spectrum.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib18.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Theoretical Population Biology</span><span id="bib.bib18.7.2" class="ltx_text" style="font-size:90%;">, 71(1), 2007.
</span>
</span>
</li>
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib19.2.2.1" class="ltx_text" style="font-size:90%;">[19]</span></span>
<span class="ltx_bibblock"><span id="bib.bib19.4.1" class="ltx_text" style="font-size:90%;">
R. A. Fisher.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib19.5.1" class="ltx_text" style="font-size:90%;">The distribution of gene ratios for rare mutations.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib19.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Proceedings of the Royal Society of Edinburgh</span><span id="bib.bib19.7.2" class="ltx_text" style="font-size:90%;">, 50, 1931.
</span>
</span>
</li>
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib20.2.2.1" class="ltx_text" style="font-size:90%;">[20]</span></span>
<span class="ltx_bibblock"><span id="bib.bib20.4.1" class="ltx_text" style="font-size:90%;">
M. Fredrikson, E. Lantz, S. Jha, S. Lin, D. Page, and T. Ristenpart.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib20.5.1" class="ltx_text" style="font-size:90%;">Privacy in pharmacogenetics: An end-to-end case study of
personalized warfarin dosing.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib20.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib20.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">USENIX Security</span><span id="bib.bib20.8.3" class="ltx_text" style="font-size:90%;">, 2014.
</span>
</span>
</li>
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib21.2.2.1" class="ltx_text" style="font-size:90%;">[21]</span></span>
<span class="ltx_bibblock"><span id="bib.bib21.4.1" class="ltx_text" style="font-size:90%;">
Genetics Home Reference.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib21.5.1" class="ltx_text" style="font-size:90%;">What is Precision Medicine?
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://medlineplus.gov/genetics/understanding/precisionmedicine/definition/" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://medlineplus.gov/genetics/understanding/precisionmedicine/definition/</a><span id="bib.bib21.6.1" class="ltx_text" style="font-size:90%;">,
2020.
</span>
</span>
</li>
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib22.2.2.1" class="ltx_text" style="font-size:90%;">[22]</span></span>
<span class="ltx_bibblock"><span id="bib.bib22.4.1" class="ltx_text" style="font-size:90%;">
I. Gulrajani, F. Ahmed, M. Arjovsky, V. Dumoulin, and A. Courville.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib22.5.1" class="ltx_text" style="font-size:90%;">Improved Training of Wasserstein GANs.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib22.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib22.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">NeurIPS</span><span id="bib.bib22.8.3" class="ltx_text" style="font-size:90%;">, 2017.
</span>
</span>
</li>
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib23.2.2.1" class="ltx_text" style="font-size:90%;">[23]</span></span>
<span class="ltx_bibblock"><span id="bib.bib23.4.1" class="ltx_text" style="font-size:90%;">
M. Gymrek, A. L. McGuire, D. Golan, E. Halperin, and Y. Erlich.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib23.5.1" class="ltx_text" style="font-size:90%;">Identifying personal genomes by surname inference.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib23.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Science</span><span id="bib.bib23.7.2" class="ltx_text" style="font-size:90%;">, 339(6117), 2013.
</span>
</span>
</li>
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib24.2.2.1" class="ltx_text" style="font-size:90%;">[24]</span></span>
<span class="ltx_bibblock"><span id="bib.bib24.4.1" class="ltx_text" style="font-size:90%;">
J. Hayes, L. Melis, G. Danezis, and E. De Cristofaro.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib24.5.1" class="ltx_text" style="font-size:90%;">LOGAN: Membership inference attacks against generative models.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib24.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Proceedings on Privacy Enhancing Technologies</span><span id="bib.bib24.7.2" class="ltx_text" style="font-size:90%;">, 2019.
</span>
</span>
</li>
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib25.2.2.1" class="ltx_text" style="font-size:90%;">[25]</span></span>
<span class="ltx_bibblock"><span id="bib.bib25.4.1" class="ltx_text" style="font-size:90%;">
K. He, X. Zhang, S. Ren, and J. Sun.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib25.5.1" class="ltx_text" style="font-size:90%;">Deep Residual Learning for Image Recognition.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib25.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib25.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">IEEE CVPR</span><span id="bib.bib25.8.3" class="ltx_text" style="font-size:90%;">, 2016.
</span>
</span>
</li>
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib26.2.2.1" class="ltx_text" style="font-size:90%;">[26]</span></span>
<span class="ltx_bibblock"><span id="bib.bib26.4.1" class="ltx_text" style="font-size:90%;">
B. Hilprecht, M. Härterich, and D. Bernau.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib26.5.1" class="ltx_text" style="font-size:90%;">Monte Carlo and Reconstruction Membership Inference Attacks against
Generative Models.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib26.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">PETS</span><span id="bib.bib26.7.2" class="ltx_text" style="font-size:90%;">, 2019.
</span>
</span>
</li>
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib27.2.2.1" class="ltx_text" style="font-size:90%;">[27]</span></span>
<span class="ltx_bibblock"><span id="bib.bib27.4.1" class="ltx_text" style="font-size:90%;">
J. L. Hodges.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib27.5.1" class="ltx_text" style="font-size:90%;">The significance probability of the Smirnov two-sample test.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib27.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Arkiv för Matematik</span><span id="bib.bib27.7.2" class="ltx_text" style="font-size:90%;">, 3(5), 1958.
</span>
</span>
</li>
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib28.2.2.1" class="ltx_text" style="font-size:90%;">[28]</span></span>
<span class="ltx_bibblock"><span id="bib.bib28.4.1" class="ltx_text" style="font-size:90%;">
K. E. Holsinger and B. S. Weir.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib28.5.1" class="ltx_text" style="font-size:90%;">Genetics in geographically structured populations: defining,
estimating and interpreting FST.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib28.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Nature Reviews Genetics</span><span id="bib.bib28.7.2" class="ltx_text" style="font-size:90%;">, 10(9), 2009.
</span>
</span>
</li>
<li id="bib.bib29" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib29.2.2.1" class="ltx_text" style="font-size:90%;">[29]</span></span>
<span class="ltx_bibblock"><span id="bib.bib29.4.1" class="ltx_text" style="font-size:90%;">
N. Homer, S. Szelinger, M. Redman, D. Duggan, W. Tembe, J. Muehling, J. V.
Pearson, D. A. Stephan, S. F. Nelson, and D. W. Craig.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib29.5.1" class="ltx_text" style="font-size:90%;">Resolving individuals contributing trace amounts of DNA to highly
complex mixtures using high-density SNP genotyping microarrays.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib29.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">PLoS Genetics</span><span id="bib.bib29.7.2" class="ltx_text" style="font-size:90%;">, 4(8), 2008.
</span>
</span>
</li>
<li id="bib.bib30" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib30.2.2.1" class="ltx_text" style="font-size:90%;">[30]</span></span>
<span class="ltx_bibblock"><span id="bib.bib30.4.1" class="ltx_text" style="font-size:90%;">
B. Hui, Y. Yang, H. Yuan, P. Burlina, N. Z. Gong, and Y. Cao.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib30.5.1" class="ltx_text" style="font-size:90%;">Practical blind membership inference attack via differential
comparisons.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib30.6.1" class="ltx_text" style="font-size:90%;">arXiv preprint arXiv:2101.01341, 2021.
</span>
</span>
</li>
<li id="bib.bib31" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib31.2.2.1" class="ltx_text" style="font-size:90%;">[31]</span></span>
<span class="ltx_bibblock"><span id="bib.bib31.4.1" class="ltx_text" style="font-size:90%;">
M. Humbert, E. Ayday, J.-P. Hubaux, and A. Telenti.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib31.5.1" class="ltx_text" style="font-size:90%;">Addressing the concerns of the lacks family: quantification of kin
genomic privacy.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib31.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib31.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">ACM CCS</span><span id="bib.bib31.8.3" class="ltx_text" style="font-size:90%;">, 2013.
</span>
</span>
</li>
<li id="bib.bib32" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib32.2.2.1" class="ltx_text" style="font-size:90%;">[32]</span></span>
<span class="ltx_bibblock"><span id="bib.bib32.4.1" class="ltx_text" style="font-size:90%;">
H. K. Im, E. R. Gamazon, D. L. Nicolae, and N. J. Cox.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib32.5.1" class="ltx_text" style="font-size:90%;">On sharing quantitative trait GWAS results in an era of
multiple-omics data and the limits of genomic privacy.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib32.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">The American Journal of Human Genetics</span><span id="bib.bib32.7.2" class="ltx_text" style="font-size:90%;">, 90(4), 2012.
</span>
</span>
</li>
<li id="bib.bib33" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib33.2.2.1" class="ltx_text" style="font-size:90%;">[33]</span></span>
<span class="ltx_bibblock"><span id="bib.bib33.4.1" class="ltx_text" style="font-size:90%;">
A. Johnson and V. Shmatikov.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib33.5.1" class="ltx_text" style="font-size:90%;">Privacy-preserving data exploration in genome-wide association
studies.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib33.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib33.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">ACM KDD</span><span id="bib.bib33.8.3" class="ltx_text" style="font-size:90%;">, 2013.
</span>
</span>
</li>
<li id="bib.bib34" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib34.2.2.1" class="ltx_text" style="font-size:90%;">[34]</span></span>
<span class="ltx_bibblock"><span id="bib.bib34.4.1" class="ltx_text" style="font-size:90%;">
J. Jordon, J. Yoon, and M. Van Der Schaar.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib34.5.1" class="ltx_text" style="font-size:90%;">PATE-GAN: Generating synthetic data with differential privacy
guarantees.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib34.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib34.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">ICLR</span><span id="bib.bib34.8.3" class="ltx_text" style="font-size:90%;">, 2018.
</span>
</span>
</li>
<li id="bib.bib35" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib35.2.2.1" class="ltx_text" style="font-size:90%;">[35]</span></span>
<span class="ltx_bibblock"><span id="bib.bib35.4.1" class="ltx_text" style="font-size:90%;">
N. Killoran.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib35.5.1" class="ltx_text" style="font-size:90%;">Generating and designing DNA.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://github.com/co9olguy/Generating-and-designing-DNA" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://github.com/co9olguy/Generating-and-designing-DNA</a><span id="bib.bib35.6.1" class="ltx_text" style="font-size:90%;">, 2021.
</span>
</span>
</li>
<li id="bib.bib36" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib36.2.2.1" class="ltx_text" style="font-size:90%;">[36]</span></span>
<span class="ltx_bibblock"><span id="bib.bib36.4.1" class="ltx_text" style="font-size:90%;">
N. Killoran, L. Lee, A. Delong, D. Duvenaud, and B. Frey.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib36.5.1" class="ltx_text" style="font-size:90%;">Generating and designing DNA with deep generative models.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib36.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib36.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">NIPS Computational Biology Workshop</span><span id="bib.bib36.8.3" class="ltx_text" style="font-size:90%;">, 2017.
</span>
</span>
</li>
<li id="bib.bib37" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib37.2.2.1" class="ltx_text" style="font-size:90%;">[37]</span></span>
<span class="ltx_bibblock"><span id="bib.bib37.4.1" class="ltx_text" style="font-size:90%;">
Y. LeCun, B. Boser, J. S. Denker, D. Henderson, R. E. Howard, W. Hubbard, and
L. D. Jackel.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib37.5.1" class="ltx_text" style="font-size:90%;">Backpropagation applied to handwritten zip code recognition.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib37.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Neural Comp.</span><span id="bib.bib37.7.2" class="ltx_text" style="font-size:90%;">, 1(4), 1989.
</span>
</span>
</li>
<li id="bib.bib38" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib38.2.2.1" class="ltx_text" style="font-size:90%;">[38]</span></span>
<span class="ltx_bibblock"><span id="bib.bib38.4.1" class="ltx_text" style="font-size:90%;">
K. Leino and M. Fredrikson.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib38.5.1" class="ltx_text" style="font-size:90%;">Stolen Memories: Leveraging Model Memorization for Calibrated
White-Box Membership Inference.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib38.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib38.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">USENIX Security Symposium</span><span id="bib.bib38.8.3" class="ltx_text" style="font-size:90%;">. USENIX, 2020.
</span>
</span>
</li>
<li id="bib.bib39" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib39.2.2.1" class="ltx_text" style="font-size:90%;">[39]</span></span>
<span class="ltx_bibblock"><span id="bib.bib39.4.1" class="ltx_text" style="font-size:90%;">
I. Mironov.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib39.5.1" class="ltx_text" style="font-size:90%;">Rényi differential privacy.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib39.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib39.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">CSF</span><span id="bib.bib39.8.3" class="ltx_text" style="font-size:90%;">, 2017.
</span>
</span>
</li>
<li id="bib.bib40" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib40.2.2.1" class="ltx_text" style="font-size:90%;">[40]</span></span>
<span class="ltx_bibblock"><span id="bib.bib40.4.1" class="ltx_text" style="font-size:90%;">
A. Mordvintsev, C. Olah, and M. Tyka.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib40.5.1" class="ltx_text" style="font-size:90%;">Inceptionism: Going Deeper into Neural Networks.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://ai.googleblog.com/2015/06/inceptionism-going-deeper-into-neural.html" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://ai.googleblog.com/2015/06/inceptionism-going-deeper-into-neural.html</a><span id="bib.bib40.6.1" class="ltx_text" style="font-size:90%;">,
2015.
</span>
</span>
</li>
<li id="bib.bib41" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib41.2.2.1" class="ltx_text" style="font-size:90%;">[41]</span></span>
<span class="ltx_bibblock"><span id="bib.bib41.4.1" class="ltx_text" style="font-size:90%;">
M. Nasr, R. Shokri, and A. Houmansadr.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib41.5.1" class="ltx_text" style="font-size:90%;">Machine Learning with Membership Privacy using Adversarial
Regularization.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib41.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib41.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">ACM CCS</span><span id="bib.bib41.8.3" class="ltx_text" style="font-size:90%;">. ACM, 2018.
</span>
</span>
</li>
<li id="bib.bib42" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib42.2.2.1" class="ltx_text" style="font-size:90%;">[42]</span></span>
<span class="ltx_bibblock"><span id="bib.bib42.4.1" class="ltx_text" style="font-size:90%;">
National Human Genome Research Institute.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib42.5.1" class="ltx_text" style="font-size:90%;">International HapMap Project.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://www.genome.gov/10001688/international-hapmap-project" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://www.genome.gov/10001688/international-hapmap-project</a><span id="bib.bib42.6.1" class="ltx_text" style="font-size:90%;">,
2012.
</span>
</span>
</li>
<li id="bib.bib43" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib43.2.2.1" class="ltx_text" style="font-size:90%;">[43]</span></span>
<span class="ltx_bibblock"><span id="bib.bib43.4.1" class="ltx_text" style="font-size:90%;">
National Human Genome Research Institute.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib43.5.1" class="ltx_text" style="font-size:90%;">Genome-Wide Association Studies Fact Sheet.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://www.genome.gov/about-genomics/fact-sheets/Genome-Wide-Association-Studies-Fact-Sheet" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://www.genome.gov/about-genomics/fact-sheets/Genome-Wide-Association-Studies-Fact-Sheet</a><span id="bib.bib43.6.1" class="ltx_text" style="font-size:90%;">,
2021.
</span>
</span>
</li>
<li id="bib.bib44" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib44.2.2.1" class="ltx_text" style="font-size:90%;">[44]</span></span>
<span class="ltx_bibblock"><span id="bib.bib44.4.1" class="ltx_text" style="font-size:90%;">
M. Naveed, E. Ayday, E. W. Clayton, J. Fellay, C. A. Gunter, J.-P. Hubaux,
B. A. Malin, and X. Wang.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib44.5.1" class="ltx_text" style="font-size:90%;">Privacy In The Genomic Era.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib44.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">ACM Computing Surveys</span><span id="bib.bib44.7.2" class="ltx_text" style="font-size:90%;">, 48(1), 2015.
</span>
</span>
</li>
<li id="bib.bib45" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib45.2.2.1" class="ltx_text" style="font-size:90%;">[45]</span></span>
<span class="ltx_bibblock"><span id="bib.bib45.4.1" class="ltx_text" style="font-size:90%;">
NHS England.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib45.5.1" class="ltx_text" style="font-size:90%;">A&amp;E Synthetic Data.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://data.england.nhs.uk/dataset/a-e-synthetic-data" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://data.england.nhs.uk/dataset/a-e-synthetic-data</a><span id="bib.bib45.6.1" class="ltx_text" style="font-size:90%;">, 2021.
</span>
</span>
</li>
<li id="bib.bib46" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib46.2.2.1" class="ltx_text" style="font-size:90%;">[46]</span></span>
<span class="ltx_bibblock"><span id="bib.bib46.4.1" class="ltx_text" style="font-size:90%;">
NIH.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib46.5.1" class="ltx_text" style="font-size:90%;">Genomic Data Sharing.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://osp.od.nih.gov/scientific-sharing/genomic-data-sharing-faqs/" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://osp.od.nih.gov/scientific-sharing/genomic-data-sharing-faqs/</a><span id="bib.bib46.6.1" class="ltx_text" style="font-size:90%;">,
2021.
</span>
</span>
</li>
<li id="bib.bib47" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib47.2.2.1" class="ltx_text" style="font-size:90%;">[47]</span></span>
<span class="ltx_bibblock"><span id="bib.bib47.4.1" class="ltx_text" style="font-size:90%;">
J. Pickrell.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib47.5.1" class="ltx_text" style="font-size:90%;">Genetic maps for the 1000 Genomes Project variants.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://github.com/joepickrell/1000-genomes-genetic-maps" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://github.com/joepickrell/1000-genomes-genetic-maps</a><span id="bib.bib47.6.1" class="ltx_text" style="font-size:90%;">, 2021.
</span>
</span>
</li>
<li id="bib.bib48" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib48.2.2.1" class="ltx_text" style="font-size:90%;">[48]</span></span>
<span class="ltx_bibblock"><span id="bib.bib48.4.1" class="ltx_text" style="font-size:90%;">
F. Privé, Z. Zhu, and B. J. Vilhjalmsson.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib48.5.1" class="ltx_text" style="font-size:90%;">Finding hidden treasures in summary statistics from genome-wide
association studies.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib48.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Nature Genetics</span><span id="bib.bib48.7.2" class="ltx_text" style="font-size:90%;">, 53(4):431–432, 2021.
</span>
</span>
</li>
<li id="bib.bib49" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib49.2.2.1" class="ltx_text" style="font-size:90%;">[49]</span></span>
<span class="ltx_bibblock"><span id="bib.bib49.4.1" class="ltx_text" style="font-size:90%;">
J. L. Raisaro, F. Tramer, Z. Ji, D. Bu, Y. Zhao, K. Carey, D. Lloyd, H. Sofia,
D. Baker, P. Flicek, et al.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib49.5.1" class="ltx_text" style="font-size:90%;">Addressing Beacon re-identification attacks: quantification and
mitigation of privacy risks.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib49.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Journal of the American Medical Informatics Association</span><span id="bib.bib49.7.2" class="ltx_text" style="font-size:90%;">, 24(4),
2017.
</span>
</span>
</li>
<li id="bib.bib50" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib50.2.2.1" class="ltx_text" style="font-size:90%;">[50]</span></span>
<span class="ltx_bibblock"><span id="bib.bib50.4.1" class="ltx_text" style="font-size:90%;">
A. R. Rogers and C. Huff.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib50.5.1" class="ltx_text" style="font-size:90%;">Linkage disequilibrium between loci with unknown phase.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib50.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Genetics</span><span id="bib.bib50.7.2" class="ltx_text" style="font-size:90%;">, 182, 2009.
</span>
</span>
</li>
<li id="bib.bib51" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib51.2.2.1" class="ltx_text" style="font-size:90%;">[51]</span></span>
<span class="ltx_bibblock"><span id="bib.bib51.4.1" class="ltx_text" style="font-size:90%;">
A. Sablayrolles, M. Douze, C. Schmid, Y. Ollivier, and H. Jégou.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib51.5.1" class="ltx_text" style="font-size:90%;">White-box vs Black-box: Bayes Optimal Strategies for Membership
Inference.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib51.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib51.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">ICML</span><span id="bib.bib51.8.3" class="ltx_text" style="font-size:90%;">, 2019.
</span>
</span>
</li>
<li id="bib.bib52" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib52.2.2.1" class="ltx_text" style="font-size:90%;">[52]</span></span>
<span class="ltx_bibblock"><span id="bib.bib52.4.1" class="ltx_text" style="font-size:90%;">
P. Sadrach.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib52.5.1" class="ltx_text" style="font-size:90%;">How Artificial Intelligence Can Revolutionize Healthcare.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://builtin.com/healthcare-technology/how-ai-revolutionize-healthcare" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://builtin.com/healthcare-technology/how-ai-revolutionize-healthcare</a><span id="bib.bib52.6.1" class="ltx_text" style="font-size:90%;">,
2020.
</span>
</span>
</li>
<li id="bib.bib53" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib53.2.2.1" class="ltx_text" style="font-size:90%;">[53]</span></span>
<span class="ltx_bibblock"><span id="bib.bib53.4.1" class="ltx_text" style="font-size:90%;">
A. Salem, Y. Zhang, M. Humbert, P. Berrang, M. Fritz, and M. Backes.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib53.5.1" class="ltx_text" style="font-size:90%;">ML-Leaks: Model and Data Independent Membership Inference Attacks
and Defenses on Machine Learning Models.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib53.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib53.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">NDSS</span><span id="bib.bib53.8.3" class="ltx_text" style="font-size:90%;">, 2019.
</span>
</span>
</li>
<li id="bib.bib54" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib54.2.2.1" class="ltx_text" style="font-size:90%;">[54]</span></span>
<span class="ltx_bibblock"><span id="bib.bib54.4.1" class="ltx_text" style="font-size:90%;">
S. S. Samani, Z. Huang, E. Ayday, M. Elliot, J. Fellay, J.-P. Hubaux, and
Z. Kutalik.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib54.5.1" class="ltx_text" style="font-size:90%;">Quantifying genomic privacy via inference attack with high-order SNV
correlations.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib54.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib54.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">IEEE Security and Privacy Workshops</span><span id="bib.bib54.8.3" class="ltx_text" style="font-size:90%;">, 2015.
</span>
</span>
</li>
<li id="bib.bib55" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib55.2.2.1" class="ltx_text" style="font-size:90%;">[55]</span></span>
<span class="ltx_bibblock"><span id="bib.bib55.4.1" class="ltx_text" style="font-size:90%;">
R. Shokri, M. Stronati, C. Song, and V. Shmatikov.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib55.5.1" class="ltx_text" style="font-size:90%;">Membership inference attacks against machine learning models.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib55.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib55.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">IEEE S&amp;P</span><span id="bib.bib55.8.3" class="ltx_text" style="font-size:90%;">, 2017.
</span>
</span>
</li>
<li id="bib.bib56" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib56.2.2.1" class="ltx_text" style="font-size:90%;">[56]</span></span>
<span class="ltx_bibblock"><span id="bib.bib56.4.1" class="ltx_text" style="font-size:90%;">
S. S. Shringarpure and C. D. Bustamante.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib56.5.1" class="ltx_text" style="font-size:90%;">Privacy risks from genomic data-sharing beacons.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib56.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">The American Journal of Human Genetics</span><span id="bib.bib56.7.2" class="ltx_text" style="font-size:90%;">, 97(5), 2015.
</span>
</span>
</li>
<li id="bib.bib57" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib57.2.2.1" class="ltx_text" style="font-size:90%;">[57]</span></span>
<span class="ltx_bibblock"><span id="bib.bib57.4.1" class="ltx_text" style="font-size:90%;">
K. Simonyan, A. Vedaldi, and A. Zisserman.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib57.5.1" class="ltx_text" style="font-size:90%;">Deep Inside Convolutional Networks: Visualising Image Classification
Models and Saliency Maps.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib57.6.1" class="ltx_text" style="font-size:90%;">arXiv:1312.6034, 2013.
</span>
</span>
</li>
<li id="bib.bib58" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib58.2.2.1" class="ltx_text" style="font-size:90%;">[58]</span></span>
<span class="ltx_bibblock"><span id="bib.bib58.4.1" class="ltx_text" style="font-size:90%;">
P. Smolensky.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib58.5.1" class="ltx_text" style="font-size:90%;">Information processing in dynamical systems: Foundations of harmony
theory.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib58.6.1" class="ltx_text" style="font-size:90%;">Technical report, Colorado Univ at Boulder Dept of Computer Science,
1986.
</span>
</span>
</li>
<li id="bib.bib59" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib59.2.2.1" class="ltx_text" style="font-size:90%;">[59]</span></span>
<span class="ltx_bibblock"><span id="bib.bib59.4.1" class="ltx_text" style="font-size:90%;">
T. Stadler, B. Oprisanu, and C. Troncoso.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib59.5.1" class="ltx_text" style="font-size:90%;">Synthetic Data – A Privacy Mirage.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib59.6.1" class="ltx_text" style="font-size:90%;">arXiv:2011.07018, 2020.
</span>
</span>
</li>
<li id="bib.bib60" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib60.2.2.1" class="ltx_text" style="font-size:90%;">[60]</span></span>
<span class="ltx_bibblock"><span id="bib.bib60.4.1" class="ltx_text" style="font-size:90%;">
A. Telenti, E. Ayday, and J. P. Hubaux.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib60.5.1" class="ltx_text" style="font-size:90%;">On genomics, kin, and privacy.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib60.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">F1000Research</span><span id="bib.bib60.7.2" class="ltx_text" style="font-size:90%;">, 3, 2014.
</span>
</span>
</li>
<li id="bib.bib61" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib61.2.2.1" class="ltx_text" style="font-size:90%;">[61]</span></span>
<span class="ltx_bibblock"><span id="bib.bib61.4.1" class="ltx_text" style="font-size:90%;">
A. Torfi and E. A. Fox.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib61.5.1" class="ltx_text" style="font-size:90%;">CorGAN: Correlation-capturing convolutional generative adversarial
networks for generating synthetic healthcare records.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib61.6.1" class="ltx_text" style="font-size:90%;">arXiv:2001.09346, 2020.
</span>
</span>
</li>
<li id="bib.bib62" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib62.2.2.1" class="ltx_text" style="font-size:90%;">[62]</span></span>
<span class="ltx_bibblock"><span id="bib.bib62.4.1" class="ltx_text" style="font-size:90%;">
A. Torfi, E. A. Fox, and C. K. Reddy.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib62.5.1" class="ltx_text" style="font-size:90%;">Differentially Private Synthetic Medical Data Generation using
Convolutional GANs.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib62.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">arXiv:2012.11774</span><span id="bib.bib62.7.2" class="ltx_text" style="font-size:90%;">, 2020.
</span>
</span>
</li>
<li id="bib.bib63" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib63.2.2.1" class="ltx_text" style="font-size:90%;">[63]</span></span>
<span class="ltx_bibblock"><span id="bib.bib63.4.1" class="ltx_text" style="font-size:90%;">
A. Tucker, Z. Wang, Y. Rotalinti, and P. Myles.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib63.5.1" class="ltx_text" style="font-size:90%;">Generating high-fidelity synthetic patient data for assessing
machine learning healthcare software.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib63.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">NPJ Digital Medicine</span><span id="bib.bib63.7.2" class="ltx_text" style="font-size:90%;">, 3(1), 2020.
</span>
</span>
</li>
<li id="bib.bib64" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib64.2.2.1" class="ltx_text" style="font-size:90%;">[64]</span></span>
<span class="ltx_bibblock"><span id="bib.bib64.4.1" class="ltx_text" style="font-size:90%;">
UK Government.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib64.5.1" class="ltx_text" style="font-size:90%;">Press Release – New synthetic datasets to assist COVID-19 and
cardiovascular research.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://bit.ly/3eFCFB4" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://bit.ly/3eFCFB4</a><span id="bib.bib64.6.1" class="ltx_text" style="font-size:90%;">, 2020.
</span>
</span>
</li>
<li id="bib.bib65" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib65.2.2.1" class="ltx_text" style="font-size:90%;">[65]</span></span>
<span class="ltx_bibblock"><span id="bib.bib65.4.1" class="ltx_text" style="font-size:90%;">
N. Von Thenen, E. Ayday, and A. E. Cicek.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib65.5.1" class="ltx_text" style="font-size:90%;">Re-identification of individuals in genomic data-sharing beacons via
allele inference.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib65.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Bioinformatics</span><span id="bib.bib65.7.2" class="ltx_text" style="font-size:90%;">, 35(3), 2019.
</span>
</span>
</li>
<li id="bib.bib66" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib66.2.2.1" class="ltx_text" style="font-size:90%;">[66]</span></span>
<span class="ltx_bibblock"><span id="bib.bib66.4.1" class="ltx_text" style="font-size:90%;">
R. Wang, Y. F. Li, X. Wang, H. Tang, and X. Zhou.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib66.5.1" class="ltx_text" style="font-size:90%;">Learning your identity and disease from research papers: information
leaks in genome wide association study.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib66.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib66.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">ACM CCS</span><span id="bib.bib66.8.3" class="ltx_text" style="font-size:90%;">, 2009.
</span>
</span>
</li>
<li id="bib.bib67" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib67.2.2.1" class="ltx_text" style="font-size:90%;">[67]</span></span>
<span class="ltx_bibblock"><span id="bib.bib67.4.1" class="ltx_text" style="font-size:90%;">
S. Wang, X. Jiang, S. Singh, R. Marmor, L. Bonomi, D. Fox, M. Dow, and
L. Ohno-Machado.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib67.5.1" class="ltx_text" style="font-size:90%;">Genome Privacy: Challenges, Technical Approaches to Mitigate Risk,
and Ethical Considerations in the United States.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib67.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">Annals of the New York Academy of Sciences</span><span id="bib.bib67.7.2" class="ltx_text" style="font-size:90%;">, 1387(1), 2017.
</span>
</span>
</li>
<li id="bib.bib68" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib68.2.2.1" class="ltx_text" style="font-size:90%;">[68]</span></span>
<span class="ltx_bibblock"><span id="bib.bib68.4.1" class="ltx_text" style="font-size:90%;">
Z. Wang, P. Myles, and A. Tucker.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib68.5.1" class="ltx_text" style="font-size:90%;">Generating and Evaluating Synthetic UK Primary Care Data: Preserving
Data Utility &amp; Patient Privacy.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib68.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib68.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">IEEE International Symposium on Computer-Based Medical
Systems</span><span id="bib.bib68.8.3" class="ltx_text" style="font-size:90%;">, 2019.
</span>
</span>
</li>
<li id="bib.bib69" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib69.2.2.1" class="ltx_text" style="font-size:90%;">[69]</span></span>
<span class="ltx_bibblock"><span id="bib.bib69.4.1" class="ltx_text" style="font-size:90%;">
L. Xie, K. Lin, S. Wang, F. Wang, and J. Zhou.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib69.5.1" class="ltx_text" style="font-size:90%;">Differentially private generative adversarial network.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib69.6.1" class="ltx_text" style="font-size:90%;">arXiv preprint arXiv:1802.06739, 2018.
</span>
</span>
</li>
<li id="bib.bib70" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib70.2.2.1" class="ltx_text" style="font-size:90%;">[70]</span></span>
<span class="ltx_bibblock"><span id="bib.bib70.4.1" class="ltx_text" style="font-size:90%;">
A. Yale, S. Dash, R. Dutta, I. Guyon, A. Pavao, and K. Bennett.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib70.5.1" class="ltx_text" style="font-size:90%;">Privacy preserving synthetic health data.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib70.6.1" class="ltx_text" style="font-size:90%;">2019.
</span>
</span>
</li>
<li id="bib.bib71" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib71.2.2.1" class="ltx_text" style="font-size:90%;">[71]</span></span>
<span class="ltx_bibblock"><span id="bib.bib71.4.1" class="ltx_text" style="font-size:90%;">
B. Yelmen, A. Decelle, L. Ongaro, D. Marnetto, F. Montinaro, C. Furtlehner,
L. Pagani, and F. Jay.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib71.5.1" class="ltx_text" style="font-size:90%;">Creating Artificial Human Genomes Using Generative Models.
</span>
</span>
<span class="ltx_bibblock"><a target="_blank" href="https://www.biorxiv.org/content/10.1101/769091v2" title="" class="ltx_ref ltx_url" style="font-size:90%;">https://www.biorxiv.org/content/10.1101/769091v2</a><span id="bib.bib71.6.1" class="ltx_text" style="font-size:90%;">, 2019.
</span>
</span>
</li>
<li id="bib.bib72" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib72.2.2.1" class="ltx_text" style="font-size:90%;">[72]</span></span>
<span class="ltx_bibblock"><span id="bib.bib72.4.1" class="ltx_text" style="font-size:90%;">
J. Yoon, L. Drumright, and M. Schaar.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib72.5.1" class="ltx_text" style="font-size:90%;">Anonymization Through Data Synthesis Using Generative Adversarial
Networks (ADS-GAN).
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib72.6.1" class="ltx_text ltx_font_italic" style="font-size:90%;">IEEE Journal of Biomedical and Health Informatics</span><span id="bib.bib72.7.2" class="ltx_text" style="font-size:90%;">, PP, 2020.
</span>
</span>
</li>
<li id="bib.bib73" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib73.2.2.1" class="ltx_text" style="font-size:90%;">[73]</span></span>
<span class="ltx_bibblock"><span id="bib.bib73.4.1" class="ltx_text" style="font-size:90%;">
J. Yosinski, J. Clune, A. Nguyen, T. Fuchs, and H. Lipson.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib73.5.1" class="ltx_text" style="font-size:90%;">Understanding Neural Networks Through Deep Visualization.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib73.6.1" class="ltx_text" style="font-size:90%;">arXiv:1506.06579, 2015.
</span>
</span>
</li>
<li id="bib.bib74" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem"><span id="bib.bib74.2.2.1" class="ltx_text" style="font-size:90%;">[74]</span></span>
<span class="ltx_bibblock"><span id="bib.bib74.4.1" class="ltx_text" style="font-size:90%;">
X. Zhou, B. Peng, Y. F. Li, Y. Chen, H. Tang, and X. Wang.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib74.5.1" class="ltx_text" style="font-size:90%;">To release or not to release: evaluating information leaks in
aggregate human-genome data.
</span>
</span>
<span class="ltx_bibblock"><span id="bib.bib74.6.1" class="ltx_text" style="font-size:90%;">In </span><span id="bib.bib74.7.2" class="ltx_text ltx_font_italic" style="font-size:90%;">ESORICS</span><span id="bib.bib74.8.3" class="ltx_text" style="font-size:90%;">, 2011.
</span>
</span>
</li>
</ul>
</section>
<figure id="S7.F11" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S7.F11.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x15.png" id="S7.F11.sf1.g1" class="ltx_graphics ltx_img_landscape" width="346" height="96" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>MAF for CEU</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S7.F11.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x16.png" id="S7.F11.sf2.g1" class="ltx_graphics ltx_img_landscape" width="415" height="92" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>WGAN discriminator losses for all datasets</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 11: </span>WGANs experiments for 100/1,000/10,000 iterations.</figcaption>
</figure>
<figure id="S7.F12" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S7.F12.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x17.png" id="S7.F12.sf1.g1" class="ltx_graphics ltx_img_landscape" width="456" height="75" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S7.F12.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x18.png" id="S7.F12.sf2.g1" class="ltx_graphics ltx_img_landscape" width="456" height="75" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S7.F12.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x19.png" id="S7.F12.sf3.g1" class="ltx_graphics ltx_img_landscape" width="456" height="75" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(c) </span>1000 Genomes</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 12: </span>Alternate allele correlation for the CEU population, the CHB population, and the 1000 Genomes dataset.</figcaption>
</figure>
<figure id="S7.F13" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_3">
<figure id="S7.F13.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x20.png" id="S7.F13.sf1.g1" class="ltx_graphics ltx_img_landscape" width="147" height="97" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU</figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_3">
<figure id="S7.F13.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x21.png" id="S7.F13.sf2.g1" class="ltx_graphics ltx_img_landscape" width="147" height="97" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB</figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_3">
<figure id="S7.F13.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x22.png" id="S7.F13.sf3.g1" class="ltx_graphics ltx_img_landscape" width="150" height="97" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(c) </span>1000 Genomes</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 13: </span>Frequency spectrum analysis for the CEU population, the CHB population, and the 1000 Genomes dataset.</figcaption>
</figure>
<figure id="S7.F14" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S7.F14.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x23.png" id="S7.F14.sf1.g1" class="ltx_graphics ltx_img_landscape" width="456" height="68" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S7.F14.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x24.png" id="S7.F14.sf2.g1" class="ltx_graphics ltx_img_landscape" width="456" height="68" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S7.F14.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x25.png" id="S7.F14.sf3.g1" class="ltx_graphics ltx_img_landscape" width="456" height="68" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(c) </span>1000 Genomes</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 14: </span>Pairwise Euclidean Genetic Distance (EGD) between individuals.</figcaption>
</figure>
<figure id="S7.F15" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S7.F15.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x26.png" id="S7.F15.sf1.g1" class="ltx_graphics ltx_img_landscape" width="230" height="51" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU Population</figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S7.F15.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x27.png" id="S7.F15.sf2.g1" class="ltx_graphics ltx_img_landscape" width="230" height="51" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB Population</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S7.F15.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x28.png" id="S7.F15.sf3.g1" class="ltx_graphics ltx_img_landscape" width="230" height="51" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(c) </span>1000 Genomes</figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S7.F15.sf4" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x29.png" id="S7.F15.sf4.g1" class="ltx_graphics ltx_img_landscape" width="230" height="51" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(d) </span>1000 Genomes, excluding the GAN and Rec-GAN models</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 15: </span>2D Principal Component Analysis (PCA) visualization of the real and synthetic sequences.</figcaption>
</figure>
<section id="A1" class="ltx_appendix">
<h2 class="ltx_title ltx_title_appendix">
<span class="ltx_tag ltx_tag_appendix">Appendix A </span>WGAN settings</h2>

<div id="A1.p1" class="ltx_para">
<p id="A1.p1.1" class="ltx_p">We experimented with training the WGAN models for various iterations (up to 10,000) and observed that the utility metrics stabilized after around the 100th iteration.
Specifically, it can be seen that for MAF in the CEU dataset in Fig. <a href="#S7.F11.sf1" title="In Figure 11 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">11a</span></a> training the WGAN model for more than 100 iterations does not result in better metrics.
The discriminator training and validation losses converge to the same values around 100th iteration for all datasets as can be seen in Fig. <a href="#S7.F11.sf2" title="In Figure 11 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">11b</span></a>.
Therefore, in our evaluation we train the WGAN model for 100 iterations, as opposed to the 100,000 proposed in the original implementations.
Overall, we believe this could be due to the characteristics of the datasets as, for example, the sequences in our evaluation all have a length of 1,000.
Killoran et al. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib36" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">36</span></a>]</cite> experiment with DNA sequences of length around 50.</p>
</div>
</section>
<section id="A2" class="ltx_appendix">
<h2 class="ltx_title ltx_title_appendix">
<span class="ltx_tag ltx_tag_appendix">Appendix B </span>Additional Figures from Experiments</h2>

<div id="A2.p1" class="ltx_para">
<p id="A2.p1.1" class="ltx_p">In this section, we include some additional experiments complementing our utility evaluation presented in Section <a href="#S4" title="4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.
More precisely, Figure <a href="#S7.F12" title="Figure 12 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">12</span></a> presents the alternate allele correlation (AAC) between the real and the synthetic datasets, while in Figure <a href="#S7.F13" title="Figure 13 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">13</span></a>, we plot the scaled folded Site Frequency Spectrum (SFS), which is the distribution of counts of minor alleles in a sample calculated over all segregating sites.
Finally, Figure <a href="#S7.F14" title="Figure 14 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">14</span></a> shows the Euclidean Genetic Distance (EGD) between the samples in each dataset.</p>
</div>
</section>
<section id="A3" class="ltx_appendix">
<h2 class="ltx_title ltx_title_appendix">
<span class="ltx_tag ltx_tag_appendix">Appendix C </span>Principal Component Analysis (PCA)</h2>

<figure id="A3.F16" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2102.03314/assets/graphs/plots/legend.png" id="A3.F16.g1" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="113" height="14" alt="Refer to caption"></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="A3.F16.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x30.png" id="A3.F16.sf1.g1" class="ltx_graphics ltx_img_landscape" width="184" height="138" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>CEU Population</figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="A3.F16.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2102.03314/assets/x31.png" id="A3.F16.sf2.g1" class="ltx_graphics ltx_img_landscape" width="184" height="138" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>CHB Population</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 16: </span>Fixation index values vs Privacy gain for the HapMap populations.</figcaption>
</figure>
<div id="A3.p1" class="ltx_para">
<p id="A3.p1.1" class="ltx_p">Here, we study the difference between synthetic and real data vis-à-vis a principal component analysis on the corresponding samples.
We extract the first two principal components and project the real and synthetic datasets on these two components to show how the synthetic samples are distributed compared to the real data.</p>
</div>
<div id="A3.p2" class="ltx_para">
<p id="A3.p2.1" class="ltx_p">Fig. <a href="#S7.F15" title="Figure 15 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">15</span></a> presents this 2D visualization.
Recomb has a close distribution to the real data for both HapMap populations, which, according to <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref"><span class="ltx_text" style="font-size:90%;">54</span></a>]</cite>, is because the genetic recombination model considers all the correlations between SNPs and builds a higher-order model. For the 1000 Genomes, as for the other utility metrics studied in Section <a href="#S4" title="4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>, the GAN and Rec-GAN models perform quite poorly, generating samples with a different distribution than the real samples.
Therefore, in Figure <a href="#S7.F15.sf4" title="In Figure 15 ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">15d</span></a>, we exclude them in order to take a closer look at the non-GAN-based models.
Here, we can better observe that the RBM-generated samples have the closest distribution to the real data.
In contrast to the HapMap populations, the samples from Recomb are all centered around 0 and fail to simulate the distribution given by the real data, and similar results are in the case of samples generated by Rec-RBM.</p>
</div>
</section>
<section id="A4" class="ltx_appendix">
<h2 class="ltx_title ltx_title_appendix">
<span class="ltx_tag ltx_tag_appendix">Appendix D </span>Privacy Gain vs. <math id="A4.1.m1.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="A4.1.m1.1b"><msub id="A4.1.m1.1.1" xref="A4.1.m1.1.1.cmml"><mi id="A4.1.m1.1.1.2" xref="A4.1.m1.1.1.2.cmml">F</mi><mrow id="A4.1.m1.1.1.3" xref="A4.1.m1.1.1.3.cmml"><mi id="A4.1.m1.1.1.3.2" xref="A4.1.m1.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="A4.1.m1.1.1.3.1" xref="A4.1.m1.1.1.3.1.cmml">​</mo><mi id="A4.1.m1.1.1.3.3" xref="A4.1.m1.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="A4.1.m1.1c"><apply id="A4.1.m1.1.1.cmml" xref="A4.1.m1.1.1"><csymbol cd="ambiguous" id="A4.1.m1.1.1.1.cmml" xref="A4.1.m1.1.1">subscript</csymbol><ci id="A4.1.m1.1.1.2.cmml" xref="A4.1.m1.1.1.2">𝐹</ci><apply id="A4.1.m1.1.1.3.cmml" xref="A4.1.m1.1.1.3"><times id="A4.1.m1.1.1.3.1.cmml" xref="A4.1.m1.1.1.3.1"></times><ci id="A4.1.m1.1.1.3.2.cmml" xref="A4.1.m1.1.1.3.2">𝑆</ci><ci id="A4.1.m1.1.1.3.3.cmml" xref="A4.1.m1.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="A4.1.m1.1d">F_{ST}</annotation></semantics></math>
</h2>

<div id="A4.p1" class="ltx_para">
<p id="A4.p1.3" class="ltx_p">To provide a quick visualization of the <span id="A4.p1.3.1" class="ltx_text ltx_font_italic">trade-offs</span> between privacy and utility, we also plot the <math id="A4.p1.1.m1.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="A4.p1.1.m1.1a"><msub id="A4.p1.1.m1.1.1" xref="A4.p1.1.m1.1.1.cmml"><mi id="A4.p1.1.m1.1.1.2" xref="A4.p1.1.m1.1.1.2.cmml">F</mi><mrow id="A4.p1.1.m1.1.1.3" xref="A4.p1.1.m1.1.1.3.cmml"><mi id="A4.p1.1.m1.1.1.3.2" xref="A4.p1.1.m1.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="A4.p1.1.m1.1.1.3.1" xref="A4.p1.1.m1.1.1.3.1.cmml">​</mo><mi id="A4.p1.1.m1.1.1.3.3" xref="A4.p1.1.m1.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="A4.p1.1.m1.1b"><apply id="A4.p1.1.m1.1.1.cmml" xref="A4.p1.1.m1.1.1"><csymbol cd="ambiguous" id="A4.p1.1.m1.1.1.1.cmml" xref="A4.p1.1.m1.1.1">subscript</csymbol><ci id="A4.p1.1.m1.1.1.2.cmml" xref="A4.p1.1.m1.1.1.2">𝐹</ci><apply id="A4.p1.1.m1.1.1.3.cmml" xref="A4.p1.1.m1.1.1.3"><times id="A4.p1.1.m1.1.1.3.1.cmml" xref="A4.p1.1.m1.1.1.3.1"></times><ci id="A4.p1.1.m1.1.1.3.2.cmml" xref="A4.p1.1.m1.1.1.3.2">𝑆</ci><ci id="A4.p1.1.m1.1.1.3.3.cmml" xref="A4.p1.1.m1.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="A4.p1.1.m1.1c">F_{ST}</annotation></semantics></math> (see Section <a href="#S4.SS2" title="4.2 Population Statistics ‣ 4 Utility Evaluation ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">4.2</span></a>) against the PG in Figure <a href="#A3.F16" title="Figure 16 ‣ Appendix C Principal Component Analysis (PCA) ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">16</span></a> for the two smaller HapMap Populations.
For this set of experiments, we randomly sample 10 targets from each dataset.
We train a Logistic Regression attack classifier using 5 shadow models, using 100 synthetic datasets for each of them.
We then compute the PG and <math id="A4.p1.2.m2.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="A4.p1.2.m2.1a"><msub id="A4.p1.2.m2.1.1" xref="A4.p1.2.m2.1.1.cmml"><mi id="A4.p1.2.m2.1.1.2" xref="A4.p1.2.m2.1.1.2.cmml">F</mi><mrow id="A4.p1.2.m2.1.1.3" xref="A4.p1.2.m2.1.1.3.cmml"><mi id="A4.p1.2.m2.1.1.3.2" xref="A4.p1.2.m2.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="A4.p1.2.m2.1.1.3.1" xref="A4.p1.2.m2.1.1.3.1.cmml">​</mo><mi id="A4.p1.2.m2.1.1.3.3" xref="A4.p1.2.m2.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="A4.p1.2.m2.1b"><apply id="A4.p1.2.m2.1.1.cmml" xref="A4.p1.2.m2.1.1"><csymbol cd="ambiguous" id="A4.p1.2.m2.1.1.1.cmml" xref="A4.p1.2.m2.1.1">subscript</csymbol><ci id="A4.p1.2.m2.1.1.2.cmml" xref="A4.p1.2.m2.1.1.2">𝐹</ci><apply id="A4.p1.2.m2.1.1.3.cmml" xref="A4.p1.2.m2.1.1.3"><times id="A4.p1.2.m2.1.1.3.1.cmml" xref="A4.p1.2.m2.1.1.3.1"></times><ci id="A4.p1.2.m2.1.1.3.2.cmml" xref="A4.p1.2.m2.1.1.3.2">𝑆</ci><ci id="A4.p1.2.m2.1.1.3.3.cmml" xref="A4.p1.2.m2.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="A4.p1.2.m2.1c">F_{ST}</annotation></semantics></math> on 100 synthetic datasets, with a split of 50 sets generated from a training set including the target and 50 sets generated without.
We report the PG and <math id="A4.p1.3.m3.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="A4.p1.3.m3.1a"><msub id="A4.p1.3.m3.1.1" xref="A4.p1.3.m3.1.1.cmml"><mi id="A4.p1.3.m3.1.1.2" xref="A4.p1.3.m3.1.1.2.cmml">F</mi><mrow id="A4.p1.3.m3.1.1.3" xref="A4.p1.3.m3.1.1.3.cmml"><mi id="A4.p1.3.m3.1.1.3.2" xref="A4.p1.3.m3.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="A4.p1.3.m3.1.1.3.1" xref="A4.p1.3.m3.1.1.3.1.cmml">​</mo><mi id="A4.p1.3.m3.1.1.3.3" xref="A4.p1.3.m3.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="A4.p1.3.m3.1b"><apply id="A4.p1.3.m3.1.1.cmml" xref="A4.p1.3.m3.1.1"><csymbol cd="ambiguous" id="A4.p1.3.m3.1.1.1.cmml" xref="A4.p1.3.m3.1.1">subscript</csymbol><ci id="A4.p1.3.m3.1.1.2.cmml" xref="A4.p1.3.m3.1.1.2">𝐹</ci><apply id="A4.p1.3.m3.1.1.3.cmml" xref="A4.p1.3.m3.1.1.3"><times id="A4.p1.3.m3.1.1.3.1.cmml" xref="A4.p1.3.m3.1.1.3.1"></times><ci id="A4.p1.3.m3.1.1.3.2.cmml" xref="A4.p1.3.m3.1.1.3.2">𝑆</ci><ci id="A4.p1.3.m3.1.1.3.3.cmml" xref="A4.p1.3.m3.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="A4.p1.3.m3.1c">F_{ST}</annotation></semantics></math> as the average across all targets and synthetic datasets.</p>
</div>
<div id="A4.p2" class="ltx_para">
<p id="A4.p2.1" class="ltx_p">Recall that the lower the <math id="A4.p2.1.m1.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="A4.p2.1.m1.1a"><msub id="A4.p2.1.m1.1.1" xref="A4.p2.1.m1.1.1.cmml"><mi id="A4.p2.1.m1.1.1.2" xref="A4.p2.1.m1.1.1.2.cmml">F</mi><mrow id="A4.p2.1.m1.1.1.3" xref="A4.p2.1.m1.1.1.3.cmml"><mi id="A4.p2.1.m1.1.1.3.2" xref="A4.p2.1.m1.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="A4.p2.1.m1.1.1.3.1" xref="A4.p2.1.m1.1.1.3.1.cmml">​</mo><mi id="A4.p2.1.m1.1.1.3.3" xref="A4.p2.1.m1.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="A4.p2.1.m1.1b"><apply id="A4.p2.1.m1.1.1.cmml" xref="A4.p2.1.m1.1.1"><csymbol cd="ambiguous" id="A4.p2.1.m1.1.1.1.cmml" xref="A4.p2.1.m1.1.1">subscript</csymbol><ci id="A4.p2.1.m1.1.1.2.cmml" xref="A4.p2.1.m1.1.1.2">𝐹</ci><apply id="A4.p2.1.m1.1.1.3.cmml" xref="A4.p2.1.m1.1.1.3"><times id="A4.p2.1.m1.1.1.3.1.cmml" xref="A4.p2.1.m1.1.1.3.1"></times><ci id="A4.p2.1.m1.1.1.3.2.cmml" xref="A4.p2.1.m1.1.1.3.2">𝑆</ci><ci id="A4.p2.1.m1.1.1.3.3.cmml" xref="A4.p2.1.m1.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="A4.p2.1.m1.1c">F_{ST}</annotation></semantics></math> value the closer the synthetic samples are to the real data, and PG should be above the random guess baseline of 0.25 for the synthetic data to offer better privacy protection than releasing the real.
Put simply, the ideal “place” in the plots in Figure <a href="#A3.F16" title="Figure 16 ‣ Appendix C Principal Component Analysis (PCA) ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">16</span></a> is the top left.</p>
</div>
<div id="A4.p3" class="ltx_para">
<p id="A4.p3.2" class="ltx_p">For the CEU population (Figure <a href="#A3.F16.sf1" title="In Figure 16 ‣ Appendix C Principal Component Analysis (PCA) ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">16a</span></a>), the synthetic Recomb samples have the best overall utility; however, the average PG is below 0.25.
In line with our utility evaluation, we find that the hybrid model Rec-RBM model generates samples with a <math id="A4.p3.1.m1.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="A4.p3.1.m1.1a"><msub id="A4.p3.1.m1.1.1" xref="A4.p3.1.m1.1.1.cmml"><mi id="A4.p3.1.m1.1.1.2" xref="A4.p3.1.m1.1.1.2.cmml">F</mi><mrow id="A4.p3.1.m1.1.1.3" xref="A4.p3.1.m1.1.1.3.cmml"><mi id="A4.p3.1.m1.1.1.3.2" xref="A4.p3.1.m1.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="A4.p3.1.m1.1.1.3.1" xref="A4.p3.1.m1.1.1.3.1.cmml">​</mo><mi id="A4.p3.1.m1.1.1.3.3" xref="A4.p3.1.m1.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="A4.p3.1.m1.1b"><apply id="A4.p3.1.m1.1.1.cmml" xref="A4.p3.1.m1.1.1"><csymbol cd="ambiguous" id="A4.p3.1.m1.1.1.1.cmml" xref="A4.p3.1.m1.1.1">subscript</csymbol><ci id="A4.p3.1.m1.1.1.2.cmml" xref="A4.p3.1.m1.1.1.2">𝐹</ci><apply id="A4.p3.1.m1.1.1.3.cmml" xref="A4.p3.1.m1.1.1.3"><times id="A4.p3.1.m1.1.1.3.1.cmml" xref="A4.p3.1.m1.1.1.3.1"></times><ci id="A4.p3.1.m1.1.1.3.2.cmml" xref="A4.p3.1.m1.1.1.3.2">𝑆</ci><ci id="A4.p3.1.m1.1.1.3.3.cmml" xref="A4.p3.1.m1.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="A4.p3.1.m1.1c">F_{ST}</annotation></semantics></math> value closer to the real data than the RBM model.
Interestingly, while for Histogram, Correlation, and Ensemble feature sets, the PG for the Rec-RBM is close to 0.25, there is a significant privacy loss for Naive.
In contrast, for the samples generated by the RBM, the Naive yields the highest PG across all feature sets.
The samples generated by the WGAN also have a <math id="A4.p3.2.m2.1" class="ltx_Math" alttext="F_{ST}" display="inline"><semantics id="A4.p3.2.m2.1a"><msub id="A4.p3.2.m2.1.1" xref="A4.p3.2.m2.1.1.cmml"><mi id="A4.p3.2.m2.1.1.2" xref="A4.p3.2.m2.1.1.2.cmml">F</mi><mrow id="A4.p3.2.m2.1.1.3" xref="A4.p3.2.m2.1.1.3.cmml"><mi id="A4.p3.2.m2.1.1.3.2" xref="A4.p3.2.m2.1.1.3.2.cmml">S</mi><mo lspace="0em" rspace="0em" id="A4.p3.2.m2.1.1.3.1" xref="A4.p3.2.m2.1.1.3.1.cmml">​</mo><mi id="A4.p3.2.m2.1.1.3.3" xref="A4.p3.2.m2.1.1.3.3.cmml">T</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="A4.p3.2.m2.1b"><apply id="A4.p3.2.m2.1.1.cmml" xref="A4.p3.2.m2.1.1"><csymbol cd="ambiguous" id="A4.p3.2.m2.1.1.1.cmml" xref="A4.p3.2.m2.1.1">subscript</csymbol><ci id="A4.p3.2.m2.1.1.2.cmml" xref="A4.p3.2.m2.1.1.2">𝐹</ci><apply id="A4.p3.2.m2.1.1.3.cmml" xref="A4.p3.2.m2.1.1.3"><times id="A4.p3.2.m2.1.1.3.1.cmml" xref="A4.p3.2.m2.1.1.3.1"></times><ci id="A4.p3.2.m2.1.1.3.2.cmml" xref="A4.p3.2.m2.1.1.3.2">𝑆</ci><ci id="A4.p3.2.m2.1.1.3.3.cmml" xref="A4.p3.2.m2.1.1.3.3">𝑇</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="A4.p3.2.m2.1c">F_{ST}</annotation></semantics></math> value close to 0; however, for the Histogram, Correlation and Ensemble, the PG values are below 0.25.</p>
</div>
<div id="A4.p4" class="ltx_para">
<p id="A4.p4.1" class="ltx_p">For CHB (Figure <a href="#A3.F16.sf2" title="In Figure 16 ‣ Appendix C Principal Component Analysis (PCA) ‣ On Utility and Privacy in Synthetic Genomic DataPublished in the Proceedings of the 29th Network and Distributed System Security Symposium (NDSS 2022)." class="ltx_ref"><span class="ltx_text ltx_ref_tag">16b</span></a>), the utility results are similar to the CEU population.
However, all targets apart from the ones generated by RecRBM have a lower PG than the CEU population.
This reiterates the fact that one cannot reliably use synthetic data as a good privacy mechanism, as the value of PG is unpredictable and can fluctuate based on target and training set combinations chosen.</p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
</section>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/2102.03313" class="ar5iv-nav-button ar5iv-nav-button-prev">◄</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/2102.03314" class="ar5iv-text-button ar5iv-severity-warning">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+2102.03314">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/2102.03314" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/2102.03315" class="ar5iv-nav-button ar5iv-nav-button-next">►</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Thu Mar  7 02:27:27 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "×";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
