<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[2304.12012] \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications</title><meta property="og:description" content="The real-world implementation of federated learning is complex and requires research and development actions at the crossroad between different domains ranging from data science, to software programming, networking, an…">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="\fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="\fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/2304.12012">

<!--Generated on Thu Feb 29 12:37:20 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<h1 class="ltx_title ltx_title_document">
<span id="id1.id1" class="ltx_ERROR undefined">\fbm</span>:
<br class="ltx_break">Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Francesco Cremonesi
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, Epione Research Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Marc Vesin
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, SED Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Sergen Cansiz
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, SED Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Yannick Bouillard
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, SED Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Irene Balelli
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, Epione Research Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Lucia Innocenti
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, Epione Research Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Santiago Silva
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, Epione Research Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Samy-Safwan Ayed
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, Epione Research Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Riccardo Taiello
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, Epione Research Group, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Laetita Kameni
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Accenture Labs, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Richard Vidal
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Accenture Labs, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Fanny Orlhac
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Laboratoire d’Imagerie Translationnelle en Oncologie (LITO), U1288 Inserm, Université Paris-Saclay, Centre de Recherche de l’Institut Curie, Orsay, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Christophe Nioche
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Laboratoire d’Imagerie Translationnelle en Oncologie (LITO), U1288 Inserm, Université Paris-Saclay, Centre de Recherche de l’Institut Curie, Orsay, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Nathan Lapel
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Nuclear Medicine Department, Henri Becquerel Center, Rouen, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Bastien Houis
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Nuclear Medicine Department, Henri Becquerel Center, Rouen, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Romain Modzelewski
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Nuclear Medicine Department, Henri Becquerel Center, Rouen, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Olivier Humbert
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">UCA, Université Côte d’Azur, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Melek Önen
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">EURECOM, France
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Marco Lorenzi
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Université Côte d’Azur, Inria Sophia Antipolis, Epione Research Group, France
</span></span></span>
</div>

<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p id="id2.id1" class="ltx_p">The real-world implementation of federated learning is complex and requires research and development actions at the crossroad between different domains ranging from data science, to software programming, networking, and security. While today several FL libraries are proposed to data scientists and users, most of these frameworks are not designed to find seamless application in medical use-cases, due to the specific challenges and requirements of working with medical data and hospital infrastructures. Moreover, governance, design principles, and security assumptions of these frameworks are generally not clearly illustrated, thus preventing the adoption in sensitive applications.
Motivated by the current technological landscape of FL in healthcare, in this document we present <span id="id2.id1.1" class="ltx_ERROR undefined">\fbm</span>: a research and development initiative aiming at translating federated learning (FL) into real-world medical research applications. We describe our design space, targeted users, domain constraints, and how these factors affect our current and future software architecture.</p>
</div>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>

<div id="S1.p1" class="ltx_para">
<p id="S1.p1.1" class="ltx_p">The need for large amounts of data to develop Artificial Intelligence (AI) in healthcare has motivated a number of national and international initiatives aimed at creating medical data lakes accessible to researchers, such as the French Health Data Hub <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx10" title="" class="ltx_ref">10</a>]</cite>, the UK BioBank <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx59" title="" class="ltx_ref">59</a>]</cite>, the US ADNI <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx26" title="" class="ltx_ref">26</a>]</cite> and TCGA <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx60" title="" class="ltx_ref">60</a>]</cite>, among the many <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx58" title="" class="ltx_ref">58</a>, <a href="#bib.bibx40" title="" class="ltx_ref">40</a>, <a href="#bib.bibx7" title="" class="ltx_ref">7</a>]</cite>.
In spite of these initiatives, there are still major bottlenecks preventing the widespread availability of large centralized repositories of healthcare information <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx63" title="" class="ltx_ref">63</a>]</cite>.</p>
</div>
<div id="S1.p2" class="ltx_para">
<p id="S1.p2.1" class="ltx_p">To overcome these limitations, Federated Learning (FL) has been proposed as a working paradigm to enable the training of ML models on large datasets from diverse sources while guaranteeing the respect of data privacy and governance.
The basic paradigm of FL consists of iterating the following steps: i) model training is performed locally in the hospitals starting from a common initialization, ii) the resulting model parameters are subsequently shared (instead of the data) and aggregated, to define a global model iii) transmitted back to the hospitals to initiate a new local training step.
Under certain conditions <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx39" title="" class="ltx_ref">39</a>]</cite>, this procedure is guaranteed to converge to a final global model representing an optimal consensus among the hospitals participating in the experiment.
FL is particularly suited for applications in sensitive domains, such as healthcare and biomedical research <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx48" title="" class="ltx_ref">48</a>, <a href="#bib.bibx9" title="" class="ltx_ref">9</a>, <a href="#bib.bibx13" title="" class="ltx_ref">13</a>]</cite>.
The current societal and economical interest in FL for healthcare is paramount <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx56" title="" class="ltx_ref">56</a>, <a href="#bib.bibx52" title="" class="ltx_ref">52</a>]</cite>, as demonstrated by the several large-scale medical research projects based on FL at the national and international level, focusing for example on rare hematological diseases<span id="footnote1" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_tag ltx_tag_note">1</span><a target="_blank" href="https://genomed4all.eu/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://genomed4all.eu/</a></span></span></span>, drug development<span id="footnote2" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">2</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">2</sup><span class="ltx_tag ltx_tag_note">2</span><a target="_blank" href="https://www.melloddy.eu/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.melloddy.eu/</a></span></span></span>, blood cancer<span id="footnote3" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">3</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">3</sup><span class="ltx_tag ltx_tag_note">3</span><a target="_blank" href="https://www.harmony-alliance.eu/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.harmony-alliance.eu/</a></span></span></span>, among the many <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx15" title="" class="ltx_ref">15</a>, <a href="#bib.bibx53" title="" class="ltx_ref">53</a>, <a href="#bib.bibx19" title="" class="ltx_ref">19</a>]</cite>.</p>
</div>
<div id="S1.p3" class="ltx_para">
<p id="S1.p3.1" class="ltx_p">In spite of the current popularity, the real-world implementation of FL is complex and requires research and development actions at the crossroad between different domains spanning data science, software programming, networking, and security.
Today, several FL software frameworks are currently being proposed to data scientists and users, based on different design spaces, goals, and with varying degrees of software maturity.
Nevertheless, most of these frameworks are not designed to find seamless application in medical use-cases, due to the specific challenges and requirements of working with medical data and hospital infrastructures.
Furthermore, several widely-used FL libraries oftentimes devote little attention to describe the design space and guiding principles, while the spotlight is often placed on describing implementation details without justifying particular design choices.</p>
</div>
<div id="S1.p4" class="ltx_para">
<p id="S1.p4.1" class="ltx_p">In this document we present <span id="S1.p4.1.1" class="ltx_ERROR undefined">\fbm</span>: a research and development initiative aiming at translating FL into real-world medical research applications, motivated by the current technological landscape of FL for healthcare. We describe our design space, targeted users, domain constraints, and how these factors affect our current and future software architecture.
While implementation details may change with the evolution of new technologies and contributions from a growing pool of developers, we believe that a description of <span id="S1.p4.1.2" class="ltx_ERROR undefined">\fbm</span>’s philosophy and guiding principles, along with the relevant architectural details, provides a faithful and transparent representation of our goal and ambitions.</p>
</div>
<section id="S1.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.1 </span>Contribution</h3>

<div id="S1.SS1.p1" class="ltx_para">
<p id="S1.SS1.p1.1" class="ltx_p">While several FL frameworks are currently available, none of them have documented a clear set of design principles and guiding concepts inspired by the application domain, nor a demonstration of how their architecture and implementation satisfy the specific requirements of FL in clinical applications.
In what follows, we identify the requirements arising from the needs of medical data owners and biomedical data scientists, and show how <span id="S1.SS1.p1.1.1" class="ltx_ERROR undefined">\fbm</span>addresses the specific challenges of this domain.</p>
</div>
<div id="S1.SS1.p2" class="ltx_para">
<p id="S1.SS1.p2.1" class="ltx_p">To meet the strict security requirements typical of medical environments, <span id="S1.SS1.p2.1.1" class="ltx_ERROR undefined">\fbm</span>focuses on empowering the user with a tight control of data management and model training process.
This is based on a deployment workflow enabling easy setup of its main components, on prototypes of a data and model governance system with a graphical interface for non-technical users such as clinical data managers and physicians, on a Jupyter notebook interface for researchers and data scientists, and on extensive documentation and tutorials targeting biomedical data scientists as well as health data providers<span id="footnote4" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">4</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">4</sup><span class="ltx_tag ltx_tag_note">4</span><a target="_blank" href="https://fedbiomed.gitlabpages.inria.fr/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://fedbiomed.gitlabpages.inria.fr/</a></span></span></span>.
Overall, the straightforward design of <span id="S1.SS1.p2.1.2" class="ltx_ERROR undefined">\fbm</span>aims at simplifying the development and deployment of federated learning analysis in real-world healthcare research.</p>
</div>
</section>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Federated Learning for biomedical research applications: design considerations</h2>

<div id="S2.p1" class="ltx_para">
<p id="S2.p1.1" class="ltx_p">The application of ML methods, and in particular of FL, in the context of medical data analysis presents unique challenges requiring a targeted approach.
First and foremost, is the conflict between the need for large datasets to train ML models and data sharing regulations, such as the European General Data Protection Regulation (GDPR)<span id="footnote5" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">5</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">5</sup><span class="ltx_tag ltx_tag_note">5</span><a target="_blank" href="https://gdpr-info.eu/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://gdpr-info.eu/</a></span></span></span> and the US Health Insurance Portability and Accountability Act (HIPAA)<span id="footnote6" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">6</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">6</sup><span class="ltx_tag ltx_tag_note">6</span><a target="_blank" href="https://www.cdc.gov/phlp/publications/topic/hipaa.html" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.cdc.gov/phlp/publications/topic/hipaa.html</a></span></span></span>.
Additionally, ethical, economical, and technical barriers hinder the sharing of medical information <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx63" title="" class="ltx_ref">63</a>, <a href="#bib.bibx61" title="" class="ltx_ref">61</a>]</cite>.
The entities that provide medical records for FL experiments are therefore compelled to enforce strict data governance policies on their data, requiring strong security and privacy guarantees as well as retaining the ability to exert fine-grained control over data processing and flow.</p>
</div>
<div id="S2.p2" class="ltx_para">
<p id="S2.p2.1" class="ltx_p">Medical data are generally not directly amenable to FL analysis: they are often stored in unstructured, potentially proprietary formats inside independent silos, leading to large degrees of heterogeneity making data potentially biased and difficult to compare <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx24" title="" class="ltx_ref">24</a>, <a href="#bib.bibx61" title="" class="ltx_ref">61</a>]</cite>.
For example, the lack of standardized coded definitions may lead to noisy labels, with a detrimental effect on training performance, especially for medical data where missingness is a common problem <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx21" title="" class="ltx_ref">21</a>]</cite>.
Big data in medicine is characterized by relatively low volumes with high information density <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx24" title="" class="ltx_ref">24</a>]</cite>, and often requires integration of multiple data acquisition methods, bringing all the challenges associated with the analysis of multimodal data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx30" title="" class="ltx_ref">30</a>]</cite>.
Furthermore, hospital IT infrastructure has been designed to support clinical and billing operations, but is not optimized for data analytics <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx24" title="" class="ltx_ref">24</a>]</cite>.
The procedures for installing and operating FL software may vary wildly across different hospitals, leading to difficulties during the deployment process and inconsistencies in the execution.</p>
</div>
<div id="S2.p3" class="ltx_para">
<p id="S2.p3.1" class="ltx_p">In a biomedical research setting, the prototypical FL experiment consists of a dynamic and highly interactive series of training rounds interspersed with sessions of model and hyperparameter tuning, interpretation of partial results with domain experts, and general debugging.
This process requires a large degree of interactivity that can be in contrast with the design of FL systems for other applications that are more focused on batch execution of training rounds, model stability, and high availability of the infrastructure.</p>
</div>
<div id="S2.p4" class="ltx_para">
<p id="S2.p4.1" class="ltx_p">Any software being operated in the context of biomedical research must satisfy strict security rules arising not only from data privacy concerns but also from intellectual property and compliance to guidelines <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx12" title="" class="ltx_ref">12</a>]</cite>.
The FL process itself exposes multiples vulnerabilities such as model inversion, membership inference, and model poisoning attacks <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx6" title="" class="ltx_ref">6</a>, <a href="#bib.bibx57" title="" class="ltx_ref">57</a>]</cite>, which must be mitigated through a combination of classical cybersecurity approaches — such as e.g. encryption, firewalls, malware protection, and network segmentation— as well as FL-specific techniques such as secure aggregation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx36" title="" class="ltx_ref">36</a>]</cite>.</p>
</div>
<section id="S2.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.1 </span>Primary Requirements</h3>

<div id="S2.SS1.p1" class="ltx_para">
<p id="S2.SS1.p1.1" class="ltx_p">From the challenges described above we derive a set of requirements that <span id="S2.SS1.p1.1.1" class="ltx_ERROR undefined">\fbm</span>aims at satisfying, categorized in four primary (i.e. must-have), four secondary (i.e. nice-to-have) and three minor (i.e. could-have) requirements. We summarize the challenges and requirements in Table <a href="#S2.T1" title="Table 1 ‣ Security ‣ 2.1 Primary Requirements ‣ 2 Federated Learning for biomedical research applications: design considerations ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.</p>
</div>
<section id="S2.SS1.SSSx1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">Data and model governance</h4>

<div id="S2.SS1.SSSx1.p1" class="ltx_para">
<p id="S2.SS1.SSSx1.p1.1" class="ltx_p">We define these as the granting the following rights to clinical data nodes: i) the ability to review, add or revoke at any time the availability of any given dataset for federated training; ii) the ability to approve, audit and monitor the execution of specific FL workflows; iii) the ability to review, audit and customize the deployment of the software infrastructure.
Furthermore, as the time of data managers and clinical experts is often a scarce resource, the ability to exercise such fine-grained control on data governance and processing should be provided in a simple user interface requiring only minimal learning efforts.</p>
</div>
</section>
<section id="S2.SS1.SSSx2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">Integration with biomedical data sources</h4>

<div id="S2.SS1.SSSx2.p1" class="ltx_para">
<p id="S2.SS1.SSSx2.p1.1" class="ltx_p">An FL framework targeting interoperability should at the very least be aware of existing interoperability efforts in medical data management and analysis, and ideally should offer direct and seamless integration for the management of data in such formats.</p>
</div>
</section>
<section id="S2.SS1.SSSx3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">Researcher interactivity</h4>

<div id="S2.SS1.SSSx3.p1" class="ltx_para">
<p id="S2.SS1.SSSx3.p1.1" class="ltx_p">FL frameworks suited for research purposes should provide the ability to launch, stop and generally manipulate the training process, modify model and training parameters on the fly, resume training from checkpoints, and monitor convergence, while respecting the FL paradigm, the data providers’ privacy, and node’s governance requirements.</p>
</div>
</section>
<section id="S2.SS1.SSSx4" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">Security</h4>

<div id="S2.SS1.SSSx4.p1" class="ltx_para">
<p id="S2.SS1.SSSx4.p1.1" class="ltx_p">An FL framework for the biomedical research domain should provide a secure environment for the FL technical infrastructure, minimizing the surface of attacks on the data providers’ systems through the framework itself, implementing network segmentation, securing network communication, and insulating the execution from external attackers.
Furthermore, the framework should support and allow for easy activation of gradient protection, against model-targeted attacks such as model poisoning, membership inference, and model inversion.</p>
</div>
<figure id="S2.T1" class="ltx_table">
<table id="S2.T1.1" class="ltx_tabular ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S2.T1.1.1.1" class="ltx_tr">
<th id="S2.T1.1.1.1.1" class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_l ltx_border_r ltx_border_t" style="padding-bottom:4.30554pt;">
<span id="S2.T1.1.1.1.1.1" class="ltx_inline-block ltx_align_top">
<span id="S2.T1.1.1.1.1.1.1" class="ltx_p"><span id="S2.T1.1.1.1.1.1.1.1" class="ltx_text ltx_font_bold">Challenges</span></span>
</span>
</th>
<th id="S2.T1.1.1.1.2" class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_r ltx_border_t" style="padding-bottom:4.30554pt;">
<span id="S2.T1.1.1.1.2.1" class="ltx_inline-block ltx_align_top">
<span id="S2.T1.1.1.1.2.1.1" class="ltx_p"><span id="S2.T1.1.1.1.2.1.1.1" class="ltx_text ltx_font_bold">Primary requirements</span></span>
</span>
</th>
<th id="S2.T1.1.1.1.3" class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_r ltx_border_t" style="padding-bottom:4.30554pt;">
<span id="S2.T1.1.1.1.3.1" class="ltx_inline-block ltx_align_top">
<span id="S2.T1.1.1.1.3.1.1" class="ltx_p"><span id="S2.T1.1.1.1.3.1.1.1" class="ltx_text ltx_font_bold">Secondary and minor</span></span>
</span>
</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S2.T1.1.2.1" class="ltx_tr">
<td id="S2.T1.1.2.1.1" class="ltx_td ltx_align_justify ltx_border_b ltx_border_l ltx_border_r ltx_border_tt">
<span id="S2.T1.1.2.1.1.1" class="ltx_inline-block ltx_align_top">
<span id="S2.I1" class="ltx_itemize">
<span id="S2.I1.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i1.p1" class="ltx_para">
<span id="S2.I1.i1.p1.1" class="ltx_p">Data privacy and protection</span>
</span></span>
<span id="S2.I1.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i2.p1" class="ltx_para">
<span id="S2.I1.i2.p1.1" class="ltx_p">Siloed data</span>
</span></span>
<span id="S2.I1.i3" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i3.p1" class="ltx_para">
<span id="S2.I1.i3.p1.1" class="ltx_p">Heterogeneous, unstructured data</span>
</span></span>
<span id="S2.I1.i4" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i4.p1" class="ltx_para">
<span id="S2.I1.i4.p1.1" class="ltx_p">Integration of multiple data modalities</span>
</span></span>
<span id="S2.I1.i5" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i5.p1" class="ltx_para">
<span id="S2.I1.i5.p1.1" class="ltx_p">Dynamic model development cycle</span>
</span></span>
<span id="S2.I1.i6" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i6.p1" class="ltx_para">
<span id="S2.I1.i6.p1.1" class="ltx_p">Cyberattacks</span>
</span></span>
<span id="S2.I1.i7" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i7.p1" class="ltx_para">
<span id="S2.I1.i7.p1.1" class="ltx_p">Model and gradient attacks</span>
</span></span>
<span id="S2.I1.i8" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i8.p1" class="ltx_para">
<span id="S2.I1.i8.p1.1" class="ltx_p">Suboptimal IT infrastructure and computing capabilities</span>
</span></span>
<span id="S2.I1.i9" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">•</span> 
<span id="S2.I1.i9.p1" class="ltx_para">
<span id="S2.I1.i9.p1.1" class="ltx_p">IP protection</span>
</span></span>
</span>
</span>
</td>
<td id="S2.T1.1.2.1.2" class="ltx_td ltx_align_justify ltx_border_b ltx_border_r ltx_border_tt">
<span id="S2.T1.1.2.1.2.1" class="ltx_inline-block ltx_align_top">
<span id="S2.T1.1.2.1.2.1.1" class="ltx_p"><span id="S2.T1.1.2.1.2.1.1.1" class="ltx_text ltx_font_bold">Data and model governance
<br class="ltx_break"></span>Powerful and easy-to-use governance tools for nodes.
<br class="ltx_break"><span id="S2.T1.1.2.1.2.1.1.2" class="ltx_text ltx_font_bold">Integration with biomedical data sources
<br class="ltx_break"></span>Support biomedical and interoperability standards; support diverse computing infrastructures.

<br class="ltx_break"><span id="S2.T1.1.2.1.2.1.1.3" class="ltx_text ltx_font_bold">Researcher interactivity
<br class="ltx_break"></span>Support a highly interactive, dynamic model development cycle steered by the researcher.

<br class="ltx_break"><span id="S2.T1.1.2.1.2.1.1.4" class="ltx_text ltx_font_bold">Security
<br class="ltx_break"></span>Secure software and related IT infrastructure against hacking; support model and gradient protection strategies.

<br class="ltx_break"></span>
</span>
</td>
<td id="S2.T1.1.2.1.3" class="ltx_td ltx_align_justify ltx_border_b ltx_border_r ltx_border_tt">
<span id="S2.T1.1.2.1.3.1" class="ltx_inline-block ltx_align_top">
<span id="S2.I2" class="ltx_itemize">
<span id="S2.I2.i1" class="ltx_item" style="list-style-type:none;"> 
<span id="S2.I2.i1.p1" class="ltx_para">
<span id="S2.I2.i1.p1.1" class="ltx_p">federated pre/post processing</span>
</span></span>
<span id="S2.I2.i2" class="ltx_item" style="list-style-type:none;"> 
<span id="S2.I2.i2.p1" class="ltx_para">
<span id="S2.I2.i2.p1.1" class="ltx_p">support for several ML libraries</span>
</span></span>
<span id="S2.I2.i3" class="ltx_item" style="list-style-type:none;"> 
<span id="S2.I2.i3.p1" class="ltx_para">
<span id="S2.I2.i3.p1.1" class="ltx_p">portability</span>
</span></span>
<span id="S2.I2.i4" class="ltx_item" style="list-style-type:none;"> 
<span id="S2.I2.i4.p1" class="ltx_para">
<span id="S2.I2.i4.p1.1" class="ltx_p">drop-out tolerance</span>
</span></span>
<span id="S2.I2.i5" class="ltx_item" style="list-style-type:none;"> 
<span id="S2.I2.i5.p1" class="ltx_para">
<span id="S2.I2.i5.p1.1" class="ltx_p">high availability</span>
</span></span>
<span id="S2.I2.i6" class="ltx_item" style="list-style-type:none;"> 
<span id="S2.I2.i6.p1" class="ltx_para">
<span id="S2.I2.i6.p1.1" class="ltx_p">scalability</span>
</span></span>
<span id="S2.I2.i7" class="ltx_item" style="list-style-type:none;"> 
<span id="S2.I2.i7.p1" class="ltx_para">
<span id="S2.I2.i7.p1.1" class="ltx_p">lightweight</span>
</span></span>
</span>
</span>
</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 1: </span>Summary of challenges and requirements addressed by fed-BioMed in the application of FL to biomedical research.</figcaption>
</figure>
</section>
</section>
<section id="S2.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.2 </span>Secondary and Minor Requirements</h3>

<div id="S2.SS2.p1" class="ltx_para">
<p id="S2.SS2.p1.1" class="ltx_p"><span id="S2.SS2.p1.1.1" class="ltx_text ltx_font_bold">Federated pre/post-processing</span>, i.e. the framework’s ability to support data or model pre/post-processing in a federated approach.</p>
</div>
<div id="S2.SS2.p2" class="ltx_para">
<p id="S2.SS2.p2.1" class="ltx_p"><span id="S2.SS2.p2.1.1" class="ltx_text ltx_font_bold">Supported ML libraries/frameworks</span>, meaning the framework’s capability to seamlessly integrate with a variety of state-of-the-art ML libraries and providing state-of-the-art FL algorithms.</p>
</div>
<div id="S2.SS2.p3" class="ltx_para">
<p id="S2.SS2.p3.1" class="ltx_p"><span id="S2.SS2.p3.1.1" class="ltx_text ltx_font_bold">Portability</span>, intended as reproducibility of the development environment as obtained e.g. by the use of containers and virtual environments.</p>
</div>
<div id="S2.SS2.p4" class="ltx_para">
<p id="S2.SS2.p4.1" class="ltx_p"><span id="S2.SS2.p4.1.1" class="ltx_text ltx_font_bold">Drop-out tolerance</span>, as in the framework’s resilience to node drop-outs, other unexpected failure events, or the framework’s ability to provide fine-grained node selection during training. Note that in a controlled cross-silo setting unexpected drop-outs are usually considered less relevant than in a cross-device FL deployment.</p>
</div>
<div id="S2.SS2.p5" class="ltx_para">
<p id="S2.SS2.p5.1" class="ltx_p">Finally, we note that some other requirements are often cited in the broader context of FL applications but are of relatively minor importance to the domain of biomedical research, such as e.g. high availability of servers, scalability in terms of number of clients, and lightweight software implementation.</p>
</div>
</section>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Background: FL Frameworks Landscape</h2>

<div id="S3.p1" class="ltx_para">
<p id="S3.p1.1" class="ltx_p">The number of FL frameworks has dramatically surged in recent years, witnessing the growing interest in the applications of this technology <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx31" title="" class="ltx_ref">31</a>, <a href="#bib.bibx56" title="" class="ltx_ref">56</a>, <a href="#bib.bibx44" title="" class="ltx_ref">44</a>]</cite>.
We are aware of well-established products whose focus is not related to biomedical applications, but whose implementation does not exclude future deployments in this domain, such as e.g. Tensorflow Federated (TFF) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx5" title="" class="ltx_ref">5</a>]</cite>, FedML <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx23" title="" class="ltx_ref">23</a>]</cite>, IBM-FL <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx34" title="" class="ltx_ref">34</a>]</cite>, FATE <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx33" title="" class="ltx_ref">33</a>]</cite>, PaddleFL <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx35" title="" class="ltx_ref">35</a>]</cite>, and PySyft <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx51" title="" class="ltx_ref">51</a>]</cite>.
Henceforth, we restrict our analysis to SubstraFL <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx46" title="" class="ltx_ref">46</a>]</cite>, OpenFL <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx17" title="" class="ltx_ref">17</a>]</cite>, Flare <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx50" title="" class="ltx_ref">50</a>]</cite>, and Flower <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx4" title="" class="ltx_ref">4</a>]</cite>, which are frameworks that have already been applied in medical use-cases, in accordance with the focus of this paper.
Table <a href="#S3.T2" title="Table 2 ‣ 3 Background: FL Frameworks Landscape ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> summarizes our frameworks review in light of the requirements identified in section <a href="#S2.SS1" title="2.1 Primary Requirements ‣ 2 Federated Learning for biomedical research applications: design considerations ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.1</span></a>.</p>
</div>
<figure id="S3.T2" class="ltx_table">
<table id="S3.T2.1" class="ltx_tabular ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S3.T2.1.1.1" class="ltx_tr">
<th id="S3.T2.1.1.1.1" class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_th_row ltx_border_l ltx_border_r ltx_border_t" style="padding-bottom:3.87498pt;"></th>
<th id="S3.T2.1.1.1.2" class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_t" style="padding-bottom:3.87498pt;">
<span id="S3.T2.1.1.1.2.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.1.1.2.1.1" class="ltx_p"><span id="S3.T2.1.1.1.2.1.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">SubstraFL</span></span>
</span>
</th>
<th id="S3.T2.1.1.1.3" class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_t" style="padding-bottom:3.87498pt;">
<span id="S3.T2.1.1.1.3.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.1.1.3.1.1" class="ltx_p"><span id="S3.T2.1.1.1.3.1.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">OpenFL</span></span>
</span>
</th>
<th id="S3.T2.1.1.1.4" class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_t" style="padding-bottom:3.87498pt;">
<span id="S3.T2.1.1.1.4.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.1.1.4.1.1" class="ltx_p ltx_align_left"><span id="S3.T2.1.1.1.4.1.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Flare</span></span>
</span>
</th>
<th id="S3.T2.1.1.1.5" class="ltx_td ltx_align_justify ltx_th ltx_th_column ltx_border_r ltx_border_t" style="padding-bottom:3.87498pt;">
<span id="S3.T2.1.1.1.5.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.1.1.5.1.1" class="ltx_p"><span id="S3.T2.1.1.1.5.1.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Flower</span></span>
</span>
</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S3.T2.1.2.1" class="ltx_tr">
<th id="S3.T2.1.2.1.1" class="ltx_td ltx_align_justify ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_tt" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.2.1.1.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.2.1.1.1.1" class="ltx_p"><span id="S3.T2.1.2.1.1.1.1.1" class="ltx_text" style="font-size:90%;">Design focus</span></span>
</span>
</th>
<td id="S3.T2.1.2.1.2" class="ltx_td ltx_align_justify ltx_border_tt" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.2.1.2.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.2.1.2.1.1" class="ltx_p"><span id="S3.T2.1.2.1.2.1.1.1" class="ltx_text" style="font-size:90%;">collaboration, privacy, </span><span id="S3.T2.1.2.1.2.1.1.2" class="ltx_text" style="font-size:90%;">traceability</span></span>
</span>
</td>
<td id="S3.T2.1.2.1.3" class="ltx_td ltx_align_justify ltx_border_tt" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.2.1.3.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.2.1.3.1.1" class="ltx_p"><span id="S3.T2.1.2.1.3.1.1.1" class="ltx_text" style="font-size:90%;">cybersecurity</span></span>
</span>
</td>
<td id="S3.T2.1.2.1.4" class="ltx_td ltx_align_justify ltx_border_tt" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.2.1.4.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.2.1.4.1.1" class="ltx_p ltx_align_left"><span id="S3.T2.1.2.1.4.1.1.1" class="ltx_text" style="font-size:90%;">scalability, flexibility, lightweight</span></span>
</span>
</td>
<td id="S3.T2.1.2.1.5" class="ltx_td ltx_align_justify ltx_border_r ltx_border_tt" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.2.1.5.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.2.1.5.1.1" class="ltx_p"><span id="S3.T2.1.2.1.5.1.1.1" class="ltx_text" style="font-size:90%;">scalability, flexibility, agnostic to clients, communication and privacy</span></span>
</span>
</td>
</tr>
<tr id="S3.T2.1.3.2" class="ltx_tr">
<th id="S3.T2.1.3.2.1" class="ltx_td ltx_align_justify ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.3.2.1.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.3.2.1.1.1" class="ltx_p"><span id="S3.T2.1.3.2.1.1.1.1" class="ltx_text" style="font-size:90%;">Data </span><span id="S3.T2.1.3.2.1.1.1.2" class="ltx_text" style="font-size:90%;">governance</span></span>
</span>
</th>
<td id="S3.T2.1.3.2.2" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.3.2.2.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.3.2.2.1.1" class="ltx_p"><span id="S3.T2.1.3.2.2.1.1.1" class="ltx_text" style="font-size:90%;">Distributed ledger, RBAC, GUI</span></span>
</span>
</td>
<td id="S3.T2.1.3.2.3" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.3.2.3.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.3.2.3.1.1" class="ltx_p"><span id="S3.T2.1.3.2.3.1.1.1" class="ltx_text" style="font-size:90%;">HTTP API, Python SDK</span></span>
</span>
</td>
<td id="S3.T2.1.3.2.4" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.3.2.4.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.3.2.4.1.1" class="ltx_p ltx_align_left"><span id="S3.T2.1.3.2.4.1.1.1" class="ltx_text" style="font-size:90%;">configuration files, Python SDK</span></span>
</span>
</td>
<td id="S3.T2.1.3.2.5" class="ltx_td ltx_align_justify ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.3.2.5.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.3.2.5.1.1" class="ltx_p"><span id="S3.T2.1.3.2.5.1.1.1" class="ltx_text" style="font-size:90%;">Python SDK</span></span>
</span>
</td>
</tr>
<tr id="S3.T2.1.4.3" class="ltx_tr">
<th id="S3.T2.1.4.3.1" class="ltx_td ltx_align_justify ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.4.3.1.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.4.3.1.1.1" class="ltx_p"><span id="S3.T2.1.4.3.1.1.1.1" class="ltx_text" style="font-size:90%;">Model </span><span id="S3.T2.1.4.3.1.1.1.2" class="ltx_text" style="font-size:90%;">governance</span></span>
</span>
</th>
<td id="S3.T2.1.4.3.2" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.4.3.2.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.4.3.2.1.1" class="ltx_p"><span id="S3.T2.1.4.3.2.1.1.1" class="ltx_text" style="font-size:90%;">Distributed ledger, RBAC, GUI</span></span>
</span>
</td>
<td id="S3.T2.1.4.3.3" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.4.3.3.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.4.3.3.1.1" class="ltx_p"><span id="S3.T2.1.4.3.3.1.1.1" class="ltx_text" style="font-size:90%;">None built-in</span></span>
</span>
</td>
<td id="S3.T2.1.4.3.4" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.4.3.4.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.4.3.4.1.1" class="ltx_p ltx_align_left"><span id="S3.T2.1.4.3.4.1.1.1" class="ltx_text" style="font-size:90%;">None built-in</span></span>
</span>
</td>
<td id="S3.T2.1.4.3.5" class="ltx_td ltx_align_justify ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.4.3.5.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.4.3.5.1.1" class="ltx_p"><span id="S3.T2.1.4.3.5.1.1.1" class="ltx_text" style="font-size:90%;">None built-in</span></span>
</span>
</td>
</tr>
<tr id="S3.T2.1.5.4" class="ltx_tr">
<th id="S3.T2.1.5.4.1" class="ltx_td ltx_align_justify ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.5.4.1.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.5.4.1.1.1" class="ltx_p"><span id="S3.T2.1.5.4.1.1.1.1" class="ltx_text" style="font-size:90%;">Integration with medical domain</span></span>
</span>
</th>
<td id="S3.T2.1.5.4.2" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.5.4.2.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.5.4.2.1.1" class="ltx_p"><span id="S3.T2.1.5.4.2.1.1.1" class="ltx_text" style="font-size:90%;">Only </span><span id="S3.T2.1.5.4.2.1.1.2" class="ltx_text" style="font-size:90%;">demonstrators</span></span>
</span>
</td>
<td id="S3.T2.1.5.4.3" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.5.4.3.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.5.4.3.1.1" class="ltx_p"><span id="S3.T2.1.5.4.3.1.1.1" class="ltx_text" style="font-size:90%;">Only </span><span id="S3.T2.1.5.4.3.1.1.2" class="ltx_text" style="font-size:90%;">demonstrators</span></span>
</span>
</td>
<td id="S3.T2.1.5.4.4" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.5.4.4.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.5.4.4.1.1" class="ltx_p ltx_align_left"><span id="S3.T2.1.5.4.4.1.1.1" class="ltx_text" style="font-size:90%;">Only </span><span id="S3.T2.1.5.4.4.1.1.2" class="ltx_text" style="font-size:90%;">demonstrators</span></span>
</span>
</td>
<td id="S3.T2.1.5.4.5" class="ltx_td ltx_align_justify ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.5.4.5.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.5.4.5.1.1" class="ltx_p"><span id="S3.T2.1.5.4.5.1.1.1" class="ltx_text" style="font-size:90%;">None built-in</span></span>
</span>
</td>
</tr>
<tr id="S3.T2.1.6.5" class="ltx_tr">
<th id="S3.T2.1.6.5.1" class="ltx_td ltx_align_justify ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.6.5.1.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.6.5.1.1.1" class="ltx_p"><span id="S3.T2.1.6.5.1.1.1.1" class="ltx_text" style="font-size:90%;">Researcher </span><span id="S3.T2.1.6.5.1.1.1.2" class="ltx_text" style="font-size:90%;">interactivity</span></span>
</span>
</th>
<td id="S3.T2.1.6.5.2" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.6.5.2.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.6.5.2.1.1" class="ltx_p"><span id="S3.T2.1.6.5.2.1.1.1" class="ltx_text" style="font-size:90%;">Explicit focus on production</span></span>
</span>
</td>
<td id="S3.T2.1.6.5.3" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.6.5.3.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.6.5.3.1.1" class="ltx_p"><span id="S3.T2.1.6.5.3.1.1.1" class="ltx_text" style="font-size:90%;">Limited</span></span>
</span>
</td>
<td id="S3.T2.1.6.5.4" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.6.5.4.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.6.5.4.1.1" class="ltx_p ltx_align_left"><span id="S3.T2.1.6.5.4.1.1.1" class="ltx_text" style="font-size:90%;">Limited</span></span>
</span>
</td>
<td id="S3.T2.1.6.5.5" class="ltx_td ltx_align_justify ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.6.5.5.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.6.5.5.1.1" class="ltx_p"><span id="S3.T2.1.6.5.5.1.1.1" class="ltx_text" style="font-size:90%;">High interactivity through Python SDK</span></span>
</span>
</td>
</tr>
<tr id="S3.T2.1.7.6" class="ltx_tr">
<th id="S3.T2.1.7.6.1" class="ltx_td ltx_align_justify ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.7.6.1.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.7.6.1.1.1" class="ltx_p"><span id="S3.T2.1.7.6.1.1.1.1" class="ltx_text" style="font-size:90%;">Cybersecurity</span></span>
</span>
</th>
<td id="S3.T2.1.7.6.2" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.7.6.2.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.7.6.2.1.1" class="ltx_p"><span id="S3.T2.1.7.6.2.1.1.1" class="ltx_text" style="font-size:90%;">encrypted </span><span id="S3.T2.1.7.6.2.1.1.2" class="ltx_text" style="font-size:90%;">communication</span></span>
</span>
</td>
<td id="S3.T2.1.7.6.3" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.7.6.3.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.7.6.3.1.1" class="ltx_p"><span id="S3.T2.1.7.6.3.1.1.1" class="ltx_text" style="font-size:90%;">TEE, </span><span id="S3.T2.1.7.6.3.1.1.2" class="ltx_text" style="font-size:90%;">encrypted</span><span id="S3.T2.1.7.6.3.1.1.3" class="ltx_text" style="font-size:90%;"> </span><span id="S3.T2.1.7.6.3.1.1.4" class="ltx_text" style="font-size:90%;">communication</span><span id="S3.T2.1.7.6.3.1.1.5" class="ltx_text" style="font-size:90%;">, PKI</span></span>
</span>
</td>
<td id="S3.T2.1.7.6.4" class="ltx_td ltx_align_justify ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.7.6.4.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.7.6.4.1.1" class="ltx_p ltx_align_left"><span id="S3.T2.1.7.6.4.1.1.1" class="ltx_text" style="font-size:90%;">encrypted </span><span id="S3.T2.1.7.6.4.1.1.2" class="ltx_text" style="font-size:90%;">communication</span></span>
</span>
</td>
<td id="S3.T2.1.7.6.5" class="ltx_td ltx_align_justify ltx_border_r ltx_border_t" style="padding-bottom:30.99988pt;">
<span id="S3.T2.1.7.6.5.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.7.6.5.1.1" class="ltx_p"><span id="S3.T2.1.7.6.5.1.1.1" class="ltx_text" style="font-size:90%;">encrypted communication</span></span>
</span>
</td>
</tr>
<tr id="S3.T2.1.8.7" class="ltx_tr">
<th id="S3.T2.1.8.7.1" class="ltx_td ltx_align_justify ltx_th ltx_th_row ltx_border_b ltx_border_l ltx_border_r ltx_border_t">
<span id="S3.T2.1.8.7.1.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.8.7.1.1.1" class="ltx_p"><span id="S3.T2.1.8.7.1.1.1.1" class="ltx_text" style="font-size:90%;">Model and </span><span id="S3.T2.1.8.7.1.1.1.2" class="ltx_text" style="font-size:90%;">update</span><span id="S3.T2.1.8.7.1.1.1.3" class="ltx_text" style="font-size:90%;"> attacks</span></span>
</span>
</th>
<td id="S3.T2.1.8.7.2" class="ltx_td ltx_align_justify ltx_border_b ltx_border_t">
<span id="S3.T2.1.8.7.2.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.8.7.2.1.1" class="ltx_p"><span id="S3.T2.1.8.7.2.1.1.1" class="ltx_text" style="font-size:90%;">Future development</span></span>
</span>
</td>
<td id="S3.T2.1.8.7.3" class="ltx_td ltx_align_justify ltx_border_b ltx_border_t">
<span id="S3.T2.1.8.7.3.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.8.7.3.1.1" class="ltx_p"><span id="S3.T2.1.8.7.3.1.1.1" class="ltx_text" style="font-size:90%;">None built-in</span></span>
</span>
</td>
<td id="S3.T2.1.8.7.4" class="ltx_td ltx_align_justify ltx_border_b ltx_border_t">
<span id="S3.T2.1.8.7.4.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.8.7.4.1.1" class="ltx_p ltx_align_left"><span id="S3.T2.1.8.7.4.1.1.1" class="ltx_text" style="font-size:90%;">Only demonstrators</span></span>
</span>
</td>
<td id="S3.T2.1.8.7.5" class="ltx_td ltx_align_justify ltx_border_b ltx_border_r ltx_border_t">
<span id="S3.T2.1.8.7.5.1" class="ltx_inline-block ltx_align_top">
<span id="S3.T2.1.8.7.5.1.1" class="ltx_p"><span id="S3.T2.1.8.7.5.1.1.1" class="ltx_text" style="font-size:90%;">Secure aggregation</span></span>
</span>
</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 2: </span>Comparison of FL frameworks with respect to the requirements identified in Section <a href="#S2.SS1" title="2.1 Primary Requirements ‣ 2 Federated Learning for biomedical research applications: design considerations ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.1</span></a>.
The label “Only demonstrators” signifies that a feature is not built in the framework, but that a demonstrator has been provided in the form of a reference implementation, a tutorial, or documentation.</figcaption>
</figure>
<div id="S3.p2" class="ltx_para">
<p id="S3.p2.1" class="ltx_p">LabeliaLab’s SubstraFL framework is a Python library based on the Substra software developed by the company Owkin <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx46" title="" class="ltx_ref">46</a>, <a href="#bib.bibx20" title="" class="ltx_ref">20</a>]</cite>.
It is currently being used in healthcare applications for drug discovery in the context of the Mellody project <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx8" title="" class="ltx_ref">8</a>]</cite>, as well as oncology, anatomopathology and fertility in the context of the Healthchain project <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx45" title="" class="ltx_ref">45</a>, <a href="#bib.bibx43" title="" class="ltx_ref">43</a>]</cite>.
SubstraFL’s architecture is based on a fully-decentralized distributed ledger, and is designed upon the three core principles of collaboration, privacy, and traceability.
Data governance is implemented through operations on the distributed ledger, which also guarantees traceability of all ML operations within the consortium.
Operational roles are well-defined, and a permissions regime inspired by the Role-Based Access Control paradigm (RBAC) can be used to enable additional governance measures for handling remote <em id="S3.p2.1.1" class="ltx_emph ltx_font_italic">assets</em> (algorithms and data).
Furthermore, a Graphical User Interface (GUI) is provided to simplify the management of assets on nodes.
While SubstraFL has been deployed in healthcare settings, to our knowledge no tools specifically dedicated to the management of healthcare or biomedical assets are provided with the library, nor are there any tutorials or deployment examples focused on this domain.
SubstraFL’s target use-case is the execution of FL experiments at scale, and therefore the library is intended to be mainly used in production environments.
For this reason, it may not be an ideal choice for exploratory workflows requiring a high degree of interactivity during model development and training.
The distributed ledger at the core of SubstraFL’s architecture also acts as a security feature by providing a strong guarantee against the alteration of training and inference metadata, and potentially against model poisoning attacks.
No explicit description of encrypted secured communications, such as TLS, is currently provided in the framework, while differential privacy and secure aggregation approaches are not integrated in the library. Finally, governance and roadmap of SubstraFL are centralised, preventing the development of an open-source community around the project.
</p>
</div>
<div id="S3.p3" class="ltx_para">
<p id="S3.p3.1" class="ltx_p">OpenFL is another Python-based FL library, originally designed for a healthcare application but later expanded to be use case-agnostic <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx17" title="" class="ltx_ref">17</a>]</cite>.
It has been used in the context of medical applications for a global FL deployment aimed at detecting glioblastoma sub-compartment boundaries <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx47" title="" class="ltx_ref">47</a>]</cite>.
OpenFL has been designed to support multi-institutional collaborations with a strong focus on cybersecurity.
The system architecture is based on the star topology paradigm with the aggregator as the central node and collaborators as edge nodes, authenticated through PKI certificates and communicating via encrypted TLS connections.
Governance is implemented mainly through API operations or code snippets written by data owners.
For example, the code implementing a <em id="S3.p3.1.1" class="ltx_emph ltx_font_italic">ShardDescriptor</em> needs to be present on each node.
In some setups, data scientists may query centralized datasets and experiment registries holding metadata describing the whole federation.
We could not find any core functionalities in the library that are specific to the integration with biomedical data sources.
However, tutorials with a medical focus are provided in the examples section and can serve as a rough template for simple biomedical applications.
From the point of view of interactivity, OpenFL’s documentation and experiment API seems to implicitly emphasize batch execution of training experiments rather than model exploration, even though some features, such as e.g. the simple porting of model descriptions from the serial to the federated approach, do enable some degree of interactivity.
OpenFL has a strong declared focus on cybersecurity.
The usage of hardware-level features such as Trusted Execution Environments (TEE), as well as more conventional network-level measures such as TLS-encrypted communication and PKI certificates guarantee high degree of protection against attacks.
Nevertheless, fully exploiting the capabilities of OpenFL is bound to the adoption of proprietary hardware.
This condition is not necessarily compatible with practical deployment of FL in hospitals, and may critically prevent the adoption of the software in typical real-world scenarios.
</p>
</div>
<div id="S3.p4" class="ltx_para">
<p id="S3.p4.1" class="ltx_p">Flare is yet another Python FL library, developed by Nvidia <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx50" title="" class="ltx_ref">50</a>]</cite>, which has been recently used in a world-wide federation of clinical sites to develop a new model for triaging patients affected by COVID-19 <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx14" title="" class="ltx_ref">14</a>]</cite>, as well as other healthcare-related demonstrations for e.g. classification and segmentation tasks on medical images <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx49" title="" class="ltx_ref">49</a>, <a href="#bib.bibx54" title="" class="ltx_ref">54</a>]</cite>.
Flare has been designed on the principles of scalability, flexibility and lightweight, and is targeted towards cross-silo FL, not limited to healthcare, supporting both production settings as well as simulated FL for researchers.
Flare’s architecture is built on the paradigm of one <em id="S3.p4.1.1" class="ltx_emph ltx_font_italic">Controller</em> distributing <em id="S3.p4.1.2" class="ltx_emph ltx_font_italic">tasks</em> to several <em id="S3.p4.1.3" class="ltx_emph ltx_font_italic">workers</em>, thus leading to a more generic framework with respect to the other libraries analysed here.
Governance is handled through configuration files and a Python code managed manually by node administrators.
Specifications for a specific training experiment are provided partially by the data scientist (e.g. the model) and partially by the clients (e.g. the learners).
To our knowledge, no special tools are provided to node administrators for the management of data nor for fine-grained control over algorithm execution.
Moreover, while Flare’s utility has been showcased in healthcare settings, it still aims to be a generic framework, and as such we could not identify features of the library aimed specifically at the integration with biomedical data sources.
Similarly as with OpenFL, the capabilities of Flare are oriented towards the adoption of specialized hardware, as demonstrated by the focus of the project on GPU usage, scalability, and high availability, which in our experience does not correspond to a prototypical hospital deployment scenario.
Flare’s reliance on configuration files leads to a somewhat complex ecosystem that could potentially lead to a more rigid structure, lacking some of the flexibility that medical researchers may wish for during an exploratory model development phase.
From a cybersecurity point of view, Flare offers protection against man in the middle and impersonation attacks, through the use of encrypted communication, SSL authentication, and a robust provisioning workflow.
Furthermore, the concept of local <em id="S3.p4.1.4" class="ltx_emph ltx_font_italic">filters</em> enables the implementation of secure aggregation, homomorphic encryption, and differential privacy, which are however not available as part of the standard features of the framework.</p>
</div>
<div id="S3.p5" class="ltx_para">
<p id="S3.p5.1" class="ltx_p">Recently, the widely-used Flower framework <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx4" title="" class="ltx_ref">4</a>]</cite> has announced a collaboration with the Swedish decentralized AI project in order to improve the national healthcare ecosystem<span id="footnote7" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">7</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">7</sup><span class="ltx_tag ltx_tag_note">7</span><a target="_blank" href="https://flower.dev/conf/flower-summit-2022/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://flower.dev/conf/flower-summit-2022/</a></span></span></span>.
The design principles at the basis of Flower’s architecture are scalability, flexibility, and generality w.r.t ML, communication and privacy frameworks.
Given the general focus, Flower does not come pre-equipped with any data governance tools, nor does it provide any specific tools for the integration with biomedical data formats.
On the other hand, Flower’s flexible structure makes it highly amenable to highly interactive workflows, which are also further customizable thanks to the extensible <span id="S3.p5.1.1" class="ltx_text ltx_font_typewriter">Strategy</span> class.
Flower offers native support for encrypted communication through its use of gRPC <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx18" title="" class="ltx_ref">18</a>]</cite>.
Their Secure Aggregation implementation, named <span id="S3.p5.1.2" class="ltx_text ltx_font_typewriter">Salvia</span>, remains to our knowledge a proof of concept that is not yet built into the library.</p>
</div>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Methods</h2>

<section id="S4.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Design space and goals of <span id="S4.SS1.1.1" class="ltx_ERROR undefined">\fbm</span>
</h3>

<div id="S4.SS1.p1" class="ltx_para">
<p id="S4.SS1.p1.1" class="ltx_p">The main purpose of <span id="S4.SS1.p1.1.1" class="ltx_ERROR undefined">\fbm</span>is to enable seamless collaboration between medical investigators, data providers, and data scientists in a high trust, highly interactive research environment.
First and foremost, we aim at providing secure tools for the governance of personal biomedical data and the federated training of models on such data.
Additionally, we strive to make all our interfaces, especially those facing the medical researchers and data providers, easy to use even for non-technical users.
Our second goal is to make the interface for the data scientist as interactive and flexible as possible to allow fast turnaround during the development of new ML models and strategies, while respecting the privacy needs of the data providers.</p>
</div>
<div id="S4.SS1.p2" class="ltx_para">
<p id="S4.SS1.p2.1" class="ltx_p">In the context of the life cycle of a FL experiment, we make a clear-cut distinction between research applications and model deployment in production.
Both begin with a data generation and preparation step, which is typically within the scope of a specific clinical investigation.
In the case of secondary use of data, this step includes extracting data from an already existing hospital database, cleaning and wrangling the data, and applying the necessary anonymization or pseudonymization procedures.
This step is followed by a model development phase, comprising federated data preprocessing, training, validation, and hyperparameter tuning.
Sometimes the model development phase is split in two sub-phases: an initial exploration where a subset of the data is centralized, followed by the actual federated training phase.
In the development phase, emphasis is put on the scientific process of hypothesis formulation, experiment and model design, architecture and hyperparameter search, and training.
Finally, once a stable version has been reached, the model can be deployed in production mode, where emphasis is placed more on inference, reliability, robustness and high availability.
The main scope of <span id="S4.SS1.p2.1.1" class="ltx_ERROR undefined">\fbm</span>is to support the deployment and translation of AI to biomedical research and healthcare during the model development phase, but we also support deployments in production mode.
To this end, we are actively working and collaborating with experts in medical data analysis, optimization, security, databases, and visualization.
Figure <a href="#S4.F1" title="Figure 1 ‣ 4.1 Design space and goals of \fbm ‣ 4 Methods ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> shows the typical workflow of a FL experiment, with the main design space of <span id="S4.SS1.p2.1.2" class="ltx_ERROR undefined">\fbm</span>highlighted in the shaded area and the supported deployment modes highlighted in the dashed area.</p>
</div>
<figure id="S4.F1" class="ltx_figure"><img src="/html/2304.12012/assets/FL-workflow.png" id="S4.F1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="592" height="324" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>Workflow of an FL experiment from the point of view of a clinical data provider. The shaded area represents <span id="S4.F1.2.1" class="ltx_ERROR undefined">\fbm</span>’s design space in terms of functionalities and targeted usage.
</figcaption>
</figure>
<div id="S4.SS1.p3" class="ltx_para">
<p id="S4.SS1.p3.1" class="ltx_p">Our current design goals include using <span id="S4.SS1.p3.1.1" class="ltx_ERROR undefined">\fbm</span>in a high-trust environment, where all parties are honest and have an open, secure channel of communication outside of it.
We target mainly research consortia composed of a relatively small number of data providers (e.g. university hospitals or medical research centers) and data consumers (e.g. data scientists and researchers with an expertise on biomedical data).
In the future, we aim to extend our paradigm to account for less-trusted environments, such as e.g. in the presence of malicious clients.</p>
</div>
<div id="S4.SS1.p4" class="ltx_para">
<p id="S4.SS1.p4.1" class="ltx_p">Federated learning requires the availability of computing infrastructure at the edge nodes: rather than focusing on high performance computing, we target the more common scenario of clinical sites with varying computing capabilities.
We do not make the assumption that the edge nodes’ computing infrastructure is able to sustain a high availability cycle; instead we consider that the research work will be organized in periods of active testing and model training, alternating with periods of independent work where the computing infrastructure at the edge nodes may be disconnected.</p>
</div>
<div id="S4.SS1.p5" class="ltx_para">
<p id="S4.SS1.p5.1" class="ltx_p">During periods of research activity and model training, we assume that all actors involved will be available and able to communicate with low response latency.
Our researcher and medical data provider interfaces are designed with this underlying assumption, thus enabling us to provide what we deem to be a good balance between automatizing some governance processes while keeping a human-in-the-loop approach to maximize security.</p>
</div>
<div id="S4.SS1.p6" class="ltx_para">
<p id="S4.SS1.p6.1" class="ltx_p">The data scientists using <span id="S4.SS1.p6.1.1" class="ltx_ERROR undefined">\fbm</span>, called <em id="S4.SS1.p6.1.2" class="ltx_emph ltx_font_italic">researchers</em> in our notation, are assumed to be knowledgeable users of at least one of the supported ML frameworks, and to be able to read API documentation and code basic Python functions complying with it.
We also assume that they are able to discuss with biomedical researchers and data providers about technical aspects involving both the preparation of data (e.g. developing a common data model and format) and the development or interpretation of ML models for biomedical data.
Coherently with the FL paradigm, we make the assumption that the edge nodes wish to protect their data from arbitrary access of the researchers while preventing data leakages, from either a malicious source or a manipulation mistake.</p>
</div>
</section>
<section id="S4.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>Design and functional architecture</h3>

<figure id="S4.F2" class="ltx_figure"><img src="/html/2304.12012/assets/fedbiomed-pillars.png" id="S4.F2.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="592" height="218" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>High-level architecture and design pillars of <span id="S4.F2.3.1" class="ltx_ERROR undefined">\fbm</span>: node-side governance and control, interoperability with medical data standards and infrastructure, researcher interactivity, and data privacy and security. <span id="S4.F2.4.2" class="ltx_ERROR undefined">\fbm</span>is composed of three components: the researcher, a data scientist responsible for designing and steering the training of the ML model; the nodes, i.e. the clinical data providers; and the network, responsible for brokering all communication between the researcher and nodes. Each component has been designed following the Fed-BioMed requirements (Section 2), and the figure highlights which requirement affects the architectural subcomponents.</figcaption>
</figure>
<div id="S4.SS2.p1" class="ltx_para">
<p id="S4.SS2.p1.1" class="ltx_p">The functional architecture of <span id="S4.SS2.p1.1.1" class="ltx_ERROR undefined">\fbm</span>, as shown in Figure <a href="#S4.F2" title="Figure 2 ‣ 4.2 Design and functional architecture ‣ 4 Methods ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>, is based on the design space described in Section <a href="#S4.SS1" title="4.1 Design space and goals of \fbm ‣ 4 Methods ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4.1</span></a> and the requirements identified in Section <a href="#S2" title="2 Federated Learning for biomedical research applications: design considerations ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
The <span id="S4.SS2.p1.1.2" class="ltx_ERROR undefined">\fbm</span>ecosystem comprises three architectural components: the network, the researcher, and one or more nodes.
The network is responsible for brokering the communication between all <span id="S4.SS2.p1.1.3" class="ltx_ERROR undefined">\fbm</span>components, the researcher is responsible for configuring and driving the federated learning experiment, as well as aggregating the trained models, while the nodes are responsible for local data governance and actually performing the training.</p>
</div>
<div id="S4.SS2.p2" class="ltx_para">
<p id="S4.SS2.p2.1" class="ltx_p">A standard training experiment is described in Figure <a href="#S4.F3" title="Figure 3 ‣ 4.2 Design and functional architecture ‣ 4 Methods ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, where nodes are first required to make their data available for training by inserting an appropriate metadata entry in a locally-stored database, and assigning unique identifying tags to those data.
This process is simplified by the web-GUI built-in to our framework, but a CLI is also available for programmatic approaches.
Optionally, node-specific customizations to the data loading process may be specified here through a plugin system called <span id="S4.SS2.p2.1.1" class="ltx_text ltx_font_typewriter">DataLoadingPlan</span>, with the intention of reducing the data formatting burden by providing a logical layer between the researcher and the actual data format as stored locally.
Then researchers define a <span id="S4.SS2.p2.1.2" class="ltx_text ltx_font_typewriter">TrainingPlan</span>, i.e. an object containing the description of the ML model and aggregation parameters, the code for the training and validation routines, a data loading and preprocessing routine, and other training-specific information such as e.g. the description of the optimizer.
The <span id="S4.SS2.p2.1.3" class="ltx_text ltx_font_typewriter">TrainingPlan</span> is packaged in an <span id="S4.SS2.p2.1.4" class="ltx_text ltx_font_typewriter">Experiment</span> object along with the aggregator, training arguments, and data identifier tags.</p>
</div>
<div id="S4.SS2.p3" class="ltx_para">
<p id="S4.SS2.p3.1" class="ltx_p">The researcher then issues a train command through the network’s message broker, which is broadcasted to all the nodes.
Nodes that identify the requested data tags within their metadata database may begin training: first the <span id="S4.SS2.p3.1.1" class="ltx_text ltx_font_typewriter">TrainingPlan</span> object is recreated on the node, then data are loaded, preprocessed using both node-specific customizations as well as researcher-specified functions, and finally the training routine of the training plan is executed.
For convenience, <span id="S4.SS2.p3.1.2" class="ltx_ERROR undefined">\fbm</span>comes with pre-packaged <span id="S4.SS2.p3.1.3" class="ltx_text ltx_font_typewriter">TrainingPlans</span> for widely-used frameworks, thus requiring minimal configuration from researchers.</p>
</div>
<figure id="S4.F3" class="ltx_figure"><img src="/html/2304.12012/assets/experiment-loop-v2.png" id="S4.F3.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="333" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>Prototypical FL training workflow in <span id="S4.F3.2.1" class="ltx_ERROR undefined">\fbm</span>. Icons indicate the steps where the design was influenced by a particular requirement. First the nodes mark their dataset as available for federated training, while the researcher defines and obtains approval for a TrainingPlan. Then the researcher may launch multiple experiments, and within each experiment interactively launch multiple rounds of training. In each round, data are loaded locally on the nodes and the researcher-defined training routine is executed. Communication of model parameters and metadata always happens through the Network component.</figcaption>
</figure>
<section id="S4.SS2.SSSx1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">Node-side governance</h4>

<div id="S4.SS2.SSSx1.p1" class="ltx_para">
<span id="S4.SS2.SSSx1.p1.1" class="ltx_ERROR undefined">\fbm</span>
<p id="S4.SS2.SSSx1.p1.2" class="ltx_p">empowers nodes by maximising the amount of control they have over the execution of a FL experiment.
In <span id="S4.SS2.SSSx1.p1.2.1" class="ltx_ERROR undefined">\fbm</span>, there is no notion of a trusted third party to which nodes must delegate full trust.
For training, we make the assumption that datasets have already been treated to comply to existing data privacy regulations (e.g. pseudonymisation), and that they have been at least partially harmonised, both semantically and syntactically, to a pre-defined format common to all the nodes participating in the experiment.</p>
</div>
<div id="S4.SS2.SSSx1.p2" class="ltx_para">
<p id="S4.SS2.SSSx1.p2.1" class="ltx_p">When an experiment is launched, the datasets corresponding to the requested tags are loaded in the node’s worker instance and prepared for training.
A researcher-defined <span id="S4.SS2.SSSx1.p2.1.1" class="ltx_text ltx_font_typewriter">training_data</span> function is then used to load the data and apply local pre-processing.
Although this is not yet strictly enforced, this function is expected to use one of the dataset classes defined by <span id="S4.SS2.SSSx1.p2.1.2" class="ltx_ERROR undefined">\fbm</span>providing controlled access to filesystem resources and best compatibility with the rest of the <span id="S4.SS2.SSSx1.p2.1.3" class="ltx_ERROR undefined">\fbm</span>workflow.
It is also possible to emulate a federated data pre-processing pipeline by assembling multiple experiments, thanks to <span id="S4.SS2.SSSx1.p2.1.4" class="ltx_ERROR undefined">\fbm</span>’s interactive approach.</p>
</div>
<div id="S4.SS2.SSSx1.p3" class="ltx_para">
<span id="S4.SS2.SSSx1.p3.1" class="ltx_ERROR undefined">\fbm</span>
<p id="S4.SS2.SSSx1.p3.2" class="ltx_p">’s answer to the obvious tradeoff between allowing researchers to execute custom code snippets on the worker nodes and the associated security issues, is to provide a mechanism called <em id="S4.SS2.SSSx1.p3.2.1" class="ltx_emph ltx_font_italic">training plan approval</em>.
When enabled, this feature prevents the execution of code contained in <span id="S4.SS2.SSSx1.p3.2.2" class="ltx_text ltx_font_typewriter">TrainingPlans</span> that has not yet been inspected and approved by the node.
A hash of the code is computed and checked at every training execution to prevent substitution attacks, and while this feature is not a definitive fail-safe measure against malicious users, it provides an additional layer of protection, as it empowers nodes and fosters researcher-node collaboration through shared responsibility.
Furthermore, nodes are granted the right to override certain training parameters, regardless of the researcher’s original request, for matters concerning security and resource usage.</p>
</div>
</section>
<section id="S4.SS2.SSSx2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">Integration with biomedical data sources</h4>

<div id="S4.SS2.SSSx2.p1" class="ltx_para">
<p id="S4.SS2.SSSx2.p1.1" class="ltx_p">Like most FL frameworks, <span id="S4.SS2.SSSx2.p1.1.1" class="ltx_ERROR undefined">\fbm</span>does not offer a direct connection with raw data sources such as hospital EHR, PACS, and other clinical IT systems.
Instead, clinical data managers are requested to perform a one-time data preparation task to extract and wrangle the data.
Contrary to other frameworks, <span id="S4.SS2.SSSx2.p1.1.2" class="ltx_ERROR undefined">\fbm</span>tries to minimize this effort by implementing dedicated mechanisms and offering built-in integration with widely-used data standards, at the cost of losing some generality by restricting its focus on biomedical data.</p>
</div>
<div id="S4.SS2.SSSx2.p2" class="ltx_para">
<span id="S4.SS2.SSSx2.p2.1" class="ltx_ERROR undefined">\fbm</span>
<p id="S4.SS2.SSSx2.p2.2" class="ltx_p">introduces the notion of a <span id="S4.SS2.SSSx2.p2.2.1" class="ltx_text ltx_font_typewriter">DataLoadingPlan</span>, meaning a set of customizations configured by the node which allow changing the way data is presented to the researcher.
Built-in <span id="S4.SS2.SSSx2.p2.2.2" class="ltx_text ltx_font_typewriter">DataLoadingPlans</span> are already integrated in <span id="S4.SS2.SSSx2.p2.2.3" class="ltx_ERROR undefined">\fbm</span>’s GUI, thus providing a mechanism for data harmonization that does not require extensive effort on the node side.</p>
</div>
<div id="S4.SS2.SSSx2.p3" class="ltx_para">
<span id="S4.SS2.SSSx2.p3.1" class="ltx_ERROR undefined">\fbm</span>
<p id="S4.SS2.SSSx2.p3.2" class="ltx_p">also provides a built-in suite of dataset classes that provide an interface to common standards.
For example, our <span id="S4.SS2.SSSx2.p3.2.1" class="ltx_text ltx_font_typewriter">MedicalFolderDataset</span> class is inspired by the BIDS standard<span id="footnote8" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">8</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">8</sup><span class="ltx_tag ltx_tag_note">8</span><a target="_blank" href="https://bids.neuroimaging.io/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://bids.neuroimaging.io/</a></span></span></span> as well as PyTorch’s <span id="S4.SS2.SSSx2.p3.2.2" class="ltx_text ltx_font_typewriter">ImageFolder</span>.
Furthermore, our generic <span id="S4.SS2.SSSx2.p3.2.3" class="ltx_text ltx_font_typewriter">TabularDataset</span> class supports any standard that may be reduced to csv format.
As part of our continuous development effort, we plan to significantly expand this suite to include several other healthcare interoperability standards, depending on the needs of our collaborators.</p>
</div>
<div id="S4.SS2.SSSx2.p4" class="ltx_para">
<p id="S4.SS2.SSSx2.p4.1" class="ltx_p">Hospitals participating in the same FL experiment may have different computing infrastructures.
<span id="S4.SS2.SSSx2.p4.1.1" class="ltx_ERROR undefined">\fbm</span>supports execution on containers, virtual machines, bare-metal CPUs and GPUs in the same experiment, allowing for the reconciliation of an heterogeneous computing infrastructure across nodes.</p>
</div>
</section>
<section id="S4.SS2.SSSx3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">Researcher interactivity</h4>

<div id="S4.SS2.SSSx3.p1" class="ltx_para">
<p id="S4.SS2.SSSx3.p1.1" class="ltx_p">The researcher-facing side of <span id="S4.SS2.SSSx3.p1.1.1" class="ltx_ERROR undefined">\fbm</span>is a Python SDK for configuring and managing FL experiments, preferably via jupyter notebooks.
To create an experiment, researchers first instantiate a class called <span id="S4.SS2.SSSx3.p1.1.2" class="ltx_text ltx_font_typewriter">TrainingPlan</span> containing the definition of the federated data and the training routine.
The code written by data scientists in the <span id="S4.SS2.SSSx3.p1.1.3" class="ltx_text ltx_font_typewriter">TrainingPlan</span> is designed to be as similar as possible to the serial local version of the training loop.
This interface is intentionally generic to support a wide variety of use cases, from workflows allowing to compute federated summary statistics to general distributed optimization based for example on Stochastic Gradient Descent (SGD), Expectation Maximization (EM), and Variational Inference (VI).
For convenience, we offer specialized <span id="S4.SS2.SSSx3.p1.1.4" class="ltx_text ltx_font_typewriter">TrainingPlans</span> for specific optimization strategies such as SGD, and for the ML framework being used.</p>
</div>
<div id="S4.SS2.SSSx3.p2" class="ltx_para">
<p id="S4.SS2.SSSx3.p2.1" class="ltx_p">Our approach promotes researcher interactivity via the <span id="S4.SS2.SSSx3.p2.1.1" class="ltx_text ltx_font_typewriter">Experiment</span> class, which allows to easily set the participating nodes, the FL strategy, and includes the <span id="S4.SS2.SSSx3.p2.1.2" class="ltx_text ltx_font_typewriter">TrainingPlan</span> itself.
The <span id="S4.SS2.SSSx3.p2.1.3" class="ltx_text ltx_font_typewriter">Experiment</span> is provided with a logging functionality and integration with tensorboard <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx38" title="" class="ltx_ref">38</a>]</cite>.
The modular design of the <span id="S4.SS2.SSSx3.p2.1.4" class="ltx_ERROR undefined">\fbm</span>training loop allows researchers to dynamically adjust hyper-parameters on the fly.
Furthermore, a checkpointing system allows saving and loading the state of an experiment in persistent memory.</p>
</div>
<div id="S4.SS2.SSSx3.p3" class="ltx_para">
<p id="S4.SS2.SSSx3.p3.1" class="ltx_p">In a highly secure environment where training plans must be approved by each node, minor changes in a training plan may lead to insufferable delays for obtaining multiple approvals.
Therefore, we make a distinction between the <span id="S4.SS2.SSSx3.p3.1.1" class="ltx_text ltx_font_typewriter">TrainingPlan</span> source code and a set of training and model arguments.
The former must be approved by the node, but it may leverage the latter as a way to dynamically configure details about the experiment that lie within the node’s acceptable ranges.
For example, a <span id="S4.SS2.SSSx3.p3.1.2" class="ltx_text ltx_font_typewriter">TrainingPlan</span> may include a dropout layer but the dropout rate would be specified as a model argument, thus providing security guarantees to the node while allowing flexibility for the researcher.</p>
</div>
</section>
<section id="S4.SS2.SSSx4" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">Cybersecurity</h4>

<div id="S4.SS2.SSSx4.p1" class="ltx_para">
<span id="S4.SS2.SSSx4.p1.1" class="ltx_ERROR undefined">\fbm</span>
<p id="S4.SS2.SSSx4.p1.2" class="ltx_p">’s current threat model assumes that nodes and server are honest-but-curious, nodes are independent actors and do not collude, and researchers may be malicious.
This set of assumptions corresponds to our current application scenarios in hospital networks, where clients are trusted and strong protection is required with respect to researcher’s manipulations.
Thus, the first measure that we put in place is deploying the whole <span id="S4.SS2.SSSx4.p1.2.1" class="ltx_ERROR undefined">\fbm</span>ecosystem inside a VPN, to effectively isolate the execution and protect it against external attacks.
In the honest-but-curious model, an entity that gains access to the model weights may still attempt to re-identify individual samples through model inversion attacks.
<span id="S4.SS2.SSSx4.p1.2.2" class="ltx_ERROR undefined">\fbm</span>implements both differential privacy as well as secure aggregation based on additively homomorphic encryption <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx27" title="" class="ltx_ref">27</a>, <a href="#bib.bibx36" title="" class="ltx_ref">36</a>]</cite>, with currently some limitations concerning the supported ML frameworks and aggregation methods.</p>
</div>
<div id="S4.SS2.SSSx4.p2" class="ltx_para">
<span id="S4.SS2.SSSx4.p2.1" class="ltx_ERROR undefined">\fbm</span>
<p id="S4.SS2.SSSx4.p2.2" class="ltx_p">’s architecture offers one final layer of protection, by insulating the researcher from the nodes through the existence of a middle component, the network, that brokers all communication between them.
Furthermore, our <span id="S4.SS2.SSSx4.p2.2.1" class="ltx_text ltx_font_typewriter">TrainingPlan</span> approval mechanism can also be viewed as a security feature, by allowing nodes to review any code intended to be executed within their systems.</p>
</div>
<div id="S4.SS2.SSSx4.p3" class="ltx_para">
<p id="S4.SS2.SSSx4.p3.1" class="ltx_p">In our short-term roadmap we plan to strengthen our security features by adopting a tighter threat model.
The first measure that we intend to introduce is encrypted communication inside the VPN, preventing potentially malicious actors who have gained internal access from being able to listen-in on the communication exchanges.
Secondly, we intend to introduce trusted digital certificates to improve our protection against impersonation attacks.</p>
</div>
</section>
</section>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Status report</h2>

<section id="S5.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.1 </span>Current state of the implementation and future plans</h3>

<div id="S5.SS1.p1" class="ltx_para">
<span id="S5.SS1.p1.1" class="ltx_ERROR undefined">\fbm</span>
<p id="S5.SS1.p1.2" class="ltx_p">is a constantly evolving project, with an active group of core maintainers striving to issue monthly releases.
The current maturity of our library has been acknowledged to allow a first real-world deployment and validation within a federation of some members from the UniCancer consortium, as described in Section <a href="#S5.SS2" title="5.2 Demonstration on real-world federated hospitals networks ‣ 5 Status report ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5.2</span></a>, thus characterizing its Technology Readiness Level as TLR 5<span id="footnote9" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">9</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">9</sup><span class="ltx_tag ltx_tag_note">9</span><a target="_blank" href="https://ec.europa.eu/research/participants/data/ref/h2020/wp/2014_2015/annexes/h2020-wp1415-annex-g-trl_en.pdf" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://ec.europa.eu/research/participants/data/ref/h2020/wp/2014_2015/annexes/h2020-wp1415-annex-g-trl_en.pdf</a></span></span></span>.
However, we strive to constantly improve several aspects of the implementation in future releases, following a development roadmap inspired by the design pillars described in this paper and actively driven by the needs of our growing community of users.
Building on our functional description in Section <a href="#S4.SS2" title="4.2 Design and functional architecture ‣ 4 Methods ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4.2</span></a>, we provide in the Supplementary Information <a href="#S8.SS2" title="8.2 Technical implementation details ‣ 8 Supplementary Information ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8.2</span></a> a fine-grained description of the core library, interface and cybersecurity functionalities currently implemented in <span id="S5.SS1.p1.2.1" class="ltx_ERROR undefined">\fbm</span>.</p>
</div>
</section>
<section id="S5.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.2 </span>Demonstration on real-world federated hospitals networks</h3>

<div id="S5.SS2.p1" class="ltx_para">
<p id="S5.SS2.p1.1" class="ltx_p">To illustrate the capabilities of Fed-BioMed for real world applications of federated learning in hospitals networks, in this section we report the results obtained from the end-to-end deployment and training of a federated prostate segmentation model, based on the data hosted by three different medical institutes from the Unicancer consortium <span id="footnote10" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">10</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">10</sup><span class="ltx_tag ltx_tag_note">10</span><a target="_blank" href="https://www.unicancer.fr/en/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.unicancer.fr/en/</a></span></span></span>.</p>
</div>
<div id="S5.SS2.p2" class="ltx_para">
<p id="S5.SS2.p2.1" class="ltx_p">The French hospitals involved are the <em id="S5.SS2.p2.1.1" class="ltx_emph ltx_font_italic">Centre Henri Becquerel</em>, Rouen, the <em id="S5.SS2.p2.1.2" class="ltx_emph ltx_font_italic">Institut Curie</em>, Orsay, and the <em id="S5.SS2.p2.1.3" class="ltx_emph ltx_font_italic">Centre Antoine Lacassagne</em>, Nice. Each hospital loaded into the Fed-BioMed client the respective dataset composed by prostate magnetic resonance images (MRIs), and associated prostate segmentation masks. The dataset considered for this test were the following:</p>
</div>
<div id="S5.SS2.p3" class="ltx_para">
<ul id="S5.I1" class="ltx_itemize">
<li id="S5.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S5.I1.i1.p1" class="ltx_para">
<p id="S5.I1.i1.p1.1" class="ltx_p"><span id="S5.I1.i1.p1.1.1" class="ltx_text ltx_font_bold">Medical Segmentation Decathlon - Prostate</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx1" title="" class="ltx_ref">1</a>]</cite>, composed by 32 prostate MRIs for training, with respective masks of peripheral and transition zone, merged into a single mask for the whole prostate. This dataset was hosted at <em id="S5.I1.i1.p1.1.2" class="ltx_emph ltx_font_italic">Institut Curie</em> (CURIE).</p>
</div>
</li>
<li id="S5.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S5.I1.i2.p1" class="ltx_para">
<p id="S5.I1.i2.p1.1" class="ltx_p"><span id="S5.I1.i2.p1.1.1" class="ltx_text ltx_font_bold">Promise12</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx32" title="" class="ltx_ref">32</a>]</cite>, consisting of 50 training cases obtained with different scanners. Of those, 27 cases were acquired without using an endorectal coil. This dataset was hosted at <em id="S5.I1.i2.p1.1.2" class="ltx_emph ltx_font_italic">Centre Henri Becquerel</em> (CHB).</p>
</div>
</li>
<li id="S5.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span> 
<div id="S5.I1.i3.p1" class="ltx_para">
<p id="S5.I1.i3.p1.1" class="ltx_p"><span id="S5.I1.i3.p1.1.1" class="ltx_text ltx_font_bold">ProstateX</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx2" title="" class="ltx_ref">2</a>]</cite>, containing prostate MRIs acquired by using two different scanners, providing prostate segmentation masks for 189 cases <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx11" title="" class="ltx_ref">11</a>]</cite>. This dataset was hosted at <em id="S5.I1.i3.p1.1.2" class="ltx_emph ltx_font_italic">Centre Antoine Lacassagne</em> (CAL).</p>
</div>
</li>
</ul>
</div>
<div id="S5.SS2.p4" class="ltx_para">
<p id="S5.SS2.p4.1" class="ltx_p">Each of the three hospitals was assigned a single dataset from the ones listed above, which was subsequently loaded into the Fed-BioMed node client.
At each site, the data was further divided in a training subset (90% of samples) and a holdout subset (10% of samples).
The resulting federated learning setup reflects a realistic scenario presenting the typical data heterogeneity of multi-centric medical imaging applications (see Figure <a href="#S5.F4.sf1" title="In Figure 4 ‣ 5.2.3 FL runtime overhead ‣ 5.2 Demonstration on real-world federated hospitals networks ‣ 5 Status report ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4a</span></a> and Appendix <a href="#S8.SS1" title="8.1 Real-world use case scenario: prostate segmentation ‣ 8 Supplementary Information ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8.1</span></a> for more details).
The use of publicly available data is in the scope of this experiment, and is aimed at demonstrating the FL task on a reproducible benchmark, allowing the comparison of the results with respect to the centralized scenario.
Moreover, this setup facilitated the ethical approval of this experiment by each clinical center, avoiding the cumbersome process related to the use of hospital data.</p>
</div>
<section id="S5.SS2.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.2.1 </span>Experimental details</h4>

<div id="S5.SS2.SSS1.p1" class="ltx_para">
<p id="S5.SS2.SSS1.p1.1" class="ltx_p">Within this setup, training hyperparameters were previously identified via a simulated FL scenario on the same data.
The chosen aggregation method used was FedAvg <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx39" title="" class="ltx_ref">39</a>]</cite>, and the <span id="S5.SS2.SSS1.p1.1.1" class="ltx_text ltx_font_typewriter">TrainingPlanApproval</span> security feature was enabled.
On the hospitals node side, the Fed-BioMed nodes were running on GPU-enabled machines, even though the existence of specific hardware is not a requirement for running Fed-BioMed.
The central aggregator and the network component were hosted on a separate server provided by Inria.
The training experiment consisted of 40 rounds with 25 local gradient updates each.
Additional details are provided in the Supplementary Material <a href="#S8.SS1" title="8.1 Real-world use case scenario: prostate segmentation ‣ 8 Supplementary Information ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8.1</span></a>.</p>
</div>
</section>
<section id="S5.SS2.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.2.2 </span>FL does not affect final model’s performance</h4>

<div id="S5.SS2.SSS2.p1" class="ltx_para">
<p id="S5.SS2.SSS2.p1.5" class="ltx_p">After training, the model obtained a Dice score of <math id="S5.SS2.SSS2.p1.1.m1.1" class="ltx_Math" alttext="0.868" display="inline"><semantics id="S5.SS2.SSS2.p1.1.m1.1a"><mn id="S5.SS2.SSS2.p1.1.m1.1.1" xref="S5.SS2.SSS2.p1.1.m1.1.1.cmml">0.868</mn><annotation-xml encoding="MathML-Content" id="S5.SS2.SSS2.p1.1.m1.1b"><cn type="float" id="S5.SS2.SSS2.p1.1.m1.1.1.cmml" xref="S5.SS2.SSS2.p1.1.m1.1.1">0.868</cn></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.SSS2.p1.1.m1.1c">0.868</annotation></semantics></math>.
To assess the generalizability of this result, we performed a 5-fold cross-validation with similar 90-10 splits in both a centralized and a simulated federated setting.
The cross-validation Dice score for the simulated FL model was <math id="S5.SS2.SSS2.p1.2.m2.1" class="ltx_Math" alttext="0.854\pm 0.028" display="inline"><semantics id="S5.SS2.SSS2.p1.2.m2.1a"><mrow id="S5.SS2.SSS2.p1.2.m2.1.1" xref="S5.SS2.SSS2.p1.2.m2.1.1.cmml"><mn id="S5.SS2.SSS2.p1.2.m2.1.1.2" xref="S5.SS2.SSS2.p1.2.m2.1.1.2.cmml">0.854</mn><mo id="S5.SS2.SSS2.p1.2.m2.1.1.1" xref="S5.SS2.SSS2.p1.2.m2.1.1.1.cmml">±</mo><mn id="S5.SS2.SSS2.p1.2.m2.1.1.3" xref="S5.SS2.SSS2.p1.2.m2.1.1.3.cmml">0.028</mn></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.SSS2.p1.2.m2.1b"><apply id="S5.SS2.SSS2.p1.2.m2.1.1.cmml" xref="S5.SS2.SSS2.p1.2.m2.1.1"><csymbol cd="latexml" id="S5.SS2.SSS2.p1.2.m2.1.1.1.cmml" xref="S5.SS2.SSS2.p1.2.m2.1.1.1">plus-or-minus</csymbol><cn type="float" id="S5.SS2.SSS2.p1.2.m2.1.1.2.cmml" xref="S5.SS2.SSS2.p1.2.m2.1.1.2">0.854</cn><cn type="float" id="S5.SS2.SSS2.p1.2.m2.1.1.3.cmml" xref="S5.SS2.SSS2.p1.2.m2.1.1.3">0.028</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.SSS2.p1.2.m2.1c">0.854\pm 0.028</annotation></semantics></math>, while for the centralized (CL) model it was <math id="S5.SS2.SSS2.p1.3.m3.1" class="ltx_Math" alttext="0.850\pm 0.035" display="inline"><semantics id="S5.SS2.SSS2.p1.3.m3.1a"><mrow id="S5.SS2.SSS2.p1.3.m3.1.1" xref="S5.SS2.SSS2.p1.3.m3.1.1.cmml"><mn id="S5.SS2.SSS2.p1.3.m3.1.1.2" xref="S5.SS2.SSS2.p1.3.m3.1.1.2.cmml">0.850</mn><mo id="S5.SS2.SSS2.p1.3.m3.1.1.1" xref="S5.SS2.SSS2.p1.3.m3.1.1.1.cmml">±</mo><mn id="S5.SS2.SSS2.p1.3.m3.1.1.3" xref="S5.SS2.SSS2.p1.3.m3.1.1.3.cmml">0.035</mn></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.SSS2.p1.3.m3.1b"><apply id="S5.SS2.SSS2.p1.3.m3.1.1.cmml" xref="S5.SS2.SSS2.p1.3.m3.1.1"><csymbol cd="latexml" id="S5.SS2.SSS2.p1.3.m3.1.1.1.cmml" xref="S5.SS2.SSS2.p1.3.m3.1.1.1">plus-or-minus</csymbol><cn type="float" id="S5.SS2.SSS2.p1.3.m3.1.1.2.cmml" xref="S5.SS2.SSS2.p1.3.m3.1.1.2">0.850</cn><cn type="float" id="S5.SS2.SSS2.p1.3.m3.1.1.3.cmml" xref="S5.SS2.SSS2.p1.3.m3.1.1.3">0.035</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.SSS2.p1.3.m3.1c">0.850\pm 0.035</annotation></semantics></math>.
The difference in cross-validation scores between the simulated FL and CL models is not statistically significant (<math id="S5.SS2.SSS2.p1.4.m4.1" class="ltx_Math" alttext="p=0.63" display="inline"><semantics id="S5.SS2.SSS2.p1.4.m4.1a"><mrow id="S5.SS2.SSS2.p1.4.m4.1.1" xref="S5.SS2.SSS2.p1.4.m4.1.1.cmml"><mi id="S5.SS2.SSS2.p1.4.m4.1.1.2" xref="S5.SS2.SSS2.p1.4.m4.1.1.2.cmml">p</mi><mo id="S5.SS2.SSS2.p1.4.m4.1.1.1" xref="S5.SS2.SSS2.p1.4.m4.1.1.1.cmml">=</mo><mn id="S5.SS2.SSS2.p1.4.m4.1.1.3" xref="S5.SS2.SSS2.p1.4.m4.1.1.3.cmml">0.63</mn></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.SSS2.p1.4.m4.1b"><apply id="S5.SS2.SSS2.p1.4.m4.1.1.cmml" xref="S5.SS2.SSS2.p1.4.m4.1.1"><eq id="S5.SS2.SSS2.p1.4.m4.1.1.1.cmml" xref="S5.SS2.SSS2.p1.4.m4.1.1.1"></eq><ci id="S5.SS2.SSS2.p1.4.m4.1.1.2.cmml" xref="S5.SS2.SSS2.p1.4.m4.1.1.2">𝑝</ci><cn type="float" id="S5.SS2.SSS2.p1.4.m4.1.1.3.cmml" xref="S5.SS2.SSS2.p1.4.m4.1.1.3">0.63</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.SSS2.p1.4.m4.1c">p=0.63</annotation></semantics></math>).
Furthermore, the original model trained in the real-world setting has a realistic performance that falls within <math id="S5.SS2.SSS2.p1.5.m5.1" class="ltx_Math" alttext="0.5\sigma" display="inline"><semantics id="S5.SS2.SSS2.p1.5.m5.1a"><mrow id="S5.SS2.SSS2.p1.5.m5.1.1" xref="S5.SS2.SSS2.p1.5.m5.1.1.cmml"><mn id="S5.SS2.SSS2.p1.5.m5.1.1.2" xref="S5.SS2.SSS2.p1.5.m5.1.1.2.cmml">0.5</mn><mo lspace="0em" rspace="0em" id="S5.SS2.SSS2.p1.5.m5.1.1.1" xref="S5.SS2.SSS2.p1.5.m5.1.1.1.cmml">​</mo><mi id="S5.SS2.SSS2.p1.5.m5.1.1.3" xref="S5.SS2.SSS2.p1.5.m5.1.1.3.cmml">σ</mi></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.SSS2.p1.5.m5.1b"><apply id="S5.SS2.SSS2.p1.5.m5.1.1.cmml" xref="S5.SS2.SSS2.p1.5.m5.1.1"><times id="S5.SS2.SSS2.p1.5.m5.1.1.1.cmml" xref="S5.SS2.SSS2.p1.5.m5.1.1.1"></times><cn type="float" id="S5.SS2.SSS2.p1.5.m5.1.1.2.cmml" xref="S5.SS2.SSS2.p1.5.m5.1.1.2">0.5</cn><ci id="S5.SS2.SSS2.p1.5.m5.1.1.3.cmml" xref="S5.SS2.SSS2.p1.5.m5.1.1.3">𝜎</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.SSS2.p1.5.m5.1c">0.5\sigma</annotation></semantics></math> of the average of the simulated FL cross-validation scores.
The distribution of Dice scores for all cross-validation folds combined, shown in Figure <a href="#S5.F4.sf3" title="In Figure 4 ‣ 5.2.3 FL runtime overhead ‣ 5.2 Demonstration on real-world federated hospitals networks ‣ 5 Status report ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4c</span></a>, are qualitatively similar between the CL and FL models.
The final segmentations are also visually indistinguishable, as shown in the bottom row of Figure <a href="#S5.F4.sf4" title="In Figure 4 ‣ 5.2.3 FL runtime overhead ‣ 5.2 Demonstration on real-world federated hospitals networks ‣ 5 Status report ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4d</span></a>.</p>
</div>
</section>
<section id="S5.SS2.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.2.3 </span>FL runtime overhead</h4>

<div id="S5.SS2.SSS3.p1" class="ltx_para">
<p id="S5.SS2.SSS3.p1.1" class="ltx_p">Each round required on average just over 60 seconds of wallclock time for completion.
Federated Learning was found to induce a significant overhead on training time, ranging from 39% to 56% of the overall experiment wallclock time, as shown in Figure <a href="#S5.F4.sf2" title="In Figure 4 ‣ 5.2.3 FL runtime overhead ‣ 5.2 Demonstration on real-world federated hospitals networks ‣ 5 Status report ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4b</span></a>.
This is in contrast with other studies that report negligible overhead <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx62" title="" class="ltx_ref">62</a>, <a href="#bib.bibx53" title="" class="ltx_ref">53</a>]</cite>, and can be explained by the smaller dataset sizes in our experiment, associated with a smaller number of samples used for training within each round.
We believe that our experiment represents a realistic common scenario for most real-world hospital deployments where data may be scarce and high performance computing not applicable.
Some <span id="S5.SS2.SSS3.p1.1.1" class="ltx_ERROR undefined">\fbm</span>implementation details, such as an hard-coded delay at round initialization, may also impact this measure and we are investigating whether useful runtime gains could be made through software improvements.</p>
</div>
<figure id="S5.F4" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S5.F4.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2304.12012/assets/histogram-pixel-intensity.png" id="S5.F4.sf1.g1" class="ltx_graphics ltx_img_landscape" width="296" height="234" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span></figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S5.F4.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2304.12012/assets/FL-overhead-breakdown.png" id="S5.F4.sf2.g1" class="ltx_graphics ltx_img_landscape" width="296" height="235" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span></figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S5.F4.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2304.12012/assets/dsc-score-compare-centralized.png" id="S5.F4.sf3.g1" class="ltx_graphics ltx_img_square" width="296" height="284" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(c) </span></figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S5.F4.sf4" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2304.12012/assets/example-segmentatin.png" id="S5.F4.sf4.g1" class="ltx_graphics ltx_img_square" width="296" height="310" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(d) </span></figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 4: </span><span id="S5.F4.5.1" class="ltx_text ltx_font_bold">a)</span> Pixel intensity distribution grouped by clinical site.
Prostate images exhibit significant differences between sites, especially in the case of Site 2.
<span id="S5.F4.6.2" class="ltx_text ltx_font_bold">b)</span> Breakdown of FL experiment wallclock runtime.
FL introduces a significant overhead in our case, likely attributable to the relatively small number of samples seen by the model in each FL round.
<span id="S5.F4.7.3" class="ltx_text ltx_font_bold">c)</span> Distribution of Dice scores for centralized (CL) and federated (FL) models, combining all cross-validation folds.
The performance of the two models has a statistically significant difference, but with a small effect size.
The figure shows boxplots inside the violin plots.
The top bottom of each boxplot depict the 3rd and 1st quartile of each measure.
The white line and the red x indicate the median and mean values, respectively.
The whiskers depict the extremal observations still within 1.5 times the interquartile range.
<span id="S5.F4.8.4" class="ltx_text ltx_font_bold">d)</span> Clockwise from top left: an example raw image, the ground truth segmentation, the FL model prediction and the CL model prediction.</figcaption>
</figure>
</section>
</section>
</section>
<section id="S6" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Lessons learned</h2>

<section id="S6.SS0.SSS0.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_title_paragraph">Answering to the specific needs of biomedical research applications</h5>

<div id="S6.SS0.SSS0.Px1.p1" class="ltx_para">
<p id="S6.SS0.SSS0.Px1.p1.1" class="ltx_p">The field of biomedical research is characterized by specific needs that do not necessarily generalize to other fields of application of FL, such as tighter regulations for data protection, very large degrees of heterogeneity, and others identified in Section <a href="#S2.SS1" title="2.1 Primary Requirements ‣ 2 Federated Learning for biomedical research applications: design considerations ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.1</span></a>.
FL frameworks, on the other hand, are often developed with a generic approach in mind, striving for generality in terms of application domains.
While this approach ensures that such frameworks can be more widely used, this may lead to a sub-optimal experience for users who are interested only in a specific application domain.
We believe that some communities, and in particular the biomedical research field, may greatly benefit from personalized framework development approaches able to cater to specific requirements such as providing the necessary governance tools, ensuring their usability by members of the application domain’s community, and providing a software architecture tailored to the privacy or performance needs.</p>
</div>
</section>
<section id="S6.SS0.SSS0.Px2" class="ltx_paragraph">
<h5 class="ltx_title ltx_title_paragraph">Model development and debugging in a federated setting</h5>

<div id="S6.SS0.SSS0.Px2.p1" class="ltx_para">
<p id="S6.SS0.SSS0.Px2.p1.1" class="ltx_p">The development of a new ML model almost always requires an initial exploratory phase where data are examined, data preprocessing pipelines are designed, the model architecture is refined, and finally hyperparameters are tuned.
Most of the strategies for such operations usually involve some data manipulation and observation, for example it may be insightful to examine the data samples with the highest losses for a given model.
In a federated setting, all of this is not possible because data are not allowed to travel outside of the source institution.
This leads to a mismatch between who has the right to examine the data — i.e. the clinical data sources —, and who has the knowledge to interpret it — i.e. the data scientists —.
<span id="S6.SS0.SSS0.Px2.p1.1.1" class="ltx_ERROR undefined">\fbm</span>is designed to operate in high-trust environments where the communication between these two entities is encouraged, therefore we may imagine a scenario where the data scientist instructs an operator on the clinical side to look at specific images and try to identify abnormal patterns.
Furthermore, our modular <span id="S6.SS0.SSS0.Px2.p1.1.2" class="ltx_text ltx_font_typewriter">TrainingPlan</span> design can support advanced explainable techniques, while still guaranteeing node’s data privacy through our <span id="S6.SS0.SSS0.Px2.p1.1.3" class="ltx_text ltx_font_typewriter">TrainingPlan</span> approval process.
Despite these mitigation efforts, the issue of remotely debugging ML models in a privacy-preserving way remains a difficult problem to solve.</p>
</div>
</section>
<section id="S6.SS0.SSS0.Px3" class="ltx_paragraph">
<h5 class="ltx_title ltx_title_paragraph">Data preparation prior to federated training</h5>

<div id="S6.SS0.SSS0.Px3.p1" class="ltx_para">
<p id="S6.SS0.SSS0.Px3.p1.1" class="ltx_p">Data in clinical databases are typically stored in proprietary formats that are not generally readily usable for ML analysis.
However, all FL frameworks, including <span id="S6.SS0.SSS0.Px3.p1.1.1" class="ltx_ERROR undefined">\fbm</span>, make the assumption that the input data is provided in a format ready for ingestion by an ML model, or maybe a preprocessing pipeline.
This gap has been, in our experience, a major source of frustration and delays, with the burden of data export and conversion falling usually on clinical data managers lacking the budget and training for such operations.
<span id="S6.SS0.SSS0.Px3.p1.1.2" class="ltx_ERROR undefined">\fbm</span>tries to mitigate this by providing <span id="S6.SS0.SSS0.Px3.p1.1.3" class="ltx_text ltx_font_typewriter">Dataset</span> classes that can be populated with data that has been exported but not highly transformed, for example our <span id="S6.SS0.SSS0.Px3.p1.1.4" class="ltx_text ltx_font_typewriter">MedicalFolderDataset</span> class that can handle data in a simplified BIDS structure.
In the future, we also plan to extend list of supported formats with some that are closer to the raw data exported by hospital IT systems, such as DICOM for imaging, OMOP or FHIR for clinical data, VCF files for genomics, and others.
An ideal scenario would include a direct integration with the hospital PACS or EHR systems, however the proprietary nature of most of this software and the strict security rules unfortunately make this an unlikely scenario.
Regardless of the feasibility, we believe that FL frameworks targeted at biomedical research applications must simplify the process of data preparation and be able to interoperate with standard data formats in their raw form, to ease the burden on clinical data managers and improve data reusability.</p>
</div>
</section>
<section id="S6.SS0.SSS0.Px4" class="ltx_paragraph">
<h5 class="ltx_title ltx_title_paragraph">Data privacy and regulation for real-world deployment</h5>

<div id="S6.SS0.SSS0.Px4.p1" class="ltx_para">
<p id="S6.SS0.SSS0.Px4.p1.1" class="ltx_p">Contrary to controlled academic environments, real-world deployment scenarios are characterized by strict privacy and security measures.
Some notable examples include: avoiding training if a client’s dataset has too few samples, requiring secure aggregation and differential privacy measures, restricting the data flow in the network, and ensuring compliance with hospitals firewall policies.
The integration in <span id="S6.SS0.SSS0.Px4.p1.1.1" class="ltx_ERROR undefined">\fbm</span>of such security measures to meet the requirements asked by Data Protection Officers of hospitals has been a long, iterative process consisting of multiple rounds of discussion needed to achieve a common understanding and a shared vocabulary.
Ultimately, this dialectic process is what allowed us to carry out our first deployment in a collaborative clinical project.</p>
</div>
</section>
</section>
<section id="S7" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">7 </span>Conclusion</h2>

<div id="S7.p1" class="ltx_para">
<p id="S7.p1.1" class="ltx_p">We have presented version v4.1 of <span id="S7.p1.1.1" class="ltx_ERROR undefined">\fbm</span>, a Python-based FL framework aimed at supporting real-world deployments for biomedical and healthcare research applications.
This document focuses on <span id="S7.p1.1.2" class="ltx_ERROR undefined">\fbm</span>’s philosophy, guiding principles, and relevant details of the current architecture.
<span id="S7.p1.1.3" class="ltx_ERROR undefined">\fbm</span>is an open-source project available in our public repository<span id="footnote11" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">11</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">11</sup><span class="ltx_tag ltx_tag_note">11</span><a target="_blank" href="https://gitlab.inria.fr/fedbiomed/fedbiomed" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://gitlab.inria.fr/fedbiomed/fedbiomed</a></span></span></span> under an Apache license, and we welcome contributions from the community.
We also provide extensive user documentation and tutorials, in addition to an API reference for developers<span id="footnote12" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">12</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">12</sup><span class="ltx_tag ltx_tag_note">12</span><a target="_blank" href="https://fedbiomed.gitlabpages.inria.fr/v4.1/getting-started/what-is-fedbiomed/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://fedbiomed.gitlabpages.inria.fr/v4.1/getting-started/what-is-fedbiomed/</a></span></span></span>.
The development of <span id="S7.p1.1.4" class="ltx_ERROR undefined">\fbm</span>is an ongoing effort currently brought forward by a group of vetted collaborators coordinated by a small number of core developers.
In the future, we plan to grow our community of users and developers, and integrate open-source contributions through a consortium.
Our short-term implementation roadmap includes finalizing our implementation of Secure Aggregation, securing client-server communications, and improving even further the usability for clinical data providers.
The domain of healthcare and clinical research is characterized by specific challenges and requirements that are difficult or impossible to satisfy with the currently available frameworks.
We hope that <span id="S7.p1.1.5" class="ltx_ERROR undefined">\fbm</span>will lower the threshold to apply FL in this domain, enabling clinical data scientists to perform experiments and train ML models within a federated consortium of data providers in a secure, practical, and privacy-preserving way.</p>
</div>
</section>
<section id="S8" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">8 </span>Supplementary Information</h2>

<section id="S8.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">8.1 </span>Real-world use case scenario: prostate segmentation</h3>

<div id="S8.SS1.p1" class="ltx_para">
<p id="S8.SS1.p1.1" class="ltx_p">Data were gathered following the process described in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx25" title="" class="ltx_ref">25</a>]</cite> from three public datasets <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx1" title="" class="ltx_ref">1</a>, <a href="#bib.bibx32" title="" class="ltx_ref">32</a>, <a href="#bib.bibx2" title="" class="ltx_ref">2</a>]</cite>.
Each clinical site received data corresponding to a single source, to reflect the more realistic scenario of high heterogeneity of data among clinical sites.
Details of the data distribution are given in Table <a href="#S8.T3" title="Table 3 ‣ 8.1 Real-world use case scenario: prostate segmentation ‣ 8 Supplementary Information ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, while details of the experiment setup are given in Table <a href="#S8.T4" title="Table 4 ‣ 8.1 Real-world use case scenario: prostate segmentation ‣ 8 Supplementary Information ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.
We used <span id="S8.SS1.p1.1.1" class="ltx_ERROR undefined">\fbm</span>v4.1.2 relied on MONAI’s (v1.0.1) implementation of UNet <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx29" title="" class="ltx_ref">29</a>]</cite> and data transformations, and on Pytorch’s SGD implementation, for definition of the model and optimizer in each site’s local training.</p>
</div>
<figure id="S8.T3" class="ltx_table">
<table id="S8.T3.1" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S8.T3.1.1.1" class="ltx_tr">
<th id="S8.T3.1.1.1.1" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_th_row ltx_border_r">Center</th>
<th id="S8.T3.1.1.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row">Dataset</th>
<th id="S8.T3.1.1.1.3" class="ltx_td ltx_align_center ltx_th ltx_th_column">Train samples</th>
<th id="S8.T3.1.1.1.4" class="ltx_td ltx_align_center ltx_th ltx_th_column">holdout samples</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S8.T3.1.2.1" class="ltx_tr">
<th id="S8.T3.1.2.1.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_r ltx_border_t">CAL</th>
<th id="S8.T3.1.2.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_t">ProstateX <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx2" title="" class="ltx_ref">2</a>]</cite>
</th>
<td id="S8.T3.1.2.1.3" class="ltx_td ltx_align_center ltx_border_t">147</td>
<td id="S8.T3.1.2.1.4" class="ltx_td ltx_align_center ltx_border_t">37</td>
</tr>
<tr id="S8.T3.1.3.2" class="ltx_tr">
<th id="S8.T3.1.3.2.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_r">CHB</th>
<th id="S8.T3.1.3.2.2" class="ltx_td ltx_align_left ltx_th ltx_th_row">Promise <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx32" title="" class="ltx_ref">32</a>]</cite>
</th>
<td id="S8.T3.1.3.2.3" class="ltx_td ltx_align_center">21</td>
<td id="S8.T3.1.3.2.4" class="ltx_td ltx_align_center">6</td>
</tr>
<tr id="S8.T3.1.4.3" class="ltx_tr">
<th id="S8.T3.1.4.3.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_r">CURIE</th>
<th id="S8.T3.1.4.3.2" class="ltx_td ltx_align_left ltx_th ltx_th_row">Decathlon <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx1" title="" class="ltx_ref">1</a>]</cite>
</th>
<td id="S8.T3.1.4.3.3" class="ltx_td ltx_align_center">25</td>
<td id="S8.T3.1.4.3.4" class="ltx_td ltx_align_center">7</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 3: </span>Distribution of data among clinical sites during the FL experiment, including the number of samples in each cross-validation fold.</figcaption>
</figure>
<figure id="S8.T4" class="ltx_table">
<table id="S8.T4.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S8.T4.1.1.1" class="ltx_tr">
<td id="S8.T4.1.1.1.1" class="ltx_td ltx_align_left" rowspan="3"><span id="S8.T4.1.1.1.1.1" class="ltx_text">Data</span></td>
<td id="S8.T4.1.1.1.2" class="ltx_td ltx_align_left ltx_border_r">Transformation</td>
<td id="S8.T4.1.1.1.3" class="ltx_td ltx_align_left">Center cropping and padding</td>
</tr>
<tr id="S8.T4.1.2.2" class="ltx_tr">
<td id="S8.T4.1.2.2.1" class="ltx_td ltx_align_left ltx_border_r">Common shape</td>
<td id="S8.T4.1.2.2.2" class="ltx_td ltx_align_left">320, 320, 16</td>
</tr>
<tr id="S8.T4.1.3.3" class="ltx_tr">
<td id="S8.T4.1.3.3.1" class="ltx_td ltx_align_left ltx_border_r">Transformation</td>
<td id="S8.T4.1.3.3.2" class="ltx_td ltx_align_left">Intensity normalization</td>
</tr>
<tr id="S8.T4.1.4.4" class="ltx_tr">
<td id="S8.T4.1.4.4.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="5"><span id="S8.T4.1.4.4.1.1" class="ltx_text">UNet</span></td>
<td id="S8.T4.1.4.4.2" class="ltx_td ltx_align_left ltx_border_r ltx_border_t">channels</td>
<td id="S8.T4.1.4.4.3" class="ltx_td ltx_align_left ltx_border_t">16, 32, 64, 128, 256</td>
</tr>
<tr id="S8.T4.1.5.5" class="ltx_tr">
<td id="S8.T4.1.5.5.1" class="ltx_td ltx_align_left ltx_border_r">strides</td>
<td id="S8.T4.1.5.5.2" class="ltx_td ltx_align_left">2, 2, 2, 2</td>
</tr>
<tr id="S8.T4.1.6.6" class="ltx_tr">
<td id="S8.T4.1.6.6.1" class="ltx_td ltx_align_left ltx_border_r">residual units</td>
<td id="S8.T4.1.6.6.2" class="ltx_td ltx_align_left">3</td>
</tr>
<tr id="S8.T4.1.7.7" class="ltx_tr">
<td id="S8.T4.1.7.7.1" class="ltx_td ltx_align_left ltx_border_r">normalization</td>
<td id="S8.T4.1.7.7.2" class="ltx_td ltx_align_left">Batch normalization</td>
</tr>
<tr id="S8.T4.1.8.8" class="ltx_tr">
<td id="S8.T4.1.8.8.1" class="ltx_td ltx_align_left ltx_border_r">dropout</td>
<td id="S8.T4.1.8.8.2" class="ltx_td ltx_align_left">0.3</td>
</tr>
<tr id="S8.T4.1.9.9" class="ltx_tr">
<td id="S8.T4.1.9.9.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="6"><span id="S8.T4.1.9.9.1.1" class="ltx_text">Local</span></td>
<td id="S8.T4.1.9.9.2" class="ltx_td ltx_align_left ltx_border_r ltx_border_t">Optimizer</td>
<td id="S8.T4.1.9.9.3" class="ltx_td ltx_align_left ltx_border_t">SGD</td>
</tr>
<tr id="S8.T4.1.10.10" class="ltx_tr">
<td id="S8.T4.1.10.10.1" class="ltx_td ltx_align_left ltx_border_r">Learning rate</td>
<td id="S8.T4.1.10.10.2" class="ltx_td ltx_align_left">0.1</td>
</tr>
<tr id="S8.T4.1.11.11" class="ltx_tr">
<td id="S8.T4.1.11.11.1" class="ltx_td ltx_align_left ltx_border_r">Momentum</td>
<td id="S8.T4.1.11.11.2" class="ltx_td ltx_align_left">0.9</td>
</tr>
<tr id="S8.T4.1.12.12" class="ltx_tr">
<td id="S8.T4.1.12.12.1" class="ltx_td ltx_align_left ltx_border_r">Batch size</td>
<td id="S8.T4.1.12.12.2" class="ltx_td ltx_align_left">8</td>
</tr>
<tr id="S8.T4.1.13.13" class="ltx_tr">
<td id="S8.T4.1.13.13.1" class="ltx_td ltx_align_left ltx_border_r">Local updates</td>
<td id="S8.T4.1.13.13.2" class="ltx_td ltx_align_left">25</td>
</tr>
<tr id="S8.T4.1.14.14" class="ltx_tr">
<td id="S8.T4.1.14.14.1" class="ltx_td ltx_align_left ltx_border_r">Loss</td>
<td id="S8.T4.1.14.14.2" class="ltx_td ltx_align_left">Dice loss</td>
</tr>
<tr id="S8.T4.1.15.15" class="ltx_tr">
<td id="S8.T4.1.15.15.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S8.T4.1.15.15.1.1" class="ltx_text">Federated</span></td>
<td id="S8.T4.1.15.15.2" class="ltx_td ltx_align_left ltx_border_r ltx_border_t">Aggregation</td>
<td id="S8.T4.1.15.15.3" class="ltx_td ltx_align_left ltx_border_t">FedAVG</td>
</tr>
<tr id="S8.T4.1.16.16" class="ltx_tr">
<td id="S8.T4.1.16.16.1" class="ltx_td ltx_align_left ltx_border_r">Rounds</td>
<td id="S8.T4.1.16.16.2" class="ltx_td ltx_align_left">40</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 4: </span>FL experimental setup details.</figcaption>
</figure>
</section>
<section id="S8.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">8.2 </span>Technical implementation details</h3>

<div id="S8.SS2.p1" class="ltx_para">
<p id="S8.SS2.p1.1" class="ltx_p">Building on our functional description in Section <a href="#S4.SS2" title="4.2 Design and functional architecture ‣ 4 Methods ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4.2</span></a>, we provide here a more detailed explanation of <span id="S8.SS2.p1.1.1" class="ltx_ERROR undefined">\fbm</span>’s implementation details, as shown in Figure <a href="#S4.F2" title="Figure 2 ‣ 4.2 Design and functional architecture ‣ 4 Methods ‣ \fbm: Open, Transparent and Trusted Federated Learning for Real-world Healthcare Applications" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>, trying to make a clear distinction between those that, as of today, have already reached a satisfactory level of maturity and those that are expected to improve in the near future, or may change at some point due to the dynamic nature of our project.</p>
</div>
<section id="S8.SS2.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">8.2.1 </span>Core library functionalities</h4>

<div id="S8.SS2.SSS1.p1" class="ltx_para">
<p id="S8.SS2.SSS1.p1.1" class="ltx_p">On the node side we have implemented: a database of dataset metadata based on the TinyDB<span id="footnote13" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">13</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">13</sup><span class="ltx_tag ltx_tag_note">13</span><a target="_blank" href="https://pypi.org/project/tinydb/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://pypi.org/project/tinydb/</a></span></span></span> package; a task manager based on persist-queue<span id="footnote14" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">14</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">14</sup><span class="ltx_tag ltx_tag_note">14</span><a target="_blank" href="https://github.com/peter-wangxu/persist-queue" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/peter-wangxu/persist-queue</a></span></span></span>; a suite of <span id="S8.SS2.SSS1.p1.1.1" class="ltx_text ltx_font_typewriter">Dataset</span> classes to represent tabular, medical imaging (BIDS <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx22" title="" class="ltx_ref">22</a>]</cite>), and other data formats; and a <span id="S8.SS2.SSS1.p1.1.2" class="ltx_text ltx_font_typewriter">Round</span> class handling all the logic for the execution of one federated training round.
In the future, we plan to vastly extend our library of supported data formats to include other imaging formats such as e.g. DICOM, add support for NLP formats, and integrate interoperability standards and data models such as i2b2 and OMOP<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx42" title="" class="ltx_ref">42</a>, <a href="#bib.bibx41" title="" class="ltx_ref">41</a>, <a href="#bib.bibx55" title="" class="ltx_ref">55</a>]</cite>.
Furthermore, we may consider moving to more complex database solutions if dataset management becomes too cumbersome, and similarly we may consider more advanced task management systems should the need arise among clinical data providers for better resource handling and improved performance.
On the researcher side, an <span id="S8.SS2.SSS1.p1.1.3" class="ltx_text ltx_font_typewriter">Experiment</span> class represents the entry-point for the configuration and steering of a FL experiment, while the <span id="S8.SS2.SSS1.p1.1.4" class="ltx_text ltx_font_typewriter">TrainingPlan</span> class contains all the logic and code to be shipped to nodes for remote execution.
The highly interactive design of the researcher-facing classes represents one of the highlights of <span id="S8.SS2.SSS1.p1.1.5" class="ltx_ERROR undefined">\fbm</span>, and in the future we plan to improve this feature by moving towards an ever more modular design with clearer separation of responsibilities, a hierarchical approach, and improved user documentation.</p>
</div>
<div id="S8.SS2.SSS1.p2" class="ltx_para">
<p id="S8.SS2.SSS1.p2.1" class="ltx_p">Communication between the researcher and the nodes happens through an intermediate component called the <span id="S8.SS2.SSS1.p2.1.1" class="ltx_text ltx_font_typewriter">Network</span>.
This component is based on two technologies: MQTT for the brokering of short messages (e.g. train, search datasets, and others) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx3" title="" class="ltx_ref">3</a>]</cite>, and an HTTP API based on Django REST<span id="footnote15" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">15</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">15</sup><span class="ltx_tag ltx_tag_note">15</span><a target="_blank" href="https://www.django-rest-framework.org/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.django-rest-framework.org/</a></span></span></span> for exchanging larger files such as model parameters.
MQTT was chosen because of its natural mapping to a star topology and good support of broadcast operations, which we use for discovering datasets and clients.
However, in the future we plan to evaluate other approaches, more commonly used in the literature, such as relying on gRPC or other remote execution protocols.
In the current implementation, nodes are pure slaves that execute commands issued by the researcher.
In the future, we plan to evaluate endowing nodes with a more active role, for example inverting the direction of communication such that the nodes would request tasks instead of passively receiving them, which would entail security benefits (reduced attack surface on the nodes), while shifting some of the complexity from the node implementation to the network component.</p>
</div>
</section>
<section id="S8.SS2.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">8.2.2 </span>User interfaces</h4>

<div id="S8.SS2.SSS2.p1" class="ltx_para">
<p id="S8.SS2.SSS2.p1.1" class="ltx_p">In terms of user interfaces, on the node side we have implemented both a Command Line Interface (CLI) and a Graphical User Interface (GUI).
Our GUI implementation is composed of a web client based on React, and a Flask backend server that manages the interactions with the <span id="S8.SS2.SSS2.p1.1.1" class="ltx_ERROR undefined">\fbm</span>library, in particular with the dataset database.
This simplifies the process of managing datasets (CRUD operations <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx37" title="" class="ltx_ref">37</a>]</cite>), training plan approval by data science experts on the node side, and has the general goal of facilitating governance operations on data and models.
Currently, the GUI is limited to executing on the same machine as the node process, but in the future we plan to decouple the two through a micro-services approach, as well as improve the RBAC with better security and account management.
Another feature that we plan to implement in the future is adding experiment monitoring functionalities for the node, as well as allowing basic experiment steering functionalities such as stopping an experiment through the GUI.
The researcher’s interface is a Python SDK, mainly designed to be executed by interpreter such as Jython and within notebooks.
The researcher may monitor the training via Tensorboard, which plots common metrics such as training loss as well as custom ones defined via a plugin system.
Defining the best interface for the researcher in terms of tradeoff between interactivity, security, and simplicity is one of our main goals, and we constantly strive to improve the API for the <span id="S8.SS2.SSS2.p1.1.2" class="ltx_text ltx_font_typewriter">Experiment</span> and <span id="S8.SS2.SSS2.p1.1.3" class="ltx_text ltx_font_typewriter">TrainingPlan</span> classes.</p>
</div>
</section>
<section id="S8.SS2.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">8.2.3 </span>Cybersecurity and model security</h4>

<div id="S8.SS2.SSS3.p1" class="ltx_para">
<p id="S8.SS2.SSS3.p1.1" class="ltx_p">Cybersecurity is one of the main design pillars of <span id="S8.SS2.SSS3.p1.1.1" class="ltx_ERROR undefined">\fbm</span>.
As described above, we have implemented a VPN deployment mode based on the Wireguard framework to isolate the execution of <span id="S8.SS2.SSS3.p1.1.2" class="ltx_ERROR undefined">\fbm</span>from external parties <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx16" title="" class="ltx_ref">16</a>]</cite>.
The VPN deployment mode is shipped with the software, and designed to be easily configurable and deployable.
One of our short-term goals is to implement TLS-encrypted communication within the VPN, by exploring suitable MQTT and Django-REST extensions.
To protect nodes from the execution of arbitrary code, a training plan approval mode can be enabled.
This process is handled by the GUI on the Node side, making it easy and immediate for clinical data providers to conduct their reviews.
In this configuration, when a researcher sends a train request to the node, the hash of the training plan is compared to the hashes of a set of training plans that have been previously reviewed and approved by the node, and is executed only in the case of a match.
Differential Privacy can also be enabled in <span id="S8.SS2.SSS3.p1.1.3" class="ltx_ERROR undefined">\fbm</span>, when using PyTorch as the training backend, by leveraging the Opacus library and Tensorflow’s accountant<span id="footnote16" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">16</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">16</sup><span class="ltx_tag ltx_tag_note">16</span><a target="_blank" href="https://raw.githubusercontent.com/tensorflow/privacy/7eea74a6a1cf15e2d2bd890722400edd0e470db8/research/hyperparameters_2022/rdp_accountant.py" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://raw.githubusercontent.com/tensorflow/privacy/7eea74a6a1cf15e2d2bd890722400edd0e470db8/research/hyperparameters_2022/rdp_accountant.py</a></span></span></span>.
Secure aggregation is being implemented based on additively homomorphic encryption <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx27" title="" class="ltx_ref">27</a>, <a href="#bib.bibx36" title="" class="ltx_ref">36</a>]</cite>, with the computation of keys based on a multi-party computation (MPC) approach inspired by the MP-SPDZ benchmark suite <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bibx28" title="" class="ltx_ref">28</a>]</cite>.</p>
</div>
</section>
</section>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
<li id="bib.bibx1" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">Michela Antonelli et al.
</span>
<span class="ltx_bibblock">“The medical segmentation decathlon”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx1.1.1" class="ltx_emph ltx_font_italic">Nature communications</em> <span id="bib.bibx1.2.2" class="ltx_text ltx_font_bold">13.1</span>
</span>
<span class="ltx_bibblock">Nature Publishing Group UK London, 2022, pp. 4128
</span>
</li>
<li id="bib.bibx2" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">Samuel G Armato III et al.
</span>
<span class="ltx_bibblock">“PROSTATEx Challenges for computerized classification of prostate lesions from multiparametric magnetic resonance images”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx2.1.1" class="ltx_emph ltx_font_italic">Journal of Medical Imaging</em> <span id="bib.bibx2.2.2" class="ltx_text ltx_font_bold">5.4</span>
</span>
<span class="ltx_bibblock">Society of Photo-Optical Instrumentation Engineers, 2018, pp. 044501–044501
</span>
</li>
<li id="bib.bibx3" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">Andrew Banks, Ed Briggs, Ken Borgendale and Rahul Gupta
</span>
<span class="ltx_bibblock">“MQTT Version 5.0”, 2019
</span>
</li>
<li id="bib.bibx4" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">Daniel J Beutel et al.
</span>
<span class="ltx_bibblock">“Flower: A friendly federated learning research framework”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx4.1.1" class="ltx_emph ltx_font_italic">arXiv preprint arXiv:2007.14390</em>, 2020
</span>
</li>
<li id="bib.bibx5" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">Keith Bonawitz et al.
</span>
<span class="ltx_bibblock">“Towards federated learning at scale: System design”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx5.1.1" class="ltx_emph ltx_font_italic">Proceedings of Machine Learning and Systems</em> <span id="bib.bibx5.2.2" class="ltx_text ltx_font_bold">1</span>, 2019, pp. 374–388
</span>
</li>
<li id="bib.bibx6" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">Nader Bouacida and Prasant Mohapatra
</span>
<span class="ltx_bibblock">“Vulnerabilities in federated learning”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx6.1.1" class="ltx_emph ltx_font_italic">IEEE Access</em> <span id="bib.bibx6.2.2" class="ltx_text ltx_font_bold">9</span>
</span>
<span class="ltx_bibblock">IEEE, 2021, pp. 63229–63249
</span>
</li>
<li id="bib.bibx7" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">Kenneth Clark et al.
</span>
<span class="ltx_bibblock">“The Cancer Imaging Archive (TCIA): maintaining and operating a public information repository”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx7.1.1" class="ltx_emph ltx_font_italic">Journal of digital imaging</em> <span id="bib.bibx7.2.2" class="ltx_text ltx_font_bold">26.6</span>
</span>
<span class="ltx_bibblock">Springer, 2013, pp. 1045–1057
</span>
</li>
<li id="bib.bibx8" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">Mellody consortium
</span>
<span class="ltx_bibblock">“Mellody” Accessed on 28th November 2022, 2019
</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://www.melloddy.eu/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.melloddy.eu/</a>
</span>
</li>
<li id="bib.bibx9" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">Matthew G Crowson et al.
</span>
<span class="ltx_bibblock">“A systematic review of federated learning applications for biomedical data”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx9.1.1" class="ltx_emph ltx_font_italic">PLOS Digital Health</em> <span id="bib.bibx9.2.2" class="ltx_text ltx_font_bold">1.5</span>
</span>
<span class="ltx_bibblock">Public Library of Science San Francisco, CA USA, 2022, pp. e0000033
</span>
</li>
<li id="bib.bibx10" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">Marc Cuggia and Stéphanie Combes
</span>
<span class="ltx_bibblock">“The French Health Data Hub and the German Medical Informatics Initiatives: two national projects to promote data sharing in healthcare”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx10.1.1" class="ltx_emph ltx_font_italic">Yearbook of medical informatics</em> <span id="bib.bibx10.2.2" class="ltx_text ltx_font_bold">28.01</span>
</span>
<span class="ltx_bibblock">Georg Thieme Verlag KG, 2019, pp. 195–202
</span>
</li>
<li id="bib.bibx11" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock">Renato Cuocolo et al.
</span>
<span class="ltx_bibblock">“Quality control and whole-gland, zonal and lesion annotations for the PROSTATEx challenge public dataset”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx11.1.1" class="ltx_emph ltx_font_italic">European Journal of Radiology</em> <span id="bib.bibx11.2.2" class="ltx_text ltx_font_bold">138</span>
</span>
<span class="ltx_bibblock">Elsevier, 2021, pp. 109647
</span>
</li>
<li id="bib.bibx12" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock">European Union Agency Cybersecurity (ENISA)
</span>
<span class="ltx_bibblock">“ICT Security Certification Opportunities in the Healthcare Sector”
</span>
<span class="ltx_bibblock">European Union Agency For NetworkInformation Security Attiki, Greece, 2018
</span>
</li>
<li id="bib.bibx13" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock">Erfan Darzidehkalani, Mohammad Ghasemi-Rad and Pma Ooijen
</span>
<span class="ltx_bibblock">“Federated Learning in Medical Imaging: Part I: Toward Multicentral Health Care Ecosystems”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx13.1.1" class="ltx_emph ltx_font_italic">Journal of the American College of Radiology</em>
</span>
<span class="ltx_bibblock">Elsevier, 2022
</span>
</li>
<li id="bib.bibx14" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock">Ittai Dayan et al.
</span>
<span class="ltx_bibblock">“Federated learning for predicting clinical outcomes in patients with COVID-19”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx14.1.1" class="ltx_emph ltx_font_italic">Nature medicine</em> <span id="bib.bibx14.2.2" class="ltx_text ltx_font_bold">27.10</span>
</span>
<span class="ltx_bibblock">Nature Publishing Group, 2021, pp. 1735–1743
</span>
</li>
<li id="bib.bibx15" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock">Timo M. Deist et al.
</span>
<span class="ltx_bibblock">“Distributed learning on 20 000+ lung cancer patients – The Personal Health Train”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx15.1.1" class="ltx_emph ltx_font_italic">Radiotherapy and Oncology</em> <span id="bib.bibx15.2.2" class="ltx_text ltx_font_bold">144</span>, 2020, pp. 189–200
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/10.1016/j.radonc.2019.11.019" title="" class="ltx_ref ltx_href">10.1016/j.radonc.2019.11.019</a>
</span>
</li>
<li id="bib.bibx16" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock">Jason A Donenfeld
</span>
<span class="ltx_bibblock">“Wireguard: next generation kernel network tunnel”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx16.1.1" class="ltx_emph ltx_font_italic">NDSS</em>, 2017, pp. 1–12
</span>
</li>
<li id="bib.bibx17" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock">Patrick Foley et al.
</span>
<span class="ltx_bibblock">“OpenFL: the open federated learning library”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx17.1.1" class="ltx_emph ltx_font_italic">Physics in Medicine &amp; Biology</em>
</span>
<span class="ltx_bibblock">IOP Publishing, 2022
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/10.1088/1361-6560/ac97d9" title="" class="ltx_ref ltx_href">10.1088/1361-6560/ac97d9</a>
</span>
</li>
<li id="bib.bibx18" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock">Cloud Native Computing Foundation
</span>
<span class="ltx_bibblock">“high performance, open-source universal RPC framework” Accessed on 16 December 2022, 2018
</span>
<span class="ltx_bibblock">URL: <a href="%5Curl%7Bhttps://grpc.io/%7D" title="" class="ltx_ref ltx_url ltx_font_typewriter">%5Curl%7Bhttps://grpc.io/%7D</a>
</span>
</li>
<li id="bib.bibx19" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock">David Froelicher et al.
</span>
<span class="ltx_bibblock">“Truly privacy-preserving federated analytics for precision medicine with multiparty homomorphic encryption”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx19.1.1" class="ltx_emph ltx_font_italic">Nature Communications</em> <span id="bib.bibx19.2.2" class="ltx_text ltx_font_bold">12.1</span>, 2021, pp. 5910
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/10.1038/s41467-021-25972-y" title="" class="ltx_ref ltx_href">10.1038/s41467-021-25972-y</a>
</span>
</li>
<li id="bib.bibx20" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock">Mathieu N Galtier and Camille Marini
</span>
<span class="ltx_bibblock">“Substra: a framework for privacy-preserving, traceable and collaborative machine learning”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx20.1.1" class="ltx_emph ltx_font_italic">arXiv preprint arXiv:1910.11567</em>, 2019
</span>
</li>
<li id="bib.bibx21" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock">Marzyeh Ghassemi et al.
</span>
<span class="ltx_bibblock">“A Review of Challenges and Opportunities in Machine Learning for Health”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx21.1.1" class="ltx_emph ltx_font_italic">AMIA Summits on Translational Science Proceedings</em> <span id="bib.bibx21.2.2" class="ltx_text ltx_font_bold">2020</span>, 2020, pp. 191–200
</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233077/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233077/</a>
</span>
</li>
<li id="bib.bibx22" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[22]</span>
<span class="ltx_bibblock">Krzysztof J Gorgolewski et al.
</span>
<span class="ltx_bibblock">“The brain imaging data structure, a format for organizing and describing outputs of neuroimaging experiments”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx22.1.1" class="ltx_emph ltx_font_italic">Scientific data</em> <span id="bib.bibx22.2.2" class="ltx_text ltx_font_bold">3.1</span>
</span>
<span class="ltx_bibblock">Nature Publishing Group, 2016, pp. 1–9
</span>
</li>
<li id="bib.bibx23" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[23]</span>
<span class="ltx_bibblock">Chaoyang He et al.
</span>
<span class="ltx_bibblock">“Fedml: A research library and benchmark for federated machine learning”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx23.1.1" class="ltx_emph ltx_font_italic">arXiv preprint arXiv:2007.13518</em>, 2020
</span>
</li>
<li id="bib.bibx24" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[24]</span>
<span class="ltx_bibblock">Tim Hulsen et al.
</span>
<span class="ltx_bibblock">“From big data to precision medicine”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx24.1.1" class="ltx_emph ltx_font_italic">Frontiers in medicine</em>
</span>
<span class="ltx_bibblock">Frontiers, 2019, pp. 34
</span>
</li>
<li id="bib.bibx25" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[25]</span>
<span class="ltx_bibblock">Lucia et al. Innocenti
</span>
<span class="ltx_bibblock">“Consensus-Based Methods as a Cost-Effective Alternative to Federated Learning: Benchmark on Prostate MRI Segmentation” under review, 2023
</span>
</li>
<li id="bib.bibx26" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[26]</span>
<span class="ltx_bibblock">Clifford R Jack Jr et al.
</span>
<span class="ltx_bibblock">“The Alzheimer’s disease neuroimaging initiative (ADNI): MRI methods”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx26.1.1" class="ltx_emph ltx_font_italic">Journal of Magnetic Resonance Imaging: An Official Journal of the International Society for Magnetic Resonance in Medicine</em> <span id="bib.bibx26.2.2" class="ltx_text ltx_font_bold">27.4</span>
</span>
<span class="ltx_bibblock">Wiley Online Library, 2008, pp. 685–691
</span>
</li>
<li id="bib.bibx27" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[27]</span>
<span class="ltx_bibblock">Marc Joye and Benoît Libert
</span>
<span class="ltx_bibblock">“A scalable scheme for privacy-preserving aggregation of time-series data”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx27.1.1" class="ltx_emph ltx_font_italic">International Conference on Financial Cryptography and Data Security</em>, 2013, pp. 111–125
</span>
<span class="ltx_bibblock">Springer
</span>
</li>
<li id="bib.bibx28" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[28]</span>
<span class="ltx_bibblock">Marcel Keller
</span>
<span class="ltx_bibblock">“MP-SPDZ: A Versatile Framework for Multi-Party Computation”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx28.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 2020 ACM SIGSAC Conference on Computer and Communications Security</em>, 2020
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/10.1145/3372297.3417872" title="" class="ltx_ref ltx_href">10.1145/3372297.3417872</a>
</span>
</li>
<li id="bib.bibx29" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[29]</span>
<span class="ltx_bibblock">Eric Kerfoot et al.
</span>
<span class="ltx_bibblock">“Left-ventricle quantification using residual U-Net”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx29.1.1" class="ltx_emph ltx_font_italic">Statistical Atlases and Computational Models of the Heart. Atrial Segmentation and LV Quantification Challenges: 9th International Workshop, STACOM 2018, Held in Conjunction with MICCAI 2018, Granada, Spain, September 16, 2018, Revised Selected Papers 9</em>, 2019, pp. 371–380
</span>
<span class="ltx_bibblock">Springer
</span>
</li>
<li id="bib.bibx30" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[30]</span>
<span class="ltx_bibblock">Adrienne Kline et al.
</span>
<span class="ltx_bibblock">“Multimodal machine learning in precision health: A scoping review”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx30.1.1" class="ltx_emph ltx_font_italic">npj Digital Medicine</em> <span id="bib.bibx30.2.2" class="ltx_text ltx_font_bold">5.1</span>
</span>
<span class="ltx_bibblock">Nature Publishing Group, 2022, pp. 1–14
</span>
</li>
<li id="bib.bibx31" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[31]</span>
<span class="ltx_bibblock">Qinbin Li et al.
</span>
<span class="ltx_bibblock">“A Survey on Federated Learning Systems: Vision, Hype and Reality for Data Privacy and Protection”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx31.1.1" class="ltx_emph ltx_font_italic">IEEE Transactions on Knowledge and Data Engineering</em>, 2021, pp. 1–1
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/10.1109/TKDE.2021.3124599" title="" class="ltx_ref ltx_href">10.1109/TKDE.2021.3124599</a>
</span>
</li>
<li id="bib.bibx32" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[32]</span>
<span class="ltx_bibblock">Geert Litjens et al.
</span>
<span class="ltx_bibblock">“Evaluation of prostate segmentation algorithms for MRI: the PROMISE12 challenge”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx32.1.1" class="ltx_emph ltx_font_italic">Medical image analysis</em> <span id="bib.bibx32.2.2" class="ltx_text ltx_font_bold">18.2</span>
</span>
<span class="ltx_bibblock">Elsevier, 2014, pp. 359–373
</span>
</li>
<li id="bib.bibx33" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[33]</span>
<span class="ltx_bibblock">Yang Liu et al.
</span>
<span class="ltx_bibblock">“FATE: An Industrial Grade Platform for Collaborative Learning With Data Protection.”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx33.1.1" class="ltx_emph ltx_font_italic">J. Mach. Learn. Res.</em> <span id="bib.bibx33.2.2" class="ltx_text ltx_font_bold">22.226</span>, 2021, pp. 1–6
</span>
</li>
<li id="bib.bibx34" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[34]</span>
<span class="ltx_bibblock">Heiko Ludwig et al.
</span>
<span class="ltx_bibblock">“Ibm federated learning: an enterprise framework white paper v0. 1”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx34.1.1" class="ltx_emph ltx_font_italic">arXiv preprint arXiv:2007.10987</em>, 2020
</span>
</li>
<li id="bib.bibx35" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[35]</span>
<span class="ltx_bibblock">Yanjun Ma, Dianhai Yu, Tian Wu and Haifeng Wang
</span>
<span class="ltx_bibblock">“PaddlePaddle: An open-source deep learning platform from industrial practice”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx35.1.1" class="ltx_emph ltx_font_italic">Frontiers of Data and Domputing</em> <span id="bib.bibx35.2.2" class="ltx_text ltx_font_bold">1.1</span>, 2019, pp. 105–115
</span>
</li>
<li id="bib.bibx36" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[36]</span>
<span class="ltx_bibblock">Mohamad Mansouri, Melek Önen and Wafa Ben Jaballah
</span>
<span class="ltx_bibblock">“Learning from failures: Secure and fault-tolerant aggregation for federated learning” © ACM, 2022. This is the author&amp;#039;s version of the work. It is posted here by permission of ACM for your personal use. Not for redistribution. The definitive version was published in ACSAC 2022, Annual Computer Security Applications Conference, 5-9 December 2022, Austin, Texas, USA https://doi.org/10.1145/3564625.3568135
</span>
<span class="ltx_bibblock">In <em id="bib.bibx36.1.1" class="ltx_emph ltx_font_italic">ACSAC 2022, Annual Computer Security Applications Conference, 5-9 December 2022, Austin, Texas, USA</em>, 2022
</span>
</li>
<li id="bib.bibx37" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[37]</span>
<span class="ltx_bibblock">James Martin
</span>
<span class="ltx_bibblock">“Managing the data base environment”
</span>
<span class="ltx_bibblock">Prentice Hall PTR, 1983
</span>
</li>
<li id="bib.bibx38" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[38]</span>
<span class="ltx_bibblock"> Martín Abadi et al.
</span>
<span class="ltx_bibblock">“TensorFlow: Large-Scale Machine Learning on Heterogeneous Systems” Software available from tensorflow.org, 2015
</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://www.tensorflow.org/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.tensorflow.org/</a>
</span>
</li>
<li id="bib.bibx39" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[39]</span>
<span class="ltx_bibblock">H McMahan et al.
</span>
<span class="ltx_bibblock">“Communication-efficient learning of deep networks from decentralized data. arXiv e-prints”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx39.1.1" class="ltx_emph ltx_font_italic">Artificial intelligence and statistics. PMLR.</em>, 2017
</span>
</li>
<li id="bib.bibx40" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[40]</span>
<span class="ltx_bibblock">Bjoern H Menze et al.
</span>
<span class="ltx_bibblock">“The multimodal brain tumor image segmentation benchmark (BRATS)”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx40.1.1" class="ltx_emph ltx_font_italic">IEEE transactions on medical imaging</em> <span id="bib.bibx40.2.2" class="ltx_text ltx_font_bold">34.10</span>
</span>
<span class="ltx_bibblock">IEEE, 2014, pp. 1993–2024
</span>
</li>
<li id="bib.bibx41" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[41]</span>
<span class="ltx_bibblock">Shawn N Murphy et al.
</span>
<span class="ltx_bibblock">“Serving the enterprise and beyond with informatics for integrating biology and the bedside (i2b2)”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx41.1.1" class="ltx_emph ltx_font_italic">Journal of the American Medical Informatics Association</em> <span id="bib.bibx41.2.2" class="ltx_text ltx_font_bold">17.2</span>
</span>
<span class="ltx_bibblock">BMJ Group, 2010, pp. 124–130
</span>
</li>
<li id="bib.bibx42" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[42]</span>
<span class="ltx_bibblock">Mario Mustra, Kresimir Delac and Mislav Grgic
</span>
<span class="ltx_bibblock">“Overview of the DICOM standard”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx42.1.1" class="ltx_emph ltx_font_italic">2008 50th International Symposium ELMAR</em> <span id="bib.bibx42.2.2" class="ltx_text ltx_font_bold">1</span>, 2008, pp. 39–44
</span>
<span class="ltx_bibblock">IEEE
</span>
</li>
<li id="bib.bibx43" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[43]</span>
<span class="ltx_bibblock">Jean Ogier du Terrail et al.
</span>
<span class="ltx_bibblock">“Federated learning for predicting histological response to neoadjuvant chemotherapy in triple-negative breast cancer”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx43.1.1" class="ltx_emph ltx_font_italic">Nature Medicine</em>
</span>
<span class="ltx_bibblock">Nature Publishing Group US New York, 2023, pp. 1–12
</span>
</li>
<li id="bib.bibx44" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[44]</span>
<span class="ltx_bibblock">Roseline Oluwaseun Ogundokun, Sanjay Misra, Rytis Maskeliunas and Robertas Damasevicius
</span>
<span class="ltx_bibblock">“A Review on Federated Learning and Machine Learning Approaches: Categorization, Application Areas, and Blockchain Technology”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx44.1.1" class="ltx_emph ltx_font_italic">Information</em> <span id="bib.bibx44.2.2" class="ltx_text ltx_font_bold">13.5</span>, 2022, pp. 263
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/10.3390/info13050263" title="" class="ltx_ref ltx_href">10.3390/info13050263</a>
</span>
</li>
<li id="bib.bibx45" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[45]</span>
<span class="ltx_bibblock"> Owkin
</span>
<span class="ltx_bibblock">“Healthchain” Accessed on 28th November 2022, 2019
</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://www.labelia.org/fr/healthchain-project" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.labelia.org/fr/healthchain-project</a>
</span>
</li>
<li id="bib.bibx46" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[46]</span>
<span class="ltx_bibblock"> Owkin
</span>
<span class="ltx_bibblock">“Substrafl overview” Accessed on: 28th November 2022, Revision b152eecf, 2022
</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://docs.substra.org/en/stable/substrafl_doc/substrafl_overview.html#" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://docs.substra.org/en/stable/substrafl_doc/substrafl_overview.html#</a>
</span>
</li>
<li id="bib.bibx47" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[47]</span>
<span class="ltx_bibblock">Sarthak Pati et al.
</span>
<span class="ltx_bibblock">“Federated Learning Enables Big Data for Rare Cancer Boundary Detection”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx47.1.1" class="ltx_emph ltx_font_italic">Nature Communication</em>, 2022
</span>
</li>
<li id="bib.bibx48" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[48]</span>
<span class="ltx_bibblock">Nicola Rieke et al.
</span>
<span class="ltx_bibblock">“The future of digital health with federated learning”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx48.1.1" class="ltx_emph ltx_font_italic">NPJ digital medicine</em> <span id="bib.bibx48.2.2" class="ltx_text ltx_font_bold">3.1</span>
</span>
<span class="ltx_bibblock">Nature Publishing Group, 2020, pp. 1–7
</span>
</li>
<li id="bib.bibx49" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[49]</span>
<span class="ltx_bibblock">Holger R Roth et al.
</span>
<span class="ltx_bibblock">“Federated learning for breast density classification: A real-world implementation”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx49.1.1" class="ltx_emph ltx_font_italic">Domain adaptation and representation transfer, and distributed and collaborative learning</em>
</span>
<span class="ltx_bibblock">Springer, 2020, pp. 181–191
</span>
</li>
<li id="bib.bibx50" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[50]</span>
<span class="ltx_bibblock">Holger R Roth et al.
</span>
<span class="ltx_bibblock">“FLARE: Federated Learning from Simulation to Real-World”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx50.1.1" class="ltx_emph ltx_font_italic">Workshop on Federated Learning: Recent Advances and New Challenges (in Conjunction with NeurIPS 2022)</em>
</span>
</li>
<li id="bib.bibx51" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[51]</span>
<span class="ltx_bibblock">Theo Ryffel et al.
</span>
<span class="ltx_bibblock">“A generic framework for privacy preserving deep learning”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx51.1.1" class="ltx_emph ltx_font_italic">arXiv preprint arXiv:1811.04017</em>, 2018
</span>
</li>
<li id="bib.bibx52" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[52]</span>
<span class="ltx_bibblock">Adam Sadilek et al.
</span>
<span class="ltx_bibblock">“Privacy-first health research with federated learning”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx52.1.1" class="ltx_emph ltx_font_italic">NPJ digital medicine</em> <span id="bib.bibx52.2.2" class="ltx_text ltx_font_bold">4.1</span>
</span>
<span class="ltx_bibblock">Nature Publishing Group, 2021, pp. 1–8
</span>
</li>
<li id="bib.bibx53" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[53]</span>
<span class="ltx_bibblock">Oliver Lester Saldanha et al.
</span>
<span class="ltx_bibblock">“Swarm learning for decentralized artificial intelligence in cancer histopathology”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx53.1.1" class="ltx_emph ltx_font_italic">Nature Medicine</em> <span id="bib.bibx53.2.2" class="ltx_text ltx_font_bold">28.6</span>, 2022, pp. 1232–1239
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/10.1038/s41591-022-01768-5" title="" class="ltx_ref ltx_href">10.1038/s41591-022-01768-5</a>
</span>
</li>
<li id="bib.bibx54" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[54]</span>
<span class="ltx_bibblock">Karthik V Sarma et al.
</span>
<span class="ltx_bibblock">“Federated learning improves site performance in multicenter deep learning without data sharing”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx54.1.1" class="ltx_emph ltx_font_italic">Journal of the American Medical Informatics Association</em> <span id="bib.bibx54.2.2" class="ltx_text ltx_font_bold">28.6</span>
</span>
<span class="ltx_bibblock">Oxford University Press, 2021, pp. 1259–1264
</span>
</li>
<li id="bib.bibx55" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[55]</span>
<span class="ltx_bibblock">Observational Health Data Sciences and  Informatics
</span>
<span class="ltx_bibblock">“The Book of OHDSI”
</span>
<span class="ltx_bibblock">Independently published, 2019
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/%5Curl%7Bhttps://doi.org/10.5281/zenodo.4265255" title="" class="ltx_ref ltx_href">“url–https://doi.org/10.5281/zenodo.4265255</a>
</span>
</li>
<li id="bib.bibx56" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[56]</span>
<span class="ltx_bibblock">Momina Shaheen, Muhammad Shoaib Farooq, Tariq Umer and Byung-Seo Kim
</span>
<span class="ltx_bibblock">“Applications of federated learning; Taxonomy, challenges, and research trends”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx56.1.1" class="ltx_emph ltx_font_italic">Electronics</em> <span id="bib.bibx56.2.2" class="ltx_text ltx_font_bold">11.4</span>
</span>
<span class="ltx_bibblock">MDPI, 2022, pp. 670
</span>
</li>
<li id="bib.bibx57" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[57]</span>
<span class="ltx_bibblock">Chi-Ren Shyu et al.
</span>
<span class="ltx_bibblock">“A systematic review of federated learning in the healthcare area: From the perspective of data properties and applications”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx57.1.1" class="ltx_emph ltx_font_italic">Applied Sciences</em> <span id="bib.bibx57.2.2" class="ltx_text ltx_font_bold">11.23</span>
</span>
<span class="ltx_bibblock">MDPI, 2021, pp. 11191
</span>
</li>
<li id="bib.bibx58" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[58]</span>
<span class="ltx_bibblock">Olaf Sporns, Giulio Tononi and Rolf Kötter
</span>
<span class="ltx_bibblock">“The human connectome: a structural description of the human brain”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx58.1.1" class="ltx_emph ltx_font_italic">PLoS computational biology</em> <span id="bib.bibx58.2.2" class="ltx_text ltx_font_bold">1.4</span>
</span>
<span class="ltx_bibblock">Public Library of Science San Francisco, USA, 2005, pp. e42
</span>
</li>
<li id="bib.bibx59" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[59]</span>
<span class="ltx_bibblock">Cathie Sudlow et al.
</span>
<span class="ltx_bibblock">“UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx59.1.1" class="ltx_emph ltx_font_italic">PLoS medicine</em> <span id="bib.bibx59.2.2" class="ltx_text ltx_font_bold">12.3</span>
</span>
<span class="ltx_bibblock">Public Library of Science, 2015, pp. e1001779
</span>
</li>
<li id="bib.bibx60" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[60]</span>
<span class="ltx_bibblock">K. Tomczak, P. Czerwińska and M. Wiznerowicz
</span>
<span class="ltx_bibblock">“The cancer genome atlas (tcga): an immeasurable source of knowledge.”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx60.1.1" class="ltx_emph ltx_font_italic">Contemporary Oncology</em> <span id="bib.bibx60.2.2" class="ltx_text ltx_font_bold">19</span>, 2015
</span>
</li>
<li id="bib.bibx61" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[61]</span>
<span class="ltx_bibblock">Eric J Topol
</span>
<span class="ltx_bibblock">“High-performance medicine: the convergence of human and artificial intelligence”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx61.1.1" class="ltx_emph ltx_font_italic">Nature medicine</em> <span id="bib.bibx61.2.2" class="ltx_text ltx_font_bold">25.1</span>
</span>
<span class="ltx_bibblock">Nature Publishing Group, 2019, pp. 44–56
</span>
</li>
<li id="bib.bibx62" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[62]</span>
<span class="ltx_bibblock">Daniel Truhn et al.
</span>
<span class="ltx_bibblock">“Encrypted federated learning for secure decentralized collaboration in cancer image analysis”
</span>
<span class="ltx_bibblock">medRxiv, 2022
</span>
<span class="ltx_bibblock">DOI: <a target="_blank" href="https://dx.doi.org/10.1101/2022.07.28.22277288" title="" class="ltx_ref ltx_href">10.1101/2022.07.28.22277288</a>
</span>
</li>
<li id="bib.bibx63" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[63]</span>
<span class="ltx_bibblock">Willem G Van Panhuis et al.
</span>
<span class="ltx_bibblock">“A systematic review of barriers to data sharing in public health”
</span>
<span class="ltx_bibblock">In <em id="bib.bibx63.1.1" class="ltx_emph ltx_font_italic">BMC public health</em> <span id="bib.bibx63.2.2" class="ltx_text ltx_font_bold">14.1</span>
</span>
<span class="ltx_bibblock">BioMed Central, 2014, pp. 1–9
</span>
</li>
</ul>
</section>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/2304.12011" class="ar5iv-nav-button ar5iv-nav-button-prev">◄</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/2304.12012" class="ar5iv-text-button ar5iv-severity-error">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+2304.12012">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/2304.12012" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/2304.12013" class="ar5iv-nav-button ar5iv-nav-button-next">►</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Thu Feb 29 12:37:20 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "×";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
