<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[2311.06364] Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/2311.06364">

<!--Generated on Tue Feb 27 19:32:15 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<div id="p1" class="ltx_para">
<span id="p1.1" class="ltx_ERROR undefined">\newcites</span>
<p id="p1.2" class="ltx_p">suppSupplementary References




 









</p>
</div>
<h1 class="ltx_title ltx_title_document">Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Maxime Delmas
</span><span class="ltx_author_notes">Corresponding author: maxime.delmas@idiap.ch
<span class="ltx_contact ltx_role_affiliation">Idiap Research Institute, Switzerland
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Magdalena Wysocka
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Digital Experimental Cancer Medicine Team, Cancer Biomarker Centre, CRUK Manchester Institute
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Andr√© Freitas
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Idiap Research Institute, Switzerland
</span>
<span class="ltx_contact ltx_role_affiliation">Digital Experimental Cancer Medicine Team, Cancer Biomarker Centre, CRUK Manchester Institute
</span>
<span class="ltx_contact ltx_role_affiliation">Department of Computer Science, University of Manchester
</span></span></span>
</div>

<section id="Sx1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Abstract</h2>

<div id="Sx1.p1" class="ltx_para ltx_noindent">
<p id="Sx1.p1.1" class="ltx_p">The sparsity of labelled data is an obstacle to the development of Relation Extraction models and the completion of databases in various biomedical areas. While being of high interest in drug-discovery, the natural-products literature, reporting the identification of potential bioactive compounds from organisms, is a concrete example of such an overlooked topic. To mark the start of this new task, we created the first curated evaluation dataset and extracted literature items from the LOTUS database to build training sets. To this end, we developed a new sampler inspired by diversity metrics in ecology, named Greedy Maximum Entropy sampler, or GME-sampler (<a target="_blank" href="https://github.com/idiap/gme-sampler" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/idiap/gme-sampler</a>). The strategic optimization of both balance and diversity of the selected items in the evaluation set is important given the resource-intensive nature of manual curation. After quantifying the noise in the training set, in the form of discrepancies between the input abstracts text and the expected output labels, we explored different strategies accordingly. Framing the task as an end-to-end Relation Extraction, we evaluated the performance of standard fine-tuning with models such as BioGPT, GPT-2 and Seq2Rel, as a generative task, and few-shot learning with open Large Language Models (LLaMa 7B-65B). Interestingly, the training sets built with the GME-sampler also exhibit a tendency to tip the precision-recall trade-off of trained models in favour of recall. In addition to their evaluation in few-shot settings, we explore the potential of open Large Language Models (Vicuna-13B) as synthetic data generator and propose a new workflow for this purpose. All evaluated models exhibited substantial improvements when fine-tuned on synthetic abstracts rather than the original noisy data. We provide our best performing (<math id="Sx1.p1.1.m1.1" class="ltx_Math" alttext="\text{f1-score}=59.0" display="inline"><semantics id="Sx1.p1.1.m1.1a"><mrow id="Sx1.p1.1.m1.1.1" xref="Sx1.p1.1.m1.1.1.cmml"><mtext id="Sx1.p1.1.m1.1.1.2" xref="Sx1.p1.1.m1.1.1.2a.cmml">f1-score</mtext><mo id="Sx1.p1.1.m1.1.1.1" xref="Sx1.p1.1.m1.1.1.1.cmml">=</mo><mn id="Sx1.p1.1.m1.1.1.3" xref="Sx1.p1.1.m1.1.1.3.cmml">59.0</mn></mrow><annotation-xml encoding="MathML-Content" id="Sx1.p1.1.m1.1b"><apply id="Sx1.p1.1.m1.1.1.cmml" xref="Sx1.p1.1.m1.1.1"><eq id="Sx1.p1.1.m1.1.1.1.cmml" xref="Sx1.p1.1.m1.1.1.1"></eq><ci id="Sx1.p1.1.m1.1.1.2a.cmml" xref="Sx1.p1.1.m1.1.1.2"><mtext id="Sx1.p1.1.m1.1.1.2.cmml" xref="Sx1.p1.1.m1.1.1.2">f1-score</mtext></ci><cn type="float" id="Sx1.p1.1.m1.1.1.3.cmml" xref="Sx1.p1.1.m1.1.1.3">59.0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Sx1.p1.1.m1.1c">\text{f1-score}=59.0</annotation></semantics></math>) BioGPT-Large model for end-to-end RE of natural-products relationships along with all the generated synthetic data and the evaluation dataset. See more details at <a target="_blank" href="https://github.com/idiap/abroad-re" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/idiap/abroad-re</a>.</p>
</div>
</section>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>

<div id="S1.p1" class="ltx_para ltx_noindent">
<p id="S1.p1.1" class="ltx_p">The biomedical literature constitutes a vast but still underexploited reservoir of knowledge, the growth of which reflects the expansion of topics and areas of applications. However, the diversity and morphological richness of bio-entities and the complexity of the relationships expressed between them, contrast with the sparsity of the available labelled data. While some domains can already benefit from efficient extraction models (e.g chemical-disease relationships) for database completion, less popular domains, like the natural product (NP) literature, are often overlooked. NPs are chemical compounds produced by living organisms (plants, bacteria, fungi, etc.), exhibiting a wide range of structure and functions and offering a vast reservoir of potential therapeutic molecules. The isolation and identification of NPs is primarily reported in the scientific literature and also disseminated in different public databases (eg. COCONUT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib1" title="" class="ltx_ref">1</a>]</cite>, KNApSAck <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib2" title="" class="ltx_ref">2</a>]</cite>, etc.). Recently, the LOTUS initiative <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite> has successfully established an Open and FAIR standard resource for natural product chemistry through a rigorous harmonization of a heterogenous set of databases. However, the extent of the NP landscape is not reflected by the content of the databases, which are incomplete and exhibit an imbalanced coverage toward model organisms (eg. <span id="S1.p1.1.1" class="ltx_text ltx_font_italic">A. Thaliana</span>). While a significant portion of the existing literature remains unannotated, there is also a continuous surge of new publications reporting novel relationships that could contribute to filling this gap.</p>
</div>
<div id="S1.p2" class="ltx_para ltx_noindent">
<p id="S1.p2.1" class="ltx_p">Enriching such knowledge bases requires jointly performing Named Entity Recognition (NER) and Relation Extraction (RE). In this case, NER is defined as a sub-task which consists of identifying the boundaries and classifying the type of named entities (i.e an organism "Isaria sinclairii" and a chemical "<span id="S1.p2.1.1" class="ltx_text ltx_font_italic">fingolimod</span>"<span id="footnote1" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_tag ltx_tag_note">1</span><a target="_blank" href="https://lotus.naturalproducts.net/compound/lotus_id/LTS0203935" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://lotus.naturalproducts.net/compound/lotus_id/LTS0203935</a></span></span></span>). The subsequent RE step is the semantic classification of the relations between two (or more) entities. To complete NP databases, the objective is to extract the "<span id="S1.p2.1.2" class="ltx_text ltx_font_italic">produces</span>" or "<span id="S1.p2.1.3" class="ltx_text ltx_font_italic">is isolated from</span>" relationships between organisms and chemicals. Noteworthy, others types of relationships can also be expressed, such as "<span id="S1.p2.1.4" class="ltx_text ltx_font_italic">inhibits the growth of</span>". Traditional deep learning models exhibiting SOTA performance on NER and RE (separately or in so-called <span id="S1.p2.1.5" class="ltx_text ltx_font_italic">end-to-end</span> models) relies on a large set of labelled data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib4" title="" class="ltx_ref">4</a>, <a href="#bib.bib5" title="" class="ltx_ref">5</a>, <a href="#bib.bib6" title="" class="ltx_ref">6</a>]</cite>. However, while datasets like Linneaus <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib7" title="" class="ltx_ref">7</a>]</cite> have been successfully applied for organisms recognition, existing chemical NER datasets (i.e CHEMDNER <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite>) do not provide sufficient coverage on the NP literature and do not adequately capture their morphological specificities. Along with the typically long systematic names of metabolites (eg. <span id="S1.p2.1.6" class="ltx_text ltx_font_typewriter">3‚Äô-[gamma-hydroxymethyl-(E)-gamma-methylallyl]-2,4,2‚Äô,4‚Äô-tetrahydroxychalcone 11‚Äô-O-coumarate<span id="footnote2" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">2</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">2</sup><span class="ltx_tag ltx_tag_note"><span id="footnote2.1.1.1" class="ltx_text ltx_font_serif">2</span></span><span id="footnote2.5" class="ltx_text">PMID: 11678652</span></span></span></span></span>), many chemical mentions are defined as Multiples co-joined enumerations, where entities are mentioned in non-continuous strings such as "cystodiones A-D" or "wortmannins C and D", and are particularly frequent. These chemical mentions must be correctly identified and expanded to recover the full list of entities, which also adds complexity to the decoding process. Finally, to the best of our knowledge, no datasets are available for the subsequent RE step <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib9" title="" class="ltx_ref">9</a>]</cite>. The aforementioned constraints are frequently encountered in BioNLP, when venturing beyond the well-studied chemical-disease associations or protein-protein interactions.</p>
</div>
<div id="S1.p3" class="ltx_para ltx_noindent">
<p id="S1.p3.1" class="ltx_p">Meanwhile, the abundance of unlabelled textual data has been instrumental in driving recent breakthroughs in representation learning <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib10" title="" class="ltx_ref">10</a>]</cite> and the development of the foundational models (e.g GPT and LLaMA model families). The zero/few-shot learning capabilities of Large Language Models (LLMs) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib11" title="" class="ltx_ref">11</a>, <a href="#bib.bib12" title="" class="ltx_ref">12</a>]</cite> make them serious candidates for performing a task with only a handful of examples. Moreover, conversations (chatbots) and instructions-tuned models <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib13" title="" class="ltx_ref">13</a>]</cite> also represent a promising opportunity for synthetic data generation to alleviate the main problem, namely the lack of labelled data within the target distribution. Indeed, beyond the sophistication of model architectures, data availability and quality are limiting factors for the extraction performance, but often neglected <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib14" title="" class="ltx_ref">14</a>]</cite>.</p>
</div>
<div id="S1.p4" class="ltx_para ltx_noindent">
<p id="S1.p4.1" class="ltx_p">In order to address these scarcity constraints, we propose an end-to-end generative extraction paradigm, which introduces two novel methodological contributions. Firstly, we introduce a diversity-optimised sampling strategy, which minimises the selection of items for the parsimonious creation of evaluation gold-standards and training sets. This component minimises the popularity biases and associated imbalance towards entities which are over-expressed in the literature (e.g. model organisms and recurring substances), allowing for a systematic (entropy-based) method to maximise diversity and measure the utility of new annotations. Secondly, we use the generative expressivity of models fine-tuned on conversations and instructions for creating within distribution synthetic data, to support the construction of end-to-end joint NER-RE extraction models. In this framework, the diversity sampled entities and associated relations are linguistically embedded within synthetically generated text. The overall framework is depicted in Figure <a href="#S1.F1" title="Figure 1 ‚Ä£ 1 Introduction ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.</p>
</div>
<div id="S1.p5" class="ltx_para ltx_noindent">
<p id="S1.p5.1" class="ltx_p">More formally, this paper aims to investigate the following research hypotheses (RHs) as supporting mechanisms for addressing these limitations, using NPs as a validation domain:</p>
</div>
<div id="S1.p6" class="ltx_para ltx_noindent">
<ul id="S1.I1" class="ltx_itemize">
<li id="S1.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">‚Ä¢</span> 
<div id="S1.I1.i1.p1" class="ltx_para">
<p id="S1.I1.i1.p1.1" class="ltx_p"><span id="S1.I1.i1.p1.1.1" class="ltx_text ltx_font_bold">RH1: Diversity-optimised sampling provide a valuable selection of items to build training and evaluation datasets for RE.</span></p>
</div>
</li>
<li id="S1.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">‚Ä¢</span> 
<div id="S1.I1.i2.p1" class="ltx_para ltx_noindent">
<p id="S1.I1.i2.p1.1" class="ltx_p"><span id="S1.I1.i2.p1.1.1" class="ltx_text ltx_font_bold">RH2: In a practical scenario with noisy labels, LLMs can be more beneficial as synthetic data generator than unsupervised predictors.</span></p>
</div>
</li>
</ul>
</div>
<figure id="S1.F1" class="ltx_figure"><img src="/html/2311.06364/assets/Figures/Introduction/schema-hypothesis.png" id="S1.F1.1.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="120" height="170" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>Combining diversity-optimised sampling and synthetic data generation.</figcaption>
</figure>
<div id="S1.p7" class="ltx_para ltx_noindent">
<p id="S1.p7.4" class="ltx_p">With the aim of providing an end-to-end RE model to help expanding natural products databases, we started by building a training and evaluation dataset. Inspired by the metrics used in ecology, we first proposed the Greedy-Maximum-Entropy sampler (GME-sampler) to extract a diversity-optimised sample from the LOTUS database. By manually annotating the top-diverse items, we proposed the first evaluation dataset for this task, which can serve as a benchmark for future developments in this area. Following a descriptive analysis of the remaining data and quantifying the noise present in the form of discrepancies between raw input text and annotated (standardized) labels, we evaluated various modelling approaches. First, we compared the performance of standard fine-tuning techniques on the available noisy data to the few-shot learning capabilities of open LLMs. Leveraging the generative capabilities of a LLM (Vicuna-13B), we then proposed a novel synthetic abstract generation pipeline and demonstrated the significant performance improvements (on average <math id="S1.p7.1.m1.1" class="ltx_Math" alttext="24.7\%" display="inline"><semantics id="S1.p7.1.m1.1a"><mrow id="S1.p7.1.m1.1.1" xref="S1.p7.1.m1.1.1.cmml"><mn id="S1.p7.1.m1.1.1.2" xref="S1.p7.1.m1.1.1.2.cmml">24.7</mn><mo id="S1.p7.1.m1.1.1.1" xref="S1.p7.1.m1.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="S1.p7.1.m1.1b"><apply id="S1.p7.1.m1.1.1.cmml" xref="S1.p7.1.m1.1.1"><csymbol cd="latexml" id="S1.p7.1.m1.1.1.1.cmml" xref="S1.p7.1.m1.1.1.1">percent</csymbol><cn type="float" id="S1.p7.1.m1.1.1.2.cmml" xref="S1.p7.1.m1.1.1.2">24.7</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.p7.1.m1.1c">24.7\%</annotation></semantics></math> in <span id="S1.p7.4.1" class="ltx_text ltx_markedasmath">f1-score</span>) brought by these new training data on the evaluated models. In line with these results, we have made available our best-performing BioGPT-Large model (<math id="S1.p7.3.m3.1" class="ltx_Math" alttext="\text{f1-score}=59.0" display="inline"><semantics id="S1.p7.3.m3.1a"><mrow id="S1.p7.3.m3.1.1" xref="S1.p7.3.m3.1.1.cmml"><mtext id="S1.p7.3.m3.1.1.2" xref="S1.p7.3.m3.1.1.2a.cmml">f1-score</mtext><mo id="S1.p7.3.m3.1.1.1" xref="S1.p7.3.m3.1.1.1.cmml">=</mo><mn id="S1.p7.3.m3.1.1.3" xref="S1.p7.3.m3.1.1.3.cmml">59.0</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.p7.3.m3.1b"><apply id="S1.p7.3.m3.1.1.cmml" xref="S1.p7.3.m3.1.1"><eq id="S1.p7.3.m3.1.1.1.cmml" xref="S1.p7.3.m3.1.1.1"></eq><ci id="S1.p7.3.m3.1.1.2a.cmml" xref="S1.p7.3.m3.1.1.2"><mtext id="S1.p7.3.m3.1.1.2.cmml" xref="S1.p7.3.m3.1.1.2">f1-score</mtext></ci><cn type="float" id="S1.p7.3.m3.1.1.3.cmml" xref="S1.p7.3.m3.1.1.3">59.0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.p7.3.m3.1c">\text{f1-score}=59.0</annotation></semantics></math>) and the <math id="S1.p7.4.m4.1" class="ltx_Math" alttext="\approx 25000" display="inline"><semantics id="S1.p7.4.m4.1a"><mrow id="S1.p7.4.m4.1.1" xref="S1.p7.4.m4.1.1.cmml"><mi id="S1.p7.4.m4.1.1.2" xref="S1.p7.4.m4.1.1.2.cmml"></mi><mo id="S1.p7.4.m4.1.1.1" xref="S1.p7.4.m4.1.1.1.cmml">‚âà</mo><mn id="S1.p7.4.m4.1.1.3" xref="S1.p7.4.m4.1.1.3.cmml">25000</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.p7.4.m4.1b"><apply id="S1.p7.4.m4.1.1.cmml" xref="S1.p7.4.m4.1.1"><approx id="S1.p7.4.m4.1.1.1.cmml" xref="S1.p7.4.m4.1.1.1"></approx><csymbol cd="latexml" id="S1.p7.4.m4.1.1.2.cmml" xref="S1.p7.4.m4.1.1.2">absent</csymbol><cn type="integer" id="S1.p7.4.m4.1.1.3.cmml" xref="S1.p7.4.m4.1.1.3">25000</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.p7.4.m4.1c">\approx 25000</annotation></semantics></math> synthetic abstracts on which it has been trained. A synthetic diagram of the different strategies explored in this work is presented in Figure <a href="#S1.F2" title="Figure 2 ‚Ä£ 1 Introduction ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>. The main contributions of the work are summarised below:</p>
<ul id="S1.I2" class="ltx_itemize">
<li id="S1.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">‚Ä¢</span> 
<div id="S1.I2.i1.p1" class="ltx_para">
<p id="S1.I2.i1.p1.1" class="ltx_p">A diversity-optimised sampler (GME-sampler) for building diverse and balanced datasets for RE (see <a target="_blank" href="https://github.com/idiap/gme-sampler" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/idiap/gme-sampler</a>).</p>
</div>
</li>
<li id="S1.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">‚Ä¢</span> 
<div id="S1.I2.i2.p1" class="ltx_para">
<p id="S1.I2.i2.p1.1" class="ltx_p">The first curated evaluation dataset for RE between organisms and natural-products (see <a target="_blank" href="https://zenodo.org/records/8422007" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://zenodo.org/records/8422007</a>).</p>
</div>
</li>
<li id="S1.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">‚Ä¢</span> 
<div id="S1.I2.i3.p1" class="ltx_para">
<p id="S1.I2.i3.p1.1" class="ltx_p">An evaluation of different strategies for RE with noisy labels.</p>
</div>
</li>
<li id="S1.I2.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">‚Ä¢</span> 
<div id="S1.I2.i4.p1" class="ltx_para">
<p id="S1.I2.i4.p1.1" class="ltx_p">A framework for synthetic data generation via chatbot or instruction-tuned models and produced training datasets (see <a target="_blank" href="https://github.com/idiap/abroad-re" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/idiap/abroad-re</a> and <a target="_blank" href="https://zenodo.org/records/8422294" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://zenodo.org/records/8422294</a>).</p>
</div>
</li>
<li id="S1.I2.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">‚Ä¢</span> 
<div id="S1.I2.i5.p1" class="ltx_para ltx_noindent">
<p id="S1.I2.i5.p1.1" class="ltx_p">A set of ready-to-use BioGPT fine-tuned models (see <a target="_blank" href="https://huggingface.co/mdelmas/BioGPT-Large-Natural-Products-RE-Diversity-synt-v1.0" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://huggingface.co/mdelmas/BioGPT-Large-Natural-Products-RE-Diversity-synt-v1.0</a>)</p>
</div>
</li>
</ul>
</div>
<figure id="S1.F2" class="ltx_figure"><img src="/html/2311.06364/assets/Figures/Introduction/workflow-diagram.png" id="S1.F2.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="479" height="345" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>A global diagram of the workflow presented in this work.</figcaption>
</figure>
<section id="S1.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.1 </span>Related work</h3>

<div id="S1.SS1.p1" class="ltx_para ltx_noindent">
<p id="S1.SS1.p1.1" class="ltx_p">Biomedical RE <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib15" title="" class="ltx_ref">15</a>, <a href="#bib.bib16" title="" class="ltx_ref">16</a>, <a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite> encompasses various subtypes, depending on the considered bio-entities, such as drug-drug interactions <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib18" title="" class="ltx_ref">18</a>]</cite>, chemical-disease relationships <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib19" title="" class="ltx_ref">19</a>]</cite>, gene-disease associations <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib20" title="" class="ltx_ref">20</a>]</cite>, and protein-protein interactions <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib21" title="" class="ltx_ref">21</a>]</cite>, among the most popular. Investigating the overlooked NPs relationships necessitated the exploration of several interconnected sub-tasks, including the selection and partitioning of a dataset, the generation of synthetic data and the assessment of various end-to-end RE strategies. This section provides a review of the closely-related works that align with these three development axes.</p>
</div>
<section id="S1.SS1.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">1.1.1 </span>Splitting datasets and impact of diversity</h4>

<div id="S1.SS1.SSS1.p1" class="ltx_para ltx_noindent">
<p id="S1.SS1.SSS1.p1.1" class="ltx_p">Data selection and partitioning methods can significantly impact the generalization performance of supervised models. <cite class="ltx_cite ltx_citemacro_citet">Xu and Goodacre [<a href="#bib.bib22" title="" class="ltx_ref">22</a>]</cite> evaluated various splitting techniques, including K-S <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib23" title="" class="ltx_ref">23</a>]</cite> and SPXY <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib24" title="" class="ltx_ref">24</a>]</cite>, and emphasized the importance of maintaining a balance between training and test sets for a reliable evaluation of models. Like the recently proposed SPlit <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib25" title="" class="ltx_ref">25</a>]</cite> method, these approaches aim to select a representative subset of the data, leveraging different distance metrics. Unlike the Euclidean or Energy-based distances used in aforementioned methods, the GME-sampler uses an entropy-based metric to capture diversity and select representative evaluation and training sets.
Regarding diversity, <cite class="ltx_cite ltx_citemacro_citet">Yu et¬†al. [<a href="#bib.bib26" title="" class="ltx_ref">26</a>]</cite> investigated various diversity-based metrics for selecting training data, and demonstrated their positive impact on the performance of NER models. Additionally, other works have highlighted the significance of effective data selection over a naive increase of the dataset size for training <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib27" title="" class="ltx_ref">27</a>, <a href="#bib.bib28" title="" class="ltx_ref">28</a>, <a href="#bib.bib29" title="" class="ltx_ref">29</a>]</cite>.</p>
</div>
</section>
<section id="S1.SS1.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">1.1.2 </span>Synthetic data generation</h4>

<div id="S1.SS1.SSS2.p1" class="ltx_para ltx_noindent">
<p id="S1.SS1.SSS2.p1.1" class="ltx_p">Training neural RE models strongly rely on a substantial and diverse set of training data. However, annotating large datasets with experts is time-consuming and costly. To overcome this limitation, many works explored approaches such as Data-augmentation (DA) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib30" title="" class="ltx_ref">30</a>, <a href="#bib.bib31" title="" class="ltx_ref">31</a>, <a href="#bib.bib32" title="" class="ltx_ref">32</a>]</cite> and Distant Supervision (DS) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib33" title="" class="ltx_ref">33</a>, <a href="#bib.bib34" title="" class="ltx_ref">34</a>]</cite>, which enable the expansion of the dataset size by creating new training examples from existing ones, or, by assigning pseudo-labels to external, unlabelled data. In the biomedical domain, the RE challenge ChemProt <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib35" title="" class="ltx_ref">35</a>, <a href="#bib.bib36" title="" class="ltx_ref">36</a>]</cite> or Protein-Protein interactions extraction <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>, have recently benefited from the application of these methods. Synthetic data generation (SDG) goes beyond DA or DS by creating fully synthetic datasets, i.e paired input text and output labels. A significant body of influential works has leveraged the generative capabilities of LLMs to propose different SDG strategies in zero-shot <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>, <a href="#bib.bib39" title="" class="ltx_ref">39</a>, <a href="#bib.bib40" title="" class="ltx_ref">40</a>, <a href="#bib.bib41" title="" class="ltx_ref">41</a>, <a href="#bib.bib42" title="" class="ltx_ref">42</a>, <a href="#bib.bib43" title="" class="ltx_ref">43</a>, <a href="#bib.bib44" title="" class="ltx_ref">44</a>, <a href="#bib.bib45" title="" class="ltx_ref">45</a>]</cite>, few-shot <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib46" title="" class="ltx_ref">46</a>, <a href="#bib.bib47" title="" class="ltx_ref">47</a>, <a href="#bib.bib48" title="" class="ltx_ref">48</a>, <a href="#bib.bib49" title="" class="ltx_ref">49</a>, <a href="#bib.bib50" title="" class="ltx_ref">50</a>]</cite>, or by fine-tuning <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib51" title="" class="ltx_ref">51</a>, <a href="#bib.bib52" title="" class="ltx_ref">52</a>, <a href="#bib.bib53" title="" class="ltx_ref">53</a>]</cite>. Similarly to this work, <cite class="ltx_cite ltx_citemacro_citet">Josifoski et¬†al. [<a href="#bib.bib54" title="" class="ltx_ref">54</a>]</cite> also proposed to reverse the task and used LLMs from OpenAI to generate plausible input text based on expected output triplets from Wikidata. <cite class="ltx_cite ltx_citemacro_citet">Tang et¬†al. [<a href="#bib.bib55" title="" class="ltx_ref">55</a>]</cite> compared the performance of LLM (ChatGPT) in directly extracting information from unstructured clinical text to its potential use as synthetic data generator for DA. <cite class="ltx_cite ltx_citemacro_citet">Veselovsky et¬†al. [<a href="#bib.bib56" title="" class="ltx_ref">56</a>]</cite> evaluated various prompting strategies to improve diversity and alignment between synthetic and real-world data distributions for sarcasm detection. <cite class="ltx_cite ltx_citemacro_citet">Yang et¬†al. [<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> combined synthetic data generation with a diversity-augmentation component for common sense reasoning. <cite class="ltx_cite ltx_citemacro_citet">Aggarwal et¬†al. [<a href="#bib.bib58" title="" class="ltx_ref">58</a>]</cite> applied SDG to biomedical NER, while <cite class="ltx_cite ltx_citemacro_citet">Xu et¬†al. [<a href="#bib.bib59" title="" class="ltx_ref">59</a>]</cite> used a two-stage training procedure on synthetic and golden data, notably for extracting protein interactions with the ChemProt dataset. In contrast, this work proposes to leverage Open LLMs to generate synthetic abstracts based on a list of verbalised main findings. The diversity of the generations is increased and guided by the entropy-based sampling of the seed articles which originally report these findings, as well as a set of crafted patterns of expressions.</p>
</div>
</section>
<section id="S1.SS1.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">1.1.3 </span>End-to-End Relation Extraction</h4>

<div id="S1.SS1.SSS3.p1" class="ltx_para ltx_noindent">
<p id="S1.SS1.SSS3.p1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_citet">Kambar et¬†al. [<a href="#bib.bib60" title="" class="ltx_ref">60</a>]</cite> classifies various strategies and highlights the potential of end-to-end (or joint) NER and RE methods to overcome limitations of the traditional pipeline approaches. In the biomedical domain, <cite class="ltx_cite ltx_citemacro_citet">Li et¬†al. [<a href="#bib.bib61" title="" class="ltx_ref">61</a>]</cite> proposed a Bi-LSTM for drug adverse effects extraction, while <cite class="ltx_cite ltx_citemacro_citet">Esmail Zadeh Nojoo¬†Kambar et¬†al. [<a href="#bib.bib62" title="" class="ltx_ref">62</a>]</cite> introduced a GNN for chemical-protein interactions. Recent approaches frames the task in a generative "text-to-text" process, using sequence-to-sequence models, by lineralising the expected relations as a text string to be decoded from the input. Seq2Rel <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite> and REBEL <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib63" title="" class="ltx_ref">63</a>]</cite> proposed different linearisation schemas, and <cite class="ltx_cite ltx_citemacro_citet">Zhang et¬†al. [<a href="#bib.bib64" title="" class="ltx_ref">64</a>], Zeng et¬†al. [<a href="#bib.bib65" title="" class="ltx_ref">65</a>]</cite> notably assessed the biases caused by the forced order of relationships during training. <cite class="ltx_cite ltx_citemacro_citet">Hou et¬†al. [<a href="#bib.bib66" title="" class="ltx_ref">66</a>]</cite> trained a sequence-to-sequence model for drug-target interactions extraction, and <cite class="ltx_cite ltx_citemacro_citet">Zeng et¬†al. [<a href="#bib.bib67" title="" class="ltx_ref">67</a>]</cite> introduced a copy mechanism. Additionally, <cite class="ltx_cite ltx_citemacro_citet">Eberts and Ulges [<a href="#bib.bib68" title="" class="ltx_ref">68</a>]</cite> used four task-specific sub-components, and <cite class="ltx_cite ltx_citemacro_citet">Paolini et¬†al. [<a href="#bib.bib69" title="" class="ltx_ref">69</a>]</cite> utilized a translation mechanism. Finally, BioGPT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib4" title="" class="ltx_ref">4</a>]</cite> demonstrated SOTA performance on several biomedical datasets using an autoregressive approach, providing the input text as context.</p>
</div>
</section>
</section>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Proposed approach</h2>

<div id="S2.p1" class="ltx_para ltx_noindent">
<p id="S2.p1.1" class="ltx_p">This section describes the different methodology used in this work. We start by describing our first contribution, the GME-sampler, in section <a href="#S2.SS1" title="2.1 Greedy Maximum Entropy Sampling (GME) ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.1</span></a>. The few-shot learning and fine-tuning strategies evaluated for the RE task are then described in sections <a href="#S2.SS2.SSS1" title="2.2.1 Few-shot learning with open LLM ‚Ä£ 2.2 Different strategies for Relation Extraction ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.2.1</span></a> and <a href="#S2.SS2.SSS2" title="2.2.2 Fine-tuning ‚Ä£ 2.2 Different strategies for Relation Extraction ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.2.2</span></a>. The synthetic abstract generation procedure is described in <a href="#S2.SS3" title="2.3 Synthetic abstracts generation ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.3</span></a>. Finally, details on the evaluation, experimental setup and implementation details are provided in supplementary <a href="#S1a" title="S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S1</span></a>.</p>
</div>
<section id="S2.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.1 </span>Greedy Maximum Entropy Sampling (GME)</h3>

<div id="S2.SS1.p1" class="ltx_para ltx_noindent">
<p id="S2.SS1.p1.18" class="ltx_p">The objective is to extract a sample <math id="S2.SS1.p1.1.m1.1" class="ltx_Math" alttext="S" display="inline"><semantics id="S2.SS1.p1.1.m1.1a"><mi id="S2.SS1.p1.1.m1.1.1" xref="S2.SS1.p1.1.m1.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.1.m1.1b"><ci id="S2.SS1.p1.1.m1.1.1.cmml" xref="S2.SS1.p1.1.m1.1.1">ùëÜ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.1.m1.1c">S</annotation></semantics></math> of documents from an initial set <math id="S2.SS1.p1.2.m2.1" class="ltx_Math" alttext="D" display="inline"><semantics id="S2.SS1.p1.2.m2.1a"><mi id="S2.SS1.p1.2.m2.1.1" xref="S2.SS1.p1.2.m2.1.1.cmml">D</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.2.m2.1b"><ci id="S2.SS1.p1.2.m2.1.1.cmml" xref="S2.SS1.p1.2.m2.1.1">ùê∑</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.2.m2.1c">D</annotation></semantics></math> with an optimized diversity of mentioned organisms and chemicals: <math id="S2.SS1.p1.3.m3.1" class="ltx_Math" alttext="S\subset D" display="inline"><semantics id="S2.SS1.p1.3.m3.1a"><mrow id="S2.SS1.p1.3.m3.1.1" xref="S2.SS1.p1.3.m3.1.1.cmml"><mi id="S2.SS1.p1.3.m3.1.1.2" xref="S2.SS1.p1.3.m3.1.1.2.cmml">S</mi><mo id="S2.SS1.p1.3.m3.1.1.1" xref="S2.SS1.p1.3.m3.1.1.1.cmml">‚äÇ</mo><mi id="S2.SS1.p1.3.m3.1.1.3" xref="S2.SS1.p1.3.m3.1.1.3.cmml">D</mi></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.3.m3.1b"><apply id="S2.SS1.p1.3.m3.1.1.cmml" xref="S2.SS1.p1.3.m3.1.1"><subset id="S2.SS1.p1.3.m3.1.1.1.cmml" xref="S2.SS1.p1.3.m3.1.1.1"></subset><ci id="S2.SS1.p1.3.m3.1.1.2.cmml" xref="S2.SS1.p1.3.m3.1.1.2">ùëÜ</ci><ci id="S2.SS1.p1.3.m3.1.1.3.cmml" xref="S2.SS1.p1.3.m3.1.1.3">ùê∑</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.3.m3.1c">S\subset D</annotation></semantics></math>, <math id="S2.SS1.p1.4.m4.1" class="ltx_Math" alttext="|S|=l" display="inline"><semantics id="S2.SS1.p1.4.m4.1a"><mrow id="S2.SS1.p1.4.m4.1.2" xref="S2.SS1.p1.4.m4.1.2.cmml"><mrow id="S2.SS1.p1.4.m4.1.2.2.2" xref="S2.SS1.p1.4.m4.1.2.2.1.cmml"><mo stretchy="false" id="S2.SS1.p1.4.m4.1.2.2.2.1" xref="S2.SS1.p1.4.m4.1.2.2.1.1.cmml">|</mo><mi id="S2.SS1.p1.4.m4.1.1" xref="S2.SS1.p1.4.m4.1.1.cmml">S</mi><mo stretchy="false" id="S2.SS1.p1.4.m4.1.2.2.2.2" xref="S2.SS1.p1.4.m4.1.2.2.1.1.cmml">|</mo></mrow><mo id="S2.SS1.p1.4.m4.1.2.1" xref="S2.SS1.p1.4.m4.1.2.1.cmml">=</mo><mi id="S2.SS1.p1.4.m4.1.2.3" xref="S2.SS1.p1.4.m4.1.2.3.cmml">l</mi></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.4.m4.1b"><apply id="S2.SS1.p1.4.m4.1.2.cmml" xref="S2.SS1.p1.4.m4.1.2"><eq id="S2.SS1.p1.4.m4.1.2.1.cmml" xref="S2.SS1.p1.4.m4.1.2.1"></eq><apply id="S2.SS1.p1.4.m4.1.2.2.1.cmml" xref="S2.SS1.p1.4.m4.1.2.2.2"><abs id="S2.SS1.p1.4.m4.1.2.2.1.1.cmml" xref="S2.SS1.p1.4.m4.1.2.2.2.1"></abs><ci id="S2.SS1.p1.4.m4.1.1.cmml" xref="S2.SS1.p1.4.m4.1.1">ùëÜ</ci></apply><ci id="S2.SS1.p1.4.m4.1.2.3.cmml" xref="S2.SS1.p1.4.m4.1.2.3">ùëô</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.4.m4.1c">|S|=l</annotation></semantics></math> and <math id="S2.SS1.p1.5.m5.1" class="ltx_Math" alttext="|D|=L" display="inline"><semantics id="S2.SS1.p1.5.m5.1a"><mrow id="S2.SS1.p1.5.m5.1.2" xref="S2.SS1.p1.5.m5.1.2.cmml"><mrow id="S2.SS1.p1.5.m5.1.2.2.2" xref="S2.SS1.p1.5.m5.1.2.2.1.cmml"><mo stretchy="false" id="S2.SS1.p1.5.m5.1.2.2.2.1" xref="S2.SS1.p1.5.m5.1.2.2.1.1.cmml">|</mo><mi id="S2.SS1.p1.5.m5.1.1" xref="S2.SS1.p1.5.m5.1.1.cmml">D</mi><mo stretchy="false" id="S2.SS1.p1.5.m5.1.2.2.2.2" xref="S2.SS1.p1.5.m5.1.2.2.1.1.cmml">|</mo></mrow><mo id="S2.SS1.p1.5.m5.1.2.1" xref="S2.SS1.p1.5.m5.1.2.1.cmml">=</mo><mi id="S2.SS1.p1.5.m5.1.2.3" xref="S2.SS1.p1.5.m5.1.2.3.cmml">L</mi></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.5.m5.1b"><apply id="S2.SS1.p1.5.m5.1.2.cmml" xref="S2.SS1.p1.5.m5.1.2"><eq id="S2.SS1.p1.5.m5.1.2.1.cmml" xref="S2.SS1.p1.5.m5.1.2.1"></eq><apply id="S2.SS1.p1.5.m5.1.2.2.1.cmml" xref="S2.SS1.p1.5.m5.1.2.2.2"><abs id="S2.SS1.p1.5.m5.1.2.2.1.1.cmml" xref="S2.SS1.p1.5.m5.1.2.2.2.1"></abs><ci id="S2.SS1.p1.5.m5.1.1.cmml" xref="S2.SS1.p1.5.m5.1.1">ùê∑</ci></apply><ci id="S2.SS1.p1.5.m5.1.2.3.cmml" xref="S2.SS1.p1.5.m5.1.2.3">ùêø</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.5.m5.1c">|D|=L</annotation></semantics></math>. The initial set <math id="S2.SS1.p1.6.m6.1" class="ltx_Math" alttext="D" display="inline"><semantics id="S2.SS1.p1.6.m6.1a"><mi id="S2.SS1.p1.6.m6.1.1" xref="S2.SS1.p1.6.m6.1.1.cmml">D</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.6.m6.1b"><ci id="S2.SS1.p1.6.m6.1.1.cmml" xref="S2.SS1.p1.6.m6.1.1">ùê∑</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.6.m6.1c">D</annotation></semantics></math> correspond to the LOTUS dataset, in which each document <math id="S2.SS1.p1.7.m7.1" class="ltx_Math" alttext="d" display="inline"><semantics id="S2.SS1.p1.7.m7.1a"><mi id="S2.SS1.p1.7.m7.1.1" xref="S2.SS1.p1.7.m7.1.1.cmml">d</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.7.m7.1b"><ci id="S2.SS1.p1.7.m7.1.1.cmml" xref="S2.SS1.p1.7.m7.1.1">ùëë</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.7.m7.1c">d</annotation></semantics></math> reports a set of relations between organism(s) and isolated natural product(s): <math id="S2.SS1.p1.8.m8.4" class="ltx_Math" alttext="d=\{r_{1},r_{2},...,r_{n_{d}}\}" display="inline"><semantics id="S2.SS1.p1.8.m8.4a"><mrow id="S2.SS1.p1.8.m8.4.4" xref="S2.SS1.p1.8.m8.4.4.cmml"><mi id="S2.SS1.p1.8.m8.4.4.5" xref="S2.SS1.p1.8.m8.4.4.5.cmml">d</mi><mo id="S2.SS1.p1.8.m8.4.4.4" xref="S2.SS1.p1.8.m8.4.4.4.cmml">=</mo><mrow id="S2.SS1.p1.8.m8.4.4.3.3" xref="S2.SS1.p1.8.m8.4.4.3.4.cmml"><mo stretchy="false" id="S2.SS1.p1.8.m8.4.4.3.3.4" xref="S2.SS1.p1.8.m8.4.4.3.4.cmml">{</mo><msub id="S2.SS1.p1.8.m8.2.2.1.1.1" xref="S2.SS1.p1.8.m8.2.2.1.1.1.cmml"><mi id="S2.SS1.p1.8.m8.2.2.1.1.1.2" xref="S2.SS1.p1.8.m8.2.2.1.1.1.2.cmml">r</mi><mn id="S2.SS1.p1.8.m8.2.2.1.1.1.3" xref="S2.SS1.p1.8.m8.2.2.1.1.1.3.cmml">1</mn></msub><mo id="S2.SS1.p1.8.m8.4.4.3.3.5" xref="S2.SS1.p1.8.m8.4.4.3.4.cmml">,</mo><msub id="S2.SS1.p1.8.m8.3.3.2.2.2" xref="S2.SS1.p1.8.m8.3.3.2.2.2.cmml"><mi id="S2.SS1.p1.8.m8.3.3.2.2.2.2" xref="S2.SS1.p1.8.m8.3.3.2.2.2.2.cmml">r</mi><mn id="S2.SS1.p1.8.m8.3.3.2.2.2.3" xref="S2.SS1.p1.8.m8.3.3.2.2.2.3.cmml">2</mn></msub><mo id="S2.SS1.p1.8.m8.4.4.3.3.6" xref="S2.SS1.p1.8.m8.4.4.3.4.cmml">,</mo><mi mathvariant="normal" id="S2.SS1.p1.8.m8.1.1" xref="S2.SS1.p1.8.m8.1.1.cmml">‚Ä¶</mi><mo id="S2.SS1.p1.8.m8.4.4.3.3.7" xref="S2.SS1.p1.8.m8.4.4.3.4.cmml">,</mo><msub id="S2.SS1.p1.8.m8.4.4.3.3.3" xref="S2.SS1.p1.8.m8.4.4.3.3.3.cmml"><mi id="S2.SS1.p1.8.m8.4.4.3.3.3.2" xref="S2.SS1.p1.8.m8.4.4.3.3.3.2.cmml">r</mi><msub id="S2.SS1.p1.8.m8.4.4.3.3.3.3" xref="S2.SS1.p1.8.m8.4.4.3.3.3.3.cmml"><mi id="S2.SS1.p1.8.m8.4.4.3.3.3.3.2" xref="S2.SS1.p1.8.m8.4.4.3.3.3.3.2.cmml">n</mi><mi id="S2.SS1.p1.8.m8.4.4.3.3.3.3.3" xref="S2.SS1.p1.8.m8.4.4.3.3.3.3.3.cmml">d</mi></msub></msub><mo stretchy="false" id="S2.SS1.p1.8.m8.4.4.3.3.8" xref="S2.SS1.p1.8.m8.4.4.3.4.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.8.m8.4b"><apply id="S2.SS1.p1.8.m8.4.4.cmml" xref="S2.SS1.p1.8.m8.4.4"><eq id="S2.SS1.p1.8.m8.4.4.4.cmml" xref="S2.SS1.p1.8.m8.4.4.4"></eq><ci id="S2.SS1.p1.8.m8.4.4.5.cmml" xref="S2.SS1.p1.8.m8.4.4.5">ùëë</ci><set id="S2.SS1.p1.8.m8.4.4.3.4.cmml" xref="S2.SS1.p1.8.m8.4.4.3.3"><apply id="S2.SS1.p1.8.m8.2.2.1.1.1.cmml" xref="S2.SS1.p1.8.m8.2.2.1.1.1"><csymbol cd="ambiguous" id="S2.SS1.p1.8.m8.2.2.1.1.1.1.cmml" xref="S2.SS1.p1.8.m8.2.2.1.1.1">subscript</csymbol><ci id="S2.SS1.p1.8.m8.2.2.1.1.1.2.cmml" xref="S2.SS1.p1.8.m8.2.2.1.1.1.2">ùëü</ci><cn type="integer" id="S2.SS1.p1.8.m8.2.2.1.1.1.3.cmml" xref="S2.SS1.p1.8.m8.2.2.1.1.1.3">1</cn></apply><apply id="S2.SS1.p1.8.m8.3.3.2.2.2.cmml" xref="S2.SS1.p1.8.m8.3.3.2.2.2"><csymbol cd="ambiguous" id="S2.SS1.p1.8.m8.3.3.2.2.2.1.cmml" xref="S2.SS1.p1.8.m8.3.3.2.2.2">subscript</csymbol><ci id="S2.SS1.p1.8.m8.3.3.2.2.2.2.cmml" xref="S2.SS1.p1.8.m8.3.3.2.2.2.2">ùëü</ci><cn type="integer" id="S2.SS1.p1.8.m8.3.3.2.2.2.3.cmml" xref="S2.SS1.p1.8.m8.3.3.2.2.2.3">2</cn></apply><ci id="S2.SS1.p1.8.m8.1.1.cmml" xref="S2.SS1.p1.8.m8.1.1">‚Ä¶</ci><apply id="S2.SS1.p1.8.m8.4.4.3.3.3.cmml" xref="S2.SS1.p1.8.m8.4.4.3.3.3"><csymbol cd="ambiguous" id="S2.SS1.p1.8.m8.4.4.3.3.3.1.cmml" xref="S2.SS1.p1.8.m8.4.4.3.3.3">subscript</csymbol><ci id="S2.SS1.p1.8.m8.4.4.3.3.3.2.cmml" xref="S2.SS1.p1.8.m8.4.4.3.3.3.2">ùëü</ci><apply id="S2.SS1.p1.8.m8.4.4.3.3.3.3.cmml" xref="S2.SS1.p1.8.m8.4.4.3.3.3.3"><csymbol cd="ambiguous" id="S2.SS1.p1.8.m8.4.4.3.3.3.3.1.cmml" xref="S2.SS1.p1.8.m8.4.4.3.3.3.3">subscript</csymbol><ci id="S2.SS1.p1.8.m8.4.4.3.3.3.3.2.cmml" xref="S2.SS1.p1.8.m8.4.4.3.3.3.3.2">ùëõ</ci><ci id="S2.SS1.p1.8.m8.4.4.3.3.3.3.3.cmml" xref="S2.SS1.p1.8.m8.4.4.3.3.3.3.3">ùëë</ci></apply></apply></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.8.m8.4c">d=\{r_{1},r_{2},...,r_{n_{d}}\}</annotation></semantics></math>, where <math id="S2.SS1.p1.9.m9.1" class="ltx_Math" alttext="n_{d}" display="inline"><semantics id="S2.SS1.p1.9.m9.1a"><msub id="S2.SS1.p1.9.m9.1.1" xref="S2.SS1.p1.9.m9.1.1.cmml"><mi id="S2.SS1.p1.9.m9.1.1.2" xref="S2.SS1.p1.9.m9.1.1.2.cmml">n</mi><mi id="S2.SS1.p1.9.m9.1.1.3" xref="S2.SS1.p1.9.m9.1.1.3.cmml">d</mi></msub><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.9.m9.1b"><apply id="S2.SS1.p1.9.m9.1.1.cmml" xref="S2.SS1.p1.9.m9.1.1"><csymbol cd="ambiguous" id="S2.SS1.p1.9.m9.1.1.1.cmml" xref="S2.SS1.p1.9.m9.1.1">subscript</csymbol><ci id="S2.SS1.p1.9.m9.1.1.2.cmml" xref="S2.SS1.p1.9.m9.1.1.2">ùëõ</ci><ci id="S2.SS1.p1.9.m9.1.1.3.cmml" xref="S2.SS1.p1.9.m9.1.1.3">ùëë</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.9.m9.1c">n_{d}</annotation></semantics></math> is the number of reported relations in <math id="S2.SS1.p1.10.m10.1" class="ltx_Math" alttext="d" display="inline"><semantics id="S2.SS1.p1.10.m10.1a"><mi id="S2.SS1.p1.10.m10.1.1" xref="S2.SS1.p1.10.m10.1.1.cmml">d</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.10.m10.1b"><ci id="S2.SS1.p1.10.m10.1.1.cmml" xref="S2.SS1.p1.10.m10.1.1">ùëë</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.10.m10.1c">d</annotation></semantics></math>. A relation <math id="S2.SS1.p1.11.m11.2" class="ltx_Math" alttext="r_{k}=(o_{i},c_{j})" display="inline"><semantics id="S2.SS1.p1.11.m11.2a"><mrow id="S2.SS1.p1.11.m11.2.2" xref="S2.SS1.p1.11.m11.2.2.cmml"><msub id="S2.SS1.p1.11.m11.2.2.4" xref="S2.SS1.p1.11.m11.2.2.4.cmml"><mi id="S2.SS1.p1.11.m11.2.2.4.2" xref="S2.SS1.p1.11.m11.2.2.4.2.cmml">r</mi><mi id="S2.SS1.p1.11.m11.2.2.4.3" xref="S2.SS1.p1.11.m11.2.2.4.3.cmml">k</mi></msub><mo id="S2.SS1.p1.11.m11.2.2.3" xref="S2.SS1.p1.11.m11.2.2.3.cmml">=</mo><mrow id="S2.SS1.p1.11.m11.2.2.2.2" xref="S2.SS1.p1.11.m11.2.2.2.3.cmml"><mo stretchy="false" id="S2.SS1.p1.11.m11.2.2.2.2.3" xref="S2.SS1.p1.11.m11.2.2.2.3.cmml">(</mo><msub id="S2.SS1.p1.11.m11.1.1.1.1.1" xref="S2.SS1.p1.11.m11.1.1.1.1.1.cmml"><mi id="S2.SS1.p1.11.m11.1.1.1.1.1.2" xref="S2.SS1.p1.11.m11.1.1.1.1.1.2.cmml">o</mi><mi id="S2.SS1.p1.11.m11.1.1.1.1.1.3" xref="S2.SS1.p1.11.m11.1.1.1.1.1.3.cmml">i</mi></msub><mo id="S2.SS1.p1.11.m11.2.2.2.2.4" xref="S2.SS1.p1.11.m11.2.2.2.3.cmml">,</mo><msub id="S2.SS1.p1.11.m11.2.2.2.2.2" xref="S2.SS1.p1.11.m11.2.2.2.2.2.cmml"><mi id="S2.SS1.p1.11.m11.2.2.2.2.2.2" xref="S2.SS1.p1.11.m11.2.2.2.2.2.2.cmml">c</mi><mi id="S2.SS1.p1.11.m11.2.2.2.2.2.3" xref="S2.SS1.p1.11.m11.2.2.2.2.2.3.cmml">j</mi></msub><mo stretchy="false" id="S2.SS1.p1.11.m11.2.2.2.2.5" xref="S2.SS1.p1.11.m11.2.2.2.3.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.11.m11.2b"><apply id="S2.SS1.p1.11.m11.2.2.cmml" xref="S2.SS1.p1.11.m11.2.2"><eq id="S2.SS1.p1.11.m11.2.2.3.cmml" xref="S2.SS1.p1.11.m11.2.2.3"></eq><apply id="S2.SS1.p1.11.m11.2.2.4.cmml" xref="S2.SS1.p1.11.m11.2.2.4"><csymbol cd="ambiguous" id="S2.SS1.p1.11.m11.2.2.4.1.cmml" xref="S2.SS1.p1.11.m11.2.2.4">subscript</csymbol><ci id="S2.SS1.p1.11.m11.2.2.4.2.cmml" xref="S2.SS1.p1.11.m11.2.2.4.2">ùëü</ci><ci id="S2.SS1.p1.11.m11.2.2.4.3.cmml" xref="S2.SS1.p1.11.m11.2.2.4.3">ùëò</ci></apply><interval closure="open" id="S2.SS1.p1.11.m11.2.2.2.3.cmml" xref="S2.SS1.p1.11.m11.2.2.2.2"><apply id="S2.SS1.p1.11.m11.1.1.1.1.1.cmml" xref="S2.SS1.p1.11.m11.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.SS1.p1.11.m11.1.1.1.1.1.1.cmml" xref="S2.SS1.p1.11.m11.1.1.1.1.1">subscript</csymbol><ci id="S2.SS1.p1.11.m11.1.1.1.1.1.2.cmml" xref="S2.SS1.p1.11.m11.1.1.1.1.1.2">ùëú</ci><ci id="S2.SS1.p1.11.m11.1.1.1.1.1.3.cmml" xref="S2.SS1.p1.11.m11.1.1.1.1.1.3">ùëñ</ci></apply><apply id="S2.SS1.p1.11.m11.2.2.2.2.2.cmml" xref="S2.SS1.p1.11.m11.2.2.2.2.2"><csymbol cd="ambiguous" id="S2.SS1.p1.11.m11.2.2.2.2.2.1.cmml" xref="S2.SS1.p1.11.m11.2.2.2.2.2">subscript</csymbol><ci id="S2.SS1.p1.11.m11.2.2.2.2.2.2.cmml" xref="S2.SS1.p1.11.m11.2.2.2.2.2.2">ùëê</ci><ci id="S2.SS1.p1.11.m11.2.2.2.2.2.3.cmml" xref="S2.SS1.p1.11.m11.2.2.2.2.2.3">ùëó</ci></apply></interval></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.11.m11.2c">r_{k}=(o_{i},c_{j})</annotation></semantics></math> involves the organism <math id="S2.SS1.p1.12.m12.1" class="ltx_Math" alttext="o_{i}" display="inline"><semantics id="S2.SS1.p1.12.m12.1a"><msub id="S2.SS1.p1.12.m12.1.1" xref="S2.SS1.p1.12.m12.1.1.cmml"><mi id="S2.SS1.p1.12.m12.1.1.2" xref="S2.SS1.p1.12.m12.1.1.2.cmml">o</mi><mi id="S2.SS1.p1.12.m12.1.1.3" xref="S2.SS1.p1.12.m12.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.12.m12.1b"><apply id="S2.SS1.p1.12.m12.1.1.cmml" xref="S2.SS1.p1.12.m12.1.1"><csymbol cd="ambiguous" id="S2.SS1.p1.12.m12.1.1.1.cmml" xref="S2.SS1.p1.12.m12.1.1">subscript</csymbol><ci id="S2.SS1.p1.12.m12.1.1.2.cmml" xref="S2.SS1.p1.12.m12.1.1.2">ùëú</ci><ci id="S2.SS1.p1.12.m12.1.1.3.cmml" xref="S2.SS1.p1.12.m12.1.1.3">ùëñ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.12.m12.1c">o_{i}</annotation></semantics></math> and the chemical <math id="S2.SS1.p1.13.m13.1" class="ltx_Math" alttext="c_{j}" display="inline"><semantics id="S2.SS1.p1.13.m13.1a"><msub id="S2.SS1.p1.13.m13.1.1" xref="S2.SS1.p1.13.m13.1.1.cmml"><mi id="S2.SS1.p1.13.m13.1.1.2" xref="S2.SS1.p1.13.m13.1.1.2.cmml">c</mi><mi id="S2.SS1.p1.13.m13.1.1.3" xref="S2.SS1.p1.13.m13.1.1.3.cmml">j</mi></msub><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.13.m13.1b"><apply id="S2.SS1.p1.13.m13.1.1.cmml" xref="S2.SS1.p1.13.m13.1.1"><csymbol cd="ambiguous" id="S2.SS1.p1.13.m13.1.1.1.cmml" xref="S2.SS1.p1.13.m13.1.1">subscript</csymbol><ci id="S2.SS1.p1.13.m13.1.1.2.cmml" xref="S2.SS1.p1.13.m13.1.1.2">ùëê</ci><ci id="S2.SS1.p1.13.m13.1.1.3.cmml" xref="S2.SS1.p1.13.m13.1.1.3">ùëó</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.13.m13.1c">c_{j}</annotation></semantics></math>. The set of organisms and chemicals are denoted as <math id="S2.SS1.p1.14.m14.1" class="ltx_Math" alttext="O" display="inline"><semantics id="S2.SS1.p1.14.m14.1a"><mi id="S2.SS1.p1.14.m14.1.1" xref="S2.SS1.p1.14.m14.1.1.cmml">O</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.14.m14.1b"><ci id="S2.SS1.p1.14.m14.1.1.cmml" xref="S2.SS1.p1.14.m14.1.1">ùëÇ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.14.m14.1c">O</annotation></semantics></math> and <math id="S2.SS1.p1.15.m15.1" class="ltx_Math" alttext="C" display="inline"><semantics id="S2.SS1.p1.15.m15.1a"><mi id="S2.SS1.p1.15.m15.1.1" xref="S2.SS1.p1.15.m15.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.15.m15.1b"><ci id="S2.SS1.p1.15.m15.1.1.cmml" xref="S2.SS1.p1.15.m15.1.1">ùê∂</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.15.m15.1c">C</annotation></semantics></math>, respectively.
<br class="ltx_break">Then, given a set <math id="S2.SS1.p1.16.m16.1" class="ltx_Math" alttext="S" display="inline"><semantics id="S2.SS1.p1.16.m16.1a"><mi id="S2.SS1.p1.16.m16.1.1" xref="S2.SS1.p1.16.m16.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.16.m16.1b"><ci id="S2.SS1.p1.16.m16.1.1.cmml" xref="S2.SS1.p1.16.m16.1.1">ùëÜ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.16.m16.1c">S</annotation></semantics></math> of documents, the probability that a reported relation involve the organism <math id="S2.SS1.p1.17.m17.1" class="ltx_Math" alttext="o_{i}" display="inline"><semantics id="S2.SS1.p1.17.m17.1a"><msub id="S2.SS1.p1.17.m17.1.1" xref="S2.SS1.p1.17.m17.1.1.cmml"><mi id="S2.SS1.p1.17.m17.1.1.2" xref="S2.SS1.p1.17.m17.1.1.2.cmml">o</mi><mi id="S2.SS1.p1.17.m17.1.1.3" xref="S2.SS1.p1.17.m17.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.17.m17.1b"><apply id="S2.SS1.p1.17.m17.1.1.cmml" xref="S2.SS1.p1.17.m17.1.1"><csymbol cd="ambiguous" id="S2.SS1.p1.17.m17.1.1.1.cmml" xref="S2.SS1.p1.17.m17.1.1">subscript</csymbol><ci id="S2.SS1.p1.17.m17.1.1.2.cmml" xref="S2.SS1.p1.17.m17.1.1.2">ùëú</ci><ci id="S2.SS1.p1.17.m17.1.1.3.cmml" xref="S2.SS1.p1.17.m17.1.1.3">ùëñ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.17.m17.1c">o_{i}</annotation></semantics></math> (and similarly for the chemical <math id="S2.SS1.p1.18.m18.1" class="ltx_Math" alttext="c_{j}" display="inline"><semantics id="S2.SS1.p1.18.m18.1a"><msub id="S2.SS1.p1.18.m18.1.1" xref="S2.SS1.p1.18.m18.1.1.cmml"><mi id="S2.SS1.p1.18.m18.1.1.2" xref="S2.SS1.p1.18.m18.1.1.2.cmml">c</mi><mi id="S2.SS1.p1.18.m18.1.1.3" xref="S2.SS1.p1.18.m18.1.1.3.cmml">j</mi></msub><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.18.m18.1b"><apply id="S2.SS1.p1.18.m18.1.1.cmml" xref="S2.SS1.p1.18.m18.1.1"><csymbol cd="ambiguous" id="S2.SS1.p1.18.m18.1.1.1.cmml" xref="S2.SS1.p1.18.m18.1.1">subscript</csymbol><ci id="S2.SS1.p1.18.m18.1.1.2.cmml" xref="S2.SS1.p1.18.m18.1.1.2">ùëê</ci><ci id="S2.SS1.p1.18.m18.1.1.3.cmml" xref="S2.SS1.p1.18.m18.1.1.3">ùëó</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.18.m18.1c">c_{j}</annotation></semantics></math>) is</p>
<table id="S2.E1" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S2.E1.m1.5" class="ltx_Math" alttext="P(o_{i})=\frac{\displaystyle\sum_{d\in S}|\{r_{k}:o_{i}\in r_{k}\}^{d}|}{\displaystyle\sum_{d\in S}n_{d}},\text{ with }|\{r_{k}:o_{i}\in r_{k}\}^{d}|\text{ the number of relations involving $o_{i}$ in $d$.}" display="block"><semantics id="S2.E1.m1.5a"><mrow id="S2.E1.m1.5.5" xref="S2.E1.m1.5.5.cmml"><mrow id="S2.E1.m1.4.4.1" xref="S2.E1.m1.4.4.1.cmml"><mi id="S2.E1.m1.4.4.1.3" xref="S2.E1.m1.4.4.1.3.cmml">P</mi><mo lspace="0em" rspace="0em" id="S2.E1.m1.4.4.1.2" xref="S2.E1.m1.4.4.1.2.cmml">‚Äã</mo><mrow id="S2.E1.m1.4.4.1.1.1" xref="S2.E1.m1.4.4.1.1.1.1.cmml"><mo stretchy="false" id="S2.E1.m1.4.4.1.1.1.2" xref="S2.E1.m1.4.4.1.1.1.1.cmml">(</mo><msub id="S2.E1.m1.4.4.1.1.1.1" xref="S2.E1.m1.4.4.1.1.1.1.cmml"><mi id="S2.E1.m1.4.4.1.1.1.1.2" xref="S2.E1.m1.4.4.1.1.1.1.2.cmml">o</mi><mi id="S2.E1.m1.4.4.1.1.1.1.3" xref="S2.E1.m1.4.4.1.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="S2.E1.m1.4.4.1.1.1.3" xref="S2.E1.m1.4.4.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S2.E1.m1.5.5.3" xref="S2.E1.m1.5.5.3.cmml">=</mo><mrow id="S2.E1.m1.5.5.2.1" xref="S2.E1.m1.5.5.2.2.cmml"><mfrac id="S2.E1.m1.3.3" xref="S2.E1.m1.3.3.cmml"><mrow id="S2.E1.m1.3.3.1" xref="S2.E1.m1.3.3.1.cmml"><mstyle displaystyle="true" id="S2.E1.m1.3.3.1.2" xref="S2.E1.m1.3.3.1.2.cmml"><munder id="S2.E1.m1.3.3.1.2a" xref="S2.E1.m1.3.3.1.2.cmml"><mo movablelimits="false" id="S2.E1.m1.3.3.1.2.2" xref="S2.E1.m1.3.3.1.2.2.cmml">‚àë</mo><mrow id="S2.E1.m1.3.3.1.2.3" xref="S2.E1.m1.3.3.1.2.3.cmml"><mi id="S2.E1.m1.3.3.1.2.3.2" xref="S2.E1.m1.3.3.1.2.3.2.cmml">d</mi><mo id="S2.E1.m1.3.3.1.2.3.1" xref="S2.E1.m1.3.3.1.2.3.1.cmml">‚àà</mo><mi id="S2.E1.m1.3.3.1.2.3.3" xref="S2.E1.m1.3.3.1.2.3.3.cmml">S</mi></mrow></munder></mstyle><mrow id="S2.E1.m1.3.3.1.1.1" xref="S2.E1.m1.3.3.1.1.2.cmml"><mo stretchy="false" id="S2.E1.m1.3.3.1.1.1.2" xref="S2.E1.m1.3.3.1.1.2.1.cmml">|</mo><msup id="S2.E1.m1.3.3.1.1.1.1" xref="S2.E1.m1.3.3.1.1.1.1.cmml"><mrow id="S2.E1.m1.3.3.1.1.1.1.2.2" xref="S2.E1.m1.3.3.1.1.1.1.2.3.cmml"><mo stretchy="false" id="S2.E1.m1.3.3.1.1.1.1.2.2.3" xref="S2.E1.m1.3.3.1.1.1.1.2.3.1.cmml">{</mo><msub id="S2.E1.m1.3.3.1.1.1.1.1.1.1" xref="S2.E1.m1.3.3.1.1.1.1.1.1.1.cmml"><mi id="S2.E1.m1.3.3.1.1.1.1.1.1.1.2" xref="S2.E1.m1.3.3.1.1.1.1.1.1.1.2.cmml">r</mi><mi id="S2.E1.m1.3.3.1.1.1.1.1.1.1.3" xref="S2.E1.m1.3.3.1.1.1.1.1.1.1.3.cmml">k</mi></msub><mo lspace="0.278em" rspace="0.278em" id="S2.E1.m1.3.3.1.1.1.1.2.2.4" xref="S2.E1.m1.3.3.1.1.1.1.2.3.1.cmml">:</mo><mrow id="S2.E1.m1.3.3.1.1.1.1.2.2.2" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.cmml"><msub id="S2.E1.m1.3.3.1.1.1.1.2.2.2.2" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.cmml"><mi id="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.2" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.2.cmml">o</mi><mi id="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.3" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.3.cmml">i</mi></msub><mo id="S2.E1.m1.3.3.1.1.1.1.2.2.2.1" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.1.cmml">‚àà</mo><msub id="S2.E1.m1.3.3.1.1.1.1.2.2.2.3" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.cmml"><mi id="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.2" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.2.cmml">r</mi><mi id="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.3" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.3.cmml">k</mi></msub></mrow><mo stretchy="false" id="S2.E1.m1.3.3.1.1.1.1.2.2.5" xref="S2.E1.m1.3.3.1.1.1.1.2.3.1.cmml">}</mo></mrow><mi id="S2.E1.m1.3.3.1.1.1.1.4" xref="S2.E1.m1.3.3.1.1.1.1.4.cmml">d</mi></msup><mo stretchy="false" id="S2.E1.m1.3.3.1.1.1.3" xref="S2.E1.m1.3.3.1.1.2.1.cmml">|</mo></mrow></mrow><mrow id="S2.E1.m1.3.3.3" xref="S2.E1.m1.3.3.3.cmml"><mstyle displaystyle="true" id="S2.E1.m1.3.3.3.1" xref="S2.E1.m1.3.3.3.1.cmml"><munder id="S2.E1.m1.3.3.3.1a" xref="S2.E1.m1.3.3.3.1.cmml"><mo movablelimits="false" id="S2.E1.m1.3.3.3.1.2" xref="S2.E1.m1.3.3.3.1.2.cmml">‚àë</mo><mrow id="S2.E1.m1.3.3.3.1.3" xref="S2.E1.m1.3.3.3.1.3.cmml"><mi id="S2.E1.m1.3.3.3.1.3.2" xref="S2.E1.m1.3.3.3.1.3.2.cmml">d</mi><mo id="S2.E1.m1.3.3.3.1.3.1" xref="S2.E1.m1.3.3.3.1.3.1.cmml">‚àà</mo><mi id="S2.E1.m1.3.3.3.1.3.3" xref="S2.E1.m1.3.3.3.1.3.3.cmml">S</mi></mrow></munder></mstyle><msub id="S2.E1.m1.3.3.3.2" xref="S2.E1.m1.3.3.3.2.cmml"><mi id="S2.E1.m1.3.3.3.2.2" xref="S2.E1.m1.3.3.3.2.2.cmml">n</mi><mi id="S2.E1.m1.3.3.3.2.3" xref="S2.E1.m1.3.3.3.2.3.cmml">d</mi></msub></mrow></mfrac><mo id="S2.E1.m1.5.5.2.1.2" xref="S2.E1.m1.5.5.2.2.cmml">,</mo><mrow id="S2.E1.m1.5.5.2.1.1" xref="S2.E1.m1.5.5.2.1.1.cmml"><mtext id="S2.E1.m1.5.5.2.1.1.3" xref="S2.E1.m1.5.5.2.1.1.3a.cmml">¬†with¬†</mtext><mo lspace="0em" rspace="0em" id="S2.E1.m1.5.5.2.1.1.2" xref="S2.E1.m1.5.5.2.1.1.2.cmml">‚Äã</mo><mrow id="S2.E1.m1.5.5.2.1.1.1.1" xref="S2.E1.m1.5.5.2.1.1.1.2.cmml"><mo stretchy="false" id="S2.E1.m1.5.5.2.1.1.1.1.2" xref="S2.E1.m1.5.5.2.1.1.1.2.1.cmml">|</mo><msup id="S2.E1.m1.5.5.2.1.1.1.1.1" xref="S2.E1.m1.5.5.2.1.1.1.1.1.cmml"><mrow id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.3.cmml"><mo stretchy="false" id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.3" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.3.1.cmml">{</mo><msub id="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1" xref="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.cmml"><mi id="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.2" xref="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.2.cmml">r</mi><mi id="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.3" xref="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.3.cmml">k</mi></msub><mo lspace="0.278em" rspace="0.278em" id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.4" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.3.1.cmml">:</mo><mrow id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.cmml"><msub id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.cmml"><mi id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.2" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.2.cmml">o</mi><mi id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.3" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.3.cmml">i</mi></msub><mo id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.1" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.1.cmml">‚àà</mo><msub id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.cmml"><mi id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.2" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.2.cmml">r</mi><mi id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.3" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.3.cmml">k</mi></msub></mrow><mo stretchy="false" id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.5" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.3.1.cmml">}</mo></mrow><mi id="S2.E1.m1.5.5.2.1.1.1.1.1.4" xref="S2.E1.m1.5.5.2.1.1.1.1.1.4.cmml">d</mi></msup><mo stretchy="false" id="S2.E1.m1.5.5.2.1.1.1.1.3" xref="S2.E1.m1.5.5.2.1.1.1.2.1.cmml">|</mo></mrow><mo lspace="0em" rspace="0em" id="S2.E1.m1.5.5.2.1.1.2a" xref="S2.E1.m1.5.5.2.1.1.2.cmml">‚Äã</mo><mrow id="S2.E1.m1.2.2.2" xref="S2.E1.m1.2.2.2d.cmml"><mtext id="S2.E1.m1.2.2.2a" xref="S2.E1.m1.2.2.2d.cmml">¬†the number of relations involving¬†</mtext><msub id="S2.E1.m1.1.1.1.m1.1.1" xref="S2.E1.m1.1.1.1.m1.1.1.cmml"><mi id="S2.E1.m1.1.1.1.m1.1.1.2" xref="S2.E1.m1.1.1.1.m1.1.1.2.cmml">o</mi><mi id="S2.E1.m1.1.1.1.m1.1.1.3" xref="S2.E1.m1.1.1.1.m1.1.1.3.cmml">i</mi></msub><mtext id="S2.E1.m1.2.2.2b" xref="S2.E1.m1.2.2.2d.cmml">¬†in¬†</mtext><mi id="S2.E1.m1.2.2.2.m2.1.1" xref="S2.E1.m1.2.2.2.m2.1.1.cmml">d</mi><mtext id="S2.E1.m1.2.2.2c" xref="S2.E1.m1.2.2.2d.cmml">.</mtext></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.E1.m1.5b"><apply id="S2.E1.m1.5.5.cmml" xref="S2.E1.m1.5.5"><eq id="S2.E1.m1.5.5.3.cmml" xref="S2.E1.m1.5.5.3"></eq><apply id="S2.E1.m1.4.4.1.cmml" xref="S2.E1.m1.4.4.1"><times id="S2.E1.m1.4.4.1.2.cmml" xref="S2.E1.m1.4.4.1.2"></times><ci id="S2.E1.m1.4.4.1.3.cmml" xref="S2.E1.m1.4.4.1.3">ùëÉ</ci><apply id="S2.E1.m1.4.4.1.1.1.1.cmml" xref="S2.E1.m1.4.4.1.1.1"><csymbol cd="ambiguous" id="S2.E1.m1.4.4.1.1.1.1.1.cmml" xref="S2.E1.m1.4.4.1.1.1">subscript</csymbol><ci id="S2.E1.m1.4.4.1.1.1.1.2.cmml" xref="S2.E1.m1.4.4.1.1.1.1.2">ùëú</ci><ci id="S2.E1.m1.4.4.1.1.1.1.3.cmml" xref="S2.E1.m1.4.4.1.1.1.1.3">ùëñ</ci></apply></apply><list id="S2.E1.m1.5.5.2.2.cmml" xref="S2.E1.m1.5.5.2.1"><apply id="S2.E1.m1.3.3.cmml" xref="S2.E1.m1.3.3"><divide id="S2.E1.m1.3.3.2.cmml" xref="S2.E1.m1.3.3"></divide><apply id="S2.E1.m1.3.3.1.cmml" xref="S2.E1.m1.3.3.1"><apply id="S2.E1.m1.3.3.1.2.cmml" xref="S2.E1.m1.3.3.1.2"><csymbol cd="ambiguous" id="S2.E1.m1.3.3.1.2.1.cmml" xref="S2.E1.m1.3.3.1.2">subscript</csymbol><sum id="S2.E1.m1.3.3.1.2.2.cmml" xref="S2.E1.m1.3.3.1.2.2"></sum><apply id="S2.E1.m1.3.3.1.2.3.cmml" xref="S2.E1.m1.3.3.1.2.3"><in id="S2.E1.m1.3.3.1.2.3.1.cmml" xref="S2.E1.m1.3.3.1.2.3.1"></in><ci id="S2.E1.m1.3.3.1.2.3.2.cmml" xref="S2.E1.m1.3.3.1.2.3.2">ùëë</ci><ci id="S2.E1.m1.3.3.1.2.3.3.cmml" xref="S2.E1.m1.3.3.1.2.3.3">ùëÜ</ci></apply></apply><apply id="S2.E1.m1.3.3.1.1.2.cmml" xref="S2.E1.m1.3.3.1.1.1"><abs id="S2.E1.m1.3.3.1.1.2.1.cmml" xref="S2.E1.m1.3.3.1.1.1.2"></abs><apply id="S2.E1.m1.3.3.1.1.1.1.cmml" xref="S2.E1.m1.3.3.1.1.1.1"><csymbol cd="ambiguous" id="S2.E1.m1.3.3.1.1.1.1.3.cmml" xref="S2.E1.m1.3.3.1.1.1.1">superscript</csymbol><apply id="S2.E1.m1.3.3.1.1.1.1.2.3.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2"><csymbol cd="latexml" id="S2.E1.m1.3.3.1.1.1.1.2.3.1.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.3">conditional-set</csymbol><apply id="S2.E1.m1.3.3.1.1.1.1.1.1.1.cmml" xref="S2.E1.m1.3.3.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.E1.m1.3.3.1.1.1.1.1.1.1.1.cmml" xref="S2.E1.m1.3.3.1.1.1.1.1.1.1">subscript</csymbol><ci id="S2.E1.m1.3.3.1.1.1.1.1.1.1.2.cmml" xref="S2.E1.m1.3.3.1.1.1.1.1.1.1.2">ùëü</ci><ci id="S2.E1.m1.3.3.1.1.1.1.1.1.1.3.cmml" xref="S2.E1.m1.3.3.1.1.1.1.1.1.1.3">ùëò</ci></apply><apply id="S2.E1.m1.3.3.1.1.1.1.2.2.2.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2"><in id="S2.E1.m1.3.3.1.1.1.1.2.2.2.1.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.1"></in><apply id="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.2"><csymbol cd="ambiguous" id="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.1.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.2">subscript</csymbol><ci id="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.2.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.2">ùëú</ci><ci id="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.3.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.2.3">ùëñ</ci></apply><apply id="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.3"><csymbol cd="ambiguous" id="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.1.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.3">subscript</csymbol><ci id="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.2.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.2">ùëü</ci><ci id="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.3.cmml" xref="S2.E1.m1.3.3.1.1.1.1.2.2.2.3.3">ùëò</ci></apply></apply></apply><ci id="S2.E1.m1.3.3.1.1.1.1.4.cmml" xref="S2.E1.m1.3.3.1.1.1.1.4">ùëë</ci></apply></apply></apply><apply id="S2.E1.m1.3.3.3.cmml" xref="S2.E1.m1.3.3.3"><apply id="S2.E1.m1.3.3.3.1.cmml" xref="S2.E1.m1.3.3.3.1"><csymbol cd="ambiguous" id="S2.E1.m1.3.3.3.1.1.cmml" xref="S2.E1.m1.3.3.3.1">subscript</csymbol><sum id="S2.E1.m1.3.3.3.1.2.cmml" xref="S2.E1.m1.3.3.3.1.2"></sum><apply id="S2.E1.m1.3.3.3.1.3.cmml" xref="S2.E1.m1.3.3.3.1.3"><in id="S2.E1.m1.3.3.3.1.3.1.cmml" xref="S2.E1.m1.3.3.3.1.3.1"></in><ci id="S2.E1.m1.3.3.3.1.3.2.cmml" xref="S2.E1.m1.3.3.3.1.3.2">ùëë</ci><ci id="S2.E1.m1.3.3.3.1.3.3.cmml" xref="S2.E1.m1.3.3.3.1.3.3">ùëÜ</ci></apply></apply><apply id="S2.E1.m1.3.3.3.2.cmml" xref="S2.E1.m1.3.3.3.2"><csymbol cd="ambiguous" id="S2.E1.m1.3.3.3.2.1.cmml" xref="S2.E1.m1.3.3.3.2">subscript</csymbol><ci id="S2.E1.m1.3.3.3.2.2.cmml" xref="S2.E1.m1.3.3.3.2.2">ùëõ</ci><ci id="S2.E1.m1.3.3.3.2.3.cmml" xref="S2.E1.m1.3.3.3.2.3">ùëë</ci></apply></apply></apply><apply id="S2.E1.m1.5.5.2.1.1.cmml" xref="S2.E1.m1.5.5.2.1.1"><times id="S2.E1.m1.5.5.2.1.1.2.cmml" xref="S2.E1.m1.5.5.2.1.1.2"></times><ci id="S2.E1.m1.5.5.2.1.1.3a.cmml" xref="S2.E1.m1.5.5.2.1.1.3"><mtext id="S2.E1.m1.5.5.2.1.1.3.cmml" xref="S2.E1.m1.5.5.2.1.1.3">¬†with¬†</mtext></ci><apply id="S2.E1.m1.5.5.2.1.1.1.2.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1"><abs id="S2.E1.m1.5.5.2.1.1.1.2.1.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.2"></abs><apply id="S2.E1.m1.5.5.2.1.1.1.1.1.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.E1.m1.5.5.2.1.1.1.1.1.3.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1">superscript</csymbol><apply id="S2.E1.m1.5.5.2.1.1.1.1.1.2.3.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2"><csymbol cd="latexml" id="S2.E1.m1.5.5.2.1.1.1.1.1.2.3.1.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.3">conditional-set</csymbol><apply id="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.1.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.2.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.2">ùëü</ci><ci id="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.3.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.1.1.1.3">ùëò</ci></apply><apply id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2"><in id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.1.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.1"></in><apply id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2"><csymbol cd="ambiguous" id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.1.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2">subscript</csymbol><ci id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.2.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.2">ùëú</ci><ci id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.3.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.2.3">ùëñ</ci></apply><apply id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3"><csymbol cd="ambiguous" id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.1.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3">subscript</csymbol><ci id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.2.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.2">ùëü</ci><ci id="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.3.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.2.2.2.3.3">ùëò</ci></apply></apply></apply><ci id="S2.E1.m1.5.5.2.1.1.1.1.1.4.cmml" xref="S2.E1.m1.5.5.2.1.1.1.1.1.4">ùëë</ci></apply></apply><ci id="S2.E1.m1.2.2.2d.cmml" xref="S2.E1.m1.2.2.2"><mrow id="S2.E1.m1.2.2.2.cmml" xref="S2.E1.m1.2.2.2"><mtext id="S2.E1.m1.2.2.2a.cmml" xref="S2.E1.m1.2.2.2">¬†the number of relations involving¬†</mtext><msub id="S2.E1.m1.1.1.1.m1.1.1.cmml" xref="S2.E1.m1.1.1.1.m1.1.1"><mi id="S2.E1.m1.1.1.1.m1.1.1.2.cmml" xref="S2.E1.m1.1.1.1.m1.1.1.2">o</mi><mi id="S2.E1.m1.1.1.1.m1.1.1.3.cmml" xref="S2.E1.m1.1.1.1.m1.1.1.3">i</mi></msub><mtext id="S2.E1.m1.2.2.2b.cmml" xref="S2.E1.m1.2.2.2">¬†in¬†</mtext><mi id="S2.E1.m1.2.2.2.m2.1.1.cmml" xref="S2.E1.m1.2.2.2.m2.1.1">d</mi><mtext id="S2.E1.m1.2.2.2c.cmml" xref="S2.E1.m1.2.2.2">.</mtext></mrow></ci></apply></list></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.E1.m1.5c">P(o_{i})=\frac{\displaystyle\sum_{d\in S}|\{r_{k}:o_{i}\in r_{k}\}^{d}|}{\displaystyle\sum_{d\in S}n_{d}},\text{ with }|\{r_{k}:o_{i}\in r_{k}\}^{d}|\text{ the number of relations involving $o_{i}$ in $d$.}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(1)</span></td>
</tr></tbody>
</table>
<p id="S2.SS1.p1.32" class="ltx_p">It follows that the diversity of organisms or chemicals can be measured with the Shannon‚Äôs entropy over the probability distributions of elements of <math id="S2.SS1.p1.19.m1.1" class="ltx_Math" alttext="O" display="inline"><semantics id="S2.SS1.p1.19.m1.1a"><mi id="S2.SS1.p1.19.m1.1.1" xref="S2.SS1.p1.19.m1.1.1.cmml">O</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.19.m1.1b"><ci id="S2.SS1.p1.19.m1.1.1.cmml" xref="S2.SS1.p1.19.m1.1.1">ùëÇ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.19.m1.1c">O</annotation></semantics></math> and <math id="S2.SS1.p1.20.m2.1" class="ltx_Math" alttext="C" display="inline"><semantics id="S2.SS1.p1.20.m2.1a"><mi id="S2.SS1.p1.20.m2.1.1" xref="S2.SS1.p1.20.m2.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.20.m2.1b"><ci id="S2.SS1.p1.20.m2.1.1.cmml" xref="S2.SS1.p1.20.m2.1.1">ùê∂</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.20.m2.1c">C</annotation></semantics></math> in the sample <math id="S2.SS1.p1.21.m3.1" class="ltx_Math" alttext="S" display="inline"><semantics id="S2.SS1.p1.21.m3.1a"><mi id="S2.SS1.p1.21.m3.1.1" xref="S2.SS1.p1.21.m3.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.21.m3.1b"><ci id="S2.SS1.p1.21.m3.1.1.cmml" xref="S2.SS1.p1.21.m3.1.1">ùëÜ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.21.m3.1c">S</annotation></semantics></math>. Expressed with entropy, the diversity reflects the uncertainty about the organism, or the chemical, which is attached to a relation reported in an article <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib70" title="" class="ltx_ref">70</a>, <a href="#bib.bib71" title="" class="ltx_ref">71</a>]</cite>. For organisms (resp. chemicals) the Shannon‚Äôs entropy in the sample <math id="S2.SS1.p1.22.m4.1" class="ltx_Math" alttext="S" display="inline"><semantics id="S2.SS1.p1.22.m4.1a"><mi id="S2.SS1.p1.22.m4.1.1" xref="S2.SS1.p1.22.m4.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.22.m4.1b"><ci id="S2.SS1.p1.22.m4.1.1.cmml" xref="S2.SS1.p1.22.m4.1.1">ùëÜ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.22.m4.1c">S</annotation></semantics></math> is <math id="S2.SS1.p1.23.m5.3" class="ltx_Math" alttext="H_{S}(O)=-\displaystyle\sum_{o_{i}\in O}P(o_{i})\log P(o_{i})" display="inline"><semantics id="S2.SS1.p1.23.m5.3a"><mrow id="S2.SS1.p1.23.m5.3.3" xref="S2.SS1.p1.23.m5.3.3.cmml"><mrow id="S2.SS1.p1.23.m5.3.3.4" xref="S2.SS1.p1.23.m5.3.3.4.cmml"><msub id="S2.SS1.p1.23.m5.3.3.4.2" xref="S2.SS1.p1.23.m5.3.3.4.2.cmml"><mi id="S2.SS1.p1.23.m5.3.3.4.2.2" xref="S2.SS1.p1.23.m5.3.3.4.2.2.cmml">H</mi><mi id="S2.SS1.p1.23.m5.3.3.4.2.3" xref="S2.SS1.p1.23.m5.3.3.4.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.SS1.p1.23.m5.3.3.4.1" xref="S2.SS1.p1.23.m5.3.3.4.1.cmml">‚Äã</mo><mrow id="S2.SS1.p1.23.m5.3.3.4.3.2" xref="S2.SS1.p1.23.m5.3.3.4.cmml"><mo stretchy="false" id="S2.SS1.p1.23.m5.3.3.4.3.2.1" xref="S2.SS1.p1.23.m5.3.3.4.cmml">(</mo><mi id="S2.SS1.p1.23.m5.1.1" xref="S2.SS1.p1.23.m5.1.1.cmml">O</mi><mo stretchy="false" id="S2.SS1.p1.23.m5.3.3.4.3.2.2" xref="S2.SS1.p1.23.m5.3.3.4.cmml">)</mo></mrow></mrow><mo id="S2.SS1.p1.23.m5.3.3.3" xref="S2.SS1.p1.23.m5.3.3.3.cmml">=</mo><mrow id="S2.SS1.p1.23.m5.3.3.2" xref="S2.SS1.p1.23.m5.3.3.2.cmml"><mo id="S2.SS1.p1.23.m5.3.3.2a" xref="S2.SS1.p1.23.m5.3.3.2.cmml">‚àí</mo><mrow id="S2.SS1.p1.23.m5.3.3.2.2" xref="S2.SS1.p1.23.m5.3.3.2.2.cmml"><mstyle displaystyle="true" id="S2.SS1.p1.23.m5.3.3.2.2.3" xref="S2.SS1.p1.23.m5.3.3.2.2.3.cmml"><munder id="S2.SS1.p1.23.m5.3.3.2.2.3a" xref="S2.SS1.p1.23.m5.3.3.2.2.3.cmml"><mo movablelimits="false" id="S2.SS1.p1.23.m5.3.3.2.2.3.2" xref="S2.SS1.p1.23.m5.3.3.2.2.3.2.cmml">‚àë</mo><mrow id="S2.SS1.p1.23.m5.3.3.2.2.3.3" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.cmml"><msub id="S2.SS1.p1.23.m5.3.3.2.2.3.3.2" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.cmml"><mi id="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.2" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.2.cmml">o</mi><mi id="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.3" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.3.cmml">i</mi></msub><mo id="S2.SS1.p1.23.m5.3.3.2.2.3.3.1" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.1.cmml">‚àà</mo><mi id="S2.SS1.p1.23.m5.3.3.2.2.3.3.3" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.3.cmml">O</mi></mrow></munder></mstyle><mrow id="S2.SS1.p1.23.m5.3.3.2.2.2" xref="S2.SS1.p1.23.m5.3.3.2.2.2.cmml"><mi id="S2.SS1.p1.23.m5.3.3.2.2.2.4" xref="S2.SS1.p1.23.m5.3.3.2.2.2.4.cmml">P</mi><mo lspace="0em" rspace="0em" id="S2.SS1.p1.23.m5.3.3.2.2.2.3" xref="S2.SS1.p1.23.m5.3.3.2.2.2.3.cmml">‚Äã</mo><mrow id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.cmml"><mo stretchy="false" id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.2" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.cmml">(</mo><msub id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.cmml"><mi id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.2" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.2.cmml">o</mi><mi id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.3" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.3" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.cmml">)</mo></mrow><mo lspace="0.167em" rspace="0em" id="S2.SS1.p1.23.m5.3.3.2.2.2.3a" xref="S2.SS1.p1.23.m5.3.3.2.2.2.3.cmml">‚Äã</mo><mrow id="S2.SS1.p1.23.m5.3.3.2.2.2.5" xref="S2.SS1.p1.23.m5.3.3.2.2.2.5.cmml"><mi id="S2.SS1.p1.23.m5.3.3.2.2.2.5.1" xref="S2.SS1.p1.23.m5.3.3.2.2.2.5.1.cmml">log</mi><mo lspace="0.167em" id="S2.SS1.p1.23.m5.3.3.2.2.2.5a" xref="S2.SS1.p1.23.m5.3.3.2.2.2.5.cmml">‚Å°</mo><mi id="S2.SS1.p1.23.m5.3.3.2.2.2.5.2" xref="S2.SS1.p1.23.m5.3.3.2.2.2.5.2.cmml">P</mi></mrow><mo lspace="0em" rspace="0em" id="S2.SS1.p1.23.m5.3.3.2.2.2.3b" xref="S2.SS1.p1.23.m5.3.3.2.2.2.3.cmml">‚Äã</mo><mrow id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.cmml"><mo stretchy="false" id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.2" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.cmml">(</mo><msub id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.cmml"><mi id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.2" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.2.cmml">o</mi><mi id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.3" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.3.cmml">i</mi></msub><mo stretchy="false" id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.3" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.cmml">)</mo></mrow></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.23.m5.3b"><apply id="S2.SS1.p1.23.m5.3.3.cmml" xref="S2.SS1.p1.23.m5.3.3"><eq id="S2.SS1.p1.23.m5.3.3.3.cmml" xref="S2.SS1.p1.23.m5.3.3.3"></eq><apply id="S2.SS1.p1.23.m5.3.3.4.cmml" xref="S2.SS1.p1.23.m5.3.3.4"><times id="S2.SS1.p1.23.m5.3.3.4.1.cmml" xref="S2.SS1.p1.23.m5.3.3.4.1"></times><apply id="S2.SS1.p1.23.m5.3.3.4.2.cmml" xref="S2.SS1.p1.23.m5.3.3.4.2"><csymbol cd="ambiguous" id="S2.SS1.p1.23.m5.3.3.4.2.1.cmml" xref="S2.SS1.p1.23.m5.3.3.4.2">subscript</csymbol><ci id="S2.SS1.p1.23.m5.3.3.4.2.2.cmml" xref="S2.SS1.p1.23.m5.3.3.4.2.2">ùêª</ci><ci id="S2.SS1.p1.23.m5.3.3.4.2.3.cmml" xref="S2.SS1.p1.23.m5.3.3.4.2.3">ùëÜ</ci></apply><ci id="S2.SS1.p1.23.m5.1.1.cmml" xref="S2.SS1.p1.23.m5.1.1">ùëÇ</ci></apply><apply id="S2.SS1.p1.23.m5.3.3.2.cmml" xref="S2.SS1.p1.23.m5.3.3.2"><minus id="S2.SS1.p1.23.m5.3.3.2.3.cmml" xref="S2.SS1.p1.23.m5.3.3.2"></minus><apply id="S2.SS1.p1.23.m5.3.3.2.2.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2"><apply id="S2.SS1.p1.23.m5.3.3.2.2.3.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3"><csymbol cd="ambiguous" id="S2.SS1.p1.23.m5.3.3.2.2.3.1.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3">subscript</csymbol><sum id="S2.SS1.p1.23.m5.3.3.2.2.3.2.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3.2"></sum><apply id="S2.SS1.p1.23.m5.3.3.2.2.3.3.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3"><in id="S2.SS1.p1.23.m5.3.3.2.2.3.3.1.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.1"></in><apply id="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.2"><csymbol cd="ambiguous" id="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.1.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.2">subscript</csymbol><ci id="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.2.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.2">ùëú</ci><ci id="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.3.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.2.3">ùëñ</ci></apply><ci id="S2.SS1.p1.23.m5.3.3.2.2.3.3.3.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.3.3.3">ùëÇ</ci></apply></apply><apply id="S2.SS1.p1.23.m5.3.3.2.2.2.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2"><times id="S2.SS1.p1.23.m5.3.3.2.2.2.3.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.3"></times><ci id="S2.SS1.p1.23.m5.3.3.2.2.2.4.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.4">ùëÉ</ci><apply id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.cmml" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.1.cmml" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1">subscript</csymbol><ci id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.2.cmml" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.2">ùëú</ci><ci id="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.3.cmml" xref="S2.SS1.p1.23.m5.2.2.1.1.1.1.1.1.3">ùëñ</ci></apply><apply id="S2.SS1.p1.23.m5.3.3.2.2.2.5.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.5"><log id="S2.SS1.p1.23.m5.3.3.2.2.2.5.1.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.5.1"></log><ci id="S2.SS1.p1.23.m5.3.3.2.2.2.5.2.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.5.2">ùëÉ</ci></apply><apply id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1"><csymbol cd="ambiguous" id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.1.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1">subscript</csymbol><ci id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.2.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.2">ùëú</ci><ci id="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.3.cmml" xref="S2.SS1.p1.23.m5.3.3.2.2.2.2.1.1.3">ùëñ</ci></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.23.m5.3c">H_{S}(O)=-\displaystyle\sum_{o_{i}\in O}P(o_{i})\log P(o_{i})</annotation></semantics></math>. Adding a new document <math id="S2.SS1.p1.24.m6.1" class="ltx_Math" alttext="\displaystyle d" display="inline"><semantics id="S2.SS1.p1.24.m6.1a"><mi id="S2.SS1.p1.24.m6.1.1" xref="S2.SS1.p1.24.m6.1.1.cmml">d</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.24.m6.1b"><ci id="S2.SS1.p1.24.m6.1.1.cmml" xref="S2.SS1.p1.24.m6.1.1">ùëë</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.24.m6.1c">\displaystyle d</annotation></semantics></math> to <math id="S2.SS1.p1.25.m7.1" class="ltx_Math" alttext="S" display="inline"><semantics id="S2.SS1.p1.25.m7.1a"><mi id="S2.SS1.p1.25.m7.1.1" xref="S2.SS1.p1.25.m7.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.25.m7.1b"><ci id="S2.SS1.p1.25.m7.1.1.cmml" xref="S2.SS1.p1.25.m7.1.1">ùëÜ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.25.m7.1c">S</annotation></semantics></math> will update the probability distributions of <math id="S2.SS1.p1.26.m8.1" class="ltx_Math" alttext="O" display="inline"><semantics id="S2.SS1.p1.26.m8.1a"><mi id="S2.SS1.p1.26.m8.1.1" xref="S2.SS1.p1.26.m8.1.1.cmml">O</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.26.m8.1b"><ci id="S2.SS1.p1.26.m8.1.1.cmml" xref="S2.SS1.p1.26.m8.1.1">ùëÇ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.26.m8.1c">O</annotation></semantics></math> (resp. <math id="S2.SS1.p1.27.m9.1" class="ltx_Math" alttext="C" display="inline"><semantics id="S2.SS1.p1.27.m9.1a"><mi id="S2.SS1.p1.27.m9.1.1" xref="S2.SS1.p1.27.m9.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.27.m9.1b"><ci id="S2.SS1.p1.27.m9.1.1.cmml" xref="S2.SS1.p1.27.m9.1.1">ùê∂</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.27.m9.1c">C</annotation></semantics></math>), and the new observed entropy will be <math id="S2.SS1.p1.28.m10.1" class="ltx_Math" alttext="H_{\displaystyle S_{+d}}(O)" display="inline"><semantics id="S2.SS1.p1.28.m10.1a"><mrow id="S2.SS1.p1.28.m10.1.2" xref="S2.SS1.p1.28.m10.1.2.cmml"><msub id="S2.SS1.p1.28.m10.1.2.2" xref="S2.SS1.p1.28.m10.1.2.2.cmml"><mi id="S2.SS1.p1.28.m10.1.2.2.2" xref="S2.SS1.p1.28.m10.1.2.2.2.cmml">H</mi><msub id="S2.SS1.p1.28.m10.1.2.2.3" xref="S2.SS1.p1.28.m10.1.2.2.3.cmml"><mi mathsize="142%" id="S2.SS1.p1.28.m10.1.2.2.3.2" xref="S2.SS1.p1.28.m10.1.2.2.3.2.cmml">S</mi><mrow id="S2.SS1.p1.28.m10.1.2.2.3.3" xref="S2.SS1.p1.28.m10.1.2.2.3.3.cmml"><mo mathsize="140%" id="S2.SS1.p1.28.m10.1.2.2.3.3a" xref="S2.SS1.p1.28.m10.1.2.2.3.3.cmml">+</mo><mi mathsize="140%" id="S2.SS1.p1.28.m10.1.2.2.3.3.2" xref="S2.SS1.p1.28.m10.1.2.2.3.3.2.cmml">d</mi></mrow></msub></msub><mo lspace="0em" rspace="0em" id="S2.SS1.p1.28.m10.1.2.1" xref="S2.SS1.p1.28.m10.1.2.1.cmml">‚Äã</mo><mrow id="S2.SS1.p1.28.m10.1.2.3.2" xref="S2.SS1.p1.28.m10.1.2.cmml"><mo stretchy="false" id="S2.SS1.p1.28.m10.1.2.3.2.1" xref="S2.SS1.p1.28.m10.1.2.cmml">(</mo><mi id="S2.SS1.p1.28.m10.1.1" xref="S2.SS1.p1.28.m10.1.1.cmml">O</mi><mo stretchy="false" id="S2.SS1.p1.28.m10.1.2.3.2.2" xref="S2.SS1.p1.28.m10.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.28.m10.1b"><apply id="S2.SS1.p1.28.m10.1.2.cmml" xref="S2.SS1.p1.28.m10.1.2"><times id="S2.SS1.p1.28.m10.1.2.1.cmml" xref="S2.SS1.p1.28.m10.1.2.1"></times><apply id="S2.SS1.p1.28.m10.1.2.2.cmml" xref="S2.SS1.p1.28.m10.1.2.2"><csymbol cd="ambiguous" id="S2.SS1.p1.28.m10.1.2.2.1.cmml" xref="S2.SS1.p1.28.m10.1.2.2">subscript</csymbol><ci id="S2.SS1.p1.28.m10.1.2.2.2.cmml" xref="S2.SS1.p1.28.m10.1.2.2.2">ùêª</ci><apply id="S2.SS1.p1.28.m10.1.2.2.3.cmml" xref="S2.SS1.p1.28.m10.1.2.2.3"><csymbol cd="ambiguous" id="S2.SS1.p1.28.m10.1.2.2.3.1.cmml" xref="S2.SS1.p1.28.m10.1.2.2.3">subscript</csymbol><ci id="S2.SS1.p1.28.m10.1.2.2.3.2.cmml" xref="S2.SS1.p1.28.m10.1.2.2.3.2">ùëÜ</ci><apply id="S2.SS1.p1.28.m10.1.2.2.3.3.cmml" xref="S2.SS1.p1.28.m10.1.2.2.3.3"><plus id="S2.SS1.p1.28.m10.1.2.2.3.3.1.cmml" xref="S2.SS1.p1.28.m10.1.2.2.3.3"></plus><ci id="S2.SS1.p1.28.m10.1.2.2.3.3.2.cmml" xref="S2.SS1.p1.28.m10.1.2.2.3.3.2">ùëë</ci></apply></apply></apply><ci id="S2.SS1.p1.28.m10.1.1.cmml" xref="S2.SS1.p1.28.m10.1.1">ùëÇ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.28.m10.1c">H_{\displaystyle S_{+d}}(O)</annotation></semantics></math> where <math id="S2.SS1.p1.29.m11.1" class="ltx_Math" alttext="\displaystyle S_{+d}=S\cup\{d\}" display="inline"><semantics id="S2.SS1.p1.29.m11.1a"><mrow id="S2.SS1.p1.29.m11.1.2" xref="S2.SS1.p1.29.m11.1.2.cmml"><msub id="S2.SS1.p1.29.m11.1.2.2" xref="S2.SS1.p1.29.m11.1.2.2.cmml"><mi id="S2.SS1.p1.29.m11.1.2.2.2" xref="S2.SS1.p1.29.m11.1.2.2.2.cmml">S</mi><mrow id="S2.SS1.p1.29.m11.1.2.2.3" xref="S2.SS1.p1.29.m11.1.2.2.3.cmml"><mo id="S2.SS1.p1.29.m11.1.2.2.3a" xref="S2.SS1.p1.29.m11.1.2.2.3.cmml">+</mo><mi id="S2.SS1.p1.29.m11.1.2.2.3.2" xref="S2.SS1.p1.29.m11.1.2.2.3.2.cmml">d</mi></mrow></msub><mo id="S2.SS1.p1.29.m11.1.2.1" xref="S2.SS1.p1.29.m11.1.2.1.cmml">=</mo><mrow id="S2.SS1.p1.29.m11.1.2.3" xref="S2.SS1.p1.29.m11.1.2.3.cmml"><mi id="S2.SS1.p1.29.m11.1.2.3.2" xref="S2.SS1.p1.29.m11.1.2.3.2.cmml">S</mi><mo id="S2.SS1.p1.29.m11.1.2.3.1" xref="S2.SS1.p1.29.m11.1.2.3.1.cmml">‚à™</mo><mrow id="S2.SS1.p1.29.m11.1.2.3.3.2" xref="S2.SS1.p1.29.m11.1.2.3.3.1.cmml"><mo stretchy="false" id="S2.SS1.p1.29.m11.1.2.3.3.2.1" xref="S2.SS1.p1.29.m11.1.2.3.3.1.cmml">{</mo><mi id="S2.SS1.p1.29.m11.1.1" xref="S2.SS1.p1.29.m11.1.1.cmml">d</mi><mo stretchy="false" id="S2.SS1.p1.29.m11.1.2.3.3.2.2" xref="S2.SS1.p1.29.m11.1.2.3.3.1.cmml">}</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.29.m11.1b"><apply id="S2.SS1.p1.29.m11.1.2.cmml" xref="S2.SS1.p1.29.m11.1.2"><eq id="S2.SS1.p1.29.m11.1.2.1.cmml" xref="S2.SS1.p1.29.m11.1.2.1"></eq><apply id="S2.SS1.p1.29.m11.1.2.2.cmml" xref="S2.SS1.p1.29.m11.1.2.2"><csymbol cd="ambiguous" id="S2.SS1.p1.29.m11.1.2.2.1.cmml" xref="S2.SS1.p1.29.m11.1.2.2">subscript</csymbol><ci id="S2.SS1.p1.29.m11.1.2.2.2.cmml" xref="S2.SS1.p1.29.m11.1.2.2.2">ùëÜ</ci><apply id="S2.SS1.p1.29.m11.1.2.2.3.cmml" xref="S2.SS1.p1.29.m11.1.2.2.3"><plus id="S2.SS1.p1.29.m11.1.2.2.3.1.cmml" xref="S2.SS1.p1.29.m11.1.2.2.3"></plus><ci id="S2.SS1.p1.29.m11.1.2.2.3.2.cmml" xref="S2.SS1.p1.29.m11.1.2.2.3.2">ùëë</ci></apply></apply><apply id="S2.SS1.p1.29.m11.1.2.3.cmml" xref="S2.SS1.p1.29.m11.1.2.3"><union id="S2.SS1.p1.29.m11.1.2.3.1.cmml" xref="S2.SS1.p1.29.m11.1.2.3.1"></union><ci id="S2.SS1.p1.29.m11.1.2.3.2.cmml" xref="S2.SS1.p1.29.m11.1.2.3.2">ùëÜ</ci><set id="S2.SS1.p1.29.m11.1.2.3.3.1.cmml" xref="S2.SS1.p1.29.m11.1.2.3.3.2"><ci id="S2.SS1.p1.29.m11.1.1.cmml" xref="S2.SS1.p1.29.m11.1.1">ùëë</ci></set></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.29.m11.1c">\displaystyle S_{+d}=S\cup\{d\}</annotation></semantics></math>. Therefore, to optimize the diversity over organisms and chemicals, the document <math id="S2.SS1.p1.30.m12.1" class="ltx_math_unparsed" alttext="d*" display="inline"><semantics id="S2.SS1.p1.30.m12.1a"><mrow id="S2.SS1.p1.30.m12.1b"><mi id="S2.SS1.p1.30.m12.1.1">d</mi><mo lspace="0.222em" id="S2.SS1.p1.30.m12.1.2">‚àó</mo></mrow><annotation encoding="application/x-tex" id="S2.SS1.p1.30.m12.1c">d*</annotation></semantics></math> added to <math id="S2.SS1.p1.31.m13.1" class="ltx_Math" alttext="S" display="inline"><semantics id="S2.SS1.p1.31.m13.1a"><mi id="S2.SS1.p1.31.m13.1.1" xref="S2.SS1.p1.31.m13.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.31.m13.1b"><ci id="S2.SS1.p1.31.m13.1.1.cmml" xref="S2.SS1.p1.31.m13.1.1">ùëÜ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.31.m13.1c">S</annotation></semantics></math> minimizes the distance to the utopian point <math id="S2.SS1.p1.32.m14.4" class="ltx_Math" alttext="(log|O|,log|C|)" display="inline"><semantics id="S2.SS1.p1.32.m14.4a"><mrow id="S2.SS1.p1.32.m14.4.4.2" xref="S2.SS1.p1.32.m14.4.4.3.cmml"><mo stretchy="false" id="S2.SS1.p1.32.m14.4.4.2.3" xref="S2.SS1.p1.32.m14.4.4.3.cmml">(</mo><mrow id="S2.SS1.p1.32.m14.3.3.1.1" xref="S2.SS1.p1.32.m14.3.3.1.1.cmml"><mi id="S2.SS1.p1.32.m14.3.3.1.1.2" xref="S2.SS1.p1.32.m14.3.3.1.1.2.cmml">l</mi><mo lspace="0em" rspace="0em" id="S2.SS1.p1.32.m14.3.3.1.1.1" xref="S2.SS1.p1.32.m14.3.3.1.1.1.cmml">‚Äã</mo><mi id="S2.SS1.p1.32.m14.3.3.1.1.3" xref="S2.SS1.p1.32.m14.3.3.1.1.3.cmml">o</mi><mo lspace="0em" rspace="0em" id="S2.SS1.p1.32.m14.3.3.1.1.1a" xref="S2.SS1.p1.32.m14.3.3.1.1.1.cmml">‚Äã</mo><mi id="S2.SS1.p1.32.m14.3.3.1.1.4" xref="S2.SS1.p1.32.m14.3.3.1.1.4.cmml">g</mi><mo lspace="0em" rspace="0em" id="S2.SS1.p1.32.m14.3.3.1.1.1b" xref="S2.SS1.p1.32.m14.3.3.1.1.1.cmml">‚Äã</mo><mrow id="S2.SS1.p1.32.m14.3.3.1.1.5.2" xref="S2.SS1.p1.32.m14.3.3.1.1.5.1.cmml"><mo stretchy="false" id="S2.SS1.p1.32.m14.3.3.1.1.5.2.1" xref="S2.SS1.p1.32.m14.3.3.1.1.5.1.1.cmml">|</mo><mi id="S2.SS1.p1.32.m14.1.1" xref="S2.SS1.p1.32.m14.1.1.cmml">O</mi><mo stretchy="false" id="S2.SS1.p1.32.m14.3.3.1.1.5.2.2" xref="S2.SS1.p1.32.m14.3.3.1.1.5.1.1.cmml">|</mo></mrow></mrow><mo id="S2.SS1.p1.32.m14.4.4.2.4" xref="S2.SS1.p1.32.m14.4.4.3.cmml">,</mo><mrow id="S2.SS1.p1.32.m14.4.4.2.2" xref="S2.SS1.p1.32.m14.4.4.2.2.cmml"><mi id="S2.SS1.p1.32.m14.4.4.2.2.2" xref="S2.SS1.p1.32.m14.4.4.2.2.2.cmml">l</mi><mo lspace="0em" rspace="0em" id="S2.SS1.p1.32.m14.4.4.2.2.1" xref="S2.SS1.p1.32.m14.4.4.2.2.1.cmml">‚Äã</mo><mi id="S2.SS1.p1.32.m14.4.4.2.2.3" xref="S2.SS1.p1.32.m14.4.4.2.2.3.cmml">o</mi><mo lspace="0em" rspace="0em" id="S2.SS1.p1.32.m14.4.4.2.2.1a" xref="S2.SS1.p1.32.m14.4.4.2.2.1.cmml">‚Äã</mo><mi id="S2.SS1.p1.32.m14.4.4.2.2.4" xref="S2.SS1.p1.32.m14.4.4.2.2.4.cmml">g</mi><mo lspace="0em" rspace="0em" id="S2.SS1.p1.32.m14.4.4.2.2.1b" xref="S2.SS1.p1.32.m14.4.4.2.2.1.cmml">‚Äã</mo><mrow id="S2.SS1.p1.32.m14.4.4.2.2.5.2" xref="S2.SS1.p1.32.m14.4.4.2.2.5.1.cmml"><mo stretchy="false" id="S2.SS1.p1.32.m14.4.4.2.2.5.2.1" xref="S2.SS1.p1.32.m14.4.4.2.2.5.1.1.cmml">|</mo><mi id="S2.SS1.p1.32.m14.2.2" xref="S2.SS1.p1.32.m14.2.2.cmml">C</mi><mo stretchy="false" id="S2.SS1.p1.32.m14.4.4.2.2.5.2.2" xref="S2.SS1.p1.32.m14.4.4.2.2.5.1.1.cmml">|</mo></mrow></mrow><mo stretchy="false" id="S2.SS1.p1.32.m14.4.4.2.5" xref="S2.SS1.p1.32.m14.4.4.3.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.32.m14.4b"><interval closure="open" id="S2.SS1.p1.32.m14.4.4.3.cmml" xref="S2.SS1.p1.32.m14.4.4.2"><apply id="S2.SS1.p1.32.m14.3.3.1.1.cmml" xref="S2.SS1.p1.32.m14.3.3.1.1"><times id="S2.SS1.p1.32.m14.3.3.1.1.1.cmml" xref="S2.SS1.p1.32.m14.3.3.1.1.1"></times><ci id="S2.SS1.p1.32.m14.3.3.1.1.2.cmml" xref="S2.SS1.p1.32.m14.3.3.1.1.2">ùëô</ci><ci id="S2.SS1.p1.32.m14.3.3.1.1.3.cmml" xref="S2.SS1.p1.32.m14.3.3.1.1.3">ùëú</ci><ci id="S2.SS1.p1.32.m14.3.3.1.1.4.cmml" xref="S2.SS1.p1.32.m14.3.3.1.1.4">ùëî</ci><apply id="S2.SS1.p1.32.m14.3.3.1.1.5.1.cmml" xref="S2.SS1.p1.32.m14.3.3.1.1.5.2"><abs id="S2.SS1.p1.32.m14.3.3.1.1.5.1.1.cmml" xref="S2.SS1.p1.32.m14.3.3.1.1.5.2.1"></abs><ci id="S2.SS1.p1.32.m14.1.1.cmml" xref="S2.SS1.p1.32.m14.1.1">ùëÇ</ci></apply></apply><apply id="S2.SS1.p1.32.m14.4.4.2.2.cmml" xref="S2.SS1.p1.32.m14.4.4.2.2"><times id="S2.SS1.p1.32.m14.4.4.2.2.1.cmml" xref="S2.SS1.p1.32.m14.4.4.2.2.1"></times><ci id="S2.SS1.p1.32.m14.4.4.2.2.2.cmml" xref="S2.SS1.p1.32.m14.4.4.2.2.2">ùëô</ci><ci id="S2.SS1.p1.32.m14.4.4.2.2.3.cmml" xref="S2.SS1.p1.32.m14.4.4.2.2.3">ùëú</ci><ci id="S2.SS1.p1.32.m14.4.4.2.2.4.cmml" xref="S2.SS1.p1.32.m14.4.4.2.2.4">ùëî</ci><apply id="S2.SS1.p1.32.m14.4.4.2.2.5.1.cmml" xref="S2.SS1.p1.32.m14.4.4.2.2.5.2"><abs id="S2.SS1.p1.32.m14.4.4.2.2.5.1.1.cmml" xref="S2.SS1.p1.32.m14.4.4.2.2.5.2.1"></abs><ci id="S2.SS1.p1.32.m14.2.2.cmml" xref="S2.SS1.p1.32.m14.2.2">ùê∂</ci></apply></apply></interval></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.32.m14.4c">(log|O|,log|C|)</annotation></semantics></math> (maximal observable entropy over organisms and chemicals):</p>
<table id="S2.E2" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S2.E2.m1.1" class="ltx_math_unparsed" alttext="d*=\operatorname*{arg\,min}_{d}||(H_{\displaystyle S_{+d}}(O),H_{\displaystyle S_{+d}}(C))-(log|O|,log|C|)||." display="block"><semantics id="S2.E2.m1.1a"><mrow id="S2.E2.m1.1b"><mi id="S2.E2.m1.1.1">d</mi><mo lspace="0.222em" rspace="0em" id="S2.E2.m1.1.2">‚àó</mo><mo lspace="0em" id="S2.E2.m1.1.3">=</mo><munder id="S2.E2.m1.1.4"><mrow id="S2.E2.m1.1.4.2"><mi id="S2.E2.m1.1.4.2.2">arg</mi><mo lspace="0.170em" rspace="0em" id="S2.E2.m1.1.4.2.1">‚Äã</mo><mi id="S2.E2.m1.1.4.2.3">min</mi></mrow><mi id="S2.E2.m1.1.4.3">d</mi></munder><mo fence="false" rspace="0.167em" stretchy="false" id="S2.E2.m1.1.5">|</mo><mo fence="false" rspace="0.167em" stretchy="false" id="S2.E2.m1.1.6">|</mo><mrow id="S2.E2.m1.1.7"><mo stretchy="false" id="S2.E2.m1.1.7.1">(</mo><msub id="S2.E2.m1.1.7.2"><mi id="S2.E2.m1.1.7.2.2">H</mi><msub id="S2.E2.m1.1.7.2.3"><mi mathsize="142%" id="S2.E2.m1.1.7.2.3.2">S</mi><mrow id="S2.E2.m1.1.7.2.3.3"><mo mathsize="140%" id="S2.E2.m1.1.7.2.3.3a">+</mo><mi mathsize="140%" id="S2.E2.m1.1.7.2.3.3.2">d</mi></mrow></msub></msub><mrow id="S2.E2.m1.1.7.3"><mo stretchy="false" id="S2.E2.m1.1.7.3.1">(</mo><mi id="S2.E2.m1.1.7.3.2">O</mi><mo stretchy="false" id="S2.E2.m1.1.7.3.3">)</mo></mrow><mo id="S2.E2.m1.1.7.4">,</mo><msub id="S2.E2.m1.1.7.5"><mi id="S2.E2.m1.1.7.5.2">H</mi><msub id="S2.E2.m1.1.7.5.3"><mi mathsize="142%" id="S2.E2.m1.1.7.5.3.2">S</mi><mrow id="S2.E2.m1.1.7.5.3.3"><mo mathsize="140%" id="S2.E2.m1.1.7.5.3.3a">+</mo><mi mathsize="140%" id="S2.E2.m1.1.7.5.3.3.2">d</mi></mrow></msub></msub><mrow id="S2.E2.m1.1.7.6"><mo stretchy="false" id="S2.E2.m1.1.7.6.1">(</mo><mi id="S2.E2.m1.1.7.6.2">C</mi><mo stretchy="false" id="S2.E2.m1.1.7.6.3">)</mo></mrow><mo stretchy="false" id="S2.E2.m1.1.7.7">)</mo></mrow><mo id="S2.E2.m1.1.8">‚àí</mo><mrow id="S2.E2.m1.1.9"><mo stretchy="false" id="S2.E2.m1.1.9.1">(</mo><mi id="S2.E2.m1.1.9.2">l</mi><mi id="S2.E2.m1.1.9.3">o</mi><mi id="S2.E2.m1.1.9.4">g</mi><mo fence="false" rspace="0.167em" stretchy="false" id="S2.E2.m1.1.9.5">|</mo><mi id="S2.E2.m1.1.9.6">O</mi><mo fence="false" rspace="0.167em" stretchy="false" id="S2.E2.m1.1.9.7">|</mo><mo id="S2.E2.m1.1.9.8">,</mo><mi id="S2.E2.m1.1.9.9">l</mi><mi id="S2.E2.m1.1.9.10">o</mi><mi id="S2.E2.m1.1.9.11">g</mi><mo fence="false" rspace="0.167em" stretchy="false" id="S2.E2.m1.1.9.12">|</mo><mi id="S2.E2.m1.1.9.13">C</mi><mo fence="false" rspace="0.167em" stretchy="false" id="S2.E2.m1.1.9.14">|</mo><mo stretchy="false" id="S2.E2.m1.1.9.15">)</mo></mrow><mo fence="false" rspace="0.167em" stretchy="false" id="S2.E2.m1.1.10">|</mo><mo fence="false" stretchy="false" id="S2.E2.m1.1.11">|</mo><mo lspace="0.167em" id="S2.E2.m1.1.12">.</mo></mrow><annotation encoding="application/x-tex" id="S2.E2.m1.1c">d*=\operatorname*{arg\,min}_{d}||(H_{\displaystyle S_{+d}}(O),H_{\displaystyle S_{+d}}(C))-(log|O|,log|C|)||.</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(2)</span></td>
</tr></tbody>
</table>
<p id="S2.SS1.p1.42" class="ltx_p">The proposed sampling approach is a simple greedy algorithm that, at each step, selects and adds the new document <math id="S2.SS1.p1.33.m1.1" class="ltx_math_unparsed" alttext="d*" display="inline"><semantics id="S2.SS1.p1.33.m1.1a"><mrow id="S2.SS1.p1.33.m1.1b"><mi id="S2.SS1.p1.33.m1.1.1">d</mi><mo lspace="0.222em" id="S2.SS1.p1.33.m1.1.2">‚àó</mo></mrow><annotation encoding="application/x-tex" id="S2.SS1.p1.33.m1.1c">d*</annotation></semantics></math> from <math id="S2.SS1.p1.34.m2.1" class="ltx_Math" alttext="D" display="inline"><semantics id="S2.SS1.p1.34.m2.1a"><mi id="S2.SS1.p1.34.m2.1.1" xref="S2.SS1.p1.34.m2.1.1.cmml">D</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.34.m2.1b"><ci id="S2.SS1.p1.34.m2.1.1.cmml" xref="S2.SS1.p1.34.m2.1.1">ùê∑</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.34.m2.1c">D</annotation></semantics></math>, maximising the diversity on organisms and chemicals (see eq.<a href="#S2.E2" title="In 2.1 Greedy Maximum Entropy Sampling (GME) ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>). We refer to it as diversity-sampling. As in ecology, diversity is intrinsically related to the number of distinct biological entities (e.g organisms) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib72" title="" class="ltx_ref">72</a>]</cite>, but, for the purpose of using the samples as training or evaluation sets, the balance of the distribution is prioritized over a higher number of rare entities <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib70" title="" class="ltx_ref">70</a>]</cite>. This latter behavior is a natural consequence of using the Shannon‚Äôs entropy. From this perspective, the method can also be seen as a ranking procedure, and a sample is determined by selecting the first top <math id="S2.SS1.p1.35.m3.1" class="ltx_Math" alttext="n" display="inline"><semantics id="S2.SS1.p1.35.m3.1a"><mi id="S2.SS1.p1.35.m3.1.1" xref="S2.SS1.p1.35.m3.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.35.m3.1b"><ci id="S2.SS1.p1.35.m3.1.1.cmml" xref="S2.SS1.p1.35.m3.1.1">ùëõ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.35.m3.1c">n</annotation></semantics></math> ranked items. The selection of an appropriate sample size <math id="S2.SS1.p1.36.m4.1" class="ltx_Math" alttext="l" display="inline"><semantics id="S2.SS1.p1.36.m4.1a"><mi id="S2.SS1.p1.36.m4.1.1" xref="S2.SS1.p1.36.m4.1.1.cmml">l</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.36.m4.1b"><ci id="S2.SS1.p1.36.m4.1.1.cmml" xref="S2.SS1.p1.36.m4.1.1">ùëô</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.36.m4.1c">l</annotation></semantics></math> is also a critical, but often overlooked factor. By monitoring <math id="S2.SS1.p1.37.m5.1" class="ltx_Math" alttext="H_{S}(O)" display="inline"><semantics id="S2.SS1.p1.37.m5.1a"><mrow id="S2.SS1.p1.37.m5.1.2" xref="S2.SS1.p1.37.m5.1.2.cmml"><msub id="S2.SS1.p1.37.m5.1.2.2" xref="S2.SS1.p1.37.m5.1.2.2.cmml"><mi id="S2.SS1.p1.37.m5.1.2.2.2" xref="S2.SS1.p1.37.m5.1.2.2.2.cmml">H</mi><mi id="S2.SS1.p1.37.m5.1.2.2.3" xref="S2.SS1.p1.37.m5.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.SS1.p1.37.m5.1.2.1" xref="S2.SS1.p1.37.m5.1.2.1.cmml">‚Äã</mo><mrow id="S2.SS1.p1.37.m5.1.2.3.2" xref="S2.SS1.p1.37.m5.1.2.cmml"><mo stretchy="false" id="S2.SS1.p1.37.m5.1.2.3.2.1" xref="S2.SS1.p1.37.m5.1.2.cmml">(</mo><mi id="S2.SS1.p1.37.m5.1.1" xref="S2.SS1.p1.37.m5.1.1.cmml">O</mi><mo stretchy="false" id="S2.SS1.p1.37.m5.1.2.3.2.2" xref="S2.SS1.p1.37.m5.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.37.m5.1b"><apply id="S2.SS1.p1.37.m5.1.2.cmml" xref="S2.SS1.p1.37.m5.1.2"><times id="S2.SS1.p1.37.m5.1.2.1.cmml" xref="S2.SS1.p1.37.m5.1.2.1"></times><apply id="S2.SS1.p1.37.m5.1.2.2.cmml" xref="S2.SS1.p1.37.m5.1.2.2"><csymbol cd="ambiguous" id="S2.SS1.p1.37.m5.1.2.2.1.cmml" xref="S2.SS1.p1.37.m5.1.2.2">subscript</csymbol><ci id="S2.SS1.p1.37.m5.1.2.2.2.cmml" xref="S2.SS1.p1.37.m5.1.2.2.2">ùêª</ci><ci id="S2.SS1.p1.37.m5.1.2.2.3.cmml" xref="S2.SS1.p1.37.m5.1.2.2.3">ùëÜ</ci></apply><ci id="S2.SS1.p1.37.m5.1.1.cmml" xref="S2.SS1.p1.37.m5.1.1">ùëÇ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.37.m5.1c">H_{S}(O)</annotation></semantics></math> and <math id="S2.SS1.p1.38.m6.1" class="ltx_Math" alttext="H_{S}(C)" display="inline"><semantics id="S2.SS1.p1.38.m6.1a"><mrow id="S2.SS1.p1.38.m6.1.2" xref="S2.SS1.p1.38.m6.1.2.cmml"><msub id="S2.SS1.p1.38.m6.1.2.2" xref="S2.SS1.p1.38.m6.1.2.2.cmml"><mi id="S2.SS1.p1.38.m6.1.2.2.2" xref="S2.SS1.p1.38.m6.1.2.2.2.cmml">H</mi><mi id="S2.SS1.p1.38.m6.1.2.2.3" xref="S2.SS1.p1.38.m6.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.SS1.p1.38.m6.1.2.1" xref="S2.SS1.p1.38.m6.1.2.1.cmml">‚Äã</mo><mrow id="S2.SS1.p1.38.m6.1.2.3.2" xref="S2.SS1.p1.38.m6.1.2.cmml"><mo stretchy="false" id="S2.SS1.p1.38.m6.1.2.3.2.1" xref="S2.SS1.p1.38.m6.1.2.cmml">(</mo><mi id="S2.SS1.p1.38.m6.1.1" xref="S2.SS1.p1.38.m6.1.1.cmml">C</mi><mo stretchy="false" id="S2.SS1.p1.38.m6.1.2.3.2.2" xref="S2.SS1.p1.38.m6.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.38.m6.1b"><apply id="S2.SS1.p1.38.m6.1.2.cmml" xref="S2.SS1.p1.38.m6.1.2"><times id="S2.SS1.p1.38.m6.1.2.1.cmml" xref="S2.SS1.p1.38.m6.1.2.1"></times><apply id="S2.SS1.p1.38.m6.1.2.2.cmml" xref="S2.SS1.p1.38.m6.1.2.2"><csymbol cd="ambiguous" id="S2.SS1.p1.38.m6.1.2.2.1.cmml" xref="S2.SS1.p1.38.m6.1.2.2">subscript</csymbol><ci id="S2.SS1.p1.38.m6.1.2.2.2.cmml" xref="S2.SS1.p1.38.m6.1.2.2.2">ùêª</ci><ci id="S2.SS1.p1.38.m6.1.2.2.3.cmml" xref="S2.SS1.p1.38.m6.1.2.2.3">ùëÜ</ci></apply><ci id="S2.SS1.p1.38.m6.1.1.cmml" xref="S2.SS1.p1.38.m6.1.1">ùê∂</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.38.m6.1c">H_{S}(C)</annotation></semantics></math> during the iterative construction of <math id="S2.SS1.p1.39.m7.1" class="ltx_Math" alttext="S" display="inline"><semantics id="S2.SS1.p1.39.m7.1a"><mi id="S2.SS1.p1.39.m7.1.1" xref="S2.SS1.p1.39.m7.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.39.m7.1b"><ci id="S2.SS1.p1.39.m7.1.1.cmml" xref="S2.SS1.p1.39.m7.1.1">ùëÜ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.39.m7.1c">S</annotation></semantics></math> (until <math id="S2.SS1.p1.40.m8.1" class="ltx_Math" alttext="l=L" display="inline"><semantics id="S2.SS1.p1.40.m8.1a"><mrow id="S2.SS1.p1.40.m8.1.1" xref="S2.SS1.p1.40.m8.1.1.cmml"><mi id="S2.SS1.p1.40.m8.1.1.2" xref="S2.SS1.p1.40.m8.1.1.2.cmml">l</mi><mo id="S2.SS1.p1.40.m8.1.1.1" xref="S2.SS1.p1.40.m8.1.1.1.cmml">=</mo><mi id="S2.SS1.p1.40.m8.1.1.3" xref="S2.SS1.p1.40.m8.1.1.3.cmml">L</mi></mrow><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.40.m8.1b"><apply id="S2.SS1.p1.40.m8.1.1.cmml" xref="S2.SS1.p1.40.m8.1.1"><eq id="S2.SS1.p1.40.m8.1.1.1.cmml" xref="S2.SS1.p1.40.m8.1.1.1"></eq><ci id="S2.SS1.p1.40.m8.1.1.2.cmml" xref="S2.SS1.p1.40.m8.1.1.2">ùëô</ci><ci id="S2.SS1.p1.40.m8.1.1.3.cmml" xref="S2.SS1.p1.40.m8.1.1.3">ùêø</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.40.m8.1c">l=L</annotation></semantics></math>), it is possible to determine the step <math id="S2.SS1.p1.41.m9.1" class="ltx_Math" alttext="l" display="inline"><semantics id="S2.SS1.p1.41.m9.1a"><mi id="S2.SS1.p1.41.m9.1.1" xref="S2.SS1.p1.41.m9.1.1.cmml">l</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.41.m9.1b"><ci id="S2.SS1.p1.41.m9.1.1.cmml" xref="S2.SS1.p1.41.m9.1.1">ùëô</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.41.m9.1c">l</annotation></semantics></math> at which diversity starts to deteriorate and sampling should be stopped, i.e. when the new added documents provide relationships for already frequently reported entities in <math id="S2.SS1.p1.42.m10.1" class="ltx_Math" alttext="S" display="inline"><semantics id="S2.SS1.p1.42.m10.1a"><mi id="S2.SS1.p1.42.m10.1.1" xref="S2.SS1.p1.42.m10.1.1.cmml">S</mi><annotation-xml encoding="MathML-Content" id="S2.SS1.p1.42.m10.1b"><ci id="S2.SS1.p1.42.m10.1.1.cmml" xref="S2.SS1.p1.42.m10.1.1">ùëÜ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS1.p1.42.m10.1c">S</annotation></semantics></math>. The GME-sampler initially designed for the purpose of extracting data from LOTUS has also been implemented as a standalone library and is readily applicable to alternative contexts with <span id="S2.SS1.p1.42.1" class="ltx_text ltx_font_typewriter">N-ary</span> relations (e.g Pharmacogenomics: <span id="S2.SS1.p1.42.2" class="ltx_text ltx_font_italic">Variant - Drug - Adverse event</span>). See code available at <a target="_blank" href="https://github.com/idiap/gme-sampler" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/idiap/gme-sampler</a>.</p>
</div>
</section>
<section id="S2.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.2 </span>Different strategies for Relation Extraction</h3>

<section id="S2.SS2.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.2.1 </span>Few-shot learning with open LLM</h4>

<div id="S2.SS2.SSS1.p1" class="ltx_para ltx_noindent">
<p id="S2.SS2.SSS1.p1.1" class="ltx_p">In few-shot settings, the model is prompted with <math id="S2.SS2.SSS1.p1.1.m1.1" class="ltx_Math" alttext="K" display="inline"><semantics id="S2.SS2.SSS1.p1.1.m1.1a"><mi id="S2.SS2.SSS1.p1.1.m1.1.1" xref="S2.SS2.SSS1.p1.1.m1.1.1.cmml">K</mi><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS1.p1.1.m1.1b"><ci id="S2.SS2.SSS1.p1.1.m1.1.1.cmml" xref="S2.SS2.SSS1.p1.1.m1.1.1">ùêæ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS1.p1.1.m1.1c">K</annotation></semantics></math> input-completion example pairs and one final input, with the objective of accurately generating the completion for the final input <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib12" title="" class="ltx_ref">12</a>]</cite>. Considering the limited size of the context-window (2048 tokens), we carefully selected archetypical parts of diverse abstracts that exemplify various patterns and specificities of reporting NP relationships. More details in supplementary <a href="#S1.SS1a" title="S1.1 Few-shot learning details ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S1.1</span></a>.</p>
</div>
</section>
<section id="S2.SS2.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.2.2 </span>Fine-tuning</h4>

<div id="S2.SS2.SSS2.p1" class="ltx_para ltx_noindent">
<p id="S2.SS2.SSS2.p1.5" class="ltx_p">The task was framed as a special case of end-to-end relation extraction with a single relation. Several factors influenced this design: the need for a generalised NER component that is not dictionary-based, as new species and NPs are discovered and named continuously, the scarcity of training data to segment the task in a NER-RE pipeline, and, the specific decoding requirements related to the NP relationships of "Multiples" entities. Given an input text <math id="S2.SS2.SSS2.p1.1.m1.1" class="ltx_Math" alttext="X" display="inline"><semantics id="S2.SS2.SSS2.p1.1.m1.1a"><mi id="S2.SS2.SSS2.p1.1.m1.1.1" xref="S2.SS2.SSS2.p1.1.m1.1.1.cmml">X</mi><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.1.m1.1b"><ci id="S2.SS2.SSS2.p1.1.m1.1.1.cmml" xref="S2.SS2.SSS2.p1.1.m1.1.1">ùëã</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.1.m1.1c">X</annotation></semantics></math> reporting relations <math id="S2.SS2.SSS2.p1.2.m2.3" class="ltx_Math" alttext="\{r_{1},r_{2},r_{3}\}" display="inline"><semantics id="S2.SS2.SSS2.p1.2.m2.3a"><mrow id="S2.SS2.SSS2.p1.2.m2.3.3.3" xref="S2.SS2.SSS2.p1.2.m2.3.3.4.cmml"><mo stretchy="false" id="S2.SS2.SSS2.p1.2.m2.3.3.3.4" xref="S2.SS2.SSS2.p1.2.m2.3.3.4.cmml">{</mo><msub id="S2.SS2.SSS2.p1.2.m2.1.1.1.1" xref="S2.SS2.SSS2.p1.2.m2.1.1.1.1.cmml"><mi id="S2.SS2.SSS2.p1.2.m2.1.1.1.1.2" xref="S2.SS2.SSS2.p1.2.m2.1.1.1.1.2.cmml">r</mi><mn id="S2.SS2.SSS2.p1.2.m2.1.1.1.1.3" xref="S2.SS2.SSS2.p1.2.m2.1.1.1.1.3.cmml">1</mn></msub><mo id="S2.SS2.SSS2.p1.2.m2.3.3.3.5" xref="S2.SS2.SSS2.p1.2.m2.3.3.4.cmml">,</mo><msub id="S2.SS2.SSS2.p1.2.m2.2.2.2.2" xref="S2.SS2.SSS2.p1.2.m2.2.2.2.2.cmml"><mi id="S2.SS2.SSS2.p1.2.m2.2.2.2.2.2" xref="S2.SS2.SSS2.p1.2.m2.2.2.2.2.2.cmml">r</mi><mn id="S2.SS2.SSS2.p1.2.m2.2.2.2.2.3" xref="S2.SS2.SSS2.p1.2.m2.2.2.2.2.3.cmml">2</mn></msub><mo id="S2.SS2.SSS2.p1.2.m2.3.3.3.6" xref="S2.SS2.SSS2.p1.2.m2.3.3.4.cmml">,</mo><msub id="S2.SS2.SSS2.p1.2.m2.3.3.3.3" xref="S2.SS2.SSS2.p1.2.m2.3.3.3.3.cmml"><mi id="S2.SS2.SSS2.p1.2.m2.3.3.3.3.2" xref="S2.SS2.SSS2.p1.2.m2.3.3.3.3.2.cmml">r</mi><mn id="S2.SS2.SSS2.p1.2.m2.3.3.3.3.3" xref="S2.SS2.SSS2.p1.2.m2.3.3.3.3.3.cmml">3</mn></msub><mo stretchy="false" id="S2.SS2.SSS2.p1.2.m2.3.3.3.7" xref="S2.SS2.SSS2.p1.2.m2.3.3.4.cmml">}</mo></mrow><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.2.m2.3b"><set id="S2.SS2.SSS2.p1.2.m2.3.3.4.cmml" xref="S2.SS2.SSS2.p1.2.m2.3.3.3"><apply id="S2.SS2.SSS2.p1.2.m2.1.1.1.1.cmml" xref="S2.SS2.SSS2.p1.2.m2.1.1.1.1"><csymbol cd="ambiguous" id="S2.SS2.SSS2.p1.2.m2.1.1.1.1.1.cmml" xref="S2.SS2.SSS2.p1.2.m2.1.1.1.1">subscript</csymbol><ci id="S2.SS2.SSS2.p1.2.m2.1.1.1.1.2.cmml" xref="S2.SS2.SSS2.p1.2.m2.1.1.1.1.2">ùëü</ci><cn type="integer" id="S2.SS2.SSS2.p1.2.m2.1.1.1.1.3.cmml" xref="S2.SS2.SSS2.p1.2.m2.1.1.1.1.3">1</cn></apply><apply id="S2.SS2.SSS2.p1.2.m2.2.2.2.2.cmml" xref="S2.SS2.SSS2.p1.2.m2.2.2.2.2"><csymbol cd="ambiguous" id="S2.SS2.SSS2.p1.2.m2.2.2.2.2.1.cmml" xref="S2.SS2.SSS2.p1.2.m2.2.2.2.2">subscript</csymbol><ci id="S2.SS2.SSS2.p1.2.m2.2.2.2.2.2.cmml" xref="S2.SS2.SSS2.p1.2.m2.2.2.2.2.2">ùëü</ci><cn type="integer" id="S2.SS2.SSS2.p1.2.m2.2.2.2.2.3.cmml" xref="S2.SS2.SSS2.p1.2.m2.2.2.2.2.3">2</cn></apply><apply id="S2.SS2.SSS2.p1.2.m2.3.3.3.3.cmml" xref="S2.SS2.SSS2.p1.2.m2.3.3.3.3"><csymbol cd="ambiguous" id="S2.SS2.SSS2.p1.2.m2.3.3.3.3.1.cmml" xref="S2.SS2.SSS2.p1.2.m2.3.3.3.3">subscript</csymbol><ci id="S2.SS2.SSS2.p1.2.m2.3.3.3.3.2.cmml" xref="S2.SS2.SSS2.p1.2.m2.3.3.3.3.2">ùëü</ci><cn type="integer" id="S2.SS2.SSS2.p1.2.m2.3.3.3.3.3.cmml" xref="S2.SS2.SSS2.p1.2.m2.3.3.3.3.3">3</cn></apply></set></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.2.m2.3c">\{r_{1},r_{2},r_{3}\}</annotation></semantics></math> between organisms <math id="S2.SS2.SSS2.p1.3.m3.1" class="ltx_Math" alttext="o_{1}" display="inline"><semantics id="S2.SS2.SSS2.p1.3.m3.1a"><msub id="S2.SS2.SSS2.p1.3.m3.1.1" xref="S2.SS2.SSS2.p1.3.m3.1.1.cmml"><mi id="S2.SS2.SSS2.p1.3.m3.1.1.2" xref="S2.SS2.SSS2.p1.3.m3.1.1.2.cmml">o</mi><mn id="S2.SS2.SSS2.p1.3.m3.1.1.3" xref="S2.SS2.SSS2.p1.3.m3.1.1.3.cmml">1</mn></msub><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.3.m3.1b"><apply id="S2.SS2.SSS2.p1.3.m3.1.1.cmml" xref="S2.SS2.SSS2.p1.3.m3.1.1"><csymbol cd="ambiguous" id="S2.SS2.SSS2.p1.3.m3.1.1.1.cmml" xref="S2.SS2.SSS2.p1.3.m3.1.1">subscript</csymbol><ci id="S2.SS2.SSS2.p1.3.m3.1.1.2.cmml" xref="S2.SS2.SSS2.p1.3.m3.1.1.2">ùëú</ci><cn type="integer" id="S2.SS2.SSS2.p1.3.m3.1.1.3.cmml" xref="S2.SS2.SSS2.p1.3.m3.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.3.m3.1c">o_{1}</annotation></semantics></math> and natural-products <math id="S2.SS2.SSS2.p1.4.m4.1" class="ltx_Math" alttext="c_{1:3}" display="inline"><semantics id="S2.SS2.SSS2.p1.4.m4.1a"><msub id="S2.SS2.SSS2.p1.4.m4.1.1" xref="S2.SS2.SSS2.p1.4.m4.1.1.cmml"><mi id="S2.SS2.SSS2.p1.4.m4.1.1.2" xref="S2.SS2.SSS2.p1.4.m4.1.1.2.cmml">c</mi><mrow id="S2.SS2.SSS2.p1.4.m4.1.1.3" xref="S2.SS2.SSS2.p1.4.m4.1.1.3.cmml"><mn id="S2.SS2.SSS2.p1.4.m4.1.1.3.2" xref="S2.SS2.SSS2.p1.4.m4.1.1.3.2.cmml">1</mn><mo lspace="0.278em" rspace="0.278em" id="S2.SS2.SSS2.p1.4.m4.1.1.3.1" xref="S2.SS2.SSS2.p1.4.m4.1.1.3.1.cmml">:</mo><mn id="S2.SS2.SSS2.p1.4.m4.1.1.3.3" xref="S2.SS2.SSS2.p1.4.m4.1.1.3.3.cmml">3</mn></mrow></msub><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.4.m4.1b"><apply id="S2.SS2.SSS2.p1.4.m4.1.1.cmml" xref="S2.SS2.SSS2.p1.4.m4.1.1"><csymbol cd="ambiguous" id="S2.SS2.SSS2.p1.4.m4.1.1.1.cmml" xref="S2.SS2.SSS2.p1.4.m4.1.1">subscript</csymbol><ci id="S2.SS2.SSS2.p1.4.m4.1.1.2.cmml" xref="S2.SS2.SSS2.p1.4.m4.1.1.2">ùëê</ci><apply id="S2.SS2.SSS2.p1.4.m4.1.1.3.cmml" xref="S2.SS2.SSS2.p1.4.m4.1.1.3"><ci id="S2.SS2.SSS2.p1.4.m4.1.1.3.1.cmml" xref="S2.SS2.SSS2.p1.4.m4.1.1.3.1">:</ci><cn type="integer" id="S2.SS2.SSS2.p1.4.m4.1.1.3.2.cmml" xref="S2.SS2.SSS2.p1.4.m4.1.1.3.2">1</cn><cn type="integer" id="S2.SS2.SSS2.p1.4.m4.1.1.3.3.cmml" xref="S2.SS2.SSS2.p1.4.m4.1.1.3.3">3</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.4.m4.1c">c_{1:3}</annotation></semantics></math> like: "Three new metabolites, gloeophyllins A-C (1-3) have been isolated from the solid cultures of Gloeophyllum abietinum.", the expected output is the linearised list of relations <math id="S2.SS2.SSS2.p1.5.m5.1" class="ltx_Math" alttext="Y" display="inline"><semantics id="S2.SS2.SSS2.p1.5.m5.1a"><mi id="S2.SS2.SSS2.p1.5.m5.1.1" xref="S2.SS2.SSS2.p1.5.m5.1.1.cmml">Y</mi><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.5.m5.1b"><ci id="S2.SS2.SSS2.p1.5.m5.1.1.cmml" xref="S2.SS2.SSS2.p1.5.m5.1.1">ùëå</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.5.m5.1c">Y</annotation></semantics></math>: "Gloeophyllum abietinum produces gloeophyllin A; Gloeophyllum abietinum produces gloeophyllin B; Gloeophyllum abietinum produces gloeophyllin C", following previous works recommendations <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib4" title="" class="ltx_ref">4</a>, <a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite>. During fine-tuning, the objective function is then defined as:</p>
<table id="S2.E3" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S2.E3.m1.5" class="ltx_Math" alttext="L(\theta)=\sum\limits_{t=1}^{|Y|}{\log p(y_{t}|X,y_{&lt;t},\theta)}" display="block"><semantics id="S2.E3.m1.5a"><mrow id="S2.E3.m1.5.5" xref="S2.E3.m1.5.5.cmml"><mrow id="S2.E3.m1.5.5.3" xref="S2.E3.m1.5.5.3.cmml"><mi id="S2.E3.m1.5.5.3.2" xref="S2.E3.m1.5.5.3.2.cmml">L</mi><mo lspace="0em" rspace="0em" id="S2.E3.m1.5.5.3.1" xref="S2.E3.m1.5.5.3.1.cmml">‚Äã</mo><mrow id="S2.E3.m1.5.5.3.3.2" xref="S2.E3.m1.5.5.3.cmml"><mo stretchy="false" id="S2.E3.m1.5.5.3.3.2.1" xref="S2.E3.m1.5.5.3.cmml">(</mo><mi id="S2.E3.m1.2.2" xref="S2.E3.m1.2.2.cmml">Œ∏</mi><mo stretchy="false" id="S2.E3.m1.5.5.3.3.2.2" xref="S2.E3.m1.5.5.3.cmml">)</mo></mrow></mrow><mo rspace="0.111em" id="S2.E3.m1.5.5.2" xref="S2.E3.m1.5.5.2.cmml">=</mo><mrow id="S2.E3.m1.5.5.1" xref="S2.E3.m1.5.5.1.cmml"><munderover id="S2.E3.m1.5.5.1.2" xref="S2.E3.m1.5.5.1.2.cmml"><mo movablelimits="false" id="S2.E3.m1.5.5.1.2.2.2" xref="S2.E3.m1.5.5.1.2.2.2.cmml">‚àë</mo><mrow id="S2.E3.m1.5.5.1.2.2.3" xref="S2.E3.m1.5.5.1.2.2.3.cmml"><mi id="S2.E3.m1.5.5.1.2.2.3.2" xref="S2.E3.m1.5.5.1.2.2.3.2.cmml">t</mi><mo id="S2.E3.m1.5.5.1.2.2.3.1" xref="S2.E3.m1.5.5.1.2.2.3.1.cmml">=</mo><mn id="S2.E3.m1.5.5.1.2.2.3.3" xref="S2.E3.m1.5.5.1.2.2.3.3.cmml">1</mn></mrow><mrow id="S2.E3.m1.1.1.1.3" xref="S2.E3.m1.1.1.1.2.cmml"><mo stretchy="false" id="S2.E3.m1.1.1.1.3.1" xref="S2.E3.m1.1.1.1.2.1.cmml">|</mo><mi id="S2.E3.m1.1.1.1.1" xref="S2.E3.m1.1.1.1.1.cmml">Y</mi><mo stretchy="false" id="S2.E3.m1.1.1.1.3.2" xref="S2.E3.m1.1.1.1.2.1.cmml">|</mo></mrow></munderover><mrow id="S2.E3.m1.5.5.1.1" xref="S2.E3.m1.5.5.1.1.cmml"><mrow id="S2.E3.m1.5.5.1.1.3" xref="S2.E3.m1.5.5.1.1.3.cmml"><mi id="S2.E3.m1.5.5.1.1.3.1" xref="S2.E3.m1.5.5.1.1.3.1.cmml">log</mi><mo lspace="0.167em" id="S2.E3.m1.5.5.1.1.3a" xref="S2.E3.m1.5.5.1.1.3.cmml">‚Å°</mo><mi id="S2.E3.m1.5.5.1.1.3.2" xref="S2.E3.m1.5.5.1.1.3.2.cmml">p</mi></mrow><mo lspace="0em" rspace="0em" id="S2.E3.m1.5.5.1.1.2" xref="S2.E3.m1.5.5.1.1.2.cmml">‚Äã</mo><mrow id="S2.E3.m1.5.5.1.1.1.1" xref="S2.E3.m1.5.5.1.1.1.1.1.cmml"><mo stretchy="false" id="S2.E3.m1.5.5.1.1.1.1.2" xref="S2.E3.m1.5.5.1.1.1.1.1.cmml">(</mo><mrow id="S2.E3.m1.5.5.1.1.1.1.1" xref="S2.E3.m1.5.5.1.1.1.1.1.cmml"><msub id="S2.E3.m1.5.5.1.1.1.1.1.3" xref="S2.E3.m1.5.5.1.1.1.1.1.3.cmml"><mi id="S2.E3.m1.5.5.1.1.1.1.1.3.2" xref="S2.E3.m1.5.5.1.1.1.1.1.3.2.cmml">y</mi><mi id="S2.E3.m1.5.5.1.1.1.1.1.3.3" xref="S2.E3.m1.5.5.1.1.1.1.1.3.3.cmml">t</mi></msub><mo fence="false" id="S2.E3.m1.5.5.1.1.1.1.1.2" xref="S2.E3.m1.5.5.1.1.1.1.1.2.cmml">|</mo><mrow id="S2.E3.m1.5.5.1.1.1.1.1.1.1" xref="S2.E3.m1.5.5.1.1.1.1.1.1.2.cmml"><mi id="S2.E3.m1.3.3" xref="S2.E3.m1.3.3.cmml">X</mi><mo id="S2.E3.m1.5.5.1.1.1.1.1.1.1.2" xref="S2.E3.m1.5.5.1.1.1.1.1.1.2.cmml">,</mo><msub id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.cmml"><mi id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.2" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.2.cmml">y</mi><mrow id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.cmml"><mi id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.2" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.2.cmml"></mi><mo id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.1" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.1.cmml">&lt;</mo><mi id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.3" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.3.cmml">t</mi></mrow></msub><mo id="S2.E3.m1.5.5.1.1.1.1.1.1.1.3" xref="S2.E3.m1.5.5.1.1.1.1.1.1.2.cmml">,</mo><mi id="S2.E3.m1.4.4" xref="S2.E3.m1.4.4.cmml">Œ∏</mi></mrow></mrow><mo stretchy="false" id="S2.E3.m1.5.5.1.1.1.1.3" xref="S2.E3.m1.5.5.1.1.1.1.1.cmml">)</mo></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.E3.m1.5b"><apply id="S2.E3.m1.5.5.cmml" xref="S2.E3.m1.5.5"><eq id="S2.E3.m1.5.5.2.cmml" xref="S2.E3.m1.5.5.2"></eq><apply id="S2.E3.m1.5.5.3.cmml" xref="S2.E3.m1.5.5.3"><times id="S2.E3.m1.5.5.3.1.cmml" xref="S2.E3.m1.5.5.3.1"></times><ci id="S2.E3.m1.5.5.3.2.cmml" xref="S2.E3.m1.5.5.3.2">ùêø</ci><ci id="S2.E3.m1.2.2.cmml" xref="S2.E3.m1.2.2">ùúÉ</ci></apply><apply id="S2.E3.m1.5.5.1.cmml" xref="S2.E3.m1.5.5.1"><apply id="S2.E3.m1.5.5.1.2.cmml" xref="S2.E3.m1.5.5.1.2"><csymbol cd="ambiguous" id="S2.E3.m1.5.5.1.2.1.cmml" xref="S2.E3.m1.5.5.1.2">superscript</csymbol><apply id="S2.E3.m1.5.5.1.2.2.cmml" xref="S2.E3.m1.5.5.1.2"><csymbol cd="ambiguous" id="S2.E3.m1.5.5.1.2.2.1.cmml" xref="S2.E3.m1.5.5.1.2">subscript</csymbol><sum id="S2.E3.m1.5.5.1.2.2.2.cmml" xref="S2.E3.m1.5.5.1.2.2.2"></sum><apply id="S2.E3.m1.5.5.1.2.2.3.cmml" xref="S2.E3.m1.5.5.1.2.2.3"><eq id="S2.E3.m1.5.5.1.2.2.3.1.cmml" xref="S2.E3.m1.5.5.1.2.2.3.1"></eq><ci id="S2.E3.m1.5.5.1.2.2.3.2.cmml" xref="S2.E3.m1.5.5.1.2.2.3.2">ùë°</ci><cn type="integer" id="S2.E3.m1.5.5.1.2.2.3.3.cmml" xref="S2.E3.m1.5.5.1.2.2.3.3">1</cn></apply></apply><apply id="S2.E3.m1.1.1.1.2.cmml" xref="S2.E3.m1.1.1.1.3"><abs id="S2.E3.m1.1.1.1.2.1.cmml" xref="S2.E3.m1.1.1.1.3.1"></abs><ci id="S2.E3.m1.1.1.1.1.cmml" xref="S2.E3.m1.1.1.1.1">ùëå</ci></apply></apply><apply id="S2.E3.m1.5.5.1.1.cmml" xref="S2.E3.m1.5.5.1.1"><times id="S2.E3.m1.5.5.1.1.2.cmml" xref="S2.E3.m1.5.5.1.1.2"></times><apply id="S2.E3.m1.5.5.1.1.3.cmml" xref="S2.E3.m1.5.5.1.1.3"><log id="S2.E3.m1.5.5.1.1.3.1.cmml" xref="S2.E3.m1.5.5.1.1.3.1"></log><ci id="S2.E3.m1.5.5.1.1.3.2.cmml" xref="S2.E3.m1.5.5.1.1.3.2">ùëù</ci></apply><apply id="S2.E3.m1.5.5.1.1.1.1.1.cmml" xref="S2.E3.m1.5.5.1.1.1.1"><csymbol cd="latexml" id="S2.E3.m1.5.5.1.1.1.1.1.2.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.2">conditional</csymbol><apply id="S2.E3.m1.5.5.1.1.1.1.1.3.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.3"><csymbol cd="ambiguous" id="S2.E3.m1.5.5.1.1.1.1.1.3.1.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.3">subscript</csymbol><ci id="S2.E3.m1.5.5.1.1.1.1.1.3.2.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.3.2">ùë¶</ci><ci id="S2.E3.m1.5.5.1.1.1.1.1.3.3.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.3.3">ùë°</ci></apply><list id="S2.E3.m1.5.5.1.1.1.1.1.1.2.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1"><ci id="S2.E3.m1.3.3.cmml" xref="S2.E3.m1.3.3">ùëã</ci><apply id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.1.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.2.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.2">ùë¶</ci><apply id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3"><lt id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.1.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.1"></lt><csymbol cd="latexml" id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.2.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.2">absent</csymbol><ci id="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.3.cmml" xref="S2.E3.m1.5.5.1.1.1.1.1.1.1.1.3.3">ùë°</ci></apply></apply><ci id="S2.E3.m1.4.4.cmml" xref="S2.E3.m1.4.4">ùúÉ</ci></list></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.E3.m1.5c">L(\theta)=\sum\limits_{t=1}^{|Y|}{\log p(y_{t}|X,y_{&lt;t},\theta)}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(3)</span></td>
</tr></tbody>
</table>
<p id="S2.SS2.SSS2.p1.11" class="ltx_p">For efficient fine-tuning with minimal memory and parameter requirements, we adopted the QLoRA approach <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib73" title="" class="ltx_ref">73</a>]</cite>. The method extends the Low-rank Adaptation (LoRA) technique <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite>, which involves freezing the original model weights and training only a small set of parameters, known as adapters. Given the original weight of a layer <math id="S2.SS2.SSS2.p1.6.m1.1" class="ltx_Math" alttext="W" display="inline"><semantics id="S2.SS2.SSS2.p1.6.m1.1a"><mi id="S2.SS2.SSS2.p1.6.m1.1.1" xref="S2.SS2.SSS2.p1.6.m1.1.1.cmml">W</mi><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.6.m1.1b"><ci id="S2.SS2.SSS2.p1.6.m1.1.1.cmml" xref="S2.SS2.SSS2.p1.6.m1.1.1">ùëä</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.6.m1.1c">W</annotation></semantics></math>, with <math id="S2.SS2.SSS2.p1.7.m2.1" class="ltx_Math" alttext="h=Wx" display="inline"><semantics id="S2.SS2.SSS2.p1.7.m2.1a"><mrow id="S2.SS2.SSS2.p1.7.m2.1.1" xref="S2.SS2.SSS2.p1.7.m2.1.1.cmml"><mi id="S2.SS2.SSS2.p1.7.m2.1.1.2" xref="S2.SS2.SSS2.p1.7.m2.1.1.2.cmml">h</mi><mo id="S2.SS2.SSS2.p1.7.m2.1.1.1" xref="S2.SS2.SSS2.p1.7.m2.1.1.1.cmml">=</mo><mrow id="S2.SS2.SSS2.p1.7.m2.1.1.3" xref="S2.SS2.SSS2.p1.7.m2.1.1.3.cmml"><mi id="S2.SS2.SSS2.p1.7.m2.1.1.3.2" xref="S2.SS2.SSS2.p1.7.m2.1.1.3.2.cmml">W</mi><mo lspace="0em" rspace="0em" id="S2.SS2.SSS2.p1.7.m2.1.1.3.1" xref="S2.SS2.SSS2.p1.7.m2.1.1.3.1.cmml">‚Äã</mo><mi id="S2.SS2.SSS2.p1.7.m2.1.1.3.3" xref="S2.SS2.SSS2.p1.7.m2.1.1.3.3.cmml">x</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.7.m2.1b"><apply id="S2.SS2.SSS2.p1.7.m2.1.1.cmml" xref="S2.SS2.SSS2.p1.7.m2.1.1"><eq id="S2.SS2.SSS2.p1.7.m2.1.1.1.cmml" xref="S2.SS2.SSS2.p1.7.m2.1.1.1"></eq><ci id="S2.SS2.SSS2.p1.7.m2.1.1.2.cmml" xref="S2.SS2.SSS2.p1.7.m2.1.1.2">‚Ñé</ci><apply id="S2.SS2.SSS2.p1.7.m2.1.1.3.cmml" xref="S2.SS2.SSS2.p1.7.m2.1.1.3"><times id="S2.SS2.SSS2.p1.7.m2.1.1.3.1.cmml" xref="S2.SS2.SSS2.p1.7.m2.1.1.3.1"></times><ci id="S2.SS2.SSS2.p1.7.m2.1.1.3.2.cmml" xref="S2.SS2.SSS2.p1.7.m2.1.1.3.2">ùëä</ci><ci id="S2.SS2.SSS2.p1.7.m2.1.1.3.3.cmml" xref="S2.SS2.SSS2.p1.7.m2.1.1.3.3">ùë•</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.7.m2.1c">h=Wx</annotation></semantics></math>, the adapters operate as an update to the initial weights <math id="S2.SS2.SSS2.p1.8.m3.1" class="ltx_Math" alttext="W+\Delta W" display="inline"><semantics id="S2.SS2.SSS2.p1.8.m3.1a"><mrow id="S2.SS2.SSS2.p1.8.m3.1.1" xref="S2.SS2.SSS2.p1.8.m3.1.1.cmml"><mi id="S2.SS2.SSS2.p1.8.m3.1.1.2" xref="S2.SS2.SSS2.p1.8.m3.1.1.2.cmml">W</mi><mo id="S2.SS2.SSS2.p1.8.m3.1.1.1" xref="S2.SS2.SSS2.p1.8.m3.1.1.1.cmml">+</mo><mrow id="S2.SS2.SSS2.p1.8.m3.1.1.3" xref="S2.SS2.SSS2.p1.8.m3.1.1.3.cmml"><mi mathvariant="normal" id="S2.SS2.SSS2.p1.8.m3.1.1.3.2" xref="S2.SS2.SSS2.p1.8.m3.1.1.3.2.cmml">Œî</mi><mo lspace="0em" rspace="0em" id="S2.SS2.SSS2.p1.8.m3.1.1.3.1" xref="S2.SS2.SSS2.p1.8.m3.1.1.3.1.cmml">‚Äã</mo><mi id="S2.SS2.SSS2.p1.8.m3.1.1.3.3" xref="S2.SS2.SSS2.p1.8.m3.1.1.3.3.cmml">W</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.8.m3.1b"><apply id="S2.SS2.SSS2.p1.8.m3.1.1.cmml" xref="S2.SS2.SSS2.p1.8.m3.1.1"><plus id="S2.SS2.SSS2.p1.8.m3.1.1.1.cmml" xref="S2.SS2.SSS2.p1.8.m3.1.1.1"></plus><ci id="S2.SS2.SSS2.p1.8.m3.1.1.2.cmml" xref="S2.SS2.SSS2.p1.8.m3.1.1.2">ùëä</ci><apply id="S2.SS2.SSS2.p1.8.m3.1.1.3.cmml" xref="S2.SS2.SSS2.p1.8.m3.1.1.3"><times id="S2.SS2.SSS2.p1.8.m3.1.1.3.1.cmml" xref="S2.SS2.SSS2.p1.8.m3.1.1.3.1"></times><ci id="S2.SS2.SSS2.p1.8.m3.1.1.3.2.cmml" xref="S2.SS2.SSS2.p1.8.m3.1.1.3.2">Œî</ci><ci id="S2.SS2.SSS2.p1.8.m3.1.1.3.3.cmml" xref="S2.SS2.SSS2.p1.8.m3.1.1.3.3">ùëä</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.8.m3.1c">W+\Delta W</annotation></semantics></math> and give <math id="S2.SS2.SSS2.p1.9.m4.1" class="ltx_Math" alttext="h=Wx+\Delta Wx" display="inline"><semantics id="S2.SS2.SSS2.p1.9.m4.1a"><mrow id="S2.SS2.SSS2.p1.9.m4.1.1" xref="S2.SS2.SSS2.p1.9.m4.1.1.cmml"><mi id="S2.SS2.SSS2.p1.9.m4.1.1.2" xref="S2.SS2.SSS2.p1.9.m4.1.1.2.cmml">h</mi><mo id="S2.SS2.SSS2.p1.9.m4.1.1.1" xref="S2.SS2.SSS2.p1.9.m4.1.1.1.cmml">=</mo><mrow id="S2.SS2.SSS2.p1.9.m4.1.1.3" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.cmml"><mrow id="S2.SS2.SSS2.p1.9.m4.1.1.3.2" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.2.cmml"><mi id="S2.SS2.SSS2.p1.9.m4.1.1.3.2.2" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.2.2.cmml">W</mi><mo lspace="0em" rspace="0em" id="S2.SS2.SSS2.p1.9.m4.1.1.3.2.1" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.2.1.cmml">‚Äã</mo><mi id="S2.SS2.SSS2.p1.9.m4.1.1.3.2.3" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.2.3.cmml">x</mi></mrow><mo id="S2.SS2.SSS2.p1.9.m4.1.1.3.1" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.1.cmml">+</mo><mrow id="S2.SS2.SSS2.p1.9.m4.1.1.3.3" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.cmml"><mi mathvariant="normal" id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.2" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.2.cmml">Œî</mi><mo lspace="0em" rspace="0em" id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.1" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.1.cmml">‚Äã</mo><mi id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.3" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.3.cmml">W</mi><mo lspace="0em" rspace="0em" id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.1a" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.1.cmml">‚Äã</mo><mi id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.4" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.4.cmml">x</mi></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.SS2.SSS2.p1.9.m4.1b"><apply id="S2.SS2.SSS2.p1.9.m4.1.1.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1"><eq id="S2.SS2.SSS2.p1.9.m4.1.1.1.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.1"></eq><ci id="S2.SS2.SSS2.p1.9.m4.1.1.2.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.2">‚Ñé</ci><apply id="S2.SS2.SSS2.p1.9.m4.1.1.3.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3"><plus id="S2.SS2.SSS2.p1.9.m4.1.1.3.1.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.1"></plus><apply id="S2.SS2.SSS2.p1.9.m4.1.1.3.2.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.2"><times id="S2.SS2.SSS2.p1.9.m4.1.1.3.2.1.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.2.1"></times><ci id="S2.SS2.SSS2.p1.9.m4.1.1.3.2.2.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.2.2">ùëä</ci><ci id="S2.SS2.SSS2.p1.9.m4.1.1.3.2.3.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.2.3">ùë•</ci></apply><apply id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3"><times id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.1.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.1"></times><ci id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.2.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.2">Œî</ci><ci id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.3.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.3">ùëä</ci><ci id="S2.SS2.SSS2.p1.9.m4.1.1.3.3.4.cmml" xref="S2.SS2.SSS2.p1.9.m4.1.1.3.3.4">ùë•</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS2.SSS2.p1.9.m4.1c">h=Wx+\Delta Wx</annotation></semantics></math>. With QLoRA, the LoRA strategy is integrated with quantization to minimize the memory footprint. It uses a low-precision storage data type (<span id="S2.SS2.SSS2.p1.11.1" class="ltx_text ltx_markedasmath">NF4</span>) and a computation data type (<span id="S2.SS2.SSS2.p1.11.2" class="ltx_text ltx_markedasmath">BFloat16</span>) for the forward and backward passes. Two models were evaluated using this strategy: BioGPT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib4" title="" class="ltx_ref">4</a>]</cite> and GPT-2 <span id="S2.SS2.SSS2.p1.11.3" class="ltx_text ltx_font_italic">Medium</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib75" title="" class="ltx_ref">75</a>]</cite>. Both models share the same architecture, but BioGPT and its tokenizer were trained on PubMed items and achieved SOTA performance on three end-to-end RE tasks. As <cite class="ltx_cite ltx_citemacro_citet">Luo et¬†al. [<a href="#bib.bib4" title="" class="ltx_ref">4</a>]</cite>, we also add Seq2rel <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite> as a second baseline using a sequence-to-sequence approach. See implementation details and hyperparameter tuning in supplementary <a href="#S1.SS2" title="S1.2 Fine-tuning details ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S1.2</span></a>.</p>
</div>
</section>
</section>
<section id="S2.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.3 </span>Synthetic abstracts generation</h3>

<div id="S2.SS3.p1" class="ltx_para">
<p id="S2.SS3.p1.12" class="ltx_p">A general overview of the synthetic abstract generation is provided in Figure <a href="#S2.F3" title="Figure 3 ‚Ä£ 2.3 Synthetic abstracts generation ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>. The goal is to leverage the generative capabilities of instructions and conversations tuned models to correct the discrepancies between the output labels and the input text. Consistency is maintained by grounding the generated abstracts on key elements from an original seed abstract: title, keyphrases extracted from the abstract (and title), and verbalized main findings. The main findings represent the set of relations <math id="S2.SS3.p1.1.m1.4" class="ltx_Math" alttext="\{r_{1},r_{2},...,r_{n}\}" display="inline"><semantics id="S2.SS3.p1.1.m1.4a"><mrow id="S2.SS3.p1.1.m1.4.4.3" xref="S2.SS3.p1.1.m1.4.4.4.cmml"><mo stretchy="false" id="S2.SS3.p1.1.m1.4.4.3.4" xref="S2.SS3.p1.1.m1.4.4.4.cmml">{</mo><msub id="S2.SS3.p1.1.m1.2.2.1.1" xref="S2.SS3.p1.1.m1.2.2.1.1.cmml"><mi id="S2.SS3.p1.1.m1.2.2.1.1.2" xref="S2.SS3.p1.1.m1.2.2.1.1.2.cmml">r</mi><mn id="S2.SS3.p1.1.m1.2.2.1.1.3" xref="S2.SS3.p1.1.m1.2.2.1.1.3.cmml">1</mn></msub><mo id="S2.SS3.p1.1.m1.4.4.3.5" xref="S2.SS3.p1.1.m1.4.4.4.cmml">,</mo><msub id="S2.SS3.p1.1.m1.3.3.2.2" xref="S2.SS3.p1.1.m1.3.3.2.2.cmml"><mi id="S2.SS3.p1.1.m1.3.3.2.2.2" xref="S2.SS3.p1.1.m1.3.3.2.2.2.cmml">r</mi><mn id="S2.SS3.p1.1.m1.3.3.2.2.3" xref="S2.SS3.p1.1.m1.3.3.2.2.3.cmml">2</mn></msub><mo id="S2.SS3.p1.1.m1.4.4.3.6" xref="S2.SS3.p1.1.m1.4.4.4.cmml">,</mo><mi mathvariant="normal" id="S2.SS3.p1.1.m1.1.1" xref="S2.SS3.p1.1.m1.1.1.cmml">‚Ä¶</mi><mo id="S2.SS3.p1.1.m1.4.4.3.7" xref="S2.SS3.p1.1.m1.4.4.4.cmml">,</mo><msub id="S2.SS3.p1.1.m1.4.4.3.3" xref="S2.SS3.p1.1.m1.4.4.3.3.cmml"><mi id="S2.SS3.p1.1.m1.4.4.3.3.2" xref="S2.SS3.p1.1.m1.4.4.3.3.2.cmml">r</mi><mi id="S2.SS3.p1.1.m1.4.4.3.3.3" xref="S2.SS3.p1.1.m1.4.4.3.3.3.cmml">n</mi></msub><mo stretchy="false" id="S2.SS3.p1.1.m1.4.4.3.8" xref="S2.SS3.p1.1.m1.4.4.4.cmml">}</mo></mrow><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.1.m1.4b"><set id="S2.SS3.p1.1.m1.4.4.4.cmml" xref="S2.SS3.p1.1.m1.4.4.3"><apply id="S2.SS3.p1.1.m1.2.2.1.1.cmml" xref="S2.SS3.p1.1.m1.2.2.1.1"><csymbol cd="ambiguous" id="S2.SS3.p1.1.m1.2.2.1.1.1.cmml" xref="S2.SS3.p1.1.m1.2.2.1.1">subscript</csymbol><ci id="S2.SS3.p1.1.m1.2.2.1.1.2.cmml" xref="S2.SS3.p1.1.m1.2.2.1.1.2">ùëü</ci><cn type="integer" id="S2.SS3.p1.1.m1.2.2.1.1.3.cmml" xref="S2.SS3.p1.1.m1.2.2.1.1.3">1</cn></apply><apply id="S2.SS3.p1.1.m1.3.3.2.2.cmml" xref="S2.SS3.p1.1.m1.3.3.2.2"><csymbol cd="ambiguous" id="S2.SS3.p1.1.m1.3.3.2.2.1.cmml" xref="S2.SS3.p1.1.m1.3.3.2.2">subscript</csymbol><ci id="S2.SS3.p1.1.m1.3.3.2.2.2.cmml" xref="S2.SS3.p1.1.m1.3.3.2.2.2">ùëü</ci><cn type="integer" id="S2.SS3.p1.1.m1.3.3.2.2.3.cmml" xref="S2.SS3.p1.1.m1.3.3.2.2.3">2</cn></apply><ci id="S2.SS3.p1.1.m1.1.1.cmml" xref="S2.SS3.p1.1.m1.1.1">‚Ä¶</ci><apply id="S2.SS3.p1.1.m1.4.4.3.3.cmml" xref="S2.SS3.p1.1.m1.4.4.3.3"><csymbol cd="ambiguous" id="S2.SS3.p1.1.m1.4.4.3.3.1.cmml" xref="S2.SS3.p1.1.m1.4.4.3.3">subscript</csymbol><ci id="S2.SS3.p1.1.m1.4.4.3.3.2.cmml" xref="S2.SS3.p1.1.m1.4.4.3.3.2">ùëü</ci><ci id="S2.SS3.p1.1.m1.4.4.3.3.3.cmml" xref="S2.SS3.p1.1.m1.4.4.3.3.3">ùëõ</ci></apply></set></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.1.m1.4c">\{r_{1},r_{2},...,r_{n}\}</annotation></semantics></math> between organisms and NPs reported in the seed article. Both the keyphrase extraction and the subsequent generation step can be framed as instructions-guided tasks: "<span id="S2.SS3.p1.12.3" class="ltx_text ltx_font_italic">Extract a list of keywords ‚Ä¶</span>", "<span id="S2.SS3.p1.12.4" class="ltx_text ltx_font_italic">Create a scientific abstract ‚Ä¶</span>". As the extracted keywords and keyphrases will provide an essential context to constrain the generation of the synthetic abstracts, it is also arguably advantageous that the both tasks are carried by the same model.
<br class="ltx_break">The extraction of keywords (illustrated in Box <span id="S2.SS3.p1.12.5" class="ltx_text ltx_font_bold">A</span> of Figure <a href="#S2.F3" title="Figure 3 ‚Ä£ 2.3 Synthetic abstracts generation ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>) involves prompting the model with appropriate instructions and filtering any outputs corresponding to organism or chemical names using an exclusion list. The exclusion list is built from the available PubTator‚Äôs annotations <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib76" title="" class="ltx_ref">76</a>]</cite> of named entities (if any) and the list of synonyms available from the LOTUS annotations. This filter is essential to ensure the consistency of the generated abstract by avoiding any contextual information that could interfere with the provided main findings. Together with the original title, the keyphrases provide a biological context in which to express the NP relationships.
<br class="ltx_break">By explicitly formalizing the expected patterns in upstream instructions, the expression of NP relationships during the generation step can be more efficiently controlled. The findings-verbaliser module operates as a sampler to emulate and combine various patterns of expression that can be observed in the literature. It incorporates 5 possible transformations: (1) members of a same chemical class<span id="footnote3" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">3</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">3</sup><span class="ltx_tag ltx_tag_note">3</span>according to NP-classifier <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite> annotations in LOTUS</span></span></span> can be replaced by the simple mention of the class (e.g: a list of chemicals <math id="S2.SS3.p1.2.m2.1" class="ltx_Math" alttext="c_{1:5}" display="inline"><semantics id="S2.SS3.p1.2.m2.1a"><msub id="S2.SS3.p1.2.m2.1.1" xref="S2.SS3.p1.2.m2.1.1.cmml"><mi id="S2.SS3.p1.2.m2.1.1.2" xref="S2.SS3.p1.2.m2.1.1.2.cmml">c</mi><mrow id="S2.SS3.p1.2.m2.1.1.3" xref="S2.SS3.p1.2.m2.1.1.3.cmml"><mn id="S2.SS3.p1.2.m2.1.1.3.2" xref="S2.SS3.p1.2.m2.1.1.3.2.cmml">1</mn><mo lspace="0.278em" rspace="0.278em" id="S2.SS3.p1.2.m2.1.1.3.1" xref="S2.SS3.p1.2.m2.1.1.3.1.cmml">:</mo><mn id="S2.SS3.p1.2.m2.1.1.3.3" xref="S2.SS3.p1.2.m2.1.1.3.3.cmml">5</mn></mrow></msub><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.2.m2.1b"><apply id="S2.SS3.p1.2.m2.1.1.cmml" xref="S2.SS3.p1.2.m2.1.1"><csymbol cd="ambiguous" id="S2.SS3.p1.2.m2.1.1.1.cmml" xref="S2.SS3.p1.2.m2.1.1">subscript</csymbol><ci id="S2.SS3.p1.2.m2.1.1.2.cmml" xref="S2.SS3.p1.2.m2.1.1.2">ùëê</ci><apply id="S2.SS3.p1.2.m2.1.1.3.cmml" xref="S2.SS3.p1.2.m2.1.1.3"><ci id="S2.SS3.p1.2.m2.1.1.3.1.cmml" xref="S2.SS3.p1.2.m2.1.1.3.1">:</ci><cn type="integer" id="S2.SS3.p1.2.m2.1.1.3.2.cmml" xref="S2.SS3.p1.2.m2.1.1.3.2">1</cn><cn type="integer" id="S2.SS3.p1.2.m2.1.1.3.3.cmml" xref="S2.SS3.p1.2.m2.1.1.3.3">5</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.2.m2.1c">c_{1:5}</annotation></semantics></math> is replaced by the more concise mention "<span id="S2.SS3.p1.12.6" class="ltx_text ltx_font_italic">Five Meroterpenoids</span>"); (2) Lists of chemical derivates can be contracted (e.g "<span id="S2.SS3.p1.12.7" class="ltx_text ltx_font_italic">Cystodione A-D</span>"); (3) The order of relationships is systematically shuffled; (4) Chemicals can be numbered (e.g "<span id="S2.SS3.p1.12.8" class="ltx_text ltx_font_italic">Cystodione A-D</span> (1-4)"); (5) Directions of the relationship can change from "<math id="S2.SS3.p1.3.m3.1" class="ltx_Math" alttext="O" display="inline"><semantics id="S2.SS3.p1.3.m3.1a"><mi id="S2.SS3.p1.3.m3.1.1" xref="S2.SS3.p1.3.m3.1.1.cmml">O</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.3.m3.1b"><ci id="S2.SS3.p1.3.m3.1.1.cmml" xref="S2.SS3.p1.3.m3.1.1">ùëÇ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.3.m3.1c">O</annotation></semantics></math><span id="S2.SS3.p1.4.1" class="ltx_text ltx_font_italic"> produces <math id="S2.SS3.p1.4.1.m1.1" class="ltx_Math" alttext="C" display="inline"><semantics id="S2.SS3.p1.4.1.m1.1a"><mi id="S2.SS3.p1.4.1.m1.1.1" xref="S2.SS3.p1.4.1.m1.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.4.1.m1.1b"><ci id="S2.SS3.p1.4.1.m1.1.1.cmml" xref="S2.SS3.p1.4.1.m1.1.1">ùê∂</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.4.1.m1.1c">C</annotation></semantics></math></span>" to "<math id="S2.SS3.p1.5.m4.1" class="ltx_Math" alttext="C" display="inline"><semantics id="S2.SS3.p1.5.m4.1a"><mi id="S2.SS3.p1.5.m4.1.1" xref="S2.SS3.p1.5.m4.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.5.m4.1b"><ci id="S2.SS3.p1.5.m4.1.1.cmml" xref="S2.SS3.p1.5.m4.1.1">ùê∂</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.5.m4.1c">C</annotation></semantics></math><span id="S2.SS3.p1.6.2" class="ltx_text ltx_font_italic"> was isolated from <math id="S2.SS3.p1.6.2.m1.1" class="ltx_Math" alttext="O" display="inline"><semantics id="S2.SS3.p1.6.2.m1.1a"><mi id="S2.SS3.p1.6.2.m1.1.1" xref="S2.SS3.p1.6.2.m1.1.1.cmml">O</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.6.2.m1.1b"><ci id="S2.SS3.p1.6.2.m1.1.1.cmml" xref="S2.SS3.p1.6.2.m1.1.1">ùëÇ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.6.2.m1.1c">O</annotation></semantics></math></span>". See the box <span id="S2.SS3.p1.12.9" class="ltx_text ltx_font_bold">B</span> of Figure <a href="#S2.F3" title="Figure 3 ‚Ä£ 2.3 Synthetic abstracts generation ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a> and more details in supplementary <a href="#S1.SS3" title="S1.3 Main-findings verbalisation patterns ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S1.3</span></a>. These different transformations are reminiscent of the strategies commonly used in data augmentation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib31" title="" class="ltx_ref">31</a>]</cite>.
<br class="ltx_break">For each input seed abstract, <math id="S2.SS3.p1.7.m5.1" class="ltx_Math" alttext="m" display="inline"><semantics id="S2.SS3.p1.7.m5.1a"><mi id="S2.SS3.p1.7.m5.1.1" xref="S2.SS3.p1.7.m5.1.1.cmml">m</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.7.m5.1b"><ci id="S2.SS3.p1.7.m5.1.1.cmml" xref="S2.SS3.p1.7.m5.1.1">ùëö</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.7.m5.1c">m</annotation></semantics></math> instructions are sampled and assembled following this procedure and forwarded to the model for generation (Box <span id="S2.SS3.p1.12.10" class="ltx_text ltx_font_bold">C</span>). Finally, the selector module selects a top <math id="S2.SS3.p1.8.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S2.SS3.p1.8.m6.1a"><mi id="S2.SS3.p1.8.m6.1.1" xref="S2.SS3.p1.8.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.8.m6.1b"><ci id="S2.SS3.p1.8.m6.1.1.cmml" xref="S2.SS3.p1.8.m6.1.1">ùëò</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.8.m6.1c">k</annotation></semantics></math>, from the <math id="S2.SS3.p1.9.m7.1" class="ltx_Math" alttext="m" display="inline"><semantics id="S2.SS3.p1.9.m7.1a"><mi id="S2.SS3.p1.9.m7.1.1" xref="S2.SS3.p1.9.m7.1.1.cmml">m</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.9.m7.1b"><ci id="S2.SS3.p1.9.m7.1.1.cmml" xref="S2.SS3.p1.9.m7.1.1">ùëö</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.9.m7.1c">m</annotation></semantics></math> generated abstracts, ensuring that at least a proportion <math id="S2.SS3.p1.10.m8.1" class="ltx_Math" alttext="q" display="inline"><semantics id="S2.SS3.p1.10.m8.1a"><mi id="S2.SS3.p1.10.m8.1.1" xref="S2.SS3.p1.10.m8.1.1.cmml">q</mi><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.10.m8.1b"><ci id="S2.SS3.p1.10.m8.1.1.cmml" xref="S2.SS3.p1.10.m8.1.1">ùëû</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.10.m8.1c">q</annotation></semantics></math> of the expected relations have the labels of the involved organisms and chemicals explicitly mentioned in the generated abstract (Box <span id="S2.SS3.p1.12.11" class="ltx_text ltx_font_bold">D</span>). Regarding the expected output labels, the replacement operated by transformation (1) also applies: the initial relations <math id="S2.SS3.p1.11.m9.1" class="ltx_Math" alttext="r_{1:5}" display="inline"><semantics id="S2.SS3.p1.11.m9.1a"><msub id="S2.SS3.p1.11.m9.1.1" xref="S2.SS3.p1.11.m9.1.1.cmml"><mi id="S2.SS3.p1.11.m9.1.1.2" xref="S2.SS3.p1.11.m9.1.1.2.cmml">r</mi><mrow id="S2.SS3.p1.11.m9.1.1.3" xref="S2.SS3.p1.11.m9.1.1.3.cmml"><mn id="S2.SS3.p1.11.m9.1.1.3.2" xref="S2.SS3.p1.11.m9.1.1.3.2.cmml">1</mn><mo lspace="0.278em" rspace="0.278em" id="S2.SS3.p1.11.m9.1.1.3.1" xref="S2.SS3.p1.11.m9.1.1.3.1.cmml">:</mo><mn id="S2.SS3.p1.11.m9.1.1.3.3" xref="S2.SS3.p1.11.m9.1.1.3.3.cmml">5</mn></mrow></msub><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.11.m9.1b"><apply id="S2.SS3.p1.11.m9.1.1.cmml" xref="S2.SS3.p1.11.m9.1.1"><csymbol cd="ambiguous" id="S2.SS3.p1.11.m9.1.1.1.cmml" xref="S2.SS3.p1.11.m9.1.1">subscript</csymbol><ci id="S2.SS3.p1.11.m9.1.1.2.cmml" xref="S2.SS3.p1.11.m9.1.1.2">ùëü</ci><apply id="S2.SS3.p1.11.m9.1.1.3.cmml" xref="S2.SS3.p1.11.m9.1.1.3"><ci id="S2.SS3.p1.11.m9.1.1.3.1.cmml" xref="S2.SS3.p1.11.m9.1.1.3.1">:</ci><cn type="integer" id="S2.SS3.p1.11.m9.1.1.3.2.cmml" xref="S2.SS3.p1.11.m9.1.1.3.2">1</cn><cn type="integer" id="S2.SS3.p1.11.m9.1.1.3.3.cmml" xref="S2.SS3.p1.11.m9.1.1.3.3">5</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.11.m9.1c">r_{1:5}</annotation></semantics></math> involving the 5 meroterpenoids are replaced by a single relation <math id="S2.SS3.p1.12.m10.1" class="ltx_Math" alttext="r_{6}" display="inline"><semantics id="S2.SS3.p1.12.m10.1a"><msub id="S2.SS3.p1.12.m10.1.1" xref="S2.SS3.p1.12.m10.1.1.cmml"><mi id="S2.SS3.p1.12.m10.1.1.2" xref="S2.SS3.p1.12.m10.1.1.2.cmml">r</mi><mn id="S2.SS3.p1.12.m10.1.1.3" xref="S2.SS3.p1.12.m10.1.1.3.cmml">6</mn></msub><annotation-xml encoding="MathML-Content" id="S2.SS3.p1.12.m10.1b"><apply id="S2.SS3.p1.12.m10.1.1.cmml" xref="S2.SS3.p1.12.m10.1.1"><csymbol cd="ambiguous" id="S2.SS3.p1.12.m10.1.1.1.cmml" xref="S2.SS3.p1.12.m10.1.1">subscript</csymbol><ci id="S2.SS3.p1.12.m10.1.1.2.cmml" xref="S2.SS3.p1.12.m10.1.1.2">ùëü</ci><cn type="integer" id="S2.SS3.p1.12.m10.1.1.3.cmml" xref="S2.SS3.p1.12.m10.1.1.3">6</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.SS3.p1.12.m10.1c">r_{6}</annotation></semantics></math> involving "Meroterpenoids" as chemical entity. In contrast, transformation (2) does not affect the output labels, requiring the model to expand the list of relations involving each derivative (see Box <span id="S2.SS3.p1.12.12" class="ltx_text ltx_font_bold">D</span> - <span id="S2.SS3.p1.12.13" class="ltx_text ltx_font_italic">Output labels</span>). Also, the loss in eq.<a href="#S2.E3" title="In 2.2.2 Fine-tuning ‚Ä£ 2.2 Different strategies for Relation Extraction ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a> (like in Seq2rel) is permutation sensitive, but the order created by the transformation (3), which also applied to the output labels, is almost systematically respected by the model in the generated abstract, alleviating this issue. Transformations (4) and (5) have no influence on the output labels.</p>
</div>
<figure id="S2.F3" class="ltx_figure"><img src="/html/2311.06364/assets/Figures/Methods/figure-method-abstract-generation.png" id="S2.F3.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="538" height="303" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>Description of the synthetic abstract generation process</figcaption>
</figure>
</section>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Empirical experiments</h2>

<figure id="S3.F4" class="ltx_figure"><img src="/html/2311.06364/assets/Figures/Results/Figure1.png" id="S3.F4.1.g1" class="ltx_graphics ltx_centering ltx_img_square" width="538" height="524" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure 4: </span><span id="S3.F4.19.1" class="ltx_text ltx_font_bold">A:</span> Distribution of the cumulative proportion of reported relations per fraction of chemicals (plain line) and organisms (dashed line), ordered by their contributions. The same relations but reported from different articles are considered as distinct items. From the both curves, it can be estimated that 20% of the most represented chemicals holds more than 68% of the relations, and 20% of the organisms hold more than 72%. <span id="S3.F4.20.2" class="ltx_text ltx_font_bold">B:</span> Repartition of the number of reported relations, organised per kingdom of the subject organism in 4 datasets: the original (<span id="S3.F4.21.3" class="ltx_text ltx_font_italic">full</span>) dataset, the pre-processed dataset, the <span id="S3.F4.22.4" class="ltx_text ltx_font_italic">Diversity</span> sample and <span id="S3.F4.23.5" class="ltx_text ltx_font_italic">Random</span> samples. For <span id="S3.F4.24.6" class="ltx_text ltx_font_italic">Random</span> samples, proportions are averaged over 5 samples. <span id="S3.F4.25.7" class="ltx_text ltx_font_bold">C:</span> Statistics of the number of distinct organisms, relations and chemicals in the <span id="S3.F4.26.8" class="ltx_text ltx_font_italic">Diversity</span> sample compared to <span id="S3.F4.27.9" class="ltx_text ltx_font_italic">Random</span> samples. All samples contain 2000 articles. <span id="S3.F4.28.10" class="ltx_text ltx_font_bold">D:</span> Similarly to <span id="S3.F4.29.11" class="ltx_text ltx_font_bold">A</span>, Distribution of the cumulative proportion of reported relations per fraction of chemicals (top) and organisms (bottom), ordered by their contributions in three different type of samples: <span id="S3.F4.30.12" class="ltx_text ltx_font_italic">Pre-processed</span>, <span id="S3.F4.31.13" class="ltx_text ltx_font_italic">Diversity</span> and <span id="S3.F4.32.14" class="ltx_text ltx_font_italic">Random</span>. <span id="S3.F4.33.15" class="ltx_text ltx_font_bold">E:</span> Distribution of the frequency of mention in distinct articles of chemicals (top) and organisms (bottom). <span id="S3.F4.34.16" class="ltx_text ltx_font_bold">F:</span> Mismatches between standardized labels of organisms and chemicals and their original literal mentions in the abstracts of articles reporting the relationships. <span id="S3.F4.35.17" class="ltx_text ltx_font_italic">Multiple</span> entities correspond to chemical mentions that are not expressed in a continuous string. See details in supplementary <a href="#S2.SS3a" title="S2.3 Mismatches between standardized labels and original abstracts ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2.3</span></a>.</figcaption>
</figure>
<section id="S3.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.1 </span>Imbalanced repartition of reported relations and coverage on biological kingdoms</h3>

<div id="S3.SS1.p1" class="ltx_para ltx_noindent">
<p id="S3.SS1.p1.2" class="ltx_p">As reported in the original release of the LOTUS dataset <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite>, the imbalance in the data distribution manifests at two main levels: the repartition of the number of reported relationships per organism (resp. chemicals) and the coverage of biological kingdoms. These observations were reproduced from the latest available snapshot of the LOTUS dataset (v10-01-2023)<span id="footnote4" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">4</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">4</sup><span class="ltx_tag ltx_tag_note">4</span><a target="_blank" href="https://zenodo.org/record/7534071" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://zenodo.org/record/7534071</a></span></span></span>, containing more than <math id="S3.SS1.p1.1.m1.2" class="ltx_Math" alttext="533,000" display="inline"><semantics id="S3.SS1.p1.1.m1.2a"><mrow id="S3.SS1.p1.1.m1.2.3.2" xref="S3.SS1.p1.1.m1.2.3.1.cmml"><mn id="S3.SS1.p1.1.m1.1.1" xref="S3.SS1.p1.1.m1.1.1.cmml">533</mn><mo id="S3.SS1.p1.1.m1.2.3.2.1" xref="S3.SS1.p1.1.m1.2.3.1.cmml">,</mo><mn id="S3.SS1.p1.1.m1.2.2" xref="S3.SS1.p1.1.m1.2.2.cmml">000</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS1.p1.1.m1.2b"><list id="S3.SS1.p1.1.m1.2.3.1.cmml" xref="S3.SS1.p1.1.m1.2.3.2"><cn type="integer" id="S3.SS1.p1.1.m1.1.1.cmml" xref="S3.SS1.p1.1.m1.1.1">533</cn><cn type="integer" id="S3.SS1.p1.1.m1.2.2.cmml" xref="S3.SS1.p1.1.m1.2.2">000</cn></list></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p1.1.m1.2c">533,000</annotation></semantics></math> distinct relations between organisms and NPs, reported from more than <math id="S3.SS1.p1.2.m2.2" class="ltx_Math" alttext="88,000" display="inline"><semantics id="S3.SS1.p1.2.m2.2a"><mrow id="S3.SS1.p1.2.m2.2.3.2" xref="S3.SS1.p1.2.m2.2.3.1.cmml"><mn id="S3.SS1.p1.2.m2.1.1" xref="S3.SS1.p1.2.m2.1.1.cmml">88</mn><mo id="S3.SS1.p1.2.m2.2.3.2.1" xref="S3.SS1.p1.2.m2.2.3.1.cmml">,</mo><mn id="S3.SS1.p1.2.m2.2.2" xref="S3.SS1.p1.2.m2.2.2.cmml">000</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS1.p1.2.m2.2b"><list id="S3.SS1.p1.2.m2.2.3.1.cmml" xref="S3.SS1.p1.2.m2.2.3.2"><cn type="integer" id="S3.SS1.p1.2.m2.1.1.cmml" xref="S3.SS1.p1.2.m2.1.1">88</cn><cn type="integer" id="S3.SS1.p1.2.m2.2.2.cmml" xref="S3.SS1.p1.2.m2.2.2">000</cn></list></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.p1.2.m2.2c">88,000</annotation></semantics></math> articles. As expected, a small fraction of the organisms (resp. chemicals) attracts a large proportion of the relations: more than 72% of relations involve only 20% of the organisms (Figure <a href="#S3.F4" title="Figure 4 ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.A). Beside these pareto distributions <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib78" title="" class="ltx_ref">78</a>]</cite>, the imbalance in the repartition of the relations across biological kingdoms is also important: 80% are related to <span id="S3.SS1.p1.2.1" class="ltx_text ltx_font_italic">Archaeplatida</span> (Figure <a href="#S3.F4" title="Figure 4 ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.B <span id="S3.SS1.p1.2.2" class="ltx_text ltx_font_italic">top-left</span>). Considering these both biases is essential to extract a valuable sample. This motivated the use of the GME-sampler in a stratified way, to maximise diversity and reduce the pareto effect, while ensuring a more balanced coverage across biological kingdoms.</p>
</div>
</section>
<section id="S3.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.2 </span>Dataset pre-processing</h3>

<div id="S3.SS2.p1" class="ltx_para ltx_noindent">
<p id="S3.SS2.p1.1" class="ltx_p">The original dataset was first preprocessed and filtered prior to sampling to eliminate various sources of perturbations and unusable data in subsequent steps. Specifically, only documents with publicly available abstracts on PubMed were selected, and these were further filtered based on the number of reported relations. Indeed, a manual inspection of a subset of articles revealed that documents reporting large numbers of relations (<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib80" title="" class="ltx_ref">80</a>, <a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite>) often propose genome-scale metabolic reconstructions, large screening analyses, or database releases. Although these documents may report hundreds of relationships, they are typically not expressed in the abstracts, making them useless examples for building a RE model. Only articles reporting less than 20 relations (corresponding to the quantile 93%) were then selected. Compared to organism names, the length of chemical names can exhibit extreme variability and exceed hundreds of characters depending on the nomenclature. To mitigate the issues posed by these lengthy labels, which are inordinate to decode and could consume an excessive portion of the context window, only relations involving chemicals with a label length <math id="S3.SS2.p1.1.m1.1" class="ltx_Math" alttext="l\leq 60" display="inline"><semantics id="S3.SS2.p1.1.m1.1a"><mrow id="S3.SS2.p1.1.m1.1.1" xref="S3.SS2.p1.1.m1.1.1.cmml"><mi id="S3.SS2.p1.1.m1.1.1.2" xref="S3.SS2.p1.1.m1.1.1.2.cmml">l</mi><mo id="S3.SS2.p1.1.m1.1.1.1" xref="S3.SS2.p1.1.m1.1.1.1.cmml">‚â§</mo><mn id="S3.SS2.p1.1.m1.1.1.3" xref="S3.SS2.p1.1.m1.1.1.3.cmml">60</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS2.p1.1.m1.1b"><apply id="S3.SS2.p1.1.m1.1.1.cmml" xref="S3.SS2.p1.1.m1.1.1"><leq id="S3.SS2.p1.1.m1.1.1.1.cmml" xref="S3.SS2.p1.1.m1.1.1.1"></leq><ci id="S3.SS2.p1.1.m1.1.1.2.cmml" xref="S3.SS2.p1.1.m1.1.1.2">ùëô</ci><cn type="integer" id="S3.SS2.p1.1.m1.1.1.3.cmml" xref="S3.SS2.p1.1.m1.1.1.3">60</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS2.p1.1.m1.1c">l\leq 60</annotation></semantics></math> characters were retained. More details in supplementary <a href="#S2.SS1a" title="S2.1 Chemical lengths thresholding ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2.1</span></a>. See the global pre-processing statistics in supplementary table <a href="#S2.T5" title="Table S5 ‚Ä£ S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S5</span></a> and kingdom coverage in Figure <a href="#S3.F4" title="Figure 4 ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.B top-right.</p>
</div>
</section>
<section id="S3.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.3 </span>Building a diversity-augmented dataset</h3>

<figure id="S3.F5" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/entropy-degradation.png" id="S3.F5.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="658" height="366" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure 5: </span><span id="S3.F5.15.1" class="ltx_text ltx_font_bold">Top-panel:</span> Evolution of the Entropy over the distribution of reported organisms (<math id="S3.F5.7.m1.1" class="ltx_Math" alttext="H_{\displaystyle S}(O)" display="inline"><semantics id="S3.F5.7.m1.1b"><mrow id="S3.F5.7.m1.1.2" xref="S3.F5.7.m1.1.2.cmml"><msub id="S3.F5.7.m1.1.2.2" xref="S3.F5.7.m1.1.2.2.cmml"><mi id="S3.F5.7.m1.1.2.2.2" xref="S3.F5.7.m1.1.2.2.2.cmml">H</mi><mi mathsize="142%" id="S3.F5.7.m1.1.2.2.3" xref="S3.F5.7.m1.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S3.F5.7.m1.1.2.1" xref="S3.F5.7.m1.1.2.1.cmml">‚Äã</mo><mrow id="S3.F5.7.m1.1.2.3.2" xref="S3.F5.7.m1.1.2.cmml"><mo stretchy="false" id="S3.F5.7.m1.1.2.3.2.1" xref="S3.F5.7.m1.1.2.cmml">(</mo><mi id="S3.F5.7.m1.1.1" xref="S3.F5.7.m1.1.1.cmml">O</mi><mo stretchy="false" id="S3.F5.7.m1.1.2.3.2.2" xref="S3.F5.7.m1.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.F5.7.m1.1c"><apply id="S3.F5.7.m1.1.2.cmml" xref="S3.F5.7.m1.1.2"><times id="S3.F5.7.m1.1.2.1.cmml" xref="S3.F5.7.m1.1.2.1"></times><apply id="S3.F5.7.m1.1.2.2.cmml" xref="S3.F5.7.m1.1.2.2"><csymbol cd="ambiguous" id="S3.F5.7.m1.1.2.2.1.cmml" xref="S3.F5.7.m1.1.2.2">subscript</csymbol><ci id="S3.F5.7.m1.1.2.2.2.cmml" xref="S3.F5.7.m1.1.2.2.2">ùêª</ci><ci id="S3.F5.7.m1.1.2.2.3.cmml" xref="S3.F5.7.m1.1.2.2.3">ùëÜ</ci></apply><ci id="S3.F5.7.m1.1.1.cmml" xref="S3.F5.7.m1.1.1">ùëÇ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F5.7.m1.1d">H_{\displaystyle S}(O)</annotation></semantics></math>) and chemicals (<math id="S3.F5.8.m2.1" class="ltx_Math" alttext="H_{\displaystyle S}(C)" display="inline"><semantics id="S3.F5.8.m2.1b"><mrow id="S3.F5.8.m2.1.2" xref="S3.F5.8.m2.1.2.cmml"><msub id="S3.F5.8.m2.1.2.2" xref="S3.F5.8.m2.1.2.2.cmml"><mi id="S3.F5.8.m2.1.2.2.2" xref="S3.F5.8.m2.1.2.2.2.cmml">H</mi><mi mathsize="142%" id="S3.F5.8.m2.1.2.2.3" xref="S3.F5.8.m2.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S3.F5.8.m2.1.2.1" xref="S3.F5.8.m2.1.2.1.cmml">‚Äã</mo><mrow id="S3.F5.8.m2.1.2.3.2" xref="S3.F5.8.m2.1.2.cmml"><mo stretchy="false" id="S3.F5.8.m2.1.2.3.2.1" xref="S3.F5.8.m2.1.2.cmml">(</mo><mi id="S3.F5.8.m2.1.1" xref="S3.F5.8.m2.1.1.cmml">C</mi><mo stretchy="false" id="S3.F5.8.m2.1.2.3.2.2" xref="S3.F5.8.m2.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.F5.8.m2.1c"><apply id="S3.F5.8.m2.1.2.cmml" xref="S3.F5.8.m2.1.2"><times id="S3.F5.8.m2.1.2.1.cmml" xref="S3.F5.8.m2.1.2.1"></times><apply id="S3.F5.8.m2.1.2.2.cmml" xref="S3.F5.8.m2.1.2.2"><csymbol cd="ambiguous" id="S3.F5.8.m2.1.2.2.1.cmml" xref="S3.F5.8.m2.1.2.2">subscript</csymbol><ci id="S3.F5.8.m2.1.2.2.2.cmml" xref="S3.F5.8.m2.1.2.2.2">ùêª</ci><ci id="S3.F5.8.m2.1.2.2.3.cmml" xref="S3.F5.8.m2.1.2.2.3">ùëÜ</ci></apply><ci id="S3.F5.8.m2.1.1.cmml" xref="S3.F5.8.m2.1.1">ùê∂</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F5.8.m2.1d">H_{\displaystyle S}(C)</annotation></semantics></math>) by adding iteratively a new article (<math id="S3.F5.9.m3.1" class="ltx_math_unparsed" alttext="d*" display="inline"><semantics id="S3.F5.9.m3.1b"><mrow id="S3.F5.9.m3.1c"><mi id="S3.F5.9.m3.1.1">d</mi><mo lspace="0.222em" id="S3.F5.9.m3.1.2">‚àó</mo></mrow><annotation encoding="application/x-tex" id="S3.F5.9.m3.1d">d*</annotation></semantics></math>) in the built dataset, stratified by biological kingdoms. The step (500) when 80% of the maximal entropy is reached in all the kingdom‚Äôs subsets, for organisms and chemicals, is indicated with the black arrow (more details in supplementary table <a href="#S2.T7" title="Table S7 ‚Ä£ S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S7</span></a>). <span id="S3.F5.16.2" class="ltx_text ltx_font_bold">Bottom-panel: </span> Zoom of the evolution of <math id="S3.F5.10.m4.1" class="ltx_Math" alttext="H_{\displaystyle S}(O)" display="inline"><semantics id="S3.F5.10.m4.1b"><mrow id="S3.F5.10.m4.1.2" xref="S3.F5.10.m4.1.2.cmml"><msub id="S3.F5.10.m4.1.2.2" xref="S3.F5.10.m4.1.2.2.cmml"><mi id="S3.F5.10.m4.1.2.2.2" xref="S3.F5.10.m4.1.2.2.2.cmml">H</mi><mi mathsize="142%" id="S3.F5.10.m4.1.2.2.3" xref="S3.F5.10.m4.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S3.F5.10.m4.1.2.1" xref="S3.F5.10.m4.1.2.1.cmml">‚Äã</mo><mrow id="S3.F5.10.m4.1.2.3.2" xref="S3.F5.10.m4.1.2.cmml"><mo stretchy="false" id="S3.F5.10.m4.1.2.3.2.1" xref="S3.F5.10.m4.1.2.cmml">(</mo><mi id="S3.F5.10.m4.1.1" xref="S3.F5.10.m4.1.1.cmml">O</mi><mo stretchy="false" id="S3.F5.10.m4.1.2.3.2.2" xref="S3.F5.10.m4.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.F5.10.m4.1c"><apply id="S3.F5.10.m4.1.2.cmml" xref="S3.F5.10.m4.1.2"><times id="S3.F5.10.m4.1.2.1.cmml" xref="S3.F5.10.m4.1.2.1"></times><apply id="S3.F5.10.m4.1.2.2.cmml" xref="S3.F5.10.m4.1.2.2"><csymbol cd="ambiguous" id="S3.F5.10.m4.1.2.2.1.cmml" xref="S3.F5.10.m4.1.2.2">subscript</csymbol><ci id="S3.F5.10.m4.1.2.2.2.cmml" xref="S3.F5.10.m4.1.2.2.2">ùêª</ci><ci id="S3.F5.10.m4.1.2.2.3.cmml" xref="S3.F5.10.m4.1.2.2.3">ùëÜ</ci></apply><ci id="S3.F5.10.m4.1.1.cmml" xref="S3.F5.10.m4.1.1">ùëÇ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F5.10.m4.1d">H_{\displaystyle S}(O)</annotation></semantics></math> and <math id="S3.F5.11.m5.1" class="ltx_Math" alttext="H_{\displaystyle S}(C)" display="inline"><semantics id="S3.F5.11.m5.1b"><mrow id="S3.F5.11.m5.1.2" xref="S3.F5.11.m5.1.2.cmml"><msub id="S3.F5.11.m5.1.2.2" xref="S3.F5.11.m5.1.2.2.cmml"><mi id="S3.F5.11.m5.1.2.2.2" xref="S3.F5.11.m5.1.2.2.2.cmml">H</mi><mi mathsize="142%" id="S3.F5.11.m5.1.2.2.3" xref="S3.F5.11.m5.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S3.F5.11.m5.1.2.1" xref="S3.F5.11.m5.1.2.1.cmml">‚Äã</mo><mrow id="S3.F5.11.m5.1.2.3.2" xref="S3.F5.11.m5.1.2.cmml"><mo stretchy="false" id="S3.F5.11.m5.1.2.3.2.1" xref="S3.F5.11.m5.1.2.cmml">(</mo><mi id="S3.F5.11.m5.1.1" xref="S3.F5.11.m5.1.1.cmml">C</mi><mo stretchy="false" id="S3.F5.11.m5.1.2.3.2.2" xref="S3.F5.11.m5.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.F5.11.m5.1c"><apply id="S3.F5.11.m5.1.2.cmml" xref="S3.F5.11.m5.1.2"><times id="S3.F5.11.m5.1.2.1.cmml" xref="S3.F5.11.m5.1.2.1"></times><apply id="S3.F5.11.m5.1.2.2.cmml" xref="S3.F5.11.m5.1.2.2"><csymbol cd="ambiguous" id="S3.F5.11.m5.1.2.2.1.cmml" xref="S3.F5.11.m5.1.2.2">subscript</csymbol><ci id="S3.F5.11.m5.1.2.2.2.cmml" xref="S3.F5.11.m5.1.2.2.2">ùêª</ci><ci id="S3.F5.11.m5.1.2.2.3.cmml" xref="S3.F5.11.m5.1.2.2.3">ùëÜ</ci></apply><ci id="S3.F5.11.m5.1.1.cmml" xref="S3.F5.11.m5.1.1">ùê∂</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F5.11.m5.1d">H_{\displaystyle S}(C)</annotation></semantics></math> in the first 500 added articles. For each curve, the knee-points (<span id="S3.F5.17.3" class="ltx_text ltx_font_italic">bending</span> points) with the corresponding rank and associated entropies are indicated.</figcaption>
</figure>
<section id="S3.SS3.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.1 </span>Diversity-sampling on organisms and chemicals</h4>

<div id="S3.SS3.SSS1.p1" class="ltx_para ltx_noindent">
<p id="S3.SS3.SSS1.p1.6" class="ltx_p">The preprocessed dataset was first stratified according to the taxonomic classification (kingdoms) of the organisms associated with the relations reported in each document. Subsequently, the GME-sampler was applied to each subset (Figure <a href="#S3.F5" title="Figure 5 ‚Ä£ 3.3 Building a diversity-augmented dataset ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>:Top-panel) to monitor the evolution of the diversity metrics (<math id="S3.SS3.SSS1.p1.1.m1.1" class="ltx_Math" alttext="H_{S}(O)" display="inline"><semantics id="S3.SS3.SSS1.p1.1.m1.1a"><mrow id="S3.SS3.SSS1.p1.1.m1.1.2" xref="S3.SS3.SSS1.p1.1.m1.1.2.cmml"><msub id="S3.SS3.SSS1.p1.1.m1.1.2.2" xref="S3.SS3.SSS1.p1.1.m1.1.2.2.cmml"><mi id="S3.SS3.SSS1.p1.1.m1.1.2.2.2" xref="S3.SS3.SSS1.p1.1.m1.1.2.2.2.cmml">H</mi><mi id="S3.SS3.SSS1.p1.1.m1.1.2.2.3" xref="S3.SS3.SSS1.p1.1.m1.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S3.SS3.SSS1.p1.1.m1.1.2.1" xref="S3.SS3.SSS1.p1.1.m1.1.2.1.cmml">‚Äã</mo><mrow id="S3.SS3.SSS1.p1.1.m1.1.2.3.2" xref="S3.SS3.SSS1.p1.1.m1.1.2.cmml"><mo stretchy="false" id="S3.SS3.SSS1.p1.1.m1.1.2.3.2.1" xref="S3.SS3.SSS1.p1.1.m1.1.2.cmml">(</mo><mi id="S3.SS3.SSS1.p1.1.m1.1.1" xref="S3.SS3.SSS1.p1.1.m1.1.1.cmml">O</mi><mo stretchy="false" id="S3.SS3.SSS1.p1.1.m1.1.2.3.2.2" xref="S3.SS3.SSS1.p1.1.m1.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.SS3.SSS1.p1.1.m1.1b"><apply id="S3.SS3.SSS1.p1.1.m1.1.2.cmml" xref="S3.SS3.SSS1.p1.1.m1.1.2"><times id="S3.SS3.SSS1.p1.1.m1.1.2.1.cmml" xref="S3.SS3.SSS1.p1.1.m1.1.2.1"></times><apply id="S3.SS3.SSS1.p1.1.m1.1.2.2.cmml" xref="S3.SS3.SSS1.p1.1.m1.1.2.2"><csymbol cd="ambiguous" id="S3.SS3.SSS1.p1.1.m1.1.2.2.1.cmml" xref="S3.SS3.SSS1.p1.1.m1.1.2.2">subscript</csymbol><ci id="S3.SS3.SSS1.p1.1.m1.1.2.2.2.cmml" xref="S3.SS3.SSS1.p1.1.m1.1.2.2.2">ùêª</ci><ci id="S3.SS3.SSS1.p1.1.m1.1.2.2.3.cmml" xref="S3.SS3.SSS1.p1.1.m1.1.2.2.3">ùëÜ</ci></apply><ci id="S3.SS3.SSS1.p1.1.m1.1.1.cmml" xref="S3.SS3.SSS1.p1.1.m1.1.1">ùëÇ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS3.SSS1.p1.1.m1.1c">H_{S}(O)</annotation></semantics></math> and <math id="S3.SS3.SSS1.p1.2.m2.1" class="ltx_Math" alttext="H_{S}(C)" display="inline"><semantics id="S3.SS3.SSS1.p1.2.m2.1a"><mrow id="S3.SS3.SSS1.p1.2.m2.1.2" xref="S3.SS3.SSS1.p1.2.m2.1.2.cmml"><msub id="S3.SS3.SSS1.p1.2.m2.1.2.2" xref="S3.SS3.SSS1.p1.2.m2.1.2.2.cmml"><mi id="S3.SS3.SSS1.p1.2.m2.1.2.2.2" xref="S3.SS3.SSS1.p1.2.m2.1.2.2.2.cmml">H</mi><mi id="S3.SS3.SSS1.p1.2.m2.1.2.2.3" xref="S3.SS3.SSS1.p1.2.m2.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S3.SS3.SSS1.p1.2.m2.1.2.1" xref="S3.SS3.SSS1.p1.2.m2.1.2.1.cmml">‚Äã</mo><mrow id="S3.SS3.SSS1.p1.2.m2.1.2.3.2" xref="S3.SS3.SSS1.p1.2.m2.1.2.cmml"><mo stretchy="false" id="S3.SS3.SSS1.p1.2.m2.1.2.3.2.1" xref="S3.SS3.SSS1.p1.2.m2.1.2.cmml">(</mo><mi id="S3.SS3.SSS1.p1.2.m2.1.1" xref="S3.SS3.SSS1.p1.2.m2.1.1.cmml">C</mi><mo stretchy="false" id="S3.SS3.SSS1.p1.2.m2.1.2.3.2.2" xref="S3.SS3.SSS1.p1.2.m2.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.SS3.SSS1.p1.2.m2.1b"><apply id="S3.SS3.SSS1.p1.2.m2.1.2.cmml" xref="S3.SS3.SSS1.p1.2.m2.1.2"><times id="S3.SS3.SSS1.p1.2.m2.1.2.1.cmml" xref="S3.SS3.SSS1.p1.2.m2.1.2.1"></times><apply id="S3.SS3.SSS1.p1.2.m2.1.2.2.cmml" xref="S3.SS3.SSS1.p1.2.m2.1.2.2"><csymbol cd="ambiguous" id="S3.SS3.SSS1.p1.2.m2.1.2.2.1.cmml" xref="S3.SS3.SSS1.p1.2.m2.1.2.2">subscript</csymbol><ci id="S3.SS3.SSS1.p1.2.m2.1.2.2.2.cmml" xref="S3.SS3.SSS1.p1.2.m2.1.2.2.2">ùêª</ci><ci id="S3.SS3.SSS1.p1.2.m2.1.2.2.3.cmml" xref="S3.SS3.SSS1.p1.2.m2.1.2.2.3">ùëÜ</ci></apply><ci id="S3.SS3.SSS1.p1.2.m2.1.1.cmml" xref="S3.SS3.SSS1.p1.2.m2.1.1">ùê∂</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS3.SSS1.p1.2.m2.1c">H_{S}(C)</annotation></semantics></math>) and determine an optimal sample size. Indeed, the GME-sampler operates as a ranking method, where the article selected at step <math id="S3.SS3.SSS1.p1.3.m3.1" class="ltx_Math" alttext="n" display="inline"><semantics id="S3.SS3.SSS1.p1.3.m3.1a"><mi id="S3.SS3.SSS1.p1.3.m3.1.1" xref="S3.SS3.SSS1.p1.3.m3.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="S3.SS3.SSS1.p1.3.m3.1b"><ci id="S3.SS3.SSS1.p1.3.m3.1.1.cmml" xref="S3.SS3.SSS1.p1.3.m3.1.1">ùëõ</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.SS3.SSS1.p1.3.m3.1c">n</annotation></semantics></math>, is the one which contributes the most to the diversity of the set of the <math id="S3.SS3.SSS1.p1.4.m4.1" class="ltx_Math" alttext="n-1" display="inline"><semantics id="S3.SS3.SSS1.p1.4.m4.1a"><mrow id="S3.SS3.SSS1.p1.4.m4.1.1" xref="S3.SS3.SSS1.p1.4.m4.1.1.cmml"><mi id="S3.SS3.SSS1.p1.4.m4.1.1.2" xref="S3.SS3.SSS1.p1.4.m4.1.1.2.cmml">n</mi><mo id="S3.SS3.SSS1.p1.4.m4.1.1.1" xref="S3.SS3.SSS1.p1.4.m4.1.1.1.cmml">‚àí</mo><mn id="S3.SS3.SSS1.p1.4.m4.1.1.3" xref="S3.SS3.SSS1.p1.4.m4.1.1.3.cmml">1</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS3.SSS1.p1.4.m4.1b"><apply id="S3.SS3.SSS1.p1.4.m4.1.1.cmml" xref="S3.SS3.SSS1.p1.4.m4.1.1"><minus id="S3.SS3.SSS1.p1.4.m4.1.1.1.cmml" xref="S3.SS3.SSS1.p1.4.m4.1.1.1"></minus><ci id="S3.SS3.SSS1.p1.4.m4.1.1.2.cmml" xref="S3.SS3.SSS1.p1.4.m4.1.1.2">ùëõ</ci><cn type="integer" id="S3.SS3.SSS1.p1.4.m4.1.1.3.cmml" xref="S3.SS3.SSS1.p1.4.m4.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS3.SSS1.p1.4.m4.1c">n-1</annotation></semantics></math> articles selected upstream. For both organisms and chemicals, diversity increases rapidly in the first hundred ranked items, followed by a plateau. Specifically for organisms (regardless of the kingdom), diversity showed a decline in the second half of the sampled items (supplementary table <a href="#S2.T6" title="Table S6 ‚Ä£ S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S6</span></a>). This is the signal that the addition of new articles provide relations for already well-covered organisms and disrupted the existing balance in the organism distribution. In contrast, the impact of newly added articles on chemicals is negligible, likely because they represent a larger set of distinct entities (supplementary table <a href="#S2.T5" title="Table S5 ‚Ä£ S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S5</span></a>). To keep a reasonable balance between diversity and sample size, we decided to only retain the top <math id="S3.SS3.SSS1.p1.5.m5.1" class="ltx_Math" alttext="n=500" display="inline"><semantics id="S3.SS3.SSS1.p1.5.m5.1a"><mrow id="S3.SS3.SSS1.p1.5.m5.1.1" xref="S3.SS3.SSS1.p1.5.m5.1.1.cmml"><mi id="S3.SS3.SSS1.p1.5.m5.1.1.2" xref="S3.SS3.SSS1.p1.5.m5.1.1.2.cmml">n</mi><mo id="S3.SS3.SSS1.p1.5.m5.1.1.1" xref="S3.SS3.SSS1.p1.5.m5.1.1.1.cmml">=</mo><mn id="S3.SS3.SSS1.p1.5.m5.1.1.3" xref="S3.SS3.SSS1.p1.5.m5.1.1.3.cmml">500</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS3.SSS1.p1.5.m5.1b"><apply id="S3.SS3.SSS1.p1.5.m5.1.1.cmml" xref="S3.SS3.SSS1.p1.5.m5.1.1"><eq id="S3.SS3.SSS1.p1.5.m5.1.1.1.cmml" xref="S3.SS3.SSS1.p1.5.m5.1.1.1"></eq><ci id="S3.SS3.SSS1.p1.5.m5.1.1.2.cmml" xref="S3.SS3.SSS1.p1.5.m5.1.1.2">ùëõ</ci><cn type="integer" id="S3.SS3.SSS1.p1.5.m5.1.1.3.cmml" xref="S3.SS3.SSS1.p1.5.m5.1.1.3">500</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS3.SSS1.p1.5.m5.1c">n=500</annotation></semantics></math> ranked articles per kingdoms, ensuring at least <math id="S3.SS3.SSS1.p1.6.m6.1" class="ltx_Math" alttext="80\%" display="inline"><semantics id="S3.SS3.SSS1.p1.6.m6.1a"><mrow id="S3.SS3.SSS1.p1.6.m6.1.1" xref="S3.SS3.SSS1.p1.6.m6.1.1.cmml"><mn id="S3.SS3.SSS1.p1.6.m6.1.1.2" xref="S3.SS3.SSS1.p1.6.m6.1.1.2.cmml">80</mn><mo id="S3.SS3.SSS1.p1.6.m6.1.1.1" xref="S3.SS3.SSS1.p1.6.m6.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="S3.SS3.SSS1.p1.6.m6.1b"><apply id="S3.SS3.SSS1.p1.6.m6.1.1.cmml" xref="S3.SS3.SSS1.p1.6.m6.1.1"><csymbol cd="latexml" id="S3.SS3.SSS1.p1.6.m6.1.1.1.cmml" xref="S3.SS3.SSS1.p1.6.m6.1.1.1">percent</csymbol><cn type="integer" id="S3.SS3.SSS1.p1.6.m6.1.1.2.cmml" xref="S3.SS3.SSS1.p1.6.m6.1.1.2">80</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS3.SSS1.p1.6.m6.1c">80\%</annotation></semantics></math> of the maximal observed entropy on both organisms and chemicals (Figure <a href="#S3.F5" title="Figure 5 ‚Ä£ 3.3 Building a diversity-augmented dataset ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>:Bottom-panel). The proportions of maximal observed entropy at alternative sample sizes are presented in supplementary table <a href="#S2.T7" title="Table S7 ‚Ä£ S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S7</span></a>.
<br class="ltx_break">The impact of the diversity-sampling strategy is evaluated by comparing the composition of the sample against 5 random samples of equivalent sizes<span id="footnote5" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">5</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">5</sup><span class="ltx_tag ltx_tag_note">5</span>Each random sample is composed of 500 random literature items sampled per kingdoms</span></span></span>. The original diversity sample and the extracted random samples are respectively denoted as <span id="S3.SS3.SSS1.p1.6.1" class="ltx_text ltx_font_italic">Diversity</span> and <span id="S3.SS3.SSS1.p1.6.2" class="ltx_text ltx_font_italic">Random</span> samples. While showing similar kingdoms‚Äô coverage because of the common stratification procedure (Figure <a href="#S3.F4" title="Figure 4 ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.B bottom), the diversity sample is, as expected, significantly richer in terms of distinct number of chemicals, organisms and relations (Figure <a href="#S3.F4" title="Figure 4 ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.C. This improved diversity is also reflected in a reduced pareto effect for the distribution of the organisms (negligible for chemicals), and overlap between the entities reported in each article (Figures <a href="#S3.F4" title="Figure 4 ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.D and E). A more comprehensive comparison against others possible sampling strategies for diversity are discussed in supplementary <a href="#S2.SS2a" title="S2.2 Diversity-sampling: alternative comparisons ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2.2</span></a>.</p>
</div>
</section>
<section id="S3.SS3.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.2 </span>Distance between standardised annotations and original text</h4>

<div id="S3.SS3.SSS2.p1" class="ltx_para ltx_noindent">
<p id="S3.SS3.SSS2.p1.1" class="ltx_p">Several works emphasize the importance of data quality over quantity for fine-tuning language models <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib73" title="" class="ltx_ref">73</a>, <a href="#bib.bib84" title="" class="ltx_ref">84</a>]</cite>. LOTUS data are recognized as being of high-quality, particularly because of the harmonization, cleaning and validation steps of the workflow, aligning original records from several open NP databases into standardized structures and organisms in Wikidata. Although this is essential to ensure data FAIRness, these processes logically distance the standardised entries from their original literal mentions in the referenced articles. To get a rough estimate of this distance, the <span id="S3.SS3.SSS2.p1.1.1" class="ltx_text ltx_font_italic">Diversity</span> and the 5 <span id="S3.SS3.SSS2.p1.1.2" class="ltx_text ltx_font_italic">Random</span> samples were merged into a single <span id="S3.SS3.SSS2.p1.1.3" class="ltx_text ltx_font_italic">Extended</span> dataset. Then, we estimated the proportion of the labels of the standardized entities that could be found in the original abstracts of articles reporting the relationships. Details of this estimation in supplementary <a href="#S2.SS3a" title="S2.3 Mismatches between standardized labels and original abstracts ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2.3</span></a>. More than 2/3 of the organisms labels are effectively retrieved in the original abstract, while less than half of the chemical names can be retrieved, even considering their synonyms (Figures <a href="#S3.F4" title="Figure 4 ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.F). Assuming that these two types of mismatches are independent, only 1/3 of the reported pairs would be completely found in an abstract. Finally, some reported NP relationships are simply not expressed in the abstract of the cited reference, but have been reported from the body of the article or supplementary materials<span id="footnote6" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">6</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">6</sup><span class="ltx_tag ltx_tag_note">6</span>However, we have chosen to focus only on abstracts and not on full texts because of their much greater availability and their synthetic forms.</span></span></span>. Whether they are derived from the <span id="S3.SS3.SSS2.p1.1.4" class="ltx_text ltx_font_italic">Diversity</span> or a <span id="S3.SS3.SSS2.p1.1.5" class="ltx_text ltx_font_italic">Random</span> sample, these noisy examples make the training of a model challenging because some labels to be predicted are missing from the input text <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib85" title="" class="ltx_ref">85</a>, <a href="#bib.bib86" title="" class="ltx_ref">86</a>]</cite>. In this context, alternative strategies like Zero-shot or Few-shot learning (also called In-Context Learning) based on open LLMs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib87" title="" class="ltx_ref">87</a>, <a href="#bib.bib88" title="" class="ltx_ref">88</a>]</cite> also need to be considered.</p>
</div>
</section>
<section id="S3.SS3.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.3 </span>Creating a manually curated evaluation dataset</h4>

<div id="S3.SS3.SSS3.p1" class="ltx_para ltx_noindent">
<p id="S3.SS3.SSS3.p1.1" class="ltx_p">If these discrepancies certainly affect the training of a model, they are a more sensitive issue in an evaluation set <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib89" title="" class="ltx_ref">89</a>]</cite>. Also, if diversity can be an important feature for a training set <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib26" title="" class="ltx_ref">26</a>]</cite>, it is arguably also important for an evaluation set <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite>. While smaller by design, the evaluation set needs to be representative. Finally, since the manual curation of an evaluation set is an expensive and time-consuming task, the selected set of entries need to be chosen carefully <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib14" title="" class="ltx_ref">14</a>]</cite>. Considering the last points, the knee-points of the entropy curves (where the entropy increases weaker by new added articles) obtained with the GME-sampler suggest relevant tradeoffs between sample size and diversity (<a href="#S3.F5" title="Figure 5 ‚Ä£ 3.3 Building a diversity-augmented dataset ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>:Bottom-panel) early in the sampling. Nonetheless, as they vary between different biological kingdoms, on organisms and chemicals, and could be too restrictive, the extended set of the top 50 items from each kingdom was extracted, resulting in an evaluation dataset of 200 abstracts. The abstracts were manually curated by an expert, annotating all instances of mentioned organism-NP relationships in their order of appearance in the original text and using established identifiers such as Wikidata IDs and PubChem IDs. As isolated chemicals are sometimes grouped into chemical families for the sake of brevity in abstracts, all mentions of a more general chemical family were also annotated. The curated evaluation set is publicly available at <a target="_blank" href="https://zenodo.org/records/8422007" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://zenodo.org/records/8422007</a>. Details about the curation protocol are available in supplementary <a href="#S2.SS4" title="S2.4 Dataset curation protocol ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2.4</span></a>. The benefit of the diversity-sampling for building the evaluation set is discussed in more details in supplementary <a href="#S2.SS5" title="S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2.5</span></a>.</p>
</div>
</section>
</section>
<section id="S3.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.4 </span>Few-shot learning approaches the performance of standard fine-tuning on raw data</h3>

<div id="S3.SS4.p1" class="ltx_para ltx_noindent">
<p id="S3.SS4.p1.2" class="ltx_p">The mismatches between the standardized labels and the original abstracts have therefore been corrected for the evaluation set. However, due to the considerable investment of time and resources required for this task, the same corrections were not applied on the remaining data available for training. In this particular context of noisy data for end-to-end RE, two strategies were evaluated: standard fine-tuning and few-shot learning, the latter being able to rely only on a few manually selected examples. The performance of the fine-tuning strategy were evaluated based on all available examples (<span id="S3.SS4.p1.2.1" class="ltx_text ltx_font_typewriter">Full</span>) and 3 alternatives train/valid datasets derived from the initial <span id="S3.SS4.p1.2.2" class="ltx_text ltx_font_italic">Diversity</span>, <span id="S3.SS4.p1.2.3" class="ltx_text ltx_font_italic">Random</span> and <span id="S3.SS4.p1.2.4" class="ltx_text ltx_font_italic">Extended</span> samples, here after referred as: <span id="S3.SS4.p1.2.5" class="ltx_text ltx_font_typewriter">Diversity-raw</span>, <span id="S3.SS4.p1.2.6" class="ltx_text ltx_font_typewriter">Random-raw</span> and their union <span id="S3.SS4.p1.2.7" class="ltx_text ltx_font_typewriter">Extended-raw</span>. Their respective sizes and splits are detailed in supplementary table <a href="#S2.T8" title="Table S8 ‚Ä£ S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S8</span></a>. All datasets were used to train 3 models for end-to-end RE: Seq2rel, BioGPT and GPT-2. Six open LLMs were also evaluated in few-shot learning settings: LLaMA 7B, 13B, 30B and 65B, along with two models, respectively fine-tuned on instructions and conversations and derived from LLaMA 7B and 13B: Alpaca-7B and Vicuna-13B.
<br class="ltx_break">Best performance in fine-tuning settings was achieved by BioGPT (Table <a href="#S3.T1" title="Table 1 ‚Ä£ 3.4 Few-shot learning approaches the performance of standard fine-tuning on raw data ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>). Regardless of the training dataset<span id="footnote7" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">7</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">7</sup><span class="ltx_tag ltx_tag_note">7</span>Except for <span id="footnote7.1" class="ltx_text ltx_font_typewriter">Full</span></span></span></span>, it consistently outperformed Seq2rel and GPT-2 and demonstrated an <span id="S3.SS4.p1.2.8" class="ltx_text ltx_markedasmath">f1-score</span> of <math id="S3.SS4.p1.2.m2.1" class="ltx_Math" alttext="32.5\%" display="inline"><semantics id="S3.SS4.p1.2.m2.1a"><mrow id="S3.SS4.p1.2.m2.1.1" xref="S3.SS4.p1.2.m2.1.1.cmml"><mn id="S3.SS4.p1.2.m2.1.1.2" xref="S3.SS4.p1.2.m2.1.1.2.cmml">32.5</mn><mo id="S3.SS4.p1.2.m2.1.1.1" xref="S3.SS4.p1.2.m2.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="S3.SS4.p1.2.m2.1b"><apply id="S3.SS4.p1.2.m2.1.1.cmml" xref="S3.SS4.p1.2.m2.1.1"><csymbol cd="latexml" id="S3.SS4.p1.2.m2.1.1.1.cmml" xref="S3.SS4.p1.2.m2.1.1.1">percent</csymbol><cn type="float" id="S3.SS4.p1.2.m2.1.1.2.cmml" xref="S3.SS4.p1.2.m2.1.1.2">32.5</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS4.p1.2.m2.1c">32.5\%</annotation></semantics></math> when trained on <span id="S3.SS4.p1.2.9" class="ltx_text ltx_font_typewriter">Extended-raw</span>. We also evaluated the influence of the different training datasets on models performance. The results indicate that models trained on <span id="S3.SS4.p1.2.10" class="ltx_text ltx_font_typewriter">Diversity-raw</span> outperformed those models trained on <span id="S3.SS4.p1.2.11" class="ltx_text ltx_font_typewriter">Random-raw</span>, with a notable improvement in recall at the expense of precision. Merging the datasets into a larger (<span id="S3.SS4.p1.2.12" class="ltx_text ltx_font_typewriter">Extended-raw</span>) also resulted in improved performance for all models. However, expanding the dataset to all available examples only barely improved the previous performance and surprisingly underperformed with BioGPT. In few-shot learning scenarios, the best performance was obtained with LLaMA-65B and decline with smaller models. Although the performance was inferior compared to fine-tuned alternatives, the models achieved reasonable scores considering the limited number of archetypal examples provided. These results also emphasize the potential of few-shot learning or prompt-tuning based approaches in practical context with low-resources.</p>
</div>
<figure id="S3.T1" class="ltx_table">
<table id="S3.T1.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S3.T1.1.1" class="ltx_tr">
<td id="S3.T1.1.1.1" class="ltx_td ltx_align_left ltx_border_tt"><span id="S3.T1.1.1.1.1" class="ltx_text ltx_font_bold">model</span></td>
<td id="S3.T1.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T1.1.1.2.1" class="ltx_text ltx_font_bold">Training</span></td>
<td id="S3.T1.1.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T1.1.1.3.1" class="ltx_text ltx_font_bold">precision</span></td>
<td id="S3.T1.1.1.4" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T1.1.1.4.1" class="ltx_text ltx_font_bold">recall</span></td>
<td id="S3.T1.1.1.5" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T1.1.1.5.1" class="ltx_text ltx_font_bold">f1</span></td>
<td id="S3.T1.1.1.6" class="ltx_td ltx_border_tt"></td>
</tr>
<tr id="S3.T1.1.2" class="ltx_tr">
<td id="S3.T1.1.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T1.1.2.1.1" class="ltx_text ltx_font_bold">LLaMA-7B</span></td>
<td id="S3.T1.1.2.2" class="ltx_td ltx_align_right ltx_border_t" rowspan="6"><span id="S3.T1.1.2.2.1" class="ltx_text">Few-shot learning (5-shot)</span></td>
<td id="S3.T1.1.2.3" class="ltx_td ltx_align_right ltx_border_t">27.0</td>
<td id="S3.T1.1.2.4" class="ltx_td ltx_align_right ltx_border_t">9.04</td>
<td id="S3.T1.1.2.5" class="ltx_td ltx_align_right ltx_border_t">13.55</td>
<td id="S3.T1.1.2.6" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T1.1.3" class="ltx_tr">
<td id="S3.T1.1.3.1" class="ltx_td ltx_align_left"><span id="S3.T1.1.3.1.1" class="ltx_text ltx_font_bold">LLaMA-13B</span></td>
<td id="S3.T1.1.3.2" class="ltx_td ltx_align_right">35.64</td>
<td id="S3.T1.1.3.3" class="ltx_td ltx_align_right">23.64</td>
<td id="S3.T1.1.3.4" class="ltx_td ltx_align_right">28.49</td>
<td id="S3.T1.1.3.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.4" class="ltx_tr">
<td id="S3.T1.1.4.1" class="ltx_td ltx_align_left"><span id="S3.T1.1.4.1.1" class="ltx_text ltx_font_bold">LLaMA-30B</span></td>
<td id="S3.T1.1.4.2" class="ltx_td ltx_align_right">38.51</td>
<td id="S3.T1.1.4.3" class="ltx_td ltx_align_right"><span id="S3.T1.1.4.3.1" class="ltx_text ltx_framed ltx_framed_underline">23.24</span></td>
<td id="S3.T1.1.4.4" class="ltx_td ltx_align_right">28.99</td>
<td id="S3.T1.1.4.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.5" class="ltx_tr">
<td id="S3.T1.1.5.1" class="ltx_td ltx_align_left"><span id="S3.T1.1.5.1.1" class="ltx_text ltx_font_bold">LLaMA-65B</span></td>
<td id="S3.T1.1.5.2" class="ltx_td ltx_align_right"><span id="S3.T1.1.5.2.1" class="ltx_text ltx_framed ltx_framed_underline">40.16</span></td>
<td id="S3.T1.1.5.3" class="ltx_td ltx_align_right">22.97</td>
<td id="S3.T1.1.5.4" class="ltx_td ltx_align_right"><span id="S3.T1.1.5.4.1" class="ltx_text ltx_framed ltx_framed_underline">29.23</span></td>
<td id="S3.T1.1.5.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.6" class="ltx_tr">
<td id="S3.T1.1.6.1" class="ltx_td ltx_align_left"><span id="S3.T1.1.6.1.1" class="ltx_text ltx_font_bold">Alpaca-7B</span></td>
<td id="S3.T1.1.6.2" class="ltx_td ltx_align_right">15.14</td>
<td id="S3.T1.1.6.3" class="ltx_td ltx_align_right">2.21</td>
<td id="S3.T1.1.6.4" class="ltx_td ltx_align_right">5.86</td>
<td id="S3.T1.1.6.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.7" class="ltx_tr">
<td id="S3.T1.1.7.1" class="ltx_td ltx_align_left"><span id="S3.T1.1.7.1.1" class="ltx_text ltx_font_bold">Vicuna-13B</span></td>
<td id="S3.T1.1.7.2" class="ltx_td ltx_align_right">38.4</td>
<td id="S3.T1.1.7.3" class="ltx_td ltx_align_right">20.43</td>
<td id="S3.T1.1.7.4" class="ltx_td ltx_align_right">26.48</td>
<td id="S3.T1.1.7.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.8" class="ltx_tr">
<td id="S3.T1.1.8.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T1.1.8.1.1" class="ltx_text ltx_font_bold">Seq2rel</span></td>
<td id="S3.T1.1.8.2" class="ltx_td ltx_align_right ltx_border_t"><span id="S3.T1.1.8.2.1" class="ltx_text ltx_font_typewriter">Random-raw</span></td>
<td id="S3.T1.1.8.3" class="ltx_td ltx_align_right ltx_border_t">43.2 +/- (6.67)</td>
<td id="S3.T1.1.8.4" class="ltx_td ltx_align_right ltx_border_t">4.8 +/- (1.16)</td>
<td id="S3.T1.1.8.5" class="ltx_td ltx_align_right ltx_border_t">8.6 +/- (2.00)</td>
<td id="S3.T1.1.8.6" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T1.1.9" class="ltx_tr">
<td id="S3.T1.1.9.1" class="ltx_td ltx_align_right"><span id="S3.T1.1.9.1.1" class="ltx_text ltx_font_typewriter">Diversty-raw</span></td>
<td id="S3.T1.1.9.2" class="ltx_td ltx_align_right">39.6</td>
<td id="S3.T1.1.9.3" class="ltx_td ltx_align_right">5.4</td>
<td id="S3.T1.1.9.4" class="ltx_td ltx_align_right">9.5</td>
<td id="S3.T1.1.9.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.10" class="ltx_tr">
<td id="S3.T1.1.10.1" class="ltx_td ltx_align_right"><span id="S3.T1.1.10.1.1" class="ltx_text ltx_font_typewriter">Extended-raw</span></td>
<td id="S3.T1.1.10.2" class="ltx_td ltx_align_right"><span id="S3.T1.1.10.2.1" class="ltx_text ltx_font_bold">47.3</span></td>
<td id="S3.T1.1.10.3" class="ltx_td ltx_align_right">5.8</td>
<td id="S3.T1.1.10.4" class="ltx_td ltx_align_right">10.4</td>
<td id="S3.T1.1.10.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.11" class="ltx_tr">
<td id="S3.T1.1.11.1" class="ltx_td"></td>
<td id="S3.T1.1.11.2" class="ltx_td ltx_align_right"><span id="S3.T1.1.11.2.1" class="ltx_text ltx_font_typewriter">Full</span></td>
<td id="S3.T1.1.11.3" class="ltx_td ltx_align_right">45.6</td>
<td id="S3.T1.1.11.4" class="ltx_td ltx_align_right">7.1</td>
<td id="S3.T1.1.11.5" class="ltx_td ltx_align_right">12.2</td>
<td id="S3.T1.1.11.6" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.12" class="ltx_tr">
<td id="S3.T1.1.12.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T1.1.12.1.1" class="ltx_text ltx_font_bold">GPT-2</span></td>
<td id="S3.T1.1.12.2" class="ltx_td ltx_align_right ltx_border_t"><span id="S3.T1.1.12.2.1" class="ltx_text ltx_font_typewriter">Random-raw</span></td>
<td id="S3.T1.1.12.3" class="ltx_td ltx_align_right ltx_border_t">32.5 +/- (4.83)</td>
<td id="S3.T1.1.12.4" class="ltx_td ltx_align_right ltx_border_t">11.8 +/- (5.25)</td>
<td id="S3.T1.1.12.5" class="ltx_td ltx_align_right ltx_border_t">15.0 +/- (2.54)</td>
<td id="S3.T1.1.12.6" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T1.1.13" class="ltx_tr">
<td id="S3.T1.1.13.1" class="ltx_td ltx_align_right"><span id="S3.T1.1.13.1.1" class="ltx_text ltx_font_typewriter">Diversty-raw</span></td>
<td id="S3.T1.1.13.2" class="ltx_td ltx_align_right">22.3</td>
<td id="S3.T1.1.13.3" class="ltx_td ltx_align_right">19.2</td>
<td id="S3.T1.1.13.4" class="ltx_td ltx_align_right">20.6</td>
<td id="S3.T1.1.13.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.14" class="ltx_tr">
<td id="S3.T1.1.14.1" class="ltx_td ltx_align_right"><span id="S3.T1.1.14.1.1" class="ltx_text ltx_font_typewriter">Extended-raw</span></td>
<td id="S3.T1.1.14.2" class="ltx_td ltx_align_right">44.8</td>
<td id="S3.T1.1.14.3" class="ltx_td ltx_align_right">21.7</td>
<td id="S3.T1.1.14.4" class="ltx_td ltx_align_right">29.3</td>
<td id="S3.T1.1.14.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.15" class="ltx_tr">
<td id="S3.T1.1.15.1" class="ltx_td"></td>
<td id="S3.T1.1.15.2" class="ltx_td ltx_align_right"><span id="S3.T1.1.15.2.1" class="ltx_text ltx_font_typewriter">Full</span></td>
<td id="S3.T1.1.15.3" class="ltx_td ltx_align_right">47.5</td>
<td id="S3.T1.1.15.4" class="ltx_td ltx_align_right">22.5</td>
<td id="S3.T1.1.15.5" class="ltx_td ltx_align_right">30.5</td>
<td id="S3.T1.1.15.6" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.16" class="ltx_tr">
<td id="S3.T1.1.16.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T1.1.16.1.1" class="ltx_text ltx_font_bold">BioGPT</span></td>
<td id="S3.T1.1.16.2" class="ltx_td ltx_align_right ltx_border_t"><span id="S3.T1.1.16.2.1" class="ltx_text ltx_font_typewriter">Random-raw</span></td>
<td id="S3.T1.1.16.3" class="ltx_td ltx_align_right ltx_border_t">47.2 +/- (4.01)</td>
<td id="S3.T1.1.16.4" class="ltx_td ltx_align_right ltx_border_t">19.8 +/- (2.71)</td>
<td id="S3.T1.1.16.5" class="ltx_td ltx_align_right ltx_border_t">27.6 +/- (2.48)</td>
<td id="S3.T1.1.16.6" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T1.1.17" class="ltx_tr">
<td id="S3.T1.1.17.1" class="ltx_td ltx_align_right"><span id="S3.T1.1.17.1.1" class="ltx_text ltx_font_typewriter">Diversty-raw</span></td>
<td id="S3.T1.1.17.2" class="ltx_td ltx_align_right">37.1</td>
<td id="S3.T1.1.17.3" class="ltx_td ltx_align_right"><span id="S3.T1.1.17.3.1" class="ltx_text ltx_font_bold">28.4</span></td>
<td id="S3.T1.1.17.4" class="ltx_td ltx_align_right">32.2</td>
<td id="S3.T1.1.17.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.18" class="ltx_tr">
<td id="S3.T1.1.18.1" class="ltx_td ltx_align_right"><span id="S3.T1.1.18.1.1" class="ltx_text ltx_font_typewriter">Extended-raw</span></td>
<td id="S3.T1.1.18.2" class="ltx_td ltx_align_right">42.2</td>
<td id="S3.T1.1.18.3" class="ltx_td ltx_align_right">26.5</td>
<td id="S3.T1.1.18.4" class="ltx_td ltx_align_right"><span id="S3.T1.1.18.4.1" class="ltx_text ltx_font_bold">32.5</span></td>
<td id="S3.T1.1.18.5" class="ltx_td"></td>
</tr>
<tr id="S3.T1.1.19" class="ltx_tr">
<td id="S3.T1.1.19.1" class="ltx_td ltx_border_bb"></td>
<td id="S3.T1.1.19.2" class="ltx_td ltx_align_right ltx_border_bb"><span id="S3.T1.1.19.2.1" class="ltx_text ltx_font_typewriter">Full</span></td>
<td id="S3.T1.1.19.3" class="ltx_td ltx_align_right ltx_border_bb">46.7</td>
<td id="S3.T1.1.19.4" class="ltx_td ltx_align_right ltx_border_bb">21.3</td>
<td id="S3.T1.1.19.5" class="ltx_td ltx_align_right ltx_border_bb">29.3</td>
<td id="S3.T1.1.19.6" class="ltx_td ltx_border_bb"></td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 1: </span>Performance of 5-shot <span id="S3.T1.8.1" class="ltx_text ltx_font_italic">in-context</span> learning using LLaMA and LLaMA-instructed models compared to <span id="S3.T1.9.2" class="ltx_text ltx_font_italic">seq2rel</span>, GPT-2, BioGPT finetuned models. Three types of training dataset are evaluated: the diversity sample (<span id="S3.T1.10.3" class="ltx_text ltx_font_typewriter">Diversty-raw</span>), 5 random samples (<span id="S3.T1.11.4" class="ltx_text ltx_font_typewriter">Random-raw</span>) and the extended sample (<span id="S3.T1.12.5" class="ltx_text ltx_font_typewriter">Extended-raw</span>), which is the union of the previous samples. <span id="S3.T1.13.6" class="ltx_text ltx_font_typewriter">Full</span> is a dataset contained all available examples from the LOTUS snapshot, except the 200 used in the evaluation set. Best performance via finetuning are bold, while best performance in few-shot settings are underlined.</figcaption>
</figure>
</section>
<section id="S3.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.5 </span>Reversing the task: Generation of synthetic data with open LLMs</h3>

<div id="S3.SS5.p1" class="ltx_para ltx_noindent">
<p id="S3.SS5.p1.2" class="ltx_p">While LLMs cannot compete in terms of performance with fine-tuned approaches in the evaluated settings, their generative abilities could be used alternatively to address the main bottleneck: the discrepancies between the input text and the labels in the training data. It requires going beyond distant supervision or data augmentation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib31" title="" class="ltx_ref">31</a>, <a href="#bib.bib91" title="" class="ltx_ref">91</a>, <a href="#bib.bib33" title="" class="ltx_ref">33</a>]</cite>. The former involves mapping relationships from a Knowledge Base to a large corpus of text to generate pseudo-labels, whereas the latter entails applying a range of transformations, permutations, or morphings to a core set of high-quality examples. In contrast, the adaptive described approach propose to generate a set of synthetic input abstracts from a pre-defined context and a set of expected output labels (i.e organism - NP relationships).
<br class="ltx_break">To maintain consistency, each synthetic abstract is based on the context and results reported from an original seed abstract. The first step is to generate the instructions to prompt the selected LLM for generation. The instructions are composed of a title, a list of keywords and the verbalised main findings (Method <a href="#S2.SS3" title="2.3 Synthetic abstracts generation ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.3</span></a>). We decided to use the open source Vicuna-13B <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib92" title="" class="ltx_ref">92</a>]</cite><span id="footnote8" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">8</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">8</sup><span class="ltx_tag ltx_tag_note">8</span>version v1.3 from 22/06/2023: <a target="_blank" href="https://huggingface.co/lmsys/vicuna-13b-v1.3" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://huggingface.co/lmsys/vicuna-13b-v1.3</a></span></span></span>, a LLaMA-13B model fine-tuned on user-shared conversations collected from ShareGPT<span id="footnote9" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">9</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">9</sup><span class="ltx_tag ltx_tag_note">9</span><a target="_blank" href="https://sharegpt.com/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://sharegpt.com/</a></span></span></span>, which outperforms alternatives of equivalent sizes on several benchmarks <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib73" title="" class="ltx_ref">73</a>]</cite>. For each input seed abstract, the top-10 extracted keywords were used in the built instruction. As this is a crucial step, the performance of Vicuna-13B to extract keywords have been evaluated on the SemEval2017-Task10 dataset <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib93" title="" class="ltx_ref">93</a>]</cite> in supplementary <a href="#S2.SS6" title="S2.6 Evaluation of Keyword Extraction on the SemEVAL2017 dataset ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2.6</span></a>. To diversify the generated abstracts, <math id="S3.SS5.p1.1.m1.1" class="ltx_Math" alttext="m=10" display="inline"><semantics id="S3.SS5.p1.1.m1.1a"><mrow id="S3.SS5.p1.1.m1.1.1" xref="S3.SS5.p1.1.m1.1.1.cmml"><mi id="S3.SS5.p1.1.m1.1.1.2" xref="S3.SS5.p1.1.m1.1.1.2.cmml">m</mi><mo id="S3.SS5.p1.1.m1.1.1.1" xref="S3.SS5.p1.1.m1.1.1.1.cmml">=</mo><mn id="S3.SS5.p1.1.m1.1.1.3" xref="S3.SS5.p1.1.m1.1.1.3.cmml">10</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS5.p1.1.m1.1b"><apply id="S3.SS5.p1.1.m1.1.1.cmml" xref="S3.SS5.p1.1.m1.1.1"><eq id="S3.SS5.p1.1.m1.1.1.1.cmml" xref="S3.SS5.p1.1.m1.1.1.1"></eq><ci id="S3.SS5.p1.1.m1.1.1.2.cmml" xref="S3.SS5.p1.1.m1.1.1.2">ùëö</ci><cn type="integer" id="S3.SS5.p1.1.m1.1.1.3.cmml" xref="S3.SS5.p1.1.m1.1.1.3">10</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS5.p1.1.m1.1c">m=10</annotation></semantics></math> instructions prompts with different verbalisation patterns were then sampled per initial seed article. Finally, only the top <math id="S3.SS5.p1.2.m2.1" class="ltx_Math" alttext="k=3" display="inline"><semantics id="S3.SS5.p1.2.m2.1a"><mrow id="S3.SS5.p1.2.m2.1.1" xref="S3.SS5.p1.2.m2.1.1.cmml"><mi id="S3.SS5.p1.2.m2.1.1.2" xref="S3.SS5.p1.2.m2.1.1.2.cmml">k</mi><mo id="S3.SS5.p1.2.m2.1.1.1" xref="S3.SS5.p1.2.m2.1.1.1.cmml">=</mo><mn id="S3.SS5.p1.2.m2.1.1.3" xref="S3.SS5.p1.2.m2.1.1.3.cmml">3</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS5.p1.2.m2.1b"><apply id="S3.SS5.p1.2.m2.1.1.cmml" xref="S3.SS5.p1.2.m2.1.1"><eq id="S3.SS5.p1.2.m2.1.1.1.cmml" xref="S3.SS5.p1.2.m2.1.1.1"></eq><ci id="S3.SS5.p1.2.m2.1.1.2.cmml" xref="S3.SS5.p1.2.m2.1.1.2">ùëò</ci><cn type="integer" id="S3.SS5.p1.2.m2.1.1.3.cmml" xref="S3.SS5.p1.2.m2.1.1.3">3</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS5.p1.2.m2.1c">k=3</annotation></semantics></math> most relevant synthesized abstracts per seed were selected with the simple, yet effective, selector module.
<br class="ltx_break">To evaluate the impact of diversity-sampling on the seed articles used for synthetic generation, we created two new datasets: <span id="S3.SS5.p1.2.1" class="ltx_text ltx_font_typewriter">Diversity-synth</span> and <span id="S3.SS5.p1.2.2" class="ltx_text ltx_font_typewriter">Random-synth</span>, derived from the original abstracts in the <span id="S3.SS5.p1.2.3" class="ltx_text ltx_font_typewriter">Diversity-raw</span> and <span id="S3.SS5.p1.2.4" class="ltx_text ltx_font_typewriter">Random-raw</span> datasets, respectively. Several illustrative examples of synthetic abstracts from <span id="S3.SS5.p1.2.5" class="ltx_text ltx_font_typewriter">Diversity-synth</span> are discussed in supplementary <a href="#S2.SS7" title="S2.7 LLM prompting for synthetic abstract generation ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2.7</span></a>, highlighting both the variability and the potential caveats (errors, hallucinations) of the process. As with the original data, <span id="S3.SS5.p1.2.6" class="ltx_text ltx_font_typewriter">Diversity-synt</span> and <span id="S3.SS5.p1.2.7" class="ltx_text ltx_font_typewriter">Random-synt</span> were merged in <span id="S3.SS5.p1.2.8" class="ltx_text ltx_font_typewriter">Extended-synt</span> to measure the impact of the dataset size. Statistics of the generated datasets are presented in supplementary table <a href="#S2.T8" title="Table S8 ‚Ä£ S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S8</span></a>. In total, more than 25,000 synthetic abstracts were generated from the 7901 originally contained in the <span id="S3.SS5.p1.2.9" class="ltx_text ltx_font_typewriter">raw</span> datasets. From <span id="S3.SS5.p1.2.10" class="ltx_text ltx_font_typewriter">Diversity-raw</span>, 200 initial items were excluded by the selector module and 162 on average for <span id="S3.SS5.p1.2.11" class="ltx_text ltx_font_typewriter">Random-raw</span>. While the distinct numbers of entities/relations dropped in synthetic datasets, the selector guarantees that these labels are part of the generated abstracts. Furthermore, the generation process enables the integration of examples with chemical classes in the input text and expected labels, which were not available in the original data.
<br class="ltx_break"></p>
</div>
</section>
<section id="S3.SS6" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.6 </span>Training on synthetic data improved performance over noisy raw data</h3>

<div id="S3.SS6.p1" class="ltx_para ltx_noindent">
<p id="S3.SS6.p1.6" class="ltx_p">The synthetic datasets were used to train new instances of the previously evaluated models: Seq2rel, GPT-2 and BioGPT. Although the synthetic training sets (<span id="S3.SS6.p1.6.1" class="ltx_text ltx_font_typewriter">Diversity-synt</span> and <span id="S3.SS6.p1.6.2" class="ltx_text ltx_font_typewriter">Random-synt</span>) are almost half the size of <span id="S3.SS6.p1.6.3" class="ltx_text ltx_font_typewriter">Extended-raw</span> (resp. 3562 and 3798 compared to 7111 examples), on which was established the previous baseline with BioGPT (<math id="S3.SS6.p1.1.m1.1" class="ltx_Math" alttext="\text{f1-score}=32.5" display="inline"><semantics id="S3.SS6.p1.1.m1.1a"><mrow id="S3.SS6.p1.1.m1.1.1" xref="S3.SS6.p1.1.m1.1.1.cmml"><mtext id="S3.SS6.p1.1.m1.1.1.2" xref="S3.SS6.p1.1.m1.1.1.2a.cmml">f1-score</mtext><mo id="S3.SS6.p1.1.m1.1.1.1" xref="S3.SS6.p1.1.m1.1.1.1.cmml">=</mo><mn id="S3.SS6.p1.1.m1.1.1.3" xref="S3.SS6.p1.1.m1.1.1.3.cmml">32.5</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS6.p1.1.m1.1b"><apply id="S3.SS6.p1.1.m1.1.1.cmml" xref="S3.SS6.p1.1.m1.1.1"><eq id="S3.SS6.p1.1.m1.1.1.1.cmml" xref="S3.SS6.p1.1.m1.1.1.1"></eq><ci id="S3.SS6.p1.1.m1.1.1.2a.cmml" xref="S3.SS6.p1.1.m1.1.1.2"><mtext id="S3.SS6.p1.1.m1.1.1.2.cmml" xref="S3.SS6.p1.1.m1.1.1.2">f1-score</mtext></ci><cn type="float" id="S3.SS6.p1.1.m1.1.1.3.cmml" xref="S3.SS6.p1.1.m1.1.1.3">32.5</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS6.p1.1.m1.1c">\text{f1-score}=32.5</annotation></semantics></math>), all the trained models demonstrated improved performance (see Table <a href="#S3.T2" title="Table 2 ‚Ä£ 3.6 Training on synthetic data improved performance over noisy raw data ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>). The ranking of the models and the impact of the synthetic training sets on the final performance align with the previous observations on the original data. BioGPT models persistently outperformed Seq2rel and GPT-2, and the training on <span id="S3.SS6.p1.6.4" class="ltx_text ltx_font_typewriter">Diversity-synt</span> resulted in an improved recall at the expense of precision compared to <span id="S3.SS6.p1.6.5" class="ltx_text ltx_font_typewriter">Random-synt</span>. However, the GPT-2 models trained on <span id="S3.SS6.p1.6.6" class="ltx_text ltx_font_typewriter">Random-synt</span> on average outperformed the one trained on <span id="S3.SS6.p1.6.7" class="ltx_text ltx_font_typewriter">Diversity-synt</span>, a departure from the trend observed with Seq2rel and BioGPT. Again, the best performance is achieved by BioGPT trained on the merged set, with <math id="S3.SS6.p1.2.m2.1" class="ltx_Math" alttext="\text{f1-score}=53.8" display="inline"><semantics id="S3.SS6.p1.2.m2.1a"><mrow id="S3.SS6.p1.2.m2.1.1" xref="S3.SS6.p1.2.m2.1.1.cmml"><mtext id="S3.SS6.p1.2.m2.1.1.2" xref="S3.SS6.p1.2.m2.1.1.2a.cmml">f1-score</mtext><mo id="S3.SS6.p1.2.m2.1.1.1" xref="S3.SS6.p1.2.m2.1.1.1.cmml">=</mo><mn id="S3.SS6.p1.2.m2.1.1.3" xref="S3.SS6.p1.2.m2.1.1.3.cmml">53.8</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS6.p1.2.m2.1b"><apply id="S3.SS6.p1.2.m2.1.1.cmml" xref="S3.SS6.p1.2.m2.1.1"><eq id="S3.SS6.p1.2.m2.1.1.1.cmml" xref="S3.SS6.p1.2.m2.1.1.1"></eq><ci id="S3.SS6.p1.2.m2.1.1.2a.cmml" xref="S3.SS6.p1.2.m2.1.1.2"><mtext id="S3.SS6.p1.2.m2.1.1.2.cmml" xref="S3.SS6.p1.2.m2.1.1.2">f1-score</mtext></ci><cn type="float" id="S3.SS6.p1.2.m2.1.1.3.cmml" xref="S3.SS6.p1.2.m2.1.1.3">53.8</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS6.p1.2.m2.1c">\text{f1-score}=53.8</annotation></semantics></math>.
<br class="ltx_break">Finally, two BioGPT-Large models were trained on the <span id="S3.SS6.p1.6.8" class="ltx_text ltx_font_typewriter">Diversity-synt</span> and <span id="S3.SS6.p1.6.9" class="ltx_text ltx_font_typewriter">Extended-synt</span> (see Table <a href="#S3.T3" title="Table 3 ‚Ä£ 3.6 Training on synthetic data improved performance over noisy raw data ‚Ä£ 3 Empirical experiments ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>). The model trained on <span id="S3.SS6.p1.6.10" class="ltx_text ltx_font_typewriter">Diversity-synt</span> achieved <math id="S3.SS6.p1.3.m3.1" class="ltx_Math" alttext="\text{f1-score}=57.2" display="inline"><semantics id="S3.SS6.p1.3.m3.1a"><mrow id="S3.SS6.p1.3.m3.1.1" xref="S3.SS6.p1.3.m3.1.1.cmml"><mtext id="S3.SS6.p1.3.m3.1.1.2" xref="S3.SS6.p1.3.m3.1.1.2a.cmml">f1-score</mtext><mo id="S3.SS6.p1.3.m3.1.1.1" xref="S3.SS6.p1.3.m3.1.1.1.cmml">=</mo><mn id="S3.SS6.p1.3.m3.1.1.3" xref="S3.SS6.p1.3.m3.1.1.3.cmml">57.2</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.SS6.p1.3.m3.1b"><apply id="S3.SS6.p1.3.m3.1.1.cmml" xref="S3.SS6.p1.3.m3.1.1"><eq id="S3.SS6.p1.3.m3.1.1.1.cmml" xref="S3.SS6.p1.3.m3.1.1.1"></eq><ci id="S3.SS6.p1.3.m3.1.1.2a.cmml" xref="S3.SS6.p1.3.m3.1.1.2"><mtext id="S3.SS6.p1.3.m3.1.1.2.cmml" xref="S3.SS6.p1.3.m3.1.1.2">f1-score</mtext></ci><cn type="float" id="S3.SS6.p1.3.m3.1.1.3.cmml" xref="S3.SS6.p1.3.m3.1.1.3">57.2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS6.p1.3.m3.1c">\text{f1-score}=57.2</annotation></semantics></math>, comparable to the new best model trained on the much larger merged set (<math id="S3.SS6.p1.4.m4.1" class="ltx_math_unparsed" alttext="\text{f1-score}==59.0" display="inline"><semantics id="S3.SS6.p1.4.m4.1a"><mrow id="S3.SS6.p1.4.m4.1b"><mtext id="S3.SS6.p1.4.m4.1.1">f1-score</mtext><mo rspace="0em" id="S3.SS6.p1.4.m4.1.2">=</mo><mo lspace="0em" id="S3.SS6.p1.4.m4.1.3">=</mo><mn id="S3.SS6.p1.4.m4.1.4">59.0</mn></mrow><annotation encoding="application/x-tex" id="S3.SS6.p1.4.m4.1c">\text{f1-score}==59.0</annotation></semantics></math>) and also demonstrated a better recall (<math id="S3.SS6.p1.5.m5.1" class="ltx_Math" alttext="56.90" display="inline"><semantics id="S3.SS6.p1.5.m5.1a"><mn id="S3.SS6.p1.5.m5.1.1" xref="S3.SS6.p1.5.m5.1.1.cmml">56.90</mn><annotation-xml encoding="MathML-Content" id="S3.SS6.p1.5.m5.1b"><cn type="float" id="S3.SS6.p1.5.m5.1.1.cmml" xref="S3.SS6.p1.5.m5.1.1">56.90</cn></annotation-xml><annotation encoding="application/x-tex" id="S3.SS6.p1.5.m5.1c">56.90</annotation></semantics></math> against <math id="S3.SS6.p1.6.m6.1" class="ltx_Math" alttext="51.6" display="inline"><semantics id="S3.SS6.p1.6.m6.1a"><mn id="S3.SS6.p1.6.m6.1.1" xref="S3.SS6.p1.6.m6.1.1.cmml">51.6</mn><annotation-xml encoding="MathML-Content" id="S3.SS6.p1.6.m6.1b"><cn type="float" id="S3.SS6.p1.6.m6.1.1.cmml" xref="S3.SS6.p1.6.m6.1.1">51.6</cn></annotation-xml><annotation encoding="application/x-tex" id="S3.SS6.p1.6.m6.1c">51.6</annotation></semantics></math>).</p>
</div>
<figure id="S3.T2" class="ltx_table">
<table id="S3.T2.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S3.T2.1.1" class="ltx_tr">
<td id="S3.T2.1.1.1" class="ltx_td ltx_align_left ltx_border_tt"><span id="S3.T2.1.1.1.1" class="ltx_text ltx_font_bold">model</span></td>
<td id="S3.T2.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T2.1.1.2.1" class="ltx_text ltx_font_bold">Dataset</span></td>
<td id="S3.T2.1.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T2.1.1.3.1" class="ltx_text ltx_font_bold">precision</span></td>
<td id="S3.T2.1.1.4" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T2.1.1.4.1" class="ltx_text ltx_font_bold">recall</span></td>
<td id="S3.T2.1.1.5" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T2.1.1.5.1" class="ltx_text ltx_font_bold">f1</span></td>
<td id="S3.T2.1.1.6" class="ltx_td ltx_border_tt"></td>
</tr>
<tr id="S3.T2.1.2" class="ltx_tr">
<td id="S3.T2.1.2.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T2.1.2.1.1" class="ltx_text ltx_font_bold">Seq2rel</span></td>
<td id="S3.T2.1.2.2" class="ltx_td ltx_align_right ltx_border_t"><span id="S3.T2.1.2.2.1" class="ltx_text ltx_font_typewriter">Random-synt</span></td>
<td id="S3.T2.1.2.3" class="ltx_td ltx_align_right ltx_border_t"><span id="S3.T2.1.2.3.1" class="ltx_text ltx_framed ltx_framed_underline">62.4 +/- (1.03)</span></td>
<td id="S3.T2.1.2.4" class="ltx_td ltx_align_right ltx_border_t">26.8 +/- (1.96)</td>
<td id="S3.T2.1.2.5" class="ltx_td ltx_align_right ltx_border_t">37.5 +/- (1.90)</td>
<td id="S3.T2.1.2.6" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T2.1.3" class="ltx_tr">
<td id="S3.T2.1.3.1" class="ltx_td ltx_align_right"><span id="S3.T2.1.3.1.1" class="ltx_text ltx_font_typewriter">Diversty-synt</span></td>
<td id="S3.T2.1.3.2" class="ltx_td ltx_align_right">61.5</td>
<td id="S3.T2.1.3.3" class="ltx_td ltx_align_right">30.7</td>
<td id="S3.T2.1.3.4" class="ltx_td ltx_align_right">40.1</td>
<td id="S3.T2.1.3.5" class="ltx_td"></td>
</tr>
<tr id="S3.T2.1.4" class="ltx_tr">
<td id="S3.T2.1.4.1" class="ltx_td ltx_align_right"><span id="S3.T2.1.4.1.1" class="ltx_text ltx_font_typewriter">Extended-synt</span></td>
<td id="S3.T2.1.4.2" class="ltx_td ltx_align_right"><span id="S3.T2.1.4.2.1" class="ltx_text ltx_font_bold">65.1</span></td>
<td id="S3.T2.1.4.3" class="ltx_td ltx_align_right">29.9</td>
<td id="S3.T2.1.4.4" class="ltx_td ltx_align_right">41.0</td>
<td id="S3.T2.1.4.5" class="ltx_td"></td>
</tr>
<tr id="S3.T2.1.5" class="ltx_tr">
<td id="S3.T2.1.5.1" class="ltx_td ltx_align_left" rowspan="3"><span id="S3.T2.1.5.1.1" class="ltx_text ltx_font_bold">GPT-2</span></td>
<td id="S3.T2.1.5.2" class="ltx_td ltx_align_right"><span id="S3.T2.1.5.2.1" class="ltx_text ltx_font_typewriter">Random-synt</span></td>
<td id="S3.T2.1.5.3" class="ltx_td ltx_align_right">42.6 +/- (2.89)</td>
<td id="S3.T2.1.5.4" class="ltx_td ltx_align_right">32.7 +/- (2.81)</td>
<td id="S3.T2.1.5.5" class="ltx_td ltx_align_right">37.2 +/- (2.80)</td>
<td id="S3.T2.1.5.6" class="ltx_td"></td>
</tr>
<tr id="S3.T2.1.6" class="ltx_tr">
<td id="S3.T2.1.6.1" class="ltx_td ltx_align_right"><span id="S3.T2.1.6.1.1" class="ltx_text ltx_font_typewriter">Diversty-synt</span></td>
<td id="S3.T2.1.6.2" class="ltx_td ltx_align_right">28.5</td>
<td id="S3.T2.1.6.3" class="ltx_td ltx_align_right">39.4</td>
<td id="S3.T2.1.6.4" class="ltx_td ltx_align_right">33.0</td>
<td id="S3.T2.1.6.5" class="ltx_td"></td>
</tr>
<tr id="S3.T2.1.7" class="ltx_tr">
<td id="S3.T2.1.7.1" class="ltx_td ltx_align_right"><span id="S3.T2.1.7.1.1" class="ltx_text ltx_font_typewriter">Extended-synt</span></td>
<td id="S3.T2.1.7.2" class="ltx_td ltx_align_right">52.0</td>
<td id="S3.T2.1.7.3" class="ltx_td ltx_align_right">44.6</td>
<td id="S3.T2.1.7.4" class="ltx_td ltx_align_right">48.0</td>
<td id="S3.T2.1.7.5" class="ltx_td"></td>
</tr>
<tr id="S3.T2.1.8" class="ltx_tr">
<td id="S3.T2.1.8.1" class="ltx_td ltx_align_left ltx_border_bb" rowspan="3"><span id="S3.T2.1.8.1.1" class="ltx_text ltx_font_bold">BioGPT</span></td>
<td id="S3.T2.1.8.2" class="ltx_td ltx_align_right"><span id="S3.T2.1.8.2.1" class="ltx_text ltx_font_typewriter">Random-synt</span></td>
<td id="S3.T2.1.8.3" class="ltx_td ltx_align_right">56.4 +/- (2.26)</td>
<td id="S3.T2.1.8.4" class="ltx_td ltx_align_right">38.8 +/- (1.92)</td>
<td id="S3.T2.1.8.5" class="ltx_td ltx_align_right">46.0 +/- 1.08</td>
<td id="S3.T2.1.8.6" class="ltx_td"></td>
</tr>
<tr id="S3.T2.1.9" class="ltx_tr">
<td id="S3.T2.1.9.1" class="ltx_td ltx_align_right"><span id="S3.T2.1.9.1.1" class="ltx_text ltx_font_typewriter">Diversty-synt</span></td>
<td id="S3.T2.1.9.2" class="ltx_td ltx_align_right">52.5</td>
<td id="S3.T2.1.9.3" class="ltx_td ltx_align_right"><span id="S3.T2.1.9.3.1" class="ltx_text ltx_framed ltx_framed_underline">41.2</span></td>
<td id="S3.T2.1.9.4" class="ltx_td ltx_align_right"><span id="S3.T2.1.9.4.1" class="ltx_text ltx_framed ltx_framed_underline">46.2</span></td>
<td id="S3.T2.1.9.5" class="ltx_td"></td>
</tr>
<tr id="S3.T2.1.10" class="ltx_tr">
<td id="S3.T2.1.10.1" class="ltx_td ltx_align_right ltx_border_bb"><span id="S3.T2.1.10.1.1" class="ltx_text ltx_font_typewriter">Extended-synt</span></td>
<td id="S3.T2.1.10.2" class="ltx_td ltx_align_right ltx_border_bb">63.7</td>
<td id="S3.T2.1.10.3" class="ltx_td ltx_align_right ltx_border_bb"><span id="S3.T2.1.10.3.1" class="ltx_text ltx_font_bold">46.5</span></td>
<td id="S3.T2.1.10.4" class="ltx_td ltx_align_right ltx_border_bb"><span id="S3.T2.1.10.4.1" class="ltx_text ltx_font_bold">53.8</span></td>
<td id="S3.T2.1.10.5" class="ltx_td ltx_border_bb"></td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 2: </span>Performance of Seq2rel, GPT-2 and BioGPT models fine-tuned on synthetic data. Three types of training dataset are evaluated: the diversity sample (<span id="S3.T2.6.1" class="ltx_text ltx_font_typewriter">Diversty-synt</span>), 5 random samples (<span id="S3.T2.7.2" class="ltx_text ltx_font_typewriter">Random-synt</span>) and the extended sample (<span id="S3.T2.8.3" class="ltx_text ltx_font_typewriter">Extended-synt</span>), which is the union of the previous samples, all synthetically generated from the corresponding seed original samples. For <span id="S3.T2.9.4" class="ltx_text ltx_font_italic">Random-synt</span> samples, results are averaged and standard deviations are reported. Best performance are bold, and second best performance are underlined.</figcaption>
</figure>
<figure id="S3.T3" class="ltx_table">
<table id="S3.T3.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S3.T3.1.1" class="ltx_tr">
<td id="S3.T3.1.1.1" class="ltx_td ltx_align_left ltx_border_tt"><span id="S3.T3.1.1.1.1" class="ltx_text ltx_font_bold">model</span></td>
<td id="S3.T3.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T3.1.1.2.1" class="ltx_text ltx_font_bold">Dataset</span></td>
<td id="S3.T3.1.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T3.1.1.3.1" class="ltx_text ltx_font_bold">precision</span></td>
<td id="S3.T3.1.1.4" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T3.1.1.4.1" class="ltx_text ltx_font_bold">recall</span></td>
<td id="S3.T3.1.1.5" class="ltx_td ltx_align_right ltx_border_tt"><span id="S3.T3.1.1.5.1" class="ltx_text ltx_font_bold">f1</span></td>
<td id="S3.T3.1.1.6" class="ltx_td ltx_border_tt"></td>
</tr>
<tr id="S3.T3.1.2" class="ltx_tr">
<td id="S3.T3.1.2.1" class="ltx_td ltx_align_left ltx_border_t">BioGPT-Large</td>
<td id="S3.T3.1.2.2" class="ltx_td ltx_align_right ltx_border_t"><span id="S3.T3.1.2.2.1" class="ltx_text ltx_font_typewriter">Diversty-synt</span></td>
<td id="S3.T3.1.2.3" class="ltx_td ltx_align_right ltx_border_t">57.50</td>
<td id="S3.T3.1.2.4" class="ltx_td ltx_align_right ltx_border_t"><span id="S3.T3.1.2.4.1" class="ltx_text ltx_font_bold">56.90</span></td>
<td id="S3.T3.1.2.5" class="ltx_td ltx_align_right ltx_border_t">57.20</td>
<td id="S3.T3.1.2.6" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T3.1.3" class="ltx_tr">
<td id="S3.T3.1.3.1" class="ltx_td ltx_align_left ltx_border_bb">BioGPT-Large</td>
<td id="S3.T3.1.3.2" class="ltx_td ltx_align_right ltx_border_bb"><span id="S3.T3.1.3.2.1" class="ltx_text ltx_font_typewriter">Extended-synt</span></td>
<td id="S3.T3.1.3.3" class="ltx_td ltx_align_right ltx_border_bb"><span id="S3.T3.1.3.3.1" class="ltx_text ltx_font_bold">69.0</span></td>
<td id="S3.T3.1.3.4" class="ltx_td ltx_align_right ltx_border_bb">51.6</td>
<td id="S3.T3.1.3.5" class="ltx_td ltx_align_right ltx_border_bb"><span id="S3.T3.1.3.5.1" class="ltx_text ltx_font_bold">59.0</span></td>
<td id="S3.T3.1.3.6" class="ltx_td ltx_border_bb"></td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 3: </span>Performance of finetuned bioGPT-Large models on the synthetic diversity sample and the Extended synthetic sample.</figcaption>
</figure>
</section>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Discussion</h2>

<div id="S4.p1" class="ltx_para ltx_noindent">
<p id="S4.p1.1" class="ltx_p">The application of deep learning models for the completion of biomedical knowledge bases is largely limited by the availability and quality of domain-specific labelled data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite>. Therefore, we adopted a data-centric methodology <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib94" title="" class="ltx_ref">94</a>, <a href="#bib.bib95" title="" class="ltx_ref">95</a>]</cite>. In order to address the data imbalance and optimised the manual curation process, we proposed the GME-sampler inspired by diversity metrics commonly used in ecology. The sampler was applied on the pre-processed LOTUS dataset (separately on each biological kingdom) to extract a subset of documents, ensuring a diverse set of organisms and chemicals in the reported relations. The compositional analysis revealed a higher number of distinct entities in the extracted sample, but also a better balance considering the fixed number of items. Diversity has been recognized as an important factor in training set for representation learning and improving the generalization performance of models <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib96" title="" class="ltx_ref">96</a>, <a href="#bib.bib26" title="" class="ltx_ref">26</a>]</cite>, essential for the NER sub-task. By forcing diversity into the relation partners (organisms and chemicals), we also expect it to be improved in their mentioning contexts. Considering the time and domain expertise requirements to annotate an evaluation dataset, the diversity metric was also used for partitioning. We extracted and manually annotated a representative subset by extracting the 200 top-diverse items. We hope that this manually-curated evaluation dataset will help the community to build upon this work.
<br class="ltx_break">Despite a smaller number of trained parameters, BioGPT and GPT-2 fine-tuned with QLoRa clearly outperformed Seq2rel (see supplementary <a href="#S1.SS2" title="S1.2 Fine-tuning details ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S1.2</span></a>). This highlighted the benefit of the larger pre-training, but also the effectiveness of the QLoRA strategy, where low-rank updates of a large, but quantized model, achieve better performance than the full fine-tuning of a smaller model, for a lower parameter budget <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib97" title="" class="ltx_ref">97</a>, <a href="#bib.bib98" title="" class="ltx_ref">98</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite>. While based on the same architecture, improvements of BioGPT over GPT-2 can be attributed both to the pre-training on PubMed and also to the dedicated tokenizer (see supplementary figure <a href="#S2.F10" title="Figure S10 ‚Ä£ S2.8 Tokenized length of abstracts ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S10</span></a>). Beyond the architectures of the models, the training dataset also had a significant impact on the performance. A comparison between models trained on the largest (<span id="S4.p1.1.1" class="ltx_text ltx_font_typewriter">Extended-raw</span>) and the diversity-optimized dataset revealed that the latter achieved competitive results despite its smaller size. Additionally, results also suggest that improving the diversity of the provided set of examples for training can improve the recall of the model, at the expense of precision. Increasing the number of training examples with noisy data also have limited benefits, as suggested by the comparison with the training dataset extended to all available data (no sampling, no stratification) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib99" title="" class="ltx_ref">99</a>, <a href="#bib.bib100" title="" class="ltx_ref">100</a>, <a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite>. Finally, few-shot learning techniques leveraging open LLMs exhibit reasonable performance (see LLaMA-65B) and can be particularly valuable when only limited or noisy data are available. However, their larger size may incur higher management costs, necessitating careful consideration of resource allocation.
<br class="ltx_break">Instead of using them to directly perform the task, we then propose to use them to generate synthetic examples and alleviate the noise of the dataset. However, evaluating the quality of the generated abstracts is challenging. Although the process is prone to hallucinations, the factuality is not the key criteria, as long as the generated texts are credible, meaning that they are coherent and adhere to the established syntax, style and patterns, of expression of the relations in human-written abstracts. Since the training sets of LLMs contain scientific articles and abstracts, they have absorbed their stylistic and syntactic specificities. The generation of synthetic data could then be seen as a form of knowledge distillation. Moreover, while previous studies have suggested that LLMs may not be knowledgeable <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib101" title="" class="ltx_ref">101</a>, <a href="#bib.bib102" title="" class="ltx_ref">102</a>, <a href="#bib.bib103" title="" class="ltx_ref">103</a>]</cite>, other investigations have highlighted the remarkable capabilities of chatbot and instructions-tuned models in following style instructions <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib104" title="" class="ltx_ref">104</a>, <a href="#bib.bib105" title="" class="ltx_ref">105</a>]</cite>. Then, a relevant evaluation criteria for these synthetic data is the improvement on the performance they provided.
<br class="ltx_break">The study revealed that the trained model exhibited a substantial increase in performance when transitioning from raw noisy data to synthetic data. This shift in data sources did not alter the previously observed trend: BioGPT outperformed other models, and the diversity-optimised sampling had a positive effect on the recall of trained models when used to select the seed articles. Most importantly, we noticed that the transition from original to synthetic data had a more determinant impact on the performance improvements than the choice of the model architecture (Seq2rel, GPT-2, BioGPT). For instance, the influence of synthetic data on the performance of BioGPT and GPT-2 is greater than the difference between the two fine-tuned models. The performance of Seq2rel was also enhanced almost by a factor of 4, notably narrowing the gap with GPT models. Similarly, scaling-up the architecture with Bio-Large (<math id="S4.p1.1.m1.1" class="ltx_math_unparsed" alttext="&gt;4.5\times" display="inline"><semantics id="S4.p1.1.m1.1a"><mrow id="S4.p1.1.m1.1b"><mo id="S4.p1.1.m1.1.1">&gt;</mo><mn id="S4.p1.1.m1.1.2">4.5</mn><mo lspace="0.222em" id="S4.p1.1.m1.1.3">√ó</mo></mrow><annotation encoding="application/x-tex" id="S4.p1.1.m1.1c">&gt;4.5\times</annotation></semantics></math> larger) indeed resulted in improved performance, but comparable to the previous enhancement obtained with synthetic data. Also, we noticed a clear impact of the training dataset, with the best observed recall achieved with <span id="S4.p1.1.2" class="ltx_text ltx_font_typewriter">Diversity-synt</span>. These results also support the data-centric view, even in low-resource scenarios, by demonstrating improved performance over other strategies such as few-shot learning <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib106" title="" class="ltx_ref">106</a>]</cite>.</p>
</div>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Limits and future work</h2>

<div id="S5.p1" class="ltx_para ltx_noindent">
<p id="S5.p1.1" class="ltx_p">Fine-tuned methods exhibit superior performance compared to zero-shot/few-shot approaches. However, the basic prompting approach used in the experiments may not fully demonstrate the capabilities of the models, and alternative strategies have been proposed <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib107" title="" class="ltx_ref">107</a>, <a href="#bib.bib108" title="" class="ltx_ref">108</a>, <a href="#bib.bib87" title="" class="ltx_ref">87</a>]</cite>. Nevertheless, <cite class="ltx_cite ltx_citemacro_citet">Guti√©rrez et¬†al. [<a href="#bib.bib109" title="" class="ltx_ref">109</a>]</cite> noted that even with these improvements, the models still lack the accuracy of fine-tuned approaches with qualitative data. The use of LLMs to generate abstracts also has some evident limitations. The generated abstracts exhibit a narrow range of styles to express the relationships between organisms and chemicals compared to human-written abstracts. We suppose that the synthetic data mostly improved the recognition of organism and chemical entities, this subtask being inherently embedded in the ultimate task of decoding the relationships.
<br class="ltx_break">Although the proposed framework is effective, it cannot guarantee the true diversity of the generated abstracts and the final selection may be very similar. Secondly, the selector module does not ensure that the relations are semantically expressed in the generated abstracts, as it only checks for the explicit mention of the entities. Finally, all generated examples are designed as "positive" cases, meaning that a relation is always expected, which may not be the case in practical applications. The developed models are intended for use on a large corpus of articles and the input documents can be either selected by an upstream retriever component, or, the predictions can be re-evaluated by a downstream selector. Continuing with this data-centric view, future works will prioritize improving the three key components (instructions builder, generator, and selector) to improve the diversity of the synthetic abstracts, rather than focusing on the architecture of the trained models.
<br class="ltx_break">Given the highly dynamic nature of the LLMs research area, we anticipate significant advancements in model architecture and accessibility to arise from the research community. At the date of writing, the release of Llama (and Llama2) has paved the way for the creation of more open-license models, such as the next-generation of Vicuna<span id="footnote10" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">10</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">10</sup><span class="ltx_tag ltx_tag_note">10</span><a target="_blank" href="https://huggingface.co/lmsys/vicuna-13b-v1.5" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://huggingface.co/lmsys/vicuna-13b-v1.5</a></span></span></span> or PMC-Llama trained on the biomedical literature. The development of multilingual open LLMs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib110" title="" class="ltx_ref">110</a>]</cite> also offers opportunities for synthetic data generation in promising areas, such as the extraction of plant-disease relationships from Traditional Chinese Medicine prescriptions, where the scarcity of labelled data is limiting <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib111" title="" class="ltx_ref">111</a>]</cite>.</p>
</div>
</section>
<section id="S6" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Conclusion</h2>

<div id="S6.p1" class="ltx_para ltx_noindent">
<p id="S6.p1.1" class="ltx_p">With the aim of assisting the completion of NP databases, we provide the first training and evaluation datasets along with the first trained models for end-to-end RE of relationships between organisms and chemicals. Along with these main results, we explored different strategies and proposed new developments to address the problematics raised in this biomedical context. We empirically showed the benefit of the proposed GME-sampler for building a diverse and balanced evaluation dataset, as well as its positive impact on the recall via the training data. The results also indicate that the opportunities brought by the open LLMs in scenarios with few data or weakly labelled, may not lie only in their zero/few-shot learning abilities, but also in their great potential as synthetic data generator. They could open the door for the extraction of previously unexplored relationships between biomedical entities expressed in the literature, a prerequisite to unlock new paths of inferences in knowledge discovery.</p>
</div>
</section>
<section id="Sx2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Funding</h2>

<div id="Sx2.p1" class="ltx_para ltx_noindent">
<p id="Sx2.p1.1" class="ltx_p">This work was supported by the IDIAP Research Institute and has been done in collaboration with the company Inflamalps SA and is supported by the Ark Foundation. This project has received funding from the European Union‚Äôs Horizon 2020 research and innovation programme under grant agreement No 965397. The funding bodies played no role in the design of the study, research, writing and publication of the paper.</p>
</div>
</section>
<section id="Sx3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Acknowledgements</h2>

<div id="Sx3.p1" class="ltx_para ltx_noindent">
<p id="Sx3.p1.1" class="ltx_p">The authors are thankful to Vincent Mutel, Jo√´l Dumoulin, Joel Rossier and Colombine Verzat for their help during the project. We are grateful to Olena Hrynenko for proofreading the mathematical formulations. We are also grateful to the authors behind LOTUS, BioGPT and Seq2rel for sharing their data or code.</p>
</div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sorokina et¬†al. [2021]</span>
<span class="ltx_bibblock">
Maria Sorokina, Peter Merseburger, Kohulan Rajan, Mehmet¬†Aziz Yirik, and Christoph Steinbeck.

</span>
<span class="ltx_bibblock">COCONUT online: collection of open natural products database.

</span>
<span class="ltx_bibblock"><em id="bib.bib1.1.1" class="ltx_emph ltx_font_italic">Journal of Cheminformatics</em>, 13(1):1‚Äì13, 2021.

</span>
<span class="ltx_bibblock">Publisher: BioMed Central.

</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Shinbo et¬†al. [2006]</span>
<span class="ltx_bibblock">
Y¬†Shinbo, Y¬†Nakamura, Md¬†Altaf-Ul-Amin, H¬†Asahi, K¬†Kurokawa, M¬†Arita, K¬†Saito, D¬†Ohta, D¬†Shibata, and S¬†Kanaya.

</span>
<span class="ltx_bibblock">KNApSAcK: a comprehensive species-metabolite relationship database.

</span>
<span class="ltx_bibblock"><em id="bib.bib2.1.1" class="ltx_emph ltx_font_italic">Plant metabolomics</em>, pages 165‚Äì181, 2006.

</span>
<span class="ltx_bibblock">Publisher: Springer.

</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Rutz et¬†al. [2022]</span>
<span class="ltx_bibblock">
Adriano Rutz, Maria Sorokina, Jakub Galgonek, Daniel Mietchen, Egon Willighagen, Arnaud Gaudry, James¬†G Graham, Ralf Stephan, Roderic Page, Ji≈ô√≠ Vondr√°≈°ek, Christoph Steinbeck, Guido¬†F Pauli, Jean-Luc Wolfender, Jonathan Bisson, and Pierre-Marie Allard.

</span>
<span class="ltx_bibblock">The LOTUS initiative for open knowledge management in natural products research.

</span>
<span class="ltx_bibblock"><em id="bib.bib3.1.1" class="ltx_emph ltx_font_italic">eLife</em>, 11:e70780, May 2022.

</span>
<span class="ltx_bibblock">ISSN 2050-084X.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.7554/eLife.70780" title="" class="ltx_ref ltx_href">10.7554/eLife.70780</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.7554/eLife.70780" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.7554/eLife.70780</a>.

</span>
<span class="ltx_bibblock">Publisher: eLife Sciences Publications, Ltd.

</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Luo et¬†al. [2022a]</span>
<span class="ltx_bibblock">
Renqian Luo, Liai Sun, Yingce Xia, Tao Qin, Sheng Zhang, Hoifung Poon, and Tie-Yan Liu.

</span>
<span class="ltx_bibblock">BioGPT: generative pre-trained transformer for biomedical text generation and mining.

</span>
<span class="ltx_bibblock"><em id="bib.bib4.1.1" class="ltx_emph ltx_font_italic">Briefings in Bioinformatics</em>, 23(6):bbac409, November 2022a.

</span>
<span class="ltx_bibblock">ISSN 1477-4054.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1093/bib/bbac409" title="" class="ltx_ref ltx_href">10.1093/bib/bbac409</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1093/bib/bbac409" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1093/bib/bbac409</a>.

</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Giorgi et¬†al. [2022]</span>
<span class="ltx_bibblock">
John Giorgi, Gary¬†D. Bader, and Bo¬†Wang.

</span>
<span class="ltx_bibblock">A sequence-to-sequence approach for document-level relation extraction, April 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2204.01098" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2204.01098</a>.

</span>
<span class="ltx_bibblock">arXiv:2204.01098 [cs].

</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wang et¬†al. [2020]</span>
<span class="ltx_bibblock">
Difeng Wang, Wei Hu, Ermei Cao, and Weijian Sun.

</span>
<span class="ltx_bibblock">Global-to-Local Neural Networks for Document-Level Relation Extraction.

</span>
<span class="ltx_bibblock">In <em id="bib.bib6.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP)</em>, pages 3711‚Äì3721, Online, November 2020. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2020.emnlp-main.303" title="" class="ltx_ref ltx_href">10.18653/v1/2020.emnlp-main.303</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2020.emnlp-main.303" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2020.emnlp-main.303</a>.

</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gerner et¬†al. [2010]</span>
<span class="ltx_bibblock">
Martin Gerner, Goran Nenadic, and Casey¬†M. Bergman.

</span>
<span class="ltx_bibblock">LINNAEUS: A species name identification system for biomedical literature.

</span>
<span class="ltx_bibblock"><em id="bib.bib7.1.1" class="ltx_emph ltx_font_italic">BMC Bioinformatics</em>, 11(1):85, February 2010.

</span>
<span class="ltx_bibblock">ISSN 1471-2105.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1186/1471-2105-11-85" title="" class="ltx_ref ltx_href">10.1186/1471-2105-11-85</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1186/1471-2105-11-85" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1186/1471-2105-11-85</a>.

</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Krallinger et¬†al. [2015]</span>
<span class="ltx_bibblock">
Martin Krallinger, Obdulia Rabal, Florian Leitner, Miguel Vazquez, David Salgado, Zhiyong Lu, Robert Leaman, Yanan Lu, Donghong Ji, Daniel¬†M. Lowe, Roger¬†A. Sayle, Riza¬†Theresa Batista-Navarro, Rafal Rak, Torsten Huber, Tim Rockt√§schel, S√©rgio Matos, David Campos, Buzhou Tang, Hua Xu, Tsendsuren Munkhdalai, Keun¬†Ho Ryu, SV¬†Ramanan, Senthil Nathan, Slavko ≈Ωitnik, Marko Bajec, Lutz Weber, Matthias Irmer, Saber¬†A. Akhondi, Jan¬†A. Kors, Shuo Xu, Xin An, Utpal¬†Kumar Sikdar, Asif Ekbal, Masaharu Yoshioka, Thaer¬†M. Dieb, Miji Choi, Karin Verspoor, Madian Khabsa, C.¬†Lee Giles, Hongfang Liu, Komandur¬†Elayavilli Ravikumar, Andre Lamurias, Francisco¬†M. Couto, Hong-Jie Dai, Richard Tzong-Han Tsai, Caglar Ata, Tolga Can, Anabel Usi√©, Rui Alves, Isabel Segura-Bedmar, Paloma Mart√≠nez, Julen Oyarzabal, and Alfonso Valencia.

</span>
<span class="ltx_bibblock">The CHEMDNER corpus of chemicals and drugs and its annotation principles.

</span>
<span class="ltx_bibblock"><em id="bib.bib8.1.1" class="ltx_emph ltx_font_italic">Journal of Cheminformatics</em>, 7(1):S2, January 2015.

</span>
<span class="ltx_bibblock">ISSN 1758-2946.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1186/1758-2946-7-S1-S2" title="" class="ltx_ref ltx_href">10.1186/1758-2946-7-S1-S2</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1186/1758-2946-7-S1-S2" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1186/1758-2946-7-S1-S2</a>.

</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Luo et¬†al. [2022b]</span>
<span class="ltx_bibblock">
Ling Luo, Po-Ting Lai, Chih-Hsuan Wei, Cecilia¬†N Arighi, and Zhiyong Lu.

</span>
<span class="ltx_bibblock">BioRED: a rich biomedical relation extraction dataset.

</span>
<span class="ltx_bibblock"><em id="bib.bib9.1.1" class="ltx_emph ltx_font_italic">Briefings in Bioinformatics</em>, 23(5):bbac282, September 2022b.

</span>
<span class="ltx_bibblock">ISSN 1477-4054.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1093/bib/bbac282" title="" class="ltx_ref ltx_href">10.1093/bib/bbac282</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1093/bib/bbac282" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1093/bib/bbac282</a>.

</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wysocki et¬†al. [2023]</span>
<span class="ltx_bibblock">
Oskar Wysocki, Zili Zhou, Paul O‚ÄôRegan, Deborah Ferreira, Magdalena Wysocka, D√≥nal Landers, and Andr√© Freitas.

</span>
<span class="ltx_bibblock">Transformers and the representation of biomedical background knowledge.

</span>
<span class="ltx_bibblock"><em id="bib.bib10.1.1" class="ltx_emph ltx_font_italic">Computational Linguistics</em>, 49(1):73‚Äì115, March 2023.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1162/coli_a_00462" title="" class="ltx_ref ltx_href">10.1162/coli_a_00462</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2023.cl-1.2" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2023.cl-1.2</a>.

</span>
</li>
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kojima et¬†al. [2023]</span>
<span class="ltx_bibblock">
Takeshi Kojima, Shixiang¬†Shane Gu, Machel Reid, Yutaka Matsuo, and Yusuke Iwasawa.

</span>
<span class="ltx_bibblock">Large Language Models are Zero-Shot Reasoners.

</span>
<span class="ltx_bibblock"><em id="bib.bib11.1.1" class="ltx_emph ltx_font_italic">Advances in neural information processing systems</em>, 2023.

</span>
</li>
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Brown et¬†al. [2020]</span>
<span class="ltx_bibblock">
Tom¬†B Brown, Benjamin Mann, Nick Ryder, Melanie Subbiah, Jared Kaplan, Prafulla Dhariwal, Arvind Neelakantan, Pranav Shyam, Girish Sastry, Amanda Askell, Sandhini Agarwal, Ariel Herbert-Voss, Gretchen Krueger, and Tom Henighan.

</span>
<span class="ltx_bibblock">Language Models are Few-Shot Learners.

</span>
<span class="ltx_bibblock"><em id="bib.bib12.1.1" class="ltx_emph ltx_font_italic">Advances in neural information processing systems</em>, 2020.

</span>
</li>
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhang et¬†al. [2023]</span>
<span class="ltx_bibblock">
Shengyu Zhang, Linfeng Dong, Xiaoya Li, Sen Zhang, Xiaofei Sun, Shuhe Wang, Jiwei Li, Runyi Hu, Tianwei Zhang, Fei Wu, and Guoyin Wang.

</span>
<span class="ltx_bibblock">Instruction Tuning for Large Language Models: A Survey, September 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2308.10792" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2308.10792</a>.

</span>
<span class="ltx_bibblock">arXiv:2308.10792 [cs].

</span>
</li>
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sambasivan et¬†al. [2021]</span>
<span class="ltx_bibblock">
Nithya Sambasivan, Shivani Kapania, Hannah Highfill, Diana Akrong, Praveen Paritosh, and Lora¬†M Aroyo.

</span>
<span class="ltx_bibblock">‚ÄúEveryone wants to do the model work, not the data work‚Äù: Data Cascades in High-Stakes AI.

</span>
<span class="ltx_bibblock">In <em id="bib.bib14.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 2021 CHI Conference on Human Factors in Computing Systems</em>, CHI ‚Äô21, pages 1‚Äì15, New York, NY, USA, May 2021. Association for Computing Machinery.

</span>
<span class="ltx_bibblock">ISBN 978-1-4503-8096-6.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1145/3411764.3445518" title="" class="ltx_ref ltx_href">10.1145/3411764.3445518</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1145/3411764.3445518" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1145/3411764.3445518</a>.

</span>
</li>
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Shahab [2017]</span>
<span class="ltx_bibblock">
Elham Shahab.

</span>
<span class="ltx_bibblock">A Short Survey of Biomedical Relation Extraction Techniques, July 2017.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/1707.05850" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/1707.05850</a>.

</span>
<span class="ltx_bibblock">arXiv:1707.05850 [cs].

</span>
</li>
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhao et¬†al. [2020]</span>
<span class="ltx_bibblock">
Sendong Zhao, Chang Su, Chang Su, Zhiyong Lu, Zhiyong Lu, Zhiyong Lu, and Fei Wang.

</span>
<span class="ltx_bibblock">Recent advances in biomedical literature mining.

</span>
<span class="ltx_bibblock"><em id="bib.bib16.1.1" class="ltx_emph ltx_font_italic">Briefings in Bioinformatics</em>, 22(3), May 2020.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1093/bib/bbaa057" title="" class="ltx_ref ltx_href">10.1093/bib/bbaa057</a>.

</span>
<span class="ltx_bibblock">MAG ID: 3028004046.

</span>
</li>
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhao et¬†al. [2023]</span>
<span class="ltx_bibblock">
Xiaoyan Zhao, Yang Deng, Min Yang, Lingzhi Wang, Rui Zhang, Hong Cheng, Wai Lam, Ying Shen, and Ruifeng Xu.

</span>
<span class="ltx_bibblock">A Comprehensive Survey on Deep Learning for Relation Extraction: Recent Advances and New Frontiers, June 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2306.02051" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2306.02051</a>.

</span>
<span class="ltx_bibblock">arXiv:2306.02051 [cs].

</span>
</li>
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhang et¬†al. [2020a]</span>
<span class="ltx_bibblock">
Tianlin Zhang, Jiaxu Leng, and Ying Liu.

</span>
<span class="ltx_bibblock">Deep learning for drug-drug interaction extraction from the literature: a review.

</span>
<span class="ltx_bibblock"><em id="bib.bib18.1.1" class="ltx_emph ltx_font_italic">Briefings in Bioinformatics</em>, 21(5):1609‚Äì1627, September 2020a.

</span>
<span class="ltx_bibblock">ISSN 1477-4054.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1093/bib/bbz087" title="" class="ltx_ref ltx_href">10.1093/bib/bbz087</a>.

</span>
</li>
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li et¬†al. [2016]</span>
<span class="ltx_bibblock">
Jiao Li, Yueping Sun, Robin¬†J. Johnson, Daniela Sciaky, Chih-Hsuan Wei, Robert Leaman, Allan¬†Peter Davis, Carolyn¬†J. Mattingly, Thomas¬†C. Wiegers, and Zhiyong Lu.

</span>
<span class="ltx_bibblock">BioCreative V CDR task corpus: a resource for chemical disease relation extraction.

</span>
<span class="ltx_bibblock"><em id="bib.bib19.1.1" class="ltx_emph ltx_font_italic">Database: The Journal of Biological Databases and Curation</em>, 2016:baw068, May 2016.

</span>
<span class="ltx_bibblock">ISSN 1758-0463.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1093/database/baw068" title="" class="ltx_ref ltx_href">10.1093/database/baw068</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4860626/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4860626/</a>.

</span>
</li>
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Su et¬†al. [2021]</span>
<span class="ltx_bibblock">
Junhao Su, Ye¬†Wu, Hing-Fung Ting, Tak-Wah Lam, and Ruibang Luo.

</span>
<span class="ltx_bibblock">RENET2: high-performance full-text gene‚Äìdisease relation extraction with iterative training data expansion.

</span>
<span class="ltx_bibblock"><em id="bib.bib20.1.1" class="ltx_emph ltx_font_italic">NAR Genomics and Bioinformatics</em>, 3(3):lqab062, June 2021.

</span>
<span class="ltx_bibblock">ISSN 2631-9268.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1093/nargab/lqab062" title="" class="ltx_ref ltx_href">10.1093/nargab/lqab062</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://academic.oup.com/nargab/article/doi/10.1093/nargab/lqab062/6315218" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://academic.oup.com/nargab/article/doi/10.1093/nargab/lqab062/6315218</a>.

</span>
</li>
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ahmed et¬†al. [2019]</span>
<span class="ltx_bibblock">
Mahtab Ahmed, Jumayel Islam, Muhammad¬†Rifayat Samee, and Robert¬†E. Mercer.

</span>
<span class="ltx_bibblock">Identifying Protein-Protein Interaction Using Tree LSTM and Structured Attention.

</span>
<span class="ltx_bibblock">In <em id="bib.bib21.1.1" class="ltx_emph ltx_font_italic">2019 IEEE 13th International Conference on Semantic Computing (ICSC)</em>, pages 224‚Äì231, January 2019.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1109/ICOSC.2019.8665584" title="" class="ltx_ref ltx_href">10.1109/ICOSC.2019.8665584</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://ieeexplore.ieee.org/document/8665584" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://ieeexplore.ieee.org/document/8665584</a>.

</span>
<span class="ltx_bibblock">ISSN: 2325-6516.

</span>
</li>
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Xu and Goodacre [2018]</span>
<span class="ltx_bibblock">
Yun Xu and Royston Goodacre.

</span>
<span class="ltx_bibblock">On Splitting Training and Validation Set: A Comparative Study of Cross-Validation, Bootstrap and Systematic Sampling for Estimating the Generalization Performance of Supervised Learning.

</span>
<span class="ltx_bibblock"><em id="bib.bib22.1.1" class="ltx_emph ltx_font_italic">Journal of Analysis and Testing</em>, 2(3):249‚Äì262, July 2018.

</span>
<span class="ltx_bibblock">ISSN 2509-4696.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1007/s41664-018-0068-2" title="" class="ltx_ref ltx_href">10.1007/s41664-018-0068-2</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1007/s41664-018-0068-2" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1007/s41664-018-0068-2</a>.

</span>
</li>
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kennard and Stone [1969]</span>
<span class="ltx_bibblock">
R.¬†W. Kennard and L.¬†A. Stone.

</span>
<span class="ltx_bibblock">Computer Aided Design of Experiments.

</span>
<span class="ltx_bibblock"><em id="bib.bib23.1.1" class="ltx_emph ltx_font_italic">Technometrics</em>, 11(1):137‚Äì148, 1969.

</span>
<span class="ltx_bibblock">ISSN 0040-1706.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.2307/1266770" title="" class="ltx_ref ltx_href">10.2307/1266770</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.jstor.org/stable/1266770" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.jstor.org/stable/1266770</a>.

</span>
<span class="ltx_bibblock">Publisher: [Taylor &amp; Francis, Ltd., American Statistical Association, American Society for Quality].

</span>
</li>
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Galvao et¬†al. [2005]</span>
<span class="ltx_bibblock">
Roberto Kawakami¬†Harrop Galvao, M√°rio C√©sar¬†Ugulino Araujo, Gledson¬†Em√≠dio Jos√©, Marcio Jos√©¬†Coelho Pontes, Edvan¬†Cirino Silva, and Teresa Cristina¬†Bezerra Saldanha.

</span>
<span class="ltx_bibblock">A method for calibration and validation subset partitioning.

</span>
<span class="ltx_bibblock"><em id="bib.bib24.1.1" class="ltx_emph ltx_font_italic">Talanta</em>, 67(4):736‚Äì740, 2005.

</span>
<span class="ltx_bibblock">Publisher: Elsevier.

</span>
</li>
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Joseph and Vakayil [2022]</span>
<span class="ltx_bibblock">
V.¬†Roshan Joseph and Akhil Vakayil.

</span>
<span class="ltx_bibblock">SPlit: An Optimal Method for Data Splitting.

</span>
<span class="ltx_bibblock"><em id="bib.bib25.1.1" class="ltx_emph ltx_font_italic">Technometrics</em>, 64(2):166‚Äì176, April 2022.

</span>
<span class="ltx_bibblock">ISSN 0040-1706.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1080/00401706.2021.1921037" title="" class="ltx_ref ltx_href">10.1080/00401706.2021.1921037</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1080/00401706.2021.1921037" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1080/00401706.2021.1921037</a>.

</span>
<span class="ltx_bibblock">Publisher: Taylor &amp; Francis _eprint: https://doi.org/10.1080/00401706.2021.1921037.

</span>
</li>
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Yu et¬†al. [2022]</span>
<span class="ltx_bibblock">
Yu¬†Yu, Shahram Khadivi, and Jia Xu.

</span>
<span class="ltx_bibblock">Can Data Diversity Enhance Learning Generalization?

</span>
<span class="ltx_bibblock">In <em id="bib.bib26.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 29th international conference on computational linguistics</em>, 2022.

</span>
</li>
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Axelrod et¬†al. [2011]</span>
<span class="ltx_bibblock">
Amittai Axelrod, Xiaodong He, and Jianfeng Gao.

</span>
<span class="ltx_bibblock">Domain Adaptation via Pseudo In-Domain Data Selection.

</span>
<span class="ltx_bibblock">In <em id="bib.bib27.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 2011 conference on empirical methods in natural language processing</em>, 2011.

</span>
</li>
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Fan et¬†al. [2017]</span>
<span class="ltx_bibblock">
Yang Fan, Fei Tian, Tao Qin, Jiang Bian, and Tie-Yan Liu.

</span>
<span class="ltx_bibblock">Learning What Data to Learn, February 2017.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/1702.08635" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/1702.08635</a>.

</span>
<span class="ltx_bibblock">arXiv:1702.08635 [cs, stat].

</span>
</li>
<li id="bib.bib29" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Feng et¬†al. [2018]</span>
<span class="ltx_bibblock">
Jun Feng, Minlie Huang, Li¬†Zhao, Yang Yang, and Xiaoyan Zhu.

</span>
<span class="ltx_bibblock">Reinforcement Learning for Relation Classification from Noisy Data, August 2018.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/1808.08013" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/1808.08013</a>.

</span>
<span class="ltx_bibblock">arXiv:1808.08013 [cs, stat].

</span>
</li>
<li id="bib.bib30" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hu et¬†al. [2023]</span>
<span class="ltx_bibblock">
Xuming Hu, Aiwei Liu, Zeqi Tan, Xin Zhang, Chenwei Zhang, Irwin King, and Philip¬†S. Yu.

</span>
<span class="ltx_bibblock">GDA: Generative Data Augmentation Techniques for Relation Extraction Tasks, June 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2305.16663" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2305.16663</a>.

</span>
<span class="ltx_bibblock">arXiv:2305.16663 [cs].

</span>
</li>
<li id="bib.bib31" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Feng et¬†al. [2021]</span>
<span class="ltx_bibblock">
Steven¬†Y. Feng, Varun Gangal, Jason Wei, Sarath Chandar, Soroush Vosoughi, Teruko Mitamura, and Eduard Hovy.

</span>
<span class="ltx_bibblock">A Survey of Data Augmentation Approaches for NLP, December 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2105.03075" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2105.03075</a>.

</span>
<span class="ltx_bibblock">arXiv:2105.03075 [cs].

</span>
</li>
<li id="bib.bib32" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Pellicer et¬†al. [2023]</span>
<span class="ltx_bibblock">
Lucas Francisco Amaral¬†Orosco Pellicer, Taynan¬†Maier Ferreira, and Anna Helena¬†Reali Costa.

</span>
<span class="ltx_bibblock">Data augmentation techniques in natural language processing.

</span>
<span class="ltx_bibblock"><em id="bib.bib32.1.1" class="ltx_emph ltx_font_italic">Applied Soft Computing</em>, 132:109803, January 2023.

</span>
<span class="ltx_bibblock">ISSN 1568-4946.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1016/j.asoc.2022.109803" title="" class="ltx_ref ltx_href">10.1016/j.asoc.2022.109803</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S1568494622008523" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.sciencedirect.com/science/article/pii/S1568494622008523</a>.

</span>
</li>
<li id="bib.bib33" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Smirnova and Cudr√©-Mauroux [2018]</span>
<span class="ltx_bibblock">
Alisa Smirnova and Philippe Cudr√©-Mauroux.

</span>
<span class="ltx_bibblock">Relation Extraction Using Distant Supervision: A Survey.

</span>
<span class="ltx_bibblock"><em id="bib.bib33.1.1" class="ltx_emph ltx_font_italic">ACM Computing Surveys</em>, 51(5):106:1‚Äì106:35, November 2018.

</span>
<span class="ltx_bibblock">ISSN 0360-0300.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1145/3241741" title="" class="ltx_ref ltx_href">10.1145/3241741</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1145/3241741" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1145/3241741</a>.

</span>
</li>
<li id="bib.bib34" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mintz et¬†al. [2009]</span>
<span class="ltx_bibblock">
Mike Mintz, Steven Bills, Rion Snow, and Dan Jurafsky.

</span>
<span class="ltx_bibblock">Distant supervision for relation extraction without labeled data.

</span>
<span class="ltx_bibblock">In <em id="bib.bib34.1.1" class="ltx_emph ltx_font_italic">Proceedings of the Joint Conference of the 47th Annual Meeting of the ACL and the 4th International Joint Conference on Natural Language Processing of the AFNLP</em>, pages 1003‚Äì1011, 2009.

</span>
</li>
<li id="bib.bib35" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Yoon et¬†al. [2023]</span>
<span class="ltx_bibblock">
Wonjin Yoon, Sean Yi, Richard Jackson, Hyunjae Kim, Sunkyu Kim, and Jaewoo Kang.

</span>
<span class="ltx_bibblock">Biomedical relation extraction with knowledge base‚Äìrefined weak supervision.

</span>
<span class="ltx_bibblock"><em id="bib.bib35.1.1" class="ltx_emph ltx_font_italic">Database</em>, 2023:baad054, January 2023.

</span>
<span class="ltx_bibblock">ISSN 1758-0463.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1093/database/baad054" title="" class="ltx_ref ltx_href">10.1093/database/baad054</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1093/database/baad054" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1093/database/baad054</a>.

</span>
</li>
<li id="bib.bib36" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Iinuma et¬†al. [2022]</span>
<span class="ltx_bibblock">
Naoki Iinuma, Makoto Miwa, and Yutaka Sasaki.

</span>
<span class="ltx_bibblock">Improving Supervised Drug-Protein Relation Extraction with Distantly Supervised Models.

</span>
<span class="ltx_bibblock">In <em id="bib.bib36.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 21st Workshop on Biomedical Language Processing</em>, pages 161‚Äì170, Dublin, Ireland, 2022. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2022.bionlp-1.16" title="" class="ltx_ref ltx_href">10.18653/v1/2022.bionlp-1.16</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2022.bionlp-1.16" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2022.bionlp-1.16</a>.

</span>
</li>
<li id="bib.bib37" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Su et¬†al. [2019]</span>
<span class="ltx_bibblock">
Peng Su, Gang Li, Cathy Wu, and K.¬†Vijay-Shanker.

</span>
<span class="ltx_bibblock">Using distant supervision to augment manually annotated data for relation extraction.

</span>
<span class="ltx_bibblock"><em id="bib.bib37.1.1" class="ltx_emph ltx_font_italic">PLOS ONE</em>, 14(7):e0216913, July 2019.

</span>
<span class="ltx_bibblock">ISSN 1932-6203.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1371/journal.pone.0216913" title="" class="ltx_ref ltx_href">10.1371/journal.pone.0216913</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0216913" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0216913</a>.

</span>
<span class="ltx_bibblock">Publisher: Public Library of Science.

</span>
</li>
<li id="bib.bib38" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ye et¬†al. [2022]</span>
<span class="ltx_bibblock">
Jiacheng Ye, Jiahui Gao, Qintong Li, Hang Xu, Jiangtao Feng, Zhiyong Wu, Tao Yu, and Lingpeng Kong.

</span>
<span class="ltx_bibblock">ZeroGen: Efficient Zero-shot Learning via Dataset Generation, October 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2202.07922" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2202.07922</a>.

</span>
<span class="ltx_bibblock">arXiv:2202.07922 [cs].

</span>
</li>
<li id="bib.bib39" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gao et¬†al. [2023]</span>
<span class="ltx_bibblock">
Jiahui Gao, Renjie Pi, Yong Lin, Hang Xu, Jiacheng Ye, Zhiyong Wu, Weizhong Zhang, Xiaodan Liang, Zhenguo Li, and Lingpeng Kong.

</span>
<span class="ltx_bibblock">Self-Guided Noise-Free Data Generation for Efficient Zero-Shot Learning, February 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2205.12679" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2205.12679</a>.

</span>
<span class="ltx_bibblock">arXiv:2205.12679 [cs].

</span>
</li>
<li id="bib.bib40" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Schick and Sch√ºtze [2021]</span>
<span class="ltx_bibblock">
Timo Schick and Hinrich Sch√ºtze.

</span>
<span class="ltx_bibblock">Generating Datasets with Pretrained Language Models.

</span>
<span class="ltx_bibblock">In <em id="bib.bib40.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 2021 Conference on Empirical Methods in Natural Language Processing</em>, pages 6943‚Äì6951, Online and Punta Cana, Dominican Republic, 2021. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2021.emnlp-main.555" title="" class="ltx_ref ltx_href">10.18653/v1/2021.emnlp-main.555</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2021.emnlp-main.555" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2021.emnlp-main.555</a>.

</span>
</li>
<li id="bib.bib41" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">He et¬†al. [2022]</span>
<span class="ltx_bibblock">
Xuanli He, Islam Nassar, Jamie Kiros, Gholamreza Haffari, and Mohammad Norouzi.

</span>
<span class="ltx_bibblock">Generate, Annotate, and Learn: NLP with Synthetic Text, May 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2106.06168" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2106.06168</a>.

</span>
<span class="ltx_bibblock">arXiv:2106.06168 [cs].

</span>
</li>
<li id="bib.bib42" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wang et¬†al. [2021]</span>
<span class="ltx_bibblock">
Zirui Wang, Adams¬†Wei Yu, Orhan Firat, and Yuan Cao.

</span>
<span class="ltx_bibblock">Towards Zero-Label Language Learning, September 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2109.09193" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2109.09193</a>.

</span>
<span class="ltx_bibblock">arXiv:2109.09193 [cs].

</span>
</li>
<li id="bib.bib43" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Smith et¬†al. [2022]</span>
<span class="ltx_bibblock">
Ryan Smith, Jason¬†A. Fries, Braden Hancock, and Stephen¬†H. Bach.

</span>
<span class="ltx_bibblock">Language Models in the Loop: Incorporating Prompting into Weak Supervision, May 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2205.02318" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2205.02318</a>.

</span>
<span class="ltx_bibblock">arXiv:2205.02318 [cs].

</span>
</li>
<li id="bib.bib44" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Meng et¬†al. [2022]</span>
<span class="ltx_bibblock">
Yu¬†Meng, Jiaxin Huang, Yu¬†Zhang, and Jiawei Han.

</span>
<span class="ltx_bibblock">Generating Training Data with Language Models: Towards Zero-Shot Language Understanding, October 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2202.04538" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2202.04538</a>.

</span>
<span class="ltx_bibblock">arXiv:2202.04538 [cs].

</span>
</li>
<li id="bib.bib45" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kumar et¬†al. [2020]</span>
<span class="ltx_bibblock">
Varun Kumar, Ashutosh Choudhary, and Eunah Cho.

</span>
<span class="ltx_bibblock">Data Augmentation using Pre-trained Transformer Models.

</span>
<span class="ltx_bibblock"><em id="bib.bib45.1.1" class="ltx_emph ltx_font_italic">arXiv preprint arXiv:2003.02245</em>, 2020.

</span>
</li>
<li id="bib.bib46" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Bonifacio et¬†al. [2022]</span>
<span class="ltx_bibblock">
Luiz Bonifacio, Hugo Abonizio, Marzieh Fadaee, and Rodrigo Nogueira.

</span>
<span class="ltx_bibblock">InPars: Data Augmentation for Information Retrieval using Large Language Models, February 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2202.05144" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2202.05144</a>.

</span>
<span class="ltx_bibblock">arXiv:2202.05144 [cs].

</span>
</li>
<li id="bib.bib47" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Dai et¬†al. [2022]</span>
<span class="ltx_bibblock">
Zhuyun Dai, Vincent¬†Y. Zhao, Ji¬†Ma, Yi¬†Luan, Jianmo Ni, Jing Lu, Anton Bakalov, Kelvin Guu, Keith¬†B. Hall, and Ming-Wei Chang.

</span>
<span class="ltx_bibblock">Promptagator: Few-shot Dense Retrieval From 8 Examples, September 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2209.11755" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2209.11755</a>.

</span>
<span class="ltx_bibblock">arXiv:2209.11755 [cs].

</span>
</li>
<li id="bib.bib48" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Meng et¬†al. [2023]</span>
<span class="ltx_bibblock">
Yu¬†Meng, Martin Michalski, Jiaxin Huang, Yu¬†Zhang, Tarek Abdelzaher, and Jiawei Han.

</span>
<span class="ltx_bibblock">Tuning Language Models as Training Data Generators for Augmentation-Enhanced Few-Shot Learning, May 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2211.03044" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2211.03044</a>.

</span>
<span class="ltx_bibblock">arXiv:2211.03044 [cs].

</span>
</li>
<li id="bib.bib49" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chen et¬†al. [2022a]</span>
<span class="ltx_bibblock">
Maximillian Chen, Alexandros Papangelis, Chenyang Tao, Andy Rosenbaum, Seokhwan Kim, Yang Liu, Zhou Yu, and Dilek Hakkani-Tur.

</span>
<span class="ltx_bibblock">Weakly Supervised Data Augmentation Through Prompting for Dialogue Understanding, November 2022a.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2210.14169" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2210.14169</a>.

</span>
<span class="ltx_bibblock">arXiv:2210.14169 [cs].

</span>
</li>
<li id="bib.bib50" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Yoo et¬†al. [2021]</span>
<span class="ltx_bibblock">
Kang¬†Min Yoo, Dongju Park, Jaewook Kang, Sang-Woo Lee, and Woomyeong Park.

</span>
<span class="ltx_bibblock">GPT3Mix: Leveraging Large-scale Language Models for Text Augmentation, November 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2104.08826" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2104.08826</a>.

</span>
<span class="ltx_bibblock">arXiv:2104.08826 [cs].

</span>
</li>
<li id="bib.bib51" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Anaby-Tavor et¬†al. [2019]</span>
<span class="ltx_bibblock">
Ateret Anaby-Tavor, Boaz Carmeli, Esther Goldbraich, Amir Kantor, George Kour, Segev Shlomov, Naama Tepper, and Naama Zwerdling.

</span>
<span class="ltx_bibblock">Not Enough Data? Deep Learning to the Rescue!, November 2019.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/1911.03118" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/1911.03118</a>.

</span>
<span class="ltx_bibblock">arXiv:1911.03118 [cs].

</span>
</li>
<li id="bib.bib52" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Papanikolaou and Pierleoni [2020]</span>
<span class="ltx_bibblock">
Yannis Papanikolaou and Andrea Pierleoni.

</span>
<span class="ltx_bibblock">DARE: Data Augmented Relation Extraction with GPT-2, April 2020.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2004.13845" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2004.13845</a>.

</span>
<span class="ltx_bibblock">arXiv:2004.13845 [cs, stat].

</span>
</li>
<li id="bib.bib53" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hartvigsen et¬†al. [2022]</span>
<span class="ltx_bibblock">
Thomas Hartvigsen, Saadia Gabriel, Hamid Palangi, Maarten Sap, Dipankar Ray, and Ece Kamar.

</span>
<span class="ltx_bibblock">ToxiGen: A Large-Scale Machine-Generated Dataset for Adversarial and Implicit Hate Speech Detection, July 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2203.09509" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2203.09509</a>.

</span>
<span class="ltx_bibblock">arXiv:2203.09509 [cs].

</span>
</li>
<li id="bib.bib54" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Josifoski et¬†al. [2023]</span>
<span class="ltx_bibblock">
Martin Josifoski, Marija Sakota, Maxime Peyrard, and Robert West.

</span>
<span class="ltx_bibblock">Exploiting Asymmetry for Synthetic Training Data Generation: SynthIE and the Case of Information Extraction, March 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2303.04132" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2303.04132</a>.

</span>
<span class="ltx_bibblock">arXiv:2303.04132 [cs].

</span>
</li>
<li id="bib.bib55" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Tang et¬†al. [2023]</span>
<span class="ltx_bibblock">
Ruixiang Tang, Xiaotian Han, Xiaoqian Jiang, and Xia Hu.

</span>
<span class="ltx_bibblock">Does Synthetic Data Generation of LLMs Help Clinical Text Mining?, March 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://arxiv.org/abs/2303.04360v2" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://arxiv.org/abs/2303.04360v2</a>.

</span>
</li>
<li id="bib.bib56" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Veselovsky et¬†al. [2023]</span>
<span class="ltx_bibblock">
Veniamin Veselovsky, Manoel¬†Horta Ribeiro, Akhil Arora, Martin Josifoski, Ashton Anderson, and Robert West.

</span>
<span class="ltx_bibblock">Generating Faithful Synthetic Data with Large Language Models: A Case Study in Computational Social Science, May 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2305.15041" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2305.15041</a>.

</span>
<span class="ltx_bibblock">arXiv:2305.15041 [cs].

</span>
</li>
<li id="bib.bib57" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Yang et¬†al. [2020]</span>
<span class="ltx_bibblock">
Yiben Yang, Chaitanya Malaviya, Jared Fernandez, Swabha Swayamdipta, Ronan Le¬†Bras, Ji-Ping Wang, Chandra Bhagavatula, Yejin Choi, and Doug Downey.

</span>
<span class="ltx_bibblock">Generative Data Augmentation for Commonsense Reasoning.

</span>
<span class="ltx_bibblock">In <em id="bib.bib57.1.1" class="ltx_emph ltx_font_italic">Findings of the Association for Computational Linguistics: EMNLP 2020</em>, pages 1008‚Äì1025, Online, 2020. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2020.findings-emnlp.90" title="" class="ltx_ref ltx_href">10.18653/v1/2020.findings-emnlp.90</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.aclweb.org/anthology/2020.findings-emnlp.90" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.aclweb.org/anthology/2020.findings-emnlp.90</a>.

</span>
</li>
<li id="bib.bib58" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Aggarwal et¬†al. [2023]</span>
<span class="ltx_bibblock">
Karan Aggarwal, Henry Jin, and Aitzaz Ahmad.

</span>
<span class="ltx_bibblock">ECG-QALM: Entity-Controlled Synthetic Text Generation using Contextual Q&amp;A for NER.

</span>
<span class="ltx_bibblock">In Anna Rogers, Jordan Boyd-Graber, and Naoaki Okazaki, editors, <em id="bib.bib58.1.1" class="ltx_emph ltx_font_italic">Findings of the Association for Computational Linguistics: ACL 2023</em>, pages 5649‚Äì5660, Toronto, Canada, July 2023. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2023.findings-acl.349" title="" class="ltx_ref ltx_href">10.18653/v1/2023.findings-acl.349</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2023.findings-acl.349" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2023.findings-acl.349</a>.

</span>
</li>
<li id="bib.bib59" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Xu et¬†al. [2023]</span>
<span class="ltx_bibblock">
Benfeng Xu, Quan Wang, Yajuan Lyu, Dai Dai, Yongdong Zhang, and Zhendong Mao.

</span>
<span class="ltx_bibblock">S2ynRE: Two-stage Self-training with Synthetic data for Low-resource Relation Extraction.

</span>
<span class="ltx_bibblock">In <em id="bib.bib59.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 61st Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers)</em>, pages 8186‚Äì8207, Toronto, Canada, 2023. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2023.acl-long.455" title="" class="ltx_ref ltx_href">10.18653/v1/2023.acl-long.455</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2023.acl-long.455" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2023.acl-long.455</a>.

</span>
</li>
<li id="bib.bib60" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kambar et¬†al. [2022]</span>
<span class="ltx_bibblock">
Mina Esmail Zadeh¬†Nojoo Kambar, Armin Esmaeilzadeh, and Maryam Heidari.

</span>
<span class="ltx_bibblock">A survey on deep learning techniques for joint named entities and relation extraction.

</span>
<span class="ltx_bibblock">In <em id="bib.bib60.1.1" class="ltx_emph ltx_font_italic">2022 IEEE World AI IoT Congress (AIIoT)</em>, pages 218‚Äì224. IEEE, 2022.

</span>
</li>
<li id="bib.bib61" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li et¬†al. [2017]</span>
<span class="ltx_bibblock">
Fei Li, Meishan Zhang, Guohong Fu, and Donghong Ji.

</span>
<span class="ltx_bibblock">A neural joint model for entity and relation extraction from biomedical text.

</span>
<span class="ltx_bibblock"><em id="bib.bib61.1.1" class="ltx_emph ltx_font_italic">BMC Bioinformatics</em>, 18(1):198, March 2017.

</span>
<span class="ltx_bibblock">ISSN 1471-2105.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1186/s12859-017-1609-9" title="" class="ltx_ref ltx_href">10.1186/s12859-017-1609-9</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1186/s12859-017-1609-9" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1186/s12859-017-1609-9</a>.

</span>
</li>
<li id="bib.bib62" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Esmail Zadeh Nojoo¬†Kambar et¬†al. [2022]</span>
<span class="ltx_bibblock">
Mina Esmail Zadeh Nojoo¬†Kambar, Armin Esmaeilzadeh, and Kazem Taghva.

</span>
<span class="ltx_bibblock">Chemical-Gene Relation Extraction with¬†Graph Neural Networks and¬†BERT Encoder.

</span>
<span class="ltx_bibblock">In Kevin Daimi and Abeer Al¬†Sadoon, editors, <em id="bib.bib62.1.1" class="ltx_emph ltx_font_italic">Proceedings of the ICR‚Äô22 International Conference on Innovations in Computing Research</em>, Advances in Intelligent Systems and Computing, pages 166‚Äì179, Cham, 2022. Springer International Publishing.

</span>
<span class="ltx_bibblock">ISBN 978-3-031-14054-9.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1007/978-3-031-14054-9_17" title="" class="ltx_ref ltx_href">10.1007/978-3-031-14054-9_17</a>.

</span>
</li>
<li id="bib.bib63" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Huguet¬†Cabot and Navigli [2021]</span>
<span class="ltx_bibblock">
Pere-Llu√≠s Huguet¬†Cabot and Roberto Navigli.

</span>
<span class="ltx_bibblock">REBEL: Relation Extraction By End-to-end Language generation.

</span>
<span class="ltx_bibblock">In <em id="bib.bib63.1.1" class="ltx_emph ltx_font_italic">Findings of the Association for Computational Linguistics: EMNLP 2021</em>, pages 2370‚Äì2381, Punta Cana, Dominican Republic, 2021. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2021.findings-emnlp.204" title="" class="ltx_ref ltx_href">10.18653/v1/2021.findings-emnlp.204</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2021.findings-emnlp.204" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2021.findings-emnlp.204</a>.

</span>
</li>
<li id="bib.bib64" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhang et¬†al. [2020b]</span>
<span class="ltx_bibblock">
Ranran¬†Haoran Zhang, Qianying Liu, Aysa¬†Xuemo Fan, Heng Ji, Daojian Zeng, Fei Cheng, Daisuke Kawahara, and Sadao Kurohashi.

</span>
<span class="ltx_bibblock">Minimize Exposure Bias of Seq2Seq Models in Joint Entity and Relation Extraction.

</span>
<span class="ltx_bibblock">In <em id="bib.bib64.1.1" class="ltx_emph ltx_font_italic">Findings of the Association for Computational Linguistics: EMNLP 2020</em>, pages 236‚Äì246, Online, 2020b. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2020.findings-emnlp.23" title="" class="ltx_ref ltx_href">10.18653/v1/2020.findings-emnlp.23</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.aclweb.org/anthology/2020.findings-emnlp.23" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.aclweb.org/anthology/2020.findings-emnlp.23</a>.

</span>
</li>
<li id="bib.bib65" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zeng et¬†al. [2019]</span>
<span class="ltx_bibblock">
Xiangrong Zeng, Shizhu He, Daojian Zeng, Kang Liu, Shengping Liu, and Jun Zhao.

</span>
<span class="ltx_bibblock">Learning the Extraction Order of Multiple Relational Facts in a Sentence with Reinforcement Learning.

</span>
<span class="ltx_bibblock">In <em id="bib.bib65.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing (EMNLP-IJCNLP)</em>, pages 367‚Äì377, Hong Kong, China, 2019. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/D19-1035" title="" class="ltx_ref ltx_href">10.18653/v1/D19-1035</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.aclweb.org/anthology/D19-1035" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.aclweb.org/anthology/D19-1035</a>.

</span>
</li>
<li id="bib.bib66" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hou et¬†al. [2021]</span>
<span class="ltx_bibblock">
Yutai Hou, Yingce Xia, Lijun Wu, Shufang Xie, Yang Fan, Jinhua Zhu, Wanxiang Che, Tao Qin, and Tie-Yan Liu.

</span>
<span class="ltx_bibblock">Discovering Drug-Target Interaction Knowledge from Biomedical Literature, September 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2109.13187" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2109.13187</a>.

</span>
<span class="ltx_bibblock">arXiv:2109.13187 [cs].

</span>
</li>
<li id="bib.bib67" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zeng et¬†al. [2018]</span>
<span class="ltx_bibblock">
Xiangrong Zeng, Daojian Zeng, Shizhu He, Kang Liu, and Jun Zhao.

</span>
<span class="ltx_bibblock">Extracting Relational Facts by an End-to-End Neural Model with Copy Mechanism.

</span>
<span class="ltx_bibblock">In <em id="bib.bib67.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 56th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers)</em>, pages 506‚Äì514, Melbourne, Australia, 2018. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/P18-1047" title="" class="ltx_ref ltx_href">10.18653/v1/P18-1047</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://aclweb.org/anthology/P18-1047" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://aclweb.org/anthology/P18-1047</a>.

</span>
</li>
<li id="bib.bib68" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Eberts and Ulges [2021]</span>
<span class="ltx_bibblock">
Markus Eberts and Adrian Ulges.

</span>
<span class="ltx_bibblock">An End-to-end Model for Entity-level Relation Extraction using Multi-instance Learning.

</span>
<span class="ltx_bibblock">In <em id="bib.bib68.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 16th Conference of the European Chapter of the Association for Computational Linguistics: Main Volume</em>, pages 3650‚Äì3660, Online, 2021. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2021.eacl-main.319" title="" class="ltx_ref ltx_href">10.18653/v1/2021.eacl-main.319</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2021.eacl-main.319" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2021.eacl-main.319</a>.

</span>
</li>
<li id="bib.bib69" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Paolini et¬†al. [2021]</span>
<span class="ltx_bibblock">
Giovanni Paolini, Ben Athiwaratkun, Jason Krone, Jie Ma, Alessandro Achille, Rishita Anubhai, Cicero Nogueira¬†dos Santos, Bing Xiang, and Stefano Soatto.

</span>
<span class="ltx_bibblock">Structured Prediction as Translation between Augmented Natural Languages, December 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2101.05779" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2101.05779</a>.

</span>
<span class="ltx_bibblock">arXiv:2101.05779 [cs].

</span>
</li>
<li id="bib.bib70" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Leinster [2022]</span>
<span class="ltx_bibblock">
Tom Leinster.

</span>
<span class="ltx_bibblock">Entropy and Diversity: The Axiomatic Approach, October 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2012.02113" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2012.02113</a>.

</span>
<span class="ltx_bibblock">arXiv:2012.02113 [cs, math, q-bio].

</span>
</li>
<li id="bib.bib71" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Jost [2006]</span>
<span class="ltx_bibblock">
Lou Jost.

</span>
<span class="ltx_bibblock">Entropy and diversity.

</span>
<span class="ltx_bibblock"><em id="bib.bib71.1.1" class="ltx_emph ltx_font_italic">Oikos</em>, 113(2):363‚Äì375, 2006.

</span>
<span class="ltx_bibblock">ISSN 1600-0706.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1111/j.2006.0030-1299.14714.x" title="" class="ltx_ref ltx_href">10.1111/j.2006.0030-1299.14714.x</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.2006.0030-1299.14714.x" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://onlinelibrary.wiley.com/doi/abs/10.1111/j.2006.0030-1299.14714.x</a>.

</span>
<span class="ltx_bibblock">_eprint: https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.2006.0030-1299.14714.x.

</span>
</li>
<li id="bib.bib72" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hill [1973]</span>
<span class="ltx_bibblock">
M.¬†O. Hill.

</span>
<span class="ltx_bibblock">Diversity and Evenness: A Unifying Notation and Its Consequences.

</span>
<span class="ltx_bibblock"><em id="bib.bib72.1.1" class="ltx_emph ltx_font_italic">Ecology</em>, 54(2):427‚Äì432, 1973.

</span>
<span class="ltx_bibblock">ISSN 0012-9658.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.2307/1934352" title="" class="ltx_ref ltx_href">10.2307/1934352</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.jstor.org/stable/1934352" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.jstor.org/stable/1934352</a>.

</span>
<span class="ltx_bibblock">Publisher: Ecological Society of America.

</span>
</li>
<li id="bib.bib73" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Dettmers et¬†al. [2023]</span>
<span class="ltx_bibblock">
Tim Dettmers, Artidoro Pagnoni, Ari Holtzman, and Luke Zettlemoyer.

</span>
<span class="ltx_bibblock">Qlora: Efficient finetuning of quantized llms.

</span>
<span class="ltx_bibblock"><em id="bib.bib73.1.1" class="ltx_emph ltx_font_italic">arXiv preprint arXiv:2305.14314</em>, 2023.

</span>
</li>
<li id="bib.bib74" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hu et¬†al. [2021]</span>
<span class="ltx_bibblock">
Edward¬†J. Hu, Yelong Shen, Phillip Wallis, Zeyuan Allen-Zhu, Yuanzhi Li, Shean Wang, Lu¬†Wang, and Weizhu Chen.

</span>
<span class="ltx_bibblock">LoRA: Low-Rank Adaptation of Large Language Models, October 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2106.09685" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2106.09685</a>.

</span>
<span class="ltx_bibblock">arXiv:2106.09685 [cs].

</span>
</li>
<li id="bib.bib75" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Radford et¬†al. [2019]</span>
<span class="ltx_bibblock">
Alec Radford, Jeffrey Wu, Rewon Child, David Luan, Dario Amodei, and Ilya Sutskever.

</span>
<span class="ltx_bibblock">Language Models are Unsupervised Multitask Learners.

</span>
<span class="ltx_bibblock"><em id="bib.bib75.1.1" class="ltx_emph ltx_font_italic">OpenAI blog</em>, 2019.

</span>
</li>
<li id="bib.bib76" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wei et¬†al. [2019]</span>
<span class="ltx_bibblock">
Chih-Hsuan Wei, Alexis Allot, Robert Leaman, and Zhiyong Lu.

</span>
<span class="ltx_bibblock">PubTator central: automated concept annotation for biomedical full text articles.

</span>
<span class="ltx_bibblock"><em id="bib.bib76.1.1" class="ltx_emph ltx_font_italic">Nucleic Acids Research</em>, 47(W1):W587‚ÄìW593, July 2019.

</span>
<span class="ltx_bibblock">ISSN 0305-1048.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1093/nar/gkz389" title="" class="ltx_ref ltx_href">10.1093/nar/gkz389</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1093/nar/gkz389" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1093/nar/gkz389</a>.

</span>
</li>
<li id="bib.bib77" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kim et¬†al. [2021]</span>
<span class="ltx_bibblock">
Hyun¬†Woo Kim, Mingxun Wang, Christopher¬†A Leber, Louis-F√©lix Nothias, Raphael Reher, Kyo¬†Bin Kang, Justin¬†JJ Van Der¬†Hooft, Pieter¬†C Dorrestein, William¬†H Gerwick, and Garrison¬†W Cottrell.

</span>
<span class="ltx_bibblock">NPClassifier: A deep neural network-based structural classification tool for natural products.

</span>
<span class="ltx_bibblock"><em id="bib.bib77.1.1" class="ltx_emph ltx_font_italic">Journal of natural products</em>, 84(11):2795‚Äì2807, 2021.

</span>
<span class="ltx_bibblock">Publisher: ACS Publications.

</span>
</li>
<li id="bib.bib78" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Newman [2005]</span>
<span class="ltx_bibblock">
Mark¬†EJ Newman.

</span>
<span class="ltx_bibblock">Power laws, Pareto distributions and Zipf‚Äôs law.

</span>
<span class="ltx_bibblock"><em id="bib.bib78.1.1" class="ltx_emph ltx_font_italic">Contemporary physics</em>, 46(5):323‚Äì351, 2005.

</span>
<span class="ltx_bibblock">Publisher: Taylor &amp; Francis.

</span>
</li>
<li id="bib.bib79" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Swainston et¬†al. [2016]</span>
<span class="ltx_bibblock">
Neil Swainston, Kieran Smallbone, Hooman Hefzi, Paul¬†D. Dobson, Judy Brewer, Michael Hanscho, Daniel¬†C. Zielinski, Kok¬†Siong Ang, Natalie¬†J. Gardiner, Jahir¬†M. Gutierrez, Sarantos Kyriakopoulos, Meiyappan Lakshmanan, Shangzhong Li, Joanne¬†K. Liu, Veronica¬†S. Mart√≠nez, Camila¬†A. Orellana, Lake-Ee Quek, Alex Thomas, Juergen Zanghellini, Nicole Borth, Dong-Yup Lee, Lars¬†K. Nielsen, Douglas¬†B. Kell, Nathan¬†E. Lewis, and Pedro Mendes.

</span>
<span class="ltx_bibblock">Recon 2.2: from reconstruction to model of human metabolism.

</span>
<span class="ltx_bibblock"><em id="bib.bib79.1.1" class="ltx_emph ltx_font_italic">Metabolomics</em>, 12(7):109, June 2016.

</span>
<span class="ltx_bibblock">ISSN 1573-3890.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1007/s11306-016-1051-4" title="" class="ltx_ref ltx_href">10.1007/s11306-016-1051-4</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1007/s11306-016-1051-4" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1007/s11306-016-1051-4</a>.

</span>
</li>
<li id="bib.bib80" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Thiele et¬†al. [2013]</span>
<span class="ltx_bibblock">
Ines Thiele, Neil Swainston, Ronan M.¬†T. Fleming, Andreas Hoppe, Swagatika Sahoo, Maike¬†K. Aurich, Hulda Haraldsdottir, Monica¬†L. Mo, Ottar Rolfsson, Miranda¬†D. Stobbe, Stefan¬†G. Thorleifsson, Rasmus Agren, Christian B√∂lling, Sergio Bordel, Arvind¬†K. Chavali, Paul Dobson, Warwick¬†B. Dunn, Lukas Endler, David Hala, Michael Hucka, Duncan Hull, Daniel Jameson, Neema Jamshidi, Jon¬†J. Jonsson, Nick Juty, Sarah Keating, Intawat Nookaew, Nicolas Le¬†Nov√®re, Naglis Malys, Alexander Mazein, Jason¬†A. Papin, Nathan¬†D. Price, Evgeni Selkov, Martin¬†I. Sigurdsson, Evangelos Simeonidis, Nikolaus Sonnenschein, Kieran Smallbone, Anatoly Sorokin, Johannes H. G.¬†M. van Beek, Dieter Weichart, Igor Goryanin, Jens Nielsen, Hans¬†V. Westerhoff, Douglas¬†B. Kell, Pedro Mendes, and Bernhard¬†O Palsson.

</span>
<span class="ltx_bibblock">A community-driven global reconstruction of human metabolism.

</span>
<span class="ltx_bibblock"><em id="bib.bib80.1.1" class="ltx_emph ltx_font_italic">Nature Biotechnology</em>, 31(5):419‚Äì425, May 2013.

</span>
<span class="ltx_bibblock">ISSN 1546-1696.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1038/nbt.2488" title="" class="ltx_ref ltx_href">10.1038/nbt.2488</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.nature.com/articles/nbt.2488" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.nature.com/articles/nbt.2488</a>.

</span>
<span class="ltx_bibblock">Number: 5 Publisher: Nature Publishing Group.

</span>
</li>
<li id="bib.bib81" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Stefanini et¬†al. [2017]</span>
<span class="ltx_bibblock">
Irene Stefanini, Silvia Carlin, Noemi Tocci, Davide Albanese, Claudio Donati, Pietro Franceschi, Michele Paris, Alberto Zenato, Silvano Tempesta, Alberto Bronzato, Urska Vrhovsek, Fulvio Mattivi, and Duccio Cavalieri.

</span>
<span class="ltx_bibblock">Core Microbiota and Metabolome of Vitis vinifera L. cv. Corvina Grapes and Musts.

</span>
<span class="ltx_bibblock"><em id="bib.bib81.1.1" class="ltx_emph ltx_font_italic">Frontiers in Microbiology</em>, 8, 2017.

</span>
<span class="ltx_bibblock">ISSN 1664-302X.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.frontiersin.org/articles/10.3389/fmicb.2017.00457" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.frontiersin.org/articles/10.3389/fmicb.2017.00457</a>.

</span>
</li>
<li id="bib.bib82" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Thompson et¬†al. [2006]</span>
<span class="ltx_bibblock">
Lilian¬†U. Thompson, Beatrice¬†A. Boucher, Zhen Liu, Michelle Cotterchio, and Nancy Kreiger.

</span>
<span class="ltx_bibblock">Phytoestrogen Content of Foods Consumed in Canada, Including Isoflavones, Lignans, and Coumestan.

</span>
<span class="ltx_bibblock"><em id="bib.bib82.1.1" class="ltx_emph ltx_font_italic">Nutrition and Cancer</em>, 54(2):184‚Äì201, July 2006.

</span>
<span class="ltx_bibblock">ISSN 0163-5581.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1207/s15327914nc5402_5" title="" class="ltx_ref ltx_href">10.1207/s15327914nc5402_5</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.1207/s15327914nc5402_5" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.1207/s15327914nc5402_5</a>.

</span>
<span class="ltx_bibblock">Publisher: Routledge _eprint: https://doi.org/10.1207/s15327914nc5402_5.

</span>
</li>
<li id="bib.bib83" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhou et¬†al. [2023]</span>
<span class="ltx_bibblock">
Chunting Zhou, Pengfei Liu, Puxin Xu, Srini Iyer, Jiao Sun, Yuning Mao, Xuezhe Ma, Avia Efrat, Ping Yu, Lili Yu, Susan Zhang, Gargi Ghosh, Mike Lewis, Luke Zettlemoyer, and Omer Levy.

</span>
<span class="ltx_bibblock">LIMA: Less Is More for Alignment, May 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2305.11206" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2305.11206</a>.

</span>
<span class="ltx_bibblock">arXiv:2305.11206 [cs].

</span>
</li>
<li id="bib.bib84" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li et¬†al. [2023]</span>
<span class="ltx_bibblock">
Ming Li, Yong Zhang, Zhitao Li, Jiuhai Chen, Lichang Chen, Ning Cheng, Jianzong Wang, Tianyi Zhou, and Jing Xiao.

</span>
<span class="ltx_bibblock">From Quantity to Quality: Boosting LLM Performance with Self-Guided Data Selection for Instruction Tuning, September 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2308.12032" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2308.12032</a>.

</span>
<span class="ltx_bibblock">arXiv:2308.12032 [cs].

</span>
</li>
<li id="bib.bib85" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Northcutt et¬†al. [2022]</span>
<span class="ltx_bibblock">
Curtis¬†G. Northcutt, Lu¬†Jiang, and Isaac¬†L. Chuang.

</span>
<span class="ltx_bibblock">Confident Learning: Estimating Uncertainty in Dataset Labels, August 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/1911.00068" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/1911.00068</a>.

</span>
<span class="ltx_bibblock">arXiv:1911.00068 [cs, stat].

</span>
</li>
<li id="bib.bib86" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Jain et¬†al. [2020]</span>
<span class="ltx_bibblock">
Abhinav Jain, Hima Patel, Lokesh Nagalapatti, Nitin Gupta, Sameep Mehta, Shanmukha Guttula, Shashank Mujumdar, Shazia Afzal, Ruhi Sharma¬†Mittal, and Vitobha Munigala.

</span>
<span class="ltx_bibblock">Overview and importance of data quality for machine learning tasks.

</span>
<span class="ltx_bibblock">In <em id="bib.bib86.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 26th ACM SIGKDD international conference on knowledge discovery &amp; data mining</em>, pages 3561‚Äì3562, 2020.

</span>
</li>
<li id="bib.bib87" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liu et¬†al. [2021]</span>
<span class="ltx_bibblock">
Jiachang Liu, Dinghan Shen, Yizhe Zhang, Bill Dolan, Lawrence Carin, and Weizhu Chen.

</span>
<span class="ltx_bibblock">What Makes Good In-Context Examples for GPT-$3$?, January 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2101.06804" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2101.06804</a>.

</span>
<span class="ltx_bibblock">arXiv:2101.06804 [cs].

</span>
</li>
<li id="bib.bib88" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chen et¬†al. [2022b]</span>
<span class="ltx_bibblock">
Yanda Chen, Ruiqi Zhong, Sheng Zha, George Karypis, and He¬†He.

</span>
<span class="ltx_bibblock">Meta-learning via Language Model In-context Tuning.

</span>
<span class="ltx_bibblock">In <em id="bib.bib88.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 60th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers)</em>, pages 719‚Äì730, Dublin, Ireland, May 2022b. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2022.acl-long.53" title="" class="ltx_ref ltx_href">10.18653/v1/2022.acl-long.53</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2022.acl-long.53" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2022.acl-long.53</a>.

</span>
</li>
<li id="bib.bib89" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Northcutt et¬†al. [2021]</span>
<span class="ltx_bibblock">
Curtis¬†G. Northcutt, Anish Athalye, and Jonas Mueller.

</span>
<span class="ltx_bibblock">Pervasive Label Errors in Test Sets Destabilize Machine Learning Benchmarks, November 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2103.14749" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2103.14749</a>.

</span>
<span class="ltx_bibblock">arXiv:2103.14749 [cs, stat].

</span>
</li>
<li id="bib.bib90" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liang et¬†al. [2022]</span>
<span class="ltx_bibblock">
Weixin Liang, Girmaw¬†Abebe Tadesse, Daniel Ho, L.¬†Fei-Fei, Matei Zaharia, Ce¬†Zhang, and James Zou.

</span>
<span class="ltx_bibblock">Advances, challenges and opportunities in creating data for trustworthy AI.

</span>
<span class="ltx_bibblock"><em id="bib.bib90.1.1" class="ltx_emph ltx_font_italic">Nature Machine Intelligence</em>, 4(8):669‚Äì677, August 2022.

</span>
<span class="ltx_bibblock">ISSN 2522-5839.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1038/s42256-022-00516-1" title="" class="ltx_ref ltx_href">10.1038/s42256-022-00516-1</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.nature.com/articles/s42256-022-00516-1" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.nature.com/articles/s42256-022-00516-1</a>.

</span>
<span class="ltx_bibblock">Number: 8 Publisher: Nature Publishing Group.

</span>
</li>
<li id="bib.bib91" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Shang et¬†al. [2018]</span>
<span class="ltx_bibblock">
Jingbo Shang, Liyuan Liu, Xiaotao Gu, Xiang Ren, Teng Ren, and Jiawei Han.

</span>
<span class="ltx_bibblock">Learning Named Entity Tagger using Domain-Specific Dictionary.

</span>
<span class="ltx_bibblock">In <em id="bib.bib91.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 2018 Conference on Empirical Methods in Natural Language Processing</em>, pages 2054‚Äì2064, Brussels, Belgium, 2018. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/D18-1230" title="" class="ltx_ref ltx_href">10.18653/v1/D18-1230</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://aclweb.org/anthology/D18-1230" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://aclweb.org/anthology/D18-1230</a>.

</span>
</li>
<li id="bib.bib92" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chiang et¬†al. [2023]</span>
<span class="ltx_bibblock">
Wei-Lin Chiang, Zhuohan Li, Zi¬†Lin, Ying Sheng, Zhanghao Wu, Hao Zhang, Lianmin Zheng, Siyuan Zhuang, Yonghao Zhuang, Joseph¬†E. Gonzalez, Ion Stoica, and Eric¬†P. Xing.

</span>
<span class="ltx_bibblock">Vicuna: An Open-Source Chatbot Impressing GPT-4 with 90%* ChatGPT Quality, March 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://lmsys.org/blog/2023-03-30-vicuna/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://lmsys.org/blog/2023-03-30-vicuna/</a>.

</span>
</li>
<li id="bib.bib93" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Augenstein et¬†al. [2017]</span>
<span class="ltx_bibblock">
Isabelle Augenstein, Mrinal Das, Sebastian Riedel, Lakshmi Vikraman, and Andrew McCallum.

</span>
<span class="ltx_bibblock">SemEval 2017 task 10: ScienceIE - extracting keyphrases and relations from scientific publications.

</span>
<span class="ltx_bibblock">In Steven Bethard, Marine Carpuat, Marianna Apidianaki, Saif¬†M. Mohammad, Daniel Cer, and David Jurgens, editors, <em id="bib.bib93.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 11th International Workshop on Semantic Evaluation (SemEval-2017)</em>, pages 546‚Äì555, Vancouver, Canada, August 2017. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/S17-2091" title="" class="ltx_ref ltx_href">10.18653/v1/S17-2091</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/S17-2091" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/S17-2091</a>.

</span>
</li>
<li id="bib.bib94" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mazumder et¬†al. [2023]</span>
<span class="ltx_bibblock">
Mark Mazumder, Colby Banbury, Xiaozhe Yao, Bojan Karla≈°, William¬†Gaviria Rojas, Sudnya Diamos, Greg Diamos, Lynn He, Alicia Parrish, Hannah¬†Rose Kirk, Jessica Quaye, Charvi Rastogi, Douwe Kiela, David Jurado, David Kanter, Rafael Mosquera, Juan Ciro, Lora Aroyo, Bilge Acun, Lingjiao Chen, Mehul¬†Smriti Raje, Max Bartolo, Sabri Eyuboglu, Amirata Ghorbani, Emmett Goodman, Oana Inel, Tariq Kane, Christine¬†R. Kirkpatrick, Tzu-Sheng Kuo, Jonas Mueller, Tristan Thrush, Joaquin Vanschoren, Margaret Warren, Adina Williams, Serena Yeung, Newsha Ardalani, Praveen Paritosh, Ce¬†Zhang, James Zou, Carole-Jean Wu, Cody Coleman, Andrew Ng, Peter Mattson, and Vijay¬†Janapa Reddi.

</span>
<span class="ltx_bibblock">DataPerf: Benchmarks for Data-Centric AI Development, July 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2207.10062" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2207.10062</a>.

</span>
<span class="ltx_bibblock">arXiv:2207.10062 [cs].

</span>
</li>
<li id="bib.bib95" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zha et¬†al. [2023]</span>
<span class="ltx_bibblock">
Daochen Zha, Zaid¬†Pervaiz Bhat, Kwei-Herng Lai, Fan Yang, Zhimeng Jiang, Shaochen Zhong, and Xia Hu.

</span>
<span class="ltx_bibblock">Data-centric Artificial Intelligence: A Survey, June 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2303.10158" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2303.10158</a>.

</span>
<span class="ltx_bibblock">arXiv:2303.10158 [cs].

</span>
</li>
<li id="bib.bib96" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gong et¬†al. [2019]</span>
<span class="ltx_bibblock">
Zhiqiang Gong, Ping Zhong, and Weidong Hu.

</span>
<span class="ltx_bibblock">Diversity in Machine Learning.

</span>
<span class="ltx_bibblock"><em id="bib.bib96.1.1" class="ltx_emph ltx_font_italic">IEEE Access</em>, 7:64323‚Äì64350, 2019.

</span>
<span class="ltx_bibblock">ISSN 2169-3536.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1109/ACCESS.2019.2917620" title="" class="ltx_ref ltx_href">10.1109/ACCESS.2019.2917620</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/1807.01477" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/1807.01477</a>.

</span>
<span class="ltx_bibblock">arXiv:1807.01477 [cs].

</span>
</li>
<li id="bib.bib97" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Aghajanyan et¬†al. [2020]</span>
<span class="ltx_bibblock">
Armen Aghajanyan, Luke Zettlemoyer, and Sonal Gupta.

</span>
<span class="ltx_bibblock">Intrinsic dimensionality explains the effectiveness of language model fine-tuning, 2020.

</span>
</li>
<li id="bib.bib98" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Dettmers et¬†al. [2022]</span>
<span class="ltx_bibblock">
Tim Dettmers, Mike Lewis, Sam Shleifer, and Luke Zettlemoyer.

</span>
<span class="ltx_bibblock">8-bit Optimizers via Block-wise Quantization, June 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2110.02861" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2110.02861</a>.

</span>
<span class="ltx_bibblock">arXiv:2110.02861 [cs].

</span>
</li>
<li id="bib.bib99" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Salhofer et¬†al. [2022]</span>
<span class="ltx_bibblock">
Eileen Salhofer, Xing¬†Lan Liu, and Roman Kern.

</span>
<span class="ltx_bibblock">Impact of Training Instance Selection on Domain-Specific Entity Extraction using BERT.

</span>
<span class="ltx_bibblock">In <em id="bib.bib99.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 2022 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies: Student Research Workshop</em>, pages 83‚Äì88, Hybrid: Seattle, Washington + Online, 2022. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2022.naacl-srw.11" title="" class="ltx_ref ltx_href">10.18653/v1/2022.naacl-srw.11</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2022.naacl-srw.11" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2022.naacl-srw.11</a>.

</span>
</li>
<li id="bib.bib100" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Prusa et¬†al. [2015]</span>
<span class="ltx_bibblock">
Joseph Prusa, Taghi¬†M. Khoshgoftaar, and Naeem Seliya.

</span>
<span class="ltx_bibblock">The Effect of Dataset Size on Training Tweet Sentiment Classifiers.

</span>
<span class="ltx_bibblock">In <em id="bib.bib100.1.1" class="ltx_emph ltx_font_italic">2015 IEEE 14th International Conference on Machine Learning and Applications (ICMLA)</em>, pages 96‚Äì102, December 2015.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1109/ICMLA.2015.22" title="" class="ltx_ref ltx_href">10.1109/ICMLA.2015.22</a>.

</span>
</li>
<li id="bib.bib101" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Cao et¬†al. [2021]</span>
<span class="ltx_bibblock">
Boxi Cao, Hongyu Lin, Xianpei Han, Le¬†Sun, Lingyong Yan, Meng Liao, Tong Xue, and Jin Xu.

</span>
<span class="ltx_bibblock">Knowledgeable or educated guess? revisiting language models as knowledge bases.

</span>
<span class="ltx_bibblock"><em id="bib.bib101.1.1" class="ltx_emph ltx_font_italic">arXiv preprint arXiv:2106.09231</em>, 2021.

</span>
</li>
<li id="bib.bib102" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Si et¬†al. [2023]</span>
<span class="ltx_bibblock">
Chenglei Si, Zhe Gan, Zhengyuan Yang, Shuohang Wang, Jianfeng Wang, Jordan Boyd-Graber, and Lijuan Wang.

</span>
<span class="ltx_bibblock">Prompting GPT-3 To Be Reliable, February 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2210.09150" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2210.09150</a>.

</span>
<span class="ltx_bibblock">arXiv:2210.09150 [cs].

</span>
</li>
<li id="bib.bib103" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mallen et¬†al. [2023]</span>
<span class="ltx_bibblock">
Alex Mallen, Akari Asai, Victor Zhong, Rajarshi Das, Daniel Khashabi, and Hannaneh Hajishirzi.

</span>
<span class="ltx_bibblock">When Not to Trust Language Models: Investigating Effectiveness of Parametric and Non-Parametric Memories, July 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2212.10511" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2212.10511</a>.

</span>
<span class="ltx_bibblock">arXiv:2212.10511 [cs].

</span>
</li>
<li id="bib.bib104" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Pu and Demberg [2023]</span>
<span class="ltx_bibblock">
Dongqi Pu and Vera Demberg.

</span>
<span class="ltx_bibblock">ChatGPT vs Human-authored Text: Insights into Controllable Text Summarization and Sentence Style Transfer, June 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2306.07799" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2306.07799</a>.

</span>
<span class="ltx_bibblock">arXiv:2306.07799 [cs].

</span>
</li>
<li id="bib.bib105" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chia et¬†al. [2023]</span>
<span class="ltx_bibblock">
Yew¬†Ken Chia, Pengfei Hong, Lidong Bing, and Soujanya Poria.

</span>
<span class="ltx_bibblock">INSTRUCTEVAL: Towards Holistic Evaluation of Instruction-Tuned Large Language Models, June 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2306.04757" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2306.04757</a>.

</span>
<span class="ltx_bibblock">arXiv:2306.04757 [cs].

</span>
</li>
<li id="bib.bib106" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Xu et¬†al. [2022]</span>
<span class="ltx_bibblock">
Xin Xu, Xiang Chen, Ningyu Zhang, Xin Xie, Xi¬†Chen, and Huajun Chen.

</span>
<span class="ltx_bibblock">Towards Realistic Low-resource Relation Extraction: A Benchmark with Empirical Baseline Study.

</span>
<span class="ltx_bibblock">In <em id="bib.bib106.1.1" class="ltx_emph ltx_font_italic">Findings of the Association for Computational Linguistics: EMNLP 2022</em>, pages 413‚Äì427, Abu Dhabi, United Arab Emirates, 2022. Association for Computational Linguistics.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2022.findings-emnlp.29" title="" class="ltx_ref ltx_href">10.18653/v1/2022.findings-emnlp.29</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://aclanthology.org/2022.findings-emnlp.29" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://aclanthology.org/2022.findings-emnlp.29</a>.

</span>
</li>
<li id="bib.bib107" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhao et¬†al. [2021]</span>
<span class="ltx_bibblock">
Tony¬†Z. Zhao, Eric Wallace, Shi Feng, Dan Klein, and Sameer Singh.

</span>
<span class="ltx_bibblock">Calibrate Before Use: Improving Few-Shot Performance of Language Models, June 2021.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2102.09690" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2102.09690</a>.

</span>
<span class="ltx_bibblock">arXiv:2102.09690 [cs].

</span>
</li>
<li id="bib.bib108" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wu et¬†al. [2022]</span>
<span class="ltx_bibblock">
Zhiyong Wu, Yaoxiang Wang, Jiacheng Ye, and Lingpeng Kong.

</span>
<span class="ltx_bibblock">Self-adaptive In-context Learning, December 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2212.10375" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2212.10375</a>.

</span>
<span class="ltx_bibblock">arXiv:2212.10375 [cs].

</span>
</li>
<li id="bib.bib109" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Guti√©rrez et¬†al. [2022]</span>
<span class="ltx_bibblock">
Bernal¬†Jim√©nez Guti√©rrez, Nikolas McNeal, Clay Washington, You Chen, Lang Li, Huan Sun, and Yu¬†Su.

</span>
<span class="ltx_bibblock">Thinking about GPT-3 In-Context Learning for Biomedical IE? Think Again, November 2022.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2203.08410" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2203.08410</a>.

</span>
<span class="ltx_bibblock">arXiv:2203.08410 [cs].

</span>
</li>
<li id="bib.bib110" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Workshop et¬†al. [2023]</span>
<span class="ltx_bibblock">
BigScience Workshop, Teven¬†Le Scao, Angela Fan, Christopher Akiki, Ellie Pavlick, Suzana Iliƒá, Daniel Hesslow, Roman Castagn√©, Alexandra¬†Sasha Luccioni, Fran√ßois Yvon, Matthias Gall√©, Jonathan Tow, Alexander¬†M. Rush, Stella Biderman, Albert Webson, Pawan¬†Sasanka Ammanamanchi, Thomas Wang, Beno√Æt Sagot, Niklas Muennighoff, Albert¬†Villanova del Moral, Olatunji Ruwase, Rachel Bawden, Stas Bekman, Angelina McMillan-Major, Iz¬†Beltagy, Huu Nguyen, Lucile Saulnier, Samson Tan, Pedro¬†Ortiz Suarez, Victor Sanh, Hugo Lauren√ßon, Yacine Jernite, Julien Launay, Margaret Mitchell, Colin Raffel, Aaron Gokaslan, Adi Simhi, Aitor Soroa, Alham¬†Fikri Aji, Amit Alfassy, Anna Rogers, Ariel¬†Kreisberg Nitzav, Canwen Xu, Chenghao Mou, Chris Emezue, Christopher Klamm, Colin Leong, Daniel van Strien, David¬†Ifeoluwa Adelani, Dragomir Radev, Eduardo¬†Gonz√°lez Ponferrada, Efrat Levkovizh, Ethan Kim, Eyal¬†Bar Natan, Francesco De¬†Toni, G√©rard Dupont, Germ√°n Kruszewski, Giada Pistilli, Hady Elsahar, Hamza Benyamina, Hieu Tran, Ian
Yu, Idris Abdulmumin, Isaac Johnson, Itziar Gonzalez-Dios, Javier de¬†la Rosa, Jenny Chim, Jesse Dodge, Jian Zhu, Jonathan Chang, J√∂rg Frohberg, Joseph Tobing, Joydeep Bhattacharjee, Khalid Almubarak, Kimbo Chen, Kyle Lo, Leandro Von¬†Werra, Leon Weber, Long Phan, Loubna¬†Ben allal, Ludovic Tanguy, Manan Dey, Manuel¬†Romero Mu√±oz, Maraim Masoud, Mar√≠a Grandury, Mario ≈†a≈°ko, Max Huang, Maximin Coavoux, Mayank Singh, Mike Tian-Jian Jiang, Minh¬†Chien Vu, Mohammad¬†A. Jauhar, Mustafa Ghaleb, Nishant Subramani, Nora Kassner, Nurulaqilla Khamis, Olivier Nguyen, Omar Espejel, Ona de¬†Gibert, Paulo Villegas, Peter Henderson, Pierre Colombo, Priscilla Amuok, Quentin Lhoest, Rheza Harliman, Rishi Bommasani, Roberto¬†Luis L√≥pez, Rui Ribeiro, Salomey Osei, Sampo Pyysalo, Sebastian Nagel, Shamik Bose, Shamsuddeen¬†Hassan Muhammad, Shanya Sharma, Shayne Longpre, Somaieh Nikpoor, Stanislav Silberberg, Suhas Pai, Sydney Zink, Tiago¬†Timponi Torrent, Timo Schick, Tristan Thrush, Valentin Danchev, Vassilina Nikoulina, Veronika
Laippala, Violette Lepercq, Vrinda Prabhu, Zaid Alyafeai, Zeerak Talat, Arun Raja, Benjamin Heinzerling, Chenglei Si, Davut¬†Emre Ta≈üar, Elizabeth Salesky, Sabrina¬†J. Mielke, Wilson¬†Y. Lee, Abheesht Sharma, Andrea Santilli, Antoine Chaffin, Arnaud Stiegler, Debajyoti Datta, Eliza Szczechla, Gunjan Chhablani, Han Wang, Harshit Pandey, Hendrik Strobelt, Jason¬†Alan Fries, Jos Rozen, Leo Gao, Lintang Sutawika, M.¬†Saiful Bari, Maged¬†S. Al-shaibani, Matteo Manica, Nihal Nayak, Ryan Teehan, Samuel Albanie, Sheng Shen, Srulik Ben-David, Stephen¬†H. Bach, Taewoon Kim, Tali Bers, Thibault Fevry, Trishala Neeraj, Urmish Thakker, Vikas Raunak, Xiangru Tang, Zheng-Xin Yong, Zhiqing Sun, Shaked Brody, Yallow Uri, Hadar Tojarieh, Adam Roberts, Hyung¬†Won Chung, Jaesung Tae, Jason Phang, Ofir Press, Conglong Li, Deepak Narayanan, Hatim Bourfoune, Jared Casper, Jeff Rasley, Max Ryabinin, Mayank Mishra, Minjia Zhang, Mohammad Shoeybi, Myriam Peyrounette, Nicolas Patry, Nouamane Tazi, Omar Sanseviero, Patrick von Platen, Pierre
Cornette, Pierre¬†Fran√ßois Lavall√©e, R√©mi Lacroix, Samyam Rajbhandari, Sanchit Gandhi, Shaden Smith, St√©phane Requena, Suraj Patil, Tim Dettmers, Ahmed Baruwa, Amanpreet Singh, Anastasia Cheveleva, Anne-Laure Ligozat, Arjun Subramonian, Aur√©lie N√©v√©ol, Charles Lovering, Dan Garrette, Deepak Tunuguntla, Ehud Reiter, Ekaterina Taktasheva, Ekaterina Voloshina, Eli Bogdanov, Genta¬†Indra Winata, Hailey Schoelkopf, Jan-Christoph Kalo, Jekaterina Novikova, Jessica¬†Zosa Forde, Jordan Clive, Jungo Kasai, Ken Kawamura, Liam Hazan, Marine Carpuat, Miruna Clinciu, Najoung Kim, Newton Cheng, Oleg Serikov, Omer Antverg, Oskar van¬†der Wal, Rui Zhang, Ruochen Zhang, Sebastian Gehrmann, Shachar Mirkin, Shani Pais, Tatiana Shavrina, Thomas Scialom, Tian Yun, Tomasz Limisiewicz, Verena Rieser, Vitaly Protasov, Vladislav Mikhailov, Yada Pruksachatkun, Yonatan Belinkov, Zachary Bamberger, Zdenƒõk Kasner, Alice Rueda, Amanda Pestana, Amir Feizpour, Ammar Khan, Amy Faranak, Ana Santos, Anthony Hevia, Antigona Unldreaj,
Arash Aghagol, Arezoo Abdollahi, Aycha Tammour, Azadeh HajiHosseini, Bahareh Behroozi, Benjamin Ajibade, Bharat Saxena, Carlos¬†Mu√±oz Ferrandis, Daniel McDuff, Danish Contractor, David Lansky, Davis David, Douwe Kiela, Duong¬†A. Nguyen, Edward Tan, Emi Baylor, Ezinwanne Ozoani, Fatima Mirza, Frankline Ononiwu, Habib Rezanejad, Hessie Jones, Indrani Bhattacharya, Irene Solaiman, Irina Sedenko, Isar Nejadgholi, Jesse Passmore, Josh Seltzer, Julio¬†Bonis Sanz, Livia Dutra, Mairon Samagaio, Maraim Elbadri, Margot Mieskes, Marissa Gerchick, Martha Akinlolu, Michael McKenna, Mike Qiu, Muhammed Ghauri, Mykola Burynok, Nafis Abrar, Nazneen Rajani, Nour Elkott, Nour Fahmy, Olanrewaju Samuel, Ran An, Rasmus Kromann, Ryan Hao, Samira Alizadeh, Sarmad Shubber, Silas Wang, Sourav Roy, Sylvain Viguier, Thanh Le, Tobi Oyebade, Trieu Le, Yoyo Yang, Zach Nguyen, Abhinav¬†Ramesh Kashyap, Alfredo Palasciano, Alison Callahan, Anima Shukla, Antonio Miranda-Escalada, Ayush Singh, Benjamin Beilharz, Bo¬†Wang, Caio Brito, Chenxi Zhou,
Chirag Jain, Chuxin Xu, Cl√©mentine Fourrier, Daniel¬†Le√≥n Peri√±√°n, Daniel Molano, Dian Yu, Enrique Manjavacas, Fabio Barth, Florian Fuhrimann, Gabriel Altay, Giyaseddin Bayrak, Gully Burns, Helena¬†U. Vrabec, Imane Bello, Ishani Dash, Jihyun Kang, John Giorgi, Jonas Golde, Jose¬†David Posada, Karthik¬†Rangasai Sivaraman, Lokesh Bulchandani, Lu¬†Liu, Luisa Shinzato, Madeleine¬†Hahn de¬†Bykhovetz, Maiko Takeuchi, Marc P√†mies, Maria¬†A. Castillo, Marianna Nezhurina, Mario S√§nger, Matthias Samwald, Michael Cullan, Michael Weinberg, Michiel De¬†Wolf, Mina Mihaljcic, Minna Liu, Moritz Freidank, Myungsun Kang, Natasha Seelam, Nathan Dahlberg, Nicholas¬†Michio Broad, Nikolaus Muellner, Pascale Fung, Patrick Haller, Ramya Chandrasekhar, Renata Eisenberg, Robert Martin, Rodrigo Canalli, Rosaline Su, Ruisi Su, Samuel Cahyawijaya, Samuele Garda, Shlok¬†S. Deshmukh, Shubhanshu Mishra, Sid Kiblawi, Simon Ott, Sinee Sang-aroonsiri, Srishti Kumar, Stefan Schweter, Sushil Bharati, Tanmay Laud, Th√©o Gigant, Tomoya Kainuma,
Wojciech Kusa, Yanis Labrak, Yash¬†Shailesh Bajaj, Yash Venkatraman, Yifan Xu, Yingxin Xu, Yu¬†Xu, Zhe Tan, Zhongli Xie, Zifan Ye, Mathilde Bras, Younes Belkada, and Thomas Wolf.

</span>
<span class="ltx_bibblock">BLOOM: A 176B-Parameter Open-Access Multilingual Language Model, June 2023.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2211.05100" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2211.05100</a>.

</span>
<span class="ltx_bibblock">arXiv:2211.05100 [cs].

</span>
</li>
<li id="bib.bib111" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li et¬†al. [2022]</span>
<span class="ltx_bibblock">
Xu¬†Li, Jing Ren, Wen Zhang, Zhiming Zhang, Jinchao Yu, Jiawei Wu, He¬†Sun, Shuiping Zhou, Kaijing Yan, Xijun Yan, and Wenjia Wang.

</span>
<span class="ltx_bibblock">LTM-TCM: A comprehensive database for the linking of Traditional Chinese Medicine with modern medicine at molecular and phenotypic levels.

</span>
<span class="ltx_bibblock"><em id="bib.bib111.1.1" class="ltx_emph ltx_font_italic">Pharmacological Research</em>, 178:106185, April 2022.

</span>
<span class="ltx_bibblock">ISSN 1043-6618.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.1016/j.phrs.2022.106185" title="" class="ltx_ref ltx_href">10.1016/j.phrs.2022.106185</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S104366182200130X" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.sciencedirect.com/science/article/pii/S104366182200130X</a>.

</span>
</li>
<li id="bib.bib112" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Akiba et¬†al. [2019]</span>
<span class="ltx_bibblock">
Takuya Akiba, Shotaro Sano, Toshihiko Yanase, Takeru Ohta, and Masanori Koyama.

</span>
<span class="ltx_bibblock">Optuna: A next-generation hyperparameter optimization framework, 2019.

</span>
</li>
<li id="bib.bib113" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mangrulkar et¬†al. [2022]</span>
<span class="ltx_bibblock">
Sourab Mangrulkar, Sylvain Gugger, Lysandre Debut, Younes Belkada, Sayak Paul, and Benjamin Bossan.

</span>
<span class="ltx_bibblock">Peft: State-of-the-art parameter-efficient fine-tuning methods.

</span>
<span class="ltx_bibblock"><a target="_blank" href="https://github.com/huggingface/peft" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/huggingface/peft</a>, 2022.

</span>
</li>
<li id="bib.bib114" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chung et¬†al. [2023]</span>
<span class="ltx_bibblock">
John Joon¬†Young Chung, Ece Kamar, and Saleema Amershi.

</span>
<span class="ltx_bibblock">Increasing Diversity While Maintaining Accuracy: Text Data Generation with Large Language Models and Human Interventions.

</span>
<span class="ltx_bibblock">In <em id="bib.bib114.1.1" class="ltx_emph ltx_font_italic">Proceedings of the 61st Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers)</em>, pages 575‚Äì593, 2023.

</span>
<span class="ltx_bibblock">doi:<a target="_blank" href="https://doi.org/10.18653/v1/2023.acl-long.34" title="" class="ltx_ref ltx_href">10.18653/v1/2023.acl-long.34</a>.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="http://arxiv.org/abs/2306.04140" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://arxiv.org/abs/2306.04140</a>.

</span>
<span class="ltx_bibblock">arXiv:2306.04140 [cs].

</span>
</li>
<li id="bib.bib115" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Grootendorst [2020]</span>
<span class="ltx_bibblock">
Maarten Grootendorst.

</span>
<span class="ltx_bibblock">Keybert: Minimal keyword extraction with bert., 2020.

</span>
<span class="ltx_bibblock">URL <a target="_blank" href="https://doi.org/10.5281/zenodo.4461265" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://doi.org/10.5281/zenodo.4461265</a>.

</span>
</li>
</ul>
</section>
<div class="ltx_pagination ltx_role_newpage"></div>
<section id="S1a" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">S1 </span>Experimental setup and implementation details</h2>

<section id="S1.SS1a" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S1.1 </span>Few-shot learning details</h3>

<div id="S1.SS1a.p1" class="ltx_para ltx_noindent">
<p id="S1.SS1a.p1.2" class="ltx_p">Considering their particular fine-tuning, small adjustments were provided to the prompt presented in Figure <a href="#S1.F1a" title="Figure S1 ‚Ä£ S1.1 Few-shot learning details ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S1</span></a> for Alpaca-7B and Vicuna-13B. All models were also quantized for memory efficient inferences (Table <a href="#S1.T1" title="Table S1 ‚Ä£ S1.4 Evaluation details ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S1</span></a>). Considering our available resources, we were not able to use the <span id="S1.SS1a.p1.2.1" class="ltx_text ltx_font_typewriter">q8</span> (8 bits) quantization for LLaMA models <math id="S1.SS1a.p1.1.m1.1" class="ltx_Math" alttext="&gt;13B" display="inline"><semantics id="S1.SS1a.p1.1.m1.1a"><mrow id="S1.SS1a.p1.1.m1.1.1" xref="S1.SS1a.p1.1.m1.1.1.cmml"><mi id="S1.SS1a.p1.1.m1.1.1.2" xref="S1.SS1a.p1.1.m1.1.1.2.cmml"></mi><mo id="S1.SS1a.p1.1.m1.1.1.1" xref="S1.SS1a.p1.1.m1.1.1.1.cmml">&gt;</mo><mrow id="S1.SS1a.p1.1.m1.1.1.3" xref="S1.SS1a.p1.1.m1.1.1.3.cmml"><mn id="S1.SS1a.p1.1.m1.1.1.3.2" xref="S1.SS1a.p1.1.m1.1.1.3.2.cmml">13</mn><mo lspace="0em" rspace="0em" id="S1.SS1a.p1.1.m1.1.1.3.1" xref="S1.SS1a.p1.1.m1.1.1.3.1.cmml">‚Äã</mo><mi id="S1.SS1a.p1.1.m1.1.1.3.3" xref="S1.SS1a.p1.1.m1.1.1.3.3.cmml">B</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.SS1a.p1.1.m1.1b"><apply id="S1.SS1a.p1.1.m1.1.1.cmml" xref="S1.SS1a.p1.1.m1.1.1"><gt id="S1.SS1a.p1.1.m1.1.1.1.cmml" xref="S1.SS1a.p1.1.m1.1.1.1"></gt><csymbol cd="latexml" id="S1.SS1a.p1.1.m1.1.1.2.cmml" xref="S1.SS1a.p1.1.m1.1.1.2">absent</csymbol><apply id="S1.SS1a.p1.1.m1.1.1.3.cmml" xref="S1.SS1a.p1.1.m1.1.1.3"><times id="S1.SS1a.p1.1.m1.1.1.3.1.cmml" xref="S1.SS1a.p1.1.m1.1.1.3.1"></times><cn type="integer" id="S1.SS1a.p1.1.m1.1.1.3.2.cmml" xref="S1.SS1a.p1.1.m1.1.1.3.2">13</cn><ci id="S1.SS1a.p1.1.m1.1.1.3.3.cmml" xref="S1.SS1a.p1.1.m1.1.1.3.3">ùêµ</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS1a.p1.1.m1.1c">&gt;13B</annotation></semantics></math> and improvements in performance could then be expected. In parallel, we noticed significant performance degradations when using <math id="S1.SS1a.p1.2.m2.1" class="ltx_Math" alttext="q4" display="inline"><semantics id="S1.SS1a.p1.2.m2.1a"><mrow id="S1.SS1a.p1.2.m2.1.1" xref="S1.SS1a.p1.2.m2.1.1.cmml"><mi id="S1.SS1a.p1.2.m2.1.1.2" xref="S1.SS1a.p1.2.m2.1.1.2.cmml">q</mi><mo lspace="0em" rspace="0em" id="S1.SS1a.p1.2.m2.1.1.1" xref="S1.SS1a.p1.2.m2.1.1.1.cmml">‚Äã</mo><mn id="S1.SS1a.p1.2.m2.1.1.3" xref="S1.SS1a.p1.2.m2.1.1.3.cmml">4</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.SS1a.p1.2.m2.1b"><apply id="S1.SS1a.p1.2.m2.1.1.cmml" xref="S1.SS1a.p1.2.m2.1.1"><times id="S1.SS1a.p1.2.m2.1.1.1.cmml" xref="S1.SS1a.p1.2.m2.1.1.1"></times><ci id="S1.SS1a.p1.2.m2.1.1.2.cmml" xref="S1.SS1a.p1.2.m2.1.1.2">ùëû</ci><cn type="integer" id="S1.SS1a.p1.2.m2.1.1.3.cmml" xref="S1.SS1a.p1.2.m2.1.1.3">4</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS1a.p1.2.m2.1c">q4</annotation></semantics></math> (4 bits). We used <span id="S1.SS1a.p1.2.2" class="ltx_text ltx_font_typewriter">llama.cpp<span id="footnote11" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">11</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">11</sup><span class="ltx_tag ltx_tag_note"><span id="footnote11.1.1.1" class="ltx_text ltx_font_serif">11</span></span><a target="_blank" href="https://github.com/ggerganov/llama.cpp" title="" class="ltx_ref ltx_url">https://github.com/ggerganov/llama.cpp</a></span></span></span></span> for quantization and inferences.</p>
</div>
<figure id="S1.F1a" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/archetypical-examples-ICL.png" id="S1.F1a.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="272" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S1: </span>Few-shot learning prompt containing five archetypical examples used on the Llama 7B, 13B, 30B and 65B.</figcaption>
</figure>
</section>
<section id="S1.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S1.2 </span>Fine-tuning details</h3>

<div id="S1.SS2.p1" class="ltx_para ltx_noindent">
<p id="S1.SS2.p1.8" class="ltx_p"><cite class="ltx_cite ltx_citemacro_citet">Dettmers et¬†al. [<a href="#bib.bib73" title="" class="ltx_ref">73</a>]</cite> demonstrated the efficacy of the QLoRA approach by showing that the loss in performance due to quantization can be fully recovered through subsequent fine-tuning of the adapters and that increasing the number of adapters is crucial to match full fine-tuning performance. By exploiting the memory benefits of the <span id="S1.SS2.p1.8.1" class="ltx_text ltx_font_typewriter">NF4</span> data type, we applied LoRA adapters to all linear blocks (except the initial embeddings layer) of the BioGPT and GPT-2 models. Details on the number of trained parameters are presented in Table <a href="#S1.T2" title="Table S2 ‚Ä£ S1.4 Evaluation details ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2</span></a>. During training, the special tokens <span id="S1.SS2.p1.8.2" class="ltx_text ltx_font_typewriter">&lt;BOS&gt;</span> and <span id="S1.SS2.p1.8.3" class="ltx_text ltx_font_typewriter">&lt;EOS&gt;</span> are used to delimitate the input <math id="S1.SS2.p1.1.m1.1" class="ltx_Math" alttext="X" display="inline"><semantics id="S1.SS2.p1.1.m1.1a"><mi id="S1.SS2.p1.1.m1.1.1" xref="S1.SS2.p1.1.m1.1.1.cmml">X</mi><annotation-xml encoding="MathML-Content" id="S1.SS2.p1.1.m1.1b"><ci id="S1.SS2.p1.1.m1.1.1.cmml" xref="S1.SS2.p1.1.m1.1.1">ùëã</ci></annotation-xml><annotation encoding="application/x-tex" id="S1.SS2.p1.1.m1.1c">X</annotation></semantics></math> and the expected linearised output <math id="S1.SS2.p1.2.m2.1" class="ltx_Math" alttext="Y" display="inline"><semantics id="S1.SS2.p1.2.m2.1a"><mi id="S1.SS2.p1.2.m2.1.1" xref="S1.SS2.p1.2.m2.1.1.cmml">Y</mi><annotation-xml encoding="MathML-Content" id="S1.SS2.p1.2.m2.1b"><ci id="S1.SS2.p1.2.m2.1.1.cmml" xref="S1.SS2.p1.2.m2.1.1">ùëå</ci></annotation-xml><annotation encoding="application/x-tex" id="S1.SS2.p1.2.m2.1c">Y</annotation></semantics></math>, such as [<math id="S1.SS2.p1.3.m3.1" class="ltx_Math" alttext="X" display="inline"><semantics id="S1.SS2.p1.3.m3.1a"><mi id="S1.SS2.p1.3.m3.1.1" xref="S1.SS2.p1.3.m3.1.1.cmml">X</mi><annotation-xml encoding="MathML-Content" id="S1.SS2.p1.3.m3.1b"><ci id="S1.SS2.p1.3.m3.1.1.cmml" xref="S1.SS2.p1.3.m3.1.1">ùëã</ci></annotation-xml><annotation encoding="application/x-tex" id="S1.SS2.p1.3.m3.1c">X</annotation></semantics></math>, <span id="S1.SS2.p1.8.4" class="ltx_text ltx_font_typewriter">&lt;EOS&gt;</span> <span id="S1.SS2.p1.8.5" class="ltx_text ltx_font_typewriter">&lt;BOS&gt;</span>, <math id="S1.SS2.p1.4.m4.1" class="ltx_Math" alttext="Y" display="inline"><semantics id="S1.SS2.p1.4.m4.1a"><mi id="S1.SS2.p1.4.m4.1.1" xref="S1.SS2.p1.4.m4.1.1.cmml">Y</mi><annotation-xml encoding="MathML-Content" id="S1.SS2.p1.4.m4.1b"><ci id="S1.SS2.p1.4.m4.1.1.cmml" xref="S1.SS2.p1.4.m4.1.1">ùëå</ci></annotation-xml><annotation encoding="application/x-tex" id="S1.SS2.p1.4.m4.1c">Y</annotation></semantics></math>, <span id="S1.SS2.p1.8.6" class="ltx_text ltx_font_typewriter">&lt;EOS&gt;</span>]. The <span id="S1.SS2.p1.8.7" class="ltx_text ltx_font_typewriter">&lt;BOS&gt;</span> token triggers the RE task at inference time.
<br class="ltx_break">For BioGPT, we evaluated different hyperparameter settings using Optuna <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib112" title="" class="ltx_ref">112</a>]</cite>. We applied the TPE (Tree-structured Parzen Estimator) algorithm with a pruner on median to speed up the process. The hyperparameters tuning was done on the <span id="S1.SS2.p1.8.8" class="ltx_text ltx_font_typewriter">Diversity-synt</span> dataset, using the <span id="S1.SS2.p1.8.9" class="ltx_text ltx_markedasmath">f1-score</span> on the validation set as evaluation criteria. As in <cite class="ltx_cite ltx_citemacro_citet">Giorgi et¬†al. [<a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite>, we used greedy decoding during the hyperparameter tuning step and subsequently tune the decoding strategy on the best obtained configuration. The list of hyperparameters is presented in Table <a href="#S1.T3" title="Table S3 ‚Ä£ S1.4 Evaluation details ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S3</span></a>. No significant impact of the batch-size on the performances was noticed, although larger values tends to improve the stability of the training. As already observed in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib97" title="" class="ltx_ref">97</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite>, increasing the rank of the LoRA adapters from <math id="S1.SS2.p1.6.m6.1" class="ltx_Math" alttext="r=8" display="inline"><semantics id="S1.SS2.p1.6.m6.1a"><mrow id="S1.SS2.p1.6.m6.1.1" xref="S1.SS2.p1.6.m6.1.1.cmml"><mi id="S1.SS2.p1.6.m6.1.1.2" xref="S1.SS2.p1.6.m6.1.1.2.cmml">r</mi><mo id="S1.SS2.p1.6.m6.1.1.1" xref="S1.SS2.p1.6.m6.1.1.1.cmml">=</mo><mn id="S1.SS2.p1.6.m6.1.1.3" xref="S1.SS2.p1.6.m6.1.1.3.cmml">8</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.SS2.p1.6.m6.1b"><apply id="S1.SS2.p1.6.m6.1.1.cmml" xref="S1.SS2.p1.6.m6.1.1"><eq id="S1.SS2.p1.6.m6.1.1.1.cmml" xref="S1.SS2.p1.6.m6.1.1.1"></eq><ci id="S1.SS2.p1.6.m6.1.1.2.cmml" xref="S1.SS2.p1.6.m6.1.1.2">ùëü</ci><cn type="integer" id="S1.SS2.p1.6.m6.1.1.3.cmml" xref="S1.SS2.p1.6.m6.1.1.3">8</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS2.p1.6.m6.1c">r=8</annotation></semantics></math> to <math id="S1.SS2.p1.7.m7.1" class="ltx_Math" alttext="r=16" display="inline"><semantics id="S1.SS2.p1.7.m7.1a"><mrow id="S1.SS2.p1.7.m7.1.1" xref="S1.SS2.p1.7.m7.1.1.cmml"><mi id="S1.SS2.p1.7.m7.1.1.2" xref="S1.SS2.p1.7.m7.1.1.2.cmml">r</mi><mo id="S1.SS2.p1.7.m7.1.1.1" xref="S1.SS2.p1.7.m7.1.1.1.cmml">=</mo><mn id="S1.SS2.p1.7.m7.1.1.3" xref="S1.SS2.p1.7.m7.1.1.3.cmml">16</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.SS2.p1.7.m7.1b"><apply id="S1.SS2.p1.7.m7.1.1.cmml" xref="S1.SS2.p1.7.m7.1.1"><eq id="S1.SS2.p1.7.m7.1.1.1.cmml" xref="S1.SS2.p1.7.m7.1.1.1"></eq><ci id="S1.SS2.p1.7.m7.1.1.2.cmml" xref="S1.SS2.p1.7.m7.1.1.2">ùëü</ci><cn type="integer" id="S1.SS2.p1.7.m7.1.1.3.cmml" xref="S1.SS2.p1.7.m7.1.1.3">16</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS2.p1.7.m7.1c">r=16</annotation></semantics></math> only led to marginal improvements. The best configuration obtained for BioGPT was reused for BioGPT-Large and GPT-2. For all evaluated datasets, models were then trained during 15 epochs (10 for BioGPT-Large) with 100 warm-up steps and the best epoch was selected using the validation set. We used the available implementation of QLoRA with PEFT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib113" title="" class="ltx_ref">113</a>]</cite>. We used the recommended <math id="S1.SS2.p1.8.m8.1" class="ltx_Math" alttext="8-bits" display="inline"><semantics id="S1.SS2.p1.8.m8.1a"><mrow id="S1.SS2.p1.8.m8.1.1" xref="S1.SS2.p1.8.m8.1.1.cmml"><mn id="S1.SS2.p1.8.m8.1.1.2" xref="S1.SS2.p1.8.m8.1.1.2.cmml">8</mn><mo id="S1.SS2.p1.8.m8.1.1.1" xref="S1.SS2.p1.8.m8.1.1.1.cmml">‚àí</mo><mrow id="S1.SS2.p1.8.m8.1.1.3" xref="S1.SS2.p1.8.m8.1.1.3.cmml"><mi id="S1.SS2.p1.8.m8.1.1.3.2" xref="S1.SS2.p1.8.m8.1.1.3.2.cmml">b</mi><mo lspace="0em" rspace="0em" id="S1.SS2.p1.8.m8.1.1.3.1" xref="S1.SS2.p1.8.m8.1.1.3.1.cmml">‚Äã</mo><mi id="S1.SS2.p1.8.m8.1.1.3.3" xref="S1.SS2.p1.8.m8.1.1.3.3.cmml">i</mi><mo lspace="0em" rspace="0em" id="S1.SS2.p1.8.m8.1.1.3.1a" xref="S1.SS2.p1.8.m8.1.1.3.1.cmml">‚Äã</mo><mi id="S1.SS2.p1.8.m8.1.1.3.4" xref="S1.SS2.p1.8.m8.1.1.3.4.cmml">t</mi><mo lspace="0em" rspace="0em" id="S1.SS2.p1.8.m8.1.1.3.1b" xref="S1.SS2.p1.8.m8.1.1.3.1.cmml">‚Äã</mo><mi id="S1.SS2.p1.8.m8.1.1.3.5" xref="S1.SS2.p1.8.m8.1.1.3.5.cmml">s</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.SS2.p1.8.m8.1b"><apply id="S1.SS2.p1.8.m8.1.1.cmml" xref="S1.SS2.p1.8.m8.1.1"><minus id="S1.SS2.p1.8.m8.1.1.1.cmml" xref="S1.SS2.p1.8.m8.1.1.1"></minus><cn type="integer" id="S1.SS2.p1.8.m8.1.1.2.cmml" xref="S1.SS2.p1.8.m8.1.1.2">8</cn><apply id="S1.SS2.p1.8.m8.1.1.3.cmml" xref="S1.SS2.p1.8.m8.1.1.3"><times id="S1.SS2.p1.8.m8.1.1.3.1.cmml" xref="S1.SS2.p1.8.m8.1.1.3.1"></times><ci id="S1.SS2.p1.8.m8.1.1.3.2.cmml" xref="S1.SS2.p1.8.m8.1.1.3.2">ùëè</ci><ci id="S1.SS2.p1.8.m8.1.1.3.3.cmml" xref="S1.SS2.p1.8.m8.1.1.3.3">ùëñ</ci><ci id="S1.SS2.p1.8.m8.1.1.3.4.cmml" xref="S1.SS2.p1.8.m8.1.1.3.4">ùë°</ci><ci id="S1.SS2.p1.8.m8.1.1.3.5.cmml" xref="S1.SS2.p1.8.m8.1.1.3.5">ùë†</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS2.p1.8.m8.1c">8-bits</annotation></semantics></math> paged AdamW optimizer<span id="footnote12" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">12</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">12</sup><span class="ltx_tag ltx_tag_note">12</span><a target="_blank" href="https://github.com/TimDettmers/bitsandbytes" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/TimDettmers/bitsandbytes</a></span></span></span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib98" title="" class="ltx_ref">98</a>]</cite>.
<br class="ltx_break">Similarly, we tuned the hyperparameters of Seq2rel on the <span id="S1.SS2.p1.8.10" class="ltx_text ltx_font_typewriter">Diversity-synt</span> dataset (See Table <a href="#S1.T4" title="Table S4 ‚Ä£ S1.4 Evaluation details ‚Ä£ S1 Experimental setup and implementation details ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S4</span></a>). All the fine-tuning experiments were conducted on an NVIDIA GeForce RTX 3090.</p>
</div>
</section>
<section id="S1.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S1.3 </span>Main-findings verbalisation patterns</h3>

<div id="S1.SS3.p1" class="ltx_para ltx_noindent">
<p id="S1.SS3.p1.5" class="ltx_p">To emulate different patterns of expression of the NP relationships, 5 transformations are applied: (1) chemical class replacement, (2) derivates contraction, (3) shuffling; (4) numbering, (5) relation directionality. The findings-verbaliser module operates as a sampler, and each transformation has an assigned probability. In the conducted experiments, we used <math id="S1.SS3.p1.1.m1.1" class="ltx_Math" alttext="p_{1}=0.2" display="inline"><semantics id="S1.SS3.p1.1.m1.1a"><mrow id="S1.SS3.p1.1.m1.1.1" xref="S1.SS3.p1.1.m1.1.1.cmml"><msub id="S1.SS3.p1.1.m1.1.1.2" xref="S1.SS3.p1.1.m1.1.1.2.cmml"><mi id="S1.SS3.p1.1.m1.1.1.2.2" xref="S1.SS3.p1.1.m1.1.1.2.2.cmml">p</mi><mn id="S1.SS3.p1.1.m1.1.1.2.3" xref="S1.SS3.p1.1.m1.1.1.2.3.cmml">1</mn></msub><mo id="S1.SS3.p1.1.m1.1.1.1" xref="S1.SS3.p1.1.m1.1.1.1.cmml">=</mo><mn id="S1.SS3.p1.1.m1.1.1.3" xref="S1.SS3.p1.1.m1.1.1.3.cmml">0.2</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.SS3.p1.1.m1.1b"><apply id="S1.SS3.p1.1.m1.1.1.cmml" xref="S1.SS3.p1.1.m1.1.1"><eq id="S1.SS3.p1.1.m1.1.1.1.cmml" xref="S1.SS3.p1.1.m1.1.1.1"></eq><apply id="S1.SS3.p1.1.m1.1.1.2.cmml" xref="S1.SS3.p1.1.m1.1.1.2"><csymbol cd="ambiguous" id="S1.SS3.p1.1.m1.1.1.2.1.cmml" xref="S1.SS3.p1.1.m1.1.1.2">subscript</csymbol><ci id="S1.SS3.p1.1.m1.1.1.2.2.cmml" xref="S1.SS3.p1.1.m1.1.1.2.2">ùëù</ci><cn type="integer" id="S1.SS3.p1.1.m1.1.1.2.3.cmml" xref="S1.SS3.p1.1.m1.1.1.2.3">1</cn></apply><cn type="float" id="S1.SS3.p1.1.m1.1.1.3.cmml" xref="S1.SS3.p1.1.m1.1.1.3">0.2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS3.p1.1.m1.1c">p_{1}=0.2</annotation></semantics></math>, <math id="S1.SS3.p1.2.m2.1" class="ltx_Math" alttext="p_{2}=0.9" display="inline"><semantics id="S1.SS3.p1.2.m2.1a"><mrow id="S1.SS3.p1.2.m2.1.1" xref="S1.SS3.p1.2.m2.1.1.cmml"><msub id="S1.SS3.p1.2.m2.1.1.2" xref="S1.SS3.p1.2.m2.1.1.2.cmml"><mi id="S1.SS3.p1.2.m2.1.1.2.2" xref="S1.SS3.p1.2.m2.1.1.2.2.cmml">p</mi><mn id="S1.SS3.p1.2.m2.1.1.2.3" xref="S1.SS3.p1.2.m2.1.1.2.3.cmml">2</mn></msub><mo id="S1.SS3.p1.2.m2.1.1.1" xref="S1.SS3.p1.2.m2.1.1.1.cmml">=</mo><mn id="S1.SS3.p1.2.m2.1.1.3" xref="S1.SS3.p1.2.m2.1.1.3.cmml">0.9</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.SS3.p1.2.m2.1b"><apply id="S1.SS3.p1.2.m2.1.1.cmml" xref="S1.SS3.p1.2.m2.1.1"><eq id="S1.SS3.p1.2.m2.1.1.1.cmml" xref="S1.SS3.p1.2.m2.1.1.1"></eq><apply id="S1.SS3.p1.2.m2.1.1.2.cmml" xref="S1.SS3.p1.2.m2.1.1.2"><csymbol cd="ambiguous" id="S1.SS3.p1.2.m2.1.1.2.1.cmml" xref="S1.SS3.p1.2.m2.1.1.2">subscript</csymbol><ci id="S1.SS3.p1.2.m2.1.1.2.2.cmml" xref="S1.SS3.p1.2.m2.1.1.2.2">ùëù</ci><cn type="integer" id="S1.SS3.p1.2.m2.1.1.2.3.cmml" xref="S1.SS3.p1.2.m2.1.1.2.3">2</cn></apply><cn type="float" id="S1.SS3.p1.2.m2.1.1.3.cmml" xref="S1.SS3.p1.2.m2.1.1.3">0.9</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS3.p1.2.m2.1c">p_{2}=0.9</annotation></semantics></math>, <math id="S1.SS3.p1.3.m3.1" class="ltx_Math" alttext="p_{3}=0.25" display="inline"><semantics id="S1.SS3.p1.3.m3.1a"><mrow id="S1.SS3.p1.3.m3.1.1" xref="S1.SS3.p1.3.m3.1.1.cmml"><msub id="S1.SS3.p1.3.m3.1.1.2" xref="S1.SS3.p1.3.m3.1.1.2.cmml"><mi id="S1.SS3.p1.3.m3.1.1.2.2" xref="S1.SS3.p1.3.m3.1.1.2.2.cmml">p</mi><mn id="S1.SS3.p1.3.m3.1.1.2.3" xref="S1.SS3.p1.3.m3.1.1.2.3.cmml">3</mn></msub><mo id="S1.SS3.p1.3.m3.1.1.1" xref="S1.SS3.p1.3.m3.1.1.1.cmml">=</mo><mn id="S1.SS3.p1.3.m3.1.1.3" xref="S1.SS3.p1.3.m3.1.1.3.cmml">0.25</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.SS3.p1.3.m3.1b"><apply id="S1.SS3.p1.3.m3.1.1.cmml" xref="S1.SS3.p1.3.m3.1.1"><eq id="S1.SS3.p1.3.m3.1.1.1.cmml" xref="S1.SS3.p1.3.m3.1.1.1"></eq><apply id="S1.SS3.p1.3.m3.1.1.2.cmml" xref="S1.SS3.p1.3.m3.1.1.2"><csymbol cd="ambiguous" id="S1.SS3.p1.3.m3.1.1.2.1.cmml" xref="S1.SS3.p1.3.m3.1.1.2">subscript</csymbol><ci id="S1.SS3.p1.3.m3.1.1.2.2.cmml" xref="S1.SS3.p1.3.m3.1.1.2.2">ùëù</ci><cn type="integer" id="S1.SS3.p1.3.m3.1.1.2.3.cmml" xref="S1.SS3.p1.3.m3.1.1.2.3">3</cn></apply><cn type="float" id="S1.SS3.p1.3.m3.1.1.3.cmml" xref="S1.SS3.p1.3.m3.1.1.3">0.25</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS3.p1.3.m3.1c">p_{3}=0.25</annotation></semantics></math> and <math id="S1.SS3.p1.4.m4.1" class="ltx_Math" alttext="p_{4}=0.9" display="inline"><semantics id="S1.SS3.p1.4.m4.1a"><mrow id="S1.SS3.p1.4.m4.1.1" xref="S1.SS3.p1.4.m4.1.1.cmml"><msub id="S1.SS3.p1.4.m4.1.1.2" xref="S1.SS3.p1.4.m4.1.1.2.cmml"><mi id="S1.SS3.p1.4.m4.1.1.2.2" xref="S1.SS3.p1.4.m4.1.1.2.2.cmml">p</mi><mn id="S1.SS3.p1.4.m4.1.1.2.3" xref="S1.SS3.p1.4.m4.1.1.2.3.cmml">4</mn></msub><mo id="S1.SS3.p1.4.m4.1.1.1" xref="S1.SS3.p1.4.m4.1.1.1.cmml">=</mo><mn id="S1.SS3.p1.4.m4.1.1.3" xref="S1.SS3.p1.4.m4.1.1.3.cmml">0.9</mn></mrow><annotation-xml encoding="MathML-Content" id="S1.SS3.p1.4.m4.1b"><apply id="S1.SS3.p1.4.m4.1.1.cmml" xref="S1.SS3.p1.4.m4.1.1"><eq id="S1.SS3.p1.4.m4.1.1.1.cmml" xref="S1.SS3.p1.4.m4.1.1.1"></eq><apply id="S1.SS3.p1.4.m4.1.1.2.cmml" xref="S1.SS3.p1.4.m4.1.1.2"><csymbol cd="ambiguous" id="S1.SS3.p1.4.m4.1.1.2.1.cmml" xref="S1.SS3.p1.4.m4.1.1.2">subscript</csymbol><ci id="S1.SS3.p1.4.m4.1.1.2.2.cmml" xref="S1.SS3.p1.4.m4.1.1.2.2">ùëù</ci><cn type="integer" id="S1.SS3.p1.4.m4.1.1.2.3.cmml" xref="S1.SS3.p1.4.m4.1.1.2.3">4</cn></apply><cn type="float" id="S1.SS3.p1.4.m4.1.1.3.cmml" xref="S1.SS3.p1.4.m4.1.1.3">0.9</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS3.p1.4.m4.1c">p_{4}=0.9</annotation></semantics></math> for the corresponding transformations. The values were empirically estimated from observed behaviours in the literature. To enhance the diversity of the generated abstracts, the temperature parameter is also randomly sampled at each generation: <math id="S1.SS3.p1.5.m5.4" class="ltx_Math" alttext="t\in\{0.5,0.6,0.7,0.8\}" display="inline"><semantics id="S1.SS3.p1.5.m5.4a"><mrow id="S1.SS3.p1.5.m5.4.5" xref="S1.SS3.p1.5.m5.4.5.cmml"><mi id="S1.SS3.p1.5.m5.4.5.2" xref="S1.SS3.p1.5.m5.4.5.2.cmml">t</mi><mo id="S1.SS3.p1.5.m5.4.5.1" xref="S1.SS3.p1.5.m5.4.5.1.cmml">‚àà</mo><mrow id="S1.SS3.p1.5.m5.4.5.3.2" xref="S1.SS3.p1.5.m5.4.5.3.1.cmml"><mo stretchy="false" id="S1.SS3.p1.5.m5.4.5.3.2.1" xref="S1.SS3.p1.5.m5.4.5.3.1.cmml">{</mo><mn id="S1.SS3.p1.5.m5.1.1" xref="S1.SS3.p1.5.m5.1.1.cmml">0.5</mn><mo id="S1.SS3.p1.5.m5.4.5.3.2.2" xref="S1.SS3.p1.5.m5.4.5.3.1.cmml">,</mo><mn id="S1.SS3.p1.5.m5.2.2" xref="S1.SS3.p1.5.m5.2.2.cmml">0.6</mn><mo id="S1.SS3.p1.5.m5.4.5.3.2.3" xref="S1.SS3.p1.5.m5.4.5.3.1.cmml">,</mo><mn id="S1.SS3.p1.5.m5.3.3" xref="S1.SS3.p1.5.m5.3.3.cmml">0.7</mn><mo id="S1.SS3.p1.5.m5.4.5.3.2.4" xref="S1.SS3.p1.5.m5.4.5.3.1.cmml">,</mo><mn id="S1.SS3.p1.5.m5.4.4" xref="S1.SS3.p1.5.m5.4.4.cmml">0.8</mn><mo stretchy="false" id="S1.SS3.p1.5.m5.4.5.3.2.5" xref="S1.SS3.p1.5.m5.4.5.3.1.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.SS3.p1.5.m5.4b"><apply id="S1.SS3.p1.5.m5.4.5.cmml" xref="S1.SS3.p1.5.m5.4.5"><in id="S1.SS3.p1.5.m5.4.5.1.cmml" xref="S1.SS3.p1.5.m5.4.5.1"></in><ci id="S1.SS3.p1.5.m5.4.5.2.cmml" xref="S1.SS3.p1.5.m5.4.5.2">ùë°</ci><set id="S1.SS3.p1.5.m5.4.5.3.1.cmml" xref="S1.SS3.p1.5.m5.4.5.3.2"><cn type="float" id="S1.SS3.p1.5.m5.1.1.cmml" xref="S1.SS3.p1.5.m5.1.1">0.5</cn><cn type="float" id="S1.SS3.p1.5.m5.2.2.cmml" xref="S1.SS3.p1.5.m5.2.2">0.6</cn><cn type="float" id="S1.SS3.p1.5.m5.3.3.cmml" xref="S1.SS3.p1.5.m5.3.3">0.7</cn><cn type="float" id="S1.SS3.p1.5.m5.4.4.cmml" xref="S1.SS3.p1.5.m5.4.4">0.8</cn></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.SS3.p1.5.m5.4c">t\in\{0.5,0.6,0.7,0.8\}</annotation></semantics></math>, as similarly evaluated by <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib114" title="" class="ltx_ref">114</a>]</cite>. Similarly to few-shot learning, we also used <span id="S1.SS3.p1.5.1" class="ltx_text ltx_font_typewriter">llama.cpp</span> through the python bindings library <span id="S1.SS3.p1.5.2" class="ltx_text ltx_font_typewriter">llama-cpp-python<span id="footnote13" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">13</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">13</sup><span class="ltx_tag ltx_tag_note"><span id="footnote13.1.1.1" class="ltx_text ltx_font_serif">13</span></span><a target="_blank" href="https://github.com/abetlen/llama-cpp-python" title="" class="ltx_ref ltx_url">https://github.com/abetlen/llama-cpp-python</a></span></span></span></span> for inference in generating the synthetic abstracts.</p>
</div>
</section>
<section id="S1.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S1.4 </span>Evaluation details</h3>

<div id="S1.SS4.p1" class="ltx_para ltx_noindent">
<p id="S1.SS4.p1.1" class="ltx_p">All evaluated models (in fine-tuning and few-shot settings) were evaluated for end-to-end RE, jointly performing NER and RE, framed as a generative task. The performances of the tested models were assessed by measuring the <span id="S1.SS4.p1.1.1" class="ltx_text ltx_font_typewriter">f1-score</span> over the predicted relations extracted from the decoded outputs. An extracted relation is considered correct only if the head (an organism) and the tail (a chemical) entities exactly match the ground-truth labels.</p>
</div>
<figure id="S1.T1" class="ltx_table">
<table id="S1.T1.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S1.T1.1.1" class="ltx_tr">
<td id="S1.T1.1.1.1" class="ltx_td ltx_align_left ltx_border_tt"><span id="S1.T1.1.1.1.1" class="ltx_text ltx_font_bold">model</span></td>
<td id="S1.T1.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S1.T1.1.1.2.1" class="ltx_text ltx_font_bold">Quantization type (size in GB)</span></td>
</tr>
<tr id="S1.T1.1.2" class="ltx_tr">
<td id="S1.T1.1.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S1.T1.1.2.1.1" class="ltx_text ltx_font_bold">Llama-7B</span></td>
<td id="S1.T1.1.2.2" class="ltx_td ltx_align_right ltx_border_t">q8 (6.8 GB)</td>
</tr>
<tr id="S1.T1.1.3" class="ltx_tr">
<td id="S1.T1.1.3.1" class="ltx_td ltx_align_left"><span id="S1.T1.1.3.1.1" class="ltx_text ltx_font_bold">Llama-13B</span></td>
<td id="S1.T1.1.3.2" class="ltx_td ltx_align_right">q8 (13.2 GB)</td>
</tr>
<tr id="S1.T1.1.4" class="ltx_tr">
<td id="S1.T1.1.4.1" class="ltx_td ltx_align_left"><span id="S1.T1.1.4.1.1" class="ltx_text ltx_font_bold">Llama-30B</span></td>
<td id="S1.T1.1.4.2" class="ltx_td ltx_align_right">q5_K_M (21.9GB)</td>
</tr>
<tr id="S1.T1.1.5" class="ltx_tr">
<td id="S1.T1.1.5.1" class="ltx_td ltx_align_left"><span id="S1.T1.1.5.1.1" class="ltx_text ltx_font_bold">Llama-65B</span></td>
<td id="S1.T1.1.5.2" class="ltx_td ltx_align_right">q5_K_M (44.1GB)</td>
</tr>
<tr id="S1.T1.1.6" class="ltx_tr">
<td id="S1.T1.1.6.1" class="ltx_td ltx_align_left"><span id="S1.T1.1.6.1.1" class="ltx_text ltx_font_bold">Alpaca-7B</span></td>
<td id="S1.T1.1.6.2" class="ltx_td ltx_align_right">q8 (6.8 GB)</td>
</tr>
<tr id="S1.T1.1.7" class="ltx_tr">
<td id="S1.T1.1.7.1" class="ltx_td ltx_align_left ltx_border_bb"><span id="S1.T1.1.7.1.1" class="ltx_text ltx_font_bold">Vicuna-13B</span></td>
<td id="S1.T1.1.7.2" class="ltx_td ltx_align_right ltx_border_bb">q8 (13.2 GB)</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S1: </span>Table of quantizations procedures applied on the used LLMs. See <a target="_blank" href="https://github.com/ggerganov/llama.cpp" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://github.com/ggerganov/llama.cpp</a>.</figcaption>
</figure>
<figure id="S1.T2" class="ltx_table">
<table id="S1.T2.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S1.T2.1.1" class="ltx_tr">
<td id="S1.T2.1.1.1" class="ltx_td ltx_border_tt"></td>
<td id="S1.T2.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S1.T2.1.1.2.1" class="ltx_text ltx_font_bold">Total parameters</span></td>
<td id="S1.T2.1.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S1.T2.1.1.3.1" class="ltx_text ltx_font_bold">Trainable parameters</span></td>
</tr>
<tr id="S1.T2.1.2" class="ltx_tr">
<td id="S1.T2.1.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S1.T2.1.2.1.1" class="ltx_text ltx_font_bold">Seq2rel</span></td>
<td id="S1.T2.1.2.2" class="ltx_td ltx_align_right ltx_border_t">118546185</td>
<td id="S1.T2.1.2.3" class="ltx_td ltx_align_right ltx_border_t">118546185 (100%)</td>
</tr>
<tr id="S1.T2.1.3" class="ltx_tr">
<td id="S1.T2.1.3.1" class="ltx_td ltx_align_left"><span id="S1.T2.1.3.1.1" class="ltx_text ltx_font_bold">BioGPT</span></td>
<td id="S1.T2.1.3.2" class="ltx_td ltx_align_right">350649472</td>
<td id="S1.T2.1.3.3" class="ltx_td ltx_align_right">3886208 (1.11%)</td>
</tr>
<tr id="S1.T2.1.4" class="ltx_tr">
<td id="S1.T2.1.4.1" class="ltx_td ltx_align_left"><span id="S1.T2.1.4.1.1" class="ltx_text ltx_font_bold">BioGPT-Large</span></td>
<td id="S1.T2.1.4.2" class="ltx_td ltx_align_right">1582722536</td>
<td id="S1.T2.1.4.3" class="ltx_td ltx_align_right">11533736 (0.73%)</td>
</tr>
<tr id="S1.T2.1.5" class="ltx_tr">
<td id="S1.T2.1.5.1" class="ltx_td ltx_align_left ltx_border_bb"><span id="S1.T2.1.5.1.1" class="ltx_text ltx_font_bold">GPT-2 <span id="S1.T2.1.5.1.1.1" class="ltx_text ltx_font_italic">Medium</span></span></td>
<td id="S1.T2.1.5.2" class="ltx_td ltx_align_right ltx_border_bb">358381208</td>
<td id="S1.T2.1.5.3" class="ltx_td ltx_align_right ltx_border_bb">3555992 (0.99%)</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S2: </span>Statistics of the number of trainable parameters per evaluated models.</figcaption>
</figure>
<figure id="S1.T3" class="ltx_table">
<table id="S1.T3.15" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S1.T3.15.1" class="ltx_tr">
<td id="S1.T3.15.1.1" class="ltx_td ltx_border_tt"></td>
<td id="S1.T3.15.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S1.T3.15.1.2.1" class="ltx_text ltx_font_bold">Tuned ?</span></td>
<td id="S1.T3.15.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S1.T3.15.1.3.1" class="ltx_text ltx_font_bold">Value</span></td>
</tr>
<tr id="S1.T3.15.2" class="ltx_tr">
<td id="S1.T3.15.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S1.T3.15.2.1.1" class="ltx_text ltx_font_bold">Training</span></td>
<td id="S1.T3.15.2.2" class="ltx_td ltx_border_t"></td>
<td id="S1.T3.15.2.3" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S1.T3.15.3" class="ltx_tr">
<td id="S1.T3.15.3.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.3.1.1" class="ltx_text ltx_font_typewriter">Batch size</span></td>
<td id="S1.T3.15.3.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T3.15.3.3" class="ltx_td ltx_align_right">16 (12)</td>
</tr>
<tr id="S1.T3.15.4" class="ltx_tr">
<td id="S1.T3.15.4.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.4.1.1" class="ltx_text ltx_font_typewriter">Number of epochs</span></td>
<td id="S1.T3.15.4.2" class="ltx_td ltx_align_right">no</td>
<td id="S1.T3.15.4.3" class="ltx_td ltx_align_right">15</td>
</tr>
<tr id="S1.T3.15.5" class="ltx_tr">
<td id="S1.T3.15.5.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.5.1.1" class="ltx_text ltx_font_typewriter">LoRa r</span></td>
<td id="S1.T3.15.5.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T3.15.5.3" class="ltx_td ltx_align_right">8</td>
</tr>
<tr id="S1.T3.15.6" class="ltx_tr">
<td id="S1.T3.15.6.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.6.1.1" class="ltx_text ltx_font_typewriter">LoRa alpha</span></td>
<td id="S1.T3.15.6.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T3.15.6.3" class="ltx_td ltx_align_right">16</td>
</tr>
<tr id="S1.T3.15.7" class="ltx_tr">
<td id="S1.T3.15.7.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.7.1.1" class="ltx_text ltx_font_typewriter">Learning rate</span></td>
<td id="S1.T3.15.7.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T3.15.7.3" class="ltx_td ltx_align_right">1.00e-4</td>
</tr>
<tr id="S1.T3.15.8" class="ltx_tr">
<td id="S1.T3.15.8.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.8.1.1" class="ltx_text ltx_font_typewriter">Weight decay</span></td>
<td id="S1.T3.15.8.2" class="ltx_td ltx_align_right">no</td>
<td id="S1.T3.15.8.3" class="ltx_td ltx_align_right">0.01</td>
</tr>
<tr id="S1.T3.15.9" class="ltx_tr">
<td id="S1.T3.15.9.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.9.1.1" class="ltx_text ltx_font_typewriter">Gradient accumulation steps</span></td>
<td id="S1.T3.15.9.2" class="ltx_td ltx_align_right">no*</td>
<td id="S1.T3.15.9.3" class="ltx_td ltx_align_right">5</td>
</tr>
<tr id="S1.T3.15.10" class="ltx_tr">
<td id="S1.T3.15.10.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.10.1.1" class="ltx_text ltx_font_typewriter">LoRA dropout</span></td>
<td id="S1.T3.15.10.2" class="ltx_td ltx_align_right">no</td>
<td id="S1.T3.15.10.3" class="ltx_td ltx_align_right">0.05</td>
</tr>
<tr id="S1.T3.15.11" class="ltx_tr">
<td id="S1.T3.15.11.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.11.1.1" class="ltx_text ltx_font_typewriter">LoRa target modules</span></td>
<td id="S1.T3.15.11.2" class="ltx_td ltx_align_right">no</td>
<td id="S1.T3.15.11.3" class="ltx_td ltx_align_right">q_proj, k_proj, v_proj, out_proj, fc1, fc2, output_projection</td>
</tr>
<tr id="S1.T3.15.12" class="ltx_tr">
<td id="S1.T3.15.12.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.12.1.1" class="ltx_text ltx_font_bold">Decoding</span></td>
<td id="S1.T3.15.12.2" class="ltx_td"></td>
<td id="S1.T3.15.12.3" class="ltx_td"></td>
</tr>
<tr id="S1.T3.15.13" class="ltx_tr">
<td id="S1.T3.15.13.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.13.1.1" class="ltx_text ltx_font_typewriter">strategy</span></td>
<td id="S1.T3.15.13.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T3.15.13.3" class="ltx_td ltx_align_right">beam search</td>
</tr>
<tr id="S1.T3.15.14" class="ltx_tr">
<td id="S1.T3.15.14.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.14.1.1" class="ltx_text ltx_font_typewriter">beam size</span></td>
<td id="S1.T3.15.14.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T3.15.14.3" class="ltx_td ltx_align_right">3</td>
</tr>
<tr id="S1.T3.15.15" class="ltx_tr">
<td id="S1.T3.15.15.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.15.1.1" class="ltx_text ltx_font_typewriter">stopping criteria</span></td>
<td id="S1.T3.15.15.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T3.15.15.3" class="ltx_td ltx_align_right">never</td>
</tr>
<tr id="S1.T3.15.16" class="ltx_tr">
<td id="S1.T3.15.16.1" class="ltx_td ltx_align_left"><span id="S1.T3.15.16.1.1" class="ltx_text ltx_font_typewriter">length penality</span></td>
<td id="S1.T3.15.16.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T3.15.16.3" class="ltx_td ltx_align_right">1.5</td>
</tr>
<tr id="S1.T3.15.17" class="ltx_tr">
<td id="S1.T3.15.17.1" class="ltx_td ltx_align_left ltx_border_bb"><span id="S1.T3.15.17.1.1" class="ltx_text ltx_font_typewriter">temperature</span></td>
<td id="S1.T3.15.17.2" class="ltx_td ltx_align_right ltx_border_bb">no</td>
<td id="S1.T3.15.17.3" class="ltx_td ltx_align_right ltx_border_bb">0</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S3: </span>Hyperparameters values used for BioGPT. Hyperparameters were fine-tuned using Optuna on the <span id="S1.T3.25.1" class="ltx_text ltx_font_italic">Diversity-synt</span> dataset. Values between parentheses correspond to adaptation for BioGPT-Large. The following settings were evaluated: <span id="S1.T3.26.2" class="ltx_text ltx_font_typewriter">batch-size</span> <math id="S1.T3.8.m1.3" class="ltx_Math" alttext="\in\{4,8,16\}" display="inline"><semantics id="S1.T3.8.m1.3b"><mrow id="S1.T3.8.m1.3.4" xref="S1.T3.8.m1.3.4.cmml"><mi id="S1.T3.8.m1.3.4.2" xref="S1.T3.8.m1.3.4.2.cmml"></mi><mo id="S1.T3.8.m1.3.4.1" xref="S1.T3.8.m1.3.4.1.cmml">‚àà</mo><mrow id="S1.T3.8.m1.3.4.3.2" xref="S1.T3.8.m1.3.4.3.1.cmml"><mo stretchy="false" id="S1.T3.8.m1.3.4.3.2.1" xref="S1.T3.8.m1.3.4.3.1.cmml">{</mo><mn id="S1.T3.8.m1.1.1" xref="S1.T3.8.m1.1.1.cmml">4</mn><mo id="S1.T3.8.m1.3.4.3.2.2" xref="S1.T3.8.m1.3.4.3.1.cmml">,</mo><mn id="S1.T3.8.m1.2.2" xref="S1.T3.8.m1.2.2.cmml">8</mn><mo id="S1.T3.8.m1.3.4.3.2.3" xref="S1.T3.8.m1.3.4.3.1.cmml">,</mo><mn id="S1.T3.8.m1.3.3" xref="S1.T3.8.m1.3.3.cmml">16</mn><mo stretchy="false" id="S1.T3.8.m1.3.4.3.2.4" xref="S1.T3.8.m1.3.4.3.1.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.T3.8.m1.3c"><apply id="S1.T3.8.m1.3.4.cmml" xref="S1.T3.8.m1.3.4"><in id="S1.T3.8.m1.3.4.1.cmml" xref="S1.T3.8.m1.3.4.1"></in><csymbol cd="latexml" id="S1.T3.8.m1.3.4.2.cmml" xref="S1.T3.8.m1.3.4.2">absent</csymbol><set id="S1.T3.8.m1.3.4.3.1.cmml" xref="S1.T3.8.m1.3.4.3.2"><cn type="integer" id="S1.T3.8.m1.1.1.cmml" xref="S1.T3.8.m1.1.1">4</cn><cn type="integer" id="S1.T3.8.m1.2.2.cmml" xref="S1.T3.8.m1.2.2">8</cn><cn type="integer" id="S1.T3.8.m1.3.3.cmml" xref="S1.T3.8.m1.3.3">16</cn></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.T3.8.m1.3d">\in\{4,8,16\}</annotation></semantics></math>; <span id="S1.T3.27.3" class="ltx_text ltx_font_typewriter">learning-rate</span> <math id="S1.T3.9.m2.2" class="ltx_Math" alttext="\in[1e-6;1e-3]" display="inline"><semantics id="S1.T3.9.m2.2b"><mrow id="S1.T3.9.m2.2.2" xref="S1.T3.9.m2.2.2.cmml"><mi id="S1.T3.9.m2.2.2.4" xref="S1.T3.9.m2.2.2.4.cmml"></mi><mo id="S1.T3.9.m2.2.2.3" xref="S1.T3.9.m2.2.2.3.cmml">‚àà</mo><mrow id="S1.T3.9.m2.2.2.2.2" xref="S1.T3.9.m2.2.2.2.3.cmml"><mo stretchy="false" id="S1.T3.9.m2.2.2.2.2.3" xref="S1.T3.9.m2.2.2.2.3.cmml">[</mo><mrow id="S1.T3.9.m2.1.1.1.1.1" xref="S1.T3.9.m2.1.1.1.1.1.cmml"><mrow id="S1.T3.9.m2.1.1.1.1.1.2" xref="S1.T3.9.m2.1.1.1.1.1.2.cmml"><mn id="S1.T3.9.m2.1.1.1.1.1.2.2" xref="S1.T3.9.m2.1.1.1.1.1.2.2.cmml">1</mn><mo lspace="0em" rspace="0em" id="S1.T3.9.m2.1.1.1.1.1.2.1" xref="S1.T3.9.m2.1.1.1.1.1.2.1.cmml">‚Äã</mo><mi id="S1.T3.9.m2.1.1.1.1.1.2.3" xref="S1.T3.9.m2.1.1.1.1.1.2.3.cmml">e</mi></mrow><mo id="S1.T3.9.m2.1.1.1.1.1.1" xref="S1.T3.9.m2.1.1.1.1.1.1.cmml">‚àí</mo><mn id="S1.T3.9.m2.1.1.1.1.1.3" xref="S1.T3.9.m2.1.1.1.1.1.3.cmml">6</mn></mrow><mo id="S1.T3.9.m2.2.2.2.2.4" xref="S1.T3.9.m2.2.2.2.3.cmml">;</mo><mrow id="S1.T3.9.m2.2.2.2.2.2" xref="S1.T3.9.m2.2.2.2.2.2.cmml"><mrow id="S1.T3.9.m2.2.2.2.2.2.2" xref="S1.T3.9.m2.2.2.2.2.2.2.cmml"><mn id="S1.T3.9.m2.2.2.2.2.2.2.2" xref="S1.T3.9.m2.2.2.2.2.2.2.2.cmml">1</mn><mo lspace="0em" rspace="0em" id="S1.T3.9.m2.2.2.2.2.2.2.1" xref="S1.T3.9.m2.2.2.2.2.2.2.1.cmml">‚Äã</mo><mi id="S1.T3.9.m2.2.2.2.2.2.2.3" xref="S1.T3.9.m2.2.2.2.2.2.2.3.cmml">e</mi></mrow><mo id="S1.T3.9.m2.2.2.2.2.2.1" xref="S1.T3.9.m2.2.2.2.2.2.1.cmml">‚àí</mo><mn id="S1.T3.9.m2.2.2.2.2.2.3" xref="S1.T3.9.m2.2.2.2.2.2.3.cmml">3</mn></mrow><mo stretchy="false" id="S1.T3.9.m2.2.2.2.2.5" xref="S1.T3.9.m2.2.2.2.3.cmml">]</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.T3.9.m2.2c"><apply id="S1.T3.9.m2.2.2.cmml" xref="S1.T3.9.m2.2.2"><in id="S1.T3.9.m2.2.2.3.cmml" xref="S1.T3.9.m2.2.2.3"></in><csymbol cd="latexml" id="S1.T3.9.m2.2.2.4.cmml" xref="S1.T3.9.m2.2.2.4">absent</csymbol><list id="S1.T3.9.m2.2.2.2.3.cmml" xref="S1.T3.9.m2.2.2.2.2"><apply id="S1.T3.9.m2.1.1.1.1.1.cmml" xref="S1.T3.9.m2.1.1.1.1.1"><minus id="S1.T3.9.m2.1.1.1.1.1.1.cmml" xref="S1.T3.9.m2.1.1.1.1.1.1"></minus><apply id="S1.T3.9.m2.1.1.1.1.1.2.cmml" xref="S1.T3.9.m2.1.1.1.1.1.2"><times id="S1.T3.9.m2.1.1.1.1.1.2.1.cmml" xref="S1.T3.9.m2.1.1.1.1.1.2.1"></times><cn type="integer" id="S1.T3.9.m2.1.1.1.1.1.2.2.cmml" xref="S1.T3.9.m2.1.1.1.1.1.2.2">1</cn><ci id="S1.T3.9.m2.1.1.1.1.1.2.3.cmml" xref="S1.T3.9.m2.1.1.1.1.1.2.3">ùëí</ci></apply><cn type="integer" id="S1.T3.9.m2.1.1.1.1.1.3.cmml" xref="S1.T3.9.m2.1.1.1.1.1.3">6</cn></apply><apply id="S1.T3.9.m2.2.2.2.2.2.cmml" xref="S1.T3.9.m2.2.2.2.2.2"><minus id="S1.T3.9.m2.2.2.2.2.2.1.cmml" xref="S1.T3.9.m2.2.2.2.2.2.1"></minus><apply id="S1.T3.9.m2.2.2.2.2.2.2.cmml" xref="S1.T3.9.m2.2.2.2.2.2.2"><times id="S1.T3.9.m2.2.2.2.2.2.2.1.cmml" xref="S1.T3.9.m2.2.2.2.2.2.2.1"></times><cn type="integer" id="S1.T3.9.m2.2.2.2.2.2.2.2.cmml" xref="S1.T3.9.m2.2.2.2.2.2.2.2">1</cn><ci id="S1.T3.9.m2.2.2.2.2.2.2.3.cmml" xref="S1.T3.9.m2.2.2.2.2.2.2.3">ùëí</ci></apply><cn type="integer" id="S1.T3.9.m2.2.2.2.2.2.3.cmml" xref="S1.T3.9.m2.2.2.2.2.2.3">3</cn></apply></list></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.T3.9.m2.2d">\in[1e-6;1e-3]</annotation></semantics></math>; <span id="S1.T3.28.4" class="ltx_text ltx_font_typewriter">LoRA configurations</span> <math id="S1.T3.10.m3.1" class="ltx_math_unparsed" alttext="\in\{(r=4,\alpha=4),(r=4,\alpha=8),(r=4,\alpha=16),(r=8,\alpha=8),(r=8,\alpha=16),(r=8,\alpha=32),(r=16,\alpha=16),(r=16,\alpha=32),(r=,16\alpha=64)\}" display="inline"><semantics id="S1.T3.10.m3.1b"><mrow id="S1.T3.10.m3.1c"><mo id="S1.T3.10.m3.1.1">‚àà</mo><mrow id="S1.T3.10.m3.1.2"><mo stretchy="false" id="S1.T3.10.m3.1.2.1">{</mo><mrow id="S1.T3.10.m3.1.2.2"><mo stretchy="false" id="S1.T3.10.m3.1.2.2.1">(</mo><mi id="S1.T3.10.m3.1.2.2.2">r</mi><mo id="S1.T3.10.m3.1.2.2.3">=</mo><mn id="S1.T3.10.m3.1.2.2.4">4</mn><mo id="S1.T3.10.m3.1.2.2.5">,</mo><mi id="S1.T3.10.m3.1.2.2.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.2.7">=</mo><mn id="S1.T3.10.m3.1.2.2.8">4</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.2.9">)</mo></mrow><mo id="S1.T3.10.m3.1.2.3">,</mo><mrow id="S1.T3.10.m3.1.2.4"><mo stretchy="false" id="S1.T3.10.m3.1.2.4.1">(</mo><mi id="S1.T3.10.m3.1.2.4.2">r</mi><mo id="S1.T3.10.m3.1.2.4.3">=</mo><mn id="S1.T3.10.m3.1.2.4.4">4</mn><mo id="S1.T3.10.m3.1.2.4.5">,</mo><mi id="S1.T3.10.m3.1.2.4.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.4.7">=</mo><mn id="S1.T3.10.m3.1.2.4.8">8</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.4.9">)</mo></mrow><mo id="S1.T3.10.m3.1.2.5">,</mo><mrow id="S1.T3.10.m3.1.2.6"><mo stretchy="false" id="S1.T3.10.m3.1.2.6.1">(</mo><mi id="S1.T3.10.m3.1.2.6.2">r</mi><mo id="S1.T3.10.m3.1.2.6.3">=</mo><mn id="S1.T3.10.m3.1.2.6.4">4</mn><mo id="S1.T3.10.m3.1.2.6.5">,</mo><mi id="S1.T3.10.m3.1.2.6.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.6.7">=</mo><mn id="S1.T3.10.m3.1.2.6.8">16</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.6.9">)</mo></mrow><mo id="S1.T3.10.m3.1.2.7">,</mo><mrow id="S1.T3.10.m3.1.2.8"><mo stretchy="false" id="S1.T3.10.m3.1.2.8.1">(</mo><mi id="S1.T3.10.m3.1.2.8.2">r</mi><mo id="S1.T3.10.m3.1.2.8.3">=</mo><mn id="S1.T3.10.m3.1.2.8.4">8</mn><mo id="S1.T3.10.m3.1.2.8.5">,</mo><mi id="S1.T3.10.m3.1.2.8.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.8.7">=</mo><mn id="S1.T3.10.m3.1.2.8.8">8</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.8.9">)</mo></mrow><mo id="S1.T3.10.m3.1.2.9">,</mo><mrow id="S1.T3.10.m3.1.2.10"><mo stretchy="false" id="S1.T3.10.m3.1.2.10.1">(</mo><mi id="S1.T3.10.m3.1.2.10.2">r</mi><mo id="S1.T3.10.m3.1.2.10.3">=</mo><mn id="S1.T3.10.m3.1.2.10.4">8</mn><mo id="S1.T3.10.m3.1.2.10.5">,</mo><mi id="S1.T3.10.m3.1.2.10.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.10.7">=</mo><mn id="S1.T3.10.m3.1.2.10.8">16</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.10.9">)</mo></mrow><mo id="S1.T3.10.m3.1.2.11">,</mo><mrow id="S1.T3.10.m3.1.2.12"><mo stretchy="false" id="S1.T3.10.m3.1.2.12.1">(</mo><mi id="S1.T3.10.m3.1.2.12.2">r</mi><mo id="S1.T3.10.m3.1.2.12.3">=</mo><mn id="S1.T3.10.m3.1.2.12.4">8</mn><mo id="S1.T3.10.m3.1.2.12.5">,</mo><mi id="S1.T3.10.m3.1.2.12.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.12.7">=</mo><mn id="S1.T3.10.m3.1.2.12.8">32</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.12.9">)</mo></mrow><mo id="S1.T3.10.m3.1.2.13">,</mo><mrow id="S1.T3.10.m3.1.2.14"><mo stretchy="false" id="S1.T3.10.m3.1.2.14.1">(</mo><mi id="S1.T3.10.m3.1.2.14.2">r</mi><mo id="S1.T3.10.m3.1.2.14.3">=</mo><mn id="S1.T3.10.m3.1.2.14.4">16</mn><mo id="S1.T3.10.m3.1.2.14.5">,</mo><mi id="S1.T3.10.m3.1.2.14.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.14.7">=</mo><mn id="S1.T3.10.m3.1.2.14.8">16</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.14.9">)</mo></mrow><mo id="S1.T3.10.m3.1.2.15">,</mo><mrow id="S1.T3.10.m3.1.2.16"><mo stretchy="false" id="S1.T3.10.m3.1.2.16.1">(</mo><mi id="S1.T3.10.m3.1.2.16.2">r</mi><mo id="S1.T3.10.m3.1.2.16.3">=</mo><mn id="S1.T3.10.m3.1.2.16.4">16</mn><mo id="S1.T3.10.m3.1.2.16.5">,</mo><mi id="S1.T3.10.m3.1.2.16.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.16.7">=</mo><mn id="S1.T3.10.m3.1.2.16.8">32</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.16.9">)</mo></mrow><mo id="S1.T3.10.m3.1.2.17">,</mo><mrow id="S1.T3.10.m3.1.2.18"><mo stretchy="false" id="S1.T3.10.m3.1.2.18.1">(</mo><mi id="S1.T3.10.m3.1.2.18.2">r</mi><mo rspace="0em" id="S1.T3.10.m3.1.2.18.3">=</mo><mo id="S1.T3.10.m3.1.2.18.4">,</mo><mn id="S1.T3.10.m3.1.2.18.5">16</mn><mi id="S1.T3.10.m3.1.2.18.6">Œ±</mi><mo id="S1.T3.10.m3.1.2.18.7">=</mo><mn id="S1.T3.10.m3.1.2.18.8">64</mn><mo stretchy="false" id="S1.T3.10.m3.1.2.18.9">)</mo></mrow><mo stretchy="false" id="S1.T3.10.m3.1.2.19">}</mo></mrow></mrow><annotation encoding="application/x-tex" id="S1.T3.10.m3.1d">\in\{(r=4,\alpha=4),(r=4,\alpha=8),(r=4,\alpha=16),(r=8,\alpha=8),(r=8,\alpha=16),(r=8,\alpha=32),(r=16,\alpha=16),(r=16,\alpha=32),(r=,16\alpha=64)\}</annotation></semantics></math>. <span id="S1.T3.29.5" class="ltx_text ltx_font_typewriter">Gradient accumulation steps</span> values were directly scaled in inverse proportion to the <span id="S1.T3.30.6" class="ltx_text ltx_font_typewriter">batch-size</span>: <math id="S1.T3.11.m4.3" class="ltx_Math" alttext="\{20,10,5\}" display="inline"><semantics id="S1.T3.11.m4.3b"><mrow id="S1.T3.11.m4.3.4.2" xref="S1.T3.11.m4.3.4.1.cmml"><mo stretchy="false" id="S1.T3.11.m4.3.4.2.1" xref="S1.T3.11.m4.3.4.1.cmml">{</mo><mn id="S1.T3.11.m4.1.1" xref="S1.T3.11.m4.1.1.cmml">20</mn><mo id="S1.T3.11.m4.3.4.2.2" xref="S1.T3.11.m4.3.4.1.cmml">,</mo><mn id="S1.T3.11.m4.2.2" xref="S1.T3.11.m4.2.2.cmml">10</mn><mo id="S1.T3.11.m4.3.4.2.3" xref="S1.T3.11.m4.3.4.1.cmml">,</mo><mn id="S1.T3.11.m4.3.3" xref="S1.T3.11.m4.3.3.cmml">5</mn><mo stretchy="false" id="S1.T3.11.m4.3.4.2.4" xref="S1.T3.11.m4.3.4.1.cmml">}</mo></mrow><annotation-xml encoding="MathML-Content" id="S1.T3.11.m4.3c"><set id="S1.T3.11.m4.3.4.1.cmml" xref="S1.T3.11.m4.3.4.2"><cn type="integer" id="S1.T3.11.m4.1.1.cmml" xref="S1.T3.11.m4.1.1">20</cn><cn type="integer" id="S1.T3.11.m4.2.2.cmml" xref="S1.T3.11.m4.2.2">10</cn><cn type="integer" id="S1.T3.11.m4.3.3.cmml" xref="S1.T3.11.m4.3.3">5</cn></set></annotation-xml><annotation encoding="application/x-tex" id="S1.T3.11.m4.3d">\{20,10,5\}</annotation></semantics></math>. For the decoding strategies, the following settings were also evaluated: <span id="S1.T3.31.7" class="ltx_text ltx_font_typewriter">beam-size</span> <math id="S1.T3.12.m5.2" class="ltx_Math" alttext="\in\{3,5\}" display="inline"><semantics id="S1.T3.12.m5.2b"><mrow id="S1.T3.12.m5.2.3" xref="S1.T3.12.m5.2.3.cmml"><mi id="S1.T3.12.m5.2.3.2" xref="S1.T3.12.m5.2.3.2.cmml"></mi><mo id="S1.T3.12.m5.2.3.1" xref="S1.T3.12.m5.2.3.1.cmml">‚àà</mo><mrow id="S1.T3.12.m5.2.3.3.2" xref="S1.T3.12.m5.2.3.3.1.cmml"><mo stretchy="false" id="S1.T3.12.m5.2.3.3.2.1" xref="S1.T3.12.m5.2.3.3.1.cmml">{</mo><mn id="S1.T3.12.m5.1.1" xref="S1.T3.12.m5.1.1.cmml">3</mn><mo id="S1.T3.12.m5.2.3.3.2.2" xref="S1.T3.12.m5.2.3.3.1.cmml">,</mo><mn id="S1.T3.12.m5.2.2" xref="S1.T3.12.m5.2.2.cmml">5</mn><mo stretchy="false" id="S1.T3.12.m5.2.3.3.2.3" xref="S1.T3.12.m5.2.3.3.1.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.T3.12.m5.2c"><apply id="S1.T3.12.m5.2.3.cmml" xref="S1.T3.12.m5.2.3"><in id="S1.T3.12.m5.2.3.1.cmml" xref="S1.T3.12.m5.2.3.1"></in><csymbol cd="latexml" id="S1.T3.12.m5.2.3.2.cmml" xref="S1.T3.12.m5.2.3.2">absent</csymbol><set id="S1.T3.12.m5.2.3.3.1.cmml" xref="S1.T3.12.m5.2.3.3.2"><cn type="integer" id="S1.T3.12.m5.1.1.cmml" xref="S1.T3.12.m5.1.1">3</cn><cn type="integer" id="S1.T3.12.m5.2.2.cmml" xref="S1.T3.12.m5.2.2">5</cn></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.T3.12.m5.2d">\in\{3,5\}</annotation></semantics></math>; <span id="S1.T3.32.8" class="ltx_text ltx_font_typewriter">stopping criteria</span> <math id="S1.T3.13.m6.3" class="ltx_Math" alttext="\in\{\text{True},\text{False},\text{never}\}" display="inline"><semantics id="S1.T3.13.m6.3b"><mrow id="S1.T3.13.m6.3.4" xref="S1.T3.13.m6.3.4.cmml"><mi id="S1.T3.13.m6.3.4.2" xref="S1.T3.13.m6.3.4.2.cmml"></mi><mo id="S1.T3.13.m6.3.4.1" xref="S1.T3.13.m6.3.4.1.cmml">‚àà</mo><mrow id="S1.T3.13.m6.3.4.3.2" xref="S1.T3.13.m6.3.4.3.1.cmml"><mo stretchy="false" id="S1.T3.13.m6.3.4.3.2.1" xref="S1.T3.13.m6.3.4.3.1.cmml">{</mo><mtext id="S1.T3.13.m6.1.1" xref="S1.T3.13.m6.1.1a.cmml">True</mtext><mo id="S1.T3.13.m6.3.4.3.2.2" xref="S1.T3.13.m6.3.4.3.1.cmml">,</mo><mtext id="S1.T3.13.m6.2.2" xref="S1.T3.13.m6.2.2a.cmml">False</mtext><mo id="S1.T3.13.m6.3.4.3.2.3" xref="S1.T3.13.m6.3.4.3.1.cmml">,</mo><mtext id="S1.T3.13.m6.3.3" xref="S1.T3.13.m6.3.3a.cmml">never</mtext><mo stretchy="false" id="S1.T3.13.m6.3.4.3.2.4" xref="S1.T3.13.m6.3.4.3.1.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.T3.13.m6.3c"><apply id="S1.T3.13.m6.3.4.cmml" xref="S1.T3.13.m6.3.4"><in id="S1.T3.13.m6.3.4.1.cmml" xref="S1.T3.13.m6.3.4.1"></in><csymbol cd="latexml" id="S1.T3.13.m6.3.4.2.cmml" xref="S1.T3.13.m6.3.4.2">absent</csymbol><set id="S1.T3.13.m6.3.4.3.1.cmml" xref="S1.T3.13.m6.3.4.3.2"><ci id="S1.T3.13.m6.1.1a.cmml" xref="S1.T3.13.m6.1.1"><mtext id="S1.T3.13.m6.1.1.cmml" xref="S1.T3.13.m6.1.1">True</mtext></ci><ci id="S1.T3.13.m6.2.2a.cmml" xref="S1.T3.13.m6.2.2"><mtext id="S1.T3.13.m6.2.2.cmml" xref="S1.T3.13.m6.2.2">False</mtext></ci><ci id="S1.T3.13.m6.3.3a.cmml" xref="S1.T3.13.m6.3.3"><mtext id="S1.T3.13.m6.3.3.cmml" xref="S1.T3.13.m6.3.3">never</mtext></ci></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.T3.13.m6.3d">\in\{\text{True},\text{False},\text{never}\}</annotation></semantics></math>; <span id="S1.T3.33.9" class="ltx_text ltx_font_typewriter">length penality</span> <math id="S1.T3.14.m7.2" class="ltx_Math" alttext="\in[0,3]" display="inline"><semantics id="S1.T3.14.m7.2b"><mrow id="S1.T3.14.m7.2.3" xref="S1.T3.14.m7.2.3.cmml"><mi id="S1.T3.14.m7.2.3.2" xref="S1.T3.14.m7.2.3.2.cmml"></mi><mo id="S1.T3.14.m7.2.3.1" xref="S1.T3.14.m7.2.3.1.cmml">‚àà</mo><mrow id="S1.T3.14.m7.2.3.3.2" xref="S1.T3.14.m7.2.3.3.1.cmml"><mo stretchy="false" id="S1.T3.14.m7.2.3.3.2.1" xref="S1.T3.14.m7.2.3.3.1.cmml">[</mo><mn id="S1.T3.14.m7.1.1" xref="S1.T3.14.m7.1.1.cmml">0</mn><mo id="S1.T3.14.m7.2.3.3.2.2" xref="S1.T3.14.m7.2.3.3.1.cmml">,</mo><mn id="S1.T3.14.m7.2.2" xref="S1.T3.14.m7.2.2.cmml">3</mn><mo stretchy="false" id="S1.T3.14.m7.2.3.3.2.3" xref="S1.T3.14.m7.2.3.3.1.cmml">]</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.T3.14.m7.2c"><apply id="S1.T3.14.m7.2.3.cmml" xref="S1.T3.14.m7.2.3"><in id="S1.T3.14.m7.2.3.1.cmml" xref="S1.T3.14.m7.2.3.1"></in><csymbol cd="latexml" id="S1.T3.14.m7.2.3.2.cmml" xref="S1.T3.14.m7.2.3.2">absent</csymbol><interval closure="closed" id="S1.T3.14.m7.2.3.3.1.cmml" xref="S1.T3.14.m7.2.3.3.2"><cn type="integer" id="S1.T3.14.m7.1.1.cmml" xref="S1.T3.14.m7.1.1">0</cn><cn type="integer" id="S1.T3.14.m7.2.2.cmml" xref="S1.T3.14.m7.2.2">3</cn></interval></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.T3.14.m7.2d">\in[0,3]</annotation></semantics></math>.</figcaption>
</figure>
<figure id="S1.T4" class="ltx_table">
<table id="S1.T4.7" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S1.T4.7.1" class="ltx_tr">
<td id="S1.T4.7.1.1" class="ltx_td ltx_border_tt"></td>
<td id="S1.T4.7.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S1.T4.7.1.2.1" class="ltx_text ltx_font_bold">Tuned ?</span></td>
<td id="S1.T4.7.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S1.T4.7.1.3.1" class="ltx_text ltx_font_bold">Value</span></td>
</tr>
<tr id="S1.T4.7.2" class="ltx_tr">
<td id="S1.T4.7.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S1.T4.7.2.1.1" class="ltx_text ltx_font_bold">Training</span></td>
<td id="S1.T4.7.2.2" class="ltx_td ltx_border_t"></td>
<td id="S1.T4.7.2.3" class="ltx_td ltx_align_right ltx_border_t">//</td>
</tr>
<tr id="S1.T4.7.3" class="ltx_tr">
<td id="S1.T4.7.3.1" class="ltx_td ltx_align_left">decodr learning rate</td>
<td id="S1.T4.7.3.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T4.7.3.3" class="ltx_td ltx_align_right">9.00e-4</td>
</tr>
<tr id="S1.T4.7.4" class="ltx_tr">
<td id="S1.T4.7.4.1" class="ltx_td ltx_align_left">batch size</td>
<td id="S1.T4.7.4.2" class="ltx_td ltx_align_right">no</td>
<td id="S1.T4.7.4.3" class="ltx_td ltx_align_right">4</td>
</tr>
<tr id="S1.T4.7.5" class="ltx_tr">
<td id="S1.T4.7.5.1" class="ltx_td ltx_align_left">number of epochs</td>
<td id="S1.T4.7.5.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T4.7.5.3" class="ltx_td ltx_align_right">20</td>
</tr>
<tr id="S1.T4.7.6" class="ltx_tr">
<td id="S1.T4.7.6.1" class="ltx_td ltx_align_left">gradient accumulation steps</td>
<td id="S1.T4.7.6.2" class="ltx_td ltx_align_right">no</td>
<td id="S1.T4.7.6.3" class="ltx_td ltx_align_right">10</td>
</tr>
<tr id="S1.T4.7.7" class="ltx_tr">
<td id="S1.T4.7.7.1" class="ltx_td ltx_align_left">others</td>
<td id="S1.T4.7.7.2" class="ltx_td ltx_align_right">no</td>
<td id="S1.T4.7.7.3" class="ltx_td ltx_align_right">identifical to seq2rel‚Äôs CDR config</td>
</tr>
<tr id="S1.T4.7.8" class="ltx_tr">
<td id="S1.T4.7.8.1" class="ltx_td ltx_align_left"><span id="S1.T4.7.8.1.1" class="ltx_text ltx_font_bold">Decoding</span></td>
<td id="S1.T4.7.8.2" class="ltx_td"></td>
<td id="S1.T4.7.8.3" class="ltx_td ltx_align_right">//</td>
</tr>
<tr id="S1.T4.7.9" class="ltx_tr">
<td id="S1.T4.7.9.1" class="ltx_td ltx_align_left">beam size</td>
<td id="S1.T4.7.9.2" class="ltx_td ltx_align_right">yes</td>
<td id="S1.T4.7.9.3" class="ltx_td ltx_align_right">5</td>
</tr>
<tr id="S1.T4.7.10" class="ltx_tr">
<td id="S1.T4.7.10.1" class="ltx_td ltx_align_left ltx_border_bb">length penality</td>
<td id="S1.T4.7.10.2" class="ltx_td ltx_align_right ltx_border_bb">yes</td>
<td id="S1.T4.7.10.3" class="ltx_td ltx_align_right ltx_border_bb">1</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S4: </span>Hyperparameters values used for Seq2rel. Hyperparameters were fine-tuned using Optuna on the <span id="S1.T4.12.1" class="ltx_text ltx_font_italic">Diversity-synt</span> dataset. All non-mentioned parameters were set according to the CDR configuration in the original Seq2rel‚Äôs paper. The following configurations were evaluated: decoder‚Äôs <span id="S1.T4.13.2" class="ltx_text ltx_font_typewriter">learning-rate</span> <math id="S1.T4.4.m1.2" class="ltx_Math" alttext="\in[1e-6;1e-3]" display="inline"><semantics id="S1.T4.4.m1.2b"><mrow id="S1.T4.4.m1.2.2" xref="S1.T4.4.m1.2.2.cmml"><mi id="S1.T4.4.m1.2.2.4" xref="S1.T4.4.m1.2.2.4.cmml"></mi><mo id="S1.T4.4.m1.2.2.3" xref="S1.T4.4.m1.2.2.3.cmml">‚àà</mo><mrow id="S1.T4.4.m1.2.2.2.2" xref="S1.T4.4.m1.2.2.2.3.cmml"><mo stretchy="false" id="S1.T4.4.m1.2.2.2.2.3" xref="S1.T4.4.m1.2.2.2.3.cmml">[</mo><mrow id="S1.T4.4.m1.1.1.1.1.1" xref="S1.T4.4.m1.1.1.1.1.1.cmml"><mrow id="S1.T4.4.m1.1.1.1.1.1.2" xref="S1.T4.4.m1.1.1.1.1.1.2.cmml"><mn id="S1.T4.4.m1.1.1.1.1.1.2.2" xref="S1.T4.4.m1.1.1.1.1.1.2.2.cmml">1</mn><mo lspace="0em" rspace="0em" id="S1.T4.4.m1.1.1.1.1.1.2.1" xref="S1.T4.4.m1.1.1.1.1.1.2.1.cmml">‚Äã</mo><mi id="S1.T4.4.m1.1.1.1.1.1.2.3" xref="S1.T4.4.m1.1.1.1.1.1.2.3.cmml">e</mi></mrow><mo id="S1.T4.4.m1.1.1.1.1.1.1" xref="S1.T4.4.m1.1.1.1.1.1.1.cmml">‚àí</mo><mn id="S1.T4.4.m1.1.1.1.1.1.3" xref="S1.T4.4.m1.1.1.1.1.1.3.cmml">6</mn></mrow><mo id="S1.T4.4.m1.2.2.2.2.4" xref="S1.T4.4.m1.2.2.2.3.cmml">;</mo><mrow id="S1.T4.4.m1.2.2.2.2.2" xref="S1.T4.4.m1.2.2.2.2.2.cmml"><mrow id="S1.T4.4.m1.2.2.2.2.2.2" xref="S1.T4.4.m1.2.2.2.2.2.2.cmml"><mn id="S1.T4.4.m1.2.2.2.2.2.2.2" xref="S1.T4.4.m1.2.2.2.2.2.2.2.cmml">1</mn><mo lspace="0em" rspace="0em" id="S1.T4.4.m1.2.2.2.2.2.2.1" xref="S1.T4.4.m1.2.2.2.2.2.2.1.cmml">‚Äã</mo><mi id="S1.T4.4.m1.2.2.2.2.2.2.3" xref="S1.T4.4.m1.2.2.2.2.2.2.3.cmml">e</mi></mrow><mo id="S1.T4.4.m1.2.2.2.2.2.1" xref="S1.T4.4.m1.2.2.2.2.2.1.cmml">‚àí</mo><mn id="S1.T4.4.m1.2.2.2.2.2.3" xref="S1.T4.4.m1.2.2.2.2.2.3.cmml">3</mn></mrow><mo stretchy="false" id="S1.T4.4.m1.2.2.2.2.5" xref="S1.T4.4.m1.2.2.2.3.cmml">]</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.T4.4.m1.2c"><apply id="S1.T4.4.m1.2.2.cmml" xref="S1.T4.4.m1.2.2"><in id="S1.T4.4.m1.2.2.3.cmml" xref="S1.T4.4.m1.2.2.3"></in><csymbol cd="latexml" id="S1.T4.4.m1.2.2.4.cmml" xref="S1.T4.4.m1.2.2.4">absent</csymbol><list id="S1.T4.4.m1.2.2.2.3.cmml" xref="S1.T4.4.m1.2.2.2.2"><apply id="S1.T4.4.m1.1.1.1.1.1.cmml" xref="S1.T4.4.m1.1.1.1.1.1"><minus id="S1.T4.4.m1.1.1.1.1.1.1.cmml" xref="S1.T4.4.m1.1.1.1.1.1.1"></minus><apply id="S1.T4.4.m1.1.1.1.1.1.2.cmml" xref="S1.T4.4.m1.1.1.1.1.1.2"><times id="S1.T4.4.m1.1.1.1.1.1.2.1.cmml" xref="S1.T4.4.m1.1.1.1.1.1.2.1"></times><cn type="integer" id="S1.T4.4.m1.1.1.1.1.1.2.2.cmml" xref="S1.T4.4.m1.1.1.1.1.1.2.2">1</cn><ci id="S1.T4.4.m1.1.1.1.1.1.2.3.cmml" xref="S1.T4.4.m1.1.1.1.1.1.2.3">ùëí</ci></apply><cn type="integer" id="S1.T4.4.m1.1.1.1.1.1.3.cmml" xref="S1.T4.4.m1.1.1.1.1.1.3">6</cn></apply><apply id="S1.T4.4.m1.2.2.2.2.2.cmml" xref="S1.T4.4.m1.2.2.2.2.2"><minus id="S1.T4.4.m1.2.2.2.2.2.1.cmml" xref="S1.T4.4.m1.2.2.2.2.2.1"></minus><apply id="S1.T4.4.m1.2.2.2.2.2.2.cmml" xref="S1.T4.4.m1.2.2.2.2.2.2"><times id="S1.T4.4.m1.2.2.2.2.2.2.1.cmml" xref="S1.T4.4.m1.2.2.2.2.2.2.1"></times><cn type="integer" id="S1.T4.4.m1.2.2.2.2.2.2.2.cmml" xref="S1.T4.4.m1.2.2.2.2.2.2.2">1</cn><ci id="S1.T4.4.m1.2.2.2.2.2.2.3.cmml" xref="S1.T4.4.m1.2.2.2.2.2.2.3">ùëí</ci></apply><cn type="integer" id="S1.T4.4.m1.2.2.2.2.2.3.cmml" xref="S1.T4.4.m1.2.2.2.2.2.3">3</cn></apply></list></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.T4.4.m1.2d">\in[1e-6;1e-3]</annotation></semantics></math>; <span id="S1.T4.14.3" class="ltx_text ltx_font_typewriter">beam-size</span> <math id="S1.T4.5.m2.2" class="ltx_Math" alttext="\in[3,5]" display="inline"><semantics id="S1.T4.5.m2.2b"><mrow id="S1.T4.5.m2.2.3" xref="S1.T4.5.m2.2.3.cmml"><mi id="S1.T4.5.m2.2.3.2" xref="S1.T4.5.m2.2.3.2.cmml"></mi><mo id="S1.T4.5.m2.2.3.1" xref="S1.T4.5.m2.2.3.1.cmml">‚àà</mo><mrow id="S1.T4.5.m2.2.3.3.2" xref="S1.T4.5.m2.2.3.3.1.cmml"><mo stretchy="false" id="S1.T4.5.m2.2.3.3.2.1" xref="S1.T4.5.m2.2.3.3.1.cmml">[</mo><mn id="S1.T4.5.m2.1.1" xref="S1.T4.5.m2.1.1.cmml">3</mn><mo id="S1.T4.5.m2.2.3.3.2.2" xref="S1.T4.5.m2.2.3.3.1.cmml">,</mo><mn id="S1.T4.5.m2.2.2" xref="S1.T4.5.m2.2.2.cmml">5</mn><mo stretchy="false" id="S1.T4.5.m2.2.3.3.2.3" xref="S1.T4.5.m2.2.3.3.1.cmml">]</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.T4.5.m2.2c"><apply id="S1.T4.5.m2.2.3.cmml" xref="S1.T4.5.m2.2.3"><in id="S1.T4.5.m2.2.3.1.cmml" xref="S1.T4.5.m2.2.3.1"></in><csymbol cd="latexml" id="S1.T4.5.m2.2.3.2.cmml" xref="S1.T4.5.m2.2.3.2">absent</csymbol><interval closure="closed" id="S1.T4.5.m2.2.3.3.1.cmml" xref="S1.T4.5.m2.2.3.3.2"><cn type="integer" id="S1.T4.5.m2.1.1.cmml" xref="S1.T4.5.m2.1.1">3</cn><cn type="integer" id="S1.T4.5.m2.2.2.cmml" xref="S1.T4.5.m2.2.2">5</cn></interval></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.T4.5.m2.2d">\in[3,5]</annotation></semantics></math>; <span id="S1.T4.15.4" class="ltx_text ltx_font_typewriter">length penality</span> <math id="S1.T4.6.m3.2" class="ltx_Math" alttext="\in[1,3]" display="inline"><semantics id="S1.T4.6.m3.2b"><mrow id="S1.T4.6.m3.2.3" xref="S1.T4.6.m3.2.3.cmml"><mi id="S1.T4.6.m3.2.3.2" xref="S1.T4.6.m3.2.3.2.cmml"></mi><mo id="S1.T4.6.m3.2.3.1" xref="S1.T4.6.m3.2.3.1.cmml">‚àà</mo><mrow id="S1.T4.6.m3.2.3.3.2" xref="S1.T4.6.m3.2.3.3.1.cmml"><mo stretchy="false" id="S1.T4.6.m3.2.3.3.2.1" xref="S1.T4.6.m3.2.3.3.1.cmml">[</mo><mn id="S1.T4.6.m3.1.1" xref="S1.T4.6.m3.1.1.cmml">1</mn><mo id="S1.T4.6.m3.2.3.3.2.2" xref="S1.T4.6.m3.2.3.3.1.cmml">,</mo><mn id="S1.T4.6.m3.2.2" xref="S1.T4.6.m3.2.2.cmml">3</mn><mo stretchy="false" id="S1.T4.6.m3.2.3.3.2.3" xref="S1.T4.6.m3.2.3.3.1.cmml">]</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S1.T4.6.m3.2c"><apply id="S1.T4.6.m3.2.3.cmml" xref="S1.T4.6.m3.2.3"><in id="S1.T4.6.m3.2.3.1.cmml" xref="S1.T4.6.m3.2.3.1"></in><csymbol cd="latexml" id="S1.T4.6.m3.2.3.2.cmml" xref="S1.T4.6.m3.2.3.2">absent</csymbol><interval closure="closed" id="S1.T4.6.m3.2.3.3.1.cmml" xref="S1.T4.6.m3.2.3.3.2"><cn type="integer" id="S1.T4.6.m3.1.1.cmml" xref="S1.T4.6.m3.1.1">1</cn><cn type="integer" id="S1.T4.6.m3.2.2.cmml" xref="S1.T4.6.m3.2.2">3</cn></interval></apply></annotation-xml><annotation encoding="application/x-tex" id="S1.T4.6.m3.2d">\in[1,3]</annotation></semantics></math>.</figcaption>
</figure>
</section>
</section>
<section id="S2a" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">S2 </span>Supplementary materials</h2>

<section id="S2.SS1a" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S2.1 </span>Chemical lengths thresholding</h3>

<div id="S2.SS1a.p1" class="ltx_para ltx_noindent">
<p id="S2.SS1a.p1.1" class="ltx_p">To determine a reasonable threshold for filtering chemical labels with excessive length, we conducted a comparative analysis of the distribution of label lengths in LOTUS (derived from Wikidata) versus their corresponding IUPAC names (See supplementary Figure <a href="#S2.F2" title="Figure S2 ‚Ä£ S2.1 Chemical lengths thresholding ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S2</span></a>). While the respective median and mean values clearly suggest that most of the available chemicals are identified with common names (i.e. shorter), the long right-tail of labels exhibit a length comparable to IUPAC names. These longer labels are often too lengthy to be practical for use in training examples for the targeted RE task. By estimated the limit when 90% of the chemical labels in LOTUS are at least as long as their corresponding IUPAC name, we estimated that a threshold of 60 characters effectively filters out excessively long labels.</p>
</div>
<figure id="S2.F2" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/chemical-label-nchar.png" id="S2.F2.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="337" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S2: </span>Distribution of number of characters in chemical names between Wikidata labels and their corresponding IUPAC name. The integrated graph indicates the proportion of chemicals for which the length of the Wikidata label is longer or equal to the IUPAC name, by increasing number of characters in the Wikidata label. The black vertical line correspond to the chosen threshold at 60 characters.</figcaption>
</figure>
</section>
<section id="S2.SS2a" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S2.2 </span>Diversity-sampling: alternative comparisons</h3>

<figure id="S2.F3a" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/supp-eval-datasets.png" id="S2.F3a.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="337" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S3: </span>Additional evaluation statistics for the diversity-sampling method compared to other potential sampling procedures. In "<span id="S2.F3a.9.1" class="ltx_text ltx_font_italic">Top-organisms sample</span>", the top 500 articles with the most distinct organisms (individually) were extracted per biological kingdoms. Similarly for relations and chemicals, with "<span id="S2.F3a.10.2" class="ltx_text ltx_font_italic">Top-relations sample</span>" and "<span id="S2.F3a.11.3" class="ltx_text ltx_font_italic">Top-chemicals sample</span>" respectively. <span id="S2.F3a.12.4" class="ltx_text ltx_font_bold">A:</span> Statistics of the number of distinct organisms, relations, and chemicals in the <span id="S2.F3a.13.5" class="ltx_text ltx_font_italic">Diversity</span> sample compared to alternative samples. All samples contain 2000 articles. <span id="S2.F3a.14.6" class="ltx_text ltx_font_bold">B:</span> Distribution of the cumulative proportion of reported relations per fraction of chemicals (top) and organisms (bottom), ordered by their contributions in the different samples. <span id="S2.F3a.15.7" class="ltx_text ltx_font_bold">C:</span> Distribution of the frequency of mention in distinct articles of chemicals (top) and organisms (bottom) in the different samples.</figcaption>
</figure>
<div id="S2.SS2a.p1" class="ltx_para ltx_noindent">
<p id="S2.SS2a.p1.1" class="ltx_p">The diversity-sampling strategy was evaluated against three new possible strategies for improving the diversity in the extracted set. In <span id="S2.SS2a.p1.1.1" class="ltx_text ltx_font_italic">Top-organisms</span>, the top 500 articles with the most distinct organisms (individually) were extracted per biological kingdoms. Similarly done for relations and chemicals with <span id="S2.SS2a.p1.1.2" class="ltx_text ltx_font_italic">Top-relations</span> and <span id="S2.SS2a.p1.1.3" class="ltx_text ltx_font_italic">Top-chemicals</span>. As expected, the <span id="S2.SS2a.p1.1.4" class="ltx_text ltx_font_italic">Top-relations</span> strategy led to the largest set of distinct relations (Figures <a href="#S2.F3a" title="Figure S3 ‚Ä£ S2.2 Diversity-sampling: alternative comparisons ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S3</span></a>.A), followed by <span id="S2.SS2a.p1.1.5" class="ltx_text ltx_font_italic">Top-chemicals</span> and the proposed diversity-sampling. However, this improvement comes at the expense of a poorer diversity in terms of organisms, but also balance in their distribution (Figures <a href="#S2.F3a" title="Figure S3 ‚Ä£ S2.2 Diversity-sampling: alternative comparisons ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S3</span></a>.B and C.). Interestingly, the <span id="S2.SS2a.p1.1.6" class="ltx_text ltx_font_italic">Top-organisms</span> strategy led to a smaller set of entities compared to the diversity-sampling. This highlights that in case of imbalanced distribution of entities over the sampled items (i.e. some model organisms attract more articles than non-model organisms.), the simple <span id="S2.SS2a.p1.1.7" class="ltx_text ltx_font_italic">Top-organisms</span> strategy do not consider this potential redundancy, whereas its prevention is an explicit objective with the proposed approach. Overall, the evaluated metrics suggest that the diversity-sampling with the GME-sampler offers a valuable compromise between these alternative strategies.</p>
</div>
</section>
<section id="S2.SS3a" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S2.3 </span>Mismatches between standardized labels and original abstracts</h3>

<div id="S2.SS3a.p1" class="ltx_para ltx_noindent">
<p id="S2.SS3a.p1.1" class="ltx_p">The 7,901 available abstracts from the literature references in the <span id="S2.SS3a.p1.1.1" class="ltx_text ltx_font_italic">Extended</span> dataset were extracted using the NCBI E-utilities <span id="S2.SS3a.p1.1.2" class="ltx_text ltx_font_typewriter">efectch</span> service. All the organism labels available on Wikidata were directly matched on the abstracts. Using the PubChem exchange service, all the synonyms (direct synonyms of the molecule and synonyms of its stereoisomers) were extracted whether a PubChem ID was available. In total, 653,749 synonyms were extracted. A chemical entity was considered as mentioned in the abstract when there is an exact match of its name or one of its synonym in the abstract. Some chemicals, however, may also only be implicitly mentioned in an abstract. Indeed, the isolation of multiple derivatives, such as Atroviridin A, B and C, is typical reported as ‚ÄúAtroviridins A-C‚Äù. Then, Atroviridin B would not be explicitly mentioned and has to be infered. All chemicals which could be part of such expressions were identified using a set of regular expressions and were treated separately to not wrongly inflate the proportion of chemicals not mentioned in the abstracts. Nonetheless, it is worth mentioning that a non-negligeable part of these <span id="S2.SS3a.p1.1.3" class="ltx_text ltx_font_italic">multiple</span> chemical entities are simply not mentioned in the abstract, either explicitly or implicitly. For instance, see the original mentions of <span id="S2.SS3a.p1.1.4" class="ltx_text ltx_font_italic">malyngamide A<span id="footnote14" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">14</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">14</sup><span class="ltx_tag ltx_tag_note"><span id="footnote14.1.1.1" class="ltx_text ltx_font_upright">14</span></span><a target="_blank" href="https://www.wikidata.org/wiki/Q27135775" title="" class="ltx_ref ltx_url ltx_font_typewriter ltx_font_upright">https://www.wikidata.org/wiki/Q27135775</a></span></span></span></span> in PMID 10924193, 11076568, and 21341718.</p>
</div>
</section>
<section id="S2.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S2.4 </span>Dataset curation protocol</h3>

<div id="S2.SS4.p1" class="ltx_para ltx_noindent">
<p id="S2.SS4.p1.1" class="ltx_p"><span id="S2.SS4.p1.1.1" class="ltx_text ltx_font_bold">Biocurator: </span>The dataset was curated by a single curator with a PhD in microbiology and prior experiences in manual curation.
<br class="ltx_break"></p>
</div>
<div id="S2.SS4.p2" class="ltx_para ltx_noindent">
<p id="S2.SS4.p2.1" class="ltx_p"><span id="S2.SS4.p2.1.1" class="ltx_text ltx_font_bold">Articles selection:</span> Articles were selected using the proposed GME-sampler, by extracting the top-200 literature references which maximise the diversity of named entities. All selected articles have a PMID, an available abstract, a title, and are available online on PubMed. No filter was applied based on the journal or the publication date.
<br class="ltx_break"></p>
</div>
<div id="S2.SS4.p3" class="ltx_para ltx_noindent">
<p id="S2.SS4.p3.1" class="ltx_p"><span id="S2.SS4.p3.1.1" class="ltx_text ltx_font_bold">Objective:</span> The curator targeted the relations between organisms (<span id="S2.SS4.p3.1.2" class="ltx_text ltx_font_italic">head</span>) and their isolated natural products (<span id="S2.SS4.p3.1.3" class="ltx_text ltx_font_italic">tail</span>) in the abstracts. Only organisms and chemicals that are involved in NP relationships are extracted. For example, organisms on which the activity of a compound is tested (e.g. a pathogen like <span id="S2.SS4.p3.1.4" class="ltx_text ltx_font_italic">Bacillus cereus</span>) are not annotated. The available LOTUS annotations were always used as a starting point.
<br class="ltx_break"></p>
</div>
<div id="S2.SS4.p4" class="ltx_para ltx_noindent">
<p id="S2.SS4.p4.1" class="ltx_p"><span id="S2.SS4.p4.1.1" class="ltx_text ltx_font_bold">Annotation of chemical entities: </span> All chemical entities are categorized as either singular chemical (e.g. hispaglabridin A) or chemical classes (e.g. Isoflavanoids). The nature of these entities was cross-validated with the standard ChEBI Ontology when necessary. For singular chemicals, information about their chemical class is also extracted if it is mentioned in the article. Importantly, the label of the chemical entity is annotated as it is mentioned in the abstract. To align with the original LOTUS data, Wikidata and PubChem identifiers were assigned to chemicals and classes when available. In cases of ambiguity, the curator refers to the full-text (if available) to obtain more detailed information and assign the correct standardized entity. If the entity is not found in Wikidata, a dedicated identifier in the format "{pmid}CHEM{N}" is assigned instead, e.g. "11421752CHEM1".
<br class="ltx_break"></p>
</div>
<div id="S2.SS4.p5" class="ltx_para ltx_noindent">
<p id="S2.SS4.p5.1" class="ltx_p"><span id="S2.SS4.p5.1.1" class="ltx_text ltx_font_bold">Annotation of organism entities:</span> Similarly to chemicals, the name of the organism is annotated exactly as it appears in the abstract. When only the genus is determined (e.g. Plakinastrella sp.), the genus name serves as the label.
<br class="ltx_break"></p>
</div>
<div id="S2.SS4.p6" class="ltx_para ltx_noindent">
<p id="S2.SS4.p6.1" class="ltx_p"><span id="S2.SS4.p6.1.1" class="ltx_text ltx_font_bold">Annotation of relations:</span> The output labels only include relations explicitly mentioned in the abstract, while relations mentioned in the full-text are excluded. The relations are annotated based on their order of appearance in the abstract. If there are more than one organism, the relations of the first organism are annotated first, followed by the relations of the other organisms in order of appearance.</p>
</div>
<div id="S2.SS4.p7" class="ltx_para ltx_noindent">
<p id="S2.SS4.p7.1" class="ltx_p"><span id="S2.SS4.p7.1.1" class="ltx_text ltx_font_bold">Export:</span> The annotations are exported in a <span id="S2.SS4.p7.1.2" class="ltx_text ltx_font_typewriter">JSON</span>-format as illustrated in Figure <a href="#S2.F4" title="Figure S4 ‚Ä£ S2.4 Dataset curation protocol ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S4</span></a> along with more statistics on the annotation.</p>
</div>
<figure id="S2.F4" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/example-format-curated-set.png" id="S2.F4.1.g1" class="ltx_graphics ltx_centering ltx_img_square" width="598" height="496" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S4: </span>An example of a curated literature reference in the evaluation dataset with supplementary statistics.</figcaption>
</figure>
</section>
<section id="S2.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S2.5 </span>Details of the composition of datasets</h3>

<figure id="S2.T5" class="ltx_table">
<table id="S2.T5.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S2.T5.1.1" class="ltx_tr">
<td id="S2.T5.1.1.1" class="ltx_td ltx_border_tt"></td>
<td id="S2.T5.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T5.1.1.2.1" class="ltx_text ltx_font_bold"># Organisms</span></td>
<td id="S2.T5.1.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T5.1.1.3.1" class="ltx_text ltx_font_bold"># Chemicals</span></td>
<td id="S2.T5.1.1.4" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T5.1.1.4.1" class="ltx_text ltx_font_bold"># Relations</span></td>
<td id="S2.T5.1.1.5" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T5.1.1.5.1" class="ltx_text ltx_font_bold"># References</span></td>
</tr>
<tr id="S2.T5.1.2" class="ltx_tr">
<td id="S2.T5.1.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T5.1.2.1.1" class="ltx_text ltx_font_bold">Original dataset</span></td>
<td id="S2.T5.1.2.2" class="ltx_td ltx_align_right ltx_border_t">36803</td>
<td id="S2.T5.1.2.3" class="ltx_td ltx_align_right ltx_border_t">220783</td>
<td id="S2.T5.1.2.4" class="ltx_td ltx_align_right ltx_border_t">533347</td>
<td id="S2.T5.1.2.5" class="ltx_td ltx_align_right ltx_border_t">88810</td>
</tr>
<tr id="S2.T5.1.3" class="ltx_tr">
<td id="S2.T5.1.3.1" class="ltx_td ltx_align_left ltx_border_bb"><span id="S2.T5.1.3.1.1" class="ltx_text ltx_font_bold">Pre-processed dataset</span></td>
<td id="S2.T5.1.3.2" class="ltx_td ltx_align_right ltx_border_bb">14890</td>
<td id="S2.T5.1.3.3" class="ltx_td ltx_align_right ltx_border_bb">56310</td>
<td id="S2.T5.1.3.4" class="ltx_td ltx_align_right ltx_border_bb">102528</td>
<td id="S2.T5.1.3.5" class="ltx_td ltx_align_right ltx_border_bb">32616</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S5: </span>Impact of the pre-processing of the number of organisms, chemicals and relations.</figcaption>
</figure>
<figure id="S2.T6" class="ltx_table">
<table id="S2.T6.2" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S2.T6.2.2" class="ltx_tr">
<td id="S2.T6.2.2.3" class="ltx_td ltx_align_left ltx_border_tt"><span id="S2.T6.2.2.3.1" class="ltx_text ltx_font_bold">Kingdom</span></td>
<td id="S2.T6.2.2.4" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T6.2.2.4.1" class="ltx_text ltx_font_bold">N</span></td>
<td id="S2.T6.1.1.1" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T6.1.1.1.1" class="ltx_text ltx_font_bold">max <math id="S2.T6.1.1.1.1.m1.1" class="ltx_Math" alttext="H_{S}(O)" display="inline"><semantics id="S2.T6.1.1.1.1.m1.1a"><mrow id="S2.T6.1.1.1.1.m1.1.2" xref="S2.T6.1.1.1.1.m1.1.2.cmml"><msub id="S2.T6.1.1.1.1.m1.1.2.2" xref="S2.T6.1.1.1.1.m1.1.2.2.cmml"><mi id="S2.T6.1.1.1.1.m1.1.2.2.2" xref="S2.T6.1.1.1.1.m1.1.2.2.2.cmml">H</mi><mi id="S2.T6.1.1.1.1.m1.1.2.2.3" xref="S2.T6.1.1.1.1.m1.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.T6.1.1.1.1.m1.1.2.1" xref="S2.T6.1.1.1.1.m1.1.2.1.cmml">‚Äã</mo><mrow id="S2.T6.1.1.1.1.m1.1.2.3.2" xref="S2.T6.1.1.1.1.m1.1.2.cmml"><mo stretchy="false" id="S2.T6.1.1.1.1.m1.1.2.3.2.1" xref="S2.T6.1.1.1.1.m1.1.2.cmml">(</mo><mi id="S2.T6.1.1.1.1.m1.1.1" xref="S2.T6.1.1.1.1.m1.1.1.cmml">O</mi><mo stretchy="false" id="S2.T6.1.1.1.1.m1.1.2.3.2.2" xref="S2.T6.1.1.1.1.m1.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.T6.1.1.1.1.m1.1b"><apply id="S2.T6.1.1.1.1.m1.1.2.cmml" xref="S2.T6.1.1.1.1.m1.1.2"><times id="S2.T6.1.1.1.1.m1.1.2.1.cmml" xref="S2.T6.1.1.1.1.m1.1.2.1"></times><apply id="S2.T6.1.1.1.1.m1.1.2.2.cmml" xref="S2.T6.1.1.1.1.m1.1.2.2"><csymbol cd="ambiguous" id="S2.T6.1.1.1.1.m1.1.2.2.1.cmml" xref="S2.T6.1.1.1.1.m1.1.2.2">subscript</csymbol><ci id="S2.T6.1.1.1.1.m1.1.2.2.2.cmml" xref="S2.T6.1.1.1.1.m1.1.2.2.2">ùêª</ci><ci id="S2.T6.1.1.1.1.m1.1.2.2.3.cmml" xref="S2.T6.1.1.1.1.m1.1.2.2.3">ùëÜ</ci></apply><ci id="S2.T6.1.1.1.1.m1.1.1.cmml" xref="S2.T6.1.1.1.1.m1.1.1">ùëÇ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.T6.1.1.1.1.m1.1c">H_{S}(O)</annotation></semantics></math> (rank)</span></td>
<td id="S2.T6.2.2.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T6.2.2.2.1" class="ltx_text ltx_font_bold">max <math id="S2.T6.2.2.2.1.m1.1" class="ltx_Math" alttext="H_{S}(C)" display="inline"><semantics id="S2.T6.2.2.2.1.m1.1a"><mrow id="S2.T6.2.2.2.1.m1.1.2" xref="S2.T6.2.2.2.1.m1.1.2.cmml"><msub id="S2.T6.2.2.2.1.m1.1.2.2" xref="S2.T6.2.2.2.1.m1.1.2.2.cmml"><mi id="S2.T6.2.2.2.1.m1.1.2.2.2" xref="S2.T6.2.2.2.1.m1.1.2.2.2.cmml">H</mi><mi id="S2.T6.2.2.2.1.m1.1.2.2.3" xref="S2.T6.2.2.2.1.m1.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.T6.2.2.2.1.m1.1.2.1" xref="S2.T6.2.2.2.1.m1.1.2.1.cmml">‚Äã</mo><mrow id="S2.T6.2.2.2.1.m1.1.2.3.2" xref="S2.T6.2.2.2.1.m1.1.2.cmml"><mo stretchy="false" id="S2.T6.2.2.2.1.m1.1.2.3.2.1" xref="S2.T6.2.2.2.1.m1.1.2.cmml">(</mo><mi id="S2.T6.2.2.2.1.m1.1.1" xref="S2.T6.2.2.2.1.m1.1.1.cmml">C</mi><mo stretchy="false" id="S2.T6.2.2.2.1.m1.1.2.3.2.2" xref="S2.T6.2.2.2.1.m1.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.T6.2.2.2.1.m1.1b"><apply id="S2.T6.2.2.2.1.m1.1.2.cmml" xref="S2.T6.2.2.2.1.m1.1.2"><times id="S2.T6.2.2.2.1.m1.1.2.1.cmml" xref="S2.T6.2.2.2.1.m1.1.2.1"></times><apply id="S2.T6.2.2.2.1.m1.1.2.2.cmml" xref="S2.T6.2.2.2.1.m1.1.2.2"><csymbol cd="ambiguous" id="S2.T6.2.2.2.1.m1.1.2.2.1.cmml" xref="S2.T6.2.2.2.1.m1.1.2.2">subscript</csymbol><ci id="S2.T6.2.2.2.1.m1.1.2.2.2.cmml" xref="S2.T6.2.2.2.1.m1.1.2.2.2">ùêª</ci><ci id="S2.T6.2.2.2.1.m1.1.2.2.3.cmml" xref="S2.T6.2.2.2.1.m1.1.2.2.3">ùëÜ</ci></apply><ci id="S2.T6.2.2.2.1.m1.1.1.cmml" xref="S2.T6.2.2.2.1.m1.1.1">ùê∂</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.T6.2.2.2.1.m1.1c">H_{S}(C)</annotation></semantics></math> (rank)</span></td>
</tr>
<tr id="S2.T6.2.3" class="ltx_tr">
<td id="S2.T6.2.3.1" class="ltx_td ltx_align_left ltx_border_t">Archaeplastida</td>
<td id="S2.T6.2.3.2" class="ltx_td ltx_align_right ltx_border_t">19491</td>
<td id="S2.T6.2.3.3" class="ltx_td ltx_align_right ltx_border_t">8.73 (10512)</td>
<td id="S2.T6.2.3.4" class="ltx_td ltx_align_right ltx_border_t">9.81 (10713)</td>
</tr>
<tr id="S2.T6.2.4" class="ltx_tr">
<td id="S2.T6.2.4.1" class="ltx_td ltx_align_left">Fungi</td>
<td id="S2.T6.2.4.2" class="ltx_td ltx_align_right">5023</td>
<td id="S2.T6.2.4.3" class="ltx_td ltx_align_right">7.18 (2519)</td>
<td id="S2.T6.2.4.4" class="ltx_td ltx_align_right">9.33 (5023)</td>
</tr>
<tr id="S2.T6.2.5" class="ltx_tr">
<td id="S2.T6.2.5.1" class="ltx_td ltx_align_left">Metazoa</td>
<td id="S2.T6.2.5.2" class="ltx_td ltx_align_right">1920</td>
<td id="S2.T6.2.5.3" class="ltx_td ltx_align_right">6.72 (1304)</td>
<td id="S2.T6.2.5.4" class="ltx_td ltx_align_right">8.33 (1920)</td>
</tr>
<tr id="S2.T6.2.6" class="ltx_tr">
<td id="S2.T6.2.6.1" class="ltx_td ltx_align_left ltx_border_bb">Not Attributed (Bacteria or Algae)</td>
<td id="S2.T6.2.6.2" class="ltx_td ltx_align_right ltx_border_bb">6666</td>
<td id="S2.T6.2.6.3" class="ltx_td ltx_align_right ltx_border_bb">6.96 (2503)</td>
<td id="S2.T6.2.6.4" class="ltx_td ltx_align_right ltx_border_bb">8.90 (6666)</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S6: </span>Maximal number <math id="S2.T6.6.m1.1" class="ltx_Math" alttext="N" display="inline"><semantics id="S2.T6.6.m1.1b"><mi id="S2.T6.6.m1.1.1" xref="S2.T6.6.m1.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="S2.T6.6.m1.1c"><ci id="S2.T6.6.m1.1.1.cmml" xref="S2.T6.6.m1.1.1">ùëÅ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.T6.6.m1.1d">N</annotation></semantics></math> of literature items per kingdoms along with the value and the rank of the maximal reached entropies on organisms <math id="S2.T6.7.m2.1" class="ltx_Math" alttext="H_{S}(O)" display="inline"><semantics id="S2.T6.7.m2.1b"><mrow id="S2.T6.7.m2.1.2" xref="S2.T6.7.m2.1.2.cmml"><msub id="S2.T6.7.m2.1.2.2" xref="S2.T6.7.m2.1.2.2.cmml"><mi id="S2.T6.7.m2.1.2.2.2" xref="S2.T6.7.m2.1.2.2.2.cmml">H</mi><mi id="S2.T6.7.m2.1.2.2.3" xref="S2.T6.7.m2.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.T6.7.m2.1.2.1" xref="S2.T6.7.m2.1.2.1.cmml">‚Äã</mo><mrow id="S2.T6.7.m2.1.2.3.2" xref="S2.T6.7.m2.1.2.cmml"><mo stretchy="false" id="S2.T6.7.m2.1.2.3.2.1" xref="S2.T6.7.m2.1.2.cmml">(</mo><mi id="S2.T6.7.m2.1.1" xref="S2.T6.7.m2.1.1.cmml">O</mi><mo stretchy="false" id="S2.T6.7.m2.1.2.3.2.2" xref="S2.T6.7.m2.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.T6.7.m2.1c"><apply id="S2.T6.7.m2.1.2.cmml" xref="S2.T6.7.m2.1.2"><times id="S2.T6.7.m2.1.2.1.cmml" xref="S2.T6.7.m2.1.2.1"></times><apply id="S2.T6.7.m2.1.2.2.cmml" xref="S2.T6.7.m2.1.2.2"><csymbol cd="ambiguous" id="S2.T6.7.m2.1.2.2.1.cmml" xref="S2.T6.7.m2.1.2.2">subscript</csymbol><ci id="S2.T6.7.m2.1.2.2.2.cmml" xref="S2.T6.7.m2.1.2.2.2">ùêª</ci><ci id="S2.T6.7.m2.1.2.2.3.cmml" xref="S2.T6.7.m2.1.2.2.3">ùëÜ</ci></apply><ci id="S2.T6.7.m2.1.1.cmml" xref="S2.T6.7.m2.1.1">ùëÇ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.T6.7.m2.1d">H_{S}(O)</annotation></semantics></math> and chemicals <math id="S2.T6.8.m3.1" class="ltx_Math" alttext="H_{S}(C)" display="inline"><semantics id="S2.T6.8.m3.1b"><mrow id="S2.T6.8.m3.1.2" xref="S2.T6.8.m3.1.2.cmml"><msub id="S2.T6.8.m3.1.2.2" xref="S2.T6.8.m3.1.2.2.cmml"><mi id="S2.T6.8.m3.1.2.2.2" xref="S2.T6.8.m3.1.2.2.2.cmml">H</mi><mi id="S2.T6.8.m3.1.2.2.3" xref="S2.T6.8.m3.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.T6.8.m3.1.2.1" xref="S2.T6.8.m3.1.2.1.cmml">‚Äã</mo><mrow id="S2.T6.8.m3.1.2.3.2" xref="S2.T6.8.m3.1.2.cmml"><mo stretchy="false" id="S2.T6.8.m3.1.2.3.2.1" xref="S2.T6.8.m3.1.2.cmml">(</mo><mi id="S2.T6.8.m3.1.1" xref="S2.T6.8.m3.1.1.cmml">C</mi><mo stretchy="false" id="S2.T6.8.m3.1.2.3.2.2" xref="S2.T6.8.m3.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.T6.8.m3.1c"><apply id="S2.T6.8.m3.1.2.cmml" xref="S2.T6.8.m3.1.2"><times id="S2.T6.8.m3.1.2.1.cmml" xref="S2.T6.8.m3.1.2.1"></times><apply id="S2.T6.8.m3.1.2.2.cmml" xref="S2.T6.8.m3.1.2.2"><csymbol cd="ambiguous" id="S2.T6.8.m3.1.2.2.1.cmml" xref="S2.T6.8.m3.1.2.2">subscript</csymbol><ci id="S2.T6.8.m3.1.2.2.2.cmml" xref="S2.T6.8.m3.1.2.2.2">ùêª</ci><ci id="S2.T6.8.m3.1.2.2.3.cmml" xref="S2.T6.8.m3.1.2.2.3">ùëÜ</ci></apply><ci id="S2.T6.8.m3.1.1.cmml" xref="S2.T6.8.m3.1.1">ùê∂</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.T6.8.m3.1d">H_{S}(C)</annotation></semantics></math>.</figcaption>
</figure>
<figure id="S2.T7" class="ltx_table">
<table id="S2.T7.5" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S2.T7.5.1" class="ltx_tr">
<td id="S2.T7.5.1.1" class="ltx_td ltx_border_tt"></td>
<td id="S2.T7.5.1.2" class="ltx_td ltx_align_center ltx_border_tt" colspan="4"><span id="S2.T7.5.1.2.1" class="ltx_text ltx_font_bold">Organisms (% of Max Entropy)</span></td>
<td id="S2.T7.5.1.3" class="ltx_td ltx_align_center ltx_border_tt" colspan="4"><span id="S2.T7.5.1.3.1" class="ltx_text ltx_font_bold">Chemicals (% of Max Entropy)</span></td>
</tr>
<tr id="S2.T7.5.2" class="ltx_tr">
<td id="S2.T7.5.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T7.5.2.1.1" class="ltx_text ltx_font_bold">Kingdom</span></td>
<td id="S2.T7.5.2.2" class="ltx_td ltx_align_right ltx_border_t"><span id="S2.T7.5.2.2.1" class="ltx_text ltx_font_bold">n=250</span></td>
<td id="S2.T7.5.2.3" class="ltx_td ltx_align_right ltx_border_t"><span id="S2.T7.5.2.3.1" class="ltx_text ltx_font_bold">n=500</span></td>
<td id="S2.T7.5.2.4" class="ltx_td ltx_align_right ltx_border_t"><span id="S2.T7.5.2.4.1" class="ltx_text ltx_font_bold">n=1000</span></td>
<td id="S2.T7.5.2.5" class="ltx_td ltx_align_right ltx_border_t"><span id="S2.T7.5.2.5.1" class="ltx_text ltx_font_bold">n=2000</span></td>
<td id="S2.T7.5.2.6" class="ltx_td ltx_align_right ltx_border_t"><span id="S2.T7.5.2.6.1" class="ltx_text ltx_font_bold">n=250</span></td>
<td id="S2.T7.5.2.7" class="ltx_td ltx_align_right ltx_border_t"><span id="S2.T7.5.2.7.1" class="ltx_text ltx_font_bold">n=500</span></td>
<td id="S2.T7.5.2.8" class="ltx_td ltx_align_right ltx_border_t"><span id="S2.T7.5.2.8.1" class="ltx_text ltx_font_bold">n=1000</span></td>
<td id="S2.T7.5.2.9" class="ltx_td ltx_align_right ltx_border_t"><span id="S2.T7.5.2.9.1" class="ltx_text ltx_font_bold">n=2000</span></td>
</tr>
<tr id="S2.T7.5.3" class="ltx_tr">
<td id="S2.T7.5.3.1" class="ltx_td ltx_align_left">Archaeplastida</td>
<td id="S2.T7.5.3.2" class="ltx_td ltx_align_right">75.5</td>
<td id="S2.T7.5.3.3" class="ltx_td ltx_align_right">80.5</td>
<td id="S2.T7.5.3.4" class="ltx_td ltx_align_right">86</td>
<td id="S2.T7.5.3.5" class="ltx_td ltx_align_right">91.7</td>
<td id="S2.T7.5.3.6" class="ltx_td ltx_align_right">76.9</td>
<td id="S2.T7.5.3.7" class="ltx_td ltx_align_right">83.7</td>
<td id="S2.T7.5.3.8" class="ltx_td ltx_align_right">89.6</td>
<td id="S2.T7.5.3.9" class="ltx_td ltx_align_right">94.3</td>
</tr>
<tr id="S2.T7.5.4" class="ltx_tr">
<td id="S2.T7.5.4.1" class="ltx_td ltx_align_left">Fungi</td>
<td id="S2.T7.5.4.2" class="ltx_td ltx_align_right">80.7</td>
<td id="S2.T7.5.4.3" class="ltx_td ltx_align_right">89</td>
<td id="S2.T7.5.4.4" class="ltx_td ltx_align_right">96.6</td>
<td id="S2.T7.5.4.5" class="ltx_td ltx_align_right">99.8</td>
<td id="S2.T7.5.4.6" class="ltx_td ltx_align_right">83.7</td>
<td id="S2.T7.5.4.7" class="ltx_td ltx_align_right">88.1</td>
<td id="S2.T7.5.4.8" class="ltx_td ltx_align_right">92.3</td>
<td id="S2.T7.5.4.9" class="ltx_td ltx_align_right">95.1</td>
</tr>
<tr id="S2.T7.5.5" class="ltx_tr">
<td id="S2.T7.5.5.1" class="ltx_td ltx_align_left">Metazoa</td>
<td id="S2.T7.5.5.2" class="ltx_td ltx_align_right">84.4</td>
<td id="S2.T7.5.5.3" class="ltx_td ltx_align_right">93</td>
<td id="S2.T7.5.5.4" class="ltx_td ltx_align_right">99.5</td>
<td id="S2.T7.5.5.5" class="ltx_td ltx_align_right">96.1</td>
<td id="S2.T7.5.5.6" class="ltx_td ltx_align_right">90</td>
<td id="S2.T7.5.5.7" class="ltx_td ltx_align_right">94.6</td>
<td id="S2.T7.5.5.8" class="ltx_td ltx_align_right">97.4</td>
<td id="S2.T7.5.5.9" class="ltx_td ltx_align_right">100</td>
</tr>
<tr id="S2.T7.5.6" class="ltx_tr">
<td id="S2.T7.5.6.1" class="ltx_td ltx_align_left ltx_border_bb">Not Attributed (Bacteria or Algae)</td>
<td id="S2.T7.5.6.2" class="ltx_td ltx_align_right ltx_border_bb">82.3</td>
<td id="S2.T7.5.6.3" class="ltx_td ltx_align_right ltx_border_bb">90.7</td>
<td id="S2.T7.5.6.4" class="ltx_td ltx_align_right ltx_border_bb">96.7</td>
<td id="S2.T7.5.6.5" class="ltx_td ltx_align_right ltx_border_bb">99.9</td>
<td id="S2.T7.5.6.6" class="ltx_td ltx_align_right ltx_border_bb">85.1</td>
<td id="S2.T7.5.6.7" class="ltx_td ltx_align_right ltx_border_bb">89.6</td>
<td id="S2.T7.5.6.8" class="ltx_td ltx_align_right ltx_border_bb">93.7</td>
<td id="S2.T7.5.6.9" class="ltx_td ltx_align_right ltx_border_bb">97.2</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S7: </span>Percentage of the maximal (observed) entropies <math id="S2.T7.3.m1.1" class="ltx_Math" alttext="H_{\displaystyle S}(O)" display="inline"><semantics id="S2.T7.3.m1.1b"><mrow id="S2.T7.3.m1.1.2" xref="S2.T7.3.m1.1.2.cmml"><msub id="S2.T7.3.m1.1.2.2" xref="S2.T7.3.m1.1.2.2.cmml"><mi id="S2.T7.3.m1.1.2.2.2" xref="S2.T7.3.m1.1.2.2.2.cmml">H</mi><mi mathsize="142%" id="S2.T7.3.m1.1.2.2.3" xref="S2.T7.3.m1.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.T7.3.m1.1.2.1" xref="S2.T7.3.m1.1.2.1.cmml">‚Äã</mo><mrow id="S2.T7.3.m1.1.2.3.2" xref="S2.T7.3.m1.1.2.cmml"><mo stretchy="false" id="S2.T7.3.m1.1.2.3.2.1" xref="S2.T7.3.m1.1.2.cmml">(</mo><mi id="S2.T7.3.m1.1.1" xref="S2.T7.3.m1.1.1.cmml">O</mi><mo stretchy="false" id="S2.T7.3.m1.1.2.3.2.2" xref="S2.T7.3.m1.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.T7.3.m1.1c"><apply id="S2.T7.3.m1.1.2.cmml" xref="S2.T7.3.m1.1.2"><times id="S2.T7.3.m1.1.2.1.cmml" xref="S2.T7.3.m1.1.2.1"></times><apply id="S2.T7.3.m1.1.2.2.cmml" xref="S2.T7.3.m1.1.2.2"><csymbol cd="ambiguous" id="S2.T7.3.m1.1.2.2.1.cmml" xref="S2.T7.3.m1.1.2.2">subscript</csymbol><ci id="S2.T7.3.m1.1.2.2.2.cmml" xref="S2.T7.3.m1.1.2.2.2">ùêª</ci><ci id="S2.T7.3.m1.1.2.2.3.cmml" xref="S2.T7.3.m1.1.2.2.3">ùëÜ</ci></apply><ci id="S2.T7.3.m1.1.1.cmml" xref="S2.T7.3.m1.1.1">ùëÇ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.T7.3.m1.1d">H_{\displaystyle S}(O)</annotation></semantics></math> and <math id="S2.T7.4.m2.1" class="ltx_Math" alttext="H_{\displaystyle S}(C)" display="inline"><semantics id="S2.T7.4.m2.1b"><mrow id="S2.T7.4.m2.1.2" xref="S2.T7.4.m2.1.2.cmml"><msub id="S2.T7.4.m2.1.2.2" xref="S2.T7.4.m2.1.2.2.cmml"><mi id="S2.T7.4.m2.1.2.2.2" xref="S2.T7.4.m2.1.2.2.2.cmml">H</mi><mi mathsize="142%" id="S2.T7.4.m2.1.2.2.3" xref="S2.T7.4.m2.1.2.2.3.cmml">S</mi></msub><mo lspace="0em" rspace="0em" id="S2.T7.4.m2.1.2.1" xref="S2.T7.4.m2.1.2.1.cmml">‚Äã</mo><mrow id="S2.T7.4.m2.1.2.3.2" xref="S2.T7.4.m2.1.2.cmml"><mo stretchy="false" id="S2.T7.4.m2.1.2.3.2.1" xref="S2.T7.4.m2.1.2.cmml">(</mo><mi id="S2.T7.4.m2.1.1" xref="S2.T7.4.m2.1.1.cmml">C</mi><mo stretchy="false" id="S2.T7.4.m2.1.2.3.2.2" xref="S2.T7.4.m2.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.T7.4.m2.1c"><apply id="S2.T7.4.m2.1.2.cmml" xref="S2.T7.4.m2.1.2"><times id="S2.T7.4.m2.1.2.1.cmml" xref="S2.T7.4.m2.1.2.1"></times><apply id="S2.T7.4.m2.1.2.2.cmml" xref="S2.T7.4.m2.1.2.2"><csymbol cd="ambiguous" id="S2.T7.4.m2.1.2.2.1.cmml" xref="S2.T7.4.m2.1.2.2">subscript</csymbol><ci id="S2.T7.4.m2.1.2.2.2.cmml" xref="S2.T7.4.m2.1.2.2.2">ùêª</ci><ci id="S2.T7.4.m2.1.2.2.3.cmml" xref="S2.T7.4.m2.1.2.2.3">ùëÜ</ci></apply><ci id="S2.T7.4.m2.1.1.cmml" xref="S2.T7.4.m2.1.1">ùê∂</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.T7.4.m2.1d">H_{\displaystyle S}(C)</annotation></semantics></math> at different steps: 250, 500, 1000 and 2000 top ranked articles. </figcaption>
</figure>
<figure id="S2.T8" class="ltx_table">
<table id="S2.T8.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S2.T8.1.1.1" class="ltx_tr">
<td id="S2.T8.1.1.1.1" class="ltx_td ltx_align_left ltx_border_tt"><span id="S2.T8.1.1.1.1.1" class="ltx_text ltx_font_bold">Dataset</span></td>
<td id="S2.T8.1.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T8.1.1.1.2.1" class="ltx_text ltx_font_bold">Part.</span></td>
<th id="S2.T8.1.1.1.3" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_th_row ltx_border_tt">
<span id="S2.T8.1.1.1.3.1" class="ltx_text"></span> <span id="S2.T8.1.1.1.3.2" class="ltx_text">
<span id="S2.T8.1.1.1.3.2.1" class="ltx_tabular ltx_align_middle">
<span id="S2.T8.1.1.1.3.2.1.1" class="ltx_tr">
<span id="S2.T8.1.1.1.3.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S2.T8.1.1.1.3.2.1.1.1.1" class="ltx_text ltx_font_bold"># Relations</span></span></span>
<span id="S2.T8.1.1.1.3.2.1.2" class="ltx_tr">
<span id="S2.T8.1.1.1.3.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">(w. chem / w. class)</span></span>
</span></span><span id="S2.T8.1.1.1.3.3" class="ltx_text"></span>
</th>
<td id="S2.T8.1.1.1.4" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T8.1.1.1.4.1" class="ltx_text ltx_font_bold"># Organisms</span></td>
<th id="S2.T8.1.1.1.5" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_th_row ltx_border_tt">
<span id="S2.T8.1.1.1.5.1" class="ltx_text"></span> <span id="S2.T8.1.1.1.5.2" class="ltx_text">
<span id="S2.T8.1.1.1.5.2.1" class="ltx_tabular ltx_align_middle">
<span id="S2.T8.1.1.1.5.2.1.1" class="ltx_tr">
<span id="S2.T8.1.1.1.5.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S2.T8.1.1.1.5.2.1.1.1.1" class="ltx_text ltx_font_bold"># Chemical entities</span></span></span>
<span id="S2.T8.1.1.1.5.2.1.2" class="ltx_tr">
<span id="S2.T8.1.1.1.5.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">(chem. / class.)</span></span>
</span></span><span id="S2.T8.1.1.1.5.3" class="ltx_text"></span>
</th>
<td id="S2.T8.1.1.1.6" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T8.1.1.1.6.1" class="ltx_text ltx_font_bold"># References</span></td>
</tr>
<tr id="S2.T8.1.2.2" class="ltx_tr">
<td id="S2.T8.1.2.2.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S2.T8.1.2.2.1.1" class="ltx_text ltx_font_bold">Diversty-raw</span></td>
<td id="S2.T8.1.2.2.2" class="ltx_td ltx_align_right ltx_border_t">train</td>
<td id="S2.T8.1.2.2.3" class="ltx_td ltx_align_right ltx_border_t">12666</td>
<td id="S2.T8.1.2.2.4" class="ltx_td ltx_align_right ltx_border_t">2644</td>
<td id="S2.T8.1.2.2.5" class="ltx_td ltx_align_right ltx_border_t">10311</td>
<td id="S2.T8.1.2.2.6" class="ltx_td ltx_align_right ltx_border_t">1519*</td>
</tr>
<tr id="S2.T8.1.3.3" class="ltx_tr">
<td id="S2.T8.1.3.3.1" class="ltx_td ltx_align_right">valid</td>
<td id="S2.T8.1.3.3.2" class="ltx_td ltx_align_right">1425</td>
<td id="S2.T8.1.3.3.3" class="ltx_td ltx_align_right">301</td>
<td id="S2.T8.1.3.3.4" class="ltx_td ltx_align_right">1211</td>
<td id="S2.T8.1.3.3.5" class="ltx_td ltx_align_right">168*</td>
</tr>
<tr id="S2.T8.1.4.4" class="ltx_tr">
<td id="S2.T8.1.4.4.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S2.T8.1.4.4.1.1" class="ltx_text ltx_font_bold">Random-raw</span></td>
<td id="S2.T8.1.4.4.2" class="ltx_td ltx_align_right ltx_border_t">train</td>
<td id="S2.T8.1.4.4.3" class="ltx_td ltx_align_right ltx_border_t">5102</td>
<td id="S2.T8.1.4.4.4" class="ltx_td ltx_align_right ltx_border_t">1434</td>
<td id="S2.T8.1.4.4.5" class="ltx_td ltx_align_right ltx_border_t">4286</td>
<td id="S2.T8.1.4.4.6" class="ltx_td ltx_align_right ltx_border_t">1531*</td>
</tr>
<tr id="S2.T8.1.5.5" class="ltx_tr">
<td id="S2.T8.1.5.5.1" class="ltx_td ltx_align_right">valid</td>
<td id="S2.T8.1.5.5.2" class="ltx_td ltx_align_right">657</td>
<td id="S2.T8.1.5.5.3" class="ltx_td ltx_align_right">220</td>
<td id="S2.T8.1.5.5.4" class="ltx_td ltx_align_right">584</td>
<td id="S2.T8.1.5.5.5" class="ltx_td ltx_align_right">189*</td>
</tr>
<tr id="S2.T8.1.6.6" class="ltx_tr">
<td id="S2.T8.1.6.6.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S2.T8.1.6.6.1.1" class="ltx_text ltx_font_bold">Extended-raw</span></td>
<td id="S2.T8.1.6.6.2" class="ltx_td ltx_align_right ltx_border_t">train</td>
<td id="S2.T8.1.6.6.3" class="ltx_td ltx_align_right ltx_border_t">27952</td>
<td id="S2.T8.1.6.6.4" class="ltx_td ltx_align_right ltx_border_t">5642</td>
<td id="S2.T8.1.6.6.5" class="ltx_td ltx_align_right ltx_border_t">21028</td>
<td id="S2.T8.1.6.6.6" class="ltx_td ltx_align_right ltx_border_t">7111*</td>
</tr>
<tr id="S2.T8.1.7.7" class="ltx_tr">
<td id="S2.T8.1.7.7.1" class="ltx_td ltx_align_right">valid</td>
<td id="S2.T8.1.7.7.2" class="ltx_td ltx_align_right">3355</td>
<td id="S2.T8.1.7.7.3" class="ltx_td ltx_align_right">932</td>
<td id="S2.T8.1.7.7.4" class="ltx_td ltx_align_right">2741</td>
<td id="S2.T8.1.7.7.5" class="ltx_td ltx_align_right">790*</td>
</tr>
<tr id="S2.T8.1.8.8" class="ltx_tr">
<td id="S2.T8.1.8.8.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S2.T8.1.8.8.1.1" class="ltx_text ltx_font_bold">Full</span></td>
<td id="S2.T8.1.8.8.2" class="ltx_td ltx_align_right ltx_border_t">train</td>
<td id="S2.T8.1.8.8.3" class="ltx_td ltx_align_right ltx_border_t">90326</td>
<td id="S2.T8.1.8.8.4" class="ltx_td ltx_align_right ltx_border_t">13208</td>
<td id="S2.T8.1.8.8.5" class="ltx_td ltx_align_right ltx_border_t">51658</td>
<td id="S2.T8.1.8.8.6" class="ltx_td ltx_align_right ltx_border_t">28286</td>
</tr>
<tr id="S2.T8.1.9.9" class="ltx_tr">
<td id="S2.T8.1.9.9.1" class="ltx_td ltx_align_right">valid</td>
<td id="S2.T8.1.9.9.2" class="ltx_td ltx_align_right">1533</td>
<td id="S2.T8.1.9.9.3" class="ltx_td ltx_align_right">484</td>
<td id="S2.T8.1.9.9.4" class="ltx_td ltx_align_right">1288</td>
<td id="S2.T8.1.9.9.5" class="ltx_td ltx_align_right">430</td>
</tr>
<tr id="S2.T8.1.10.10" class="ltx_tr">
<td id="S2.T8.1.10.10.1" class="ltx_td ltx_align_left" rowspan="2"><span id="S2.T8.1.10.10.1.1" class="ltx_text ltx_font_bold">Diversity-synt</span></td>
<td id="S2.T8.1.10.10.2" class="ltx_td ltx_align_right">train</td>
<td id="S2.T8.1.10.10.3" class="ltx_td ltx_align_right">11547 (10764 / 783)</td>
<td id="S2.T8.1.10.10.4" class="ltx_td ltx_align_right">2154</td>
<td id="S2.T8.1.10.10.5" class="ltx_td ltx_align_right">(9108 / 61)</td>
<td id="S2.T8.1.10.10.6" class="ltx_td ltx_align_right">3562</td>
</tr>
<tr id="S2.T8.1.11.11" class="ltx_tr">
<td id="S2.T8.1.11.11.1" class="ltx_td ltx_align_right">valid</td>
<td id="S2.T8.1.11.11.2" class="ltx_td ltx_align_right">1197 (1096 / 101)</td>
<td id="S2.T8.1.11.11.3" class="ltx_td ltx_align_right">220</td>
<td id="S2.T8.1.11.11.4" class="ltx_td ltx_align_right">(998 / 37)</td>
<td id="S2.T8.1.11.11.5" class="ltx_td ltx_align_right">389</td>
</tr>
<tr id="S2.T8.1.12.12" class="ltx_tr">
<td id="S2.T8.1.12.12.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S2.T8.1.12.12.1.1" class="ltx_text ltx_font_bold">Random-synt</span></td>
<td id="S2.T8.1.12.12.2" class="ltx_td ltx_align_right ltx_border_t">train</td>
<td id="S2.T8.1.12.12.3" class="ltx_td ltx_align_right ltx_border_t">4825 (4474 / 351)</td>
<td id="S2.T8.1.12.12.4" class="ltx_td ltx_align_right ltx_border_t">1267</td>
<td id="S2.T8.1.12.12.5" class="ltx_td ltx_align_right ltx_border_t">(3854 / 53 )</td>
<td id="S2.T8.1.12.12.6" class="ltx_td ltx_align_right ltx_border_t">3798</td>
</tr>
<tr id="S2.T8.1.13.13" class="ltx_tr">
<td id="S2.T8.1.13.13.1" class="ltx_td ltx_align_right">valid</td>
<td id="S2.T8.1.13.13.2" class="ltx_td ltx_align_right">609 (561 / 47)</td>
<td id="S2.T8.1.13.13.3" class="ltx_td ltx_align_right">190</td>
<td id="S2.T8.1.13.13.4" class="ltx_td ltx_align_right">(507 / 22)</td>
<td id="S2.T8.1.13.13.5" class="ltx_td ltx_align_right">460</td>
</tr>
<tr id="S2.T8.1.14.14" class="ltx_tr">
<td id="S2.T8.1.14.14.1" class="ltx_td ltx_align_left ltx_border_bb ltx_border_t" rowspan="2"><span id="S2.T8.1.14.14.1.1" class="ltx_text ltx_font_bold">Extended-synt</span></td>
<td id="S2.T8.1.14.14.2" class="ltx_td ltx_align_right ltx_border_t">train</td>
<td id="S2.T8.1.14.14.3" class="ltx_td ltx_align_right ltx_border_t">28614 (26373 / 2242)</td>
<td id="S2.T8.1.14.14.4" class="ltx_td ltx_align_right ltx_border_t">5258</td>
<td id="S2.T8.1.14.14.5" class="ltx_td ltx_align_right ltx_border_t">(20404 / 69)</td>
<td id="S2.T8.1.14.14.6" class="ltx_td ltx_align_right ltx_border_t">23985</td>
</tr>
<tr id="S2.T8.1.15.15" class="ltx_tr">
<td id="S2.T8.1.15.15.1" class="ltx_td ltx_align_right ltx_border_bb">valid</td>
<td id="S2.T8.1.15.15.2" class="ltx_td ltx_align_right ltx_border_bb">1444 (1332 / 112)</td>
<td id="S2.T8.1.15.15.3" class="ltx_td ltx_align_right ltx_border_bb">432</td>
<td id="S2.T8.1.15.15.4" class="ltx_td ltx_align_right ltx_border_bb">(1122 / 37)</td>
<td id="S2.T8.1.15.15.5" class="ltx_td ltx_align_right ltx_border_bb">1254</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S8: </span>Statistics about created datasets‚Äô content. For <span id="S2.T8.8.1" class="ltx_text ltx_font_typewriter">Diversty-raw</span>, the top-50 articles per biological kingdoms (with an available abstract) were reserved for the evaluation set. The train/valid sets are composed of the remaining items split in 90:10. A similar split was performed on the initial 5 random samples to obtain the train/valid datasets of equivalent sizes, referred as the <span id="S2.T8.9.2" class="ltx_text ltx_font_typewriter">Random-raw</span> datasets. Their count statistics are averaged over the 5 seeds. <span id="S2.T8.10.3" class="ltx_text ltx_font_typewriter">Extended-raw</span> is the fusion of the <span id="S2.T8.11.4" class="ltx_text ltx_font_typewriter">Diversty-raw</span> plus the 5 <span id="S2.T8.12.5" class="ltx_text ltx_font_typewriter">Random-raw</span> datasets. <span id="S2.T8.13.6" class="ltx_text ltx_font_typewriter">Full</span> is a dataset containing all available examples from the LOTUS snapshot, except the 200 used in the evaluation set. For synthetic datasets, the number of relations, as well as the number of distinct chemicals, is split between chemical entities and chemical classes.</figcaption>
</figure>
<section id="S2.SS5.SSS0.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_title_paragraph">Curated evaluation dataset: </h5>

<div id="S2.SS5.SSS0.Px1.p1" class="ltx_para ltx_noindent">
<p id="S2.SS5.SSS0.Px1.p1.1" class="ltx_p">The composition of the curated evaluation dataset, in terms of number of distinct entities and relationships, is compared to 5 random sets of equivalent sizes. Firstly, 13 abstracts did not mention any relationships between organisms and chemicals in the curated dataset. Secondly, for the random sets, statistics were directly estimated from the LOTUS annotations. Then, they may represent an overestimate of the actual number of distinct entities, given that a manual curation could potentially eliminate some irrelevant annotations that are actually not mention in the abstracts. They should therefore be regarded as an approximate upper bound. Considering the last points, the proposed strategy for selecting the evaluation set has significantly improved the diversity, as highlighted in Table <a href="#S2.T9" title="Table S9 ‚Ä£ Curated evaluation dataset: ‚Ä£ S2.5 Details of the composition of datasets ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S9</span></a>.</p>
</div>
<figure id="S2.T9" class="ltx_table">
<table id="S2.T9.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S2.T9.1.1" class="ltx_tr">
<td id="S2.T9.1.1.1" class="ltx_td ltx_border_tt"></td>
<td id="S2.T9.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T9.1.1.2.1" class="ltx_text ltx_font_bold"># Organisms</span></td>
<td id="S2.T9.1.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T9.1.1.3.1" class="ltx_text ltx_font_bold"># Chemicals</span></td>
<td id="S2.T9.1.1.4" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T9.1.1.4.1" class="ltx_text ltx_font_bold"># Relations</span></td>
<td id="S2.T9.1.1.5" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T9.1.1.5.1" class="ltx_text ltx_font_bold"># References</span></td>
</tr>
<tr id="S2.T9.1.2" class="ltx_tr">
<td id="S2.T9.1.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T9.1.2.1.1" class="ltx_text ltx_font_bold">eval-set (top200 diversity)</span></td>
<td id="S2.T9.1.2.2" class="ltx_td ltx_align_right ltx_border_t">275</td>
<td id="S2.T9.1.2.3" class="ltx_td ltx_align_right ltx_border_t">1197 ( 1092 / 105)</td>
<td id="S2.T9.1.2.4" class="ltx_td ltx_align_right ltx_border_t">1488 (1297 / 191)</td>
<td id="S2.T9.1.2.5" class="ltx_td ltx_align_right ltx_border_t">200 (187*)</td>
</tr>
<tr id="S2.T9.1.3" class="ltx_tr">
<td id="S2.T9.1.3.1" class="ltx_td ltx_align_left ltx_border_bb"><span id="S2.T9.1.3.1.1" class="ltx_text ltx_font_bold">Random (200 articles)</span></td>
<td id="S2.T9.1.3.2" class="ltx_td ltx_align_right ltx_border_bb">238</td>
<td id="S2.T9.1.3.3" class="ltx_td ltx_align_right ltx_border_bb">610</td>
<td id="S2.T9.1.3.4" class="ltx_td ltx_align_right ltx_border_bb">699</td>
<td id="S2.T9.1.3.5" class="ltx_td ltx_align_right ltx_border_bb">200</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S9: </span>Statistics of the number of organisms, chemicals, and relations in the top-200 abstracts selected and curated in the evaluation set, compared to 200 randomly selected items (statistics averaged over 5 random seeds). For the evaluation set, the number of annotated distinct chemical compounds and chemical classes are respectively indicated between parentheses. In the curated evaluation set, 13 references had no relation directly expressed in the abstract.</figcaption>
</figure>
</section>
</section>
<section id="S2.SS6" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S2.6 </span>Evaluation of Keyword Extraction on the SemEVAL2017 dataset</h3>

<figure id="S2.T10" class="ltx_table">
<table id="S2.T10.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S2.T10.1.1" class="ltx_tr">
<td id="S2.T10.1.1.1" class="ltx_td ltx_border_tt"></td>
<td id="S2.T10.1.1.2" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T10.1.1.2.1" class="ltx_text ltx_font_bold">TP in Top10</span></td>
<td id="S2.T10.1.1.3" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T10.1.1.3.1" class="ltx_text ltx_font_bold">FP in Top10</span></td>
<td id="S2.T10.1.1.4" class="ltx_td ltx_align_right ltx_border_tt"><span id="S2.T10.1.1.4.1" class="ltx_text ltx_font_bold">Precision</span></td>
</tr>
<tr id="S2.T10.1.2" class="ltx_tr">
<td id="S2.T10.1.2.1" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T10.1.2.1.1" class="ltx_text ltx_font_bold">KeyBERT (all-MiniLM-L6-v2)</span></td>
<td id="S2.T10.1.2.2" class="ltx_td ltx_align_right ltx_border_t">96</td>
<td id="S2.T10.1.2.3" class="ltx_td ltx_align_right ltx_border_t">904</td>
<td id="S2.T10.1.2.4" class="ltx_td ltx_align_right ltx_border_t">9.6</td>
</tr>
<tr id="S2.T10.1.3" class="ltx_tr">
<td id="S2.T10.1.3.1" class="ltx_td ltx_align_left ltx_border_bb"><span id="S2.T10.1.3.1.1" class="ltx_text ltx_font_bold">Vicuna-13B</span></td>
<td id="S2.T10.1.3.2" class="ltx_td ltx_align_right ltx_border_bb">321</td>
<td id="S2.T10.1.3.3" class="ltx_td ltx_align_right ltx_border_bb">669</td>
<td id="S2.T10.1.3.4" class="ltx_td ltx_align_right ltx_border_bb"><span id="S2.T10.1.3.4.1" class="ltx_text ltx_font_bold">32.424</span></td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table S10: </span>Comparison of the performance of the prompted Vicuna-13B LLM and KeyBERT for keywords/keyphrases extraction on the <span id="S2.T10.3.1" class="ltx_text ltx_font_italic">SemEVAL2017</span> test set. The evaluation was only done on the top-10 extracted keywords for the both methods, to use the same configuration as in the experiments.</figcaption>
</figure>
<div id="S2.SS6.p1" class="ltx_para ltx_noindent">
<p id="S2.SS6.p1.1" class="ltx_p">The SemEVAL2017 <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib93" title="" class="ltx_ref">93</a>]</cite> evaluation dataset consists of 100 paragraphs, extracted from scientific publications in various domains, with on average 17.23 annotated keyphrases. While 3 subtasks are proposed in this challenge (classification and semantic relation), we only focused on the mention-level keyphrases identification. To consider similar settings as used for synthetic abstract generation, we evaluated the precision in the top-10 extracted keywords. The comparison is done by exact-match and results are presented in Table <a href="#S2.T10" title="Table S10 ‚Ä£ S2.6 Evaluation of Keyword Extraction on the SemEVAL2017 dataset ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S10</span></a>. Vicuna-13B largely outperforms the KeyBERT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib115" title="" class="ltx_ref">115</a>]</cite> baseline and show more than acceptable performance in zero-shot settings. KeyBERT was used with standard parameters: <span id="S2.SS6.p1.1.1" class="ltx_text ltx_font_typewriter">keyphrase_ngram_range: (1,2), stop_words: None, use_mmr: True, diversity: 0.7</span> and BERT model <span id="S2.SS6.p1.1.2" class="ltx_text ltx_font_typewriter">all-MiniLM-L6-v2</span> for base embeddings.</p>
</div>
</section>
<section id="S2.SS7" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S2.7 </span>LLM prompting for synthetic abstract generation</h3>

<div id="S2.SS7.p1" class="ltx_para ltx_noindent">
<p id="S2.SS7.p1.1" class="ltx_p">The following section provides archetypal examples to illustrate the diversity engendered by the synthetic abstract generation process. Recall that each generation is calibrated with an original title, a set of keyphrases derived from the original abstract and verbalised main-findings. In the latter, 5 main transformations can be applied to improve the diversity of the generation (see Method <a href="#S2.SS3" title="2.3 Synthetic abstracts generation ‚Ä£ 2 Proposed approach ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.3</span></a>).
<br class="ltx_break">These transformations allow for the generation of multiple alternative synthetic abstracts, which emulate different syntaxes or styles for communicating the isolation of the same set of compounds (see supplementary Figure <a href="#S2.F5" title="Figure S5 ‚Ä£ S2.7 LLM prompting for synthetic abstract generation ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S5</span></a>).
<span id="S2.SS7.p1.1.1" class="ltx_text ltx_font_bold">A</span> serves as a reference for a <span id="S2.SS7.p1.1.2" class="ltx_text ltx_font_italic">standard</span> instruction/generation. The example <span id="S2.SS7.p1.1.3" class="ltx_text ltx_font_bold">B</span> introduces variations by reshuffling the order of the mentioned chemicals and then numbering them. In <span id="S2.SS7.p1.1.4" class="ltx_text ltx_font_bold">C</span>, different subsets of compounds were substituted with their associated chemical families. In <span id="S2.SS7.p1.1.5" class="ltx_text ltx_font_bold">A</span> and <span id="S2.SS7.p1.1.6" class="ltx_text ltx_font_bold">B</span>, the expected output labels align with the verbalised main findings, e.g: "<span id="S2.SS7.p1.1.7" class="ltx_text ltx_font_italic">Lachnum papyraceum produces 6-Methoxymellein; Lachnum papyraceum produces 4-Chloro-6-methoxymellein, ‚Ä¶</span>". In <span id="S2.SS7.p1.1.8" class="ltx_text ltx_font_bold">C</span>, they are substituted by the chemical classes: <span id="S2.SS7.p1.1.9" class="ltx_text ltx_font_italic">Lachnum papyraceum produces Coumarins; etc.</span>.
<br class="ltx_break">However, for "Multiple" chemicals contraction (Supplementary Figure <a href="#S2.F6" title="Figure S6 ‚Ä£ S2.7 LLM prompting for synthetic abstract generation ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S6</span></a>), while the synthetic text mention "cytosporones J-N, pestalasins A-E", the outputs are expected to be expanded like: "Cytosporone J, Cytosporone K, Cytosporone L, ‚Ä¶, Pestalasin A, Pestalasin B, ‚Ä¶, Pestalasin E". Verbalised relations can also exhibit a <span id="S2.SS7.p1.1.10" class="ltx_text ltx_markedasmath">N:M</span> pattern, when multiple compounds are isolated from multiple organisms, showcasing the model creative generation abilities (Supplementary Figure <a href="#S2.F7" title="Figure S7 ‚Ä£ S2.7 LLM prompting for synthetic abstract generation ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S7</span></a>).
<br class="ltx_break">The generation process is subject to certain limitations and can occasionally produce inaccuracies of a similar nature to those that were intended to be mitigated. In Supplementary Figure <a href="#S2.F8" title="Figure S8 ‚Ä£ S2.7 LLM prompting for synthetic abstract generation ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S8</span></a>, while it is explicitly indicated in the instruction part that "Tagetes erecta produces two Flavonoids", this information does not appear in the generated abstract. Additionally, the NPs isolated from <span id="S2.SS7.p1.1.11" class="ltx_text ltx_font_italic">Tagetes lucida</span> are qualified as Flavonoids, which is a wrong assertion, i.e a hallucination. The synthetic abstracts frequently exhibit instances of hallucinations, yet, these do not significantly impair their utility for the specific task of RE, as long as they do not pertain to the expression of the relationships (see Supplementary Figure <a href="#S2.F9" title="Figure S9 ‚Ä£ S2.7 LLM prompting for synthetic abstract generation ‚Ä£ S2 Supplementary materials ‚Ä£ Relation Extraction in underexplored biomedical domains: A diversity-optimised sampling and synthetic data generation approach" class="ltx_ref"><span class="ltx_text ltx_ref_tag">S9</span></a>).</p>
</div>
<figure id="S2.F5" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/examples-synthetic-generation/ex1.png" id="S2.F5.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="337" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S5: </span>3 generated abstracts from the seed article PMID: 7730162. <span id="S2.F5.8.1" class="ltx_text ltx_font_bold">A</span> is a standard generation. In <span id="S2.F5.9.2" class="ltx_text ltx_font_bold">B</span>, the chemicals are shuffled and numbered. In <span id="S2.F5.10.3" class="ltx_text ltx_font_bold">C</span> chemicals were substituted with their corresponding chemical families. All generation were produced with <math id="S2.F5.3.m1.1" class="ltx_Math" alttext="temp=0.7" display="inline"><semantics id="S2.F5.3.m1.1b"><mrow id="S2.F5.3.m1.1.1" xref="S2.F5.3.m1.1.1.cmml"><mrow id="S2.F5.3.m1.1.1.2" xref="S2.F5.3.m1.1.1.2.cmml"><mi id="S2.F5.3.m1.1.1.2.2" xref="S2.F5.3.m1.1.1.2.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.F5.3.m1.1.1.2.1" xref="S2.F5.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F5.3.m1.1.1.2.3" xref="S2.F5.3.m1.1.1.2.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.F5.3.m1.1.1.2.1b" xref="S2.F5.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F5.3.m1.1.1.2.4" xref="S2.F5.3.m1.1.1.2.4.cmml">m</mi><mo lspace="0em" rspace="0em" id="S2.F5.3.m1.1.1.2.1c" xref="S2.F5.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F5.3.m1.1.1.2.5" xref="S2.F5.3.m1.1.1.2.5.cmml">p</mi></mrow><mo id="S2.F5.3.m1.1.1.1" xref="S2.F5.3.m1.1.1.1.cmml">=</mo><mn id="S2.F5.3.m1.1.1.3" xref="S2.F5.3.m1.1.1.3.cmml">0.7</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.F5.3.m1.1c"><apply id="S2.F5.3.m1.1.1.cmml" xref="S2.F5.3.m1.1.1"><eq id="S2.F5.3.m1.1.1.1.cmml" xref="S2.F5.3.m1.1.1.1"></eq><apply id="S2.F5.3.m1.1.1.2.cmml" xref="S2.F5.3.m1.1.1.2"><times id="S2.F5.3.m1.1.1.2.1.cmml" xref="S2.F5.3.m1.1.1.2.1"></times><ci id="S2.F5.3.m1.1.1.2.2.cmml" xref="S2.F5.3.m1.1.1.2.2">ùë°</ci><ci id="S2.F5.3.m1.1.1.2.3.cmml" xref="S2.F5.3.m1.1.1.2.3">ùëí</ci><ci id="S2.F5.3.m1.1.1.2.4.cmml" xref="S2.F5.3.m1.1.1.2.4">ùëö</ci><ci id="S2.F5.3.m1.1.1.2.5.cmml" xref="S2.F5.3.m1.1.1.2.5">ùëù</ci></apply><cn type="float" id="S2.F5.3.m1.1.1.3.cmml" xref="S2.F5.3.m1.1.1.3">0.7</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F5.3.m1.1d">temp=0.7</annotation></semantics></math> and are training examples from the <span id="S2.F5.11.4" class="ltx_text ltx_font_typewriter">Diversity-synt</span> dataset.</figcaption>
</figure>
<figure id="S2.F6" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/examples-synthetic-generation/ex2.png" id="S2.F6.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="175" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S6: </span>An example of an abstract generation with multiple-chemical in the instruction. The original seed article is PMID 19762244. The generation was produced with <math id="S2.F6.3.m1.1" class="ltx_Math" alttext="temp=0.7" display="inline"><semantics id="S2.F6.3.m1.1b"><mrow id="S2.F6.3.m1.1.1" xref="S2.F6.3.m1.1.1.cmml"><mrow id="S2.F6.3.m1.1.1.2" xref="S2.F6.3.m1.1.1.2.cmml"><mi id="S2.F6.3.m1.1.1.2.2" xref="S2.F6.3.m1.1.1.2.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.F6.3.m1.1.1.2.1" xref="S2.F6.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F6.3.m1.1.1.2.3" xref="S2.F6.3.m1.1.1.2.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.F6.3.m1.1.1.2.1b" xref="S2.F6.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F6.3.m1.1.1.2.4" xref="S2.F6.3.m1.1.1.2.4.cmml">m</mi><mo lspace="0em" rspace="0em" id="S2.F6.3.m1.1.1.2.1c" xref="S2.F6.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F6.3.m1.1.1.2.5" xref="S2.F6.3.m1.1.1.2.5.cmml">p</mi></mrow><mo id="S2.F6.3.m1.1.1.1" xref="S2.F6.3.m1.1.1.1.cmml">=</mo><mn id="S2.F6.3.m1.1.1.3" xref="S2.F6.3.m1.1.1.3.cmml">0.7</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.F6.3.m1.1c"><apply id="S2.F6.3.m1.1.1.cmml" xref="S2.F6.3.m1.1.1"><eq id="S2.F6.3.m1.1.1.1.cmml" xref="S2.F6.3.m1.1.1.1"></eq><apply id="S2.F6.3.m1.1.1.2.cmml" xref="S2.F6.3.m1.1.1.2"><times id="S2.F6.3.m1.1.1.2.1.cmml" xref="S2.F6.3.m1.1.1.2.1"></times><ci id="S2.F6.3.m1.1.1.2.2.cmml" xref="S2.F6.3.m1.1.1.2.2">ùë°</ci><ci id="S2.F6.3.m1.1.1.2.3.cmml" xref="S2.F6.3.m1.1.1.2.3">ùëí</ci><ci id="S2.F6.3.m1.1.1.2.4.cmml" xref="S2.F6.3.m1.1.1.2.4">ùëö</ci><ci id="S2.F6.3.m1.1.1.2.5.cmml" xref="S2.F6.3.m1.1.1.2.5">ùëù</ci></apply><cn type="float" id="S2.F6.3.m1.1.1.3.cmml" xref="S2.F6.3.m1.1.1.3">0.7</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F6.3.m1.1d">temp=0.7</annotation></semantics></math>. It is a training example from the <span id="S2.F6.5.1" class="ltx_text ltx_font_typewriter">Diversity-synt</span> dataset.</figcaption>
</figure>
<figure id="S2.F7" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/examples-synthetic-generation/ex3.png" id="S2.F7.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="237" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S7: </span>An example of an abstract generation with a <span id="S2.F7.6.1" class="ltx_text ltx_font_typewriter">N:M</span>-type relations. The original seed article is PMID 20382006. The generation was produced with <math id="S2.F7.3.m1.1" class="ltx_Math" alttext="temp=0.7" display="inline"><semantics id="S2.F7.3.m1.1b"><mrow id="S2.F7.3.m1.1.1" xref="S2.F7.3.m1.1.1.cmml"><mrow id="S2.F7.3.m1.1.1.2" xref="S2.F7.3.m1.1.1.2.cmml"><mi id="S2.F7.3.m1.1.1.2.2" xref="S2.F7.3.m1.1.1.2.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.F7.3.m1.1.1.2.1" xref="S2.F7.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F7.3.m1.1.1.2.3" xref="S2.F7.3.m1.1.1.2.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.F7.3.m1.1.1.2.1b" xref="S2.F7.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F7.3.m1.1.1.2.4" xref="S2.F7.3.m1.1.1.2.4.cmml">m</mi><mo lspace="0em" rspace="0em" id="S2.F7.3.m1.1.1.2.1c" xref="S2.F7.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F7.3.m1.1.1.2.5" xref="S2.F7.3.m1.1.1.2.5.cmml">p</mi></mrow><mo id="S2.F7.3.m1.1.1.1" xref="S2.F7.3.m1.1.1.1.cmml">=</mo><mn id="S2.F7.3.m1.1.1.3" xref="S2.F7.3.m1.1.1.3.cmml">0.7</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.F7.3.m1.1c"><apply id="S2.F7.3.m1.1.1.cmml" xref="S2.F7.3.m1.1.1"><eq id="S2.F7.3.m1.1.1.1.cmml" xref="S2.F7.3.m1.1.1.1"></eq><apply id="S2.F7.3.m1.1.1.2.cmml" xref="S2.F7.3.m1.1.1.2"><times id="S2.F7.3.m1.1.1.2.1.cmml" xref="S2.F7.3.m1.1.1.2.1"></times><ci id="S2.F7.3.m1.1.1.2.2.cmml" xref="S2.F7.3.m1.1.1.2.2">ùë°</ci><ci id="S2.F7.3.m1.1.1.2.3.cmml" xref="S2.F7.3.m1.1.1.2.3">ùëí</ci><ci id="S2.F7.3.m1.1.1.2.4.cmml" xref="S2.F7.3.m1.1.1.2.4">ùëö</ci><ci id="S2.F7.3.m1.1.1.2.5.cmml" xref="S2.F7.3.m1.1.1.2.5">ùëù</ci></apply><cn type="float" id="S2.F7.3.m1.1.1.3.cmml" xref="S2.F7.3.m1.1.1.3">0.7</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F7.3.m1.1d">temp=0.7</annotation></semantics></math>. It is a training example from the <span id="S2.F7.7.2" class="ltx_text ltx_font_typewriter">Diversity-synt</span> dataset.</figcaption>
</figure>
<figure id="S2.F8" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/examples-synthetic-generation/ex4.png" id="S2.F8.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="228" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S8: </span>An example of an abstract generation with incorrectly added and removed relations. The original seed article is PMID 19127719. The generation was produced with <math id="S2.F8.3.m1.1" class="ltx_Math" alttext="temp=0.5" display="inline"><semantics id="S2.F8.3.m1.1b"><mrow id="S2.F8.3.m1.1.1" xref="S2.F8.3.m1.1.1.cmml"><mrow id="S2.F8.3.m1.1.1.2" xref="S2.F8.3.m1.1.1.2.cmml"><mi id="S2.F8.3.m1.1.1.2.2" xref="S2.F8.3.m1.1.1.2.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.F8.3.m1.1.1.2.1" xref="S2.F8.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F8.3.m1.1.1.2.3" xref="S2.F8.3.m1.1.1.2.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.F8.3.m1.1.1.2.1b" xref="S2.F8.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F8.3.m1.1.1.2.4" xref="S2.F8.3.m1.1.1.2.4.cmml">m</mi><mo lspace="0em" rspace="0em" id="S2.F8.3.m1.1.1.2.1c" xref="S2.F8.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F8.3.m1.1.1.2.5" xref="S2.F8.3.m1.1.1.2.5.cmml">p</mi></mrow><mo id="S2.F8.3.m1.1.1.1" xref="S2.F8.3.m1.1.1.1.cmml">=</mo><mn id="S2.F8.3.m1.1.1.3" xref="S2.F8.3.m1.1.1.3.cmml">0.5</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.F8.3.m1.1c"><apply id="S2.F8.3.m1.1.1.cmml" xref="S2.F8.3.m1.1.1"><eq id="S2.F8.3.m1.1.1.1.cmml" xref="S2.F8.3.m1.1.1.1"></eq><apply id="S2.F8.3.m1.1.1.2.cmml" xref="S2.F8.3.m1.1.1.2"><times id="S2.F8.3.m1.1.1.2.1.cmml" xref="S2.F8.3.m1.1.1.2.1"></times><ci id="S2.F8.3.m1.1.1.2.2.cmml" xref="S2.F8.3.m1.1.1.2.2">ùë°</ci><ci id="S2.F8.3.m1.1.1.2.3.cmml" xref="S2.F8.3.m1.1.1.2.3">ùëí</ci><ci id="S2.F8.3.m1.1.1.2.4.cmml" xref="S2.F8.3.m1.1.1.2.4">ùëö</ci><ci id="S2.F8.3.m1.1.1.2.5.cmml" xref="S2.F8.3.m1.1.1.2.5">ùëù</ci></apply><cn type="float" id="S2.F8.3.m1.1.1.3.cmml" xref="S2.F8.3.m1.1.1.3">0.5</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.3.m1.1d">temp=0.5</annotation></semantics></math>. It is a training example from the <span id="S2.F8.5.1" class="ltx_text ltx_font_typewriter">Diversity-synt</span> dataset.</figcaption>
</figure>
<figure id="S2.F9" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/examples-synthetic-generation/ex5.png" id="S2.F9.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="199" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S9: </span>An example of an abstract generation with a context hallucination. The original seed article is PMID 23758051. The generation was produced with <math id="S2.F9.3.m1.1" class="ltx_Math" alttext="temp=0.5" display="inline"><semantics id="S2.F9.3.m1.1b"><mrow id="S2.F9.3.m1.1.1" xref="S2.F9.3.m1.1.1.cmml"><mrow id="S2.F9.3.m1.1.1.2" xref="S2.F9.3.m1.1.1.2.cmml"><mi id="S2.F9.3.m1.1.1.2.2" xref="S2.F9.3.m1.1.1.2.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S2.F9.3.m1.1.1.2.1" xref="S2.F9.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F9.3.m1.1.1.2.3" xref="S2.F9.3.m1.1.1.2.3.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.F9.3.m1.1.1.2.1b" xref="S2.F9.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F9.3.m1.1.1.2.4" xref="S2.F9.3.m1.1.1.2.4.cmml">m</mi><mo lspace="0em" rspace="0em" id="S2.F9.3.m1.1.1.2.1c" xref="S2.F9.3.m1.1.1.2.1.cmml">‚Äã</mo><mi id="S2.F9.3.m1.1.1.2.5" xref="S2.F9.3.m1.1.1.2.5.cmml">p</mi></mrow><mo id="S2.F9.3.m1.1.1.1" xref="S2.F9.3.m1.1.1.1.cmml">=</mo><mn id="S2.F9.3.m1.1.1.3" xref="S2.F9.3.m1.1.1.3.cmml">0.5</mn></mrow><annotation-xml encoding="MathML-Content" id="S2.F9.3.m1.1c"><apply id="S2.F9.3.m1.1.1.cmml" xref="S2.F9.3.m1.1.1"><eq id="S2.F9.3.m1.1.1.1.cmml" xref="S2.F9.3.m1.1.1.1"></eq><apply id="S2.F9.3.m1.1.1.2.cmml" xref="S2.F9.3.m1.1.1.2"><times id="S2.F9.3.m1.1.1.2.1.cmml" xref="S2.F9.3.m1.1.1.2.1"></times><ci id="S2.F9.3.m1.1.1.2.2.cmml" xref="S2.F9.3.m1.1.1.2.2">ùë°</ci><ci id="S2.F9.3.m1.1.1.2.3.cmml" xref="S2.F9.3.m1.1.1.2.3">ùëí</ci><ci id="S2.F9.3.m1.1.1.2.4.cmml" xref="S2.F9.3.m1.1.1.2.4">ùëö</ci><ci id="S2.F9.3.m1.1.1.2.5.cmml" xref="S2.F9.3.m1.1.1.2.5">ùëù</ci></apply><cn type="float" id="S2.F9.3.m1.1.1.3.cmml" xref="S2.F9.3.m1.1.1.3">0.5</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F9.3.m1.1d">temp=0.5</annotation></semantics></math>. It is a training example from the <span id="S2.F9.5.1" class="ltx_text ltx_font_typewriter">Diversity-synt</span> dataset.</figcaption>
</figure>
<div class="ltx_pagination ltx_role_newpage"></div>
</section>
<section id="S2.SS8" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">S2.8 </span>Tokenized length of abstracts</h3>

<figure id="S2.F10" class="ltx_figure"><img src="/html/2311.06364/assets/Supplementary/SupplementaryFigures/tokenized-asbtract-sizes-diff.png" id="S2.F10.1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="479" height="269" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure S10: </span>Differences in size of the tokenized abstracts from the <span id="S2.F10.5.1" class="ltx_text ltx_font_italic">Extended-raw</span> dataset (23985 abstracts) between <span id="S2.F10.6.2" class="ltx_text ltx_font_typewriter">GPT-2</span> and <span id="S2.F10.7.3" class="ltx_text ltx_font_typewriter">BioGPT</span> tokenizers. The dedicated tokenizer of BioGPT allows for a more efficient tokenization of the abstracts.</figcaption>
</figure>
</section>
</section>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/2311.06363" class="ar5iv-nav-button ar5iv-nav-button-prev">‚óÑ</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/2311.06364" class="ar5iv-text-button ar5iv-severity-error">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+2311.06364">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/2311.06364" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/2311.06365" class="ar5iv-nav-button ar5iv-nav-button-next">‚ñ∫</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Tue Feb 27 19:32:15 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "√ó";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
