<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[2403.03368] Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures</title><meta property="og:description" content="The effectiveness of clopidogrel, a widely used antiplatelet medication, varies significantly among individuals, necessitating the development of precise predictive models to optimize patient care. In this study, we le…">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/2403.03368">

<!--Generated on Fri Apr  5 14:57:49 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line" lang="en">
<h1 class="ltx_title ltx_title_document">Leveraging Federated Learning for 
<br class="ltx_break">Automatic Detection of Clopidogrel Treatment Failures</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Samuel Kim and Min Sang Kim 
<br class="ltx_break">
<br class="ltx_break">Cipherome Inc., San Jose, CA, U.S.A. 
<br class="ltx_break"><span id="id1.1.id1" class="ltx_text ltx_font_typewriter" style="font-size:90%;">{sam.kim, min.kim}@cipherome.com<span id="id1.1.id1.1" class="ltx_text ltx_font_serif"> 
<br class="ltx_break"></span></span>
</span></span>
</div>

<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p id="id2.id1" class="ltx_p"><span id="id2.id1.1" class="ltx_text">The effectiveness of clopidogrel, a widely used antiplatelet medication, varies significantly among individuals, necessitating the development of precise predictive models to optimize patient care. In this study, we leverage federated learning strategies to address clopidogrel treatment failure detection. Our research harnesses the collaborative power of multiple healthcare institutions, allowing them to jointly train machine learning models while safeguarding sensitive patient data. Utilizing the UK Biobank dataset, which encompasses a vast and diverse population, we partitioned the data based on geographic centers and evaluated the performance of federated learning. Our results show that while centralized training achieves higher Area Under the Curve (AUC) values and faster convergence, federated learning approaches can substantially narrow this performance gap. Our findings underscore the potential of federated learning in addressing clopidogrel treatment failure detection, offering a promising avenue for enhancing patient care through personalized treatment strategies while respecting data privacy. This study contributes to the growing body of research on federated learning in healthcare and lays the groundwork for secure and privacy-preserving predictive models for various medical conditions.</span></p>
</div>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">I </span><span id="S1.1.1" class="ltx_text ltx_font_smallcaps">Introduction</span>
</h2>

<div id="S1.p1" class="ltx_para">
<p id="S1.p1.1" class="ltx_p">The prediction of adverse reactions to clopidogrel, a widely prescribed antiplatelet medication, is a critical challenge in modern healthcare. Individual responses to clopidogrel can vary significantly, leading to varying degrees of treatment efficacy and the potential for adverse events. Traditional approaches to adverse reaction prediction typically involve centralized data analysis, which entails the aggregation of patient data from diverse sources into a single, consolidated repository <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib1" title="" class="ltx_ref">1</a>, <a href="#bib.bib2" title="" class="ltx_ref">2</a>]</cite>. However, this centralized approach presents inherent challenges, including data privacy concerns, security risks, and regulatory complexities, particularly in the realm of healthcare data.</p>
</div>
<div id="S1.p2" class="ltx_para">
<p id="S1.p2.1" class="ltx_p">In recent years, federated learning has emerged as a transformative paradigm for addressing these issues. Federated learning enables collaborative model training across decentralized data sources while keeping raw data localized and secure. In the context of predicting adverse reactions to clopidogrel, federated learning offers a promising avenue to leverage insights from multiple healthcare institutions without compromising patient confidentiality.</p>
</div>
<div id="S1.p3" class="ltx_para">
<p id="S1.p3.1" class="ltx_p">While initially proposed for machine learning in mobile environments <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite>, federated learning has garnered increasing attention within the healthcare domain, primarily owing to its privacy-preserving capabilities <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib4" title="" class="ltx_ref">4</a>]</cite>. Researchers have been actively exploring its applications in addressing a wide array of medical conditions, including Alzheimer’s disease <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite> and COVID-19 <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib6" title="" class="ltx_ref">6</a>, <a href="#bib.bib7" title="" class="ltx_ref">7</a>]</cite>. Pati <span id="S1.p3.1.1" class="ltx_text ltx_font_italic">et al.</span> conducted a substantial-scale federated learning study aimed at developing a tumor boundary detection model <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite>. They spanned 71 geographically distinct sites across six continents and demonstrated the superiority of their approach over conventional methods.</p>
</div>
<div id="S1.p4" class="ltx_para">
<p id="S1.p4.1" class="ltx_p">This paper focuses on a novel approach to detect clopidogrel treatment failures that harnesses the power of federated learning strategies. We present a federated learning framework designed to facilitate joint model training across diverse healthcare institutions, allowing them to pool their collective knowledge without exposing sensitive patient data. In particular, we use patients’ medical history including diagnoses, procedures, and prescriptions. Our approach seeks to enhance the accuracy and generalizability of clopidogrel adverse reaction prediction models by aggregating insights from a multitude of sources.</p>
</div>
<figure id="S1.F1" class="ltx_figure"><img src="/html/2403.03368/assets/figures/scenarios.png" id="S1.F1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="318" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span id="S1.F1.2.1.1" class="ltx_text" style="font-size:90%;">Figure 1</span>: </span><span id="S1.F1.3.2" class="ltx_text" style="font-size:90%;">Experimental scenarios to illustrate how data is handled; 1) localized training, 2) centralized training, and 3) federated learning. </span></figcaption>
</figure>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">II </span><span id="S2.1.1" class="ltx_text ltx_font_smallcaps">Data</span>
</h2>

<section id="S2.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S2.SS1.4.1.1" class="ltx_text">II-A</span> </span><span id="S2.SS1.5.2" class="ltx_text ltx_font_italic">Dataset</span>
</h3>

<div id="S2.SS1.p1" class="ltx_para">
<p id="S2.SS1.p1.1" class="ltx_p">We used the UK Biobank which consists of data collected from 502,527 participants  <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib9" title="" class="ltx_ref">9</a>]</cite>. Volunteers aged 40 to 70 were recruited from England, Scotland, and Wales and invited to assessment centers between 2006 and 2010. Data collected during the visits included biosamples, physical examination measurements, and questionnaire answers followed by interviews. Genomic data, both sequencing and genotyping, were generated using the biosamples collected. Also, an extensive and comprehensive medical history data was made available through hospital in-patient and primary care data from external data sources.</p>
</div>
<div id="S2.SS1.p2" class="ltx_para">
<p id="S2.SS1.p2.1" class="ltx_p">We extracted prescriptions, diagnoses and procedure records, along with dates, for all participants from the UK Biobank. All prescription records were from general practitioner (GP) data coded in Read and British National Formulary (BNF) depending on the data supplier. Diagnoses and procedure records were from hospital in-patient records only and coded in ICD-9/10 and OPCS-3/4 respectively.</p>
</div>
</section>
<section id="S2.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S2.SS2.4.1.1" class="ltx_text">II-B</span> </span><span id="S2.SS2.5.2" class="ltx_text ltx_font_italic">Annotation</span>
</h3>

<div id="S2.SS2.p1" class="ltx_para">
<p id="S2.SS2.p1.1" class="ltx_p">Treatment failure (TF) was defined as having a TF event within one year of the very first clopidogrel prescription. Clopidogrel prescriptions were identified using the substance or brand names of clopidogrel or respective read codes, while TF events include ischemic stroke, myocardial infarct, stent thrombosis and recurrent thrombosis or stenting. All of these annotations were carefully done by clinical professions reviewing patients’ medical records.</p>
</div>
<div id="S2.SS2.p2" class="ltx_para">
<p id="S2.SS2.p2.1" class="ltx_p">Subjects with events occurring within 7 days of the first prescription were excluded as it was unclear whether those events were associated with clopidogrel. The visit had to be through the emergency room to be valid in order to exclude follow up visits from previous events. From the dataset, we found 9,867 subjects with clopidogrel prescriptions. Among them, we labeled 1,824 patients as TF cases and 6,859 as control cases; 1,184 subjects were excluded due to data inconsistencies or ambiguities.</p>
</div>
</section>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">III </span><span id="S3.1.1" class="ltx_text ltx_font_smallcaps">Methods</span>
</h2>

<section id="S3.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S3.SS1.4.1.1" class="ltx_text">III-A</span> </span><span id="S3.SS1.5.2" class="ltx_text ltx_font_italic">Experimental scenarios</span>
</h3>

<div id="S3.SS1.p1" class="ltx_para">
<p id="S3.SS1.p1.1" class="ltx_p">The study was conducted in three scenarios to show the benefits of the federated learning method as shown in Fig. <a href="#S1.F1" title="Figure 1 ‣ I Introduction ‣ Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>; 1) locally fragmented datasets, 2) an aggregated dataset as a whole, and 3) federated learning. The first and the second scenarios assume two opposite cases; the first is for the case that there is no way for researchers to share their datasets, while the second is for the case that researchers have full access to others’ datasets. We hypothesized that the performance in terms of accuracy between the two extreme scenarios is significantly different because of the lack of training data and model generalization that the first scenario inherently has. We also hypothesized that the performance of the federated learning method converges toward the one with the aggregated dataset as the proposed method should mitigate the problems with the fragmented datasets, therefore, attaining high predictive performance while ensuring the security of patient data.</p>
</div>
<div id="S3.SS1.p2" class="ltx_para">
<p id="S3.SS1.p2.1" class="ltx_p">Assuming that medical history data of individuals are stored in geographically distinct locations, we split the clopidogrel response dataset into 22 groups based on the UK Biobank assessment centre (field id 54-0.0) which includes the center location information of the initial assessment visit at which participants were recruited and consent given. The reason we chose the location of the first baseline survey is that most of participants only have the first baseline survey.</p>
</div>
<figure id="S3.F2" class="ltx_figure"><img src="/html/2403.03368/assets/figures/num_subject_per_center.png" id="S3.F2.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="328" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span id="S3.F2.2.1.1" class="ltx_text" style="font-size:90%;">Figure 2</span>: </span><span id="S3.F2.3.2" class="ltx_text" style="font-size:90%;">Number of patients with clopidogrel prescription in individual centers</span></figcaption>
</figure>
<figure id="S3.F3" class="ltx_figure"><img src="/html/2403.03368/assets/figures/testset_subject_per_center.png" id="S3.F3.g1" class="ltx_graphics ltx_img_landscape" width="598" height="331" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure"><span id="S3.F3.2.1.1" class="ltx_text" style="font-size:90%;">Figure 3</span>: </span><span id="S3.F3.3.2" class="ltx_text" style="font-size:90%;">Number of patients that are selected as test set in idividual centers.</span></figcaption>
</figure>
<div id="S3.SS1.p3" class="ltx_para">
<p id="S3.SS1.p3.1" class="ltx_p">Fig. <a href="#S3.F2" title="Figure 2 ‣ III-A Experimental scenarios ‣ III Methods ‣ Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> provides a visual representation of the patient distribution across multiple centers in descending order of the number of patients. As anticipated, the distribution of subjects is not uniform; certain centers have an abundance of patients, with numbers exceeding a thousand, while others have fewer than ten patients. To ensure a fair and comprehensive evaluation of our model, we meticulously constructed a test set that accurately reflects the diversity and distribution of patients across these centers. This test set was created by randomly selecting 20% of the entire cohort of clopidogrel patients, taking into account both the prevalence of treatment failures and the distribution of patients across the various centers. It’s important to note that this test set was exclusively reserved for model evaluation and was deliberately excluded from any aspect of the training process. Fig. <a href="#S3.F3" title="Figure 3 ‣ III-A Experimental scenarios ‣ III Methods ‣ Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a> shows the distribution of selected patients in the test set.</p>
</div>
</section>
<section id="S3.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S3.SS2.4.1.1" class="ltx_text">III-B</span> </span><span id="S3.SS2.5.2" class="ltx_text ltx_font_italic">Federated learning</span>
</h3>

<div id="S3.SS2.p1" class="ltx_para">
<p id="S3.SS2.p1.1" class="ltx_p">To consolidate the model parameters from multiple participating centers, we used FedAvg algorithm implemented in NVIDIA NVFlare framework <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib10" title="" class="ltx_ref">10</a>]</cite>. In the FedAvg algorithm, each participating center trains a local machine learning model using its own data. These local models are then communicated to a central server, which computes a weighted average of these models to create a global model update. This global model update is subsequently sent back to each participating device, where it is incorporated into the local model. This iterative process continues until the global model converges to a desirable state.</p>
</div>
<figure id="S3.T1" class="ltx_table">
<table id="S3.T1.2" class="ltx_tabular ltx_centering ltx_align_middle">
<tr id="S3.T1.2.1" class="ltx_tr">
<td id="S3.T1.2.1.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">Architecture</td>
<td id="S3.T1.2.1.2" class="ltx_td ltx_align_center ltx_border_t">AUC</td>
</tr>
<tr id="S3.T1.2.2" class="ltx_tr">
<td id="S3.T1.2.2.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_tt">FCN</td>
<td id="S3.T1.2.2.2" class="ltx_td ltx_align_center ltx_border_tt">0.793</td>
</tr>
<tr id="S3.T1.2.3" class="ltx_tr">
<td id="S3.T1.2.3.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">GRU</td>
<td id="S3.T1.2.3.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t">0.957</td>
</tr>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table"><span id="S3.T1.3.1.1" class="ltx_text" style="font-size:90%;">Table I</span>: </span><span id="S3.T1.4.2" class="ltx_text" style="font-size:90%;">AUCs of ML architectures in centralized training scenario.</span></figcaption>
</figure>
<figure id="S3.F4" class="ltx_figure"><img src="/html/2403.03368/assets/figures/center_local_aucs.png" id="S3.F4.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="334" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span id="S3.F4.2.1.1" class="ltx_text" style="font-size:90%;">Figure 4</span>: </span><span id="S3.F4.3.2" class="ltx_text" style="font-size:90%;">AUCs of locally trained models with their own data.</span></figcaption>
</figure>
<figure id="S3.F5" class="ltx_figure"><img src="/html/2403.03368/assets/figures/size_vs_auc.png" id="S3.F5.g1" class="ltx_graphics ltx_img_square" width="598" height="590" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure"><span id="S3.F5.2.1.1" class="ltx_text" style="font-size:90%;">Figure 5</span>: </span><span id="S3.F5.3.2" class="ltx_text" style="font-size:90%;">Scatter plot of the AUCs of locally trained models with respect to their cohort sizes.</span></figcaption>
</figure>
<figure id="S3.F6" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S3.F6.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2403.03368/assets/figures/fc_learning_curve.png" id="S3.F6.sf1.g1" class="ltx_graphics ltx_img_landscape" width="598" height="462" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure"><span id="S3.F6.sf1.2.1.1" class="ltx_text" style="font-size:90%;">(a)</span> </span><span id="S3.F6.sf1.3.2" class="ltx_text" style="font-size:90%;">FCN - Learning Curves</span></figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S3.F6.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2403.03368/assets/figures/fc_best_auc.png" id="S3.F6.sf2.g1" class="ltx_graphics ltx_img_landscape" width="598" height="462" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure"><span id="S3.F6.sf2.2.1.1" class="ltx_text" style="font-size:90%;">(b)</span> </span><span id="S3.F6.sf2.3.2" class="ltx_text" style="font-size:90%;">FCN - AUC</span></figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S3.F6.sf3" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2403.03368/assets/figures/rnn_learning_curve.png" id="S3.F6.sf3.g1" class="ltx_graphics ltx_img_landscape" width="598" height="462" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure"><span id="S3.F6.sf3.2.1.1" class="ltx_text" style="font-size:90%;">(c)</span> </span><span id="S3.F6.sf3.3.2" class="ltx_text" style="font-size:90%;">GRU - Learning Curves</span></figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S3.F6.sf4" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2403.03368/assets/figures/rnn_best_auc.png" id="S3.F6.sf4.g1" class="ltx_graphics ltx_img_landscape" width="598" height="462" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure"><span id="S3.F6.sf4.2.1.1" class="ltx_text" style="font-size:90%;">(d)</span> </span><span id="S3.F6.sf4.3.2" class="ltx_text" style="font-size:90%;">GRU - AUC</span></figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure"><span id="S3.F6.2.1.1" class="ltx_text" style="font-size:90%;">Figure 6</span>: </span><span id="S3.F6.3.2" class="ltx_text" style="font-size:90%;">AUCs of the models that are trained in a federated learning scenario: (a)(b) for the FCN architecture and (c)(d) for the GRU architecture.</span></figcaption>
</figure>
</section>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">IV </span><span id="S4.1.1" class="ltx_text ltx_font_smallcaps">Experimental Results</span>
</h2>

<div id="S4.p1" class="ltx_para">
<p id="S4.p1.1" class="ltx_p">We primarily used two neural network architectures; fully connected network (FCN) and gated recurrent unit (GRU). While we applied a multi-hot encoder for FCN, we flattened the codes of the visits and concatenated them across all visits so that a patient has a sentence-like sequence of codes that can be fed into GRU. Table <a href="#S3.T1" title="Table I ‣ III-B Federated learning ‣ III Methods ‣ Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures" class="ltx_ref"><span class="ltx_text ltx_ref_tag">I</span></a> shows the AUC values of each ML architecture in the conventional centralized training scenario. The results shows that the recurrent architecture outperforms the fully connected architecture. This performance advantage can be attributed to the GRU’s ability to capture and model the evolving dynamics and temporal dependencies inherent in patients’ medical histories, a capability that the FCN, relying on a bag-of-words approach, does not possess.</p>
</div>
<div id="S4.p2" class="ltx_para">
<p id="S4.p2.1" class="ltx_p">Figure <a href="#S3.F4" title="Figure 4 ‣ III-B Federated learning ‣ III Methods ‣ Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a> provides a clear depiction of the Area Under the Curve (AUC) values attained by individual models trained locally, using only their respective locally available datasets. As anticipated, the performance of these local models notably lags behind that achieved in the centralized training scenario, indicated by the dotted lines in the figure.</p>
</div>
<div id="S4.p3" class="ltx_para">
<p id="S4.p3.1" class="ltx_p">In Figure <a href="#S3.F5" title="Figure 5 ‣ III-B Federated learning ‣ III Methods ‣ Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>, we explore the intriguing relationship between the size of the local dataset and the corresponding model performance. This figure reveals a consistent trend across both model architectures: as the size of the local dataset increases, the performance of the local model also improves. This observation aligns with our expectations, as larger datasets enable models to discern more generalized patterns and enhance their predictive capabilities. This phenomenon underscores the significance of data quantity in the training process, emphasizing that access to more extensive and diverse datasets can yield models with superior generalization and prediction abilities, regardless of the chosen architecture.</p>
</div>
<div id="S4.p4" class="ltx_para">
<p id="S4.p4.1" class="ltx_p">In our federated learning scenario, we conducted a series of experiments involving varying numbers of participating centers. To simplify the analysis, we organized the centers in descending order based on the number of subjects within each center, incrementally incorporating them into the federated learning process.</p>
</div>
<div id="S4.p5" class="ltx_para">
<p id="S4.p5.1" class="ltx_p">Fig. <a href="#S3.F6" title="Figure 6 ‣ III-B Federated learning ‣ III Methods ‣ Leveraging Federated Learning for Automatic Detection of Clopidogrel Treatment Failures" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a> illustrates the performance of federated learning concerning the number of participating centers. While it is evident that conventional centralized training achieves the highest Area Under the Curve (AUC) values and converges most rapidly when compared to models trained through federated learning, it is noteworthy that the performance gap between these approaches can be substantially narrowed under certain federated learning configurations.</p>
</div>
<div id="S4.p6" class="ltx_para">
<p id="S4.p6.1" class="ltx_p">Specifically, our FCN model achieved an AUC of 0.777 with the involvement of all 22 participating centers, while the GRU model reached an impressive AUC of 0.940 with the participation of just 8 centers. However, it’s important to observe that the performance starts to degrade as the number of participating centers increases beyond a certain point, especially for the GRU architecture. This trend suggests that the simple averaging method employed to consolidate model parameters from multiple centers may not be the most effective solution, particularly when models from some participating centers exhibit poor performance, as observed in the case of the GRU architecture.</p>
</div>
<div id="S4.p7" class="ltx_para">
<p id="S4.p7.1" class="ltx_p">As part of our future research endeavors, we plan to delve deeper into the consolidation of model parameters, exploring methodologies that assign varying weights to these parameters based on their individual performance. This approach holds the potential to further enhance the performance of federated learning models in scenarios involving a diverse range of participating centers.</p>
</div>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">V </span><span id="S5.1.1" class="ltx_text ltx_font_smallcaps">Conclusion</span>
</h2>

<div id="S5.p1" class="ltx_para">
<p id="S5.p1.1" class="ltx_p">Our study highlighted the potential of federated learning in addressing clopidogrel treatment failure detection, offering a promising avenue for enhancing patient care through personalized treatment strategies while respecting data privacy. As we look to the future, our research paves the way for further investigations into advanced model consolidation techniques that can optimize federated learning in healthcare scenarios.</p>
</div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">
Samuel Kim, In Gu Sean Lee, Mijeong Irene Ban, and Jane Chiang,

</span>
<span class="ltx_bibblock">“Detection and prediction of clopidogrel treatment failures using longitudinal structured electronic health records,” 2023.

</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">
Sunwoong Choi and Samuel Kim,

</span>
<span class="ltx_bibblock">“Data augmentation method for modeling health records with applications to clopidogrel treatment failure detection,” 2024.

</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">
H. Brendan McMahan, Eider Moore, Daniel Ramage, Seth Hampson, and Blaise Agüera y Arcas,

</span>
<span class="ltx_bibblock">“Communication-Efficient Learning of Deep Networks from Decentralized Data,” Feb. 2017,

</span>
<span class="ltx_bibblock">arXiv:1602.05629 [cs].

</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">
Jie Xu, Benjamin S. Glicksberg, Chang Su, Peter Walker, Jiang Bian, and Fei Wang,

</span>
<span class="ltx_bibblock">“Federated Learning for Healthcare Informatics,”

</span>
<span class="ltx_bibblock"><span id="bib.bib4.1.1" class="ltx_text ltx_font_italic">Journal of Healthcare Informatics Research</span>, vol. 5, no. 1, pp. 1–19, Mar. 2021.

</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">
Syed Irfan Ali Meerza, Zhuohang Li, Luyang Liu, Jiaxin Zhang, and Jian Liu,

</span>
<span class="ltx_bibblock">“Fair and Privacy-Preserving Alzheimer’s Disease Diagnosis Based on Spontaneous Speech Analysis via Federated Learning,”

</span>
<span class="ltx_bibblock">in <span id="bib.bib5.1.1" class="ltx_text ltx_font_italic">2022 44th Annual International Conference of the IEEE Engineering in Medicine &amp; Biology Society (EMBC)</span>, Glasgow, Scotland, United Kingdom, July 2022, pp. 1362–1365, IEEE.

</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">
Yoo Jeong Ha, Gusang Lee, Minjae Yoo, Soyi Jung, Seehwan Yoo, and Joongheon Kim,

</span>
<span class="ltx_bibblock">“Multi-Site Split Learning for Privacy-Preserving Medical Systems under Data Imbalance Constraints: A Feasibility Study with COVID-19, X-Ray, and Cholesterol Dataset,”

</span>
<span class="ltx_bibblock">preprint, In Review, Sept. 2021.

</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">
Ittai Dayan, Holger Roth, Aoxiao Zhong, Ahmed Harouni, Amilcare Gentili, Anas Z Abidin, Andrew Liu, Anthony Beardsworth Costa, Bradford J. Wood, Chien-Sung Tsai, Chih-Hung Wang, Chun-Nan Hsu, CK Lee, Peiying Ruan, Daguang Xu, Dufan Wu, Eddie Huang, Felipe Campos Kitamura, Griffin Lacey, Gustavo César de Antônio Corradi, Gustavo Nino, Hao-Hsin Shin, Hirofumi Obinata, Hui Ren, Jason C. Crane, Jesse Tetreault, Jiahui Guan, John W. Garrett, Joshua D Kaggie, Jung Gil Park, Keith Dreyer, Krishna Juluru, Kristopher Kersten, Marcio Aloisio Bezerra Cavalcanti Rockenbach, Marius George Linguraru, Masoom A. Haider, Meena AbdelMaseeh, Nicola Rieke, Pablo F. Damasceno, Pedro Mario Cruz e Silva, Pochuan Wang, Sheng Xu, Shuichi Kawano, Sira Sriswasdi, Soo Young Park, Thomas M. Grist, Varun Buch, Watsamon Jantarabenjakul, Weichung Wang, Won Young Tak, Xiang Li, Xihong Lin, Young Joon Kwon, Abood Quraini, Andrew Feng, Andrew N Priest, Baris Turkbey, Benjamin Glicksberg, Bernardo Bizzo, Byung Seok Kim, Carlos Tor-Díez,
Chia-Cheng Lee, Chia-Jung Hsu, Chin Lin, Chiu-Ling Lai, Christopher P. Hess, Colin Compas, Deepeksha Bhatia, Eric K Oermann, Evan Leibovitz, Hisashi Sasaki, Hitoshi Mori, Isaac Yang, Jae Ho Sohn, Krishna Nand Keshava Murthy, Li-Chen Fu, Matheus Ribeiro Furtado de Mendonça, Mike Fralick, Min Kyu Kang, Mohammad Adil, Natalie Gangai, Peerapon Vateekul, Pierre Elnajjar, Sarah Hickman, Sharmila Majumdar, Shelley L. McLeod, Sheridan Reed, Stefan Gräf, Stephanie Harmon, Tatsuya Kodama, Thanyawee Puthanakit, Tony Mazzulli, Vitor Lima de Lavor, Yothin Rakvongthai, Yu Rim Lee, Yuhong Wen, Fiona J Gilbert, Mona G. Flores, and Quanzheng Li,

</span>
<span class="ltx_bibblock">“Federated learning for predicting clinical outcomes in patients with COVID-19,”

</span>
<span class="ltx_bibblock"><span id="bib.bib7.1.1" class="ltx_text ltx_font_italic">Nature medicine</span>, vol. 27, no. 10, pp. 1735–1743, Oct. 2021.

</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">
Sarthak Pati,

</span>
<span class="ltx_bibblock">“Federated learning enables big data for rare cancer boundary detection,”

</span>
<span class="ltx_bibblock"><span id="bib.bib8.1.1" class="ltx_text ltx_font_italic">Nature Communications</span>, 2022.

</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">
Cathie Sudlow et al.,

</span>
<span class="ltx_bibblock">“Uk biobank: An open access resource for identifying the causes of a wide range of complex diseases of middle and old age,”

</span>
<span class="ltx_bibblock"><span id="bib.bib9.1.1" class="ltx_text ltx_font_italic">PLOS Medicine</span>, vol. 12, no. 3, pp. 1–10, 03 2015.

</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">
Holger R. Roth, Yan Cheng, Yuhong Wen, Isaac Yang, Ziyue Xu, Yuan-Ting Hsieh, Kristopher Kersten, Ahmed Harouni, Can Zhao, Kevin Lu, Zhihong Zhang, Wenqi Li, Andriy Myronenko, Dong Yang, Sean Yang, Nicola Rieke, Abood Quraini, Chester Chen, Daguang Xu, Nic Ma, Prerna Dogra, Mona Flores, and Andrew Feng,

</span>
<span class="ltx_bibblock">“NVIDIA FLARE: Federated Learning from Simulation to Real-World,” Apr. 2023,

</span>
<span class="ltx_bibblock">arXiv:2210.13291 [cs].

</span>
</li>
</ul>
</section>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/2403.03367" class="ar5iv-nav-button ar5iv-nav-button-prev">◄</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/2403.03368" class="ar5iv-text-button ar5iv-severity-ok">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+2403.03368">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/2403.03368" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/2403.03369" class="ar5iv-nav-button ar5iv-nav-button-next">►</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Fri Apr  5 14:57:49 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "×";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
