<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[2407.00116] Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges</title><meta property="og:description" content="This paper presents a comprehensive systematic review of generative models (GANs, VAEs, DMs, and LLMs) used to synthesize various medical data types, including imaging (dermoscopic, mammographic, ultrasound, CT, MRI, a…">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/2407.00116">

<!--Generated on Mon Aug  5 16:23:58 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_fleqn">
<h1 class="ltx_title ltx_title_document">Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges </h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Mahmoud Ibrahim <sup id="id1.1.id1" class="ltx_sup">*</sup>
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_email"><a href="mailto:mahmoud.ibrahim@vito.be">mahmoud.ibrahim@vito.be</a>
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Yasmina Al Khalil
</span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Sina Amirrajab
</span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Chang Sun
</span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Marcel Breeuwer
</span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Josien Pluim
</span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Bart Elen
</span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Gökhan Ertaylan
</span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Michel Dumontier
</span></span>
</div>

<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p id="id2.id1" class="ltx_p">This paper presents a comprehensive systematic review of generative models (GANs, VAEs, DMs, and LLMs) used to synthesize various medical data types, including imaging (dermoscopic, mammographic, ultrasound, CT, MRI, and X-ray), text, time-series, and tabular data (EHR). Unlike previous narrowly focused reviews, our study encompasses a broad array of medical data modalities and explores various generative models. Our search strategy queries databases such as Scopus, PubMed, and ArXiv, focusing on recent works from January 2021 to November 2023, excluding reviews and perspectives. This period emphasizes recent advancements beyond GANs, which have been extensively covered previously.</p>
<p id="id3.id2" class="ltx_p">The survey reveals insights from three key aspects: (1) Synthesis applications and purpose of synthesis, (2) generation techniques, and (3) evaluation methods. It highlights clinically valid synthesis applications, demonstrating the potential of synthetic data to tackle diverse clinical requirements. While conditional models incorporating class labels, segmentation masks and image translations are prevalent, there is a gap in utilizing prior clinical knowledge and patient-specific context, suggesting a need for more personalized synthesis approaches and emphasizing the importance of tailoring generative approaches to the unique characteristics of medical data. Additionally, there is a significant gap in using synthetic data beyond augmentation, such as for validation and evaluation of downstream medical AI models. The survey uncovers that the lack of standardized evaluation methodologies tailored to medical images is a barrier to clinical application, underscoring the need for in-depth evaluation approaches, benchmarking, and comparative studies to promote openness and collaboration.</p>
</div>
<div class="ltx_classification">
<h6 class="ltx_title ltx_title_classification">keywords: </h6>
Generative models , Synthetic data , Medical Data , EHR and Physiological Signals , Medical Imaging , Medical Text

</div>
<span id="id1" class="ltx_note ltx_note_frontmatter ltx_role_journal"><sup class="ltx_note_mark">†</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">†</sup><span class="ltx_note_type">journal: </span>Artificial Intelligence Review</span></span></span>
<div id="p1" class="ltx_para">
<span id="p1.1" class="ltx_ERROR undefined">\affiliation</span>
<p id="p1.2" class="ltx_p">[first]organization=Institute of Data Science, Faculty of Science and Engineering, Maastricht University,
city=Maastricht,
country=The Netherlands</p>
</div>
<div id="p2" class="ltx_para">
<span id="p2.1" class="ltx_ERROR undefined">\affiliation</span>
<p id="p2.2" class="ltx_p">[second]organization=Department of Advanced Computing Sciences, Faculty of Science and Engineering, Maastricht University,
city=Maastricht,
country=The Netherlands</p>
</div>
<div id="p3" class="ltx_para">
<span id="p3.1" class="ltx_ERROR undefined">\affiliation</span>
<p id="p3.2" class="ltx_p">[third]organization=VITO,
country=Belgium</p>
</div>
<div id="p4" class="ltx_para">
<span id="p4.1" class="ltx_ERROR undefined">\affiliation</span>
<p id="p4.2" class="ltx_p">[fourth]organization=Department of Biomedical Engineering, Eindhoven University of Technology
,
city=Eindhoven,
country=The Netherlands</p>
</div>
<nav class="ltx_TOC ltx_list_toc ltx_toc_toc"><h6 class="ltx_title ltx_title_contents">Contents</h6>
<ol class="ltx_toclist">
<li class="ltx_tocentry ltx_tocentry_section">
<a href="#S1" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1 </span>Introduction</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S1.SS1" title="In 1 Introduction ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1.1 </span>Related work</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S1.SS2" title="In 1 Introduction ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1.2 </span>Review methods and protocol</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S1.SS3" title="In 1 Introduction ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1.3 </span>Search results</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a href="#S2" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2 </span>Synthesis application, generative technologies and evaluations</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S2.SS1" title="In 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.1 </span>Synthesis applications and purpose of synthesis</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection">
<a href="#S2.SS2" title="In 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2 </span>Generation techniques</span></a>
<ol class="ltx_toclist ltx_toclist_subsection">
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S2.SS2.SSS1" title="In 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2.1 </span>Differences in generative models working mechanism per datatype</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S2.SS2.SSS2" title="In 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2.2 </span>GANs</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S2.SS2.SSS3" title="In 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2.3 </span>Diffusion models</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S2.SS2.SSS4" title="In 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2.4 </span>Language models:</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S2.SS3" title="In 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.3 </span>Evaluation</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a href="#S3" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3 </span>Results</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection">
<a href="#S3.SS1" title="In 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1 </span>EHR</span></a>
<ol class="ltx_toclist ltx_toclist_subsection">
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS1.SSS1" title="In 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1.1 </span>Generative models- Longitudinal format</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS1.SSS2" title="In 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1.2 </span>Generative models- Time-dependent format</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS1.SSS3" title="In 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1.3 </span>Generative models- Aggregated format</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS1.SSS4" title="In 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1.4 </span>Generative models- Snapshot format</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS1.SSS5" title="In 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1.5 </span>Diverse papers from diverse formats</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_subsection">
<a href="#S3.SS2" title="In 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.2 </span>Time-series sgnals</span></a>
<ol class="ltx_toclist ltx_toclist_subsection">
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS2.SSS1" title="In 3.2 Time-series sgnals ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.2.1 </span>ECGs</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS2.SSS2" title="In 3.2 Time-series sgnals ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.2.2 </span>EEGs and other signals</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_subsection">
<a href="#S3.SS3" title="In 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3 </span>Medical imaging</span></a>
<ol class="ltx_toclist ltx_toclist_subsection">
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS3.SSS1" title="In 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3.1 </span>Dermatoscopy</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS3.SSS2" title="In 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3.2 </span>Mammography</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS3.SSS3" title="In 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3.3 </span>Ultrasound</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS3.SSS4" title="In 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3.4 </span>MRI</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS3.SSS5" title="In 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3.5 </span>CT</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS3.SSS6" title="In 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3.6 </span>X-Ray</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS3.SSS7" title="In 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3.7 </span>OCT</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S3.SS3.SSS8" title="In 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3.8 </span>Multiple Modalities</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S3.SS4" title="In 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.4 </span>Medical text</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a href="#S4" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4 </span>Discussion</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S4.SS1" title="In 4 Discussion ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.1 </span>Synthesis applications and purpose of synthesis</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S4.SS2" title="In 4 Discussion ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.2 </span>Generation techniques</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S4.SS3" title="In 4 Discussion ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.3 </span>Evaluation</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section"><a href="#S5" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5 </span>Recommendations</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a href="#S6" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">6 </span>Conclusion</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a href="#S7" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">7 </span>Acknowledgements</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a href="#S8" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">8 </span>Statements and Declarations</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a href="#S9" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">9 </span>CRediT author statement</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section">
<a href="#S10" title="In Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">10 </span>Appendix</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a href="#S10.SS1" title="In 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">10.1 </span>Evolution of the usage of generation techniques over years</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection">
<a href="#S10.SS2" title="In 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">10.2 </span>Synthetic Data Evaluation</span></a>
<ol class="ltx_toclist ltx_toclist_subsection">
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S10.SS2.SSS1" title="In 10.2 Synthetic Data Evaluation ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">10.2.1 </span>Privacy Metrics</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsubsection"><a href="#S10.SS2.SSS2" title="In 10.2 Synthetic Data Evaluation ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">10.2.2 </span>Fidelity Metrics</span></a></li>
</ol>
</li>
</ol>
</li>
</ol></nav>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>

<div id="S1.p1" class="ltx_para">
<p id="S1.p1.1" class="ltx_p">The advent of Artificial Intelligence (AI) and Machine Learning (ML) has revolutionized numerous fields, including healthcare. These technologies have the potential to transform medical research and clinical practice, offering new avenues for diagnosis, treatment, and patient care. However, the application of AI and ML in healthcare depends on the availability of large and high-quality datasets, with diverse modalities and acquisition properties. In many instances, such datasets are not readily available due to privacy concerns, restricted sharing policies, complex acquisition techniques, expensive annotation costs, as well as limited diversity in real world data. This has led to the emergence of synthetic data generation, a promising solution that leverages generative models to create artificial data that mimics real-world datasets.</p>
</div>
<div id="S1.p2" class="ltx_para">
<p id="S1.p2.1" class="ltx_p">Synthetic data could serve various critical purposes in data science and machine learning, mainly facilitating data sharing while protecting privacy, augmenting existing datasets, and promoting fairness and equity in AI applications <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib1" title="" class="ltx_ref">1</a>]</cite>.
Generative models are computational algorithms capable of learning and capturing complex data distributions, enabling the generation of new samples that closely resemble real data. By leveraging techniques such as Generative Adversarial Networks (GANs), Variational Autoencoders (VAEs), Diffusion Models (DMs), and Large Language Models (LLMs), researchers can create synthetic medical data across various modalities, including imaging, text, time-series, and tabular data.</p>
</div>
<div id="S1.p3" class="ltx_para">
<p id="S1.p3.1" class="ltx_p">Our survey aims to provide a holistic understanding of the applications of these generative models in generating medical synthetic data. We delve into three key aspects: <span id="S1.p3.1.1" class="ltx_text ltx_font_italic">the purpose of synthesis, generation techniques, and evaluation methods</span>. We highlight the potential of synthetic data in addressing various clinical needs and identify gaps in current practices, such as the need for more personalized synthesis approaches and standardized evaluation methodologies. Moreover, we emphasize the importance of tailoring generative approaches to the unique characteristics of medical data and call for more in-depth evaluation approaches relevant to clinical applications. Our study encourages benchmarking and comparative studies to promote openness and collaboration in this field.</p>
</div>
<div id="S1.p4" class="ltx_para">
<p id="S1.p4.1" class="ltx_p">In essence, this survey paper serves as a valuable resource for researchers and practitioners interested in leveraging generative models for synthesizing medical data. By shedding light on the current practices, potential, and challenges in the field of synthetic medical data generation, we hope to spur further research and innovation in this critical area of healthcare AI and ML.</p>
</div>
<div id="S1.p5" class="ltx_para">
<p id="S1.p5.1" class="ltx_p">The structure of this paper is further organized into different chapters. Section <a href="#S2" title="2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> provides an overview of the synthesis applications, generative models, and evaluation methods that are common across the different data types. The concepts explained in this section are essential to understand the detailed results in section <a href="#S3" title="3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, where the findings from the surveyed papers are presented and divided into four sections, each focusing on a specific type of medical data: Electronic Health Records (EHR) in section <a href="#S3.SS1" title="3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.1</span></a>, physiological signals in section <a href="#S3.SS2" title="3.2 Time-series sgnals ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.2</span></a>, medical images in section <a href="#S3.SS3" title="3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3</span></a>, and medical text in section <a href="#S3.SS4" title="3.4 Medical text ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.4</span></a>. This organization allows for a detailed exploration of the use of generative models for each data type, providing readers with a comprehensive understanding of the current state of synthetic medical data generation. Section <a href="#S4" title="4 Discussion ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a> reveal the insights and conclusions collected from the surveyed papers. Section <a href="#S5" title="5 Recommendations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a> provides recommendations to be taken into account for future research, and <a href="#S6" title="6 Conclusion ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a> concludes the paper. The reader is referred to the table of contents for a smooth navigation of the paper.</p>
</div>
<section id="S1.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.1 </span>Related work</h3>

<figure id="S1.F1" class="ltx_figure"><img src="/html/2407.00116/assets/related.jpg" id="S1.F1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="323" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 1: </span>Specialization trends in related survey papers- The papers are either focused on a single data type, or combine a single data type with one specific technology, or are exclusively dedicated to data augmentation applications.</figcaption>
</figure>
<div id="S1.SS1.p1" class="ltx_para">
<p id="S1.SS1.p1.1" class="ltx_p">Despite the growing interest in generative models for medical data synthesis, the review of existing survey papers in the field reveals a notable trend: a tendency towards narrow specialization, as seen in Fig.<a href="#S1.F1" title="Fig. 1 ‣ 1.1 Related work ‣ 1 Introduction ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>. These papers often focus on a single data type, such as brain and heart imaging <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib2" title="" class="ltx_ref">2</a>]</cite> or EHR <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite>, or they combine a single data type with one specific technology, like medical images with diffusion models <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib4" title="" class="ltx_ref">4</a>]</cite> , EHR with GANs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite>, or Electrocardiography (ECG) with GANs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib6" title="" class="ltx_ref">6</a>]</cite>. Additionally, there is a subset of papers dedicated exclusively to data augmentation applications, which similarly specialize in either a single data type <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib7" title="" class="ltx_ref">7</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite> or a combination of one modality and one technology <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib9" title="" class="ltx_ref">9</a>]</cite>.</p>
</div>
<div id="S1.SS1.p2" class="ltx_para">
<p id="S1.SS1.p2.1" class="ltx_p">This pattern indicates a gap in the literature where comprehensive, multi-faceted analyses providing holistic overviews are less prevalent. In contrast, the proposed survey paper seeks to address this gap by covering a broader spectrum. It spans multiple data types, encompassing medical imaging, tabular EHRs, physiological signals, and clinical text notes. Furthermore, it explores various generative models, extending beyond GANs which have been extensively covered in literature, to include diffusion models and language models. The survey also emphasizes the aspect of conditional generation, which is not focused on in similar work. Additionally, we have considered a specific timeline for our review, consciously excluding older papers that have already been covered in various surveys. This approach aims to provide a more holistic understanding of the application of generative models in medical research, moving away from the trend of focusing on singular aspects. Such a comprehensive analysis could significantly enrich the medical field by offering insights that are potentially overlooked by more narrowly focused studies.</p>
</div>
</section>
<section id="S1.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.2 </span>Review methods and protocol</h3>

<div id="S1.SS2.p1" class="ltx_para">
<p id="S1.SS2.p1.1" class="ltx_p">Our review methodology and protocol aimed to address several key objectives and research questions. Firstly, we sought to explore the applications and purpose of synthesis beyond data augmentation in medical research. Additionally, we aimed to identify the latest generative models utilized for generating synthetic medical data. Lastly, we aimed to investigate the common protocols for evaluating synthetic data and the trade-offs between the different evaluation dimensions.</p>
</div>
<div id="S1.SS2.p2" class="ltx_para">
<p id="S1.SS2.p2.1" class="ltx_p">Our review is scoped to cover a broad spectrum of modalities, including tabular data (specifically EHRs), physiological signals (primarily ECGs and Electroencephalography (EEGs)), clinical text notes, and a variety of medical images such as dermatoscopic images, mammographic images, Utrasound (US) scans, Computed Tomography (CT) scans, Magnetic Resonance Imaging (MRI) scans, Optical Coherence Tomography (OCT) scans, and X-rays. We consider an array of generative models, including GANs, VAEs, DMs, and LLMs.</p>
</div>
<div id="S1.SS2.p3" class="ltx_para">
<p id="S1.SS2.p3.1" class="ltx_p">Our search strategy involves querying several databases, namely Scopus, PubMed, and ArXiv. We repeated our search query for each modality, using terms related to “synthetic data generation,” “conditional generation,” “generative models,” and other related keywords. To ensure the relevance and timeliness of our findings, we limit our search to papers and preprints published within the time frame of January 2021 to November 2023. Furthermore, we exclude review papers from our search to maintain our focus on original research and primary studies. This approach ensures a comprehensive and up-to-date review of the field.</p>
</div>
</section>
<section id="S1.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.3 </span>Search results</h3>

<div id="S1.SS3.p1" class="ltx_para">
<p id="S1.SS3.p1.1" class="ltx_p">The details of the literature screening processes according to the Preferred Reporting Items for Systematic reviews and Meta-Analyses (PRISMA) guidelines <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib10" title="" class="ltx_ref">10</a>]</cite> are shown in Fig.<a href="#S1.F2" title="Fig. 2 ‣ 1.3 Search results ‣ 1 Introduction ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>. Overall, 249 papers are included in the review, distributed over the different modalities as shown in Fig.<a href="#S1.F3" title="Fig. 3 ‣ 1.3 Search results ‣ 1 Introduction ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>.</p>
</div>
<figure id="S1.F2" class="ltx_figure"><img src="/html/2407.00116/assets/PRISMA_OVERALL.jpg" id="S1.F2.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="419" height="267" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 2: </span>PRISMA flowchart of the literature screening process</figcaption>
</figure>
<figure id="S1.F3" class="ltx_figure"><img src="/html/2407.00116/assets/output.png" id="S1.F3.g1" class="ltx_graphics ltx_centering ltx_img_square" width="538" height="503" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 3: </span>An overview of the number of surveyed papers per modality</figcaption>
</figure>
</section>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Synthesis application, generative technologies and evaluations</h2>

<div id="S2.p1" class="ltx_para">
<p id="S2.p1.1" class="ltx_p">This section presents a strong foundational understanding of key concepts gathered from various surveyed papers concerning the overarching themes of (i) synthesis applications, (ii) generative technologies, and (iii) evaluation methodologies in medical data synthesis. The objective is to lay the groundwork by explaining fundamental concepts and approaches that are commonly encountered in the survey. This aids the reader in building the necessary background to understand the detailed and more specific findings presented in section <a href="#S3" title="3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>.
First, we discusses different synthesis applications in section <a href="#S2.SS1" title="2.1 Synthesis applications and purpose of synthesis ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.1</span></a>, followed by an exploration of common generative models <a href="#S2.SS2" title="2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.2</span></a>. Finally, in section <a href="#S2.SS3" title="2.3 Evaluation ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.3</span></a>, we delve into the diverse evaluation methods employed in the surveyed studies, offering a comprehensive view of the research landscape in this field.</p>
</div>
<div id="S2.p2" class="ltx_para">
<p id="S2.p2.1" class="ltx_p">In our paper, we introduce specific definitions for the terms ”data types” and ”modalities”. We classify EHR, images, text, and signals as distinct data types. Within each data type, we define various modalities. For example, within medical images, modalities include CT, MRI, and others. Additionally, within each modality, there exist different modalities; for example, within MRI, modalities such as T1 and T2 weights are distinguished.</p>
</div>
<section id="S2.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.1 </span>Synthesis applications and purpose of synthesis</h3>

<figure id="S2.F4" class="ltx_figure"><img src="/html/2407.00116/assets/synthesis_applications_new.jpg" id="S2.F4.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="473" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 4: </span>(a) The different data types and modalities covered in the survey. The different EHR formats also represent the synthesis applications of EHR. (b) Overview of synthesis applications of the imaging and signals data types. The left side focuses on imaging data, while the right side focuses on signals. An example of inter-modal synthesis involves the transformation of a brain CT scan to an MRI in imaging or an ECG signal into a synthetic PPG signal in the signals data type, which helps in scenarios where certain imaging or signal modalities might be unavailable. Intra-modal synthesis involves the conversion of a brain MRI T2 sequence into a T1 sequence or transforming a single lead into a 10 lead synthetic ECG signal. Attribute-conditioned synthesis shows a patient-specific brain MRI or an ECG signal being generated that matches attributes like age, BMI, ethnicity, and gender. Class or semantic map-conditioned synthesis shows a synthetic brain MRI with tumor being generated using a binary mask of a brain tumor or a synthetic ECG signal being generated based on class labels, such as C1 and C2. This can be useful for generating labeled datasets for training. The figure also illustrates text-guided synthesis, such as generating a synthetic chest X-ray based on a textual description like ”moderate bilateral pleural effusions” or a synthetic ECG signal based on textual descriptions like ”Left Bundle Branch Block”. (c) Synthesis applications of the text data type</figcaption>
</figure>
<figure id="S2.F5" class="ltx_figure"><img src="/html/2407.00116/assets/conditionalvsunconditional.png" id="S2.F5.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="299" height="102" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 5: </span>Unconditional (left) vs conditional generation (right)</figcaption>
</figure>
<div id="S2.SS1.p1" class="ltx_para">
<p id="S2.SS1.p1.1" class="ltx_p">Generative models are a powerful tool in machine learning and can be broadly categorized into two types: unconditional and conditional. Unconditional models take a random variable as input, allowing for the application of unconditional synthesis. On the other hand, conditional generative models introduce an additional layer of control by incorporating external information or context during the generation process that serve as additional guidance for the model. These can include images, text, semantic maps, class labels, attributes, and signals, as demonstrated in Fig.<a href="#S2.F5" title="Fig. 5 ‣ 2.1 Synthesis applications and purpose of synthesis ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>. This added control allows conditional models to be used in a variety of synthesis applications specific to each data type shown in Fig.<a href="#S2.F4" title="Fig. 4 ‣ 2.1 Synthesis applications and purpose of synthesis ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.</p>
</div>
<div id="S2.SS1.p2" class="ltx_para">
<p id="S2.SS1.p2.1" class="ltx_p">For the EHR data type, the synthesis depends on the EHR format, which includes (i) Longitudinal EHR, consisting of medical codes from various patient visits; (ii) Aggregated EHR, which consists of longitudinal visits of the patients condensed into a single row; (iii) Time-dependent EHR, which consists of time-series readings from one patient’s visit; and (iv) Snapshot EHR consisting a single snapshot of a patient’s EHR focusing on selected attributes.</p>
</div>
<div id="S2.SS1.p3" class="ltx_para">
<p id="S2.SS1.p3.1" class="ltx_p">Regarding the physiological signals and imaging data types, prominent applications include (i) Inter-modal translation which involves converting data from one modality to another. For example, in the realm of images, this could involve translating CT scans to MR images. In the signals data type, this could mean converting ECG readings to PPG signals.
(ii) Intra-modal translation which involves translating data within the same modality. For instance, in images, this could mean translating across different MRI contrasts: T1-weighted images to T2-weighted images. In signals , this could involve converting single-lead ECG inputs into a complete 12-lead set.
(iii) Class or semantic map-based synthesis that involves generating data conditioned on a certain class label or a certain segmentation mask.
(iv) Attribute-based Synthesis where the generated data is conditioned to correspond with specific subject characteristics and patient demographics, such as age, sex, and Body Mass Index (BMI). These approaches suggest a higher level of personalization in the synthetic data, potentially leading to more accurate representations of patient-specific medical data.
(v) Text-based synthesis demonstrates innovative approach for integrating descriptive text into synthetic medical data generation by conditioning on clinical text reports for example. This could involve text to image, text to signal, or text to table synthesis.</p>
</div>
<div id="S2.SS1.p4" class="ltx_para">
<p id="S2.SS1.p4.1" class="ltx_p">For the medical text synthesis, we categorize the applications based on their generation purposes into (i) Natural Language Processing (NLP) enhancement, where the generated synthetic text data is used to improve the performance of NLP tasks. A typical NLP task in medical text generation includes name entity recognition, information / concept extraction, relation extraction, semantic similarity, summarization, and question answering. For example, the Name Entity Recognition (NER) performance on real clinical notes are limited by the insufficient amount and completeness of the dataset. By adding generated synthetic medical text can improve the NER task in this scenario; (ii) text augmentation, where the generated text is used to augment the existing text such as generating discharge summaries, patients reports, clinical notes and case reports of certain diseases especially when the real-world clinical notes are limited and sensitive to share or use for research; (iii) text de-identification by replacing personal identifiable and sensitive information about individuals in order to preserve patients privacy in the clinical text. These applications have a focus on particular attributes (such as names, address, or diagnosis) in the text .</p>
</div>
</section>
<section id="S2.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.2 </span>Generation techniques</h3>

<div id="S2.SS2.p1" class="ltx_para">
<p id="S2.SS2.p1.1" class="ltx_p">Our examination of various papers revealed a range of techniques utilized for generating synthetic medical data, with a primary focus on GANs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib11" title="" class="ltx_ref">11</a>]</cite>, VAEs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib12" title="" class="ltx_ref">12</a>]</cite> and recent advancements in diffusion models <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib13" title="" class="ltx_ref">13</a>]</cite> and language models. See Fig.<a href="#S2.F6" title="Fig. 6 ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a> for the basic working mechanisms of the different generative models.</p>
</div>
<figure id="S2.F6" class="ltx_figure"><img src="/html/2407.00116/assets/generative_models.jpg" id="S2.F6.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="299" height="204" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 6: </span>Generative Models. GANs follow adversarial training to implicitly model the real distribution. VAEs employs variational inference techniques <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib14" title="" class="ltx_ref">14</a>, <a href="#bib.bib15" title="" class="ltx_ref">15</a>]</cite> to approximate real distribution, while diffusion models gradually add noise to the input and reverse it back</figcaption>
</figure>
<div id="S2.SS2.p2" class="ltx_para">
<p id="S2.SS2.p2.1" class="ltx_p">Firstly, we discuss the differences in how various generative models operate across different datatypes: EHR, medical images, medical text, and physiological signals.
Next, we highlight notable state-of-the-art GANs frequently referenced in the literature.
Following this, we explore diffusion models, including denoising diffusion probabilistic models and latent diffusion models, alongside discussions on different text embedding methods.
Finally, we examine language models, concluding our overview of the diverse techniques employed in generating synthetic medical data.</p>
</div>
<section id="S2.SS2.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.2.1 </span>Differences in generative models working mechanism per datatype</h4>

<div id="S2.SS2.SSS1.p1" class="ltx_para">
<ul id="S2.I1" class="ltx_itemize">
<li id="S2.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S2.I1.i1.p1" class="ltx_para">
<p id="S2.I1.i1.p1.1" class="ltx_p"><span id="S2.I1.i1.p1.1.1" class="ltx_text ltx_font_bold">Handling discrete variables:</span> GANs, initially introduced for generating realistic continuous images, face challenges when applied to discrete data. Similarly, many diffusion models are Gaussian processes, operating in continuous spaces and not the discrete space. To address these issues, various techniques have been introduced. (Variational) autoencoders are commonly employed before GANs and diffusion models to condense high-dimensional and heterogeneous features into latent representations. Notably, some diffusion models like Tabular Denoising Diffusion Probabilistic Models (tabDDPM) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite> adopt a mixed sequence diffusion approach <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib17" title="" class="ltx_ref">17</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>]</cite>, employing Gaussian diffusion for continuous variables and multinomial diffusion <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib19" title="" class="ltx_ref">19</a>]</cite> for discrete variables. Others treat discrete variables similar to real-valued sequences but with further post-processing of the model output, transforming continuous variables into discrete.</p>
</div>
</li>
<li id="S2.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S2.I1.i2.p1" class="ltx_para">
<p id="S2.I1.i2.p1.1" class="ltx_p"><span id="S2.I1.i2.p1.1.1" class="ltx_text ltx_font_bold">Handling time aspect:</span> Notably, conventional vanilla GANs and diffusion models lack the ability to generate time-series data, making them unsuitable for directly creating time-dependent EHR data or signals. As a remedy, specialized sequential Deep Learning (DL) models like Recurrent Neural Networks (RNNs), Gated Recurrent Units (GRUs), and Long Short-term Memory(LSTMs) are employed within the architectures of the GANs or diffusion models.</p>
</div>
</li>
</ul>
</div>
</section>
<section id="S2.SS2.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.2.2 </span>GANs</h4>

<figure id="S2.F7" class="ltx_figure"><img src="/html/2407.00116/assets/gan_overview.jpg" id="S2.F7.g1" class="ltx_graphics ltx_centering ltx_img_square" width="598" height="558" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 7: </span>Comprehensive overview of various GAN architectures, each tailored for specific applications and improvements in synthetic data generation. The Vanilla GAN (a) illustrates the basic GAN structure with a generator creating samples from noise and a discriminator evaluating them. The Conditional GAN (b) integrates conditional variables to guide the generation process, enabling the production of more targeted outputs. The PGGAN (c) employs a series of generator and discriminator pairs (G1, D1 to Gn, Dn) that progressively increase the resolution and detail of the generated samples. CycleGAN (d) features a dual-generator system for effective style transfer, where each generator learns to translate images between two distinct domains, and the outputs are cycled back as inputs to maintain consistency. The Pix2Pix (e) uses aligned pairs of images for precise image-to-image translation tasks, converting segmented images to realistic photos using a trained generator. The Generator G is usually an encoder-decoder net or U-Net with skip connection. DCGAN, LSGAN, and WGAN (f), use different loss functions like BCE, MSE, and Wasserstein to refine the discriminator’s accuracy. StyleGAN (g) features a sophisticated generator with a mapping network that inputs noise, refines the generation through style blocks, and uses a discriminator with Wasserstein loss. SPADEGAN (h ) utilizes spatially-adaptive normalization to modulate the synthesis based on the input segmentation map, with a discriminator employing Wasserstein loss. VQ-GAN (i) merges GAN and VAE elements by encoding inputs into a latent space, quantizing them for efficient representation, and then reconstructing the output to improve image quality. x and x_syn, as well as y and y_syn denote input and synthesized data pairs, while c represents the conditional variable. Similarly, x_1 and x_s1, x_2 and x_s2, x_n and x_sn represent real and synthesized data pairs. </figcaption>
</figure>
<div id="S2.SS2.SSS2.p1" class="ltx_para">
<p id="S2.SS2.SSS2.p1.1" class="ltx_p">Adversarial approaches, such as GANs, involve a generator and discriminator, trained to outperform each other, hence the term ”adversarial”. The generator network adapts its distribution and generates a new, reliable distribution, while the discriminator network learns to differentiate between real and augmented data. The work in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib11" title="" class="ltx_ref">11</a>]</cite> pioneered GANs for generating realistic images, particularly dealing with continuous data, often referred to as the vanilla GAN (Fig.<a href="#S2.F7" title="Fig. 7 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>a). However, such models have been extended to different domains, such as audio and EEG signal generation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib20" title="" class="ltx_ref">20</a>, <a href="#bib.bib21" title="" class="ltx_ref">21</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib22" title="" class="ltx_ref">22</a>]</cite> proposed the Deep Convolutional GAN (DCGAN), that combines the original GAN with convolutional neural networks for better and more stable training. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib23" title="" class="ltx_ref">23</a>]</cite> proposed the Wasserstein GAN (WGAN), while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib24" title="" class="ltx_ref">24</a>]</cite> proposed the Least Squares GAN (LSGAN), to address challenges like mode collapse and vanishing gradients typically encountered in GANs (see Fig.<a href="#S2.F7" title="Fig. 7 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>f). WGAN replaces binary classification with the Wasserstein distance. The training of WGAN was enhanced with the introduction of WGAN-GP in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib25" title="" class="ltx_ref">25</a>]</cite>, which integrates a gradient penalty technique. Progressive Growing GAN (PGGAN)<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib26" title="" class="ltx_ref">26</a>]</cite> is an extension of GAN training, ensuring stability for generators producing large, high-quality images. Traditional GANs struggle with stability when working with larger sizes, attempting to balance both structure and fine details. This challenge worsens as resolutions increase, often required for medical image generation, leading to training failures, while memory constraints on Graphics Processing Units (GPUs) further necessitate reducing batch size, introducing instability. As illustrated in Fig.<a href="#S2.F7" title="Fig. 7 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>c, PGGAN addresses these issues by incrementally increasing model size during training, starting small and gradually adding layers until achieving the desired image size.</p>
</div>
<div id="S2.SS2.SSS2.p2" class="ltx_para">
<p id="S2.SS2.SSS2.p2.1" class="ltx_p">GANs enable the generation of diverse image, video, or audio data from random input sampled from a normal distribution <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib27" title="" class="ltx_ref">27</a>]</cite>. However, vanilla, unconditional GANs lack control over the generated samples’ appearance or class. To address this, Conditional Generative Adversarial Networks (cGANs) are introduced, which allow for conditional generation by incorporating semantic input or a condition (c), like image class, into the generation process <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib28" title="" class="ltx_ref">28</a>]</cite>, as shown in Fig.<a href="#S2.F7" title="Fig. 7 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>b. Both the generator and discriminator in cGANs are conditioned on auxiliary information, enabling the model to learn complex mappings from diverse contextual inputs to corresponding outputs.</p>
</div>
<div id="S2.SS2.SSS2.p3" class="ltx_para">
<p id="S2.SS2.SSS2.p3.1" class="ltx_p">Conditional GANs have popularized the use of image conditioning in image-to-image translation tasks. For such tasks, models like Pix2Pix <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib29" title="" class="ltx_ref">29</a>]</cite> (see Fig.<a href="#S2.F7" title="Fig. 7 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>e) and its high-resolution counterpart Pix2PixHD <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib30" title="" class="ltx_ref">30</a>]</cite> rely on supervised training with paired datasets. Pix2PixHD addresses some limitations of Pix2Pix, specifically catering to high-resolution image translation. CycleGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib31" title="" class="ltx_ref">31</a>]</cite> offers an alternative by excelling in unsupervised image translation, eliminating the need for paired datasets. CycleGAN utilizes cycle consistency, allowing generated images from one generator to serve as input for the other, with the output matching the original image, enabling consistency in both directions, as shown in Fig.<a href="#S2.F7" title="Fig. 7 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>d. This capability proves valuable in scenarios where obtaining paired training data is challenging. While CycleGAN shines in unsupervised learning, it is not optimal for high-resolution image-to-image translation. Consequently, paired approaches are often preferred for medical data generation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib32" title="" class="ltx_ref">32</a>]</cite>.</p>
</div>
<div id="S2.SS2.SSS2.p4" class="ltx_para">
<p id="S2.SS2.SSS2.p4.1" class="ltx_p">Other notable GAN-based approaches frequently employed for medical data generation, include StyleGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib33" title="" class="ltx_ref">33</a>]</cite>, SPADE GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib34" title="" class="ltx_ref">34</a>]</cite>, and VQ-GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib35" title="" class="ltx_ref">35</a>]</cite>. Style Generative Adversarial Network (StyleGAN) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib33" title="" class="ltx_ref">33</a>]</cite>, depicted in Fig.<a href="#S2.F7" title="Fig. 7 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>g, represents a significant extension to the GAN architecture. It introduces changes like a mapping network for latent space, allowing control over style at various points in the generator model, and the incorporation of noise for variation. The StyleGAN generator and discriminator models are trained using the PGGAN method. This model not only produces high-quality, high-resolution images but also provides control over style at different detail levels through manipulation of style vectors and noise. A cutting-edge image translation model, akin to Pix2Pix, is SPADE GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib34" title="" class="ltx_ref">34</a>]</cite>, which introduces spatially-adaptive (SPADE) normalization. Unlike previous models, which employed segmentation maps only at the input layer, SPADE GAN ensures the segmentation map is inputted across all intermediate layers, preserving its information throughout the model’s depth. (see Fig.<a href="#S2.F7" title="Fig. 7 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>h).</p>
</div>
<div id="S2.SS2.SSS2.p5" class="ltx_para">
<p id="S2.SS2.SSS2.p5.1" class="ltx_p">The VAE introduces a framework for compressing data into a lower-dimensional space <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib12" title="" class="ltx_ref">12</a>]</cite>. The Vector-Quantized Variational Autoencoder (VQ-VAE) model enhances representation by mapping inputs to a continuous space and then discretizing these into tokens, facilitated by an encoder, decoder, and a learnable codebook <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib36" title="" class="ltx_ref">36</a>]</cite>. The Vector-Quantized Generative Adversarial Network (VQ-GAN) fuses VAE and GAN models with vector quantization (VQ) to create data, including images, from noise, and employs VQ to elevate the quality by turning outputs into discrete values. This technique not only improves the definition and edges of generated data but also provides examples like images with more precise and clearer contours compared to outputs from conventional GANs, benefiting from the combined strengths of GANs and VQ’s detailed representation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib35" title="" class="ltx_ref">35</a>]</cite>.</p>
</div>
<figure id="S2.F8" class="ltx_figure"><img src="/html/2407.00116/assets/DM.jpg" id="S2.F8.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="598" height="202" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 8: </span>Diffusion Models. <span id="S2.F8.21.1" class="ltx_text ltx_font_bold">a,b,c</span> constitute the components of the Stable diffusion pipeline. <span id="S2.F8.22.2" class="ltx_text ltx_font_bold">a</span>: (Variational)AutoEncoder component for compressing the high-dimensional inputs <math id="S2.F8.9.m1.1" class="ltx_Math" alttext="X_{0}" display="inline"><semantics id="S2.F8.9.m1.1b"><msub id="S2.F8.9.m1.1.1" xref="S2.F8.9.m1.1.1.cmml"><mi id="S2.F8.9.m1.1.1.2" xref="S2.F8.9.m1.1.1.2.cmml">X</mi><mn id="S2.F8.9.m1.1.1.3" xref="S2.F8.9.m1.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="S2.F8.9.m1.1c"><apply id="S2.F8.9.m1.1.1.cmml" xref="S2.F8.9.m1.1.1"><csymbol cd="ambiguous" id="S2.F8.9.m1.1.1.1.cmml" xref="S2.F8.9.m1.1.1">subscript</csymbol><ci id="S2.F8.9.m1.1.1.2.cmml" xref="S2.F8.9.m1.1.1.2">𝑋</ci><cn type="integer" id="S2.F8.9.m1.1.1.3.cmml" xref="S2.F8.9.m1.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.9.m1.1d">X_{0}</annotation></semantics></math> into the lower-dimensional latent representations <math id="S2.F8.10.m2.1" class="ltx_Math" alttext="Z_{0}" display="inline"><semantics id="S2.F8.10.m2.1b"><msub id="S2.F8.10.m2.1.1" xref="S2.F8.10.m2.1.1.cmml"><mi id="S2.F8.10.m2.1.1.2" xref="S2.F8.10.m2.1.1.2.cmml">Z</mi><mn id="S2.F8.10.m2.1.1.3" xref="S2.F8.10.m2.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="S2.F8.10.m2.1c"><apply id="S2.F8.10.m2.1.1.cmml" xref="S2.F8.10.m2.1.1"><csymbol cd="ambiguous" id="S2.F8.10.m2.1.1.1.cmml" xref="S2.F8.10.m2.1.1">subscript</csymbol><ci id="S2.F8.10.m2.1.1.2.cmml" xref="S2.F8.10.m2.1.1.2">𝑍</ci><cn type="integer" id="S2.F8.10.m2.1.1.3.cmml" xref="S2.F8.10.m2.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.10.m2.1d">Z_{0}</annotation></semantics></math> and mapping them back to the pixel space. <math id="S2.F8.11.m3.1" class="ltx_Math" alttext="X_{0}" display="inline"><semantics id="S2.F8.11.m3.1b"><msub id="S2.F8.11.m3.1.1" xref="S2.F8.11.m3.1.1.cmml"><mi id="S2.F8.11.m3.1.1.2" xref="S2.F8.11.m3.1.1.2.cmml">X</mi><mn id="S2.F8.11.m3.1.1.3" xref="S2.F8.11.m3.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="S2.F8.11.m3.1c"><apply id="S2.F8.11.m3.1.1.cmml" xref="S2.F8.11.m3.1.1"><csymbol cd="ambiguous" id="S2.F8.11.m3.1.1.1.cmml" xref="S2.F8.11.m3.1.1">subscript</csymbol><ci id="S2.F8.11.m3.1.1.2.cmml" xref="S2.F8.11.m3.1.1.2">𝑋</ci><cn type="integer" id="S2.F8.11.m3.1.1.3.cmml" xref="S2.F8.11.m3.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.11.m3.1d">X_{0}</annotation></semantics></math> can be an image, a signal, or a table with heterogeneous features. <span id="S2.F8.23.3" class="ltx_text ltx_font_bold">b</span>: DDPM process consisting of a forward diffusion process and a backward one. <math id="S2.F8.12.m4.1" class="ltx_Math" alttext="Z_{0}" display="inline"><semantics id="S2.F8.12.m4.1b"><msub id="S2.F8.12.m4.1.1" xref="S2.F8.12.m4.1.1.cmml"><mi id="S2.F8.12.m4.1.1.2" xref="S2.F8.12.m4.1.1.2.cmml">Z</mi><mn id="S2.F8.12.m4.1.1.3" xref="S2.F8.12.m4.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="S2.F8.12.m4.1c"><apply id="S2.F8.12.m4.1.1.cmml" xref="S2.F8.12.m4.1.1"><csymbol cd="ambiguous" id="S2.F8.12.m4.1.1.1.cmml" xref="S2.F8.12.m4.1.1">subscript</csymbol><ci id="S2.F8.12.m4.1.1.2.cmml" xref="S2.F8.12.m4.1.1.2">𝑍</ci><cn type="integer" id="S2.F8.12.m4.1.1.3.cmml" xref="S2.F8.12.m4.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.12.m4.1d">Z_{0}</annotation></semantics></math> is destructed by adding noise iteratively. <math id="S2.F8.13.m5.1" class="ltx_Math" alttext="Z_{0}" display="inline"><semantics id="S2.F8.13.m5.1b"><msub id="S2.F8.13.m5.1.1" xref="S2.F8.13.m5.1.1.cmml"><mi id="S2.F8.13.m5.1.1.2" xref="S2.F8.13.m5.1.1.2.cmml">Z</mi><mn id="S2.F8.13.m5.1.1.3" xref="S2.F8.13.m5.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="S2.F8.13.m5.1c"><apply id="S2.F8.13.m5.1.1.cmml" xref="S2.F8.13.m5.1.1"><csymbol cd="ambiguous" id="S2.F8.13.m5.1.1.1.cmml" xref="S2.F8.13.m5.1.1">subscript</csymbol><ci id="S2.F8.13.m5.1.1.2.cmml" xref="S2.F8.13.m5.1.1.2">𝑍</ci><cn type="integer" id="S2.F8.13.m5.1.1.3.cmml" xref="S2.F8.13.m5.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.13.m5.1d">Z_{0}</annotation></semantics></math> is then reconstructed from <math id="S2.F8.14.m6.1" class="ltx_Math" alttext="Z_{T}" display="inline"><semantics id="S2.F8.14.m6.1b"><msub id="S2.F8.14.m6.1.1" xref="S2.F8.14.m6.1.1.cmml"><mi id="S2.F8.14.m6.1.1.2" xref="S2.F8.14.m6.1.1.2.cmml">Z</mi><mi id="S2.F8.14.m6.1.1.3" xref="S2.F8.14.m6.1.1.3.cmml">T</mi></msub><annotation-xml encoding="MathML-Content" id="S2.F8.14.m6.1c"><apply id="S2.F8.14.m6.1.1.cmml" xref="S2.F8.14.m6.1.1"><csymbol cd="ambiguous" id="S2.F8.14.m6.1.1.1.cmml" xref="S2.F8.14.m6.1.1">subscript</csymbol><ci id="S2.F8.14.m6.1.1.2.cmml" xref="S2.F8.14.m6.1.1.2">𝑍</ci><ci id="S2.F8.14.m6.1.1.3.cmml" xref="S2.F8.14.m6.1.1.3">𝑇</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.14.m6.1d">Z_{T}</annotation></semantics></math> iteratively using a neural network. Within the pipeline of Latent Diffusion Model, the DDPM process is performed in the latent space. Originally in the DDPM, the diffusion is performed in the original space. <span id="S2.F8.24.4" class="ltx_text ltx_font_bold">c</span>: Optionally, an embedded conditioning <math id="S2.F8.15.m7.1" class="ltx_Math" alttext="\tau(y)" display="inline"><semantics id="S2.F8.15.m7.1b"><mrow id="S2.F8.15.m7.1.2" xref="S2.F8.15.m7.1.2.cmml"><mi id="S2.F8.15.m7.1.2.2" xref="S2.F8.15.m7.1.2.2.cmml">τ</mi><mo lspace="0em" rspace="0em" id="S2.F8.15.m7.1.2.1" xref="S2.F8.15.m7.1.2.1.cmml">​</mo><mrow id="S2.F8.15.m7.1.2.3.2" xref="S2.F8.15.m7.1.2.cmml"><mo stretchy="false" id="S2.F8.15.m7.1.2.3.2.1" xref="S2.F8.15.m7.1.2.cmml">(</mo><mi id="S2.F8.15.m7.1.1" xref="S2.F8.15.m7.1.1.cmml">y</mi><mo stretchy="false" id="S2.F8.15.m7.1.2.3.2.2" xref="S2.F8.15.m7.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.F8.15.m7.1c"><apply id="S2.F8.15.m7.1.2.cmml" xref="S2.F8.15.m7.1.2"><times id="S2.F8.15.m7.1.2.1.cmml" xref="S2.F8.15.m7.1.2.1"></times><ci id="S2.F8.15.m7.1.2.2.cmml" xref="S2.F8.15.m7.1.2.2">𝜏</ci><ci id="S2.F8.15.m7.1.1.cmml" xref="S2.F8.15.m7.1.1">𝑦</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.15.m7.1d">\tau(y)</annotation></semantics></math> can be added, where y could be other images, radiological reports, semantic maps, class labels… In SD, the embedding function <math id="S2.F8.16.m8.1" class="ltx_Math" alttext="\tau" display="inline"><semantics id="S2.F8.16.m8.1b"><mi id="S2.F8.16.m8.1.1" xref="S2.F8.16.m8.1.1.cmml">τ</mi><annotation-xml encoding="MathML-Content" id="S2.F8.16.m8.1c"><ci id="S2.F8.16.m8.1.1.cmml" xref="S2.F8.16.m8.1.1">𝜏</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.F8.16.m8.1d">\tau</annotation></semantics></math> is based on CLIP</figcaption>
</figure>
</section>
<section id="S2.SS2.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.2.3 </span>Diffusion models</h4>

<div id="S2.SS2.SSS3.p1" class="ltx_para">
<p id="S2.SS2.SSS3.p1.1" class="ltx_p">The original concept of the diffusion model, introduced by Diffusion Probabilistic Models (DPMs) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib13" title="" class="ltx_ref">13</a>]</cite>, draws inspiration from non-equilibrium statistical physics. The key idea is to systematically and iteratively destroy structure in a data distribution through a forward diffusion process. Subsequently, the reverse diffusion process is learned and applied to restore the structure in the data.</p>
</div>
<section id="S2.SS2.SSS3.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_title_paragraph">DDPM</h5>

<div id="S2.SS2.SSS3.Px1.p1" class="ltx_para">
<p id="S2.SS2.SSS3.Px1.p1.1" class="ltx_p">The initial practical implementation of the diffusion model in the context of images was presented by <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>, introducing Denoising Diffusion Probabilistic Models (DDPM). This approach destroys data by iteratively adding Gaussian noise to the image according to the Markov chain. To learn the reverse process, a deep neural network is needed to recover the data. Fig.<a href="#S2.F8" title="Fig. 8 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>b illustrates the working mechanism of the DDPM. The current best practice for image diffusion models is to use U-Net <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite> architectures for denoising. However, these architectures are tailored for image generation tasks, and may not be a viable option for other data types, especially when the training data is limited. Moreover, U-Net architectures struggle to retain temporal dynamic information and lack flexibility in handling varying input sequence lengths, thus proving inadequate for managing sequential information. In addressing this limitation, some studies <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite> resort to Bidirectional Recurrent Neural Networks (BRNNs) implemented with either LSTM or GRU units. A significant drawback of diffusion models is that they operate sequentially on the entire image during both training and inference. Consequently, they demand substantial computational resources and time, making both the training to be slow as well generating an image after training. As a result, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib39" title="" class="ltx_ref">39</a>]</cite> speed up image generation by redefining the diffusion process as a non-Markovian one, which allows for skipping steps in the denoising process, without requiring all past states to be visited before the current state.</p>
</div>
</section>
<section id="S2.SS2.SSS3.Px2" class="ltx_paragraph">
<h5 class="ltx_title ltx_title_paragraph">Latent Diffusion Models- The Stable Diffusion Pipeline</h5>

<div id="S2.SS2.SSS3.Px2.p1" class="ltx_para">
<p id="S2.SS2.SSS3.Px2.p1.1" class="ltx_p">To address computational inefficiencies of DDPMs, Latent Diffusion Model (LDM) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib40" title="" class="ltx_ref">40</a>]</cite> was introduced initially for images, which uses encoders to compress images from their original size in the pixel space into a smaller representation in the latent space, as illustrated in Fig.<a href="#S2.F8" title="Fig. 8 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>a. Therefore, in latent diffusion, the diffusion process is performed on the latent representations rather than the original images (Fig.<a href="#S2.F8" title="Fig. 8 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>b), allowing LDMs to model long-range dependencies within the data and enabling training on limited computational resources while retaining their quality and flexibility. Stable diffusion, a foundation model built based on LDM, introduces text conditioning to the model for additional control over the generation process and consists of three main components: (i) the VAE for compression (Fig.<a href="#S2.F8" title="Fig. 8 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>a), (ii) a U-net based diffusion process in the latent space (Fig.<a href="#S2.F8" title="Fig. 8 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>b), and (iii) a conditioning mechanism that embeds a prompt describing the image using a Contrastive Language-Image Pre-training (CLIP) text encoder <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib41" title="" class="ltx_ref">41</a>]</cite> (Fig.<a href="#S2.F8" title="Fig. 8 ‣ 2.2.2 GANs ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>c). CLIP creates a numeric representation (embedding vector) of the prompt, mapping both the text and images into the same representational space, enabling comparison and similarity quantification. In addition to CLIP, other text encoders such as pre-trained T5X model <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib42" title="" class="ltx_ref">42</a>]</cite>, medBERT encoder <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib43" title="" class="ltx_ref">43</a>]</cite>, a Byte Pair Encoding (BPE) tokenizer <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite>, and Self-alignment pretraining for BERT (SAPBERT) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib45" title="" class="ltx_ref">45</a>]</cite> are commonly used in different publications to enable text conditioning of the models.</p>
</div>
<div id="S2.SS2.SSS3.Px2.p2" class="ltx_para">
<p id="S2.SS2.SSS3.Px2.p2.1" class="ltx_p">The Stable Diffusion model is widely utilized in recent publications, where the pre-trained foundation model is fine-tuned on various medical modalities without the need to initiate training from scratch. Moreover, the idea of latent diffusion models, which involves performing the diffusion process in the latent space, has been extended to both the EHR and signals data types. In these extensions, a low-dimensional representation of the high-dimensional features, including discrete ones, is learned using an auto encoder before the diffusion process.</p>
</div>
</section>
</section>
<section id="S2.SS2.SSS4" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">2.2.4 </span>Language models:</h4>

<div id="S2.SS2.SSS4.p1" class="ltx_para">
<p id="S2.SS2.SSS4.p1.1" class="ltx_p">A fundamental aspect of language models is enhancing the linguistic capabilities of machines to understand the probability of sequences of words, enabling them to predict future or missing words in sentences <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib46" title="" class="ltx_ref">46</a>]</cite>. The development of language models (shown in Fig.<a href="#S2.F9" title="Fig. 9 ‣ 2.2.4 Language models: ‣ 2.2 Generation techniques ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">9</span></a>) started from Statistical language models which were grounded in probability theories with a restriction to predicting the next word in a sequence based on a fixed number of previous words. However, these models face significant challenges in accurately predicting the next word due to the exponential growth in possible word sequences as the sentence length increases.</p>
</div>
<div id="S2.SS2.SSS4.p2" class="ltx_para">
<p id="S2.SS2.SSS4.p2.1" class="ltx_p">Furthermore, more comprehensive processing methods - Neural Language Models - have been developed utilizing neural networks, such as RNNs and LSTM models. These advanced models excel in recognizing longer-distance relationships (compared to statistical language models) and contextual subtleties in texts, employing word embeddings to grasp semantic similarities among words, thus facilitating the generation of novel word combinations. RNNs and LSTM models have the advantages of creating semantically and syntactically correct content and capturing correlations across sentences. Rather than taking words and text as input, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib47" title="" class="ltx_ref">47</a>]</cite> generate synthetic chief complaints from EHR data. The generated complaints maintain certain epidemiological details and the relationships between the chief complaints and the discharge diagnosis code. However, both models have limitations in the text length that can be understood and processed and suffer from generating more complex and hierarchical text.</p>
</div>
<div id="S2.SS2.SSS4.p3" class="ltx_para">
<p id="S2.SS2.SSS4.p3.1" class="ltx_p">Subsequently, pre-trained language models such as BERT (Bidirectional Encoder Representations from Transformers) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib48" title="" class="ltx_ref">48</a>]</cite> and GPT-2 (Generative Pretrained Transformer 2) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib49" title="" class="ltx_ref">49</a>]</cite>, were proposed and garnered significant attention from language processing research due to its outstanding performance and efficiency. BERT is well-known for its architecture design which consists of pre-training deep bidirectional representations from unlabeled text and analyzing text data from both left to right and right to left. BERT is pre-trained on the BooksCorpus and English Wikipedia (3300M words in total). To generate medical text with specific domain knowledge, extended BERT models such as Bio-BERT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib50" title="" class="ltx_ref">50</a>]</cite> and Clinical-BERT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib51" title="" class="ltx_ref">51</a>]</cite> were developed by training on additional medical corpus such as text from PubMed, clinical reports, doctors’ notes. Similar to BERT, GPT-2 applied transformer-based architecture which uses a self-attention mechanism to weigh the importance of different words in single or multiple sentences. GPT-2 is trained on WebText data (40GB text) and focuses on unidirectional (left-to-right) text processing and generation.</p>
</div>
<div id="S2.SS2.SSS4.p4" class="ltx_para">
<p id="S2.SS2.SSS4.p4.1" class="ltx_p">In recent years, large language models such as GPT-4, Llama (Large Language Model Meta AI) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib52" title="" class="ltx_ref">52</a>]</cite>, and FLAN-T5 (Fine-tuned LAnguage Net with a Text-To-Text Transfer Transformer) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib53" title="" class="ltx_ref">53</a>]</cite>, have significantly enhanced the capabilities of prior language models, showcasing remarkable proficiency across a spectrum of downstream applications. Owing to its high efficiency and scalability, the transformer model has become a foundation for most large language models including GPT-4 and Llama. GPT-4 is the latest updated version of GPT models which can understand larger and more complex text and generate more coherent and contextually relevant text compared to previous GPTs, while Llama (1 and 2) enhanced training efficiency and model scalability and accessibility for broader users. Different from GPT-4 and Llama, FLAN-T5 is based on the Text-to-Text Transfer Transformer model which transforms all the natural language processing tasks to text-to-text problems and trains on various tasks presented in natural language instructions. FLAN-T5 has shown outstanding performance in understanding and executing a set of tasks and has the capability to adapt to varied instructions and tasks.</p>
</div>
<div id="S2.SS2.SSS4.p5" class="ltx_para">
<p id="S2.SS2.SSS4.p5.1" class="ltx_p">General language models are designed to deal with a variety of tasks related to natural language understanding and generation such as text generation, translation, question answering, summarization, named entity recognition, text classification, and many others. To tailor them for clinical and medical texts, the language models are trained on medical literature (such as PubMed), patients records, case reports, clinical notes to handle the specialized and complex medical data. The main tasks for medical language models are augmenting clinical text data for downstream tasks (such as disease classification), generating summarization from lengthy clinical documentation, identifying personal health information, and generating anonymized data to replace sensitive personal information, and question answers to support clinical decision making process.</p>
</div>
<figure id="S2.F9" class="ltx_figure"><img src="/html/2407.00116/assets/lm_models.jpg" id="S2.F9.g1" class="ltx_graphics ltx_centering ltx_img_square" width="240" height="234" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 9: </span>Language models clusters</figcaption>
</figure>
</section>
</section>
<section id="S2.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.3 </span>Evaluation</h3>

<div id="S2.SS3.p1" class="ltx_para">
<p id="S2.SS3.p1.1" class="ltx_p">The evaluation of synthetic data is a multifaceted task, encompassing several dimensions that collectively determine the quality and usability of the synthetic data. We categorize the evaluation as utility, fidelity, diversity, qualitative assessment, clinical validation, and privacy. Each dimension presents unique challenges and considerations, and their comprehensive evaluation is crucial to ensure the effectiveness of synthetic data. In this survey, we analyze each of these evaluation approaches, aiming to provide a holistic view of the current synthetic data evaluation practices. The remaining of this section gives an overview of all evaluation categories considered in this paper, offering insights into their significance and the common metrics employed.</p>
</div>
<div id="S2.SS3.p2" class="ltx_para">
<p id="S2.SS3.p2.1" class="ltx_p"><span id="S2.SS3.p2.1.1" class="ltx_text ltx_font_bold">Evaluation of utility</span> asses the use of the synthetic data for specific tasks, such as providing additional data for improving a downstream medical AI model. Several settings exist in which the synthetic data can be used for a downstream task. The synthetic data can be used to fully train the downstream model(replacing the need to access real data), or to augment real training data. Moreover, the synthetic data can be used for validating or testing the models. These settings can be employed in different downstream applications such as supervised classification, prediction, supervised regressions, image segmentation, even reinforcement learning, with the specific metrics of each application.</p>
</div>
<div id="S2.SS3.p3" class="ltx_para">
<p id="S2.SS3.p3.1" class="ltx_p"><span id="S2.SS3.p3.1.1" class="ltx_text ltx_font_bold">Evaluation of privacy</span> is a critical factor that ensures the synthetic dataset cannot be exploited to reveal sensitive information about individuals in the original dataset. When it comes to EHR data, privacy evaluation typically includes testing the synthetic data for the following risks:</p>
</div>
<div id="S2.SS3.p4" class="ltx_para">
<ul id="S2.I2" class="ltx_itemize">
<li id="S2.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S2.I2.i1.p1" class="ltx_para">
<p id="S2.I2.i1.p1.1" class="ltx_p"><span id="S2.I2.i1.p1.1.1" class="ltx_text ltx_font_bold">Membership inference risk:</span> Risk against membership inference attack, where an adversary attempts to determine whether a specific data record was used in the training of a machine learning model.</p>
</div>
</li>
<li id="S2.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S2.I2.i2.p1" class="ltx_para">
<p id="S2.I2.i2.p1.1" class="ltx_p"><span id="S2.I2.i2.p1.1.1" class="ltx_text ltx_font_bold">Attribute inference risk:</span> Risk against attribute inference attack, where an adversary wants to infer private attributes of individuals based on the available data or model’s predictions.</p>
</div>
</li>
<li id="S2.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span> 
<div id="S2.I2.i3.p1" class="ltx_para">
<p id="S2.I2.i3.p1.1" class="ltx_p"><span id="S2.I2.i3.p1.1.1" class="ltx_text ltx_font_bold">Nearest neighbor adversarial accuracy risk:</span> Risk that is based on the Nearest Neighbor Adversarial Accuracy (NNAA) metric <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref">54</a>]</cite>. This risk assesses the privacy risk of synthetic data by analyzing how closely the nearest neighbor (the most similar record) in the synthetic dataset resembles any real individual. A high similarity raises the risk of re-identification.</p>
</div>
</li>
</ul>
<p id="S2.SS3.p4.1" class="ltx_p">Table <a href="#S10.T15" title="Table 15 ‣ 10.1 Evolution of the usage of generation techniques over years ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">15</span></a> in the appendix displays the usage of these metrics in the surveyed papers.</p>
</div>
<div id="S2.SS3.p5" class="ltx_para">
<p id="S2.SS3.p5.1" class="ltx_p">Researchers usually apply Differential Privacy (DP) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib55" title="" class="ltx_ref">55</a>]</cite> mechanisms to mitigate privacy risks. DP is a rigorous and mathematically grounded framework designed to protect the privacy of individuals in the dataset. By introducing randomness into data queries or the data generation process, differential privacy ensures that the synthetic data doesn’t compromise the privacy of any individual in the original dataset. It’s often considered the gold standard for privacy protection in data analysis.</p>
</div>
<div id="S2.SS3.p6" class="ltx_para">
<p id="S2.SS3.p6.1" class="ltx_p"><span id="S2.SS3.p6.1.1" class="ltx_text ltx_font_bold">Evaluation of fidelity</span> assesses how well the synthetic data reflects the real-world characteristics and features of the original dataset, both at the individual sample level and across the entire data distribution. High fidelity indicates that the synthetic data closely resembles the original data in terms of appearance and statistical properties. Notably, fidelity metrics differ across different data types due to their inherent structural and inherent differences. Table <a href="#S2.T1" title="Table 1 ‣ 2.3 Evaluation ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> summarizes fidelity metrics for different types of medical data. Tables <a href="#S10.T16" title="Table 16 ‣ Fidelity Metrics for EHR ‣ 10.2.2 Fidelity Metrics ‣ 10.2 Synthetic Data Evaluation ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">16</span></a> and <a href="#S10.T17" title="Table 17 ‣ Fidelity Metrics for EHR ‣ 10.2.2 Fidelity Metrics ‣ 10.2 Synthetic Data Evaluation ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">17</span></a> in the appendix display the different fidelity metrics as long as the papers using them. In most of the cases, the fidelity metrics used are originally designed for different types of data like natural images, not taking into account the characteristics and statistics of typical medical image modalities. For instance, FID (Fréchet Inception Distance) and IS (Inception Score) are widely used to evaluate the similarity between synthetic and real images, but they may not accurately reflect the subtle variations and noise patterns inherent in medical data, as these metrics are underpinned by a feature extractor pre-trained on the ImageNet dataset, which is more relevant for natural images.</p>
</div>
<div id="S2.SS3.p7" class="ltx_para">
<p id="S2.SS3.p7.1" class="ltx_p"><span id="S2.SS3.p7.1.1" class="ltx_text ltx_font_bold">Evaluation of diversity</span> measures the extent to which synthetic data captures the full variability of the real data.
Following <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib56" title="" class="ltx_ref">56</a>]</cite>, we consider diversity and fidelity as distinct yet complementary metrics in evaluating synthetic data. This ensures that the generative models are not only evaluated for reproducing average or common scenarios but also generating the broad spectrum of real-world variations.
Table <a href="#S2.T2" title="Table 2 ‣ 2.3 Evaluation ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> summarizes diversity metrics for different types of medical data.</p>
</div>
<div id="S2.SS3.p8" class="ltx_para">
<p id="S2.SS3.p8.1" class="ltx_p"><span id="S2.SS3.p8.1.1" class="ltx_text ltx_font_bold">Clinical evaluation</span> involves assessing the synthetic data’s correct representation of anatomical details consistent with real-world clinical scenarios, its ability to replicate key clinical features and characteristics, and its adherence to established medical guidelines and practices. These anatomical and biological details may be overlooked by fidelity metrics, and thus a clinically relevant evaluation is pivotal in introducing synthetic images into the medical field. Clinical validation also aim to verify that synthetic data can effectively support medical decision-making, patient care, and clinical outcomes, ultimately enhancing its utility and trustworthiness in healthcare applications.</p>
</div>
<div id="S2.SS3.p9" class="ltx_para">
<p id="S2.SS3.p9.1" class="ltx_p"><span id="S2.SS3.p9.1.1" class="ltx_text ltx_font_bold">Qualitative evaluation</span> including human assessment, also known as a visual Turing Test, which involves medical experts reviewing the synthetic data to try to distinguish it from the real data. In some cases, qualitative evaluation only involves data visualization using techniques such as t-distributed Stochastic Neighbor Embedding (t-sne) and Principal Component Analysis (PCA), to compare the distribution of real and synthetic data visually.</p>
</div>
<figure id="S2.T1" class="ltx_table">
<table id="S2.T1.1" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S2.T1.1.1.1" class="ltx_tr">
<th id="S2.T1.1.1.1.1" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_th_row ltx_border_t"><span id="S2.T1.1.1.1.1.1" class="ltx_text ltx_font_bold" style="font-size:80%;">Modality</span></th>
<th id="S2.T1.1.1.1.2" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S2.T1.1.1.1.2.1" class="ltx_text ltx_font_bold" style="font-size:80%;">Purpose of evaluation</span></th>
<th id="S2.T1.1.1.1.3" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S2.T1.1.1.1.3.1" class="ltx_text ltx_font_bold" style="font-size:80%;">Fidelity Metric</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S2.T1.1.2.1" class="ltx_tr">
<th id="S2.T1.1.2.1.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_t" rowspan="13"><span id="S2.T1.1.2.1.1.1" class="ltx_text" style="font-size:80%;">EHR</span></th>
<td id="S2.T1.1.2.1.2" class="ltx_td ltx_align_left ltx_border_t" rowspan="5"><span id="S2.T1.1.2.1.2.1" class="ltx_text" style="font-size:80%;">Dimension wise distributional similarity</span></td>
<td id="S2.T1.1.2.1.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.2.1.3.1" class="ltx_text" style="font-size:80%;">Bernoulli Success probability</span></td>
</tr>
<tr id="S2.T1.1.3.2" class="ltx_tr">
<td id="S2.T1.1.3.2.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.3.2.1.1" class="ltx_text" style="font-size:80%;">Chi-square test</span></td>
</tr>
<tr id="S2.T1.1.4.3" class="ltx_tr">
<td id="S2.T1.1.4.3.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.4.3.1.1" class="ltx_text" style="font-size:80%;">Kullback-Leibler divergence (KLD)</span></td>
</tr>
<tr id="S2.T1.1.5.4" class="ltx_tr">
<td id="S2.T1.1.5.4.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.5.4.1.1" class="ltx_text" style="font-size:80%;">Kolmogorov–Smirnov Test</span></td>
</tr>
<tr id="S2.T1.1.6.5" class="ltx_tr">
<td id="S2.T1.1.6.5.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.6.5.1.1" class="ltx_text" style="font-size:80%;">Pearson test</span></td>
</tr>
<tr id="S2.T1.1.7.6" class="ltx_tr">
<td id="S2.T1.1.7.6.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="4"><span id="S2.T1.1.7.6.1.1" class="ltx_text" style="font-size:80%;">Joint Distribution Similarity</span></td>
<td id="S2.T1.1.7.6.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.7.6.2.1" class="ltx_text" style="font-size:80%;">Jensen-Shannon Divergence</span></td>
</tr>
<tr id="S2.T1.1.8.7" class="ltx_tr">
<td id="S2.T1.1.8.7.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.8.7.1.1" class="ltx_text" style="font-size:80%;">Maximum-Mean Discrepancy (MMD)</span></td>
</tr>
<tr id="S2.T1.1.9.8" class="ltx_tr">
<td id="S2.T1.1.9.8.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.9.8.1.1" class="ltx_text" style="font-size:80%;">Wasserstein distance</span></td>
</tr>
<tr id="S2.T1.1.10.9" class="ltx_tr">
<td id="S2.T1.1.10.9.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.10.9.1.1" class="ltx_text" style="font-size:80%;">Propensity score</span></td>
</tr>
<tr id="S2.T1.1.11.10" class="ltx_tr">
<td id="S2.T1.1.11.10.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S2.T1.1.11.10.1.1" class="ltx_text" style="font-size:80%;">Inter-dimensional Relationship Similarity</span></td>
<td id="S2.T1.1.11.10.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.11.10.2.1" class="ltx_text" style="font-size:80%;">Pearson pairwise correlation</span></td>
</tr>
<tr id="S2.T1.1.12.11" class="ltx_tr">
<td id="S2.T1.1.12.11.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.12.11.1.1" class="ltx_text" style="font-size:80%;">Pairwise correlation difference</span></td>
</tr>
<tr id="S2.T1.1.13.12" class="ltx_tr">
<td id="S2.T1.1.13.12.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.13.12.1.1" class="ltx_text" style="font-size:80%;">Dimension-wise prediction</span></td>
</tr>
<tr id="S2.T1.1.14.13" class="ltx_tr">
<td id="S2.T1.1.14.13.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S2.T1.1.14.13.1.1" class="ltx_text" style="font-size:80%;">Latent Distribution similarity</span></td>
<td id="S2.T1.1.14.13.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.14.13.2.1" class="ltx_text" style="font-size:80%;">Log-cluster metric</span></td>
</tr>
<tr id="S2.T1.1.15.14" class="ltx_tr">
<th id="S2.T1.1.15.14.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.15.14.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.15.14.2.1" class="ltx_text" style="font-size:80%;">Latent space representation</span></td>
</tr>
<tr id="S2.T1.1.16.15" class="ltx_tr">
<th id="S2.T1.1.16.15.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.16.15.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.16.15.2.1" class="ltx_text" style="font-size:80%;">specific-language model</span></td>
<td id="S2.T1.1.16.15.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.16.15.3.1" class="ltx_text" style="font-size:80%;">Perplexity</span></td>
</tr>
<tr id="S2.T1.1.17.16" class="ltx_tr">
<th id="S2.T1.1.17.16.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.17.16.2" class="ltx_td ltx_align_left" rowspan="2"><span id="S2.T1.1.17.16.2.1" class="ltx_text" style="font-size:80%;">specific-time-series</span></td>
<td id="S2.T1.1.17.16.3" class="ltx_td ltx_align_left"><span id="S2.T1.1.17.16.3.1" class="ltx_text" style="font-size:80%;">Autocorrelation function</span></td>
</tr>
<tr id="S2.T1.1.18.17" class="ltx_tr">
<th id="S2.T1.1.18.17.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.18.17.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.18.17.2.1" class="ltx_text" style="font-size:80%;">Patient trajectories</span></td>
</tr>
<tr id="S2.T1.1.19.18" class="ltx_tr">
<th id="S2.T1.1.19.18.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_t" rowspan="7"><span id="S2.T1.1.19.18.1.1" class="ltx_text" style="font-size:80%;">Imaging Data</span></th>
<td id="S2.T1.1.19.18.2" class="ltx_td ltx_align_left ltx_border_t" rowspan="7"><span id="S2.T1.1.19.18.2.1" class="ltx_text" style="font-size:80%;">Pixel-wise similarity (image based)</span></td>
<td id="S2.T1.1.19.18.3" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S2.T1.1.20.19" class="ltx_tr">
<td id="S2.T1.1.20.19.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.20.19.1.1" class="ltx_text" style="font-size:80%;">(Root/Normalized) Mean squared error (MSE, RMSE, NRMSE)</span></td>
</tr>
<tr id="S2.T1.1.21.20" class="ltx_tr">
<td id="S2.T1.1.21.20.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.21.20.1.1" class="ltx_text" style="font-size:80%;">Structural similarity index (SSIM)</span></td>
</tr>
<tr id="S2.T1.1.22.21" class="ltx_tr">
<td id="S2.T1.1.22.21.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.22.21.1.1" class="ltx_text" style="font-size:80%;">Multi Scale SSIM (MS SSIM)</span></td>
</tr>
<tr id="S2.T1.1.23.22" class="ltx_tr">
<td id="S2.T1.1.23.22.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.23.22.1.1" class="ltx_text" style="font-size:80%;">Peak signal-to-noise ratio (PSNR)</span></td>
</tr>
<tr id="S2.T1.1.24.23" class="ltx_tr">
<td id="S2.T1.1.24.23.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.24.23.1.1" class="ltx_text" style="font-size:80%;">Universal Quality Index (UQI)</span></td>
</tr>
<tr id="S2.T1.1.25.24" class="ltx_tr">
<td id="S2.T1.1.25.24.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.25.24.1.1" class="ltx_text" style="font-size:80%;">Contrast Noise Ratio (CNR)</span></td>
</tr>
<tr id="S2.T1.1.26.25" class="ltx_tr">
<th id="S2.T1.1.26.25.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.26.25.2" class="ltx_td ltx_align_left ltx_border_t" rowspan="7"><span id="S2.T1.1.26.25.2.1" class="ltx_text" style="font-size:80%;">Feature-wise similarity (distrubution based)</span></td>
<td id="S2.T1.1.26.25.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.26.25.3.1" class="ltx_text" style="font-size:80%;">Fréchet inception distance (FID)</span></td>
</tr>
<tr id="S2.T1.1.27.26" class="ltx_tr">
<th id="S2.T1.1.27.26.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.27.26.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.27.26.2.1" class="ltx_text" style="font-size:80%;">Kernel Inception Distance (KID)</span></td>
</tr>
<tr id="S2.T1.1.28.27" class="ltx_tr">
<th id="S2.T1.1.28.27.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.28.27.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.28.27.2.1" class="ltx_text" style="font-size:80%;">Inception score (IS)</span></td>
</tr>
<tr id="S2.T1.1.29.28" class="ltx_tr">
<th id="S2.T1.1.29.28.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.29.28.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.29.28.2.1" class="ltx_text" style="font-size:80%;">Maximum Mean Discrepancy (MMD)</span></td>
</tr>
<tr id="S2.T1.1.30.29" class="ltx_tr">
<th id="S2.T1.1.30.29.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.30.29.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.30.29.2.1" class="ltx_text" style="font-size:80%;">Learned Perceptual Image Patch Similarity (LPIPS)</span></td>
</tr>
<tr id="S2.T1.1.31.30" class="ltx_tr">
<th id="S2.T1.1.31.30.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.31.30.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.31.30.2.1" class="ltx_text" style="font-size:80%;">Feature Similarity Index (FSIM)</span></td>
</tr>
<tr id="S2.T1.1.32.31" class="ltx_tr">
<th id="S2.T1.1.32.31.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.32.31.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.32.31.2.1" class="ltx_text" style="font-size:80%;">feature distribution similarity (FDS)</span></td>
</tr>
<tr id="S2.T1.1.33.32" class="ltx_tr">
<th id="S2.T1.1.33.32.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.33.32.2" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S2.T1.1.33.32.2.1" class="ltx_text" style="font-size:80%;">Image/Text Alignment</span></td>
<td id="S2.T1.1.33.32.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.33.32.3.1" class="ltx_text" style="font-size:80%;">Bilingual Evaluation Understudy (BLEU)</span></td>
</tr>
<tr id="S2.T1.1.34.33" class="ltx_tr">
<th id="S2.T1.1.34.33.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.34.33.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.34.33.2.1" class="ltx_text" style="font-size:80%;">Recall-Oriented Understudy for Gisting Evaluation (ROUGE)</span></td>
</tr>
<tr id="S2.T1.1.35.34" class="ltx_tr">
<th id="S2.T1.1.35.34.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.35.34.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.35.34.2.1" class="ltx_text" style="font-size:80%;">Contrastive Language–Image Pre-training score (CLIP)</span></td>
</tr>
<tr id="S2.T1.1.36.35" class="ltx_tr">
<th id="S2.T1.1.36.35.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_t" rowspan="3"><span id="S2.T1.1.36.35.1.1" class="ltx_text" style="font-size:80%;">Time-series Data</span></th>
<td id="S2.T1.1.36.35.2" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S2.T1.1.36.35.2.1" class="ltx_text" style="font-size:80%;">Temporal sequences similarity</span></td>
<td id="S2.T1.1.36.35.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.36.35.3.1" class="ltx_text" style="font-size:80%;">Dynamic Time Warping (DTW)</span></td>
</tr>
<tr id="S2.T1.1.37.36" class="ltx_tr">
<td id="S2.T1.1.37.36.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.37.36.1.1" class="ltx_text" style="font-size:80%;">Time Warp Edit Distance (TWED)</span></td>
</tr>
<tr id="S2.T1.1.38.37" class="ltx_tr">
<td id="S2.T1.1.38.37.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S2.T1.1.38.37.1.1" class="ltx_text" style="font-size:80%;">Temporal Correlation</span></td>
<td id="S2.T1.1.38.37.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.38.37.2.1" class="ltx_text" style="font-size:80%;">Autocorrelation</span></td>
</tr>
<tr id="S2.T1.1.39.38" class="ltx_tr">
<th id="S2.T1.1.39.38.1" class="ltx_td ltx_th ltx_th_row"></th>
<td id="S2.T1.1.39.38.2" class="ltx_td ltx_align_left"><span id="S2.T1.1.39.38.2.1" class="ltx_text" style="font-size:80%;">Cross-correlation (CC)</span></td>
</tr>
<tr id="S2.T1.1.40.39" class="ltx_tr">
<th id="S2.T1.1.40.39.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_t" rowspan="4"><span id="S2.T1.1.40.39.1.1" class="ltx_text" style="font-size:80%;">Medical Text</span></th>
<td id="S2.T1.1.40.39.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.40.39.2.1" class="ltx_text" style="font-size:80%;">Similarity between machine and human translation</span></td>
<td id="S2.T1.1.40.39.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.40.39.3.1" class="ltx_text" style="font-size:80%;">Bilingual Evaluation Understudy (BLEU)</span></td>
</tr>
<tr id="S2.T1.1.41.40" class="ltx_tr">
<td id="S2.T1.1.41.40.1" class="ltx_td ltx_align_left ltx_border_b ltx_border_t" rowspan="4"><span id="S2.T1.1.41.40.1.1" class="ltx_text" style="font-size:80%;">Similarity between real and generated text</span></td>
<td id="S2.T1.1.41.40.2" class="ltx_td ltx_align_left ltx_border_t"><span id="S2.T1.1.41.40.2.1" class="ltx_text" style="font-size:80%;">Cosine similarities</span></td>
</tr>
<tr id="S2.T1.1.42.41" class="ltx_tr">
<td id="S2.T1.1.42.41.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.42.41.1.1" class="ltx_text" style="font-size:80%;">Jaccard Similarity</span></td>
</tr>
<tr id="S2.T1.1.43.42" class="ltx_tr">
<td id="S2.T1.1.43.42.1" class="ltx_td ltx_align_left"><span id="S2.T1.1.43.42.1.1" class="ltx_text" style="font-size:80%;">ROUGE-N recall</span></td>
</tr>
<tr id="S2.T1.1.44.43" class="ltx_tr">
<th id="S2.T1.1.44.43.1" class="ltx_td ltx_th ltx_th_row ltx_border_b"></th>
<td id="S2.T1.1.44.43.2" class="ltx_td ltx_align_left ltx_border_b"><span id="S2.T1.1.44.43.2.1" class="ltx_text" style="font-size:80%;">G2 - Test</span></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:80%;"><span class="ltx_tag ltx_tag_table">Table 1: </span>Fidelity Metrics</figcaption>
</figure>
<figure id="S2.T2" class="ltx_table">
<table id="S2.T2.1" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S2.T2.1.1.1" class="ltx_tr">
<th id="S2.T2.1.1.1.1" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt"><span id="S2.T2.1.1.1.1.1" class="ltx_text ltx_font_bold" style="font-size:80%;">Modality</span></th>
<th id="S2.T2.1.1.1.2" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt"><span id="S2.T2.1.1.1.2.1" class="ltx_text ltx_font_bold" style="font-size:80%;">Diversity Metric</span></th>
<th id="S2.T2.1.1.1.3" class="ltx_td ltx_nopad_r ltx_align_center ltx_th ltx_th_column ltx_border_tt"><span id="S2.T2.1.1.1.3.1" class="ltx_text ltx_font_bold" style="font-size:80%;">Usage</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S2.T2.1.2.1" class="ltx_tr">
<td id="S2.T2.1.2.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="5"><span id="S2.T2.1.2.1.1.1" class="ltx_text" style="font-size:80%;">EHR</span></td>
<td id="S2.T2.1.2.1.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S2.T2.1.2.1.2.1" class="ltx_text" style="font-size:80%;">Category coverage</span></td>
<td id="S2.T2.1.2.1.3" class="ltx_td ltx_nopad_r ltx_align_center ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.2.1.3.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib57" title="" class="ltx_ref">57</a><span id="S2.T2.1.2.1.3.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.3.2" class="ltx_tr">
<td id="S2.T2.1.3.2.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.3.2.1.1" class="ltx_text" style="font-size:80%;">Generated Disease types</span></td>
<td id="S2.T2.1.3.2.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.3.2.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib58" title="" class="ltx_ref">58</a><span id="S2.T2.1.3.2.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.4.3" class="ltx_tr">
<td id="S2.T2.1.4.3.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.4.3.1.1" class="ltx_text" style="font-size:80%;">Required sample number to generate all diseases (RN)</span></td>
<td id="S2.T2.1.4.3.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.4.3.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib58" title="" class="ltx_ref">58</a><span id="S2.T2.1.4.3.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.5.4" class="ltx_tr">
<td id="S2.T2.1.5.4.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.5.4.1.1" class="ltx_text" style="font-size:80%;">Normalized Distance</span></td>
<td id="S2.T2.1.5.4.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.5.4.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib58" title="" class="ltx_ref">58</a><span id="S2.T2.1.5.4.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.6.5" class="ltx_tr">
<td id="S2.T2.1.6.5.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.6.5.1.1" class="ltx_text" style="font-size:80%;">Medical concept abundance</span></td>
<td id="S2.T2.1.6.5.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.6.5.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib8" title="" class="ltx_ref">8</a><span id="S2.T2.1.6.5.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.7.6" class="ltx_tr">
<td id="S2.T2.1.7.6.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S2.T2.1.7.6.1.1" class="ltx_text" style="font-size:80%;">Signals</span></td>
<td id="S2.T2.1.7.6.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S2.T2.1.7.6.2.1" class="ltx_text" style="font-size:80%;">Diversity of Samples Score</span></td>
<td id="S2.T2.1.7.6.3" class="ltx_td ltx_nopad_r ltx_align_center ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.7.6.3.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib59" title="" class="ltx_ref">59</a><span id="S2.T2.1.7.6.3.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.8.7" class="ltx_tr">
<td id="S2.T2.1.8.7.1" class="ltx_td ltx_align_center ltx_border_bb ltx_border_t" rowspan="7"><span id="S2.T2.1.8.7.1.1" class="ltx_text" style="font-size:80%;">Imaging</span></td>
<td id="S2.T2.1.8.7.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S2.T2.1.8.7.2.1" class="ltx_text" style="font-size:80%;">MS-SSIM</span></td>
<td id="S2.T2.1.8.7.3" class="ltx_td ltx_nopad_r ltx_align_center ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.8.7.3.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib60" title="" class="ltx_ref">60</a>, <a href="#bib.bib61" title="" class="ltx_ref">61</a>, <a href="#bib.bib62" title="" class="ltx_ref">62</a><span id="S2.T2.1.8.7.3.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.9.8" class="ltx_tr">
<td id="S2.T2.1.9.8.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.9.8.1.1" class="ltx_text" style="font-size:80%;">Nearest SSIM difference</span></td>
<td id="S2.T2.1.9.8.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.9.8.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib63" title="" class="ltx_ref">63</a><span id="S2.T2.1.9.8.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.10.9" class="ltx_tr">
<td id="S2.T2.1.10.9.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.10.9.1.1" class="ltx_text" style="font-size:80%;">Visualization(Diversity)</span></td>
<td id="S2.T2.1.10.9.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.10.9.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib64" title="" class="ltx_ref">64</a><span id="S2.T2.1.10.9.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.11.10" class="ltx_tr">
<td id="S2.T2.1.11.10.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.11.10.1.1" class="ltx_text" style="font-size:80%;">Diversity score (DS)</span></td>
<td id="S2.T2.1.11.10.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.11.10.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib65" title="" class="ltx_ref">65</a><span id="S2.T2.1.11.10.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.12.11" class="ltx_tr">
<td id="S2.T2.1.12.11.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.12.11.1.1" class="ltx_text" style="font-size:80%;">LPIPS</span></td>
<td id="S2.T2.1.12.11.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.12.11.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib66" title="" class="ltx_ref">66</a>, <a href="#bib.bib67" title="" class="ltx_ref">67</a>, <a href="#bib.bib68" title="" class="ltx_ref">68</a><span id="S2.T2.1.12.11.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.13.12" class="ltx_tr">
<td id="S2.T2.1.13.12.1" class="ltx_td ltx_align_center"><span id="S2.T2.1.13.12.1.1" class="ltx_text" style="font-size:80%;">Hammind Distance</span></td>
<td id="S2.T2.1.13.12.2" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.13.12.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib68" title="" class="ltx_ref">68</a><span id="S2.T2.1.13.12.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
<tr id="S2.T2.1.14.13" class="ltx_tr">
<td id="S2.T2.1.14.13.1" class="ltx_td ltx_align_center ltx_border_bb"><span id="S2.T2.1.14.13.1.1" class="ltx_text" style="font-size:80%;">Euclindean Distance</span></td>
<td id="S2.T2.1.14.13.2" class="ltx_td ltx_nopad_r ltx_align_center ltx_border_bb"><cite class="ltx_cite ltx_citemacro_cite"><span id="S2.T2.1.14.13.2.1.1" class="ltx_text" style="font-size:80%;">[</span><a href="#bib.bib69" title="" class="ltx_ref">69</a><span id="S2.T2.1.14.13.2.2.2" class="ltx_text" style="font-size:80%;">]</span></cite></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:80%;"><span class="ltx_tag ltx_tag_table">Table 2: </span>Diversity Metrics</figcaption>
</figure>
</section>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Results</h2>

<div id="S3.p1" class="ltx_para">
<p id="S3.p1.1" class="ltx_p">In our survey, we’ve meticulously distinguished between various modalities within the realm of medical data: EHR, Physiological Signals, Medical Images, and Clinical Notes, even though some might consider all these modalities as part of a patient’s EHR. Each of these categories represents a distinct dimension of patient health data, each contributing unique insights and challenges.</p>
</div>
<div id="S3.p2" class="ltx_para">
<p id="S3.p2.1" class="ltx_p">EHR encompass a variety of formats, including longitudinal EHR, time-dependent EHR, aggregated EHR, and snapshot EHR. These different formats capture the patient’s medical history over time, ranging from specific episodes of care to comprehensive summaries.</p>
</div>
<div id="S3.p3" class="ltx_para">
<p id="S3.p3.1" class="ltx_p">Physiological Signals, on the other hand, capture real-time physiological data such as ECG, EEG, and various other signals. These signals provide crucial information about a patient’s physiological state and are often used in monitoring and diagnosing medical conditions.</p>
</div>
<div id="S3.p4" class="ltx_para">
<p id="S3.p4.1" class="ltx_p">Medical Images are vital, and include modalities such as Ultrasound, Mammography, Dermoscopy, MRI, CT, X-Ray, OCT, as well as combinations of multiple modalities. Each imaging modality offers unique insights into different aspects of the patient’s anatomy and pathology.</p>
</div>
<div id="S3.p5" class="ltx_para">
<p id="S3.p5.1" class="ltx_p">Clinical Notes provide detailed textual descriptions of patient observations, diagnoses, treatments, and other relevant information. These notes offer valuable context and insight into the patient’s condition, treatment history, and healthcare journey.</p>
</div>
<div id="S3.p6" class="ltx_para">
<p id="S3.p6.1" class="ltx_p">This section transitions from general foundational knowledge to a more focused examination of findings obtained from the surveyed papers, organized according to different modalities. For each modality, we present a table summarizing various aspects of the reviewed papers, including the synthesis applications, models utilized for generation, and the adopted evaluation framework. The table content encompasses details such as the type of synthesis applications, specifics of the models employed, evaluation metrics, and code availability. We aimed to maintain a consistent table structure across modalities to facilitate a better understanding of methodologies and outcomes in medical data synthesis across diverse domains, although not always feasible. This structured approach enhances comprehension and comparison across different modalities in the field.</p>
</div>
<section id="S3.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.1 </span>EHR</h3>

<div id="S3.SS1.p1" class="ltx_para">
<p id="S3.SS1.p1.1" class="ltx_p">EHRs are inherently heterogeneous, encompassing various types of data, including demographic data, medical codes assigned to diagnoses and procedures such as the International Classification of Diseases (ICD) codes, and time-series vital signs, among others. EHR typically consists of both static (e.g. demographics) and sequential numerical features (e.g. blood pressure), as well as multi-categorical features (e.g. medical codes). Given this complexity and heterogeneity, researchers and practitioners often differentiate four formats of EHR when developing generative models and generating synthetic data. These formats include longitudinal, time-dependent, aggregated, and snapshot, each with its own challenges and complexities, from modeling temporal dependencies and inter-dependencies between features to generating high-dimensional data and mixed-type features. Fig.<a href="#S3.F10" title="Fig. 10 ‣ 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a> demonstrates different EHR data formats.</p>
</div>
<div id="S3.SS1.p2" class="ltx_para">
<p id="S3.SS1.p2.1" class="ltx_p">In this paper, we categorize different generative models based on their ability to handle the complexities of EHR data. We achieve this through a comprehensive evaluation that considers a diverse set of properties critical for generating realistic and heterogeneous EHR data. These properties include: capturing temporal relationships between events, generating data conditional on specific patient characteristics, creating data for uncommon diseases or underrepresented populations, effectively handling the high-dimensionality of EHR data, modeling missing data patterns, and generating both static and temporal features jointly. This comprehensive evaluation provides a thorough assessment of the current state-of-the-art in synthetic data generation for EHRs.</p>
</div>
<div id="S3.SS1.p3" class="ltx_para">
<p id="S3.SS1.p3.1" class="ltx_p">We start with an overview of early work in EHR synthesis and move to the different formats (longitudinal, time dependent, aggregated, and snapshot). A summary of the models is presented in Table <a href="#S3.T3" title="Table 3 ‣ 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, indicating the technology used for generation, type of features and different properties that the generative models are capable of, as well as the evaluation framework.</p>
</div>
<figure id="S3.F10" class="ltx_figure"><img src="/html/2407.00116/assets/ehr_format.jpg" id="S3.F10.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="449" height="190" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 10: </span>Different formats that EHR data exist in literature. <span id="S3.F10.29.1" class="ltx_text ltx_font_bold">A. Longitudinal:</span> The most comprehensive yet least accessible due to privacy concerns, longitudinal EHRs track multiple attributes over the span of a patient’s life. A patient record <math id="S3.F10.13.m1.1" class="ltx_Math" alttext="R_{n}" display="inline"><semantics id="S3.F10.13.m1.1b"><msub id="S3.F10.13.m1.1.1" xref="S3.F10.13.m1.1.1.cmml"><mi id="S3.F10.13.m1.1.1.2" xref="S3.F10.13.m1.1.1.2.cmml">R</mi><mi id="S3.F10.13.m1.1.1.3" xref="S3.F10.13.m1.1.1.3.cmml">n</mi></msub><annotation-xml encoding="MathML-Content" id="S3.F10.13.m1.1c"><apply id="S3.F10.13.m1.1.1.cmml" xref="S3.F10.13.m1.1.1"><csymbol cd="ambiguous" id="S3.F10.13.m1.1.1.1.cmml" xref="S3.F10.13.m1.1.1">subscript</csymbol><ci id="S3.F10.13.m1.1.1.2.cmml" xref="S3.F10.13.m1.1.1.2">𝑅</ci><ci id="S3.F10.13.m1.1.1.3.cmml" xref="S3.F10.13.m1.1.1.3">𝑛</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.13.m1.1d">R_{n}</annotation></semantics></math> consists of a sequence of visits of a patient to the healthcare provider <math id="S3.F10.14.m2.4" class="ltx_Math" alttext="R_{n}=V_{1},V_{2},\ldots,V_{T}" display="inline"><semantics id="S3.F10.14.m2.4b"><mrow id="S3.F10.14.m2.4.4" xref="S3.F10.14.m2.4.4.cmml"><msub id="S3.F10.14.m2.4.4.5" xref="S3.F10.14.m2.4.4.5.cmml"><mi id="S3.F10.14.m2.4.4.5.2" xref="S3.F10.14.m2.4.4.5.2.cmml">R</mi><mi id="S3.F10.14.m2.4.4.5.3" xref="S3.F10.14.m2.4.4.5.3.cmml">n</mi></msub><mo id="S3.F10.14.m2.4.4.4" xref="S3.F10.14.m2.4.4.4.cmml">=</mo><mrow id="S3.F10.14.m2.4.4.3.3" xref="S3.F10.14.m2.4.4.3.4.cmml"><msub id="S3.F10.14.m2.2.2.1.1.1" xref="S3.F10.14.m2.2.2.1.1.1.cmml"><mi id="S3.F10.14.m2.2.2.1.1.1.2" xref="S3.F10.14.m2.2.2.1.1.1.2.cmml">V</mi><mn id="S3.F10.14.m2.2.2.1.1.1.3" xref="S3.F10.14.m2.2.2.1.1.1.3.cmml">1</mn></msub><mo id="S3.F10.14.m2.4.4.3.3.4" xref="S3.F10.14.m2.4.4.3.4.cmml">,</mo><msub id="S3.F10.14.m2.3.3.2.2.2" xref="S3.F10.14.m2.3.3.2.2.2.cmml"><mi id="S3.F10.14.m2.3.3.2.2.2.2" xref="S3.F10.14.m2.3.3.2.2.2.2.cmml">V</mi><mn id="S3.F10.14.m2.3.3.2.2.2.3" xref="S3.F10.14.m2.3.3.2.2.2.3.cmml">2</mn></msub><mo id="S3.F10.14.m2.4.4.3.3.5" xref="S3.F10.14.m2.4.4.3.4.cmml">,</mo><mi mathvariant="normal" id="S3.F10.14.m2.1.1" xref="S3.F10.14.m2.1.1.cmml">…</mi><mo id="S3.F10.14.m2.4.4.3.3.6" xref="S3.F10.14.m2.4.4.3.4.cmml">,</mo><msub id="S3.F10.14.m2.4.4.3.3.3" xref="S3.F10.14.m2.4.4.3.3.3.cmml"><mi id="S3.F10.14.m2.4.4.3.3.3.2" xref="S3.F10.14.m2.4.4.3.3.3.2.cmml">V</mi><mi id="S3.F10.14.m2.4.4.3.3.3.3" xref="S3.F10.14.m2.4.4.3.3.3.3.cmml">T</mi></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.14.m2.4c"><apply id="S3.F10.14.m2.4.4.cmml" xref="S3.F10.14.m2.4.4"><eq id="S3.F10.14.m2.4.4.4.cmml" xref="S3.F10.14.m2.4.4.4"></eq><apply id="S3.F10.14.m2.4.4.5.cmml" xref="S3.F10.14.m2.4.4.5"><csymbol cd="ambiguous" id="S3.F10.14.m2.4.4.5.1.cmml" xref="S3.F10.14.m2.4.4.5">subscript</csymbol><ci id="S3.F10.14.m2.4.4.5.2.cmml" xref="S3.F10.14.m2.4.4.5.2">𝑅</ci><ci id="S3.F10.14.m2.4.4.5.3.cmml" xref="S3.F10.14.m2.4.4.5.3">𝑛</ci></apply><list id="S3.F10.14.m2.4.4.3.4.cmml" xref="S3.F10.14.m2.4.4.3.3"><apply id="S3.F10.14.m2.2.2.1.1.1.cmml" xref="S3.F10.14.m2.2.2.1.1.1"><csymbol cd="ambiguous" id="S3.F10.14.m2.2.2.1.1.1.1.cmml" xref="S3.F10.14.m2.2.2.1.1.1">subscript</csymbol><ci id="S3.F10.14.m2.2.2.1.1.1.2.cmml" xref="S3.F10.14.m2.2.2.1.1.1.2">𝑉</ci><cn type="integer" id="S3.F10.14.m2.2.2.1.1.1.3.cmml" xref="S3.F10.14.m2.2.2.1.1.1.3">1</cn></apply><apply id="S3.F10.14.m2.3.3.2.2.2.cmml" xref="S3.F10.14.m2.3.3.2.2.2"><csymbol cd="ambiguous" id="S3.F10.14.m2.3.3.2.2.2.1.cmml" xref="S3.F10.14.m2.3.3.2.2.2">subscript</csymbol><ci id="S3.F10.14.m2.3.3.2.2.2.2.cmml" xref="S3.F10.14.m2.3.3.2.2.2.2">𝑉</ci><cn type="integer" id="S3.F10.14.m2.3.3.2.2.2.3.cmml" xref="S3.F10.14.m2.3.3.2.2.2.3">2</cn></apply><ci id="S3.F10.14.m2.1.1.cmml" xref="S3.F10.14.m2.1.1">…</ci><apply id="S3.F10.14.m2.4.4.3.3.3.cmml" xref="S3.F10.14.m2.4.4.3.3.3"><csymbol cd="ambiguous" id="S3.F10.14.m2.4.4.3.3.3.1.cmml" xref="S3.F10.14.m2.4.4.3.3.3">subscript</csymbol><ci id="S3.F10.14.m2.4.4.3.3.3.2.cmml" xref="S3.F10.14.m2.4.4.3.3.3.2">𝑉</ci><ci id="S3.F10.14.m2.4.4.3.3.3.3.cmml" xref="S3.F10.14.m2.4.4.3.3.3.3">𝑇</ci></apply></list></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.14.m2.4d">R_{n}=V_{1},V_{2},\ldots,V_{T}</annotation></semantics></math>, where each visit consists of a varying number of different data types (medical codes, lab results, diagnoses, clinical notes). In majority of cases, Each visit <math id="S3.F10.15.m3.1" class="ltx_Math" alttext="V_{t}" display="inline"><semantics id="S3.F10.15.m3.1b"><msub id="S3.F10.15.m3.1.1" xref="S3.F10.15.m3.1.1.cmml"><mi id="S3.F10.15.m3.1.1.2" xref="S3.F10.15.m3.1.1.2.cmml">V</mi><mi id="S3.F10.15.m3.1.1.3" xref="S3.F10.15.m3.1.1.3.cmml">t</mi></msub><annotation-xml encoding="MathML-Content" id="S3.F10.15.m3.1c"><apply id="S3.F10.15.m3.1.1.cmml" xref="S3.F10.15.m3.1.1"><csymbol cd="ambiguous" id="S3.F10.15.m3.1.1.1.cmml" xref="S3.F10.15.m3.1.1">subscript</csymbol><ci id="S3.F10.15.m3.1.1.2.cmml" xref="S3.F10.15.m3.1.1.2">𝑉</ci><ci id="S3.F10.15.m3.1.1.3.cmml" xref="S3.F10.15.m3.1.1.3">𝑡</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.15.m3.1d">V_{t}</annotation></semantics></math> is a <math id="S3.F10.16.m4.1" class="ltx_Math" alttext="K" display="inline"><semantics id="S3.F10.16.m4.1b"><mi id="S3.F10.16.m4.1.1" xref="S3.F10.16.m4.1.1.cmml">K</mi><annotation-xml encoding="MathML-Content" id="S3.F10.16.m4.1c"><ci id="S3.F10.16.m4.1.1.cmml" xref="S3.F10.16.m4.1.1">𝐾</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.16.m4.1d">K</annotation></semantics></math>-dimensional binary vector, where <math id="S3.F10.17.m5.1" class="ltx_Math" alttext="V_{T}[k]=1" display="inline"><semantics id="S3.F10.17.m5.1b"><mrow id="S3.F10.17.m5.1.2" xref="S3.F10.17.m5.1.2.cmml"><mrow id="S3.F10.17.m5.1.2.2" xref="S3.F10.17.m5.1.2.2.cmml"><msub id="S3.F10.17.m5.1.2.2.2" xref="S3.F10.17.m5.1.2.2.2.cmml"><mi id="S3.F10.17.m5.1.2.2.2.2" xref="S3.F10.17.m5.1.2.2.2.2.cmml">V</mi><mi id="S3.F10.17.m5.1.2.2.2.3" xref="S3.F10.17.m5.1.2.2.2.3.cmml">T</mi></msub><mo lspace="0em" rspace="0em" id="S3.F10.17.m5.1.2.2.1" xref="S3.F10.17.m5.1.2.2.1.cmml">​</mo><mrow id="S3.F10.17.m5.1.2.2.3.2" xref="S3.F10.17.m5.1.2.2.3.1.cmml"><mo stretchy="false" id="S3.F10.17.m5.1.2.2.3.2.1" xref="S3.F10.17.m5.1.2.2.3.1.1.cmml">[</mo><mi id="S3.F10.17.m5.1.1" xref="S3.F10.17.m5.1.1.cmml">k</mi><mo stretchy="false" id="S3.F10.17.m5.1.2.2.3.2.2" xref="S3.F10.17.m5.1.2.2.3.1.1.cmml">]</mo></mrow></mrow><mo id="S3.F10.17.m5.1.2.1" xref="S3.F10.17.m5.1.2.1.cmml">=</mo><mn id="S3.F10.17.m5.1.2.3" xref="S3.F10.17.m5.1.2.3.cmml">1</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.17.m5.1c"><apply id="S3.F10.17.m5.1.2.cmml" xref="S3.F10.17.m5.1.2"><eq id="S3.F10.17.m5.1.2.1.cmml" xref="S3.F10.17.m5.1.2.1"></eq><apply id="S3.F10.17.m5.1.2.2.cmml" xref="S3.F10.17.m5.1.2.2"><times id="S3.F10.17.m5.1.2.2.1.cmml" xref="S3.F10.17.m5.1.2.2.1"></times><apply id="S3.F10.17.m5.1.2.2.2.cmml" xref="S3.F10.17.m5.1.2.2.2"><csymbol cd="ambiguous" id="S3.F10.17.m5.1.2.2.2.1.cmml" xref="S3.F10.17.m5.1.2.2.2">subscript</csymbol><ci id="S3.F10.17.m5.1.2.2.2.2.cmml" xref="S3.F10.17.m5.1.2.2.2.2">𝑉</ci><ci id="S3.F10.17.m5.1.2.2.2.3.cmml" xref="S3.F10.17.m5.1.2.2.2.3">𝑇</ci></apply><apply id="S3.F10.17.m5.1.2.2.3.1.cmml" xref="S3.F10.17.m5.1.2.2.3.2"><csymbol cd="latexml" id="S3.F10.17.m5.1.2.2.3.1.1.cmml" xref="S3.F10.17.m5.1.2.2.3.2.1">delimited-[]</csymbol><ci id="S3.F10.17.m5.1.1.cmml" xref="S3.F10.17.m5.1.1">𝑘</ci></apply></apply><cn type="integer" id="S3.F10.17.m5.1.2.3.cmml" xref="S3.F10.17.m5.1.2.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.17.m5.1d">V_{T}[k]=1</annotation></semantics></math> if the <math id="S3.F10.18.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S3.F10.18.m6.1b"><mi id="S3.F10.18.m6.1.1" xref="S3.F10.18.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S3.F10.18.m6.1c"><ci id="S3.F10.18.m6.1.1.cmml" xref="S3.F10.18.m6.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.18.m6.1d">k</annotation></semantics></math>-th code for patient <math id="S3.F10.19.m7.1" class="ltx_Math" alttext="n" display="inline"><semantics id="S3.F10.19.m7.1b"><mi id="S3.F10.19.m7.1.1" xref="S3.F10.19.m7.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="S3.F10.19.m7.1c"><ci id="S3.F10.19.m7.1.1.cmml" xref="S3.F10.19.m7.1.1">𝑛</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.19.m7.1d">n</annotation></semantics></math> was observed at visit <math id="S3.F10.20.m8.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S3.F10.20.m8.1b"><mi id="S3.F10.20.m8.1.1" xref="S3.F10.20.m8.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S3.F10.20.m8.1c"><ci id="S3.F10.20.m8.1.1.cmml" xref="S3.F10.20.m8.1.1">𝑡</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.20.m8.1d">t</annotation></semantics></math> and <math id="S3.F10.21.m9.1" class="ltx_Math" alttext="0" display="inline"><semantics id="S3.F10.21.m9.1b"><mn id="S3.F10.21.m9.1.1" xref="S3.F10.21.m9.1.1.cmml">0</mn><annotation-xml encoding="MathML-Content" id="S3.F10.21.m9.1c"><cn type="integer" id="S3.F10.21.m9.1.1.cmml" xref="S3.F10.21.m9.1.1">0</cn></annotation-xml></semantics></math> otherwise. <math id="S3.F10.22.m10.1" class="ltx_Math" alttext="K" display="inline"><semantics id="S3.F10.22.m10.1b"><mi id="S3.F10.22.m10.1.1" xref="S3.F10.22.m10.1.1.cmml">K</mi><annotation-xml encoding="MathML-Content" id="S3.F10.22.m10.1c"><ci id="S3.F10.22.m10.1.1.cmml" xref="S3.F10.22.m10.1.1">𝐾</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.22.m10.1d">K</annotation></semantics></math> denotes the cardinality of the set of possible codes.An Electronic Health Record (EHR) consists of a set of patient records <math id="S3.F10.23.m11.5" class="ltx_Math" alttext="D=\{R_{1},\ldots,R_{n},\ldots,R_{N}\}" display="inline"><semantics id="S3.F10.23.m11.5b"><mrow id="S3.F10.23.m11.5.5" xref="S3.F10.23.m11.5.5.cmml"><mi id="S3.F10.23.m11.5.5.5" xref="S3.F10.23.m11.5.5.5.cmml">D</mi><mo id="S3.F10.23.m11.5.5.4" xref="S3.F10.23.m11.5.5.4.cmml">=</mo><mrow id="S3.F10.23.m11.5.5.3.3" xref="S3.F10.23.m11.5.5.3.4.cmml"><mo stretchy="false" id="S3.F10.23.m11.5.5.3.3.4" xref="S3.F10.23.m11.5.5.3.4.cmml">{</mo><msub id="S3.F10.23.m11.3.3.1.1.1" xref="S3.F10.23.m11.3.3.1.1.1.cmml"><mi id="S3.F10.23.m11.3.3.1.1.1.2" xref="S3.F10.23.m11.3.3.1.1.1.2.cmml">R</mi><mn id="S3.F10.23.m11.3.3.1.1.1.3" xref="S3.F10.23.m11.3.3.1.1.1.3.cmml">1</mn></msub><mo id="S3.F10.23.m11.5.5.3.3.5" xref="S3.F10.23.m11.5.5.3.4.cmml">,</mo><mi mathvariant="normal" id="S3.F10.23.m11.1.1" xref="S3.F10.23.m11.1.1.cmml">…</mi><mo id="S3.F10.23.m11.5.5.3.3.6" xref="S3.F10.23.m11.5.5.3.4.cmml">,</mo><msub id="S3.F10.23.m11.4.4.2.2.2" xref="S3.F10.23.m11.4.4.2.2.2.cmml"><mi id="S3.F10.23.m11.4.4.2.2.2.2" xref="S3.F10.23.m11.4.4.2.2.2.2.cmml">R</mi><mi id="S3.F10.23.m11.4.4.2.2.2.3" xref="S3.F10.23.m11.4.4.2.2.2.3.cmml">n</mi></msub><mo id="S3.F10.23.m11.5.5.3.3.7" xref="S3.F10.23.m11.5.5.3.4.cmml">,</mo><mi mathvariant="normal" id="S3.F10.23.m11.2.2" xref="S3.F10.23.m11.2.2.cmml">…</mi><mo id="S3.F10.23.m11.5.5.3.3.8" xref="S3.F10.23.m11.5.5.3.4.cmml">,</mo><msub id="S3.F10.23.m11.5.5.3.3.3" xref="S3.F10.23.m11.5.5.3.3.3.cmml"><mi id="S3.F10.23.m11.5.5.3.3.3.2" xref="S3.F10.23.m11.5.5.3.3.3.2.cmml">R</mi><mi id="S3.F10.23.m11.5.5.3.3.3.3" xref="S3.F10.23.m11.5.5.3.3.3.3.cmml">N</mi></msub><mo stretchy="false" id="S3.F10.23.m11.5.5.3.3.9" xref="S3.F10.23.m11.5.5.3.4.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S3.F10.23.m11.5c"><apply id="S3.F10.23.m11.5.5.cmml" xref="S3.F10.23.m11.5.5"><eq id="S3.F10.23.m11.5.5.4.cmml" xref="S3.F10.23.m11.5.5.4"></eq><ci id="S3.F10.23.m11.5.5.5.cmml" xref="S3.F10.23.m11.5.5.5">𝐷</ci><set id="S3.F10.23.m11.5.5.3.4.cmml" xref="S3.F10.23.m11.5.5.3.3"><apply id="S3.F10.23.m11.3.3.1.1.1.cmml" xref="S3.F10.23.m11.3.3.1.1.1"><csymbol cd="ambiguous" id="S3.F10.23.m11.3.3.1.1.1.1.cmml" xref="S3.F10.23.m11.3.3.1.1.1">subscript</csymbol><ci id="S3.F10.23.m11.3.3.1.1.1.2.cmml" xref="S3.F10.23.m11.3.3.1.1.1.2">𝑅</ci><cn type="integer" id="S3.F10.23.m11.3.3.1.1.1.3.cmml" xref="S3.F10.23.m11.3.3.1.1.1.3">1</cn></apply><ci id="S3.F10.23.m11.1.1.cmml" xref="S3.F10.23.m11.1.1">…</ci><apply id="S3.F10.23.m11.4.4.2.2.2.cmml" xref="S3.F10.23.m11.4.4.2.2.2"><csymbol cd="ambiguous" id="S3.F10.23.m11.4.4.2.2.2.1.cmml" xref="S3.F10.23.m11.4.4.2.2.2">subscript</csymbol><ci id="S3.F10.23.m11.4.4.2.2.2.2.cmml" xref="S3.F10.23.m11.4.4.2.2.2.2">𝑅</ci><ci id="S3.F10.23.m11.4.4.2.2.2.3.cmml" xref="S3.F10.23.m11.4.4.2.2.2.3">𝑛</ci></apply><ci id="S3.F10.23.m11.2.2.cmml" xref="S3.F10.23.m11.2.2">…</ci><apply id="S3.F10.23.m11.5.5.3.3.3.cmml" xref="S3.F10.23.m11.5.5.3.3.3"><csymbol cd="ambiguous" id="S3.F10.23.m11.5.5.3.3.3.1.cmml" xref="S3.F10.23.m11.5.5.3.3.3">subscript</csymbol><ci id="S3.F10.23.m11.5.5.3.3.3.2.cmml" xref="S3.F10.23.m11.5.5.3.3.3.2">𝑅</ci><ci id="S3.F10.23.m11.5.5.3.3.3.3.cmml" xref="S3.F10.23.m11.5.5.3.3.3.3">𝑁</ci></apply></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.23.m11.5d">D=\{R_{1},\ldots,R_{n},\ldots,R_{N}\}</annotation></semantics></math>, where <math id="S3.F10.24.m12.1" class="ltx_Math" alttext="N" display="inline"><semantics id="S3.F10.24.m12.1b"><mi id="S3.F10.24.m12.1.1" xref="S3.F10.24.m12.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="S3.F10.24.m12.1c"><ci id="S3.F10.24.m12.1.1.cmml" xref="S3.F10.24.m12.1.1">𝑁</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.F10.24.m12.1d">N</annotation></semantics></math> denotes the number of patients in the set. In this form, the temporal correlations between the visits should be maintained. <span id="S3.F10.30.2" class="ltx_text ltx_font_bold">B. Time-dependent EHR:</span> This format captures regular, time-specific Data from monitoring devices, recorded at regular time intervals, focusing on physiological measurements such as Heart rate, respiratory rate,SpO2. <span id="S3.F10.31.3" class="ltx_text ltx_font_bold">C. Aggregate:</span> This format flattens the diagnosis, medications, and procedure codes in the longitudinal data of a patient into aggregated data points represented in a single row with binary or count aggregations. The temporal and sequential aspect of the data is lost with this kind of processing, offering a cumulative overview instead <span id="S3.F10.32.4" class="ltx_text ltx_font_bold">D. Snapshot or Cross-subsectional:</span> This is the most common format for public medical data, presenting a single static snapshot of a patient’s EHR focusing on selected attributes. </figcaption>
</figure>
<figure id="S3.T3" class="ltx_table">
<table id="S3.T3.2" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T3.2.3.1" class="ltx_tr">
<td id="S3.T3.2.3.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T3.2.3.1.1.1" class="ltx_text ltx_font_bold" style="font-size:50%;">Synthesis</span></td>
<td id="S3.T3.2.3.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T3.2.3.1.2.1" class="ltx_text ltx_font_bold" style="font-size:50%;">Type</span></td>
<td id="S3.T3.2.3.1.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T3.2.3.1.3.1" class="ltx_text ltx_font_bold" style="font-size:50%;">Features</span></td>
<td id="S3.T3.2.3.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T3.2.3.1.4.1" class="ltx_text ltx_font_bold" style="font-size:50%;">Technology</span></td>
<td id="S3.T3.2.3.1.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T3.2.3.1.5.1" class="ltx_text ltx_font_bold" style="font-size:50%;">Paper</span></td>
<td id="S3.T3.2.3.1.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T3.2.3.1.6.1" class="ltx_text ltx_font_bold" style="font-size:50%;">DT</span></td>
<td id="S3.T3.2.3.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="6"><span id="S3.T3.2.3.1.7.1" class="ltx_text ltx_font_bold" style="font-size:50%;">Features</span></td>
<td id="S3.T3.2.3.1.8" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T3.2.3.1.8.1" class="ltx_text" style="font-size:50%;">Evaluation</span></td>
<td id="S3.T3.2.3.1.9" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T3.2.3.1.9.1" class="ltx_text ltx_font_bold" style="font-size:50%;">Code</span></td>
</tr>
<tr id="S3.T3.2.4.2" class="ltx_tr">
<td id="S3.T3.2.4.2.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.1.1" class="ltx_text" style="font-size:50%;">T</span></td>
<td id="S3.T3.2.4.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.2.1" class="ltx_text" style="font-size:50%;">C</span></td>
<td id="S3.T3.2.4.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.3.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.4.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.4.1" class="ltx_text" style="font-size:50%;">H</span></td>
<td id="S3.T3.2.4.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.5.1" class="ltx_text" style="font-size:50%;">M</span></td>
<td id="S3.T3.2.4.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.6.1" class="ltx_text" style="font-size:50%;">J</span></td>
<td id="S3.T3.2.4.2.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T3.2.4.2.7.1" class="ltx_text" style="font-size:50%;">U</span></td>
<td id="S3.T3.2.4.2.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.8.1" class="ltx_text" style="font-size:50%;">F</span></td>
<td id="S3.T3.2.4.2.9" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.9.1" class="ltx_text" style="font-size:50%;">D</span></td>
<td id="S3.T3.2.4.2.10" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.10.1" class="ltx_text" style="font-size:50%;">Q</span></td>
<td id="S3.T3.2.4.2.11" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.11.1" class="ltx_text" style="font-size:50%;">C</span></td>
<td id="S3.T3.2.4.2.12" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.4.2.12.1" class="ltx_text" style="font-size:50%;">P</span></td>
</tr>
<tr id="S3.T3.2.5.3" class="ltx_tr">
<td id="S3.T3.2.5.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.5.3.1.1" class="ltx_text" style="font-size:50%;">Train</span></td>
<td id="S3.T3.2.5.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.5.3.2.1" class="ltx_text" style="font-size:50%;">Test</span></td>
</tr>
<tr id="S3.T3.2.6.4" class="ltx_tr">
<td id="S3.T3.2.6.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T3.2.6.4.1.1" class="ltx_text" style="font-size:50%;">Longitudinal</span></td>
<td id="S3.T3.2.6.4.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T3.2.6.4.2.1" class="ltx_text" style="font-size:50%;">GAN</span></td>
<td id="S3.T3.2.6.4.3" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.3.1" class="ltx_text" style="font-size:50%;">Dis</span></td>
<td id="S3.T3.2.6.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T3.2.6.4.4.1" class="ltx_text" style="font-size:50%;">WGAN</span></td>
<td id="S3.T3.2.6.4.5" class="ltx_td ltx_align_center ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.6.4.5.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib70" title="" class="ltx_ref">70</a><span id="S3.T3.2.6.4.5.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.6.4.6" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.6.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.6.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.7.1" class="ltx_text" style="font-size:50%;">✓(V)</span></td>
<td id="S3.T3.2.6.4.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.6.4.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.6.4.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.10.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.6.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.6.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.6.4.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.13.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.2.6.4.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.14.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.6.4.15" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.15.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.6.4.16" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.6.4.17" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.6.4.18" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.6.4.19" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.6.4.19.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.6.4.20" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T3.2.7.5" class="ltx_tr">
<td id="S3.T3.2.7.5.1" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.1.1" class="ltx_text" style="font-size:50%;">Dis</span></td>
<td id="S3.T3.2.7.5.2" class="ltx_td ltx_align_left"><span id="S3.T3.2.7.5.2.1" class="ltx_text" style="font-size:50%;">WGAN-GP</span></td>
<td id="S3.T3.2.7.5.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.3.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib58" title="" class="ltx_ref">58</a>]</cite></span></td>
<td id="S3.T3.2.7.5.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.4.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.7.5.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.7.5.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.6.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.7.5.7" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.7.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.7.5.8" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.8.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.7.5.9" class="ltx_td"></td>
<td id="S3.T3.2.7.5.10" class="ltx_td"></td>
<td id="S3.T3.2.7.5.11" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T3.2.7.5.11.1" class="ltx_text" style="font-size:50%;">**</span></td>
<td id="S3.T3.2.7.5.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.12.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.7.5.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.13.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.7.5.14" class="ltx_td"></td>
<td id="S3.T3.2.7.5.15" class="ltx_td"></td>
<td id="S3.T3.2.7.5.16" class="ltx_td"></td>
<td id="S3.T3.2.7.5.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.7.5.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.2.8.6" class="ltx_tr">
<td id="S3.T3.2.8.6.1" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.1.1" class="ltx_text" style="font-size:50%;">VAE</span></td>
<td id="S3.T3.2.8.6.2" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.2.1" class="ltx_text" style="font-size:50%;">Dis</span></td>
<td id="S3.T3.2.8.6.3" class="ltx_td ltx_align_left"><span id="S3.T3.2.8.6.3.1" class="ltx_text" style="font-size:50%;">EHR VAE</span></td>
<td id="S3.T3.2.8.6.4" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.8.6.4.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib71" title="" class="ltx_ref">71</a><span id="S3.T3.2.8.6.4.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.8.6.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.5.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.8.6.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.6.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.8.6.7" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.7.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.8.6.8" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.8.1" class="ltx_text" style="font-size:50%;">-</span></td>
<td id="S3.T3.2.8.6.9" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.9.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.8.6.10" class="ltx_td"></td>
<td id="S3.T3.2.8.6.11" class="ltx_td"></td>
<td id="S3.T3.2.8.6.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.12.1" class="ltx_text" style="font-size:50%;">S,A</span></td>
<td id="S3.T3.2.8.6.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.13.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.8.6.14" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.14.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.8.6.15" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.15.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.8.6.16" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.16.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.8.6.17" class="ltx_td"></td>
<td id="S3.T3.2.8.6.18" class="ltx_td ltx_align_center"><span id="S3.T3.2.8.6.18.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.8.6.19" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.9.7" class="ltx_tr">
<td id="S3.T3.2.9.7.1" class="ltx_td ltx_align_center"><span id="S3.T3.2.9.7.1.1" class="ltx_text" style="font-size:50%;">DM</span></td>
<td id="S3.T3.2.9.7.2" class="ltx_td ltx_align_center"><span id="S3.T3.2.9.7.2.1" class="ltx_text" style="font-size:50%;">Dis</span></td>
<td id="S3.T3.2.9.7.3" class="ltx_td ltx_align_left"><span id="S3.T3.2.9.7.3.1" class="ltx_text" style="font-size:50%;">DDPM</span></td>
<td id="S3.T3.2.9.7.4" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.9.7.4.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib72" title="" class="ltx_ref">72</a><span id="S3.T3.2.9.7.4.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.9.7.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.9.7.5.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.9.7.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.9.7.6.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.9.7.7" class="ltx_td"></td>
<td id="S3.T3.2.9.7.8" class="ltx_td"></td>
<td id="S3.T3.2.9.7.9" class="ltx_td ltx_align_center"><span id="S3.T3.2.9.7.9.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.9.7.10" class="ltx_td"></td>
<td id="S3.T3.2.9.7.11" class="ltx_td"></td>
<td id="S3.T3.2.9.7.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.9.7.12.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.9.7.13" class="ltx_td"></td>
<td id="S3.T3.2.9.7.14" class="ltx_td"></td>
<td id="S3.T3.2.9.7.15" class="ltx_td"></td>
<td id="S3.T3.2.9.7.16" class="ltx_td"></td>
<td id="S3.T3.2.9.7.17" class="ltx_td"></td>
<td id="S3.T3.2.9.7.18" class="ltx_td"></td>
<td id="S3.T3.2.9.7.19" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.10.8" class="ltx_tr">
<td id="S3.T3.2.10.8.1" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T3.2.10.8.1.1" class="ltx_text" style="font-size:50%;">LM</span></td>
<td id="S3.T3.2.10.8.2" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.2.1" class="ltx_text" style="font-size:50%;">Dis</span></td>
<td id="S3.T3.2.10.8.3" class="ltx_td ltx_align_left"><span id="S3.T3.2.10.8.3.1" class="ltx_text" style="font-size:50%;">Prompt based LM</span></td>
<td id="S3.T3.2.10.8.4" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.10.8.4.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib73" title="" class="ltx_ref">73</a><span id="S3.T3.2.10.8.4.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.10.8.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.5.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.10.8.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.6.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.10.8.7" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.7.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.10.8.8" class="ltx_td"></td>
<td id="S3.T3.2.10.8.9" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.9.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.10.8.10" class="ltx_td"></td>
<td id="S3.T3.2.10.8.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.11.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.10.8.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.12.1" class="ltx_text" style="font-size:50%;">S,A</span></td>
<td id="S3.T3.2.10.8.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.13.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.10.8.14" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.14.1" class="ltx_text" style="font-size:50%;">✓(Perp)</span></td>
<td id="S3.T3.2.10.8.15" class="ltx_td"></td>
<td id="S3.T3.2.10.8.16" class="ltx_td"></td>
<td id="S3.T3.2.10.8.17" class="ltx_td"></td>
<td id="S3.T3.2.10.8.18" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.18.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.10.8.19" class="ltx_td ltx_align_center"><span id="S3.T3.2.10.8.19.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.2.11.9" class="ltx_tr">
<td id="S3.T3.2.11.9.1" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.1.1" class="ltx_text" style="font-size:50%;">Mix</span></td>
<td id="S3.T3.2.11.9.2" class="ltx_td ltx_align_left"><span id="S3.T3.2.11.9.2.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.11.9.2.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.11.9.2.1.1.1" class="ltx_tr">
<span id="S3.T3.2.11.9.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">Hierarchical</span></span>
<span id="S3.T3.2.11.9.2.1.1.2" class="ltx_tr">
<span id="S3.T3.2.11.9.2.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">auto-regressive LM</span></span>
</span></span></td>
<td id="S3.T3.2.11.9.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.3.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite></span></td>
<td id="S3.T3.2.11.9.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.4.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.11.9.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.11.9.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.6.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.11.9.7" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.7.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.11.9.8" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.8.1" class="ltx_text" style="font-size:50%;">✓✓</span></td>
<td id="S3.T3.2.11.9.9" class="ltx_td"></td>
<td id="S3.T3.2.11.9.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.10.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.11.9.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.11.1" class="ltx_text" style="font-size:50%;">S,A</span></td>
<td id="S3.T3.2.11.9.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.12.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.11.9.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.13.1" class="ltx_text" style="font-size:50%;">✓(Perp)</span></td>
<td id="S3.T3.2.11.9.14" class="ltx_td"></td>
<td id="S3.T3.2.11.9.15" class="ltx_td"></td>
<td id="S3.T3.2.11.9.16" class="ltx_td"></td>
<td id="S3.T3.2.11.9.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.11.9.18" class="ltx_td ltx_align_center"><span id="S3.T3.2.11.9.18.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.2.12.10" class="ltx_tr">
<td id="S3.T3.2.12.10.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="8"><span id="S3.T3.2.12.10.1.1" class="ltx_text" style="font-size:50%;">Timeseries</span></td>
<td id="S3.T3.2.12.10.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.2.1" class="ltx_text" style="font-size:50%;">GAN</span></td>
<td id="S3.T3.2.12.10.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="8"><span id="S3.T3.2.12.10.3.1" class="ltx_text" style="font-size:50%;">Mix</span></td>
<td id="S3.T3.2.12.10.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T3.2.12.10.4.1" class="ltx_text" style="font-size:50%;">WGAN</span></td>
<td id="S3.T3.2.12.10.5" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.5.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib75" title="" class="ltx_ref">75</a>]</cite></span></td>
<td id="S3.T3.2.12.10.6" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.6.1" class="ltx_text" style="font-size:50%;">Class,Pred</span></td>
<td id="S3.T3.2.12.10.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.7.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.12.10.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.8.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.12.10.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.12.10.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.10.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.12.10.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.12.10.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.12.10.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.13.1" class="ltx_text" style="font-size:50%;">S,R</span></td>
<td id="S3.T3.2.12.10.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.14.1" class="ltx_text" style="font-size:50%;">S,R</span></td>
<td id="S3.T3.2.12.10.15" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.12.10.16" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.12.10.17" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.12.10.18" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.12.10.19" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.12.10.19.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.12.10.20" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T3.2.13.11" class="ltx_tr">
<td id="S3.T3.2.13.11.1" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.1.1" class="ltx_text" style="font-size:50%;">VAE</span></td>
<td id="S3.T3.2.13.11.2" class="ltx_td ltx_align_left"><span id="S3.T3.2.13.11.2.1" class="ltx_text" style="font-size:50%;">Dynamical VAE</span></td>
<td id="S3.T3.2.13.11.3" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.13.11.3.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib76" title="" class="ltx_ref">76</a><span id="S3.T3.2.13.11.3.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.13.11.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.4.1" class="ltx_text" style="font-size:50%;">Class</span></td>
<td id="S3.T3.2.13.11.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.13.11.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.6.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.13.11.7" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.7.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.13.11.8" class="ltx_td"></td>
<td id="S3.T3.2.13.11.9" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.9.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.13.11.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.10.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.13.11.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.11.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.2.13.11.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.12.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.13.11.13" class="ltx_td"></td>
<td id="S3.T3.2.13.11.14" class="ltx_td"></td>
<td id="S3.T3.2.13.11.15" class="ltx_td"></td>
<td id="S3.T3.2.13.11.16" class="ltx_td"></td>
<td id="S3.T3.2.13.11.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.13.11.18" class="ltx_td ltx_align_center"><span id="S3.T3.2.13.11.18.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.2.14.12" class="ltx_tr">
<td id="S3.T3.2.14.12.1" class="ltx_td ltx_align_center" rowspan="4"><span id="S3.T3.2.14.12.1.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.14.12.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.14.12.1.1.1.1" class="ltx_tr">
<span id="S3.T3.2.14.12.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Hybrid :</span></span>
<span id="S3.T3.2.14.12.1.1.1.2" class="ltx_tr">
<span id="S3.T3.2.14.12.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">VAE</span></span>
<span id="S3.T3.2.14.12.1.1.1.3" class="ltx_tr">
<span id="S3.T3.2.14.12.1.1.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">+</span></span>
<span id="S3.T3.2.14.12.1.1.1.4" class="ltx_tr">
<span id="S3.T3.2.14.12.1.1.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center">GAN</span></span>
</span></span></td>
<td id="S3.T3.2.14.12.2" class="ltx_td ltx_align_left"><span id="S3.T3.2.14.12.2.1" class="ltx_text" style="font-size:50%;">AE &amp; WGAN-GP</span></td>
<td id="S3.T3.2.14.12.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.3.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite></span></td>
<td id="S3.T3.2.14.12.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.4.1" class="ltx_text" style="font-size:50%;">Class</span></td>
<td id="S3.T3.2.14.12.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.14.12.6" class="ltx_td"></td>
<td id="S3.T3.2.14.12.7" class="ltx_td"></td>
<td id="S3.T3.2.14.12.8" class="ltx_td"></td>
<td id="S3.T3.2.14.12.9" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.9.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.14.12.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.10.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.14.12.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.11.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.2.14.12.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.12.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.14.12.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.13.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.14.12.14" class="ltx_td"></td>
<td id="S3.T3.2.14.12.15" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.15.1" class="ltx_text ltx_font_bold" style="font-size:50%;">*</span></td>
<td id="S3.T3.2.14.12.16" class="ltx_td"></td>
<td id="S3.T3.2.14.12.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.14.12.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.14.12.18" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.15.13" class="ltx_tr">
<td id="S3.T3.2.15.13.1" class="ltx_td ltx_align_left"><span id="S3.T3.2.15.13.1.1" class="ltx_text" style="font-size:50%;">AE &amp; WGAN-GP</span></td>
<td id="S3.T3.2.15.13.2" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.15.13.2.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib78" title="" class="ltx_ref">78</a><span id="S3.T3.2.15.13.2.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.15.13.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.15.13.3.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.15.13.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.15.13.4.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.15.13.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.15.13.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.15.13.6" class="ltx_td"></td>
<td id="S3.T3.2.15.13.7" class="ltx_td"></td>
<td id="S3.T3.2.15.13.8" class="ltx_td"></td>
<td id="S3.T3.2.15.13.9" class="ltx_td"></td>
<td id="S3.T3.2.15.13.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.15.13.10.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.2.15.13.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.15.13.11.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.15.13.12" class="ltx_td"></td>
<td id="S3.T3.2.15.13.13" class="ltx_td"></td>
<td id="S3.T3.2.15.13.14" class="ltx_td"></td>
<td id="S3.T3.2.15.13.15" class="ltx_td"></td>
<td id="S3.T3.2.15.13.16" class="ltx_td ltx_align_center"><span id="S3.T3.2.15.13.16.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.15.13.17" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.16.14" class="ltx_tr">
<td id="S3.T3.2.16.14.1" class="ltx_td ltx_align_left"><span id="S3.T3.2.16.14.1.1" class="ltx_text" style="font-size:50%;">VAE &amp; LSTM GANs</span></td>
<td id="S3.T3.2.16.14.2" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.2.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite></span></td>
<td id="S3.T3.2.16.14.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.3.1" class="ltx_text" style="font-size:50%;">Class</span></td>
<td id="S3.T3.2.16.14.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.4.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.16.14.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.16.14.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.6.1" class="ltx_text ltx_font_bold" style="font-size:50%;">-</span></td>
<td id="S3.T3.2.16.14.7" class="ltx_td"></td>
<td id="S3.T3.2.16.14.8" class="ltx_td"></td>
<td id="S3.T3.2.16.14.9" class="ltx_td"></td>
<td id="S3.T3.2.16.14.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.10.1" class="ltx_text" style="font-size:50%;">S,A</span></td>
<td id="S3.T3.2.16.14.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.11.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.16.14.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.12.1" class="ltx_text" style="font-size:50%;">✓(time)</span></td>
<td id="S3.T3.2.16.14.13" class="ltx_td"></td>
<td id="S3.T3.2.16.14.14" class="ltx_td"></td>
<td id="S3.T3.2.16.14.15" class="ltx_td"></td>
<td id="S3.T3.2.16.14.16" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.16.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.16.14.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.16.14.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.1.1" class="ltx_tr">
<td id="S3.T3.1.1.2" class="ltx_td ltx_align_left">
<table id="S3.T3.1.1.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T3.1.1.2.1.1" class="ltx_tr">
<td id="S3.T3.1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T3.1.1.2.1.1.1.1" class="ltx_text" style="font-size:50%;">Combined Time-GAN</span></td>
</tr>
<tr id="S3.T3.1.1.2.1.2" class="ltx_tr">
<td id="S3.T3.1.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T3.1.1.2.1.2.1.1" class="ltx_text" style="font-size:50%;">&amp; ADS-GAN</span></td>
</tr>
</table>
</td>
<td id="S3.T3.1.1.3" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.1.1.3.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib80" title="" class="ltx_ref">80</a><span id="S3.T3.1.1.3.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.1.1.4" class="ltx_td ltx_align_center"><span id="S3.T3.1.1.4.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.1.1.5" class="ltx_td ltx_align_center"><span id="S3.T3.1.1.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.1.1.6" class="ltx_td"></td>
<td id="S3.T3.1.1.7" class="ltx_td"></td>
<td id="S3.T3.1.1.8" class="ltx_td"></td>
<td id="S3.T3.1.1.9" class="ltx_td"></td>
<td id="S3.T3.1.1.10" class="ltx_td ltx_align_center"><span id="S3.T3.1.1.10.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.1.1.11" class="ltx_td ltx_align_center"><span id="S3.T3.1.1.11.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.1.1.12" class="ltx_td ltx_align_center"><span id="S3.T3.1.1.12.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.1.1.13" class="ltx_td ltx_align_center"><span id="S3.T3.1.1.13.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.1.1.14" class="ltx_td ltx_align_center"><span id="S3.T3.1.1.14.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.1.1.15" class="ltx_td ltx_align_center"><span id="S3.T3.1.1.15.1" class="ltx_text ltx_font_bold" style="font-size:50%;">*</span></td>
<td id="S3.T3.1.1.16" class="ltx_td"></td>
<td id="S3.T3.1.1.1" class="ltx_td ltx_align_center">
<span id="S3.T3.1.1.1.1" class="ltx_text" style="font-size:50%;">✓(</span><math id="S3.T3.1.1.1.m1.1" class="ltx_Math" alttext="\epsilon" display="inline"><semantics id="S3.T3.1.1.1.m1.1a"><mi mathsize="50%" id="S3.T3.1.1.1.m1.1.1" xref="S3.T3.1.1.1.m1.1.1.cmml">ϵ</mi><annotation-xml encoding="MathML-Content" id="S3.T3.1.1.1.m1.1b"><ci id="S3.T3.1.1.1.m1.1.1.cmml" xref="S3.T3.1.1.1.m1.1.1">italic-ϵ</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T3.1.1.1.m1.1c">\epsilon</annotation></semantics></math><span id="S3.T3.1.1.1.2" class="ltx_text" style="font-size:50%;">)</span>
</td>
<td id="S3.T3.1.1.17" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.17.15" class="ltx_tr">
<td id="S3.T3.2.17.15.1" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T3.2.17.15.1.1" class="ltx_text" style="font-size:50%;">DM</span></td>
<td id="S3.T3.2.17.15.2" class="ltx_td ltx_align_left"><span id="S3.T3.2.17.15.2.1" class="ltx_text" style="font-size:50%;">DDPM</span></td>
<td id="S3.T3.2.17.15.3" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.17.15.3.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib57" title="" class="ltx_ref">57</a><span id="S3.T3.2.17.15.3.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.17.15.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.17.15.4.1" class="ltx_text" style="font-size:50%;">RL</span></td>
<td id="S3.T3.2.17.15.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.17.15.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.17.15.6" class="ltx_td"></td>
<td id="S3.T3.2.17.15.7" class="ltx_td"></td>
<td id="S3.T3.2.17.15.8" class="ltx_td"></td>
<td id="S3.T3.2.17.15.9" class="ltx_td"></td>
<td id="S3.T3.2.17.15.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.17.15.10.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.17.15.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.17.15.11.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.2.17.15.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.17.15.12.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.17.15.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.17.15.13.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.17.15.14" class="ltx_td ltx_align_center"><span id="S3.T3.2.17.15.14.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.17.15.15" class="ltx_td"></td>
<td id="S3.T3.2.17.15.16" class="ltx_td"></td>
<td id="S3.T3.2.17.15.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.17.15.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.17.15.18" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.18.16" class="ltx_tr">
<td id="S3.T3.2.18.16.1" class="ltx_td ltx_align_left"><span id="S3.T3.2.18.16.1.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.18.16.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.18.16.1.1.1.1" class="ltx_tr">
<span id="S3.T3.2.18.16.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">mixed sequence</span></span>
<span id="S3.T3.2.18.16.1.1.1.2" class="ltx_tr">
<span id="S3.T3.2.18.16.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">DDPM</span></span>
</span></span></td>
<td id="S3.T3.2.18.16.2" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.2.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite></span></td>
<td id="S3.T3.2.18.16.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.3.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.18.16.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.4.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.18.16.5" class="ltx_td"></td>
<td id="S3.T3.2.18.16.6" class="ltx_td"></td>
<td id="S3.T3.2.18.16.7" class="ltx_td"></td>
<td id="S3.T3.2.18.16.8" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.8.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.18.16.9" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.9.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.18.16.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.10.1" class="ltx_text" style="font-size:50%;">S,A</span></td>
<td id="S3.T3.2.18.16.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.11.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.18.16.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.12.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.18.16.13" class="ltx_td"></td>
<td id="S3.T3.2.18.16.14" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.14.1" class="ltx_text ltx_font_bold" style="font-size:50%;">*</span></td>
<td id="S3.T3.2.18.16.15" class="ltx_td"></td>
<td id="S3.T3.2.18.16.16" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.16.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.18.16.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.18.16.17.1" class="ltx_text" style="font-size:50%;">***</span></td>
</tr>
<tr id="S3.T3.2.19.17" class="ltx_tr">
<td id="S3.T3.2.19.17.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.1.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.19.17.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.19.17.1.1.1.1" class="ltx_tr">
<span id="S3.T3.2.19.17.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Aggregate,</span></span>
<span id="S3.T3.2.19.17.1.1.1.2" class="ltx_tr">
<span id="S3.T3.2.19.17.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Signals,</span></span>
<span id="S3.T3.2.19.17.1.1.1.3" class="ltx_tr">
<span id="S3.T3.2.19.17.1.1.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">&amp; Snapshot</span></span>
</span></span></td>
<td id="S3.T3.2.19.17.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.2.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.19.17.2.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.19.17.2.1.1.1" class="ltx_tr">
<span id="S3.T3.2.19.17.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Hybrid:</span></span>
<span id="S3.T3.2.19.17.2.1.1.2" class="ltx_tr">
<span id="S3.T3.2.19.17.2.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">AE+GAN</span></span>
</span></span></td>
<td id="S3.T3.2.19.17.3" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.3.1" class="ltx_text" style="font-size:50%;">Dis</span></td>
<td id="S3.T3.2.19.17.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T3.2.19.17.4.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.19.17.4.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.19.17.4.1.1.1" class="ltx_tr">
<span id="S3.T3.2.19.17.4.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">convolutional AE</span></span>
<span id="S3.T3.2.19.17.4.1.1.2" class="ltx_tr">
<span id="S3.T3.2.19.17.4.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">&amp; GAN</span></span>
</span></span></td>
<td id="S3.T3.2.19.17.5" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.5.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib81" title="" class="ltx_ref">81</a>]</cite></span></td>
<td id="S3.T3.2.19.17.6" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.6.1" class="ltx_text" style="font-size:50%;">None</span></td>
<td id="S3.T3.2.19.17.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.7.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.19.17.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.8.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.19.17.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.19.17.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.10.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.19.17.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.19.17.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.19.17.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.19.17.14" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.19.17.15" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.15.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.19.17.16" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.19.17.17" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.19.17.18" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.19.17.19" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.19.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓(RDP)</span></td>
<td id="S3.T3.2.19.17.20" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.19.17.20.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.2.20.18" class="ltx_tr">
<td id="S3.T3.2.20.18.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T3.2.20.18.1.1" class="ltx_text" style="font-size:50%;">Aggregate</span></td>
<td id="S3.T3.2.20.18.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.20.18.2.1" class="ltx_text" style="font-size:50%;">GAN</span></td>
<td id="S3.T3.2.20.18.3" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.20.18.3.1" class="ltx_text" style="font-size:50%;">Mix</span></td>
<td id="S3.T3.2.20.18.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T3.2.20.18.4.1" class="ltx_text" style="font-size:50%;">EMR-WGAN based</span></td>
<td id="S3.T3.2.20.18.5" class="ltx_td ltx_align_center ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.20.18.5.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib82" title="" class="ltx_ref">82</a><span id="S3.T3.2.20.18.5.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.20.18.6" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.20.18.6.1" class="ltx_text" style="font-size:50%;">None</span></td>
<td id="S3.T3.2.20.18.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.20.18.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.20.18.8.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.20.18.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.20.18.9.1" class="ltx_text ltx_font_bold" style="font-size:50%;">-</span></td>
<td id="S3.T3.2.20.18.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.20.18.10.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.20.18.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.20.18.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.20.18.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.20.18.14" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.20.18.15" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.20.18.15.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.20.18.16" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.20.18.17" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.20.18.18" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.20.18.19" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.20.18.19.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.20.18.20" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T3.2.21.19" class="ltx_tr">
<td id="S3.T3.2.21.19.1" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T3.2.21.19.1.1" class="ltx_text" style="font-size:50%;">DM</span></td>
<td id="S3.T3.2.21.19.2" class="ltx_td ltx_align_center"><span id="S3.T3.2.21.19.2.1" class="ltx_text" style="font-size:50%;">Mix</span></td>
<td id="S3.T3.2.21.19.3" class="ltx_td ltx_align_left"><span id="S3.T3.2.21.19.3.1" class="ltx_text" style="font-size:50%;">AE followed by SDE</span></td>
<td id="S3.T3.2.21.19.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.21.19.4.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib83" title="" class="ltx_ref">83</a>]</cite></span></td>
<td id="S3.T3.2.21.19.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.21.19.5.1" class="ltx_text" style="font-size:50%;">Class</span></td>
<td id="S3.T3.2.21.19.6" class="ltx_td"></td>
<td id="S3.T3.2.21.19.7" class="ltx_td"></td>
<td id="S3.T3.2.21.19.8" class="ltx_td"></td>
<td id="S3.T3.2.21.19.9" class="ltx_td ltx_align_center"><span id="S3.T3.2.21.19.9.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.21.19.10" class="ltx_td"></td>
<td id="S3.T3.2.21.19.11" class="ltx_td"></td>
<td id="S3.T3.2.21.19.12" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T3.2.21.19.12.1" class="ltx_text ltx_font_bold" style="font-size:50%;">SRA</span></td>
<td id="S3.T3.2.21.19.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.21.19.13.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.21.19.14" class="ltx_td ltx_align_center"><span id="S3.T3.2.21.19.14.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.21.19.15" class="ltx_td ltx_align_center"><span id="S3.T3.2.21.19.15.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.21.19.16" class="ltx_td"></td>
<td id="S3.T3.2.21.19.17" class="ltx_td"></td>
<td id="S3.T3.2.21.19.18" class="ltx_td ltx_align_center"><span id="S3.T3.2.21.19.18.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.2.22.20" class="ltx_tr">
<td id="S3.T3.2.22.20.1" class="ltx_td ltx_align_center"><span id="S3.T3.2.22.20.1.1" class="ltx_text" style="font-size:50%;">Dis</span></td>
<td id="S3.T3.2.22.20.2" class="ltx_td ltx_align_left"><span id="S3.T3.2.22.20.2.1" class="ltx_text" style="font-size:50%;">SDE</span></td>
<td id="S3.T3.2.22.20.3" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.22.20.3.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib8" title="" class="ltx_ref">8</a><span id="S3.T3.2.22.20.3.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.22.20.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.22.20.4.1" class="ltx_text" style="font-size:50%;">Class</span></td>
<td id="S3.T3.2.22.20.5" class="ltx_td"></td>
<td id="S3.T3.2.22.20.6" class="ltx_td"></td>
<td id="S3.T3.2.22.20.7" class="ltx_td"></td>
<td id="S3.T3.2.22.20.8" class="ltx_td ltx_align_center"><span id="S3.T3.2.22.20.8.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.22.20.9" class="ltx_td"></td>
<td id="S3.T3.2.22.20.10" class="ltx_td"></td>
<td id="S3.T3.2.22.20.11" class="ltx_td"></td>
<td id="S3.T3.2.22.20.12" class="ltx_td"></td>
<td id="S3.T3.2.22.20.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.22.20.13.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.22.20.14" class="ltx_td ltx_align_center"><span id="S3.T3.2.22.20.14.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.22.20.15" class="ltx_td"></td>
<td id="S3.T3.2.22.20.16" class="ltx_td"></td>
<td id="S3.T3.2.22.20.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.22.20.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.22.20.18" class="ltx_td ltx_align_center"><span id="S3.T3.2.22.20.18.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.2.23.21" class="ltx_tr">
<td id="S3.T3.2.23.21.1" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T3.2.23.21.1.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.23.21.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.23.21.1.1.1.1" class="ltx_tr">
<span id="S3.T3.2.23.21.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Aggregate,</span></span>
<span id="S3.T3.2.23.21.1.1.1.2" class="ltx_tr">
<span id="S3.T3.2.23.21.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">&amp; Snapshot</span></span>
</span></span></td>
<td id="S3.T3.2.23.21.2" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T3.2.23.21.2.1" class="ltx_text" style="font-size:50%;">DM</span></td>
<td id="S3.T3.2.23.21.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.23.21.3.1" class="ltx_text" style="font-size:50%;">D*,C</span></td>
<td id="S3.T3.2.23.21.4" class="ltx_td ltx_align_left"><span id="S3.T3.2.23.21.4.1" class="ltx_text" style="font-size:50%;">DDPM</span></td>
<td id="S3.T3.2.23.21.5" class="ltx_td ltx_align_center"><span id="S3.T3.2.23.21.5.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib84" title="" class="ltx_ref">84</a>]</cite></span></td>
<td id="S3.T3.2.23.21.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.23.21.6.1" class="ltx_text" style="font-size:50%;">Class</span></td>
<td id="S3.T3.2.23.21.7" class="ltx_td"></td>
<td id="S3.T3.2.23.21.8" class="ltx_td ltx_align_center"><span id="S3.T3.2.23.21.8.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.23.21.9" class="ltx_td"></td>
<td id="S3.T3.2.23.21.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.23.21.10.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.23.21.11" class="ltx_td"></td>
<td id="S3.T3.2.23.21.12" class="ltx_td"></td>
<td id="S3.T3.2.23.21.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.23.21.13.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.2.23.21.14" class="ltx_td ltx_align_center"><span id="S3.T3.2.23.21.14.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.23.21.15" class="ltx_td ltx_align_center"><span id="S3.T3.2.23.21.15.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.23.21.16" class="ltx_td"></td>
<td id="S3.T3.2.23.21.17" class="ltx_td"></td>
<td id="S3.T3.2.23.21.18" class="ltx_td"></td>
<td id="S3.T3.2.23.21.19" class="ltx_td"></td>
<td id="S3.T3.2.23.21.20" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.24.22" class="ltx_tr">
<td id="S3.T3.2.24.22.1" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.1.1" class="ltx_text" style="font-size:50%;">D*,M</span></td>
<td id="S3.T3.2.24.22.2" class="ltx_td ltx_align_left"><span id="S3.T3.2.24.22.2.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.24.22.2.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.24.22.2.1.1.1" class="ltx_tr">
<span id="S3.T3.2.24.22.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">mixed sequence</span></span>
<span id="S3.T3.2.24.22.2.1.1.2" class="ltx_tr">
<span id="S3.T3.2.24.22.2.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">DDPM</span></span>
</span></span></td>
<td id="S3.T3.2.24.22.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.3.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib18" title="" class="ltx_ref">18</a>]</cite></span></td>
<td id="S3.T3.2.24.22.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.4.1" class="ltx_text" style="font-size:50%;">Class</span></td>
<td id="S3.T3.2.24.22.5" class="ltx_td"></td>
<td id="S3.T3.2.24.22.6" class="ltx_td"></td>
<td id="S3.T3.2.24.22.7" class="ltx_td"></td>
<td id="S3.T3.2.24.22.8" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.8.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.24.22.9" class="ltx_td"></td>
<td id="S3.T3.2.24.22.10" class="ltx_td"></td>
<td id="S3.T3.2.24.22.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.11.1" class="ltx_text" style="font-size:50%;">S,A</span></td>
<td id="S3.T3.2.24.22.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.12.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.24.22.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.13.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.24.22.14" class="ltx_td"></td>
<td id="S3.T3.2.24.22.15" class="ltx_td"></td>
<td id="S3.T3.2.24.22.16" class="ltx_td"></td>
<td id="S3.T3.2.24.22.17" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.24.22.18" class="ltx_td ltx_align_center"><span id="S3.T3.2.24.22.18.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
<tr id="S3.T3.2.25.23" class="ltx_tr">
<td id="S3.T3.2.25.23.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="4"><span id="S3.T3.2.25.23.1.1" class="ltx_text" style="font-size:50%;">Snapshot</span></td>
<td id="S3.T3.2.25.23.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="4"><span id="S3.T3.2.25.23.2.1" class="ltx_text" style="font-size:50%;">GAN</span></td>
<td id="S3.T3.2.25.23.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="4"><span id="S3.T3.2.25.23.3.1" class="ltx_text" style="font-size:50%;">Mix</span></td>
<td id="S3.T3.2.25.23.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T3.2.25.23.4.1" class="ltx_text" style="font-size:50%;">WGAN-GP based</span></td>
<td id="S3.T3.2.25.23.5" class="ltx_td ltx_align_center ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.25.23.5.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib54" title="" class="ltx_ref">54</a><span id="S3.T3.2.25.23.5.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.25.23.6" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.25.23.6.1" class="ltx_text" style="font-size:50%;">None</span></td>
<td id="S3.T3.2.25.23.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.25.23.10.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.25.23.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.14" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.15" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.25.23.15.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.25.23.16" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.17" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.25.23.17.1" class="ltx_text ltx_font_bold" style="font-size:50%;">*</span></td>
<td id="S3.T3.2.25.23.18" class="ltx_td ltx_border_t"></td>
<td id="S3.T3.2.25.23.19" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T3.2.25.23.19.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.25.23.20" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T3.2.2" class="ltx_tr">
<td id="S3.T3.2.2.2" class="ltx_td ltx_align_left"><span id="S3.T3.2.2.2.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.2.2.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.2.2.1.1.1" class="ltx_tr">
<span id="S3.T3.2.2.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">ADS-GAN:</span></span>
<span id="S3.T3.2.2.2.1.1.2" class="ltx_tr">
<span id="S3.T3.2.2.2.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">WGAN-GP based</span></span>
</span></span></td>
<td id="S3.T3.2.2.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.2.3.1" class="ltx_text" style="font-size:50%;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite></span></td>
<td id="S3.T3.2.2.4" class="ltx_td ltx_align_center"><span id="S3.T3.2.2.4.1" class="ltx_text" style="font-size:50%;">Pred</span></td>
<td id="S3.T3.2.2.5" class="ltx_td"></td>
<td id="S3.T3.2.2.6" class="ltx_td ltx_align_center"><span id="S3.T3.2.2.6.1" class="ltx_text ltx_font_bold" style="font-size:50%;">*</span></td>
<td id="S3.T3.2.2.7" class="ltx_td"></td>
<td id="S3.T3.2.2.8" class="ltx_td"></td>
<td id="S3.T3.2.2.9" class="ltx_td"></td>
<td id="S3.T3.2.2.10" class="ltx_td"></td>
<td id="S3.T3.2.2.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.2.11.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.2.2.12" class="ltx_td ltx_align_center"><span id="S3.T3.2.2.12.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.2.13" class="ltx_td ltx_align_center"><span id="S3.T3.2.2.13.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.2.14" class="ltx_td"></td>
<td id="S3.T3.2.2.15" class="ltx_td"></td>
<td id="S3.T3.2.2.16" class="ltx_td"></td>
<td id="S3.T3.2.2.1" class="ltx_td ltx_align_center"><span id="S3.T3.2.2.1.1" class="ltx_text ltx_font_bold" style="font-size:50%;">✓(<math id="S3.T3.2.2.1.1.1.m1.1" class="ltx_Math" alttext="\epsilon" display="inline"><semantics id="S3.T3.2.2.1.1.1.m1.1a"><mi id="S3.T3.2.2.1.1.1.m1.1.1" xref="S3.T3.2.2.1.1.1.m1.1.1.cmml">ϵ</mi><annotation-xml encoding="MathML-Content" id="S3.T3.2.2.1.1.1.m1.1b"><ci id="S3.T3.2.2.1.1.1.m1.1.1.cmml" xref="S3.T3.2.2.1.1.1.m1.1.1">italic-ϵ</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T3.2.2.1.1.1.m1.1c">\epsilon</annotation></semantics></math>)</span></td>
<td id="S3.T3.2.2.17" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.26.24" class="ltx_tr">
<td id="S3.T3.2.26.24.1" class="ltx_td ltx_align_left"><span id="S3.T3.2.26.24.1.1" class="ltx_text" style="font-size:50%;">MLP gen &amp; disc</span></td>
<td id="S3.T3.2.26.24.2" class="ltx_td ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T3.2.26.24.2.1.1" class="ltx_text" style="font-size:50%;">[</span><a href="#bib.bib86" title="" class="ltx_ref">86</a><span id="S3.T3.2.26.24.2.2.2" class="ltx_text" style="font-size:50%;">]</span></cite></td>
<td id="S3.T3.2.26.24.3" class="ltx_td ltx_align_center"><span id="S3.T3.2.26.24.3.1" class="ltx_text" style="font-size:50%;">Class,Reg</span></td>
<td id="S3.T3.2.26.24.4" class="ltx_td"></td>
<td id="S3.T3.2.26.24.5" class="ltx_td"></td>
<td id="S3.T3.2.26.24.6" class="ltx_td"></td>
<td id="S3.T3.2.26.24.7" class="ltx_td"></td>
<td id="S3.T3.2.26.24.8" class="ltx_td"></td>
<td id="S3.T3.2.26.24.9" class="ltx_td"></td>
<td id="S3.T3.2.26.24.10" class="ltx_td ltx_align_center"><span id="S3.T3.2.26.24.10.1" class="ltx_text" style="font-size:50%;">S</span></td>
<td id="S3.T3.2.26.24.11" class="ltx_td ltx_align_center"><span id="S3.T3.2.26.24.11.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.26.24.12" class="ltx_td"></td>
<td id="S3.T3.2.26.24.13" class="ltx_td"></td>
<td id="S3.T3.2.26.24.14" class="ltx_td ltx_align_center"><span id="S3.T3.2.26.24.14.1" class="ltx_text ltx_font_bold" style="font-size:50%;">*</span></td>
<td id="S3.T3.2.26.24.15" class="ltx_td"></td>
<td id="S3.T3.2.26.24.16" class="ltx_td ltx_align_center"><span id="S3.T3.2.26.24.16.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.26.24.17" class="ltx_td"></td>
</tr>
<tr id="S3.T3.2.27.25" class="ltx_tr">
<td id="S3.T3.2.27.25.1" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T3.2.27.25.1.1" class="ltx_text" style="font-size:50%;">CTGAN</span></td>
<td id="S3.T3.2.27.25.2" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T3.2.27.25.2.1" class="ltx_text" style="font-size:50%;">
<span id="S3.T3.2.27.25.2.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T3.2.27.25.2.1.1.1" class="ltx_tr">
<span id="S3.T3.2.27.25.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib87" title="" class="ltx_ref">87</a>]</cite>,</span></span>
<span id="S3.T3.2.27.25.2.1.1.2" class="ltx_tr">
<span id="S3.T3.2.27.25.2.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib88" title="" class="ltx_ref">88</a>]</cite>,</span></span>
<span id="S3.T3.2.27.25.2.1.1.3" class="ltx_tr">
<span id="S3.T3.2.27.25.2.1.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib89" title="" class="ltx_ref">89</a>]</cite>,</span></span>
<span id="S3.T3.2.27.25.2.1.1.4" class="ltx_tr">
<span id="S3.T3.2.27.25.2.1.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite>,</span></span>
</span></span></td>
<td id="S3.T3.2.27.25.3" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T3.2.27.25.3.1" class="ltx_text" style="font-size:50%;">Class</span></td>
<td id="S3.T3.2.27.25.4" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.5" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T3.2.27.25.5.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.27.25.6" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.7" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.8" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.9" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.10" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T3.2.27.25.10.1" class="ltx_text" style="font-size:50%;">S,A</span></td>
<td id="S3.T3.2.27.25.11" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T3.2.27.25.11.1" class="ltx_text" style="font-size:50%;">R</span></td>
<td id="S3.T3.2.27.25.12" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T3.2.27.25.12.1" class="ltx_text" style="font-size:50%;">✓</span></td>
<td id="S3.T3.2.27.25.13" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.14" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.15" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.16" class="ltx_td ltx_border_b"></td>
<td id="S3.T3.2.27.25.17" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T3.2.27.25.17.1" class="ltx_text" style="font-size:50%;">✓</span></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:50%;"><span class="ltx_tag ltx_tag_table">Table 3: </span>Overview of all summarized and discussed studies on EHR synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. The table lists the key properties evaluated for generative models of EHRs, including (T) preservation of temporal correlations and dependencies, (C) ability for conditional generation, (R) generation of uncommon diseases or underrepresented sub-populations, (H) capability of generating high-dimensional formats, (M) modeling missing patterns, (J)joint generation of static and temporal features. In the privacy column of the evaluation, the check mark indicates the presence of privacy evaluation, while the text in parenthesis indicate the method that the model follow to preserve privacy. In majority of the cases, the privacy is inherent, meaning that it is assumed in the design of the model. Evaluation is split according to the evaluation metrics reported, covering utility (U), fidelity (F), diversity (D), qualitative assessment (Q), clinical utility (C) and privacy (P) evaluation. In evaluation of utility (U), R stands for real data, S stands for synthetic data, and A stands for augmented data. SRA stands for Synthetic ranking agreement. In evaluation of fidelity, (perp) stands for perplexity measures, whereas (time) stands for time-series metrics. * stands for qualitative evaluation consisting of visualization of some plots only. ** stands for pretraining on synthetic, then fine tuning on real. *** stands for availability of code for original method only. RDP stands for Renyi DP. Dis stands for discrete features. C stands for continuous features. </figcaption>
</figure>
<section id="S3.SS1.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.1.1 </span>Generative models- Longitudinal format </h4>

<div id="S3.SS1.SSS1.p1" class="ltx_para">
<p id="S3.SS1.SSS1.p1.1" class="ltx_p">Table <a href="#S3.T3" title="Table 3 ‣ 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a> summarizes several models that generate synthetic longitudinal EHR data, emphasizing their ability to handle the high dimensionality of medical datasets, mainly discrete features, such as those including ICD codes. These models are capable of preserving temporal correlations across patient visits, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib70" title="" class="ltx_ref">70</a>]</cite> additionally maintaining these correlations at the visit level. Despite their capabilities, none of the models explicitly address the patterns of missing data in their design.</p>
</div>
<div id="S3.SS1.SSS1.p2" class="ltx_para">
<p id="S3.SS1.SSS1.p2.1" class="ltx_p">Beside GANs, VAE, DM, and LM have been utilized to generate longitudinal EHR. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib70" title="" class="ltx_ref">70</a>]</cite> approached the problem using a two-stage strategy. The initial stage sequentially estimates temporal patterns and expected patient state from visits, incorporating a self-attention layer and using RNNs. The subsequent stage generates data, conditioned on the expected patient state, utilizing WGAN and the previous EMR-WGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib91" title="" class="ltx_ref">91</a>]</cite> work. The model in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib58" title="" class="ltx_ref">58</a>]</cite> uses Wasserstein GAN with gradient penalty (WGAN-GP) training with a GRU generator to recursively generate sequences and a sequential discriminator that can simultaneously distinguish whether individual visits are real and whether the entire sequence are real. The model proposed is focused on the generation of uncommon diseases, which is particularly beneficial for downstream tasks involving these less frequent conditions. To generate uncommon diseases, the conditional vector is smoothed into a conditional matrix for all visits to avoid the disease appearing only in the first visit or in every visit due to the characteristics of RNN-based models. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref">71</a>]</cite> used hierarchically factorized conditional VAE to generate discrete EHR sequences specific to medical conditions of interest</p>
</div>
<div id="S3.SS1.SSS1.p3" class="ltx_para">
<p id="S3.SS1.SSS1.p3.1" class="ltx_p">The models in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib73" title="" class="ltx_ref">73</a>]</cite> stand out by following a language model approach, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite> using hierarchical auto-regressive language model, treating EHR data as natural language sequences, predicting the probability of the presence of a potential medical code given the patient’s medical history and the previous codes. This model is uniquely capable of synthesizing both discrete (medical codes) and continuous (average values of lab tests) features, with extremely high dimensional, emulating the heterogeneous character of EHRs. Moreover, it uses demographic and chronic disease phenotypes for conditional generation, which helps create balanced datasets for training downstream tasks on rare conditions.
The model in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib73" title="" class="ltx_ref">73</a>]</cite> allows for prompt-based generation and offers the flexibility to modulate diversity using a variety of methods found in the natural language generation literature. This is in contrast to GANs and VAEs, which rely on sampling random noise vectors to inject diversity, an approach that is uncontrollable and compromises the quality of the generated content.</p>
</div>
<div id="S3.SS1.SSS1.p4" class="ltx_para">
<p id="S3.SS1.SSS1.p4.1" class="ltx_p">Models for synthesizing longitudinal EHR outlined in Table <a href="#S3.T3" title="Table 3 ‣ 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a> are assumed to be designed with inherent privacy-preserving features within their architectural frameworks, and privacy evaluation is performed in most cases. While all models were evaluated for their utility in downstream tasks specifically for training downstream AI models or augmenting the training set, the diffusion model in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib72" title="" class="ltx_ref">72</a>]</cite> lacked fidelity evaluation. For evaluating the fidelity of the data from the language models, metrics adopted from language literature are used such as perplexity measure <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib73" title="" class="ltx_ref">73</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite>. Notably, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib58" title="" class="ltx_ref">58</a>]</cite> evaluates for the diversity of the generated disease types, checking for the presence and representation of all possible diseases and medical codes in the synthetic data.</p>
</div>
</section>
<section id="S3.SS1.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.1.2 </span>Generative models- Time-dependent format </h4>

<div id="S3.SS1.SSS2.p1" class="ltx_para">
<p id="S3.SS1.SSS2.p1.1" class="ltx_p">The capacity to preserve temporal dependencies and correlations among clinical features is a critical functionality for models generating synthetic time dependent EHR data. The challenge of high-dimensional data generation is less pronounced in time-dependent than longitudinal format since the extensive data typically associated with medical codes is not utilized. The WGAN based model <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib75" title="" class="ltx_ref">75</a>]</cite> is the exception, with the ability to generate high-dimensional, time-dependent records featuring 714 attributes. Conversely, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite>, in its initial design, limited the dimensionality of variables to less than 100, potentially constraining its capacity to represent the complexity of medical data fully. Models like the ones in<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib76" title="" class="ltx_ref">76</a>, <a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite> focus as well on accurately reflecting the heterogeneous nature of real-world EHR through a combination of static and temporal features. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib76" title="" class="ltx_ref">76</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>, <a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite> are all proficient in handling the patterns of missing data, a critical aspect that <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib78" title="" class="ltx_ref">78</a>]</cite> opts to eliminate through data imputation, potentially losing valuable information inherent in the absence of data . Iconically, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite> manages varying time-series sequence lengths.</p>
</div>
<div id="S3.SS1.SSS2.p2" class="ltx_para">
<p id="S3.SS1.SSS2.p2.1" class="ltx_p">The idea of combining an autoencoder with a GAN is widely used, mainly to establish a shared latent space representation encompassing mixed types of EHRs features, including both continuous and discrete-valued data, prior to the generation process. For instance, both <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib78" title="" class="ltx_ref">78</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite> employ autoencoder networks followed by a WGAN-GP. While <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib78" title="" class="ltx_ref">78</a>]</cite> utilizes a LSTM based recurrent autoencoder, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite> employs a sequential encoder-decoder network. Expanding upon this strategy, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite> incorporates a dual LSTM-based VAE. It then employs bilaterally coupled LSTM GANs to ensure capturing temporal correlations within diverse EHR features. TIME-ADS-GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib80" title="" class="ltx_ref">80</a>]</cite> combines research on time-dependent synthetic data (TimeGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib92" title="" class="ltx_ref">92</a>]</cite>) and privacy preserving static synthetic data (ADS-GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite>), where the TimeGAN also utilize the latent space of the autoencoder for better synthesis.</p>
</div>
<div id="S3.SS1.SSS2.p3" class="ltx_para">
<p id="S3.SS1.SSS2.p3.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite> are novel approaches that utilize diffusion models to generate mixed feature data. While <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> resorts to Gaussian diffusion for generating discrete sequences, treating them similarly to real-valued sequences but with further post-processing of the model output, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite> on the other hand simultaneously generates (without need for post-processing) time series discrete (using multinomial diffusion <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib19" title="" class="ltx_ref">19</a>]</cite>) and continuous (using Gaussian diffusion) data that preserve temporal dependencies because of the use of a BRNN instead of U-net in the backward diffusion step.</p>
</div>
<div id="S3.SS1.SSS2.p4" class="ltx_para">
<p id="S3.SS1.SSS2.p4.1" class="ltx_p">Multiple models in this category allow for conditional generation, including <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib75" title="" class="ltx_ref">75</a>, <a href="#bib.bib76" title="" class="ltx_ref">76</a>, <a href="#bib.bib78" title="" class="ltx_ref">78</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite> utilizes condition-specific labels (outcomes: ICU mortality, hospital mortality, 30-day re-admission, hospital discharge …) to improve model performance, although its conditional generation capabilities are limited and not extendable to patient-specific EHRs based on more granular level information. In contrast, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib76" title="" class="ltx_ref">76</a>]</cite> excels in this area by enabling conditional generation that considers patient demographic data, effectively addressing the needs of underrepresented sub-populations and contributing to fairness in downstream machine learning applications.</p>
</div>
<div id="S3.SS1.SSS2.p5" class="ltx_para">
<p id="S3.SS1.SSS2.p5.1" class="ltx_p">The privacy of <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite> is empirically evaluated using membership inference attack and differential privacy. Differential privacy analysis is performed by evaluating the performance of the same downstream medical intervention prediction tasks used for utility evaluation but under various differential privacy budgets. Similarly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib75" title="" class="ltx_ref">75</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>, <a href="#bib.bib78" title="" class="ltx_ref">78</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite> are evaluated against various adversary privacy attacks. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib76" title="" class="ltx_ref">76</a>]</cite> indirectly supports privacy by ensuring that synthetic data do not replicate the training data using memorization analysis (latent space nearest neighbor). While all models were evaluated for their utility, only <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> conducted a diversity analysis to evaluate the coverage of all demographic features present in the real dataset as well as uniquely evaluating the utility of their model for the downstream task of reinforcement learning. However, a limitation in the evaluation of this form of time-dependent synthetic EHR arises from the scarcity of time-specific metrics. Instead, the authors opted to assess the temporal aspect of the data by evaluating the utility for temporal tasks.</p>
</div>
</section>
<section id="S3.SS1.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.1.3 </span>Generative models- Aggregated format </h4>

<div id="S3.SS1.SSS3.p1" class="ltx_para">
<p id="S3.SS1.SSS3.p1.1" class="ltx_p">In the context of aggregated format generation for EHR data, all models demonstrate a range of capabilities, particularly in handling high-dimensional data. However, a significant limitation across all these models is their lack of consideration for missing data patterns. Additionally, they do not integrate both static features, like patient demographics, and temporal features, such as medical readings over time, in their generation process. This gap limits their ability to fully replicate the nuanced and comprehensive nature of real-world EHR data. Beyond the primary focus on aggregated format EHR, several models have also shown effectiveness in other EHR formats. For example, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib81" title="" class="ltx_ref">81</a>]</cite> have been effective with EEG and ECG data, demonstrating their ability to preserve temporal correlations and maintain the integrity of time-series patterns. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib84" title="" class="ltx_ref">84</a>]</cite> proved to be effective in generating snapshot format data as well.</p>
</div>
<div id="S3.SS1.SSS3.p2" class="ltx_para">
<p id="S3.SS1.SSS3.p2.1" class="ltx_p">Early works on GANs for EHRs were centered around producing structured, discrete high dimensional aggregated EHRs, including diagnosis and billing ICD codes <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib93" title="" class="ltx_ref">93</a>]</cite>. The initial GAN architecture that generated discrete EHR, medGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib94" title="" class="ltx_ref">94</a>]</cite>, utilized an autoencoder to address the original GAN architecture’s inability to generate discrete features in combination with a mini-batch averaging to mitigate mode collapse. Following the success of medGAN in generating discrete data, medWGAN and medBGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib95" title="" class="ltx_ref">95</a>]</cite> were introduced, replacing the standard GAN with a WGAN-GP <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib25" title="" class="ltx_ref">25</a>]</cite> and boundary-seeking GANs (BGAN) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib96" title="" class="ltx_ref">96</a>]</cite>, respectively. The authors notably contributed to enhancing the quality of the generated data, surpassing that of the original medGAN. In MC-medGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib97" title="" class="ltx_ref">97</a>]</cite>, adaptations to medGAN were proposed to facilitate a superior representation of multi-categorical data . Other endeavors have concentrated on enhancing training stability, such as the approach proposed in Electronic Medical Record Wasserstein GAN (EMR-WGAN) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib91" title="" class="ltx_ref">91</a>]</cite>. This approach bypasses the autoencoder architecture used in medGAN, allowing the generator to directly produce synthetic data. Additionally, the authors implemented a filtering strategy to enhance GAN training specifically for clinical concepts and medical codes with low prevalence.</p>
</div>
<div id="S3.SS1.SSS3.p3" class="ltx_para">
<p id="S3.SS1.SSS3.p3.1" class="ltx_p">More research has been conducted in this direction,expanding the scope to cover not only discrete feature types but heterogeneous type (continuous and discrete) as well. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib81" title="" class="ltx_ref">81</a>]</cite> employed a convolutional AE and convolutional GAN, while the model in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite> is based on EMR-WGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib91" title="" class="ltx_ref">91</a>]</cite> and generate mixed feature types. Recent models utilize diffusion models for generating aggregated format EHR.<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib83" title="" class="ltx_ref">83</a>]</cite> was the first to do so, followed by <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib84" title="" class="ltx_ref">84</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib83" title="" class="ltx_ref">83</a>]</cite> compress the high-dimensional mixed features into a low-dimensional representation using an autoencoder and perform Stochastic Differential Equation (SDE) diffusion in the latent space, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite> apply SDE directly to discrete values, treating them like continuous numbers. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib84" title="" class="ltx_ref">84</a>]</cite>, on the other hand, use a DDPM without requiring a pre-trained autoencoder, utilizing a modified U-net using 1D convolutional layers to accompany the tabular type of data, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib18" title="" class="ltx_ref">18</a>]</cite> adopt the TabDDPM model <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite> and follow a mixed diffusion approach to generate realistic mixed-type tabular EHRs.</p>
</div>
<div id="S3.SS1.SSS3.p4" class="ltx_para">
<p id="S3.SS1.SSS3.p4.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib84" title="" class="ltx_ref">84</a>]</cite> support conditional generation, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite> allowing for conditioning on demographic features such as age and gender, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib84" title="" class="ltx_ref">84</a>]</cite> facilitates class-based conditioning by incorporating the idea of a classifier-guided sampling process <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib98" title="" class="ltx_ref">98</a>]</cite>. Despite these capabilities, these models do not specifically focus on generating data for uncommon diseases or underrepresented conditions, potentially overlooking important aspects of diversity in healthcare.</p>
</div>
<div id="S3.SS1.SSS3.p5" class="ltx_para">
<p id="S3.SS1.SSS3.p5.1" class="ltx_p">Empirical privacy risk analysis against various privacy attacks was performed on <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib83" title="" class="ltx_ref">83</a>]</cite> and demonstrated low privacy disclosure risk. However, there’s room for improvement in the privacy mechanisms of these models, such as the potential incorporation of differential privacy, rather than relying solely on inherent privacy traits of the generative model. RDP-CGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib81" title="" class="ltx_ref">81</a>]</cite> stands out by explicitly implementing Renyi DP <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib99" title="" class="ltx_ref">99</a>]</cite>, a relaxed variant of traditional differential privacy, into the conditional GAN architecture. Meanwhile, the use of TabDDPM in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib18" title="" class="ltx_ref">18</a>]</cite>, despite proving its excellence in data quality, utility, and augmentation, faces challenges when evaluated for privacy, underscoring the often-present trade-off between privacy and utility in synthetic data generation.</p>
</div>
</section>
<section id="S3.SS1.SSS4" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.1.4 </span>Generative models- Snapshot format </h4>

<div id="S3.SS1.SSS4.p1" class="ltx_para">
<p id="S3.SS1.SSS4.p1.1" class="ltx_p">As the snapshot format lack the time aspect, none of the models are designed to preserve temporal correlations within the datasets they handle. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref">54</a>]</cite> stands out as the only model capable of handling high-dimensional data (handling 324 features). Missing data patterns are not addressed by any of the models, nor the joint static and temporal features representation.
The models in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref">54</a>, <a href="#bib.bib85" title="" class="ltx_ref">85</a>, <a href="#bib.bib86" title="" class="ltx_ref">86</a>, <a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite> were all evaluated for privacy, with ADS-GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite> explicitly focusing on preserving privacy by minimizing patient identifiability . They mathematically define <math id="S3.SS1.SSS4.p1.1.m1.1" class="ltx_Math" alttext="\epsilon" display="inline"><semantics id="S3.SS1.SSS4.p1.1.m1.1a"><mi id="S3.SS1.SSS4.p1.1.m1.1.1" xref="S3.SS1.SSS4.p1.1.m1.1.1.cmml">ϵ</mi><annotation-xml encoding="MathML-Content" id="S3.SS1.SSS4.p1.1.m1.1b"><ci id="S3.SS1.SSS4.p1.1.m1.1.1.cmml" xref="S3.SS1.SSS4.p1.1.m1.1.1">italic-ϵ</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.SS1.SSS4.p1.1.m1.1c">\epsilon</annotation></semantics></math>-identifiability based on the probability of re-identification given the combination of all data on any individual patient. ADS-GAN use a modified conditional GAN framework where conditioning variables are not pre-determined but rather optimized from the real patient records, while minimizing patient identifiability. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref">54</a>]</cite> introduced an array of metrics to evaluate realism and privacy together called nearest neighbor adversarial accuracy (NNAA). <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib86" title="" class="ltx_ref">86</a>]</cite> later propose a privacy preserving GAN that uses these metrics to assess the privacy risk.</p>
</div>
<div id="S3.SS1.SSS4.p2" class="ltx_para">
<p id="S3.SS1.SSS4.p2.1" class="ltx_p">Snapshot format data usually have the same properties of tabular data: correlated features, mixed feature type, highly imbalanced categorical features and non-Gaussian distribution of features. These properties impose challenges on the synthetic data generation process. Conditional tabular GANs (CTGANs) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib100" title="" class="ltx_ref">100</a>]</cite> have been recognized for their effectiveness in generating tabular data, mitigating the challenges and difficulties of this task, and have been adapted to ensure privacy, while allowing for conditional generation. These models have been widely used in snapshot EHR format synthesis.The introduction of DP-CTGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib87" title="" class="ltx_ref">87</a>]</cite> offers a differentially private variant that incorporates random perturbation into the GAN’s critic for privacy protection. The federated version, FDP-CTGAN, extends this approach to a distributed data setting, evaluated across multiple medical datasets. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib88" title="" class="ltx_ref">88</a>]</cite> uses CTGANs for data augmentation purposes (with varying sizes of augmentation) on classification problems. However, the authors of the paper resort to other data balancing methods like SMOTE and ADASYN instead of using the CTGAN for data balancing as well. The authors of <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib89" title="" class="ltx_ref">89</a>]</cite>, on the other hand, use CTGAN as well to address the issue of poor performance of underrepresented sub-populations. They train CTGAN for each sub-population separately and incorporate synthetic samples into the training set of each, consequently increasing the generalizability of predictive models towards underrepresented sub-populations. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite> utilizes synthetic data generated using CTGAN as a proxy for scarce real-world EHR for the patient length-of-stay multi-class classification task.</p>
</div>
</section>
<section id="S3.SS1.SSS5" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.1.5 </span>Diverse papers from diverse formats</h4>

<div id="S3.SS1.SSS5.p1" class="ltx_para">
<p id="S3.SS1.SSS5.p1.1" class="ltx_p">Following the success of the HealthGAN model in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib75" title="" class="ltx_ref">75</a>, <a href="#bib.bib54" title="" class="ltx_ref">54</a>]</cite>, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib101" title="" class="ltx_ref">101</a>]</cite> extends the utility, privacy, and fidelity evaluation to include fairness evaluation as well. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib102" title="" class="ltx_ref">102</a>]</cite> implemented federated learning to train separate GANs locally at each organization, and then combining them into a central GAN that can generate synthetic discrete EHR, such as ICD-9 codes. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib103" title="" class="ltx_ref">103</a>]</cite> evaluates four different GAN architectures: Tabular GAN (TGAN), CTGAN, WGAN-GP, and conditional TableGAN (CTABGAN) on four tabular medical datasets, three of which are in the snapshot format, and one is an EEG dataset. The aim of this study is to investigate the applicability of tabular GANs to medical data and analyze whether the GAN architectures specific to tabular data outperform the more general WGAN-GP by evaluation of model quality, fidelity and privacy.</p>
</div>
</section>
</section>
<section id="S3.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.2 </span>Time-series sgnals</h3>

<div id="S3.SS2.p1" class="ltx_para">
<p id="S3.SS2.p1.1" class="ltx_p">In this section, we explore the generation of synthetic data for physiological signals, such as ECG and EEG. These signals are crucial for monitoring patient health. Our investigation delves into the synthesis techniques and evaluations frameworks specific to these type of signals, addressing challenges and opportunities in generating realistic physiological signals.</p>
</div>
<section id="S3.SS2.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.2.1 </span>ECGs</h4>

<figure id="S3.T4" class="ltx_table">
<table id="S3.T4.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T4.1.1.1" class="ltx_tr">
<td id="S3.T4.1.1.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T4.1.1.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T4.1.1.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T4.1.1.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T4.1.1.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T4.1.1.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T4.1.1.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T4.1.1.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T4.1.1.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T4.1.1.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T4.1.1.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T4.1.1.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T4.1.1.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T4.1.1.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T4.1.1.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T4.1.1.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T4.1.2.2" class="ltx_tr">
<td id="S3.T4.1.2.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T4.1.2.2.1.1" class="ltx_text" style="font-size:70%;">U</span></td>
<td id="S3.T4.1.2.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.2.2.2.1" class="ltx_text" style="font-size:70%;">F</span></td>
<td id="S3.T4.1.2.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.2.2.3.1" class="ltx_text" style="font-size:70%;">D</span></td>
<td id="S3.T4.1.2.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.2.2.4.1" class="ltx_text" style="font-size:70%;">Q</span></td>
<td id="S3.T4.1.2.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.2.2.5.1" class="ltx_text" style="font-size:70%;">C</span></td>
<td id="S3.T4.1.2.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.2.2.6.1" class="ltx_text" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T4.1.3.3" class="ltx_tr">
<td id="S3.T4.1.3.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.3.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T4.1.3.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.3.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T4.1.4.4" class="ltx_tr">
<td id="S3.T4.1.4.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="8"><span id="S3.T4.1.4.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T4.1.4.4.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.4.4.2.1" class="ltx_text" style="font-size:70%;">Other</span></td>
<td id="S3.T4.1.4.4.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.4.4.3.1" class="ltx_text" style="font-size:70%;">12-lead</span></td>
<td id="S3.T4.1.4.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.4.4.4.1" class="ltx_text" style="font-size:70%;">Bi-LSTM and CNN</span></td>
<td id="S3.T4.1.4.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.4.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib104" title="" class="ltx_ref">104</a><span id="S3.T4.1.4.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.4.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.4.4.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.4.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.4.4.7.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T4.1.4.4.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.4.4.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.4.4.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.4.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.4.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.4.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.4.4.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.4.4.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T4.1.5.5" class="ltx_tr">
<td id="S3.T4.1.5.5.1" class="ltx_td ltx_align_center" rowspan="6"><span id="S3.T4.1.5.5.1.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T4.1.5.5.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.5.5.2.1" class="ltx_text" style="font-size:70%;">single-lead</span></td>
<td id="S3.T4.1.5.5.3" class="ltx_td ltx_align_left">
<table id="S3.T4.1.5.5.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T4.1.5.5.3.1.1" class="ltx_tr">
<td id="S3.T4.1.5.5.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.5.5.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Composite GAN:</span></td>
</tr>
<tr id="S3.T4.1.5.5.3.1.2" class="ltx_tr">
<td id="S3.T4.1.5.5.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.5.5.3.1.2.1.1" class="ltx_text" style="font-size:70%;">LSTM-GAN and DCGAN</span></td>
</tr>
</table>
</td>
<td id="S3.T4.1.5.5.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.5.5.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib105" title="" class="ltx_ref">105</a><span id="S3.T4.1.5.5.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.5.5.5" class="ltx_td ltx_align_left"><span id="S3.T4.1.5.5.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.5.5.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.5.5.6.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T4.1.5.5.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.5.5.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.5.5.8" class="ltx_td"></td>
<td id="S3.T4.1.5.5.9" class="ltx_td"></td>
<td id="S3.T4.1.5.5.10" class="ltx_td"></td>
<td id="S3.T4.1.5.5.11" class="ltx_td"></td>
<td id="S3.T4.1.5.5.12" class="ltx_td"></td>
<td id="S3.T4.1.5.5.13" class="ltx_td"></td>
</tr>
<tr id="S3.T4.1.6.6" class="ltx_tr">
<td id="S3.T4.1.6.6.1" class="ltx_td ltx_align_left"><span id="S3.T4.1.6.6.1.1" class="ltx_text" style="font-size:70%;">single and 2-lead</span></td>
<td id="S3.T4.1.6.6.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.6.6.2.1" class="ltx_text" style="font-size:70%;">LSTM-based:TS-GAN</span></td>
<td id="S3.T4.1.6.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.6.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib106" title="" class="ltx_ref">106</a><span id="S3.T4.1.6.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.6.6.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.6.6.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.6.6.5" class="ltx_td ltx_align_center"><span id="S3.T4.1.6.6.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T4.1.6.6.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.6.6.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.6.6.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.6.6.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.6.6.8" class="ltx_td"></td>
<td id="S3.T4.1.6.6.9" class="ltx_td ltx_align_center"><span id="S3.T4.1.6.6.9.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T4.1.6.6.10" class="ltx_td"></td>
<td id="S3.T4.1.6.6.11" class="ltx_td"></td>
<td id="S3.T4.1.6.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T4.1.7.7" class="ltx_tr">
<td id="S3.T4.1.7.7.1" class="ltx_td ltx_align_left"><span id="S3.T4.1.7.7.1.1" class="ltx_text" style="font-size:70%;">12-lead</span></td>
<td id="S3.T4.1.7.7.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.7.7.2.1" class="ltx_text" style="font-size:70%;">LSTM-based:BiLSTM</span></td>
<td id="S3.T4.1.7.7.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.7.7.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib107" title="" class="ltx_ref">107</a><span id="S3.T4.1.7.7.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.7.7.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.7.7.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T4.1.7.7.5" class="ltx_td"></td>
<td id="S3.T4.1.7.7.6" class="ltx_td"></td>
<td id="S3.T4.1.7.7.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.7.7.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.7.7.8" class="ltx_td"></td>
<td id="S3.T4.1.7.7.9" class="ltx_td"></td>
<td id="S3.T4.1.7.7.10" class="ltx_td"></td>
<td id="S3.T4.1.7.7.11" class="ltx_td"></td>
<td id="S3.T4.1.7.7.12" class="ltx_td"></td>
</tr>
<tr id="S3.T4.1.8.8" class="ltx_tr">
<td id="S3.T4.1.8.8.1" class="ltx_td ltx_align_left"><span id="S3.T4.1.8.8.1.1" class="ltx_text" style="font-size:70%;">single-lead</span></td>
<td id="S3.T4.1.8.8.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.8.8.2.1" class="ltx_text" style="font-size:70%;">Multiple GANs</span></td>
<td id="S3.T4.1.8.8.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.8.8.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib93" title="" class="ltx_ref">93</a><span id="S3.T4.1.8.8.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.8.8.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.8.8.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.8.8.5" class="ltx_td ltx_align_center"><span id="S3.T4.1.8.8.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T4.1.8.8.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.8.8.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.8.8.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.8.8.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.8.8.8" class="ltx_td"></td>
<td id="S3.T4.1.8.8.9" class="ltx_td ltx_align_center"><span id="S3.T4.1.8.8.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.8.8.10" class="ltx_td ltx_align_center"><span id="S3.T4.1.8.8.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.8.8.11" class="ltx_td"></td>
<td id="S3.T4.1.8.8.12" class="ltx_td ltx_align_center"><span id="S3.T4.1.8.8.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.9.9" class="ltx_tr">
<td id="S3.T4.1.9.9.1" class="ltx_td ltx_align_left"><span id="S3.T4.1.9.9.1.1" class="ltx_text" style="font-size:70%;">short (10s) 12-lead</span></td>
<td id="S3.T4.1.9.9.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.9.9.2.1" class="ltx_text" style="font-size:70%;">WaveGAN,Pulse2Pulse</span></td>
<td id="S3.T4.1.9.9.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.9.9.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib108" title="" class="ltx_ref">108</a><span id="S3.T4.1.9.9.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.9.9.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.9.9.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T4.1.9.9.5" class="ltx_td"></td>
<td id="S3.T4.1.9.9.6" class="ltx_td"></td>
<td id="S3.T4.1.9.9.7" class="ltx_td"></td>
<td id="S3.T4.1.9.9.8" class="ltx_td"></td>
<td id="S3.T4.1.9.9.9" class="ltx_td"></td>
<td id="S3.T4.1.9.9.10" class="ltx_td"></td>
<td id="S3.T4.1.9.9.11" class="ltx_td"></td>
<td id="S3.T4.1.9.9.12" class="ltx_td ltx_align_center"><span id="S3.T4.1.9.9.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.10.10" class="ltx_tr">
<td id="S3.T4.1.10.10.1" class="ltx_td ltx_align_left"><span id="S3.T4.1.10.10.1.1" class="ltx_text" style="font-size:70%;">single-lead</span></td>
<td id="S3.T4.1.10.10.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.10.10.2.1" class="ltx_text" style="font-size:70%;">image-based:TTS-GAN</span></td>
<td id="S3.T4.1.10.10.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.10.10.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib109" title="" class="ltx_ref">109</a><span id="S3.T4.1.10.10.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.10.10.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.10.10.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T4.1.10.10.5" class="ltx_td"></td>
<td id="S3.T4.1.10.10.6" class="ltx_td"></td>
<td id="S3.T4.1.10.10.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.10.10.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.10.10.8" class="ltx_td"></td>
<td id="S3.T4.1.10.10.9" class="ltx_td ltx_align_center"><span id="S3.T4.1.10.10.9.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T4.1.10.10.10" class="ltx_td"></td>
<td id="S3.T4.1.10.10.11" class="ltx_td"></td>
<td id="S3.T4.1.10.10.12" class="ltx_td ltx_align_center"><span id="S3.T4.1.10.10.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.11.11" class="ltx_tr">
<td id="S3.T4.1.11.11.1" class="ltx_td ltx_align_center"><span id="S3.T4.1.11.11.1.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T4.1.11.11.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.11.11.2.1" class="ltx_text" style="font-size:70%;">2-lead</span></td>
<td id="S3.T4.1.11.11.3" class="ltx_td ltx_align_left"><span id="S3.T4.1.11.11.3.1" class="ltx_text" style="font-size:70%;">image-based:DDPM</span></td>
<td id="S3.T4.1.11.11.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.11.11.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib110" title="" class="ltx_ref">110</a><span id="S3.T4.1.11.11.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.11.11.5" class="ltx_td ltx_align_left"><span id="S3.T4.1.11.11.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.11.11.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.11.11.6.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T4.1.11.11.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.11.11.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.11.11.8" class="ltx_td ltx_align_center"><span id="S3.T4.1.11.11.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.11.11.9" class="ltx_td"></td>
<td id="S3.T4.1.11.11.10" class="ltx_td"></td>
<td id="S3.T4.1.11.11.11" class="ltx_td"></td>
<td id="S3.T4.1.11.11.12" class="ltx_td"></td>
<td id="S3.T4.1.11.11.13" class="ltx_td ltx_align_center"><span id="S3.T4.1.11.11.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.12.12" class="ltx_tr">
<td id="S3.T4.1.12.12.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T4.1.12.12.1.1" class="ltx_text" style="font-size:70%;">Class or Label Cond.</span></td>
<td id="S3.T4.1.12.12.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.12.12.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T4.1.12.12.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.12.12.3.1" class="ltx_text" style="font-size:70%;">12-lead</span></td>
<td id="S3.T4.1.12.12.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.12.12.4.1" class="ltx_text" style="font-size:70%;">DSAT-ECG</span></td>
<td id="S3.T4.1.12.12.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.12.12.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib111" title="" class="ltx_ref">111</a><span id="S3.T4.1.12.12.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.12.12.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.12.12.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.12.12.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.12.12.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.12.12.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.12.12.8.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T4.1.12.12.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.12.12.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.12.12.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.12.12.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.12.12.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.12.12.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.12.12.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T4.1.13.13" class="ltx_tr">
<td id="S3.T4.1.13.13.1" class="ltx_td ltx_align_left"><span id="S3.T4.1.13.13.1.1" class="ltx_text" style="font-size:70%;">2-lead</span></td>
<td id="S3.T4.1.13.13.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.13.13.2.1" class="ltx_text" style="font-size:70%;">DDPM-based:DiffECG</span></td>
<td id="S3.T4.1.13.13.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.13.13.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib112" title="" class="ltx_ref">112</a><span id="S3.T4.1.13.13.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.13.13.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.13.13.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.13.13.5" class="ltx_td ltx_align_center"><span id="S3.T4.1.13.13.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T4.1.13.13.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.13.13.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.13.13.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.13.13.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.13.13.8" class="ltx_td"></td>
<td id="S3.T4.1.13.13.9" class="ltx_td ltx_align_center"><span id="S3.T4.1.13.13.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.13.13.10" class="ltx_td"></td>
<td id="S3.T4.1.13.13.11" class="ltx_td"></td>
<td id="S3.T4.1.13.13.12" class="ltx_td"></td>
</tr>
<tr id="S3.T4.1.14.14" class="ltx_tr">
<td id="S3.T4.1.14.14.1" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T4.1.14.14.1.1" class="ltx_text" style="font-size:70%;">GAN,VAE</span></td>
<td id="S3.T4.1.14.14.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.14.14.2.1" class="ltx_text" style="font-size:70%;">single-lead</span></td>
<td id="S3.T4.1.14.14.3" class="ltx_td ltx_align_left"><span id="S3.T4.1.14.14.3.1" class="ltx_text" style="font-size:70%;">CVAE,CWGAN</span></td>
<td id="S3.T4.1.14.14.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.14.14.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib113" title="" class="ltx_ref">113</a><span id="S3.T4.1.14.14.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.14.14.5" class="ltx_td ltx_align_left"><span id="S3.T4.1.14.14.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.14.14.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.14.14.6.1" class="ltx_text" style="font-size:70%;">S,R</span></td>
<td id="S3.T4.1.14.14.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.14.14.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T4.1.14.14.8" class="ltx_td"></td>
<td id="S3.T4.1.14.14.9" class="ltx_td"></td>
<td id="S3.T4.1.14.14.10" class="ltx_td"></td>
<td id="S3.T4.1.14.14.11" class="ltx_td"></td>
<td id="S3.T4.1.14.14.12" class="ltx_td"></td>
<td id="S3.T4.1.14.14.13" class="ltx_td"></td>
</tr>
<tr id="S3.T4.1.15.15" class="ltx_tr">
<td id="S3.T4.1.15.15.1" class="ltx_td"></td>
<td id="S3.T4.1.15.15.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.15.15.2.1" class="ltx_text" style="font-size:70%;">PHYSIOGAN</span></td>
<td id="S3.T4.1.15.15.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.15.15.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib59" title="" class="ltx_ref">59</a><span id="S3.T4.1.15.15.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.15.15.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.15.15.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.15.15.5" class="ltx_td ltx_align_center"><span id="S3.T4.1.15.15.5.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T4.1.15.15.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.15.15.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.15.15.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.15.15.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.15.15.8" class="ltx_td ltx_align_center"><span id="S3.T4.1.15.15.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.15.15.9" class="ltx_td"></td>
<td id="S3.T4.1.15.15.10" class="ltx_td"></td>
<td id="S3.T4.1.15.15.11" class="ltx_td"></td>
<td id="S3.T4.1.15.15.12" class="ltx_td ltx_align_center"><span id="S3.T4.1.15.15.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.16.16" class="ltx_tr">
<td id="S3.T4.1.16.16.1" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T4.1.16.16.1.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T4.1.16.16.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.16.16.2.1" class="ltx_text" style="font-size:70%;">single-lead</span></td>
<td id="S3.T4.1.16.16.3" class="ltx_td ltx_align_left">
<table id="S3.T4.1.16.16.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T4.1.16.16.3.1.1" class="ltx_tr">
<td id="S3.T4.1.16.16.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.16.16.3.1.1.1.1" class="ltx_text" style="font-size:70%;">WGAN-GP-based:</span></td>
</tr>
<tr id="S3.T4.1.16.16.3.1.2" class="ltx_tr">
<td id="S3.T4.1.16.16.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.16.16.3.1.2.1.1" class="ltx_text" style="font-size:70%;">AC-WGAN-GP</span></td>
</tr>
</table>
</td>
<td id="S3.T4.1.16.16.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.16.16.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib114" title="" class="ltx_ref">114</a><span id="S3.T4.1.16.16.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.16.16.5" class="ltx_td ltx_align_left"><span id="S3.T4.1.16.16.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.16.16.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.16.16.6.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T4.1.16.16.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.16.16.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.16.16.8" class="ltx_td"></td>
<td id="S3.T4.1.16.16.9" class="ltx_td"></td>
<td id="S3.T4.1.16.16.10" class="ltx_td"></td>
<td id="S3.T4.1.16.16.11" class="ltx_td"></td>
<td id="S3.T4.1.16.16.12" class="ltx_td"></td>
<td id="S3.T4.1.16.16.13" class="ltx_td ltx_align_center"><span id="S3.T4.1.16.16.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.17.17" class="ltx_tr">
<td id="S3.T4.1.17.17.1" class="ltx_td ltx_align_left"><span id="S3.T4.1.17.17.1.1" class="ltx_text" style="font-size:70%;">beat generation</span></td>
<td id="S3.T4.1.17.17.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.17.17.2.1" class="ltx_text" style="font-size:70%;">DCCGAN</span></td>
<td id="S3.T4.1.17.17.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.17.17.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib115" title="" class="ltx_ref">115</a><span id="S3.T4.1.17.17.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.17.17.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.17.17.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.17.17.5" class="ltx_td ltx_align_center"><span id="S3.T4.1.17.17.5.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T4.1.17.17.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.17.17.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.17.17.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.17.17.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.17.17.8" class="ltx_td"></td>
<td id="S3.T4.1.17.17.9" class="ltx_td ltx_align_center"><span id="S3.T4.1.17.17.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.17.17.10" class="ltx_td"></td>
<td id="S3.T4.1.17.17.11" class="ltx_td"></td>
<td id="S3.T4.1.17.17.12" class="ltx_td"></td>
</tr>
<tr id="S3.T4.1.18.18" class="ltx_tr">
<td id="S3.T4.1.18.18.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.18.18.1.1" class="ltx_text" style="font-size:70%;">Attribute Cond</span></td>
<td id="S3.T4.1.18.18.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.18.18.2.1" class="ltx_text" style="font-size:70%;">VAE</span></td>
<td id="S3.T4.1.18.18.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.18.18.3.1" class="ltx_text" style="font-size:70%;">12-lead</span></td>
<td id="S3.T4.1.18.18.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.18.18.4.1" class="ltx_text" style="font-size:70%;">cVAE</span></td>
<td id="S3.T4.1.18.18.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.18.18.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib116" title="" class="ltx_ref">116</a><span id="S3.T4.1.18.18.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.18.18.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.18.18.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T4.1.18.18.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.18.18.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.18.18.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.18.18.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.18.18.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.18.18.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.18.18.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.18.18.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.18.18.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T4.1.19.19" class="ltx_tr">
<td id="S3.T4.1.19.19.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.19.19.1.1" class="ltx_text" style="font-size:70%;">Text-guided</span></td>
<td id="S3.T4.1.19.19.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.19.19.2.1" class="ltx_text" style="font-size:70%;">VAE,GAN</span></td>
<td id="S3.T4.1.19.19.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.19.19.3.1" class="ltx_text" style="font-size:70%;">12-lead</span></td>
<td id="S3.T4.1.19.19.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.19.19.4.1" class="ltx_text" style="font-size:70%;">Auto-TTE</span></td>
<td id="S3.T4.1.19.19.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.19.19.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib117" title="" class="ltx_ref">117</a><span id="S3.T4.1.19.19.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.19.19.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.19.19.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.19.19.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.19.19.7.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T4.1.19.19.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.19.19.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.19.19.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.19.19.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.19.19.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.19.19.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.19.19.11" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.19.19.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.19.19.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.19.19.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.19.19.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.19.19.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.20.20" class="ltx_tr">
<td id="S3.T4.1.20.20.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.20.20.1.1" class="ltx_text" style="font-size:70%;">Intra-modal trans.</span></td>
<td id="S3.T4.1.20.20.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.20.20.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T4.1.20.20.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.20.20.3.1" class="ltx_text" style="font-size:70%;">single-lead to 12-lead</span></td>
<td id="S3.T4.1.20.20.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.20.20.4.1" class="ltx_text" style="font-size:70%;">Bi-LSTM and CNN</span></td>
<td id="S3.T4.1.20.20.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.20.20.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib118" title="" class="ltx_ref">118</a><span id="S3.T4.1.20.20.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.20.20.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.20.20.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T4.1.20.20.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.20.20.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.20.20.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.20.20.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.20.20.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.20.20.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.20.20.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.20.20.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.20.20.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.20.20.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.21.21" class="ltx_tr">
<td id="S3.T4.1.21.21.1" class="ltx_td ltx_align_left"><span id="S3.T4.1.21.21.1.1" class="ltx_text" style="font-size:70%;">2-lead to 12 LEAD</span></td>
<td id="S3.T4.1.21.21.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.21.21.2.1" class="ltx_text" style="font-size:70%;">StarGAN v2</span></td>
<td id="S3.T4.1.21.21.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.21.21.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib119" title="" class="ltx_ref">119</a><span id="S3.T4.1.21.21.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.21.21.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.21.21.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.21.21.5" class="ltx_td ltx_align_center"><span id="S3.T4.1.21.21.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T4.1.21.21.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.21.21.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.21.21.7" class="ltx_td"></td>
<td id="S3.T4.1.21.21.8" class="ltx_td"></td>
<td id="S3.T4.1.21.21.9" class="ltx_td ltx_align_center"><span id="S3.T4.1.21.21.9.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T4.1.21.21.10" class="ltx_td"></td>
<td id="S3.T4.1.21.21.11" class="ltx_td"></td>
<td id="S3.T4.1.21.21.12" class="ltx_td ltx_align_center"><span id="S3.T4.1.21.21.12.1" class="ltx_text" style="font-size:70%;">**</span></td>
</tr>
<tr id="S3.T4.1.22.22" class="ltx_tr">
<td id="S3.T4.1.22.22.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.22.22.1.1" class="ltx_text" style="font-size:70%;">Inter-modal trans.</span></td>
<td id="S3.T4.1.22.22.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.22.22.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T4.1.22.22.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.22.22.3.1" class="ltx_text" style="font-size:70%;">PPG-to-ECG</span></td>
<td id="S3.T4.1.22.22.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.22.22.4.1" class="ltx_text" style="font-size:70%;">RDDM</span></td>
<td id="S3.T4.1.22.22.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.22.22.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib120" title="" class="ltx_ref">120</a><span id="S3.T4.1.22.22.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.22.22.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.22.22.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.22.22.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.22.22.7.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T4.1.22.22.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.22.22.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.22.22.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.22.22.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.22.22.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.22.22.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.22.22.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.22.22.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.22.22.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.22.22.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.22.22.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.23.23" class="ltx_tr">
<td id="S3.T4.1.23.23.1" class="ltx_td ltx_align_center"><span id="S3.T4.1.23.23.1.1" class="ltx_text" style="font-size:70%;">GAN,AE</span></td>
<td id="S3.T4.1.23.23.2" class="ltx_td ltx_align_left">
<table id="S3.T4.1.23.23.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T4.1.23.23.2.1.1" class="ltx_tr">
<td id="S3.T4.1.23.23.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.23.23.2.1.1.1.1" class="ltx_text" style="font-size:70%;">PCG to ECG,</span></td>
</tr>
<tr id="S3.T4.1.23.23.2.1.2" class="ltx_tr">
<td id="S3.T4.1.23.23.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.23.23.2.1.2.1.1" class="ltx_text" style="font-size:70%;">VCG to 12-lead transfer</span></td>
</tr>
</table>
</td>
<td id="S3.T4.1.23.23.3" class="ltx_td ltx_align_left">
<table id="S3.T4.1.23.23.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T4.1.23.23.3.1.1" class="ltx_tr">
<td id="S3.T4.1.23.23.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.23.23.3.1.1.1.1" class="ltx_text" style="font-size:70%;">classical GAN,adversarial AE,</span></td>
</tr>
<tr id="S3.T4.1.23.23.3.1.2" class="ltx_tr">
<td id="S3.T4.1.23.23.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.23.23.3.1.2.1.1" class="ltx_text" style="font-size:70%;">modality transfer GAN</span></td>
</tr>
</table>
</td>
<td id="S3.T4.1.23.23.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.23.23.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib121" title="" class="ltx_ref">121</a><span id="S3.T4.1.23.23.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.23.23.5" class="ltx_td ltx_align_left"><span id="S3.T4.1.23.23.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.23.23.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.23.23.6.1" class="ltx_text" style="font-size:70%;">S,R</span></td>
<td id="S3.T4.1.23.23.7" class="ltx_td ltx_align_center"><span id="S3.T4.1.23.23.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T4.1.23.23.8" class="ltx_td ltx_align_center"><span id="S3.T4.1.23.23.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.23.23.9" class="ltx_td"></td>
<td id="S3.T4.1.23.23.10" class="ltx_td ltx_align_center"><span id="S3.T4.1.23.23.10.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T4.1.23.23.11" class="ltx_td"></td>
<td id="S3.T4.1.23.23.12" class="ltx_td"></td>
<td id="S3.T4.1.23.23.13" class="ltx_td"></td>
</tr>
<tr id="S3.T4.1.24.24" class="ltx_tr">
<td id="S3.T4.1.24.24.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="3"><span id="S3.T4.1.24.24.1.1" class="ltx_text" style="font-size:70%;">Other Conditioning</span></td>
<td id="S3.T4.1.24.24.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T4.1.24.24.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T4.1.24.24.3" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T4.1.24.24.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T4.1.24.24.3.1.1" class="ltx_tr">
<td id="S3.T4.1.24.24.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.24.24.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Conditioned on</span></td>
</tr>
<tr id="S3.T4.1.24.24.3.1.2" class="ltx_tr">
<td id="S3.T4.1.24.24.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.24.24.3.1.2.1.1" class="ltx_text" style="font-size:70%;">clinical Knowledge</span></td>
</tr>
</table>
</td>
<td id="S3.T4.1.24.24.4" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T4.1.24.24.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T4.1.24.24.4.1.1" class="ltx_tr">
<td id="S3.T4.1.24.24.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.24.24.4.1.1.1.1" class="ltx_text" style="font-size:70%;">WGAN-GP-based:</span></td>
</tr>
<tr id="S3.T4.1.24.24.4.1.2" class="ltx_tr">
<td id="S3.T4.1.24.24.4.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.24.24.4.1.2.1.1" class="ltx_text" style="font-size:70%;">CardiacGen</span></td>
</tr>
</table>
</td>
<td id="S3.T4.1.24.24.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.24.24.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib122" title="" class="ltx_ref">122</a><span id="S3.T4.1.24.24.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.24.24.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T4.1.24.24.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.24.24.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.24.24.7.1" class="ltx_text" style="font-size:70%;">S,A</span></td>
<td id="S3.T4.1.24.24.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.24.24.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.24.24.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.24.24.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.24.24.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.24.24.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.24.24.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.24.24.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.24.24.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T4.1.24.24.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T4.1.24.24.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T4.1.25.25" class="ltx_tr">
<td id="S3.T4.1.25.25.1" class="ltx_td ltx_align_left">
<table id="S3.T4.1.25.25.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T4.1.25.25.1.1.1" class="ltx_tr">
<td id="S3.T4.1.25.25.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.25.25.1.1.1.1.1" class="ltx_text" style="font-size:70%;">Conditioned on other</span></td>
</tr>
<tr id="S3.T4.1.25.25.1.1.2" class="ltx_tr">
<td id="S3.T4.1.25.25.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.25.25.1.1.2.1.1" class="ltx_text" style="font-size:70%;">ECG statements</span></td>
</tr>
</table>
</td>
<td id="S3.T4.1.25.25.2" class="ltx_td ltx_align_left"><span id="S3.T4.1.25.25.2.1" class="ltx_text" style="font-size:70%;">Conditional GAN</span></td>
<td id="S3.T4.1.25.25.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.25.25.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib123" title="" class="ltx_ref">123</a><span id="S3.T4.1.25.25.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.25.25.4" class="ltx_td ltx_align_left"><span id="S3.T4.1.25.25.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.25.25.5" class="ltx_td ltx_align_center"><span id="S3.T4.1.25.25.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T4.1.25.25.6" class="ltx_td ltx_align_center"><span id="S3.T4.1.25.25.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T4.1.25.25.7" class="ltx_td"></td>
<td id="S3.T4.1.25.25.8" class="ltx_td"></td>
<td id="S3.T4.1.25.25.9" class="ltx_td ltx_align_center"><span id="S3.T4.1.25.25.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.25.25.10" class="ltx_td"></td>
<td id="S3.T4.1.25.25.11" class="ltx_td"></td>
<td id="S3.T4.1.25.25.12" class="ltx_td"></td>
</tr>
<tr id="S3.T4.1.26.26" class="ltx_tr">
<td id="S3.T4.1.26.26.1" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T4.1.26.26.1.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T4.1.26.26.2" class="ltx_td ltx_align_left ltx_border_b">
<table id="S3.T4.1.26.26.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T4.1.26.26.2.1.1" class="ltx_tr">
<td id="S3.T4.1.26.26.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.26.26.2.1.1.1.1" class="ltx_text" style="font-size:70%;">Conditioned on other</span></td>
</tr>
<tr id="S3.T4.1.26.26.2.1.2" class="ltx_tr">
<td id="S3.T4.1.26.26.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T4.1.26.26.2.1.2.1.1" class="ltx_text" style="font-size:70%;">ECG statements</span></td>
</tr>
</table>
</td>
<td id="S3.T4.1.26.26.3" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T4.1.26.26.3.1" class="ltx_text" style="font-size:70%;">SSSD-ECG</span></td>
<td id="S3.T4.1.26.26.4" class="ltx_td ltx_align_left ltx_border_b"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T4.1.26.26.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib124" title="" class="ltx_ref">124</a><span id="S3.T4.1.26.26.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T4.1.26.26.5" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T4.1.26.26.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T4.1.26.26.6" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T4.1.26.26.6.1" class="ltx_text" style="font-size:70%;">S,R</span></td>
<td id="S3.T4.1.26.26.7" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T4.1.26.26.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T4.1.26.26.8" class="ltx_td ltx_border_b"></td>
<td id="S3.T4.1.26.26.9" class="ltx_td ltx_border_b"></td>
<td id="S3.T4.1.26.26.10" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T4.1.26.26.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T4.1.26.26.11" class="ltx_td ltx_border_b"></td>
<td id="S3.T4.1.26.26.12" class="ltx_td ltx_border_b"></td>
<td id="S3.T4.1.26.26.13" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T4.1.26.26.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 4: </span>Overview of all summarized and discussed studies on <span id="S3.T4.6.1" class="ltx_text ltx_font_bold">ECG</span> synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), fidelity (F), diversity (D), qualitative assessment (Q), clinical utility (C) and privacy (P) evaluation. In evaluation of utility (U), R stands for real data, S stands for synthetic data, and A stands for augmented data. * stands for qualitative evaluation consisting of visualization of some plots only.** stands for availability of code for original method only. Acronyms: Deep convolutional conditional GAN (DCCGAN), Region-Disentangled Diffusion Model (RDDM)</figcaption>
</figure>
<div id="S3.SS2.SSS1.p1" class="ltx_para">
<p id="S3.SS2.SSS1.p1.1" class="ltx_p">Various technologies are employed in the synthesis of ECG signals with many proposed methods, such as Pulse2Pulse, WaveGAN, BiLSTM GAN, and TS-GAN, tailored to ECG data specifics such as morphology (appearance), durations, and intervals. The models were assessed for their ability to generate various synthetic ECG formats, including 2-lead synthesis, 12-lead synthesis, single-lead synthesis. ECG signals are commonly recorded in various formats, depending on the number of leads used. Each lead measures the electrical activity of the heart from a different angle, providing unique information about heart function. Each type of ECG provides a different level of detail and is suited for different clinical needs and settings. The 12-lead ECG is the most informative for diagnostic purposes, while single-lead ECGs are useful for continuous, everyday heart rhythm monitoring. An ECG signal is divided into heartbeats (also known as cardiac cycles), and a typical normal ECG heartbeat consists of three waves, the P wave, the QRS complex which contain the R peak, and the T wave.</p>
</div>
<div id="S3.SS2.SSS1.p2" class="ltx_para">
<p id="S3.SS2.SSS1.p2.1" class="ltx_p">Atrial Fibrillation (AF) is a cardiac disorder where abnormal electrical impulses start firing in the atria, causing a faster heart rate, with two known clinical markers: The P waves are either absent or replaced by fibrillatory wave, and the distance between consecutive R peaks is irregular, causing a phenomena called Heart Rate Variability (HRV). To simulate the two AF markers, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib105" title="" class="ltx_ref">105</a>]</cite> employs a dual-GAN approach, combining LSTM-GAN to generate the R-R interval time-series and DCGAN for generating signal morphology in terms of missing P waves or presence of abnormal fibrillatory waves before the QRS complex. This underscores the necessity of composite models to capture the intricate characteristics of the signal.</p>
</div>
<div id="S3.SS2.SSS1.p3" class="ltx_para">
<p id="S3.SS2.SSS1.p3.1" class="ltx_p">Most existing approaches to multi-lead ECG synthesis generate ECG lead signals that are independent of each other. To mitigate this, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib107" title="" class="ltx_ref">107</a>]</cite> introduces a 2 dimension bidirectional LSTM GAN , where the second dimension captures the physiological and spatial correlation among ECG leads of the same recording when synthesizing 12-lead ECGs.<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib106" title="" class="ltx_ref">106</a>]</cite> also presents a LSTM based GAN architecture, where the discriminator and generator are based on LSTMs, which demonstrates superiority over previous LSTM-based models like TimeGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib92" title="" class="ltx_ref">92</a>]</cite> and C-RNN-GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib125" title="" class="ltx_ref">125</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib104" title="" class="ltx_ref">104</a>]</cite> follows a hybrid approach combining LSTM and Convolutional Neural Network (CNN) layers, where the model utilizes bi-LSTM layers to extract temporal features and CNN layers to extract spatial features from ECG signals.</p>
</div>
<div id="S3.SS2.SSS1.p4" class="ltx_para">
<p id="S3.SS2.SSS1.p4.1" class="ltx_p">The GAN technologies are further diversified in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib108" title="" class="ltx_ref">108</a>]</cite>, which develops two state-of-the-art GAN methods, Pulse2Pulse (inspired by the U-Net architecture) and WaveGAN, to generate synthetic 12-lead ECG signals. Here, the Pulse2Pulse GAN outperforms its counterpart, highlighting the importance of architecture choice in generating realistic ECGs.<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib110" title="" class="ltx_ref">110</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib109" title="" class="ltx_ref">109</a>]</cite> share a common approach to treating time-series data akin to images, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib110" title="" class="ltx_ref">110</a>]</cite> employing a diffusion model for this purpose, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib109" title="" class="ltx_ref">109</a>]</cite> utilizes tts-gan, a transformer-based GAN architecture. Both studies pivot away from traditional time series modeling, instead adopting image processing paradigms for ECGs.</p>
</div>
<div id="S3.SS2.SSS1.p5" class="ltx_para">
<p id="S3.SS2.SSS1.p5.1" class="ltx_p">ECGs are typically categorized based on heartbeat types, including normal beats (N), premature ventricular contraction beats (V), and fusion beats (F), with various models leveraging these class labels for conditioned generation tasks. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib114" title="" class="ltx_ref">114</a>]</cite> developed a WGAN-GP inspired model that generates class-conditioned single-lead ECG data. The work in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref">59</a>]</cite> introduces an innovative hybrid model that combines the generative capabilities of GANs with VAEs. The resulting hybrid model consists of recurrent neural networks encoder and decoder, as well as a discriminator component to improve the quality of the class conditional generation. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib112" title="" class="ltx_ref">112</a>]</cite> employs a DDPM based model for a range of tasks including ECG signal generation, partial signal completion, and full heartbeat forecasting. This model is capable of conditional generation across various classes, indicating a flexible approach to the creation of ECG data.</p>
</div>
<div id="S3.SS2.SSS1.p6" class="ltx_para">
<p id="S3.SS2.SSS1.p6.1" class="ltx_p">Conditional VAEs play a role in conditional generation, as demonstrated by <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib113" title="" class="ltx_ref">113</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib116" title="" class="ltx_ref">116</a>]</cite>. The work in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib113" title="" class="ltx_ref">113</a>]</cite> showcased the VAE model’s faster convergence and reduced complexity (fewer parameters) compared to traditional GAN models. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib116" title="" class="ltx_ref">116</a>]</cite> also utilizes conditional VAEs to generate 12-lead ECG signals that are conditioned on specific subject characteristics, such as age, sex, and BMI, as well as image-derived data, which provides information on heart position and orientation from Cardiac MRI. This approach suggest a higher level of personalization in the synthetic data, potentially leading to more accurate representations of patient-specific ECG signals.</p>
</div>
<div id="S3.SS2.SSS1.p7" class="ltx_para">
<p id="S3.SS2.SSS1.p7.1" class="ltx_p">Structured State Space Model (SSSM) is a linear state space transition equation that proved to be a promising model to captures long-term dependencies in time series data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib126" title="" class="ltx_ref">126</a>]</cite>. This is achieved by discretizing the input and output relations and representing them as a convolution operation. SSSD-ECG <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib124" title="" class="ltx_ref">124</a>]</cite> and DSAT-ECG <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib111" title="" class="ltx_ref">111</a>]</cite> combine SSSM with diffusion models to generate synthetic ECG signals. The reader is referred to <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib127" title="" class="ltx_ref">127</a>]</cite> for more information about the integration of SSSM with diffusion models. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib124" title="" class="ltx_ref">124</a>]</cite> condition on representative referential ECG signals to guide the generation process, demonstrating superior performance over prior GAN-based models such as WaveGAN and Pulse2Pulse. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib111" title="" class="ltx_ref">111</a>]</cite>, on the other hand, synthesize conditional 12-lead ECG based on 12 heart rhythm classes, outperforming that of SSSD-ECG in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib124" title="" class="ltx_ref">124</a>]</cite> and earlier GAN works, including WaveGAN and Pulse2Pulse, in terms of fidelity and utility evaluation.</p>
</div>
<div id="S3.SS2.SSS1.p8" class="ltx_para">
<p id="S3.SS2.SSS1.p8.1" class="ltx_p">Other than conditioning on class labels, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib122" title="" class="ltx_ref">122</a>]</cite> relies on two WGAN-GP-based models conditioned on HRV characteristics and pulse morphological properties, underscoring the potential of incorporating domain knowledge into the generative process to generate synthetic but physiologically plausible cardiac signals.</p>
</div>
<div id="S3.SS2.SSS1.p9" class="ltx_para">
<p id="S3.SS2.SSS1.p9.1" class="ltx_p">Auto-TTE, presented in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib117" title="" class="ltx_ref">117</a>]</cite>, introduces an auto-regressive model integrating text descriptors and raw ECG data for comprehensive 12-lead ECG synthesis, allowing for patient-specific input like age and gender, marking a significant leap in the field.The generation process in Auto-TTE involves two phases. Initially, raw ECG signals are converted into a sequence of discrete tokens using a VQ-VAE. These tokens, alongside text tokens processed by a BPE tokenizer and patient specific tokens like age and gender, are then fed into an autoregressive Transformer-decoder model. This model is trained to generate ECG tokens autoregressively, which are then decoded back into ECG signals using Hifi-GAN decoder architecture, completing the synthesis process.</p>
</div>
<div id="S3.SS2.SSS1.p10" class="ltx_para">
<p id="S3.SS2.SSS1.p10.1" class="ltx_p">In another application, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib118" title="" class="ltx_ref">118</a>]</cite> presents a model capable of translating single-lead ECG inputs into a complete 12-lead set, utilizing a GAN framework with a bidirectional grid LSTM generator and a CNN discriminator, showcasing another innovative instance of LSTM and CNN hybrid model architecture. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib119" title="" class="ltx_ref">119</a>]</cite> also focuses on synthesizing twelve-lead ECGs from two asynchronous leads. Drawing inspiration from GAN-based image-to-image translation models, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib119" title="" class="ltx_ref">119</a>]</cite> develops a model primarily influenced by StarGAN v2 and its adaptations in speech processing to synthesize twelve-lead ECGs from two asynchronous leads. These translation capabilities are pivotal for enhancing the applicability of ECG data in clinical settings where different lead configurations are used.</p>
</div>
<div id="S3.SS2.SSS1.p11" class="ltx_para">
<p id="S3.SS2.SSS1.p11.1" class="ltx_p">The evaluation criteria of the synthetic data primarily concentrated on fidelity and utility, while diversity received limited attention in the cited studies <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref">59</a>, <a href="#bib.bib117" title="" class="ltx_ref">117</a>]</cite>. Generic fidelity metrics, including Frechet Distance, RMSE, and MMD, were prevalent. Signal-specific metrics such as dynamic time warping and time warp edit distance, that consider the temporal alignment of two time sequences, were introduced in only a handful of studies <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib93" title="" class="ltx_ref">93</a>, <a href="#bib.bib110" title="" class="ltx_ref">110</a>, <a href="#bib.bib111" title="" class="ltx_ref">111</a>, <a href="#bib.bib115" title="" class="ltx_ref">115</a>]</cite>. Some studies extended the evaluation by incorporating qualitative analysis from medical experts or by engaging in clinical validation. This involved comparing generated beats to templates and assessing the realism of HRV features in synthetic data. However, privacy was not explicitly mentioned in any of the papers, , indicating an area for potential future emphasis in model development and evaluation.</p>
</div>
</section>
<section id="S3.SS2.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.2.2 </span>EEGs and other signals</h4>

<div id="S3.SS2.SSS2.p1" class="ltx_para">
<p id="S3.SS2.SSS2.p1.1" class="ltx_p">Under the scope of unconditional synthesis, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib128" title="" class="ltx_ref">128</a>]</cite> employs a latent diffusion model to generate 30-second windows of synthetic EEG signals, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib129" title="" class="ltx_ref">129</a>]</cite> uses a generative model called Causal Recurrent Variational Autoencoder (CR-VAE) integrating Granger causality <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib130" title="" class="ltx_ref">130</a>]</cite> into a recurrent VAE framework to infer relationships between different time series variables. Given the success of language models, specifically GPT-2 <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib49" title="" class="ltx_ref">49</a>]</cite>, in different applications such as patent claims <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib131" title="" class="ltx_ref">131</a>]</cite> and stock market analysis <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib132" title="" class="ltx_ref">132</a>]</cite>, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib133" title="" class="ltx_ref">133</a>]</cite> wants to explore the capabilities of language models in a brand new field of application: the generation of bio-synthetic signals. For this reason, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib133" title="" class="ltx_ref">133</a>]</cite> trains GPT-2 language models for generating synthetic biological signals, where a model is trained for each class separately.</p>
</div>
<div id="S3.SS2.SSS2.p2" class="ltx_para">
<p id="S3.SS2.SSS2.p2.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib134" title="" class="ltx_ref">134</a>]</cite> uses Class conditional Wasserstein GAN (CWGAN) for generating synthetic EEG signals. However, instead of using the raw EEG signal directly, features extracted from the EEG are utilized due to the high dimensionality of the raw signal. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib135" title="" class="ltx_ref">135</a>]</cite> uses DDPMs to generate realistic synthetic data for a variety of physiological signals (LFP, ECoG, EEG) and evaluates if dataset-specific features such as sharp wave-ripples and ross-channel couplings were reproduced in the generated data. The work in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib136" title="" class="ltx_ref">136</a>]</cite> employs a diffusion model (DiffEEG) that uses the Short-time Fourier transform (STFT) spectrograms of the EEG signal segments as the condition for synthesis.</p>
</div>
<figure id="S3.T5" class="ltx_table">
<table id="S3.T5.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T5.1.1.1" class="ltx_tr">
<td id="S3.T5.1.1.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T5.1.1.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T5.1.1.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T5.1.1.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T5.1.1.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T5.1.1.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T5.1.1.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T5.1.1.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T5.1.1.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T5.1.1.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">paper</span></td>
<td id="S3.T5.1.1.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T5.1.1.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T5.1.1.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T5.1.1.1.7.1" class="ltx_text" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T5.1.1.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T5.1.1.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T5.1.2.2" class="ltx_tr">
<td id="S3.T5.1.2.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T5.1.2.2.1.1" class="ltx_text" style="font-size:70%;">U</span></td>
<td id="S3.T5.1.2.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T5.1.2.2.2.1" class="ltx_text" style="font-size:70%;">F</span></td>
<td id="S3.T5.1.2.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T5.1.2.2.3.1" class="ltx_text" style="font-size:70%;">D</span></td>
<td id="S3.T5.1.2.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T5.1.2.2.4.1" class="ltx_text" style="font-size:70%;">Q</span></td>
<td id="S3.T5.1.2.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T5.1.2.2.5.1" class="ltx_text" style="font-size:70%;">C</span></td>
<td id="S3.T5.1.2.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T5.1.2.2.6.1" class="ltx_text" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T5.1.3.3" class="ltx_tr">
<td id="S3.T5.1.3.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.3.3.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Train</span></td>
<td id="S3.T5.1.3.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.3.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T5.1.4.4" class="ltx_tr">
<td id="S3.T5.1.4.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="4"><span id="S3.T5.1.4.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T5.1.4.4.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.4.4.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T5.1.4.4.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T5.1.4.4.3.1" class="ltx_text" style="font-size:70%;">EEG</span></td>
<td id="S3.T5.1.4.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T5.1.4.4.4.1" class="ltx_text" style="font-size:70%;">LDM</span></td>
<td id="S3.T5.1.4.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T5.1.4.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib128" title="" class="ltx_ref">128</a><span id="S3.T5.1.4.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T5.1.4.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T5.1.4.4.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T5.1.4.4.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.4.4.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.4.4.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.4.4.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T5.1.4.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.4.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.4.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.4.4.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.4.4.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.4.4.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T5.1.5.5" class="ltx_tr">
<td id="S3.T5.1.5.5.1" class="ltx_td ltx_align_center"><span id="S3.T5.1.5.5.1.1" class="ltx_text" style="font-size:70%;">VAE</span></td>
<td id="S3.T5.1.5.5.2" class="ltx_td ltx_align_left"><span id="S3.T5.1.5.5.2.1" class="ltx_text" style="font-size:70%;">EEG,.Fmri</span></td>
<td id="S3.T5.1.5.5.3" class="ltx_td ltx_align_left"><span id="S3.T5.1.5.5.3.1" class="ltx_text" style="font-size:70%;">CRVAE</span></td>
<td id="S3.T5.1.5.5.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T5.1.5.5.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib129" title="" class="ltx_ref">129</a><span id="S3.T5.1.5.5.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T5.1.5.5.5" class="ltx_td ltx_align_left"><span id="S3.T5.1.5.5.5.1" class="ltx_text" style="font-size:70%;">Predict next samples</span></td>
<td id="S3.T5.1.5.5.6" class="ltx_td ltx_align_center"><span id="S3.T5.1.5.5.6.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T5.1.5.5.7" class="ltx_td ltx_align_center"><span id="S3.T5.1.5.5.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T5.1.5.5.8" class="ltx_td ltx_align_center"><span id="S3.T5.1.5.5.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T5.1.5.5.9" class="ltx_td"></td>
<td id="S3.T5.1.5.5.10" class="ltx_td ltx_align_center"><span id="S3.T5.1.5.5.10.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T5.1.5.5.11" class="ltx_td"></td>
<td id="S3.T5.1.5.5.12" class="ltx_td"></td>
<td id="S3.T5.1.5.5.13" class="ltx_td ltx_align_center"><span id="S3.T5.1.5.5.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T5.1.6.6" class="ltx_tr">
<td id="S3.T5.1.6.6.1" class="ltx_td ltx_align_center"><span id="S3.T5.1.6.6.1.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T5.1.6.6.2" class="ltx_td ltx_align_left"><span id="S3.T5.1.6.6.2.1" class="ltx_text" style="font-size:70%;">stereo EEG</span></td>
<td id="S3.T5.1.6.6.3" class="ltx_td ltx_align_left"><span id="S3.T5.1.6.6.3.1" class="ltx_text" style="font-size:70%;">TGAN</span></td>
<td id="S3.T5.1.6.6.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T5.1.6.6.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib137" title="" class="ltx_ref">137</a><span id="S3.T5.1.6.6.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T5.1.6.6.5" class="ltx_td ltx_align_left"><span id="S3.T5.1.6.6.5.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T5.1.6.6.6" class="ltx_td ltx_align_center"><span id="S3.T5.1.6.6.6.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T5.1.6.6.7" class="ltx_td ltx_align_center"><span id="S3.T5.1.6.6.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T5.1.6.6.8" class="ltx_td"></td>
<td id="S3.T5.1.6.6.9" class="ltx_td"></td>
<td id="S3.T5.1.6.6.10" class="ltx_td ltx_align_center"><span id="S3.T5.1.6.6.10.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T5.1.6.6.11" class="ltx_td"></td>
<td id="S3.T5.1.6.6.12" class="ltx_td"></td>
<td id="S3.T5.1.6.6.13" class="ltx_td"></td>
</tr>
<tr id="S3.T5.1.7.7" class="ltx_tr">
<td id="S3.T5.1.7.7.1" class="ltx_td ltx_align_center"><span id="S3.T5.1.7.7.1.1" class="ltx_text" style="font-size:70%;">LLM</span></td>
<td id="S3.T5.1.7.7.2" class="ltx_td ltx_align_left"><span id="S3.T5.1.7.7.2.1" class="ltx_text" style="font-size:70%;">EEG,EMG</span></td>
<td id="S3.T5.1.7.7.3" class="ltx_td ltx_align_left"><span id="S3.T5.1.7.7.3.1" class="ltx_text" style="font-size:70%;">GPT2</span></td>
<td id="S3.T5.1.7.7.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T5.1.7.7.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib133" title="" class="ltx_ref">133</a><span id="S3.T5.1.7.7.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T5.1.7.7.5" class="ltx_td ltx_align_left"><span id="S3.T5.1.7.7.5.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T5.1.7.7.6" class="ltx_td ltx_align_center"><span id="S3.T5.1.7.7.6.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T5.1.7.7.7" class="ltx_td ltx_align_center"><span id="S3.T5.1.7.7.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T5.1.7.7.8" class="ltx_td"></td>
<td id="S3.T5.1.7.7.9" class="ltx_td"></td>
<td id="S3.T5.1.7.7.10" class="ltx_td"></td>
<td id="S3.T5.1.7.7.11" class="ltx_td"></td>
<td id="S3.T5.1.7.7.12" class="ltx_td"></td>
<td id="S3.T5.1.7.7.13" class="ltx_td ltx_align_center"><span id="S3.T5.1.7.7.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T5.1.8.8" class="ltx_tr">
<td id="S3.T5.1.8.8.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T5.1.8.8.1.1" class="ltx_text" style="font-size:70%;">Class-conditional</span></td>
<td id="S3.T5.1.8.8.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.8.8.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T5.1.8.8.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T5.1.8.8.3.1" class="ltx_text" style="font-size:70%;">EEG,LFP, ECoG,</span></td>
<td id="S3.T5.1.8.8.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T5.1.8.8.4.1" class="ltx_text" style="font-size:70%;">DDPM</span></td>
<td id="S3.T5.1.8.8.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T5.1.8.8.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib135" title="" class="ltx_ref">135</a><span id="S3.T5.1.8.8.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T5.1.8.8.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T5.1.8.8.6.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T5.1.8.8.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.8.8.7.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T5.1.8.8.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.8.8.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T5.1.8.8.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.8.8.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T5.1.8.8.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.8.8.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.8.8.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T5.1.8.8.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T5.1.8.8.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T5.1.8.8.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T5.1.9.9" class="ltx_tr">
<td id="S3.T5.1.9.9.1" class="ltx_td ltx_align_center"><span id="S3.T5.1.9.9.1.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T5.1.9.9.2" class="ltx_td ltx_align_left"><span id="S3.T5.1.9.9.2.1" class="ltx_text" style="font-size:70%;">EEG</span></td>
<td id="S3.T5.1.9.9.3" class="ltx_td ltx_align_left"><span id="S3.T5.1.9.9.3.1" class="ltx_text" style="font-size:70%;">Conditional WGAN</span></td>
<td id="S3.T5.1.9.9.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T5.1.9.9.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib134" title="" class="ltx_ref">134</a><span id="S3.T5.1.9.9.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T5.1.9.9.5" class="ltx_td ltx_align_left"><span id="S3.T5.1.9.9.5.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T5.1.9.9.6" class="ltx_td ltx_align_center"><span id="S3.T5.1.9.9.6.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T5.1.9.9.7" class="ltx_td ltx_align_center"><span id="S3.T5.1.9.9.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T5.1.9.9.8" class="ltx_td"></td>
<td id="S3.T5.1.9.9.9" class="ltx_td"></td>
<td id="S3.T5.1.9.9.10" class="ltx_td"></td>
<td id="S3.T5.1.9.9.11" class="ltx_td"></td>
<td id="S3.T5.1.9.9.12" class="ltx_td"></td>
<td id="S3.T5.1.9.9.13" class="ltx_td"></td>
</tr>
<tr id="S3.T5.1.10.10" class="ltx_tr">
<td id="S3.T5.1.10.10.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T5.1.10.10.1.1" class="ltx_text" style="font-size:70%;">Other Conditioning</span></td>
<td id="S3.T5.1.10.10.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T5.1.10.10.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T5.1.10.10.3" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T5.1.10.10.3.1" class="ltx_text" style="font-size:70%;">STFT spectrograms</span></td>
<td id="S3.T5.1.10.10.4" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T5.1.10.10.4.1" class="ltx_text" style="font-size:70%;">DiffEEG</span></td>
<td id="S3.T5.1.10.10.5" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T5.1.10.10.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib136" title="" class="ltx_ref">136</a><span id="S3.T5.1.10.10.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T5.1.10.10.6" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T5.1.10.10.6.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T5.1.10.10.7" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T5.1.10.10.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T5.1.10.10.8" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T5.1.10.10.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T5.1.10.10.9" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T5.1.10.10.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T5.1.10.10.10" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T5.1.10.10.11" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T5.1.10.10.12" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T5.1.10.10.13" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T5.1.10.10.14" class="ltx_td ltx_border_b ltx_border_t"></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 5: </span>Overview of all summarized and discussed studies on <span id="S3.T5.6.1" class="ltx_text ltx_font_bold">EEG</span> synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), fidelity (F), diversity (D), qualitative assessment (Q), clinical utility (C) and privacy (P) evaluation. In evaluation of utility (U), R stands for real data, S stands for synthetic data, and A stands for augmented data. * stands for qualitative evaluation consisting of visualization of some plots only.Acronyms: Large Language Model (LLM), causal recurrent VAE (CRVAE), temporal GAN (TGAN)</figcaption>
</figure>
</section>
</section>
<section id="S3.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.3 </span>Medical imaging</h3>

<div id="S3.SS3.p1" class="ltx_para">
<p id="S3.SS3.p1.1" class="ltx_p">In this section, we investigate the synthesis of medical image data across various modalities essential for diagnostic imaging. Medical imaging plays a crucial role in diagnosis, treatment planning, and monitoring of various medical conditions. Our exploration delves into the synthesis techniques tailored to each imaging modality, ranging from dermatoscopy (also known as dermoscopy) <a href="#S3.SS3.SSS1" title="3.3.1 Dermatoscopy ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3.1</span></a>, mammography <a href="#S3.SS3.SSS2" title="3.3.2 Mammography ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3.2</span></a>, ultrasound <a href="#S3.SS3.SSS3" title="3.3.3 Ultrasound ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3.3</span></a>, to MRI <a href="#S3.SS3.SSS4" title="3.3.4 MRI ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3.4</span></a>, CT scans <a href="#S3.SS3.SSS5" title="3.3.5 CT ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3.5</span></a>, X-rays <a href="#S3.SS3.SSS6" title="3.3.6 X-Ray ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3.6</span></a>, OCTs <a href="#S3.SS3.SSS7" title="3.3.7 OCT ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3.7</span></a>, and multiple modalities <a href="#S3.SS3.SSS8" title="3.3.8 Multiple Modalities ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3.3.8</span></a>. Through the analysis of synthesis methods and evaluation strategies, we aim to enhance understanding of synthetic medical image data generation and its applications in healthcare research.</p>
</div>
<section id="S3.SS3.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.1 </span>Dermatoscopy </h4>

<figure id="S3.T6" class="ltx_table">
<table id="S3.T6.1" class="ltx_tabular ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T6.1.1.1" class="ltx_tr">
<td id="S3.T6.1.1.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T6.1.1.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T6.1.1.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T6.1.1.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T6.1.1.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T6.1.1.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T6.1.1.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T6.1.1.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T6.1.1.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T6.1.1.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T6.1.1.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T6.1.1.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T6.1.1.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T6.1.1.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T6.1.1.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T6.1.1.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T6.1.2.2" class="ltx_tr">
<td id="S3.T6.1.2.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T6.1.2.2.1.1" class="ltx_text" style="font-size:70%;">U</span></td>
<td id="S3.T6.1.2.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T6.1.2.2.2.1" class="ltx_text" style="font-size:70%;">F</span></td>
<td id="S3.T6.1.2.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T6.1.2.2.3.1" class="ltx_text" style="font-size:70%;">D</span></td>
<td id="S3.T6.1.2.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T6.1.2.2.4.1" class="ltx_text" style="font-size:70%;">Q</span></td>
<td id="S3.T6.1.2.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T6.1.2.2.5.1" class="ltx_text" style="font-size:70%;">C</span></td>
<td id="S3.T6.1.2.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T6.1.2.2.6.1" class="ltx_text" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T6.1.3.3" class="ltx_tr">
<td id="S3.T6.1.3.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.3.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T6.1.3.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.3.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T6.1.4.4" class="ltx_tr">
<td id="S3.T6.1.4.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="4"><span id="S3.T6.1.4.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T6.1.4.4.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="4"><span id="S3.T6.1.4.4.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T6.1.4.4.3" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.4.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T6.1.4.4.4.1" class="ltx_text" style="font-size:70%;">PGGAN</span></td>
<td id="S3.T6.1.4.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.4.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib138" title="" class="ltx_ref">138</a><span id="S3.T6.1.4.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.4.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T6.1.4.4.6.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.4.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.4.4.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T6.1.4.4.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.4.4.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T6.1.4.4.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.4.4.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.4.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.4.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.4.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.4.4.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.4.4.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T6.1.5.5" class="ltx_tr">
<td id="S3.T6.1.5.5.1" class="ltx_td ltx_align_left"><span id="S3.T6.1.5.5.1.1" class="ltx_text" style="font-size:70%;">Rosacea</span></td>
<td id="S3.T6.1.5.5.2" class="ltx_td ltx_align_left"><span id="S3.T6.1.5.5.2.1" class="ltx_text" style="font-size:70%;">StyleGAN2-ADA</span></td>
<td id="S3.T6.1.5.5.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.5.5.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib139" title="" class="ltx_ref">139</a><span id="S3.T6.1.5.5.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.5.5.4" class="ltx_td ltx_align_left"><span id="S3.T6.1.5.5.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T6.1.5.5.5" class="ltx_td"></td>
<td id="S3.T6.1.5.5.6" class="ltx_td"></td>
<td id="S3.T6.1.5.5.7" class="ltx_td ltx_align_center"><span id="S3.T6.1.5.5.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.5.5.8" class="ltx_td"></td>
<td id="S3.T6.1.5.5.9" class="ltx_td ltx_align_center"><span id="S3.T6.1.5.5.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.5.5.10" class="ltx_td"></td>
<td id="S3.T6.1.5.5.11" class="ltx_td"></td>
<td id="S3.T6.1.5.5.12" class="ltx_td ltx_align_center"><span id="S3.T6.1.5.5.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T6.1.6.6" class="ltx_tr">
<td id="S3.T6.1.6.6.1" class="ltx_td"></td>
<td id="S3.T6.1.6.6.2" class="ltx_td ltx_align_left"><span id="S3.T6.1.6.6.2.1" class="ltx_text" style="font-size:70%;">StyleGAN2</span></td>
<td id="S3.T6.1.6.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.6.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib140" title="" class="ltx_ref">140</a><span id="S3.T6.1.6.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.6.6.4" class="ltx_td ltx_align_left"><span id="S3.T6.1.6.6.4.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.6.6.5" class="ltx_td ltx_align_center"><span id="S3.T6.1.6.6.5.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T6.1.6.6.6" class="ltx_td ltx_align_center"><span id="S3.T6.1.6.6.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T6.1.6.6.7" class="ltx_td ltx_align_center"><span id="S3.T6.1.6.6.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.6.6.8" class="ltx_td"></td>
<td id="S3.T6.1.6.6.9" class="ltx_td"></td>
<td id="S3.T6.1.6.6.10" class="ltx_td"></td>
<td id="S3.T6.1.6.6.11" class="ltx_td"></td>
<td id="S3.T6.1.6.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T6.1.7.7" class="ltx_tr">
<td id="S3.T6.1.7.7.1" class="ltx_td"></td>
<td id="S3.T6.1.7.7.2" class="ltx_td ltx_align_left"><span id="S3.T6.1.7.7.2.1" class="ltx_text" style="font-size:70%;">SLA-StyleGAN</span></td>
<td id="S3.T6.1.7.7.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.7.7.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib141" title="" class="ltx_ref">141</a><span id="S3.T6.1.7.7.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.7.7.4" class="ltx_td ltx_align_left"><span id="S3.T6.1.7.7.4.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.7.7.5" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T6.1.7.7.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.7.7.6" class="ltx_td ltx_align_center"><span id="S3.T6.1.7.7.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.7.7.7" class="ltx_td"></td>
<td id="S3.T6.1.7.7.8" class="ltx_td"></td>
<td id="S3.T6.1.7.7.9" class="ltx_td"></td>
<td id="S3.T6.1.7.7.10" class="ltx_td"></td>
<td id="S3.T6.1.7.7.11" class="ltx_td"></td>
</tr>
<tr id="S3.T6.1.8.8" class="ltx_tr">
<td id="S3.T6.1.8.8.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T6.1.8.8.1.1" class="ltx_text" style="font-size:70%;">Class or Label Cond.</span></td>
<td id="S3.T6.1.8.8.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T6.1.8.8.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T6.1.8.8.3" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.8.8.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S3.T6.1.8.8.4.1" class="ltx_text" style="font-size:70%;">StyleGAN2-ADA</span></td>
<td id="S3.T6.1.8.8.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.8.8.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib142" title="" class="ltx_ref">142</a><span id="S3.T6.1.8.8.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.8.8.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T6.1.8.8.6.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.8.8.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.8.8.7.1" class="ltx_text" style="font-size:70%;">S,A</span></td>
<td id="S3.T6.1.8.8.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.8.8.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T6.1.8.8.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.8.8.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.8.8.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.8.8.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.8.8.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.8.8.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.8.8.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T6.1.8.8.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T6.1.9.9" class="ltx_tr">
<td id="S3.T6.1.9.9.1" class="ltx_td"></td>
<td id="S3.T6.1.9.9.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.9.9.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib143" title="" class="ltx_ref">143</a><span id="S3.T6.1.9.9.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.9.9.3" class="ltx_td ltx_align_left"><span id="S3.T6.1.9.9.3.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.9.9.4" class="ltx_td ltx_align_center"><span id="S3.T6.1.9.9.4.1" class="ltx_text" style="font-size:70%;">S,A</span></td>
<td id="S3.T6.1.9.9.5" class="ltx_td ltx_align_center"><span id="S3.T6.1.9.9.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T6.1.9.9.6" class="ltx_td ltx_align_center"><span id="S3.T6.1.9.9.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.9.9.7" class="ltx_td ltx_align_center"><span id="S3.T6.1.9.9.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.9.9.8" class="ltx_td ltx_align_center"><span id="S3.T6.1.9.9.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.9.9.9" class="ltx_td"></td>
<td id="S3.T6.1.9.9.10" class="ltx_td ltx_align_center"><span id="S3.T6.1.9.9.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.9.9.11" class="ltx_td ltx_align_center"><span id="S3.T6.1.9.9.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T6.1.10.10" class="ltx_tr">
<td id="S3.T6.1.10.10.1" class="ltx_td"></td>
<td id="S3.T6.1.10.10.2" class="ltx_td ltx_align_left"><span id="S3.T6.1.10.10.2.1" class="ltx_text" style="font-size:70%;">cGAN</span></td>
<td id="S3.T6.1.10.10.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.10.10.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib144" title="" class="ltx_ref">144</a><span id="S3.T6.1.10.10.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.10.10.4" class="ltx_td ltx_align_left"><span id="S3.T6.1.10.10.4.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.10.10.5" class="ltx_td ltx_align_center"><span id="S3.T6.1.10.10.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T6.1.10.10.6" class="ltx_td ltx_align_center"><span id="S3.T6.1.10.10.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T6.1.10.10.7" class="ltx_td"></td>
<td id="S3.T6.1.10.10.8" class="ltx_td"></td>
<td id="S3.T6.1.10.10.9" class="ltx_td"></td>
<td id="S3.T6.1.10.10.10" class="ltx_td"></td>
<td id="S3.T6.1.10.10.11" class="ltx_td"></td>
<td id="S3.T6.1.10.10.12" class="ltx_td"></td>
</tr>
<tr id="S3.T6.1.11.11" class="ltx_tr">
<td id="S3.T6.1.11.11.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T6.1.11.11.1.1" class="ltx_text" style="font-size:70%;">Text-guided</span></td>
<td id="S3.T6.1.11.11.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T6.1.11.11.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T6.1.11.11.3" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.11.11.4" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T6.1.11.11.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T6.1.11.11.4.1.1" class="ltx_tr">
<td id="S3.T6.1.11.11.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T6.1.11.11.4.1.1.1.1" class="ltx_text" style="font-size:70%;">LDM, Stable Diffusion,</span></td>
</tr>
<tr id="S3.T6.1.11.11.4.1.2" class="ltx_tr">
<td id="S3.T6.1.11.11.4.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T6.1.11.11.4.1.2.1.1" class="ltx_text" style="font-size:70%;">Fine tuned stable diffusion</span></td>
</tr>
</table>
</td>
<td id="S3.T6.1.11.11.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.11.11.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib145" title="" class="ltx_ref">145</a><span id="S3.T6.1.11.11.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.11.11.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T6.1.11.11.6.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.11.11.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T6.1.11.11.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.11.11.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.11.11.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.11.11.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.11.11.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.11.11.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T6.1.11.11.13" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T6.1.12.12" class="ltx_tr">
<td id="S3.T6.1.12.12.1" class="ltx_td"></td>
<td id="S3.T6.1.12.12.2" class="ltx_td ltx_align_left"><span id="S3.T6.1.12.12.2.1" class="ltx_text" style="font-size:70%;">LDM</span></td>
<td id="S3.T6.1.12.12.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.12.12.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib146" title="" class="ltx_ref">146</a><span id="S3.T6.1.12.12.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.12.12.4" class="ltx_td ltx_align_left"><span id="S3.T6.1.12.12.4.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.12.12.5" class="ltx_td ltx_align_center"><span id="S3.T6.1.12.12.5.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T6.1.12.12.6" class="ltx_td ltx_align_center"><span id="S3.T6.1.12.12.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T6.1.12.12.7" class="ltx_td"></td>
<td id="S3.T6.1.12.12.8" class="ltx_td"></td>
<td id="S3.T6.1.12.12.9" class="ltx_td"></td>
<td id="S3.T6.1.12.12.10" class="ltx_td"></td>
<td id="S3.T6.1.12.12.11" class="ltx_td"></td>
<td id="S3.T6.1.12.12.12" class="ltx_td"></td>
</tr>
<tr id="S3.T6.1.13.13" class="ltx_tr">
<td id="S3.T6.1.13.13.1" class="ltx_td"></td>
<td id="S3.T6.1.13.13.2" class="ltx_td ltx_align_left"><span id="S3.T6.1.13.13.2.1" class="ltx_text" style="font-size:70%;">DALL-E2</span></td>
<td id="S3.T6.1.13.13.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.13.13.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib147" title="" class="ltx_ref">147</a><span id="S3.T6.1.13.13.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.13.13.4" class="ltx_td ltx_align_left"><span id="S3.T6.1.13.13.4.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.13.13.5" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T6.1.13.13.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.13.13.6" class="ltx_td"></td>
<td id="S3.T6.1.13.13.7" class="ltx_td"></td>
<td id="S3.T6.1.13.13.8" class="ltx_td"></td>
<td id="S3.T6.1.13.13.9" class="ltx_td"></td>
<td id="S3.T6.1.13.13.10" class="ltx_td"></td>
<td id="S3.T6.1.13.13.11" class="ltx_td"></td>
</tr>
<tr id="S3.T6.1.14.14" class="ltx_tr">
<td id="S3.T6.1.14.14.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T6.1.14.14.1.1" class="ltx_text" style="font-size:70%;">Intra-modal trans.</span></td>
<td id="S3.T6.1.14.14.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T6.1.14.14.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T6.1.14.14.3" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T6.1.14.14.3.1" class="ltx_text" style="font-size:70%;">Majority to Minority</span></td>
<td id="S3.T6.1.14.14.4" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T6.1.14.14.4.1" class="ltx_text" style="font-size:70%;">CycleGAN</span></td>
<td id="S3.T6.1.14.14.5" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T6.1.14.14.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib148" title="" class="ltx_ref">148</a><span id="S3.T6.1.14.14.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T6.1.14.14.6" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T6.1.14.14.6.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T6.1.14.14.7" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T6.1.14.14.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T6.1.14.14.8" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T6.1.14.14.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T6.1.14.14.9" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T6.1.14.14.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T6.1.14.14.10" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T6.1.14.14.11" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T6.1.14.14.12" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T6.1.14.14.13" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T6.1.14.14.14" class="ltx_td ltx_border_b ltx_border_t"></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 6: </span>Overview of all summarized and discussed studies on <span id="S3.T6.6.1" class="ltx_text ltx_font_bold">dermatology</span> synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), fidelity (F), diversity (D), qualitative assessment (Q), clinical utility (C) and privacy (P) evaluation. In evaluation of utility (U), R stands for real data, S stands for synthetic data, and A stands for augmented data </figcaption>
</figure>
<div id="S3.SS3.SSS1.p1" class="ltx_para">
<p id="S3.SS3.SSS1.p1.1" class="ltx_p">The research on generative models in the field of dermoscopic imaging involves GANs and diffusion models. GANs, including variants like PGGAN, StyleGAN2 and its adaptations , and CycleGAN, have been used for data augmentation, class imbalance correction, and improving classification performance across diverse open source datasets like ISIC, Fitzpatrick 17k, ISIC, and BCN10000. These datasets are valuable resources for dermatology research, with Fitzpatrick 17k being annotated with Fitzpatrick skin types <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib149" title="" class="ltx_ref">149</a>]</cite>, a six-point scale that categorizes skin based on color and reaction to sun exposure.
<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib138" title="" class="ltx_ref">138</a>]</cite> utilized PGGAN to generate synthetic skin images and used them for data augmentation, outperforming common augmentation tools, like Unsupervised Data Augmentation for Consistency Training (UDA) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib150" title="" class="ltx_ref">150</a>]</cite>, Random Eraser <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib151" title="" class="ltx_ref">151</a>]</cite>, and random data augmentation. Further GANs employed in the unconditional synthesis utilize adaptations of StyleGAN2 <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib139" title="" class="ltx_ref">139</a>, <a href="#bib.bib140" title="" class="ltx_ref">140</a>, <a href="#bib.bib141" title="" class="ltx_ref">141</a>]</cite>. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib139" title="" class="ltx_ref">139</a>]</cite>, the StyleGAN2 with adaptive discriminator augmentation (StyleGAN2-ADA) model is fine-tuned on a small limited Rosacea dataset. The synthetic data was qualitatively evaluated by specialist dermatologists.
In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib140" title="" class="ltx_ref">140</a>]</cite>, StyleGAN2 network together with the Differentiable Augmentation (DiffAugment) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib152" title="" class="ltx_ref">152</a>]</cite> is used to balance the number of images per class and enhance the performance of the skin lesion images classifier.<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib141" title="" class="ltx_ref">141</a>]</cite> considered an adaptation of the original StyleGAN, with modifications to enhance style control mechanisms, focusing on replicating specific features such as hair, color, and lesion shapes in the images. By introducing variability through noise, the model generated a diverse set of synthetic samples.</p>
</div>
<div id="S3.SS3.SSS1.p2" class="ltx_para">
<p id="S3.SS3.SSS1.p2.1" class="ltx_p">StyleGAN2-ADA was also used in studies for conditional synthesis<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib142" title="" class="ltx_ref">142</a>, <a href="#bib.bib143" title="" class="ltx_ref">143</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib142" title="" class="ltx_ref">142</a>]</cite> examined if the generated data makes the classification models more prone to biases, specifically those associated with specific artifacts (e.g.: hairs, frames, or rulers present in the image). The results show that while GANs tend to amplify prominent biases present in the training data, they can mitigate less common ones. The study also observed that models trained on a mix of real and synthetic data were more resilient to bias, maintaining consistent predictions even when biases were introduced into the data. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib143" title="" class="ltx_ref">143</a>]</cite> considered StyleGAN2-ADA in a federated learning setup.</p>
</div>
<div id="S3.SS3.SSS1.p3" class="ltx_para">
<p id="S3.SS3.SSS1.p3.1" class="ltx_p">Recently, different variants of diffusion models such as DALL.E-2 and LDM-based Stable Diffusion are employed for generating high-quality synthetic images <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib145" title="" class="ltx_ref">145</a>, <a href="#bib.bib146" title="" class="ltx_ref">146</a>, <a href="#bib.bib147" title="" class="ltx_ref">147</a>]</cite>, with a focus text-to-image synthesis applications.Specifically, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib147" title="" class="ltx_ref">147</a>]</cite> uses DALL.E-2 to generate synthetic images from different skin types using structured text prompts specifying the type of medical condition, the location of the condition, the skin color of the patient, as well as the gender. This work show that the generated synthetic data can improve the classification of skin disease especially for underrepresented Fitzpatrick skin types. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib145" title="" class="ltx_ref">145</a>]</cite> fine-tuned the Stable Diffusion on a small number of training images from Fitzpatrick 17k using DreamBooth <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib153" title="" class="ltx_ref">153</a>]</cite> method to generate images of skin diseases conditioned on text. The synthetic data was used for augmenting training data for an disease classification problem, and showed that it can mitigate class imbalance for under-represented skin-types and improve the classification result, outperforming basic image transformations (comprising random flipping, cropping, rotating, warping, and lighting changes).</p>
</div>
<div id="S3.SS3.SSS1.p4" class="ltx_para">
<p id="S3.SS3.SSS1.p4.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib148" title="" class="ltx_ref">148</a>]</cite> employed CycleGANs to tackle the challenge of class imbalance in skin disease classifiers by generating synthetic images. Given the seven classes of images in HAM10000, they introduced a novel training strategy for the generators, taking the majority class images and pairing them with each of the 6 minority classes separately to facilitate the generation of minority class images, enhancing the classification accuracy of the downstream models.</p>
</div>
<div id="S3.SS3.SSS1.p5" class="ltx_para">
<p id="S3.SS3.SSS1.p5.1" class="ltx_p">While majority of papers evaluate the fidelity and utility of the generated data, only two papers <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib139" title="" class="ltx_ref">139</a>, <a href="#bib.bib143" title="" class="ltx_ref">143</a>]</cite> conduct a qualitative assessment, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib143" title="" class="ltx_ref">143</a>]</cite> utilizing 4 experts, 2 of which are dermatologist and 2 are DL experts, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib139" title="" class="ltx_ref">139</a>]</cite> employ three dermatologist, which raise questions about the validity and the statistical significance of such qualitative assessment. Moreover, only one paper <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib143" title="" class="ltx_ref">143</a>]</cite> consider privacy aspect of the generative model and the generated data. To assess the authenticity of the generative model, the authors project real dataset samples into the latent space of the generator and measure the similarity between pairs of real and generated images. This same paper measures diversity of the generated images by calculating the recall of the visual Turing test.</p>
</div>
</section>
<section id="S3.SS3.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.2 </span>Mammography</h4>

<figure id="S3.T7" class="ltx_table">
<table id="S3.T7.3" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T7.3.4.1" class="ltx_tr">
<td id="S3.T7.3.4.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T7.3.4.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T7.3.4.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T7.3.4.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T7.3.4.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T7.3.4.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T7.3.4.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T7.3.4.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T7.3.4.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T7.3.4.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T7.3.4.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T7.3.4.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T7.3.4.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T7.3.4.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T7.3.4.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T7.3.4.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T7.3.5.2" class="ltx_tr">
<td id="S3.T7.3.5.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T7.3.5.2.1.1" class="ltx_text" style="font-size:70%;">U</span></td>
<td id="S3.T7.3.5.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T7.3.5.2.2.1" class="ltx_text" style="font-size:70%;">F</span></td>
<td id="S3.T7.3.5.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T7.3.5.2.3.1" class="ltx_text" style="font-size:70%;">D</span></td>
<td id="S3.T7.3.5.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T7.3.5.2.4.1" class="ltx_text" style="font-size:70%;">Q</span></td>
<td id="S3.T7.3.5.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T7.3.5.2.5.1" class="ltx_text" style="font-size:70%;">C</span></td>
<td id="S3.T7.3.5.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T7.3.5.2.6.1" class="ltx_text" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T7.3.6.3" class="ltx_tr">
<td id="S3.T7.3.6.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.6.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T7.3.6.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.6.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T7.3.7.4" class="ltx_tr">
<td id="S3.T7.3.7.4.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.7.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T7.3.7.4.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.7.4.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T7.3.7.4.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.7.4.3.1" class="ltx_text" style="font-size:70%;">patch-based</span></td>
<td id="S3.T7.3.7.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.7.4.4.1" class="ltx_text" style="font-size:70%;">DCGAN, WGAN-GP</span></td>
<td id="S3.T7.3.7.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.7.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib154" title="" class="ltx_ref">154</a><span id="S3.T7.3.7.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.7.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.7.4.6.1" class="ltx_text" style="font-size:70%;">class</span></td>
<td id="S3.T7.3.7.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.7.4.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T7.3.7.4.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.7.4.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T7.3.7.4.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.7.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.7.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.7.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.7.4.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.7.4.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T7.3.8.5" class="ltx_tr">
<td id="S3.T7.3.8.5.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T7.3.8.5.1.1" class="ltx_text" style="font-size:70%;">Class or Label Cond.</span></td>
<td id="S3.T7.3.8.5.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T7.3.8.5.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T7.3.8.5.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.8.5.3.1" class="ltx_text" style="font-size:70%;">DFM</span></td>
<td id="S3.T7.3.8.5.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.8.5.4.1" class="ltx_text" style="font-size:70%;">cGAN</span></td>
<td id="S3.T7.3.8.5.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.8.5.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib155" title="" class="ltx_ref">155</a><span id="S3.T7.3.8.5.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.8.5.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.8.5.6.1" class="ltx_text" style="font-size:70%;">class</span></td>
<td id="S3.T7.3.8.5.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T7.3.8.5.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.8.5.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.8.5.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.8.5.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.8.5.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.8.5.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.8.5.13" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T7.3.9.6" class="ltx_tr">
<td id="S3.T7.3.9.6.1" class="ltx_td ltx_align_left"><span id="S3.T7.3.9.6.1.1" class="ltx_text" style="font-size:70%;">ROI-based</span></td>
<td id="S3.T7.3.9.6.2" class="ltx_td ltx_align_left"><span id="S3.T7.3.9.6.2.1" class="ltx_text" style="font-size:70%;">ROImammoGAN</span></td>
<td id="S3.T7.3.9.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.9.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib156" title="" class="ltx_ref">156</a><span id="S3.T7.3.9.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.9.6.4" class="ltx_td ltx_align_left"><span id="S3.T7.3.9.6.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T7.3.9.6.5" class="ltx_td"></td>
<td id="S3.T7.3.9.6.6" class="ltx_td"></td>
<td id="S3.T7.3.9.6.7" class="ltx_td ltx_align_center"><span id="S3.T7.3.9.6.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.9.6.8" class="ltx_td"></td>
<td id="S3.T7.3.9.6.9" class="ltx_td"></td>
<td id="S3.T7.3.9.6.10" class="ltx_td"></td>
<td id="S3.T7.3.9.6.11" class="ltx_td"></td>
<td id="S3.T7.3.9.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T7.3.10.7" class="ltx_tr">
<td id="S3.T7.3.10.7.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.10.7.1.1" class="ltx_text" style="font-size:70%;">Text-guided</span></td>
<td id="S3.T7.3.10.7.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.10.7.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T7.3.10.7.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.10.7.3.1" class="ltx_text" style="font-size:70%;">FFDM</span></td>
<td id="S3.T7.3.10.7.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.10.7.4.1" class="ltx_text" style="font-size:70%;">FineTuned StableDiffusion</span></td>
<td id="S3.T7.3.10.7.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.10.7.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib157" title="" class="ltx_ref">157</a><span id="S3.T7.3.10.7.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.10.7.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.10.7.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T7.3.10.7.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.10.7.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.10.7.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.10.7.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.10.7.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.10.7.11" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.10.7.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.10.7.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.10.7.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.10.7.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.10.7.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T7.3.11.8" class="ltx_tr">
<td id="S3.T7.3.11.8.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="7"><span id="S3.T7.3.11.8.1.1" class="ltx_text" style="font-size:70%;">Intra-modal trans.</span></td>
<td id="S3.T7.3.11.8.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="7"><span id="S3.T7.3.11.8.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T7.3.11.8.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.11.8.3.1" class="ltx_text" style="font-size:70%;">Low-to-high density FFDM</span></td>
<td id="S3.T7.3.11.8.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.11.8.4.1" class="ltx_text" style="font-size:70%;">CycleGAN</span></td>
<td id="S3.T7.3.11.8.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.11.8.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib158" title="" class="ltx_ref">158</a><span id="S3.T7.3.11.8.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.11.8.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T7.3.11.8.6.1" class="ltx_text" style="font-size:70%;">det</span></td>
<td id="S3.T7.3.11.8.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.11.8.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T7.3.11.8.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.11.8.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T7.3.11.8.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.11.8.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.11.8.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.11.8.11" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.11.8.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.11.8.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.11.8.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T7.3.11.8.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T7.3.11.8.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T7.1.1" class="ltx_tr">
<td id="S3.T7.1.1.1" class="ltx_td ltx_align_left">
<span id="S3.T7.1.1.1.1" class="ltx_text" style="font-size:70%;">Single-view </span><math id="S3.T7.1.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T7.1.1.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T7.1.1.1.m1.1.1" xref="S3.T7.1.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T7.1.1.1.m1.1b"><ci id="S3.T7.1.1.1.m1.1.1.cmml" xref="S3.T7.1.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T7.1.1.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T7.1.1.1.2" class="ltx_text" style="font-size:70%;"> Two-view</span>
</td>
<td id="S3.T7.1.1.2" class="ltx_td ltx_align_left"><span id="S3.T7.1.1.2.1" class="ltx_text" style="font-size:70%;">CR-GAN</span></td>
<td id="S3.T7.1.1.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.1.1.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib159" title="" class="ltx_ref">159</a><span id="S3.T7.1.1.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.1.1.4" class="ltx_td ltx_align_left"><span id="S3.T7.1.1.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T7.1.1.5" class="ltx_td"></td>
<td id="S3.T7.1.1.6" class="ltx_td"></td>
<td id="S3.T7.1.1.7" class="ltx_td ltx_align_center"><span id="S3.T7.1.1.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.1.1.8" class="ltx_td"></td>
<td id="S3.T7.1.1.9" class="ltx_td"></td>
<td id="S3.T7.1.1.10" class="ltx_td"></td>
<td id="S3.T7.1.1.11" class="ltx_td"></td>
<td id="S3.T7.1.1.12" class="ltx_td ltx_align_center"><span id="S3.T7.1.1.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T7.3.12.9" class="ltx_tr">
<td id="S3.T7.3.12.9.1" class="ltx_td ltx_align_left" rowspan="2"><span id="S3.T7.3.12.9.1.1" class="ltx_text" style="font-size:70%;">Opposite breast synthesis</span></td>
<td id="S3.T7.3.12.9.2" class="ltx_td ltx_align_left" rowspan="2"><span id="S3.T7.3.12.9.2.1" class="ltx_text" style="font-size:70%;">Pix2Pix</span></td>
<td id="S3.T7.3.12.9.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.12.9.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib160" title="" class="ltx_ref">160</a><span id="S3.T7.3.12.9.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.12.9.4" class="ltx_td ltx_align_left"><span id="S3.T7.3.12.9.4.1" class="ltx_text" style="font-size:70%;">det</span></td>
<td id="S3.T7.3.12.9.5" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T7.3.12.9.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.12.9.6" class="ltx_td ltx_align_center"><span id="S3.T7.3.12.9.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.12.9.7" class="ltx_td"></td>
<td id="S3.T7.3.12.9.8" class="ltx_td"></td>
<td id="S3.T7.3.12.9.9" class="ltx_td"></td>
<td id="S3.T7.3.12.9.10" class="ltx_td"></td>
<td id="S3.T7.3.12.9.11" class="ltx_td"></td>
</tr>
<tr id="S3.T7.3.13.10" class="ltx_tr">
<td id="S3.T7.3.13.10.1" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.13.10.1.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib161" title="" class="ltx_ref">161</a><span id="S3.T7.3.13.10.1.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.13.10.2" class="ltx_td ltx_align_left"><span id="S3.T7.3.13.10.2.1" class="ltx_text" style="font-size:70%;">det</span></td>
<td id="S3.T7.3.13.10.3" class="ltx_td ltx_align_center"><span id="S3.T7.3.13.10.3.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T7.3.13.10.4" class="ltx_td ltx_align_center"><span id="S3.T7.3.13.10.4.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T7.3.13.10.5" class="ltx_td"></td>
<td id="S3.T7.3.13.10.6" class="ltx_td"></td>
<td id="S3.T7.3.13.10.7" class="ltx_td"></td>
<td id="S3.T7.3.13.10.8" class="ltx_td"></td>
<td id="S3.T7.3.13.10.9" class="ltx_td"></td>
<td id="S3.T7.3.13.10.10" class="ltx_td"></td>
</tr>
<tr id="S3.T7.3.14.11" class="ltx_tr">
<td id="S3.T7.3.14.11.1" class="ltx_td ltx_align_left"><span id="S3.T7.3.14.11.1.1" class="ltx_text" style="font-size:70%;">Lesion mask</span></td>
<td id="S3.T7.3.14.11.2" class="ltx_td ltx_align_left"><span id="S3.T7.3.14.11.2.1" class="ltx_text" style="font-size:70%;">TMP-GAN</span></td>
<td id="S3.T7.3.14.11.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.14.11.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib162" title="" class="ltx_ref">162</a><span id="S3.T7.3.14.11.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.14.11.4" class="ltx_td ltx_align_left"><span id="S3.T7.3.14.11.4.1" class="ltx_text" style="font-size:70%;">det</span></td>
<td id="S3.T7.3.14.11.5" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T7.3.14.11.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.14.11.6" class="ltx_td"></td>
<td id="S3.T7.3.14.11.7" class="ltx_td"></td>
<td id="S3.T7.3.14.11.8" class="ltx_td"></td>
<td id="S3.T7.3.14.11.9" class="ltx_td"></td>
<td id="S3.T7.3.14.11.10" class="ltx_td"></td>
<td id="S3.T7.3.14.11.11" class="ltx_td"></td>
</tr>
<tr id="S3.T7.2.2" class="ltx_tr">
<td id="S3.T7.2.2.1" class="ltx_td ltx_align_left">
<span id="S3.T7.2.2.1.1" class="ltx_text" style="font-size:70%;">DFM </span><math id="S3.T7.2.2.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T7.2.2.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T7.2.2.1.m1.1.1" xref="S3.T7.2.2.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T7.2.2.1.m1.1b"><ci id="S3.T7.2.2.1.m1.1.1.cmml" xref="S3.T7.2.2.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T7.2.2.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T7.2.2.1.2" class="ltx_text" style="font-size:70%;"> FFDM</span>
</td>
<td id="S3.T7.2.2.2" class="ltx_td ltx_align_left"><span id="S3.T7.2.2.2.1" class="ltx_text" style="font-size:70%;">HRGAN, based on CycleGAN</span></td>
<td id="S3.T7.2.2.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.2.2.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib163" title="" class="ltx_ref">163</a><span id="S3.T7.2.2.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.2.2.4" class="ltx_td ltx_align_left"><span id="S3.T7.2.2.4.1" class="ltx_text" style="font-size:70%;">Seg,calc</span></td>
<td id="S3.T7.2.2.5" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T7.2.2.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.2.2.6" class="ltx_td"></td>
<td id="S3.T7.2.2.7" class="ltx_td"></td>
<td id="S3.T7.2.2.8" class="ltx_td"></td>
<td id="S3.T7.2.2.9" class="ltx_td"></td>
<td id="S3.T7.2.2.10" class="ltx_td"></td>
<td id="S3.T7.2.2.11" class="ltx_td"></td>
</tr>
<tr id="S3.T7.3.3" class="ltx_tr">
<td id="S3.T7.3.3.1" class="ltx_td ltx_align_left ltx_border_b">
<span id="S3.T7.3.3.1.1" class="ltx_text" style="font-size:70%;">Seg masks </span><math id="S3.T7.3.3.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T7.3.3.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T7.3.3.1.m1.1.1" xref="S3.T7.3.3.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T7.3.3.1.m1.1b"><ci id="S3.T7.3.3.1.m1.1.1.cmml" xref="S3.T7.3.3.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T7.3.3.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T7.3.3.1.2" class="ltx_text" style="font-size:70%;"> mass images</span>
</td>
<td id="S3.T7.3.3.2" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T7.3.3.2.1" class="ltx_text" style="font-size:70%;">DCGAN, InfillingGAN</span></td>
<td id="S3.T7.3.3.3" class="ltx_td ltx_align_left ltx_border_b"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T7.3.3.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib164" title="" class="ltx_ref">164</a><span id="S3.T7.3.3.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T7.3.3.4" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T7.3.3.4.1" class="ltx_text" style="font-size:70%;">det</span></td>
<td id="S3.T7.3.3.5" class="ltx_td ltx_align_center ltx_border_b" colspan="2"><span id="S3.T7.3.3.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.3.6" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T7.3.3.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T7.3.3.7" class="ltx_td ltx_border_b"></td>
<td id="S3.T7.3.3.8" class="ltx_td ltx_border_b"></td>
<td id="S3.T7.3.3.9" class="ltx_td ltx_border_b"></td>
<td id="S3.T7.3.3.10" class="ltx_td ltx_border_b"></td>
<td id="S3.T7.3.3.11" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T7.3.3.11.1" class="ltx_text" style="font-size:70%;">*</span></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 7: </span>Overview of all summarized and discussed studies on <span id="S3.T7.8.1" class="ltx_text ltx_font_bold">mammography</span> synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), fidelity (F), diversity (D), qualitative assessment (Q), clinical utility (C) and privacy (P) evaluation. In evaluation of utility (U), R stands for real data, S stands for synthetic data, and A stands for augmented data. * stands for availability of code for original method only. Acronyms: Region of interest (ROI),detection (det), segmentation (seg), calcification (calc), classification (class) conditional GAN (cGAN), complete representation GAN (CR-GAN), texture-constrained multichannel progressive GAN (TMP-GAN), high resolution GAN (HRGAN) </figcaption>
</figure>
<div id="S3.SS3.SSS2.p1" class="ltx_para">
<p id="S3.SS3.SSS2.p1.1" class="ltx_p">The development of AI software products to enhance breast screening outcomes relies heavily on access to well-curated images. Over time, mammography technology has undergone significant evolution, transitioning from Digitized Screen-film Mammograms (DFMs) to Full-Field Digital Mammography (FFDM) and Digital Breast Tomosynthesis (DBT), thus revolutionizing breast cancer screening practices. Initially, DFMs involved X-ray images captured on film and later digitized for analysis. However, the introduction of FFDM marked a crucial advancement by allowing direct digital acquisition of breast images, eliminating the need for film processing and enabling immediate image availability. FFDM offered numerous benefits, including improved image quality, faster acquisition times, and enhanced visualization capabilities for radiologists.
Building upon the advantages of FFDM, DBT emerged as a groundbreaking technique in breast imaging. DBT captures multiple X-ray images from different angles, reconstructing three-dimensional breast tissue images. This innovative approach overcomes the limitations of traditional mammography by reducing tissue overlap and providing clearer, more detailed images, especially in dense breast tissue, but at the expense of higher radiation dose.
Aside from imaging techniques, the mammographic view plays a crucial role in breast cancer detection. While two-view mammography (mediolateral-oblique and craniocaudal views) is the current standard for breast cancer screening, single-view mammography remains in use in certain regions, particularly for screening specific age groups of women. However, the effectiveness of detecting cancer is comparatively lower with single-view mammography than with the two-view approach, as the breast cancer may be visually obscured by overlapping mammary glands in two-dimensional mammograms <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib165" title="" class="ltx_ref">165</a>]</cite>.
Moreover, breast density poses an additional challenge in mammography and breast cancer detection. Dense breasts can obscure or mimic masses, making cancer detection more challenging. Additionally, extremely dense breasts are associated with a higher risk of cancer compared to low-density breasts <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib166" title="" class="ltx_ref">166</a>]</cite>.</p>
</div>
<div id="S3.SS3.SSS2.p2" class="ltx_para">
<p id="S3.SS3.SSS2.p2.1" class="ltx_p">Intra-model translation within the mammography modality emerges as a promising approach to address some of these challenges. One primary hurdle is the scarcity of labeled FFDMs. Despite FFDMs becoming the preferred screening method over DFMs, datasets predominantly remain in DFM format. To overcome this, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib163" title="" class="ltx_ref">163</a>]</cite> proposes a method to convert DFMs into FFDMs. They enhance the CycleGAN with advanced techniques like Pair with Constraint (PWC) training and gradient map input for discriminators, aiming to bridge the gap between available datasets and the preferred FFDM format. To overcome the limitation of using single-view mammograms over two-view mammograms in cancer detection, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib159" title="" class="ltx_ref">159</a>]</cite> develops a method to generate two-view mammograms from single-view images, enhancing cancer detection capabilities. To mitigate the low detection accuracy of dense breasts, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib158" title="" class="ltx_ref">158</a>]</cite> employs CycleGANs to enhance mass detection in dense breasts. They generate synthetic dense mammograms from low-density counterparts, augmenting training data to improve detection accuracy. Through these methods, researchers aim to enhance the effectiveness of DL algorithms in breast cancer screening.</p>
</div>
<div id="S3.SS3.SSS2.p3" class="ltx_para">
<p id="S3.SS3.SSS2.p3.1" class="ltx_p">Generative models also serve various purposes in synthetic mammogram generation, including unconditional, class conditioned, and text conditioned synthesis. Researchers utilize diverse public datasets containing different lesion types and cases (normal, benign, and malignant). Examples include CBIS-DDSM <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib167" title="" class="ltx_ref">167</a>]</cite> and MIAS<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib168" title="" class="ltx_ref">168</a>]</cite> for DFMs, and OPTIMAM <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib169" title="" class="ltx_ref">169</a>]</cite> and INBREAST <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib170" title="" class="ltx_ref">170</a>]</cite> for FFDMs. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib154" title="" class="ltx_ref">154</a>]</cite> use DCGAN to generate non-healthy mammogram patches containing both malignant and benign lesion. To overcome mode collapse in DCGAN and to increase training stability, the authors substitute DCGAN’s binary cross-entropy loss with a Wasserstein distance based loss function, ending up with WGAN-GP based training. The generated lesions patches are used to augment training data for patch classification models in different clinical centers, simulating the idea of sharing generative models instead of private data.
<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib155" title="" class="ltx_ref">155</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib156" title="" class="ltx_ref">156</a>]</cite> utilize GANs for class conditional synthesis, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib155" title="" class="ltx_ref">155</a>]</cite> generating DFMs with diverse conditions (normal, benign, and malignant), while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib156" title="" class="ltx_ref">156</a>]</cite> generate region of interest (ROI) based digital mammograms with different abnormalities. Fine-tuned stable diffusion model was used for text conditioned healthy FFDM generation, specifying the description of the mammogram including view, breast density, breast area and vendor, in addition to a stable diffusion in-painting model used to generate synthetic lesions in desired regions of the normal mammogram <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib157" title="" class="ltx_ref">157</a>]</cite>.</p>
</div>
<div id="S3.SS3.SSS2.p4" class="ltx_para">
<p id="S3.SS3.SSS2.p4.1" class="ltx_p">Evaluation of synthetic mammography data mainly focuses on utility and fidelity, with limited consideration for clinical validation, privacy, diversity metrics, and comparative analyses between models. Qualitative assessment is performed in two papers, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib158" title="" class="ltx_ref">158</a>]</cite> employing a reader study involving two breast radiologists and one surgical oncologist while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib157" title="" class="ltx_ref">157</a>]</cite> involved only one radiologist to determine whether the synthetic images were distinguishable from the real ones. The limited number of experts involved in both studies raise concerns about the validity and statistical significance of the qualitative assessment.</p>
</div>
</section>
<section id="S3.SS3.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.3 </span>Ultrasound</h4>

<figure id="S3.T8" class="ltx_table">
<table id="S3.T8.8" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T8.8.9.1" class="ltx_tr">
<td id="S3.T8.8.9.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T8.8.9.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T8.8.9.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T8.8.9.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T8.8.9.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T8.8.9.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T8.8.9.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T8.8.9.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T8.8.9.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T8.8.9.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T8.8.9.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T8.8.9.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T8.8.9.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T8.8.9.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T8.8.9.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T8.8.9.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T8.8.10.2" class="ltx_tr">
<td id="S3.T8.8.10.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T8.8.10.2.1.1" class="ltx_text" style="font-size:70%;">U</span></td>
<td id="S3.T8.8.10.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T8.8.10.2.2.1" class="ltx_text" style="font-size:70%;">F</span></td>
<td id="S3.T8.8.10.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T8.8.10.2.3.1" class="ltx_text" style="font-size:70%;">D</span></td>
<td id="S3.T8.8.10.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T8.8.10.2.4.1" class="ltx_text" style="font-size:70%;">Q</span></td>
<td id="S3.T8.8.10.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T8.8.10.2.5.1" class="ltx_text" style="font-size:70%;">C</span></td>
<td id="S3.T8.8.10.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T8.8.10.2.6.1" class="ltx_text" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T8.8.11.3" class="ltx_tr">
<td id="S3.T8.8.11.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.8.11.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T8.8.11.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.8.11.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T8.8.12.4" class="ltx_tr">
<td id="S3.T8.8.12.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T8.8.12.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T8.8.12.4.2" class="ltx_td ltx_align_center ltx_border_t">
<span id="S3.T8.8.12.4.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">H:</span><span id="S3.T8.8.12.4.2.2" class="ltx_text" style="font-size:70%;">DM,GAN</span>
</td>
<td id="S3.T8.8.12.4.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.8.12.4.3.1" class="ltx_text" style="font-size:70%;">brain fetal</span></td>
<td id="S3.T8.8.12.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.8.12.4.4.1" class="ltx_text" style="font-size:70%;">DSR-GAN,TB-GAN</span></td>
<td id="S3.T8.8.12.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.12.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib171" title="" class="ltx_ref">171</a><span id="S3.T8.8.12.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.12.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.8.12.4.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T8.8.12.4.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.12.4.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.12.4.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.8.12.4.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.12.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.12.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.12.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.12.4.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.12.4.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.8.12.4.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T8.1.1" class="ltx_tr">
<td id="S3.T8.1.1.2" class="ltx_td ltx_align_center">
<span id="S3.T8.1.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">H:</span><span id="S3.T8.1.1.2.2" class="ltx_text" style="font-size:70%;">GAN,VAE</span>
</td>
<td id="S3.T8.1.1.3" class="ltx_td ltx_align_left"><span id="S3.T8.1.1.3.1" class="ltx_text" style="font-size:70%;">thyroid</span></td>
<td id="S3.T8.1.1.1" class="ltx_td ltx_align_left">
<span id="S3.T8.1.1.1.1" class="ltx_text" style="font-size:70%;">Improved </span><math id="S3.T8.1.1.1.m1.1" class="ltx_Math" alttext="\alpha" display="inline"><semantics id="S3.T8.1.1.1.m1.1a"><mi mathsize="70%" id="S3.T8.1.1.1.m1.1.1" xref="S3.T8.1.1.1.m1.1.1.cmml">α</mi><annotation-xml encoding="MathML-Content" id="S3.T8.1.1.1.m1.1b"><ci id="S3.T8.1.1.1.m1.1.1.cmml" xref="S3.T8.1.1.1.m1.1.1">𝛼</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T8.1.1.1.m1.1c">\alpha</annotation></semantics></math><span id="S3.T8.1.1.1.2" class="ltx_text" style="font-size:70%;">-WGAN-GP</span>
</td>
<td id="S3.T8.1.1.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.1.1.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib172" title="" class="ltx_ref">172</a><span id="S3.T8.1.1.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.1.1.5" class="ltx_td ltx_align_left"><span id="S3.T8.1.1.5.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T8.1.1.6" class="ltx_td ltx_align_center"><span id="S3.T8.1.1.6.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T8.1.1.7" class="ltx_td ltx_align_center"><span id="S3.T8.1.1.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T8.1.1.8" class="ltx_td ltx_align_center"><span id="S3.T8.1.1.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.1.1.9" class="ltx_td"></td>
<td id="S3.T8.1.1.10" class="ltx_td"></td>
<td id="S3.T8.1.1.11" class="ltx_td"></td>
<td id="S3.T8.1.1.12" class="ltx_td"></td>
<td id="S3.T8.1.1.13" class="ltx_td"></td>
</tr>
<tr id="S3.T8.8.13.5" class="ltx_tr">
<td id="S3.T8.8.13.5.1" class="ltx_td ltx_align_center" rowspan="4"><span id="S3.T8.8.13.5.1.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T8.8.13.5.2" class="ltx_td ltx_align_left"><span id="S3.T8.8.13.5.2.1" class="ltx_text" style="font-size:70%;">fetal brain</span></td>
<td id="S3.T8.8.13.5.3" class="ltx_td ltx_align_left"><span id="S3.T8.8.13.5.3.1" class="ltx_text" style="font-size:70%;">Stylegan2-ada</span></td>
<td id="S3.T8.8.13.5.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.13.5.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib173" title="" class="ltx_ref">173</a><span id="S3.T8.8.13.5.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.13.5.5" class="ltx_td ltx_align_left"><span id="S3.T8.8.13.5.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T8.8.13.5.6" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T8.8.13.5.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.13.5.7" class="ltx_td ltx_align_center"><span id="S3.T8.8.13.5.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.13.5.8" class="ltx_td"></td>
<td id="S3.T8.8.13.5.9" class="ltx_td"></td>
<td id="S3.T8.8.13.5.10" class="ltx_td"></td>
<td id="S3.T8.8.13.5.11" class="ltx_td"></td>
<td id="S3.T8.8.13.5.12" class="ltx_td ltx_align_center"><span id="S3.T8.8.13.5.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T8.8.14.6" class="ltx_tr">
<td id="S3.T8.8.14.6.1" class="ltx_td ltx_align_left"><span id="S3.T8.8.14.6.1.1" class="ltx_text" style="font-size:70%;">breast</span></td>
<td id="S3.T8.8.14.6.2" class="ltx_td ltx_align_left"><span id="S3.T8.8.14.6.2.1" class="ltx_text" style="font-size:70%;">StyleGAN2 variants</span></td>
<td id="S3.T8.8.14.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.14.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib174" title="" class="ltx_ref">174</a><span id="S3.T8.8.14.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.14.6.4" class="ltx_td ltx_align_left"><span id="S3.T8.8.14.6.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T8.8.14.6.5" class="ltx_td ltx_align_center"><span id="S3.T8.8.14.6.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T8.8.14.6.6" class="ltx_td ltx_align_center"><span id="S3.T8.8.14.6.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T8.8.14.6.7" class="ltx_td ltx_align_center"><span id="S3.T8.8.14.6.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.14.6.8" class="ltx_td"></td>
<td id="S3.T8.8.14.6.9" class="ltx_td"></td>
<td id="S3.T8.8.14.6.10" class="ltx_td"></td>
<td id="S3.T8.8.14.6.11" class="ltx_td"></td>
<td id="S3.T8.8.14.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T8.8.15.7" class="ltx_tr">
<td id="S3.T8.8.15.7.1" class="ltx_td ltx_align_left"><span id="S3.T8.8.15.7.1.1" class="ltx_text" style="font-size:70%;">liver</span></td>
<td id="S3.T8.8.15.7.2" class="ltx_td ltx_align_left"><span id="S3.T8.8.15.7.2.1" class="ltx_text" style="font-size:70%;">StackGAN</span></td>
<td id="S3.T8.8.15.7.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.15.7.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib175" title="" class="ltx_ref">175</a><span id="S3.T8.8.15.7.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.15.7.4" class="ltx_td ltx_align_left"><span id="S3.T8.8.15.7.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T8.8.15.7.5" class="ltx_td ltx_align_center"><span id="S3.T8.8.15.7.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T8.8.15.7.6" class="ltx_td ltx_align_center"><span id="S3.T8.8.15.7.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T8.8.15.7.7" class="ltx_td ltx_align_center"><span id="S3.T8.8.15.7.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.15.7.8" class="ltx_td"></td>
<td id="S3.T8.8.15.7.9" class="ltx_td ltx_align_center"><span id="S3.T8.8.15.7.9.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T8.8.15.7.10" class="ltx_td"></td>
<td id="S3.T8.8.15.7.11" class="ltx_td"></td>
<td id="S3.T8.8.15.7.12" class="ltx_td"></td>
</tr>
<tr id="S3.T8.8.16.8" class="ltx_tr">
<td id="S3.T8.8.16.8.1" class="ltx_td ltx_align_left"><span id="S3.T8.8.16.8.1.1" class="ltx_text" style="font-size:70%;">Breast</span></td>
<td id="S3.T8.8.16.8.2" class="ltx_td ltx_align_left"><span id="S3.T8.8.16.8.2.1" class="ltx_text" style="font-size:70%;">TripleGAN</span></td>
<td id="S3.T8.8.16.8.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.16.8.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib176" title="" class="ltx_ref">176</a><span id="S3.T8.8.16.8.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.16.8.4" class="ltx_td ltx_align_left"><span id="S3.T8.8.16.8.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T8.8.16.8.5" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T8.8.16.8.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.16.8.6" class="ltx_td"></td>
<td id="S3.T8.8.16.8.7" class="ltx_td"></td>
<td id="S3.T8.8.16.8.8" class="ltx_td ltx_align_center"><span id="S3.T8.8.16.8.8.1" class="ltx_text" style="font-size:70%;">✓(2)</span></td>
<td id="S3.T8.8.16.8.9" class="ltx_td"></td>
<td id="S3.T8.8.16.8.10" class="ltx_td"></td>
<td id="S3.T8.8.16.8.11" class="ltx_td"></td>
</tr>
<tr id="S3.T8.8.17.9" class="ltx_tr">
<td id="S3.T8.8.17.9.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T8.8.17.9.1.1" class="ltx_text" style="font-size:70%;">Class or Label Cond.</span></td>
<td id="S3.T8.8.17.9.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T8.8.17.9.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T8.8.17.9.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.8.17.9.3.1" class="ltx_text" style="font-size:70%;">Muscles</span></td>
<td id="S3.T8.8.17.9.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.8.17.9.4.1" class="ltx_text" style="font-size:70%;">GAN-CA</span></td>
<td id="S3.T8.8.17.9.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.17.9.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib177" title="" class="ltx_ref">177</a><span id="S3.T8.8.17.9.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.17.9.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.8.17.9.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T8.8.17.9.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T8.8.17.9.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.17.9.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.17.9.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.17.9.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.17.9.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.17.9.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.8.17.9.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.8.17.9.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T8.2.2" class="ltx_tr">
<td id="S3.T8.2.2.1" class="ltx_td ltx_align_left">
<span id="S3.T8.2.2.1.1" class="ltx_text" style="font-size:70%;">nodule mask </span><math id="S3.T8.2.2.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T8.2.2.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T8.2.2.1.m1.1.1" xref="S3.T8.2.2.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T8.2.2.1.m1.1b"><ci id="S3.T8.2.2.1.m1.1.1.cmml" xref="S3.T8.2.2.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T8.2.2.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T8.2.2.1.2" class="ltx_text" style="font-size:70%;"> US</span>
</td>
<td id="S3.T8.2.2.2" class="ltx_td ltx_align_left"><span id="S3.T8.2.2.2.1" class="ltx_text" style="font-size:70%;">Phased GAN</span></td>
<td id="S3.T8.2.2.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.2.2.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib178" title="" class="ltx_ref">178</a><span id="S3.T8.2.2.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.2.2.4" class="ltx_td ltx_align_left"><span id="S3.T8.2.2.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T8.2.2.5" class="ltx_td ltx_align_center"><span id="S3.T8.2.2.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T8.2.2.6" class="ltx_td ltx_align_center"><span id="S3.T8.2.2.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T8.2.2.7" class="ltx_td"></td>
<td id="S3.T8.2.2.8" class="ltx_td"></td>
<td id="S3.T8.2.2.9" class="ltx_td"></td>
<td id="S3.T8.2.2.10" class="ltx_td"></td>
<td id="S3.T8.2.2.11" class="ltx_td"></td>
<td id="S3.T8.2.2.12" class="ltx_td"></td>
</tr>
<tr id="S3.T8.8.18.10" class="ltx_tr">
<td id="S3.T8.8.18.10.1" class="ltx_td ltx_align_center"><span id="S3.T8.8.18.10.1.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T8.8.18.10.2" class="ltx_td ltx_align_left">
<table id="S3.T8.8.18.10.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T8.8.18.10.2.1.1" class="ltx_tr">
<td id="S3.T8.8.18.10.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T8.8.18.10.2.1.1.1.1" class="ltx_text" style="font-size:70%;">cardiac anatomical</span></td>
</tr>
<tr id="S3.T8.8.18.10.2.1.2" class="ltx_tr">
<td id="S3.T8.8.18.10.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T8.8.18.10.2.1.2.1.1" class="ltx_text" style="font-size:70%;">semantic label maps</span></td>
</tr>
</table>
</td>
<td id="S3.T8.8.18.10.3" class="ltx_td ltx_align_left">
<table id="S3.T8.8.18.10.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T8.8.18.10.3.1.1" class="ltx_tr">
<td id="S3.T8.8.18.10.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T8.8.18.10.3.1.1.1.1" class="ltx_text" style="font-size:70%;">specifically semantic</span></td>
</tr>
<tr id="S3.T8.8.18.10.3.1.2" class="ltx_tr">
<td id="S3.T8.8.18.10.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T8.8.18.10.3.1.2.1.1" class="ltx_text" style="font-size:70%;">DDPM</span></td>
</tr>
</table>
</td>
<td id="S3.T8.8.18.10.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.18.10.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib179" title="" class="ltx_ref">179</a><span id="S3.T8.8.18.10.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.18.10.5" class="ltx_td ltx_align_left"><span id="S3.T8.8.18.10.5.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T8.8.18.10.6" class="ltx_td ltx_align_center"><span id="S3.T8.8.18.10.6.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T8.8.18.10.7" class="ltx_td ltx_align_center"><span id="S3.T8.8.18.10.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T8.8.18.10.8" class="ltx_td"></td>
<td id="S3.T8.8.18.10.9" class="ltx_td"></td>
<td id="S3.T8.8.18.10.10" class="ltx_td"></td>
<td id="S3.T8.8.18.10.11" class="ltx_td"></td>
<td id="S3.T8.8.18.10.12" class="ltx_td"></td>
<td id="S3.T8.8.18.10.13" class="ltx_td ltx_align_center"><span id="S3.T8.8.18.10.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T8.3.3" class="ltx_tr">
<td id="S3.T8.3.3.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="4"><span id="S3.T8.3.3.2.1" class="ltx_text" style="font-size:70%;">Intra-modal trans.</span></td>
<td id="S3.T8.3.3.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="4"><span id="S3.T8.3.3.3.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T8.3.3.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S3.T8.3.3.1.1" class="ltx_text" style="font-size:70%;">B-mode <math id="S3.T8.3.3.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T8.3.3.1.1.m1.1a"><mo stretchy="false" id="S3.T8.3.3.1.1.m1.1.1" xref="S3.T8.3.3.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T8.3.3.1.1.m1.1b"><ci id="S3.T8.3.3.1.1.m1.1.1.cmml" xref="S3.T8.3.3.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T8.3.3.1.1.m1.1c">\rightarrow</annotation></semantics></math> EUS</span></td>
<td id="S3.T8.3.3.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.3.3.4.1" class="ltx_text" style="font-size:70%;">U-net based gen</span></td>
<td id="S3.T8.3.3.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.3.3.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib180" title="" class="ltx_ref">180</a><span id="S3.T8.3.3.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.3.3.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.3.3.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T8.3.3.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T8.3.3.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.3.3.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.3.3.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.3.3.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.3.3.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.3.3.10.1" class="ltx_text" style="font-size:70%;">✓(2)</span></td>
<td id="S3.T8.3.3.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.3.3.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.3.3.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.3.3.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T8.8.19.11" class="ltx_tr">
<td id="S3.T8.8.19.11.1" class="ltx_td ltx_align_left"><span id="S3.T8.8.19.11.1.1" class="ltx_text" style="font-size:70%;">AUE-net:Pix2Pix based</span></td>
<td id="S3.T8.8.19.11.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.19.11.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib181" title="" class="ltx_ref">181</a><span id="S3.T8.8.19.11.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.19.11.3" class="ltx_td ltx_align_left"><span id="S3.T8.8.19.11.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T8.8.19.11.4" class="ltx_td"></td>
<td id="S3.T8.8.19.11.5" class="ltx_td"></td>
<td id="S3.T8.8.19.11.6" class="ltx_td ltx_align_center"><span id="S3.T8.8.19.11.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.19.11.7" class="ltx_td"></td>
<td id="S3.T8.8.19.11.8" class="ltx_td ltx_align_center"><span id="S3.T8.8.19.11.8.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T8.8.19.11.9" class="ltx_td"></td>
<td id="S3.T8.8.19.11.10" class="ltx_td"></td>
<td id="S3.T8.8.19.11.11" class="ltx_td"></td>
</tr>
<tr id="S3.T8.4.4" class="ltx_tr">
<td id="S3.T8.4.4.1" class="ltx_td ltx_align_left">
<span id="S3.T8.4.4.1.1" class="ltx_text" style="font-size:70%;">FC sketch </span><math id="S3.T8.4.4.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T8.4.4.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T8.4.4.1.m1.1.1" xref="S3.T8.4.4.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T8.4.4.1.m1.1b"><ci id="S3.T8.4.4.1.m1.1.1.cmml" xref="S3.T8.4.4.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T8.4.4.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T8.4.4.1.2" class="ltx_text" style="font-size:70%;"> FC views</span>
</td>
<td id="S3.T8.4.4.2" class="ltx_td ltx_align_left"><span id="S3.T8.4.4.2.1" class="ltx_text" style="font-size:70%;">PSFFGAN</span></td>
<td id="S3.T8.4.4.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.4.4.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib182" title="" class="ltx_ref">182</a><span id="S3.T8.4.4.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.4.4.4" class="ltx_td ltx_align_left"><span id="S3.T8.4.4.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T8.4.4.5" class="ltx_td"></td>
<td id="S3.T8.4.4.6" class="ltx_td"></td>
<td id="S3.T8.4.4.7" class="ltx_td ltx_align_center"><span id="S3.T8.4.4.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.4.4.8" class="ltx_td"></td>
<td id="S3.T8.4.4.9" class="ltx_td ltx_align_center"><span id="S3.T8.4.4.9.1" class="ltx_text" style="font-size:70%;">✓(2)</span></td>
<td id="S3.T8.4.4.10" class="ltx_td"></td>
<td id="S3.T8.4.4.11" class="ltx_td"></td>
<td id="S3.T8.4.4.12" class="ltx_td"></td>
</tr>
<tr id="S3.T8.8.20.12" class="ltx_tr">
<td id="S3.T8.8.20.12.1" class="ltx_td ltx_align_left">
<table id="S3.T8.8.20.12.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T8.8.20.12.1.1.1" class="ltx_tr">
<td id="S3.T8.8.20.12.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T8.8.20.12.1.1.1.1.1" class="ltx_text" style="font-size:70%;">lung, hip joint, and ovary-</span></td>
</tr>
<tr id="S3.T8.8.20.12.1.1.2" class="ltx_tr">
<td id="S3.T8.8.20.12.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T8.8.20.12.1.1.2.1.1" class="ltx_text" style="font-size:70%;">sketch guided</span></td>
</tr>
</table>
</td>
<td id="S3.T8.8.20.12.2" class="ltx_td ltx_align_left"><span id="S3.T8.8.20.12.2.1" class="ltx_text" style="font-size:70%;">PGGAN</span></td>
<td id="S3.T8.8.20.12.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.20.12.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib183" title="" class="ltx_ref">183</a><span id="S3.T8.8.20.12.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.20.12.4" class="ltx_td ltx_align_left"><span id="S3.T8.8.20.12.4.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T8.8.20.12.5" class="ltx_td ltx_align_center"><span id="S3.T8.8.20.12.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T8.8.20.12.6" class="ltx_td ltx_align_center"><span id="S3.T8.8.20.12.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T8.8.20.12.7" class="ltx_td ltx_align_center"><span id="S3.T8.8.20.12.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.20.12.8" class="ltx_td"></td>
<td id="S3.T8.8.20.12.9" class="ltx_td"></td>
<td id="S3.T8.8.20.12.10" class="ltx_td"></td>
<td id="S3.T8.8.20.12.11" class="ltx_td"></td>
<td id="S3.T8.8.20.12.12" class="ltx_td ltx_align_center"><span id="S3.T8.8.20.12.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T8.5.5" class="ltx_tr">
<td id="S3.T8.5.5.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="5"><span id="S3.T8.5.5.2.1" class="ltx_text" style="font-size:70%;">Inter-modal trans.</span></td>
<td id="S3.T8.5.5.3" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.5.5.3.1" class="ltx_text" style="font-size:70%;">VAE</span></td>
<td id="S3.T8.5.5.1" class="ltx_td ltx_align_left ltx_border_t">
<span id="S3.T8.5.5.1.1" class="ltx_text" style="font-size:70%;">T2 MR </span><math id="S3.T8.5.5.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T8.5.5.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T8.5.5.1.m1.1.1" xref="S3.T8.5.5.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T8.5.5.1.m1.1b"><ci id="S3.T8.5.5.1.m1.1.1.cmml" xref="S3.T8.5.5.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T8.5.5.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T8.5.5.1.2" class="ltx_text" style="font-size:70%;"> iUS</span>
</td>
<td id="S3.T8.5.5.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.5.5.4.1" class="ltx_text" style="font-size:70%;">MHVAE</span></td>
<td id="S3.T8.5.5.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.5.5.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib184" title="" class="ltx_ref">184</a><span id="S3.T8.5.5.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.5.5.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T8.5.5.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T8.5.5.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.5.5.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.5.5.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.5.5.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.5.5.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.5.5.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.5.5.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.5.5.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T8.5.5.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T8.5.5.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T8.6.6" class="ltx_tr">
<td id="S3.T8.6.6.2" class="ltx_td ltx_align_center ltx_border_b" rowspan="4"><span id="S3.T8.6.6.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T8.6.6.1" class="ltx_td ltx_align_left">
<span id="S3.T8.6.6.1.1" class="ltx_text" style="font-size:70%;">pMR </span><math id="S3.T8.6.6.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T8.6.6.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T8.6.6.1.m1.1.1" xref="S3.T8.6.6.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T8.6.6.1.m1.1b"><ci id="S3.T8.6.6.1.m1.1.1.cmml" xref="S3.T8.6.6.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T8.6.6.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T8.6.6.1.2" class="ltx_text" style="font-size:70%;"> iUS</span>
</td>
<td id="S3.T8.6.6.3" class="ltx_td ltx_align_left"><span id="S3.T8.6.6.3.1" class="ltx_text" style="font-size:70%;">ApGAN</span></td>
<td id="S3.T8.6.6.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.6.6.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib185" title="" class="ltx_ref">185</a><span id="S3.T8.6.6.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.6.6.5" class="ltx_td ltx_align_left"><span id="S3.T8.6.6.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T8.6.6.6" class="ltx_td ltx_align_center"><span id="S3.T8.6.6.6.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T8.6.6.7" class="ltx_td ltx_align_center"><span id="S3.T8.6.6.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T8.6.6.8" class="ltx_td ltx_align_center"><span id="S3.T8.6.6.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.6.6.9" class="ltx_td"></td>
<td id="S3.T8.6.6.10" class="ltx_td ltx_align_center"><span id="S3.T8.6.6.10.1" class="ltx_text" style="font-size:70%;">✓(5)</span></td>
<td id="S3.T8.6.6.11" class="ltx_td ltx_align_center"><span id="S3.T8.6.6.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.6.6.12" class="ltx_td"></td>
<td id="S3.T8.6.6.13" class="ltx_td"></td>
</tr>
<tr id="S3.T8.7.7" class="ltx_tr">
<td id="S3.T8.7.7.1" class="ltx_td ltx_align_left" rowspan="2"><span id="S3.T8.7.7.1.1" class="ltx_text" style="font-size:70%;">CT <math id="S3.T8.7.7.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T8.7.7.1.1.m1.1a"><mo stretchy="false" id="S3.T8.7.7.1.1.m1.1.1" xref="S3.T8.7.7.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T8.7.7.1.1.m1.1b"><ci id="S3.T8.7.7.1.1.m1.1.1.cmml" xref="S3.T8.7.7.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T8.7.7.1.1.m1.1c">\rightarrow</annotation></semantics></math> US</span></td>
<td id="S3.T8.7.7.2" class="ltx_td ltx_align_left"><span id="S3.T8.7.7.2.1" class="ltx_text" style="font-size:70%;">MR-CycleGAN</span></td>
<td id="S3.T8.7.7.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.7.7.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib186" title="" class="ltx_ref">186</a><span id="S3.T8.7.7.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.7.7.4" class="ltx_td ltx_align_left"><span id="S3.T8.7.7.4.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T8.7.7.5" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T8.7.7.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.7.7.6" class="ltx_td"></td>
<td id="S3.T8.7.7.7" class="ltx_td"></td>
<td id="S3.T8.7.7.8" class="ltx_td"></td>
<td id="S3.T8.7.7.9" class="ltx_td"></td>
<td id="S3.T8.7.7.10" class="ltx_td"></td>
<td id="S3.T8.7.7.11" class="ltx_td"></td>
</tr>
<tr id="S3.T8.8.21.13" class="ltx_tr">
<td id="S3.T8.8.21.13.1" class="ltx_td ltx_align_left"><span id="S3.T8.8.21.13.1.1" class="ltx_text" style="font-size:70%;">CycleGAN</span></td>
<td id="S3.T8.8.21.13.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.21.13.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib187" title="" class="ltx_ref">187</a><span id="S3.T8.8.21.13.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.21.13.3" class="ltx_td ltx_align_left"><span id="S3.T8.8.21.13.3.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T8.8.21.13.4" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T8.8.21.13.4.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.21.13.5" class="ltx_td ltx_align_center"><span id="S3.T8.8.21.13.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.21.13.6" class="ltx_td"></td>
<td id="S3.T8.8.21.13.7" class="ltx_td"></td>
<td id="S3.T8.8.21.13.8" class="ltx_td"></td>
<td id="S3.T8.8.21.13.9" class="ltx_td"></td>
<td id="S3.T8.8.21.13.10" class="ltx_td"></td>
</tr>
<tr id="S3.T8.8.8" class="ltx_tr">
<td id="S3.T8.8.8.1" class="ltx_td ltx_align_left ltx_border_b">
<span id="S3.T8.8.8.1.1" class="ltx_text" style="font-size:70%;">CT AMs </span><math id="S3.T8.8.8.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T8.8.8.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T8.8.8.1.m1.1.1" xref="S3.T8.8.8.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T8.8.8.1.m1.1b"><ci id="S3.T8.8.8.1.m1.1.1.cmml" xref="S3.T8.8.8.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T8.8.8.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T8.8.8.1.2" class="ltx_text" style="font-size:70%;"> US</span>
</td>
<td id="S3.T8.8.8.2" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T8.8.8.2.1" class="ltx_text" style="font-size:70%;">3D Pix2Pix</span></td>
<td id="S3.T8.8.8.3" class="ltx_td ltx_align_left ltx_border_b"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T8.8.8.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib188" title="" class="ltx_ref">188</a><span id="S3.T8.8.8.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T8.8.8.4" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T8.8.8.4.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T8.8.8.5" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T8.8.8.5.1" class="ltx_text" style="font-size:70%;">S,A</span></td>
<td id="S3.T8.8.8.6" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T8.8.8.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T8.8.8.7" class="ltx_td ltx_border_b"></td>
<td id="S3.T8.8.8.8" class="ltx_td ltx_border_b"></td>
<td id="S3.T8.8.8.9" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T8.8.8.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T8.8.8.10" class="ltx_td ltx_border_b"></td>
<td id="S3.T8.8.8.11" class="ltx_td ltx_border_b"></td>
<td id="S3.T8.8.8.12" class="ltx_td ltx_border_b"></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 8: </span>Overview of all summarized and discussed studies on <span id="S3.T8.13.1" class="ltx_text ltx_font_bold">Ultrasound</span> synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), fidelity (F), diversity (D), qualitative assessment (Q), clinical utility (C) and privacy (P) evaluation. In the Type column, H stands for hybrid approach combining two different models. In evaluation of utility (U), R stands for real data, S stands for synthetic data, and A stands for augmented data. In qualitative evaluation, the number between parenthesis indicate the number of specialists employed, while * stands for visual inspection of visualization plots. Acronyms: generator (gen),Multi-Modal Hierarchical Latent Representation VAE (MHVAE), anatomy preserving GAN (ApGAN), Pseudo-Siamese Feature Fusion Generative Adversarial Network (PSFFGAN), GAN with coordinate attention mechanism (GAN-CA), Diffusion-Super-Resolution-GAN (DSR-GAN), Brightness mode (B-mode), intra-operative US (iUS),Elastography Ultrasound (EUS), four-chamber (FC), anatomical model (AM), preoperative magnetic resonance (pMR), Transformer-based-GAN (TB-GAN), StyleGAN2 variants: StyleGAN2 ADA, StyleGAN2 DiffAug</figcaption>
</figure>
<div id="S3.SS3.SSS3.p1" class="ltx_para">
<p id="S3.SS3.SSS3.p1.1" class="ltx_p">Ultrasound synthesis using generative models spans various organs, including fetal brain, thyroid, breast, liver, muscles, and more, and different ultrasound techniques, such as Brightness Mode (B-mode) ultrasound and Elastography Ultrasound (EUS). B-mode ultrasound, commonly known as conventional ultrasound, is a widely used imaging technique in medical diagnostics. It generates 2D grayscale images of soft tissues, offering detailed anatomical information in a non-invasive manner. Despite its safety and relative comfort for patients, B-mode ultrasound has limitations in accurately assessing tissue stiffness and distinguishing between various types of lesions or abnormalities. To address these limitations, EUS is employed as an advanced technique. It evaluates tissue stiffness or elasticity by measuring the response of tissue to compression or shear waves, providing additional functional information beyond traditional B-mode ultrasound. Elastography often displays tissue stiffness using a color scale. This makes it valuable for assessing tissue pathology and differentiating between benign and malignant lesions. However, EUS may require specialized equipment and training, and its accuracy can be affected by factors such as operator technique and tissue depth <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib189" title="" class="ltx_ref">189</a>]</cite>.</p>
</div>
<div id="S3.SS3.SSS3.p2" class="ltx_para">
<p id="S3.SS3.SSS3.p2.1" class="ltx_p">To generate the synthetic ultrasound under different applications of synthesis, various generative models are employed.
For instance, in unconditional synthesis, a hybrid approach combining diffusion models and GANs was adopted in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib171" title="" class="ltx_ref">171</a>]</cite> to synthesize fetal brain images. This method, known as DSR-GAN, comprises a DDPM followed by a Super-Resolution-GAN. Additionally, StyleSwin, a Transformer-based GAN (TB-GAN), was utilized in this study.The results indicated that the DSR-GAN outperformed the TB-GAN in terms of FID, showcasing the potential of using hybrid generative models for improved image synthesis. Similarly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib172" title="" class="ltx_ref">172</a>]</cite> employed a hybrid approach, combining GANs and VAEs to generate synthetic thyroid ultrasound images. Similar to applications in dermoscopy imaging, different variants of StyleGAN2, like StyleGAN2-ADA and DiffAug, are employed in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib174" title="" class="ltx_ref">174</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib173" title="" class="ltx_ref">173</a>]</cite> to address the issue of limited training data. To generate class-conditioned images, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib173" title="" class="ltx_ref">173</a>]</cite> train a generator for each class separately. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib175" title="" class="ltx_ref">175</a>]</cite>, a two-stage stacked GAN architecture was introduced for synthesizing realistic B-mode liver ultrasound images, with the second stage responsible for enhancing tissue details.</p>
</div>
<div id="S3.SS3.SSS3.p3" class="ltx_para">
<p id="S3.SS3.SSS3.p3.1" class="ltx_p">Models focusing on class and label-conditioned synthesis also exist, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib177" title="" class="ltx_ref">177</a>]</cite> utilizing different classes of the idiopathic inflammatory myopathies disease and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib178" title="" class="ltx_ref">178</a>, <a href="#bib.bib179" title="" class="ltx_ref">179</a>]</cite> conditioning on semantic label maps. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib177" title="" class="ltx_ref">177</a>]</cite>, a GAN with a coordinate attention mechanism (GAN-CA) synthesizes muscle ultrasound images, focusing on disease features rather than muscle contour information by segmenting the data into subsets based on the muscle site. Meanwhile, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib179" title="" class="ltx_ref">179</a>]</cite> generates cardiac synthetic data guided by cardiac anatomical semantic label maps. In breast ultrasound classification, some cases are considered to be hard, as they contain overlapping benign and malignant nodules with mixed characteristics of both. Addressing the challenge of classifying hard cases, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib178" title="" class="ltx_ref">178</a>]</cite> introduces a phased GAN that consists of one generator and two discriminators to generate complex cases. The first discriminator help the generator to generate normal samples, while the second discriminator help the generation of hard samples. The generated hard samples enhance the classifiers’ ability to classify them.</p>
</div>
<div id="S3.SS3.SSS3.p4" class="ltx_para">
<p id="S3.SS3.SSS3.p4.1" class="ltx_p">Intra-modal translation is a common application, with techniques mainly employing GANs to translate sketches into images or B-mode to EUS. To make the structural details of generated images more realistic, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib183" title="" class="ltx_ref">183</a>]</cite> introduce auxiliary sketch guidance into a cGAN, and adopt a PGGAN training strategy to generate high-resolution images. Inspired by the conditional GAN and by integrating the optimization strategy of WGAN-GP, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib182" title="" class="ltx_ref">182</a>]</cite> propose a model to synthesizes fetal four-chamber (FC) views from given sketch images. The generator of the model consists of US image encoder, sketch image decoder, and a US image decoder, while the discriminator is based on PatchGAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib190" title="" class="ltx_ref">190</a>]</cite>.
Generating EUS from conventional B-mode ultrasound, as in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib180" title="" class="ltx_ref">180</a>, <a href="#bib.bib181" title="" class="ltx_ref">181</a>]</cite>, offers a promising solution to address the challenges associated with capturing EUS directly. Because the aim of elasticity generation is to accurately evaluate the degree of elasticity of the tumor, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib180" title="" class="ltx_ref">180</a>]</cite> utilize a local tumor discriminator to determine whether the tumor area is real or fake in addition to the global discriminator used to classify whether the input image is real or fake. Moreover, because the elastography image mainly relies on color to distinguish tissue elasticity , <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib180" title="" class="ltx_ref">180</a>]</cite> utilize color re-balancing module to re-weight L1 loss during training based on the color rarity.<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib181" title="" class="ltx_ref">181</a>]</cite> employs an Pix2Pix based architecture combined with various attention modules and color loss features to generate EUS images from conventional ultrasound images.</p>
</div>
<div id="S3.SS3.SSS3.p5" class="ltx_para">
<p id="S3.SS3.SSS3.p5.1" class="ltx_p">Inter-modal translation plays a significant role in US synthesis, encompassing translations from MR or CT scans to US images. In some cases during surgery, obtaining high-quality intra-operative US (iUS) images with clear anatomical structures can be difficult due to various factors such as tissue deformation, blood flow, and surgical instruments. To address the lack of iUS data, researchers propose generating iUS images using Preoperative Magnetic Resonance (pMR) scans. These approach utilizes various techniques, including VAEs and popular GAN models such as CycleGAN and Pix2Pix. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib184" title="" class="ltx_ref">184</a>]</cite> introduce Multi-Modal Hierarchical Latent Representation VAE (MHVAE), the first multi-modal VAE approach with a hierarchical latent representation for unified medical image synthesis, mainly focusing on translating MR to iUS. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib185" title="" class="ltx_ref">185</a>]</cite> proposed an anatomy preserving GAN (ApGAN) to generate intra-operative ultrasound (Sim-iUS) of liver from pMR. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib188" title="" class="ltx_ref">188</a>]</cite>, 3D Pix2Pix GANs were trained using high-resolution anatomical models derived from CT scans as input.</p>
</div>
<div id="S3.SS3.SSS3.p6" class="ltx_para">
<p id="S3.SS3.SSS3.p6.1" class="ltx_p">The evaluation of synthetic ultrasound images predominantly focuses on utility, fidelity assessment, and qualitative analysis, with a few studies incorporating clinical validation. For instance, in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib185" title="" class="ltx_ref">185</a>]</cite>, clinical assessment encompassed various anatomical structures and focal areas to convince physicians of the authenticity of the synthetic data, including unique features like annular structures of the gallbladder or kidney and other relevant distances. As part of the qualitative assessment, several studies employed visual Turing tests, as seen in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib176" title="" class="ltx_ref">176</a>, <a href="#bib.bib180" title="" class="ltx_ref">180</a>, <a href="#bib.bib182" title="" class="ltx_ref">182</a>, <a href="#bib.bib185" title="" class="ltx_ref">185</a>, <a href="#bib.bib191" title="" class="ltx_ref">191</a>]</cite>. These tests typically involved specialists evaluating the realism of the synthetic images, with a limitation that the majority of studies employ two specialists only, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib185" title="" class="ltx_ref">185</a>]</cite> employed five specialists for a more comprehensive assessment. Moreover, the fidelity and utility of the synthetic data of many proposed models were compared against state-of-the-art synthesis methods, including off-the-shelf models like CycleGAN, Pix2Pix, Interpretable Representation Learning by Information Maximizing Generative Adversarial Nets (InfoGAN) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib192" title="" class="ltx_ref">192</a>]</cite>, GAN, and DCGAN, as well as traditional augmentation methods. Comparisons in most cases are limited by the fact that off-the-shelf models are not optimized for specific medical modalities.</p>
</div>
</section>
<section id="S3.SS3.SSS4" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.4 </span>MRI</h4>

<div id="S3.SS3.SSS4.p1" class="ltx_para">
<p id="S3.SS3.SSS4.p1.1" class="ltx_p">MRI has become a ubiquitous medical imaging tool owing to its remarkable capability to generate a plethora of contrast mechanisms in soft tissues using specific pulse sequences. While acquiring images of the same anatomy with different contrasts offer a comprehensive diagnostic information, acquisition constraints, scan time limitations, inherent noise and artifacts may compromise the integrity of acquired data. In these scenarios, the ability to synthesize missing or corrupted contrast through computational algorithms holds significant promise for enhancing diagnostic utility.</p>
</div>
<div id="S3.SS3.SSS4.p2" class="ltx_para">
<p id="S3.SS3.SSS4.p2.1" class="ltx_p">Within the application of intra-modality translation, various generative models convert images from one appearance (source domain) to another (target domain). In many cases, variants of CycleGAN and Pix2Pix models are widely used for unpaired and paired translations, respectively. The authors show promising results for translating the contrast of one MRI sequence to another in both brain <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib193" title="" class="ltx_ref">193</a>, <a href="#bib.bib194" title="" class="ltx_ref">194</a>, <a href="#bib.bib195" title="" class="ltx_ref">195</a>, <a href="#bib.bib196" title="" class="ltx_ref">196</a>, <a href="#bib.bib197" title="" class="ltx_ref">197</a>]</cite> and cardiac MRI <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib198" title="" class="ltx_ref">198</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib199" title="" class="ltx_ref">199</a>]</cite> proposes a GANs-based approach for inter-modal translation from Brain CT to MRI and cardiac MRI to CT to develop a segmentation model on the target domain without labels.</p>
</div>
<div id="S3.SS3.SSS4.p3" class="ltx_para">
<p id="S3.SS3.SSS4.p3.1" class="ltx_p">Several studies have tackled the issue of missing or corrupted contrasts in multi-contrast brain MRI by using different generative models for the task of contrast-to-contrast translation. The goal of the image synthesis is to generate a missing contrast from other available images, for instance generating a T2-weighted image from T1-weighted image. Most proposed methods rely on GANs with modifications to handle multiple image sources and combine complementary features from images of the same anatomy with different contrasts. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib200" title="" class="ltx_ref">200</a>]</cite> propose a multi-stream GAN approach, named mustGAN that aggregates information across multiple source images and effectively utilizes both the shared and complementary features of multiple source images through a combination of multiple one-to-one streams and a joint many-to-one stream, resulting in superior performance in multi-contrast MRI synthesis compared to existing methods. Similarly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib201" title="" class="ltx_ref">201</a>]</cite> explored the use of the Pix2Pix GAN model for generating multi-modal brain MRI data, aimed at enhancing lesion segmentation and classification tasks, demonstrating that using generated images yielded significantly better performance compared to simply replacing missing images with blank or duplicate ones.</p>
</div>
<div id="S3.SS3.SSS4.p4" class="ltx_para">
<p id="S3.SS3.SSS4.p4.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib202" title="" class="ltx_ref">202</a>]</cite> addresses the challenge of predicting Alzheimer’s disease progression from 3D brain MRI data by proposing a Temporal Recurrent GAN (TR-GAN). The study aims to enable multi-session prediction and synthesis of missing longitudinal data, allowing for a more accurate observation of Alzheimer’s disease progression. The TR-GAN model employs recurrent connections to generate future sessions with variable lengths and incorporates specific modules to encourage the generation of high-quality MRI data. The study demonstrates the effectiveness of TR-GAN in generating future MRI sessions and outperforms other GAN architectures in image quality evaluation metrics such as MSE, MS-SSIM, and PSNR.
<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib203" title="" class="ltx_ref">203</a>]</cite> introduces a 3D context-aware generative adversarial network (CoCa-GAN) for synthesizing multi-modality glioma MRI images. It maps input modalities into a shared feature space to synthesize the target modality and complete tumor segmentation. The method’s contributions include utilizing a common feature space for encoding multi-modality information, employing separate encoders for decomposing each modality into high-level features, and implementing tumor attention for learning lesion-specific representations.</p>
</div>
<div id="S3.SS3.SSS4.p5" class="ltx_para">
<p id="S3.SS3.SSS4.p5.1" class="ltx_p">Label-conditioned generation, where the generative model uses segmentation masks as input, has shown promise for creating images in various MRI modalities. One of the key goals is to address medical data scarcity when training downstream DL models, for instance generating images with labels for the development of a DL-based organ segmentation. This approach offers two advantages: increased control over image generation by conditioning of the anatomy present in the labels and the ability to directly use synthetic data with ground truth labels to augment supervised segmentation tasks. Several frameworks based on the SPADE GAN <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib34" title="" class="ltx_ref">34</a>]</cite> architecture have been developed for generating cardiac MRI images, specifically aimed at tackling data scarcity in multi-vendor settings and augmenting existing datasets for building generalizable segmentation networks <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib204" title="" class="ltx_ref">204</a>, <a href="#bib.bib205" title="" class="ltx_ref">205</a>, <a href="#bib.bib206" title="" class="ltx_ref">206</a>, <a href="#bib.bib207" title="" class="ltx_ref">207</a>, <a href="#bib.bib208" title="" class="ltx_ref">208</a>]</cite>. The authors found that incorporating anatomical variations, such as changes in heart shape during synthesis of cardiac MRI data and tailoring the generation to the downstream task significantly reduced segmentation failures <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib209" title="" class="ltx_ref">209</a>]</cite> and led to the development of a more robust segmentation model <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib206" title="" class="ltx_ref">206</a>]</cite>. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib207" title="" class="ltx_ref">207</a>]</cite>, a novel framework is introduced for disentangling images into spatial anatomy factors and corresponding imaging representations to enable controllable image synthesis. This framework allows combining anatomical factors to create new plausible heart anatomies in synthesized cardiac MR images. The authors propose the concept of a disentangled anatomy arithmetic GAN (DAA-GAN) in the generative model and assess the impact of data augmentation on classification and segmentation tasks. They demonstrate that synthetic data augmentation not only enhances learning on balanced data but also improves model performance on underrepresented classes in both disease classification and segmentation tasks.</p>
</div>
<div id="S3.SS3.SSS4.p6" class="ltx_para">
<p id="S3.SS3.SSS4.p6.1" class="ltx_p">While few studies have explored diffusion models, Med-DDPM <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib210" title="" class="ltx_ref">210</a>]</cite> and Brain-SPADE <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib211" title="" class="ltx_ref">211</a>]</cite> present methods for generating 3D semantic medical images using diffusion models. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib211" title="" class="ltx_ref">211</a>]</cite> demonstrates that the segmentation models trained on synthetic data can perform comparable to those trained on real data, even generalizing to unseen data distributions. However, despite the realistic appearance of Med-DDPM’s generated images, the study found that advanced data augmentation using nnU-Net <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib212" title="" class="ltx_ref">212</a>]</cite> led to better results, suggesting a gap between real and synthetic data quality.</p>
</div>
<div id="S3.SS3.SSS4.p7" class="ltx_para">
<p id="S3.SS3.SSS4.p7.1" class="ltx_p">Researchers have also explored unconditional image generation for various modalities <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib213" title="" class="ltx_ref">213</a>, <a href="#bib.bib214" title="" class="ltx_ref">214</a>, <a href="#bib.bib215" title="" class="ltx_ref">215</a>]</cite>, as well as generation conditioned on additional information like age, gender, brain volume, and time frame <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib60" title="" class="ltx_ref">60</a>, <a href="#bib.bib216" title="" class="ltx_ref">216</a>]</cite>. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib60" title="" class="ltx_ref">60</a>]</cite>, an LDM is proposed to generate synthetic MRI images of the adult human brain, conditioned on factors such as age, gender, ventricular volume, and brain volume. The realism of the synthetic data is evaluated using FID score and diversity with the MS-SSIM for 1000 synthetic samples. The method’s age conditional ability is verified in a brain age prediction task, and the volume conditional ability is analyzed by an external model, demonstrating a high correlation between the obtained volume and the inputted value. Conditional LDMs are also applied in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib217" title="" class="ltx_ref">217</a>]</cite> for multi-sequence and multi-contrast generation of prostate MR images, conditioned on both text and images. The synthesized quality is evaluated by a clinician to identify generated pathology, and a model for lesion identification is developed on the augmented data with synthetic examples.</p>
</div>
<div id="S3.SS3.SSS4.p8" class="ltx_para">
<p id="S3.SS3.SSS4.p8.1" class="ltx_p">Despite generating high quality images, few studies adequately evaluate the usefulness of synthetic data for downstream tasks like improving DL based segmentation or classification models. Many studies solely focus on qualitative evaluation of the synthesis results without usability studies <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib202" title="" class="ltx_ref">202</a>, <a href="#bib.bib218" title="" class="ltx_ref">218</a>, <a href="#bib.bib219" title="" class="ltx_ref">219</a>, <a href="#bib.bib220" title="" class="ltx_ref">220</a>, <a href="#bib.bib221" title="" class="ltx_ref">221</a>]</cite>, highlighting the need for further research to assess the usability and clinical applicability of these generated images. Similarly, there is limited evaluation of image diversity and privacy preserving characteristics of the synthetic MRI data.</p>
</div>
<figure id="S3.T9" class="ltx_table">
<table id="S3.T9.10" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T9.10.11.1" class="ltx_tr">
<td id="S3.T9.10.11.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T9.10.11.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T9.10.11.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T9.10.11.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T9.10.11.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T9.10.11.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T9.10.11.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T9.10.11.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T9.10.11.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T9.10.11.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T9.10.11.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T9.10.11.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T9.10.11.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T9.10.11.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T9.10.11.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T9.10.11.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T9.10.12.2" class="ltx_tr">
<td id="S3.T9.10.12.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T9.10.12.2.1.1" class="ltx_text" style="font-size:70%;">U</span></td>
<td id="S3.T9.10.12.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T9.10.12.2.2.1" class="ltx_text" style="font-size:70%;">F</span></td>
<td id="S3.T9.10.12.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T9.10.12.2.3.1" class="ltx_text" style="font-size:70%;">D</span></td>
<td id="S3.T9.10.12.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T9.10.12.2.4.1" class="ltx_text" style="font-size:70%;">Q</span></td>
<td id="S3.T9.10.12.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T9.10.12.2.5.1" class="ltx_text" style="font-size:70%;">C</span></td>
<td id="S3.T9.10.12.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T9.10.12.2.6.1" class="ltx_text" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T9.10.13.3" class="ltx_tr">
<td id="S3.T9.10.13.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.13.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T9.10.13.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.13.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T9.10.14.4" class="ltx_tr">
<td id="S3.T9.10.14.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T9.10.14.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T9.10.14.4.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.14.4.2.1" class="ltx_text" style="font-size:70%;">GANs</span></td>
<td id="S3.T9.10.14.4.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.14.4.3.1" class="ltx_text" style="font-size:70%;">Brain</span></td>
<td id="S3.T9.10.14.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.14.4.4.1" class="ltx_text" style="font-size:70%;">Pix2Pix, WGANGP</span></td>
<td id="S3.T9.10.14.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.14.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib213" title="" class="ltx_ref">213</a><span id="S3.T9.10.14.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.14.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.14.4.6.1" class="ltx_text" style="font-size:70%;">Age est.</span></td>
<td id="S3.T9.10.14.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.14.4.7.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.14.4.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.14.4.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.14.4.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.14.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.14.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.14.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.14.4.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.14.4.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.14.4.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.15.5" class="ltx_tr">
<td id="S3.T9.10.15.5.1" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T9.10.15.5.1.1" class="ltx_text" style="font-size:70%;">DMs</span></td>
<td id="S3.T9.10.15.5.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.15.5.2.1" class="ltx_text" style="font-size:70%;">Brain T1</span></td>
<td id="S3.T9.10.15.5.3" class="ltx_td ltx_align_left"><span id="S3.T9.10.15.5.3.1" class="ltx_text" style="font-size:70%;">3D-DDPM</span></td>
<td id="S3.T9.10.15.5.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.15.5.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib214" title="" class="ltx_ref">214</a><span id="S3.T9.10.15.5.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.15.5.5" class="ltx_td ltx_align_left"><span id="S3.T9.10.15.5.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.15.5.6" class="ltx_td"></td>
<td id="S3.T9.10.15.5.7" class="ltx_td"></td>
<td id="S3.T9.10.15.5.8" class="ltx_td ltx_align_center"><span id="S3.T9.10.15.5.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.15.5.9" class="ltx_td"></td>
<td id="S3.T9.10.15.5.10" class="ltx_td ltx_align_center"><span id="S3.T9.10.15.5.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.15.5.11" class="ltx_td"></td>
<td id="S3.T9.10.15.5.12" class="ltx_td"></td>
<td id="S3.T9.10.15.5.13" class="ltx_td ltx_align_center"><span id="S3.T9.10.15.5.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.16.6" class="ltx_tr">
<td id="S3.T9.10.16.6.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.16.6.1.1" class="ltx_text" style="font-size:70%;">Knee , PCCTA</span></td>
<td id="S3.T9.10.16.6.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.16.6.2.1" class="ltx_text" style="font-size:70%;">LDM</span></td>
<td id="S3.T9.10.16.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.16.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib215" title="" class="ltx_ref">215</a><span id="S3.T9.10.16.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.16.6.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.16.6.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.16.6.5" class="ltx_td"></td>
<td id="S3.T9.10.16.6.6" class="ltx_td"></td>
<td id="S3.T9.10.16.6.7" class="ltx_td"></td>
<td id="S3.T9.10.16.6.8" class="ltx_td"></td>
<td id="S3.T9.10.16.6.9" class="ltx_td"></td>
<td id="S3.T9.10.16.6.10" class="ltx_td"></td>
<td id="S3.T9.10.16.6.11" class="ltx_td ltx_align_center"><span id="S3.T9.10.16.6.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.16.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.17.7" class="ltx_tr">
<td id="S3.T9.10.17.7.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="10"><span id="S3.T9.10.17.7.1.1" class="ltx_text" style="font-size:70%;">Class or Label Cond.</span></td>
<td id="S3.T9.10.17.7.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T9.10.17.7.2.1" class="ltx_text" style="font-size:70%;">GANs</span></td>
<td id="S3.T9.10.17.7.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.17.7.3.1" class="ltx_text" style="font-size:70%;">Cardiac cine, label</span></td>
<td id="S3.T9.10.17.7.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.17.7.4.1" class="ltx_text" style="font-size:70%;">SPADE GAN</span></td>
<td id="S3.T9.10.17.7.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.17.7.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib205" title="" class="ltx_ref">205</a><span id="S3.T9.10.17.7.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.17.7.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.17.7.6.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.17.7.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.17.7.7.1" class="ltx_text" style="font-size:70%;">R, S, A</span></td>
<td id="S3.T9.10.17.7.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.17.7.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.17.7.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.17.7.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.17.7.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.17.7.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.17.7.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.17.7.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.17.7.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.18.8" class="ltx_tr">
<td id="S3.T9.10.18.8.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.18.8.1.1" class="ltx_text" style="font-size:70%;">Cardiac cine, label</span></td>
<td id="S3.T9.10.18.8.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.18.8.2.1" class="ltx_text" style="font-size:70%;">SDNet</span></td>
<td id="S3.T9.10.18.8.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.18.8.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib207" title="" class="ltx_ref">207</a><span id="S3.T9.10.18.8.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.18.8.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.18.8.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.18.8.5" class="ltx_td"></td>
<td id="S3.T9.10.18.8.6" class="ltx_td"></td>
<td id="S3.T9.10.18.8.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.18.8.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.18.8.8" class="ltx_td"></td>
<td id="S3.T9.10.18.8.9" class="ltx_td"></td>
<td id="S3.T9.10.18.8.10" class="ltx_td"></td>
<td id="S3.T9.10.18.8.11" class="ltx_td"></td>
<td id="S3.T9.10.18.8.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.19.9" class="ltx_tr">
<td id="S3.T9.10.19.9.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.19.9.1.1" class="ltx_text" style="font-size:70%;">Breast</span></td>
<td id="S3.T9.10.19.9.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.19.9.2.1" class="ltx_text" style="font-size:70%;">Pix2PixHD</span></td>
<td id="S3.T9.10.19.9.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.19.9.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib222" title="" class="ltx_ref">222</a><span id="S3.T9.10.19.9.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.19.9.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.19.9.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.19.9.5" class="ltx_td"></td>
<td id="S3.T9.10.19.9.6" class="ltx_td"></td>
<td id="S3.T9.10.19.9.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.19.9.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.19.9.8" class="ltx_td"></td>
<td id="S3.T9.10.19.9.9" class="ltx_td"></td>
<td id="S3.T9.10.19.9.10" class="ltx_td"></td>
<td id="S3.T9.10.19.9.11" class="ltx_td"></td>
<td id="S3.T9.10.19.9.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.20.10" class="ltx_tr">
<td id="S3.T9.10.20.10.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.20.10.1.1" class="ltx_text" style="font-size:70%;">Cardiac LGE, label</span></td>
<td id="S3.T9.10.20.10.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.20.10.2.1" class="ltx_text" style="font-size:70%;">SPADE GAN</span></td>
<td id="S3.T9.10.20.10.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.20.10.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib206" title="" class="ltx_ref">206</a><span id="S3.T9.10.20.10.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.20.10.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.20.10.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.20.10.5" class="ltx_td ltx_align_center"><span id="S3.T9.10.20.10.5.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.20.10.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.20.10.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.20.10.7" class="ltx_td"></td>
<td id="S3.T9.10.20.10.8" class="ltx_td"></td>
<td id="S3.T9.10.20.10.9" class="ltx_td"></td>
<td id="S3.T9.10.20.10.10" class="ltx_td"></td>
<td id="S3.T9.10.20.10.11" class="ltx_td"></td>
<td id="S3.T9.10.20.10.12" class="ltx_td ltx_align_center"><span id="S3.T9.10.20.10.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.21.11" class="ltx_tr">
<td id="S3.T9.10.21.11.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.21.11.1.1" class="ltx_text" style="font-size:70%;">Cardiac cine, label</span></td>
<td id="S3.T9.10.21.11.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.21.11.2.1" class="ltx_text" style="font-size:70%;">SPADE GAN</span></td>
<td id="S3.T9.10.21.11.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.21.11.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib204" title="" class="ltx_ref">204</a><span id="S3.T9.10.21.11.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.21.11.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.21.11.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.21.11.5" class="ltx_td ltx_align_center"><span id="S3.T9.10.21.11.5.1" class="ltx_text" style="font-size:70%;">R, S, A</span></td>
<td id="S3.T9.10.21.11.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.21.11.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.21.11.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.21.11.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.21.11.8" class="ltx_td"></td>
<td id="S3.T9.10.21.11.9" class="ltx_td ltx_align_center"><span id="S3.T9.10.21.11.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.21.11.10" class="ltx_td"></td>
<td id="S3.T9.10.21.11.11" class="ltx_td"></td>
<td id="S3.T9.10.21.11.12" class="ltx_td ltx_align_center"><span id="S3.T9.10.21.11.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.22.12" class="ltx_tr">
<td id="S3.T9.10.22.12.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.22.12.1.1" class="ltx_text" style="font-size:70%;">Cardiac cine, label</span></td>
<td id="S3.T9.10.22.12.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.22.12.2.1" class="ltx_text" style="font-size:70%;">SPADE GAN</span></td>
<td id="S3.T9.10.22.12.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.22.12.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib209" title="" class="ltx_ref">209</a><span id="S3.T9.10.22.12.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.22.12.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.22.12.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.22.12.5" class="ltx_td ltx_align_center"><span id="S3.T9.10.22.12.5.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.22.12.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.22.12.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.22.12.7" class="ltx_td"></td>
<td id="S3.T9.10.22.12.8" class="ltx_td"></td>
<td id="S3.T9.10.22.12.9" class="ltx_td ltx_align_center"><span id="S3.T9.10.22.12.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.22.12.10" class="ltx_td ltx_align_center"><span id="S3.T9.10.22.12.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.22.12.11" class="ltx_td"></td>
<td id="S3.T9.10.22.12.12" class="ltx_td ltx_align_center"><span id="S3.T9.10.22.12.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.23.13" class="ltx_tr">
<td id="S3.T9.10.23.13.1" class="ltx_td ltx_align_center"><span id="S3.T9.10.23.13.1.1" class="ltx_text" style="font-size:70%;">VAE</span></td>
<td id="S3.T9.10.23.13.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.23.13.2.1" class="ltx_text" style="font-size:70%;">Brain lesion</span></td>
<td id="S3.T9.10.23.13.3" class="ltx_td ltx_align_left"><span id="S3.T9.10.23.13.3.1" class="ltx_text" style="font-size:70%;">Progressive VAE</span></td>
<td id="S3.T9.10.23.13.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.23.13.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib197" title="" class="ltx_ref">197</a><span id="S3.T9.10.23.13.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.23.13.5" class="ltx_td ltx_align_left"><span id="S3.T9.10.23.13.5.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.23.13.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.23.13.6.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.23.13.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.23.13.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.23.13.8" class="ltx_td ltx_align_center"><span id="S3.T9.10.23.13.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.23.13.9" class="ltx_td"></td>
<td id="S3.T9.10.23.13.10" class="ltx_td ltx_align_center"><span id="S3.T9.10.23.13.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.23.13.11" class="ltx_td"></td>
<td id="S3.T9.10.23.13.12" class="ltx_td"></td>
<td id="S3.T9.10.23.13.13" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.24.14" class="ltx_tr">
<td id="S3.T9.10.24.14.1" class="ltx_td ltx_align_center"><span id="S3.T9.10.24.14.1.1" class="ltx_text" style="font-size:70%;">VAEs, GANs</span></td>
<td id="S3.T9.10.24.14.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.24.14.2.1" class="ltx_text" style="font-size:70%;">Cardiac cine, label</span></td>
<td id="S3.T9.10.24.14.3" class="ltx_td ltx_align_left"><span id="S3.T9.10.24.14.3.1" class="ltx_text" style="font-size:70%;">VAE, SPADE GANs</span></td>
<td id="S3.T9.10.24.14.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.24.14.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib208" title="" class="ltx_ref">208</a><span id="S3.T9.10.24.14.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.24.14.5" class="ltx_td ltx_align_left"><span id="S3.T9.10.24.14.5.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.24.14.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.24.14.6.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.24.14.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.24.14.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.24.14.8" class="ltx_td"></td>
<td id="S3.T9.10.24.14.9" class="ltx_td"></td>
<td id="S3.T9.10.24.14.10" class="ltx_td ltx_align_center"><span id="S3.T9.10.24.14.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.24.14.11" class="ltx_td ltx_align_center"><span id="S3.T9.10.24.14.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.24.14.12" class="ltx_td"></td>
<td id="S3.T9.10.24.14.13" class="ltx_td ltx_align_center"><span id="S3.T9.10.24.14.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.25.15" class="ltx_tr">
<td id="S3.T9.10.25.15.1" class="ltx_td ltx_align_center"><span id="S3.T9.10.25.15.1.1" class="ltx_text" style="font-size:70%;">DMs</span></td>
<td id="S3.T9.10.25.15.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.25.15.2.1" class="ltx_text" style="font-size:70%;">Brain T1, label</span></td>
<td id="S3.T9.10.25.15.3" class="ltx_td ltx_align_left"><span id="S3.T9.10.25.15.3.1" class="ltx_text" style="font-size:70%;">MED-DDPM</span></td>
<td id="S3.T9.10.25.15.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.25.15.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib210" title="" class="ltx_ref">210</a><span id="S3.T9.10.25.15.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.25.15.5" class="ltx_td ltx_align_left"><span id="S3.T9.10.25.15.5.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.25.15.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.25.15.6.1" class="ltx_text" style="font-size:70%;">R, S, A</span></td>
<td id="S3.T9.10.25.15.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.25.15.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.25.15.8" class="ltx_td ltx_align_center"><span id="S3.T9.10.25.15.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.25.15.9" class="ltx_td"></td>
<td id="S3.T9.10.25.15.10" class="ltx_td ltx_align_center"><span id="S3.T9.10.25.15.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.25.15.11" class="ltx_td"></td>
<td id="S3.T9.10.25.15.12" class="ltx_td"></td>
<td id="S3.T9.10.25.15.13" class="ltx_td ltx_align_center"><span id="S3.T9.10.25.15.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.26.16" class="ltx_tr">
<td id="S3.T9.10.26.16.1" class="ltx_td ltx_align_center"><span id="S3.T9.10.26.16.1.1" class="ltx_text" style="font-size:70%;">DMs, GANs</span></td>
<td id="S3.T9.10.26.16.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.26.16.2.1" class="ltx_text" style="font-size:70%;">Brain T1, FLAIR, label</span></td>
<td id="S3.T9.10.26.16.3" class="ltx_td ltx_align_left"><span id="S3.T9.10.26.16.3.1" class="ltx_text" style="font-size:70%;">brainSPADE</span></td>
<td id="S3.T9.10.26.16.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.26.16.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib211" title="" class="ltx_ref">211</a><span id="S3.T9.10.26.16.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.26.16.5" class="ltx_td ltx_align_left"><span id="S3.T9.10.26.16.5.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.26.16.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.26.16.6.1" class="ltx_text" style="font-size:70%;">R, S, A</span></td>
<td id="S3.T9.10.26.16.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.26.16.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.26.16.8" class="ltx_td"></td>
<td id="S3.T9.10.26.16.9" class="ltx_td"></td>
<td id="S3.T9.10.26.16.10" class="ltx_td"></td>
<td id="S3.T9.10.26.16.11" class="ltx_td"></td>
<td id="S3.T9.10.26.16.12" class="ltx_td"></td>
<td id="S3.T9.10.26.16.13" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.27.17" class="ltx_tr">
<td id="S3.T9.10.27.17.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T9.10.27.17.1.1" class="ltx_text" style="font-size:70%;">Attribute Cond</span></td>
<td id="S3.T9.10.27.17.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.27.17.2.1" class="ltx_text" style="font-size:70%;">DMs</span></td>
<td id="S3.T9.10.27.17.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.27.17.3.1" class="ltx_text" style="font-size:70%;">Brain, Cardiac</span></td>
<td id="S3.T9.10.27.17.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.27.17.4.1" class="ltx_text" style="font-size:70%;">SADM</span></td>
<td id="S3.T9.10.27.17.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.27.17.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib216" title="" class="ltx_ref">216</a><span id="S3.T9.10.27.17.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.27.17.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.10.27.17.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.27.17.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.27.17.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.27.17.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.27.17.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.27.17.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.27.17.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.27.17.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.27.17.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.10.27.17.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.10.27.17.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.28.18" class="ltx_tr">
<td id="S3.T9.10.28.18.1" class="ltx_td ltx_align_center"><span id="S3.T9.10.28.18.1.1" class="ltx_text" style="font-size:70%;">LDM</span></td>
<td id="S3.T9.10.28.18.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.28.18.2.1" class="ltx_text" style="font-size:70%;">Brain</span></td>
<td id="S3.T9.10.28.18.3" class="ltx_td ltx_align_left"><span id="S3.T9.10.28.18.3.1" class="ltx_text" style="font-size:70%;">CLDM</span></td>
<td id="S3.T9.10.28.18.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.28.18.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib60" title="" class="ltx_ref">60</a><span id="S3.T9.10.28.18.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.28.18.5" class="ltx_td ltx_align_left"><span id="S3.T9.10.28.18.5.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T9.10.28.18.6" class="ltx_td"></td>
<td id="S3.T9.10.28.18.7" class="ltx_td"></td>
<td id="S3.T9.10.28.18.8" class="ltx_td ltx_align_center"><span id="S3.T9.10.28.18.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.28.18.9" class="ltx_td ltx_align_center"><span id="S3.T9.10.28.18.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.28.18.10" class="ltx_td"></td>
<td id="S3.T9.10.28.18.11" class="ltx_td"></td>
<td id="S3.T9.10.28.18.12" class="ltx_td"></td>
<td id="S3.T9.10.28.18.13" class="ltx_td"></td>
</tr>
<tr id="S3.T9.2.2" class="ltx_tr">
<td id="S3.T9.2.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="17"><span id="S3.T9.2.2.3.1" class="ltx_text" style="font-size:70%;">Intra-modal trans.</span></td>
<td id="S3.T9.2.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="10"><span id="S3.T9.2.2.4.1" class="ltx_text" style="font-size:70%;">GANs</span></td>
<td id="S3.T9.2.2.2" class="ltx_td ltx_align_left ltx_border_t">
<span id="S3.T9.2.2.2.1" class="ltx_text" style="font-size:70%;">Brain T1</span><math id="S3.T9.1.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.1.1.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.1.1.1.m1.1.1" xref="S3.T9.1.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.1.1.1.m1.1b"><ci id="S3.T9.1.1.1.m1.1.1.cmml" xref="S3.T9.1.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.1.1.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.2.2.2.2" class="ltx_text" style="font-size:70%;">T2, T2</span><math id="S3.T9.2.2.2.m2.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.2.2.2.m2.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.2.2.2.m2.1.1" xref="S3.T9.2.2.2.m2.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.2.2.2.m2.1b"><ci id="S3.T9.2.2.2.m2.1.1.cmml" xref="S3.T9.2.2.2.m2.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.2.2.2.m2.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.2.2.2.3" class="ltx_text" style="font-size:70%;">T1</span>
</td>
<td id="S3.T9.2.2.5" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.2.2.5.1" class="ltx_text" style="font-size:70%;">ST-cGAN</span></td>
<td id="S3.T9.2.2.6" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.2.2.6.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib194" title="" class="ltx_ref">194</a><span id="S3.T9.2.2.6.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.2.2.7" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T9.2.2.7.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.2.2.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.2.2.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.2.2.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T9.2.2.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.2.2.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.2.2.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.2.2.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.2.2.14" class="ltx_td ltx_border_t"></td>
<td id="S3.T9.2.2.15" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T9.10.29.19" class="ltx_tr">
<td id="S3.T9.10.29.19.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.29.19.1.1" class="ltx_text" style="font-size:70%;">Brain missing contrast</span></td>
<td id="S3.T9.10.29.19.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.29.19.2.1" class="ltx_text" style="font-size:70%;">CoCa-GAN</span></td>
<td id="S3.T9.10.29.19.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.29.19.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib203" title="" class="ltx_ref">203</a><span id="S3.T9.10.29.19.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.29.19.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.29.19.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.29.19.5" class="ltx_td ltx_align_center"><span id="S3.T9.10.29.19.5.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.29.19.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.29.19.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.29.19.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.29.19.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.29.19.8" class="ltx_td"></td>
<td id="S3.T9.10.29.19.9" class="ltx_td ltx_align_center"><span id="S3.T9.10.29.19.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.29.19.10" class="ltx_td"></td>
<td id="S3.T9.10.29.19.11" class="ltx_td"></td>
<td id="S3.T9.10.29.19.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.30.20" class="ltx_tr">
<td id="S3.T9.10.30.20.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.30.20.1.1" class="ltx_text" style="font-size:70%;">Brain missing contrast</span></td>
<td id="S3.T9.10.30.20.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.30.20.2.1" class="ltx_text" style="font-size:70%;">LR-cGAN</span></td>
<td id="S3.T9.10.30.20.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.30.20.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib221" title="" class="ltx_ref">221</a><span id="S3.T9.10.30.20.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.30.20.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.30.20.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.30.20.5" class="ltx_td"></td>
<td id="S3.T9.10.30.20.6" class="ltx_td"></td>
<td id="S3.T9.10.30.20.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.30.20.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.30.20.8" class="ltx_td"></td>
<td id="S3.T9.10.30.20.9" class="ltx_td"></td>
<td id="S3.T9.10.30.20.10" class="ltx_td"></td>
<td id="S3.T9.10.30.20.11" class="ltx_td"></td>
<td id="S3.T9.10.30.20.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.31.21" class="ltx_tr">
<td id="S3.T9.10.31.21.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.31.21.1.1" class="ltx_text" style="font-size:70%;">Brain missing contrast</span></td>
<td id="S3.T9.10.31.21.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.31.21.2.1" class="ltx_text" style="font-size:70%;">MustGAN</span></td>
<td id="S3.T9.10.31.21.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.31.21.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib200" title="" class="ltx_ref">200</a><span id="S3.T9.10.31.21.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.31.21.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.31.21.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.31.21.5" class="ltx_td"></td>
<td id="S3.T9.10.31.21.6" class="ltx_td"></td>
<td id="S3.T9.10.31.21.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.31.21.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.31.21.8" class="ltx_td"></td>
<td id="S3.T9.10.31.21.9" class="ltx_td ltx_align_center"><span id="S3.T9.10.31.21.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.31.21.10" class="ltx_td"></td>
<td id="S3.T9.10.31.21.11" class="ltx_td"></td>
<td id="S3.T9.10.31.21.12" class="ltx_td ltx_align_center"><span id="S3.T9.10.31.21.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.32.22" class="ltx_tr">
<td id="S3.T9.10.32.22.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.32.22.1.1" class="ltx_text" style="font-size:70%;">Brain missing sessions</span></td>
<td id="S3.T9.10.32.22.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.32.22.2.1" class="ltx_text" style="font-size:70%;">TR-GAN</span></td>
<td id="S3.T9.10.32.22.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.32.22.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib202" title="" class="ltx_ref">202</a><span id="S3.T9.10.32.22.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.32.22.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.32.22.4.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T9.10.32.22.5" class="ltx_td ltx_align_center"><span id="S3.T9.10.32.22.5.1" class="ltx_text" style="font-size:70%;">R, S, A</span></td>
<td id="S3.T9.10.32.22.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.32.22.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.32.22.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.32.22.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.32.22.8" class="ltx_td"></td>
<td id="S3.T9.10.32.22.9" class="ltx_td"></td>
<td id="S3.T9.10.32.22.10" class="ltx_td"></td>
<td id="S3.T9.10.32.22.11" class="ltx_td"></td>
<td id="S3.T9.10.32.22.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.33.23" class="ltx_tr">
<td id="S3.T9.10.33.23.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.33.23.1.1" class="ltx_text" style="font-size:70%;">Brain different sequences</span></td>
<td id="S3.T9.10.33.23.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.33.23.2.1" class="ltx_text" style="font-size:70%;">Pix2Pix</span></td>
<td id="S3.T9.10.33.23.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.33.23.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib193" title="" class="ltx_ref">193</a><span id="S3.T9.10.33.23.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.33.23.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.33.23.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.33.23.5" class="ltx_td ltx_align_center"><span id="S3.T9.10.33.23.5.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.33.23.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.33.23.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.33.23.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.33.23.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.33.23.8" class="ltx_td"></td>
<td id="S3.T9.10.33.23.9" class="ltx_td ltx_align_center"><span id="S3.T9.10.33.23.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.33.23.10" class="ltx_td"></td>
<td id="S3.T9.10.33.23.11" class="ltx_td"></td>
<td id="S3.T9.10.33.23.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.34.24" class="ltx_tr">
<td id="S3.T9.10.34.24.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.34.24.1.1" class="ltx_text" style="font-size:70%;">Brain missing contrast</span></td>
<td id="S3.T9.10.34.24.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.34.24.2.1" class="ltx_text" style="font-size:70%;">DualMMP-GAN</span></td>
<td id="S3.T9.10.34.24.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.34.24.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib195" title="" class="ltx_ref">195</a><span id="S3.T9.10.34.24.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.34.24.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.34.24.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.34.24.5" class="ltx_td ltx_align_center"><span id="S3.T9.10.34.24.5.1" class="ltx_text" style="font-size:70%;">R, S, A</span></td>
<td id="S3.T9.10.34.24.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.34.24.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.34.24.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.34.24.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.34.24.8" class="ltx_td"></td>
<td id="S3.T9.10.34.24.9" class="ltx_td"></td>
<td id="S3.T9.10.34.24.10" class="ltx_td"></td>
<td id="S3.T9.10.34.24.11" class="ltx_td"></td>
<td id="S3.T9.10.34.24.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.35.25" class="ltx_tr">
<td id="S3.T9.10.35.25.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.35.25.1.1" class="ltx_text" style="font-size:70%;">Brain missing contrast</span></td>
<td id="S3.T9.10.35.25.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.35.25.2.1" class="ltx_text" style="font-size:70%;">Pix2Pix-GAN</span></td>
<td id="S3.T9.10.35.25.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.35.25.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib201" title="" class="ltx_ref">201</a><span id="S3.T9.10.35.25.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.35.25.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.35.25.4.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T9.10.35.25.5" class="ltx_td ltx_align_center"><span id="S3.T9.10.35.25.5.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.35.25.6" class="ltx_td ltx_align_center"><span id="S3.T9.10.35.25.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.35.25.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.35.25.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.35.25.8" class="ltx_td"></td>
<td id="S3.T9.10.35.25.9" class="ltx_td ltx_align_center"><span id="S3.T9.10.35.25.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.35.25.10" class="ltx_td"></td>
<td id="S3.T9.10.35.25.11" class="ltx_td"></td>
<td id="S3.T9.10.35.25.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.3.3" class="ltx_tr">
<td id="S3.T9.3.3.1" class="ltx_td ltx_align_left">
<span id="S3.T9.3.3.1.1" class="ltx_text" style="font-size:70%;">Brain T1, T2, T2c</span><math id="S3.T9.3.3.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.3.3.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.3.3.1.m1.1.1" xref="S3.T9.3.3.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.3.3.1.m1.1b"><ci id="S3.T9.3.3.1.m1.1.1.cmml" xref="S3.T9.3.3.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.3.3.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.3.3.1.2" class="ltx_text" style="font-size:70%;">FLAIR</span>
</td>
<td id="S3.T9.3.3.2" class="ltx_td ltx_align_left"><span id="S3.T9.3.3.2.1" class="ltx_text" style="font-size:70%;">encoder-GANs</span></td>
<td id="S3.T9.3.3.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.3.3.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib196" title="" class="ltx_ref">196</a><span id="S3.T9.3.3.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.3.3.4" class="ltx_td ltx_align_left"><span id="S3.T9.3.3.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.3.3.5" class="ltx_td"></td>
<td id="S3.T9.3.3.6" class="ltx_td"></td>
<td id="S3.T9.3.3.7" class="ltx_td ltx_align_center"><span id="S3.T9.3.3.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.3.3.8" class="ltx_td"></td>
<td id="S3.T9.3.3.9" class="ltx_td"></td>
<td id="S3.T9.3.3.10" class="ltx_td"></td>
<td id="S3.T9.3.3.11" class="ltx_td"></td>
<td id="S3.T9.3.3.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.4.4" class="ltx_tr">
<td id="S3.T9.4.4.1" class="ltx_td ltx_align_left">
<span id="S3.T9.4.4.1.1" class="ltx_text" style="font-size:70%;">Cardiac cine</span><math id="S3.T9.4.4.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.4.4.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.4.4.1.m1.1.1" xref="S3.T9.4.4.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.4.4.1.m1.1b"><ci id="S3.T9.4.4.1.m1.1.1.cmml" xref="S3.T9.4.4.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.4.4.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.4.4.1.2" class="ltx_text" style="font-size:70%;">tagged</span>
</td>
<td id="S3.T9.4.4.2" class="ltx_td ltx_align_left"><span id="S3.T9.4.4.2.1" class="ltx_text" style="font-size:70%;">CycleGAN</span></td>
<td id="S3.T9.4.4.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.4.4.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib198" title="" class="ltx_ref">198</a><span id="S3.T9.4.4.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.4.4.4" class="ltx_td ltx_align_left"><span id="S3.T9.4.4.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.4.4.5" class="ltx_td ltx_align_center"><span id="S3.T9.4.4.5.1" class="ltx_text" style="font-size:70%;">R, S</span></td>
<td id="S3.T9.4.4.6" class="ltx_td ltx_align_center"><span id="S3.T9.4.4.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.4.4.7" class="ltx_td"></td>
<td id="S3.T9.4.4.8" class="ltx_td"></td>
<td id="S3.T9.4.4.9" class="ltx_td"></td>
<td id="S3.T9.4.4.10" class="ltx_td"></td>
<td id="S3.T9.4.4.11" class="ltx_td"></td>
<td id="S3.T9.4.4.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.5.5" class="ltx_tr">
<td id="S3.T9.5.5.2" class="ltx_td ltx_align_center"><span id="S3.T9.5.5.2.1" class="ltx_text" style="font-size:70%;">VAE</span></td>
<td id="S3.T9.5.5.1" class="ltx_td ltx_align_left">
<span id="S3.T9.5.5.1.1" class="ltx_text" style="font-size:70%;">Brain FLAIR,T2</span><math id="S3.T9.5.5.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.5.5.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.5.5.1.m1.1.1" xref="S3.T9.5.5.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.5.5.1.m1.1b"><ci id="S3.T9.5.5.1.m1.1.1.cmml" xref="S3.T9.5.5.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.5.5.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.5.5.1.2" class="ltx_text" style="font-size:70%;">T1</span>
</td>
<td id="S3.T9.5.5.3" class="ltx_td ltx_align_left"><span id="S3.T9.5.5.3.1" class="ltx_text" style="font-size:70%;">VAE</span></td>
<td id="S3.T9.5.5.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.5.5.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib218" title="" class="ltx_ref">218</a><span id="S3.T9.5.5.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.5.5.5" class="ltx_td ltx_align_left"><span id="S3.T9.5.5.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.5.5.6" class="ltx_td"></td>
<td id="S3.T9.5.5.7" class="ltx_td"></td>
<td id="S3.T9.5.5.8" class="ltx_td ltx_align_center"><span id="S3.T9.5.5.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.5.5.9" class="ltx_td"></td>
<td id="S3.T9.5.5.10" class="ltx_td"></td>
<td id="S3.T9.5.5.11" class="ltx_td"></td>
<td id="S3.T9.5.5.12" class="ltx_td"></td>
<td id="S3.T9.5.5.13" class="ltx_td ltx_align_center"><span id="S3.T9.5.5.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.36.26" class="ltx_tr">
<td id="S3.T9.10.36.26.1" class="ltx_td ltx_align_center" rowspan="3"><span id="S3.T9.10.36.26.1.1" class="ltx_text" style="font-size:70%;">DMs</span></td>
<td id="S3.T9.10.36.26.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.36.26.2.1" class="ltx_text" style="font-size:70%;">Brain slice generation</span></td>
<td id="S3.T9.10.36.26.3" class="ltx_td ltx_align_left"><span id="S3.T9.10.36.26.3.1" class="ltx_text" style="font-size:70%;">DDPM</span></td>
<td id="S3.T9.10.36.26.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.36.26.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib223" title="" class="ltx_ref">223</a><span id="S3.T9.10.36.26.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.36.26.5" class="ltx_td ltx_align_left"><span id="S3.T9.10.36.26.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.36.26.6" class="ltx_td"></td>
<td id="S3.T9.10.36.26.7" class="ltx_td"></td>
<td id="S3.T9.10.36.26.8" class="ltx_td ltx_align_center"><span id="S3.T9.10.36.26.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.36.26.9" class="ltx_td"></td>
<td id="S3.T9.10.36.26.10" class="ltx_td"></td>
<td id="S3.T9.10.36.26.11" class="ltx_td"></td>
<td id="S3.T9.10.36.26.12" class="ltx_td"></td>
<td id="S3.T9.10.36.26.13" class="ltx_td ltx_align_center"><span id="S3.T9.10.36.26.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.6.6" class="ltx_tr">
<td id="S3.T9.6.6.1" class="ltx_td ltx_align_left">
<span id="S3.T9.6.6.1.1" class="ltx_text" style="font-size:70%;">Cardiac ED</span><math id="S3.T9.6.6.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.6.6.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.6.6.1.m1.1.1" xref="S3.T9.6.6.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.6.6.1.m1.1b"><ci id="S3.T9.6.6.1.m1.1.1.cmml" xref="S3.T9.6.6.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.6.6.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.6.6.1.2" class="ltx_text" style="font-size:70%;">ES phases</span>
</td>
<td id="S3.T9.6.6.2" class="ltx_td ltx_align_left"><span id="S3.T9.6.6.2.1" class="ltx_text" style="font-size:70%;">DDPM</span></td>
<td id="S3.T9.6.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.6.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib224" title="" class="ltx_ref">224</a><span id="S3.T9.6.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.6.6.4" class="ltx_td ltx_align_left"><span id="S3.T9.6.6.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.6.6.5" class="ltx_td"></td>
<td id="S3.T9.6.6.6" class="ltx_td"></td>
<td id="S3.T9.6.6.7" class="ltx_td ltx_align_center"><span id="S3.T9.6.6.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.6.6.8" class="ltx_td"></td>
<td id="S3.T9.6.6.9" class="ltx_td"></td>
<td id="S3.T9.6.6.10" class="ltx_td"></td>
<td id="S3.T9.6.6.11" class="ltx_td"></td>
<td id="S3.T9.6.6.12" class="ltx_td ltx_align_center"><span id="S3.T9.6.6.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T9.10.37.27" class="ltx_tr">
<td id="S3.T9.10.37.27.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.37.27.1.1" class="ltx_text" style="font-size:70%;">Brain with tumor</span></td>
<td id="S3.T9.10.37.27.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.37.27.2.1" class="ltx_text" style="font-size:70%;">DDPM</span></td>
<td id="S3.T9.10.37.27.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.37.27.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib225" title="" class="ltx_ref">225</a><span id="S3.T9.10.37.27.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.37.27.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.37.27.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.37.27.5" class="ltx_td"></td>
<td id="S3.T9.10.37.27.6" class="ltx_td"></td>
<td id="S3.T9.10.37.27.7" class="ltx_td"></td>
<td id="S3.T9.10.37.27.8" class="ltx_td"></td>
<td id="S3.T9.10.37.27.9" class="ltx_td"></td>
<td id="S3.T9.10.37.27.10" class="ltx_td"></td>
<td id="S3.T9.10.37.27.11" class="ltx_td"></td>
<td id="S3.T9.10.37.27.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.38.28" class="ltx_tr">
<td id="S3.T9.10.38.28.1" class="ltx_td ltx_align_center" rowspan="3"><span id="S3.T9.10.38.28.1.1" class="ltx_text" style="font-size:70%;">LDM</span></td>
<td id="S3.T9.10.38.28.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.38.28.2.1" class="ltx_text" style="font-size:70%;">Brain missing contrast</span></td>
<td id="S3.T9.10.38.28.3" class="ltx_td ltx_align_left"><span id="S3.T9.10.38.28.3.1" class="ltx_text" style="font-size:70%;">CoLa-Diff , LDM</span></td>
<td id="S3.T9.10.38.28.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.38.28.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib219" title="" class="ltx_ref">219</a><span id="S3.T9.10.38.28.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.38.28.5" class="ltx_td ltx_align_left"><span id="S3.T9.10.38.28.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.38.28.6" class="ltx_td"></td>
<td id="S3.T9.10.38.28.7" class="ltx_td"></td>
<td id="S3.T9.10.38.28.8" class="ltx_td ltx_align_center"><span id="S3.T9.10.38.28.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.38.28.9" class="ltx_td"></td>
<td id="S3.T9.10.38.28.10" class="ltx_td ltx_align_center"><span id="S3.T9.10.38.28.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.38.28.11" class="ltx_td"></td>
<td id="S3.T9.10.38.28.12" class="ltx_td"></td>
<td id="S3.T9.10.38.28.13" class="ltx_td"></td>
</tr>
<tr id="S3.T9.8.8" class="ltx_tr">
<td id="S3.T9.8.8.2" class="ltx_td ltx_align_left">
<span id="S3.T9.8.8.2.1" class="ltx_text" style="font-size:70%;">Brain SWI</span><math id="S3.T9.7.7.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.7.7.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.7.7.1.m1.1.1" xref="S3.T9.7.7.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.7.7.1.m1.1b"><ci id="S3.T9.7.7.1.m1.1.1.cmml" xref="S3.T9.7.7.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.7.7.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.8.8.2.2" class="ltx_text" style="font-size:70%;">MRA, T1</span><math id="S3.T9.8.8.2.m2.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.8.8.2.m2.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.8.8.2.m2.1.1" xref="S3.T9.8.8.2.m2.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.8.8.2.m2.1b"><ci id="S3.T9.8.8.2.m2.1.1.cmml" xref="S3.T9.8.8.2.m2.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.8.8.2.m2.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.8.8.2.3" class="ltx_text" style="font-size:70%;">T2</span>
</td>
<td id="S3.T9.8.8.3" class="ltx_td ltx_align_left"><span id="S3.T9.8.8.3.1" class="ltx_text" style="font-size:70%;">LDM</span></td>
<td id="S3.T9.8.8.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.8.8.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib226" title="" class="ltx_ref">226</a><span id="S3.T9.8.8.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.8.8.5" class="ltx_td ltx_align_left"><span id="S3.T9.8.8.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.8.8.6" class="ltx_td"></td>
<td id="S3.T9.8.8.7" class="ltx_td"></td>
<td id="S3.T9.8.8.8" class="ltx_td ltx_align_center"><span id="S3.T9.8.8.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.8.8.9" class="ltx_td"></td>
<td id="S3.T9.8.8.10" class="ltx_td ltx_align_center"><span id="S3.T9.8.8.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.8.8.11" class="ltx_td"></td>
<td id="S3.T9.8.8.12" class="ltx_td"></td>
<td id="S3.T9.8.8.13" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.39.29" class="ltx_tr">
<td id="S3.T9.10.39.29.1" class="ltx_td ltx_align_left"><span id="S3.T9.10.39.29.1.1" class="ltx_text" style="font-size:70%;">Brain missing contrast</span></td>
<td id="S3.T9.10.39.29.2" class="ltx_td ltx_align_left"><span id="S3.T9.10.39.29.2.1" class="ltx_text" style="font-size:70%;">LDM</span></td>
<td id="S3.T9.10.39.29.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.39.29.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib220" title="" class="ltx_ref">220</a><span id="S3.T9.10.39.29.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.39.29.4" class="ltx_td ltx_align_left"><span id="S3.T9.10.39.29.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T9.10.39.29.5" class="ltx_td"></td>
<td id="S3.T9.10.39.29.6" class="ltx_td"></td>
<td id="S3.T9.10.39.29.7" class="ltx_td ltx_align_center"><span id="S3.T9.10.39.29.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T9.10.39.29.8" class="ltx_td"></td>
<td id="S3.T9.10.39.29.9" class="ltx_td"></td>
<td id="S3.T9.10.39.29.10" class="ltx_td"></td>
<td id="S3.T9.10.39.29.11" class="ltx_td"></td>
<td id="S3.T9.10.39.29.12" class="ltx_td"></td>
</tr>
<tr id="S3.T9.10.10" class="ltx_tr">
<td id="S3.T9.10.10.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T9.10.10.3.1" class="ltx_text" style="font-size:70%;">Inter-modal trans.</span></td>
<td id="S3.T9.10.10.4" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T9.10.10.4.1" class="ltx_text" style="font-size:70%;">GANs</span></td>
<td id="S3.T9.10.10.2" class="ltx_td ltx_align_left ltx_border_b ltx_border_t">
<table id="S3.T9.10.10.2.2" class="ltx_tabular ltx_align_middle">
<tr id="S3.T9.9.9.1.1.1" class="ltx_tr">
<td id="S3.T9.9.9.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">
<span id="S3.T9.9.9.1.1.1.1.1" class="ltx_text" style="font-size:70%;">Brain CT</span><math id="S3.T9.9.9.1.1.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.9.9.1.1.1.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.9.9.1.1.1.1.m1.1.1" xref="S3.T9.9.9.1.1.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.9.9.1.1.1.1.m1.1b"><ci id="S3.T9.9.9.1.1.1.1.m1.1.1.cmml" xref="S3.T9.9.9.1.1.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.9.9.1.1.1.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.9.9.1.1.1.1.2" class="ltx_text" style="font-size:70%;">MRI,</span>
</td>
</tr>
<tr id="S3.T9.10.10.2.2.2" class="ltx_tr">
<td id="S3.T9.10.10.2.2.2.1" class="ltx_td ltx_nopad_r ltx_align_left">
<span id="S3.T9.10.10.2.2.2.1.1" class="ltx_text" style="font-size:70%;">Cardiac MRI</span><math id="S3.T9.10.10.2.2.2.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.10.10.2.2.2.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T9.10.10.2.2.2.1.m1.1.1" xref="S3.T9.10.10.2.2.2.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.10.10.2.2.2.1.m1.1b"><ci id="S3.T9.10.10.2.2.2.1.m1.1.1.cmml" xref="S3.T9.10.10.2.2.2.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.10.10.2.2.2.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T9.10.10.2.2.2.1.2" class="ltx_text" style="font-size:70%;">CT</span>
</td>
</tr>
</table>
</td>
<td id="S3.T9.10.10.5" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T9.10.10.5.1" class="ltx_text" style="font-size:70%;">DDA-GAN</span></td>
<td id="S3.T9.10.10.6" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T9.10.10.6.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib199" title="" class="ltx_ref">199</a><span id="S3.T9.10.10.6.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T9.10.10.7" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T9.10.10.7.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T9.10.10.8" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T9.10.10.8.1" class="ltx_text" style="font-size:70%;">R, A</span></td>
<td id="S3.T9.10.10.9" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T9.10.10.9.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T9.10.10.10" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T9.10.10.11" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T9.10.10.12" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T9.10.10.13" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T9.10.10.14" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T9.10.10.15" class="ltx_td ltx_border_b ltx_border_t"></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 9: </span>Overview of all summarized and discussed studies on <span id="S3.T9.24.1" class="ltx_text ltx_font_bold">MRI</span> synthesis with information about the technology utilized, the downstream task, the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), fidelity (F), diversity (D), visual quality (Q), clinical utility (C) and privacy (P) evaluation. Missing contrast generation can include any combination of available source(s) of contrast to target contrast, such as T1, T2 <math id="S3.T9.14.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.14.m1.1b"><mo stretchy="false" id="S3.T9.14.m1.1.1" xref="S3.T9.14.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.14.m1.1c"><ci id="S3.T9.14.m1.1.1.cmml" xref="S3.T9.14.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.14.m1.1d">\rightarrow</annotation></semantics></math> FLAIR, T1, PD <math id="S3.T9.15.m2.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.15.m2.1b"><mo stretchy="false" id="S3.T9.15.m2.1.1" xref="S3.T9.15.m2.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.15.m2.1c"><ci id="S3.T9.15.m2.1.1.cmml" xref="S3.T9.15.m2.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.15.m2.1d">\rightarrow</annotation></semantics></math> T2, FLAIR, T2 <math id="S3.T9.16.m3.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T9.16.m3.1b"><mo stretchy="false" id="S3.T9.16.m3.1.1" xref="S3.T9.16.m3.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T9.16.m3.1c"><ci id="S3.T9.16.m3.1.1.cmml" xref="S3.T9.16.m3.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T9.16.m3.1d">\rightarrow</annotation></semantics></math>T1, etc. </figcaption>
</figure>
</section>
<section id="S3.SS3.SSS5" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.5 </span>CT</h4>

<div id="S3.SS3.SSS5.p1" class="ltx_para">
<p id="S3.SS3.SSS5.p1.1" class="ltx_p">Radiation therapy relies on CT as the primary imaging tool for treatment planning, offering precise tissue geometry visualization and electron density conversion crucial for dose calculations <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib227" title="" class="ltx_ref">227</a>]</cite>. Although MRI complements CT by providing superior soft-tissue contrast without ionizing radiation, it lacks the electron density data necessary for accurate dose calculations <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib228" title="" class="ltx_ref">228</a>]</cite>. In addition, cone-beam CT is commonly utilized for patient positioning and monitoring before, during, or after dose delivery <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib229" title="" class="ltx_ref">229</a>]</cite>. Integrating information from both modalities involves registering MRI to CT, a process prone to systematic errors affecting treatment accuracy <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib228" title="" class="ltx_ref">228</a>, <a href="#bib.bib230" title="" class="ltx_ref">230</a>]</cite>. MR-only radiotherapy seeks to overcome registration issues <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib231" title="" class="ltx_ref">231</a>]</cite>, yet the absence of tissue attenuation data in MRI requires methods to convert MR data for precise dose calculations equivalent to CT.
Similarly, various techniques aim to enhance CBCT quality by generating synthetic CT from alternative imaging modalities <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib232" title="" class="ltx_ref">232</a>]</cite>. Despite its significance in image-guided adaptive radiation therapy, CBCT faces limited usage due to scatter noise and truncated projections, leading to reconstruction challenges and artifacts like shading, streaking, and cupping <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib233" title="" class="ltx_ref">233</a>]</cite>. Consequently, CBCT is rarely employed for online plan adaptation. Conversion of CBCT to CT could enable precise dose computation and enhance the quality of image-guided adaptive radiation therapy for patients. In addition, efforts to reduce radiation dose in CT imaging have gained significant attention due to concerns about patient exposure <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib234" title="" class="ltx_ref">234</a>]</cite>. Various methods have been explored, including adjustments to tube current, voltage, and x-ray intensity, as well as protocol revisions. However, these approaches often compromise image quality, impacting diagnostic accuracy. Consequently, recent focus has shifted towards developing low-dose CT (LDCT) restoration algorithms using diverse DL techniques.</p>
</div>
<div id="S3.SS3.SSS5.p2" class="ltx_para">
<p id="S3.SS3.SSS5.p2.1" class="ltx_p">As such, most proposed approaches focus on image-to-image translation for synthetic CT generation (Table <a href="#S3.T10" title="Table 10 ‣ 3.3.5 CT ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a>), mostly from MR images. Some methods extend to generating CBCT or low-dose CT images. While these typically require extensive paired or unpaired MRI and CT data, unsupervised methods are still rare. However, the emergence of diffusion-based models has inspired interest in unsupervised and text-to-image approaches, offering potential for realistic tissue structure generation without the dependency on source images.</p>
</div>
<div id="S3.SS3.SSS5.p3" class="ltx_para">
<p id="S3.SS3.SSS5.p3.1" class="ltx_p">Generative methods play a crucial role in synthesizing realistic CT scans, frequently employed for data augmentation across various tasks like segmentation and classification. Using a PGGAN, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib235" title="" class="ltx_ref">235</a>]</cite> augments data for segmenting prostate and organs in 3D pelvic CT data. Building on this, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib236" title="" class="ltx_ref">236</a>]</cite> employs PGGANs for unsupervised synthesis of high-resolution CT scans, yielding outputs that are often indistinguishable from real CT scans, as demonstrated in a comprehensive Visual Turing test study. However, challenges persist in accurately generating smaller anatomical structures, indicating limitations in current methods due to homogeneous data training. Addressing this, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib65" title="" class="ltx_ref">65</a>]</cite> introduces a hybrid model using a DDPM with a Swin-transformer-based network for abdomen and pelvic CT scan generation, achieving state-of-the-art performance, in terms of synthetic image quality and diversity, compared to existing generative models such as WGAN, PGGAN, SGAN and DDPM. Challenges exist in extending this approach to 3D synthesis due to complexity and resource-intensive requirements.</p>
</div>
<div id="S3.SS3.SSS5.p4" class="ltx_para">
<p id="S3.SS3.SSS5.p4.1" class="ltx_p">Numerous applications in CT imaging revolve around generating synthetic lung and chest CT images, such as <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib237" title="" class="ltx_ref">237</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib238" title="" class="ltx_ref">238</a>]</cite>, utilizing a 3D CT-GAN, enabling a gradual generation of CT volumes by generating sub-component slices and slabs, reducing the GPU memory requirements. Interest in clinical applications of unsupervised and/or unconditional generative methods has expanded to include COVID-19 detection <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib239" title="" class="ltx_ref">239</a>, <a href="#bib.bib240" title="" class="ltx_ref">240</a>, <a href="#bib.bib241" title="" class="ltx_ref">241</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib239" title="" class="ltx_ref">239</a>]</cite> presents a 3D cycleGAN framework tailored for generating non-contrast CT images from contrast-enhanced ones, leveraging a sizable dataset of more than 1000 COVID-19 scans. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib241" title="" class="ltx_ref">241</a>]</cite>, the authors explore COVID-19 detection in CT scans using vision transformers, augmented with synthetic images from SAGAN-ResNet. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib240" title="" class="ltx_ref">240</a>]</cite> delves into the challenges of employing large diffusion models, proposing a lightweight variant with a U-Net architecture and step information injection, stressing the importance of specific operations in encoder-decoder models to maintain synthetic image quality.</p>
</div>
<div id="S3.SS3.SSS5.p5" class="ltx_para">
<p id="S3.SS3.SSS5.p5.1" class="ltx_p">Recent advancements in exploring conditional models have emphasized the injection of greater control into the generation process, resulting in the production of higher quality synthetic images. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib242" title="" class="ltx_ref">242</a>]</cite> leverages a VAE to compress image space, and then conditions a latent diffusion model on its output to synthesize CT images for the denoising of LDCT images. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib243" title="" class="ltx_ref">243</a>]</cite> introduces MedGen3D, a novel diffusion-based generative framework, which utilizes conditioning on mask sequences generated by a Multi-Condition Diffusion Probabilistic Model (MC-DPM), considering both the position or direction in which sequences are generates (forward or backward), as well as the sequence content. These conditions guide the synthesis of realistic medical images aligned with masks while maintaining spatial consistency across adjacent slices.</p>
</div>
<div id="S3.SS3.SSS5.p6" class="ltx_para">
<p id="S3.SS3.SSS5.p6.1" class="ltx_p">Conditioning on label maps has been thoroughly explored to guide the accurate anatomical synthesis of tissues in chest and brain <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib244" title="" class="ltx_ref">244</a>]</cite>, liver <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib245" title="" class="ltx_ref">245</a>]</cite>, as well as lung CT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib246" title="" class="ltx_ref">246</a>, <a href="#bib.bib247" title="" class="ltx_ref">247</a>, <a href="#bib.bib248" title="" class="ltx_ref">248</a>, <a href="#bib.bib249" title="" class="ltx_ref">249</a>, <a href="#bib.bib67" title="" class="ltx_ref">67</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib244" title="" class="ltx_ref">244</a>]</cite> benchmarks variations of PGGAN and StyleGAN for synthetic medical image generation, revealing insights into their performance under variable conditions, such as the variation in label combinations, sample size and spatial resolution. Reducing class numbers improves synthetic data quality, but caution is needed to avoid overfitting on rare classes, while refining label conditioning mechanisms is crucial for enhancing overall predictive performance. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib245" title="" class="ltx_ref">245</a>]</cite> presents a 3D GAN framework for liver CT image synthesis using vascular segmentation masks and region-based weight-balancing with a stable Multiple Gradient Descent Algorithm (MGDA). While promising, challenges remain in accurately labeling vascular structures, underscoring the need for more explicit information for precise 3D liver image synthesis. Variations of the Pix2Pix model are commonly used for synthesizing lung images with diverse pathological cases <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib246" title="" class="ltx_ref">246</a>]</cite>, generating free-form lesion images from tumor sketches using StyleGAN concepts for accurate style translation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib67" title="" class="ltx_ref">67</a>]</cite>, and lung nodule synthesis <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib247" title="" class="ltx_ref">247</a>]</cite> for data augmentation, employing window-guided semantic learning to control the generation of smaller semantic features. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib248" title="" class="ltx_ref">248</a>]</cite> investigates InfoGAN for generating shape-controlled tumor images and evaluates their potential for histological classification of lung cancer with CT scans. InfoGAN outperforms WGAN in controlling tumor size and chest wall presence, improving image features. However, both GANs still fall short of fully capturing the entire distribution of real images, likely due to differences in generated tumor shapes. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib249" title="" class="ltx_ref">249</a>]</cite> explores Semantic Diffusion Models (SDM) for generating high-quality pulmonary CT images compared to other SOTA approaches, such as SPADE GANs, while prioritizing lung nodule detection.</p>
</div>
<div id="S3.SS3.SSS5.p7" class="ltx_para">
<p id="S3.SS3.SSS5.p7.1" class="ltx_p">Text-to-image synthesis is gaining traction for diversifying data augmentation, offering text-controlled generation of synthetic images to address challenges like rare case simulation and privacy concerns by creating images without the need for sharing sensitive or identifiable data, typically sourced from real images. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib250" title="" class="ltx_ref">250</a>]</cite> utilizes radiology reports for high-resolution 3D medical image synthesis, starting with low-resolution synthesis guided by tokenized reports and up-scaled using a super-resolution module. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib251" title="" class="ltx_ref">251</a>]</cite> introduces a text-conditional CT generation method, combining a pre-trained language model, transformer-based 3D chest CT generation, and a super-resolution diffusion model. Limitations include the lack of benchmarks, reliance on 2D super-resolution, and considerations for model generalization, applicability to non-chest tissues, and clinical relevance evaluation.</p>
</div>
<figure id="S3.T10" class="ltx_table">
<table id="S3.T10.11" class="ltx_tabular ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T10.11.12.1" class="ltx_tr">
<td id="S3.T10.11.12.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T10.11.12.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T10.11.12.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T10.11.12.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T10.11.12.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T10.11.12.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T10.11.12.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T10.11.12.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T10.11.12.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T10.11.12.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T10.11.12.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T10.11.12.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T10.11.12.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T10.11.12.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T10.11.12.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T10.11.12.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T10.11.13.2" class="ltx_tr">
<td id="S3.T10.11.13.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T10.11.13.2.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">U</span></td>
<td id="S3.T10.11.13.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T10.11.13.2.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">F</span></td>
<td id="S3.T10.11.13.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T10.11.13.2.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">D</span></td>
<td id="S3.T10.11.13.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T10.11.13.2.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Q</span></td>
<td id="S3.T10.11.13.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T10.11.13.2.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">C</span></td>
<td id="S3.T10.11.13.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T10.11.13.2.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T10.11.14.3" class="ltx_tr">
<td id="S3.T10.11.14.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.14.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T10.11.14.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.14.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T10.11.15.4" class="ltx_tr">
<td id="S3.T10.11.15.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="8"><span id="S3.T10.11.15.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T10.11.15.4.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T10.11.15.4.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T10.11.15.4.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.15.4.3.1" class="ltx_text" style="font-size:70%;">Pelvic CT</span></td>
<td id="S3.T10.11.15.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.15.4.4.1" class="ltx_text" style="font-size:70%;">PGGAN</span></td>
<td id="S3.T10.11.15.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.15.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib235" title="" class="ltx_ref">235</a><span id="S3.T10.11.15.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.15.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.15.4.6.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T10.11.15.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.15.4.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T10.11.15.4.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.15.4.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.15.4.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.15.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.15.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.15.4.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.15.4.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.15.4.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.15.4.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T10.11.16.5" class="ltx_tr">
<td id="S3.T10.11.16.5.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.16.5.1.1" class="ltx_text" style="font-size:70%;">Whole body CT</span></td>
<td id="S3.T10.11.16.5.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.16.5.2.1" class="ltx_text" style="font-size:70%;">PGGAN</span></td>
<td id="S3.T10.11.16.5.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.16.5.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib236" title="" class="ltx_ref">236</a><span id="S3.T10.11.16.5.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.16.5.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.16.5.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.16.5.5" class="ltx_td"></td>
<td id="S3.T10.11.16.5.6" class="ltx_td"></td>
<td id="S3.T10.11.16.5.7" class="ltx_td"></td>
<td id="S3.T10.11.16.5.8" class="ltx_td"></td>
<td id="S3.T10.11.16.5.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.16.5.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.16.5.10" class="ltx_td"></td>
<td id="S3.T10.11.16.5.11" class="ltx_td"></td>
<td id="S3.T10.11.16.5.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.17.6" class="ltx_tr">
<td id="S3.T10.11.17.6.1" class="ltx_td ltx_align_left" rowspan="2"><span id="S3.T10.11.17.6.1.1" class="ltx_text" style="font-size:70%;">Lung CT</span></td>
<td id="S3.T10.11.17.6.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.17.6.2.1" class="ltx_text" style="font-size:70%;">CT-SGAN with BiLSTM</span></td>
<td id="S3.T10.11.17.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.17.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib237" title="" class="ltx_ref">237</a><span id="S3.T10.11.17.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.17.6.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.17.6.4.1" class="ltx_text" style="font-size:70%;">Det.</span></td>
<td id="S3.T10.11.17.6.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.17.6.5.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T10.11.17.6.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.17.6.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.17.6.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.17.6.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.17.6.8" class="ltx_td"></td>
<td id="S3.T10.11.17.6.9" class="ltx_td"></td>
<td id="S3.T10.11.17.6.10" class="ltx_td"></td>
<td id="S3.T10.11.17.6.11" class="ltx_td"></td>
<td id="S3.T10.11.17.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.18.7" class="ltx_tr">
<td id="S3.T10.11.18.7.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.18.7.1.1" class="ltx_text" style="font-size:70%;">3D CT-GAN</span></td>
<td id="S3.T10.11.18.7.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.18.7.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib238" title="" class="ltx_ref">238</a><span id="S3.T10.11.18.7.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.18.7.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.18.7.3.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T10.11.18.7.4" class="ltx_td ltx_align_center"><span id="S3.T10.11.18.7.4.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T10.11.18.7.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.18.7.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.18.7.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.18.7.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.18.7.7" class="ltx_td"></td>
<td id="S3.T10.11.18.7.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.18.7.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.18.7.9" class="ltx_td"></td>
<td id="S3.T10.11.18.7.10" class="ltx_td"></td>
<td id="S3.T10.11.18.7.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.19.8" class="ltx_tr">
<td id="S3.T10.11.19.8.1" class="ltx_td ltx_align_left" rowspan="2"><span id="S3.T10.11.19.8.1.1" class="ltx_text" style="font-size:70%;">COVID19</span></td>
<td id="S3.T10.11.19.8.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.19.8.2.1" class="ltx_text" style="font-size:70%;">SA-GAN ResNet</span></td>
<td id="S3.T10.11.19.8.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.19.8.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib241" title="" class="ltx_ref">241</a><span id="S3.T10.11.19.8.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.19.8.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.19.8.4.1" class="ltx_text" style="font-size:70%;">Det.</span></td>
<td id="S3.T10.11.19.8.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.19.8.5.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T10.11.19.8.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.19.8.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.19.8.7" class="ltx_td"></td>
<td id="S3.T10.11.19.8.8" class="ltx_td"></td>
<td id="S3.T10.11.19.8.9" class="ltx_td"></td>
<td id="S3.T10.11.19.8.10" class="ltx_td"></td>
<td id="S3.T10.11.19.8.11" class="ltx_td"></td>
<td id="S3.T10.11.19.8.12" class="ltx_td ltx_align_center"><span id="S3.T10.11.19.8.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T10.11.20.9" class="ltx_tr">
<td id="S3.T10.11.20.9.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.20.9.1.1" class="ltx_text" style="font-size:70%;">3D patch-based cycle-GAN</span></td>
<td id="S3.T10.11.20.9.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.20.9.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib239" title="" class="ltx_ref">239</a><span id="S3.T10.11.20.9.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.20.9.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.20.9.3.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T10.11.20.9.4" class="ltx_td ltx_align_center"><span id="S3.T10.11.20.9.4.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.20.9.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.20.9.5.1" class="ltx_text" style="font-size:70%;">R, S</span></td>
<td id="S3.T10.11.20.9.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.20.9.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.20.9.7" class="ltx_td"></td>
<td id="S3.T10.11.20.9.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.20.9.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.20.9.9" class="ltx_td"></td>
<td id="S3.T10.11.20.9.10" class="ltx_td"></td>
<td id="S3.T10.11.20.9.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.21.10" class="ltx_tr">
<td id="S3.T10.11.21.10.1" class="ltx_td ltx_align_center" rowspan="2"><span id="S3.T10.11.21.10.1.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T10.11.21.10.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.21.10.2.1" class="ltx_text" style="font-size:70%;">Abd. &amp; pelvic CT</span></td>
<td id="S3.T10.11.21.10.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.21.10.3.1" class="ltx_text" style="font-size:70%;">MT-DDPM</span></td>
<td id="S3.T10.11.21.10.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.21.10.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib65" title="" class="ltx_ref">65</a><span id="S3.T10.11.21.10.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.21.10.5" class="ltx_td ltx_align_left"><span id="S3.T10.11.21.10.5.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T10.11.21.10.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.21.10.6.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T10.11.21.10.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.21.10.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.21.10.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.21.10.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.21.10.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.21.10.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.21.10.10" class="ltx_td ltx_align_center"><span id="S3.T10.11.21.10.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.21.10.11" class="ltx_td"></td>
<td id="S3.T10.11.21.10.12" class="ltx_td"></td>
<td id="S3.T10.11.21.10.13" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.22.11" class="ltx_tr">
<td id="S3.T10.11.22.11.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.22.11.1.1" class="ltx_text" style="font-size:70%;">COVID19</span></td>
<td id="S3.T10.11.22.11.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.22.11.2.1" class="ltx_text" style="font-size:70%;">U-Net DM</span></td>
<td id="S3.T10.11.22.11.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.22.11.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib240" title="" class="ltx_ref">240</a><span id="S3.T10.11.22.11.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.22.11.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.22.11.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.22.11.5" class="ltx_td"></td>
<td id="S3.T10.11.22.11.6" class="ltx_td"></td>
<td id="S3.T10.11.22.11.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.22.11.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.22.11.8" class="ltx_td"></td>
<td id="S3.T10.11.22.11.9" class="ltx_td"></td>
<td id="S3.T10.11.22.11.10" class="ltx_td"></td>
<td id="S3.T10.11.22.11.11" class="ltx_td"></td>
<td id="S3.T10.11.22.11.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.23.12" class="ltx_tr">
<td id="S3.T10.11.23.12.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="7"><span id="S3.T10.11.23.12.1.1" class="ltx_text" style="font-size:70%;">Class or Label Cond.</span></td>
<td id="S3.T10.11.23.12.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T10.11.23.12.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T10.11.23.12.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.23.12.3.1" class="ltx_text" style="font-size:70%;">Brain CT</span></td>
<td id="S3.T10.11.23.12.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.23.12.4.1" class="ltx_text" style="font-size:70%;">PGGAN, cpd-GAN</span></td>
<td id="S3.T10.11.23.12.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.23.12.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib244" title="" class="ltx_ref">244</a><span id="S3.T10.11.23.12.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.23.12.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.23.12.6.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T10.11.23.12.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.23.12.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T10.11.23.12.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.23.12.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.23.12.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.23.12.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.23.12.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.23.12.11" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.23.12.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.23.12.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.23.12.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.23.12.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.23.12.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.23.12.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T10.11.24.13" class="ltx_tr">
<td id="S3.T10.11.24.13.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.24.13.1.1" class="ltx_text" style="font-size:70%;">Liver CT</span></td>
<td id="S3.T10.11.24.13.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.24.13.2.1" class="ltx_text" style="font-size:70%;">U-Net patch-GAN</span></td>
<td id="S3.T10.11.24.13.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.24.13.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib245" title="" class="ltx_ref">245</a><span id="S3.T10.11.24.13.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.24.13.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.24.13.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.24.13.5" class="ltx_td"></td>
<td id="S3.T10.11.24.13.6" class="ltx_td"></td>
<td id="S3.T10.11.24.13.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.24.13.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.24.13.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.24.13.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.24.13.9" class="ltx_td"></td>
<td id="S3.T10.11.24.13.10" class="ltx_td ltx_align_center"><span id="S3.T10.11.24.13.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.24.13.11" class="ltx_td"></td>
<td id="S3.T10.11.24.13.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.25.14" class="ltx_tr">
<td id="S3.T10.11.25.14.1" class="ltx_td ltx_align_left" rowspan="5"><span id="S3.T10.11.25.14.1.1" class="ltx_text" style="font-size:70%;">Lung CT</span></td>
<td id="S3.T10.11.25.14.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.25.14.2.1" class="ltx_text" style="font-size:70%;">Pix2Pix</span></td>
<td id="S3.T10.11.25.14.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.25.14.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib246" title="" class="ltx_ref">246</a><span id="S3.T10.11.25.14.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.25.14.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.25.14.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.25.14.5" class="ltx_td"></td>
<td id="S3.T10.11.25.14.6" class="ltx_td"></td>
<td id="S3.T10.11.25.14.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.25.14.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.25.14.8" class="ltx_td"></td>
<td id="S3.T10.11.25.14.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.25.14.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.25.14.10" class="ltx_td"></td>
<td id="S3.T10.11.25.14.11" class="ltx_td"></td>
<td id="S3.T10.11.25.14.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.26.15" class="ltx_tr">
<td id="S3.T10.11.26.15.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.26.15.1.1" class="ltx_text" style="font-size:70%;">Info GAN</span></td>
<td id="S3.T10.11.26.15.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.26.15.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib248" title="" class="ltx_ref">248</a><span id="S3.T10.11.26.15.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.26.15.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.26.15.3.1" class="ltx_text" style="font-size:70%;">Class.*</span></td>
<td id="S3.T10.11.26.15.4" class="ltx_td ltx_align_center"><span id="S3.T10.11.26.15.4.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T10.11.26.15.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.26.15.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.26.15.6" class="ltx_td"></td>
<td id="S3.T10.11.26.15.7" class="ltx_td"></td>
<td id="S3.T10.11.26.15.8" class="ltx_td"></td>
<td id="S3.T10.11.26.15.9" class="ltx_td"></td>
<td id="S3.T10.11.26.15.10" class="ltx_td"></td>
<td id="S3.T10.11.26.15.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.27.16" class="ltx_tr">
<td id="S3.T10.11.27.16.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.27.16.1.1" class="ltx_text" style="font-size:70%;">Style Pix2Pix</span></td>
<td id="S3.T10.11.27.16.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.27.16.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib67" title="" class="ltx_ref">67</a><span id="S3.T10.11.27.16.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.27.16.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.27.16.3.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T10.11.27.16.4" class="ltx_td ltx_align_center"><span id="S3.T10.11.27.16.4.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T10.11.27.16.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.27.16.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.27.16.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.27.16.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.27.16.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.27.16.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.27.16.8" class="ltx_td"></td>
<td id="S3.T10.11.27.16.9" class="ltx_td"></td>
<td id="S3.T10.11.27.16.10" class="ltx_td"></td>
<td id="S3.T10.11.27.16.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.28.17" class="ltx_tr">
<td id="S3.T10.11.28.17.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.28.17.1.1" class="ltx_text" style="font-size:70%;">MGGAN + Pix2Pix + WGSLN</span></td>
<td id="S3.T10.11.28.17.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.28.17.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib247" title="" class="ltx_ref">247</a><span id="S3.T10.11.28.17.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.28.17.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.28.17.3.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T10.11.28.17.4" class="ltx_td ltx_align_center"><span id="S3.T10.11.28.17.4.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T10.11.28.17.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.28.17.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.28.17.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.28.17.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.28.17.7" class="ltx_td"></td>
<td id="S3.T10.11.28.17.8" class="ltx_td"></td>
<td id="S3.T10.11.28.17.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.28.17.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.28.17.10" class="ltx_td"></td>
<td id="S3.T10.11.28.17.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.29.18" class="ltx_tr">
<td id="S3.T10.11.29.18.1" class="ltx_td ltx_align_center"><span id="S3.T10.11.29.18.1.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T10.11.29.18.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.29.18.2.1" class="ltx_text" style="font-size:70%;">SDM</span></td>
<td id="S3.T10.11.29.18.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.29.18.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib249" title="" class="ltx_ref">249</a><span id="S3.T10.11.29.18.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.29.18.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.29.18.4.1" class="ltx_text" style="font-size:70%;">Det.</span></td>
<td id="S3.T10.11.29.18.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.29.18.5.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T10.11.29.18.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.29.18.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.29.18.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.29.18.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.29.18.8" class="ltx_td"></td>
<td id="S3.T10.11.29.18.9" class="ltx_td"></td>
<td id="S3.T10.11.29.18.10" class="ltx_td"></td>
<td id="S3.T10.11.29.18.11" class="ltx_td"></td>
<td id="S3.T10.11.29.18.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.30.19" class="ltx_tr">
<td id="S3.T10.11.30.19.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T10.11.30.19.1.1" class="ltx_text" style="font-size:70%;">Attribute Cond.</span></td>
<td id="S3.T10.11.30.19.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T10.11.30.19.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T10.11.30.19.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.30.19.3.1" class="ltx_text" style="font-size:70%;">Abd. &amp; head CT</span></td>
<td id="S3.T10.11.30.19.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.30.19.4.1" class="ltx_text" style="font-size:70%;">CLDM</span></td>
<td id="S3.T10.11.30.19.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.30.19.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib242" title="" class="ltx_ref">242</a><span id="S3.T10.11.30.19.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.30.19.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.30.19.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.30.19.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.30.19.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.30.19.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.30.19.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.30.19.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.30.19.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.30.19.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.30.19.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.30.19.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T10.11.31.20" class="ltx_tr">
<td id="S3.T10.11.31.20.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.31.20.1.1" class="ltx_text" style="font-size:70%;">Thoracic CT</span></td>
<td id="S3.T10.11.31.20.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.31.20.2.1" class="ltx_text" style="font-size:70%;">Cond. DPM + Semantic DM</span></td>
<td id="S3.T10.11.31.20.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.31.20.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib243" title="" class="ltx_ref">243</a><span id="S3.T10.11.31.20.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.31.20.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.31.20.4.1" class="ltx_text" style="font-size:70%;">Seg.*</span></td>
<td id="S3.T10.11.31.20.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.31.20.5.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T10.11.31.20.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.31.20.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.31.20.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.31.20.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.31.20.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.31.20.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.31.20.9" class="ltx_td"></td>
<td id="S3.T10.11.31.20.10" class="ltx_td"></td>
<td id="S3.T10.11.31.20.11" class="ltx_td"></td>
<td id="S3.T10.11.31.20.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.32.21" class="ltx_tr">
<td id="S3.T10.11.32.21.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T10.11.32.21.1.1" class="ltx_text" style="font-size:70%;">Text-guided</span></td>
<td id="S3.T10.11.32.21.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.32.21.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T10.11.32.21.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.32.21.3.1" class="ltx_text" style="font-size:70%;">Lung CT</span></td>
<td id="S3.T10.11.32.21.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.32.21.4.1" class="ltx_text" style="font-size:70%;">BERT + cond. DDPM</span></td>
<td id="S3.T10.11.32.21.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.32.21.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib250" title="" class="ltx_ref">250</a><span id="S3.T10.11.32.21.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.32.21.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.11.32.21.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.32.21.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.32.21.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.32.21.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.32.21.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.32.21.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.32.21.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.32.21.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.32.21.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.32.21.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.11.32.21.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.11.32.21.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T10.11.33.22" class="ltx_tr">
<td id="S3.T10.11.33.22.1" class="ltx_td ltx_align_center"><span id="S3.T10.11.33.22.1.1" class="ltx_text" style="font-size:70%;">Hybrid</span></td>
<td id="S3.T10.11.33.22.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.33.22.2.1" class="ltx_text" style="font-size:70%;">Chest CT</span></td>
<td id="S3.T10.11.33.22.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.33.22.3.1" class="ltx_text" style="font-size:70%;">LLM, ViT, transf.-MaskGT</span></td>
<td id="S3.T10.11.33.22.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.33.22.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib251" title="" class="ltx_ref">251</a><span id="S3.T10.11.33.22.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.33.22.5" class="ltx_td ltx_align_left"><span id="S3.T10.11.33.22.5.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T10.11.33.22.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.33.22.6.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T10.11.33.22.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.33.22.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.33.22.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.33.22.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.33.22.9" class="ltx_td"></td>
<td id="S3.T10.11.33.22.10" class="ltx_td"></td>
<td id="S3.T10.11.33.22.11" class="ltx_td"></td>
<td id="S3.T10.11.33.22.12" class="ltx_td"></td>
<td id="S3.T10.11.33.22.13" class="ltx_td ltx_align_center"><span id="S3.T10.11.33.22.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T10.1.1" class="ltx_tr">
<td id="S3.T10.1.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="10"><span id="S3.T10.1.1.2.1" class="ltx_text" style="font-size:70%;">Intra-modal trans.</span></td>
<td id="S3.T10.1.1.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="8"><span id="S3.T10.1.1.3.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T10.1.1.1" class="ltx_td ltx_align_left ltx_border_t">
<span id="S3.T10.1.1.1.1" class="ltx_text" style="font-size:70%;">CBCT </span><math id="S3.T10.1.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.1.1.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.1.1.1.m1.1.1" xref="S3.T10.1.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.1.1.1.m1.1b"><ci id="S3.T10.1.1.1.m1.1.1.cmml" xref="S3.T10.1.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.1.1.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.1.1.1.2" class="ltx_text" style="font-size:70%;"> CT</span>
</td>
<td id="S3.T10.1.1.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.1.1.4.1" class="ltx_text" style="font-size:70%;">R2ACNN</span></td>
<td id="S3.T10.1.1.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.1.1.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib252" title="" class="ltx_ref">252</a><span id="S3.T10.1.1.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.1.1.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.1.1.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.1.1.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.1.1.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.1.1.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.1.1.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.1.1.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.1.1.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.1.1.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.1.1.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.1.1.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.1.1.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T10.2.2" class="ltx_tr">
<td id="S3.T10.2.2.1" class="ltx_td ltx_align_left">
<span id="S3.T10.2.2.1.1" class="ltx_text" style="font-size:70%;">CT </span><math id="S3.T10.2.2.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.2.2.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.2.2.1.m1.1.1" xref="S3.T10.2.2.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.2.2.1.m1.1b"><ci id="S3.T10.2.2.1.m1.1.1.cmml" xref="S3.T10.2.2.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.2.2.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.2.2.1.2" class="ltx_text" style="font-size:70%;">CTA</span>
</td>
<td id="S3.T10.2.2.2" class="ltx_td ltx_align_left"><span id="S3.T10.2.2.2.1" class="ltx_text" style="font-size:70%;">DCT-GAN</span></td>
<td id="S3.T10.2.2.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.2.2.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib253" title="" class="ltx_ref">253</a><span id="S3.T10.2.2.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.2.2.4" class="ltx_td ltx_align_left"><span id="S3.T10.2.2.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.2.2.5" class="ltx_td"></td>
<td id="S3.T10.2.2.6" class="ltx_td"></td>
<td id="S3.T10.2.2.7" class="ltx_td ltx_align_center"><span id="S3.T10.2.2.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.2.2.8" class="ltx_td"></td>
<td id="S3.T10.2.2.9" class="ltx_td"></td>
<td id="S3.T10.2.2.10" class="ltx_td"></td>
<td id="S3.T10.2.2.11" class="ltx_td"></td>
<td id="S3.T10.2.2.12" class="ltx_td ltx_align_center"><span id="S3.T10.2.2.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T10.3.3" class="ltx_tr">
<td id="S3.T10.3.3.1" class="ltx_td ltx_align_left">
<span id="S3.T10.3.3.1.1" class="ltx_text" style="font-size:70%;">NDCT </span><math id="S3.T10.3.3.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.3.3.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.3.3.1.m1.1.1" xref="S3.T10.3.3.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.3.3.1.m1.1b"><ci id="S3.T10.3.3.1.m1.1.1.cmml" xref="S3.T10.3.3.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.3.3.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.3.3.1.2" class="ltx_text" style="font-size:70%;"> LDCT</span>
</td>
<td id="S3.T10.3.3.2" class="ltx_td ltx_align_left"><span id="S3.T10.3.3.2.1" class="ltx_text" style="font-size:70%;">GAN-NETL</span></td>
<td id="S3.T10.3.3.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.3.3.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib254" title="" class="ltx_ref">254</a><span id="S3.T10.3.3.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.3.3.4" class="ltx_td ltx_align_left"><span id="S3.T10.3.3.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.3.3.5" class="ltx_td"></td>
<td id="S3.T10.3.3.6" class="ltx_td"></td>
<td id="S3.T10.3.3.7" class="ltx_td ltx_align_center"><span id="S3.T10.3.3.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.3.3.8" class="ltx_td"></td>
<td id="S3.T10.3.3.9" class="ltx_td"></td>
<td id="S3.T10.3.3.10" class="ltx_td ltx_align_center"><span id="S3.T10.3.3.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.3.3.11" class="ltx_td"></td>
<td id="S3.T10.3.3.12" class="ltx_td ltx_align_center"><span id="S3.T10.3.3.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T10.4.4" class="ltx_tr">
<td id="S3.T10.4.4.1" class="ltx_td ltx_align_left">
<span id="S3.T10.4.4.1.1" class="ltx_text" style="font-size:70%;">dCT </span><math id="S3.T10.4.4.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.4.4.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.4.4.1.m1.1.1" xref="S3.T10.4.4.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.4.4.1.m1.1b"><ci id="S3.T10.4.4.1.m1.1.1.cmml" xref="S3.T10.4.4.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.4.4.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.4.4.1.2" class="ltx_text" style="font-size:70%;"> CT</span>
</td>
<td id="S3.T10.4.4.2" class="ltx_td ltx_align_left"><span id="S3.T10.4.4.2.1" class="ltx_text" style="font-size:70%;">Pix2Pix</span></td>
<td id="S3.T10.4.4.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.4.4.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib255" title="" class="ltx_ref">255</a><span id="S3.T10.4.4.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.4.4.4" class="ltx_td ltx_align_left"><span id="S3.T10.4.4.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.4.4.5" class="ltx_td"></td>
<td id="S3.T10.4.4.6" class="ltx_td"></td>
<td id="S3.T10.4.4.7" class="ltx_td ltx_align_center"><span id="S3.T10.4.4.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.4.4.8" class="ltx_td"></td>
<td id="S3.T10.4.4.9" class="ltx_td"></td>
<td id="S3.T10.4.4.10" class="ltx_td ltx_align_center"><span id="S3.T10.4.4.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.4.4.11" class="ltx_td"></td>
<td id="S3.T10.4.4.12" class="ltx_td ltx_align_center"><span id="S3.T10.4.4.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T10.5.5" class="ltx_tr">
<td id="S3.T10.5.5.1" class="ltx_td ltx_align_left">
<span id="S3.T10.5.5.1.1" class="ltx_text" style="font-size:70%;">CBCT </span><math id="S3.T10.5.5.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.5.5.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.5.5.1.m1.1.1" xref="S3.T10.5.5.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.5.5.1.m1.1b"><ci id="S3.T10.5.5.1.m1.1.1.cmml" xref="S3.T10.5.5.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.5.5.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.5.5.1.2" class="ltx_text" style="font-size:70%;"> CT</span>
</td>
<td id="S3.T10.5.5.2" class="ltx_td ltx_align_left"><span id="S3.T10.5.5.2.1" class="ltx_text" style="font-size:70%;">Attention GAN</span></td>
<td id="S3.T10.5.5.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.5.5.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib256" title="" class="ltx_ref">256</a><span id="S3.T10.5.5.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.5.5.4" class="ltx_td ltx_align_left"><span id="S3.T10.5.5.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.5.5.5" class="ltx_td"></td>
<td id="S3.T10.5.5.6" class="ltx_td"></td>
<td id="S3.T10.5.5.7" class="ltx_td ltx_align_center"><span id="S3.T10.5.5.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.5.5.8" class="ltx_td"></td>
<td id="S3.T10.5.5.9" class="ltx_td"></td>
<td id="S3.T10.5.5.10" class="ltx_td ltx_align_center"><span id="S3.T10.5.5.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.5.5.11" class="ltx_td"></td>
<td id="S3.T10.5.5.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.34.23" class="ltx_tr">
<td id="S3.T10.11.34.23.1" class="ltx_td ltx_align_left" rowspan="3"><span id="S3.T10.11.34.23.1.1" class="ltx_text" style="font-size:70%;">COVID19</span></td>
<td id="S3.T10.11.34.23.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.34.23.2.1" class="ltx_text" style="font-size:70%;">StarGANv2</span></td>
<td id="S3.T10.11.34.23.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.34.23.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib257" title="" class="ltx_ref">257</a><span id="S3.T10.11.34.23.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.34.23.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.34.23.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T10.11.34.23.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.34.23.5.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T10.11.34.23.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.34.23.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.34.23.7" class="ltx_td"></td>
<td id="S3.T10.11.34.23.8" class="ltx_td"></td>
<td id="S3.T10.11.34.23.9" class="ltx_td"></td>
<td id="S3.T10.11.34.23.10" class="ltx_td"></td>
<td id="S3.T10.11.34.23.11" class="ltx_td"></td>
<td id="S3.T10.11.34.23.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.35.24" class="ltx_tr">
<td id="S3.T10.11.35.24.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.35.24.1.1" class="ltx_text" style="font-size:70%;">sRD-GAN</span></td>
<td id="S3.T10.11.35.24.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.35.24.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib66" title="" class="ltx_ref">66</a><span id="S3.T10.11.35.24.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.35.24.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.35.24.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.35.24.4" class="ltx_td"></td>
<td id="S3.T10.11.35.24.5" class="ltx_td"></td>
<td id="S3.T10.11.35.24.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.35.24.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.35.24.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.35.24.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.35.24.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.35.24.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.35.24.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.35.24.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.35.24.10" class="ltx_td"></td>
<td id="S3.T10.11.35.24.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.36.25" class="ltx_tr">
<td id="S3.T10.11.36.25.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.36.25.1.1" class="ltx_text" style="font-size:70%;">Cycle GAN</span></td>
<td id="S3.T10.11.36.25.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.36.25.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib258" title="" class="ltx_ref">258</a><span id="S3.T10.11.36.25.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.36.25.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.36.25.3.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T10.11.36.25.4" class="ltx_td ltx_align_center"><span id="S3.T10.11.36.25.4.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T10.11.36.25.5" class="ltx_td ltx_align_center"><span id="S3.T10.11.36.25.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.11.36.25.6" class="ltx_td"></td>
<td id="S3.T10.11.36.25.7" class="ltx_td"></td>
<td id="S3.T10.11.36.25.8" class="ltx_td"></td>
<td id="S3.T10.11.36.25.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.36.25.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.36.25.10" class="ltx_td"></td>
<td id="S3.T10.11.36.25.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.6.6" class="ltx_tr">
<td id="S3.T10.6.6.2" class="ltx_td ltx_align_center"><span id="S3.T10.6.6.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T10.6.6.1" class="ltx_td ltx_align_left">
<span id="S3.T10.6.6.1.1" class="ltx_text" style="font-size:70%;">CBCT </span><math id="S3.T10.6.6.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.6.6.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.6.6.1.m1.1.1" xref="S3.T10.6.6.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.6.6.1.m1.1b"><ci id="S3.T10.6.6.1.m1.1.1.cmml" xref="S3.T10.6.6.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.6.6.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.6.6.1.2" class="ltx_text" style="font-size:70%;"> CT</span>
</td>
<td id="S3.T10.6.6.3" class="ltx_td ltx_align_left"><span id="S3.T10.6.6.3.1" class="ltx_text" style="font-size:70%;">Cond. DDPM</span></td>
<td id="S3.T10.6.6.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.6.6.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib259" title="" class="ltx_ref">259</a><span id="S3.T10.6.6.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.6.6.5" class="ltx_td ltx_align_left"><span id="S3.T10.6.6.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.6.6.6" class="ltx_td"></td>
<td id="S3.T10.6.6.7" class="ltx_td"></td>
<td id="S3.T10.6.6.8" class="ltx_td ltx_align_center"><span id="S3.T10.6.6.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.6.6.9" class="ltx_td"></td>
<td id="S3.T10.6.6.10" class="ltx_td"></td>
<td id="S3.T10.6.6.11" class="ltx_td ltx_align_center"><span id="S3.T10.6.6.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.6.6.12" class="ltx_td"></td>
<td id="S3.T10.6.6.13" class="ltx_td"></td>
</tr>
<tr id="S3.T10.7.7" class="ltx_tr">
<td id="S3.T10.7.7.2" class="ltx_td ltx_align_center"><span id="S3.T10.7.7.2.1" class="ltx_text" style="font-size:70%;">Transf.</span></td>
<td id="S3.T10.7.7.1" class="ltx_td ltx_align_left">
<span id="S3.T10.7.7.1.1" class="ltx_text" style="font-size:70%;">insp. </span><math id="S3.T10.7.7.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.7.7.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.7.7.1.m1.1.1" xref="S3.T10.7.7.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.7.7.1.m1.1b"><ci id="S3.T10.7.7.1.m1.1.1.cmml" xref="S3.T10.7.7.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.7.7.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.7.7.1.2" class="ltx_text" style="font-size:70%;"> exp. CT</span>
</td>
<td id="S3.T10.7.7.3" class="ltx_td ltx_align_left"><span id="S3.T10.7.7.3.1" class="ltx_text" style="font-size:70%;">SWIN</span></td>
<td id="S3.T10.7.7.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.7.7.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib260" title="" class="ltx_ref">260</a><span id="S3.T10.7.7.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.7.7.5" class="ltx_td ltx_align_left"><span id="S3.T10.7.7.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.7.7.6" class="ltx_td"></td>
<td id="S3.T10.7.7.7" class="ltx_td"></td>
<td id="S3.T10.7.7.8" class="ltx_td ltx_align_center"><span id="S3.T10.7.7.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.7.7.9" class="ltx_td ltx_align_center"><span id="S3.T10.7.7.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.7.7.10" class="ltx_td"></td>
<td id="S3.T10.7.7.11" class="ltx_td ltx_align_center"><span id="S3.T10.7.7.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.7.7.12" class="ltx_td"></td>
<td id="S3.T10.7.7.13" class="ltx_td"></td>
</tr>
<tr id="S3.T10.8.8" class="ltx_tr">
<td id="S3.T10.8.8.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="18"><span id="S3.T10.8.8.2.1" class="ltx_text" style="font-size:70%;">Inter-modal trans.</span></td>
<td id="S3.T10.8.8.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="13"><span id="S3.T10.8.8.3.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T10.8.8.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="11"><span id="S3.T10.8.8.1.1" class="ltx_text" style="font-size:70%;">MR <math id="S3.T10.8.8.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.8.8.1.1.m1.1a"><mo stretchy="false" id="S3.T10.8.8.1.1.m1.1.1" xref="S3.T10.8.8.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.8.8.1.1.m1.1b"><ci id="S3.T10.8.8.1.1.m1.1.1.cmml" xref="S3.T10.8.8.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.8.8.1.1.m1.1c">\rightarrow</annotation></semantics></math> CT</span></td>
<td id="S3.T10.8.8.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.8.8.4.1" class="ltx_text" style="font-size:70%;">CTF GAN (Pix2Pix)</span></td>
<td id="S3.T10.8.8.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.8.8.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib261" title="" class="ltx_ref">261</a><span id="S3.T10.8.8.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.8.8.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T10.8.8.6.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T10.8.8.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.8.8.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T10.8.8.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.8.8.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T10.8.8.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T10.8.8.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.8.8.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.8.8.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.8.8.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.8.8.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T10.8.8.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T10.11.37.26" class="ltx_tr">
<td id="S3.T10.11.37.26.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.37.26.1.1" class="ltx_text" style="font-size:70%;">3D Pix2Pix</span></td>
<td id="S3.T10.11.37.26.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.37.26.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib262" title="" class="ltx_ref">262</a><span id="S3.T10.11.37.26.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.37.26.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.37.26.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.37.26.4" class="ltx_td"></td>
<td id="S3.T10.11.37.26.5" class="ltx_td"></td>
<td id="S3.T10.11.37.26.6" class="ltx_td"></td>
<td id="S3.T10.11.37.26.7" class="ltx_td"></td>
<td id="S3.T10.11.37.26.8" class="ltx_td"></td>
<td id="S3.T10.11.37.26.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.37.26.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.37.26.10" class="ltx_td"></td>
<td id="S3.T10.11.37.26.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.38.27" class="ltx_tr">
<td id="S3.T10.11.38.27.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.38.27.1.1" class="ltx_text" style="font-size:70%;">Pix2Pix + StyleGAN</span></td>
<td id="S3.T10.11.38.27.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.38.27.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib263" title="" class="ltx_ref">263</a><span id="S3.T10.11.38.27.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.38.27.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.38.27.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.38.27.4" class="ltx_td"></td>
<td id="S3.T10.11.38.27.5" class="ltx_td"></td>
<td id="S3.T10.11.38.27.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.38.27.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.38.27.7" class="ltx_td"></td>
<td id="S3.T10.11.38.27.8" class="ltx_td"></td>
<td id="S3.T10.11.38.27.9" class="ltx_td"></td>
<td id="S3.T10.11.38.27.10" class="ltx_td"></td>
<td id="S3.T10.11.38.27.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.39.28" class="ltx_tr">
<td id="S3.T10.11.39.28.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.39.28.1.1" class="ltx_text" style="font-size:70%;">Multi-cycle GAN</span></td>
<td id="S3.T10.11.39.28.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.39.28.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib264" title="" class="ltx_ref">264</a><span id="S3.T10.11.39.28.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.39.28.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.39.28.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.39.28.4" class="ltx_td"></td>
<td id="S3.T10.11.39.28.5" class="ltx_td"></td>
<td id="S3.T10.11.39.28.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.39.28.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.39.28.7" class="ltx_td"></td>
<td id="S3.T10.11.39.28.8" class="ltx_td"></td>
<td id="S3.T10.11.39.28.9" class="ltx_td"></td>
<td id="S3.T10.11.39.28.10" class="ltx_td"></td>
<td id="S3.T10.11.39.28.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.40.29" class="ltx_tr">
<td id="S3.T10.11.40.29.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.40.29.1.1" class="ltx_text" style="font-size:70%;">U-Net cycleGAN</span></td>
<td id="S3.T10.11.40.29.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.40.29.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib265" title="" class="ltx_ref">265</a><span id="S3.T10.11.40.29.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.40.29.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.40.29.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.40.29.4" class="ltx_td"></td>
<td id="S3.T10.11.40.29.5" class="ltx_td"></td>
<td id="S3.T10.11.40.29.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.40.29.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.40.29.7" class="ltx_td"></td>
<td id="S3.T10.11.40.29.8" class="ltx_td"></td>
<td id="S3.T10.11.40.29.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.40.29.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.40.29.10" class="ltx_td"></td>
<td id="S3.T10.11.40.29.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.41.30" class="ltx_tr">
<td id="S3.T10.11.41.30.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.41.30.1.1" class="ltx_text" style="font-size:70%;">Cycle GAN + attention</span></td>
<td id="S3.T10.11.41.30.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.41.30.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib266" title="" class="ltx_ref">266</a><span id="S3.T10.11.41.30.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.41.30.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.41.30.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.41.30.4" class="ltx_td"></td>
<td id="S3.T10.11.41.30.5" class="ltx_td"></td>
<td id="S3.T10.11.41.30.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.41.30.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.41.30.7" class="ltx_td"></td>
<td id="S3.T10.11.41.30.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.41.30.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.41.30.9" class="ltx_td"></td>
<td id="S3.T10.11.41.30.10" class="ltx_td"></td>
<td id="S3.T10.11.41.30.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.42.31" class="ltx_tr">
<td id="S3.T10.11.42.31.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.42.31.1.1" class="ltx_text" style="font-size:70%;">Cycle GAN</span></td>
<td id="S3.T10.11.42.31.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.42.31.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib267" title="" class="ltx_ref">267</a><span id="S3.T10.11.42.31.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.42.31.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.42.31.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.42.31.4" class="ltx_td"></td>
<td id="S3.T10.11.42.31.5" class="ltx_td"></td>
<td id="S3.T10.11.42.31.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.42.31.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.42.31.7" class="ltx_td"></td>
<td id="S3.T10.11.42.31.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.42.31.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.42.31.9" class="ltx_td"></td>
<td id="S3.T10.11.42.31.10" class="ltx_td"></td>
<td id="S3.T10.11.42.31.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.43.32" class="ltx_tr">
<td id="S3.T10.11.43.32.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.43.32.1.1" class="ltx_text" style="font-size:70%;">3D Cycle GAN</span></td>
<td id="S3.T10.11.43.32.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.43.32.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib268" title="" class="ltx_ref">268</a><span id="S3.T10.11.43.32.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.43.32.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.43.32.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.43.32.4" class="ltx_td"></td>
<td id="S3.T10.11.43.32.5" class="ltx_td"></td>
<td id="S3.T10.11.43.32.6" class="ltx_td"></td>
<td id="S3.T10.11.43.32.7" class="ltx_td"></td>
<td id="S3.T10.11.43.32.8" class="ltx_td"></td>
<td id="S3.T10.11.43.32.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.43.32.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.43.32.10" class="ltx_td"></td>
<td id="S3.T10.11.43.32.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.44.33" class="ltx_tr">
<td id="S3.T10.11.44.33.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.44.33.1.1" class="ltx_text" style="font-size:70%;">Pix2Pix</span></td>
<td id="S3.T10.11.44.33.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.44.33.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib269" title="" class="ltx_ref">269</a><span id="S3.T10.11.44.33.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.44.33.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.44.33.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.44.33.4" class="ltx_td"></td>
<td id="S3.T10.11.44.33.5" class="ltx_td"></td>
<td id="S3.T10.11.44.33.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.44.33.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.44.33.7" class="ltx_td"></td>
<td id="S3.T10.11.44.33.8" class="ltx_td"></td>
<td id="S3.T10.11.44.33.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.44.33.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.44.33.10" class="ltx_td"></td>
<td id="S3.T10.11.44.33.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.45.34" class="ltx_tr">
<td id="S3.T10.11.45.34.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.45.34.1.1" class="ltx_text" style="font-size:70%;">Cycle GAN, structGAN, Flow CGAN</span></td>
<td id="S3.T10.11.45.34.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.45.34.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib270" title="" class="ltx_ref">270</a><span id="S3.T10.11.45.34.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.45.34.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.45.34.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.45.34.4" class="ltx_td"></td>
<td id="S3.T10.11.45.34.5" class="ltx_td"></td>
<td id="S3.T10.11.45.34.6" class="ltx_td"></td>
<td id="S3.T10.11.45.34.7" class="ltx_td"></td>
<td id="S3.T10.11.45.34.8" class="ltx_td"></td>
<td id="S3.T10.11.45.34.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.45.34.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.45.34.10" class="ltx_td"></td>
<td id="S3.T10.11.45.34.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.46.35" class="ltx_tr">
<td id="S3.T10.11.46.35.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.46.35.1.1" class="ltx_text" style="font-size:70%;">U-Net-based WGAN</span></td>
<td id="S3.T10.11.46.35.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.46.35.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib271" title="" class="ltx_ref">271</a><span id="S3.T10.11.46.35.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.46.35.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.46.35.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.46.35.4" class="ltx_td"></td>
<td id="S3.T10.11.46.35.5" class="ltx_td"></td>
<td id="S3.T10.11.46.35.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.46.35.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.46.35.7" class="ltx_td"></td>
<td id="S3.T10.11.46.35.8" class="ltx_td"></td>
<td id="S3.T10.11.46.35.9" class="ltx_td ltx_align_center"><span id="S3.T10.11.46.35.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.46.35.10" class="ltx_td"></td>
<td id="S3.T10.11.46.35.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.9.9" class="ltx_tr">
<td id="S3.T10.9.9.1" class="ltx_td ltx_align_left">
<span id="S3.T10.9.9.1.1" class="ltx_text" style="font-size:70%;">PET/MRI </span><math id="S3.T10.9.9.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.9.9.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.9.9.1.m1.1.1" xref="S3.T10.9.9.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.9.9.1.m1.1b"><ci id="S3.T10.9.9.1.m1.1.1.cmml" xref="S3.T10.9.9.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.9.9.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.9.9.1.2" class="ltx_text" style="font-size:70%;"> CT</span>
</td>
<td id="S3.T10.9.9.2" class="ltx_td ltx_align_left"><span id="S3.T10.9.9.2.1" class="ltx_text" style="font-size:70%;">Patch GAN</span></td>
<td id="S3.T10.9.9.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.9.9.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib272" title="" class="ltx_ref">272</a><span id="S3.T10.9.9.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.9.9.4" class="ltx_td ltx_align_left"><span id="S3.T10.9.9.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.9.9.5" class="ltx_td"></td>
<td id="S3.T10.9.9.6" class="ltx_td"></td>
<td id="S3.T10.9.9.7" class="ltx_td ltx_align_center"><span id="S3.T10.9.9.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.9.9.8" class="ltx_td ltx_align_center"><span id="S3.T10.9.9.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.9.9.9" class="ltx_td"></td>
<td id="S3.T10.9.9.10" class="ltx_td"></td>
<td id="S3.T10.9.9.11" class="ltx_td"></td>
<td id="S3.T10.9.9.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.10.10" class="ltx_tr">
<td id="S3.T10.10.10.1" class="ltx_td ltx_align_left">
<span id="S3.T10.10.10.1.1" class="ltx_text" style="font-size:70%;">US </span><math id="S3.T10.10.10.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.10.10.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T10.10.10.1.m1.1.1" xref="S3.T10.10.10.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.10.10.1.m1.1b"><ci id="S3.T10.10.10.1.m1.1.1.cmml" xref="S3.T10.10.10.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.10.10.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T10.10.10.1.2" class="ltx_text" style="font-size:70%;"> CT</span>
</td>
<td id="S3.T10.10.10.2" class="ltx_td ltx_align_left"><span id="S3.T10.10.10.2.1" class="ltx_text" style="font-size:70%;">cGAN + SRGAN</span></td>
<td id="S3.T10.10.10.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.10.10.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib273" title="" class="ltx_ref">273</a><span id="S3.T10.10.10.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.10.10.4" class="ltx_td ltx_align_left"><span id="S3.T10.10.10.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.10.10.5" class="ltx_td"></td>
<td id="S3.T10.10.10.6" class="ltx_td"></td>
<td id="S3.T10.10.10.7" class="ltx_td ltx_align_center"><span id="S3.T10.10.10.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.10.10.8" class="ltx_td"></td>
<td id="S3.T10.10.10.9" class="ltx_td"></td>
<td id="S3.T10.10.10.10" class="ltx_td ltx_align_center"><span id="S3.T10.10.10.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.10.10.11" class="ltx_td"></td>
<td id="S3.T10.10.10.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.11" class="ltx_tr">
<td id="S3.T10.11.11.2" class="ltx_td ltx_align_center" rowspan="3"><span id="S3.T10.11.11.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T10.11.11.1" class="ltx_td ltx_align_left ltx_border_b" rowspan="5"><span id="S3.T10.11.11.1.1" class="ltx_text" style="font-size:70%;">MR <math id="S3.T10.11.11.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T10.11.11.1.1.m1.1a"><mo stretchy="false" id="S3.T10.11.11.1.1.m1.1.1" xref="S3.T10.11.11.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T10.11.11.1.1.m1.1b"><ci id="S3.T10.11.11.1.1.m1.1.1.cmml" xref="S3.T10.11.11.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T10.11.11.1.1.m1.1c">\rightarrow</annotation></semantics></math> CT</span></td>
<td id="S3.T10.11.11.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.11.3.1" class="ltx_text" style="font-size:70%;">SynDiff</span></td>
<td id="S3.T10.11.11.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.11.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib274" title="" class="ltx_ref">274</a><span id="S3.T10.11.11.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.11.5" class="ltx_td ltx_align_left"><span id="S3.T10.11.11.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.11.6" class="ltx_td"></td>
<td id="S3.T10.11.11.7" class="ltx_td"></td>
<td id="S3.T10.11.11.8" class="ltx_td ltx_align_center"><span id="S3.T10.11.11.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.11.9" class="ltx_td"></td>
<td id="S3.T10.11.11.10" class="ltx_td"></td>
<td id="S3.T10.11.11.11" class="ltx_td"></td>
<td id="S3.T10.11.11.12" class="ltx_td"></td>
<td id="S3.T10.11.11.13" class="ltx_td ltx_align_center"><span id="S3.T10.11.11.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T10.11.47.36" class="ltx_tr">
<td id="S3.T10.11.47.36.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.47.36.1.1" class="ltx_text" style="font-size:70%;">DDPM + SDE</span></td>
<td id="S3.T10.11.47.36.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.47.36.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib275" title="" class="ltx_ref">275</a><span id="S3.T10.11.47.36.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.47.36.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.47.36.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.47.36.4" class="ltx_td"></td>
<td id="S3.T10.11.47.36.5" class="ltx_td"></td>
<td id="S3.T10.11.47.36.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.47.36.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.47.36.7" class="ltx_td"></td>
<td id="S3.T10.11.47.36.8" class="ltx_td"></td>
<td id="S3.T10.11.47.36.9" class="ltx_td"></td>
<td id="S3.T10.11.47.36.10" class="ltx_td"></td>
<td id="S3.T10.11.47.36.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.48.37" class="ltx_tr">
<td id="S3.T10.11.48.37.1" class="ltx_td ltx_align_left"><span id="S3.T10.11.48.37.1.1" class="ltx_text" style="font-size:70%;">DDMM</span></td>
<td id="S3.T10.11.48.37.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.48.37.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib276" title="" class="ltx_ref">276</a><span id="S3.T10.11.48.37.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.48.37.3" class="ltx_td ltx_align_left"><span id="S3.T10.11.48.37.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.48.37.4" class="ltx_td"></td>
<td id="S3.T10.11.48.37.5" class="ltx_td"></td>
<td id="S3.T10.11.48.37.6" class="ltx_td ltx_align_center"><span id="S3.T10.11.48.37.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.48.37.7" class="ltx_td"></td>
<td id="S3.T10.11.48.37.8" class="ltx_td"></td>
<td id="S3.T10.11.48.37.9" class="ltx_td"></td>
<td id="S3.T10.11.48.37.10" class="ltx_td"></td>
<td id="S3.T10.11.48.37.11" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.49.38" class="ltx_tr">
<td id="S3.T10.11.49.38.1" class="ltx_td ltx_align_center ltx_border_b" rowspan="2"><span id="S3.T10.11.49.38.1.1" class="ltx_text" style="font-size:70%;">Hybrid</span></td>
<td id="S3.T10.11.49.38.2" class="ltx_td ltx_align_left"><span id="S3.T10.11.49.38.2.1" class="ltx_text" style="font-size:70%;">Swin-VNET</span></td>
<td id="S3.T10.11.49.38.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.49.38.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib277" title="" class="ltx_ref">277</a><span id="S3.T10.11.49.38.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.49.38.4" class="ltx_td ltx_align_left"><span id="S3.T10.11.49.38.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.49.38.5" class="ltx_td"></td>
<td id="S3.T10.11.49.38.6" class="ltx_td"></td>
<td id="S3.T10.11.49.38.7" class="ltx_td ltx_align_center"><span id="S3.T10.11.49.38.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.49.38.8" class="ltx_td"></td>
<td id="S3.T10.11.49.38.9" class="ltx_td"></td>
<td id="S3.T10.11.49.38.10" class="ltx_td ltx_align_center"><span id="S3.T10.11.49.38.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.49.38.11" class="ltx_td"></td>
<td id="S3.T10.11.49.38.12" class="ltx_td"></td>
</tr>
<tr id="S3.T10.11.50.39" class="ltx_tr">
<td id="S3.T10.11.50.39.1" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T10.11.50.39.1.1" class="ltx_text" style="font-size:70%;">RTCGAN-CNN/transf.</span></td>
<td id="S3.T10.11.50.39.2" class="ltx_td ltx_align_left ltx_border_b"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T10.11.50.39.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib278" title="" class="ltx_ref">278</a><span id="S3.T10.11.50.39.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T10.11.50.39.3" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T10.11.50.39.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T10.11.50.39.4" class="ltx_td ltx_border_b"></td>
<td id="S3.T10.11.50.39.5" class="ltx_td ltx_border_b"></td>
<td id="S3.T10.11.50.39.6" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T10.11.50.39.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.50.39.7" class="ltx_td ltx_border_b"></td>
<td id="S3.T10.11.50.39.8" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T10.11.50.39.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.50.39.9" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T10.11.50.39.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T10.11.50.39.10" class="ltx_td ltx_border_b"></td>
<td id="S3.T10.11.50.39.11" class="ltx_td ltx_border_b"></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 10: </span>Overview of all summarized and discussed studies on <span id="S3.T10.16.1" class="ltx_text ltx_font_bold">CT</span> synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), whereby automated machine learning models are trained with real (R), synthetic (S) or augmented with synthetic data (A), and tested on real or synthetic data, fidelity (F), diversity (D), visual quality (Q), clinical utility (C) and privacy (P) evaluation. * stands for studies that have utilized synthesized images for pre-training. </figcaption>
</figure>
<div id="S3.SS3.SSS5.p8" class="ltx_para">
<p id="S3.SS3.SSS5.p8.1" class="ltx_p">Applications of generative models for LDCT synthesis include <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib254" title="" class="ltx_ref">254</a>]</cite>, which introduces a GAN with Noise Encoding Transfer Learning (GAN-NETL) to generate paired Normal-dose CT (NDCT) and LDCT datasets, addressing variations in LDCT noise across scanners. For MRI-only radiotherapy, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib261" title="" class="ltx_ref">261</a>]</cite> proposes joint synthesis and segmentation using a Pix2Pix GAN, transferring semantic information for anatomical structure correction in synthetic CT images. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib262" title="" class="ltx_ref">262</a>]</cite> extends the Pix2Pix generator for 3D CT synthesis from T1-weighted MR images within a patch-based conditional GAN, showing high correlation with real CT.</p>
</div>
<div id="S3.SS3.SSS5.p9" class="ltx_para">
<p id="S3.SS3.SSS5.p9.1" class="ltx_p">To tackle the lack of paired data, unpaired synthesis with cycle GANs has become popular across various tasks, including generating synthetic abdominal and pelvic CT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib235" title="" class="ltx_ref">235</a>, <a href="#bib.bib65" title="" class="ltx_ref">65</a>]</cite>, prostate <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib267" title="" class="ltx_ref">267</a>]</cite>, lumbar spine <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib268" title="" class="ltx_ref">268</a>]</cite> in the realm of inter-modal translation, and COVID-19 <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib67" title="" class="ltx_ref">67</a>, <a href="#bib.bib257" title="" class="ltx_ref">257</a>, <a href="#bib.bib258" title="" class="ltx_ref">258</a>]</cite> in intra-modal translation applications. To address structural consistency issues between MRI and CT images inherent in cycle GAN-based synthesis for inter-modal translation, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib264" title="" class="ltx_ref">264</a>]</cite> propose a multi-cycle GAN with a pseudo-cycle consistent module and domain control module. This ensures high-quality and realistic CT image generation. Similarly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib270" title="" class="ltx_ref">270</a>]</cite> introduce augmented cycle consistent GANs, injecting structural information and optical flow consistency constraints. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib266" title="" class="ltx_ref">266</a>]</cite> present a paired-unpaired unsupervised attention-guided GAN, combining Wasserstein GAN adversarial loss with content and L1 losses, capturing fine structures and improving global consistency. Additionally, to mitigate spatial inconsistencies often generated by 2D cycle GANs in synthetic 3D CT images, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib265" title="" class="ltx_ref">265</a>]</cite> employ a double U-Net cycle GAN for 2.5D synthesis.</p>
</div>
<div id="S3.SS3.SSS5.p10" class="ltx_para">
<p id="S3.SS3.SSS5.p10.1" class="ltx_p">Cone beam CT (CBCT) images are crucial for image-guided therapy but often suffer from low contrast and artifacts, impacting adaptive radiotherapy dose calculations. Synthetic CT addresses these challenges, with <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib256" title="" class="ltx_ref">256</a>]</cite> using an attention-guided GAN for unpaired low-dose CBCT to CT synthesis, emphasizing artifact correction due to X-ray scatter and respiratory movements. Similarly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib252" title="" class="ltx_ref">252</a>]</cite> introduces a sequence-aware contrastive generative network to enhance CBCT image quality. Other approaches include <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib253" title="" class="ltx_ref">253</a>]</cite> using a cascaded GAN for synthesizing plain CT images into CTA images, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib279" title="" class="ltx_ref">279</a>]</cite> applying styleGAN2 with a tailored loss function for breast CT images, and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib263" title="" class="ltx_ref">263</a>]</cite> manipulating styleGAN2’s latent space for diverse CT image generation.</p>
</div>
<div id="S3.SS3.SSS5.p11" class="ltx_para">
<p id="S3.SS3.SSS5.p11.1" class="ltx_p">Intra- and inter-modality translation models, like <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib274" title="" class="ltx_ref">274</a>]</cite>, propose a cycle-consistent architecture for diffusion-based synthesis between CT and MRI scans. Similarly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib275" title="" class="ltx_ref">275</a>]</cite> explores diffusion and score-matching models for CT-MRI image conversion, showcasing superior image quality with faithful anatomical details and avoiding over-smoothing and artifacts. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib276" title="" class="ltx_ref">276</a>]</cite> introduces a denoising diffusion model for cross-modal medical image synthesis, allowing for adjustment of CT projection numbers and enhancing CT image fidelity. Meanwhile, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib277" title="" class="ltx_ref">277</a>]</cite> proposes a method for generating CT scans from MRI data, employing a transformer-based DDPM with a shifter-window transformer network for the diffusion process, facilitating the synthesis of synthetic CT from MRI.</p>
</div>
<div id="S3.SS3.SSS5.p12" class="ltx_para">
<p id="S3.SS3.SSS5.p12.1" class="ltx_p">While many studies use fidelity metrics, comprehensive evaluations of newly proposed models, especially those employing diffusion or transformer-based approaches, are lacking <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib240" title="" class="ltx_ref">240</a>, <a href="#bib.bib242" title="" class="ltx_ref">242</a>, <a href="#bib.bib243" title="" class="ltx_ref">243</a>, <a href="#bib.bib249" title="" class="ltx_ref">249</a>, <a href="#bib.bib275" title="" class="ltx_ref">275</a>, <a href="#bib.bib277" title="" class="ltx_ref">277</a>]</cite>. The practical utility of generated images in downstream tasks is often overlooked, raising concerns about clinical applicability. A crucial need for realistic synthetic medical images is highlighted, emphasizing the importance of a visual Turing test with radiologists <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib65" title="" class="ltx_ref">65</a>, <a href="#bib.bib246" title="" class="ltx_ref">246</a>, <a href="#bib.bib266" title="" class="ltx_ref">266</a>]</cite>. However, these tests often involve few participants and limited samples, introducing biases, and challenges, particularly in assessing 2D slices representing 3D data.</p>
</div>
<div id="S3.SS3.SSS5.p13" class="ltx_para">
<p id="S3.SS3.SSS5.p13.1" class="ltx_p">Examining model limitations, studies like <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib238" title="" class="ltx_ref">238</a>, <a href="#bib.bib246" title="" class="ltx_ref">246</a>]</cite> reveal the presence of artificial features and challenges in the realistic synthesis of certain anatomical regions, such as the thoracoabdominal junction. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib238" title="" class="ltx_ref">238</a>]</cite> suggests modifications to the visual Turing test and stresses involving multiple radiologists for robust evaluations. Insights from <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib244" title="" class="ltx_ref">244</a>]</cite> indicate challenges in discriminating between real and synthetic images at lower spatial resolutions, underlining the importance of image quality and label accuracy in realistic synthesis. Interestingly, classification accuracy improves with higher spatial resolution due to detailed features and visual artifacts, aligning with similar findings. These considerations emphasize ongoing challenges in evaluating synthetic medical images, highlighting the need for a standardized approach in assessing generative models in clinical contexts.</p>
</div>
<div id="S3.SS3.SSS5.p14" class="ltx_para">
<p id="S3.SS3.SSS5.p14.1" class="ltx_p">Metrics like FID and MMD, though effective for natural images, may overlook crucial anatomical details in medical images, as evidenced by the small gap between methods. A clinically relevant evaluation is thus pivotal in introducing synthetic images into the medical field. CT synthetic images are primarily assessed by calculating differences in Hounsfield units per organ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib235" title="" class="ltx_ref">235</a>, <a href="#bib.bib247" title="" class="ltx_ref">247</a>, <a href="#bib.bib252" title="" class="ltx_ref">252</a>, <a href="#bib.bib256" title="" class="ltx_ref">256</a>, <a href="#bib.bib259" title="" class="ltx_ref">259</a>, <a href="#bib.bib272" title="" class="ltx_ref">272</a>, <a href="#bib.bib273" title="" class="ltx_ref">273</a>, <a href="#bib.bib267" title="" class="ltx_ref">267</a>, <a href="#bib.bib277" title="" class="ltx_ref">277</a>, <a href="#bib.bib278" title="" class="ltx_ref">278</a>]</cite>, crucial for reflecting tissue density on CT scans. Given CT’s significance in radiation oncology, dose distribution comparison is common <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib252" title="" class="ltx_ref">252</a>, <a href="#bib.bib255" title="" class="ltx_ref">255</a>, <a href="#bib.bib256" title="" class="ltx_ref">256</a>, <a href="#bib.bib262" title="" class="ltx_ref">262</a>, <a href="#bib.bib267" title="" class="ltx_ref">267</a>, <a href="#bib.bib269" title="" class="ltx_ref">269</a>, <a href="#bib.bib271" title="" class="ltx_ref">271</a>, <a href="#bib.bib273" title="" class="ltx_ref">273</a>]</cite>. Additional studies evaluate organ contour quality and target volumes for planning and dose calculation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib245" title="" class="ltx_ref">245</a>, <a href="#bib.bib256" title="" class="ltx_ref">256</a>, <a href="#bib.bib269" title="" class="ltx_ref">269</a>, <a href="#bib.bib273" title="" class="ltx_ref">273</a>, <a href="#bib.bib278" title="" class="ltx_ref">278</a>]</cite>. A subset of studies delves into clinically-relevant anatomical measurements from synthetic images, showcasing diverse applications. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib260" title="" class="ltx_ref">260</a>]</cite> calculates biomarkers for air trapping and emphysema, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib262" title="" class="ltx_ref">262</a>]</cite> compares skull properties for transcranial ultrasound procedures, and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib268" title="" class="ltx_ref">268</a>]</cite> assesses errors in lumbar spine synthetic CT images. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib250" title="" class="ltx_ref">250</a>]</cite> conducts an in-depth analysis of diffusion-based text-guided image synthesis, evaluating anatomical preservation in lung vessels and airways. This method, utilizing specific input prompts from real radiology reports, enhances the evaluation of text-based models, considering clinical indices like pleural effusion and cardiothoracic ratio. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib272" title="" class="ltx_ref">272</a>]</cite> examines attenuation correction in PET images from synthetic CT, compared to current MRI methods.</p>
</div>
<div id="S3.SS3.SSS5.p15" class="ltx_para">
<p id="S3.SS3.SSS5.p15.1" class="ltx_p">These methods offer insights into generative model effectiveness for clinical use, surpassing traditional metrics. However, integrating clinical information into the generation process remains uncommon, and evaluating privacy preservation with synthetic data is seldom done. Notably, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib244" title="" class="ltx_ref">244</a>]</cite> stands out as one of the few studies employing nearest neighbor matching and cosine distance to demonstrate that GAN models may inherently offer privacy through stochastic gradient descent. Nevertheless, the study emphasizes the imperative need to formulate and quantify privacy guarantees.</p>
</div>
<div id="S3.SS3.SSS5.p16" class="ltx_para">
<p id="S3.SS3.SSS5.p16.1" class="ltx_p">Lastly, a crucial approach in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib280" title="" class="ltx_ref">280</a>]</cite> uniquely advocates for employing synthetic data in AI model validation, utilizing it as a part of the training process to prevent overfitting and facilitate model selection. The study introduces synthetic tumors in an extensive validation dataset, incorporated into a continual learning framework. Models trained and validated on dynamically expanding synthetic data consistently outperform those relying solely on real-world data for tumor segmentation. The findings highlight the efficacy of synthetic data in mitigating overfitting and supporting early cancer detection, suggesting its potential for large-scale testing, especially in scenarios where obtaining real data is challenging, as is often the case in clinical settings.</p>
</div>
</section>
<section id="S3.SS3.SSS6" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.6 </span>X-Ray</h4>

<div id="S3.SS3.SSS6.p1" class="ltx_para">
<p id="S3.SS3.SSS6.p1.1" class="ltx_p">Common applications of X-ray image synthesis (Table <a href="#S3.T11" title="Table 11 ‣ 3.3.6 X-Ray ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">11</span></a>) involve generating images with pathological features like lesions in Chest X-rays (CXR). Lesions vary in shape and size, posing challenges for detection, especially in deep learning methods reliant on extensive training data. Generating X-ray images of COVID-19 patients has also attracted significant interest, with various approaches using different conditional and unconditional methods to enhance the diversity of the COVID-19 patient population, leading to more accurate detection. Consequently, translating between X-rays of normal cases and those showing indications of pneumonia or tumors is frequently performed to improve the performance of anomaly detection algorithms.</p>
</div>
<div id="S3.SS3.SSS6.p2" class="ltx_para">
<p id="S3.SS3.SSS6.p2.1" class="ltx_p">Common methods for unconditional synthesis of X-ray images involve WGAN-GPs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib281" title="" class="ltx_ref">281</a>, <a href="#bib.bib282" title="" class="ltx_ref">282</a>]</cite>, DCGANs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib282" title="" class="ltx_ref">282</a>, <a href="#bib.bib283" title="" class="ltx_ref">283</a>]</cite>, and PGGANs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib283" title="" class="ltx_ref">283</a>, <a href="#bib.bib284" title="" class="ltx_ref">284</a>, <a href="#bib.bib285" title="" class="ltx_ref">285</a>]</cite>, known for high-resolution synthetic image generation. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib281" title="" class="ltx_ref">281</a>]</cite> pioneers WGANs for realistic knee joint X-rays, validated by 15 medical experts, showing notable realism and improved osteoarthritis severity classification accuracy.
<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib282" title="" class="ltx_ref">282</a>]</cite> notes DCGAN’s instability during COVID-19 CXR generation. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib286" title="" class="ltx_ref">286</a>]</cite> stabilizes training with multi-scale gradient flow and self-attention mechanisms. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib287" title="" class="ltx_ref">287</a>]</cite> explores co-evolutionary learning, and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib288" title="" class="ltx_ref">288</a>]</cite> leverages a PGGAN for chest X-ray synthesis, useful in cardiomegaly <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib289" title="" class="ltx_ref">289</a>]</cite> and abnormality detection <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib285" title="" class="ltx_ref">285</a>]</cite>. While <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib281" title="" class="ltx_ref">281</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib283" title="" class="ltx_ref">283</a>]</cite> investigate the utility of synthetic images for osteoarthritis severity and abnormality classification across various training scenarios, other approaches mainly focus on augmenting with synthetic data alone <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib282" title="" class="ltx_ref">282</a>, <a href="#bib.bib285" title="" class="ltx_ref">285</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib289" title="" class="ltx_ref">289</a>]</cite> suggests synthetic data for cardiothoracic ratio calculation in cardiomegaly. Despite using FID scores <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib282" title="" class="ltx_ref">282</a>, <a href="#bib.bib283" title="" class="ltx_ref">283</a>, <a href="#bib.bib286" title="" class="ltx_ref">286</a>, <a href="#bib.bib287" title="" class="ltx_ref">287</a>]</cite>, studies often lack detailed downstream task analysis <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib281" title="" class="ltx_ref">281</a>, <a href="#bib.bib282" title="" class="ltx_ref">282</a>, <a href="#bib.bib285" title="" class="ltx_ref">285</a>, <a href="#bib.bib288" title="" class="ltx_ref">288</a>, <a href="#bib.bib289" title="" class="ltx_ref">289</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib283" title="" class="ltx_ref">283</a>]</cite> attempts downstream task analysis, but the evaluation lacks detail, raising concerns about practical applicability in clinical settings.</p>
</div>
<div id="S3.SS3.SSS6.p3" class="ltx_para">
<p id="S3.SS3.SSS6.p3.1" class="ltx_p">Recent studies exploring conditional approaches for data generation include employing a DDPM-based model with shared latent noise for semantic consistency in synthesizing image/label pairs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib290" title="" class="ltx_ref">290</a>]</cite>, as well as latent class optimization for synthesis of diverse pathologies <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib284" title="" class="ltx_ref">284</a>]</cite>. In parallel, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib291" title="" class="ltx_ref">291</a>]</cite> synthesizes pathology by translating disease-containing medical images into disease-free ones using a cycleGAN and a domain label as a conditional input. In the evaluation process, both <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib290" title="" class="ltx_ref">290</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib284" title="" class="ltx_ref">284</a>]</cite>extensively assess synthetic images, including a thorough examination in downstream segmentation tasks, such as semantic and instance-aware segmentation, for both in-domain and out-of-domain samples. Furthermore, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib284" title="" class="ltx_ref">284</a>]</cite> stands out as one of the first studies to delve into a comprehensive analysis of the applicability of FID for quality assessment, showcasing its effectiveness in distinguishing not only between various pathologies and normal X-rays but also among different pathologies themselves.</p>
</div>
<div id="S3.SS3.SSS6.p4" class="ltx_para">
<p id="S3.SS3.SSS6.p4.1" class="ltx_p">Some studies extend the use of label-conditioned synthetic methods beyond basic label utilization, broadening data generation capabilities. For instance, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib292" title="" class="ltx_ref">292</a>]</cite> uses a DCGAN-based shape generator with size modulation for precise control of lung nodule shape and diameter. Incorporating PGGAN enhances visual plausibility, adopting a coarse-to-fine generation approach. Notably, they demonstrate that deliberate selection of challenging examples improves augmentation significantly. In a different approach, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib293" title="" class="ltx_ref">293</a>]</cite> employs StyleGAN2 and Bayesian image reconstruction for ROI-conditioned synthesis of CXR images. It aims to generate synthetic data while preserving clinically relevant features, proposing a potential data sharing strategy for privacy protection. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib294" title="" class="ltx_ref">294</a>]</cite> merges PGGAN with Pix2PixHD GAN to facilitate multi-stage generation from dots, representing various anatomical parts and serving as seeds for subsequent steps that produce high-resolution images. These methods, tailored for specific augmentation tasks, highlight the importance of designing synthesis procedures around downstream challenges, leading to better-performing models and higher-quality generated images.</p>
</div>
<div id="S3.SS3.SSS6.p5" class="ltx_para">
<p id="S3.SS3.SSS6.p5.1" class="ltx_p">In the realm of medical text-to-image synthesis, latent diffusion models, as described in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib62" title="" class="ltx_ref">62</a>]</cite>, employ strategies like reduced image sizes conditioned on textual inputs and incorporate reverse diffusion steps to enhance denoising. Notably, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib295" title="" class="ltx_ref">295</a>]</cite> introduces semantic diffusion, which involves translating latent variables into image space using a decoder and refining/upscaling through a super-resolution diffusion process. Models aiming to align generated reports with X-ray images, as discussed in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib296" title="" class="ltx_ref">296</a>, <a href="#bib.bib297" title="" class="ltx_ref">297</a>, <a href="#bib.bib298" title="" class="ltx_ref">298</a>]</cite>, encounter challenges such as false positives <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib62" title="" class="ltx_ref">62</a>, <a href="#bib.bib299" title="" class="ltx_ref">299</a>]</cite>. Joint learning of language embeddings and image generation reveals limitations, prompting <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib295" title="" class="ltx_ref">295</a>]</cite> to propose a cascaded latent diffusion model that includes an autoencoder component. Despite the advancements, comprehensive evaluations of image quality and clinical applicability, as highlighted in studies like <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib295" title="" class="ltx_ref">295</a>, <a href="#bib.bib296" title="" class="ltx_ref">296</a>, <a href="#bib.bib297" title="" class="ltx_ref">297</a>]</cite>, are still lacking. Furthermore, the potential utility of these models in data augmentation remains largely unexplored, and there is a paucity of comparative analyses against other state-of-the-art methodologies.</p>
</div>
<figure id="S3.T11" class="ltx_table">
<table id="S3.T11.7" class="ltx_tabular ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T11.7.8.1" class="ltx_tr">
<td id="S3.T11.7.8.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T11.7.8.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T11.7.8.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T11.7.8.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T11.7.8.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T11.7.8.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T11.7.8.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T11.7.8.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T11.7.8.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T11.7.8.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T11.7.8.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T11.7.8.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T11.7.8.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T11.7.8.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T11.7.8.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T11.7.8.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T11.7.9.2" class="ltx_tr">
<td id="S3.T11.7.9.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T11.7.9.2.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">U</span></td>
<td id="S3.T11.7.9.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T11.7.9.2.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">F</span></td>
<td id="S3.T11.7.9.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T11.7.9.2.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">D</span></td>
<td id="S3.T11.7.9.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T11.7.9.2.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Q</span></td>
<td id="S3.T11.7.9.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T11.7.9.2.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">C</span></td>
<td id="S3.T11.7.9.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T11.7.9.2.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T11.7.10.3" class="ltx_tr">
<td id="S3.T11.7.10.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.10.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T11.7.10.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.10.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T11.7.11.4" class="ltx_tr">
<td id="S3.T11.7.11.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="8"><span id="S3.T11.7.11.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T11.7.11.4.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="8"><span id="S3.T11.7.11.4.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T11.7.11.4.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.7.11.4.3.1" class="ltx_text" style="font-size:70%;">Knee joint XR</span></td>
<td id="S3.T11.7.11.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.7.11.4.4.1" class="ltx_text" style="font-size:70%;">WGAN</span></td>
<td id="S3.T11.7.11.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.11.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib281" title="" class="ltx_ref">281</a><span id="S3.T11.7.11.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.11.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.7.11.4.6.1" class="ltx_text" style="font-size:70%;">Class.*</span></td>
<td id="S3.T11.7.11.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.11.4.7.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T11.7.11.4.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.11.4.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.11.4.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.11.4.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.11.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.11.4.11" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.11.4.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.11.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.11.4.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.11.4.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.11.4.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T11.7.12.5" class="ltx_tr">
<td id="S3.T11.7.12.5.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.12.5.1.1" class="ltx_text" style="font-size:70%;">Pneumonia</span></td>
<td id="S3.T11.7.12.5.2" class="ltx_td ltx_align_left"><span id="S3.T11.7.12.5.2.1" class="ltx_text" style="font-size:70%;">DCGAN + PGGAN</span></td>
<td id="S3.T11.7.12.5.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.12.5.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib283" title="" class="ltx_ref">283</a><span id="S3.T11.7.12.5.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.12.5.4" class="ltx_td ltx_align_left"><span id="S3.T11.7.12.5.4.1" class="ltx_text" style="font-size:70%;">Class.*</span></td>
<td id="S3.T11.7.12.5.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.12.5.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T11.7.12.5.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.12.5.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.12.5.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.12.5.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.12.5.8" class="ltx_td"></td>
<td id="S3.T11.7.12.5.9" class="ltx_td"></td>
<td id="S3.T11.7.12.5.10" class="ltx_td"></td>
<td id="S3.T11.7.12.5.11" class="ltx_td"></td>
<td id="S3.T11.7.12.5.12" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.13.6" class="ltx_tr">
<td id="S3.T11.7.13.6.1" class="ltx_td ltx_align_left" rowspan="3"><span id="S3.T11.7.13.6.1.1" class="ltx_text" style="font-size:70%;">COVID19</span></td>
<td id="S3.T11.7.13.6.2" class="ltx_td ltx_align_left"><span id="S3.T11.7.13.6.2.1" class="ltx_text" style="font-size:70%;">DCGAN + WGAN-GP</span></td>
<td id="S3.T11.7.13.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.13.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib282" title="" class="ltx_ref">282</a><span id="S3.T11.7.13.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.13.6.4" class="ltx_td ltx_align_left"><span id="S3.T11.7.13.6.4.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.7.13.6.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.13.6.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T11.7.13.6.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.13.6.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.13.6.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.13.6.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.13.6.8" class="ltx_td"></td>
<td id="S3.T11.7.13.6.9" class="ltx_td"></td>
<td id="S3.T11.7.13.6.10" class="ltx_td"></td>
<td id="S3.T11.7.13.6.11" class="ltx_td"></td>
<td id="S3.T11.7.13.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.14.7" class="ltx_tr">
<td id="S3.T11.7.14.7.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.14.7.1.1" class="ltx_text" style="font-size:70%;">MSG-SAGAN</span></td>
<td id="S3.T11.7.14.7.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.14.7.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib286" title="" class="ltx_ref">286</a><span id="S3.T11.7.14.7.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.14.7.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.14.7.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T11.7.14.7.4" class="ltx_td"></td>
<td id="S3.T11.7.14.7.5" class="ltx_td"></td>
<td id="S3.T11.7.14.7.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.14.7.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.14.7.7" class="ltx_td"></td>
<td id="S3.T11.7.14.7.8" class="ltx_td"></td>
<td id="S3.T11.7.14.7.9" class="ltx_td"></td>
<td id="S3.T11.7.14.7.10" class="ltx_td"></td>
<td id="S3.T11.7.14.7.11" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.15.8" class="ltx_tr">
<td id="S3.T11.7.15.8.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.15.8.1.1" class="ltx_text" style="font-size:70%;">MLPs, CNNs</span></td>
<td id="S3.T11.7.15.8.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.15.8.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib287" title="" class="ltx_ref">287</a><span id="S3.T11.7.15.8.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.15.8.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.15.8.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T11.7.15.8.4" class="ltx_td"></td>
<td id="S3.T11.7.15.8.5" class="ltx_td"></td>
<td id="S3.T11.7.15.8.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.15.8.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.15.8.7" class="ltx_td"></td>
<td id="S3.T11.7.15.8.8" class="ltx_td"></td>
<td id="S3.T11.7.15.8.9" class="ltx_td"></td>
<td id="S3.T11.7.15.8.10" class="ltx_td"></td>
<td id="S3.T11.7.15.8.11" class="ltx_td ltx_align_center"><span id="S3.T11.7.15.8.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T11.7.16.9" class="ltx_tr">
<td id="S3.T11.7.16.9.1" class="ltx_td ltx_align_left" rowspan="3"><span id="S3.T11.7.16.9.1.1" class="ltx_text" style="font-size:70%;">CXR</span></td>
<td id="S3.T11.7.16.9.2" class="ltx_td ltx_align_left" rowspan="3"><span id="S3.T11.7.16.9.2.1" class="ltx_text" style="font-size:70%;">PGGAN</span></td>
<td id="S3.T11.7.16.9.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.16.9.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib288" title="" class="ltx_ref">288</a><span id="S3.T11.7.16.9.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.16.9.4" class="ltx_td ltx_align_left"><span id="S3.T11.7.16.9.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T11.7.16.9.5" class="ltx_td"></td>
<td id="S3.T11.7.16.9.6" class="ltx_td"></td>
<td id="S3.T11.7.16.9.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.16.9.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.16.9.8" class="ltx_td ltx_align_center"><span id="S3.T11.7.16.9.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.16.9.9" class="ltx_td"></td>
<td id="S3.T11.7.16.9.10" class="ltx_td"></td>
<td id="S3.T11.7.16.9.11" class="ltx_td"></td>
<td id="S3.T11.7.16.9.12" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.17.10" class="ltx_tr">
<td id="S3.T11.7.17.10.1" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.17.10.1.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib285" title="" class="ltx_ref">285</a><span id="S3.T11.7.17.10.1.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.17.10.2" class="ltx_td ltx_align_left"><span id="S3.T11.7.17.10.2.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.7.17.10.3" class="ltx_td ltx_align_center"><span id="S3.T11.7.17.10.3.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T11.7.17.10.4" class="ltx_td ltx_align_center"><span id="S3.T11.7.17.10.4.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.17.10.5" class="ltx_td"></td>
<td id="S3.T11.7.17.10.6" class="ltx_td"></td>
<td id="S3.T11.7.17.10.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.17.10.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.17.10.8" class="ltx_td"></td>
<td id="S3.T11.7.17.10.9" class="ltx_td"></td>
<td id="S3.T11.7.17.10.10" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.18.11" class="ltx_tr">
<td id="S3.T11.7.18.11.1" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.18.11.1.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib289" title="" class="ltx_ref">289</a><span id="S3.T11.7.18.11.1.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.18.11.2" class="ltx_td ltx_align_left"><span id="S3.T11.7.18.11.2.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.7.18.11.3" class="ltx_td ltx_align_center"><span id="S3.T11.7.18.11.3.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.18.11.4" class="ltx_td ltx_align_center"><span id="S3.T11.7.18.11.4.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T11.7.18.11.5" class="ltx_td"></td>
<td id="S3.T11.7.18.11.6" class="ltx_td"></td>
<td id="S3.T11.7.18.11.7" class="ltx_td"></td>
<td id="S3.T11.7.18.11.8" class="ltx_td ltx_align_center"><span id="S3.T11.7.18.11.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.18.11.9" class="ltx_td"></td>
<td id="S3.T11.7.18.11.10" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.19.12" class="ltx_tr">
<td id="S3.T11.7.19.12.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="4"><span id="S3.T11.7.19.12.1.1" class="ltx_text" style="font-size:70%;">Class or Label Cond.</span></td>
<td id="S3.T11.7.19.12.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T11.7.19.12.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T11.7.19.12.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="4"><span id="S3.T11.7.19.12.3.1" class="ltx_text" style="font-size:70%;">CXR</span></td>
<td id="S3.T11.7.19.12.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.7.19.12.4.1" class="ltx_text" style="font-size:70%;">DCGAN + patchGAN</span></td>
<td id="S3.T11.7.19.12.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.19.12.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib292" title="" class="ltx_ref">292</a><span id="S3.T11.7.19.12.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.19.12.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.7.19.12.6.1" class="ltx_text" style="font-size:70%;">Det.</span></td>
<td id="S3.T11.7.19.12.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.19.12.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T11.7.19.12.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.19.12.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.19.12.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.19.12.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.19.12.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.19.12.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.19.12.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.19.12.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.19.12.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.19.12.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T11.7.20.13" class="ltx_tr">
<td id="S3.T11.7.20.13.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.20.13.1.1" class="ltx_text" style="font-size:70%;">StyleGAN2 + Bayesian recon.</span></td>
<td id="S3.T11.7.20.13.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.20.13.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib293" title="" class="ltx_ref">293</a><span id="S3.T11.7.20.13.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.20.13.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.20.13.3.1" class="ltx_text" style="font-size:70%;">Class.*</span></td>
<td id="S3.T11.7.20.13.4" class="ltx_td ltx_align_center"><span id="S3.T11.7.20.13.4.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T11.7.20.13.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.20.13.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.20.13.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.20.13.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.20.13.7" class="ltx_td"></td>
<td id="S3.T11.7.20.13.8" class="ltx_td"></td>
<td id="S3.T11.7.20.13.9" class="ltx_td"></td>
<td id="S3.T11.7.20.13.10" class="ltx_td"></td>
<td id="S3.T11.7.20.13.11" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.21.14" class="ltx_tr">
<td id="S3.T11.7.21.14.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.21.14.1.1" class="ltx_text" style="font-size:70%;">PGGAN + Pix2PixHD</span></td>
<td id="S3.T11.7.21.14.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.21.14.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib294" title="" class="ltx_ref">294</a><span id="S3.T11.7.21.14.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.21.14.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.21.14.3.1" class="ltx_text" style="font-size:70%;">Seg.*</span></td>
<td id="S3.T11.7.21.14.4" class="ltx_td ltx_align_center"><span id="S3.T11.7.21.14.4.1" class="ltx_text" style="font-size:70%;">R, S</span></td>
<td id="S3.T11.7.21.14.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.21.14.5.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T11.7.21.14.6" class="ltx_td"></td>
<td id="S3.T11.7.21.14.7" class="ltx_td"></td>
<td id="S3.T11.7.21.14.8" class="ltx_td"></td>
<td id="S3.T11.7.21.14.9" class="ltx_td"></td>
<td id="S3.T11.7.21.14.10" class="ltx_td"></td>
<td id="S3.T11.7.21.14.11" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.22.15" class="ltx_tr">
<td id="S3.T11.7.22.15.1" class="ltx_td ltx_align_center"><span id="S3.T11.7.22.15.1.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T11.7.22.15.2" class="ltx_td ltx_align_left"><span id="S3.T11.7.22.15.2.1" class="ltx_text" style="font-size:70%;">DDPMs</span></td>
<td id="S3.T11.7.22.15.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.22.15.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib290" title="" class="ltx_ref">290</a><span id="S3.T11.7.22.15.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.22.15.4" class="ltx_td ltx_align_left"><span id="S3.T11.7.22.15.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T11.7.22.15.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.22.15.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T11.7.22.15.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.22.15.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.22.15.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.22.15.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.22.15.8" class="ltx_td ltx_align_center"><span id="S3.T11.7.22.15.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.22.15.9" class="ltx_td"></td>
<td id="S3.T11.7.22.15.10" class="ltx_td"></td>
<td id="S3.T11.7.22.15.11" class="ltx_td"></td>
<td id="S3.T11.7.22.15.12" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.23.16" class="ltx_tr">
<td id="S3.T11.7.23.16.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S3.T11.7.23.16.1.1" class="ltx_text" style="font-size:70%;">Attribute conditioning</span></td>
<td id="S3.T11.7.23.16.2" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S3.T11.7.23.16.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T11.7.23.16.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.7.23.16.3.1" class="ltx_text" style="font-size:70%;">CXR</span></td>
<td id="S3.T11.7.23.16.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.7.23.16.4.1" class="ltx_text" style="font-size:70%;">PGGAN with latent space optim.</span></td>
<td id="S3.T11.7.23.16.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.23.16.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib284" title="" class="ltx_ref">284</a><span id="S3.T11.7.23.16.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.23.16.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.7.23.16.6.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.7.23.16.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.23.16.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T11.7.23.16.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.23.16.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.23.16.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.23.16.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.23.16.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.23.16.11" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.7.23.16.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.23.16.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.23.16.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.7.23.16.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T11.7.24.17" class="ltx_tr">
<td id="S3.T11.7.24.17.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.24.17.1.1" class="ltx_text" style="font-size:70%;">COVID19</span></td>
<td id="S3.T11.7.24.17.2" class="ltx_td ltx_align_left"><span id="S3.T11.7.24.17.2.1" class="ltx_text" style="font-size:70%;">SD-GAN</span></td>
<td id="S3.T11.7.24.17.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.24.17.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib291" title="" class="ltx_ref">291</a><span id="S3.T11.7.24.17.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.24.17.4" class="ltx_td ltx_align_left"><span id="S3.T11.7.24.17.4.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.7.24.17.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.24.17.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T11.7.24.17.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.24.17.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.24.17.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.24.17.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.24.17.8" class="ltx_td ltx_align_center"><span id="S3.T11.7.24.17.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.24.17.9" class="ltx_td"></td>
<td id="S3.T11.7.24.17.10" class="ltx_td"></td>
<td id="S3.T11.7.24.17.11" class="ltx_td"></td>
<td id="S3.T11.7.24.17.12" class="ltx_td ltx_align_center"><span id="S3.T11.7.24.17.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T11.1.1" class="ltx_tr">
<td id="S3.T11.1.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T11.1.1.2.1" class="ltx_text" style="font-size:70%;">Text-guided</span></td>
<td id="S3.T11.1.1.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T11.1.1.3.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T11.1.1.1" class="ltx_td ltx_align_left ltx_border_t">
<span id="S3.T11.1.1.1.1" class="ltx_text" style="font-size:70%;">CXR </span><math id="S3.T11.1.1.1.m1.1" class="ltx_Math" alttext="\&amp;" display="inline"><semantics id="S3.T11.1.1.1.m1.1a"><mo mathsize="70%" id="S3.T11.1.1.1.m1.1.1" xref="S3.T11.1.1.1.m1.1.1.cmml">&amp;</mo><annotation-xml encoding="MathML-Content" id="S3.T11.1.1.1.m1.1b"><and id="S3.T11.1.1.1.m1.1.1.cmml" xref="S3.T11.1.1.1.m1.1.1"></and></annotation-xml><annotation encoding="application/x-tex" id="S3.T11.1.1.1.m1.1c">\&amp;</annotation></semantics></math><span id="S3.T11.1.1.1.2" class="ltx_text" style="font-size:70%;"> Reports</span>
</td>
<td id="S3.T11.1.1.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.1.1.4.1" class="ltx_text" style="font-size:70%;">Semantic DM &amp; SR diffusion</span></td>
<td id="S3.T11.1.1.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.1.1.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib295" title="" class="ltx_ref">295</a><span id="S3.T11.1.1.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.1.1.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.1.1.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T11.1.1.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.1.1.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.1.1.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.1.1.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.1.1.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.1.1.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.1.1.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.1.1.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.1.1.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.1.1.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.1.1.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T11.2.2" class="ltx_tr">
<td id="S3.T11.2.2.1" class="ltx_td ltx_align_left">
<span id="S3.T11.2.2.1.1" class="ltx_text" style="font-size:70%;">CXR </span><math id="S3.T11.2.2.1.m1.1" class="ltx_Math" alttext="\&amp;" display="inline"><semantics id="S3.T11.2.2.1.m1.1a"><mo mathsize="70%" id="S3.T11.2.2.1.m1.1.1" xref="S3.T11.2.2.1.m1.1.1.cmml">&amp;</mo><annotation-xml encoding="MathML-Content" id="S3.T11.2.2.1.m1.1b"><and id="S3.T11.2.2.1.m1.1.1.cmml" xref="S3.T11.2.2.1.m1.1.1"></and></annotation-xml><annotation encoding="application/x-tex" id="S3.T11.2.2.1.m1.1c">\&amp;</annotation></semantics></math><span id="S3.T11.2.2.1.2" class="ltx_text" style="font-size:70%;"> Impressions</span>
</td>
<td id="S3.T11.2.2.2" class="ltx_td ltx_align_left"><span id="S3.T11.2.2.2.1" class="ltx_text" style="font-size:70%;">Stable diffusion v2</span></td>
<td id="S3.T11.2.2.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.2.2.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib62" title="" class="ltx_ref">62</a><span id="S3.T11.2.2.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.2.2.4" class="ltx_td ltx_align_left"><span id="S3.T11.2.2.4.1" class="ltx_text" style="font-size:70%;">Det.</span></td>
<td id="S3.T11.2.2.5" class="ltx_td"></td>
<td id="S3.T11.2.2.6" class="ltx_td ltx_align_center"><span id="S3.T11.2.2.6.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T11.2.2.7" class="ltx_td ltx_align_center"><span id="S3.T11.2.2.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.2.2.8" class="ltx_td ltx_align_center"><span id="S3.T11.2.2.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.2.2.9" class="ltx_td"></td>
<td id="S3.T11.2.2.10" class="ltx_td"></td>
<td id="S3.T11.2.2.11" class="ltx_td"></td>
<td id="S3.T11.2.2.12" class="ltx_td ltx_align_center"><span id="S3.T11.2.2.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T11.3.3" class="ltx_tr">
<td id="S3.T11.3.3.2" class="ltx_td ltx_align_left" rowspan="4"><span id="S3.T11.3.3.2.1" class="ltx_text" style="font-size:70%;">Hybrid</span></td>
<td id="S3.T11.3.3.1" class="ltx_td ltx_align_left" rowspan="4"><span id="S3.T11.3.3.1.1" class="ltx_text" style="font-size:70%;">CXR <math id="S3.T11.3.3.1.1.m1.1" class="ltx_Math" alttext="\&amp;" display="inline"><semantics id="S3.T11.3.3.1.1.m1.1a"><mo id="S3.T11.3.3.1.1.m1.1.1" xref="S3.T11.3.3.1.1.m1.1.1.cmml">&amp;</mo><annotation-xml encoding="MathML-Content" id="S3.T11.3.3.1.1.m1.1b"><and id="S3.T11.3.3.1.1.m1.1.1.cmml" xref="S3.T11.3.3.1.1.m1.1.1"></and></annotation-xml><annotation encoding="application/x-tex" id="S3.T11.3.3.1.1.m1.1c">\&amp;</annotation></semantics></math> Reports</span></td>
<td id="S3.T11.3.3.3" class="ltx_td ltx_align_left"><span id="S3.T11.3.3.3.1" class="ltx_text" style="font-size:70%;">VQ-GAN + GPT-Neox LLM</span></td>
<td id="S3.T11.3.3.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.3.3.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib297" title="" class="ltx_ref">297</a><span id="S3.T11.3.3.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.3.3.5" class="ltx_td ltx_align_left"><span id="S3.T11.3.3.5.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T11.3.3.6" class="ltx_td"></td>
<td id="S3.T11.3.3.7" class="ltx_td"></td>
<td id="S3.T11.3.3.8" class="ltx_td ltx_align_center"><span id="S3.T11.3.3.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.3.3.9" class="ltx_td ltx_align_center"><span id="S3.T11.3.3.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.3.3.10" class="ltx_td"></td>
<td id="S3.T11.3.3.11" class="ltx_td ltx_align_center"><span id="S3.T11.3.3.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.3.3.12" class="ltx_td"></td>
<td id="S3.T11.3.3.13" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.25.18" class="ltx_tr">
<td id="S3.T11.7.25.18.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.25.18.1.1" class="ltx_text" style="font-size:70%;">SDM-VAE + CLIP text-enc.</span></td>
<td id="S3.T11.7.25.18.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.25.18.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib299" title="" class="ltx_ref">299</a><span id="S3.T11.7.25.18.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.25.18.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.25.18.3.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.7.25.18.4" class="ltx_td ltx_align_center"><span id="S3.T11.7.25.18.4.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T11.7.25.18.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.25.18.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.25.18.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.25.18.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.25.18.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.25.18.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.25.18.8" class="ltx_td"></td>
<td id="S3.T11.7.25.18.9" class="ltx_td ltx_align_center"><span id="S3.T11.7.25.18.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.25.18.10" class="ltx_td"></td>
<td id="S3.T11.7.25.18.11" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.26.19" class="ltx_tr">
<td id="S3.T11.7.26.19.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.26.19.1.1" class="ltx_text" style="font-size:70%;">VQ-GAN + bidirec. transf.</span></td>
<td id="S3.T11.7.26.19.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.26.19.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib298" title="" class="ltx_ref">298</a><span id="S3.T11.7.26.19.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.26.19.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.26.19.3.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.7.26.19.4" class="ltx_td"></td>
<td id="S3.T11.7.26.19.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.26.19.5.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T11.7.26.19.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.26.19.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.26.19.7" class="ltx_td"></td>
<td id="S3.T11.7.26.19.8" class="ltx_td ltx_align_center"><span id="S3.T11.7.26.19.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.26.19.9" class="ltx_td ltx_align_center"><span id="S3.T11.7.26.19.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.26.19.10" class="ltx_td"></td>
<td id="S3.T11.7.26.19.11" class="ltx_td ltx_align_center"><span id="S3.T11.7.26.19.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T11.7.27.20" class="ltx_tr">
<td id="S3.T11.7.27.20.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.27.20.1.1" class="ltx_text" style="font-size:70%;">Cycle GAN + LSTM</span></td>
<td id="S3.T11.7.27.20.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.27.20.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib296" title="" class="ltx_ref">296</a><span id="S3.T11.7.27.20.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.27.20.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.27.20.3.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T11.7.27.20.4" class="ltx_td"></td>
<td id="S3.T11.7.27.20.5" class="ltx_td"></td>
<td id="S3.T11.7.27.20.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.27.20.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.27.20.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.27.20.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.27.20.8" class="ltx_td ltx_align_center"><span id="S3.T11.7.27.20.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.27.20.9" class="ltx_td"></td>
<td id="S3.T11.7.27.20.10" class="ltx_td"></td>
<td id="S3.T11.7.27.20.11" class="ltx_td"></td>
</tr>
<tr id="S3.T11.4.4" class="ltx_tr">
<td id="S3.T11.4.4.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T11.4.4.2.1" class="ltx_text" style="font-size:70%;">Intra-modal trans.</span></td>
<td id="S3.T11.4.4.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T11.4.4.3.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T11.4.4.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S3.T11.4.4.1.1" class="ltx_text" style="font-size:70%;">Normal <math id="S3.T11.4.4.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T11.4.4.1.1.m1.1a"><mo stretchy="false" id="S3.T11.4.4.1.1.m1.1.1" xref="S3.T11.4.4.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T11.4.4.1.1.m1.1b"><ci id="S3.T11.4.4.1.1.m1.1.1.cmml" xref="S3.T11.4.4.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T11.4.4.1.1.m1.1c">\rightarrow</annotation></semantics></math> Abn. CXR</span></td>
<td id="S3.T11.4.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.4.4.4.1" class="ltx_text" style="font-size:70%;">cond. GAN + spatial transf.</span></td>
<td id="S3.T11.4.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.4.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib300" title="" class="ltx_ref">300</a><span id="S3.T11.4.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.4.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T11.4.4.6.1" class="ltx_text" style="font-size:70%;">Det.</span></td>
<td id="S3.T11.4.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.4.4.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T11.4.4.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T11.4.4.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.4.4.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.4.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.4.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.4.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.4.4.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T11.4.4.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T11.7.28.21" class="ltx_tr">
<td id="S3.T11.7.28.21.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.28.21.1.1" class="ltx_text" style="font-size:70%;">Cycle GAN + classifier</span></td>
<td id="S3.T11.7.28.21.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.28.21.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib301" title="" class="ltx_ref">301</a><span id="S3.T11.7.28.21.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.28.21.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.28.21.3.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.7.28.21.4" class="ltx_td ltx_align_center"><span id="S3.T11.7.28.21.4.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T11.7.28.21.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.28.21.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.28.21.6" class="ltx_td"></td>
<td id="S3.T11.7.28.21.7" class="ltx_td"></td>
<td id="S3.T11.7.28.21.8" class="ltx_td"></td>
<td id="S3.T11.7.28.21.9" class="ltx_td"></td>
<td id="S3.T11.7.28.21.10" class="ltx_td"></td>
<td id="S3.T11.7.28.21.11" class="ltx_td"></td>
</tr>
<tr id="S3.T11.5.5" class="ltx_tr">
<td id="S3.T11.5.5.1" class="ltx_td ltx_align_left" rowspan="2"><span id="S3.T11.5.5.1.1" class="ltx_text" style="font-size:70%;">Normal <math id="S3.T11.5.5.1.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T11.5.5.1.1.m1.1a"><mo stretchy="false" id="S3.T11.5.5.1.1.m1.1.1" xref="S3.T11.5.5.1.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T11.5.5.1.1.m1.1b"><ci id="S3.T11.5.5.1.1.m1.1.1.cmml" xref="S3.T11.5.5.1.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T11.5.5.1.1.m1.1c">\rightarrow</annotation></semantics></math> COVID19</span></td>
<td id="S3.T11.5.5.2" class="ltx_td ltx_align_left"><span id="S3.T11.5.5.2.1" class="ltx_text" style="font-size:70%;">Adaptive Cycle GAN</span></td>
<td id="S3.T11.5.5.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.5.5.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib302" title="" class="ltx_ref">302</a><span id="S3.T11.5.5.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.5.5.4" class="ltx_td ltx_align_left"><span id="S3.T11.5.5.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T11.5.5.5" class="ltx_td"></td>
<td id="S3.T11.5.5.6" class="ltx_td"></td>
<td id="S3.T11.5.5.7" class="ltx_td ltx_align_center"><span id="S3.T11.5.5.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.5.5.8" class="ltx_td"></td>
<td id="S3.T11.5.5.9" class="ltx_td ltx_align_center"><span id="S3.T11.5.5.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.5.5.10" class="ltx_td"></td>
<td id="S3.T11.5.5.11" class="ltx_td"></td>
<td id="S3.T11.5.5.12" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.29.22" class="ltx_tr">
<td id="S3.T11.7.29.22.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.29.22.1.1" class="ltx_text" style="font-size:70%;">Cycle GAN</span></td>
<td id="S3.T11.7.29.22.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.29.22.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib303" title="" class="ltx_ref">303</a><span id="S3.T11.7.29.22.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.29.22.3" class="ltx_td ltx_align_left"><span id="S3.T11.7.29.22.3.1" class="ltx_text" style="font-size:70%;">Det.*</span></td>
<td id="S3.T11.7.29.22.4" class="ltx_td ltx_align_center"><span id="S3.T11.7.29.22.4.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T11.7.29.22.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.29.22.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.29.22.6" class="ltx_td"></td>
<td id="S3.T11.7.29.22.7" class="ltx_td"></td>
<td id="S3.T11.7.29.22.8" class="ltx_td"></td>
<td id="S3.T11.7.29.22.9" class="ltx_td"></td>
<td id="S3.T11.7.29.22.10" class="ltx_td"></td>
<td id="S3.T11.7.29.22.11" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.30.23" class="ltx_tr">
<td id="S3.T11.7.30.23.1" class="ltx_td ltx_align_left"><span id="S3.T11.7.30.23.1.1" class="ltx_text" style="font-size:70%;">Urinary stones</span></td>
<td id="S3.T11.7.30.23.2" class="ltx_td ltx_align_left"><span id="S3.T11.7.30.23.2.1" class="ltx_text" style="font-size:70%;">U-Net cond. GAN</span></td>
<td id="S3.T11.7.30.23.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.30.23.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib304" title="" class="ltx_ref">304</a><span id="S3.T11.7.30.23.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.30.23.4" class="ltx_td ltx_align_left"><span id="S3.T11.7.30.23.4.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T11.7.30.23.5" class="ltx_td ltx_align_center"><span id="S3.T11.7.30.23.5.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T11.7.30.23.6" class="ltx_td ltx_align_center"><span id="S3.T11.7.30.23.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T11.7.30.23.7" class="ltx_td ltx_align_center"><span id="S3.T11.7.30.23.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.30.23.8" class="ltx_td"></td>
<td id="S3.T11.7.30.23.9" class="ltx_td"></td>
<td id="S3.T11.7.30.23.10" class="ltx_td"></td>
<td id="S3.T11.7.30.23.11" class="ltx_td"></td>
<td id="S3.T11.7.30.23.12" class="ltx_td"></td>
</tr>
<tr id="S3.T11.6.6" class="ltx_tr">
<td id="S3.T11.6.6.1" class="ltx_td ltx_align_left">
<span id="S3.T11.6.6.1.1" class="ltx_text" style="font-size:70%;">Normal </span><math id="S3.T11.6.6.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T11.6.6.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T11.6.6.1.m1.1.1" xref="S3.T11.6.6.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T11.6.6.1.m1.1b"><ci id="S3.T11.6.6.1.m1.1.1.cmml" xref="S3.T11.6.6.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T11.6.6.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T11.6.6.1.2" class="ltx_text" style="font-size:70%;"> Pneumonia</span>
</td>
<td id="S3.T11.6.6.2" class="ltx_td ltx_align_left"><span id="S3.T11.6.6.2.1" class="ltx_text" style="font-size:70%;">Cond. + Cycle GAN</span></td>
<td id="S3.T11.6.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.6.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib305" title="" class="ltx_ref">305</a><span id="S3.T11.6.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.6.6.4" class="ltx_td ltx_align_left"><span id="S3.T11.6.6.4.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T11.6.6.5" class="ltx_td"></td>
<td id="S3.T11.6.6.6" class="ltx_td ltx_align_center"><span id="S3.T11.6.6.6.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T11.6.6.7" class="ltx_td"></td>
<td id="S3.T11.6.6.8" class="ltx_td"></td>
<td id="S3.T11.6.6.9" class="ltx_td"></td>
<td id="S3.T11.6.6.10" class="ltx_td"></td>
<td id="S3.T11.6.6.11" class="ltx_td"></td>
<td id="S3.T11.6.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T11.7.7" class="ltx_tr">
<td id="S3.T11.7.7.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T11.7.7.2.1" class="ltx_text" style="font-size:70%;">Inter-modal trans.</span></td>
<td id="S3.T11.7.7.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T11.7.7.3.1" class="ltx_text" style="font-size:70%;">Hybrid</span></td>
<td id="S3.T11.7.7.1" class="ltx_td ltx_align_left ltx_border_b ltx_border_t">
<span id="S3.T11.7.7.1.1" class="ltx_text" style="font-size:70%;">Spine US </span><math id="S3.T11.7.7.1.m1.1" class="ltx_Math" alttext="\rightarrow" display="inline"><semantics id="S3.T11.7.7.1.m1.1a"><mo mathsize="70%" stretchy="false" id="S3.T11.7.7.1.m1.1.1" xref="S3.T11.7.7.1.m1.1.1.cmml">→</mo><annotation-xml encoding="MathML-Content" id="S3.T11.7.7.1.m1.1b"><ci id="S3.T11.7.7.1.m1.1.1.cmml" xref="S3.T11.7.7.1.m1.1.1">→</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T11.7.7.1.m1.1c">\rightarrow</annotation></semantics></math><span id="S3.T11.7.7.1.2" class="ltx_text" style="font-size:70%;"> XR</span>
</td>
<td id="S3.T11.7.7.4" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T11.7.7.4.1" class="ltx_text" style="font-size:70%;">DDPM with ViT</span></td>
<td id="S3.T11.7.7.5" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T11.7.7.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib306" title="" class="ltx_ref">306</a><span id="S3.T11.7.7.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T11.7.7.6" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><span id="S3.T11.7.7.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T11.7.7.7" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T11.7.7.8" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T11.7.7.9" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T11.7.7.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.7.10" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T11.7.7.11" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T11.7.7.12" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T11.7.7.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T11.7.7.13" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T11.7.7.14" class="ltx_td ltx_border_b ltx_border_t"></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 11: </span>Overview of all summarized and discussed studies on <span id="S3.T11.12.1" class="ltx_text ltx_font_bold">X-Ray</span> (XR) synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), whereby automated machine learning models are trained with real (R), synthetic (S) or augmented with synthetic data (A), and tested on real or synthetic data, fidelity (F), diversity (D), visual quality (Q), clinical utility (C) and privacy (P) evaluation. * stands for studies that have utilized synthesized images for pre-training. CXR stands for Chest X-Ray. Papers that specify the downstream task without training data, signify instances where a pre-existing model was directly applied or tested on synthetic data. </figcaption>
</figure>
<div id="S3.SS3.SSS6.p6" class="ltx_para">
<p id="S3.SS3.SSS6.p6.1" class="ltx_p">In evaluating the quality of generated X-ray images, most studies use downstream tasks and generic fidelity and diversity metrics that potentially overlook nuances specific to medical imaging. However, a few studies, such as <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib281" title="" class="ltx_ref">281</a>]</cite>, conduct human perception quality and clinical usability evaluations. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib281" title="" class="ltx_ref">281</a>]</cite> performed a Visual Turing test with specialists, revealing that synthetic X-ray images are often mistaken for real ones, suggesting sufficient realism for deceiving experts. Another study, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib285" title="" class="ltx_ref">285</a>]</cite>, conducted an extensive Turing test with 400 images, demonstrating that radiologists struggled to distinguish between synthetic and real CXRs, especially in abnormal cases, containing lung lesions. In the context of CXRs, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib284" title="" class="ltx_ref">284</a>]</cite> finds that synthetic data is often mistaken for real, but improved replication of fine anatomical details is required for truly indistinguishable samples. The study also evaluates FID’s applicability to synthetic CXRs, noting that small-scale details may not be reproduced at the highest quality. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib298" title="" class="ltx_ref">298</a>]</cite> conducts human evaluation with board-certified clinicians, revealing that the model generates realistic CXRs but lacks precise alignment with reports. The proposed method excels in generating view-specific X-rays but occasionally struggles with fine details. Furthermore, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib302" title="" class="ltx_ref">302</a>]</cite> employs the Universal Image Quality Index (UIQI) and Visual Information Fidelity (VIF) to compare synthetic and real images. The quantitative metrics suggest challenges for GANs in translating high-quality synthetic images, possibly due to insufficient training samples. A few studies offer insights into the use of synthetic images for clinical tasks. For instance, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib298" title="" class="ltx_ref">298</a>]</cite> employs synthetic images for a 14-class diagnosis of anomalies in X-ray images, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib289" title="" class="ltx_ref">289</a>]</cite> calculates ratios to classify cardiomegaly in CXR images, and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib297" title="" class="ltx_ref">297</a>]</cite> evaluates findings generated from synthetic images, identifying false positives and misdiagnoses. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib306" title="" class="ltx_ref">306</a>]</cite> measures the Cobb angle difference between synthetic and original X-ray images, demonstrating effective use in evaluating scoliosis. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib281" title="" class="ltx_ref">281</a>]</cite> explores privacy concerns by assessing the nearest real image neighbors to synthetic images, used as criteria for selecting training data to ensure privacy in downstream tasks; however, a proper evaluation of preserved privacy is lacking.</p>
</div>
</section>
<section id="S3.SS3.SSS7" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.7 </span>OCT</h4>

<figure id="S3.T12" class="ltx_table">
<table id="S3.T12.40" class="ltx_tabular ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T12.40.41.1" class="ltx_tr">
<td id="S3.T12.40.41.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T12.40.41.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T12.40.41.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T12.40.41.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T12.40.41.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T12.40.41.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></td>
<td id="S3.T12.40.41.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T12.40.41.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T12.40.41.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T12.40.41.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T12.40.41.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T12.40.41.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Downstream Task</span></td>
<td id="S3.T12.40.41.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T12.40.41.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T12.40.41.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T12.40.41.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T12.40.42.2" class="ltx_tr">
<td id="S3.T12.40.42.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T12.40.42.2.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">U</span></td>
<td id="S3.T12.40.42.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T12.40.42.2.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">F</span></td>
<td id="S3.T12.40.42.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T12.40.42.2.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">D</span></td>
<td id="S3.T12.40.42.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T12.40.42.2.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Q</span></td>
<td id="S3.T12.40.42.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T12.40.42.2.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">C</span></td>
<td id="S3.T12.40.42.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T12.40.42.2.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T12.40.43.3" class="ltx_tr">
<td id="S3.T12.40.43.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T12.40.43.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T12.40.43.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T12.40.43.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T12.1.1" class="ltx_tr">
<td id="S3.T12.1.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="7"><span id="S3.T12.1.1.2.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T12.1.1.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="5"><span id="S3.T12.1.1.3.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T12.1.1.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.1.1.4.1" class="ltx_text" style="font-size:70%;">AS OCT</span></td>
<td id="S3.T12.1.1.5" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.1.1.5.1" class="ltx_text" style="font-size:70%;">PGGAN</span></td>
<td id="S3.T12.1.1.6" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.1.1.6.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib307" title="" class="ltx_ref">307</a><span id="S3.T12.1.1.6.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.1.1.7" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.1.1.7.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T12.1.1.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T12.1.1.8.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T12.1.1.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T12.1.1.9.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.1.1.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.1.1.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.1.1.1" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.1.1.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.1.1.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.1.1.1.m1.1.1" xref="S3.T12.1.1.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.1.1.1.m1.1b"><ci id="S3.T12.1.1.1.m1.1.1.cmml" xref="S3.T12.1.1.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.1.1.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.1.1.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.1.1.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.1.1.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T12.3.3" class="ltx_tr">
<td id="S3.T12.3.3.3" class="ltx_td ltx_align_left"><span id="S3.T12.3.3.3.1" class="ltx_text" style="font-size:70%;">AS OCT</span></td>
<td id="S3.T12.3.3.4" class="ltx_td ltx_align_left"><span id="S3.T12.3.3.4.1" class="ltx_text" style="font-size:70%;">Cycle + patch GAN</span></td>
<td id="S3.T12.3.3.5" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.3.3.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib308" title="" class="ltx_ref">308</a><span id="S3.T12.3.3.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.3.3.6" class="ltx_td ltx_align_left"><span id="S3.T12.3.3.6.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T12.3.3.7" class="ltx_td ltx_align_center"><span id="S3.T12.3.3.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T12.3.3.8" class="ltx_td ltx_align_center"><span id="S3.T12.3.3.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.2.2.1" class="ltx_td ltx_align_center"><math id="S3.T12.2.2.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.2.2.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.2.2.1.m1.1.1" xref="S3.T12.2.2.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.2.2.1.m1.1b"><ci id="S3.T12.2.2.1.m1.1.1.cmml" xref="S3.T12.2.2.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.2.2.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.3.3.9" class="ltx_td"></td>
<td id="S3.T12.3.3.10" class="ltx_td"></td>
<td id="S3.T12.3.3.2" class="ltx_td ltx_align_center"><math id="S3.T12.3.3.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.3.3.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.3.3.2.m1.1.1" xref="S3.T12.3.3.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.3.3.2.m1.1b"><ci id="S3.T12.3.3.2.m1.1.1.cmml" xref="S3.T12.3.3.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.3.3.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.3.3.11" class="ltx_td"></td>
<td id="S3.T12.3.3.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.4.4" class="ltx_tr">
<td id="S3.T12.4.4.2" class="ltx_td ltx_align_left"><span id="S3.T12.4.4.2.1" class="ltx_text" style="font-size:70%;">SD DM, NOR, AMD</span></td>
<td id="S3.T12.4.4.3" class="ltx_td ltx_align_left"><span id="S3.T12.4.4.3.1" class="ltx_text" style="font-size:70%;">Vanilla GAN</span></td>
<td id="S3.T12.4.4.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.4.4.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib309" title="" class="ltx_ref">309</a><span id="S3.T12.4.4.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.4.4.5" class="ltx_td ltx_align_left"><span id="S3.T12.4.4.5.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T12.4.4.6" class="ltx_td ltx_align_center"><span id="S3.T12.4.4.6.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T12.4.4.7" class="ltx_td ltx_align_center"><span id="S3.T12.4.4.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.4.4.1" class="ltx_td ltx_align_center"><math id="S3.T12.4.4.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.4.4.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.4.4.1.m1.1.1" xref="S3.T12.4.4.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.4.4.1.m1.1b"><ci id="S3.T12.4.4.1.m1.1.1.cmml" xref="S3.T12.4.4.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.4.4.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.4.4.8" class="ltx_td"></td>
<td id="S3.T12.4.4.9" class="ltx_td"></td>
<td id="S3.T12.4.4.10" class="ltx_td"></td>
<td id="S3.T12.4.4.11" class="ltx_td"></td>
<td id="S3.T12.4.4.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.40.44.4" class="ltx_tr">
<td id="S3.T12.40.44.4.1" class="ltx_td ltx_align_left"><span id="S3.T12.40.44.4.1.1" class="ltx_text" style="font-size:70%;">Retinal OCT</span></td>
<td id="S3.T12.40.44.4.2" class="ltx_td ltx_align_left"><span id="S3.T12.40.44.4.2.1" class="ltx_text" style="font-size:70%;">StyleGAN2-ADA</span></td>
<td id="S3.T12.40.44.4.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.40.44.4.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib310" title="" class="ltx_ref">310</a><span id="S3.T12.40.44.4.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.40.44.4.4" class="ltx_td ltx_align_left"><span id="S3.T12.40.44.4.4.1" class="ltx_text" style="font-size:70%;">Class.*</span></td>
<td id="S3.T12.40.44.4.5" class="ltx_td ltx_align_center"><span id="S3.T12.40.44.4.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T12.40.44.4.6" class="ltx_td ltx_align_center"><span id="S3.T12.40.44.4.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.40.44.4.7" class="ltx_td"></td>
<td id="S3.T12.40.44.4.8" class="ltx_td"></td>
<td id="S3.T12.40.44.4.9" class="ltx_td"></td>
<td id="S3.T12.40.44.4.10" class="ltx_td"></td>
<td id="S3.T12.40.44.4.11" class="ltx_td"></td>
<td id="S3.T12.40.44.4.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.7.7" class="ltx_tr">
<td id="S3.T12.7.7.4" class="ltx_td ltx_align_left"><span id="S3.T12.7.7.4.1" class="ltx_text" style="font-size:70%;">FH OCT</span></td>
<td id="S3.T12.7.7.5" class="ltx_td ltx_align_left"><span id="S3.T12.7.7.5.1" class="ltx_text" style="font-size:70%;">StyleGAN2-ADA</span></td>
<td id="S3.T12.7.7.6" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.7.7.6.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib311" title="" class="ltx_ref">311</a><span id="S3.T12.7.7.6.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.7.7.7" class="ltx_td ltx_align_left"><span id="S3.T12.7.7.7.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T12.7.7.8" class="ltx_td ltx_align_center"><span id="S3.T12.7.7.8.1" class="ltx_text" style="font-size:70%;">R,A</span></td>
<td id="S3.T12.7.7.9" class="ltx_td ltx_align_center"><span id="S3.T12.7.7.9.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.7.7.10" class="ltx_td"></td>
<td id="S3.T12.7.7.11" class="ltx_td"></td>
<td id="S3.T12.5.5.1" class="ltx_td ltx_align_center"><math id="S3.T12.5.5.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.5.5.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.5.5.1.m1.1.1" xref="S3.T12.5.5.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.5.5.1.m1.1b"><ci id="S3.T12.5.5.1.m1.1.1.cmml" xref="S3.T12.5.5.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.5.5.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.6.6.2" class="ltx_td ltx_align_center"><math id="S3.T12.6.6.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.6.6.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.6.6.2.m1.1.1" xref="S3.T12.6.6.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.6.6.2.m1.1b"><ci id="S3.T12.6.6.2.m1.1.1.cmml" xref="S3.T12.6.6.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.6.6.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.7.7.12" class="ltx_td"></td>
<td id="S3.T12.7.7.3" class="ltx_td ltx_align_center"><math id="S3.T12.7.7.3.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.7.7.3.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.7.7.3.m1.1.1" xref="S3.T12.7.7.3.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.7.7.3.m1.1b"><ci id="S3.T12.7.7.3.m1.1.1.cmml" xref="S3.T12.7.7.3.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.7.7.3.m1.1c">\checkmark</annotation></semantics></math></td>
</tr>
<tr id="S3.T12.9.9" class="ltx_tr">
<td id="S3.T12.9.9.3" class="ltx_td ltx_align_center"><span id="S3.T12.9.9.3.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T12.9.9.4" class="ltx_td ltx_align_left"><span id="S3.T12.9.9.4.1" class="ltx_text" style="font-size:70%;">AS OCT</span></td>
<td id="S3.T12.9.9.5" class="ltx_td ltx_align_left"><span id="S3.T12.9.9.5.1" class="ltx_text" style="font-size:70%;">DDPM</span></td>
<td id="S3.T12.9.9.6" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.9.9.6.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib312" title="" class="ltx_ref">312</a><span id="S3.T12.9.9.6.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.9.9.7" class="ltx_td ltx_align_left"><span id="S3.T12.9.9.7.1" class="ltx_text" style="font-size:70%;">Seg./Det.</span></td>
<td id="S3.T12.9.9.8" class="ltx_td ltx_align_center"><span id="S3.T12.9.9.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.9.9.9" class="ltx_td ltx_align_center"><span id="S3.T12.9.9.9.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T12.8.8.1" class="ltx_td ltx_align_center"><math id="S3.T12.8.8.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.8.8.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.8.8.1.m1.1.1" xref="S3.T12.8.8.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.8.8.1.m1.1b"><ci id="S3.T12.8.8.1.m1.1.1.cmml" xref="S3.T12.8.8.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.8.8.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.9.9.10" class="ltx_td"></td>
<td id="S3.T12.9.9.11" class="ltx_td"></td>
<td id="S3.T12.9.9.2" class="ltx_td ltx_align_center"><math id="S3.T12.9.9.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.9.9.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.9.9.2.m1.1.1" xref="S3.T12.9.9.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.9.9.2.m1.1b"><ci id="S3.T12.9.9.2.m1.1.1.cmml" xref="S3.T12.9.9.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.9.9.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.9.9.12" class="ltx_td"></td>
<td id="S3.T12.9.9.13" class="ltx_td"></td>
</tr>
<tr id="S3.T12.10.10" class="ltx_tr">
<td id="S3.T12.10.10.2" class="ltx_td ltx_align_center"><span id="S3.T12.10.10.2.1" class="ltx_text" style="font-size:70%;">Hybrid</span></td>
<td id="S3.T12.10.10.3" class="ltx_td ltx_align_left"><span id="S3.T12.10.10.3.1" class="ltx_text" style="font-size:70%;">Retinal OCT</span></td>
<td id="S3.T12.10.10.4" class="ltx_td ltx_align_left"><span id="S3.T12.10.10.4.1" class="ltx_text" style="font-size:70%;">DDPM + transf.</span></td>
<td id="S3.T12.10.10.5" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.10.10.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib313" title="" class="ltx_ref">313</a><span id="S3.T12.10.10.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.10.10.6" class="ltx_td ltx_align_left"><span id="S3.T12.10.10.6.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T12.10.10.7" class="ltx_td ltx_align_center"><span id="S3.T12.10.10.7.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T12.10.10.8" class="ltx_td ltx_align_center"><span id="S3.T12.10.10.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.10.10.1" class="ltx_td ltx_align_center"><math id="S3.T12.10.10.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.10.10.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.10.10.1.m1.1.1" xref="S3.T12.10.10.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.10.10.1.m1.1b"><ci id="S3.T12.10.10.1.m1.1.1.cmml" xref="S3.T12.10.10.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.10.10.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.10.10.9" class="ltx_td"></td>
<td id="S3.T12.10.10.10" class="ltx_td"></td>
<td id="S3.T12.10.10.11" class="ltx_td"></td>
<td id="S3.T12.10.10.12" class="ltx_td"></td>
<td id="S3.T12.10.10.13" class="ltx_td"></td>
</tr>
<tr id="S3.T12.15.15" class="ltx_tr">
<td id="S3.T12.15.15.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T12.15.15.6.1" class="ltx_text" style="font-size:70%;">
<span id="S3.T12.15.15.6.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T12.15.15.6.1.1.1" class="ltx_tr">
<span id="S3.T12.15.15.6.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Class or Label</span></span>
<span id="S3.T12.15.15.6.1.1.2" class="ltx_tr">
<span id="S3.T12.15.15.6.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">cond.</span></span>
</span></span></td>
<td id="S3.T12.15.15.7" class="ltx_td ltx_align_center ltx_border_t" rowspan="5"><span id="S3.T12.15.15.7.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T12.15.15.8" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S3.T12.15.15.8.1" class="ltx_text" style="font-size:70%;">Retinal OCT</span></td>
<td id="S3.T12.15.15.9" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.15.15.9.1" class="ltx_text" style="font-size:70%;">U-Net + patch GAN</span></td>
<td id="S3.T12.15.15.10" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.15.15.10.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib314" title="" class="ltx_ref">314</a><span id="S3.T12.15.15.10.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.15.15.11" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.15.15.11.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T12.15.15.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T12.15.15.12.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.15.15.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T12.15.15.13.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T12.11.11.1" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.11.11.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.11.11.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.11.11.1.m1.1.1" xref="S3.T12.11.11.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.11.11.1.m1.1b"><ci id="S3.T12.11.11.1.m1.1.1.cmml" xref="S3.T12.11.11.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.11.11.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.12.12.2" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.12.12.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.12.12.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.12.12.2.m1.1.1" xref="S3.T12.12.12.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.12.12.2.m1.1b"><ci id="S3.T12.12.12.2.m1.1.1.cmml" xref="S3.T12.12.12.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.12.12.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.13.13.3" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.13.13.3.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.13.13.3.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.13.13.3.m1.1.1" xref="S3.T12.13.13.3.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.13.13.3.m1.1b"><ci id="S3.T12.13.13.3.m1.1.1.cmml" xref="S3.T12.13.13.3.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.13.13.3.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.15.15.14" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.14.14.4" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.14.14.4.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.14.14.4.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.14.14.4.m1.1.1" xref="S3.T12.14.14.4.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.14.14.4.m1.1b"><ci id="S3.T12.14.14.4.m1.1.1.cmml" xref="S3.T12.14.14.4.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.14.14.4.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.15.15.5" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.15.15.5.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.15.15.5.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.15.15.5.m1.1.1" xref="S3.T12.15.15.5.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.15.15.5.m1.1b"><ci id="S3.T12.15.15.5.m1.1.1.cmml" xref="S3.T12.15.15.5.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.15.15.5.m1.1c">\checkmark</annotation></semantics></math></td>
</tr>
<tr id="S3.T12.40.45.5" class="ltx_tr">
<td id="S3.T12.40.45.5.1" class="ltx_td ltx_align_left"><span id="S3.T12.40.45.5.1.1" class="ltx_text" style="font-size:70%;">Modified Pix2Pix</span></td>
<td id="S3.T12.40.45.5.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.40.45.5.2.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib315" title="" class="ltx_ref">315</a><span id="S3.T12.40.45.5.2.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.40.45.5.3" class="ltx_td ltx_align_left"><span id="S3.T12.40.45.5.3.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T12.40.45.5.4" class="ltx_td ltx_align_center"><span id="S3.T12.40.45.5.4.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T12.40.45.5.5" class="ltx_td ltx_align_center"><span id="S3.T12.40.45.5.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.40.45.5.6" class="ltx_td"></td>
<td id="S3.T12.40.45.5.7" class="ltx_td"></td>
<td id="S3.T12.40.45.5.8" class="ltx_td"></td>
<td id="S3.T12.40.45.5.9" class="ltx_td"></td>
<td id="S3.T12.40.45.5.10" class="ltx_td"></td>
<td id="S3.T12.40.45.5.11" class="ltx_td"></td>
</tr>
<tr id="S3.T12.16.16" class="ltx_tr">
<td id="S3.T12.16.16.2" class="ltx_td ltx_align_left"><span id="S3.T12.16.16.2.1" class="ltx_text" style="font-size:70%;">Chorio-retinal OCT</span></td>
<td id="S3.T12.16.16.3" class="ltx_td ltx_align_left"><span id="S3.T12.16.16.3.1" class="ltx_text" style="font-size:70%;">Modified DCGAB</span></td>
<td id="S3.T12.16.16.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.16.16.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib316" title="" class="ltx_ref">316</a><span id="S3.T12.16.16.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.16.16.5" class="ltx_td ltx_align_left"><span id="S3.T12.16.16.5.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T12.16.16.6" class="ltx_td ltx_align_center"><span id="S3.T12.16.16.6.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T12.16.16.7" class="ltx_td ltx_align_center"><span id="S3.T12.16.16.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.16.16.1" class="ltx_td ltx_align_center"><math id="S3.T12.16.16.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.16.16.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.16.16.1.m1.1.1" xref="S3.T12.16.16.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.16.16.1.m1.1b"><ci id="S3.T12.16.16.1.m1.1.1.cmml" xref="S3.T12.16.16.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.16.16.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.16.16.8" class="ltx_td"></td>
<td id="S3.T12.16.16.9" class="ltx_td"></td>
<td id="S3.T12.16.16.10" class="ltx_td"></td>
<td id="S3.T12.16.16.11" class="ltx_td"></td>
<td id="S3.T12.16.16.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.17.17" class="ltx_tr">
<td id="S3.T12.17.17.2" class="ltx_td ltx_align_left"><span id="S3.T12.17.17.2.1" class="ltx_text" style="font-size:70%;">SD OCT</span></td>
<td id="S3.T12.17.17.3" class="ltx_td ltx_align_left"><span id="S3.T12.17.17.3.1" class="ltx_text" style="font-size:70%;">StyleGAN2</span></td>
<td id="S3.T12.17.17.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.17.17.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib317" title="" class="ltx_ref">317</a><span id="S3.T12.17.17.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.17.17.5" class="ltx_td ltx_align_left"><span id="S3.T12.17.17.5.1" class="ltx_text" style="font-size:70%;">Seg./Class.</span></td>
<td id="S3.T12.17.17.6" class="ltx_td ltx_align_center"><span id="S3.T12.17.17.6.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T12.17.17.7" class="ltx_td ltx_align_center"><span id="S3.T12.17.17.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.17.17.1" class="ltx_td ltx_align_center"><math id="S3.T12.17.17.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.17.17.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.17.17.1.m1.1.1" xref="S3.T12.17.17.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.17.17.1.m1.1b"><ci id="S3.T12.17.17.1.m1.1.1.cmml" xref="S3.T12.17.17.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.17.17.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.17.17.8" class="ltx_td"></td>
<td id="S3.T12.17.17.9" class="ltx_td"></td>
<td id="S3.T12.17.17.10" class="ltx_td"></td>
<td id="S3.T12.17.17.11" class="ltx_td"></td>
<td id="S3.T12.17.17.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.40.46.6" class="ltx_tr">
<td id="S3.T12.40.46.6.1" class="ltx_td ltx_align_left"><span id="S3.T12.40.46.6.1.1" class="ltx_text" style="font-size:70%;">Coronary atrial plaques</span></td>
<td id="S3.T12.40.46.6.2" class="ltx_td ltx_align_left"><span id="S3.T12.40.46.6.2.1" class="ltx_text" style="font-size:70%;">cond. GAN</span></td>
<td id="S3.T12.40.46.6.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.40.46.6.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib318" title="" class="ltx_ref">318</a><span id="S3.T12.40.46.6.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.40.46.6.4" class="ltx_td ltx_align_left"><span id="S3.T12.40.46.6.4.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T12.40.46.6.5" class="ltx_td ltx_align_center"><span id="S3.T12.40.46.6.5.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T12.40.46.6.6" class="ltx_td ltx_align_center"><span id="S3.T12.40.46.6.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.40.46.6.7" class="ltx_td"></td>
<td id="S3.T12.40.46.6.8" class="ltx_td"></td>
<td id="S3.T12.40.46.6.9" class="ltx_td"></td>
<td id="S3.T12.40.46.6.10" class="ltx_td"></td>
<td id="S3.T12.40.46.6.11" class="ltx_td"></td>
<td id="S3.T12.40.46.6.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.18.18" class="ltx_tr">
<td id="S3.T12.18.18.2" class="ltx_td ltx_align_center"><span id="S3.T12.18.18.2.1" class="ltx_text" style="font-size:70%;">VAE</span></td>
<td id="S3.T12.18.18.3" class="ltx_td ltx_align_left"><span id="S3.T12.18.18.3.1" class="ltx_text" style="font-size:70%;">Retinal (diseased) OCT</span></td>
<td id="S3.T12.18.18.4" class="ltx_td ltx_align_left"><span id="S3.T12.18.18.4.1" class="ltx_text" style="font-size:70%;">Modified cond. VAE</span></td>
<td id="S3.T12.18.18.5" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.18.18.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib319" title="" class="ltx_ref">319</a><span id="S3.T12.18.18.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.18.18.6" class="ltx_td ltx_align_left"><span id="S3.T12.18.18.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T12.18.18.7" class="ltx_td"></td>
<td id="S3.T12.18.18.8" class="ltx_td"></td>
<td id="S3.T12.18.18.9" class="ltx_td"></td>
<td id="S3.T12.18.18.10" class="ltx_td"></td>
<td id="S3.T12.18.18.11" class="ltx_td"></td>
<td id="S3.T12.18.18.12" class="ltx_td"></td>
<td id="S3.T12.18.18.13" class="ltx_td"></td>
<td id="S3.T12.18.18.1" class="ltx_td ltx_align_center"><math id="S3.T12.18.18.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.18.18.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.18.18.1.m1.1.1" xref="S3.T12.18.18.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.18.18.1.m1.1b"><ci id="S3.T12.18.18.1.m1.1.1.cmml" xref="S3.T12.18.18.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.18.18.1.m1.1c">\checkmark</annotation></semantics></math></td>
</tr>
<tr id="S3.T12.21.21" class="ltx_tr">
<td id="S3.T12.21.21.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="6"><span id="S3.T12.21.21.4.1" class="ltx_text" style="font-size:70%;">Attribute cond.</span></td>
<td id="S3.T12.21.21.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="6"><span id="S3.T12.21.21.5.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T12.21.21.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.21.21.6.1" class="ltx_text" style="font-size:70%;">SD OCT</span></td>
<td id="S3.T12.21.21.7" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.21.21.7.1" class="ltx_text" style="font-size:70%;">BAABGAN</span></td>
<td id="S3.T12.21.21.8" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.21.21.8.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib320" title="" class="ltx_ref">320</a><span id="S3.T12.21.21.8.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.21.21.9" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.21.21.9.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T12.21.21.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.21.21.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.19.19.1" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.19.19.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.19.19.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.19.19.1.m1.1.1" xref="S3.T12.19.19.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.19.19.1.m1.1b"><ci id="S3.T12.19.19.1.m1.1.1.cmml" xref="S3.T12.19.19.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.19.19.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.21.21.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.20.20.2" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.20.20.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.20.20.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.20.20.2.m1.1.1" xref="S3.T12.20.20.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.20.20.2.m1.1b"><ci id="S3.T12.20.20.2.m1.1.1.cmml" xref="S3.T12.20.20.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.20.20.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.21.21.3" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.21.21.3.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.21.21.3.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.21.21.3.m1.1.1" xref="S3.T12.21.21.3.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.21.21.3.m1.1b"><ci id="S3.T12.21.21.3.m1.1.1.cmml" xref="S3.T12.21.21.3.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.21.21.3.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.21.21.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.21.21.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T12.24.24" class="ltx_tr">
<td id="S3.T12.24.24.4" class="ltx_td ltx_align_left"><span id="S3.T12.24.24.4.1" class="ltx_text" style="font-size:70%;">DME OCT</span></td>
<td id="S3.T12.24.24.5" class="ltx_td ltx_align_left"><span id="S3.T12.24.24.5.1" class="ltx_text" style="font-size:70%;">DDFA-GAN</span></td>
<td id="S3.T12.24.24.6" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.24.24.6.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib69" title="" class="ltx_ref">69</a><span id="S3.T12.24.24.6.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.24.24.7" class="ltx_td ltx_align_left"><span id="S3.T12.24.24.7.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T12.24.24.8" class="ltx_td ltx_align_center"><span id="S3.T12.24.24.8.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T12.24.24.9" class="ltx_td ltx_align_center"><span id="S3.T12.24.24.9.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.22.22.1" class="ltx_td ltx_align_center"><math id="S3.T12.22.22.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.22.22.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.22.22.1.m1.1.1" xref="S3.T12.22.22.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.22.22.1.m1.1b"><ci id="S3.T12.22.22.1.m1.1.1.cmml" xref="S3.T12.22.22.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.22.22.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.23.23.2" class="ltx_td ltx_align_center"><math id="S3.T12.23.23.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.23.23.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.23.23.2.m1.1.1" xref="S3.T12.23.23.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.23.23.2.m1.1b"><ci id="S3.T12.23.23.2.m1.1.1.cmml" xref="S3.T12.23.23.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.23.23.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.24.24.10" class="ltx_td"></td>
<td id="S3.T12.24.24.11" class="ltx_td"></td>
<td id="S3.T12.24.24.12" class="ltx_td"></td>
<td id="S3.T12.24.24.3" class="ltx_td ltx_align_center"><math id="S3.T12.24.24.3.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.24.24.3.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.24.24.3.m1.1.1" xref="S3.T12.24.24.3.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.24.24.3.m1.1b"><ci id="S3.T12.24.24.3.m1.1.1.cmml" xref="S3.T12.24.24.3.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.24.24.3.m1.1c">\checkmark</annotation></semantics></math></td>
</tr>
<tr id="S3.T12.25.25" class="ltx_tr">
<td id="S3.T12.25.25.2" class="ltx_td ltx_align_left"><span id="S3.T12.25.25.2.1" class="ltx_text" style="font-size:70%;">AMD OCT and CFP</span></td>
<td id="S3.T12.25.25.3" class="ltx_td ltx_align_left"><span id="S3.T12.25.25.3.1" class="ltx_text" style="font-size:70%;">Pix2PixHD</span></td>
<td id="S3.T12.25.25.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.25.25.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib321" title="" class="ltx_ref">321</a><span id="S3.T12.25.25.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.25.25.5" class="ltx_td ltx_align_left"><span id="S3.T12.25.25.5.1" class="ltx_text" style="font-size:70%;">Class.*</span></td>
<td id="S3.T12.25.25.6" class="ltx_td ltx_align_center"><span id="S3.T12.25.25.6.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T12.25.25.7" class="ltx_td ltx_align_center"><span id="S3.T12.25.25.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.25.25.8" class="ltx_td"></td>
<td id="S3.T12.25.25.9" class="ltx_td"></td>
<td id="S3.T12.25.25.10" class="ltx_td"></td>
<td id="S3.T12.25.25.11" class="ltx_td"></td>
<td id="S3.T12.25.25.12" class="ltx_td"></td>
<td id="S3.T12.25.25.1" class="ltx_td ltx_align_center"><math id="S3.T12.25.25.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.25.25.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.25.25.1.m1.1.1" xref="S3.T12.25.25.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.25.25.1.m1.1b"><ci id="S3.T12.25.25.1.m1.1.1.cmml" xref="S3.T12.25.25.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.25.25.1.m1.1c">\checkmark</annotation></semantics></math></td>
</tr>
<tr id="S3.T12.27.27" class="ltx_tr">
<td id="S3.T12.27.27.3" class="ltx_td ltx_align_left" rowspan="3"><span id="S3.T12.27.27.3.1" class="ltx_text" style="font-size:70%;">Retinal OCT</span></td>
<td id="S3.T12.27.27.4" class="ltx_td ltx_align_left"><span id="S3.T12.27.27.4.1" class="ltx_text" style="font-size:70%;">Contrastive CNN-based</span></td>
<td id="S3.T12.27.27.5" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.27.27.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib322" title="" class="ltx_ref">322</a><span id="S3.T12.27.27.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.27.27.6" class="ltx_td ltx_align_left"><span id="S3.T12.27.27.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T12.27.27.7" class="ltx_td"></td>
<td id="S3.T12.27.27.8" class="ltx_td"></td>
<td id="S3.T12.27.27.9" class="ltx_td ltx_align_left"><span id="S3.T12.27.27.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T12.27.27.10" class="ltx_td"></td>
<td id="S3.T12.26.26.1" class="ltx_td ltx_align_center"><math id="S3.T12.26.26.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.26.26.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.26.26.1.m1.1.1" xref="S3.T12.26.26.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.26.26.1.m1.1b"><ci id="S3.T12.26.26.1.m1.1.1.cmml" xref="S3.T12.26.26.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.26.26.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.27.27.2" class="ltx_td ltx_align_center"><math id="S3.T12.27.27.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.27.27.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.27.27.2.m1.1.1" xref="S3.T12.27.27.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.27.27.2.m1.1b"><ci id="S3.T12.27.27.2.m1.1.1.cmml" xref="S3.T12.27.27.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.27.27.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.27.27.11" class="ltx_td"></td>
<td id="S3.T12.27.27.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.28.28" class="ltx_tr">
<td id="S3.T12.28.28.2" class="ltx_td ltx_align_left"><span id="S3.T12.28.28.2.1" class="ltx_text" style="font-size:70%;">FOF-GAN</span></td>
<td id="S3.T12.28.28.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.28.28.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib323" title="" class="ltx_ref">323</a><span id="S3.T12.28.28.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.28.28.4" class="ltx_td ltx_align_left"><span id="S3.T12.28.28.4.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T12.28.28.5" class="ltx_td ltx_align_center"><span id="S3.T12.28.28.5.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.28.28.6" class="ltx_td ltx_align_center"><span id="S3.T12.28.28.6.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T12.28.28.1" class="ltx_td ltx_align_left"><math id="S3.T12.28.28.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.28.28.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.28.28.1.m1.1.1" xref="S3.T12.28.28.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.28.28.1.m1.1b"><ci id="S3.T12.28.28.1.m1.1.1.cmml" xref="S3.T12.28.28.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.28.28.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.28.28.7" class="ltx_td"></td>
<td id="S3.T12.28.28.8" class="ltx_td"></td>
<td id="S3.T12.28.28.9" class="ltx_td"></td>
<td id="S3.T12.28.28.10" class="ltx_td"></td>
<td id="S3.T12.28.28.11" class="ltx_td"></td>
</tr>
<tr id="S3.T12.30.30" class="ltx_tr">
<td id="S3.T12.30.30.3" class="ltx_td ltx_align_left"><span id="S3.T12.30.30.3.1" class="ltx_text" style="font-size:70%;">StarGAN</span></td>
<td id="S3.T12.30.30.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.30.30.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib324" title="" class="ltx_ref">324</a><span id="S3.T12.30.30.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.30.30.5" class="ltx_td ltx_align_left"><span id="S3.T12.30.30.5.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T12.30.30.6" class="ltx_td ltx_align_center"><span id="S3.T12.30.30.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.30.30.7" class="ltx_td ltx_align_center"><span id="S3.T12.30.30.7.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T12.30.30.8" class="ltx_td"></td>
<td id="S3.T12.30.30.9" class="ltx_td"></td>
<td id="S3.T12.29.29.1" class="ltx_td ltx_align_center"><math id="S3.T12.29.29.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.29.29.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.29.29.1.m1.1.1" xref="S3.T12.29.29.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.29.29.1.m1.1b"><ci id="S3.T12.29.29.1.m1.1.1.cmml" xref="S3.T12.29.29.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.29.29.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.30.30.2" class="ltx_td ltx_align_center"><math id="S3.T12.30.30.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.30.30.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.30.30.2.m1.1.1" xref="S3.T12.30.30.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.30.30.2.m1.1b"><ci id="S3.T12.30.30.2.m1.1.1.cmml" xref="S3.T12.30.30.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.30.30.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.30.30.10" class="ltx_td"></td>
<td id="S3.T12.30.30.11" class="ltx_td"></td>
</tr>
<tr id="S3.T12.32.32" class="ltx_tr">
<td id="S3.T12.32.32.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T12.32.32.3.1" class="ltx_text" style="font-size:70%;">Intra-modal trans.</span></td>
<td id="S3.T12.32.32.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T12.32.32.4.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T12.32.32.5" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.32.32.5.1" class="ltx_text" style="font-size:70%;">Pre-T - Post-T OCT</span></td>
<td id="S3.T12.32.32.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.32.32.6.1" class="ltx_text" style="font-size:70%;">Pix2PixHD</span></td>
<td id="S3.T12.32.32.7" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.32.32.7.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib325" title="" class="ltx_ref">325</a><span id="S3.T12.32.32.7.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.32.32.8" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.32.32.8.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T12.32.32.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.32.32.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.32.32.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.32.32.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.31.31.1" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.31.31.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.31.31.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.31.31.1.m1.1.1" xref="S3.T12.31.31.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.31.31.1.m1.1b"><ci id="S3.T12.31.31.1.m1.1.1.cmml" xref="S3.T12.31.31.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.31.31.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.32.32.2" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.32.32.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.32.32.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.32.32.2.m1.1.1" xref="S3.T12.32.32.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.32.32.2.m1.1b"><ci id="S3.T12.32.32.2.m1.1.1.cmml" xref="S3.T12.32.32.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.32.32.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.32.32.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.32.32.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T12.34.34" class="ltx_tr">
<td id="S3.T12.34.34.3" class="ltx_td ltx_align_left"><span id="S3.T12.34.34.3.1" class="ltx_text" style="font-size:70%;">OCT - PS-OCT</span></td>
<td id="S3.T12.34.34.4" class="ltx_td ltx_align_left"><span id="S3.T12.34.34.4.1" class="ltx_text" style="font-size:70%;">Pix2PixHD</span></td>
<td id="S3.T12.34.34.5" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.34.34.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib326" title="" class="ltx_ref">326</a><span id="S3.T12.34.34.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.34.34.6" class="ltx_td ltx_align_left"><span id="S3.T12.34.34.6.1" class="ltx_text" style="font-size:70%;">Class.</span></td>
<td id="S3.T12.34.34.7" class="ltx_td ltx_align_center"><span id="S3.T12.34.34.7.1" class="ltx_text" style="font-size:70%;">R,S</span></td>
<td id="S3.T12.34.34.8" class="ltx_td ltx_align_center"><span id="S3.T12.34.34.8.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.33.33.1" class="ltx_td ltx_align_center"><math id="S3.T12.33.33.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.33.33.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.33.33.1.m1.1.1" xref="S3.T12.33.33.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.33.33.1.m1.1b"><ci id="S3.T12.33.33.1.m1.1.1.cmml" xref="S3.T12.33.33.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.33.33.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.34.34.9" class="ltx_td"></td>
<td id="S3.T12.34.34.2" class="ltx_td ltx_align_center"><math id="S3.T12.34.34.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.34.34.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.34.34.2.m1.1.1" xref="S3.T12.34.34.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.34.34.2.m1.1b"><ci id="S3.T12.34.34.2.m1.1.1.cmml" xref="S3.T12.34.34.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.34.34.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.34.34.10" class="ltx_td"></td>
<td id="S3.T12.34.34.11" class="ltx_td"></td>
<td id="S3.T12.34.34.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.35.35" class="ltx_tr">
<td id="S3.T12.35.35.2" class="ltx_td ltx_align_left"><span id="S3.T12.35.35.2.1" class="ltx_text" style="font-size:70%;">Inter-device SD-OCT</span></td>
<td id="S3.T12.35.35.3" class="ltx_td ltx_align_left"><span id="S3.T12.35.35.3.1" class="ltx_text" style="font-size:70%;">CycleGAN</span></td>
<td id="S3.T12.35.35.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.35.35.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib327" title="" class="ltx_ref">327</a><span id="S3.T12.35.35.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.35.35.5" class="ltx_td ltx_align_left"><span id="S3.T12.35.35.5.1" class="ltx_text" style="font-size:70%;">Seg.</span></td>
<td id="S3.T12.35.35.6" class="ltx_td ltx_align_center"><span id="S3.T12.35.35.6.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.35.35.7" class="ltx_td ltx_align_center"><span id="S3.T12.35.35.7.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T12.35.35.8" class="ltx_td"></td>
<td id="S3.T12.35.35.9" class="ltx_td"></td>
<td id="S3.T12.35.35.10" class="ltx_td"></td>
<td id="S3.T12.35.35.1" class="ltx_td ltx_align_center"><math id="S3.T12.35.35.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.35.35.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.35.35.1.m1.1.1" xref="S3.T12.35.35.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.35.35.1.m1.1b"><ci id="S3.T12.35.35.1.m1.1.1.cmml" xref="S3.T12.35.35.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.35.35.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.35.35.11" class="ltx_td"></td>
<td id="S3.T12.35.35.12" class="ltx_td"></td>
</tr>
<tr id="S3.T12.39.39" class="ltx_tr">
<td id="S3.T12.39.39.5" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="2"><span id="S3.T12.39.39.5.1" class="ltx_text" style="font-size:70%;">Inter-modal trans.</span></td>
<td id="S3.T12.39.39.6" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="2"><span id="S3.T12.39.39.6.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T12.39.39.7" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.39.39.7.1" class="ltx_text" style="font-size:70%;">FA - OCT</span></td>
<td id="S3.T12.39.39.8" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.39.39.8.1" class="ltx_text" style="font-size:70%;">Pix2Pix</span></td>
<td id="S3.T12.39.39.9" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.39.39.9.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib68" title="" class="ltx_ref">68</a><span id="S3.T12.39.39.9.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.39.39.10" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T12.39.39.10.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T12.39.39.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.39.39.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.36.36.1" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.36.36.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.36.36.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.36.36.1.m1.1.1" xref="S3.T12.36.36.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.36.36.1.m1.1b"><ci id="S3.T12.36.36.1.m1.1.1.cmml" xref="S3.T12.36.36.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.36.36.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.37.37.2" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.37.37.2.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.37.37.2.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.37.37.2.m1.1.1" xref="S3.T12.37.37.2.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.37.37.2.m1.1b"><ci id="S3.T12.37.37.2.m1.1.1.cmml" xref="S3.T12.37.37.2.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.37.37.2.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.38.38.3" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.38.38.3.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.38.38.3.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.38.38.3.m1.1.1" xref="S3.T12.38.38.3.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.38.38.3.m1.1b"><ci id="S3.T12.38.38.3.m1.1.1.cmml" xref="S3.T12.38.38.3.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.38.38.3.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.39.39.4" class="ltx_td ltx_align_center ltx_border_t"><math id="S3.T12.39.39.4.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.39.39.4.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.39.39.4.m1.1.1" xref="S3.T12.39.39.4.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.39.39.4.m1.1b"><ci id="S3.T12.39.39.4.m1.1.1.cmml" xref="S3.T12.39.39.4.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.39.39.4.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.39.39.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T12.39.39.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T12.40.40" class="ltx_tr">
<td id="S3.T12.40.40.2" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T12.40.40.2.1" class="ltx_text" style="font-size:70%;">Fundus - OCT</span></td>
<td id="S3.T12.40.40.3" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T12.40.40.3.1" class="ltx_text" style="font-size:70%;">U-net-based</span></td>
<td id="S3.T12.40.40.4" class="ltx_td ltx_align_left ltx_border_b"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T12.40.40.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib328" title="" class="ltx_ref">328</a><span id="S3.T12.40.40.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T12.40.40.5" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T12.40.40.5.1" class="ltx_text" style="font-size:70%;">Class.*</span></td>
<td id="S3.T12.40.40.6" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T12.40.40.6.1" class="ltx_text" style="font-size:70%;">R,S,A</span></td>
<td id="S3.T12.40.40.7" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T12.40.40.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T12.40.40.1" class="ltx_td ltx_align_center ltx_border_b"><math id="S3.T12.40.40.1.m1.1" class="ltx_Math" alttext="\checkmark" display="inline"><semantics id="S3.T12.40.40.1.m1.1a"><mi mathsize="70%" mathvariant="normal" id="S3.T12.40.40.1.m1.1.1" xref="S3.T12.40.40.1.m1.1.1.cmml">✓</mi><annotation-xml encoding="MathML-Content" id="S3.T12.40.40.1.m1.1b"><ci id="S3.T12.40.40.1.m1.1.1.cmml" xref="S3.T12.40.40.1.m1.1.1">✓</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T12.40.40.1.m1.1c">\checkmark</annotation></semantics></math></td>
<td id="S3.T12.40.40.8" class="ltx_td ltx_border_b"></td>
<td id="S3.T12.40.40.9" class="ltx_td ltx_border_b"></td>
<td id="S3.T12.40.40.10" class="ltx_td ltx_border_b"></td>
<td id="S3.T12.40.40.11" class="ltx_td ltx_border_b"></td>
<td id="S3.T12.40.40.12" class="ltx_td ltx_border_b"></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 12: </span>Overview of all summarized and discussed studies on <span id="S3.T12.45.1" class="ltx_text ltx_font_bold">OCT</span> synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), whereby automated machine learning models are trained with real (R), synthetic (S) or augmented with synthetic data (A), and tested on real or synthetic data, fidelity (F), diversity (D), visual quality (Q), clinical utility (C) and privacy (P) evaluation. AS, SD, DM, NOR, AMD, FH, DME, CFP, Pre-T, Post-T, PS, FA stand for anterior segment, spectral domain, diabetic maculopathy, normal, age-related macular degeneration, foveal hypoplasia, diabetic macular edema, color fundus photography, pre-therapy, post-therapy, polarization sensitive OCT and fluorescein angiography, respectively. * stands for studies that have utilized synthesized images for pre-training. </figcaption>
</figure>
<div id="S3.SS3.SSS7.p1" class="ltx_para">
<p id="S3.SS3.SSS7.p1.1" class="ltx_p">Non-invasive retinal OCT is crucial for diagnosing eye diseases, as features like retinal layer shapes and thicknesses strongly correlate with conditions such as glaucoma, age-related macular degeneration, and diabetic macular edema. Thus, in recent years, there has been an increase in the development of automated methods for identifying macular pathologies <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib329" title="" class="ltx_ref">329</a>]</cite>. However, these approaches face challenges due to limited data availability and imbalanced datasets, given the rarity of pathological findings. Therefore, the primary goal of OCT image synthesis (Table <a href="#S3.T12" title="Table 12 ‣ 3.3.7 OCT ‣ 3.3 Medical imaging ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">12</span></a>) is to augment data for algorithms, enabling not just disease detection but also enhancing the comprehension of retinal diseases.</p>
</div>
<div id="S3.SS3.SSS7.p2" class="ltx_para">
<p id="S3.SS3.SSS7.p2.1" class="ltx_p">The array of unconditional generative methods, as exemplified by <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib307" title="" class="ltx_ref">307</a>, <a href="#bib.bib308" title="" class="ltx_ref">308</a>, <a href="#bib.bib309" title="" class="ltx_ref">309</a>, <a href="#bib.bib310" title="" class="ltx_ref">310</a>, <a href="#bib.bib311" title="" class="ltx_ref">311</a>, <a href="#bib.bib312" title="" class="ltx_ref">312</a>, <a href="#bib.bib313" title="" class="ltx_ref">313</a>]</cite>, underscores a notable improvement in downstream tasks when the generation process is meticulously tailored to the specific task and the modality of interest, such as various types of OCT images. These studies reveal that the careful consideration of generative architectures significantly enhances the performance of subsequent analyses. Addressing challenges like limited dataset availability, artifact removal, and domain shift issues through the adoption of techniques like PGGANs, structure-consistency GANs, and StyleGAN2-ADA contributes significantly to the generation of diverse and realistic synthetic images. The effectiveness of these unconditional methods is further emphasized in tasks like classification, despeckling, and segmentation, showcasing their versatility in diverse medical image analyses. The integration of diffusion-based methods, exemplified by <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib313" title="" class="ltx_ref">313</a>]</cite>, further accentuates this point, showcasing the adaptability of generative architectures on different tasks. These findings collectively emphasize the importance of thoughtfully tailoring generative approaches to the intricacies of medical image analysis, promoting their adoption in the domain of OCT image generation and analysis.</p>
</div>
<div id="S3.SS3.SSS7.p3" class="ltx_para">
<p id="S3.SS3.SSS7.p3.1" class="ltx_p">Highlighting the importance of conditioning for accurate representation of retinal pathologies in OCT images, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib319" title="" class="ltx_ref">319</a>]</cite> employs a conditional VAE with contrastive learning to discriminate between diseases in the embedding space. Similarly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib323" title="" class="ltx_ref">323</a>]</cite> introduces a FastGAN with a Fourier Attention Block (FAB), improving feature map rescaling based on frequency contributions. Inspired by Spectral Domain OCT (SD-OCT) that utilizes Fourier/spectral detection for improving OCT imaging resolution, the generator uses the Fast Fourier Transform (FFT) and inverse-FFT algorithms to eliminate noise and enhance details. Results demonstrate FAB’s effectiveness in noise reduction, but lower accuracies in classifying certain pathologies indicate the need for incorporating more distinctive features. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib324" title="" class="ltx_ref">324</a>]</cite> underscores GANs’ importance in evaluating subject-specific eye development and tracking subtle retinal changes over time. This study introduces a counterfactual GAN based on starGAN, synthesizing high-resolution counterfactual OCT images and longitudinal time series from retrospective data. These images allow exploration of hypothetical scenarios by altering certain subject characteristics while preserving identity and image acquisition settings. Despite clinical evaluations aligning with previous studies, limitations include potential correlation of disease features with age or sex, which may result in the GAN altering them when generating counterfactual images, as well as the inability to learn relationships for unseen subject groups. Expert evaluations reveal challenges in distinguishing real from synthetic images, highlighting the algorithm’s realism in mimicking retinal changes and emphasizing the need for further research into counterfactual approaches in medical imaging for a deeper understanding of subject-specific changes over time.</p>
</div>
<div id="S3.SS3.SSS7.p4" class="ltx_para">
<p id="S3.SS3.SSS7.p4.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib320" title="" class="ltx_ref">320</a>]</cite> presents a biomarkers-aware asymmetric bibranch GAN for post-therapeutic (PT) OCT image generation. Using Adaptive Memory Batch Normalization (AMBN), the method transfers knowledge from large-scale data to the target branch, trained on small-scale paired data. The generated images closely align with clinical requirements, aiding in treatment adjustment and monitoring of neovascular age-related macular degeneration (nAMD) progression, whereby over 95% of synthetic SD-OCT images effectively convey clinically relevant information and predict biomarker presence accurately. Additionally, the study shows significant advancements in predicting therapeutic effects over longer periods, up to 12 months after initial treatment, compared to existing methods. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib69" title="" class="ltx_ref">69</a>]</cite> employs a dual-discriminator Fourier acquisitive GAN to generate realistic OCT images, leveraging Fourier domain similarity. Meanwhile, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib321" title="" class="ltx_ref">321</a>]</cite> uses a Pix2PixHD GAN for multi-modal age-related macular degeneration categorization with CAM manipulation to enhance image diversity. Recognizing the limitations of 2D OCT methods, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib321" title="" class="ltx_ref">321</a>]</cite> underscores the importance of 3D volumes aligned with routine clinical examinations. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib322" title="" class="ltx_ref">322</a>]</cite> enhances cross-subsectional resolution using a fully convolutional approach, validated for realism by medical experts. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib322" title="" class="ltx_ref">322</a>]</cite> also introduces BRISQUE, a perceptual image quality indicator, effectively evaluating image quality without a reference. These studies contribute nuanced perspectives to OCT image synthesis, emphasizing practical applications and diagnostic advancements.</p>
</div>
<div id="S3.SS3.SSS7.p5" class="ltx_para">
<p id="S3.SS3.SSS7.p5.1" class="ltx_p">In the absence of paired data for training, unpaired architectures, like cycle-consistent GAN variations, are employed for image harmonization, ensuring consistency of features across different acquisition sites. However, challenges like instability and contrast inversion limit their reliability in real-world medical imaging. Methods in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib314" title="" class="ltx_ref">314</a>, <a href="#bib.bib315" title="" class="ltx_ref">315</a>, <a href="#bib.bib316" title="" class="ltx_ref">316</a>, <a href="#bib.bib317" title="" class="ltx_ref">317</a>, <a href="#bib.bib318" title="" class="ltx_ref">318</a>]</cite> preserve anatomical layout during generation, assuming consistent morphological shape across imaging sites. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib314" title="" class="ltx_ref">314</a>]</cite> introduces a segmentation-renormalized image translation approach using a residual U-Net, enhancing image harmonization and robustness against perturbations. It explores Kernel Inception Distance (KID) alongside FID for evaluating visual fidelity, suggesting a multitask autoencoder trained on medical images for feature extraction and FID/KID calculation. Several studies have made significant advancements in medical imaging using GANs. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib315" title="" class="ltx_ref">315</a>]</cite> achieved a 94.83% accuracy in detecting DME fluid with synthetic data using a Pix2Pix U-net-based generator for OCT image synthesis. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib316" title="" class="ltx_ref">316</a>]</cite> demonstrated comparable performance between synthetic and real data for OCT patch generation using a DCGAN. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib317" title="" class="ltx_ref">317</a>]</cite> enhanced chorio-retinal segmentation by optimizing generator choice with an improved StyleGAN2, leading to substantial improvements in patch classification. Lastly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib318" title="" class="ltx_ref">318</a>]</cite> improved plaque detection accuracy by 15.8% using conditional GANs with class encoding, aligning well with clinical findings.</p>
</div>
<div id="S3.SS3.SSS7.p6" class="ltx_para">
<p id="S3.SS3.SSS7.p6.1" class="ltx_p">Translation-based approaches in OCT imaging address critical clinical needs, like predicting treatment responses through generating post-therapeutic OCT images from pre-therapeutic ones. For example, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib325" title="" class="ltx_ref">325</a>]</cite> adapts Pix2PixHD to forecast responses to anti-vascular endothelial growth factors (anti-VEGF) therapy in retinal vein occlusion patients. The generated synthetic OCT images closely match real ones, aiding in treatment response prediction and patient follow-up. Moreover, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib326" title="" class="ltx_ref">326</a>]</cite> introduces a method for synthesizing polarization-sensitive (PS) OCT images using a modified Pix2Pix GAN, offering an alternative to complex hardware. Validation demonstrates interchangeability between synthetic and real PS-OCT images, promising applications in cancer diagnosis. Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib327" title="" class="ltx_ref">327</a>]</cite> tackles differences in image intensities and signal-to-noise ratios between instruments using an unpaired cycleGAN-based domain adaptation network. This enables cross-instrumental image analysis, crucial for deep learning model training and clinical segmentation tasks.</p>
</div>
<div id="S3.SS3.SSS7.p7" class="ltx_para">
<p id="S3.SS3.SSS7.p7.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib68" title="" class="ltx_ref">68</a>]</cite> presents a method using inter-modality translation to synthesize OCT color-coded macular thickness maps from fluorescein angiography (FA) data and vice versa, aiding in diagnosing diabetic macular edema. By integrating FA’s physiological data with OCT’s structural data, a more precise assessment of treatment response becomes possible, despite limitations in training data and generator architecture. Similarly, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib328" title="" class="ltx_ref">328</a>]</cite> employs GANs, specifically the U-Net generator, for generating OCT images from fundus images to diagnose glaucoma. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib328" title="" class="ltx_ref">328</a>]</cite> demonstrates that incremental training improves model accuracy and adaptability across different hospital settings, showcasing the potential of generated images in glaucoma identification.</p>
</div>
<div id="S3.SS3.SSS7.p8" class="ltx_para">
<p id="S3.SS3.SSS7.p8.1" class="ltx_p">Synthesized OCT image quality evaluation typically mirrors assessments in other modalities, relying on objective fidelity and diversity metrics. Visual Turing tests, conducted in a limited number of studies <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib307" title="" class="ltx_ref">307</a>, <a href="#bib.bib322" title="" class="ltx_ref">322</a>, <a href="#bib.bib325" title="" class="ltx_ref">325</a>, <a href="#bib.bib68" title="" class="ltx_ref">68</a>]</cite> with 2-3 experts, indicate that artifacts around crucial regions and unusual noise patterns often hint at synthetic samples. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib307" title="" class="ltx_ref">307</a>]</cite>, specialists noted lower visibility of scleral spurs and anterior segment structures in synthetic AS-OCT ACA images, critical for imaging the anterior chamber angle, aiding identification. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib68" title="" class="ltx_ref">68</a>]</cite> revealed the model’s ability to represent leakage areas in FA frames with color codes but also generated inconsistent background vessels, signaling synthetic images. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib324" title="" class="ltx_ref">324</a>]</cite>, despite difficulty distinguishing real from fake images in many cases, features like choroid and vitreous areas, pathologic features, and shadowing artifacts hinted at synthetic images. Several studies have assessed the clinical utility of synthetic OCT images, providing insights into image quality and applicability. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib320" title="" class="ltx_ref">320</a>]</cite>, analysis by retinal clinicians of generated post-therapeutic images unveiled predictive accuracy for short and long-term treatment effects, facilitating effective doctor-patient communication and adjustments in treatment strategy. The proposed BAABGAN also demonstrated effective prediction of biomarkers. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib324" title="" class="ltx_ref">324</a>]</cite> introduced a neural network generating high-resolution counterfactual OCT images, enabling exploration of hypothetical scenarios for retinal aging research. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib311" title="" class="ltx_ref">311</a>]</cite> estimated foveal hypoplasia grade and visual acuity using deep learning, incorporating real and GAN-generated images. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib325" title="" class="ltx_ref">325</a>]</cite> focused on calculating central macular thickness, while<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib327" title="" class="ltx_ref">327</a>]</cite> compared retinal volumes.Additionally, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib312" title="" class="ltx_ref">312</a>]</cite> addressed despeckling in AS-OCT images. Despite these advances, the evaluation of synthetic images is lacking, especially in terms of diversity, quality and clinical utility, with privacy preservation mostly unaddressed in current methods.</p>
</div>
</section>
<section id="S3.SS3.SSS8" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.3.8 </span>Multiple Modalities</h4>

<figure id="S3.T13" class="ltx_table">
<table id="S3.T13.1" class="ltx_tabular ltx_centering ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S3.T13.1.2.1" class="ltx_tr">
<td id="S3.T13.1.2.1.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T13.1.2.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></td>
<td id="S3.T13.1.2.1.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T13.1.2.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></td>
<td id="S3.T13.1.2.1.3" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T13.1.2.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modalities</span></td>
<td id="S3.T13.1.2.1.4" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T13.1.2.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></td>
<td id="S3.T13.1.2.1.5" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T13.1.2.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></td>
<td id="S3.T13.1.2.1.6" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S3.T13.1.2.1.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">DT</span></td>
<td id="S3.T13.1.2.1.7" class="ltx_td ltx_align_center ltx_border_t" colspan="7"><span id="S3.T13.1.2.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></td>
<td id="S3.T13.1.2.1.8" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T13.1.2.1.8.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></td>
</tr>
<tr id="S3.T13.1.3.2" class="ltx_tr">
<td id="S3.T13.1.3.2.1" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T13.1.3.2.1.1" class="ltx_text" style="font-size:70%;">U</span></td>
<td id="S3.T13.1.3.2.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T13.1.3.2.2.1" class="ltx_text" style="font-size:70%;">F</span></td>
<td id="S3.T13.1.3.2.3" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T13.1.3.2.3.1" class="ltx_text" style="font-size:70%;">D</span></td>
<td id="S3.T13.1.3.2.4" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T13.1.3.2.4.1" class="ltx_text" style="font-size:70%;">Q</span></td>
<td id="S3.T13.1.3.2.5" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T13.1.3.2.5.1" class="ltx_text" style="font-size:70%;">C</span></td>
<td id="S3.T13.1.3.2.6" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T13.1.3.2.6.1" class="ltx_text" style="font-size:70%;">P</span></td>
</tr>
<tr id="S3.T13.1.4.3" class="ltx_tr">
<td id="S3.T13.1.4.3.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.4.3.1.1" class="ltx_text" style="font-size:70%;">Train</span></td>
<td id="S3.T13.1.4.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.4.3.2.1" class="ltx_text" style="font-size:70%;">Test</span></td>
</tr>
<tr id="S3.T13.1.5.4" class="ltx_tr">
<td id="S3.T13.1.5.4.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T13.1.5.4.1.1" class="ltx_text" style="font-size:70%;">Unconditional</span></td>
<td id="S3.T13.1.5.4.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.5.4.2.1" class="ltx_text" style="font-size:70%;">GAN,DM</span></td>
<td id="S3.T13.1.5.4.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.5.4.3.1" class="ltx_text" style="font-size:70%;">MR-brain,Chest x-ray</span></td>
<td id="S3.T13.1.5.4.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.5.4.4.1" class="ltx_text" style="font-size:70%;">StyleGAN, DDPM</span></td>
<td id="S3.T13.1.5.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.5.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib330" title="" class="ltx_ref">330</a><span id="S3.T13.1.5.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.5.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.5.4.6.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T13.1.5.4.7" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.5.4.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.5.4.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.5.4.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.5.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.5.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.5.4.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.5.4.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.5.4.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.5.4.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T13.1.6.5" class="ltx_tr">
<td id="S3.T13.1.6.5.1" class="ltx_td ltx_align_center">
<table id="S3.T13.1.6.5.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.6.5.1.1.1" class="ltx_tr">
<td id="S3.T13.1.6.5.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">
<span id="S3.T13.1.6.5.1.1.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">H:</span><span id="S3.T13.1.6.5.1.1.1.1.2" class="ltx_text" style="font-size:70%;">VAE,</span>
</td>
</tr>
<tr id="S3.T13.1.6.5.1.1.2" class="ltx_tr">
<td id="S3.T13.1.6.5.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.6.5.1.1.2.1.1" class="ltx_text" style="font-size:70%;">DDPM</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.6.5.2" class="ltx_td ltx_align_left">
<table id="S3.T13.1.6.5.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.6.5.2.1.1" class="ltx_tr">
<td id="S3.T13.1.6.5.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.6.5.2.1.1.1.1" class="ltx_text" style="font-size:70%;">3D:CT-lung, MR-brain,</span></td>
</tr>
<tr id="S3.T13.1.6.5.2.1.2" class="ltx_tr">
<td id="S3.T13.1.6.5.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.6.5.2.1.2.1.1" class="ltx_text" style="font-size:70%;">MR-breast, MR-knee</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.6.5.3" class="ltx_td ltx_align_left">
<table id="S3.T13.1.6.5.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.6.5.3.1.1" class="ltx_tr">
<td id="S3.T13.1.6.5.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.6.5.3.1.1.1.1" class="ltx_text" style="font-size:70%;">VQ-GAN followed</span></td>
</tr>
<tr id="S3.T13.1.6.5.3.1.2" class="ltx_tr">
<td id="S3.T13.1.6.5.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.6.5.3.1.2.1.1" class="ltx_text" style="font-size:70%;">by DDPM</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.6.5.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.6.5.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib61" title="" class="ltx_ref">61</a><span id="S3.T13.1.6.5.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.6.5.5" class="ltx_td ltx_align_left"><span id="S3.T13.1.6.5.5.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T13.1.6.5.6" class="ltx_td ltx_align_center"><span id="S3.T13.1.6.5.6.1" class="ltx_text" style="font-size:70%;">A</span></td>
<td id="S3.T13.1.6.5.7" class="ltx_td ltx_align_center"><span id="S3.T13.1.6.5.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T13.1.6.5.8" class="ltx_td"></td>
<td id="S3.T13.1.6.5.9" class="ltx_td ltx_align_center"><span id="S3.T13.1.6.5.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.6.5.10" class="ltx_td ltx_align_center"><span id="S3.T13.1.6.5.10.1" class="ltx_text" style="font-size:70%;">✓(2)</span></td>
<td id="S3.T13.1.6.5.11" class="ltx_td ltx_align_center"><span id="S3.T13.1.6.5.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.6.5.12" class="ltx_td"></td>
<td id="S3.T13.1.6.5.13" class="ltx_td ltx_align_center"><span id="S3.T13.1.6.5.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T13.1.7.6" class="ltx_tr">
<td id="S3.T13.1.7.6.1" class="ltx_td ltx_align_center"><span id="S3.T13.1.7.6.1.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T13.1.7.6.2" class="ltx_td ltx_align_left">
<table id="S3.T13.1.7.6.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.7.6.2.1.1" class="ltx_tr">
<td id="S3.T13.1.7.6.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.7.6.2.1.1.1.1" class="ltx_text" style="font-size:70%;">CT-abdomen, CT-pelvic,</span></td>
</tr>
<tr id="S3.T13.1.7.6.2.1.2" class="ltx_tr">
<td id="S3.T13.1.7.6.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.7.6.2.1.2.1.1" class="ltx_text" style="font-size:70%;">Chest x-ray, MR-heart</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.7.6.3" class="ltx_td ltx_align_left"><span id="S3.T13.1.7.6.3.1" class="ltx_text" style="font-size:70%;">MT-DDPM</span></td>
<td id="S3.T13.1.7.6.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.7.6.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib63" title="" class="ltx_ref">63</a><span id="S3.T13.1.7.6.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.7.6.5" class="ltx_td ltx_align_left"><span id="S3.T13.1.7.6.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T13.1.7.6.6" class="ltx_td ltx_align_center"><span id="S3.T13.1.7.6.6.1" class="ltx_text" style="font-size:70%;">S,A</span></td>
<td id="S3.T13.1.7.6.7" class="ltx_td ltx_align_center"><span id="S3.T13.1.7.6.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T13.1.7.6.8" class="ltx_td ltx_align_center"><span id="S3.T13.1.7.6.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.7.6.9" class="ltx_td ltx_align_center"><span id="S3.T13.1.7.6.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.7.6.10" class="ltx_td ltx_align_center"><span id="S3.T13.1.7.6.10.1" class="ltx_text" style="font-size:70%;">✓(3)</span></td>
<td id="S3.T13.1.7.6.11" class="ltx_td"></td>
<td id="S3.T13.1.7.6.12" class="ltx_td"></td>
<td id="S3.T13.1.7.6.13" class="ltx_td"></td>
</tr>
<tr id="S3.T13.1.8.7" class="ltx_tr">
<td id="S3.T13.1.8.7.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="3"><span id="S3.T13.1.8.7.1.1" class="ltx_text" style="font-size:70%;">
<span id="S3.T13.1.8.7.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T13.1.8.7.1.1.1.1" class="ltx_tr">
<span id="S3.T13.1.8.7.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">Class or Label</span></span>
<span id="S3.T13.1.8.7.1.1.1.2" class="ltx_tr">
<span id="S3.T13.1.8.7.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">cond.</span></span>
</span></span></td>
<td id="S3.T13.1.8.7.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.8.7.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T13.1.8.7.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.8.7.3.1" class="ltx_text" style="font-size:70%;">CT-lung, MR-brain</span></td>
<td id="S3.T13.1.8.7.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.8.7.4.1" class="ltx_text" style="font-size:70%;">HA-GAN</span></td>
<td id="S3.T13.1.8.7.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.8.7.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib64" title="" class="ltx_ref">64</a><span id="S3.T13.1.8.7.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.8.7.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.8.7.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T13.1.8.7.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T13.1.8.7.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.8.7.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.8.7.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.8.7.9" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.8.7.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.8.7.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.8.7.11" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.8.7.11.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.8.7.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.8.7.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.8.7.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T13.1.9.8" class="ltx_tr">
<td id="S3.T13.1.9.8.1" class="ltx_td ltx_align_center"><span id="S3.T13.1.9.8.1.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T13.1.9.8.2" class="ltx_td ltx_align_left">
<table id="S3.T13.1.9.8.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.9.8.2.1.1" class="ltx_tr">
<td id="S3.T13.1.9.8.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.9.8.2.1.1.1.1" class="ltx_text" style="font-size:70%;">Conditioned on seg</span></td>
</tr>
<tr id="S3.T13.1.9.8.2.1.2" class="ltx_tr">
<td id="S3.T13.1.9.8.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.9.8.2.1.2.1.1" class="ltx_text" style="font-size:70%;">mask: cardiac cine-MRI,</span></td>
</tr>
<tr id="S3.T13.1.9.8.2.1.3" class="ltx_tr">
<td id="S3.T13.1.9.8.2.1.3.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.9.8.2.1.3.1.1" class="ltx_text" style="font-size:70%;">liver CT, and Fundus images</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.9.8.3" class="ltx_td ltx_align_left"><span id="S3.T13.1.9.8.3.1" class="ltx_text" style="font-size:70%;">Multiple GANs</span></td>
<td id="S3.T13.1.9.8.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.9.8.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib331" title="" class="ltx_ref">331</a><span id="S3.T13.1.9.8.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.9.8.5" class="ltx_td ltx_align_left"><span id="S3.T13.1.9.8.5.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T13.1.9.8.6" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T13.1.9.8.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.9.8.7" class="ltx_td ltx_align_center"><span id="S3.T13.1.9.8.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.9.8.8" class="ltx_td"></td>
<td id="S3.T13.1.9.8.9" class="ltx_td ltx_align_center"><span id="S3.T13.1.9.8.9.1" class="ltx_text" style="font-size:70%;">✓(4)</span></td>
<td id="S3.T13.1.9.8.10" class="ltx_td"></td>
<td id="S3.T13.1.9.8.11" class="ltx_td"></td>
<td id="S3.T13.1.9.8.12" class="ltx_td"></td>
</tr>
<tr id="S3.T13.1.10.9" class="ltx_tr">
<td id="S3.T13.1.10.9.1" class="ltx_td ltx_align_center"><span id="S3.T13.1.10.9.1.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T13.1.10.9.2" class="ltx_td ltx_align_left">
<table id="S3.T13.1.10.9.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.10.9.2.1.1" class="ltx_tr">
<td id="S3.T13.1.10.9.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T13.1.10.9.2.1.1.1.1" class="ltx_text" style="font-size:70%;">Chest x-ray, Histopathology,</span></td>
</tr>
<tr id="S3.T13.1.10.9.2.1.2" class="ltx_tr">
<td id="S3.T13.1.10.9.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T13.1.10.9.2.1.2.1.1" class="ltx_text" style="font-size:70%;">Opthalmology Fundus images</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.10.9.3" class="ltx_td ltx_align_left">
<table id="S3.T13.1.10.9.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.10.9.3.1.1" class="ltx_tr">
<td id="S3.T13.1.10.9.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.10.9.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Medfusion:</span></td>
</tr>
<tr id="S3.T13.1.10.9.3.1.2" class="ltx_tr">
<td id="S3.T13.1.10.9.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.10.9.3.1.2.1.1" class="ltx_text" style="font-size:70%;">Conditional LDM</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.10.9.4" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.10.9.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib332" title="" class="ltx_ref">332</a><span id="S3.T13.1.10.9.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.10.9.5" class="ltx_td ltx_align_left"><span id="S3.T13.1.10.9.5.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T13.1.10.9.6" class="ltx_td ltx_align_center"><span id="S3.T13.1.10.9.6.1" class="ltx_text" style="font-size:70%;">S</span></td>
<td id="S3.T13.1.10.9.7" class="ltx_td ltx_align_center"><span id="S3.T13.1.10.9.7.1" class="ltx_text" style="font-size:70%;">R</span></td>
<td id="S3.T13.1.10.9.8" class="ltx_td ltx_align_center"><span id="S3.T13.1.10.9.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.10.9.9" class="ltx_td ltx_align_center"><span id="S3.T13.1.10.9.9.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.10.9.10" class="ltx_td"></td>
<td id="S3.T13.1.10.9.11" class="ltx_td"></td>
<td id="S3.T13.1.10.9.12" class="ltx_td"></td>
<td id="S3.T13.1.10.9.13" class="ltx_td ltx_align_center"><span id="S3.T13.1.10.9.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T13.1.11.10" class="ltx_tr">
<td id="S3.T13.1.11.10.1" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.11.10.1.1" class="ltx_text" style="font-size:70%;">Attribute cond.</span></td>
<td id="S3.T13.1.11.10.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.11.10.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T13.1.11.10.3" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T13.1.11.10.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.11.10.3.1.1" class="ltx_tr">
<td id="S3.T13.1.11.10.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.11.10.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Chest x-ray, Dermatoscopy,</span></td>
</tr>
<tr id="S3.T13.1.11.10.3.1.2" class="ltx_tr">
<td id="S3.T13.1.11.10.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.11.10.3.1.2.1.1" class="ltx_text" style="font-size:70%;">Histopathology</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.11.10.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.11.10.4.1" class="ltx_text" style="font-size:70%;">DDPM</span></td>
<td id="S3.T13.1.11.10.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.11.10.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib333" title="" class="ltx_ref">333</a><span id="S3.T13.1.11.10.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.11.10.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.11.10.6.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T13.1.11.10.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T13.1.11.10.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.11.10.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.11.10.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.11.10.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.11.10.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.11.10.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.11.10.13" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T13.1.12.11" class="ltx_tr">
<td id="S3.T13.1.12.11.1" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T13.1.12.11.1.1" class="ltx_text" style="font-size:70%;">Text-guided</span></td>
<td id="S3.T13.1.12.11.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T13.1.12.11.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T13.1.12.11.3" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T13.1.12.11.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.12.11.3.1.1" class="ltx_tr">
<td id="S3.T13.1.12.11.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.12.11.3.1.1.1.1" class="ltx_text" style="font-size:70%;">breast Ultrasound, spleen CT,</span></td>
</tr>
<tr id="S3.T13.1.12.11.3.1.2" class="ltx_tr">
<td id="S3.T13.1.12.11.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.12.11.3.1.2.1.1" class="ltx_text" style="font-size:70%;">prostate MR</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.12.11.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.12.11.4.1" class="ltx_text" style="font-size:70%;">EMIT-Diff</span></td>
<td id="S3.T13.1.12.11.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.12.11.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib334" title="" class="ltx_ref">334</a><span id="S3.T13.1.12.11.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.12.11.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.12.11.6.1" class="ltx_text" style="font-size:70%;">Seg</span></td>
<td id="S3.T13.1.12.11.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T13.1.12.11.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.12.11.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.12.11.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.12.11.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.12.11.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.12.11.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.12.11.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.12.11.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.12.11.13.1" class="ltx_text" style="font-size:70%;">**</span></td>
</tr>
<tr id="S3.T13.1.13.12" class="ltx_tr">
<td id="S3.T13.1.13.12.1" class="ltx_td ltx_align_left">
<table id="S3.T13.1.13.12.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.13.12.1.1.1" class="ltx_tr">
<td id="S3.T13.1.13.12.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.13.12.1.1.1.1.1" class="ltx_text" style="font-size:70%;">Chest x-ray, Histopathology,</span></td>
</tr>
<tr id="S3.T13.1.13.12.1.1.2" class="ltx_tr">
<td id="S3.T13.1.13.12.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.13.12.1.1.2.1.1" class="ltx_text" style="font-size:70%;">MR-prostate</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.13.12.2" class="ltx_td ltx_align_left">
<table id="S3.T13.1.13.12.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.13.12.2.1.1" class="ltx_tr">
<td id="S3.T13.1.13.12.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.13.12.2.1.1.1.1" class="ltx_text" style="font-size:70%;">FineTuned</span></td>
</tr>
<tr id="S3.T13.1.13.12.2.1.2" class="ltx_tr">
<td id="S3.T13.1.13.12.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.13.12.2.1.2.1.1" class="ltx_text" style="font-size:70%;">StableDiffusion</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.13.12.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.13.12.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib335" title="" class="ltx_ref">335</a><span id="S3.T13.1.13.12.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.13.12.4" class="ltx_td ltx_align_left"><span id="S3.T13.1.13.12.4.1" class="ltx_text" style="font-size:70%;">Class</span></td>
<td id="S3.T13.1.13.12.5" class="ltx_td ltx_align_center" colspan="2"><span id="S3.T13.1.13.12.5.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.13.12.6" class="ltx_td ltx_align_center"><span id="S3.T13.1.13.12.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.13.12.7" class="ltx_td"></td>
<td id="S3.T13.1.13.12.8" class="ltx_td"></td>
<td id="S3.T13.1.13.12.9" class="ltx_td"></td>
<td id="S3.T13.1.13.12.10" class="ltx_td"></td>
<td id="S3.T13.1.13.12.11" class="ltx_td"></td>
</tr>
<tr id="S3.T13.1.14.13" class="ltx_tr">
<td id="S3.T13.1.14.13.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_t" rowspan="3"><span id="S3.T13.1.14.13.1.1" class="ltx_text" style="font-size:70%;">Other cond.</span></td>
<td id="S3.T13.1.14.13.2" class="ltx_td ltx_align_center ltx_border_t" rowspan="2"><span id="S3.T13.1.14.13.2.1" class="ltx_text" style="font-size:70%;">DM</span></td>
<td id="S3.T13.1.14.13.3" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.14.13.3.1" class="ltx_text" style="font-size:70%;">prostate-MRI, chest x-ray</span></td>
<td id="S3.T13.1.14.13.4" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T13.1.14.13.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.14.13.4.1.1" class="ltx_tr">
<td id="S3.T13.1.14.13.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.14.13.4.1.1.1.1" class="ltx_text" style="font-size:70%;">Med-cDiff:</span></td>
</tr>
<tr id="S3.T13.1.14.13.4.1.2" class="ltx_tr">
<td id="S3.T13.1.14.13.4.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.14.13.4.1.2.1.1" class="ltx_text" style="font-size:70%;">Conditional DDPM</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.14.13.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.14.13.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib336" title="" class="ltx_ref">336</a><span id="S3.T13.1.14.13.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.14.13.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T13.1.14.13.6.1" class="ltx_text" style="font-size:70%;">Seg,Class</span></td>
<td id="S3.T13.1.14.13.7" class="ltx_td ltx_align_center ltx_border_t" colspan="2"><span id="S3.T13.1.14.13.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.14.13.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T13.1.14.13.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.14.13.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.14.13.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.14.13.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.14.13.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T13.1.14.13.13" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T13.1.15.14" class="ltx_tr">
<td id="S3.T13.1.15.14.1" class="ltx_td ltx_align_left">
<table id="S3.T13.1.15.14.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.15.14.1.1.1" class="ltx_tr">
<td id="S3.T13.1.15.14.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.15.14.1.1.1.1.1" class="ltx_text" style="font-size:70%;">Conditioned on 2perpendicular</span></td>
</tr>
<tr id="S3.T13.1.15.14.1.1.2" class="ltx_tr">
<td id="S3.T13.1.15.14.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.15.14.1.1.2.1.1" class="ltx_text" style="font-size:70%;">2d DMs as a 3d prior: MRI, CT</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.15.14.2" class="ltx_td ltx_align_left"><span id="S3.T13.1.15.14.2.1" class="ltx_text" style="font-size:70%;">TPDM</span></td>
<td id="S3.T13.1.15.14.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.15.14.3.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib337" title="" class="ltx_ref">337</a><span id="S3.T13.1.15.14.3.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.15.14.4" class="ltx_td ltx_align_left"><span id="S3.T13.1.15.14.4.1" class="ltx_text" style="font-size:70%;">None</span></td>
<td id="S3.T13.1.15.14.5" class="ltx_td"></td>
<td id="S3.T13.1.15.14.6" class="ltx_td"></td>
<td id="S3.T13.1.15.14.7" class="ltx_td ltx_align_center"><span id="S3.T13.1.15.14.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.15.14.8" class="ltx_td"></td>
<td id="S3.T13.1.15.14.9" class="ltx_td"></td>
<td id="S3.T13.1.15.14.10" class="ltx_td"></td>
<td id="S3.T13.1.15.14.11" class="ltx_td"></td>
<td id="S3.T13.1.15.14.12" class="ltx_td ltx_align_center"><span id="S3.T13.1.15.14.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T13.1.1" class="ltx_tr">
<td id="S3.T13.1.1.2" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T13.1.1.2.1" class="ltx_text" style="font-size:70%;">GAN</span></td>
<td id="S3.T13.1.1.3" class="ltx_td ltx_align_left ltx_border_b">
<table id="S3.T13.1.1.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T13.1.1.3.1.1" class="ltx_tr">
<td id="S3.T13.1.1.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.1.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Combining Chest x-ray</span></td>
</tr>
<tr id="S3.T13.1.1.3.1.2" class="ltx_tr">
<td id="S3.T13.1.1.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T13.1.1.3.1.2.1.1" class="ltx_text" style="font-size:70%;">images and Tabular Data</span></td>
</tr>
</table>
</td>
<td id="S3.T13.1.1.1" class="ltx_td ltx_align_left ltx_border_b">
<math id="S3.T13.1.1.1.m1.1" class="ltx_Math" alttext="\alpha" display="inline"><semantics id="S3.T13.1.1.1.m1.1a"><mi mathsize="70%" id="S3.T13.1.1.1.m1.1.1" xref="S3.T13.1.1.1.m1.1.1.cmml">α</mi><annotation-xml encoding="MathML-Content" id="S3.T13.1.1.1.m1.1b"><ci id="S3.T13.1.1.1.m1.1.1.cmml" xref="S3.T13.1.1.1.m1.1.1">𝛼</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.T13.1.1.1.m1.1c">\alpha</annotation></semantics></math><span id="S3.T13.1.1.1.1" class="ltx_text" style="font-size:70%;">GAN and CTGAN</span>
</td>
<td id="S3.T13.1.1.4" class="ltx_td ltx_align_left ltx_border_b"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T13.1.1.4.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib338" title="" class="ltx_ref">338</a><span id="S3.T13.1.1.4.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T13.1.1.5" class="ltx_td ltx_align_left ltx_border_b"><span id="S3.T13.1.1.5.1" class="ltx_text" style="font-size:70%;">Class,Reg</span></td>
<td id="S3.T13.1.1.6" class="ltx_td ltx_align_center ltx_border_b" colspan="2"><span id="S3.T13.1.1.6.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T13.1.1.7" class="ltx_td ltx_border_b"></td>
<td id="S3.T13.1.1.8" class="ltx_td ltx_border_b"></td>
<td id="S3.T13.1.1.9" class="ltx_td ltx_align_center ltx_border_b"><span id="S3.T13.1.1.9.1" class="ltx_text" style="font-size:70%;">*</span></td>
<td id="S3.T13.1.1.10" class="ltx_td ltx_border_b"></td>
<td id="S3.T13.1.1.11" class="ltx_td ltx_border_b"></td>
<td id="S3.T13.1.1.12" class="ltx_td ltx_border_b"></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 13: </span>Overview of all summarized and discussed studies on <span id="S3.T13.6.1" class="ltx_text ltx_font_bold">multiple modalities</span> synthesis with information about the technology utilized, the downstream task (DT), the evaluation procedure and code availability. Evaluation is split according to the evaluation metrics reported, covering utility (U), fidelity (F), diversity (D), qualitative assessment (Q), clinical utility (C) and privacy (P) evaluation.In the Type column, H stands for hybrid approach combining two different models. In evaluation of utility (U), R stands for real data, S stands for synthetic data, and A stands for augmented data. In qualitative evaluation, the number between parenthesis indicate the number of specialists employed, while * stands for visual inspection of visualization plots.** stands for the availability of code of the original method. Acronyms: Classification (class), Segmentation (seg), regression (reg), conditioning (cond),Denoising Diffusion Probabilistic Model (DDPM), Transformer-based Denoising Diffusion Probabilistic Model (MT-DDPM),Hierarchical Amortized GAN (HA-GAN),Two Perpendicular 2D DMs (TPDM) </figcaption>
</figure>
<div id="S3.SS3.SSS8.p1" class="ltx_para">
<p id="S3.SS3.SSS8.p1.1" class="ltx_p">In preceding sections, the focus was primarily on papers centered around generative models targeting a single modality, such as exclusive generation of MRIs or CT scans. This section delves into papers exploring models capable of generating multiple modalities using the same architecture, thereby avoiding significant modifications. For instance, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib61" title="" class="ltx_ref">61</a>]</cite> demonstrated that diffusion models are capable of generating realistic 3D synthetic data across four different anatomical regions, two modalities (CT and MRI), and three different resolutions. Remarkably, the diffusion model achieved this without the need for fine-tuning any hyper-parameters to adapt to the different datasets, despite the small training datasets. In addition to the diffusion model, a VQ-GAN architecture was used to compress the 3D CT and MRI images into a latent space, where the DDPM process was performed. In <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib330" title="" class="ltx_ref">330</a>]</cite>, a comparison was made between the performance of a GAN and a diffusion model in generating synthetic brain MRI and chest X-rays, with a focus on memorization of training data.<cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib63" title="" class="ltx_ref">63</a>]</cite> utilize Swin-transformer in the design of a diffusion model to enhance the image synthesis quality, applied on multiple imaging modalities. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib334" title="" class="ltx_ref">334</a>]</cite> adopts a holistic approach to medical image synthesis, utilizing controllable diffusion with DDPMs incorporating edge information to generate realistic and diverse synthetic ultrasounds, spleen CTs, or prostate MRI images while preserving essential characteristics. Initial pretraining on RadImageNet involves triplets for diverse prompts, while edge information from RadImageNet, guided by a HED algorithm (Holistically-Nested Edge Detection) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib339" title="" class="ltx_ref">339</a>]</cite>, enhances realistic image generation. Subsequent fine-tuning on a task-specific dataset adapts the diffusion model to unique characteristics.</p>
</div>
<div id="S3.SS3.SSS8.p2" class="ltx_para">
<p id="S3.SS3.SSS8.p2.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib331" title="" class="ltx_ref">331</a>]</cite> employed multiple GANs conditioned on segmentation label masks for different MRI, CT, and retinal images, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib332" title="" class="ltx_ref">332</a>]</cite> utilized a class conditional LDM on various modalities including chest x-ray, histopathology, and fundus images. Notably, a DDPM was employed in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib333" title="" class="ltx_ref">333</a>]</cite> conditioned on either the diagnostic label alone or in conjunction with a property such as hospital or a sensitive attribute (e.g.: ethnicity,sex). Augmenting the training data with synthetic data improved both the accuracy and fairness of downstream classification models, particularly for under-represented groups in the training data.</p>
</div>
<div id="S3.SS3.SSS8.p3" class="ltx_para">
<p id="S3.SS3.SSS8.p3.1" class="ltx_p">Training diffusion-based models for text-to-image generation from the scratch demands access to extensive datasets with image-caption pairs and significant computational resources. However, in the context of medical image generation, the availability of large, publicly accessible datasets containing text reports is limited. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib335" title="" class="ltx_ref">335</a>]</cite> proposes a solution by demonstrating that pre-trained Stable Diffusion models, initially trained on natural images, can be adapted to diverse medical imaging modalities through textual inversion <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib340" title="" class="ltx_ref">340</a>]</cite>. To introduce a medical modality as a new concept to a pre-trained diffusion model, textual inversion finds a vector in the text embedding space that best represents the new concept. This is achieved by freezing the entire architecture except for this specific embedding vector, and then performing backpropagation with a similarity loss using a small set of example images.</p>
</div>
<div id="S3.SS3.SSS8.p4" class="ltx_para">
<p id="S3.SS3.SSS8.p4.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib338" title="" class="ltx_ref">338</a>]</cite> is the only paper in our survey that generates synthetic hybrid data that combines both images and non-image data, whereby two GANs are used to create synthetic records containing both CXRs and non-image clinical data. This is done by reducing the dimensionality of the images using the pretrained encoder of the GAN.</p>
</div>
<div id="S3.SS3.SSS8.p5" class="ltx_para">
<p id="S3.SS3.SSS8.p5.1" class="ltx_p">While the majority of papers focus on utility and fidelity evaluation of the synthetic data, only few papers considered measures of diversity <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib61" title="" class="ltx_ref">61</a>, <a href="#bib.bib63" title="" class="ltx_ref">63</a>, <a href="#bib.bib332" title="" class="ltx_ref">332</a>, <a href="#bib.bib330" title="" class="ltx_ref">330</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib63" title="" class="ltx_ref">63</a>]</cite> used a diversity score (DS) to measure how diverse the synthetic images were. The DS, or the nearest SSIM difference, is the KL divergence between the distributions of nearest SSIMs in real and synthetic images. To find the nearest SSIM for the synthetic images, the authors calculated the SSIM of each synthetic image with all the others and found the closest match. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib330" title="" class="ltx_ref">330</a>]</cite> argues that if the SSIM was calculated between each synthetic image and all training images, then this score would quantify the memorization of the training data by the model. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib61" title="" class="ltx_ref">61</a>]</cite> also used SSIM to measure diversity of synthetic images, by calculating the SSIM between pairs of synthetic data to check how similar they are, and thus investigate model’s ability to generate diverse images.</p>
</div>
<div id="S3.SS3.SSS8.p6" class="ltx_para">
<p id="S3.SS3.SSS8.p6.1" class="ltx_p">Few studies consider qualitative assessment of the generated data, with a limitation of considering small number of experts. For instance, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib61" title="" class="ltx_ref">61</a>]</cite> relied on the evaluation of image quality by only two human experts in terms of realistic image appearance, anatomical correctness, and consistency between slices, while <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib63" title="" class="ltx_ref">63</a>]</cite> involved three experts, and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib331" title="" class="ltx_ref">331</a>]</cite> considered four experts.</p>
</div>
<div id="S3.SS3.SSS8.p7" class="ltx_para">
<p id="S3.SS3.SSS8.p7.1" class="ltx_p">Finally, the evaluation of memorization i.e. that generative models can simply generate samples that are copies of the training data, is the focus of <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib330" title="" class="ltx_ref">330</a>]</cite>. It involved calculating pairwise image correlation between each synthetic image (from a selection of only 100 synthetic images for analysis) and the entire set of training images, and the findings indicated that diffusion models exhibit a higher susceptibility to memorizing the training images than GANs.</p>
</div>
</section>
</section>
<section id="S3.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.4 </span>Medical text</h3>

<figure id="S3.T14" class="ltx_table">
<table id="S3.T14.1" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S3.T14.1.1.1" class="ltx_tr">
<th id="S3.T14.1.1.1.1" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" rowspan="2"><span id="S3.T14.1.1.1.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Application</span></th>
<th id="S3.T14.1.1.1.2" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" rowspan="2"><span id="S3.T14.1.1.1.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Type</span></th>
<th id="S3.T14.1.1.1.3" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_t" rowspan="2"><span id="S3.T14.1.1.1.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Modality</span></th>
<th id="S3.T14.1.1.1.4" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" rowspan="2"><span id="S3.T14.1.1.1.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Technology</span></th>
<th id="S3.T14.1.1.1.5" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" rowspan="2"><span id="S3.T14.1.1.1.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Paper</span></th>
<th id="S3.T14.1.1.1.6" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" rowspan="2"><span id="S3.T14.1.1.1.6.1" class="ltx_text" style="font-size:70%;">
<span id="S3.T14.1.1.1.6.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T14.1.1.1.6.1.1.1" class="ltx_tr">
<span id="S3.T14.1.1.1.6.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.1.1.6.1.1.1.1.1" class="ltx_text ltx_font_bold">Downstream</span></span></span>
<span id="S3.T14.1.1.1.6.1.1.2" class="ltx_tr">
<span id="S3.T14.1.1.1.6.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.1.1.6.1.1.2.1.1" class="ltx_text ltx_font_bold">Task</span></span></span>
</span></span></th>
<th id="S3.T14.1.1.1.7" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" colspan="6"><span id="S3.T14.1.1.1.7.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Evaluation</span></th>
<th id="S3.T14.1.1.1.8" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" rowspan="2"><span id="S3.T14.1.1.1.8.1" class="ltx_text" style="font-size:70%;">
<span id="S3.T14.1.1.1.8.1.1" class="ltx_tabular ltx_align_middle">
<span id="S3.T14.1.1.1.8.1.1.1" class="ltx_tr">
<span id="S3.T14.1.1.1.8.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.1.1.8.1.1.1.1.1" class="ltx_text ltx_font_bold">Comparison</span></span></span>
</span></span></th>
<th id="S3.T14.1.1.1.9" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t" rowspan="2"><span id="S3.T14.1.1.1.9.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Code</span></th>
</tr>
<tr id="S3.T14.1.2.2" class="ltx_tr">
<th id="S3.T14.1.2.2.1" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.2.2.1.1" class="ltx_text ltx_font_bold" style="font-size:70%;">U</span></th>
<th id="S3.T14.1.2.2.2" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.2.2.2.1" class="ltx_text ltx_font_bold" style="font-size:70%;">F</span></th>
<th id="S3.T14.1.2.2.3" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.2.2.3.1" class="ltx_text ltx_font_bold" style="font-size:70%;">D</span></th>
<th id="S3.T14.1.2.2.4" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.2.2.4.1" class="ltx_text ltx_font_bold" style="font-size:70%;">Q</span></th>
<th id="S3.T14.1.2.2.5" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.2.2.5.1" class="ltx_text ltx_font_bold" style="font-size:70%;">C</span></th>
<th id="S3.T14.1.2.2.6" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.2.2.6.1" class="ltx_text ltx_font_bold" style="font-size:70%;">P</span></th>
</tr>
<tr id="S3.T14.1.3.3" class="ltx_tr">
<th id="S3.T14.1.3.3.1" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t">
<table id="S3.T14.1.3.3.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.3.3.1.1.1" class="ltx_tr">
<td id="S3.T14.1.3.3.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.3.3.1.1.1.1.1" class="ltx_text" style="font-size:70%;">NLP Enhancement</span></td>
</tr>
</table>
</th>
<th id="S3.T14.1.3.3.2" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.3.3.2.1" class="ltx_text" style="font-size:70%;">LLM</span></th>
<th id="S3.T14.1.3.3.3" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_t">
<table id="S3.T14.1.3.3.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.3.3.3.1.1" class="ltx_tr">
<td id="S3.T14.1.3.3.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.3.3.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Discharge summary</span></td>
</tr>
</table>
</th>
<th id="S3.T14.1.3.3.4" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_t">
<table id="S3.T14.1.3.3.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.3.3.4.1.1" class="ltx_tr">
<td id="S3.T14.1.3.3.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.3.3.4.1.1.1.1" class="ltx_text" style="font-size:70%;">LLaMa</span></td>
</tr>
<tr id="S3.T14.1.3.3.4.1.2" class="ltx_tr">
<td id="S3.T14.1.3.3.4.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.3.3.4.1.2.1.1" class="ltx_text" style="font-size:70%;">GPT-3.5, 4</span></td>
</tr>
</table>
</th>
<th id="S3.T14.1.3.3.5" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T14.1.3.3.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib341" title="" class="ltx_ref">341</a><span id="S3.T14.1.3.3.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></th>
<th id="S3.T14.1.3.3.6" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.3.3.6.1" class="ltx_text" style="font-size:70%;">NLP Tasks</span></th>
<th id="S3.T14.1.3.3.7" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.3.3.7.1" class="ltx_text" style="font-size:70%;">✓</span></th>
<th id="S3.T14.1.3.3.8" class="ltx_td ltx_th ltx_th_column ltx_border_t"></th>
<th id="S3.T14.1.3.3.9" class="ltx_td ltx_th ltx_th_column ltx_border_t"></th>
<th id="S3.T14.1.3.3.10" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.3.3.10.1" class="ltx_text" style="font-size:70%;">✓</span></th>
<th id="S3.T14.1.3.3.11" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.3.3.11.1" class="ltx_text" style="font-size:70%;">✓</span></th>
<th id="S3.T14.1.3.3.12" class="ltx_td ltx_th ltx_th_column ltx_border_t"></th>
<th id="S3.T14.1.3.3.13" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.3.3.13.1" class="ltx_text" style="font-size:70%;">✓</span></th>
<th id="S3.T14.1.3.3.14" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span id="S3.T14.1.3.3.14.1" class="ltx_text" style="font-size:70%;">✓</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S3.T14.1.4.1" class="ltx_tr">
<td id="S3.T14.1.4.1.1" class="ltx_td ltx_align_center ltx_border_t">
<table id="S3.T14.1.4.1.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.4.1.1.1.1" class="ltx_tr">
<td id="S3.T14.1.4.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.4.1.1.1.1.1.1" class="ltx_text" style="font-size:70%;">NLP Enhancement</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.4.1.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.4.1.2.1" class="ltx_text" style="font-size:70%;">LLM</span></td>
<td id="S3.T14.1.4.1.3" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.4.1.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.4.1.3.1.1" class="ltx_tr">
<td id="S3.T14.1.4.1.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.4.1.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Diabetic ketoacidosis</span></td>
</tr>
<tr id="S3.T14.1.4.1.3.1.2" class="ltx_tr">
<td id="S3.T14.1.4.1.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.4.1.3.1.2.1.1" class="ltx_text" style="font-size:70%;">guideline</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.4.1.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T14.1.4.1.4.1" class="ltx_text" style="font-size:70%;">GPT-4</span></td>
<td id="S3.T14.1.4.1.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T14.1.4.1.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib342" title="" class="ltx_ref">342</a><span id="S3.T14.1.4.1.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T14.1.4.1.6" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.4.1.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.4.1.6.1.1" class="ltx_tr">
<td id="S3.T14.1.4.1.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.4.1.6.1.1.1.1" class="ltx_text" style="font-size:70%;">Question</span></td>
</tr>
<tr id="S3.T14.1.4.1.6.1.2" class="ltx_tr">
<td id="S3.T14.1.4.1.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.4.1.6.1.2.1.1" class="ltx_text" style="font-size:70%;">Answering</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.4.1.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.4.1.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.4.1.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.4.1.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.4.1.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.4.1.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.4.1.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.4.1.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.4.1.14" class="ltx_td ltx_border_t"></td>
</tr>
<tr id="S3.T14.1.5.2" class="ltx_tr">
<td id="S3.T14.1.5.2.1" class="ltx_td ltx_align_center ltx_border_t">
<table id="S3.T14.1.5.2.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.5.2.1.1.1" class="ltx_tr">
<td id="S3.T14.1.5.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.5.2.1.1.1.1.1" class="ltx_text" style="font-size:70%;">NLP Enhancement</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.5.2.2" class="ltx_td ltx_align_center ltx_border_t">
<table id="S3.T14.1.5.2.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.5.2.2.1.1" class="ltx_tr">
<td id="S3.T14.1.5.2.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.5.2.2.1.1.1.1" class="ltx_text" style="font-size:70%;">RNN,</span></td>
</tr>
<tr id="S3.T14.1.5.2.2.1.2" class="ltx_tr">
<td id="S3.T14.1.5.2.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.5.2.2.1.2.1.1" class="ltx_text" style="font-size:70%;">GAN,</span></td>
</tr>
<tr id="S3.T14.1.5.2.2.1.3" class="ltx_tr">
<td id="S3.T14.1.5.2.2.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.5.2.2.1.3.1.1" class="ltx_text" style="font-size:70%;">PLM</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.5.2.3" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.5.2.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.5.2.3.1.1" class="ltx_tr">
<td id="S3.T14.1.5.2.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.5.2.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Patient history,</span></td>
</tr>
<tr id="S3.T14.1.5.2.3.1.2" class="ltx_tr">
<td id="S3.T14.1.5.2.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.5.2.3.1.2.1.1" class="ltx_text" style="font-size:70%;">present Illness</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.5.2.4" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.5.2.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.5.2.4.1.1" class="ltx_tr">
<td id="S3.T14.1.5.2.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.5.2.4.1.1.1.1" class="ltx_text" style="font-size:70%;">CharRNN, GAN,</span></td>
</tr>
<tr id="S3.T14.1.5.2.4.1.2" class="ltx_tr">
<td id="S3.T14.1.5.2.4.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.5.2.4.1.2.1.1" class="ltx_text" style="font-size:70%;">GPT-2, CTRL</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.5.2.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T14.1.5.2.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib343" title="" class="ltx_ref">343</a><span id="S3.T14.1.5.2.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T14.1.5.2.6" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.5.2.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.5.2.6.1.1" class="ltx_tr">
<td id="S3.T14.1.5.2.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.5.2.6.1.1.1.1" class="ltx_text" style="font-size:70%;">Name entity</span></td>
</tr>
<tr id="S3.T14.1.5.2.6.1.2" class="ltx_tr">
<td id="S3.T14.1.5.2.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.5.2.6.1.2.1.1" class="ltx_text" style="font-size:70%;">recognition</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.5.2.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.5.2.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.5.2.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.5.2.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.5.2.10" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.5.2.10.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.5.2.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.5.2.12" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.5.2.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.5.2.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.5.2.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T14.1.6.3" class="ltx_tr">
<td id="S3.T14.1.6.3.1" class="ltx_td ltx_align_center ltx_border_t">
<table id="S3.T14.1.6.3.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.6.3.1.1.1" class="ltx_tr">
<td id="S3.T14.1.6.3.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.6.3.1.1.1.1.1" class="ltx_text" style="font-size:70%;">Text augmentation,</span></td>
</tr>
<tr id="S3.T14.1.6.3.1.1.2" class="ltx_tr">
<td id="S3.T14.1.6.3.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.6.3.1.1.2.1.1" class="ltx_text" style="font-size:70%;">de-identification</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.6.3.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.6.3.2.1" class="ltx_text" style="font-size:70%;">PLM</span></td>
<td id="S3.T14.1.6.3.3" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.6.3.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.6.3.3.1.1" class="ltx_tr">
<td id="S3.T14.1.6.3.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.6.3.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Injury records</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.6.3.4" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T14.1.6.3.4.1" class="ltx_text" style="font-size:70%;">BERT</span></td>
<td id="S3.T14.1.6.3.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T14.1.6.3.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib344" title="" class="ltx_ref">344</a><span id="S3.T14.1.6.3.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T14.1.6.3.6" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.6.3.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.6.3.6.1.1" class="ltx_tr">
<td id="S3.T14.1.6.3.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.6.3.6.1.1.1.1" class="ltx_text" style="font-size:70%;">Replace sensitive</span></td>
</tr>
<tr id="S3.T14.1.6.3.6.1.2" class="ltx_tr">
<td id="S3.T14.1.6.3.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.6.3.6.1.2.1.1" class="ltx_text" style="font-size:70%;">data</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.6.3.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.6.3.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.6.3.8" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.6.3.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.6.3.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.6.3.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.6.3.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.6.3.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.6.3.13" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.6.3.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.6.3.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T14.1.7.4" class="ltx_tr">
<td id="S3.T14.1.7.4.1" class="ltx_td ltx_align_center ltx_border_t">
<table id="S3.T14.1.7.4.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.7.4.1.1.1" class="ltx_tr">
<td id="S3.T14.1.7.4.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.7.4.1.1.1.1.1" class="ltx_text" style="font-size:70%;">Text augmentation</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.7.4.2" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.7.4.2.1" class="ltx_text" style="font-size:70%;">PLM</span></td>
<td id="S3.T14.1.7.4.3" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.7.4.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.7.4.3.1.1" class="ltx_tr">
<td id="S3.T14.1.7.4.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.7.4.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Discharge summary</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.7.4.4" class="ltx_td ltx_align_left ltx_border_t">
<table id="S3.T14.1.7.4.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.7.4.4.1.1" class="ltx_tr">
<td id="S3.T14.1.7.4.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.7.4.4.1.1.1.1" class="ltx_text" style="font-size:70%;">Transformer,</span></td>
</tr>
<tr id="S3.T14.1.7.4.4.1.2" class="ltx_tr">
<td id="S3.T14.1.7.4.4.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.7.4.4.1.2.1.1" class="ltx_text" style="font-size:70%;">GPT-2</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.7.4.5" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T14.1.7.4.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib345" title="" class="ltx_ref">345</a><span id="S3.T14.1.7.4.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T14.1.7.4.6" class="ltx_td ltx_align_left ltx_border_t"><span id="S3.T14.1.7.4.6.1" class="ltx_text" style="font-size:70%;">Classification</span></td>
<td id="S3.T14.1.7.4.7" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.7.4.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.7.4.8" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.7.4.8.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.7.4.9" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.7.4.10" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.7.4.11" class="ltx_td ltx_border_t"></td>
<td id="S3.T14.1.7.4.12" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.7.4.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.7.4.13" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.7.4.13.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.7.4.14" class="ltx_td ltx_align_center ltx_border_t"><span id="S3.T14.1.7.4.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
<tr id="S3.T14.1.8.5" class="ltx_tr">
<td id="S3.T14.1.8.5.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_t">
<table id="S3.T14.1.8.5.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.8.5.1.1.1" class="ltx_tr">
<td id="S3.T14.1.8.5.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.8.5.1.1.1.1.1" class="ltx_text" style="font-size:70%;">De-identification,</span></td>
</tr>
<tr id="S3.T14.1.8.5.1.1.2" class="ltx_tr">
<td id="S3.T14.1.8.5.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.8.5.1.1.2.1.1" class="ltx_text" style="font-size:70%;">Text augmentation</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.8.5.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T14.1.8.5.2.1" class="ltx_text" style="font-size:70%;">PLM</span></td>
<td id="S3.T14.1.8.5.3" class="ltx_td ltx_align_left ltx_border_b ltx_border_t">
<table id="S3.T14.1.8.5.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.8.5.3.1.1" class="ltx_tr">
<td id="S3.T14.1.8.5.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S3.T14.1.8.5.3.1.1.1.1" class="ltx_text" style="font-size:70%;">Discharge summary</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.8.5.4" class="ltx_td ltx_align_left ltx_border_b ltx_border_t">
<table id="S3.T14.1.8.5.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.8.5.4.1.1" class="ltx_tr">
<td id="S3.T14.1.8.5.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.8.5.4.1.1.1.1" class="ltx_text" style="font-size:70%;">LSTM, GPT-2</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.8.5.5" class="ltx_td ltx_align_left ltx_border_b ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite"><span id="S3.T14.1.8.5.5.1.1" class="ltx_text" style="font-size:70%;">[</span><a href="#bib.bib346" title="" class="ltx_ref">346</a><span id="S3.T14.1.8.5.5.2.2" class="ltx_text" style="font-size:70%;">]</span></cite></td>
<td id="S3.T14.1.8.5.6" class="ltx_td ltx_align_left ltx_border_b ltx_border_t">
<table id="S3.T14.1.8.5.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S3.T14.1.8.5.6.1.1" class="ltx_tr">
<td id="S3.T14.1.8.5.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.8.5.6.1.1.1.1" class="ltx_text" style="font-size:70%;">Name entity</span></td>
</tr>
<tr id="S3.T14.1.8.5.6.1.2" class="ltx_tr">
<td id="S3.T14.1.8.5.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left"><span id="S3.T14.1.8.5.6.1.2.1.1" class="ltx_text" style="font-size:70%;">recognition</span></td>
</tr>
</table>
</td>
<td id="S3.T14.1.8.5.7" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T14.1.8.5.7.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.8.5.8" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T14.1.8.5.9" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T14.1.8.5.10" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T14.1.8.5.11" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T14.1.8.5.12" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T14.1.8.5.12.1" class="ltx_text" style="font-size:70%;">✓</span></td>
<td id="S3.T14.1.8.5.13" class="ltx_td ltx_border_b ltx_border_t"></td>
<td id="S3.T14.1.8.5.14" class="ltx_td ltx_align_center ltx_border_b ltx_border_t"><span id="S3.T14.1.8.5.14.1" class="ltx_text" style="font-size:70%;">✓</span></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 14: </span>Reviewed generative models for <span id="S3.T14.6.1" class="ltx_text ltx_font_bold">medical text</span>. Acronyms: Large Language Model (LLM),Large Language Model Meta AI (LLaMa), Generative Pre-Trained Transformer (GPT), Pre-trained Language Models (PLM), Natural Language Processing (NLP), Recurrent Neural Netwrok (RNN), Character Recurrent Neural Network (CharRNN), Conditional Transformer Language (CTRL), Bidirectional Encoder Representations from Transformers (BERT)</figcaption>
</figure>
<div id="S3.SS4.p1" class="ltx_para">
<p id="S3.SS4.p1.1" class="ltx_p">Different from synthesizing medical imaging or signal data, medical text generation focuses on language models and transformers. As we mentioned before, the main purposes of generating medical text are categorized into (i) enhancing NLP tasks such as Name entity recognition, concept extraction, relation extraction, and question answering, (ii) augmenting medical text data where the high-quality clinical notes are limited and privacy sensitive to use for research, (iii) replacing sensitive information in the real data with generated synthetic data. A variety of medical text data types have been utilized to train the models including</p>
<ol id="S3.I1" class="ltx_enumerate">
<li id="S3.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S3.I1.i1.p1" class="ltx_para">
<p id="S3.I1.i1.p1.1" class="ltx_p">discharge summary, a clinical document summarizing the patient’s hospitalization from admission to discharge to provide a comprehensive overview of the patient’s diagnosis, treatments, condition at discharge, and recommendations for follow-up care;</p>
</div>
</li>
<li id="S3.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S3.I1.i2.p1" class="ltx_para">
<p id="S3.I1.i2.p1.1" class="ltx_p">disease guideline (a.k.a clinical guideline or practice guideline), a systematically developed documentation to assist medical professionals and patient decisions about appropriate healthcare for specific clinical situations;</p>
</div>
</li>
<li id="S3.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span> 
<div id="S3.I1.i3.p1" class="ltx_para">
<p id="S3.I1.i3.p1.1" class="ltx_p">case report, a detailed report of the symptoms, diagnosis, treatment, and follow-up of an individual patient;</p>
</div>
</li>
<li id="S3.I1.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">4.</span> 
<div id="S3.I1.i4.p1" class="ltx_para">
<p id="S3.I1.i4.p1.1" class="ltx_p">history of present illness which is a detailed documentation of the development of the patient’s illness, including the onset of symptoms, duration, intensity, and any factors that aggravate or relieve them;</p>
</div>
</li>
<li id="S3.I1.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">5.</span> 
<div id="S3.I1.i5.p1" class="ltx_para">
<p id="S3.I1.i5.p1.1" class="ltx_p">chief complaint which describes the primary reason or symptoms that brought the patient to seek medical attention</p>
</div>
</li>
</ol>
</div>
<div id="S3.SS4.p2" class="ltx_para">
<p id="S3.SS4.p2.1" class="ltx_p">Some additional input can also be used for medical text generation such as EHR data to generate clinical notes and predict outcomes <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib345" title="" class="ltx_ref">345</a>]</cite>, diagnostic reports where studies take radiology, pathology, and other diagnostic images as input to generate imaging reports with detailed descriptions and diagnosis, clinical trial reports where studies take extensive medical documents as input to generate summarization of the reports.</p>
</div>
<div id="S3.SS4.p3" class="ltx_para">
<p id="S3.SS4.p3.1" class="ltx_p">In medical text generation, there are no standards or well-accepted evaluation metrics to assess the fidelity, accuracy, and clinical utility for the generators. Moreover, an additional crucial aspect in evaluating synthetic medical text is the truthfulness and correctness of the generated data, ensuring that the information presented adheres to current medical guidelines and practices. If a medical text generator were to generate inaccurate or misleading information (such as hallucinating facts), the implications could be far more serious than with general-purpose text generators. Such errors could mislead healthcare professionals, impacting their decision-making processes and potentially affecting patient care outcomes. Another important aspect is the understand-ability and readability by humans, mainly medical professionals and patients</p>
</div>
<div id="S3.SS4.p4" class="ltx_para">
<p id="S3.SS4.p4.1" class="ltx_p">Pre-trained and large language models such as BERT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib48" title="" class="ltx_ref">48</a>]</cite> and GPT <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib49" title="" class="ltx_ref">49</a>]</cite> are widely applied in medical text generation applications. Similarly, there are commonly used training text datasets such as MIMIC clinical notes data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib347" title="" class="ltx_ref">347</a>, <a href="#bib.bib348" title="" class="ltx_ref">348</a>]</cite>, i2b2 (Informatics for Integrating Biology and the Bedside) data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib349" title="" class="ltx_ref">349</a>]</cite>, and n2c2 (National NLP Clinical Challenges) data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib350" title="" class="ltx_ref">350</a>]</cite>. The i2b2 and n2c2 are well-known initiatives that provide datasets for NLP research in the biomedical and clinical domains.</p>
</div>
<div id="S3.SS4.p5" class="ltx_para">
<p id="S3.SS4.p5.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib341" title="" class="ltx_ref">341</a>]</cite> developed a generator based on LLaMA <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib52" title="" class="ltx_ref">52</a>]</cite> but particularly focusing on clinical note generation using case reports, discharge summaries, and clinical notes from multiple sources including MIMIC-III <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib347" title="" class="ltx_ref">347</a>]</cite>, MIMIC-IV <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib348" title="" class="ltx_ref">348</a>]</cite>, i2b2 <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib349" title="" class="ltx_ref">349</a>]</cite>, and CASI <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib351" title="" class="ltx_ref">351</a>]</cite>. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib341" title="" class="ltx_ref">341</a>]</cite> first generated 158K synthetic clinical notes from case reports from the PubMed central patients dataset <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib352" title="" class="ltx_ref">352</a>]</cite> using GPT-3.5. Then, incorporating the generated clinical notes, the proposed generator was fine-tuned based on LLaMA using discharge summaries from MIMIC-III database to capture the peculiarities in clinical notes such as capturing the relations between two medical terminologies, recognizing the reasoning process in the notes. The proposed model is fully open-source and trained on open-access datasets. The study evaluated the utility of the generator by conducting the NLP tasks using generated text (e.g., summarization, concept extraction, relationship extraction, name entity recognition, question answering) focusing on accuracy, relevancy, and completeness. They applied GPT-4 to design specific evaluation prompts and also validated by four real clinicians. The study demonstrated their model outperformed other open-source instruction fine-tuned LLMs such as Alpaca <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib353" title="" class="ltx_ref">353</a>]</cite> and medical domain-specific models such as MedAlpaca <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib354" title="" class="ltx_ref">354</a>]</cite> and ChatDoctor <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib355" title="" class="ltx_ref">355</a>]</cite>. The study demonstrated there is no significant difference between models trained on actual and synthetic clinical notes. However, in the training aspect, the proposed model is limited to dealing with only discharge summaries in one-turn instruction following tasks which means the user can ask one question or give one prompt then generator will provide an answer or generated text based on this single input.</p>
</div>
<div id="S3.SS4.p6" class="ltx_para">
<p id="S3.SS4.p6.1" class="ltx_p">In the evaluation, the study did not analyze whether and to what extent the model would hallucinate the outcome, which is one of the major concerns for the language model use in the clinical practice.</p>
</div>
<div id="S3.SS4.p7" class="ltx_para">
<p id="S3.SS4.p7.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib342" title="" class="ltx_ref">342</a>]</cite> utilized ChatGPT-4 and a Link Retrievel Plug-In to retrieve the medical information, integrate and analyze the collected information, and compile them into a human-understandable answer using three international guidelines on treating diabetic ketoacidosis. The study designed two prompts to 1) retrieve and integrate information focusing on diagnostic criteria, risk factors, signs and symptoms, investigations, and treatments of diabetic ketoacidosis and 2) generate comprehensive and evidence-based answers to the questions. The study aims to enhance the traceability and retrieval accuracy of ChatGPT-4 and claims their work maintained the accuracy, reliability, and quality of the generated answers. However, there are not sufficient systematic and quantitative experiments and evaluations being conducted in the study to support this conclusion. Furthermore, the truthfulness which assesses if the language models hallucinate the facts is not considered and evaluated sufficiently. This reflects many other use cases in using generative models in practical applications in medicine and healthcare domain.</p>
</div>
<div id="S3.SS4.p8" class="ltx_para">
<p id="S3.SS4.p8.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib343" title="" class="ltx_ref">343</a>]</cite> used a text dataset of 570 patient History and Present Illness (HPI) description from i2b2 and n2c2 challenges to train four different generative models - Character Recurrent Neural Network (CharRNN), Sub-sequence Generative Adversarial Network (SegGAN), GPT-2, and Conditional Transformer Language (CTRL). The four models were constructed based on different architectures and with different focuses. CharRNN is a character-level RNN which has shown its capability to tackle the out-of-vocabulary problem. Out-of-vocabulary challenge occurs when a word or phrase is not in the training data’s vocabulary or dictionary used by a language model. The language models learn to understand and generate text only based on the training data, which means they can only recognize and interpret words in their training process and unable to process or understand the out-of-vocabulary words directly. SegGAN follows the traditional GAN framework, adapted to deal with discrete text output and able to adversarially learn on both the entire sequences (i.e. sentences, paragraphs, documents) and sub-sequences (i.e., words, phrases). GPT-2, as a large transformer-based pre-trained language model, has the capability to generate diverse and massive synthetic text data, while CTRL is a parameter conditional transformer language model which is conditioned on control codes that govern the content and task-specific behavior. Control words are specific words or phrases that specify domain, subdomain, entities, relationships between entities to steer or guide the generation process of the model towards a desired outcome. The study fine-tuned the GPT-2 and CTRL models to generate synthetic clinical HPI corpus. The models’ performance was assessed using the Bilingual Evaluation Understudy (BLEU) metrics <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib356" title="" class="ltx_ref">356</a>]</cite>, to select the best model to create a synthetic corpus of 500 HPI subsections. BLEU examines the similarity of the machine translation and human translation in n-gram representation. An n-gram is a contiguous sequence of n items from a text, where these items can be words for text generation. After manually annotating clinical entities such as problems, treatments, and tests in real and synthetic text, NER models were developed on both real and synthetic datasets to conduct a utility evaluation. The study showed NER models that were trained on synthetic corpus achieved slightly higher performance than that of the real corpus. The NER models that were trained on the augmented corpus (real + synthetic) achieved better performance than that trained on the real corpus only. The study concluded the generated synthetic HPI text can be used to enhanced the development of clinical NLP models.</p>
</div>
<div id="S3.SS4.p9" class="ltx_para">
<p id="S3.SS4.p9.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib344" title="" class="ltx_ref">344</a>]</cite> proposed a model generating partially synthetic clinical text to protect against re-identification for individual patient and preserve the characteristics information from the population. The proposed model has two key processes - the artificial masking and the content substitution. Masking process utilized the tokenization function in BERT model to break down the sentence into tokens, added special tokens to indicate the beginning and end of sentences, then converted all the tokens into corresponding IDs in the pre-trained BERT vocabulary. Then, the tokenized data was fed into BERT for training. To mask the personally identifiable information in an efficient manner, the study embedded a blacklist and a white list to direct the masking. The blacklist and whitelist are customisable for different use cases. For instance, the blacklist can include words without meaning such as standard punctuation, stop words, while the whitelist contains sensitive attributes such as personally identifiable information. The model will avoid masking words in the blacklist, while the words in the whitelist will be masked with higher probabilities than words that are not in the lists. Masking sensitive words may result in obfuscation in some records. Therefore, the second step in the generative model is to replace defined sensitive information (i.e. words or phrases in the whitelist) of each document with text in similar documents in the dataset using Rapid Automatic Keyword Extraction <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib357" title="" class="ltx_ref">357</a>]</cite> and TextRank <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib358" title="" class="ltx_ref">358</a>]</cite> methods. The model was trained on 153K injury reports including textual injury descriptions with gender, age, and injury and illness labels and MIMIC III discharge summaries. The study evaluated data privacy by comparing the similarity between original and generated synthetic texts in word choice, word order, and word frequency. The satisfactory privacy-preserved synthetic text, which is well-obfuscated text, should be significantly different from the original text in word choice, word order, and word frequency. The privacy evaluation utilized BLEU scores, ranging from 0 to 1, measuring the similarity, with lower scores indicating better obfuscation. The data utility of the generated synthetic text was assessed by training a classification model on the original text and applying the trained model to the synthetic text to calculate the data utility loss. Data utility loss was measured by comparing the classification performances between the original and obfuscated text (partially synthetic text). Finally, readability of the generated text was evaluated by a machine evaluator which is a fine-tuned text classifier based on a BERT model to determine if the text input is a real clinical note or generated from a machine.</p>
</div>
<div id="S3.SS4.p10" class="ltx_para">
<p id="S3.SS4.p10.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib345" title="" class="ltx_ref">345</a>]</cite> proposed a differentially private and self-attention generative neural network model to generate synthetic medical texts. The generated text does not contain any identifying information from any individual if the input datasets were private. The study proposed a model as an autoregressive variation of the transformer model introduced by <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib359" title="" class="ltx_ref">359</a>]</cite>. In the proposed model, each of the decoder blocks is composed of a multi-headed masked self-attention layer and a simple feed-forward network. The autoregressive feature means when masked self-attention stops the model from seeing tokens that are at the right of the current position which allows the model to take into account the last token when generating the next token. Furthermore, the study applied a differentially private Gaussian mechanism technique by modifying the optimization algorithms in their generator training process. This means in each training iteration, the original weights are adjusted by adding noise values derived from the Gaussian mechanism. This process generates differentially private tokens, resulting in an input layer distinct from the original and ensuring a certain level of privacy guarantee. Similar to <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib341" title="" class="ltx_ref">341</a>]</cite> and <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib343" title="" class="ltx_ref">343</a>]</cite>, this study also used discharge summaries from MIMIC-III and i2b2 datasets. The model performance is evaluated by measuring the data utility on the word-level, document-level, and corpus-level. On the word-level, the study used BLEU score, Jaccard similarity, and G2 Test respectively to analyze the word co-occurrence or similarity, assessing whether the generated text has similar word distributions as the original text. On the document-level, data utility was analyzed by performing a classification task on original and synthetic data to determine whether the underlying document contains a certain disease or not. The study trained classifier on original text then tested on synthetic text, and trained on synthetic text then tested on original. The corpus-level utility, containing all documents, was assessed by an adversarial classification task. The study trained multiple classifiers on combined datasets of original and generated text and predicted whether a document is original or synthetic. The purpose of the classifier is to distinguish between original and synthetic text which is analogous to the Turing test, where a human evaluator predicts the data to be original or synthetic. Regarding data privacy, the study applied different privacy in the model and emphasized on the privacy-preserving feature that proposed model has, but the study did not provide sufficient privacy evaluation on generated text.</p>
</div>
<div id="S3.SS4.p11" class="ltx_para">
<p id="S3.SS4.p11.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib346" title="" class="ltx_ref">346</a>]</cite> developed two language models - LSTM-based model and GPT-2 (Transformer) based model.
The study sampled medical documents from the EHRs of 39 healthcare organizations in the Netherlands. Three domains of healthcare are represented within this sample: elderly care, mental care and disabled care. The models were evaluated in terms of the utility and privacy of the synthetic text. The utility was assessed by using synthetic text as a replacement of the real data to protect sensitive personal information and using synthetic text as a data augmentation approach to enrich the real medical documents. To evaluate data privacy, a user study was conducted with participants who were presented with the synthetic documents with high risks of privacy disclosure to judge if the documents contain sensitive information. The study observed the distribution of annotations in the training text and the diversity of the generated text have significant impact on the downstream task performance. The study reported that non-English language training faces additional challenges due to the lack of clear structure and conformity of the languages. Regarding data privacy and utility balance, the study argued that synthetic text does not need to be realistic for utility in downstream tasks so that privacy protection can be priotized. However, it is necessary to provide a mathematical privacy preservation in synthetic text generators such as using differential privacy methods.</p>
</div>
</section>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Discussion</h2>

<div id="S4.p1" class="ltx_para">
<p id="S4.p1.1" class="ltx_p">The survey of the different papers across different data types and modalities has yielded notable findings that we categorize into goal of synthesis, generation methods, and evaluation methods. For each insight or conclusion, we provide a non-exhaustive list of supporting references, previously discussed in section <a href="#S3" title="3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>. For further details, readers are directed to this section.</p>
</div>
<section id="S4.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Synthesis applications and purpose of synthesis</h3>

<div id="S4.SS1.p1" class="ltx_para">
<p id="S4.SS1.p1.1" class="ltx_p"><span id="S4.SS1.p1.1.1" class="ltx_text ltx_font_bold">Diversity of clinically valid synthesis applications</span>:
Conditional generative models have demonstrated impressive capabilities in synthesizing missing or corrupted modalities across a diverse range of applications. As depicted in Fig.<a href="#S2.F4" title="Fig. 4 ‣ 2.1 Synthesis applications and purpose of synthesis ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>, these applications vary depending on the data types involved. For example, in signals and images, both intra and inter modal translation between different modalities are common for various clinically valid reasons, such as synthesizing missing or corrupted MRI contrast when acquiring such images is not feasible <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib194" title="" class="ltx_ref">194</a>, <a href="#bib.bib203" title="" class="ltx_ref">203</a>, <a href="#bib.bib221" title="" class="ltx_ref">221</a>]</cite>, translating low-to-high density mammograms <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib158" title="" class="ltx_ref">158</a>]</cite> or DFM to FFDM mammograms <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib163" title="" class="ltx_ref">163</a>]</cite>, or B-mode to Elastography US <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib180" title="" class="ltx_ref">180</a>, <a href="#bib.bib181" title="" class="ltx_ref">181</a>]</cite>. In the context of EHR, the synthesis and the choice of generative model depends on the specific format and requirements of the EHR data. For instance, the capability to manage high dimensionality is pivotal when handling longitudinal <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref">71</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite> and aggregated <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib81" title="" class="ltx_ref">81</a>]</cite> EHR data, but its significance diminishes in the context of time-dependent <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite> and snapshot <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite> formats. Conversely, the preservation of temporal dependencies among clinical features remains crucial for both longitudinal and time-dependent formats. In medical text generation, the generated data serve multiple purposes beyond augmenting training data <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib344" title="" class="ltx_ref">344</a>, <a href="#bib.bib345" title="" class="ltx_ref">345</a>]</cite> . They can be also used for NLP enhancement, such as entity extraction <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib343" title="" class="ltx_ref">343</a>]</cite> and question-answering systems <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib342" title="" class="ltx_ref">342</a>]</cite> , as well as text replacement for privacy reasons <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib346" title="" class="ltx_ref">346</a>]</cite>, as depicted in Table <a href="#S3.T14" title="Table 14 ‣ 3.4 Medical text ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">14</span></a>.</p>
</div>
<div id="S4.SS1.p2" class="ltx_para">
<p id="S4.SS1.p2.1" class="ltx_p"><span id="S4.SS1.p2.1.1" class="ltx_text ltx_font_bold">Limited utilization of synthetic data beyond augmentation:</span> Despite its potential, synthetic data is underutilized beyond augmentation. This valuable resource, capable of generating diverse datasets varying in quality and anatomy, could serve as a comprehensive validation and testing set during model development <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib273" title="" class="ltx_ref">273</a>, <a href="#bib.bib280" title="" class="ltx_ref">280</a>, <a href="#bib.bib62" title="" class="ltx_ref">62</a>, <a href="#bib.bib323" title="" class="ltx_ref">323</a>, <a href="#bib.bib324" title="" class="ltx_ref">324</a>, <a href="#bib.bib360" title="" class="ltx_ref">360</a>]</cite>. It has the potential to uncover algorithm limitations early on, particularly for challenging cases that may not be accessible due to data sharing policies or rarity, facilitating smoother translation of algorithms into clinical practice.</p>
</div>
<div id="S4.SS1.p3" class="ltx_para">
<p id="S4.SS1.p3.1" class="ltx_p"><span id="S4.SS1.p3.1.1" class="ltx_text ltx_font_bold">Most generative approaches are designed to handle the lack of training data but thorough analysis of how this is achieved is missing:</span> While numerous methodologies have been proposed to address the scarcity of training data, the specific elements of the generated data that contribute to improved outcomes remain largely unexplored. This is particularly true when the use of synthetic data alone does not yield any enhancement in performance <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib361" title="" class="ltx_ref">361</a>, <a href="#bib.bib237" title="" class="ltx_ref">237</a>, <a href="#bib.bib257" title="" class="ltx_ref">257</a>, <a href="#bib.bib362" title="" class="ltx_ref">362</a>, <a href="#bib.bib363" title="" class="ltx_ref">363</a>, <a href="#bib.bib364" title="" class="ltx_ref">364</a>]</cite>. Only a handful of studies have indicated that the type of synthetic data incorporated into the training process can significantly impact results <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib263" title="" class="ltx_ref">263</a>, <a href="#bib.bib292" title="" class="ltx_ref">292</a>]</cite>. For instance, incorporating challenging cases such as dense mammograms <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib158" title="" class="ltx_ref">158</a>]</cite>, severe examples of lung nodules in X-ray images or overlapping benign and malignant nodules in breast ultrasound <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib178" title="" class="ltx_ref">178</a>]</cite>, or underrepresented groups <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib148" title="" class="ltx_ref">148</a>, <a href="#bib.bib147" title="" class="ltx_ref">147</a>]</cite>, has been shown to enhance performance. This underscores the potential benefits of active learning strategies, either during the data generation process or in selecting synthetic samples to be included in the training set.</p>
</div>
</section>
<section id="S4.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>Generation techniques </h3>

<div id="S4.SS2.p1" class="ltx_para">
<p id="S4.SS2.p1.1" class="ltx_p"><span id="S4.SS2.p1.1.1" class="ltx_text ltx_font_bold">The leap of textual conditioning opens future directions:</span>
Recent advancements in text-conditional generative models offer an innovative approach to integrating descriptive data into synthetic medical data generation. This can involve structured text input specifying medical conditions and demographic data or utilizing medical reports as a source of clinically relevant guidance for the model <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib117" title="" class="ltx_ref">117</a>, <a href="#bib.bib145" title="" class="ltx_ref">145</a>, <a href="#bib.bib157" title="" class="ltx_ref">157</a>, <a href="#bib.bib250" title="" class="ltx_ref">250</a>, <a href="#bib.bib295" title="" class="ltx_ref">295</a>, <a href="#bib.bib334" title="" class="ltx_ref">334</a>]</cite>. Readers are directed to the ”Text-guided” row in the tables for more information. Text-guided conditioning has been already established across various modalities, although some modalities are lagging behind and have not yet implemented this approach .</p>
</div>
<div id="S4.SS2.p2" class="ltx_para">
<p id="S4.SS2.p2.1" class="ltx_p"><span id="S4.SS2.p2.1.1" class="ltx_text ltx_font_bold">Conditional Models- Limited leverage of clinical knowledge, patient context, and text:</span>
A significant gap exists in the utilization of prior clinical knowledge, the availability of complimentary information accompanying the acquisition, as well as pathological processes, biomarkers and radiology reports to inform the generation process <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib252" title="" class="ltx_ref">252</a>, <a href="#bib.bib258" title="" class="ltx_ref">258</a>, <a href="#bib.bib279" title="" class="ltx_ref">279</a>, <a href="#bib.bib320" title="" class="ltx_ref">320</a>, <a href="#bib.bib68" title="" class="ltx_ref">68</a>, <a href="#bib.bib365" title="" class="ltx_ref">365</a>]</cite>. Most models fail to take advantage of these valuable resources, leading to potentially sub-optimal results. Similarly, conditioning based on subject characteristics patient and demographics is rarely utilized, except in isolated instances <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref">54</a>, <a href="#bib.bib76" title="" class="ltx_ref">76</a>, <a href="#bib.bib147" title="" class="ltx_ref">147</a>, <a href="#bib.bib157" title="" class="ltx_ref">157</a>]</cite>. This highlights the need to focus on more personalized synthesis tailored to specific subject characteristics and patient demographics, prioritizing fairness and inclusivity in medical data generation. Strong evidence suggests that conditional models leveraging such textual information, clinical knowledge, or other images/data inputs consistently produce higher quality and more realistic data compared to their non-conditional counterparts.</p>
</div>
<div id="S4.SS2.p3" class="ltx_para">
<p id="S4.SS2.p3.1" class="ltx_p"><span id="S4.SS2.p3.1.1" class="ltx_text ltx_font_bold">Limited tailoring of generative approaches to the specifics of medical data:</span> Most synthetic methods proposed for medical image generation still largely rely on loss functions and architectures initially designed for natural images, without considering the intricate aspects of medical image acquisition, physics and well defined statistics that can guide the generative process <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib250" title="" class="ltx_ref">250</a>, <a href="#bib.bib255" title="" class="ltx_ref">255</a>, <a href="#bib.bib279" title="" class="ltx_ref">279</a>, <a href="#bib.bib365" title="" class="ltx_ref">365</a>]</cite>. Moreover, a significant number of models are designed to generate small patches (especially 3D) and lead to the generation of artificial features that can lead to large errors in anomaly detection and other tasks <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib236" title="" class="ltx_ref">236</a>, <a href="#bib.bib249" title="" class="ltx_ref">249</a>, <a href="#bib.bib259" title="" class="ltx_ref">259</a>, <a href="#bib.bib272" title="" class="ltx_ref">272</a>]</cite>. Computational limitations also restrict recent transformer-based approaches to slice-based training <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib236" title="" class="ltx_ref">236</a>, <a href="#bib.bib251" title="" class="ltx_ref">251</a>, <a href="#bib.bib366" title="" class="ltx_ref">366</a>]</cite> and diffusion models to low-resolution image generation.
In contrast, EHR generative models draw from established deep learning architectures like CTGANs <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib89" title="" class="ltx_ref">89</a>]</cite> but address unique challenges such as synthesizing mixed-type data with static (e.g., demographics data) and sequential features (e.g., vital sign time-series) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib73" title="" class="ltx_ref">73</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>]</cite>, numerical features (e.g., blood pressure), and categorical features with many categories (e.g., medical codes) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib58" title="" class="ltx_ref">58</a>, <a href="#bib.bib71" title="" class="ltx_ref">71</a>]</cite>. Some of these models also consider modeling missing data patterns for informative analyses <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib76" title="" class="ltx_ref">76</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite>. See Table <a href="#S3.T3" title="Table 3 ‣ 3.1 EHR ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a> for more information. Similarly, generative models for physiological signals leverage techniques for time-series data, some customizing to signal specifics like morphology or inter-lead <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib105" title="" class="ltx_ref">105</a>, <a href="#bib.bib122" title="" class="ltx_ref">122</a>]</cite>.
Regarding medical text generation, the application of pre-trained models and large language models has recently become prominent in the field. These models are based on general foundation language models such as BERT and GPT and then adapted for medical applications through training or fine-tuning with specialized datasets, such as PubMed abstracts or clinical notes from the MIMIC database <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib342" title="" class="ltx_ref">342</a>, <a href="#bib.bib341" title="" class="ltx_ref">341</a>]</cite>. See Table <a href="#S3.T14" title="Table 14 ‣ 3.4 Medical text ‣ 3 Results ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">14</span></a> for more information. However, this approach has limitations in that these advanced medical text generators may not fully grasp the underlying principles or logic of medical knowledge.</p>
</div>
</section>
<section id="S4.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.3 </span>Evaluation</h3>

<div id="S4.SS3.p1" class="ltx_para">
<p id="S4.SS3.p1.1" class="ltx_p"><span id="S4.SS3.p1.1.1" class="ltx_text ltx_font_bold">Limited benchmarking, comparative studies and the use of small private datasets</span>
The absence of comparative studies between different methods and the lack of a well-defined benchmark for comparison leaves a gap in understanding the relative strengths and weaknesses of various synthetic data generation approaches and therefore highlights the need for a more systematic approach <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib237" title="" class="ltx_ref">237</a>, <a href="#bib.bib242" title="" class="ltx_ref">242</a>, <a href="#bib.bib281" title="" class="ltx_ref">281</a>, <a href="#bib.bib299" title="" class="ltx_ref">299</a>, <a href="#bib.bib307" title="" class="ltx_ref">307</a>, <a href="#bib.bib325" title="" class="ltx_ref">325</a>]</cite>. Moreover, the reliance on small, often private datasets for training and testing generative models raises concerns about the generalizability and usability of the generated data in clinical settings <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib255" title="" class="ltx_ref">255</a>, <a href="#bib.bib271" title="" class="ltx_ref">271</a>, <a href="#bib.bib362" title="" class="ltx_ref">362</a>, <a href="#bib.bib367" title="" class="ltx_ref">367</a>]</cite>. Most studies report both training and validation of the proposed synthesis approaches on significantly limited datasets - typically up to several hundreds of samples are used for training with a substantially smaller set for validation/testing. In addition, there is a severe lack of investigation of the effects of training data itself <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib239" title="" class="ltx_ref">239</a>]</cite>.</p>
</div>
<div id="S4.SS3.p2" class="ltx_para">
<p id="S4.SS3.p2.1" class="ltx_p"><span id="S4.SS3.p2.1.1" class="ltx_text ltx_font_bold">Limited evaluation frameworks</span>: Most approaches focus on quantitative evaluation using fidelity metrics that are not designed for the specific kind of medical data. For example, in the case of medical images, the fidelity metrics employed are tailored for diverse data types such as natural images, overlooking the distinctive characteristics and statistical properties inherent to typical medical imaging modalities <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib240" title="" class="ltx_ref">240</a>, <a href="#bib.bib67" title="" class="ltx_ref">67</a>, <a href="#bib.bib254" title="" class="ltx_ref">254</a>, <a href="#bib.bib286" title="" class="ltx_ref">286</a>, <a href="#bib.bib293" title="" class="ltx_ref">293</a>, <a href="#bib.bib316" title="" class="ltx_ref">316</a>]</cite>. Similarly, the evaluation metrics used for time-dependent EHR and physiological signals may overlook critical temporal features essential for an accurate evaluation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib93" title="" class="ltx_ref">93</a>, <a href="#bib.bib110" title="" class="ltx_ref">110</a>, <a href="#bib.bib111" title="" class="ltx_ref">111</a>, <a href="#bib.bib115" title="" class="ltx_ref">115</a>]</cite>. In medical text generation, there are no standards or well-accepted evaluation metrics to assess the fidelity of the generated data. Sole reliance on such generic metrics, without incorporating additional measures of clinical validation, utility, and human visual perception studies can lead to misleading conclusions about the true quality and suitability of synthetic medical data.</p>
</div>
<div id="S4.SS3.p3" class="ltx_para">
<p id="S4.SS3.p3.1" class="ltx_p">However, despite the crucial role of clinical validation in making sure that crucial anatomical details are maintained in the generated data, there is a notable lack in its proper implementation, hindering the translation of synthetic data into practical clinical settings. Studies that do perform clinical validation, typically perform it on a small subset of the generated data, signifying a need for evaluation on large internal and external validation datasets <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib256" title="" class="ltx_ref">256</a>, <a href="#bib.bib260" title="" class="ltx_ref">260</a>, <a href="#bib.bib262" title="" class="ltx_ref">262</a>, <a href="#bib.bib271" title="" class="ltx_ref">271</a>, <a href="#bib.bib272" title="" class="ltx_ref">272</a>, <a href="#bib.bib273" title="" class="ltx_ref">273</a>]</cite>.</p>
</div>
<div id="S4.SS3.p4" class="ltx_para">
<p id="S4.SS3.p4.1" class="ltx_p">Moreover, human assessments are still limited and face significant design shortcomings. Typically, these studies involve a limited number of experts, often ranging from 1 to 3, which raises concerns about the statistical significance of the results <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib65" title="" class="ltx_ref">65</a>, <a href="#bib.bib244" title="" class="ltx_ref">244</a>, <a href="#bib.bib246" title="" class="ltx_ref">246</a>, <a href="#bib.bib247" title="" class="ltx_ref">247</a>, <a href="#bib.bib325" title="" class="ltx_ref">325</a>, <a href="#bib.bib368" title="" class="ltx_ref">368</a>]</cite>. Furthermore, the design of these studies is often unclear, especially regarding the selection criteria of synthetic samples for evaluation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib236" title="" class="ltx_ref">236</a>, <a href="#bib.bib238" title="" class="ltx_ref">238</a>, <a href="#bib.bib239" title="" class="ltx_ref">239</a>, <a href="#bib.bib66" title="" class="ltx_ref">66</a>]</cite>. To enhance the effectiveness of these studies, they should be designed to gather reports from experts on indications that suggest certain samples are synthetic, providing the ability to correct for these artifacts in future approaches <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib238" title="" class="ltx_ref">238</a>]</cite>. In other instances, human visual assessment is reduced to a mere check of a few image samples for realism or a visual comparison of images from different methods, or merely inspecting visualizations such as PCA or t-SNE plots.</p>
</div>
<div id="S4.SS3.p5" class="ltx_para">
<p id="S4.SS3.p5.1" class="ltx_p">A more comprehensive evaluation of synthetic data should include measures of diversity, ensuring that the synthetic data encompasses the full range of anatomical and pathological variations encountered in clinical practice <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib255" title="" class="ltx_ref">255</a>, <a href="#bib.bib271" title="" class="ltx_ref">271</a>, <a href="#bib.bib362" title="" class="ltx_ref">362</a>, <a href="#bib.bib367" title="" class="ltx_ref">367</a>]</cite>. However, many current methods for generating synthetic data focus on reproducing specific image characteristics rather than generating a truly diverse set of instances. The lack of well-defined diversity metrics adds to the challenge, with researchers employing their own individual definitions <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib330" title="" class="ltx_ref">330</a>, <a href="#bib.bib61" title="" class="ltx_ref">61</a>, <a href="#bib.bib63" title="" class="ltx_ref">63</a>, <a href="#bib.bib332" title="" class="ltx_ref">332</a>]</cite>.</p>
</div>
<div id="S4.SS3.p6" class="ltx_para">
<p id="S4.SS3.p6.1" class="ltx_p"><span id="S4.SS3.p6.1.1" class="ltx_text ltx_font_bold">Limited evaluation of recent approaches based on diffusion and transformers:</span> Current research on diffusion models for data generation indicates they are more stable compared to GANs, with the ability to avoid mode collapse and vanishing gradients and add more control over the sampling process. However, a lot of diffusion models presented so far are yet to be properly tested and evaluated - utility studies are largely missing, as well as clinical validation or detailed qualitative and fidelity analysis of the generated data. Moreover, training is still performed on limited data that may lack adequate variability <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib240" title="" class="ltx_ref">240</a>, <a href="#bib.bib242" title="" class="ltx_ref">242</a>, <a href="#bib.bib246" title="" class="ltx_ref">246</a>, <a href="#bib.bib260" title="" class="ltx_ref">260</a>, <a href="#bib.bib259" title="" class="ltx_ref">259</a>, <a href="#bib.bib274" title="" class="ltx_ref">274</a>]</cite>.</p>
</div>
<div id="S4.SS3.p7" class="ltx_para">
<p id="S4.SS3.p7.1" class="ltx_p"><span id="S4.SS3.p7.1.1" class="ltx_text ltx_font_bold">Privacy disparities in synthetic medical data:</span>
The assessment of privacy protection in generated medical data, encompassing images, EHR, and signals, faces notable disparities in standardized methodologies. Despite assertions of privacy capabilities and the potential for widespread data sharing, the absence of well-defined evaluation methods for medical images, signals, and text compared to EHR raises concerns. EHR privacy assessments commonly involve testing synthetic data for vulnerabilities against potential adversary attacks and demonstrating resilience against various threats. In contrast, privacy discussions in medical images primarily focus on the memorization of training data, leaving questions about identity disclosure unexplored. The extent to which generators and diffusion models memorize or replicate data, and their impact on tracing back learned distributions to original training data, remains unclear. While EHR benefits from established evaluation practices, addressing the significant gaps in privacy protection evaluation for other data types is crucial for ensuring the responsible deployment of synthetic data in medical contexts.</p>
</div>
</section>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Recommendations</h2>

<div id="S5.p1" class="ltx_para">
<p id="S5.p1.1" class="ltx_p">Based on the insights collected from the surveyed papers, we propose the following recommendations to be taken into account for future research:</p>
</div>
<div id="S5.p2" class="ltx_para">
<p id="S5.p2.1" class="ltx_p"><span id="S5.p2.1.1" class="ltx_text ltx_font_bold">Generative approaches should be tailored to the medical data and downstream task or their intended use:</span> As specified earlier, generative approaches should be more tailored to the specifics of the medical data and also to the specific task at hand, as there are different requirements for each tasks <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib242" title="" class="ltx_ref">242</a>, <a href="#bib.bib257" title="" class="ltx_ref">257</a>, <a href="#bib.bib258" title="" class="ltx_ref">258</a>, <a href="#bib.bib254" title="" class="ltx_ref">254</a>, <a href="#bib.bib261" title="" class="ltx_ref">261</a>, <a href="#bib.bib366" title="" class="ltx_ref">366</a>]</cite>. For instance, requirements for segmentation/classification are substantial different compared to registration or denoising tasks .</p>
</div>
<div id="S5.p3" class="ltx_para">
<p id="S5.p3.1" class="ltx_p"><span id="S5.p3.1.1" class="ltx_text ltx_font_bold">Synthetic data should be utilized beyond data augmentation to train and evaluate medical AI software:</span> Synthetic data should not only serve as a tool for data augmentation and training downstream models but also for validation and testing purposes <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib298" title="" class="ltx_ref">298</a>, <a href="#bib.bib294" title="" class="ltx_ref">294</a>, <a href="#bib.bib289" title="" class="ltx_ref">289</a>, <a href="#bib.bib312" title="" class="ltx_ref">312</a>, <a href="#bib.bib327" title="" class="ltx_ref">327</a>]</cite>. Moreover, it can be used for the development of explainability mechanisms to improve not only the confidence of the proposed analysis models, but also the confidence of clinicians in utilizing automated methods in clinical practice.</p>
</div>
<div id="S5.p4" class="ltx_para">
<p id="S5.p4.1" class="ltx_p"><span id="S5.p4.1.1" class="ltx_text ltx_font_bold">Prioritizing fairness and inclusivity in medical data generation:</span>
Due to its scarcity, there is a need to focus on more personalized synthesis tailored to specific subject characteristics and patient demographics, prioritizing fairness and inclusivity in medical data generation.</p>
</div>
<div id="S5.p5" class="ltx_para">
<p id="S5.p5.1" class="ltx_p"><span id="S5.p5.1.1" class="ltx_text ltx_font_bold">Adapting language models to medical tasks should be further approached with more care:</span> A thorough analysis is needed on the interpretability capabilities of diffusion models when it comes to the understanding and translation of radiological reports to images.</p>
</div>
<div id="S5.p6" class="ltx_para">
<p id="S5.p6.1" class="ltx_p"><span id="S5.p6.1.1" class="ltx_text ltx_font_bold">A call for benchmarking and comparative studies to promote openness and collaboration:
</span>
The use of established and well defined benchmarks and open-source datasets would not only accelerate progress in the field but also provide valuable insights into the strengths and weaknesses of various techniques, guiding researchers towards the optimal tools for specific clinical applications.</p>
</div>
<div id="S5.p7" class="ltx_para">
<p id="S5.p7.1" class="ltx_p"><span id="S5.p7.1.1" class="ltx_text ltx_font_bold">A need for more comprehensive evaluation:</span> There is a need for holistic assessment covering more specific fidelity metrics, proper clinical validation and human assessment, and well-defined diversity metrics. Moreover, there is a need for well established privacy evaluation practices, especially for data types where there is a significant gap. This is crucial for ensuring the responsible deployment of synthetic data in medical contexts.</p>
</div>
</section>
<section id="S6" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Conclusion</h2>

<div id="S6.p1" class="ltx_para">
<p id="S6.p1.1" class="ltx_p">The advent of AI and ML has a great potential to revolutionize healthcare, with synthetic data generation emerging as a promising solution to the challenges of data availability and diversity. Despite the growing interest in generative models for medical data synthesis, existing reviews often have limited scope, focusing on specific modalities or data types, or overlooking certain types of generative models beyond basic data augmentation techniques. This survey paper aims to fulfill this need by providing a comprehensive examination of the use of generative models, such as GANs, VAEs, LLMs, and DMs, in synthesizing medical data across various modalities, including imaging, text, time-series, and tabular data. Through a meticulous search strategy involving databases such as Scopus, PubMed, and arXiv, this survey paper identifies and analyzes 249 relevant studies published from January 2021 to November 2023, excluding review and perspective papers. The survey aimed to provide a holistic understanding of the applications of these generative models in generating synthetic medical data,shedding light on the current practices, potential, and challenges in this field, delving into three key aspects: the purpose of synthesis, generation techniques, and evaluation methods.</p>
</div>
<div id="S6.p2" class="ltx_para">
<p id="S6.p2.1" class="ltx_p">Our findings reveal a diversity of clinically valid synthesis applications, yet also highlight a limited exploration of synthetic data’s potential and its utilization beyond augmentation. Most generative approaches are designed to handle the lack of training data, but a thorough analysis of how this is achieved is often missing.</p>
</div>
<div id="S6.p3" class="ltx_para">
<p id="S6.p3.1" class="ltx_p">In terms of generation techniques, the advent of textual conditioning opens up exciting future directions. However, there is limited leverage of clinical knowledge, patient context, and text in conditional models, and generative approaches are often not tailored to the specifics of medical data.</p>
</div>
<div id="S6.p4" class="ltx_para">
<p id="S6.p4.1" class="ltx_p">Evaluation methods also present challenges, with limited benchmarking, comparative studies, and the use of small private datasets. Evaluation frameworks are often limited, and recent approaches based on diffusion and transformers are not thoroughly evaluated. Privacy disparities in synthetic medical data also pose a significant concern.</p>
</div>
<div id="S6.p5" class="ltx_para">
<p id="S6.p5.1" class="ltx_p">In essence, while synthetic data generation holds immense potential in transforming healthcare research and practice, there are notable gaps and challenges that need to be addressed. There is a need for more personalized synthesis approaches, standardized evaluation methodologies, and in-depth evaluation approaches relevant to clinical applications. The field would also benefit from more benchmarking and comparative studies to promote openness and collaboration.</p>
</div>
<div id="S6.p6" class="ltx_para">
<p id="S6.p6.1" class="ltx_p">This survey paper serves as a valuable resource for researchers and practitioners interested in leveraging generative models for synthesizing medical data. By providing a holistic understanding of the field, we hope to spur further research and innovation in this critical area of healthcare AI and ML. The insights gained from this survey underscore the importance of continued exploration and innovation in synthetic medical data generation, with the ultimate goal of advancing patient care and clinical practice.</p>
</div>
</section>
<section id="S7" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">7 </span>Acknowledgements</h2>

<div id="S7.p1" class="ltx_para">
<p id="S7.p1.1" class="ltx_p">This work was funded by the European Union under the Horizon Europe grant 101095435. Views and opinions expressed are however those of the author(s) only and do not necessarily reflect those of the European Union. Neither the European Union nor the granting authority can be held responsible for them.</p>
</div>
<figure id="S7.1" class="ltx_figure"><img src="/html/2407.00116/assets/eu.png" id="S7.1.g1" class="ltx_graphics ltx_img_landscape" width="299" height="63" alt="[Uncaptioned image]">
</figure>
</section>
<section id="S8" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">8 </span>Statements and Declarations</h2>

<div id="S8.p1" class="ltx_para">
<p id="S8.p1.1" class="ltx_p">The authors have no competing interests to declare that are relevant to the content of this article.</p>
</div>
</section>
<section id="S9" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">9 </span>CRediT author statement</h2>

<div id="S9.p1" class="ltx_para">
<p id="S9.p1.1" class="ltx_p"><span id="S9.p1.1.1" class="ltx_text ltx_font_bold">Mahmoud Ibrahim:</span> Conceptualization, Methodology, Investigation, Data Curation, Writing, Visualization, Project Administration.<span id="S9.p1.1.2" class="ltx_text ltx_font_bold">Yasmina Al Khalil:</span> Methodology, Investigation, Writing, Visualization. <span id="S9.p1.1.3" class="ltx_text ltx_font_bold">Sina Amirrajab</span>: Methodology, Investigation, Writing. <span id="S9.p1.1.4" class="ltx_text ltx_font_bold">Chang Sun:</span> Methodology, Investigation, Writing, Funding Acquisition. <span id="S9.p1.1.5" class="ltx_text ltx_font_bold">Bart Elen:</span> Writing - Review and Editing, Funding Acquisition, Supervision. <span id="S9.p1.1.6" class="ltx_text ltx_font_bold">Gokhan Ertaylan:</span> Funding Acquisition, Supervision. <span id="S9.p1.1.7" class="ltx_text ltx_font_bold">Michel Dumontier:</span> Funding Acquisition, Supervision.</p>
</div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">
Rajotte JF, Bergen R, Buckeridge DL, El Emam K, Ng R, Strome E.

</span>
<span class="ltx_bibblock">Synthetic Data as an Enabler for Machine Learning Applications in Medicine.

</span>
<span class="ltx_bibblock">iScience. 2022;25(11):105331.

</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">
Liu Y, Dwivedi G, Boussaid F, Bennamoun M.

</span>
<span class="ltx_bibblock">3D Brain and Heart Volume Generative Models: A Survey.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">
Murtaza H, Ahmed M, Khan NF, Murtaza G, Zafar S, Bano A.

</span>
<span class="ltx_bibblock">Synthetic Data Generation: State of the Art in Health Care Domain.

</span>
<span class="ltx_bibblock">Computer Science Review. 2023;48:100546.

</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">
Kazerouni A, Aghdam EK, Heidari M, Azad R, Fayyaz M, Hacihaliloglu I, et al.

</span>
<span class="ltx_bibblock">Diffusion Models in Medical Imaging: A Comprehensive Survey.

</span>
<span class="ltx_bibblock">Medical Image Analysis. 2023;88:102846.

</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">
Koo H, Kim TE.

</span>
<span class="ltx_bibblock">A Comprehensive Survey on Generative Diffusion Models for Structured Data.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">
Berger L, Haberbusch M, Moscato F.

</span>
<span class="ltx_bibblock">Generative Adversarial Networks in Electrocardiogram Synthesis: Recent Developments and Challenges.

</span>
<span class="ltx_bibblock">Artificial Intelligence in Medicine. 2023;143:102632.

</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">
Kebaili A, Lapuyade-Lahorgue J, Ruan S.

</span>
<span class="ltx_bibblock">Deep Learning Approaches for Data Augmentation in Medical Imaging: A Review.

</span>
<span class="ltx_bibblock">Journal of Imaging. 2023;9(4):81.

</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">
Yuan H, Zhou S, Yu S.

</span>
<span class="ltx_bibblock">EHRDiff: Exploring Realistic EHR Synthesis with Diffusion Models.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">
Chen Y, Yang XH, Wei Z, Heidari AA, Zheng N, Li Z, et al.

</span>
<span class="ltx_bibblock">Generative Adversarial Networks in Medical Image Augmentation: A Review.

</span>
<span class="ltx_bibblock">Computers in Biology and Medicine. 2022;144:105382.

</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">
Page MJ, McKenzie JE, Bossuyt PM, Boutron I, Hoffmann TC, Mulrow CD, et al.

</span>
<span class="ltx_bibblock">PRISMA 2020 statement: systematic review and meta-analysis protocols (PRISMA-P) 2021 extension.

</span>
<span class="ltx_bibblock">BMJ. 2021;374:n71.

</span>
</li>
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock">
Goodfellow I, Pouget-Abadie J, Mirza M, Xu B, Warde-Farley D, Ozair S, et al.

</span>
<span class="ltx_bibblock">Generative adversarial nets.

</span>
<span class="ltx_bibblock">Advances in neural information processing systems. 2014;27.

</span>
</li>
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock">
Kingma DP, Welling M.

</span>
<span class="ltx_bibblock">Auto-Encoding Variational Bayes.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock">
Sohl-Dickstein J, Weiss EA, Maheswaranathan N, Ganguli S.

</span>
<span class="ltx_bibblock">Deep Unsupervised Learning using Nonequilibrium Thermodynamics.

</span>
<span class="ltx_bibblock">arxiv. 2015.

</span>
</li>
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock">
Jordan MI, Ghahramani Z, Jaakkola TS, Saul LK.

</span>
<span class="ltx_bibblock">An Introduction to Variational Methods for Graphical Models.

</span>
<span class="ltx_bibblock">In: Learning in Graphical Models. Springer; 1999. p. 156-71.

</span>
</li>
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock">
Ganguly A, Earp SWF.

</span>
<span class="ltx_bibblock">An Introduction to Variational Inference.

</span>
<span class="ltx_bibblock">arxiv. 2021.

</span>
</li>
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock">
Kotelnikov A, Baranchuk D, Rubachev I, Babenko A.

</span>
<span class="ltx_bibblock">TabDDPM: Modelling Tabular Data with Diffusion Models.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock">
Tian M, Chen B, Guo A, Jiang S, Zhang AR.

</span>
<span class="ltx_bibblock">Fast and Reliable Generation of EHR Time Series via Diffusion Models.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock">
Ceritli, T , Ghosheh, G O , Chauhan, V K , et al.

</span>
<span class="ltx_bibblock">Synthesizing Mixed-type Electronic Health Records Using Diffusion Models.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock">
Hoogeboom E, Nielsen D, Jaini P, Forré P, Welling M.

</span>
<span class="ltx_bibblock">Argmax Flows and Multinomial Diffusion: Learning Categorical Distributions.

</span>
<span class="ltx_bibblock">arxiv. 2021.

</span>
</li>
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock">
Donahue C, McAuley J, Puckette M.

</span>
<span class="ltx_bibblock">Synthesizing audio with GANs.

</span>
<span class="ltx_bibblock">arXiv. 2018.

</span>
</li>
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock">
Raoof I, Gupta MK.

</span>
<span class="ltx_bibblock">A Study of Semi Supervised Based Approaches for Motor Imagery Signal Generation.

</span>
<span class="ltx_bibblock">In: 2021 Third International Conference on Inventive Research in Computing Applications (ICIRCA). IEEE; 2021. p. 1312-6.

</span>
</li>
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[22]</span>
<span class="ltx_bibblock">
Radford A, Metz L, Chintala S.

</span>
<span class="ltx_bibblock">Unsupervised representation learning with deep convolutional generative adversarial networks.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:151106434. 2015.

</span>
</li>
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[23]</span>
<span class="ltx_bibblock">
Arjovsky M, Chintala S, Bottou L.

</span>
<span class="ltx_bibblock">Wasserstein generative adversarial networks.

</span>
<span class="ltx_bibblock">In: International conference on machine learning. PMLR; 2017. p. 214-23.

</span>
</li>
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[24]</span>
<span class="ltx_bibblock">
Mao X, Li Q, Xie H, Lau RY, Wang Z, Paul Smolley S.

</span>
<span class="ltx_bibblock">Least squares generative adversarial networks.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE international conference on computer vision; 2017. p. 2794-802.

</span>
</li>
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[25]</span>
<span class="ltx_bibblock">
Gulrajani I, Ahmed F, Arjovsky M, Dumoulin V, Courville AC.

</span>
<span class="ltx_bibblock">Improved training of wasserstein gans.

</span>
<span class="ltx_bibblock">Advances in neural information processing systems. 2017;30.

</span>
</li>
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[26]</span>
<span class="ltx_bibblock">
Karras T, Aila T, Laine S, Lehtinen J.

</span>
<span class="ltx_bibblock">Progressive growing of gans for improved quality, stability, and variation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:171010196. 2017.

</span>
</li>
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[27]</span>
<span class="ltx_bibblock">
Alqahtani H, Kavakli-Thorne M, Kumar G.

</span>
<span class="ltx_bibblock">Applications of generative adversarial networks (gans): An updated review.

</span>
<span class="ltx_bibblock">Archives of Computational Methods in Engineering. 2021;28:525-52.

</span>
</li>
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[28]</span>
<span class="ltx_bibblock">
Mirza M, Osindero S.

</span>
<span class="ltx_bibblock">Conditional generative adversarial nets.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:14111784. 2014.

</span>
</li>
<li id="bib.bib29" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[29]</span>
<span class="ltx_bibblock">
Isola P, Zhu JY, Zhou T, Efros AA.

</span>
<span class="ltx_bibblock">Image-to-Image Translation with Conditional Adversarial Networks.

</span>
<span class="ltx_bibblock">arxiv. 2018.

</span>
</li>
<li id="bib.bib30" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[30]</span>
<span class="ltx_bibblock">
Wang TC, Liu MY, Zhu JY, Tao A, Kautz J, Catanzaro B.

</span>
<span class="ltx_bibblock">High-Resolution Image Synthesis and Semantic Manipulation with Conditional GANs.

</span>
<span class="ltx_bibblock">arxiv. 2018.

</span>
</li>
<li id="bib.bib31" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[31]</span>
<span class="ltx_bibblock">
Zhu JY, Park T, Isola P, Efros AA.

</span>
<span class="ltx_bibblock">Unpaired Image-to-Image Translation Using Cycle-Consistent Adversarial Networks.

</span>
<span class="ltx_bibblock">arxiv. 2020.

</span>
</li>
<li id="bib.bib32" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[32]</span>
<span class="ltx_bibblock">
Wang T, Lei Y, Fu Y, Wynne JF, Curran WJ, Liu T, et al.

</span>
<span class="ltx_bibblock">A review on medical imaging synthesis using deep learning and its clinical applications.

</span>
<span class="ltx_bibblock">Journal of applied clinical medical physics. 2021;22(1):11-36.

</span>
</li>
<li id="bib.bib33" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[33]</span>
<span class="ltx_bibblock">
Karras T, Laine S, Aila T.

</span>
<span class="ltx_bibblock">A style-based generator architecture for generative adversarial networks.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE/CVF conference on computer vision and pattern recognition; 2019. p. 4401-10.

</span>
</li>
<li id="bib.bib34" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[34]</span>
<span class="ltx_bibblock">
Park T, Liu MY, Wang TC, Zhu JY.

</span>
<span class="ltx_bibblock">Semantic image synthesis with spatially-adaptive normalization.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE/CVF conference on computer vision and pattern recognition; 2019. p. 2337-46.

</span>
</li>
<li id="bib.bib35" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[35]</span>
<span class="ltx_bibblock">
Esser P, Rombach R, Ommer B.

</span>
<span class="ltx_bibblock">Taming transformers for high-resolution image synthesis.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE/CVF conference on computer vision and pattern recognition; 2021. p. 12873-83.

</span>
</li>
<li id="bib.bib36" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[36]</span>
<span class="ltx_bibblock">
Oord Avd, Vinyals O, Kavukcuoglu K.

</span>
<span class="ltx_bibblock">Neural discrete representation learning.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:171100937. 2017.

</span>
</li>
<li id="bib.bib37" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[37]</span>
<span class="ltx_bibblock">
Ho J, Jain A, Abbeel P.

</span>
<span class="ltx_bibblock">Denoising Diffusion Probabilistic Models.

</span>
<span class="ltx_bibblock">arxiv. 2020.

</span>
</li>
<li id="bib.bib38" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[38]</span>
<span class="ltx_bibblock">
Ronneberger O, Fischer P, Brox T.

</span>
<span class="ltx_bibblock">U-Net: Convolutional Networks for Biomedical Image Segmentation.

</span>
<span class="ltx_bibblock">arxiv. 2015.

</span>
</li>
<li id="bib.bib39" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[39]</span>
<span class="ltx_bibblock">
Song J, Meng C, Ermon S.

</span>
<span class="ltx_bibblock">Denoising Diffusion Implicit Models.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib40" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[40]</span>
<span class="ltx_bibblock">
Rombach R, Blattmann A, Lorenz D, Esser P, Ommer B.

</span>
<span class="ltx_bibblock">High-Resolution Image Synthesis with Latent Diffusion Models.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib41" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[41]</span>
<span class="ltx_bibblock">
Radford A, Kim JW, Hallacy C, Ramesh A, Goh G, Agarwal S, et al.

</span>
<span class="ltx_bibblock">Learning Transferable Visual Models From Natural Language Supervision.

</span>
<span class="ltx_bibblock">arxiv. 2021.

</span>
</li>
<li id="bib.bib42" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[42]</span>
<span class="ltx_bibblock">
Roberts A, Chung HW, Levskaya A, Mishra G, Bradbury J, Andor D, et al.

</span>
<span class="ltx_bibblock">Scaling Up Models and Data with <span id="bib.bib42.3.1" class="ltx_text ltx_markedasmath ltx_font_typewriter">t5x</span> and <span id="bib.bib42.4.2" class="ltx_text ltx_markedasmath ltx_font_typewriter">seqio</span>.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib43" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[43]</span>
<span class="ltx_bibblock">
Rasmy L, Xiang Y, Xie Z, Tao C, Zhi D.

</span>
<span class="ltx_bibblock">Med-BERT: pre-trained contextualized embeddings on large-scale structured electronic health records for disease prediction.

</span>
<span class="ltx_bibblock">arxiv. 2020.

</span>
</li>
<li id="bib.bib44" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[44]</span>
<span class="ltx_bibblock">
Sennrich R, Haddow B, Birch A.

</span>
<span class="ltx_bibblock">Neural Machine Translation of Rare Words with Subword Units.

</span>
<span class="ltx_bibblock">arxiv. 2016.

</span>
</li>
<li id="bib.bib45" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[45]</span>
<span class="ltx_bibblock">
Liu F, Shareghi E, Meng Z, Basaldella M, Collier N.

</span>
<span class="ltx_bibblock">Self-Alignment Pretraining for Biomedical Entity Representations.

</span>
<span class="ltx_bibblock">arxiv. 2021.

</span>
</li>
<li id="bib.bib46" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[46]</span>
<span class="ltx_bibblock">
Zhao WX, Zhou K, Li J, Tang T, Wang X, Hou Y, et al.

</span>
<span class="ltx_bibblock">A survey of large language models.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230318223. 2023.

</span>
</li>
<li id="bib.bib47" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[47]</span>
<span class="ltx_bibblock">
Lee SH.

</span>
<span class="ltx_bibblock">Natural language generation for electronic health records.

</span>
<span class="ltx_bibblock">NPJ digital medicine. 2018;1(1):63.

</span>
</li>
<li id="bib.bib48" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[48]</span>
<span class="ltx_bibblock">
Devlin J, Chang MW, Lee K, Toutanova K.

</span>
<span class="ltx_bibblock">Bert: Pre-training of deep bidirectional transformers for language understanding.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:181004805. 2018.

</span>
</li>
<li id="bib.bib49" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[49]</span>
<span class="ltx_bibblock">
Radford A, Wu J, Child R, Luan D, Amodei D, Sutskever I, et al.

</span>
<span class="ltx_bibblock">Language models are unsupervised multitask learners.

</span>
<span class="ltx_bibblock">OpenAI blog. 2019;1(8):9.

</span>
</li>
<li id="bib.bib50" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[50]</span>
<span class="ltx_bibblock">
Lee J, Yoon W, Kim S, Kim D, Kim S, So CH, et al.

</span>
<span class="ltx_bibblock">BioBERT: a pre-trained biomedical language representation model for biomedical text mining.

</span>
<span class="ltx_bibblock">Bioinformatics. 2020;36(4):1234-40.

</span>
</li>
<li id="bib.bib51" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[51]</span>
<span class="ltx_bibblock">
Alsentzer E, Murphy JR, Boag W, Weng WH, Jin D, Naumann T, et al.

</span>
<span class="ltx_bibblock">Publicly available clinical BERT embeddings.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:190403323. 2019.

</span>
</li>
<li id="bib.bib52" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[52]</span>
<span class="ltx_bibblock">
Touvron H, Lavril T, Izacard G, Martinet X, Lachaux MA, Lacroix T, et al.

</span>
<span class="ltx_bibblock">Llama: Open and efficient foundation language models.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230213971. 2023.

</span>
</li>
<li id="bib.bib53" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[53]</span>
<span class="ltx_bibblock">
Chung HW, Hou L, Longpre S, Zoph B, Tay Y, Fedus W, et al.

</span>
<span class="ltx_bibblock">Scaling instruction-finetuned language models.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:221011416. 2022.

</span>
</li>
<li id="bib.bib54" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[54]</span>
<span class="ltx_bibblock">
Yale A, Dash S, Dutta R, Guyon I, Pavao A, Bennett KP.

</span>
<span class="ltx_bibblock">Generation and Evaluation of Privacy Preserving Synthetic Health Data.

</span>
<span class="ltx_bibblock">Neurocomputing. 2020;416:244-55.

</span>
</li>
<li id="bib.bib55" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[55]</span>
<span class="ltx_bibblock">
Dwork C.

</span>
<span class="ltx_bibblock">Differential Privacy: A Survey of Results.

</span>
<span class="ltx_bibblock">In: Agrawal M, Du D, Duan Z, Li A, editors. Theory and Applications of Models of Computation. vol. 4978. Springer Berlin Heidelberg; 2008. p. 1-19.

</span>
</li>
<li id="bib.bib56" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[56]</span>
<span class="ltx_bibblock">
Alaa AM, van Breugel B, Saveliev E, van der Schaar M.

</span>
<span class="ltx_bibblock">How Faithful is your Synthetic Data? Sample-level Metrics for Evaluating and Auditing Generative Models.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib57" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[57]</span>
<span class="ltx_bibblock">
Kuo NIH, Jorm L, Barbieri S.

</span>
<span class="ltx_bibblock">Synthetic Health-related Longitudinal Data with Mixed-type Variables Generated Using Diffusion Models.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib58" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[58]</span>
<span class="ltx_bibblock">
Lu C, Reddy CK, Wang P, Nie D, Ning Y.

</span>
<span class="ltx_bibblock">Multi-Label Clinical Time-Series Generation via Conditional GAN.

</span>
<span class="ltx_bibblock">IEEE Transactions on Knowledge and Data Engineering. 2023:1-13.

</span>
</li>
<li id="bib.bib59" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[59]</span>
<span class="ltx_bibblock">
Alzantot, M , Garcia, L , Srivastava, M .

</span>
<span class="ltx_bibblock">PhysioGAN: Training High Fidelity Generative Model for Physiological Sensor Readings.

</span>
<span class="ltx_bibblock">arXiv. 2022.

</span>
</li>
<li id="bib.bib60" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[60]</span>
<span class="ltx_bibblock">
Pinaya WHL, Tudosiu PD, Dafflon J, da Costa PF, Fernandez V, Nachev P, et al.

</span>
<span class="ltx_bibblock">Brain Imaging Generation with Latent Diffusion Models.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib61" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[61]</span>
<span class="ltx_bibblock">
Khader F, Müller-Franzes G, Tayebi Arasteh S, Han T, Haarburger C, Schulze-Hagen M, et al.

</span>
<span class="ltx_bibblock">Denoising Diffusion Probabilistic Models for 3D Medical Image Generation.

</span>
<span class="ltx_bibblock">Scientific Reports. 2023;13(1):7303.

</span>
</li>
<li id="bib.bib62" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[62]</span>
<span class="ltx_bibblock">
Dombrowski M, Reynaud H, Müller JP, Baugh M, Kainz B.

</span>
<span class="ltx_bibblock">Trade-offs in Fine-tuned Diffusion Models Between Accuracy and Interpretability.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib63" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[63]</span>
<span class="ltx_bibblock">
Pan S, Wang T, Qiu RLJ, Axente M, Chang CW, Peng J, et al.

</span>
<span class="ltx_bibblock">2D Medical Image Synthesis Using Transformer-Based Denoising Diffusion Probabilistic Model.

</span>
<span class="ltx_bibblock">Physics in Medicine and Biology. 2023;68(10).

</span>
</li>
<li id="bib.bib64" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[64]</span>
<span class="ltx_bibblock">
Sun L, Chen J, Xu Y, Gong M, Yu K, Batmanghelich K.

</span>
<span class="ltx_bibblock">Hierarchical Amortized GAN for 3D High Resolution Medical Image Synthesis.

</span>
<span class="ltx_bibblock">IEEE journal of biomedical and health informatics. 2022;26(8):3966-75.

</span>
</li>
<li id="bib.bib65" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[65]</span>
<span class="ltx_bibblock">
Pan S, Wang T, Qiu RL, Axente M, Chang CW, Peng J, et al.

</span>
<span class="ltx_bibblock">2D medical image synthesis using transformer-based denoising diffusion probabilistic model.

</span>
<span class="ltx_bibblock">Physics in Medicine &amp; Biology. 2023;68(10):105004.

</span>
</li>
<li id="bib.bib66" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[66]</span>
<span class="ltx_bibblock">
Lee KW, Chin RKY.

</span>
<span class="ltx_bibblock">Diverse covid-19 CT image-to-image translation with stacked residual dropout.

</span>
<span class="ltx_bibblock">Bioengineering. 2022;9(11):698.

</span>
</li>
<li id="bib.bib67" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[67]</span>
<span class="ltx_bibblock">
Toda R, Teramoto A, Kondo M, Imaizumi K, Saito K, Fujita H.

</span>
<span class="ltx_bibblock">Lung cancer CT image generation from a free-form sketch using style-based pix2pix for data augmentation.

</span>
<span class="ltx_bibblock">Scientific reports. 2022;12(1):12867.

</span>
</li>
<li id="bib.bib68" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[68]</span>
<span class="ltx_bibblock">
Abdelmotaal H, Sharaf M, Soliman W, Wasfi E, Kedwany SM.

</span>
<span class="ltx_bibblock">Bridging the resources gap: deep learning for fluorescein angiography and optical coherence tomography macular thickness map image translation.

</span>
<span class="ltx_bibblock">BMC ophthalmology. 2022;22(1):355.

</span>
</li>
<li id="bib.bib69" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[69]</span>
<span class="ltx_bibblock">
Tajmirriahi M, Kafieh R, Amini Z, Lakshminarayanan V.

</span>
<span class="ltx_bibblock">A dual-discriminator Fourier acquisitive GAN for generating retinal optical coherence tomography images.

</span>
<span class="ltx_bibblock">IEEE Transactions on Instrumentation and Measurement. 2022;71:1-8.

</span>
</li>
<li id="bib.bib70" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[70]</span>
<span class="ltx_bibblock">
Zhang Z, Yan C, Lasko TA, Sun J, Malin BA.

</span>
<span class="ltx_bibblock">SynTEG: A Framework for Temporal Structured Electronic Health Data Simulation.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association : JAMIA. 2021;28(3):596-604.

</span>
</li>
<li id="bib.bib71" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[71]</span>
<span class="ltx_bibblock">
Biswal S, Ghosh S, Duke J, Malin B, Stewart W, Xiao C, et al.

</span>
<span class="ltx_bibblock">EVA: Generating Longitudinal Electronic Health Records Using Conditional Variational Autoencoders.

</span>
<span class="ltx_bibblock">In: Proceedings of Machine Learning Research. vol. 149; 2021. p. 260-82.

</span>
</li>
<li id="bib.bib72" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[72]</span>
<span class="ltx_bibblock">
Zhong Y, Cui S, Wang J, Wang X, Yin Z, Wang Y, et al.

</span>
<span class="ltx_bibblock">MedDiffusion: Boosting Health Risk Prediction via Diffusion-based Data Augmentation.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib73" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[73]</span>
<span class="ltx_bibblock">
Wang Z, Sun J.

</span>
<span class="ltx_bibblock">PromptEHR: Conditional Electronic Healthcare Records Generation with Prompt Learning.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib74" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[74]</span>
<span class="ltx_bibblock">
Theodorou B, Xiao C, Sun J.

</span>
<span class="ltx_bibblock">Synthesize High-Dimensional Longitudinal Electronic Health Records via Hierarchical Autoregressive Language Model.

</span>
<span class="ltx_bibblock">Nature Communications. 2023;14(1):5305.

</span>
</li>
<li id="bib.bib75" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[75]</span>
<span class="ltx_bibblock">
Dash S, Yale A, Guyon I, Bennett KP.

</span>
<span class="ltx_bibblock">Medical Time-Series Data Generation Using Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 12299 LNAI; 2020. p. 382-91.

</span>
</li>
<li id="bib.bib76" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[76]</span>
<span class="ltx_bibblock">
Bing S, Dittadi A, Bauer S, Schwab P.

</span>
<span class="ltx_bibblock">Conditional Generation of Medical Time Series for Extrapolation to Underrepresented Populations.

</span>
<span class="ltx_bibblock">PLOS digital health. 2022;1(7):e0000074.

</span>
</li>
<li id="bib.bib77" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[77]</span>
<span class="ltx_bibblock">
Yoon J, Mizrahi M, Ghalaty NF, Jarvinen T, Ravi AS, Brune P, et al.

</span>
<span class="ltx_bibblock">EHR-Safe: Generating High-Fidelity and Privacy-Preserving Synthetic Electronic Health Records.

</span>
<span class="ltx_bibblock">NPJ digital medicine. 2023;6(1):141.

</span>
</li>
<li id="bib.bib78" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[78]</span>
<span class="ltx_bibblock">
Sun S, Wang F, Rashidian S, Kurc T, Abell-Hart K, Hajagos J, et al.

</span>
<span class="ltx_bibblock">Generating Longitudinal Synthetic EHR Data with Recurrent Autoencoders and Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 12921 LNCS; 2021. p. 153-65.

</span>
</li>
<li id="bib.bib79" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[79]</span>
<span class="ltx_bibblock">
Li J, Cairns BJ, Li J, Zhu T.

</span>
<span class="ltx_bibblock">Generating Synthetic Mixed-Type Longitudinal Electronic Health Records for Artificial Intelligent Applications.

</span>
<span class="ltx_bibblock">NPJ digital medicine. 2023;6(1):98.

</span>
</li>
<li id="bib.bib80" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[80]</span>
<span class="ltx_bibblock">
van Hoorn R, Bakkes T, Tokoutsi Z, de Jong Y, Bouwman RA, Pechenizkiy M.

</span>
<span class="ltx_bibblock">Generating Privacy-Preserving Longitudinal Synthetic Data.

</span>
<span class="ltx_bibblock">In: NeurIPS 2023 Workshop on Synthetic Data Generation with Generative AI; 2023. p. .

</span>
<span class="ltx_bibblock">Available from: <a target="_blank" href="https://openreview.net/forum?id=Xr13v66xxT" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://openreview.net/forum?id=Xr13v66xxT</a>.

</span>
</li>
<li id="bib.bib81" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[81]</span>
<span class="ltx_bibblock">
Torfi A, Fox EA, Reddy CK.

</span>
<span class="ltx_bibblock">Differentially Private Synthetic Medical Data Generation Using Convolutional GANs.

</span>
<span class="ltx_bibblock">Information Sciences. 2022;586:485-500.

</span>
</li>
<li id="bib.bib82" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[82]</span>
<span class="ltx_bibblock">
Yan C, Zhang Z, Nyemba S, Malin BA.

</span>
<span class="ltx_bibblock">Generating Electronic Health Records with Multiple Data Types and Constraints.

</span>
<span class="ltx_bibblock">In: AMIA Annual Symposium Proceedings. arXiv; 2020. p. 1335–1344.

</span>
</li>
<li id="bib.bib83" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[83]</span>
<span class="ltx_bibblock">
Naseer AA, Walker B, Landon C, Ambrosy A, Fudim M, Wysham N, et al.

</span>
<span class="ltx_bibblock">ScoEHR: Generating Synthetic Electronic Health Records Using Continuous-time Diffusion Models.

</span>
<span class="ltx_bibblock">In: Proceedings of Machine Learning Research 219:1–22, 2023; 2023. p. 489-508.

</span>
</li>
<li id="bib.bib84" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[84]</span>
<span class="ltx_bibblock">
He H, Zhao S, Xi Y, Ho JC.

</span>
<span class="ltx_bibblock">MedDiff: Generating Electronic Health Records Using Accelerated Denoising Diffusion Model.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib85" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[85]</span>
<span class="ltx_bibblock">
Yoon J, Drumright LN, van der Schaar M.

</span>
<span class="ltx_bibblock">Anonymization Through Data Synthesis Using Generative Adversarial Networks (ADS-GAN).

</span>
<span class="ltx_bibblock">IEEE journal of biomedical and health informatics. 2020;24(8):2378-88.

</span>
</li>
<li id="bib.bib86" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[86]</span>
<span class="ltx_bibblock">
Venugopal R, Shafqat N, Venugopal I, Tillbury BMJ, Stafford HD, Bourazeri A.

</span>
<span class="ltx_bibblock">Privacy Preserving Generative Adversarial Networks to Model Electronic Health Records.

</span>
<span class="ltx_bibblock">Neural networks : the official journal of the International Neural Network Society. 2022;153:339-48.

</span>
</li>
<li id="bib.bib87" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[87]</span>
<span class="ltx_bibblock">
Fang ML, Dhami DS, Kersting K.

</span>
<span class="ltx_bibblock">DP-CTGAN: Differentially Private Medical Data Generation Using CTGANs.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 13263 LNAI; 2022. p. 178-88.

</span>
</li>
<li id="bib.bib88" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[88]</span>
<span class="ltx_bibblock">
Rodriguez-Almeida AJ, Fabelo H, Ortega S, Deniz A, Balea-Fernandez FJ, Quevedo E, et al.

</span>
<span class="ltx_bibblock">Synthetic Patient Data Generation and Evaluation in Disease Prediction Using Small and Imbalanced Datasets.

</span>
<span class="ltx_bibblock">IEEE journal of biomedical and health informatics. 2023;27(6):2670-80.

</span>
</li>
<li id="bib.bib89" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[89]</span>
<span class="ltx_bibblock">
Perets, O , Rappoport, N .

</span>
<span class="ltx_bibblock">Ensemble Synthetic EHR Generation for Increasing Subpopulation Model’s Performance.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib90" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[90]</span>
<span class="ltx_bibblock">
Bietsch D, Stahlbock R, Voß S.

</span>
<span class="ltx_bibblock">Synthetic Data as a Proxy for Real-World Electronic Health Records in the Patient Length of Stay Prediction.

</span>
<span class="ltx_bibblock">Sustainability (Switzerland). 2023;15(18).

</span>
</li>
<li id="bib.bib91" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[91]</span>
<span class="ltx_bibblock">
Zhang Z, Yan C, Mesa DA, Sun J, Malin BA.

</span>
<span class="ltx_bibblock">Ensuring Electronic Medical Record Simulation through Better Training, Modeling, and Evaluation.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association : JAMIA. 2019;27(1):99-108.

</span>
</li>
<li id="bib.bib92" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[92]</span>
<span class="ltx_bibblock">
Yoon J, Jarrett D, van der Schaar M.

</span>
<span class="ltx_bibblock">Time-Series Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">In: Advances in Neural Information Processing Systems. vol. 32. Curran Associates, Inc.; 2019. p. .

</span>
</li>
<li id="bib.bib93" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[93]</span>
<span class="ltx_bibblock">
Adib, E , Afghah, F , Prevost, J J .

</span>
<span class="ltx_bibblock">Synthetic ECG Signal Generation Using Generative Neural Networks.

</span>
<span class="ltx_bibblock">arXiv. 2021.

</span>
</li>
<li id="bib.bib94" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[94]</span>
<span class="ltx_bibblock">
Choi E, Biswal S, Malin B, Duke J, Stewart WF, Sun J.

</span>
<span class="ltx_bibblock">Generating Multi-label Discrete Patient Records Using Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">In: Proceedings of the 2nd Machine Learning for Healthcare Conference. PMLR; 2017. p. 286-305.

</span>
</li>
<li id="bib.bib95" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[95]</span>
<span class="ltx_bibblock">
Baowaly MK, Lin CC, Liu CL, Chen KT.

</span>
<span class="ltx_bibblock">Synthesizing Electronic Health Records Using Improved Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association : JAMIA. 2019;26(3):228-41.

</span>
</li>
<li id="bib.bib96" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[96]</span>
<span class="ltx_bibblock">
Hjelm RD, Jacob AP, Che T, Trischler A, Cho K, Bengio Y.

</span>
<span class="ltx_bibblock">Boundary-Seeking Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">arxiv. 2018.

</span>
</li>
<li id="bib.bib97" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[97]</span>
<span class="ltx_bibblock">
Goncalves A, Ray P, Soper B, Stevens J, Coyle L, Sales AP.

</span>
<span class="ltx_bibblock">Generation and Evaluation of Synthetic Patient Data.

</span>
<span class="ltx_bibblock">BMC medical research methodology. 2020;20(1):108.

</span>
</li>
<li id="bib.bib98" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[98]</span>
<span class="ltx_bibblock">
Ho J, Salimans T.

</span>
<span class="ltx_bibblock">Classifier-Free Diffusion Guidance.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib99" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[99]</span>
<span class="ltx_bibblock">
Mironov I.

</span>
<span class="ltx_bibblock">Rényi Differential Privacy.

</span>
<span class="ltx_bibblock">In: 2017 IEEE 30th Computer Security Foundations Symposium (CSF). IEEE; 2017. p. .

</span>
</li>
<li id="bib.bib100" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[100]</span>
<span class="ltx_bibblock">
Xu L, Skoularidou M, Cuesta-Infante A, Veeramachaneni K.

</span>
<span class="ltx_bibblock">Modeling Tabular data using Conditional GAN.

</span>
<span class="ltx_bibblock">arxiv. 2019.

</span>
</li>
<li id="bib.bib101" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[101]</span>
<span class="ltx_bibblock">
Bhanot, K , Baldini, I , Wei, D , et al.

</span>
<span class="ltx_bibblock">Downstream Fairness Caveats with Synthetic Healthcare Data.

</span>
<span class="ltx_bibblock">arxiv. 2022.

</span>
</li>
<li id="bib.bib102" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[102]</span>
<span class="ltx_bibblock">
Weldon, J , Ward, T , Brophy, E .

</span>
<span class="ltx_bibblock">Generation of Synthetic Electronic Health Records Using a Federated GAN.

</span>
<span class="ltx_bibblock">arxiv. 2021.

</span>
</li>
<li id="bib.bib103" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[103]</span>
<span class="ltx_bibblock">
Nik AHZ, Riegler MA, Halvorsen P, Storås AM.

</span>
<span class="ltx_bibblock">Generation of Synthetic Tabular Healthcare Data Using Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 13833 LNCS; 2023. p. 434-46.

</span>
</li>
<li id="bib.bib104" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[104]</span>
<span class="ltx_bibblock">
Yoo H, Moon J, Kim JH, Joo HJ.

</span>
<span class="ltx_bibblock">Design and Technical Validation to Generate a Synthetic 12-Lead Electrocardiogram Dataset to Promote Artificial Intelligence Research.

</span>
<span class="ltx_bibblock">Health Information Science and Systems. 2023;11(1).

</span>
</li>
<li id="bib.bib105" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[105]</span>
<span class="ltx_bibblock">
Banerjee R, Ghose A.

</span>
<span class="ltx_bibblock">Synthesis of Realistic ECG Waveforms Using a Composite Generative Adversarial Network for Classification of Atrial Fibrillation.

</span>
<span class="ltx_bibblock">In: European Signal Processing Conference. vol. 2021-August; 2021. p. 1145-9.

</span>
</li>
<li id="bib.bib106" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[106]</span>
<span class="ltx_bibblock">
Yang Z, Li Y, Zhou G.

</span>
<span class="ltx_bibblock">TS-GAN: Time-series GAN for Sensor-based Health Data Augmentation.

</span>
<span class="ltx_bibblock">ACM Transactions on Computing for Healthcare. 2023;4(2).

</span>
</li>
<li id="bib.bib107" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[107]</span>
<span class="ltx_bibblock">
Zhang, Y -H , Babaeizadeh, S .

</span>
<span class="ltx_bibblock">Synthesis of Standard 12-Lead Electrocardiograms Using Two Dimensional Generative Adversarial Network.

</span>
<span class="ltx_bibblock">Journal of Electrocardiology. 2021.

</span>
</li>
<li id="bib.bib108" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[108]</span>
<span class="ltx_bibblock">
Thambawita V, Isaksen JL, Hicks SA, Ghouse J, Ahlberg G, Linneberg A, et al.

</span>
<span class="ltx_bibblock">DeepFake Electrocardiograms Using Generative Adversarial Networks Are the Beginning of the End for Privacy Issues in Medicine.

</span>
<span class="ltx_bibblock">Scientific Reports. 2021;11(1).

</span>
</li>
<li id="bib.bib109" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[109]</span>
<span class="ltx_bibblock">
Li X, Metsis V, Wang H, Ngu AHH.

</span>
<span class="ltx_bibblock">TTS-GAN: A Transformer-Based Time-Series Generative Adversarial Network.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 13263 LNAI; 2022. p. 133-43.

</span>
</li>
<li id="bib.bib110" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[110]</span>
<span class="ltx_bibblock">
Adib, E , Fernandez, A , Afghah, F , et al.

</span>
<span class="ltx_bibblock">Synthetic ECG Signal Generation Using Probabilistic Diffusion Models.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib111" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[111]</span>
<span class="ltx_bibblock">
Zama MH, Schwenker F.

</span>
<span class="ltx_bibblock">ECG Synthesis via Diffusion-Based State Space Augmented Transformer.

</span>
<span class="ltx_bibblock">Sensors. 2023;23(19).

</span>
</li>
<li id="bib.bib112" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[112]</span>
<span class="ltx_bibblock">
Neifar, N , Ben-Hamadou, A , Mdhaffar, A , et al.

</span>
<span class="ltx_bibblock">DiffECG: A Generalized Probabilistic Diffusion Model for ECG Signals Synthesis.

</span>
<span class="ltx_bibblock">arxiv. 2023.

</span>
</li>
<li id="bib.bib113" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[113]</span>
<span class="ltx_bibblock">
Xia Y, Wang W, Wang K.

</span>
<span class="ltx_bibblock">ECG Signal Generation Based on Conditional Generative Models.

</span>
<span class="ltx_bibblock">Biomedical Signal Processing and Control. 2023;82:104587.

</span>
</li>
<li id="bib.bib114" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[114]</span>
<span class="ltx_bibblock">
Adib, E , Afghah, F , Prevost, J J . Arrhythmia Classification Using CGAN-Augmented ECG Signals; 2022.

</span>
</li>
<li id="bib.bib115" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[115]</span>
<span class="ltx_bibblock">
Nankani D, Dutta Baruah R.

</span>
<span class="ltx_bibblock">Improved Diagnostic Performance of Arrhythmia Classification Using Conditional GAN Augmented Heartbeats.

</span>
<span class="ltx_bibblock">In: Intelligent Systems Reference Library. vol. 217; 2022. p. 275-304.

</span>
</li>
<li id="bib.bib116" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[116]</span>
<span class="ltx_bibblock">
Sang Y, Beetz M, Grau V.

</span>
<span class="ltx_bibblock">Generation of 12-Lead Electrocardiogram with Subject-Specific, Image-Derived Characteristics Using a Conditional Variational Autoencoder.

</span>
<span class="ltx_bibblock">In: Proceedings - International Symposium on Biomedical Imaging. vol. 2022-March; 2022. .

</span>
</li>
<li id="bib.bib117" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[117]</span>
<span class="ltx_bibblock">
Chung, H , Kim, J , Kwon, J -M , et al.. TEXT-TO-ECG: 12-LEAD ELECTROCARDIOGRAM SYNTHESIS CONDITIONED ON CLINICAL TEXT REPORTS; 2023.

</span>
</li>
<li id="bib.bib118" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[118]</span>
<span class="ltx_bibblock">
Bagga, M , Jeon, H , Issokson, A . ECGNet: A Generative Adversarial Network (GAN) Approach to the Synthesis of 12-Lead ECG Signals from Single Lead Inputs; 2023.

</span>
</li>
<li id="bib.bib119" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[119]</span>
<span class="ltx_bibblock">
Jo, Y -Y , Choi, Y S , Jang, J -H , et al.

</span>
<span class="ltx_bibblock">ECGT2T: Towards Synthesizing Twelve-Lead Electrocardiograms from Two Asynchronous Leads.

</span>
<span class="ltx_bibblock">arXiv. 2021.

</span>
</li>
<li id="bib.bib120" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[120]</span>
<span class="ltx_bibblock">
Shome, D , Sarkar, P , Etemad, A .

</span>
<span class="ltx_bibblock">Region-Disentangled Diffusion Model for High-Fidelity PPG-to-ECG Translation.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib121" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[121]</span>
<span class="ltx_bibblock">
Dissanayake T, Fernando T, Denman S, Sridharan S, Fookes C.

</span>
<span class="ltx_bibblock">Generalized Generative Deep Learning Models for Biosignal Synthesis and Modality Transfer.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics. 2023;27(2):968-79.

</span>
</li>
<li id="bib.bib122" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[122]</span>
<span class="ltx_bibblock">
Agarwal E.

</span>
<span class="ltx_bibblock">CardiacGen: A Hierarchical Deep Generative Model for Cardiac Signals.

</span>
<span class="ltx_bibblock">In: Machine Learning for Health; 2022. .

</span>
</li>
<li id="bib.bib123" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[123]</span>
<span class="ltx_bibblock">
Neifar N, Mdhaffar A, Ben-Hamadou A, Jmaiel M, Freisleben B.

</span>
<span class="ltx_bibblock">Disentangling Temporal and Amplitude Variations in ECG Synthesis Using Anchored GANs.

</span>
<span class="ltx_bibblock">In: Proceedings of the ACM Symposium on Applied Computing; 2022. p. 645-52.

</span>
</li>
<li id="bib.bib124" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[124]</span>
<span class="ltx_bibblock">
Alcaraz JML, Strodthoff N.

</span>
<span class="ltx_bibblock">Diffusion-Based Conditional ECG Generation with Structured State Space Models.

</span>
<span class="ltx_bibblock">Computers in Biology and Medicine. 2023;163.

</span>
</li>
<li id="bib.bib125" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[125]</span>
<span class="ltx_bibblock">
Mogren O. C-RNN-GAN: Continuous recurrent neural networks with adversarial training; 2016.

</span>
</li>
<li id="bib.bib126" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[126]</span>
<span class="ltx_bibblock">
Gu A, Goel K, Ré C. Efficiently Modeling Long Sequences with Structured State Spaces; 2022.

</span>
</li>
<li id="bib.bib127" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[127]</span>
<span class="ltx_bibblock">
Alcaraz JML, Strodthoff N. Diffusion-based Time Series Imputation and Forecasting with Structured State Space Models; 2023.

</span>
</li>
<li id="bib.bib128" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[128]</span>
<span class="ltx_bibblock">
Aristimunha B, de Camargo RY, Chevallier S, Thomas AG, Lucena O, Cardoso MJ, et al.

</span>
<span class="ltx_bibblock">Synthetic Sleep EEG Signal Generation Using Latent Diffusion Models.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib129" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[129]</span>
<span class="ltx_bibblock">
Li H, Yu S, Principe J.

</span>
<span class="ltx_bibblock">Causal Recurrent Variational Autoencoder for Medical Time Series Generation.

</span>
<span class="ltx_bibblock">In: Proceedings of the 37th AAAI Conference on Artificial Intelligence, AAAI 2023. vol. 37; 2023. p. 8562-70.

</span>
</li>
<li id="bib.bib130" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[130]</span>
<span class="ltx_bibblock">
Granger CWJ.

</span>
<span class="ltx_bibblock">Investigating Causal Relations by Econometric Models and Cross-spectral Methods.

</span>
<span class="ltx_bibblock">Econometrica. 1969;37(3):424-38.

</span>
<span class="ltx_bibblock">Available from: <a target="_blank" href="http://www.jstor.org/stable/1912791" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://www.jstor.org/stable/1912791</a>.

</span>
</li>
<li id="bib.bib131" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[131]</span>
<span class="ltx_bibblock">
Lee JS, Hsiang J. Patent Claim Generation by Fine-Tuning OpenAI GPT-2; 2019.

</span>
</li>
<li id="bib.bib132" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[132]</span>
<span class="ltx_bibblock">
Nishi Y, Suge A, Takahashi H.

</span>
<span class="ltx_bibblock">Text analysis on the stock market in the automotive industry through fake news generated by GPT-2.

</span>
<span class="ltx_bibblock">Proceedings of the Artificial Intelligence of and for Business. 2019.

</span>
</li>
<li id="bib.bib133" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[133]</span>
<span class="ltx_bibblock">
Bird JJ, Pritchard M, Fratini A, Ekart A, Faria DR.

</span>
<span class="ltx_bibblock">Synthetic Biological Signals Machine-Generated by GPT-2 Improve the Classification of EEG and EMG through Data Augmentation.

</span>
<span class="ltx_bibblock">IEEE Robotics and Automation Letters. 2021;6(2):3498-504.

</span>
</li>
<li id="bib.bib134" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[134]</span>
<span class="ltx_bibblock">
Kalashami MP, Pedram MM, Sadr H.

</span>
<span class="ltx_bibblock">EEG Feature Extraction and Data Augmentation in Emotion Recognition.

</span>
<span class="ltx_bibblock">Computational Intelligence and Neuroscience. 2022;2022.

</span>
</li>
<li id="bib.bib135" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[135]</span>
<span class="ltx_bibblock">
Vetter, J , Macke, J H , Gao, R .

</span>
<span class="ltx_bibblock">Generating Realistic Neurophysiological Time Series with Denoising Diffusion Probabilistic Models.

</span>
<span class="ltx_bibblock">bioRxiv. 2023.

</span>
</li>
<li id="bib.bib136" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[136]</span>
<span class="ltx_bibblock">
Shu, K , Zhao, Y , Wu, L , et al.

</span>
<span class="ltx_bibblock">Data Augmentation for Seizure Prediction with Generative Diffusion Model.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib137" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[137]</span>
<span class="ltx_bibblock">
Ganti B, Chaitanya G, Balamurugan RS, Nagaraj N, Balasubramanian K, Pati S.

</span>
<span class="ltx_bibblock">Time-Series Generative Adversarial Network Approach of Deep Learning Improves Seizure Detection From the Human Thalamic SEEG.

</span>
<span class="ltx_bibblock">Frontiers in Neurology. 2022;13.

</span>
</li>
<li id="bib.bib138" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[138]</span>
<span class="ltx_bibblock">
Pérez E, Ventura S.

</span>
<span class="ltx_bibblock">Progressive Growing of Generative Adversarial Networks for Improving Data Augmentation and Skin Cancer Diagnosis.

</span>
<span class="ltx_bibblock">Artificial intelligence in medicine. 2023;141:102556.

</span>
</li>
<li id="bib.bib139" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[139]</span>
<span class="ltx_bibblock">
Mohanty, A ,  Sutherland, A ,  Bezbradica, M , et al.

</span>
<span class="ltx_bibblock">High Fidelity Synthetic Face Generation for Rosacea Skin Condition from Limited Data.

</span>
<span class="ltx_bibblock">Research Square. 2023.

</span>
</li>
<li id="bib.bib140" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[140]</span>
<span class="ltx_bibblock">
Teodoro AAM, Silva DH, Rosa RL, Saadi M, Wuttisittikulkij L, Mumtaz RA, et al.

</span>
<span class="ltx_bibblock">A Skin Cancer Classification Approach Using GAN and RoI-Based Attention Mechanism.

</span>
<span class="ltx_bibblock">Journal of Signal Processing Systems. 2023;95(2-3):211-24.

</span>
</li>
<li id="bib.bib141" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[141]</span>
<span class="ltx_bibblock">
Zhao C, Shuai R, Ma L, Liu W, Hu D, Wu M.

</span>
<span class="ltx_bibblock">Dermoscopy Image Classification Based on StyleGAN and DenseNet201.

</span>
<span class="ltx_bibblock">IEEE Access. 2021;9:8659-79.

</span>
</li>
<li id="bib.bib142" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[142]</span>
<span class="ltx_bibblock">
Mikołajczyk A, Majchrowska S, Carrasco Limeros S.

</span>
<span class="ltx_bibblock">The (de)Biasing Effect of GAN-Based Augmentation Methods on Skin Lesion Images.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 13438 LNCS; 2022. p. 437-47.

</span>
</li>
<li id="bib.bib143" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[143]</span>
<span class="ltx_bibblock">
Limeros, S C ,  Majchrowska, S ,  Zoubi, M K , et al.

</span>
<span class="ltx_bibblock">GAN-based Generative Modelling for Dermatological Applications - Comparative Study.

</span>
<span class="ltx_bibblock">arXiv. 2022.

</span>
</li>
<li id="bib.bib144" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[144]</span>
<span class="ltx_bibblock">
Kaur R, Gholamhosseini H, Sinha R.

</span>
<span class="ltx_bibblock">Synthetic Images Generation Using Conditional Generative Adversarial Network for Skin Cancer Classification.

</span>
<span class="ltx_bibblock">In: IEEE Region 10 Annual International Conference, Proceedings/TENCON. vol. 2021-December; 2021. p. 381-6.

</span>
</li>
<li id="bib.bib145" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[145]</span>
<span class="ltx_bibblock">
Sagers, L W ,  Diao, J A ,  Melas-Kyriazi, L , et al.

</span>
<span class="ltx_bibblock">AUGMENTING MEDICAL IMAGE CLASSIFIERS WITH SYNTHETIC DATA FROM LATENT DIFFUSION MODELS.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib146" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[146]</span>
<span class="ltx_bibblock">
Akrout, M ,  Gyepesi, B ,  Holló, P , et al.

</span>
<span class="ltx_bibblock">Diffusion-Based Data Augmentation for Skin Disease Classification: Impact Across Original Medical Datasets to Fully Synthetic Images.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib147" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[147]</span>
<span class="ltx_bibblock">
Sagers, L W ,  Groh, M ,  Adamson, A S , et al.

</span>
<span class="ltx_bibblock">Improving Dermatology Classifiers across Populations Using Images Generated by Large Diffusion Models.

</span>
<span class="ltx_bibblock">arXiv. 2022.

</span>
</li>
<li id="bib.bib148" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[148]</span>
<span class="ltx_bibblock">
Wang H, Qi Q, Sun W, Li X, Dong B, Yao C.

</span>
<span class="ltx_bibblock">Classification of Skin Lesions with Generative Adversarial Networks and Improved MobileNetV2.

</span>
<span class="ltx_bibblock">International Journal of Imaging Systems and Technology. 2023;33(5):1561-76.

</span>
</li>
<li id="bib.bib149" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[149]</span>
<span class="ltx_bibblock">
TB; F. The validity and practicality of sun-reactive skin types I through VI. U.S. National Library of Medicine;.

</span>
<span class="ltx_bibblock">Available from: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/3377516/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://pubmed.ncbi.nlm.nih.gov/3377516/</a>.

</span>
</li>
<li id="bib.bib150" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[150]</span>
<span class="ltx_bibblock">
Xie Q, Dai Z, Hovy E, Luong MT, Le QV. Unsupervised Data Augmentation for Consistency Training; 2020.

</span>
</li>
<li id="bib.bib151" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[151]</span>
<span class="ltx_bibblock">
Zhong Z, Zheng L, Kang G, Li S, Yang Y. Random Erasing Data Augmentation; 2017.

</span>
</li>
<li id="bib.bib152" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[152]</span>
<span class="ltx_bibblock">
Zhao S, Liu Z, Lin J, Zhu JY, Han S. Differentiable Augmentation for Data-Efficient GAN Training; 2020.

</span>
</li>
<li id="bib.bib153" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[153]</span>
<span class="ltx_bibblock">
Ruiz N, Li Y, Jampani V, Pritch Y, Rubinstein M, Aberman K. DreamBooth: Fine Tuning Text-to-Image Diffusion Models for Subject-Driven Generation; 2023.

</span>
</li>
<li id="bib.bib154" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[154]</span>
<span class="ltx_bibblock">
Szafranowska Z, Osuala R, Breier B, Kushibar K, Lekadir K, Diaz O.

</span>
<span class="ltx_bibblock">Sharing Generative Models Instead of Private Data: A Simulation Study on Mammography Patch Classification.

</span>
<span class="ltx_bibblock">In: Proceedings of SPIE - The International Society for Optical Engineering. vol. 12286; 2022. .

</span>
</li>
<li id="bib.bib155" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[155]</span>
<span class="ltx_bibblock">
Joseph AJ, Dwivedi P, Joseph J, Francis S, Pournami PN, Jayaraj PB, et al.

</span>
<span class="ltx_bibblock">Prior-Guided Generative Adversarial Network for Mammogram Synthesis.

</span>
<span class="ltx_bibblock">Biomedical Signal Processing and Control. 2024;87.

</span>
</li>
<li id="bib.bib156" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[156]</span>
<span class="ltx_bibblock">
Oyelade ON, Ezugwu AE, Almutairi MS, Saha AK, Abualigah L, Chiroma H.

</span>
<span class="ltx_bibblock">A Generative Adversarial Network for Synthetization of Regions of Interest Based on Digital Mammograms.

</span>
<span class="ltx_bibblock">Scientific Reports. 2022;12(1).

</span>
</li>
<li id="bib.bib157" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[157]</span>
<span class="ltx_bibblock">
Montoya-del Angel R, Sam-Millan K, Vilanova JC, Martí R. MAM-E: Mammographic Synthetic Image Generation with Diffusion Models; 2023.

</span>
</li>
<li id="bib.bib158" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[158]</span>
<span class="ltx_bibblock">
Garrucho L, Kushibar K, Osuala R, Diaz O, Catanese A, del Riego J, et al.

</span>
<span class="ltx_bibblock">High-Resolution Synthesis of High-Density Breast Mammograms: Application to Improved Fairness in Deep Learning Based Mass Detection.

</span>
<span class="ltx_bibblock">Frontiers in Oncology. 2023;12.

</span>
</li>
<li id="bib.bib159" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[159]</span>
<span class="ltx_bibblock">
Yamazaki A, Ishida T.

</span>
<span class="ltx_bibblock">Two-View Mammogram Synthesis from Single-View Data Using Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">Applied Sciences (Switzerland). 2022;12(23).

</span>
</li>
<li id="bib.bib160" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[160]</span>
<span class="ltx_bibblock">
Aizaz Z, Khare K, Khursheed A, Tirmizi A.

</span>
<span class="ltx_bibblock">Pix2Pix Generative Adversarial Networks (GAN) for Breast Cancer Detection.

</span>
<span class="ltx_bibblock">In: 2022 5th International Conference on Multimedia, Signal Processing and Communication Technologies, IMPACT 2022; 2022. .

</span>
</li>
<li id="bib.bib161" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[161]</span>
<span class="ltx_bibblock">
Lee J, Nishikawa RM.

</span>
<span class="ltx_bibblock">Identifying Women with Mammographically- Occult Breast Cancer Leveraging GAN-Simulated Mammograms.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging. 2022;41(1):225-36.

</span>
</li>
<li id="bib.bib162" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[162]</span>
<span class="ltx_bibblock">
Guan Q, Chen Y, Wei Z, Heidari AA, Hu H, Yang XH, et al.

</span>
<span class="ltx_bibblock">Medical Image Augmentation for Lesion Detection Using a Texture-Constrained Multichannel Progressive GAN.

</span>
<span class="ltx_bibblock">Computers in Biology and Medicine. 2022;145.

</span>
</li>
<li id="bib.bib163" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[163]</span>
<span class="ltx_bibblock">
Zhou Y, Wei J, Wu D, Zhang Y.

</span>
<span class="ltx_bibblock">Generating Full-Field Digital Mammogram From Digitized Screen-Film Mammogram for Breast Cancer Screening With High-Resolution Generative Adversarial Network.

</span>
<span class="ltx_bibblock">Frontiers in Oncology. 2022;12.

</span>
</li>
<li id="bib.bib164" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[164]</span>
<span class="ltx_bibblock">
Shen T, Hao K, Gou C, Wang FY.

</span>
<span class="ltx_bibblock">Mass Image Synthesis in Mammogram with Contextual Information Based on GANs.

</span>
<span class="ltx_bibblock">Computer Methods and Programs in Biomedicine. 2021;202.

</span>
</li>
<li id="bib.bib165" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[165]</span>
<span class="ltx_bibblock">
Giess CS, Frost EP, Birdwell RL.

</span>
<span class="ltx_bibblock">Interpreting one-view mammographic findings: Minimizing callbacks while maximizing cancer detection.

</span>
<span class="ltx_bibblock">Radiographics. 2014;34(4):928-40.

</span>
</li>
<li id="bib.bib166" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[166]</span>
<span class="ltx_bibblock">
Kerlikowske K, Zhu W, Tosteson AN, Sprague BL, Tice JA, Lehman CD, et al.

</span>
<span class="ltx_bibblock">Identifying women with dense breasts at high risk for interval cancer: a cohort study.

</span>
<span class="ltx_bibblock">Breast Cancer Surveillance Consortium (BCSC). 2015;19(2):236-48.

</span>
</li>
<li id="bib.bib167" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[167]</span>
<span class="ltx_bibblock">
Lee RS, Gimenez F, Hoogi A, Miyake KK, Gorovoy M, Rubin DL.

</span>
<span class="ltx_bibblock">A curated mammography data set for use in computer-aided detection and diagnosis research.

</span>
<span class="ltx_bibblock">Scientific Data. 2017 Dec;4(1).

</span>
</li>
<li id="bib.bib168" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[168]</span>
<span class="ltx_bibblock">
Bowyer KW.

</span>
<span class="ltx_bibblock">Digital Image Database with gold standard and performance metrics for Mammographic Image Analysis Research. 1999 Aug.

</span>
</li>
<li id="bib.bib169" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[169]</span>
<span class="ltx_bibblock">
Halling-Brown MD, Warren LM, Ward D, Lewis E, Mackenzie A, Wallis MG, et al.. OPTIMAM Mammography Image Database: a large scale resource of mammography images and clinical data; 2020.

</span>
</li>
<li id="bib.bib170" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[170]</span>
<span class="ltx_bibblock">
Moreira IC, Amaral I, Domingues I, Cardoso A, Cardoso MJ, Cardoso JS.

</span>
<span class="ltx_bibblock">INbreast: Toward a Full-Field Digital Mammographic Database.

</span>
<span class="ltx_bibblock">Academic Radiology. 2011;19(2):236-48.

</span>
</li>
<li id="bib.bib171" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[171]</span>
<span class="ltx_bibblock">
Iskandar, M ,  Mannering, H ,  Sun, Z , et al.

</span>
<span class="ltx_bibblock">Towards Realistic Ultrasound Fetal Brain Imaging Synthesis.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib172" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[172]</span>
<span class="ltx_bibblock">
Liang J, Chen J.

</span>
<span class="ltx_bibblock">Data Augmentation of Thyroid Ultrasound Images Using Generative Adversarial Network.

</span>
<span class="ltx_bibblock">In: IEEE International Ultrasonics Symposium, IUS; 2021. .

</span>
</li>
<li id="bib.bib173" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[173]</span>
<span class="ltx_bibblock">
Montero A, Bonet-Carne E, Burgos-Artizzu XP.

</span>
<span class="ltx_bibblock">Generative Adversarial Networks to Improve Fetal Brain Fine-Grained Plane Classification.

</span>
<span class="ltx_bibblock">Sensors (Basel, Switzerland). 2021;21(23).

</span>
</li>
<li id="bib.bib174" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[174]</span>
<span class="ltx_bibblock">
Maack L, Holstein L, Schlaefer A.

</span>
<span class="ltx_bibblock">GANs for Generation of Synthetic Ultrasound Images from Small Datasets.

</span>
<span class="ltx_bibblock">Current Directions in Biomedical Engineering. 2022;8(1):17-20.

</span>
</li>
<li id="bib.bib175" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[175]</span>
<span class="ltx_bibblock">
Che H, Ramanathan S, Foran DJ, Nosher JL, Patel VM, Hacihaliloglu I.

</span>
<span class="ltx_bibblock">Realistic Ultrasound Image Synthesis for Improved Classification of Liver Disease.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 12967 LNCS; 2021. p. 179-88.

</span>
</li>
<li id="bib.bib176" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[176]</span>
<span class="ltx_bibblock">
Pang T, Wong JHD, Ng WL, Chan CS.

</span>
<span class="ltx_bibblock">Semi-Supervised GAN-based Radiomics Model for Data Augmentation in Breast Ultrasound Mass Classification.

</span>
<span class="ltx_bibblock">Computer Methods and Programs in Biomedicine. 2021;203.

</span>
</li>
<li id="bib.bib177" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[177]</span>
<span class="ltx_bibblock">
Tan H, Lang X, He B, Lu Y, Zhang Y.

</span>
<span class="ltx_bibblock">GAN-based Medical Image Augmentation for Improving CNN Performance in Myositis Ultrasound Image Classification.

</span>
<span class="ltx_bibblock">In: 2023 6th International Conference on Electronics Technology, ICET 2023; 2023. p. 1329-33.

</span>
</li>
<li id="bib.bib178" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[178]</span>
<span class="ltx_bibblock">
Gong X, Zhou H, Gu Y, Guo Y.

</span>
<span class="ltx_bibblock">Breast Ultrasound Image Classification with Hard Sample Generation and Semi-Supervised Learning.

</span>
<span class="ltx_bibblock">Biomedical Signal Processing and Control. 2023;86.

</span>
</li>
<li id="bib.bib179" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[179]</span>
<span class="ltx_bibblock">
Stojanovski D, Hermida U, Lamata P, Beqiri A, Gomez A.

</span>
<span class="ltx_bibblock">Echo from Noise: Synthetic Ultrasound Image Generation Using Diffusion Models for Real Image Segmentation.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 14337 LNCS; 2023. p. 34-43.

</span>
</li>
<li id="bib.bib180" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[180]</span>
<span class="ltx_bibblock">
Yao Z, Luo T, Dong YJ, Jia XH, Deng YH, Wu GQ, et al.

</span>
<span class="ltx_bibblock">Virtual Elastography Ultrasound via Generative Adversarial Network for Breast Cancer Diagnosis.

</span>
<span class="ltx_bibblock">Nature Communications. 2023;14(1).

</span>
</li>
<li id="bib.bib181" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[181]</span>
<span class="ltx_bibblock">
Zhang Q, Zhao J, Long X, Luo Q, Wang R, Ding X, et al.

</span>
<span class="ltx_bibblock">AUE-Net: Automated Generation of Ultrasound Elastography Using Generative Adversarial Network.

</span>
<span class="ltx_bibblock">Diagnostics. 2022;12(2).

</span>
</li>
<li id="bib.bib182" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[182]</span>
<span class="ltx_bibblock">
Qiao S, Pan S, Luo G, Pang S, Chen T, Singh AK, et al.

</span>
<span class="ltx_bibblock">A Pseudo-Siamese Feature Fusion Generative Adversarial Network for Synthesizing High-Quality Fetal Four-Chamber Views.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics. 2023;27(3):1193-204.

</span>
</li>
<li id="bib.bib183" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[183]</span>
<span class="ltx_bibblock">
Liang J, Yang X, Huang Y, Li H, He S, Hu X, et al.

</span>
<span class="ltx_bibblock">Sketch Guided and Progressive Growing GAN for Realistic and Editable Ultrasound Image Synthesis.

</span>
<span class="ltx_bibblock">Medical Image Analysis. 2022;79.

</span>
</li>
<li id="bib.bib184" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[184]</span>
<span class="ltx_bibblock">
Dorent R, Haouchine N, Kogl F, Joutard S, Juvekar P, Torio E, et al.

</span>
<span class="ltx_bibblock">Unified Brain MR-Ultrasound Synthesis Using Multi-modal Hierarchical Representations.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (Including Subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 14229 LNCS; 2023. p. 448-58.

</span>
</li>
<li id="bib.bib185" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[185]</span>
<span class="ltx_bibblock">
Chen L, Liao H, Kong W, Zhang D, Chen F.

</span>
<span class="ltx_bibblock">Anatomy Preserving GAN for Realistic Simulation of Intraoperative Liver Ultrasound Images.

</span>
<span class="ltx_bibblock">Computer Methods and Programs in Biomedicine. 2023;240.

</span>
</li>
<li id="bib.bib186" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[186]</span>
<span class="ltx_bibblock">
Jin S, Monkam P, Lu W.

</span>
<span class="ltx_bibblock">Echocardiography Segmentation Based on Cross-modal Data Augmentation Method.

</span>
<span class="ltx_bibblock">In: IEEE International Ultrasonics Symposium, IUS. vol. 2022-October; 2022. .

</span>
</li>
<li id="bib.bib187" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[187]</span>
<span class="ltx_bibblock">
Song Y, Zheng J, Lei L, Ni Z, Zhao B, Hu Y.

</span>
<span class="ltx_bibblock">CT2US: Cross-modal Transfer Learning for Kidney Segmentation in Ultrasound Images with Synthesized Data.

</span>
<span class="ltx_bibblock">Ultrasonics. 2022;122.

</span>
</li>
<li id="bib.bib188" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[188]</span>
<span class="ltx_bibblock">
Tiago C, Gilbert A, Beela AS, Aase SA, Snare SR, Šprem J, et al.

</span>
<span class="ltx_bibblock">A Data Augmentation Pipeline to Generate Synthetic Labeled Datasets of 3D Echocardiography Images Using a GAN.

</span>
<span class="ltx_bibblock">IEEE Access. 2022;10:98803-15.

</span>
</li>
<li id="bib.bib189" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[189]</span>
<span class="ltx_bibblock">
Sigrist RM, Liau J, El Kaffas A, Chammas MC, Willmann JK.

</span>
<span class="ltx_bibblock">Ultrasound elastography: review of techniques and clinical applications.

</span>
<span class="ltx_bibblock">Theranostics. 2017;7(5):1303.

</span>
</li>
<li id="bib.bib190" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[190]</span>
<span class="ltx_bibblock">
Isola P, Zhu JY, Zhou T, Efros AA. Image-to-Image Translation with Conditional Adversarial Networks; 2018.

</span>
</li>
<li id="bib.bib191" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[191]</span>
<span class="ltx_bibblock">
Gilbert A, Marciniak M, Rodero C, Lamata P, Samset E, McLeod K.

</span>
<span class="ltx_bibblock">Generating Synthetic Labeled Data from Existing Anatomical Models: An Example with Echocardiography Segmentation.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging. 2021;40(10):2783-94.

</span>
</li>
<li id="bib.bib192" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[192]</span>
<span class="ltx_bibblock">
Chen X, Duan Y, Houthooft R, Schulman J, Sutskever I, Abbeel P. InfoGAN: Interpretable Representation Learning by Information Maximizing Generative Adversarial Nets; 2016.

</span>
</li>
<li id="bib.bib193" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[193]</span>
<span class="ltx_bibblock">
La Rosa F, Yu T, Barquero G, Thiran JP, Granziera C, Bach Cuadra M.

</span>
<span class="ltx_bibblock">MPRAGE to MP2RAGE UNI translation via generative adversarial network improves the automatic tissue and lesion segmentation in multiple sclerosis patients.

</span>
<span class="ltx_bibblock">Comput Biol Med. 2021;132:104297.

</span>
<span class="ltx_bibblock">Place: United States.

</span>
</li>
<li id="bib.bib194" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[194]</span>
<span class="ltx_bibblock">
Qin Z, Liu Z, Zhu P, Ling W.

</span>
<span class="ltx_bibblock">Style transfer in conditional GANs for cross-modality synthesis of brain magnetic resonance images.

</span>
<span class="ltx_bibblock">Comput Biol Med. 2022;148:105928.

</span>
<span class="ltx_bibblock">Place: United States.

</span>
</li>
<li id="bib.bib195" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[195]</span>
<span class="ltx_bibblock">
Zhu L, He Q, Huang Y, Zhang Z, Zeng J, Lu L, et al.

</span>
<span class="ltx_bibblock">DualMMP-GAN: Dual-scale multi-modality perceptual generative adversarial network for medical image segmentation.

</span>
<span class="ltx_bibblock">Comput Biol Med. 2022;144:105387.

</span>
<span class="ltx_bibblock">Place: United States.

</span>
</li>
<li id="bib.bib196" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[196]</span>
<span class="ltx_bibblock">
Fei Y, Zhan B, Hong M, Wu X, Zhou J, Wang Y.

</span>
<span class="ltx_bibblock">Deep learning-based multi-modal computing with feature disentanglement for MRI image synthesis.

</span>
<span class="ltx_bibblock">Med Phys. 2021;48(7):3778-89.

</span>
<span class="ltx_bibblock">Place: United States.

</span>
</li>
<li id="bib.bib197" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[197]</span>
<span class="ltx_bibblock">
Huo J, Vakharia V, Wu C, Sharan A, Ko A, Ourselin S, et al.

</span>
<span class="ltx_bibblock">Brain Lesion Synthesis via Progressive Adversarial Variational Auto-Encoder.

</span>
<span class="ltx_bibblock">In: Zhao C, Svoboda D, Wolterink JM, Escobar M, editors. Simulation and Synthesis in Medical Imaging. Lecture Notes in Computer Science. Springer International Publishing; 2022. p. 101-11.

</span>
</li>
<li id="bib.bib198" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[198]</span>
<span class="ltx_bibblock">
Dhaene AP, Loecher M, Wilson AJ, Ennis DB.

</span>
<span class="ltx_bibblock">Myocardial Segmentation of Tagged Magnetic Resonance Images with Transfer Learning Using Generative Cine-To-Tagged Dataset Transformation.

</span>
<span class="ltx_bibblock">Bioengineering. 2023;10(2).

</span>
</li>
<li id="bib.bib199" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[199]</span>
<span class="ltx_bibblock">
Chen X, Lian C, Wang L, Deng H, Kuang T, Fung SH, et al.

</span>
<span class="ltx_bibblock">Diverse data augmentation for learning image segmentation with cross-modality annotations.

</span>
<span class="ltx_bibblock">Med Image Anal. 2021;71:102060.

</span>
<span class="ltx_bibblock">Place: Netherlands.

</span>
</li>
<li id="bib.bib200" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[200]</span>
<span class="ltx_bibblock">
Yurt M, Dar SU, Erdem A, Erdem E, Oguz KK, Çukur T.

</span>
<span class="ltx_bibblock">mustGAN: multi-stream Generative Adversarial Networks for MR Image Synthesis.

</span>
<span class="ltx_bibblock">Med Image Anal. 2021;70:101944.

</span>
<span class="ltx_bibblock">Place: Netherlands.

</span>
</li>
<li id="bib.bib201" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[201]</span>
<span class="ltx_bibblock">
Moshe YH, Buchsweiler Y, Teicher M, Artzi M.

</span>
<span class="ltx_bibblock">Handling Missing MRI Data in Brain Tumors Classification Tasks: Usage of Synthetic Images vs. Duplicate Images and Empty Images.

</span>
<span class="ltx_bibblock">J Magn Reson Imaging. 2023.

</span>
<span class="ltx_bibblock">Place: United States.

</span>
</li>
<li id="bib.bib202" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[202]</span>
<span class="ltx_bibblock">
Fan CC, Peng L, Wang T, Yang H, Zhou XH, Ni ZL, et al.

</span>
<span class="ltx_bibblock">TR-GAN: Multi-Session Future MRI Prediction With Temporal Recurrent Generative Adversarial Network.

</span>
<span class="ltx_bibblock">IEEE Trans Med Imaging. 2022;41(8):1925-37.

</span>
<span class="ltx_bibblock">Place: United States.

</span>
</li>
<li id="bib.bib203" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[203]</span>
<span class="ltx_bibblock">
Huang P, Li D, Jiao Z, Wei D, Cao B, Mo Z, et al.

</span>
<span class="ltx_bibblock">Common feature learning for brain tumor MRI synthesis by context-aware generative adversarial network.

</span>
<span class="ltx_bibblock">Med Image Anal. 2022;79:102472.

</span>
<span class="ltx_bibblock">Place: Netherlands.

</span>
</li>
<li id="bib.bib204" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[204]</span>
<span class="ltx_bibblock">
Amirrajab S, Al Khalil Y, Lorenz C, Weese J, Pluim J, Breeuwer M.

</span>
<span class="ltx_bibblock">Label-informed cardiac magnetic resonance image synthesis through conditional generative adversarial networks.

</span>
<span class="ltx_bibblock">Comput Med Imaging Graph. 2022;101:102123.

</span>
<span class="ltx_bibblock">Place: United States.

</span>
</li>
<li id="bib.bib205" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[205]</span>
<span class="ltx_bibblock">
Al Khalil Y, Amirrajab S, Lorenz C, Weese J, Pluim J, Breeuwer M.

</span>
<span class="ltx_bibblock">On the usability of synthetic data for improving the robustness of deep learning-based segmentation of cardiac magnetic resonance images.

</span>
<span class="ltx_bibblock">Medical Image Analysis. 2023;84.

</span>
</li>
<li id="bib.bib206" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[206]</span>
<span class="ltx_bibblock">
Lustermans DRPRM, Amirrajab S, Veta M, Breeuwer M, Scannell CM.

</span>
<span class="ltx_bibblock">Optimized automated cardiac MR scar quantification with GAN-based data augmentation.

</span>
<span class="ltx_bibblock">Comput Methods Programs Biomed. 2022;226:107116.

</span>
<span class="ltx_bibblock">Place: Ireland.

</span>
</li>
<li id="bib.bib207" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[207]</span>
<span class="ltx_bibblock">
Thermos S, Liu X, O’Neil A, Tsaftaris SA.

</span>
<span class="ltx_bibblock">Controllable Cardiac Synthesis via Disentangled Anatomy Arithmetic.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 12903 LNCS; 2021. p. 160-70.

</span>
</li>
<li id="bib.bib208" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[208]</span>
<span class="ltx_bibblock">
Amirrajab S, Lorenz C, Weese J, Pluim J, Breeuwer M.

</span>
<span class="ltx_bibblock">Pathology Synthesis of 3D Consistent Cardiac MR Images Using 2D VAEs and GANs.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 13570 LNCS; 2022. p. 34-42.

</span>
</li>
<li id="bib.bib209" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[209]</span>
<span class="ltx_bibblock">
Al Khalil Y, Amirrajab S, Lorenz C, Weese J, Pluim J, Breeuwer M.

</span>
<span class="ltx_bibblock">Reducing segmentation failures in cardiac MRI via late feature fusion and GAN-based augmentation.

</span>
<span class="ltx_bibblock">Computers in Biology and Medicine. 2023;161.

</span>
</li>
<li id="bib.bib210" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[210]</span>
<span class="ltx_bibblock">
Dorjsembe Z, Pao HK, Odonchimed S, Xiao F. Conditional Diffusion Models for Semantic 3D Medical Image Synthesis. arXiv; 2023.

</span>
</li>
<li id="bib.bib211" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[211]</span>
<span class="ltx_bibblock">
Fernandez V, Pinaya WHL, Borges P, Tudosiu PD, Graham MS, Vercauteren T, et al.

</span>
<span class="ltx_bibblock">Can segmentation models be trained with fully synthetically generated data?

</span>
<span class="ltx_bibblock">vol. 13570; 2022. p. 79-90.

</span>
</li>
<li id="bib.bib212" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[212]</span>
<span class="ltx_bibblock">
Isensee F, Jaeger PF, Kohl SA, Petersen J, Maier-Hein KH.

</span>
<span class="ltx_bibblock">nnU-Net: a self-configuring method for deep learning-based biomedical image segmentation.

</span>
<span class="ltx_bibblock">Nature methods. 2021;18(2):203-11.

</span>
</li>
<li id="bib.bib213" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[213]</span>
<span class="ltx_bibblock">
Li R, Bastiani M, Auer D, Wagner C, Chen X.

</span>
<span class="ltx_bibblock">Image Augmentation Using a Task Guided Generative Adversarial Network for Age Estimation on Brain MRI.

</span>
<span class="ltx_bibblock">In: Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics). vol. 12722 LNCS; 2021. p. 350-60.

</span>
</li>
<li id="bib.bib214" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[214]</span>
<span class="ltx_bibblock">
Dorjsembe Z, Odonchimed S, Xiao F.

</span>
<span class="ltx_bibblock">Three-Dimensional Medical Image Synthesis with Denoising Diffusion Probabilistic Models.

</span>
<span class="ltx_bibblock">In: Medical Imaging with Deep Learning; 2022. .

</span>
</li>
<li id="bib.bib215" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[215]</span>
<span class="ltx_bibblock">
Dar SUH, Ghanaat A, Kahmann J, Ayx I, Papavassiliu T, Schoenberg SO, et al.. Investigating Data Memorization in 3D Latent Diffusion Models for Medical Image Synthesis. arXiv; 2023.

</span>
</li>
<li id="bib.bib216" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[216]</span>
<span class="ltx_bibblock">
Yoon JS, Zhang C, Suk HI, Guo J, Li X.

</span>
<span class="ltx_bibblock">SADM: Sequence-Aware Diffusion Model for Longitudinal Medical Image Generation.

</span>
<span class="ltx_bibblock">vol. 13939; 2023. p. 388-400.

</span>
</li>
<li id="bib.bib217" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[217]</span>
<span class="ltx_bibblock">
Saeed SU, Syer T, Yan W, Yang Q, Emberton M, Punwani S, et al.. Bi-Parametric Prostate MR Image Synthesis Using Pathology and Sequence-Conditioned Stable Diffusion; 2023.

</span>
</li>
<li id="bib.bib218" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[218]</span>
<span class="ltx_bibblock">
Hu Q, Li H, Zhang J. Domain-Adaptive 3D Medical Image Synthesis: An Efficient Unsupervised Approach. arXiv; 2022.

</span>
</li>
<li id="bib.bib219" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[219]</span>
<span class="ltx_bibblock">
Jiang L, Mao Y, Chen X, Wang X, Li C. CoLa-Diff: Conditional Latent Diffusion Model for Multi-Modal MRI Synthesis. arXiv; 2023.

</span>
</li>
<li id="bib.bib220" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[220]</span>
<span class="ltx_bibblock">
Meng X, Gu Y, Pan Y, Wang N, Xue P, Lu M, et al.. A Novel Unified Conditional Score-based Generative Framework for Multi-modal Medical Image Completion. arXiv; 2022.

</span>
</li>
<li id="bib.bib221" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[221]</span>
<span class="ltx_bibblock">
Zhan B, Li D, Wang Y, Ma Z, Wu X, Zhou J, et al.

</span>
<span class="ltx_bibblock">LR-cGAN: Latent representation based conditional generative adversarial network for multi-modality MRI synthesis.

</span>
<span class="ltx_bibblock">Biomedical Signal Processing and Control. 2021;66.

</span>
</li>
<li id="bib.bib222" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[222]</span>
<span class="ltx_bibblock">
Teixeira JF, Dias M, Batista E, Costa J, Teixeira LF, Oliveira HP.

</span>
<span class="ltx_bibblock">Adversarial data augmentation on breast mri segmentation.

</span>
<span class="ltx_bibblock">Applied Sciences (Switzerland). 2021;11(10).

</span>
</li>
<li id="bib.bib223" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[223]</span>
<span class="ltx_bibblock">
Peng W, Adeli E, Bosschieter T, Park SH, Zhao Q, Pohl KM. Generating Realistic Brain MRIs via a Conditional Diffusion Probabilistic Model. arXiv; 2023.

</span>
</li>
<li id="bib.bib224" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[224]</span>
<span class="ltx_bibblock">
Kim B, Ye JC. Diffusion Deformable Model for 4D Temporal Medical Image Generation. arXiv; 2022.

</span>
</li>
<li id="bib.bib225" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[225]</span>
<span class="ltx_bibblock">
Wolleb J, Sandkühler R, Bieder F, Cattin PC. The Swiss Army Knife for Image-to-Image Translation: Multi-Task Diffusion Models. arXiv; 2022.

</span>
</li>
<li id="bib.bib226" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[226]</span>
<span class="ltx_bibblock">
Zhu L, Xue Z, Jin Z, Liu X, He J, Liu Z, et al.. Make-A-Volume: Leveraging Latent Diffusion Models for Cross-Modality 3D Brain MRI Synthesis. arXiv; 2023.

</span>
</li>
<li id="bib.bib227" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[227]</span>
<span class="ltx_bibblock">
Seco J, Evans PM.

</span>
<span class="ltx_bibblock">Assessing the effect of electron density in photon dose calculations.

</span>
<span class="ltx_bibblock">Medical physics. 2006;33(2):540-52.

</span>
</li>
<li id="bib.bib228" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[228]</span>
<span class="ltx_bibblock">
Nyholm T, Nyberg M, Karlsson MG, Karlsson M.

</span>
<span class="ltx_bibblock">Systematisation of spatial uncertainties for comparison between a MR and a CT-based radiotherapy workflow for prostate treatments.

</span>
<span class="ltx_bibblock">Radiation oncology. 2009;4:1-9.

</span>
</li>
<li id="bib.bib229" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[229]</span>
<span class="ltx_bibblock">
Jaffray DA.

</span>
<span class="ltx_bibblock">Image-guided radiotherapy: from current concept to future perspectives.

</span>
<span class="ltx_bibblock">Nature reviews Clinical oncology. 2012;9(12):688-99.

</span>
</li>
<li id="bib.bib230" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[230]</span>
<span class="ltx_bibblock">
Ulin K, Urie MM, Cherlow JM.

</span>
<span class="ltx_bibblock">Results of a multi-institutional benchmark test for cranial CT/MR image registration.

</span>
<span class="ltx_bibblock">International Journal of Radiation Oncology* Biology* Physics. 2010;77(5):1584-9.

</span>
</li>
<li id="bib.bib231" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[231]</span>
<span class="ltx_bibblock">
Nyholm T, Jonsson J.

</span>
<span class="ltx_bibblock">Counterpoint: opportunities and challenges of a magnetic resonance imaging–only radiotherapy work flow.

</span>
<span class="ltx_bibblock">In: Seminars in radiation oncology. vol. 24. Elsevier; 2014. p. 175-80.

</span>
</li>
<li id="bib.bib232" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[232]</span>
<span class="ltx_bibblock">
Taasti VT, Klages P, Parodi K, Muren LP.

</span>
<span class="ltx_bibblock">Developments in deep learning based corrections of cone beam computed tomography to enable dose calculations for adaptive radiotherapy.

</span>
<span class="ltx_bibblock">Physics and Imaging in Radiation Oncology. 2020;15:77-9.

</span>
</li>
<li id="bib.bib233" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[233]</span>
<span class="ltx_bibblock">
Zhu L, Wang J, Xing L.

</span>
<span class="ltx_bibblock">Noise suppression in scatter correction for cone-beam CT.

</span>
<span class="ltx_bibblock">Medical physics. 2009;36(3):741-52.

</span>
</li>
<li id="bib.bib234" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[234]</span>
<span class="ltx_bibblock">
Antypas EJ, Sokhandon F, Farah M, Emerson S, Bis KG, Tien H, et al.

</span>
<span class="ltx_bibblock">A comprehensive approach to CT radiation dose reduction: one institution’s experience.

</span>
<span class="ltx_bibblock">American Journal of Roentgenology. 2011;197(4):935-40.

</span>
</li>
<li id="bib.bib235" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[235]</span>
<span class="ltx_bibblock">
Zhang Z, Zhao T, Gay H, Zhang W, Sun B.

</span>
<span class="ltx_bibblock">Semi-supervised semantic segmentation of prostate and organs-at-risk on 3D pelvic CT images.

</span>
<span class="ltx_bibblock">Biomedical Physics &amp; Engineering Express. 2021;7(6):065023.

</span>
</li>
<li id="bib.bib236" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[236]</span>
<span class="ltx_bibblock">
Park HY, Bae HJ, Hong GS, Kim M, Yun J, Park S, et al.

</span>
<span class="ltx_bibblock">Realistic high-resolution body computed tomography image synthesis by using progressive growing generative adversarial network: visual turing test.

</span>
<span class="ltx_bibblock">JMIR medical informatics. 2021;9(3):e23328.

</span>
</li>
<li id="bib.bib237" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[237]</span>
<span class="ltx_bibblock">
Pesaranghader A, Wang Y, Havaei M.

</span>
<span class="ltx_bibblock">CT-SGAN: computed tomography synthesis GAN.

</span>
<span class="ltx_bibblock">In: Deep Generative Models, and Data Augmentation, Labelling, and Imperfections: First Workshop, DGM4MICCAI 2021, and First Workshop, DALI 2021, Held in Conjunction with MICCAI 2021, Strasbourg, France, October 1, 2021, Proceedings 1. Springer; 2021. p. 67-79.

</span>
</li>
<li id="bib.bib238" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[238]</span>
<span class="ltx_bibblock">
Semiletov A, Vatian A, Krychkov M, Khanzhina N, Klochkov A, Zubanenko A, et al.

</span>
<span class="ltx_bibblock">Comparative Evaluation of Lung Cancer CT Image Synthesis with Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">In: International Conference on Computational Science. Springer; 2021. p. 593-608.

</span>
</li>
<li id="bib.bib239" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[239]</span>
<span class="ltx_bibblock">
Kalantar R, Hindocha S, Hunter B, Sharma B, Khan N, Koh DM, et al.

</span>
<span class="ltx_bibblock">Non-contrast CT synthesis using patch-based cycle-consistent generative adversarial network (Cycle-GAN) for radiomics and deep learning in the era of COVID-19.

</span>
<span class="ltx_bibblock">Scientific Reports. 2023;13(1):10568.

</span>
</li>
<li id="bib.bib240" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[240]</span>
<span class="ltx_bibblock">
Nguyen LX, Aung PS, Le HQ, Park SB, Hong CS.

</span>
<span class="ltx_bibblock">A new chapter for medical image generation: The stable diffusion method.

</span>
<span class="ltx_bibblock">In: 2023 International Conference on Information Networking (ICOIN). IEEE; 2023. p. 483-6.

</span>
</li>
<li id="bib.bib241" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[241]</span>
<span class="ltx_bibblock">
Ambita AAE, Boquio ENV, Naval Jr PC.

</span>
<span class="ltx_bibblock">Covit-gan: vision transformer forcovid-19 detection in ct scan imageswith self-attention gan forDataAugmentation.

</span>
<span class="ltx_bibblock">In: International Conference on Artificial Neural Networks. Springer; 2021. p. 587-98.

</span>
</li>
<li id="bib.bib242" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[242]</span>
<span class="ltx_bibblock">
Shi Y, Wang G.

</span>
<span class="ltx_bibblock">Conversion of the Mayo LDCT data to synthetic equivalent through the diffusion model for training denoising networks with a theoretically perfect privacy.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230106604. 2023.

</span>
</li>
<li id="bib.bib243" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[243]</span>
<span class="ltx_bibblock">
Han K, Xiong Y, You C, Khosravi P, Sun S, Yan X, et al.

</span>
<span class="ltx_bibblock">Medgen3d: A deep generative framework for paired 3d image and mask generation.

</span>
<span class="ltx_bibblock">In: International Conference on Medical Image Computing and Computer-Assisted Intervention. Springer; 2023. p. 759-69.

</span>
</li>
<li id="bib.bib244" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[244]</span>
<span class="ltx_bibblock">
DuMont Schütte A, Hetzel J, Gatidis S, Hepp T, Dietz B, Bauer S, et al.

</span>
<span class="ltx_bibblock">Overcoming barriers to data sharing with medical image generation: a comprehensive evaluation.

</span>
<span class="ltx_bibblock">NPJ digital medicine. 2021;4(1):141.

</span>
</li>
<li id="bib.bib245" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[245]</span>
<span class="ltx_bibblock">
Xiao Q, Zhao L.

</span>
<span class="ltx_bibblock">End-to-End 3D Liver CT Image Synthesis from Vasculature Using a Multi-Task Conditional Generative Adversarial Network.

</span>
<span class="ltx_bibblock">Applied Sciences. 2023;13(11):6784.

</span>
</li>
<li id="bib.bib246" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[246]</span>
<span class="ltx_bibblock">
Mendes J, Pereira T, Silva F, Frade J, Morgado J, Freitas C, et al.

</span>
<span class="ltx_bibblock">Lung CT image synthesis using GANs.

</span>
<span class="ltx_bibblock">Expert Systems with Applications. 2023;215:119350.

</span>
</li>
<li id="bib.bib247" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[247]</span>
<span class="ltx_bibblock">
Wang Q, Zhang X, Zhang W, Gao M, Huang S, Wang J, et al.

</span>
<span class="ltx_bibblock">Realistic lung nodule synthesis with multi-target co-guided adversarial mechanism.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging. 2021;40(9):2343-53.

</span>
</li>
<li id="bib.bib248" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[248]</span>
<span class="ltx_bibblock">
Toda R, Teramoto A, Tsujimoto M, Toyama H, Imaizumi K, Saito K, et al.

</span>
<span class="ltx_bibblock">Synthetic CT image generation of shape-controlled lung cancer using semi-conditional InfoGAN and its applicability for type classification.

</span>
<span class="ltx_bibblock">International Journal of Computer Assisted Radiology and Surgery. 2021;16:241-51.

</span>
</li>
<li id="bib.bib249" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[249]</span>
<span class="ltx_bibblock">
Zhao X, Hou B.

</span>
<span class="ltx_bibblock">High-fidelity image synthesis from pulmonary nodule lesion maps using semantic diffusion model.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230501138. 2023.

</span>
</li>
<li id="bib.bib250" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[250]</span>
<span class="ltx_bibblock">
Xu Y, Sun L, Peng W, Visweswaran S, Batmanghelich K.

</span>
<span class="ltx_bibblock">MedSyn: Text-guided Anatomy-aware Synthesis of High-Fidelity 3D CT Images.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:231003559. 2023.

</span>
</li>
<li id="bib.bib251" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[251]</span>
<span class="ltx_bibblock">
Hamamci IE, Er S, Simsar E, Tezcan A, Simsek AG, Almas F, et al.

</span>
<span class="ltx_bibblock">GenerateCT: text-guided 3D chest CT generation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230516037. 2023.

</span>
</li>
<li id="bib.bib252" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[252]</span>
<span class="ltx_bibblock">
Liu Y, Chen A, Li Y, Lai H, Huang S, Yang X.

</span>
<span class="ltx_bibblock">CT synthesis from CBCT using a sequence-aware contrastive generative network.

</span>
<span class="ltx_bibblock">Computerized Medical Imaging and Graphics. 2023;109:102300.

</span>
</li>
<li id="bib.bib253" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[253]</span>
<span class="ltx_bibblock">
Chen H, Yan S, Xie M, Huang J.

</span>
<span class="ltx_bibblock">Application of cascaded GAN based on CT scan in the diagnosis of aortic dissection.

</span>
<span class="ltx_bibblock">Computer Methods and Programs in Biomedicine. 2022;226:107130.

</span>
</li>
<li id="bib.bib254" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[254]</span>
<span class="ltx_bibblock">
Li M, Wang J, Chen Y, Tang Y, Wu Z, Qi Y, et al.

</span>
<span class="ltx_bibblock">Low-dose CT image synthesis for domain adaptation imaging using a generative adversarial network with noise encoding transfer learning.

</span>
<span class="ltx_bibblock">IEEE transactions on medical imaging. 2023.

</span>
</li>
<li id="bib.bib255" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[255]</span>
<span class="ltx_bibblock">
Hooshangnejad H, Chen Q, Feng X, Zhang R, Ding K.

</span>
<span class="ltx_bibblock">deepPERFECT: Novel Deep Learning CT Synthesis Method for Expeditious Pancreatic Cancer Radiotherapy.

</span>
<span class="ltx_bibblock">Cancers. 2023;15(11):3061.

</span>
</li>
<li id="bib.bib256" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[256]</span>
<span class="ltx_bibblock">
Gao L, Xie K, Wu X, Lu Z, Li C, Sun J, et al.

</span>
<span class="ltx_bibblock">Generating synthetic CT from low-dose cone-beam CT by using generative adversarial networks for adaptive radiotherapy.

</span>
<span class="ltx_bibblock">Radiation Oncology. 2021;16:1-16.

</span>
</li>
<li id="bib.bib257" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[257]</span>
<span class="ltx_bibblock">
Krinski BA, Ruiz DV, Laroca R, Todt E.

</span>
<span class="ltx_bibblock">DACov: a deeper analysis of data augmentation on the computed tomography segmentation problem.

</span>
<span class="ltx_bibblock">Computer Methods in Biomechanics and Biomedical Engineering: Imaging &amp; Visualization. 2023;11(5):1724-41.

</span>
</li>
<li id="bib.bib258" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[258]</span>
<span class="ltx_bibblock">
Fang C, Liu Y, Liu Y, Liu M, Qiu X, Li Y, et al.

</span>
<span class="ltx_bibblock">Label-free coronavirus disease 2019 lesion segmentation based on synthetic healthy lung image subtraction.

</span>
<span class="ltx_bibblock">Medical Physics. 2022;49(7):4632-41.

</span>
</li>
<li id="bib.bib259" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[259]</span>
<span class="ltx_bibblock">
Peng J, Qiu RL, Wynne JF, Chang CW, Pan S, Wang T, et al.

</span>
<span class="ltx_bibblock">CBCT-Based synthetic CT image generation using conditional denoising diffusion probabilistic model.

</span>
<span class="ltx_bibblock">Medical physics. 2023.

</span>
</li>
<li id="bib.bib260" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[260]</span>
<span class="ltx_bibblock">
Chaudhary MF, Gerard SE, Christensen GE, Cooper CB, Schroeder JD, Hoffman EA, et al.

</span>
<span class="ltx_bibblock">LungViT: Ensembling Cascade of Texture Sensitive Hierarchical Vision Transformers for Cross-Volume Chest CT Image-to-Image Translation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:221002625. 2022.

</span>
</li>
<li id="bib.bib261" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[261]</span>
<span class="ltx_bibblock">
Zhang Y, Zhong L, Shu H, Dai Z, Zheng K, Chen Z, et al.

</span>
<span class="ltx_bibblock">Cross-task feedback fusion gan for joint mr-ct synthesis and segmentation of target and organs-at-risk.

</span>
<span class="ltx_bibblock">IEEE Transactions on Artificial Intelligence. 2022.

</span>
</li>
<li id="bib.bib262" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[262]</span>
<span class="ltx_bibblock">
Liu H, Sigona MK, Manuel TJ, Chen LM, Dawant BM, Caskey CF.

</span>
<span class="ltx_bibblock">Evaluation of synthetically generated CT for use in transcranial focused ultrasound procedures.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:221014775. 2022.

</span>
</li>
<li id="bib.bib263" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[263]</span>
<span class="ltx_bibblock">
Tronchin L, Vu MH, Soda P, Löfstedt T.

</span>
<span class="ltx_bibblock">LatentAugment: Data Augmentation via Guided Manipulation of GAN’s Latent Space.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230711375. 2023.

</span>
</li>
<li id="bib.bib264" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[264]</span>
<span class="ltx_bibblock">
Liu Y, Chen A, Shi H, Huang S, Zheng W, Liu Z, et al.

</span>
<span class="ltx_bibblock">CT synthesis from MRI using multi-cycle GAN for head-and-neck radiation therapy.

</span>
<span class="ltx_bibblock">Computerized medical imaging and graphics. 2021;91:101953.

</span>
</li>
<li id="bib.bib265" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[265]</span>
<span class="ltx_bibblock">
Sun B, Jia S, Jiang X, Jia F.

</span>
<span class="ltx_bibblock">Double U-Net CycleGAN for 3D MR to CT image synthesis.

</span>
<span class="ltx_bibblock">International Journal of Computer Assisted Radiology and Surgery. 2023;18(1):149-56.

</span>
</li>
<li id="bib.bib266" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[266]</span>
<span class="ltx_bibblock">
Abu-Srhan A, Almallahi I, Abushariah MA, Mahafza W, Al-Kadi OS.

</span>
<span class="ltx_bibblock">Paired-unpaired Unsupervised Attention Guided GAN with transfer learning for bidirectional brain MR-CT synthesis.

</span>
<span class="ltx_bibblock">Computers in Biology and Medicine. 2021;136:104763.

</span>
</li>
<li id="bib.bib267" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[267]</span>
<span class="ltx_bibblock">
Yoo GS, Luu HM, Kim H, Park W, Pyo H, Han Y, et al.

</span>
<span class="ltx_bibblock">Feasibility of synthetic computed tomography images generated from magnetic resonance imaging scans using various deep learning methods in the planning of radiation therapy for prostate cancer.

</span>
<span class="ltx_bibblock">Cancers. 2021;14(1):40.

</span>
</li>
<li id="bib.bib268" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[268]</span>
<span class="ltx_bibblock">
Roberts M, Hinton G, Wells AJ, Van Der Veken J, Bajger M, Lee G, et al.

</span>
<span class="ltx_bibblock">Imaging evaluation of a proposed 3D generative model for MRI to CT translation in the lumbar spine.

</span>
<span class="ltx_bibblock">The Spine Journal. 2023;23(11):1602-12.

</span>
</li>
<li id="bib.bib269" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[269]</span>
<span class="ltx_bibblock">
Ma X, Chen X, Li J, Wang Y, Men K, Dai J.

</span>
<span class="ltx_bibblock">MRI-only radiotherapy planning for nasopharyngeal carcinoma using deep learning.

</span>
<span class="ltx_bibblock">Frontiers in oncology. 2021;11:713617.

</span>
</li>
<li id="bib.bib270" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[270]</span>
<span class="ltx_bibblock">
Vajpayee R, Agrawal V, Krishnamurthi G.

</span>
<span class="ltx_bibblock">Structurally-constrained optical-flow-guided adversarial generation of synthetic CT for MR-only radiotherapy treatment planning.

</span>
<span class="ltx_bibblock">Scientific Reports. 2022;12(1):14855.

</span>
</li>
<li id="bib.bib271" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[271]</span>
<span class="ltx_bibblock">
Joseph J, Hemanth C, Pulinthanathu Narayanan P, Pottekkattuvalappil Balakrishnan J, Puzhakkal N.

</span>
<span class="ltx_bibblock">Computed tomography image generation from magnetic resonance imaging using Wasserstein metric for MR-only radiation therapy.

</span>
<span class="ltx_bibblock">International Journal of Imaging Systems and Technology. 2022;32(6):2080-93.

</span>
</li>
<li id="bib.bib272" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[272]</span>
<span class="ltx_bibblock">
Schaefferkoetter J, Yan J, Moon S, Chan R, Ortega C, Metser U, et al.

</span>
<span class="ltx_bibblock">Deep learning for whole-body medical image generation.

</span>
<span class="ltx_bibblock">European journal of nuclear medicine and molecular imaging. 2021;48(12):3817-26.

</span>
</li>
<li id="bib.bib273" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[273]</span>
<span class="ltx_bibblock">
Sun H, Lu Z, Fan R, Xiong W, Xie K, Ni X, et al.

</span>
<span class="ltx_bibblock">Research on obtaining pseudo CT images based on stacked generative adversarial network.

</span>
<span class="ltx_bibblock">Quantitative Imaging in Medicine and Surgery. 2021;11(5):1983.

</span>
</li>
<li id="bib.bib274" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[274]</span>
<span class="ltx_bibblock">
Özbey M, Dalmaz O, Dar SU, Bedel HA, Özturk Ş, Güngör A, et al.

</span>
<span class="ltx_bibblock">Unsupervised medical image translation with adversarial diffusion models.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging. 2023.

</span>
</li>
<li id="bib.bib275" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[275]</span>
<span class="ltx_bibblock">
Lyu Q, Wang G.

</span>
<span class="ltx_bibblock">Conversion between CT and MRI images using diffusion and score-matching models.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:220912104. 2022.

</span>
</li>
<li id="bib.bib276" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[276]</span>
<span class="ltx_bibblock">
Li X, Shang K, Wang G, Butala MD.

</span>
<span class="ltx_bibblock">Ddmm-synth: A denoising diffusion model for cross-modal medical image synthesis with sparse-view measurement embedding.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230315770. 2023.

</span>
</li>
<li id="bib.bib277" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[277]</span>
<span class="ltx_bibblock">
Pan S, Abouei E, Wynne J, Chang CW, Wang T, Qiu RL, et al.

</span>
<span class="ltx_bibblock">Synthetic CT generation from MRI using 3D transformer-based denoising diffusion model.

</span>
<span class="ltx_bibblock">Medical Physics. 2023.

</span>
</li>
<li id="bib.bib278" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[278]</span>
<span class="ltx_bibblock">
Zhao B, Cheng T, Zhang X, Wang J, Zhu H, Zhao R, et al.

</span>
<span class="ltx_bibblock">CT synthesis from MR in the pelvic area using Residual Transformer Conditional GAN.

</span>
<span class="ltx_bibblock">Computerized medical imaging and graphics. 2023;103:102150.

</span>
</li>
<li id="bib.bib279" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[279]</span>
<span class="ltx_bibblock">
Kim G, Baek J.

</span>
<span class="ltx_bibblock">Power-law spectrum-based objective function to train a generative adversarial network with transfer learning for the synthetic breast CT image.

</span>
<span class="ltx_bibblock">Physics in Medicine &amp; Biology. 2023;68(20):205007.

</span>
</li>
<li id="bib.bib280" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[280]</span>
<span class="ltx_bibblock">
Hu Q, Yuille A, Zhou Z.

</span>
<span class="ltx_bibblock">Synthetic Data as Validation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:231016052. 2023.

</span>
</li>
<li id="bib.bib281" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[281]</span>
<span class="ltx_bibblock">
Prezja F, Paloneva J, Pölönen I, Niinimäki E, Äyrämö S.

</span>
<span class="ltx_bibblock">DeepFake knee osteoarthritis X-rays from generative adversarial neural networks deceive medical experts and offer augmentation potential to automatic classification.

</span>
<span class="ltx_bibblock">Scientific Reports. 2022;12(1):18573.

</span>
</li>
<li id="bib.bib282" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[282]</span>
<span class="ltx_bibblock">
Ng MF, Hargreaves CA.

</span>
<span class="ltx_bibblock">Generative Adversarial Networks for the Synthesis of Chest X-ray Images.

</span>
<span class="ltx_bibblock">Engineering Proceedings. 2023;31(1):84.

</span>
</li>
<li id="bib.bib283" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[283]</span>
<span class="ltx_bibblock">
Qin X, Bui FM, Nguyen HH, Han Z.

</span>
<span class="ltx_bibblock">Learning from limited and imbalanced medical images with finer synthetic images from gans.

</span>
<span class="ltx_bibblock">Ieee Access. 2022;10:91663-77.

</span>
</li>
<li id="bib.bib284" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[284]</span>
<span class="ltx_bibblock">
Segal B, Rubin DM, Rubin G, Pantanowitz A.

</span>
<span class="ltx_bibblock">Evaluating the clinical realism of synthetic chest x-rays generated using progressively growing gans.

</span>
<span class="ltx_bibblock">SN Computer Science. 2021;2(4):321.

</span>
</li>
<li id="bib.bib285" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[285]</span>
<span class="ltx_bibblock">
Jang M, Bae Hj, Kim M, Park SY, Son Ay, Choi SJ, et al.

</span>
<span class="ltx_bibblock">Image Turing test and its applications on synthetic chest radiographs by using the progressive growing generative adversarial network.

</span>
<span class="ltx_bibblock">Scientific Reports. 2023;13(1):2356.

</span>
</li>
<li id="bib.bib286" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[286]</span>
<span class="ltx_bibblock">
Saad MM, Rehmani MH, O’Reilly R.

</span>
<span class="ltx_bibblock">A self-attention guided multi-scale gradient GAN for diversified x-ray image synthesis.

</span>
<span class="ltx_bibblock">In: Irish Conference on Artificial Intelligence and Cognitive Science. Springer; 2022. p. 18-31.

</span>
</li>
<li id="bib.bib287" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[287]</span>
<span class="ltx_bibblock">
Flores D, Hemberg E, Toutouh J, O’Reily UM.

</span>
<span class="ltx_bibblock">Coevolutionary generative adversarial networks for medical image augumentation at scale.

</span>
<span class="ltx_bibblock">In: Proceedings of the Genetic and Evolutionary Computation Conference; 2022. p. 367-76.

</span>
</li>
<li id="bib.bib288" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[288]</span>
<span class="ltx_bibblock">
Weber T, Ingrisch M, Bischl B, Rügamer D.

</span>
<span class="ltx_bibblock">Implicit embeddings via GAN inversion for high resolution chest radiographs.

</span>
<span class="ltx_bibblock">In: MICCAI Workshop on Medical Applications with Disentanglements. Springer; 2022. p. 22-32.

</span>
</li>
<li id="bib.bib289" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[289]</span>
<span class="ltx_bibblock">
Kusakunniran W, Saiviroonporn P, Siriapisith T, Tongdee T, Uraiverotchanakorn A, Leesakul S, et al.

</span>
<span class="ltx_bibblock">Automatic measurement of cardiothoracic ratio in chest x-ray images with ProGAN-generated dataset.

</span>
<span class="ltx_bibblock">Applied Computing and Informatics. 2023;(ahead-of-print).

</span>
</li>
<li id="bib.bib290" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[290]</span>
<span class="ltx_bibblock">
Huy PN, Quan TM.

</span>
<span class="ltx_bibblock">Denoising Diffusion Medical Models.

</span>
<span class="ltx_bibblock">In: 2023 IEEE 20th International Symposium on Biomedical Imaging (ISBI). IEEE; 2023. p. 1-5.

</span>
</li>
<li id="bib.bib291" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[291]</span>
<span class="ltx_bibblock">
Kausar T, Lu Y, Kausar A, Ali M, Yousaf A.

</span>
<span class="ltx_bibblock">SD-GAN: A style distribution transfer generative adversarial network for Covid-19 detection through X-ray images.

</span>
<span class="ltx_bibblock">IEEE Access. 2023;11:24545-60.

</span>
</li>
<li id="bib.bib292" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[292]</span>
<span class="ltx_bibblock">
Shen Z, Ouyang X, Xiao B, Cheng JZ, Shen D, Wang Q.

</span>
<span class="ltx_bibblock">Image synthesis with disentangled attributes for chest X-ray nodule augmentation and detection.

</span>
<span class="ltx_bibblock">Medical Image Analysis. 2023;84:102708.

</span>
</li>
<li id="bib.bib293" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[293]</span>
<span class="ltx_bibblock">
Kim Y, Lee JH, Kim C, Jin KN, Park CM.

</span>
<span class="ltx_bibblock">GAN based ROI conditioned synthesis of medical image for data augmentation.

</span>
<span class="ltx_bibblock">In: Medical Imaging 2023: Image Processing. vol. 12464. SPIE; 2023. p. 739-45.

</span>
</li>
<li id="bib.bib294" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[294]</span>
<span class="ltx_bibblock">
Ciano G, Andreini P, Mazzierli T, Bianchini M, Scarselli F.

</span>
<span class="ltx_bibblock">A multi-stage GAN for multi-organ chest X-ray image generation and segmentation.

</span>
<span class="ltx_bibblock">Mathematics. 2021;9(22):2896.

</span>
</li>
<li id="bib.bib295" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[295]</span>
<span class="ltx_bibblock">
Weber T, Ingrisch M, Bischl B, Rügamer D.

</span>
<span class="ltx_bibblock">Cascaded latent diffusion models for high-resolution chest x-ray synthesis.

</span>
<span class="ltx_bibblock">In: Pacific-Asia Conference on Knowledge Discovery and Data Mining. Springer; 2023. p. 180-91.

</span>
</li>
<li id="bib.bib296" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[296]</span>
<span class="ltx_bibblock">
Mahapatra D.

</span>
<span class="ltx_bibblock">Cyclic generative adversarial networks with congruent image-report generation for explainable medical image analysis.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:221108424. 2022.

</span>
</li>
<li id="bib.bib297" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[297]</span>
<span class="ltx_bibblock">
Lee S, Kim WJ, Chang J, Ye JC.

</span>
<span class="ltx_bibblock">LLM-CXR: Instruction-Finetuned LLM for CXR Image Understanding and Generation.

</span>
<span class="ltx_bibblock">In: The Twelfth International Conference on Learning Representations; 2023. .

</span>
</li>
<li id="bib.bib298" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[298]</span>
<span class="ltx_bibblock">
Lee H, Lee DY, Kim W, Kim JH, Kim T, Kim J, et al.

</span>
<span class="ltx_bibblock">UniXGen: A Unified Vision-Language Model for Multi-View Chest X-ray Generation and Report Generation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230212172. 2023.

</span>
</li>
<li id="bib.bib299" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[299]</span>
<span class="ltx_bibblock">
Chambon P, Bluethgen C, Delbrouck JB, Van der Sluijs R, Połacin M, Chaves JMZ, et al.

</span>
<span class="ltx_bibblock">Roentgen: Vision-language foundation model for chest x-ray generation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:221112737. 2022.

</span>
</li>
<li id="bib.bib300" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[300]</span>
<span class="ltx_bibblock">
Kim E, Lee S, Lee KM.

</span>
<span class="ltx_bibblock">Abnormality detection in chest X-Ray via residual-saliency from normal generation.

</span>
<span class="ltx_bibblock">IEEE Access. 2023;11:21799-810.

</span>
</li>
<li id="bib.bib301" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[301]</span>
<span class="ltx_bibblock">
Sanchez K, Hinojosa C, Arguello H, Kouamé D, Meyrignac O, Basarab A.

</span>
<span class="ltx_bibblock">CX-DaGAN: Domain adaptation for pneumonia diagnosis on a small chest X-ray dataset.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging. 2022;41(11):3278-88.

</span>
</li>
<li id="bib.bib302" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[302]</span>
<span class="ltx_bibblock">
Liang Z, Huang JX, Antani S.

</span>
<span class="ltx_bibblock">Image translation by ad CycleGAN for COVID-19 X-ray images: a new approach for controllable GAN.

</span>
<span class="ltx_bibblock">Sensors. 2022;22(24):9628.

</span>
</li>
<li id="bib.bib303" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[303]</span>
<span class="ltx_bibblock">
Bargshady G, Zhou X, Barua PD, Gururajan R, Li Y, Acharya UR.

</span>
<span class="ltx_bibblock">Application of CycleGAN and transfer learning techniques for automated detection of COVID-19 using X-ray images.

</span>
<span class="ltx_bibblock">Pattern Recognition Letters. 2022;153:67-74.

</span>
</li>
<li id="bib.bib304" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[304]</span>
<span class="ltx_bibblock">
Preedanan W, Suzuki K, Kondo T, Kobayashi M, Tanaka H, Ishioka J, et al.

</span>
<span class="ltx_bibblock">Improvement of urinary stone segmentation using GAN-based urinary stones inpainting augmentation.

</span>
<span class="ltx_bibblock">IEEE Access. 2022;10:115131-42.

</span>
</li>
<li id="bib.bib305" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[305]</span>
<span class="ltx_bibblock">
Jin Y, Chang W, Ko B.

</span>
<span class="ltx_bibblock">Generating Chest X–ray Progression of Pneumonia using Conditional Cycle Generative Adversarial Networks.

</span>
<span class="ltx_bibblock">IEEE Access. 2023.

</span>
</li>
<li id="bib.bib306" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[306]</span>
<span class="ltx_bibblock">
Zhou Y, Wu C, Wang X, Zheng Y.

</span>
<span class="ltx_bibblock">UXDiff: Synthesis of X-Ray Image from Ultrasound Coronal Image of Spine with Diffusion Probabilistic Network.

</span>
<span class="ltx_bibblock">In: International Conference on Medical Image Computing and Computer-Assisted Intervention. Springer; 2023. p. 3-12.

</span>
</li>
<li id="bib.bib307" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[307]</span>
<span class="ltx_bibblock">
Zheng C, Bian F, Li L, Xie X, Liu H, Liang J, et al.

</span>
<span class="ltx_bibblock">Assessment of generative adversarial networks for synthetic anterior segment optical coherence tomography images in closed-angle detection.

</span>
<span class="ltx_bibblock">Translational Vision Science &amp; Technology. 2021;10(4):34-4.

</span>
</li>
<li id="bib.bib308" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[308]</span>
<span class="ltx_bibblock">
Bai G, Li S, Zhang H, Higashita R, Liu J, Li J, et al.

</span>
<span class="ltx_bibblock">A Structure-Consistency GAN for Unpaired AS-OCT Image Inpainting.

</span>
<span class="ltx_bibblock">In: International Workshop on Ophthalmic Medical Image Analysis. Springer; 2023. p. 142-51.

</span>
</li>
<li id="bib.bib309" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[309]</span>
<span class="ltx_bibblock">
Zhao M, Lu Z, Zhu S, Wang X, Feng J.

</span>
<span class="ltx_bibblock">Automatic generation of retinal optical coherence tomography images based on generative adversarial networks.

</span>
<span class="ltx_bibblock">Medical Physics. 2022;49(11):7357-67.

</span>
</li>
<li id="bib.bib310" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[310]</span>
<span class="ltx_bibblock">
Sun LC, Pao SI, Huang KH, Wei CY, Lin KF, Chen PN.

</span>
<span class="ltx_bibblock">Generative adversarial network-based deep learning approach in classification of retinal conditions with optical coherence tomography images.

</span>
<span class="ltx_bibblock">Graefe’s Archive for Clinical and Experimental Ophthalmology. 2023;261(5):1399-412.

</span>
</li>
<li id="bib.bib311" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[311]</span>
<span class="ltx_bibblock">
Malechka VV, Duong D, Bordonada KD, Turriff A, Blain D, Murphy E, et al.

</span>
<span class="ltx_bibblock">Investigating determinants and evaluating deep learning training approaches for visual acuity in foveal hypoplasia.

</span>
<span class="ltx_bibblock">Ophthalmology Science. 2023;3(1):100225.

</span>
</li>
<li id="bib.bib312" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[312]</span>
<span class="ltx_bibblock">
Li S, Higashita R, Fu H, Li H, Niu J, Liu J.

</span>
<span class="ltx_bibblock">Content-Preserving Diffusion Model for Unsupervised AS-OCT Image Despeckling.

</span>
<span class="ltx_bibblock">In: International Conference on Medical Image Computing and Computer-Assisted Intervention. Springer; 2023. p. 660-70.

</span>
</li>
<li id="bib.bib313" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[313]</span>
<span class="ltx_bibblock">
Zhang S, Liu J, Hu B, Mao Z.

</span>
<span class="ltx_bibblock">GH-DDM: the generalized hybrid denoising diffusion model for medical image generation.

</span>
<span class="ltx_bibblock">Multimedia Systems. 2023;29(3):1335-45.

</span>
</li>
<li id="bib.bib314" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[314]</span>
<span class="ltx_bibblock">
Ren M, Dey N, Fishbaugh J, Gerig G.

</span>
<span class="ltx_bibblock">Segmentation-renormalized deep feature modulation for unpaired image harmonization.

</span>
<span class="ltx_bibblock">IEEE transactions on medical imaging. 2021;40(6):1519-30.

</span>
</li>
<li id="bib.bib315" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[315]</span>
<span class="ltx_bibblock">
Vidal PL, de Moura J, Novo J, Penedo MG, Ortega M.

</span>
<span class="ltx_bibblock">Image-to-image translation with generative adversarial networks via retinal masks for realistic optical coherence tomography imaging of diabetic macular edema disorders.

</span>
<span class="ltx_bibblock">Biomedical Signal Processing and Control. 2023;79:104098.

</span>
</li>
<li id="bib.bib316" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[316]</span>
<span class="ltx_bibblock">
Kugelman J, Alonso-Caneiro D, Read SA, Vincent SJ, Chen FK, Collins MJ.

</span>
<span class="ltx_bibblock">Data augmentation for patch-based OCT chorio-retinal segmentation using generative adversarial networks.

</span>
<span class="ltx_bibblock">Neural Computing and Applications. 2021;33(13):7393-408.

</span>
</li>
<li id="bib.bib317" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[317]</span>
<span class="ltx_bibblock">
Kugelman J, Alonso-Caneiro D, Read SA, Vincent SJ, Collins MJ.

</span>
<span class="ltx_bibblock">Enhanced OCT chorio-retinal segmentation in low-data settings with semi-supervised GAN augmentation using cross-localisation.

</span>
<span class="ltx_bibblock">Computer Vision and Image Understanding. 2023;237:103852.

</span>
</li>
<li id="bib.bib318" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[318]</span>
<span class="ltx_bibblock">
Zafar H, Zafar J, Sharif F.

</span>
<span class="ltx_bibblock">GANs-Based Intracoronary Optical Coherence Tomography Image Augmentation for Improved Plaques Characterization Using Deep Neural Networks.

</span>
<span class="ltx_bibblock">Optics. 2023;4(2):288-99.

</span>
</li>
<li id="bib.bib319" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[319]</span>
<span class="ltx_bibblock">
Kaplan S, Lensu L.

</span>
<span class="ltx_bibblock">Contrastive learning for generating optical coherence tomography images of the retina.

</span>
<span class="ltx_bibblock">In: International Workshop on Simulation and Synthesis in Medical Imaging. Springer; 2022. p. 112-21.

</span>
</li>
<li id="bib.bib320" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[320]</span>
<span class="ltx_bibblock">
Zhao P, Song X, Xi X, Nie X, Meng X, Qu Y, et al.

</span>
<span class="ltx_bibblock">Biomarkers-Aware Asymmetric Bibranch GAN with Adaptive Memory Batch Normalization for Prediction of Anti-VEGF Treatment Response in Neovascular Age-related Macular Degeneration.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics. 2023.

</span>
</li>
<li id="bib.bib321" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[321]</span>
<span class="ltx_bibblock">
Wang W, Li X, Xu Z, Yu W, Zhao J, Ding D, et al.

</span>
<span class="ltx_bibblock">Learning two-stream CNN for multi-modal age-related macular degeneration categorization.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics. 2022;26(8):4111-22.

</span>
</li>
<li id="bib.bib322" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[322]</span>
<span class="ltx_bibblock">
López-Varela E, Barreira N, Pascual NO, Castillo MRA, Penedo MG.

</span>
<span class="ltx_bibblock">Generation of synthetic intermediate slices in 3D OCT cubes for improving pathology detection and monitoring.

</span>
<span class="ltx_bibblock">Computers in Biology and Medicine. 2023;163:107214.

</span>
</li>
<li id="bib.bib323" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[323]</span>
<span class="ltx_bibblock">
Ge N, Liu Y, Xu X, Zhang X, Jiang M.

</span>
<span class="ltx_bibblock">A Fast Generative Adversarial Network for High-Fidelity Optical Coherence Tomography Image Synthesis.

</span>
<span class="ltx_bibblock">In: Photonics. vol. 9. MDPI; 2022. p. 944.

</span>
</li>
<li id="bib.bib324" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[324]</span>
<span class="ltx_bibblock">
Menten MJ, Holland R, Leingang O, Bogunović H, Hagag AM, Kaye R, et al.

</span>
<span class="ltx_bibblock">Exploring healthy retinal aging with deep learning.

</span>
<span class="ltx_bibblock">Ophthalmology Science. 2023;3(3):100294.

</span>
</li>
<li id="bib.bib325" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[325]</span>
<span class="ltx_bibblock">
Xu F, Yu X, Gao Y, Ning X, Huang Z, Wei M, et al.

</span>
<span class="ltx_bibblock">Predicting OCT images of short-term response to anti-VEGF treatment for retinal vein occlusion using generative adversarial network.

</span>
<span class="ltx_bibblock">Frontiers in Bioengineering and Biotechnology. 2022;10:914964.

</span>
</li>
<li id="bib.bib326" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[326]</span>
<span class="ltx_bibblock">
Sun Y, Wang J, Shi J, Boppart SA.

</span>
<span class="ltx_bibblock">Synthetic polarization-sensitive optical coherence tomography by deep learning.

</span>
<span class="ltx_bibblock">npj Digital Medicine. 2021;4(1):105.

</span>
</li>
<li id="bib.bib327" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[327]</span>
<span class="ltx_bibblock">
Mukherjee S, De Silva T, Jayakar G, Grisso P, Wiley H, Keenan T, et al.

</span>
<span class="ltx_bibblock">Device specific SD-OCT retinal layer segmentation using cycle-generative adversarial networks in patients with AMD.

</span>
<span class="ltx_bibblock">In: Medical Imaging 2022: Computer-Aided Diagnosis. vol. 12033. SPIE; 2022. p. 903-9.

</span>
</li>
<li id="bib.bib328" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[328]</span>
<span class="ltx_bibblock">
Chang CW, Chang CY, Lin YY, Su WW, Chen HSL.

</span>
<span class="ltx_bibblock">A glaucoma detection system based on generative adversarial network and incremental learning.

</span>
<span class="ltx_bibblock">Applied Sciences. 2023;13(4):2195.

</span>
</li>
<li id="bib.bib329" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[329]</span>
<span class="ltx_bibblock">
Melo T, Cardoso J, Carneiro Â, Campilho A, Mendonça AM.

</span>
<span class="ltx_bibblock">Oct image synthesis through deep generative models.

</span>
<span class="ltx_bibblock">In: 2023 IEEE 36th International Symposium on Computer-Based Medical Systems (CBMS). IEEE; 2023. p. 561-6.

</span>
</li>
<li id="bib.bib330" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[330]</span>
<span class="ltx_bibblock">
Akbar MU, Wang W, Eklund A.

</span>
<span class="ltx_bibblock">Beware of Diffusion Models for Synthesizing Medical Images – A Comparison with GANs in Terms of Memorizing Brain MRI and Chest x-Ray Images.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib331" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[331]</span>
<span class="ltx_bibblock">
Skandarani Y, Jodoin PM, Lalande A.

</span>
<span class="ltx_bibblock">GANs for Medical Image Synthesis: An Empirical Study [Article].

</span>
<span class="ltx_bibblock">Journal of Imaging. 2023;9(3).

</span>
</li>
<li id="bib.bib332" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[332]</span>
<span class="ltx_bibblock">
Müller-Franzes G, Niehues JM, Khader F, Arasteh ST, Haarburger C, Kuhl C, et al.

</span>
<span class="ltx_bibblock">A Multimodal Comparison of Latent Denoising Diffusion Probabilistic Models and Generative Adversarial Networks for Medical Image Synthesis.

</span>
<span class="ltx_bibblock">Scientific Reports. 2023;13(1):12098.

</span>
</li>
<li id="bib.bib333" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[333]</span>
<span class="ltx_bibblock">
Ktena, I ,  Wiles, O ,  Albuquerque, I , et al.. Generative Models Improve Fairness of Medical Classifiers under Distribution Shifts; 2023.

</span>
</li>
<li id="bib.bib334" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[334]</span>
<span class="ltx_bibblock">
Zhang Z, Yao L, Wang B, Jha D, Keles E, Medetalibeyoglu A, et al.. EMIT-Diff: Enhancing Medical Image Segmentation via Text-Guided Diffusion Model; 2023.

</span>
</li>
<li id="bib.bib335" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[335]</span>
<span class="ltx_bibblock">
de Wilde B, Saha A, ten Broek RPG, Huisman H. Medical Diffusion on a Budget: Textual Inversion for Medical Image Generation; 2023.

</span>
</li>
<li id="bib.bib336" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[336]</span>
<span class="ltx_bibblock">
Hung ALY, Zhao K, Zheng H, Yan R, Raman SS, Terzopoulos D, et al.

</span>
<span class="ltx_bibblock">Med-cDiff: Conditional Medical Image Generation with Diffusion Models.

</span>
<span class="ltx_bibblock">Bioengineering. 2023;10(11):1258.

</span>
</li>
<li id="bib.bib337" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[337]</span>
<span class="ltx_bibblock">
Lee S, Chung H, Park M, Park J, Ryu WS, Ye JC. Improving 3D Imaging with Pre-Trained Perpendicular 2D Diffusion Models; 2023.

</span>
</li>
<li id="bib.bib338" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[338]</span>
<span class="ltx_bibblock">
Kikuchi, T ,  Hanaoka, S ,  Nakao, T , et al.

</span>
<span class="ltx_bibblock">Method for Generating Synthetic Data Combining Chest Radiography Images with Tabular Clinical Information Using Dual Generative Models.

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib339" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[339]</span>
<span class="ltx_bibblock">
Xie S, Tu Z.

</span>
<span class="ltx_bibblock">Holistically-nested edge detection.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE international conference on computer vision; 2015. p. 1395-403.

</span>
</li>
<li id="bib.bib340" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[340]</span>
<span class="ltx_bibblock">
Gal R, Alaluf Y, Atzmon Y, Patashnik O, Bermano AH, Chechik G, et al.. An Image is Worth One Word: Personalizing Text-to-Image Generation using Textual Inversion; 2022.

</span>
</li>
<li id="bib.bib341" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[341]</span>
<span class="ltx_bibblock">
Kweon S, Kim J, Kim J, Im S, Cho E, Bae S, et al.

</span>
<span class="ltx_bibblock">Publicly Shareable Clinical Large Language Model Built on Synthetic Clinical Notes.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230900237. 2023.

</span>
</li>
<li id="bib.bib342" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[342]</span>
<span class="ltx_bibblock">
Hamed E, Sharif A, Eid A, Alfehaidi A, Alberry M, Ramish A.

</span>
<span class="ltx_bibblock">Advancing artificial intelligence for clinical knowledge retrieval: a case study using ChatGPT-4 and link retrieval plug-in to analyze diabetic ketoacidosis guidelines.

</span>
<span class="ltx_bibblock">Cureus. 2023;15(7).

</span>
</li>
<li id="bib.bib343" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[343]</span>
<span class="ltx_bibblock">
Li J, Zhou Y, Jiang X, Natarajan K, Pakhomov SV, Liu H, et al.

</span>
<span class="ltx_bibblock">Are synthetic clinical notes useful for real natural language processing tasks: A case study on clinical entity recognition.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association. 2021;28(10):2193-201.

</span>
</li>
<li id="bib.bib344" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[344]</span>
<span class="ltx_bibblock">
Zhou N, Wu Q, Wu Z, Marino S, Dinov ID.

</span>
<span class="ltx_bibblock">DataSifterText: Partially Synthetic Text Generation for Sensitive Clinical Notes.

</span>
<span class="ltx_bibblock">Journal of Medical Systems. 2022;46(12):96.

</span>
</li>
<li id="bib.bib345" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[345]</span>
<span class="ltx_bibblock">
Al Aziz MM, Ahmed T, Faequa T, Jiang X, Yao Y, Mohammed N.

</span>
<span class="ltx_bibblock">Differentially private medical texts generation using generative neural networks.

</span>
<span class="ltx_bibblock">ACM Transactions on Computing for Healthcare (HEALTH). 2021;3(1):1-27.

</span>
</li>
<li id="bib.bib346" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[346]</span>
<span class="ltx_bibblock">
Libbi CA, Trienes J, Trieschnigg D, Seifert C.

</span>
<span class="ltx_bibblock">Generating synthetic training data for supervised de-identification of electronic health records.

</span>
<span class="ltx_bibblock">Future Internet. 2021;13(5):136.

</span>
</li>
<li id="bib.bib347" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[347]</span>
<span class="ltx_bibblock">
Johnson AE, Pollard TJ, Shen L, Lehman LwH, Feng M, Ghassemi M, et al.

</span>
<span class="ltx_bibblock">MIMIC-III, a freely accessible critical care database.

</span>
<span class="ltx_bibblock">Scientific data. 2016;3(1):1-9.

</span>
</li>
<li id="bib.bib348" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[348]</span>
<span class="ltx_bibblock">
Johnson A, Pollard T, Horng S, Celi LA, Mark R. MIMIC-IV-Note: Deidentified free-text clinical notes. PhysioNet; 2023.

</span>
</li>
<li id="bib.bib349" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[349]</span>
<span class="ltx_bibblock">
Uzuner Ö, Luo Y, Szolovits P.

</span>
<span class="ltx_bibblock">Evaluating the state-of-the-art in automatic de-identification.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association. 2007;14(5):550-63.

</span>
</li>
<li id="bib.bib350" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[350]</span>
<span class="ltx_bibblock">
Henry S, Buchan K, Filannino M, Stubbs A, Uzuner O.

</span>
<span class="ltx_bibblock">2018 n2c2 shared task on adverse drug events and medication extraction in electronic health records.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association. 2020;27(1):3-12.

</span>
</li>
<li id="bib.bib351" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[351]</span>
<span class="ltx_bibblock">
Moon S, Pakhomov S, Liu N, Ryan JO, Melton GB.

</span>
<span class="ltx_bibblock">A sense inventory for clinical abbreviations and acronyms created using clinical notes and medical dictionary resources.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association. 2014;21(2):299-307.

</span>
</li>
<li id="bib.bib352" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[352]</span>
<span class="ltx_bibblock">
Zhao Z, Jin Q, Chen F, Peng T, Yu S.

</span>
<span class="ltx_bibblock">Pmc-patients: A large-scale dataset of patient summaries and relations for benchmarking retrieval-based clinical decision support systems.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:220213876. 2022.

</span>
</li>
<li id="bib.bib353" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[353]</span>
<span class="ltx_bibblock">
Taori R, Gulrajani I, Zhang T, Dubois Y, Li X, Guestrin C, et al.. Stanford alpaca: An instruction-following llama model; 2023.

</span>
</li>
<li id="bib.bib354" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[354]</span>
<span class="ltx_bibblock">
Han T, Adams LC, Papaioannou JM, Grundmann P, Oberhauser T, Löser A, et al.

</span>
<span class="ltx_bibblock">MedAlpaca–An Open-Source Collection of Medical Conversational AI Models and Training Data.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230408247. 2023.

</span>
</li>
<li id="bib.bib355" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[355]</span>
<span class="ltx_bibblock">
Yunxiang L, Zihan L, Kai Z, Ruilong D, You Z.

</span>
<span class="ltx_bibblock">Chatdoctor: A medical chat model fine-tuned on llama model using medical domain knowledge.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230314070. 2023.

</span>
</li>
<li id="bib.bib356" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[356]</span>
<span class="ltx_bibblock">
Papineni K, Roukos S, Ward T, Zhu WJ.

</span>
<span class="ltx_bibblock">Bleu: a method for automatic evaluation of machine translation.

</span>
<span class="ltx_bibblock">In: Proceedings of the 40th annual meeting of the Association for Computational Linguistics; 2002. p. 311-8.

</span>
</li>
<li id="bib.bib357" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[357]</span>
<span class="ltx_bibblock">
Rose S, Engel D, Cramer N, Cowley W.

</span>
<span class="ltx_bibblock">Automatic keyword extraction from individual documents.

</span>
<span class="ltx_bibblock">Text mining: applications and theory. 2010:1-20.

</span>
</li>
<li id="bib.bib358" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[358]</span>
<span class="ltx_bibblock">
Mihalcea R, Tarau P.

</span>
<span class="ltx_bibblock">Textrank: Bringing order into text.

</span>
<span class="ltx_bibblock">In: Proceedings of the 2004 conference on empirical methods in natural language processing; 2004. p. 404-11.

</span>
</li>
<li id="bib.bib359" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[359]</span>
<span class="ltx_bibblock">
Vaswani A, Shazeer N, Parmar N, Uszkoreit J, Jones L, Gomez AN, et al.

</span>
<span class="ltx_bibblock">Attention is all you need.

</span>
<span class="ltx_bibblock">Advances in neural information processing systems. 2017;30.

</span>
</li>
<li id="bib.bib360" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[360]</span>
<span class="ltx_bibblock">
Shibata H, Hanaoka S, Cao Y, Yoshikawa M, Takenaga T, Nomura Y, et al.

</span>
<span class="ltx_bibblock">Local differential privacy image generation using flow-based deep generative models.

</span>
<span class="ltx_bibblock">Applied Sciences. 2023;13(18):10132.

</span>
</li>
<li id="bib.bib361" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[361]</span>
<span class="ltx_bibblock">
Zhang Z, Yao L, Wang B, Jha D, Keles E, Medetalibeyoglu A, et al.

</span>
<span class="ltx_bibblock">EMIT-Diff: Enhancing Medical Image Segmentation via Text-Guided Diffusion Model.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:231012868. 2023.

</span>
</li>
<li id="bib.bib362" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[362]</span>
<span class="ltx_bibblock">
Menon S, Mangalagiri J, Galita J, Morris M, Saboury B, Yesha Y, et al.

</span>
<span class="ltx_bibblock">CCS-GAN: COVID-19 CT Scan Generation and Classification with Very Few Positive Training Images.

</span>
<span class="ltx_bibblock">Journal of Digital Imaging. 2023;36(4):1376-89.

</span>
</li>
<li id="bib.bib363" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[363]</span>
<span class="ltx_bibblock">
Lee EH, Kelly B, Altinmakas E, Dogan H, Mohammadzadeh M, Colak E, et al.

</span>
<span class="ltx_bibblock">AI Models Close to your Chest: Robust Federated Learning Strategies for Multi-site CT (preprint).

</span>
<span class="ltx_bibblock">arXiv. 2023.

</span>
</li>
<li id="bib.bib364" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[364]</span>
<span class="ltx_bibblock">
Liu Y, Hou YJ, Qin CX, Li XH, Li SJ, Wang B, et al.

</span>
<span class="ltx_bibblock">A Data Augmentation Method and the Embedding Mechanism for Detection and Classification of Pulmonary Nodules on Small Samples.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:230312801. 2023.

</span>
</li>
<li id="bib.bib365" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[365]</span>
<span class="ltx_bibblock">
Zhang X, Sisniega A, Zbijewski WB, Lee J, Jones CK, Wu P, et al.

</span>
<span class="ltx_bibblock">Combining physics-based models with deep learning image synthesis and uncertainty in intraoperative cone-beam CT of the brain.

</span>
<span class="ltx_bibblock">Medical physics. 2023;50(5):2607-24.

</span>
</li>
<li id="bib.bib366" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[366]</span>
<span class="ltx_bibblock">
Pesaranghader A, Wang Y, Havaei M.

</span>
<span class="ltx_bibblock">CT-SGAN: computed tomography synthesis GAN.

</span>
<span class="ltx_bibblock">In: Deep Generative Models, and Data Augmentation, Labelling, and Imperfections: First Workshop, DGM4MICCAI 2021, and First Workshop, DALI 2021, Held in Conjunction with MICCAI 2021, Strasbourg, France, October 1, 2021, Proceedings 1. Springer; 2021. p. 67-79.

</span>
</li>
<li id="bib.bib367" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[367]</span>
<span class="ltx_bibblock">
Estakhraji SIZ, Pirasteh A, Bradshaw T, McMillan A.

</span>
<span class="ltx_bibblock">On the effect of training database size for MR-based synthetic CT generation in the head.

</span>
<span class="ltx_bibblock">Computerized Medical Imaging and Graphics. 2023;107:102227.

</span>
</li>
<li id="bib.bib368" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[368]</span>
<span class="ltx_bibblock">
Liu S, Hu W, Xu F, Chen W, Liu J, Yu X, et al.

</span>
<span class="ltx_bibblock">Prediction of OCT images of short-term response to anti-VEGF treatment for diabetic macular edema using different generative adversarial networks.

</span>
<span class="ltx_bibblock">Photodiagnosis and Photodynamic Therapy. 2023;41:103272.

</span>
</li>
<li id="bib.bib369" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[369]</span>
<span class="ltx_bibblock">
Torfi A, Fox EA.

</span>
<span class="ltx_bibblock">CorGAN: Correlation-capturing Convolutional Generative Adversarial Networks for Generating Synthetic Healthcare Records.

</span>
<span class="ltx_bibblock">In: Proceedings of the 33rd International Florida Artificial Intelligence Research Society Conference, FLAIRS 2020; 2020. p. 335-40.

</span>
</li>
</ul>
</section>
<section id="S10" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">10 </span>Appendix</h2>

<section id="S10.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">10.1 </span>Evolution of the usage of generation techniques over years</h3>

<figure id="S10.F11" class="ltx_figure"><img src="/html/2407.00116/assets/EVOLUTION_OVER_YEARS.png" id="S10.F11.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="419" height="264" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Fig. 11: </span>Evolution of used techniques (GANs, DM, VAE, etc.) over years</figcaption>
</figure>
<div id="S10.SS1.p1" class="ltx_para">
<p id="S10.SS1.p1.1" class="ltx_p">Based on Fig.<a href="#S10.F11" title="Fig. 11 ‣ 10.1 Evolution of the usage of generation techniques over years ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">11</span></a>, we notice the following:</p>
<ul id="S10.I1" class="ltx_itemize">
<li id="S10.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S10.I1.i1.p1" class="ltx_para">
<p id="S10.I1.i1.p1.1" class="ltx_p"><span id="S10.I1.i1.p1.1.1" class="ltx_text ltx_font_bold">Dominance of GANs:</span> GANs consistently dominate the number of publications across all half-year periods. This indicates a strong and sustained interest in GANs within the research community,</p>
</div>
</li>
<li id="S10.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S10.I1.i2.p1" class="ltx_para">
<p id="S10.I1.i2.p1.1" class="ltx_p"><span id="S10.I1.i2.p1.1.1" class="ltx_text ltx_font_bold">Rising interest in Diffusion Models (DM):</span> There is a notable increase in publications related to Diffusion Models (DM), particularly in the latter half of 2022 and 2023. This suggests growing recognition of the potential of DM in generating high-quality synthetic data.</p>
</div>
</li>
<li id="S10.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span> 
<div id="S10.I1.i3.p1" class="ltx_para">
<p id="S10.I1.i3.p1.1" class="ltx_p"><span id="S10.I1.i3.p1.1.1" class="ltx_text ltx_font_bold">Emergence of novel synthetic data generation techniques:</span> Other model types, including LLMs and Transformers, show periodic spikes in publication numbers. This reflects the expanding scope of this field.</p>
</div>
</li>
</ul>
</div>
<figure id="S10.T15" class="ltx_table">
<table id="S10.T15.1" class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S10.T15.1.1.1" class="ltx_tr">
<th id="S10.T15.1.1.1.1" class="ltx_td ltx_align_justify ltx_align_top ltx_th ltx_th_column ltx_border_t">
<span id="S10.T15.1.1.1.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.1.1.1.1.1" class="ltx_p" style="width:97.6pt;"><span id="S10.T15.1.1.1.1.1.1.1" class="ltx_text ltx_font_bold">Privacy Risk</span></span>
</span>
</th>
<th id="S10.T15.1.1.1.2" class="ltx_td ltx_align_justify ltx_align_top ltx_th ltx_th_column ltx_border_t">
<span id="S10.T15.1.1.1.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.1.1.2.1.1" class="ltx_p" style="width:97.6pt;"><span id="S10.T15.1.1.1.2.1.1.1" class="ltx_text ltx_font_bold">References</span></span>
</span>
</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S10.T15.1.2.1" class="ltx_tr">
<td id="S10.T15.1.2.1.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T15.1.2.1.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.2.1.1.1.1" class="ltx_p" style="width:97.6pt;">Membership Inference</span>
</span>
</td>
<td id="S10.T15.1.2.1.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T15.1.2.1.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.2.1.2.1.1" class="ltx_p" style="width:97.6pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib369" title="" class="ltx_ref">369</a>, <a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib70" title="" class="ltx_ref">70</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>, <a href="#bib.bib73" title="" class="ltx_ref">73</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite>, Presence Disclosure <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref">71</a>]</cite></span>
</span>
</td>
</tr>
<tr id="S10.T15.1.3.2" class="ltx_tr">
<td id="S10.T15.1.3.2.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T15.1.3.2.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.3.2.1.1.1" class="ltx_p" style="width:97.6pt;">Attribute Inference Attack</span>
</span>
</td>
<td id="S10.T15.1.3.2.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T15.1.3.2.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.3.2.2.1.1" class="ltx_p" style="width:97.6pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib70" title="" class="ltx_ref">70</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>, <a href="#bib.bib73" title="" class="ltx_ref">73</a>]</cite></span>
</span>
</td>
</tr>
<tr id="S10.T15.1.4.3" class="ltx_tr">
<td id="S10.T15.1.4.3.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T15.1.4.3.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.4.3.1.1.1" class="ltx_p" style="width:97.6pt;">Nearest Neighbor Adversarial Accuracy Risk (nnaa)</span>
</span>
</td>
<td id="S10.T15.1.4.3.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T15.1.4.3.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.4.3.2.1.1" class="ltx_p" style="width:97.6pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib17" title="" class="ltx_ref">17</a>, <a href="#bib.bib54" title="" class="ltx_ref">54</a>, <a href="#bib.bib86" title="" class="ltx_ref">86</a>]</cite></span>
</span>
</td>
</tr>
<tr id="S10.T15.1.5.4" class="ltx_tr">
<td id="S10.T15.1.5.4.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T15.1.5.4.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.5.4.1.1.1" class="ltx_p" style="width:97.6pt;">Re-identification (or De-anonymization)</span>
</span>
</td>
<td id="S10.T15.1.5.4.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T15.1.5.4.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.5.4.2.1.1" class="ltx_p" style="width:97.6pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite></span>
</span>
</td>
</tr>
<tr id="S10.T15.1.6.5" class="ltx_tr">
<td id="S10.T15.1.6.5.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_b ltx_border_t">
<span id="S10.T15.1.6.5.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.6.5.1.1.1" class="ltx_p" style="width:97.6pt;">Other</span>
</span>
</td>
<td id="S10.T15.1.6.5.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_b ltx_border_t">
<span id="S10.T15.1.6.5.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T15.1.6.5.2.1.1" class="ltx_p" style="width:97.6pt;">Distance to Closest Record <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib18" title="" class="ltx_ref">18</a>]</cite>, Counting Identical Rows <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite>, Latent Space Nearest Neighbor <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib76" title="" class="ltx_ref">76</a>]</cite>, Re-identification Attack <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite>, Differential Privacy <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite>, KNN Estimation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib78" title="" class="ltx_ref">78</a>]</cite></span>
</span>
</td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 15: </span>Summary of Privacy Risk Metrics</figcaption>
</figure>
</section>
<section id="S10.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">10.2 </span>Synthetic Data Evaluation</h3>

<div id="S10.SS2.p1" class="ltx_para">
<p id="S10.SS2.p1.1" class="ltx_p">As mentioned in section <a href="#S2.SS3" title="2.3 Evaluation ‣ 2 Synthesis application, generative technologies and evaluations ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2.3</span></a>, synthetic data evaluation involves multiple dimensions. Below, more information on some of these dimensions is provided.</p>
</div>
<section id="S10.SS2.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">10.2.1 </span>Privacy Metrics</h4>

<div id="S10.SS2.SSS1.p1" class="ltx_para">
<p id="S10.SS2.SSS1.p1.1" class="ltx_p">Table <a href="#S10.T15" title="Table 15 ‣ 10.1 Evolution of the usage of generation techniques over years ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">15</span></a> in the appendix displays commonly used metrics and risks in evaluating privacy for EHR data.</p>
</div>
</section>
<section id="S10.SS2.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">10.2.2 </span>Fidelity Metrics</h4>

<div id="S10.SS2.SSS2.p1" class="ltx_para">
<p id="S10.SS2.SSS2.p1.1" class="ltx_p">The different fidelity metrics used in the surveyed papers are displayed in Tables <a href="#S10.T16" title="Table 16 ‣ Fidelity Metrics for EHR ‣ 10.2.2 Fidelity Metrics ‣ 10.2 Synthetic Data Evaluation ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">16</span></a> and <a href="#S10.T17" title="Table 17 ‣ Fidelity Metrics for EHR ‣ 10.2.2 Fidelity Metrics ‣ 10.2 Synthetic Data Evaluation ‣ 10 Appendix ‣ Generative AI for Synthetic Data Across Multiple Medical Modalities: A Systematic Review of Recent Developments and Challenges" class="ltx_ref"><span class="ltx_text ltx_ref_tag">17</span></a>. For the imaging data type, only top frequently used metrics are displayed.</p>
</div>
<section id="S10.SS2.SSS2.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_title_paragraph">Fidelity Metrics for EHR</h5>

<div id="S10.SS2.SSS2.Px1.p1" class="ltx_para">
<p id="S10.SS2.SSS2.Px1.p1.1" class="ltx_p">The different metrics used for evaluating EHR are categorized according to the following:</p>
</div>
<div id="S10.SS2.SSS2.Px1.p2" class="ltx_para">
<ul id="S10.I2" class="ltx_itemize">
<li id="S10.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S10.I2.i1.p1" class="ltx_para">
<p id="S10.I2.i1.p1.1" class="ltx_p"><span id="S10.I2.i1.p1.1.1" class="ltx_text ltx_font_bold">Dimension Wise distributional similarity:</span> This category of metrics focuses on the resemblance of the distribution of individual features between the real and synthetic datasets. <span id="S10.I2.i1.p1.1.2" class="ltx_text ltx_font_bold">Dimension-wise probability</span> is a family of tests that compare the distribution of features of synthetic data to those of real data. Depending on the type of the feature, different tests and metrics are used. <span id="S10.I2.i1.p1.1.3" class="ltx_text ltx_font_italic">Bernoulli Success probability</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib369" title="" class="ltx_ref">369</a>, <a href="#bib.bib84" title="" class="ltx_ref">84</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite> ,<span id="S10.I2.i1.p1.1.4" class="ltx_text ltx_font_italic">Pearson Chi-square test</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib90" title="" class="ltx_ref">90</a>, <a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite>, and <span id="S10.I2.i1.p1.1.5" class="ltx_text ltx_font_italic">analysis of variance F-test</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> are commonly used for binary features (which is the case when ICD-9 codes are present), while <span id="S10.I2.i1.p1.1.6" class="ltx_text ltx_font_italic">Student T-test</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite> and <span id="S10.I2.i1.p1.1.7" class="ltx_text ltx_font_italic">Snedecor’s F-test</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite>. is used for continuous features. <span id="S10.I2.i1.p1.1.8" class="ltx_text ltx_font_bold">Dimension-wise distribution</span> is another term used in the literature <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib83" title="" class="ltx_ref">83</a>]</cite> that focuses on evaluating the resemblance between synthetic and real data at the feature level. Although these publications use the same term, they refer to slightly different aspects and methodologies. Both descriptions use the means to compare the real and synthetic binary features. However, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib83" title="" class="ltx_ref">83</a>]</cite> extends the evaluation by using the 1-Wasserstein distance for continuous values. The <span id="S10.I2.i1.p1.1.9" class="ltx_text ltx_font_italic">Kolmogorov–Smirnov Test</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>, <a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite> is also commonly used to check if the real and synthetic data come from the same distribution. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> use <span id="S10.I2.i1.p1.1.10" class="ltx_text ltx_font_italic">three sigma rule test<cite class="ltx_cite ltx_citemacro_cite"><span id="S10.I2.i1.p1.1.10.1.1" class="ltx_text ltx_font_upright">[</span><a href="#bib.bib57" title="" class="ltx_ref">57</a><span id="S10.I2.i1.p1.1.10.2.2" class="ltx_text ltx_font_upright">]</span></cite></span> to check if the values of the synthetic data are within realistic ranges of the real data, especially when the KS-test fail. The <span id="S10.I2.i1.p1.1.11" class="ltx_text ltx_font_italic">Kernel Density Estimation <cite class="ltx_cite ltx_citemacro_cite"><span id="S10.I2.i1.p1.1.11.1.1" class="ltx_text ltx_font_upright">[</span><a href="#bib.bib84" title="" class="ltx_ref">84</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a><span id="S10.I2.i1.p1.1.11.2.2" class="ltx_text ltx_font_upright">]</span></cite></span> is used as well to compare the probability density functions of the real and synthetic data. The <span id="S10.I2.i1.p1.1.12" class="ltx_text ltx_font_italic">Kullback Leibler Divergence</span> (KLD) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> is a divergence measure that is also used to measure the divergence of the synthetic distribution from the real one. <span id="S10.I2.i1.p1.1.13" class="ltx_text ltx_font_bold">Dimension-wise statistics</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib70" title="" class="ltx_ref">70</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite> is widely used, where the univariate statistics (mean, standard deviations..) of the synthetic and real features are compared.</p>
</div>
</li>
<li id="S10.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S10.I2.i2.p1" class="ltx_para">
<p id="S10.I2.i2.p1.1" class="ltx_p"><span id="S10.I2.i2.p1.1.1" class="ltx_text ltx_font_bold">Joint Distribution Similarity</span>: assesses how well the synthetic data replicates the real data in terms of the overall distribution that encompasses all features simultaneously. It’s about the global structure of the data, ensuring that the synthetic data captures the complex, multi-dimensional relationships as they exist in the real data. In this sense, multiple metrics are used, like the <span id="S10.I2.i2.p1.1.2" class="ltx_text ltx_font_italic">Jensen-Shannon Divergence</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib85" title="" class="ltx_ref">85</a>, <a href="#bib.bib58" title="" class="ltx_ref">58</a>]</cite> which is a symmetrical extension of the KLD. <span id="S10.I2.i2.p1.1.3" class="ltx_text ltx_font_italic">Maximum-Mean Discrepancy</span> measures the dissimilarity between two probability distributions and is widely used <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite>. Measures like the <span id="S10.I2.i2.p1.1.4" class="ltx_text ltx_font_italic">Wasserstein distance</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite>, the <span id="S10.I2.i2.p1.1.5" class="ltx_text ltx_font_italic">Cross-type Conditional Distribution</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite>, and<span id="S10.I2.i2.p1.1.6" class="ltx_text ltx_font_italic"> first-order proximity</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite> are also used, each with a different way to measure the joint distributional similarity. The discriminative score, usually called propensity score as well, measures the overall resemblance of the synthetic data to the real data without explicitly calculating statistical distances. It involves a post-hoc classifier that tries to discriminate the real samples from the synthetic ones.</p>
</div>
</li>
<li id="S10.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span> 
<div id="S10.I2.i3.p1" class="ltx_para">
<p id="S10.I2.i3.p1.1" class="ltx_p"><span id="S10.I2.i3.p1.1.1" class="ltx_text ltx_font_bold">Inter-dimensional Relationship Similarity</span>: focuses on the relationships and correlations between pairs or groups of features within the data. It evaluates how well the synthetic data maintains the dependencies and interactions between different features, as observed in the real data. Correlation-based metrics like <span id="S10.I2.i3.p1.1.2" class="ltx_text ltx_font_italic">Pearson pairwise correlation</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref">71</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite>, <span id="S10.I2.i3.p1.1.3" class="ltx_text ltx_font_italic">pairwise correlation difference</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite>, <span id="S10.I2.i3.p1.1.4" class="ltx_text ltx_font_italic">Kendall’s tau rank correlation</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> tend to capture if the inter-dimensional correlations of the real-data are maintained in the synthetic data. <span id="S10.I2.i3.p1.1.5" class="ltx_text ltx_font_italic">Dimension-wise prediction test </span>is a widely common used method <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib369" title="" class="ltx_ref">369</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite> that is said to catch the quality and inter-dimensional relationships of features of the synthetic data. This metric is calculated by systematically selecting each feature as the target in turn, while utilizing the remaining features to train a classifier. Such an approach offers a more holistic evaluation compared to the conventional prediction utility metric, which is usually limited to specific downstream tasks. This metric is more comprehensive than the standard one, as it can assess the utility of all features, including those that may not significantly influence the task-specific outcome, thus providing a broader insight into the data’s overall quality and utility. <span id="S10.I2.i3.p1.1.6" class="ltx_text ltx_font_italic">Frequent Association Rules</span>, as used in <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite>, are employed to verify whether the association rules identified in the real data remain applicable in the synthetic data. It ensures that the underlying relationships and associations of the real-data are maintained.</p>
</div>
</li>
<li id="S10.I2.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">4.</span> 
<div id="S10.I2.i4.p1" class="ltx_para">
<p id="S10.I2.i4.p1.1" class="ltx_p"><span id="S10.I2.i4.p1.1.1" class="ltx_text ltx_font_bold">Latent Distribution similarity:</span> This family of metrics tend to evaluate if the distribution of synthetic and real data is similar as well in the latent space. The <span id="S10.I2.i4.p1.1.2" class="ltx_text ltx_font_italic">log-cluster metric</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite> measures the clustering pattern similarities between the real and synthetic data. <span id="S10.I2.i4.p1.1.3" class="ltx_text ltx_font_italic">Latent space representation<cite class="ltx_cite ltx_citemacro_cite"><span id="S10.I2.i4.p1.1.3.1.1" class="ltx_text ltx_font_upright">[</span><a href="#bib.bib70" title="" class="ltx_ref">70</a><span id="S10.I2.i4.p1.1.3.2.2" class="ltx_text ltx_font_upright">]</span></cite></span> and latent space evaluation <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite> metrics also fall into this family.</p>
</div>
</li>
<li id="S10.I2.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">5.</span> 
<div id="S10.I2.i5.p1" class="ltx_para">
<p id="S10.I2.i5.p1.1" class="ltx_p"><span id="S10.I2.i5.p1.1.1" class="ltx_text ltx_font_bold">Special Fidelity metrics</span>: Other metrics that fall under the fidelity category tend to evaluate special features of the synthesis process, like the diversity of the synthetic data. Diversity is the ability to cover the variability of the synthetic data. For example, <span id="S10.I2.i5.p1.1.2" class="ltx_text ltx_font_italic">category coverage</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> and <span id="S10.I2.i5.p1.1.3" class="ltx_text ltx_font_italic">Generated Disease types<cite class="ltx_cite ltx_citemacro_cite"><span id="S10.I2.i5.p1.1.3.1.1" class="ltx_text ltx_font_upright">[</span><a href="#bib.bib58" title="" class="ltx_ref">58</a><span id="S10.I2.i5.p1.1.3.2.2" class="ltx_text ltx_font_upright">]</span></cite></span> are concerned with the presence and representation of all possible categories, usually diseases and medical codes, in the synthetic data. <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite> focuses on generating uncommon diseases, and as a result, introduced (1) <span id="S10.I2.i5.p1.1.4" class="ltx_text ltx_font_italic">Medical Concept Abundance</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite> that evaluates how diverse and representative these medical codes are in the synthetic data compared to the real data., (2) <span id="S10.I2.i5.p1.1.5" class="ltx_text ltx_font_italic">Required sample number to generate all diseases (RN)</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite> to evaluate the ability to generate uncommon diseases, and (3) <span id="S10.I2.i5.p1.1.6" class="ltx_text ltx_font_italic">Normalized distance</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite> to further evaluate the distribution of uncommon diseases. Some other metrics that focus on the quality of generating time-series synthetic data, are available, like <span id="S10.I2.i5.p1.1.7" class="ltx_text ltx_font_italic">autocorrelation</span> function <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite> and <span id="S10.I2.i5.p1.1.8" class="ltx_text ltx_font_italic">patient trajectories</span> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite>.</p>
</div>
</li>
</ul>
</div>
<figure id="S10.T16" class="ltx_table">
<div id="S10.T16.1" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:826.6pt;height:627pt;vertical-align:-0.0pt;"><span class="ltx_transformed_inner" style="width:627.0pt;transform:translate(99.76pt,100.26pt) rotate(-90deg) ;">
<table id="S10.T16.1.1" class="ltx_tabular ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S10.T16.1.1.1.1" class="ltx_tr">
<td id="S10.T16.1.1.1.1.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt">
<span id="S10.T16.1.1.1.1.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.1.1.1.1.1" class="ltx_p" style="width:43.4pt;">Modality</span>
</span>
</td>
<td id="S10.T16.1.1.1.1.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt">
<span id="S10.T16.1.1.1.1.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.1.1.2.1.1" class="ltx_p" style="width:86.7pt;">Purpose of Evaluation</span>
</span>
</td>
<td id="S10.T16.1.1.1.1.3" class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt">
<span id="S10.T16.1.1.1.1.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.1.1.3.1.1" class="ltx_p" style="width:130.1pt;">Fidelity Metric</span>
</span>
</td>
<td id="S10.T16.1.1.1.1.4" class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt">
<span id="S10.T16.1.1.1.1.4.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.1.1.4.1.1" class="ltx_p" style="width:281.9pt;">Usage</span>
</span>
</td>
<td id="S10.T16.1.1.1.1.5" class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt">
<span id="S10.T16.1.1.1.1.5.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.1.1.5.1.1" class="ltx_p" style="width:21.7pt;">Count</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.2.2" class="ltx_tr">
<td id="S10.T16.1.1.2.2.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_b ltx_border_t" rowspan="21">
<span id="S10.T16.1.1.2.2.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.2.2.1.1.1" class="ltx_p" style="width:43.4pt;"><span id="S10.T16.1.1.2.2.1.1.1.1" class="ltx_text">Imaging</span></span>
</span>
</td>
<td id="S10.T16.1.1.2.2.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" rowspan="11">
<span id="S10.T16.1.1.2.2.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.2.2.2.1.1" class="ltx_p" style="width:86.7pt;"><span id="S10.T16.1.1.2.2.2.1.1.1" class="ltx_text">
<span id="S10.T16.1.1.2.2.2.1.1.1.1" class="ltx_inline-block ltx_parbox ltx_align_middle" style="width:78.0pt;">
<span id="S10.T16.1.1.2.2.2.1.1.1.1.1" class="ltx_p">Pixel-wise similarity (image based)</span>
</span></span></span>
</span>
</td>
<td id="S10.T16.1.1.2.2.3" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.2.2.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.2.2.3.1.1" class="ltx_p" style="width:130.1pt;">Structural Similarity Index (SSIM)</span>
</span>
</td>
<td id="S10.T16.1.1.2.2.4" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.2.2.4.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.2.2.4.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib174" title="" class="ltx_ref">174</a>, <a href="#bib.bib180" title="" class="ltx_ref">180</a>, <a href="#bib.bib181" title="" class="ltx_ref">181</a>, <a href="#bib.bib182" title="" class="ltx_ref">182</a>, <a href="#bib.bib184" title="" class="ltx_ref">184</a>, <a href="#bib.bib185" title="" class="ltx_ref">185</a>, <a href="#bib.bib156" title="" class="ltx_ref">156</a>, <a href="#bib.bib159" title="" class="ltx_ref">159</a>, <a href="#bib.bib160" title="" class="ltx_ref">160</a>, <a href="#bib.bib204" title="" class="ltx_ref">204</a>, <a href="#bib.bib196" title="" class="ltx_ref">196</a>, <a href="#bib.bib218" title="" class="ltx_ref">218</a>, <a href="#bib.bib203" title="" class="ltx_ref">203</a>, <a href="#bib.bib197" title="" class="ltx_ref">197</a>, <a href="#bib.bib219" title="" class="ltx_ref">219</a>, <a href="#bib.bib193" title="" class="ltx_ref">193</a>, <a href="#bib.bib220" title="" class="ltx_ref">220</a>, <a href="#bib.bib201" title="" class="ltx_ref">201</a>, <a href="#bib.bib194" title="" class="ltx_ref">194</a>, <a href="#bib.bib222" title="" class="ltx_ref">222</a>, <a href="#bib.bib216" title="" class="ltx_ref">216</a>, <a href="#bib.bib200" title="" class="ltx_ref">200</a>, <a href="#bib.bib221" title="" class="ltx_ref">221</a>, <a href="#bib.bib195" title="" class="ltx_ref">195</a>, <a href="#bib.bib226" title="" class="ltx_ref">226</a>, <a href="#bib.bib337" title="" class="ltx_ref">337</a>, <a href="#bib.bib254" title="" class="ltx_ref">254</a>, <a href="#bib.bib242" title="" class="ltx_ref">242</a>, <a href="#bib.bib274" title="" class="ltx_ref">274</a>, <a href="#bib.bib252" title="" class="ltx_ref">252</a>, <a href="#bib.bib253" title="" class="ltx_ref">253</a>, <a href="#bib.bib261" title="" class="ltx_ref">261</a>, <a href="#bib.bib275" title="" class="ltx_ref">275</a>, <a href="#bib.bib276" title="" class="ltx_ref">276</a>, <a href="#bib.bib263" title="" class="ltx_ref">263</a>, <a href="#bib.bib278" title="" class="ltx_ref">278</a>, <a href="#bib.bib265" title="" class="ltx_ref">265</a>, <a href="#bib.bib256" title="" class="ltx_ref">256</a>, <a href="#bib.bib273" title="" class="ltx_ref">273</a>, <a href="#bib.bib246" title="" class="ltx_ref">246</a>, <a href="#bib.bib271" title="" class="ltx_ref">271</a>, <a href="#bib.bib260" title="" class="ltx_ref">260</a>, <a href="#bib.bib67" title="" class="ltx_ref">67</a>, <a href="#bib.bib266" title="" class="ltx_ref">266</a>, <a href="#bib.bib309" title="" class="ltx_ref">309</a>, <a href="#bib.bib68" title="" class="ltx_ref">68</a>, <a href="#bib.bib314" title="" class="ltx_ref">314</a>, <a href="#bib.bib326" title="" class="ltx_ref">326</a>, <a href="#bib.bib328" title="" class="ltx_ref">328</a>, <a href="#bib.bib299" title="" class="ltx_ref">299</a>, <a href="#bib.bib292" title="" class="ltx_ref">292</a>, <a href="#bib.bib304" title="" class="ltx_ref">304</a>, <a href="#bib.bib306" title="" class="ltx_ref">306</a>, <a href="#bib.bib293" title="" class="ltx_ref">293</a>, <a href="#bib.bib288" title="" class="ltx_ref">288</a>, <a href="#bib.bib295" title="" class="ltx_ref">295</a>, <a href="#bib.bib290" title="" class="ltx_ref">290</a>, <a href="#bib.bib291" title="" class="ltx_ref">291</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.2.2.5" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.2.2.5.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.2.2.5.1.1" class="ltx_p" style="width:21.7pt;">58</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.3.3" class="ltx_tr">
<td id="S10.T16.1.1.3.3.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.3.3.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.3.3.1.1.1" class="ltx_p" style="width:130.1pt;">Peak Signal-to-Noise Ratio (PSNR)</span>
</span>
</td>
<td id="S10.T16.1.1.3.3.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.3.3.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.3.3.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib181" title="" class="ltx_ref">181</a>, <a href="#bib.bib184" title="" class="ltx_ref">184</a>, <a href="#bib.bib185" title="" class="ltx_ref">185</a>, <a href="#bib.bib156" title="" class="ltx_ref">156</a>, <a href="#bib.bib159" title="" class="ltx_ref">159</a>, <a href="#bib.bib160" title="" class="ltx_ref">160</a>, <a href="#bib.bib204" title="" class="ltx_ref">204</a>, <a href="#bib.bib202" title="" class="ltx_ref">202</a>, <a href="#bib.bib196" title="" class="ltx_ref">196</a>, <a href="#bib.bib203" title="" class="ltx_ref">203</a>, <a href="#bib.bib197" title="" class="ltx_ref">197</a>, <a href="#bib.bib219" title="" class="ltx_ref">219</a>, <a href="#bib.bib224" title="" class="ltx_ref">224</a>, <a href="#bib.bib193" title="" class="ltx_ref">193</a>, <a href="#bib.bib220" title="" class="ltx_ref">220</a>, <a href="#bib.bib201" title="" class="ltx_ref">201</a>, <a href="#bib.bib194" title="" class="ltx_ref">194</a>, <a href="#bib.bib216" title="" class="ltx_ref">216</a>, <a href="#bib.bib200" title="" class="ltx_ref">200</a>, <a href="#bib.bib221" title="" class="ltx_ref">221</a>, <a href="#bib.bib195" title="" class="ltx_ref">195</a>, <a href="#bib.bib226" title="" class="ltx_ref">226</a>, <a href="#bib.bib337" title="" class="ltx_ref">337</a>, <a href="#bib.bib274" title="" class="ltx_ref">274</a>, <a href="#bib.bib252" title="" class="ltx_ref">252</a>, <a href="#bib.bib253" title="" class="ltx_ref">253</a>, <a href="#bib.bib259" title="" class="ltx_ref">259</a>, <a href="#bib.bib261" title="" class="ltx_ref">261</a>, <a href="#bib.bib275" title="" class="ltx_ref">275</a>, <a href="#bib.bib276" title="" class="ltx_ref">276</a>, <a href="#bib.bib263" title="" class="ltx_ref">263</a>, <a href="#bib.bib277" title="" class="ltx_ref">277</a>, <a href="#bib.bib278" title="" class="ltx_ref">278</a>, <a href="#bib.bib264" title="" class="ltx_ref">264</a>, <a href="#bib.bib265" title="" class="ltx_ref">265</a>, <a href="#bib.bib256" title="" class="ltx_ref">256</a>, <a href="#bib.bib273" title="" class="ltx_ref">273</a>, <a href="#bib.bib267" title="" class="ltx_ref">267</a>, <a href="#bib.bib260" title="" class="ltx_ref">260</a>, <a href="#bib.bib67" title="" class="ltx_ref">67</a>, <a href="#bib.bib266" title="" class="ltx_ref">266</a>, <a href="#bib.bib309" title="" class="ltx_ref">309</a>, <a href="#bib.bib68" title="" class="ltx_ref">68</a>, <a href="#bib.bib328" title="" class="ltx_ref">328</a>, <a href="#bib.bib292" title="" class="ltx_ref">292</a>, <a href="#bib.bib302" title="" class="ltx_ref">302</a>, <a href="#bib.bib304" title="" class="ltx_ref">304</a>, <a href="#bib.bib306" title="" class="ltx_ref">306</a>, <a href="#bib.bib288" title="" class="ltx_ref">288</a>, <a href="#bib.bib295" title="" class="ltx_ref">295</a>, <a href="#bib.bib291" title="" class="ltx_ref">291</a>, <a href="#bib.bib242" title="" class="ltx_ref">242</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.3.3.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.3.3.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.3.3.3.1.1" class="ltx_p" style="width:21.7pt;">52</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.4.4" class="ltx_tr">
<td id="S10.T16.1.1.4.4.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.4.4.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.4.4.1.1.1" class="ltx_p" style="width:130.1pt;">Mean Absolute Error (MAE)</span>
</span>
</td>
<td id="S10.T16.1.1.4.4.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.4.4.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.4.4.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib220" title="" class="ltx_ref">220</a>, <a href="#bib.bib226" title="" class="ltx_ref">226</a>, <a href="#bib.bib334" title="" class="ltx_ref">334</a>, <a href="#bib.bib112" title="" class="ltx_ref">112</a>, <a href="#bib.bib252" title="" class="ltx_ref">252</a>, <a href="#bib.bib261" title="" class="ltx_ref">261</a>, <a href="#bib.bib263" title="" class="ltx_ref">263</a>, <a href="#bib.bib278" title="" class="ltx_ref">278</a>, <a href="#bib.bib264" title="" class="ltx_ref">264</a>, <a href="#bib.bib265" title="" class="ltx_ref">265</a>, <a href="#bib.bib256" title="" class="ltx_ref">256</a>, <a href="#bib.bib273" title="" class="ltx_ref">273</a>, <a href="#bib.bib267" title="" class="ltx_ref">267</a>, <a href="#bib.bib269" title="" class="ltx_ref">269</a>, <a href="#bib.bib271" title="" class="ltx_ref">271</a>, <a href="#bib.bib316" title="" class="ltx_ref">316</a>, <a href="#bib.bib317" title="" class="ltx_ref">317</a>, <a href="#bib.bib292" title="" class="ltx_ref">292</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.4.4.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.4.4.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.4.4.3.1.1" class="ltx_p" style="width:21.7pt;">18</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.5.5" class="ltx_tr">
<td id="S10.T16.1.1.5.5.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.5.5.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.5.5.1.1.1" class="ltx_p" style="width:130.1pt;">Multi-Scale Structural Similarity Index (MS-SSIM)</span>
</span>
</td>
<td id="S10.T16.1.1.5.5.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.5.5.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.5.5.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib172" title="" class="ltx_ref">172</a>, <a href="#bib.bib182" title="" class="ltx_ref">182</a>, <a href="#bib.bib183" title="" class="ltx_ref">183</a>, <a href="#bib.bib157" title="" class="ltx_ref">157</a>, <a href="#bib.bib210" title="" class="ltx_ref">210</a>, <a href="#bib.bib214" title="" class="ltx_ref">214</a>, <a href="#bib.bib202" title="" class="ltx_ref">202</a>, <a href="#bib.bib225" title="" class="ltx_ref">225</a>, <a href="#bib.bib334" title="" class="ltx_ref">334</a>, <a href="#bib.bib128" title="" class="ltx_ref">128</a>, <a href="#bib.bib277" title="" class="ltx_ref">277</a>, <a href="#bib.bib69" title="" class="ltx_ref">69</a>, <a href="#bib.bib328" title="" class="ltx_ref">328</a>, <a href="#bib.bib286" title="" class="ltx_ref">286</a>, <a href="#bib.bib283" title="" class="ltx_ref">283</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.5.5.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.5.5.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.5.5.3.1.1" class="ltx_p" style="width:21.7pt;">15</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.6.6" class="ltx_tr">
<td id="S10.T16.1.1.6.6.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.6.6.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.6.6.1.1.1" class="ltx_p" style="width:130.1pt;">Mean Squared Error (MSE)</span>
</span>
</td>
<td id="S10.T16.1.1.6.6.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.6.6.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.6.6.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib185" title="" class="ltx_ref">185</a>, <a href="#bib.bib187" title="" class="ltx_ref">187</a>, <a href="#bib.bib210" title="" class="ltx_ref">210</a>, <a href="#bib.bib334" title="" class="ltx_ref">334</a>, <a href="#bib.bib112" title="" class="ltx_ref">112</a>, <a href="#bib.bib253" title="" class="ltx_ref">253</a>, <a href="#bib.bib275" title="" class="ltx_ref">275</a>, <a href="#bib.bib67" title="" class="ltx_ref">67</a>, <a href="#bib.bib314" title="" class="ltx_ref">314</a>, <a href="#bib.bib302" title="" class="ltx_ref">302</a>, <a href="#bib.bib304" title="" class="ltx_ref">304</a>, <a href="#bib.bib295" title="" class="ltx_ref">295</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.6.6.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.6.6.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.6.6.3.1.1" class="ltx_p" style="width:21.7pt;">12</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.7.7" class="ltx_tr">
<td id="S10.T16.1.1.7.7.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.7.7.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.7.7.1.1.1" class="ltx_p" style="width:130.1pt;">Root Mean Square Error (RMSE)</span>
</span>
</td>
<td id="S10.T16.1.1.7.7.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.7.7.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.7.7.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib195" title="" class="ltx_ref">195</a>, <a href="#bib.bib334" title="" class="ltx_ref">334</a>, <a href="#bib.bib122" title="" class="ltx_ref">122</a>, <a href="#bib.bib112" title="" class="ltx_ref">112</a>, <a href="#bib.bib115" title="" class="ltx_ref">115</a>, <a href="#bib.bib120" title="" class="ltx_ref">120</a>, <a href="#bib.bib254" title="" class="ltx_ref">254</a>, <a href="#bib.bib242" title="" class="ltx_ref">242</a>, <a href="#bib.bib252" title="" class="ltx_ref">252</a>, <a href="#bib.bib269" title="" class="ltx_ref">269</a>, <a href="#bib.bib271" title="" class="ltx_ref">271</a>, <a href="#bib.bib302" title="" class="ltx_ref">302</a>, <a href="#bib.bib255" title="" class="ltx_ref">255</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.7.7.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.7.7.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.7.7.3.1.1" class="ltx_p" style="width:21.7pt;">13</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.8.8" class="ltx_tr">
<td id="S10.T16.1.1.8.8.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.8.8.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.8.8.1.1.1" class="ltx_p" style="width:130.1pt;">Normalized Mean Squared Error (NMSE)</span>
</span>
</td>
<td id="S10.T16.1.1.8.8.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.8.8.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.8.8.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib202" title="" class="ltx_ref">202</a>, <a href="#bib.bib203" title="" class="ltx_ref">203</a>, <a href="#bib.bib197" title="" class="ltx_ref">197</a>, <a href="#bib.bib224" title="" class="ltx_ref">224</a>, <a href="#bib.bib260" title="" class="ltx_ref">260</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.8.8.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.8.8.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.8.8.3.1.1" class="ltx_p" style="width:21.7pt;">5</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.9.9" class="ltx_tr">
<td id="S10.T16.1.1.9.9.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.9.9.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.9.9.1.1.1" class="ltx_p" style="width:130.1pt;">Normalized Root Mean Square Error (NRMSE)</span>
</span>
</td>
<td id="S10.T16.1.1.9.9.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.9.9.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.9.9.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib204" title="" class="ltx_ref">204</a>, <a href="#bib.bib196" title="" class="ltx_ref">196</a>, <a href="#bib.bib193" title="" class="ltx_ref">193</a>, <a href="#bib.bib216" title="" class="ltx_ref">216</a>, <a href="#bib.bib221" title="" class="ltx_ref">221</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.9.9.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.9.9.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.9.9.3.1.1" class="ltx_p" style="width:21.7pt;">5</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.10.10" class="ltx_tr">
<td id="S10.T16.1.1.10.10.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.10.10.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.10.10.1.1.1" class="ltx_p" style="width:130.1pt;">Mean Error (ME)</span>
</span>
</td>
<td id="S10.T16.1.1.10.10.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.10.10.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.10.10.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib264" title="" class="ltx_ref">264</a>, <a href="#bib.bib267" title="" class="ltx_ref">267</a>, <a href="#bib.bib269" title="" class="ltx_ref">269</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.10.10.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.10.10.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.10.10.3.1.1" class="ltx_p" style="width:21.7pt;">3</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.11.11" class="ltx_tr">
<td id="S10.T16.1.1.11.11.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.11.11.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.11.11.1.1.1" class="ltx_p" style="width:130.1pt;">Universal Quality Index (UQI)</span>
</span>
</td>
<td id="S10.T16.1.1.11.11.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.11.11.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.11.11.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib266" title="" class="ltx_ref">266</a>, <a href="#bib.bib290" title="" class="ltx_ref">290</a>, <a href="#bib.bib302" title="" class="ltx_ref">302</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.11.11.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.11.11.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.11.11.3.1.1" class="ltx_p" style="width:21.7pt;">3</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.12.12" class="ltx_tr">
<td id="S10.T16.1.1.12.12.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.12.12.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.12.12.1.1.1" class="ltx_p" style="width:130.1pt;">contrast noise ratio (CNR)</span>
</span>
</td>
<td id="S10.T16.1.1.12.12.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.12.12.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.12.12.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib308" title="" class="ltx_ref">308</a>, <a href="#bib.bib62" title="" class="ltx_ref">62</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.12.12.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.12.12.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.12.12.3.1.1" class="ltx_p" style="width:21.7pt;">2</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.13.13" class="ltx_tr">
<td id="S10.T16.1.1.13.13.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" rowspan="7">
<span id="S10.T16.1.1.13.13.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.13.13.1.1.1" class="ltx_p" style="width:86.7pt;"><span id="S10.T16.1.1.13.13.1.1.1.1" class="ltx_text">
<span id="S10.T16.1.1.13.13.1.1.1.1.1" class="ltx_inline-block ltx_parbox ltx_align_middle" style="width:78.0pt;">
<span id="S10.T16.1.1.13.13.1.1.1.1.1.1" class="ltx_p">Feature-wise similarity (distrubution based)</span>
</span></span></span>
</span>
</td>
<td id="S10.T16.1.1.13.13.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.13.13.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.13.13.2.1.1" class="ltx_p" style="width:130.1pt;">Frechet Inception Distance (FID)</span>
</span>
</td>
<td id="S10.T16.1.1.13.13.3" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.13.13.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.13.13.3.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib139" title="" class="ltx_ref">139</a>, <a href="#bib.bib140" title="" class="ltx_ref">140</a>, <a href="#bib.bib141" title="" class="ltx_ref">141</a>, <a href="#bib.bib148" title="" class="ltx_ref">148</a>, <a href="#bib.bib142" title="" class="ltx_ref">142</a>, <a href="#bib.bib143" title="" class="ltx_ref">143</a>, <a href="#bib.bib171" title="" class="ltx_ref">171</a>, <a href="#bib.bib173" title="" class="ltx_ref">173</a>, <a href="#bib.bib174" title="" class="ltx_ref">174</a>, <a href="#bib.bib175" title="" class="ltx_ref">175</a>, <a href="#bib.bib181" title="" class="ltx_ref">181</a>, <a href="#bib.bib182" title="" class="ltx_ref">182</a>, <a href="#bib.bib183" title="" class="ltx_ref">183</a>, <a href="#bib.bib185" title="" class="ltx_ref">185</a>, <a href="#bib.bib156" title="" class="ltx_ref">156</a>, <a href="#bib.bib157" title="" class="ltx_ref">157</a>, <a href="#bib.bib158" title="" class="ltx_ref">158</a>, <a href="#bib.bib60" title="" class="ltx_ref">60</a>, <a href="#bib.bib222" title="" class="ltx_ref">222</a>, <a href="#bib.bib207" title="" class="ltx_ref">207</a>, <a href="#bib.bib225" title="" class="ltx_ref">225</a>, <a href="#bib.bib330" title="" class="ltx_ref">330</a>, <a href="#bib.bib63" title="" class="ltx_ref">63</a>, <a href="#bib.bib335" title="" class="ltx_ref">335</a>, <a href="#bib.bib336" title="" class="ltx_ref">336</a>, <a href="#bib.bib331" title="" class="ltx_ref">331</a>, <a href="#bib.bib64" title="" class="ltx_ref">64</a>, <a href="#bib.bib332" title="" class="ltx_ref">332</a>, <a href="#bib.bib110" title="" class="ltx_ref">110</a>, <a href="#bib.bib93" title="" class="ltx_ref">93</a>, <a href="#bib.bib112" title="" class="ltx_ref">112</a>, <a href="#bib.bib115" title="" class="ltx_ref">115</a>, <a href="#bib.bib117" title="" class="ltx_ref">117</a>, <a href="#bib.bib118" title="" class="ltx_ref">118</a>, <a href="#bib.bib120" title="" class="ltx_ref">120</a>, <a href="#bib.bib128" title="" class="ltx_ref">128</a>, <a href="#bib.bib254" title="" class="ltx_ref">254</a>, <a href="#bib.bib65" title="" class="ltx_ref">65</a>, <a href="#bib.bib245" title="" class="ltx_ref">245</a>, <a href="#bib.bib251" title="" class="ltx_ref">251</a>, <a href="#bib.bib243" title="" class="ltx_ref">243</a>, <a href="#bib.bib246" title="" class="ltx_ref">246</a>, <a href="#bib.bib247" title="" class="ltx_ref">247</a>, <a href="#bib.bib249" title="" class="ltx_ref">249</a>, <a href="#bib.bib260" title="" class="ltx_ref">260</a>, <a href="#bib.bib250" title="" class="ltx_ref">250</a>, <a href="#bib.bib66" title="" class="ltx_ref">66</a>, <a href="#bib.bib258" title="" class="ltx_ref">258</a>, <a href="#bib.bib67" title="" class="ltx_ref">67</a>, <a href="#bib.bib240" title="" class="ltx_ref">240</a>, <a href="#bib.bib237" title="" class="ltx_ref">237</a>, <a href="#bib.bib309" title="" class="ltx_ref">309</a>, <a href="#bib.bib320" title="" class="ltx_ref">320</a>, <a href="#bib.bib68" title="" class="ltx_ref">68</a>, <a href="#bib.bib314" title="" class="ltx_ref">314</a>, <a href="#bib.bib69" title="" class="ltx_ref">69</a>, <a href="#bib.bib323" title="" class="ltx_ref">323</a>, <a href="#bib.bib316" title="" class="ltx_ref">316</a>, <a href="#bib.bib313" title="" class="ltx_ref">313</a>, <a href="#bib.bib297" title="" class="ltx_ref">297</a>, <a href="#bib.bib62" title="" class="ltx_ref">62</a>, <a href="#bib.bib299" title="" class="ltx_ref">299</a>, <a href="#bib.bib298" title="" class="ltx_ref">298</a>, <a href="#bib.bib281" title="" class="ltx_ref">281</a>, <a href="#bib.bib282" title="" class="ltx_ref">282</a>, <a href="#bib.bib292" title="" class="ltx_ref">292</a>, <a href="#bib.bib302" title="" class="ltx_ref">302</a>, <a href="#bib.bib286" title="" class="ltx_ref">286</a>, <a href="#bib.bib295" title="" class="ltx_ref">295</a>, <a href="#bib.bib287" title="" class="ltx_ref">287</a>, <a href="#bib.bib290" title="" class="ltx_ref">290</a>, <a href="#bib.bib284" title="" class="ltx_ref">284</a>, <a href="#bib.bib283" title="" class="ltx_ref">283</a>, <a href="#bib.bib291" title="" class="ltx_ref">291</a>, <a href="#bib.bib238" title="" class="ltx_ref">238</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.13.13.4" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.13.13.4.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.13.13.4.1.1" class="ltx_p" style="width:21.7pt;">75</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.14.14" class="ltx_tr">
<td id="S10.T16.1.1.14.14.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.14.14.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.14.14.1.1.1" class="ltx_p" style="width:130.1pt;">Inception Score (IS)</span>
</span>
</td>
<td id="S10.T16.1.1.14.14.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.14.14.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.14.14.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib138" title="" class="ltx_ref">138</a>, <a href="#bib.bib141" title="" class="ltx_ref">141</a>, <a href="#bib.bib148" title="" class="ltx_ref">148</a>, <a href="#bib.bib175" title="" class="ltx_ref">175</a>, <a href="#bib.bib330" title="" class="ltx_ref">330</a>, <a href="#bib.bib63" title="" class="ltx_ref">63</a>, <a href="#bib.bib64" title="" class="ltx_ref">64</a>, <a href="#bib.bib65" title="" class="ltx_ref">65</a>, <a href="#bib.bib237" title="" class="ltx_ref">237</a>, <a href="#bib.bib287" title="" class="ltx_ref">287</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.14.14.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.14.14.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.14.14.3.1.1" class="ltx_p" style="width:21.7pt;">10</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.15.15" class="ltx_tr">
<td id="S10.T16.1.1.15.15.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.15.15.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.15.15.1.1.1" class="ltx_p" style="width:130.1pt;">Kernel Inception Distance (KID)</span>
</span>
</td>
<td id="S10.T16.1.1.15.15.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.15.15.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.15.15.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib139" title="" class="ltx_ref">139</a>, <a href="#bib.bib142" title="" class="ltx_ref">142</a>, <a href="#bib.bib143" title="" class="ltx_ref">143</a>, <a href="#bib.bib183" title="" class="ltx_ref">183</a>, <a href="#bib.bib185" title="" class="ltx_ref">185</a>, <a href="#bib.bib332" title="" class="ltx_ref">332</a>, <a href="#bib.bib245" title="" class="ltx_ref">245</a>, <a href="#bib.bib314" title="" class="ltx_ref">314</a>, <a href="#bib.bib295" title="" class="ltx_ref">295</a>, <a href="#bib.bib290" title="" class="ltx_ref">290</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.15.15.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.15.15.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.15.15.3.1.1" class="ltx_p" style="width:21.7pt;">10</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.16.16" class="ltx_tr">
<td id="S10.T16.1.1.16.16.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.16.16.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.16.16.1.1.1" class="ltx_p" style="width:130.1pt;">Maximum Mean Discrepancy (MMD)</span>
</span>
</td>
<td id="S10.T16.1.1.16.16.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.16.16.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.16.16.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib172" title="" class="ltx_ref">172</a>, <a href="#bib.bib210" title="" class="ltx_ref">210</a>, <a href="#bib.bib194" title="" class="ltx_ref">194</a>, <a href="#bib.bib225" title="" class="ltx_ref">225</a>, <a href="#bib.bib64" title="" class="ltx_ref">64</a>, <a href="#bib.bib106" title="" class="ltx_ref">106</a>, <a href="#bib.bib107" title="" class="ltx_ref">107</a>, <a href="#bib.bib111" title="" class="ltx_ref">111</a>, <a href="#bib.bib112" title="" class="ltx_ref">112</a>, <a href="#bib.bib115" title="" class="ltx_ref">115</a>, <a href="#bib.bib116" title="" class="ltx_ref">116</a>, <a href="#bib.bib129" title="" class="ltx_ref">129</a>, <a href="#bib.bib250" title="" class="ltx_ref">250</a>, <a href="#bib.bib238" title="" class="ltx_ref">238</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.16.16.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.16.16.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.16.16.3.1.1" class="ltx_p" style="width:21.7pt;">14</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.17.17" class="ltx_tr">
<td id="S10.T16.1.1.17.17.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.17.17.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.17.17.1.1.1" class="ltx_p" style="width:130.1pt;">Learned Perceptual Image Patch Similarity (LPIPS)</span>
</span>
</td>
<td id="S10.T16.1.1.17.17.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.17.17.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.17.17.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib183" title="" class="ltx_ref">183</a>, <a href="#bib.bib184" title="" class="ltx_ref">184</a>, <a href="#bib.bib164" title="" class="ltx_ref">164</a>, <a href="#bib.bib336" title="" class="ltx_ref">336</a>, <a href="#bib.bib245" title="" class="ltx_ref">245</a>, <a href="#bib.bib243" title="" class="ltx_ref">243</a>, <a href="#bib.bib260" title="" class="ltx_ref">260</a>, <a href="#bib.bib291" title="" class="ltx_ref">291</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.17.17.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.17.17.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.17.17.3.1.1" class="ltx_p" style="width:21.7pt;">8</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.18.18" class="ltx_tr">
<td id="S10.T16.1.1.18.18.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.18.18.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.18.18.1.1.1" class="ltx_p" style="width:130.1pt;">Feature Similarity Index (FSIM)</span>
</span>
</td>
<td id="S10.T16.1.1.18.18.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.18.18.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.18.18.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib156" title="" class="ltx_ref">156</a>, <a href="#bib.bib159" title="" class="ltx_ref">159</a>, <a href="#bib.bib194" title="" class="ltx_ref">194</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.18.18.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.18.18.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.18.18.3.1.1" class="ltx_p" style="width:21.7pt;">3</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.19.19" class="ltx_tr">
<td id="S10.T16.1.1.19.19.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.19.19.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.19.19.1.1.1" class="ltx_p" style="width:130.1pt;">feature distribution similarity (FDS)</span>
</span>
</td>
<td id="S10.T16.1.1.19.19.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.19.19.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.19.19.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib65" title="" class="ltx_ref">65</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.19.19.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.19.19.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.19.19.3.1.1" class="ltx_p" style="width:21.7pt;">1</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.20.20" class="ltx_tr">
<td id="S10.T16.1.1.20.20.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_b ltx_border_t" rowspan="3">
<span id="S10.T16.1.1.20.20.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.20.20.1.1.1" class="ltx_p" style="width:86.7pt;"><span id="S10.T16.1.1.20.20.1.1.1.1" class="ltx_text">
<span id="S10.T16.1.1.20.20.1.1.1.1.1" class="ltx_inline-block ltx_parbox ltx_align_middle" style="width:78.0pt;">
<span id="S10.T16.1.1.20.20.1.1.1.1.1.1" class="ltx_p">Image-text alignment/accuracy</span>
</span></span></span>
</span>
</td>
<td id="S10.T16.1.1.20.20.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.20.20.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.20.20.2.1.1" class="ltx_p" style="width:130.1pt;">Bilingual Evaluation Understudy (BLEU)</span>
</span>
</td>
<td id="S10.T16.1.1.20.20.3" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.20.20.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.20.20.3.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib296" title="" class="ltx_ref">296</a>, <a href="#bib.bib298" title="" class="ltx_ref">298</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.20.20.4" class="ltx_td ltx_align_justify ltx_align_top ltx_border_t">
<span id="S10.T16.1.1.20.20.4.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.20.20.4.1.1" class="ltx_p" style="width:21.7pt;">2</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.21.21" class="ltx_tr">
<td id="S10.T16.1.1.21.21.1" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.21.21.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.21.21.1.1.1" class="ltx_p" style="width:130.1pt;">Recall-Oriented Understudy for Gisting Evaluation (ROUGE)</span>
</span>
</td>
<td id="S10.T16.1.1.21.21.2" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.21.21.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.21.21.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib296" title="" class="ltx_ref">296</a>, <a href="#bib.bib299" title="" class="ltx_ref">299</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.21.21.3" class="ltx_td ltx_align_justify ltx_align_top">
<span id="S10.T16.1.1.21.21.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.21.21.3.1.1" class="ltx_p" style="width:21.7pt;">2</span>
</span>
</td>
</tr>
<tr id="S10.T16.1.1.22.22" class="ltx_tr">
<td id="S10.T16.1.1.22.22.1" class="ltx_td ltx_align_justify ltx_align_top ltx_border_b">
<span id="S10.T16.1.1.22.22.1.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.22.22.1.1.1" class="ltx_p" style="width:130.1pt;">Contrastive Language–Image Pre-training score (CLIP)</span>
</span>
</td>
<td id="S10.T16.1.1.22.22.2" class="ltx_td ltx_align_justify ltx_align_top ltx_border_b">
<span id="S10.T16.1.1.22.22.2.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.22.22.2.1.1" class="ltx_p" style="width:281.9pt;"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib251" title="" class="ltx_ref">251</a>]</cite></span>
</span>
</td>
<td id="S10.T16.1.1.22.22.3" class="ltx_td ltx_align_justify ltx_align_top ltx_border_b">
<span id="S10.T16.1.1.22.22.3.1" class="ltx_inline-block ltx_align_top">
<span id="S10.T16.1.1.22.22.3.1.1" class="ltx_p" style="width:21.7pt;">1</span>
</span>
</td>
</tr>
</tbody>
</table>
</span></div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 16: </span>Fidelity Metrics for Imaging Data</figcaption>
</figure>
<figure id="S10.T17" class="ltx_table">
<div id="S10.T17.1" class="ltx_inline-block ltx_transformed_outer" style="width:577.0pt;height:912.5pt;vertical-align:-0.0pt;"><span class="ltx_transformed_inner" style="width:912.5pt;transform:translate(-167.75pt,-167.25pt) rotate(-90deg) ;">
<table id="S10.T17.1.1" class="ltx_tabular ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S10.T17.1.1.2.1" class="ltx_tr">
<td id="S10.T17.1.1.2.1.1" class="ltx_td ltx_align_left ltx_border_t">Modality</td>
<td id="S10.T17.1.1.2.1.2" class="ltx_td ltx_align_left ltx_border_t">Purpose of Evaluation</td>
<td id="S10.T17.1.1.2.1.3" class="ltx_td ltx_align_left ltx_border_t">Fidelity Metric</td>
<td id="S10.T17.1.1.2.1.4" class="ltx_td ltx_align_left ltx_border_t">Usage</td>
<td id="S10.T17.1.1.2.1.5" class="ltx_td ltx_align_left ltx_border_t">Count</td>
</tr>
<tr id="S10.T17.1.1.3.2" class="ltx_tr">
<td id="S10.T17.1.1.3.2.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="27"><span id="S10.T17.1.1.3.2.1.1" class="ltx_text">EHR</span></td>
<td id="S10.T17.1.1.3.2.2" class="ltx_td ltx_align_left ltx_border_t" rowspan="10"><span id="S10.T17.1.1.3.2.2.1" class="ltx_text">Dimension Wise distributional similarity</span></td>
<td id="S10.T17.1.1.3.2.3" class="ltx_td ltx_align_left ltx_border_t">Bernoulli success probability</td>
<td id="S10.T17.1.1.3.2.4" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib369" title="" class="ltx_ref">369</a>, <a href="#bib.bib84" title="" class="ltx_ref">84</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite></td>
<td id="S10.T17.1.1.3.2.5" class="ltx_td ltx_align_left ltx_border_t">4</td>
</tr>
<tr id="S10.T17.1.1.4.3" class="ltx_tr">
<td id="S10.T17.1.1.4.3.1" class="ltx_td ltx_align_left">Pearson Chi-Squared Test</td>
<td id="S10.T17.1.1.4.3.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib90" title="" class="ltx_ref">90</a>, <a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite></td>
<td id="S10.T17.1.1.4.3.3" class="ltx_td ltx_align_left">2</td>
</tr>
<tr id="S10.T17.1.1.5.4" class="ltx_tr">
<td id="S10.T17.1.1.5.4.1" class="ltx_td ltx_align_left">Analysis of variance F-test</td>
<td id="S10.T17.1.1.5.4.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite></td>
<td id="S10.T17.1.1.5.4.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.6.5" class="ltx_tr">
<td id="S10.T17.1.1.6.5.1" class="ltx_td ltx_align_left">Student’s t-test</td>
<td id="S10.T17.1.1.6.5.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite></td>
<td id="S10.T17.1.1.6.5.3" class="ltx_td ltx_align_left">2</td>
</tr>
<tr id="S10.T17.1.1.7.6" class="ltx_tr">
<td id="S10.T17.1.1.7.6.1" class="ltx_td ltx_align_left">Dimension-wise distribution</td>
<td id="S10.T17.1.1.7.6.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite></td>
<td id="S10.T17.1.1.7.6.3" class="ltx_td ltx_align_left">2</td>
</tr>
<tr id="S10.T17.1.1.8.7" class="ltx_tr">
<td id="S10.T17.1.1.8.7.1" class="ltx_td ltx_align_left">dimension-wise statistics</td>
<td id="S10.T17.1.1.8.7.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib70" title="" class="ltx_ref">70</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite></td>
<td id="S10.T17.1.1.8.7.3" class="ltx_td ltx_align_left">4</td>
</tr>
<tr id="S10.T17.1.1.9.8" class="ltx_tr">
<td id="S10.T17.1.1.9.8.1" class="ltx_td ltx_align_left">Kolmogorov–Smirnov Test</td>
<td id="S10.T17.1.1.9.8.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib77" title="" class="ltx_ref">77</a>, <a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite></td>
<td id="S10.T17.1.1.9.8.3" class="ltx_td ltx_align_left">4</td>
</tr>
<tr id="S10.T17.1.1.10.9" class="ltx_tr">
<td id="S10.T17.1.1.10.9.1" class="ltx_td ltx_align_left">three sigma rule test</td>
<td id="S10.T17.1.1.10.9.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite></td>
<td id="S10.T17.1.1.10.9.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.11.10" class="ltx_tr">
<td id="S10.T17.1.1.11.10.1" class="ltx_td ltx_align_left">Kernel Density Estimation</td>
<td id="S10.T17.1.1.11.10.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib84" title="" class="ltx_ref">84</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite></td>
<td id="S10.T17.1.1.11.10.3" class="ltx_td ltx_align_left">2</td>
</tr>
<tr id="S10.T17.1.1.12.11" class="ltx_tr">
<td id="S10.T17.1.1.12.11.1" class="ltx_td ltx_align_left">Kullback Leibler Divergence</td>
<td id="S10.T17.1.1.12.11.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite></td>
<td id="S10.T17.1.1.12.11.3" class="ltx_td ltx_align_left">2</td>
</tr>
<tr id="S10.T17.1.1.13.12" class="ltx_tr">
<td id="S10.T17.1.1.13.12.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="6"><span id="S10.T17.1.1.13.12.1.1" class="ltx_text">inter-dimensional similarity</span></td>
<td id="S10.T17.1.1.13.12.2" class="ltx_td ltx_align_left ltx_border_t">Pearson pairwise correlation</td>
<td id="S10.T17.1.1.13.12.3" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib71" title="" class="ltx_ref">71</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite></td>
<td id="S10.T17.1.1.13.12.4" class="ltx_td ltx_align_left ltx_border_t">2</td>
</tr>
<tr id="S10.T17.1.1.14.13" class="ltx_tr">
<td id="S10.T17.1.1.14.13.1" class="ltx_td ltx_align_left">pairwise correlation difference</td>
<td id="S10.T17.1.1.14.13.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite></td>
<td id="S10.T17.1.1.14.13.3" class="ltx_td ltx_align_left">4</td>
</tr>
<tr id="S10.T17.1.1.1" class="ltx_tr">
<td id="S10.T17.1.1.1.1" class="ltx_td ltx_align_left">Kendall’s <math id="S10.T17.1.1.1.1.m1.1" class="ltx_Math" alttext="\tau" display="inline"><semantics id="S10.T17.1.1.1.1.m1.1a"><mi id="S10.T17.1.1.1.1.m1.1.1" xref="S10.T17.1.1.1.1.m1.1.1.cmml">τ</mi><annotation-xml encoding="MathML-Content" id="S10.T17.1.1.1.1.m1.1b"><ci id="S10.T17.1.1.1.1.m1.1.1.cmml" xref="S10.T17.1.1.1.1.m1.1.1">𝜏</ci></annotation-xml><annotation encoding="application/x-tex" id="S10.T17.1.1.1.1.m1.1c">\tau</annotation></semantics></math> rank correlation</td>
<td id="S10.T17.1.1.1.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite></td>
<td id="S10.T17.1.1.1.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.15.14" class="ltx_tr">
<td id="S10.T17.1.1.15.14.1" class="ltx_td ltx_align_left">Dimension-wise prediction</td>
<td id="S10.T17.1.1.15.14.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib369" title="" class="ltx_ref">369</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>]</cite></td>
<td id="S10.T17.1.1.15.14.3" class="ltx_td ltx_align_left">6</td>
</tr>
<tr id="S10.T17.1.1.16.15" class="ltx_tr">
<td id="S10.T17.1.1.16.15.1" class="ltx_td ltx_align_left">Frequent Association Rules</td>
<td id="S10.T17.1.1.16.15.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite></td>
<td id="S10.T17.1.1.16.15.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.17.16" class="ltx_tr">
<td id="S10.T17.1.1.17.16.1" class="ltx_td ltx_align_left">Dimension-wise correlation</td>
<td id="S10.T17.1.1.17.16.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite></td>
<td id="S10.T17.1.1.17.16.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.18.17" class="ltx_tr">
<td id="S10.T17.1.1.18.17.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="5"><span id="S10.T17.1.1.18.17.1.1" class="ltx_text">Joint distribution similarity</span></td>
<td id="S10.T17.1.1.18.17.2" class="ltx_td ltx_align_left ltx_border_t">Jensen-Shannon Divergence</td>
<td id="S10.T17.1.1.18.17.3" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib85" title="" class="ltx_ref">85</a>, <a href="#bib.bib58" title="" class="ltx_ref">58</a>]</cite></td>
<td id="S10.T17.1.1.18.17.4" class="ltx_td ltx_align_left ltx_border_t">2</td>
</tr>
<tr id="S10.T17.1.1.19.18" class="ltx_tr">
<td id="S10.T17.1.1.19.18.1" class="ltx_td ltx_align_left">Maximum-Mean Discrepancy</td>
<td id="S10.T17.1.1.19.18.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib18" title="" class="ltx_ref">18</a>, <a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite></td>
<td id="S10.T17.1.1.19.18.3" class="ltx_td ltx_align_left">4</td>
</tr>
<tr id="S10.T17.1.1.20.19" class="ltx_tr">
<td id="S10.T17.1.1.20.19.1" class="ltx_td ltx_align_left">Wasserstein distance</td>
<td id="S10.T17.1.1.20.19.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib85" title="" class="ltx_ref">85</a>]</cite></td>
<td id="S10.T17.1.1.20.19.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.21.20" class="ltx_tr">
<td id="S10.T17.1.1.21.20.1" class="ltx_td ltx_align_left">Cross-type Conditional Distribution,first-order proximity</td>
<td id="S10.T17.1.1.21.20.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite></td>
<td id="S10.T17.1.1.21.20.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.22.21" class="ltx_tr">
<td id="S10.T17.1.1.22.21.1" class="ltx_td ltx_align_left">discriminatve score</td>
<td id="S10.T17.1.1.22.21.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a>, <a href="#bib.bib17" title="" class="ltx_ref">17</a>, <a href="#bib.bib90" title="" class="ltx_ref">90</a>]</cite></td>
<td id="S10.T17.1.1.22.21.3" class="ltx_td ltx_align_left">5</td>
</tr>
<tr id="S10.T17.1.1.23.22" class="ltx_tr">
<td id="S10.T17.1.1.23.22.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="3"><span id="S10.T17.1.1.23.22.1.1" class="ltx_text">Latent distribtuion similarity</span></td>
<td id="S10.T17.1.1.23.22.2" class="ltx_td ltx_align_left ltx_border_t">Log-cluster</td>
<td id="S10.T17.1.1.23.22.3" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite></td>
<td id="S10.T17.1.1.23.22.4" class="ltx_td ltx_align_left ltx_border_t">3</td>
</tr>
<tr id="S10.T17.1.1.24.23" class="ltx_tr">
<td id="S10.T17.1.1.24.23.1" class="ltx_td ltx_align_left">latent space evaluation metrics</td>
<td id="S10.T17.1.1.24.23.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib70" title="" class="ltx_ref">70</a>]</cite></td>
<td id="S10.T17.1.1.24.23.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.25.24" class="ltx_tr">
<td id="S10.T17.1.1.25.24.1" class="ltx_td ltx_align_left">latent space representation</td>
<td id="S10.T17.1.1.25.24.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib82" title="" class="ltx_ref">82</a>]</cite></td>
<td id="S10.T17.1.1.25.24.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.26.25" class="ltx_tr">
<td id="S10.T17.1.1.26.25.1" class="ltx_td ltx_align_left ltx_border_t">specific-language model</td>
<td id="S10.T17.1.1.26.25.2" class="ltx_td ltx_align_left ltx_border_t">PERPLEXITY</td>
<td id="S10.T17.1.1.26.25.3" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib74" title="" class="ltx_ref">74</a>, <a href="#bib.bib73" title="" class="ltx_ref">73</a>]</cite></td>
<td id="S10.T17.1.1.26.25.4" class="ltx_td ltx_align_left ltx_border_t">2</td>
</tr>
<tr id="S10.T17.1.1.27.26" class="ltx_tr">
<td id="S10.T17.1.1.27.26.1" class="ltx_td ltx_align_left">specific-time-series</td>
<td id="S10.T17.1.1.27.26.2" class="ltx_td ltx_align_left">autocorrelation function</td>
<td id="S10.T17.1.1.27.26.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite></td>
<td id="S10.T17.1.1.27.26.4" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.28.27" class="ltx_tr">
<td id="S10.T17.1.1.28.27.1" class="ltx_td ltx_align_left">specific-time-series</td>
<td id="S10.T17.1.1.28.27.2" class="ltx_td ltx_align_left">patient trajectories</td>
<td id="S10.T17.1.1.28.27.3" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib79" title="" class="ltx_ref">79</a>]</cite></td>
<td id="S10.T17.1.1.28.27.4" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.29.28" class="ltx_tr">
<td id="S10.T17.1.1.29.28.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S10.T17.1.1.29.28.1.1" class="ltx_text">Signals</span></td>
<td id="S10.T17.1.1.29.28.2" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S10.T17.1.1.29.28.2.1" class="ltx_text">Time</span></td>
<td id="S10.T17.1.1.29.28.3" class="ltx_td ltx_align_left ltx_border_t">Dynamic Time Warping (DTW)</td>
<td id="S10.T17.1.1.29.28.4" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib110" title="" class="ltx_ref">110</a>, <a href="#bib.bib93" title="" class="ltx_ref">93</a>, <a href="#bib.bib111" title="" class="ltx_ref">111</a>, <a href="#bib.bib115" title="" class="ltx_ref">115</a>]</cite></td>
<td id="S10.T17.1.1.29.28.5" class="ltx_td ltx_align_left ltx_border_t">4</td>
</tr>
<tr id="S10.T17.1.1.30.29" class="ltx_tr">
<td id="S10.T17.1.1.30.29.1" class="ltx_td ltx_align_left">Time Warp Edit Distance (TWED)</td>
<td id="S10.T17.1.1.30.29.2" class="ltx_td ltx_align_left"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib115" title="" class="ltx_ref">115</a>]</cite></td>
<td id="S10.T17.1.1.30.29.3" class="ltx_td ltx_align_left">1</td>
</tr>
<tr id="S10.T17.1.1.31.30" class="ltx_tr">
<td id="S10.T17.1.1.31.30.1" class="ltx_td ltx_align_left ltx_border_b ltx_border_t" rowspan="2"><span id="S10.T17.1.1.31.30.1.1" class="ltx_text">Text</span></td>
<td id="S10.T17.1.1.31.30.2" class="ltx_td ltx_align_left ltx_border_t">Similarity between machine and human translation</td>
<td id="S10.T17.1.1.31.30.3" class="ltx_td ltx_align_left ltx_border_t">Bilingual Evaluation Understudy (BLEU)</td>
<td id="S10.T17.1.1.31.30.4" class="ltx_td ltx_align_left ltx_border_t"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib343" title="" class="ltx_ref">343</a>, <a href="#bib.bib344" title="" class="ltx_ref">344</a>, <a href="#bib.bib345" title="" class="ltx_ref">345</a>]</cite></td>
<td id="S10.T17.1.1.31.30.5" class="ltx_td ltx_align_left ltx_border_t">3</td>
</tr>
<tr id="S10.T17.1.1.32.31" class="ltx_tr">
<td id="S10.T17.1.1.32.31.1" class="ltx_td ltx_align_left ltx_border_b">Similarity between real and generated text</td>
<td id="S10.T17.1.1.32.31.2" class="ltx_td ltx_align_left ltx_border_b">ROUGE-N recall</td>
<td id="S10.T17.1.1.32.31.3" class="ltx_td ltx_align_left ltx_border_b"><cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib346" title="" class="ltx_ref">346</a>]</cite></td>
<td id="S10.T17.1.1.32.31.4" class="ltx_td ltx_align_left ltx_border_b">1</td>
</tr>
</tbody>
</table>
</span></div>
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 17: </span>Fidelity Metrics for EHR, Signals, and Text</figcaption>
</figure>
<div class="ltx_pagination ltx_role_newpage"></div>
</section>
</section>
</section>
</section>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/2407.00115" class="ar5iv-nav-button ar5iv-nav-button-prev">◄</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/2407.00116" class="ar5iv-text-button ar5iv-severity-error">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+2407.00116">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/2407.00116" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/2407.00118" class="ar5iv-nav-button ar5iv-nav-button-next">►</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Mon Aug  5 16:23:58 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "×";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
