<!DOCTYPE html>
<html lang="en">
<head>
<meta content="text/html; charset=utf-8" http-equiv="content-type"/>
<title>Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data</title>
<!--Generated on Wed Sep 18 15:44:46 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport"/>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv.0.7.9.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv-fonts.0.7.9.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/latexml_styles.css" rel="stylesheet" type="text/css"/>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/html2canvas/1.3.3/html2canvas.min.js"></script>
<script src="/static/browse/0.3.4/js/addons_new.js"></script>
<script src="/static/browse/0.3.4/js/feedbackOverlay.js"></script>
<base href="/html/2409.12215v1/"/></head>
<body>
<nav class="ltx_page_navbar">
<nav class="ltx_TOC">
<ol class="ltx_toclist">
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S1" title="In Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1 </span>Introduction</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2" title="In Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2 </span>Methods</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS1" title="In 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.1 </span>Model selection criteria</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS2" title="In 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2 </span>Software Environment</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS3" title="In 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.3 </span>Code Modularity</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS4" title="In 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.4 </span>Data Availability and Preprocessing</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS5" title="In 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.5 </span>Reproducing key performance scores</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3" title="In Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3 </span>Results</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.SS1" title="In 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1 </span>Software Environment</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.SS2" title="In 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.2 </span>Code Modularity</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.SS3" title="In 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.3 </span>Data Availability and Preprocessing</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.SS4" title="In 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.4 </span>Reproducing key performance scores</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S4" title="In Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4 </span>Discussion and Recommendations</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S5" title="In Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5 </span>Conclusion</span></a></li>
</ol></nav>
</nav>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<h1 class="ltx_title ltx_title_document">Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Jamie C. Overbeek<sup class="ltx_sup" id="id3.2.id1">∗</sup>
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Data Science and Learning Division, Argonne National Laboratory, Lemont, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Alexander Partin<sup class="ltx_sup" id="id4.2.id1">∗</sup>
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Data Science and Learning Division, Argonne National Laboratory, Lemont, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Thomas S. Brettin
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Data Science and Learning Division, Argonne National Laboratory, Lemont, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Nicholas Chia
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Data Science and Learning Division, Argonne National Laboratory, Lemont, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Oleksandr Narykov
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Data Science and Learning Division, Argonne National Laboratory, Lemont, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Priyanka Vasanthakumari
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Data Science and Learning Division, Argonne National Laboratory, Lemont, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Andreas Wilke
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Data Science and Learning Division, Argonne National Laboratory, Lemont, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Yitan Zhu
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Data Science and Learning Division, Argonne National Laboratory, Lemont, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Austin Clyde
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Computer Science, The University of Chicago, Chicago, IL, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Sara Jones
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Frederick National Laboratory for Cancer Research, Frederick, MD, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Rohan Gnanaolivu
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Quantitative Health Sciences, Mayo Clinic, Rochester, MN, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Yuanhang Liu
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Quantitative Health Sciences, Mayo Clinic, Rochester, MN, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Jun Jiang
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Quantitative Health Sciences, Mayo Clinic, Rochester, MN, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Chen Wang
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Quantitative Health Sciences, Mayo Clinic, Rochester, MN, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Carter Knutson
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Pacific Northwest National Laboratory, Richland, WA, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Andrew McNaughton
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Pacific Northwest National Laboratory, Richland, WA, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Neeraj Kumar
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Pacific Northwest National Laboratory, Richland, WA, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Gayara Demini Fernando
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Statistics, University of Nebraska–Lincoln, Lincoln, NE, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Souparno Ghosh
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Statistics, University of Nebraska–Lincoln, Lincoln, NE, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Cesar Sanchez-Villalobos
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Electrical &amp; Computer Engineering, Texas Tech University, Lubbock, TX, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Ruibo Zhang
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Electrical &amp; Computer Engineering, Texas Tech University, Lubbock, TX, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Ranadip Pal
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Electrical &amp; Computer Engineering, Texas Tech University, Lubbock, TX, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">M. Ryan Weil
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Frederick National Laboratory for Cancer Research, Frederick, MD, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Rick L. Stevens
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Computing, Environment and Life Sciences, Argonne National Laboratory, Lemont, IL, USA
</span>
<span class="ltx_contact ltx_role_affiliation">Department of Computer Science, The University of Chicago, Chicago, IL, USA
</span></span></span>
</div>
<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p class="ltx_p" id="id5.id1">Cancer drug response prediction (DRP) models present a promising approach towards precision oncology, tailoring treatments to individual patient profiles.
While deep learning (DL) methods have shown great potential in this area, models that can be successfully translated into clinical practice and shed light on the molecular mechanisms underlying treatment response will likely emerge from collaborative research efforts. This highlights the need for reusable and adaptable models that can be improved and tested by the wider scientific community.
In this study, we present a scoring system for assessing the reusability of prediction DRP models, and apply it to 17 peer-reviewed DL-based DRP models. As part of the IMPROVE (Innovative Methodologies and New Data for Predictive Oncology Model Evaluation) project, which aims to develop methods for systematic evaluation and comparison DL models across scientific domains, we analyzed these 17 DRP models focusing on three key categories: software environment, code modularity, and data availability and preprocessing. While not the primary focus, we also attempted to reproduce key performance metrics to verify model behavior and adaptability.
Our assessment of 17 DRP models reveals both strengths and shortcomings in model reusability. Issues like underspecified software environments, lack of workflow documentation, and omission of data preprocessing scripts create challenges for users attempting to reproduce or build upon these models. To promote rigorous practices and open-source sharing, we offer recommendations for developing and sharing prediction models. Following these recommendations can address many of the issues identified in this study, improving model reusability without adding significant burdens on researchers.
This work offers the first comprehensive assessment of reusability and reproducibility across diverse DRP models, providing insights into current model sharing practices and promoting standards within the DRP and broader AI-enabled scientific research community. The community will develop further guidelines as engagement with this topic increases.
</p>
</div>
<span class="ltx_note ltx_role_footnotetext" id="footnotex1"><sup class="ltx_note_mark">*</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">*</sup><span class="ltx_note_type">footnotetext: </span>These authors contributed equally to this work. Please cite as Overbeek JC and Partin A et al. Correspondence: joverbeek@anl.gov, apartin@anl.gov</span></span></span>
<div class="ltx_para ltx_noindent" id="p1">
<p class="ltx_p" id="p1.5"><em class="ltx_emph ltx_font_bold ltx_font_italic" id="p1.5.1">K</em><span class="ltx_text ltx_font_bold" id="p1.5.2">eywords</span> drug response prediction  <math alttext="\cdot" class="ltx_Math" display="inline" id="p1.1.m1.1"><semantics id="p1.1.m1.1a"><mo id="p1.1.m1.1.1" xref="p1.1.m1.1.1.cmml">⋅</mo><annotation-xml encoding="MathML-Content" id="p1.1.m1.1b"><ci id="p1.1.m1.1.1.cmml" xref="p1.1.m1.1.1">⋅</ci></annotation-xml><annotation encoding="application/x-tex" id="p1.1.m1.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p1.1.m1.1d">⋅</annotation></semantics></math>
precision oncology  <math alttext="\cdot" class="ltx_Math" display="inline" id="p1.2.m2.1"><semantics id="p1.2.m2.1a"><mo id="p1.2.m2.1.1" xref="p1.2.m2.1.1.cmml">⋅</mo><annotation-xml encoding="MathML-Content" id="p1.2.m2.1b"><ci id="p1.2.m2.1.1.cmml" xref="p1.2.m2.1.1">⋅</ci></annotation-xml><annotation encoding="application/x-tex" id="p1.2.m2.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p1.2.m2.1d">⋅</annotation></semantics></math>
reusability  <math alttext="\cdot" class="ltx_Math" display="inline" id="p1.3.m3.1"><semantics id="p1.3.m3.1a"><mo id="p1.3.m3.1.1" xref="p1.3.m3.1.1.cmml">⋅</mo><annotation-xml encoding="MathML-Content" id="p1.3.m3.1b"><ci id="p1.3.m3.1.1.cmml" xref="p1.3.m3.1.1">⋅</ci></annotation-xml><annotation encoding="application/x-tex" id="p1.3.m3.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p1.3.m3.1d">⋅</annotation></semantics></math>
reproducibility  <math alttext="\cdot" class="ltx_Math" display="inline" id="p1.4.m4.1"><semantics id="p1.4.m4.1a"><mo id="p1.4.m4.1.1" xref="p1.4.m4.1.1.cmml">⋅</mo><annotation-xml encoding="MathML-Content" id="p1.4.m4.1b"><ci id="p1.4.m4.1.1.cmml" xref="p1.4.m4.1.1">⋅</ci></annotation-xml><annotation encoding="application/x-tex" id="p1.4.m4.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p1.4.m4.1d">⋅</annotation></semantics></math>
predictive modeling  <math alttext="\cdot" class="ltx_Math" display="inline" id="p1.5.m5.1"><semantics id="p1.5.m5.1a"><mo id="p1.5.m5.1.1" xref="p1.5.m5.1.1.cmml">⋅</mo><annotation-xml encoding="MathML-Content" id="p1.5.m5.1b"><ci id="p1.5.m5.1.1.cmml" xref="p1.5.m5.1.1">⋅</ci></annotation-xml><annotation encoding="application/x-tex" id="p1.5.m5.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p1.5.m5.1d">⋅</annotation></semantics></math>
deep learning</p>
</div>
<section class="ltx_section" id="S1">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>
<div class="ltx_para ltx_noindent" id="S1.p1">
<p class="ltx_p" id="S1.p1.1">There is a growing interest in utilizing predictive modeling to advance precision oncology <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib1" title="">1</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib2" title="">2</a>]</cite>. Among various approaches, drug response prediction (DRP) models hold promise, as they predict the response of cancer to drug treatments by analyzing tumor and drug properties <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib3" title="">3</a>]</cite>.
Technological advances in genomic profiling and drug sensitivity screenings have led to an abundance of data <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib4" title="">4</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib5" title="">5</a>]</cite>, driving the exploration of machine learning (ML) and especially deep learning (DL) methods, for DRP <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib6" title="">6</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib7" title="">7</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib8" title="">8</a>]</cite>.
Numerous papers have been published, exploring various model architectures, training schemes, and data representations, demonstrating the predictive performance of different approaches in retrospective analyses with preclinical drug screening data <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib6" title="">6</a>]</cite>.
While this surge of publications represents great progress in the field, it also makes it extremely challenging to navigate the various methodologies and identify the most effective approaches to improve the utility of DRP models in precision oncology.</p>
</div>
<div class="ltx_para ltx_noindent" id="S1.p2">
<p class="ltx_p" id="S1.p2.1">To demonstrate superiority, most existing papers compare the performance of their DL models against conventional, out-of-the-box ML baselines <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib6" title="">6</a>]</cite>.
However, to demonstrate substantive contributions, newly proposed models should undergo rigorous cross-comparisons with multiple state-of-the-art models from the research community.
Such comparisons typically require models to use consistent data representations, data splits, and evaluation schemes, which often necessitate modifications to the original code of the models.
Therefore, the ability to efficiently integrate community models into diverse comparison analyses is directly linked to model <span class="ltx_text ltx_font_italic" id="S1.p2.1.1">reusability</span>.
While several studies have performed comprehensive model comparisons focusing on prediction performance <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib9" title="">9</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib10" title="">10</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib11" title="">11</a>]</cite>, there is minimal discussion provided about the reusability of these models.</p>
</div>
<div class="ltx_para ltx_noindent" id="S1.p3">
<p class="ltx_p" id="S1.p3.1">Given the potential challenges of working with community models, having insights into the ease of using and <span class="ltx_text ltx_font_italic" id="S1.p3.1.1">reusing</span> models can substantially influence researchers’ decision-making process when selecting models for comparison studies or adapting them to novel contexts.
So et al. <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib12" title="">12</a>]</cite> conducted extensive analysis with TCRP <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib13" title="">13</a>]</cite>, assessing its reusability on ’omics and drug response datasets beyond the original publication, including cell line <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib14" title="">14</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib15" title="">15</a>]</cite>, patient-derived xenograft <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib16" title="">16</a>]</cite>, and clinical trial data <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib17" title="">17</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib18" title="">18</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib19" title="">19</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib20" title="">20</a>]</cite>.
Their conclusion of TCRP’s successful reusability primarily relied on its prediction performance across contexts.
However, the challenges encountered during model adaptation can play an equally critical role in model selection.
While providing extensive discussion on these challenges, a concrete system of criteria assessing ease of use was lacking.
A set of well-defined criteria for assessing reusability can facilitate efficient model selection.
These criteria could take on a form of reusability ranking, evaluating models based on factors relating to code accessibility and modularity, quality of documentation, and compatibility with popular frameworks.
This would allow researchers to quickly identify models that are not only high-performing but also user-friendly, fostering greater collaboration and innovation in AI-assisted precision oncology.
</p>
</div>
<div class="ltx_para ltx_noindent" id="S1.p4">
<p class="ltx_p" id="S1.p4.1">In this paper, we propose a scoring system to assess model reusability and apply it to 17 DRP models from peer-reviewed publications.
This work is part of the IMPROVE project (Innovative Methodologies and New Data for Predictive Oncology Model Evaluation)<span class="ltx_note ltx_role_footnote" id="footnote1"><sup class="ltx_note_mark">*</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">*</sup><span class="ltx_tag ltx_tag_note">*</span>https://computational.cancer.gov/about/improve</span></span></span>, which aims to develop computational methods for the systematic evaluation and comparison of DL models across scientific domains, with DRP as a driving use case.
From over a hundred published DL-based DRP models <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib6" title="">6</a>]</cite>, we selected 17 based on the year of publication, deep learning framework used, and other criteria outlined in detail in Section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS1" title="2.1 Model selection criteria ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.1</span></a>.
Each researcher from our collaborative team analyzed one or two models.
The reusability assessment consisted of three major categories:</p>
<ol class="ltx_enumerate" id="S1.I1">
<li class="ltx_item" id="S1.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span>
<div class="ltx_para" id="S1.I1.i1.p1">
<p class="ltx_p" id="S1.I1.i1.p1.1"><span class="ltx_text ltx_font_bold" id="S1.I1.i1.p1.1.1">Software Environment</span> (Section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS2" title="2.2 Software Environment ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.2</span></a>): Scoring models based on the availability of information for setting up the computational environment.</p>
</div>
</li>
<li class="ltx_item" id="S1.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span>
<div class="ltx_para" id="S1.I1.i2.p1">
<p class="ltx_p" id="S1.I1.i2.p1.1"><span class="ltx_text ltx_font_bold" id="S1.I1.i2.p1.1.1">Code Modularity</span> (Section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS3" title="2.3 Code Modularity ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.3</span></a>): Evaluating the ability to execute major ML steps individually, including data preprocessing, model training, and inference, and assessing the description of this modularity in the documentation.</p>
</div>
</li>
<li class="ltx_item" id="S1.I1.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span>
<div class="ltx_para ltx_noindent" id="S1.I1.i3.p1">
<p class="ltx_p" id="S1.I1.i3.p1.1"><span class="ltx_text ltx_font_bold" id="S1.I1.i3.p1.1.1">Data Availability and Preprocessing</span> (Section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS4" title="2.4 Data Availability and Preprocessing ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.4</span></a>): Assessing the availability of data and preprocessing code that transforms raw data into model-ready files.</p>
</div>
</li>
</ol>
<p class="ltx_p" id="S1.p4.2">We aimed to reproduce key performance metrics for each model to ensure expected behavior and adaptability for new contexts (Section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS5" title="2.5 Reproducing key performance scores ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.5</span></a>).</p>
</div>
<div class="ltx_para ltx_noindent" id="S1.p5">
<p class="ltx_p" id="S1.p5.1">This study represents the first comprehensive assessment of model reusability and reproducibility across diverse DRP models. Evaluating 17 peer-reviewed models provides insights into current model sharing and documentation practices in the DRP research community. Our findings aim to highlight model sharing challenges and promote standards for sharing DRP and AI-enabled scientific models.
</p>
</div>
<figure class="ltx_table" id="S1.T1">
<div class="ltx_inline-block ltx_align_center ltx_transformed_outer" id="S1.T1.1" style="width:433.6pt;height:197.7pt;vertical-align:-0.6pt;"><span class="ltx_transformed_inner" style="transform:translate(-139.6pt,63.4pt) scale(0.608386851015964,0.608386851015964) ;">
<table class="ltx_tabular ltx_guessed_headers ltx_align_middle" id="S1.T1.1.1">
<thead class="ltx_thead">
<tr class="ltx_tr" id="S1.T1.1.1.1.1">
<th class="ltx_td ltx_th ltx_th_row ltx_border_tt" id="S1.T1.1.1.1.1.1"></th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_tt" id="S1.T1.1.1.1.1.2">Model</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_tt" id="S1.T1.1.1.1.1.3">Year</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_tt" id="S1.T1.1.1.1.1.4">Framework</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_tt" id="S1.T1.1.1.1.1.5">Cancer ’Omics Features</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_tt" id="S1.T1.1.1.1.1.6">Drug Features</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_tt" id="S1.T1.1.1.1.1.7">Treatment Response</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="S1.T1.1.1.2.1">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_t" id="S1.T1.1.1.2.1.1">1</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_t" id="S1.T1.1.1.2.1.2">BiG-DRP <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib21" title="">21</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_t" id="S1.T1.1.1.2.1.3">2022</th>
<td class="ltx_td ltx_align_left ltx_border_t" id="S1.T1.1.1.2.1.4">PyTorch</td>
<td class="ltx_td ltx_align_left ltx_border_t" id="S1.T1.1.1.2.1.5">GE</td>
<td class="ltx_td ltx_align_left ltx_border_t" id="S1.T1.1.1.2.1.6">DD</td>
<td class="ltx_td ltx_align_left ltx_border_t" id="S1.T1.1.1.2.1.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.3.2">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.3.2.1">2</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.3.2.2">DeepCDR <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib22" title="">22</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.3.2.3">2020</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.3.2.4">TF-Keras</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.3.2.5">GE, Methyl, Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.3.2.6">MG</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.3.2.7">IC50, Bin-IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.4.3">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.4.3.1">3</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.4.3.2">DeepTTA <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib23" title="">23</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.4.3.3">2022</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.4.3.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.4.3.5">GE</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.4.3.6">ESPF</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.4.3.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.5.4">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.5.4.1">4</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.5.4.2">DRPreter <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib24" title="">24</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.5.4.3">2022</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.5.4.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.5.4.5">GE</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.5.4.6">MG</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.5.4.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.6.5">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.6.5.1">5</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.6.5.2">DrugCell <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib25" title="">25</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.6.5.3">2020</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.6.5.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.6.5.5">Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.6.5.6">Morgan FP</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.6.5.7">AUC</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.7.6">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.7.6.1">6</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.7.6.2">DrugGCN <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib26" title="">26</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.7.6.3">2021</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.7.6.4">TF</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.7.6.5">GE</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.7.6.6">None</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.7.6.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.8.7">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.8.7.1">7</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.8.7.2">DualGCN <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib27" title="">27</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.8.7.3">2022</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.8.7.4">TF-Keras</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.8.7.5">CNV, GE</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.8.7.6">MG</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.8.7.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.9.8">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.9.8.1">8</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.9.8.2">GraphCDR <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib28" title="">28</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.9.8.3">2021</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.9.8.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.9.8.5">GE, Methyl, Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.9.8.6">MG</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.9.8.7">Bin-IC50 (GDSC), Bin-AAC (CCLE)</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.10.9">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.10.9.1">9</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.10.9.2">GraphDRP <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib29" title="">29</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.10.9.3">2022</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.10.9.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.10.9.5">CNV, Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.10.9.6">MG</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.10.9.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.11.10">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.11.10.1">10</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.11.10.2">HiDRA <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib30" title="">30</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.11.10.3">2021</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.11.10.4">TF-Keras</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.11.10.5">GE</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.11.10.6">Morgan FP</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.11.10.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.12.11">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.12.11.1">11</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.12.11.2">IGTD <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib31" title="">31</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.12.11.3">2021</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.12.11.4">TF-Keras</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.12.11.5">GE</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.12.11.6">DD</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.12.11.7">AUC</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.13.12">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.13.12.1">12</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.13.12.2">MOLI <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib32" title="">32</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.13.12.3">2019</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.13.12.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.13.12.5">CNV, GE, Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.13.12.6">None</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.13.12.7">Bin-IC50 (GDSC), Bin (PDXE, TCGA)</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.14.13">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.14.13.1">13</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.14.13.2">PathDSP <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib33" title="">33</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.14.13.3">2021</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.14.13.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.14.13.5">CNV, GE, Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.14.13.6">Morgan FP</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.14.13.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.15.14">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.15.14.1">14</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.15.14.2">SWNet <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib34" title="">34</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.15.14.3">2021</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.15.14.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.15.14.5">GE, Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.15.14.6">MG, Morgan FP</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.15.14.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.16.15">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.16.15.1">15</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.16.15.2">tCNNs <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib35" title="">35</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.16.15.3">2019</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.16.15.4">TF</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.16.15.5">CNV, Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.16.15.6">SMILES (one-hot enc.)</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.16.15.7">IC50</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.17.16">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S1.T1.1.1.17.16.1">16</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.17.16.2">TCRP <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib13" title="">13</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S1.T1.1.1.17.16.3">2021</th>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.17.16.4">PyTorch</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.17.16.5">GE, Mu</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.17.16.6">None</td>
<td class="ltx_td ltx_align_left" id="S1.T1.1.1.17.16.7">AUC</td>
</tr>
<tr class="ltx_tr" id="S1.T1.1.1.18.17">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_bb" id="S1.T1.1.1.18.17.1">17</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_bb" id="S1.T1.1.1.18.17.2">TGSA <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib36" title="">36</a>]</cite>
</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_bb" id="S1.T1.1.1.18.17.3">2022</th>
<td class="ltx_td ltx_align_left ltx_border_bb" id="S1.T1.1.1.18.17.4">PyTorch</td>
<td class="ltx_td ltx_align_left ltx_border_bb" id="S1.T1.1.1.18.17.5">CNV, GE, Mu</td>
<td class="ltx_td ltx_align_left ltx_border_bb" id="S1.T1.1.1.18.17.6">MG</td>
<td class="ltx_td ltx_align_left ltx_border_bb" id="S1.T1.1.1.18.17.7">IC50</td>
</tr>
</tbody>
</table>
</span></div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 1: </span><span class="ltx_text ltx_font_bold" id="S1.T1.6.1">Models and attributes used in this study.</span>
All models except DrugCell and TCRP were trained and predicted with the GDSC (Genomics of Drug Sensitivity in Cancer <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib37" title="">37</a>]</cite>) dataset. DrugCell was trained on combined GDSC and CTRP (Cancer Therapeutic Response Portal <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib38" title="">38</a>]</cite>) data, and TCRP was pre-trained on GDSC and fine-tuned on PDTC (patient-derived tumor cells).
<span class="ltx_text ltx_font_bold" id="S1.T1.7.2">Cancer and Drug Features</span> include commonly used representations in drug response prediction models (note that some models incorporate unique representations not detailed in this table).
The <span class="ltx_text ltx_font_bold" id="S1.T1.8.3">Treatment Response</span> indicates the outcome predicted by the model.
<span class="ltx_text ltx_font_bold" id="S1.T1.9.4">Abbreviations</span>; TF, TensorFlow; GE, gene expression; Methyl, methylation; Mu, mutation; CNV, copy number variation; DD, drug descriptors; MG, molecular graph; ESPF, explainable substructure partition fingerprint; Morgan FP, Morgan fingerprint; SMILES, simplified molecular input line entry system; IC50, half-maximal inhibitory concentration; AUC, area under the dose-response curve; Bin, binary (i.e. response divided into sensitive and resistant); AAC, area above the dose-response curve.
</figcaption>
</figure>
</section>
<section class="ltx_section" id="S2">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Methods</h2>
<div class="ltx_para ltx_noindent" id="S2.p1">
<p class="ltx_p" id="S2.p1.1">The Methods section outlines our approach for selecting community models and the subsequent analysis conducted on these models. Section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS1" title="2.1 Model selection criteria ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.1</span></a> details the criteria used for model selection, ensuring a representative and relevant subset. Sections <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS2" title="2.2 Software Environment ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.2</span></a> to <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS4" title="2.4 Data Availability and Preprocessing ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.4</span></a> describe the proposed reusability scoring system, which evaluates models based on software environment setup, code modularity, and data preprocessing. Additionally, we conducted a reproducibility analysis of key performance metrics to validate the models’ expected behavior and adaptability.</p>
</div>
<section class="ltx_subsection" id="S2.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.1 </span>Model selection criteria</h3>
<div class="ltx_para ltx_noindent" id="S2.SS1.p1">
<p class="ltx_p" id="S2.SS1.p1.1">We previously assembled a list of models that use DL methods for DRP, encompassing over one hundred original research papers. Given this collection, we utilize the following selection criteria to choose a subset of models for this study:</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS1.p2">
<ol class="ltx_enumerate" id="S2.I1">
<li class="ltx_item" id="S2.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span>
<div class="ltx_para" id="S2.I1.i1.p1">
<p class="ltx_p" id="S2.I1.i1.p1.1">The model is accompanied by open-source code, with preference to more recent publications (2019 onwards).</p>
</div>
</li>
<li class="ltx_item" id="S2.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span>
<div class="ltx_para" id="S2.I1.i2.p1">
<p class="ltx_p" id="S2.I1.i2.p1.1">The model is deep learning-based and implemented in PyTorch or TensorFlow/Keras. </p>
</div>
</li>
<li class="ltx_item" id="S2.I1.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span>
<div class="ltx_para" id="S2.I1.i3.p1">
<p class="ltx_p" id="S2.I1.i3.p1.1">The model predicts monotherapy drug response values with end-to-end learning.</p>
</div>
</li>
<li class="ltx_item" id="S2.I1.i4" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">4.</span>
<div class="ltx_para ltx_noindent" id="S2.I1.i4.p1">
<p class="ltx_p" id="S2.I1.i4.p1.1">The code repository associated with the model contains a basic description of the steps needed to use the model and reproduce the results.</p>
</div>
</li>
</ol>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS1.p3">
<p class="ltx_p" id="S2.SS1.p3.1">Following the aforementioned criteria, we selected a non-comprehensive set of 17 models that our team could successfully set up and run, summarized in Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S1.T1" title="Table 1 ‣ 1 Introduction ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">1</span></a>. This is not an exhaustive list of all models meeting the selection criteria, but rather a subset feasible to analyze within our time constraints. Thus, the exclusion of any specific model was due to time limitations and not a judgment of its reusability.</p>
</div>
</section>
<section class="ltx_subsection" id="S2.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.2 </span>Software Environment</h3>
<div class="ltx_para ltx_noindent" id="S2.SS2.p1">
<p class="ltx_p" id="S2.SS2.p1.1">Proper setup of the model’s computational environment is crucial for reproducing the results as described in the original research papers. The environment setup typically involves installing required packages using Conda or pip, and often requires specific versions to avoid compatibility issues. The required software environment is generally detailed in a configuration file (e.g., requirements.txt, environment.yml) or described in a README file associated with the repository. Correctly setting up the environment ensures that the model functions as expected, while incomplete or outdated setups can lead to errors and multiple reinstallation attempts. Repositories that fail to provide complete environment information present a challenge, especially for users who may not have deep expertise in package management or troubleshooting.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS2.p2">
<p class="ltx_p" id="S2.SS2.p2.1">Below is the scoring system used to assess the quality of the software environment information provided in the documentation of code repositories:</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS2.p3">
<p class="ltx_p" id="S2.SS2.p3.1"><span class="ltx_text ltx_font_bold" id="S2.SS2.p3.1.1">Score of 4</span>.
The repository includes a requirements file or README that lists all necessary packages and specific version numbers. This detailed information allows users to set up the exact environment required, minimizing the risk of compatibility issues or unexpected behavior.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS2.p4">
<p class="ltx_p" id="S2.SS2.p4.1"><span class="ltx_text ltx_font_bold" id="S2.SS2.p4.1.1">Score of 3</span>.
The repository contains a requirements file or README with all necessary packages but with limited or no specification of version numbers. Although users can generally set up the environment, the lack of precise version details can lead to issues with incompatible package versions.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS2.p5">
<p class="ltx_p" id="S2.SS2.p5.1"><span class="ltx_text ltx_font_bold" id="S2.SS2.p5.1.1">Score of 2</span>.
The repository includes a requirements file or README with only a subset of the required packages. This incomplete list makes it difficult for users to set up the environment properly, potentially causing significant errors and requiring additional research to identify the missing packages.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS2.p6">
<p class="ltx_p" id="S2.SS2.p6.1"><span class="ltx_text ltx_font_bold" id="S2.SS2.p6.1.1">Score of 1</span>.
The repository contains no information on the required software environment. Users would have to infer or guess which packages are needed, likely resulting in significant reproducibility issues and extensive troubleshooting.</p>
</div>
</section>
<section class="ltx_subsection" id="S2.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.3 </span>Code Modularity</h3>
<div class="ltx_para ltx_noindent" id="S2.SS3.p1">
<p class="ltx_p" id="S2.SS3.p1.1">Enhanced modularity in code repositories facilitates easier understanding of models and improves their adaptability to various contexts. In the context of a DRP workflow (and ML in general), three key components of modular code typically include data preparation, model training, and inference runs <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib6" title="">6</a>]</cite>. These steps can either be integrated into a single script or separated into individual scripts. Proper documentation is crucial for clearly understanding these steps, enabling users to reproduce, adapt, or extend the workflow. This is especially important when working with external data or modifying components of the model.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS3.p2">
<p class="ltx_p" id="S2.SS3.p2.1">A comprehensive README file in a code repository should clearly explain each step of the DRP workflow, providing sufficient detail for users to understand how to prepare data, train the model, and run inference. Command line examples and code snippets are invaluable for illustrating how to execute the three steps. The quality of documentation significantly impacts how quickly and easily users can implement the workflow without needing to refer back to the associated manuscript or other sources for guidance.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS3.p3">
<p class="ltx_p" id="S2.SS3.p3.1">Below is the scoring system used to evaluate the documentation of code repositories regarding the DRP workflow and its components:</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS3.p4">
<p class="ltx_p" id="S2.SS3.p4.1"><span class="ltx_text ltx_font_bold" id="S2.SS3.p4.1.1">Score of 4</span>.
The README provides a comprehensive explanation of each step in the DRP workflow, including data preparation, model training, and inference runs. The steps may be combined into a single script or separated into different scripts, as long as the README clearly outlines the purpose and execution order of each script. The key is to ensure that the documentation enables users to understand and reproduce the entire workflow seamlessly, regardless of the script organization.
This ensures that users can quickly implement the steps without requiring additional information from the manuscript.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS3.p5">
<p class="ltx_p" id="S2.SS3.p5.1"><span class="ltx_text ltx_font_bold" id="S2.SS3.p5.1.1">Score of 3</span>.
The README covers the entire workflow, but code examples are only provided for some steps. The explanation is mostly clear, but might have minor gaps in detail.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS3.p6">
<p class="ltx_p" id="S2.SS3.p6.1"><span class="ltx_text ltx_font_bold" id="S2.SS3.p6.1.1">Score of 2</span>.
The README provides only a partial description of the workflow, with significant details missing. Code examples are minimal or absent, and users are likely to encounter obstacles in implementing the code without additional guidance.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS3.p7">
<p class="ltx_p" id="S2.SS3.p7.1"><span class="ltx_text ltx_font_bold" id="S2.SS3.p7.1.1">Score of 1</span>.
The README does not provide any workflow explanation, nor does it offer examples or other guidance on executing the code.</p>
</div>
</section>
<section class="ltx_subsection" id="S2.SS4">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.4 </span>Data Availability and Preprocessing</h3>
<div class="ltx_para ltx_noindent" id="S2.SS4.p1">
<p class="ltx_p" id="S2.SS4.p1.1">Data preprocessing involves transforming raw data, including ’omics representations, drug features, and drug response values, into model-ready data. Scripts that perform this transformation are essential for assessing reusability in DRP models. Comprehensive preprocessing scripts enable users to test models with new data and compare results from models trained on various datasets. If a repository provides only preprocessed datasets without the corresponding raw data and preprocessing scripts, it becomes challenging, if not impossible, to adapt the model for other data sources. In certain cases, repositories share preprocessed datasets without explaining how the data were created. In this study, we excluded models due to a lack of available raw data or because the preprocessing steps were too vague to understand.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS4.p2">
<p class="ltx_p" id="S2.SS4.p2.1">Below is grading criteria we used to assess the quality and completeness of data preprocessing:</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS4.p3">
<p class="ltx_p" id="S2.SS4.p3.1"><span class="ltx_text ltx_font_bold" id="S2.SS4.p3.1.1">Score of 4</span>.
All necessary preprocessing scripts are available in the repository, and the data required to run these scripts is either included in the repository or detailed instructions are provided for download. This ensures that users can recreate the entire data preparation process from raw data to model-ready data.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS4.p4">
<p class="ltx_p" id="S2.SS4.p4.1"><span class="ltx_text ltx_font_bold" id="S2.SS4.p4.1.1">Score of 3</span>.
All necessary preprocessing scripts are available in the repository, but the data required to run these scripts, or instructions to download it, are missing. While users can understand the preprocessing steps, they cannot fully reproduce the data preparation process without the data.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS4.p5">
<p class="ltx_p" id="S2.SS4.p5.1"><span class="ltx_text ltx_font_bold" id="S2.SS4.p5.1.1">Score of 2</span>.
The repository contains limited preprocessing scripts, leaving significant gaps in the data preparation process.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS4.p6">
<p class="ltx_p" id="S2.SS4.p6.1"><span class="ltx_text ltx_font_bold" id="S2.SS4.p6.1.1">Score of 1</span>.
The repository does not provide any preprocessing scripts, making it virtually impossible to use the model with external datasets or recreate the original data preparation process.</p>
</div>
</section>
<section class="ltx_subsection" id="S2.SS5">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.5 </span>Reproducing key performance scores</h3>
<div class="ltx_para ltx_noindent" id="S2.SS5.p1">
<p class="ltx_p" id="S2.SS5.p1.1">The methods and results reported in research papers should ideally be reproducible by external researchers. Reproducing results helps ensure that a model behaves as expected and can be extended for further analysis, such as comparison with other models. In this section, we do not aim to reproduce all the results reported in the papers. Instead, our goal is to reproduce only key performance metrics from each paper listed in Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S1.T1" title="Table 1 ‣ 1 Introduction ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">1</span></a>, ensuring the proper usage of models.
In addition, to further assess the ease of reusing the models, we recorded the total time spent setting up the environment and getting the model running without errors.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS5.p2">
<p class="ltx_p" id="S2.SS5.p2.1">The reproduced scores are referred to here as <span class="ltx_text ltx_font_italic" id="S2.SS5.p2.1.1">reference scores</span>. These scores can serve as a reference, ensuring that the model’s behavior has not changed after adapting the code for other analyses.
The reference scores used in this analysis include common metrics for assessing the performance of prediction models, such as <span class="ltx_text ltx_font_italic" id="S2.SS5.p2.1.2">R squared</span>, Pearson correlation coefficient (PCC), Spearman correlation coefficient (SCC), and area under the receiver operating curve (AUROC). Since not all publications presented all scores, the choice of which scores to recreate was subjective.</p>
</div>
<div class="ltx_para ltx_noindent" id="S2.SS5.p3">
<p class="ltx_p" id="S2.SS5.p3.1">Our goal was to minimize code modifications to preserve the authentic code structure as much as possible. Whenever feasible, we preserved the original data splits (such as folds or random seeds) and data partitions (like cell line-blind, drug-blind, or random). Changes to these partitions can substantially influence a model’s performance, though delving into such effects is outside the scope of this study.</p>
</div>
<figure class="ltx_table" id="S2.T2">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 2: </span>A summary of the scoring system for model reproducibility <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS1" title="2.1 Model selection criteria ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.1</span></a>.</figcaption>
<table class="ltx_tabular ltx_guessed_headers ltx_align_middle" id="S2.T2.1">
<thead class="ltx_thead">
<tr class="ltx_tr" id="S2.T2.1.1.1">
<th class="ltx_td ltx_th ltx_th_column ltx_border_r ltx_border_tt" id="S2.T2.1.1.1.1"></th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S2.T2.1.1.1.2">Score</th>
<th class="ltx_td ltx_align_justify ltx_align_top ltx_th ltx_th_column ltx_border_tt" id="S2.T2.1.1.1.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.1.1.3.1">
<span class="ltx_p" id="S2.T2.1.1.1.3.1.1" style="width:227.6pt;">Description</span>
</span>
</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="S2.T2.1.2.1">
<td class="ltx_td ltx_align_left ltx_border_r ltx_border_t" id="S2.T2.1.2.1.1">Software Environment (section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS2" title="2.2 Software Environment ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.2</span></a>)</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S2.T2.1.2.1.2">1</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="S2.T2.1.2.1.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.2.1.3.1">
<span class="ltx_p" id="S2.T2.1.2.1.3.1.1" style="width:227.6pt;">No information on required software environment.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.3.2">
<td class="ltx_td ltx_border_r" id="S2.T2.1.3.2.1"></td>
<td class="ltx_td ltx_align_center" id="S2.T2.1.3.2.2">2</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="S2.T2.1.3.2.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.3.2.3.1">
<span class="ltx_p" id="S2.T2.1.3.2.3.1.1" style="width:227.6pt;">Only a subset of required packages is specified.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.4.3">
<td class="ltx_td ltx_border_r" id="S2.T2.1.4.3.1"></td>
<td class="ltx_td ltx_align_center" id="S2.T2.1.4.3.2">3</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="S2.T2.1.4.3.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.4.3.3.1">
<span class="ltx_p" id="S2.T2.1.4.3.3.1.1" style="width:227.6pt;">All packages are listed, but version numbers are limited or missing.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.5.4">
<td class="ltx_td ltx_border_r" id="S2.T2.1.5.4.1"></td>
<td class="ltx_td ltx_align_center" id="S2.T2.1.5.4.2">4</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="S2.T2.1.5.4.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.5.4.3.1">
<span class="ltx_p" id="S2.T2.1.5.4.3.1.1" style="width:227.6pt;">All packages with specific version numbers are listed in the requirements file or README.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.6.5">
<td class="ltx_td ltx_align_left ltx_border_r ltx_border_t" id="S2.T2.1.6.5.1">Code Modularity (section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS3" title="2.3 Code Modularity ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.3</span></a>)</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S2.T2.1.6.5.2">1</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="S2.T2.1.6.5.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.6.5.3.1">
<span class="ltx_p" id="S2.T2.1.6.5.3.1.1" style="width:227.6pt;">The README has no workflow explanation or code examples.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.7.6">
<td class="ltx_td ltx_border_r" id="S2.T2.1.7.6.1"></td>
<td class="ltx_td ltx_align_center" id="S2.T2.1.7.6.2">2</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="S2.T2.1.7.6.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.7.6.3.1">
<span class="ltx_p" id="S2.T2.1.7.6.3.1.1" style="width:227.6pt;">The README offers only a partial workflow description, with limited or no code examples.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.8.7">
<td class="ltx_td ltx_border_r" id="S2.T2.1.8.7.1"></td>
<td class="ltx_td ltx_align_center" id="S2.T2.1.8.7.2">3</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="S2.T2.1.8.7.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.8.7.3.1">
<span class="ltx_p" id="S2.T2.1.8.7.3.1.1" style="width:227.6pt;">The README covers the entire workflow, but code examples are provided for only some steps.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.9.8">
<td class="ltx_td ltx_border_r" id="S2.T2.1.9.8.1"></td>
<td class="ltx_td ltx_align_center" id="S2.T2.1.9.8.2">4</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="S2.T2.1.9.8.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.9.8.3.1">
<span class="ltx_p" id="S2.T2.1.9.8.3.1.1" style="width:227.6pt;">The README comprehensively explains every step of the DRP workflow, including code examples.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.10.9">
<td class="ltx_td ltx_align_left ltx_border_r ltx_border_t" id="S2.T2.1.10.9.1">Data Availability and Preprocessing (sec. <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS4" title="2.4 Data Availability and Preprocessing ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.4</span></a>)</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S2.T2.1.10.9.2">1</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="S2.T2.1.10.9.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.10.9.3.1">
<span class="ltx_p" id="S2.T2.1.10.9.3.1.1" style="width:227.6pt;">No preprocessing scripts are provided.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.11.10">
<td class="ltx_td ltx_border_r" id="S2.T2.1.11.10.1"></td>
<td class="ltx_td ltx_align_center" id="S2.T2.1.11.10.2">2</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="S2.T2.1.11.10.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.11.10.3.1">
<span class="ltx_p" id="S2.T2.1.11.10.3.1.1" style="width:227.6pt;">Only some preprocessing scripts are available.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.12.11">
<td class="ltx_td ltx_border_r" id="S2.T2.1.12.11.1"></td>
<td class="ltx_td ltx_align_center" id="S2.T2.1.12.11.2">3</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="S2.T2.1.12.11.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.12.11.3.1">
<span class="ltx_p" id="S2.T2.1.12.11.3.1.1" style="width:227.6pt;">All preprocessing scripts are included, but data or download instructions are missing.</span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="S2.T2.1.13.12">
<td class="ltx_td ltx_border_bb ltx_border_r" id="S2.T2.1.13.12.1"></td>
<td class="ltx_td ltx_align_center ltx_border_bb" id="S2.T2.1.13.12.2">4</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="S2.T2.1.13.12.3">
<span class="ltx_inline-block ltx_align_top" id="S2.T2.1.13.12.3.1">
<span class="ltx_p" id="S2.T2.1.13.12.3.1.1" style="width:227.6pt;">All preprocessing scripts and data (or clear download instructions) are provided.</span>
</span>
</td>
</tr>
</tbody>
</table>
</figure>
</section>
</section>
<section class="ltx_section" id="S3">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Results</h2>
<div class="ltx_para ltx_noindent" id="S3.p1">
<p class="ltx_p" id="S3.p1.1">The results summarized below are structured according to the reusability scoring system and reproducibility of references scores described in section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2" title="2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2</span></a>. Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.T3" title="Table 3 ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">3</span></a> and Figures <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.F1" title="Figure 1 ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">1</span></a> and <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.F2" title="Figure 2 ‣ 3.4 Reproducing key performance scores ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2</span></a> present the statistical details of these results.</p>
</div>
<figure class="ltx_table" id="S3.T3">
<div class="ltx_inline-block ltx_align_center ltx_transformed_outer" id="S3.T3.1" style="width:433.6pt;height:311.5pt;vertical-align:-1.0pt;"><span class="ltx_transformed_inner" style="transform:translate(-9.4pt,6.8pt) scale(0.958304217139887,0.958304217139887) ;">
<table class="ltx_tabular ltx_guessed_headers ltx_align_middle" id="S3.T3.1.1">
<thead class="ltx_thead">
<tr class="ltx_tr" id="S3.T3.1.1.1.1">
<th class="ltx_td ltx_th ltx_th_row ltx_border_tt" id="S3.T3.1.1.1.1.1"></th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_tt" id="S3.T3.1.1.1.1.2">Model</th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S3.T3.1.1.1.1.3">Year</th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S3.T3.1.1.1.1.4">Environment Score</th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S3.T3.1.1.1.1.5">Modularity Score</th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S3.T3.1.1.1.1.6">Preprocessing Score</th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S3.T3.1.1.1.1.7">Time (h)</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="S3.T3.1.1.2.1">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_t" id="S3.T3.1.1.2.1.1">1</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_t" id="S3.T3.1.1.2.1.2">BiG-DRP</th>
<td class="ltx_td ltx_align_center ltx_border_t" id="S3.T3.1.1.2.1.3">2022</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S3.T3.1.1.2.1.4">3</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S3.T3.1.1.2.1.5">4</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S3.T3.1.1.2.1.6">1</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S3.T3.1.1.2.1.7">3</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.3.2">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.3.2.1">2</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.3.2.2">DeepCDR</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.3.2.3">2020</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.3.2.4">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.3.2.5">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.3.2.6">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.3.2.7">8</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.4.3">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.4.3.1">3</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.4.3.2">DeepTTA</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.4.3.3">2022</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.4.3.4">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.4.3.5">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.4.3.6">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.4.3.7">5</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.5.4">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.5.4.1">4</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.5.4.2">DRPreter</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.5.4.3">2022</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.5.4.4">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.5.4.5">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.5.4.6">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.5.4.7">0.5</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.6.5">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.6.5.1">5</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.6.5.2">DrugCell</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.6.5.3">2020</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.6.5.4">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.6.5.5">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.6.5.6">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.6.5.7">0.1</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.7.6">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.7.6.1">6</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.7.6.2">DrugGCN</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.7.6.3">2021</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.7.6.4">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.7.6.5">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.7.6.6">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.7.6.7">11</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.8.7">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.8.7.1">7</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.8.7.2">DualGCN</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.8.7.3">2022</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.8.7.4">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.8.7.5">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.8.7.6">1</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.8.7.7">12</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.9.8">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.9.8.1">8</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.9.8.2">GraphCDR</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.9.8.3">2021</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.9.8.4">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.9.8.5">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.9.8.6">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.9.8.7">2</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.10.9">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.10.9.1">9</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.10.9.2">GraphDRP</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.10.9.3">2022</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.10.9.4">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.10.9.5">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.10.9.6">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.10.9.7">8</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.11.10">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.11.10.1">10</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.11.10.2">HiDRA</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.11.10.3">2021</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.11.10.4">1</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.11.10.5">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.11.10.6">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.11.10.7">10</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.12.11">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.12.11.1">11</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.12.11.2">IGTD</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.12.11.3">2021</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.12.11.4">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.12.11.5">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.12.11.6">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.12.11.7">0.5</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.13.12">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.13.12.1">12</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.13.12.2">MOLI</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.13.12.3">2019</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.13.12.4">1</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.13.12.5">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.13.12.6">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.13.12.7">8</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.14.13">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.14.13.1">13</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.14.13.2">PathDSP</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.14.13.3">2021</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.14.13.4">1</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.14.13.5">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.14.13.6">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.14.13.7">2</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.15.14">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.15.14.1">14</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.15.14.2">SWNet</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.15.14.3">2021</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.15.14.4">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.15.14.5">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.15.14.6">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.15.14.7">3</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.16.15">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.16.15.1">15</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.16.15.2">tCNNs</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.16.15.3">2019</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.16.15.4">1</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.16.15.5">1</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.16.15.6">4</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.16.15.7">2</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.17.16">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.17.16.1">16</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S3.T3.1.1.17.16.2">TCRP</th>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.17.16.3">2021</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.17.16.4">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.17.16.5">3</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.17.16.6">2</td>
<td class="ltx_td ltx_align_center" id="S3.T3.1.1.17.16.7">10</td>
</tr>
<tr class="ltx_tr" id="S3.T3.1.1.18.17">
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_bb" id="S3.T3.1.1.18.17.1">17</th>
<th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_bb" id="S3.T3.1.1.18.17.2">TGSA</th>
<td class="ltx_td ltx_align_center ltx_border_bb" id="S3.T3.1.1.18.17.3">2022</td>
<td class="ltx_td ltx_align_center ltx_border_bb" id="S3.T3.1.1.18.17.4">3</td>
<td class="ltx_td ltx_align_center ltx_border_bb" id="S3.T3.1.1.18.17.5">3</td>
<td class="ltx_td ltx_align_center ltx_border_bb" id="S3.T3.1.1.18.17.6">3</td>
<td class="ltx_td ltx_align_center ltx_border_bb" id="S3.T3.1.1.18.17.7">4</td>
</tr>
</tbody>
</table>
</span></div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 3: </span><span class="ltx_text ltx_font_bold" id="S3.T3.3.1">List of models and scoring results</span>. Scoring results obtained as part of the reusability and reproducibility analysis are listed for the models. The scoring metrics include software environment (section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.SS1" title="3.1 Software Environment ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">3.1</span></a>), code modularity (section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.SS2" title="3.2 Code Modularity ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">3.2</span></a>), and data availability and preprocessing (section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.SS3" title="3.3 Data Availability and Preprocessing ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">3.3</span></a>). The column Time indicates the time required to set up the environment and run the model without errors.</figcaption>
</figure>
<figure class="ltx_figure" id="S3.F1"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="225" id="S3.F1.g1" src="extracted/5861452/score_histograms.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>
Histograms showing the distribution of model scores for each category, with the number of models shown above each bar.
</figcaption>
</figure>
<section class="ltx_subsection" id="S3.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.1 </span>Software Environment</h3>
<div class="ltx_para ltx_noindent" id="S3.SS1.p1">
<p class="ltx_p" id="S3.SS1.p1.1">Seven of 17 models successfully met the full criteria, with the software environment containing all required packages and their respective version numbers (see Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.T3" title="Table 3 ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">3</span></a> and Figure <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.F1" title="Figure 1 ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">1</span></a>).
For models with incomplete or missing environment descriptions, we determined compatible package versions through trial and error or by directly contacting the authors.
Outdated package versions can also increase time required for installation. DRPreter specified a version of PyTorch no longer supported by Conda, so we manually pip installed the package.
According to some users, installing compatible dependencies was the most time-consuming step in the entire study.
Conversely, DrugCell, which was containerized with Docker, required the least amount of time to set up and run.
We observe that rebuilding environments can be a time-consuming and error-prone process, even for experienced users, often requiring trial and error to resolve dependency conflicts and ensure proper configuration.</p>
</div>
</section>
<section class="ltx_subsection" id="S3.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.2 </span>Code Modularity</h3>
<div class="ltx_para ltx_noindent" id="S3.SS2.p1">
<p class="ltx_p" id="S3.SS2.p1.1">According to our scoring system described in Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.T2" title="Table 2 ‣ 2.5 Reproducing key performance scores ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2</span></a>, the highest score is awarded to models that provided command line examples for each step of the DRP workflow in their documentation, including preprocessing, training, and inference.
These models are more likely to be reusable in various contexts.
As shown in Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.T3" title="Table 3 ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">3</span></a> and Figure <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.F1" title="Figure 1 ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">1</span></a>, only four models (24% of the total) met this requirement by providing command-line examples for the entire workflow. Another seven models had documentation that explained the complete workflow but lacked command line examples for the individual steps. The top-scoring models, and particularly SWNet, described data files, included installation instructions, and gave code examples for different use cases. It is important to note that a qualitative assessment of the documentation was a factor in the model selection process (see Section <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.SS1" title="2.1 Model selection criteria ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2.1</span></a>). Consequently, many models were discarded during the initial selection phase due to inadequate documentation.</p>
</div>
</section>
<section class="ltx_subsection" id="S3.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.3 </span>Data Availability and Preprocessing</h3>
<div class="ltx_para ltx_noindent" id="S3.SS3.p1">
<p class="ltx_p" id="S3.SS3.p1.1">Based on our analysis, eight models provided both the data and the preprocessing scripts required to transform it into the form required by the model, earning a score of 4 in this category.
The repositories varied in the information they provided, ranging from model-ready datasets without preprocessing scripts to full preprocessing steps but without the raw data required for preprocessing.
Only two models scored 1, lacking both a preprocessing script and raw data to generate model-ready files.
Common challenges in this category included hard-coded paths, missing preprocessing functions, and improperly structured files in the directory tree, all of which generally required troubleshooting.
Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.T2" title="Table 2 ‣ 2.5 Reproducing key performance scores ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2</span></a> and Figure <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.F1" title="Figure 1 ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">1</span></a> show the distribution of scores, reflecting availability of data and preprocessing scripts.</p>
</div>
</section>
<section class="ltx_subsection" id="S3.SS4">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.4 </span>Reproducing key performance scores</h3>
<div class="ltx_para ltx_noindent" id="S3.SS4.p1">
<p class="ltx_p" id="S3.SS4.p1.1">The time taken to re-implement the models, excluding the actual model training time, varied widely, ranging from less than an hour to over twelve hours, as reported in Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.T3" title="Table 3 ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">3</span></a>, with a median time of 4 hours.
The reported times for individual models were influenced, in part, by the familiarity of the model curators with the DL framework and model architecture. Furthermore, challenges such as setting up the required environment, missing code or data files, and minor bugs like hard-coded paths were reported as significant factors contributing to the longer setup times. In contrast, DrugCell, which was containerized using Docker, proved to be the fastest model to set up.</p>
</div>
<div class="ltx_para ltx_noindent" id="S3.SS4.p2">
<p class="ltx_p" id="S3.SS4.p2.1">Due to the difference in evaluation schemes and reported metrics across publications, the reproducibility results between various models are not directly comparable.
We quantify reproducibility using the percentage deviation of the reproduced score from the original one with </p>
</div>
<div class="ltx_para ltx_noindent" id="S3.SS4.p3">
<table class="ltx_equation ltx_eqn_table" id="S3.E1">
<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math alttext="d=100*(m_{r}-m_{o})/m_{o}," class="ltx_Math" display="block" id="S3.E1.m1.1"><semantics id="S3.E1.m1.1a"><mrow id="S3.E1.m1.1.1.1" xref="S3.E1.m1.1.1.1.1.cmml"><mrow id="S3.E1.m1.1.1.1.1" xref="S3.E1.m1.1.1.1.1.cmml"><mi id="S3.E1.m1.1.1.1.1.3" xref="S3.E1.m1.1.1.1.1.3.cmml">d</mi><mo id="S3.E1.m1.1.1.1.1.2" xref="S3.E1.m1.1.1.1.1.2.cmml">=</mo><mrow id="S3.E1.m1.1.1.1.1.1" xref="S3.E1.m1.1.1.1.1.1.cmml"><mrow id="S3.E1.m1.1.1.1.1.1.1" xref="S3.E1.m1.1.1.1.1.1.1.cmml"><mn id="S3.E1.m1.1.1.1.1.1.1.3" xref="S3.E1.m1.1.1.1.1.1.1.3.cmml">100</mn><mo id="S3.E1.m1.1.1.1.1.1.1.2" lspace="0.222em" rspace="0.222em" xref="S3.E1.m1.1.1.1.1.1.1.2.cmml">∗</mo><mrow id="S3.E1.m1.1.1.1.1.1.1.1.1" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.cmml"><mo id="S3.E1.m1.1.1.1.1.1.1.1.1.2" stretchy="false" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.cmml">(</mo><mrow id="S3.E1.m1.1.1.1.1.1.1.1.1.1" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.cmml"><msub id="S3.E1.m1.1.1.1.1.1.1.1.1.1.2" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.cmml"><mi id="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.2" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.2.cmml">m</mi><mi id="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.3" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">r</mi></msub><mo id="S3.E1.m1.1.1.1.1.1.1.1.1.1.1" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.1.cmml">−</mo><msub id="S3.E1.m1.1.1.1.1.1.1.1.1.1.3" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.cmml"><mi id="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.2" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.2.cmml">m</mi><mi id="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.3" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.3.cmml">o</mi></msub></mrow><mo id="S3.E1.m1.1.1.1.1.1.1.1.1.3" stretchy="false" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S3.E1.m1.1.1.1.1.1.2" xref="S3.E1.m1.1.1.1.1.1.2.cmml">/</mo><msub id="S3.E1.m1.1.1.1.1.1.3" xref="S3.E1.m1.1.1.1.1.1.3.cmml"><mi id="S3.E1.m1.1.1.1.1.1.3.2" xref="S3.E1.m1.1.1.1.1.1.3.2.cmml">m</mi><mi id="S3.E1.m1.1.1.1.1.1.3.3" xref="S3.E1.m1.1.1.1.1.1.3.3.cmml">o</mi></msub></mrow></mrow><mo id="S3.E1.m1.1.1.1.2" xref="S3.E1.m1.1.1.1.1.cmml">,</mo></mrow><annotation-xml encoding="MathML-Content" id="S3.E1.m1.1b"><apply id="S3.E1.m1.1.1.1.1.cmml" xref="S3.E1.m1.1.1.1"><eq id="S3.E1.m1.1.1.1.1.2.cmml" xref="S3.E1.m1.1.1.1.1.2"></eq><ci id="S3.E1.m1.1.1.1.1.3.cmml" xref="S3.E1.m1.1.1.1.1.3">𝑑</ci><apply id="S3.E1.m1.1.1.1.1.1.cmml" xref="S3.E1.m1.1.1.1.1.1"><divide id="S3.E1.m1.1.1.1.1.1.2.cmml" xref="S3.E1.m1.1.1.1.1.1.2"></divide><apply id="S3.E1.m1.1.1.1.1.1.1.cmml" xref="S3.E1.m1.1.1.1.1.1.1"><times id="S3.E1.m1.1.1.1.1.1.1.2.cmml" xref="S3.E1.m1.1.1.1.1.1.1.2"></times><cn id="S3.E1.m1.1.1.1.1.1.1.3.cmml" type="integer" xref="S3.E1.m1.1.1.1.1.1.1.3">100</cn><apply id="S3.E1.m1.1.1.1.1.1.1.1.1.1.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1"><minus id="S3.E1.m1.1.1.1.1.1.1.1.1.1.1.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.1"></minus><apply id="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.1.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.2">subscript</csymbol><ci id="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.2.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.2">𝑚</ci><ci id="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.2.3">𝑟</ci></apply><apply id="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.3"><csymbol cd="ambiguous" id="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.1.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.3">subscript</csymbol><ci id="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.2.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.2">𝑚</ci><ci id="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.3.cmml" xref="S3.E1.m1.1.1.1.1.1.1.1.1.1.3.3">𝑜</ci></apply></apply></apply><apply id="S3.E1.m1.1.1.1.1.1.3.cmml" xref="S3.E1.m1.1.1.1.1.1.3"><csymbol cd="ambiguous" id="S3.E1.m1.1.1.1.1.1.3.1.cmml" xref="S3.E1.m1.1.1.1.1.1.3">subscript</csymbol><ci id="S3.E1.m1.1.1.1.1.1.3.2.cmml" xref="S3.E1.m1.1.1.1.1.1.3.2">𝑚</ci><ci id="S3.E1.m1.1.1.1.1.1.3.3.cmml" xref="S3.E1.m1.1.1.1.1.1.3.3">𝑜</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.E1.m1.1c">d=100*(m_{r}-m_{o})/m_{o},</annotation><annotation encoding="application/x-llamapun" id="S3.E1.m1.1d">italic_d = 100 ∗ ( italic_m start_POSTSUBSCRIPT italic_r end_POSTSUBSCRIPT - italic_m start_POSTSUBSCRIPT italic_o end_POSTSUBSCRIPT ) / italic_m start_POSTSUBSCRIPT italic_o end_POSTSUBSCRIPT ,</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right" rowspan="1"><span class="ltx_tag ltx_tag_equation ltx_align_right">(1)</span></td>
</tr></tbody>
</table>
</div>
<div class="ltx_para ltx_noindent" id="S3.SS4.p4">
<p class="ltx_p" id="S3.SS4.p4.3">where <math alttext="m_{o}" class="ltx_Math" display="inline" id="S3.SS4.p4.1.m1.1"><semantics id="S3.SS4.p4.1.m1.1a"><msub id="S3.SS4.p4.1.m1.1.1" xref="S3.SS4.p4.1.m1.1.1.cmml"><mi id="S3.SS4.p4.1.m1.1.1.2" xref="S3.SS4.p4.1.m1.1.1.2.cmml">m</mi><mi id="S3.SS4.p4.1.m1.1.1.3" xref="S3.SS4.p4.1.m1.1.1.3.cmml">o</mi></msub><annotation-xml encoding="MathML-Content" id="S3.SS4.p4.1.m1.1b"><apply id="S3.SS4.p4.1.m1.1.1.cmml" xref="S3.SS4.p4.1.m1.1.1"><csymbol cd="ambiguous" id="S3.SS4.p4.1.m1.1.1.1.cmml" xref="S3.SS4.p4.1.m1.1.1">subscript</csymbol><ci id="S3.SS4.p4.1.m1.1.1.2.cmml" xref="S3.SS4.p4.1.m1.1.1.2">𝑚</ci><ci id="S3.SS4.p4.1.m1.1.1.3.cmml" xref="S3.SS4.p4.1.m1.1.1.3">𝑜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS4.p4.1.m1.1c">m_{o}</annotation><annotation encoding="application/x-llamapun" id="S3.SS4.p4.1.m1.1d">italic_m start_POSTSUBSCRIPT italic_o end_POSTSUBSCRIPT</annotation></semantics></math> and <math alttext="m_{r}" class="ltx_Math" display="inline" id="S3.SS4.p4.2.m2.1"><semantics id="S3.SS4.p4.2.m2.1a"><msub id="S3.SS4.p4.2.m2.1.1" xref="S3.SS4.p4.2.m2.1.1.cmml"><mi id="S3.SS4.p4.2.m2.1.1.2" xref="S3.SS4.p4.2.m2.1.1.2.cmml">m</mi><mi id="S3.SS4.p4.2.m2.1.1.3" xref="S3.SS4.p4.2.m2.1.1.3.cmml">r</mi></msub><annotation-xml encoding="MathML-Content" id="S3.SS4.p4.2.m2.1b"><apply id="S3.SS4.p4.2.m2.1.1.cmml" xref="S3.SS4.p4.2.m2.1.1"><csymbol cd="ambiguous" id="S3.SS4.p4.2.m2.1.1.1.cmml" xref="S3.SS4.p4.2.m2.1.1">subscript</csymbol><ci id="S3.SS4.p4.2.m2.1.1.2.cmml" xref="S3.SS4.p4.2.m2.1.1.2">𝑚</ci><ci id="S3.SS4.p4.2.m2.1.1.3.cmml" xref="S3.SS4.p4.2.m2.1.1.3">𝑟</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.SS4.p4.2.m2.1c">m_{r}</annotation><annotation encoding="application/x-llamapun" id="S3.SS4.p4.2.m2.1d">italic_m start_POSTSUBSCRIPT italic_r end_POSTSUBSCRIPT</annotation></semantics></math>, are respectively, the original and the reproduced scores, and <math alttext="d" class="ltx_Math" display="inline" id="S3.SS4.p4.3.m3.1"><semantics id="S3.SS4.p4.3.m3.1a"><mi id="S3.SS4.p4.3.m3.1.1" xref="S3.SS4.p4.3.m3.1.1.cmml">d</mi><annotation-xml encoding="MathML-Content" id="S3.SS4.p4.3.m3.1b"><ci id="S3.SS4.p4.3.m3.1.1.cmml" xref="S3.SS4.p4.3.m3.1.1">𝑑</ci></annotation-xml><annotation encoding="application/x-tex" id="S3.SS4.p4.3.m3.1c">d</annotation><annotation encoding="application/x-llamapun" id="S3.SS4.p4.3.m3.1d">italic_d</annotation></semantics></math> is the percentage deviation.</p>
</div>
<div class="ltx_para ltx_noindent" id="S3.SS4.p5">
<p class="ltx_p" id="S3.SS4.p5.1">As shown in Figure <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.F2" title="Figure 2 ‣ 3.4 Reproducing key performance scores ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2</span></a>, eleven models demonstrated deviations of less than 5% for all selected metrics. Of the remaining six models, BiG-DRP and DeepTTA exhibited substantial deviation in root mean squared error (RMSE) but not in other metrics.
For BiG-DRP, the differences in RMSE between the published and reproduced results appear to be due to discrepancies between the model implemented in the code repository and the results published in the paper. Specifically, the paper reports RMSE using z-score normalized IC50 values, whereas the repository produces non-normalized scores.
DeepTTA showed an approximately 10% deviation in RMSE, but the deviations in PCC and SCC were relatively small (less than 2%).
The performance of TCRP was evaluated on a per-drug basis <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib13" title="">13</a>]</cite>. To assess reproducibility, we selected the drug with the highest reported PCC value from the paper. Our analysis revealed a deviation of slightly over 5% between the reported and reproduced PCC values for that specific drug.
Lastly, our analysis of the DrugCell model showed the highest deviation of approximately 14% when comparing the original and reproduced SCC scores.
</p>
</div>
<figure class="ltx_figure" id="S3.F2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="333" id="S3.F2.g1" src="extracted/5861452/PCC_scores.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Percentage deviation of reproduced metrics, where the indicated metric is published. A negative score (red) means the reproduced value is lower than the published value. Pearson’s correlation coefficient (PCC), coefficient of determination (<math alttext="R^{2}" class="ltx_Math" display="inline" id="S3.F2.2.m1.1"><semantics id="S3.F2.2.m1.1b"><msup id="S3.F2.2.m1.1.1" xref="S3.F2.2.m1.1.1.cmml"><mi id="S3.F2.2.m1.1.1.2" xref="S3.F2.2.m1.1.1.2.cmml">R</mi><mn id="S3.F2.2.m1.1.1.3" xref="S3.F2.2.m1.1.1.3.cmml">2</mn></msup><annotation-xml encoding="MathML-Content" id="S3.F2.2.m1.1c"><apply id="S3.F2.2.m1.1.1.cmml" xref="S3.F2.2.m1.1.1"><csymbol cd="ambiguous" id="S3.F2.2.m1.1.1.1.cmml" xref="S3.F2.2.m1.1.1">superscript</csymbol><ci id="S3.F2.2.m1.1.1.2.cmml" xref="S3.F2.2.m1.1.1.2">𝑅</ci><cn id="S3.F2.2.m1.1.1.3.cmml" type="integer" xref="S3.F2.2.m1.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.F2.2.m1.1d">R^{2}</annotation><annotation encoding="application/x-llamapun" id="S3.F2.2.m1.1e">italic_R start_POSTSUPERSCRIPT 2 end_POSTSUPERSCRIPT</annotation></semantics></math>), area under the curve (AUC), and Spearman correlation coefficient (SCC) shown.</figcaption>
</figure>
</section>
</section>
<section class="ltx_section" id="S4">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Discussion and Recommendations</h2>
<div class="ltx_para ltx_noindent" id="S4.p1">
<p class="ltx_p" id="S4.p1.1">Multiple validation and assessment studies have analyzed various aspects of DRP models, such as generalization across cell-line studies <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib39" title="">39</a>]</cite>, data scaling properties <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib40" title="">40</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib41" title="">41</a>]</cite>, drug representations <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib42" title="">42</a>, <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib43" title="">43</a>]</cite>, and interpretability <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib44" title="">44</a>]</cite>, while primarily focusing on prediction performance. The current study, however, provides a different view of examining models, specifically geared towards reusability and extensibility rather than prediction performance. The goal here is not to compare models, but rather to understand what aspects of reusability are most overlooked and should receive more attention from developers in the future.
In this paper, we introduce a scoring system to quantify the reusability of prediction models (Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S2.T2" title="Table 2 ‣ 2.5 Reproducing key performance scores ‣ 2 Methods ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2</span></a>) and rigorously apply it to 17 DRP models sourced from peer-reviewed literature (Table <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S1.T1" title="Table 1 ‣ 1 Introduction ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">1</span></a>).
Our analysis reveals that the vast majority of models fall short of perfect scores, indicating potential challenges that external users may encounter when attempting to utilize or build upon these models.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p2">
<p class="ltx_p" id="S4.p2.1">Underspecified software environments force users to determine required package versions through trial and error.
Missing description and examples of the workflow require users to sift through various scripts in the repository in an attempt to piece together a functioning pipeline, further complicating code reusability.
These challenges significantly increase the time needed to understand and reimplement a model.
Notably, data preprocessing steps are frequently omitted, making dataset reconstruction challenging or even impossible, which directly impacts the ability to apply the model to new datasets or extend the model to various analyses.
</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p3">
<p class="ltx_p" id="S4.p3.1">We find reference scores are a useful tool for diagnosing problems while reimplementing a model, and when altering a model to accept a new dataset or enhance performance. Time to obtain reference scores is significantly reduced by providing a container for the model. Most models show moderate underperformance compared to original metrics (Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#S3.F2" title="Figure 2 ‣ 3.4 Reproducing key performance scores ‣ 3 Results ‣ Assessing Reusability of Deep Learning-Based Monotherapy Drug Response Prediction Models Trained with Omics Data"><span class="ltx_text ltx_ref_tag">2</span></a>), but a low reference score does not mean that a model’s approach or design is flawed. Rather, the case may be that the publication did not provide sufficient steps to recreate the model without prior knowledge.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p4">
<p class="ltx_p" id="S4.p4.1">Challenges reported in this study align with findings from the work of So et al. <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib12" title="">12</a>]</cite>, who investigated the reproducibility and reusability of the TCRP model proposed by Ma et al <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2409.12215v1#bib.bib13" title="">13</a>]</cite>. Ma et al. did not provide final hyperparameters, requiring reoptimization and making comparisons between results difficult. Including command line examples with parameters could address this issue. Also, as So et al. noted, some drug results were not replicable. We considered TCRP’s preprocessing code to be incomplete, which is reflected in a score of 2 for the ’Data Availability and Preprocessing’ category.
Our PCC reproducibility score for TCRP, based on KU-55933, is 0.53 compared to the published 0.56 and So et al.’s 0.38. While all results show KU-55933 as the top scorer with similar trends between drugs, the reasons for the substantial differences in PCC are unclear. These reimplementation challenges, though not invalidating the model, make it harder to reuse and limit its overall impact.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p5">
<p class="ltx_p" id="S4.p5.1">Despite our efforts to create clear and straightforward guidelines for the scoring system, individual bias may still influence scoring. After the initial submission of scores, we cross-reviewed all entries, identified instances where categories were incorrectly scored, updated the scores, and confirmed them with the original scorers.
As large language models (LLMs) are extensively explored in various scientific domains, they can automate redundant and clearly defined tasks. LLMs can be fine-tuned to auto-score model repositories based on the proposed scoring system, assisting developers in sharing high-quality models and helping the community analyze the potential reusability of public models. </p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p6">
<p class="ltx_p" id="S4.p6.1">We observe that many papers introduce novel DRP approaches to improve prediction performance, often claiming state-of-the-art results, while the field lacks agreed-upon benchmark datasets. This suggests that research groups often work in isolation rather than collaboratively, with authors typically not expecting others to adopt and extend their work. To promote rigorous practices and open-source sharing, the field would benefit from clear, standardized guidelines for developing and sharing DRP models. Adhering to these guidelines can mitigate many of the identified issues in this study, enhancing model reusability without imposing significant additional burdens on researchers.
Below, we provide recommendations to achieve this.
</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p7">
<p class="ltx_p" id="S4.p7.1"><span class="ltx_text ltx_font_bold" id="S4.p7.1.1">Recommendation 1</span>: <span class="ltx_text ltx_font_bold" id="S4.p7.1.2">Environment Setup and Containerization</span>.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p8">
<p class="ltx_p" id="S4.p8.1">The code repository should include clear instructions for recreating the computational environment required for the model. This typically involves providing a requirements.txt file (pip installation) or an environment.yml (Conda environment), ideally accompanied by command-line instructions for environment setup.
Furthermore, including containerization files, such as Dockerfile for Docker or definition file (ending with <span class="ltx_text ltx_font_italic" id="S4.p8.1.1">.def</span>) for Singularity, along with instructions for building and running containers can significantly enhance usability and mitigate hardware-related compatibility issues. Specifying the exact versions of key dependencies and Python version is critical.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p9">
<p class="ltx_p" id="S4.p9.1"><span class="ltx_text ltx_font_bold" id="S4.p9.1.1">Recommendation 2</span>: <span class="ltx_text ltx_font_bold" id="S4.p9.1.2">Data and Preprocessing Scripts</span>.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p10">
<p class="ltx_p" id="S4.p10.1">The code repository should include all necessary data preprocessing scripts, and ideally, all the necessary data itself. This includes incorporating feature selection, both by providing explicit feature lists and by automating their selection during preprocessing and dataset generation stages. If data inclusion is restricted due to size or licensing constraints, the repository should provide scripts or clear instructions to facilitate data acquisition. Automating data acquisition and preprocessing eliminates the need for users to manually reproduce complex preprocessing steps based on often incomplete documentation.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p11">
<p class="ltx_p" id="S4.p11.1"><span class="ltx_text ltx_font_bold" id="S4.p11.1.1">Recommendation 3</span>: <span class="ltx_text ltx_font_bold" id="S4.p11.1.2">Code Modularity and Documentation</span>.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p12">
<p class="ltx_p" id="S4.p12.1">To enhance understanding and adaptability, ensure that your code repository is modular, separating key components of the DRP workflow—data preparation, model training, and inference runs—into clearly defined steps.
Whether these steps are integrated into a single script or divided into multiple scripts, the documentation should outline the purpose and execution order of each script.
Include command-line examples to facilitate user implementation and reproduction of the workflow, minimizing the need to refer back to the manuscript or other sources.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p13">
<p class="ltx_p" id="S4.p13.1"><span class="ltx_text ltx_font_bold" id="S4.p13.1.1">Recommendation 4</span>: <span class="ltx_text ltx_font_bold" id="S4.p13.1.2">Testing in a Fresh Environment</span>.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.p14">
<p class="ltx_p" id="S4.p14.1">Before publishing your repository, test the complete workflow by cloning it onto a separate system, setting it up, and running the code. This process helps identify smaller issues, such as hard-coded paths or misnamed files, that may not be apparent otherwise. By ensuring the repository functions correctly in a fresh installation, you address key issues and improve the overall robustness of your code.</p>
</div>
</section>
<section class="ltx_section" id="S5">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Conclusion</h2>
<div class="ltx_para ltx_noindent" id="S5.p1">
<p class="ltx_p" id="S5.p1.1">The lack of proper documentation, modular source code, and full preprocessing information in open-source DRP models hinders their understanding and reuse. This limits their adoption by the scientific community and, consequently, their potential impact on precision oncology.
In this study, we conducted a comprehensive evaluation of 17 DRP models and demonstrate the challenges in their reusability and reproducibility.
Our findings underscore the need for standardized code sharing best practices, not only in DRP but across all AI-enabled scientific models.
The proposed recommendations aim to foster a more transparent and collaborative research environment, facilitating knowledge sharing and driving progress across various scientific domains.
Ultimately, our recommendations are a starting point for community-developed guidelines; greater engagement from model developers on this topic will lead to a fuller understanding of what makes a model reusable.
</p>
</div>
</section>
<section class="ltx_section" id="Sx1">
<h2 class="ltx_title ltx_title_section">Funding</h2>
<div class="ltx_para ltx_noindent" id="Sx1.p1">
<p class="ltx_p" id="Sx1.p1.1">The research was part of the IMPROVE project under the NCI-DOE Collaboration Program that has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, Task Order No. 75N91019F00134 and from the Department of Energy under Award Number ACO22002-001-00000.</p>
</div>
</section>
<section class="ltx_section" id="Sx2">
<h2 class="ltx_title ltx_title_section">Code availability</h2>
<div class="ltx_para ltx_noindent" id="Sx2.p1">
<p class="ltx_p" id="Sx2.p1.1">This study focuses on the qualitative aspects of reusing and implementing currently available code bases for cancer drug response prediction. The results in this paper do not pertain to the scientific merits, claims, or applicability of the cited research studies. The implementations and modifications made as a result of this effort are available in this GitHub repository: <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/JDACS4C-IMPROVE" title="">https://github.com/JDACS4C-IMPROVE</a>.</p>
</div>
</section>
<section class="ltx_bibliography" id="bib">
<h2 class="ltx_title ltx_title_bibliography">References</h2>
<ul class="ltx_biblist">
<li class="ltx_bibitem" id="bib.bib1">
<span class="ltx_tag ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">
Daniela Senft, Mark D. M. Leiserson, Eytan Ruppin, and Ze’ev A. Ronai.

</span>
<span class="ltx_bibblock">Precision Oncology: The Road Ahead.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib1.1.1">Trends in Molecular Medicine</span>, 23(10):874–898, October 2017.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib2">
<span class="ltx_tag ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">
Pedro J. Ballester and Javier Carmona.

</span>
<span class="ltx_bibblock">Artificial intelligence for the next generation of precision oncology.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib2.1.1">npj Precision Oncology</span>, 5(1):1–3, August 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib3">
<span class="ltx_tag ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">
Pedro J Ballester, Rick Stevens, Benjamin Haibe-Kains, R Stephanie Huang, and Tero Aittokallio.

</span>
<span class="ltx_bibblock">Artificial intelligence for drug response prediction in disease models.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib3.1.1">Briefings in Bioinformatics</span>, (bbab450), October 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib4">
<span class="ltx_tag ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">
Chayanit Piyawajanusorn, Linh C Nguyen, Ghita Ghislat, and Pedro J Ballester.

</span>
<span class="ltx_bibblock">A gentle introduction to understanding preclinical data for cancer pharmaco-omic modeling.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib4.1.1">Briefings in Bioinformatics</span>, 22(6), November 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib5">
<span class="ltx_tag ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">
Yu-Chiao Chiu, Hung-I Harry Chen, Aparna Gorthi, Milad Mostavi, Siyuan Zheng, Yufei Huang, and Yidong Chen.

</span>
<span class="ltx_bibblock">Deep learning of pharmacogenomics resources: moving towards precision oncology.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib5.1.1">Briefings in Bioinformatics</span>, 21(6):2066–2083, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib6">
<span class="ltx_tag ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">
Alexander Partin, Thomas S. Brettin, Yitan Zhu, Oleksandr Narykov, Austin Clyde, Jamie Overbeek, and Rick L. Stevens.

</span>
<span class="ltx_bibblock">Deep learning methods for drug response prediction in cancer: Predominant and emerging trends.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib6.1.1">Frontiers in Medicine</span>, 10, 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib7">
<span class="ltx_tag ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">
Farzaneh Firoozbakht, Behnam Yousefi, and Benno Schwikowski.

</span>
<span class="ltx_bibblock">An overview of machine learning methods for monotherapy drug response prediction.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib7.1.1">Briefings in Bioinformatics</span>, 23(1), January 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib8">
<span class="ltx_tag ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">
George Adam, Ladislav Rampášek, Zhaleh Safikhani, Petr Smirnov, Benjamin Haibe-Kains, and Anna Goldenberg.

</span>
<span class="ltx_bibblock">Machine learning approaches to drug response prediction: challenges and recent progress.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib8.1.1">npj Precision Oncology</span>, 4(1), 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib9">
<span class="ltx_tag ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">
James C Costello, Laura M Heiser, Elisabeth Georgii, Mehmet Gönen, Michael P Menden, Nicholas J Wang, Mukesh Bansal, Muhammad Ammad-Ud-Din, Petteri Hintsanen, Suleiman A Khan, John-Patrick Mpindi, Olli Kallioniemi, Antti Honkela, Tero Aittokallio, Krister Wennerberg, Nci Dream Community, James J Collins, Dan Gallahan, Dinah Singer, Julio Saez-Rodriguez, Samuel Kaski, Joe W Gray, and Gustavo Stolovitzky.

</span>
<span class="ltx_bibblock">A community effort to assess and improve drug sensitivity prediction algorithms.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib9.1.1">Nature Biotechnology</span>, 32(12):1202–1212, 2014.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib10">
<span class="ltx_tag ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">
Michael P. Menden, Dennis Wang, Mike J. Mason, Bence Szalai, Krishna C. Bulusu, Yuanfang Guan, Thomas Yu, Jaewoo Kang, Minji Jeon, Russ Wolfinger, Tin Nguyen, Mikhail Zaslavskiy, In Sock Jang, Zara Ghazoui, Mehmet Eren Ahsen, Robert Vogel, Elias Chaibub Neto, Thea Norman, Eric K. Y. Tang, Mathew J. Garnett, Giovanni Y. Di Veroli, Stephen Fawell, Gustavo Stolovitzky, Justin Guinney, Jonathan R. Dry, and Julio Saez-Rodriguez.

</span>
<span class="ltx_bibblock">Community assessment to advance computational prediction of cancer drug combinations in a pharmacogenomic screen.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib10.1.1">Nature Communications</span>, 10(1):2674, June 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib11">
<span class="ltx_tag ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock">
Jinyu Chen and Louxin Zhang.

</span>
<span class="ltx_bibblock">A survey and systematic assessment of computational methods for drug response prediction.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib11.1.1">Briefings in Bioinformatics</span>, 22(1):232–246, January 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib12">
<span class="ltx_tag ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock">
Emily So, Fengqing Yu, Bo Wang, and Benjamin Haibe-Kains.

</span>
<span class="ltx_bibblock">Reusability report: Evaluating reproducibility and reusability of a fine-tuned model to predict drug response in cancer patient samples.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib12.1.1">Nature Machine Intelligence</span>, 5(7):792–798, July 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib13">
<span class="ltx_tag ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock">
Jianzhu Ma, Samson H. Fong, Yunan Luo, Christopher J. Bakkenist, John Paul Shen, Soufiane Mourragui, Lodewyk F. A. Wessels, Marc Hafner, Roded Sharan, Jian Peng, and Trey Ideker.

</span>
<span class="ltx_bibblock">Few-shot learning creates predictive models of drug response that translate from high-throughput screens to individual patients.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib13.1.1">Nature Cancer</span>, 2(2):233–244, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib14">
<span class="ltx_tag ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock">
Zhaleh Safikhani, Petr Smirnov, Kelsie L Thu, Jennifer Silvester, Nehme El-Hachem, Rene Quevedo, Mathieu Lupien, Tak W Mak, David Cescon, and Benjamin Haibe-Kains.

</span>
<span class="ltx_bibblock">Gene isoforms as expression-based biomarkers predictive of drug response in vitro.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib14.1.1">Nature communications</span>, 8(1):1126, 2017.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib15">
<span class="ltx_tag ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock">
KL Thu, J Silvester, MJ Elliott, W Ba-Alawi, MH Duncan, AC Elia, AS Mer, P Smirnov, Z Safikhani, B Haibe-Kains, et al.

</span>
<span class="ltx_bibblock">Disruption of the anaphase-promoting complex confers resistance to ttk inhibitors in triple-negative breast cancer.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib15.1.1">Proceedings of the National Academy of Sciences</span>, 115(7):E1570–E1577, 2018.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib16">
<span class="ltx_tag ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock">
Hui Gao, Joshua M Korn, Stéphane Ferretti, John E Monahan, Youzhen Wang, Mallika Singh, Chao Zhang, Christian Schnell, Guizhi Yang, Yun Zhang, et al.

</span>
<span class="ltx_bibblock">High-throughput screening using patient-derived tumor xenografts to predict clinical trial drug response.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib16.1.1">Nature medicine</span>, 21(11):1318–1325, 2015.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib17">
<span class="ltx_tag ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock">
Christos Hatzis, Lajos Pusztai, Vicente Valero, Daniel J Booser, Laura Esserman, Ana Lluch, Tatiana Vidaurre, Frankie Holmes, Eduardo Souchon, Hongkun Wang, et al.

</span>
<span class="ltx_bibblock">A genomic predictor of response and survival following taxane-anthracycline chemotherapy for invasive breast cancer.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib17.1.1">Jama</span>, 305(18):1873–1881, 2011.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib18">
<span class="ltx_tag ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock">
Mitsuya Itoh, Takayuki Iwamoto, Junji Matsuoka, Tomohiro Nogami, Takayuki Motoki, Tadahiko Shien, Naruto Taira, Naoki Niikura, Naoki Hayashi, Shoichiro Ohtani, et al.

</span>
<span class="ltx_bibblock">Estrogen receptor (er) mrna expression and molecular subtype distribution in er-negative/progesterone receptor-positive breast cancers.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib18.1.1">Breast cancer research and treatment</span>, 143:403–409, 2014.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib19">
<span class="ltx_tag ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock">
Oana Baldasici, Loredana Balacescu, Daniel Cruceriu, Andrei Roman, Carmen Lisencu, Bogdan Fetica, Simona Visan, Andrei Cismaru, Ancuta Jurj, Lucian Barbu-Tudoran, et al.

</span>
<span class="ltx_bibblock">Circulating small evs mirnas as predictors of pathological response to neo-adjuvant therapy in breast cancer patients.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib19.1.1">International Journal of Molecular Sciences</span>, 23(20):12625, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib20">
<span class="ltx_tag ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock">
Christine E Horak, Lajos Pusztai, Guan Xing, Ovidiu C Trifan, Cristina Saura, Ling-Ming Tseng, Stephen Chan, Rosanne Welcher, and David Liu.

</span>
<span class="ltx_bibblock">Biomarker analysis of neoadjuvant doxorubicin/cyclophosphamide followed by ixabepilone or paclitaxel in early-stage breast cancer.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib20.1.1">Clinical Cancer Research</span>, 19(6):1587–1595, 2013.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib21">
<span class="ltx_tag ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock">
David Earl Hostallero, Yihui Li, and Amin Emad.

</span>
<span class="ltx_bibblock">Looking at the BiG picture: incorporating bipartite graphs in drug response prediction.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib21.1.1">Bioinformatics</span>, 38(14):3609–3620, 06 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib22">
<span class="ltx_tag ltx_tag_bibitem">[22]</span>
<span class="ltx_bibblock">
Qiao Liu, Zhiqiang Hu, Rui Jiang, and Mu Zhou.

</span>
<span class="ltx_bibblock">Deepcdr: a hybrid graph convolutional network for predicting cancer drug response.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib22.1.1">Bioinformatics</span>, 36(Supplement_2):i911–i918, 12 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib23">
<span class="ltx_tag ltx_tag_bibitem">[23]</span>
<span class="ltx_bibblock">
Likun Jiang, Changzhi Jiang, Xinyu Yu, Rao Fu, Shuting Jin, and Xiangrong Liu.

</span>
<span class="ltx_bibblock">DeepTTA: a transformer-based model for predicting cancer drug response.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib23.1.1">Briefings in Bioinformatics</span>, 23(3):bbac100, 03 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib24">
<span class="ltx_tag ltx_tag_bibitem">[24]</span>
<span class="ltx_bibblock">
Jihye Shin, Yinhua Piao, Dongmin Bang, Sun Kim, and Kyuri Jo.

</span>
<span class="ltx_bibblock">Drpreter: interpretable anticancer drug response prediction using knowledge-guided graph neural networks and transformer.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib24.1.1">International Journal of Molecular Sciences</span>, 23(22):13919, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib25">
<span class="ltx_tag ltx_tag_bibitem">[25]</span>
<span class="ltx_bibblock">
Brent M Kuenzi, Jisoo Park, Samson H Fong, Kyle S Sanchez, John Lee, Jason F Kreisberg, Jianzhu Ma, and Trey Ideker.

</span>
<span class="ltx_bibblock">Predicting drug response and synergy using a deep learning model of human cancer cells.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib25.1.1">Cancer cell</span>, 38(5):672–684, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib26">
<span class="ltx_tag ltx_tag_bibitem">[26]</span>
<span class="ltx_bibblock">
Seonghun Kim, Seockhun Bae, Yinhua Piao, and Kyuri Jo.

</span>
<span class="ltx_bibblock">Graph convolutional network for drug response prediction using gene expression data.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib26.1.1">Mathematics</span>, 9(7):772, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib27">
<span class="ltx_tag ltx_tag_bibitem">[27]</span>
<span class="ltx_bibblock">
Tianxing Ma, Qiao Liu, Haochen Li, Mu Zhou, Rui Jiang, and Xuegong Zhang.

</span>
<span class="ltx_bibblock">Dualgcn: a dual graph convolutional network model to predict cancer drug response.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib27.1.1">BMC bioinformatics</span>, 23(Suppl 4):129, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib28">
<span class="ltx_tag ltx_tag_bibitem">[28]</span>
<span class="ltx_bibblock">
Xuan Liu, Congzhi Song, Feng Huang, Haitao Fu, Wenjie Xiao, and Wen Zhang.

</span>
<span class="ltx_bibblock">GraphCDR: a graph neural network method with contrastive learning for cancer drug response prediction.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib28.1.1">Briefings in Bioinformatics</span>, 23(1):bbab457, 11 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib29">
<span class="ltx_tag ltx_tag_bibitem">[29]</span>
<span class="ltx_bibblock">
Tuan Nguyen, Giang TT Nguyen, Thin Nguyen, and Duc-Hau Le.

</span>
<span class="ltx_bibblock">Graph convolutional networks for drug response prediction.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib29.1.1">IEEE/ACM transactions on computational biology and bioinformatics</span>, 19(1):146–154, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib30">
<span class="ltx_tag ltx_tag_bibitem">[30]</span>
<span class="ltx_bibblock">
Iljung Jin and Hojung Nam.

</span>
<span class="ltx_bibblock">Hidra: Hierarchical network for drug response prediction with attention.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib30.1.1">Journal of Chemical Information and Modeling</span>, 61(8):3858–3867, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib31">
<span class="ltx_tag ltx_tag_bibitem">[31]</span>
<span class="ltx_bibblock">
Yitan Zhu, Thomas Brettin, Fangfang Xia, Alexander Partin, Maulik Shukla, Hyunseung Yoo, Yvonne A Evrard, James H Doroshow, and Rick L Stevens.

</span>
<span class="ltx_bibblock">Converting tabular data into images for deep learning with convolutional neural networks.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib31.1.1">Scientific reports</span>, 11(1):1–11, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib32">
<span class="ltx_tag ltx_tag_bibitem">[32]</span>
<span class="ltx_bibblock">
Hossein Sharifi-Noghabi, Olga Zolotareva, Colin C Collins, and Martin Ester.

</span>
<span class="ltx_bibblock">MOLI: multi-omics late integration with deep neural networks for drug response prediction.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib32.1.1">Bioinformatics</span>, 35(14):i501–i509, 07 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib33">
<span class="ltx_tag ltx_tag_bibitem">[33]</span>
<span class="ltx_bibblock">
Yi-Ching Tang and Assaf Gottlieb.

</span>
<span class="ltx_bibblock">Explainable drug sensitivity prediction through cancer pathway enrichment.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib33.1.1">Scientific reports</span>, 11(1):3128, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib34">
<span class="ltx_tag ltx_tag_bibitem">[34]</span>
<span class="ltx_bibblock">
Zhaorui Zuo, Penglei Wang, Xiaowei Chen, Li Tian, Hui Ge, and Dahong Qian.

</span>
<span class="ltx_bibblock">Swnet: a deep learning model for drug response prediction from cancer genomic signatures and compound chemical structures.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib34.1.1">BMC bioinformatics</span>, 22:1–16, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib35">
<span class="ltx_tag ltx_tag_bibitem">[35]</span>
<span class="ltx_bibblock">
Pengfei Liu, Hongjian Li, Shuai Li, and Kwong-Sak Leung.

</span>
<span class="ltx_bibblock">Improving prediction of phenotypic drug response on cancer cell lines using deep convolutional network.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib35.1.1">BMC bioinformatics</span>, 20:1–14, 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib36">
<span class="ltx_tag ltx_tag_bibitem">[36]</span>
<span class="ltx_bibblock">
Yiheng Zhu, Zhenqiu Ouyang, Wenbo Chen, Ruiwei Feng, Danny Z Chen, Ji Cao, and Jian Wu.

</span>
<span class="ltx_bibblock">Tgsa: protein–protein association-based twin graph neural networks for drug response prediction with similarity augmentation.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib36.1.1">Bioinformatics</span>, 38(2):461–468, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib37">
<span class="ltx_tag ltx_tag_bibitem">[37]</span>
<span class="ltx_bibblock">
Wanjuan Yang, Jorge Soares, Patricia Greninger, Elena J Edelman, Howard Lightfoot, Simon Forbes, Nidhi Bindal, Dave Beare, James A Smith, I Richard Thompson, et al.

</span>
<span class="ltx_bibblock">Genomics of drug sensitivity in cancer (gdsc): a resource for therapeutic biomarker discovery in cancer cells.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib37.1.1">Nucleic acids research</span>, 41(D1):D955–D961, 2012.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib38">
<span class="ltx_tag ltx_tag_bibitem">[38]</span>
<span class="ltx_bibblock">
Brinton Seashore-Ludlow, Matthew G Rees, Jaime H Cheah, Murat Cokol, Edmund V Price, Matthew E Coletti, Victor Jones, Nicole E Bodycombe, Christian K Soule, Joshua Gould, et al.

</span>
<span class="ltx_bibblock">Harnessing connectivity in a large-scale small-molecule sensitivity dataset.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib38.1.1">Cancer discovery</span>, 5(11):1210–1223, 2015.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib39">
<span class="ltx_tag ltx_tag_bibitem">[39]</span>
<span class="ltx_bibblock">
Fangfang Xia, Jonathan Allen, Prasanna Balaprakash, Thomas Brettin, Cristina Garcia-Cardona, Austin Clyde, Judith Cohn, James Doroshow, Xiaotian Duan, Veronika Dubinkina, Yvonne Evrard, Ya Ju Fan, Jason Gans, Stewart He, Pinyi Lu, Sergei Maslov, Alexander Partin, Maulik Shukla, Eric Stahlberg, Justin M Wozniak, Hyunseung Yoo, George Zaki, Yitan Zhu, and Rick Stevens.

</span>
<span class="ltx_bibblock">A cross-study analysis of drug response prediction in cancer cell lines.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib39.1.1">Briefings in Bioinformatics</span>, 23(1):bbab356, January 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib40">
<span class="ltx_tag ltx_tag_bibitem">[40]</span>
<span class="ltx_bibblock">
Alexander Partin, Thomas Brettin, Yvonne A. Evrard, Yitan Zhu, Hyunseung Yoo, Fangfang Xia, Songhao Jiang, Austin Clyde, Maulik Shukla, Michael Fonstein, James H. Doroshow, and Rick L. Stevens.

</span>
<span class="ltx_bibblock">Learning curves for drug response prediction in cancer cell lines.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib40.1.1">BMC Bioinformatics</span>, 22(1), May 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib41">
<span class="ltx_tag ltx_tag_bibitem">[41]</span>
<span class="ltx_bibblock">
Nikhil Branson, Pedro R Cutillas, and Conrad Bessant.

</span>
<span class="ltx_bibblock">Comparison of multiple modalities for drug response prediction with learning curves using neural networks and XGBoost.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib41.1.1">Bioinformatics Advances</span>, 4(1):vbad190, January 2024.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib42">
<span class="ltx_tag ltx_tag_bibitem">[42]</span>
<span class="ltx_bibblock">
Austin Clyde, Tom Brettin, Alexander Partin, Maulik Shaulik, Hyunseung Yoo, Yvonne Evrard, Yitan Zhu, Fangfang Xia, and Rick Stevens.

</span>
<span class="ltx_bibblock">A Systematic Approach to Featurization for Cancer Drug Sensitivity Predictions with Deep Learning.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib42.1.1">arXiv:2005.00095 [cs, q-bio]</span>, May 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib43">
<span class="ltx_tag ltx_tag_bibitem">[43]</span>
<span class="ltx_bibblock">
Delora Baptista, João Correia, Bruno Pereira, and Miguel Rocha.

</span>
<span class="ltx_bibblock">Evaluating molecular representations in machine learning models for drug response prediction and interpretability.

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib43.1.1">Journal of Integrative Bioinformatics</span>, August 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib44">
<span class="ltx_tag ltx_tag_bibitem">[44]</span>
<span class="ltx_bibblock">
Yihui Li, David Earl Hostallero, and Amin Emad.

</span>
<span class="ltx_bibblock">Interpretable deep learning architectures for improving drug response prediction performance: myth or reality?

</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_font_italic" id="bib.bib44.1.1">Bioinformatics</span>, 39(6):btad390, June 2023.

</span>
</li>
</ul>
</section>
</article>
</div>
<footer class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Wed Sep 18 15:44:46 2024 by <a class="ltx_LaTeXML_logo" href="http://dlmf.nist.gov/LaTeXML/"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img alt="Mascot Sammy" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg=="/></a>
</div></footer>
</div>
</body>
</html>
