<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[2409.18907] In-depth Analysis of Privacy Threats in Federated Learning for Medical Data</title><meta property="og:description" content="Federated learning is emerging as a promising machine learning technique in the medical field for analyzing medical images, as it is considered an effective method to safeguard sensitive patient data and comply with prâ€¦">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="In-depth Analysis of Privacy Threats in Federated Learning for Medical Data">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="In-depth Analysis of Privacy Threats in Federated Learning for Medical Data">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/2409.18907">

<!--Generated on Sat Oct  5 19:24:20 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="keywords" lang="en" content="
Federated Learning,  Gradient Leakage Attack,  Medical Image Analysis,  Privacy Risk.
">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<h1 class="ltx_title ltx_title_document">In-depth Analysis of Privacy Threats in Federated Learning for Medical Data</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Badhan Chandra Das, M. Hadi Amini,  and Yanzhao Wu
</span><span class="ltx_author_notes">Badhan Chandra Das, M. Hadi Amini, and Yanzhao Wu are with the Knight Foundation School of Computing and Information Sciences (KFSCIS), Florida International University (FIU), Miami, FL-33199, USA. Badhan Chandra Das and M. Hadi Amini are also with the Sustainability, Optimization, and Learning for InterDependent networks laboratory (solid lab) at FIU. (E-mails: bdas004@fiu.edu, moamini@fiu.edu, yawu@fiu.edu). 
<br class="ltx_break">Corresponding authors: M. Hadi Amini and Yanzhao Wu.
<br class="ltx_break">This paper is an extended version of our conference paperÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib1" title="" class="ltx_ref">1</a>]</cite>.</span></span>
</div>

<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p id="id4.id1" class="ltx_p">Federated learning is emerging as a promising machine learning technique in the medical field for analyzing medical images, as it is considered an effective method to safeguard sensitive patient data and comply with privacy regulations. However, recent studies have revealed that the default settings of federated learning may inadvertently expose private training data to privacy attacks. Thus, the intensity of such privacy risks and potential mitigation strategies in the medical domain remain unclear. In this paper, we make three original contributions to privacy risk analysis and mitigation in federated learning for medical data. First, we propose a holistic framework, MedPFL, for analyzing privacy risks in processing medical data in the federated learning environment and developing effective mitigation strategies for protecting privacy. Second, through our empirical analysis, we demonstrate the severe privacy risks in federated learning to process medical images, where adversaries can accurately reconstruct private medical images by performing privacy attacks. Third, we illustrate that the prevalent defense mechanism of adding random noises may not always be effective in protecting medical images against privacy attacks in federated learning, which poses unique and pressing challenges related to protecting the privacy of medical data. Furthermore, the paper discusses several unique research questions related to the privacy protection of medical data in the federated learning environment. We conduct extensive experiments on several benchmark medical image datasets to analyze and mitigate the privacy risks associated with federated learning for medical data.</p>
</div>
<div class="ltx_keywords">
<h6 class="ltx_title ltx_title_keywords">Index Terms: </h6>
Federated Learning, Gradient Leakage Attack, Medical Image Analysis, Privacy Risk.

</div>
<span id="id1a" class="ltx_note ltx_note_frontmatter ltx_role_publicationid"><sup class="ltx_note_mark">â€ </sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">â€ </sup><span class="ltx_note_type">publicationid: </span>pubid:  </span></span></span>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">I </span><span id="S1.1.1" class="ltx_text ltx_font_smallcaps">Introduction</span>
</h2>

<div id="S1.p1" class="ltx_para">
<p id="S1.p1.1" class="ltx_p">Federated Learning (FL) is an emergent Machine Learning (ML) technique where training data is distributed across multiple clients instead of a central server to protect privacy. In this approach, the training occurs locally on each client (also known as participants) and the model parameters are aggregated on a central serverÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib2" title="" class="ltx_ref">2</a>]</cite>, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite>. One of the most significant advantages of FL is that it can mitigate the systemic privacy risks of traditional centralized ML by keeping the private data decentralized on the clientsâ€™ end and only sharing the extracted gradient updates to the central server. There are several additional benefits of FL except decentralizing the private training data including, scalability and efficiencyÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>, <a href="#bib.bib4" title="" class="ltx_ref">4</a>]</cite>. FL ensures scalability by allowing seamless integration of a large number of edge devices on clients into the learning process. It also demonstrates enhanced efficiency by collaborating among participating devices, which enables them to collectively contribute to updating the shared global model through their private training data. There are several FL algorithms prevalent such as FedAvgÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib2" title="" class="ltx_ref">2</a>]</cite>, FedProxÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib5" title="" class="ltx_ref">5</a>]</cite>, FedGANÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib6" title="" class="ltx_ref">6</a>]</cite>, and ProxyFLÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib7" title="" class="ltx_ref">7</a>]</cite>.</p>
</div>
<div id="S1.p2" class="ltx_para">
<p id="S1.p2.1" class="ltx_p">In the healthcare sector, the integration of ML and Convolutional Neural Networks (CNN) algorithms are common for the analysis of diverse medical data such as medical images, health records, and text-based doctorâ€™s adviceÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib8" title="" class="ltx_ref">8</a>]</cite>, <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib9" title="" class="ltx_ref">9</a>]</cite>. These algorithms are also being utilized for prediction purposesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib10" title="" class="ltx_ref">10</a>]</cite>. Those models help to make better decision and recommendation systems in the healthcare contextÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib11" title="" class="ltx_ref">11</a>]</cite>. Additionally, FL emerged as a promising learning technique in the medical domain due to its decentralized nature of private training dataÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib3" title="" class="ltx_ref">3</a>]</cite>. It facilitates keeping the patientsâ€™ sensitive health records private at their corresponding ends. FL ensures privacy-preserving ML by collaborating with multiple distributed clients, such as hospitals or clinics, without sharing sensitive raw dataÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib12" title="" class="ltx_ref">12</a>]</cite>.</p>
</div>
<div id="S1.p3" class="ltx_para">
<p id="S1.p3.1" class="ltx_p">Medical data, for example, X-ray images, diabetic test reports, and Magnetic Resonance Imaging (MRI) scans are considered highly sensitive records. Because those contain confidential details such as individualsâ€™ names, dates of birth, and comprehensive medical histories, which collectively serve as unique identifiers of an individual. The exposure of such sensitive information can yield severe consequences for patients, ranging from social stigma and discrimination to potential job loss and insurance coverage denial. To mitigate these risks, numerous data protection regulations have been imposed globally, underscoring the critical importance of safeguarding individualsâ€™ health-related information. The most common regulations include the Health Insurance Portability and Accountability Act (HIPAA)Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib13" title="" class="ltx_ref">13</a>]</cite>, the California Consumer Privacy Act (CCPA)Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib14" title="" class="ltx_ref">14</a>]</cite>, and the European Union General Data Protection Regulation (GDPR)Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib15" title="" class="ltx_ref">15</a>]</cite> are the most common. These regulatory boards aim to ensure the privacy and security of medical data, shielding patients from the risk of unauthorized exposure. Thus, it ensures the robust security and confidentiality of healthcare data.</p>
</div>
<div id="S1.p4" class="ltx_para">
<p id="S1.p4.1" class="ltx_p">Although the primary purpose of FL is to prevent leaving private training data from local devices to mitigate privacy risks, recent studies outlined that the default privacy schema in FL is inadequate to prevent privacy leakage attacks. Studies revealed that FL systems are susceptible to privacy leakage attacks where the adversaries intercept the local gradient updates transmitted by the clients before model aggregation. It can reconstruct the clientsâ€™ private training data with high reconstruction accuracy, thus covertly and illegally exposing clientsâ€™ privacyÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>, <a href="#bib.bib17" title="" class="ltx_ref">17</a>]</cite>. This vulnerability poses a severe threat to the security of FL systems, compromising the protection of client privacy <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib18" title="" class="ltx_ref">18</a>]</cite>. The occurrence of such privacy attacks underscores the pressing need for robust privacy-preserving mechanisms within FL frameworks to ensure the confidentiality of sensitive data.</p>
</div>
<div id="S1.p5" class="ltx_para">
<p id="S1.p5.1" class="ltx_p">Protecting sensitive medical data from unauthorized access is crucial to uphold confidentiality, privacy, and the trust of patients so that legal standards can be met, and patientsâ€™ privacy can be safeguarded. Despite the benefits, FL faces significant privacy vulnerabilities that pose serious threats to its application in the medical domain. Therefore, it is imperative to investigate these privacy risks and devise effective mitigation strategies to defend against privacy attacks targeting FL applications within the healthcare domain. The key contributions of this paper are as follows.</p>
</div>
<div id="S1.p6" class="ltx_para">
<ol id="S1.I1" class="ltx_enumerate">
<li id="S1.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S1.I1.i1.p1" class="ltx_para">
<p id="S1.I1.i1.p1.1" class="ltx_p">We introduce MedPFL, a systematic framework designed for the <span id="S1.I1.i1.p1.1.1" class="ltx_text ltx_font_bold">Med</span>ical Data <span id="S1.I1.i1.p1.1.2" class="ltx_text ltx_font_bold">P</span>rivacy risk analysis and mitigation in <span id="S1.I1.i1.p1.1.3" class="ltx_text ltx_font_bold">F</span>ederated <span id="S1.I1.i1.p1.1.4" class="ltx_text ltx_font_bold">L</span>earning. The framework consists of real-world medical datasets, deep learning models, and a variety of attack and defense mechanisms. It also includes evaluation metrics for a thorough assessment of the effectiveness of different attack methods and defense strategies across various configurations.</p>
</div>
</li>
<li id="S1.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S1.I1.i2.p1" class="ltx_para">
<p id="S1.I1.i2.p1.1" class="ltx_p">Our research highlights the significant privacy risks associated with the use of federated learning for the analysis of medical images. Through empirical evaluations, we demonstrate how adversaries can execute privacy attacks to accurately reconstruct private medical data. This finding underscores the vulnerability of FL systems to privacy breaches and the need for robust defenses.</p>
</div>
</li>
<li id="S1.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span> 
<div id="S1.I1.i3.p1" class="ltx_para">
<p id="S1.I1.i3.p1.1" class="ltx_p">In response to the identified privacy risks, we explore various defense configurations within the FL settings. By integrating different levels of random noise, we aim to protect private medical data effectively.</p>
</div>
</li>
<li id="S1.I1.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">4.</span> 
<div id="S1.I1.i4.p1" class="ltx_para">
<p id="S1.I1.i4.p1.1" class="ltx_p">Our investigation reveals several challenges involved in defending against privacy attacks in FL, particularly in the context of medical data, where the urge for privacy protection is exceptionally high.</p>
</div>
</li>
<li id="S1.I1.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">5.</span> 
<div id="S1.I1.i5.p1" class="ltx_para">
<p id="S1.I1.i5.p1.1" class="ltx_p">We raise and discuss several research questions upon performing the experiments on different attack methods and defense mechanisms with various configurations on medical image datasets in Section <a href="#S7" title="VII Discussion â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">VII</span></a>.</p>
</div>
</li>
</ol>
</div>
<div id="S1.p7" class="ltx_para">
<p id="S1.p7.1" class="ltx_p">Systematic experiments are conducted on representative medical image datasets in order to analyze the challenges of protecting privacy for medical images in FL with visual examples of several real-world scenarios. We conjecture this investigation will capture the interest of researchers, developers, and stakeholders in the relevant domain of privacy-preserving methods in FL for medical purposes.</p>
</div>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">II </span><span id="S2.1.1" class="ltx_text ltx_font_smallcaps">Motivation</span>
</h2>

<div id="S2.p1" class="ltx_para">
<p id="S2.p1.1" class="ltx_p">The adoption of FL within the medical domain has been accelerated, primarily to safeguard private medical data while training ML models for tasks such as COVID-19 detection using chest X-ray imagesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib19" title="" class="ltx_ref">19</a>]</cite>, and skin disease detection using dermoscopy imagesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib20" title="" class="ltx_ref">20</a>]</cite>. However, recent studies have shown the inherent privacy-related challenges in FL <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib1" title="" class="ltx_ref">1</a>, <a href="#bib.bib21" title="" class="ltx_ref">21</a>, <a href="#bib.bib22" title="" class="ltx_ref">22</a>]</cite>. Despite numerous efforts to obscure personal health data, there remains a risk of patient information being re-identified, as evidenced by studies showcasing the potential re-identificationÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib23" title="" class="ltx_ref">23</a>]</cite> of individuals from DICOM imagesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib24" title="" class="ltx_ref">24</a>]</cite>.</p>
</div>
<div id="S2.p2" class="ltx_para">
<p id="S2.p2.1" class="ltx_p">Furthermore, adversaries could steal the data or access the algorithm from non-encrypted networksÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib25" title="" class="ltx_ref">25</a>]</cite>.
Medical images have been reported to be susceptible to adversarial attacks due to numerous reasons, such as ambiguous ground truth <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib26" title="" class="ltx_ref">26</a>]</cite> and highly standardized formatÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib27" title="" class="ltx_ref">27</a>]</cite>. Therefore, the default settings of FL are still insufficient to protect the privacy of medical images. The aforementioned studies have only shown how medical images (e.g., X-ray, and MRI scans) can be leaked from the various FL environments. However, to the best of our knowledge, there is not yet a comprehensive framework for analyzing and mitigating privacy risks in FL to protect private medical data.
Moreover, the stringent regulations on medical data protection, coupled with their distinct characteristics compared with generic data, make it imperative and much more challenging to investigate and develop effective privacy-preserving techniques for protecting medical data. For example, we highlight below several unique features of medical images.</p>
</div>
<div id="S2.p3" class="ltx_para">
<ol id="S2.I1" class="ltx_enumerate">
<li id="S2.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span> 
<div id="S2.I1.i1.p1" class="ltx_para">
<p id="S2.I1.i1.p1.1" class="ltx_p"><span id="S2.I1.i1.p1.1.1" class="ltx_text ltx_font_bold">Complexity and variability of medical images:</span> Medical images, such as MRI, CT scans, and ultrasound, are more complex and heterogeneous than general images (e.g., FigureÂ <a href="#S2.F1" title="Figure 1 â€£ II Motivation â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>). They often contain noise, artifacts, and distortionsÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib28" title="" class="ltx_ref">28</a>]</cite> that make data interpretation and analysis more challenging.</p>
</div>
</li>
<li id="S2.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span> 
<div id="S2.I1.i2.p1" class="ltx_para">
<p id="S2.I1.i2.p1.1" class="ltx_p"><span id="S2.I1.i2.p1.1.1" class="ltx_text ltx_font_bold">High dimensionality:</span> Medical images may have higher dimensionality. For instance, a CT scan can have hundreds of slices, each of which contains a large number of pixels. This high dimensionality requires more computational power and specialized algorithms to process, which may also impact the chance of privacy leakage.</p>
</div>
</li>
<li id="S2.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">3.</span> 
<div id="S2.I1.i3.p1" class="ltx_para">
<p id="S2.I1.i3.p1.1" class="ltx_p"><span id="S2.I1.i3.p1.1.1" class="ltx_text ltx_font_bold">Specificity of medical domain:</span> Medical images often contain specific features and structures that are not present in general images, and the interpretation of these features requires specialized knowledge and data analysis models in the medical domain.</p>
</div>
</li>
<li id="S2.I1.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">4.</span> 
<div id="S2.I1.i4.p1" class="ltx_para">
<p id="S2.I1.i4.p1.1" class="ltx_p"><span id="S2.I1.i4.p1.1.1" class="ltx_text ltx_font_bold">Statistical distribution derivation:</span> The statistical distribution of medical images often deviates from the generic images, which creates significant differences between these two data categories in terms of processing, privacy protection, and execution time. We will discuss it in SectionÂ <a href="#S7" title="VII Discussion â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">VII</span></a> along with experiments.</p>
</div>
</li>
</ol>
</div>
<figure id="S2.F1" class="ltx_figure"><img src="/html/2409.18907/assets/x1.png" id="S2.F1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="368" height="218" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>Samples of Generic Images (CIFAR-10) and Medical Images</figcaption>
</figure>
<div id="S2.p4" class="ltx_para">
<p id="S2.p4.1" class="ltx_p">Therefore, medical images require dedicated studies and need to be handled with more care than generic data types. In subsequent sections, we perform experiments to compare several privacy attacks to analyze privacy risks and different defense mechanisms with different configurations to prevent privacy leakage for three representative medical images in FL. Also, we demonstrate how different defense configurations impact the performance of the model. Furthermore, we distinguish different characteristics of generic images and medical images, along with several unique research challenges in terms of protecting medical data from privacy attacks.</p>
</div>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">III </span><span id="S3.1.1" class="ltx_text ltx_font_smallcaps">Related Works</span>
</h2>

<div id="S3.p1" class="ltx_para">
<p id="S3.p1.1" class="ltx_p">Though FL was proposed to protect clientsâ€™ private data, researchers have shown that the FL is prone to be attacked by adversaries from both security and privacy perspectives <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib29" title="" class="ltx_ref">29</a>]</cite> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib30" title="" class="ltx_ref">30</a>]</cite>.</p>
</div>
<section id="S3.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S3.SS1.5.1.1" class="ltx_text">III-A</span> </span><span id="S3.SS1.6.2" class="ltx_text ltx_font_italic">Security Attacks</span>
</h3>

<div id="S3.SS1.p1" class="ltx_para">
<p id="S3.SS1.p1.1" class="ltx_p">In FL, a malicious user or an adversary takes advantage of the vulnerabilities <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib31" title="" class="ltx_ref">31</a>]</cite> and gains control of one or more participants (i.e., clients) within the FL environment so that it can cause the malfunction of the whole system <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib29" title="" class="ltx_ref">29</a>]</cite>.</p>
</div>
<div id="S3.SS1.p2" class="ltx_para">
<p id="S3.SS1.p2.1" class="ltx_p">Poisoning attacks are the most common types of security attacksÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib32" title="" class="ltx_ref">32</a>]</cite>. The basic concept of poisoning attacks in FL refers to a scenario where a malicious user from the participants in the FL inserts poisonous data samples or parameters intending to malfunction the whole system. Poisoning can happen in both data levelÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib33" title="" class="ltx_ref">33</a>]</cite> and model levelÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib34" title="" class="ltx_ref">34</a>]</cite>. For data poisoning attacks in FL, malicious clients inject mislabeled, corrupted, or poisoned data into their local training data and attempt to update the global model with the poisoned dataÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib35" title="" class="ltx_ref">35</a>]</cite>. That eventually yields degraded performance of the system <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib33" title="" class="ltx_ref">33</a>]</cite>, which is the primary goal of the data poisoning attack. On the other hand, in poisoning attacks, rather than modifying the training data directly, the attacker/malicious client intentionally manipulates the gradient updates before sending them to the central serverÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib36" title="" class="ltx_ref">36</a>]</cite>. Backdoor attacks are also prevalent in the FL setting, where the attacker aims to inject a desired malicious task into the existing modelÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib29" title="" class="ltx_ref">29</a>]</cite>.</p>
</div>
</section>
<section id="S3.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S3.SS2.5.1.1" class="ltx_text">III-B</span> </span><span id="S3.SS2.6.2" class="ltx_text ltx_font_italic">Privacy Attacks</span>
</h3>

<div id="S3.SS2.p1" class="ltx_para">
<p id="S3.SS2.p1.1" class="ltx_p">The privacy attacks in FL primarily focused on inferring sensitive information about participantsâ€™ private training data based on the gradients they send to the central server <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib29" title="" class="ltx_ref">29</a>]</cite>. Training data leakage through reconstruction attacks is the most prevalent in this category.</p>
</div>
<div id="S3.SS2.p2" class="ltx_para">
<p id="S3.SS2.p2.1" class="ltx_p">Recent studiesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref">37</a>, <a href="#bib.bib16" title="" class="ltx_ref">16</a>, <a href="#bib.bib38" title="" class="ltx_ref">38</a>, <a href="#bib.bib39" title="" class="ltx_ref">39</a>, <a href="#bib.bib40" title="" class="ltx_ref">40</a>, <a href="#bib.bib41" title="" class="ltx_ref">41</a>, <a href="#bib.bib42" title="" class="ltx_ref">42</a>, <a href="#bib.bib43" title="" class="ltx_ref">43</a>, <a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite> have demonstrated that the clientsâ€™ private training data in FL environment can be reconstructed by the adversary with high reconstruction accuracy. Several privacy leakage attack techniques have been proposed including Client Privacy Leakage (CPL)Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>, Deep Leakage from Gradients (DLG)Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>, Improved DLG (iDLG) <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite>, and Inverting Gradients (GradInv)Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite>. These attack methods illustrate that the default privacy scheme in FL might not offer adequate protection against privacy leakage attacks in the default FL environment. These attack methods underscore the need for enhanced privacy-preserving mechanisms in FL to effectively safeguard participantsâ€™ sensitive data. Random noise insertion, e.g., Gaussian noiseÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib45" title="" class="ltx_ref">45</a>]</cite> or Laplacian noiseÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib46" title="" class="ltx_ref">46</a>]</cite> might be an effective method for defending against privacy attacks in deep learning models. However, there is a significant lack of comprehensive and systematic studies addressing the potential threats posed by these privacy attacks to medical data within FL settings.</p>
</div>
<figure id="S3.F2" class="ltx_figure"><img src="/html/2409.18907/assets/x2.png" id="S3.F2.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="375" height="137" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Overview of MedPFL: a framework for <span id="S3.F2.5.1" class="ltx_text ltx_font_bold">Med</span>ical Data <span id="S3.F2.6.2" class="ltx_text ltx_font_bold">P</span>rivacy risk analysis and mitigation in <span id="S3.F2.7.3" class="ltx_text ltx_font_bold">F</span>ederated <span id="S3.F2.8.4" class="ltx_text ltx_font_bold">L</span>earning</figcaption>
</figure>
<div id="S3.SS2.p3" class="ltx_para">
<p id="S3.SS2.p3.1" class="ltx_p">In the medical domain, Kaissis et al.Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib47" title="" class="ltx_ref">47</a>]</cite> surveyed privacy-preservation techniques, which are designed for classifying chest X-rays and segmenting CT scans in deep learning trainingÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib48" title="" class="ltx_ref">48</a>]</cite>. A personalized local differential privacy in the FL scheme was illustrated by Shen et al.Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib49" title="" class="ltx_ref">49</a>]</cite> on the MNIST dataset to overcome the challenges, (e.g., inadequate or excessive privacy protection due to the same privacy budget) of the existing local differential privacy-based FL scheme. The use of Differential Privacy (DP) to protect medical data from such privacy attacks has been discussed by Liu et al.Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib50" title="" class="ltx_ref">50</a>]</cite>. Adnan et al.Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib51" title="" class="ltx_ref">51</a>]</cite> proposed DP in FL on histopathology images, but they did not explicitly mention any vulnerabilities or unique challenges of medical images in a decentralized environment. Aouedi et al.Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib52" title="" class="ltx_ref">52</a>]</cite> highlighted the several challenges in FL, focusing on privacy and security concerns, issues related to client synchronization, and the complexity arising from the presence of non-IID datasets.</p>
</div>
<div id="S3.SS2.p4" class="ltx_para">
<p id="S3.SS2.p4.1" class="ltx_p">We propose a comprehensive framework to specifically analyze the privacy risks inherent in medical data and their mitigation strategies in the FL environments. Moreover, our research has identified a set of unique challenges and distinguishing features, notably the intricate nature, higher-dimensional aspects, and latent pathological information inherent in medical images. These factors significantly amplify privacy concerns surrounding medical data in FL scenariosÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib27" title="" class="ltx_ref">27</a>]</cite>. Therefore, the extent of vulnerability of FL applications in the medical sector to privacy attacks lacks in-depth studies, along with the optimal approaches for mitigating such risks. This poses critical challenges in employing FL for processing sensitive private medical data, such as skin cancer images, X-ray images, and MRI scans of patients. Addressing these gaps in understanding is imperative for ensuring the security and privacy of sensitive medical data within FL frameworks.</p>
</div>
</section>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">IV </span><span id="S4.1.1" class="ltx_text ltx_font_smallcaps">Framework Overview</span>
</h2>

<div id="S4.p1" class="ltx_para">
<p id="S4.p1.1" class="ltx_p">We propose MedPFL, a comprehensive framework that addresses the critical need for privacy risk analysis and mitigation in FL, particularly in the medical domain. It comprises five key components that aim to streamline the evaluation, comparison, and mitigation of privacy risks associated with processing medical data within FL environments. FigureÂ <a href="#S3.F2" title="Figure 2 â€£ III-B Privacy Attacks â€£ III Related Works â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> illustrates these components and major workflows for privacy attacks on the trained model with private medical datasets, and defense mechanisms for safeguarding the private medical images from being attacked. We also incorporate several evaluation metrics to measure the efficacy of both attack and defense mechanisms.</p>
</div>
<section id="S4.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S4.SS1.5.1.1" class="ltx_text">IV-A</span> </span><span id="S4.SS1.6.2" class="ltx_text ltx_font_italic">Datasets</span>
</h3>

<div id="S4.SS1.p1" class="ltx_para">
<p id="S4.SS1.p1.1" class="ltx_p">The proposed framework offers a collection of real-world medical datasets from publicly available sources. These datasets represent diverse forms of medical data, including Melanoma Skin Cancer imagesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib53" title="" class="ltx_ref">53</a>]</cite>, COVID-19 X-ray imagesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref">54</a>]</cite>, and Brain Tumor MRI scansÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib55" title="" class="ltx_ref">55</a>]</cite>. These datasets serve as valuable resources to assess the potential privacy risks associated with medical data. Additionally, our framework provides easy-to-use APIs to facilitate the integration of new datasets.
The details of the datasets that we used in this study will be introduced later in SectionÂ <a href="#S6" title="VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">VI</span></a> (experimental analysis).</p>
</div>
</section>
<section id="S4.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S4.SS2.5.1.1" class="ltx_text">IV-B</span> </span><span id="S4.SS2.6.2" class="ltx_text ltx_font_italic">Models</span>
</h3>

<div id="S4.SS2.p1" class="ltx_para">
<p id="S4.SS2.p1.1" class="ltx_p">Our framework supports a variety of deep-learning models for medical data processing. For instance, we incorporate LeNetÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib56" title="" class="ltx_ref">56</a>]</cite> and ResNetÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> for medical image classification tasks, and U-NetÂ  <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib58" title="" class="ltx_ref">58</a>]</cite> for biomedical image segmentation. These models represent the mainstream Deep Neural Network (DNN) architectures for medical data analysis. It allows us to reflect the research challenges by scrutinizing their privacy vulnerabilities and investigating potential mitigation techniques. For now, the framework supports classification tasks, in the future, we are planning to incorporate other learning tasks, e.g., medical image segmentation.</p>
</div>
</section>
<section id="S4.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S4.SS3.5.1.1" class="ltx_text">IV-C</span> </span><span id="S4.SS3.6.2" class="ltx_text ltx_font_italic">Attack Methods</span>
</h3>

<div id="S4.SS3.p1" class="ltx_para">
<p id="S4.SS3.p1.1" class="ltx_p">To assess the privacy risks associated with medical data, we implemented a set of attack methods within our framework. These include CPLÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>, DLGÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>, iDLGÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite>, and GradInvÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite>. Leveraging these attack techniques, we conducted privacy attacks against diverse medical datasets and models to evaluate their potential privacy risks. Evaluation of these privacy attack methods is conducted using well-established metrics such as Attack Success Rate (ASR), Mean Squared Error (MSE), and Structural Similarity Index Measure (SSIM).</p>
</div>
</section>
<section id="S4.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S4.SS4.5.1.1" class="ltx_text">IV-D</span> </span><span id="S4.SS4.6.2" class="ltx_text ltx_font_italic">Defense Mechanisms</span>
</h3>

<div id="S4.SS4.p1" class="ltx_para">
<p id="S4.SS4.p1.1" class="ltx_p">In a conventional FL environment, several defense mechanisms have been proposed to prevent various types of privacy attacks. These defenses include gradient perturbationÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>, gradient compressionÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>, secure multi-party computationÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref">59</a>]</cite>, and DP techniquesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib45" title="" class="ltx_ref">45</a>]</cite>. However, there is a lack of comprehensive studies on the efficacy and optimal configurations of these defense mechanisms against a wide range of privacy attacks within the realm of medical data. In response to this gap, our MedPFL framework offers a range of defense mechanisms, facilitating in-depth investigations into their effectiveness and the factors influencing their performance against privacy attacks on medical data. Additionally, we provide a set of evaluation metrics designed to assess the efficacy of these defense mechanisms in safeguarding the privacy of medical data.</p>
</div>
</section>
<section id="S4.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S4.SS5.5.1.1" class="ltx_text">IV-E</span> </span><span id="S4.SS5.6.2" class="ltx_text ltx_font_italic">Evaluation Metrics</span>
</h3>

<div id="S4.SS5.p1" class="ltx_para">
<p id="S4.SS5.p1.1" class="ltx_p">Here, we briefly introduce three major evaluation metrics that we used in this study.</p>
</div>
<div id="S4.SS5.p2" class="ltx_para">
<p id="S4.SS5.p2.1" class="ltx_p"><span id="S4.SS5.p2.1.1" class="ltx_text ltx_font_bold">Attack Success Rate (ASR)</span> is the percentage of the number of successfully reconstructed samples over the number of samples attackedÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>. This metric can be used to evaluate the performance of various attack methods and defense mechanisms. A higher ASR value indicates the high efficacy of a privacy attack method and lower ASR values refer to better performance of the defense mechanisms.</p>
</div>
<div id="S4.SS5.p3" class="ltx_para">
<p id="S4.SS5.p3.1" class="ltx_p"><span id="S4.SS5.p3.1.1" class="ltx_text ltx_font_bold">Mean Squared Error (MSE)</span> is used to quantify the average squared difference between the pixel values of two images, providing a numerical measure of the dissimilarity or error between themÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib60" title="" class="ltx_ref">60</a>]</cite>.
Lower MSE implies higher image similarity, implying smaller average pixel intensity differences, while higher MSE implies greater dissimilarity with larger average differences. MSE is used for evaluating both attack and defense mechanisms in this study. Since MSE refers to the difference between the original private image and the attack reconstructed one, for the attack methods, the lower MSE values indicate the attack is more successful, and exactly the opposite for the defense.</p>
</div>
<div id="S4.SS5.p4" class="ltx_para">
<p id="S4.SS5.p4.1" class="ltx_p"><span id="S4.SS5.p4.1.1" class="ltx_text ltx_font_bold">Structural Similarity Index Measure (SSIM)</span> provides a measure of the structural similarity between two images, taking into account not only pixel intensity differences but also spatial information and human visual perceptionÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib61" title="" class="ltx_ref">61</a>]</cite>. It ranges from 0 to 1 with higher values implying greater image similarity. SSIM is used for evaluating both attack and defense mechanisms. Here, SSIM values present the similarity between the original private image and the attack reconstructed one. So, for the attack methods, higher SSIM values indicate the attack is more successful, and exactly the opposite for defense.</p>
</div>
<figure id="alg1" class="ltx_float ltx_float_algorithm ltx_framed ltx_framed_top">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_float"><span id="alg1.21.1.1" class="ltx_text ltx_font_bold">Algorithm 1</span> </span> <em id="alg1.22.2" class="ltx_emph ltx_font_italic">FedSGD</em>. <em id="alg1.23.3" class="ltx_emph ltx_font_italic">#</em> of clients <math id="alg1.9.m1.1" class="ltx_Math" alttext="C" display="inline"><semantics id="alg1.9.m1.1b"><mi id="alg1.9.m1.1.1" xref="alg1.9.m1.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="alg1.9.m1.1c"><ci id="alg1.9.m1.1.1.cmml" xref="alg1.9.m1.1.1">ğ¶</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.9.m1.1d">C</annotation></semantics></math>, learning rate <math id="alg1.10.m2.1" class="ltx_Math" alttext="\eta" display="inline"><semantics id="alg1.10.m2.1b"><mi id="alg1.10.m2.1.1" xref="alg1.10.m2.1.1.cmml">Î·</mi><annotation-xml encoding="MathML-Content" id="alg1.10.m2.1c"><ci id="alg1.10.m2.1.1.cmml" xref="alg1.10.m2.1.1">ğœ‚</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.10.m2.1d">\eta</annotation></semantics></math>, <em id="alg1.24.4" class="ltx_emph ltx_font_italic">#</em> of local epoch <math id="alg1.11.m3.1" class="ltx_Math" alttext="E" display="inline"><semantics id="alg1.11.m3.1b"><mi id="alg1.11.m3.1.1" xref="alg1.11.m3.1.1.cmml">E</mi><annotation-xml encoding="MathML-Content" id="alg1.11.m3.1c"><ci id="alg1.11.m3.1.1.cmml" xref="alg1.11.m3.1.1">ğ¸</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.11.m3.1d">E</annotation></semantics></math>, <math id="alg1.12.m4.1" class="ltx_Math" alttext="n_{k}" display="inline"><semantics id="alg1.12.m4.1b"><msub id="alg1.12.m4.1.1" xref="alg1.12.m4.1.1.cmml"><mi id="alg1.12.m4.1.1.2" xref="alg1.12.m4.1.1.2.cmml">n</mi><mi id="alg1.12.m4.1.1.3" xref="alg1.12.m4.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="alg1.12.m4.1c"><apply id="alg1.12.m4.1.1.cmml" xref="alg1.12.m4.1.1"><csymbol cd="ambiguous" id="alg1.12.m4.1.1.1.cmml" xref="alg1.12.m4.1.1">subscript</csymbol><ci id="alg1.12.m4.1.1.2.cmml" xref="alg1.12.m4.1.1.2">ğ‘›</ci><ci id="alg1.12.m4.1.1.3.cmml" xref="alg1.12.m4.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.12.m4.1d">n_{k}</annotation></semantics></math> is the number of data samples associated with client <math id="alg1.13.m5.1" class="ltx_Math" alttext="k" display="inline"><semantics id="alg1.13.m5.1b"><mi id="alg1.13.m5.1.1" xref="alg1.13.m5.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="alg1.13.m5.1c"><ci id="alg1.13.m5.1.1.cmml" xref="alg1.13.m5.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.13.m5.1d">k</annotation></semantics></math> from set of <math id="alg1.14.m6.1" class="ltx_Math" alttext="P_{k}" display="inline"><semantics id="alg1.14.m6.1b"><msub id="alg1.14.m6.1.1" xref="alg1.14.m6.1.1.cmml"><mi id="alg1.14.m6.1.1.2" xref="alg1.14.m6.1.1.2.cmml">P</mi><mi id="alg1.14.m6.1.1.3" xref="alg1.14.m6.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="alg1.14.m6.1c"><apply id="alg1.14.m6.1.1.cmml" xref="alg1.14.m6.1.1"><csymbol cd="ambiguous" id="alg1.14.m6.1.1.1.cmml" xref="alg1.14.m6.1.1">subscript</csymbol><ci id="alg1.14.m6.1.1.2.cmml" xref="alg1.14.m6.1.1.2">ğ‘ƒ</ci><ci id="alg1.14.m6.1.1.3.cmml" xref="alg1.14.m6.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.14.m6.1d">P_{k}</annotation></semantics></math>, gradient computed by each client <math id="alg1.15.m7.1" class="ltx_Math" alttext="k" display="inline"><semantics id="alg1.15.m7.1b"><mi id="alg1.15.m7.1.1" xref="alg1.15.m7.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="alg1.15.m7.1c"><ci id="alg1.15.m7.1.1.cmml" xref="alg1.15.m7.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.15.m7.1d">k</annotation></semantics></math>, <math id="alg1.16.m8.1" class="ltx_Math" alttext="g_{k}" display="inline"><semantics id="alg1.16.m8.1b"><msub id="alg1.16.m8.1.1" xref="alg1.16.m8.1.1.cmml"><mi id="alg1.16.m8.1.1.2" xref="alg1.16.m8.1.1.2.cmml">g</mi><mi id="alg1.16.m8.1.1.3" xref="alg1.16.m8.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="alg1.16.m8.1c"><apply id="alg1.16.m8.1.1.cmml" xref="alg1.16.m8.1.1"><csymbol cd="ambiguous" id="alg1.16.m8.1.1.1.cmml" xref="alg1.16.m8.1.1">subscript</csymbol><ci id="alg1.16.m8.1.1.2.cmml" xref="alg1.16.m8.1.1.2">ğ‘”</ci><ci id="alg1.16.m8.1.1.3.cmml" xref="alg1.16.m8.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.16.m8.1d">g_{k}</annotation></semantics></math> <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib62" title="" class="ltx_ref">62</a>]</cite>.
</figcaption>
<div id="alg1.25" class="ltx_listing ltx_listing">
<div id="alg1.l1" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l1.1.1.1" class="ltx_text" style="font-size:80%;">1:</span></span>Â Â <span id="alg1.l1.2" class="ltx_text ltx_font_bold">Serverâ€™s execution:</span>

</div>
<div id="alg1.l2" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l2.1.1.1" class="ltx_text" style="font-size:80%;">2:</span></span>Â Â Initialize <math id="alg1.l2.m1.1" class="ltx_Math" alttext="\omega_{0}" display="inline"><semantics id="alg1.l2.m1.1a"><msub id="alg1.l2.m1.1.1" xref="alg1.l2.m1.1.1.cmml"><mi id="alg1.l2.m1.1.1.2" xref="alg1.l2.m1.1.1.2.cmml">Ï‰</mi><mn id="alg1.l2.m1.1.1.3" xref="alg1.l2.m1.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="alg1.l2.m1.1b"><apply id="alg1.l2.m1.1.1.cmml" xref="alg1.l2.m1.1.1"><csymbol cd="ambiguous" id="alg1.l2.m1.1.1.1.cmml" xref="alg1.l2.m1.1.1">subscript</csymbol><ci id="alg1.l2.m1.1.1.2.cmml" xref="alg1.l2.m1.1.1.2">ğœ”</ci><cn type="integer" id="alg1.l2.m1.1.1.3.cmml" xref="alg1.l2.m1.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l2.m1.1c">\omega_{0}</annotation></semantics></math>

</div>
<div id="alg1.l3" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l3.1.1.1" class="ltx_text" style="font-size:80%;">3:</span></span>Â Â <math id="alg1.l3.m1.1" class="ltx_Math" alttext="k\leftarrow" display="inline"><semantics id="alg1.l3.m1.1a"><mrow id="alg1.l3.m1.1.1" xref="alg1.l3.m1.1.1.cmml"><mi id="alg1.l3.m1.1.1.2" xref="alg1.l3.m1.1.1.2.cmml">k</mi><mo stretchy="false" id="alg1.l3.m1.1.1.1" xref="alg1.l3.m1.1.1.1.cmml">â†</mo><mi id="alg1.l3.m1.1.1.3" xref="alg1.l3.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg1.l3.m1.1b"><apply id="alg1.l3.m1.1.1.cmml" xref="alg1.l3.m1.1.1"><ci id="alg1.l3.m1.1.1.1.cmml" xref="alg1.l3.m1.1.1.1">â†</ci><ci id="alg1.l3.m1.1.1.2.cmml" xref="alg1.l3.m1.1.1.2">ğ‘˜</ci><csymbol cd="latexml" id="alg1.l3.m1.1.1.3.cmml" xref="alg1.l3.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l3.m1.1c">k\leftarrow</annotation></semantics></math>(random set of clients from <math id="alg1.l3.m2.1" class="ltx_Math" alttext="C" display="inline"><semantics id="alg1.l3.m2.1a"><mi id="alg1.l3.m2.1.1" xref="alg1.l3.m2.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="alg1.l3.m2.1b"><ci id="alg1.l3.m2.1.1.cmml" xref="alg1.l3.m2.1.1">ğ¶</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l3.m2.1c">C</annotation></semantics></math>)

</div>
<div id="alg1.l4" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l4.1.1.1" class="ltx_text" style="font-size:80%;">4:</span></span>Â Â <span id="alg1.l4.2" class="ltx_text ltx_font_bold">for</span>Â iterations <math id="alg1.l4.m1.3" class="ltx_Math" alttext="t=1,2,..." display="inline"><semantics id="alg1.l4.m1.3a"><mrow id="alg1.l4.m1.3.4" xref="alg1.l4.m1.3.4.cmml"><mi id="alg1.l4.m1.3.4.2" xref="alg1.l4.m1.3.4.2.cmml">t</mi><mo id="alg1.l4.m1.3.4.1" xref="alg1.l4.m1.3.4.1.cmml">=</mo><mrow id="alg1.l4.m1.3.4.3.2" xref="alg1.l4.m1.3.4.3.1.cmml"><mn id="alg1.l4.m1.1.1" xref="alg1.l4.m1.1.1.cmml">1</mn><mo id="alg1.l4.m1.3.4.3.2.1" xref="alg1.l4.m1.3.4.3.1.cmml">,</mo><mn id="alg1.l4.m1.2.2" xref="alg1.l4.m1.2.2.cmml">2</mn><mo id="alg1.l4.m1.3.4.3.2.2" xref="alg1.l4.m1.3.4.3.1.cmml">,</mo><mi mathvariant="normal" id="alg1.l4.m1.3.3" xref="alg1.l4.m1.3.3.cmml">â€¦</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l4.m1.3b"><apply id="alg1.l4.m1.3.4.cmml" xref="alg1.l4.m1.3.4"><eq id="alg1.l4.m1.3.4.1.cmml" xref="alg1.l4.m1.3.4.1"></eq><ci id="alg1.l4.m1.3.4.2.cmml" xref="alg1.l4.m1.3.4.2">ğ‘¡</ci><list id="alg1.l4.m1.3.4.3.1.cmml" xref="alg1.l4.m1.3.4.3.2"><cn type="integer" id="alg1.l4.m1.1.1.cmml" xref="alg1.l4.m1.1.1">1</cn><cn type="integer" id="alg1.l4.m1.2.2.cmml" xref="alg1.l4.m1.2.2">2</cn><ci id="alg1.l4.m1.3.3.cmml" xref="alg1.l4.m1.3.3">â€¦</ci></list></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l4.m1.3c">t=1,2,...</annotation></semantics></math>Â <span id="alg1.l4.3" class="ltx_text ltx_font_bold">do</span>



</div>
<div id="alg1.l5" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l5.2.1.1" class="ltx_text" style="font-size:80%;">5:</span></span>Â Â Â Â Â <span id="alg1.l5.1" class="ltx_text ltx_font_bold">ClientUpdate(<math id="alg1.l5.1.m1.2" class="ltx_Math" alttext="k,\omega_{t}" display="inline"><semantics id="alg1.l5.1.m1.2a"><mrow id="alg1.l5.1.m1.2.2.1" xref="alg1.l5.1.m1.2.2.2.cmml"><mi id="alg1.l5.1.m1.1.1" xref="alg1.l5.1.m1.1.1.cmml">k</mi><mo id="alg1.l5.1.m1.2.2.1.2" xref="alg1.l5.1.m1.2.2.2.cmml">,</mo><msub id="alg1.l5.1.m1.2.2.1.1" xref="alg1.l5.1.m1.2.2.1.1.cmml"><mi id="alg1.l5.1.m1.2.2.1.1.2" xref="alg1.l5.1.m1.2.2.1.1.2.cmml">Ï‰</mi><mi id="alg1.l5.1.m1.2.2.1.1.3" xref="alg1.l5.1.m1.2.2.1.1.3.cmml">t</mi></msub></mrow><annotation-xml encoding="MathML-Content" id="alg1.l5.1.m1.2b"><list id="alg1.l5.1.m1.2.2.2.cmml" xref="alg1.l5.1.m1.2.2.1"><ci id="alg1.l5.1.m1.1.1.cmml" xref="alg1.l5.1.m1.1.1">ğ‘˜</ci><apply id="alg1.l5.1.m1.2.2.1.1.cmml" xref="alg1.l5.1.m1.2.2.1.1"><csymbol cd="ambiguous" id="alg1.l5.1.m1.2.2.1.1.1.cmml" xref="alg1.l5.1.m1.2.2.1.1">subscript</csymbol><ci id="alg1.l5.1.m1.2.2.1.1.2.cmml" xref="alg1.l5.1.m1.2.2.1.1.2">ğœ”</ci><ci id="alg1.l5.1.m1.2.2.1.1.3.cmml" xref="alg1.l5.1.m1.2.2.1.1.3">ğ‘¡</ci></apply></list></annotation-xml><annotation encoding="application/x-tex" id="alg1.l5.1.m1.2c">k,\omega_{t}</annotation></semantics></math>)</span>

</div>
<div id="alg1.l6" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l6.1.1.1" class="ltx_text" style="font-size:80%;">6:</span></span>Â Â Â Â Â <math id="alg1.l6.m1.1" class="ltx_Math" alttext="\omega_{t+1}\leftarrow\omega_{t}-\eta\sum_{k=1}^{k}\frac{n_{k}}{n}g_{k}" display="inline"><semantics id="alg1.l6.m1.1a"><mrow id="alg1.l6.m1.1.1" xref="alg1.l6.m1.1.1.cmml"><msub id="alg1.l6.m1.1.1.2" xref="alg1.l6.m1.1.1.2.cmml"><mi id="alg1.l6.m1.1.1.2.2" xref="alg1.l6.m1.1.1.2.2.cmml">Ï‰</mi><mrow id="alg1.l6.m1.1.1.2.3" xref="alg1.l6.m1.1.1.2.3.cmml"><mi id="alg1.l6.m1.1.1.2.3.2" xref="alg1.l6.m1.1.1.2.3.2.cmml">t</mi><mo id="alg1.l6.m1.1.1.2.3.1" xref="alg1.l6.m1.1.1.2.3.1.cmml">+</mo><mn id="alg1.l6.m1.1.1.2.3.3" xref="alg1.l6.m1.1.1.2.3.3.cmml">1</mn></mrow></msub><mo stretchy="false" id="alg1.l6.m1.1.1.1" xref="alg1.l6.m1.1.1.1.cmml">â†</mo><mrow id="alg1.l6.m1.1.1.3" xref="alg1.l6.m1.1.1.3.cmml"><msub id="alg1.l6.m1.1.1.3.2" xref="alg1.l6.m1.1.1.3.2.cmml"><mi id="alg1.l6.m1.1.1.3.2.2" xref="alg1.l6.m1.1.1.3.2.2.cmml">Ï‰</mi><mi id="alg1.l6.m1.1.1.3.2.3" xref="alg1.l6.m1.1.1.3.2.3.cmml">t</mi></msub><mo id="alg1.l6.m1.1.1.3.1" xref="alg1.l6.m1.1.1.3.1.cmml">âˆ’</mo><mrow id="alg1.l6.m1.1.1.3.3" xref="alg1.l6.m1.1.1.3.3.cmml"><mi id="alg1.l6.m1.1.1.3.3.2" xref="alg1.l6.m1.1.1.3.3.2.cmml">Î·</mi><mo lspace="0em" rspace="0em" id="alg1.l6.m1.1.1.3.3.1" xref="alg1.l6.m1.1.1.3.3.1.cmml">â€‹</mo><mrow id="alg1.l6.m1.1.1.3.3.3" xref="alg1.l6.m1.1.1.3.3.3.cmml"><msubsup id="alg1.l6.m1.1.1.3.3.3.1" xref="alg1.l6.m1.1.1.3.3.3.1.cmml"><mo id="alg1.l6.m1.1.1.3.3.3.1.2.2" xref="alg1.l6.m1.1.1.3.3.3.1.2.2.cmml">âˆ‘</mo><mrow id="alg1.l6.m1.1.1.3.3.3.1.2.3" xref="alg1.l6.m1.1.1.3.3.3.1.2.3.cmml"><mi id="alg1.l6.m1.1.1.3.3.3.1.2.3.2" xref="alg1.l6.m1.1.1.3.3.3.1.2.3.2.cmml">k</mi><mo id="alg1.l6.m1.1.1.3.3.3.1.2.3.1" xref="alg1.l6.m1.1.1.3.3.3.1.2.3.1.cmml">=</mo><mn id="alg1.l6.m1.1.1.3.3.3.1.2.3.3" xref="alg1.l6.m1.1.1.3.3.3.1.2.3.3.cmml">1</mn></mrow><mi id="alg1.l6.m1.1.1.3.3.3.1.3" xref="alg1.l6.m1.1.1.3.3.3.1.3.cmml">k</mi></msubsup><mrow id="alg1.l6.m1.1.1.3.3.3.2" xref="alg1.l6.m1.1.1.3.3.3.2.cmml"><mfrac id="alg1.l6.m1.1.1.3.3.3.2.2" xref="alg1.l6.m1.1.1.3.3.3.2.2.cmml"><msub id="alg1.l6.m1.1.1.3.3.3.2.2.2" xref="alg1.l6.m1.1.1.3.3.3.2.2.2.cmml"><mi id="alg1.l6.m1.1.1.3.3.3.2.2.2.2" xref="alg1.l6.m1.1.1.3.3.3.2.2.2.2.cmml">n</mi><mi id="alg1.l6.m1.1.1.3.3.3.2.2.2.3" xref="alg1.l6.m1.1.1.3.3.3.2.2.2.3.cmml">k</mi></msub><mi id="alg1.l6.m1.1.1.3.3.3.2.2.3" xref="alg1.l6.m1.1.1.3.3.3.2.2.3.cmml">n</mi></mfrac><mo lspace="0em" rspace="0em" id="alg1.l6.m1.1.1.3.3.3.2.1" xref="alg1.l6.m1.1.1.3.3.3.2.1.cmml">â€‹</mo><msub id="alg1.l6.m1.1.1.3.3.3.2.3" xref="alg1.l6.m1.1.1.3.3.3.2.3.cmml"><mi id="alg1.l6.m1.1.1.3.3.3.2.3.2" xref="alg1.l6.m1.1.1.3.3.3.2.3.2.cmml">g</mi><mi id="alg1.l6.m1.1.1.3.3.3.2.3.3" xref="alg1.l6.m1.1.1.3.3.3.2.3.3.cmml">k</mi></msub></mrow></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l6.m1.1b"><apply id="alg1.l6.m1.1.1.cmml" xref="alg1.l6.m1.1.1"><ci id="alg1.l6.m1.1.1.1.cmml" xref="alg1.l6.m1.1.1.1">â†</ci><apply id="alg1.l6.m1.1.1.2.cmml" xref="alg1.l6.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l6.m1.1.1.2.1.cmml" xref="alg1.l6.m1.1.1.2">subscript</csymbol><ci id="alg1.l6.m1.1.1.2.2.cmml" xref="alg1.l6.m1.1.1.2.2">ğœ”</ci><apply id="alg1.l6.m1.1.1.2.3.cmml" xref="alg1.l6.m1.1.1.2.3"><plus id="alg1.l6.m1.1.1.2.3.1.cmml" xref="alg1.l6.m1.1.1.2.3.1"></plus><ci id="alg1.l6.m1.1.1.2.3.2.cmml" xref="alg1.l6.m1.1.1.2.3.2">ğ‘¡</ci><cn type="integer" id="alg1.l6.m1.1.1.2.3.3.cmml" xref="alg1.l6.m1.1.1.2.3.3">1</cn></apply></apply><apply id="alg1.l6.m1.1.1.3.cmml" xref="alg1.l6.m1.1.1.3"><minus id="alg1.l6.m1.1.1.3.1.cmml" xref="alg1.l6.m1.1.1.3.1"></minus><apply id="alg1.l6.m1.1.1.3.2.cmml" xref="alg1.l6.m1.1.1.3.2"><csymbol cd="ambiguous" id="alg1.l6.m1.1.1.3.2.1.cmml" xref="alg1.l6.m1.1.1.3.2">subscript</csymbol><ci id="alg1.l6.m1.1.1.3.2.2.cmml" xref="alg1.l6.m1.1.1.3.2.2">ğœ”</ci><ci id="alg1.l6.m1.1.1.3.2.3.cmml" xref="alg1.l6.m1.1.1.3.2.3">ğ‘¡</ci></apply><apply id="alg1.l6.m1.1.1.3.3.cmml" xref="alg1.l6.m1.1.1.3.3"><times id="alg1.l6.m1.1.1.3.3.1.cmml" xref="alg1.l6.m1.1.1.3.3.1"></times><ci id="alg1.l6.m1.1.1.3.3.2.cmml" xref="alg1.l6.m1.1.1.3.3.2">ğœ‚</ci><apply id="alg1.l6.m1.1.1.3.3.3.cmml" xref="alg1.l6.m1.1.1.3.3.3"><apply id="alg1.l6.m1.1.1.3.3.3.1.cmml" xref="alg1.l6.m1.1.1.3.3.3.1"><csymbol cd="ambiguous" id="alg1.l6.m1.1.1.3.3.3.1.1.cmml" xref="alg1.l6.m1.1.1.3.3.3.1">superscript</csymbol><apply id="alg1.l6.m1.1.1.3.3.3.1.2.cmml" xref="alg1.l6.m1.1.1.3.3.3.1"><csymbol cd="ambiguous" id="alg1.l6.m1.1.1.3.3.3.1.2.1.cmml" xref="alg1.l6.m1.1.1.3.3.3.1">subscript</csymbol><sum id="alg1.l6.m1.1.1.3.3.3.1.2.2.cmml" xref="alg1.l6.m1.1.1.3.3.3.1.2.2"></sum><apply id="alg1.l6.m1.1.1.3.3.3.1.2.3.cmml" xref="alg1.l6.m1.1.1.3.3.3.1.2.3"><eq id="alg1.l6.m1.1.1.3.3.3.1.2.3.1.cmml" xref="alg1.l6.m1.1.1.3.3.3.1.2.3.1"></eq><ci id="alg1.l6.m1.1.1.3.3.3.1.2.3.2.cmml" xref="alg1.l6.m1.1.1.3.3.3.1.2.3.2">ğ‘˜</ci><cn type="integer" id="alg1.l6.m1.1.1.3.3.3.1.2.3.3.cmml" xref="alg1.l6.m1.1.1.3.3.3.1.2.3.3">1</cn></apply></apply><ci id="alg1.l6.m1.1.1.3.3.3.1.3.cmml" xref="alg1.l6.m1.1.1.3.3.3.1.3">ğ‘˜</ci></apply><apply id="alg1.l6.m1.1.1.3.3.3.2.cmml" xref="alg1.l6.m1.1.1.3.3.3.2"><times id="alg1.l6.m1.1.1.3.3.3.2.1.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.1"></times><apply id="alg1.l6.m1.1.1.3.3.3.2.2.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.2"><divide id="alg1.l6.m1.1.1.3.3.3.2.2.1.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.2"></divide><apply id="alg1.l6.m1.1.1.3.3.3.2.2.2.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.2.2"><csymbol cd="ambiguous" id="alg1.l6.m1.1.1.3.3.3.2.2.2.1.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.2.2">subscript</csymbol><ci id="alg1.l6.m1.1.1.3.3.3.2.2.2.2.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.2.2.2">ğ‘›</ci><ci id="alg1.l6.m1.1.1.3.3.3.2.2.2.3.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.2.2.3">ğ‘˜</ci></apply><ci id="alg1.l6.m1.1.1.3.3.3.2.2.3.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.2.3">ğ‘›</ci></apply><apply id="alg1.l6.m1.1.1.3.3.3.2.3.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.3"><csymbol cd="ambiguous" id="alg1.l6.m1.1.1.3.3.3.2.3.1.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.3">subscript</csymbol><ci id="alg1.l6.m1.1.1.3.3.3.2.3.2.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.3.2">ğ‘”</ci><ci id="alg1.l6.m1.1.1.3.3.3.2.3.3.cmml" xref="alg1.l6.m1.1.1.3.3.3.2.3.3">ğ‘˜</ci></apply></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l6.m1.1c">\omega_{t+1}\leftarrow\omega_{t}-\eta\sum_{k=1}^{k}\frac{n_{k}}{n}g_{k}</annotation></semantics></math>, <math id="alg1.l6.m2.1" class="ltx_Math" alttext="\Bigl{[}\sum_{k=1}^{k}\frac{n_{k}}{n}g_{k}=\nabla L(w_{t})\Bigr{]}" display="inline"><semantics id="alg1.l6.m2.1a"><mrow id="alg1.l6.m2.1.1.1" xref="alg1.l6.m2.1.1.2.cmml"><mo maxsize="160%" minsize="160%" id="alg1.l6.m2.1.1.1.2" xref="alg1.l6.m2.1.1.2.1.cmml">[</mo><mrow id="alg1.l6.m2.1.1.1.1" xref="alg1.l6.m2.1.1.1.1.cmml"><mrow id="alg1.l6.m2.1.1.1.1.3" xref="alg1.l6.m2.1.1.1.1.3.cmml"><msubsup id="alg1.l6.m2.1.1.1.1.3.1" xref="alg1.l6.m2.1.1.1.1.3.1.cmml"><mo lspace="0em" id="alg1.l6.m2.1.1.1.1.3.1.2.2" xref="alg1.l6.m2.1.1.1.1.3.1.2.2.cmml">âˆ‘</mo><mrow id="alg1.l6.m2.1.1.1.1.3.1.2.3" xref="alg1.l6.m2.1.1.1.1.3.1.2.3.cmml"><mi id="alg1.l6.m2.1.1.1.1.3.1.2.3.2" xref="alg1.l6.m2.1.1.1.1.3.1.2.3.2.cmml">k</mi><mo id="alg1.l6.m2.1.1.1.1.3.1.2.3.1" xref="alg1.l6.m2.1.1.1.1.3.1.2.3.1.cmml">=</mo><mn id="alg1.l6.m2.1.1.1.1.3.1.2.3.3" xref="alg1.l6.m2.1.1.1.1.3.1.2.3.3.cmml">1</mn></mrow><mi id="alg1.l6.m2.1.1.1.1.3.1.3" xref="alg1.l6.m2.1.1.1.1.3.1.3.cmml">k</mi></msubsup><mrow id="alg1.l6.m2.1.1.1.1.3.2" xref="alg1.l6.m2.1.1.1.1.3.2.cmml"><mfrac id="alg1.l6.m2.1.1.1.1.3.2.2" xref="alg1.l6.m2.1.1.1.1.3.2.2.cmml"><msub id="alg1.l6.m2.1.1.1.1.3.2.2.2" xref="alg1.l6.m2.1.1.1.1.3.2.2.2.cmml"><mi id="alg1.l6.m2.1.1.1.1.3.2.2.2.2" xref="alg1.l6.m2.1.1.1.1.3.2.2.2.2.cmml">n</mi><mi id="alg1.l6.m2.1.1.1.1.3.2.2.2.3" xref="alg1.l6.m2.1.1.1.1.3.2.2.2.3.cmml">k</mi></msub><mi id="alg1.l6.m2.1.1.1.1.3.2.2.3" xref="alg1.l6.m2.1.1.1.1.3.2.2.3.cmml">n</mi></mfrac><mo lspace="0em" rspace="0em" id="alg1.l6.m2.1.1.1.1.3.2.1" xref="alg1.l6.m2.1.1.1.1.3.2.1.cmml">â€‹</mo><msub id="alg1.l6.m2.1.1.1.1.3.2.3" xref="alg1.l6.m2.1.1.1.1.3.2.3.cmml"><mi id="alg1.l6.m2.1.1.1.1.3.2.3.2" xref="alg1.l6.m2.1.1.1.1.3.2.3.2.cmml">g</mi><mi id="alg1.l6.m2.1.1.1.1.3.2.3.3" xref="alg1.l6.m2.1.1.1.1.3.2.3.3.cmml">k</mi></msub></mrow></mrow><mo id="alg1.l6.m2.1.1.1.1.2" xref="alg1.l6.m2.1.1.1.1.2.cmml">=</mo><mrow id="alg1.l6.m2.1.1.1.1.1" xref="alg1.l6.m2.1.1.1.1.1.cmml"><mrow id="alg1.l6.m2.1.1.1.1.1.3" xref="alg1.l6.m2.1.1.1.1.1.3.cmml"><mo rspace="0.167em" id="alg1.l6.m2.1.1.1.1.1.3.1" xref="alg1.l6.m2.1.1.1.1.1.3.1.cmml">âˆ‡</mo><mi id="alg1.l6.m2.1.1.1.1.1.3.2" xref="alg1.l6.m2.1.1.1.1.1.3.2.cmml">L</mi></mrow><mo lspace="0em" rspace="0em" id="alg1.l6.m2.1.1.1.1.1.2" xref="alg1.l6.m2.1.1.1.1.1.2.cmml">â€‹</mo><mrow id="alg1.l6.m2.1.1.1.1.1.1.1" xref="alg1.l6.m2.1.1.1.1.1.1.1.1.cmml"><mo stretchy="false" id="alg1.l6.m2.1.1.1.1.1.1.1.2" xref="alg1.l6.m2.1.1.1.1.1.1.1.1.cmml">(</mo><msub id="alg1.l6.m2.1.1.1.1.1.1.1.1" xref="alg1.l6.m2.1.1.1.1.1.1.1.1.cmml"><mi id="alg1.l6.m2.1.1.1.1.1.1.1.1.2" xref="alg1.l6.m2.1.1.1.1.1.1.1.1.2.cmml">w</mi><mi id="alg1.l6.m2.1.1.1.1.1.1.1.1.3" xref="alg1.l6.m2.1.1.1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="alg1.l6.m2.1.1.1.1.1.1.1.3" xref="alg1.l6.m2.1.1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow></mrow><mo maxsize="160%" minsize="160%" id="alg1.l6.m2.1.1.1.3" xref="alg1.l6.m2.1.1.2.1.cmml">]</mo></mrow><annotation-xml encoding="MathML-Content" id="alg1.l6.m2.1b"><apply id="alg1.l6.m2.1.1.2.cmml" xref="alg1.l6.m2.1.1.1"><csymbol cd="latexml" id="alg1.l6.m2.1.1.2.1.cmml" xref="alg1.l6.m2.1.1.1.2">delimited-[]</csymbol><apply id="alg1.l6.m2.1.1.1.1.cmml" xref="alg1.l6.m2.1.1.1.1"><eq id="alg1.l6.m2.1.1.1.1.2.cmml" xref="alg1.l6.m2.1.1.1.1.2"></eq><apply id="alg1.l6.m2.1.1.1.1.3.cmml" xref="alg1.l6.m2.1.1.1.1.3"><apply id="alg1.l6.m2.1.1.1.1.3.1.cmml" xref="alg1.l6.m2.1.1.1.1.3.1"><csymbol cd="ambiguous" id="alg1.l6.m2.1.1.1.1.3.1.1.cmml" xref="alg1.l6.m2.1.1.1.1.3.1">superscript</csymbol><apply id="alg1.l6.m2.1.1.1.1.3.1.2.cmml" xref="alg1.l6.m2.1.1.1.1.3.1"><csymbol cd="ambiguous" id="alg1.l6.m2.1.1.1.1.3.1.2.1.cmml" xref="alg1.l6.m2.1.1.1.1.3.1">subscript</csymbol><sum id="alg1.l6.m2.1.1.1.1.3.1.2.2.cmml" xref="alg1.l6.m2.1.1.1.1.3.1.2.2"></sum><apply id="alg1.l6.m2.1.1.1.1.3.1.2.3.cmml" xref="alg1.l6.m2.1.1.1.1.3.1.2.3"><eq id="alg1.l6.m2.1.1.1.1.3.1.2.3.1.cmml" xref="alg1.l6.m2.1.1.1.1.3.1.2.3.1"></eq><ci id="alg1.l6.m2.1.1.1.1.3.1.2.3.2.cmml" xref="alg1.l6.m2.1.1.1.1.3.1.2.3.2">ğ‘˜</ci><cn type="integer" id="alg1.l6.m2.1.1.1.1.3.1.2.3.3.cmml" xref="alg1.l6.m2.1.1.1.1.3.1.2.3.3">1</cn></apply></apply><ci id="alg1.l6.m2.1.1.1.1.3.1.3.cmml" xref="alg1.l6.m2.1.1.1.1.3.1.3">ğ‘˜</ci></apply><apply id="alg1.l6.m2.1.1.1.1.3.2.cmml" xref="alg1.l6.m2.1.1.1.1.3.2"><times id="alg1.l6.m2.1.1.1.1.3.2.1.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.1"></times><apply id="alg1.l6.m2.1.1.1.1.3.2.2.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.2"><divide id="alg1.l6.m2.1.1.1.1.3.2.2.1.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.2"></divide><apply id="alg1.l6.m2.1.1.1.1.3.2.2.2.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.2.2"><csymbol cd="ambiguous" id="alg1.l6.m2.1.1.1.1.3.2.2.2.1.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.2.2">subscript</csymbol><ci id="alg1.l6.m2.1.1.1.1.3.2.2.2.2.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.2.2.2">ğ‘›</ci><ci id="alg1.l6.m2.1.1.1.1.3.2.2.2.3.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.2.2.3">ğ‘˜</ci></apply><ci id="alg1.l6.m2.1.1.1.1.3.2.2.3.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.2.3">ğ‘›</ci></apply><apply id="alg1.l6.m2.1.1.1.1.3.2.3.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.3"><csymbol cd="ambiguous" id="alg1.l6.m2.1.1.1.1.3.2.3.1.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.3">subscript</csymbol><ci id="alg1.l6.m2.1.1.1.1.3.2.3.2.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.3.2">ğ‘”</ci><ci id="alg1.l6.m2.1.1.1.1.3.2.3.3.cmml" xref="alg1.l6.m2.1.1.1.1.3.2.3.3">ğ‘˜</ci></apply></apply></apply><apply id="alg1.l6.m2.1.1.1.1.1.cmml" xref="alg1.l6.m2.1.1.1.1.1"><times id="alg1.l6.m2.1.1.1.1.1.2.cmml" xref="alg1.l6.m2.1.1.1.1.1.2"></times><apply id="alg1.l6.m2.1.1.1.1.1.3.cmml" xref="alg1.l6.m2.1.1.1.1.1.3"><ci id="alg1.l6.m2.1.1.1.1.1.3.1.cmml" xref="alg1.l6.m2.1.1.1.1.1.3.1">âˆ‡</ci><ci id="alg1.l6.m2.1.1.1.1.1.3.2.cmml" xref="alg1.l6.m2.1.1.1.1.1.3.2">ğ¿</ci></apply><apply id="alg1.l6.m2.1.1.1.1.1.1.1.1.cmml" xref="alg1.l6.m2.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="alg1.l6.m2.1.1.1.1.1.1.1.1.1.cmml" xref="alg1.l6.m2.1.1.1.1.1.1.1">subscript</csymbol><ci id="alg1.l6.m2.1.1.1.1.1.1.1.1.2.cmml" xref="alg1.l6.m2.1.1.1.1.1.1.1.1.2">ğ‘¤</ci><ci id="alg1.l6.m2.1.1.1.1.1.1.1.1.3.cmml" xref="alg1.l6.m2.1.1.1.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l6.m2.1c">\Bigl{[}\sum_{k=1}^{k}\frac{n_{k}}{n}g_{k}=\nabla L(w_{t})\Bigr{]}</annotation></semantics></math>

</div>
<div id="alg1.l7" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l7.1.1.1" class="ltx_text" style="font-size:80%;">7:</span></span>Â Â <span id="alg1.l7.2" class="ltx_text ltx_font_bold">end</span>Â <span id="alg1.l7.3" class="ltx_text ltx_font_bold">for</span>
</div>
<div id="alg1.l8" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l8.1.1.1" class="ltx_text" style="font-size:80%;">8:</span></span>Â Â <span id="alg1.l8.2" class="ltx_text ltx_font_bold">Clientsâ€™ execution:</span>

</div>
<div id="alg1.l9" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l9.2.1.1" class="ltx_text" style="font-size:80%;">9:</span></span>Â Â <span id="alg1.l9.1" class="ltx_text ltx_font_bold">ClientUpdate(<math id="alg1.l9.1.m1.2" class="ltx_Math" alttext="k,\omega_{t}" display="inline"><semantics id="alg1.l9.1.m1.2a"><mrow id="alg1.l9.1.m1.2.2.1" xref="alg1.l9.1.m1.2.2.2.cmml"><mi id="alg1.l9.1.m1.1.1" xref="alg1.l9.1.m1.1.1.cmml">k</mi><mo id="alg1.l9.1.m1.2.2.1.2" xref="alg1.l9.1.m1.2.2.2.cmml">,</mo><msub id="alg1.l9.1.m1.2.2.1.1" xref="alg1.l9.1.m1.2.2.1.1.cmml"><mi id="alg1.l9.1.m1.2.2.1.1.2" xref="alg1.l9.1.m1.2.2.1.1.2.cmml">Ï‰</mi><mi id="alg1.l9.1.m1.2.2.1.1.3" xref="alg1.l9.1.m1.2.2.1.1.3.cmml">t</mi></msub></mrow><annotation-xml encoding="MathML-Content" id="alg1.l9.1.m1.2b"><list id="alg1.l9.1.m1.2.2.2.cmml" xref="alg1.l9.1.m1.2.2.1"><ci id="alg1.l9.1.m1.1.1.cmml" xref="alg1.l9.1.m1.1.1">ğ‘˜</ci><apply id="alg1.l9.1.m1.2.2.1.1.cmml" xref="alg1.l9.1.m1.2.2.1.1"><csymbol cd="ambiguous" id="alg1.l9.1.m1.2.2.1.1.1.cmml" xref="alg1.l9.1.m1.2.2.1.1">subscript</csymbol><ci id="alg1.l9.1.m1.2.2.1.1.2.cmml" xref="alg1.l9.1.m1.2.2.1.1.2">ğœ”</ci><ci id="alg1.l9.1.m1.2.2.1.1.3.cmml" xref="alg1.l9.1.m1.2.2.1.1.3">ğ‘¡</ci></apply></list></annotation-xml><annotation encoding="application/x-tex" id="alg1.l9.1.m1.2c">k,\omega_{t}</annotation></semantics></math>):</span>

</div>
<div id="alg1.l10" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l10.1.1.1" class="ltx_text" style="font-size:80%;">10:</span></span>Â Â <math id="alg1.l10.m1.1" class="ltx_Math" alttext="g_{k}\leftarrow\nabla L_{k}(\omega_{t})" display="inline"><semantics id="alg1.l10.m1.1a"><mrow id="alg1.l10.m1.1.1" xref="alg1.l10.m1.1.1.cmml"><msub id="alg1.l10.m1.1.1.3" xref="alg1.l10.m1.1.1.3.cmml"><mi id="alg1.l10.m1.1.1.3.2" xref="alg1.l10.m1.1.1.3.2.cmml">g</mi><mi id="alg1.l10.m1.1.1.3.3" xref="alg1.l10.m1.1.1.3.3.cmml">k</mi></msub><mo stretchy="false" id="alg1.l10.m1.1.1.2" xref="alg1.l10.m1.1.1.2.cmml">â†</mo><mrow id="alg1.l10.m1.1.1.1" xref="alg1.l10.m1.1.1.1.cmml"><mrow id="alg1.l10.m1.1.1.1.3" xref="alg1.l10.m1.1.1.1.3.cmml"><mo rspace="0.167em" id="alg1.l10.m1.1.1.1.3.1" xref="alg1.l10.m1.1.1.1.3.1.cmml">âˆ‡</mo><msub id="alg1.l10.m1.1.1.1.3.2" xref="alg1.l10.m1.1.1.1.3.2.cmml"><mi id="alg1.l10.m1.1.1.1.3.2.2" xref="alg1.l10.m1.1.1.1.3.2.2.cmml">L</mi><mi id="alg1.l10.m1.1.1.1.3.2.3" xref="alg1.l10.m1.1.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="alg1.l10.m1.1.1.1.2" xref="alg1.l10.m1.1.1.1.2.cmml">â€‹</mo><mrow id="alg1.l10.m1.1.1.1.1.1" xref="alg1.l10.m1.1.1.1.1.1.1.cmml"><mo stretchy="false" id="alg1.l10.m1.1.1.1.1.1.2" xref="alg1.l10.m1.1.1.1.1.1.1.cmml">(</mo><msub id="alg1.l10.m1.1.1.1.1.1.1" xref="alg1.l10.m1.1.1.1.1.1.1.cmml"><mi id="alg1.l10.m1.1.1.1.1.1.1.2" xref="alg1.l10.m1.1.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="alg1.l10.m1.1.1.1.1.1.1.3" xref="alg1.l10.m1.1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="alg1.l10.m1.1.1.1.1.1.3" xref="alg1.l10.m1.1.1.1.1.1.1.cmml">)</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l10.m1.1b"><apply id="alg1.l10.m1.1.1.cmml" xref="alg1.l10.m1.1.1"><ci id="alg1.l10.m1.1.1.2.cmml" xref="alg1.l10.m1.1.1.2">â†</ci><apply id="alg1.l10.m1.1.1.3.cmml" xref="alg1.l10.m1.1.1.3"><csymbol cd="ambiguous" id="alg1.l10.m1.1.1.3.1.cmml" xref="alg1.l10.m1.1.1.3">subscript</csymbol><ci id="alg1.l10.m1.1.1.3.2.cmml" xref="alg1.l10.m1.1.1.3.2">ğ‘”</ci><ci id="alg1.l10.m1.1.1.3.3.cmml" xref="alg1.l10.m1.1.1.3.3">ğ‘˜</ci></apply><apply id="alg1.l10.m1.1.1.1.cmml" xref="alg1.l10.m1.1.1.1"><times id="alg1.l10.m1.1.1.1.2.cmml" xref="alg1.l10.m1.1.1.1.2"></times><apply id="alg1.l10.m1.1.1.1.3.cmml" xref="alg1.l10.m1.1.1.1.3"><ci id="alg1.l10.m1.1.1.1.3.1.cmml" xref="alg1.l10.m1.1.1.1.3.1">âˆ‡</ci><apply id="alg1.l10.m1.1.1.1.3.2.cmml" xref="alg1.l10.m1.1.1.1.3.2"><csymbol cd="ambiguous" id="alg1.l10.m1.1.1.1.3.2.1.cmml" xref="alg1.l10.m1.1.1.1.3.2">subscript</csymbol><ci id="alg1.l10.m1.1.1.1.3.2.2.cmml" xref="alg1.l10.m1.1.1.1.3.2.2">ğ¿</ci><ci id="alg1.l10.m1.1.1.1.3.2.3.cmml" xref="alg1.l10.m1.1.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="alg1.l10.m1.1.1.1.1.1.1.cmml" xref="alg1.l10.m1.1.1.1.1.1"><csymbol cd="ambiguous" id="alg1.l10.m1.1.1.1.1.1.1.1.cmml" xref="alg1.l10.m1.1.1.1.1.1">subscript</csymbol><ci id="alg1.l10.m1.1.1.1.1.1.1.2.cmml" xref="alg1.l10.m1.1.1.1.1.1.1.2">ğœ”</ci><ci id="alg1.l10.m1.1.1.1.1.1.1.3.cmml" xref="alg1.l10.m1.1.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l10.m1.1c">g_{k}\leftarrow\nabla L_{k}(\omega_{t})</annotation></semantics></math>

</div>
<div id="alg1.l11" class="ltx_listingline">
<span class="ltx_tag ltx_tag_listingline"><span id="alg1.l11.1.1.1" class="ltx_text" style="font-size:80%;">11:</span></span>Â Â Return <math id="alg1.l11.m1.1" class="ltx_Math" alttext="g_{k}" display="inline"><semantics id="alg1.l11.m1.1a"><msub id="alg1.l11.m1.1.1" xref="alg1.l11.m1.1.1.cmml"><mi id="alg1.l11.m1.1.1.2" xref="alg1.l11.m1.1.1.2.cmml">g</mi><mi id="alg1.l11.m1.1.1.3" xref="alg1.l11.m1.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="alg1.l11.m1.1b"><apply id="alg1.l11.m1.1.1.cmml" xref="alg1.l11.m1.1.1"><csymbol cd="ambiguous" id="alg1.l11.m1.1.1.1.cmml" xref="alg1.l11.m1.1.1">subscript</csymbol><ci id="alg1.l11.m1.1.1.2.cmml" xref="alg1.l11.m1.1.1.2">ğ‘”</ci><ci id="alg1.l11.m1.1.1.3.cmml" xref="alg1.l11.m1.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l11.m1.1c">g_{k}</annotation></semantics></math> to the central server.

</div>
</div>
</figure>
</section>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">V </span><span id="S5.1.1" class="ltx_text ltx_font_smallcaps">Methodology</span>
</h2>

<div id="S5.p1" class="ltx_para">
<p id="S5.p1.1" class="ltx_p">In the healthcare system, FL involves the decentralization of ML models from a central server to be distributed across a group of hospitals and clinics, referred to as client nodes. Since it is uncertain that all the clients are available, a small number is chosen from the pool of participants to participate in collaborative learning during each iteration.</p>
</div>
<section id="S5.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS1.5.1.1" class="ltx_text">V-A</span> </span><span id="S5.SS1.6.2" class="ltx_text ltx_font_italic">FL Architecture</span>
</h3>

<div id="S5.SS1.p1" class="ltx_para">
<p id="S5.SS1.p1.8" class="ltx_p">We present the representative FL aggregation algorithm, FedSGDÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib62" title="" class="ltx_ref">62</a>]</cite>, in Algorithm <a href="#alg1" title="Algorithm 1 â€£ IV-E Evaluation Metrics â€£ IV Framework Overview â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>. First, the central server initiates the global model <math id="S5.SS1.p1.1.m1.1" class="ltx_Math" alttext="\omega_{0}" display="inline"><semantics id="S5.SS1.p1.1.m1.1a"><msub id="S5.SS1.p1.1.m1.1.1" xref="S5.SS1.p1.1.m1.1.1.cmml"><mi id="S5.SS1.p1.1.m1.1.1.2" xref="S5.SS1.p1.1.m1.1.1.2.cmml">Ï‰</mi><mn id="S5.SS1.p1.1.m1.1.1.3" xref="S5.SS1.p1.1.m1.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="S5.SS1.p1.1.m1.1b"><apply id="S5.SS1.p1.1.m1.1.1.cmml" xref="S5.SS1.p1.1.m1.1.1"><csymbol cd="ambiguous" id="S5.SS1.p1.1.m1.1.1.1.cmml" xref="S5.SS1.p1.1.m1.1.1">subscript</csymbol><ci id="S5.SS1.p1.1.m1.1.1.2.cmml" xref="S5.SS1.p1.1.m1.1.1.2">ğœ”</ci><cn type="integer" id="S5.SS1.p1.1.m1.1.1.3.cmml" xref="S5.SS1.p1.1.m1.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p1.1.m1.1c">\omega_{0}</annotation></semantics></math> and shares it with the selected clients at round <math id="S5.SS1.p1.2.m2.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S5.SS1.p1.2.m2.1a"><mi id="S5.SS1.p1.2.m2.1.1" xref="S5.SS1.p1.2.m2.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S5.SS1.p1.2.m2.1b"><ci id="S5.SS1.p1.2.m2.1.1.cmml" xref="S5.SS1.p1.2.m2.1.1">ğ‘¡</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p1.2.m2.1c">t</annotation></semantics></math>. Each client <math id="S5.SS1.p1.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S5.SS1.p1.3.m3.1a"><mi id="S5.SS1.p1.3.m3.1.1" xref="S5.SS1.p1.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S5.SS1.p1.3.m3.1b"><ci id="S5.SS1.p1.3.m3.1.1.cmml" xref="S5.SS1.p1.3.m3.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p1.3.m3.1c">k</annotation></semantics></math> performs training and computes the gradient <math id="S5.SS1.p1.4.m4.1" class="ltx_Math" alttext="g_{k}" display="inline"><semantics id="S5.SS1.p1.4.m4.1a"><msub id="S5.SS1.p1.4.m4.1.1" xref="S5.SS1.p1.4.m4.1.1.cmml"><mi id="S5.SS1.p1.4.m4.1.1.2" xref="S5.SS1.p1.4.m4.1.1.2.cmml">g</mi><mi id="S5.SS1.p1.4.m4.1.1.3" xref="S5.SS1.p1.4.m4.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="S5.SS1.p1.4.m4.1b"><apply id="S5.SS1.p1.4.m4.1.1.cmml" xref="S5.SS1.p1.4.m4.1.1"><csymbol cd="ambiguous" id="S5.SS1.p1.4.m4.1.1.1.cmml" xref="S5.SS1.p1.4.m4.1.1">subscript</csymbol><ci id="S5.SS1.p1.4.m4.1.1.2.cmml" xref="S5.SS1.p1.4.m4.1.1.2">ğ‘”</ci><ci id="S5.SS1.p1.4.m4.1.1.3.cmml" xref="S5.SS1.p1.4.m4.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p1.4.m4.1c">g_{k}</annotation></semantics></math> of the loss function with respect to model weights on their private training data at iteration <math id="S5.SS1.p1.5.m5.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S5.SS1.p1.5.m5.1a"><mi id="S5.SS1.p1.5.m5.1.1" xref="S5.SS1.p1.5.m5.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S5.SS1.p1.5.m5.1b"><ci id="S5.SS1.p1.5.m5.1.1.cmml" xref="S5.SS1.p1.5.m5.1.1">ğ‘¡</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p1.5.m5.1c">t</annotation></semantics></math> and sends <math id="S5.SS1.p1.6.m6.1" class="ltx_Math" alttext="g_{k}" display="inline"><semantics id="S5.SS1.p1.6.m6.1a"><msub id="S5.SS1.p1.6.m6.1.1" xref="S5.SS1.p1.6.m6.1.1.cmml"><mi id="S5.SS1.p1.6.m6.1.1.2" xref="S5.SS1.p1.6.m6.1.1.2.cmml">g</mi><mi id="S5.SS1.p1.6.m6.1.1.3" xref="S5.SS1.p1.6.m6.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="S5.SS1.p1.6.m6.1b"><apply id="S5.SS1.p1.6.m6.1.1.cmml" xref="S5.SS1.p1.6.m6.1.1"><csymbol cd="ambiguous" id="S5.SS1.p1.6.m6.1.1.1.cmml" xref="S5.SS1.p1.6.m6.1.1">subscript</csymbol><ci id="S5.SS1.p1.6.m6.1.1.2.cmml" xref="S5.SS1.p1.6.m6.1.1.2">ğ‘”</ci><ci id="S5.SS1.p1.6.m6.1.1.3.cmml" xref="S5.SS1.p1.6.m6.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p1.6.m6.1c">g_{k}</annotation></semantics></math> back to the central server. Then, the central server aggregates the gradients from the selected participating clients <math id="S5.SS1.p1.7.m7.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S5.SS1.p1.7.m7.1a"><mi id="S5.SS1.p1.7.m7.1.1" xref="S5.SS1.p1.7.m7.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S5.SS1.p1.7.m7.1b"><ci id="S5.SS1.p1.7.m7.1.1.cmml" xref="S5.SS1.p1.7.m7.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p1.7.m7.1c">k</annotation></semantics></math>, weighted by the number of data samples associated with <math id="S5.SS1.p1.8.m8.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S5.SS1.p1.8.m8.1a"><mi id="S5.SS1.p1.8.m8.1.1" xref="S5.SS1.p1.8.m8.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S5.SS1.p1.8.m8.1b"><ci id="S5.SS1.p1.8.m8.1.1.cmml" xref="S5.SS1.p1.8.m8.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS1.p1.8.m8.1c">k</annotation></semantics></math>.</p>
</div>
</section>
<section id="S5.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS2.5.1.1" class="ltx_text">V-B</span> </span><span id="S5.SS2.6.2" class="ltx_text ltx_font_italic">FL Architecture in Medical Domain</span>
</h3>

<div id="S5.SS2.p1" class="ltx_para">
<p id="S5.SS2.p1.7" class="ltx_p">In the healthcare sector, FL entails duplicating ML models from a central server and disseminating them among a set of clients, including clinics, hospitals, and healthcare organizations. The procedure typically works as follows. Initially, in <span id="S5.SS2.p1.7.1" class="ltx_text ltx_font_italic">step 1</span>, each client receives a global model denoted as <math id="S5.SS2.p1.1.m1.1" class="ltx_Math" alttext="\omega_{0}" display="inline"><semantics id="S5.SS2.p1.1.m1.1a"><msub id="S5.SS2.p1.1.m1.1.1" xref="S5.SS2.p1.1.m1.1.1.cmml"><mi id="S5.SS2.p1.1.m1.1.1.2" xref="S5.SS2.p1.1.m1.1.1.2.cmml">Ï‰</mi><mn id="S5.SS2.p1.1.m1.1.1.3" xref="S5.SS2.p1.1.m1.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="S5.SS2.p1.1.m1.1b"><apply id="S5.SS2.p1.1.m1.1.1.cmml" xref="S5.SS2.p1.1.m1.1.1"><csymbol cd="ambiguous" id="S5.SS2.p1.1.m1.1.1.1.cmml" xref="S5.SS2.p1.1.m1.1.1">subscript</csymbol><ci id="S5.SS2.p1.1.m1.1.1.2.cmml" xref="S5.SS2.p1.1.m1.1.1.2">ğœ”</ci><cn type="integer" id="S5.SS2.p1.1.m1.1.1.3.cmml" xref="S5.SS2.p1.1.m1.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p1.1.m1.1c">\omega_{0}</annotation></semantics></math> at round <math id="S5.SS2.p1.2.m2.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S5.SS2.p1.2.m2.1a"><mi id="S5.SS2.p1.2.m2.1.1" xref="S5.SS2.p1.2.m2.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S5.SS2.p1.2.m2.1b"><ci id="S5.SS2.p1.2.m2.1.1.cmml" xref="S5.SS2.p1.2.m2.1.1">ğ‘¡</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p1.2.m2.1c">t</annotation></semantics></math> from the trusted centralized server for client <math id="S5.SS2.p1.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S5.SS2.p1.3.m3.1a"><mi id="S5.SS2.p1.3.m3.1.1" xref="S5.SS2.p1.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S5.SS2.p1.3.m3.1b"><ci id="S5.SS2.p1.3.m3.1.1.cmml" xref="S5.SS2.p1.3.m3.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p1.3.m3.1c">k</annotation></semantics></math>. In <span id="S5.SS2.p1.7.2" class="ltx_text ltx_font_italic">step 2</span>, each client updates the local model and computes gradient, <math id="S5.SS2.p1.4.m4.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S5.SS2.p1.4.m4.1a"><mrow id="S5.SS2.p1.4.m4.1.1" xref="S5.SS2.p1.4.m4.1.1.cmml"><mrow id="S5.SS2.p1.4.m4.1.1.3" xref="S5.SS2.p1.4.m4.1.1.3.cmml"><mo rspace="0.167em" id="S5.SS2.p1.4.m4.1.1.3.1" xref="S5.SS2.p1.4.m4.1.1.3.1.cmml">âˆ‡</mo><msub id="S5.SS2.p1.4.m4.1.1.3.2" xref="S5.SS2.p1.4.m4.1.1.3.2.cmml"><mi id="S5.SS2.p1.4.m4.1.1.3.2.2" xref="S5.SS2.p1.4.m4.1.1.3.2.2.cmml">L</mi><mi id="S5.SS2.p1.4.m4.1.1.3.2.3" xref="S5.SS2.p1.4.m4.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S5.SS2.p1.4.m4.1.1.2" xref="S5.SS2.p1.4.m4.1.1.2.cmml">â€‹</mo><mrow id="S5.SS2.p1.4.m4.1.1.1.1" xref="S5.SS2.p1.4.m4.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS2.p1.4.m4.1.1.1.1.2" xref="S5.SS2.p1.4.m4.1.1.1.1.1.cmml">(</mo><msub id="S5.SS2.p1.4.m4.1.1.1.1.1" xref="S5.SS2.p1.4.m4.1.1.1.1.1.cmml"><mi id="S5.SS2.p1.4.m4.1.1.1.1.1.2" xref="S5.SS2.p1.4.m4.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S5.SS2.p1.4.m4.1.1.1.1.1.3" xref="S5.SS2.p1.4.m4.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.SS2.p1.4.m4.1.1.1.1.3" xref="S5.SS2.p1.4.m4.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.p1.4.m4.1b"><apply id="S5.SS2.p1.4.m4.1.1.cmml" xref="S5.SS2.p1.4.m4.1.1"><times id="S5.SS2.p1.4.m4.1.1.2.cmml" xref="S5.SS2.p1.4.m4.1.1.2"></times><apply id="S5.SS2.p1.4.m4.1.1.3.cmml" xref="S5.SS2.p1.4.m4.1.1.3"><ci id="S5.SS2.p1.4.m4.1.1.3.1.cmml" xref="S5.SS2.p1.4.m4.1.1.3.1">âˆ‡</ci><apply id="S5.SS2.p1.4.m4.1.1.3.2.cmml" xref="S5.SS2.p1.4.m4.1.1.3.2"><csymbol cd="ambiguous" id="S5.SS2.p1.4.m4.1.1.3.2.1.cmml" xref="S5.SS2.p1.4.m4.1.1.3.2">subscript</csymbol><ci id="S5.SS2.p1.4.m4.1.1.3.2.2.cmml" xref="S5.SS2.p1.4.m4.1.1.3.2.2">ğ¿</ci><ci id="S5.SS2.p1.4.m4.1.1.3.2.3.cmml" xref="S5.SS2.p1.4.m4.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S5.SS2.p1.4.m4.1.1.1.1.1.cmml" xref="S5.SS2.p1.4.m4.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS2.p1.4.m4.1.1.1.1.1.1.cmml" xref="S5.SS2.p1.4.m4.1.1.1.1">subscript</csymbol><ci id="S5.SS2.p1.4.m4.1.1.1.1.1.2.cmml" xref="S5.SS2.p1.4.m4.1.1.1.1.1.2">ğœ”</ci><ci id="S5.SS2.p1.4.m4.1.1.1.1.1.3.cmml" xref="S5.SS2.p1.4.m4.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p1.4.m4.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math>, utilizing its private medical data, <math id="S5.SS2.p1.5.m5.2" class="ltx_Math" alttext="T(k,t)" display="inline"><semantics id="S5.SS2.p1.5.m5.2a"><mrow id="S5.SS2.p1.5.m5.2.3" xref="S5.SS2.p1.5.m5.2.3.cmml"><mi id="S5.SS2.p1.5.m5.2.3.2" xref="S5.SS2.p1.5.m5.2.3.2.cmml">T</mi><mo lspace="0em" rspace="0em" id="S5.SS2.p1.5.m5.2.3.1" xref="S5.SS2.p1.5.m5.2.3.1.cmml">â€‹</mo><mrow id="S5.SS2.p1.5.m5.2.3.3.2" xref="S5.SS2.p1.5.m5.2.3.3.1.cmml"><mo stretchy="false" id="S5.SS2.p1.5.m5.2.3.3.2.1" xref="S5.SS2.p1.5.m5.2.3.3.1.cmml">(</mo><mi id="S5.SS2.p1.5.m5.1.1" xref="S5.SS2.p1.5.m5.1.1.cmml">k</mi><mo id="S5.SS2.p1.5.m5.2.3.3.2.2" xref="S5.SS2.p1.5.m5.2.3.3.1.cmml">,</mo><mi id="S5.SS2.p1.5.m5.2.2" xref="S5.SS2.p1.5.m5.2.2.cmml">t</mi><mo stretchy="false" id="S5.SS2.p1.5.m5.2.3.3.2.3" xref="S5.SS2.p1.5.m5.2.3.3.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.p1.5.m5.2b"><apply id="S5.SS2.p1.5.m5.2.3.cmml" xref="S5.SS2.p1.5.m5.2.3"><times id="S5.SS2.p1.5.m5.2.3.1.cmml" xref="S5.SS2.p1.5.m5.2.3.1"></times><ci id="S5.SS2.p1.5.m5.2.3.2.cmml" xref="S5.SS2.p1.5.m5.2.3.2">ğ‘‡</ci><interval closure="open" id="S5.SS2.p1.5.m5.2.3.3.1.cmml" xref="S5.SS2.p1.5.m5.2.3.3.2"><ci id="S5.SS2.p1.5.m5.1.1.cmml" xref="S5.SS2.p1.5.m5.1.1">ğ‘˜</ci><ci id="S5.SS2.p1.5.m5.2.2.cmml" xref="S5.SS2.p1.5.m5.2.2">ğ‘¡</ci></interval></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p1.5.m5.2c">T(k,t)</annotation></semantics></math>. Subsequently, in <span id="S5.SS2.p1.7.3" class="ltx_text ltx_font_italic">step 3</span>, gradients, <math id="S5.SS2.p1.6.m6.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S5.SS2.p1.6.m6.1a"><mrow id="S5.SS2.p1.6.m6.1.1" xref="S5.SS2.p1.6.m6.1.1.cmml"><mrow id="S5.SS2.p1.6.m6.1.1.3" xref="S5.SS2.p1.6.m6.1.1.3.cmml"><mo rspace="0.167em" id="S5.SS2.p1.6.m6.1.1.3.1" xref="S5.SS2.p1.6.m6.1.1.3.1.cmml">âˆ‡</mo><msub id="S5.SS2.p1.6.m6.1.1.3.2" xref="S5.SS2.p1.6.m6.1.1.3.2.cmml"><mi id="S5.SS2.p1.6.m6.1.1.3.2.2" xref="S5.SS2.p1.6.m6.1.1.3.2.2.cmml">L</mi><mi id="S5.SS2.p1.6.m6.1.1.3.2.3" xref="S5.SS2.p1.6.m6.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S5.SS2.p1.6.m6.1.1.2" xref="S5.SS2.p1.6.m6.1.1.2.cmml">â€‹</mo><mrow id="S5.SS2.p1.6.m6.1.1.1.1" xref="S5.SS2.p1.6.m6.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS2.p1.6.m6.1.1.1.1.2" xref="S5.SS2.p1.6.m6.1.1.1.1.1.cmml">(</mo><msub id="S5.SS2.p1.6.m6.1.1.1.1.1" xref="S5.SS2.p1.6.m6.1.1.1.1.1.cmml"><mi id="S5.SS2.p1.6.m6.1.1.1.1.1.2" xref="S5.SS2.p1.6.m6.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S5.SS2.p1.6.m6.1.1.1.1.1.3" xref="S5.SS2.p1.6.m6.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.SS2.p1.6.m6.1.1.1.1.3" xref="S5.SS2.p1.6.m6.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.SS2.p1.6.m6.1b"><apply id="S5.SS2.p1.6.m6.1.1.cmml" xref="S5.SS2.p1.6.m6.1.1"><times id="S5.SS2.p1.6.m6.1.1.2.cmml" xref="S5.SS2.p1.6.m6.1.1.2"></times><apply id="S5.SS2.p1.6.m6.1.1.3.cmml" xref="S5.SS2.p1.6.m6.1.1.3"><ci id="S5.SS2.p1.6.m6.1.1.3.1.cmml" xref="S5.SS2.p1.6.m6.1.1.3.1">âˆ‡</ci><apply id="S5.SS2.p1.6.m6.1.1.3.2.cmml" xref="S5.SS2.p1.6.m6.1.1.3.2"><csymbol cd="ambiguous" id="S5.SS2.p1.6.m6.1.1.3.2.1.cmml" xref="S5.SS2.p1.6.m6.1.1.3.2">subscript</csymbol><ci id="S5.SS2.p1.6.m6.1.1.3.2.2.cmml" xref="S5.SS2.p1.6.m6.1.1.3.2.2">ğ¿</ci><ci id="S5.SS2.p1.6.m6.1.1.3.2.3.cmml" xref="S5.SS2.p1.6.m6.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S5.SS2.p1.6.m6.1.1.1.1.1.cmml" xref="S5.SS2.p1.6.m6.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS2.p1.6.m6.1.1.1.1.1.1.cmml" xref="S5.SS2.p1.6.m6.1.1.1.1">subscript</csymbol><ci id="S5.SS2.p1.6.m6.1.1.1.1.1.2.cmml" xref="S5.SS2.p1.6.m6.1.1.1.1.1.2">ğœ”</ci><ci id="S5.SS2.p1.6.m6.1.1.1.1.1.3.cmml" xref="S5.SS2.p1.6.m6.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p1.6.m6.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math>, are transmitted to the central server. After that, the central server aggregates these local model updates (gradients) received from <math id="S5.SS2.p1.7.m7.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S5.SS2.p1.7.m7.1a"><mi id="S5.SS2.p1.7.m7.1.1" xref="S5.SS2.p1.7.m7.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S5.SS2.p1.7.m7.1b"><ci id="S5.SS2.p1.7.m7.1.1.cmml" xref="S5.SS2.p1.7.m7.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS2.p1.7.m7.1c">k</annotation></semantics></math> clients and adjusts its global model, often employing an aggregation technique such as FedSGDÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib62" title="" class="ltx_ref">62</a>]</cite>. This iterative process continues until satisfying predetermined stopping criteria, such as reaching a specified number of iterations or achieving a desired level of accuracy.</p>
</div>
</section>
<section id="S5.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS3.5.1.1" class="ltx_text">V-C</span> </span><span id="S5.SS3.6.2" class="ltx_text ltx_font_italic">Attack Method on Medical Images</span>
</h3>

<figure id="S5.F3" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S5.F3.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2409.18907/assets/x3.png" id="S5.F3.sf1.g1" class="ltx_graphics ltx_img_landscape" width="276" height="97" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(a) </span>Attack</figcaption>
</figure>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<figure id="S5.F3.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="/html/2409.18907/assets/x4.png" id="S5.F3.sf2.g1" class="ltx_graphics ltx_img_landscape" width="252" height="96" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">(b) </span>Defense</figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>Overview of Attack Method and Defense Mechanism</figcaption>
</figure>
<div id="S5.SS3.p1" class="ltx_para">
<p id="S5.SS3.p1.5" class="ltx_p">In Figure <a href="#S5.F3" title="Figure 3 â€£ V-C Attack Method on Medical Images â€£ V Methodology â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>(a), we illustrate how an adversary could intercept the local gradient update and reconstruct patientsâ€™ sensitive medical information. During <span id="S5.SS3.p1.5.1" class="ltx_text ltx_font_italic">step 3</span>, when the local model update (gradients) <math id="S5.SS3.p1.1.m1.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S5.SS3.p1.1.m1.1a"><mrow id="S5.SS3.p1.1.m1.1.1" xref="S5.SS3.p1.1.m1.1.1.cmml"><mrow id="S5.SS3.p1.1.m1.1.1.3" xref="S5.SS3.p1.1.m1.1.1.3.cmml"><mo rspace="0.167em" id="S5.SS3.p1.1.m1.1.1.3.1" xref="S5.SS3.p1.1.m1.1.1.3.1.cmml">âˆ‡</mo><msub id="S5.SS3.p1.1.m1.1.1.3.2" xref="S5.SS3.p1.1.m1.1.1.3.2.cmml"><mi id="S5.SS3.p1.1.m1.1.1.3.2.2" xref="S5.SS3.p1.1.m1.1.1.3.2.2.cmml">L</mi><mi id="S5.SS3.p1.1.m1.1.1.3.2.3" xref="S5.SS3.p1.1.m1.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S5.SS3.p1.1.m1.1.1.2" xref="S5.SS3.p1.1.m1.1.1.2.cmml">â€‹</mo><mrow id="S5.SS3.p1.1.m1.1.1.1.1" xref="S5.SS3.p1.1.m1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS3.p1.1.m1.1.1.1.1.2" xref="S5.SS3.p1.1.m1.1.1.1.1.1.cmml">(</mo><msub id="S5.SS3.p1.1.m1.1.1.1.1.1" xref="S5.SS3.p1.1.m1.1.1.1.1.1.cmml"><mi id="S5.SS3.p1.1.m1.1.1.1.1.1.2" xref="S5.SS3.p1.1.m1.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S5.SS3.p1.1.m1.1.1.1.1.1.3" xref="S5.SS3.p1.1.m1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.SS3.p1.1.m1.1.1.1.1.3" xref="S5.SS3.p1.1.m1.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.SS3.p1.1.m1.1b"><apply id="S5.SS3.p1.1.m1.1.1.cmml" xref="S5.SS3.p1.1.m1.1.1"><times id="S5.SS3.p1.1.m1.1.1.2.cmml" xref="S5.SS3.p1.1.m1.1.1.2"></times><apply id="S5.SS3.p1.1.m1.1.1.3.cmml" xref="S5.SS3.p1.1.m1.1.1.3"><ci id="S5.SS3.p1.1.m1.1.1.3.1.cmml" xref="S5.SS3.p1.1.m1.1.1.3.1">âˆ‡</ci><apply id="S5.SS3.p1.1.m1.1.1.3.2.cmml" xref="S5.SS3.p1.1.m1.1.1.3.2"><csymbol cd="ambiguous" id="S5.SS3.p1.1.m1.1.1.3.2.1.cmml" xref="S5.SS3.p1.1.m1.1.1.3.2">subscript</csymbol><ci id="S5.SS3.p1.1.m1.1.1.3.2.2.cmml" xref="S5.SS3.p1.1.m1.1.1.3.2.2">ğ¿</ci><ci id="S5.SS3.p1.1.m1.1.1.3.2.3.cmml" xref="S5.SS3.p1.1.m1.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S5.SS3.p1.1.m1.1.1.1.1.1.cmml" xref="S5.SS3.p1.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS3.p1.1.m1.1.1.1.1.1.1.cmml" xref="S5.SS3.p1.1.m1.1.1.1.1">subscript</csymbol><ci id="S5.SS3.p1.1.m1.1.1.1.1.1.2.cmml" xref="S5.SS3.p1.1.m1.1.1.1.1.1.2">ğœ”</ci><ci id="S5.SS3.p1.1.m1.1.1.1.1.1.3.cmml" xref="S5.SS3.p1.1.m1.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS3.p1.1.m1.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math> is transmitted to the central server, an attacker <math id="S5.SS3.p1.2.m2.1" class="ltx_Math" alttext="a_{i}" display="inline"><semantics id="S5.SS3.p1.2.m2.1a"><msub id="S5.SS3.p1.2.m2.1.1" xref="S5.SS3.p1.2.m2.1.1.cmml"><mi id="S5.SS3.p1.2.m2.1.1.2" xref="S5.SS3.p1.2.m2.1.1.2.cmml">a</mi><mi id="S5.SS3.p1.2.m2.1.1.3" xref="S5.SS3.p1.2.m2.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="S5.SS3.p1.2.m2.1b"><apply id="S5.SS3.p1.2.m2.1.1.cmml" xref="S5.SS3.p1.2.m2.1.1"><csymbol cd="ambiguous" id="S5.SS3.p1.2.m2.1.1.1.cmml" xref="S5.SS3.p1.2.m2.1.1">subscript</csymbol><ci id="S5.SS3.p1.2.m2.1.1.2.cmml" xref="S5.SS3.p1.2.m2.1.1.2">ğ‘</ci><ci id="S5.SS3.p1.2.m2.1.1.3.cmml" xref="S5.SS3.p1.2.m2.1.1.3">ğ‘–</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS3.p1.2.m2.1c">a_{i}</annotation></semantics></math> could intercept this transmission and obtain the local model update for the respective client <math id="S5.SS3.p1.3.m3.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S5.SS3.p1.3.m3.1a"><mi id="S5.SS3.p1.3.m3.1.1" xref="S5.SS3.p1.3.m3.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S5.SS3.p1.3.m3.1b"><ci id="S5.SS3.p1.3.m3.1.1.cmml" xref="S5.SS3.p1.3.m3.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS3.p1.3.m3.1c">k</annotation></semantics></math>. Subsequently, the attacker could scrutinize the periodic local model updates to execute privacy attacks, as documented in prior works such asÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>, <a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>, potentially leading to the reconstruction of client <math id="S5.SS3.p1.4.m4.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S5.SS3.p1.4.m4.1a"><mi id="S5.SS3.p1.4.m4.1.1" xref="S5.SS3.p1.4.m4.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S5.SS3.p1.4.m4.1b"><ci id="S5.SS3.p1.4.m4.1.1.cmml" xref="S5.SS3.p1.4.m4.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS3.p1.4.m4.1c">k</annotation></semantics></math>â€™s private data. Typically, the privacy attack method starts by initiating dummy data and labels (e.g., <math id="S5.SS3.p1.5.m5.2" class="ltx_Math" alttext="x^{\prime},y^{\prime}" display="inline"><semantics id="S5.SS3.p1.5.m5.2a"><mrow id="S5.SS3.p1.5.m5.2.2.2" xref="S5.SS3.p1.5.m5.2.2.3.cmml"><msup id="S5.SS3.p1.5.m5.1.1.1.1" xref="S5.SS3.p1.5.m5.1.1.1.1.cmml"><mi id="S5.SS3.p1.5.m5.1.1.1.1.2" xref="S5.SS3.p1.5.m5.1.1.1.1.2.cmml">x</mi><mo id="S5.SS3.p1.5.m5.1.1.1.1.3" xref="S5.SS3.p1.5.m5.1.1.1.1.3.cmml">â€²</mo></msup><mo id="S5.SS3.p1.5.m5.2.2.2.3" xref="S5.SS3.p1.5.m5.2.2.3.cmml">,</mo><msup id="S5.SS3.p1.5.m5.2.2.2.2" xref="S5.SS3.p1.5.m5.2.2.2.2.cmml"><mi id="S5.SS3.p1.5.m5.2.2.2.2.2" xref="S5.SS3.p1.5.m5.2.2.2.2.2.cmml">y</mi><mo id="S5.SS3.p1.5.m5.2.2.2.2.3" xref="S5.SS3.p1.5.m5.2.2.2.2.3.cmml">â€²</mo></msup></mrow><annotation-xml encoding="MathML-Content" id="S5.SS3.p1.5.m5.2b"><list id="S5.SS3.p1.5.m5.2.2.3.cmml" xref="S5.SS3.p1.5.m5.2.2.2"><apply id="S5.SS3.p1.5.m5.1.1.1.1.cmml" xref="S5.SS3.p1.5.m5.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS3.p1.5.m5.1.1.1.1.1.cmml" xref="S5.SS3.p1.5.m5.1.1.1.1">superscript</csymbol><ci id="S5.SS3.p1.5.m5.1.1.1.1.2.cmml" xref="S5.SS3.p1.5.m5.1.1.1.1.2">ğ‘¥</ci><ci id="S5.SS3.p1.5.m5.1.1.1.1.3.cmml" xref="S5.SS3.p1.5.m5.1.1.1.1.3">â€²</ci></apply><apply id="S5.SS3.p1.5.m5.2.2.2.2.cmml" xref="S5.SS3.p1.5.m5.2.2.2.2"><csymbol cd="ambiguous" id="S5.SS3.p1.5.m5.2.2.2.2.1.cmml" xref="S5.SS3.p1.5.m5.2.2.2.2">superscript</csymbol><ci id="S5.SS3.p1.5.m5.2.2.2.2.2.cmml" xref="S5.SS3.p1.5.m5.2.2.2.2.2">ğ‘¦</ci><ci id="S5.SS3.p1.5.m5.2.2.2.2.3.cmml" xref="S5.SS3.p1.5.m5.2.2.2.2.3">â€²</ci></apply></list></annotation-xml><annotation encoding="application/x-tex" id="S5.SS3.p1.5.m5.2c">x^{\prime},y^{\prime}</annotation></semantics></math>) of the same size as the private training data. Then these â€œdummy dataâ€ are fed into the models and get â€œdummy gradientsâ€.</p>
</div>
<div id="S5.SS3.p2" class="ltx_para">
<table id="S5.E1" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S5.E1.m1.3" class="ltx_Math" alttext="\nabla\omega^{\prime}=\frac{\partial L(F(x^{\prime},\omega),y^{\prime})}{\partial\omega}" display="block"><semantics id="S5.E1.m1.3a"><mrow id="S5.E1.m1.3.4" xref="S5.E1.m1.3.4.cmml"><mrow id="S5.E1.m1.3.4.2" xref="S5.E1.m1.3.4.2.cmml"><mo rspace="0.167em" id="S5.E1.m1.3.4.2.1" xref="S5.E1.m1.3.4.2.1.cmml">âˆ‡</mo><msup id="S5.E1.m1.3.4.2.2" xref="S5.E1.m1.3.4.2.2.cmml"><mi id="S5.E1.m1.3.4.2.2.2" xref="S5.E1.m1.3.4.2.2.2.cmml">Ï‰</mi><mo id="S5.E1.m1.3.4.2.2.3" xref="S5.E1.m1.3.4.2.2.3.cmml">â€²</mo></msup></mrow><mo id="S5.E1.m1.3.4.1" xref="S5.E1.m1.3.4.1.cmml">=</mo><mfrac id="S5.E1.m1.3.3" xref="S5.E1.m1.3.3.cmml"><mrow id="S5.E1.m1.3.3.3" xref="S5.E1.m1.3.3.3.cmml"><mo rspace="0em" id="S5.E1.m1.3.3.3.4" xref="S5.E1.m1.3.3.3.4.cmml">âˆ‚</mo><mrow id="S5.E1.m1.3.3.3.3" xref="S5.E1.m1.3.3.3.3.cmml"><mi id="S5.E1.m1.3.3.3.3.4" xref="S5.E1.m1.3.3.3.3.4.cmml">L</mi><mo lspace="0em" rspace="0em" id="S5.E1.m1.3.3.3.3.3" xref="S5.E1.m1.3.3.3.3.3.cmml">â€‹</mo><mrow id="S5.E1.m1.3.3.3.3.2.2" xref="S5.E1.m1.3.3.3.3.2.3.cmml"><mo stretchy="false" id="S5.E1.m1.3.3.3.3.2.2.3" xref="S5.E1.m1.3.3.3.3.2.3.cmml">(</mo><mrow id="S5.E1.m1.2.2.2.2.1.1.1" xref="S5.E1.m1.2.2.2.2.1.1.1.cmml"><mi id="S5.E1.m1.2.2.2.2.1.1.1.3" xref="S5.E1.m1.2.2.2.2.1.1.1.3.cmml">F</mi><mo lspace="0em" rspace="0em" id="S5.E1.m1.2.2.2.2.1.1.1.2" xref="S5.E1.m1.2.2.2.2.1.1.1.2.cmml">â€‹</mo><mrow id="S5.E1.m1.2.2.2.2.1.1.1.1.1" xref="S5.E1.m1.2.2.2.2.1.1.1.1.2.cmml"><mo stretchy="false" id="S5.E1.m1.2.2.2.2.1.1.1.1.1.2" xref="S5.E1.m1.2.2.2.2.1.1.1.1.2.cmml">(</mo><msup id="S5.E1.m1.2.2.2.2.1.1.1.1.1.1" xref="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.cmml"><mi id="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.2" xref="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.2.cmml">x</mi><mo id="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.3" xref="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.3.cmml">â€²</mo></msup><mo id="S5.E1.m1.2.2.2.2.1.1.1.1.1.3" xref="S5.E1.m1.2.2.2.2.1.1.1.1.2.cmml">,</mo><mi id="S5.E1.m1.1.1.1.1" xref="S5.E1.m1.1.1.1.1.cmml">Ï‰</mi><mo stretchy="false" id="S5.E1.m1.2.2.2.2.1.1.1.1.1.4" xref="S5.E1.m1.2.2.2.2.1.1.1.1.2.cmml">)</mo></mrow></mrow><mo id="S5.E1.m1.3.3.3.3.2.2.4" xref="S5.E1.m1.3.3.3.3.2.3.cmml">,</mo><msup id="S5.E1.m1.3.3.3.3.2.2.2" xref="S5.E1.m1.3.3.3.3.2.2.2.cmml"><mi id="S5.E1.m1.3.3.3.3.2.2.2.2" xref="S5.E1.m1.3.3.3.3.2.2.2.2.cmml">y</mi><mo id="S5.E1.m1.3.3.3.3.2.2.2.3" xref="S5.E1.m1.3.3.3.3.2.2.2.3.cmml">â€²</mo></msup><mo stretchy="false" id="S5.E1.m1.3.3.3.3.2.2.5" xref="S5.E1.m1.3.3.3.3.2.3.cmml">)</mo></mrow></mrow></mrow><mrow id="S5.E1.m1.3.3.5" xref="S5.E1.m1.3.3.5.cmml"><mo rspace="0em" id="S5.E1.m1.3.3.5.1" xref="S5.E1.m1.3.3.5.1.cmml">âˆ‚</mo><mi id="S5.E1.m1.3.3.5.2" xref="S5.E1.m1.3.3.5.2.cmml">Ï‰</mi></mrow></mfrac></mrow><annotation-xml encoding="MathML-Content" id="S5.E1.m1.3b"><apply id="S5.E1.m1.3.4.cmml" xref="S5.E1.m1.3.4"><eq id="S5.E1.m1.3.4.1.cmml" xref="S5.E1.m1.3.4.1"></eq><apply id="S5.E1.m1.3.4.2.cmml" xref="S5.E1.m1.3.4.2"><ci id="S5.E1.m1.3.4.2.1.cmml" xref="S5.E1.m1.3.4.2.1">âˆ‡</ci><apply id="S5.E1.m1.3.4.2.2.cmml" xref="S5.E1.m1.3.4.2.2"><csymbol cd="ambiguous" id="S5.E1.m1.3.4.2.2.1.cmml" xref="S5.E1.m1.3.4.2.2">superscript</csymbol><ci id="S5.E1.m1.3.4.2.2.2.cmml" xref="S5.E1.m1.3.4.2.2.2">ğœ”</ci><ci id="S5.E1.m1.3.4.2.2.3.cmml" xref="S5.E1.m1.3.4.2.2.3">â€²</ci></apply></apply><apply id="S5.E1.m1.3.3.cmml" xref="S5.E1.m1.3.3"><divide id="S5.E1.m1.3.3.4.cmml" xref="S5.E1.m1.3.3"></divide><apply id="S5.E1.m1.3.3.3.cmml" xref="S5.E1.m1.3.3.3"><partialdiff id="S5.E1.m1.3.3.3.4.cmml" xref="S5.E1.m1.3.3.3.4"></partialdiff><apply id="S5.E1.m1.3.3.3.3.cmml" xref="S5.E1.m1.3.3.3.3"><times id="S5.E1.m1.3.3.3.3.3.cmml" xref="S5.E1.m1.3.3.3.3.3"></times><ci id="S5.E1.m1.3.3.3.3.4.cmml" xref="S5.E1.m1.3.3.3.3.4">ğ¿</ci><interval closure="open" id="S5.E1.m1.3.3.3.3.2.3.cmml" xref="S5.E1.m1.3.3.3.3.2.2"><apply id="S5.E1.m1.2.2.2.2.1.1.1.cmml" xref="S5.E1.m1.2.2.2.2.1.1.1"><times id="S5.E1.m1.2.2.2.2.1.1.1.2.cmml" xref="S5.E1.m1.2.2.2.2.1.1.1.2"></times><ci id="S5.E1.m1.2.2.2.2.1.1.1.3.cmml" xref="S5.E1.m1.2.2.2.2.1.1.1.3">ğ¹</ci><interval closure="open" id="S5.E1.m1.2.2.2.2.1.1.1.1.2.cmml" xref="S5.E1.m1.2.2.2.2.1.1.1.1.1"><apply id="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.cmml" xref="S5.E1.m1.2.2.2.2.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.1.cmml" xref="S5.E1.m1.2.2.2.2.1.1.1.1.1.1">superscript</csymbol><ci id="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.2.cmml" xref="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.2">ğ‘¥</ci><ci id="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.3.cmml" xref="S5.E1.m1.2.2.2.2.1.1.1.1.1.1.3">â€²</ci></apply><ci id="S5.E1.m1.1.1.1.1.cmml" xref="S5.E1.m1.1.1.1.1">ğœ”</ci></interval></apply><apply id="S5.E1.m1.3.3.3.3.2.2.2.cmml" xref="S5.E1.m1.3.3.3.3.2.2.2"><csymbol cd="ambiguous" id="S5.E1.m1.3.3.3.3.2.2.2.1.cmml" xref="S5.E1.m1.3.3.3.3.2.2.2">superscript</csymbol><ci id="S5.E1.m1.3.3.3.3.2.2.2.2.cmml" xref="S5.E1.m1.3.3.3.3.2.2.2.2">ğ‘¦</ci><ci id="S5.E1.m1.3.3.3.3.2.2.2.3.cmml" xref="S5.E1.m1.3.3.3.3.2.2.2.3">â€²</ci></apply></interval></apply></apply><apply id="S5.E1.m1.3.3.5.cmml" xref="S5.E1.m1.3.3.5"><partialdiff id="S5.E1.m1.3.3.5.1.cmml" xref="S5.E1.m1.3.3.5.1"></partialdiff><ci id="S5.E1.m1.3.3.5.2.cmml" xref="S5.E1.m1.3.3.5.2">ğœ”</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.E1.m1.3c">\nabla\omega^{\prime}=\frac{\partial L(F(x^{\prime},\omega),y^{\prime})}{\partial\omega}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(1)</span></td>
</tr></tbody>
</table>
<p id="S5.SS3.p2.1" class="ltx_p">Optimizing the dummy gradients close to the original gradients on the private data also makes the dummy data close to the real
private data. Given gradients at a certain step, we obtain the training
data by minimizing the loss function as follows.</p>
<table id="S5.E2" class="ltx_equationgroup ltx_eqn_table">
<tbody>
<tr id="S5.E2X" class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_td ltx_align_right ltx_eqn_cell"><math id="S5.E2X.2.1.1.m1.2" class="ltx_Math" alttext="\displaystyle x^{\prime},y^{\prime}" display="inline"><semantics id="S5.E2X.2.1.1.m1.2a"><mrow id="S5.E2X.2.1.1.m1.2.2.2" xref="S5.E2X.2.1.1.m1.2.2.3.cmml"><msup id="S5.E2X.2.1.1.m1.1.1.1.1" xref="S5.E2X.2.1.1.m1.1.1.1.1.cmml"><mi id="S5.E2X.2.1.1.m1.1.1.1.1.2" xref="S5.E2X.2.1.1.m1.1.1.1.1.2.cmml">x</mi><mo id="S5.E2X.2.1.1.m1.1.1.1.1.3" xref="S5.E2X.2.1.1.m1.1.1.1.1.3.cmml">â€²</mo></msup><mo id="S5.E2X.2.1.1.m1.2.2.2.3" xref="S5.E2X.2.1.1.m1.2.2.3.cmml">,</mo><msup id="S5.E2X.2.1.1.m1.2.2.2.2" xref="S5.E2X.2.1.1.m1.2.2.2.2.cmml"><mi id="S5.E2X.2.1.1.m1.2.2.2.2.2" xref="S5.E2X.2.1.1.m1.2.2.2.2.2.cmml">y</mi><mo id="S5.E2X.2.1.1.m1.2.2.2.2.3" xref="S5.E2X.2.1.1.m1.2.2.2.2.3.cmml">â€²</mo></msup></mrow><annotation-xml encoding="MathML-Content" id="S5.E2X.2.1.1.m1.2b"><list id="S5.E2X.2.1.1.m1.2.2.3.cmml" xref="S5.E2X.2.1.1.m1.2.2.2"><apply id="S5.E2X.2.1.1.m1.1.1.1.1.cmml" xref="S5.E2X.2.1.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="S5.E2X.2.1.1.m1.1.1.1.1.1.cmml" xref="S5.E2X.2.1.1.m1.1.1.1.1">superscript</csymbol><ci id="S5.E2X.2.1.1.m1.1.1.1.1.2.cmml" xref="S5.E2X.2.1.1.m1.1.1.1.1.2">ğ‘¥</ci><ci id="S5.E2X.2.1.1.m1.1.1.1.1.3.cmml" xref="S5.E2X.2.1.1.m1.1.1.1.1.3">â€²</ci></apply><apply id="S5.E2X.2.1.1.m1.2.2.2.2.cmml" xref="S5.E2X.2.1.1.m1.2.2.2.2"><csymbol cd="ambiguous" id="S5.E2X.2.1.1.m1.2.2.2.2.1.cmml" xref="S5.E2X.2.1.1.m1.2.2.2.2">superscript</csymbol><ci id="S5.E2X.2.1.1.m1.2.2.2.2.2.cmml" xref="S5.E2X.2.1.1.m1.2.2.2.2.2">ğ‘¦</ci><ci id="S5.E2X.2.1.1.m1.2.2.2.2.3.cmml" xref="S5.E2X.2.1.1.m1.2.2.2.2.3">â€²</ci></apply></list></annotation-xml><annotation encoding="application/x-tex" id="S5.E2X.2.1.1.m1.2c">\displaystyle x^{\prime},y^{\prime}</annotation></semantics></math></td>
<td class="ltx_td ltx_align_left ltx_eqn_cell"><math id="S5.E2X.3.2.2.m1.3" class="ltx_Math" alttext="\displaystyle=\arg\min_{x^{\prime},y^{\prime}}||\nabla\omega^{\prime}-\nabla\omega||^{2}" display="inline"><semantics id="S5.E2X.3.2.2.m1.3a"><mrow id="S5.E2X.3.2.2.m1.3.3" xref="S5.E2X.3.2.2.m1.3.3.cmml"><mi id="S5.E2X.3.2.2.m1.3.3.3" xref="S5.E2X.3.2.2.m1.3.3.3.cmml"></mi><mo id="S5.E2X.3.2.2.m1.3.3.2" xref="S5.E2X.3.2.2.m1.3.3.2.cmml">=</mo><mrow id="S5.E2X.3.2.2.m1.3.3.1" xref="S5.E2X.3.2.2.m1.3.3.1.cmml"><mrow id="S5.E2X.3.2.2.m1.3.3.1.3" xref="S5.E2X.3.2.2.m1.3.3.1.3.cmml"><mi id="S5.E2X.3.2.2.m1.3.3.1.3.1" xref="S5.E2X.3.2.2.m1.3.3.1.3.1.cmml">arg</mi><mo lspace="0.167em" id="S5.E2X.3.2.2.m1.3.3.1.3a" xref="S5.E2X.3.2.2.m1.3.3.1.3.cmml">â¡</mo><munder id="S5.E2X.3.2.2.m1.3.3.1.3.2" xref="S5.E2X.3.2.2.m1.3.3.1.3.2.cmml"><mi id="S5.E2X.3.2.2.m1.3.3.1.3.2.2" xref="S5.E2X.3.2.2.m1.3.3.1.3.2.2.cmml">min</mi><mrow id="S5.E2X.3.2.2.m1.2.2.2.2" xref="S5.E2X.3.2.2.m1.2.2.2.3.cmml"><msup id="S5.E2X.3.2.2.m1.1.1.1.1.1" xref="S5.E2X.3.2.2.m1.1.1.1.1.1.cmml"><mi id="S5.E2X.3.2.2.m1.1.1.1.1.1.2" xref="S5.E2X.3.2.2.m1.1.1.1.1.1.2.cmml">x</mi><mo id="S5.E2X.3.2.2.m1.1.1.1.1.1.3" xref="S5.E2X.3.2.2.m1.1.1.1.1.1.3.cmml">â€²</mo></msup><mo id="S5.E2X.3.2.2.m1.2.2.2.2.3" xref="S5.E2X.3.2.2.m1.2.2.2.3.cmml">,</mo><msup id="S5.E2X.3.2.2.m1.2.2.2.2.2" xref="S5.E2X.3.2.2.m1.2.2.2.2.2.cmml"><mi id="S5.E2X.3.2.2.m1.2.2.2.2.2.2" xref="S5.E2X.3.2.2.m1.2.2.2.2.2.2.cmml">y</mi><mo id="S5.E2X.3.2.2.m1.2.2.2.2.2.3" xref="S5.E2X.3.2.2.m1.2.2.2.2.2.3.cmml">â€²</mo></msup></mrow></munder></mrow><mo lspace="0em" rspace="0em" id="S5.E2X.3.2.2.m1.3.3.1.2" xref="S5.E2X.3.2.2.m1.3.3.1.2.cmml">â€‹</mo><msup id="S5.E2X.3.2.2.m1.3.3.1.1" xref="S5.E2X.3.2.2.m1.3.3.1.1.cmml"><mrow id="S5.E2X.3.2.2.m1.3.3.1.1.1.1" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.2.cmml"><mo stretchy="false" id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.2" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.2.1.cmml">â€–</mo><mrow id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.cmml"><mrow id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.cmml"><mo rspace="0.167em" id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.1" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.1.cmml">âˆ‡</mo><msup id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.cmml"><mi id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.2" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.2.cmml">Ï‰</mi><mo id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.3" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.3.cmml">â€²</mo></msup></mrow><mo id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.1" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.1.cmml">âˆ’</mo><mrow id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.cmml"><mo rspace="0.167em" id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.1" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.1.cmml">âˆ‡</mo><mi id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.2" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.2.cmml">Ï‰</mi></mrow></mrow><mo stretchy="false" id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.3" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.2.1.cmml">â€–</mo></mrow><mn id="S5.E2X.3.2.2.m1.3.3.1.1.3" xref="S5.E2X.3.2.2.m1.3.3.1.1.3.cmml">2</mn></msup></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.E2X.3.2.2.m1.3b"><apply id="S5.E2X.3.2.2.m1.3.3.cmml" xref="S5.E2X.3.2.2.m1.3.3"><eq id="S5.E2X.3.2.2.m1.3.3.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.2"></eq><csymbol cd="latexml" id="S5.E2X.3.2.2.m1.3.3.3.cmml" xref="S5.E2X.3.2.2.m1.3.3.3">absent</csymbol><apply id="S5.E2X.3.2.2.m1.3.3.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1"><times id="S5.E2X.3.2.2.m1.3.3.1.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.2"></times><apply id="S5.E2X.3.2.2.m1.3.3.1.3.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.3"><arg id="S5.E2X.3.2.2.m1.3.3.1.3.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.3.1"></arg><apply id="S5.E2X.3.2.2.m1.3.3.1.3.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.3.2"><csymbol cd="ambiguous" id="S5.E2X.3.2.2.m1.3.3.1.3.2.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.3.2">subscript</csymbol><min id="S5.E2X.3.2.2.m1.3.3.1.3.2.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.3.2.2"></min><list id="S5.E2X.3.2.2.m1.2.2.2.3.cmml" xref="S5.E2X.3.2.2.m1.2.2.2.2"><apply id="S5.E2X.3.2.2.m1.1.1.1.1.1.cmml" xref="S5.E2X.3.2.2.m1.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.E2X.3.2.2.m1.1.1.1.1.1.1.cmml" xref="S5.E2X.3.2.2.m1.1.1.1.1.1">superscript</csymbol><ci id="S5.E2X.3.2.2.m1.1.1.1.1.1.2.cmml" xref="S5.E2X.3.2.2.m1.1.1.1.1.1.2">ğ‘¥</ci><ci id="S5.E2X.3.2.2.m1.1.1.1.1.1.3.cmml" xref="S5.E2X.3.2.2.m1.1.1.1.1.1.3">â€²</ci></apply><apply id="S5.E2X.3.2.2.m1.2.2.2.2.2.cmml" xref="S5.E2X.3.2.2.m1.2.2.2.2.2"><csymbol cd="ambiguous" id="S5.E2X.3.2.2.m1.2.2.2.2.2.1.cmml" xref="S5.E2X.3.2.2.m1.2.2.2.2.2">superscript</csymbol><ci id="S5.E2X.3.2.2.m1.2.2.2.2.2.2.cmml" xref="S5.E2X.3.2.2.m1.2.2.2.2.2.2">ğ‘¦</ci><ci id="S5.E2X.3.2.2.m1.2.2.2.2.2.3.cmml" xref="S5.E2X.3.2.2.m1.2.2.2.2.2.3">â€²</ci></apply></list></apply></apply><apply id="S5.E2X.3.2.2.m1.3.3.1.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1"><csymbol cd="ambiguous" id="S5.E2X.3.2.2.m1.3.3.1.1.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1">superscript</csymbol><apply id="S5.E2X.3.2.2.m1.3.3.1.1.1.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1"><csymbol cd="latexml" id="S5.E2X.3.2.2.m1.3.3.1.1.1.2.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.2">norm</csymbol><apply id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1"><minus id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.1"></minus><apply id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2"><ci id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.1">âˆ‡</ci><apply id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2"><csymbol cd="ambiguous" id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2">superscript</csymbol><ci id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.2">ğœ”</ci><ci id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.3.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.2.2.3">â€²</ci></apply></apply><apply id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3"><ci id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.1.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.1">âˆ‡</ci><ci id="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.2.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.1.1.1.3.2">ğœ”</ci></apply></apply></apply><cn type="integer" id="S5.E2X.3.2.2.m1.3.3.1.1.3.cmml" xref="S5.E2X.3.2.2.m1.3.3.1.1.3">2</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.E2X.3.2.2.m1.3c">\displaystyle=\arg\min_{x^{\prime},y^{\prime}}||\nabla\omega^{\prime}-\nabla\omega||^{2}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="2" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equationgroup ltx_align_right">(2)</span></td>
</tr>
<tr id="S5.E2Xa" class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_td ltx_eqn_cell"></td>
<td class="ltx_td ltx_align_left ltx_eqn_cell"><math id="S5.E2Xa.2.1.1.m1.6" class="ltx_Math" alttext="\displaystyle=\arg\min_{x^{\prime},y^{\prime}}||\frac{\partial L(F(x^{\prime},\omega),y^{\prime})}{\partial\omega}-\nabla\omega||^{2}" display="inline"><semantics id="S5.E2Xa.2.1.1.m1.6a"><mrow id="S5.E2Xa.2.1.1.m1.6.6" xref="S5.E2Xa.2.1.1.m1.6.6.cmml"><mi id="S5.E2Xa.2.1.1.m1.6.6.3" xref="S5.E2Xa.2.1.1.m1.6.6.3.cmml"></mi><mo id="S5.E2Xa.2.1.1.m1.6.6.2" xref="S5.E2Xa.2.1.1.m1.6.6.2.cmml">=</mo><mrow id="S5.E2Xa.2.1.1.m1.6.6.1" xref="S5.E2Xa.2.1.1.m1.6.6.1.cmml"><mrow id="S5.E2Xa.2.1.1.m1.6.6.1.3" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.cmml"><mi id="S5.E2Xa.2.1.1.m1.6.6.1.3.1" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.1.cmml">arg</mi><mo lspace="0.167em" id="S5.E2Xa.2.1.1.m1.6.6.1.3a" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.cmml">â¡</mo><munder id="S5.E2Xa.2.1.1.m1.6.6.1.3.2" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.2.cmml"><mi id="S5.E2Xa.2.1.1.m1.6.6.1.3.2.2" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.2.2.cmml">min</mi><mrow id="S5.E2Xa.2.1.1.m1.2.2.2.2" xref="S5.E2Xa.2.1.1.m1.2.2.2.3.cmml"><msup id="S5.E2Xa.2.1.1.m1.1.1.1.1.1" xref="S5.E2Xa.2.1.1.m1.1.1.1.1.1.cmml"><mi id="S5.E2Xa.2.1.1.m1.1.1.1.1.1.2" xref="S5.E2Xa.2.1.1.m1.1.1.1.1.1.2.cmml">x</mi><mo id="S5.E2Xa.2.1.1.m1.1.1.1.1.1.3" xref="S5.E2Xa.2.1.1.m1.1.1.1.1.1.3.cmml">â€²</mo></msup><mo id="S5.E2Xa.2.1.1.m1.2.2.2.2.3" xref="S5.E2Xa.2.1.1.m1.2.2.2.3.cmml">,</mo><msup id="S5.E2Xa.2.1.1.m1.2.2.2.2.2" xref="S5.E2Xa.2.1.1.m1.2.2.2.2.2.cmml"><mi id="S5.E2Xa.2.1.1.m1.2.2.2.2.2.2" xref="S5.E2Xa.2.1.1.m1.2.2.2.2.2.2.cmml">y</mi><mo id="S5.E2Xa.2.1.1.m1.2.2.2.2.2.3" xref="S5.E2Xa.2.1.1.m1.2.2.2.2.2.3.cmml">â€²</mo></msup></mrow></munder></mrow><mo lspace="0em" rspace="0em" id="S5.E2Xa.2.1.1.m1.6.6.1.2" xref="S5.E2Xa.2.1.1.m1.6.6.1.2.cmml">â€‹</mo><msup id="S5.E2Xa.2.1.1.m1.6.6.1.1" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.cmml"><mrow id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.2.cmml"><mo stretchy="false" id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.2" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.2.1.cmml">â€–</mo><mrow id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.cmml"><mstyle displaystyle="true" id="S5.E2Xa.2.1.1.m1.5.5" xref="S5.E2Xa.2.1.1.m1.5.5.cmml"><mfrac id="S5.E2Xa.2.1.1.m1.5.5a" xref="S5.E2Xa.2.1.1.m1.5.5.cmml"><mrow id="S5.E2Xa.2.1.1.m1.5.5.3" xref="S5.E2Xa.2.1.1.m1.5.5.3.cmml"><mo rspace="0em" id="S5.E2Xa.2.1.1.m1.5.5.3.4" xref="S5.E2Xa.2.1.1.m1.5.5.3.4.cmml">âˆ‚</mo><mrow id="S5.E2Xa.2.1.1.m1.5.5.3.3" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.cmml"><mi id="S5.E2Xa.2.1.1.m1.5.5.3.3.4" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.4.cmml">L</mi><mo lspace="0em" rspace="0em" id="S5.E2Xa.2.1.1.m1.5.5.3.3.3" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.3.cmml">â€‹</mo><mrow id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.3.cmml"><mo stretchy="false" id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.3" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.3.cmml">(</mo><mrow id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.cmml"><mi id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.3" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.3.cmml">F</mi><mo lspace="0em" rspace="0em" id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.2" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.2.cmml">â€‹</mo><mrow id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.2.cmml"><mo stretchy="false" id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.2" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.2.cmml">(</mo><msup id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.cmml"><mi id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.2" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.2.cmml">x</mi><mo id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.3" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.3.cmml">â€²</mo></msup><mo id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.3" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.2.cmml">,</mo><mi id="S5.E2Xa.2.1.1.m1.3.3.1.1" xref="S5.E2Xa.2.1.1.m1.3.3.1.1.cmml">Ï‰</mi><mo stretchy="false" id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.4" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.2.cmml">)</mo></mrow></mrow><mo id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.4" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.3.cmml">,</mo><msup id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.cmml"><mi id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.2" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.2.cmml">y</mi><mo id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.3" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.3.cmml">â€²</mo></msup><mo stretchy="false" id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.5" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.3.cmml">)</mo></mrow></mrow></mrow><mrow id="S5.E2Xa.2.1.1.m1.5.5.5" xref="S5.E2Xa.2.1.1.m1.5.5.5.cmml"><mo rspace="0em" id="S5.E2Xa.2.1.1.m1.5.5.5.1" xref="S5.E2Xa.2.1.1.m1.5.5.5.1.cmml">âˆ‚</mo><mi id="S5.E2Xa.2.1.1.m1.5.5.5.2" xref="S5.E2Xa.2.1.1.m1.5.5.5.2.cmml">Ï‰</mi></mrow></mfrac></mstyle><mo id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.1" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.1.cmml">âˆ’</mo><mrow id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.cmml"><mo rspace="0.167em" id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.1" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.1.cmml">âˆ‡</mo><mi id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.2" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.2.cmml">Ï‰</mi></mrow></mrow><mo stretchy="false" id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.3" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.2.1.cmml">â€–</mo></mrow><mn id="S5.E2Xa.2.1.1.m1.6.6.1.1.3" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.3.cmml">2</mn></msup></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.E2Xa.2.1.1.m1.6b"><apply id="S5.E2Xa.2.1.1.m1.6.6.cmml" xref="S5.E2Xa.2.1.1.m1.6.6"><eq id="S5.E2Xa.2.1.1.m1.6.6.2.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.2"></eq><csymbol cd="latexml" id="S5.E2Xa.2.1.1.m1.6.6.3.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.3">absent</csymbol><apply id="S5.E2Xa.2.1.1.m1.6.6.1.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1"><times id="S5.E2Xa.2.1.1.m1.6.6.1.2.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.2"></times><apply id="S5.E2Xa.2.1.1.m1.6.6.1.3.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.3"><arg id="S5.E2Xa.2.1.1.m1.6.6.1.3.1.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.1"></arg><apply id="S5.E2Xa.2.1.1.m1.6.6.1.3.2.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.2"><csymbol cd="ambiguous" id="S5.E2Xa.2.1.1.m1.6.6.1.3.2.1.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.2">subscript</csymbol><min id="S5.E2Xa.2.1.1.m1.6.6.1.3.2.2.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.3.2.2"></min><list id="S5.E2Xa.2.1.1.m1.2.2.2.3.cmml" xref="S5.E2Xa.2.1.1.m1.2.2.2.2"><apply id="S5.E2Xa.2.1.1.m1.1.1.1.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.E2Xa.2.1.1.m1.1.1.1.1.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.1.1.1.1.1">superscript</csymbol><ci id="S5.E2Xa.2.1.1.m1.1.1.1.1.1.2.cmml" xref="S5.E2Xa.2.1.1.m1.1.1.1.1.1.2">ğ‘¥</ci><ci id="S5.E2Xa.2.1.1.m1.1.1.1.1.1.3.cmml" xref="S5.E2Xa.2.1.1.m1.1.1.1.1.1.3">â€²</ci></apply><apply id="S5.E2Xa.2.1.1.m1.2.2.2.2.2.cmml" xref="S5.E2Xa.2.1.1.m1.2.2.2.2.2"><csymbol cd="ambiguous" id="S5.E2Xa.2.1.1.m1.2.2.2.2.2.1.cmml" xref="S5.E2Xa.2.1.1.m1.2.2.2.2.2">superscript</csymbol><ci id="S5.E2Xa.2.1.1.m1.2.2.2.2.2.2.cmml" xref="S5.E2Xa.2.1.1.m1.2.2.2.2.2.2">ğ‘¦</ci><ci id="S5.E2Xa.2.1.1.m1.2.2.2.2.2.3.cmml" xref="S5.E2Xa.2.1.1.m1.2.2.2.2.2.3">â€²</ci></apply></list></apply></apply><apply id="S5.E2Xa.2.1.1.m1.6.6.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1"><csymbol cd="ambiguous" id="S5.E2Xa.2.1.1.m1.6.6.1.1.2.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1">superscript</csymbol><apply id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.2.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1"><csymbol cd="latexml" id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.2.1.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.2">norm</csymbol><apply id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1"><minus id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.1"></minus><apply id="S5.E2Xa.2.1.1.m1.5.5.cmml" xref="S5.E2Xa.2.1.1.m1.5.5"><divide id="S5.E2Xa.2.1.1.m1.5.5.4.cmml" xref="S5.E2Xa.2.1.1.m1.5.5"></divide><apply id="S5.E2Xa.2.1.1.m1.5.5.3.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3"><partialdiff id="S5.E2Xa.2.1.1.m1.5.5.3.4.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.4"></partialdiff><apply id="S5.E2Xa.2.1.1.m1.5.5.3.3.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.3"><times id="S5.E2Xa.2.1.1.m1.5.5.3.3.3.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.3"></times><ci id="S5.E2Xa.2.1.1.m1.5.5.3.3.4.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.4">ğ¿</ci><interval closure="open" id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.3.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2"><apply id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1"><times id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.2.cmml" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.2"></times><ci id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.3.cmml" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.3">ğ¹</ci><interval closure="open" id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.2.cmml" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1"><apply id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1">superscript</csymbol><ci id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.2.cmml" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.2">ğ‘¥</ci><ci id="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.3.cmml" xref="S5.E2Xa.2.1.1.m1.4.4.2.2.1.1.1.1.1.1.3">â€²</ci></apply><ci id="S5.E2Xa.2.1.1.m1.3.3.1.1.cmml" xref="S5.E2Xa.2.1.1.m1.3.3.1.1">ğœ”</ci></interval></apply><apply id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2"><csymbol cd="ambiguous" id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.1.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2">superscript</csymbol><ci id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.2.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.2">ğ‘¦</ci><ci id="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.3.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.3.3.2.2.2.3">â€²</ci></apply></interval></apply></apply><apply id="S5.E2Xa.2.1.1.m1.5.5.5.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.5"><partialdiff id="S5.E2Xa.2.1.1.m1.5.5.5.1.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.5.1"></partialdiff><ci id="S5.E2Xa.2.1.1.m1.5.5.5.2.cmml" xref="S5.E2Xa.2.1.1.m1.5.5.5.2">ğœ”</ci></apply></apply><apply id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2"><ci id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.1.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.1">âˆ‡</ci><ci id="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.2.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.1.1.1.2.2">ğœ”</ci></apply></apply></apply><cn type="integer" id="S5.E2Xa.2.1.1.m1.6.6.1.1.3.cmml" xref="S5.E2Xa.2.1.1.m1.6.6.1.1.3">2</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.E2Xa.2.1.1.m1.6c">\displaystyle=\arg\min_{x^{\prime},y^{\prime}}||\frac{\partial L(F(x^{\prime},\omega),y^{\prime})}{\partial\omega}-\nabla\omega||^{2}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr>
</tbody>
</table>
</div>
<div id="S5.SS3.p3" class="ltx_para">
<p id="S5.SS3.p3.1" class="ltx_p">The attacker performs the privacy attacks by taking the dummy data, dummy label, and the local model updates (gradients obtained from intercepting <span id="S5.SS3.p3.1.1" class="ltx_text ltx_font_italic">step 3</span>) of the client from local training. Additionally, the attacker utilizes the shared global model so that it can iteratively update the dummy data and labels to reconstruct the clientâ€™s private training data. This iterative process involves updating the dummy data and labels to minimize the disparity between the local gradients computed on the private data and the dummy gradients computed on the dummy data and labels, denoted as <math id="S5.SS3.p3.1.m1.1" class="ltx_Math" alttext="||\nabla\omega^{\prime}-\nabla\omega||^{2}" display="inline"><semantics id="S5.SS3.p3.1.m1.1a"><msup id="S5.SS3.p3.1.m1.1.1" xref="S5.SS3.p3.1.m1.1.1.cmml"><mrow id="S5.SS3.p3.1.m1.1.1.1.1" xref="S5.SS3.p3.1.m1.1.1.1.2.cmml"><mo stretchy="false" id="S5.SS3.p3.1.m1.1.1.1.1.2" xref="S5.SS3.p3.1.m1.1.1.1.2.1.cmml">â€–</mo><mrow id="S5.SS3.p3.1.m1.1.1.1.1.1" xref="S5.SS3.p3.1.m1.1.1.1.1.1.cmml"><mrow id="S5.SS3.p3.1.m1.1.1.1.1.1.2" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.cmml"><mo rspace="0.167em" id="S5.SS3.p3.1.m1.1.1.1.1.1.2.1" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.1.cmml">âˆ‡</mo><msup id="S5.SS3.p3.1.m1.1.1.1.1.1.2.2" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.cmml"><mi id="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.2" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.2.cmml">Ï‰</mi><mo id="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.3" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.3.cmml">â€²</mo></msup></mrow><mo id="S5.SS3.p3.1.m1.1.1.1.1.1.1" xref="S5.SS3.p3.1.m1.1.1.1.1.1.1.cmml">âˆ’</mo><mrow id="S5.SS3.p3.1.m1.1.1.1.1.1.3" xref="S5.SS3.p3.1.m1.1.1.1.1.1.3.cmml"><mo rspace="0.167em" id="S5.SS3.p3.1.m1.1.1.1.1.1.3.1" xref="S5.SS3.p3.1.m1.1.1.1.1.1.3.1.cmml">âˆ‡</mo><mi id="S5.SS3.p3.1.m1.1.1.1.1.1.3.2" xref="S5.SS3.p3.1.m1.1.1.1.1.1.3.2.cmml">Ï‰</mi></mrow></mrow><mo stretchy="false" id="S5.SS3.p3.1.m1.1.1.1.1.3" xref="S5.SS3.p3.1.m1.1.1.1.2.1.cmml">â€–</mo></mrow><mn id="S5.SS3.p3.1.m1.1.1.3" xref="S5.SS3.p3.1.m1.1.1.3.cmml">2</mn></msup><annotation-xml encoding="MathML-Content" id="S5.SS3.p3.1.m1.1b"><apply id="S5.SS3.p3.1.m1.1.1.cmml" xref="S5.SS3.p3.1.m1.1.1"><csymbol cd="ambiguous" id="S5.SS3.p3.1.m1.1.1.2.cmml" xref="S5.SS3.p3.1.m1.1.1">superscript</csymbol><apply id="S5.SS3.p3.1.m1.1.1.1.2.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1"><csymbol cd="latexml" id="S5.SS3.p3.1.m1.1.1.1.2.1.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.2">norm</csymbol><apply id="S5.SS3.p3.1.m1.1.1.1.1.1.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1"><minus id="S5.SS3.p3.1.m1.1.1.1.1.1.1.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.1"></minus><apply id="S5.SS3.p3.1.m1.1.1.1.1.1.2.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2"><ci id="S5.SS3.p3.1.m1.1.1.1.1.1.2.1.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.1">âˆ‡</ci><apply id="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.2"><csymbol cd="ambiguous" id="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.1.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.2">superscript</csymbol><ci id="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.2.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.2">ğœ”</ci><ci id="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.3.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.2.2.3">â€²</ci></apply></apply><apply id="S5.SS3.p3.1.m1.1.1.1.1.1.3.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.3"><ci id="S5.SS3.p3.1.m1.1.1.1.1.1.3.1.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.3.1">âˆ‡</ci><ci id="S5.SS3.p3.1.m1.1.1.1.1.1.3.2.cmml" xref="S5.SS3.p3.1.m1.1.1.1.1.1.3.2">ğœ”</ci></apply></apply></apply><cn type="integer" id="S5.SS3.p3.1.m1.1.1.3.cmml" xref="S5.SS3.p3.1.m1.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS3.p3.1.m1.1c">||\nabla\omega^{\prime}-\nabla\omega||^{2}</annotation></semantics></math>. This process, facilitated by the shared global model, gradually aligns the dummy data with the private training data, exacerbating privacy breaches.</p>
</div>
</section>
<section id="S5.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S5.SS4.5.1.1" class="ltx_text">V-D</span> </span><span id="S5.SS4.6.2" class="ltx_text ltx_font_italic">Defense Mechanism for Medical Images</span>
</h3>

<div id="S5.SS4.p1" class="ltx_para">
<p id="S5.SS4.p1.2" class="ltx_p">In the vanilla FL context, various methods exist to thwart privacy leakage attacks, which could potentially safeguard medical data privacy. One such method is gradient perturbationÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>, which entails injecting a controlled amount of Laplacian or Gaussian noise into the local model update
<math id="S5.SS4.p1.1.m1.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S5.SS4.p1.1.m1.1a"><mrow id="S5.SS4.p1.1.m1.1.1" xref="S5.SS4.p1.1.m1.1.1.cmml"><mrow id="S5.SS4.p1.1.m1.1.1.3" xref="S5.SS4.p1.1.m1.1.1.3.cmml"><mo rspace="0.167em" id="S5.SS4.p1.1.m1.1.1.3.1" xref="S5.SS4.p1.1.m1.1.1.3.1.cmml">âˆ‡</mo><msub id="S5.SS4.p1.1.m1.1.1.3.2" xref="S5.SS4.p1.1.m1.1.1.3.2.cmml"><mi id="S5.SS4.p1.1.m1.1.1.3.2.2" xref="S5.SS4.p1.1.m1.1.1.3.2.2.cmml">L</mi><mi id="S5.SS4.p1.1.m1.1.1.3.2.3" xref="S5.SS4.p1.1.m1.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S5.SS4.p1.1.m1.1.1.2" xref="S5.SS4.p1.1.m1.1.1.2.cmml">â€‹</mo><mrow id="S5.SS4.p1.1.m1.1.1.1.1" xref="S5.SS4.p1.1.m1.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS4.p1.1.m1.1.1.1.1.2" xref="S5.SS4.p1.1.m1.1.1.1.1.1.cmml">(</mo><msub id="S5.SS4.p1.1.m1.1.1.1.1.1" xref="S5.SS4.p1.1.m1.1.1.1.1.1.cmml"><mi id="S5.SS4.p1.1.m1.1.1.1.1.1.2" xref="S5.SS4.p1.1.m1.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S5.SS4.p1.1.m1.1.1.1.1.1.3" xref="S5.SS4.p1.1.m1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.SS4.p1.1.m1.1.1.1.1.3" xref="S5.SS4.p1.1.m1.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.SS4.p1.1.m1.1b"><apply id="S5.SS4.p1.1.m1.1.1.cmml" xref="S5.SS4.p1.1.m1.1.1"><times id="S5.SS4.p1.1.m1.1.1.2.cmml" xref="S5.SS4.p1.1.m1.1.1.2"></times><apply id="S5.SS4.p1.1.m1.1.1.3.cmml" xref="S5.SS4.p1.1.m1.1.1.3"><ci id="S5.SS4.p1.1.m1.1.1.3.1.cmml" xref="S5.SS4.p1.1.m1.1.1.3.1">âˆ‡</ci><apply id="S5.SS4.p1.1.m1.1.1.3.2.cmml" xref="S5.SS4.p1.1.m1.1.1.3.2"><csymbol cd="ambiguous" id="S5.SS4.p1.1.m1.1.1.3.2.1.cmml" xref="S5.SS4.p1.1.m1.1.1.3.2">subscript</csymbol><ci id="S5.SS4.p1.1.m1.1.1.3.2.2.cmml" xref="S5.SS4.p1.1.m1.1.1.3.2.2">ğ¿</ci><ci id="S5.SS4.p1.1.m1.1.1.3.2.3.cmml" xref="S5.SS4.p1.1.m1.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S5.SS4.p1.1.m1.1.1.1.1.1.cmml" xref="S5.SS4.p1.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS4.p1.1.m1.1.1.1.1.1.1.cmml" xref="S5.SS4.p1.1.m1.1.1.1.1">subscript</csymbol><ci id="S5.SS4.p1.1.m1.1.1.1.1.1.2.cmml" xref="S5.SS4.p1.1.m1.1.1.1.1.1.2">ğœ”</ci><ci id="S5.SS4.p1.1.m1.1.1.1.1.1.3.cmml" xref="S5.SS4.p1.1.m1.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS4.p1.1.m1.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math> during <span id="S5.SS4.p1.2.1" class="ltx_text ltx_font_italic">step 3</span> (refer to FigureÂ <a href="#S5.F3" title="Figure 3 â€£ V-C Attack Method on Medical Images â€£ V Methodology â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>(b)). By introducing noise to <math id="S5.SS4.p1.2.m2.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S5.SS4.p1.2.m2.1a"><mrow id="S5.SS4.p1.2.m2.1.1" xref="S5.SS4.p1.2.m2.1.1.cmml"><mrow id="S5.SS4.p1.2.m2.1.1.3" xref="S5.SS4.p1.2.m2.1.1.3.cmml"><mo rspace="0.167em" id="S5.SS4.p1.2.m2.1.1.3.1" xref="S5.SS4.p1.2.m2.1.1.3.1.cmml">âˆ‡</mo><msub id="S5.SS4.p1.2.m2.1.1.3.2" xref="S5.SS4.p1.2.m2.1.1.3.2.cmml"><mi id="S5.SS4.p1.2.m2.1.1.3.2.2" xref="S5.SS4.p1.2.m2.1.1.3.2.2.cmml">L</mi><mi id="S5.SS4.p1.2.m2.1.1.3.2.3" xref="S5.SS4.p1.2.m2.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S5.SS4.p1.2.m2.1.1.2" xref="S5.SS4.p1.2.m2.1.1.2.cmml">â€‹</mo><mrow id="S5.SS4.p1.2.m2.1.1.1.1" xref="S5.SS4.p1.2.m2.1.1.1.1.1.cmml"><mo stretchy="false" id="S5.SS4.p1.2.m2.1.1.1.1.2" xref="S5.SS4.p1.2.m2.1.1.1.1.1.cmml">(</mo><msub id="S5.SS4.p1.2.m2.1.1.1.1.1" xref="S5.SS4.p1.2.m2.1.1.1.1.1.cmml"><mi id="S5.SS4.p1.2.m2.1.1.1.1.1.2" xref="S5.SS4.p1.2.m2.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S5.SS4.p1.2.m2.1.1.1.1.1.3" xref="S5.SS4.p1.2.m2.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S5.SS4.p1.2.m2.1.1.1.1.3" xref="S5.SS4.p1.2.m2.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S5.SS4.p1.2.m2.1b"><apply id="S5.SS4.p1.2.m2.1.1.cmml" xref="S5.SS4.p1.2.m2.1.1"><times id="S5.SS4.p1.2.m2.1.1.2.cmml" xref="S5.SS4.p1.2.m2.1.1.2"></times><apply id="S5.SS4.p1.2.m2.1.1.3.cmml" xref="S5.SS4.p1.2.m2.1.1.3"><ci id="S5.SS4.p1.2.m2.1.1.3.1.cmml" xref="S5.SS4.p1.2.m2.1.1.3.1">âˆ‡</ci><apply id="S5.SS4.p1.2.m2.1.1.3.2.cmml" xref="S5.SS4.p1.2.m2.1.1.3.2"><csymbol cd="ambiguous" id="S5.SS4.p1.2.m2.1.1.3.2.1.cmml" xref="S5.SS4.p1.2.m2.1.1.3.2">subscript</csymbol><ci id="S5.SS4.p1.2.m2.1.1.3.2.2.cmml" xref="S5.SS4.p1.2.m2.1.1.3.2.2">ğ¿</ci><ci id="S5.SS4.p1.2.m2.1.1.3.2.3.cmml" xref="S5.SS4.p1.2.m2.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S5.SS4.p1.2.m2.1.1.1.1.1.cmml" xref="S5.SS4.p1.2.m2.1.1.1.1"><csymbol cd="ambiguous" id="S5.SS4.p1.2.m2.1.1.1.1.1.1.cmml" xref="S5.SS4.p1.2.m2.1.1.1.1">subscript</csymbol><ci id="S5.SS4.p1.2.m2.1.1.1.1.1.2.cmml" xref="S5.SS4.p1.2.m2.1.1.1.1.1.2">ğœ”</ci><ci id="S5.SS4.p1.2.m2.1.1.1.1.1.3.cmml" xref="S5.SS4.p1.2.m2.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.SS4.p1.2.m2.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math>, this approach introduces uncertainty into local updates, obscuring details and hindering adversaries from accurately reconstructing private data, such as medical images.</p>
</div>
<div id="S5.SS4.p2" class="ltx_para">
<p id="S5.SS4.p2.1" class="ltx_p">DP can be leveraged to provide theoretical guarantees in gradient perturbation to allow the sharing of confidential data and safeguard the privacy of the individuals whose data is being utilizedÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib63" title="" class="ltx_ref">63</a>]</cite>. DP-based mechanisms introduce a controlled amount of noise to the private data in a manner that preserves the statistical characteristics and obscures the actual values of individual data points. By doing so, DP prevents malicious entities from identifying specific data points, thereby ensuring the privacy of the individuals concerned. As represented in FigureÂ <a href="#S5.F3" title="Figure 3 â€£ V-C Attack Method on Medical Images â€£ V Methodology â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>(b), when we add a controlled amount of noise to the gradient at <span id="S5.SS4.p2.1.1" class="ltx_text ltx_font_italic">step 3</span> of the process, it can prevent accurate reconstruction of private training data.
Also, gradient compressionÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite> can be utilized to defend against privacy leakage attacks in FL.
Secure multi-party computation is another category of defense mechanisms where multiple parties perform computations without revealing their sensitive data to each otherÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib59" title="" class="ltx_ref">59</a>]</cite>.</p>
</div>
</section>
</section>
<section id="S6" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">VI </span><span id="S6.1.1" class="ltx_text ltx_font_smallcaps">Experimental Analysis</span>
</h2>

<div id="S6.p1" class="ltx_para">
<p id="S6.p1.1" class="ltx_p">The experiments are conducted on a GPU server with an NVIDIA RTX A6000 with 48 GB memory.</p>
</div>
<figure id="S6.T1" class="ltx_table">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">TABLE I: </span>Dataset Information and Properties</figcaption>
<div id="S6.T1.1" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:273.7pt;height:103.1pt;vertical-align:-0.0pt;"><span class="ltx_transformed_inner" style="transform:translate(-51.9pt,19.6pt) scale(0.725,0.725) ;">
<table id="S6.T1.1.1" class="ltx_tabular ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S6.T1.1.1.1.1" class="ltx_tr">
<th id="S6.T1.1.1.1.1.1" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_l ltx_border_r ltx_border_t"><span id="S6.T1.1.1.1.1.1.1" class="ltx_text ltx_font_bold" style="font-size:90%;">Dataset</span></th>
<th id="S6.T1.1.1.1.1.2" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_r ltx_border_t"><span id="S6.T1.1.1.1.1.2.1" class="ltx_text ltx_font_bold" style="font-size:90%;"># of Samples</span></th>
<th id="S6.T1.1.1.1.1.3" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_r ltx_border_t"><span id="S6.T1.1.1.1.1.3.1" class="ltx_text ltx_font_bold" style="font-size:90%;"># of Classes and Names</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S6.T1.1.1.2.1" class="ltx_tr">
<td id="S6.T1.1.1.2.1.1" class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t"><span id="S6.T1.1.1.2.1.1.1" class="ltx_text" style="font-size:90%;">Melanoma Skin Cancer</span></td>
<td id="S6.T1.1.1.2.1.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T1.1.1.2.1.2.1" class="ltx_text" style="font-size:90%;">10000</span></td>
<td id="S6.T1.1.1.2.1.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T1.1.1.2.1.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T1.1.1.2.1.3.1.1" class="ltx_tr">
<td id="S6.T1.1.1.2.1.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T1.1.1.2.1.3.1.1.1.1" class="ltx_text" style="font-size:90%;">2</span></td>
</tr>
<tr id="S6.T1.1.1.2.1.3.1.2" class="ltx_tr">
<td id="S6.T1.1.1.2.1.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T1.1.1.2.1.3.1.2.1.1" class="ltx_text" style="font-size:90%;">(Benign and malignant)</span></td>
</tr>
</table>
</td>
</tr>
<tr id="S6.T1.1.1.3.2" class="ltx_tr">
<td id="S6.T1.1.1.3.2.1" class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t"><span id="S6.T1.1.1.3.2.1.1" class="ltx_text" style="font-size:90%;">COVID-19 X-ray</span></td>
<td id="S6.T1.1.1.3.2.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T1.1.1.3.2.2.1" class="ltx_text" style="font-size:90%;">317</span></td>
<td id="S6.T1.1.1.3.2.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T1.1.1.3.2.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T1.1.1.3.2.3.1.1" class="ltx_tr">
<td id="S6.T1.1.1.3.2.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T1.1.1.3.2.3.1.1.1.1" class="ltx_text" style="font-size:90%;">3</span></td>
</tr>
<tr id="S6.T1.1.1.3.2.3.1.2" class="ltx_tr">
<td id="S6.T1.1.1.3.2.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T1.1.1.3.2.3.1.2.1.1" class="ltx_text" style="font-size:90%;">(COVID, Normal, Viral Pneumonia)</span></td>
</tr>
</table>
</td>
</tr>
<tr id="S6.T1.1.1.4.3" class="ltx_tr">
<td id="S6.T1.1.1.4.3.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_l ltx_border_r ltx_border_t"><span id="S6.T1.1.1.4.3.1.1" class="ltx_text" style="font-size:90%;">Brain Tumor MRI Images</span></td>
<td id="S6.T1.1.1.4.3.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S6.T1.1.1.4.3.2.1" class="ltx_text" style="font-size:90%;">7022</span></td>
<td id="S6.T1.1.1.4.3.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">
<table id="S6.T1.1.1.4.3.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T1.1.1.4.3.3.1.1" class="ltx_tr">
<td id="S6.T1.1.1.4.3.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T1.1.1.4.3.3.1.1.1.1" class="ltx_text" style="font-size:90%;">4</span></td>
</tr>
<tr id="S6.T1.1.1.4.3.3.1.2" class="ltx_tr">
<td id="S6.T1.1.1.4.3.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T1.1.1.4.3.3.1.2.1.1" class="ltx_text" style="font-size:90%;">(Pituitary, Glioma, Meningioma, No tumor)</span></td>
</tr>
</table>
</td>
</tr>
</tbody>
</table>
</span></div>
</figure>
<section id="S6.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS1.5.1.1" class="ltx_text">VI-A</span> </span><span id="S6.SS1.6.2" class="ltx_text ltx_font_italic">Dataset Properties and Preprocessing</span>
</h3>

<div id="S6.SS1.p1" class="ltx_para">
<p id="S6.SS1.p1.2" class="ltx_p">We conduct experiments employing various attack methods on three representative medical image datasets: Melanoma Skin CancerÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib53" title="" class="ltx_ref">53</a>]</cite>, COVID-19 X-rayÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib54" title="" class="ltx_ref">54</a>]</cite>, and Brain Tumor MRI ImagesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib55" title="" class="ltx_ref">55</a>]</cite>. These datasets contain 10,000, 317, and 7,022 samples respectively, distributed across two, three, and four classes as outlined in TableÂ <a href="#S6.T1" title="TABLE I â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">I</span></a>. Each dataset comprises images of different sizes, for instance, the Melanoma Skin Cancer dataset contains images of size 300<math id="S6.SS1.p1.1.m1.1" class="ltx_Math" alttext="\times" display="inline"><semantics id="S6.SS1.p1.1.m1.1a"><mo id="S6.SS1.p1.1.m1.1.1" xref="S6.SS1.p1.1.m1.1.1.cmml">Ã—</mo><annotation-xml encoding="MathML-Content" id="S6.SS1.p1.1.m1.1b"><times id="S6.SS1.p1.1.m1.1.1.cmml" xref="S6.SS1.p1.1.m1.1.1"></times></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p1.1.m1.1c">\times</annotation></semantics></math>300. Also, COVID-19 X-ray images and Brain Tumor MRI Images exhibit varying shapes. In our experiment, all images were resized to 32<math id="S6.SS1.p1.2.m2.1" class="ltx_Math" alttext="\times" display="inline"><semantics id="S6.SS1.p1.2.m2.1a"><mo id="S6.SS1.p1.2.m2.1.1" xref="S6.SS1.p1.2.m2.1.1.cmml">Ã—</mo><annotation-xml encoding="MathML-Content" id="S6.SS1.p1.2.m2.1b"><times id="S6.SS1.p1.2.m2.1.1.cmml" xref="S6.SS1.p1.2.m2.1.1"></times></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p1.2.m2.1c">\times</annotation></semantics></math>32 and we performed data normalization using mean and standard deviation during our preprocessing phase. We employ a 4-layer CNN architecture, incorporating a fully connected layer to process input images with three channels, as outlined in CPLÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>, DLGÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>, and iDLGÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite> respectively on the medical image datasets. Additionally, we conduct GradInv attacksÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite> on all three datasets to compare the attack performance with CPL, DLG, and iDLG respectively. The GradInv <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite> attack is designed to reconstruct private training images from both ResNet-18Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> which is pre-trained on ImageNetÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib64" title="" class="ltx_ref">64</a>]</cite> and its untrained version.</p>
</div>
<figure id="S6.F4" class="ltx_figure"><img src="/html/2409.18907/assets/x5.png" id="S6.F4.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="163" height="112" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 4: </span>Samples of attack failure cases of all datasets.</figcaption>
</figure>
</section>
<section id="S6.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS2.5.1.1" class="ltx_text">VI-B</span> </span><span id="S6.SS2.6.2" class="ltx_text ltx_font_italic">Attack Method Configuration</span>
</h3>

<div id="S6.SS2.p1" class="ltx_para">
<p id="S6.SS2.p1.7" class="ltx_p">CPLÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>, DLGÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>, and iDLGÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite> follow a similar approach in order to reconstruct clientâ€™s private data. The attacker intercepts the local model update <math id="S6.SS2.p1.1.m1.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S6.SS2.p1.1.m1.1a"><mrow id="S6.SS2.p1.1.m1.1.1" xref="S6.SS2.p1.1.m1.1.1.cmml"><mrow id="S6.SS2.p1.1.m1.1.1.3" xref="S6.SS2.p1.1.m1.1.1.3.cmml"><mo rspace="0.167em" id="S6.SS2.p1.1.m1.1.1.3.1" xref="S6.SS2.p1.1.m1.1.1.3.1.cmml">âˆ‡</mo><msub id="S6.SS2.p1.1.m1.1.1.3.2" xref="S6.SS2.p1.1.m1.1.1.3.2.cmml"><mi id="S6.SS2.p1.1.m1.1.1.3.2.2" xref="S6.SS2.p1.1.m1.1.1.3.2.2.cmml">L</mi><mi id="S6.SS2.p1.1.m1.1.1.3.2.3" xref="S6.SS2.p1.1.m1.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S6.SS2.p1.1.m1.1.1.2" xref="S6.SS2.p1.1.m1.1.1.2.cmml">â€‹</mo><mrow id="S6.SS2.p1.1.m1.1.1.1.1" xref="S6.SS2.p1.1.m1.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.SS2.p1.1.m1.1.1.1.1.2" xref="S6.SS2.p1.1.m1.1.1.1.1.1.cmml">(</mo><msub id="S6.SS2.p1.1.m1.1.1.1.1.1" xref="S6.SS2.p1.1.m1.1.1.1.1.1.cmml"><mi id="S6.SS2.p1.1.m1.1.1.1.1.1.2" xref="S6.SS2.p1.1.m1.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S6.SS2.p1.1.m1.1.1.1.1.1.3" xref="S6.SS2.p1.1.m1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S6.SS2.p1.1.m1.1.1.1.1.3" xref="S6.SS2.p1.1.m1.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p1.1.m1.1b"><apply id="S6.SS2.p1.1.m1.1.1.cmml" xref="S6.SS2.p1.1.m1.1.1"><times id="S6.SS2.p1.1.m1.1.1.2.cmml" xref="S6.SS2.p1.1.m1.1.1.2"></times><apply id="S6.SS2.p1.1.m1.1.1.3.cmml" xref="S6.SS2.p1.1.m1.1.1.3"><ci id="S6.SS2.p1.1.m1.1.1.3.1.cmml" xref="S6.SS2.p1.1.m1.1.1.3.1">âˆ‡</ci><apply id="S6.SS2.p1.1.m1.1.1.3.2.cmml" xref="S6.SS2.p1.1.m1.1.1.3.2"><csymbol cd="ambiguous" id="S6.SS2.p1.1.m1.1.1.3.2.1.cmml" xref="S6.SS2.p1.1.m1.1.1.3.2">subscript</csymbol><ci id="S6.SS2.p1.1.m1.1.1.3.2.2.cmml" xref="S6.SS2.p1.1.m1.1.1.3.2.2">ğ¿</ci><ci id="S6.SS2.p1.1.m1.1.1.3.2.3.cmml" xref="S6.SS2.p1.1.m1.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S6.SS2.p1.1.m1.1.1.1.1.1.cmml" xref="S6.SS2.p1.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="S6.SS2.p1.1.m1.1.1.1.1.1.1.cmml" xref="S6.SS2.p1.1.m1.1.1.1.1">subscript</csymbol><ci id="S6.SS2.p1.1.m1.1.1.1.1.1.2.cmml" xref="S6.SS2.p1.1.m1.1.1.1.1.1.2">ğœ”</ci><ci id="S6.SS2.p1.1.m1.1.1.1.1.1.3.cmml" xref="S6.SS2.p1.1.m1.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p1.1.m1.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math> corresponding to a client <math id="S6.SS2.p1.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S6.SS2.p1.2.m2.1a"><mi id="S6.SS2.p1.2.m2.1.1" xref="S6.SS2.p1.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p1.2.m2.1b"><ci id="S6.SS2.p1.2.m2.1.1.cmml" xref="S6.SS2.p1.2.m2.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p1.2.m2.1c">k</annotation></semantics></math> at iteration <math id="S6.SS2.p1.3.m3.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S6.SS2.p1.3.m3.1a"><mi id="S6.SS2.p1.3.m3.1.1" xref="S6.SS2.p1.3.m3.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p1.3.m3.1b"><ci id="S6.SS2.p1.3.m3.1.1.cmml" xref="S6.SS2.p1.3.m3.1.1">ğ‘¡</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p1.3.m3.1c">t</annotation></semantics></math>. They initialize a dummy image of the same dimensions as the training data, along with a dummy label. The dummy image is then iteratively updated to minimize the <math id="S6.SS2.p1.4.m4.1" class="ltx_Math" alttext="L2" display="inline"><semantics id="S6.SS2.p1.4.m4.1a"><mrow id="S6.SS2.p1.4.m4.1.1" xref="S6.SS2.p1.4.m4.1.1.cmml"><mi id="S6.SS2.p1.4.m4.1.1.2" xref="S6.SS2.p1.4.m4.1.1.2.cmml">L</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p1.4.m4.1.1.1" xref="S6.SS2.p1.4.m4.1.1.1.cmml">â€‹</mo><mn id="S6.SS2.p1.4.m4.1.1.3" xref="S6.SS2.p1.4.m4.1.1.3.cmml">2</mn></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p1.4.m4.1b"><apply id="S6.SS2.p1.4.m4.1.1.cmml" xref="S6.SS2.p1.4.m4.1.1"><times id="S6.SS2.p1.4.m4.1.1.1.cmml" xref="S6.SS2.p1.4.m4.1.1.1"></times><ci id="S6.SS2.p1.4.m4.1.1.2.cmml" xref="S6.SS2.p1.4.m4.1.1.2">ğ¿</ci><cn type="integer" id="S6.SS2.p1.4.m4.1.1.3.cmml" xref="S6.SS2.p1.4.m4.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p1.4.m4.1c">L2</annotation></semantics></math> distance between the actual gradient <math id="S6.SS2.p1.5.m5.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S6.SS2.p1.5.m5.1a"><mrow id="S6.SS2.p1.5.m5.1.1" xref="S6.SS2.p1.5.m5.1.1.cmml"><mrow id="S6.SS2.p1.5.m5.1.1.3" xref="S6.SS2.p1.5.m5.1.1.3.cmml"><mo rspace="0.167em" id="S6.SS2.p1.5.m5.1.1.3.1" xref="S6.SS2.p1.5.m5.1.1.3.1.cmml">âˆ‡</mo><msub id="S6.SS2.p1.5.m5.1.1.3.2" xref="S6.SS2.p1.5.m5.1.1.3.2.cmml"><mi id="S6.SS2.p1.5.m5.1.1.3.2.2" xref="S6.SS2.p1.5.m5.1.1.3.2.2.cmml">L</mi><mi id="S6.SS2.p1.5.m5.1.1.3.2.3" xref="S6.SS2.p1.5.m5.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S6.SS2.p1.5.m5.1.1.2" xref="S6.SS2.p1.5.m5.1.1.2.cmml">â€‹</mo><mrow id="S6.SS2.p1.5.m5.1.1.1.1" xref="S6.SS2.p1.5.m5.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.SS2.p1.5.m5.1.1.1.1.2" xref="S6.SS2.p1.5.m5.1.1.1.1.1.cmml">(</mo><msub id="S6.SS2.p1.5.m5.1.1.1.1.1" xref="S6.SS2.p1.5.m5.1.1.1.1.1.cmml"><mi id="S6.SS2.p1.5.m5.1.1.1.1.1.2" xref="S6.SS2.p1.5.m5.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S6.SS2.p1.5.m5.1.1.1.1.1.3" xref="S6.SS2.p1.5.m5.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S6.SS2.p1.5.m5.1.1.1.1.3" xref="S6.SS2.p1.5.m5.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p1.5.m5.1b"><apply id="S6.SS2.p1.5.m5.1.1.cmml" xref="S6.SS2.p1.5.m5.1.1"><times id="S6.SS2.p1.5.m5.1.1.2.cmml" xref="S6.SS2.p1.5.m5.1.1.2"></times><apply id="S6.SS2.p1.5.m5.1.1.3.cmml" xref="S6.SS2.p1.5.m5.1.1.3"><ci id="S6.SS2.p1.5.m5.1.1.3.1.cmml" xref="S6.SS2.p1.5.m5.1.1.3.1">âˆ‡</ci><apply id="S6.SS2.p1.5.m5.1.1.3.2.cmml" xref="S6.SS2.p1.5.m5.1.1.3.2"><csymbol cd="ambiguous" id="S6.SS2.p1.5.m5.1.1.3.2.1.cmml" xref="S6.SS2.p1.5.m5.1.1.3.2">subscript</csymbol><ci id="S6.SS2.p1.5.m5.1.1.3.2.2.cmml" xref="S6.SS2.p1.5.m5.1.1.3.2.2">ğ¿</ci><ci id="S6.SS2.p1.5.m5.1.1.3.2.3.cmml" xref="S6.SS2.p1.5.m5.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S6.SS2.p1.5.m5.1.1.1.1.1.cmml" xref="S6.SS2.p1.5.m5.1.1.1.1"><csymbol cd="ambiguous" id="S6.SS2.p1.5.m5.1.1.1.1.1.1.cmml" xref="S6.SS2.p1.5.m5.1.1.1.1">subscript</csymbol><ci id="S6.SS2.p1.5.m5.1.1.1.1.1.2.cmml" xref="S6.SS2.p1.5.m5.1.1.1.1.1.2">ğœ”</ci><ci id="S6.SS2.p1.5.m5.1.1.1.1.1.3.cmml" xref="S6.SS2.p1.5.m5.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p1.5.m5.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math> computed on the private training data and the dummy gradients computed on the dummy data. In other words, the attack aims to find a dummy data sample that produces gradients as close as possible to the intercepted gradients from the clientâ€™s local update. For executing the privacy attack, we employ the L-BFGS optimization method, as suggested by CPLÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib38" title="" class="ltx_ref">38</a>]</cite>, DLGÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib37" title="" class="ltx_ref">37</a>]</cite>, and iDLGÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite>. The GradInvÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite> attack is basically conducted by leveraging the gradients, <math id="S6.SS2.p1.6.m6.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S6.SS2.p1.6.m6.1a"><mrow id="S6.SS2.p1.6.m6.1.1" xref="S6.SS2.p1.6.m6.1.1.cmml"><mrow id="S6.SS2.p1.6.m6.1.1.3" xref="S6.SS2.p1.6.m6.1.1.3.cmml"><mo rspace="0.167em" id="S6.SS2.p1.6.m6.1.1.3.1" xref="S6.SS2.p1.6.m6.1.1.3.1.cmml">âˆ‡</mo><msub id="S6.SS2.p1.6.m6.1.1.3.2" xref="S6.SS2.p1.6.m6.1.1.3.2.cmml"><mi id="S6.SS2.p1.6.m6.1.1.3.2.2" xref="S6.SS2.p1.6.m6.1.1.3.2.2.cmml">L</mi><mi id="S6.SS2.p1.6.m6.1.1.3.2.3" xref="S6.SS2.p1.6.m6.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S6.SS2.p1.6.m6.1.1.2" xref="S6.SS2.p1.6.m6.1.1.2.cmml">â€‹</mo><mrow id="S6.SS2.p1.6.m6.1.1.1.1" xref="S6.SS2.p1.6.m6.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.SS2.p1.6.m6.1.1.1.1.2" xref="S6.SS2.p1.6.m6.1.1.1.1.1.cmml">(</mo><msub id="S6.SS2.p1.6.m6.1.1.1.1.1" xref="S6.SS2.p1.6.m6.1.1.1.1.1.cmml"><mi id="S6.SS2.p1.6.m6.1.1.1.1.1.2" xref="S6.SS2.p1.6.m6.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S6.SS2.p1.6.m6.1.1.1.1.1.3" xref="S6.SS2.p1.6.m6.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S6.SS2.p1.6.m6.1.1.1.1.3" xref="S6.SS2.p1.6.m6.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p1.6.m6.1b"><apply id="S6.SS2.p1.6.m6.1.1.cmml" xref="S6.SS2.p1.6.m6.1.1"><times id="S6.SS2.p1.6.m6.1.1.2.cmml" xref="S6.SS2.p1.6.m6.1.1.2"></times><apply id="S6.SS2.p1.6.m6.1.1.3.cmml" xref="S6.SS2.p1.6.m6.1.1.3"><ci id="S6.SS2.p1.6.m6.1.1.3.1.cmml" xref="S6.SS2.p1.6.m6.1.1.3.1">âˆ‡</ci><apply id="S6.SS2.p1.6.m6.1.1.3.2.cmml" xref="S6.SS2.p1.6.m6.1.1.3.2"><csymbol cd="ambiguous" id="S6.SS2.p1.6.m6.1.1.3.2.1.cmml" xref="S6.SS2.p1.6.m6.1.1.3.2">subscript</csymbol><ci id="S6.SS2.p1.6.m6.1.1.3.2.2.cmml" xref="S6.SS2.p1.6.m6.1.1.3.2.2">ğ¿</ci><ci id="S6.SS2.p1.6.m6.1.1.3.2.3.cmml" xref="S6.SS2.p1.6.m6.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S6.SS2.p1.6.m6.1.1.1.1.1.cmml" xref="S6.SS2.p1.6.m6.1.1.1.1"><csymbol cd="ambiguous" id="S6.SS2.p1.6.m6.1.1.1.1.1.1.cmml" xref="S6.SS2.p1.6.m6.1.1.1.1">subscript</csymbol><ci id="S6.SS2.p1.6.m6.1.1.1.1.1.2.cmml" xref="S6.SS2.p1.6.m6.1.1.1.1.1.2">ğœ”</ci><ci id="S6.SS2.p1.6.m6.1.1.1.1.1.3.cmml" xref="S6.SS2.p1.6.m6.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p1.6.m6.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math>, of the local training data to reconstruct the original images utilizing a network composed of fully-connected layers. This attack iteratively analyzes the <math id="S6.SS2.p1.7.m7.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S6.SS2.p1.7.m7.1a"><mrow id="S6.SS2.p1.7.m7.1.1" xref="S6.SS2.p1.7.m7.1.1.cmml"><mrow id="S6.SS2.p1.7.m7.1.1.3" xref="S6.SS2.p1.7.m7.1.1.3.cmml"><mo rspace="0.167em" id="S6.SS2.p1.7.m7.1.1.3.1" xref="S6.SS2.p1.7.m7.1.1.3.1.cmml">âˆ‡</mo><msub id="S6.SS2.p1.7.m7.1.1.3.2" xref="S6.SS2.p1.7.m7.1.1.3.2.cmml"><mi id="S6.SS2.p1.7.m7.1.1.3.2.2" xref="S6.SS2.p1.7.m7.1.1.3.2.2.cmml">L</mi><mi id="S6.SS2.p1.7.m7.1.1.3.2.3" xref="S6.SS2.p1.7.m7.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S6.SS2.p1.7.m7.1.1.2" xref="S6.SS2.p1.7.m7.1.1.2.cmml">â€‹</mo><mrow id="S6.SS2.p1.7.m7.1.1.1.1" xref="S6.SS2.p1.7.m7.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.SS2.p1.7.m7.1.1.1.1.2" xref="S6.SS2.p1.7.m7.1.1.1.1.1.cmml">(</mo><msub id="S6.SS2.p1.7.m7.1.1.1.1.1" xref="S6.SS2.p1.7.m7.1.1.1.1.1.cmml"><mi id="S6.SS2.p1.7.m7.1.1.1.1.1.2" xref="S6.SS2.p1.7.m7.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S6.SS2.p1.7.m7.1.1.1.1.1.3" xref="S6.SS2.p1.7.m7.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S6.SS2.p1.7.m7.1.1.1.1.3" xref="S6.SS2.p1.7.m7.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p1.7.m7.1b"><apply id="S6.SS2.p1.7.m7.1.1.cmml" xref="S6.SS2.p1.7.m7.1.1"><times id="S6.SS2.p1.7.m7.1.1.2.cmml" xref="S6.SS2.p1.7.m7.1.1.2"></times><apply id="S6.SS2.p1.7.m7.1.1.3.cmml" xref="S6.SS2.p1.7.m7.1.1.3"><ci id="S6.SS2.p1.7.m7.1.1.3.1.cmml" xref="S6.SS2.p1.7.m7.1.1.3.1">âˆ‡</ci><apply id="S6.SS2.p1.7.m7.1.1.3.2.cmml" xref="S6.SS2.p1.7.m7.1.1.3.2"><csymbol cd="ambiguous" id="S6.SS2.p1.7.m7.1.1.3.2.1.cmml" xref="S6.SS2.p1.7.m7.1.1.3.2">subscript</csymbol><ci id="S6.SS2.p1.7.m7.1.1.3.2.2.cmml" xref="S6.SS2.p1.7.m7.1.1.3.2.2">ğ¿</ci><ci id="S6.SS2.p1.7.m7.1.1.3.2.3.cmml" xref="S6.SS2.p1.7.m7.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S6.SS2.p1.7.m7.1.1.1.1.1.cmml" xref="S6.SS2.p1.7.m7.1.1.1.1"><csymbol cd="ambiguous" id="S6.SS2.p1.7.m7.1.1.1.1.1.1.cmml" xref="S6.SS2.p1.7.m7.1.1.1.1">subscript</csymbol><ci id="S6.SS2.p1.7.m7.1.1.1.1.1.2.cmml" xref="S6.SS2.p1.7.m7.1.1.1.1.1.2">ğœ”</ci><ci id="S6.SS2.p1.7.m7.1.1.1.1.1.3.cmml" xref="S6.SS2.p1.7.m7.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p1.7.m7.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math> under the condition of non-zero gradients, optimizing the angle-based loss function (cosine similarity), to reconstruct the private data. Adam is employed as the optimization algorithm.</p>
</div>
</section>
<section id="S6.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS3.5.1.1" class="ltx_text">VI-C</span> </span><span id="S6.SS3.6.2" class="ltx_text ltx_font_italic">Performance of Attack Methods</span>
</h3>

<div id="S6.SS3.p1" class="ltx_para">
<p id="S6.SS3.p1.1" class="ltx_p">Here, we present the performance comparison of the four attack methods that we investigated in this research (i.e., CPL, DLG, iDLG, and GradInv) on different medical image datasets in TableÂ <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a>. We chose 100 randomly sampled images for all the attack methods for all three datasets. During the ASR computation, if the SSIM between the original private image and the attack-reconstructed image is above or equal to <math id="S6.SS3.p1.1.m1.1" class="ltx_Math" alttext="0.9" display="inline"><semantics id="S6.SS3.p1.1.m1.1a"><mn id="S6.SS3.p1.1.m1.1.1" xref="S6.SS3.p1.1.m1.1.1.cmml">0.9</mn><annotation-xml encoding="MathML-Content" id="S6.SS3.p1.1.m1.1b"><cn type="float" id="S6.SS3.p1.1.m1.1.1.cmml" xref="S6.SS3.p1.1.m1.1.1">0.9</cn></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.p1.1.m1.1c">0.9</annotation></semantics></math>, we consider that case as a successful attack. In our experiment, we found very high MSE values and very low SSIM values between the original training image and the reconstructed one for several attack failure cases in various attack methods. We show some of the samples from the iDLG attack method in Figure <a href="#S6.F4" title="Figure 4 â€£ VI-A Dataset Properties and Preprocessing â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a> along with their corresponding high MSE and low SSIM values. Such high MSE and low SSIM values, i.e., attack failure cases, might cause bias in the attack performance evaluation. Therefore, we considered the average SSIM and MSE values shown in Table <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a>, which has been calculated only for successful attacks as per ASR requirements for all attack methods.</p>
</div>
<figure id="S6.T2" class="ltx_table">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">TABLE II: </span>Performance Comparison of GradInv., CPL, DLG, and iDLG attack methods on all three medical datasets.</figcaption>
<div id="S6.T2.1" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:618.8pt;height:346.6pt;vertical-align:-1.0pt;"><span class="ltx_transformed_inner" style="transform:translate(0.0pt,0.0pt) scale(1.0,1.0) ;">
<table id="S6.T2.1.1" class="ltx_tabular ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr id="S6.T2.1.1.1.1" class="ltx_tr">
<th id="S6.T2.1.1.1.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_l ltx_border_r ltx_border_t">Dataset</th>
<th id="S6.T2.1.1.1.1.2" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_r ltx_border_t">Method</th>
<th id="S6.T2.1.1.1.1.3" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_r ltx_border_t">Model</th>
<th id="S6.T2.1.1.1.1.4" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_r ltx_border_t">
<table id="S6.T2.1.1.1.1.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T2.1.1.1.1.4.1.1" class="ltx_tr">
<td id="S6.T2.1.1.1.1.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Attack Success</td>
</tr>
<tr id="S6.T2.1.1.1.1.4.1.2" class="ltx_tr">
<td id="S6.T2.1.1.1.1.4.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Rate</td>
</tr>
</table>
</th>
<th id="S6.T2.1.1.1.1.5" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_r ltx_border_t">
<table id="S6.T2.1.1.1.1.5.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T2.1.1.1.1.5.1.1" class="ltx_tr">
<td id="S6.T2.1.1.1.1.5.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Avg.</td>
</tr>
<tr id="S6.T2.1.1.1.1.5.1.2" class="ltx_tr">
<td id="S6.T2.1.1.1.1.5.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">SSIM for</td>
</tr>
<tr id="S6.T2.1.1.1.1.5.1.3" class="ltx_tr">
<td id="S6.T2.1.1.1.1.5.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">Successful Attacks</td>
</tr>
</table>
</th>
<th id="S6.T2.1.1.1.1.6" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_r ltx_border_t">
<table id="S6.T2.1.1.1.1.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T2.1.1.1.1.6.1.1" class="ltx_tr">
<td id="S6.T2.1.1.1.1.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Avg.</td>
</tr>
<tr id="S6.T2.1.1.1.1.6.1.2" class="ltx_tr">
<td id="S6.T2.1.1.1.1.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">MSE for</td>
</tr>
<tr id="S6.T2.1.1.1.1.6.1.3" class="ltx_tr">
<td id="S6.T2.1.1.1.1.6.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">Successful Attacks</td>
</tr>
</table>
</th>
<th id="S6.T2.1.1.1.1.7" class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_r ltx_border_t">
<table id="S6.T2.1.1.1.1.7.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T2.1.1.1.1.7.1.1" class="ltx_tr">
<td id="S6.T2.1.1.1.1.7.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Avg. Attack</td>
</tr>
<tr id="S6.T2.1.1.1.1.7.1.2" class="ltx_tr">
<td id="S6.T2.1.1.1.1.7.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Execution Time</td>
</tr>
<tr id="S6.T2.1.1.1.1.7.1.3" class="ltx_tr">
<td id="S6.T2.1.1.1.1.7.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">per image (in seconds)</td>
</tr>
</table>
</th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr id="S6.T2.1.1.2.1" class="ltx_tr">
<th id="S6.T2.1.1.2.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" rowspan="4"><span id="S6.T2.1.1.2.1.1.1" class="ltx_text">
<span id="S6.T2.1.1.2.1.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S6.T2.1.1.2.1.1.1.1.1" class="ltx_tr">
<span id="S6.T2.1.1.2.1.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">Melanoma</span></span>
<span id="S6.T2.1.1.2.1.1.1.1.2" class="ltx_tr">
<span id="S6.T2.1.1.2.1.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">Skin</span></span>
<span id="S6.T2.1.1.2.1.1.1.1.3" class="ltx_tr">
<span id="S6.T2.1.1.2.1.1.1.1.3.1" class="ltx_td ltx_nopad_r ltx_align_left">Cancer</span></span>
<span id="S6.T2.1.1.2.1.1.1.1.4" class="ltx_tr">
<span id="S6.T2.1.1.2.1.1.1.1.4.1" class="ltx_td ltx_nopad_r ltx_align_left">Dataset</span></span>
</span></span></th>
<td id="S6.T2.1.1.2.1.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t" rowspan="2"><span id="S6.T2.1.1.2.1.2.1" class="ltx_text">GradInv.</span></td>
<td id="S6.T2.1.1.2.1.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">ResNet-18 Untrained</td>
<td id="S6.T2.1.1.2.1.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.2.1.4.1" class="ltx_text ltx_font_bold">76%</span></td>
<td id="S6.T2.1.1.2.1.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9473</td>
<td id="S6.T2.1.1.2.1.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.5301</td>
<td id="S6.T2.1.1.2.1.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">5940.94</td>
</tr>
<tr id="S6.T2.1.1.3.2" class="ltx_tr">
<td id="S6.T2.1.1.3.2.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T2.1.1.3.2.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T2.1.1.3.2.1.1.1" class="ltx_tr">
<td id="S6.T2.1.1.3.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">ResNet-18 Trained</td>
</tr>
</table>
</td>
<td id="S6.T2.1.1.3.2.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">72%</td>
<td id="S6.T2.1.1.3.2.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9389</td>
<td id="S6.T2.1.1.3.2.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.5621</td>
<td id="S6.T2.1.1.3.2.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">4880.58</td>
</tr>
<tr id="S6.T2.1.1.4.3" class="ltx_tr">
<td id="S6.T2.1.1.4.3.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CPL</td>
<td id="S6.T2.1.1.4.3.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.4.3.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">49%</td>
<td id="S6.T2.1.1.4.3.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.4.3.4.1" class="ltx_text ltx_font_bold">0.9996</span></td>
<td id="S6.T2.1.1.4.3.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.4.3.5.1" class="ltx_text ltx_font_bold">4.1 * E-7</span></td>
<td id="S6.T2.1.1.4.3.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">50.8471</td>
</tr>
<tr id="S6.T2.1.1.5.4" class="ltx_tr">
<td id="S6.T2.1.1.5.4.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">DLG</td>
<td id="S6.T2.1.1.5.4.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.5.4.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">47%</td>
<td id="S6.T2.1.1.5.4.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9569</td>
<td id="S6.T2.1.1.5.4.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">6.28 * E-3</td>
<td id="S6.T2.1.1.5.4.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">60.1282</td>
</tr>
<tr id="S6.T2.1.1.6.5" class="ltx_tr">
<th id="S6.T2.1.1.6.5.1" class="ltx_td ltx_th ltx_th_row ltx_border_l ltx_border_r"></th>
<td id="S6.T2.1.1.6.5.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">iDLG</td>
<td id="S6.T2.1.1.6.5.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.6.5.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">47%</td>
<td id="S6.T2.1.1.6.5.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9667</td>
<td id="S6.T2.1.1.6.5.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">2.83 * E-4</td>
<td id="S6.T2.1.1.6.5.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">65.6132</td>
</tr>
<tr id="S6.T2.1.1.7.6" class="ltx_tr">
<th id="S6.T2.1.1.7.6.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" rowspan="4"><span id="S6.T2.1.1.7.6.1.1" class="ltx_text">
<span id="S6.T2.1.1.7.6.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S6.T2.1.1.7.6.1.1.1.1" class="ltx_tr">
<span id="S6.T2.1.1.7.6.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">Covid-19</span></span>
<span id="S6.T2.1.1.7.6.1.1.1.2" class="ltx_tr">
<span id="S6.T2.1.1.7.6.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">X-ray</span></span>
<span id="S6.T2.1.1.7.6.1.1.1.3" class="ltx_tr">
<span id="S6.T2.1.1.7.6.1.1.1.3.1" class="ltx_td ltx_nopad_r ltx_align_left">Dataset</span></span>
</span></span></th>
<td id="S6.T2.1.1.7.6.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t" rowspan="2"><span id="S6.T2.1.1.7.6.2.1" class="ltx_text">GradInv.</span></td>
<td id="S6.T2.1.1.7.6.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">ResNet-18 Untrained</td>
<td id="S6.T2.1.1.7.6.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.7.6.4.1" class="ltx_text ltx_font_bold">75%</span></td>
<td id="S6.T2.1.1.7.6.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9305</td>
<td id="S6.T2.1.1.7.6.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.6763</td>
<td id="S6.T2.1.1.7.6.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">5908.71</td>
</tr>
<tr id="S6.T2.1.1.8.7" class="ltx_tr">
<td id="S6.T2.1.1.8.7.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T2.1.1.8.7.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T2.1.1.8.7.1.1.1" class="ltx_tr">
<td id="S6.T2.1.1.8.7.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">ResNet-18 Trained</td>
</tr>
</table>
</td>
<td id="S6.T2.1.1.8.7.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">30%</td>
<td id="S6.T2.1.1.8.7.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9252</td>
<td id="S6.T2.1.1.8.7.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.6954</td>
<td id="S6.T2.1.1.8.7.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">4967.69</td>
</tr>
<tr id="S6.T2.1.1.9.8" class="ltx_tr">
<td id="S6.T2.1.1.9.8.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CPL</td>
<td id="S6.T2.1.1.9.8.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.9.8.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">55%</td>
<td id="S6.T2.1.1.9.8.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.9.8.4.1" class="ltx_text ltx_font_bold">0.9999</span></td>
<td id="S6.T2.1.1.9.8.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.9.8.5.1" class="ltx_text ltx_font_bold">2.88 * E-7</span></td>
<td id="S6.T2.1.1.9.8.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">92.2457</td>
</tr>
<tr id="S6.T2.1.1.10.9" class="ltx_tr">
<td id="S6.T2.1.1.10.9.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">DLG</td>
<td id="S6.T2.1.1.10.9.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.10.9.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">55%</td>
<td id="S6.T2.1.1.10.9.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9785</td>
<td id="S6.T2.1.1.10.9.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">3.39 * E-3</td>
<td id="S6.T2.1.1.10.9.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">125.6201</td>
</tr>
<tr id="S6.T2.1.1.11.10" class="ltx_tr">
<th id="S6.T2.1.1.11.10.1" class="ltx_td ltx_th ltx_th_row ltx_border_l ltx_border_r"></th>
<td id="S6.T2.1.1.11.10.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">iDLG</td>
<td id="S6.T2.1.1.11.10.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.11.10.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">56%</td>
<td id="S6.T2.1.1.11.10.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9957</td>
<td id="S6.T2.1.1.11.10.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">1.42 * E-4</td>
<td id="S6.T2.1.1.11.10.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">81.4562</td>
</tr>
<tr id="S6.T2.1.1.12.11" class="ltx_tr">
<th id="S6.T2.1.1.12.11.1" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" rowspan="4"><span id="S6.T2.1.1.12.11.1.1" class="ltx_text">
<span id="S6.T2.1.1.12.11.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S6.T2.1.1.12.11.1.1.1.1" class="ltx_tr">
<span id="S6.T2.1.1.12.11.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">Brain</span></span>
<span id="S6.T2.1.1.12.11.1.1.1.2" class="ltx_tr">
<span id="S6.T2.1.1.12.11.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_left">Tumor</span></span>
<span id="S6.T2.1.1.12.11.1.1.1.3" class="ltx_tr">
<span id="S6.T2.1.1.12.11.1.1.1.3.1" class="ltx_td ltx_nopad_r ltx_align_left">MRI</span></span>
<span id="S6.T2.1.1.12.11.1.1.1.4" class="ltx_tr">
<span id="S6.T2.1.1.12.11.1.1.1.4.1" class="ltx_td ltx_nopad_r ltx_align_left">Dataset</span></span>
</span></span></th>
<td id="S6.T2.1.1.12.11.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t" rowspan="2"><span id="S6.T2.1.1.12.11.2.1" class="ltx_text">GradInv.</span></td>
<td id="S6.T2.1.1.12.11.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">ResNet-18 Untrained</td>
<td id="S6.T2.1.1.12.11.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">73%</td>
<td id="S6.T2.1.1.12.11.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9491</td>
<td id="S6.T2.1.1.12.11.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.6061</td>
<td id="S6.T2.1.1.12.11.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">5953.77</td>
</tr>
<tr id="S6.T2.1.1.13.12" class="ltx_tr">
<td id="S6.T2.1.1.13.12.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T2.1.1.13.12.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T2.1.1.13.12.1.1.1" class="ltx_tr">
<td id="S6.T2.1.1.13.12.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">ResNet-18 Trained</td>
</tr>
</table>
</td>
<td id="S6.T2.1.1.13.12.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">13%</td>
<td id="S6.T2.1.1.13.12.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9168</td>
<td id="S6.T2.1.1.13.12.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.8728</td>
<td id="S6.T2.1.1.13.12.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">4933.24</td>
</tr>
<tr id="S6.T2.1.1.14.13" class="ltx_tr">
<td id="S6.T2.1.1.14.13.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CPL</td>
<td id="S6.T2.1.1.14.13.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.14.13.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.14.13.3.1" class="ltx_text ltx_font_bold">76%</span></td>
<td id="S6.T2.1.1.14.13.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.14.13.4.1" class="ltx_text ltx_font_bold">0.9999</span></td>
<td id="S6.T2.1.1.14.13.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T2.1.1.14.13.5.1" class="ltx_text ltx_font_bold">4.57 * E-7</span></td>
<td id="S6.T2.1.1.14.13.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">103.126</td>
</tr>
<tr id="S6.T2.1.1.15.14" class="ltx_tr">
<td id="S6.T2.1.1.15.14.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">DLG</td>
<td id="S6.T2.1.1.15.14.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.15.14.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">71%</td>
<td id="S6.T2.1.1.15.14.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.9445</td>
<td id="S6.T2.1.1.15.14.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">1.51 * E-2</td>
<td id="S6.T2.1.1.15.14.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">116.485</td>
</tr>
<tr id="S6.T2.1.1.16.15" class="ltx_tr">
<th id="S6.T2.1.1.16.15.1" class="ltx_td ltx_th ltx_th_row ltx_border_b ltx_border_l ltx_border_r"></th>
<td id="S6.T2.1.1.16.15.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">iDLG</td>
<td id="S6.T2.1.1.16.15.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">CNN</td>
<td id="S6.T2.1.1.16.15.4" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">72%</td>
<td id="S6.T2.1.1.16.15.5" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">0.9972</td>
<td id="S6.T2.1.1.16.15.6" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">3.95 * E-5</td>
<td id="S6.T2.1.1.16.15.7" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">75.7268</td>
</tr>
</tbody>
</table>
</span></div>
</figure>
<figure id="S6.F7" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2409.18907/assets/x6.png" id="S6.F7.g1" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="194" height="49" alt="Refer to caption"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 5: </span>CPL attack on Melanoma Skin Cancer Dataset</figcaption><div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2409.18907/assets/x7.png" id="S6.F7.g2" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="194" height="46" alt="Refer to caption"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 6: </span>CPL attack on COVID-19 X-ray Dataset</figcaption><div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1"><img src="/html/2409.18907/assets/x8.png" id="S6.F7.g3" class="ltx_graphics ltx_centering ltx_figure_panel ltx_img_landscape" width="196" height="47" alt="Refer to caption"></div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 7: </span>CPL attack on Brain Tumor MRI Images Dataset</figcaption>
</figure>
<div id="S6.SS3.p2" class="ltx_para">
<p id="S6.SS3.p2.3" class="ltx_p">In TableÂ <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a>, we note the ASR values (4th column) for the CPL, DLG, and iDLG attack methods, indicating that approximately 50<math id="S6.SS3.p2.1.m1.1" class="ltx_Math" alttext="\%" display="inline"><semantics id="S6.SS3.p2.1.m1.1a"><mo id="S6.SS3.p2.1.m1.1.1" xref="S6.SS3.p2.1.m1.1.1.cmml">%</mo><annotation-xml encoding="MathML-Content" id="S6.SS3.p2.1.m1.1b"><csymbol cd="latexml" id="S6.SS3.p2.1.m1.1.1.cmml" xref="S6.SS3.p2.1.m1.1.1">percent</csymbol></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.p2.1.m1.1c">\%</annotation></semantics></math>, 55<math id="S6.SS3.p2.2.m2.1" class="ltx_Math" alttext="\%" display="inline"><semantics id="S6.SS3.p2.2.m2.1a"><mo id="S6.SS3.p2.2.m2.1.1" xref="S6.SS3.p2.2.m2.1.1.cmml">%</mo><annotation-xml encoding="MathML-Content" id="S6.SS3.p2.2.m2.1b"><csymbol cd="latexml" id="S6.SS3.p2.2.m2.1.1.cmml" xref="S6.SS3.p2.2.m2.1.1">percent</csymbol></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.p2.2.m2.1c">\%</annotation></semantics></math>, and 75<math id="S6.SS3.p2.3.m3.1" class="ltx_Math" alttext="\%" display="inline"><semantics id="S6.SS3.p2.3.m3.1a"><mo id="S6.SS3.p2.3.m3.1.1" xref="S6.SS3.p2.3.m3.1.1.cmml">%</mo><annotation-xml encoding="MathML-Content" id="S6.SS3.p2.3.m3.1b"><csymbol cd="latexml" id="S6.SS3.p2.3.m3.1.1.cmml" xref="S6.SS3.p2.3.m3.1.1">percent</csymbol></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.p2.3.m3.1c">\%</annotation></semantics></math> of images have been successfully reconstructed for Melanoma Skin Cancer, COVID-19 X-ray, and Brain Tumor MRI Images, respectively. SSIM and MSE for successful attacks are also shown in the 5th and 6th column in Table <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a>. Additionally, we compare the execution time required for performing the privacy attacks across all scenarios (7th column in Table <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a>). FigureÂ <a href="#S6.F7" title="Figure 7 â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>, FigureÂ <a href="#S6.F7" title="Figure 7 â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>, and FigureÂ <a href="#S6.F7" title="Figure 7 â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a> illustrate the original training image and the intermediate reconstructed images resulting from a successful CPL attack on Melanoma Skin Cancer, COVID-19 X-ray, and Brain Tumor MRI Images, respectively, over 200 iterations. We also show some representative medical images and the corresponding reconstructed images after performing iDLG attackÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib44" title="" class="ltx_ref">44</a>]</cite> in Figure <a href="#S6.F8" title="Figure 8 â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a> from all three datasets. TableÂ <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a> and these figures demonstrate that patientsâ€™ private medical images can be precisely reconstructed with minimal noise from the gradients in the FL environment.</p>
</div>
<figure id="S6.F8" class="ltx_figure"><img src="/html/2409.18907/assets/x9.png" id="S6.F8.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="461" height="185" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 8: </span>iDLG attack on Medical Image Datasets (Top row: original private training images, Bottom Row: reconstructed images).</figcaption>
</figure>
<div id="S6.SS3.p3" class="ltx_para">
<p id="S6.SS3.p3.1" class="ltx_p">We also present reconstructed images after performing GradInv attack on all the datasets for both ResNet-18 <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib57" title="" class="ltx_ref">57</a>]</cite> trained and untrained versions with CPL, DLG, and iDLG attack methods. We show the performance of GradInv attack method in the first two rows of TableÂ <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a> on all corresponding datasets. We observed that the performance of GradInv is better on the untrained ResNet-18 model than the trained ResNet-18 in terms of all evaluation metrics (higher ASR, higher SSIM, and lower MSE). However, the reconstruction process of the GradInv attack for the original private medical image on the untrained ResNet-18 takes a little more time than the trained ResNet-18, which is consistent with the results of GradInvÂ <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib16" title="" class="ltx_ref">16</a>]</cite>. FigureÂ <a href="#S6.F9" title="Figure 9 â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">9</span></a> visually illustrates the attack performance by the GradInv on Melanoma Skin Cancer, COVID-19 X-ray, and Brain Tumor MRI datasets respectively for 24,000 iterations which supports our observations from TableÂ <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a>. Comparing the performance of CPL, DLG, and iDLG (in Table <a href="#S6.T2" title="TABLE II â€£ VI-C Performance of Attack Methods â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a>), we observe that GradInv can reconstruct high-quality images by analyzing models with advanced architecture, such as ResNet-18. Though GradInv takes much longer time than CPL, DLG, and iDLG, the performance is better (sometimes similar) in terms of reconstruction quality as well as ASR evaluation metric.</p>
</div>
<figure id="S6.F9" class="ltx_figure"><img src="/html/2409.18907/assets/x10.png" id="S6.F9.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="461" height="229" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 9: </span>GradInv attack recovered images after 24000 iterations on both untrained and trained ResNet-18 (Top row: original, Middle row: recovered from untrained ResNet-18, Bottom Row: recovered from trained ResNet-18).</figcaption>
</figure>
</section>
<section id="S6.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS4.5.1.1" class="ltx_text">VI-D</span> </span><span id="S6.SS4.6.2" class="ltx_text ltx_font_italic">Defense Mechanism Configuration</span>
</h3>

<div id="S6.SS4.p1" class="ltx_para">
<p id="S6.SS4.p1.2" class="ltx_p">In this paper, we perform gradient perturbation as a defense mechanism for preventing privacy leakage attacks. Specifically, we investigate the insertion of a controlled amount of Laplacian noise, characterized by zero mean, into the gradients <math id="S6.SS4.p1.1.m1.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S6.SS4.p1.1.m1.1a"><mrow id="S6.SS4.p1.1.m1.1.1" xref="S6.SS4.p1.1.m1.1.1.cmml"><mrow id="S6.SS4.p1.1.m1.1.1.3" xref="S6.SS4.p1.1.m1.1.1.3.cmml"><mo rspace="0.167em" id="S6.SS4.p1.1.m1.1.1.3.1" xref="S6.SS4.p1.1.m1.1.1.3.1.cmml">âˆ‡</mo><msub id="S6.SS4.p1.1.m1.1.1.3.2" xref="S6.SS4.p1.1.m1.1.1.3.2.cmml"><mi id="S6.SS4.p1.1.m1.1.1.3.2.2" xref="S6.SS4.p1.1.m1.1.1.3.2.2.cmml">L</mi><mi id="S6.SS4.p1.1.m1.1.1.3.2.3" xref="S6.SS4.p1.1.m1.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S6.SS4.p1.1.m1.1.1.2" xref="S6.SS4.p1.1.m1.1.1.2.cmml">â€‹</mo><mrow id="S6.SS4.p1.1.m1.1.1.1.1" xref="S6.SS4.p1.1.m1.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.SS4.p1.1.m1.1.1.1.1.2" xref="S6.SS4.p1.1.m1.1.1.1.1.1.cmml">(</mo><msub id="S6.SS4.p1.1.m1.1.1.1.1.1" xref="S6.SS4.p1.1.m1.1.1.1.1.1.cmml"><mi id="S6.SS4.p1.1.m1.1.1.1.1.1.2" xref="S6.SS4.p1.1.m1.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S6.SS4.p1.1.m1.1.1.1.1.1.3" xref="S6.SS4.p1.1.m1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S6.SS4.p1.1.m1.1.1.1.1.3" xref="S6.SS4.p1.1.m1.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS4.p1.1.m1.1b"><apply id="S6.SS4.p1.1.m1.1.1.cmml" xref="S6.SS4.p1.1.m1.1.1"><times id="S6.SS4.p1.1.m1.1.1.2.cmml" xref="S6.SS4.p1.1.m1.1.1.2"></times><apply id="S6.SS4.p1.1.m1.1.1.3.cmml" xref="S6.SS4.p1.1.m1.1.1.3"><ci id="S6.SS4.p1.1.m1.1.1.3.1.cmml" xref="S6.SS4.p1.1.m1.1.1.3.1">âˆ‡</ci><apply id="S6.SS4.p1.1.m1.1.1.3.2.cmml" xref="S6.SS4.p1.1.m1.1.1.3.2"><csymbol cd="ambiguous" id="S6.SS4.p1.1.m1.1.1.3.2.1.cmml" xref="S6.SS4.p1.1.m1.1.1.3.2">subscript</csymbol><ci id="S6.SS4.p1.1.m1.1.1.3.2.2.cmml" xref="S6.SS4.p1.1.m1.1.1.3.2.2">ğ¿</ci><ci id="S6.SS4.p1.1.m1.1.1.3.2.3.cmml" xref="S6.SS4.p1.1.m1.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S6.SS4.p1.1.m1.1.1.1.1.1.cmml" xref="S6.SS4.p1.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="S6.SS4.p1.1.m1.1.1.1.1.1.1.cmml" xref="S6.SS4.p1.1.m1.1.1.1.1">subscript</csymbol><ci id="S6.SS4.p1.1.m1.1.1.1.1.1.2.cmml" xref="S6.SS4.p1.1.m1.1.1.1.1.1.2">ğœ”</ci><ci id="S6.SS4.p1.1.m1.1.1.1.1.1.3.cmml" xref="S6.SS4.p1.1.m1.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS4.p1.1.m1.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math> during transmission to the central server, as illustrated in FigureÂ <a href="#S5.F3" title="Figure 3 â€£ V-C Attack Method on Medical Images â€£ V Methodology â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>(b). The addition of noise to the <math id="S6.SS4.p1.2.m2.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S6.SS4.p1.2.m2.1a"><mrow id="S6.SS4.p1.2.m2.1.1" xref="S6.SS4.p1.2.m2.1.1.cmml"><mrow id="S6.SS4.p1.2.m2.1.1.3" xref="S6.SS4.p1.2.m2.1.1.3.cmml"><mo rspace="0.167em" id="S6.SS4.p1.2.m2.1.1.3.1" xref="S6.SS4.p1.2.m2.1.1.3.1.cmml">âˆ‡</mo><msub id="S6.SS4.p1.2.m2.1.1.3.2" xref="S6.SS4.p1.2.m2.1.1.3.2.cmml"><mi id="S6.SS4.p1.2.m2.1.1.3.2.2" xref="S6.SS4.p1.2.m2.1.1.3.2.2.cmml">L</mi><mi id="S6.SS4.p1.2.m2.1.1.3.2.3" xref="S6.SS4.p1.2.m2.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S6.SS4.p1.2.m2.1.1.2" xref="S6.SS4.p1.2.m2.1.1.2.cmml">â€‹</mo><mrow id="S6.SS4.p1.2.m2.1.1.1.1" xref="S6.SS4.p1.2.m2.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.SS4.p1.2.m2.1.1.1.1.2" xref="S6.SS4.p1.2.m2.1.1.1.1.1.cmml">(</mo><msub id="S6.SS4.p1.2.m2.1.1.1.1.1" xref="S6.SS4.p1.2.m2.1.1.1.1.1.cmml"><mi id="S6.SS4.p1.2.m2.1.1.1.1.1.2" xref="S6.SS4.p1.2.m2.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S6.SS4.p1.2.m2.1.1.1.1.1.3" xref="S6.SS4.p1.2.m2.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S6.SS4.p1.2.m2.1.1.1.1.3" xref="S6.SS4.p1.2.m2.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS4.p1.2.m2.1b"><apply id="S6.SS4.p1.2.m2.1.1.cmml" xref="S6.SS4.p1.2.m2.1.1"><times id="S6.SS4.p1.2.m2.1.1.2.cmml" xref="S6.SS4.p1.2.m2.1.1.2"></times><apply id="S6.SS4.p1.2.m2.1.1.3.cmml" xref="S6.SS4.p1.2.m2.1.1.3"><ci id="S6.SS4.p1.2.m2.1.1.3.1.cmml" xref="S6.SS4.p1.2.m2.1.1.3.1">âˆ‡</ci><apply id="S6.SS4.p1.2.m2.1.1.3.2.cmml" xref="S6.SS4.p1.2.m2.1.1.3.2"><csymbol cd="ambiguous" id="S6.SS4.p1.2.m2.1.1.3.2.1.cmml" xref="S6.SS4.p1.2.m2.1.1.3.2">subscript</csymbol><ci id="S6.SS4.p1.2.m2.1.1.3.2.2.cmml" xref="S6.SS4.p1.2.m2.1.1.3.2.2">ğ¿</ci><ci id="S6.SS4.p1.2.m2.1.1.3.2.3.cmml" xref="S6.SS4.p1.2.m2.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S6.SS4.p1.2.m2.1.1.1.1.1.cmml" xref="S6.SS4.p1.2.m2.1.1.1.1"><csymbol cd="ambiguous" id="S6.SS4.p1.2.m2.1.1.1.1.1.1.cmml" xref="S6.SS4.p1.2.m2.1.1.1.1">subscript</csymbol><ci id="S6.SS4.p1.2.m2.1.1.1.1.1.2.cmml" xref="S6.SS4.p1.2.m2.1.1.1.1.1.2">ğœ”</ci><ci id="S6.SS4.p1.2.m2.1.1.1.1.1.3.cmml" xref="S6.SS4.p1.2.m2.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS4.p1.2.m2.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math> introduces uncertainty in local updates, obscures fine details, and thwarts adversariesâ€™ attempts to accurately reconstruct private medical image data. Through our empirical analysis, we find that the default level of noise may not always provide sufficient defense. Therefore, we experiment with varying levels of noise to identify robust defense configurations.</p>
</div>
<figure id="S6.T3" class="ltx_table">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">TABLE III: </span>CPL attack and defense mechanism performance for different noise levels on three benchmark datasets and CIFAR-10</figcaption>
<div id="S6.T3.1" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:309.8pt;height:313.6pt;vertical-align:-0.0pt;"><span class="ltx_transformed_inner" style="transform:translate(-27.3pt,27.7pt) scale(0.85,0.85) ;">
<table id="S6.T3.1.1" class="ltx_tabular ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S6.T3.1.1.1.1" class="ltx_tr">
<td id="S6.T3.1.1.1.1.1" class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t" colspan="2" rowspan="2"><span id="S6.T3.1.1.1.1.1.1" class="ltx_text ltx_font_bold">Dataset</span></td>
<td id="S6.T3.1.1.1.1.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t" colspan="2"><span id="S6.T3.1.1.1.1.2.1" class="ltx_text ltx_font_bold">CPL Attack</span></td>
<td id="S6.T3.1.1.1.1.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t" colspan="3"><span id="S6.T3.1.1.1.1.3.1" class="ltx_text ltx_font_bold">Defense</span></td>
</tr>
<tr id="S6.T3.1.1.2.2" class="ltx_tr">
<td id="S6.T3.1.1.2.2.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T3.1.1.2.2.1.1" class="ltx_text ltx_font_bold">MSE</span></td>
<td id="S6.T3.1.1.2.2.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T3.1.1.2.2.2.1" class="ltx_text ltx_font_bold">SSIM</span></td>
<td id="S6.T3.1.1.2.2.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.2.2.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.2.2.3.1.1" class="ltx_tr">
<td id="S6.T3.1.1.2.2.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.2.2.3.1.1.1.1" class="ltx_text ltx_font_bold">Noise</span></td>
</tr>
<tr id="S6.T3.1.1.2.2.3.1.2" class="ltx_tr">
<td id="S6.T3.1.1.2.2.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.2.2.3.1.2.1.1" class="ltx_text ltx_font_bold">Levels</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.2.2.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T3.1.1.2.2.4.1" class="ltx_text ltx_font_bold">MSE</span></td>
<td id="S6.T3.1.1.2.2.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S6.T3.1.1.2.2.5.1" class="ltx_text ltx_font_bold">SSIM</span></td>
</tr>
<tr id="S6.T3.1.1.3.3" class="ltx_tr">
<td id="S6.T3.1.1.3.3.1" class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t">1.</td>
<td id="S6.T3.1.1.3.3.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">Melanoma Skin Cancer</td>
<td id="S6.T3.1.1.3.3.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.0762</td>
<td id="S6.T3.1.1.3.3.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.50</td>
<td id="S6.T3.1.1.3.3.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.3.3.5.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.3.3.5.1.1" class="ltx_tr">
<td id="S6.T3.1.1.3.3.5.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">100</td>
</tr>
<tr id="S6.T3.1.1.3.3.5.1.2" class="ltx_tr">
<td id="S6.T3.1.1.3.3.5.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">200</td>
</tr>
<tr id="S6.T3.1.1.3.3.5.1.3" class="ltx_tr">
<td id="S6.T3.1.1.3.3.5.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">300</td>
</tr>
<tr id="S6.T3.1.1.3.3.5.1.4" class="ltx_tr">
<td id="S6.T3.1.1.3.3.5.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.3.3.5.1.4.1.1" class="ltx_text ltx_font_bold">400</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.3.3.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.3.3.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.3.3.6.1.1" class="ltx_tr">
<td id="S6.T3.1.1.3.3.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">0.1306</td>
</tr>
<tr id="S6.T3.1.1.3.3.6.1.2" class="ltx_tr">
<td id="S6.T3.1.1.3.3.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">0.1468</td>
</tr>
<tr id="S6.T3.1.1.3.3.6.1.3" class="ltx_tr">
<td id="S6.T3.1.1.3.3.6.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">0.1497</td>
</tr>
<tr id="S6.T3.1.1.3.3.6.1.4" class="ltx_tr">
<td id="S6.T3.1.1.3.3.6.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.3.3.6.1.4.1.1" class="ltx_text ltx_font_bold">0.1503</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.3.3.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.3.3.7.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.3.3.7.1.1" class="ltx_tr">
<td id="S6.T3.1.1.3.3.7.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0154</td>
</tr>
<tr id="S6.T3.1.1.3.3.7.1.2" class="ltx_tr">
<td id="S6.T3.1.1.3.3.7.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0131</td>
</tr>
<tr id="S6.T3.1.1.3.3.7.1.3" class="ltx_tr">
<td id="S6.T3.1.1.3.3.7.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0121</td>
</tr>
<tr id="S6.T3.1.1.3.3.7.1.4" class="ltx_tr">
<td id="S6.T3.1.1.3.3.7.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.3.3.7.1.4.1.1" class="ltx_text ltx_font_bold">0.0101</span></td>
</tr>
</table>
</td>
</tr>
<tr id="S6.T3.1.1.4.4" class="ltx_tr">
<td id="S6.T3.1.1.4.4.1" class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t">2.</td>
<td id="S6.T3.1.1.4.4.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">COVID-19 X-ray</td>
<td id="S6.T3.1.1.4.4.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.0641</td>
<td id="S6.T3.1.1.4.4.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.55</td>
<td id="S6.T3.1.1.4.4.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.4.4.5.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.4.4.5.1.1" class="ltx_tr">
<td id="S6.T3.1.1.4.4.5.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">100</td>
</tr>
<tr id="S6.T3.1.1.4.4.5.1.2" class="ltx_tr">
<td id="S6.T3.1.1.4.4.5.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">200</td>
</tr>
<tr id="S6.T3.1.1.4.4.5.1.3" class="ltx_tr">
<td id="S6.T3.1.1.4.4.5.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">300</td>
</tr>
<tr id="S6.T3.1.1.4.4.5.1.4" class="ltx_tr">
<td id="S6.T3.1.1.4.4.5.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.4.4.5.1.4.1.1" class="ltx_text ltx_font_bold">400</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.4.4.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.4.4.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.4.4.6.1.1" class="ltx_tr">
<td id="S6.T3.1.1.4.4.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0013</td>
</tr>
<tr id="S6.T3.1.1.4.4.6.1.2" class="ltx_tr">
<td id="S6.T3.1.1.4.4.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0157</td>
</tr>
<tr id="S6.T3.1.1.4.4.6.1.3" class="ltx_tr">
<td id="S6.T3.1.1.4.4.6.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0206</td>
</tr>
<tr id="S6.T3.1.1.4.4.6.1.4" class="ltx_tr">
<td id="S6.T3.1.1.4.4.6.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.4.4.6.1.4.1.1" class="ltx_text ltx_font_bold">0.0578</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.4.4.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.4.4.7.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.4.4.7.1.1" class="ltx_tr">
<td id="S6.T3.1.1.4.4.7.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">0.9815</td>
</tr>
<tr id="S6.T3.1.1.4.4.7.1.2" class="ltx_tr">
<td id="S6.T3.1.1.4.4.7.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">0.7410</td>
</tr>
<tr id="S6.T3.1.1.4.4.7.1.3" class="ltx_tr">
<td id="S6.T3.1.1.4.4.7.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">0.7000</td>
</tr>
<tr id="S6.T3.1.1.4.4.7.1.4" class="ltx_tr">
<td id="S6.T3.1.1.4.4.7.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.4.4.7.1.4.1.1" class="ltx_text ltx_font_bold">0.4605</span></td>
</tr>
</table>
</td>
</tr>
<tr id="S6.T3.1.1.5.5" class="ltx_tr">
<td id="S6.T3.1.1.5.5.1" class="ltx_td ltx_align_center ltx_border_l ltx_border_r ltx_border_t">3.</td>
<td id="S6.T3.1.1.5.5.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">Brain Tumor MRI Images</td>
<td id="S6.T3.1.1.5.5.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.0586</td>
<td id="S6.T3.1.1.5.5.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.75</td>
<td id="S6.T3.1.1.5.5.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.5.5.5.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.5.5.5.1.1" class="ltx_tr">
<td id="S6.T3.1.1.5.5.5.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">100</td>
</tr>
<tr id="S6.T3.1.1.5.5.5.1.2" class="ltx_tr">
<td id="S6.T3.1.1.5.5.5.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">200</td>
</tr>
<tr id="S6.T3.1.1.5.5.5.1.3" class="ltx_tr">
<td id="S6.T3.1.1.5.5.5.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">300</td>
</tr>
<tr id="S6.T3.1.1.5.5.5.1.4" class="ltx_tr">
<td id="S6.T3.1.1.5.5.5.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.5.5.5.1.4.1.1" class="ltx_text ltx_font_bold">400</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.5.5.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.5.5.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.5.5.6.1.1" class="ltx_tr">
<td id="S6.T3.1.1.5.5.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0207</td>
</tr>
<tr id="S6.T3.1.1.5.5.6.1.2" class="ltx_tr">
<td id="S6.T3.1.1.5.5.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0686</td>
</tr>
<tr id="S6.T3.1.1.5.5.6.1.3" class="ltx_tr">
<td id="S6.T3.1.1.5.5.6.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0300</td>
</tr>
<tr id="S6.T3.1.1.5.5.6.1.4" class="ltx_tr">
<td id="S6.T3.1.1.5.5.6.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.5.5.6.1.4.1.1" class="ltx_text ltx_font_bold">0.1705</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.5.5.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.5.5.7.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.5.5.7.1.1" class="ltx_tr">
<td id="S6.T3.1.1.5.5.7.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">0.6657</td>
</tr>
<tr id="S6.T3.1.1.5.5.7.1.2" class="ltx_tr">
<td id="S6.T3.1.1.5.5.7.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">0.2383</td>
</tr>
<tr id="S6.T3.1.1.5.5.7.1.3" class="ltx_tr">
<td id="S6.T3.1.1.5.5.7.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">0.3661</td>
</tr>
<tr id="S6.T3.1.1.5.5.7.1.4" class="ltx_tr">
<td id="S6.T3.1.1.5.5.7.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.5.5.7.1.4.1.1" class="ltx_text ltx_font_bold">0.0699</span></td>
</tr>
</table>
</td>
</tr>
<tr id="S6.T3.1.1.6.6" class="ltx_tr">
<td id="S6.T3.1.1.6.6.1" class="ltx_td ltx_align_center ltx_border_b ltx_border_l ltx_border_r ltx_border_t">4.</td>
<td id="S6.T3.1.1.6.6.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">CIFAR-10</td>
<td id="S6.T3.1.1.6.6.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">0.0222</td>
<td id="S6.T3.1.1.6.6.4" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">0.86</td>
<td id="S6.T3.1.1.6.6.5" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.6.6.5.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.6.6.5.1.1" class="ltx_tr">
<td id="S6.T3.1.1.6.6.5.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">100</td>
</tr>
<tr id="S6.T3.1.1.6.6.5.1.2" class="ltx_tr">
<td id="S6.T3.1.1.6.6.5.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">200</td>
</tr>
<tr id="S6.T3.1.1.6.6.5.1.3" class="ltx_tr">
<td id="S6.T3.1.1.6.6.5.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">300</td>
</tr>
<tr id="S6.T3.1.1.6.6.5.1.4" class="ltx_tr">
<td id="S6.T3.1.1.6.6.5.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.6.6.5.1.4.1.1" class="ltx_text ltx_font_bold">400</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.6.6.6" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.6.6.6.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.6.6.6.1.1" class="ltx_tr">
<td id="S6.T3.1.1.6.6.6.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0265</td>
</tr>
<tr id="S6.T3.1.1.6.6.6.1.2" class="ltx_tr">
<td id="S6.T3.1.1.6.6.6.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0292</td>
</tr>
<tr id="S6.T3.1.1.6.6.6.1.3" class="ltx_tr">
<td id="S6.T3.1.1.6.6.6.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">0.0346</td>
</tr>
<tr id="S6.T3.1.1.6.6.6.1.4" class="ltx_tr">
<td id="S6.T3.1.1.6.6.6.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.6.6.6.1.4.1.1" class="ltx_text ltx_font_bold">0.0475</span></td>
</tr>
</table>
</td>
<td id="S6.T3.1.1.6.6.7" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">
<table id="S6.T3.1.1.6.6.7.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T3.1.1.6.6.7.1.1" class="ltx_tr">
<td id="S6.T3.1.1.6.6.7.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">0.4965</td>
</tr>
<tr id="S6.T3.1.1.6.6.7.1.2" class="ltx_tr">
<td id="S6.T3.1.1.6.6.7.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">0.4366</td>
</tr>
<tr id="S6.T3.1.1.6.6.7.1.3" class="ltx_tr">
<td id="S6.T3.1.1.6.6.7.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">0.3270</td>
</tr>
<tr id="S6.T3.1.1.6.6.7.1.4" class="ltx_tr">
<td id="S6.T3.1.1.6.6.7.1.4.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T3.1.1.6.6.7.1.4.1.1" class="ltx_text ltx_font_bold">0.2295</span></td>
</tr>
</table>
</td>
</tr>
</tbody>
</table>
</span></div>
</figure>
</section>
<section id="S6.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection"><span id="S6.SS5.5.1.1" class="ltx_text">VI-E</span> </span><span id="S6.SS5.6.2" class="ltx_text ltx_font_italic">Performance of Defense Mechanisms.</span>
</h3>

<div id="S6.SS5.p1" class="ltx_para">
<p id="S6.SS5.p1.1" class="ltx_p">The aim of defense against privacy attacks is to enhance the dissimilarity, measured by metrics such as MSE, and minimize the similarity, quantified by metrics like SSIM, between the original private training images and the reconstructed images by performing privacy attacks. As presented in TableÂ <a href="#S6.T3" title="TABLE III â€£ VI-D Defense Mechanism Configuration â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">III</span></a>, we observe that defense becomes stronger as we enhance the level of Laplacian noise to the gradients as shown in FigureÂ <a href="#S5.F3" title="Figure 3 â€£ V-C Attack Method on Medical Images â€£ V Methodology â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>(b). Introducing random noise to <math id="S6.SS5.p1.1.m1.1" class="ltx_Math" alttext="\nabla L_{k}(\omega_{t})" display="inline"><semantics id="S6.SS5.p1.1.m1.1a"><mrow id="S6.SS5.p1.1.m1.1.1" xref="S6.SS5.p1.1.m1.1.1.cmml"><mrow id="S6.SS5.p1.1.m1.1.1.3" xref="S6.SS5.p1.1.m1.1.1.3.cmml"><mo rspace="0.167em" id="S6.SS5.p1.1.m1.1.1.3.1" xref="S6.SS5.p1.1.m1.1.1.3.1.cmml">âˆ‡</mo><msub id="S6.SS5.p1.1.m1.1.1.3.2" xref="S6.SS5.p1.1.m1.1.1.3.2.cmml"><mi id="S6.SS5.p1.1.m1.1.1.3.2.2" xref="S6.SS5.p1.1.m1.1.1.3.2.2.cmml">L</mi><mi id="S6.SS5.p1.1.m1.1.1.3.2.3" xref="S6.SS5.p1.1.m1.1.1.3.2.3.cmml">k</mi></msub></mrow><mo lspace="0em" rspace="0em" id="S6.SS5.p1.1.m1.1.1.2" xref="S6.SS5.p1.1.m1.1.1.2.cmml">â€‹</mo><mrow id="S6.SS5.p1.1.m1.1.1.1.1" xref="S6.SS5.p1.1.m1.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.SS5.p1.1.m1.1.1.1.1.2" xref="S6.SS5.p1.1.m1.1.1.1.1.1.cmml">(</mo><msub id="S6.SS5.p1.1.m1.1.1.1.1.1" xref="S6.SS5.p1.1.m1.1.1.1.1.1.cmml"><mi id="S6.SS5.p1.1.m1.1.1.1.1.1.2" xref="S6.SS5.p1.1.m1.1.1.1.1.1.2.cmml">Ï‰</mi><mi id="S6.SS5.p1.1.m1.1.1.1.1.1.3" xref="S6.SS5.p1.1.m1.1.1.1.1.1.3.cmml">t</mi></msub><mo stretchy="false" id="S6.SS5.p1.1.m1.1.1.1.1.3" xref="S6.SS5.p1.1.m1.1.1.1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS5.p1.1.m1.1b"><apply id="S6.SS5.p1.1.m1.1.1.cmml" xref="S6.SS5.p1.1.m1.1.1"><times id="S6.SS5.p1.1.m1.1.1.2.cmml" xref="S6.SS5.p1.1.m1.1.1.2"></times><apply id="S6.SS5.p1.1.m1.1.1.3.cmml" xref="S6.SS5.p1.1.m1.1.1.3"><ci id="S6.SS5.p1.1.m1.1.1.3.1.cmml" xref="S6.SS5.p1.1.m1.1.1.3.1">âˆ‡</ci><apply id="S6.SS5.p1.1.m1.1.1.3.2.cmml" xref="S6.SS5.p1.1.m1.1.1.3.2"><csymbol cd="ambiguous" id="S6.SS5.p1.1.m1.1.1.3.2.1.cmml" xref="S6.SS5.p1.1.m1.1.1.3.2">subscript</csymbol><ci id="S6.SS5.p1.1.m1.1.1.3.2.2.cmml" xref="S6.SS5.p1.1.m1.1.1.3.2.2">ğ¿</ci><ci id="S6.SS5.p1.1.m1.1.1.3.2.3.cmml" xref="S6.SS5.p1.1.m1.1.1.3.2.3">ğ‘˜</ci></apply></apply><apply id="S6.SS5.p1.1.m1.1.1.1.1.1.cmml" xref="S6.SS5.p1.1.m1.1.1.1.1"><csymbol cd="ambiguous" id="S6.SS5.p1.1.m1.1.1.1.1.1.1.cmml" xref="S6.SS5.p1.1.m1.1.1.1.1">subscript</csymbol><ci id="S6.SS5.p1.1.m1.1.1.1.1.1.2.cmml" xref="S6.SS5.p1.1.m1.1.1.1.1.1.2">ğœ”</ci><ci id="S6.SS5.p1.1.m1.1.1.1.1.1.3.cmml" xref="S6.SS5.p1.1.m1.1.1.1.1.1.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p1.1.m1.1c">\nabla L_{k}(\omega_{t})</annotation></semantics></math> makes the extraction of sensitive information from local gradients more challenging in FL environment. FigureÂ <a href="#S6.F10" title="Figure 10 â€£ VI-E Performance of Defense Mechanisms. â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a> visualizes the outcome of the defense mechanism for noise levels at 100, 200, 300, and 400 respectively for the Brain Tumor MRI dataset. At lower noise levels, such as 100, private information remains susceptible to extraction under the CPL attack, as shown by the first row of FigureÂ <a href="#S6.F10" title="Figure 10 â€£ VI-E Performance of Defense Mechanisms. â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a>. This suggests that only adding a standard amount of random noise may not always offer sufficient privacy protection for medical data within FL settings.</p>
</div>
<figure id="S6.F10" class="ltx_figure"><img src="/html/2409.18907/assets/x11.png" id="S6.F10.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="192" height="82" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 10: </span>Defense to CPL attack on Brain Tumor MRI Dataset</figcaption>
</figure>
<div id="S6.SS5.p2" class="ltx_para">
<p id="S6.SS5.p2.1" class="ltx_p">Furthermore, we extend our analysis to include a comparison of privacy risks of medical images with a generic dataset, CIFAR-10Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib65" title="" class="ltx_ref">65</a>]</cite>, and present the corresponding SSIM and MSE values for equivalent noise levels. This comparison outlined in the last row of TableÂ <a href="#S6.T3" title="TABLE III â€£ VI-D Defense Mechanism Configuration â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">III</span></a>, provides insights into the privacy risks between generic datasets and medical datasets. The comparison of SSIM and MSE values for all four noise levels in CIFAR-10 reveals that the standard level of noise applied to gradients may provide sufficient privacy protection for generic images. However, it may not always provide a sufficiently robust defense for medical images.</p>
</div>
<figure id="S6.T4" class="ltx_table">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">TABLE IV: </span>Statistical Distributions of Medical Image Datasets and Generic Image Dataset</figcaption>
<table id="S6.T4.1" class="ltx_tabular ltx_guessed_headers ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S6.T4.1.1.1" class="ltx_tr">
<th id="S6.T4.1.1.1.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" rowspan="2"><span id="S6.T4.1.1.1.1.1" class="ltx_text ltx_font_bold">Datasets</span></th>
<td id="S6.T4.1.1.1.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t" colspan="2"><span id="S6.T4.1.1.1.2.1" class="ltx_text ltx_font_bold">Properties</span></td>
</tr>
<tr id="S6.T4.1.2.2" class="ltx_tr">
<td id="S6.T4.1.2.2.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T4.1.2.2.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T4.1.2.2.1.1.1" class="ltx_tr">
<td id="S6.T4.1.2.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T4.1.2.2.1.1.1.1.1" class="ltx_text ltx_font_bold">Mean</span></td>
</tr>
</table>
</td>
<td id="S6.T4.1.2.2.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S6.T4.1.2.2.2.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T4.1.2.2.2.1.1" class="ltx_tr">
<td id="S6.T4.1.2.2.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S6.T4.1.2.2.2.1.1.1.1" class="ltx_text ltx_font_bold">Standard Deviation</span></td>
</tr>
</table>
</td>
</tr>
<tr id="S6.T4.1.3.3" class="ltx_tr">
<th id="S6.T4.1.3.3.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t">
<table id="S6.T4.1.3.3.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T4.1.3.3.1.1.1" class="ltx_tr">
<td id="S6.T4.1.3.3.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Melanoma Skin</td>
</tr>
<tr id="S6.T4.1.3.3.1.1.2" class="ltx_tr">
<td id="S6.T4.1.3.3.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Cancer</td>
</tr>
</table>
</th>
<td id="S6.T4.1.3.3.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">[0.7160, 0.5668, 0.5441]</td>
<td id="S6.T4.1.3.3.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">[0.2207, 0.2087, 0.2222]</td>
</tr>
<tr id="S6.T4.1.4.4" class="ltx_tr">
<th id="S6.T4.1.4.4.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t">
<table id="S6.T4.1.4.4.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T4.1.4.4.1.1.1" class="ltx_tr">
<td id="S6.T4.1.4.4.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">COVID-19</td>
</tr>
<tr id="S6.T4.1.4.4.1.1.2" class="ltx_tr">
<td id="S6.T4.1.4.4.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">X-ray</td>
</tr>
</table>
</th>
<td id="S6.T4.1.4.4.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">[0.4949, 0.4950, 0.4953]</td>
<td id="S6.T4.1.4.4.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">[0.2687, 0.2687, 0.2688]</td>
</tr>
<tr id="S6.T4.1.5.5" class="ltx_tr">
<th id="S6.T4.1.5.5.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t">
<table id="S6.T4.1.5.5.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S6.T4.1.5.5.1.1.1" class="ltx_tr">
<td id="S6.T4.1.5.5.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Brain Tumor</td>
</tr>
<tr id="S6.T4.1.5.5.1.1.2" class="ltx_tr">
<td id="S6.T4.1.5.5.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">MRI Scans</td>
</tr>
</table>
</th>
<td id="S6.T4.1.5.5.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">[0.1869, 0.1869, 0.1870]</td>
<td id="S6.T4.1.5.5.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">[0.1763, 0.1763, 0.1763]</td>
</tr>
<tr id="S6.T4.1.6.6" class="ltx_tr">
<th id="S6.T4.1.6.6.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_b ltx_border_l ltx_border_r ltx_border_t">CIFAR-10</th>
<td id="S6.T4.1.6.6.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">[0.4914, 0.4822, 0.4467]</td>
<td id="S6.T4.1.6.6.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">[0.2471, 0.2434, 0.2615]</td>
</tr>
</tbody>
</table>
</figure>
</section>
</section>
<section id="S7" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">VII </span><span id="S7.1.1" class="ltx_text ltx_font_smallcaps">Discussion</span>
</h2>

<div id="S7.p1" class="ltx_para">
<p id="S7.p1.1" class="ltx_p">After performing the experiments for different attack methods and defense mechanisms with various configurations on three medical datasets, we discuss and address several research questions (<span id="S7.p1.1.1" class="ltx_text ltx_font_bold">RQ</span>).</p>
</div>
<div id="S7.p2" class="ltx_para">
<p id="S7.p2.1" class="ltx_p"><span id="S7.p2.1.1" class="ltx_text ltx_font_bold">RQ1: What are the unique challenges for privacy protection of medical images?</span> Apart from the complexity, and high dimensionality, as mentioned in SectionÂ <a href="#S2" title="II Motivation â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">II</span></a>, the statistical distribution of medical images often deviates from the generic images, which implies significant differences between these two data categories in terms of processing, privacy protection, and execution time. From the distribution of the datasets as shown in 3 channels tensor format in Table V, for medical images (first three rows) and the generic image (CIFAR-10)Â <cite class="ltx_cite ltx_citemacro_cite">[<a href="#bib.bib65" title="" class="ltx_ref">65</a>]</cite>, we can observe that the mean and standard deviation of medical images may differ from generic images.</p>
</div>
<figure id="S7.T5" class="ltx_table">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">TABLE V: </span>Model Performance of trained Resnet-18 on original images, recovered images by CPL attack, and recovered images under perturbed (Laplacian noise) gradients for three medical datasets and CIFAR-10.</figcaption>
<div id="S7.T5.1" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:431.4pt;height:140.4pt;vertical-align:-1.0pt;"><span class="ltx_transformed_inner" style="transform:translate(-4.4pt,1.4pt) scale(0.98,0.98) ;">
<table id="S7.T5.1.1" class="ltx_tabular ltx_guessed_headers ltx_align_middle">
<tbody class="ltx_tbody">
<tr id="S7.T5.1.1.1.1" class="ltx_tr">
<th id="S7.T5.1.1.1.1.1" class="ltx_td ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" colspan="2"></th>
<td id="S7.T5.1.1.1.1.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t" colspan="4"><span id="S7.T5.1.1.1.1.2.1" class="ltx_text ltx_font_bold">Accuracy</span></td>
</tr>
<tr id="S7.T5.1.1.2.2" class="ltx_tr">
<th id="S7.T5.1.1.2.2.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" colspan="2">
<table id="S7.T5.1.1.2.2.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S7.T5.1.1.2.2.1.1.1" class="ltx_tr">
<td id="S7.T5.1.1.2.2.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S7.T5.1.1.2.2.1.1.1.1.1" class="ltx_text ltx_font_bold">Data Source</span></td>
</tr>
</table>
</th>
<td id="S7.T5.1.1.2.2.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S7.T5.1.1.2.2.2.1" class="ltx_text ltx_font_bold">CIFAR-10</span></td>
<td id="S7.T5.1.1.2.2.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S7.T5.1.1.2.2.3.1" class="ltx_tabular ltx_align_middle">
<tr id="S7.T5.1.1.2.2.3.1.1" class="ltx_tr">
<td id="S7.T5.1.1.2.2.3.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S7.T5.1.1.2.2.3.1.1.1.1" class="ltx_text ltx_font_bold">Melanoma Skin</span></td>
</tr>
<tr id="S7.T5.1.1.2.2.3.1.2" class="ltx_tr">
<td id="S7.T5.1.1.2.2.3.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S7.T5.1.1.2.2.3.1.2.1.1" class="ltx_text ltx_font_bold">Cancer</span></td>
</tr>
</table>
</td>
<td id="S7.T5.1.1.2.2.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S7.T5.1.1.2.2.4.1" class="ltx_tabular ltx_align_middle">
<tr id="S7.T5.1.1.2.2.4.1.1" class="ltx_tr">
<td id="S7.T5.1.1.2.2.4.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S7.T5.1.1.2.2.4.1.1.1.1" class="ltx_text ltx_font_bold">COVID-19</span></td>
</tr>
<tr id="S7.T5.1.1.2.2.4.1.2" class="ltx_tr">
<td id="S7.T5.1.1.2.2.4.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S7.T5.1.1.2.2.4.1.2.1.1" class="ltx_text ltx_font_bold">X-ray</span></td>
</tr>
</table>
</td>
<td id="S7.T5.1.1.2.2.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<table id="S7.T5.1.1.2.2.5.1" class="ltx_tabular ltx_align_middle">
<tr id="S7.T5.1.1.2.2.5.1.1" class="ltx_tr">
<td id="S7.T5.1.1.2.2.5.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S7.T5.1.1.2.2.5.1.1.1.1" class="ltx_text ltx_font_bold">Brain Tumor</span></td>
</tr>
<tr id="S7.T5.1.1.2.2.5.1.2" class="ltx_tr">
<td id="S7.T5.1.1.2.2.5.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><span id="S7.T5.1.1.2.2.5.1.2.1.1" class="ltx_text ltx_font_bold">MRI Scans</span></td>
</tr>
</table>
</td>
</tr>
<tr id="S7.T5.1.1.3.3" class="ltx_tr">
<th id="S7.T5.1.1.3.3.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" colspan="2">
<table id="S7.T5.1.1.3.3.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S7.T5.1.1.3.3.1.1.1" class="ltx_tr">
<td id="S7.T5.1.1.3.3.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Original Data</td>
</tr>
</table>
</th>
<td id="S7.T5.1.1.3.3.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.82</td>
<td id="S7.T5.1.1.3.3.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.97</td>
<td id="S7.T5.1.1.3.3.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.95</td>
<td id="S7.T5.1.1.3.3.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.96</td>
</tr>
<tr id="S7.T5.1.1.4.4" class="ltx_tr">
<th id="S7.T5.1.1.4.4.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t" colspan="2">
<table id="S7.T5.1.1.4.4.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S7.T5.1.1.4.4.1.1.1" class="ltx_tr">
<td id="S7.T5.1.1.4.4.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Recovered Data</td>
</tr>
</table>
</th>
<td id="S7.T5.1.1.4.4.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.74</td>
<td id="S7.T5.1.1.4.4.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.95</td>
<td id="S7.T5.1.1.4.4.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.87</td>
<td id="S7.T5.1.1.4.4.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.74</td>
</tr>
<tr id="S7.T5.1.1.5.5" class="ltx_tr">
<th id="S7.T5.1.1.5.5.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_b ltx_border_l ltx_border_r ltx_border_t" rowspan="2"><span id="S7.T5.1.1.5.5.1.1" class="ltx_text">
<span id="S7.T5.1.1.5.5.1.1.1" class="ltx_tabular ltx_align_middle">
<span id="S7.T5.1.1.5.5.1.1.1.1" class="ltx_tr">
<span id="S7.T5.1.1.5.5.1.1.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Perturbed gradients with</span></span>
<span id="S7.T5.1.1.5.5.1.1.1.2" class="ltx_tr">
<span id="S7.T5.1.1.5.5.1.1.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Different Noise Levels</span></span>
</span></span></th>
<th id="S7.T5.1.1.5.5.2" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_r ltx_border_t">100</th>
<td id="S7.T5.1.1.5.5.3" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">0.12</td>
<td id="S7.T5.1.1.5.5.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S7.T5.1.1.5.5.4.1" class="ltx_text ltx_font_bold">0.96</span></td>
<td id="S7.T5.1.1.5.5.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S7.T5.1.1.5.5.5.1" class="ltx_text ltx_font_bold">0.90</span></td>
<td id="S7.T5.1.1.5.5.6" class="ltx_td ltx_align_center ltx_border_r ltx_border_t"><span id="S7.T5.1.1.5.5.6.1" class="ltx_text ltx_font_bold">0.92</span></td>
</tr>
<tr id="S7.T5.1.1.6.6" class="ltx_tr">
<th id="S7.T5.1.1.6.6.1" class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_b ltx_border_r ltx_border_t">400</th>
<td id="S7.T5.1.1.6.6.2" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t">0.09</td>
<td id="S7.T5.1.1.6.6.3" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S7.T5.1.1.6.6.3.1" class="ltx_text ltx_font_bold">0.93</span></td>
<td id="S7.T5.1.1.6.6.4" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S7.T5.1.1.6.6.4.1" class="ltx_text ltx_font_bold">0.88</span></td>
<td id="S7.T5.1.1.6.6.5" class="ltx_td ltx_align_center ltx_border_b ltx_border_r ltx_border_t"><span id="S7.T5.1.1.6.6.5.1" class="ltx_text ltx_font_bold">0.87</span></td>
</tr>
</tbody>
</table>
</span></div>
</figure>
<div id="S7.p3" class="ltx_para">
<p id="S7.p3.1" class="ltx_p"><span id="S7.p3.1.1" class="ltx_text ltx_font_bold">RQ2: Which level of Laplacian noise is enough to safeguard the privacy of medical images?</span> As shown and discussed in the previous section, medical image datasets (Brain Tumor MRI dataset, Figure <a href="#S6.F10" title="Figure 10 â€£ VI-E Performance of Defense Mechanisms. â€£ VI Experimental Analysis â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a>) are still susceptible to being revealed even if we add the highest level of noise compared with a generic image dataset (e.g., CIFAR-10). We show several samples in FigureÂ <a href="#S7.F11" title="Figure 11 â€£ VII Discussion â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">11</span></a> after adding the lowest and highest levels of Laplacian noise to the generic CIFAR-10 image and the medical images that we studied in this research respectively. From these visual examples, it can be clearly observed that the lowest level of noise (100) is enough to protect the content of CIFAR-10 images. On the other hand, by looking at the medical images of all datasets, even if we add the highest level of noise (400), a major portion of the contents is still visible, which may reveal their categories. Thus, it requires further studies to build a strong defense for medical images.</p>
</div>
<figure id="S7.F11" class="ltx_figure"><img src="/html/2409.18907/assets/x12.png" id="S7.F11.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="165" height="76" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 11: </span>Dataset Comparison Under Different Noise Levels</figcaption>
</figure>
<div id="S7.p4" class="ltx_para">
<p id="S7.p4.1" class="ltx_p"><span id="S7.p4.1.1" class="ltx_text ltx_font_bold">RQ3: If we keep increasing the noise level to make a stronger defense against privacy leakage attacks, how does it impact the model performance?</span> In order to check the model performance on the recovered images by CPL attack and the reconstructed images after perturbing gradients with different noise levels, we employed trained ResNet-18 to evaluate image classification accuracy on the original data, recovered data, and recovered data under perturbed gradients as shown in Table <a href="#S7.T5" title="TABLE V â€£ VII Discussion â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">V</span></a>. We observe a slight performance drop for reconstructed images by CPL, and a significant drop occurred for the recovered data under perturbed gradients at any noise levels in terms of classification accuracy for the generic image dataset (CIFAR-10). On the contrary, for all medical image datasets under the perturbed gradients, although we observed a slight performance drop in the classification accuracy, it still remains high. The reason behind such high accuracy is that even if we added the highest level of noise to the gradients, most of the contents were visible for all medical images (see medical images in FigureÂ <a href="#S7.F11" title="Figure 11 â€£ VII Discussion â€£ In-depth Analysis of Privacy Threats in Federated Learning for Medical Data" class="ltx_ref"><span class="ltx_text ltx_ref_tag">11</span></a>).
This further confirms the unique research challenges associated with privacy protection for medical images.</p>
</div>
</section>
<section id="S8" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">VIII </span><span id="S8.1.1" class="ltx_text ltx_font_smallcaps">Conclusion</span>
</h2>

<div id="S8.p1" class="ltx_para">
<p id="S8.p1.1" class="ltx_p">This paper introduces MedPFL, a framework designed to facilitate the analysis and mitigation of privacy risks associated with medical images in the FL environment. We demonstrate the substantial privacy risks inherent in utilizing FL for medical data processing, where sensitive patient data can be susceptible to recovery by adversaries through various privacy attacks. In our study, we employ different levels of random noise as a defense mechanism against these privacy attacks. However, we observe that while higher levels of noise can offer stronger privacy protection, however, adding random noise may not always adequately safeguard medical images within FL environments. Through the experiments of real-world scenarios involving multiple privacy attacks on medical images across three benchmark datasets, we underscore the critical challenges associated with mitigating privacy risks within FL, particularly within the medical domain. In the future, we intend to explore other types of privacy attacks and devise innovative privacy-preserving techniques tailored specifically to safeguarding medical data within FL settings. Also, we plan to incorporate different learning tasks, e.g., medical image segmentation into the MedPFL framework.</p>
</div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">
B.Â C. Das, M.Â H. Amini, and Y.Â Wu, â€œPrivacy risks analysis and mitigation in federated learning for medical images,â€ in <span id="bib.bib1.1.1" class="ltx_text ltx_font_italic">2023 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</span>, pp.Â 1870â€“1873, IEEE, 2023.

</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">
B.Â McMahan, E.Â Moore, D.Â Ramage, S.Â Hampson, and B.Â A. yÂ Arcas, â€œCommunication-efficient learning of deep networks from decentralized data,â€ in <span id="bib.bib2.1.1" class="ltx_text ltx_font_italic">Artificial intelligence and statistics</span>, pp.Â 1273â€“1282, PMLR, 2017.

</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">
L.Â Li, Y.Â Fan, M.Â Tse, and K.-Y. Lin, â€œA review of applications in federated learning,â€ <span id="bib.bib3.1.1" class="ltx_text ltx_font_italic">Computers &amp; Industrial Engineering</span>, vol.Â 149, p.Â 106854, 2020.

</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">
T.Â Zhang and etÂ al., â€œFederated learning for the internet of things: applications, challenges, and opportunities,â€ <span id="bib.bib4.1.1" class="ltx_text ltx_font_italic">IEEE Internet of Things Magazine</span>, vol.Â 5, no.Â 1, pp.Â 24â€“29, 2022.

</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">
T.Â e.Â a. Li, â€œFederated optimization in heterogeneous networks,â€ <span id="bib.bib5.1.1" class="ltx_text ltx_font_italic">Proceedings of Machine learning and systems</span>, vol.Â 2, pp.Â 429â€“450, 2020.

</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">
M.Â Rasouli, T.Â Sun, and R.Â Rajagopal, â€œFedgan: Federated generative adversarial networks for distributed data,â€ <span id="bib.bib6.1.1" class="ltx_text ltx_font_italic">arXiv preprint arXiv:2006.07228</span>, 2020.

</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">
S.Â Kalra, J.Â Wen, J.Â C. Cresswell, M.Â Volkovs, and H.Â R. Tizhoosh, â€œDecentralized federated learning through proxy model sharing,â€ <span id="bib.bib7.1.1" class="ltx_text ltx_font_italic">Nature communications</span>, vol.Â 14, no.Â 1, p.Â 2899, 2023.

</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">
A.Â Qayyum, J.Â Qadir, M.Â Bilal, and A.Â Al-Fuqaha, â€œSecure and robust machine learning for healthcare: A survey,â€ <span id="bib.bib8.1.1" class="ltx_text ltx_font_italic">IEEE Reviews in Biomedical Engineering</span>, vol.Â 14, pp.Â 156â€“180, 2020.

</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">
K.Â Shailaja, B.Â Seetharamulu, and M.Â Jabbar, â€œMachine learning in healthcare: A review,â€ in <span id="bib.bib9.1.1" class="ltx_text ltx_font_italic">2018 Second international conference on electronics, communication and aerospace technology (ICECA)</span>, pp.Â 910â€“914, IEEE, 2018.

</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">
H.Â Habehh and S.Â Gohel, â€œMachine learning in healthcare,â€ <span id="bib.bib10.1.1" class="ltx_text ltx_font_italic">Current genomics</span>, vol.Â 22, no.Â 4, p.Â 291, 2021.

</span>
</li>
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock">
A.Â K. Sahoo, C.Â Pradhan, R.Â K. Barik, and H.Â Dubey, â€œDeepReco: deep learning based health recommender system using collaborative filtering,â€ <span id="bib.bib11.1.1" class="ltx_text ltx_font_italic">Computation</span>, vol.Â 7, no.Â 2, p.Â 25, 2019.

</span>
</li>
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock">
D.Â C. Nguyen, Q.-V. Pham, P.Â N. Pathirana, M.Â Ding, A.Â Seneviratne, Z.Â Lin, O.Â Dobre, and W.-J. Hwang, â€œFederated learning for smart healthcare: A survey,â€ <span id="bib.bib12.1.1" class="ltx_text ltx_font_italic">ACM Computing Surveys (CSUR)</span>, vol.Â 55, no.Â 3, pp.Â 1â€“37, 2022.

</span>
</li>
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock">
V.Â S. Cheng and P.Â C. Hung, â€œHealth insurance portability and accountability act (HIPPA) compliant access control model for web services,â€ <span id="bib.bib13.1.1" class="ltx_text ltx_font_italic">International Journal of Healthcare Information Systems and Informatics (IJHISI)</span>, vol.Â 1, no.Â 1, pp.Â 22â€“39, 2006.

</span>
</li>
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock">
E.Â Goldman, â€œAn introduction to the california consumer privacy act (CCPA),â€ <span id="bib.bib14.1.1" class="ltx_text ltx_font_italic">Santa Clara Univ. Legal Studies Research Paper</span>, 2020.

</span>
</li>
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock">
P.Â Regulation, â€œGeneral data protection regulation,â€ <span id="bib.bib15.1.1" class="ltx_text ltx_font_italic">Intouch</span>, vol.Â 25, pp.Â 1â€“5, 2018.

</span>
</li>
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock">
J.Â Geiping, H.Â Bauermeister, H.Â DrÃ¶ge, and M.Â Moeller, â€œInverting gradients-how easy is it to break privacy in federated learning?,â€ <span id="bib.bib16.1.1" class="ltx_text ltx_font_italic">Advances in Neural Information Processing Systems</span>, vol.Â 33, pp.Â 16937â€“16947, 2020.

</span>
</li>
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock">
L.Â H. Fowl, J.Â Geiping, W.Â Czaja, M.Â Goldblum, and T.Â Goldstein, â€œRobbing the Fed: Directly obtaining private data in federated learning with modified models,â€ in <span id="bib.bib17.1.1" class="ltx_text ltx_font_italic">International Conference on Learning Representations</span>, 2022.

</span>
</li>
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock">
C.Â Zhang, Y.Â Xie, H.Â Bai, B.Â Yu, W.Â Li, and Y.Â Gao, â€œA survey on federated learning,â€ <span id="bib.bib18.1.1" class="ltx_text ltx_font_italic">Knowledge-Based Systems</span>, vol.Â 216, p.Â 106775, 2021.

</span>
</li>
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock">
B.Â Yan and etÂ al., â€œExperiments of federated learning for COVID-19 chest x-ray images,â€ in <span id="bib.bib19.1.1" class="ltx_text ltx_font_italic">Advances in Artificial Intelligence and Security: 7th International Conference, ICAIS 2021, Dublin, Ireland, July 19-23, 2021, Proceedings, Part II 7</span>, pp.Â 41â€“53, Springer, 2021.

</span>
</li>
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock">
M.Â A. Hashmani, S.Â M. Jameel, S.Â S.Â H. Rizvi, and S.Â Shukla, â€œAn adaptive federated machine learning-based intelligent system for skin disease detection: A step toward an intelligent dermoscopy device,â€ <span id="bib.bib20.1.1" class="ltx_text ltx_font_italic">Applied Sciences</span>, vol.Â 11, no.Â 5, p.Â 2145, 2021.

</span>
</li>
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock">
N.Â Truong, K.Â Sun, S.Â Wang, F.Â Guitton, and Y.Â Guo, â€œPrivacy preservation in federated learning: An insightful survey from the gdpr perspective,â€ <span id="bib.bib21.1.1" class="ltx_text ltx_font_italic">Computers &amp; Security</span>, vol.Â 110, p.Â 102402, 2021.

</span>
</li>
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[22]</span>
<span class="ltx_bibblock">
M.Â Ali, F.Â Naeem, M.Â Tariq, and G.Â Kaddoum, â€œFederated learning for privacy preservation in smart healthcare systems: A comprehensive survey,â€ <span id="bib.bib22.1.1" class="ltx_text ltx_font_italic">IEEE journal of biomedical and health informatics</span>, vol.Â 27, no.Â 2, pp.Â 778â€“789, 2022.

</span>
</li>
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[23]</span>
<span class="ltx_bibblock">
A.Â Mileva, L.Â Caviglione, A.Â Velinov, S.Â Wendzel, and V.Â Dimitrova, â€œRisks and opportunities for information hiding in DICOM standard,â€ in <span id="bib.bib23.1.1" class="ltx_text ltx_font_italic">Proceedings of the 16th International Conference on Availability, Reliability and Security</span>, pp.Â 1â€“8, 2021.

</span>
</li>
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[24]</span>
<span class="ltx_bibblock">
M.Â Aiello, G.Â Esposito, G.Â Pagliari, P.Â Borrelli, V.Â Brancato, and M.Â Salvatore, â€œHow does dicom support big data management? investigating its use in medical imaging community,â€ <span id="bib.bib24.1.1" class="ltx_text ltx_font_italic">Insights into Imaging</span>, vol.Â 12, no.Â 1, p.Â 164, 2021.

</span>
</li>
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[25]</span>
<span class="ltx_bibblock">
E.Â Darzidehkalani, M.Â Ghasemi-Rad, and P.Â van Ooijen, â€œFederated learning in medical imaging: Part II: methods, challenges, and considerations,â€ <span id="bib.bib25.1.1" class="ltx_text ltx_font_italic">Journal of the American College of Radiology</span>, vol.Â 19, no.Â 8, pp.Â 975â€“982, 2022.

</span>
</li>
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[26]</span>
<span class="ltx_bibblock">
S.Â G. Finlayson, H.Â W. Chung, I.Â S. Kohane, and A.Â L. Beam, â€œAdversarial attacks against medical deep learning systems,â€ <span id="bib.bib26.1.1" class="ltx_text ltx_font_italic">arXiv preprint arXiv:1804.05296</span>, 2018.

</span>
</li>
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[27]</span>
<span class="ltx_bibblock">
J.Â H. Yoo, H.Â Jeong, J.Â Lee, and T.-M. Chung, â€œOpen problems in medical federated learning,â€ <span id="bib.bib27.1.1" class="ltx_text ltx_font_italic">International Journal of Web Information Systems</span>, no.Â ahead-of-print, 2022.

</span>
</li>
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[28]</span>
<span class="ltx_bibblock">
K.Â D. Toennies, <span id="bib.bib28.1.1" class="ltx_text ltx_font_italic">Guide to medical image analysis</span>.

</span>
<span class="ltx_bibblock">Springer, 2017.

</span>
</li>
<li id="bib.bib29" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[29]</span>
<span class="ltx_bibblock">
V.Â Mothukuri, R.Â M. Parizi, S.Â Pouriyeh, Y.Â Huang, A.Â Dehghantanha, and G.Â Srivastava, â€œA survey on security and privacy of federated learning,â€ <span id="bib.bib29.1.1" class="ltx_text ltx_font_italic">Future Generation Computer Systems</span>, vol.Â 115, pp.Â 619â€“640, 2021.

</span>
</li>
<li id="bib.bib30" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[30]</span>
<span class="ltx_bibblock">
L.Â Lyu and etÂ al., â€œPrivacy and robustness in federated learning: Attacks and defenses,â€ <span id="bib.bib30.1.1" class="ltx_text ltx_font_italic">IEEE transactions on neural networks and learning systems</span>, 2022.

</span>
</li>
<li id="bib.bib31" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[31]</span>
<span class="ltx_bibblock">
J.Â Men and etÂ al., â€œFinding sands in the eyes: vulnerabilities discovery in IoT with eufuzzer on human machine interface,â€ <span id="bib.bib31.1.1" class="ltx_text ltx_font_italic">IEEE Access</span>, vol.Â 7, pp.Â 103751â€“103759, 2019.

</span>
</li>
<li id="bib.bib32" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[32]</span>
<span class="ltx_bibblock">
L.Â MuÃ±oz-GonzÃ¡lez, B.Â Biggio, A.Â Demontis, A.Â Paudice, V.Â Wongrassamee, E.Â C. Lupu, and F.Â Roli, â€œTowards poisoning of deep learning algorithms with back-gradient optimization,â€ in <span id="bib.bib32.1.1" class="ltx_text ltx_font_italic">Proceedings of the 10th ACM workshop on artificial intelligence and security</span>, pp.Â 27â€“38, 2017.

</span>
</li>
<li id="bib.bib33" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[33]</span>
<span class="ltx_bibblock">
V.Â Tolpegin, S.Â Truex, M.Â E. Gursoy, and L.Â Liu, â€œData poisoning attacks against federated learning systems,â€ in <span id="bib.bib33.1.1" class="ltx_text ltx_font_italic">Computer Securityâ€“ESORICS 2020: 25th European Symposium on Research in Computer Security, ESORICS 2020, Guildford, UK, September 14â€“18, 2020, Proceedings, Part I 25</span>, pp.Â 480â€“501, Springer, 2020.

</span>
</li>
<li id="bib.bib34" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[34]</span>
<span class="ltx_bibblock">
A.Â N. Bhagoji, S.Â Chakraborty, P.Â Mittal, and S.Â Calo, â€œModel poisoning attacks in federated learning,â€ in <span id="bib.bib34.1.1" class="ltx_text ltx_font_italic">Proc. Workshop Secur. Mach. Learn.(SecML) 32nd Conf. Neural Inf. Process. Syst.(NeurIPS)</span>, pp.Â 1â€“23, 2018.

</span>
</li>
<li id="bib.bib35" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[35]</span>
<span class="ltx_bibblock">
C.Â Fung, C.Â J. Yoon, and I.Â Beschastnikh, â€œMitigating sybils in federated learning poisoning,â€ <span id="bib.bib35.1.1" class="ltx_text ltx_font_italic">arXiv preprint arXiv:1808.04866</span>, 2018.

</span>
</li>
<li id="bib.bib36" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[36]</span>
<span class="ltx_bibblock">
A.Â N. Bhagoji, S.Â Chakraborty, P.Â Mittal, and S.Â Calo, â€œAnalyzing federated learning through an adversarial lens,â€ in <span id="bib.bib36.1.1" class="ltx_text ltx_font_italic">International Conference on Machine Learning</span>, pp.Â 634â€“643, PMLR, 2019.

</span>
</li>
<li id="bib.bib37" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[37]</span>
<span class="ltx_bibblock">
L.Â Zhu, Z.Â Liu, and S.Â Han, â€œDeep leakage from gradients,â€ <span id="bib.bib37.1.1" class="ltx_text ltx_font_italic">Advances in neural information processing systems</span>, vol.Â 32, 2019.

</span>
</li>
<li id="bib.bib38" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[38]</span>
<span class="ltx_bibblock">
W.Â Wei <span id="bib.bib38.1.1" class="ltx_text ltx_font_italic">etÂ al.</span>, â€œA framework for evaluating client privacy leakages in federated learning,â€ in <span id="bib.bib38.2.2" class="ltx_text ltx_font_italic">Computer Security-25th European Symposium on Research in Computer Security, ESORICS 2020, Guildford, UK, September 14â€“18, 2020, Proceedings, Part I 25</span>, pp.Â 545â€“566, Springer, 2020.

</span>
</li>
<li id="bib.bib39" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[39]</span>
<span class="ltx_bibblock">
W.Â Wei, L.Â Liu, Y.Â Wu, G.Â Su, and A.Â Iyengar, â€œGradient-leakage resilient federated learning,â€ in <span id="bib.bib39.1.1" class="ltx_text ltx_font_italic">2021 IEEE 41st International Conference on Distributed Computing Systems (ICDCS)</span>, pp.Â 797â€“807, IEEE, 2021.

</span>
</li>
<li id="bib.bib40" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[40]</span>
<span class="ltx_bibblock">
P.Â Liu, X.Â Xu, and W.Â Wang, â€œThreats, attacks and defenses to federated learning: issues, taxonomy and perspectives,â€ <span id="bib.bib40.1.1" class="ltx_text ltx_font_italic">Cybersecurity</span>, vol.Â 5, no.Â 1, pp.Â 1â€“19, 2022.

</span>
</li>
<li id="bib.bib41" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[41]</span>
<span class="ltx_bibblock">
M.Â E. Dahlgaard, M.Â W. JÃ¸rgensen, N.Â A. Fuglsang, and H.Â Nassar, â€œAnalysing the influence of attack configurations on the reconstruction of medical images in federated learning,â€ <span id="bib.bib41.1.1" class="ltx_text ltx_font_italic">arXiv preprint arXiv:2204.13808</span>, 2022.

</span>
</li>
<li id="bib.bib42" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[42]</span>
<span class="ltx_bibblock">
W.Â Wei, L.Â Liu, J.Â Zhou, K.-H. Chow, and Y.Â Wu, â€œSecuring distributed SGD against gradient leakage threats,â€ <span id="bib.bib42.1.1" class="ltx_text ltx_font_italic">IEEE Transactions on Parallel and Distributed Systems</span>, vol.Â 34, no.Â 7, pp.Â 2040â€“2054, 2023.

</span>
</li>
<li id="bib.bib43" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[43]</span>
<span class="ltx_bibblock">
W.Â Wei, K.-H. Chow, F.Â Ilhan, Y.Â Wu, and L.Â Liu, â€œModel cloaking against gradient leakage,â€ in <span id="bib.bib43.1.1" class="ltx_text ltx_font_italic">2023 IEEE International Conference on Data Mining (ICDM)</span>, pp.Â 1403â€“1408, Dec 2023.

</span>
</li>
<li id="bib.bib44" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[44]</span>
<span class="ltx_bibblock">
B.Â Zhao, K.Â R. Mopuri, and H.Â Bilen, â€œiDLG: Improved deep leakage from gradients,â€ <span id="bib.bib44.1.1" class="ltx_text ltx_font_italic">arXiv preprint arXiv:2001.02610</span>, 2020.

</span>
</li>
<li id="bib.bib45" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[45]</span>
<span class="ltx_bibblock">
M.Â Abadi, A.Â Chu, I.Â Goodfellow, H.Â B. McMahan, I.Â Mironov, K.Â Talwar, and L.Â Zhang, â€œDeep learning with differential privacy,â€ in <span id="bib.bib45.1.1" class="ltx_text ltx_font_italic">Proceedings of the 2016 ACM SIGSAC conference on computer and communications security</span>, pp.Â 308â€“318, 2016.

</span>
</li>
<li id="bib.bib46" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[46]</span>
<span class="ltx_bibblock">
L.Â Melis, G.Â Danezis, and E.Â DeÂ Cristofaro, â€œEfficient private statistics with succinct sketches,â€ <span id="bib.bib46.1.1" class="ltx_text ltx_font_italic">arXiv preprint arXiv:1508.06110</span>, 2015.

</span>
</li>
<li id="bib.bib47" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[47]</span>
<span class="ltx_bibblock">
G.Â A. Kaissis, M.Â R. Makowski, D.Â RÃ¼ckert, and R.Â F. Braren, â€œSecure, privacy-preserving and federated machine learning in medical imaging,â€ <span id="bib.bib47.1.1" class="ltx_text ltx_font_italic">Nature Machine Intelligence</span>, vol.Â 2, no.Â 6, pp.Â 305â€“311, 2020.

</span>
</li>
<li id="bib.bib48" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[48]</span>
<span class="ltx_bibblock">
G.Â Kaissis, A.Â Ziller, J.Â Passerat-Palmbach, T.Â Ryffel, D.Â Usynin, A.Â Trask, I.Â LimaÂ Jr, J.Â Mancuso, F.Â Jungmann, M.-M. Steinborn, <span id="bib.bib48.1.1" class="ltx_text ltx_font_italic">etÂ al.</span>, â€œEnd-to-end privacy preserving deep learning on multi-institutional medical imaging,â€ <span id="bib.bib48.2.2" class="ltx_text ltx_font_italic">Nature Machine Intelligence</span>, vol.Â 3, no.Â 6, pp.Â 473â€“484, 2021.

</span>
</li>
<li id="bib.bib49" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[49]</span>
<span class="ltx_bibblock">
X.Â Shen, H.Â Jiang, Y.Â Chen, B.Â Wang, and L.Â Gao, â€œPLDP-FL: Federated learning with personalized local differential privacy,â€ <span id="bib.bib49.1.1" class="ltx_text ltx_font_italic">Entropy</span>, vol.Â 25, no.Â 3, p.Â 485, 2023.

</span>
</li>
<li id="bib.bib50" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[50]</span>
<span class="ltx_bibblock">
W.Â Liu, Y.Â Zhang, H.Â Yang, and Q.Â Meng, â€œA survey on differential privacy for medical data analysis,â€ <span id="bib.bib50.1.1" class="ltx_text ltx_font_italic">Annals of Data Science</span>, pp.Â 1â€“15, 2023.

</span>
</li>
<li id="bib.bib51" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[51]</span>
<span class="ltx_bibblock">
M.Â Adnan, S.Â Kalra, J.Â C. Cresswell, G.Â W. Taylor, and H.Â R. Tizhoosh, â€œFederated learning and differential privacy for medical image analysis,â€ <span id="bib.bib51.1.1" class="ltx_text ltx_font_italic">Scientific reports</span>, vol.Â 12, no.Â 1, p.Â 1953, 2022.

</span>
</li>
<li id="bib.bib52" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[52]</span>
<span class="ltx_bibblock">
O.Â Aouedi, A.Â Sacco, K.Â Piamrat, and G.Â Marchetto, â€œHandling privacy-sensitive medical data with federated learning: challenges and future directions,â€ <span id="bib.bib52.1.1" class="ltx_text ltx_font_italic">IEEE Journal of Biomedical and Health Informatics</span>, vol.Â 27, no.Â 2, pp.Â 790â€“803, 2022.

</span>
</li>
<li id="bib.bib53" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[53]</span>
<span class="ltx_bibblock">
M.Â H. Javid, â€œMelanoma skin cancer dataset of 10000 images.â€ <a target="_blank" href="https://www.kaggle.com/dsv/3376422" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.kaggle.com/dsv/3376422</a>, 2022.

</span>
</li>
<li id="bib.bib54" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[54]</span>
<span class="ltx_bibblock">
â€œCovid-19 image dataset.â€ <a href="www.kaggle.com/datasets/pranavraikokte/covid19-image-dataset" title="" class="ltx_ref ltx_url ltx_font_typewriter">www.kaggle.com/datasets/pranavraikokte/covid19-image-dataset</a>, 2020.

</span>
</li>
<li id="bib.bib55" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[55]</span>
<span class="ltx_bibblock">
M.Â Nickparvar, â€œBrain tumor mri dataset.â€ <a target="_blank" href="https://www.kaggle.com/dsv/2645886" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.kaggle.com/dsv/2645886</a>, 2021.

</span>
</li>
<li id="bib.bib56" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[56]</span>
<span class="ltx_bibblock">
Y.Â LeCun, L.Â Bottou, Y.Â Bengio, and P.Â Haffner, â€œGradient-based learning applied to document recognition,â€ <span id="bib.bib56.1.1" class="ltx_text ltx_font_italic">Proceedings of the IEEE</span>, vol.Â 86, no.Â 11, pp.Â 2278â€“2324, 1998.

</span>
</li>
<li id="bib.bib57" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[57]</span>
<span class="ltx_bibblock">
K.Â He, X.Â Zhang, S.Â Ren, and J.Â Sun, â€œDeep residual learning for image recognition,â€ in <span id="bib.bib57.1.1" class="ltx_text ltx_font_italic">Proceedings of the IEEE conference on computer vision and pattern recognition</span>, pp.Â 770â€“778, 2016.

</span>
</li>
<li id="bib.bib58" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[58]</span>
<span class="ltx_bibblock">
O.Â Ronneberger, P.Â Fischer, and T.Â Brox, â€œU-net: Convolutional networks for biomedical image segmentation,â€ in <span id="bib.bib58.1.1" class="ltx_text ltx_font_italic">Medical Image Computing and Computer-Assisted Interventionâ€“MICCAI 2015: 18th International Conference, Munich, Germany, October 5-9, 2015, Proceedings, Part III 18</span>, pp.Â 234â€“241, Springer, 2015.

</span>
</li>
<li id="bib.bib59" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[59]</span>
<span class="ltx_bibblock">
O.Â Goldreich, â€œSecure multi-party computation,â€ <span id="bib.bib59.1.1" class="ltx_text ltx_font_italic">Manuscript. Preliminary version</span>, vol.Â 78, no.Â 110, 1998.

</span>
</li>
<li id="bib.bib60" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[60]</span>
<span class="ltx_bibblock">
H.Â L. Tan <span id="bib.bib60.1.1" class="ltx_text ltx_font_italic">etÂ al.</span>, â€œA perceptually relevant MSE-based image quality metric,â€ <span id="bib.bib60.2.2" class="ltx_text ltx_font_italic">IEEE Transactions on Image Processing</span>, vol.Â 22, no.Â 11, pp.Â 4447â€“4459, 2013.

</span>
</li>
<li id="bib.bib61" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[61]</span>
<span class="ltx_bibblock">
Z.Â Wang, A.Â C. Bovik, H.Â R. Sheikh, and E.Â P. Simoncelli, â€œImage quality assessment: from error visibility to structural similarity,â€ <span id="bib.bib61.1.1" class="ltx_text ltx_font_italic">IEEE transactions on image processing</span>, vol.Â 13, no.Â 4, pp.Â 600â€“612, 2004.

</span>
</li>
<li id="bib.bib62" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[62]</span>
<span class="ltx_bibblock">
M.Â Du, â€œFederated learning.â€ <a target="_blank" href="https://inst.eecs.berkeley.edu/~cs294-163/fa19/slides/federated-learning.pdf" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://inst.eecs.berkeley.edu/~cs294-163/fa19/slides/federated-learning.pdf</a>, 2019.

</span>
</li>
<li id="bib.bib63" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[63]</span>
<span class="ltx_bibblock">
D.Â Cynthia, â€œDifferential privacy in automata, languages and programming, bugliesi michele, preneel bart, sassone vladimiro, and wegener ingo,â€ 2006.

</span>
</li>
<li id="bib.bib64" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[64]</span>
<span class="ltx_bibblock">
J.Â Deng, W.Â Dong, R.Â Socher, L.-J. Li, K.Â Li, and L.Â Fei-Fei, â€œImagenet: A large-scale hierarchical image database,â€ in <span id="bib.bib64.1.1" class="ltx_text ltx_font_italic">2009 IEEE conference on computer vision and pattern recognition</span>, pp.Â 248â€“255, Ieee, 2009.

</span>
</li>
<li id="bib.bib65" class="ltx_bibitem">
<span class="ltx_tag ltx_tag_bibitem">[65]</span>
<span class="ltx_bibblock">
A.Â Krizhevsky, â€œCIFAR-10 (canadian institute for advanced research),â€ tech. rep., Canadian Institute For Advanced Research, 2009.

</span>
</li>
</ul>
</section>
<div class="ltx_pagination ltx_role_newpage"></div>
<figure id="id1" class="ltx_float biography">
<table id="id1.1" class="ltx_tabular">
<tr id="id1.1.1" class="ltx_tr">
<td id="id1.1.1.1" class="ltx_td"><img src="/html/2409.18907/assets/x13.png" id="id1.1.1.1.g1" class="ltx_graphics ltx_img_portrait" width="75" height="96" alt="[Uncaptioned image]"></td>
<td id="id1.1.1.2" class="ltx_td">
<span id="id1.1.1.2.1" class="ltx_inline-block">
<span id="id1.1.1.2.1.1" class="ltx_p"><span id="id1.1.1.2.1.1.1" class="ltx_text ltx_font_bold">Badhan Chandra Das</span>  is a Ph.D. Candidate at Knight Foundation School of Computing and Information Sciences (KFSCIS), Florida International University (FIU). His research includes privacy-preserving machine learning algorithms, computer vision, and their real-world applications.</span>
</span>
</td>
</tr>
</table>
</figure>
<figure id="id2" class="ltx_float biography">
<table id="id2.1" class="ltx_tabular">
<tr id="id2.1.1" class="ltx_tr">
<td id="id2.1.1.1" class="ltx_td"><img src="/html/2409.18907/assets/x14.png" id="id2.1.1.1.g1" class="ltx_graphics ltx_img_portrait" width="77" height="96" alt="[Uncaptioned image]"></td>
<td id="id2.1.1.2" class="ltx_td">
<span id="id2.1.1.2.1" class="ltx_inline-block">
<span id="id2.1.1.2.1.1" class="ltx_p"><span id="id2.1.1.2.1.1.1" class="ltx_text ltx_font_bold">M. Hadi Amini, Senior Member, IEEE</span>  is an Assistant Professor at Knight Foundation School of Computing and Information Sciences at Florida International University. He is the director of Sustainability, Optimization, and Learning for InterDependent networks laboratory (www.solidlab.network). He received his Ph.D. in Electrical and Computer Engineering from Carnegie Mellon University in 2019, where he received his M.Sc. degree in 2015. He also holds a doctoral degree in Computer Science and Technology. He serves as the Director and PI of ADvanced education and research for Machine learning-driven critical Infrastructure REsilience (ADMIRE) Center, Supported by the U.S. DHS; and
Associate Director of the National Center for Transportation Cybersecurity and Resiliency (TraCR), Supported by the U.S. DOT. He is an Associate Editor of <span id="id2.1.1.2.1.1.2" class="ltx_text ltx_font_italic">IEEE Transactions on Information Forensics and Security</span>. His research interests include secure and privacy-preserving distributed optimization and learning algorithms, interdependent networks, and cyber-physical-social security and resilience.</span>
</span>
</td>
</tr>
</table>
</figure>
<figure id="id3" class="ltx_float biography">
<table id="id3.1" class="ltx_tabular">
<tr id="id3.1.1" class="ltx_tr">
<td id="id3.1.1.1" class="ltx_td"><img src="/html/2409.18907/assets/x15.png" id="id3.1.1.1.g1" class="ltx_graphics ltx_img_portrait" width="72" height="96" alt="[Uncaptioned image]"></td>
<td id="id3.1.1.2" class="ltx_td">
<span id="id3.1.1.2.1" class="ltx_inline-block">
<span id="id3.1.1.2.1.1" class="ltx_p"><span id="id3.1.1.2.1.1.1" class="ltx_text ltx_font_bold">Yanzhao Wu</span>  is an Assistant Professor in the Knight Foundation School of Computing and Information Sciences (KFSCIS) at Florida International University (FIU). He obtained his Bachelorâ€™s degree from University of Science and Technology of China (USTC) in 2017 and then received his PhD in Computer Science from Georgia Institute of Technology in 2022. His research interests are primarily centered on the intersection of machine learning and computing systems, including machine learning algorithm and system co-design, large language models (LLMs), edge AI, privacy-preserving machine learning, deep learning, big data analytics, and their real-world applications.</span>
</span>
</td>
</tr>
</table>
</figure>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/2409.18906" class="ar5iv-nav-button ar5iv-nav-button-prev">â—„</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/2409.18907" class="ar5iv-text-button ar5iv-severity-ok">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+2409.18907">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/2409.18907" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/2409.18908" class="ar5iv-nav-button ar5iv-nav-button-next">â–º</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Sat Oct  5 19:24:20 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "Ã—";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
