<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[2409.09727] From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/2409.09727">

<!--Generated on Sat Oct  5 21:39:27 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.8.0.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<h1 class="ltx_title ltx_title_document">From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Ming Li
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Bioengineering Department and Imperial-X, Imperial College London, London W12 7SL, UK
</span>
<span class="ltx_contact ltx_role_affiliation">National Heart and Lung Institute, Imperial College London, London SW7 2AZ, UK
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Pengcheng Xu
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Athinoula A. Martinos Center for Biomedical Imaging, Massachusetts General Hospital, Boston, Massachusetts, USA
</span>
<span class="ltx_contact ltx_role_affiliation">State Key Laboratory of Extreme Photonics and Instrumentation, College of Optical Science and Engineering, Zhejiang University, Hangzhou, China
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Junjie Hu
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">National Heart and Lung Institute, Imperial College London, London SW7 2AZ, UK
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Zeyu Tang
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Bioengineering Department and Imperial-X, Imperial College London, London W12 7SL, UK
</span>
<span class="ltx_contact ltx_role_affiliation">Tri-Institutional Computational Biology &amp; Medicine Program, Weill Cornell Medicine of Cornell University, New York, USA
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Guang Yang
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Bioengineering Department and Imperial-X, Imperial College London, London W12 7SL, UK
</span>
<span class="ltx_contact ltx_role_affiliation">National Heart and Lung Institute, Imperial College London, London SW7 2AZ, UK
</span>
<span class="ltx_contact ltx_role_affiliation">Cardiovascular Research Centre, Royal Brompton Hospital, London SW3 6NP, UK
</span>
<span class="ltx_contact ltx_role_affiliation">School of Biomedical Engineering &amp; Imaging Sciences, Kingâ€™s College London, London WC2R 2LS, UK
</span>
<span class="ltx_contact ltx_role_affiliation">Correspondence: <span id="id1.1.id1" class="ltx_text ltx_font_typewriter">g.yang@imperial.ac.uk</span>
</span></span></span>
</div>

<section id="Sx1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Abstract</h2>

<div id="Sx1.p1" class="ltx_para">
<p id="Sx1.p1.1" class="ltx_p">Federated learning holds great potential for enabling large-scale healthcare research and collaboration across multiple centres while ensuring data privacy and security are not compromised.
Although numerous recent studies suggest or utilize federated learning based methods in healthcare, it remains unclear which ones have potential clinical utility.
This review paper considers and analyzes the most recent studies up to May 2024 that describe federated learning based methods in healthcare.
After a thorough review, we find that the vast majority are not appropriate for clinical use due to their methodological flaws and/or underlying biases which include but are not limited to privacy concerns, generalization issues, and communication costs.
As a result, the effectiveness of federated learning in healthcare is significantly compromised.
To overcome these challenges, we provide recommendations and promising opportunities that might be implemented to resolve these problems and improve the quality of model development in federated learning with healthcare.</p>
</div>
</section>
<section id="Sx2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Keywords</h2>

<div id="Sx2.p1" class="ltx_para">
<p id="Sx2.p1.1" class="ltx_p">Federated Learning, Healthcare, Pitfalls, Challenges, Recommendations, Opportunities.</p>
</div>
</section>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>

<div id="S1.p1" class="ltx_para">
<p id="S1.p1.1" class="ltx_p">The integration of Artificial Intelligence (AI) into healthcare research has started a transformative era, catalyzing unprecedented advancements in patient care, diagnostic precision, and therapeutic efficacyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib1" title="" class="ltx_ref">1</a></sup></cite>.
However, developing robust AI models requires a vast amount of multi-centre data.
A notable example is the genome-wide association studies, when confined to data from a single institution, are often limited by sample size, failing to identify established biomarkersÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib2" title="" class="ltx_ref">2</a></sup></cite>.
This underscores the imperative for collaborative data sharing among institutions.
Standard AI approaches rely on centralized datasets for model training, but in healthcare, centralization is complex due to various factors such as privacy concerns, regulatory constraints, as well as legal, ethical and technological barriers to data sharingÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib3" title="" class="ltx_ref">3</a></sup></cite>.</p>
</div>
<div id="S1.p2" class="ltx_para">
<p id="S1.p2.1" class="ltx_p">Federated Learning (FL) emerges as a revolutionary paradigm, promising the collaborative training of AI models across distributed datasets without data sharingÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib4" title="" class="ltx_ref">4</a></sup></cite>.
By enabling privacy-preserving data analysis across multiple data silos, FL can exploit the full potential of worldwide healthcare data across different demographics, unlocking insights unattainable by isolated institutions.
Models trained in a federated fashion are potentially able to yield even less biased decisions and higher sensitivity to rare cases as they are exposed to a more complete data distribution.
Recent studies have shown that models trained by FL can achieve performance comparable to the ones trained on centrally hosted datasets and superior to models that only see isolated single-institutional dataÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib5" title="" class="ltx_ref">5</a></sup>, <sup class="ltx_sup"><a href="#bib.bib6" title="" class="ltx_ref">6</a></sup></cite>.
Notably, early studies into FL, particularly in areas like brain tumourÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib6" title="" class="ltx_ref">6</a></sup></cite>, triple negative breast cancerÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib7" title="" class="ltx_ref">7</a></sup></cite> and COVID-19Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib8" title="" class="ltx_ref">8</a></sup></cite>, also have begun to illustrate the potential for generalizability beyond a single institution.</p>
</div>
<div id="S1.p3" class="ltx_para">
<p id="S1.p3.1" class="ltx_p">Todayâ€™s pioneering large-scale initiatives span academic research, clinical applications, and industrial translations, collectively advancing FL in healthcare.
<span id="S1.p3.1.1" class="ltx_text ltx_font_italic">Within academic research,</span> consortia such as Trustworthy Federated Data Analytics (TFDA)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib9" title="" class="ltx_ref">9</a></sup></cite> and theÂ <cite class="ltx_cite ltx_citemacro_cite"><sup id="S1.p3.1.2.1" class="ltx_sup"><a href="#bib.bib10" title="" class="ltx_ref">10</a></sup></cite> spearhead decentralized research across institutions. An illustrative example is the international collaboration employing FL to develop AI models for mammogram assessment, which outperformed single-institutional models and exhibited enhanced generalizabilityÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib11" title="" class="ltx_ref">11</a></sup></cite>.
<span id="S1.p3.1.3" class="ltx_text ltx_font_italic">Moving to clinical applications,</span> projects like HealthChainÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib12" title="" class="ltx_ref">12</a></sup></cite> and DRAGONÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib13" title="" class="ltx_ref">13</a></sup></cite> aim to deploy FL across multiple hospitals in Europe, facilitating the prediction of treatment responses for cancer and COVID-19. By aiding clinicians in treatment decisions based on histology slides and CT images, FL demonstrated direct clinical impact. Another large scale project is the Federated Tumour Segmentation (FeTS) initiativeÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib14" title="" class="ltx_ref">14</a></sup></cite>, which involves 30 institutions globally, that utilize FL to improve tumour boundary detection across various cancers.
<span id="S1.p3.1.4" class="ltx_text ltx_font_italic">In the industrial domain,</span> collaborative efforts likeÂ <cite class="ltx_cite ltx_citemacro_cite"><sup id="S1.p3.1.5.1" class="ltx_sup"><a href="#bib.bib15" title="" class="ltx_ref">15</a></sup></cite> demonstrate how competing companies can optimize the drug discovery process through multi-task FL while protecting their proprietary data.</p>
</div>
<div id="S1.p4" class="ltx_para">
<p id="S1.p4.1" class="ltx_p">Despite FLâ€™s promising advantages, integrating it within healthcare still faces methodological flaws and underlying biases.
These encompass but are not limited to, addressing privacy concernsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib8" title="" class="ltx_ref">8</a></sup>, <sup class="ltx_sup"><a href="#bib.bib16" title="" class="ltx_ref">16</a></sup></cite>, generalization issuesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib17" title="" class="ltx_ref">17</a></sup></cite>, communication costsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib18" title="" class="ltx_ref">18</a></sup></cite>, and the non-independent and identically distributed (non-IID) nature of healthcare data across institutionsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib19" title="" class="ltx_ref">19</a></sup></cite>, safeguarding patient data against sophisticated inference attacks that could potentially deanonymize sensitive information from model updatesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib20" title="" class="ltx_ref">20</a></sup></cite>, and the necessity for standardization across FL implementations.
Moreover, thereâ€™s a pressing need for models that not only exhibit robust performance across diverse datasets but are also interpretable and transparent in their predictions and decision-making processesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib21" title="" class="ltx_ref">21</a></sup>, <sup class="ltx_sup"><a href="#bib.bib22" title="" class="ltx_ref">22</a></sup></cite>.</p>
</div>
<div id="S1.p5" class="ltx_para">
<p id="S1.p5.1" class="ltx_p">To facilitate the implementation of FL in healthcare, we have considered and analyzed the most recent studies, delving into the practical application of FL in healthcare.
We provide numerous recommendations and promising opportunities, which, if followed appropriately, might be able to mitigate current pitfalls and challenges, ultimately leading to high-quality development and reliable reporting of results in FL with healthcare.
Our review makes contributions as follows:</p>
<ul id="S1.I1" class="ltx_itemize">
<li id="S1.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S1.I1.i1.p1" class="ltx_para">
<p id="S1.I1.i1.p1.1" class="ltx_p">Quantifying and evaluating the integrity and variation of most recent and advanced FL technologies in healthcare to identify challenges, flaws and pitfalls;</p>
</div>
</li>
<li id="S1.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S1.I1.i2.p1" class="ltx_para">
<p id="S1.I1.i2.p1.1" class="ltx_p">Providing a taxonomized, in-depth analysis and discussion of various aspects of FL within healthcare;</p>
</div>
</li>
<li id="S1.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S1.I1.i3.p1" class="ltx_para">
<p id="S1.I1.i3.p1.1" class="ltx_p">Offering evidence-based guidelines and recommendations to enhance the quality of FL development, ensuring fair and reproducible comparisons of FL strategies, while also identifying emerging trends and suggesting future opportunities for improving patient outcomes and streamlining clinical workflows.</p>
</div>
</li>
</ul>
</div>
<div id="S1.p6" class="ltx_para">
<p id="S1.p6.1" class="ltx_p">The rest of this review is organized as follows.
<span id="S1.p6.1.1" class="ltx_text ltx_font_italic">Section 2</span> offers the background and preliminaries of FL.
<span id="S1.p6.1.2" class="ltx_text ltx_font_italic">Section 3</span> presents the screening procedure of this review.
<span id="S1.p6.1.3" class="ltx_text ltx_font_italic">Section 4</span> summarizes the key results of this review.
<span id="S1.p6.1.4" class="ltx_text ltx_font_italic">Section 5</span> discusses the most recent advances, challenges and pitfalls for implementing FL in healthcare, providing recommended practices to avoid current pitfalls as well as outlining potential future research directions and opportunities.</p>
</div>
<figure id="alg1" class="ltx_float ltx_minipage ltx_align_center ltx_align_middle ltx_framed ltx_framed_top" style="width:303.5pt;">

<figcaption class="ltx_caption" style="font-size:70%;"><span class="ltx_tag ltx_tag_float"><span id="alg1.24.1.1" class="ltx_text ltx_font_bold">Algorithm 1</span> </span> FL with FedAvg Training process. <math id="alg1.7.m1.1" class="ltx_Math" alttext="K" display="inline"><semantics id="alg1.7.m1.1b"><mi id="alg1.7.m1.1.1" xref="alg1.7.m1.1.1.cmml">K</mi><annotation-xml encoding="MathML-Content" id="alg1.7.m1.1c"><ci id="alg1.7.m1.1.1.cmml" xref="alg1.7.m1.1.1">ğ¾</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.7.m1.1d">K</annotation></semantics></math> clients are indexed by <math id="alg1.8.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="alg1.8.m2.1b"><mi id="alg1.8.m2.1.1" xref="alg1.8.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="alg1.8.m2.1c"><ci id="alg1.8.m2.1.1.cmml" xref="alg1.8.m2.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.8.m2.1d">k</annotation></semantics></math>; <math id="alg1.9.m3.1" class="ltx_Math" alttext="C" display="inline"><semantics id="alg1.9.m3.1b"><mi id="alg1.9.m3.1.1" xref="alg1.9.m3.1.1.cmml">C</mi><annotation-xml encoding="MathML-Content" id="alg1.9.m3.1c"><ci id="alg1.9.m3.1.1.cmml" xref="alg1.9.m3.1.1">ğ¶</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.9.m3.1d">C</annotation></semantics></math>-fractions of clients are selected at each round; <math id="alg1.10.m4.1" class="ltx_Math" alttext="E" display="inline"><semantics id="alg1.10.m4.1b"><mi id="alg1.10.m4.1.1" xref="alg1.10.m4.1.1.cmml">E</mi><annotation-xml encoding="MathML-Content" id="alg1.10.m4.1c"><ci id="alg1.10.m4.1.1.cmml" xref="alg1.10.m4.1.1">ğ¸</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.10.m4.1d">E</annotation></semantics></math> is the number of local epochs; <math id="alg1.11.m5.1" class="ltx_Math" alttext="B" display="inline"><semantics id="alg1.11.m5.1b"><mi id="alg1.11.m5.1.1" xref="alg1.11.m5.1.1.cmml">B</mi><annotation-xml encoding="MathML-Content" id="alg1.11.m5.1c"><ci id="alg1.11.m5.1.1.cmml" xref="alg1.11.m5.1.1">ğµ</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.11.m5.1d">B</annotation></semantics></math> is the local mini-batch size; <math id="alg1.12.m6.1" class="ltx_Math" alttext="\eta" display="inline"><semantics id="alg1.12.m6.1b"><mi id="alg1.12.m6.1.1" xref="alg1.12.m6.1.1.cmml">Î·</mi><annotation-xml encoding="MathML-Content" id="alg1.12.m6.1c"><ci id="alg1.12.m6.1.1.cmml" xref="alg1.12.m6.1.1">ğœ‚</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.12.m6.1d">\eta</annotation></semantics></math> is the learning rate.</figcaption><div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<p id="alg1.25" class="ltx_p ltx_figure_panel"><span id="alg1.25.1" class="ltx_text ltx_font_italic" style="font-size:70%;">ServerExecutes:<span id="alg1.25.1.1" class="ltx_text ltx_font_upright"></span></span></p>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<div id="alg1.26" class="ltx_listing ltx_figure_panel ltx_listing">
<div id="alg1.l1" class="ltx_listingline">
<span id="alg1.l1.1" class="ltx_text" style="font-size:70%;">â€„initialize the parameters of model </span><math id="alg1.l1.m1.1" class="ltx_Math" alttext="\theta_{0}" display="inline"><semantics id="alg1.l1.m1.1a"><msub id="alg1.l1.m1.1.1" xref="alg1.l1.m1.1.1.cmml"><mi mathsize="70%" id="alg1.l1.m1.1.1.2" xref="alg1.l1.m1.1.1.2.cmml">Î¸</mi><mn mathsize="70%" id="alg1.l1.m1.1.1.3" xref="alg1.l1.m1.1.1.3.cmml">0</mn></msub><annotation-xml encoding="MathML-Content" id="alg1.l1.m1.1b"><apply id="alg1.l1.m1.1.1.cmml" xref="alg1.l1.m1.1.1"><csymbol cd="ambiguous" id="alg1.l1.m1.1.1.1.cmml" xref="alg1.l1.m1.1.1">subscript</csymbol><ci id="alg1.l1.m1.1.1.2.cmml" xref="alg1.l1.m1.1.1.2">ğœƒ</ci><cn type="integer" id="alg1.l1.m1.1.1.3.cmml" xref="alg1.l1.m1.1.1.3">0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1.m1.1c">\theta_{0}</annotation></semantics></math><span id="alg1.l1.2" class="ltx_text" style="font-size:70%;">
</span>
</div>
<div id="alg1.l2" class="ltx_listingline">
<span id="alg1.l2.1" class="ltx_text" style="font-size:70%;">â€„</span><span id="alg1.l2.2" class="ltx_text ltx_font_bold" style="font-size:70%;">for</span><span id="alg1.l2.3" class="ltx_text" style="font-size:70%;">Â each round </span><math id="alg1.l2.m1.4" class="ltx_Math" alttext="t=1,2,...,T" display="inline"><semantics id="alg1.l2.m1.4a"><mrow id="alg1.l2.m1.4.5" xref="alg1.l2.m1.4.5.cmml"><mi mathsize="70%" id="alg1.l2.m1.4.5.2" xref="alg1.l2.m1.4.5.2.cmml">t</mi><mo mathsize="70%" id="alg1.l2.m1.4.5.1" xref="alg1.l2.m1.4.5.1.cmml">=</mo><mrow id="alg1.l2.m1.4.5.3.2" xref="alg1.l2.m1.4.5.3.1.cmml"><mn mathsize="70%" id="alg1.l2.m1.1.1" xref="alg1.l2.m1.1.1.cmml">1</mn><mo mathsize="70%" id="alg1.l2.m1.4.5.3.2.1" xref="alg1.l2.m1.4.5.3.1.cmml">,</mo><mn mathsize="70%" id="alg1.l2.m1.2.2" xref="alg1.l2.m1.2.2.cmml">2</mn><mo mathsize="70%" id="alg1.l2.m1.4.5.3.2.2" xref="alg1.l2.m1.4.5.3.1.cmml">,</mo><mi mathsize="70%" mathvariant="normal" id="alg1.l2.m1.3.3" xref="alg1.l2.m1.3.3.cmml">â€¦</mi><mo mathsize="70%" id="alg1.l2.m1.4.5.3.2.3" xref="alg1.l2.m1.4.5.3.1.cmml">,</mo><mi mathsize="70%" id="alg1.l2.m1.4.4" xref="alg1.l2.m1.4.4.cmml">T</mi></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l2.m1.4b"><apply id="alg1.l2.m1.4.5.cmml" xref="alg1.l2.m1.4.5"><eq id="alg1.l2.m1.4.5.1.cmml" xref="alg1.l2.m1.4.5.1"></eq><ci id="alg1.l2.m1.4.5.2.cmml" xref="alg1.l2.m1.4.5.2">ğ‘¡</ci><list id="alg1.l2.m1.4.5.3.1.cmml" xref="alg1.l2.m1.4.5.3.2"><cn type="integer" id="alg1.l2.m1.1.1.cmml" xref="alg1.l2.m1.1.1">1</cn><cn type="integer" id="alg1.l2.m1.2.2.cmml" xref="alg1.l2.m1.2.2">2</cn><ci id="alg1.l2.m1.3.3.cmml" xref="alg1.l2.m1.3.3">â€¦</ci><ci id="alg1.l2.m1.4.4.cmml" xref="alg1.l2.m1.4.4">ğ‘‡</ci></list></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l2.m1.4c">t=1,2,...,T</annotation></semantics></math><span id="alg1.l2.4" class="ltx_text" style="font-size:70%;">Â </span><span id="alg1.l2.5" class="ltx_text ltx_font_bold" style="font-size:70%;">do</span><span id="alg1.l2.6" class="ltx_text" style="font-size:70%;">


</span>
</div>
<div id="alg1.l3" class="ltx_listingline">
<span id="alg1.l3.1" class="ltx_text" style="font-size:70%;">Â Â Â Â </span><math id="alg1.l3.m1.1" class="ltx_Math" alttext="m\leftarrow" display="inline"><semantics id="alg1.l3.m1.1a"><mrow id="alg1.l3.m1.1.1" xref="alg1.l3.m1.1.1.cmml"><mi mathsize="70%" id="alg1.l3.m1.1.1.2" xref="alg1.l3.m1.1.1.2.cmml">m</mi><mo mathsize="70%" stretchy="false" id="alg1.l3.m1.1.1.1" xref="alg1.l3.m1.1.1.1.cmml">â†</mo><mi id="alg1.l3.m1.1.1.3" xref="alg1.l3.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg1.l3.m1.1b"><apply id="alg1.l3.m1.1.1.cmml" xref="alg1.l3.m1.1.1"><ci id="alg1.l3.m1.1.1.1.cmml" xref="alg1.l3.m1.1.1.1">â†</ci><ci id="alg1.l3.m1.1.1.2.cmml" xref="alg1.l3.m1.1.1.2">ğ‘š</ci><csymbol cd="latexml" id="alg1.l3.m1.1.1.3.cmml" xref="alg1.l3.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l3.m1.1c">m\leftarrow</annotation></semantics></math><span id="alg1.l3.2" class="ltx_text" style="font-size:70%;">max</span><math id="alg1.l3.m2.2" class="ltx_Math" alttext="(C\cdot K,1)" display="inline"><semantics id="alg1.l3.m2.2a"><mrow id="alg1.l3.m2.2.2.1" xref="alg1.l3.m2.2.2.2.cmml"><mo maxsize="70%" minsize="70%" id="alg1.l3.m2.2.2.1.2" xref="alg1.l3.m2.2.2.2.cmml">(</mo><mrow id="alg1.l3.m2.2.2.1.1" xref="alg1.l3.m2.2.2.1.1.cmml"><mi mathsize="70%" id="alg1.l3.m2.2.2.1.1.2" xref="alg1.l3.m2.2.2.1.1.2.cmml">C</mi><mo lspace="0.222em" mathsize="70%" rspace="0.222em" id="alg1.l3.m2.2.2.1.1.1" xref="alg1.l3.m2.2.2.1.1.1.cmml">â‹…</mo><mi mathsize="70%" id="alg1.l3.m2.2.2.1.1.3" xref="alg1.l3.m2.2.2.1.1.3.cmml">K</mi></mrow><mo mathsize="70%" id="alg1.l3.m2.2.2.1.3" xref="alg1.l3.m2.2.2.2.cmml">,</mo><mn mathsize="70%" id="alg1.l3.m2.1.1" xref="alg1.l3.m2.1.1.cmml">1</mn><mo maxsize="70%" minsize="70%" id="alg1.l3.m2.2.2.1.4" xref="alg1.l3.m2.2.2.2.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="alg1.l3.m2.2b"><interval closure="open" id="alg1.l3.m2.2.2.2.cmml" xref="alg1.l3.m2.2.2.1"><apply id="alg1.l3.m2.2.2.1.1.cmml" xref="alg1.l3.m2.2.2.1.1"><ci id="alg1.l3.m2.2.2.1.1.1.cmml" xref="alg1.l3.m2.2.2.1.1.1">â‹…</ci><ci id="alg1.l3.m2.2.2.1.1.2.cmml" xref="alg1.l3.m2.2.2.1.1.2">ğ¶</ci><ci id="alg1.l3.m2.2.2.1.1.3.cmml" xref="alg1.l3.m2.2.2.1.1.3">ğ¾</ci></apply><cn type="integer" id="alg1.l3.m2.1.1.cmml" xref="alg1.l3.m2.1.1">1</cn></interval></annotation-xml><annotation encoding="application/x-tex" id="alg1.l3.m2.2c">(C\cdot K,1)</annotation></semantics></math><span id="alg1.l3.3" class="ltx_text" style="font-size:70%;">
</span>
</div>
<div id="alg1.l4" class="ltx_listingline">
<span id="alg1.l4.1" class="ltx_text" style="font-size:70%;">Â Â Â Â </span><math id="alg1.l4.m1.1" class="ltx_Math" alttext="S_{t}\leftarrow" display="inline"><semantics id="alg1.l4.m1.1a"><mrow id="alg1.l4.m1.1.1" xref="alg1.l4.m1.1.1.cmml"><msub id="alg1.l4.m1.1.1.2" xref="alg1.l4.m1.1.1.2.cmml"><mi mathsize="70%" id="alg1.l4.m1.1.1.2.2" xref="alg1.l4.m1.1.1.2.2.cmml">S</mi><mi mathsize="70%" id="alg1.l4.m1.1.1.2.3" xref="alg1.l4.m1.1.1.2.3.cmml">t</mi></msub><mo mathsize="70%" stretchy="false" id="alg1.l4.m1.1.1.1" xref="alg1.l4.m1.1.1.1.cmml">â†</mo><mi id="alg1.l4.m1.1.1.3" xref="alg1.l4.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg1.l4.m1.1b"><apply id="alg1.l4.m1.1.1.cmml" xref="alg1.l4.m1.1.1"><ci id="alg1.l4.m1.1.1.1.cmml" xref="alg1.l4.m1.1.1.1">â†</ci><apply id="alg1.l4.m1.1.1.2.cmml" xref="alg1.l4.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l4.m1.1.1.2.1.cmml" xref="alg1.l4.m1.1.1.2">subscript</csymbol><ci id="alg1.l4.m1.1.1.2.2.cmml" xref="alg1.l4.m1.1.1.2.2">ğ‘†</ci><ci id="alg1.l4.m1.1.1.2.3.cmml" xref="alg1.l4.m1.1.1.2.3">ğ‘¡</ci></apply><csymbol cd="latexml" id="alg1.l4.m1.1.1.3.cmml" xref="alg1.l4.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l4.m1.1c">S_{t}\leftarrow</annotation></semantics></math><span id="alg1.l4.2" class="ltx_text" style="font-size:70%;"> (random set of </span><math id="alg1.l4.m2.1" class="ltx_Math" alttext="m" display="inline"><semantics id="alg1.l4.m2.1a"><mi mathsize="70%" id="alg1.l4.m2.1.1" xref="alg1.l4.m2.1.1.cmml">m</mi><annotation-xml encoding="MathML-Content" id="alg1.l4.m2.1b"><ci id="alg1.l4.m2.1.1.cmml" xref="alg1.l4.m2.1.1">ğ‘š</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l4.m2.1c">m</annotation></semantics></math><span id="alg1.l4.3" class="ltx_text" style="font-size:70%;"> clients)
</span>
</div>
<div id="alg1.l5" class="ltx_listingline">
<span id="alg1.l5.1" class="ltx_text" style="font-size:70%;">Â Â Â Â </span><span id="alg1.l5.2" class="ltx_text ltx_font_bold" style="font-size:70%;">for</span><span id="alg1.l5.3" class="ltx_text" style="font-size:70%;">Â each client </span><math id="alg1.l5.m1.1" class="ltx_Math" alttext="k\in S_{t}" display="inline"><semantics id="alg1.l5.m1.1a"><mrow id="alg1.l5.m1.1.1" xref="alg1.l5.m1.1.1.cmml"><mi mathsize="70%" id="alg1.l5.m1.1.1.2" xref="alg1.l5.m1.1.1.2.cmml">k</mi><mo mathsize="70%" id="alg1.l5.m1.1.1.1" xref="alg1.l5.m1.1.1.1.cmml">âˆˆ</mo><msub id="alg1.l5.m1.1.1.3" xref="alg1.l5.m1.1.1.3.cmml"><mi mathsize="70%" id="alg1.l5.m1.1.1.3.2" xref="alg1.l5.m1.1.1.3.2.cmml">S</mi><mi mathsize="70%" id="alg1.l5.m1.1.1.3.3" xref="alg1.l5.m1.1.1.3.3.cmml">t</mi></msub></mrow><annotation-xml encoding="MathML-Content" id="alg1.l5.m1.1b"><apply id="alg1.l5.m1.1.1.cmml" xref="alg1.l5.m1.1.1"><in id="alg1.l5.m1.1.1.1.cmml" xref="alg1.l5.m1.1.1.1"></in><ci id="alg1.l5.m1.1.1.2.cmml" xref="alg1.l5.m1.1.1.2">ğ‘˜</ci><apply id="alg1.l5.m1.1.1.3.cmml" xref="alg1.l5.m1.1.1.3"><csymbol cd="ambiguous" id="alg1.l5.m1.1.1.3.1.cmml" xref="alg1.l5.m1.1.1.3">subscript</csymbol><ci id="alg1.l5.m1.1.1.3.2.cmml" xref="alg1.l5.m1.1.1.3.2">ğ‘†</ci><ci id="alg1.l5.m1.1.1.3.3.cmml" xref="alg1.l5.m1.1.1.3.3">ğ‘¡</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l5.m1.1c">k\in S_{t}</annotation></semantics></math><span id="alg1.l5.4" class="ltx_text" style="font-size:70%;"> </span><span id="alg1.l5.5" class="ltx_text ltx_font_bold" style="font-size:70%;">in parallel</span><span id="alg1.l5.6" class="ltx_text" style="font-size:70%;">Â </span><span id="alg1.l5.7" class="ltx_text ltx_font_bold" style="font-size:70%;">do</span><span id="alg1.l5.8" class="ltx_text" style="font-size:70%;">


</span>
</div>
<div id="alg1.l6" class="ltx_listingline">
<span id="alg1.l6.1" class="ltx_text" style="font-size:70%;">Â Â Â Â Â Â Â </span><math id="alg1.l6.m1.1" class="ltx_Math" alttext="\theta_{t+1}^{k}\leftarrow" display="inline"><semantics id="alg1.l6.m1.1a"><mrow id="alg1.l6.m1.1.1" xref="alg1.l6.m1.1.1.cmml"><msubsup id="alg1.l6.m1.1.1.2" xref="alg1.l6.m1.1.1.2.cmml"><mi mathsize="70%" id="alg1.l6.m1.1.1.2.2.2" xref="alg1.l6.m1.1.1.2.2.2.cmml">Î¸</mi><mrow id="alg1.l6.m1.1.1.2.2.3" xref="alg1.l6.m1.1.1.2.2.3.cmml"><mi mathsize="70%" id="alg1.l6.m1.1.1.2.2.3.2" xref="alg1.l6.m1.1.1.2.2.3.2.cmml">t</mi><mo mathsize="70%" id="alg1.l6.m1.1.1.2.2.3.1" xref="alg1.l6.m1.1.1.2.2.3.1.cmml">+</mo><mn mathsize="70%" id="alg1.l6.m1.1.1.2.2.3.3" xref="alg1.l6.m1.1.1.2.2.3.3.cmml">1</mn></mrow><mi mathsize="70%" id="alg1.l6.m1.1.1.2.3" xref="alg1.l6.m1.1.1.2.3.cmml">k</mi></msubsup><mo mathsize="70%" stretchy="false" id="alg1.l6.m1.1.1.1" xref="alg1.l6.m1.1.1.1.cmml">â†</mo><mi id="alg1.l6.m1.1.1.3" xref="alg1.l6.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg1.l6.m1.1b"><apply id="alg1.l6.m1.1.1.cmml" xref="alg1.l6.m1.1.1"><ci id="alg1.l6.m1.1.1.1.cmml" xref="alg1.l6.m1.1.1.1">â†</ci><apply id="alg1.l6.m1.1.1.2.cmml" xref="alg1.l6.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l6.m1.1.1.2.1.cmml" xref="alg1.l6.m1.1.1.2">superscript</csymbol><apply id="alg1.l6.m1.1.1.2.2.cmml" xref="alg1.l6.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l6.m1.1.1.2.2.1.cmml" xref="alg1.l6.m1.1.1.2">subscript</csymbol><ci id="alg1.l6.m1.1.1.2.2.2.cmml" xref="alg1.l6.m1.1.1.2.2.2">ğœƒ</ci><apply id="alg1.l6.m1.1.1.2.2.3.cmml" xref="alg1.l6.m1.1.1.2.2.3"><plus id="alg1.l6.m1.1.1.2.2.3.1.cmml" xref="alg1.l6.m1.1.1.2.2.3.1"></plus><ci id="alg1.l6.m1.1.1.2.2.3.2.cmml" xref="alg1.l6.m1.1.1.2.2.3.2">ğ‘¡</ci><cn type="integer" id="alg1.l6.m1.1.1.2.2.3.3.cmml" xref="alg1.l6.m1.1.1.2.2.3.3">1</cn></apply></apply><ci id="alg1.l6.m1.1.1.2.3.cmml" xref="alg1.l6.m1.1.1.2.3">ğ‘˜</ci></apply><csymbol cd="latexml" id="alg1.l6.m1.1.1.3.cmml" xref="alg1.l6.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l6.m1.1c">\theta_{t+1}^{k}\leftarrow</annotation></semantics></math><span id="alg1.l6.2" class="ltx_text" style="font-size:70%;"> ClientUpdate</span><math id="alg1.l6.m2.2" class="ltx_Math" alttext="(k,\theta_{t})" display="inline"><semantics id="alg1.l6.m2.2a"><mrow id="alg1.l6.m2.2.2.1" xref="alg1.l6.m2.2.2.2.cmml"><mo maxsize="70%" minsize="70%" id="alg1.l6.m2.2.2.1.2" xref="alg1.l6.m2.2.2.2.cmml">(</mo><mi mathsize="70%" id="alg1.l6.m2.1.1" xref="alg1.l6.m2.1.1.cmml">k</mi><mo mathsize="70%" id="alg1.l6.m2.2.2.1.3" xref="alg1.l6.m2.2.2.2.cmml">,</mo><msub id="alg1.l6.m2.2.2.1.1" xref="alg1.l6.m2.2.2.1.1.cmml"><mi mathsize="70%" id="alg1.l6.m2.2.2.1.1.2" xref="alg1.l6.m2.2.2.1.1.2.cmml">Î¸</mi><mi mathsize="70%" id="alg1.l6.m2.2.2.1.1.3" xref="alg1.l6.m2.2.2.1.1.3.cmml">t</mi></msub><mo maxsize="70%" minsize="70%" id="alg1.l6.m2.2.2.1.4" xref="alg1.l6.m2.2.2.2.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="alg1.l6.m2.2b"><interval closure="open" id="alg1.l6.m2.2.2.2.cmml" xref="alg1.l6.m2.2.2.1"><ci id="alg1.l6.m2.1.1.cmml" xref="alg1.l6.m2.1.1">ğ‘˜</ci><apply id="alg1.l6.m2.2.2.1.1.cmml" xref="alg1.l6.m2.2.2.1.1"><csymbol cd="ambiguous" id="alg1.l6.m2.2.2.1.1.1.cmml" xref="alg1.l6.m2.2.2.1.1">subscript</csymbol><ci id="alg1.l6.m2.2.2.1.1.2.cmml" xref="alg1.l6.m2.2.2.1.1.2">ğœƒ</ci><ci id="alg1.l6.m2.2.2.1.1.3.cmml" xref="alg1.l6.m2.2.2.1.1.3">ğ‘¡</ci></apply></interval></annotation-xml><annotation encoding="application/x-tex" id="alg1.l6.m2.2c">(k,\theta_{t})</annotation></semantics></math><span id="alg1.l6.3" class="ltx_text" style="font-size:70%;">
</span>
</div>
<div id="alg1.l7" class="ltx_listingline">
<span id="alg1.l7.1" class="ltx_text" style="font-size:70%;">Â Â Â Â </span><span id="alg1.l7.2" class="ltx_text ltx_font_bold" style="font-size:70%;">end</span><span id="alg1.l7.3" class="ltx_text" style="font-size:70%;">Â </span><span id="alg1.l7.4" class="ltx_text ltx_font_bold" style="font-size:70%;">for</span>
</div>
<div id="alg1.l8" class="ltx_listingline">
<span id="alg1.l8.1" class="ltx_text" style="font-size:70%;">Â Â Â Â </span><math id="alg1.l8.m1.1" class="ltx_Math" alttext="\theta_{t+1}\leftarrow\sum_{k=1}^{K}w_{k}\theta_{t+1}^{k}" display="inline"><semantics id="alg1.l8.m1.1a"><mrow id="alg1.l8.m1.1.1" xref="alg1.l8.m1.1.1.cmml"><msub id="alg1.l8.m1.1.1.2" xref="alg1.l8.m1.1.1.2.cmml"><mi mathsize="70%" id="alg1.l8.m1.1.1.2.2" xref="alg1.l8.m1.1.1.2.2.cmml">Î¸</mi><mrow id="alg1.l8.m1.1.1.2.3" xref="alg1.l8.m1.1.1.2.3.cmml"><mi mathsize="70%" id="alg1.l8.m1.1.1.2.3.2" xref="alg1.l8.m1.1.1.2.3.2.cmml">t</mi><mo mathsize="70%" id="alg1.l8.m1.1.1.2.3.1" xref="alg1.l8.m1.1.1.2.3.1.cmml">+</mo><mn mathsize="70%" id="alg1.l8.m1.1.1.2.3.3" xref="alg1.l8.m1.1.1.2.3.3.cmml">1</mn></mrow></msub><mo mathsize="70%" rspace="0.111em" stretchy="false" id="alg1.l8.m1.1.1.1" xref="alg1.l8.m1.1.1.1.cmml">â†</mo><mrow id="alg1.l8.m1.1.1.3" xref="alg1.l8.m1.1.1.3.cmml"><msubsup id="alg1.l8.m1.1.1.3.1" xref="alg1.l8.m1.1.1.3.1.cmml"><mo maxsize="70%" minsize="70%" stretchy="true" id="alg1.l8.m1.1.1.3.1.2.2" xref="alg1.l8.m1.1.1.3.1.2.2.cmml">âˆ‘</mo><mrow id="alg1.l8.m1.1.1.3.1.2.3" xref="alg1.l8.m1.1.1.3.1.2.3.cmml"><mi mathsize="70%" id="alg1.l8.m1.1.1.3.1.2.3.2" xref="alg1.l8.m1.1.1.3.1.2.3.2.cmml">k</mi><mo mathsize="70%" id="alg1.l8.m1.1.1.3.1.2.3.1" xref="alg1.l8.m1.1.1.3.1.2.3.1.cmml">=</mo><mn mathsize="70%" id="alg1.l8.m1.1.1.3.1.2.3.3" xref="alg1.l8.m1.1.1.3.1.2.3.3.cmml">1</mn></mrow><mi mathsize="70%" id="alg1.l8.m1.1.1.3.1.3" xref="alg1.l8.m1.1.1.3.1.3.cmml">K</mi></msubsup><mrow id="alg1.l8.m1.1.1.3.2" xref="alg1.l8.m1.1.1.3.2.cmml"><msub id="alg1.l8.m1.1.1.3.2.2" xref="alg1.l8.m1.1.1.3.2.2.cmml"><mi mathsize="70%" id="alg1.l8.m1.1.1.3.2.2.2" xref="alg1.l8.m1.1.1.3.2.2.2.cmml">w</mi><mi mathsize="70%" id="alg1.l8.m1.1.1.3.2.2.3" xref="alg1.l8.m1.1.1.3.2.2.3.cmml">k</mi></msub><mo lspace="0em" rspace="0em" id="alg1.l8.m1.1.1.3.2.1" xref="alg1.l8.m1.1.1.3.2.1.cmml">â€‹</mo><msubsup id="alg1.l8.m1.1.1.3.2.3" xref="alg1.l8.m1.1.1.3.2.3.cmml"><mi mathsize="70%" id="alg1.l8.m1.1.1.3.2.3.2.2" xref="alg1.l8.m1.1.1.3.2.3.2.2.cmml">Î¸</mi><mrow id="alg1.l8.m1.1.1.3.2.3.2.3" xref="alg1.l8.m1.1.1.3.2.3.2.3.cmml"><mi mathsize="70%" id="alg1.l8.m1.1.1.3.2.3.2.3.2" xref="alg1.l8.m1.1.1.3.2.3.2.3.2.cmml">t</mi><mo mathsize="70%" id="alg1.l8.m1.1.1.3.2.3.2.3.1" xref="alg1.l8.m1.1.1.3.2.3.2.3.1.cmml">+</mo><mn mathsize="70%" id="alg1.l8.m1.1.1.3.2.3.2.3.3" xref="alg1.l8.m1.1.1.3.2.3.2.3.3.cmml">1</mn></mrow><mi mathsize="70%" id="alg1.l8.m1.1.1.3.2.3.3" xref="alg1.l8.m1.1.1.3.2.3.3.cmml">k</mi></msubsup></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l8.m1.1b"><apply id="alg1.l8.m1.1.1.cmml" xref="alg1.l8.m1.1.1"><ci id="alg1.l8.m1.1.1.1.cmml" xref="alg1.l8.m1.1.1.1">â†</ci><apply id="alg1.l8.m1.1.1.2.cmml" xref="alg1.l8.m1.1.1.2"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.2.1.cmml" xref="alg1.l8.m1.1.1.2">subscript</csymbol><ci id="alg1.l8.m1.1.1.2.2.cmml" xref="alg1.l8.m1.1.1.2.2">ğœƒ</ci><apply id="alg1.l8.m1.1.1.2.3.cmml" xref="alg1.l8.m1.1.1.2.3"><plus id="alg1.l8.m1.1.1.2.3.1.cmml" xref="alg1.l8.m1.1.1.2.3.1"></plus><ci id="alg1.l8.m1.1.1.2.3.2.cmml" xref="alg1.l8.m1.1.1.2.3.2">ğ‘¡</ci><cn type="integer" id="alg1.l8.m1.1.1.2.3.3.cmml" xref="alg1.l8.m1.1.1.2.3.3">1</cn></apply></apply><apply id="alg1.l8.m1.1.1.3.cmml" xref="alg1.l8.m1.1.1.3"><apply id="alg1.l8.m1.1.1.3.1.cmml" xref="alg1.l8.m1.1.1.3.1"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.3.1.1.cmml" xref="alg1.l8.m1.1.1.3.1">superscript</csymbol><apply id="alg1.l8.m1.1.1.3.1.2.cmml" xref="alg1.l8.m1.1.1.3.1"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.3.1.2.1.cmml" xref="alg1.l8.m1.1.1.3.1">subscript</csymbol><sum id="alg1.l8.m1.1.1.3.1.2.2.cmml" xref="alg1.l8.m1.1.1.3.1.2.2"></sum><apply id="alg1.l8.m1.1.1.3.1.2.3.cmml" xref="alg1.l8.m1.1.1.3.1.2.3"><eq id="alg1.l8.m1.1.1.3.1.2.3.1.cmml" xref="alg1.l8.m1.1.1.3.1.2.3.1"></eq><ci id="alg1.l8.m1.1.1.3.1.2.3.2.cmml" xref="alg1.l8.m1.1.1.3.1.2.3.2">ğ‘˜</ci><cn type="integer" id="alg1.l8.m1.1.1.3.1.2.3.3.cmml" xref="alg1.l8.m1.1.1.3.1.2.3.3">1</cn></apply></apply><ci id="alg1.l8.m1.1.1.3.1.3.cmml" xref="alg1.l8.m1.1.1.3.1.3">ğ¾</ci></apply><apply id="alg1.l8.m1.1.1.3.2.cmml" xref="alg1.l8.m1.1.1.3.2"><times id="alg1.l8.m1.1.1.3.2.1.cmml" xref="alg1.l8.m1.1.1.3.2.1"></times><apply id="alg1.l8.m1.1.1.3.2.2.cmml" xref="alg1.l8.m1.1.1.3.2.2"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.3.2.2.1.cmml" xref="alg1.l8.m1.1.1.3.2.2">subscript</csymbol><ci id="alg1.l8.m1.1.1.3.2.2.2.cmml" xref="alg1.l8.m1.1.1.3.2.2.2">ğ‘¤</ci><ci id="alg1.l8.m1.1.1.3.2.2.3.cmml" xref="alg1.l8.m1.1.1.3.2.2.3">ğ‘˜</ci></apply><apply id="alg1.l8.m1.1.1.3.2.3.cmml" xref="alg1.l8.m1.1.1.3.2.3"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.3.2.3.1.cmml" xref="alg1.l8.m1.1.1.3.2.3">superscript</csymbol><apply id="alg1.l8.m1.1.1.3.2.3.2.cmml" xref="alg1.l8.m1.1.1.3.2.3"><csymbol cd="ambiguous" id="alg1.l8.m1.1.1.3.2.3.2.1.cmml" xref="alg1.l8.m1.1.1.3.2.3">subscript</csymbol><ci id="alg1.l8.m1.1.1.3.2.3.2.2.cmml" xref="alg1.l8.m1.1.1.3.2.3.2.2">ğœƒ</ci><apply id="alg1.l8.m1.1.1.3.2.3.2.3.cmml" xref="alg1.l8.m1.1.1.3.2.3.2.3"><plus id="alg1.l8.m1.1.1.3.2.3.2.3.1.cmml" xref="alg1.l8.m1.1.1.3.2.3.2.3.1"></plus><ci id="alg1.l8.m1.1.1.3.2.3.2.3.2.cmml" xref="alg1.l8.m1.1.1.3.2.3.2.3.2">ğ‘¡</ci><cn type="integer" id="alg1.l8.m1.1.1.3.2.3.2.3.3.cmml" xref="alg1.l8.m1.1.1.3.2.3.2.3.3">1</cn></apply></apply><ci id="alg1.l8.m1.1.1.3.2.3.3.cmml" xref="alg1.l8.m1.1.1.3.2.3.3">ğ‘˜</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l8.m1.1c">\theta_{t+1}\leftarrow\sum_{k=1}^{K}w_{k}\theta_{t+1}^{k}</annotation></semantics></math><span id="alg1.l8.2" class="ltx_text" style="font-size:70%;">
</span>
</div>
<div id="alg1.l9" class="ltx_listingline">
<span id="alg1.l9.1" class="ltx_text" style="font-size:70%;">â€„</span><span id="alg1.l9.2" class="ltx_text ltx_font_bold" style="font-size:70%;">end</span><span id="alg1.l9.3" class="ltx_text" style="font-size:70%;">Â </span><span id="alg1.l9.4" class="ltx_text ltx_font_bold" style="font-size:70%;">for</span>
</div>
</div>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<p id="alg1.14" class="ltx_p ltx_figure_panel"><span id="alg1.14.2" class="ltx_text ltx_font_italic" style="font-size:70%;">ClientUpdate<math id="alg1.13.1.m1.2" class="ltx_Math" alttext="(k,\theta)" display="inline"><semantics id="alg1.13.1.m1.2a"><mrow id="alg1.13.1.m1.2.3.2" xref="alg1.13.1.m1.2.3.1.cmml"><mo stretchy="false" id="alg1.13.1.m1.2.3.2.1" xref="alg1.13.1.m1.2.3.1.cmml">(</mo><mi id="alg1.13.1.m1.1.1" xref="alg1.13.1.m1.1.1.cmml">k</mi><mo id="alg1.13.1.m1.2.3.2.2" xref="alg1.13.1.m1.2.3.1.cmml">,</mo><mi id="alg1.13.1.m1.2.2" xref="alg1.13.1.m1.2.2.cmml">Î¸</mi><mo stretchy="false" id="alg1.13.1.m1.2.3.2.3" xref="alg1.13.1.m1.2.3.1.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="alg1.13.1.m1.2b"><interval closure="open" id="alg1.13.1.m1.2.3.1.cmml" xref="alg1.13.1.m1.2.3.2"><ci id="alg1.13.1.m1.1.1.cmml" xref="alg1.13.1.m1.1.1">ğ‘˜</ci><ci id="alg1.13.1.m1.2.2.cmml" xref="alg1.13.1.m1.2.2">ğœƒ</ci></interval></annotation-xml><annotation encoding="application/x-tex" id="alg1.13.1.m1.2c">(k,\theta)</annotation></semantics></math>:<span id="alg1.14.2.1" class="ltx_text ltx_font_upright">
run on client <math id="alg1.14.2.1.m1.1" class="ltx_Math" alttext="k" display="inline"><semantics id="alg1.14.2.1.m1.1a"><mi id="alg1.14.2.1.m1.1.1" xref="alg1.14.2.1.m1.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="alg1.14.2.1.m1.1b"><ci id="alg1.14.2.1.m1.1.1.cmml" xref="alg1.14.2.1.m1.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.14.2.1.m1.1c">k</annotation></semantics></math></span></span></p>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<div id="alg1.27" class="ltx_listing ltx_figure_panel ltx_listing">
<div id="alg1.l1a" class="ltx_listingline">
<span id="alg1.l1a.1" class="ltx_text" style="font-size:70%;">â€„</span><math id="alg1.l1a.m1.1" class="ltx_Math" alttext="\mathcal{B}\leftarrow" display="inline"><semantics id="alg1.l1a.m1.1a"><mrow id="alg1.l1a.m1.1.1" xref="alg1.l1a.m1.1.1.cmml"><mi class="ltx_font_mathcaligraphic" mathsize="70%" id="alg1.l1a.m1.1.1.2" xref="alg1.l1a.m1.1.1.2.cmml">â„¬</mi><mo mathsize="70%" stretchy="false" id="alg1.l1a.m1.1.1.1" xref="alg1.l1a.m1.1.1.1.cmml">â†</mo><mi id="alg1.l1a.m1.1.1.3" xref="alg1.l1a.m1.1.1.3.cmml"></mi></mrow><annotation-xml encoding="MathML-Content" id="alg1.l1a.m1.1b"><apply id="alg1.l1a.m1.1.1.cmml" xref="alg1.l1a.m1.1.1"><ci id="alg1.l1a.m1.1.1.1.cmml" xref="alg1.l1a.m1.1.1.1">â†</ci><ci id="alg1.l1a.m1.1.1.2.cmml" xref="alg1.l1a.m1.1.1.2">â„¬</ci><csymbol cd="latexml" id="alg1.l1a.m1.1.1.3.cmml" xref="alg1.l1a.m1.1.1.3">absent</csymbol></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1a.m1.1c">\mathcal{B}\leftarrow</annotation></semantics></math><span id="alg1.l1a.2" class="ltx_text" style="font-size:70%;"> (split </span><math id="alg1.l1a.m2.1" class="ltx_Math" alttext="\mathcal{P}_{k}" display="inline"><semantics id="alg1.l1a.m2.1a"><msub id="alg1.l1a.m2.1.1" xref="alg1.l1a.m2.1.1.cmml"><mi class="ltx_font_mathcaligraphic" mathsize="70%" id="alg1.l1a.m2.1.1.2" xref="alg1.l1a.m2.1.1.2.cmml">ğ’«</mi><mi mathsize="70%" id="alg1.l1a.m2.1.1.3" xref="alg1.l1a.m2.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="alg1.l1a.m2.1b"><apply id="alg1.l1a.m2.1.1.cmml" xref="alg1.l1a.m2.1.1"><csymbol cd="ambiguous" id="alg1.l1a.m2.1.1.1.cmml" xref="alg1.l1a.m2.1.1">subscript</csymbol><ci id="alg1.l1a.m2.1.1.2.cmml" xref="alg1.l1a.m2.1.1.2">ğ’«</ci><ci id="alg1.l1a.m2.1.1.3.cmml" xref="alg1.l1a.m2.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1a.m2.1c">\mathcal{P}_{k}</annotation></semantics></math><span id="alg1.l1a.3" class="ltx_text" style="font-size:70%;"> into batches of size </span><math id="alg1.l1a.m3.1" class="ltx_Math" alttext="B" display="inline"><semantics id="alg1.l1a.m3.1a"><mi mathsize="70%" id="alg1.l1a.m3.1.1" xref="alg1.l1a.m3.1.1.cmml">B</mi><annotation-xml encoding="MathML-Content" id="alg1.l1a.m3.1b"><ci id="alg1.l1a.m3.1.1.cmml" xref="alg1.l1a.m3.1.1">ğµ</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l1a.m3.1c">B</annotation></semantics></math><span id="alg1.l1a.4" class="ltx_text" style="font-size:70%;">)
</span>
</div>
<div id="alg1.l2a" class="ltx_listingline">
<span id="alg1.l2a.1" class="ltx_text" style="font-size:70%;">â€„</span><span id="alg1.l2a.2" class="ltx_text ltx_font_bold" style="font-size:70%;">for</span><span id="alg1.l2a.3" class="ltx_text" style="font-size:70%;">Â each local epoch </span><math id="alg1.l2a.m1.1" class="ltx_Math" alttext="i" display="inline"><semantics id="alg1.l2a.m1.1a"><mi mathsize="70%" id="alg1.l2a.m1.1.1" xref="alg1.l2a.m1.1.1.cmml">i</mi><annotation-xml encoding="MathML-Content" id="alg1.l2a.m1.1b"><ci id="alg1.l2a.m1.1.1.cmml" xref="alg1.l2a.m1.1.1">ğ‘–</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l2a.m1.1c">i</annotation></semantics></math><span id="alg1.l2a.4" class="ltx_text" style="font-size:70%;"> from </span><math id="alg1.l2a.m2.1" class="ltx_Math" alttext="1" display="inline"><semantics id="alg1.l2a.m2.1a"><mn mathsize="70%" id="alg1.l2a.m2.1.1" xref="alg1.l2a.m2.1.1.cmml">1</mn><annotation-xml encoding="MathML-Content" id="alg1.l2a.m2.1b"><cn type="integer" id="alg1.l2a.m2.1.1.cmml" xref="alg1.l2a.m2.1.1">1</cn></annotation-xml><annotation encoding="application/x-tex" id="alg1.l2a.m2.1c">1</annotation></semantics></math><span id="alg1.l2a.5" class="ltx_text" style="font-size:70%;"> to </span><math id="alg1.l2a.m3.1" class="ltx_Math" alttext="E" display="inline"><semantics id="alg1.l2a.m3.1a"><mi mathsize="70%" id="alg1.l2a.m3.1.1" xref="alg1.l2a.m3.1.1.cmml">E</mi><annotation-xml encoding="MathML-Content" id="alg1.l2a.m3.1b"><ci id="alg1.l2a.m3.1.1.cmml" xref="alg1.l2a.m3.1.1">ğ¸</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l2a.m3.1c">E</annotation></semantics></math><span id="alg1.l2a.6" class="ltx_text" style="font-size:70%;">Â </span><span id="alg1.l2a.7" class="ltx_text ltx_font_bold" style="font-size:70%;">do</span><span id="alg1.l2a.8" class="ltx_text" style="font-size:70%;">


</span>
</div>
<div id="alg1.l3a" class="ltx_listingline">
<span id="alg1.l3a.1" class="ltx_text" style="font-size:70%;">Â Â Â Â </span><span id="alg1.l3a.2" class="ltx_text ltx_font_bold" style="font-size:70%;">for</span><span id="alg1.l3a.3" class="ltx_text" style="font-size:70%;">Â batch </span><math id="alg1.l3a.m1.1" class="ltx_Math" alttext="b\in\mathcal{B}" display="inline"><semantics id="alg1.l3a.m1.1a"><mrow id="alg1.l3a.m1.1.1" xref="alg1.l3a.m1.1.1.cmml"><mi mathsize="70%" id="alg1.l3a.m1.1.1.2" xref="alg1.l3a.m1.1.1.2.cmml">b</mi><mo mathsize="70%" id="alg1.l3a.m1.1.1.1" xref="alg1.l3a.m1.1.1.1.cmml">âˆˆ</mo><mi class="ltx_font_mathcaligraphic" mathsize="70%" id="alg1.l3a.m1.1.1.3" xref="alg1.l3a.m1.1.1.3.cmml">â„¬</mi></mrow><annotation-xml encoding="MathML-Content" id="alg1.l3a.m1.1b"><apply id="alg1.l3a.m1.1.1.cmml" xref="alg1.l3a.m1.1.1"><in id="alg1.l3a.m1.1.1.1.cmml" xref="alg1.l3a.m1.1.1.1"></in><ci id="alg1.l3a.m1.1.1.2.cmml" xref="alg1.l3a.m1.1.1.2">ğ‘</ci><ci id="alg1.l3a.m1.1.1.3.cmml" xref="alg1.l3a.m1.1.1.3">â„¬</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l3a.m1.1c">b\in\mathcal{B}</annotation></semantics></math><span id="alg1.l3a.4" class="ltx_text" style="font-size:70%;">Â </span><span id="alg1.l3a.5" class="ltx_text ltx_font_bold" style="font-size:70%;">do</span><span id="alg1.l3a.6" class="ltx_text" style="font-size:70%;">


</span>
</div>
<div id="alg1.l4a" class="ltx_listingline">
<span id="alg1.l4a.1" class="ltx_text" style="font-size:70%;">Â Â Â Â Â Â Â </span><math id="alg1.l4a.m1.2" class="ltx_Math" alttext="\theta\leftarrow\theta-\eta\nabla l(\theta;b)" display="inline"><semantics id="alg1.l4a.m1.2a"><mrow id="alg1.l4a.m1.2.3" xref="alg1.l4a.m1.2.3.cmml"><mi mathsize="70%" id="alg1.l4a.m1.2.3.2" xref="alg1.l4a.m1.2.3.2.cmml">Î¸</mi><mo mathsize="70%" stretchy="false" id="alg1.l4a.m1.2.3.1" xref="alg1.l4a.m1.2.3.1.cmml">â†</mo><mrow id="alg1.l4a.m1.2.3.3" xref="alg1.l4a.m1.2.3.3.cmml"><mi mathsize="70%" id="alg1.l4a.m1.2.3.3.2" xref="alg1.l4a.m1.2.3.3.2.cmml">Î¸</mi><mo mathsize="70%" id="alg1.l4a.m1.2.3.3.1" xref="alg1.l4a.m1.2.3.3.1.cmml">âˆ’</mo><mrow id="alg1.l4a.m1.2.3.3.3" xref="alg1.l4a.m1.2.3.3.3.cmml"><mi mathsize="70%" id="alg1.l4a.m1.2.3.3.3.2" xref="alg1.l4a.m1.2.3.3.3.2.cmml">Î·</mi><mo lspace="0.167em" rspace="0em" id="alg1.l4a.m1.2.3.3.3.1" xref="alg1.l4a.m1.2.3.3.3.1.cmml">â€‹</mo><mrow id="alg1.l4a.m1.2.3.3.3.3" xref="alg1.l4a.m1.2.3.3.3.3.cmml"><mo mathsize="70%" rspace="0.167em" id="alg1.l4a.m1.2.3.3.3.3.1" xref="alg1.l4a.m1.2.3.3.3.3.1.cmml">âˆ‡</mo><mi mathsize="70%" id="alg1.l4a.m1.2.3.3.3.3.2" xref="alg1.l4a.m1.2.3.3.3.3.2.cmml">l</mi></mrow><mo lspace="0em" rspace="0em" id="alg1.l4a.m1.2.3.3.3.1a" xref="alg1.l4a.m1.2.3.3.3.1.cmml">â€‹</mo><mrow id="alg1.l4a.m1.2.3.3.3.4.2" xref="alg1.l4a.m1.2.3.3.3.4.1.cmml"><mo maxsize="70%" minsize="70%" id="alg1.l4a.m1.2.3.3.3.4.2.1" xref="alg1.l4a.m1.2.3.3.3.4.1.cmml">(</mo><mi mathsize="70%" id="alg1.l4a.m1.1.1" xref="alg1.l4a.m1.1.1.cmml">Î¸</mi><mo mathsize="70%" id="alg1.l4a.m1.2.3.3.3.4.2.2" xref="alg1.l4a.m1.2.3.3.3.4.1.cmml">;</mo><mi mathsize="70%" id="alg1.l4a.m1.2.2" xref="alg1.l4a.m1.2.2.cmml">b</mi><mo maxsize="70%" minsize="70%" id="alg1.l4a.m1.2.3.3.3.4.2.3" xref="alg1.l4a.m1.2.3.3.3.4.1.cmml">)</mo></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="alg1.l4a.m1.2b"><apply id="alg1.l4a.m1.2.3.cmml" xref="alg1.l4a.m1.2.3"><ci id="alg1.l4a.m1.2.3.1.cmml" xref="alg1.l4a.m1.2.3.1">â†</ci><ci id="alg1.l4a.m1.2.3.2.cmml" xref="alg1.l4a.m1.2.3.2">ğœƒ</ci><apply id="alg1.l4a.m1.2.3.3.cmml" xref="alg1.l4a.m1.2.3.3"><minus id="alg1.l4a.m1.2.3.3.1.cmml" xref="alg1.l4a.m1.2.3.3.1"></minus><ci id="alg1.l4a.m1.2.3.3.2.cmml" xref="alg1.l4a.m1.2.3.3.2">ğœƒ</ci><apply id="alg1.l4a.m1.2.3.3.3.cmml" xref="alg1.l4a.m1.2.3.3.3"><times id="alg1.l4a.m1.2.3.3.3.1.cmml" xref="alg1.l4a.m1.2.3.3.3.1"></times><ci id="alg1.l4a.m1.2.3.3.3.2.cmml" xref="alg1.l4a.m1.2.3.3.3.2">ğœ‚</ci><apply id="alg1.l4a.m1.2.3.3.3.3.cmml" xref="alg1.l4a.m1.2.3.3.3.3"><ci id="alg1.l4a.m1.2.3.3.3.3.1.cmml" xref="alg1.l4a.m1.2.3.3.3.3.1">âˆ‡</ci><ci id="alg1.l4a.m1.2.3.3.3.3.2.cmml" xref="alg1.l4a.m1.2.3.3.3.3.2">ğ‘™</ci></apply><list id="alg1.l4a.m1.2.3.3.3.4.1.cmml" xref="alg1.l4a.m1.2.3.3.3.4.2"><ci id="alg1.l4a.m1.1.1.cmml" xref="alg1.l4a.m1.1.1">ğœƒ</ci><ci id="alg1.l4a.m1.2.2.cmml" xref="alg1.l4a.m1.2.2">ğ‘</ci></list></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="alg1.l4a.m1.2c">\theta\leftarrow\theta-\eta\nabla l(\theta;b)</annotation></semantics></math><span id="alg1.l4a.2" class="ltx_text" style="font-size:70%;">
</span>
</div>
<div id="alg1.l5a" class="ltx_listingline">
<span id="alg1.l5a.1" class="ltx_text" style="font-size:70%;">Â Â Â Â </span><span id="alg1.l5a.2" class="ltx_text ltx_font_bold" style="font-size:70%;">end</span><span id="alg1.l5a.3" class="ltx_text" style="font-size:70%;">Â </span><span id="alg1.l5a.4" class="ltx_text ltx_font_bold" style="font-size:70%;">for</span>
</div>
<div id="alg1.l6a" class="ltx_listingline">
<span id="alg1.l6a.1" class="ltx_text" style="font-size:70%;">â€„</span><span id="alg1.l6a.2" class="ltx_text ltx_font_bold" style="font-size:70%;">end</span><span id="alg1.l6a.3" class="ltx_text" style="font-size:70%;">Â </span><span id="alg1.l6a.4" class="ltx_text ltx_font_bold" style="font-size:70%;">for</span>
</div>
<div id="alg1.l7a" class="ltx_listingline">
<span id="alg1.l7a.1" class="ltx_text" style="font-size:70%;">â€„</span><span id="alg1.l7a.2" class="ltx_text ltx_font_bold" style="font-size:70%;">return</span><span id="alg1.l7a.3" class="ltx_text" style="font-size:70%;"> Â </span><math id="alg1.l7a.m1.1" class="ltx_Math" alttext="\theta" display="inline"><semantics id="alg1.l7a.m1.1a"><mi mathsize="70%" id="alg1.l7a.m1.1.1" xref="alg1.l7a.m1.1.1.cmml">Î¸</mi><annotation-xml encoding="MathML-Content" id="alg1.l7a.m1.1b"><ci id="alg1.l7a.m1.1.1.cmml" xref="alg1.l7a.m1.1.1">ğœƒ</ci></annotation-xml><annotation encoding="application/x-tex" id="alg1.l7a.m1.1c">\theta</annotation></semantics></math><span id="alg1.l7a.4" class="ltx_text" style="font-size:70%;"> to server
</span>
</div>
</div>
</div>
</div>
</figure>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Preliminaries</h2>

<div id="S2.p1" class="ltx_para">
<p id="S2.p1.1" class="ltx_p">FL, introduced in 2017Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib4" title="" class="ltx_ref">4</a></sup></cite> as federated averaging (FedAvg), is an approach that trains models across multiple clients without centralizing data.
In FL, each client (e.g., hospitals and institutions) keeps their private data locally and contributes to a shared model by sending updates like gradients or parameters to a central server. This server coordinates the training process, aggregates updates, and broadcasts the refined model back to clients.
The goal of FL is to minimize the global objective function with parameters <math id="S2.p1.1.m1.1" class="ltx_Math" alttext="\theta" display="inline"><semantics id="S2.p1.1.m1.1a"><mi id="S2.p1.1.m1.1.1" xref="S2.p1.1.m1.1.1.cmml">Î¸</mi><annotation-xml encoding="MathML-Content" id="S2.p1.1.m1.1b"><ci id="S2.p1.1.m1.1.1.cmml" xref="S2.p1.1.m1.1.1">ğœƒ</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.1.m1.1c">\theta</annotation></semantics></math> defined as:</p>
<table id="S2.E1" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S2.E1.m1.8" class="ltx_Math" alttext="\sum_{k=1}^{K}w_{k}F_{k}(\theta)\quad where\quad F_{k}(w)=\frac{1}{n_{k}}\sum_{i\in\mathcal{P}_{k}}l(x_{i},y_{i},\theta)" display="block"><semantics id="S2.E1.m1.8a"><mrow id="S2.E1.m1.8.8" xref="S2.E1.m1.8.8.cmml"><mrow id="S2.E1.m1.6.6.3.3" xref="S2.E1.m1.6.6.3.4.cmml"><mrow id="S2.E1.m1.4.4.1.1.1" xref="S2.E1.m1.4.4.1.1.1.cmml"><munderover id="S2.E1.m1.4.4.1.1.1.1" xref="S2.E1.m1.4.4.1.1.1.1.cmml"><mo movablelimits="false" id="S2.E1.m1.4.4.1.1.1.1.2.2" xref="S2.E1.m1.4.4.1.1.1.1.2.2.cmml">âˆ‘</mo><mrow id="S2.E1.m1.4.4.1.1.1.1.2.3" xref="S2.E1.m1.4.4.1.1.1.1.2.3.cmml"><mi id="S2.E1.m1.4.4.1.1.1.1.2.3.2" xref="S2.E1.m1.4.4.1.1.1.1.2.3.2.cmml">k</mi><mo id="S2.E1.m1.4.4.1.1.1.1.2.3.1" xref="S2.E1.m1.4.4.1.1.1.1.2.3.1.cmml">=</mo><mn id="S2.E1.m1.4.4.1.1.1.1.2.3.3" xref="S2.E1.m1.4.4.1.1.1.1.2.3.3.cmml">1</mn></mrow><mi id="S2.E1.m1.4.4.1.1.1.1.3" xref="S2.E1.m1.4.4.1.1.1.1.3.cmml">K</mi></munderover><mrow id="S2.E1.m1.4.4.1.1.1.2" xref="S2.E1.m1.4.4.1.1.1.2.cmml"><msub id="S2.E1.m1.4.4.1.1.1.2.2" xref="S2.E1.m1.4.4.1.1.1.2.2.cmml"><mi id="S2.E1.m1.4.4.1.1.1.2.2.2" xref="S2.E1.m1.4.4.1.1.1.2.2.2.cmml">w</mi><mi id="S2.E1.m1.4.4.1.1.1.2.2.3" xref="S2.E1.m1.4.4.1.1.1.2.2.3.cmml">k</mi></msub><mo lspace="0em" rspace="0em" id="S2.E1.m1.4.4.1.1.1.2.1" xref="S2.E1.m1.4.4.1.1.1.2.1.cmml">â€‹</mo><msub id="S2.E1.m1.4.4.1.1.1.2.3" xref="S2.E1.m1.4.4.1.1.1.2.3.cmml"><mi id="S2.E1.m1.4.4.1.1.1.2.3.2" xref="S2.E1.m1.4.4.1.1.1.2.3.2.cmml">F</mi><mi id="S2.E1.m1.4.4.1.1.1.2.3.3" xref="S2.E1.m1.4.4.1.1.1.2.3.3.cmml">k</mi></msub><mo lspace="0em" rspace="0em" id="S2.E1.m1.4.4.1.1.1.2.1a" xref="S2.E1.m1.4.4.1.1.1.2.1.cmml">â€‹</mo><mrow id="S2.E1.m1.4.4.1.1.1.2.4.2" xref="S2.E1.m1.4.4.1.1.1.2.cmml"><mo stretchy="false" id="S2.E1.m1.4.4.1.1.1.2.4.2.1" xref="S2.E1.m1.4.4.1.1.1.2.cmml">(</mo><mi id="S2.E1.m1.1.1" xref="S2.E1.m1.1.1.cmml">Î¸</mi><mo stretchy="false" id="S2.E1.m1.4.4.1.1.1.2.4.2.2" xref="S2.E1.m1.4.4.1.1.1.2.cmml">)</mo></mrow></mrow></mrow><mspace width="1em" id="S2.E1.m1.6.6.3.3.4" xref="S2.E1.m1.6.6.3.4.cmml"></mspace><mrow id="S2.E1.m1.5.5.2.2.2" xref="S2.E1.m1.5.5.2.2.2.cmml"><mi id="S2.E1.m1.5.5.2.2.2.2" xref="S2.E1.m1.5.5.2.2.2.2.cmml">w</mi><mo lspace="0em" rspace="0em" id="S2.E1.m1.5.5.2.2.2.1" xref="S2.E1.m1.5.5.2.2.2.1.cmml">â€‹</mo><mi id="S2.E1.m1.5.5.2.2.2.3" xref="S2.E1.m1.5.5.2.2.2.3.cmml">h</mi><mo lspace="0em" rspace="0em" id="S2.E1.m1.5.5.2.2.2.1a" xref="S2.E1.m1.5.5.2.2.2.1.cmml">â€‹</mo><mi id="S2.E1.m1.5.5.2.2.2.4" xref="S2.E1.m1.5.5.2.2.2.4.cmml">e</mi><mo lspace="0em" rspace="0em" id="S2.E1.m1.5.5.2.2.2.1b" xref="S2.E1.m1.5.5.2.2.2.1.cmml">â€‹</mo><mi id="S2.E1.m1.5.5.2.2.2.5" xref="S2.E1.m1.5.5.2.2.2.5.cmml">r</mi><mo lspace="0em" rspace="0em" id="S2.E1.m1.5.5.2.2.2.1c" xref="S2.E1.m1.5.5.2.2.2.1.cmml">â€‹</mo><mi id="S2.E1.m1.5.5.2.2.2.6" xref="S2.E1.m1.5.5.2.2.2.6.cmml">e</mi></mrow><mspace width="1em" id="S2.E1.m1.6.6.3.3.5" xref="S2.E1.m1.6.6.3.4.cmml"></mspace><mrow id="S2.E1.m1.6.6.3.3.3" xref="S2.E1.m1.6.6.3.3.3.cmml"><msub id="S2.E1.m1.6.6.3.3.3.2" xref="S2.E1.m1.6.6.3.3.3.2.cmml"><mi id="S2.E1.m1.6.6.3.3.3.2.2" xref="S2.E1.m1.6.6.3.3.3.2.2.cmml">F</mi><mi id="S2.E1.m1.6.6.3.3.3.2.3" xref="S2.E1.m1.6.6.3.3.3.2.3.cmml">k</mi></msub><mo lspace="0em" rspace="0em" id="S2.E1.m1.6.6.3.3.3.1" xref="S2.E1.m1.6.6.3.3.3.1.cmml">â€‹</mo><mrow id="S2.E1.m1.6.6.3.3.3.3.2" xref="S2.E1.m1.6.6.3.3.3.cmml"><mo stretchy="false" id="S2.E1.m1.6.6.3.3.3.3.2.1" xref="S2.E1.m1.6.6.3.3.3.cmml">(</mo><mi id="S2.E1.m1.2.2" xref="S2.E1.m1.2.2.cmml">w</mi><mo stretchy="false" id="S2.E1.m1.6.6.3.3.3.3.2.2" xref="S2.E1.m1.6.6.3.3.3.cmml">)</mo></mrow></mrow></mrow><mo id="S2.E1.m1.8.8.6" xref="S2.E1.m1.8.8.6.cmml">=</mo><mrow id="S2.E1.m1.8.8.5" xref="S2.E1.m1.8.8.5.cmml"><mfrac id="S2.E1.m1.8.8.5.4" xref="S2.E1.m1.8.8.5.4.cmml"><mn id="S2.E1.m1.8.8.5.4.2" xref="S2.E1.m1.8.8.5.4.2.cmml">1</mn><msub id="S2.E1.m1.8.8.5.4.3" xref="S2.E1.m1.8.8.5.4.3.cmml"><mi id="S2.E1.m1.8.8.5.4.3.2" xref="S2.E1.m1.8.8.5.4.3.2.cmml">n</mi><mi id="S2.E1.m1.8.8.5.4.3.3" xref="S2.E1.m1.8.8.5.4.3.3.cmml">k</mi></msub></mfrac><mo lspace="0em" rspace="0em" id="S2.E1.m1.8.8.5.3" xref="S2.E1.m1.8.8.5.3.cmml">â€‹</mo><mrow id="S2.E1.m1.8.8.5.2" xref="S2.E1.m1.8.8.5.2.cmml"><munder id="S2.E1.m1.8.8.5.2.3" xref="S2.E1.m1.8.8.5.2.3.cmml"><mo movablelimits="false" id="S2.E1.m1.8.8.5.2.3.2" xref="S2.E1.m1.8.8.5.2.3.2.cmml">âˆ‘</mo><mrow id="S2.E1.m1.8.8.5.2.3.3" xref="S2.E1.m1.8.8.5.2.3.3.cmml"><mi id="S2.E1.m1.8.8.5.2.3.3.2" xref="S2.E1.m1.8.8.5.2.3.3.2.cmml">i</mi><mo id="S2.E1.m1.8.8.5.2.3.3.1" xref="S2.E1.m1.8.8.5.2.3.3.1.cmml">âˆˆ</mo><msub id="S2.E1.m1.8.8.5.2.3.3.3" xref="S2.E1.m1.8.8.5.2.3.3.3.cmml"><mi class="ltx_font_mathcaligraphic" id="S2.E1.m1.8.8.5.2.3.3.3.2" xref="S2.E1.m1.8.8.5.2.3.3.3.2.cmml">ğ’«</mi><mi id="S2.E1.m1.8.8.5.2.3.3.3.3" xref="S2.E1.m1.8.8.5.2.3.3.3.3.cmml">k</mi></msub></mrow></munder><mrow id="S2.E1.m1.8.8.5.2.2" xref="S2.E1.m1.8.8.5.2.2.cmml"><mi id="S2.E1.m1.8.8.5.2.2.4" xref="S2.E1.m1.8.8.5.2.2.4.cmml">l</mi><mo lspace="0em" rspace="0em" id="S2.E1.m1.8.8.5.2.2.3" xref="S2.E1.m1.8.8.5.2.2.3.cmml">â€‹</mo><mrow id="S2.E1.m1.8.8.5.2.2.2.2" xref="S2.E1.m1.8.8.5.2.2.2.3.cmml"><mo stretchy="false" id="S2.E1.m1.8.8.5.2.2.2.2.3" xref="S2.E1.m1.8.8.5.2.2.2.3.cmml">(</mo><msub id="S2.E1.m1.7.7.4.1.1.1.1.1" xref="S2.E1.m1.7.7.4.1.1.1.1.1.cmml"><mi id="S2.E1.m1.7.7.4.1.1.1.1.1.2" xref="S2.E1.m1.7.7.4.1.1.1.1.1.2.cmml">x</mi><mi id="S2.E1.m1.7.7.4.1.1.1.1.1.3" xref="S2.E1.m1.7.7.4.1.1.1.1.1.3.cmml">i</mi></msub><mo id="S2.E1.m1.8.8.5.2.2.2.2.4" xref="S2.E1.m1.8.8.5.2.2.2.3.cmml">,</mo><msub id="S2.E1.m1.8.8.5.2.2.2.2.2" xref="S2.E1.m1.8.8.5.2.2.2.2.2.cmml"><mi id="S2.E1.m1.8.8.5.2.2.2.2.2.2" xref="S2.E1.m1.8.8.5.2.2.2.2.2.2.cmml">y</mi><mi id="S2.E1.m1.8.8.5.2.2.2.2.2.3" xref="S2.E1.m1.8.8.5.2.2.2.2.2.3.cmml">i</mi></msub><mo id="S2.E1.m1.8.8.5.2.2.2.2.5" xref="S2.E1.m1.8.8.5.2.2.2.3.cmml">,</mo><mi id="S2.E1.m1.3.3" xref="S2.E1.m1.3.3.cmml">Î¸</mi><mo stretchy="false" id="S2.E1.m1.8.8.5.2.2.2.2.6" xref="S2.E1.m1.8.8.5.2.2.2.3.cmml">)</mo></mrow></mrow></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.E1.m1.8b"><apply id="S2.E1.m1.8.8.cmml" xref="S2.E1.m1.8.8"><eq id="S2.E1.m1.8.8.6.cmml" xref="S2.E1.m1.8.8.6"></eq><list id="S2.E1.m1.6.6.3.4.cmml" xref="S2.E1.m1.6.6.3.3"><apply id="S2.E1.m1.4.4.1.1.1.cmml" xref="S2.E1.m1.4.4.1.1.1"><apply id="S2.E1.m1.4.4.1.1.1.1.cmml" xref="S2.E1.m1.4.4.1.1.1.1"><csymbol cd="ambiguous" id="S2.E1.m1.4.4.1.1.1.1.1.cmml" xref="S2.E1.m1.4.4.1.1.1.1">superscript</csymbol><apply id="S2.E1.m1.4.4.1.1.1.1.2.cmml" xref="S2.E1.m1.4.4.1.1.1.1"><csymbol cd="ambiguous" id="S2.E1.m1.4.4.1.1.1.1.2.1.cmml" xref="S2.E1.m1.4.4.1.1.1.1">subscript</csymbol><sum id="S2.E1.m1.4.4.1.1.1.1.2.2.cmml" xref="S2.E1.m1.4.4.1.1.1.1.2.2"></sum><apply id="S2.E1.m1.4.4.1.1.1.1.2.3.cmml" xref="S2.E1.m1.4.4.1.1.1.1.2.3"><eq id="S2.E1.m1.4.4.1.1.1.1.2.3.1.cmml" xref="S2.E1.m1.4.4.1.1.1.1.2.3.1"></eq><ci id="S2.E1.m1.4.4.1.1.1.1.2.3.2.cmml" xref="S2.E1.m1.4.4.1.1.1.1.2.3.2">ğ‘˜</ci><cn type="integer" id="S2.E1.m1.4.4.1.1.1.1.2.3.3.cmml" xref="S2.E1.m1.4.4.1.1.1.1.2.3.3">1</cn></apply></apply><ci id="S2.E1.m1.4.4.1.1.1.1.3.cmml" xref="S2.E1.m1.4.4.1.1.1.1.3">ğ¾</ci></apply><apply id="S2.E1.m1.4.4.1.1.1.2.cmml" xref="S2.E1.m1.4.4.1.1.1.2"><times id="S2.E1.m1.4.4.1.1.1.2.1.cmml" xref="S2.E1.m1.4.4.1.1.1.2.1"></times><apply id="S2.E1.m1.4.4.1.1.1.2.2.cmml" xref="S2.E1.m1.4.4.1.1.1.2.2"><csymbol cd="ambiguous" id="S2.E1.m1.4.4.1.1.1.2.2.1.cmml" xref="S2.E1.m1.4.4.1.1.1.2.2">subscript</csymbol><ci id="S2.E1.m1.4.4.1.1.1.2.2.2.cmml" xref="S2.E1.m1.4.4.1.1.1.2.2.2">ğ‘¤</ci><ci id="S2.E1.m1.4.4.1.1.1.2.2.3.cmml" xref="S2.E1.m1.4.4.1.1.1.2.2.3">ğ‘˜</ci></apply><apply id="S2.E1.m1.4.4.1.1.1.2.3.cmml" xref="S2.E1.m1.4.4.1.1.1.2.3"><csymbol cd="ambiguous" id="S2.E1.m1.4.4.1.1.1.2.3.1.cmml" xref="S2.E1.m1.4.4.1.1.1.2.3">subscript</csymbol><ci id="S2.E1.m1.4.4.1.1.1.2.3.2.cmml" xref="S2.E1.m1.4.4.1.1.1.2.3.2">ğ¹</ci><ci id="S2.E1.m1.4.4.1.1.1.2.3.3.cmml" xref="S2.E1.m1.4.4.1.1.1.2.3.3">ğ‘˜</ci></apply><ci id="S2.E1.m1.1.1.cmml" xref="S2.E1.m1.1.1">ğœƒ</ci></apply></apply><apply id="S2.E1.m1.5.5.2.2.2.cmml" xref="S2.E1.m1.5.5.2.2.2"><times id="S2.E1.m1.5.5.2.2.2.1.cmml" xref="S2.E1.m1.5.5.2.2.2.1"></times><ci id="S2.E1.m1.5.5.2.2.2.2.cmml" xref="S2.E1.m1.5.5.2.2.2.2">ğ‘¤</ci><ci id="S2.E1.m1.5.5.2.2.2.3.cmml" xref="S2.E1.m1.5.5.2.2.2.3">â„</ci><ci id="S2.E1.m1.5.5.2.2.2.4.cmml" xref="S2.E1.m1.5.5.2.2.2.4">ğ‘’</ci><ci id="S2.E1.m1.5.5.2.2.2.5.cmml" xref="S2.E1.m1.5.5.2.2.2.5">ğ‘Ÿ</ci><ci id="S2.E1.m1.5.5.2.2.2.6.cmml" xref="S2.E1.m1.5.5.2.2.2.6">ğ‘’</ci></apply><apply id="S2.E1.m1.6.6.3.3.3.cmml" xref="S2.E1.m1.6.6.3.3.3"><times id="S2.E1.m1.6.6.3.3.3.1.cmml" xref="S2.E1.m1.6.6.3.3.3.1"></times><apply id="S2.E1.m1.6.6.3.3.3.2.cmml" xref="S2.E1.m1.6.6.3.3.3.2"><csymbol cd="ambiguous" id="S2.E1.m1.6.6.3.3.3.2.1.cmml" xref="S2.E1.m1.6.6.3.3.3.2">subscript</csymbol><ci id="S2.E1.m1.6.6.3.3.3.2.2.cmml" xref="S2.E1.m1.6.6.3.3.3.2.2">ğ¹</ci><ci id="S2.E1.m1.6.6.3.3.3.2.3.cmml" xref="S2.E1.m1.6.6.3.3.3.2.3">ğ‘˜</ci></apply><ci id="S2.E1.m1.2.2.cmml" xref="S2.E1.m1.2.2">ğ‘¤</ci></apply></list><apply id="S2.E1.m1.8.8.5.cmml" xref="S2.E1.m1.8.8.5"><times id="S2.E1.m1.8.8.5.3.cmml" xref="S2.E1.m1.8.8.5.3"></times><apply id="S2.E1.m1.8.8.5.4.cmml" xref="S2.E1.m1.8.8.5.4"><divide id="S2.E1.m1.8.8.5.4.1.cmml" xref="S2.E1.m1.8.8.5.4"></divide><cn type="integer" id="S2.E1.m1.8.8.5.4.2.cmml" xref="S2.E1.m1.8.8.5.4.2">1</cn><apply id="S2.E1.m1.8.8.5.4.3.cmml" xref="S2.E1.m1.8.8.5.4.3"><csymbol cd="ambiguous" id="S2.E1.m1.8.8.5.4.3.1.cmml" xref="S2.E1.m1.8.8.5.4.3">subscript</csymbol><ci id="S2.E1.m1.8.8.5.4.3.2.cmml" xref="S2.E1.m1.8.8.5.4.3.2">ğ‘›</ci><ci id="S2.E1.m1.8.8.5.4.3.3.cmml" xref="S2.E1.m1.8.8.5.4.3.3">ğ‘˜</ci></apply></apply><apply id="S2.E1.m1.8.8.5.2.cmml" xref="S2.E1.m1.8.8.5.2"><apply id="S2.E1.m1.8.8.5.2.3.cmml" xref="S2.E1.m1.8.8.5.2.3"><csymbol cd="ambiguous" id="S2.E1.m1.8.8.5.2.3.1.cmml" xref="S2.E1.m1.8.8.5.2.3">subscript</csymbol><sum id="S2.E1.m1.8.8.5.2.3.2.cmml" xref="S2.E1.m1.8.8.5.2.3.2"></sum><apply id="S2.E1.m1.8.8.5.2.3.3.cmml" xref="S2.E1.m1.8.8.5.2.3.3"><in id="S2.E1.m1.8.8.5.2.3.3.1.cmml" xref="S2.E1.m1.8.8.5.2.3.3.1"></in><ci id="S2.E1.m1.8.8.5.2.3.3.2.cmml" xref="S2.E1.m1.8.8.5.2.3.3.2">ğ‘–</ci><apply id="S2.E1.m1.8.8.5.2.3.3.3.cmml" xref="S2.E1.m1.8.8.5.2.3.3.3"><csymbol cd="ambiguous" id="S2.E1.m1.8.8.5.2.3.3.3.1.cmml" xref="S2.E1.m1.8.8.5.2.3.3.3">subscript</csymbol><ci id="S2.E1.m1.8.8.5.2.3.3.3.2.cmml" xref="S2.E1.m1.8.8.5.2.3.3.3.2">ğ’«</ci><ci id="S2.E1.m1.8.8.5.2.3.3.3.3.cmml" xref="S2.E1.m1.8.8.5.2.3.3.3.3">ğ‘˜</ci></apply></apply></apply><apply id="S2.E1.m1.8.8.5.2.2.cmml" xref="S2.E1.m1.8.8.5.2.2"><times id="S2.E1.m1.8.8.5.2.2.3.cmml" xref="S2.E1.m1.8.8.5.2.2.3"></times><ci id="S2.E1.m1.8.8.5.2.2.4.cmml" xref="S2.E1.m1.8.8.5.2.2.4">ğ‘™</ci><vector id="S2.E1.m1.8.8.5.2.2.2.3.cmml" xref="S2.E1.m1.8.8.5.2.2.2.2"><apply id="S2.E1.m1.7.7.4.1.1.1.1.1.cmml" xref="S2.E1.m1.7.7.4.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.E1.m1.7.7.4.1.1.1.1.1.1.cmml" xref="S2.E1.m1.7.7.4.1.1.1.1.1">subscript</csymbol><ci id="S2.E1.m1.7.7.4.1.1.1.1.1.2.cmml" xref="S2.E1.m1.7.7.4.1.1.1.1.1.2">ğ‘¥</ci><ci id="S2.E1.m1.7.7.4.1.1.1.1.1.3.cmml" xref="S2.E1.m1.7.7.4.1.1.1.1.1.3">ğ‘–</ci></apply><apply id="S2.E1.m1.8.8.5.2.2.2.2.2.cmml" xref="S2.E1.m1.8.8.5.2.2.2.2.2"><csymbol cd="ambiguous" id="S2.E1.m1.8.8.5.2.2.2.2.2.1.cmml" xref="S2.E1.m1.8.8.5.2.2.2.2.2">subscript</csymbol><ci id="S2.E1.m1.8.8.5.2.2.2.2.2.2.cmml" xref="S2.E1.m1.8.8.5.2.2.2.2.2.2">ğ‘¦</ci><ci id="S2.E1.m1.8.8.5.2.2.2.2.2.3.cmml" xref="S2.E1.m1.8.8.5.2.2.2.2.2.3">ğ‘–</ci></apply><ci id="S2.E1.m1.3.3.cmml" xref="S2.E1.m1.3.3">ğœƒ</ci></vector></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.E1.m1.8c">\sum_{k=1}^{K}w_{k}F_{k}(\theta)\quad where\quad F_{k}(w)=\frac{1}{n_{k}}\sum_{i\in\mathcal{P}_{k}}l(x_{i},y_{i},\theta)</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(1)</span></td>
</tr></tbody>
</table>
<p id="S2.p1.11" class="ltx_p">where <math id="S2.p1.2.m1.1" class="ltx_Math" alttext="K" display="inline"><semantics id="S2.p1.2.m1.1a"><mi id="S2.p1.2.m1.1.1" xref="S2.p1.2.m1.1.1.cmml">K</mi><annotation-xml encoding="MathML-Content" id="S2.p1.2.m1.1b"><ci id="S2.p1.2.m1.1.1.cmml" xref="S2.p1.2.m1.1.1">ğ¾</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.2.m1.1c">K</annotation></semantics></math> is the number of clients, the weights <math id="S2.p1.3.m2.1" class="ltx_Math" alttext="w_{k}" display="inline"><semantics id="S2.p1.3.m2.1a"><msub id="S2.p1.3.m2.1.1" xref="S2.p1.3.m2.1.1.cmml"><mi id="S2.p1.3.m2.1.1.2" xref="S2.p1.3.m2.1.1.2.cmml">w</mi><mi id="S2.p1.3.m2.1.1.3" xref="S2.p1.3.m2.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="S2.p1.3.m2.1b"><apply id="S2.p1.3.m2.1.1.cmml" xref="S2.p1.3.m2.1.1"><csymbol cd="ambiguous" id="S2.p1.3.m2.1.1.1.cmml" xref="S2.p1.3.m2.1.1">subscript</csymbol><ci id="S2.p1.3.m2.1.1.2.cmml" xref="S2.p1.3.m2.1.1.2">ğ‘¤</ci><ci id="S2.p1.3.m2.1.1.3.cmml" xref="S2.p1.3.m2.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.3.m2.1c">w_{k}</annotation></semantics></math> represents the proportional significance or scale of each local dataset, <math id="S2.p1.4.m3.1" class="ltx_Math" alttext="n_{k}" display="inline"><semantics id="S2.p1.4.m3.1a"><msub id="S2.p1.4.m3.1.1" xref="S2.p1.4.m3.1.1.cmml"><mi id="S2.p1.4.m3.1.1.2" xref="S2.p1.4.m3.1.1.2.cmml">n</mi><mi id="S2.p1.4.m3.1.1.3" xref="S2.p1.4.m3.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="S2.p1.4.m3.1b"><apply id="S2.p1.4.m3.1.1.cmml" xref="S2.p1.4.m3.1.1"><csymbol cd="ambiguous" id="S2.p1.4.m3.1.1.1.cmml" xref="S2.p1.4.m3.1.1">subscript</csymbol><ci id="S2.p1.4.m3.1.1.2.cmml" xref="S2.p1.4.m3.1.1.2">ğ‘›</ci><ci id="S2.p1.4.m3.1.1.3.cmml" xref="S2.p1.4.m3.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.4.m3.1c">n_{k}</annotation></semantics></math> is the number of training data on client <math id="S2.p1.5.m4.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S2.p1.5.m4.1a"><mi id="S2.p1.5.m4.1.1" xref="S2.p1.5.m4.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S2.p1.5.m4.1b"><ci id="S2.p1.5.m4.1.1.cmml" xref="S2.p1.5.m4.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.5.m4.1c">k</annotation></semantics></math>; <math id="S2.p1.6.m5.1" class="ltx_Math" alttext="\mathcal{P}_{k}" display="inline"><semantics id="S2.p1.6.m5.1a"><msub id="S2.p1.6.m5.1.1" xref="S2.p1.6.m5.1.1.cmml"><mi class="ltx_font_mathcaligraphic" id="S2.p1.6.m5.1.1.2" xref="S2.p1.6.m5.1.1.2.cmml">ğ’«</mi><mi id="S2.p1.6.m5.1.1.3" xref="S2.p1.6.m5.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="S2.p1.6.m5.1b"><apply id="S2.p1.6.m5.1.1.cmml" xref="S2.p1.6.m5.1.1"><csymbol cd="ambiguous" id="S2.p1.6.m5.1.1.1.cmml" xref="S2.p1.6.m5.1.1">subscript</csymbol><ci id="S2.p1.6.m5.1.1.2.cmml" xref="S2.p1.6.m5.1.1.2">ğ’«</ci><ci id="S2.p1.6.m5.1.1.3.cmml" xref="S2.p1.6.m5.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.6.m5.1c">\mathcal{P}_{k}</annotation></semantics></math> is the set of indices of data points on client <math id="S2.p1.7.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S2.p1.7.m6.1a"><mi id="S2.p1.7.m6.1.1" xref="S2.p1.7.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S2.p1.7.m6.1b"><ci id="S2.p1.7.m6.1.1.cmml" xref="S2.p1.7.m6.1.1">ğ‘˜</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.7.m6.1c">k</annotation></semantics></math>, and <math id="S2.p1.8.m7.1" class="ltx_Math" alttext="n_{k}=|\mathcal{P}_{k}|" display="inline"><semantics id="S2.p1.8.m7.1a"><mrow id="S2.p1.8.m7.1.1" xref="S2.p1.8.m7.1.1.cmml"><msub id="S2.p1.8.m7.1.1.3" xref="S2.p1.8.m7.1.1.3.cmml"><mi id="S2.p1.8.m7.1.1.3.2" xref="S2.p1.8.m7.1.1.3.2.cmml">n</mi><mi id="S2.p1.8.m7.1.1.3.3" xref="S2.p1.8.m7.1.1.3.3.cmml">k</mi></msub><mo id="S2.p1.8.m7.1.1.2" xref="S2.p1.8.m7.1.1.2.cmml">=</mo><mrow id="S2.p1.8.m7.1.1.1.1" xref="S2.p1.8.m7.1.1.1.2.cmml"><mo stretchy="false" id="S2.p1.8.m7.1.1.1.1.2" xref="S2.p1.8.m7.1.1.1.2.1.cmml">|</mo><msub id="S2.p1.8.m7.1.1.1.1.1" xref="S2.p1.8.m7.1.1.1.1.1.cmml"><mi class="ltx_font_mathcaligraphic" id="S2.p1.8.m7.1.1.1.1.1.2" xref="S2.p1.8.m7.1.1.1.1.1.2.cmml">ğ’«</mi><mi id="S2.p1.8.m7.1.1.1.1.1.3" xref="S2.p1.8.m7.1.1.1.1.1.3.cmml">k</mi></msub><mo stretchy="false" id="S2.p1.8.m7.1.1.1.1.3" xref="S2.p1.8.m7.1.1.1.2.1.cmml">|</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.p1.8.m7.1b"><apply id="S2.p1.8.m7.1.1.cmml" xref="S2.p1.8.m7.1.1"><eq id="S2.p1.8.m7.1.1.2.cmml" xref="S2.p1.8.m7.1.1.2"></eq><apply id="S2.p1.8.m7.1.1.3.cmml" xref="S2.p1.8.m7.1.1.3"><csymbol cd="ambiguous" id="S2.p1.8.m7.1.1.3.1.cmml" xref="S2.p1.8.m7.1.1.3">subscript</csymbol><ci id="S2.p1.8.m7.1.1.3.2.cmml" xref="S2.p1.8.m7.1.1.3.2">ğ‘›</ci><ci id="S2.p1.8.m7.1.1.3.3.cmml" xref="S2.p1.8.m7.1.1.3.3">ğ‘˜</ci></apply><apply id="S2.p1.8.m7.1.1.1.2.cmml" xref="S2.p1.8.m7.1.1.1.1"><abs id="S2.p1.8.m7.1.1.1.2.1.cmml" xref="S2.p1.8.m7.1.1.1.1.2"></abs><apply id="S2.p1.8.m7.1.1.1.1.1.cmml" xref="S2.p1.8.m7.1.1.1.1.1"><csymbol cd="ambiguous" id="S2.p1.8.m7.1.1.1.1.1.1.cmml" xref="S2.p1.8.m7.1.1.1.1.1">subscript</csymbol><ci id="S2.p1.8.m7.1.1.1.1.1.2.cmml" xref="S2.p1.8.m7.1.1.1.1.1.2">ğ’«</ci><ci id="S2.p1.8.m7.1.1.1.1.1.3.cmml" xref="S2.p1.8.m7.1.1.1.1.1.3">ğ‘˜</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.8.m7.1c">n_{k}=|\mathcal{P}_{k}|</annotation></semantics></math>; <math id="S2.p1.9.m8.1" class="ltx_Math" alttext="F_{k}(\theta)" display="inline"><semantics id="S2.p1.9.m8.1a"><mrow id="S2.p1.9.m8.1.2" xref="S2.p1.9.m8.1.2.cmml"><msub id="S2.p1.9.m8.1.2.2" xref="S2.p1.9.m8.1.2.2.cmml"><mi id="S2.p1.9.m8.1.2.2.2" xref="S2.p1.9.m8.1.2.2.2.cmml">F</mi><mi id="S2.p1.9.m8.1.2.2.3" xref="S2.p1.9.m8.1.2.2.3.cmml">k</mi></msub><mo lspace="0em" rspace="0em" id="S2.p1.9.m8.1.2.1" xref="S2.p1.9.m8.1.2.1.cmml">â€‹</mo><mrow id="S2.p1.9.m8.1.2.3.2" xref="S2.p1.9.m8.1.2.cmml"><mo stretchy="false" id="S2.p1.9.m8.1.2.3.2.1" xref="S2.p1.9.m8.1.2.cmml">(</mo><mi id="S2.p1.9.m8.1.1" xref="S2.p1.9.m8.1.1.cmml">Î¸</mi><mo stretchy="false" id="S2.p1.9.m8.1.2.3.2.2" xref="S2.p1.9.m8.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.p1.9.m8.1b"><apply id="S2.p1.9.m8.1.2.cmml" xref="S2.p1.9.m8.1.2"><times id="S2.p1.9.m8.1.2.1.cmml" xref="S2.p1.9.m8.1.2.1"></times><apply id="S2.p1.9.m8.1.2.2.cmml" xref="S2.p1.9.m8.1.2.2"><csymbol cd="ambiguous" id="S2.p1.9.m8.1.2.2.1.cmml" xref="S2.p1.9.m8.1.2.2">subscript</csymbol><ci id="S2.p1.9.m8.1.2.2.2.cmml" xref="S2.p1.9.m8.1.2.2.2">ğ¹</ci><ci id="S2.p1.9.m8.1.2.2.3.cmml" xref="S2.p1.9.m8.1.2.2.3">ğ‘˜</ci></apply><ci id="S2.p1.9.m8.1.1.cmml" xref="S2.p1.9.m8.1.1">ğœƒ</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.9.m8.1c">F_{k}(\theta)</annotation></semantics></math> is the local objective function; <math id="S2.p1.10.m9.3" class="ltx_Math" alttext="l(x_{i},y_{i},\theta)" display="inline"><semantics id="S2.p1.10.m9.3a"><mrow id="S2.p1.10.m9.3.3" xref="S2.p1.10.m9.3.3.cmml"><mi id="S2.p1.10.m9.3.3.4" xref="S2.p1.10.m9.3.3.4.cmml">l</mi><mo lspace="0em" rspace="0em" id="S2.p1.10.m9.3.3.3" xref="S2.p1.10.m9.3.3.3.cmml">â€‹</mo><mrow id="S2.p1.10.m9.3.3.2.2" xref="S2.p1.10.m9.3.3.2.3.cmml"><mo stretchy="false" id="S2.p1.10.m9.3.3.2.2.3" xref="S2.p1.10.m9.3.3.2.3.cmml">(</mo><msub id="S2.p1.10.m9.2.2.1.1.1" xref="S2.p1.10.m9.2.2.1.1.1.cmml"><mi id="S2.p1.10.m9.2.2.1.1.1.2" xref="S2.p1.10.m9.2.2.1.1.1.2.cmml">x</mi><mi id="S2.p1.10.m9.2.2.1.1.1.3" xref="S2.p1.10.m9.2.2.1.1.1.3.cmml">i</mi></msub><mo id="S2.p1.10.m9.3.3.2.2.4" xref="S2.p1.10.m9.3.3.2.3.cmml">,</mo><msub id="S2.p1.10.m9.3.3.2.2.2" xref="S2.p1.10.m9.3.3.2.2.2.cmml"><mi id="S2.p1.10.m9.3.3.2.2.2.2" xref="S2.p1.10.m9.3.3.2.2.2.2.cmml">y</mi><mi id="S2.p1.10.m9.3.3.2.2.2.3" xref="S2.p1.10.m9.3.3.2.2.2.3.cmml">i</mi></msub><mo id="S2.p1.10.m9.3.3.2.2.5" xref="S2.p1.10.m9.3.3.2.3.cmml">,</mo><mi id="S2.p1.10.m9.1.1" xref="S2.p1.10.m9.1.1.cmml">Î¸</mi><mo stretchy="false" id="S2.p1.10.m9.3.3.2.2.6" xref="S2.p1.10.m9.3.3.2.3.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S2.p1.10.m9.3b"><apply id="S2.p1.10.m9.3.3.cmml" xref="S2.p1.10.m9.3.3"><times id="S2.p1.10.m9.3.3.3.cmml" xref="S2.p1.10.m9.3.3.3"></times><ci id="S2.p1.10.m9.3.3.4.cmml" xref="S2.p1.10.m9.3.3.4">ğ‘™</ci><vector id="S2.p1.10.m9.3.3.2.3.cmml" xref="S2.p1.10.m9.3.3.2.2"><apply id="S2.p1.10.m9.2.2.1.1.1.cmml" xref="S2.p1.10.m9.2.2.1.1.1"><csymbol cd="ambiguous" id="S2.p1.10.m9.2.2.1.1.1.1.cmml" xref="S2.p1.10.m9.2.2.1.1.1">subscript</csymbol><ci id="S2.p1.10.m9.2.2.1.1.1.2.cmml" xref="S2.p1.10.m9.2.2.1.1.1.2">ğ‘¥</ci><ci id="S2.p1.10.m9.2.2.1.1.1.3.cmml" xref="S2.p1.10.m9.2.2.1.1.1.3">ğ‘–</ci></apply><apply id="S2.p1.10.m9.3.3.2.2.2.cmml" xref="S2.p1.10.m9.3.3.2.2.2"><csymbol cd="ambiguous" id="S2.p1.10.m9.3.3.2.2.2.1.cmml" xref="S2.p1.10.m9.3.3.2.2.2">subscript</csymbol><ci id="S2.p1.10.m9.3.3.2.2.2.2.cmml" xref="S2.p1.10.m9.3.3.2.2.2.2">ğ‘¦</ci><ci id="S2.p1.10.m9.3.3.2.2.2.3.cmml" xref="S2.p1.10.m9.3.3.2.2.2.3">ğ‘–</ci></apply><ci id="S2.p1.10.m9.1.1.cmml" xref="S2.p1.10.m9.1.1">ğœƒ</ci></vector></apply></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.10.m9.3c">l(x_{i},y_{i},\theta)</annotation></semantics></math> is the loss of the prediction on sample <math id="S2.p1.11.m10.2" class="ltx_Math" alttext="(x_{i},y_{i})" display="inline"><semantics id="S2.p1.11.m10.2a"><mrow id="S2.p1.11.m10.2.2.2" xref="S2.p1.11.m10.2.2.3.cmml"><mo stretchy="false" id="S2.p1.11.m10.2.2.2.3" xref="S2.p1.11.m10.2.2.3.cmml">(</mo><msub id="S2.p1.11.m10.1.1.1.1" xref="S2.p1.11.m10.1.1.1.1.cmml"><mi id="S2.p1.11.m10.1.1.1.1.2" xref="S2.p1.11.m10.1.1.1.1.2.cmml">x</mi><mi id="S2.p1.11.m10.1.1.1.1.3" xref="S2.p1.11.m10.1.1.1.1.3.cmml">i</mi></msub><mo id="S2.p1.11.m10.2.2.2.4" xref="S2.p1.11.m10.2.2.3.cmml">,</mo><msub id="S2.p1.11.m10.2.2.2.2" xref="S2.p1.11.m10.2.2.2.2.cmml"><mi id="S2.p1.11.m10.2.2.2.2.2" xref="S2.p1.11.m10.2.2.2.2.2.cmml">y</mi><mi id="S2.p1.11.m10.2.2.2.2.3" xref="S2.p1.11.m10.2.2.2.2.3.cmml">i</mi></msub><mo stretchy="false" id="S2.p1.11.m10.2.2.2.5" xref="S2.p1.11.m10.2.2.3.cmml">)</mo></mrow><annotation-xml encoding="MathML-Content" id="S2.p1.11.m10.2b"><interval closure="open" id="S2.p1.11.m10.2.2.3.cmml" xref="S2.p1.11.m10.2.2.2"><apply id="S2.p1.11.m10.1.1.1.1.cmml" xref="S2.p1.11.m10.1.1.1.1"><csymbol cd="ambiguous" id="S2.p1.11.m10.1.1.1.1.1.cmml" xref="S2.p1.11.m10.1.1.1.1">subscript</csymbol><ci id="S2.p1.11.m10.1.1.1.1.2.cmml" xref="S2.p1.11.m10.1.1.1.1.2">ğ‘¥</ci><ci id="S2.p1.11.m10.1.1.1.1.3.cmml" xref="S2.p1.11.m10.1.1.1.1.3">ğ‘–</ci></apply><apply id="S2.p1.11.m10.2.2.2.2.cmml" xref="S2.p1.11.m10.2.2.2.2"><csymbol cd="ambiguous" id="S2.p1.11.m10.2.2.2.2.1.cmml" xref="S2.p1.11.m10.2.2.2.2">subscript</csymbol><ci id="S2.p1.11.m10.2.2.2.2.2.cmml" xref="S2.p1.11.m10.2.2.2.2.2">ğ‘¦</ci><ci id="S2.p1.11.m10.2.2.2.2.3.cmml" xref="S2.p1.11.m10.2.2.2.2.3">ğ‘–</ci></apply></interval></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.11.m10.2c">(x_{i},y_{i})</annotation></semantics></math>.</p>
</div>
<div id="S2.p2" class="ltx_para">
<p id="S2.p2.3" class="ltx_p">The traditional centralized FL training process is detailed in AlgorithmÂ <a href="#alg1" title="Algorithm 1 â€£ 1 Introduction â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>, it involves <math id="S2.p2.1.m1.1" class="ltx_Math" alttext="T" display="inline"><semantics id="S2.p2.1.m1.1a"><mi id="S2.p2.1.m1.1.1" xref="S2.p2.1.m1.1.1.cmml">T</mi><annotation-xml encoding="MathML-Content" id="S2.p2.1.m1.1b"><ci id="S2.p2.1.m1.1.1.cmml" xref="S2.p2.1.m1.1.1">ğ‘‡</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.p2.1.m1.1c">T</annotation></semantics></math> communication rounds between server and clients.
Specifically, in the <math id="S2.p2.2.m2.1" class="ltx_Math" alttext="t" display="inline"><semantics id="S2.p2.2.m2.1a"><mi id="S2.p2.2.m2.1.1" xref="S2.p2.2.m2.1.1.cmml">t</mi><annotation-xml encoding="MathML-Content" id="S2.p2.2.m2.1b"><ci id="S2.p2.2.m2.1.1.cmml" xref="S2.p2.2.m2.1.1">ğ‘¡</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.p2.2.m2.1c">t</annotation></semantics></math>-th communication round, each client first downloads the current global model from the server.
Then each client trains its local model using the local dataset for <math id="S2.p2.3.m3.1" class="ltx_Math" alttext="E" display="inline"><semantics id="S2.p2.3.m3.1a"><mi id="S2.p2.3.m3.1.1" xref="S2.p2.3.m3.1.1.cmml">E</mi><annotation-xml encoding="MathML-Content" id="S2.p2.3.m3.1b"><ci id="S2.p2.3.m3.1.1.cmml" xref="S2.p2.3.m3.1.1">ğ¸</ci></annotation-xml><annotation encoding="application/x-tex" id="S2.p2.3.m3.1c">E</annotation></semantics></math> local epochs.
Next, the server collects the model updates of all selected clients and aggregates them into a new global model.
FL training is accomplished by repeating the above round until the global model meets the desired performance criteria.</p>
</div>
<figure id="S2.F1" class="ltx_figure"><img src="/html/2409.09727/assets/x1.png" id="S2.F1.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="276" height="148" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>Difference between centralized FL paradigm and decentralized FL paradigm. Centralized FL relies on a central server to manage the training. While decentralized FL eliminates the need for a central server. Instead, clients can directly communicate with connected ones.</figcaption>
</figure>
<div id="S2.p3" class="ltx_para">
<p id="S2.p3.1" class="ltx_p">In practice, the rapid development of FL has propelled the field beyond the traditional centralized paradigm, as shown in FigureÂ <a href="#S2.F1" title="Figure 1 â€£ 2 Preliminaries â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
For instance, the integration of blockchainÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib23" title="" class="ltx_ref">23</a></sup></cite> and swarm learningÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib24" title="" class="ltx_ref">24</a></sup></cite> has transitioned FL towards decentralized paradigms, such as peer-to-peer, sequential, and cyclic computing, which enhance data privacy, security, and traceability by enabling secure data transactions and consensus mechanisms.
Throughout this evolution, the scope of updates exchanged during communication has expanded.
The updates now encompass not only model parameters or gradients but also partial model parametersÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib25" title="" class="ltx_ref">25</a></sup></cite>, statistical informationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib26" title="" class="ltx_ref">26</a></sup></cite>, and predictions from knowledge distillation techniques, such as logitsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib27" title="" class="ltx_ref">27</a></sup></cite>.
This expansion helps reduce communication costs, enhance privacy, and enable multi-task learning where only certain parameters are updated collaboratively.</p>
</div>
<div id="S2.p4" class="ltx_para">
<p id="S2.p4.1" class="ltx_p">Beyond centralized or decentralized topologies, FL has evolved to address complex scenarios caused by various feature and sample distributions, leading to the development of <span id="S2.p4.1.1" class="ltx_text ltx_font_italic">Horizontal Federated Learning (HFL)</span>, where the data from different clients has significant overlap in the feature space but little overlap in the sample space;
<span id="S2.p4.1.2" class="ltx_text ltx_font_italic">Vertical Federated Learning (VFL)</span>, where the data from different clients has very little overlap in the feature space while significant overlap in the sample space;
and <span id="S2.p4.1.3" class="ltx_text ltx_font_italic">Federated Transfer Learning (FTL)</span>, which leverages knowledge transfer to manage scenarios where clients have little overlap in both feature and sample spaces.
FigureÂ <a href="#S2.F2" title="Figure 2 â€£ 2 Preliminaries â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> provides a clear illustration of the differences among HFL, VFL, and FTL.
HFL is the most common scenario in FL studies. For example, a vast number of people use wearables, such as the Apple Watch, to monitor their health conditions. These sensors generate massive amounts of data that share the same feature space, which can be used for collaborative model training. See Clients 1 and 2 in FigureÂ <a href="#S2.F2" title="Figure 2 â€£ 2 Preliminaries â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> for an example.
VFL aligns well with real-world applications. For instance, a patient may have medical records at different hospitals, each containing distinct features. With many samples, these hospitals will possess more comprehensive feature sets, enabling them to jointly learn the model more accurately. Refer to FigureÂ <a href="#S2.F2" title="Figure 2 â€£ 2 Preliminaries â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> to see Clients 1 and 3 for an example.
FTL is also well-suited for healthcare scenarios, especially when dealing with tabular EHR dataÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib28" title="" class="ltx_ref">28</a></sup></cite>. For example, see Clients 2 and 4 in FigureÂ <a href="#S2.F2" title="Figure 2 â€£ 2 Preliminaries â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>, where there is only a small intersection in the feature space, different clients have varying combinations of healthcare data and patients.</p>
</div>
<div id="S2.p5" class="ltx_para">
<p id="S2.p5.1" class="ltx_p">FL can be further categorized into cross-silo FL and cross-device FL based on the scale and attributes of participating clients.
<span id="S2.p5.1.1" class="ltx_text ltx_font_italic">Cross-silo FL</span> is tailored for scenarios where a limited number of participating clients, such as various hospitals, medical centres, and institutions, engage in all stages of FL training.
<span id="S2.p5.1.2" class="ltx_text ltx_font_italic">Cross-device FL</span>, on the other hand, is designed for scenarios with a multitude of participating clients, typically devices with limited data storage capabilities, such as wearables and Internet of Medical Things (IoMT) devices.</p>
</div>
<figure id="S2.F2" class="ltx_figure"><img src="/html/2409.09727/assets/x2.png" id="S2.F2.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="276" height="180" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Visual representation of three categories of FL, illustrating their distribution across feature and sample spaces.</figcaption>
</figure>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Method</h2>

<div id="S3.p1" class="ltx_para">
<p id="S3.p1.1" class="ltx_p">Our review was conducted in accordance with the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines.As shown in FigureÂ <a href="#S3.F3" title="Figure 3 â€£ 3.2 Study Selection â€£ 3 Method â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, the flow diagram outlines the search, inclusion and exclusion procedures.
We carried out a comprehensive search of the most recent studies focusing on advanced FL technologies within healthcare domain up to May 2024.</p>
</div>
<section id="S3.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.1 </span>Literature Search</h3>

<div id="S3.SS1.p1" class="ltx_para">
<p id="S3.SS1.p1.1" class="ltx_p">We conducted a systematic search using PubMed, Web of Science, Scopus, Science Direct, IEEEXplore, ArXiv, Springerlink, ACM Digital Library, and Google Scholar.
Any study up to May 2024 that involved the use of FL technologies in healthcare based on a simulated or real distributed scenario was included.
The search phrases included the following keywords: â€œFederate Learningâ€, â€œHealthcareâ€, â€œPrivacy-Preservingâ€, â€œMedicalâ€, â€œBiomedicalâ€, â€œDecentralized Learningâ€, and â€œPrivacyâ€, using Boolean operators such as â€œAND/ORâ€ and various combinations of these keywords.
As shown in FigureÂ <a href="#S3.F3" title="Figure 3 â€£ 3.2 Study Selection â€£ 3 Method â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, initially, a total of 1,149 studies were identified.</p>
</div>
</section>
<section id="S3.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.2 </span>Study Selection</h3>

<div id="S3.SS2.p1" class="ltx_para">
<p id="S3.SS2.p1.1" class="ltx_p">We defined clear and transparent inclusion and exclusion criteria as follows.
<span id="S3.SS2.p1.1.1" class="ltx_text ltx_font_italic">Inclusion criteria:</span>
(1) Studies involving the implementation of FL in the healthcare domain;
(2) Studies that, while not explicitly focused on healthcare applications, involve or utilize healthcare data or scenarios in their experiments;
(3) Studies on the design or optimization of FL frameworks/workflows that cover the healthcare domain;
(4) Studies in English language.
<span id="S3.SS2.p1.1.2" class="ltx_text ltx_font_italic">Exclusion criteria:</span>
(1) Duplicate studies;
(2) Studies such as surveys, reviews, opinions, editorial letters, book chapters, and theses;
(3) Studies unrelated to FL or those using FL for non-healthcare applications;
(4) Non-English language studies.</p>
</div>
<figure id="S3.F3" class="ltx_figure"><img src="/html/2409.09727/assets/x3.png" id="S3.F3.g1" class="ltx_graphics ltx_centering ltx_img_square" width="276" height="239" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>PRISMA flow diagram for our review, highlighting the inclusion and exclusion of studies at each stage.</figcaption>
</figure>
<div id="S3.SS2.p2" class="ltx_para">
<p id="S3.SS2.p2.1" class="ltx_p">Based on the above criteria, the screening procedure was conducted independently by two groups of authors (Group A: M.L. and P.X.; Group B: J.H. and Z.T.) to eliminate bias and ambiguity.
Two groups confirmed the selected studies and resolved any conflicts or inconsistencies through discussion between the groups.
The study selection process is outlined in FigureÂ <a href="#S3.F3" title="Figure 3 â€£ 3.2 Study Selection â€£ 3 Method â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>.
Initially, a total of 43 duplicate studies were removed.
Subsequently, the titles and abstracts were carefully screened, leading to the exclusion of 402 unqualified and irrelevant studies.
Next, the eligibility of the remaining 704 studies was assessed through full-text screening.
Finally, after further evaluation, 597 studies were deemed ineligible, and 107 studies were included in our final review.</p>
</div>
</section>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Results</h2>

<section id="S4.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Application and Data</h3>

<div id="S4.SS1.p1" class="ltx_para">
<p id="S4.SS1.p1.1" class="ltx_p">Included studies explored a broad range of healthcare specialties, including general medicineÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib6" title="" class="ltx_ref">6</a></sup></cite>, cardiologyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib29" title="" class="ltx_ref">29</a></sup></cite>, oncologyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib30" title="" class="ltx_ref">30</a></sup></cite>, ophthalmologyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib31" title="" class="ltx_ref">31</a></sup></cite>, drug discoveryÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib15" title="" class="ltx_ref">15</a></sup></cite>, multiomicsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib32" title="" class="ltx_ref">32</a></sup></cite>, dermatologyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib33" title="" class="ltx_ref">33</a></sup></cite>, and radiologyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib34" title="" class="ltx_ref">34</a></sup></cite>.
Most studies focused on tasks such as classification (67/107), segmentation (20/107), and detection (8/107), with additional applications in regressionÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib35" title="" class="ltx_ref">35</a></sup></cite>, clusteringÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib32" title="" class="ltx_ref">32</a></sup></cite>, reconstructionÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib36" title="" class="ltx_ref">36</a></sup>, <sup class="ltx_sup"><a href="#bib.bib37" title="" class="ltx_ref">37</a></sup></cite>, feature selectionÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib38" title="" class="ltx_ref">38</a></sup>, <sup class="ltx_sup"><a href="#bib.bib39" title="" class="ltx_ref">39</a></sup></cite>, data synthesisÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib40" title="" class="ltx_ref">40</a></sup>, <sup class="ltx_sup"><a href="#bib.bib41" title="" class="ltx_ref">41</a></sup>, <sup class="ltx_sup"><a href="#bib.bib42" title="" class="ltx_ref">42</a></sup></cite>, and biomedical language processÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib43" title="" class="ltx_ref">43</a></sup></cite>.
In terms of data types, medical imaging, including MRI, CT, and X-rays, was the most frequently used (55/107), followed by clinical data and electronic health records (EHR) (24/107), skin images (6/107), retinal images (6/107), histopathology slides (16/107), multiomics data (3/107), and biomedical language data (1/107). Some studies involved multiple data types, while 8 studies did not specify the type of data used or used non-healthcare dataÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib44" title="" class="ltx_ref">44</a></sup>, <sup class="ltx_sup"><a href="#bib.bib45" title="" class="ltx_ref">45</a></sup>, <sup class="ltx_sup"><a href="#bib.bib46" title="" class="ltx_ref">46</a></sup></cite>.</p>
</div>
<figure id="S4.T1" class="ltx_table">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 1: </span>Key results of included studies.</figcaption>
<div id="S4.T1.9.9" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:18.1pt;height:332.9pt;vertical-align:-0.0pt;"><span class="ltx_transformed_inner" style="transform:translate(-320.0pt,5894.9pt) scale(0.0274607644363595,0.0274607644363595) ;">
<table id="S4.T1.9.9.9" class="ltx_tabular ltx_align_middle">
<tr id="S4.T1.3.3.3.3" class="ltx_tr">
<td id="S4.T1.3.3.3.3.4" class="ltx_td ltx_align_left ltx_border_tt">Item</td>
<td id="S4.T1.1.1.1.1.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_tt" style="width:256.1pt;"><span id="S4.T1.1.1.1.1.1.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S4.T1.1.1.1.1.1.1.p1" class="ltx_para ltx_noindent">
<span id="S4.T1.1.1.1.1.1.1.p1.1" class="ltx_p"><span id="S4.T1.1.1.1.1.1.1.p1.1.1" class="ltx_text"></span><span id="S4.T1.1.1.1.1.1.1.p1.1.2" class="ltx_text">
<span id="S4.T1.1.1.1.1.1.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S4.T1.1.1.1.1.1.1.p1.1.2.1.1" class="ltx_tr">
<span id="S4.T1.1.1.1.1.1.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">Characteristics</span></span>
</span></span><span id="S4.T1.1.1.1.1.1.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S4.T1.2.2.2.2.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_tt" style="width:99.6pt;"><span id="S4.T1.2.2.2.2.2.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S4.T1.2.2.2.2.2.1.p1" class="ltx_para ltx_noindent">
<span id="S4.T1.2.2.2.2.2.1.p1.1" class="ltx_p"><span id="S4.T1.2.2.2.2.2.1.p1.1.1" class="ltx_text"></span><span id="S4.T1.2.2.2.2.2.1.p1.1.2" class="ltx_text">
<span id="S4.T1.2.2.2.2.2.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S4.T1.2.2.2.2.2.1.p1.1.2.1.1" class="ltx_tr">
<span id="S4.T1.2.2.2.2.2.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">Number of Study</span></span>
</span></span><span id="S4.T1.2.2.2.2.2.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S4.T1.3.3.3.3.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_tt" style="width:56.9pt;"><span id="S4.T1.3.3.3.3.3.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S4.T1.3.3.3.3.3.1.p1" class="ltx_para ltx_noindent">
<span id="S4.T1.3.3.3.3.3.1.p1.1" class="ltx_p"><span id="S4.T1.3.3.3.3.3.1.p1.1.1" class="ltx_text"></span><span id="S4.T1.3.3.3.3.3.1.p1.1.2" class="ltx_text">
<span id="S4.T1.3.3.3.3.3.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S4.T1.3.3.3.3.3.1.p1.1.2.1.1" class="ltx_tr">
<span id="S4.T1.3.3.3.3.3.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_left">Examples</span></span>
</span></span><span id="S4.T1.3.3.3.3.3.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
</tr>
<tr id="S4.T1.6.6.6.6" class="ltx_tr">
<td id="S4.T1.6.6.6.6.4" class="ltx_td ltx_align_left ltx_border_t">Cohort Size</td>
<td id="S4.T1.6.6.6.6.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.6.6.6.6.3.3" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.4.4.4.4.1.1.1" class="ltx_p"><math id="S4.T1.4.4.4.4.1.1.1.m1.1" class="ltx_Math" alttext="\leq 100" display="inline"><semantics id="S4.T1.4.4.4.4.1.1.1.m1.1a"><mrow id="S4.T1.4.4.4.4.1.1.1.m1.1.1" xref="S4.T1.4.4.4.4.1.1.1.m1.1.1.cmml"><mi id="S4.T1.4.4.4.4.1.1.1.m1.1.1.2" xref="S4.T1.4.4.4.4.1.1.1.m1.1.1.2.cmml"></mi><mo id="S4.T1.4.4.4.4.1.1.1.m1.1.1.1" xref="S4.T1.4.4.4.4.1.1.1.m1.1.1.1.cmml">â‰¤</mo><mn id="S4.T1.4.4.4.4.1.1.1.m1.1.1.3" xref="S4.T1.4.4.4.4.1.1.1.m1.1.1.3.cmml">100</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.4.4.4.4.1.1.1.m1.1b"><apply id="S4.T1.4.4.4.4.1.1.1.m1.1.1.cmml" xref="S4.T1.4.4.4.4.1.1.1.m1.1.1"><leq id="S4.T1.4.4.4.4.1.1.1.m1.1.1.1.cmml" xref="S4.T1.4.4.4.4.1.1.1.m1.1.1.1"></leq><csymbol cd="latexml" id="S4.T1.4.4.4.4.1.1.1.m1.1.1.2.cmml" xref="S4.T1.4.4.4.4.1.1.1.m1.1.1.2">absent</csymbol><cn type="integer" id="S4.T1.4.4.4.4.1.1.1.m1.1.1.3.cmml" xref="S4.T1.4.4.4.4.1.1.1.m1.1.1.3">100</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.4.4.4.4.1.1.1.m1.1c">\leq 100</annotation></semantics></math></span>
<span id="S4.T1.5.5.5.5.2.2.2" class="ltx_p"><math id="S4.T1.5.5.5.5.2.2.2.m1.1" class="ltx_Math" alttext="100-1000" display="inline"><semantics id="S4.T1.5.5.5.5.2.2.2.m1.1a"><mrow id="S4.T1.5.5.5.5.2.2.2.m1.1.1" xref="S4.T1.5.5.5.5.2.2.2.m1.1.1.cmml"><mn id="S4.T1.5.5.5.5.2.2.2.m1.1.1.2" xref="S4.T1.5.5.5.5.2.2.2.m1.1.1.2.cmml">100</mn><mo id="S4.T1.5.5.5.5.2.2.2.m1.1.1.1" xref="S4.T1.5.5.5.5.2.2.2.m1.1.1.1.cmml">âˆ’</mo><mn id="S4.T1.5.5.5.5.2.2.2.m1.1.1.3" xref="S4.T1.5.5.5.5.2.2.2.m1.1.1.3.cmml">1000</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.5.5.5.5.2.2.2.m1.1b"><apply id="S4.T1.5.5.5.5.2.2.2.m1.1.1.cmml" xref="S4.T1.5.5.5.5.2.2.2.m1.1.1"><minus id="S4.T1.5.5.5.5.2.2.2.m1.1.1.1.cmml" xref="S4.T1.5.5.5.5.2.2.2.m1.1.1.1"></minus><cn type="integer" id="S4.T1.5.5.5.5.2.2.2.m1.1.1.2.cmml" xref="S4.T1.5.5.5.5.2.2.2.m1.1.1.2">100</cn><cn type="integer" id="S4.T1.5.5.5.5.2.2.2.m1.1.1.3.cmml" xref="S4.T1.5.5.5.5.2.2.2.m1.1.1.3">1000</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.5.5.5.5.2.2.2.m1.1c">100-1000</annotation></semantics></math></span>
<span id="S4.T1.6.6.6.6.3.3.3" class="ltx_p"><math id="S4.T1.6.6.6.6.3.3.3.m1.1" class="ltx_Math" alttext="\geq 1000" display="inline"><semantics id="S4.T1.6.6.6.6.3.3.3.m1.1a"><mrow id="S4.T1.6.6.6.6.3.3.3.m1.1.1" xref="S4.T1.6.6.6.6.3.3.3.m1.1.1.cmml"><mi id="S4.T1.6.6.6.6.3.3.3.m1.1.1.2" xref="S4.T1.6.6.6.6.3.3.3.m1.1.1.2.cmml"></mi><mo id="S4.T1.6.6.6.6.3.3.3.m1.1.1.1" xref="S4.T1.6.6.6.6.3.3.3.m1.1.1.1.cmml">â‰¥</mo><mn id="S4.T1.6.6.6.6.3.3.3.m1.1.1.3" xref="S4.T1.6.6.6.6.3.3.3.m1.1.1.3.cmml">1000</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.6.6.6.6.3.3.3.m1.1b"><apply id="S4.T1.6.6.6.6.3.3.3.m1.1.1.cmml" xref="S4.T1.6.6.6.6.3.3.3.m1.1.1"><geq id="S4.T1.6.6.6.6.3.3.3.m1.1.1.1.cmml" xref="S4.T1.6.6.6.6.3.3.3.m1.1.1.1"></geq><csymbol cd="latexml" id="S4.T1.6.6.6.6.3.3.3.m1.1.1.2.cmml" xref="S4.T1.6.6.6.6.3.3.3.m1.1.1.2">absent</csymbol><cn type="integer" id="S4.T1.6.6.6.6.3.3.3.m1.1.1.3.cmml" xref="S4.T1.6.6.6.6.3.3.3.m1.1.1.3">1000</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.6.6.6.6.3.3.3.m1.1c">\geq 1000</annotation></semantics></math></span>
<span id="S4.T1.6.6.6.6.3.3.4" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.6.6.6.6.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.6.6.6.6.5.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.6.6.6.6.5.1.1" class="ltx_p">6</span>
<span id="S4.T1.6.6.6.6.5.1.2" class="ltx_p">9</span>
<span id="S4.T1.6.6.6.6.5.1.3" class="ltx_p">24</span>
<span id="S4.T1.6.6.6.6.5.1.4" class="ltx_p">68</span>
</span>
</td>
<td id="S4.T1.6.6.6.6.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.6.6.6.6.6.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.6.6.6.6.6.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.6.6.6.6.6.1.1.1.1" class="ltx_sup"><a href="#bib.bib23" title="" class="ltx_ref">23</a>, <a href="#bib.bib47" title="" class="ltx_ref">47</a>, <a href="#bib.bib48" title="" class="ltx_ref">48</a></sup></cite></span>
<span id="S4.T1.6.6.6.6.6.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.6.6.6.6.6.1.2.1.1" class="ltx_sup"><a href="#bib.bib5" title="" class="ltx_ref">5</a>, <a href="#bib.bib6" title="" class="ltx_ref">6</a>, <a href="#bib.bib7" title="" class="ltx_ref">7</a></sup></cite></span>
<span id="S4.T1.6.6.6.6.6.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.6.6.6.6.6.1.3.1.1" class="ltx_sup"><a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib35" title="" class="ltx_ref">35</a>, <a href="#bib.bib14" title="" class="ltx_ref">14</a></sup></cite></span>
<span id="S4.T1.6.6.6.6.6.1.4" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.6.6.6.6.6.1.4.1.1" class="ltx_sup"><a href="#bib.bib49" title="" class="ltx_ref">49</a>, <a href="#bib.bib50" title="" class="ltx_ref">50</a>, <a href="#bib.bib36" title="" class="ltx_ref">36</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.10" class="ltx_tr">
<td id="S4.T1.9.9.9.10.1" class="ltx_td ltx_align_left ltx_border_t">Task</td>
<td id="S4.T1.9.9.9.10.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.10.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.10.2.1.1" class="ltx_p">Classification</span>
<span id="S4.T1.9.9.9.10.2.1.2" class="ltx_p">Segmentation</span>
<span id="S4.T1.9.9.9.10.2.1.3" class="ltx_p">Detection</span>
<span id="S4.T1.9.9.9.10.2.1.4" class="ltx_p">Others</span>
</span>
</td>
<td id="S4.T1.9.9.9.10.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.10.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.10.3.1.1" class="ltx_p">74</span>
<span id="S4.T1.9.9.9.10.3.1.2" class="ltx_p">22</span>
<span id="S4.T1.9.9.9.10.3.1.3" class="ltx_p">8</span>
<span id="S4.T1.9.9.9.10.3.1.4" class="ltx_p">10</span>
</span>
</td>
<td id="S4.T1.9.9.9.10.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.10.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.10.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.10.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib51" title="" class="ltx_ref">51</a>, <a href="#bib.bib52" title="" class="ltx_ref">52</a>, <a href="#bib.bib53" title="" class="ltx_ref">53</a></sup></cite></span>
<span id="S4.T1.9.9.9.10.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.10.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib54" title="" class="ltx_ref">54</a>, <a href="#bib.bib55" title="" class="ltx_ref">55</a>, <a href="#bib.bib56" title="" class="ltx_ref">56</a></sup></cite></span>
<span id="S4.T1.9.9.9.10.4.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.10.4.1.3.1.1" class="ltx_sup"><a href="#bib.bib19" title="" class="ltx_ref">19</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib58" title="" class="ltx_ref">58</a></sup></cite></span>
<span id="S4.T1.9.9.9.10.4.1.4" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.10.4.1.4.1.1" class="ltx_sup"><a href="#bib.bib35" title="" class="ltx_ref">35</a>, <a href="#bib.bib32" title="" class="ltx_ref">32</a>, <a href="#bib.bib37" title="" class="ltx_ref">37</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.11" class="ltx_tr">
<td id="S4.T1.9.9.9.11.1" class="ltx_td ltx_align_left ltx_border_t">Data Type</td>
<td id="S4.T1.9.9.9.11.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.11.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.11.2.1.1" class="ltx_p">Medical Imaging (MRI, CT, X-rays)</span>
<span id="S4.T1.9.9.9.11.2.1.2" class="ltx_p">Clinical and EHR</span>
<span id="S4.T1.9.9.9.11.2.1.3" class="ltx_p">Skin Images</span>
<span id="S4.T1.9.9.9.11.2.1.4" class="ltx_p">Retinal Images</span>
<span id="S4.T1.9.9.9.11.2.1.5" class="ltx_p">Histopathology Slides</span>
<span id="S4.T1.9.9.9.11.2.1.6" class="ltx_p">Multiomics</span>
<span id="S4.T1.9.9.9.11.2.1.7" class="ltx_p">Biomedical Language</span>
<span id="S4.T1.9.9.9.11.2.1.8" class="ltx_p">unavailable or non-healthcare</span>
</span>
</td>
<td id="S4.T1.9.9.9.11.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.11.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.11.3.1.1" class="ltx_p">55</span>
<span id="S4.T1.9.9.9.11.3.1.2" class="ltx_p">24</span>
<span id="S4.T1.9.9.9.11.3.1.3" class="ltx_p">6</span>
<span id="S4.T1.9.9.9.11.3.1.4" class="ltx_p">6</span>
<span id="S4.T1.9.9.9.11.3.1.5" class="ltx_p">16</span>
<span id="S4.T1.9.9.9.11.3.1.6" class="ltx_p">3</span>
<span id="S4.T1.9.9.9.11.3.1.7" class="ltx_p">1</span>
<span id="S4.T1.9.9.9.11.3.1.8" class="ltx_p">8</span>
</span>
</td>
<td id="S4.T1.9.9.9.11.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.11.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.11.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.11.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib59" title="" class="ltx_ref">59</a>, <a href="#bib.bib6" title="" class="ltx_ref">6</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a></sup></cite></span>
<span id="S4.T1.9.9.9.11.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.11.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib60" title="" class="ltx_ref">60</a>, <a href="#bib.bib61" title="" class="ltx_ref">61</a>, <a href="#bib.bib62" title="" class="ltx_ref">62</a></sup></cite></span>
<span id="S4.T1.9.9.9.11.4.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.11.4.1.3.1.1" class="ltx_sup"><a href="#bib.bib49" title="" class="ltx_ref">49</a>, <a href="#bib.bib63" title="" class="ltx_ref">63</a>, <a href="#bib.bib64" title="" class="ltx_ref">64</a></sup></cite></span>
<span id="S4.T1.9.9.9.11.4.1.4" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.11.4.1.4.1.1" class="ltx_sup"><a href="#bib.bib65" title="" class="ltx_ref">65</a>, <a href="#bib.bib63" title="" class="ltx_ref">63</a>, <a href="#bib.bib56" title="" class="ltx_ref">56</a></sup></cite></span>
<span id="S4.T1.9.9.9.11.4.1.5" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.11.4.1.5.1.1" class="ltx_sup"><a href="#bib.bib24" title="" class="ltx_ref">24</a>, <a href="#bib.bib66" title="" class="ltx_ref">66</a>, <a href="#bib.bib67" title="" class="ltx_ref">67</a></sup></cite></span>
<span id="S4.T1.9.9.9.11.4.1.6" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.11.4.1.6.1.1" class="ltx_sup"><a href="#bib.bib32" title="" class="ltx_ref">32</a>, <a href="#bib.bib68" title="" class="ltx_ref">68</a>, <a href="#bib.bib69" title="" class="ltx_ref">69</a></sup></cite></span>
<span id="S4.T1.9.9.9.11.4.1.7" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.11.4.1.7.1.1" class="ltx_sup"><a href="#bib.bib43" title="" class="ltx_ref">43</a></sup></cite></span>
<span id="S4.T1.9.9.9.11.4.1.8" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.11.4.1.8.1.1" class="ltx_sup"><a href="#bib.bib70" title="" class="ltx_ref">70</a>, <a href="#bib.bib46" title="" class="ltx_ref">46</a>, <a href="#bib.bib45" title="" class="ltx_ref">45</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.9" class="ltx_tr">
<td id="S4.T1.9.9.9.9.4" class="ltx_td ltx_align_left ltx_border_t">Number of Clients</td>
<td id="S4.T1.9.9.9.9.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.9.3.3" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.7.7.7.7.1.1.1" class="ltx_p"><math id="S4.T1.7.7.7.7.1.1.1.m1.1" class="ltx_Math" alttext="\leq 10" display="inline"><semantics id="S4.T1.7.7.7.7.1.1.1.m1.1a"><mrow id="S4.T1.7.7.7.7.1.1.1.m1.1.1" xref="S4.T1.7.7.7.7.1.1.1.m1.1.1.cmml"><mi id="S4.T1.7.7.7.7.1.1.1.m1.1.1.2" xref="S4.T1.7.7.7.7.1.1.1.m1.1.1.2.cmml"></mi><mo id="S4.T1.7.7.7.7.1.1.1.m1.1.1.1" xref="S4.T1.7.7.7.7.1.1.1.m1.1.1.1.cmml">â‰¤</mo><mn id="S4.T1.7.7.7.7.1.1.1.m1.1.1.3" xref="S4.T1.7.7.7.7.1.1.1.m1.1.1.3.cmml">10</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.7.7.7.7.1.1.1.m1.1b"><apply id="S4.T1.7.7.7.7.1.1.1.m1.1.1.cmml" xref="S4.T1.7.7.7.7.1.1.1.m1.1.1"><leq id="S4.T1.7.7.7.7.1.1.1.m1.1.1.1.cmml" xref="S4.T1.7.7.7.7.1.1.1.m1.1.1.1"></leq><csymbol cd="latexml" id="S4.T1.7.7.7.7.1.1.1.m1.1.1.2.cmml" xref="S4.T1.7.7.7.7.1.1.1.m1.1.1.2">absent</csymbol><cn type="integer" id="S4.T1.7.7.7.7.1.1.1.m1.1.1.3.cmml" xref="S4.T1.7.7.7.7.1.1.1.m1.1.1.3">10</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.7.7.7.7.1.1.1.m1.1c">\leq 10</annotation></semantics></math></span>
<span id="S4.T1.8.8.8.8.2.2.2" class="ltx_p"><math id="S4.T1.8.8.8.8.2.2.2.m1.1" class="ltx_Math" alttext="10-50" display="inline"><semantics id="S4.T1.8.8.8.8.2.2.2.m1.1a"><mrow id="S4.T1.8.8.8.8.2.2.2.m1.1.1" xref="S4.T1.8.8.8.8.2.2.2.m1.1.1.cmml"><mn id="S4.T1.8.8.8.8.2.2.2.m1.1.1.2" xref="S4.T1.8.8.8.8.2.2.2.m1.1.1.2.cmml">10</mn><mo id="S4.T1.8.8.8.8.2.2.2.m1.1.1.1" xref="S4.T1.8.8.8.8.2.2.2.m1.1.1.1.cmml">âˆ’</mo><mn id="S4.T1.8.8.8.8.2.2.2.m1.1.1.3" xref="S4.T1.8.8.8.8.2.2.2.m1.1.1.3.cmml">50</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.8.8.8.8.2.2.2.m1.1b"><apply id="S4.T1.8.8.8.8.2.2.2.m1.1.1.cmml" xref="S4.T1.8.8.8.8.2.2.2.m1.1.1"><minus id="S4.T1.8.8.8.8.2.2.2.m1.1.1.1.cmml" xref="S4.T1.8.8.8.8.2.2.2.m1.1.1.1"></minus><cn type="integer" id="S4.T1.8.8.8.8.2.2.2.m1.1.1.2.cmml" xref="S4.T1.8.8.8.8.2.2.2.m1.1.1.2">10</cn><cn type="integer" id="S4.T1.8.8.8.8.2.2.2.m1.1.1.3.cmml" xref="S4.T1.8.8.8.8.2.2.2.m1.1.1.3">50</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.8.8.8.8.2.2.2.m1.1c">10-50</annotation></semantics></math></span>
<span id="S4.T1.9.9.9.9.3.3.3" class="ltx_p"><math id="S4.T1.9.9.9.9.3.3.3.m1.1" class="ltx_Math" alttext="\geq 50" display="inline"><semantics id="S4.T1.9.9.9.9.3.3.3.m1.1a"><mrow id="S4.T1.9.9.9.9.3.3.3.m1.1.1" xref="S4.T1.9.9.9.9.3.3.3.m1.1.1.cmml"><mi id="S4.T1.9.9.9.9.3.3.3.m1.1.1.2" xref="S4.T1.9.9.9.9.3.3.3.m1.1.1.2.cmml"></mi><mo id="S4.T1.9.9.9.9.3.3.3.m1.1.1.1" xref="S4.T1.9.9.9.9.3.3.3.m1.1.1.1.cmml">â‰¥</mo><mn id="S4.T1.9.9.9.9.3.3.3.m1.1.1.3" xref="S4.T1.9.9.9.9.3.3.3.m1.1.1.3.cmml">50</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.9.9.9.9.3.3.3.m1.1b"><apply id="S4.T1.9.9.9.9.3.3.3.m1.1.1.cmml" xref="S4.T1.9.9.9.9.3.3.3.m1.1.1"><geq id="S4.T1.9.9.9.9.3.3.3.m1.1.1.1.cmml" xref="S4.T1.9.9.9.9.3.3.3.m1.1.1.1"></geq><csymbol cd="latexml" id="S4.T1.9.9.9.9.3.3.3.m1.1.1.2.cmml" xref="S4.T1.9.9.9.9.3.3.3.m1.1.1.2">absent</csymbol><cn type="integer" id="S4.T1.9.9.9.9.3.3.3.m1.1.1.3.cmml" xref="S4.T1.9.9.9.9.3.3.3.m1.1.1.3">50</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.9.9.9.9.3.3.3.m1.1c">\geq 50</annotation></semantics></math></span>
<span id="S4.T1.9.9.9.9.3.3.4" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.9.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.9.5.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.9.5.1.1" class="ltx_p">57</span>
<span id="S4.T1.9.9.9.9.5.1.2" class="ltx_p">17</span>
<span id="S4.T1.9.9.9.9.5.1.3" class="ltx_p">6</span>
<span id="S4.T1.9.9.9.9.5.1.4" class="ltx_p">27</span>
</span>
</td>
<td id="S4.T1.9.9.9.9.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.9.6.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.9.6.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.9.6.1.1.1.1" class="ltx_sup"><a href="#bib.bib7" title="" class="ltx_ref">7</a>, <a href="#bib.bib11" title="" class="ltx_ref">11</a>, <a href="#bib.bib15" title="" class="ltx_ref">15</a></sup></cite></span>
<span id="S4.T1.9.9.9.9.6.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.9.6.1.2.1.1" class="ltx_sup"><a href="#bib.bib5" title="" class="ltx_ref">5</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib9" title="" class="ltx_ref">9</a></sup></cite></span>
<span id="S4.T1.9.9.9.9.6.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.9.6.1.3.1.1" class="ltx_sup"><a href="#bib.bib61" title="" class="ltx_ref">61</a>, <a href="#bib.bib70" title="" class="ltx_ref">70</a>, <a href="#bib.bib71" title="" class="ltx_ref">71</a></sup></cite></span>
<span id="S4.T1.9.9.9.9.6.1.4" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.9.6.1.4.1.1" class="ltx_sup"><a href="#bib.bib72" title="" class="ltx_ref">72</a>, <a href="#bib.bib73" title="" class="ltx_ref">73</a>, <a href="#bib.bib46" title="" class="ltx_ref">46</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.12" class="ltx_tr">
<td id="S4.T1.9.9.9.12.1" class="ltx_td ltx_align_left ltx_border_t">Topology</td>
<td id="S4.T1.9.9.9.12.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.12.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.12.2.1.1" class="ltx_p">Centralized</span>
<span id="S4.T1.9.9.9.12.2.1.2" class="ltx_p">Decentralized</span>
</span>
</td>
<td id="S4.T1.9.9.9.12.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.12.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.12.3.1.1" class="ltx_p">95</span>
<span id="S4.T1.9.9.9.12.3.1.2" class="ltx_p">12</span>
</span>
</td>
<td id="S4.T1.9.9.9.12.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.12.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.12.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.12.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib51" title="" class="ltx_ref">51</a>, <a href="#bib.bib52" title="" class="ltx_ref">52</a>, <a href="#bib.bib36" title="" class="ltx_ref">36</a></sup></cite></span>
<span id="S4.T1.9.9.9.12.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.12.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib74" title="" class="ltx_ref">74</a>, <a href="#bib.bib75" title="" class="ltx_ref">75</a>, <a href="#bib.bib44" title="" class="ltx_ref">44</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.13" class="ltx_tr">
<td id="S4.T1.9.9.9.13.1" class="ltx_td ltx_align_left ltx_border_t">Type of Federation</td>
<td id="S4.T1.9.9.9.13.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.13.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.13.2.1.1" class="ltx_p">Cross-Silo</span>
<span id="S4.T1.9.9.9.13.2.1.2" class="ltx_p">Cross-Device</span>
<span id="S4.T1.9.9.9.13.2.1.3" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.13.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.13.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.13.3.1.1" class="ltx_p">98</span>
<span id="S4.T1.9.9.9.13.3.1.2" class="ltx_p">7</span>
<span id="S4.T1.9.9.9.13.3.1.3" class="ltx_p">2</span>
</span>
</td>
<td id="S4.T1.9.9.9.13.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.13.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.13.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.13.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib76" title="" class="ltx_ref">76</a>, <a href="#bib.bib37" title="" class="ltx_ref">37</a>, <a href="#bib.bib56" title="" class="ltx_ref">56</a></sup></cite></span>
<span id="S4.T1.9.9.9.13.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.13.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib75" title="" class="ltx_ref">75</a>, <a href="#bib.bib74" title="" class="ltx_ref">74</a>, <a href="#bib.bib77" title="" class="ltx_ref">77</a></sup></cite></span>
<span id="S4.T1.9.9.9.13.4.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.13.4.1.3.1.1" class="ltx_sup"><a href="#bib.bib46" title="" class="ltx_ref">46</a>, <a href="#bib.bib73" title="" class="ltx_ref">73</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.14" class="ltx_tr">
<td id="S4.T1.9.9.9.14.1" class="ltx_td ltx_align_left ltx_border_t">Scenario</td>
<td id="S4.T1.9.9.9.14.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.14.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.14.2.1.1" class="ltx_p">Deployment</span>
<span id="S4.T1.9.9.9.14.2.1.2" class="ltx_p">Simulation</span>
</span>
</td>
<td id="S4.T1.9.9.9.14.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.14.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.14.3.1.1" class="ltx_p">10</span>
<span id="S4.T1.9.9.9.14.3.1.2" class="ltx_p">97</span>
</span>
</td>
<td id="S4.T1.9.9.9.14.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.14.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.14.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.14.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib78" title="" class="ltx_ref">78</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib80" title="" class="ltx_ref">80</a></sup></cite></span>
<span id="S4.T1.9.9.9.14.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.14.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib64" title="" class="ltx_ref">64</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.15" class="ltx_tr">
<td id="S4.T1.9.9.9.15.1" class="ltx_td ltx_align_left ltx_border_t">Framework</td>
<td id="S4.T1.9.9.9.15.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.15.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.15.2.1.1" class="ltx_p">Custom-designed</span>
<span id="S4.T1.9.9.9.15.2.1.2" class="ltx_p">Open-source Options</span>
<span id="S4.T1.9.9.9.15.2.1.3" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.15.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.15.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.15.3.1.1" class="ltx_p">78</span>
<span id="S4.T1.9.9.9.15.3.1.2" class="ltx_p">13</span>
<span id="S4.T1.9.9.9.15.3.1.3" class="ltx_p">16</span>
</span>
</td>
<td id="S4.T1.9.9.9.15.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.15.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.15.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.15.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib82" title="" class="ltx_ref">82</a>, <a href="#bib.bib81" title="" class="ltx_ref">81</a>, <a href="#bib.bib83" title="" class="ltx_ref">83</a></sup></cite></span>
<span id="S4.T1.9.9.9.15.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.15.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib13" title="" class="ltx_ref">13</a>, <a href="#bib.bib8" title="" class="ltx_ref">8</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a></sup></cite></span>
<span id="S4.T1.9.9.9.15.4.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.15.4.1.3.1.1" class="ltx_sup"><a href="#bib.bib50" title="" class="ltx_ref">50</a>, <a href="#bib.bib64" title="" class="ltx_ref">64</a>, <a href="#bib.bib84" title="" class="ltx_ref">84</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.16" class="ltx_tr">
<td id="S4.T1.9.9.9.16.1" class="ltx_td ltx_align_left ltx_border_t">Data Curation</td>
<td id="S4.T1.9.9.9.16.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.16.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.16.2.1.1" class="ltx_p">Standardization &amp; Harmonization</span>
<span id="S4.T1.9.9.9.16.2.1.2" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.16.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.16.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.16.3.1.1" class="ltx_p">12</span>
<span id="S4.T1.9.9.9.16.3.1.2" class="ltx_p">95</span>
</span>
</td>
<td id="S4.T1.9.9.9.16.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.16.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.16.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.16.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib7" title="" class="ltx_ref">7</a>, <a href="#bib.bib66" title="" class="ltx_ref">66</a>, <a href="#bib.bib23" title="" class="ltx_ref">23</a></sup></cite></span>
<span id="S4.T1.9.9.9.16.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.16.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib49" title="" class="ltx_ref">49</a>, <a href="#bib.bib85" title="" class="ltx_ref">85</a>, <a href="#bib.bib86" title="" class="ltx_ref">86</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.17" class="ltx_tr">
<td id="S4.T1.9.9.9.17.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="4"><span id="S4.T1.9.9.9.17.1.1" class="ltx_text">Data Partition</span></td>
<td id="S4.T1.9.9.9.17.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.17.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.17.2.1.1" class="ltx_p">HFL</span>
<span id="S4.T1.9.9.9.17.2.1.2" class="ltx_p">VFL</span>
<span id="S4.T1.9.9.9.17.2.1.3" class="ltx_p">FTL</span>
<span id="S4.T1.9.9.9.17.2.1.4" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.17.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.17.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.17.3.1.1" class="ltx_p">94</span>
<span id="S4.T1.9.9.9.17.3.1.2" class="ltx_p">3</span>
<span id="S4.T1.9.9.9.17.3.1.3" class="ltx_p">1</span>
<span id="S4.T1.9.9.9.17.3.1.4" class="ltx_p">12</span>
</span>
</td>
<td id="S4.T1.9.9.9.17.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.17.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.17.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.17.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib87" title="" class="ltx_ref">87</a>, <a href="#bib.bib88" title="" class="ltx_ref">88</a>, <a href="#bib.bib89" title="" class="ltx_ref">89</a></sup></cite></span>
<span id="S4.T1.9.9.9.17.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.17.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib28" title="" class="ltx_ref">28</a>, <a href="#bib.bib48" title="" class="ltx_ref">48</a>, <a href="#bib.bib36" title="" class="ltx_ref">36</a></sup></cite></span>
<span id="S4.T1.9.9.9.17.4.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.17.4.1.3.1.1" class="ltx_sup"><a href="#bib.bib77" title="" class="ltx_ref">77</a></sup></cite></span>
<span id="S4.T1.9.9.9.17.4.1.4" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.17.4.1.4.1.1" class="ltx_sup"><a href="#bib.bib40" title="" class="ltx_ref">40</a>, <a href="#bib.bib90" title="" class="ltx_ref">90</a>, <a href="#bib.bib39" title="" class="ltx_ref">39</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.18" class="ltx_tr">
<td id="S4.T1.9.9.9.18.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.18.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.18.1.1.1" class="ltx_p">Natural Split</span>
<span id="S4.T1.9.9.9.18.1.1.2" class="ltx_p">Simulate</span>
<span id="S4.T1.9.9.9.18.1.1.3" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.18.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.18.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.18.2.1.1" class="ltx_p">37</span>
<span id="S4.T1.9.9.9.18.2.1.2" class="ltx_p">67</span>
<span id="S4.T1.9.9.9.18.2.1.3" class="ltx_p">3</span>
</span>
</td>
<td id="S4.T1.9.9.9.18.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.18.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.18.3.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.18.3.1.1.1.1" class="ltx_sup"><a href="#bib.bib75" title="" class="ltx_ref">75</a>, <a href="#bib.bib91" title="" class="ltx_ref">91</a>, <a href="#bib.bib37" title="" class="ltx_ref">37</a></sup></cite></span>
<span id="S4.T1.9.9.9.18.3.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.18.3.1.2.1.1" class="ltx_sup"><a href="#bib.bib92" title="" class="ltx_ref">92</a>, <a href="#bib.bib43" title="" class="ltx_ref">43</a>, <a href="#bib.bib86" title="" class="ltx_ref">86</a></sup></cite></span>
<span id="S4.T1.9.9.9.18.3.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.18.3.1.3.1.1" class="ltx_sup"><a href="#bib.bib93" title="" class="ltx_ref">93</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a>, <a href="#bib.bib9" title="" class="ltx_ref">9</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.19" class="ltx_tr">
<td id="S4.T1.9.9.9.19.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.19.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.19.1.1.1" class="ltx_p">Train/Test/Val Details per Client</span>
</span>
</td>
<td id="S4.T1.9.9.9.19.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.19.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.19.2.1.1" class="ltx_p">17</span>
</span>
</td>
<td id="S4.T1.9.9.9.19.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.19.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.19.3.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.19.3.1.1.1.1" class="ltx_sup"><a href="#bib.bib43" title="" class="ltx_ref">43</a>, <a href="#bib.bib89" title="" class="ltx_ref">89</a>, <a href="#bib.bib94" title="" class="ltx_ref">94</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.20" class="ltx_tr">
<td id="S4.T1.9.9.9.20.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.20.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.20.1.1.1" class="ltx_p">Holdout Cohort for Evaluation</span>
</span>
</td>
<td id="S4.T1.9.9.9.20.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.20.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.20.2.1.1" class="ltx_p">12</span>
</span>
</td>
<td id="S4.T1.9.9.9.20.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.20.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.20.3.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.20.3.1.1.1.1" class="ltx_sup"><a href="#bib.bib49" title="" class="ltx_ref">49</a>, <a href="#bib.bib37" title="" class="ltx_ref">37</a>, <a href="#bib.bib66" title="" class="ltx_ref">66</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.21" class="ltx_tr">
<td id="S4.T1.9.9.9.21.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="2"><span id="S4.T1.9.9.9.21.1.1" class="ltx_text">Model</span></td>
<td id="S4.T1.9.9.9.21.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.21.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.21.2.1.1" class="ltx_p">Deep Learning</span>
<span id="S4.T1.9.9.9.21.2.1.2" class="ltx_p">Traditional Machine Learning</span>
<span id="S4.T1.9.9.9.21.2.1.3" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.21.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.21.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.21.3.1.1" class="ltx_p">83</span>
<span id="S4.T1.9.9.9.21.3.1.2" class="ltx_p">7</span>
<span id="S4.T1.9.9.9.21.3.1.3" class="ltx_p">17</span>
</span>
</td>
<td id="S4.T1.9.9.9.21.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.21.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.21.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.21.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib78" title="" class="ltx_ref">78</a>, <a href="#bib.bib80" title="" class="ltx_ref">80</a>, <a href="#bib.bib57" title="" class="ltx_ref">57</a></sup></cite></span>
<span id="S4.T1.9.9.9.21.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.21.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib7" title="" class="ltx_ref">7</a>, <a href="#bib.bib35" title="" class="ltx_ref">35</a>, <a href="#bib.bib62" title="" class="ltx_ref">62</a></sup></cite></span>
<span id="S4.T1.9.9.9.21.4.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.21.4.1.3.1.1" class="ltx_sup"><a href="#bib.bib95" title="" class="ltx_ref">95</a>, <a href="#bib.bib13" title="" class="ltx_ref">13</a>, <a href="#bib.bib79" title="" class="ltx_ref">79</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.22" class="ltx_tr">
<td id="S4.T1.9.9.9.22.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.22.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.22.1.1.1" class="ltx_p">Initialization: Random</span>
<span id="S4.T1.9.9.9.22.1.1.2" class="ltx_p">Initialization: Pretrained/Foundation Models</span>
<span id="S4.T1.9.9.9.22.1.1.3" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.22.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.22.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.22.2.1.1" class="ltx_p">18</span>
<span id="S4.T1.9.9.9.22.2.1.2" class="ltx_p">5</span>
<span id="S4.T1.9.9.9.22.2.1.3" class="ltx_p">84</span>
</span>
</td>
<td id="S4.T1.9.9.9.22.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.22.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.22.3.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.22.3.1.1.1.1" class="ltx_sup"><a href="#bib.bib7" title="" class="ltx_ref">7</a>, <a href="#bib.bib28" title="" class="ltx_ref">28</a>, <a href="#bib.bib65" title="" class="ltx_ref">65</a></sup></cite></span>
<span id="S4.T1.9.9.9.22.3.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.22.3.1.2.1.1" class="ltx_sup"><a href="#bib.bib96" title="" class="ltx_ref">96</a>, <a href="#bib.bib33" title="" class="ltx_ref">33</a>, <a href="#bib.bib43" title="" class="ltx_ref">43</a></sup></cite></span>
<span id="S4.T1.9.9.9.22.3.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.22.3.1.3.1.1" class="ltx_sup"><a href="#bib.bib97" title="" class="ltx_ref">97</a>, <a href="#bib.bib83" title="" class="ltx_ref">83</a>, <a href="#bib.bib53" title="" class="ltx_ref">53</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.23" class="ltx_tr">
<td id="S4.T1.9.9.9.23.1" class="ltx_td ltx_align_left ltx_border_t" rowspan="6"><span id="S4.T1.9.9.9.23.1.1" class="ltx_text">Optimization</span></td>
<td id="S4.T1.9.9.9.23.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.23.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.23.2.1.1" class="ltx_p">System Heterogeneity</span>
</span>
</td>
<td id="S4.T1.9.9.9.23.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.23.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.23.3.1.1" class="ltx_p">0</span>
</span>
</td>
<td id="S4.T1.9.9.9.23.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.23.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.23.4.1.1" class="ltx_p">-</span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.24" class="ltx_tr">
<td id="S4.T1.9.9.9.24.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.24.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.24.1.1.1" class="ltx_p">Generalization in Open Domain</span>
</span>
</td>
<td id="S4.T1.9.9.9.24.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.24.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.24.2.1.1" class="ltx_p">15</span>
</span>
</td>
<td id="S4.T1.9.9.9.24.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.24.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.24.3.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.24.3.1.1.1.1" class="ltx_sup"><a href="#bib.bib7" title="" class="ltx_ref">7</a>, <a href="#bib.bib28" title="" class="ltx_ref">28</a>, <a href="#bib.bib33" title="" class="ltx_ref">33</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.25" class="ltx_tr">
<td id="S4.T1.9.9.9.25.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.25.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.25.1.1.1" class="ltx_p">Communication Efficiency</span>
</span>
</td>
<td id="S4.T1.9.9.9.25.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.25.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.25.2.1.1" class="ltx_p">19</span>
</span>
</td>
<td id="S4.T1.9.9.9.25.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.25.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.25.3.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.25.3.1.1.1.1" class="ltx_sup"><a href="#bib.bib61" title="" class="ltx_ref">61</a>, <a href="#bib.bib27" title="" class="ltx_ref">27</a>, <a href="#bib.bib98" title="" class="ltx_ref">98</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.26" class="ltx_tr">
<td id="S4.T1.9.9.9.26.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.26.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.26.1.1.1" class="ltx_p">Theoretical Convergence Analysis</span>
</span>
</td>
<td id="S4.T1.9.9.9.26.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.26.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.26.2.1.1" class="ltx_p">0</span>
</span>
</td>
<td id="S4.T1.9.9.9.26.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.26.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.26.3.1.1" class="ltx_p">-</span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.27" class="ltx_tr">
<td id="S4.T1.9.9.9.27.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.27.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.27.1.1.1" class="ltx_p">Temporal Data Dynamics</span>
</span>
</td>
<td id="S4.T1.9.9.9.27.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.27.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.27.2.1.1" class="ltx_p">2</span>
</span>
</td>
<td id="S4.T1.9.9.9.27.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.27.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.27.3.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.27.3.1.1.1.1" class="ltx_sup"><a href="#bib.bib57" title="" class="ltx_ref">57</a>, <a href="#bib.bib28" title="" class="ltx_ref">28</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.28" class="ltx_tr">
<td id="S4.T1.9.9.9.28.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.28.1.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.28.1.1.1" class="ltx_p">Synchronous Aggregation</span>
<span id="S4.T1.9.9.9.28.1.1.2" class="ltx_p">Asynchronous Aggregation</span>
</span>
</td>
<td id="S4.T1.9.9.9.28.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.28.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.28.2.1.1" class="ltx_p">92</span>
<span id="S4.T1.9.9.9.28.2.1.2" class="ltx_p">15</span>
</span>
</td>
<td id="S4.T1.9.9.9.28.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.28.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.28.3.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.28.3.1.1.1.1" class="ltx_sup"><a href="#bib.bib19" title="" class="ltx_ref">19</a>, <a href="#bib.bib23" title="" class="ltx_ref">23</a>, <a href="#bib.bib24" title="" class="ltx_ref">24</a></sup></cite></span>
<span id="S4.T1.9.9.9.28.3.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.28.3.1.2.1.1" class="ltx_sup"><a href="#bib.bib32" title="" class="ltx_ref">32</a>, <a href="#bib.bib33" title="" class="ltx_ref">33</a>, <a href="#bib.bib34" title="" class="ltx_ref">34</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.29" class="ltx_tr">
<td id="S4.T1.9.9.9.29.1" class="ltx_td ltx_align_left ltx_border_t">Privacy and Security</td>
<td id="S4.T1.9.9.9.29.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.29.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.29.2.1.1" class="ltx_p">Model Updates Protection</span>
</span>
</td>
<td id="S4.T1.9.9.9.29.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.29.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.29.3.1.1" class="ltx_p">41</span>
</span>
</td>
<td id="S4.T1.9.9.9.29.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.29.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.29.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.29.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib19" title="" class="ltx_ref">19</a>, <a href="#bib.bib99" title="" class="ltx_ref">99</a>, <a href="#bib.bib64" title="" class="ltx_ref">64</a></sup></cite></span>
</span>
</td>
</tr>
<tr id="S4.T1.9.9.9.30" class="ltx_tr">
<td id="S4.T1.9.9.9.30.1" class="ltx_td ltx_align_left ltx_border_bb ltx_border_t">Open Source</td>
<td id="S4.T1.9.9.9.30.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_t" style="width:256.1pt;">
<span id="S4.T1.9.9.9.30.2.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.30.2.1.1" class="ltx_p">Code Available</span>
<span id="S4.T1.9.9.9.30.2.1.2" class="ltx_p">Trained Model Available</span>
<span id="S4.T1.9.9.9.30.2.1.3" class="ltx_p">unavailable</span>
</span>
</td>
<td id="S4.T1.9.9.9.30.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_t" style="width:99.6pt;">
<span id="S4.T1.9.9.9.30.3.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.30.3.1.1" class="ltx_p">29</span>
<span id="S4.T1.9.9.9.30.3.1.2" class="ltx_p">1</span>
<span id="S4.T1.9.9.9.30.3.1.3" class="ltx_p">78</span>
</span>
</td>
<td id="S4.T1.9.9.9.30.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_t" style="width:56.9pt;">
<span id="S4.T1.9.9.9.30.4.1" class="ltx_inline-block ltx_align_top">
<span id="S4.T1.9.9.9.30.4.1.1" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.30.4.1.1.1.1" class="ltx_sup"><a href="#bib.bib51" title="" class="ltx_ref">51</a>, <a href="#bib.bib52" title="" class="ltx_ref">52</a>, <a href="#bib.bib65" title="" class="ltx_ref">65</a></sup></cite></span>
<span id="S4.T1.9.9.9.30.4.1.2" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.30.4.1.2.1.1" class="ltx_sup"><a href="#bib.bib8" title="" class="ltx_ref">8</a></sup></cite></span>
<span id="S4.T1.9.9.9.30.4.1.3" class="ltx_p"><cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.T1.9.9.9.30.4.1.3.1.1" class="ltx_sup"><a href="#bib.bib100" title="" class="ltx_ref">100</a>, <a href="#bib.bib47" title="" class="ltx_ref">47</a>, <a href="#bib.bib98" title="" class="ltx_ref">98</a></sup></cite></span>
</span>
</td>
</tr>
</table>
</span></div>
</figure>
</section>
<section id="S4.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>Topology, Scenario and Framework</h3>

<div id="S4.SS2.p1" class="ltx_para">
<p id="S4.SS2.p1.1" class="ltx_p">The centralized FL paradigm dominates current implementations, with 95 out of 107 studies following this topology.
Only 10 studies reported real-world deployments in distributed clinical settings, while the rest remained in the realm of prototypes or simulations.
In terms of frameworks, the majority (78/107) utilized custom-designed FL frameworks, while a smaller number (13/107) employed open-source options such as FlowerÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib13" title="" class="ltx_ref">13</a></sup></cite>, FlareÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib8" title="" class="ltx_ref">8</a></sup>, <sup class="ltx_sup"><a href="#bib.bib79" title="" class="ltx_ref">79</a></sup>, <sup class="ltx_sup"><a href="#bib.bib11" title="" class="ltx_ref">11</a></sup></cite>, SubstraFLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib7" title="" class="ltx_ref">7</a></sup>, <sup class="ltx_sup"><a href="#bib.bib15" title="" class="ltx_ref">15</a></sup></cite>, TFFÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib35" title="" class="ltx_ref">35</a></sup></cite>, OpenFLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib14" title="" class="ltx_ref">14</a></sup></cite>, PySyftÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib34" title="" class="ltx_ref">34</a></sup>, <sup class="ltx_sup"><a href="#bib.bib47" title="" class="ltx_ref">47</a></sup>, <sup class="ltx_sup"><a href="#bib.bib101" title="" class="ltx_ref">101</a></sup>, <sup class="ltx_sup"><a href="#bib.bib66" title="" class="ltx_ref">66</a></sup></cite>, and FedBioMedÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib80" title="" class="ltx_ref">80</a></sup></cite>.
16 studies did not specify the framework used.
Further details about open-source frameworks can be found in TableÂ <a href="#S5.T2" title="Table 2 â€£ Recommendations &amp; Opportunities â€£ 5.1.1 Domination of Simulation Studies â€£ 5.1 Scenario and Framework â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.</p>
</div>
</section>
<section id="S4.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.3 </span>Data Curation and Partition</h3>

<div id="S4.SS3.p1" class="ltx_para">
<p id="S4.SS3.p1.1" class="ltx_p">Among the reviewed studies, only 12 provided details on the processes of data standardization and harmonization.
Regarding data partition, HFL was the predominant approach, with 94 out of 107 studies focusing solely on it.
In contrast, VFL was explored in only 3 studiesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib36" title="" class="ltx_ref">36</a></sup>, <sup class="ltx_sup"><a href="#bib.bib28" title="" class="ltx_ref">28</a></sup>, <sup class="ltx_sup"><a href="#bib.bib48" title="" class="ltx_ref">48</a></sup></cite>, while 2 studies considered both HFL and VFL in combinationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib28" title="" class="ltx_ref">28</a></sup>, <sup class="ltx_sup"><a href="#bib.bib48" title="" class="ltx_ref">48</a></sup></cite>.
Only 1 study discussed FTLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib77" title="" class="ltx_ref">77</a></sup></cite>.
Notably, 12 studies did not mention this aspect at all.
The majority of studies addressed only one type of data heterogeneity, such as quantity skew or label skew, without considering multiple factors simultaneously. Moreover, 37 studies employed natural data splits for training and/or evaluation, while the rest relied on artificial splits.
Only 17 studies detailed their training, testing or validation sets and 12 studies split a holdout cohort for evaluation.</p>
</div>
</section>
<section id="S4.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.4 </span>Model</h3>

<div id="S4.SS4.p1" class="ltx_para">
<p id="S4.SS4.p1.1" class="ltx_p">Among reviewed studies, Convolutional Neural Networks (CNNs) were the most commonly utilized (80/107), including both custom models specifically designed for healthcare tasks and well-established architectures like ResNet, DenseNet, MobileNet, and U-NetÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib100" title="" class="ltx_ref">100</a></sup>, <sup class="ltx_sup"><a href="#bib.bib47" title="" class="ltx_ref">47</a></sup>, <sup class="ltx_sup"><a href="#bib.bib98" title="" class="ltx_ref">98</a></sup></cite>.
Additionally, Recurrent Neural Networks (RNNs) have been incorporated to leverage their strengths in handling complex healthcare dataÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib48" title="" class="ltx_ref">48</a></sup>, <sup class="ltx_sup"><a href="#bib.bib57" title="" class="ltx_ref">57</a></sup>, <sup class="ltx_sup"><a href="#bib.bib102" title="" class="ltx_ref">102</a></sup></cite>.
Some studies also employed custom Multi-Layer Perceptrons (MLPs) and attention mechanisms to further boost model performanceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib70" title="" class="ltx_ref">70</a></sup>, <sup class="ltx_sup"><a href="#bib.bib103" title="" class="ltx_ref">103</a></sup>, <sup class="ltx_sup"><a href="#bib.bib61" title="" class="ltx_ref">61</a></sup></cite>.
<cite class="ltx_cite ltx_citemacro_cite"><sup id="S4.SS4.p1.1.1.1" class="ltx_sup"><a href="#bib.bib43" title="" class="ltx_ref">43</a></sup></cite> utilized large language models for distributed biomedical natural language processing.
Beyond deep learning approaches, several studies explored traditional machine learning (ML) algorithms, including linear models and ensemble methods. Notable examples include logistic regressionÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib70" title="" class="ltx_ref">70</a></sup></cite>, support vector machinesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib84" title="" class="ltx_ref">84</a></sup></cite>, fuzzy clusteringÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib104" title="" class="ltx_ref">104</a></sup></cite>, and decision treesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib71" title="" class="ltx_ref">71</a></sup>, <sup class="ltx_sup"><a href="#bib.bib105" title="" class="ltx_ref">105</a></sup></cite>.
Interestingly, only 23 studies explicitly discussed their initialization strategies for model training.
Among these, the majority opted for random initialization, while a mere five clearly stated that they utilized pretrained or foundation models as their starting pointÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib96" title="" class="ltx_ref">96</a></sup>, <sup class="ltx_sup"><a href="#bib.bib106" title="" class="ltx_ref">106</a></sup>, <sup class="ltx_sup"><a href="#bib.bib33" title="" class="ltx_ref">33</a></sup>, <sup class="ltx_sup"><a href="#bib.bib29" title="" class="ltx_ref">29</a></sup>, <sup class="ltx_sup"><a href="#bib.bib43" title="" class="ltx_ref">43</a></sup></cite>.</p>
</div>
</section>
<section id="S4.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.5 </span>Optimization</h3>

<div id="S4.SS5.p1" class="ltx_para">
<p id="S4.SS5.p1.1" class="ltx_p">Most studies addressed either data or model heterogeneity, and none of them considered system heterogeneity. Only 15 studies evaluated model generalization ability in unseen open domains.
A total of 19 studies focused on improving communication efficiency, employing techniques such as knowledge distillationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib51" title="" class="ltx_ref">51</a></sup>, <sup class="ltx_sup"><a href="#bib.bib99" title="" class="ltx_ref">99</a></sup>, <sup class="ltx_sup"><a href="#bib.bib96" title="" class="ltx_ref">96</a></sup></cite>, gradient quantizationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib61" title="" class="ltx_ref">61</a></sup></cite>, one-shot FLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib98" title="" class="ltx_ref">98</a></sup></cite>, split learningÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib101" title="" class="ltx_ref">101</a></sup></cite>, and tensor factorizationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib95" title="" class="ltx_ref">95</a></sup>, <sup class="ltx_sup"><a href="#bib.bib107" title="" class="ltx_ref">107</a></sup></cite>.
In terms of convergence analysis, a few studies (21/107) reported metrics such as communication rounds and costs, as well as overall convergence time, but none provided a theoretical understanding of convergence dynamics.
Only two studies considered temporal data dynamics in model learningÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib28" title="" class="ltx_ref">28</a></sup>, <sup class="ltx_sup"><a href="#bib.bib57" title="" class="ltx_ref">57</a></sup></cite>.
Regarding synchronization, 15 studies employed asynchronous aggregation instead of synchronous aggregation, particularly in applications involving wearablesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib75" title="" class="ltx_ref">75</a></sup></cite> and IoMT devicesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib57" title="" class="ltx_ref">57</a></sup></cite>.</p>
</div>
</section>
<section id="S4.SS6" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.6 </span>Privacy and Security</h3>

<div id="S4.SS6.p1" class="ltx_para">
<p id="S4.SS6.p1.1" class="ltx_p">Only 41 studies addressed the exchange of model updates with privacy guarantees. The most commonly used techniques for safeguarding model updates included Differential Privacy (DP), Homomorphic Encryption (HE), Secure Multi-Party Computation (SMPC), knowledge distillation, and partial model exchange. However, metadata such as sample sizes and distributions were frequently shared without protection, particularly in methods based on FedAvgÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib106" title="" class="ltx_ref">106</a></sup>, <sup class="ltx_sup"><a href="#bib.bib108" title="" class="ltx_ref">108</a></sup></cite>.
To mitigate the risk of adversaries inferring raw data, synthetic data was employed in some casesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib19" title="" class="ltx_ref">19</a></sup>, <sup class="ltx_sup"><a href="#bib.bib98" title="" class="ltx_ref">98</a></sup>, <sup class="ltx_sup"><a href="#bib.bib42" title="" class="ltx_ref">42</a></sup>, <sup class="ltx_sup"><a href="#bib.bib85" title="" class="ltx_ref">85</a></sup></cite>. Additionally, swarm learning and blockchain were utilized to secure the communication processÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib24" title="" class="ltx_ref">24</a></sup>, <sup class="ltx_sup"><a href="#bib.bib19" title="" class="ltx_ref">19</a></sup></cite>.</p>
</div>
</section>
<section id="S4.SS7" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.7 </span>Fairness and Incentive</h3>

<div id="S4.SS7.p1" class="ltx_para">
<p id="S4.SS7.p1.1" class="ltx_p">Only three studies have discussed issues related to fairness and/or incentives in healthcare FLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib109" title="" class="ltx_ref">109</a></sup>, <sup class="ltx_sup"><a href="#bib.bib45" title="" class="ltx_ref">45</a></sup>, <sup class="ltx_sup"><a href="#bib.bib53" title="" class="ltx_ref">53</a></sup></cite>, with just one of these studies specifically exploring the complexities of both fairness and incentives in detailÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib53" title="" class="ltx_ref">53</a></sup></cite>.</p>
</div>
</section>
<section id="S4.SS8" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.8 </span>Evaluation</h3>

<div id="S4.SS8.p1" class="ltx_para">
<p id="S4.SS8.p1.1" class="ltx_p">Most studies used conventional ML metrics for evaluation, such as accuracy, precision, area under the receiver operating characteristic curve (AUC), sensitivity/recall, specificity, F1-score, Dice score, Intersection over Union (IoU), Hausdorff Distance (HD), and loss value. Additionally, many studies performed comparisons against classical centralized models or localized models, and conducted ablation studies.
However, only a few studies (26/107) addressed critical aspects unique to FL, such as communication overhead, resource consumption, scalability, generalization, privacy, fairness, and security concerns.
As for benchmarking, just one study provided relatively comprehensive benchmarks across multiple healthcare datasetsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib52" title="" class="ltx_ref">52</a></sup></cite>.
Interpretability was explored in seven studies, either through feature selectionÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib62" title="" class="ltx_ref">62</a></sup>, <sup class="ltx_sup"><a href="#bib.bib38" title="" class="ltx_ref">38</a></sup></cite>, attention mapsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib39" title="" class="ltx_ref">39</a></sup>, <sup class="ltx_sup"><a href="#bib.bib51" title="" class="ltx_ref">51</a></sup>, <sup class="ltx_sup"><a href="#bib.bib66" title="" class="ltx_ref">66</a></sup></cite>, or tree-based modelsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib7" title="" class="ltx_ref">7</a></sup>, <sup class="ltx_sup"><a href="#bib.bib35" title="" class="ltx_ref">35</a></sup></cite>. While 29 studies released their source code, only one also made the trained model publicly availableÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib8" title="" class="ltx_ref">8</a></sup></cite>.</p>
</div>
<div id="S4.SS8.p2" class="ltx_para">
<p id="S4.SS8.p2.1" class="ltx_p">We provide a more detailed summary of the key results in TableÂ <a href="#S4.T1" title="Table 1 â€£ 4.1 Application and Data â€£ 4 Results â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.</p>
</div>
</section>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>From Challenges and Pitfalls to Recommended Solutions and Future Opportunities</h2>

<div id="S5.p1" class="ltx_para">
<p id="S5.p1.1" class="ltx_p">After a thorough review of the most recent and advanced FL studies, we find various challenges and pitfalls that still limit the implementation of FL in healthcare.
In this section, we introduce a clear taxonomy, as depicted in FigureÂ <a href="#S5.F5" title="Figure 5 â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>, focusing on the challenges and pitfalls, and further providing recommended solutions and opportunities.
We adhere to the best practice workflow in FL for discussion in the following subsections.</p>
</div>
<figure id="S5.F4" class="ltx_figure"><img src="/html/2409.09727/assets/x4.png" id="S5.F4.g1" class="ltx_graphics ltx_centering ltx_img_square" width="277" height="238" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 4: </span>Simulation scenario VS. real-world distributed scenario.</figcaption>
</figure>
<figure id="S5.F5" class="ltx_figure"><img src="/html/2409.09727/assets/x5.png" id="S5.F5.g1" class="ltx_graphics ltx_centering ltx_img_square" width="461" height="567" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 5: </span>Taxonomy of challenges and pitfalls (red blocks) as well as recommended solutions and opportunities (green blocks).</figcaption>
</figure>
<section id="S5.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.1 </span>Scenario and Framework</h3>

<section id="S5.SS1.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.1.1 </span>Domination of Simulation Studies</h4>

<div id="S5.SS1.SSS1.p1" class="ltx_para">
<p id="S5.SS1.SSS1.p1.1" class="ltx_p">The majority of existing studies have been confined to simulation environments, with only 10 studies incorporating real-world distributed clinical scenarios. This indicates that the application of FL in healthcare is still in its nascent stage.
The complexity of deploying FL across a real-world network of hospitals and institutions has significantly hindered its progress.
Most studies have operated within controlled, simulated settings where data is pooled and then artificially partitioned to represent distributed environments. The simulated clients interact with a simulated server, coordinating model updates in a manner that is highly controlled and predictable.
In contrast, real-world scenarios involve each client working with inherently distributed, heterogeneous, and locale-specific data. The interaction between real server and clients is far more complex, requiring secure protocols, real-time communication, and the ability to handle diverse datasets across various institutions. This disparity between simulation and real-world environments is illustrated in FigureÂ <a href="#S5.F4" title="Figure 4 â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.</p>
</div>
<div id="S5.SS1.SSS1.p2" class="ltx_para">
<p id="S5.SS1.SSS1.p2.1" class="ltx_p">Moreover, very few studies have explored FL practices at a national or international scaleÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib8" title="" class="ltx_ref">8</a></sup>, <sup class="ltx_sup"><a href="#bib.bib62" title="" class="ltx_ref">62</a></sup></cite>.
Notable examples include the Collaborative Data Analysis (CODA)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib78" title="" class="ltx_ref">78</a></sup></cite>, <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS1.SSS1.p2.1.1.1" class="ltx_sup"><a href="#bib.bib15" title="" class="ltx_ref">15</a></sup></cite>, HealthChainÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib12" title="" class="ltx_ref">12</a></sup></cite> and DRAGONÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib13" title="" class="ltx_ref">13</a></sup></cite>.
CODA tested FLâ€™s feasibility across eight hospitals in Canada by enrolling patients with suspected or confirmed COVID-19 over three years.
Melody deployed multi-task FL among 10 pharmaceutical companies to optimize the drug discovery process.
HealthChain and DRAGON implemented FL across multiple hospitals in Europe, facilitating the prediction of treatment responses for cancer and COVID-19 patients.</p>
</div>
<div id="S5.SS1.SSS1.p3" class="ltx_para">
<p id="S5.SS1.SSS1.p3.1" class="ltx_p">Despite these rare promising examples, the majority of FL studies remain proof-of-concept and the broader deployment of FL in healthcare remains largely undocumented. There is still a lack of clarity on how FL nodes are set up within individual hospitals, the methods for delivering local models to these nodes, the protocols enabling interaction between nodes and aggregators, and the mechanisms triggering new training rounds, etc.</p>
</div>
<section id="S5.SS1.SSS1.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS1.SSS1.Px1.p1" class="ltx_para">
<ul id="S5.I1" class="ltx_itemize">
<li id="S5.I1.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I1.i1.p1" class="ltx_para">
<p id="S5.I1.i1.p1.1" class="ltx_p"><span id="S5.I1.i1.p1.1.1" class="ltx_text ltx_font_italic">For researchers</span>, it is recommended to continue leveraging simulation environments to rapidly prototype and evaluate FL algorithms. Simulations offer precise control over experimental conditions, which is essential for understanding the underlying mechanics of FL and its behaviour under various scenarios. However, researchers should acknowledge the limitations of simulations and anticipate the challenges that real-world deployments may introduce. Beyond simulations, to enhance data diversity, collaboration and generalization, efforts should be made to implement FL at a national and international scale, with cloud computing offering scalable resources and seamless implementation across institutions.</p>
</div>
</li>
<li id="S5.I1.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I1.i2.p1" class="ltx_para">
<p id="S5.I1.i2.p1.1" class="ltx_p"><span id="S5.I1.i2.p1.1.1" class="ltx_text ltx_font_italic">For clinicians</span> interested in applying FL to enhance clinical diagnostics and prognostics, it is crucial to comprehend both the potential benefits and limitations of FL. Clinicians should collaborate closely with researchers and engineers to identify promising use cases for FL in clinical practice. This may involve conducting pilot studies to assess the feasibility and effectiveness of FL in specific clinical scenarios. Clinicians should advocate for the integration of FL into existing healthcare workflows to ensure a seamless transition from research to practice. Additionally, their feedback on the usability and impact of FL systems is vital for guiding further refinements.</p>
</div>
</li>
<li id="S5.I1.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I1.i3.p1" class="ltx_para">
<p id="S5.I1.i3.p1.1" class="ltx_p"><span id="S5.I1.i3.p1.1.1" class="ltx_text ltx_font_italic">For engineers</span>, the focus should be on addressing the practical challenges of FL deployment in real-world settings. This includes ensuring the interoperability of different hospital systems, safeguarding data privacy and security, and managing the communication overhead of networks. Engineers should aim to develop robust and scalable solutions adaptable to the heterogeneous IT infrastructures across healthcare institutions. Close collaboration with clinicians and researchers is essential to ensure that FL systems meet healthcare-specific needs and comply with regulatory standards.</p>
</div>
</li>
</ul>
</div>
<figure id="S5.T2" class="ltx_table">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 2: </span>Capabilities and features of current popular FL frameworks.</figcaption>
<div id="S5.T2.1" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:224.5pt;height:71.1pt;vertical-align:-0.0pt;"><span class="ltx_transformed_inner" style="transform:translate(-322.3pt,102.1pt) scale(0.258285705484089,0.258285705484089) ;">
<table id="S5.T2.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S5.T2.1.1.1" class="ltx_tr">
<td id="S5.T2.1.1.1.1" class="ltx_td ltx_align_left ltx_border_tt">Framework</td>
<td id="S5.T2.1.1.1.2" class="ltx_td ltx_align_center ltx_border_tt">Developer</td>
<td id="S5.T2.1.1.1.3" class="ltx_td ltx_align_center ltx_border_tt">
<span id="S5.T2.1.1.1.3.1" class="ltx_text"></span> <span id="S5.T2.1.1.1.3.2" class="ltx_text">
<span id="S5.T2.1.1.1.3.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T2.1.1.1.3.2.1.1" class="ltx_tr">
<span id="S5.T2.1.1.1.3.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Secure</span></span>
<span id="S5.T2.1.1.1.3.2.1.2" class="ltx_tr">
<span id="S5.T2.1.1.1.3.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Aggregation</span></span>
</span></span><span id="S5.T2.1.1.1.3.3" class="ltx_text"></span></td>
<td id="S5.T2.1.1.1.4" class="ltx_td ltx_align_center ltx_border_tt">
<span id="S5.T2.1.1.1.4.1" class="ltx_text"></span> <span id="S5.T2.1.1.1.4.2" class="ltx_text">
<span id="S5.T2.1.1.1.4.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T2.1.1.1.4.2.1.1" class="ltx_tr">
<span id="S5.T2.1.1.1.4.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Communication</span></span>
<span id="S5.T2.1.1.1.4.2.1.2" class="ltx_tr">
<span id="S5.T2.1.1.1.4.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Efficiency</span></span>
</span></span><span id="S5.T2.1.1.1.4.3" class="ltx_text"></span></td>
<td id="S5.T2.1.1.1.5" class="ltx_td ltx_align_center ltx_border_tt">
<span id="S5.T2.1.1.1.5.1" class="ltx_text"></span> <span id="S5.T2.1.1.1.5.2" class="ltx_text">
<span id="S5.T2.1.1.1.5.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T2.1.1.1.5.2.1.1" class="ltx_tr">
<span id="S5.T2.1.1.1.5.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Healthcare</span></span>
<span id="S5.T2.1.1.1.5.2.1.2" class="ltx_tr">
<span id="S5.T2.1.1.1.5.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Adaptation</span></span>
</span></span><span id="S5.T2.1.1.1.5.3" class="ltx_text"></span></td>
<td id="S5.T2.1.1.1.6" class="ltx_td ltx_align_center ltx_border_tt">Traceability</td>
<td id="S5.T2.1.1.1.7" class="ltx_td ltx_align_center ltx_border_tt">Deployment</td>
<td id="S5.T2.1.1.1.8" class="ltx_td ltx_align_center ltx_border_tt">
<span id="S5.T2.1.1.1.8.1" class="ltx_text"></span> <span id="S5.T2.1.1.1.8.2" class="ltx_text">
<span id="S5.T2.1.1.1.8.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T2.1.1.1.8.2.1.1" class="ltx_tr">
<span id="S5.T2.1.1.1.8.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Foundation</span></span>
<span id="S5.T2.1.1.1.8.2.1.2" class="ltx_tr">
<span id="S5.T2.1.1.1.8.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Model</span></span>
</span></span><span id="S5.T2.1.1.1.8.3" class="ltx_text"></span></td>
<td id="S5.T2.1.1.1.9" class="ltx_td ltx_align_center ltx_border_tt">Scalability</td>
<td id="S5.T2.1.1.1.10" class="ltx_td ltx_align_center ltx_border_tt">
<span id="S5.T2.1.1.1.10.1" class="ltx_text"></span> <span id="S5.T2.1.1.1.10.2" class="ltx_text">
<span id="S5.T2.1.1.1.10.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T2.1.1.1.10.2.1.1" class="ltx_tr">
<span id="S5.T2.1.1.1.10.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Clound</span></span>
<span id="S5.T2.1.1.1.10.2.1.2" class="ltx_tr">
<span id="S5.T2.1.1.1.10.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Friendly</span></span>
</span></span><span id="S5.T2.1.1.1.10.3" class="ltx_text"></span></td>
</tr>
<tr id="S5.T2.1.1.2" class="ltx_tr">
<td id="S5.T2.1.1.2.1" class="ltx_td ltx_align_left ltx_border_t">FlareÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib79" title="" class="ltx_ref">79</a></sup></cite>
</td>
<td id="S5.T2.1.1.2.2" class="ltx_td ltx_align_center ltx_border_t">Nvidia</td>
<td id="S5.T2.1.1.2.3" class="ltx_td ltx_align_center ltx_border_t">DP, HE</td>
<td id="S5.T2.1.1.2.4" class="ltx_td ltx_align_center ltx_border_t">-</td>
<td id="S5.T2.1.1.2.5" class="ltx_td ltx_align_center ltx_border_t">-</td>
<td id="S5.T2.1.1.2.6" class="ltx_td ltx_align_center ltx_border_t">âœ“</td>
<td id="S5.T2.1.1.2.7" class="ltx_td ltx_align_center ltx_border_t">âœ“</td>
<td id="S5.T2.1.1.2.8" class="ltx_td ltx_align_center ltx_border_t">âœ“</td>
<td id="S5.T2.1.1.2.9" class="ltx_td ltx_align_center ltx_border_t">âœ“</td>
<td id="S5.T2.1.1.2.10" class="ltx_td ltx_align_center ltx_border_t">âœ“</td>
</tr>
<tr id="S5.T2.1.1.3" class="ltx_tr">
<td id="S5.T2.1.1.3.1" class="ltx_td ltx_align_left">FedMLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib110" title="" class="ltx_ref">110</a></sup></cite>
</td>
<td id="S5.T2.1.1.3.2" class="ltx_td ltx_align_center">TensorOpera</td>
<td id="S5.T2.1.1.3.3" class="ltx_td ltx_align_center">DP, HE</td>
<td id="S5.T2.1.1.3.4" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.3.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.3.6" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.3.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.3.8" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.3.9" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.3.10" class="ltx_td ltx_align_center">âœ“</td>
</tr>
<tr id="S5.T2.1.1.4" class="ltx_tr">
<td id="S5.T2.1.1.4.1" class="ltx_td ltx_align_left">FederatedScopeÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib111" title="" class="ltx_ref">111</a></sup></cite>
</td>
<td id="S5.T2.1.1.4.2" class="ltx_td ltx_align_center">Alibaba</td>
<td id="S5.T2.1.1.4.3" class="ltx_td ltx_align_center">DP, SMPC</td>
<td id="S5.T2.1.1.4.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.4.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.4.6" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.4.7" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.4.8" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.4.9" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.4.10" class="ltx_td ltx_align_center">-</td>
</tr>
<tr id="S5.T2.1.1.5" class="ltx_tr">
<td id="S5.T2.1.1.5.1" class="ltx_td ltx_align_left">FlowerÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib112" title="" class="ltx_ref">112</a></sup></cite>
</td>
<td id="S5.T2.1.1.5.2" class="ltx_td ltx_align_center">FlowerLab</td>
<td id="S5.T2.1.1.5.3" class="ltx_td ltx_align_center">DP</td>
<td id="S5.T2.1.1.5.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.5.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.5.6" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.5.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.5.8" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.5.9" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.5.10" class="ltx_td ltx_align_center">âœ“</td>
</tr>
<tr id="S5.T2.1.1.6" class="ltx_tr">
<td id="S5.T2.1.1.6.1" class="ltx_td ltx_align_left">FATEÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib113" title="" class="ltx_ref">113</a></sup></cite>
</td>
<td id="S5.T2.1.1.6.2" class="ltx_td ltx_align_center">WeBank</td>
<td id="S5.T2.1.1.6.3" class="ltx_td ltx_align_center">DP, HE, SMPC</td>
<td id="S5.T2.1.1.6.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.6.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.6.6" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.6.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.6.8" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.6.9" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.6.10" class="ltx_td ltx_align_center">âœ“</td>
</tr>
<tr id="S5.T2.1.1.7" class="ltx_tr">
<td id="S5.T2.1.1.7.1" class="ltx_td ltx_align_left">SubstraFLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib114" title="" class="ltx_ref">114</a></sup></cite>
</td>
<td id="S5.T2.1.1.7.2" class="ltx_td ltx_align_center">Owkin</td>
<td id="S5.T2.1.1.7.3" class="ltx_td ltx_align_center">DP, SMPC</td>
<td id="S5.T2.1.1.7.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.7.5" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.7.6" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.7.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.7.8" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.7.9" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.7.10" class="ltx_td ltx_align_center">âœ“</td>
</tr>
<tr id="S5.T2.1.1.8" class="ltx_tr">
<td id="S5.T2.1.1.8.1" class="ltx_td ltx_align_left">PySyftÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib115" title="" class="ltx_ref">115</a></sup></cite>
</td>
<td id="S5.T2.1.1.8.2" class="ltx_td ltx_align_center">OpenMined</td>
<td id="S5.T2.1.1.8.3" class="ltx_td ltx_align_center">DP</td>
<td id="S5.T2.1.1.8.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.8.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.8.6" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.8.7" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.8.8" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.8.9" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.8.10" class="ltx_td ltx_align_center">-</td>
</tr>
<tr id="S5.T2.1.1.9" class="ltx_tr">
<td id="S5.T2.1.1.9.1" class="ltx_td ltx_align_left">OpenFLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib116" title="" class="ltx_ref">116</a></sup></cite>
</td>
<td id="S5.T2.1.1.9.2" class="ltx_td ltx_align_center">Intel</td>
<td id="S5.T2.1.1.9.3" class="ltx_td ltx_align_center">DP</td>
<td id="S5.T2.1.1.9.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.9.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.9.6" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.9.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.9.8" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.9.9" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.9.10" class="ltx_td ltx_align_center">-</td>
</tr>
<tr id="S5.T2.1.1.10" class="ltx_tr">
<td id="S5.T2.1.1.10.1" class="ltx_td ltx_align_left">TFFÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib117" title="" class="ltx_ref">117</a></sup></cite>
</td>
<td id="S5.T2.1.1.10.2" class="ltx_td ltx_align_center">Google</td>
<td id="S5.T2.1.1.10.3" class="ltx_td ltx_align_center">DP</td>
<td id="S5.T2.1.1.10.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.10.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.10.6" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.10.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.10.8" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.10.9" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.10.10" class="ltx_td ltx_align_center">-</td>
</tr>
<tr id="S5.T2.1.1.11" class="ltx_tr">
<td id="S5.T2.1.1.11.1" class="ltx_td ltx_align_left">Fed-BioMedÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib80" title="" class="ltx_ref">80</a></sup></cite>
</td>
<td id="S5.T2.1.1.11.2" class="ltx_td ltx_align_center">Inria</td>
<td id="S5.T2.1.1.11.3" class="ltx_td ltx_align_center">DP</td>
<td id="S5.T2.1.1.11.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.11.5" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.11.6" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.11.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.11.8" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.11.9" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.11.10" class="ltx_td ltx_align_center">-</td>
</tr>
<tr id="S5.T2.1.1.12" class="ltx_tr">
<td id="S5.T2.1.1.12.1" class="ltx_td ltx_align_left">IBM FLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib118" title="" class="ltx_ref">118</a></sup></cite>
</td>
<td id="S5.T2.1.1.12.2" class="ltx_td ltx_align_center">IBM</td>
<td id="S5.T2.1.1.12.3" class="ltx_td ltx_align_center">DP, SMPC</td>
<td id="S5.T2.1.1.12.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.12.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.12.6" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.12.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.12.8" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.12.9" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.12.10" class="ltx_td ltx_align_center">-</td>
</tr>
<tr id="S5.T2.1.1.13" class="ltx_tr">
<td id="S5.T2.1.1.13.1" class="ltx_td ltx_align_left">PaddleFLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib119" title="" class="ltx_ref">119</a></sup></cite>
</td>
<td id="S5.T2.1.1.13.2" class="ltx_td ltx_align_center">Baidu</td>
<td id="S5.T2.1.1.13.3" class="ltx_td ltx_align_center">DP, SMPC</td>
<td id="S5.T2.1.1.13.4" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.13.5" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.13.6" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.13.7" class="ltx_td ltx_align_center">âœ“</td>
<td id="S5.T2.1.1.13.8" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.13.9" class="ltx_td ltx_align_center">-</td>
<td id="S5.T2.1.1.13.10" class="ltx_td ltx_align_center">-</td>
</tr>
<tr id="S5.T2.1.1.14" class="ltx_tr">
<td id="S5.T2.1.1.14.1" class="ltx_td ltx_align_left ltx_border_bb">SAFEFLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib120" title="" class="ltx_ref">120</a></sup></cite>
</td>
<td id="S5.T2.1.1.14.2" class="ltx_td ltx_align_center ltx_border_bb">ENCRYPTO</td>
<td id="S5.T2.1.1.14.3" class="ltx_td ltx_align_center ltx_border_bb">SMPC</td>
<td id="S5.T2.1.1.14.4" class="ltx_td ltx_align_center ltx_border_bb">-</td>
<td id="S5.T2.1.1.14.5" class="ltx_td ltx_align_center ltx_border_bb">-</td>
<td id="S5.T2.1.1.14.6" class="ltx_td ltx_align_center ltx_border_bb">-</td>
<td id="S5.T2.1.1.14.7" class="ltx_td ltx_align_center ltx_border_bb">-</td>
<td id="S5.T2.1.1.14.8" class="ltx_td ltx_align_center ltx_border_bb">-</td>
<td id="S5.T2.1.1.14.9" class="ltx_td ltx_align_center ltx_border_bb">-</td>
<td id="S5.T2.1.1.14.10" class="ltx_td ltx_align_center ltx_border_bb">-</td>
</tr>
</table>
</span></div>
</figure>
</section>
</section>
<section id="S5.SS1.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.1.2 </span>Deficiency in FL Frameworks Development and Usage</h4>

<div id="S5.SS1.SSS2.p1" class="ltx_para">
<p id="S5.SS1.SSS2.p1.1" class="ltx_p">Most studies developed their own FL frameworks, often not strictly aligning with standard FL protocols, particularly when confined to single machine simulation studies.
Meanwhile, some simulation studies used industrial-grade frameworks, which introduce unnecessary complexity and resource demands for simulation and prototype research.
Some other studies utilized lightweight open-source FL frameworks, although prevalent, frequently lack healthcare-specific adaptations, leading to deficiencies in privacy, security, and regulatory compliance.
Common shortcomings across current FL frameworks include a lack of healthcare adaptations, as most frameworks are not tailored to meet healthcare-specific requirements, which include stringent privacy, security, and regulatory standards.
Additionally, many frameworks do not address the need for communication efficiency, which is essential for the practical deployment of FL in resource-constrained environments.
Limited support for traceability also hinders accountability and transparency in FL.
Furthermore, while some frameworks offer scalability and cloud compatibility, many do not, which can limit their ability to handle large-scale healthcare data and integrate with existing cloud infrastructures.
Here, we inventory the most popular FL frameworks in TableÂ <a href="#S5.T2" title="Table 2 â€£ Recommendations &amp; Opportunities â€£ 5.1.1 Domination of Simulation Studies â€£ 5.1 Scenario and Framework â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>, with emphasis on those adapted for healthcare, and outline their features.</p>
</div>
<section id="S5.SS1.SSS2.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS1.SSS2.Px1.p1" class="ltx_para">
<p id="S5.SS1.SSS2.Px1.p1.1" class="ltx_p">Researchers aiming to swiftly prototype and test novel concepts can benefit from frameworks that incorporate comprehensive simulator modules. These tools allow for the rapid iteration and validation of ideas within a controlled simulation environment, which can be critical for the initial stages of research and development.
Engineers seeking to deploy FL in real-world scenarios should consider frameworks tailored to the specific needs of the healthcare domain. These frameworks should offer healthcare specific adaptations to ensure compatibility with medical data formats, regulatory compliance, and the unique challenges of healthcare data analysis.
Users facing computational constraints are encouraged to explore cloud-friendly frameworks that leverage cloud computing services such as Azure and AWS. These platforms can alleviate the burden of substantial computational demands and the complexities of local infrastructure development. Moreover, cloud computing can significantly mitigate the risk of network issues that may arise from client-hosted infrastructures with varying capabilities.
More specifically, we propose the following suggestions for FL framework selection, usage, and development:</p>
<ul id="S5.I2" class="ltx_itemize">
<li id="S5.I2.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I2.i1.p1" class="ltx_para">
<p id="S5.I2.i1.p1.1" class="ltx_p"><span id="S5.I2.i1.p1.1.1" class="ltx_text ltx_font_italic">Secure Communication:</span> The integrity of the FL system hinges on secure communication protocols, where encryption should be employedÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib121" title="" class="ltx_ref">121</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I2.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I2.i2.p1" class="ltx_para">
<p id="S5.I2.i2.p1.1" class="ltx_p"><span id="S5.I2.i2.p1.1.1" class="ltx_text ltx_font_italic">Flexible Deployment:</span> To streamline the deployment process, FL frameworks should support secure, reliable, and flexible deployment methods. They should integrate seamlessly with existing IT and data science infrastructures, facilitating a routine and uneventful deployment experienceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib122" title="" class="ltx_ref">122</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I2.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I2.i3.p1" class="ltx_para">
<p id="S5.I2.i3.p1.1" class="ltx_p"><span id="S5.I2.i3.p1.1.1" class="ltx_text ltx_font_italic">Data Science Workflow Support:</span> Given the necessity for diverse data providers to achieve robust FL outcomes, frameworks should support a comprehensive data science workflow. The ideal framework should be agnostic to both the model and the data, accommodating a wide range of data types and analytical methodsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib80" title="" class="ltx_ref">80</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I2.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I2.i4.p1" class="ltx_para">
<p id="S5.I2.i4.p1.1" class="ltx_p"><span id="S5.I2.i4.p1.1.1" class="ltx_text ltx_font_italic">Scalability:</span> Scalability is a key consideration for FL frameworks, which must accommodate an increasing number of participants and the corresponding complexity. Addressing scalability challenges, particularly with privacy-enhancing technologies such as synthetic data or HE, is crucial for the long-term viability of FL initiativesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib123" title="" class="ltx_ref">123</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I2.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I2.i5.p1" class="ltx_para">
<p id="S5.I2.i5.p1.1" class="ltx_p"><span id="S5.I2.i5.p1.1.1" class="ltx_text ltx_font_italic">Future-proofing:</span> FL frameworks should be designed with future-proofing in mind, anticipating emerging use cases, evolving security threats, and new privacy concerns. It should facilitate the dynamic participation of data providers, adapt computational resources to fluctuating client numbers, and implement regular system updates to address privacy and security challengesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib122" title="" class="ltx_ref">122</a></sup></cite>.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
</section>
<section id="S5.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.2 </span>Data</h3>

<section id="S5.SS2.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.2.1 </span>Unclear in Data Standardization and Harmonization</h4>

<div id="S5.SS2.SSS1.p1" class="ltx_para">
<p id="S5.SS2.SSS1.p1.1" class="ltx_p">Healthcare data are often collected and stored in diverse and proprietary formats that do not always adhere to international standards and terminologies, complicating data linkage and reuse.
For example, structured clinical data usually contains features that vary with differences in clinical practice across institutionsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib124" title="" class="ltx_ref">124</a></sup></cite>, such as diabetes diagnosis, which can involve different glucose measurement methods with varying cut-off points, resulting in hidden heterogeneity that may be overlooked in subsequent statistical analyses.
A crucial step before implementing FL in healthcare is to ensure data standardization, harmonization, and interoperability across different cohorts, which are key to the success of FL (FigureÂ <a href="#S5.F6" title="Figure 6 â€£ 5.2.1 Unclear in Data Standardization and Harmonization â€£ 5.2 Data â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a>).</p>
</div>
<div id="S5.SS2.SSS1.p2" class="ltx_para">
<p id="S5.SS2.SSS1.p2.1" class="ltx_p">Most simulation studies processed data centrally and generate artificially partitioned datasets without considering the distributed nature of various data silos. This oversight extends to the lack of discussion on how datasets at each client are curated for use in experiments.
Despite this, almost all FL frameworks assume the input data is preformatted for model training or preprocessing pipelines. This assumption leads to significant frustration and delays, as the burden of data export and conversion typically falls on clinical data managers who may lack the necessary budget and training.
Moreover, among the included studies, only two performed quality or integrity checks on the data. <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS2.SSS1.p2.1.1.1" class="ltx_sup"><a href="#bib.bib125" title="" class="ltx_ref">125</a></sup></cite> excluded samples with impossible values (e.g., negative heart rates) and inconsistent feature values, while <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS2.SSS1.p2.1.2.1" class="ltx_sup"><a href="#bib.bib126" title="" class="ltx_ref">126</a></sup></cite> used Principal Component Analysis to filter out noise.
Few studies addressed structural or informative missingness, with only <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS2.SSS1.p2.1.3.1" class="ltx_sup"><a href="#bib.bib57" title="" class="ltx_ref">57</a></sup></cite> and <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS2.SSS1.p2.1.4.1" class="ltx_sup"><a href="#bib.bib38" title="" class="ltx_ref">38</a></sup></cite> considering imputation methods while also deleting features with high missingness rates. Poor quality imputation and handling of non-random missingness can bias model training.</p>
</div>
<figure id="S5.F6" class="ltx_figure"><img src="/html/2409.09727/assets/x6.png" id="S5.F6.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="230" height="157" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 6: </span>Data Standardization and Harmonization.</figcaption>
</figure>
<section id="S5.SS2.SSS1.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS2.SSS1.Px1.p1" class="ltx_para">
<ul id="S5.I3" class="ltx_itemize">
<li id="S5.I3.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I3.i1.p1" class="ltx_para">
<p id="S5.I3.i1.p1.1" class="ltx_p"><span id="S5.I3.i1.p1.1.1" class="ltx_text ltx_font_italic">Adopt Standardized Formats:</span> Data standardization, harmonization, and interoperability across clients can be facilitated through formats such as Fast Healthcare Interoperability Resources (FHIR) for electronic health records (EHR) data and Digital Imaging and Communications in Medicine (DICOM) for imaging data.</p>
</div>
</li>
<li id="S5.I3.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I3.i2.p1" class="ltx_para">
<p id="S5.I3.i2.p1.1" class="ltx_p"><span id="S5.I3.i2.p1.1.1" class="ltx_text ltx_font_italic">Simplify Data Preparation:</span> FL frameworks targeting healthcare research must simplify data preparation and ensure interoperability with standard data formats. This approach eases the burden on clinical data managers and improves data reusability.</p>
</div>
</li>
<li id="S5.I3.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I3.i3.p1" class="ltx_para">
<p id="S5.I3.i3.p1.1" class="ltx_p"><span id="S5.I3.i3.p1.1.1" class="ltx_text ltx_font_italic">Use Consistent Protocols:</span> Collaborating clients must use consistent preprocessing protocols to standardize data to a Common Data Model (CDM) such as the Observational Medical Outcomes Partnership (OMOP)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib127" title="" class="ltx_ref">127</a></sup></cite>. Harmonizing healthcare data to a CDM like OMOP ensures it is interoperable with other clinical datasets, enabling effective merging and analysis across distributed sources and platformsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib93" title="" class="ltx_ref">93</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I3.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I3.i4.p1" class="ltx_para">
<p id="S5.I3.i4.p1.1" class="ltx_p"><span id="S5.I3.i4.p1.1.1" class="ltx_text ltx_font_italic">Automate Data Pipelines:</span> Extraction, transformation, and loading pipelines that automate the conversion of raw data to analysis-ready/training-ready data are needed to further simplify data standardization and harmonization.</p>
</div>
</li>
<li id="S5.I3.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I3.i5.p1" class="ltx_para">
<p id="S5.I3.i5.p1.1" class="ltx_p"><span id="S5.I3.i5.p1.1.1" class="ltx_text ltx_font_italic">Ensure Secure Access:</span> Secure access to fully standardized, harmonized, and interoperable large datasets through encryption methods can significantly accelerate clinical research within the federation.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS2.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.2.2 </span>Issues in Data Partition</h4>

<div id="S5.SS2.SSS2.p1" class="ltx_para">
<p id="S5.SS2.SSS2.p1.1" class="ltx_p">Among all included studies, only a few leveraged natural splits to replicate data collection processes across different hospitals or institutions. For instance, <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS2.SSS2.p1.1.1.1" class="ltx_sup"><a href="#bib.bib30" title="" class="ltx_ref">30</a></sup></cite> employed the CheXpert dataset, <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS2.SSS2.p1.1.2.1" class="ltx_sup"><a href="#bib.bib128" title="" class="ltx_ref">128</a></sup></cite> worked with the chest X-ray dataset and <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS2.SSS2.p1.1.3.1" class="ltx_sup"><a href="#bib.bib39" title="" class="ltx_ref">39</a></sup></cite> extracted metadata from Tissue Source sites in the TCGA dataset for their studies.
These datasets naturally reflect the heterogeneity found in real-world clinical data across hospitals and institutions, making them more suitable for FL studies in healthcareÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib129" title="" class="ltx_ref">129</a></sup>, <sup class="ltx_sup"><a href="#bib.bib58" title="" class="ltx_ref">58</a></sup></cite>.</p>
</div>
<div id="S5.SS2.SSS2.p2" class="ltx_para">
<p id="S5.SS2.SSS2.p2.1" class="ltx_p">Simulation studies typically used heuristics to artificially create heterogeneous data partitions from a pooled dataset, assigning these partitions to simulated clients, as illustrated in FigureÂ <a href="#S5.F4" title="Figure 4 â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.
Common synthetic partitioning methods for classification tasks include assigning samples from a limited number of classes to each client, using Dirichlet distribution sampling on class labels, and employing the Pachinko Allocation Method (PAM) when labels have a hierarchical structureÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib4" title="" class="ltx_ref">4</a></sup></cite>. For regression tasks, Gaussian Mixture clustering based on t-SNE feature representations has been used to partition datasets among clientsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib130" title="" class="ltx_ref">130</a></sup></cite>.</p>
</div>
<div id="S5.SS2.SSS2.p3" class="ltx_para">
<p id="S5.SS2.SSS2.p3.1" class="ltx_p">Nonetheless, synthetic partitioning methods may not accurately reflect the intricate heterogeneity found in real-world scenariosÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib90" title="" class="ltx_ref">90</a></sup></cite>.
Examples from digital histopathology illustrate the limitations of synthetic partitioning methodsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib131" title="" class="ltx_ref">131</a></sup></cite>. In digital histopathology, tissue samples are extracted, stained, and digitized, leading to data heterogeneity due to factors such as patient demographics, staining techniques, physical slide storage methods, and digitization processes. Although advancements in staining normalization have reduced some heterogeneity, other sources remain challenging to replicate synthetically, and some may even be unknownÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib132" title="" class="ltx_ref">132</a></sup></cite>. These underscore the necessity of conducting cohort experiments with natural splits to ensure that FL models are robust across varied clinical settings. This issue also extends to other areas, including radiology, dermatology, and retinal image analysis.</p>
</div>
<div id="S5.SS2.SSS2.p4" class="ltx_para">
<p id="S5.SS2.SSS2.p4.1" class="ltx_p">Even among studies that adopted synthetic partitioning methods, the strategies employed are often limited, primarily focusing on scenarios such as quantity skew. These studies addressed only a narrow aspect of heterogeneity. For instance, label skew, where the distribution of labels differs across clients, and feature skew, where clients have different feature distributions, are frequently overlooked. As a result, the synthetic partitions created in these studies may not adequately represent the complex and varied heterogeneous conditions, potentially leading to less robustness in diverse healthcare environments.
In SectionÂ <a href="#S5.SS4.SSS1" title="5.4.1 Heterogeneity Issues â€£ 5.4 Optimization â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5.4.1</span></a>, we provide a comprehensive discussion of various types of skew and heterogeneity.</p>
</div>
<div id="S5.SS2.SSS2.p5" class="ltx_para">
<p id="S5.SS2.SSS2.p5.1" class="ltx_p">Another significant issue is the lack of clear definitions and descriptions for train and test set partitions across clients in many studies. 84% included studies failed to explicitly define how these partitions are handled for each client, leading to potential ambiguity in evaluating model performance. This concern is particularly critical in the context of personalized FL, where each clientâ€™s test set should be unique to accurately reflect individual data distributions.</p>
</div>
<section id="S5.SS2.SSS2.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS2.SSS2.Px1.p1" class="ltx_para">
<ul id="S5.I4" class="ltx_itemize">
<li id="S5.I4.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I4.i1.p1" class="ltx_para">
<p id="S5.I4.i1.p1.1" class="ltx_p"><span id="S5.I4.i1.p1.1.1" class="ltx_text ltx_font_italic">Complement Simulation Studies with Real-World Data Evaluations:</span> While simulation studies using artificially partitioned datasets can provide valuable insights, it is essential to validate these findings through evaluations on real-world, naturally partitioned datasets. This multi-stage evaluation process ensures that models are tested in both controlled environments and realistic deployment scenarios, improving their generalizability and robustnessÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib133" title="" class="ltx_ref">133</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I4.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I4.i2.p1" class="ltx_para">
<p id="S5.I4.i2.p1.1" class="ltx_p"><span id="S5.I4.i2.p1.1.1" class="ltx_text ltx_font_italic">Adaptive Partitioning Based on Data Distribution:</span> Researchers should consider using adaptive partitioning techniques that account for the underlying data distribution and specific characteristics of each clientâ€™s data. This can create more realistic and representative partitions, especially in scenarios where data is highly heterogeneous.</p>
</div>
</li>
<li id="S5.I4.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I4.i3.p1" class="ltx_para">
<p id="S5.I4.i3.p1.1" class="ltx_p"><span id="S5.I4.i3.p1.1.1" class="ltx_text ltx_font_italic">Incorporate Multiple Types of Skew:</span> Researchers should broaden the scope of their synthetic partitioning strategies to include not just quantity skew but also label skew and feature skew. This would create a more realistic representation of the heterogeneity found in real-world datasets, allowing FL models to be more robust and generalizable across diverse clinical settingsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib134" title="" class="ltx_ref">134</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I4.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I4.i4.p1" class="ltx_para">
<p id="S5.I4.i4.p1.1" class="ltx_p"><span id="S5.I4.i4.p1.1.1" class="ltx_text ltx_font_italic">Retaining Client-Specific Test Sets:</span> Consider the non-IID nature of data in FL, we suggest that researchers retain a portion of data within each client as a dedicated test set rather than relying on a single, global test set for all clients. This approach provides a more accurate and reliable evaluation, reflecting the unique data distributions of each client, which is particularly important in personalized FL scenariosÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib133" title="" class="ltx_ref">133</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I4.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I4.i5.p1" class="ltx_para">
<p id="S5.I4.i5.p1.1" class="ltx_p"><span id="S5.I4.i5.p1.1.1" class="ltx_text ltx_font_italic">Incorporation of Domain Knowledge:</span> Incorporate domain knowledge into the partitioning process can enhance the relevance of synthetic data splits. In medical imaging, for example, understanding the clinical context and variability in imaging protocols across different institutions can inform more meaningful data partitioning strategies.</p>
</div>
</li>
<li id="S5.I4.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I4.i6.p1" class="ltx_para">
<p id="S5.I4.i6.p1.1" class="ltx_p"><span id="S5.I4.i6.p1.1.1" class="ltx_text ltx_font_italic">Transparency and Reproducibility:</span> Researchers should provide detailed documentation of their data partitioning strategies, including the rationale behind their choices and any domain-specific considerations. This transparency will enable others to replicate and build upon their work effectively.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
</section>
<section id="S5.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.3 </span>Model</h3>

<section id="S5.SS3.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.3.1 </span>Inadequate Model Selection and Development</h4>

<div id="S5.SS3.SSS1.p1" class="ltx_para">
<p id="S5.SS3.SSS1.p1.1" class="ltx_p">The studies reviewed exhibit a wide range of model complexities, from advanced, parameter-heavy architectures to traditional ML techniques.
However, several issues persist in model selection and development.
<span id="S5.SS3.SSS1.p1.1.1" class="ltx_text ltx_font_italic">Firstly</span>, there is an over-reliance on complex models, particularly CNNs, which dominate due to their high performance but pose challenges in resource-constrained healthcare environments. Their complexity complicates reproducibility and generalizability across different settings, particularly when custom architectures are involved.
<span id="S5.SS3.SSS1.p1.1.2" class="ltx_text ltx_font_italic">Secondly</span>, the lack of standardization in model selection leads to variability in methodologies, making it difficult to compare results across studies and generalize findings. This inconsistency hampers the ability to benchmark performance across different FL applications.
<span id="S5.SS3.SSS1.p1.1.3" class="ltx_text ltx_font_italic">Moreover</span>, simpler and more interpretable models are underutilized. While deep learning models offer high performance, traditional ML algorithms, which are easier to interpret and less resource-intensive, are often overlooked. These models could be more suitable for certain healthcare applications where interpretability and transparency are critical for clinical decision-making, offering a practical alternative that balances performance with the need for clarity and trustworthiness in healthcare settings.
<span id="S5.SS3.SSS1.p1.1.4" class="ltx_text ltx_font_italic">Another challenge</span> is the limited focus on personalization. Many studies prioritized a single global model, which may not be optimal for all clients due to the heterogeneity in healthcare data. Personalized FL approaches, tailored to individual client data distributions, remain underdeveloped and require further research.
<span id="S5.SS3.SSS1.p1.1.5" class="ltx_text ltx_font_italic">Lastly</span>, scalability concerns arise with complex models, particularly in large-scale healthcare networks. The communication and computational overhead of training such models can become prohibitive, highlighting the need for more scalable FL solutions to ensure practical deployment.</p>
</div>
<section id="S5.SS3.SSS1.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS3.SSS1.Px1.p1" class="ltx_para">
<ul id="S5.I5" class="ltx_itemize">
<li id="S5.I5.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I5.i1.p1" class="ltx_para">
<p id="S5.I5.i1.p1.1" class="ltx_p"><span id="S5.I5.i1.p1.1.1" class="ltx_text ltx_font_italic">Standardization for Research and Benchmarking:</span> There should be an emphasis on standardizing model architectures to enable better comparison and benchmarking across studies, especially when trying to establish baseline performance or when deploying FL in more homogeneous environments. Researchers should also consider developing more lightweight models that can be deployed in resource-constrained environments without sacrificing performance.</p>
</div>
</li>
<li id="S5.I5.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I5.i2.p1" class="ltx_para">
<p id="S5.I5.i2.p1.1" class="ltx_p"><span id="S5.I5.i2.p1.1.1" class="ltx_text ltx_font_italic">Exploring Simpler, More Interpretable Models:</span> Researchers should not overlook simpler, traditional ML models, particularly in scenarios where interpretability is crucial. These models can be more practical and equally effective in certain healthcare applications, providing a balance between performance and interpretabilityÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib135" title="" class="ltx_ref">135</a></sup>, <sup class="ltx_sup"><a href="#bib.bib21" title="" class="ltx_ref">21</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I5.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I5.i3.p1" class="ltx_para">
<p id="S5.I5.i3.p1.1" class="ltx_p"><span id="S5.I5.i3.p1.1.1" class="ltx_text ltx_font_italic">Enhancing Personalization in FL:</span> More work is needed to develop and refine personalized FL approaches that can adapt to the diverse data distributions encountered in healthcare. This could involve hybrid models that combine the strengths of both global and local models, as well as more sophisticated techniques for model adaptationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib49" title="" class="ltx_ref">49</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I5.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I5.i4.p1" class="ltx_para">
<p id="S5.I5.i4.p1.1" class="ltx_p"><span id="S5.I5.i4.p1.1.1" class="ltx_text ltx_font_italic">Addressing Model Heterogeneity:</span> Given the diverse requirements and constraints across different institutions, it is crucial to develop FL strategies that can effectively manage model heterogeneity. This includes exploring federated ensemble learning methods, which allow the aggregation of heterogeneous models and can lead to more robust and accurate predictionsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib136" title="" class="ltx_ref">136</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I5.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I5.i5.p1" class="ltx_para">
<p id="S5.I5.i5.p1.1" class="ltx_p"><span id="S5.I5.i5.p1.1.1" class="ltx_text ltx_font_italic">Improving Scalability and Efficiency:</span> Future studies should prioritize the design of scalable FL models that can handle an increasing number of clients without excessive computational and communication costs. This could involve the development of more efficient algorithms and the use of federated distillation techniques to reduce model size and complexityÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib137" title="" class="ltx_ref">137</a></sup></cite>.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS3.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.3.2 </span>Negligence in Initialization</h4>

<div id="S5.SS3.SSS2.p1" class="ltx_para">
<p id="S5.SS3.SSS2.p1.1" class="ltx_p">Most of the included studies began federated training from a random initialization, a method that, while effective in IID scenarios, can be less optimal for handling non-IID data. In healthcare, where data distribution often varies significantly across institutions due to differences in patient demographics, clinical practices, or data collection methods, random initialization can lead to slower convergence, increased communication costs, and potentially suboptimal local optimaÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib138" title="" class="ltx_ref">138</a></sup>, <sup class="ltx_sup"><a href="#bib.bib139" title="" class="ltx_ref">139</a></sup></cite>.</p>
</div>
<div id="S5.SS3.SSS2.p2" class="ltx_para">
<p id="S5.SS3.SSS2.p2.1" class="ltx_p">A significant issue is the lack of standardization in model initialization approaches. Many studies either adopted random initialization without justification or entirely omitted the description of their initialization method. This inconsistency can result in significant variations in model performance and convergence rates, making it difficult to compare results across different studies and settings. Additionally, if the initial model is biased towards the data distribution of certain participants, it might not perform well across all clients, leading to fairness issues and suboptimal overall performance.
Moreover, these challenges are further exacerbated by the heterogeneity of computational resources available across institutions. Some advanced initialization methods, such as those involving foundation models or pretraining on large-scale datasets, may be computationally expensive and thus infeasible for resource-constrained participantsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib138" title="" class="ltx_ref">138</a></sup>, <sup class="ltx_sup"><a href="#bib.bib96" title="" class="ltx_ref">96</a></sup>, <sup class="ltx_sup"><a href="#bib.bib43" title="" class="ltx_ref">43</a></sup></cite>.</p>
</div>
<div id="S5.SS3.SSS2.p3" class="ltx_para">
<p id="S5.SS3.SSS2.p3.1" class="ltx_p">Personalization in model initialization is another underexplored area. Personalized initialization techniques, which tailor the starting point to the specific data distribution of each client, are critical for improving local model performance and accelerating convergence. However, research into these techniques, such as model-agnostic meta-learning and partial initialization for finding a good initialization, remains limited within FL for healthcareÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib140" title="" class="ltx_ref">140</a></sup>, <sup class="ltx_sup"><a href="#bib.bib141" title="" class="ltx_ref">141</a></sup></cite>.</p>
</div>
<section id="S5.SS3.SSS2.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS3.SSS2.Px1.p1" class="ltx_para">
<ul id="S5.I6" class="ltx_itemize">
<li id="S5.I6.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I6.i1.p1" class="ltx_para">
<p id="S5.I6.i1.p1.1" class="ltx_p"><span id="S5.I6.i1.p1.1.1" class="ltx_text ltx_font_italic">Transfer Learning for Initialization:</span> Utilize pretraining or foundation models for initialization can provide a strong starting point for FL. This approach has been shown to not only speed up convergence but also mitigate the effects of both data and system heterogeneityÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib96" title="" class="ltx_ref">96</a></sup>, <sup class="ltx_sup"><a href="#bib.bib43" title="" class="ltx_ref">43</a></sup>, <sup class="ltx_sup"><a href="#bib.bib138" title="" class="ltx_ref">138</a></sup></cite>, potentially closing the performance gap between FL and centralized learningÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib139" title="" class="ltx_ref">139</a></sup></cite>. However, researchers should carefully consider the similarity between the source and target datasets to avoid negative transfer effects.</p>
</div>
</li>
<li id="S5.I6.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I6.i2.p1" class="ltx_para">
<p id="S5.I6.i2.p1.1" class="ltx_p"><span id="S5.I6.i2.p1.1.1" class="ltx_text ltx_font_italic">Personalized Initialization:</span> Implement personalized initialization methods, such as model-agnostic meta-learning and partial initialization, can help customize the starting point of the model based on local data characteristics, further enhancing model performance on individual clients and improving overall system convergenceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib140" title="" class="ltx_ref">140</a></sup>, <sup class="ltx_sup"><a href="#bib.bib141" title="" class="ltx_ref">141</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I6.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I6.i3.p1" class="ltx_para">
<p id="S5.I6.i3.p1.1" class="ltx_p"><span id="S5.I6.i3.p1.1.1" class="ltx_text ltx_font_italic">Standardization of Initialization Procedures:</span> To ensure consistency and comparability across studies, there should be a push towards standardizing initialization methods in FL research. Providing clear descriptions and justifications for the chosen initialization strategy is essential for reproducibility and for advancing the field.</p>
</div>
</li>
<li id="S5.I6.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I6.i4.p1" class="ltx_para">
<p id="S5.I6.i4.p1.1" class="ltx_p"><span id="S5.I6.i4.p1.1.1" class="ltx_text ltx_font_italic">Consideration of Resource Constraints:</span> When selecting initialization methods, researchers should account for the varying computational resources across participants. Techniques that balance initialization quality with computational feasibility, such as hierarchical model training or using lightweight pretrained models, are critical to ensuring broader applicability of FL in healthcareÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib96" title="" class="ltx_ref">96</a></sup>, <sup class="ltx_sup"><a href="#bib.bib43" title="" class="ltx_ref">43</a></sup>, <sup class="ltx_sup"><a href="#bib.bib138" title="" class="ltx_ref">138</a></sup></cite>.</p>
</div>
</li>
</ul>
</div>
<figure id="S5.F7" class="ltx_figure"><img src="/html/2409.09727/assets/x7.png" id="S5.F7.g1" class="ltx_graphics ltx_centering ltx_img_square" width="276" height="242" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 7: </span>Drift issue in non-IID. In IID scenario, the global optima <math id="S5.F7.8.m1.1" class="ltx_Math" alttext="w^{*}" display="inline"><semantics id="S5.F7.8.m1.1b"><msup id="S5.F7.8.m1.1.1" xref="S5.F7.8.m1.1.1.cmml"><mi id="S5.F7.8.m1.1.1.2" xref="S5.F7.8.m1.1.1.2.cmml">w</mi><mo id="S5.F7.8.m1.1.1.3" xref="S5.F7.8.m1.1.1.3.cmml">âˆ—</mo></msup><annotation-xml encoding="MathML-Content" id="S5.F7.8.m1.1c"><apply id="S5.F7.8.m1.1.1.cmml" xref="S5.F7.8.m1.1.1"><csymbol cd="ambiguous" id="S5.F7.8.m1.1.1.1.cmml" xref="S5.F7.8.m1.1.1">superscript</csymbol><ci id="S5.F7.8.m1.1.1.2.cmml" xref="S5.F7.8.m1.1.1.2">ğ‘¤</ci><times id="S5.F7.8.m1.1.1.3.cmml" xref="S5.F7.8.m1.1.1.3"></times></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.F7.8.m1.1d">w^{*}</annotation></semantics></math> aligns closely with the local optima <math id="S5.F7.9.m2.1" class="ltx_Math" alttext="w^{*}_{1}" display="inline"><semantics id="S5.F7.9.m2.1b"><msubsup id="S5.F7.9.m2.1.1" xref="S5.F7.9.m2.1.1.cmml"><mi id="S5.F7.9.m2.1.1.2.2" xref="S5.F7.9.m2.1.1.2.2.cmml">w</mi><mn id="S5.F7.9.m2.1.1.3" xref="S5.F7.9.m2.1.1.3.cmml">1</mn><mo id="S5.F7.9.m2.1.1.2.3" xref="S5.F7.9.m2.1.1.2.3.cmml">âˆ—</mo></msubsup><annotation-xml encoding="MathML-Content" id="S5.F7.9.m2.1c"><apply id="S5.F7.9.m2.1.1.cmml" xref="S5.F7.9.m2.1.1"><csymbol cd="ambiguous" id="S5.F7.9.m2.1.1.1.cmml" xref="S5.F7.9.m2.1.1">subscript</csymbol><apply id="S5.F7.9.m2.1.1.2.cmml" xref="S5.F7.9.m2.1.1"><csymbol cd="ambiguous" id="S5.F7.9.m2.1.1.2.1.cmml" xref="S5.F7.9.m2.1.1">superscript</csymbol><ci id="S5.F7.9.m2.1.1.2.2.cmml" xref="S5.F7.9.m2.1.1.2.2">ğ‘¤</ci><times id="S5.F7.9.m2.1.1.2.3.cmml" xref="S5.F7.9.m2.1.1.2.3"></times></apply><cn type="integer" id="S5.F7.9.m2.1.1.3.cmml" xref="S5.F7.9.m2.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.F7.9.m2.1d">w^{*}_{1}</annotation></semantics></math> and <math id="S5.F7.10.m3.1" class="ltx_Math" alttext="w^{*}_{2}" display="inline"><semantics id="S5.F7.10.m3.1b"><msubsup id="S5.F7.10.m3.1.1" xref="S5.F7.10.m3.1.1.cmml"><mi id="S5.F7.10.m3.1.1.2.2" xref="S5.F7.10.m3.1.1.2.2.cmml">w</mi><mn id="S5.F7.10.m3.1.1.3" xref="S5.F7.10.m3.1.1.3.cmml">2</mn><mo id="S5.F7.10.m3.1.1.2.3" xref="S5.F7.10.m3.1.1.2.3.cmml">âˆ—</mo></msubsup><annotation-xml encoding="MathML-Content" id="S5.F7.10.m3.1c"><apply id="S5.F7.10.m3.1.1.cmml" xref="S5.F7.10.m3.1.1"><csymbol cd="ambiguous" id="S5.F7.10.m3.1.1.1.cmml" xref="S5.F7.10.m3.1.1">subscript</csymbol><apply id="S5.F7.10.m3.1.1.2.cmml" xref="S5.F7.10.m3.1.1"><csymbol cd="ambiguous" id="S5.F7.10.m3.1.1.2.1.cmml" xref="S5.F7.10.m3.1.1">superscript</csymbol><ci id="S5.F7.10.m3.1.1.2.2.cmml" xref="S5.F7.10.m3.1.1.2.2">ğ‘¤</ci><times id="S5.F7.10.m3.1.1.2.3.cmml" xref="S5.F7.10.m3.1.1.2.3"></times></apply><cn type="integer" id="S5.F7.10.m3.1.1.3.cmml" xref="S5.F7.10.m3.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.F7.10.m3.1d">w^{*}_{2}</annotation></semantics></math>. Consequently, the aggregated model <math id="S5.F7.11.m4.1" class="ltx_Math" alttext="w^{t+1}" display="inline"><semantics id="S5.F7.11.m4.1b"><msup id="S5.F7.11.m4.1.1" xref="S5.F7.11.m4.1.1.cmml"><mi id="S5.F7.11.m4.1.1.2" xref="S5.F7.11.m4.1.1.2.cmml">w</mi><mrow id="S5.F7.11.m4.1.1.3" xref="S5.F7.11.m4.1.1.3.cmml"><mi id="S5.F7.11.m4.1.1.3.2" xref="S5.F7.11.m4.1.1.3.2.cmml">t</mi><mo id="S5.F7.11.m4.1.1.3.1" xref="S5.F7.11.m4.1.1.3.1.cmml">+</mo><mn id="S5.F7.11.m4.1.1.3.3" xref="S5.F7.11.m4.1.1.3.3.cmml">1</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="S5.F7.11.m4.1c"><apply id="S5.F7.11.m4.1.1.cmml" xref="S5.F7.11.m4.1.1"><csymbol cd="ambiguous" id="S5.F7.11.m4.1.1.1.cmml" xref="S5.F7.11.m4.1.1">superscript</csymbol><ci id="S5.F7.11.m4.1.1.2.cmml" xref="S5.F7.11.m4.1.1.2">ğ‘¤</ci><apply id="S5.F7.11.m4.1.1.3.cmml" xref="S5.F7.11.m4.1.1.3"><plus id="S5.F7.11.m4.1.1.3.1.cmml" xref="S5.F7.11.m4.1.1.3.1"></plus><ci id="S5.F7.11.m4.1.1.3.2.cmml" xref="S5.F7.11.m4.1.1.3.2">ğ‘¡</ci><cn type="integer" id="S5.F7.11.m4.1.1.3.3.cmml" xref="S5.F7.11.m4.1.1.3.3">1</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.F7.11.m4.1d">w^{t+1}</annotation></semantics></math> remains near the global optima. However, in non-IID scenario, the global optima <math id="S5.F7.12.m5.1" class="ltx_Math" alttext="w^{*}" display="inline"><semantics id="S5.F7.12.m5.1b"><msup id="S5.F7.12.m5.1.1" xref="S5.F7.12.m5.1.1.cmml"><mi id="S5.F7.12.m5.1.1.2" xref="S5.F7.12.m5.1.1.2.cmml">w</mi><mo id="S5.F7.12.m5.1.1.3" xref="S5.F7.12.m5.1.1.3.cmml">âˆ—</mo></msup><annotation-xml encoding="MathML-Content" id="S5.F7.12.m5.1c"><apply id="S5.F7.12.m5.1.1.cmml" xref="S5.F7.12.m5.1.1"><csymbol cd="ambiguous" id="S5.F7.12.m5.1.1.1.cmml" xref="S5.F7.12.m5.1.1">superscript</csymbol><ci id="S5.F7.12.m5.1.1.2.cmml" xref="S5.F7.12.m5.1.1.2">ğ‘¤</ci><times id="S5.F7.12.m5.1.1.3.cmml" xref="S5.F7.12.m5.1.1.3"></times></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.F7.12.m5.1d">w^{*}</annotation></semantics></math> may be significantly distant from <math id="S5.F7.13.m6.1" class="ltx_Math" alttext="w^{*}_{1}" display="inline"><semantics id="S5.F7.13.m6.1b"><msubsup id="S5.F7.13.m6.1.1" xref="S5.F7.13.m6.1.1.cmml"><mi id="S5.F7.13.m6.1.1.2.2" xref="S5.F7.13.m6.1.1.2.2.cmml">w</mi><mn id="S5.F7.13.m6.1.1.3" xref="S5.F7.13.m6.1.1.3.cmml">1</mn><mo id="S5.F7.13.m6.1.1.2.3" xref="S5.F7.13.m6.1.1.2.3.cmml">âˆ—</mo></msubsup><annotation-xml encoding="MathML-Content" id="S5.F7.13.m6.1c"><apply id="S5.F7.13.m6.1.1.cmml" xref="S5.F7.13.m6.1.1"><csymbol cd="ambiguous" id="S5.F7.13.m6.1.1.1.cmml" xref="S5.F7.13.m6.1.1">subscript</csymbol><apply id="S5.F7.13.m6.1.1.2.cmml" xref="S5.F7.13.m6.1.1"><csymbol cd="ambiguous" id="S5.F7.13.m6.1.1.2.1.cmml" xref="S5.F7.13.m6.1.1">superscript</csymbol><ci id="S5.F7.13.m6.1.1.2.2.cmml" xref="S5.F7.13.m6.1.1.2.2">ğ‘¤</ci><times id="S5.F7.13.m6.1.1.2.3.cmml" xref="S5.F7.13.m6.1.1.2.3"></times></apply><cn type="integer" id="S5.F7.13.m6.1.1.3.cmml" xref="S5.F7.13.m6.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.F7.13.m6.1d">w^{*}_{1}</annotation></semantics></math>, leading to <math id="S5.F7.14.m7.1" class="ltx_Math" alttext="w^{t+1}" display="inline"><semantics id="S5.F7.14.m7.1b"><msup id="S5.F7.14.m7.1.1" xref="S5.F7.14.m7.1.1.cmml"><mi id="S5.F7.14.m7.1.1.2" xref="S5.F7.14.m7.1.1.2.cmml">w</mi><mrow id="S5.F7.14.m7.1.1.3" xref="S5.F7.14.m7.1.1.3.cmml"><mi id="S5.F7.14.m7.1.1.3.2" xref="S5.F7.14.m7.1.1.3.2.cmml">t</mi><mo id="S5.F7.14.m7.1.1.3.1" xref="S5.F7.14.m7.1.1.3.1.cmml">+</mo><mn id="S5.F7.14.m7.1.1.3.3" xref="S5.F7.14.m7.1.1.3.3.cmml">1</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="S5.F7.14.m7.1c"><apply id="S5.F7.14.m7.1.1.cmml" xref="S5.F7.14.m7.1.1"><csymbol cd="ambiguous" id="S5.F7.14.m7.1.1.1.cmml" xref="S5.F7.14.m7.1.1">superscript</csymbol><ci id="S5.F7.14.m7.1.1.2.cmml" xref="S5.F7.14.m7.1.1.2">ğ‘¤</ci><apply id="S5.F7.14.m7.1.1.3.cmml" xref="S5.F7.14.m7.1.1.3"><plus id="S5.F7.14.m7.1.1.3.1.cmml" xref="S5.F7.14.m7.1.1.3.1"></plus><ci id="S5.F7.14.m7.1.1.3.2.cmml" xref="S5.F7.14.m7.1.1.3.2">ğ‘¡</ci><cn type="integer" id="S5.F7.14.m7.1.1.3.3.cmml" xref="S5.F7.14.m7.1.1.3.3">1</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S5.F7.14.m7.1d">w^{t+1}</annotation></semantics></math> being far from the ideal solution.</figcaption>
</figure>
</section>
</section>
</section>
<section id="S5.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.4 </span>Optimization</h3>

<section id="S5.SS4.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.4.1 </span>Heterogeneity Issues</h4>

<div id="S5.SS4.SSS1.p1" class="ltx_para">
<p id="S5.SS4.SSS1.p1.1" class="ltx_p">In FL for healthcare, heterogeneity refers to the variability in data, models, and systems across different hospitals and institutions. This variability poses significant challenges to FLâ€™s performance and its ability to generalize well across diverse environments. The key types of heterogeneity in FL for healthcare include <span id="S5.SS4.SSS1.p1.1.1" class="ltx_text ltx_font_italic">statistical heterogeneity</span>, <span id="S5.SS4.SSS1.p1.1.2" class="ltx_text ltx_font_italic">model heterogeneity</span>, and <span id="S5.SS4.SSS1.p1.1.3" class="ltx_text ltx_font_italic">system heterogeneity</span>.</p>
</div>
<div id="S5.SS4.SSS1.p2" class="ltx_para">
<p id="S5.SS4.SSS1.p2.1" class="ltx_p"><span id="S5.SS4.SSS1.p2.1.1" class="ltx_text ltx_font_italic">Statistical heterogeneity</span> arises due to the non-IID nature of healthcare data across various institutions, which is characterized by demographic differences, instrumentation biases, distinct data acquisition protocols, and human operations, etcÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib142" title="" class="ltx_ref">142</a></sup></cite>.
For instance, variations in CT scan quality across sites can lead to inconsistencies in the correlation between imaging data and corresponding site-specific EHR data. These inconsistencies severely degrade FL performance, with accuracy drops of up to 50%Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib4" title="" class="ltx_ref">4</a></sup></cite>, necessitating additional communication rounds for convergenceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib143" title="" class="ltx_ref">143</a></sup></cite>.
Statistical heterogeneity can also result in clients overfitting to their local data, leading to poor generalization on data from other clients, making simple parameter averaging an ineffective aggregation strategyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib144" title="" class="ltx_ref">144</a></sup></cite>.
Since local models are optimized towards different local optima, the aggregated global model may drift from the true global optima, causing a biased minimum and significantly slowing down convergence as illustrated in FigureÂ <a href="#S5.F7" title="Figure 7 â€£ Recommendations &amp; Opportunities â€£ 5.3.2 Negligence in Initialization â€£ 5.3 Model â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>.
In healthcare, statistical heterogeneity can be broadly characterized by four forms, including:</p>
<ul id="S5.I7" class="ltx_itemize">
<li id="S5.I7.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I7.i1.p1" class="ltx_para">
<p id="S5.I7.i1.p1.1" class="ltx_p"><span id="S5.I7.i1.p1.1.1" class="ltx_text ltx_font_italic">Quantity Skew.</span> The number of training samples differs greatly across clients, leading to imbalanced data distributions. Models tend to optimizing for clients with more data, potentially neglecting those with less, further reducing the generalization abilityÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib134" title="" class="ltx_ref">134</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I7.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I7.i2.p1" class="ltx_para">
<p id="S5.I7.i2.p1.1" class="ltx_p"><span id="S5.I7.i2.p1.1.1" class="ltx_text ltx_font_italic">Label Skew.</span> The distribution of labels varies across clients. For instance, in the context of COVID-19, hospitals in regions heavily impacted by the pandemic may have a higher proportion of positive cases, whereas other regions might have predominantly negative cases. This label imbalance can lead to biased modelsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib145" title="" class="ltx_ref">145</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I7.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I7.i3.p1" class="ltx_para">
<p id="S5.I7.i3.p1.1" class="ltx_p"><span id="S5.I7.i3.p1.1.1" class="ltx_text ltx_font_italic">Feature Skew.</span> Different clients may have access to different features for the same sample cohort. For example, some institutions might only have access to EHR data, while others may have additional imaging modalities like X-rays or MRIsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib140" title="" class="ltx_ref">140</a></sup>, <sup class="ltx_sup"><a href="#bib.bib17" title="" class="ltx_ref">17</a></sup></cite>. This is especially common in VFL scenarios.</p>
</div>
</li>
<li id="S5.I7.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I7.i4.p1" class="ltx_para">
<p id="S5.I7.i4.p1.1" class="ltx_p"><span id="S5.I7.i4.p1.1.1" class="ltx_text ltx_font_italic">Quality Skew.</span> This arises from varying data quality across clients, often due to issues like label noise, data acquisition noise, or processing discrepancies. Clients with high-quality, accurately labeled data contribute more effectively to model learning, while those with noisy or inaccurate labels can introduce errors, undermining the modelâ€™s generalization and convergenceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib146" title="" class="ltx_ref">146</a></sup></cite>.</p>
</div>
</li>
</ul>
<p id="S5.SS4.SSS1.p2.2" class="ltx_p">FigureÂ <a href="#S5.F8" title="Figure 8 â€£ 5.4.1 Heterogeneity Issues â€£ 5.4 Optimization â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a> provides an illustration of these skew forms.
Numerous studies have been focusing on one of these skews in healthcare.
<cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS4.SSS1.p2.2.1.1" class="ltx_sup"><a href="#bib.bib98" title="" class="ltx_ref">98</a></sup></cite> introduced a generative replay method by employing a VAE to synthesize medical images, enabling clients to train on a combined dataset of real and generated data, thus mitigating data quantity skew.
<cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS4.SSS1.p2.2.2.1" class="ltx_sup"><a href="#bib.bib67" title="" class="ltx_ref">67</a></sup></cite> employed a frequency-based approach for medical data harmonization in FL, where images are processed in the frequency domain to retain local phase information and synchronize amplitudes across clients, thereby mitigating feature skew.
<cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS4.SSS1.p2.2.3.1" class="ltx_sup"><a href="#bib.bib131" title="" class="ltx_ref">131</a></sup></cite> leveraged a Dirichlet distribution for modeling categorical probabilities in medical data, applying uncertainty calibration and diversity relaxation to enhance label annotation by focusing on samples with high uncertainty and low similarity, thus reducing label quality skew.</p>
</div>
<figure id="S5.F8" class="ltx_figure"><img src="/html/2409.09727/assets/x8.png" id="S5.F8.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="364" height="281" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 8: </span>Illustration of different skew forms in statistical heterogeneity, including quantity, feature, label, and quality skew. Medical image sources: Chest X-raysÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib147" title="" class="ltx_ref">147</a></sup></cite>, CT scansÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib148" title="" class="ltx_ref">148</a></sup></cite>, Ultrasound imagesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib149" title="" class="ltx_ref">149</a></sup></cite>, and Whole Slide Imaging (WSI)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib150" title="" class="ltx_ref">150</a></sup></cite>.</figcaption>
</figure>
<div id="S5.SS4.SSS1.p3" class="ltx_para">
<p id="S5.SS4.SSS1.p3.1" class="ltx_p"><span id="S5.SS4.SSS1.p3.1.1" class="ltx_text ltx_font_italic">Model heterogeneity</span> occurs when different clients use varying model architectures due to differences in hardware capabilities, data types, or specific institutional requirements. For example, one hospital may use a complex deep learning model for image analysis, while another uses a simpler model due to computational constraints. This divergence complicates the process of aggregating model updates in FL, as different architectures may have different fitting capabilities, performance characteristics, and requirements for convergenceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib151" title="" class="ltx_ref">151</a></sup></cite>.
To address model heterogeneity, approaches like knowledge distillation have been employed, where a â€œstudentâ€ model is used to transfer knowledge from various â€œteacherâ€ models in different clientsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib151" title="" class="ltx_ref">151</a></sup></cite>. However, these methods often require auxiliary public datasets for transferring knowledge, raising privacy concerns and increasing the computational burden on resource-constrained institutionsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib152" title="" class="ltx_ref">152</a></sup>, <sup class="ltx_sup"><a href="#bib.bib151" title="" class="ltx_ref">151</a></sup></cite>.
An alternative approach is the use of a lightweight â€œmessengerâ€ model, as proposed by <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS4.SSS1.p3.1.2.1" class="ltx_sup"><a href="#bib.bib99" title="" class="ltx_ref">99</a></sup></cite>, which carries concentrated information from one client to another, reducing the need for a full model exchange. This method allows for efficient aggregation and distribution of knowledge without the overhead of auxiliary public datasets.</p>
</div>
<div id="S5.SS4.SSS1.p4" class="ltx_para">
<p id="S5.SS4.SSS1.p4.1" class="ltx_p"><span id="S5.SS4.SSS1.p4.1.1" class="ltx_text ltx_font_italic">System heterogeneity</span> refers to the differences in computational capabilities, network architectures, and resource availability across clients. For example, some hospitals may have advanced computing infrastructure, while others may have limited hardware resources. This variability may affect the efficiency of FL, particularly in aggregating models trained on non-IID dataÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib134" title="" class="ltx_ref">134</a></sup>, <sup class="ltx_sup"><a href="#bib.bib136" title="" class="ltx_ref">136</a></sup></cite>. Most studies have focused on addressing statistical or model heterogeneity, but system heterogeneity is equally important. Differences in hardware resources can lead to discrepancies in how models perform and converge across clients. Furthermore, the need for additional local computation and storage resources, as required by methods like knowledge distillation, can be a burden for institutions with limited resourcesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib27" title="" class="ltx_ref">27</a></sup>, <sup class="ltx_sup"><a href="#bib.bib152" title="" class="ltx_ref">152</a></sup></cite>.</p>
</div>
<section id="S5.SS4.SSS1.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS4.SSS1.Px1.p1" class="ltx_para">
<ul id="S5.I8" class="ltx_itemize">
<li id="S5.I8.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i1.p1" class="ltx_para">
<p id="S5.I8.i1.p1.1" class="ltx_p"><span id="S5.I8.i1.p1.1.1" class="ltx_text ltx_font_italic">Data Harmonization:</span> Apply data harmonization techniques locally at each client site to minimize variability in data distributions and improve the consistency of FL modelsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib67" title="" class="ltx_ref">67</a></sup>, <sup class="ltx_sup"><a href="#bib.bib153" title="" class="ltx_ref">153</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I8.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i2.p1" class="ltx_para">
<p id="S5.I8.i2.p1.1" class="ltx_p"><span id="S5.I8.i2.p1.1.1" class="ltx_text ltx_font_italic">Data Synthesis and Augmentation:</span> Use data synthesis and augmentation techniques to generate additional data for underrepresented classes in the local data. Techniques such as GAN, VAE, and diffusion models can be leveraged to create synthetic data that preserves privacy while boosting model generalization.</p>
</div>
</li>
<li id="S5.I8.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i3.p1" class="ltx_para">
<p id="S5.I8.i3.p1.1" class="ltx_p"><span id="S5.I8.i3.p1.1.1" class="ltx_text ltx_font_italic">Feature Alignment and Data Imputation:</span> Use methods like feature alignment and data imputation during training to ensure models learn from consistent data across clients. For example, FedHealthÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib77" title="" class="ltx_ref">77</a></sup></cite> demonstrated the ability to infer missing modalities in healthcare data through FTL.</p>
</div>
</li>
<li id="S5.I8.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i4.p1" class="ltx_para">
<p id="S5.I8.i4.p1.1" class="ltx_p"><span id="S5.I8.i4.p1.1.1" class="ltx_text ltx_font_italic">Bias Checks and Corrections:</span> Implement continuous monitoring for bias within and across clients throughout the FL process. Correct identified biases to prevent model performance degradation. For instance, <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.I8.i4.p1.1.2.1" class="ltx_sup"><a href="#bib.bib131" title="" class="ltx_ref">131</a></sup></cite> utilized uncertainty calibration and diversity relaxation to dynamically correct annotations for high-uncertainty, low-similarity samples.</p>
</div>
</li>
<li id="S5.I8.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i5.p1" class="ltx_para">
<p id="S5.I8.i5.p1.1" class="ltx_p"><span id="S5.I8.i5.p1.1.1" class="ltx_text ltx_font_italic">Advanced Optimization Techniques:</span> Enhance FL robustness against client drift and heterogeneity by employing advanced optimization techniques such as FedProxÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib154" title="" class="ltx_ref">154</a></sup></cite>, SCAFFOLDÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib155" title="" class="ltx_ref">155</a></sup></cite>, and MOONÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib156" title="" class="ltx_ref">156</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I8.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i6.p1" class="ltx_para">
<p id="S5.I8.i6.p1.1" class="ltx_p"><span id="S5.I8.i6.p1.1.1" class="ltx_text ltx_font_italic">Disentangled Representation Learning (RDL):</span> Integrate DRL into federated models to disentangle underlying invariant factors, making models more robust to data heterogeneity across different institutionsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib76" title="" class="ltx_ref">76</a></sup>, <sup class="ltx_sup"><a href="#bib.bib157" title="" class="ltx_ref">157</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I8.i7" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i7.p1" class="ltx_para">
<p id="S5.I8.i7.p1.1" class="ltx_p"><span id="S5.I8.i7.p1.1.1" class="ltx_text ltx_font_italic">Fine-Tuning and Personalization:</span> Improve model performance on domain-specific tasks by fine-tuning models locally with client-specific data and annotations. Extend techniques from meta-learning and multi-task learning to support personalized or device-specific modeling in FLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib140" title="" class="ltx_ref">140</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I8.i8" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i8.p1" class="ltx_para">
<p id="S5.I8.i8.p1.1" class="ltx_p"><span id="S5.I8.i8.p1.1.1" class="ltx_text ltx_font_italic">Comprehensive Heterogeneity Handling:</span> Address the complex and multifaceted nature of data skew and heterogeneity more comprehensively. Real-world scenarios often involve intricate combinations of quantity, label, feature, and quality skew, along with system and model heterogeneity. Developing more robust and flexible methods to handle these diverse challenges will be crucial for improving FLâ€™s effectiveness in healthcare.</p>
</div>
</li>
<li id="S5.I8.i9" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I8.i9.p1" class="ltx_para">
<p id="S5.I8.i9.p1.1" class="ltx_p"><span id="S5.I8.i9.p1.1.1" class="ltx_text ltx_font_italic">Adaptive FL Frameworks:</span> Future research should focus on developing more adaptive FL frameworks capable of accommodating a wide range of model architectures and system configurations.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS4.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.4.2 </span>Open Domain Problem</h4>

<div id="S5.SS4.SSS2.p1" class="ltx_para">
<p id="S5.SS4.SSS2.p1.1" class="ltx_p">A key challenge in healthcare FL is the poor generalization of models to open domains, where unseen data lies beyond the federationâ€™s scope. A mere 14% of the included studies have validated their methods on such external data, underscoring a significant research gap. Current FL strategies predominantly focus on boosting performance within the federation, frequently overlooking the essential need for model adaptability to new, unseen environments.</p>
</div>
<div id="S5.SS4.SSS2.p2" class="ltx_para">
<p id="S5.SS4.SSS2.p2.1" class="ltx_p">Studies have shown that even slight differences in devices or acquisition protocols can result in a significant distribution shift, thereby reducing the modelâ€™s effectiveness when applied to new, unseen datasets. This issue is particularly acute in healthcare applications like diabetic retinopathy screening in fundus images, where the diversity of cameras and settings across different institutions can lead to poor model performance on external data. <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS4.SSS2.p2.1.1.1" class="ltx_sup"><a href="#bib.bib31" title="" class="ltx_ref">31</a></sup></cite> addressed this challenge by introducing a frequency-based domain generalization approach in FL. They enabled privacy preserving exchange of distribution information across clients through continuous frequency space interpolation and designed a boundary-oriented episodic learning scheme to expose local training to domain shifts and enhance model generalizability in ambiguous boundary regions.
However, the proposed method can be impractical for real-world applications due to its reliance on extensive network bandwidth and computational resources required for Fourier transform computation in frequency space interpolation.</p>
</div>
<section id="S5.SS4.SSS2.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS4.SSS2.Px1.p1" class="ltx_para">
<ul id="S5.I9" class="ltx_itemize">
<li id="S5.I9.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I9.i1.p1" class="ltx_para">
<p id="S5.I9.i1.p1.1" class="ltx_p"><span id="S5.I9.i1.p1.1.1" class="ltx_text ltx_font_italic">Domain Generalization Techniques:</span> These techniques aim to create models that are robust to distribution shifts by learning domain-invariant features. Methods such as data augmentation, adversarial training, and meta-learning have been explored to improve the generalization capabilities of FL models to unseen domains.</p>
</div>
</li>
<li id="S5.I9.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I9.i2.p1" class="ltx_para">
<p id="S5.I9.i2.p1.1" class="ltx_p"><span id="S5.I9.i2.p1.1.1" class="ltx_text ltx_font_italic">Transfer Learning and Fine-Tuning:</span> Apply transfer learning and fine-tuning on small amounts of unlabeled data from the open domain can help adapt the federated model to new environmentsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib77" title="" class="ltx_ref">77</a></sup></cite>. Self-supervised learning techniques like contrastive learning can further facilitate this processÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib37" title="" class="ltx_ref">37</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I9.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I9.i3.p1" class="ltx_para">
<p id="S5.I9.i3.p1.1" class="ltx_p"><span id="S5.I9.i3.p1.1.1" class="ltx_text ltx_font_italic">Ensemble Learning:</span> Combine multiple models trained on different clients via sophisticated ensemble methods can enhance the robustness of the final prediction, making it more generalizable to open domainsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib151" title="" class="ltx_ref">151</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I9.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I9.i4.p1" class="ltx_para">
<p id="S5.I9.i4.p1.1" class="ltx_p"><span id="S5.I9.i4.p1.1.1" class="ltx_text ltx_font_italic">Data Synthesis and Simulation:</span> Generate synthetic data that mimics the characteristics of potential open domains can be used to pretrain or fine-tune the federated model, improving its generalization to unseen environmentsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib31" title="" class="ltx_ref">31</a></sup></cite>.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS4.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.4.3 </span>Burdens in Communication</h4>

<div id="S5.SS4.SSS3.p1" class="ltx_para">
<p id="S5.SS4.SSS3.p1.1" class="ltx_p">Communication is a significant bottleneck in the implementation of FL in healthcare. In FL, each client needs to frequently communicate with the central server. This communication can be orders of magnitude slower than local computation due to constraints such as bandwidth, latency, and powerÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib95" title="" class="ltx_ref">95</a></sup></cite>.</p>
</div>
<div id="S5.SS4.SSS3.p2" class="ltx_para">
<p id="S5.SS4.SSS3.p2.1" class="ltx_p">The communication bottleneck in FL arises from several factors.
<span id="S5.SS4.SSS3.p2.1.1" class="ltx_text ltx_font_italic">First</span>, the number of clients involved in an FL system can be very large (e.g., wearables and IoMT), leading to significant communication overhead. Each communication round requires sending model updates between the clients and the central server, which can be expensive in terms of time and resourcesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib95" title="" class="ltx_ref">95</a></sup>, <sup class="ltx_sup"><a href="#bib.bib4" title="" class="ltx_ref">4</a></sup></cite>.
<span id="S5.SS4.SSS3.p2.1.2" class="ltx_text ltx_font_italic">Second</span>, ensuring data privacy and security in FL is crucial, especially in sensitive domains like healthcare. The need for encryption and secure communication protocols adds to the computational and communication overhead. Encrypting model updates can significantly increase the size of the data being transmitted, further straining the communication channels and requiring more sophisticated algorithms to balance privacy and efficiencyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib4" title="" class="ltx_ref">4</a></sup></cite>.
<span id="S5.SS4.SSS3.p2.1.3" class="ltx_text ltx_font_italic">Third</span>, as FL tasks become more complex, the size of the models involved increases. Modern large-scale models, such as large language models (LLMs) and foundation models (FMs), can have billions of parameters, resulting in model sizes that require significant bandwidth to transmit. This issue is exacerbated when using standard communication protocols like gRPC, which have size limits on single messages (e.g., 2 GB). Typical LLMs and FMs can exceed these limits, necessitating the model to be split into smaller chunks for transmission, adding additional overhead and complexity to the systemÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib158" title="" class="ltx_ref">158</a></sup></cite>.</p>
</div>
<div id="S5.SS4.SSS3.p3" class="ltx_para">
<p id="S5.SS4.SSS3.p3.1" class="ltx_p">Included studies primarily concentrated on enhancing communication encryption techniques, with the aim of either reducing the volume of data exchangedÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib63" title="" class="ltx_ref">63</a></sup>, <sup class="ltx_sup"><a href="#bib.bib104" title="" class="ltx_ref">104</a></sup></cite> or minimizing the number of communication roundsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib54" title="" class="ltx_ref">54</a></sup></cite>.
Additionally, several studies explored methods for achieving fully decentralized communication without central serverÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib55" title="" class="ltx_ref">55</a></sup>, <sup class="ltx_sup"><a href="#bib.bib159" title="" class="ltx_ref">159</a></sup></cite>.
Improvements in encryption often involved methods for securely sharing secret keys among clientsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib83" title="" class="ltx_ref">83</a></sup>, <sup class="ltx_sup"><a href="#bib.bib81" title="" class="ltx_ref">81</a></sup></cite>, encryption mechanisms for safeguarding exchanged dataÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib82" title="" class="ltx_ref">82</a></sup>, <sup class="ltx_sup"><a href="#bib.bib64" title="" class="ltx_ref">64</a></sup></cite>, and techniques for perturbing model outputs at each client using a secret keyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib160" title="" class="ltx_ref">160</a></sup></cite>.
To decrease the amount of data transmitted, some studies proposed transferring only a subset of model parametersÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib141" title="" class="ltx_ref">141</a></sup>, <sup class="ltx_sup"><a href="#bib.bib161" title="" class="ltx_ref">161</a></sup>, <sup class="ltx_sup"><a href="#bib.bib95" title="" class="ltx_ref">95</a></sup></cite> or employing strategies like compressingÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib97" title="" class="ltx_ref">97</a></sup></cite>, maskingÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib61" title="" class="ltx_ref">61</a></sup></cite>, and quantizing gradientsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib95" title="" class="ltx_ref">95</a></sup></cite> or model outputs before exchangeÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib27" title="" class="ltx_ref">27</a></sup></cite>.
Reducing the number of communication rounds was addressed through model designÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib91" title="" class="ltx_ref">91</a></sup>, <sup class="ltx_sup"><a href="#bib.bib162" title="" class="ltx_ref">162</a></sup>, <sup class="ltx_sup"><a href="#bib.bib98" title="" class="ltx_ref">98</a></sup></cite>, aggregating updates based on elapsed time instead of epochsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib97" title="" class="ltx_ref">97</a></sup>, <sup class="ltx_sup"><a href="#bib.bib107" title="" class="ltx_ref">107</a></sup></cite>, and evaluating the potential benefit of an update before communicationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib81" title="" class="ltx_ref">81</a></sup></cite>.
Other studies focused on detecting attacks during communicationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib163" title="" class="ltx_ref">163</a></sup></cite>, developing authentication systems for clientsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib46" title="" class="ltx_ref">46</a></sup></cite>, and improving client management systemsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib45" title="" class="ltx_ref">45</a></sup>, <sup class="ltx_sup"><a href="#bib.bib64" title="" class="ltx_ref">64</a></sup></cite>.</p>
</div>
<section id="S5.SS4.SSS3.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS4.SSS3.Px1.p1" class="ltx_para">
<ul id="S5.I10" class="ltx_itemize">
<li id="S5.I10.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I10.i1.p1" class="ltx_para">
<p id="S5.I10.i1.p1.1" class="ltx_p"><span id="S5.I10.i1.p1.1.1" class="ltx_text ltx_font_italic">Model Optimization:</span> Utilize parameter-efficient, lightweight models such as MobileNet and SqueezeNet to reduce computation and memory overhead. Explore model pruningÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib164" title="" class="ltx_ref">164</a></sup></cite> and low-rank adaptation (LoRA)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib165" title="" class="ltx_ref">165</a></sup></cite> techniques to decrease model size or regularize model weights without significantly impacting accuracy. Consider split learningÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib101" title="" class="ltx_ref">101</a></sup>, <sup class="ltx_sup"><a href="#bib.bib25" title="" class="ltx_ref">25</a></sup></cite> to share intermediate activations or a subset of the model instead of the full model, thus reducing communication costs.</p>
</div>
</li>
<li id="S5.I10.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I10.i2.p1" class="ltx_para">
<p id="S5.I10.i2.p1.1" class="ltx_p"><span id="S5.I10.i2.p1.1.1" class="ltx_text ltx_font_italic">Gradient Compression:</span> Implement gradient compression methods, including sparsification and quantization, to reduce the size of model updatesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib18" title="" class="ltx_ref">18</a></sup></cite>. Integrate AI-driven compression techniques to dynamically adjust quantization and sparsification levels based on model performance and network conditions.</p>
</div>
</li>
<li id="S5.I10.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I10.i3.p1" class="ltx_para">
<p id="S5.I10.i3.p1.1" class="ltx_p"><span id="S5.I10.i3.p1.1.1" class="ltx_text ltx_font_italic">Reduce Communication Rounds:</span> Minimize the number of communication rounds by adopting methods such as One-Shot FLÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib166" title="" class="ltx_ref">166</a></sup></cite>, which requires only a single round of communication to achieve effective model training.</p>
</div>
</li>
<li id="S5.I10.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I10.i4.p1" class="ltx_para">
<p id="S5.I10.i4.p1.1" class="ltx_p"><span id="S5.I10.i4.p1.1.1" class="ltx_text ltx_font_italic">Adopt Decentralized Approaches:</span> Explore decentralized training methods to eliminate the need for a central server, thereby alleviating communication bottlenecks and improving scalability.</p>
</div>
</li>
<li id="S5.I10.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I10.i5.p1" class="ltx_para">
<p id="S5.I10.i5.p1.1" class="ltx_p"><span id="S5.I10.i5.p1.1.1" class="ltx_text ltx_font_italic">Knowledge Distillation:</span> Use knowledge distillation techniques to transmit only essential distilled knowledge, such as logits from the final layer, rather than the entire modelÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib152" title="" class="ltx_ref">152</a></sup>, <sup class="ltx_sup"><a href="#bib.bib166" title="" class="ltx_ref">166</a></sup>, <sup class="ltx_sup"><a href="#bib.bib50" title="" class="ltx_ref">50</a></sup></cite>. This approach can reduce communication overhead while maintaining model performance.</p>
</div>
</li>
<li id="S5.I10.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I10.i6.p1" class="ltx_para">
<p id="S5.I10.i6.p1.1" class="ltx_p"><span id="S5.I10.i6.p1.1.1" class="ltx_text ltx_font_italic">Data Distillation:</span>
Apply data distillation to generate synthetic data summaries or distilled representations of the original datasets or updates. Clients can then share smaller, more concise updates with the central server, enhancing communication efficiencyÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib167" title="" class="ltx_ref">167</a></sup>, <sup class="ltx_sup"><a href="#bib.bib168" title="" class="ltx_ref">168</a></sup></cite>.</p>
</div>
</li>
<li id="S5.I10.i7" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I10.i7.p1" class="ltx_para">
<p id="S5.I10.i7.p1.1" class="ltx_p"><span id="S5.I10.i7.p1.1.1" class="ltx_text ltx_font_italic">Develop Specialized Protocols:</span>
Design and implement lightweight communication protocols tailored for FL, focusing on bandwidth efficiency and robustness against network instability.</p>
</div>
</li>
<li id="S5.I10.i8" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I10.i8.p1" class="ltx_para">
<p id="S5.I10.i8.p1.1" class="ltx_p"><span id="S5.I10.i8.p1.1.1" class="ltx_text ltx_font_italic">Design Scalable and Flexible Framework:</span> Develop FL frameworks that adapt to various communication environments and model complexities. Incorporate dynamic communication schedules to optimize efficiency and performance.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS4.SSS4" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.4.4 </span>Plain Convergence Analysis</h4>

<div id="S5.SS4.SSS4.p1" class="ltx_para">
<p id="S5.SS4.SSS4.p1.1" class="ltx_p">Federated optimization in healthcare aims to adapt models to local data distributions while integrating global information.
The inherent heterogeneity across hospitals and institutions often leads to instability and slow convergence in federated trainingÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib154" title="" class="ltx_ref">154</a></sup></cite>.
However, comprehensive convergence analysis is frequently lacking in current studies.</p>
</div>
<div id="S5.SS4.SSS4.p2" class="ltx_para">
<p id="S5.SS4.SSS4.p2.1" class="ltx_p">Most studies tend to provide only plain convergence analysis, focusing on reporting metrics such as the number of local epochs, communication rounds, and overall convergence time.
While these metrics are useful, they do not offer a deep theoretical understanding of the convergence dynamics.
This lack of rigorous analysis limits our understanding of how and why certain FL algorithms perform well (or poorly) in specific healthcare applications.</p>
</div>
<div id="S5.SS4.SSS4.p3" class="ltx_para">
<p id="S5.SS4.SSS4.p3.1" class="ltx_p">Only a handful of studiesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib159" title="" class="ltx_ref">159</a></sup>, <sup class="ltx_sup"><a href="#bib.bib61" title="" class="ltx_ref">61</a></sup>, <sup class="ltx_sup"><a href="#bib.bib95" title="" class="ltx_ref">95</a></sup>, <sup class="ltx_sup"><a href="#bib.bib104" title="" class="ltx_ref">104</a></sup></cite> have focused on the convergence of FL in healthcare settings.
These studies typically relied on Stochastic Gradient Descent (SGD) as the foundational optimization method due to its effectiveness in smooth optimization problems, under assumptions such as the existence of lower bounds, Lipschitz smoothness, and bounded varianceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib159" title="" class="ltx_ref">159</a></sup>, <sup class="ltx_sup"><a href="#bib.bib61" title="" class="ltx_ref">61</a></sup></cite>.
However, SGD-based FL algorithms often struggle with nonsmooth optimization problems, which are common in healthcare data due to irregularities in data distributions and the presence of outliers.</p>
</div>
<section id="S5.SS4.SSS4.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS4.SSS4.Px1.p1" class="ltx_para">
<ul id="S5.I11" class="ltx_itemize">
<li id="S5.I11.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I11.i1.p1" class="ltx_para">
<p id="S5.I11.i1.p1.1" class="ltx_p"><span id="S5.I11.i1.p1.1.1" class="ltx_text ltx_font_italic">Theoretical Convergence Guarantees:</span> Establish rigorous theoretical methods that provide convergence guarantees for FL in healthcare. This includes deriving bounds on convergence rates and understanding the conditions under which proposed algorithms perform optimally, particularly in non-IID data settings.</p>
</div>
</li>
<li id="S5.I11.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I11.i2.p1" class="ltx_para">
<p id="S5.I11.i2.p1.1" class="ltx_p"><span id="S5.I11.i2.p1.1.1" class="ltx_text ltx_font_italic">Real-World Validation:</span> Prioritize validating convergence analysis and algorithmic improvements on real-world healthcare datasets with natural split rather than relying solely on synthetic partitions. This will ensure that the proposed methods are practical and effective in real healthcare environments.</p>
</div>
</li>
<li id="S5.I11.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I11.i3.p1" class="ltx_para">
<p id="S5.I11.i3.p1.1" class="ltx_p"><span id="S5.I11.i3.p1.1.1" class="ltx_text ltx_font_italic">Advanced Optimization:</span> Develop optimization algorithms tailored to the specific challenges of healthcare data, such as non-convexity, non-smoothness, and heterogeneity. Techniques like adaptive learning rates, momentum methods, and second-order methods could be explored to enhance convergence rates and stability in FL scenarios.</p>
</div>
</li>
<li id="S5.I11.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I11.i4.p1" class="ltx_para">
<p id="S5.I11.i4.p1.1" class="ltx_p"><span id="S5.I11.i4.p1.1.1" class="ltx_text ltx_font_italic">Principled Communication Termination:</span> Implement principled methods for terminating communication rounds, potentially based on the performance of the global model at each client on a validation cohort or evaluation data held at the central aggregator. Early stopping based on local convergence could also be beneficial, as it would reduce unnecessary computation and communication costs.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS4.SSS5" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.4.5 </span>Temporal Dynamics and Revoke Issues</h4>

<div id="S5.SS4.SSS5.p1" class="ltx_para">
<p id="S5.SS4.SSS5.p1.1" class="ltx_p">Healthcare dataâ€™s inherent time dependence is critical, especially for diseases with distinct progression or treatment timelines, such as cancer and chronic conditions like diabetes. However, included studies often overlooked these temporal dynamics when partitioning data, potentially leading to models that inaccurately reflect disease progression.
For example, COVID-19 characteristics, such as ground-glass opacities in lung CT scans, evolve with the diseaseâ€™s progressionÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib169" title="" class="ltx_ref">169</a></sup></cite>. Ignoring such temporal dynamics can result in models that are overfitted to specific stages of a disease and unable to generalize across different phasesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib170" title="" class="ltx_ref">170</a></sup>, <sup class="ltx_sup"><a href="#bib.bib171" title="" class="ltx_ref">171</a></sup>, <sup class="ltx_sup"><a href="#bib.bib172" title="" class="ltx_ref">172</a></sup></cite>.
Furthermore, the dynamic nature of data at each participating hospital or institution complicates the situation. Hospitals continuously acquire new data and may also remove or modify existing data due to errors or other factors.
This dynamic data environment requires FL models to adapt without frequent retraining, as new data might cause model drift, while data removal can leave critical gaps in the modelâ€™s understanding, particularly if the removed data represents rare or critical cases.</p>
</div>
<div id="S5.SS4.SSS5.p2" class="ltx_para">
<p id="S5.SS4.SSS5.p2.1" class="ltx_p">Another critical but overlooked issue in FL is data revocation, which becomes necessary when specific data must be withdrawn due to privacy concerns, regulatory requirements, patient requests, or participant requests.
Current FL setups, designed for iterative data aggregation, struggle with â€œunlearningâ€ specific contributions without requiring complete model retraining.
Emerging research highlights the need for mechanisms that allow for efficient data revocation without compromising the integrity of the model. For instance, methods have been proposed to facilitate client â€œunlearningâ€ in FL, enabling the removal of data contributions from specific clients without significant degradation in model performanceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib173" title="" class="ltx_ref">173</a></sup></cite>. This is especially important in healthcare, where patient consent might be withdrawn, or new privacy laws might mandate the deletion of certain data.</p>
</div>
<section id="S5.SS4.SSS5.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS4.SSS5.Px1.p1" class="ltx_para">
<ul id="S5.I12" class="ltx_itemize">
<li id="S5.I12.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I12.i1.p1" class="ltx_para">
<p id="S5.I12.i1.p1.1" class="ltx_p"><span id="S5.I12.i1.p1.1.1" class="ltx_text ltx_font_italic">Incorporate Temporal Dynamics:</span> FL frameworks should be adapted to account for the temporal aspects of healthcare data. This could involve time-aware partitioning of data and the development of models that can learn from time-series data, ensuring better generalization across different stages of disease progression.</p>
</div>
</li>
<li id="S5.I12.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I12.i2.p1" class="ltx_para">
<p id="S5.I12.i2.p1.1" class="ltx_p"><span id="S5.I12.i2.p1.1.1" class="ltx_text ltx_font_italic">Support Dynamic Data Management:</span> FL models should include mechanisms for continuous learning to adapt to the addition and removal of data within the federation. Techniques like incremental learning or online learning could be employed to keep models updated with the latest data while minimizing the need for complete retraining.</p>
</div>
</li>
<li id="S5.I12.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I12.i3.p1" class="ltx_para">
<p id="S5.I12.i3.p1.1" class="ltx_p"><span id="S5.I12.i3.p1.1.1" class="ltx_text ltx_font_italic">Data Revocation:</span> Develop and integrate efficient data revocation techniques in FL frameworks. Approaches such as machine unlearningÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib173" title="" class="ltx_ref">173</a></sup></cite> can be refined to allow the removal of specific data contributions while minimizing the impact on overall model performance. This will be critical for maintaining compliance with privacy laws and upholding patient rights in healthcare applications.</p>
</div>
</li>
<li id="S5.I12.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I12.i4.p1" class="ltx_para">
<p id="S5.I12.i4.p1.1" class="ltx_p"><span id="S5.I12.i4.p1.1.1" class="ltx_text ltx_font_italic">Active Learning Integration:</span> Incorporate active learning strategies into FL to selectively query the most informative data points during the training process. This would help in focusing on critical data that improves model performance over time, especially in cases where temporal dynamics are at play.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS4.SSS6" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.4.6 </span>Synchronization Issues</h4>

<div id="S5.SS4.SSS6.p1" class="ltx_para">
<p id="S5.SS4.SSS6.p1.1" class="ltx_p">Synchronization of updates across different clients also poses significant challenges for FL in healthcare.
The variation in computational resources, network conditions, and data availability among clients can lead to different training speeds and delayed model updates.</p>
</div>
<div id="S5.SS4.SSS6.p2" class="ltx_para">
<p id="S5.SS4.SSS6.p2.1" class="ltx_p">In <span id="S5.SS4.SSS6.p2.1.1" class="ltx_text ltx_font_italic">Synchronous FL</span>, all clients must complete their training and send their updates before the global model aggregation, which is straightforward but can be inefficient. This approach works well in environments where data is immediately available, such as in a centralized hospital picture archiving and communication system (PACS).
However, in real-world scenarios, where data acquisition might be delayed due to network issues or the unavailability of input/output devices, synchronous updates can result in significant idle times and resource underutilization. Additionally, clients in an FL network, particularly smaller healthcare entities (e.g., wearables), may not be active during every communication round, further delaying the global model update and potentially degrading overall system performance.</p>
</div>
<div id="S5.SS4.SSS6.p3" class="ltx_para">
<p id="S5.SS4.SSS6.p3.1" class="ltx_p"><span id="S5.SS4.SSS6.p3.1.1" class="ltx_text ltx_font_italic">Asynchronous FL</span>, on the other hand, allows clients to send updates independently, without waiting for other clients to finish their training. This approach is more flexible and can accommodate variations in client availability and computational power, thereby improving the overall efficiency of the FL process. However, asynchronous updates can introduce challenges related to the consistency of the global model, as updates from slower or less reliable clients may arrive out of sync with the rest of the system.</p>
</div>
<section id="S5.SS4.SSS6.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS4.SSS6.Px1.p1" class="ltx_para">
<ul id="S5.I13" class="ltx_itemize">
<li id="S5.I13.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I13.i1.p1" class="ltx_para">
<p id="S5.I13.i1.p1.1" class="ltx_p"><span id="S5.I13.i1.p1.1.1" class="ltx_text ltx_font_italic">Hybrid Synchronization:</span> Investigate hybrid synchronization methods that combine the benefits of synchronous and asynchronous updates. For example, employing synchronous updates for critical clients with high-quality data and asynchronous updates for less reliable or slower clients could balance consistency and efficiency.</p>
</div>
</li>
<li id="S5.I13.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I13.i2.p1" class="ltx_para">
<p id="S5.I13.i2.p1.1" class="ltx_p"><span id="S5.I13.i2.p1.1.1" class="ltx_text ltx_font_italic">Consistency and Anomaly Check:</span> When employing asynchronous updates, it is crucial to assess the consistency and anomaly of updates received from different clients. Divergent updates, where one clientâ€™s model update significantly differs from others, could indicate issues such as heterogeneity or anomalous behaviour in the training process. These discrepancies need to be carefully managed to prevent the global model from diverging.</p>
</div>
</li>
<li id="S5.I13.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I13.i3.p1" class="ltx_para">
<p id="S5.I13.i3.p1.1" class="ltx_p"><span id="S5.I13.i3.p1.1.1" class="ltx_text ltx_font_italic">Asynchronous FL for Wearables and IoMT Devices:</span> Asynchronous FL is particularly suitable for wearables and IoMT, where devices may have intermittent connectivity. Leveraging the Async-FL paradigm can pave the way for implementing the next generation of smart and remote healthcare monitoring systems at a mass scale.</p>
</div>
</li>
</ul>
</div>
<figure id="S5.T3" class="ltx_table">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 3: </span>Summary of privacy and security attacks in FL applied to healthcare.</figcaption>
<div id="S5.T3.6" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:7.4pt;height:142.3pt;vertical-align:-0.0pt;"><span class="ltx_transformed_inner" style="transform:translate(-261.3pt,5035.2pt) scale(0.0139302212500939,0.0139302212500939) ;">
<table id="S5.T3.6.6" class="ltx_tabular ltx_align_middle">
<tr id="S5.T3.6.6.6" class="ltx_tr">
<td id="S5.T3.6.6.6.7" class="ltx_td ltx_align_center ltx_border_r ltx_border_tt">Attacks</td>
<td id="S5.T3.1.1.1.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_tt" style="width:170.7pt;"><span id="S5.T3.1.1.1.1.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T3.1.1.1.1.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T3.1.1.1.1.1.p1.1" class="ltx_p"><span id="S5.T3.1.1.1.1.1.p1.1.1" class="ltx_text"></span> <span id="S5.T3.1.1.1.1.1.p1.1.2" class="ltx_text">
<span id="S5.T3.1.1.1.1.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.1.1.1.1.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T3.1.1.1.1.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Description</span></span>
</span></span><span id="S5.T3.1.1.1.1.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S5.T3.2.2.2.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_tt" style="width:170.7pt;"><span id="S5.T3.2.2.2.2.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T3.2.2.2.2.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T3.2.2.2.2.1.p1.1" class="ltx_p"><span id="S5.T3.2.2.2.2.1.p1.1.1" class="ltx_text"></span> <span id="S5.T3.2.2.2.2.1.p1.1.2" class="ltx_text">
<span id="S5.T3.2.2.2.2.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.2.2.2.2.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T3.2.2.2.2.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Risks</span></span>
</span></span><span id="S5.T3.2.2.2.2.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S5.T3.3.3.3.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_tt" style="width:45.5pt;"><span id="S5.T3.3.3.3.3.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T3.3.3.3.3.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T3.3.3.3.3.1.p1.1" class="ltx_p"><span id="S5.T3.3.3.3.3.1.p1.1.1" class="ltx_text"></span> <span id="S5.T3.3.3.3.3.1.p1.1.2" class="ltx_text">
<span id="S5.T3.3.3.3.3.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.3.3.3.3.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T3.3.3.3.3.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Attack</span></span>
<span id="S5.T3.3.3.3.3.1.p1.1.2.1.2" class="ltx_tr">
<span id="S5.T3.3.3.3.3.1.p1.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Difficulty</span></span>
</span></span><span id="S5.T3.3.3.3.3.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S5.T3.4.4.4.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_tt" style="width:85.4pt;"><span id="S5.T3.4.4.4.4.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T3.4.4.4.4.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T3.4.4.4.4.1.p1.1" class="ltx_p"><span id="S5.T3.4.4.4.4.1.p1.1.1" class="ltx_text"></span> <span id="S5.T3.4.4.4.4.1.p1.1.2" class="ltx_text">
<span id="S5.T3.4.4.4.4.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.4.4.4.4.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T3.4.4.4.4.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Impact</span></span>
<span id="S5.T3.4.4.4.4.1.p1.1.2.1.2" class="ltx_tr">
<span id="S5.T3.4.4.4.4.1.p1.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Scope</span></span>
</span></span><span id="S5.T3.4.4.4.4.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S5.T3.5.5.5.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_tt" style="width:105.3pt;"><span id="S5.T3.5.5.5.5.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T3.5.5.5.5.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T3.5.5.5.5.1.p1.1" class="ltx_p"><span id="S5.T3.5.5.5.5.1.p1.1.1" class="ltx_text"></span> <span id="S5.T3.5.5.5.5.1.p1.1.2" class="ltx_text">
<span id="S5.T3.5.5.5.5.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.5.5.5.5.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T3.5.5.5.5.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Detection</span></span>
<span id="S5.T3.5.5.5.5.1.p1.1.2.1.2" class="ltx_tr">
<span id="S5.T3.5.5.5.5.1.p1.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Methods</span></span>
</span></span><span id="S5.T3.5.5.5.5.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S5.T3.6.6.6.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_tt" style="width:105.3pt;"><span id="S5.T3.6.6.6.6.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T3.6.6.6.6.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T3.6.6.6.6.1.p1.1" class="ltx_p"><span id="S5.T3.6.6.6.6.1.p1.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.6.6.1.p1.1.2" class="ltx_text">
<span id="S5.T3.6.6.6.6.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.6.6.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.6.6.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Defense</span></span>
<span id="S5.T3.6.6.6.6.1.p1.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.6.6.1.p1.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Strategies</span></span>
</span></span><span id="S5.T3.6.6.6.6.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
</tr>
<tr id="S5.T3.6.6.7" class="ltx_tr">
<td id="S5.T3.6.6.7.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T3.6.6.7.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.7.1.2" class="ltx_text">
<span id="S5.T3.6.6.7.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.7.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.7.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Model Inversion</span></span>
<span id="S5.T3.6.6.7.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.7.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib174" title="" class="ltx_ref">174</a></sup></cite></span></span>
</span></span><span id="S5.T3.6.6.7.1.3" class="ltx_text"></span></td>
<td id="S5.T3.6.6.7.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.7.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.7.2.1.1" class="ltx_p">Reconstruct the actual samples (e.g., patient medical images or genetic data) from the model or updates</span>
</span>
</td>
<td id="S5.T3.6.6.7.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.7.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.7.3.1.1" class="ltx_p">Potential leakage of original patient data</span>
</span>
</td>
<td id="S5.T3.6.6.7.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:45.5pt;">
<span id="S5.T3.6.6.7.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.7.4.1.1" class="ltx_p">High</span>
</span>
</td>
<td id="S5.T3.6.6.7.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:85.4pt;">
<span id="S5.T3.6.6.7.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.7.5.1.1" class="ltx_p">Data privacy, patient confidentiality</span>
</span>
</td>
<td id="S5.T3.6.6.7.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.7.6.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.7.6.1.1" class="ltx_p">Anomaly detection in model outputs &amp; updates</span>
</span>
</td>
<td id="S5.T3.6.6.7.7" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.7.7.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.7.7.1.1" class="ltx_p">DP</span>
</span>
</td>
</tr>
<tr id="S5.T3.6.6.8" class="ltx_tr">
<td id="S5.T3.6.6.8.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T3.6.6.8.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.8.1.2" class="ltx_text">
<span id="S5.T3.6.6.8.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.8.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.8.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Membership Inference</span></span>
<span id="S5.T3.6.6.8.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.8.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib175" title="" class="ltx_ref">175</a></sup></cite></span></span>
</span></span><span id="S5.T3.6.6.8.1.3" class="ltx_text"></span></td>
<td id="S5.T3.6.6.8.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.8.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.8.2.1.1" class="ltx_p">Determining if a specific patientâ€™s record was part of the modelâ€™s training set by analyzing its outputs</span>
</span>
</td>
<td id="S5.T3.6.6.8.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.8.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.8.3.1.1" class="ltx_p">Compromises patient confidentiality, leading to potential unauthorized access to sensitive health information</span>
</span>
</td>
<td id="S5.T3.6.6.8.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:45.5pt;">
<span id="S5.T3.6.6.8.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.8.4.1.1" class="ltx_p">Medium</span>
</span>
</td>
<td id="S5.T3.6.6.8.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:85.4pt;">
<span id="S5.T3.6.6.8.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.8.5.1.1" class="ltx_p">Data privacy, patient confidentiality</span>
</span>
</td>
<td id="S5.T3.6.6.8.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.8.6.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.8.6.1.1" class="ltx_p">Monitoring for unusual model behaviour, particularly overconfident predictions</span>
</span>
</td>
<td id="S5.T3.6.6.8.7" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.8.7.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.8.7.1.1" class="ltx_p">Regularization techniques, DP, and robust model validation</span>
</span>
</td>
</tr>
<tr id="S5.T3.6.6.9" class="ltx_tr">
<td id="S5.T3.6.6.9.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T3.6.6.9.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.9.1.2" class="ltx_text">
<span id="S5.T3.6.6.9.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.9.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.9.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Data Attribute Inference</span></span>
<span id="S5.T3.6.6.9.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.9.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib176" title="" class="ltx_ref">176</a></sup></cite></span></span>
</span></span><span id="S5.T3.6.6.9.1.3" class="ltx_text"></span></td>
<td id="S5.T3.6.6.9.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.9.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.9.2.1.1" class="ltx_p">Infer individual sampleâ€™s attributes (like race, gender, age) or gain aggregate statistical insights about the entire training set from model parameters and updates</span>
</span>
</td>
<td id="S5.T3.6.6.9.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.9.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.9.3.1.1" class="ltx_p">Leakage of patient privacy</span>
</span>
</td>
<td id="S5.T3.6.6.9.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:45.5pt;">
<span id="S5.T3.6.6.9.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.9.4.1.1" class="ltx_p">Medium</span>
</span>
</td>
<td id="S5.T3.6.6.9.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:85.4pt;">
<span id="S5.T3.6.6.9.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.9.5.1.1" class="ltx_p">Data privacy, patient confidentiality, communication security</span>
</span>
</td>
<td id="S5.T3.6.6.9.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.9.6.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.9.6.1.1" class="ltx_p">Gradient analysis, privacy audits</span>
</span>
</td>
<td id="S5.T3.6.6.9.7" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.9.7.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.9.7.1.1" class="ltx_p">Secure aggregation, DP</span>
</span>
</td>
</tr>
<tr id="S5.T3.6.6.10" class="ltx_tr">
<td id="S5.T3.6.6.10.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T3.6.6.10.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.10.1.2" class="ltx_text">
<span id="S5.T3.6.6.10.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.10.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.10.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Data Poisoning</span></span>
<span id="S5.T3.6.6.10.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.10.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib177" title="" class="ltx_ref">177</a></sup></cite></span></span>
</span></span><span id="S5.T3.6.6.10.1.3" class="ltx_text"></span></td>
<td id="S5.T3.6.6.10.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.10.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.10.2.1.1" class="ltx_p">Malicious participants alter local data or labels to degrade the global modelâ€™s performance</span>
</span>
</td>
<td id="S5.T3.6.6.10.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.10.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.10.3.1.1" class="ltx_p">Significant drop in model performance, potentially misleading patient diagnoses</span>
</span>
</td>
<td id="S5.T3.6.6.10.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:45.5pt;">
<span id="S5.T3.6.6.10.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.10.4.1.1" class="ltx_p">Medium</span>
</span>
</td>
<td id="S5.T3.6.6.10.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:85.4pt;">
<span id="S5.T3.6.6.10.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.10.5.1.1" class="ltx_p">Model accuracy, patient safety</span>
</span>
</td>
<td id="S5.T3.6.6.10.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.10.6.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.10.6.1.1" class="ltx_p">Monitoring for abnormal model behaviour, statistical checks</span>
</span>
</td>
<td id="S5.T3.6.6.10.7" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.10.7.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.10.7.1.1" class="ltx_p">Robust aggregation methods, anomaly detection in model updates</span>
</span>
</td>
</tr>
<tr id="S5.T3.6.6.11" class="ltx_tr">
<td id="S5.T3.6.6.11.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T3.6.6.11.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.11.1.2" class="ltx_text">
<span id="S5.T3.6.6.11.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.11.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.11.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Model Poisoning</span></span>
<span id="S5.T3.6.6.11.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.11.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib72" title="" class="ltx_ref">72</a></sup></cite></span></span>
</span></span><span id="S5.T3.6.6.11.1.3" class="ltx_text"></span></td>
<td id="S5.T3.6.6.11.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.11.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.11.2.1.1" class="ltx_p">Malicious participants upload tampered model parameters to manipulate the global model</span>
</span>
</td>
<td id="S5.T3.6.6.11.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.11.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.11.3.1.1" class="ltx_p">Global model may be intentionally corrupted, affecting its performance on real data</span>
</span>
</td>
<td id="S5.T3.6.6.11.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:45.5pt;">
<span id="S5.T3.6.6.11.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.11.4.1.1" class="ltx_p">High</span>
</span>
</td>
<td id="S5.T3.6.6.11.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:85.4pt;">
<span id="S5.T3.6.6.11.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.11.5.1.1" class="ltx_p">Model integrity, patient safety</span>
</span>
</td>
<td id="S5.T3.6.6.11.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.11.6.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.11.6.1.1" class="ltx_p">Anomaly detection in model updates</span>
</span>
</td>
<td id="S5.T3.6.6.11.7" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.11.7.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.11.7.1.1" class="ltx_p">Robust aggregation, DP</span>
</span>
</td>
</tr>
<tr id="S5.T3.6.6.12" class="ltx_tr">
<td id="S5.T3.6.6.12.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T3.6.6.12.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.12.1.2" class="ltx_text">
<span id="S5.T3.6.6.12.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.12.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.12.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Denial of Service</span></span>
<span id="S5.T3.6.6.12.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.12.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib73" title="" class="ltx_ref">73</a></sup></cite></span></span>
</span></span><span id="S5.T3.6.6.12.1.3" class="ltx_text"></span></td>
<td id="S5.T3.6.6.12.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.12.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.12.2.1.1" class="ltx_p">Disrupt the FL system by overwhelming it with requests or blocking normal data flow</span>
</span>
</td>
<td id="S5.T3.6.6.12.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.12.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.12.3.1.1" class="ltx_p">Training process delays or interruptions, affecting time-sensitive medical applications</span>
</span>
</td>
<td id="S5.T3.6.6.12.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:45.5pt;">
<span id="S5.T3.6.6.12.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.12.4.1.1" class="ltx_p">Medium</span>
</span>
</td>
<td id="S5.T3.6.6.12.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:85.4pt;">
<span id="S5.T3.6.6.12.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.12.5.1.1" class="ltx_p">System availability and security, training efficiency</span>
</span>
</td>
<td id="S5.T3.6.6.12.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.12.6.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.12.6.1.1" class="ltx_p">Monitoring network traffic for anomalies</span>
</span>
</td>
<td id="S5.T3.6.6.12.7" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.12.7.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.12.7.1.1" class="ltx_p">Rate limiting, robust network design, redundancy mechanisms</span>
</span>
</td>
</tr>
<tr id="S5.T3.6.6.13" class="ltx_tr">
<td id="S5.T3.6.6.13.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T3.6.6.13.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.13.1.2" class="ltx_text">
<span id="S5.T3.6.6.13.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.13.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.13.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">HE Attack</span></span>
<span id="S5.T3.6.6.13.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.13.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib64" title="" class="ltx_ref">64</a></sup></cite></span></span>
</span></span><span id="S5.T3.6.6.13.1.3" class="ltx_text"></span></td>
<td id="S5.T3.6.6.13.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.13.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.13.2.1.1" class="ltx_p">Decrypt encrypted model updates to access sensitive information</span>
</span>
</td>
<td id="S5.T3.6.6.13.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.13.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.13.3.1.1" class="ltx_p">Encryption key leakage may lead to a breakdown in data protection, compromising patient privacy</span>
</span>
</td>
<td id="S5.T3.6.6.13.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:45.5pt;">
<span id="S5.T3.6.6.13.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.13.4.1.1" class="ltx_p">High</span>
</span>
</td>
<td id="S5.T3.6.6.13.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:85.4pt;">
<span id="S5.T3.6.6.13.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.13.5.1.1" class="ltx_p">Data privacy, model integrity</span>
</span>
</td>
<td id="S5.T3.6.6.13.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.13.6.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.13.6.1.1" class="ltx_p">Encryption integrity checks, key management audits</span>
</span>
</td>
<td id="S5.T3.6.6.13.7" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.13.7.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.13.7.1.1" class="ltx_p">Secure key management, SMPC</span>
</span>
</td>
</tr>
<tr id="S5.T3.6.6.14" class="ltx_tr">
<td id="S5.T3.6.6.14.1" class="ltx_td ltx_align_center ltx_border_bb ltx_border_r ltx_border_t">
<span id="S5.T3.6.6.14.1.1" class="ltx_text"></span> <span id="S5.T3.6.6.14.1.2" class="ltx_text">
<span id="S5.T3.6.6.14.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T3.6.6.14.1.2.1.1" class="ltx_tr">
<span id="S5.T3.6.6.14.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">User Withdrawal Attack</span></span>
<span id="S5.T3.6.6.14.1.2.1.2" class="ltx_tr">
<span id="S5.T3.6.6.14.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center"><cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib44" title="" class="ltx_ref">44</a></sup></cite></span></span>
</span></span><span id="S5.T3.6.6.14.1.3" class="ltx_text"></span></td>
<td id="S5.T3.6.6.14.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.14.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.14.2.1.1" class="ltx_p">Participants withdraw from training, but their prior updates still affect the global model</span>
</span>
</td>
<td id="S5.T3.6.6.14.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_r ltx_border_t" style="width:170.7pt;">
<span id="S5.T3.6.6.14.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.14.3.1.1" class="ltx_p">Updates from withdrawn users may contain errors or malicious data, degrading the global model</span>
</span>
</td>
<td id="S5.T3.6.6.14.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_r ltx_border_t" style="width:45.5pt;">
<span id="S5.T3.6.6.14.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.14.4.1.1" class="ltx_p">Low</span>
</span>
</td>
<td id="S5.T3.6.6.14.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_r ltx_border_t" style="width:85.4pt;">
<span id="S5.T3.6.6.14.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.14.5.1.1" class="ltx_p">Model integrity</span>
</span>
</td>
<td id="S5.T3.6.6.14.6" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_r ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.14.6.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.14.6.1.1" class="ltx_p">Tracking user participation and contribution consistency</span>
</span>
</td>
<td id="S5.T3.6.6.14.7" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_t" style="width:105.3pt;">
<span id="S5.T3.6.6.14.7.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T3.6.6.14.7.1.1" class="ltx_p">Periodic re-evaluation of model contributions, machine unlearning</span>
</span>
</td>
</tr>
</table>
</span></div>
</figure>
</section>
</section>
</section>
<section id="S5.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.5 </span>Privacy and Security</h3>

<div id="S5.SS5.p1" class="ltx_para">
<p id="S5.SS5.p1.1" class="ltx_p">Two critical privacy and security issues exist in current studies. First, it is concerning that 62% of the included studies did not encrypt model updates during communication. This lack of encryption leaves FL system vulnerable to interception, posing significant security risks. Second, statistical information such as sample sizes and distributions were often shared alongside model updates, particularly in FedAvg-based methodsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib106" title="" class="ltx_ref">106</a></sup>, <sup class="ltx_sup"><a href="#bib.bib108" title="" class="ltx_ref">108</a></sup></cite>. This practice can expose participants with large datasets to targeted attacks if an adversary intercepts the communication or compromises the aggregatorÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib158" title="" class="ltx_ref">158</a></sup></cite>.</p>
</div>
<div id="S5.SS5.p2" class="ltx_para">
<p id="S5.SS5.p2.1" class="ltx_p">Despite the advantage of FL in healthcare without directly exchanging or sharing local data, it is not immune to privacy and security risks.
Adversaries can analyze changes in model updates over time to infer sensitive information or exploit system vulnerabilities to conduct targeted attacks using techniques like model inversionÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib178" title="" class="ltx_ref">178</a></sup></cite>, membership inferenceÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib179" title="" class="ltx_ref">179</a></sup></cite>, and poisoningÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib177" title="" class="ltx_ref">177</a></sup></cite>.
Additionally, clients may unintentionally reveal private data during the FL process. This can happen when the client memorizes previous model and gradient updates, leading to the leakage of sensitive informationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib65" title="" class="ltx_ref">65</a></sup>, <sup class="ltx_sup"><a href="#bib.bib16" title="" class="ltx_ref">16</a></sup></cite>. Furthermore, methods involving the sharing of a few data samples for augmentation or disclosing local data distributions during knowledge transfer can also result in privacy breachesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib99" title="" class="ltx_ref">99</a></sup>, <sup class="ltx_sup"><a href="#bib.bib50" title="" class="ltx_ref">50</a></sup></cite>.
These adversarial and unintentional exposures undermine the privacy and security of the FL process, necessitating robust countermeasures and continuous vigilance to safeguard the integrity and confidentiality of the FL process.</p>
</div>
<div id="S5.SS5.p3" class="ltx_para">
<p id="S5.SS5.p3.1" class="ltx_p">In TableÂ <a href="#S5.T3" title="Table 3 â€£ Recommendations &amp; Opportunities â€£ 5.4.6 Synchronization Issues â€£ 5.4 Optimization â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, we provide a comprehensive overview of various privacy and security attacks identified in the context of FL in healthcare.
By summarizing their key characteristics, specific risks they introduce, as well as potential defence strategies and other relevant factors. This table serves as a critical resource for understanding the complexities and vulnerabilities associated with FL in healthcare, offering insights into both the challenges and possible solutions for enhancing privacy and security in this domain.</p>
</div>
<div id="S5.SS5.p4" class="ltx_para">
<p id="S5.SS5.p4.1" class="ltx_p">Among all included studies, methods for data privacy and security protection generally fall into two broad categories, namely <span id="S5.SS5.p4.1.1" class="ltx_text ltx_font_italic">cryptography</span> and <span id="S5.SS5.p4.1.2" class="ltx_text ltx_font_italic">perturbation</span> techniques. Each of these methods has its advantages and shortcomings, as summarised in TableÂ <a href="#S5.T4" title="Table 4 â€£ 5.5 Privacy and Security â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>.</p>
</div>
<div id="S5.SS5.p5" class="ltx_para">
<p id="S5.SS5.p5.1" class="ltx_p"><span id="S5.SS5.p5.1.1" class="ltx_text ltx_font_italic">Cryptography</span> encompass a variety of methods, with homomorphic encryption (HE)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib180" title="" class="ltx_ref">180</a></sup></cite> and secure multi-party computation (SMPC)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib181" title="" class="ltx_ref">181</a></sup></cite> being among the most popular.
HE enables computations directly on encrypted data without the need for decryption. This ensures that data remains encrypted throughout processing, storage, and transmission, thus providing robust data security and privacy. HE allows operations on ciphertexts, with the results, once decrypted, accurately reflecting the outcomes of operations performed on the original plaintext data.
HE is particularly valuable in FL due to its ability to safeguard data privacy during computation. Recent research has highlighted HEâ€™s effectiveness in various healthcare applications, including oncology and medical genetics. For instance, <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS5.p5.1.2.1" class="ltx_sup"><a href="#bib.bib69" title="" class="ltx_ref">69</a></sup></cite> demonstrated HEâ€™s potential for truly private federated evaluations, and <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS5.p5.1.3.1" class="ltx_sup"><a href="#bib.bib77" title="" class="ltx_ref">77</a></sup></cite> successfully utilized HE for model aggregation in FL.
While SMPC enables multiple parties to collaboratively compute a function over their combined data while keeping each partyâ€™s data private. Each participant holds a piece of encrypted or encoded data, and the computation is designed so that no party can access the othersâ€™ data or infer anything beyond the final result. SMPC ensures the confidentiality of both input data and computation results, making it robust against adversarial attacks and suitable for scenarios with multiple untrusted parties. It is increasingly used in healthcare and other sensitive applications to enhance privacy. For instance, research shows that SMPC can improve privacy in FL with medical datasets by addressing risks related to malicious models and enhancing the confidentiality of model aggregationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib47" title="" class="ltx_ref">47</a></sup>, <sup class="ltx_sup"><a href="#bib.bib72" title="" class="ltx_ref">72</a></sup></cite>.</p>
</div>
<div id="S5.SS5.p6" class="ltx_para">
<p id="S5.SS5.p6.1" class="ltx_p">Nevertheless, HE typically involves high storage and computational overheads. Encrypted data requires significant processing power, and the complexity of HE schemes can introduce a single point of failure, where a single server manages all encrypted dataÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib69" title="" class="ltx_ref">69</a></sup></cite>. Additionally, managing encryption keys securely is crucial, as any compromise in key management can jeopardize the entire systemâ€™s securityÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib182" title="" class="ltx_ref">182</a></sup></cite>.
Also, SMPC often incurs high computational and communication overhead. The process of encrypting, encoding, and splitting data can be computationally intensive. Moreover, coordinating the computations across multiple parties requires substantial communication resources, which can become a bottleneck as the number of participants growsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib183" title="" class="ltx_ref">183</a></sup></cite>.</p>
</div>
<div id="S5.SS5.p7" class="ltx_para">
<p id="S5.SS5.p7.2" class="ltx_p"><span id="S5.SS5.p7.2.1" class="ltx_text ltx_font_italic">Perturbation</span> techniques, with differential privacy (DP)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib184" title="" class="ltx_ref">184</a></sup></cite> being the most prominent, are crucial for protecting sensitive healthcare data.
DP quantifies the risk of exposing individual data points and ensures that the inclusion or exclusion of any single data point has minimal impact on the modelâ€™s output by introducing randomness into the modelâ€™s results.
In FL, DP is typically implemented by adding noise to the local updates or gradients before they are aggregated. This noise is designed to mask the contributions of individual data points, thus making it difficult to infer any single data point from the modelâ€™s outputs.
DP provides a quantifiable measure of privacy through parameters such as <math id="S5.SS5.p7.1.m1.1" class="ltx_Math" alttext="\epsilon" display="inline"><semantics id="S5.SS5.p7.1.m1.1a"><mi id="S5.SS5.p7.1.m1.1.1" xref="S5.SS5.p7.1.m1.1.1.cmml">Ïµ</mi><annotation-xml encoding="MathML-Content" id="S5.SS5.p7.1.m1.1b"><ci id="S5.SS5.p7.1.m1.1.1.cmml" xref="S5.SS5.p7.1.m1.1.1">italic-Ïµ</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS5.p7.1.m1.1c">\epsilon</annotation></semantics></math> and <math id="S5.SS5.p7.2.m2.1" class="ltx_Math" alttext="\delta" display="inline"><semantics id="S5.SS5.p7.2.m2.1a"><mi id="S5.SS5.p7.2.m2.1.1" xref="S5.SS5.p7.2.m2.1.1.cmml">Î´</mi><annotation-xml encoding="MathML-Content" id="S5.SS5.p7.2.m2.1b"><ci id="S5.SS5.p7.2.m2.1.1.cmml" xref="S5.SS5.p7.2.m2.1.1">ğ›¿</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS5.p7.2.m2.1c">\delta</annotation></semantics></math>, allowing for a clear understanding of the trade-off between privacy and model utility.
Recent studies in healthcare have shown that FL with DP can achieve comparable accuracy to non-DP models in specific tasks, with a performance gap of less than 5%, proving DPâ€™s efficacy with minimal accuracy compromiseÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib106" title="" class="ltx_ref">106</a></sup>, <sup class="ltx_sup"><a href="#bib.bib108" title="" class="ltx_ref">108</a></sup>, <sup class="ltx_sup"><a href="#bib.bib158" title="" class="ltx_ref">158</a></sup></cite>.</p>
</div>
<div id="S5.SS5.p8" class="ltx_para">
<p id="S5.SS5.p8.3" class="ltx_p">However, the main challenge with DP is balancing the privacy budget with model performance. Adding noise typically reduces model accuracy by obscuring data patterns, which is a critical consideration in DP implementationÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib65" title="" class="ltx_ref">65</a></sup></cite>. Managing the privacy budget <math id="S5.SS5.p8.1.m1.1" class="ltx_Math" alttext="\epsilon" display="inline"><semantics id="S5.SS5.p8.1.m1.1a"><mi id="S5.SS5.p8.1.m1.1.1" xref="S5.SS5.p8.1.m1.1.1.cmml">Ïµ</mi><annotation-xml encoding="MathML-Content" id="S5.SS5.p8.1.m1.1b"><ci id="S5.SS5.p8.1.m1.1.1.cmml" xref="S5.SS5.p8.1.m1.1.1">italic-Ïµ</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS5.p8.1.m1.1c">\epsilon</annotation></semantics></math> involves a trade-off: a smaller <math id="S5.SS5.p8.2.m2.1" class="ltx_Math" alttext="\epsilon" display="inline"><semantics id="S5.SS5.p8.2.m2.1a"><mi id="S5.SS5.p8.2.m2.1.1" xref="S5.SS5.p8.2.m2.1.1.cmml">Ïµ</mi><annotation-xml encoding="MathML-Content" id="S5.SS5.p8.2.m2.1b"><ci id="S5.SS5.p8.2.m2.1.1.cmml" xref="S5.SS5.p8.2.m2.1.1">italic-Ïµ</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS5.p8.2.m2.1c">\epsilon</annotation></semantics></math> enhances privacy but may reduce performance, and a larger <math id="S5.SS5.p8.3.m3.1" class="ltx_Math" alttext="\epsilon" display="inline"><semantics id="S5.SS5.p8.3.m3.1a"><mi id="S5.SS5.p8.3.m3.1.1" xref="S5.SS5.p8.3.m3.1.1.cmml">Ïµ</mi><annotation-xml encoding="MathML-Content" id="S5.SS5.p8.3.m3.1b"><ci id="S5.SS5.p8.3.m3.1.1.cmml" xref="S5.SS5.p8.3.m3.1.1">italic-Ïµ</ci></annotation-xml><annotation encoding="application/x-tex" id="S5.SS5.p8.3.m3.1c">\epsilon</annotation></semantics></math> offers less privacy protection. Effective management of this trade-off is essential, requiring careful attention to privacy needs and performance goalsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib128" title="" class="ltx_ref">128</a></sup></cite>.
Implementing DP also introduces additional computational overhead due to the noise addition and gradient clipping processes. This overhead can affect the scalability and efficiency of FL systems, particularly in scenarios with large numbers of clients or dataÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib185" title="" class="ltx_ref">185</a></sup></cite>.</p>
</div>
<figure id="S5.T4" class="ltx_table">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 4: </span>Comparison of Cryptography, perturbation, and other techniques used for privacy and security protection in FL for healthcare.</figcaption>
<div id="S5.T4.3" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:29.1pt;height:153.6pt;vertical-align:-0.0pt;"><span class="ltx_transformed_inner" style="transform:translate(-555.0pt,2931.3pt) scale(0.0255387325226728,0.0255387325226728) ;">
<table id="S5.T4.3.3" class="ltx_tabular ltx_align_middle">
<tr id="S5.T4.3.3.3" class="ltx_tr">
<td id="S5.T4.3.3.3.4" class="ltx_td ltx_align_center ltx_border_r ltx_border_tt">
<span id="S5.T4.3.3.3.4.1" class="ltx_text"></span> <span id="S5.T4.3.3.3.4.2" class="ltx_text">
<span id="S5.T4.3.3.3.4.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.3.4.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.3.4.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Category</span></span>
</span></span><span id="S5.T4.3.3.3.4.3" class="ltx_text"></span></td>
<td id="S5.T4.3.3.3.5" class="ltx_td ltx_align_center ltx_border_r ltx_border_tt">
<span id="S5.T4.3.3.3.5.1" class="ltx_text"></span> <span id="S5.T4.3.3.3.5.2" class="ltx_text">
<span id="S5.T4.3.3.3.5.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.3.5.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.3.5.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Technique</span></span>
</span></span><span id="S5.T4.3.3.3.5.3" class="ltx_text"></span></td>
<td id="S5.T4.1.1.1.1" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_tt" style="width:199.2pt;"><span id="S5.T4.1.1.1.1.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T4.1.1.1.1.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T4.1.1.1.1.1.p1.1" class="ltx_p"><span id="S5.T4.1.1.1.1.1.p1.1.1" class="ltx_text"></span> <span id="S5.T4.1.1.1.1.1.p1.1.2" class="ltx_text">
<span id="S5.T4.1.1.1.1.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.1.1.1.1.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T4.1.1.1.1.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Description</span></span>
</span></span><span id="S5.T4.1.1.1.1.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S5.T4.2.2.2.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_tt" style="width:284.5pt;"><span id="S5.T4.2.2.2.2.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T4.2.2.2.2.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T4.2.2.2.2.1.p1.1" class="ltx_p"><span id="S5.T4.2.2.2.2.1.p1.1.1" class="ltx_text"></span> <span id="S5.T4.2.2.2.2.1.p1.1.2" class="ltx_text">
<span id="S5.T4.2.2.2.2.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.2.2.2.2.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T4.2.2.2.2.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Advantages</span></span>
</span></span><span id="S5.T4.2.2.2.2.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
<td id="S5.T4.3.3.3.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_tt" style="width:284.5pt;"><span id="S5.T4.3.3.3.3.1" class="ltx_inline-logical-block ltx_align_top">
<span id="S5.T4.3.3.3.3.1.p1" class="ltx_para ltx_noindent">
<span id="S5.T4.3.3.3.3.1.p1.1" class="ltx_p"><span id="S5.T4.3.3.3.3.1.p1.1.1" class="ltx_text"></span> <span id="S5.T4.3.3.3.3.1.p1.1.2" class="ltx_text">
<span id="S5.T4.3.3.3.3.1.p1.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.3.3.1.p1.1.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.3.3.1.p1.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Limitations</span></span>
</span></span><span id="S5.T4.3.3.3.3.1.p1.1.3" class="ltx_text"></span></span>
</span></span></td>
</tr>
<tr id="S5.T4.3.3.4" class="ltx_tr">
<td id="S5.T4.3.3.4.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t" rowspan="2"><span id="S5.T4.3.3.4.1.1" class="ltx_text">Cryptography</span></td>
<td id="S5.T4.3.3.4.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T4.3.3.4.2.1" class="ltx_text"></span> <span id="S5.T4.3.3.4.2.2" class="ltx_text">
<span id="S5.T4.3.3.4.2.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.4.2.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.4.2.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Homomorphic</span></span>
<span id="S5.T4.3.3.4.2.2.1.2" class="ltx_tr">
<span id="S5.T4.3.3.4.2.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Encryption</span></span>
</span></span><span id="S5.T4.3.3.4.2.3" class="ltx_text"></span></td>
<td id="S5.T4.3.3.4.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:199.2pt;">
<span id="S5.T4.3.3.4.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T4.3.3.4.3.1.1" class="ltx_p">Enable computations on encrypted data without decryption, maintaining data encryption throughout</span>
</span>
</td>
<td id="S5.T4.3.3.4.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.4.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I14" class="ltx_itemize">
<span id="S5.I14.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I14.i1.p1" class="ltx_para">
<span id="S5.I14.i1.p1.1" class="ltx_p">Provide strong data security and privacy</span>
</span></span>
<span id="S5.I14.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I14.i2.p1" class="ltx_para">
<span id="S5.I14.i2.p1.1" class="ltx_p">Useful for various healthcare applications</span>
</span></span>
<span id="S5.I14.i3" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I14.i3.p1" class="ltx_para">
<span id="S5.I14.i3.p1.1" class="ltx_p">Effective in federated evaluations and model aggregation</span>
</span></span>
</span>
</span>
</td>
<td id="S5.T4.3.3.4.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.4.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I15" class="ltx_itemize">
<span id="S5.I15.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I15.i1.p1" class="ltx_para">
<span id="S5.I15.i1.p1.1" class="ltx_p">High computational and storage overhead</span>
</span></span>
<span id="S5.I15.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I15.i2.p1" class="ltx_para">
<span id="S5.I15.i2.p1.1" class="ltx_p">Complex schemes may introduce single points of failure</span>
</span></span>
<span id="S5.I15.i3" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I15.i3.p1" class="ltx_para">
<span id="S5.I15.i3.p1.1" class="ltx_p">Key management challenges</span>
</span></span>
</span>
</span>
</td>
</tr>
<tr id="S5.T4.3.3.5" class="ltx_tr">
<td id="S5.T4.3.3.5.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T4.3.3.5.1.1" class="ltx_text"></span> <span id="S5.T4.3.3.5.1.2" class="ltx_text">
<span id="S5.T4.3.3.5.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.5.1.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.5.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Secure</span></span>
<span id="S5.T4.3.3.5.1.2.1.2" class="ltx_tr">
<span id="S5.T4.3.3.5.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Multi-Party</span></span>
<span id="S5.T4.3.3.5.1.2.1.3" class="ltx_tr">
<span id="S5.T4.3.3.5.1.2.1.3.1" class="ltx_td ltx_nopad_r ltx_align_center">Computation</span></span>
</span></span><span id="S5.T4.3.3.5.1.3" class="ltx_text"></span></td>
<td id="S5.T4.3.3.5.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:199.2pt;">
<span id="S5.T4.3.3.5.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T4.3.3.5.2.1.1" class="ltx_p">Allow multiple parties to collaboratively compute a function on their combined data while keeping data private</span>
</span>
</td>
<td id="S5.T4.3.3.5.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.5.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I16" class="ltx_itemize">
<span id="S5.I16.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I16.i1.p1" class="ltx_para">
<span id="S5.I16.i1.p1.1" class="ltx_p">Ensure confidentiality of input data and computation results</span>
</span></span>
<span id="S5.I16.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I16.i2.p1" class="ltx_para">
<span id="S5.I16.i2.p1.1" class="ltx_p">Robust against adversarial attacks</span>
</span></span>
<span id="S5.I16.i3" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I16.i3.p1" class="ltx_para">
<span id="S5.I16.i3.p1.1" class="ltx_p">Enhance privacy in federated settings</span>
</span></span>
</span>
</span>
</td>
<td id="S5.T4.3.3.5.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.5.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I17" class="ltx_itemize">
<span id="S5.I17.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I17.i1.p1" class="ltx_para">
<span id="S5.I17.i1.p1.1" class="ltx_p">High computational and communication overhead</span>
</span></span>
<span id="S5.I17.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I17.i2.p1" class="ltx_para">
<span id="S5.I17.i2.p1.1" class="ltx_p">Scalability issues with increasing number of participants</span>
</span></span>
<span id="S5.I17.i3" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I17.i3.p1" class="ltx_para">
<span id="S5.I17.i3.p1.1" class="ltx_p">Coordination complexity</span>
</span></span>
</span>
</span>
</td>
</tr>
<tr id="S5.T4.3.3.6" class="ltx_tr">
<td id="S5.T4.3.3.6.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T4.3.3.6.1.1" class="ltx_text"></span> <span id="S5.T4.3.3.6.1.2" class="ltx_text">
<span id="S5.T4.3.3.6.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.6.1.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.6.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Perturbation</span></span>
</span></span><span id="S5.T4.3.3.6.1.3" class="ltx_text"></span></td>
<td id="S5.T4.3.3.6.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T4.3.3.6.2.1" class="ltx_text"></span> <span id="S5.T4.3.3.6.2.2" class="ltx_text">
<span id="S5.T4.3.3.6.2.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.6.2.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.6.2.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Differential</span></span>
<span id="S5.T4.3.3.6.2.2.1.2" class="ltx_tr">
<span id="S5.T4.3.3.6.2.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Privacy</span></span>
</span></span><span id="S5.T4.3.3.6.2.3" class="ltx_text"></span></td>
<td id="S5.T4.3.3.6.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:199.2pt;">
<span id="S5.T4.3.3.6.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T4.3.3.6.3.1.1" class="ltx_p">Adds noise to local updates or gradients to mask individual data contributions, ensuring privacy</span>
</span>
</td>
<td id="S5.T4.3.3.6.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.6.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I18" class="ltx_itemize">
<span id="S5.I18.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I18.i1.p1" class="ltx_para">
<span id="S5.I18.i1.p1.1" class="ltx_p">Provide strong privacy guarantees with quantifiable privacy budget</span>
</span></span>
<span id="S5.I18.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I18.i2.p1" class="ltx_para">
<span id="S5.I18.i2.p1.1" class="ltx_p">Achieve competitive performance in healthcare applications</span>
</span></span>
</span>
</span>
</td>
<td id="S5.T4.3.3.6.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.6.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I19" class="ltx_itemize">
<span id="S5.I19.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I19.i1.p1" class="ltx_para">
<span id="S5.I19.i1.p1.1" class="ltx_p">Trade-off between privacy and model accuracy</span>
</span></span>
<span id="S5.I19.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I19.i2.p1" class="ltx_para">
<span id="S5.I19.i2.p1.1" class="ltx_p">Additional computational overhead</span>
</span></span>
</span>
</span>
</td>
</tr>
<tr id="S5.T4.3.3.7" class="ltx_tr">
<td id="S5.T4.3.3.7.1" class="ltx_td ltx_align_center ltx_border_bb ltx_border_r ltx_border_t" rowspan="2"><span id="S5.T4.3.3.7.1.1" class="ltx_text">Others</span></td>
<td id="S5.T4.3.3.7.2" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">
<span id="S5.T4.3.3.7.2.1" class="ltx_text"></span> <span id="S5.T4.3.3.7.2.2" class="ltx_text">
<span id="S5.T4.3.3.7.2.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.7.2.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.7.2.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Blockchain</span></span>
</span></span><span id="S5.T4.3.3.7.2.3" class="ltx_text"></span></td>
<td id="S5.T4.3.3.7.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:199.2pt;">
<span id="S5.T4.3.3.7.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T4.3.3.7.3.1.1" class="ltx_p">A decentralized ledger system that secures data sharing across all clients, reducing single points of failure</span>
</span>
</td>
<td id="S5.T4.3.3.7.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_r ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.7.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I20" class="ltx_itemize">
<span id="S5.I20.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I20.i1.p1" class="ltx_para">
<span id="S5.I20.i1.p1.1" class="ltx_p">Decentralized data sharing and management through distributed ledger</span>
</span></span>
<span id="S5.I20.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I20.i2.p1" class="ltx_para">
<span id="S5.I20.i2.p1.1" class="ltx_p">Enhanced data integrity and security with no single point of failure</span>
</span></span>
<span id="S5.I20.i3" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I20.i3.p1" class="ltx_para">
<span id="S5.I20.i3.p1.1" class="ltx_p">Effective at preventing and mitigating poisoning attacks with verification schemes</span>
</span></span>
</span>
</span>
</td>
<td id="S5.T4.3.3.7.5" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.7.5.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I21" class="ltx_itemize">
<span id="S5.I21.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I21.i1.p1" class="ltx_para">
<span id="S5.I21.i1.p1.1" class="ltx_p">High communication and computational costs, which may limit scalability</span>
</span></span>
<span id="S5.I21.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I21.i2.p1" class="ltx_para">
<span id="S5.I21.i2.p1.1" class="ltx_p">Potential challenges in key management and the need for substantial processing power</span>
</span></span>
</span>
</span>
</td>
</tr>
<tr id="S5.T4.3.3.8" class="ltx_tr">
<td id="S5.T4.3.3.8.1" class="ltx_td ltx_align_center ltx_border_bb ltx_border_r ltx_border_t">
<span id="S5.T4.3.3.8.1.1" class="ltx_text"></span> <span id="S5.T4.3.3.8.1.2" class="ltx_text">
<span id="S5.T4.3.3.8.1.2.1" class="ltx_tabular ltx_align_middle">
<span id="S5.T4.3.3.8.1.2.1.1" class="ltx_tr">
<span id="S5.T4.3.3.8.1.2.1.1.1" class="ltx_td ltx_nopad_r ltx_align_center">Swarm</span></span>
<span id="S5.T4.3.3.8.1.2.1.2" class="ltx_tr">
<span id="S5.T4.3.3.8.1.2.1.2.1" class="ltx_td ltx_nopad_r ltx_align_center">Learning</span></span>
</span></span><span id="S5.T4.3.3.8.1.3" class="ltx_text"></span></td>
<td id="S5.T4.3.3.8.2" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_r ltx_border_t" style="width:199.2pt;">
<span id="S5.T4.3.3.8.2.1" class="ltx_inline-block ltx_align_top">
<span id="S5.T4.3.3.8.2.1.1" class="ltx_p">A fully decentralized model training approach without a central aggregator</span>
</span>
</td>
<td id="S5.T4.3.3.8.3" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_r ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.8.3.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I22" class="ltx_itemize">
<span id="S5.I22.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I22.i1.p1" class="ltx_para">
<span id="S5.I22.i1.p1.1" class="ltx_p">Enhance resilience against attack</span>
</span></span>
<span id="S5.I22.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I22.i2.p1" class="ltx_para">
<span id="S5.I22.i2.p1.1" class="ltx_p">Effective in handling non-IID data, making it suitable for diverse and heterogeneous datasets</span>
</span></span>
<span id="S5.I22.i3" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I22.i3.p1" class="ltx_para">
<span id="S5.I22.i3.p1.1" class="ltx_p">Dynamically integrate decentralized hardware infrastructures</span>
</span></span>
</span>
</span>
</td>
<td id="S5.T4.3.3.8.4" class="ltx_td ltx_align_justify ltx_align_middle ltx_border_bb ltx_border_t" style="width:284.5pt;">
<span id="S5.T4.3.3.8.4.1" class="ltx_inline-block ltx_align_top">
<span id="S5.I23" class="ltx_itemize">
<span id="S5.I23.i1" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I23.i1.p1" class="ltx_para">
<span id="S5.I23.i1.p1.1" class="ltx_p">Latency issues due to peer-to-peer communication, potentially slowing down the training process</span>
</span></span>
<span id="S5.I23.i2" class="ltx_item" style="list-style-type:none;"><span class="ltx_tag ltx_tag_item">â€¢</span> 
<span id="S5.I23.i2.p1" class="ltx_para">
<span id="S5.I23.i2.p1.1" class="ltx_p">The absence of a central aggregator may limit certain coordination and optimization capabilities</span>
</span></span>
</span>
</span>
</td>
</tr>
</table>
</span></div>
</figure>
<div id="S5.SS5.p9" class="ltx_para">
<p id="S5.SS5.p9.1" class="ltx_p"><span id="S5.SS5.p9.1.1" class="ltx_text ltx_font_italic">Beyond</span> cryptography and perturbation methods, blockchainÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib23" title="" class="ltx_ref">23</a></sup></cite> and swarm learning (SL)Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib68" title="" class="ltx_ref">68</a></sup>, <sup class="ltx_sup"><a href="#bib.bib24" title="" class="ltx_ref">24</a></sup></cite> have gained lots of attention for enhancing privacy and security in healthcare.
<span id="S5.SS5.p9.1.2" class="ltx_text ltx_font_italic">Blockchain</span> is increasingly being integrated with FL to address data security and trust issues by replacing the central server with a decentralized privacy protocol. The key advantage of blockchain lies in its distributed ledger system, which ensures that data is securely shared and maintained across all clients without relying on a central authority. This decentralization reduces the risk of single points of failure and enhances data integrity. Furthermore, blockchainâ€™s verification schemes play a crucial role in the FL process, helping to detect and mitigate threats such as poisoning attacks. For instance, a blockchain-based FL framework developed by <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS5.p9.1.3.1" class="ltx_sup"><a href="#bib.bib75" title="" class="ltx_ref">75</a></sup></cite> combined DP and gradient-verification protocols to enhance security in IoMT devices, significantly reducing the success rate of poisoning attacks in tasks like diabetes prediction. Another approach by <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS5.p9.1.4.1" class="ltx_sup"><a href="#bib.bib74" title="" class="ltx_ref">74</a></sup></cite> utilized blockchain alongside an intrusion detection system to monitor and prevent malicious activities during model training, further securing federated healthcare networks. However, while blockchain offers robust security features, its integration with FL is often challenged by high communication and computational costs, which can limit its scalability and efficiency.
<span id="S5.SS5.p9.1.5" class="ltx_text ltx_font_italic">Swarm Learning</span> takes a different approach by decentralizing not just the privacy protocol but the entire model training process. In SL, there is no central aggregator, instead, decentralized hardware infrastructures work together to securely onboard clients and collaboratively generate a global model. This decentralized approach enhances the networkâ€™s resilience against attacks and is particularly effective in scenarios where data is non-IID, such as in the prediction of conditions like COVID-19 and leukemiaÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib68" title="" class="ltx_ref">68</a></sup>, <sup class="ltx_sup"><a href="#bib.bib24" title="" class="ltx_ref">24</a></sup></cite>.
However, SLâ€™s reliance on peer-to-peer communication can introduce latency issues, which may slow down the training process, particularly in environments with varying network conditions.</p>
</div>
<section id="S5.SS5.SSS0.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS5.SSS0.Px1.p1" class="ltx_para">
<ul id="S5.I24" class="ltx_itemize">
<li id="S5.I24.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I24.i1.p1" class="ltx_para">
<p id="S5.I24.i1.p1.1" class="ltx_p"><span id="S5.I24.i1.p1.1.1" class="ltx_text ltx_font_italic">Balancing Privacy and Model Utility:</span> Privacy-preserving FL systems must carefully balance privacy protection with model performance. It is essential to implement techniques that provide strong privacy guarantees while minimizing the impact on model accuracy.</p>
</div>
</li>
<li id="S5.I24.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I24.i2.p1" class="ltx_para">
<p id="S5.I24.i2.p1.1" class="ltx_p"><span id="S5.I24.i2.p1.1.1" class="ltx_text ltx_font_italic">Quantifying Privacy Levels:</span> Establish clear metrics for quantifying privacy levels. All participants should agree on the acceptable privacy thresholds, ensuring that these thresholds align with the collaborative research goals and regulatory requirements. This quantification should be transparent and well-documented to foster trust and compliance among stakeholders.</p>
</div>
</li>
<li id="S5.I24.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I24.i3.p1" class="ltx_para">
<p id="S5.I24.i3.p1.1" class="ltx_p"><span id="S5.I24.i3.p1.1.1" class="ltx_text ltx_font_italic">Comprehensive Privacy Enforcement:</span> Privacy protections should be applied uniformly across all components of the FL ecosystem, including clients, central servers, and communication channels. It is also critical to ensure the join or leave of participants does not compromise the federationâ€™s privacy promises.</p>
</div>
</li>
<li id="S5.I24.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I24.i4.p1" class="ltx_para">
<p id="S5.I24.i4.p1.1" class="ltx_p"><span id="S5.I24.i4.p1.1.1" class="ltx_text ltx_font_italic">Empirical and Theoretical Trade-offs:</span> Address the trade-offs between privacy and model performance requires both theoretical insights and empirical validation. Researchers should focus on understanding how various privacy-preserving techniques impact different aspects of model performance and utility. This includes investigating how privacy budgets, noise levels, and other parameters affect the overall effectiveness of the FL system.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS6" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.6 </span>Fairness and Incentive</h3>

<div id="S5.SS6.p1" class="ltx_para">
<p id="S5.SS6.p1.1" class="ltx_p">In FL for healthcare, research on fairness and incentive mechanisms is relatively underexplored, with only <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS6.p1.1.1.1" class="ltx_sup"><a href="#bib.bib53" title="" class="ltx_ref">53</a></sup></cite> delved into the intricacies of both fairness and incentive in FL for healthcare.</p>
</div>
<div id="S5.SS6.p2" class="ltx_para">
<p id="S5.SS6.p2.1" class="ltx_p">Fairness in FL refers to the equitable distribution of model performance across participants. In healthcare, it often means ensuring that ML models perform consistently across different healthcare providers and demographic groups or patient attributes. These fairness considerations are essential because disparities in model accuracy can lead to unequal treatment outcomes.
<cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS6.p2.1.1.1" class="ltx_sup"><a href="#bib.bib53" title="" class="ltx_ref">53</a></sup></cite> introduced several types of fairness, including horizontal fairness, where different hospitals receive comparable model accuracy, and vertical fairness, which focuses on ensuring that model performance is balanced across different demographic or medical attributes. They also proposed multilevel fairness, which seeks to address both client-level and attribute-level fairness simultaneously, and agnostic distribution fairness, aiming to generalize the modelâ€™s fairness to non-participating entities, such as hospitals outside the federation.</p>
</div>
<div id="S5.SS6.p3" class="ltx_para">
<p id="S5.SS6.p3.1" class="ltx_p">Incentive mechanisms are equally important in FL, as healthcare institutions often require motivation to participate in the federation. While regulatory constraints can mandate FL within organizations, voluntary participation in broader FL networks typically relies on clear incentives. For instance, hospitals engaging in FL for tasks like chest radiography classification or COVID-19 detection benefit from access to models that are more accurate than those developed using only local data. A well-designed incentive structure should ensure that participants who contribute more, whether through higher-quality data or computational resources, receive proportionately higher rewards. These rewards could be financial, reputational, or in the form of improved access to infrastructure.</p>
</div>
<section id="S5.SS6.SSS0.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS6.SSS0.Px1.p1" class="ltx_para">
<ul id="S5.I25" class="ltx_itemize">
<li id="S5.I25.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I25.i1.p1" class="ltx_para">
<p id="S5.I25.i1.p1.1" class="ltx_p"><span id="S5.I25.i1.p1.1.1" class="ltx_text ltx_font_italic">Dynamic Fairness Mechanisms:</span> Future research should focus on developing dynamic fairness mechanisms that can adapt to changes in data distributions and contributions from different healthcare providers. This would ensure that fairness is maintained even as the data and participants evolve over time.</p>
</div>
</li>
<li id="S5.I25.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I25.i2.p1" class="ltx_para">
<p id="S5.I25.i2.p1.1" class="ltx_p"><span id="S5.I25.i2.p1.1.1" class="ltx_text ltx_font_italic">Transparent Contribution Metrics:</span> Establish transparent and robust methods for quantifying each participantâ€™s contribution is essential. Accurate measurement of contributions in terms of data volume, quality, and computational resources will facilitate the creation of fair incentive structures.</p>
</div>
</li>
<li id="S5.I25.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I25.i3.p1" class="ltx_para">
<p id="S5.I25.i3.p1.1" class="ltx_p"><span id="S5.I25.i3.p1.1.1" class="ltx_text ltx_font_italic">Broader Incentive:</span> Incentive mechanisms should extend beyond financial rewards to include non-monetary incentives, such as reputation enhancement, access to advanced computational resources, and improved patient outcomes. This broader incentive could encourage more diverse participation from healthcare institutions.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS7" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.7 </span>Evaluation</h3>

<section id="S5.SS7.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.7.1 </span>Gaps in Evaluation Metrics</h4>

<div id="S5.SS7.SSS1.p1" class="ltx_para">
<p id="S5.SS7.SSS1.p1.1" class="ltx_p">The evaluation of FL models in healthcare heavily focuses on conventional ML metrics, such as accuracy, precision, AUC, sensitivity/recall, specificity, F1-score, Dice score, IoU, HD, and loss valueÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib186" title="" class="ltx_ref">186</a></sup></cite>.
FL models are typically compared against classical centralized models or localized models, with ablation studies commonly used to isolate the impact of specific modifications.
Most studies overlooked critical aspects unique to FL, such as communication overhead, resource consumption, privacy, and security concerns, thus failing to capture the complexity of FL systems.</p>
</div>
<div id="S5.SS7.SSS1.p2" class="ltx_para">
<p id="S5.SS7.SSS1.p2.1" class="ltx_p">FL involves frequent communication, which can introduce delays and increase costs. However, only a minority of studies (18%) included communication efficiency in their evaluation. Metrics such as communication cost, number of communication rounds, and latency are crucial for understanding the effectiveness of FL systemsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib109" title="" class="ltx_ref">109</a></sup>, <sup class="ltx_sup"><a href="#bib.bib34" title="" class="ltx_ref">34</a></sup></cite>.</p>
</div>
<div id="S5.SS7.SSS1.p3" class="ltx_para">
<p id="S5.SS7.SSS1.p3.1" class="ltx_p">Resource consumption is another crucial factor that has been underexplored, with only 12% of the reviewed studies measuring computational costs in FL evaluation. Key metrics such as training timeÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib97" title="" class="ltx_ref">97</a></sup></cite>, encryption timeÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib187" title="" class="ltx_ref">187</a></sup></cite>, CPU and memory consumptionÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib57" title="" class="ltx_ref">57</a></sup></cite> are necessary for evaluating the computational efficiency of FL systemsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib47" title="" class="ltx_ref">47</a></sup></cite>.</p>
</div>
<div id="S5.SS7.SSS1.p4" class="ltx_para">
<p id="S5.SS7.SSS1.p4.1" class="ltx_p">Privacy and security evaluations are fundamental for FL in healthcare. Despite this, only 16% of the reviewed studies assessed these aspects, with methods focusing on vulnerabilities to attacks such as model inversion attacks, and DP guarantees.
For instance, <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS7.SSS1.p4.1.1.1" class="ltx_sup"><a href="#bib.bib42" title="" class="ltx_ref">42</a></sup></cite> evaluated the influence of model inversion attacks on synthetic medical images in FL settings.
<cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS7.SSS1.p4.1.2.1" class="ltx_sup"><a href="#bib.bib32" title="" class="ltx_ref">32</a></sup></cite> offered theoretical guarantees for DP to assess the systemâ€™s privacy resilience.
<cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS7.SSS1.p4.1.3.1" class="ltx_sup"><a href="#bib.bib65" title="" class="ltx_ref">65</a></sup></cite> assessed various inversion attacks on medical images to measure and visualize potential data leakage in FL.</p>
</div>
<div id="S5.SS7.SSS1.p5" class="ltx_para">
<p id="S5.SS7.SSS1.p5.1" class="ltx_p">Scalability, a critical factor for the widespread adoption of FL in healthcare, has similarly been insufficiently evaluated. Variations in client numbers and patient populations need to be thoroughly assessed to ensure that FL systems can scale across large healthcare networks. Studies by <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS7.SSS1.p5.1.1.1" class="ltx_sup"><a href="#bib.bib36" title="" class="ltx_ref">36</a></sup></cite> and <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS7.SSS1.p5.1.2.1" class="ltx_sup"><a href="#bib.bib78" title="" class="ltx_ref">78</a></sup></cite> have demonstrated how FL models can handle varying clients and patient populations.</p>
</div>
<div id="S5.SS7.SSS1.p6" class="ltx_para">
<p id="S5.SS7.SSS1.p6.1" class="ltx_p">Finally, generalizability and robustness are crucial, especially given the heterogeneity of healthcare data.
As discussed in SectionÂ <a href="#S5.SS4.SSS1" title="5.4.1 Heterogeneity Issues â€£ 5.4 Optimization â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5.4.1</span></a>, most included studies focused on narrow aspects of data skew and heterogeneity, limiting the applicability of their findings across diverse healthcare institutions. To fully evaluate the performance of FL in healthcare, models must be tested across varying types of data skew and heterogeneity. Without this, the true potential and limitations of FL in heterogeneous healthcare environments cannot be fully understood.</p>
</div>
<figure id="S5.T5" class="ltx_table">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 5: </span>Comprehensive Evaluation Metrics for FL in Healthcare.</figcaption>
<div id="S5.T5.1" class="ltx_inline-block ltx_align_center ltx_transformed_outer" style="width:186.7pt;height:48.8pt;vertical-align:-0.4pt;"><span class="ltx_transformed_inner" style="transform:translate(-149.8pt,38.8pt) scale(0.383886307004898,0.383886307004898) ;">
<table id="S5.T5.1.1" class="ltx_tabular ltx_align_middle">
<tr id="S5.T5.1.1.1" class="ltx_tr">
<td id="S5.T5.1.1.1.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_tt">Aspect</td>
<td id="S5.T5.1.1.1.2" class="ltx_td ltx_align_center ltx_border_tt">Recommended Metrics</td>
</tr>
<tr id="S5.T5.1.1.2" class="ltx_tr">
<td id="S5.T5.1.1.2.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">Performance</td>
<td id="S5.T5.1.1.2.2" class="ltx_td ltx_align_center ltx_border_t">Accuracy, AUC, Precision, Sensitivity/Recall, F1-Score, Dice, IoU, HD, Loss</td>
</tr>
<tr id="S5.T5.1.1.3" class="ltx_tr">
<td id="S5.T5.1.1.3.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">Communication Efficiency</td>
<td id="S5.T5.1.1.3.2" class="ltx_td ltx_align_center ltx_border_t">Communication Cost, Communication Rounds, Latency</td>
</tr>
<tr id="S5.T5.1.1.4" class="ltx_tr">
<td id="S5.T5.1.1.4.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">Resource Consumption</td>
<td id="S5.T5.1.1.4.2" class="ltx_td ltx_align_center ltx_border_t">Training Time, CPU/Memory Usage, Encryption Time</td>
</tr>
<tr id="S5.T5.1.1.5" class="ltx_tr">
<td id="S5.T5.1.1.5.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">Privacy and Security</td>
<td id="S5.T5.1.1.5.2" class="ltx_td ltx_align_center ltx_border_t">Attacks listed in TableÂ <a href="#S5.T3" title="Table 3 â€£ Recommendations &amp; Opportunities â€£ 5.4.6 Synchronization Issues â€£ 5.4 Optimization â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, DP guarantees</td>
</tr>
<tr id="S5.T5.1.1.6" class="ltx_tr">
<td id="S5.T5.1.1.6.1" class="ltx_td ltx_align_center ltx_border_r ltx_border_t">Scalability</td>
<td id="S5.T5.1.1.6.2" class="ltx_td ltx_align_center ltx_border_t">Client Numbers, Patient Populations</td>
</tr>
<tr id="S5.T5.1.1.7" class="ltx_tr">
<td id="S5.T5.1.1.7.1" class="ltx_td ltx_align_center ltx_border_bb ltx_border_r ltx_border_t">Generalizability/Robustness</td>
<td id="S5.T5.1.1.7.2" class="ltx_td ltx_align_center ltx_border_bb ltx_border_t">Data Skew Scenarios listed in SectionÂ <a href="#S5.SS4.SSS1" title="5.4.1 Heterogeneity Issues â€£ 5.4 Optimization â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5.4.1</span></a>
</td>
</tr>
</table>
</span></div>
</figure>
<section id="S5.SS7.SSS1.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS7.SSS1.Px1.p1" class="ltx_para">
<p id="S5.SS7.SSS1.Px1.p1.1" class="ltx_p">In summary, there is a pressing need for a more comprehensive evaluation framework for FL in healthcare. This framework should encompass traditional performance metrics, communication efficiency, resource consumption, privacy and security, scalability, and generalizability. TableÂ <a href="#S5.T5" title="Table 5 â€£ 5.7.1 Gaps in Evaluation Metrics â€£ 5.7 Evaluation â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a> summarizes the key aspects and recommended evaluation metrics that should be considered in future research. By adopting this comprehensive approach, researchers can ensure that FL models are not only effective but also scalable, efficient, and secure for real-world healthcare applications.</p>
</div>
</section>
</section>
<section id="S5.SS7.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.7.2 </span>Insufficient Benchmarking</h4>

<div id="S5.SS7.SSS2.p1" class="ltx_para">
<p id="S5.SS7.SSS2.p1.1" class="ltx_p">Numerous FL algorithms have been proposed to address the challenges posed by non-IID data. However, systematic benchmarking of these algorithms is scarce.
Existing studies employed insufficient data partitioning strategies that failed to capture the diversity of real-world healthcare data distributions.
Most of them focused on only one or two types of data skew, limiting the scope of analysis and preventing a holistic understanding of algorithm performance under varied conditions.
This limitation extends to other critical aspects of FL evaluation as well. To date, no study has provided a comprehensive evaluation covering performance metrics, communication efficiency, resource consumption, privacy and security, scalability, and generalizability. Such evaluations are essential for a robust understanding of FLâ€™s applicability in healthcare.
</p>
</div>
<div id="S5.SS7.SSS2.p2" class="ltx_para">
<p id="S5.SS7.SSS2.p2.1" class="ltx_p">The absence of comprehensive and universally accepted datasets across various healthcare domains also hinders FL benchmarking. Depending on the research objectives, FL experiments may use datasets that vary significantly in scope and focus, such as medical image classification, segmentation, or reconstruction. Currently, there is no standardized, curated collection of large-scale healthcare datasets across various domains, specifically designed for FL research, which makes it difficult to ensure consistency in benchmarking.</p>
</div>
<div id="S5.SS7.SSS2.p3" class="ltx_para">
<p id="S5.SS7.SSS2.p3.1" class="ltx_p">Only one study has reported standardized benchmarking, but it included a limited set of healthcare datasets and failed to integrate key constraints of FL in healthcare, particularly privacy, efficiency, and generalizabilityÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib52" title="" class="ltx_ref">52</a></sup></cite>.</p>
</div>
<section id="S5.SS7.SSS2.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS7.SSS2.Px1.p1" class="ltx_para">
<ul id="S5.I26" class="ltx_itemize">
<li id="S5.I26.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I26.i1.p1" class="ltx_para">
<p id="S5.I26.i1.p1.1" class="ltx_p"><span id="S5.I26.i1.p1.1.1" class="ltx_text ltx_font_italic">Natural Client Splits and Metrics Definition:</span> Datasets should incorporate a natural client partition reflecting real-world healthcare scenarios, with clearly defined tasks and evaluation metrics. This will facilitate realistic and meaningful benchmarking.</p>
</div>
</li>
<li id="S5.I26.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I26.i2.p1" class="ltx_para">
<p id="S5.I26.i2.p1.1" class="ltx_p"><span id="S5.I26.i2.p1.1.1" class="ltx_text ltx_font_italic">Reproducible Train/Test Splits:</span> Ensure datasets have predefined and documented train/test splits for each client, enabling reproducible experiments and comparisons across different studies.</p>
</div>
</li>
<li id="S5.I26.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I26.i3.p1" class="ltx_para">
<p id="S5.I26.i3.p1.1" class="ltx_p"><span id="S5.I26.i3.p1.1.1" class="ltx_text ltx_font_italic">Baseline Models for Comparison:</span> Provide baseline models for each task, including a reference implementation for training on pooled data. This will help researchers compare FL performance against traditional centralized learning approaches.</p>
</div>
</li>
<li id="S5.I26.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I26.i4.p1" class="ltx_para">
<p id="S5.I26.i4.p1.1" class="ltx_p"><span id="S5.I26.i4.p1.1.1" class="ltx_text ltx_font_italic">Standardized API for FL Algorithms:</span> Standardize the API for FL algorithms to ensure compatibility with the dataset API, allowing for seamless benchmarking of different FL strategies.</p>
</div>
</li>
<li id="S5.I26.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I26.i5.p1" class="ltx_para">
<p id="S5.I26.i5.p1.1" class="ltx_p"><span id="S5.I26.i5.p1.1.1" class="ltx_text ltx_font_italic">Framework-Agnostic Algorithm Implementation:</span> Offer plain Python code for various FL algorithms that is independent of specific FL frameworks, ensuring flexibility and broader accessibility.</p>
</div>
</li>
<li id="S5.I26.i6" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I26.i6.p1" class="ltx_para">
<p id="S5.I26.i6.p1.1" class="ltx_p"><span id="S5.I26.i6.p1.1.1" class="ltx_text ltx_font_italic">Comprehensive Evaluation:</span> Cover basic performance metrics, communication efficiency, resource consumption, privacy, security, scalability, and generalizability. Follow the guidelines suggested in SectionÂ <a href="#S5.SS7.SSS1" title="5.7.1 Gaps in Evaluation Metrics â€£ 5.7 Evaluation â€£ 5 From Challenges and Pitfalls to Recommended Solutions and Future Opportunities â€£ From Challenges and Pitfalls to Recommendations and Opportunities: Implementing Federated Learning in Healthcare" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5.7.1</span></a>.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS7.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.7.3 </span>Lack of Interpretability</h4>

<div id="S5.SS7.SSS3.p1" class="ltx_para">
<p id="S5.SS7.SSS3.p1.1" class="ltx_p">FL models allow decentralized data processing, but their black-box nature makes it difficult to understand how decisions are made. This is particularly critical in healthcare, where incorrect decisions, such as misdiagnoses or inappropriate treatment recommendations, can have severe consequencesÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib22" title="" class="ltx_ref">22</a></sup></cite>. The opacity of these models raises concerns about trust and accountability, as medical professionals must be able to explain and justify the decisions made by such systemsÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib21" title="" class="ltx_ref">21</a></sup></cite>.</p>
</div>
<div id="S5.SS7.SSS3.p2" class="ltx_para">
<p id="S5.SS7.SSS3.p2.1" class="ltx_p">The primary challenge in achieving interpretability in FL stems from the decentralized nature of data. Since each clientâ€™s data remains private, the global model lacks direct access to local datasets, making it harder to detect biases, noisy features, or irrelevant data points. Additionally, resource constraints (e.g., limited computational power and communication bandwidth) can hinder the integration of advanced interpretability techniques.
Moreover, multiple stakeholders are involved in the decision-making process in FL: the central server needs to understand the significance of certain features, while clients must comprehend how their data contributes to the modelâ€™s performance.</p>
</div>
<div id="S5.SS7.SSS3.p3" class="ltx_para">
<p id="S5.SS7.SSS3.p3.1" class="ltx_p">Interpretable feature selection is an essential component of addressing these interpretability challenges in FL.
By identifying the most relevant features and filtering out noisy or redundant data, FL models can not only improve performance but also increase transparency in their decision-making processes. In healthcare, this is particularly important, as clinicians need to understand which clinical factors the model considers most relevant. For instance, <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS7.SSS3.p3.1.1.1" class="ltx_sup"><a href="#bib.bib62" title="" class="ltx_ref">62</a></sup></cite> utilized SHAP values to analyze the correlation between 20 clinical features and COVID-19 outcomes in FL settings, finding that eosinophil count had the greatest influence on predictions.
<cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS7.SSS3.p3.1.2.1" class="ltx_sup"><a href="#bib.bib188" title="" class="ltx_ref">188</a></sup></cite> introduced a mutual information-based approach to select relevant features in a decentralized manner, while <cite class="ltx_cite ltx_citemacro_cite"><sup id="S5.SS7.SSS3.p3.1.3.1" class="ltx_sup"><a href="#bib.bib189" title="" class="ltx_ref">189</a></sup></cite> proposed an unsupervised technique to detect outlier features and group similar ones via hierarchical clustering in FL.</p>
</div>
<div id="S5.SS7.SSS3.p4" class="ltx_para">
<p id="S5.SS7.SSS3.p4.1" class="ltx_p">In addition to feature selection, model-specific techniques such as tree-based FL provide further opportunities for interpretability by allowing models to function as â€œwhite-boxesâ€Â <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib135" title="" class="ltx_ref">135</a></sup>, <sup class="ltx_sup"><a href="#bib.bib190" title="" class="ltx_ref">190</a></sup></cite>. These methods leverage the modelâ€™s internal structure to explain its behavior.
However, such approaches are often highly specific to particular types of data and may not be broadly applicable across different healthcare domains.
Similarly, while gradient-based explanations and attention mechanisms offer another interpretability route, their effectiveness is sometimes limited due to weak correlations between these methods and the actual decision-making processes of the modelÂ <cite class="ltx_cite ltx_citemacro_citep"><sup class="ltx_sup"><a href="#bib.bib51" title="" class="ltx_ref">51</a></sup>, <sup class="ltx_sup"><a href="#bib.bib89" title="" class="ltx_ref">89</a></sup></cite>.</p>
</div>
<section id="S5.SS7.SSS3.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS7.SSS3.Px1.p1" class="ltx_para">
<ul id="S5.I27" class="ltx_itemize">
<li id="S5.I27.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I27.i1.p1" class="ltx_para">
<p id="S5.I27.i1.p1.1" class="ltx_p"><span id="S5.I27.i1.p1.1.1" class="ltx_text ltx_font_italic">Incorporation of Domain Knowledge:</span> Integrate domain-specific knowledge into FL models to enhance interpretability. In healthcare, leveraging medical knowledge (e.g., known correlations between symptoms and diseases) can help guide feature selection and model design, making it easier to explain model outputs to clinicians.</p>
</div>
</li>
<li id="S5.I27.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I27.i2.p1" class="ltx_para">
<p id="S5.I27.i2.p1.1" class="ltx_p"><span id="S5.I27.i2.p1.1.1" class="ltx_text ltx_font_italic">Inherit and Expand Explainable Models:</span> Techniques such as federated decision trees, and rule-based methods could be explored to build models that are transparent by design.</p>
</div>
</li>
<li id="S5.I27.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I27.i3.p1" class="ltx_para">
<p id="S5.I27.i3.p1.1" class="ltx_p"><span id="S5.I27.i3.p1.1.1" class="ltx_text ltx_font_italic">Client-Side Interpretability Tools:</span> Create lightweight, client-side tools for interpretability that allow individual clients to better understand how their data contributes to the global model. These tools should be resource-efficient to accommodate the limited computational power and bandwidth of many FL clients, particularly in remote healthcare settings.</p>
</div>
</li>
<li id="S5.I27.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I27.i4.p1" class="ltx_para">
<p id="S5.I27.i4.p1.1" class="ltx_p"><span id="S5.I27.i4.p1.1.1" class="ltx_text ltx_font_italic">Interpretable Aggregation:</span> Explore novel aggregation methods that not only combine client updates but also explain why certain updates were prioritized over others. These methods could use techniques such as explainable boosting or weighted aggregation based on feature importance to make the global model more transparent.</p>
</div>
</li>
<li id="S5.I27.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I27.i5.p1" class="ltx_para">
<p id="S5.I27.i5.p1.1" class="ltx_p"><span id="S5.I27.i5.p1.1.1" class="ltx_text ltx_font_italic">Collaboration Between AI and Healthcare Professionals:</span> Foster collaborations between AI experts and healthcare professionals to ensure that the interpretability techniques developed for FL models align with the needs and expectations of medical practitioners. Regular feedback loops could be established to fine-tune models for both accuracy and interpretability.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
<section id="S5.SS7.SSS4" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">5.7.4 </span>Poor Documentation and Reproducibility</h4>

<div id="S5.SS7.SSS4.p1" class="ltx_para">
<p id="S5.SS7.SSS4.p1.1" class="ltx_p">The reproducibility of FL in healthcare is significantly hindered by several critical issues related to documentation, custom implementations, open-source code availability, and the use of private data.</p>
</div>
<div id="S5.SS7.SSS4.p2" class="ltx_para">
<p id="S5.SS7.SSS4.p2.1" class="ltx_p">Firstly, inadequate documentation is a major obstacle. Many included studies lacked crucial details required for reproducing results, such as the methods for data preprocessing, data imputation and augmentation, model initialization, optimization algorithms, and choice of key hyperparameters. This absence of detailed documentation makes it challenging to replicate the reported findings accurately. Additionally, there is often a lack of clarity regarding the data exchanged between clients and the central server. Terms like â€œmodel parametersâ€ and â€œmodel updatesâ€ are frequently used without precise definitions, which leads to ambiguity about whether these terms refer to gradients, model weights, or other parameters.</p>
</div>
<div id="S5.SS7.SSS4.p3" class="ltx_para">
<p id="S5.SS7.SSS4.p3.1" class="ltx_p">Secondly, the widespread use of custom FL frameworks exacerbates the issue. Many studies chose to develop their own implementations instead of utilizing established, open-source frameworks. Given the complexity of FL systems, custom implementations are more prone to errors and may lack the robustness of well-tested frameworks. This practice can lead to inconsistencies and difficulties in reproducing results.</p>
</div>
<div id="S5.SS7.SSS4.p4" class="ltx_para">
<p id="S5.SS7.SSS4.p4.1" class="ltx_p">Thirdly, the availability of open-source code is critically limited. Only 27% of the reviewed studies made their code publicly available, and none released their trained models. This lack of transparency severely hampers the ability to independently assess and validate model performance, further obstructing reproducibility and impeding future research and application of FL techniques.</p>
</div>
<div id="S5.SS7.SSS4.p5" class="ltx_para">
<p id="S5.SS7.SSS4.p5.1" class="ltx_p">Additionally, many studies relied on private data and did not test their methods on publicly available datasets. This practice further complicates reproducibility and fairness in comparisons, as proprietary datasets restrict the ability to conduct fair evaluations and verify results.</p>
</div>
<section id="S5.SS7.SSS4.Px1" class="ltx_paragraph">
<h5 class="ltx_title ltx_font_bold ltx_title_paragraph">Recommendations &amp; Opportunities</h5>

<div id="S5.SS7.SSS4.Px1.p1" class="ltx_para">
<ul id="S5.I28" class="ltx_itemize">
<li id="S5.I28.i1" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I28.i1.p1" class="ltx_para">
<p id="S5.I28.i1.p1.1" class="ltx_p"><span id="S5.I28.i1.p1.1.1" class="ltx_text ltx_font_italic">Code &amp; Model Release:</span> Prioritize the release of well-documented code and trained models to facilitate independent performance evaluations and advance the field collaboratively.</p>
</div>
</li>
<li id="S5.I28.i2" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I28.i2.p1" class="ltx_para">
<p id="S5.I28.i2.p1.1" class="ltx_p"><span id="S5.I28.i2.p1.1.1" class="ltx_text ltx_font_italic">Checklist:</span> Create a comprehensive FL methodology checklist could improve documentation practices in future studies.</p>
</div>
</li>
<li id="S5.I28.i3" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I28.i3.p1" class="ltx_para">
<p id="S5.I28.i3.p1.1" class="ltx_p"><span id="S5.I28.i3.p1.1.1" class="ltx_text ltx_font_italic">Piplines Documentation:</span> Implement full-stack FL pipelines with documentation can simplify AI development for healthcare institutions, making advanced methods more accessible to users with information technology expertise.</p>
</div>
</li>
<li id="S5.I28.i4" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I28.i4.p1" class="ltx_para">
<p id="S5.I28.i4.p1.1" class="ltx_p"><span id="S5.I28.i4.p1.1.1" class="ltx_text ltx_font_italic">Evaluation on Public Datasets:</span> Encourage the use of public datasets for evaluation is also crucial to ensure fair comparisons and enhance the generalizability of results.</p>
</div>
</li>
<li id="S5.I28.i5" class="ltx_item" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">â€¢</span> 
<div id="S5.I28.i5.p1" class="ltx_para">
<p id="S5.I28.i5.p1.1" class="ltx_p"><span id="S5.I28.i5.p1.1.1" class="ltx_text ltx_font_italic">Framework:</span> Extend existing FL frameworks rather than developing new ones should be promoted to reduce the risk of errors and support community-driven validation and improvement.</p>
</div>
</li>
</ul>
</div>
</section>
</section>
</section>
</section>
<section id="S6" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Conclusions</h2>

<div id="S6.p1" class="ltx_para">
<p id="S6.p1.1" class="ltx_p">In this review, we find that the application of FL to healthcare is still in its relative infancy, with most studies focusing on prediction tasks and often lacking robust demonstrations of clinically significant outcomes.
We delve into the challenges and pitfalls of existing solutions and offer practical guidelines for selecting the most appropriate techniques based on specific application scenarios.
Additionally, we identify open research challenges that need to be addressed in the near future. We also highlight the importance of establishing standardized methodologies and protocols, as well as promoting the release of open-source code to ensure reproducibility and transparency in FL development in healthcare.
We hope that this review will spark new ideas and inspire numerous possibilities for research and application in healthcare FL.</p>
</div>
</section>
<section id="Sx3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Author Contributions</h2>

<div id="Sx3.p1" class="ltx_para">
<p id="Sx3.p1.1" class="ltx_p">Method and investigation: M.L., P.X., J.H., and Z.T.;
writing: M.L., P.X., J.H., and Z.T.;
supervision and review: G.Y.
All authors have read and agreed to this manuscript.</p>
</div>
</section>
<section id="Sx4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Acknowledgments</h2>

<div id="Sx4.p1" class="ltx_para">
<p id="Sx4.p1.1" class="ltx_p">This study was supported in part by the ERC IMI (101005122), the H2020 (952172), the MRC (MC/PC/21013), the Royal Society (IEC\NSFC\211235), the NVIDIA Academic Hardware Grant Program, the SABER project supported by Boehringer Ingelheim Ltd, NIHR Imperial Biomedical Research Centre (RDA01), Wellcome Leap Dynamic Resilience, UKRI guarantee funding for Horizon Europe MSCA Postdoctoral Fellowships (EP/Z002206/1), and the UKRI Future Leaders Fellowship (MR/V023799/1).</p>
</div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liu etÂ al. 2021a</span>
<span class="ltx_bibblock">
Liu, X., Fan, Y., Li, S., Chen, M., Li, M., Hau, W.Â K., Zhang, H., Xu, L., and Lee, A. P.-W. (2021a). Deep learning-based automated left ventricular ejection fraction assessment using 2-d echocardiography.

</span>
<span class="ltx_bibblock">American Journal of Physiology-Heart and Circulatory Physiology <em id="bib.bib1.1.1" class="ltx_emph ltx_font_italic">321</em>, H390â€“H399.

</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Newton etÂ al. 2013</span>
<span class="ltx_bibblock">
Newton, K.Â M., Peissig, P.Â L., Kho, A.Â N., Bielinski, S.Â J., Berg, R.Â L., Choudhary, V., Basford, M., Chute, C.Â G., Kullo, I.Â J., Li, R. etÂ al. (2013). Validation of electronic medical record-based phenotyping algorithms: results and lessons learned from the emerge network.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association <em id="bib.bib2.1.1" class="ltx_emph ltx_font_italic">20</em>, e147â€“e154.

</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ngiam and Khor 2019</span>
<span class="ltx_bibblock">
Ngiam, K.Â Y., and Khor, W. (2019). Big data and machine learning algorithms for health-care delivery.

</span>
<span class="ltx_bibblock">The Lancet Oncology <em id="bib.bib3.1.1" class="ltx_emph ltx_font_italic">20</em>, e262â€“e273.

</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">McMahan etÂ al. 2017</span>
<span class="ltx_bibblock">
McMahan, B., Moore, E., Ramage, D., Hampson, S., and yÂ Arcas, B.Â A.

</span>
<span class="ltx_bibblock">Communication-efficient learning of deep networks from decentralized data.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib4.1.1" class="ltx_emph ltx_font_italic">Artificial intelligence and statistics</em>. PMLR (2017):( 1273â€“1282).

</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sheller etÂ al. 2019</span>
<span class="ltx_bibblock">
Sheller, M.Â J., Reina, G.Â A., Edwards, B., Martin, J., and Bakas, S.

</span>
<span class="ltx_bibblock">Multi-institutional deep learning modeling without sharing patient data: A feasibility study on brain tumor segmentation.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib5.1.1" class="ltx_emph ltx_font_italic">Brainlesion: Glioma, Multiple Sclerosis, Stroke and Traumatic Brain Injuries: 4th International Workshop, BrainLes 2018, Held in Conjunction with MICCAI 2018, Granada, Spain, September 16, 2018, Revised Selected Papers, Part I 4</em>. Springer (2019):( 92â€“104).

</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sheller etÂ al. 2020</span>
<span class="ltx_bibblock">
Sheller, M.Â J., Edwards, B., Reina, G.Â A., Martin, J., Pati, S., Kotrotsou, A., Milchenko, M., Xu, W., Marcus, D., Colen, R.Â R. etÂ al. (2020). Federated learning in medicine: facilitating multi-institutional collaborations without sharing patient data.

</span>
<span class="ltx_bibblock">Scientific reports <em id="bib.bib6.1.1" class="ltx_emph ltx_font_italic">10</em>, 12598.

</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">OgierÂ du Terrail etÂ al. 2023</span>
<span class="ltx_bibblock">
OgierÂ du Terrail, J., Leopold, A., Joly, C., BÃ©guier, C., Andreux, M., Maussion, C., Schmauch, B., Tramel, E.Â W., Bendjebbar, E., Zaslavskiy, M. etÂ al. (2023). Federated learning for predicting histological response to neoadjuvant chemotherapy in triple-negative breast cancer.

</span>
<span class="ltx_bibblock">Nature medicine <em id="bib.bib7.1.1" class="ltx_emph ltx_font_italic">29</em>, 135â€“146.

</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Dayan etÂ al. 2021</span>
<span class="ltx_bibblock">
Dayan, I., Roth, H.Â R., Zhong, A., Harouni, A., Gentili, A., Abidin, A.Â Z., Liu, A., Costa, A.Â B., Wood, B.Â J., Tsai, C.-S. etÂ al. (2021). Federated learning for predicting clinical outcomes in patients with covid-19.

</span>
<span class="ltx_bibblock">Nature medicine <em id="bib.bib8.1.1" class="ltx_emph ltx_font_italic">27</em>, 1735â€“1743.

</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Helmholtz Association 2024</span>
<span class="ltx_bibblock">
Helmholtz Association.

</span>
<span class="ltx_bibblock">Trustworthy federated data analytics (tfda).

</span>
<span class="ltx_bibblock">Web Page (2024).

</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://tfda.hmsp.center" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://tfda.hmsp.center</a> accessed: 2024-04-24.

</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">German Cancer Consortium (2024) DKTK</span>
<span class="ltx_bibblock">German Cancer Consortium (DKTK).

</span>
<span class="ltx_bibblock">Joint imaging platform.

</span>
<span class="ltx_bibblock">Web Page (2024).

</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://jip.dktk.dkfz.de/jiphomepage/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://jip.dktk.dkfz.de/jiphomepage/</a> accessed: 2024-04-24.

</span>
</li>
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">NVIDIA 2020</span>
<span class="ltx_bibblock">
NVIDIA.

</span>
<span class="ltx_bibblock">Medical institutions collaborate to improve mammogram assessment ai.

</span>
<span class="ltx_bibblock">Web Page (2020).

</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://blogs.nvidia.com/blog/federated-learning-mammogram-assessment/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://blogs.nvidia.com/blog/federated-learning-mammogram-assessment/</a> accessed: 2020-05-28.

</span>
</li>
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Healthchain Consortium 2024</span>
<span class="ltx_bibblock">
Healthchain Consortium.

</span>
<span class="ltx_bibblock">Healthchain consortium.

</span>
<span class="ltx_bibblock">Web Page (2024).

</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="http://healthchain-i3.eu" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://healthchain-i3.eu</a> accessed: 2024-04-24.

</span>
</li>
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">European Lung Foundation 2024</span>
<span class="ltx_bibblock">
European Lung Foundation.

</span>
<span class="ltx_bibblock">Dragon project (2024).

</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://europeanlung.org/dragon/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://europeanlung.org/dragon/</a> accessed: 2024-04-24.

</span>
</li>
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">FeTS-AI 2024</span>
<span class="ltx_bibblock">
FeTS-AI.

</span>
<span class="ltx_bibblock">The federated tumor segmentation (fets) initiative (2024).

</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://www.fets.ai" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.fets.ai</a> accessed: 2024-04-24.

</span>
</li>
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Melody Project 2024</span>
<span class="ltx_bibblock">
Melody Project.

</span>
<span class="ltx_bibblock">Machine learning ledger orchestration for drug discovery (2024).

</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://www.melloddy.eu" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.melloddy.eu</a> accessed: 2024-04-24.

</span>
</li>
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Fang etÂ al. 2022</span>
<span class="ltx_bibblock">
Fang, M.Â L., Dhami, D.Â S., and Kersting, K.

</span>
<span class="ltx_bibblock">Dp-ctgan: Differentially private medical data generation using ctgans.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib16.1.1" class="ltx_emph ltx_font_italic">International Conference on Artificial Intelligence in Medicine</em>. Springer (2022):( 178â€“188).

</span>
</li>
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kline etÂ al. 2022</span>
<span class="ltx_bibblock">
Kline, A., Wang, H., Li, Y., Dennis, S., Hutch, M., Xu, Z., Wang, F., Cheng, F., and Luo, Y. (2022). Multimodal machine learning in precision health: A scoping review.

</span>
<span class="ltx_bibblock">npj Digital Medicine <em id="bib.bib17.1.1" class="ltx_emph ltx_font_italic">5</em>, 171.

</span>
</li>
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Rothchild etÂ al. 2020</span>
<span class="ltx_bibblock">
Rothchild, D., Panda, A., Ullah, E., Ivkin, N., Stoica, I., Braverman, V., Gonzalez, J., and Arora, R.

</span>
<span class="ltx_bibblock">Fetchsgd: Communication-efficient federated learning with sketching.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib18.1.1" class="ltx_emph ltx_font_italic">International Conference on Machine Learning</em>. PMLR (2020):( 8253â€“8265).

</span>
</li>
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Nguyen etÂ al. 2021</span>
<span class="ltx_bibblock">
Nguyen, D.Â C., Ding, M., Pathirana, P.Â N., Seneviratne, A., and Zomaya, A.Â Y. (2021). Federated learning for covid-19 detection with generative adversarial networks in edge cloud computing.

</span>
<span class="ltx_bibblock">IEEE Internet of Things Journal <em id="bib.bib19.1.1" class="ltx_emph ltx_font_italic">9</em>, 10257â€“10271.

</span>
</li>
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Bouacida and Mohapatra 2021</span>
<span class="ltx_bibblock">
Bouacida, N., and Mohapatra, P. (2021). Vulnerabilities in federated learning.

</span>
<span class="ltx_bibblock">IEEE Access <em id="bib.bib20.1.1" class="ltx_emph ltx_font_italic">9</em>, 63229â€“63249.

</span>
</li>
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2023a</span>
<span class="ltx_bibblock">
Li, A., Liu, R., Hu, M., Tuan, L.Â A., and Yu, H. (2023a). Towards interpretable federated learning.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2302.13473.

</span>
</li>
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2022a</span>
<span class="ltx_bibblock">
Li, M., Fang, Y., Tang, Z., Onuorah, C., Xia, J., DelÂ Ser, J., Walsh, S., and Yang, G. (2022a). Explainable covid-19 infections identification and delineation using calibrated pseudo labels.

</span>
<span class="ltx_bibblock">IEEE Transactions on Emerging Topics in Computational Intelligence <em id="bib.bib22.1.1" class="ltx_emph ltx_font_italic">7</em>, 26â€“35.

</span>
</li>
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kumar etÂ al. 2021</span>
<span class="ltx_bibblock">
Kumar, R., Khan, A.Â A., Kumar, J., Golilarz, N.Â A., Zhang, S., Ting, Y., Zheng, C., Wang, W. etÂ al. (2021). Blockchain-federated-learning and deep learning models for covid-19 detection using ct imaging.

</span>
<span class="ltx_bibblock">IEEE Sensors Journal <em id="bib.bib23.1.1" class="ltx_emph ltx_font_italic">21</em>, 16301â€“16314.

</span>
</li>
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Saldanha etÂ al. 2022</span>
<span class="ltx_bibblock">
Saldanha, O.Â L., Quirke, P., West, N.Â P., James, J.Â A., Loughrey, M.Â B., Grabsch, H.Â I., Salto-Tellez, M., Alwers, E., Cifci, D., GhaffariÂ Laleh, N. etÂ al. (2022). Swarm learning for decentralized artificial intelligence in cancer histopathology.

</span>
<span class="ltx_bibblock">Nature medicine <em id="bib.bib24.1.1" class="ltx_emph ltx_font_italic">28</em>, 1232â€“1239.

</span>
</li>
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Thapa etÂ al. 2022</span>
<span class="ltx_bibblock">
Thapa, C., Arachchige, P. C.Â M., Camtepe, S., and Sun, L.

</span>
<span class="ltx_bibblock">Splitfed: When federated learning meets split learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib25.1.1" class="ltx_emph ltx_font_italic">Proceedings of the AAAI Conference on Artificial Intelligence</em> vol.Â 36 (2022):( 8485â€“8493).

</span>
</li>
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhu etÂ al. 2021</span>
<span class="ltx_bibblock">
Zhu, Z., Hong, J., and Zhou, J.

</span>
<span class="ltx_bibblock">Data-free knowledge distillation for heterogeneous federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib26.1.1" class="ltx_emph ltx_font_italic">International conference on machine learning</em>. PMLR (2021):( 12878â€“12889).

</span>
</li>
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li and Yang 2023</span>
<span class="ltx_bibblock">
Li, M., and Yang, G.

</span>
<span class="ltx_bibblock">Data-free distillation improves efficiency and privacy in federated thorax disease analysis.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib27.1.1" class="ltx_emph ltx_font_italic">2023 IEEE EMBS Special Topic Conference on Data Science and Engineering in Healthcare, Medicine and Biology</em>. IEEE (2023):( 131â€“132).

</span>
</li>
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liu etÂ al. 2022</span>
<span class="ltx_bibblock">
Liu, D., Fox, K., Weber, G., and Miller, T. (2022). Confederated learning in healthcare: Training machine learning models using disconnected data separated by individual, data type and identity for large-scale health system intelligence.

</span>
<span class="ltx_bibblock">Journal of Biomedical Informatics <em id="bib.bib28.1.1" class="ltx_emph ltx_font_italic">134</em>, 104151.

</span>
</li>
<li id="bib.bib29" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Linardos etÂ al. 2022</span>
<span class="ltx_bibblock">
Linardos, A., Kushibar, K., Walsh, S., Gkontra, P., and Lekadir, K. (2022). Federated learning for multi-center imaging diagnostics: a simulation study in cardiovascular disease.

</span>
<span class="ltx_bibblock">Scientific Reports <em id="bib.bib29.1.1" class="ltx_emph ltx_font_italic">12</em>, 3551.

</span>
</li>
<li id="bib.bib30" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chakravarty etÂ al. 2021</span>
<span class="ltx_bibblock">
Chakravarty, A., Kar, A., Sethuraman, R., and Sheet, D.

</span>
<span class="ltx_bibblock">Federated learning for site aware chest radiograph screening.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib30.1.1" class="ltx_emph ltx_font_italic">2021 IEEE 18th International Symposium on Biomedical Imaging (ISBI)</em>. IEEE (2021):( 1077â€“1081).

</span>
</li>
<li id="bib.bib31" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liu etÂ al. 2021b</span>
<span class="ltx_bibblock">
Liu, Q., Chen, C., Qin, J., Dou, Q., and Heng, P.-A. (2021b). Feddg: Federated domain generalization on medical image segmentation via episodic learning in continuous frequency space.

</span>
<span class="ltx_bibblock">The IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR).

</span>
</li>
<li id="bib.bib32" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhou etÂ al. 2024a</span>
<span class="ltx_bibblock">
Zhou, J., Chen, S., Wu, Y., Li, H., Zhang, B., Zhou, L., Hu, Y., Xiang, Z., Li, Z., Chen, N. etÂ al. (2024a). Ppml-omics: a privacy-preserving federated machine learning method protects patientsâ€™ privacy in omic data.

</span>
<span class="ltx_bibblock">Science Advances <em id="bib.bib32.1.1" class="ltx_emph ltx_font_italic">10</em>, eadh8601.

</span>
</li>
<li id="bib.bib33" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">HaggenmÃ¼ller etÂ al. 2024</span>
<span class="ltx_bibblock">
HaggenmÃ¼ller, S., Schmitt, M., Krieghoff-Henning, E., Hekler, A., Maron, R.Â C., Wies, C., Utikal, J.Â S., Meier, F., Hobelsberger, S., Gellrich, F.Â F. etÂ al. (2024). Federated learning for decentralized artificial intelligence in melanoma diagnostics.

</span>
<span class="ltx_bibblock">JAMA dermatology <em id="bib.bib33.1.1" class="ltx_emph ltx_font_italic">160</em>, 303â€“311.

</span>
</li>
<li id="bib.bib34" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Malik etÂ al. 2023</span>
<span class="ltx_bibblock">
Malik, H., Naeem, A., Naqvi, R.Â A., and Loh, W.-K. (2023). Dmfl_net: A federated learning-based framework for the classification of covid-19 from multiple chest diseases using x-rays.

</span>
<span class="ltx_bibblock">Sensors <em id="bib.bib34.1.1" class="ltx_emph ltx_font_italic">23</em>, 743.

</span>
</li>
<li id="bib.bib35" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sadilek etÂ al. 2021</span>
<span class="ltx_bibblock">
Sadilek, A., Liu, L., Nguyen, D., Kamruzzaman, M., Serghiou, S., Rader, B., Ingerman, A., Mellem, S., Kairouz, P., Nsoesie, E.Â O. etÂ al. (2021). Privacy-first health research with federated learning.

</span>
<span class="ltx_bibblock">NPJ digital medicine <em id="bib.bib35.1.1" class="ltx_emph ltx_font_italic">4</em>, 132.

</span>
</li>
<li id="bib.bib36" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Yan etÂ al. 2024</span>
<span class="ltx_bibblock">
Yan, Y., Wang, H., Huang, Y., He, N., Zhu, L., Xu, Y., Li, Y., and Zheng, Y. (2024). Cross-modal vertical federated learning for mri reconstruction.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics.

</span>
</li>
<li id="bib.bib37" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zou etÂ al. 2023</span>
<span class="ltx_bibblock">
Zou, J., Pei, T., Li, C., Wu, R., and Wang, S. (2023). Self-supervised federated learning for fast mr imaging.

</span>
<span class="ltx_bibblock">IEEE Transactions on Instrumentation and Measurement.

</span>
</li>
<li id="bib.bib38" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sun etÂ al. 2021a</span>
<span class="ltx_bibblock">
Sun, W., Chen, Y., Yang, X., Cao, J., and Song, Y.

</span>
<span class="ltx_bibblock">Fedio: Bridge inner-and outer-hospital information for perioperative complications prognostic prediction via federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib38.1.1" class="ltx_emph ltx_font_italic">2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</em>. IEEE (2021a):( 3215â€“3221).

</span>
</li>
<li id="bib.bib39" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lu etÂ al. 2022a</span>
<span class="ltx_bibblock">
Lu, M.Â Y., Chen, R.Â J., Kong, D., Lipkova, J., Singh, R., Williamson, D.Â F., Chen, T.Â Y., and Mahmood, F. (2022a). Federated learning for computational pathology on gigapixel whole slide images.

</span>
<span class="ltx_bibblock">Medical image analysis <em id="bib.bib39.1.1" class="ltx_emph ltx_font_italic">76</em>, 102298.

</span>
</li>
<li id="bib.bib40" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wang etÂ al. 2023a</span>
<span class="ltx_bibblock">
Wang, J., Xie, G., Huang, Y., Lyu, J., Zheng, F., Zheng, Y., and Jin, Y. (2023a). Fedmed-gan: Federated domain translation on unsupervised cross-modality brain image synthesis.

</span>
<span class="ltx_bibblock">Neurocomputing <em id="bib.bib40.1.1" class="ltx_emph ltx_font_italic">546</em>, 126282.

</span>
</li>
<li id="bib.bib41" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Dalmaz etÂ al. 2024</span>
<span class="ltx_bibblock">
Dalmaz, O., Mirza, M.Â U., Elmas, G., Ozbey, M., Dar, S.Â U., Ceyani, E., Oguz, K.Â K., Avestimehr, S., and Ã‡ukur, T. (2024). One model to unite them all: Personalized federated learning of multi-contrast mri synthesis.

</span>
<span class="ltx_bibblock">Medical Image Analysis <em id="bib.bib41.1.1" class="ltx_emph ltx_font_italic">94</em>, 103121.

</span>
</li>
<li id="bib.bib42" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Jin and Li 2023</span>
<span class="ltx_bibblock">
Jin, R., and Li, X. (2023). Backdoor attack and defense in federated generative adversarial network-based medical image synthesis.

</span>
<span class="ltx_bibblock">Medical Image Analysis <em id="bib.bib42.1.1" class="ltx_emph ltx_font_italic">90</em>, 102965.

</span>
</li>
<li id="bib.bib43" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Peng etÂ al. 2024</span>
<span class="ltx_bibblock">
Peng, L., Luo, G., Zhou, S., Chen, J., Xu, Z., Sun, J., and Zhang, R. (2024). An in-depth evaluation of federated learning on biomedical natural language processing for information extraction.

</span>
<span class="ltx_bibblock">NPJ Digital Medicine <em id="bib.bib43.1.1" class="ltx_emph ltx_font_italic">7</em>, 127.

</span>
</li>
<li id="bib.bib44" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Tian etÂ al. 2023</span>
<span class="ltx_bibblock">
Tian, Y., Wang, S., Xiong, J., Bi, R., Zhou, Z., and Bhuiyan, M. Z.Â A. (2023). Robust and privacy-preserving decentralized deep federated learning training: Focusing on digital healthcare applications.

</span>
<span class="ltx_bibblock">IEEE/ACM Transactions on computational biology and bioinformatics.

</span>
</li>
<li id="bib.bib45" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2022b</span>
<span class="ltx_bibblock">
Li, L., Yu, X., Cai, X., He, X., and Liu, Y. (2022b). Contract-theory-based incentive mechanism for federated learning in health crowdsensing.

</span>
<span class="ltx_bibblock">IEEE Internet of Things Journal <em id="bib.bib45.1.1" class="ltx_emph ltx_font_italic">10</em>, 4475â€“4489.

</span>
</li>
<li id="bib.bib46" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wang etÂ al. 2023b</span>
<span class="ltx_bibblock">
Wang, W., Li, X., Qiu, X., Zhang, X., Brusic, V., and Zhao, J. (2023b). A privacy preserving framework for federated learning in smart healthcare systems.

</span>
<span class="ltx_bibblock">Information Processing &amp; Management <em id="bib.bib46.1.1" class="ltx_emph ltx_font_italic">60</em>, 103167.

</span>
</li>
<li id="bib.bib47" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kalapaaking etÂ al. 2022</span>
<span class="ltx_bibblock">
Kalapaaking, A.Â P., Stephanie, V., Khalil, I., Atiquzzaman, M., Yi, X., and Almashor, M. (2022). Smpc-based federated learning for 6g-enabled internet of medical things.

</span>
<span class="ltx_bibblock">IEEE Network <em id="bib.bib47.1.1" class="ltx_emph ltx_font_italic">36</em>, 182â€“189.

</span>
</li>
<li id="bib.bib48" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Che etÂ al. 2022</span>
<span class="ltx_bibblock">
Che, S., Kong, Z., Peng, H., Sun, L., Leow, A., Chen, Y., and He, L. (2022). Federated multi-view learning for private medical data integration and analysis.

</span>
<span class="ltx_bibblock">ACM Transactions on Intelligent Systems and Technology (TIST) <em id="bib.bib48.1.1" class="ltx_emph ltx_font_italic">13</em>, 1â€“23.

</span>
</li>
<li id="bib.bib49" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhang etÂ al. 2023a</span>
<span class="ltx_bibblock">
Zhang, R., Fan, Z., Xu, Q., Yao, J., Zhang, Y., and Wang, Y.

</span>
<span class="ltx_bibblock">Grace: A generalized and personalized federated learning method for medical imaging.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib49.1.1" class="ltx_emph ltx_font_italic">International Conference on Medical Image Computing and Computer-Assisted Intervention</em>. Springer (2023a):( 14â€“24).

</span>
</li>
<li id="bib.bib50" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Madni etÂ al. 2023</span>
<span class="ltx_bibblock">
Madni, H.Â A., Umer, R.Â M., and Foresti, G.Â L.

</span>
<span class="ltx_bibblock">Federated learning for data and model heterogeneity in medical imaging.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib50.1.1" class="ltx_emph ltx_font_italic">International Conference on Image Analysis and Processing</em>. Springer (2023):( 167â€“178).

</span>
</li>
<li id="bib.bib51" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gong etÂ al. 2021</span>
<span class="ltx_bibblock">
Gong, X., Sharma, A., Karanam, S., Wu, Z., Chen, T., Doermann, D., and Innanje, A.

</span>
<span class="ltx_bibblock">Ensemble attention distillation for privacy-preserving federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib51.1.1" class="ltx_emph ltx_font_italic">Proceedings of the IEEE/CVF International Conference on Computer Vision</em> (2021):( 15076â€“15086).

</span>
</li>
<li id="bib.bib52" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">OgierÂ du Terrail etÂ al. 2022</span>
<span class="ltx_bibblock">
OgierÂ du Terrail, J., Ayed, S.-S., Cyffers, E., Grimberg, F., He, C., Loeb, R., Mangold, P., Marchand, T., Marfoq, O., Mushtaq, E. etÂ al. (2022). Flamby: Datasets and benchmarks for cross-silo federated learning in realistic healthcare settings.

</span>
<span class="ltx_bibblock">Advances in Neural Information Processing Systems <em id="bib.bib52.1.1" class="ltx_emph ltx_font_italic">35</em>, 5315â€“5334.

</span>
</li>
<li id="bib.bib53" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhang etÂ al. 2024</span>
<span class="ltx_bibblock">
Zhang, F., Shuai, Z., Kuang, K., Wu, F., Zhuang, Y., and Xiao, J. (2024). Unified fair federated learning for digital healthcare.

</span>
<span class="ltx_bibblock">Patterns <em id="bib.bib53.1.1" class="ltx_emph ltx_font_italic">5</em>.

</span>
</li>
<li id="bib.bib54" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Souza etÂ al. 2021</span>
<span class="ltx_bibblock">
Souza, R., Tuladhar, A., Mouches, P., Wilms, M., Tyagi, L., and Forkert, N.Â D.

</span>
<span class="ltx_bibblock">Multi-institutional travelling model for tumor segmentation in mri datasets.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib54.1.1" class="ltx_emph ltx_font_italic">International MICCAI Brainlesion Workshop</em>. Springer (2021):( 420â€“432).

</span>
</li>
<li id="bib.bib55" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Tedeschini etÂ al. 2022</span>
<span class="ltx_bibblock">
Tedeschini, B.Â C., Savazzi, S., Stoklasa, R., Barbieri, L., Stathopoulos, I., Nicoli, M., and Serio, L. (2022). Decentralized federated learning for healthcare networks: A case study on tumor segmentation.

</span>
<span class="ltx_bibblock">IEEE access <em id="bib.bib55.1.1" class="ltx_emph ltx_font_italic">10</em>, 8693â€“8708.

</span>
</li>
<li id="bib.bib56" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lin etÂ al. 2023</span>
<span class="ltx_bibblock">
Lin, L., Wu, J., Liu, Y., Wong, K.Â K., and Tang, X.

</span>
<span class="ltx_bibblock">Unifying and personalizing weakly-supervised federated medical image segmentation via adaptive representation and aggregation.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib56.1.1" class="ltx_emph ltx_font_italic">International Workshop on Machine Learning in Medical Imaging</em>. Springer (2023):( 196â€“206).

</span>
</li>
<li id="bib.bib57" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Alam and Rahmani 2023</span>
<span class="ltx_bibblock">
Alam, M.Â U., and Rahmani, R. (2023). Fedsepsis: A federated multi-modal deep learning-based internet of medical things application for early detection of sepsis from electronic health records using raspberry pi and jetson nano devices.

</span>
<span class="ltx_bibblock">Sensors <em id="bib.bib57.1.1" class="ltx_emph ltx_font_italic">23</em>, 970.

</span>
</li>
<li id="bib.bib58" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Baheti etÂ al. 2020</span>
<span class="ltx_bibblock">
Baheti, P., Sikka, M., Arya, K., and Rajesh, R.

</span>
<span class="ltx_bibblock">Federated learning on distributed medical records for detection of lung nodules.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib58.1.1" class="ltx_emph ltx_font_italic">VISIGRAPP (4: VISAPP)</em> (2020):( 445â€“451).

</span>
</li>
<li id="bib.bib59" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhang etÂ al. 2021</span>
<span class="ltx_bibblock">
Zhang, W., Zhou, T., Lu, Q., Wang, X., Zhu, C., Sun, H., Wang, Z., Lo, S.Â K., and Wang, F.-Y. (2021). Dynamic-fusion-based federated learning for covid-19 detection.

</span>
<span class="ltx_bibblock">IEEE Internet of Things Journal <em id="bib.bib59.1.1" class="ltx_emph ltx_font_italic">8</em>, 15884â€“15891.

</span>
</li>
<li id="bib.bib60" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sakib etÂ al. 2021</span>
<span class="ltx_bibblock">
Sakib, S., Fouda, M.Â M., Fadlullah, Z.Â M., Abualsaud, K., Yaacoub, E., and Guizani, M.

</span>
<span class="ltx_bibblock">Asynchronous federated learning-based ecg analysis for arrhythmia detection.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib60.1.1" class="ltx_emph ltx_font_italic">2021 IEEE International Mediterranean Conference on Communications and Networking (MeditCom)</em>. IEEE (2021):( 277â€“282).

</span>
</li>
<li id="bib.bib61" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kerkouche etÂ al. 2021</span>
<span class="ltx_bibblock">
Kerkouche, R., Acs, G., Castelluccia, C., and GenevÃ¨s, P.

</span>
<span class="ltx_bibblock">Privacy-preserving and bandwidth-efficient federated learning: An application to in-hospital mortality prediction.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib61.1.1" class="ltx_emph ltx_font_italic">Proceedings of the conference on health, inference, and learning</em> (2021):( 25â€“35).

</span>
</li>
<li id="bib.bib62" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Soltan etÂ al. 2024</span>
<span class="ltx_bibblock">
Soltan, A.Â A., Thakur, A., Yang, J., Chauhan, A., Dâ€™Cruz, L.Â G., Dickson, P., Soltan, M.Â A., Thickett, D.Â R., Eyre, D.Â W., Zhu, T. etÂ al. (2024). A scalable federated learning solution for secondary care using low-cost microcomputing: privacy-preserving development and evaluation of a covid-19 screening test in uk hospitals.

</span>
<span class="ltx_bibblock">The Lancet Digital Health <em id="bib.bib62.1.1" class="ltx_emph ltx_font_italic">6</em>, e93â€“e104.

</span>
</li>
<li id="bib.bib63" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lian etÂ al. 2022</span>
<span class="ltx_bibblock">
Lian, Z., Yang, Q., Wang, W., Zeng, Q., Alazab, M., Zhao, H., and Su, C. (2022). Deep-fel: Decentralized, efficient and privacy-enhanced federated edge learning for healthcare cyber physical systems.

</span>
<span class="ltx_bibblock">IEEE Transactions on Network Science and Engineering <em id="bib.bib63.1.1" class="ltx_emph ltx_font_italic">9</em>, 3558â€“3569.

</span>
</li>
<li id="bib.bib64" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhang etÂ al. 2022</span>
<span class="ltx_bibblock">
Zhang, L., Xu, J., Vijayakumar, P., Sharma, P.Â K., and Ghosh, U. (2022). Homomorphic encryption-based privacy-preserving federated learning in iot-enabled healthcare system.

</span>
<span class="ltx_bibblock">IEEE Transactions on Network Science and Engineering <em id="bib.bib64.1.1" class="ltx_emph ltx_font_italic">10</em>, 2864â€“2880.

</span>
</li>
<li id="bib.bib65" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hatamizadeh etÂ al. 2023</span>
<span class="ltx_bibblock">
Hatamizadeh, A., Yin, H., Molchanov, P., Myronenko, A., Li, W., Dogra, P., Feng, A., Flores, M.Â G., Kautz, J., Xu, D. etÂ al. (2023). Do gradient inversion attacks make federated learning unsafe?

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging <em id="bib.bib65.1.1" class="ltx_emph ltx_font_italic">42</em>, 2044â€“2056.

</span>
</li>
<li id="bib.bib66" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Truhn etÂ al. 2024</span>
<span class="ltx_bibblock">
Truhn, D., Arasteh, S.Â T., Saldanha, O.Â L., MÃ¼ller-Franzes, G., Khader, F., Quirke, P., West, N.Â P., Gray, R., Hutchins, G.Â G., James, J.Â A. etÂ al. (2024). Encrypted federated learning for secure decentralized collaboration in cancer image analysis.

</span>
<span class="ltx_bibblock">Medical image analysis <em id="bib.bib66.1.1" class="ltx_emph ltx_font_italic">92</em>, 103059.

</span>
</li>
<li id="bib.bib67" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Jiang etÂ al. 2022</span>
<span class="ltx_bibblock">
Jiang, M., Wang, Z., and Dou, Q.

</span>
<span class="ltx_bibblock">Harmofl: Harmonizing local and global drifts in federated learning on heterogeneous medical images.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib67.1.1" class="ltx_emph ltx_font_italic">Proceedings of the AAAI Conference on Artificial Intelligence</em> vol.Â 36 (2022):( 1087â€“1095).

</span>
</li>
<li id="bib.bib68" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Warnat-Herresthal etÂ al. 2021</span>
<span class="ltx_bibblock">
Warnat-Herresthal, S., Schultze, H., Shastry, K.Â L., Manamohan, S., Mukherjee, S., Garg, V., Sarveswara, R., HÃ¤ndler, K., Pickkers, P., Aziz, N.Â A. etÂ al. (2021). Swarm learning for decentralized and confidential clinical machine learning.

</span>
<span class="ltx_bibblock">Nature <em id="bib.bib68.1.1" class="ltx_emph ltx_font_italic">594</em>, 265â€“270.

</span>
</li>
<li id="bib.bib69" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Froelicher etÂ al. 2021</span>
<span class="ltx_bibblock">
Froelicher, D., Troncoso-Pastoriza, J.Â R., Raisaro, J.Â L., Cuendet, M.Â A., Sousa, J.Â S., Cho, H., Berger, B., Fellay, J., and Hubaux, J.-P. (2021). Truly privacy-preserving federated analytics for precision medicine with multiparty homomorphic encryption.

</span>
<span class="ltx_bibblock">Nature communications <em id="bib.bib69.1.1" class="ltx_emph ltx_font_italic">12</em>, 5910.

</span>
</li>
<li id="bib.bib70" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Repetto and LaÂ Torre 2022</span>
<span class="ltx_bibblock">
Repetto, M., and LaÂ Torre, D.

</span>
<span class="ltx_bibblock">Federated learning through goal programming: a computational study in cancer detection.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib70.1.1" class="ltx_emph ltx_font_italic">2022 5th International Conference on Signal Processing and Information Security (ICSPIS)</em>. IEEE (2022):( 80â€“85).

</span>
</li>
<li id="bib.bib71" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Balkus etÂ al. 2022</span>
<span class="ltx_bibblock">
Balkus, S.Â V., Fang, H., and Wang, H.

</span>
<span class="ltx_bibblock">Federated fuzzy clustering for longitudinal health data.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib71.1.1" class="ltx_emph ltx_font_italic">2022 IEEE/ACM Conference on Connected Health: Applications, Systems and Engineering Technologies (CHASE)</em>. IEEE (2022):( 128â€“132).

</span>
</li>
<li id="bib.bib72" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kalapaaking etÂ al. 2023</span>
<span class="ltx_bibblock">
Kalapaaking, A.Â P., Khalil, I., and Yi, X. (2023). Blockchain-based federated learning with smpc model verification against poisoning attack for healthcare systems.

</span>
<span class="ltx_bibblock">IEEE Transactions on Emerging Topics in Computing <em id="bib.bib72.1.1" class="ltx_emph ltx_font_italic">12</em>, 269â€“280.

</span>
</li>
<li id="bib.bib73" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Salim etÂ al. 2024</span>
<span class="ltx_bibblock">
Salim, M.Â M., Sangthong, Y., Deng, X., and Park, J.Â H.

</span>
<span class="ltx_bibblock">Federated learning-enabled zero-day ddos attack detection scheme in healthcare 4.0 (2024).

</span>
</li>
<li id="bib.bib74" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Rehman etÂ al. 2022</span>
<span class="ltx_bibblock">
Rehman, A., Abbas, S., Khan, M., Ghazal, T.Â M., Adnan, K.Â M., and Mosavi, A. (2022). A secure healthcare 5.0 system based on blockchain technology entangled with federated learning technique.

</span>
<span class="ltx_bibblock">Computers in Biology and Medicine <em id="bib.bib74.1.1" class="ltx_emph ltx_font_italic">150</em>, 106019.

</span>
</li>
<li id="bib.bib75" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chang etÂ al. 2021</span>
<span class="ltx_bibblock">
Chang, Y., Fang, C., and Sun, W. (2021). A blockchain-based federated learning method for smart healthcare.

</span>
<span class="ltx_bibblock">Computational Intelligence and Neuroscience <em id="bib.bib75.1.1" class="ltx_emph ltx_font_italic">2021</em>.

</span>
</li>
<li id="bib.bib76" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Bercea etÂ al. 2021</span>
<span class="ltx_bibblock">
Bercea, C.Â I., Wiestler, B., Rueckert, D., and Albarqouni, S. (2021). Feddis: Disentangled federated learning for unsupervised brain pathology segmentation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2103.03705.

</span>
</li>
<li id="bib.bib77" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chen etÂ al. 2020a</span>
<span class="ltx_bibblock">
Chen, Y., Qin, X., Wang, J., Yu, C., and Gao, W. (2020a). Fedhealth: A federated transfer learning framework for wearable healthcare.

</span>
<span class="ltx_bibblock">IEEE Intelligent Systems <em id="bib.bib77.1.1" class="ltx_emph ltx_font_italic">35</em>, 83â€“93.

</span>
</li>
<li id="bib.bib78" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mullie etÂ al. 2024</span>
<span class="ltx_bibblock">
Mullie, L., Afilalo, J., Archambault, P., Bouchakri, R., Brown, K., Buckeridge, D.Â L., Cavayas, Y.Â A., Turgeon, A.Â F., Martineau, D., Lamontagne, F. etÂ al. (2024). Coda: an open-source platform for federated analysis and machine learning on distributed healthcare data.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association <em id="bib.bib78.1.1" class="ltx_emph ltx_font_italic">31</em>, 651â€“665.

</span>
</li>
<li id="bib.bib79" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Roth etÂ al. 2022</span>
<span class="ltx_bibblock">
Roth, H.Â R., Cheng, Y., Wen, Y., Yang, I., Xu, Z., Hsieh, Y.-T., Kersten, K., Harouni, A., Zhao, C., Lu, K. etÂ al. (2022). Nvidia flare: Federated learning from simulation to real-world.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2210.13291.

</span>
</li>
<li id="bib.bib80" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Cremonesi etÂ al. 2023</span>
<span class="ltx_bibblock">
Cremonesi, F., Vesin, M., Cansiz, S., Bouillard, Y., Balelli, I., Innocenti, L., Silva, S., Ayed, S.-S., Taiello, R., Kameni, L. etÂ al. (2023). Fed-biomed: open, transparent and trusted federated learning for real-world healthcare applications.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2304.12012.

</span>
</li>
<li id="bib.bib81" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chen etÂ al. 2020b</span>
<span class="ltx_bibblock">
Chen, H., Li, H., Xu, G., Zhang, Y., and Luo, X.

</span>
<span class="ltx_bibblock">Achieving privacy-preserving federated learning with irrelevant updates over e-health applications.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib81.1.1" class="ltx_emph ltx_font_italic">ICC 2020-2020 IEEE international conference on communications (ICC)</em>. IEEE (2020b):( 1â€“6).

</span>
</li>
<li id="bib.bib82" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2021a</span>
<span class="ltx_bibblock">
Li, J., Meng, Y., Ma, L., Du, S., Zhu, H., Pei, Q., and Shen, X. (2021a). A federated learning based privacy-preserving smart healthcare system.

</span>
<span class="ltx_bibblock">IEEE Transactions on Industrial Informatics <em id="bib.bib82.1.1" class="ltx_emph ltx_font_italic">18</em>.

</span>
</li>
<li id="bib.bib83" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sav etÂ al. 2022</span>
<span class="ltx_bibblock">
Sav, S., Bossuat, J.-P., Troncoso-Pastoriza, J.Â R., Claassen, M., and Hubaux, J.-P. (2022). Privacy-preserving federated neural network learning for disease-associated cell classification.

</span>
<span class="ltx_bibblock">Patterns <em id="bib.bib83.1.1" class="ltx_emph ltx_font_italic">3</em>.

</span>
</li>
<li id="bib.bib84" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Bey etÂ al. 2020</span>
<span class="ltx_bibblock">
Bey, R., Goussault, R., Grolleau, F., Benchoufi, M., and Porcher, R. (2020). Fold-stratified cross-validation for unbiased and privacy-preserving federated learning.

</span>
<span class="ltx_bibblock">Journal of the American Medical Informatics Association <em id="bib.bib84.1.1" class="ltx_emph ltx_font_italic">27</em>, 1244â€“1251.

</span>
</li>
<li id="bib.bib85" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Rehman etÂ al. 2024</span>
<span class="ltx_bibblock">
Rehman, A., Xing, H., Feng, L., Hussain, M., Gulzar, N., Khan, M.Â A., Hussain, A., and Saeed, D. (2024). Fedcscd-gan: A secure and collaborative framework for clinical cancer diagnosis via optimized federated learning and gan.

</span>
<span class="ltx_bibblock">Biomedical Signal Processing and Control <em id="bib.bib85.1.1" class="ltx_emph ltx_font_italic">89</em>, 105893.

</span>
</li>
<li id="bib.bib86" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lakhan etÂ al. 2024</span>
<span class="ltx_bibblock">
Lakhan, A., Hamouda, H., Abdulkareem, K.Â H., Alyahya, S., and Mohammed, M.Â A. (2024). Digital healthcare framework for patients with disabilities based on deep federated learning schemes.

</span>
<span class="ltx_bibblock">Computers in Biology and Medicine <em id="bib.bib86.1.1" class="ltx_emph ltx_font_italic">169</em>, 107845.

</span>
</li>
<li id="bib.bib87" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhou etÂ al. 2024b</span>
<span class="ltx_bibblock">
Zhou, L., Wang, M., and Zhou, N. (2024b). Distributed federated learning-based deep learning model for privacy mri brain tumor detection.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2404.10026.

</span>
</li>
<li id="bib.bib88" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Yaqoob etÂ al. 2023</span>
<span class="ltx_bibblock">
Yaqoob, M.Â M., Alsulami, M., Khan, M.Â A., Alsadie, D., Saudagar, A. K.Â J., and AlKhathami, M. (2023). Federated machine learning for skin lesion diagnosis: an asynchronous and weighted approach.

</span>
<span class="ltx_bibblock">Diagnostics <em id="bib.bib88.1.1" class="ltx_emph ltx_font_italic">13</em>, 1964.

</span>
</li>
<li id="bib.bib89" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Feng etÂ al. 2024</span>
<span class="ltx_bibblock">
Feng, B., Shi, J., Huang, L., Yang, Z., Feng, S.-T., Li, J., Chen, Q., Xue, H., Chen, X., Wan, C. etÂ al. (2024). Robustly federated learning model for identifying high-risk patients with postoperative gastric cancer recurrence.

</span>
<span class="ltx_bibblock">Nature Communications <em id="bib.bib89.1.1" class="ltx_emph ltx_font_italic">15</em>, 742.

</span>
</li>
<li id="bib.bib90" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Andreux etÂ al. 2020a</span>
<span class="ltx_bibblock">
Andreux, M., Manoel, A., Menuet, R., Saillard, C., and Simpson, C. (2020a). Federated survival analysis with discrete-time cox models.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2006.08997.

</span>
</li>
<li id="bib.bib91" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Tong etÂ al. 2022</span>
<span class="ltx_bibblock">
Tong, J., Luo, C., Islam, M.Â N., Sheils, N.Â E., Buresh, J., Edmondson, M., Merkel, P.Â A., Lautenbach, E., Duan, R., and Chen, Y. (2022). Distributed learning for heterogeneous clinical data with application to integrating covid-19 data across 230 sites.

</span>
<span class="ltx_bibblock">NPJ digital medicine <em id="bib.bib91.1.1" class="ltx_emph ltx_font_italic">5</em>, 76.

</span>
</li>
<li id="bib.bib92" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhou etÂ al. 2024c</span>
<span class="ltx_bibblock">
Zhou, X., Huang, W., Liang, W., Yan, Z., Ma, J., Pan, Y., Kevin, I., and Wang, K. (2024c). Federated distillation and blockchain empowered secure knowledge sharing for internet of medical things.

</span>
<span class="ltx_bibblock">Information Sciences <em id="bib.bib92.1.1" class="ltx_emph ltx_font_italic">662</em>, 120217.

</span>
</li>
<li id="bib.bib93" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mateus etÂ al. 2024</span>
<span class="ltx_bibblock">
Mateus, P., Moonen, J., Beran, M., Jaarsma, E., vanÂ der Landen, S.Â M., Heuvelink, J., Birhanu, M., Harms, A.Â G., Bron, E., Wolters, F.Â J. etÂ al. (2024). Data harmonization and federated learning for multi-cohort dementia research using the omop common data model: A netherlands consortium of dementia cohorts case study.

</span>
<span class="ltx_bibblock">Journal of biomedical informatics ( 104661).

</span>
</li>
<li id="bib.bib94" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mazher etÂ al. 2024</span>
<span class="ltx_bibblock">
Mazher, M., Razzak, I., Qayyum, A., Tanveer, M., Beier, S., Khan, T., and Niederer, S.Â A. (2024). Self-supervised spatial-temporal transformer fusion based federated framework for 4d cardiovascular image segmentation.

</span>
<span class="ltx_bibblock">Information Fusion <em id="bib.bib94.1.1" class="ltx_emph ltx_font_italic">106</em>, 102256.

</span>
</li>
<li id="bib.bib95" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ma etÂ al. 2021a</span>
<span class="ltx_bibblock">
Ma, J., Zhang, Q., Lou, J., Xiong, L., and Ho, J.Â C.

</span>
<span class="ltx_bibblock">Communication efficient federated generalized tensor factorization for collaborative health data analytics.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib95.1.1" class="ltx_emph ltx_font_italic">Proceedings of the Web Conference 2021</em> (2021a):( 171â€“182).

</span>
</li>
<li id="bib.bib96" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li and Yang 2024</span>
<span class="ltx_bibblock">
Li, M., and Yang, G.

</span>
<span class="ltx_bibblock">Where to begin? from random to foundation model instructed initialization in federated learning for medical image segmentation.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib96.1.1" class="ltx_emph ltx_font_italic">2024 IEEE International Symposium on Biomedical Imaging (ISBI)</em>. IEEE (2024):( 1â€“5).

</span>
</li>
<li id="bib.bib97" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wang etÂ al. 2022</span>
<span class="ltx_bibblock">
Wang, R., Lai, J., Zhang, Z., Li, X., Vijayakumar, P., and Karuppiah, M. (2022). Privacy-preserving federated learning for internet of medical things under edge computing.

</span>
<span class="ltx_bibblock">IEEE journal of biomedical and health informatics <em id="bib.bib97.1.1" class="ltx_emph ltx_font_italic">27</em>, 854â€“865.

</span>
</li>
<li id="bib.bib98" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Qu etÂ al. 2022</span>
<span class="ltx_bibblock">
Qu, L., Balachandar, N., Zhang, M., and Rubin, D. (2022). Handling data heterogeneity with generative replay in collaborative learning for medical imaging.

</span>
<span class="ltx_bibblock">Medical image analysis <em id="bib.bib98.1.1" class="ltx_emph ltx_font_italic">78</em>, 102424.

</span>
</li>
<li id="bib.bib99" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Xie etÂ al. 2024</span>
<span class="ltx_bibblock">
Xie, L., Lin, M., Luan, T., Li, C., Fang, Y., Shen, Q., and Wu, Z.

</span>
<span class="ltx_bibblock">MH-pFLID: Model heterogeneous personalized federated learning via injection and distillation for medical data analysis.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib99.1.1" class="ltx_emph ltx_font_italic">Forty-first International Conference on Machine Learning</em> (2024):.

</span>
</li>
<li id="bib.bib100" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2022c</span>
<span class="ltx_bibblock">
Li, Z., Xu, X., Cao, X., Liu, W., Zhang, Y., Chen, D., and Dai, H. (2022c). Integrated cnn and federated learning for covid-19 detection on chest x-ray images.

</span>
<span class="ltx_bibblock">IEEE/ACM Transactions on Computational Biology and Bioinformatics.

</span>
</li>
<li id="bib.bib101" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gawali etÂ al. 2021</span>
<span class="ltx_bibblock">
Gawali, M., Arvind, C., Suryavanshi, S., Madaan, H., Gaikwad, A., BhanuÂ Prakash, K., Kulkarni, V., and Pant, A.

</span>
<span class="ltx_bibblock">Comparison of privacy-preserving distributed deep learning methods in healthcare.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib101.1.1" class="ltx_emph ltx_font_italic">Medical Image Understanding and Analysis: 25th Annual Conference, MIUA 2021, Oxford, United Kingdom, July 12â€“14, 2021, Proceedings 25</em>. Springer (2021):( 457â€“471).

</span>
</li>
<li id="bib.bib102" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Paragliola and Coronato 2022</span>
<span class="ltx_bibblock">
Paragliola, G., and Coronato, A. (2022). Definition of a novel federated learning approach to reduce communication costs.

</span>
<span class="ltx_bibblock">Expert Systems with Applications <em id="bib.bib102.1.1" class="ltx_emph ltx_font_italic">189</em>, 116109.

</span>
</li>
<li id="bib.bib103" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kandati and Gadekallu 2022</span>
<span class="ltx_bibblock">
Kandati, D.Â R., and Gadekallu, T.Â R. (2022). Genetic clustered federated learning for covid-19 detection.

</span>
<span class="ltx_bibblock">Electronics <em id="bib.bib103.1.1" class="ltx_emph ltx_font_italic">11</em>, 2714.

</span>
</li>
<li id="bib.bib104" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Brisimi etÂ al. 2018</span>
<span class="ltx_bibblock">
Brisimi, T.Â S., Chen, R., Mela, T., Olshevsky, A., Paschalidis, I.Â C., and Shi, W. (2018). Federated learning of predictive models from federated electronic health records.

</span>
<span class="ltx_bibblock">International journal of medical informatics <em id="bib.bib104.1.1" class="ltx_emph ltx_font_italic">112</em>, 59â€“67.

</span>
</li>
<li id="bib.bib105" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Aminifar etÂ al. 2022</span>
<span class="ltx_bibblock">
Aminifar, A., Shokri, M., Rabbi, F., Pun, V. K.Â I., and Lamo, Y. (2022). Extremely randomized trees with privacy preservation for distributed structured health data.

</span>
<span class="ltx_bibblock">IEEE Access <em id="bib.bib105.1.1" class="ltx_emph ltx_font_italic">10</em>, 6010â€“6027.

</span>
</li>
<li id="bib.bib106" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Adnan etÂ al. 2022</span>
<span class="ltx_bibblock">
Adnan, M., Kalra, S., Cresswell, J.Â C., Taylor, G.Â W., and Tizhoosh, H.Â R. (2022). Federated learning and differential privacy for medical image analysis.

</span>
<span class="ltx_bibblock">Scientific reports <em id="bib.bib106.1.1" class="ltx_emph ltx_font_italic">12</em>, 1953.

</span>
</li>
<li id="bib.bib107" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ma etÂ al. 2021b</span>
<span class="ltx_bibblock">
Ma, J., Zhang, Q., Lou, J., Xiong, L., Bhavani, S., and Ho, J.Â C.

</span>
<span class="ltx_bibblock">Communication efficient tensor factorization for decentralized healthcare networks.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib107.1.1" class="ltx_emph ltx_font_italic">2021 IEEE International Conference on Data Mining (ICDM)</em>. IEEE (2021b):( 1216â€“1221).

</span>
</li>
<li id="bib.bib108" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ziller etÂ al. 2021</span>
<span class="ltx_bibblock">
Ziller, A., Usynin, D., Remerscheid, N., Knolle, M., Makowski, M., Braren, R., Rueckert, D., and Kaissis, G. (2021). Differentially private federated deep learning for multi-site medical image segmentation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2107.02586.

</span>
</li>
<li id="bib.bib109" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hosseini etÂ al. 2023</span>
<span class="ltx_bibblock">
Hosseini, S.Â M., Sikaroudi, M., Babaie, M., and Tizhoosh, H.Â R. (2023). Proportionally fair hospital collaborations in federated learning of histopathology images.

</span>
<span class="ltx_bibblock">IEEE transactions on medical imaging <em id="bib.bib109.1.1" class="ltx_emph ltx_font_italic">42</em>, 1982â€“1995.

</span>
</li>
<li id="bib.bib110" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">He etÂ al. 2020</span>
<span class="ltx_bibblock">
He, C., Li, S., So, J., Zeng, X., Zhang, M., Wang, H., Wang, X., Vepakomma, P., Singh, A., Qiu, H. etÂ al. (2020). Fedml: A research library and benchmark for federated machine learning.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2007.13518.

</span>
</li>
<li id="bib.bib111" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Xie etÂ al. 2022</span>
<span class="ltx_bibblock">
Xie, Y., Wang, Z., Gao, D., Chen, D., Yao, L., Kuang, W., Li, Y., Ding, B., and Zhou, J. (2022). Federatedscope: A flexible federated learning platform for heterogeneity.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2204.05011.

</span>
</li>
<li id="bib.bib112" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Beutel etÂ al. 2022</span>
<span class="ltx_bibblock">
Beutel, D.Â J., Topal, T., Mathur, A., Qiu, X., Fernandez-Marques, J., Gao, Y., Sani, L., Li, K.Â H., Parcollet, T., deÂ GusmÃ£o, P. P.Â B. etÂ al. (2022). Flower: A friendly federated learning framework.

</span>
</li>
<li id="bib.bib113" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liu etÂ al. 2021c</span>
<span class="ltx_bibblock">
Liu, Y., Fan, T., Chen, T., Xu, Q., and Yang, Q. (2021c). Fate: An industrial grade platform for collaborative learning with data protection.

</span>
<span class="ltx_bibblock">Journal of Machine Learning Research <em id="bib.bib113.1.1" class="ltx_emph ltx_font_italic">22</em>, 1â€“6.

</span>
</li>
<li id="bib.bib114" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">OWKIN 2024</span>
<span class="ltx_bibblock">
OWKIN.

</span>
<span class="ltx_bibblock">SubstraFL Overview (2024).

</span>
<span class="ltx_bibblock">URL: <a target="_blank" href="https://docs.substra.org/en/stable/substrafl_doc/substrafl_overview.html" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://docs.substra.org/en/stable/substrafl_doc/substrafl_overview.html</a>.

</span>
</li>
<li id="bib.bib115" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ryffel etÂ al. 2018</span>
<span class="ltx_bibblock">
Ryffel, T., Trask, A., Dahl, M., Wagner, B., Mancuso, J., Rueckert, D., and Passerat-Palmbach, J. (2018). A generic framework for privacy preserving deep learning.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:1811.04017.

</span>
</li>
<li id="bib.bib116" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Foley etÂ al. 2022</span>
<span class="ltx_bibblock">
Foley, P., Sheller, M.Â J., Edwards, B., Pati, S., Riviera, W., Sharma, M., Moorthy, P.Â N., Wang, S.-h., Martin, J., Mirhaji, P. etÂ al. (2022). Openfl: the open federated learning library.

</span>
<span class="ltx_bibblock">Physics in Medicine &amp; Biology <em id="bib.bib116.1.1" class="ltx_emph ltx_font_italic">67</em>, 214001.

</span>
</li>
<li id="bib.bib117" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Bonawitz etÂ al. 2019</span>
<span class="ltx_bibblock">
Bonawitz, K., Eichner, H., Grieskamp, W., Huba, D., Ingerman, A., Ivanov, V., Kiddon, C., KoneÄná»³, J., Mazzocchi, S., McMahan, B. etÂ al. (2019). Towards federated learning at scale: System design.

</span>
<span class="ltx_bibblock">Proceedings of machine learning and systems <em id="bib.bib117.1.1" class="ltx_emph ltx_font_italic">1</em>, 374â€“388.

</span>
</li>
<li id="bib.bib118" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ludwig etÂ al. 2020</span>
<span class="ltx_bibblock">
Ludwig, H., Baracaldo, N., Thomas, G., Zhou, Y., Anwar, A., Rajamoni, S., Ong, Y., Radhakrishnan, J., Verma, A., Sinn, M. etÂ al. (2020). Ibm federated learning: an enterprise framework white paper v0. 1.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2007.10987.

</span>
</li>
<li id="bib.bib119" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ma etÂ al. 2019</span>
<span class="ltx_bibblock">
Ma, Y., Yu, D., Wu, T., and Wang, H. (2019). Paddlepaddle: An open-source deep learning platform from industrial practice.

</span>
<span class="ltx_bibblock">Frontiers of Data and Domputing <em id="bib.bib119.1.1" class="ltx_emph ltx_font_italic">1</em>, 105â€“115.

</span>
</li>
<li id="bib.bib120" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gehlhar etÂ al. 2023</span>
<span class="ltx_bibblock">
Gehlhar, T., Marx, F., Schneider, T., Suresh, A., Wehrle, T., and Yalame, H.

</span>
<span class="ltx_bibblock">Safefl: Mpc-friendly framework for private and robust federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib120.1.1" class="ltx_emph ltx_font_italic">2023 IEEE Security and Privacy Workshops (SPW)</em>. IEEE (2023):( 69â€“76).

</span>
</li>
<li id="bib.bib121" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2023b</span>
<span class="ltx_bibblock">
Li, H., Li, C., Wang, J., Yang, A., Ma, Z., Zhang, Z., and Hua, D. (2023b). Review on security of federated learning and its application in healthcare.

</span>
<span class="ltx_bibblock">Future Generation Computer Systems <em id="bib.bib121.1.1" class="ltx_emph ltx_font_italic">144</em>, 271â€“290.

</span>
</li>
<li id="bib.bib122" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lo etÂ al. 2022</span>
<span class="ltx_bibblock">
Lo, S.Â K., Lu, Q., Zhu, L., Paik, H.-Y., Xu, X., and Wang, C. (2022). Architectural patterns for the design of federated learning systems.

</span>
<span class="ltx_bibblock">Journal of Systems and Software <em id="bib.bib122.1.1" class="ltx_emph ltx_font_italic">191</em>, 111357.

</span>
</li>
<li id="bib.bib123" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lai etÂ al. 2022</span>
<span class="ltx_bibblock">
Lai, F., Dai, Y., Singapuram, S., Liu, J., Zhu, X., Madhyastha, H., and Chowdhury, M.

</span>
<span class="ltx_bibblock">Fedscale: Benchmarking model and system performance of federated learning at scale.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib123.1.1" class="ltx_emph ltx_font_italic">International conference on machine learning</em>. PMLR (2022):( 11814â€“11827).

</span>
</li>
<li id="bib.bib124" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Petersmann etÂ al. 2019</span>
<span class="ltx_bibblock">
Petersmann, A., MÃ¼ller-Wieland, D., MÃ¼ller, U.Â A., Landgraf, R., Nauck, M., Freckmann, G., Heinemann, L., and Schleicher, E. (2019). Definition, classification and diagnosis of diabetes mellitus.

</span>
<span class="ltx_bibblock">Experimental and Clinical Endocrinology &amp; Diabetes <em id="bib.bib124.1.1" class="ltx_emph ltx_font_italic">127</em>, S1â€“S7.

</span>
</li>
<li id="bib.bib125" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gad and Fadlullah 2022</span>
<span class="ltx_bibblock">
Gad, G., and Fadlullah, Z. (2022). Federated learning via augmented knowledge distillation for heterogenous deep human activity recognition systems.

</span>
<span class="ltx_bibblock">Sensors <em id="bib.bib125.1.1" class="ltx_emph ltx_font_italic">23</em>, 6.

</span>
</li>
<li id="bib.bib126" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Shaik etÂ al. 2022</span>
<span class="ltx_bibblock">
Shaik, T., Tao, X., Higgins, N., Gururajan, R., Li, Y., Zhou, X., and Acharya, U.Â R. (2022). Fedstack: Personalized activity monitoring using stacked federated learning.

</span>
<span class="ltx_bibblock">Knowledge-Based Systems <em id="bib.bib126.1.1" class="ltx_emph ltx_font_italic">257</em>, 109929.

</span>
</li>
<li id="bib.bib127" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lee etÂ al. 2023</span>
<span class="ltx_bibblock">
Lee, G.Â H., Park, J., Kim, J., Kim, Y., Choi, B., Park, R.Â W., Rhee, S.Â Y., and Shin, S.-Y. (2023). Feasibility study of federated learning on the distributed research network of omop common data model.

</span>
<span class="ltx_bibblock">Healthcare Informatics Research <em id="bib.bib127.1.1" class="ltx_emph ltx_font_italic">29</em>, 168â€“173.

</span>
</li>
<li id="bib.bib128" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kaissis etÂ al. 2021</span>
<span class="ltx_bibblock">
Kaissis, G., Ziller, A., Passerat-Palmbach, J., Ryffel, T., Usynin, D., Trask, A., LimaÂ Jr, I., Mancuso, J., Jungmann, F., Steinborn, M.-M. etÂ al. (2021). End-to-end privacy preserving deep learning on multi-institutional medical imaging.

</span>
<span class="ltx_bibblock">Nature Machine Intelligence <em id="bib.bib128.1.1" class="ltx_emph ltx_font_italic">3</em>, 473â€“484.

</span>
</li>
<li id="bib.bib129" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Andreux etÂ al. 2020b</span>
<span class="ltx_bibblock">
Andreux, M., duÂ Terrail, J.Â O., Beguier, C., and Tramel, E.Â W.

</span>
<span class="ltx_bibblock">Siloed federated learning for multi-centric histopathology datasets.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib129.1.1" class="ltx_emph ltx_font_italic">Domain Adaptation and Representation Transfer, and Distributed and Collaborative Learning: Second MICCAI Workshop, DART 2020, and First MICCAI Workshop, DCL 2020, Held in Conjunction with MICCAI 2020, Lima, Peru, October 4â€“8, 2020, Proceedings 2</em>. Springer (2020b):( 129â€“139).

</span>
</li>
<li id="bib.bib130" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Philippenko and Dieuleveut 2020</span>
<span class="ltx_bibblock">
Philippenko, C., and Dieuleveut, A. (2020). Bidirectional compression in heterogeneous settings for distributed or federated learning with partial participation: tight convergence guarantees.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2006.14591.

</span>
</li>
<li id="bib.bib131" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chen etÂ al. 2024</span>
<span class="ltx_bibblock">
Chen, J., Ma, B., Cui, H., and Xia, Y.

</span>
<span class="ltx_bibblock">Think twice before selection: Federated evidential active learning for medical image analysis with domain shifts.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib131.1.1" class="ltx_emph ltx_font_italic">Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition</em> (2024):( 11439â€“11449).

</span>
</li>
<li id="bib.bib132" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Howard etÂ al. 2021</span>
<span class="ltx_bibblock">
Howard, F.Â M., Dolezal, J., Kochanny, S., Schulte, J., Chen, H., Heij, L., Huo, D., Nanda, R., Olopade, O.Â I., Kather, J.Â N. etÂ al. (2021). The impact of site-specific digital histology signatures on deep learning model accuracy and bias.

</span>
<span class="ltx_bibblock">Nature communications <em id="bib.bib132.1.1" class="ltx_emph ltx_font_italic">12</em>, 4423.

</span>
</li>
<li id="bib.bib133" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kairouz etÂ al. 2021</span>
<span class="ltx_bibblock">
Kairouz, P., McMahan, H.Â B., Avent, B., Bellet, A., Bennis, M., Bhagoji, A.Â N., Bonawitz, K., Charles, Z., Cormode, G., Cummings, R. etÂ al. (2021). Advances and open problems in federated learning.

</span>
<span class="ltx_bibblock">Foundations and trendsÂ® in machine learning <em id="bib.bib133.1.1" class="ltx_emph ltx_font_italic">14</em>, 1â€“210.

</span>
</li>
<li id="bib.bib134" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2022d</span>
<span class="ltx_bibblock">
Li, Q., Diao, Y., Chen, Q., and He, B.

</span>
<span class="ltx_bibblock">Federated learning on non-iid data silos: An experimental study.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib134.1.1" class="ltx_emph ltx_font_italic">2022 IEEE 38th international conference on data engineering (ICDE)</em>. IEEE (2022d):( 965â€“978).

</span>
</li>
<li id="bib.bib135" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Argente-Garrido etÂ al. 2024</span>
<span class="ltx_bibblock">
Argente-Garrido, A., Zuheros, C., LuzÃ³n, M., and Herrera, F. (2024). An interpretable client decision tree aggregation process for federated learning.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2404.02510.

</span>
</li>
<li id="bib.bib136" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mabrouk etÂ al. 2023</span>
<span class="ltx_bibblock">
Mabrouk, A., Redondo, R. P.Â D., AbdÂ Elaziz, M., and Kayed, M. (2023). Ensemble federated learning: An approach for collaborative pneumonia diagnosis.

</span>
<span class="ltx_bibblock">Applied Soft Computing <em id="bib.bib136.1.1" class="ltx_emph ltx_font_italic">144</em>, 110500.

</span>
</li>
<li id="bib.bib137" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ullah etÂ al. 2023</span>
<span class="ltx_bibblock">
Ullah, F., Srivastava, G., Xiao, H., Ullah, S., Lin, J. C.-W., and Zhao, Y. (2023). A scalable federated learning approach for collaborative smart healthcare systems with intermittent clients using medical imaging.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics.

</span>
</li>
<li id="bib.bib138" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Nguyen etÂ al. 2023</span>
<span class="ltx_bibblock">
Nguyen, J., Wang, J., Malik, K., Sanjabi, M., and Rabbat, M.

</span>
<span class="ltx_bibblock">Where to begin? on the impact of pre-training and initialization in federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib138.1.1" class="ltx_emph ltx_font_italic">International Conference on Learning Representations</em> (2023):.

</span>
</li>
<li id="bib.bib139" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chen etÂ al. 2023</span>
<span class="ltx_bibblock">
Chen, H.-Y., Tu, C.-H., Li, Z., Shen, H.-W., and Chao, W.-L.

</span>
<span class="ltx_bibblock">On the importance and applicability of pre-training for federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib139.1.1" class="ltx_emph ltx_font_italic">International Conference on Learning Representations</em> (2023):.

</span>
</li>
<li id="bib.bib140" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Fallah etÂ al. 2020</span>
<span class="ltx_bibblock">
Fallah, A., Mokhtari, A., and Ozdaglar, A. (2020). Personalized federated learning with theoretical guarantees: A model-agnostic meta-learning approach.

</span>
<span class="ltx_bibblock">Advances in Neural Information Processing Systems <em id="bib.bib140.1.1" class="ltx_emph ltx_font_italic">33</em>, 3557â€“3568.

</span>
</li>
<li id="bib.bib141" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Sun etÂ al. 2021b</span>
<span class="ltx_bibblock">
Sun, B., Huo, H., Yang, Y., and Bai, B. (2021b). Partialfed: Cross-domain personalized federated learning via partial initialization.

</span>
<span class="ltx_bibblock">Advances in Neural Information Processing Systems <em id="bib.bib141.1.1" class="ltx_emph ltx_font_italic">34</em>, 23309â€“23320.

</span>
</li>
<li id="bib.bib142" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Guan and Liu 2021</span>
<span class="ltx_bibblock">
Guan, H., and Liu, M. (2021). Domain adaptation for medical image analysis: a survey.

</span>
<span class="ltx_bibblock">IEEE Transactions on Biomedical Engineering.

</span>
</li>
<li id="bib.bib143" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chen etÂ al. 2021</span>
<span class="ltx_bibblock">
Chen, Y., Chai, Z., Cheng, Y., and Rangwala, H.

</span>
<span class="ltx_bibblock">Asynchronous federated learning for sensor data with concept drift.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib143.1.1" class="ltx_emph ltx_font_italic">2021 IEEE International Conference on Big Data (Big Data)</em>. IEEE (2021):( 4822â€“4831).

</span>
</li>
<li id="bib.bib144" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mora etÂ al. 2024</span>
<span class="ltx_bibblock">
Mora, A., Bujari, A., and Bellavista, P. (2024). Enhancing generalization in federated learning with heterogeneous data: A comparative literature review.

</span>
<span class="ltx_bibblock">Future Generation Computer Systems.

</span>
</li>
<li id="bib.bib145" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Ghassemi etÂ al. 2020</span>
<span class="ltx_bibblock">
Ghassemi, M., Naumann, T., Schulam, P., Beam, A.Â L., Chen, I.Â Y., and Ranganath, R. (2020). A review of challenges and opportunities in machine learning for health.

</span>
<span class="ltx_bibblock">AMIA Summits on Translational Science Proceedings <em id="bib.bib145.1.1" class="ltx_emph ltx_font_italic">2020</em>, 191.

</span>
</li>
<li id="bib.bib146" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wang etÂ al. 2024a</span>
<span class="ltx_bibblock">
Wang, L., Bian, J., and Xu, J.

</span>
<span class="ltx_bibblock">Federated learning with instance-dependent noisy label.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib146.1.1" class="ltx_emph ltx_font_italic">ICASSP 2024-2024 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP)</em>. IEEE (2024a):( 8916â€“8920).

</span>
</li>
<li id="bib.bib147" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">ofÂ Health ClinicalÂ Center 2017</span>
<span class="ltx_bibblock">
ofÂ Health ClinicalÂ Center, N.Â I.

</span>
<span class="ltx_bibblock">Chest x-ray dataset.

</span>
<span class="ltx_bibblock"><a target="_blank" href="https://nihcc.app.box.com/v/ChestXray-NIHCC" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://nihcc.app.box.com/v/ChestXray-NIHCC</a> (2017).

</span>
<span class="ltx_bibblock">Accessed: 2024-08-15.

</span>
</li>
<li id="bib.bib148" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Radiopaedia 2021</span>
<span class="ltx_bibblock">
Radiopaedia.

</span>
<span class="ltx_bibblock">Ground-glass opacification.

</span>
<span class="ltx_bibblock"><a target="_blank" href="https://radiopaedia.org/articles/ground-glass-opacification-3?lang=gb" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://radiopaedia.org/articles/ground-glass-opacification-3?lang=gb</a> (2021).

</span>
<span class="ltx_bibblock">Accessed: 2024-08-15.

</span>
</li>
<li id="bib.bib149" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">CREATIS 2019</span>
<span class="ltx_bibblock">
CREATIS.

</span>
<span class="ltx_bibblock">Camus challenge: Cardiac acquisitions for multi-structure ultrasound segmentation.

</span>
<span class="ltx_bibblock"><a target="_blank" href="https://www.creatis.insa-lyon.fr/Challenge/camus/index.html" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://www.creatis.insa-lyon.fr/Challenge/camus/index.html</a> (2019).

</span>
<span class="ltx_bibblock">Accessed: 2024-08-15.

</span>
</li>
<li id="bib.bib150" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">CAMELYON 2017</span>
<span class="ltx_bibblock">
CAMELYON.

</span>
<span class="ltx_bibblock">Camelyon17 challenge: Grand challenge on cancer metastasis detection in lymph nodes.

</span>
<span class="ltx_bibblock"><a target="_blank" href="https://camelyon17.grand-challenge.org/Home/" title="" class="ltx_ref ltx_url ltx_font_typewriter">https://camelyon17.grand-challenge.org/Home/</a> (2017).

</span>
<span class="ltx_bibblock">Accessed: 2024-08-15.

</span>
</li>
<li id="bib.bib151" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lin etÂ al. 2020</span>
<span class="ltx_bibblock">
Lin, T., Kong, L., Stich, S.Â U., and Jaggi, M. (2020). Ensemble distillation for robust model fusion in federated learning.

</span>
<span class="ltx_bibblock">Advances in Neural Information Processing Systems <em id="bib.bib151.1.1" class="ltx_emph ltx_font_italic">33</em>, 2351â€“2363.

</span>
</li>
<li id="bib.bib152" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li and Wang 2019</span>
<span class="ltx_bibblock">
Li, D., and Wang, J. (2019). Fedmd: Heterogenous federated learning via model distillation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:1910.03581.

</span>
</li>
<li id="bib.bib153" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Yan etÂ al. 2020</span>
<span class="ltx_bibblock">
Yan, Z., Wicaksana, J., Wang, Z., Yang, X., and Cheng, K.-T. (2020). Variation-aware federated learning with multi-source decentralized medical image data.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics <em id="bib.bib153.1.1" class="ltx_emph ltx_font_italic">25</em>, 2615â€“2628.

</span>
</li>
<li id="bib.bib154" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2020a</span>
<span class="ltx_bibblock">
Li, T., Sahu, A.Â K., Zaheer, M., Sanjabi, M., Talwalkar, A., and Smith, V. (2020a). Federated optimization in heterogeneous networks.

</span>
<span class="ltx_bibblock">Proceedings of Machine learning and systems <em id="bib.bib154.1.1" class="ltx_emph ltx_font_italic">2</em>, 429â€“450.

</span>
</li>
<li id="bib.bib155" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Karimireddy etÂ al. 2020</span>
<span class="ltx_bibblock">
Karimireddy, S.Â P., Kale, S., Mohri, M., Reddi, S., Stich, S., and Suresh, A.Â T.

</span>
<span class="ltx_bibblock">Scaffold: Stochastic controlled averaging for federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib155.1.1" class="ltx_emph ltx_font_italic">International conference on machine learning</em>. PMLR (2020):( 5132â€“5143).

</span>
</li>
<li id="bib.bib156" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2021b</span>
<span class="ltx_bibblock">
Li, Q., He, B., and Song, D.

</span>
<span class="ltx_bibblock">Model-contrastive federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib156.1.1" class="ltx_emph ltx_font_italic">Proceedings of the IEEE/CVF conference on computer vision and pattern recognition</em> (2021b):( 10713â€“10722).

</span>
</li>
<li id="bib.bib157" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Luo etÂ al. 2022</span>
<span class="ltx_bibblock">
Luo, Z., Wang, Y., Wang, Z., Sun, Z., and Tan, T.

</span>
<span class="ltx_bibblock">Disentangled federated learning for tackling attributes skew via invariant aggregation and diversity transferring.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib157.1.1" class="ltx_emph ltx_font_italic">International Conference on Machine Learning</em>. PMLR (2022):( 14527â€“14541).

</span>
</li>
<li id="bib.bib158" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">TayebiÂ Arasteh etÂ al. 2023</span>
<span class="ltx_bibblock">
TayebiÂ Arasteh, S., Ziller, A., Kuhl, C., Makowski, M., Nebelung, S., Braren, R., Rueckert, D., Truhn, D., and Kaissis, G. (2023). Private, fair and accurate: Training large-scale, privacy-preserving ai models in medical imaging.

</span>
<span class="ltx_bibblock">arXiv e-prints ( arXivâ€“2302).

</span>
</li>
<li id="bib.bib159" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lu etÂ al. 2020</span>
<span class="ltx_bibblock">
Lu, S., Zhang, Y., and Wang, Y.

</span>
<span class="ltx_bibblock">Decentralized federated learning for electronic health records.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib159.1.1" class="ltx_emph ltx_font_italic">2020 54th Annual Conference on Information Sciences and Systems (CISS)</em>. IEEE (2020):( 1â€“5).

</span>
</li>
<li id="bib.bib160" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Park and Ye 2022</span>
<span class="ltx_bibblock">
Park, S., and Ye, J.Â C. (2022). Multi-task distributed learning using vision transformer with random patch permutation.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging <em id="bib.bib160.1.1" class="ltx_emph ltx_font_italic">42</em>, 2091â€“2105.

</span>
</li>
<li id="bib.bib161" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lu etÂ al. 2022b</span>
<span class="ltx_bibblock">
Lu, W., Wang, J., Chen, Y., Qin, X., Xu, R., Dimitriadis, D., and Qin, T. (2022b). Personalized federated learning with adaptive batchnorm for healthcare.

</span>
<span class="ltx_bibblock">IEEE Transactions on Big Data.

</span>
</li>
<li id="bib.bib162" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Thakur etÂ al. 2021</span>
<span class="ltx_bibblock">
Thakur, A., Sharma, P., and Clifton, D.Â A. (2021). Dynamic neural graphs based federated reptile for semi-supervised multi-tasking in healthcare applications.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics <em id="bib.bib162.1.1" class="ltx_emph ltx_font_italic">26</em>, 1761â€“1772.

</span>
</li>
<li id="bib.bib163" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Cholakoska etÂ al. 2021</span>
<span class="ltx_bibblock">
Cholakoska, A., Pfitzner, B., Gjoreski, H., Rakovic, V., Arnrich, B., and Kalendar, M.

</span>
<span class="ltx_bibblock">Differentially private federated learningfor anomaly detection in ehealth networks.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib163.1.1" class="ltx_emph ltx_font_italic">Adjunct Proceedings of the 2021 ACM International Joint Conference on Pervasive and Ubiquitous Computing and Proceedings of the 2021 ACM International Symposium on Wearable Computers</em> (2021):( 514â€“518).

</span>
</li>
<li id="bib.bib164" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wen etÂ al. 2016</span>
<span class="ltx_bibblock">
Wen, W., Wu, C., Wang, Y., Chen, Y., and Li, H. (2016). Learning structured sparsity in deep neural networks.

</span>
<span class="ltx_bibblock">Advances in neural information processing systems <em id="bib.bib164.1.1" class="ltx_emph ltx_font_italic">29</em>.

</span>
</li>
<li id="bib.bib165" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hu etÂ al. 2021</span>
<span class="ltx_bibblock">
Hu, E.Â J., Shen, Y., Wallis, P., Allen-Zhu, Z., Li, Y., Wang, S., Wang, L., and Chen, W. (2021). Lora: Low-rank adaptation of large language models.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2106.09685.

</span>
</li>
<li id="bib.bib166" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Guha etÂ al. 2019</span>
<span class="ltx_bibblock">
Guha, N., Talwalkar, A., and Smith, V. (2019). One-shot federated learning.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:1902.11175.

</span>
</li>
<li id="bib.bib167" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Xiong etÂ al. 2022</span>
<span class="ltx_bibblock">
Xiong, Y., Wang, R., Cheng, M., Yu, F., and Hsieh, C.-J.

</span>
<span class="ltx_bibblock">FedDM: Iterative distribution matching for communication-efficient federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib167.1.1" class="ltx_emph ltx_font_italic">Workshop on Federated Learning: Recent Advances and New Challenges (in Conjunction with NeurIPS 2022)</em> (2022):.

</span>
</li>
<li id="bib.bib168" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Goetz and Tewari 2020</span>
<span class="ltx_bibblock">
Goetz, J., and Tewari, A. (2020). Federated learning via synthetic data.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2008.04489.

</span>
</li>
<li id="bib.bib169" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Pan etÂ al. 2020</span>
<span class="ltx_bibblock">
Pan, F., Ye, T., Sun, P., Gui, S., Liang, B., Li, L., Zheng, D., Wang, J., Hesketh, R.Â L., Yang, L., and Zheng, C. (2020). Time course of lung changes at chest ct during recovery from coronavirus disease 2019 (covid-19).

</span>
<span class="ltx_bibblock">Radiology <em id="bib.bib169.1.1" class="ltx_emph ltx_font_italic">295</em>, 715â€“721.

</span>
</li>
<li id="bib.bib170" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2019</span>
<span class="ltx_bibblock">
Li, M., Zhang, W., Yang, G., Wang, C., Zhang, H., Liu, H., Zheng, W., and Li, S.

</span>
<span class="ltx_bibblock">Recurrent aggregation learning for multi-view echocardiographic sequences segmentation.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib170.1.1" class="ltx_emph ltx_font_italic">Medical Image Computing and Computer Assisted Interventionâ€“MICCAI 2019: 22nd International Conference, Shenzhen, China, October 13â€“17, 2019, Proceedings, Part II 22</em>. Springer (2019):( 678â€“686).

</span>
</li>
<li id="bib.bib171" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2020b</span>
<span class="ltx_bibblock">
Li, M., Wang, C., Zhang, H., and Yang, G. (2020b). Mv-ran: Multiview recurrent aggregation network for echocardiographic sequences segmentation and full cardiac cycle analysis.

</span>
<span class="ltx_bibblock">Computers in biology and medicine <em id="bib.bib171.1.1" class="ltx_emph ltx_font_italic">120</em>, 103728.

</span>
</li>
<li id="bib.bib172" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2020c</span>
<span class="ltx_bibblock">
Li, M., Dong, S., Gao, Z., Feng, C., Xiong, H., Zheng, W., Ghista, D., Zhang, H., and deÂ Albuquerque, V. H.Â C. (2020c). Unified model for interpreting multi-view echocardiographic sequences without temporal information.

</span>
<span class="ltx_bibblock">Applied Soft Computing <em id="bib.bib172.1.1" class="ltx_emph ltx_font_italic">88</em>, 106049.

</span>
</li>
<li id="bib.bib173" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wang etÂ al. 2024b</span>
<span class="ltx_bibblock">
Wang, S., Liu, B., and Zuccon, G.

</span>
<span class="ltx_bibblock">How to forget clients in federated online learning to rank?

</span>
<span class="ltx_bibblock">In: <em id="bib.bib173.1.1" class="ltx_emph ltx_font_italic">European Conference on Information Retrieval</em>. Springer (2024b):( 105â€“121).

</span>
</li>
<li id="bib.bib174" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2022e</span>
<span class="ltx_bibblock">
Li, Z., Wang, L., Chen, G., Zhang, Z., Shafiq, M., and Gu, Z. (2022e). E2egi: End-to-end gradient inversion in federated learning.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics <em id="bib.bib174.1.1" class="ltx_emph ltx_font_italic">27</em>, 756â€“767.

</span>
</li>
<li id="bib.bib175" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Yu etÂ al. 2020</span>
<span class="ltx_bibblock">
Yu, D., Zhang, H. etÂ al.

</span>
<span class="ltx_bibblock">Gradient Perturbation is Underrated for Differentially Private Convex Optimization.

</span>
<span class="ltx_bibblock">In: Bessiere, C., ed. <em id="bib.bib175.1.1" class="ltx_emph ltx_font_italic">Proceedings of the Twenty-Ninth International Joint Conference on Artificial Intelligence, IJCAI-20</em> (2020):( 3117â€“3123).

</span>
</li>
<li id="bib.bib176" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Melis etÂ al. 2019</span>
<span class="ltx_bibblock">
Melis, L., Song, C., DeÂ Cristofaro, E., and Shmatikov, V.

</span>
<span class="ltx_bibblock">Exploiting unintended feature leakage in collaborative learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib176.1.1" class="ltx_emph ltx_font_italic">2019 IEEE symposium on security and privacy (SP)</em>. IEEE (2019):( 691â€“706).

</span>
</li>
<li id="bib.bib177" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Nasr etÂ al. 2019</span>
<span class="ltx_bibblock">
Nasr, M., Shokri, R., and Houmansadr, A.

</span>
<span class="ltx_bibblock">Comprehensive privacy analysis of deep learning: Passive and active white-box inference attacks against centralized and federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib177.1.1" class="ltx_emph ltx_font_italic">2019 IEEE symposium on security and privacy (SP)</em>. IEEE (2019):( 739â€“753).

</span>
</li>
<li id="bib.bib178" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wu etÂ al. 2019</span>
<span class="ltx_bibblock">
Wu, B., Zhao, S., Sun, G., Zhang, X., Su, Z., Zeng, C., and Liu, Z.

</span>
<span class="ltx_bibblock">P3sgd: Patient privacy preserving sgd for regularizing deep cnns in pathological image classification.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib178.1.1" class="ltx_emph ltx_font_italic">Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition</em> (2019):( 2099â€“2108).

</span>
</li>
<li id="bib.bib179" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2021c</span>
<span class="ltx_bibblock">
Li, J., Li, N., and Ribeiro, B.

</span>
<span class="ltx_bibblock">Membership inference attacks and defenses in classification models.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib179.1.1" class="ltx_emph ltx_font_italic">Proceedings of the Eleventh ACM Conference on Data and Application Security and Privacy</em> (2021c):( 5â€“16).

</span>
</li>
<li id="bib.bib180" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liu etÂ al. 2021d</span>
<span class="ltx_bibblock">
Liu, B., Ding, M., Shaham, S., Rahayu, W., Farokhi, F., and Lin, Z. (2021d). When machine learning meets privacy: A survey and outlook.

</span>
<span class="ltx_bibblock">ACM Computing Surveys (CSUR) <em id="bib.bib180.1.1" class="ltx_emph ltx_font_italic">54</em>, 1â€“36.

</span>
</li>
<li id="bib.bib181" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lindell 2020</span>
<span class="ltx_bibblock">
Lindell, Y. (2020). Secure multiparty computation.

</span>
<span class="ltx_bibblock">Communications of the ACM <em id="bib.bib181.1.1" class="ltx_emph ltx_font_italic">64</em>, 86â€“96.

</span>
</li>
<li id="bib.bib182" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Huang etÂ al. 2023</span>
<span class="ltx_bibblock">
Huang, W., Zhuo, M., Zhu, T., Zhou, S., and Liao, Y. (2023). Differential privacy: Review of improving utility through cryptography-based technologies.

</span>
<span class="ltx_bibblock">Concurrency and Computation: Practice and Experience <em id="bib.bib182.1.1" class="ltx_emph ltx_font_italic">35</em>, e7565.

</span>
</li>
<li id="bib.bib183" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Huang etÂ al. 2022</span>
<span class="ltx_bibblock">
Huang, C., Yao, Y., Zhang, X., Teng, D., Wang, Y., and Zhou, L.

</span>
<span class="ltx_bibblock">Robust secure aggregation with lightweight verification for federated learning.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib183.1.1" class="ltx_emph ltx_font_italic">2022 IEEE International Conference on Trust, Security and Privacy in Computing and Communications (TrustCom)</em>. IEEE (2022):( 582â€“589).

</span>
</li>
<li id="bib.bib184" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Dwork 2006</span>
<span class="ltx_bibblock">
Dwork, C.

</span>
<span class="ltx_bibblock">Differential privacy.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib184.1.1" class="ltx_emph ltx_font_italic">International colloquium on automata, languages, and programming</em>. Springer (2006):( 1â€“12).

</span>
</li>
<li id="bib.bib185" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lee and Kifer 2021</span>
<span class="ltx_bibblock">
Lee, J., and Kifer, D. (2021). Scaling up differentially private deep learning with fast per-example gradient clipping.

</span>
<span class="ltx_bibblock">Proceedings on Privacy Enhancing Technologies.

</span>
</li>
<li id="bib.bib186" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Dong etÂ al. 2018</span>
<span class="ltx_bibblock">
Dong, S., Gao, Z., Sun, S., Wang, X., Li, M., Zhang, H., Yang, G., Liu, H., Li, S. etÂ al.

</span>
<span class="ltx_bibblock">Holistic and deep feature pyramids for saliency detection.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib186.1.1" class="ltx_emph ltx_font_italic">BMVC</em> vol.Â 67 (2018):.

</span>
</li>
<li id="bib.bib187" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liu etÂ al. 2023</span>
<span class="ltx_bibblock">
Liu, W., He, Y., Wang, X., Duan, Z., Liang, W., and Liu, Y. (2023). Bfg: privacy protection framework for internet of medical things based on blockchain and federated learning.

</span>
<span class="ltx_bibblock">Connection Science <em id="bib.bib187.1.1" class="ltx_emph ltx_font_italic">35</em>, 2199951.

</span>
</li>
<li id="bib.bib188" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">CassarÃ¡ etÂ al. 2022</span>
<span class="ltx_bibblock">
CassarÃ¡, P., Gotta, A., and Valerio, L. (2022). Federated feature selection for cyber-physical systems of systems.

</span>
<span class="ltx_bibblock">IEEE Transactions on Vehicular Technology <em id="bib.bib188.1.1" class="ltx_emph ltx_font_italic">71</em>, 9937â€“9950.

</span>
</li>
<li id="bib.bib189" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Zhang etÂ al. 2023b</span>
<span class="ltx_bibblock">
Zhang, X., Mavromatis, A., Vafeas, A., Nejabati, R., and Simeonidou, D. (2023b). Federated feature selection for horizontal federated learning in iot networks.

</span>
<span class="ltx_bibblock">IEEE Internet of Things Journal <em id="bib.bib189.1.1" class="ltx_emph ltx_font_italic">10</em>, 10095â€“10112.

</span>
</li>
<li id="bib.bib190" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Li etÂ al. 2024</span>
<span class="ltx_bibblock">
Li, Q., Xie, C., Xu, X., Liu, X., Zhang, C., Li, B., He, B., and Song, D.

</span>
<span class="ltx_bibblock">Effective and efficient federated tree learning on hybrid data.

</span>
<span class="ltx_bibblock">In: <em id="bib.bib190.1.1" class="ltx_emph ltx_font_italic">The Twelfth International Conference on Learning Representations</em> (2024):.

</span>
</li>
</ul>
</section>
<div class="ltx_pagination ltx_role_newpage"></div>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/2409.09726" class="ar5iv-nav-button ar5iv-nav-button-prev">â—„</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/2409.09727" class="ar5iv-text-button ar5iv-severity-warning">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+2409.09727">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/2409.09727" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/2409.09729" class="ar5iv-nav-button ar5iv-nav-button-next">â–º</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Sat Oct  5 21:39:27 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "Ã—";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
