<article class="ltx_document ltx_authors_1line">
 <div class="ltx_para" id="p1">
  <span class="ltx_ERROR undefined" id="p1.1">
   \theorembodyfont
  </span>
  <span class="ltx_ERROR undefined" id="p1.2">
   \theoremheaderfont
  </span>
  <span class="ltx_ERROR undefined" id="p1.3">
   \theorempostheader
  </span>
  <p class="ltx_p" id="p1.4">
   :
   <span class="ltx_ERROR undefined" id="p1.4.1">
    \theoremsep
   </span>
   <br class="ltx_break"/>
   <span class="ltx_ERROR undefined" id="p1.4.2">
    \jmlrvolume
   </span>
   [VOLUME # TBD]
   <span class="ltx_ERROR undefined" id="p1.4.3">
    \jmlryear
   </span>
   2024
   <span class="ltx_ERROR undefined" id="p1.4.4">
    \jmlrworkshop
   </span>
   Machine Learning for Healthcare
  </p>
 </div>
 <h1 class="ltx_title ltx_title_document">
  CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning
 </h1>
 <div class="ltx_authors">
  <span class="ltx_creator ltx_role_author">
   <span class="ltx_personname">
    <span class="ltx_ERROR undefined" id="id1.1.id1">
     \Name
    </span>
    Ling Yue, Tianfan Fu
    <br class="ltx_break"/>
    <span class="ltx_ERROR undefined" id="id2.2.id2">
     \addr
    </span>
    Computer Science Department
   </span>
  </span>
  <span class="ltx_author_before">
  </span>
  <span class="ltx_creator ltx_role_author">
   <span class="ltx_personname">
    Rensselaer Polytechnic Institute
   </span>
  </span>
 </div>
 <div class="ltx_abstract">
  <h6 class="ltx_title ltx_title_abstract">
   Abstract
  </h6>
  <p class="ltx_p" id="id3.id1">
   Large Language Models (LLMs) and multi-agent systems have shown impressive capabilities in natural language tasks but face challenges in clinical trial applications, primarily due to limited access to external knowledge.
Recognizing the potential of advanced clinical trial tools that aggregate and predict based on the latest medical data, we propose an integrated solution to enhance their accessibility and utility.
We introduce Clinical Agent System (CT-Agent), a Clinical multi-agent system designed for clinical trial tasks, leveraging GPT-4, multi-agent architectures, LEAST-TO-MOST, and ReAct reasoning technology.
This integration not only boosts LLM performance in clinical contexts but also introduces novel functionalities.
Our system autonomously manages the entire clinical trial process, demonstrating significant efficiency improvements in our evaluations, which include both computational benchmarks and expert feedback.
  </p>
 </div>
 <section class="ltx_section" id="S1">
  <h2 class="ltx_title ltx_title_section">
   <span class="ltx_tag ltx_tag_section">
    1
   </span>
   Introduction
  </h2>
  <div class="ltx_para" id="S1.p1">
   <p class="ltx_p" id="S1.p1.1">
    The introduction of clinical multi-agent systems into the healthcare sector marks a substantial advancement in improving care quality through sophisticated computational methods and in-depth data analysis.
These systems, driven by Large Language Models
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Singhal et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib21" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023a
      </span>
     </a>
     )
    </cite>
    (LLMs) like ChatGPT
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Liu et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib13" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023
      </span>
     </a>
     )
    </cite>
    , BioGPT
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Luo et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib18" title="">
      <span class="ltx_text" style="font-size:90%;">
       2022
      </span>
     </a>
     )
    </cite>
    , ChatDoctor
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Yunxiang et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib28" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023
      </span>
     </a>
     )
    </cite>
    ,
and Med-PaLM
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Singhal et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib22" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023b
      </span>
     </a>
     )
    </cite>
    , have shown considerable success in processing and understanding medical data, providing customized care, and offering insights into intricate health conditions.
However, their use in clinical trials faces challenges, mainly due to their limited ability to access and integrate external knowledge sources, such as DrugBank
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Wishart et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib25" title="">
      <span class="ltx_text" style="font-size:90%;">
       2018
      </span>
     </a>
     )
    </cite>
    .
This research stems from the urgent need to fully utilize LLMs in clinical settings, going beyond the conversational skills of current models to include actionable and explanatory analysis leveraging extensive external data.
   </p>
  </div>
  <div class="ltx_para" id="S1.p2">
   <p class="ltx_p" id="S1.p2.1">
    Our study introduces CT-Agent, a new Clinical multi-agent system tailored for clinical trial tasks.
Utilizing the capabilities of GPT-4, combined with multi-agent system architectures, and incorporating advanced reasoning technologies like LEAST-TO-MOST
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Zhou et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib31" title="">
      <span class="ltx_text" style="font-size:90%;">
       2022
      </span>
     </a>
     )
    </cite>
    and ReAct
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Yao et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib26" title="">
      <span class="ltx_text" style="font-size:90%;">
       2022
      </span>
     </a>
     )
    </cite>
    , our solution not only boosts LLM performance in clinical scenarios but also brings new functionalities.
Our system is designed to autonomously oversee the clinical trial process, filling the void in existing implementations that mainly focus on conversational interactions without sufficient actionable outcomes.
   </p>
  </div>
  <div class="ltx_para" id="S1.p3">
   <p class="ltx_p" id="S1.p3.1">
    Prior research has highlighted the potential of LLMs in healthcare, particularly in diagnostics, patient communication, and medical research
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Singhal et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib22" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023b
      </span>
     </a>
     ;
     <span class="ltx_text" style="font-size:90%;">
      Yunxiang et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib28" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023
      </span>
     </a>
     ;
     <span class="ltx_text" style="font-size:90%;">
      Singhal et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib21" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023a
      </span>
     </a>
     )
    </cite>
    .
Yet, these investigations have not fully exploited the models for clinical trials, where understanding the complex relationships between drugs, diseases, and patient reactions is crucial.
Our research introduces a multi-agent framework that uses specialized agents for tasks such as drug information retrieval, disease analysis, and explanatory reasoning.
This strategy not only allows for a more detailed and understandable decision-making process but also significantly enhances clinical trial analysis capabilities, including predicting outcomes, deciphering reasons for failure, and estimating trial duration.
   </p>
  </div>
  <div class="ltx_para" id="S1.p4">
   <p class="ltx_p" id="S1.p4.1">
    A review of the literature indicates a growing interest in improving LLM applications in medicine.
For instance, studies like
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Li et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib12" title="">
      <span class="ltx_text" style="font-size:90%;">
       2024
      </span>
     </a>
     )
    </cite>
    discuss employing ChatGPT and BioGPT for patient data synthesis and diagnostic recommendations.
However, these discussions often focus only on the conversational aspects, overlooking the actionable intelligence and comprehensive reasoning our approach introduces.
Moreover, our method is unique in incorporating external databases and reasoning technologies like ReAct, aiming not just to interpret but also to act on the intricate network of clinical data.
   </p>
  </div>
  <div class="ltx_para" id="S1.p5">
   <p class="ltx_p" id="S1.p5.1">
    Our main contributions are summarized as follows:
   </p>
   <ul class="ltx_itemize" id="S1.I1">
    <li class="ltx_item" id="S1.I1.i1" style="list-style-type:none;">
     <span class="ltx_tag ltx_tag_item">
      •
     </span>
     <div class="ltx_para" id="S1.I1.i1.p1">
      <p class="ltx_p" id="S1.I1.i1.p1.1">
       We present Clinical Trial Multi-Agent (CT-Agent), the first multi-agent framework that elevates the conversational abilities of LLMs with actionable intelligence.
      </p>
     </div>
    </li>
    <li class="ltx_item" id="S1.I1.i2" style="list-style-type:none;">
     <span class="ltx_tag ltx_tag_item">
      •
     </span>
     <div class="ltx_para" id="S1.I1.i2.p1">
      <p class="ltx_p" id="S1.I1.i2.p1.1">
       We integrate extensive tools, and knowledge and use advanced reasoning technologies to enhance the system’s decision-making capabilities.
      </p>
     </div>
    </li>
    <li class="ltx_item" id="S1.I1.i3" style="list-style-type:none;">
     <span class="ltx_tag ltx_tag_item">
      •
     </span>
     <div class="ltx_para" id="S1.I1.i3.p1">
      <p class="ltx_p" id="S1.I1.i3.p1.1">
       CT-Agent achieves competitive predictive performance in clinical trial outcome prediction (0.7908 PR-AUC), obtaining a 0.3326 improvement over the Standard Prompt Method.
      </p>
     </div>
    </li>
   </ul>
  </div>
 </section>
 <section class="ltx_section" id="S2">
  <h2 class="ltx_title ltx_title_section">
   <span class="ltx_tag ltx_tag_section">
    2
   </span>
   Related Work
  </h2>
  <div class="ltx_para" id="S2.p1">
   <p class="ltx_p" id="S2.p1.1">
    NLP has achieved significant progress in the biomedical arena, delivering crucial insights and tools for a range of applications in healthcare and medicine. The advent of Large Language Models (LLMs) has notably advanced the medical field by embedding comprehensive medical knowledge into their training.
   </p>
  </div>
  <div class="ltx_para" id="S2.p2">
   <p class="ltx_p" id="S2.p2.1">
    For example, question answering (QA) in the medical domain represents a critical challenge in NLP, where language models are tasked with responding to specific queries using their embedded medical knowledge, e.g., MedQA (USMLE)
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Jin et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib8" title="">
      <span class="ltx_text" style="font-size:90%;">
       2020
      </span>
     </a>
     )
    </cite>
    HeadQA
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Vilares and Gómez-Rodríguez
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib23" title="">
      <span class="ltx_text" style="font-size:90%;">
       2019
      </span>
     </a>
     )
    </cite>
    , MMLU
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Hendrycks et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib6" title="">
      <span class="ltx_text" style="font-size:90%;">
       2021
      </span>
     </a>
     )
    </cite>
    , and PubMedQA
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Jin et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib9" title="">
      <span class="ltx_text" style="font-size:90%;">
       2019
      </span>
     </a>
     )
    </cite>
    .
   </p>
  </div>
  <div class="ltx_para" id="S2.p3">
   <p class="ltx_p" id="S2.p3.1">
    Despite being pretrained for general purposes, closed-source LLMs like ChatGPT
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      OpenAI
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib19" title="">
      <span class="ltx_text" style="font-size:90%;">
       2022
      </span>
     </a>
     )
    </cite>
    and GPT-4
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      OpenAI
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib20" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023
      </span>
     </a>
     )
    </cite>
    have demonstrated considerable medical capabilities in both benchmark evaluations and real-world applications.
    <cite class="ltx_cite ltx_citemacro_cite">
     <span class="ltx_text" style="font-size:90%;">
      Liévin et al.
     </span>
     (
     <a class="ltx_ref" href="#bib.bib14" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023
      </span>
     </a>
     )
    </cite>
    applied GPT-3.5 using various prompting techniques, such as Chain-of-Thought, few-shot, and retrieval augmentation, across three medical reasoning benchmarks, showcasing the model’s robust medical reasoning skills without the need for specialized fine-tuning.
Additionally, evaluations of LLMs such as ChatGPT on professional medical assessments, including the US Medical Exam
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Kung et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib10" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023
      </span>
     </a>
     )
    </cite>
    and the Otolaryngology-Head and Neck Surgery Certification Examinations
    <cite class="ltx_cite ltx_citemacro_citep">
     (
     <span class="ltx_text" style="font-size:90%;">
      Long et al.
     </span>
     ,
     <a class="ltx_ref" href="#bib.bib15" title="">
      <span class="ltx_text" style="font-size:90%;">
       2023
      </span>
     </a>
     )
    </cite>
    , have resulted in scores that meet or nearly meet passing thresholds. This performance underscores the potential of LLMs to aid in significant medical contexts, including medical education and clinical decision-making.
   </p>
  </div>
  <section class="ltx_paragraph" id="S2.SS0.SSS0.Px1">
   <h5 class="ltx_title ltx_title_paragraph">
    AI for Clinical Trial.
   </h5>
   <div class="ltx_para" id="S2.SS0.SSS0.Px1.p1">
    <p class="ltx_p" id="S2.SS0.SSS0.Px1.p1.1">
     AI has great potential to revolutionize clinical trials in a couple of problems.
Specifically,
     <cite class="ltx_cite ltx_citemacro_cite">
      <span class="ltx_text" style="font-size:90%;">
       Zhang et al.
      </span>
      (
      <a class="ltx_ref" href="#bib.bib30" title="">
       <span class="ltx_text" style="font-size:90%;">
        2020
       </span>
      </a>
      );
      <span class="ltx_text" style="font-size:90%;">
       Gao et al.
      </span>
      (
      <a class="ltx_ref" href="#bib.bib5" title="">
       <span class="ltx_text" style="font-size:90%;">
        2020
       </span>
      </a>
      )
     </cite>
     leverage AI to recruit appropriate patients that meet the requirement in eligibility criteria.
     <cite class="ltx_cite ltx_citemacro_cite">
      <span class="ltx_text" style="font-size:90%;">
       Fu et al.
      </span>
      (
      <a class="ltx_ref" href="#bib.bib4" title="">
       <span class="ltx_text" style="font-size:90%;">
        2022
       </span>
      </a>
      );
      <span class="ltx_text" style="font-size:90%;">
       Chen et al.
      </span>
      (
      <a class="ltx_ref" href="#bib.bib3" title="">
       <span class="ltx_text" style="font-size:90%;">
        2024
       </span>
      </a>
      );
      <span class="ltx_text" style="font-size:90%;">
       Lu et al.
      </span>
      (
      <a class="ltx_ref" href="#bib.bib17" title="">
       <span class="ltx_text" style="font-size:90%;">
        2024
       </span>
      </a>
      )
     </cite>
     builds machine learning models to predict the outcome of clinical trials based on clinical trial features such as drug molecule, disease code, and eligibility criteria.
In the context of clinical trials,
     <cite class="ltx_cite ltx_citemacro_citet">
      <span class="ltx_text" style="font-size:90%;">
       Wang et al.
      </span>
      (
      <a class="ltx_ref" href="#bib.bib24" title="">
       <span class="ltx_text" style="font-size:90%;">
        2024
       </span>
      </a>
      )
     </cite>
     leverages the large language model to generate patient-level digital twins to simulate clinical trials.
However, most of these works do not utilize LLM’s reasoning ability and cannot solve complex problems in clinical trials.
To the best of our knowledge, our work is the first LLM agent work for clinical trials and is able to solve complex clinical trial reasoning problems.
    </p>
   </div>
  </section>
 </section>
 <section class="ltx_section" id="S3">
  <h2 class="ltx_title ltx_title_section">
   <span class="ltx_tag ltx_tag_section">
    3
   </span>
   Methods
  </h2>
  <figure class="ltx_figure" id="S3.F1">
   <img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="230" id="S3.F1.g1" src="/html/2404.14777/assets/figure/ctagent.png" width="479"/>
   <figcaption class="ltx_caption ltx_centering">
    <span class="ltx_tag ltx_tag_figure">
     Figure 1:
    </span>
    CT-Agent framework. Given a complex problem to solve (e.g., predicting clinical trial outcome), the role of the Planning Agent is to decompose it into three subproblems: trial enrollment, drug safety to the human body, and drug efficacy to disease. These subproblems are solved by Enrollment Agent, Safety Agent, and Efficacy Agent, respectively, enhanced by calling external tools (Section
    <a class="ltx_ref" href="#S3.SS3" title="3.3 Calling External Tools ‣ 3 Methods ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
     <span class="ltx_text ltx_ref_tag">
      3.3
     </span>
    </a>
    ). Finally, the Reasoning Agent aggregates the solutions of subproblems, draws the conclusion, and makes the prediction.
   </figcaption>
  </figure>
  <section class="ltx_subsection" id="S3.SS1">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     3.1
    </span>
    Overview of CT-Agent
   </h3>
   <div class="ltx_para" id="S3.SS1.p1">
    <p class="ltx_p" id="S3.SS1.p1.1">
     Our proposed system is a conversational multi-agent framework,
analogous to a hospital staffed by various specialists.
Each agent within this system plays a distinct role,
mirroring the specialization seen in medical professionals—some focus on pharmacology,
others on diagnosing diseases, while a few are dedicated to designing clinical trials.
To process natural language inputs and generate responses that are coherent and contextually appropriate,
each agent utilizes GPT-4.
Moreover, we enhance the system’s reasoning capabilities by incorporating methodologies such as ReAct
     <cite class="ltx_cite ltx_citemacro_citep">
      (
      <span class="ltx_text" style="font-size:90%;">
       Yao et al.
      </span>
      ,
      <a class="ltx_ref" href="#bib.bib26" title="">
       <span class="ltx_text" style="font-size:90%;">
        2022
       </span>
      </a>
      )
     </cite>
     and the LEAST-TO-MOST
     <cite class="ltx_cite ltx_citemacro_citep">
      (
      <span class="ltx_text" style="font-size:90%;">
       Zhou et al.
      </span>
      ,
      <a class="ltx_ref" href="#bib.bib31" title="">
       <span class="ltx_text" style="font-size:90%;">
        2022
       </span>
      </a>
      )
     </cite>
     principle.
Following the reasoning process, the system is capable of taking actions such as searching for information,
indexing data in databases, and employing expert AI models.
By integrating this information, the system effectively simulates a highly knowledgeable doctor.
Working in concert, these agents can deliver precise, explainable solutions to user inquiries.
    </p>
   </div>
  </section>
  <section class="ltx_subsection" id="S3.SS2">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     3.2
    </span>
    Agent Roles and Responsibilities
   </h3>
   <div class="ltx_para" id="S3.SS2.p1">
    <p class="ltx_p" id="S3.SS2.p1.1">
     The CT-Agent framework integrates a diverse array of specialized agents, each employing the ReAct and LEAST-TO-MOST reasoning methods to meticulously plan their actions.
Through the use of advanced search capabilities, access to specialist models, and indexing in databases, these agents are able to execute a wide range of tasks effectively.
Below, we delve into the specific roles and responsibilities assigned to each agent within the system.
    </p>
   </div>
   <section class="ltx_subsubsection" id="S3.SS2.SSS1">
    <h4 class="ltx_title ltx_title_subsubsection">
     <span class="ltx_tag ltx_tag_subsubsection">
      3.2.1
     </span>
     Planning Agent
    </h4>
    <div class="ltx_para" id="S3.SS2.SSS1.p1">
     <p class="ltx_p" id="S3.SS2.SSS1.p1.1">
      The Planning Agent’s primary role is to strategize and determine the optimal approach to address user problems. Utilizing the LEAST-TO-MOST Reasoning method, this agent systematically decomposes complex issues into smaller, more manageable subproblems. This stepwise breakdown facilitates targeted interventions, where each subproblem is addressed by the most suitable specialist agent. In the context of clinical trials, the Planning Agent employs few-shot learning techniques to train on example scenarios. This approach enhances the agent’s ability to effectively decompose and delegate tasks within clinical contexts, ensuring precise and efficient problem-solving.
     </p>
    </div>
   </section>
   <section class="ltx_subsubsection" id="S3.SS2.SSS2">
    <h4 class="ltx_title ltx_title_subsubsection">
     <span class="ltx_tag ltx_tag_subsubsection">
      3.2.2
     </span>
     Efficacy Agent
    </h4>
    <div class="ltx_para" id="S3.SS2.SSS2.p1">
     <p class="ltx_p" id="S3.SS2.SSS2.p1.1">
      The Efficacy Agent is a specialized module within our multi-agent framework, primarily focused on assessing the therapeutic effectiveness of drugs against specified diseases
      <cite class="ltx_cite ltx_citemacro_citep">
       (
       <span class="ltx_text" style="font-size:90%;">
        Chang et al.
       </span>
       ,
       <a class="ltx_ref" href="#bib.bib1" title="">
        <span class="ltx_text" style="font-size:90%;">
         2019
        </span>
       </a>
       ;
       <span class="ltx_text" style="font-size:90%;">
        Chen et al.
       </span>
       ,
       <a class="ltx_ref" href="#bib.bib2" title="">
        <span class="ltx_text" style="font-size:90%;">
         2021
        </span>
       </a>
       ;
       <span class="ltx_text" style="font-size:90%;">
        Zhang et al.
       </span>
       ,
       <a class="ltx_ref" href="#bib.bib29" title="">
        <span class="ltx_text" style="font-size:90%;">
         2021
        </span>
       </a>
       ;
       <span class="ltx_text" style="font-size:90%;">
        Lu et al.
       </span>
       ,
       <a class="ltx_ref" href="#bib.bib16" title="">
        <span class="ltx_text" style="font-size:90%;">
         2022
        </span>
       </a>
       )
      </cite>
      . This agent utilizes advanced data retrieval and analysis techniques, drawing from rich biomedical databases such as DrugBank
      <cite class="ltx_cite ltx_citemacro_citep">
       (
       <span class="ltx_text" style="font-size:90%;">
        Wishart et al.
       </span>
       ,
       <a class="ltx_ref" href="#bib.bib25" title="">
        <span class="ltx_text" style="font-size:90%;">
         2018
        </span>
       </a>
       )
      </cite>
      and the HetioNet Knowledge Graph to ensure comprehensive and accurate evaluations.
     </p>
    </div>
    <div class="ltx_para" id="S3.SS2.SSS2.p2">
     <p class="ltx_p" id="S3.SS2.SSS2.p2.1">
      Specifically, the Efficacy Agent employs the SMILES (Simplified Molecular Input Line Entry System) notation to identify and retrieve detailed chemical and pharmacological information about drugs. This includes their molecular structure, mechanism of action, metabolism, and potential side effects, providing a holistic view of the drug’s properties.
     </p>
    </div>
    <div class="ltx_para" id="S3.SS2.SSS2.p3">
     <p class="ltx_p" id="S3.SS2.SSS2.p3.1">
      Upon receiving a query with a specific drug and disease, the Efficacy Agent performs several key functions:
     </p>
     <ul class="ltx_itemize" id="S3.I1">
      <li class="ltx_item" id="S3.I1.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I1.i1.p1">
        <p class="ltx_p" id="S3.I1.i1.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I1.i1.p1.1.1">
          Drug and Disease Profiling:
         </span>
         Retrieves up-to-date, detailed descriptions of the drug and the disease from DrugBank and other relevant databases, ensuring that users have access to reliable and comprehensive information.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I1.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I1.i2.p1">
        <p class="ltx_p" id="S3.I1.i2.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I1.i2.p1.1.1">
          Interaction Pathway Mapping:
         </span>
         Utilizes the HetioNet Knowledge Graph to trace and visualize the pathways connecting the drug to the disease. This involves identifying biological interactions, such as target proteins and genetic associations, that are crucial for understanding the drug’s potential efficacy.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I1.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I1.i3.p1">
        <p class="ltx_p" id="S3.I1.i3.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I1.i3.p1.1.1">
          Efficacy Assessment:
         </span>
         Analyzes the gathered information to evaluate the potential effectiveness of the drug against the disease, considering factors like target specificity, therapeutic indices, and evidence from clinical trials.
        </p>
       </div>
      </li>
     </ul>
    </div>
    <div class="ltx_para" id="S3.SS2.SSS2.p4">
     <p class="ltx_p" id="S3.SS2.SSS2.p4.1">
      By synthesizing data from multiple sources and employing sophisticated analytical techniques, the Efficacy Agent provides essential insights into drug-disease relationships, supporting informed decision-making in clinical and research settings.
     </p>
    </div>
   </section>
   <section class="ltx_subsubsection" id="S3.SS2.SSS3">
    <h4 class="ltx_title ltx_title_subsubsection">
     <span class="ltx_tag ltx_tag_subsubsection">
      3.2.3
     </span>
     Safety Agent
    </h4>
    <div class="ltx_para" id="S3.SS2.SSS3.p1">
     <p class="ltx_p" id="S3.SS2.SSS3.p1.1">
      The Safety Agent is integral to our CT-Agent framework, focusing specifically on the assessment of drug safety and its implications for patient health. This agent leverages a comprehensive repository of pharmacological data and historical clinical trial outcomes to evaluate the risks associated with specific drug-disease interactions. Utilizing databases such as DrugBank and clinical trial registries, the Safety Agent provides detailed insights into the historical safety profiles of drugs.
     </p>
    </div>
    <div class="ltx_para" id="S3.SS2.SSS3.p2">
     <p class="ltx_p" id="S3.SS2.SSS3.p2.1">
      Key functions of the Safety Agent include:
     </p>
     <ul class="ltx_itemize" id="S3.I2">
      <li class="ltx_item" id="S3.I2.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I2.i1.p1">
        <p class="ltx_p" id="S3.I2.i1.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I2.i1.p1.1.1">
          Drug Safety Profiling:
         </span>
         Accesses detailed safety information from databases to compile historical data on adverse drug reactions, contraindications, and warnings. This data is crucial for understanding the risk factors associated with the drug.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I2.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I2.i2.p1">
        <p class="ltx_p" id="S3.I2.i2.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I2.i2.p1.1.1">
          Historical Failure Rate Analysis:
         </span>
         Investigates past clinical trials and reported outcomes to determine the failure rates of drugs in similar contexts or against similar diseases. This analysis helps predict potential safety concerns in current applications.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I2.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I2.i3.p1">
        <p class="ltx_p" id="S3.I2.i3.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I2.i3.p1.1.1">
          Risk Assessment:
         </span>
         Employs statistical models to analyze the safety data and predict the risk of adverse effects when a drug is used to treat a particular disease. This predictive capability is vital for making informed decisions about drug prescriptions and usage.
        </p>
       </div>
      </li>
     </ul>
    </div>
    <div class="ltx_para" id="S3.SS2.SSS3.p3">
     <p class="ltx_p" id="S3.SS2.SSS3.p3.1">
      By systematically analyzing safety data and historical trial outcomes, the Safety Agent plays a crucial role in minimizing risks and enhancing patient safety in clinical settings.
     </p>
    </div>
   </section>
   <section class="ltx_subsubsection" id="S3.SS2.SSS4">
    <h4 class="ltx_title ltx_title_subsubsection">
     <span class="ltx_tag ltx_tag_subsubsection">
      3.2.4
     </span>
     Enrollment Agent
    </h4>
    <div class="ltx_para" id="S3.SS2.SSS4.p1">
     <p class="ltx_p" id="S3.SS2.SSS4.p1.1">
      Proper enrollment ensures that the trial has enough participants to statistically power the study. This is essential to detect the true effect of the intervention being tested. Insufficient enrollment can lead to inconclusive or unreliable results because the sample size determines the ability of a trial to accurately reflect the effects of a treatment.
We leverage a Hierarchical transformer-based model
      <cite class="ltx_cite ltx_citemacro_cite">
       <span class="ltx_text" style="font-size:90%;">
        Yue et al.
       </span>
       (
       <a class="ltx_ref" href="#bib.bib27" title="">
        <span class="ltx_text" style="font-size:90%;">
         2024
        </span>
       </a>
       )
      </cite>
      that takes eligibility criteria as an input feature and predicts the success rate of enrollment. It is a binary classification problem, where 1 denotes the successful enrollment while 0 does not. The details can be found in Section
      <a class="ltx_ref" href="#S3.SS3" title="3.3 Calling External Tools ‣ 3 Methods ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
       <span class="ltx_text ltx_ref_tag">
        3.3
       </span>
      </a>
      .
     </p>
    </div>
   </section>
  </section>
  <section class="ltx_subsection" id="S3.SS3">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     3.3
    </span>
    Calling External Tools
   </h3>
   <div class="ltx_para" id="S3.SS3.p1">
    <p class="ltx_p" id="S3.SS3.p1.1">
     GPT supports calling external tools (e.g., function, database retrieval) to leverage external knowledge and enhance its capability.
Specifically, suppose we have a couple of toolkits. GPT’s API can automatically detect which tool to use, which serves as glue to connect large language models to external tools.
Our system integrates a variety of external data sources and predictive AI models to support the agents’ functions.
    </p>
   </div>
   <section class="ltx_paragraph" id="S3.SS3.SSS0.Px1">
    <h5 class="ltx_title ltx_title_paragraph">
     Data Sources
    </h5>
    <div class="ltx_para" id="S3.SS3.SSS0.Px1.p1">
     <p class="ltx_p" id="S3.SS3.SSS0.Px1.p1.1">
      The use of professional datasets is pivotal in ensuring the accuracy and reliability of our agents’ information retrieval capabilities.
     </p>
     <ul class="ltx_itemize" id="S3.I3">
      <li class="ltx_item" id="S3.I3.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I3.i1.p1">
        <p class="ltx_p" id="S3.I3.i1.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I3.i1.p1.1.1">
          Drugbank:
         </span>
         Drugbank
         <cite class="ltx_cite ltx_citemacro_citep">
          (
          <span class="ltx_text" style="font-size:90%;">
           Wishart et al.
          </span>
          ,
          <a class="ltx_ref" href="#bib.bib25" title="">
           <span class="ltx_text" style="font-size:90%;">
            2018
           </span>
          </a>
          )
         </cite>
         stands out as a premier resource, offering detailed drug data, including chemical, pharmacological, and pharmaceutical information, with a focus on comprehensive drug-target interactions.
Drugbank is not only a repository of drug information but also serves as an invaluable tool for bioinformatics and cheminformatics research, providing data for over 13,000 drug entries including FDA-approved small molecule drugs, FDA-approved biopharmaceuticals (proteins, peptides, vaccines, and allergenics), and nutraceuticals.
         <br class="ltx_break"/>
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I3.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I3.i2.p1">
        <p class="ltx_p" id="S3.I3.i2.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I3.i2.p1.1.1">
          Hetionet:
         </span>
         Hetionet
         <cite class="ltx_cite ltx_citemacro_citep">
          (
          <span class="ltx_text" style="font-size:90%;">
           Himmelstein et al.
          </span>
          ,
          <a class="ltx_ref" href="#bib.bib7" title="">
           <span class="ltx_text" style="font-size:90%;">
            2017
           </span>
          </a>
          )
         </cite>
         is an integrative network of biology that encompasses a comprehensive collection of biological entities and their relationships.
It uniquely combines data from various biomedical databases covering diseases, genes, compounds, and more, into a single, coherent graph structure.
This interconnected approach allows for multifaceted analyses, including drug repurposing, genetic associations, and network medicine.
Hetionet includes over 47,000 nodes of different types (e.g., diseases, drugs, genes) and more than 2 million relationships, offering a rich dataset for computational biology and drug discovery.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I3.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I3.i3.p1">
        <p class="ltx_p" id="S3.I3.i3.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I3.i3.p1.1.1">
          LLM-generated data:
         </span>
         Large Language Models (LLM) like GPT-4 and its successors, have demonstrated remarkable capability as knowledge compressors and generators.
They can synthesize and extrapolate information from vast datasets to generate coherent, novel data points and insights.
In this research, we leverage LLMs to generate new knowledge relevant to our study, including hypothetical drug interactions, potential therapeutic targets, and model organism analyses.
This approach allows us to expand our dataset beyond traditional sources, incorporating generated insights that are validated against existing databases and literature.
The use of LLM-generated data introduces a novel dimension to our research, enabling the exploration of uncharted territories in drug discovery and biomedical research.
        </p>
       </div>
      </li>
     </ul>
    </div>
   </section>
   <section class="ltx_paragraph" id="S3.SS3.SSS0.Px2">
    <h5 class="ltx_title ltx_title_paragraph">
     Predictive AI Models
    </h5>
    <div class="ltx_para" id="S3.SS3.SSS0.Px2.p1">
     <p class="ltx_p" id="S3.SS3.SSS0.Px2.p1.1">
      We utilize multiple predictive AI models within our framework to ensure the accuracy and reliability of our agents’ abilities:
     </p>
    </div>
    <div class="ltx_para" id="S3.SS3.SSS0.Px2.p2">
     <ul class="ltx_itemize" id="S3.I4">
      <li class="ltx_item" id="S3.I4.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I4.i1.p1">
        <p class="ltx_p" id="S3.I4.i1.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I4.i1.p1.1.1">
          Enrollment Model:
         </span>
         The enrollment model is designed to predict the likelihood of successful participant enrollment in clinical trials based on the eligibility criteria, the drugs involved, and the diseases targeted. This is a hierarchical transformer-based model, integrating sentence embeddings from BioBERT
         <cite class="ltx_cite ltx_citemacro_citep">
          (
          <span class="ltx_text" style="font-size:90%;">
           Lee et al.
          </span>
          ,
          <a class="ltx_ref" href="#bib.bib11" title="">
           <span class="ltx_text" style="font-size:90%;">
            2020
           </span>
          </a>
          )
         </cite>
         to capture the nuanced medical semantics in the criteria text. In practice, the Enrollment Agent receives a query containing the drugs, diseases, and detailed eligibility criteria. It processes this information to predict the enrollment difficulty, which aids in planning and adjusting recruitment strategies for clinical trials. This capability supports more efficient trial design and can significantly impact the speed and success of new drug developments. The details can be found in
         <a class="ltx_ref" href="#A1.SS1" title="A.1 Enrollment Model ‣ Appendix A ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
          <span class="ltx_text ltx_ref_tag">
           A.1
          </span>
         </a>
         .
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I4.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I4.i2.p1">
        <p class="ltx_p" id="S3.I4.i2.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I4.i2.p1.1.1">
          Drug Risk Model:
         </span>
         The Drug Risk Model is designed to estimate the likelihood of a drug not achieving the desired therapeutic effect in clinical trials. This model is based on historical data of drug performances across various trials. Using a simple but effective approach, each drug is represented by its historical success rate, calculated as the mean of its trial outcomes (1 for success and 0 for failure).
        </p>
       </div>
       <div class="ltx_para" id="S3.I4.i2.p2">
        <p class="ltx_p" id="S3.I4.i2.p2.1">
         We store these success rates in a precomputed dictionary and utilize a lookup mechanism to assess drug risk rapidly. For drugs not found in the dictionary, a matching function approximates the closest drug name to ensure robust risk assessments. This method allows for quick and accurate risk estimations in real-time decision-making processes and is particularly useful in early-stage drug development and trial planning.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I4.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I4.i3.p1">
        <p class="ltx_p" id="S3.I4.i3.p1.1">
         <span class="ltx_text ltx_font_bold" id="S3.I4.i3.p1.1.1">
          Disease Risk Model:
         </span>
         Parallel to the Drug Risk Model, the Disease Risk Model calculates the probability of unsatisfactory treatment outcomes associated with specific diseases. This model aggregates historical trial data to determine success rates for diseases, which are then inverted to represent risk levels.
        </p>
       </div>
       <div class="ltx_para" id="S3.I4.i3.p2">
        <p class="ltx_p" id="S3.I4.i3.p2.1">
         Similar to the drugs model, each disease’s risk is precomputed and stored. The model employs sophisticated string-matching techniques to accommodate variations in disease naming conventions, ensuring accurate risk evaluations. This model aids in the prioritization of diseases in clinical research and helps in forecasting the challenges in achieving successful treatment outcomes.
        </p>
       </div>
      </li>
     </ul>
    </div>
   </section>
  </section>
  <section class="ltx_subsection" id="S3.SS4">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     3.4
    </span>
    Integration of Reasoning Technology
   </h3>
   <div class="ltx_para" id="S3.SS4.p1">
    <p class="ltx_p" id="S3.SS4.p1.1">
     To further enhance the agent’s decision-making capabilities, we integrate advanced reasoning technologies such as ReAct (recognition, action, and context)
     <cite class="ltx_cite ltx_citemacro_citep">
      (
      <span class="ltx_text" style="font-size:90%;">
       Yao et al.
      </span>
      ,
      <a class="ltx_ref" href="#bib.bib26" title="">
       <span class="ltx_text" style="font-size:90%;">
        2022
       </span>
      </a>
      )
     </cite>
     and the Least-to-Most reasoning framework
     <cite class="ltx_cite ltx_citemacro_citep">
      (
      <span class="ltx_text" style="font-size:90%;">
       Zhou et al.
      </span>
      ,
      <a class="ltx_ref" href="#bib.bib31" title="">
       <span class="ltx_text" style="font-size:90%;">
        2022
       </span>
      </a>
      )
     </cite>
     . These methodologies complement each other by providing robust mechanisms for addressing complex problems through structured and contextual analysis.
    </p>
   </div>
   <div class="ltx_para" id="S3.SS4.p2">
    <p class="ltx_p" id="S3.SS4.p2.1">
     <span class="ltx_text ltx_font_bold" id="S3.SS4.p2.1.1">
      ReAct Reasoning:
     </span>
     ReAct reasoning is a holistic approach that emphasizes the critical roles of recognition (Re), action (A), and context (Ct) in effective problem-solving. This methodology advocates for the identification of patterns or cues (recognition), the formulation and execution of a course of action (action), and the careful consideration of the surrounding circumstances (context). By integrating these elements, ReAct equips agents to make informed and precise decisions rapidly, an asset, particularly in dynamic and unpredictable environments.
    </p>
   </div>
   <div class="ltx_para" id="S3.SS4.p3">
    <p class="ltx_p" id="S3.SS4.p3.1">
     <span class="ltx_text ltx_font_bold" id="S3.SS4.p3.1.1">
      Least-to-Most Reasoning:
     </span>
     In contrast, the Least-to-Most reasoning method adopts a hierarchical approach to problem-solving. It suggests beginning with the simplest or least complex aspects and gradually progressing to address more intricate components. This structured problem-solving sequence ensures that foundational elements are thoroughly understood before advancing to tackle more complex layers of the issue. This method is valuable in educational contexts and when dealing with new or unfamiliar concepts, promoting a comprehensive understanding and preventing potential oversights.
    </p>
   </div>
   <div class="ltx_para" id="S3.SS4.p4">
    <p class="ltx_p" id="S3.SS4.p4.1">
     <span class="ltx_text ltx_font_bold" id="S3.SS4.p4.1.1">
      Synergistic Integration:
     </span>
     By combining ReAct and Least-to-Most reasoning, we can formulate a synergistic strategy that leverages the strengths of both methods. Initially, the Least-to-Most framework decomposes a problem into its elemental parts, organizing them from simplest to most complex. Subsequently, within this structured framework, ReAct reasoning is applied to each segment. This involves recognizing relevant patterns or cues, deciding on appropriate actions based on these insights, and adapting these actions by considering the immediate context. This integrative approach not only ensures a methodical breakdown of problems but also adopts solutions dynamically to meet the specific demands of each scenario.
    </p>
   </div>
  </section>
  <section class="ltx_subsection" id="S3.SS5">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     3.5
    </span>
    Workflow
   </h3>
   <div class="ltx_para" id="S3.SS5.p1">
    <p class="ltx_p" id="S3.SS5.p1.1">
     The workflow of our CT-Agent system is designed to optimize the collaboration and efficiency of multiple specialized agents to address complex medical inquiries. The process is structured in several sequential steps, as described below:
    </p>
   </div>
   <section class="ltx_paragraph" id="S3.SS5.SSS0.Px1">
    <h5 class="ltx_title ltx_title_paragraph">
     Step 1: Initial Planning and Problem Decomposition
    </h5>
    <div class="ltx_para" id="S3.SS5.SSS0.Px1.p1">
     <p class="ltx_p" id="S3.SS5.SSS0.Px1.p1.1">
      The workflow begins with the Planning Agent, which takes the lead in assessing the user’s query. Utilizing the LEAST-TO-MOST Reasoning method, this agent decomposes the complex problem into simpler, more manageable subproblems. This structured breakdown is crucial as it allows for targeted problem-solving by directing specific tasks to the most appropriate specialist agents.
     </p>
    </div>
   </section>
   <section class="ltx_paragraph" id="S3.SS5.SSS0.Px2">
    <h5 class="ltx_title ltx_title_paragraph">
     Step 2: Task Allocation to Specialist Agents
    </h5>
    <div class="ltx_para" id="S3.SS5.SSS0.Px2.p1">
     <p class="ltx_p" id="S3.SS5.SSS0.Px2.p1.1">
      Once the problem is decomposed, the Planning Agent allocates each subproblem to the respective specialist agents. For example:
     </p>
     <ul class="ltx_itemize" id="S3.I5">
      <li class="ltx_item" id="S3.I5.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I5.i1.p1">
        <p class="ltx_p" id="S3.I5.i1.p1.1">
         The Efficacy Agent is tasked with assessing drug effectiveness against specific diseases.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I5.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I5.i2.p1">
        <p class="ltx_p" id="S3.I5.i2.p1.1">
         The Safety Agent evaluates potential risks and adverse effects associated with the drug.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S3.I5.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S3.I5.i3.p1">
        <p class="ltx_p" id="S3.I5.i3.p1.1">
         The Enrollment Agent handles the feasibility and strategies for patient enrollment in clinical trials.
        </p>
       </div>
      </li>
     </ul>
     <p class="ltx_p" id="S3.SS5.SSS0.Px2.p1.2">
      Each agent operates independently, utilizing its specialized models and databases to process and analyze the assigned task.
     </p>
    </div>
   </section>
   <section class="ltx_paragraph" id="S3.SS5.SSS0.Px3">
    <h5 class="ltx_title ltx_title_paragraph">
     Step 3: Independent Agent Processing
    </h5>
    <div class="ltx_para" id="S3.SS5.SSS0.Px3.p1">
     <p class="ltx_p" id="S3.SS5.SSS0.Px3.p1.1">
      Each specialist agent processes its assigned subproblems using specific methodologies and external tools. This includes retrieving and analyzing data from sources like DrugBank and HetioNet, applying predictive models, and generating insights based on the agent’s specialty. The agents may also call external functions or databases to enhance their assessments or predictions.
     </p>
    </div>
   </section>
   <section class="ltx_paragraph" id="S3.SS5.SSS0.Px4">
    <h5 class="ltx_title ltx_title_paragraph">
     Step 4: Synthesis of Findings
    </h5>
    <div class="ltx_para" id="S3.SS5.SSS0.Px4.p1">
     <p class="ltx_p" id="S3.SS5.SSS0.Px4.p1.1">
      After each agent completes its task, the results are sent back to the Planning Agent. This agent synthesizes the findings from all the specialists, creating a comprehensive response that integrates all aspects of the problem, from drug efficacy and safety to enrollment potential.
     </p>
    </div>
   </section>
   <section class="ltx_paragraph" id="S3.SS5.SSS0.Px5">
    <h5 class="ltx_title ltx_title_paragraph">
     Step 5: Reasoning and Final Decision Making
    </h5>
    <div class="ltx_para" id="S3.SS5.SSS0.Px5.p1">
     <p class="ltx_p" id="S3.SS5.SSS0.Px5.p1.1">
      The final step involves applying the ReAct reasoning method to the synthesized findings. Here, the Planning Agent, enhanced by few-shot learning capabilities, examines the context and details of the integrated response to make informed decisions. This approach ensures that the final recommendation or solution is not only based on segmented analysis but also considers the interdependencies and broader implications of the combined agent findings.
     </p>
    </div>
   </section>
   <section class="ltx_paragraph" id="S3.SS5.SSS0.Px6">
    <h5 class="ltx_title ltx_title_paragraph">
     Step 6: Delivery of Solution
    </h5>
    <div class="ltx_para" id="S3.SS5.SSS0.Px6.p1">
     <p class="ltx_p" id="S3.SS5.SSS0.Px6.p1.1">
      The completed solution, which encompasses a detailed and reasoned response based on the collective intelligence of the multi-agent system, is then delivered to the user. This response not only addresses the initial query but also provides explanatory insights that justify the recommendations, thereby enhancing user trust and understanding.
     </p>
    </div>
    <div class="ltx_para" id="S3.SS5.SSS0.Px6.p2">
     <p class="ltx_p" id="S3.SS5.SSS0.Px6.p2.1">
      This structured workflow ensures that CT-Agent effectively mimics a collaborative team of medical specialists, offering precise and comprehensive solutions to complex medical inquiries.
     </p>
    </div>
   </section>
  </section>
 </section>
 <section class="ltx_section" id="S4">
  <h2 class="ltx_title ltx_title_section">
   <span class="ltx_tag ltx_tag_section">
    4
   </span>
   Experiment
  </h2>
  <div class="ltx_para" id="S4.p1">
   <p class="ltx_p" id="S4.p1.1">
    This section outlines the experimental design used to assess the performance of CT-Agent in the setting of clinical trials. We aim to demonstrate the superior predictive capabilities of our model by comparing it with established baseline methods.
   </p>
  </div>
  <section class="ltx_subsection" id="S4.SS1">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     4.1
    </span>
    Baseline Methods
   </h3>
   <div class="ltx_para" id="S4.SS1.p1">
    <p class="ltx_p" id="S4.SS1.p1.1">
     To ensure a comprehensive evaluation, we have selected diverse baseline methods known for their robustness in similar tasks:
    </p>
    <ol class="ltx_enumerate" id="S4.I1">
     <li class="ltx_item" id="S4.I1.i1" style="list-style-type:none;">
      <span class="ltx_tag ltx_tag_item">
       1.
      </span>
      <div class="ltx_para" id="S4.I1.i1.p1">
       <p class="ltx_p" id="S4.I1.i1.p1.1">
        <span class="ltx_text ltx_font_bold" id="S4.I1.i1.p1.1.1">
         Gradient-Boosted Decision Trees (GBDT)
        </span>
        : This method integrates embeddings for drugs, diseases, and eligibility criteria derived from BioBERT. The concatenated embeddings are then processed using LightGBM, a popular gradient-boosting framework that is highly efficient and scalable, making it suitable for handling complex datasets typical in clinical trials.
       </p>
      </div>
     </li>
     <li class="ltx_item" id="S4.I1.i2" style="list-style-type:none;">
      <span class="ltx_tag ltx_tag_item">
       2.
      </span>
      <div class="ltx_para" id="S4.I1.i2.p1">
       <p class="ltx_p" id="S4.I1.i2.p1.1">
        <span class="ltx_text ltx_font_bold" id="S4.I1.i2.p1.1.1">
         Hierarchical Attention Transformer (HAtten)
        </span>
        : Employing BioBERT embeddings for drugs and diseases, this model introduces a hierarchical attention mechanism. It systematically focuses on different granularity levels, from entire paragraphs to specific sentences within the eligibility criteria, enhancing its ability to discern relevant information. The process culminates in a two-layer Multilayer Perceptron (MLP), which aids in refining the decision process.
       </p>
      </div>
     </li>
     <li class="ltx_item" id="S4.I1.i3" style="list-style-type:none;">
      <span class="ltx_tag ltx_tag_item">
       3.
      </span>
      <div class="ltx_para" id="S4.I1.i3.p1">
       <p class="ltx_p" id="S4.I1.i3.p1.1">
        <span class="ltx_text ltx_font_bold" id="S4.I1.i3.p1.1.1">
         Standard Prompting
        </span>
        : As a control, this baseline employs large language models (LLMs) GPT-4
        <cite class="ltx_cite ltx_citemacro_citep">
         (
         <span class="ltx_text" style="font-size:90%;">
          OpenAI
         </span>
         ,
         <a class="ltx_ref" href="#bib.bib19" title="">
          <span class="ltx_text" style="font-size:90%;">
           2022
          </span>
         </a>
         )
        </cite>
        in their standard configuration. It tests the hypothesis that without tailored adaptations or integrations of external data, the pre-trained knowledge embedded within LLMs can competently perform outcome prediction in clinical trials, albeit potentially less effectively than more specialized approaches.
       </p>
      </div>
     </li>
    </ol>
   </div>
   <div class="ltx_para" id="S4.SS1.p2">
    <p class="ltx_p" id="S4.SS1.p2.1">
     This comparative analysis will help in highlighting the strengths and potential areas for improvement in CT-Agent, guiding future enhancements in the model’s architecture and its application in clinical trial settings.
    </p>
   </div>
  </section>
  <section class="ltx_subsection" id="S4.SS2">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     4.2
    </span>
    Experimental Setup
   </h3>
   <div class="ltx_para" id="S4.SS2.p1">
    <p class="ltx_p" id="S4.SS2.p1.1">
     Our experimental framework was implemented on a server equipped with an AMD Ryzen 9 3950X CPU, 64GB RAM, and an NVIDIA RTX 3080 Ti GPU. We utilized Python 3.8 for scripting and PyTorch for model implementation and training. For each experiment, we used the same seed to ensure reproducibility.
    </p>
   </div>
   <section class="ltx_subsubsection" id="S4.SS2.SSS1">
    <h4 class="ltx_title ltx_title_subsubsection">
     <span class="ltx_tag ltx_tag_subsubsection">
      4.2.1
     </span>
     Data and Resources
    </h4>
    <div class="ltx_para" id="S4.SS2.SSS1.p1">
     <p class="ltx_p" id="S4.SS2.SSS1.p1.1">
      In our investigation, we employed a variety of datasets and external resources to construct a comprehensive experimental framework. These are detailed as follows:
     </p>
    </div>
    <div class="ltx_para" id="S4.SS2.SSS1.p2">
     <ul class="ltx_itemize" id="S4.I2">
      <li class="ltx_item" id="S4.I2.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S4.I2.i1.p1">
        <p class="ltx_p" id="S4.I2.i1.p1.1">
         <span class="ltx_text ltx_font_bold" id="S4.I2.i1.p1.1.1">
          Drug Databases:
         </span>
         We used
         <em class="ltx_emph ltx_font_italic" id="S4.I2.i1.p1.1.2">
          DrugBank
         </em>
         , a comprehensive, freely accessible online database containing information on drugs and drug targets. DrugBank is instrumental for acquiring detailed pharmacological information, which supports the pharmacokinetics and molecular mechanisms of drug actions in our models.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S4.I2.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S4.I2.i2.p1">
        <p class="ltx_p" id="S4.I2.i2.p1.1">
         <span class="ltx_text ltx_font_bold" id="S4.I2.i2.p1.1.1">
          Knowledge Graphs:
         </span>
         <em class="ltx_emph ltx_font_italic" id="S4.I2.i2.p1.1.2">
          Hetionet
         </em>
         , an integrative knowledge graph of biomedical information that interlinks biological entities through relationships, was utilized. It provides a structured form of data that helps in understanding complex drug-disease relationships, gene interactions, and more, which are crucial for our predictive analytics.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S4.I2.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S4.I2.i3.p1">
        <p class="ltx_p" id="S4.I2.i3.p1.1">
         <span class="ltx_text ltx_font_bold" id="S4.I2.i3.p1.1.1">
          ClinicalTrials.gov:
         </span>
         We extracted data from
         <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://clinicaltrials.gov/" target="_blank" title="">
          https://clinicaltrials.gov/
         </a>
         , which includes information from both completed and ongoing clinical trials. This data is essential for validating our predictive models and for training them to understand clinical outcomes based on past trial data.
        </p>
       </div>
      </li>
      <li class="ltx_item" id="S4.I2.i4" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="S4.I2.i4.p1">
        <p class="ltx_p" id="S4.I2.i4.p1.1">
         <span class="ltx_text ltx_font_bold" id="S4.I2.i4.p1.1.1">
          GPT-Generated Data:
         </span>
         To augment our dataset and test the robustness of our models under varied scenarios, we generated synthetic data. This was done by prompting Generative Pre-trained Transformer (GPT) models with scenarios derived from our existing datasets, thereby creating realistic, hypothetical data scenarios for further testing.
        </p>
       </div>
      </li>
     </ul>
    </div>
    <div class="ltx_para" id="S4.SS2.SSS1.p3">
     <p class="ltx_p" id="S4.SS2.SSS1.p3.1">
      These diverse resources were meticulously integrated into our model, referred to hereafter as CT-Agent. This integration helps in providing a rich and informed context for each predictive task undertaken by our model.
     </p>
    </div>
    <div class="ltx_para" id="S4.SS2.SSS1.p4">
     <p class="ltx_p" id="S4.SS2.SSS1.p4.1">
      For our experimental validation, we randomly selected 40 training samples from the clinical trial outcome prediction benchmark provided in
      <cite class="ltx_cite ltx_citemacro_citep">
       (
       <span class="ltx_text" style="font-size:90%;">
        Fu et al.
       </span>
       ,
       <a class="ltx_ref" href="#bib.bib4" title="">
        <span class="ltx_text" style="font-size:90%;">
         2022
        </span>
       </a>
       )
      </cite>
      . An identical approach was used to select 40 samples from the test set, ensuring that our training and testing datasets were balanced and randomized, thereby providing a robust evaluation framework for our model’s performance.
     </p>
    </div>
    <figure class="ltx_table" id="S4.T1">
     <figcaption class="ltx_caption">
      <span class="ltx_tag ltx_tag_table">
       Table 1:
      </span>
      A real example of CT-Agent on clinical trial outcome prediction.
     </figcaption>
     <table class="ltx_tabular ltx_align_middle" id="S4.T1.1">
      <tbody class="ltx_tbody">
       <tr class="ltx_tr" id="S4.T1.1.1.1">
        <td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="S4.T1.1.1.1.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.1.1.1.1">
          <span class="ltx_p" id="S4.T1.1.1.1.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.1.1.1.1.1.1">
            User Input
           </span>
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.2.2">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.2.2.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.2.2.1.1">
          <span class="ltx_p" id="S4.T1.1.2.2.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.2.2.1.1.1.1">
            Problem
           </span>
           : I have designed a clinical trial and hope you can help me predict whether this trial can pass.
Features contain (1)
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.2.2.1.1.1.2">
            drug
           </span>
           : Aggrenox capsule;
(2)
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.2.2.1.1.1.3">
            disease
           </span>
           : cerebrovascular accident;
(3)
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.2.2.1.1.1.4">
            inclusion criteria
           </span>
           : …; (4)
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.2.2.1.1.1.5">
            exclusion criteria
           </span>
           : …;
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.3.3">
        <td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="S4.T1.1.3.3.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.3.3.1.1">
          <span class="ltx_p" id="S4.T1.1.3.3.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.3.3.1.1.1.1">
            Planning (Planning Agent)
           </span>
           : few-shot learning
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.4.4">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.4.4.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.4.4.1.1">
          <span class="ltx_p" id="S4.T1.1.4.4.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.4.4.1.1.1.1">
            Thought
           </span>
           : Decompose a complex problem into several easier subproblems.
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.5.5">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.5.5.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.5.5.1.1">
          <span class="ltx_p" id="S4.T1.1.5.5.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.5.5.1.1.1.1">
            Subproblem 1 (Enrollment Agent)
           </span>
           : Determine the level of enrollment feasibility based on inclusion and exclusion criteria.
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.6.6">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.6.6.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.6.6.1.1">
          <span class="ltx_p" id="S4.T1.1.6.6.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.6.6.1.1.1.1">
            Subproblem 2 (Safety Agent)
           </span>
           : Evaluate the safety of the drug “aggrenox capsule”
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.7.7">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.7.7.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.7.7.1.1">
          <span class="ltx_p" id="S4.T1.1.7.7.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.7.7.1.1.1.1">
            Subproblem 3 (Efficacy Agent)
           </span>
           : Assess the efficacy of the drug “aggrenox capsule” on the disease “cerebrovascular accident”
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.8.8">
        <td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="S4.T1.1.8.8.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.8.8.1.1">
          <span class="ltx_p" id="S4.T1.1.8.8.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.8.8.1.1.1.1">
            Solving Subproblems
           </span>
           :
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.9.9">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.9.9.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.9.9.1.1">
          <span class="ltx_p" id="S4.T1.1.9.9.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.9.9.1.1.1.1">
            Subproblem 1 (Enrollment Agent)
           </span>
           : Detect the function “enrollment prediction model” should to be called. The predicted enrollment failure rate is 0.3597.
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.10.10">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.10.10.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.10.10.1.1">
          <span class="ltx_p" id="S4.T1.1.10.10.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.10.10.1.1.1.1">
            Subproblem 2 (Safety Agent)
           </span>
           : Detect functions “disease historical statistics” and “drug historical statistics” should be called. The historical failure rate of Aggrenox capsules in clinical trials is reported as 1.0.
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.11.11">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.11.11.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.11.11.1.1">
          <span class="ltx_p" id="S4.T1.1.11.11.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.11.11.1.1.1.1">
            Subproblem 3 (Efficacy Agent)
           </span>
           : Detect functions “retrieve Hetionet knowledge graph” and “retrieve DrugBank database” should be called.
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.12.12">
        <td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="S4.T1.1.12.12.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.12.12.1.1">
          <span class="ltx_p" id="S4.T1.1.12.12.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.12.12.1.1.1.1">
            Reasoning (Reasoning Agent)
           </span>
           : few-shot learning
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.13.13">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.13.13.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.13.13.1.1">
          <span class="ltx_p" id="S4.T1.1.13.13.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.13.13.1.1.1.1">
            1. Enrollment Difficulty (Enrollment Agent)
           </span>
           : A moderate level of difficulty in enrolling participants. This could pose challenges in recruiting a sufficient number of eligible patients meeting the strict inclusion and exclusion criteria set for the trial, which can impact the overall success rate of the trial.
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.14.14">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.14.14.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.14.14.1.1">
          <span class="ltx_p" id="S4.T1.1.14.14.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.14.14.1.1.1.1">
            2. Safety of Aggrenox Capsule (Safety Agent)
           </span>
           : The historical failure rate of Aggrenox capsules (1.0) indicates a lack of success in previous trials. Evaluating the safety profile of the Aggrenox capsule in the context of cerebral infarction and cerebrovascular accidents is crucial due to the specified risk factors and exclusion criteria. The presence of risk factors like hypertension, diabetes, and other conditions among the trial participants might raise concerns about potential interactions or adverse effects of Aggrenox, contributing to the historical failure rate.
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.15.15">
        <td class="ltx_td ltx_align_justify ltx_align_top" id="S4.T1.1.15.15.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.15.15.1.1">
          <span class="ltx_p" id="S4.T1.1.15.15.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.15.15.1.1.1.1">
            3. Efficacy of Aggrenox Capsule (Efficacy Agent)
           </span>
           : The Aggrenox capsule, combining aspirin and dipyridamole, is beneficial in preventing strokes by its antiplatelet and vasodilatory effects. Given the criteria provided for the clinical trial specifically targeting patients with a history of cerebral infarction who meet certain age and risk factor requirements, Aggrenox capsule’s mechanism of action aligns with the needs of this patient population.
          </span>
         </span>
        </td>
       </tr>
       <tr class="ltx_tr" id="S4.T1.1.16.16">
        <td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="S4.T1.1.16.16.1">
         <span class="ltx_inline-block ltx_align_top" id="S4.T1.1.16.16.1.1">
          <span class="ltx_p" id="S4.T1.1.16.16.1.1.1" style="width:433.6pt;">
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.16.16.1.1.1.1">
            Draw Conclusion
           </span>
           . Therefore, considering the historical failure rate, safety concerns, and the moderate enrollment difficulty level associated with the clinical trial design and the use of the Aggrenox capsule in patients with cerebral infarction, the predicted success rate of the trial is low, at 0.0.
           <span class="ltx_text ltx_font_bold" id="S4.T1.1.16.16.1.1.1.2">
            (groundtruth is 0)
           </span>
          </span>
         </span>
        </td>
       </tr>
      </tbody>
     </table>
    </figure>
   </section>
   <section class="ltx_subsubsection" id="S4.SS2.SSS2">
    <h4 class="ltx_title ltx_title_subsubsection">
     <span class="ltx_tag ltx_tag_subsubsection">
      4.2.2
     </span>
     Procedure
    </h4>
    <div class="ltx_para" id="S4.SS2.SSS2.p1">
     <p class="ltx_p" id="S4.SS2.SSS2.p1.1">
      Each agent in CT-Agent was tasked with specific roles, as outlined in the Methods section. The Safety Agent queried drug databases, the Efficacy Agent analyzed disease information, the Enrollment Agent predicted the enrollment difficulty, and the Planning Agent synthesized these findings into actionable insights. We conducted experiments to assess the accuracy of outcome predictions and the precision of failure reason identifications.
     </p>
    </div>
   </section>
  </section>
  <section class="ltx_subsection" id="S4.SS3">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     4.3
    </span>
    Implementation Details
   </h3>
   <div class="ltx_para" id="S4.SS3.p1">
    <p class="ltx_p" id="S4.SS3.p1.1">
     In this section, we provide detailed descriptions of the implementation processes to enhance the reproducibility of our study.
    </p>
   </div>
   <section class="ltx_paragraph" id="S4.SS3.SSS0.Px1">
    <h5 class="ltx_title ltx_title_paragraph">
     Role Assignment to Agents
    </h5>
    <div class="ltx_para" id="S4.SS3.SSS0.Px1.p1">
     <p class="ltx_p" id="S4.SS3.SSS0.Px1.p1.1">
      Each agent within the CT-Agent framework is designated a specific role, which is integrated directly into the LLM’s system prompt for clarity and focus. For instance, the role of the Efficacy Agent is defined as follows:
     </p>
     <blockquote class="ltx_quote" id="S4.SS3.SSS0.Px1.p1.2">
      <p class="ltx_p" id="S4.SS3.SSS0.Px1.p1.2.1">
       ”As an efficacy expert, you have the capability to assess a drug’s efficacy against diseases by examining its effectiveness on the disease.”
      </p>
     </blockquote>
     <p class="ltx_p" id="S4.SS3.SSS0.Px1.p1.3">
      This role definition is crucial as it guides the LLM to prioritize responses based on the assigned expert domain, leveraging the model’s inherent capability to focus more acutely on instructed tasks than on general information.
     </p>
    </div>
   </section>
   <section class="ltx_paragraph" id="S4.SS3.SSS0.Px2">
    <h5 class="ltx_title ltx_title_paragraph">
     Defining External Tools
    </h5>
    <div class="ltx_para" id="S4.SS3.SSS0.Px2.p1">
     <p class="ltx_p" id="S4.SS3.SSS0.Px2.p1.1">
      External tools are defined in a structured format to facilitate their integration and usage within the LLM environment. These definitions are crafted in JSON format, specifying the function name, description, and necessary parameters. Examples can be found in the
      <a class="ltx_ref" href="#A1.SS2" title="A.2 Function Calling Definitions ‣ Appendix A ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
       <span class="ltx_text ltx_ref_tag">
        A.2
       </span>
      </a>
      .
     </p>
    </div>
    <div class="ltx_para" id="S4.SS3.SSS0.Px2.p2">
     <p class="ltx_p" id="S4.SS3.SSS0.Px2.p2.1">
      This structured approach allows for the direct transmission of function calls to the LLM, which in turn provides detailed responses including the function name and arguments. These responses enable the execution of functions locally and the retrieval of results in a structured manner.
     </p>
    </div>
   </section>
   <section class="ltx_paragraph" id="S4.SS3.SSS0.Px3">
    <h5 class="ltx_title ltx_title_paragraph">
     Enhanced Few-Shot Reasoning
    </h5>
    <div class="ltx_para" id="S4.SS3.SSS0.Px3.p1">
     <p class="ltx_p" id="S4.SS3.SSS0.Px3.p1.1">
      To improve the model’s reasoning capabilities, we incorporate examples of sub-problems and corresponding labels within the system prompt. This method, known as few-shot learning, aids the LLM in understanding the context and methodology required to solve complex problems by referencing similar, previously solved problems. This approach not only enhances the accuracy of the model’s outputs but also its ability to generalize from limited examples to new, unseen scenarios.
     </p>
    </div>
    <div class="ltx_para" id="S4.SS3.SSS0.Px3.p2">
     <p class="ltx_p" id="S4.SS3.SSS0.Px3.p2.1">
      These implementation strategies collectively ensure that each component of CT-Agent operates effectively and that the integration between different agents and external tools is seamless, fostering an environment conducive to robust, reproducible research.
     </p>
    </div>
   </section>
  </section>
  <section class="ltx_subsection" id="S4.SS4">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     4.4
    </span>
    Quantitative Results
   </h3>
   <div class="ltx_para" id="S4.SS4.p1">
    <p class="ltx_p" id="S4.SS4.p1.1">
     Table
     <a class="ltx_ref" href="#S4.T2" title="Table 2 ‣ 4.4 Quantitative Results ‣ 4 Experiment ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
      <span class="ltx_text ltx_ref_tag">
       2
      </span>
     </a>
     presents the performance of various methods. Specifically, CT-Agent obtain the highest ROC-AUC score at 0.8347 among all the compared methods. We observe that CT-Agent achieve competitive performance among all the well-established methods, e.g., GBDT.
Also, compared with the standard prompt (basic GPT) model, our method consistently improves all six evaluation metrics.
We also compare the performance of GPT-3.5 and GPT-4 for the standard prompting method in clinical trial outcome prediction. As observed in Table
     <a class="ltx_ref" href="#S4.T2" title="Table 2 ‣ 4.4 Quantitative Results ‣ 4 Experiment ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
      <span class="ltx_text ltx_ref_tag">
       2
      </span>
     </a>
     , GPT-4 demonstrates a superior performance across most metrics compared to GPT-3.5, suggesting that newer versions of large language models may offer incremental improvements in predicting clinical trial outcomes using the standard prompting method.
    </p>
   </div>
   <figure class="ltx_table" id="S4.T2">
    <figcaption class="ltx_caption ltx_centering" style="font-size:90%;">
     <span class="ltx_tag ltx_tag_table">
      Table 2:
     </span>
     Predictive performance of various methods.
    </figcaption>
    <table class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle" id="S4.T2.3">
     <thead class="ltx_thead">
      <tr class="ltx_tr" id="S4.T2.3.1.1">
       <th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_tt" id="S4.T2.3.1.1.1">
        <span class="ltx_text" id="S4.T2.3.1.1.1.1" style="font-size:90%;">
         Method
        </span>
       </th>
       <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T2.3.1.1.2">
        <span class="ltx_text" id="S4.T2.3.1.1.2.1" style="font-size:90%;">
         Accuracy
        </span>
       </th>
       <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T2.3.1.1.3">
        <span class="ltx_text" id="S4.T2.3.1.1.3.1" style="font-size:90%;">
         ROC-AUC
        </span>
       </th>
       <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T2.3.1.1.4">
        <span class="ltx_text" id="S4.T2.3.1.1.4.1" style="font-size:90%;">
         PR-AUC
        </span>
       </th>
       <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T2.3.1.1.5">
        <span class="ltx_text" id="S4.T2.3.1.1.5.1" style="font-size:90%;">
         Precision
        </span>
       </th>
       <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T2.3.1.1.6">
        <span class="ltx_text" id="S4.T2.3.1.1.6.1" style="font-size:90%;">
         Recall
        </span>
       </th>
       <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T2.3.1.1.7">
        <span class="ltx_text" id="S4.T2.3.1.1.7.1" style="font-size:90%;">
         F1
        </span>
       </th>
      </tr>
     </thead>
     <tbody class="ltx_tbody">
      <tr class="ltx_tr" id="S4.T2.3.2.1">
       <th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_t" id="S4.T2.3.2.1.1">
        <span class="ltx_text" id="S4.T2.3.2.1.1.1" style="font-size:90%;">
         GBDT
        </span>
       </th>
       <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T2.3.2.1.2">
        <span class="ltx_text" id="S4.T2.3.2.1.2.1" style="font-size:90%;">
         0.6250
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T2.3.2.1.3">
        <span class="ltx_text" id="S4.T2.3.2.1.3.1" style="font-size:90%;">
         0.8000
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T2.3.2.1.4">
        <span class="ltx_text" id="S4.T2.3.2.1.4.1" style="font-size:90%;">
         0.8669
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T2.3.2.1.5">
        <span class="ltx_text" id="S4.T2.3.2.1.5.1" style="font-size:90%;">
         0.6250
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T2.3.2.1.6">
        <span class="ltx_text ltx_font_bold" id="S4.T2.3.2.1.6.1" style="font-size:90%;">
         1.0000
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T2.3.2.1.7">
        <span class="ltx_text" id="S4.T2.3.2.1.7.1" style="font-size:90%;">
         0.7692
        </span>
       </td>
      </tr>
      <tr class="ltx_tr" id="S4.T2.3.3.2">
       <th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S4.T2.3.3.2.1">
        <span class="ltx_text" id="S4.T2.3.3.2.1.1" style="font-size:90%;">
         HAtten
        </span>
       </th>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.3.2.2">
        <span class="ltx_text ltx_font_bold" id="S4.T2.3.3.2.2.1" style="font-size:90%;">
         0.7500
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.3.2.3">
        <span class="ltx_text" id="S4.T2.3.3.2.3.1" style="font-size:90%;">
         0.7573
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.3.2.4">
        <span class="ltx_text ltx_font_bold" id="S4.T2.3.3.2.4.1" style="font-size:90%;">
         0.8718
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.3.2.5">
        <span class="ltx_text ltx_font_bold" id="S4.T2.3.3.2.5.1" style="font-size:90%;">
         0.8947
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.3.2.6">
        <span class="ltx_text" id="S4.T2.3.3.2.6.1" style="font-size:90%;">
         0.6800
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.3.2.7">
        <span class="ltx_text ltx_font_bold" id="S4.T2.3.3.2.7.1" style="font-size:90%;">
         0.7727
        </span>
       </td>
      </tr>
      <tr class="ltx_tr" id="S4.T2.3.4.3">
       <th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S4.T2.3.4.3.1">
        <span class="ltx_text" id="S4.T2.3.4.3.1.1" style="font-size:90%;">
         GPT-3.5
        </span>
       </th>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.4.3.2">
        <span class="ltx_text" id="S4.T2.3.4.3.2.1" style="font-size:90%;">
         0.5250
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.4.3.3">
        <span class="ltx_text" id="S4.T2.3.4.3.3.1" style="font-size:90%;">
         0.4853
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.4.3.4">
        <span class="ltx_text" id="S4.T2.3.4.3.4.1" style="font-size:90%;">
         0.4419
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.4.3.5">
        <span class="ltx_text" id="S4.T2.3.4.3.5.1" style="font-size:90%;">
         0.4000
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.4.3.6">
        <span class="ltx_text" id="S4.T2.3.4.3.6.1" style="font-size:90%;">
         0.5333
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.4.3.7">
        <span class="ltx_text" id="S4.T2.3.4.3.7.1" style="font-size:90%;">
         0.4571
        </span>
       </td>
      </tr>
      <tr class="ltx_tr" id="S4.T2.3.5.4">
       <th class="ltx_td ltx_align_left ltx_th ltx_th_row" id="S4.T2.3.5.4.1">
        <span class="ltx_text" id="S4.T2.3.5.4.1.1" style="font-size:90%;">
         GPT-4
        </span>
       </th>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.5.4.2">
        <span class="ltx_text" id="S4.T2.3.5.4.2.1" style="font-size:90%;">
         0.6500
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.5.4.3">
        <span class="ltx_text" id="S4.T2.3.5.4.3.1" style="font-size:90%;">
         0.6800
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.5.4.4">
        <span class="ltx_text" id="S4.T2.3.5.4.4.1" style="font-size:90%;">
         0.4582
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.5.4.5">
        <span class="ltx_text" id="S4.T2.3.5.4.5.1" style="font-size:90%;">
         0.5385
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.5.4.6">
        <span class="ltx_text" id="S4.T2.3.5.4.6.1" style="font-size:90%;">
         0.4666
        </span>
       </td>
       <td class="ltx_td ltx_align_center" id="S4.T2.3.5.4.7">
        <span class="ltx_text" id="S4.T2.3.5.4.7.1" style="font-size:90%;">
         0.5000
        </span>
       </td>
      </tr>
      <tr class="ltx_tr" id="S4.T2.3.6.5">
       <th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_bb" id="S4.T2.3.6.5.1">
        <span class="ltx_text" id="S4.T2.3.6.5.1.1" style="font-size:90%;">
         CT-Agent
        </span>
       </th>
       <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T2.3.6.5.2">
        <span class="ltx_text" id="S4.T2.3.6.5.2.1" style="font-size:90%;">
         0.7000
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T2.3.6.5.3">
        <span class="ltx_text ltx_font_bold" id="S4.T2.3.6.5.3.1" style="font-size:90%;">
         0.8347
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T2.3.6.5.4">
        <span class="ltx_text" id="S4.T2.3.6.5.4.1" style="font-size:90%;">
         0.7908
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T2.3.6.5.5">
        <span class="ltx_text" id="S4.T2.3.6.5.5.1" style="font-size:90%;">
         0.5714
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T2.3.6.5.6">
        <span class="ltx_text" id="S4.T2.3.6.5.6.1" style="font-size:90%;">
         0.8000
        </span>
       </td>
       <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T2.3.6.5.7">
        <span class="ltx_text" id="S4.T2.3.6.5.7.1" style="font-size:90%;">
         0.6667
        </span>
       </td>
      </tr>
     </tbody>
    </table>
   </figure>
  </section>
  <section class="ltx_subsection" id="S4.SS5">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     4.5
    </span>
    Case Study
   </h3>
   <div class="ltx_para" id="S4.SS5.p1">
    <p class="ltx_p" id="S4.SS5.p1.1">
     We analyze a realistic case study in Table
     <a class="ltx_ref" href="#S4.T1" title="Table 1 ‣ 4.2.1 Data and Resources ‣ 4.2 Experimental Setup ‣ 4 Experiment ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
      <span class="ltx_text ltx_ref_tag">
       1
      </span>
     </a>
     . The NCTID is NCT00311402. The trial focused on evaluating the treatment effect of the Aggrenox capsule on cerebrovascular accidents. Aggrenox is a popular drug and contains a combination of aspirin and dipyridamole. First, the user describes the problem using natural language: “I have designed a clinical trial and hope you can help me predict whether this trial can pass” and attach the drug name, disease name, and inclusion/exclusion criteria as features. Then, the Planning Agent decomposes the whole problem into three subproblems based on clinical knowledge. It was enhanced by few-shot learning, which gives some representative examples to the GPT as prompt.
These three subproblems are clinical trial enrollment, Aggrenox’s safety to human bodies, and Aggrenox’s efficacy in treating cerebrovascular accidents, which are handled by enrollment agents, safety agents, and efficacy agents, respectively.
These agents of subproblems can be solved with the help of external tools. For example, we train an enrollment success prediction model and the estimated enrollment failure probability is 0.359. Enrollment Agent automatically recognizes and call the predictive AI model and insert the results (0.359) into the text. Similarly, Safety Agent (drug safety) identifies that the historical failure rate of Aggrenox capsules is 100%, indicating its high risk.
Combining this information, the Reasoning Agent will make the final decision that the trial is highly likely to fail, which is correctly forecasted.
    </p>
   </div>
  </section>
  <section class="ltx_subsection" id="S4.SS6">
   <h3 class="ltx_title ltx_title_subsection">
    <span class="ltx_tag ltx_tag_subsection">
     4.6
    </span>
    Ablation Study
   </h3>
   <section class="ltx_subsubsection" id="S4.SS6.SSS1">
    <h4 class="ltx_title ltx_title_subsubsection">
     <span class="ltx_tag ltx_tag_subsubsection">
      4.6.1
     </span>
     Different Versions of GPT
    </h4>
    <div class="ltx_para" id="S4.SS6.SSS1.p1">
     <p class="ltx_p" id="S4.SS6.SSS1.p1.1">
      In this ablation study, we assess the effectiveness of two iterations of the Generative Pre-trained Transformer: GPT-3.5 and GPT-4. Our objective is to explore their performance in the context of predicting outcomes in clinical trials using a standard prompting approach. The comparative analysis focuses on key performance metrics, as detailed in the subsequent table.
     </p>
    </div>
    <div class="ltx_para" id="S4.SS6.SSS1.p2">
     <p class="ltx_p" id="S4.SS6.SSS1.p2.1">
      The results, as summarized in Table
      <a class="ltx_ref" href="#S4.T2" title="Table 2 ‣ 4.4 Quantitative Results ‣ 4 Experiment ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
       <span class="ltx_text ltx_ref_tag">
        2
       </span>
      </a>
      , distinctly illustrate that GPT-4 outperforms GPT-3.5 across the majority of evaluated metrics. This enhancement in performance with GPT-4 underscores the potential benefits of integrating more advanced versions of large language models in the domain of clinical trial outcome prediction.
     </p>
    </div>
   </section>
   <section class="ltx_subsubsection" id="S4.SS6.SSS2">
    <h4 class="ltx_title ltx_title_subsubsection">
     <span class="ltx_tag ltx_tag_subsubsection">
      4.6.2
     </span>
     Impact of Few-Shot Learning
    </h4>
    <div class="ltx_para" id="S4.SS6.SSS2.p1">
     <p class="ltx_p" id="S4.SS6.SSS2.p1.1">
      This segment of the ablation study investigates the influence of few-shot learning techniques when integrated into CT-Agent, a multi-agent framework employing large language models (LLMs). Our comparative analysis pits the version of the model that incorporates few-shot learning against a baseline version that does not utilize these adaptations. The comparative results are encapsulated in the table below.
     </p>
    </div>
    <figure class="ltx_table" id="S4.T3">
     <figcaption class="ltx_caption ltx_centering">
      <span class="ltx_tag ltx_tag_table">
       Table 3:
      </span>
      Impact of few-shot learning on CT-Agent performance.
     </figcaption>
     <div class="ltx_inline-block ltx_align_center ltx_transformed_outer" id="S4.T3.1" style="width:433.6pt;height:56.5pt;vertical-align:-0.0pt;">
      <span class="ltx_transformed_inner" style="transform:translate(9.7pt,-1.3pt) scale(1.04697704596815,1.04697704596815) ;">
       <table class="ltx_tabular ltx_guessed_headers ltx_align_middle" id="S4.T3.1.1">
        <thead class="ltx_thead">
         <tr class="ltx_tr" id="S4.T3.1.1.1.1">
          <th class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_th_row ltx_border_tt" id="S4.T3.1.1.1.1.1">
           Method
          </th>
          <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T3.1.1.1.1.2">
           Accuracy
          </th>
          <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T3.1.1.1.1.3">
           ROC-AUC
          </th>
          <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T3.1.1.1.1.4">
           PR-AUC
          </th>
          <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T3.1.1.1.1.5">
           Precision
          </th>
          <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T3.1.1.1.1.6">
           Recall
          </th>
          <th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_tt" id="S4.T3.1.1.1.1.7">
           F1
          </th>
         </tr>
        </thead>
        <tbody class="ltx_tbody">
         <tr class="ltx_tr" id="S4.T3.1.1.2.1">
          <th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_t" id="S4.T3.1.1.2.1.1">
           CT-Agent w few-shot
          </th>
          <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T3.1.1.2.1.2">
           0.7
          </td>
          <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T3.1.1.2.1.3">
           0.8347
          </td>
          <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T3.1.1.2.1.4">
           0.7908
          </td>
          <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T3.1.1.2.1.5">
           0.5714
          </td>
          <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T3.1.1.2.1.6">
           0.8
          </td>
          <td class="ltx_td ltx_align_center ltx_border_t" id="S4.T3.1.1.2.1.7">
           0.6667
          </td>
         </tr>
         <tr class="ltx_tr" id="S4.T3.1.1.3.2">
          <th class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_bb" id="S4.T3.1.1.3.2.1">
           CT-Agent w/o few-shot
          </th>
          <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T3.1.1.3.2.2">
           0.75
          </td>
          <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T3.1.1.3.2.3">
           0.824
          </td>
          <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T3.1.1.3.2.4">
           0.6793
          </td>
          <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T3.1.1.3.2.5">
           0.647
          </td>
          <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T3.1.1.3.2.6">
           0.7333
          </td>
          <td class="ltx_td ltx_align_center ltx_border_bb" id="S4.T3.1.1.3.2.7">
           0.6875
          </td>
         </tr>
        </tbody>
       </table>
      </span>
     </div>
    </figure>
    <div class="ltx_para" id="S4.SS6.SSS2.p2">
     <p class="ltx_p" id="S4.SS6.SSS2.p2.1">
      Table
      <a class="ltx_ref" href="#S4.T3" title="Table 3 ‣ 4.6.2 Impact of Few-Shot Learning ‣ 4.6 Ablation Study ‣ 4 Experiment ‣ CT-Agent: Clinical Trial Multi-Agent with Large Language Model-based Reasoning">
       <span class="ltx_text ltx_ref_tag">
        3
       </span>
      </a>
      reveals that while the accuracy and F1 score marginally favor the model without few-shot learning, the few-shot adapted model (CT-Agent) exhibits superior performance in terms of ROC-AUC and PR-AUC. This indicates that the incorporation of few-shot learning significantly bolsters the model’s proficiency in accurately classifying positive instances, despite a minor trade-off in overall accuracy and precision-recall balance.
     </p>
    </div>
   </section>
  </section>
 </section>
 <section class="ltx_section" id="S5">
  <h2 class="ltx_title ltx_title_section">
   <span class="ltx_tag ltx_tag_section">
    5
   </span>
   Discussion
  </h2>
  <div class="ltx_para" id="S5.p1">
   <p class="ltx_p" id="S5.p1.1">
    This study introduces the groundbreaking Multi-Agent Clinical Trial Helper (CT-Agent), a multi-agent framework that synergizes the advanced capabilities of GPT-4 with sophisticated agent architectures and cutting-edge reasoning technologies like LEAST-TO-MOST and ReAct. Our system significantly enhances the performance of large language models (LLMs) in clinical settings, managing complex trial processes and introducing novel functionalities such as predictive analytics, comprehensive failure analysis, and precise trial duration estimations.
   </p>
  </div>
  <div class="ltx_para" id="S5.p2">
   <p class="ltx_p" id="S5.p2.1">
    Our evaluations, which include computational benchmarks and expert feedback, underscore the efficiency and effectiveness of CT-Agentin improving clinical trial outcomes. This integration of LLMs with multi-agent systems not only manages the complexities inherent in clinical trials but also bridges the gap between conversational AI and actionable intelligence in healthcare. CT-Agentestablishes a new benchmark in the application of LLMs to clinical trials, promising a future where advanced AI tools play a crucial role in advancing medical research and patient care.
   </p>
  </div>
  <section class="ltx_paragraph" id="S5.SS0.SSS0.Px1">
   <h5 class="ltx_title ltx_title_paragraph">
    Limitations
   </h5>
   <div class="ltx_para" id="S5.SS0.SSS0.Px1.p1">
    <p class="ltx_p" id="S5.SS0.SSS0.Px1.p1.1">
     While the CT-Agent demonstrates the capability to automatically recognize and decompose user issues, directing them to specialized agents for resolution, it still relies significantly on human intervention for its design and configuration. This dependency on manual input for agent creation limits the system’s scalability and adaptability, particularly in dynamic environments where user requirements and contexts evolve rapidly. Further development could focus on integrating machine learning techniques to enable the CT-Agent to learn from interactions and autonomously update its problem-solving strategies, thereby reducing the need for frequent human oversight and redesign.
These points underscore the need for continuous research and development to fully realize the potential of AI-driven clinical trials, addressing both technical and clinical implications.
    </p>
   </div>
  </section>
 </section>
 <section class="ltx_bibliography" id="bib">
  <h2 class="ltx_title ltx_title_bibliography" style="font-size:90%;">
   References
  </h2>
  <ul class="ltx_biblist">
   <li class="ltx_bibitem" id="bib.bib1">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib1.5.5.1" style="font-size:90%;">
      Chang et al. (2019)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib1.7.1" style="font-size:90%;">
      Yi-Tan Chang, Eric P Hoffman, Guoqiang Yu, David M Herrington, Robert Clarke, Chiung-Ting Wu, Lulu Chen, and Yue Wang.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib1.8.1" style="font-size:90%;">
      Integrated identification of disease specific pathways using multi-omics data.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib1.9.1" style="font-size:90%;">
      bioRxiv
     </em>
     <span class="ltx_text" id="bib.bib1.10.2" style="font-size:90%;">
      , page 666065, 2019.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib2">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib2.5.5.1" style="font-size:90%;">
      Chen et al. (2021)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib2.7.1" style="font-size:90%;">
      Lulu Chen, Chiung-Ting Wu, Robert Clarke, Guoqiang Yu, Jennifer E Van Eyk, David M Herrington, and Yue Wang.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib2.8.1" style="font-size:90%;">
      Data-driven detection of subtype-specific differentially expressed genes.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib2.9.1" style="font-size:90%;">
      Scientific reports
     </em>
     <span class="ltx_text" id="bib.bib2.10.2" style="font-size:90%;">
      , 11(1):332, 2021.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib3">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib3.5.5.1" style="font-size:90%;">
      Chen et al. (2024)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib3.7.1" style="font-size:90%;">
      Tianyi Chen, Nan Hao, Yingzhou Lu, and Capucine Van Rechem.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib3.8.1" style="font-size:90%;">
      Uncertainty quantification on clinical trial outcome prediction.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib3.9.1" style="font-size:90%;">
      arXiv preprint arXiv:2401.03482
     </em>
     <span class="ltx_text" id="bib.bib3.10.2" style="font-size:90%;">
      , 2024.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib4">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib4.5.5.1" style="font-size:90%;">
      Fu et al. (2022)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib4.7.1" style="font-size:90%;">
      Tianfan Fu, Kexin Huang, Cao Xiao, Lucas M Glass, and Jimeng Sun.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib4.8.1" style="font-size:90%;">
      HINT: Hierarchical interaction network for clinical-trial-outcome predictions.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib4.9.1" style="font-size:90%;">
      Patterns
     </em>
     <span class="ltx_text" id="bib.bib4.10.2" style="font-size:90%;">
      , 3(4):100445, 2022.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib5">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib5.5.5.1" style="font-size:90%;">
      Gao et al. (2020)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib5.7.1" style="font-size:90%;">
      Junyi Gao, Cao Xiao, Lucas M Glass, and Jimeng Sun.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib5.8.1" style="font-size:90%;">
      COMPOSE: Cross-modal pseudo-siamese network for patient trial matching.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib5.9.1" style="font-size:90%;">
      In
     </span>
     <em class="ltx_emph ltx_font_italic" id="bib.bib5.10.2" style="font-size:90%;">
      Proceedings of the 26th ACM SIGKDD International Conference on Knowledge Discovery &amp; Data Mining
     </em>
     <span class="ltx_text" id="bib.bib5.11.3" style="font-size:90%;">
      , pages 803–812, 2020.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib6">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib6.5.5.1" style="font-size:90%;">
      Hendrycks et al. (2021)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib6.7.1" style="font-size:90%;">
      Dan Hendrycks, Collin Burns, Steven Basart, Andy Zou, Mantas Mazeika, Dawn Song, and Jacob Steinhardt.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib6.8.1" style="font-size:90%;">
      Measuring massive multitask language understanding.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib6.9.1" style="font-size:90%;">
      In
     </span>
     <em class="ltx_emph ltx_font_italic" id="bib.bib6.10.2" style="font-size:90%;">
      9th International Conference on Learning Representations, ICLR 2021, Virtual Event, Austria, May 3-7, 2021
     </em>
     <span class="ltx_text" id="bib.bib6.11.3" style="font-size:90%;">
      . OpenReview.net, 2021.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib6.12.1" style="font-size:90%;">
      URL
     </span>
     <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://openreview.net/forum?id=d7KBjmI3GmQ" style="font-size:90%;" target="_blank" title="">
      https://openreview.net/forum?id=d7KBjmI3GmQ
     </a>
     <span class="ltx_text" id="bib.bib6.13.2" style="font-size:90%;">
      .
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib7">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib7.5.5.1" style="font-size:90%;">
      Himmelstein et al. (2017)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib7.7.1" style="font-size:90%;">
      Daniel Scott Himmelstein, Antoine Lizee, Christine Hessler, Leo Brueggeman, Sabrina L Chen, Dexter Hadley, Ari Green, Pouya Khankhanian, and Sergio E Baranzini.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib7.8.1" style="font-size:90%;">
      Systematic integration of biomedical knowledge prioritizes drugs for repurposing.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib7.9.1" style="font-size:90%;">
      Elife
     </em>
     <span class="ltx_text" id="bib.bib7.10.2" style="font-size:90%;">
      , 6:e26726, 2017.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib8">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib8.5.5.1" style="font-size:90%;">
      Jin et al. (2020)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib8.7.1" style="font-size:90%;">
      Di Jin, Eileen Pan, Nassim Oufattole, Wei-Hung Weng, Hanyi Fang, and Peter Szolovits.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib8.8.1" style="font-size:90%;">
      What disease does this patient have? A large-scale open domain question answering dataset from medical exams.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib8.9.1" style="font-size:90%;">
      CoRR
     </em>
     <span class="ltx_text" id="bib.bib8.10.2" style="font-size:90%;">
      , abs/2009.13081, 2020.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib8.11.1" style="font-size:90%;">
      URL
     </span>
     <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://arxiv.org/abs/2009.13081" style="font-size:90%;" target="_blank" title="">
      https://arxiv.org/abs/2009.13081
     </a>
     <span class="ltx_text" id="bib.bib8.12.2" style="font-size:90%;">
      .
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib9">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib9.5.5.1" style="font-size:90%;">
      Jin et al. (2019)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib9.7.1" style="font-size:90%;">
      Qiao Jin, Bhuwan Dhingra, Zhengping Liu, William W. Cohen, and Xinghua Lu.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib9.8.1" style="font-size:90%;">
      Pubmedqa: A dataset for biomedical research question answering.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib9.9.1" style="font-size:90%;">
      In Kentaro Inui, Jing Jiang, Vincent Ng, and Xiaojun Wan, editors,
     </span>
     <em class="ltx_emph ltx_font_italic" id="bib.bib9.10.2" style="font-size:90%;">
      Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing, EMNLP-IJCNLP 2019, Hong Kong, China, November 3-7, 2019
     </em>
     <span class="ltx_text" id="bib.bib9.11.3" style="font-size:90%;">
      , pages 2567–2577. Association for Computational Linguistics, 2019.
     </span>
    </span>
    <span class="ltx_bibblock">
     <a class="ltx_ref" href="https:/doi.org/10.18653/V1/D19-1259" style="font-size:90%;" target="_blank" title="">
      10.18653/V1/D19-1259
     </a>
     <span class="ltx_text" id="bib.bib9.12.1" style="font-size:90%;">
      .
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib9.13.1" style="font-size:90%;">
      URL
     </span>
     <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://doi.org/10.18653/v1/D19-1259" style="font-size:90%;" target="_blank" title="">
      https://doi.org/10.18653/v1/D19-1259
     </a>
     <span class="ltx_text" id="bib.bib9.14.2" style="font-size:90%;">
      .
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib10">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib10.5.5.1" style="font-size:90%;">
      Kung et al. (2023)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib10.7.1" style="font-size:90%;">
      Tiffany H Kung, Morgan Cheatham, Arielle Medenilla, Czarina Sillos, Lorie De Leon, Camille Elepaño, Maria Madriaga, Rimel Aggabao, Giezel Diaz-Candido, James Maningo, et al.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib10.8.1" style="font-size:90%;">
      Performance of chatgpt on usmle: Potential for ai-assisted medical education using large language models.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib10.9.1" style="font-size:90%;">
      PLoS digital health
     </em>
     <span class="ltx_text" id="bib.bib10.10.2" style="font-size:90%;">
      , 2(2):e0000198, 2023.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib11">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib11.5.5.1" style="font-size:90%;">
      Lee et al. (2020)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib11.7.1" style="font-size:90%;">
      Jinhyuk Lee, Wonjin Yoon, Sungdong Kim, Donghyeon Kim, Sunkyu Kim, Chan Ho So, and Jaewoo Kang.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib11.8.1" style="font-size:90%;">
      BioBERT: a pre-trained biomedical language representation model for biomedical text mining.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib11.9.1" style="font-size:90%;">
      Bioinformatics
     </em>
     <span class="ltx_text" id="bib.bib11.10.2" style="font-size:90%;">
      , 36(4):1234–1240, 2020.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib12">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib12.5.5.1" style="font-size:90%;">
      Li et al. (2024)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib12.7.1" style="font-size:90%;">
      Jianning Li, Amin Dada, Behrus Puladi, Jens Kleesiek, and Jan Egger.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib12.8.1" style="font-size:90%;">
      Chatgpt in healthcare: a taxonomy and systematic review.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib12.9.1" style="font-size:90%;">
      Computer Methods and Programs in Biomedicine
     </em>
     <span class="ltx_text" id="bib.bib12.10.2" style="font-size:90%;">
      , page 108013, 2024.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib13">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib13.5.5.1" style="font-size:90%;">
      Liu et al. (2023)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib13.7.1" style="font-size:90%;">
      Jialin Liu, Changyu Wang, and Siru Liu.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib13.8.1" style="font-size:90%;">
      Utility of chatgpt in clinical practice.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib13.9.1" style="font-size:90%;">
      Journal of Medical Internet Research
     </em>
     <span class="ltx_text" id="bib.bib13.10.2" style="font-size:90%;">
      , 25:e48568, 2023.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib14">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib14.5.5.1" style="font-size:90%;">
      Liévin et al. (2023)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib14.7.1" style="font-size:90%;">
      Valentin Liévin, Christoffer Egeberg Hother, and Ole Winther.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib14.8.1" style="font-size:90%;">
      Can large language models reason about medical questions?, 2023.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib15">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib15.5.5.1" style="font-size:90%;">
      Long et al. (2023)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib15.7.1" style="font-size:90%;">
      Cai Long, Kayle Lowe, Jessica Zhang, André dos Santos, Alaa Alanazi, Daniel O’Brien, Erin Wright, and David Cote.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib15.8.1" style="font-size:90%;">
      A novel evaluation model for assessing chatgpt on otolaryngology–head and neck surgery certification examinations: Performance study.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib15.9.1" style="font-size:90%;">
      JMIR Medical Education
     </em>
     <span class="ltx_text" id="bib.bib15.10.2" style="font-size:90%;">
      , 10, 2023.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib15.11.1" style="font-size:90%;">
      URL
     </span>
     <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://api.semanticscholar.org/CorpusID:265110947" style="font-size:90%;" target="_blank" title="">
      https://api.semanticscholar.org/CorpusID:265110947
     </a>
     <span class="ltx_text" id="bib.bib15.12.2" style="font-size:90%;">
      .
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib16">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib16.5.5.1" style="font-size:90%;">
      Lu et al. (2022)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib16.7.1" style="font-size:90%;">
      Yingzhou Lu, Chiung-Ting Wu, Sarah J Parker, Zuolin Cheng, Georgia Saylor, Jennifer E Van Eyk, Guoqiang Yu, Robert Clarke, David M Herrington, and Yue Wang.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib16.8.1" style="font-size:90%;">
      COT: an efficient and accurate method for detecting marker genes among many subtypes.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib16.9.1" style="font-size:90%;">
      Bioinformatics Advances
     </em>
     <span class="ltx_text" id="bib.bib16.10.2" style="font-size:90%;">
      , 2(1):vbac037, 2022.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib17">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib17.5.5.1" style="font-size:90%;">
      Lu et al. (2024)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib17.7.1" style="font-size:90%;">
      Yingzhou Lu, Tianyi Chen, Nan Hao, Capucine Van Rechem, Jintai Chen, and Tianfan Fu.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib17.8.1" style="font-size:90%;">
      Uncertainty quantification and interpretability for clinical trial approval prediction.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib17.9.1" style="font-size:90%;">
      Health Data Science
     </em>
     <span class="ltx_text" id="bib.bib17.10.2" style="font-size:90%;">
      , 2024.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib18">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib18.5.5.1" style="font-size:90%;">
      Luo et al. (2022)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib18.7.1" style="font-size:90%;">
      Renqian Luo, Liai Sun, Yingce Xia, Tao Qin, Sheng Zhang, Hoifung Poon, and Tie-Yan Liu.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib18.8.1" style="font-size:90%;">
      Biogpt: generative pre-trained transformer for biomedical text generation and mining.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib18.9.1" style="font-size:90%;">
      Briefings in bioinformatics
     </em>
     <span class="ltx_text" id="bib.bib18.10.2" style="font-size:90%;">
      , 23(6):bbac409, 2022.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib19">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib19.4.4.1" style="font-size:90%;">
      OpenAI (2022)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib19.6.1" style="font-size:90%;">
      OpenAI.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib19.7.1" style="font-size:90%;">
      Introducing chatgpt, 2022.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib19.8.1" style="font-size:90%;">
      URL
     </span>
     <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://openai.com/blog/chatgpt" style="font-size:90%;" target="_blank" title="">
      https://openai.com/blog/chatgpt
     </a>
     <span class="ltx_text" id="bib.bib19.9.2" style="font-size:90%;">
      .
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib19.10.1" style="font-size:90%;">
      Accessed: 2023-05-11.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib20">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib20.4.4.1" style="font-size:90%;">
      OpenAI (2023)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib20.6.1" style="font-size:90%;">
      OpenAI.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib20.7.1" style="font-size:90%;">
      Gpt-4 technical report, 2023.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib21">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib21.5.5.1" style="font-size:90%;">
      Singhal et al. (2023a)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib21.7.1" style="font-size:90%;">
      Karan Singhal, Shekoofeh Azizi, Tao Tu, S Sara Mahdavi, Jason Wei, Hyung Won Chung, Nathan Scales, Ajay Tanwani, Heather Cole-Lewis, Stephen Pfohl, et al.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib21.8.1" style="font-size:90%;">
      Large language models encode clinical knowledge.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib21.9.1" style="font-size:90%;">
      Nature
     </em>
     <span class="ltx_text" id="bib.bib21.10.2" style="font-size:90%;">
      , 620(7972):172–180, 2023a.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib22">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib22.5.5.1" style="font-size:90%;">
      Singhal et al. (2023b)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib22.7.1" style="font-size:90%;">
      Karan Singhal, Tao Tu, Juraj Gottweis, Rory Sayres, Ellery Wulczyn, Le Hou, Kevin Clark, Stephen Pfohl, Heather Cole-Lewis, Darlene Neal, et al.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib22.8.1" style="font-size:90%;">
      Towards expert-level medical question answering with large language models.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib22.9.1" style="font-size:90%;">
      arXiv preprint arXiv:2305.09617
     </em>
     <span class="ltx_text" id="bib.bib22.10.2" style="font-size:90%;">
      , 2023b.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib23">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib23.4.4.1" style="font-size:90%;">
      Vilares and Gómez-Rodríguez (2019)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib23.6.1" style="font-size:90%;">
      David Vilares and Carlos Gómez-Rodríguez.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib23.7.1" style="font-size:90%;">
      HEAD-QA: A healthcare dataset for complex reasoning.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib23.8.1" style="font-size:90%;">
      In Anna Korhonen, David R. Traum, and Lluís Màrquez, editors,
     </span>
     <em class="ltx_emph ltx_font_italic" id="bib.bib23.9.2" style="font-size:90%;">
      Proceedings of the 57th Conference of the Association for Computational Linguistics, ACL 2019, Florence, Italy, July 28- August 2, 2019, Volume 1: Long Papers
     </em>
     <span class="ltx_text" id="bib.bib23.10.3" style="font-size:90%;">
      , pages 960–966. Association for Computational Linguistics, 2019.
     </span>
    </span>
    <span class="ltx_bibblock">
     <a class="ltx_ref" href="https:/doi.org/10.18653/V1/P19-1092" style="font-size:90%;" target="_blank" title="">
      10.18653/V1/P19-1092
     </a>
     <span class="ltx_text" id="bib.bib23.11.1" style="font-size:90%;">
      .
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib23.12.1" style="font-size:90%;">
      URL
     </span>
     <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://doi.org/10.18653/v1/p19-1092" style="font-size:90%;" target="_blank" title="">
      https://doi.org/10.18653/v1/p19-1092
     </a>
     <span class="ltx_text" id="bib.bib23.13.2" style="font-size:90%;">
      .
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib24">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib24.5.5.1" style="font-size:90%;">
      Wang et al. (2024)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib24.7.1" style="font-size:90%;">
      Yue Wang, Yingzhou Lu, Yinlong Xu, Zihan Ma, Hongxia Xu, Bang Du, Honghao Gao, and Jian Wu.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib24.8.1" style="font-size:90%;">
      Twin-gpt: Digital twins for clinical trials via large language model.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib24.9.1" style="font-size:90%;">
      arXiv preprint arXiv:2404.01273
     </em>
     <span class="ltx_text" id="bib.bib24.10.2" style="font-size:90%;">
      , 2024.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib25">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib25.5.5.1" style="font-size:90%;">
      Wishart et al. (2018)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib25.7.1" style="font-size:90%;">
      David S Wishart, Yannick D Feunang, An C Guo, Elvis J Lo, Ana Marcu, Jason R Grant, Tanvir Sajed, Daniel Johnson, Carin Li, Zinat Sayeeda, et al.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib25.8.1" style="font-size:90%;">
      Drugbank 5.0: a major update to the drugbank database for 2018.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib25.9.1" style="font-size:90%;">
      Nucleic acids research
     </em>
     <span class="ltx_text" id="bib.bib25.10.2" style="font-size:90%;">
      , 46(D1):D1074–D1082, 2018.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib26">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib26.5.5.1" style="font-size:90%;">
      Yao et al. (2022)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib26.7.1" style="font-size:90%;">
      Shunyu Yao, Jeffrey Zhao, Dian Yu, Nan Du, Izhak Shafran, Karthik Narasimhan, and Yuan Cao.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib26.8.1" style="font-size:90%;">
      React: Synergizing reasoning and acting in language models.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib26.9.1" style="font-size:90%;">
      arXiv preprint arXiv:2210.03629
     </em>
     <span class="ltx_text" id="bib.bib26.10.2" style="font-size:90%;">
      , 2022.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib27">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib27.5.5.1" style="font-size:90%;">
      Yue et al. (2024)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib27.7.1" style="font-size:90%;">
      Ling Yue, Jonathan Li, Md Zabirul Islam, Bolun Xia, Tianfan Fu, and Jintai Chen.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib27.8.1" style="font-size:90%;">
      Trialdura: Hierarchical attention transformer for interpretable clinical trial duration prediction.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib27.9.1" style="font-size:90%;">
      arxiv
     </em>
     <span class="ltx_text" id="bib.bib27.10.2" style="font-size:90%;">
      , 2024.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib28">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib28.5.5.1" style="font-size:90%;">
      Yunxiang et al. (2023)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib28.7.1" style="font-size:90%;">
      Li Yunxiang, Li Zihan, Zhang Kai, Dan Ruilong, and Zhang You.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib28.8.1" style="font-size:90%;">
      Chatdoctor: A medical chat model fine-tuned on llama model using medical domain knowledge.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib28.9.1" style="font-size:90%;">
      arXiv preprint arXiv:2303.14070
     </em>
     <span class="ltx_text" id="bib.bib28.10.2" style="font-size:90%;">
      , 2023.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib29">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib29.5.5.1" style="font-size:90%;">
      Zhang et al. (2021)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib29.7.1" style="font-size:90%;">
      Bai Zhang, Yi Fu, Zhen Zhang, Robert Clarke, Jennifer E Van Eyk, David M Herrington, and Yue Wang.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib29.8.1" style="font-size:90%;">
      DDN2.0: R and python packages for differential dependency network analysis of biological systems.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib29.9.1" style="font-size:90%;">
      bioRxiv
     </em>
     <span class="ltx_text" id="bib.bib29.10.2" style="font-size:90%;">
      , pages 2021–04, 2021.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib30">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib30.5.5.1" style="font-size:90%;">
      Zhang et al. (2020)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib30.7.1" style="font-size:90%;">
      Xingyao Zhang, Cao Xiao, Lucas M Glass, and Jimeng Sun.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib30.8.1" style="font-size:90%;">
      Deepenroll: Patient-trial matching with deep embedding and entailment prediction.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib30.9.1" style="font-size:90%;">
      In
     </span>
     <em class="ltx_emph ltx_font_italic" id="bib.bib30.10.2" style="font-size:90%;">
      Proceedings of The Web Conference 2020
     </em>
     <span class="ltx_text" id="bib.bib30.11.3" style="font-size:90%;">
      , pages 1029–1037, 2020.
     </span>
    </span>
   </li>
   <li class="ltx_bibitem" id="bib.bib31">
    <span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">
     <span class="ltx_text" id="bib.bib31.5.5.1" style="font-size:90%;">
      Zhou et al. (2022)
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib31.7.1" style="font-size:90%;">
      Denny Zhou, Nathanael Schärli, Le Hou, Jason Wei, Nathan Scales, Xuezhi Wang, Dale Schuurmans, Claire Cui, Olivier Bousquet, Quoc Le, et al.
     </span>
    </span>
    <span class="ltx_bibblock">
     <span class="ltx_text" id="bib.bib31.8.1" style="font-size:90%;">
      Least-to-most prompting enables complex reasoning in large language models.
     </span>
    </span>
    <span class="ltx_bibblock">
     <em class="ltx_emph ltx_font_italic" id="bib.bib31.9.1" style="font-size:90%;">
      arXiv preprint arXiv:2205.10625
     </em>
     <span class="ltx_text" id="bib.bib31.10.2" style="font-size:90%;">
      , 2022.
     </span>
    </span>
   </li>
  </ul>
 </section>
 <section class="ltx_appendix" id="A1">
  <h2 class="ltx_title ltx_title_appendix" style="font-size:90%;">
   <span class="ltx_tag ltx_tag_appendix">
    Appendix A
   </span>
  </h2>
  <section class="ltx_subsection" id="A1.SS1">
   <h3 class="ltx_title ltx_title_subsection" style="font-size:90%;">
    <span class="ltx_tag ltx_tag_subsection">
     A.1
    </span>
    Enrollment Model
   </h3>
   <section class="ltx_paragraph" id="A1.SS1.SSS0.Px1">
    <h5 class="ltx_title ltx_title_paragraph" style="font-size:90%;">
     Model Architecture
    </h5>
    <div class="ltx_para" id="A1.SS1.SSS0.Px1.p1">
     <p class="ltx_p" id="A1.SS1.SSS0.Px1.p1.1">
      <span class="ltx_text" id="A1.SS1.SSS0.Px1.p1.1.1" style="font-size:90%;">
       The architecture of the model consists of the following components
      </span>
     </p>
     <ul class="ltx_itemize" id="A1.I1">
      <li class="ltx_item" id="A1.I1.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I1.i1.p1">
        <p class="ltx_p" id="A1.I1.i1.p1.1">
         <span class="ltx_text" id="A1.I1.i1.p1.1.1" style="font-size:90%;">
          A transformer encoder layer that processes embeddings of inclusion and exclusion criteria, drugs, and diseases. This layer is designed to capture interactions across these different types of information, crucial for understanding the complexity of trial eligibility.
         </span>
        </p>
       </div>
      </li>
      <li class="ltx_item" id="A1.I1.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I1.i2.p1">
        <p class="ltx_p" id="A1.I1.i2.p1.1">
         <span class="ltx_text" id="A1.I1.i2.p1.1.1" style="font-size:90%;">
          A fully connected layer that maps the high-dimensional features from the transformer encoder to a single output, indicating the probability of successful enrollment.
         </span>
        </p>
       </div>
      </li>
      <li class="ltx_item" id="A1.I1.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I1.i3.p1">
        <p class="ltx_p" id="A1.I1.i3.p1.1">
         <span class="ltx_text" id="A1.I1.i3.p1.1.1" style="font-size:90%;">
          A sigmoid activation function applied to the output of the fully connected layer, converting it into a probability measure.
         </span>
        </p>
       </div>
      </li>
     </ul>
    </div>
   </section>
   <section class="ltx_paragraph" id="A1.SS1.SSS0.Px2">
    <h5 class="ltx_title ltx_title_paragraph" style="font-size:90%;">
     Data Processing and Feature Extraction
    </h5>
    <div class="ltx_para" id="A1.SS1.SSS0.Px2.p1">
     <p class="ltx_p" id="A1.SS1.SSS0.Px2.p1.1">
      <span class="ltx_text" id="A1.SS1.SSS0.Px2.p1.1.1" style="font-size:90%;">
       For each trial, the Enrollment Agent performs the following steps:
      </span>
     </p>
     <ul class="ltx_itemize" id="A1.I2">
      <li class="ltx_item" id="A1.I2.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I2.i1.p1">
        <p class="ltx_p" id="A1.I2.i1.p1.1">
         <span class="ltx_text ltx_font_bold" id="A1.I2.i1.p1.1.1" style="font-size:90%;">
          Criteria Segmentation:
         </span>
         <span class="ltx_text" id="A1.I2.i1.p1.1.2" style="font-size:90%;">
          The agent first segments the eligibility criteria into inclusion and exclusion categories. This segmentation allows for targeted analysis of factors that either qualify or disqualify potential participants.
         </span>
        </p>
       </div>
      </li>
      <li class="ltx_item" id="A1.I2.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I2.i2.p1">
        <p class="ltx_p" id="A1.I2.i2.p1.1">
         <span class="ltx_text ltx_font_bold" id="A1.I2.i2.p1.1.1" style="font-size:90%;">
          Embedding Generation:
         </span>
         <span class="ltx_text" id="A1.I2.i2.p1.1.2" style="font-size:90%;">
          Using the pretrained BioBERT model, the agent converts text data from the criteria, drugs, and diseases into dense vector representations. These embeddings capture deep semantic features that are essential for accurate model predictions.
         </span>
        </p>
       </div>
      </li>
      <li class="ltx_item" id="A1.I2.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I2.i3.p1">
        <p class="ltx_p" id="A1.I2.i3.p1.1">
         <span class="ltx_text ltx_font_bold" id="A1.I2.i3.p1.1.1" style="font-size:90%;">
          Feature Aggregation:
         </span>
         <span class="ltx_text" id="A1.I2.i3.p1.1.2" style="font-size:90%;">
          The embeddings are then aggregated and fed into the transformer encoder, which processes them to capture the complex dependencies among the criteria, drugs, and diseases.
         </span>
        </p>
       </div>
      </li>
     </ul>
    </div>
   </section>
   <section class="ltx_paragraph" id="A1.SS1.SSS0.Px3">
    <h5 class="ltx_title ltx_title_paragraph" style="font-size:90%;">
     Model Training and Evaluation
    </h5>
    <div class="ltx_para" id="A1.SS1.SSS0.Px3.p1">
     <p class="ltx_p" id="A1.SS1.SSS0.Px3.p1.1">
      <span class="ltx_text" id="A1.SS1.SSS0.Px3.p1.1.1" style="font-size:90%;">
       The model is trained on a dataset comprising historical trial data, where each record includes the eligibility criteria, associated drugs and diseases, and the trial outcome regarding enrollment success. The training process involves:
      </span>
     </p>
     <ul class="ltx_itemize" id="A1.I3">
      <li class="ltx_item" id="A1.I3.i1" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I3.i1.p1">
        <p class="ltx_p" id="A1.I3.i1.p1.1">
         <span class="ltx_text" id="A1.I3.i1.p1.1.1" style="font-size:90%;">
          Balancing the dataset to handle disparities in the number of successful and unsuccessful enrollments.
         </span>
        </p>
       </div>
      </li>
      <li class="ltx_item" id="A1.I3.i2" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I3.i2.p1">
        <p class="ltx_p" id="A1.I3.i2.p1.1">
         <span class="ltx_text" id="A1.I3.i2.p1.1.1" style="font-size:90%;">
          Employing a cross-entropy loss function adjusted for class imbalance, ensuring that the model accurately learns from both positive (successful enrollment) and negative (unsuccessful enrollment) examples.
         </span>
        </p>
       </div>
      </li>
      <li class="ltx_item" id="A1.I3.i3" style="list-style-type:none;">
       <span class="ltx_tag ltx_tag_item">
        •
       </span>
       <div class="ltx_para" id="A1.I3.i3.p1">
        <p class="ltx_p" id="A1.I3.i3.p1.1">
         <span class="ltx_text" id="A1.I3.i3.p1.1.1" style="font-size:90%;">
          Evaluation on the test dataset yielded an ROC-AUC score of 0.7037 and an accuracy of 0.7689, indicating effective class differentiation and prediction accuracy. This model exemplifies the integration of advanced NLP and neural architectures to improve clinical trial design and efficiency.
         </span>
        </p>
       </div>
      </li>
     </ul>
    </div>
   </section>
  </section>
  <section class="ltx_subsection" id="A1.SS2">
   <h3 class="ltx_title ltx_title_subsection" style="font-size:90%;">
    <span class="ltx_tag ltx_tag_subsection">
     A.2
    </span>
    Function Calling Definitions
   </h3>
   <div class="ltx_para" id="A1.SS2.p1">
    <p class="ltx_p" id="A1.SS2.p1.1">
     <span class="ltx_text" id="A1.SS2.p1.1.1" style="font-size:90%;">
      The function for retrieving drug information from the DrugBank database is defined as follows:
     </span>
    </p>
    <pre class="ltx_verbatim ltx_font_typewriter" id="A1.SS2.p1.2" style="font-size:90%;">
{
  "type": "function",
  "function": {
    "name": "retrieval_drugbank",
    "description": "Retrieves information about a drug from the DrugBank database
                    using the drug’s name as input.",
    "parameters": {
      "type": "object",
      "properties": {
        "drug_name": {
          "type": "string",
          "description": "The name of the drug."
        }
      },
      "required": ["drug_name"]
    }
  }
}
</pre>
   </div>
   <div class="ltx_para" id="A1.SS2.p2">
    <p class="ltx_p" id="A1.SS2.p2.1">
     <span class="ltx_text" id="A1.SS2.p2.1.1" style="font-size:90%;">
      The function for retrieving the path connecting the drug to the disease from the Hetionet Knowledge Graph is defined as follows:
     </span>
    </p>
   </div>
   <div class="ltx_para" id="A1.SS2.p3">
    <pre class="ltx_verbatim ltx_font_typewriter" id="A1.SS2.p3.1" style="font-size:90%;">
{
    "type": "function",
    "function": {
        "name": "retrieval_hetionet",
        "description": "
            Given the names of a drug and a disease, the model retrieves
            the path connecting the drug to the disease from
            the Hetionet Knowledge Graph.
            Hetionet is a comprehensive knowledge graph that integrates diverse
            biological information by connecting genes, diseases, compounds,
            and more into an interoperable framework.
            It structures real-world biomedical data into a network,
            facilitating advanced analysis and discovery of new insights
            into disease mechanisms, drug repurposing,
            and the genetic underpinnings of health and disease.",
        "parameters": {
            "type": "object",
            "properties": {
                "drug_name": {
                    "type": "string",
                    "description": "The drug name",
                },
                "disease_name": {
                    "type": "string",
                    "description": "The disease name",
                }
            },
            "required": ["drug_name", "disease_name"],
        },
    }
}
</pre>
   </div>
  </section>
 </section>
</article>
